
<html lang="en"     class="pb-page"  data-request-id="d2facd37-7eed-4987-aa8a-e3663f1d2f4a"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;issue:issue:10.1021/jmcmar.2018.61.issue-3;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/acs.jmedchem.7b00681;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Free Energy Methods in Drug Design: Prospects of “Alchemical Perturbation” in Medicinal Chemistry" /></meta><meta name="dc.Creator" content="Billy J.  Williams-Noonan" /></meta><meta name="dc.Creator" content="Elizabeth  Yuriev" /></meta><meta name="dc.Creator" content="David K.  Chalmers" /></meta><meta name="dc.Description" content="Underpinning all drug discovery projects is the interaction between a drug and its target, usually a protein. Thus, improved methods for predicting the magnitude of protein–ligand interactions have..." /></meta><meta name="Description" content="Underpinning all drug discovery projects is the interaction between a drug and its target, usually a protein. Thus, improved methods for predicting the magnitude of protein–ligand interactions have..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="August 4, 2017" /></meta><meta name="dc.Type" content="review-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.7b00681" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2017 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.7b00681" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.7b00681" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.7b00681" /></link>
        
    
    

<title>Free Energy Methods in Drug Design: Prospects of “Alchemical Perturbation” in Medicinal Chemistry | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.7b00681" /></meta><meta property="og:title" content="Free Energy Methods in Drug Design: Prospects of “Alchemical Perturbation” in Medicinal Chemistry" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b00681/20180201/images/large/jm-2017-00681c_0008.jpeg" /></meta><meta property="og:description" content="Underpinning all drug discovery projects is the interaction between a drug and its target, usually a protein. Thus, improved methods for predicting the magnitude of protein–ligand interactions have the potential to improve the efficiency of drug development. In this review, we describe the principles of free energy methods used for the calculation of protein–ligand binding free energies, the challenges associated with these methods, and recent advances developed to address these difficulties. We then present case studies from 2005 to 2017, each demonstrating that alchemical free energy methods can assist rational drug design projects. We conclude that alchemical methods are becoming a feasible reality in medicinal chemistry research due to improved computational resources and algorithms and that alchemical free energy predictions methods are close to becoming a mainstream tool for medicinal chemists." /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta><meta name="twitter:description" content="Underpinning all drug discovery projects is the interaction between a drug and its target, usually a protein. Thus, improved methods for predicting the magnitude of protein–ligand interactions have the potential to improve the efficiency of drug development. In this review, we describe the principles of free energy methods used for the calculation of protein–ligand binding free energies, the challenges associated with these methods, and recent advances developed to address these difficulties. We then present case studies from 2005 to 2017, each demonstrating that alchemical free energy methods can assist rational drug design projects. We conclude that alchemical methods are becoming a feasible reality in medicinal chemistry research due to improved computational resources and algorithms and that alchemical free energy predictions methods are close to becoming a mainstream tool for medicinal chemists." /></meta><meta name="twitter:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b00681/20180201/images/large/jm-2017-00681c_0008.jpeg" /></meta><meta name="twitter:title" content="Free Energy Methods in Drug Design: Prospects of “Alchemical Perturbation” in Medicinal Chemistry" /></meta>
        
            <meta property="og:type" content="Article" /></meta>
        

        
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.7b00681"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/ke1q~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-6ee29527b3d8ff00a106.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.7b00681">
            
        





































    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-12/achre4.2021.54.issue-12/20210615/achre4.2021.54.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-6/amrcda.2021.2.issue-6/20210625/amrcda.2021.2.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-6/aabmcb.2021.4.issue-6/20210621/aabmcb.2021.4.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-6/aaembp.2021.3.issue-6/20210622/aaembp.2021.3.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-6/aaemcq.2021.4.issue-6/20210628/aaemcq.2021.4.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-25/aamick.2021.13.issue-25/20210630/aamick.2021.13.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-6/aanmf6.2021.4.issue-6/20210625/aanmf6.2021.4.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-6/aapmcd.2021.3.issue-6/20210611/aapmcd.2021.3.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-6/abseba.2021.7.issue-6/20210614/abseba.2021.7.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-12/accacs.2021.11.issue-12/20210618/accacs.2021.11.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-6/acscii.2021.7.issue-6/20210623/acscii.2021.7.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-6/acbcct.2021.16.issue-6/20210618/acbcct.2021.16.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-3/achsc5.2021.28.issue-3/20210524/achsc5.2021.28.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-12/acncdm.2021.12.issue-12/20210616/acncdm.2021.12.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-6/aesccq.2021.5.issue-6/20210617/aesccq.2021.5.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-6/aelccp.2021.6.issue-6/20210611/aelccp.2021.6.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-6/aeecco.2021.1.issue-6/20210611/aeecco.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-6/aewcaa.2021.1.issue-6/20210611/aewcaa.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-5/afsthl.2021.1.issue-5/20210618/afsthl.2021.1.issue-5.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-6/aidcbc.2021.7.issue-6/20210611/aidcbc.2021.7.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-6/amlccd.2021.10.issue-6/20210615/amlccd.2021.10.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-6/amlcef.2021.3.issue-6/20210607/amlcef.2021.3.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-6/amclct.2021.12.issue-6/20210610/amclct.2021.12.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-6/ancac3.2021.15.issue-6/20210622/ancac3.2021.15.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-25/acsodf.2021.6.issue-25/20210629/acsodf.2021.6.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-6/apchd5.2021.8.issue-6/20210616/apchd5.2021.8.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-6/ascefj.2021.6.issue-6/20210625/ascefj.2021.6.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-25/ascecg.2021.9.issue-25/20210628/ascecg.2021.9.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-6/asbcd6.2021.10.issue-6/20210618/asbcd6.2021.10.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-25/ancham.2021.93.issue-25/20210629/ancham.2021.93.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-25/bichaw.2021.60.issue-25/20210629/bichaw.2021.60.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-6/bcches.2021.32.issue-6/20210616/bcches.2021.32.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-6/bomaf6.2021.22.issue-6/20210614/bomaf6.2021.22.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-24/cgeabj.2021.99.issue-24/20210628-01/cgeabj.2021.99.issue-24.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-6/crtoec.2021.34.issue-6/20210621/crtoec.2021.34.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-12/chreay.2021.121.issue-12/20210623/chreay.2021.121.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-12/cmatex.2021.33.issue-12/20210622/cmatex.2021.33.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-6/cgdefu.2021.21.issue-6/20210602/cgdefu.2021.21.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-12/enfuem.2021.35.issue-12/20210617/enfuem.2021.35.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-12/esthag.2021.55.issue-12/20210615/esthag.2021.55.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-6/estlcu.2021.8.issue-6/20210608/estlcu.2021.8.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-25/iecred.2021.60.issue-25/20210630/iecred.2021.60.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-12/inocaj.2021.60.issue-12/20210621/inocaj.2021.60.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-6/jaaucr.2021.1.issue-6/20210628/jaaucr.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-25/jafcau.2021.69.issue-25/20210630/jafcau.2021.69.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-6/jceaax.2021.66.issue-6/20210610/jceaax.2021.66.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-6/jceda8.2021.98.issue-6/20210608/jceda8.2021.98.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-6/jcisd8.2021.61.issue-6/20210628/jcisd8.2021.61.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-6/jctcce.2021.17.issue-6/20210608/jctcce.2021.17.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/jmcmar.2021.64.issue-12/20210624/jmcmar.2021.64.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-6/jnprdf.2021.84.issue-6/20210625/jnprdf.2021.84.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-12/joceah.2021.86.issue-12/20210618/joceah.2021.86.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-24/jpcafh.2021.125.issue-24/20210624/jpcafh.2021.125.issue-24.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-24/jpcbfk.2021.125.issue-24/20210624/jpcbfk.2021.125.issue-24.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-24/jpccck.2021.125.issue-24/20210624/jpccck.2021.125.issue-24.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-24/jpclcd.2021.12.issue-24/20210624/jpclcd.2021.12.issue-24.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-6/jprobs.2021.20.issue-6/20210604/jprobs.2021.20.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-25/jacsat.2021.143.issue-25/20210630/jacsat.2021.143.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-6/jamsef.2021.32.issue-6/20210602/jamsef.2021.32.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-25/langd5.2021.37.issue-25/20210629/langd5.2021.37.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-12/mamobx.2021.54.issue-12/20210622/mamobx.2021.54.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-6/mpohbp.2021.18.issue-6/20210607/mpohbp.2021.18.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-12/nalefd.2021.21.issue-12/20210623/nalefd.2021.21.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-12/orlef7.2021.23.issue-12/20210618/orlef7.2021.23.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-6/oprdfk.2021.25.issue-6/20210618/oprdfk.2021.25.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-12/orgnd7.2021.40.issue-12/20210628/orgnd7.2021.40.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.7b00681&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.7b00681&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.7b00681&amp;href=/doi/10.1021/acs.jmedchem.7b00681" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2018</span><span class="cit-fg-volume">, 61</span><span class="cit-fg-issue">, 3</span><span class="cit-fg-pageRange">, 638-649</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/61/3" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.7b00393" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Perspective</span><a href="/doi/10.1021/acs.jmedchem.7b01375" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Free Energy Methods in Drug Design: Prospects of “Alchemical Perturbation” in Medicinal Chemistry</span></h1><h2 class="article_header-subtitle"><span class="hlFld-SubTitle">Miniperspective</span></h2><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Billy+J.++Williams-Noonan">Billy J. Williams-Noonan</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Elizabeth++Yuriev">Elizabeth Yuriev</a></span><span class="author-xref-symbol "><sup>*</sup></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=David+K.++Chalmers">David K. Chalmers</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="loa-info-orcid"><a href="http://orcid.org/0000-0003-2366-569X" title="Orcid link"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img></a></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-text">Medicinal Chemistry, Monash Institute of Pharmaceutical Sciences, Monash University, 381 Royal Parade, Parkville, Victoria 3052, Australia</span></div><div class="corresp-info"><strong>*</strong>E.Y.: e-mail, <a href="/cdn-cgi/l/email-protection#f0959c998a9192958498de898582999586b09d9f9e918398de959485"><span class="__cf_email__" data-cfemail="f2979e9b88939097869adc8b87809b9784b29f9d9c93819adc979687">[email protected]</span></a></div><div class="corresp-info"><strong>*</strong>D.K.C.: e-mail, <a href="/cdn-cgi/l/email-protection#aacecbdcc3ce84c9c2cbc6c7cfd8d9eac7c5c4cbd9c284cfcedf"><span class="__cf_email__" data-cfemail="462227302f2268252e272a2b233435062b292827352e68232233">[email protected]</span></a></div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.7b00681&amp;href=/doi/10.1021%2Facs.jmedchem.7b00681" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2018</span></span><span class="cit-volume">, 61</span><span class="cit-issue">, 3</span><span class="cit-pageRange">, 638–649</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">July 26, 2017</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>7 May 2017</li><li><span class="item_label"><b>Published</b> online</span>4 August 2017</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 8 February 2018</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.7b00681" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.7b00681</a></div><div class="article_header-article-copyright"><strong>Copyright © 2017 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D638%26pageCount%3D12%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DBilly%2BJ.%2BWilliams-Noonan%252C%2BElizabeth%2BYuriev%252C%2BDavid%2BK.%2BChalmers%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D61%26issueNum%3D3%26contentID%3Dacs.jmedchem.7b00681%26title%3DFree%2BEnergy%2BMethods%2Bin%2BDrug%2BDesign%253A%2BProspects%2Bof%2B%25E2%2580%259CAlchemical%2BPerturbation%25E2%2580%259D%2Bin%2BMedicinal%2BChemistry%26numPages%3D12%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D649%26publicationDate%3DFebruary%2B2018">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.7b00681"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">4233</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">62</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.7b00681" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Free Energy Methods in Drug Design: Prospects of “Alchemical Perturbation” in Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Billy&quot;,&quot;last_name&quot;:&quot;J. Williams-Noonan&quot;},{&quot;first_name&quot;:&quot;Elizabeth&quot;,&quot;last_name&quot;:&quot;Yuriev&quot;},{&quot;first_name&quot;:&quot;David&quot;,&quot;last_name&quot;:&quot;K. Chalmers&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2017&quot;,&quot;month&quot;:&quot;08&quot;,&quot;day&quot;:&quot;04&quot;,&quot;issue&quot;:&quot;3&quot;,&quot;volume&quot;:&quot;61&quot;,&quot;pages&quot;:&quot;638-649&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.7b00681&quot;},&quot;abstract&quot;:&quot;Underpinning all drug discovery projects is the interaction between a drug and its target, usually a protein. Thus, improved methods for predicting the magnitude of protein–ligand interactions have the potential to improve the efficiency of drug development. In this review, we describe the principles of free energy methods used for the calculation of protein–ligand binding free energies, the challenges associated with these methods, and recent advances developed to address these difficulties. We then present case studies from 2005 to 2017, each demonstrating that alchemical free energy methods can assist rational drug design projects. We conclude that alchemical methods are becoming a feasible reality in medicinal chemistry research due to improved computational resources and algorithms and that alchemical free energy predictions methods are close to becoming a mainstream tool for medicinal chemists.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b00681&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b00681" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b00681&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b00681" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b00681&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b00681" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.7b00681&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b00681&amp;href=/doi/10.1021/acs.jmedchem.7b00681" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.7b00681" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.7b00681" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (3 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.7b00681&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.7b00681%26sid%3Dliteratum%253Aachs%26pmid%3D28745501%26genre%3Darticle%26aulast%3DWilliams-Noonan%26date%3D2018%26atitle%3DFree%2BEnergy%2BMethods%2Bin%2BDrug%2BDesign%253A%2BProspects%2Bof%2B%25E2%2580%259CAlchemical%2BPerturbation%25E2%2580%259D%2Bin%2BMedicinal%2BChemistry%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D61%26issue%3D3%26spage%3D638%26epage%3D649%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=290976" title="Free energy">Free energy</a>,</li><li><a href="/action/doSearch?ConceptID=292366" title="Ligands">Ligands</a>,</li><li><a href="/action/doSearch?ConceptID=291901" title="Chemical calculations">Chemical calculations</a>,</li><li><a href="/action/doSearch?ConceptID=291352" title="Screening assays">Screening assays</a>,</li><li><a href="/action/doSearch?ConceptID=292344" title="Electron correlation">Electron correlation</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/61/3" title="Go to Volume 61, Issue 3"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/jmcmar.2018.61.issue-3/20180208/jmcmar.2018.61.issue-3.largecover.jpg" alt="Go to Volume 61, Issue 3"/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b00681/20180201/images/medium/jm-2017-00681c_0008.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b00681/20180201/images/large/jm-2017-00681c_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00681&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Underpinning all drug discovery projects is the interaction between a drug and its target, usually a protein. Thus, improved methods for predicting the magnitude of protein–ligand interactions have the potential to improve the efficiency of drug development. In this review, we describe the principles of free energy methods used for the calculation of protein–ligand binding free energies, the challenges associated with these methods, and recent advances developed to address these difficulties. We then present case studies from 2005 to 2017, each demonstrating that alchemical free energy methods can assist rational drug design projects. We conclude that alchemical methods are becoming a feasible reality in medicinal chemistry research due to improved computational resources and algorithms and that alchemical free energy predictions methods are close to becoming a mainstream tool for medicinal chemists.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_03506" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_03506" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The hit to lead process in drug development commonly involves the synthesis of large numbers of individual compounds in an iterative process that aims to optimize compound potency and biopharmaceutical properties. When available, detailed structural information, such as can be obtained from X-ray crystal structures, can assist the design process by revealing how the ligand interacts with the protein and where a lead compound might be modified to create new or improved interactions. However, by itself, a structure of a protein–ligand complex does not directly provide information about the importance (strength) of each specific interaction or provide a quantitative estimate of how particular changes made to a ligand might alter affinity. Ultimately, the information that would best assist the medicinal chemist in the choice of the next synthetic target would be an (at least reasonably accurate) estimate of the compound affinity. Reliable estimates of compound binding properties prior to synthesis would both enable better selection of compounds for synthesis and reduce the amount of effort required to produce potent leads with appropriate biopharmaceutical properties.</div><div class="NLM_p last">Dependable methods for the prediction of ligand binding poses and affinities remain a key goal of computational medicinal chemistry, and a wide variety of methods of varying quality are available, including docking,<a onclick="showRef(event, 'ref1 ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3">(1-3)</a> the linear interaction energy (LIE)<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> and linear response approximation (LRA)<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> methods, molecular mechanics Poisson–Boltzmann (MM-PBSA) simulations,<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> and free energy perturbation (FEP)<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> and other related alchemical perturbation methods. In this review, we focus specifically on alchemical perturbation methods with unbiased and unaccelerated sampling and comment on their scope, accuracy, and reliability in medicinal chemistry. We complete this review with seven case studies from 2005 to 2017 to the expected scope and accuracy of alchemical calculations.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a></div></div><div id="sec1_1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Free Energy of Ligand Binding</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_34738" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_34738" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The process of a small molecule ligand binding to a target macromolecule (for simplicity, we will assume that the target is a protein) is a reaction where the ligand is transferred from the free state in solution, where it interacts with the solvent, to the bound state, where it makes new, generally noncovalent, bonds to the protein as shown in <a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>. The free and bound macrostates can be subdivided into collections of discrete microstates, different conformations and relative orientations of both the protein and the ligand.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b00681/20180201/images/medium/jm-2017-00681c_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b00681/20180201/images/large/jm-2017-00681c_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Equilibrium between bound and unbound states of a system containing a protein and ligand, shown subdivided into microstates. The protein is green, the ligand is brown, and the solvent is pale blue. The observed affinity of the ligand for the bound state depends on the total numbers of bound and free states (<i>n</i><sub>bound</sub> and <i>n</i><sub>free</sub>) and the energy of each state.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b00681/20180201/images/large/jm-2017-00681c_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00681&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The partition function (<i>Z</i>) is a mathematical equation that characterizes the number and energies of microstates available to a thermodynamic system. A partition function can be constructed for a particular macrostate (e.g., <i>x</i> corresponds to “bound” or ‘free) as shown in <a class="ref internalNav" href="#eq1" aria-label="eq 1">eq 1</a>;<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> here <i>n</i><sub><i>x</i></sub> is the total number of microstates belonging to macrostate <i>x</i>, <i>E</i><sub><i>i</i></sub> is the internal energy of microstate <i>i, k</i><sub>b</sub> is Boltzmann’s constant, and <i>T</i> is the temperature of the system in kelvin. The partition functions of the bound and free microstates can be used to obtain the free energy of binding using the relationship shown in <a class="ref internalNav" href="#eq2" aria-label="eq 2">eq 2</a>.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a><span class="NLM_disp-formula" id="eq1"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b00681/20180201/images/jm-2017-00681c_m001.gif" alt="" /></img><span class="labelSpan">(1)</span></span><span class="NLM_disp-formula" id="eq2"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b00681/20180201/images/jm-2017-00681c_m002.gif" alt="" /></img><span class="labelSpan">(2)</span></span></div><div class="NLM_p"><a class="ref internalNav" href="#eq1" aria-label="Equations 1">Equations 1</a> and <a class="ref internalNav" href="#eq2" aria-label="2">2</a> encapsulate the physical properties required to computationally estimate the free energy of a ligand binding to a protein. To obtain these properties, we must be able to model the following:<table class="listgroup" border="0" width="95%" list-type="label"><tr class="li1"><td valign="top">(1)</td><td colspan="5" valign="top" class="paragraph"><p class="last">Protein, ligand, and solvent structure and energies. This information is determined by the computational model of the molecular environment (usually a molecular mechanics force field).</p></td></tr><tr class="li1"><td valign="top">(2)</td><td colspan="5" valign="top" class="paragraph"><p class="last">The total number and energies of the <i>free states</i>, including the consideration of ligand conformations in solution, the “relaxed” state of the protein binding site, and interactions of ligand and binding site with solvent.</p></td></tr><tr class="li1"><td valign="top">(3)</td><td colspan="5" valign="top" class="paragraph"><p class="last">The total number and energies of the <i>bound states</i>, requiring accurate modeling of protein–ligand interactions and any changes in protein conformation that may occur upon ligand binding.</p></td></tr></table></div><div class="NLM_p">Finding <i>all</i> the free and bound states of a protein–ligand system presents an enormous computational problem, but this is not actually required. The computational cost can be reduced by sampling only a subset of the available conformational space and by using this subset to estimate the partition function (<a class="ref internalNav" href="#eq1" aria-label="eq 1">eq 1</a>). The computational requirements are reduced further by the fact that the low energy conformations make the largest contribution to the partition function, meaning that reasonable estimates for the partition function can be obtained even if the high-energy conformations are neglected.</div><div class="NLM_p last">A wide variety of approaches for estimating binding free energies have been developed that, to some extent, neglect or approximate the factors set out in points 1–3 above. For example, end point approximation methods (e.g., LIE, LRA, MM-PBSA) compute approximate binding free energies by subtracting energies of the bound and free states.<a onclick="showRef(event, 'ref4 ref5 ref6'); return false;" href="javascript:void(0);" class="ref ref4 ref5 ref6">(4-6)</a> These methods often overestimate the enthalpic benefit to ligand binding (by inadequately accounting for water movement) while underestimating the entropic penalty to binding (due to undersampling of the relevant conformers).<a onclick="showRef(event, 'ref4 ref5 ref6'); return false;" href="javascript:void(0);" class="ref ref4 ref5 ref6">(4-6)</a> As another example, steered molecular dynamics (steered MD) can be used to create a biasing force that pulls the ligand from the receptor and measure an unbinding free energy. This approach generally undersamples the relevant protein–ligand states across the unbinding pathway, leading to errors in the computed binding free energy (Δ<i>G</i>).<a onclick="showRef(event, 'ref10 ref11 ref12 ref13'); return false;" href="javascript:void(0);" class="ref ref10 ref11 ref12 ref13">(10-13)</a></div></div><div id="sec1_2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Molecular Dynamics Simulations</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_02536" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_02536" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Molecular dynamics (MD) simulations model the movement of atoms and molecules within a system, most commonly using a molecular mechanics force field to compute the interactions within and between molecules (bonding, electrostatic, and van der Waals interactions).<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> MD is based on Newtonian principles, whereby atoms move in accordance with classical mechanical laws. In the drug discovery context, important applications of MD include exploration of low energy conformations of the protein–ligand complex and modeling the conformations of the ligand-free protein and the unbound ligand in solution. Accelerated or biased MD simulations can be used to improve conformational sampling or to drive simulations toward specific end points.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> Other sampling techniques, such as Monte Carlo simulations,<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> can also be used to explore the conformational landscape.</div><div class="NLM_p">The quality of MD simulations is predominantly determined by the accuracy of the model used to calculate the structure and energetics of the molecular system of interest (i.e., the force field). Since the early days of molecular modeling, there have been steady and systematic improvements made to the force fields for biological macromolecules, resulting in the currently available, well-tested force fields such as GROMOS,<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> AMBER,<a onclick="showRef(event, 'ref18 ref19'); return false;" href="javascript:void(0);" class="ref ref18 ref19">(18, 19)</a> and CHARMM,<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> which reproduce key features of biomolecule conformation and energetics.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> However, obtaining or developing good quality force field parameters for novel small molecules that are compatible with the available biomolecular force fields is frequently problematic<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> and represents a common technical limitation to modeling protein–ligand complexes. Fortunately, this problem is being reduced through the active development of tools for the parametrization of small molecules, including the Automated Topology Builder (ATB)<a onclick="showRef(event, 'ref23 ref24'); return false;" href="javascript:void(0);" class="ref ref23 ref24">(23, 24)</a> for the GROMOS force fields, the Antechamber package,<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> which creates Generalized Amber Force Field (GAFF)<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> parameters for small molecules compatible with existing Amber parameters, and the CGenFF package<a onclick="showRef(event, 'ref27 ref28'); return false;" href="javascript:void(0);" class="ref ref27 ref28">(27, 28)</a> for CHARMM.</div><div class="NLM_p">Biomolecular MD simulations are typically set up so that the molecules of interest are surrounded by a model environment consisting mostly of water but also including appropriate buffer ions and possibly a lipid bilayer in the case of membrane-bound proteins. Simulations generally use periodic boundary conditions, meaning that each simulation cell is surrounded by multiple copies of the primary cell, which eliminates the presence of any boundary between the molecules in the simulation and surrounding vacuum. Such a vacuum–solvent interface is undesirable for a number of reasons, but particularly because it causes surface effects such as reordering of solvent and surface tension that do not represent the behavior of bulk solvent correctly. <a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a> illustrates the case of a peptide in the primary cell (represented as a red surface) and nearby water atoms (represented as sticks) surrounded by identical replicate cells. This system uses a cubic periodic cell, although other space filling polyhedra such as the rhombic dodecahedron or cubic octahedron can be used. Although they provide an improved model of bulk solution, periodic boundary conditions do themselves introduce other types of artifacts into simulations, particularly if there are strong interactions between a molecule and its periodic neighbors.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a></div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b00681/20180201/images/medium/jm-2017-00681c_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b00681/20180201/images/large/jm-2017-00681c_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Cubic periodic boundary conditions. A peptide (red) and nearby water molecules are surrounded by the 26 nearest replicates of the main cell.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b00681/20180201/images/large/jm-2017-00681c_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00681&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">During MD simulations, properties of state (temperature, pressure, volume, etc.) are regulated by the MD algorithm to maintain the system in a particular thermodynamic ensemble. Biomolecular simulations are most commonly performed using either the canonical or isothermal–isobaric thermodynamic ensemble.<a onclick="showRef(event, 'ref30 ref31'); return false;" href="javascript:void(0);" class="ref ref30 ref31">(30, 31)</a> Both systems maintain a constant number of particles (no atoms are created or destroyed) and regulate temperature by an algorithm that models contact with a large heat bath (a thermostat). In the canonical ensemble, the system is of a fixed volume (the simulation cell is of fixed dimensions). In the isothermal–isobaric system a constant average pressure is maintained by varying the cell size under the control of a barostat algorithm.</div></div><div id="sec1_3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i4">Steered Molecular Dynamics and Umbrella Sampling</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_73115" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_73115" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">In principle, the free energy of a ligand-binding reaction can be derived from unbiased molecular dynamics simulations where the ligand is allowed to travel spontaneously from the solution to the binding site. Such simulations can be performed,<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> but in practice require very large numbers of long simulations to observe ligand binding events.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> Generally, it is computationally much more efficient to restrict the freedom of the ligand and constrain it to follow a binding (or unbinding) pathway. Free energies of binding can be calculated from such constrained simulations provided the energetics of the applied constraint are taken into account.</div><div class="NLM_p">The conceptually most straightforward approach to calculating the free energy of ligand binding is to measure the attractive force between the ligand and protein as the ligand is pulled directly out of the binding site. From the measured relationship between force and displacement, the total work, or free energy of binding, can be derived. The plot of average potential energy versus displacement is known as the potential of mean force (PMF). One way to calculate the PMF is by steered MD, whereby a force is applied over the course of the simulation that pulls the ligand away from the protein (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>A). The pathway out of the receptor must be explored for sufficient time to allow the system to reach equilibrium conditions throughout the pull trajectory. In practice, this often requires significant simulation time, and so steered MD is frequently conducted under nonequilibrium conditions. As a corrective measure, an equilibrium PMF can be derived from the nonequilibrium data using the Jarzynski relationship.<a onclick="showRef(event, 'ref10 ref11 ref12 ref13'); return false;" href="javascript:void(0);" class="ref ref10 ref11 ref12 ref13">(10-13)</a> Alternatively, a more reliable PMF can be also be obtained using an enhanced sampling method known as “umbrella sampling”,<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> which uses a series of equilibrium simulations, each with the ligand restrained at fixed positions moving away from the binding pocket (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>B).<a onclick="showRef(event, 'ref13 ref35 ref36'); return false;" href="javascript:void(0);" class="ref ref13 ref35 ref36">(13, 35, 36)</a> Harmonic (parabolic) constraints are generally used (the origin of “umbrella” in the name), which force the ligand to sample the higher energy regions of the unbinding pathway that would otherwise not be sampled by an unconstrained simulation.<a onclick="showRef(event, 'ref13 ref35 ref36'); return false;" href="javascript:void(0);" class="ref ref13 ref35 ref36">(13, 35, 36)</a> The complete PMF for the binding reaction is reassembled from the individual curves calculated from the discrete simulations using an approach such as the weighted histogram analysis method (WHAM).<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> Both steered MD and umbrella sampling are often used in ligand binding studies and provide good free energy estimates,<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> but care must be taken to avoid systematic errors that can arise from incomplete sampling of the available conformational space<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> or, in the case of umbrella sampling, poor overlap between the conformations sampled in adjacent simulations.<a onclick="showRef(event, 'ref38 ref40'); return false;" href="javascript:void(0);" class="ref ref38 ref40">(38, 40)</a></div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b00681/20180201/images/medium/jm-2017-00681c_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b00681/20180201/images/large/jm-2017-00681c_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Schematic representations of steered MD and umbrella sampling simulations for the calculation of the free energy of a ligand (red) binding to a receptor (green). Steered MD simulations use a continuous force to pull the ligand out of the receptor. Umbrella sampling calculations use multiple simulations with the ligand restrained at different distances from the receptor. The final potential of mean force is assembled from the smaller sections derived from each individual simulation.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b00681/20180201/images/large/jm-2017-00681c_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00681&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec1_4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i5">Alchemical Transformations</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_31825" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_31825" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Free energy perturbation calculations are often (ironically) described as “alchemical” because they make a nonphysical transformation of one compound into a related chemical species, for example, the transformation of ethane into methanol (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>A), or cause the “disappearance” of a molecule by conversion of normal atoms, which interact physically with the surrounding environment, into dummy atoms that make no intermolecular interactions and are invisible to the surrounding environment (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>B). Simulations of the type shown in <a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>A are used to determine the difference in free energy between the interaction of two molecules with the surrounding environment, for example, the difference in solvation energies of ethane and methanol. The conversion of a molecule into noninteracting dummy atoms, as shown in <a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>B, enables the calculation of the difference in free energy between a molecule within a particular environment and the molecule in a vacuum.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b00681/20180201/images/medium/jm-2017-00681c_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b00681/20180201/images/large/jm-2017-00681c_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Examples of alchemical molecular transformations. (A) Alchemical perturbation of ethane to methanol. C, H, O, and D represent carbon, hydrogen, oxygen, and noninteracting “dummy” atoms. (B) Transformation of ethane into dummy atoms that make intramolecular interactions but do not interact with the surrounding environment. In each case, the variable λ is used to represent the progression between initial and final states. In the intermediate λ-states (0 < λ < 1), the atom properties (charge, van der Waals interaction strength) are varied smoothly between those of the initial and final states.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b00681/20180201/images/large/jm-2017-00681c_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00681&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The free energy difference between states can be calculated using alchemical perturbation methods, where the ligand is progressively transformed from the initial to the final state over a number of discrete steps. In practice, the progression between states is characterized as a function of a variable (λ) which ranges between 0 (representing the initial state) and 1 (the final state). In the intermediate λ-states, the atomic properties (e.g., atomic partial charge and van der Waals interactions) are transformed smoothly (although not necessarily linearly) from those of the initial molecule to those of the final molecule such that there are no large free energy changes between adjacent steps. At each step, a molecular dynamics simulation is performed that must adequately sample the conformations available to the system and provide appreciable overlap with the conformations available to the simulations performed at the adjacent λ-state.<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a></div><div class="NLM_p">Having run simulations where one system is perturbed into another, a variety of methods are available to calculate the total free energy change, including the original “free energy perturbation” (FEP) approach devised by Zwanzig (1954).<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> In the literature today, a derivation of this method is commonly referred to as exponential averaging (DEXP or IEXP).<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> Other methods for the computation of the free energy difference between two states are thermodynamic integration (TI), the weighted histogram analysis method (WHAM), and the Bennett acceptance ratio (BAR) and its extension the multistate Bennett acceptance ratio (MBAR) approach.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> Although the current literature often generalizes the term FEP to refer to perturbation methods in general, in this review we retain the term FEP for the original approach of Zwanzig and refer to all described statistical mechanics-based methods, including FEP, as “alchemical free energy methods” or “alchemical perturbation” techniques.</div><div class="NLM_p last">In practice, the path taken from the initial to the final state and the number of intermediate steps (λ-states) can have a significant effect on the accuracy and efficiency of the alchemical perturbation calculation.<a onclick="showRef(event, 'ref44 ref45'); return false;" href="javascript:void(0);" class="ref ref44 ref45">(44, 45)</a> Literature reports<a onclick="showRef(event, 'ref46 ref47 ref48 ref49 ref50 ref51 ref52 ref53 ref54'); return false;" href="javascript:void(0);" class="ref ref46 ref47 ref48 ref49 ref50 ref51 ref52 ref53 ref54">(46-54)</a> suggest that in the bulk solvent, complete decoupling of a molecule requires between 10 and 30 intermediate λ-states. Similarly, between 30 and 50 λ-states are required when decoupling the ligand from the binding site,<a onclick="showRef(event, 'ref46 ref47 ref48 ref49 ref50 ref51 ref52 ref53 ref54'); return false;" href="javascript:void(0);" class="ref ref46 ref47 ref48 ref49 ref50 ref51 ref52 ref53 ref54">(46-54)</a> although the number of λ-states needed varies depending on the complexity of the change between states A and B. Charged or flexible ligands may require more λ-states or longer simulations.</div></div><div id="sec1_5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i6">Absolute and Relative Free Energies of Ligand Binding</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_74276" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_74276" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Free energy calculations can be performed in what is generally termed in the current literature as “absolute” or “relative” manners which are achieved using thermodynamic cycles such as those shown in <a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>. We note that both approaches calculate a <i>free energy difference</i> between two states (Δ<i>G</i> or ΔΔ<i>G</i>) and do not calculate the value of <i>G</i> itself, which would require calculation of the complete partition function which is possible only in very simple cases (see, for example, Skouteris et al.<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a>). The absolute approach (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>A) determines the change in free energy on moving a ligand from solution to the bound state (Δ<i>G</i><sub>bind(a)</sub>) and requires simulations that progressively decouple the bound ligand from the binding site and the free ligand from the solvent. The decoupled bound ligand and the decoupled free ligand are equivalent, so the free energy of transfer between these two states is 0. To avoid delinquent wandering of the ligand as all interactions to the protein are turned off, it may be necessary to constrain the position of the decoupled ligand.<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a> Alternatively, relative alchemical perturbation methods (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>B) calculate the free energy difference between the binding of a pair of ligands, ΔΔ<i>G</i><sub>bind(a→b)</sub>, by performing sets of simulations that transform one ligand into the other in the binding site and also in solution.</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b00681/20180201/images/medium/jm-2017-00681c_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b00681/20180201/images/large/jm-2017-00681c_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Thermodynamic cycles for the calculation of free energies of ligand binding. (A) Calculation of the absolute free energy of binding of ligand <b>a</b> (Δ<i>G</i><sub>bind(a)</sub>). The red ellipse represents ligand <b>a</b>, the green structure is the protein, and the dotted white ellipse is ligand <b>a</b> where all interactions are decoupled from the surrounding environment. (B) Thermodynamic cycle to calculate the relative free energy difference in binding of ligands <b>a</b> and <b>b (</b>ΔΔ<i>G</i><sub>bind(a→b)</sub><i>)</i>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b00681/20180201/images/large/jm-2017-00681c_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00681&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec1_6" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i7">Expected Accuracy of Alchemical Perturbation Methods</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_46126" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_46126" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">For a binding affinity value measured using a high quality experimental method, such as isothermal titration calorimetry (ITC), the noise is said to be within 0.3–0.5 kcal/mol,<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a> which suggests the RMSD from experiment for an absolute alchemical perturbation calculation cannot exceed this level of accuracy. Therefore, even alchemical calculations that are run for an extremely long time using supremely high-quality force are expected to deviate by at least 0.3 kcal/mol RMSD from experiment.</div><div class="NLM_p">Absolute free energy calculations often give binding free energies (Δ<i>G</i>) within 2–3 kcal/mol RMSD of experimental determinations for uncharged ligands and up to 4 kcal/mol RMSD for flexible or charged ligands.<a onclick="showRef(event, 'ref7 ref46 ref47 ref48 ref49 ref50 ref51 ref52 ref53 ref54'); return false;" href="javascript:void(0);" class="ref ref7 ref46 ref47 ref48 ref49 ref50 ref51 ref52 ref53 ref54">(7, 46-54)</a> Comparatively, relative free energy calculations, which yield ΔΔ<i>G</i> values, are typically more accurate<a onclick="showRef(event, 'ref13 ref58 ref59 ref60 ref61 ref62 ref63 ref64 ref65 ref66 ref67'); return false;" href="javascript:void(0);" class="ref ref13 ref58 ref59 ref60 ref61 ref62 ref63 ref64 ref65 ref66 ref67">(13, 58-67)</a> because the initial and final states are generally similar (i.e., the perturbation might be performed between two molecules having a common core and differing in only a single functional group). Similarity between initial and final states reduces sampling errors and provides greater accuracy.</div><div class="NLM_p">When translating errors in free energy calculations to a corresponding effect on the dissociation equilibrium constant <i>K</i><sub>D</sub>, one could question whether an RMSD of 2–3 kcal/mol from the experimental binding free energy is useful, given this level of error corresponds to quite large deviations in predicted <i>K</i><sub>D</sub>.<a onclick="showRef(event, 'ref9 ref13'); return false;" href="javascript:void(0);" class="ref ref9 ref13">(9, 13)</a> Indeed with errors of this magnitude, it is likely that the rank order of a series of ligands may not be reliably predicted using alchemical methods.<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a></div><div class="NLM_p">There are a number of sources of error in perturbation methods including, but not limited to, the quality of the force field parameters used, the treatment of electrostatic interactions, incomplete sampling, and methodological simplifications. Often, the largest sources of error associated with alchemical free energy methods come down to inadequacies in the force field and suboptimal sampling of the energy landscape between states A and B. In the case of charged ligands, it has been shown that the free energy of binding is influenced by a variety of factors, including the size and shape of the periodic cell, the treatment of electrostatic interactions, and the presence of counterions.<a onclick="showRef(event, 'ref68 ref69'); return false;" href="javascript:void(0);" class="ref ref68 ref69">(68, 69)</a> Error is also derived from statistical noise and errors intrinsic to the computational models used to represent the system.<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a> In addition to the inherent calculation errors, alchemical free energy methods suffer from time-related limitations. Specifically, protein conformational changes and ligand transformations between binding modes generally take much longer than the alchemical perturbation calculations themselves. Therefore, the alchemical free energy calculation may not accurately measure the change in free energy between state A and state B in instances where there is a high degree of protein movement<a onclick="showRef(event, 'ref38 ref47 ref51 ref70 ref71 ref72 ref73'); return false;" href="javascript:void(0);" class="ref ref38 ref47 ref51 ref70 ref71 ref72 ref73">(38, 47, 51, 70-73)</a> or where ligand behavior in the binding site is not sufficiently sampled. It may not be possible to simulate even small protein conformational changes or multiple ligand binding modes within the time span of a few nanoseconds per λ-state. Alchemical calculations in these cases can result in erroneous values for binding free energies.<a onclick="showRef(event, 'ref74 ref75'); return false;" href="javascript:void(0);" class="ref ref74 ref75">(74, 75)</a></div><div class="NLM_p">Despite the above limitations, alchemical free energy methods are still a powerful tool that can be highly effective when combined with current lead optimization methods.<a onclick="showRef(event, 'ref9 ref13'); return false;" href="javascript:void(0);" class="ref ref9 ref13">(9, 13)</a> Shirts et al.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> have proposed that binding free energy predictions within 2–3 kcal/mol RMSD from experiment can still significantly reduce the synthetic effort required for lead optimization. In a thought experiment, they showed that screening molecules based on simulations with 2.0 kcal/mol RMSD from experiment would reduce the number of compounds that had to be synthesized to provide a 10-fold dissociation constant improvement by a factor of 3.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> With higher accuracy (lower RMSD from experiment), the average number of compounds needed to produce a 10-fold dissociation constant improvement decreases even further.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a></div><div class="NLM_p">Undersampling in relative alchemical free energy calculations, and the ensuing lack of convergence, produces errors in the calculated free energy of interconversion between molecules (ΔΔ<i>G</i>). This error is termed “hysteresis”. Hysteresis errors (ξ) become evident in cases that contain a closed cycle that does not sum to zero, such as the simple system shown in <a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a> (A → B, B → C, C → A). Thus, closed cycles provide an effective means of verifying the accuracy of a relative alchemical perturbation calculation, since the sum of ΔΔ<i>G</i><sub>A→B,</sub> ΔΔ<i>G</i><sub>B→C</sub>, and ΔΔ<i>G</i><sub>C→A</sub> should be 0. If the sum of these three binding affinity differences has a nonzero value, then the residual error (if statistically significant) can be corrected for. In this simple case, the corrective term for each “edge” is ξ/3. In more complex cases containing multiple, interconnected cycles, the algorithm proposed by Wang et al. can be used to eliminate the hysteresis.<a onclick="showRef(event, 'ref76'); return false;" href="javascript:void(0);" class="ref ref76">(76)</a></div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b00681/20180201/images/medium/jm-2017-00681c_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b00681/20180201/images/large/jm-2017-00681c_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. A cycle of relative alchemical free energy calculations. The sides of each cycle are referred to as the edges and are depicted with an arrow. Each edge describes an alchemical perturbation from one ligand to another. If the sum of all three edges is a nonzero value, as is the case on the left, then this hysteresis (ξ) can be corrected to yield a cycle with zero hysteresis (ξ′).<a onclick="showRef(event, 'ref76'); return false;" href="javascript:void(0);" class="ref ref76">(76)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b00681/20180201/images/large/jm-2017-00681c_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00681&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec1_7" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i8">Alchemical Free Energy Methods for Binding Energy Determination: Success Stories</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_47315" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_47315" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Here we present selected studies taken from the period 2005 to 2017 that implement relative or absolute free energy calculations to predict protein–ligand binding free energies. (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>). We focus specifically on work that utilizes the alchemical methods already described, leaving aside a number of possible variations in methodology (although they may be useful and widely applicable). We do not cover applications of implicit models<a onclick="showRef(event, 'ref77'); return false;" href="javascript:void(0);" class="ref ref77">(77)</a> such as the generalized Born solvent model. Alchemical free energy methods have also been employed in protein mutation studies,<a onclick="showRef(event, 'ref78 ref79 ref80 ref81'); return false;" href="javascript:void(0);" class="ref ref78 ref79 ref80 ref81">(78-81)</a> where one residue is alchemically transformed to another, to determine what biological effect this would have, and variations on these methods have been successful in a medicinal chemistry context.<a onclick="showRef(event, 'ref82 ref83'); return false;" href="javascript:void(0);" class="ref ref82 ref83">(82, 83)</a> Further, one-step perturbation methods where the kinetic and potential energies of the system are not expressed as a function of λ have also demonstrated moderate to high success.<a onclick="showRef(event, 'ref84 ref85'); return false;" href="javascript:void(0);" class="ref ref84 ref85">(84, 85)</a> Monte Carlo methods can be used to sample the molecular structures rather than MD.<a onclick="showRef(event, 'ref86 ref87 ref88 ref89 ref90'); return false;" href="javascript:void(0);" class="ref ref86 ref87 ref88 ref89 ref90">(86-90)</a> Finally, we do not discuss quantum mechanical methods, which are arguably more accurate than molecular mechanics-based MD, but they are even slower to compute due to the increased complexity involved.<a onclick="showRef(event, 'ref91 ref92 ref93'); return false;" href="javascript:void(0);" class="ref ref91 ref92 ref93">(91-93)</a></div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. A Selection of Recent Alchemical Perturbation Studies</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">year</th><th class="colsep0 rowsep0" align="center">ref (first author)</th><th class="colsep0 rowsep0" align="center">alchemical method<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center">target protein</th><th class="colsep0 rowsep0" align="center">no. ligands<a class="ref internalNav" href="#t1fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center">no. perturbations<a class="ref internalNav" href="#t1fn3" aria-label="c">c</a></th><th class="colsep0 rowsep0" align="center">hysteresis correction<a class="ref internalNav" href="#t1fn4" aria-label="d">d</a></th><th class="colsep0 rowsep0" align="center">RMSD<a class="ref internalNav" href="#t1fn5" aria-label="e">e</a> (kcal/mol)</th><th class="colsep0 rowsep0" align="center"><i>R</i><a class="ref internalNav" href="#t1fn6" aria-label="f">f</a></th><th class="colsep0 rowsep0" align="center">ρ<a class="ref internalNav" href="#t1fn7" aria-label="g">g</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">2005</td><td class="colsep0 rowsep0" align="left">Fujitani<a onclick="showRef(event, 'ref94'); return false;" href="javascript:void(0);" class="ref ref94">(94)</a></td><td class="colsep0 rowsep0" align="left">Abs</td><td class="colsep0 rowsep0" align="left">FKBP</td><td class="colsep0 rowsep0" align="left">8</td><td class="colsep0 rowsep0" align="left">8</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">∼3.2<a class="ref internalNav" href="#t1fn8" aria-label="h">h</a></td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">∼1.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">2005</td><td class="colsep0 rowsep0" align="left">Archontis<a onclick="showRef(event, 'ref95'); return false;" href="javascript:void(0);" class="ref ref95">(95)</a></td><td class="colsep0 rowsep0" align="left">Rel</td><td class="colsep0 rowsep0" align="left">glycogen phosphorylase</td><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left">low</td><td class="colsep0 rowsep0" align="left">0.90</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">2006, 2008</td><td class="colsep0 rowsep0" align="left">Steinbrecher<a onclick="showRef(event, 'ref72 ref96'); return false;" href="javascript:void(0);" class="ref ref72 ref96">(72, 96)</a></td><td class="colsep0 rowsep0" align="left">Rel</td><td class="colsep0 rowsep0" align="left">human neutrophil elastase</td><td class="colsep0 rowsep0" align="left">4</td><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">1.00</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">2006</td><td class="colsep0 rowsep0" align="left">Jayachandran<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a></td><td class="colsep0 rowsep0" align="left">Abs</td><td class="colsep0 rowsep0" align="left">FKBP</td><td class="colsep0 rowsep0" align="left">8</td><td class="colsep0 rowsep0" align="left">8</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">1.60</td><td class="colsep0 rowsep0" align="left">0.72</td><td class="colsep0 rowsep0" align="left">0.88</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">2007</td><td class="colsep0 rowsep0" align="left">Mobley<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a></td><td class="colsep0 rowsep0" align="left">Abs</td><td class="colsep0 rowsep0" align="left">T4 lysozyme</td><td class="colsep0 rowsep0" align="left">13</td><td class="colsep0 rowsep0" align="left">26</td><td class="colsep0 rowsep0" align="left">low</td><td class="colsep0 rowsep0" align="left">1.89</td><td class="colsep0 rowsep0" align="left">0.8</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">2012</td><td class="colsep0 rowsep0" align="left">Palma<a onclick="showRef(event, 'ref65'); return false;" href="javascript:void(0);" class="ref ref65">(65)</a></td><td class="colsep0 rowsep0" align="left">Rel</td><td class="colsep0 rowsep0" align="left"><i>O</i>-methyltransferase</td><td class="colsep0 rowsep0" align="left">11</td><td class="colsep0 rowsep0" align="left">44</td><td class="colsep0 rowsep0" align="left">med</td><td class="colsep0 rowsep0" align="left">0.33<a class="ref internalNav" href="#t1fn9" aria-label="i">i</a></td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">2013</td><td class="colsep0 rowsep0" align="left">Rogers<a onclick="showRef(event, 'ref97'); return false;" href="javascript:void(0);" class="ref ref97">(97)</a></td><td class="colsep0 rowsep0" align="left">Rel</td><td class="colsep0 rowsep0" align="left">MshB</td><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">2013</td><td class="colsep0 rowsep0" align="left">Wang<a onclick="showRef(event, 'ref76'); return false;" href="javascript:void(0);" class="ref ref76">(76)</a></td><td class="colsep0 rowsep0" align="left">Rel</td><td class="colsep0 rowsep0" align="left">CDK2</td><td class="colsep0 rowsep0" align="left">16</td><td class="colsep0 rowsep0" align="left">18</td><td class="colsep0 rowsep0" align="left">high</td><td class="colsep0 rowsep0" align="left">0.64</td><td class="colsep0 rowsep0" align="left">0.82</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">2014</td><td class="colsep0 rowsep0" align="left">Lee<a onclick="showRef(event, 'ref98'); return false;" href="javascript:void(0);" class="ref ref98">(98)</a></td><td class="colsep0 rowsep0" align="left">Rel</td><td class="colsep0 rowsep0" align="left">GSK3β</td><td class="colsep0 rowsep0" align="left">18</td><td class="colsep0 rowsep0" align="left">17</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">1.10<a class="ref internalNav" href="#t1fn10" aria-label="j">j</a></td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">2015</td><td class="colsep0 rowsep0" align="left">Wang<a onclick="showRef(event, 'ref67'); return false;" href="javascript:void(0);" class="ref ref67">(67)</a></td><td class="colsep0 rowsep0" align="left">Rel</td><td class="colsep0 rowsep0" align="left">multiple</td><td class="colsep0 rowsep0" align="left">199</td><td class="colsep0 rowsep0" align="left">330</td><td class="colsep0 rowsep0" align="left">high</td><td class="colsep0 rowsep0" align="left">1.10</td><td class="colsep0 rowsep0" align="left">0.75</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">2015</td><td class="colsep0 rowsep0" align="left">Steinbrecher<a onclick="showRef(event, 'ref99'); return false;" href="javascript:void(0);" class="ref ref99">(99)</a></td><td class="colsep0 rowsep0" align="left">Rel</td><td class="colsep0 rowsep0" align="left">multiple</td><td class="colsep0 rowsep0" align="left">96</td><td class="colsep0 rowsep0" align="left">157</td><td class="colsep0 rowsep0" align="left">med</td><td class="colsep0 rowsep0" align="left">1.14</td><td class="colsep0 rowsep0" align="left">0.81</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">2016</td><td class="colsep0 rowsep0" align="left">Aldeghi<a onclick="showRef(event, 'ref100'); return false;" href="javascript:void(0);" class="ref ref100">(100)</a></td><td class="colsep0 rowsep0" align="left">Abs</td><td class="colsep0 rowsep0" align="left">BRD4</td><td class="colsep0 rowsep0" align="left">11</td><td class="colsep0 rowsep0" align="left">44</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">0.80</td><td class="colsep0 rowsep0" align="left">0.84</td><td class="colsep0 rowsep0" align="left">0.82</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">2016</td><td class="colsep0 rowsep0" align="left">Ciorda<a onclick="showRef(event, 'ref101'); return false;" href="javascript:void(0);" class="ref ref101">(101)</a></td><td class="colsep0 rowsep0" align="left">Rel</td><td class="colsep0 rowsep0" align="left">BACE1</td><td class="colsep0 rowsep0" align="left">50</td><td class="colsep0 rowsep0" align="left">62</td><td class="colsep0 rowsep0" align="left">high</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">0.67- 0.83</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">2016</td><td class="colsep0 rowsep0" align="left">Eelke<a onclick="showRef(event, 'ref102'); return false;" href="javascript:void(0);" class="ref ref102">(102)</a></td><td class="colsep0 rowsep0" align="left">Rel</td><td class="colsep0 rowsep0" align="left">four GPCRs</td><td class="colsep0 rowsep0" align="left">50</td><td class="colsep0 rowsep0" align="left">90</td><td class="colsep0 rowsep0" align="left">high</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">0.62</td><td class="colsep0 rowsep0" align="left">0.55</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">2016</td><td class="colsep0 rowsep0" align="left">Ioannidis<a onclick="showRef(event, 'ref103'); return false;" href="javascript:void(0);" class="ref ref103">(103)</a></td><td class="colsep0 rowsep0" align="left">Rel</td><td class="colsep0 rowsep0" align="left">influenza M2</td><td class="colsep0 rowsep0" align="left">13</td><td class="colsep0 rowsep0" align="left">14</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">0.73–0.94</td><td class="colsep0 rowsep0" align="left">0.76–0.92</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">2017</td><td class="colsep0 rowsep0" align="left">Bhati<a onclick="showRef(event, 'ref104'); return false;" href="javascript:void(0);" class="ref ref104">(104)</a></td><td class="colsep0 rowsep0" align="left">Rel</td><td class="colsep0 rowsep0" align="left">multiple</td><td class="colsep0 rowsep0" align="left">∼50</td><td class="colsep0 rowsep0" align="left">55</td><td class="colsep0 rowsep0" align="left">low</td><td class="colsep0 rowsep0" align="left">0.80–1.40</td><td class="colsep0 rowsep0" align="left">0.80–0.90</td><td class="colsep0 rowsep0" align="left">0.80–0.91</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup>a</sup><p class="last">An absolute (Abs) or relative (Rel) alchemical perturbation protocol was used.</p></div><div class="footnote" id="t1fn2"><sup>b</sup><p class="last">Number of ligands in study.</p></div><div class="footnote" id="t1fn3"><sup>c</sup><p class="last">Number of perturbations used in the study.</p></div><div class="footnote" id="t1fn4"><sup>d</sup><p class="last">The possibility of nonconvergence was corrected using low, medium, or high effort corrective measures.</p></div><div class="footnote" id="t1fn5"><sup>e</sup><p class="last">The RMSD of the predicted values from equivalent high-quality experimental ones.</p></div><div class="footnote" id="t1fn6"><sup>f</sup><p class="last">Pearson’s coefficient of determination versus experiment.</p></div><div class="footnote" id="t1fn7"><sup>g</sup><p class="last">Spearman’s rank order correlation coefficient versus experimental data.</p></div><div class="footnote" id="t1fn8"><sup>h</sup><p class="last">A linear regression was performed between predicted and experimental data with a vertical shift of 3.2 kcal/mol, and it was found that this offset best explained the data. The rank-order of the compounds was correctly predicted despite this offset.</p></div><div class="footnote" id="t1fn9"><sup>i</sup><p class="last">Only three experimental differences were used in the generation of this value, and as such, it may not be reliable.</p></div><div class="footnote" id="t1fn10"><sup>j</sup><p class="last">Only one experimental difference was used in the generation of this value, and as such, it may not be reliable.<a onclick="showRef(event, 'ref98'); return false;" href="javascript:void(0);" class="ref ref98">(98)</a></p></div></div></div><div class="NLM_p"><a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a> presents a set of alchemical free energy studies taken from the recent literature ordered from oldest to most recent. Although this list is not exhaustive, it demonstrates the improvement in alchemical perturbation accuracy as methods, computing power, and models are being improved. Alchemical free energy studies from the period 2005 to around 2007 generally use only small numbers of ligands with a single target receptor. By comparison, a single 2015 report<a onclick="showRef(event, 'ref67'); return false;" href="javascript:void(0);" class="ref ref67">(67)</a> investigates the binding of 199 ligands, each as an inhibitor of one of eight targets, and replicates their binding affinities with high accuracy.</div><div id="sec1_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> “Computation of the Binding Affinities of Catechol-<i>O</i>-methyltransferase Inhibitors: Multisubstate Relative Free Energy Calculations” (2012)</h3><div class="NLM_p">Catechol-<i>O</i>-methyltransferase (COMT) is an enzyme that promotes the transfer of a methyl group from <i>S</i>-adenosyl-<span class="smallcaps smallerCapital">l</span>-methionine (SAM) to a variety of catecholamines and catechol-estrogens.<a onclick="showRef(event, 'ref65 ref105'); return false;" href="javascript:void(0);" class="ref ref65 ref105">(65, 105)</a> COMT is implicated in a variety of metabolic and psychiatric disease states and is the target of several drugs in clinical use. Palma et al. employed relative alchemical free energy techniques to calculate the difference in binding affinities between catechol and two known inhibitors of COMT: tolcapone and opicapone, the latter of which has extremely high, subpicomolar affinity.<a onclick="showRef(event, 'ref65'); return false;" href="javascript:void(0);" class="ref ref65">(65)</a> Opicapone was recently investigated in a phase III clinical trial and showed a significantly higher response rate than placebo.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a></div><div class="NLM_p">The authors began with catechol bound in complex with a binding site magnesium ion and perturbed the catechol structure over four steps to produce tolcapone. Catechol was also perturbed over seven steps to give the inhibitor opicapone.<a onclick="showRef(event, 'ref65'); return false;" href="javascript:void(0);" class="ref ref65">(65)</a> The total perturbation map starting from catechol and diverging into these two inhibitors is illustrated in <a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a>. Modeling of bound tolcapone, opicapone, and the intermediate alchemical points is complicated by the possibility of multiple alternate binding poses, caused by rotation about dihedral angles ϕ and θ (<a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a>). Molecular docking and molecular dynamics simulations were used to identify four feasible bound poses for opicapone. Two putative substates of tolcapone were found using a similar rationale. Subsequently, the free energy calculations were carried over four paths, progressing from catechol to each of the putative binding poses. ΔΔ<i>G</i> values for the conversion of catechol to tolcapone and opicapone were generated for each pose, and a Boltzmann weighted average of the energies of the relevant substates (two different torsion angles) was generated to give overall ΔΔ<i>G</i> values for each step of the alchemical route. The predicted ΔΔ<i>G</i> values were within 0.5 kcal/mol RMSD of experimentally determined free energy differences: this prediction was assessed by the authors to be at the theoretical limit of accuracy.</div><figure id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b00681/20180201/images/medium/jm-2017-00681c_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b00681/20180201/images/large/jm-2017-00681c_0007.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Perturbation map for the progressive transformation of catechol into tolcapone or opicapone.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b00681/20180201/images/large/jm-2017-00681c_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00681&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec1_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> “Accurate and Reliable Relative Ligand Binding Potency in Prospective Drug Discovery by Way of a Modern Free-Energy Calculation Protocol and Force Field” (2015)</h3><div class="NLM_p">This large study by Wang et al.<a onclick="showRef(event, 'ref67'); return false;" href="javascript:void(0);" class="ref ref67">(67)</a> gathers alchemical calculations for the binding of ligands to eight proteins involved in disease pathogenesis: BACE, CDK2, JNK1, MCL1, p38, PTP1B, thrombin, and Tyk2. A grand total of 330 relative alchemical free energy calculations using 199 ligands were performed, whereby one single ligand was often perturbed more than once into different ligands, with multiple permutations between pairs of ligands. A range of technical approaches were used to improve the accuracy and throughput of calculations, including enhanced sampling, a refined force field, and GPU-based acceleration of calculations. The overall methodology was termed FEP+.</div><div class="NLM_p">FEP+ generates ligand topologies using the LOMAP approach,<a onclick="showRef(event, 'ref106'); return false;" href="javascript:void(0);" class="ref ref106">(106)</a> which identifies the maximum common substructure (MCSS) of two or more molecules and generates a topology based on the common features.<a onclick="showRef(event, 'ref107'); return false;" href="javascript:void(0);" class="ref ref107">(107)</a> An improved version of the OPLS force field was implemented, producing lower RSMD error from experiment for solvation free energies than analogous force fields. Each simulation was typically performed for 5 ns per λ-state for 12 λ-states using free energy perturbation-replica exchange with solute tempering (FEP-REST). The REST algorithm<a onclick="showRef(event, 'ref76'); return false;" href="javascript:void(0);" class="ref ref76">(76)</a> runs concurrently with the production run and works by artificially decreasing the potential energy in the binding site, without altering the kinetic energy. This technique thus creates a “hot” region that allows for smoother, more rapid transitions between different possible binding modes, thereby reducing sampling times.</div><div class="NLM_p">The FEP+ protocol produced RMSDs of 1.1 kcal/mol relative to experimental binding affinity differences. The predicted and experimental binding affinities correlated with a Pearson’s <i>R</i> of 0.75, across all targets. For the separate targets utilized, in the best case scenario, the RMSD from experiment was 0.93 kcal/mol, while in the worst case scenario, the RMSD error was 1.41 kcal/mol, suggesting a consistent level accuracy across the different systems. A significant improvement in the result accuracy was attributed to improvements in the ligand force field.</div><div class="NLM_p last">This study<a onclick="showRef(event, 'ref67'); return false;" href="javascript:void(0);" class="ref ref67">(67)</a> is unusual in that it reports results from two prospective studies, where compounds have been synthesized based on alchemical perturbation calculations. Historically, alchemical free energy papers have focused primarily on retrospective predictions of binding free energies and do not usually provide analysis of the use of prospective predictions in the design of novel compounds. Although full details of the synthesized compounds are not provided and the analysis is brief, the authors report that the free energy predictions are in general reliable and that the use of alchemical perturbation calculations substantially enhances the number of synthesized potent compounds. Similarly, success at prospectively predicting protein–ligand binding free energies was more recently reported in a similar study utilizing the FEP+ protocol for ligands binding to several GPCRs.<a onclick="showRef(event, 'ref102'); return false;" href="javascript:void(0);" class="ref ref102">(102)</a></div></div><div id="sec1_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> “Accurate Binding Free Energy Predictions in Fragment Optimization” (2015)</h3><div class="NLM_p last">In this retrospective analysis, Steinbrecher et al. applied the FEP+ methodology to multiple chemical series developed in fragment-based drug design programs. This paper aggregates data from the binding of 200 fragment-based inhibitors of one of eight proteins: lysozyme, DNA ligase, Mcl-1, MUP-I, LDH, JAK-2, HSP90, and p38.<a onclick="showRef(event, 'ref99'); return false;" href="javascript:void(0);" class="ref ref99">(99)</a> In each case, good quality X-ray crystal structures were available. The authors replicated the binding affinities of each ligand to their respective targets within an RMSD of 1.14 kcal/mol from experiment with a Pearson’s <i>R</i><sup>2</sup> of 0.65 using relative alchemical perturbation. Alchemical free energy calculations were shown to be much more accurate than docking and MM-GBSA, which yielded a Pearson’s <i>R</i><sup>2</sup> between predicted and experimental binding affinities of 0.35 and 0.41, respectively. Hysteresis was shown to be statistically nonsignificant using the thermodynamic cycle closure approach used in the previous study<a onclick="showRef(event, 'ref67'); return false;" href="javascript:void(0);" class="ref ref67">(67)</a> and was not corrected for. In this way, the authors demonstrated that relative alchemical perturbation calculations performed using as little as 3–5 ns of simulation time per λ-state can be highly reliable, thereby adding to the pool of evidence that alchemical free energy methods are fast becoming ready for mainstream use in medicinal chemistry.</div></div><div id="sec1_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> “Accurate Calculation of the Absolute Free Energy of Binding for Drug Molecules” (2016)</h3><div class="NLM_p">Bromodomain proteins (BRDs) recognize the N-acetylated lysine residues present on the histone proteins that complex with DNA.<a onclick="showRef(event, 'ref108'); return false;" href="javascript:void(0);" class="ref ref108">(108)</a> Some subtypes of BRD have been linked to a variety of epigenetic disease states, including the incidence of multiple strains of cancer.<a onclick="showRef(event, 'ref108'); return false;" href="javascript:void(0);" class="ref ref108">(108)</a> Aldeghi et al. used absolute alchemical perturbation calculations to investigate the binding of a series of 11 inhibitors to BRD4, which is involved in the aberrant recognition and subsequent transcription of genes that produce apoptotic proteins.<a onclick="showRef(event, 'ref100'); return false;" href="javascript:void(0);" class="ref ref100">(100)</a> Absolute binding free energies were obtained for crystallographic BRD4-ligand complexes as well as for docked structures. This was done to ascertain the importance of using a high-quality starting pose in obtaining an accurate absolute free energy. A total of 42 λ-states were used in each simulation, and each was run for 10 ns, with 5 ns of data collection, all after a 1 ns equilibration phase.</div><div class="NLM_p last">For calculations starting with the docked pose, the overall RMSD from experiment was 1.4 kcal/mol, and correlation coefficients between predicted and experimental binding energies were Pearson’s <i>R</i> = 0.77 and Spearman’s ρ = 0.72, indicating a high level of predictive power. When starting from the crystal structure pose, the RMSD from experiment for the predictions improved to a 0.8 kcal/mol, and correlations were stronger, with <i>R</i> = 0.84 and ρ = 0.82, suggesting that, at least in this instance, the better the quality of the starting ligand pose is, the more accurate the result. These results reported by Aldeghi et al.<a onclick="showRef(event, 'ref100'); return false;" href="javascript:void(0);" class="ref ref100">(100)</a> are better than those obtained typically when using absolute alchemical perturbation methods, with more than 64% of the variance in the rank order of the tested ligands explained by the model in the best-case scenario and with the rest of the predictions being close to experimental values. This level of accuracy is quite rare for absolute free energies. Indeed it is generally more common for a relative free energy calculation to be this accurate, given the dynamic range of binding affinities being tested is quite broad.<a onclick="showRef(event, 'ref58 ref59 ref60 ref61 ref62 ref63 ref64 ref65 ref66 ref67'); return false;" href="javascript:void(0);" class="ref ref58 ref59 ref60 ref61 ref62 ref63 ref64 ref65 ref66 ref67">(58-67)</a></div></div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i14">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_26326" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_26326" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The papers discussed above clearly show that alchemical perturbation calculations have now reached a stage where they can be practically applied within drug development programs. In the most recent studies, alchemical free energy methods have been applied to large data sets with throughput that is sufficiently fast for practical application to compound design (multiple alchemical calculations per day). All of the reports above suggest that alchemical free energy methods are accurate, reliable, and robust, with relative alchemical perturbation methods being most concordant with experiment. Of the papers cited above, those with the best accuracy report results within 0.8–1.1 kcal/mol of experiment.<a onclick="showRef(event, 'ref67 ref99 ref100'); return false;" href="javascript:void(0);" class="ref ref67 ref99 ref100">(67, 99, 100)</a> One significant limitation of the current literature is that nearly all reports describe work that has been carried out in a retrospective manner, where ligand affinities are known and the binding conformations have usually been determined by X-ray crystallography. The prospective use of computational modeling is generally more difficult than retrospective analysis, and there is clearly a need for further evaluation of prospective studies such as that by Wang et al.<a onclick="showRef(event, 'ref67'); return false;" href="javascript:void(0);" class="ref ref67">(67)</a> From a technical point of view, there remains a definite need for more automated tools to set up and analyze alchemical free energy calculations so as to reduce the technical demands of these calculations and more readily allow their incorporation into the mainstream medicinal chemistry workflow. Improved software tools are currently being developed in commercial packages,<a onclick="showRef(event, 'ref67 ref99 ref109'); return false;" href="javascript:void(0);" class="ref ref67 ref99 ref109">(67, 99, 109)</a> and it is evident that there is a definite movement toward making alchemical free energy methods a more mainstream tool that requires less technical expertise.</div><div class="NLM_p last">Although the arrival of free energy calculations in medicinal chemistry has been predicted for more than 20 years,<a onclick="showRef(event, 'ref110'); return false;" href="javascript:void(0);" class="ref ref110">(110)</a> we leave the reader with the perspective that the widespread application of alchemical methods within medicinal chemistry programs is truly close. Over the past 2 decades, there have been great advances in the computer hardware and software necessary to perform complex free energy calculations and the development of sophisticated protein expression and crystallography methods that can provide good starting models for prospective calculations. Practical application of free energy methods for medicinal chemistry has been demonstrated in several of the papers discussed above, and the number of papers reporting the use of free energy calculations for large numbers of compounds is increasing. The current limitations on more widespread use perhaps lie more in aspects of practical implementation than by the technical limitations of the method. We expect that the take-up of free energy modeling in medicinal chemistry will be further encouraged by good quality reports of the use of free energy calculations in a prospective manner.</div></div><div class="NLM_back"><div class="NLM_notes" id="NOTES-d130e1495-autogenerated"><p class="first last">The authors declare no competing financial interest.</p></div><div class="article_supporting-info" id=""></div><div class="testing" data-doi="10.1021/acs.jmedchem.7b00681" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_86274" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_86274" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Elizabeth Yuriev</span> - <span class="hlFld-Affiliation affiliation">Medicinal Chemistry, Monash
Institute of Pharmaceutical Sciences, Monash
University, 381 Royal
Parade, Parkville, Victoria 3052, Australia</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#36535a5f4c575453425e184f43445f5340765b595857455e18535243"><span class="__cf_email__" data-cfemail="95f0f9fceff4f7f0e1fdbbece0e7fcf0e3d5f8fafbf4e6fdbbf0f1e0">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">David K. Chalmers</span> - <span class="hlFld-Affiliation affiliation">Medicinal Chemistry, Monash
Institute of Pharmaceutical Sciences, Monash
University, 381 Royal
Parade, Parkville, Victoria 3052, Australia</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-2366-569X" title="Orcid link">http://orcid.org/0000-0003-2366-569X</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#a9cdc8dfc0cd87cac1c8c5c4ccdbdae9c4c6c7c8dac187cccddc"><span class="__cf_email__" data-cfemail="2044415649440e4348414c4d455253604d4f4e4153480e454455">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Billy J. Williams-Noonan</span> - <span class="hlFld-Affiliation affiliation">Medicinal Chemistry, Monash
Institute of Pharmaceutical Sciences, Monash
University, 381 Royal
Parade, Parkville, Victoria 3052, Australia</span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><p>The authors declare no competing financial interest.<br /></br></p></li></ul></div><div class="bio" rid="ath1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="biographies">Biographies</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_17007" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_17007" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><figure id="ath1" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Billy J. Williams-Noonan</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00681&amp;id=BIO-d130e1474-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Billy J. Williams-Noonan</b> obtained a Bachelor of Science at Griffith University and undertook an honors year at Griffith University’s Institute for Glycomics. He is currently a doctoral student at the Monash Institute of Pharmaceutical Sciences where he is undertaking studies on the application of free energy methods in drug design.</p></figure></div><div class="bio" rid="ath2"><figure id="ath2" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Elizabeth Yuriev</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00681&amp;id=BIO-d130e1479-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Elizabeth Yuriev</b> is a Senior Lecturer at the Monash Institute of Pharmaceutical Sciences, Monash University and the Secretary of the Association of Molecular Modelers of Australasia. She obtained a Ph.D. in Computational/Bioinorganic Chemistry from Victoria University, Australia (1997). Her research interests include chemical education, protein and nucleic acid modeling, and peptide and carbohydrate docking. She is the author of a regular series of reviews of the molecular docking literature published in the <i>Journal of Molecular Recognition</i>.</p></figure></div><div class="bio" rid="ath3"><figure id="ath3" data-index="10" class="article__inlineFigure"><h2 class="fig-label">David K. Chalmers</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00681&amp;id=BIO-d130e1487-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>David K. Chalmers</b> is a Senior Lecturer within the Monash Institute of Pharmaceutical Sciences. Following his Ph.D. (University of Melbourne, 1993) he has pursued interests in drug design and delivery. He is particularly interested in the development of anti-infective compounds and in the molecular scale modeling of drug delivery processes.</p></figure></div><div class="ack" id="ACK-d130e1492-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i18">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_67287" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_67287" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">B.J.W.-N. acknowledges the support provided by Research Training Program (RTP) grant from the Australian Government.</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i19" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i19"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i20" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i20"> Abbreviations Used</h2><tr><td class="NLM_term">ATB</td><td class="NLM_def"><p class="first last">Automated Topology Builder</p></td></tr><tr><td class="NLM_term">BACE</td><td class="NLM_def"><p class="first last">β-secretase</p></td></tr><tr><td class="NLM_term">BACE1</td><td class="NLM_def"><p class="first last">β-secretase 1</p></td></tr><tr><td class="NLM_term">BAR</td><td class="NLM_def"><p class="first last">Bennett acceptance ratio</p></td></tr><tr><td class="NLM_term">BRD</td><td class="NLM_def"><p class="first last">bromodomain</p></td></tr><tr><td class="NLM_term">BRD4</td><td class="NLM_def"><p class="first last">bromodomain 4</p></td></tr><tr><td class="NLM_term">CDK2</td><td class="NLM_def"><p class="first last">cyclin dependent kinase 2</p></td></tr><tr><td class="NLM_term">CGenFF</td><td class="NLM_def"><p class="first last">CHARMM generalized force field</p></td></tr><tr><td class="NLM_term">COMT</td><td class="NLM_def"><p class="first last">catechol-<i>O</i>-methyltransferase</p></td></tr><tr><td class="NLM_term">DEXP</td><td class="NLM_def"><p class="first last">D-exponential averaging</p></td></tr><tr><td class="NLM_term">FEP</td><td class="NLM_def"><p class="first last">free energy perturbation</p></td></tr><tr><td class="NLM_term">FEP-REST</td><td class="NLM_def"><p class="first last">free energy perturbation replica exchange with solute tempering</p></td></tr><tr><td class="NLM_term">FKBP</td><td class="NLM_def"><p class="first last">FK506 binding protein</p></td></tr><tr><td class="NLM_term">GAFF</td><td class="NLM_def"><p class="first last">Ggeneralized AMBER force field</p></td></tr><tr><td class="NLM_term">GPCR</td><td class="NLM_def"><p class="first last">G-protein-coupled receptor</p></td></tr><tr><td class="NLM_term">GSK3β</td><td class="NLM_def"><p class="first last">glycogen synthase kinase 3β</p></td></tr><tr><td class="NLM_term">IEXP</td><td class="NLM_def"><p class="first last">I-exponential averaging</p></td></tr><tr><td class="NLM_term">influenza M2</td><td class="NLM_def"><p class="first last">influenza matrix 2</p></td></tr><tr><td class="NLM_term">ITC</td><td class="NLM_def"><p class="first last">isothermal titration calorimetry</p></td></tr><tr><td class="NLM_term">JNK1</td><td class="NLM_def"><p class="first last">c-Jun N-terminal kinase 1</p></td></tr><tr><td class="NLM_term">LIE</td><td class="NLM_def"><p class="first last">linear interaction energy</p></td></tr><tr><td class="NLM_term">LOMAP</td><td class="NLM_def"><p class="first last">lead optimization mapper</p></td></tr><tr><td class="NLM_term">LRA</td><td class="NLM_def"><p class="first last">linear response approximation</p></td></tr><tr><td class="NLM_term">MCL1</td><td class="NLM_def"><p class="first last">induced myeloid leukemia cell differentiation protein 1</p></td></tr><tr><td class="NLM_term">MCSS</td><td class="NLM_def"><p class="first last">maximum common substructure</p></td></tr><tr><td class="NLM_term">MD</td><td class="NLM_def"><p class="first last">molecular dynamics</p></td></tr><tr><td class="NLM_term">MM-GBSA</td><td class="NLM_def"><p class="first last">molecular mechanics generalized Born surface area</p></td></tr><tr><td class="NLM_term">MM-PBSA</td><td class="NLM_def"><p class="first last">molecular mechanics Poisson–Boltzmann surface area</p></td></tr><tr><td class="NLM_term">MshB</td><td class="NLM_def"><p class="first last">mycothiol biosynthesis protein</p></td></tr><tr><td class="NLM_term"><i>NPT</i></td><td class="NLM_def"><p class="first last">constant particles, pressure, temperature</p></td></tr><tr><td class="NLM_term">PMF</td><td class="NLM_def"><p class="first last">potential of mean force</p></td></tr><tr><td class="NLM_term">PTP1B</td><td class="NLM_def"><p class="first last">tyrosine-protein phosphatase 1B</p></td></tr><tr><td class="NLM_term">RMSD</td><td class="NLM_def"><p class="first last">root-mean-square deviation</p></td></tr><tr><td class="NLM_term">RTP</td><td class="NLM_def"><p class="first last">research training program</p></td></tr><tr><td class="NLM_term">SAM</td><td class="NLM_def"><p class="first last"><i>S</i>-adenosyl-<span class="smallcaps smallerCapital">l</span>-methionine</p></td></tr><tr><td class="NLM_term">TI</td><td class="NLM_def"><p class="first last">thermodynamic integration</p></td></tr><tr><td class="NLM_term">Tyk2</td><td class="NLM_def"><p class="first last">tyrosine kinase 2</p></td></tr><tr><td class="NLM_term">vdW</td><td class="NLM_def"><p class="first last">van der Waals</p></td></tr><tr><td class="NLM_term">WHAM</td><td class="NLM_def"><p class="first last">weighted histogram analysis method</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i21">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_07270" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_07270" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 110 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Yuriev, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Agostino, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ramsland, P. A.</span><span> </span><span class="NLM_article-title">Challenges and advances in computational docking: 2009 in review</span> <span class="citation_source-journal">J. Mol. Recognit.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x">, </span> <span class="NLM_fpage">149</span><span class="NLM_x">–</span> <span class="NLM_lpage">164</span><span class="refDoi"> DOI: 10.1002/jmr.1077</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1002%2Fjmr.1077" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=21360606" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC3MXisV2jsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2011&pages=149-164&author=E.+Yurievauthor=M.+Agostinoauthor=P.+A.+Ramsland&title=Challenges+and+advances+in+computational+docking%3A+2009+in+review&doi=10.1002%2Fjmr.1077"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Challenges and advances in computational docking: 2009 in review</span></div><div class="casAuthors">Yuriev, Elizabeth; Agostino, Mark; Ramsland, Paul A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Recognition</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">149-164</span>CODEN:
                <span class="NLM_cas:coden">JMORE4</span>;
        ISSN:<span class="NLM_cas:issn">0952-3499</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  Docking is a computational technique that places a small mol. (ligand) in the binding site of its macromol. target (receptor) and ests. its binding affinity.  This review addresses methodol. developments that have occurred in the docking field in 2009, with a particular focus on the more difficult, and sometimes controversial, aspects of this promising computational discipline.  These developments aim to address the main challenges of docking: receptor representation (such aspects as structural waters, side chain protonation, and, most of all, flexibility (from side chain rotation to domain movement)), ligand representation (protonation, tautomerism and stereoisomerism, and the effect of input conformation), as well as accounting for solvation and entropy of binding.  This review is strongly focused on docking advances in the context of drug design, specifically in virtual screening and fragment-based drug design.  Copyright © 2010 John Wiley & Sons, Ltd.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEIQpQeuq9f7Vg90H21EOLACvtfcHk0lh1hqX0cYPvNw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXisV2jsb8%253D&md5=214fb8189f4bb43675893ac6bdedf51d</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1002%2Fjmr.1077&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjmr.1077%26sid%3Dliteratum%253Aachs%26aulast%3DYuriev%26aufirst%3DE.%26aulast%3DAgostino%26aufirst%3DM.%26aulast%3DRamsland%26aufirst%3DP.%2BA.%26atitle%3DChallenges%2520and%2520advances%2520in%2520computational%2520docking%253A%25202009%2520in%2520review%26jtitle%3DJ.%2520Mol.%2520Recognit.%26date%3D2011%26volume%3D24%26spage%3D149%26epage%3D164%26doi%3D10.1002%2Fjmr.1077" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Yuriev, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ramsland, P. A.</span><span> </span><span class="NLM_article-title">Latest developments in molecular docking: 2010–2011 in review</span> <span class="citation_source-journal">J. Mol. Recognit.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">215</span><span class="NLM_x">–</span> <span class="NLM_lpage">239</span><span class="refDoi"> DOI: 10.1002/jmr.2266</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1002%2Fjmr.2266" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=23526775" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC3sXksFSrtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2013&pages=215-239&author=E.+Yurievauthor=P.+A.+Ramsland&title=Latest+developments+in+molecular+docking%3A+2010%E2%80%932011+in+review&doi=10.1002%2Fjmr.2266"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Latest developments in molecular docking: 2010-2011 in review</span></div><div class="casAuthors">Yuriev, Elizabeth; Ramsland, Paul A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Recognition</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">215-239</span>CODEN:
                <span class="NLM_cas:coden">JMORE4</span>;
        ISSN:<span class="NLM_cas:issn">0952-3499</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  The aim of docking is to accurately predict the structure of a ligand within the constraints of a receptor binding site and to correctly est. the strength of binding.  We discuss, in detail, methodol. developments that occurred in the docking field in 2010 and 2011, with a particular focus on the more difficult, and sometimes controversial, aspects of this promising computational discipline.  The main developments in docking in this period, covered in this review, are receptor flexibility, solvation, fragment docking, postprocessing, docking into homol. models, and docking comparisons.  Several new, or at least newly invigorated, advances occurred in areas such as nonlinear scoring functions, using machine-learning approaches.  This review is strongly focused on docking advances in the context of drug design, specifically in virtual screening and fragment-based drug design.  Where appropriate, we refer readers to exemplar case studies.  Copyright © 2013 John Wiley & Sons, Ltd.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGponcWWCcI4b7Vg90H21EOLACvtfcHk0ljdTKy58mspxA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXksFSrtrY%253D&md5=b712888ca6c7fd4142711fa252de2456</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1002%2Fjmr.2266&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjmr.2266%26sid%3Dliteratum%253Aachs%26aulast%3DYuriev%26aufirst%3DE.%26aulast%3DRamsland%26aufirst%3DP.%2BA.%26atitle%3DLatest%2520developments%2520in%2520molecular%2520docking%253A%25202010%25E2%2580%25932011%2520in%2520review%26jtitle%3DJ.%2520Mol.%2520Recognit.%26date%3D2013%26volume%3D26%26spage%3D215%26epage%3D239%26doi%3D10.1002%2Fjmr.2266" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Lohning, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Levonis, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Williams-Noonan, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schweiker, S.</span><span> </span><span class="NLM_article-title">A practical guide to molecular docking and homology modelling for medicinal chemists</span> <span class="citation_source-journal">Curr. Top. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">2023</span><span class="NLM_x">–</span> <span class="NLM_lpage">2040</span><span class="refDoi"> DOI: 10.2174/1568026617666170130110827</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.2174%2F1568026617666170130110827" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=28137238" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVShtr3J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2017&pages=2023-2040&author=A.+Lohningauthor=S.+Levonisauthor=B.+Williams-Noonanauthor=S.+Schweiker&title=A+practical+guide+to+molecular+docking+and+homology+modelling+for+medicinal+chemists&doi=10.2174%2F1568026617666170130110827"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">A Practical Guide to Molecular Docking and Homology Modelling for Medicinal Chemists</span></div><div class="casAuthors">Lohning, Anna E.; Levonis, Stephan M.; Williams-Noonan, Billy; Schweiker, Stephanie S.</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">2023-2040</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Elucidating details of the relationship between mol. structure and a particular biol. end point is essential for successful, rational drug discovery.  Mol. docking is a widely accepted tool for lead identification however, navigating the intricacies of the software can be daunting.  Our objective was therefore to provide a step-by-step guide for those interested in incorporating contemporary basic mol. docking and homol. modeling into their design strategy.  Three mol. docking programs, AutoDock4, SwissDock and Surflex-Dock, were compared in the context of a case study where a set of steroidal and non-steroidal ligands were docked into the human androgen receptor (hAR) using both rigid and flexible target atoms.  Metrics for comparison included how well each program predicted the X-ray structure orientation via root mean square deviation (rmsd), predicting known actives via ligand ranking and comparison to biol. data where available.  Benchmarking metrics were discussed in terms of identifying accurate and reliable results.  For cases where no three dimensional structure exists, we provided a practical example for creating a homol. model using Swiss-Model.  Results showed an rmsd between X-ray ligands from wild-type and mutant receptors and docked poses were 4.15Å and 0.83Å (SwissDock), 2.69Å and 8.80Å (AutoDock4) and 0.39Å and 0.71Å (Surflex-Dock) resp.  Surflex-Dock performed consistently well in pose prediction (less than 2Å) while Auto- Dock4 predicted known active non-steroidal antiandrogens most accurately.  Introducing flexibility into target atoms produced the largest degree of change in ligand ranking in Surflex-Dock.  We produced a viable homol. model of the P2X1 purireceptor for subsequent docking anal.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrDrBGiwd68d7Vg90H21EOLACvtfcHk0ljdTKy58mspxA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVShtr3J&md5=5dfe7ec9d9b9756e07b7cb04fee69145</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.2174%2F1568026617666170130110827&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1568026617666170130110827%26sid%3Dliteratum%253Aachs%26aulast%3DLohning%26aufirst%3DA.%26aulast%3DLevonis%26aufirst%3DS.%26aulast%3DWilliams-Noonan%26aufirst%3DB.%26aulast%3DSchweiker%26aufirst%3DS.%26atitle%3DA%2520practical%2520guide%2520to%2520molecular%2520docking%2520and%2520homology%2520modelling%2520for%2520medicinal%2520chemists%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2017%26volume%3D17%26spage%3D2023%26epage%3D2040%26doi%3D10.2174%2F1568026617666170130110827" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Aqvist, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marelius, J.</span><span> </span><span class="NLM_article-title">The linear interaction energy method for predicting ligand binding free energies</span> <span class="citation_source-journal">Comb. Chem. High Throughput Screening</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">613</span><span class="NLM_x">–</span> <span class="NLM_lpage">626</span><span class="refDoi"> DOI: 10.2174/1386207013330661</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.2174%2F1386207013330661" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=11812258" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BD38Xks1Oktg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2001&pages=613-626&author=J.+Aqvistauthor=J.+Marelius&title=The+linear+interaction+energy+method+for+predicting+ligand+binding+free+energies&doi=10.2174%2F1386207013330661"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">The linear interaction energy method for predicting ligand binding free energies</span></div><div class="casAuthors">Aqvist, Johan; Marelius, John</div><div class="citationInfo"><span class="NLM_cas:title">Combinatorial Chemistry and High Throughput Screening</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">613-626</span>CODEN:
                <span class="NLM_cas:coden">CCHSFU</span>;
        ISSN:<span class="NLM_cas:issn">1386-2073</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers</span>)
        </div><div class="casAbstract">An overview of the simplified linear interaction energy (LIE) method for calcn. of ligand binding free energies is given.  This method is based on force field estns. of the receptor-ligand interactions and thermal conformational sampling.  A notable feature is that the binding energetics can be predicted by considering only the intermol. interactions between the ligand and receptor.  The approxns. behind this approach are examd. and different parameterizations of the model are discussed.  In general, LIE type of methods appear particularly useful for computational drug lead optimization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2CwI46ka7i7Vg90H21EOLACvtfcHk0ljdTKy58mspxA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xks1Oktg%253D%253D&md5=53d20d4d9bfe7f03780d5e280d8c484d</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.2174%2F1386207013330661&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1386207013330661%26sid%3Dliteratum%253Aachs%26aulast%3DAqvist%26aufirst%3DJ.%26aulast%3DMarelius%26aufirst%3DJ.%26atitle%3DThe%2520linear%2520interaction%2520energy%2520method%2520for%2520predicting%2520ligand%2520binding%2520free%2520energies%26jtitle%3DComb.%2520Chem.%2520High%2520Throughput%2520Screening%26date%3D2001%26volume%3D4%26spage%3D613%26epage%3D626%26doi%3D10.2174%2F1386207013330661" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Makri, N.</span><span> </span><span class="NLM_article-title">The linear response approximation and its lowest order corrections: An influence functional approach</span> <span class="citation_source-journal">J. Phys. Chem. B</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">103</span><span class="NLM_x">, </span> <span class="NLM_fpage">2823</span><span class="NLM_x">–</span> <span class="NLM_lpage">2829</span><span class="refDoi"> DOI: 10.1021/jp9847540</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jp9847540" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADyaK1MXitVags70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=1999&pages=2823-2829&author=N.+Makri&title=The+linear+response+approximation+and+its+lowest+order+corrections%3A+An+influence+functional+approach&doi=10.1021%2Fjp9847540"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">The Linear Response Approximation and Its Lowest Order Corrections: An Influence Functional Approach</span></div><div class="casAuthors">Makri, Nancy</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry B</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">103</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">2823-2829</span>CODEN:
                <span class="NLM_cas:coden">JPCBFK</span>;
        ISSN:<span class="NLM_cas:issn">1089-5647</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A cumulant expansion of the influence functional entering the path integral representation of the reduced d. matrix is presented.  Truncation of the expansion at the quadratic terms allows mapping of a many-body environment onto an effective harmonic model with the same force autocorrelation function, recovering the conventional linear response limit.  It is shown that the latter becomes exact in the thermodn. limit if the bath consists of independent modes coupled to the system via coeffs. that scale appropriately with the size of the bath.  Low-order correction terms to the linear response approxn. are found to be given by multitime correlation functions of the force exerted by the bath on the system.  Within a classical approxn., such correction terms can be obtained from mol. dynamics calcns. and incorporated in the path integral, which can be evaluated by means of iterative procedures.  Numerical calcns. on an intrinsically nonlinear bath of two-level systems confirm the above behaviors and indicate that the deviations from the linear response approxn. are more significant at low temps. where quantum mech. effects play an important role to the bath dynamics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqn6ASqpCu_KbVg90H21EOLACvtfcHk0lj2htce3wip-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXitVags70%253D&md5=bf3eae88c2de1d43225304f77e24c441</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1021%2Fjp9847540&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp9847540%26sid%3Dliteratum%253Aachs%26aulast%3DMakri%26aufirst%3DN.%26atitle%3DThe%2520linear%2520response%2520approximation%2520and%2520its%2520lowest%2520order%2520corrections%253A%2520An%2520influence%2520functional%2520approach%26jtitle%3DJ.%2520Phys.%2520Chem.%2520B%26date%3D1999%26volume%3D103%26spage%3D2823%26epage%3D2829%26doi%3D10.1021%2Fjp9847540" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Hou, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, W.</span><span> </span><span class="NLM_article-title">Assessing the performance of the MM/PBSA and MM/GBSA methods. 1. The accuracy of binding free energy calculations based on molecular dynamics simulations</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">69</span><span class="NLM_x">–</span> <span class="NLM_lpage">82</span><span class="refDoi"> DOI: 10.1021/ci100275a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci100275a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVyrsrrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2011&pages=69-82&author=T.+Houauthor=J.+Wangauthor=Y.+Liauthor=W.+Wang&title=Assessing+the+performance+of+the+MM%2FPBSA+and+MM%2FGBSA+methods.+1.+The+accuracy+of+binding+free+energy+calculations+based+on+molecular+dynamics+simulations&doi=10.1021%2Fci100275a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Assessing the Performance of the MM/PBSA and MM/GBSA Methods. 1. The Accuracy of Binding Free Energy Calculations Based on Molecular Dynamics Simulations</span></div><div class="casAuthors">Hou, Tingjun; Wang, Junmei; Li, Youyong; Wang, Wei</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">69-82</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The Mol. Mechanics/Poisson-Boltzmann Surface Area (MM/PBSA) and the Mol. Mechanics/Generalized Born Surface Area (MM/GBSA) methods calc. binding free energies for macromols. by combining mol. mechanics calcns. and continuum solvation models.  To systematically evaluate the performance of these methods, the authors report here an extensive study of 59 ligands interacting with six different proteins.  First, the authors explored the effects of the length of the mol. dynamics (MD) simulation, ranging from 400 to 4800 ps, and the solute dielec. const. (1, 2, or 4) on the binding free energies predicted by MM/PBSA.  The following three important conclusions could be obsd.: (1) MD simulation length has an obvious impact on the predictions, and longer MD simulation is not always necessary to achieve better predictions.  (2) The predictions are quite sensitive to the solute dielec. const., and this parameter should be carefully detd. according to the characteristics of the protein/ligand binding interface.  (3) Conformational entropy often show large fluctuations in MD trajectories, and a large no. of snapshots are necessary to achieve stable predictions.  Next, the authors evaluated the accuracy of the binding free energies calcd. by three Generalized Born (GB) models.  The authors found that the GB model developed by Onufriev and Case was the most successful model in ranking the binding affinities of the studied inhibitors.  Finally, the authors evaluated the performance of MM/GBSA and MM/PBSA in predicting binding free energies.  The authors' results showed that MM/PBSA performed better in calcg. abs., but not necessarily relative, binding free energies than MM/GBSA.  Considering its computational efficiency, MM/GBSA can serve as a powerful tool in drug design, where correct ranking of inhibitors is often emphasized.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_kxNWs2s3grVg90H21EOLACvtfcHk0lj2htce3wip-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsVyrsrrK&md5=b9f5b7fab70a6cc76b115d202d718f87</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1021%2Fci100275a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci100275a%26sid%3Dliteratum%253Aachs%26aulast%3DHou%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DW.%26atitle%3DAssessing%2520the%2520performance%2520of%2520the%2520MM%252FPBSA%2520and%2520MM%252FGBSA%2520methods.%25201.%2520The%2520accuracy%2520of%2520binding%2520free%2520energy%2520calculations%2520based%2520on%2520molecular%2520dynamics%2520simulations%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2011%26volume%3D51%26spage%3D69%26epage%3D82%26doi%3D10.1021%2Fci100275a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Shirts, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mobley, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chodera, J. D.</span><span> </span><span class="NLM_article-title">Alchemical free energy calculations: ready for prime time?</span> <span class="citation_source-journal">Annu. Rep. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">41</span><span class="NLM_x">–</span> <span class="NLM_lpage">59</span><span class="refDoi"> DOI: 10.1016/S1574-1400(07)03004-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1016%2FS1574-1400%2807%2903004-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BD1cXkt1Wnsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2007&pages=41-59&author=M.+R.+Shirtsauthor=D.+L.+Mobleyauthor=J.+D.+Chodera&title=Alchemical+free+energy+calculations%3A+ready+for+prime+time%3F&doi=10.1016%2FS1574-1400%2807%2903004-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Alchemical free energy calculations: Ready for prime time?</span></div><div class="casAuthors">Shirts, Michael R.; Mobley, David L.; Chodera, John D.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Reports in Computational Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">41-59</span>CODEN:
                <span class="NLM_cas:coden">ARCCC3</span>;
        ISSN:<span class="NLM_cas:issn">1574-1400</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">This review discusses the recent literature on methodol. improvements in free energy computation that have made this possible.  It focuses on current research using the most common rigorously exact method, that of "alchem." transformation.  In an alchem. transformation, a chem. species is transformed into another via a pathway of nonphys. (alchem.) states.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQsadHGUu_brVg90H21EOLACvtfcHk0lj0gsjYxUXp2A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXkt1Wnsr8%253D&md5=e568b6f7dd6e879700946eefe0b3bfdb</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2FS1574-1400%2807%2903004-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1574-1400%252807%252903004-6%26sid%3Dliteratum%253Aachs%26aulast%3DShirts%26aufirst%3DM.%2BR.%26aulast%3DMobley%26aufirst%3DD.%2BL.%26aulast%3DChodera%26aufirst%3DJ.%2BD.%26atitle%3DAlchemical%2520free%2520energy%2520calculations%253A%2520ready%2520for%2520prime%2520time%253F%26jtitle%3DAnnu.%2520Rep.%2520Comput.%2520Chem.%26date%3D2007%26volume%3D3%26spage%3D41%26epage%3D59%26doi%3D10.1016%2FS1574-1400%2807%2903004-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Lees, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferreira, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Costa, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rocha, J.-F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oliveira, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lopes, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nunes, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soares-da-Silva, P.</span><span> </span><span class="NLM_article-title">Efficacy and safety of opicapone, a new COMT inhibitor, for the treatment of motor fluctuations in Parkinson’s disease patients: a phase III, randomized, double-blind, placebocontrolled, parallel-group study (BIPARK II)</span> <span class="citation_source-journal">J. Neurol. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">333</span><span class="NLM_x">, </span> <span class="NLM_fpage">e116</span><span class="refDoi"> DOI: 10.1016/j.jns.2013.07.391</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1016%2Fj.jns.2013.07.391" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=333&publication_year=2013&pages=e116&author=A.+Leesauthor=J.+Ferreiraauthor=R.+Costaauthor=J.-F.+Rochaauthor=C.+Oliveiraauthor=N.+Lopesauthor=T.+Nunesauthor=P.+Soares-da-Silva&title=Efficacy+and+safety+of+opicapone%2C+a+new+COMT+inhibitor%2C+for+the+treatment+of+motor+fluctuations+in+Parkinson%E2%80%99s+disease+patients%3A+a+phase+III%2C+randomized%2C+double-blind%2C+placebocontrolled%2C+parallel-group+study+%28BIPARK+II%29&doi=10.1016%2Fj.jns.2013.07.391"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.jns.2013.07.391&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jns.2013.07.391%26sid%3Dliteratum%253Aachs%26aulast%3DLees%26aufirst%3DA.%26aulast%3DFerreira%26aufirst%3DJ.%26aulast%3DCosta%26aufirst%3DR.%26aulast%3DRocha%26aufirst%3DJ.-F.%26aulast%3DOliveira%26aufirst%3DC.%26aulast%3DLopes%26aufirst%3DN.%26aulast%3DNunes%26aufirst%3DT.%26aulast%3DSoares-da-Silva%26aufirst%3DP.%26atitle%3DEfficacy%2520and%2520safety%2520of%2520opicapone%252C%2520a%2520new%2520COMT%2520inhibitor%252C%2520for%2520the%2520treatment%2520of%2520motor%2520fluctuations%2520in%2520Parkinson%25E2%2580%2599s%2520disease%2520patients%253A%2520a%2520phase%2520III%252C%2520randomized%252C%2520double-blind%252C%2520placebocontrolled%252C%2520parallel-group%2520study%2520%2528BIPARK%2520II%2529%26jtitle%3DJ.%2520Neurol.%2520Sci.%26date%3D2013%26volume%3D333%26spage%3De116%26doi%3D10.1016%2Fj.jns.2013.07.391" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Steinbrecher, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Labahn, A.</span><span> </span><span class="NLM_article-title">Towards accurate free energy calculations in ligand protein-binding studies</span> <span class="citation_source-journal">Curr. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">767</span><span class="NLM_x">–</span> <span class="NLM_lpage">785</span><span class="refDoi"> DOI: 10.2174/092986710790514453</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.2174%2F092986710790514453" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=20088755" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjs1Kkurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2010&pages=767-785&author=T.+Steinbrecherauthor=A.+Labahn&title=Towards+accurate+free+energy+calculations+in+ligand+protein-binding+studies&doi=10.2174%2F092986710790514453"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Towards accurate free energy calculations in ligand protein-binding studies</span></div><div class="casAuthors">Steinbrecher, Thomas; Labahn, Andreas</div><div class="citationInfo"><span class="NLM_cas:title">Current Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">767-785</span>CODEN:
                <span class="NLM_cas:coden">CMCHE7</span>;
        ISSN:<span class="NLM_cas:issn">0929-8673</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Cells contain a multitude of different chem. reaction paths running simultaneously and quite independently next to each other.  This amazing feat is enabled by mol. recognition, the ability of biomols. to form stable and specific complexes with each other and with their substrates.  A better understanding of this process, i.e. of the kinetics, structures and thermodn. properties of biomol. binding, would be invaluable in the study of biol. systems.  In addn., as the mode of action of many pharmaceuticals is based upon their inhibition or activation of biomol. targets, predictive models of small mol. receptor binding are very helpful tools in rational drug design.  Since the goal here is normally to design a new compd. with a high inhibition strength, one of the most important thermodn. properties is the binding free energy ΔG0.  The prediction of binding consts. has always been one of the major goals in the field of computational chem., because the ability to reliably assess a hypothetical compd.'s binding properties without having to synthesize it first would save a tremendous amt. of work.  The different approaches to this question range from fast and simple empirical descriptor methods to elaborate simulation protocols aimed at putting the computation of free energies onto a solid foundation of statistical thermodn.  While the later methods are still not suited for the screenings of thousands of compds. that are routinely performed in computational drug design studies, they are increasingly put to use for the detailed study of protein ligand interactions.  This review will focus on mol. mechanics force field based free energy calcns. and their application to the study of protein ligand interactions.  After a brief overview of other popular methods for the calcn. of free energies, we will describe recent advances in methodol. and a variety of exemplary studies of mol. dynamics simulation based free energy calcns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqIXr-1XFuGS7Vg90H21EOLACvtfcHk0lj0gsjYxUXp2A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjs1Kkurw%253D&md5=e06994fb21a04ba7bbc572daa97c91b6</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.2174%2F092986710790514453&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F092986710790514453%26sid%3Dliteratum%253Aachs%26aulast%3DSteinbrecher%26aufirst%3DT.%26aulast%3DLabahn%26aufirst%3DA.%26atitle%3DTowards%2520accurate%2520free%2520energy%2520calculations%2520in%2520ligand%2520protein-binding%2520studies%26jtitle%3DCurr.%2520Med.%2520Chem.%26date%3D2010%26volume%3D17%26spage%3D767%26epage%3D785%26doi%3D10.2174%2F092986710790514453" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Jarzynski, C.</span><span> </span><span class="NLM_article-title">Nonequilibrium equality for free energy differences</span> <span class="citation_source-journal">Phys. Rev. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">78</span><span class="NLM_x">, </span> <span class="NLM_fpage">2690</span><span class="NLM_x">–</span> <span class="NLM_lpage">2693</span><span class="refDoi"> DOI: 10.1103/PhysRevLett.78.2690</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1103%2FPhysRevLett.78.2690" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADyaK2sXisVSrt7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=78&publication_year=1997&pages=2690-2693&author=C.+Jarzynski&title=Nonequilibrium+equality+for+free+energy+differences&doi=10.1103%2FPhysRevLett.78.2690"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Nonequilibrium equality for free energy differences</span></div><div class="casAuthors">Jarzynski, C.</div><div class="citationInfo"><span class="NLM_cas:title">Physical Review Letters</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">78</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">2690-2693</span>CODEN:
                <span class="NLM_cas:coden">PRLTAO</span>;
        ISSN:<span class="NLM_cas:issn">0031-9007</span>.
    
            (<span class="NLM_cas:orgname">American Physical Society</span>)
        </div><div class="casAbstract">An expression is derived for the equil. free energy difference between two configurations of a system, in terms of an ensemble of finite-time measurements of the work performed in parametrically switching from one configuration to the other.  Two well-known identities emerge as limiting cases of this result.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqsl50isjLbR7Vg90H21EOLACvtfcHk0lj0gsjYxUXp2A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXisVSrt7c%253D&md5=5e572f64b7e9044fff5af96867602877</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1103%2FPhysRevLett.78.2690&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1103%252FPhysRevLett.78.2690%26sid%3Dliteratum%253Aachs%26aulast%3DJarzynski%26aufirst%3DC.%26atitle%3DNonequilibrium%2520equality%2520for%2520free%2520energy%2520differences%26jtitle%3DPhys.%2520Rev.%2520Lett.%26date%3D1997%26volume%3D78%26spage%3D2690%26epage%3D2693%26doi%3D10.1103%2FPhysRevLett.78.2690" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Jarzynski, C.</span><span> </span><span class="NLM_article-title">How does a system respond when driven away from thermal equilibrium?</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">98</span><span class="NLM_x">, </span> <span class="NLM_fpage">3636</span><span class="NLM_x">–</span> <span class="NLM_lpage">3638</span><span class="refDoi"> DOI: 10.1073/pnas.081074598</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1073%2Fpnas.081074598" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=2001&pages=3636-3638&author=C.+Jarzynski&title=How+does+a+system+respond+when+driven+away+from+thermal+equilibrium%3F&doi=10.1073%2Fpnas.081074598"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1073%2Fpnas.081074598&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.081074598%26sid%3Dliteratum%253Aachs%26aulast%3DJarzynski%26aufirst%3DC.%26atitle%3DHow%2520does%2520a%2520system%2520respond%2520when%2520driven%2520away%2520from%2520thermal%2520equilibrium%253F%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2001%26volume%3D98%26spage%3D3636%26epage%3D3638%26doi%3D10.1073%2Fpnas.081074598" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Hummer, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Szabo, A.</span><span> </span><span class="NLM_article-title">Free energy reconstruction from nonequilibrium single-molecule pulling experiments</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">98</span><span class="NLM_x">, </span> <span class="NLM_fpage">3658</span><span class="NLM_x">–</span> <span class="NLM_lpage">3661</span><span class="refDoi"> DOI: 10.1073/pnas.071034098</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1073%2Fpnas.071034098" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=11274384" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BD3MXis1Kitbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=2001&pages=3658-3661&author=G.+Hummerauthor=A.+Szabo&title=Free+energy+reconstruction+from+nonequilibrium+single-molecule+pulling+experiments&doi=10.1073%2Fpnas.071034098"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Free energy reconstruction from nonequilibrium single-molecule pulling experiments</span></div><div class="casAuthors">Hummer, Gerhard; Szabo, Attila</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">98</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3658-3661</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Laser tweezers and at. force microscopes are increasingly used to probe the interactions and mech. properties of individual mols.  Unfortunately, using such time-dependent perturbations to force rare mol. events also drives the system away from equil.  Nevertheless, we show how equil. free energy profiles can be extd. rigorously from repeated nonequil. force measurements on the basis of an extension of Jarzynski's remarkable identity between free energies and the irreversible work.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrs8JOlO8In77Vg90H21EOLACvtfcHk0lgI4OSAV2Mt7Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXis1Kitbk%253D&md5=b8b557f038efd3d60f9a97881636c120</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1073%2Fpnas.071034098&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.071034098%26sid%3Dliteratum%253Aachs%26aulast%3DHummer%26aufirst%3DG.%26aulast%3DSzabo%26aufirst%3DA.%26atitle%3DFree%2520energy%2520reconstruction%2520from%2520nonequilibrium%2520single-molecule%2520pulling%2520experiments%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2001%26volume%3D98%26spage%3D3658%26epage%3D3661%26doi%3D10.1073%2Fpnas.071034098" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Shirts, M. R.; Mobley, D. L.; Brown, S. P.</span><span> </span><span class="NLM_article-title">Free Energy Calculations In Structure-Based Drug Design</span>. In  <span class="citation_source-book">Drug Design: Structure- and Ligand-Based Approaches</span>; <span class="NLM_publisher-name">Cambridge University Press</span>,<span class="NLM_x"> </span><span class="NLM_year">2010</span><span class="NLM_x">; </span>pp  <span class="NLM_fpage">61</span><span class="NLM_x">–</span> <span class="NLM_lpage">86</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1017%2FCBO9780511730412.007" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2010&pages=61-86&author=M.+R.+Shirts&author=D.+L.+Mobley&author=S.+P.+Brown&title=Drug+Design%3A+Structure-+and+Ligand-Based+Approaches"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1017%2FCBO9780511730412.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1017%252FCBO9780511730412.007%26sid%3Dliteratum%253Aachs%26aulast%3DShirts%26aufirst%3DM.%2BR.%26atitle%3DFree%2520Energy%2520Calculations%2520In%2520Structure-Based%2520Drug%2520Design%26btitle%3DDrug%2520Design%253A%2520Structure-%2520and%2520Ligand-Based%2520Approaches%26pub%3DCambridge%2520University%2520Press%26date%3D2010%26spage%3D61%26epage%3D86" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Allen, M. P.</span><span> </span><span class="NLM_article-title">Introduction to Molecular Dynamics Simulation</span>. In  <span class="citation_source-book">Computational Soft Matter: From Synthetic Polymers to Proteins</span>; NIC Series, Vol. 23; <span class="NLM_publisher-name">NIC</span>,<span class="NLM_x"> </span><span class="NLM_year">2004</span><span class="NLM_x">; </span>pp  <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">28</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2004&pages=1-28&author=M.+P.+Allen&title=Computational+Soft+Matter%3A+From+Synthetic+Polymers+to+Proteins"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DAllen%26aufirst%3DM.%2BP.%26atitle%3DIntroduction%2520to%2520Molecular%2520Dynamics%2520Simulation%26btitle%3DComputational%2520Soft%2520Matter%253A%2520From%2520Synthetic%2520Polymers%2520to%2520Proteins%26pub%3DNIC%26date%3D2004%26spage%3D1%26epage%3D28" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Bernardi, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Melo, M. C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schulten, K.</span><span> </span><span class="NLM_article-title">Enhanced sampling techniques in molecular dynamics simulations of biological systems</span> <span class="citation_source-journal">Biochim. Biophys. Acta, Gen. Subj.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">1850</span><span class="NLM_x">, </span> <span class="NLM_fpage">872</span><span class="NLM_x">–</span> <span class="NLM_lpage">877</span><span class="refDoi"> DOI: 10.1016/j.bbagen.2014.10.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1016%2Fj.bbagen.2014.10.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=25450171" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVanurbN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1850&publication_year=2015&pages=872-877&author=R.+C.+Bernardiauthor=M.+C.+R.+Meloauthor=K.+Schulten&title=Enhanced+sampling+techniques+in+molecular+dynamics+simulations+of+biological+systems&doi=10.1016%2Fj.bbagen.2014.10.019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Enhanced sampling techniques in molecular dynamics simulations of biological systems</span></div><div class="casAuthors">Bernardi, Rafael C.; Melo, Marcelo C. R.; Schulten, Klaus</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, General Subjects</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">1850</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">872-877</span>CODEN:
                <span class="NLM_cas:coden">BBGSB3</span>;
        ISSN:<span class="NLM_cas:issn">0304-4165</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Mol. dynamics has emerged as an important research methodol. covering systems to the level of millions of atoms.  However, insufficient sampling often limits its application.  The limitation is due to rough energy landscapes, with many local min. sepd. by high-energy barriers, which govern the biomol. motion.  In the past few decades methods have been developed that address the sampling problem, such as replica-exchange mol. dynamics, metadynamics and simulated annealing.  Here the authors present an overview over theses sampling methods in an attempt to shed light on which should be selected depending on the type of system property studied.  Enhanced sampling methods have been employed for a broad range of biol. systems and the choice of a suitable method is connected to biol. and phys. characteristics of the system, in particular system size.  While metadynamics and replica-exchange mol. dynamics are the most adopted sampling methods to study biomol. dynamics, simulated annealing is well suited to characterize very flexible systems.  The use of annealing methods for a long time was restricted to simulation of small proteins; however, a variant of the method, generalized simulated annealing, can be employed at a relatively low computational cost to large macromol. complexes.  Mol. dynamics trajectories frequently do not reach all relevant conformational substates, for example those connected with biol. function, a problem that can be addressed by employing enhanced sampling algorithms.  This article is part of a Special Issue entitled Recent developments of mol. dynamics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDLxzIg4zKM7Vg90H21EOLACvtfcHk0lgI4OSAV2Mt7Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVanurbN&md5=44e3ac1aa042c35ee08aa86a6a63e78e</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.bbagen.2014.10.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbagen.2014.10.019%26sid%3Dliteratum%253Aachs%26aulast%3DBernardi%26aufirst%3DR.%2BC.%26aulast%3DMelo%26aufirst%3DM.%2BC.%2BR.%26aulast%3DSchulten%26aufirst%3DK.%26atitle%3DEnhanced%2520sampling%2520techniques%2520in%2520molecular%2520dynamics%2520simulations%2520of%2520biological%2520systems%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Gen.%2520Subj.%26date%3D2015%26volume%3D1850%26spage%3D872%26epage%3D877%26doi%3D10.1016%2Fj.bbagen.2014.10.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Harrison, R. L.</span><span> </span><span class="NLM_article-title">Introduction to monte carlo simulation</span> <span class="citation_source-journal">AIP Conf. Proc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">1204</span><span class="NLM_x">, </span> <span class="NLM_fpage">17</span><span class="NLM_x">–</span> <span class="NLM_lpage">21</span><span class="refDoi"> DOI: 10.1063/1.3295638</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1063%2F1.3295638" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=20733932" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A280%3ADC%252BC2srit1KmtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1204&publication_year=2010&pages=17-21&author=R.+L.+Harrison&title=Introduction+to+monte+carlo+simulation&doi=10.1063%2F1.3295638"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Introduction To Monte Carlo Simulation</span></div><div class="casAuthors">Harrison Robert L</div><div class="citationInfo"><span class="NLM_cas:title">AIP conference proceedings</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">1204</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">17-21</span>
        ISSN:<span class="NLM_cas:issn">0094-243X</span>.
    </div><div class="casAbstract">This paper reviews the history and principles of Monte Carlo simulation, emphasizing techniques commonly used in the simulation of medical imaging.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSk_aq2nZIvSIHWj6b1v8uTfW6udTcc2eZRmvUq8RdQY7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2srit1KmtA%253D%253D&md5=3b7d6f9770f1c9b9032f16f89805845f</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1063%2F1.3295638&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.3295638%26sid%3Dliteratum%253Aachs%26aulast%3DHarrison%26aufirst%3DR.%2BL.%26atitle%3DIntroduction%2520to%2520monte%2520carlo%2520simulation%26jtitle%3DAIP%2520Conf.%2520Proc.%26date%3D2010%26volume%3D1204%26spage%3D17%26epage%3D21%26doi%3D10.1063%2F1.3295638" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Reif, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hunenberger, P. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oostenbrink, C.</span><span> </span><span class="NLM_article-title">New interaction parameters for charged amino acid side chains in the GROMOS force field</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">3705</span><span class="NLM_x">–</span> <span class="NLM_lpage">3723</span><span class="refDoi"> DOI: 10.1021/ct300156h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct300156h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC38XlsVaksbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2012&pages=3705-3723&author=M.+M.+Reifauthor=P.+H.+Hunenbergerauthor=C.+Oostenbrink&title=New+interaction+parameters+for+charged+amino+acid+side+chains+in+the+GROMOS+force+field&doi=10.1021%2Fct300156h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">New Interaction Parameters for Charged Amino Acid Side Chains in the GROMOS Force Field</span></div><div class="casAuthors">Reif, Maria M.; Huenenberger, Philippe H.; Oostenbrink, Chris</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3705-3723</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A GROMOS force-field parameter set 54A8 is developed, which is based on the latest 54A7 set and involves a recalibration of the nonbonded interaction parameters for the charged amino acid side chains, based on ionic side chain analogs.  After a thorough anal. of the available exptl. data, conventional hydration free energies for the ammonium; mono-, di-, tri-, and tetramethyl-ammonium; formate; acetate; propanoate; imidazolium; and guanidinium ions are combined with a std. abs. intrinsic proton hydration free energy ΔGhyd-[Hg+] = -1100 kJ·mol-1 to yield abs. intrinsic single-ion hydration free energies serving as exptl. target data.  The raw hydration free energies calcd. from atomistic simulations are affected by electrostatic and finite-size artifacts, and corrections are applied to reach methodol. independence prior to comparison with these exptl. values.  Except for monomethyl-ammonium, ions with parameters derived directly from the 54A7 force field considerably underestimate (ammonium, formate, acetate, propanoate, guanidinium) or overestimate (di-, tri-, and tetramethyl-ammonium; imidazolium) the magnitude of the intrinsic hydration free energy, the largest deviation affecting the acetate ion (40.0 kJ·mol-1).  After reparameterization into 54A8, the mean and maximal abs. deviations between simulated and exptl. data over the set of 10 ions are reduced from 23.1 and 40.0 kJ·mol-1, resp., to 1.8 and 6.3 kJ·mol-1, resp.  Although the 54A7 and 54A8 parameter sets differ significantly in terms of the hydration free energies of the ions considered, other properties such as ion-water radial distribution functions and ion-ion potentials of mean force appear to be only moderately sensitive to this change.  These properties are similar for the two sets and, in the case of the ion-water radial distribution functions, in good agreement with available exptl. data.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUrAQr3bSDa7Vg90H21EOLACvtfcHk0ljqnn4MwlYrxw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XlsVaksbg%253D&md5=04f0f93b2985e1a8680498477eacfbc0</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Fct300156h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct300156h%26sid%3Dliteratum%253Aachs%26aulast%3DReif%26aufirst%3DM.%2BM.%26aulast%3DHunenberger%26aufirst%3DP.%2BH.%26aulast%3DOostenbrink%26aufirst%3DC.%26atitle%3DNew%2520interaction%2520parameters%2520for%2520charged%2520amino%2520acid%2520side%2520chains%2520in%2520the%2520GROMOS%2520force%2520field%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2012%26volume%3D8%26spage%3D3705%26epage%3D3723%26doi%3D10.1021%2Fct300156h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Lindorff-Larsen, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Piana, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Palmo, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maragakis, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klepeis, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dror, R. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaw, D. E.</span><span> </span><span class="NLM_article-title">Improved side-chain torsion potentials for the Amber ff99SB protein force field</span> <span class="citation_source-journal">Proteins: Struct., Funct., Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">78</span><span class="NLM_x">, </span> <span class="NLM_fpage">1950</span><span class="NLM_x">–</span> <span class="NLM_lpage">1958</span><span class="refDoi"> DOI: 10.1002/prot.22711</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1002%2Fprot.22711" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=20408171" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC3cXkvFegtLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=78&publication_year=2010&pages=1950-1958&author=K.+Lindorff-Larsenauthor=S.+Pianaauthor=K.+Palmoauthor=P.+Maragakisauthor=J.+L.+Klepeisauthor=R.+O.+Drorauthor=D.+E.+Shaw&title=Improved+side-chain+torsion+potentials+for+the+Amber+ff99SB+protein+force+field&doi=10.1002%2Fprot.22711"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Improved side-chain torsion potentials for the Amber ff99SB protein force field</span></div><div class="casAuthors">Lindorff-Larsen, Kresten; Piana, Stefano; Palmo, Kim; Maragakis, Paul; Klepeis, John L.; Dror, Ron O.; Shaw, David E.</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Bioinformatics</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">78</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1950-1958</span>CODEN:
                <span class="NLM_cas:coden">PSFBAF</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">Recent advances in hardware and software have enabled increasingly long mol. dynamics (MD) simulations of biomols., exposing certain limitations in the accuracy of the force fields used for such simulations and spurring efforts to refine these force fields.  Recent modifications to the Amber and CHARMM protein force fields, for example, have improved the backbone torsion potentials, remedying deficiencies in earlier versions.  Here, the authors further advance simulation accuracy by improving the amino acid side-chain torsion potentials of the Amber ff99SB force field.  First, the authors used simulations of model alpha-helical systems to identify the four residue types whose rotamer distribution differed the most from expectations based on Protein Data Bank statistics.  Second, the authors optimized the side-chain torsion potentials of these residues to match new, high-level quantum-mech. calcns.  Finally, the authors used microsecond-timescale MD simulations in explicit solvent to validate the resulting force field against a large set of exptl. NMR measurements that directly probe side-chain conformations.  The new force field, which the authors have termed Amber ff99SB-ILDN, exhibits considerably better agreement with the NMR data.  Proteins 2010. © 2010 Wiley-Liss, Inc.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrNFHrYs9024bVg90H21EOLACvtfcHk0ljqnn4MwlYrxw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXkvFegtLo%253D&md5=447a9004026e2b93f0f7beff165daa09</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1002%2Fprot.22711&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.22711%26sid%3Dliteratum%253Aachs%26aulast%3DLindorff-Larsen%26aufirst%3DK.%26aulast%3DPiana%26aufirst%3DS.%26aulast%3DPalmo%26aufirst%3DK.%26aulast%3DMaragakis%26aufirst%3DP.%26aulast%3DKlepeis%26aufirst%3DJ.%2BL.%26aulast%3DDror%26aufirst%3DR.%2BO.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26atitle%3DImproved%2520side-chain%2520torsion%2520potentials%2520for%2520the%2520Amber%2520ff99SB%2520protein%2520force%2520field%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D2010%26volume%3D78%26spage%3D1950%26epage%3D1958%26doi%3D10.1002%2Fprot.22711" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Maier, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martinez, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kasavajhala, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wickstrom, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hauser, K. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simmerling, C.</span><span> </span><span class="NLM_article-title">ff14SB: Improving the accuracy of protein side chain and backbone parameters from ff99SB</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">3696</span><span class="NLM_x">–</span> <span class="NLM_lpage">3713</span><span class="refDoi"> DOI: 10.1021/acs.jctc.5b00255</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jctc.5b00255" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFequ7rN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=3696-3713&author=J.+A.+Maierauthor=C.+Martinezauthor=K.+Kasavajhalaauthor=L.+Wickstromauthor=K.+E.+Hauserauthor=C.+Simmerling&title=ff14SB%3A+Improving+the+accuracy+of+protein+side+chain+and+backbone+parameters+from+ff99SB&doi=10.1021%2Facs.jctc.5b00255"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">ff14SB: Improving the Accuracy of Protein Side Chain and Backbone Parameters from ff99SB</span></div><div class="casAuthors">Maier, James A.; Martinez, Carmenza; Kasavajhala, Koushik; Wickstrom, Lauren; Hauser, Kevin E.; Simmerling, Carlos</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3696-3713</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mol. mechanics is powerful for its speed in atomistic simulations, but an accurate force field is required.  The Amber ff99SB force field improved protein secondary structure balance and dynamics from earlier force fields like ff99, but weaknesses in side chain rotamer and backbone secondary structure preferences have been identified.  Here, we performed a complete refit of all amino acid side chain dihedral parameters, which had been carried over from ff94.  The training set of conformations included multidimensional dihedral scans designed to improve transferability of the parameters.  Improvement in all amino acids was obtained as compared to ff99SB.  Parameters were also generated for alternate protonation states of ionizable side chains.  Av. errors in relative energies of pairs of conformations were under 1.0 kcal/mol as compared to QM, reduced 35% from ff99SB.  We also took the opportunity to make empirical adjustments to the protein backbone dihedral parameters as compared to ff99SB.  Multiple small adjustments of φ and ψ parameters were tested against NMR scalar coupling data and secondary structure content for short peptides.  The best results were obtained from a phys. motivated adjustment to the φ rotational profile that compensates for lack of ff99SB QM training data in the β-ppII transition region.  Together, these backbone and side chain modifications (hereafter called ff14SB) not only better reproduced their benchmarks, but also improved secondary structure content in small peptides and reprodn. of NMR χ1 scalar coupling measurements for proteins in soln.  We also discuss the Amber ff12SB parameter set, a preliminary version of ff14SB that includes most of its improvements.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgntnN9ES97rVg90H21EOLACvtfcHk0ljqnn4MwlYrxw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFequ7rN&md5=7b803577b3b6912cc6750cfbd356596e</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Facs.jctc.5b00255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jctc.5b00255%26sid%3Dliteratum%253Aachs%26aulast%3DMaier%26aufirst%3DJ.%2BA.%26aulast%3DMartinez%26aufirst%3DC.%26aulast%3DKasavajhala%26aufirst%3DK.%26aulast%3DWickstrom%26aufirst%3DL.%26aulast%3DHauser%26aufirst%3DK.%2BE.%26aulast%3DSimmerling%26aufirst%3DC.%26atitle%3Dff14SB%253A%2520Improving%2520the%2520accuracy%2520of%2520protein%2520side%2520chain%2520and%2520backbone%2520parameters%2520from%2520ff99SB%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2015%26volume%3D11%26spage%3D3696%26epage%3D3713%26doi%3D10.1021%2Facs.jctc.5b00255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Lee, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Swails, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yeom, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eastman, P. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lemkul, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wei, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buckner, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jeong, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qi, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jo, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pande, V. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Case, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brooks, C. L.,  III</span><span class="NLM_x">; </span><span class="NLM_contrib-group">MacKerell, A. D.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klauda, J. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Im, W.</span><span> </span><span class="NLM_article-title">CHARMM-GUI input generator for NAMD, GROMACS, AMBER, OpenMM, and CHARMM/OpenMM simulations using the CHARMM36 additive force field</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">405</span><span class="NLM_x">–</span> <span class="NLM_lpage">413</span><span class="refDoi"> DOI: 10.1021/acs.jctc.5b00935</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jctc.5b00935" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVWru7nI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=405-413&author=J.+Leeauthor=X.+Chengauthor=J.+M.+Swailsauthor=M.+S.+Yeomauthor=P.+K.+Eastmanauthor=J.+A.+Lemkulauthor=S.+Weiauthor=J.+Bucknerauthor=J.+C.+Jeongauthor=Y.+Qiauthor=S.+Joauthor=V.+S.+Pandeauthor=D.+A.+Caseauthor=C.+L.+Brooksauthor=A.+D.+MacKerellauthor=J.+B.+Klaudaauthor=W.+Im&title=CHARMM-GUI+input+generator+for+NAMD%2C+GROMACS%2C+AMBER%2C+OpenMM%2C+and+CHARMM%2FOpenMM+simulations+using+the+CHARMM36+additive+force+field&doi=10.1021%2Facs.jctc.5b00935"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">CHARMM-GUI Input Generator for NAMD, GROMACS, AMBER, OpenMM, and CHARMM/OpenMM Simulations Using the CHARMM36 Additive Force Field</span></div><div class="casAuthors">Lee, Jumin; Cheng, Xi; Swails, Jason M.; Yeom, Min Sun; Eastman, Peter K.; Lemkul, Justin A.; Wei, Shuai; Buckner, Joshua; Jeong, Jong Cheol; Qi, Yifei; Jo, Sunhwan; Pande, Vijay S.; Case, David A.; Brooks, Charles L.; MacKerell, Alexander D.; Klauda, Jeffery B.; Im, Wonpil</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">405-413</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Proper treatment of nonbonded interactions is essential for the accuracy of mol. dynamics (MD) simulations, esp. in studies of lipid bilayers.  The use of the CHARMM36 force field (C36 FF) in different MD simulation programs can result in disagreements with published simulations performed with CHARMM due to differences in the protocols used to treat the long-range and 1-4 nonbonded interactions.  In this study, we systematically test the use of the C36 lipid FF in NAMD, GROMACS, AMBER, OpenMM, and CHARMM/OpenMM.  A wide range of Lennard-Jones (LJ) cutoff schemes and integrator algorithms were tested to find the optimal simulation protocol to best match bilayer properties of six lipids with varying acyl chain satn. and head groups.  MD simulations of a 1,2-dipalmitoyl-sn-phosphatidylcholine (DPPC) bilayer were used to obtain the optimal protocol for each program.  MD simulations with all programs were found to reasonably match the DPPC bilayer properties (surface area per lipid, chain order parameters, and area compressibility modulus) obtained using the std. protocol used in CHARMM as well as from expts.  The optimal simulation protocol was then applied to the other five lipid simulations and resulted in excellent agreement between results from most simulation programs as well as with exptl. data.  AMBER compared least favorably with the expected membrane properties, which appears to be due to its use of the hard-truncation in the LJ potential vs. a force-based switching function used to smooth the LJ potential as it approaches the cutoff distance.  The optimal simulation protocol for each program has been implemented in CHARMM-GUI.  This protocol is expected to be applicable to the remainder of the additive C36 FF including the proteins, nucleic acids, carbohydrates, and small mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrioKQ5MoVZkrVg90H21EOLACvtfcHk0lj5Q-UkwfZGCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVWru7nI&md5=1e986f2c205aca8fb80442bc9c95b229</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Facs.jctc.5b00935&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jctc.5b00935%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DJ.%26aulast%3DCheng%26aufirst%3DX.%26aulast%3DSwails%26aufirst%3DJ.%2BM.%26aulast%3DYeom%26aufirst%3DM.%2BS.%26aulast%3DEastman%26aufirst%3DP.%2BK.%26aulast%3DLemkul%26aufirst%3DJ.%2BA.%26aulast%3DWei%26aufirst%3DS.%26aulast%3DBuckner%26aufirst%3DJ.%26aulast%3DJeong%26aufirst%3DJ.%2BC.%26aulast%3DQi%26aufirst%3DY.%26aulast%3DJo%26aufirst%3DS.%26aulast%3DPande%26aufirst%3DV.%2BS.%26aulast%3DCase%26aufirst%3DD.%2BA.%26aulast%3DBrooks%26aufirst%3DC.%2BL.%26aulast%3DMacKerell%26aufirst%3DA.%2BD.%26aulast%3DKlauda%26aufirst%3DJ.%2BB.%26aulast%3DIm%26aufirst%3DW.%26atitle%3DCHARMM-GUI%2520input%2520generator%2520for%2520NAMD%252C%2520GROMACS%252C%2520AMBER%252C%2520OpenMM%252C%2520and%2520CHARMM%252FOpenMM%2520simulations%2520using%2520the%2520CHARMM36%2520additive%2520force%2520field%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2016%26volume%3D12%26spage%3D405%26epage%3D413%26doi%3D10.1021%2Facs.jctc.5b00935" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Reif, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Winger, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oostenbrink, C.</span><span> </span><span class="NLM_article-title">Testing of the GROMOS force-field parameter set 54A8: structural properties of electrolyte solutions, lipid bilayers, and proteins</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">1247</span><span class="NLM_x">–</span> <span class="NLM_lpage">1264</span><span class="refDoi"> DOI: 10.1021/ct300874c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct300874c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC3sXitVektw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=1247-1264&author=M.+M.+Reifauthor=M.+Wingerauthor=C.+Oostenbrink&title=Testing+of+the+GROMOS+force-field+parameter+set+54A8%3A+structural+properties+of+electrolyte+solutions%2C+lipid+bilayers%2C+and+proteins&doi=10.1021%2Fct300874c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Testing of the GROMOS Force-Field Parameter Set 54A8: Structural Properties of Electrolyte Solutions, Lipid Bilayers, and Proteins</span></div><div class="casAuthors">Reif, Maria M.; Winger, Moritz; Oostenbrink, Chris</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">1247-1264</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The GROMOS 54A8 force field is the first of its kind to contain nonbonded parameters for charged amino acid side chains that are derived in a rigorously thermodn. fashion, namely a calibration against single-ion hydration free energies.  Considering charged moieties in soln., the most decisive signature of the GROMOS 54A8 force field in comparison to its predecessor 54A7 can probably be found in the thermodn. equil. between salt-bridged ion pair formation and hydration.  Possible shifts in this equil. might crucially affect the properties of electrolyte solns. or/and the stability of (bio)mols.  It is therefore important to investigate the consequences of the altered description of charged oligoat. species in the GROMOS 54A8 force field.  The present study focuses on examg. the ability of the GROMOS 54A8 force field to accurately model the structural properties of electrolyte solns., lipid bilayers, and proteins.  It is found that (i) aq. electrolytes involving oligoat. species (sodium acetate, methylammonium chloride, guanidinium chloride) reproduce exptl. salt activity derivs. for concns. up to 1.0 m (1.0-m) very well, and good agreement between simulated and exptl. data is also reached for sodium acetate and methylammonium chloride at 2.0 m concn., while not even qual. agreement is found for sodium chloride throughout the whole range of examd. concns., indicating a failure of the GROMOS 54A7 and 54A8 force-field parameter sets to correctly account for the balance between ion-ion and ion-water binding propensities of sodium and chloride ions; (ii) the GROMOS 54A8 force field reproduces the liq. cryst.-like phase of a hydrated DPPC bilayer at a pressure of 1 bar and a temp. of 323 K, the area per lipid being in agreement with exptl. data, whereas other structural properties (vol. per lipid, bilayer thickness) appear underestimated; (iii) the secondary structure of a range of different proteins simulated with the GROMOS 54A8 force field at pH 7 is maintained and compatible with exptl. NMR data, while, as also obsd. for the GROMOS 54A7 force field, α-helixes are slightly overstabilized with respect to 310-helixes; (iv) with the GROMOS 54A8 force field, the side chains of arginine, lysine, aspartate, and glutamate residues appear slightly more hydrated and present a slight excess of oppositely-charged s.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4C9UOi98oabVg90H21EOLACvtfcHk0lj5Q-UkwfZGCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXitVektw%253D%253D&md5=d5093a833a5de57591e4f620814d813a</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Fct300874c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct300874c%26sid%3Dliteratum%253Aachs%26aulast%3DReif%26aufirst%3DM.%2BM.%26aulast%3DWinger%26aufirst%3DM.%26aulast%3DOostenbrink%26aufirst%3DC.%26atitle%3DTesting%2520of%2520the%2520GROMOS%2520force-field%2520parameter%2520set%252054A8%253A%2520structural%2520properties%2520of%2520electrolyte%2520solutions%252C%2520lipid%2520bilayers%252C%2520and%2520proteins%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2013%26volume%3D9%26spage%3D1247%26epage%3D1264%26doi%3D10.1021%2Fct300874c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Malde, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mark, A. E.</span><span> </span><span class="NLM_article-title">179 Validation of ligands in x-ray crystal structures</span> <span class="citation_source-journal">J. Biomol. Struct. Dyn.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">33</span><span class="NLM_x"> (</span><span class="NLM_issue">Suppl. 1</span><span class="NLM_x">) </span> <span class="NLM_fpage">117</span><span class="NLM_x">–</span> <span class="NLM_lpage">177</span><span class="refDoi"> DOI: 10.1080/07391102.2015.1032816</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1080%2F07391102.2015.1032816" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=26103389" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A280%3ADC%252BC2MbmsFGnsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2015&pages=117-177&issue=Suppl.+1&author=A.+K.+Maldeauthor=A.+E.+Mark&title=179+Validation+of+ligands+in+x-ray+crystal+structures&doi=10.1080%2F07391102.2015.1032816"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">179 Validation of ligands in X-ray crystal structures</span></div><div class="casAuthors">Malde Alpeshkumar K; Mark Alan E</div><div class="citationInfo"><span class="NLM_cas:title">Journal of biomolecular structure & dynamics</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">33 Suppl 1</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">117-77</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTxJkC0jZN2SDslzEEsg76ufW6udTcc2eaVmp2iAvshArntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2MbmsFGnsg%253D%253D&md5=9d96f2720f75a6aaa7c6de1483bd23a3</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1080%2F07391102.2015.1032816&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F07391102.2015.1032816%26sid%3Dliteratum%253Aachs%26aulast%3DMalde%26aufirst%3DA.%2BK.%26aulast%3DMark%26aufirst%3DA.%2BE.%26atitle%3D179%2520Validation%2520of%2520ligands%2520in%2520x-ray%2520crystal%2520structures%26jtitle%3DJ.%2520Biomol.%2520Struct.%2520Dyn.%26date%3D2015%26volume%3D33%26issue%3DSuppl.%25201%26spage%3D117%26epage%3D177%26doi%3D10.1080%2F07391102.2015.1032816" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Malde, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zuo, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Breeze, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stroet, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poger, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nair, P. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oostenbrink, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mark, A. E.</span><span> </span><span class="NLM_article-title">An automated force field topology builder (ATB) and repository: version 1.0</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">4026</span><span class="NLM_x">–</span> <span class="NLM_lpage">4037</span><span class="refDoi"> DOI: 10.1021/ct200196m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct200196m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlOksrfO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=4026-4037&author=A.+K.+Maldeauthor=L.+Zuoauthor=M.+Breezeauthor=M.+Stroetauthor=D.+Pogerauthor=P.+C.+Nairauthor=C.+Oostenbrinkauthor=A.+E.+Mark&title=An+automated+force+field+topology+builder+%28ATB%29+and+repository%3A+version+1.0&doi=10.1021%2Fct200196m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">An Automated Force Field Topology Builder (ATB) and Repository: Version 1.0</span></div><div class="casAuthors">Malde, Alpeshkumr K.; Zuo, Le; Breeze, Matthew; Stroet, Martin; Poger, David; Nair, Pramod C.; Oostenbrink, Chris; Mark, Alan F.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">4026-4037</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The Automated force field Topol. Builder (ATB, http://compbio.biosci.uq.edu.au/atb) is a Web-accessible server that can provide topologies and parameters for a wide range of mols. appropriate for use in mol. simulations, computational drug design, and X-ray refinement.  The ATB has three primary functions: (1) to act as a repository for mols. that have been parametrized as part of the GROMOS family of force fields, (2) to act as a repository for pre-equilibrated systems for use as starting configurations in mol. dynamics simulations (solvent mixts., lipid systems pre-equilibrated to adopt a specific phase, etc.), and (3) to generate force field descriptions of novel mols. compatible with the GROMOS family of force fields in a variety of formats (GROMOS, GROMACS, and CNS).  Force field descriptions of novel mols. are derived using a multistep process in which results from quantum mech. (QM) calcns. are combined with a knowledge-based approach to ensure compatibility (as far as possible) with a specific parameter set of the GROMOS force field.  The ATB has several unique features: (1) It requires that the user stipulate the protonation and tautomeric states of the mol. (2) The symmetry of the mol. is analyzed to ensure that equiv. atoms are assigned identical parameters. (3) Charge groups are assigned automatically. (4) Where the assignment of a given parameter is ambiguous, a range of possible alternatives is provided.  The ATB also provides several validation tools to assist the user to assess the degree to which the topol. generated may be appropriate for a given task.  In addn. to detailing the steps involved in generating a force field topol. compatible with a specific GROMOS parameter set (GROMOS 53A6), the challenges involved in the automatic generation of force field parameters for at. simulations in general are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpGdRdHvU2-PLVg90H21EOLACvtfcHk0lgnAx5mCFntBA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlOksrfO&md5=172110d6b41762f68c5238e7ea127e1d</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Fct200196m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct200196m%26sid%3Dliteratum%253Aachs%26aulast%3DMalde%26aufirst%3DA.%2BK.%26aulast%3DZuo%26aufirst%3DL.%26aulast%3DBreeze%26aufirst%3DM.%26aulast%3DStroet%26aufirst%3DM.%26aulast%3DPoger%26aufirst%3DD.%26aulast%3DNair%26aufirst%3DP.%2BC.%26aulast%3DOostenbrink%26aufirst%3DC.%26aulast%3DMark%26aufirst%3DA.%2BE.%26atitle%3DAn%2520automated%2520force%2520field%2520topology%2520builder%2520%2528ATB%2529%2520and%2520repository%253A%2520version%25201.0%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2011%26volume%3D7%26spage%3D4026%26epage%3D4037%26doi%3D10.1021%2Fct200196m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Koziara, K. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stroet, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Malde, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mark, A. E.</span><span> </span><span class="NLM_article-title">Testing and validation of the automated topology builder (ATB) version 2.0: prediction of hydration free enthalpies</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">221</span><span class="NLM_x">–</span> <span class="NLM_lpage">233</span><span class="refDoi"> DOI: 10.1007/s10822-014-9713-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1007%2Fs10822-014-9713-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=24477799" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1WqtbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2014&pages=221-233&author=K.+B.+Koziaraauthor=M.+Stroetauthor=A.+K.+Maldeauthor=A.+E.+Mark&title=Testing+and+validation+of+the+automated+topology+builder+%28ATB%29+version+2.0%3A+prediction+of+hydration+free+enthalpies&doi=10.1007%2Fs10822-014-9713-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Testing and validation of the Automated Topology Builder (ATB) version 2.0: prediction of hydration free enthalpies</span></div><div class="casAuthors">Koziara, Katarzyna B.; Stroet, Martin; Malde, Alpeshkumar K.; Mark, Alan E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">221-233</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">To test and validate the Automated force field Topol. Builder and Repository (ATB; http://compbio.biosci.uq.edu.au/atb/) the hydration free enthalpies for a set of 214 drug-like mols., including 47 mols. that form part of the SAMPL4 challenge have been estd. using thermodn. integration and compared to expt.  The calcns. were performed using a fully automated protocol that incorporated a dynamic anal. of the convergence and integration error in the selection of intermediate points.  The system has been designed and implemented such that hydration free enthalpies can be obtained without manual intervention following the submission of a mol. to the ATB.  The overall av. unsigned error (AUE) using ATB 2.0 topologies for the complete set of 214 mols. was 6.7 kJ/mol and for mols. within the SAMPL4 7.5 kJ/mol.  The root mean square error (RMSE) was 9.5 and 10.0 kJ/mol resp.  However, for mols. contg. functional groups that form part of the main GROMOS force field the AUE was 3.4 kJ/mol and the RMSE was 4.0 kJ/mol.  This suggests it will be possible to further refine the parameters provided by the ATB based on hydration free enthalpies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpoglyAPJdgQbVg90H21EOLACvtfcHk0lgnAx5mCFntBA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1WqtbY%253D&md5=a3ebbc9492be9454290a9a64706640b1</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1007%2Fs10822-014-9713-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-014-9713-7%26sid%3Dliteratum%253Aachs%26aulast%3DKoziara%26aufirst%3DK.%2BB.%26aulast%3DStroet%26aufirst%3DM.%26aulast%3DMalde%26aufirst%3DA.%2BK.%26aulast%3DMark%26aufirst%3DA.%2BE.%26atitle%3DTesting%2520and%2520validation%2520of%2520the%2520automated%2520topology%2520builder%2520%2528ATB%2529%2520version%25202.0%253A%2520prediction%2520of%2520hydration%2520free%2520enthalpies%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2014%26volume%3D28%26spage%3D221%26epage%3D233%26doi%3D10.1007%2Fs10822-014-9713-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Wang, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kollman, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Case, D. A.</span><span> </span><span class="NLM_article-title">Automatic atom type and bond type perception in molecular mechanical calculations</span> <span class="citation_source-journal">J. Mol. Graphics Modell.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">247</span><span class="NLM_x">–</span> <span class="NLM_lpage">260</span><span class="refDoi"> DOI: 10.1016/j.jmgm.2005.12.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1016%2Fj.jmgm.2005.12.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=16458552" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BD28Xps1Gis7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2006&pages=247-260&author=J.+M.+Wangauthor=W.+Wangauthor=P.+A.+Kollmanauthor=D.+A.+Case&title=Automatic+atom+type+and+bond+type+perception+in+molecular+mechanical+calculations&doi=10.1016%2Fj.jmgm.2005.12.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Automatic atom type and bond type perception in molecular mechanical calculations</span></div><div class="casAuthors">Wang, Junmei; Wang, Wei; Kollman, Peter A.; Case, David A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Graphics & Modelling</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">247-260</span>CODEN:
                <span class="NLM_cas:coden">JMGMFI</span>;
        ISSN:<span class="NLM_cas:issn">1093-3263</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">In mol. mechanics (MM) studies, atom types and/or bond types of mols. are needed to det. prior to energy calcns.  The authors present here an automatic algorithm of perceiving atom types that are defined in a description table, and an automatic algorithm of assigning bond types just based on at. connectivity.  The algorithms have been implemented in a new module of the AMBER packages.  This auxiliary module, antechamber (roughly meaning "before AMBER"), can be applied to generate necessary inputs of leap-the AMBER program to generate topologies for minimization, mol. dynamics, etc., for most org. mols.  The algorithms behind the manipulations may be useful for other mol. mech. packages as well as applications that need to designate atom types and bond types.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqizj6F18nBYbVg90H21EOLACvtfcHk0lgnAx5mCFntBA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xps1Gis7g%253D&md5=8031a21d2784d5dea12e70868522aa61</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fj.jmgm.2005.12.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmgm.2005.12.005%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%2BM.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DKollman%26aufirst%3DP.%2BA.%26aulast%3DCase%26aufirst%3DD.%2BA.%26atitle%3DAutomatic%2520atom%2520type%2520and%2520bond%2520type%2520perception%2520in%2520molecular%2520mechanical%2520calculations%26jtitle%3DJ.%2520Mol.%2520Graphics%2520Modell.%26date%3D2006%26volume%3D25%26spage%3D247%26epage%3D260%26doi%3D10.1016%2Fj.jmgm.2005.12.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wolf, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caldwell, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kollman, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Case, D. A.</span><span> </span><span class="NLM_article-title">Development and testing of a general amber force field</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">1157</span><span class="NLM_x">–</span> <span class="NLM_lpage">1174</span><span class="refDoi"> DOI: 10.1002/jcc.20035</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1002%2Fjcc.20035" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=15116359" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BD2cXksFakurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2004&pages=1157-1174&author=J.+Wangauthor=R.+M.+Wolfauthor=J.+W.+Caldwellauthor=P.+A.+Kollmanauthor=D.+A.+Case&title=Development+and+testing+of+a+general+amber+force+field&doi=10.1002%2Fjcc.20035"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Development and testing of a general Amber force field</span></div><div class="casAuthors">Wang, Junmei; Wolf, Romain M.; Caldwell, James W.; Kollman, Peter A.; Case, David A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1157-1174</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">We describe here a general Amber force field (GAFF) for org. mols.  GAFF is designed to be compatible with existing Amber force fields for proteins and nucleic acids, and has parameters for most org. and pharmaceutical mols. that are composed of H, C, N, O, S, P, and halogens.  It uses a simple functional form and a limited no. of atom types, but incorporates both empirical and heuristic models to est. force consts. and partial at. charges.  The performance of GAFF in test cases is encouraging.  In test I, 74 crystallog. structures were compared to GAFF minimized structures, with a root-mean-square displacement of 0.26 Å, which is comparable to that of the Tripos 5.2 force field (0.25 Å) and better than those of MMFF 94 and CHARMm (0.47 and 0.44 Å, resp.).  In test II, gas phase minimizations were performed on 22 nucleic acid base pairs, and the minimized structures and intermol. energies were compared to MP2/6-31G* results.  The RMS of displacements and relative energies were 0.25 Å and 1.2 kcal/mol, resp.  These data are comparable to results from Parm99/RESP (0.16 Å and 1.18 kcal/mol, resp.), which were parameterized to these base pairs.  Test III looked at the relative energies of 71 conformational pairs that were used in development of the Parm99 force field.  The RMS error in relative energies (compared to expt.) is about 0.5 kcal/mol.  GAFF can be applied to wide range of mols. in an automatic fashion, making it suitable for rational drug design and database searching.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfW8Gsf8IiW7Vg90H21EOLACvtfcHk0ljpMwdxjjF3tw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXksFakurc%253D&md5=2992017a8cf51f89290ae2562403b115</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1002%2Fjcc.20035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.20035%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DWolf%26aufirst%3DR.%2BM.%26aulast%3DCaldwell%26aufirst%3DJ.%2BW.%26aulast%3DKollman%26aufirst%3DP.%2BA.%26aulast%3DCase%26aufirst%3DD.%2BA.%26atitle%3DDevelopment%2520and%2520testing%2520of%2520a%2520general%2520amber%2520force%2520field%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2004%26volume%3D25%26spage%3D1157%26epage%3D1174%26doi%3D10.1002%2Fjcc.20035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Vanommeslaeghe, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">MacKerell, A. D.,  Jr.</span><span> </span><span class="NLM_article-title">Automation of the CHARMM general force field (CGenFF) I: bond perception and atom typing</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">3144</span><span class="NLM_x">–</span> <span class="NLM_lpage">3154</span><span class="refDoi"> DOI: 10.1021/ci300363c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci300363c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs1Gns7fL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=3144-3154&author=K.+Vanommeslaegheauthor=A.+D.+MacKerell&title=Automation+of+the+CHARMM+general+force+field+%28CGenFF%29+I%3A+bond+perception+and+atom+typing&doi=10.1021%2Fci300363c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Automation of the CHARMM General Force Field (CGenFF) I: Bond Perception and Atom Typing</span></div><div class="casAuthors">Vanommeslaeghe, K.; MacKerell, A. D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3144-3154</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mol. mechanics force fields are widely used in computer-aided drug design for the study of drug-like mols. alone or interacting with biol. systems.  In simulations involving biol. macromols., the biol. part is typically represented by a specialized biomol. force field, while the drug is represented by a matching general (org.) force field.  In order to apply these general force fields to an arbitrary drug-like mol., functionality for assignment of atom types, parameters, and charges is required.  In the present article, which is part I of a series of two, we present the algorithms for bond perception and atom typing for the CHARMM General Force Field (CGenFF).  The CGenFF atom typer first assocs. attributes to the atoms and bonds in a mol., such as valence, bond order, and ring membership among others.  Of note are a no. of features that are specifically required for CGenFF.  This information is then used by the atom typing routine to assign CGenFF atom types based on a programmable decision tree.  This allows for straight-forward implementation of CGenFF's complicated atom typing rules and for equally straight-forward updating of the atom typing scheme as the force field grows.  The presented atom typer was validated by assigning correct atom types on 477 model compds. including in the training set as well as 126 test-set mols. that were constructed to specifically verify its different components.  The program may be utilized via an online implementation at https://www.paramchem.org/.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvkdOrvB3NELVg90H21EOLACvtfcHk0ljpMwdxjjF3tw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs1Gns7fL&md5=c6679293f4a2501f2bcadf2020ca1473</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Fci300363c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci300363c%26sid%3Dliteratum%253Aachs%26aulast%3DVanommeslaeghe%26aufirst%3DK.%26aulast%3DMacKerell%26aufirst%3DA.%2BD.%26atitle%3DAutomation%2520of%2520the%2520CHARMM%2520general%2520force%2520field%2520%2528CGenFF%2529%2520I%253A%2520bond%2520perception%2520and%2520atom%2520typing%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26spage%3D3144%26epage%3D3154%26doi%3D10.1021%2Fci300363c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Vanommeslaeghe, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raman, E. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">MacKerell, A. D.,  Jr.</span><span> </span><span class="NLM_article-title">Automation of the CHARMM general force field (CGenFF) II: assignment of bonded parameters and partial atomic charges</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">3155</span><span class="NLM_x">–</span> <span class="NLM_lpage">3168</span><span class="refDoi"> DOI: 10.1021/ci3003649</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci3003649" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs1Gns7fF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=3155-3168&author=K.+Vanommeslaegheauthor=E.+P.+Ramanauthor=A.+D.+MacKerell&title=Automation+of+the+CHARMM+general+force+field+%28CGenFF%29+II%3A+assignment+of+bonded+parameters+and+partial+atomic+charges&doi=10.1021%2Fci3003649"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Automation of the CHARMM General Force Field (CGenFF) II: Assignment of Bonded Parameters and Partial Atomic Charges</span></div><div class="casAuthors">Vanommeslaeghe, K.; Raman, E. Prabhu; MacKerell, A. D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3155-3168</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mol. mechanics force fields are widely used in computer-aided drug design for the study of drug candidates interacting with biol. systems.  In these simulations, the biol. part is typically represented by a specialized biomol. force field, while the drug is represented by a matching general (org.) force field.  In order to apply these general force fields to an arbitrary drug-like mol., functionality for assignment of atom types, parameters, and partial at. charges is required.  In the present article, algorithms for the assignment of parameters and charges for the CHARMM General Force Field (CGenFF) are presented.  These algorithms rely on the existing parameters and charges that were detd. as part of the parametrization of the force field.  Bonded parameters are assigned based on the similarity between the atom types that define said parameters, while charges are detd. using an extended bond-charge increment scheme.  Charge increments were optimized to reproduce the charges on model compds. that were part of the parametrization of the force field.  Case studies are presented to clarify the functioning of the algorithms and the significance of their output data.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0zgm66CLdn7Vg90H21EOLACvtfcHk0ljpMwdxjjF3tw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs1Gns7fF&md5=e676ad1f42cb1e98dd353d4d285e8d13</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Fci3003649&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci3003649%26sid%3Dliteratum%253Aachs%26aulast%3DVanommeslaeghe%26aufirst%3DK.%26aulast%3DRaman%26aufirst%3DE.%2BP.%26aulast%3DMacKerell%26aufirst%3DA.%2BD.%26atitle%3DAutomation%2520of%2520the%2520CHARMM%2520general%2520force%2520field%2520%2528CGenFF%2529%2520II%253A%2520assignment%2520of%2520bonded%2520parameters%2520and%2520partial%2520atomic%2520charges%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26spage%3D3155%26epage%3D3168%26doi%3D10.1021%2Fci3003649" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">Makov, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Payne, M. C.</span><span> </span><span class="NLM_article-title">Periodic boundary conditions in ab initio calculations</span> <span class="citation_source-journal">Phys. Rev. B: Condens. Matter Mater. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">4014</span><span class="NLM_x">–</span> <span class="NLM_lpage">4022</span><span class="refDoi"> DOI: 10.1103/PhysRevB.51.4014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1103%2FPhysRevB.51.4014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=9979237" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADyaK2MXjvVGksbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=1995&pages=4014-4022&author=G.+Makovauthor=M.+C.+Payne&title=Periodic+boundary+conditions+in+ab+initio+calculations&doi=10.1103%2FPhysRevB.51.4014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Periodic boundary conditions in ab initio calculations</span></div><div class="casAuthors">Makov, G.; Payne, M. C.</div><div class="citationInfo"><span class="NLM_cas:title">Physical Review B: Condensed Matter</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">4014-22</span>CODEN:
                <span class="NLM_cas:coden">PRBMDO</span>;
        ISSN:<span class="NLM_cas:issn">0163-1829</span>.
    
            (<span class="NLM_cas:orgname">American Physical Society</span>)
        </div><div class="casAbstract">The convergence of the electrostatic energy in calcns. using periodic boundary conditions is considered in the context of periodic solids and localized aperiodic systems in the gas and condensed phases.  Conditions for the abs. convergence of the total energy in periodic boundary conditions are obtained, and their implications for calcns. of the properties of polarized solids under the zero-field assumption are discussed.  For aperiodic systems the exact electrostatic energy functional in periodic boundary conditions is obtained.  The convergence in such systems is considered in the limit of large supercells, where, in the gas phase, the computational effort is proportional to the vol.  It is shown that for neutral localized aperiodic systems in either the gas or condensed phases, the energy can always be made to converge as O (L-5) where L is the linear dimension of the supercell.  For charged systems, convergence at this rate can be achieved after adding correction terms to the energy to account for spurious interactions induced by the periodic boundary conditions.  These terms are derived exactly for the gas phase and heuristically for the condensed phase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo13RcOpxda2bVg90H21EOLACvtfcHk0lh4UPc0DxtJPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXjvVGksbk%253D&md5=e43f50b6b2ab0d05c4f834f98f531bba</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1103%2FPhysRevB.51.4014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1103%252FPhysRevB.51.4014%26sid%3Dliteratum%253Aachs%26aulast%3DMakov%26aufirst%3DG.%26aulast%3DPayne%26aufirst%3DM.%2BC.%26atitle%3DPeriodic%2520boundary%2520conditions%2520in%2520ab%2520initio%2520calculations%26jtitle%3DPhys.%2520Rev.%2520B%253A%2520Condens.%2520Matter%2520Mater.%2520Phys.%26date%3D1995%26volume%3D51%26spage%3D4014%26epage%3D4022%26doi%3D10.1103%2FPhysRevB.51.4014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Martyna, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tobias, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klein, M. L.</span><span> </span><span class="NLM_article-title">Constant-pressure molecular-dynamics algorithms</span> <span class="citation_source-journal">J. Chem. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">101</span><span class="NLM_x">, </span> <span class="NLM_fpage">4177</span><span class="NLM_x">–</span> <span class="NLM_lpage">4189</span><span class="refDoi"> DOI: 10.1063/1.467468</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1063%2F1.467468" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADyaK2cXmtFeht7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=101&publication_year=1994&pages=4177-4189&author=G.+J.+Martynaauthor=D.+J.+Tobiasauthor=M.+L.+Klein&title=Constant-pressure+molecular-dynamics+algorithms&doi=10.1063%2F1.467468"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Constant pressure molecular dynamics algorithms</span></div><div class="casAuthors">Martyna, Glenn J.; Tobias, Douglas J.; Klein, Michael L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">101</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">4177-89</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    </div><div class="casAbstract">Modularly invariant equations of motion are derived that generate the isothermal-isobaric ensemble as their phase space avs.  Isotropic vol. fluctuations and fully flexible simulation cells as well as a hybrid scheme that naturally combines the two motions are considered.  The resulting methods are tested on two problems, a particle in a one-dimensional periodic potential and a spherical model of C60 in the solid/fluid phase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3YN7wMRJpjrVg90H21EOLACvtfcHk0lh4UPc0DxtJPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXmtFeht7o%253D&md5=c14bd79c6398b0b30541e3cbe92851b0</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1063%2F1.467468&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.467468%26sid%3Dliteratum%253Aachs%26aulast%3DMartyna%26aufirst%3DG.%2BJ.%26aulast%3DTobias%26aufirst%3DD.%2BJ.%26aulast%3DKlein%26aufirst%3DM.%2BL.%26atitle%3DConstant-pressure%2520molecular-dynamics%2520algorithms%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1994%26volume%3D101%26spage%3D4177%26epage%3D4189%26doi%3D10.1063%2F1.467468" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group">Hünenberger, P. H.</span><span> </span><span class="NLM_article-title">Thermostat Algorithms for Molecular Dynamics Simulations</span>. In  <span class="citation_source-book">Advanced Computer Simulation</span>; Advances in Polymer Science, Vol. 173; <span class="NLM_publisher-name">Springer</span>,<span class="NLM_x"> </span><span class="NLM_year">2005</span><span class="NLM_x">; </span>pp  <span class="NLM_fpage">105</span><span class="NLM_x">–</span> <span class="NLM_lpage">149</span>; DOI: <span class="refDoi"> DOI: 10.1007/b99427</span> .</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1007%2Fb99427" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2005&pages=105-149&author=P.+H.+H%C3%BCnenberger&title=Advanced+Computer+Simulation"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1007%2Fb99427&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252Fb99427%26sid%3Dliteratum%253Aachs%26aulast%3DH%25C3%25BCnenberger%26aufirst%3DP.%2BH.%26atitle%3DThermostat%2520Algorithms%2520for%2520Molecular%2520Dynamics%2520Simulations%26btitle%3DAdvanced%2520Computer%2520Simulation%26pub%3DSpringer%26date%3D2005%26spage%3D105%26epage%3D149%26doi%3D10.1007%2Fb99427" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group">Buch, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giorgino, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Fabritiis, G.</span><span> </span><span class="NLM_article-title">Complete reconstruction of an enzyme-inhibitor binding process by molecular dynamics simulations</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">108</span><span class="NLM_x">, </span> <span class="NLM_fpage">10184</span><span class="NLM_x">–</span> <span class="NLM_lpage">10189</span><span class="refDoi"> DOI: 10.1073/pnas.1103547108</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1073%2Fpnas.1103547108" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=21646537" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC3MXot1egu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=108&publication_year=2011&pages=10184-10189&author=I.+Buchauthor=T.+Giorginoauthor=G.+De+Fabritiis&title=Complete+reconstruction+of+an+enzyme-inhibitor+binding+process+by+molecular+dynamics+simulations&doi=10.1073%2Fpnas.1103547108"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Complete reconstruction of an enzyme-inhibitor binding process by molecular dynamics simulations</span></div><div class="casAuthors">Buch, Ignasi; Giorgino, Toni; De Fabritiis, Gianni</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">108</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">10184-10189, S10184/1-S10184/6</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The understanding of protein-ligand binding is of crit. importance for biomedical research, yet the process itself has been very difficult to study because of its intrinsically dynamic character.  Here, we have been able to quant. reconstruct the complete binding process of the enzyme-inhibitor complex trypsin-benzamidine by performing 495 mol. dynamics simulations of free ligand binding of 100 ns each, 187 of which produced binding events with an rmsd less than 2 Å compared to the crystal structure.  The binding paths obtained are able to capture the kinetic pathway of the inhibitor diffusing from solvent (SO) to the bound (S4) state passing through two metastable intermediate states S2 and S3.  Rather than directly entering the binding pocket the inhibitor appears to roll on the surface of the protein in its transition between S3 and the final binding pocket, whereas the transition between S2 and the bound pose requires rediffusion to S3.  An estn. of the std. free energy of binding gives ΔG° = -5.2 0.4 kcal/mol (cf. the exptl. value -6.2 kcal/mol), and a two-states kinetic model kon = (1.5 ± 0.2) × 108 M-1 s-1 and koff = (9.5 ± 3.3) × 104 s-1 for unbound to bound transitions.  The ability to reconstruct by simple diffusion the binding pathway of an enzyme-inhibitor binding process demonstrates the predictive power of unconventional high-throughput mol. simulations.  Moreover, the methodol. is directly applicable to other mol. systems and thus of general interest in biomedical and pharmaceutical research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrEgdoM19tFbbVg90H21EOLACvtfcHk0lh4UPc0DxtJPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXot1egu78%253D&md5=40fdd0ec62da71c0f11d57731ee518a1</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1103547108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1103547108%26sid%3Dliteratum%253Aachs%26aulast%3DBuch%26aufirst%3DI.%26aulast%3DGiorgino%26aufirst%3DT.%26aulast%3DDe%2BFabritiis%26aufirst%3DG.%26atitle%3DComplete%2520reconstruction%2520of%2520an%2520enzyme-inhibitor%2520binding%2520process%2520by%2520molecular%2520dynamics%2520simulations%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2011%26volume%3D108%26spage%3D10184%26epage%3D10189%26doi%3D10.1073%2Fpnas.1103547108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group">Thomas, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yuriev, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chalmers, D. K.</span><span> </span><span class="NLM_article-title">Ligand binding pathways of clozapine and haloperidol in the dopamine D-2 and D-3 receptors</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">308</span><span class="NLM_x">–</span> <span class="NLM_lpage">321</span><span class="refDoi"> DOI: 10.1021/acs.jcim.5b00457</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.5b00457" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitVGiurzO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2016&pages=308-321&author=T.+Thomasauthor=Y.+Fangauthor=E.+Yurievauthor=D.+K.+Chalmers&title=Ligand+binding+pathways+of+clozapine+and+haloperidol+in+the+dopamine+D-2+and+D-3+receptors&doi=10.1021%2Facs.jcim.5b00457"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand Binding Pathways of Clozapine and Haloperidol in the Dopamine D2 and D3 Receptors</span></div><div class="casAuthors">Thomas, Trayder; Fang, Yu; Yuriev, Elizabeth; Chalmers, David K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">308-321</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The binding of a small mol. ligand to its protein target is most often characterized by binding affinity and is typically viewed as an on/off switch.  The more complex reality is that binding involves the ligand passing through a series of intermediate states between the soln. phase and the fully bound pose.  We have performed a set of 29 unbiased mol. dynamics simulations to model the binding pathways of the dopamine receptor antagonists clozapine and haloperidol binding to the D2 and D3 dopamine receptors.  Through these simulations we have captured the binding pathways of clozapine and haloperidol from the extracellular vestibule to the orthosteric binding site and thereby, we also predict the bound pose of each ligand.  These are the first long time scale simulations of haloperidol or clozapine binding to dopamine receptors.  From these simulations, we have identified several important stages in the binding pathway, including the involvement of Tyr7.35 in a "handover" mechanism that transfers the ligand between the extracellular vestibule and Asp3.32.  We have also performed interaction and cluster analyses to det. differences in binding pathways between the D2 and D3 receptors and identified metastable states that may be of use in drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpimYd-xEW9a7Vg90H21EOLACvtfcHk0liI3zptLYNQMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitVGiurzO&md5=b5faf13609b11b8efcfebd3eb5f0a77e</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.5b00457&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.5b00457%26sid%3Dliteratum%253Aachs%26aulast%3DThomas%26aufirst%3DT.%26aulast%3DFang%26aufirst%3DY.%26aulast%3DYuriev%26aufirst%3DE.%26aulast%3DChalmers%26aufirst%3DD.%2BK.%26atitle%3DLigand%2520binding%2520pathways%2520of%2520clozapine%2520and%2520haloperidol%2520in%2520the%2520dopamine%2520D-2%2520and%2520D-3%2520receptors%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2016%26volume%3D56%26spage%3D308%26epage%3D321%26doi%3D10.1021%2Facs.jcim.5b00457" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group">Torrie, G. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Valleau, J. P.</span><span> </span><span class="NLM_article-title">Non-physical sampling distributions in Monte-Carlo free-energy estimation - umbrella sampling</span> <span class="citation_source-journal">J. Comput. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1977</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">187</span><span class="NLM_x">–</span> <span class="NLM_lpage">199</span><span class="refDoi"> DOI: 10.1016/0021-9991(77)90121-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1016%2F0021-9991%2877%2990121-8" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=1977&pages=187-199&author=G.+M.+Torrieauthor=J.+P.+Valleau&title=Non-physical+sampling+distributions+in+Monte-Carlo+free-energy+estimation+-+umbrella+sampling&doi=10.1016%2F0021-9991%2877%2990121-8"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2F0021-9991%2877%2990121-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0021-9991%252877%252990121-8%26sid%3Dliteratum%253Aachs%26aulast%3DTorrie%26aufirst%3DG.%2BM.%26aulast%3DValleau%26aufirst%3DJ.%2BP.%26atitle%3DNon-physical%2520sampling%2520distributions%2520in%2520Monte-Carlo%2520free-energy%2520estimation%2520-%2520umbrella%2520sampling%26jtitle%3DJ.%2520Comput.%2520Phys.%26date%3D1977%26volume%3D23%26spage%3D187%26epage%3D199%26doi%3D10.1016%2F0021-9991%2877%2990121-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group">Woo, H. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roux, B.</span><span> </span><span class="NLM_article-title">Calculation of absolute protein-ligand binding free energy from computer simulations</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">102</span><span class="NLM_x">, </span> <span class="NLM_fpage">6825</span><span class="NLM_x">–</span> <span class="NLM_lpage">6830</span><span class="refDoi"> DOI: 10.1073/pnas.0409005102</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1073%2Fpnas.0409005102" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=15867154" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BD2MXksVKgurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=2005&pages=6825-6830&author=H.+J.+Wooauthor=B.+Roux&title=Calculation+of+absolute+protein-ligand+binding+free+energy+from+computer+simulations&doi=10.1073%2Fpnas.0409005102"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Calculation of absolute protein-ligand binding free energy from computer simulations</span></div><div class="casAuthors">Woo, Hyung-June; Roux, Benoit</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">102</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">6825-6830</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">A general methodol. for calcg. the equil. binding const. of a flexible ligand to a protein receptor is formulated on the basis of potentials of mean force.  The overall process is decompd. into several stages that can be computed sep.: the free ligand in the bulk is first restrained into the conformation it adopts in the bound state, position, and orientation by applying biasing potentials, then it is translated into the binding site, where it is released completely.  The conformational restraining potential is based on the root-mean-square deviation of the peptide coordinates relative to its av. conformation in the bound complex.  Free energy contributions from each stage are calcd. by free energy perturbation potential of mean force techniques by using appropriate order parameters.  The present approach avoids the need to decouple the ligand from its surrounding (bulk solvent and receptor protein) as is traditionally performed in the double-decoupling scheme.  It is believed that the present formulation will be particularly useful when the solvation free energy of the ligand is very large.  As an application, the equil. binding const. of the phosphotyrosine peptide pYEEI to the Src homol. 2 domain of human Lck has been calcd.  The results are in good agreement with exptl. values.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoktN0FgqFW_LVg90H21EOLACvtfcHk0liI3zptLYNQMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXksVKgurw%253D&md5=23be06c80a875efaff0207ea8dc2b08d</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0409005102&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0409005102%26sid%3Dliteratum%253Aachs%26aulast%3DWoo%26aufirst%3DH.%2BJ.%26aulast%3DRoux%26aufirst%3DB.%26atitle%3DCalculation%2520of%2520absolute%2520protein-ligand%2520binding%2520free%2520energy%2520from%2520computer%2520simulations%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2005%26volume%3D102%26spage%3D6825%26epage%3D6830%26doi%3D10.1073%2Fpnas.0409005102" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group">Lee, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olson, M. A.</span><span> </span><span class="NLM_article-title">Calculation of absolute protein-ligand binding affinity using path and endpoint approaches</span> <span class="citation_source-journal">Biophys. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">90</span><span class="NLM_x">, </span> <span class="NLM_fpage">864</span><span class="NLM_x">–</span> <span class="NLM_lpage">877</span><span class="refDoi"> DOI: 10.1529/biophysj.105.071589</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1529%2Fbiophysj.105.071589" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=16284269" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtFGhs78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=90&publication_year=2006&pages=864-877&author=M.+S.+Leeauthor=M.+A.+Olson&title=Calculation+of+absolute+protein-ligand+binding+affinity+using+path+and+endpoint+approaches&doi=10.1529%2Fbiophysj.105.071589"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Calculation of absolute protein-ligand binding affinity using path and endpoint approaches</span></div><div class="casAuthors">Lee, Michael S.; Olson, Mark A.</div><div class="citationInfo"><span class="NLM_cas:title">Biophysical Journal</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">90</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">864-877</span>CODEN:
                <span class="NLM_cas:coden">BIOJAU</span>;
        ISSN:<span class="NLM_cas:issn">0006-3495</span>.
    
            (<span class="NLM_cas:orgname">Biophysical Society</span>)
        </div><div class="casAbstract">A comparative anal. is provided of rigorous and approx. methods for calcg. abs. binding affinities of two protein-ligand complexes: the FKBP protein bound with small mols. 4-hydroxy-2-butanone and FK506.  The authors' rigorous approach is an umbrella sampling technique where a potential of mean force is detd. by pulling the ligand out of the protein active site over several simulation windows.  The results of this approach agree well with exptl. obsd. binding affinities.  Also assessed is a commonly used approx. endpoint approach, which sep. ests. enthalpy, solvation free energy, and entropy.  The authors show that this endpoint approach has numerous variations, all of which are prone to crit. shortcomings.  For example, conventional harmonic and quasiharmonic entropy estn. procedures produce disparate results for the relatively simple protein-ligand systems studied in this work.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoAEk5VeXijpLVg90H21EOLACvtfcHk0liI3zptLYNQMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtFGhs78%253D&md5=3d25be5ba65e123249743538bbbf9e01</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1529%2Fbiophysj.105.071589&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1529%252Fbiophysj.105.071589%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DM.%2BS.%26aulast%3DOlson%26aufirst%3DM.%2BA.%26atitle%3DCalculation%2520of%2520absolute%2520protein-ligand%2520binding%2520affinity%2520using%2520path%2520and%2520endpoint%2520approaches%26jtitle%3DBiophys.%2520J.%26date%3D2006%26volume%3D90%26spage%3D864%26epage%3D877%26doi%3D10.1529%2Fbiophysj.105.071589" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group">Kumar, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bouzida, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Swendsen, R. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kollman, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosenberg, J. M.</span><span> </span><span class="NLM_article-title">The weighted histogram analysis method for free-energy calculations on biomolecules 0.1. The method</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1992</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">1011</span><span class="NLM_x">–</span> <span class="NLM_lpage">1021</span><span class="refDoi"> DOI: 10.1002/jcc.540130812</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1002%2Fjcc.540130812" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADyaK38XmtVynsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=1992&pages=1011-1021&author=S.+Kumarauthor=D.+Bouzidaauthor=R.+H.+Swendsenauthor=P.+A.+Kollmanauthor=J.+M.+Rosenberg&title=The+weighted+histogram+analysis+method+for+free-energy+calculations+on+biomolecules+0.1.+The+method&doi=10.1002%2Fjcc.540130812"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">The weighted histogram analysis method for free-energy calculations on biomolecules.  I.  The method</span></div><div class="casAuthors">Kumar, Shankar; Bouzida, Djamal; Swendsen, Robert H.; Kollman, Peter A.; Rosenberg, John M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">1992</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1011-21</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    </div><div class="casAbstract">The Weighted Histogram Anal. Method (WHAM), an extension of Ferrenberg and Swendsen's Multiple Histogram Technique, has been applied for the first time on complex biomol. Hamiltonians.  The method is presented here as an extension of the Umbrella Sampling method for free-energy and Potential of Mean Force calcns.  This algorithm possesses the following advantages over methods that are currently employed: (1) it provides a built-in est. of sampling errors thereby yielding objective ests. of the optimal location and length of addnl. simulations needed to achieve a desired level of precision; (2) it yields the "best" value of free energies by taking into account all the simulations so as to minimize the statistical errors; (3) in addn. to optimizing the links between simulations, it also allows multiple overlaps of probability distributions for obtaining better ests. of the free-energy differences.  By recasting the Ferrenberg-Swendsen Multiple Histogram equations in a form suitable for mol. mechanics type Hamiltonians, we have demonstrated the feasibility and robustness of this method by applying it to a test problem of the generation of the Potential of Mean Force profile of the pseudorotation phase angle of the sugar ring in deoxyadenosine.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLayYt_QpVO7Vg90H21EOLACvtfcHk0lhCZntt3fDWEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK38XmtVynsrs%253D&md5=5b2ad7410198f03025708a37c0fbe89d</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1002%2Fjcc.540130812&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.540130812%26sid%3Dliteratum%253Aachs%26aulast%3DKumar%26aufirst%3DS.%26aulast%3DBouzida%26aufirst%3DD.%26aulast%3DSwendsen%26aufirst%3DR.%2BH.%26aulast%3DKollman%26aufirst%3DP.%2BA.%26aulast%3DRosenberg%26aufirst%3DJ.%2BM.%26atitle%3DThe%2520weighted%2520histogram%2520analysis%2520method%2520for%2520free-energy%2520calculations%2520on%2520biomolecules%25200.1.%2520The%2520method%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D1992%26volume%3D13%26spage%3D1011%26epage%3D1021%26doi%3D10.1002%2Fjcc.540130812" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group">Mobley, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Graves, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chodera, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McReynolds, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dill, K. A.</span><span> </span><span class="NLM_article-title">Predicting absolute ligand binding free energies to a simple model site</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">371</span><span class="NLM_x">, </span> <span class="NLM_fpage">1118</span><span class="NLM_x">–</span> <span class="NLM_lpage">1134</span><span class="refDoi"> DOI: 10.1016/j.jmb.2007.06.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1016%2Fj.jmb.2007.06.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=17599350" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BD2sXot12rt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=371&publication_year=2007&pages=1118-1134&author=D.+L.+Mobleyauthor=A.+P.+Gravesauthor=J.+D.+Choderaauthor=A.+C.+McReynoldsauthor=B.+K.+Shoichetauthor=K.+A.+Dill&title=Predicting+absolute+ligand+binding+free+energies+to+a+simple+model+site&doi=10.1016%2Fj.jmb.2007.06.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Predicting Absolute Ligand Binding Free Energies to a Simple Model Site</span></div><div class="casAuthors">Mobley, David L.; Graves, Alan P.; Chodera, John D.; McReynolds, Andrea C.; Shoichet, Brian K.; Dill, Ken A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">371</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1118-1134</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A central challenge in structure-based ligand design is the accurate prediction of binding free energies.  Here the authors apply alchem. free energy calcns. in explicit solvent to predict ligand binding in a model cavity in T4 lysozyme.  Even in this simple site, there are challenges.  The authors made systematic improvements, beginning with single poses from docking, then including multiple poses, addnl. protein conformational changes, and using an improved charge model.  Computed abs. binding free energies had an RMS error of 1.9 kcal/mol relative to previously detd. exptl. values.  In blind prospective tests, the methods correctly discriminated between several true ligands and decoys in a set of putative binders identified by docking.  In these prospective tests, the RMS error in predicted binding free energies relative to those subsequently detd. exptl. was only 0.6 kcal/mol.  X-ray crystal structures of the new ligands bound in the cavity corresponded closely to predictions from the free energy calcns., but sometimes differed from those predicted by docking.  Finally, the authors examd. the impact of holding the protein rigid, as in docking, with a view to learning how approxns. made in docking affect accuracy and how they may be improved.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqD1-EYEODQ47Vg90H21EOLACvtfcHk0lhCZntt3fDWEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXot12rt7o%253D&md5=2480139daa5a0857c59e821d2ecbce18</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2007.06.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2007.06.002%26sid%3Dliteratum%253Aachs%26aulast%3DMobley%26aufirst%3DD.%2BL.%26aulast%3DGraves%26aufirst%3DA.%2BP.%26aulast%3DChodera%26aufirst%3DJ.%2BD.%26aulast%3DMcReynolds%26aufirst%3DA.%2BC.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DDill%26aufirst%3DK.%2BA.%26atitle%3DPredicting%2520absolute%2520ligand%2520binding%2520free%2520energies%2520to%2520a%2520simple%2520model%2520site%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2007%26volume%3D371%26spage%3D1118%26epage%3D1134%26doi%3D10.1016%2Fj.jmb.2007.06.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group">Mobley, D. L.</span><span> </span><span class="NLM_article-title">Let’s get honest about sampling</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">93</span><span class="NLM_x">–</span> <span class="NLM_lpage">95</span><span class="refDoi"> DOI: 10.1007/s10822-011-9497-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1007%2Fs10822-011-9497-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=22113833" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsV2ltr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2012&pages=93-95&author=D.+L.+Mobley&title=Let%E2%80%99s+get+honest+about+sampling&doi=10.1007%2Fs10822-011-9497-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Let's get honest about sampling</span></div><div class="casAuthors">Mobley, David L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">93-95</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Mol. simulations see widespread and increasing use in computation and mol. design, esp. within the area of mol. simulations applied to biomol. binding and interactions, our focus here.  However, force field accuracy remains a concern for many practitioners, and it is often not clear what level of accuracy is really needed for payoffs in a discovery setting.  Here, I argue that despite limitations of today's force fields, current simulation tools and force fields now provide the potential for real benefits in a variety of applications.  However, these same tools also provide irreproducible results which are often poorly interpreted.  Continued progress in the field requires more honesty in assessment and care in evaluation of simulation results, esp. with respect to convergence.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoqdpLWxgFypLVg90H21EOLACvtfcHk0lhCZntt3fDWEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsV2ltr0%253D&md5=9cbb5eb62d7f8c7d448d58b861135897</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1007%2Fs10822-011-9497-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-011-9497-y%26sid%3Dliteratum%253Aachs%26aulast%3DMobley%26aufirst%3DD.%2BL.%26atitle%3DLet%25E2%2580%2599s%2520get%2520honest%2520about%2520sampling%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2012%26volume%3D26%26spage%3D93%26epage%3D95%26doi%3D10.1007%2Fs10822-011-9497-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group">Zhu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hummer, G.</span><span> </span><span class="NLM_article-title">Convergence and error estimation in free energy calculations using the weighted histogram analysis method</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">33</span><span class="NLM_x">, </span> <span class="NLM_fpage">453</span><span class="NLM_x">–</span> <span class="NLM_lpage">465</span><span class="refDoi"> DOI: 10.1002/jcc.21989</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1002%2Fjcc.21989" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=22109354" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsV2rurjM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2012&pages=453-465&author=F.+Zhuauthor=G.+Hummer&title=Convergence+and+error+estimation+in+free+energy+calculations+using+the+weighted+histogram+analysis+method&doi=10.1002%2Fjcc.21989"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Convergence and error estimation in free energy calculations using the weighted histogram analysis method</span></div><div class="casAuthors">Zhu, Fangqiang; Hummer, Gerhard</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">453-465</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">The weighted histogram anal. method (WHAM) has become the std. technique for the anal. of umbrella sampling simulations.  The authors address the challenges: (1) of obtaining fast and accurate solns. of the coupled nonlinear WHAM equations, (2) of quantifying the statistical errors of the resulting free energies, (3) of diagnosing possible systematic errors, and (4) of optimally allocating of the computational resources.  Traditionally, the WHAM equations are solved by a fixed-point direct iteration method, despite poor convergence and possible numerical inaccuracies in the solns.  The authors instead solve the math. equiv. problem of maximizing a target likelihood function, by using superlinear numerical optimization algorithms with a significantly faster convergence rate.  To est. the statistical errors in one-dimensional free energy profiles obtained from WHAM, the authors note that for densely spaced umbrella windows with harmonic biasing potentials, the WHAM free energy profile can be approximated by a coarse-grained free energy obtained by integrating the mean restraining forces.  The statistical errors of the coarse-grained free energies can be estd. straightforwardly and then used for the WHAM results.  A generalization to multidimensional WHAM is described.  They also propose two simple statistical criteria to test the consistency between the histograms of adjacent umbrella windows, which help identify inadequate sampling and hysteresis in the degrees of freedom orthogonal to the reaction coordinate.  Together, the ests. of the statistical errors and the diagnostics of inconsistencies in the potentials of mean force provide a basis for the efficient allocation of computational resources in free energy simulations.  © 2011 Wiley Periodicals, Inc.  J Comput Chem, 2011.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7m-OnlpQeSrVg90H21EOLACvtfcHk0lg7tHmdZ_RtYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsV2rurjM&md5=e96ef6d56ab6cb619e1f7d6b2ba1fb32</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1002%2Fjcc.21989&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.21989%26sid%3Dliteratum%253Aachs%26aulast%3DZhu%26aufirst%3DF.%26aulast%3DHummer%26aufirst%3DG.%26atitle%3DConvergence%2520and%2520error%2520estimation%2520in%2520free%2520energy%2520calculations%2520using%2520the%2520weighted%2520histogram%2520analysis%2520method%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2012%26volume%3D33%26spage%3D453%26epage%3D465%26doi%3D10.1002%2Fjcc.21989" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group">Lu, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kofke, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woolf, T. B.</span><span> </span><span class="NLM_article-title">Improving the efficiency and reliability of free energy perturbation calculations using overlap sampling methods</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">28</span><span class="NLM_x">–</span> <span class="NLM_lpage">39</span><span class="refDoi"> DOI: 10.1002/jcc.10369</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1002%2Fjcc.10369" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=14634991" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A280%3ADC%252BD3srlsVCqtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2004&pages=28-39&author=N.+Luauthor=D.+A.+Kofkeauthor=T.+B.+Woolf&title=Improving+the+efficiency+and+reliability+of+free+energy+perturbation+calculations+using+overlap+sampling+methods&doi=10.1002%2Fjcc.10369"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Improving the efficiency and reliability of free energy perturbation calculations using overlap sampling methods</span></div><div class="casAuthors">Lu Nandou; Kofke David A; Woolf Thomas B</div><div class="citationInfo"><span class="NLM_cas:title">Journal of computational chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">28-39</span>
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    </div><div class="casAbstract">A challenge in free energy calculation for complex molecular systems by computer simulation is to obtain a reliable estimate within feasible computational time.  In this study, we suggest an answer to this challenge by exploring a simple method, overlap sampling (OS), for producing reliable free-energy results in an efficient way.  The formalism of the OS method is based on ensuring sampling of important overlapping phase space during perturbation calculations.  This technique samples both forward and reverse free energy perturbation (FEP) to improve the free-energy calculation.  It considers the asymmetry of the FEP calculation and features an ability to optimize both the precision and the accuracy of the measurement without affecting the simulation process itself.  The OS method is tested at two optimization levels: no optimization (simple OS), and full optimization (equivalent to Bennett's method), and compared to conventional FEP techniques, including the widely used direct FEP averaging method, on three alchemical mutation systems: (a) an anion transformation in water solution, (b) mutation between methanol and ethane, and (c) alchemical change of an adenosine molecule.  It is consistently shown that the reliability of free-energy estimates can be greatly improved using the OS techniques at both optimization levels, while the performance of Bennett's method is particularly striking.  In addition, the efficiency of a calculation can be significantly improved because the method is able to (a) converge to the right answer quickly, and (b) work for large perturbations.  The basic two-stage OS method can be extended to admit additional stages, if needed.  We suggest that the OS method can be used as a general perturbation technique for computing free energy differences in molecular simulations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQZ3pOA1ItyLkloaUqgkFHkfW6udTcc2eaInd2IQaHNr7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD3srlsVCqtw%253D%253D&md5=44b61ab8523f535d4d29ab4ba355ec91</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1002%2Fjcc.10369&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.10369%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DN.%26aulast%3DKofke%26aufirst%3DD.%2BA.%26aulast%3DWoolf%26aufirst%3DT.%2BB.%26atitle%3DImproving%2520the%2520efficiency%2520and%2520reliability%2520of%2520free%2520energy%2520perturbation%2520calculations%2520using%2520overlap%2520sampling%2520methods%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2004%26volume%3D25%26spage%3D28%26epage%3D39%26doi%3D10.1002%2Fjcc.10369" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group">Zwanzig, R. W.</span><span> </span><span class="NLM_article-title">High-temperature equation of state by a perturbation method. I. nonpolar gases</span> <span class="citation_source-journal">J. Chem. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1954</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">1420</span><span class="NLM_x">–</span> <span class="NLM_lpage">1426</span><span class="refDoi"> DOI: 10.1063/1.1740409</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1063%2F1.1740409" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADyaG2cXnsFCgsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=1954&pages=1420-1426&author=R.+W.+Zwanzig&title=High-temperature+equation+of+state+by+a+perturbation+method.+I.+nonpolar+gases&doi=10.1063%2F1.1740409"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">High-temperature equation of state by a perturbation method. I. Nonpolar gases</span></div><div class="casAuthors">Zwanzig, Robert W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">1954</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1420-6</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    </div><div class="casAbstract">A theoretical study was made of the equations of state of A and N at high temp. and ds.  The intermol. potential was of the Lennard-Jones form, with an adjustable rigid sphere cutoff.  A perturbation theory was developed, by which the thermodynamic properties of 1 system could be related to those of a slightly different system and to the difference in the intermol. potentials of the 2 systems.  The unperturbed system was a rigid-sphere fluid, and the Lennard-Jones potential was the perturbation.  The results were in fair agreement with expt. and can be used as an exptl. test of the theoretical rigid-sphere equation of state.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqUK-2Wdu4YdrVg90H21EOLACvtfcHk0lg7tHmdZ_RtYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaG2cXnsFCgsw%253D%253D&md5=8b0bce75afc2097b1c178dab250389a9</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1063%2F1.1740409&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.1740409%26sid%3Dliteratum%253Aachs%26aulast%3DZwanzig%26aufirst%3DR.%2BW.%26atitle%3DHigh-temperature%2520equation%2520of%2520state%2520by%2520a%2520perturbation%2520method.%2520I.%2520nonpolar%2520gases%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1954%26volume%3D22%26spage%3D1420%26epage%3D1426%26doi%3D10.1063%2F1.1740409" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group">Paliwal, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shirts, M. R.</span><span> </span><span class="NLM_article-title">A benchmark test set for alchemical free energy transformations and its use to quantify error in common free energy methods</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">4115</span><span class="NLM_x">–</span> <span class="NLM_lpage">4134</span><span class="refDoi"> DOI: 10.1021/ct2003995</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct2003995" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlaqtrvM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=4115-4134&author=H.+Paliwalauthor=M.+R.+Shirts&title=A+benchmark+test+set+for+alchemical+free+energy+transformations+and+its+use+to+quantify+error+in+common+free+energy+methods&doi=10.1021%2Fct2003995"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">A Benchmark Test Set for Alchemical Free Energy Transformations and Its Use to Quantify Error in Common Free Energy Methods</span></div><div class="casAuthors">Paliwal, Himanshu; Shirts, Michael R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">4115-4134</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">There is a significant need for improved tools to validate thermophys. quantities computed via mol. simulation.  In this paper we present the initial version of a benchmark set of testing methods for calcg. free energies of mol. transformation in soln.  This set is based on mol. changes common to many mol. design problems, such as insertion and deletion of at. sites and changing at. partial charges.  We use this benchmark set to compare the statistical efficiency, reliability, and quality of uncertainty ests. for a no. of published free energy methods, including thermodn. integration, free energy perturbation, the Bennett acceptance ratio (BAR) and its multistate equiv. MBAR.  We identify MBAR as the consistently best performing method, though other methods are frequently comparable in reliability and accuracy in many cases.  We demonstrate that assumptions of Gaussian distributed errors in free energies are usually valid for most methods studied.  We demonstrate that bootstrap error estn. is a robust and useful technique for estg. statistical variance for all free energy methods studied.  This benchmark set is provided in a no. of different file formats with the hope of becoming a useful and general tool for method comparisons.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxxPO-qPI-gLVg90H21EOLACvtfcHk0liUc9QWHELOng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlaqtrvM&md5=41383373300216165a7f1724ad535451</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Fct2003995&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct2003995%26sid%3Dliteratum%253Aachs%26aulast%3DPaliwal%26aufirst%3DH.%26aulast%3DShirts%26aufirst%3DM.%2BR.%26atitle%3DA%2520benchmark%2520test%2520set%2520for%2520alchemical%2520free%2520energy%2520transformations%2520and%2520its%2520use%2520to%2520quantify%2520error%2520in%2520common%2520free%2520energy%2520methods%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2011%26volume%3D7%26spage%3D4115%26epage%3D4134%26doi%3D10.1021%2Fct2003995" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group">Chodera, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mobley, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shirts, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dixon, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Branson, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pande, V. S.</span><span> </span><span class="NLM_article-title">Alchemical free energy methods for drug discovery: progress and challenges</span> <span class="citation_source-journal">Curr. Opin. Struct. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">150</span><span class="NLM_x">–</span> <span class="NLM_lpage">160</span><span class="refDoi"> DOI: 10.1016/j.sbi.2011.01.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1016%2Fj.sbi.2011.01.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=21349700" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC3MXkt1Kisrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2011&pages=150-160&author=J.+D.+Choderaauthor=D.+L.+Mobleyauthor=M.+R.+Shirtsauthor=R.+W.+Dixonauthor=K.+Bransonauthor=V.+S.+Pande&title=Alchemical+free+energy+methods+for+drug+discovery%3A+progress+and+challenges&doi=10.1016%2Fj.sbi.2011.01.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Alchemical free energy methods for drug discovery: Progress and challenges</span></div><div class="casAuthors">Chodera, John D.; Mobley, David L.; Shirts, Michael R.; Dixon, Richard W.; Branson, Kim; Pande, Vijay S.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Structural Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">150-160</span>CODEN:
                <span class="NLM_cas:coden">COSBEF</span>;
        ISSN:<span class="NLM_cas:issn">0959-440X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Improved rational drug design methods are needed to lower the cost and increase the success rate of drug discovery and development.  Alchem. binding free energy calcns., one potential tool for rational design, have progressed rapidly over the past decade, but still fall short of providing robust tools for pharmaceutical engineering.  Recent studies, esp. on model receptor systems, have clarified many of the challenges that must be overcome for robust predictions of binding affinity to be useful in rational design.  In this review, inspired by a recent joint academic/industry meeting organized by the authors, we discuss these challenges and suggest a no. of promising approaches for overcoming them.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobUBHsGUCIWLVg90H21EOLACvtfcHk0liUc9QWHELOng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXkt1Kisrk%253D&md5=fa17c0982d921cd1c32cd095fd34420e</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1016%2Fj.sbi.2011.01.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.sbi.2011.01.011%26sid%3Dliteratum%253Aachs%26aulast%3DChodera%26aufirst%3DJ.%2BD.%26aulast%3DMobley%26aufirst%3DD.%2BL.%26aulast%3DShirts%26aufirst%3DM.%2BR.%26aulast%3DDixon%26aufirst%3DR.%2BW.%26aulast%3DBranson%26aufirst%3DK.%26aulast%3DPande%26aufirst%3DV.%2BS.%26atitle%3DAlchemical%2520free%2520energy%2520methods%2520for%2520drug%2520discovery%253A%2520progress%2520and%2520challenges%26jtitle%3DCurr.%2520Opin.%2520Struct.%2520Biol.%26date%3D2011%26volume%3D21%26spage%3D150%26epage%3D160%26doi%3D10.1016%2Fj.sbi.2011.01.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group">De Vivo, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Masetti, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bottegoni, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cavalli, A.</span><span> </span><span class="NLM_article-title">Role of molecular dynamics and related methods in drug discovery</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">59</span><span class="NLM_x">, </span> <span class="NLM_fpage">4035</span><span class="NLM_x">–</span> <span class="NLM_lpage">4061</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01684</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01684" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtlWksLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=4035-4061&author=M.+De+Vivoauthor=M.+Masettiauthor=G.+Bottegoniauthor=A.+Cavalli&title=Role+of+molecular+dynamics+and+related+methods+in+drug+discovery&doi=10.1021%2Facs.jmedchem.5b01684"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Role of Molecular Dynamics and Related Methods in Drug Discovery</span></div><div class="casAuthors">De Vivo, Marco; Masetti, Matteo; Bottegoni, Giovanni; Cavalli, Andrea</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4035-4061</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mol. dynamics (MD) and related methods are close to becoming routine computational tools for drug discovery.  Their main advantage is in explicitly treating structural flexibility and entropic effects.  This allows a more accurate est. of the thermodn. and kinetics assocd. with drug-target recognition and binding, as better algorithms and hardware architectures increase their use.  Here, we review the theor. background of MD and enhanced sampling methods, focusing on free-energy perturbation, metadynamics, steered MD, and other methods most consistently used to study drug-target binding.  We discuss unbiased MD simulations that nowadays allow the observation of unsupervised ligand-target binding, assessing how these approaches help optimizing target affinity and drug residence time toward improved drug efficacy.  Further issues discussed include allosteric modulation and the role of water mols. in ligand binding and optimization.  We conclude by calling for more prospective studies to attest to these methods' utility in discovering novel drug candidates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQb_02a7Nmh7Vg90H21EOLACvtfcHk0liUc9QWHELOng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtlWksLs%253D&md5=b3d8a4fb5705a3555c2bb2a962420396</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01684%26sid%3Dliteratum%253Aachs%26aulast%3DDe%2BVivo%26aufirst%3DM.%26aulast%3DMasetti%26aufirst%3DM.%26aulast%3DBottegoni%26aufirst%3DG.%26aulast%3DCavalli%26aufirst%3DA.%26atitle%3DRole%2520of%2520molecular%2520dynamics%2520and%2520related%2520methods%2520in%2520drug%2520discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D4035%26epage%3D4061%26doi%3D10.1021%2Facs.jmedchem.5b01684" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deng, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roux, B.</span><span> </span><span class="NLM_article-title">Absolute binding free energy calculations using molecular dynamics simulations with restraining potentials</span> <span class="citation_source-journal">Biophys. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">91</span><span class="NLM_x">, </span> <span class="NLM_fpage">2798</span><span class="NLM_x">–</span> <span class="NLM_lpage">2814</span><span class="refDoi"> DOI: 10.1529/biophysj.106.084301</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1529%2Fbiophysj.106.084301" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=16844742" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtVGls7vP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=91&publication_year=2006&pages=2798-2814&author=J.+Wangauthor=Y.+Dengauthor=B.+Roux&title=Absolute+binding+free+energy+calculations+using+molecular+dynamics+simulations+with+restraining+potentials&doi=10.1529%2Fbiophysj.106.084301"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Absolute binding free energy calculations using molecular dynamics simulations with restraining potentials</span></div><div class="casAuthors">Wang, Jiyao; Deng, Yuqing; Roux, Benoit</div><div class="citationInfo"><span class="NLM_cas:title">Biophysical Journal</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">91</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2798-2814</span>CODEN:
                <span class="NLM_cas:coden">BIOJAU</span>;
        ISSN:<span class="NLM_cas:issn">0006-3495</span>.
    
            (<span class="NLM_cas:orgname">Biophysical Society</span>)
        </div><div class="casAbstract">The abs. (std.) binding free energy of eight FK506-related ligands to FKBP12 is calcd. using free energy perturbation mol. dynamics (FEP/MD) simulations with explicit solvent.  A no. of features are implemented to improve the accuracy and enhance the convergence of the calcns.  First, the abs. binding free energy is decompd. into sequential steps during which the ligand-surrounding interactions as well as various biasing potentials restraining the translation, orientation, and conformation of the ligand are turned "on" and "off.".  Second, sampling of the ligand conformation is enforced by a restraining potential based on the root mean-square deviation relative to the bound state conformation.  The effect of all the restraining potentials is rigorously unbiased, and it is shown explicitly that the final results are independent of all artificial restraints.  Third, the repulsive and dispersive free energy contribution arising from the Lennard-Jones interactions of the ligand with its surrounding (protein and solvent) is calcd. using the Weeks-Chandler-Andersen sepn.  This sepn. also improves convergence of the FEP/MD calcns.  Fourth, to decrease the computational cost, only a small no. of atoms in the vicinity of the binding site are simulated explicitly, while all the influence of the remaining atoms is incorporated implicitly using the generalized solvent boundary potential (GSBP) method.  With GSBP, the size of the simulated FKBP12/ligand systems is significantly reduced, from ∼25,000 to 2500.  The computations are very efficient and the statistical error is small (∼1 kcal/mol).  The calcd. binding free energies are generally in good agreement with available exptl. data and previous calcns. (within ∼2 kcal/mol).  The present results indicate that a strategy based on FEP/MD simulations of a reduced GSBP at. model sampled with conformational, translational, and orientational restraining potentials can be computationally inexpensive and accurate.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqsDftyP7DXobVg90H21EOLACvtfcHk0liUc9QWHELOng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtVGls7vP&md5=010f74f61048d03fee5dce41172dac70</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1529%2Fbiophysj.106.084301&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1529%252Fbiophysj.106.084301%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DDeng%26aufirst%3DY.%26aulast%3DRoux%26aufirst%3DB.%26atitle%3DAbsolute%2520binding%2520free%2520energy%2520calculations%2520using%2520molecular%2520dynamics%2520simulations%2520with%2520restraining%2520potentials%26jtitle%3DBiophys.%2520J.%26date%3D2006%26volume%3D91%26spage%3D2798%26epage%3D2814%26doi%3D10.1529%2Fbiophysj.106.084301" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group">Jayachandran, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shirts, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Park, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pande, V. S.</span><span> </span><span class="NLM_article-title">Parallelized-over-parts computation of absolute binding free energy with docking and molecular dynamics</span> <span class="citation_source-journal">J. Chem. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">125</span><span class="NLM_x">, </span> <span class="NLM_fpage">084901</span><span class="refDoi"> DOI: 10.1063/1.2221680</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1063%2F1.2221680" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=16965051" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BD28XptlCms7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=125&publication_year=2006&pages=084901&author=G.+Jayachandranauthor=M.+R.+Shirtsauthor=S.+Parkauthor=V.+S.+Pande&title=Parallelized-over-parts+computation+of+absolute+binding+free+energy+with+docking+and+molecular+dynamics&doi=10.1063%2F1.2221680"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Parallelized-over-parts computation of absolute binding free energy with docking and molecular dynamics</span></div><div class="casAuthors">Jayachandran, Guha; Shirts, Michael R.; Park, Sanghyun; Pande, Vijay S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">125</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">084901/1-084901/12</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    
            (<span class="NLM_cas:orgname">American Institute of Physics</span>)
        </div><div class="casAbstract">The authors present a technique for biomol. free energy calcns. that exploits highly parallelized sampling to significantly reduce the time to results.  The technique combines free energies for multiple, nonoverlapping configurational macrostates and is naturally suited to distributed computing.  The authors describe a methodol. that uses this technique with docking, mol. dynamics, and free energy perturbation to compute abs. free energies of binding quickly compared to previous methods.  The method does not require a priori knowledge of the binding pose as long as the docking technique used can generate reasonable binding modes.  The authors demonstrate the method on the protein FKBP12 and eight of its inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr1nPidX9GbT7Vg90H21EOLACvtfcHk0lig9a1-kqnk6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XptlCms7c%253D&md5=130cf0d8baef6d086d86d8c4b4fa673a</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1063%2F1.2221680&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.2221680%26sid%3Dliteratum%253Aachs%26aulast%3DJayachandran%26aufirst%3DG.%26aulast%3DShirts%26aufirst%3DM.%2BR.%26aulast%3DPark%26aufirst%3DS.%26aulast%3DPande%26aufirst%3DV.%2BS.%26atitle%3DParallelized-over-parts%2520computation%2520of%2520absolute%2520binding%2520free%2520energy%2520with%2520docking%2520and%2520molecular%2520dynamics%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D2006%26volume%3D125%26spage%3D084901%26doi%3D10.1063%2F1.2221680" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group">Mobley, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chodera, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dill, K. A.</span><span> </span><span class="NLM_article-title">On the use of orientational restraints and symmetry corrections in alchemical free energy calculations</span> <span class="citation_source-journal">J. Chem. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">125</span><span class="NLM_x">, </span> <span class="NLM_fpage">084902</span><span class="refDoi"> DOI: 10.1063/1.2221683</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1063%2F1.2221683" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=16965052" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BD28XptlCmsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=125&publication_year=2006&pages=084902&author=D.+L.+Mobleyauthor=J.+D.+Choderaauthor=K.+A.+Dill&title=On+the+use+of+orientational+restraints+and+symmetry+corrections+in+alchemical+free+energy+calculations&doi=10.1063%2F1.2221683"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">On the use of orientational restraints and symmetry corrections in alchemical free energy calculations</span></div><div class="casAuthors">Mobley, David L.; Chodera, John D.; Dill, Ken A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">125</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">084902/1-084902/16</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    
            (<span class="NLM_cas:orgname">American Institute of Physics</span>)
        </div><div class="casAbstract">Alchem. free energy calcns. are becoming a useful tool for calcg. abs. binding free energies of small mol. ligands to proteins.  Here, we find that the presence of multiple metastable ligand orientations can cause convergence problems when distance restraints alone are used.  We demonstrate that the use of orientational restraints can greatly accelerate the convergence of these calcns.  However, even with this acceleration, we find that sufficient sampling requires substantially longer simulations than are used in many published protocols.  To further accelerate convergence, we introduce a new method of configuration space decompn. by orientation which reduces required simulation lengths by at least a factor of 5 in the cases examd.  Our method is easily parallelizable, well suited for cases where a ligand co-crystal structure is not available, and can utilize initial orientations generated by docking packages.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXKw8APQNZDbVg90H21EOLACvtfcHk0lig9a1-kqnk6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XptlCmsL4%253D&md5=8be7d1d066ed42f067182227e952dfe1</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1063%2F1.2221683&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.2221683%26sid%3Dliteratum%253Aachs%26aulast%3DMobley%26aufirst%3DD.%2BL.%26aulast%3DChodera%26aufirst%3DJ.%2BD.%26aulast%3DDill%26aufirst%3DK.%2BA.%26atitle%3DOn%2520the%2520use%2520of%2520orientational%2520restraints%2520and%2520symmetry%2520corrections%2520in%2520alchemical%2520free%2520energy%2520calculations%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D2006%26volume%3D125%26spage%3D084902%26doi%3D10.1063%2F1.2221683" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group">Clark, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guarnieri, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shkurko, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wiseman, J.</span><span> </span><span class="NLM_article-title">Grand canonical Monte Carlo simulation of ligand-protein binding</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">231</span><span class="NLM_x">–</span> <span class="NLM_lpage">242</span><span class="refDoi"> DOI: 10.1021/ci050268f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci050268f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BD2MXht1Gju7zJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2006&pages=231-242&author=M.+Clarkauthor=F.+Guarnieriauthor=I.+Shkurkoauthor=J.+Wiseman&title=Grand+canonical+Monte+Carlo+simulation+of+ligand-protein+binding&doi=10.1021%2Fci050268f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Grand Canonical Monte Carlo Simulation of Ligand-Protein Binding</span></div><div class="casAuthors">Clark, Matthew; Guarnieri, Frank; Shkurko, Igor; Wiseman, Jeff</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">231-242</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A new application of the grand canonical thermodn. ensemble to compute ligand-protein binding is described.  The described method is sufficiently rapid that it is practical to compute ligand-protein binding free energies for a large no. of poses over the entire protein surface, thus identifying multiple putative ligand binding sites.  In addn., the method computes binding free energies for a large no. of poses.  The method is demonstrated by the simulation of two protein-ligand systems, thermolysin and T4 lysozyme, for which there is extensive thermodn. and crystallog. data for the binding of small, rigid ligands.  These low-mol.-wt. ligands correspond to the mol. fragments used in computational fragment-based drug design.  The simulations correctly identified the exptl. binding poses and rank ordered the affinities of ligands in each of these systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqD_W87xCkI47Vg90H21EOLACvtfcHk0lig9a1-kqnk6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXht1Gju7zJ&md5=6ff1670d21b2650f0eb0a113ab38a1c5</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1021%2Fci050268f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci050268f%26sid%3Dliteratum%253Aachs%26aulast%3DClark%26aufirst%3DM.%26aulast%3DGuarnieri%26aufirst%3DF.%26aulast%3DShkurko%26aufirst%3DI.%26aulast%3DWiseman%26aufirst%3DJ.%26atitle%3DGrand%2520canonical%2520Monte%2520Carlo%2520simulation%2520of%2520ligand-protein%2520binding%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2006%26volume%3D46%26spage%3D231%26epage%3D242%26doi%3D10.1021%2Fci050268f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group">Shirts, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mobley, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chodera, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pande, V. S.</span><span> </span><span class="NLM_article-title">Accurate and efficient corrections for missing dispersion interactions in molecular simulations</span> <span class="citation_source-journal">J. Phys. Chem. B</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">111</span><span class="NLM_x">, </span> <span class="NLM_fpage">13052</span><span class="NLM_x">–</span> <span class="NLM_lpage">13063</span><span class="refDoi"> DOI: 10.1021/jp0735987</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jp0735987" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFyiur3E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2007&pages=13052-13063&author=M.+R.+Shirtsauthor=D.+L.+Mobleyauthor=J.+D.+Choderaauthor=V.+S.+Pande&title=Accurate+and+efficient+corrections+for+missing+dispersion+interactions+in+molecular+simulations&doi=10.1021%2Fjp0735987"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Accurate and Efficient Corrections for Missing Dispersion Interactions in Molecular Simulations</span></div><div class="casAuthors">Shirts, Michael R.; Mobley, David L.; Chodera, John D.; Pande, Vijay S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry B</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">111</span>
        (<span class="NLM_cas:issue">45</span>),
    <span class="NLM_cas:pages">13052-13063</span>CODEN:
                <span class="NLM_cas:coden">JPCBFK</span>;
        ISSN:<span class="NLM_cas:issn">1520-6106</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  In simulations, mol. dispersion interactions are frequently neglected beyond a cutoff of around 1 nm.  In some cases, anal. corrections appropriate for isotropic systems are applied to the pressure and/or the potential energy.  Here, we show that in systems contg. macromols., either of these approaches introduce statistically significant errors in some obsd. properties; for example, the choice of cutoff can affect computed free energies of ligand binding to proteins by 1 to 2 kcal/mol.  We review current methods for eliminating this cutoff-dependent behavior of the dispersion energy and identify some situations where they fail.  We introduce two new formalisms, appropriate for binding free energy calcns., which overcome these failings, requiring minimal computational effort beyond the time required to run the original simulation.  When these cutoff approxns. are applied, which can be done after all simulations are completed, results are consistent across simulations run with different cutoffs.  In many situations, simulations can be run with even shorter cutoffs than typically used, resulting in increased computational efficiency.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo64pX9IFyCi7Vg90H21EOLACvtfcHk0lh1FLMkQNH7eg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFyiur3E&md5=1f45b806643cdc5265e94cbfa290669e</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1021%2Fjp0735987&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp0735987%26sid%3Dliteratum%253Aachs%26aulast%3DShirts%26aufirst%3DM.%2BR.%26aulast%3DMobley%26aufirst%3DD.%2BL.%26aulast%3DChodera%26aufirst%3DJ.%2BD.%26aulast%3DPande%26aufirst%3DV.%2BS.%26atitle%3DAccurate%2520and%2520efficient%2520corrections%2520for%2520missing%2520dispersion%2520interactions%2520in%2520molecular%2520simulations%26jtitle%3DJ.%2520Phys.%2520Chem.%2520B%26date%3D2007%26volume%3D111%26spage%3D13052%26epage%3D13063%26doi%3D10.1021%2Fjp0735987" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group">Boyce, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mobley, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rocklin, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Graves, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dill, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Predicting ligand binding affinity with alchemical free energy methods in a polar model binding site</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">394</span><span class="NLM_x">, </span> <span class="NLM_fpage">747</span><span class="NLM_x">–</span> <span class="NLM_lpage">763</span><span class="refDoi"> DOI: 10.1016/j.jmb.2009.09.049</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1016%2Fj.jmb.2009.09.049" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=19782087" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsVKnsbrF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=394&publication_year=2009&pages=747-763&author=S.+E.+Boyceauthor=D.+L.+Mobleyauthor=G.+J.+Rocklinauthor=A.+P.+Gravesauthor=K.+A.+Dillauthor=B.+K.+Shoichet&title=Predicting+ligand+binding+affinity+with+alchemical+free+energy+methods+in+a+polar+model+binding+site&doi=10.1016%2Fj.jmb.2009.09.049"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Predicting Ligand Binding Affinity with Alchemical Free Energy Methods in a Polar Model Binding Site</span></div><div class="casAuthors">Boyce, Sarah E.; Mobley, David L.; Rocklin, Gabriel J.; Graves, Alan P.; Dill, Ken A.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">394</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">747-763</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The authors present a combined exptl. and modeling study of org. ligand mols. binding to a slightly polar engineered cavity site in T4 lysozyme (L99A/M102Q).  For modeling, the authors computed alchem. abs. binding free energies.  These were blind tests performed prospectively on 13 diverse, previously untested candidate ligand mols.  The authors predicted that eight compds. would bind to the cavity and five would not; 11 of 13 predictions were correct at this level.  The RMS error to the measurable abs. binding energies was 1.8 kcal/mol.  In addn., the authors computed "relative" binding free energies for six phenol derivs. starting from two known ligands: phenol and catechol.  The av. RMS error in the relative free energy prediction was 2.5 kcal/mol (phenol) and 1.1 kcal/mol (catechol).  To understand these results at at. resoln., the authors obtained x-ray co-complex structures for nine of the diverse ligands and for all six phenol analogs.  The av. RMSD of the predicted pose to the expt. was 2.0 Å (diverse set), 1.8 Å (phenol-derived predictions), and 1.2 Å (catechol-derived predictions).  The authors found that predicting accurate affinities and rank-orderings required near-native starting orientations of the ligand in the binding site.  Unanticipated binding modes, multiple ligand binding, and protein conformational change all proved challenging for the free energy methods.  The authors believe that these results can help guide future improvements in physics-based abs. binding free energy methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgv5STaggJVrVg90H21EOLACvtfcHk0lh1FLMkQNH7eg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsVKnsbrF&md5=5cefcd3613993fed46fc328a905c79c9</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2009.09.049&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2009.09.049%26sid%3Dliteratum%253Aachs%26aulast%3DBoyce%26aufirst%3DS.%2BE.%26aulast%3DMobley%26aufirst%3DD.%2BL.%26aulast%3DRocklin%26aufirst%3DG.%2BJ.%26aulast%3DGraves%26aufirst%3DA.%2BP.%26aulast%3DDill%26aufirst%3DK.%2BA.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DPredicting%2520ligand%2520binding%2520affinity%2520with%2520alchemical%2520free%2520energy%2520methods%2520in%2520a%2520polar%2520model%2520binding%2520site%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2009%26volume%3D394%26spage%3D747%26epage%3D763%26doi%3D10.1016%2Fj.jmb.2009.09.049" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group">Gallicchio, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lapelosa, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Levy, R. M.</span><span> </span><span class="NLM_article-title">The binding energy distribution analysis method (BEDAM) for the estimation of protein-ligand binding affinities</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">2961</span><span class="NLM_x">–</span> <span class="NLM_lpage">2977</span><span class="refDoi"> DOI: 10.1021/ct1002913</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct1002913" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC3cXpvVGgtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2010&pages=2961-2977&author=E.+Gallicchioauthor=M.+Lapelosaauthor=R.+M.+Levy&title=The+binding+energy+distribution+analysis+method+%28BEDAM%29+for+the+estimation+of+protein-ligand+binding+affinities&doi=10.1021%2Fct1002913"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Binding Energy Distribution Analysis Method (BEDAM) for Estimation of Protein-Ligand Binding Affinities</span></div><div class="casAuthors">Gallicchio, Emilio; Lapelosa, Mauro; Levy, Ronald M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2961-2977</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The binding energy distribution anal. method (BEDAM) for the computation of receptor-ligand std. binding free energies with implicit solvation is presented.  The method is based on a well-established statistical mechanics theory of mol. assocn.  In the context of implicit solvation, the theory is homologous to the test particle method of solvation thermodn. with the solute-solvent potential represented by the effective binding energy of the protein-ligand complex.  Accordingly, in BEDAM the binding const. is computed by a weighted integral of the probability distribution of the binding energy obtained in the canonical ensemble in which the ligand is positioned in the binding site but the receptor and the ligand interact only with the solvent continuum.  The binding energy distribution encodes all of the phys. effects of binding.  The balance between binding enthalpy and entropy is seen in the authors' formalism as a balance between favorable and unfavorable binding modes which are coupled through the normalization of the binding energy distribution function.  An efficient computational protocol for the binding energy distribution based on the AGBNP2 implicit solvent model, parallel Hamiltonian replica exchange sampling, and histogram reweighting is developed.  Applications of the method to a set of known binders and nonbinders of the L99A and L99A/M102Q mutants of T4 lysozyme receptor are illustrated.  The method is able to discriminate without error binders from nonbinders, and the computed std. binding free energies of the binders are in good agreement with exptl. measurements.  Anal. of the binding affinities of these systems reflect the contributions from multiple conformations spanning a wide range of binding energies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqe_4e1f7O1XLVg90H21EOLACvtfcHk0lh1FLMkQNH7eg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXpvVGgtr0%253D&md5=190e552ee54264185012f053e42488b2</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1021%2Fct1002913&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct1002913%26sid%3Dliteratum%253Aachs%26aulast%3DGallicchio%26aufirst%3DE.%26aulast%3DLapelosa%26aufirst%3DM.%26aulast%3DLevy%26aufirst%3DR.%2BM.%26atitle%3DThe%2520binding%2520energy%2520distribution%2520analysis%2520method%2520%2528BEDAM%2529%2520for%2520the%2520estimation%2520of%2520protein-ligand%2520binding%2520affinities%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2010%26volume%3D6%26spage%3D2961%26epage%3D2977%26doi%3D10.1021%2Fct1002913" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group">Fujitani, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tanida, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matsuura, A.</span><span> </span><span class="NLM_article-title">Massively parallel computation of absolute binding free energy with well-equilibrated states</span> <span class="citation_source-journal">Phys. Rev. E: Stat., Nonlinear, Soft Matter Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">79</span><span class="NLM_x">, </span> <span class="NLM_fpage">021914</span><span class="refDoi"> DOI: 10.1103/PhysRevE.79.021914</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1103%2FPhysRevE.79.021914" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=19391785" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BD1MXisFOjsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=2009&pages=021914&author=H.+Fujitaniauthor=Y.+Tanidaauthor=A.+Matsuura&title=Massively+parallel+computation+of+absolute+binding+free+energy+with+well-equilibrated+states&doi=10.1103%2FPhysRevE.79.021914"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Massively parallel computation of absolute binding free energy with well-equilibrated states</span></div><div class="casAuthors">Fujitani, Hideaki; Tanida, Yoshiaki; Matsuura, Azuma</div><div class="citationInfo"><span class="NLM_cas:title">Physical Review E: Statistical, Nonlinear, and Soft Matter Physics</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">79</span>
        (<span class="NLM_cas:issue">2-1</span>),
    <span class="NLM_cas:pages">021914/1-021914/12</span>CODEN:
                <span class="NLM_cas:coden">PRESCM</span>;
        ISSN:<span class="NLM_cas:issn">1539-3755</span>.
    
            (<span class="NLM_cas:orgname">American Physical Society</span>)
        </div><div class="casAbstract">A force field formulator for org. mols. (FF-FOM) was developed to assign bond, angle, and dihedral parameters to arbitrary org. mols. in a unified manner including proteins and nucleic acids.  With the unified force field parametrization we performed massively parallel computations of abs. binding free energies for pharmaceutical target proteins and ligands.  Compared with the previous calcn. with the ff99 force field in the Amber simulation package (Amber99) and the ligand charges produced by the Austin Model 1 bond charge correction (AM1-BCC), the unified parametrization gave better abs. binding energies for the FK506 binding protein (FKBP) and ligand system.  Our method is based on extensive work measurement between thermodn. states to calc. the free energy difference and it is also the same as the traditional free energy perturbation.  There are important requirements for accurate calcns.  The first is a well-equilibrated bound structure including the conformational change of the protein induced by the binding of the ligand.  The second requirement is the convergence of the work distribution with a sufficient no. of trajectories and dense spacing of the coupling const. between the ligand and the rest of the system.  Finally, the most important requirement is the force field parametrization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrl-MtVRv_gpbVg90H21EOLACvtfcHk0lgAZZ127H-0Eg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXisFOjsbY%253D&md5=4aca64ba22cff055f1cd39619f798fda</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1103%2FPhysRevE.79.021914&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1103%252FPhysRevE.79.021914%26sid%3Dliteratum%253Aachs%26aulast%3DFujitani%26aufirst%3DH.%26aulast%3DTanida%26aufirst%3DY.%26aulast%3DMatsuura%26aufirst%3DA.%26atitle%3DMassively%2520parallel%2520computation%2520of%2520absolute%2520binding%2520free%2520energy%2520with%2520well-equilibrated%2520states%26jtitle%3DPhys.%2520Rev.%2520E%253A%2520Stat.%252C%2520Nonlinear%252C%2520Soft%2520Matter%2520Phys.%26date%3D2009%26volume%3D79%26spage%3D021914%26doi%3D10.1103%2FPhysRevE.79.021914" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group">Ytreberg, F. M.</span><span> </span><span class="NLM_article-title">Absolute FKBP binding affinities obtained via nonequilibrium unbinding simulations</span> <span class="citation_source-journal">J. Chem. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">130</span><span class="NLM_x">, </span> <span class="NLM_fpage">164906</span><span class="refDoi"> DOI: 10.1063/1.3119261</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1063%2F1.3119261" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=19405629" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BD1MXltV2nsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=130&publication_year=2009&pages=164906&author=F.+M.+Ytreberg&title=Absolute+FKBP+binding+affinities+obtained+via+nonequilibrium+unbinding+simulations&doi=10.1063%2F1.3119261"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Absolute FKBP binding affinities obtained via nonequilibrium unbinding simulations</span></div><div class="casAuthors">Ytreberg, F. Marty</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">130</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">164906/1-164906/8</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    
            (<span class="NLM_cas:orgname">American Institute of Physics</span>)
        </div><div class="casAbstract">We compute the abs. binding affinities for two ligands bound to the FKBP protein using nonequil. unbinding simulations.  The methodol. is straightforward, requiring little or no modification to many modern mol. simulation packages.  The approach makes use of a phys. pathway, eliminating the need for complicated alchem. decoupling schemes.  We compare our nonequil. results to those obtained via a fully equil. approach and to expt.  The results of this study suggest that to obtain accurate results using nonequil. approaches one should use the stiff-spring approxn. with the second cumulant expansion.  From this study we conclude that nonequil. simulation could provide a simple means to est. protein-ligand binding affinities.  (c) 2009 American Institute of Physics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbsbHuKp3bD7Vg90H21EOLACvtfcHk0lgAZZ127H-0Eg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXltV2nsbY%253D&md5=570ec255ee04e2d7884da4448eb5bbc4</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1063%2F1.3119261&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.3119261%26sid%3Dliteratum%253Aachs%26aulast%3DYtreberg%26aufirst%3DF.%2BM.%26atitle%3DAbsolute%2520FKBP%2520binding%2520affinities%2520obtained%2520via%2520nonequilibrium%2520unbinding%2520simulations%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D2009%26volume%3D130%26spage%3D164906%26doi%3D10.1063%2F1.3119261" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group">Skouteris, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Calderini, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barone, V.</span><span> </span><span class="NLM_article-title">Methods for calculating partition functions of molecules involving large amplitude and/or anharmonic motions</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">1011</span><span class="NLM_x">–</span> <span class="NLM_lpage">1018</span><span class="refDoi"> DOI: 10.1021/acs.jctc.5b01094</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jctc.5b01094" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC28XnvFCkug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=1011-1018&author=D.+Skouterisauthor=D.+Calderiniauthor=V.+Barone&title=Methods+for+calculating+partition+functions+of+molecules+involving+large+amplitude+and%2For+anharmonic+motions&doi=10.1021%2Facs.jctc.5b01094"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Methods for Calculating Partition Functions of Molecules Involving Large Amplitude and/or Anharmonic Motions</span></div><div class="casAuthors">Skouteris, D.; Calderini, D.; Barone, V.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1011-1018</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We present a method for calcg. partition functions taking into account anharmonic contributions for systems involving both small-amplitude vibrations and hindered rotations.  The Wang-Landau scheme is used in the first case, while two alternative schemes are used for hindered rotation based on imaginary time propagation and fitting of the exact energy levels as a function of quantum no.  These two schemes are shown to be complementary in their ranges of applicability (in terms of the torsional rotational const. and the relevant potential).  Partition functions for four different mols. are calcd. and compared to simpler ones obtained using a harmonic model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpgUPwtr0e_lrVg90H21EOLACvtfcHk0lgAZZ127H-0Eg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XnvFCkug%253D%253D&md5=6471c27f9d9d2baf764e20a1439eef8c</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1021%2Facs.jctc.5b01094&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jctc.5b01094%26sid%3Dliteratum%253Aachs%26aulast%3DSkouteris%26aufirst%3DD.%26aulast%3DCalderini%26aufirst%3DD.%26aulast%3DBarone%26aufirst%3DV.%26atitle%3DMethods%2520for%2520calculating%2520partition%2520functions%2520of%2520molecules%2520involving%2520large%2520amplitude%2520and%252For%2520anharmonic%2520motions%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2016%26volume%3D12%26spage%3D1011%26epage%3D1018%26doi%3D10.1021%2Facs.jctc.5b01094" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group">Boresch, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tettinger, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leitgeb, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karplus, M.</span><span> </span><span class="NLM_article-title">Absolute binding free energies: A quantitative approach for their calculation</span> <span class="citation_source-journal">J. Phys. Chem. B</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">107</span><span class="NLM_x">, </span> <span class="NLM_fpage">9535</span><span class="NLM_x">–</span> <span class="NLM_lpage">9551</span><span class="refDoi"> DOI: 10.1021/jp0217839</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jp0217839" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BD3sXmsVegsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=107&publication_year=2003&pages=9535-9551&author=S.+Boreschauthor=F.+Tettingerauthor=M.+Leitgebauthor=M.+Karplus&title=Absolute+binding+free+energies%3A+A+quantitative+approach+for+their+calculation&doi=10.1021%2Fjp0217839"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Absolute Binding Free Energies: A Quantitative Approach for Their Calculation</span></div><div class="casAuthors">Boresch, Stefan; Tettinger, Franz; Leitgeb, Martin; Karplus, Martin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry B</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">107</span>
        (<span class="NLM_cas:issue">35</span>),
    <span class="NLM_cas:pages">9535-9551</span>CODEN:
                <span class="NLM_cas:coden">JPCBFK</span>;
        ISSN:<span class="NLM_cas:issn">1520-6106</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The computation of abs. binding affinities by mol. dynamics (MD) based free energy simulations is analyzed, and an exact method to carry out such a computation is presented.  The key to obtaining converged results is the introduction of suitable, auxiliary restraints to prevent the ligand from leaving the binding site when the native ligand-receptor interactions are turned off alchem.  The authors describe a versatile set of restraints that (i) can be used in MD simulations, that (ii) restricts both the position and the orientation of the ligand, and that (iii) is defined relative to the receptor rather than relative to a fixed point in space.  The free energy cost, ΔAr, for this set of restraints can be evaluated anal.  Although the techniques were originally developed for the gas phase, the resulting expression is exact, since all contributions from solute-solvent interactions cancel from the final result.  The value of ΔAr depends only on the equil. values and force consts. of the chosen harmonic restraint terms and, therefore, can be easily calcd.  The std. state dependence of binding free energies is also investigated, and it is shown that the present approach takes this into account correctly.  The anal. expression for ΔAr is verified numerically by calcns. on the complex formed by benzene with the L99A mutant of T4 lysozyme.  The overall approach is illustrated by a complete binding free energy calcn. for a complex based on a simplified model for tyrosine bound to tyrosyl-tRNA-synthetase.  The results demonstrate the usefulness of the proposed set of restraints and confirm that the calcd. binding free energy is independent of the details of the restraints.  Comparisons are made with earlier formulations for the calcn. of binding free energies, and certain limitations of that work are described.  The relationship between ΔAr and the loss of translational and rotational entropy during a binding process is analyzed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDzHeD7X8aRbVg90H21EOLACvtfcHk0li72FAQxnzDuA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXmsVegsr4%253D&md5=a6b92d5349e6a9bd5912f528dc948ce0</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1021%2Fjp0217839&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp0217839%26sid%3Dliteratum%253Aachs%26aulast%3DBoresch%26aufirst%3DS.%26aulast%3DTettinger%26aufirst%3DF.%26aulast%3DLeitgeb%26aufirst%3DM.%26aulast%3DKarplus%26aufirst%3DM.%26atitle%3DAbsolute%2520binding%2520free%2520energies%253A%2520A%2520quantitative%2520approach%2520for%2520their%2520calculation%26jtitle%3DJ.%2520Phys.%2520Chem.%2520B%26date%3D2003%26volume%3D107%26spage%3D9535%26epage%3D9551%26doi%3D10.1021%2Fjp0217839" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group">Brown, S. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muchmore, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hajduk, P. J.</span><span> </span><span class="NLM_article-title">Healthy skepticism: assessing realistic model performance</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">420</span><span class="NLM_x">–</span> <span class="NLM_lpage">427</span><span class="refDoi"> DOI: 10.1016/j.drudis.2009.01.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1016%2Fj.drudis.2009.01.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=19340931" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A280%3ADC%252BD1M3ltlaqsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2009&pages=420-427&author=S.+P.+Brownauthor=S.+W.+Muchmoreauthor=P.+J.+Hajduk&title=Healthy+skepticism%3A+assessing+realistic+model+performance&doi=10.1016%2Fj.drudis.2009.01.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Healthy skepticism: assessing realistic model performance</span></div><div class="casAuthors">Brown Scott P; Muchmore Steven W; Hajduk Philip J</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">7-8</span>),
    <span class="NLM_cas:pages">420-7</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Although the development of computational models to aid drug discovery has become an integral part of pharmaceutical research, the application of these models often fails to produce the expected impact on productivity.  One reason for this may be that the expected performance of many models is simply not supported by the underlying data, because of often neglected effects of assay and prediction errors on the reliability of the predicted outcome.  Another significant challenge to realizing the full potential of computational models is their integration into prospective medicinal chemistry campaigns.  This article will analyze the impact of assay and prediction error on model quality, and explore scenarios where computational models can expect to have a significant influence on drug discovery research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRCGY9Wj1ldXiCEBeYuoi7AfW6udTcc2eblIs8rz4LdnLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1M3ltlaqsQ%253D%253D&md5=6ca35f48ea556a47040f91e4d84e4b7a</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2009.01.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2009.01.012%26sid%3Dliteratum%253Aachs%26aulast%3DBrown%26aufirst%3DS.%2BP.%26aulast%3DMuchmore%26aufirst%3DS.%2BW.%26aulast%3DHajduk%26aufirst%3DP.%2BJ.%26atitle%3DHealthy%2520skepticism%253A%2520assessing%2520realistic%2520model%2520performance%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2009%26volume%3D14%26spage%3D420%26epage%3D427%26doi%3D10.1016%2Fj.drudis.2009.01.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group">Reddy, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Erion, M. D.</span><span> </span><span class="NLM_article-title">Calculation of relative binding free energy differences for fructose 1,6-bisphosphatase inhibitors using the thermodynamic cycle perturbation approach</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">123</span><span class="NLM_x">, </span> <span class="NLM_fpage">6246</span><span class="NLM_x">–</span> <span class="NLM_lpage">6252</span><span class="refDoi"> DOI: 10.1021/ja0103288</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja0103288" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BD3MXktFamu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=123&publication_year=2001&pages=6246-6252&author=M.+R.+Reddyauthor=M.+D.+Erion&title=Calculation+of+relative+binding+free+energy+differences+for+fructose+1%2C6-bisphosphatase+inhibitors+using+the+thermodynamic+cycle+perturbation+approach&doi=10.1021%2Fja0103288"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Calculation of Relative Binding Free Energy Differences for Fructose 1,6-Bisphosphatase Inhibitors Using the Thermodynamic Cycle Perturbation Approach</span></div><div class="casAuthors">Reddy, M. Rami; Erion, Mark D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">123</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">6246-6252</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">An iterative, computer-assisted, drug design strategy that combines mol. design, mol. mechanics, mol. dynamics (MD), and free energy perturbation (FEP) calcns. with compd. synthesis, biochem. testing of inhibitors, and crystallog. structure detn. of protein-inhibitor complexes was successfully used to predict the rank order of a series of nucleoside monophosphate analogs as fructose 1,6-bisphosphatase (FBPase) inhibitors.  The x-ray structure of FBPase complexed with 5-aminoimidazole-4-carboxamide-1-β-D-ribofuranosyl 5'-monophosphate (ZMP) provided structural information used for subsequent analog design and free energy calcns.  The FEP protocol was validated by calcg. the free energy differences for the mutation of ZMP to AMP.  The calcd. results showed a net gain of 1.7 kcal/mol, which agreed with the exptl. result of 1.3 kcal/mol. FEP calcns. were performed for 18 other AMP analogs.  Inhibition consts. were detd. for over half of these analogs, usually after completion of the calcn., and were consistent with the predictions.  Solvation free energy differences between AMP and various AMP analogs proved to be an important factor in binding free energies, suggesting that increased desolvation costs assocd. with the addn. of polar groups to an inhibitor must be overcome by stronger ligand-protein interactions if the structural modification is to enhance inhibitor potency.  The results indicate that FEP calcns. predict relative binding affinities with high accuracy and provide valuable insight into the factors that influence inhibitor binding and therefore should greatly aid efforts to optimize initial lead compds. and reduce the time required for the discovery of new drug candidates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9zsaQP6gdLrVg90H21EOLACvtfcHk0li72FAQxnzDuA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXktFamu70%253D&md5=976a8de912c52216e26fd84a106dbca4</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1021%2Fja0103288&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja0103288%26sid%3Dliteratum%253Aachs%26aulast%3DReddy%26aufirst%3DM.%2BR.%26aulast%3DErion%26aufirst%3DM.%2BD.%26atitle%3DCalculation%2520of%2520relative%2520binding%2520free%2520energy%2520differences%2520for%2520fructose%25201%252C6-bisphosphatase%2520inhibitors%2520using%2520the%2520thermodynamic%2520cycle%2520perturbation%2520approach%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2001%26volume%3D123%26spage%3D6246%26epage%3D6252%26doi%3D10.1021%2Fja0103288" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group">Michel, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verdonk, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Essex, J. W.</span><span> </span><span class="NLM_article-title">Protein-ligand complexes: computation of the relative free energy of different scaffolds and binding modes</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">1645</span><span class="NLM_x">–</span> <span class="NLM_lpage">1655</span><span class="refDoi"> DOI: 10.1021/ct700081t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct700081t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BD2sXos1Srsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2007&pages=1645-1655&author=J.+Michelauthor=M.+L.+Verdonkauthor=J.+W.+Essex&title=Protein-ligand+complexes%3A+computation+of+the+relative+free+energy+of+different+scaffolds+and+binding+modes&doi=10.1021%2Fct700081t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Protein-Ligand Complexes: Computation of the Relative Free Energy of Different Scaffolds and Binding Modes</span></div><div class="casAuthors">Michel, Julien; Verdonk, Marcel L.; Essex, Jonathan W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1645-1655</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A methodol. for the calcn. of the free energy difference between a pair of mols. of arbitrary topol. is proposed.  The protocol relies on a dual-topol. paradigm, a softening of the intermol. interactions, and a constraint that prevents the perturbed mols. from drifting away from each other at the end states.  The equivalence and the performance of the methodol. against a single-topol. approach are demonstrated on a pair of harmonic oscillators, the calcn. of the relative solvation free energy of ethane and methanol, and the relative binding free energy of two congeneric inhibitors of cyclooxygenase 2.  The stability of two alternative binding modes of an inhibitor of cyclin-dependent kinase 2 is then investigated.  Finally, the relative binding free energy of two structurally different inhibitors of cyclin-dependent kinase 2 is calcd.  The proposed methodol. allows the study of a range of problems that are beyond the reach of traditional relative free energy calcn. protocols and should prove useful in drug design studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7vFYftcaZaLVg90H21EOLACvtfcHk0lh2xeztYDtXyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXos1Srsbw%253D&md5=ae0a799b4119091b801471cd8073b6db</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1021%2Fct700081t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct700081t%26sid%3Dliteratum%253Aachs%26aulast%3DMichel%26aufirst%3DJ.%26aulast%3DVerdonk%26aufirst%3DM.%2BL.%26aulast%3DEssex%26aufirst%3DJ.%2BW.%26atitle%3DProtein-ligand%2520complexes%253A%2520computation%2520of%2520the%2520relative%2520free%2520energy%2520of%2520different%2520scaffolds%2520and%2520binding%2520modes%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2007%26volume%3D3%26spage%3D1645%26epage%3D1655%26doi%3D10.1021%2Fct700081t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group">Mutyala, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reddy, R. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sumakanth, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reddanna, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reddy, M. R.</span><span> </span><span class="NLM_article-title">Calculation of relative binding affinities of fructose 1,6-bisphosphatase mutants with adenosine monophosphate using free energy perturbation method</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">932</span><span class="NLM_x">–</span> <span class="NLM_lpage">937</span><span class="refDoi"> DOI: 10.1002/jcc.20617</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1002%2Fjcc.20617" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=17253638" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BD2sXjtFCit7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2007&pages=932-937&author=R.+Mutyalaauthor=R.+N.+Reddyauthor=M.+Sumakanthauthor=P.+Reddannaauthor=M.+R.+Reddy&title=Calculation+of+relative+binding+affinities+of+fructose+1%2C6-bisphosphatase+mutants+with+adenosine+monophosphate+using+free+energy+perturbation+method&doi=10.1002%2Fjcc.20617"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Calculation of relative binding affinities of fructose 1,6-bisphosphatase mutants with adenosine monophosphate using free energy perturbation method</span></div><div class="casAuthors">Mutyala, Ravichandra; Reddy, R. N.; Sumakanth, M.; Reddanna, P.; Reddy, M. Rami</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">932-937</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">The free energy perturbation (FEP) methodol. is the most accurate means of estg. relative binding affinities between inhibitors and protein variants.  In this article, the importance of hydrophobic and hydrophilic residues to the binding of adenosine monophosphate (AMP) to the fructose 1,6-bisphosphatase (FBPase), a target enzyme for type-II diabetes, was examd. by FEP method.  Five mutations were made to the FBPase enzyme with AMP inhibitor bound: 113Tyr → 113Phe, 31Thr → 31Ala, 31Thr → 31Ser, 177Met → 177Ala, and 30Leu → 30Phe.  These mutations test the strength of hydrogen bonds and Van der Waals interactions between the ligand and enzyme.  The calcd. relative free energies indicated that: 113Tyr and 31Thr play an important role, each via two hydrogen bonds affecting the binding affinity of inhibitor AMP to FBPase, and any changes in these hydrogen bonds due to mutations on the protein will have significant effect on the binding affinity of AMP to FBPase, consistent to exptl. results.  Also, the free energy calcns. clearly show that the hydrophilic interactions are more important than the hydrophobic interactions of the binding pocket of FBPase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6IknNoSsHVbVg90H21EOLACvtfcHk0lh2xeztYDtXyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXjtFCit7o%253D&md5=a04352752c6bcbe5f23820b7e48152ac</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1002%2Fjcc.20617&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.20617%26sid%3Dliteratum%253Aachs%26aulast%3DMutyala%26aufirst%3DR.%26aulast%3DReddy%26aufirst%3DR.%2BN.%26aulast%3DSumakanth%26aufirst%3DM.%26aulast%3DReddanna%26aufirst%3DP.%26aulast%3DReddy%26aufirst%3DM.%2BR.%26atitle%3DCalculation%2520of%2520relative%2520binding%2520affinities%2520of%2520fructose%25201%252C6-bisphosphatase%2520mutants%2520with%2520adenosine%2520monophosphate%2520using%2520free%2520energy%2520perturbation%2520method%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2007%26volume%3D28%26spage%3D932%26epage%3D937%26doi%3D10.1002%2Fjcc.20617" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group">Reddy, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Erion, M. D.</span><span> </span><span class="NLM_article-title">Relative binding affinities of fructose-1,6-bisphosphatase inhibitors calculated using a quantum mechanics-based free energy perturbation method</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">129</span><span class="NLM_x">, </span> <span class="NLM_fpage">9296</span><span class="NLM_x">–</span> <span class="NLM_lpage">9297</span><span class="refDoi"> DOI: 10.1021/ja072905j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja072905j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=129&publication_year=2007&pages=9296-9297&author=M.+R.+Reddyauthor=M.+D.+Erion&title=Relative+binding+affinities+of+fructose-1%2C6-bisphosphatase+inhibitors+calculated+using+a+quantum+mechanics-based+free+energy+perturbation+method&doi=10.1021%2Fja072905j"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1021%2Fja072905j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja072905j%26sid%3Dliteratum%253Aachs%26aulast%3DReddy%26aufirst%3DM.%2BR.%26aulast%3DErion%26aufirst%3DM.%2BD.%26atitle%3DRelative%2520binding%2520affinities%2520of%2520fructose-1%252C6-bisphosphatase%2520inhibitors%2520calculated%2520using%2520a%2520quantum%2520mechanics-based%2520free%2520energy%2520perturbation%2520method%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2007%26volume%3D129%26spage%3D9296%26epage%3D9297%26doi%3D10.1021%2Fja072905j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group">Lee, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Park, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, S.</span><span> </span><span class="NLM_article-title">Molecular dynamics free energy simulation study to rationalize the relative activities of PPAR delta agonists</span> <span class="citation_source-journal">Bull. Korean Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x">, </span> <span class="NLM_fpage">363</span><span class="NLM_x">–</span> <span class="NLM_lpage">371</span><span class="refDoi"> DOI: 10.5012/bkcs.2008.29.2.363</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.5012%2Fbkcs.2008.29.2.363" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BD1cXjvVaktrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2008&pages=363-371&author=W.+Leeauthor=H.+Parkauthor=S.+Lee&title=Molecular+dynamics+free+energy+simulation+study+to+rationalize+the+relative+activities+of+PPAR+delta+agonists&doi=10.5012%2Fbkcs.2008.29.2.363"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular dynamics free energy simulation study to rationalize the relative activities of PPARδ agonists</span></div><div class="casAuthors">Lee, Woojin; Park, Hwangseo; Lee, Sangyoub</div><div class="citationInfo"><span class="NLM_cas:title">Bulletin of the Korean Chemical Society</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">363-371</span>CODEN:
                <span class="NLM_cas:coden">BKCSDE</span>;
        ISSN:<span class="NLM_cas:issn">0253-2964</span>.
    
            (<span class="NLM_cas:orgname">Korean Chemical Society</span>)
        </div><div class="casAbstract">As a computational method for the discovery of the effective agonists for PPARδ, we address the usefulness of mol. dynamics free energy (MDFE) simulation with explicit solvent in terms of the accuracy and the computing cost.  For this purpose, we establish an efficient computational protocol of thermodn. integration (TI) that is superior to free energy perturbation (FEP) method in parallel computing environment.  Using this protocol, the relative binding affinities of GW501516 and its derivs. for PPARδ are calcd.  The accuracy of our protocol was evaluated in two steps.  First, we devise a thermodn. cycle to calc. the abs. and relative hydration free energies of test mols.  This allows a self-consistent check for the accuracy of the calcn. protocol.  Second, the calcd. relative binding affinities of the selected ligands are compared with exptl. IC50 values.  The av. deviation of the calcd. binding free energies from the exptl. results amts. at the most to 1 kcal/mol.  The computational efficiency of current protocol is also assessed by comparing its execution times with those of the sequential version of the TI protocol.  The results show that the calcn. can be accelerated by 4 times when compared to the sequential run.  Based on the calcns. with the parallel computational protocol, a new potential agonist of GW501516 deriv. is proposed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqewQSkpQBhkLVg90H21EOLACvtfcHk0lh2xeztYDtXyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXjvVaktrw%253D&md5=e1efc86bd9c5eca3c1c25e6950db9add</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.5012%2Fbkcs.2008.29.2.363&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.5012%252Fbkcs.2008.29.2.363%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DW.%26aulast%3DPark%26aufirst%3DH.%26aulast%3DLee%26aufirst%3DS.%26atitle%3DMolecular%2520dynamics%2520free%2520energy%2520simulation%2520study%2520to%2520rationalize%2520the%2520relative%2520activities%2520of%2520PPAR%2520delta%2520agonists%26jtitle%3DBull.%2520Korean%2520Chem.%2520Soc.%26date%3D2008%26volume%3D29%26spage%3D363%26epage%3D371%26doi%3D10.5012%2Fbkcs.2008.29.2.363" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit63"><span><span class="NLM_contrib-group">Shine, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kikuchi, T.</span><span> </span><span class="NLM_article-title">Estimation of relative binding free energy based on a free energy variational principle for quantitative structure activity relationship analyses</span> <span class="citation_source-journal">Chem. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">365</span><span class="NLM_x">, </span> <span class="NLM_fpage">53</span><span class="NLM_x">–</span> <span class="NLM_lpage">59</span><span class="refDoi"> DOI: 10.1016/j.chemphys.2009.09.023</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1016%2Fj.chemphys.2009.09.023" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=365&publication_year=2009&pages=53-59&author=Y.+Shineauthor=T.+Kikuchi&title=Estimation+of+relative+binding+free+energy+based+on+a+free+energy+variational+principle+for+quantitative+structure+activity+relationship+analyses&doi=10.1016%2Fj.chemphys.2009.09.023"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1016%2Fj.chemphys.2009.09.023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chemphys.2009.09.023%26sid%3Dliteratum%253Aachs%26aulast%3DShine%26aufirst%3DY.%26aulast%3DKikuchi%26aufirst%3DT.%26atitle%3DEstimation%2520of%2520relative%2520binding%2520free%2520energy%2520based%2520on%2520a%2520free%2520energy%2520variational%2520principle%2520for%2520quantitative%2520structure%2520activity%2520relationship%2520analyses%26jtitle%3DChem.%2520Phys.%26date%3D2009%26volume%3D365%26spage%3D53%26epage%3D59%26doi%3D10.1016%2Fj.chemphys.2009.09.023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref64"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref64'); return false;" data-citation="" class="refNumLink">64</a></strong><div class="NLM_citation" id="cit64"><span><span class="NLM_contrib-group">Zhao, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caplan, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Noskov, S. Y.</span><span> </span><span class="NLM_article-title">Evaluations of the absolute and relative free energies for antidepressant binding to the amino acid membrane transporter LeuT with free energy simulations</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">1900</span><span class="NLM_x">–</span> <span class="NLM_lpage">1914</span><span class="refDoi"> DOI: 10.1021/ct9006597</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct9006597" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC3cXltlygtL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2010&pages=1900-1914&author=C.+Zhaoauthor=D.+A.+Caplanauthor=S.+Y.+Noskov&title=Evaluations+of+the+absolute+and+relative+free+energies+for+antidepressant+binding+to+the+amino+acid+membrane+transporter+LeuT+with+free+energy+simulations&doi=10.1021%2Fct9006597"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Evaluations of the Absolute and Relative Free Energies for Antidepressant Binding to the Amino Acid Membrane Transporter LeuT with Free Energy Simulations</span></div><div class="casAuthors">Zhao, Chunfeng; Caplan, David A.; Noskov, Sergei Yu.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1900-1914</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The binding of ligands to protein receptors with high affinity and specificity is central to many cellular processes.  The quest for the development of computational models capable of accurately evaluating binding affinity remains one of the main goals of modern computational biophysics.  In this work, free energy perturbation/mol. dynamics simulations were used to evaluate abs. and relative binding affinity for three different antidepressants to a sodium-dependent membrane transporter, LeuT, a bacterial homolog of human serotonin and dopamine transporters.  Dysfunction of these membrane transporters in mammals has been implicated in multiple diseases of the nervous system, including bipolar disorder and depression.  Furthermore, these proteins are key targets for antidepressants including fluoxetine (aka Prozac) and tricyclic antidepressants known to block transport activity.  In addn. to being clin. relevant, this system, where multiple crystal structures are readily available, represents an ideal testing ground for methods used to study the mol. mechanisms of ligand binding to membrane proteins.  We discuss possible pitfalls and different levels of approxn. required to evaluate binding affinity, such as the dependence of the computed affinities on the strength of constraints and the sensitivity of the computed affinities to the particular partial charges derived from restrained electrostatic potential fitting of quantum mechanics electrostatic potential.  Finally, we compare the effects of different constraint schemes on the abs. and relative binding affinities obtained from free energy simulations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrIGzu_SqTUlLVg90H21EOLACvtfcHk0lipID_4bs5hZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXltlygtL4%253D&md5=bf8f2709f5b82ebb685ac5da0a748c56</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1021%2Fct9006597&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct9006597%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DC.%26aulast%3DCaplan%26aufirst%3DD.%2BA.%26aulast%3DNoskov%26aufirst%3DS.%2BY.%26atitle%3DEvaluations%2520of%2520the%2520absolute%2520and%2520relative%2520free%2520energies%2520for%2520antidepressant%2520binding%2520to%2520the%2520amino%2520acid%2520membrane%2520transporter%2520LeuT%2520with%2520free%2520energy%2520simulations%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2010%26volume%3D6%26spage%3D1900%26epage%3D1914%26doi%3D10.1021%2Fct9006597" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref65"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref65'); return false;" data-citation="" class="refNumLink">65</a></strong><div class="NLM_citation" id="cit65"><span><span class="NLM_contrib-group">Palma, P. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bonifacio, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Loureiro, A. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soares-da-Silva, P.</span><span> </span><span class="NLM_article-title">Computation of the binding affinities of catechol-O-methyltransferase inhibitors: multisubstate relative free energy calculations</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">33</span><span class="NLM_x">, </span> <span class="NLM_fpage">970</span><span class="NLM_x">–</span> <span class="NLM_lpage">986</span><span class="refDoi"> DOI: 10.1002/jcc.22926</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1002%2Fjcc.22926" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=22278964" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1ygtbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2012&pages=970-986&author=P.+N.+Palmaauthor=M.+J.+Bonifacioauthor=A.+I.+Loureiroauthor=P.+Soares-da-Silva&title=Computation+of+the+binding+affinities+of+catechol-O-methyltransferase+inhibitors%3A+multisubstate+relative+free+energy+calculations&doi=10.1002%2Fjcc.22926"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Computation of the binding affinities of catechol-O-methyltransferase inhibitors: Multisubstate relative free energy calculations</span></div><div class="casAuthors">Palma, P. Nuno; Bonifacio, Maria Joao; Loureiro, Ana Isabel; Soares-da-Silva, Patricio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">970-986</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">Alchem. free energy simulations are amongst the most accurate techniques for the computation of the free energy changes assocd. with noncovalent protein-ligand interactions.  A procedure is presented to est. the relative binding free energies of several ligands to the same protein target where multiple, low-energy configurational substates might coexist, as opposed to one unique structure.  The contributions of all individual substates were estd., explicitly, with the free energy perturbation method, and combined in a rigorous fashion to compute the overall relative binding free energies and dissocn. consts.  It is shown that, unless the most stable bound forms are known a priori, inaccurate results may be obtained if the contributions of multiple substates are ignored.  The method was applied to study the complex formed between human catechol-O-methyltransferase and BIA 9-1067, a newly developed tight-binding inhibitor that is currently under clin. evaluation for the therapy of Parkinson's disease.  Our results reveal an exceptionally high-binding affinity (Kd in subpicomolar range) and provide insightful clues on the interactions and mechanism of inhibition.  The inhibitor is, itself, a slowly reacting substrate of the target enzyme and is released from the complex in the form of O-methylated product.  By comparing the exptl. catalytic rate (kcat) and the estd. dissocn. rate (koff) consts. of the enzyme-inhibitor complex, one can conclude that the obsd. inhibition potency (Ki) is primarily dependent on the catalytic rate const. of the inhibitor's O-methylation, rather than the rate const. of dissocn. of the complex. © 2012 Wiley Periodicals, Inc.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTjDjuZ-BtVbVg90H21EOLACvtfcHk0lipID_4bs5hZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1ygtbk%253D&md5=548b34974096e60eefb9ad605de2d8ba</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1002%2Fjcc.22926&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.22926%26sid%3Dliteratum%253Aachs%26aulast%3DPalma%26aufirst%3DP.%2BN.%26aulast%3DBonifacio%26aufirst%3DM.%2BJ.%26aulast%3DLoureiro%26aufirst%3DA.%2BI.%26aulast%3DSoares-da-Silva%26aufirst%3DP.%26atitle%3DComputation%2520of%2520the%2520binding%2520affinities%2520of%2520catechol-O-methyltransferase%2520inhibitors%253A%2520multisubstate%2520relative%2520free%2520energy%2520calculations%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2012%26volume%3D33%26spage%3D970%26epage%3D986%26doi%3D10.1002%2Fjcc.22926" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref66"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref66'); return false;" data-citation="" class="refNumLink">66</a></strong><div class="NLM_citation" id="cit66"><span><span class="NLM_contrib-group">Klimovich, P. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mobley, D. L.</span><span> </span><span class="NLM_article-title">A python tool to set up relative free energy calculations in GROMACS</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x">, </span> <span class="NLM_fpage">1007</span><span class="NLM_x">–</span> <span class="NLM_lpage">1014</span><span class="refDoi"> DOI: 10.1007/s10822-015-9873-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1007%2Fs10822-015-9873-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=26487189" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs1yktL3E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2015&pages=1007-1014&author=P.+V.+Klimovichauthor=D.+L.+Mobley&title=A+python+tool+to+set+up+relative+free+energy+calculations+in+GROMACS&doi=10.1007%2Fs10822-015-9873-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">A Python tool to set up relative free energy calculations in GROMACS</span></div><div class="casAuthors">Klimovich, Pavel V.; Mobley, David L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1007-1014</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Free energy calcns. based on mol. dynamics (MD) simulations have seen a tremendous growth in the last decade.  However, it is still difficult and tedious to set them up in an automated manner, as the majority of the present-day MD simulation packages lack that functionality.  Relative free energy calcns. are a particular challenge for several reasons, including the problem of finding a common substructure and mapping the transformation to be applied.  Here the authors present a tool, alchem.-setup.py, that automatically generates all the input files needed to perform relative solvation and binding free energy calcns. with the MD package GROMACS.  When combined with Lead Optimization Mapper (LOMAP; Liu et al. in J Comput Aided Mol Des 27(9):755-770, 2013), recently developed in the authors' group, alchem.-setup.py allows fully automated setup of relative free energy calcns. in GROMACS.  Taking a graph of the planned calcns. and a mapping, both computed by LOMAP, the authors' tool generates the topol. and coordinate files needed to perform relative free energy calcns. for a given set of mols., and provides a set of simulation input parameters.  The tool was validated by performing relative hydration free energy calcns. for a handful of mols. from the SAMPL4 challenge (Mobley et al. in J Comput Aided Mol Des 28(4):135-150, 2014).  Good agreement with previously published results and the straightforward way in which free energy calcns. can be conducted make alchem.-setup.py a promising tool for automated setup of relative solvation and binding free energy calcns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZlJT4bLywf7Vg90H21EOLACvtfcHk0lipID_4bs5hZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs1yktL3E&md5=79a8c43be9b37ba611fda7a689951254</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1007%2Fs10822-015-9873-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-015-9873-0%26sid%3Dliteratum%253Aachs%26aulast%3DKlimovich%26aufirst%3DP.%2BV.%26aulast%3DMobley%26aufirst%3DD.%2BL.%26atitle%3DA%2520python%2520tool%2520to%2520set%2520up%2520relative%2520free%2520energy%2520calculations%2520in%2520GROMACS%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2015%26volume%3D29%26spage%3D1007%26epage%3D1014%26doi%3D10.1007%2Fs10822-015-9873-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref67"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref67'); return false;" data-citation="" class="refNumLink">67</a></strong><div class="NLM_citation" id="cit67"><span><span class="NLM_contrib-group">Wang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deng, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pierce, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krilov, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lupyan, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robinson, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dahlgren, M. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greenwood, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Romero, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Masse, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knight, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Steinbrecher, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beuming, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Damm, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harder, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sherman, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brewer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wester, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murcko, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Farid, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mobley, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jorgensen, W. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berne, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friesner, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abel, R.</span><span> </span><span class="NLM_article-title">Accurate and reliable prediction of relative ligand binding potency in prospective drug discovery by way of a modern free-energy calculation protocol and force field</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">137</span><span class="NLM_x">, </span> <span class="NLM_fpage">2695</span><span class="NLM_x">–</span> <span class="NLM_lpage">2703</span><span class="refDoi"> DOI: 10.1021/ja512751q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja512751q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsF2iuro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=137&publication_year=2015&pages=2695-2703&author=L.+Wangauthor=Y.+Wuauthor=Y.+Dengauthor=B.+Kimauthor=L.+Pierceauthor=G.+Krilovauthor=D.+Lupyanauthor=S.+Robinsonauthor=M.+K.+Dahlgrenauthor=J.+Greenwoodauthor=D.+L.+Romeroauthor=C.+Masseauthor=J.+L.+Knightauthor=T.+Steinbrecherauthor=T.+Beumingauthor=W.+Dammauthor=E.+Harderauthor=W.+Shermanauthor=M.+Brewerauthor=R.+Westerauthor=M.+Murckoauthor=L.+Fryeauthor=R.+Faridauthor=T.+Linauthor=D.+L.+Mobleyauthor=W.+L.+Jorgensenauthor=B.+J.+Berneauthor=R.+A.+Friesnerauthor=R.+Abel&title=Accurate+and+reliable+prediction+of+relative+ligand+binding+potency+in+prospective+drug+discovery+by+way+of+a+modern+free-energy+calculation+protocol+and+force+field&doi=10.1021%2Fja512751q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">Accurate and Reliable Prediction of Relative Ligand Binding Potency in Prospective Drug Discovery by Way of a Modern Free-Energy Calculation Protocol and Force Field</span></div><div class="casAuthors">Wang, Lingle; Wu, Yujie; Deng, Yuqing; Kim, Byungchan; Pierce, Levi; Krilov, Goran; Lupyan, Dmitry; Robinson, Shaughnessy; Dahlgren, Markus K.; Greenwood, Jeremy; Romero, Donna L.; Masse, Craig; Knight, Jennifer L.; Steinbrecher, Thomas; Beuming, Thijs; Damm, Wolfgang; Harder, Ed; Sherman, Woody; Brewer, Mark; Wester, Ron; Murcko, Mark; Frye, Leah; Farid, Ramy; Lin, Teng; Mobley, David L.; Jorgensen, William L.; Berne, Bruce J.; Friesner, Richard A.; Abel, Robert</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">137</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2695-2703</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Designing tight-binding ligands is a primary objective of small-mol. drug discovery.  Over the past few decades, free-energy calcns. have benefited from improved force fields and sampling algorithms, as well as the advent of low-cost parallel computing.  However, it has proven to be challenging to reliably achieve the level of accuracy that would be needed to guide lead optimization (∼5× in binding affinity) for a wide range of ligands and protein targets.  Not surprisingly, widespread com. application of free-energy simulations has been limited due to the lack of large-scale validation coupled with the tech. challenges traditionally assocd. with running these types of calcns.  Here, we report an approach that achieves an unprecedented level of accuracy across a broad range of target classes and ligands, with retrospective results encompassing 200 ligands and a wide variety of chem. perturbations, many of which involve significant changes in ligand chem. structures.  In addn., we have applied the method in prospective drug discovery projects and found a significant improvement in the quality of the compds. synthesized that have been predicted to be potent.  Compds. predicted to be potent by this approach have a substantial redn. in false positives relative to compds. synthesized on the basis of other computational or medicinal chem. approaches.  Furthermore, the results are consistent with those obtained from our retrospective studies, demonstrating the robustness and broad range of applicability of this approach, which can be used to drive decisions in lead optimization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGor9UtXEyLPpLVg90H21EOLACvtfcHk0liIXdGJnQwGxg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsF2iuro%253D&md5=37a4f4a6c085f47ed531342643b6c33b</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1021%2Fja512751q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja512751q%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DL.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DDeng%26aufirst%3DY.%26aulast%3DKim%26aufirst%3DB.%26aulast%3DPierce%26aufirst%3DL.%26aulast%3DKrilov%26aufirst%3DG.%26aulast%3DLupyan%26aufirst%3DD.%26aulast%3DRobinson%26aufirst%3DS.%26aulast%3DDahlgren%26aufirst%3DM.%2BK.%26aulast%3DGreenwood%26aufirst%3DJ.%26aulast%3DRomero%26aufirst%3DD.%2BL.%26aulast%3DMasse%26aufirst%3DC.%26aulast%3DKnight%26aufirst%3DJ.%2BL.%26aulast%3DSteinbrecher%26aufirst%3DT.%26aulast%3DBeuming%26aufirst%3DT.%26aulast%3DDamm%26aufirst%3DW.%26aulast%3DHarder%26aufirst%3DE.%26aulast%3DSherman%26aufirst%3DW.%26aulast%3DBrewer%26aufirst%3DM.%26aulast%3DWester%26aufirst%3DR.%26aulast%3DMurcko%26aufirst%3DM.%26aulast%3DFrye%26aufirst%3DL.%26aulast%3DFarid%26aufirst%3DR.%26aulast%3DLin%26aufirst%3DT.%26aulast%3DMobley%26aufirst%3DD.%2BL.%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26aulast%3DBerne%26aufirst%3DB.%2BJ.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DAbel%26aufirst%3DR.%26atitle%3DAccurate%2520and%2520reliable%2520prediction%2520of%2520relative%2520ligand%2520binding%2520potency%2520in%2520prospective%2520drug%2520discovery%2520by%2520way%2520of%2520a%2520modern%2520free-energy%2520calculation%2520protocol%2520and%2520force%2520field%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2015%26volume%3D137%26spage%3D2695%26epage%3D2703%26doi%3D10.1021%2Fja512751q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref68"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref68'); return false;" data-citation="" class="refNumLink">68</a></strong><div class="NLM_citation" id="cit68"><span><span class="NLM_contrib-group">Rocklin, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mobley, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dill, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hunenberger, P. H.</span><span> </span><span class="NLM_article-title">Calculating the binding free energies of charged species based on explicit-solvent simulations employing lattice-sum methods: An accurate correction scheme for electrostatic finite-size effects</span> <span class="citation_source-journal">J. Chem. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">139</span><span class="NLM_x">, </span> <span class="NLM_fpage">184103</span><span class="refDoi"> DOI: 10.1063/1.4826261</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1063%2F1.4826261" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=24320250" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslGjsLbP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=139&publication_year=2013&pages=184103&author=G.+J.+Rocklinauthor=D.+L.+Mobleyauthor=K.+A.+Dillauthor=P.+H.+Hunenberger&title=Calculating+the+binding+free+energies+of+charged+species+based+on+explicit-solvent+simulations+employing+lattice-sum+methods%3A+An+accurate+correction+scheme+for+electrostatic+finite-size+effects&doi=10.1063%2F1.4826261"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">Calculating the binding free energies of charged species based on explicit-solvent simulations employing lattice-sum methods: An accurate correction scheme for electrostatic finite-size effects</span></div><div class="casAuthors">Rocklin, Gabriel J.; Mobley, David L.; Dill, Ken A.; Huenenberger, Philippe H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">139</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">184103/1-184103/32</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    
            (<span class="NLM_cas:orgname">American Institute of Physics</span>)
        </div><div class="casAbstract">The calcn. of a protein-ligand binding free energy based on mol. dynamics (MD) simulations generally relies on a thermodn. cycle in which the ligand is alchem. inserted into the system, both in the solvated protein and free in soln.  The corresponding ligand-insertion free energies are typically calcd. in nanoscale computational boxes simulated under periodic boundary conditions and considering electrostatic interactions defined by a periodic lattice-sum.  This is distinct from the ideal bulk situation of a system of macroscopic size simulated under non-periodic boundary conditions with Coulombic electrostatic interactions.  This discrepancy results in finite-size effects, which affect primarily the charging component of the insertion free energy, are dependent on the box size, and can be large when the ligand bears a net charge, esp. if the protein is charged as well.  This article studies finite-size effects on calcd. charging free energies using as a test case: the binding of the ligand 2-amino-5-methylthiazole (net charge +1 e) to a mutant form of yeast cytochrome c peroxidase in water.  Considering different charge isoforms of the protein (net charges -5, 0, +3, or +9 e), either in the absence or the presence of neutralizing counterions, and sizes of the cubic computational box (edges ranging from 7.42 to 11.02 nm), the potentially large magnitude of finite-size effects on the raw charging free energies (up to 17.1 kJ mol-1) is demonstrated.  Two correction schemes are then proposed to eliminate these effects, a numerical and an anal. one.  Both schemes are based on a continuum-electrostatics anal. and require performing Poisson-Boltzmann (PB) calcns. on the protein-ligand system.  While the numerical scheme requires PB calcns. under both non-periodic and periodic boundary conditions, the latter at the box size considered in the MD simulations, the anal. scheme only requires three non-periodic PB calcns. for a given system, its dependence on the box size being anal.  The latter scheme also provides insight into the phys. origin of the finite-size effects.  These two schemes also encompass a correction for discrete solvent effects that persists even in the limit of infinite box sizes.  Application of either scheme essentially eliminates the size dependence of the cor. charging free energies (maximal deviation of 1.5 kJ mol-1).  Because it is simple to apply, the anal. correction scheme offers a general soln. to the problem of finite-size effects in free-energy calcns. involving charged solutes, as encountered in calcns. concerning, e.g., protein-ligand binding, biomol. assocn., residue mutation, pKa and redox potential estn., substrate transformation, solvation, and solvent-solvent partitioning.  (c) 2013 American Institute of Physics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoIZLSmKR_DYrVg90H21EOLACvtfcHk0liIXdGJnQwGxg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslGjsLbP&md5=4114ffb1d127091ee7421d146911b381</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1063%2F1.4826261&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.4826261%26sid%3Dliteratum%253Aachs%26aulast%3DRocklin%26aufirst%3DG.%2BJ.%26aulast%3DMobley%26aufirst%3DD.%2BL.%26aulast%3DDill%26aufirst%3DK.%2BA.%26aulast%3DHunenberger%26aufirst%3DP.%2BH.%26atitle%3DCalculating%2520the%2520binding%2520free%2520energies%2520of%2520charged%2520species%2520based%2520on%2520explicit-solvent%2520simulations%2520employing%2520lattice-sum%2520methods%253A%2520An%2520accurate%2520correction%2520scheme%2520for%2520electrostatic%2520finite-size%2520effects%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D2013%26volume%3D139%26spage%3D184103%26doi%3D10.1063%2F1.4826261" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref69"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref69'); return false;" data-citation="" class="refNumLink">69</a></strong><div class="NLM_citation" id="cit69"><span><span class="NLM_contrib-group">Reif, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oostenbrink, C.</span><span> </span><span class="NLM_article-title">Net charge changes in the calculation of relative ligand-binding free energies via classical atomistic molecular dynamics simulation</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">35</span><span class="NLM_x">, </span> <span class="NLM_fpage">227</span><span class="NLM_x">–</span> <span class="NLM_lpage">243</span><span class="refDoi"> DOI: 10.1002/jcc.23490</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1002%2Fjcc.23490" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=24249099" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvVSgsLjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2014&pages=227-243&author=M.+M.+Reifauthor=C.+Oostenbrink&title=Net+charge+changes+in+the+calculation+of+relative+ligand-binding+free+energies+via+classical+atomistic+molecular+dynamics+simulation&doi=10.1002%2Fjcc.23490"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">Net charge changes in the calculation of relative ligand-binding free energies via classical atomistic molecular dynamics simulation</span></div><div class="casAuthors">Reif, Maria M.; Oostenbrink, Chris</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">227-243</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">The calcn. of binding free energies of charged species to a target mol. is a frequently encountered problem in mol. dynamics studies of (bio-)chem. thermodn.  Many important endogenous receptor-binding mols., enzyme substrates, or drug mols. have a nonzero net charge.  Abs. binding free energies, as well as binding free energies relative to another mol. with a different net charge will be affected by artifacts due to the used effective electrostatic interaction function and assocd. parameters (e.g., size of the computational box).  In the present study, charging contributions to binding free energies of small oligoat. ions to a series of model host cavities functionalized with different chem. groups are calcd. with classical atomistic mol. dynamics simulation.  Electrostatic interactions are treated using a lattice-summation scheme or a cutoff-truncation scheme with Barker-Watts reaction-field correction, and the simulations are conducted in boxes of different edge lengths.  It is illustrated that the charging free energies of the guest mols. in water and in the host strongly depend on the applied methodol. and that neglect of correction terms for the artifacts introduced by the finite size of the simulated system and the use of an effective electrostatic interaction function considerably impairs the thermodn. interpretation of guest-host interactions.  Application of correction terms for the various artifacts yields consistent results for the charging contribution to binding free energies and is thus a prerequisite for the valid interpretation or prediction of exptl. data via mol. dynamics simulation.  Anal. and correction of electrostatic artifacts according to the scheme proposed in the present study should therefore be considered an integral part of careful free-energy calcn. studies if changes in the net charge are involved.  © 2013 Wiley Periodicals, Inc.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1lirsSd7shLVg90H21EOLACvtfcHk0lg_vam3LtM82w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvVSgsLjO&md5=fc7ffb060ff30bcda792ffc68f832906</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1002%2Fjcc.23490&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.23490%26sid%3Dliteratum%253Aachs%26aulast%3DReif%26aufirst%3DM.%2BM.%26aulast%3DOostenbrink%26aufirst%3DC.%26atitle%3DNet%2520charge%2520changes%2520in%2520the%2520calculation%2520of%2520relative%2520ligand-binding%2520free%2520energies%2520via%2520classical%2520atomistic%2520molecular%2520dynamics%2520simulation%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2014%26volume%3D35%26spage%3D227%26epage%3D243%26doi%3D10.1002%2Fjcc.23490" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref70"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref70'); return false;" data-citation="" class="refNumLink">70</a></strong><div class="NLM_citation" id="cit70"><span><span class="NLM_contrib-group">Lazaridis, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Masunov, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gandolfo, F.</span><span> </span><span class="NLM_article-title">Contributions to the binding free energy of ligands to avidin and streptavidin</span> <span class="citation_source-journal">Proteins: Struct., Funct., Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">194</span><span class="NLM_x">–</span> <span class="NLM_lpage">208</span><span class="refDoi"> DOI: 10.1002/prot.10086</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1002%2Fprot.10086" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=11933066" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BD38XislGhurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2002&pages=194-208&author=T.+Lazaridisauthor=A.+Masunovauthor=F.+Gandolfo&title=Contributions+to+the+binding+free+energy+of+ligands+to+avidin+and+streptavidin&doi=10.1002%2Fprot.10086"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">Contributions to the binding free energy of ligands to avidin and streptavidin</span></div><div class="casAuthors">Lazaridis, Themis; Masunov, Artem; Gandolfo, Francois</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Genetics</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">194-208</span>CODEN:
                <span class="NLM_cas:coden">PSFGEY</span>;
        ISSN:<span class="NLM_cas:issn">0887-3585</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">The free energy of binding of a ligand to a macromol. is here formally decompd. into the (effective) energy of interaction, reorganization energy of the ligand and the macro-mol., conformational entropy change of the ligand and the macromol., and translational and rotational entropy loss of the ligand.  Mol. dynamics simulations with implicit solvation are used to evaluate these contributions in the binding of biotin, biotin analogs, and two peptides to avidin and streptavidin.  We find that the largest contribution opposing binding is the protein reorganization energy, which is calcd. to be from 10 to 30 kcal/mol for the ligands considered here.  The ligand reorganization energy is also significant for flexible ligands.  The translational/rotational entropy is 4.5-6 kcal/mol at 1 M std. state and room temp.  The calcd. binding free energies are in the correct range, but the large statistical uncertainty in the protein reorganization energy precludes precise predictions.  For some complexes, the simulations show multiple binding modes, different from the one obsd. in the crystal structure.  This finding is probably due to deficiencies in the force field but may also reflect considerable ligand flexibility.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrL-1EkQp_DCbVg90H21EOLACvtfcHk0lg_vam3LtM82w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XislGhurk%253D&md5=df50261e0738b8d62173157ad321fe3c</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1002%2Fprot.10086&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.10086%26sid%3Dliteratum%253Aachs%26aulast%3DLazaridis%26aufirst%3DT.%26aulast%3DMasunov%26aufirst%3DA.%26aulast%3DGandolfo%26aufirst%3DF.%26atitle%3DContributions%2520to%2520the%2520binding%2520free%2520energy%2520of%2520ligands%2520to%2520avidin%2520and%2520streptavidin%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D2002%26volume%3D47%26spage%3D194%26epage%3D208%26doi%3D10.1002%2Fprot.10086" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref71"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref71'); return false;" data-citation="" class="refNumLink">71</a></strong><div class="NLM_citation" id="cit71"><span><span class="NLM_contrib-group">Oostenbrink, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Gunsteren, W. F.</span><span> </span><span class="NLM_article-title">Free energies of binding of polychlorinated biphenyls to the estrogen receptor from a single simulation</span> <span class="citation_source-journal">Proteins: Struct., Funct., Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">237</span><span class="NLM_x">–</span> <span class="NLM_lpage">246</span><span class="refDoi"> DOI: 10.1002/prot.10558</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1002%2Fprot.10558" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=14696186" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BD2cXlsFWjsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2004&pages=237-246&author=C.+Oostenbrinkauthor=W.+F.+van+Gunsteren&title=Free+energies+of+binding+of+polychlorinated+biphenyls+to+the+estrogen+receptor+from+a+single+simulation&doi=10.1002%2Fprot.10558"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">Free energies of binding of polychlorinated biphenyls to the estrogen receptor from a single simulation</span></div><div class="casAuthors">Oostenbrink, Chris; van Gunsteren, Wilfred F.</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Bioinformatics</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">237-246</span>CODEN:
                <span class="NLM_cas:coden">PSFBAF</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">Relative free energies of binding to the ligand-binding domain of the estrogen receptor have been calcd. for a series of 17 hydroxylated polychlorinated biphenyls.  Because traditional thermodn. integration or perturbation approaches are hardly feasible for these nos. of compds., the one-step perturbation approach is applied and is shown to yield accurate results based on only two 2-ns mol. dynamics simulations of an unphys., judiciously chosen, ref. state.  The mean abs. difference between the calcd. and exptl. binding free energies for the 17 compds. is 3.4 kJ/mol, which illustrates the accuracy of the GROMOS biomol. force field used.  Excluding the three largest ligands from the comparison reduces the deviation to 2.0 kJ/mol (i.e., < kBT).  Apart from the relative free energy, structural information about the binding mode and binding orientation for every compd. can also be extd. from the simulation, showing that a ligand bound to its receptor cannot be represented by a single conformation, but it samples an ensemble of different orientations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqdRCeKfjW367Vg90H21EOLACvtfcHk0lg_vam3LtM82w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXlsFWjsg%253D%253D&md5=6dd0c013b944251c11bbd90d567c8f8e</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1002%2Fprot.10558&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.10558%26sid%3Dliteratum%253Aachs%26aulast%3DOostenbrink%26aufirst%3DC.%26aulast%3Dvan%2BGunsteren%26aufirst%3DW.%2BF.%26atitle%3DFree%2520energies%2520of%2520binding%2520of%2520polychlorinated%2520biphenyls%2520to%2520the%2520estrogen%2520receptor%2520from%2520a%2520single%2520simulation%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D2004%26volume%3D54%26spage%3D237%26epage%3D246%26doi%3D10.1002%2Fprot.10558" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref72"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref72'); return false;" data-citation="" class="refNumLink">72</a></strong><div class="NLM_citation" id="cit72"><span><span class="NLM_contrib-group">Steinbrecher, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Case, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Labahn, A.</span><span> </span><span class="NLM_article-title">A multistep approach to structure-based drug design: studying ligand binding at the human neutrophil elastase</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">1837</span><span class="NLM_x">–</span> <span class="NLM_lpage">1844</span><span class="refDoi"> DOI: 10.1021/jm0505720</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0505720" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BD28XhsVSqtbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=1837-1844&author=T.+Steinbrecherauthor=D.+A.+Caseauthor=A.+Labahn&title=A+multistep+approach+to+structure-based+drug+design%3A+studying+ligand+binding+at+the+human+neutrophil+elastase&doi=10.1021%2Fjm0505720"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">A Multistep Approach to Structure-Based Drug Design: Studying Ligand Binding at the Human Neutrophil Elastase</span></div><div class="casAuthors">Steinbrecher, Thomas; Case, David A.; Labahn, Andreas</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1837-1844</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In this study we show that a combination of different theor. methods is a viable approach to calc. the binding affinities of new ligands for the human neutrophile elastase.  This protease degrades elastin and likely aids neutrophils in fulfilling their immunol. functions.  Abnormally high human neutrophil elastase (HNE) levels are involved in several diseases; therefore, inhibitors of HNE are of interest as targets for drug design.  A recent study has revealed that cinnamic acid and bornyl ester derivs. bind to HNE, but ΔG0 values from ligand docking results exhibited no correlation with those calcd. from the IC50 values.  To accurately compute binding affinities, we generated possible protein ligand complex structures by ligand docking calcns.  For each of the ligands, the 30 most likely placements were used as starting points of nanosecond length mol. dynamics simulations.  The binding free energies for these complex structures were estd. using a continuum solvent (MM-PBSA) approach.  These results, along with structural data from the mol. dynamics runs, allowed the identification of a group of similar placements that serve as a model for the natural protein ligand complex structure.  This structural model was used to perform thermodn. integration (TI) calcns. to obtain the relative binding free energies of similar ligands to HNE.  The TI results were in quant. agreement with the measured binding affinities.  Thus, the presented approach can be used to generate a probable complex structure for known ligands to HNE and to use such a structure to calc. the effects of small ligand modifications on ligand binding, possibly leading to new inhibitors with improved binding affinities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpReq57qu60nLVg90H21EOLACvtfcHk0li4crnD-9OwnA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhsVSqtbc%253D&md5=af09a49b14c1f84df850f5e68cfac3bc</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1021%2Fjm0505720&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0505720%26sid%3Dliteratum%253Aachs%26aulast%3DSteinbrecher%26aufirst%3DT.%26aulast%3DCase%26aufirst%3DD.%2BA.%26aulast%3DLabahn%26aufirst%3DA.%26atitle%3DA%2520multistep%2520approach%2520to%2520structure-based%2520drug%2520design%253A%2520studying%2520ligand%2520binding%2520at%2520the%2520human%2520neutrophil%2520elastase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D1837%26epage%3D1844%26doi%3D10.1021%2Fjm0505720" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref73"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref73'); return false;" data-citation="" class="refNumLink">73</a></strong><div class="NLM_citation" id="cit73"><span><span class="NLM_contrib-group">Mobley, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chodera, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dill, K. A.</span><span> </span><span class="NLM_article-title">The confine-and-release method: obtaining correct binding free energies in the presence of protein conformational change</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">1231</span><span class="NLM_x">–</span> <span class="NLM_lpage">1235</span><span class="refDoi"> DOI: 10.1021/ct700032n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct700032n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BD2sXlsFenu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2007&pages=1231-1235&author=D.+L.+Mobleyauthor=J.+D.+Choderaauthor=K.+A.+Dill&title=The+confine-and-release+method%3A+obtaining+correct+binding+free+energies+in+the+presence+of+protein+conformational+change&doi=10.1021%2Fct700032n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">Confine-and-Release Method: Obtaining Correct Binding Free Energies in the Presence of Protein Conformational Change</span></div><div class="casAuthors">Mobley, David L.; Chodera, John D.; Dill, Ken A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1231-1235</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Free energy calcns. are increasingly being used to est. abs. and relative binding free energies of ligands to proteins.  However, computed free energies often appear to depend on the initial protein conformation, indicating incomplete sampling.  This is esp. true when proteins can change conformation on ligand binding, as free energies assocd. with these conformational changes are either ignored or assumed to be included by virtue of the sampling performed in the calcn.  Here, the authors show that, in a model protein system (a designed binding site in T4 lysozyme), conformational changes can make a difference of several kcal/mol in computed binding free energies and that they are neglected in computed binding free energies if the system remains kinetically trapped in a particular metastable state on simulation timescales.  The authors introduce a general "confine-and-release" framework for free energy calcns. that accounts for these free energies of conformational change.  The authors illustrate its use in this model system by demonstrating that an umbrella sampling protocol can obtain converged binding free energies that are independent of the starting protein structure and include these conformational change free energies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHdRFLUwdd7LVg90H21EOLACvtfcHk0li4crnD-9OwnA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXlsFenu7g%253D&md5=fa953a85593d81ee192ff6c5d7f525c2</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1021%2Fct700032n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct700032n%26sid%3Dliteratum%253Aachs%26aulast%3DMobley%26aufirst%3DD.%2BL.%26aulast%3DChodera%26aufirst%3DJ.%2BD.%26aulast%3DDill%26aufirst%3DK.%2BA.%26atitle%3DThe%2520confine-and-release%2520method%253A%2520obtaining%2520correct%2520binding%2520free%2520energies%2520in%2520the%2520presence%2520of%2520protein%2520conformational%2520change%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2007%26volume%3D3%26spage%3D1231%26epage%3D1235%26doi%3D10.1021%2Fct700032n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref74"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref74'); return false;" data-citation="" class="refNumLink">74</a></strong><div class="NLM_citation" id="cit74"><span><span class="NLM_contrib-group">Montfort, W. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perry, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fauman, E. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Finer-Moore, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maley, G. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hardy, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maley, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stroud, R. M.</span><span> </span><span class="NLM_article-title">Structure, multiple site binding, and segmental accommodation in thymidylate synthase on binding dUMP and an anti-folate</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1990</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x">, </span> <span class="NLM_fpage">6964</span><span class="NLM_x">–</span> <span class="NLM_lpage">6977</span><span class="refDoi"> DOI: 10.1021/bi00482a004</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi00482a004" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADyaK3cXksFCqtbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=1990&pages=6964-6977&author=W.+R.+Montfortauthor=K.+M.+Perryauthor=E.+B.+Faumanauthor=J.+S.+Finer-Mooreauthor=G.+F.+Maleyauthor=L.+Hardyauthor=F.+Maleyauthor=R.+M.+Stroud&title=Structure%2C+multiple+site+binding%2C+and+segmental+accommodation+in+thymidylate+synthase+on+binding+dUMP+and+an+anti-folate&doi=10.1021%2Fbi00482a004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit74R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">Structure, multiple site binding, and segmental accommodation in thymidylate synthase on binding dUMP and an anti-folate</span></div><div class="casAuthors">Montfort, William R.; Perry, Kathy M.; Fauman, Eric B.; Finer-Moore, Janet S.; Maley, Gladys F.; Hardy, Larry; Maley, Frank; Stroud, Robert M.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">1990</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">30</span>),
    <span class="NLM_cas:pages">6964-77</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    </div><div class="casAbstract">The structure of Escherichia coli thymidylate synthase (TS) complexed with the substrate, dUMP, and CB 3717, an analog of the cofactor, methylenetetrahydrofolate, was solved by multiple isomorphous replacement and refined at 1.97-Å resoln. to a residual of 18% for all data (16% for data >2σ) for a highly constrained structure.  All residues in the structure were clearly resolved and gave a very high confidence in total correctness of the structure.  The ternary complex directly suggested how methylation of dUMP takes place.  C-6 of dUMP was covalently bound to γS of cysteine-198(146) during catalysis, and the reactants were surrounded by specific H-bonds and hydrophobic interactions from conserved residues.  Comparison with the independently solved structure of unliganded TS revealed a large conformation change in the enzyme, which closes down to sequester the reactants and several highly ordered water mols. within a cavernous active center, away from bulk solvent.  A 2nd binding site for the quinazoline ring of the cofactor analog was discovered by withholding addn. of reducing agent during crystal storage.  The chem. change in the protein was slight, and from difference d. maps modification of SH groups was not directly responsible for blockade of the primary site.  The site, only partially overlapping with the primary site, was also surrounded by conserved residues and thus may play a functional role.  The ligand-induced conformational change was not a domain shift but involved the segmental accommodation of several helixes, β-strands, and loops that moved as units against the β-sheet interface between monomers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoaidrVNYJuorVg90H21EOLACvtfcHk0li4crnD-9OwnA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3cXksFCqtbY%253D&md5=a4a99e1360917b52f32615aa29b862b8</span></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1021%2Fbi00482a004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi00482a004%26sid%3Dliteratum%253Aachs%26aulast%3DMontfort%26aufirst%3DW.%2BR.%26aulast%3DPerry%26aufirst%3DK.%2BM.%26aulast%3DFauman%26aufirst%3DE.%2BB.%26aulast%3DFiner-Moore%26aufirst%3DJ.%2BS.%26aulast%3DMaley%26aufirst%3DG.%2BF.%26aulast%3DHardy%26aufirst%3DL.%26aulast%3DMaley%26aufirst%3DF.%26aulast%3DStroud%26aufirst%3DR.%2BM.%26atitle%3DStructure%252C%2520multiple%2520site%2520binding%252C%2520and%2520segmental%2520accommodation%2520in%2520thymidylate%2520synthase%2520on%2520binding%2520dUMP%2520and%2520an%2520anti-folate%26jtitle%3DBiochemistry%26date%3D1990%26volume%3D29%26spage%3D6964%26epage%3D6977%26doi%3D10.1021%2Fbi00482a004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref75"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref75'); return false;" data-citation="" class="refNumLink">75</a></strong><div class="NLM_citation" id="cit75"><span><span class="NLM_contrib-group">Mobley, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dill, K. A.</span><span> </span><span class="NLM_article-title">Binding of small-molecule ligands to proteins: “what you see” is not always “what you get”</span> <span class="citation_source-journal">Structure</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">489</span><span class="NLM_x">–</span> <span class="NLM_lpage">498</span><span class="refDoi"> DOI: 10.1016/j.str.2009.02.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1016%2Fj.str.2009.02.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=19368882" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BD1MXksVCmsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2009&pages=489-498&author=D.+L.+Mobleyauthor=K.+A.+Dill&title=Binding+of+small-molecule+ligands+to+proteins%3A+%E2%80%9Cwhat+you+see%E2%80%9D+is+not+always+%E2%80%9Cwhat+you+get%E2%80%9D&doi=10.1016%2Fj.str.2009.02.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit75R"><div class="casContent"><span class="casTitleNuber">75</span><div class="casTitle"><span class="NLM_cas:atitle">Binding of Small-Molecule Ligands to Proteins: "What You See" Is Not Always "What You Get"</span></div><div class="casAuthors">Mobley, David L.; Dill, Ken A.</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">489-498</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  We review insights from computational studies of affinities of ligands binding to proteins.  The power of structural biol. is in translating knowledge of protein structures into insights about their forces, binding, and mechanisms.  However, the complementary power of computer modeling is in showing "the rest of the story" (i.e., how motions and ensembles and alternative conformers and the entropies and forces that cannot be seen in single mol. structures also contribute to binding affinities).  Upon binding to a protein, a ligand can bind in multiple orientations; the protein or ligand can be deformed by the binding event; waters, ions, or cofactors can have unexpected involvement; and conformational or solvation entropies can sometimes play large and otherwise unpredictable roles.  Computer modeling is helping to elucidate these factors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnlrwNwfLm4rVg90H21EOLACvtfcHk0lic23n-sIQjtQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXksVCmsrk%253D&md5=993623b1444e33a45b2fe8b904dec9c1</span></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2009.02.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2009.02.010%26sid%3Dliteratum%253Aachs%26aulast%3DMobley%26aufirst%3DD.%2BL.%26aulast%3DDill%26aufirst%3DK.%2BA.%26atitle%3DBinding%2520of%2520small-molecule%2520ligands%2520to%2520proteins%253A%2520%25E2%2580%259Cwhat%2520you%2520see%25E2%2580%259D%2520is%2520not%2520always%2520%25E2%2580%259Cwhat%2520you%2520get%25E2%2580%259D%26jtitle%3DStructure%26date%3D2009%26volume%3D17%26spage%3D489%26epage%3D498%26doi%3D10.1016%2Fj.str.2009.02.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref76"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref76'); return false;" data-citation="" class="refNumLink">76</a></strong><div class="NLM_citation" id="cit76"><span><span class="NLM_contrib-group">Wang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deng, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knight, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sherman, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shelley, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abel, R.</span><span> </span><span class="NLM_article-title">Modeling local structural rearrangements using FEP/REST: application to relative binding affinity predictions of CDK2 inhibitors</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">1282</span><span class="NLM_x">–</span> <span class="NLM_lpage">1293</span><span class="refDoi"> DOI: 10.1021/ct300911a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct300911a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC3sXoslOjuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=1282-1293&author=L.+Wangauthor=Y.+Dengauthor=J.+L.+Knightauthor=Y.+Wuauthor=B.+Kimauthor=W.+Shermanauthor=J.+C.+Shelleyauthor=T.+Linauthor=R.+Abel&title=Modeling+local+structural+rearrangements+using+FEP%2FREST%3A+application+to+relative+binding+affinity+predictions+of+CDK2+inhibitors&doi=10.1021%2Fct300911a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit76R"><div class="casContent"><span class="casTitleNuber">76</span><div class="casTitle"><span class="NLM_cas:atitle">Modeling Local Structural Rearrangements Using FEP/REST: Application to Relative Binding Affinity Predictions of CDK2 Inhibitors</span></div><div class="casAuthors">Wang, Lingle; Deng, Yuqing; Knight, Jennifer L.; Wu, Yujie; Kim, Byungchan; Sherman, Woody; Shelley, John C.; Lin, Teng; Abel, Robert</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">1282-1293</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Accurate and reliable calcn. of protein-ligand binding affinities remains a hotbed of computer-aided drug design research.  Despite the potentially large impact FEP (free energy perturbation) may have in drug design projects, practical applications of FEP in industrial contexts have been limited.  In this work, we use a recently developed method, FEP/REST (free energy perturbation/replica exchange with solute tempering), to calc. the relative binding affinities for a set of congeneric ligands binding to the CDK2 receptor.  We compare the FEP/REST results with traditional FEP/MD (mol. dynamics) results and MM/GBSA (mol. mechanics/Generalized Born Surface Area model) results and examine why FEP/REST performed notably better than these other methods, as well as why certain ligand mutations lead to large increases of the binding affinity while others do not.  We also introduce a math. framework for assessing the consistency and reliability of the calcns. using cycle closures in FEP mutation paths.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrorAEUJEbEYbVg90H21EOLACvtfcHk0lic23n-sIQjtQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXoslOjuw%253D%253D&md5=cec7d69a0010904a55fcda5a277e8a1a</span></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.1021%2Fct300911a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct300911a%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DL.%26aulast%3DDeng%26aufirst%3DY.%26aulast%3DKnight%26aufirst%3DJ.%2BL.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DKim%26aufirst%3DB.%26aulast%3DSherman%26aufirst%3DW.%26aulast%3DShelley%26aufirst%3DJ.%2BC.%26aulast%3DLin%26aufirst%3DT.%26aulast%3DAbel%26aufirst%3DR.%26atitle%3DModeling%2520local%2520structural%2520rearrangements%2520using%2520FEP%252FREST%253A%2520application%2520to%2520relative%2520binding%2520affinity%2520predictions%2520of%2520CDK2%2520inhibitors%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2013%26volume%3D9%26spage%3D1282%26epage%3D1293%26doi%3D10.1021%2Fct300911a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref77"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref77'); return false;" data-citation="" class="refNumLink">77</a></strong><div class="NLM_citation" id="cit77"><span><span class="NLM_contrib-group">Dziedzic, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cisneros, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robertson, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hare, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Danford, N. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baxter, R. H. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jorgensen, W. L.</span><span> </span><span class="NLM_article-title">Design, synthesis, and protein crystallography of biaryltriazoles as potent tautomerase inhibitors of macrophage migration inhibitory factor</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">137</span><span class="NLM_x">, </span> <span class="NLM_fpage">2996</span><span class="NLM_x">–</span> <span class="NLM_lpage">3003</span><span class="refDoi"> DOI: 10.1021/ja512112j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja512112j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjtV2ntLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=137&publication_year=2015&pages=2996-3003&author=P.+Dziedzicauthor=J.+A.+Cisnerosauthor=M.+J.+Robertsonauthor=A.+A.+Hareauthor=N.+E.+Danfordauthor=R.+H.+G.+Baxterauthor=W.+L.+Jorgensen&title=Design%2C+synthesis%2C+and+protein+crystallography+of+biaryltriazoles+as+potent+tautomerase+inhibitors+of+macrophage+migration+inhibitory+factor&doi=10.1021%2Fja512112j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit77R"><div class="casContent"><span class="casTitleNuber">77</span><div class="casTitle"><span class="NLM_cas:atitle">Design, synthesis, and protein crystallography of biaryltriazoles as potent tautomerase inhibitors of macrophage migration inhibitory factor</span></div><div class="casAuthors">Dziedzic, Pawel; Cisneros, Jose A.; Robertson, Michael J.; Hare, Alissa A.; Danford, Nadia E.; Baxter, Richard H. G.; Jorgensen, William L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">137</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2996-3003</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Optimization is reported for biaryltriazoles as inhibitors of the tautomerase activity of human macrophage migration inhibitory factor (MIF), a proinflammatory cytokine assocd. with numerous inflammatory diseases and cancer.  A combined approach was taken featuring org. synthesis, enzymic assaying, crystallog., and modeling including free-energy perturbation (FEP) calcns.  X-ray crystal structures for two inhibitors bound to MIF are reported and provided a basis for the modeling efforts.  The accommodation of the inhibitors in the binding site is striking with multiple hydrogen bonds and aryl-aryl interactions.  Addnl. modeling encouraged pursuit of 5-phenoxyquinolinyl analogs, which led to the very potent compd.  Activity was further enhanced by addn. of a fluorine atom adjacent to the phenolic hydroxyl group to strengthen a key hydrogen bond.  It is also shown that phys. properties of the compds. can be modulated by variation of solvent-exposed substituents.  Several of the compds. are likely the most potent known MIF tautomerase inhibitors; the most active ones are more than 1000-fold more active than the well-studied (R)-ISO-1 and more than 200-fold more active than the chromen-4-one Orita-13.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqYFcj_0uhyz7Vg90H21EOLACvtfcHk0lgfMaxY5SosbQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjtV2ntLo%253D&md5=2b5fea70b6db3cb39f7217c39995bd06</span></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=10.1021%2Fja512112j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja512112j%26sid%3Dliteratum%253Aachs%26aulast%3DDziedzic%26aufirst%3DP.%26aulast%3DCisneros%26aufirst%3DJ.%2BA.%26aulast%3DRobertson%26aufirst%3DM.%2BJ.%26aulast%3DHare%26aufirst%3DA.%2BA.%26aulast%3DDanford%26aufirst%3DN.%2BE.%26aulast%3DBaxter%26aufirst%3DR.%2BH.%2BG.%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26atitle%3DDesign%252C%2520synthesis%252C%2520and%2520protein%2520crystallography%2520of%2520biaryltriazoles%2520as%2520potent%2520tautomerase%2520inhibitors%2520of%2520macrophage%2520migration%2520inhibitory%2520factor%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2015%26volume%3D137%26spage%3D2996%26epage%3D3003%26doi%3D10.1021%2Fja512112j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref78"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref78'); return false;" data-citation="" class="refNumLink">78</a></strong><div class="NLM_citation" id="cit78"><span><span class="NLM_contrib-group">Mutyala, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reddy, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sumakanth, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reddanna, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reddy, M. R.</span><span> </span><span class="NLM_article-title">Calculation of relative binding affinities of fructose 1, 6-bisphosphatase mutants with adenosine monophosphate using free energy perturbation method</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">932</span><span class="NLM_x">–</span> <span class="NLM_lpage">937</span><span class="refDoi"> DOI: 10.1002/jcc.20617</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1002%2Fjcc.20617" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2007&pages=932-937&author=R.+Mutyalaauthor=R.+Reddyauthor=M.+Sumakanthauthor=P.+Reddannaauthor=M.+R.+Reddy&title=Calculation+of+relative+binding+affinities+of+fructose+1%2C+6-bisphosphatase+mutants+with+adenosine+monophosphate+using+free+energy+perturbation+method&doi=10.1002%2Fjcc.20617"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit78&amp;dbid=16384&amp;doi=10.1002%2Fjcc.20617&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.20617%26sid%3Dliteratum%253Aachs%26aulast%3DMutyala%26aufirst%3DR.%26aulast%3DReddy%26aufirst%3DR.%26aulast%3DSumakanth%26aufirst%3DM.%26aulast%3DReddanna%26aufirst%3DP.%26aulast%3DReddy%26aufirst%3DM.%2BR.%26atitle%3DCalculation%2520of%2520relative%2520binding%2520affinities%2520of%2520fructose%25201%252C%25206-bisphosphatase%2520mutants%2520with%2520adenosine%2520monophosphate%2520using%2520free%2520energy%2520perturbation%2520method%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2007%26volume%3D28%26spage%3D932%26epage%3D937%26doi%3D10.1002%2Fjcc.20617" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref79"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref79'); return false;" data-citation="" class="refNumLink">79</a></strong><div class="NLM_citation" id="cit79"><span><span class="NLM_contrib-group">Vergara-Jaque, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poblete, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, E. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schulten, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gonzalez-Nilo, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chipot, C.</span><span> </span><span class="NLM_article-title">Molecular basis of drug resistance in A/H1N1 virus</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">2650</span><span class="NLM_x">–</span> <span class="NLM_lpage">2656</span><span class="refDoi"> DOI: 10.1021/ci300343w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci300343w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtlGjsbjL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=2650-2656&author=A.+Vergara-Jaqueauthor=H.+Pobleteauthor=E.+H.+Leeauthor=K.+Schultenauthor=F.+Gonzalez-Niloauthor=C.+Chipot&title=Molecular+basis+of+drug+resistance+in+A%2FH1N1+virus&doi=10.1021%2Fci300343w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit79R"><div class="casContent"><span class="casTitleNuber">79</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Basis of Drug Resistance in A/H1N1 Virus</span></div><div class="casAuthors">Vergara-Jaque, Ariela; Poblete, Horacio; Lee, Eric H.; Schulten, Klaus; Gonzalez-Nilo, Fernando; Chipot, Christophe</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2650-2656</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">New mutants of human influenza virus (A/H1N1) exhibit resistance to antiviral drugs.  The mechanism whereby they develop insensitivity to these medications is, however, not yet completely understood.  A crystallog. structure of A/H1N1 neuraminidase has been published recently.  Using mol. dynamic simulations, it is now possible to characterize at the at. level the mechanism that underlies the loss of binding affinity of the drugs.  In this study, free-energy perturbation was used to evaluate the relative binding free energies of Tamiflu and Relenza with H274Y, N294S, and Y252H neuraminidase mutants.  Our results demonstrate a remarkable correlation between theor. and exptl. data, which quant. confirms that the mutants are resistant to Tamiflu but are still strongly inhibited by Relenza.  The simulations further reveal the key interactions that govern the affinity of the two drugs for each mutant.  This information is envisioned to prove useful for the design of novel neuraminidase inhibitors and for the characterization of new potential mutants.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGraFM3nu-VENLVg90H21EOLACvtfcHk0lgfMaxY5SosbQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtlGjsbjL&md5=e2f4548450cbe16c8acc0aef73ed1c8c</span></div><a href="/servlet/linkout?suffix=cit79&amp;dbid=16384&amp;doi=10.1021%2Fci300343w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci300343w%26sid%3Dliteratum%253Aachs%26aulast%3DVergara-Jaque%26aufirst%3DA.%26aulast%3DPoblete%26aufirst%3DH.%26aulast%3DLee%26aufirst%3DE.%2BH.%26aulast%3DSchulten%26aufirst%3DK.%26aulast%3DGonzalez-Nilo%26aufirst%3DF.%26aulast%3DChipot%26aufirst%3DC.%26atitle%3DMolecular%2520basis%2520of%2520drug%2520resistance%2520in%2520A%252FH1N1%2520virus%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26spage%3D2650%26epage%3D2656%26doi%3D10.1021%2Fci300343w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref80"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref80'); return false;" data-citation="" class="refNumLink">80</a></strong><div class="NLM_citation" id="cit80"><span><span class="NLM_contrib-group">Boukharta, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gutierrez-de-Teran, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aqvist, J.</span><span> </span><span class="NLM_article-title">Computational prediction of alanine scanning and ligand binding energetics in G-protein coupled receptors</span> <span class="citation_source-journal">PLoS Comput. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">e1003585</span><span class="refDoi"> DOI: 10.1371/journal.pcbi.1003585</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1371%2Fjournal.pcbi.1003585" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=24743773" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVGls7bF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=e1003585&author=L.+Boukhartaauthor=H.+Gutierrez-de-Teranauthor=J.+Aqvist&title=Computational+prediction+of+alanine+scanning+and+ligand+binding+energetics+in+G-protein+coupled+receptors&doi=10.1371%2Fjournal.pcbi.1003585"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit80R"><div class="casContent"><span class="casTitleNuber">80</span><div class="casTitle"><span class="NLM_cas:atitle">Computational prediction of alanine scanning and ligand binding energetics in G-protein coupled receptors</span></div><div class="casAuthors">Boukharta, Lars; Gutieerrez-de-Teraen, Hugo; Aaqvist, Johan</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Computational Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">e1003585/1-e1003585/11, 11 pp.</span>CODEN:
                <span class="NLM_cas:coden">PCBLBG</span>;
        ISSN:<span class="NLM_cas:issn">1553-7358</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Site-directed mutagenesis combined with binding affinity measurements is widely used to probe the nature of ligand interactions with GPCRs.  Such expts., as well as structure-activity relationships for series of ligands, are usually interpreted with computationally derived models of ligand binding modes.  However, systematic approaches for accurate calcns. of the corresponding binding free energies are still lacking.  Here, we report a computational strategy to quant. predict the effects of alanine scanning and ligand modifications based on mol. dynamics free energy simulations.  A smooth stepwise scheme for free energy perturbation calcns. is derived and applied to a series of thirteen alanine mutations of the human neuropeptide Y1 receptor and series of eight analogous antagonists.  The robustness and accuracy of the method enables univocal interpretation of existing mutagenesis and binding data.  We show how these calcns. can be used to validate structural models and demonstrate their ability to discriminate against suboptimal ones.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoqb5_eZx503LVg90H21EOLACvtfcHk0lgpgHAflk6HwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVGls7bF&md5=478097bb0b20727daf628037457a7bd7</span></div><a href="/servlet/linkout?suffix=cit80&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pcbi.1003585&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pcbi.1003585%26sid%3Dliteratum%253Aachs%26aulast%3DBoukharta%26aufirst%3DL.%26aulast%3DGutierrez-de-Teran%26aufirst%3DH.%26aulast%3DAqvist%26aufirst%3DJ.%26atitle%3DComputational%2520prediction%2520of%2520alanine%2520scanning%2520and%2520ligand%2520binding%2520energetics%2520in%2520G-protein%2520coupled%2520receptors%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2014%26volume%3D10%26spage%3De1003585%26doi%3D10.1371%2Fjournal.pcbi.1003585" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref81"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref81'); return false;" data-citation="" class="refNumLink">81</a></strong><div class="NLM_citation" id="cit81"><span><span class="NLM_contrib-group">Maurer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Beer, S. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oostenbrink, C.</span><span> </span><span class="NLM_article-title">Calculation of relative binding free energy in the water-filled active site of oligopeptide-binding protein A</span> <span class="citation_source-journal">Molecules</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">499</span><span class="refDoi"> DOI: 10.3390/molecules21040499</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.3390%2Fmolecules21040499" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=27092480" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVCktLfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2016&pages=499&author=M.+Maurerauthor=S.+B.+de+Beerauthor=C.+Oostenbrink&title=Calculation+of+relative+binding+free+energy+in+the+water-filled+active+site+of+oligopeptide-binding+protein+A&doi=10.3390%2Fmolecules21040499"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit81R"><div class="casContent"><span class="casTitleNuber">81</span><div class="casTitle"><span class="NLM_cas:atitle">Calculation of relative binding free energy in the water-filled active site of oligopeptide-binding protein A</span></div><div class="casAuthors">Maurer, Manuela; de Beer, Stephanie B. A.; Oostenbrink, Chris</div><div class="citationInfo"><span class="NLM_cas:title">Molecules</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">499/1-499/15</span>CODEN:
                <span class="NLM_cas:coden">MOLEFW</span>;
        ISSN:<span class="NLM_cas:issn">1420-3049</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">The periplasmic oligopeptide binding protein A (OppA) represents a well-known example of water-mediated protein-ligand interactions.  Here, we perform free-energy calcns. for three different ligands binding to OppA, using a thermodn. integration approach.  The tripeptide ligands share a high structural similarity (all have the sequence KXK), but their exptl.-detd. binding free energies differ remarkably.  Thermodn. cycles were constructed for the ligands, and simulations conducted in the bound and (freely solvated) unbound states.  In the unbound state, it was obsd. that the difference in conformational freedom between alanine and glycine leads to a surprisingly slow convergence, despite their chem. similarity.  This could be overcome by increasing the softness parameter during alchem. transformations.  Discrepancies remained in the bound state however, when comparing independent simulations of the three ligands.  These difficulties could be traced to a slow relaxation of the water network within the active site.  Fluctuations in the no. of water mols. residing in the binding cavity occur mostly on a timescale larger than the simulation time along the alchem. path.  After extensive simulations, relative binding free energies that were converged to within thermal noise could be obtained, which agree well with available exptl. data.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojG9n-hQ3wDLVg90H21EOLACvtfcHk0lgpgHAflk6HwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVCktLfJ&md5=5179bbcb1baa17bef2da25d34a245951</span></div><a href="/servlet/linkout?suffix=cit81&amp;dbid=16384&amp;doi=10.3390%2Fmolecules21040499&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fmolecules21040499%26sid%3Dliteratum%253Aachs%26aulast%3DMaurer%26aufirst%3DM.%26aulast%3Dde%2BBeer%26aufirst%3DS.%2BB.%26aulast%3DOostenbrink%26aufirst%3DC.%26atitle%3DCalculation%2520of%2520relative%2520binding%2520free%2520energy%2520in%2520the%2520water-filled%2520active%2520site%2520of%2520oligopeptide-binding%2520protein%2520A%26jtitle%3DMolecules%26date%3D2016%26volume%3D21%26spage%3D499%26doi%3D10.3390%2Fmolecules21040499" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref82"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref82'); return false;" data-citation="" class="refNumLink">82</a></strong><div class="NLM_citation" id="cit82"><span><span class="NLM_contrib-group">Lai, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nagy, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garate, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oostenbrink, C.</span><span> </span><span class="NLM_article-title">Entropic and enthalpic contributions to stereospecific ligand binding from enhanced sampling methods</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">151</span><span class="NLM_x">–</span> <span class="NLM_lpage">158</span><span class="refDoi"> DOI: 10.1021/ci4006657</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci4006657" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC3sXitVWnsbrE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2014&pages=151-158&author=B.+Laiauthor=G.+Nagyauthor=J.+A.+Garateauthor=C.+Oostenbrink&title=Entropic+and+enthalpic+contributions+to+stereospecific+ligand+binding+from+enhanced+sampling+methods&doi=10.1021%2Fci4006657"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit82R"><div class="casContent"><span class="casTitleNuber">82</span><div class="casTitle"><span class="NLM_cas:atitle">Entropic and enthalpic contributions to stereospecific ligand binding from enhanced sampling methods</span></div><div class="casAuthors">Lai, Balder; Nagy, Gabor; Garate, Jose Antonio; Oostenbrink, Chris</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">151-158</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The stereoselective binding of R- and S-propranolol to the metabolic enzyme cytochrome P 450 2D6 and its mutant F483A was studied using various computational approaches.  Previously reported free-energy differences from Hamiltonian replica exchange simulations, combined with thermodn. integration, were compared to the one-step perturbation approach, combined with local-elevation enhanced sampling, and an excellent agreement between methods was obtained.  Further, the free-energy differences were interpreted in terms of enthalpic and entropic contributions where it was shown that exactly compensating contributions obscured a mol. interpretation of differences in the affinity while various reduced terms allowed a more detailed anal., which agreed with heuristic observations on the interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrU4EblLCkBJrVg90H21EOLACvtfcHk0lgpgHAflk6HwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXitVWnsbrE&md5=441379a9078f425cb709ee1f31cdfc0a</span></div><a href="/servlet/linkout?suffix=cit82&amp;dbid=16384&amp;doi=10.1021%2Fci4006657&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci4006657%26sid%3Dliteratum%253Aachs%26aulast%3DLai%26aufirst%3DB.%26aulast%3DNagy%26aufirst%3DG.%26aulast%3DGarate%26aufirst%3DJ.%2BA.%26aulast%3DOostenbrink%26aufirst%3DC.%26atitle%3DEntropic%2520and%2520enthalpic%2520contributions%2520to%2520stereospecific%2520ligand%2520binding%2520from%2520enhanced%2520sampling%2520methods%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2014%26volume%3D54%26spage%3D151%26epage%3D158%26doi%3D10.1021%2Fci4006657" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref83"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref83'); return false;" data-citation="" class="refNumLink">83</a></strong><div class="NLM_citation" id="cit83"><span><span class="NLM_contrib-group">de Ruiter, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oostenbrink, C.</span><span> </span><span class="NLM_article-title">Efficient and accurate free energy calculations on trypsin inhibitors</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">3686</span><span class="NLM_x">–</span> <span class="NLM_lpage">3695</span><span class="refDoi"> DOI: 10.1021/ct200750p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct200750p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtFensbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2012&pages=3686-3695&author=A.+de+Ruiterauthor=C.+Oostenbrink&title=Efficient+and+accurate+free+energy+calculations+on+trypsin+inhibitors&doi=10.1021%2Fct200750p"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit83R"><div class="casContent"><span class="casTitleNuber">83</span><div class="casTitle"><span class="NLM_cas:atitle">Efficient and Accurate Free Energy Calculations on Trypsin Inhibitors</span></div><div class="casAuthors">de Ruiter, Anita; Oostenbrink, Chris</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3686-3695</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Several combinations of free energy calcn. methods have been applied to det. the relative free energies of binding between eight para-substituted benzamidines in complex with the serine protease trypsin.  With the aim to improve efficiency and maintain accuracy, the linear response approxn. (LRA), linear interaction energy (LIE) and third power fitting (TPF) are combined with the one-step perturbation (OSP) to det. the polar and apolar contributions to the free energy, resp.  The combination TPF/OSP gives the most accurate results and is 4.5 times more efficient than the rigorous thermodn. integration (TI).  By projecting the electrostatic preorganization energy from the OSP simulations, an increase in efficiency of a factor 7.5 can even be achieved.  Loss of accuracy with respect to the TI data is limited to 3.9 and 5.6 kJ/mol, resp., making it an attractive approach for lead optimization programs in drug research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbsob2EXdDIbVg90H21EOLACvtfcHk0lhWuuykt2MzNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtFensbk%253D&md5=f43a25a8fd2ffc594c53f45c4a55b2ff</span></div><a href="/servlet/linkout?suffix=cit83&amp;dbid=16384&amp;doi=10.1021%2Fct200750p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct200750p%26sid%3Dliteratum%253Aachs%26aulast%3Dde%2BRuiter%26aufirst%3DA.%26aulast%3DOostenbrink%26aufirst%3DC.%26atitle%3DEfficient%2520and%2520accurate%2520free%2520energy%2520calculations%2520on%2520trypsin%2520inhibitors%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2012%26volume%3D8%26spage%3D3686%26epage%3D3695%26doi%3D10.1021%2Fct200750p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref84"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref84'); return false;" data-citation="" class="refNumLink">84</a></strong><div class="NLM_citation" id="cit84"><span><span class="NLM_contrib-group">De Beer, S. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Glattli, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hutzler, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vermeulen, N. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oostenbrink, C.</span><span> </span><span class="NLM_article-title">Molecular dynamics simulations and free energy calculations on the enzyme 4-hydroxyphenylpyruvate dioxygenase</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">32</span><span class="NLM_x">, </span> <span class="NLM_fpage">2160</span><span class="NLM_x">–</span> <span class="NLM_lpage">2169</span><span class="refDoi"> DOI: 10.1002/jcc.21798</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1002%2Fjcc.21798" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2011&pages=2160-2169&author=S.+B.+De+Beerauthor=A.+Glattliauthor=J.+Hutzlerauthor=N.+P.+Vermeulenauthor=C.+Oostenbrink&title=Molecular+dynamics+simulations+and+free+energy+calculations+on+the+enzyme+4-hydroxyphenylpyruvate+dioxygenase&doi=10.1002%2Fjcc.21798"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit84&amp;dbid=16384&amp;doi=10.1002%2Fjcc.21798&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.21798%26sid%3Dliteratum%253Aachs%26aulast%3DDe%2BBeer%26aufirst%3DS.%2BB.%26aulast%3DGlattli%26aufirst%3DA.%26aulast%3DHutzler%26aufirst%3DJ.%26aulast%3DVermeulen%26aufirst%3DN.%2BP.%26aulast%3DOostenbrink%26aufirst%3DC.%26atitle%3DMolecular%2520dynamics%2520simulations%2520and%2520free%2520energy%2520calculations%2520on%2520the%2520enzyme%25204-hydroxyphenylpyruvate%2520dioxygenase%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2011%26volume%3D32%26spage%3D2160%26epage%3D2169%26doi%3D10.1002%2Fjcc.21798" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref85"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref85'); return false;" data-citation="" class="refNumLink">85</a></strong><div class="NLM_citation" id="cit85"><span><span class="NLM_contrib-group">Chiang, Y. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y.</span><span> </span><span class="NLM_article-title">Virtual substitution scan via single-step free energy perturbation</span> <span class="citation_source-journal">Biopolymers</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">105</span><span class="NLM_x">, </span> <span class="NLM_fpage">324</span><span class="NLM_x">–</span> <span class="NLM_lpage">336</span><span class="refDoi"> DOI: 10.1002/bip.22820</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1002%2Fbip.22820" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=105&publication_year=2016&pages=324-336&author=Y.+C.+Chiangauthor=Y.+Wang&title=Virtual+substitution+scan+via+single-step+free+energy+perturbation&doi=10.1002%2Fbip.22820"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit85&amp;dbid=16384&amp;doi=10.1002%2Fbip.22820&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fbip.22820%26sid%3Dliteratum%253Aachs%26aulast%3DChiang%26aufirst%3DY.%2BC.%26aulast%3DWang%26aufirst%3DY.%26atitle%3DVirtual%2520substitution%2520scan%2520via%2520single-step%2520free%2520energy%2520perturbation%26jtitle%3DBiopolymers%26date%3D2016%26volume%3D105%26spage%3D324%26epage%3D336%26doi%3D10.1002%2Fbip.22820" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref86"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref86'); return false;" data-citation="" class="refNumLink">86</a></strong><div class="NLM_citation" id="cit86"><span><span class="NLM_contrib-group">Zeevaart, J. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, L. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thakur, V. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leung, C. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tirado-Rives, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bailey, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Domaoal, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Anderson, K. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jorgensen, W. L.</span><span> </span><span class="NLM_article-title">Optimization of azoles as anti-human immunodeficiency virus agents guided by free-energy calculations</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">130</span><span class="NLM_x">, </span> <span class="NLM_fpage">9492</span><span class="NLM_x">–</span> <span class="NLM_lpage">9499</span><span class="refDoi"> DOI: 10.1021/ja8019214</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja8019214" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BD1cXnvF2hs7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=130&publication_year=2008&pages=9492-9499&author=J.+G.+Zeevaartauthor=L.+G.+Wangauthor=V.+V.+Thakurauthor=C.+S.+Leungauthor=J.+Tirado-Rivesauthor=C.+M.+Baileyauthor=R.+A.+Domaoalauthor=K.+S.+Andersonauthor=W.+L.+Jorgensen&title=Optimization+of+azoles+as+anti-human+immunodeficiency+virus+agents+guided+by+free-energy+calculations&doi=10.1021%2Fja8019214"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit86R"><div class="casContent"><span class="casTitleNuber">86</span><div class="casTitle"><span class="NLM_cas:atitle">Optimization of Azoles as Anti-Human Immunodeficiency Virus Agents Guided by Free-Energy Calculations</span></div><div class="casAuthors">Zeevaart, Jacob G.; Wang, Ligong; Thakur, Vinay V.; Leung, Cheryl S.; Tirado-Rives, Julian; Bailey, Christopher M.; Domaoal, Robert A.; Anderson, Karen S.; Jorgensen, William L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">130</span>
        (<span class="NLM_cas:issue">29</span>),
    <span class="NLM_cas:pages">9492-9499</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Efficient optimization of an inactive 2-anilinyl-5-benzyloxadiazole core has been guided by free energy perturbation (FEP) calcns. to provide potent nonnucleoside inhibitors of human immunodeficiency virus (HIV) reverse transcriptase (NNRTIs).  An FEP "chlorine scan" was performed to identify the most promising sites for substitution of aryl hydrogens.  This yielded NNRTIs (I and II) with activities (EC50) of 820 and 310 nM for protection of human T-cells from infection by wild-type HIV-1.  FEP calcns. for addnl. substituent modifications and change of the core heterocycle readily led to oxazoles (III and IV), which were confirmed as highly potent anti-HIV agents with activities in the 10-20 nM range.  The designed compds. were also monitored for possession of desirable pharmacol. properties by use of addnl. computational tools.  Overall, the trends predicted by the FEP calcns. were well borne out by the assay results.  FEP-guided lead optimization is confirmed as a valuable tool for mol. design including drug discovery; chlorine scans are particularly attractive since they are both straightforward to perform and highly informative.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZtMlzatLNhLVg90H21EOLACvtfcHk0lhWuuykt2MzNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXnvF2hs7s%253D&md5=24df481811465307254908323aa470b3</span></div><a href="/servlet/linkout?suffix=cit86&amp;dbid=16384&amp;doi=10.1021%2Fja8019214&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja8019214%26sid%3Dliteratum%253Aachs%26aulast%3DZeevaart%26aufirst%3DJ.%2BG.%26aulast%3DWang%26aufirst%3DL.%2BG.%26aulast%3DThakur%26aufirst%3DV.%2BV.%26aulast%3DLeung%26aufirst%3DC.%2BS.%26aulast%3DTirado-Rives%26aufirst%3DJ.%26aulast%3DBailey%26aufirst%3DC.%2BM.%26aulast%3DDomaoal%26aufirst%3DR.%2BA.%26aulast%3DAnderson%26aufirst%3DK.%2BS.%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26atitle%3DOptimization%2520of%2520azoles%2520as%2520anti-human%2520immunodeficiency%2520virus%2520agents%2520guided%2520by%2520free-energy%2520calculations%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2008%26volume%3D130%26spage%3D9492%26epage%3D9499%26doi%3D10.1021%2Fja8019214" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref87"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref87'); return false;" data-citation="" class="refNumLink">87</a></strong><div class="NLM_citation" id="cit87"><span><span class="NLM_contrib-group">Jorgensen, W. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bollini, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thakur, V. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Domaoal, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spasov, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Anderson, K. S.</span><span> </span><span class="NLM_article-title">Efficient discovery of potent anti-HIV agents targeting the Tyr181Cys variant of HIV reverse transcriptase</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">133</span><span class="NLM_x">, </span> <span class="NLM_fpage">15686</span><span class="NLM_x">–</span> <span class="NLM_lpage">15696</span><span class="refDoi"> DOI: 10.1021/ja2058583</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja2058583" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFCgs73I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=133&publication_year=2011&pages=15686-15696&author=W.+L.+Jorgensenauthor=M.+Bolliniauthor=V.+V.+Thakurauthor=R.+A.+Domaoalauthor=K.+A.+Spasovauthor=K.+S.+Anderson&title=Efficient+discovery+of+potent+anti-HIV+agents+targeting+the+Tyr181Cys+variant+of+HIV+reverse+transcriptase&doi=10.1021%2Fja2058583"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit87R"><div class="casContent"><span class="casTitleNuber">87</span><div class="casTitle"><span class="NLM_cas:atitle">Efficient Discovery of Potent Anti-HIV Agents Targeting the Tyr181Cys Variant of HIV Reverse Transcriptase</span></div><div class="casAuthors">Jorgensen, William L.; Bollini, Mariela; Thakur, Vinay V.; Domaoal, Robert A.; Spasov, Krasimir A.; Anderson, Karen S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">133</span>
        (<span class="NLM_cas:issue">39</span>),
    <span class="NLM_cas:pages">15686-15696</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Non-nucleoside reverse transcriptase inhibitors (NNRTIs) that interfere with the replication of human immunodeficiency virus (HIV) are being pursued with guidance from mol. modeling including free-energy perturbation (FEP) calcns. for protein-inhibitor binding affinities.  The previously reported pyrimidinylphenylamine 1 (I) and its chloro analog 2 are potent anti-HIV agents; they inhibit replication of wild-type HIV-1 in infected human T-cells with EC50 values of 2 and 10 nM, resp.  However, they show no activity against viral strains contg. the Tyr181Cys (Y181C) mutation in HIV-RT.  Modeling indicates that the problem is likely assocd. with extensive interaction between the dimethylallyloxy substituent and Tyr181.  As an alternative, a phenoxy group is computed to be oriented in a manner diminishing the contact with Tyr181.  However, this replacement leads to a roughly 1000-fold loss of activity for 3 (2.5 μM).  The present report details the efficient, computationally driven evolution of 3 to novel NNRTIs with sub-10 nM potency toward both wild-type HIV-1 and Y181C-contg. variants.  The crit. contributors were FEP substituent scans for the phenoxy and pyrimidine rings and recognition of potential benefits of addn. of a cyanovinyl group to the phenoxy ring.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXO8wk_lbYPrVg90H21EOLACvtfcHk0lhfvdliyOkBHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFCgs73I&md5=e75db60a08ad0c22ef6abcb5e1ea4891</span></div><a href="/servlet/linkout?suffix=cit87&amp;dbid=16384&amp;doi=10.1021%2Fja2058583&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja2058583%26sid%3Dliteratum%253Aachs%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26aulast%3DBollini%26aufirst%3DM.%26aulast%3DThakur%26aufirst%3DV.%2BV.%26aulast%3DDomaoal%26aufirst%3DR.%2BA.%26aulast%3DSpasov%26aufirst%3DK.%2BA.%26aulast%3DAnderson%26aufirst%3DK.%2BS.%26atitle%3DEfficient%2520discovery%2520of%2520potent%2520anti-HIV%2520agents%2520targeting%2520the%2520Tyr181Cys%2520variant%2520of%2520HIV%2520reverse%2520transcriptase%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2011%26volume%3D133%26spage%3D15686%26epage%3D15696%26doi%3D10.1021%2Fja2058583" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref88"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref88'); return false;" data-citation="" class="refNumLink">88</a></strong><div class="NLM_citation" id="cit88"><span><span class="NLM_contrib-group">Leung, C. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leung, S. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tirado-Rives, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jorgensen, W. L.</span><span> </span><span class="NLM_article-title">Methyl effects on protein-ligand binding</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">4489</span><span class="NLM_x">–</span> <span class="NLM_lpage">4500</span><span class="refDoi"> DOI: 10.1021/jm3003697</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm3003697" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC38XlsV2iu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=4489-4500&author=C.+S.+Leungauthor=S.+S.+Leungauthor=J.+Tirado-Rivesauthor=W.+L.+Jorgensen&title=Methyl+effects+on+protein-ligand+binding&doi=10.1021%2Fjm3003697"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit88R"><div class="casContent"><span class="casTitleNuber">88</span><div class="casTitle"><span class="NLM_cas:atitle">Methyl Effects on Protein-Ligand Binding</span></div><div class="casAuthors">Leung, Cheryl S.; Leung, Siegfried S. F.; Tirado-Rives, Julian; Jorgensen, William L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4489-4500</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The effects of addn. of a Me group to a lead compd. on biol. activity are examd.  A literature anal. of >2000 cases reveals that an activity boost of a factor of 10 or more is found with an 8% frequency, and a 100-fold boost is a 1 in 200 event.  Four cases in the latter category are analyzed in depth to elucidate any unusual aspects of the protein-ligand binding, distribution of water mols., and changes in conformational energetics.  The analyses include Monte Carlo/free-energy perturbation (MC/FEP) calcns. for Me replacements in inhibitor series for p38α MAP kinase, ACK1, PTP1B, and thrombin.  Me substitutions ortho to an aryl ring can be particularly effective at improving activity by inducing a propitious conformational change.  The greatest improvements in activity arise from coupling the conformational gain with the burial of the Me group in a hydrophobic region of the protein.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoX0Czyhgy6KbVg90H21EOLACvtfcHk0lgc00FH_XGz-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XlsV2iu7Y%253D&md5=0ea58d9c1bf845d383931b6a017f9b37</span></div><a href="/servlet/linkout?suffix=cit88&amp;dbid=16384&amp;doi=10.1021%2Fjm3003697&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm3003697%26sid%3Dliteratum%253Aachs%26aulast%3DLeung%26aufirst%3DC.%2BS.%26aulast%3DLeung%26aufirst%3DS.%2BS.%26aulast%3DTirado-Rives%26aufirst%3DJ.%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26atitle%3DMethyl%2520effects%2520on%2520protein-ligand%2520binding%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D4489%26epage%3D4500%26doi%3D10.1021%2Fjm3003697" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref89"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref89'); return false;" data-citation="" class="refNumLink">89</a></strong><div class="NLM_citation" id="cit89"><span><span class="NLM_contrib-group">Cole, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tirado-Rives, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jorgensen, W. L.</span><span> </span><span class="NLM_article-title">Molecular dynamics and Monte Carlo simulations for protein-ligand binding and inhibitor design</span> <span class="citation_source-journal">Biochim. Biophys. Acta, Gen. Subj.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">1850</span><span class="NLM_x">, </span> <span class="NLM_fpage">966</span><span class="NLM_x">–</span> <span class="NLM_lpage">971</span><span class="refDoi"> DOI: 10.1016/j.bbagen.2014.08.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1016%2Fj.bbagen.2014.08.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=25196360" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsFWltrvK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1850&publication_year=2015&pages=966-971&author=D.+J.+Coleauthor=J.+Tirado-Rivesauthor=W.+L.+Jorgensen&title=Molecular+dynamics+and+Monte+Carlo+simulations+for+protein-ligand+binding+and+inhibitor+design&doi=10.1016%2Fj.bbagen.2014.08.018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit89R"><div class="casContent"><span class="casTitleNuber">89</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular dynamics and Monte Carlo simulations for protein-ligand binding and inhibitor design</span></div><div class="casAuthors">Cole, Daniel J.; Tirado-Rives, Julian; Jorgensen, William L.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, General Subjects</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">1850</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">966-971</span>CODEN:
                <span class="NLM_cas:coden">BBGSB3</span>;
        ISSN:<span class="NLM_cas:issn">0304-4165</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Non-nucleoside inhibitors of HIV reverse transcriptase are an important component of treatment against HIV infection.  Novel inhibitors are sought that increase potency against variants that contain the Tyr181Cys mutation.  Mol. dynamics based free energy perturbation simulations have been run to study factors that contribute to protein-ligand binding, and the results are compared with those from previous Monte Carlo based simulations and activity data.  Predictions of protein-ligand binding modes are very consistent for the two simulation methods; the accord is attributed to the use of an enhanced sampling protocol.  The Tyr181Cys binding pocket supports large, hydrophobic substituents, which is in good agreement with expt.  Although some discrepancies exist between the results of the two simulation methods and expt., free energy perturbation simulations can be used to rapidly test small mols. for gains in binding affinity.  Free energy perturbation methods show promise in providing fast, reliable and accurate data that can be used to complement expt. in lead optimization projects.  This article is part of a Special Issue entitled "Recent developments of mol. dynamics".</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1gmI-9_FKtrVg90H21EOLACvtfcHk0lgc00FH_XGz-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsFWltrvK&md5=2fd7025d2402b77ba6fb2e091733bc12</span></div><a href="/servlet/linkout?suffix=cit89&amp;dbid=16384&amp;doi=10.1016%2Fj.bbagen.2014.08.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbagen.2014.08.018%26sid%3Dliteratum%253Aachs%26aulast%3DCole%26aufirst%3DD.%2BJ.%26aulast%3DTirado-Rives%26aufirst%3DJ.%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26atitle%3DMolecular%2520dynamics%2520and%2520Monte%2520Carlo%2520simulations%2520for%2520protein-ligand%2520binding%2520and%2520inhibitor%2520design%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Gen.%2520Subj.%26date%3D2015%26volume%3D1850%26spage%3D966%26epage%3D971%26doi%3D10.1016%2Fj.bbagen.2014.08.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref90"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref90'); return false;" data-citation="" class="refNumLink">90</a></strong><div class="NLM_citation" id="cit90"><span><span class="NLM_contrib-group">Lovering, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aevazelis, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dehnhardt, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fitz, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Han, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janz, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaila, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McDonald, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moore, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moretto, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Papaioannou, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richard, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ryan, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wan, Z. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thorarensen, A.</span><span> </span><span class="NLM_article-title">Imidazotriazines: spleen tyrosine kinase (Syk) inhibitors identified by free-energy perturbation (FEP)</span> <span class="citation_source-journal">ChemMedChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">217</span><span class="NLM_x">–</span> <span class="NLM_lpage">233</span><span class="refDoi"> DOI: 10.1002/cmdc.201500333</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref90/cit90&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1002%2Fcmdc.201500333" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref90/cit90&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=26381330" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref90/cit90&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFWlsLnO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=217-233&author=F.+Loveringauthor=C.+Aevazelisauthor=J.+Changauthor=C.+Dehnhardtauthor=L.+Fitzauthor=S.+Hanauthor=K.+Janzauthor=J.+Leeauthor=N.+Kailaauthor=J.+McDonaldauthor=W.+Mooreauthor=A.+Morettoauthor=N.+Papaioannouauthor=D.+Richardauthor=M.+S.+Ryanauthor=Z.+K.+Wanauthor=A.+Thorarensen&title=Imidazotriazines%3A+spleen+tyrosine+kinase+%28Syk%29+inhibitors+identified+by+free-energy+perturbation+%28FEP%29&doi=10.1002%2Fcmdc.201500333"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit90R"><div class="casContent"><span class="casTitleNuber">90</span><div class="casTitle"><span class="NLM_cas:atitle">Imidazotriazines: Spleen Tyrosine Kinase (Syk) Inhibitors Identified by Free-Energy Perturbation (FEP)</span></div><div class="casAuthors">Lovering, Frank; Aevazelis, Cristina; Chang, Jeanne; Dehnhardt, Christoph; Fitz, Lori; Han, Seungil; Janz, Kristin; Lee, Julie; Kaila, Neelu; McDonald, Joseph; Moore, William; Moretto, Alessandro; Papaioannou, Nikolaos; Richard, David; Ryan, Mark S.; Wan, Zhao-Kui; Thorarensen, Atli</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">217-233</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">There has been significant interest in spleen tyrosine kinase (Syk) owing to its role in a no. of disease states, including autoimmunity, inflammation, and cancer.  Ongoing therapeutic programs have resulted in several compds. that are now in clin. use.  Herein the authors report the optimization of the imidazopyrazine core scaffold of Syk inhibitors through the use of empirical and computational approaches.  Free-energy perturbation (FEP) methods with MCPRO+ were undertaken to calc. the relative binding free energies for several alternate scaffolds.  FEP was first applied retrospectively to det. if there is any predictive value; this resulted in 12 of 13 transformations being predicted in a directionally correct manner.  FEP was then applied in a prospective manner to evaluate 17 potential targets, resulting in the realization of imidazotriazine I, which shows a tenfold improvement in activity relative to the parent compd. and no increase in atom count.  Optimization of I led to compds. with nanomolar cellular activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpNV2-OpOenz7Vg90H21EOLACvtfcHk0lgc00FH_XGz-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFWlsLnO&md5=2797465d9360299ecc5d65499e338bbe</span></div><a href="/servlet/linkout?suffix=cit90&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201500333&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201500333%26sid%3Dliteratum%253Aachs%26aulast%3DLovering%26aufirst%3DF.%26aulast%3DAevazelis%26aufirst%3DC.%26aulast%3DChang%26aufirst%3DJ.%26aulast%3DDehnhardt%26aufirst%3DC.%26aulast%3DFitz%26aufirst%3DL.%26aulast%3DHan%26aufirst%3DS.%26aulast%3DJanz%26aufirst%3DK.%26aulast%3DLee%26aufirst%3DJ.%26aulast%3DKaila%26aufirst%3DN.%26aulast%3DMcDonald%26aufirst%3DJ.%26aulast%3DMoore%26aufirst%3DW.%26aulast%3DMoretto%26aufirst%3DA.%26aulast%3DPapaioannou%26aufirst%3DN.%26aulast%3DRichard%26aufirst%3DD.%26aulast%3DRyan%26aufirst%3DM.%2BS.%26aulast%3DWan%26aufirst%3DZ.%2BK.%26aulast%3DThorarensen%26aufirst%3DA.%26atitle%3DImidazotriazines%253A%2520spleen%2520tyrosine%2520kinase%2520%2528Syk%2529%2520inhibitors%2520identified%2520by%2520free-energy%2520perturbation%2520%2528FEP%2529%26jtitle%3DChemMedChem%26date%3D2016%26volume%3D11%26spage%3D217%26epage%3D233%26doi%3D10.1002%2Fcmdc.201500333" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref91"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref91'); return false;" data-citation="" class="refNumLink">91</a></strong><div class="NLM_citation" id="cit91"><span><span class="NLM_contrib-group">Rathore, R. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sumakanth, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reddy, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reddanna, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rao, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Erion, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reddy, M. R.</span><span> </span><span class="NLM_article-title">Advances in binding free energies calculations: QM/MM-based free energy perturbation method for drug design</span> <span class="citation_source-journal">Curr. Pharm. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">4674</span><span class="NLM_x">–</span> <span class="NLM_lpage">4686</span><span class="refDoi"> DOI: 10.2174/1381612811319260002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.2174%2F1381612811319260002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=23260025" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1Wns7rO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2013&pages=4674-4686&author=R.+S.+Rathoreauthor=M.+Sumakanthauthor=M.+S.+Reddyauthor=P.+Reddannaauthor=A.+A.+Raoauthor=M.+D.+Erionauthor=M.+R.+Reddy&title=Advances+in+binding+free+energies+calculations%3A+QM%2FMM-based+free+energy+perturbation+method+for+drug+design&doi=10.2174%2F1381612811319260002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit91R"><div class="casContent"><span class="casTitleNuber">91</span><div class="casTitle"><span class="NLM_cas:atitle">Advances in Binding Free Energies Calculations: QM/MM-Based Free Energy Perturbation Method for Drug Design</span></div><div class="casAuthors">Rathore, R. S.; Sumakanth, M.; Reddy, M. Siva; Reddanna, P.; Rao, Allam Appa; Erion, Mark D.; Reddy, M. R.</div><div class="citationInfo"><span class="NLM_cas:title">Current Pharmaceutical Design</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">4674-4686</span>CODEN:
                <span class="NLM_cas:coden">CPDEFP</span>;
        ISSN:<span class="NLM_cas:issn">1381-6128</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Multiple approaches have been devised and evaluated to computationally est. binding free energies.  Results using a recently developed Quantum Mechanics (QM)/Mol. Mechanics (MM) based Free Energy Perturbation (FEP) method suggest that this method has the potential to provide the most accurate estn. of binding affinities to date.  The method treats ligands/inhibitors using QM while using MM for the rest of the system.  The method has been applied and validated for a structurally diverse set of fructose 1,6- bisphosphatase (FBPase) inhibitors suggesting that the approach has the potential to be used as an integral part of drug discovery for both lead identification lead optimization, where there is a structure available.  In addn., this QM/MM-based FEP method was shown to accurately replicate the anomalous hydration behavior exhibited by simple amines and amides suggesting that the method may also prove useful in predicting phys. properties of mols.  While the method is about 5-fold more computationally demanding than conventional FEP, it has the potential to be less demanding on the end user since it avoids development of MM force field parameters for novel ligands and thereby eliminates this time-consuming step that often contributes significantly to the inaccuracy of binding affinity predictions using conventional FEP methods.  The QM/MM-based FEP method has been extensively tested with respect to important considerations such as the length of the simulation required to obtain satisfactory convergence in the calcd. relative solvation and binding free energies for both small and large structural changes between ligands.  Future automation of the method and parallelization of the code is expected to enhance the speed and increase its use for drug design and lead optimization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8cjSLqWli5bVg90H21EOLACvtfcHk0lhFBIJTLnWvPQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1Wns7rO&md5=04862c0dd7413ff5aeaf598459d234dc</span></div><a href="/servlet/linkout?suffix=cit91&amp;dbid=16384&amp;doi=10.2174%2F1381612811319260002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1381612811319260002%26sid%3Dliteratum%253Aachs%26aulast%3DRathore%26aufirst%3DR.%2BS.%26aulast%3DSumakanth%26aufirst%3DM.%26aulast%3DReddy%26aufirst%3DM.%2BS.%26aulast%3DReddanna%26aufirst%3DP.%26aulast%3DRao%26aufirst%3DA.%2BA.%26aulast%3DErion%26aufirst%3DM.%2BD.%26aulast%3DReddy%26aufirst%3DM.%2BR.%26atitle%3DAdvances%2520in%2520binding%2520free%2520energies%2520calculations%253A%2520QM%252FMM-based%2520free%2520energy%2520perturbation%2520method%2520for%2520drug%2520design%26jtitle%3DCurr.%2520Pharm.%2520Des.%26date%3D2013%26volume%3D19%26spage%3D4674%26epage%3D4686%26doi%3D10.2174%2F1381612811319260002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref92"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref92'); return false;" data-citation="" class="refNumLink">92</a></strong><div class="NLM_citation" id="cit92"><span><span class="NLM_contrib-group">Genheden, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ryde, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soderhjelm, P.</span><span> </span><span class="NLM_article-title">Binding affinities by alchemical perturbation using QM/MM with a large QM system and polarizable MM model</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">36</span><span class="NLM_x">, </span> <span class="NLM_fpage">2114</span><span class="NLM_x">–</span> <span class="NLM_lpage">2124</span><span class="refDoi"> DOI: 10.1002/jcc.24048</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1002%2Fjcc.24048" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=26280564" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtlCisbjI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2015&pages=2114-2124&author=S.+Genhedenauthor=U.+Rydeauthor=P.+Soderhjelm&title=Binding+affinities+by+alchemical+perturbation+using+QM%2FMM+with+a+large+QM+system+and+polarizable+MM+model&doi=10.1002%2Fjcc.24048"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit92R"><div class="casContent"><span class="casTitleNuber">92</span><div class="casTitle"><span class="NLM_cas:atitle">Binding affinities by alchemical perturbation using QM/MM with a large QM system and polarizable MM model</span></div><div class="casAuthors">Genheden, Samuel; Ryde, Ulf; Soederhjelm, Paer</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">28</span>),
    <span class="NLM_cas:pages">2114-2124</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">The most general way to improve the accuracy of binding-affinity calcns. for protein-ligand systems is to use quantum-mech. (QM) methods together with rigorous alchem.-perturbation (AP) methods.  We explore this approach by calcg. the relative binding free energy of two synthetic disaccharides binding to galectin-3 at a reasonably high QM level (dispersion-cor. d. functional theory with a triple-zeta basis set) and with a sufficiently large QM system to include all short-range interactions with the ligand (744-748 atoms).  The rest of the protein is treated as a collection of at. multipoles (up to quadrupoles) and polarizabilities.  Several methods for evaluating the binding free energy from the 3600 QM calcns. are investigated in terms of stability and accuracy.  In particular, methods using QM calcns. only at the endpoints of the transformation are compared with the recently proposed non-Boltzmann Bennett acceptance ratio (NBB) method that uses QM calcns. at several stages of the transformation.  Unfortunately, none of the rigorous approaches give sufficient statistical precision.  However, a novel approx. method, involving the direct use of QM energies in the Bennett acceptance ratio method, gives similar results as NBB but with better precision, ∼3 kJ/mol.  The statistical error can be further reduced by performing a greater no. of QM calcns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqd0jf7UhCPvLVg90H21EOLACvtfcHk0lhFBIJTLnWvPQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtlCisbjI&md5=8e02ea35846d8335dfa2259d303b2e0c</span></div><a href="/servlet/linkout?suffix=cit92&amp;dbid=16384&amp;doi=10.1002%2Fjcc.24048&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.24048%26sid%3Dliteratum%253Aachs%26aulast%3DGenheden%26aufirst%3DS.%26aulast%3DRyde%26aufirst%3DU.%26aulast%3DSoderhjelm%26aufirst%3DP.%26atitle%3DBinding%2520affinities%2520by%2520alchemical%2520perturbation%2520using%2520QM%252FMM%2520with%2520a%2520large%2520QM%2520system%2520and%2520polarizable%2520MM%2520model%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2015%26volume%3D36%26spage%3D2114%26epage%3D2124%26doi%3D10.1002%2Fjcc.24048" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref93"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref93'); return false;" data-citation="" class="refNumLink">93</a></strong><div class="NLM_citation" id="cit93"><span><span class="NLM_contrib-group">Ignjatovic, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caldararu, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Munoz-Gutierrez, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adasme-Carreno, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ryde, U.</span><span> </span><span class="NLM_article-title">Binding-affinity predictions of HSP90 in the D3R Grand Challenge 2015 with docking, MM/GBSA, QM/MM, and free-energy simulations</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">30</span><span class="NLM_x">, </span> <span class="NLM_fpage">707</span><span class="NLM_x">–</span> <span class="NLM_lpage">730</span><span class="refDoi"> DOI: 10.1007/s10822-016-9942-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1007%2Fs10822-016-9942-z" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2016&pages=707-730&author=M.+M.+Ignjatovicauthor=O.+Caldararuauthor=G.+Dongauthor=C.+Munoz-Gutierrezauthor=F.+Adasme-Carrenoauthor=U.+Ryde&title=Binding-affinity+predictions+of+HSP90+in+the+D3R+Grand+Challenge+2015+with+docking%2C+MM%2FGBSA%2C+QM%2FMM%2C+and+free-energy+simulations&doi=10.1007%2Fs10822-016-9942-z"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit93&amp;dbid=16384&amp;doi=10.1007%2Fs10822-016-9942-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-016-9942-z%26sid%3Dliteratum%253Aachs%26aulast%3DIgnjatovic%26aufirst%3DM.%2BM.%26aulast%3DCaldararu%26aufirst%3DO.%26aulast%3DDong%26aufirst%3DG.%26aulast%3DMunoz-Gutierrez%26aufirst%3DC.%26aulast%3DAdasme-Carreno%26aufirst%3DF.%26aulast%3DRyde%26aufirst%3DU.%26atitle%3DBinding-affinity%2520predictions%2520of%2520HSP90%2520in%2520the%2520D3R%2520Grand%2520Challenge%25202015%2520with%2520docking%252C%2520MM%252FGBSA%252C%2520QM%252FMM%252C%2520and%2520free-energy%2520simulations%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2016%26volume%3D30%26spage%3D707%26epage%3D730%26doi%3D10.1007%2Fs10822-016-9942-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref94"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref94'); return false;" data-citation="" class="refNumLink">94</a></strong><div class="NLM_citation" id="cit94"><span><span class="NLM_contrib-group">Fujitani, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tanida, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ito, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jayachandran, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Snow, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shirts, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sorin, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pande, V. S.</span><span> </span><span class="NLM_article-title">Direct calculation of the binding free energies of FKBP ligands</span> <span class="citation_source-journal">J. Chem. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">123</span><span class="NLM_x">, </span> <span class="NLM_fpage">084108</span><span class="refDoi"> DOI: 10.1063/1.1999637</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1063%2F1.1999637" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=16164283" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BD2MXpvFCksb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=123&publication_year=2005&pages=084108&author=H.+Fujitaniauthor=Y.+Tanidaauthor=M.+Itoauthor=G.+Jayachandranauthor=C.+D.+Snowauthor=M.+R.+Shirtsauthor=E.+J.+Sorinauthor=V.+S.+Pande&title=Direct+calculation+of+the+binding+free+energies+of+FKBP+ligands&doi=10.1063%2F1.1999637"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit94R"><div class="casContent"><span class="casTitleNuber">94</span><div class="casTitle"><span class="NLM_cas:atitle">Direct calculation of the binding free energies of FKBP ligands</span></div><div class="casAuthors">Fujitani, Hideaki; Tanida, Yoshiaki; Ito, Masakatsu; Jayachandran, Guha; Snow, Christopher D.; Shirts, Michael R.; Sorin, Eric J.; Pande, Vijay S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">123</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">084108/1-084108/5</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    
            (<span class="NLM_cas:orgname">American Institute of Physics</span>)
        </div><div class="casAbstract">Direct calcns. of the abs. free energies of binding for eight ligands to FKBP protein were performed using the Fujitsu BioServer massively parallel computer.  Using the latest version of the general assisted model building with energy refinement (AMBER) force field for ligand model parameters and the Bennett acceptance ratio for computing free-energy differences, we obtained an excellent linear fit between the calcd. and exptl. binding free energies.  The rms error from a linear fit is 0.4 kcal/mol for eight ligand complexes.  In comparison with a previous study of the binding energies of these same eight ligand complexes, these results suggest that the use of improved model parameters can lead to more predictive binding ests., and that these ests. can be obtained with significantly less computer time than previously thought.  These findings make such direct methods more attractive for use in rational drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKfcZ36Ys0aLVg90H21EOLACvtfcHk0lhFBIJTLnWvPQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXpvFCksb8%253D&md5=4ee04c498d1f0053c760d93287aa408b</span></div><a href="/servlet/linkout?suffix=cit94&amp;dbid=16384&amp;doi=10.1063%2F1.1999637&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.1999637%26sid%3Dliteratum%253Aachs%26aulast%3DFujitani%26aufirst%3DH.%26aulast%3DTanida%26aufirst%3DY.%26aulast%3DIto%26aufirst%3DM.%26aulast%3DJayachandran%26aufirst%3DG.%26aulast%3DSnow%26aufirst%3DC.%2BD.%26aulast%3DShirts%26aufirst%3DM.%2BR.%26aulast%3DSorin%26aufirst%3DE.%2BJ.%26aulast%3DPande%26aufirst%3DV.%2BS.%26atitle%3DDirect%2520calculation%2520of%2520the%2520binding%2520free%2520energies%2520of%2520FKBP%2520ligands%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D2005%26volume%3D123%26spage%3D084108%26doi%3D10.1063%2F1.1999637" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref95"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref95'); return false;" data-citation="" class="refNumLink">95</a></strong><div class="NLM_citation" id="cit95"><span><span class="NLM_contrib-group">Archontis, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Watson, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Andreou, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chrysina, E. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zographos, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oikonomakos, N. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karplus, M.</span><span> </span><span class="NLM_article-title">Glycogen phosphorylase inhibitors: a free energy perturbation analysis of glucopyranose spirohydantoin analogues</span> <span class="citation_source-journal">Proteins: Struct., Funct., Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">61</span><span class="NLM_x">, </span> <span class="NLM_fpage">984</span><span class="NLM_x">–</span> <span class="NLM_lpage">998</span><span class="refDoi"> DOI: 10.1002/prot.20641</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref95/cit95&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1002%2Fprot.20641" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2005&pages=984-998&author=G.+Archontisauthor=K.+A.+Watsonauthor=Q.+Xieauthor=G.+Andreouauthor=E.+D.+Chrysinaauthor=S.+E.+Zographosauthor=N.+G.+Oikonomakosauthor=M.+Karplus&title=Glycogen+phosphorylase+inhibitors%3A+a+free+energy+perturbation+analysis+of+glucopyranose+spirohydantoin+analogues&doi=10.1002%2Fprot.20641"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit95&amp;dbid=16384&amp;doi=10.1002%2Fprot.20641&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.20641%26sid%3Dliteratum%253Aachs%26aulast%3DArchontis%26aufirst%3DG.%26aulast%3DWatson%26aufirst%3DK.%2BA.%26aulast%3DXie%26aufirst%3DQ.%26aulast%3DAndreou%26aufirst%3DG.%26aulast%3DChrysina%26aufirst%3DE.%2BD.%26aulast%3DZographos%26aufirst%3DS.%2BE.%26aulast%3DOikonomakos%26aufirst%3DN.%2BG.%26aulast%3DKarplus%26aufirst%3DM.%26atitle%3DGlycogen%2520phosphorylase%2520inhibitors%253A%2520a%2520free%2520energy%2520perturbation%2520analysis%2520of%2520glucopyranose%2520spirohydantoin%2520analogues%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D2005%26volume%3D61%26spage%3D984%26epage%3D998%26doi%3D10.1002%2Fprot.20641" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref96"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref96'); return false;" data-citation="" class="refNumLink">96</a></strong><div class="NLM_citation" id="cit96"><span><span class="NLM_contrib-group">Steinbrecher, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hrenn, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dormann, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Merfort, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Labahn, A.</span><span> </span><span class="NLM_article-title">Bornyl (3, 4, 5-trihydroxy)-cinnamate-an optimized human neutrophil elastase inhibitor designed by free energy calculations</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">2385</span><span class="NLM_x">–</span> <span class="NLM_lpage">2390</span><span class="refDoi"> DOI: 10.1016/j.bmc.2007.11.070</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1016%2Fj.bmc.2007.11.070" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2008&pages=2385-2390&author=T.+Steinbrecherauthor=A.+Hrennauthor=K.+L.+Dormannauthor=I.+Merfortauthor=A.+Labahn&title=Bornyl+%283%2C+4%2C+5-trihydroxy%29-cinnamate-an+optimized+human+neutrophil+elastase+inhibitor+designed+by+free+energy+calculations&doi=10.1016%2Fj.bmc.2007.11.070"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit96&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2007.11.070&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2007.11.070%26sid%3Dliteratum%253Aachs%26aulast%3DSteinbrecher%26aufirst%3DT.%26aulast%3DHrenn%26aufirst%3DA.%26aulast%3DDormann%26aufirst%3DK.%2BL.%26aulast%3DMerfort%26aufirst%3DI.%26aulast%3DLabahn%26aufirst%3DA.%26atitle%3DBornyl%2520%25283%252C%25204%252C%25205-trihydroxy%2529-cinnamate-an%2520optimized%2520human%2520neutrophil%2520elastase%2520inhibitor%2520designed%2520by%2520free%2520energy%2520calculations%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2008%26volume%3D16%26spage%3D2385%26epage%3D2390%26doi%3D10.1016%2Fj.bmc.2007.11.070" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref97"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref97'); return false;" data-citation="" class="refNumLink">97</a></strong><div class="NLM_citation" id="cit97"><span><span class="NLM_contrib-group">Rogers, I. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gammon, D. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Naidoo, K. J.</span><span> </span><span class="NLM_article-title">Conformational preferences of plumbagin with phenyl-1-thioglucoside conjugates in solution and bound to MshB determined by aromatic association</span> <span class="citation_source-journal">Carbohydr. Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">371</span><span class="NLM_x">, </span> <span class="NLM_fpage">52</span><span class="NLM_x">–</span> <span class="NLM_lpage">60</span><span class="refDoi"> DOI: 10.1016/j.carres.2013.02.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1016%2Fj.carres.2013.02.001" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=371&publication_year=2013&pages=52-60&author=I.+L.+Rogersauthor=D.+W.+Gammonauthor=K.+J.+Naidoo&title=Conformational+preferences+of+plumbagin+with+phenyl-1-thioglucoside+conjugates+in+solution+and+bound+to+MshB+determined+by+aromatic+association&doi=10.1016%2Fj.carres.2013.02.001"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit97&amp;dbid=16384&amp;doi=10.1016%2Fj.carres.2013.02.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.carres.2013.02.001%26sid%3Dliteratum%253Aachs%26aulast%3DRogers%26aufirst%3DI.%2BL.%26aulast%3DGammon%26aufirst%3DD.%2BW.%26aulast%3DNaidoo%26aufirst%3DK.%2BJ.%26atitle%3DConformational%2520preferences%2520of%2520plumbagin%2520with%2520phenyl-1-thioglucoside%2520conjugates%2520in%2520solution%2520and%2520bound%2520to%2520MshB%2520determined%2520by%2520aromatic%2520association%26jtitle%3DCarbohydr.%2520Res.%26date%3D2013%26volume%3D371%26spage%3D52%26epage%3D60%26doi%3D10.1016%2Fj.carres.2013.02.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref98"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref98'); return false;" data-citation="" class="refNumLink">98</a></strong><div class="NLM_citation" id="cit98"><span><span class="NLM_contrib-group">Lee, H. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hsu, W. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hsu, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, Y. C.</span><span> </span><span class="NLM_article-title">Using thermodynamic integration MD simulation to compute relative protein-ligand binding free energy of a GSK3beta kinase inhibitor and its analogs</span> <span class="citation_source-journal">J. Mol. Graphics Modell.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">37</span><span class="NLM_x">–</span> <span class="NLM_lpage">49</span><span class="refDoi"> DOI: 10.1016/j.jmgm.2014.04.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1016%2Fj.jmgm.2014.04.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=24858254" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtVCisbvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2014&pages=37-49&author=H.+C.+Leeauthor=W.+C.+Hsuauthor=A.+L.+Liuauthor=C.+J.+Hsuauthor=Y.+C.+Sun&title=Using+thermodynamic+integration+MD+simulation+to+compute+relative+protein-ligand+binding+free+energy+of+a+GSK3beta+kinase+inhibitor+and+its+analogs&doi=10.1016%2Fj.jmgm.2014.04.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit98R"><div class="casContent"><span class="casTitleNuber">98</span><div class="casTitle"><span class="NLM_cas:atitle">Using thermodynamic integration MD simulation to compute relative protein-ligand binding free energy of a GSK3β kinase inhibitor and its analogs</span></div><div class="casAuthors">Lee, Hsing-Chou; Hsu, Wen-Chi; Liu, An-Lun; Hsu, Chia-Jen; Sun, Ying-Chieh</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Graphics & Modelling</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">37-49</span>CODEN:
                <span class="NLM_cas:coden">JMGMFI</span>;
        ISSN:<span class="NLM_cas:issn">1093-3263</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Thermodn. integration mol. dynamics simulation was used to investigate how TI-MD simulation preforms in reproducing relative protein-ligand binding free energy of a pair of analogous GSK3β kinase inhibitors of available exptl. data (see Fig. 1), and to predict the affinity for other analogs.  The computation for the pair gave a ΔΔG of 1.0 kcal/mol, which was in reasonably good agreement with the exptl. value of -0.1 kcal/mol.  The error bar was estd. at 0.5 kcal/mol.  Subsequently, we employed the same protocol to proceed with simulations to find analogous inhibitors with a stronger affinity.  Four analogs with a substitution at one site inside the binding pocket were the first to be tried, but no significant enhancement in affinity was found.  Subsequent simulations for another 7 analogs was focused on substitutions at the benzene ring of another site, which gave two analogs (analogs 9 and 10) with ΔΔG values of -0.6 and -0.8 kcal/mol, resp.  Both analogs had a -OH group at the meta position and another -OH group at the ortho position at the other side of the benzene ring, as shown in Table 3.  To explore further, another 4 analogs with this characteristic were investigated.  Three analogs with ΔΔG values of -2.2, -1.7 and -1.2 kcal/mol, resp., were found.  Hydrogen bond anal. suggested that the addnl. hydrogen bonds of the added -OH groups with Gln185 and/or Asn64, which did not appear in the ref. inhibitor or as an analog with one substitution only in the examd. cases, were the main contributors to an enhanced affinity.  A prediction for better inhibitors should interest experimentalists of enzyme and/or cell assays.  Anal. of the interactions between GSK3β kinase and the investigated analogs will be useful in the design of GSK3β kinase inhibitors for compds. of this class.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxtb6DM4P2-LVg90H21EOLACvtfcHk0lg4e7rU6ZNfdA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtVCisbvI&md5=6fea8120488533812c0544f26bc0e1b9</span></div><a href="/servlet/linkout?suffix=cit98&amp;dbid=16384&amp;doi=10.1016%2Fj.jmgm.2014.04.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmgm.2014.04.010%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DH.%2BC.%26aulast%3DHsu%26aufirst%3DW.%2BC.%26aulast%3DLiu%26aufirst%3DA.%2BL.%26aulast%3DHsu%26aufirst%3DC.%2BJ.%26aulast%3DSun%26aufirst%3DY.%2BC.%26atitle%3DUsing%2520thermodynamic%2520integration%2520MD%2520simulation%2520to%2520compute%2520relative%2520protein-ligand%2520binding%2520free%2520energy%2520of%2520a%2520GSK3beta%2520kinase%2520inhibitor%2520and%2520its%2520analogs%26jtitle%3DJ.%2520Mol.%2520Graphics%2520Modell.%26date%3D2014%26volume%3D51%26spage%3D37%26epage%3D49%26doi%3D10.1016%2Fj.jmgm.2014.04.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref99"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref99'); return false;" data-citation="" class="refNumLink">99</a></strong><div class="NLM_citation" id="cit99"><span><span class="NLM_contrib-group">Steinbrecher, T. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dahlgren, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cappel, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krilov, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abel, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friesner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sherman, W.</span><span> </span><span class="NLM_article-title">Accurate binding free energy predictions in fragment optimization</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">2411</span><span class="NLM_x">–</span> <span class="NLM_lpage">2420</span><span class="refDoi"> DOI: 10.1021/acs.jcim.5b00538</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.5b00538" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs1eku7%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2015&pages=2411-2420&author=T.+B.+Steinbrecherauthor=M.+Dahlgrenauthor=D.+Cappelauthor=T.+Linauthor=L.+Wangauthor=G.+Krilovauthor=R.+Abelauthor=R.+Friesnerauthor=W.+Sherman&title=Accurate+binding+free+energy+predictions+in+fragment+optimization&doi=10.1021%2Facs.jcim.5b00538"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit99R"><div class="casContent"><span class="casTitleNuber">99</span><div class="casTitle"><span class="NLM_cas:atitle">Accurate Binding Free Energy Predictions in Fragment Optimization</span></div><div class="casAuthors">Steinbrecher, Thomas B.; Dahlgren, Markus; Cappel, Daniel; Lin, Teng; Wang, Lingle; Krilov, Goran; Abel, Robert; Friesner, Richard; Sherman, Woody</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2411-2420</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Predicting protein-ligand binding free energies is a central aim of computational structure-based drug design (SBDD) - improved accuracy in binding free energy predictions could significantly reduce costs and accelerate project timelines in lead discovery and optimization.  The recent development and validation of advanced free energy calcn. methods represents a major step toward this goal.  Accurately predicting the relative binding free energy changes of modifications to ligands is esp. valuable in the field of fragment-based drug design, since fragment screens tend to deliver initial hits of low binding affinity that require multiple rounds of synthesis to gain the requisite potency for a project.  A free energy perturbation protocol, FEP+, which was previously validated on drug-like lead compds., is suitable for the calcn. of relative binding strengths of fragment-sized compds. as well.  The authors study several pharmaceutically relevant targets with a total of >90 fragments and find that the FEP+ methodol., which uses explicit solvent mol. dynamics and physics-based scoring with no parameters adjusted, can accurately predict relative fragment binding affinities.  The calcns. afford R2-values on av. >0.5 compared to exptl. data and root-mean-square errors of ∼1.1 kcal/mol overall, demonstrating significant improvements over the docking and MM-GBSA methods tested in this work and indicating that FEP+ has the requisite predictive power to impact fragment-based affinity optimization projects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2OM1KfoQE4bVg90H21EOLACvtfcHk0lg9j-HsJ_yAug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs1eku7%252FI&md5=116425dcc073d2907c888555e9061aa8</span></div><a href="/servlet/linkout?suffix=cit99&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.5b00538&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.5b00538%26sid%3Dliteratum%253Aachs%26aulast%3DSteinbrecher%26aufirst%3DT.%2BB.%26aulast%3DDahlgren%26aufirst%3DM.%26aulast%3DCappel%26aufirst%3DD.%26aulast%3DLin%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DKrilov%26aufirst%3DG.%26aulast%3DAbel%26aufirst%3DR.%26aulast%3DFriesner%26aufirst%3DR.%26aulast%3DSherman%26aufirst%3DW.%26atitle%3DAccurate%2520binding%2520free%2520energy%2520predictions%2520in%2520fragment%2520optimization%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2015%26volume%3D55%26spage%3D2411%26epage%3D2420%26doi%3D10.1021%2Facs.jcim.5b00538" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref100"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref100'); return false;" data-citation="" class="refNumLink">100</a></strong><div class="NLM_citation" id="cit100"><span><span class="NLM_contrib-group">Aldeghi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heifetz, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bodkin, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Biggin, P. C.</span><span> </span><span class="NLM_article-title">Accurate calculation of the absolute free energy of binding for drug molecules</span> <span class="citation_source-journal">Chem. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">207</span><span class="NLM_x">–</span> <span class="NLM_lpage">218</span><span class="refDoi"> DOI: 10.1039/C5SC02678D</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1039%2FC5SC02678D" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=26798447" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFGku7%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=207-218&author=M.+Aldeghiauthor=A.+Heifetzauthor=M.+J.+Bodkinauthor=S.+Knappauthor=P.+C.+Biggin&title=Accurate+calculation+of+the+absolute+free+energy+of+binding+for+drug+molecules&doi=10.1039%2FC5SC02678D"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit100R"><div class="casContent"><span class="casTitleNuber">100</span><div class="casTitle"><span class="NLM_cas:atitle">Accurate calculation of the absolute free energy of binding for drug molecules</span></div><div class="casAuthors">Aldeghi, Matteo; Heifetz, Alexander; Bodkin, Michael J.; Knapp, Stefan; Biggin, Philip C.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Science</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">207-218</span>CODEN:
                <span class="NLM_cas:coden">CSHCCN</span>;
        ISSN:<span class="NLM_cas:issn">2041-6520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Accurate prediction of binding affinities has been a central goal of computational chem. for decades, yet remains elusive.  Despite good progress, the required accuracy for use in a drug-discovery context has not been consistently achieved for drug-like mols.  Here, we perform abs. free energy calcns. based on a thermodn. cycle for a set of diverse inhibitors binding to bromodomain-contg. protein 4 (BRD4) and demonstrate that a mean abs. error of 0.6 kcal mol-1 can be achieved.  We also show a similar level of accuracy (1.0 kcal mol-1) can be achieved in pseudo prospective approach.  Bromodomains are epigenetic mark readers that recognize acetylation motifs and regulate gene transcription, and are currently being investigated as therapeutic targets for cancer and inflammation.  The unprecedented accuracy offers the exciting prospect that the binding free energy of drug-like compds. can be predicted for pharmacol. relevant targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrrNjJ8SvAg3rVg90H21EOLACvtfcHk0lg9j-HsJ_yAug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFGku7%252FI&md5=d00cac3b87f1bd11837d6f292bd8c6e5</span></div><a href="/servlet/linkout?suffix=cit100&amp;dbid=16384&amp;doi=10.1039%2FC5SC02678D&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC5SC02678D%26sid%3Dliteratum%253Aachs%26aulast%3DAldeghi%26aufirst%3DM.%26aulast%3DHeifetz%26aufirst%3DA.%26aulast%3DBodkin%26aufirst%3DM.%2BJ.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBiggin%26aufirst%3DP.%2BC.%26atitle%3DAccurate%2520calculation%2520of%2520the%2520absolute%2520free%2520energy%2520of%2520binding%2520for%2520drug%2520molecules%26jtitle%3DChem.%2520Sci.%26date%3D2016%26volume%3D7%26spage%3D207%26epage%3D218%26doi%3D10.1039%2FC5SC02678D" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref101"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref101'); return false;" data-citation="" class="refNumLink">101</a></strong><div class="NLM_citation" id="cit101"><span><span class="NLM_contrib-group">Ciordia, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perez-Benito, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Delgado, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trabanco, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tresadern, G.</span><span> </span><span class="NLM_article-title">Application of free energy perturbation for the design of BACE1 inhibitors</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">1856</span><span class="NLM_x">–</span> <span class="NLM_lpage">1871</span><span class="refDoi"> DOI: 10.1021/acs.jcim.6b00220</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.6b00220" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht12mu7%252FK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2016&pages=1856-1871&author=M.+Ciordiaauthor=L.+Perez-Benitoauthor=F.+Delgadoauthor=A.+A.+Trabancoauthor=G.+Tresadern&title=Application+of+free+energy+perturbation+for+the+design+of+BACE1+inhibitors&doi=10.1021%2Facs.jcim.6b00220"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit101R"><div class="casContent"><span class="casTitleNuber">101</span><div class="casTitle"><span class="NLM_cas:atitle">Application of Free Energy Perturbation for the Design of BACE1 Inhibitors</span></div><div class="casAuthors">Ciordia, Myriam; Perez-Benito, Laura; Delgado, Francisca; Trabanco, Andres A.; Tresadern, Gary</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1856-1871</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Novel spiroaminodihydropyrroles probing for optimized interactions at the P3 pocket of β-secretase 1 (BACE1) were designed with the use of free energy perturbation (FEP) calcns.  The resulting mols. showed pIC50 potencies in enzymic BACE1 inhibition assays ranging from approx. 5 to 7.  Good correlation was obsd. between the predicted activity from the FEP calcns. and exptl. activity.  Simulations run with a default 5 ns approach delivered a mean unsigned error (MUE) between prediction and expt. of 0.58 and 0.91 kcal/mol for retrospective and prospective applications, resp.  With longer simulations of 10 and 20 ns, the MUE was in both cases 0.57 kcal/mol for the retrospective application, and 0.69 and 0.59 kcal/mol for the prospective application.  Other considerations that impact the quality of the calcns. are discussed.  This work provides an example of the value of FEP as a computational tool for drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpqpi-OiII-FrVg90H21EOLACvtfcHk0lg9j-HsJ_yAug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht12mu7%252FK&md5=b7577a89492867fc334cb1ec937a4eb6</span></div><a href="/servlet/linkout?suffix=cit101&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.6b00220&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.6b00220%26sid%3Dliteratum%253Aachs%26aulast%3DCiordia%26aufirst%3DM.%26aulast%3DPerez-Benito%26aufirst%3DL.%26aulast%3DDelgado%26aufirst%3DF.%26aulast%3DTrabanco%26aufirst%3DA.%2BA.%26aulast%3DTresadern%26aufirst%3DG.%26atitle%3DApplication%2520of%2520free%2520energy%2520perturbation%2520for%2520the%2520design%2520of%2520BACE1%2520inhibitors%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2016%26volume%3D56%26spage%3D1856%26epage%3D1871%26doi%3D10.1021%2Facs.jcim.6b00220" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref102"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref102'); return false;" data-citation="" class="refNumLink">102</a></strong><div class="NLM_citation" id="cit102"><span><span class="NLM_contrib-group">Lenselink, E. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Louvel, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Forti, A. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Veldhoven, J. P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Vries, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mulder-Krieger, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McRobb, F. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Negri, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goose, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abel, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Vlijmen, H. W. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, L. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harder, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sherman, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">IJzerman, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beuming, T.</span><span> </span><span class="NLM_article-title">Predicting binding bffinities for GPCR ligands using free-energy perturbation</span> <span class="citation_source-journal">ACS Omega</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">293</span><span class="NLM_x">–</span> <span class="NLM_lpage">304</span><span class="refDoi"> DOI: 10.1021/acsomega.6b00086</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsomega.6b00086" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2016&pages=293-304&author=E.+B.+Lenselinkauthor=J.+Louvelauthor=A.+F.+Fortiauthor=J.+P.+D.+van+Veldhovenauthor=H.+de+Vriesauthor=T.+Mulder-Kriegerauthor=F.+M.+McRobbauthor=A.+Negriauthor=J.+Gooseauthor=R.+Abelauthor=H.+W.+T.+van+Vlijmenauthor=L.+L.+Wangauthor=E.+Harderauthor=W.+Shermanauthor=A.+P.+IJzermanauthor=T.+Beuming&title=Predicting+binding+bffinities+for+GPCR+ligands+using+free-energy+perturbation&doi=10.1021%2Facsomega.6b00086"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit102&amp;dbid=16384&amp;doi=10.1021%2Facsomega.6b00086&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsomega.6b00086%26sid%3Dliteratum%253Aachs%26aulast%3DLenselink%26aufirst%3DE.%2BB.%26aulast%3DLouvel%26aufirst%3DJ.%26aulast%3DForti%26aufirst%3DA.%2BF.%26aulast%3Dvan%2BVeldhoven%26aufirst%3DJ.%2BP.%2BD.%26aulast%3Dde%2BVries%26aufirst%3DH.%26aulast%3DMulder-Krieger%26aufirst%3DT.%26aulast%3DMcRobb%26aufirst%3DF.%2BM.%26aulast%3DNegri%26aufirst%3DA.%26aulast%3DGoose%26aufirst%3DJ.%26aulast%3DAbel%26aufirst%3DR.%26aulast%3Dvan%2BVlijmen%26aufirst%3DH.%2BW.%2BT.%26aulast%3DWang%26aufirst%3DL.%2BL.%26aulast%3DHarder%26aufirst%3DE.%26aulast%3DSherman%26aufirst%3DW.%26aulast%3DIJzerman%26aufirst%3DA.%2BP.%26aulast%3DBeuming%26aufirst%3DT.%26atitle%3DPredicting%2520binding%2520bffinities%2520for%2520GPCR%2520ligands%2520using%2520free-energy%2520perturbation%26jtitle%3DACS%2520Omega%26date%3D2016%26volume%3D1%26spage%3D293%26epage%3D304%26doi%3D10.1021%2Facsomega.6b00086" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref103"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref103'); return false;" data-citation="" class="refNumLink">103</a></strong><div class="NLM_citation" id="cit103"><span><span class="NLM_contrib-group">Ioannidis, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Drakopoulos, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tzitzoglaki, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Homeyer, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kolarov, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gkeka, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Freudenberger, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liolios, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gauglitz, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cournia, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gohlke, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kolocouris, A.</span><span> </span><span class="NLM_article-title">Alchemical free energy calculations and isothermal titration calorimetry measurements of aminoadamantanes bound to the closed state of influenza A/M2TM</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">862</span><span class="NLM_x">–</span> <span class="NLM_lpage">876</span><span class="refDoi"> DOI: 10.1021/acs.jcim.6b00079</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.6b00079" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC28XmsFOmtrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2016&pages=862-876&author=H.+Ioannidisauthor=A.+Drakopoulosauthor=C.+Tzitzoglakiauthor=N.+Homeyerauthor=F.+Kolarovauthor=P.+Gkekaauthor=K.+Freudenbergerauthor=C.+Lioliosauthor=G.+Gauglitzauthor=Z.+Courniaauthor=H.+Gohlkeauthor=A.+Kolocouris&title=Alchemical+free+energy+calculations+and+isothermal+titration+calorimetry+measurements+of+aminoadamantanes+bound+to+the+closed+state+of+influenza+A%2FM2TM&doi=10.1021%2Facs.jcim.6b00079"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit103R"><div class="casContent"><span class="casTitleNuber">103</span><div class="casTitle"><span class="NLM_cas:atitle">Alchemical Free Energy Calculations and Isothermal Titration Calorimetry Measurements of Aminoadamantanes Bound to the Closed State of Influenza A/M2TM</span></div><div class="casAuthors">Ioannidis, Harris; Drakopoulos, Antonios; Tzitzoglaki, Christina; Homeyer, Nadine; Kolarov, Felix; Gkeka, Paraskevi; Freudenberger, Kathrin; Liolios, Christos; Gauglitz, Gunter; Cournia, Zoe; Gohlke, Holger; Kolocouris, Antonios</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">862-876</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Adamantane derivs., such as amantadine and rimantadine, have been reported to block the transmembrane domain (TM) of the M2 protein of influenza A virus (A/M2) but their clin. use has been discontinued due to evolved resistance in humans.  Although expts. and simulations have provided adequate information about the binding interaction of amantadine or rimantadine to the M2 protein, methods for predicting binding affinities of whole series of M2 inhibitors have so far been scarcely applied.  Such methods could assist in the development of novel potent inhibitors that overcome A/M2 resistance.  Here we show that alchem. free energy calcns. of ligand binding using the Bennett acceptance ratio (BAR) method are valuable for detg. the relative binding potency of A/M2 inhibitors of the aminoadamantane type covering a binding affinity range of only ∼2 kcal mol-1.  Their binding affinities measured by isothermal titrn. calorimetry (ITC) against the A/M2TM tetramer from the Udorn strain in its closed form at pH 8 were used as exptl. probes.  The binding consts. of rimantadine enantiomers against M2TMUdorn were measured for the first time and found to be equal.  Two series of alchem. free energy calcns. were performed using 1,2-dipalmitoyl-sn-glycero-3-phosphocholine (DPPC) and 1,2-dimyristoyl-sn-glycero-3-phosphocholine (DMPC) lipids to mimic the membrane environment.  A fair correlation was found for DPPC that was significantly improved using DMPC, which resembles more closely the DPC lipids used in the ITC expts.  This demonstrates that binding free energy calcns. by the BAR approach can be used to predict relative binding affinities of aminoadamantane derivs. toward M2TM with good accuracy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYKIuWLX1aP7Vg90H21EOLACvtfcHk0lgZE7ACPwtxRA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XmsFOmtrg%253D&md5=c9ad995dcf016bc19c81fa4753827f5f</span></div><a href="/servlet/linkout?suffix=cit103&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.6b00079&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.6b00079%26sid%3Dliteratum%253Aachs%26aulast%3DIoannidis%26aufirst%3DH.%26aulast%3DDrakopoulos%26aufirst%3DA.%26aulast%3DTzitzoglaki%26aufirst%3DC.%26aulast%3DHomeyer%26aufirst%3DN.%26aulast%3DKolarov%26aufirst%3DF.%26aulast%3DGkeka%26aufirst%3DP.%26aulast%3DFreudenberger%26aufirst%3DK.%26aulast%3DLiolios%26aufirst%3DC.%26aulast%3DGauglitz%26aufirst%3DG.%26aulast%3DCournia%26aufirst%3DZ.%26aulast%3DGohlke%26aufirst%3DH.%26aulast%3DKolocouris%26aufirst%3DA.%26atitle%3DAlchemical%2520free%2520energy%2520calculations%2520and%2520isothermal%2520titration%2520calorimetry%2520measurements%2520of%2520aminoadamantanes%2520bound%2520to%2520the%2520closed%2520state%2520of%2520influenza%2520A%252FM2TM%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2016%26volume%3D56%26spage%3D862%26epage%3D876%26doi%3D10.1021%2Facs.jcim.6b00079" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref104"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref104'); return false;" data-citation="" class="refNumLink">104</a></strong><div class="NLM_citation" id="cit104"><span><span class="NLM_contrib-group">Bhati, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wright, D. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coveney, P. V.</span><span> </span><span class="NLM_article-title">Rapid, accurate, precise, and reliable relative free energy prediction using ensemble based thermodynamic integration</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">210</span><span class="NLM_x">–</span> <span class="NLM_lpage">222</span><span class="refDoi"> DOI: 10.1021/acs.jctc.6b00979</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jctc.6b00979" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref104/cit104&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC28XitVGntrrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&pages=210-222&author=A.+P.+Bhatiauthor=S.+Wanauthor=D.+W.+Wrightauthor=P.+V.+Coveney&title=Rapid%2C+accurate%2C+precise%2C+and+reliable+relative+free+energy+prediction+using+ensemble+based+thermodynamic+integration&doi=10.1021%2Facs.jctc.6b00979"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit104R"><div class="casContent"><span class="casTitleNuber">104</span><div class="casTitle"><span class="NLM_cas:atitle">Rapid, Accurate, Precise, and Reliable Relative Free Energy Prediction Using Ensemble Based Thermodynamic Integration</span></div><div class="casAuthors">Bhati, Agastya P.; Wan, Shunzhou; Wright, David W.; Coveney, Peter V.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">210-222</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The accurate prediction of the binding affinities of ligands to proteins is a major goal in drug discovery and personalized medicine.  The time taken to make such predictions is of similar importance to their accuracy, precision and reliability.  In the last few years, an ensemble based mol. dynamics approach has been proposed that provides a route to reliable predictions of free energies based on the mol. mechanics Poisson-Boltzmann surface area method which meets the requirements of accuracy, precision and reliability.  Here, we describe an equiv. methodol. based on thermodn. integration to substantially improve the accuracy, precision and reliability of calcd. relative binding free energies.  We report the performance of the method when applied to a diverse set of protein targets and ligands.  The results are in very good agreement with exptl. data (90% of calcns. agree to within 1 kcal/mol) while the method is reproducible by construction.  Statistical uncertainties of the order of 0.5 kcal/mol or less are achieved.  We present a systematic account of how the uncertainty in the predictions may be estd.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhu9wYHCJpUbVg90H21EOLACvtfcHk0liVPNKQdiUAuA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitVGntrrJ&md5=510b70188112a3578030e291ce19b127</span></div><a href="/servlet/linkout?suffix=cit104&amp;dbid=16384&amp;doi=10.1021%2Facs.jctc.6b00979&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jctc.6b00979%26sid%3Dliteratum%253Aachs%26aulast%3DBhati%26aufirst%3DA.%2BP.%26aulast%3DWan%26aufirst%3DS.%26aulast%3DWright%26aufirst%3DD.%2BW.%26aulast%3DCoveney%26aufirst%3DP.%2BV.%26atitle%3DRapid%252C%2520accurate%252C%2520precise%252C%2520and%2520reliable%2520relative%2520free%2520energy%2520prediction%2520using%2520ensemble%2520based%2520thermodynamic%2520integration%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2017%26volume%3D13%26spage%3D210%26epage%3D222%26doi%3D10.1021%2Facs.jctc.6b00979" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref105"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref105'); return false;" data-citation="" class="refNumLink">105</a></strong><div class="NLM_citation" id="cit105"><span><span class="NLM_contrib-group">Kiss, L. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soares-da-Silva, P.</span><span> </span><span class="NLM_article-title">Medicinal chemistry of catechol O-methyltransferase (COMT) inhibitors and their therapeutic utility</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">8692</span><span class="NLM_x">–</span> <span class="NLM_lpage">8717</span><span class="refDoi"> DOI: 10.1021/jm500572b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm500572b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref105/cit105&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht1Git7rI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=8692-8717&author=L.+E.+Kissauthor=P.+Soares-da-Silva&title=Medicinal+chemistry+of+catechol+O-methyltransferase+%28COMT%29+inhibitors+and+their+therapeutic+utility&doi=10.1021%2Fjm500572b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit105R"><div class="casContent"><span class="casTitleNuber">105</span><div class="casTitle"><span class="NLM_cas:atitle">Medicinal Chemistry of Catechol O-Methyltransferase (COMT) Inhibitors and Their Therapeutic Utility</span></div><div class="casAuthors">Kiss, Laszlo E.; Soares-da-Silva, Patricio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">8692-8717</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Catechol O-methyltransferase (COMT) is the enzyme responsible for the O-methylation of endogenous neurotransmitters and of xenobiotic substances and hormones incorporating catecholic structures.  COMT is a druggable biol. target for the treatment of various central and peripheral nervous system disorders, including Parkinson's disease, depression, schizophrenia, and other dopamine deficiency-related diseases.  The purpose of this perspective is fourfold: (i) to summarize the physiol. role of COMT inhibitors in central and peripheral nervous system disorders; (ii) to provide the history and perspective of the medicinal chem. behind the discovery and development of COMT inhibitors; (iii) to discuss how the physicochem. properties of recognized COMT inhibitors are understood to exert influence over their pharmacol. properties; and (iv) to evaluate the clin. benefits of the most relevant COMT inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_JOcVUbOM4LVg90H21EOLACvtfcHk0liVPNKQdiUAuA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht1Git7rI&md5=60ddd0ce0b752ddb1bd8898805a8935b</span></div><a href="/servlet/linkout?suffix=cit105&amp;dbid=16384&amp;doi=10.1021%2Fjm500572b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm500572b%26sid%3Dliteratum%253Aachs%26aulast%3DKiss%26aufirst%3DL.%2BE.%26aulast%3DSoares-da-Silva%26aufirst%3DP.%26atitle%3DMedicinal%2520chemistry%2520of%2520catechol%2520O-methyltransferase%2520%2528COMT%2529%2520inhibitors%2520and%2520their%2520therapeutic%2520utility%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D8692%26epage%3D8717%26doi%3D10.1021%2Fjm500572b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref106"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref106'); return false;" data-citation="" class="refNumLink">106</a></strong><div class="NLM_citation" id="cit106"><span><span class="NLM_contrib-group">Loeffler, H. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Michel, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woods, C.</span><span> </span><span class="NLM_article-title">FESetup: automating setup for alchemical free energy simulations</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">2485</span><span class="NLM_x">–</span> <span class="NLM_lpage">2490</span><span class="refDoi"> DOI: 10.1021/acs.jcim.5b00368</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.5b00368" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref106/cit106&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslyktb7O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2015&pages=2485-2490&author=H.+H.+Loefflerauthor=J.+Michelauthor=C.+Woods&title=FESetup%3A+automating+setup+for+alchemical+free+energy+simulations&doi=10.1021%2Facs.jcim.5b00368"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit106R"><div class="casContent"><span class="casTitleNuber">106</span><div class="casTitle"><span class="NLM_cas:atitle">FESetup: Automating Setup for Alchemical Free Energy Simulations</span></div><div class="casAuthors">Loeffler, Hannes H.; Michel, Julien; Woods, Christopher</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2485-2490</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">FESetup is a new pipeline tool which can be used flexibly within larger workflows.  The tool aims to support fast and easy setup of alchem. free energy simulations for mol. simulation packages such as AMBER, GROMACS, Sire, or NAMD.  Post-processing methods like MM-PBSA and LIE can be set up as well.  Ligands are automatically parametrized with AM1-BCC, and atom mappings for a single topol. description are computed with a max. common substructure search (MCSS) algorithm.  An abstr. mol. dynamics (MD) engine can be used for equilibration prior to free energy setup or standalone.  Currently, all modern AMBER force fields are supported.  Ease of use, robustness of the code, and automation where it is feasible are the main development goals.  The project follows an open development model, and we welcome contributions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpaHPd8afIINLVg90H21EOLACvtfcHk0liVPNKQdiUAuA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslyktb7O&md5=14de057bac2999778efed9b040b3368b</span></div><a href="/servlet/linkout?suffix=cit106&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.5b00368&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.5b00368%26sid%3Dliteratum%253Aachs%26aulast%3DLoeffler%26aufirst%3DH.%2BH.%26aulast%3DMichel%26aufirst%3DJ.%26aulast%3DWoods%26aufirst%3DC.%26atitle%3DFESetup%253A%2520automating%2520setup%2520for%2520alchemical%2520free%2520energy%2520simulations%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2015%26volume%3D55%26spage%3D2485%26epage%3D2490%26doi%3D10.1021%2Facs.jcim.5b00368" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref107"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref107'); return false;" data-citation="" class="refNumLink">107</a></strong><div class="NLM_citation" id="cit107"><span><span class="NLM_contrib-group">Liu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abel, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Redmann, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Summa, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jaber, V. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lim, N. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mobley, D. L.</span><span> </span><span class="NLM_article-title">Lead optimization mapper: automating free energy calculations for lead optimization</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">27</span><span class="NLM_x">, </span> <span class="NLM_fpage">755</span><span class="NLM_x">–</span> <span class="NLM_lpage">770</span><span class="refDoi"> DOI: 10.1007/s10822-013-9678-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1007%2Fs10822-013-9678-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=24072356" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsFaqtLbP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2013&pages=755-770&author=S.+Liuauthor=Y.+Wuauthor=T.+Linauthor=R.+Abelauthor=J.+P.+Redmannauthor=C.+M.+Summaauthor=V.+R.+Jaberauthor=N.+M.+Limauthor=D.+L.+Mobley&title=Lead+optimization+mapper%3A+automating+free+energy+calculations+for+lead+optimization&doi=10.1007%2Fs10822-013-9678-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit107R"><div class="casContent"><span class="casTitleNuber">107</span><div class="casTitle"><span class="NLM_cas:atitle">Lead optimization mapper: automating free energy calculations for lead optimization</span></div><div class="casAuthors">Liu, Shuai; Wu, Yujie; Lin, Teng; Abel, Robert; Redmann, Jonathan P.; Summa, Christopher M.; Jaber, Vivian R.; Lim, Nathan M.; Mobley, David L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">755-770</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Alchem. free energy calcns. hold increasing promise as an aid to drug discovery efforts.  However, applications of these techniques in discovery projects have been relatively few, partly because of the difficulty of planning and setting up calcns.  Here, we introduce lead optimization mapper, LOMAP, an automated algorithm to plan efficient relative free energy calcns. between potential ligands within a substantial library of perhaps hundreds of compds.  In this approach, ligands are first grouped by structural similarity primarily based on the size of a (loosely defined) maximal common substructure, and then calcns. are planned within and between sets of structurally related compds.  An emphasis is placed on ensuring that relative free energies can be obtained between any pair of compds. without combining the results of too many different relative free energy calcns. (to avoid accumulation of error) and by providing some redundancy to allow for the possibility of error and consistency checking and provide some insight into when results can be expected to be unreliable.  The algorithm is discussed in detail and a Python implementation, based on both Schroedinger's and OpenEye's APIs, has been made available freely under the BSD license.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGruuGmSu4sZV7Vg90H21EOLACvtfcHk0liz-pBa1YqD5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsFaqtLbP&md5=ab9ba58e2572aa3564a52a04352f70d7</span></div><a href="/servlet/linkout?suffix=cit107&amp;dbid=16384&amp;doi=10.1007%2Fs10822-013-9678-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-013-9678-y%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DLin%26aufirst%3DT.%26aulast%3DAbel%26aufirst%3DR.%26aulast%3DRedmann%26aufirst%3DJ.%2BP.%26aulast%3DSumma%26aufirst%3DC.%2BM.%26aulast%3DJaber%26aufirst%3DV.%2BR.%26aulast%3DLim%26aufirst%3DN.%2BM.%26aulast%3DMobley%26aufirst%3DD.%2BL.%26atitle%3DLead%2520optimization%2520mapper%253A%2520automating%2520free%2520energy%2520calculations%2520for%2520lead%2520optimization%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2013%26volume%3D27%26spage%3D755%26epage%3D770%26doi%3D10.1007%2Fs10822-013-9678-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref108"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref108'); return false;" data-citation="" class="refNumLink">108</a></strong><div class="NLM_citation" id="cit108"><span><span class="NLM_contrib-group">Filippakopoulos, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span> </span><span class="NLM_article-title">Targeting bromodomains: epigenetic readers of lysine acetylation</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">337</span><span class="NLM_x">–</span> <span class="NLM_lpage">356</span><span class="refDoi"> DOI: 10.1038/nrd4286</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref108/cit108&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1038%2Fnrd4286" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref108/cit108&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=24751816" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref108/cit108&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmsFalsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=337-356&author=P.+Filippakopoulosauthor=S.+Knapp&title=Targeting+bromodomains%3A+epigenetic+readers+of+lysine+acetylation&doi=10.1038%2Fnrd4286"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit108R"><div class="casContent"><span class="casTitleNuber">108</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting bromodomains: epigenetic readers of lysine acetylation</span></div><div class="casAuthors">Filippakopoulos, Panagis; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">337-356</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Lysine acetylation is a key mechanism that regulates chromatin structure; aberrant acetylation levels have been linked to the development of several diseases.  Acetyl-lysine modifications create docking sites for bromodomains, which are small interaction modules found on diverse proteins, some of which have a key role in the acetylation-dependent assembly of transcriptional regulator complexes.  These complexes can then initiate transcriptional programs that result in phenotypic changes.  The recent discovery of potent and highly specific inhibitors for the BET (bromodomain and extra-terminal) family of bromodomains has stimulated intensive research activity in diverse therapeutic areas, particularly in oncol., where BET proteins regulate the expression of key oncogenes and anti-apoptotic proteins.  In addn., targeting BET bromodomains could hold potential for the treatment of inflammation and viral infection.  Here, we highlight recent progress in the development of bromodomain inhibitors, and their potential applications in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOSIcwBJKj4LVg90H21EOLACvtfcHk0liz-pBa1YqD5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmsFalsr4%253D&md5=77049c82ce6a19b4685e3b7950399a07</span></div><a href="/servlet/linkout?suffix=cit108&amp;dbid=16384&amp;doi=10.1038%2Fnrd4286&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4286%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DTargeting%2520bromodomains%253A%2520epigenetic%2520readers%2520of%2520lysine%2520acetylation%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2014%26volume%3D13%26spage%3D337%26epage%3D356%26doi%3D10.1038%2Fnrd4286" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref109"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref109'); return false;" data-citation="" class="refNumLink">109</a></strong><div class="NLM_citation" id="cit109"><span><span class="NLM_contrib-group">Christ, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fox, T.</span><span> </span><span class="NLM_article-title">Accuracy assessment and automation of free energy calculations for drug design</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">108</span><span class="NLM_x">–</span> <span class="NLM_lpage">120</span><span class="refDoi"> DOI: 10.1021/ci4004199</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci4004199" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref109/cit109&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvVSnur7M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2014&pages=108-120&author=C.+D.+Christauthor=T.+Fox&title=Accuracy+assessment+and+automation+of+free+energy+calculations+for+drug+design&doi=10.1021%2Fci4004199"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit109R"><div class="casContent"><span class="casTitleNuber">109</span><div class="casTitle"><span class="NLM_cas:atitle">Accuracy Assessment and Automation of Free Energy Calculations for Drug Design</span></div><div class="casAuthors">Christ, Clara D.; Fox, Thomas</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">108-120</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">As the free energy of binding of a ligand to its target is one of the crucial optimization parameters in drug design, its accurate prediction is highly desirable.  In the present study we have assessed the av. accuracy of free energy calcns. for a total of 92 ligands binding to five different targets.  To make this study and future larger scale applications possible we automated the setup procedure.  Starting from user defined binding modes, the procedure decides which ligands to connect via a perturbation based on max. common substructure criteria and produces all necessary parameter files for free energy calcns. in AMBER 11.  For the systems investigated, errors due to insufficient sampling were found to be substantial in some cases whereas differences in estimators (thermodn. integration (TI) vs. multistate Bennett acceptance ratio (MBAR)) were found to be negligible.  Anal. uncertainty ests. calcd. from a single free energy calcn. were found to be much smaller than the sample std. deviation obtained from two independent free energy calcns.  Agreement with expt. was found to be system dependent ranging from excellent to mediocre (RMSE = [0.9, 8.2, 4.7, 5.7, 8.7] kJ/mol).  When restricting analyses to free energy calcns. with sample std. deviations below 1 kJ/mol agreement with expt. improved (RMSE = [0.8, 6.9, 1.8, 3.9, 5.6] kJ/mol).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGri8r-6now_RrVg90H21EOLACvtfcHk0liz-pBa1YqD5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvVSnur7M&md5=e36c5ee86688acfa1bce48f9a413ddc8</span></div><a href="/servlet/linkout?suffix=cit109&amp;dbid=16384&amp;doi=10.1021%2Fci4004199&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci4004199%26sid%3Dliteratum%253Aachs%26aulast%3DChrist%26aufirst%3DC.%2BD.%26aulast%3DFox%26aufirst%3DT.%26atitle%3DAccuracy%2520assessment%2520and%2520automation%2520of%2520free%2520energy%2520calculations%2520for%2520drug%2520design%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2014%26volume%3D54%26spage%3D108%26epage%3D120%26doi%3D10.1021%2Fci4004199" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref110"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref110'); return false;" data-citation="" class="refNumLink">110</a></strong><div class="NLM_citation" id="cit110"><span><span class="NLM_contrib-group">Kollman, P.</span><span> </span><span class="NLM_article-title">Free energy calculations: applications to chemical and biochemical phenomena</span> <span class="citation_source-journal">Chem. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1993</span><span class="NLM_x">, </span> <span class="NLM_volume">93</span><span class="NLM_x">, </span> <span class="NLM_fpage">2395</span><span class="NLM_x">–</span> <span class="NLM_lpage">2417</span><span class="refDoi"> DOI: 10.1021/cr00023a004</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr00023a004" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref110/cit110&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADyaK3sXmt1Sktr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=93&publication_year=1993&pages=2395-2417&author=P.+Kollman&title=Free+energy+calculations%3A+applications+to+chemical+and+biochemical+phenomena&doi=10.1021%2Fcr00023a004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit110R"><div class="casContent"><span class="casTitleNuber">110</span><div class="casTitle"><span class="NLM_cas:atitle">Free energy calculations: Applications to chemical and biochemical phenomena</span></div><div class="casAuthors">Kollman, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">93</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2395-417</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    </div><div class="casAbstract">A review with 252 refs. about applications of free energy calcns. employing mol. dynamics or Monte Carlo methods to a variety of chem. and biochem. phenomena.  The focus is on applications of such calcns. to mol. solvation, mol. assocn., macromol. stability, and enzyme catalysis.  The mols. discussed range from monovalent ions and small mols. to proteins and nucleic acids.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrN1w4SyFDqCLVg90H21EOLACvtfcHk0lgYYe6-jUsHlQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXmt1Sktr0%253D&md5=f12326b24734ea092996cf22efb6ebd6</span></div><a href="/servlet/linkout?suffix=cit110&amp;dbid=16384&amp;doi=10.1021%2Fcr00023a004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr00023a004%26sid%3Dliteratum%253Aachs%26aulast%3DKollman%26aufirst%3DP.%26atitle%3DFree%2520energy%2520calculations%253A%2520applications%2520to%2520chemical%2520and%2520biochemical%2520phenomena%26jtitle%3DChem.%2520Rev.%26date%3D1993%26volume%3D93%26spage%3D2395%26epage%3D2417%26doi%3D10.1021%2Fcr00023a004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62'],'ref63':['cit63'],'ref64':['cit64'],'ref65':['cit65'],'ref66':['cit66'],'ref67':['cit67'],'ref68':['cit68'],'ref69':['cit69'],'ref70':['cit70'],'ref71':['cit71'],'ref72':['cit72'],'ref73':['cit73'],'ref74':['cit74'],'ref75':['cit75'],'ref76':['cit76'],'ref77':['cit77'],'ref78':['cit78'],'ref79':['cit79'],'ref80':['cit80'],'ref81':['cit81'],'ref82':['cit82'],'ref83':['cit83'],'ref84':['cit84'],'ref85':['cit85'],'ref86':['cit86'],'ref87':['cit87'],'ref88':['cit88'],'ref89':['cit89'],'ref90':['cit90'],'ref91':['cit91'],'ref92':['cit92'],'ref93':['cit93'],'ref94':['cit94'],'ref95':['cit95'],'ref96':['cit96'],'ref97':['cit97'],'ref98':['cit98'],'ref99':['cit99'],'ref100':['cit100'],'ref101':['cit101'],'ref102':['cit102'],'ref103':['cit103'],'ref104':['cit104'],'ref105':['cit105'],'ref106':['cit106'],'ref107':['cit107'],'ref108':['cit108'],'ref109':['cit109'],'ref110':['cit110']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 62 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Stefano Sainas, Marta Giorgis, Paola Circosta, Valentina Gaidano, Davide Bonanni, Agnese C. Pippione, Renzo Bagnati, Alice Passoni, Yaqi Qiu, Carina Florina Cojocaru, Barbara Canepa, Alessandro Bona, Barbara Rolando, Mariia Mishina, Cristina Ramondetti, Barbara Buccinnà, Marco Piccinini, Mohammad Houshmand, Alessandro Cignetti, Enrico Giraudo, Salam Al-Karadaghi, Donatella Boschi, Giuseppe Saglio, <span class="NLM_string-name hlFld-ContribAuthor">Marco L. Lolli</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting Acute Myelogenous Leukemia Using Potent Human Dihydroorotate Dehydrogenase Inhibitors Based on the 2-Hydroxypyrazolo[1,5-a]pyridine Scaffold: SAR of the Biphenyl Moiety. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (9)
                                     , 5404-5428. <a href="https://doi.org/10.1021/acs.jmedchem.0c01549" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01549</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01549&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c01549%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DTargeting%252BAcute%252BMyelogenous%252BLeukemia%252BUsing%252BPotent%252BHuman%252BDihydroorotate%252BDehydrogenase%252BInhibitors%252BBased%252Bon%252Bthe%252B2-Hydroxypyrazolo%25255B1%25252C5-a%25255Dpyridine%252BScaffold%25253A%252BSAR%252Bof%252Bthe%252BBiphenyl%252BMoiety%26aulast%3DSainas%26aufirst%3DStefano%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D04092020%26date%3D12042021%26volume%3D64%26issue%3D9%26spage%3D5404%26epage%3D5428" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Hannah M. Baumann, Vytautas Gapsys, Bert L. de Groot, <span class="NLM_string-name hlFld-ContribAuthor">David L. Mobley</span>. </span><span class="cited-content_cbyCitation_article-title">Challenges Encountered Applying Equilibrium and Nonequilibrium Binding Free Energy Calculations. </span><span class="cited-content_cbyCitation_journal-name">The Journal of Physical Chemistry B</span><span> <strong>2021,</strong> <em>125 </em>
                                    (17)
                                     , 4241-4261. <a href="https://doi.org/10.1021/acs.jpcb.0c10263" title="DOI URL">https://doi.org/10.1021/acs.jpcb.0c10263</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jpcb.0c10263&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jpcb.0c10263%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Journal%2520of%2520Physical%2520Chemistry%2520B%26atitle%3DChallenges%252BEncountered%252BApplying%252BEquilibrium%252Band%252BNonequilibrium%252BBinding%252BFree%252BEnergy%252BCalculations%26aulast%3DBaumann%26aufirst%3DHannah%2BM.%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D2021%26date%3D13112020%26date%3D07042021%26date%3D27042021%26volume%3D125%26issue%3D17%26spage%3D4241%26epage%3D4261" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Billy J. Williams-Noonan, Nevena Todorova, Ketav Kulkarni, Marie-Isabel Aguilar, <span class="NLM_string-name hlFld-ContribAuthor">Irene Yarovsky</span>. </span><span class="cited-content_cbyCitation_article-title">An Active Site Inhibitor Induces Conformational Penalties for ACE2 Recognition by the Spike Protein of SARS-CoV-2. </span><span class="cited-content_cbyCitation_journal-name">The Journal of Physical Chemistry B</span><span> <strong>2021,</strong> <em>125 </em>
                                    (10)
                                     , 2533-2550. <a href="https://doi.org/10.1021/acs.jpcb.0c11321" title="DOI URL">https://doi.org/10.1021/acs.jpcb.0c11321</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jpcb.0c11321&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jpcb.0c11321%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Journal%2520of%2520Physical%2520Chemistry%2520B%26atitle%3DAn%252BActive%252BSite%252BInhibitor%252BInduces%252BConformational%252BPenalties%252Bfor%252BACE2%252BRecognition%252Bby%252Bthe%252BSpike%252BProtein%252Bof%252BSARS-CoV-2%26aulast%3DWilliams-Noonan%26aufirst%3DBilly%2BJ.%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D2021%26date%3D21122020%26date%3D28012021%26date%3D03032021%26volume%3D125%26issue%3D10%26spage%3D2533%26epage%3D2550" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Steven Blake, Isabel Hemming, Julian Ik-Tsen Heng, <span class="NLM_string-name hlFld-ContribAuthor">Mark Agostino</span>. </span><span class="cited-content_cbyCitation_article-title">Structure-Based Approaches to Classify the Functional Impact of ZBTB18 Missense Variants in Health and Disease. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Neuroscience</span><span> <strong>2021,</strong> <em>12 </em>
                                    (6)
                                     , 979-989. <a href="https://doi.org/10.1021/acschemneuro.0c00758" title="DOI URL">https://doi.org/10.1021/acschemneuro.0c00758</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acschemneuro.0c00758&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facschemneuro.0c00758%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Neuroscience%26atitle%3DStructure-Based%252BApproaches%252Bto%252BClassify%252Bthe%252BFunctional%252BImpact%252Bof%252BZBTB18%252BMissense%252BVariants%252Bin%252BHealth%252Band%252BDisease%26aulast%3DBlake%26aufirst%3DSteven%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D24112020%26date%3D12022021%26date%3D23022021%26volume%3D12%26issue%3D6%26spage%3D979%26epage%3D989" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Miroslav Suruzhon, Michael S. Bodnarchuk, Antonella Ciancetta, Russell Viner, Ian D. Wall, <span class="NLM_string-name hlFld-ContribAuthor">Jonathan W. Essex</span>. </span><span class="cited-content_cbyCitation_article-title">Sensitivity of Binding Free Energy Calculations to Initial Protein Crystal Structure. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Theory and Computation</span><span> <strong>2021,</strong> <em>17 </em>
                                    (3)
                                     , 1806-1821. <a href="https://doi.org/10.1021/acs.jctc.0c00972" title="DOI URL">https://doi.org/10.1021/acs.jctc.0c00972</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jctc.0c00972&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jctc.0c00972%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Theory%2520and%2520Computation%26atitle%3DSensitivity%252Bof%252BBinding%252BFree%252BEnergy%252BCalculations%252Bto%252BInitial%252BProtein%252BCrystal%252BStructure%26aulast%3DSuruzhon%26aufirst%3DMiroslav%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D20092020%26date%3D03022021%26volume%3D17%26issue%3D3%26spage%3D1806%26epage%3D1821" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Mateusz K. Bieniek, Agastya P. Bhati, Shunzhou Wan, <span class="NLM_string-name hlFld-ContribAuthor">Peter V. Coveney</span>. </span><span class="cited-content_cbyCitation_article-title">TIES 20: Relative Binding Free Energy with a Flexible Superimposition Algorithm and Partial Ring Morphing. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Theory and Computation</span><span> <strong>2021,</strong> <em>17 </em>
                                    (2)
                                     , 1250-1265. <a href="https://doi.org/10.1021/acs.jctc.0c01179" title="DOI URL">https://doi.org/10.1021/acs.jctc.0c01179</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jctc.0c01179&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jctc.0c01179%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Theory%2520and%2520Computation%26atitle%3DTIES%252B20%25253A%252BRelative%252BBinding%252BFree%252BEnergy%252Bwith%252Ba%252BFlexible%252BSuperimposition%252BAlgorithm%252Band%252BPartial%252BRing%252BMorphing%26aulast%3DBieniek%26aufirst%3DMateusz%2BK.%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D10112020%26date%3D25012021%26volume%3D17%26issue%3D2%26spage%3D1250%26epage%3D1265" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Ksenia Korshunova, <span class="NLM_string-name hlFld-ContribAuthor">Paolo Carloni</span>. </span><span class="cited-content_cbyCitation_article-title">Ligand Affinities within the Open-Boundary Molecular Mechanics/Coarse-Grained Framework (I): Alchemical Transformations within the Hamiltonian Adaptive Resolution Scheme. </span><span class="cited-content_cbyCitation_journal-name">The Journal of Physical Chemistry B</span><span> <strong>2021,</strong> <em>125 </em>
                                    (3)
                                     , 789-797. <a href="https://doi.org/10.1021/acs.jpcb.0c09805" title="DOI URL">https://doi.org/10.1021/acs.jpcb.0c09805</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jpcb.0c09805&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jpcb.0c09805%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Journal%2520of%2520Physical%2520Chemistry%2520B%26atitle%3DLigand%252BAffinities%252Bwithin%252Bthe%252BOpen-Boundary%252BMolecular%252BMechanics%25252FCoarse-Grained%252BFramework%252B%252528I%252529%25253A%252BAlchemical%252BTransformations%252Bwithin%252Bthe%252BHamiltonian%252BAdaptive%252BResolution%252BScheme%26aulast%3DKorshunova%26aufirst%3DKsenia%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D2020%26date%3D30102020%26date%3D27122020%26date%3D14012021%26volume%3D125%26issue%3D3%26spage%3D789%26epage%3D797" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Fabian Bumbak, Trayder Thomas, Billy J. Noonan-Williams, Tasneem M. Vaid, Fei Yan, Alice R. Whitehead, Shoni Bruell, Martina Kocan, Xuan Tan, Margaret A. Johnson, Ross A. D. Bathgate, David K. Chalmers, Paul R. Gooley, <span class="NLM_string-name hlFld-ContribAuthor">Daniel J. Scott</span>. </span><span class="cited-content_cbyCitation_article-title">Conformational Changes in Tyrosine 11 of Neurotensin Are Required to Activate the Neurotensin Receptor 1. </span><span class="cited-content_cbyCitation_journal-name">ACS Pharmacology & Translational Science</span><span> <strong>2020,</strong> <em>3 </em>
                                    (4)
                                     , 690-705. <a href="https://doi.org/10.1021/acsptsci.0c00026" title="DOI URL">https://doi.org/10.1021/acsptsci.0c00026</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsptsci.0c00026&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsptsci.0c00026%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Pharmacology%2520%2526%2520Translational%2520Science%26atitle%3DConformational%252BChanges%252Bin%252BTyrosine%252B11%252Bof%252BNeurotensin%252BAre%252BRequired%252Bto%252BActivate%252Bthe%252BNeurotensin%252BReceptor%252B1%26aulast%3DBumbak%26aufirst%3DFabian%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D03032020%26date%3D21052020%26date%3D29042020%26volume%3D3%26issue%3D4%26spage%3D690%26epage%3D705" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Braden D. Kelly, <span class="NLM_string-name hlFld-ContribAuthor">William R. Smith</span>. </span><span class="cited-content_cbyCitation_article-title">A Simple Method for Including Polarization Effects in Solvation Free Energy Calculations When Using Fixed-Charge Force Fields: Alchemically Polarized Charges. </span><span class="cited-content_cbyCitation_journal-name">ACS Omega</span><span> <strong>2020,</strong> <em>5 </em>
                                    (28)
                                     , 17170-17181. <a href="https://doi.org/10.1021/acsomega.0c01148" title="DOI URL">https://doi.org/10.1021/acsomega.0c01148</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsomega.0c01148&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsomega.0c01148%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Omega%26atitle%3DA%252BSimple%252BMethod%252Bfor%252BIncluding%252BPolarization%252BEffects%252Bin%252BSolvation%252BFree%252BEnergy%252BCalculations%252BWhen%252BUsing%252BFixed-Charge%252BForce%252BFields%25253A%252BAlchemically%252BPolarized%252BCharges%26aulast%3DKelly%26aufirst%3DBraden%2BD.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D16032020%26date%3D11062020%26date%3D07072020%26volume%3D5%26issue%3D28%26spage%3D17170%26epage%3D17181" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Martin Reinhardt, <span class="NLM_string-name hlFld-ContribAuthor">Helmut Grubmüller</span>. </span><span class="cited-content_cbyCitation_article-title">Determining Free-Energy Differences Through Variationally Derived Intermediates. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Theory and Computation</span><span> <strong>2020,</strong> <em>16 </em>
                                    (6)
                                     , 3504-3512. <a href="https://doi.org/10.1021/acs.jctc.0c00106" title="DOI URL">https://doi.org/10.1021/acs.jctc.0c00106</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jctc.0c00106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jctc.0c00106%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Theory%2520and%2520Computation%26atitle%3DDetermining%252BFree-Energy%252BDifferences%252BThrough%252BVariationally%252BDerived%252BIntermediates%26aulast%3DReinhardt%26aufirst%3DMartin%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D31012020%26date%3D28052020%26date%3D11052020%26volume%3D16%26issue%3D6%26spage%3D3504%26epage%3D3512" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Daniel Cappel, Steven Jerome, Gerhard Hessler, <span class="NLM_string-name hlFld-ContribAuthor">Hans Matter</span>. </span><span class="cited-content_cbyCitation_article-title">Impact of Different Automated Binding Pose Generation Approaches on Relative Binding Free Energy Simulations. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (3)
                                     , 1432-1444. <a href="https://doi.org/10.1021/acs.jcim.9b01118" title="DOI URL">https://doi.org/10.1021/acs.jcim.9b01118</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.9b01118&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.9b01118%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DImpact%252Bof%252BDifferent%252BAutomated%252BBinding%252BPose%252BGeneration%252BApproaches%252Bon%252BRelative%252BBinding%252BFree%252BEnergy%252BSimulations%26aulast%3DCappel%26aufirst%3DDaniel%26date%3D2020%26date%3D2020%26date%3D2019%26date%3D03122019%26date%3D04022020%26date%3D28012020%26volume%3D60%26issue%3D3%26spage%3D1432%26epage%3D1444" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Sukanya Sasmal, Samuel C. Gill, Nathan M. Lim, <span class="NLM_string-name hlFld-ContribAuthor">David L. Mobley</span>. </span><span class="cited-content_cbyCitation_article-title">Sampling Conformational Changes of Bound Ligands Using Nonequilibrium Candidate Monte Carlo and Molecular Dynamics. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Theory and Computation</span><span> <strong>2020,</strong> <em>16 </em>
                                    (3)
                                     , 1854-1865. <a href="https://doi.org/10.1021/acs.jctc.9b01066" title="DOI URL">https://doi.org/10.1021/acs.jctc.9b01066</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jctc.9b01066&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jctc.9b01066%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Theory%2520and%2520Computation%26atitle%3DSampling%252BConformational%252BChanges%252Bof%252BBound%252BLigands%252BUsing%252BNonequilibrium%252BCandidate%252BMonte%252BCarlo%252Band%252BMolecular%252BDynamics%26aulast%3DSasmal%26aufirst%3DSukanya%26date%3D2020%26date%3D2020%26date%3D2019%26date%3D25102019%26date%3D24022020%26date%3D14022020%26volume%3D16%26issue%3D3%26spage%3D1854%26epage%3D1865" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Clauber H. S. da Costa, Vinícius Bonatto, Alberto M. dos Santos, Jerônimo Lameira, Andrei Leitão, <span class="NLM_string-name hlFld-ContribAuthor">Carlos A. Montanari</span>. </span><span class="cited-content_cbyCitation_article-title">Evaluating QM/MM Free Energy Surfaces for Ranking Cysteine Protease Covalent Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (2)
                                     , 880-889. <a href="https://doi.org/10.1021/acs.jcim.9b00847" title="DOI URL">https://doi.org/10.1021/acs.jcim.9b00847</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.9b00847&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.9b00847%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DEvaluating%252BQM%25252FMM%252BFree%252BEnergy%252BSurfaces%252Bfor%252BRanking%252BCysteine%252BProtease%252BCovalent%252BInhibitors%26aulast%3Dda%2BCosta%26aufirst%3DClauber%2BH.%2BS.%26date%3D2020%26date%3D2020%26date%3D2019%26date%3D25092019%26date%3D27012020%26date%3D16012020%26volume%3D60%26issue%3D2%26spage%3D880%26epage%3D889" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Alexander
D. Wade, <span class="NLM_string-name hlFld-ContribAuthor">David J. Huggins</span>. </span><span class="cited-content_cbyCitation_article-title">Optimization of Protein–Ligand Electrostatic Interactions Using an Alchemical Free-Energy Method. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Theory and Computation</span><span> <strong>2019,</strong> <em>15 </em>
                                    (11)
                                     , 6504-6512. <a href="https://doi.org/10.1021/acs.jctc.9b00976" title="DOI URL">https://doi.org/10.1021/acs.jctc.9b00976</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jctc.9b00976&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jctc.9b00976%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Theory%2520and%2520Computation%26atitle%3DOptimization%252Bof%252BProtein%2525E2%252580%252593Ligand%252BElectrostatic%252BInteractions%252BUsing%252Ban%252BAlchemical%252BFree-Energy%252BMethod%26aulast%3DWade%26aufirst%3DAlexander%2BD.%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D30092019%26date%3D23102019%26date%3D04102019%26volume%3D15%26issue%3D11%26spage%3D6504%26epage%3D6512" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Jonah
Z. Vilseck, Noor Sohail, Ryan L. Hayes, <span class="NLM_string-name hlFld-ContribAuthor">Charles L. Brooks III</span>. </span><span class="cited-content_cbyCitation_article-title">Overcoming Challenging Substituent Perturbations with Multisite λ-Dynamics: A Case Study Targeting β-Secretase 1. </span><span class="cited-content_cbyCitation_journal-name">The Journal of Physical Chemistry Letters</span><span> <strong>2019,</strong> <em>10 </em>
                                    (17)
                                     , 4875-4880. <a href="https://doi.org/10.1021/acs.jpclett.9b02004" title="DOI URL">https://doi.org/10.1021/acs.jpclett.9b02004</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jpclett.9b02004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jpclett.9b02004%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Journal%2520of%2520Physical%2520Chemistry%2520Letters%26atitle%3DOvercoming%252BChallenging%252BSubstituent%252BPerturbations%252Bwith%252BMultisite%252B%2525CE%2525BB-Dynamics%25253A%252BA%252BCase%252BStudy%252BTargeting%252B%2525CE%2525B2-Secretase%252B1%26aulast%3DVilseck%26aufirst%3DJonah%2BZ.%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D10072019%26date%3D06082019%26date%3D13082019%26date%3D06082019%26volume%3D10%26issue%3D17%26spage%3D4875%26epage%3D4880" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Ercheng Wang, Huiyong Sun, Junmei Wang, Zhe Wang, Hui Liu, John Z. H. Zhang, <span class="NLM_string-name hlFld-ContribAuthor">Tingjun Hou</span>. </span><span class="cited-content_cbyCitation_article-title">End-Point Binding Free Energy Calculation with MM/PBSA and MM/GBSA: Strategies and Applications in Drug Design. </span><span class="cited-content_cbyCitation_journal-name">Chemical Reviews</span><span> <strong>2019,</strong> <em>119 </em>
                                    (16)
                                     , 9478-9508. <a href="https://doi.org/10.1021/acs.chemrev.9b00055" title="DOI URL">https://doi.org/10.1021/acs.chemrev.9b00055</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.chemrev.9b00055&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.chemrev.9b00055%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Reviews%26atitle%3DEnd-Point%252BBinding%252BFree%252BEnergy%252BCalculation%252Bwith%252BMM%25252FPBSA%252Band%252BMM%25252FGBSA%25253A%252BStrategies%252Band%252BApplications%252Bin%252BDrug%252BDesign%26aulast%3DWang%26aufirst%3DErcheng%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D21012019%26date%3D24062019%26volume%3D119%26issue%3D16%26spage%3D9478%26epage%3D9508" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Emelie Flood, Céline Boiteux, Bogdan Lev, Igor Vorobyov, <span class="NLM_string-name hlFld-ContribAuthor">Toby W. Allen</span>. </span><span class="cited-content_cbyCitation_article-title">Atomistic Simulations of Membrane Ion Channel Conduction, Gating, and Modulation. </span><span class="cited-content_cbyCitation_journal-name">Chemical Reviews</span><span> <strong>2019,</strong> <em>119 </em>
                                    (13)
                                     , 7737-7832. <a href="https://doi.org/10.1021/acs.chemrev.8b00630" title="DOI URL">https://doi.org/10.1021/acs.chemrev.8b00630</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.chemrev.8b00630&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.chemrev.8b00630%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Reviews%26atitle%3DAtomistic%252BSimulations%252Bof%252BMembrane%252BIon%252BChannel%252BConduction%25252C%252BGating%25252C%252Band%252BModulation%26aulast%3DFlood%26aufirst%3DEmelie%26date%3D2019%26date%3D2019%26date%3D2018%26date%3D18102018%26date%3D27062019%26volume%3D119%26issue%3D13%26spage%3D7737%26epage%3D7832" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Vincent
D. Ustach, Sirish Kaushik Lakkaraju, Sunhwan Jo, Wenbo Yu, Wenjuan Jiang, <span class="NLM_string-name hlFld-ContribAuthor">Alexander D. MacKerell, Jr.</span>. </span><span class="cited-content_cbyCitation_article-title">Optimization and Evaluation of Site-Identification by Ligand Competitive Saturation (SILCS) as a Tool for Target-Based Ligand Optimization. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2019,</strong> <em>59 </em>
                                    (6)
                                     , 3018-3035. <a href="https://doi.org/10.1021/acs.jcim.9b00210" title="DOI URL">https://doi.org/10.1021/acs.jcim.9b00210</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.9b00210&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.9b00210%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DOptimization%252Band%252BEvaluation%252Bof%252BSite-Identification%252Bby%252BLigand%252BCompetitive%252BSaturation%252B%252528SILCS%252529%252Bas%252Ba%252BTool%252Bfor%252BTarget-Based%252BLigand%252BOptimization%26aulast%3DUstach%26aufirst%3DVincent%2BD.%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D12032019%26date%3D08052019%26date%3D29042019%26volume%3D59%26issue%3D6%26spage%3D3018%26epage%3D3035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Alexander
D. Wade, Andrea Rizzi, Yuanqing Wang, <span class="NLM_string-name hlFld-ContribAuthor">David J. Huggins</span>. </span><span class="cited-content_cbyCitation_article-title">Computational Fluorine Scanning Using Free-Energy Perturbation. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2019,</strong> <em>59 </em>
                                    (6)
                                     , 2776-2784. <a href="https://doi.org/10.1021/acs.jcim.9b00228" title="DOI URL">https://doi.org/10.1021/acs.jcim.9b00228</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.9b00228&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.9b00228%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DComputational%252BFluorine%252BScanning%252BUsing%252BFree-Energy%252BPerturbation%26aulast%3DWade%26aufirst%3DAlexander%2BD.%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D15032019%26date%3D09052019%26date%3D02052019%26volume%3D59%26issue%3D6%26spage%3D2776%26epage%3D2784" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Laura Pérez-Benito, Nil Casajuana-Martin, Mireia Jiménez-Rosés, Herman van Vlijmen, <span class="NLM_string-name hlFld-ContribAuthor">Gary Tresadern</span>. </span><span class="cited-content_cbyCitation_article-title">Predicting Activity Cliffs with Free-Energy Perturbation. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Theory and Computation</span><span> <strong>2019,</strong> <em>15 </em>
                                    (3)
                                     , 1884-1895. <a href="https://doi.org/10.1021/acs.jctc.8b01290" title="DOI URL">https://doi.org/10.1021/acs.jctc.8b01290</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jctc.8b01290&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jctc.8b01290%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Theory%2520and%2520Computation%26atitle%3DPredicting%252BActivity%252BCliffs%252Bwith%252BFree-Energy%252BPerturbation%26aulast%3DP%25C3%25A9rez-Benito%26aufirst%3DLaura%26date%3D2019%26date%3D2019%26date%3D2018%26date%3D21122018%26date%3D01032019%26date%3D18022019%26volume%3D15%26issue%3D3%26spage%3D1884%26epage%3D1895" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Toru Ekimoto, Tsutomu Yamane, <span class="NLM_string-name hlFld-ContribAuthor">Mitsunori Ikeguchi</span>. </span><span class="cited-content_cbyCitation_article-title">Elimination of Finite-Size Effects on Binding Free Energies via the Warp-Drive Method. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Theory and Computation</span><span> <strong>2018,</strong> <em>14 </em>
                                    (12)
                                     , 6544-6559. <a href="https://doi.org/10.1021/acs.jctc.8b00280" title="DOI URL">https://doi.org/10.1021/acs.jctc.8b00280</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jctc.8b00280&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jctc.8b00280%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Theory%2520and%2520Computation%26atitle%3DElimination%252Bof%252BFinite-Size%252BEffects%252Bon%252BBinding%252BFree%252BEnergies%252Bvia%252Bthe%252BWarp-Drive%252BMethod%26aulast%3DEkimoto%26aufirst%3DToru%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D20032018%26date%3D21112018%26date%3D07112018%26volume%3D14%26issue%3D12%26spage%3D6544%26epage%3D6559" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Tien-Yi Hou, Ching-Feng Weng, <span class="NLM_string-name hlFld-ContribAuthor">Max K. Leong</span>. </span><span class="cited-content_cbyCitation_article-title">Insight Analysis of Promiscuous Estrogen Receptor α-Ligand Binding by a Novel Machine Learning Scheme. </span><span class="cited-content_cbyCitation_journal-name">Chemical Research in Toxicology</span><span> <strong>2018,</strong> <em>31 </em>
                                    (8)
                                     , 799-813. <a href="https://doi.org/10.1021/acs.chemrestox.8b00130" title="DOI URL">https://doi.org/10.1021/acs.chemrestox.8b00130</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.chemrestox.8b00130&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.chemrestox.8b00130%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Research%2520in%2520Toxicology%26atitle%3DInsight%252BAnalysis%252Bof%252BPromiscuous%252BEstrogen%252BReceptor%252B%2525CE%2525B1-Ligand%252BBinding%252Bby%252Ba%252BNovel%252BMachine%252BLearning%252BScheme%26aulast%3DHou%26aufirst%3DTien-Yi%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D20052018%26date%3D30072018%26date%3D18072018%26volume%3D31%26issue%3D8%26spage%3D799%26epage%3D813" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Stefano Sainas, Agnese C. Pippione, Elisa Lupino, Marta Giorgis, Paola Circosta, Valentina Gaidano, Parveen Goyal, Davide Bonanni, Barbara Rolando, Alessandro Cignetti, Alex Ducime, Mikael Andersson, Michael Järvå, Rosmarie Friemann, Marco Piccinini, Cristina Ramondetti, Barbara Buccinnà, Salam Al-Karadaghi, Donatella Boschi, Giuseppe Saglio, <span class="NLM_string-name hlFld-ContribAuthor">Marco L. Lolli</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting Myeloid Differentiation Using Potent 2-Hydroxypyrazolo[1,5-a]pyridine Scaffold-Based Human Dihydroorotate Dehydrogenase Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>61 </em>
                                    (14)
                                     , 6034-6055. <a href="https://doi.org/10.1021/acs.jmedchem.8b00373" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b00373</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b00373&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.8b00373%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DTargeting%252BMyeloid%252BDifferentiation%252BUsing%252BPotent%252B2-Hydroxypyrazolo%25255B1%25252C5-a%25255Dpyridine%252BScaffold-Based%252BHuman%252BDihydroorotate%252BDehydrogenase%252BInhibitors%26aulast%3DSainas%26aufirst%3DStefano%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D08032018%26date%3D09072018%26date%3D25062018%26volume%3D61%26issue%3D14%26spage%3D6034%26epage%3D6055" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zoe  Cournia</span>, <span class="hlFld-ContribAuthor ">Bryce  Allen</span>, and <span class="hlFld-ContribAuthor ">Woody  Sherman</span>  . </span><span class="cited-content_cbyCitation_article-title">Relative Binding Free Energy Calculations in Drug Discovery: Recent Advances and Practical Considerations. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2017,</strong> <em>57 </em>
                                    (12)
                                     , 2911-2937. <a href="https://doi.org/10.1021/acs.jcim.7b00564" title="DOI URL">https://doi.org/10.1021/acs.jcim.7b00564</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.7b00564&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.7b00564%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DRelative%252BBinding%252BFree%252BEnergy%252BCalculations%252Bin%252BDrug%252BDiscovery%25253A%252BRecent%252BAdvances%252Band%252BPractical%252BConsiderations%26aulast%3DCournia%26aufirst%3DZoe%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D19092017%26date%3D15122017%26date%3D26122017%26volume%3D57%26issue%3D12%26spage%3D2911%26epage%3D2937" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Leili  Zhang</span>, <span class="hlFld-ContribAuthor ">Giacomo  Domeniconi</span>, <span class="hlFld-ContribAuthor ">Chih-Chieh  Yang</span>, <span class="hlFld-ContribAuthor ">Seung-gu  Kang</span>, <span class="hlFld-ContribAuthor ">Ruhong  Zhou</span>, <span class="hlFld-ContribAuthor ">Guojing  Cong</span>. </span><span class="cited-content_cbyCitation_article-title">CASTELO: clustered atom subtypes aided lead optimization—a combined machine learning and molecular modeling method. </span><span class="cited-content_cbyCitation_journal-name">BMC Bioinformatics</span><span> <strong>2021,</strong> <em>22 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s12859-021-04214-4" title="DOI URL">https://doi.org/10.1186/s12859-021-04214-4</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s12859-021-04214-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs12859-021-04214-4%26sid%3Dliteratum%253Aachs%26jtitle%3DBMC%2520Bioinformatics%26atitle%3DCASTELO%25253A%252Bclustered%252Batom%252Bsubtypes%252Baided%252Blead%252Boptimization%2525E2%252580%252594a%252Bcombined%252Bmachine%252Blearning%252Band%252Bmolecular%252Bmodeling%252Bmethod%26aulast%3DZhang%26aufirst%3DLeili%26date%3D2021%26date%3D2021%26volume%3D22%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Chandra Bhushan  Mishra</span>, <span class="hlFld-ContribAuthor ">Preeti  Pandey</span>, <span class="hlFld-ContribAuthor ">Ravi Datta  Sharma</span>, <span class="hlFld-ContribAuthor ">Md Zubbair  Malik</span>, <span class="hlFld-ContribAuthor ">Raj Kumar  Mongre</span>, <span class="hlFld-ContribAuthor ">Andrew M  Lynn</span>, <span class="hlFld-ContribAuthor ">Rajendra  Prasad</span>, <span class="hlFld-ContribAuthor ">Raok  Jeon</span>, <span class="hlFld-ContribAuthor ">Amresh  Prakash</span>. </span><span class="cited-content_cbyCitation_article-title">Identifying the natural polyphenol catechin as a multi-targeted agent against SARS-CoV-2 for the plausible therapy of COVID-19: an integrated computational approach. </span><span class="cited-content_cbyCitation_journal-name">Briefings in Bioinformatics</span><span> <strong>2021,</strong> <em>22 </em>
                                    (2)
                                     , 1346-1360. <a href="https://doi.org/10.1093/bib/bbaa378" title="DOI URL">https://doi.org/10.1093/bib/bbaa378</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1093/bib/bbaa378&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1093%2Fbib%2Fbbaa378%26sid%3Dliteratum%253Aachs%26jtitle%3DBriefings%2520in%2520Bioinformatics%26atitle%3DIdentifying%252Bthe%252Bnatural%252Bpolyphenol%252Bcatechin%252Bas%252Ba%252Bmulti-targeted%252Bagent%252Bagainst%252BSARS-CoV-2%252Bfor%252Bthe%252Bplausible%252Btherapy%252Bof%252BCOVID-19%25253A%252Ban%252Bintegrated%252Bcomputational%252Bapproach%26aulast%3DMishra%26aufirst%3DChandra%2BBhushan%26date%3D2021%26date%3D2020%26volume%3D22%26issue%3D2%26spage%3D1346%26epage%3D1360" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yuanjun  Shi</span>, <span class="hlFld-ContribAuthor ">Marie L.  Laury</span>, <span class="hlFld-ContribAuthor ">Zhi  Wang</span>, <span class="hlFld-ContribAuthor ">Jay W.  Ponder</span>. </span><span class="cited-content_cbyCitation_article-title">AMOEBA binding free energies for the SAMPL7 TrimerTrip host–guest challenge. </span><span class="cited-content_cbyCitation_journal-name">Journal of Computer-Aided Molecular Design</span><span> <strong>2021,</strong> <em>35 </em>
                                    (1)
                                     , 79-93. <a href="https://doi.org/10.1007/s10822-020-00358-2" title="DOI URL">https://doi.org/10.1007/s10822-020-00358-2</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s10822-020-00358-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs10822-020-00358-2%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Computer-Aided%2520Molecular%2520Design%26atitle%3DAMOEBA%252Bbinding%252Bfree%252Benergies%252Bfor%252Bthe%252BSAMPL7%252BTrimerTrip%252Bhost%2525E2%252580%252593guest%252Bchallenge%26aulast%3DShi%26aufirst%3DYuanjun%26date%3D2021%26date%3D2020%26volume%3D35%26issue%3D1%26spage%3D79%26epage%3D93" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Vivek  Sharma</span>, <span class="hlFld-ContribAuthor ">Jonathan  Lasham</span>, <span class="hlFld-ContribAuthor ">Outi  Haapanen</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting respiratory complex I—molecular mechanism and drug design. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,, 327-340. <a href="https://doi.org/10.1016/B978-0-12-819621-2.00014-0" title="DOI URL">https://doi.org/10.1016/B978-0-12-819621-2.00014-0</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-819621-2.00014-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-819621-2.00014-0%26sid%3Dliteratum%253Aachs%26atitle%3DTargeting%252Brespiratory%252Bcomplex%252BI%2525E2%252580%252594molecular%252Bmechanism%252Band%252Bdrug%252Bdesign%26aulast%3DSharma%26aufirst%3DVivek%26date%3D2021%26spage%3D327%26epage%3D340%26pub%3DElsevier%26atitle%3DClinical%252BBioenergetics%26date%3D2021%26volume%3D16" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jeevan  Patra</span>, <span class="hlFld-ContribAuthor ">Deepanmol  Singh</span>, <span class="hlFld-ContribAuthor ">Sapna  Jain</span>, <span class="hlFld-ContribAuthor ">Neeraj  Mahindroo</span>. </span><span class="cited-content_cbyCitation_article-title">Application of Docking for Lead Optimization. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,, 271-294. <a href="https://doi.org/10.1016/B978-0-12-822312-3.00012-6" title="DOI URL">https://doi.org/10.1016/B978-0-12-822312-3.00012-6</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-822312-3.00012-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-822312-3.00012-6%26sid%3Dliteratum%253Aachs%26atitle%3DApplication%252Bof%252BDocking%252Bfor%252BLead%252BOptimization%26aulast%3DPatra%26aufirst%3DJeevan%26date%3D2021%26spage%3D271%26epage%3D294%26pub%3DElsevier%26atitle%3DMolecular%252BDocking%252Bfor%252BComputer-Aided%252BDrug%252BDesign%26date%3D2021%26volume%3D10" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Outi M. H.  Salo-Ahen</span>, <span class="hlFld-ContribAuthor ">Ida  Alanko</span>, <span class="hlFld-ContribAuthor ">Rajendra  Bhadane</span>, <span class="hlFld-ContribAuthor ">Alexandre M. J. J.  Bonvin</span>, <span class="hlFld-ContribAuthor ">Rodrigo Vargas  Honorato</span>, <span class="hlFld-ContribAuthor ">Shakhawath  Hossain</span>, <span class="hlFld-ContribAuthor ">André H.  Juffer</span>, <span class="hlFld-ContribAuthor ">Aleksei  Kabedev</span>, <span class="hlFld-ContribAuthor ">Maija  Lahtela-Kakkonen</span>, <span class="hlFld-ContribAuthor ">Anders Støttrup  Larsen</span>, <span class="hlFld-ContribAuthor ">Eveline  Lescrinier</span>, <span class="hlFld-ContribAuthor ">Parthiban  Marimuthu</span>, <span class="hlFld-ContribAuthor ">Muhammad Usman  Mirza</span>, <span class="hlFld-ContribAuthor ">Ghulam  Mustafa</span>, <span class="hlFld-ContribAuthor ">Ariane  Nunes-Alves</span>, <span class="hlFld-ContribAuthor ">Tatu  Pantsar</span>, <span class="hlFld-ContribAuthor ">Atefeh  Saadabadi</span>, <span class="hlFld-ContribAuthor ">Kalaimathy  Singaravelu</span>, <span class="hlFld-ContribAuthor ">Michiel  Vanmeert</span>. </span><span class="cited-content_cbyCitation_article-title">Molecular Dynamics Simulations in Drug Discovery and Pharmaceutical Development. </span><span class="cited-content_cbyCitation_journal-name">Processes</span><span> <strong>2021,</strong> <em>9 </em>
                                    (1)
                                     , 71. <a href="https://doi.org/10.3390/pr9010071" title="DOI URL">https://doi.org/10.3390/pr9010071</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/pr9010071&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fpr9010071%26sid%3Dliteratum%253Aachs%26jtitle%3DProcesses%26atitle%3DMolecular%252BDynamics%252BSimulations%252Bin%252BDrug%252BDiscovery%252Band%252BPharmaceutical%252BDevelopment%26aulast%3DSalo-Ahen%26aufirst%3DOuti%2BM.%2BH.%26date%3D2021%26date%3D2020%26volume%3D9%26issue%3D1%26spage%3D71" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Himanshu  Goel</span>, <span class="hlFld-ContribAuthor ">Anthony  Hazel</span>, <span class="hlFld-ContribAuthor ">Vincent D.  Ustach</span>, <span class="hlFld-ContribAuthor ">Sunhwan  Jo</span>, <span class="hlFld-ContribAuthor ">Wenbo  Yu</span>, <span class="hlFld-ContribAuthor ">Alexander D.  MacKerell</span>. </span><span class="cited-content_cbyCitation_article-title">Rapid and accurate estimation of protein–ligand relative binding affinities using site-identification by ligand competitive saturation. </span><span class="cited-content_cbyCitation_journal-name">Chemical Science</span><span> <strong>2021,</strong> <em>38 </em><a href="https://doi.org/10.1039/D1SC01781K" title="DOI URL">https://doi.org/10.1039/D1SC01781K</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D1SC01781K&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD1SC01781K%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Science%26atitle%3DRapid%252Band%252Baccurate%252Bestimation%252Bof%252Bprotein%2525E2%252580%252593ligand%252Brelative%252Bbinding%252Baffinities%252Busing%252Bsite-identification%252Bby%252Bligand%252Bcompetitive%252Bsaturation%26aulast%3DGoel%26aufirst%3DHimanshu%26date%3D2021%26date%3D2021%26volume%3D38" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Anett  Hauser</span>, <span class="hlFld-ContribAuthor ">Songhwan  Hwang</span>, <span class="hlFld-ContribAuthor ">Han  Sun</span>, <span class="hlFld-ContribAuthor ">Christian P. R.  Hackenberger</span>. </span><span class="cited-content_cbyCitation_article-title">Combining free energy calculations with tailored enzyme activity assays to elucidate substrate binding of a phospho-lysine phosphatase. </span><span class="cited-content_cbyCitation_journal-name">Chemical Science</span><span> <strong>2020,</strong> <em>11 </em>
                                    (47)
                                     , 12655-12661. <a href="https://doi.org/10.1039/D0SC03930F" title="DOI URL">https://doi.org/10.1039/D0SC03930F</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D0SC03930F&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD0SC03930F%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Science%26atitle%3DCombining%252Bfree%252Benergy%252Bcalculations%252Bwith%252Btailored%252Benzyme%252Bactivity%252Bassays%252Bto%252Belucidate%252Bsubstrate%252Bbinding%252Bof%252Ba%252Bphospho-lysine%252Bphosphatase%26aulast%3DHauser%26aufirst%3DAnett%26date%3D2020%26date%3D2020%26volume%3D11%26issue%3D47%26spage%3D12655%26epage%3D12661" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shunzhou  Wan</span>, <span class="hlFld-ContribAuthor ">Agastya P.  Bhati</span>, <span class="hlFld-ContribAuthor ">Stefan J.  Zasada</span>, <span class="hlFld-ContribAuthor ">Peter V.  Coveney</span>. </span><span class="cited-content_cbyCitation_article-title">Rapid, accurate, precise and reproducible ligand–protein binding free energy prediction. </span><span class="cited-content_cbyCitation_journal-name">Interface Focus</span><span> <strong>2020,</strong> <em>10 </em>
                                    (6)
                                     , 20200007. <a href="https://doi.org/10.1098/rsfs.2020.0007" title="DOI URL">https://doi.org/10.1098/rsfs.2020.0007</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1098/rsfs.2020.0007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1098%2Frsfs.2020.0007%26sid%3Dliteratum%253Aachs%26jtitle%3DInterface%2520Focus%26atitle%3DRapid%25252C%252Baccurate%25252C%252Bprecise%252Band%252Breproducible%252Bligand%2525E2%252580%252593protein%252Bbinding%252Bfree%252Benergy%252Bprediction%26aulast%3DWan%26aufirst%3DShunzhou%26date%3D2020%26date%3D2020%26volume%3D10%26issue%3D6%26spage%3D20200007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Victor S.  Stroylov</span>, <span class="hlFld-ContribAuthor ">Igor V.  Svitanko</span>, <span class="hlFld-ContribAuthor ">Anna S.  Maksimenko</span>, <span class="hlFld-ContribAuthor ">Victor P.  Kislyi</span>, <span class="hlFld-ContribAuthor ">Marina N.  Semenova</span>, <span class="hlFld-ContribAuthor ">Victor V.  Semenov</span>. </span><span class="cited-content_cbyCitation_article-title">Computational modeling and target synthesis of monomethoxy-substituted o-diphenylisoxazoles with unexpectedly high antimitotic microtubule destabilizing activity. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2020,</strong> <em>30 </em>
                                    (23)
                                     , 127608. <a href="https://doi.org/10.1016/j.bmcl.2020.127608" title="DOI URL">https://doi.org/10.1016/j.bmcl.2020.127608</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2020.127608&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2020.127608%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DComputational%252Bmodeling%252Band%252Btarget%252Bsynthesis%252Bof%252Bmonomethoxy-substituted%252Bo-diphenylisoxazoles%252Bwith%252Bunexpectedly%252Bhigh%252Bantimitotic%252Bmicrotubule%252Bdestabilizing%252Bactivity%26aulast%3DStroylov%26aufirst%3DVictor%2BS.%26date%3D2020%26volume%3D30%26issue%3D23%26spage%3D127608" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kai  Liu</span>, <span class="hlFld-ContribAuthor ">Hironori  Kokubo</span>. </span><span class="cited-content_cbyCitation_article-title">Prediction of ligand binding mode among multiple cross-docking poses by molecular dynamics simulations. </span><span class="cited-content_cbyCitation_journal-name">Journal of Computer-Aided Molecular Design</span><span> <strong>2020,</strong> <em>34 </em>
                                    (11)
                                     , 1195-1205. <a href="https://doi.org/10.1007/s10822-020-00340-y" title="DOI URL">https://doi.org/10.1007/s10822-020-00340-y</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s10822-020-00340-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs10822-020-00340-y%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Computer-Aided%2520Molecular%2520Design%26atitle%3DPrediction%252Bof%252Bligand%252Bbinding%252Bmode%252Bamong%252Bmultiple%252Bcross-docking%252Bposes%252Bby%252Bmolecular%252Bdynamics%252Bsimulations%26aulast%3DLiu%26aufirst%3DKai%26date%3D2020%26date%3D2020%26volume%3D34%26issue%3D11%26spage%3D1195%26epage%3D1205" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Martin  Reinhardt</span>, <span class="hlFld-ContribAuthor ">Helmut  Grubmüller</span>. </span><span class="cited-content_cbyCitation_article-title">Variationally derived intermediates for correlated free-energy estimates between intermediate states. </span><span class="cited-content_cbyCitation_journal-name">Physical Review E</span><span> <strong>2020,</strong> <em>102 </em>
                                    (4)
                                     <a href="https://doi.org/10.1103/PhysRevE.102.043312" title="DOI URL">https://doi.org/10.1103/PhysRevE.102.043312</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1103/PhysRevE.102.043312&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1103%2FPhysRevE.102.043312%26sid%3Dliteratum%253Aachs%26jtitle%3DPhysical%2520Review%2520E%26atitle%3DVariationally%252Bderived%252Bintermediates%252Bfor%252Bcorrelated%252Bfree-energy%252Bestimates%252Bbetween%252Bintermediate%252Bstates%26aulast%3DReinhardt%26aufirst%3DMartin%26date%3D2020%26date%3D2020%26volume%3D102%26issue%3D4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Javier  Vázquez</span>, <span class="hlFld-ContribAuthor ">Manel  López</span>, <span class="hlFld-ContribAuthor ">Enric  Gibert</span>, <span class="hlFld-ContribAuthor ">Enric  Herrero</span>, <span class="hlFld-ContribAuthor ">F. Javier  Luque</span>. </span><span class="cited-content_cbyCitation_article-title">Merging Ligand-Based and Structure-Based Methods in Drug Discovery: An Overview of Combined Virtual Screening Approaches. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2020,</strong> <em>25 </em>
                                    (20)
                                     , 4723. <a href="https://doi.org/10.3390/molecules25204723" title="DOI URL">https://doi.org/10.3390/molecules25204723</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules25204723&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules25204723%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DMerging%252BLigand-Based%252Band%252BStructure-Based%252BMethods%252Bin%252BDrug%252BDiscovery%25253A%252BAn%252BOverview%252Bof%252BCombined%252BVirtual%252BScreening%252BApproaches%26aulast%3DV%25C3%25A1zquez%26aufirst%3DJavier%26date%3D2020%26date%3D2020%26volume%3D25%26issue%3D20%26spage%3D4723" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Andrew P.  Montgomery</span>, <span class="hlFld-ContribAuthor ">Christopher  Dobie</span>, <span class="hlFld-ContribAuthor ">Rémi  Szabo</span>, <span class="hlFld-ContribAuthor ">Laura  Hallam</span>, <span class="hlFld-ContribAuthor ">Marie  Ranson</span>, <span class="hlFld-ContribAuthor ">Haibo  Yu</span>, <span class="hlFld-ContribAuthor ">Danielle  Skropeta</span>. </span><span class="cited-content_cbyCitation_article-title">Design, synthesis and evaluation of carbamate-linked uridyl-based inhibitors of human ST6Gal I. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2020,</strong> <em>28 </em>
                                    (14)
                                     , 115561. <a href="https://doi.org/10.1016/j.bmc.2020.115561" title="DOI URL">https://doi.org/10.1016/j.bmc.2020.115561</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2020.115561&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2020.115561%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DDesign%25252C%252Bsynthesis%252Band%252Bevaluation%252Bof%252Bcarbamate-linked%252Buridyl-based%252Binhibitors%252Bof%252Bhuman%252BST6Gal%252BI%26aulast%3DMontgomery%26aufirst%3DAndrew%2BP.%26date%3D2020%26volume%3D28%26issue%3D14%26spage%3D115561" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Nidhi  Singh</span>, <span class="hlFld-ContribAuthor ">Wenjin  Li</span>. </span><span class="cited-content_cbyCitation_article-title">Absolute Binding Free Energy Calculations for Highly Flexible Protein MDM2 and Its Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">International Journal of Molecular Sciences</span><span> <strong>2020,</strong> <em>21 </em>
                                    (13)
                                     , 4765. <a href="https://doi.org/10.3390/ijms21134765" title="DOI URL">https://doi.org/10.3390/ijms21134765</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/ijms21134765&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fijms21134765%26sid%3Dliteratum%253Aachs%26jtitle%3DInternational%2520Journal%2520of%2520Molecular%2520Sciences%26atitle%3DAbsolute%252BBinding%252BFree%252BEnergy%252BCalculations%252Bfor%252BHighly%252BFlexible%252BProtein%252BMDM2%252Band%252BIts%252BInhibitors%26aulast%3DSingh%26aufirst%3DNidhi%26date%3D2020%26date%3D2020%26volume%3D21%26issue%3D13%26spage%3D4765" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Andreas  Krämer</span>, <span class="hlFld-ContribAuthor ">Phillip S.  Hudson</span>, <span class="hlFld-ContribAuthor ">Michael R.  Jones</span>, <span class="hlFld-ContribAuthor ">Bernard R.  Brooks</span>. </span><span class="cited-content_cbyCitation_article-title">Multi-phase Boltzmann weighting: accounting for local inhomogeneity in molecular simulations of water–octanol partition coefficients in the SAMPL6 challenge. </span><span class="cited-content_cbyCitation_journal-name">Journal of Computer-Aided Molecular Design</span><span> <strong>2020,</strong> <em>34 </em>
                                    (5)
                                     , 471-483. <a href="https://doi.org/10.1007/s10822-020-00285-2" title="DOI URL">https://doi.org/10.1007/s10822-020-00285-2</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s10822-020-00285-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs10822-020-00285-2%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Computer-Aided%2520Molecular%2520Design%26atitle%3DMulti-phase%252BBoltzmann%252Bweighting%25253A%252Baccounting%252Bfor%252Blocal%252Binhomogeneity%252Bin%252Bmolecular%252Bsimulations%252Bof%252Bwater%2525E2%252580%252593octanol%252Bpartition%252Bcoefficients%252Bin%252Bthe%252BSAMPL6%252Bchallenge%26aulast%3DKr%25C3%25A4mer%26aufirst%3DAndreas%26date%3D2020%26date%3D2020%26volume%3D34%26issue%3D5%26spage%3D471%26epage%3D483" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mariangela  Garofalo</span>, <span class="hlFld-ContribAuthor ">Giovanni  Grazioso</span>, <span class="hlFld-ContribAuthor ">Andrea  Cavalli</span>, <span class="hlFld-ContribAuthor ">Jacopo  Sgrignani</span>. </span><span class="cited-content_cbyCitation_article-title">How Computational Chemistry and Drug Delivery Techniques Can Support the Development of New Anticancer Drugs. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2020,</strong> <em>25 </em>
                                    (7)
                                     , 1756. <a href="https://doi.org/10.3390/molecules25071756" title="DOI URL">https://doi.org/10.3390/molecules25071756</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules25071756&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules25071756%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DHow%252BComputational%252BChemistry%252Band%252BDrug%252BDelivery%252BTechniques%252BCan%252BSupport%252Bthe%252BDevelopment%252Bof%252BNew%252BAnticancer%252BDrugs%26aulast%3DGarofalo%26aufirst%3DMariangela%26date%3D2020%26date%3D2020%26volume%3D25%26issue%3D7%26spage%3D1756" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Laurence  Biggers</span>, <span class="hlFld-ContribAuthor ">Hadeer  Elhabashy</span>, <span class="hlFld-ContribAuthor ">Edward  Ackad</span>, <span class="hlFld-ContribAuthor ">Mohammad S.  Yousef</span>. </span><span class="cited-content_cbyCitation_article-title">Molecular dynamics simulations of an engineered T4 lysozyme exclude helix to sheet transition, and provide insights into long distance, intra‐protein switchable motion. </span><span class="cited-content_cbyCitation_journal-name">Protein Science</span><span> <strong>2020,</strong> <em>29 </em>
                                    (2)
                                     , 542-554. <a href="https://doi.org/10.1002/pro.3780" title="DOI URL">https://doi.org/10.1002/pro.3780</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/pro.3780&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fpro.3780%26sid%3Dliteratum%253Aachs%26jtitle%3DProtein%2520Science%26atitle%3DMolecular%252Bdynamics%252Bsimulations%252Bof%252Ban%252Bengineered%252BT4%252Blysozyme%252Bexclude%252Bhelix%252Bto%252Bsheet%252Btransition%25252C%252Band%252Bprovide%252Binsights%252Binto%252Blong%252Bdistance%25252C%252Bintra%2525E2%252580%252590protein%252Bswitchable%252Bmotion%26aulast%3DBiggers%26aufirst%3DLaurence%26date%3D2020%26date%3D2019%26volume%3D29%26issue%3D2%26spage%3D542%26epage%3D554" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiao  Hu</span>, <span class="hlFld-ContribAuthor ">Irene  Maffucci</span>, <span class="hlFld-ContribAuthor ">Alessandro  Contini</span>. </span><span class="cited-content_cbyCitation_article-title">Advances in the Treatment of Explicit Water Molecules in Docking and Binding Free Energy Calculations. </span><span class="cited-content_cbyCitation_journal-name">Current Medicinal Chemistry</span><span> <strong>2020,</strong> <em>26 </em>
                                    (42)
                                     , 7598-7622. <a href="https://doi.org/10.2174/0929867325666180514110824" title="DOI URL">https://doi.org/10.2174/0929867325666180514110824</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.2174/0929867325666180514110824&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.2174%2F0929867325666180514110824%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Medicinal%2520Chemistry%26atitle%3DAdvances%252Bin%252Bthe%252BTreatment%252Bof%252BExplicit%252BWater%252BMolecules%252Bin%252BDocking%252Band%252BBinding%252BFree%252BEnergy%252BCalculations%26aulast%3DHu%26aufirst%3DXiao%26date%3D2020%26volume%3D26%26issue%3D42%26spage%3D7598%26epage%3D7622" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Volodymyr  Tkachenko</span>, <span class="hlFld-ContribAuthor ">Vladimir  Farafonov</span>, <span class="hlFld-ContribAuthor ">Viktor  Tokarev</span>, <span class="hlFld-ContribAuthor ">Irina  Tkachenko</span>. </span><span class="cited-content_cbyCitation_article-title">Study of the effectiveness of various cannabinoid receptor 1 (CB1) agonists using molecular docking and molecular dynamics modeling. </span><span class="cited-content_cbyCitation_journal-name">French-Ukrainian Journal of Chemistry</span><span> <strong>2020,</strong> <em>8 </em>
                                    (1)
                                     , 76-87. <a href="https://doi.org/10.17721/fujcV8I1P76-87" title="DOI URL">https://doi.org/10.17721/fujcV8I1P76-87</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.17721/fujcV8I1P76-87&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.17721%2FfujcV8I1P76-87%26sid%3Dliteratum%253Aachs%26jtitle%3DFrench-Ukrainian%2520Journal%2520of%2520Chemistry%26atitle%3DStudy%252Bof%252Bthe%252Beffectiveness%252Bof%252Bvarious%252Bcannabinoid%252Breceptor%252B1%252B%252528CB1%252529%252Bagonists%252Busing%252Bmolecular%252Bdocking%252Band%252Bmolecular%252Bdynamics%252Bmodeling%26aulast%3DTkachenko%26aufirst%3DVolodymyr%26date%3D2020%26volume%3D8%26issue%3D1%26spage%3D76%26epage%3D87" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shunzhou  Wan</span>, <span class="hlFld-ContribAuthor ">Gary  Tresadern</span>, <span class="hlFld-ContribAuthor ">Laura  Pérez‐Benito</span>, <span class="hlFld-ContribAuthor ">Herman  Vlijmen</span>, <span class="hlFld-ContribAuthor ">Peter V.  Coveney</span>. </span><span class="cited-content_cbyCitation_article-title">Accuracy and Precision of Alchemical Relative Free Energy Predictions with and without Replica‐Exchange. </span><span class="cited-content_cbyCitation_journal-name">Advanced Theory and Simulations</span><span> <strong>2020,</strong> <em>3 </em>
                                    (1)
                                     , 1900195. <a href="https://doi.org/10.1002/adts.201900195" title="DOI URL">https://doi.org/10.1002/adts.201900195</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/adts.201900195&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fadts.201900195%26sid%3Dliteratum%253Aachs%26jtitle%3DAdvanced%2520Theory%2520and%2520Simulations%26atitle%3DAccuracy%252Band%252BPrecision%252Bof%252BAlchemical%252BRelative%252BFree%252BEnergy%252BPredictions%252Bwith%252Band%252Bwithout%252BReplica%2525E2%252580%252590Exchange%26aulast%3DWan%26aufirst%3DShunzhou%26date%3D2020%26date%3D2019%26volume%3D3%26issue%3D1%26spage%3D1900195" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">John E.  Herr</span>, <span class="hlFld-ContribAuthor ">Kevin  Koh</span>, <span class="hlFld-ContribAuthor ">Kun  Yao</span>, <span class="hlFld-ContribAuthor ">John  Parkhill</span>. </span><span class="cited-content_cbyCitation_article-title">Compressing physics with an autoencoder: Creating an atomic species representation to improve machine learning models in the chemical sciences. </span><span class="cited-content_cbyCitation_journal-name">The Journal of Chemical Physics</span><span> <strong>2019,</strong> <em>151 </em>
                                    (8)
                                     , 084103. <a href="https://doi.org/10.1063/1.5108803" title="DOI URL">https://doi.org/10.1063/1.5108803</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1063/1.5108803&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1063%2F1.5108803%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Journal%2520of%2520Chemical%2520Physics%26atitle%3DCompressing%252Bphysics%252Bwith%252Ban%252Bautoencoder%25253A%252BCreating%252Ban%252Batomic%252Bspecies%252Brepresentation%252Bto%252Bimprove%252Bmachine%252Blearning%252Bmodels%252Bin%252Bthe%252Bchemical%252Bsciences%26aulast%3DHerr%26aufirst%3DJohn%2BE.%26date%3D2019%26volume%3D151%26issue%3D8%26spage%3D084103" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Visvaldas  Kairys</span>, <span class="hlFld-ContribAuthor ">Lina  Baranauskiene</span>, <span class="hlFld-ContribAuthor ">Migle  Kazlauskiene</span>, <span class="hlFld-ContribAuthor ">Daumantas  Matulis</span>, <span class="hlFld-ContribAuthor ">Egidijus  Kazlauskas</span>. </span><span class="cited-content_cbyCitation_article-title">Binding affinity in drug design: experimental and computational techniques. </span><span class="cited-content_cbyCitation_journal-name">Expert Opinion on Drug Discovery</span><span> <strong>2019,</strong> <em>14 </em>
                                    (8)
                                     , 755-768. <a href="https://doi.org/10.1080/17460441.2019.1623202" title="DOI URL">https://doi.org/10.1080/17460441.2019.1623202</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/17460441.2019.1623202&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F17460441.2019.1623202%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Opinion%2520on%2520Drug%2520Discovery%26atitle%3DBinding%252Baffinity%252Bin%252Bdrug%252Bdesign%25253A%252Bexperimental%252Band%252Bcomputational%252Btechniques%26aulast%3DKairys%26aufirst%3DVisvaldas%26date%3D2019%26date%3D2019%26volume%3D14%26issue%3D8%26spage%3D755%26epage%3D768" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Daniel J.  Cole</span>, <span class="hlFld-ContribAuthor ">Israel  Cabeza de Vaca</span>, <span class="hlFld-ContribAuthor ">William L.  Jorgensen</span>. </span><span class="cited-content_cbyCitation_article-title">Computation of protein–ligand binding free energies using quantum mechanical bespoke force fields. </span><span class="cited-content_cbyCitation_journal-name">MedChemComm</span><span> <strong>2019,</strong> <em>10 </em>
                                    (7)
                                     , 1116-1120. <a href="https://doi.org/10.1039/C9MD00017H" title="DOI URL">https://doi.org/10.1039/C9MD00017H</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C9MD00017H&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC9MD00017H%26sid%3Dliteratum%253Aachs%26jtitle%3DMedChemComm%26atitle%3DComputation%252Bof%252Bprotein%2525E2%252580%252593ligand%252Bbinding%252Bfree%252Benergies%252Busing%252Bquantum%252Bmechanical%252Bbespoke%252Bforce%252Bfields%26aulast%3DCole%26aufirst%3DDaniel%2BJ.%26date%3D2019%26date%3D2019%26volume%3D10%26issue%3D7%26spage%3D1116%26epage%3D1120" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Leandro  Battini</span>, <span class="hlFld-ContribAuthor ">Mariela  Bollini</span>. </span><span class="cited-content_cbyCitation_article-title">Challenges and approaches in the discovery of human immunodeficiency virus type‐1 non‐nucleoside reverse transcriptase inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Medicinal Research Reviews</span><span> <strong>2019,</strong> <em>39 </em>
                                    (4)
                                     , 1235-1273. <a href="https://doi.org/10.1002/med.21544" title="DOI URL">https://doi.org/10.1002/med.21544</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/med.21544&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fmed.21544%26sid%3Dliteratum%253Aachs%26jtitle%3DMedicinal%2520Research%2520Reviews%26atitle%3DChallenges%252Band%252Bapproaches%252Bin%252Bthe%252Bdiscovery%252Bof%252Bhuman%252Bimmunodeficiency%252Bvirus%252Btype%2525E2%252580%2525901%252Bnon%2525E2%252580%252590nucleoside%252Breverse%252Btranscriptase%252Binhibitors%26aulast%3DBattini%26aufirst%3DLeandro%26date%3D2019%26date%3D2018%26volume%3D39%26issue%3D4%26spage%3D1235%26epage%3D1273" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Marcos D.  Muñoz</span>, <span class="hlFld-ContribAuthor ">Lucas J.  Gutierrez</span>, <span class="hlFld-ContribAuthor ">Sandrine  Delignat</span>, <span class="hlFld-ContribAuthor ">Jules  Russick</span>, <span class="hlFld-ContribAuthor ">Sandra E.  Gomez Mejiba</span>, <span class="hlFld-ContribAuthor ">Sebastien  Lacroix-Desmazes</span>, <span class="hlFld-ContribAuthor ">Ricardo D.  Enriz</span>, <span class="hlFld-ContribAuthor ">Dario C.  Ramirez</span>. </span><span class="cited-content_cbyCitation_article-title">The nitrone spin trap 5,5‑dimethyl‑1‑pyrroline N‑oxide binds to toll-like receptor-2-TIR-BB-loop domain and dampens downstream inflammatory signaling. </span><span class="cited-content_cbyCitation_journal-name">Biochimica et Biophysica Acta (BBA) - Molecular Basis of Disease</span><span> <strong>2019,</strong> <em>1865 </em>
                                    (6)
                                     , 1152-1159. <a href="https://doi.org/10.1016/j.bbadis.2019.01.005" title="DOI URL">https://doi.org/10.1016/j.bbadis.2019.01.005</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bbadis.2019.01.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bbadis.2019.01.005%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochimica%2520et%2520Biophysica%2520Acta%2520%2528BBA%2529%2520-%2520Molecular%2520Basis%2520of%2520Disease%26atitle%3DThe%252Bnitrone%252Bspin%252Btrap%252B5%25252C5%2525E2%252580%252591dimethyl%2525E2%252580%2525911%2525E2%252580%252591pyrroline%252BN%2525E2%252580%252591oxide%252Bbinds%252Bto%252Btoll-like%252Breceptor-2-TIR-BB-loop%252Bdomain%252Band%252Bdampens%252Bdownstream%252Binflammatory%252Bsignaling%26aulast%3DMu%25C3%25B1oz%26aufirst%3DMarcos%2BD.%26date%3D2019%26volume%3D1865%26issue%3D6%26spage%3D1152%26epage%3D1159" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Thomas M.  Kaiser</span>, <span class="hlFld-ContribAuthor ">Pieter B.  Burger</span>. </span><span class="cited-content_cbyCitation_article-title">Error Tolerance of Machine Learning Algorithms across Contemporary Biological Targets. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2019,</strong> <em>24 </em>
                                    (11)
                                     , 2115. <a href="https://doi.org/10.3390/molecules24112115" title="DOI URL">https://doi.org/10.3390/molecules24112115</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules24112115&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules24112115%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DError%252BTolerance%252Bof%252BMachine%252BLearning%252BAlgorithms%252Bacross%252BContemporary%252BBiological%252BTargets%26aulast%3DKaiser%26aufirst%3DThomas%2BM.%26date%3D2019%26date%3D2019%26volume%3D24%26issue%3D11%26spage%3D2115" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Maral  Aminpour</span>, <span class="hlFld-ContribAuthor ">Carlo  Montemagno</span>, <span class="hlFld-ContribAuthor ">Jack A.  Tuszynski</span>. </span><span class="cited-content_cbyCitation_article-title">An Overview of Molecular Modeling for Drug Discovery with Specific Illustrative Examples of Applications. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2019,</strong> <em>24 </em>
                                    (9)
                                     , 1693. <a href="https://doi.org/10.3390/molecules24091693" title="DOI URL">https://doi.org/10.3390/molecules24091693</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules24091693&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules24091693%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DAn%252BOverview%252Bof%252BMolecular%252BModeling%252Bfor%252BDrug%252BDiscovery%252Bwith%252BSpecific%252BIllustrative%252BExamples%252Bof%252BApplications%26aulast%3DAminpour%26aufirst%3DMaral%26date%3D2019%26date%3D2019%26volume%3D24%26issue%3D9%26spage%3D1693" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">José M.  Granadino-Roldán</span>, <span class="hlFld-ContribAuthor ">Antonia S. J. S.  Mey</span>, <span class="hlFld-ContribAuthor ">Juan J.  Pérez González</span>, <span class="hlFld-ContribAuthor ">Stefano  Bosisio</span>, <span class="hlFld-ContribAuthor ">Jaime  Rubio-Martinez</span>, <span class="hlFld-ContribAuthor ">Julien  Michel</span>, . </span><span class="cited-content_cbyCitation_article-title">Effect of set up protocols on the accuracy of alchemical free energy calculation over a set of ACK1 inhibitors. </span><span class="cited-content_cbyCitation_journal-name">PLOS ONE</span><span> <strong>2019,</strong> <em>14 </em>
                                    (3)
                                     , e0213217. <a href="https://doi.org/10.1371/journal.pone.0213217" title="DOI URL">https://doi.org/10.1371/journal.pone.0213217</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1371/journal.pone.0213217&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1371%2Fjournal.pone.0213217%26sid%3Dliteratum%253Aachs%26jtitle%3DPLOS%2520ONE%26atitle%3DEffect%252Bof%252Bset%252Bup%252Bprotocols%252Bon%252Bthe%252Baccuracy%252Bof%252Balchemical%252Bfree%252Benergy%252Bcalculation%252Bover%252Ba%252Bset%252Bof%252BACK1%252Binhibitors%26aulast%3DGranadino-Rold%25C3%25A1n%26aufirst%3DJos%25C3%25A9%2BM.%26date%3D2019%26date%3D2019%26volume%3D14%26issue%3D3%26spage%3De0213217" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Myong In  Oh</span>, <span class="hlFld-ContribAuthor ">Donald F.  Weaver</span>. </span><span class="cited-content_cbyCitation_article-title">Explicit and Implicit Water Models for Biomolecular Simulations. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,<a href="https://doi.org/10.1016/B978-0-12-409547-2.14661-X" title="DOI URL">https://doi.org/10.1016/B978-0-12-409547-2.14661-X</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-409547-2.14661-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-409547-2.14661-X%26sid%3Dliteratum%253Aachs%26atitle%3DExplicit%252Band%252BImplicit%252BWater%252BModels%252Bfor%252BBiomolecular%252BSimulations%26aulast%3DOh%26aufirst%3DMyong%2BIn%26date%3D2019%26pub%3DElsevier%26atitle%3DReference%252BModule%252Bin%252BChemistry%25252C%252BMolecular%252BSciences%252Band%252BChemical%252BEngineering%26date%3D2019%26volume%3D46" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Varadharajan  Thiyagarajan</span>, <span class="hlFld-ContribAuthor ">Kuan-Wei  Lee</span>, <span class="hlFld-ContribAuthor ">Max K.  Leong</span>, <span class="hlFld-ContribAuthor ">Ching-Feng  Weng</span>. </span><span class="cited-content_cbyCitation_article-title">Potential natural mTOR inhibitors screened by
              in silico
              approach and suppress hepatic stellate cells activation. </span><span class="cited-content_cbyCitation_journal-name">Journal of Biomolecular Structure and Dynamics</span><span> <strong>2018,</strong> <em>36 </em>
                                    (16)
                                     , 4220-4234. <a href="https://doi.org/10.1080/07391102.2017.1411295" title="DOI URL">https://doi.org/10.1080/07391102.2017.1411295</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/07391102.2017.1411295&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F07391102.2017.1411295%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Biomolecular%2520Structure%2520and%2520Dynamics%26atitle%3DPotential%252Bnatural%252BmTOR%252Binhibitors%252Bscreened%252Bby%252Bin%252Bsilico%252Bapproach%252Band%252Bsuppress%252Bhepatic%252Bstellate%252Bcells%252Bactivation%26aulast%3DThiyagarajan%26aufirst%3DVaradharajan%26date%3D2018%26date%3D2017%26volume%3D36%26issue%3D16%26spage%3D4220%26epage%3D4234" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Laura  Pérez-Benito</span>, <span class="hlFld-ContribAuthor ">Henrik  Keränen</span>, <span class="hlFld-ContribAuthor ">Herman  van Vlijmen</span>, <span class="hlFld-ContribAuthor ">Gary  Tresadern</span>. </span><span class="cited-content_cbyCitation_article-title">Predicting Binding Free Energies of PDE2 Inhibitors. The Difficulties of Protein Conformation. </span><span class="cited-content_cbyCitation_journal-name">Scientific Reports</span><span> <strong>2018,</strong> <em>8 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/s41598-018-23039-5" title="DOI URL">https://doi.org/10.1038/s41598-018-23039-5</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41598-018-23039-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41598-018-23039-5%26sid%3Dliteratum%253Aachs%26jtitle%3DScientific%2520Reports%26atitle%3DPredicting%252BBinding%252BFree%252BEnergies%252Bof%252BPDE2%252BInhibitors.%252BThe%252BDifficulties%252Bof%252BProtein%252BConformation%26aulast%3DP%25C3%25A9rez-Benito%26aufirst%3DLaura%26date%3D2018%26date%3D2018%26volume%3D8%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">K.  Kellett</span>, <span class="hlFld-ContribAuthor ">S. A.  Kantonen</span>, <span class="hlFld-ContribAuthor ">B. M.  Duggan</span>, <span class="hlFld-ContribAuthor ">M. K.  Gilson</span>. </span><span class="cited-content_cbyCitation_article-title">Toward Expanded Diversity of Host–Guest Interactions via Synthesis and Characterization of Cyclodextrin Derivatives. </span><span class="cited-content_cbyCitation_journal-name">Journal of Solution Chemistry</span><span> <strong>2018,</strong> <em>47 </em>
                                    (10)
                                     , 1597-1608. <a href="https://doi.org/10.1007/s10953-018-0769-1" title="DOI URL">https://doi.org/10.1007/s10953-018-0769-1</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s10953-018-0769-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs10953-018-0769-1%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Solution%2520Chemistry%26atitle%3DToward%252BExpanded%252BDiversity%252Bof%252BHost%2525E2%252580%252593Guest%252BInteractions%252Bvia%252BSynthesis%252Band%252BCharacterization%252Bof%252BCyclodextrin%252BDerivatives%26aulast%3DKellett%26aufirst%3DK.%26date%3D2018%26date%3D2018%26volume%3D47%26issue%3D10%26spage%3D1597%26epage%3D1608" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Marie L.  Laury</span>, <span class="hlFld-ContribAuthor ">Zhi  Wang</span>, <span class="hlFld-ContribAuthor ">Aaron S.  Gordon</span>, <span class="hlFld-ContribAuthor ">Jay W.  Ponder</span>. </span><span class="cited-content_cbyCitation_article-title">Absolute binding free energies for the SAMPL6 cucurbit[8]uril host–guest challenge via the AMOEBA polarizable force field. </span><span class="cited-content_cbyCitation_journal-name">Journal of Computer-Aided Molecular Design</span><span> <strong>2018,</strong> <em>32 </em>
                                    (10)
                                     , 1087-1095. <a href="https://doi.org/10.1007/s10822-018-0147-5" title="DOI URL">https://doi.org/10.1007/s10822-018-0147-5</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s10822-018-0147-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs10822-018-0147-5%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Computer-Aided%2520Molecular%2520Design%26atitle%3DAbsolute%252Bbinding%252Bfree%252Benergies%252Bfor%252Bthe%252BSAMPL6%252Bcucurbit%25255B8%25255Duril%252Bhost%2525E2%252580%252593guest%252Bchallenge%252Bvia%252Bthe%252BAMOEBA%252Bpolarizable%252Bforce%252Bfield%26aulast%3DLaury%26aufirst%3DMarie%2BL.%26date%3D2018%26date%3D2018%26volume%3D32%26issue%3D10%26spage%3D1087%26epage%3D1095" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yiğitcan  Eken</span>, <span class="hlFld-ContribAuthor ">Prajay  Patel</span>, <span class="hlFld-ContribAuthor ">Thomas  Díaz</span>, <span class="hlFld-ContribAuthor ">Michael R.  Jones</span>, <span class="hlFld-ContribAuthor ">Angela K.  Wilson</span>. </span><span class="cited-content_cbyCitation_article-title">SAMPL6 host–guest challenge: binding free energies via a multistep approach. </span><span class="cited-content_cbyCitation_journal-name">Journal of Computer-Aided Molecular Design</span><span> <strong>2018,</strong> <em>32 </em>
                                    (10)
                                     , 1097-1115. <a href="https://doi.org/10.1007/s10822-018-0159-1" title="DOI URL">https://doi.org/10.1007/s10822-018-0159-1</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s10822-018-0159-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs10822-018-0159-1%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Computer-Aided%2520Molecular%2520Design%26atitle%3DSAMPL6%252Bhost%2525E2%252580%252593guest%252Bchallenge%25253A%252Bbinding%252Bfree%252Benergies%252Bvia%252Ba%252Bmultistep%252Bapproach%26aulast%3DEken%26aufirst%3DYi%25C4%259Fitcan%26date%3D2018%26date%3D2018%26volume%3D32%26issue%3D10%26spage%3D1097%26epage%3D1115" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Krystel  El Hage</span>, <span class="hlFld-ContribAuthor ">Padmabati  Mondal</span>, <span class="hlFld-ContribAuthor ">Markus  Meuwly</span>. </span><span class="cited-content_cbyCitation_article-title">Free energy simulations for protein ligand binding and stability. </span><span class="cited-content_cbyCitation_journal-name">Molecular Simulation</span><span> <strong>2018,</strong> <em>44 </em>
                                    (13-14)
                                     , 1044-1061. <a href="https://doi.org/10.1080/08927022.2017.1416115" title="DOI URL">https://doi.org/10.1080/08927022.2017.1416115</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/08927022.2017.1416115&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F08927022.2017.1416115%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Simulation%26atitle%3DFree%252Benergy%252Bsimulations%252Bfor%252Bprotein%252Bligand%252Bbinding%252Band%252Bstability%26aulast%3DEl%2BHage%26aufirst%3DKrystel%26date%3D2018%26date%3D2018%26volume%3D44%26issue%3D13-14%26spage%3D1044%26epage%3D1061" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Susan E.  Northfield</span>, <span class="hlFld-ContribAuthor ">Jerome  Wielens</span>, <span class="hlFld-ContribAuthor ">Stephen J.  Headey</span>, <span class="hlFld-ContribAuthor ">Billy J.  Williams‐Noonan</span>, <span class="hlFld-ContribAuthor ">Mark  Mulcair</span>, <span class="hlFld-ContribAuthor ">Martin J.  Scanlon</span>, <span class="hlFld-ContribAuthor ">Michael W.  Parker</span>, <span class="hlFld-ContribAuthor ">Philip E.  Thompson</span>, <span class="hlFld-ContribAuthor ">David K.  Chalmers</span>. </span><span class="cited-content_cbyCitation_article-title">Cyclic Hexapeptide Mimics of the LEDGF Integrase Recognition Loop in Complex with HIV‐1 Integrase. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2018,</strong> <em>13 </em>
                                    (15)
                                     , 1555-1565. <a href="https://doi.org/10.1002/cmdc.201800129" title="DOI URL">https://doi.org/10.1002/cmdc.201800129</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.201800129&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.201800129%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DCyclic%252BHexapeptide%252BMimics%252Bof%252Bthe%252BLEDGF%252BIntegrase%252BRecognition%252BLoop%252Bin%252BComplex%252Bwith%252BHIV%2525E2%252580%2525901%252BIntegrase%26aulast%3DNorthfield%26aufirst%3DSusan%2BE.%26date%3D2018%26date%3D2018%26volume%3D13%26issue%3D15%26spage%3D1555%26epage%3D1565" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Peng-fei  Li</span>, <span class="hlFld-ContribAuthor ">Xiang-yu  Jia</span>, <span class="hlFld-ContribAuthor ">Mei-ting  Wang</span>, <span class="hlFld-ContribAuthor ">Ye  Mei</span>. </span><span class="cited-content_cbyCitation_article-title">Comparison of Accuracy and Convergence Rate between Equilibrium and Nonequilibrium Alchemical Transformations for Calculation of Relative Binding Free Energy. </span><span class="cited-content_cbyCitation_journal-name">Chinese Journal of Chemical Physics</span><span> <strong>2017,</strong> <em>30 </em>
                                    (6)
                                     , 789-799. <a href="https://doi.org/10.1063/1674-0068/30/cjcp1711204" title="DOI URL">https://doi.org/10.1063/1674-0068/30/cjcp1711204</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1063/1674-0068/30/cjcp1711204&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1063%2F1674-0068%2F30%2Fcjcp1711204%26sid%3Dliteratum%253Aachs%26jtitle%3DChinese%2520Journal%2520of%2520Chemical%2520Physics%26atitle%3DComparison%252Bof%252BAccuracy%252Band%252BConvergence%252BRate%252Bbetween%252BEquilibrium%252Band%252BNonequilibrium%252BAlchemical%252BTransformations%252Bfor%252BCalculation%252Bof%252BRelative%252BBinding%252BFree%252BEnergy%26aulast%3DLi%26aufirst%3DPeng-fei%26date%3D2017%26volume%3D30%26issue%3D6%26spage%3D789%26epage%3D799" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b00681/20180201/images/medium/jm-2017-00681c_0008.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b00681/20180201/images/large/jm-2017-00681c_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00681&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b00681/20180201/images/medium/jm-2017-00681c_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b00681/20180201/images/large/jm-2017-00681c_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Equilibrium between bound and unbound states of a system containing a protein and ligand, shown subdivided into microstates. The protein is green, the ligand is brown, and the solvent is pale blue. The observed affinity of the ligand for the bound state depends on the total numbers of bound and free states (<i>n</i><sub>bound</sub> and <i>n</i><sub>free</sub>) and the energy of each state.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b00681/20180201/images/large/jm-2017-00681c_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00681&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b00681/20180201/images/medium/jm-2017-00681c_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b00681/20180201/images/large/jm-2017-00681c_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Cubic periodic boundary conditions. A peptide (red) and nearby water molecules are surrounded by the 26 nearest replicates of the main cell.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b00681/20180201/images/large/jm-2017-00681c_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00681&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b00681/20180201/images/medium/jm-2017-00681c_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b00681/20180201/images/large/jm-2017-00681c_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Schematic representations of steered MD and umbrella sampling simulations for the calculation of the free energy of a ligand (red) binding to a receptor (green). Steered MD simulations use a continuous force to pull the ligand out of the receptor. Umbrella sampling calculations use multiple simulations with the ligand restrained at different distances from the receptor. The final potential of mean force is assembled from the smaller sections derived from each individual simulation.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b00681/20180201/images/large/jm-2017-00681c_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00681&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b00681/20180201/images/medium/jm-2017-00681c_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b00681/20180201/images/large/jm-2017-00681c_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Examples of alchemical molecular transformations. (A) Alchemical perturbation of ethane to methanol. C, H, O, and D represent carbon, hydrogen, oxygen, and noninteracting “dummy” atoms. (B) Transformation of ethane into dummy atoms that make intramolecular interactions but do not interact with the surrounding environment. In each case, the variable λ is used to represent the progression between initial and final states. In the intermediate λ-states (0 < λ < 1), the atom properties (charge, van der Waals interaction strength) are varied smoothly between those of the initial and final states.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b00681/20180201/images/large/jm-2017-00681c_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00681&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b00681/20180201/images/medium/jm-2017-00681c_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b00681/20180201/images/large/jm-2017-00681c_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Thermodynamic cycles for the calculation of free energies of ligand binding. (A) Calculation of the absolute free energy of binding of ligand <b>a</b> (Δ<i>G</i><sub>bind(a)</sub>). The red ellipse represents ligand <b>a</b>, the green structure is the protein, and the dotted white ellipse is ligand <b>a</b> where all interactions are decoupled from the surrounding environment. (B) Thermodynamic cycle to calculate the relative free energy difference in binding of ligands <b>a</b> and <b>b (</b>ΔΔ<i>G</i><sub>bind(a→b)</sub><i>)</i>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b00681/20180201/images/large/jm-2017-00681c_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00681&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b00681/20180201/images/medium/jm-2017-00681c_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b00681/20180201/images/large/jm-2017-00681c_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. A cycle of relative alchemical free energy calculations. The sides of each cycle are referred to as the edges and are depicted with an arrow. Each edge describes an alchemical perturbation from one ligand to another. If the sum of all three edges is a nonzero value, as is the case on the left, then this hysteresis (ξ) can be corrected to yield a cycle with zero hysteresis (ξ′).<a onclick="showRef(event, 'ref76'); return false;" href="javascript:void(0);" class="ref ref76">(76)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b00681/20180201/images/large/jm-2017-00681c_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00681&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b00681/20180201/images/medium/jm-2017-00681c_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b00681/20180201/images/large/jm-2017-00681c_0007.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Perturbation map for the progressive transformation of catechol into tolcapone or opicapone.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b00681/20180201/images/large/jm-2017-00681c_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00681&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i21">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_67460" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_67460" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 110 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Yuriev, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Agostino, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ramsland, P. A.</span><span> </span><span class="NLM_article-title">Challenges and advances in computational docking: 2009 in review</span> <span class="citation_source-journal">J. Mol. Recognit.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x">, </span> <span class="NLM_fpage">149</span><span class="NLM_x">–</span> <span class="NLM_lpage">164</span><span class="refDoi"> DOI: 10.1002/jmr.1077</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1002%2Fjmr.1077" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=21360606" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC3MXisV2jsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2011&pages=149-164&author=E.+Yurievauthor=M.+Agostinoauthor=P.+A.+Ramsland&title=Challenges+and+advances+in+computational+docking%3A+2009+in+review&doi=10.1002%2Fjmr.1077"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Challenges and advances in computational docking: 2009 in review</span></div><div class="casAuthors">Yuriev, Elizabeth; Agostino, Mark; Ramsland, Paul A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Recognition</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">149-164</span>CODEN:
                <span class="NLM_cas:coden">JMORE4</span>;
        ISSN:<span class="NLM_cas:issn">0952-3499</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  Docking is a computational technique that places a small mol. (ligand) in the binding site of its macromol. target (receptor) and ests. its binding affinity.  This review addresses methodol. developments that have occurred in the docking field in 2009, with a particular focus on the more difficult, and sometimes controversial, aspects of this promising computational discipline.  These developments aim to address the main challenges of docking: receptor representation (such aspects as structural waters, side chain protonation, and, most of all, flexibility (from side chain rotation to domain movement)), ligand representation (protonation, tautomerism and stereoisomerism, and the effect of input conformation), as well as accounting for solvation and entropy of binding.  This review is strongly focused on docking advances in the context of drug design, specifically in virtual screening and fragment-based drug design.  Copyright © 2010 John Wiley & Sons, Ltd.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEIQpQeuq9f7Vg90H21EOLACvtfcHk0lgmu_5J9agr3w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXisV2jsb8%253D&md5=214fb8189f4bb43675893ac6bdedf51d</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1002%2Fjmr.1077&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjmr.1077%26sid%3Dliteratum%253Aachs%26aulast%3DYuriev%26aufirst%3DE.%26aulast%3DAgostino%26aufirst%3DM.%26aulast%3DRamsland%26aufirst%3DP.%2BA.%26atitle%3DChallenges%2520and%2520advances%2520in%2520computational%2520docking%253A%25202009%2520in%2520review%26jtitle%3DJ.%2520Mol.%2520Recognit.%26date%3D2011%26volume%3D24%26spage%3D149%26epage%3D164%26doi%3D10.1002%2Fjmr.1077" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Yuriev, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ramsland, P. A.</span><span> </span><span class="NLM_article-title">Latest developments in molecular docking: 2010–2011 in review</span> <span class="citation_source-journal">J. Mol. Recognit.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">215</span><span class="NLM_x">–</span> <span class="NLM_lpage">239</span><span class="refDoi"> DOI: 10.1002/jmr.2266</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1002%2Fjmr.2266" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=23526775" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC3sXksFSrtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2013&pages=215-239&author=E.+Yurievauthor=P.+A.+Ramsland&title=Latest+developments+in+molecular+docking%3A+2010%E2%80%932011+in+review&doi=10.1002%2Fjmr.2266"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Latest developments in molecular docking: 2010-2011 in review</span></div><div class="casAuthors">Yuriev, Elizabeth; Ramsland, Paul A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Recognition</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">215-239</span>CODEN:
                <span class="NLM_cas:coden">JMORE4</span>;
        ISSN:<span class="NLM_cas:issn">0952-3499</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  The aim of docking is to accurately predict the structure of a ligand within the constraints of a receptor binding site and to correctly est. the strength of binding.  We discuss, in detail, methodol. developments that occurred in the docking field in 2010 and 2011, with a particular focus on the more difficult, and sometimes controversial, aspects of this promising computational discipline.  The main developments in docking in this period, covered in this review, are receptor flexibility, solvation, fragment docking, postprocessing, docking into homol. models, and docking comparisons.  Several new, or at least newly invigorated, advances occurred in areas such as nonlinear scoring functions, using machine-learning approaches.  This review is strongly focused on docking advances in the context of drug design, specifically in virtual screening and fragment-based drug design.  Where appropriate, we refer readers to exemplar case studies.  Copyright © 2013 John Wiley & Sons, Ltd.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGponcWWCcI4b7Vg90H21EOLACvtfcHk0lgmu_5J9agr3w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXksFSrtrY%253D&md5=b712888ca6c7fd4142711fa252de2456</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1002%2Fjmr.2266&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjmr.2266%26sid%3Dliteratum%253Aachs%26aulast%3DYuriev%26aufirst%3DE.%26aulast%3DRamsland%26aufirst%3DP.%2BA.%26atitle%3DLatest%2520developments%2520in%2520molecular%2520docking%253A%25202010%25E2%2580%25932011%2520in%2520review%26jtitle%3DJ.%2520Mol.%2520Recognit.%26date%3D2013%26volume%3D26%26spage%3D215%26epage%3D239%26doi%3D10.1002%2Fjmr.2266" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Lohning, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Levonis, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Williams-Noonan, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schweiker, S.</span><span> </span><span class="NLM_article-title">A practical guide to molecular docking and homology modelling for medicinal chemists</span> <span class="citation_source-journal">Curr. Top. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">2023</span><span class="NLM_x">–</span> <span class="NLM_lpage">2040</span><span class="refDoi"> DOI: 10.2174/1568026617666170130110827</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.2174%2F1568026617666170130110827" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=28137238" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVShtr3J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2017&pages=2023-2040&author=A.+Lohningauthor=S.+Levonisauthor=B.+Williams-Noonanauthor=S.+Schweiker&title=A+practical+guide+to+molecular+docking+and+homology+modelling+for+medicinal+chemists&doi=10.2174%2F1568026617666170130110827"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">A Practical Guide to Molecular Docking and Homology Modelling for Medicinal Chemists</span></div><div class="casAuthors">Lohning, Anna E.; Levonis, Stephan M.; Williams-Noonan, Billy; Schweiker, Stephanie S.</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">2023-2040</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Elucidating details of the relationship between mol. structure and a particular biol. end point is essential for successful, rational drug discovery.  Mol. docking is a widely accepted tool for lead identification however, navigating the intricacies of the software can be daunting.  Our objective was therefore to provide a step-by-step guide for those interested in incorporating contemporary basic mol. docking and homol. modeling into their design strategy.  Three mol. docking programs, AutoDock4, SwissDock and Surflex-Dock, were compared in the context of a case study where a set of steroidal and non-steroidal ligands were docked into the human androgen receptor (hAR) using both rigid and flexible target atoms.  Metrics for comparison included how well each program predicted the X-ray structure orientation via root mean square deviation (rmsd), predicting known actives via ligand ranking and comparison to biol. data where available.  Benchmarking metrics were discussed in terms of identifying accurate and reliable results.  For cases where no three dimensional structure exists, we provided a practical example for creating a homol. model using Swiss-Model.  Results showed an rmsd between X-ray ligands from wild-type and mutant receptors and docked poses were 4.15Å and 0.83Å (SwissDock), 2.69Å and 8.80Å (AutoDock4) and 0.39Å and 0.71Å (Surflex-Dock) resp.  Surflex-Dock performed consistently well in pose prediction (less than 2Å) while Auto- Dock4 predicted known active non-steroidal antiandrogens most accurately.  Introducing flexibility into target atoms produced the largest degree of change in ligand ranking in Surflex-Dock.  We produced a viable homol. model of the P2X1 purireceptor for subsequent docking anal.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrDrBGiwd68d7Vg90H21EOLACvtfcHk0lgmu_5J9agr3w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVShtr3J&md5=5dfe7ec9d9b9756e07b7cb04fee69145</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.2174%2F1568026617666170130110827&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1568026617666170130110827%26sid%3Dliteratum%253Aachs%26aulast%3DLohning%26aufirst%3DA.%26aulast%3DLevonis%26aufirst%3DS.%26aulast%3DWilliams-Noonan%26aufirst%3DB.%26aulast%3DSchweiker%26aufirst%3DS.%26atitle%3DA%2520practical%2520guide%2520to%2520molecular%2520docking%2520and%2520homology%2520modelling%2520for%2520medicinal%2520chemists%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2017%26volume%3D17%26spage%3D2023%26epage%3D2040%26doi%3D10.2174%2F1568026617666170130110827" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Aqvist, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marelius, J.</span><span> </span><span class="NLM_article-title">The linear interaction energy method for predicting ligand binding free energies</span> <span class="citation_source-journal">Comb. Chem. High Throughput Screening</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">613</span><span class="NLM_x">–</span> <span class="NLM_lpage">626</span><span class="refDoi"> DOI: 10.2174/1386207013330661</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.2174%2F1386207013330661" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=11812258" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BD38Xks1Oktg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2001&pages=613-626&author=J.+Aqvistauthor=J.+Marelius&title=The+linear+interaction+energy+method+for+predicting+ligand+binding+free+energies&doi=10.2174%2F1386207013330661"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">The linear interaction energy method for predicting ligand binding free energies</span></div><div class="casAuthors">Aqvist, Johan; Marelius, John</div><div class="citationInfo"><span class="NLM_cas:title">Combinatorial Chemistry and High Throughput Screening</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">613-626</span>CODEN:
                <span class="NLM_cas:coden">CCHSFU</span>;
        ISSN:<span class="NLM_cas:issn">1386-2073</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers</span>)
        </div><div class="casAbstract">An overview of the simplified linear interaction energy (LIE) method for calcn. of ligand binding free energies is given.  This method is based on force field estns. of the receptor-ligand interactions and thermal conformational sampling.  A notable feature is that the binding energetics can be predicted by considering only the intermol. interactions between the ligand and receptor.  The approxns. behind this approach are examd. and different parameterizations of the model are discussed.  In general, LIE type of methods appear particularly useful for computational drug lead optimization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2CwI46ka7i7Vg90H21EOLACvtfcHk0ljexrYChyS7Vg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xks1Oktg%253D%253D&md5=53d20d4d9bfe7f03780d5e280d8c484d</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.2174%2F1386207013330661&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1386207013330661%26sid%3Dliteratum%253Aachs%26aulast%3DAqvist%26aufirst%3DJ.%26aulast%3DMarelius%26aufirst%3DJ.%26atitle%3DThe%2520linear%2520interaction%2520energy%2520method%2520for%2520predicting%2520ligand%2520binding%2520free%2520energies%26jtitle%3DComb.%2520Chem.%2520High%2520Throughput%2520Screening%26date%3D2001%26volume%3D4%26spage%3D613%26epage%3D626%26doi%3D10.2174%2F1386207013330661" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Makri, N.</span><span> </span><span class="NLM_article-title">The linear response approximation and its lowest order corrections: An influence functional approach</span> <span class="citation_source-journal">J. Phys. Chem. B</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">103</span><span class="NLM_x">, </span> <span class="NLM_fpage">2823</span><span class="NLM_x">–</span> <span class="NLM_lpage">2829</span><span class="refDoi"> DOI: 10.1021/jp9847540</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jp9847540" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADyaK1MXitVags70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=1999&pages=2823-2829&author=N.+Makri&title=The+linear+response+approximation+and+its+lowest+order+corrections%3A+An+influence+functional+approach&doi=10.1021%2Fjp9847540"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">The Linear Response Approximation and Its Lowest Order Corrections: An Influence Functional Approach</span></div><div class="casAuthors">Makri, Nancy</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry B</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">103</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">2823-2829</span>CODEN:
                <span class="NLM_cas:coden">JPCBFK</span>;
        ISSN:<span class="NLM_cas:issn">1089-5647</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A cumulant expansion of the influence functional entering the path integral representation of the reduced d. matrix is presented.  Truncation of the expansion at the quadratic terms allows mapping of a many-body environment onto an effective harmonic model with the same force autocorrelation function, recovering the conventional linear response limit.  It is shown that the latter becomes exact in the thermodn. limit if the bath consists of independent modes coupled to the system via coeffs. that scale appropriately with the size of the bath.  Low-order correction terms to the linear response approxn. are found to be given by multitime correlation functions of the force exerted by the bath on the system.  Within a classical approxn., such correction terms can be obtained from mol. dynamics calcns. and incorporated in the path integral, which can be evaluated by means of iterative procedures.  Numerical calcns. on an intrinsically nonlinear bath of two-level systems confirm the above behaviors and indicate that the deviations from the linear response approxn. are more significant at low temps. where quantum mech. effects play an important role to the bath dynamics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqn6ASqpCu_KbVg90H21EOLACvtfcHk0ljexrYChyS7Vg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXitVags70%253D&md5=bf3eae88c2de1d43225304f77e24c441</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1021%2Fjp9847540&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp9847540%26sid%3Dliteratum%253Aachs%26aulast%3DMakri%26aufirst%3DN.%26atitle%3DThe%2520linear%2520response%2520approximation%2520and%2520its%2520lowest%2520order%2520corrections%253A%2520An%2520influence%2520functional%2520approach%26jtitle%3DJ.%2520Phys.%2520Chem.%2520B%26date%3D1999%26volume%3D103%26spage%3D2823%26epage%3D2829%26doi%3D10.1021%2Fjp9847540" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Hou, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, W.</span><span> </span><span class="NLM_article-title">Assessing the performance of the MM/PBSA and MM/GBSA methods. 1. The accuracy of binding free energy calculations based on molecular dynamics simulations</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">69</span><span class="NLM_x">–</span> <span class="NLM_lpage">82</span><span class="refDoi"> DOI: 10.1021/ci100275a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci100275a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVyrsrrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2011&pages=69-82&author=T.+Houauthor=J.+Wangauthor=Y.+Liauthor=W.+Wang&title=Assessing+the+performance+of+the+MM%2FPBSA+and+MM%2FGBSA+methods.+1.+The+accuracy+of+binding+free+energy+calculations+based+on+molecular+dynamics+simulations&doi=10.1021%2Fci100275a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Assessing the Performance of the MM/PBSA and MM/GBSA Methods. 1. The Accuracy of Binding Free Energy Calculations Based on Molecular Dynamics Simulations</span></div><div class="casAuthors">Hou, Tingjun; Wang, Junmei; Li, Youyong; Wang, Wei</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">69-82</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The Mol. Mechanics/Poisson-Boltzmann Surface Area (MM/PBSA) and the Mol. Mechanics/Generalized Born Surface Area (MM/GBSA) methods calc. binding free energies for macromols. by combining mol. mechanics calcns. and continuum solvation models.  To systematically evaluate the performance of these methods, the authors report here an extensive study of 59 ligands interacting with six different proteins.  First, the authors explored the effects of the length of the mol. dynamics (MD) simulation, ranging from 400 to 4800 ps, and the solute dielec. const. (1, 2, or 4) on the binding free energies predicted by MM/PBSA.  The following three important conclusions could be obsd.: (1) MD simulation length has an obvious impact on the predictions, and longer MD simulation is not always necessary to achieve better predictions.  (2) The predictions are quite sensitive to the solute dielec. const., and this parameter should be carefully detd. according to the characteristics of the protein/ligand binding interface.  (3) Conformational entropy often show large fluctuations in MD trajectories, and a large no. of snapshots are necessary to achieve stable predictions.  Next, the authors evaluated the accuracy of the binding free energies calcd. by three Generalized Born (GB) models.  The authors found that the GB model developed by Onufriev and Case was the most successful model in ranking the binding affinities of the studied inhibitors.  Finally, the authors evaluated the performance of MM/GBSA and MM/PBSA in predicting binding free energies.  The authors' results showed that MM/PBSA performed better in calcg. abs., but not necessarily relative, binding free energies than MM/GBSA.  Considering its computational efficiency, MM/GBSA can serve as a powerful tool in drug design, where correct ranking of inhibitors is often emphasized.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_kxNWs2s3grVg90H21EOLACvtfcHk0ljexrYChyS7Vg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsVyrsrrK&md5=b9f5b7fab70a6cc76b115d202d718f87</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1021%2Fci100275a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci100275a%26sid%3Dliteratum%253Aachs%26aulast%3DHou%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DW.%26atitle%3DAssessing%2520the%2520performance%2520of%2520the%2520MM%252FPBSA%2520and%2520MM%252FGBSA%2520methods.%25201.%2520The%2520accuracy%2520of%2520binding%2520free%2520energy%2520calculations%2520based%2520on%2520molecular%2520dynamics%2520simulations%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2011%26volume%3D51%26spage%3D69%26epage%3D82%26doi%3D10.1021%2Fci100275a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Shirts, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mobley, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chodera, J. D.</span><span> </span><span class="NLM_article-title">Alchemical free energy calculations: ready for prime time?</span> <span class="citation_source-journal">Annu. Rep. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">41</span><span class="NLM_x">–</span> <span class="NLM_lpage">59</span><span class="refDoi"> DOI: 10.1016/S1574-1400(07)03004-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1016%2FS1574-1400%2807%2903004-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BD1cXkt1Wnsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2007&pages=41-59&author=M.+R.+Shirtsauthor=D.+L.+Mobleyauthor=J.+D.+Chodera&title=Alchemical+free+energy+calculations%3A+ready+for+prime+time%3F&doi=10.1016%2FS1574-1400%2807%2903004-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Alchemical free energy calculations: Ready for prime time?</span></div><div class="casAuthors">Shirts, Michael R.; Mobley, David L.; Chodera, John D.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Reports in Computational Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">41-59</span>CODEN:
                <span class="NLM_cas:coden">ARCCC3</span>;
        ISSN:<span class="NLM_cas:issn">1574-1400</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">This review discusses the recent literature on methodol. improvements in free energy computation that have made this possible.  It focuses on current research using the most common rigorously exact method, that of "alchem." transformation.  In an alchem. transformation, a chem. species is transformed into another via a pathway of nonphys. (alchem.) states.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQsadHGUu_brVg90H21EOLACvtfcHk0lgKPYXVg2qcUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXkt1Wnsr8%253D&md5=e568b6f7dd6e879700946eefe0b3bfdb</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2FS1574-1400%2807%2903004-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1574-1400%252807%252903004-6%26sid%3Dliteratum%253Aachs%26aulast%3DShirts%26aufirst%3DM.%2BR.%26aulast%3DMobley%26aufirst%3DD.%2BL.%26aulast%3DChodera%26aufirst%3DJ.%2BD.%26atitle%3DAlchemical%2520free%2520energy%2520calculations%253A%2520ready%2520for%2520prime%2520time%253F%26jtitle%3DAnnu.%2520Rep.%2520Comput.%2520Chem.%26date%3D2007%26volume%3D3%26spage%3D41%26epage%3D59%26doi%3D10.1016%2FS1574-1400%2807%2903004-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Lees, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferreira, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Costa, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rocha, J.-F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oliveira, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lopes, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nunes, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soares-da-Silva, P.</span><span> </span><span class="NLM_article-title">Efficacy and safety of opicapone, a new COMT inhibitor, for the treatment of motor fluctuations in Parkinson’s disease patients: a phase III, randomized, double-blind, placebocontrolled, parallel-group study (BIPARK II)</span> <span class="citation_source-journal">J. Neurol. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">333</span><span class="NLM_x">, </span> <span class="NLM_fpage">e116</span><span class="refDoi"> DOI: 10.1016/j.jns.2013.07.391</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1016%2Fj.jns.2013.07.391" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=333&publication_year=2013&pages=e116&author=A.+Leesauthor=J.+Ferreiraauthor=R.+Costaauthor=J.-F.+Rochaauthor=C.+Oliveiraauthor=N.+Lopesauthor=T.+Nunesauthor=P.+Soares-da-Silva&title=Efficacy+and+safety+of+opicapone%2C+a+new+COMT+inhibitor%2C+for+the+treatment+of+motor+fluctuations+in+Parkinson%E2%80%99s+disease+patients%3A+a+phase+III%2C+randomized%2C+double-blind%2C+placebocontrolled%2C+parallel-group+study+%28BIPARK+II%29&doi=10.1016%2Fj.jns.2013.07.391"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.jns.2013.07.391&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jns.2013.07.391%26sid%3Dliteratum%253Aachs%26aulast%3DLees%26aufirst%3DA.%26aulast%3DFerreira%26aufirst%3DJ.%26aulast%3DCosta%26aufirst%3DR.%26aulast%3DRocha%26aufirst%3DJ.-F.%26aulast%3DOliveira%26aufirst%3DC.%26aulast%3DLopes%26aufirst%3DN.%26aulast%3DNunes%26aufirst%3DT.%26aulast%3DSoares-da-Silva%26aufirst%3DP.%26atitle%3DEfficacy%2520and%2520safety%2520of%2520opicapone%252C%2520a%2520new%2520COMT%2520inhibitor%252C%2520for%2520the%2520treatment%2520of%2520motor%2520fluctuations%2520in%2520Parkinson%25E2%2580%2599s%2520disease%2520patients%253A%2520a%2520phase%2520III%252C%2520randomized%252C%2520double-blind%252C%2520placebocontrolled%252C%2520parallel-group%2520study%2520%2528BIPARK%2520II%2529%26jtitle%3DJ.%2520Neurol.%2520Sci.%26date%3D2013%26volume%3D333%26spage%3De116%26doi%3D10.1016%2Fj.jns.2013.07.391" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Steinbrecher, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Labahn, A.</span><span> </span><span class="NLM_article-title">Towards accurate free energy calculations in ligand protein-binding studies</span> <span class="citation_source-journal">Curr. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">767</span><span class="NLM_x">–</span> <span class="NLM_lpage">785</span><span class="refDoi"> DOI: 10.2174/092986710790514453</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.2174%2F092986710790514453" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=20088755" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjs1Kkurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2010&pages=767-785&author=T.+Steinbrecherauthor=A.+Labahn&title=Towards+accurate+free+energy+calculations+in+ligand+protein-binding+studies&doi=10.2174%2F092986710790514453"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Towards accurate free energy calculations in ligand protein-binding studies</span></div><div class="casAuthors">Steinbrecher, Thomas; Labahn, Andreas</div><div class="citationInfo"><span class="NLM_cas:title">Current Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">767-785</span>CODEN:
                <span class="NLM_cas:coden">CMCHE7</span>;
        ISSN:<span class="NLM_cas:issn">0929-8673</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Cells contain a multitude of different chem. reaction paths running simultaneously and quite independently next to each other.  This amazing feat is enabled by mol. recognition, the ability of biomols. to form stable and specific complexes with each other and with their substrates.  A better understanding of this process, i.e. of the kinetics, structures and thermodn. properties of biomol. binding, would be invaluable in the study of biol. systems.  In addn., as the mode of action of many pharmaceuticals is based upon their inhibition or activation of biomol. targets, predictive models of small mol. receptor binding are very helpful tools in rational drug design.  Since the goal here is normally to design a new compd. with a high inhibition strength, one of the most important thermodn. properties is the binding free energy ΔG0.  The prediction of binding consts. has always been one of the major goals in the field of computational chem., because the ability to reliably assess a hypothetical compd.'s binding properties without having to synthesize it first would save a tremendous amt. of work.  The different approaches to this question range from fast and simple empirical descriptor methods to elaborate simulation protocols aimed at putting the computation of free energies onto a solid foundation of statistical thermodn.  While the later methods are still not suited for the screenings of thousands of compds. that are routinely performed in computational drug design studies, they are increasingly put to use for the detailed study of protein ligand interactions.  This review will focus on mol. mechanics force field based free energy calcns. and their application to the study of protein ligand interactions.  After a brief overview of other popular methods for the calcn. of free energies, we will describe recent advances in methodol. and a variety of exemplary studies of mol. dynamics simulation based free energy calcns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqIXr-1XFuGS7Vg90H21EOLACvtfcHk0lgKPYXVg2qcUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjs1Kkurw%253D&md5=e06994fb21a04ba7bbc572daa97c91b6</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.2174%2F092986710790514453&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F092986710790514453%26sid%3Dliteratum%253Aachs%26aulast%3DSteinbrecher%26aufirst%3DT.%26aulast%3DLabahn%26aufirst%3DA.%26atitle%3DTowards%2520accurate%2520free%2520energy%2520calculations%2520in%2520ligand%2520protein-binding%2520studies%26jtitle%3DCurr.%2520Med.%2520Chem.%26date%3D2010%26volume%3D17%26spage%3D767%26epage%3D785%26doi%3D10.2174%2F092986710790514453" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Jarzynski, C.</span><span> </span><span class="NLM_article-title">Nonequilibrium equality for free energy differences</span> <span class="citation_source-journal">Phys. Rev. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">78</span><span class="NLM_x">, </span> <span class="NLM_fpage">2690</span><span class="NLM_x">–</span> <span class="NLM_lpage">2693</span><span class="refDoi"> DOI: 10.1103/PhysRevLett.78.2690</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1103%2FPhysRevLett.78.2690" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADyaK2sXisVSrt7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=78&publication_year=1997&pages=2690-2693&author=C.+Jarzynski&title=Nonequilibrium+equality+for+free+energy+differences&doi=10.1103%2FPhysRevLett.78.2690"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Nonequilibrium equality for free energy differences</span></div><div class="casAuthors">Jarzynski, C.</div><div class="citationInfo"><span class="NLM_cas:title">Physical Review Letters</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">78</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">2690-2693</span>CODEN:
                <span class="NLM_cas:coden">PRLTAO</span>;
        ISSN:<span class="NLM_cas:issn">0031-9007</span>.
    
            (<span class="NLM_cas:orgname">American Physical Society</span>)
        </div><div class="casAbstract">An expression is derived for the equil. free energy difference between two configurations of a system, in terms of an ensemble of finite-time measurements of the work performed in parametrically switching from one configuration to the other.  Two well-known identities emerge as limiting cases of this result.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqsl50isjLbR7Vg90H21EOLACvtfcHk0lgKPYXVg2qcUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXisVSrt7c%253D&md5=5e572f64b7e9044fff5af96867602877</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1103%2FPhysRevLett.78.2690&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1103%252FPhysRevLett.78.2690%26sid%3Dliteratum%253Aachs%26aulast%3DJarzynski%26aufirst%3DC.%26atitle%3DNonequilibrium%2520equality%2520for%2520free%2520energy%2520differences%26jtitle%3DPhys.%2520Rev.%2520Lett.%26date%3D1997%26volume%3D78%26spage%3D2690%26epage%3D2693%26doi%3D10.1103%2FPhysRevLett.78.2690" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Jarzynski, C.</span><span> </span><span class="NLM_article-title">How does a system respond when driven away from thermal equilibrium?</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">98</span><span class="NLM_x">, </span> <span class="NLM_fpage">3636</span><span class="NLM_x">–</span> <span class="NLM_lpage">3638</span><span class="refDoi"> DOI: 10.1073/pnas.081074598</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1073%2Fpnas.081074598" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=2001&pages=3636-3638&author=C.+Jarzynski&title=How+does+a+system+respond+when+driven+away+from+thermal+equilibrium%3F&doi=10.1073%2Fpnas.081074598"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1073%2Fpnas.081074598&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.081074598%26sid%3Dliteratum%253Aachs%26aulast%3DJarzynski%26aufirst%3DC.%26atitle%3DHow%2520does%2520a%2520system%2520respond%2520when%2520driven%2520away%2520from%2520thermal%2520equilibrium%253F%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2001%26volume%3D98%26spage%3D3636%26epage%3D3638%26doi%3D10.1073%2Fpnas.081074598" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Hummer, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Szabo, A.</span><span> </span><span class="NLM_article-title">Free energy reconstruction from nonequilibrium single-molecule pulling experiments</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">98</span><span class="NLM_x">, </span> <span class="NLM_fpage">3658</span><span class="NLM_x">–</span> <span class="NLM_lpage">3661</span><span class="refDoi"> DOI: 10.1073/pnas.071034098</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1073%2Fpnas.071034098" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=11274384" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BD3MXis1Kitbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=2001&pages=3658-3661&author=G.+Hummerauthor=A.+Szabo&title=Free+energy+reconstruction+from+nonequilibrium+single-molecule+pulling+experiments&doi=10.1073%2Fpnas.071034098"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Free energy reconstruction from nonequilibrium single-molecule pulling experiments</span></div><div class="casAuthors">Hummer, Gerhard; Szabo, Attila</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">98</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3658-3661</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Laser tweezers and at. force microscopes are increasingly used to probe the interactions and mech. properties of individual mols.  Unfortunately, using such time-dependent perturbations to force rare mol. events also drives the system away from equil.  Nevertheless, we show how equil. free energy profiles can be extd. rigorously from repeated nonequil. force measurements on the basis of an extension of Jarzynski's remarkable identity between free energies and the irreversible work.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrs8JOlO8In77Vg90H21EOLACvtfcHk0ljqRT9OXJPcug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXis1Kitbk%253D&md5=b8b557f038efd3d60f9a97881636c120</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1073%2Fpnas.071034098&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.071034098%26sid%3Dliteratum%253Aachs%26aulast%3DHummer%26aufirst%3DG.%26aulast%3DSzabo%26aufirst%3DA.%26atitle%3DFree%2520energy%2520reconstruction%2520from%2520nonequilibrium%2520single-molecule%2520pulling%2520experiments%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2001%26volume%3D98%26spage%3D3658%26epage%3D3661%26doi%3D10.1073%2Fpnas.071034098" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Shirts, M. R.; Mobley, D. L.; Brown, S. P.</span><span> </span><span class="NLM_article-title">Free Energy Calculations In Structure-Based Drug Design</span>. In  <span class="citation_source-book">Drug Design: Structure- and Ligand-Based Approaches</span>; <span class="NLM_publisher-name">Cambridge University Press</span>,<span class="NLM_x"> </span><span class="NLM_year">2010</span><span class="NLM_x">; </span>pp  <span class="NLM_fpage">61</span><span class="NLM_x">–</span> <span class="NLM_lpage">86</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1017%2FCBO9780511730412.007" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2010&pages=61-86&author=M.+R.+Shirts&author=D.+L.+Mobley&author=S.+P.+Brown&title=Drug+Design%3A+Structure-+and+Ligand-Based+Approaches"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1017%2FCBO9780511730412.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1017%252FCBO9780511730412.007%26sid%3Dliteratum%253Aachs%26aulast%3DShirts%26aufirst%3DM.%2BR.%26atitle%3DFree%2520Energy%2520Calculations%2520In%2520Structure-Based%2520Drug%2520Design%26btitle%3DDrug%2520Design%253A%2520Structure-%2520and%2520Ligand-Based%2520Approaches%26pub%3DCambridge%2520University%2520Press%26date%3D2010%26spage%3D61%26epage%3D86" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Allen, M. P.</span><span> </span><span class="NLM_article-title">Introduction to Molecular Dynamics Simulation</span>. In  <span class="citation_source-book">Computational Soft Matter: From Synthetic Polymers to Proteins</span>; NIC Series, Vol. 23; <span class="NLM_publisher-name">NIC</span>,<span class="NLM_x"> </span><span class="NLM_year">2004</span><span class="NLM_x">; </span>pp  <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">28</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2004&pages=1-28&author=M.+P.+Allen&title=Computational+Soft+Matter%3A+From+Synthetic+Polymers+to+Proteins"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DAllen%26aufirst%3DM.%2BP.%26atitle%3DIntroduction%2520to%2520Molecular%2520Dynamics%2520Simulation%26btitle%3DComputational%2520Soft%2520Matter%253A%2520From%2520Synthetic%2520Polymers%2520to%2520Proteins%26pub%3DNIC%26date%3D2004%26spage%3D1%26epage%3D28" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Bernardi, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Melo, M. C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schulten, K.</span><span> </span><span class="NLM_article-title">Enhanced sampling techniques in molecular dynamics simulations of biological systems</span> <span class="citation_source-journal">Biochim. Biophys. Acta, Gen. Subj.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">1850</span><span class="NLM_x">, </span> <span class="NLM_fpage">872</span><span class="NLM_x">–</span> <span class="NLM_lpage">877</span><span class="refDoi"> DOI: 10.1016/j.bbagen.2014.10.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1016%2Fj.bbagen.2014.10.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=25450171" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVanurbN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1850&publication_year=2015&pages=872-877&author=R.+C.+Bernardiauthor=M.+C.+R.+Meloauthor=K.+Schulten&title=Enhanced+sampling+techniques+in+molecular+dynamics+simulations+of+biological+systems&doi=10.1016%2Fj.bbagen.2014.10.019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Enhanced sampling techniques in molecular dynamics simulations of biological systems</span></div><div class="casAuthors">Bernardi, Rafael C.; Melo, Marcelo C. R.; Schulten, Klaus</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, General Subjects</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">1850</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">872-877</span>CODEN:
                <span class="NLM_cas:coden">BBGSB3</span>;
        ISSN:<span class="NLM_cas:issn">0304-4165</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Mol. dynamics has emerged as an important research methodol. covering systems to the level of millions of atoms.  However, insufficient sampling often limits its application.  The limitation is due to rough energy landscapes, with many local min. sepd. by high-energy barriers, which govern the biomol. motion.  In the past few decades methods have been developed that address the sampling problem, such as replica-exchange mol. dynamics, metadynamics and simulated annealing.  Here the authors present an overview over theses sampling methods in an attempt to shed light on which should be selected depending on the type of system property studied.  Enhanced sampling methods have been employed for a broad range of biol. systems and the choice of a suitable method is connected to biol. and phys. characteristics of the system, in particular system size.  While metadynamics and replica-exchange mol. dynamics are the most adopted sampling methods to study biomol. dynamics, simulated annealing is well suited to characterize very flexible systems.  The use of annealing methods for a long time was restricted to simulation of small proteins; however, a variant of the method, generalized simulated annealing, can be employed at a relatively low computational cost to large macromol. complexes.  Mol. dynamics trajectories frequently do not reach all relevant conformational substates, for example those connected with biol. function, a problem that can be addressed by employing enhanced sampling algorithms.  This article is part of a Special Issue entitled Recent developments of mol. dynamics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDLxzIg4zKM7Vg90H21EOLACvtfcHk0ljqRT9OXJPcug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVanurbN&md5=44e3ac1aa042c35ee08aa86a6a63e78e</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.bbagen.2014.10.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbagen.2014.10.019%26sid%3Dliteratum%253Aachs%26aulast%3DBernardi%26aufirst%3DR.%2BC.%26aulast%3DMelo%26aufirst%3DM.%2BC.%2BR.%26aulast%3DSchulten%26aufirst%3DK.%26atitle%3DEnhanced%2520sampling%2520techniques%2520in%2520molecular%2520dynamics%2520simulations%2520of%2520biological%2520systems%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Gen.%2520Subj.%26date%3D2015%26volume%3D1850%26spage%3D872%26epage%3D877%26doi%3D10.1016%2Fj.bbagen.2014.10.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Harrison, R. L.</span><span> </span><span class="NLM_article-title">Introduction to monte carlo simulation</span> <span class="citation_source-journal">AIP Conf. Proc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">1204</span><span class="NLM_x">, </span> <span class="NLM_fpage">17</span><span class="NLM_x">–</span> <span class="NLM_lpage">21</span><span class="refDoi"> DOI: 10.1063/1.3295638</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1063%2F1.3295638" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=20733932" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A280%3ADC%252BC2srit1KmtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1204&publication_year=2010&pages=17-21&author=R.+L.+Harrison&title=Introduction+to+monte+carlo+simulation&doi=10.1063%2F1.3295638"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Introduction To Monte Carlo Simulation</span></div><div class="casAuthors">Harrison Robert L</div><div class="citationInfo"><span class="NLM_cas:title">AIP conference proceedings</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">1204</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">17-21</span>
        ISSN:<span class="NLM_cas:issn">0094-243X</span>.
    </div><div class="casAbstract">This paper reviews the history and principles of Monte Carlo simulation, emphasizing techniques commonly used in the simulation of medical imaging.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSk_aq2nZIvSIHWj6b1v8uTfW6udTcc2eYU28T0lZK1I7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2srit1KmtA%253D%253D&md5=3b7d6f9770f1c9b9032f16f89805845f</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1063%2F1.3295638&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.3295638%26sid%3Dliteratum%253Aachs%26aulast%3DHarrison%26aufirst%3DR.%2BL.%26atitle%3DIntroduction%2520to%2520monte%2520carlo%2520simulation%26jtitle%3DAIP%2520Conf.%2520Proc.%26date%3D2010%26volume%3D1204%26spage%3D17%26epage%3D21%26doi%3D10.1063%2F1.3295638" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Reif, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hunenberger, P. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oostenbrink, C.</span><span> </span><span class="NLM_article-title">New interaction parameters for charged amino acid side chains in the GROMOS force field</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">3705</span><span class="NLM_x">–</span> <span class="NLM_lpage">3723</span><span class="refDoi"> DOI: 10.1021/ct300156h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct300156h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC38XlsVaksbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2012&pages=3705-3723&author=M.+M.+Reifauthor=P.+H.+Hunenbergerauthor=C.+Oostenbrink&title=New+interaction+parameters+for+charged+amino+acid+side+chains+in+the+GROMOS+force+field&doi=10.1021%2Fct300156h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">New Interaction Parameters for Charged Amino Acid Side Chains in the GROMOS Force Field</span></div><div class="casAuthors">Reif, Maria M.; Huenenberger, Philippe H.; Oostenbrink, Chris</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3705-3723</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A GROMOS force-field parameter set 54A8 is developed, which is based on the latest 54A7 set and involves a recalibration of the nonbonded interaction parameters for the charged amino acid side chains, based on ionic side chain analogs.  After a thorough anal. of the available exptl. data, conventional hydration free energies for the ammonium; mono-, di-, tri-, and tetramethyl-ammonium; formate; acetate; propanoate; imidazolium; and guanidinium ions are combined with a std. abs. intrinsic proton hydration free energy ΔGhyd-[Hg+] = -1100 kJ·mol-1 to yield abs. intrinsic single-ion hydration free energies serving as exptl. target data.  The raw hydration free energies calcd. from atomistic simulations are affected by electrostatic and finite-size artifacts, and corrections are applied to reach methodol. independence prior to comparison with these exptl. values.  Except for monomethyl-ammonium, ions with parameters derived directly from the 54A7 force field considerably underestimate (ammonium, formate, acetate, propanoate, guanidinium) or overestimate (di-, tri-, and tetramethyl-ammonium; imidazolium) the magnitude of the intrinsic hydration free energy, the largest deviation affecting the acetate ion (40.0 kJ·mol-1).  After reparameterization into 54A8, the mean and maximal abs. deviations between simulated and exptl. data over the set of 10 ions are reduced from 23.1 and 40.0 kJ·mol-1, resp., to 1.8 and 6.3 kJ·mol-1, resp.  Although the 54A7 and 54A8 parameter sets differ significantly in terms of the hydration free energies of the ions considered, other properties such as ion-water radial distribution functions and ion-ion potentials of mean force appear to be only moderately sensitive to this change.  These properties are similar for the two sets and, in the case of the ion-water radial distribution functions, in good agreement with available exptl. data.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUrAQr3bSDa7Vg90H21EOLACvtfcHk0liGV8hQUPKBmg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XlsVaksbg%253D&md5=04f0f93b2985e1a8680498477eacfbc0</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Fct300156h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct300156h%26sid%3Dliteratum%253Aachs%26aulast%3DReif%26aufirst%3DM.%2BM.%26aulast%3DHunenberger%26aufirst%3DP.%2BH.%26aulast%3DOostenbrink%26aufirst%3DC.%26atitle%3DNew%2520interaction%2520parameters%2520for%2520charged%2520amino%2520acid%2520side%2520chains%2520in%2520the%2520GROMOS%2520force%2520field%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2012%26volume%3D8%26spage%3D3705%26epage%3D3723%26doi%3D10.1021%2Fct300156h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Lindorff-Larsen, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Piana, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Palmo, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maragakis, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klepeis, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dror, R. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaw, D. E.</span><span> </span><span class="NLM_article-title">Improved side-chain torsion potentials for the Amber ff99SB protein force field</span> <span class="citation_source-journal">Proteins: Struct., Funct., Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">78</span><span class="NLM_x">, </span> <span class="NLM_fpage">1950</span><span class="NLM_x">–</span> <span class="NLM_lpage">1958</span><span class="refDoi"> DOI: 10.1002/prot.22711</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1002%2Fprot.22711" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=20408171" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC3cXkvFegtLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=78&publication_year=2010&pages=1950-1958&author=K.+Lindorff-Larsenauthor=S.+Pianaauthor=K.+Palmoauthor=P.+Maragakisauthor=J.+L.+Klepeisauthor=R.+O.+Drorauthor=D.+E.+Shaw&title=Improved+side-chain+torsion+potentials+for+the+Amber+ff99SB+protein+force+field&doi=10.1002%2Fprot.22711"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Improved side-chain torsion potentials for the Amber ff99SB protein force field</span></div><div class="casAuthors">Lindorff-Larsen, Kresten; Piana, Stefano; Palmo, Kim; Maragakis, Paul; Klepeis, John L.; Dror, Ron O.; Shaw, David E.</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Bioinformatics</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">78</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1950-1958</span>CODEN:
                <span class="NLM_cas:coden">PSFBAF</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">Recent advances in hardware and software have enabled increasingly long mol. dynamics (MD) simulations of biomols., exposing certain limitations in the accuracy of the force fields used for such simulations and spurring efforts to refine these force fields.  Recent modifications to the Amber and CHARMM protein force fields, for example, have improved the backbone torsion potentials, remedying deficiencies in earlier versions.  Here, the authors further advance simulation accuracy by improving the amino acid side-chain torsion potentials of the Amber ff99SB force field.  First, the authors used simulations of model alpha-helical systems to identify the four residue types whose rotamer distribution differed the most from expectations based on Protein Data Bank statistics.  Second, the authors optimized the side-chain torsion potentials of these residues to match new, high-level quantum-mech. calcns.  Finally, the authors used microsecond-timescale MD simulations in explicit solvent to validate the resulting force field against a large set of exptl. NMR measurements that directly probe side-chain conformations.  The new force field, which the authors have termed Amber ff99SB-ILDN, exhibits considerably better agreement with the NMR data.  Proteins 2010. © 2010 Wiley-Liss, Inc.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrNFHrYs9024bVg90H21EOLACvtfcHk0liGV8hQUPKBmg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXkvFegtLo%253D&md5=447a9004026e2b93f0f7beff165daa09</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1002%2Fprot.22711&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.22711%26sid%3Dliteratum%253Aachs%26aulast%3DLindorff-Larsen%26aufirst%3DK.%26aulast%3DPiana%26aufirst%3DS.%26aulast%3DPalmo%26aufirst%3DK.%26aulast%3DMaragakis%26aufirst%3DP.%26aulast%3DKlepeis%26aufirst%3DJ.%2BL.%26aulast%3DDror%26aufirst%3DR.%2BO.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26atitle%3DImproved%2520side-chain%2520torsion%2520potentials%2520for%2520the%2520Amber%2520ff99SB%2520protein%2520force%2520field%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D2010%26volume%3D78%26spage%3D1950%26epage%3D1958%26doi%3D10.1002%2Fprot.22711" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Maier, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martinez, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kasavajhala, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wickstrom, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hauser, K. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simmerling, C.</span><span> </span><span class="NLM_article-title">ff14SB: Improving the accuracy of protein side chain and backbone parameters from ff99SB</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">3696</span><span class="NLM_x">–</span> <span class="NLM_lpage">3713</span><span class="refDoi"> DOI: 10.1021/acs.jctc.5b00255</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jctc.5b00255" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFequ7rN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=3696-3713&author=J.+A.+Maierauthor=C.+Martinezauthor=K.+Kasavajhalaauthor=L.+Wickstromauthor=K.+E.+Hauserauthor=C.+Simmerling&title=ff14SB%3A+Improving+the+accuracy+of+protein+side+chain+and+backbone+parameters+from+ff99SB&doi=10.1021%2Facs.jctc.5b00255"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">ff14SB: Improving the Accuracy of Protein Side Chain and Backbone Parameters from ff99SB</span></div><div class="casAuthors">Maier, James A.; Martinez, Carmenza; Kasavajhala, Koushik; Wickstrom, Lauren; Hauser, Kevin E.; Simmerling, Carlos</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3696-3713</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mol. mechanics is powerful for its speed in atomistic simulations, but an accurate force field is required.  The Amber ff99SB force field improved protein secondary structure balance and dynamics from earlier force fields like ff99, but weaknesses in side chain rotamer and backbone secondary structure preferences have been identified.  Here, we performed a complete refit of all amino acid side chain dihedral parameters, which had been carried over from ff94.  The training set of conformations included multidimensional dihedral scans designed to improve transferability of the parameters.  Improvement in all amino acids was obtained as compared to ff99SB.  Parameters were also generated for alternate protonation states of ionizable side chains.  Av. errors in relative energies of pairs of conformations were under 1.0 kcal/mol as compared to QM, reduced 35% from ff99SB.  We also took the opportunity to make empirical adjustments to the protein backbone dihedral parameters as compared to ff99SB.  Multiple small adjustments of φ and ψ parameters were tested against NMR scalar coupling data and secondary structure content for short peptides.  The best results were obtained from a phys. motivated adjustment to the φ rotational profile that compensates for lack of ff99SB QM training data in the β-ppII transition region.  Together, these backbone and side chain modifications (hereafter called ff14SB) not only better reproduced their benchmarks, but also improved secondary structure content in small peptides and reprodn. of NMR χ1 scalar coupling measurements for proteins in soln.  We also discuss the Amber ff12SB parameter set, a preliminary version of ff14SB that includes most of its improvements.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgntnN9ES97rVg90H21EOLACvtfcHk0liGV8hQUPKBmg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFequ7rN&md5=7b803577b3b6912cc6750cfbd356596e</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Facs.jctc.5b00255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jctc.5b00255%26sid%3Dliteratum%253Aachs%26aulast%3DMaier%26aufirst%3DJ.%2BA.%26aulast%3DMartinez%26aufirst%3DC.%26aulast%3DKasavajhala%26aufirst%3DK.%26aulast%3DWickstrom%26aufirst%3DL.%26aulast%3DHauser%26aufirst%3DK.%2BE.%26aulast%3DSimmerling%26aufirst%3DC.%26atitle%3Dff14SB%253A%2520Improving%2520the%2520accuracy%2520of%2520protein%2520side%2520chain%2520and%2520backbone%2520parameters%2520from%2520ff99SB%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2015%26volume%3D11%26spage%3D3696%26epage%3D3713%26doi%3D10.1021%2Facs.jctc.5b00255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Lee, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Swails, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yeom, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eastman, P. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lemkul, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wei, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buckner, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jeong, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qi, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jo, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pande, V. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Case, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brooks, C. L.,  III</span><span class="NLM_x">; </span><span class="NLM_contrib-group">MacKerell, A. D.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klauda, J. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Im, W.</span><span> </span><span class="NLM_article-title">CHARMM-GUI input generator for NAMD, GROMACS, AMBER, OpenMM, and CHARMM/OpenMM simulations using the CHARMM36 additive force field</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">405</span><span class="NLM_x">–</span> <span class="NLM_lpage">413</span><span class="refDoi"> DOI: 10.1021/acs.jctc.5b00935</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jctc.5b00935" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVWru7nI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=405-413&author=J.+Leeauthor=X.+Chengauthor=J.+M.+Swailsauthor=M.+S.+Yeomauthor=P.+K.+Eastmanauthor=J.+A.+Lemkulauthor=S.+Weiauthor=J.+Bucknerauthor=J.+C.+Jeongauthor=Y.+Qiauthor=S.+Joauthor=V.+S.+Pandeauthor=D.+A.+Caseauthor=C.+L.+Brooksauthor=A.+D.+MacKerellauthor=J.+B.+Klaudaauthor=W.+Im&title=CHARMM-GUI+input+generator+for+NAMD%2C+GROMACS%2C+AMBER%2C+OpenMM%2C+and+CHARMM%2FOpenMM+simulations+using+the+CHARMM36+additive+force+field&doi=10.1021%2Facs.jctc.5b00935"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">CHARMM-GUI Input Generator for NAMD, GROMACS, AMBER, OpenMM, and CHARMM/OpenMM Simulations Using the CHARMM36 Additive Force Field</span></div><div class="casAuthors">Lee, Jumin; Cheng, Xi; Swails, Jason M.; Yeom, Min Sun; Eastman, Peter K.; Lemkul, Justin A.; Wei, Shuai; Buckner, Joshua; Jeong, Jong Cheol; Qi, Yifei; Jo, Sunhwan; Pande, Vijay S.; Case, David A.; Brooks, Charles L.; MacKerell, Alexander D.; Klauda, Jeffery B.; Im, Wonpil</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">405-413</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Proper treatment of nonbonded interactions is essential for the accuracy of mol. dynamics (MD) simulations, esp. in studies of lipid bilayers.  The use of the CHARMM36 force field (C36 FF) in different MD simulation programs can result in disagreements with published simulations performed with CHARMM due to differences in the protocols used to treat the long-range and 1-4 nonbonded interactions.  In this study, we systematically test the use of the C36 lipid FF in NAMD, GROMACS, AMBER, OpenMM, and CHARMM/OpenMM.  A wide range of Lennard-Jones (LJ) cutoff schemes and integrator algorithms were tested to find the optimal simulation protocol to best match bilayer properties of six lipids with varying acyl chain satn. and head groups.  MD simulations of a 1,2-dipalmitoyl-sn-phosphatidylcholine (DPPC) bilayer were used to obtain the optimal protocol for each program.  MD simulations with all programs were found to reasonably match the DPPC bilayer properties (surface area per lipid, chain order parameters, and area compressibility modulus) obtained using the std. protocol used in CHARMM as well as from expts.  The optimal simulation protocol was then applied to the other five lipid simulations and resulted in excellent agreement between results from most simulation programs as well as with exptl. data.  AMBER compared least favorably with the expected membrane properties, which appears to be due to its use of the hard-truncation in the LJ potential vs. a force-based switching function used to smooth the LJ potential as it approaches the cutoff distance.  The optimal simulation protocol for each program has been implemented in CHARMM-GUI.  This protocol is expected to be applicable to the remainder of the additive C36 FF including the proteins, nucleic acids, carbohydrates, and small mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrioKQ5MoVZkrVg90H21EOLACvtfcHk0liRGBUC9Gj-Iw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVWru7nI&md5=1e986f2c205aca8fb80442bc9c95b229</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Facs.jctc.5b00935&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jctc.5b00935%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DJ.%26aulast%3DCheng%26aufirst%3DX.%26aulast%3DSwails%26aufirst%3DJ.%2BM.%26aulast%3DYeom%26aufirst%3DM.%2BS.%26aulast%3DEastman%26aufirst%3DP.%2BK.%26aulast%3DLemkul%26aufirst%3DJ.%2BA.%26aulast%3DWei%26aufirst%3DS.%26aulast%3DBuckner%26aufirst%3DJ.%26aulast%3DJeong%26aufirst%3DJ.%2BC.%26aulast%3DQi%26aufirst%3DY.%26aulast%3DJo%26aufirst%3DS.%26aulast%3DPande%26aufirst%3DV.%2BS.%26aulast%3DCase%26aufirst%3DD.%2BA.%26aulast%3DBrooks%26aufirst%3DC.%2BL.%26aulast%3DMacKerell%26aufirst%3DA.%2BD.%26aulast%3DKlauda%26aufirst%3DJ.%2BB.%26aulast%3DIm%26aufirst%3DW.%26atitle%3DCHARMM-GUI%2520input%2520generator%2520for%2520NAMD%252C%2520GROMACS%252C%2520AMBER%252C%2520OpenMM%252C%2520and%2520CHARMM%252FOpenMM%2520simulations%2520using%2520the%2520CHARMM36%2520additive%2520force%2520field%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2016%26volume%3D12%26spage%3D405%26epage%3D413%26doi%3D10.1021%2Facs.jctc.5b00935" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Reif, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Winger, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oostenbrink, C.</span><span> </span><span class="NLM_article-title">Testing of the GROMOS force-field parameter set 54A8: structural properties of electrolyte solutions, lipid bilayers, and proteins</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">1247</span><span class="NLM_x">–</span> <span class="NLM_lpage">1264</span><span class="refDoi"> DOI: 10.1021/ct300874c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct300874c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC3sXitVektw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=1247-1264&author=M.+M.+Reifauthor=M.+Wingerauthor=C.+Oostenbrink&title=Testing+of+the+GROMOS+force-field+parameter+set+54A8%3A+structural+properties+of+electrolyte+solutions%2C+lipid+bilayers%2C+and+proteins&doi=10.1021%2Fct300874c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Testing of the GROMOS Force-Field Parameter Set 54A8: Structural Properties of Electrolyte Solutions, Lipid Bilayers, and Proteins</span></div><div class="casAuthors">Reif, Maria M.; Winger, Moritz; Oostenbrink, Chris</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">1247-1264</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The GROMOS 54A8 force field is the first of its kind to contain nonbonded parameters for charged amino acid side chains that are derived in a rigorously thermodn. fashion, namely a calibration against single-ion hydration free energies.  Considering charged moieties in soln., the most decisive signature of the GROMOS 54A8 force field in comparison to its predecessor 54A7 can probably be found in the thermodn. equil. between salt-bridged ion pair formation and hydration.  Possible shifts in this equil. might crucially affect the properties of electrolyte solns. or/and the stability of (bio)mols.  It is therefore important to investigate the consequences of the altered description of charged oligoat. species in the GROMOS 54A8 force field.  The present study focuses on examg. the ability of the GROMOS 54A8 force field to accurately model the structural properties of electrolyte solns., lipid bilayers, and proteins.  It is found that (i) aq. electrolytes involving oligoat. species (sodium acetate, methylammonium chloride, guanidinium chloride) reproduce exptl. salt activity derivs. for concns. up to 1.0 m (1.0-m) very well, and good agreement between simulated and exptl. data is also reached for sodium acetate and methylammonium chloride at 2.0 m concn., while not even qual. agreement is found for sodium chloride throughout the whole range of examd. concns., indicating a failure of the GROMOS 54A7 and 54A8 force-field parameter sets to correctly account for the balance between ion-ion and ion-water binding propensities of sodium and chloride ions; (ii) the GROMOS 54A8 force field reproduces the liq. cryst.-like phase of a hydrated DPPC bilayer at a pressure of 1 bar and a temp. of 323 K, the area per lipid being in agreement with exptl. data, whereas other structural properties (vol. per lipid, bilayer thickness) appear underestimated; (iii) the secondary structure of a range of different proteins simulated with the GROMOS 54A8 force field at pH 7 is maintained and compatible with exptl. NMR data, while, as also obsd. for the GROMOS 54A7 force field, α-helixes are slightly overstabilized with respect to 310-helixes; (iv) with the GROMOS 54A8 force field, the side chains of arginine, lysine, aspartate, and glutamate residues appear slightly more hydrated and present a slight excess of oppositely-charged s.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4C9UOi98oabVg90H21EOLACvtfcHk0liRGBUC9Gj-Iw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXitVektw%253D%253D&md5=d5093a833a5de57591e4f620814d813a</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Fct300874c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct300874c%26sid%3Dliteratum%253Aachs%26aulast%3DReif%26aufirst%3DM.%2BM.%26aulast%3DWinger%26aufirst%3DM.%26aulast%3DOostenbrink%26aufirst%3DC.%26atitle%3DTesting%2520of%2520the%2520GROMOS%2520force-field%2520parameter%2520set%252054A8%253A%2520structural%2520properties%2520of%2520electrolyte%2520solutions%252C%2520lipid%2520bilayers%252C%2520and%2520proteins%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2013%26volume%3D9%26spage%3D1247%26epage%3D1264%26doi%3D10.1021%2Fct300874c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Malde, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mark, A. E.</span><span> </span><span class="NLM_article-title">179 Validation of ligands in x-ray crystal structures</span> <span class="citation_source-journal">J. Biomol. Struct. Dyn.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">33</span><span class="NLM_x"> (</span><span class="NLM_issue">Suppl. 1</span><span class="NLM_x">) </span> <span class="NLM_fpage">117</span><span class="NLM_x">–</span> <span class="NLM_lpage">177</span><span class="refDoi"> DOI: 10.1080/07391102.2015.1032816</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1080%2F07391102.2015.1032816" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=26103389" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A280%3ADC%252BC2MbmsFGnsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2015&pages=117-177&issue=Suppl.+1&author=A.+K.+Maldeauthor=A.+E.+Mark&title=179+Validation+of+ligands+in+x-ray+crystal+structures&doi=10.1080%2F07391102.2015.1032816"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">179 Validation of ligands in X-ray crystal structures</span></div><div class="casAuthors">Malde Alpeshkumar K; Mark Alan E</div><div class="citationInfo"><span class="NLM_cas:title">Journal of biomolecular structure & dynamics</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">33 Suppl 1</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">117-77</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTxJkC0jZN2SDslzEEsg76ufW6udTcc2eZOXAD9PWt5vbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2MbmsFGnsg%253D%253D&md5=9d96f2720f75a6aaa7c6de1483bd23a3</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1080%2F07391102.2015.1032816&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F07391102.2015.1032816%26sid%3Dliteratum%253Aachs%26aulast%3DMalde%26aufirst%3DA.%2BK.%26aulast%3DMark%26aufirst%3DA.%2BE.%26atitle%3D179%2520Validation%2520of%2520ligands%2520in%2520x-ray%2520crystal%2520structures%26jtitle%3DJ.%2520Biomol.%2520Struct.%2520Dyn.%26date%3D2015%26volume%3D33%26issue%3DSuppl.%25201%26spage%3D117%26epage%3D177%26doi%3D10.1080%2F07391102.2015.1032816" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Malde, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zuo, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Breeze, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stroet, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poger, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nair, P. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oostenbrink, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mark, A. E.</span><span> </span><span class="NLM_article-title">An automated force field topology builder (ATB) and repository: version 1.0</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">4026</span><span class="NLM_x">–</span> <span class="NLM_lpage">4037</span><span class="refDoi"> DOI: 10.1021/ct200196m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct200196m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlOksrfO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=4026-4037&author=A.+K.+Maldeauthor=L.+Zuoauthor=M.+Breezeauthor=M.+Stroetauthor=D.+Pogerauthor=P.+C.+Nairauthor=C.+Oostenbrinkauthor=A.+E.+Mark&title=An+automated+force+field+topology+builder+%28ATB%29+and+repository%3A+version+1.0&doi=10.1021%2Fct200196m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">An Automated Force Field Topology Builder (ATB) and Repository: Version 1.0</span></div><div class="casAuthors">Malde, Alpeshkumr K.; Zuo, Le; Breeze, Matthew; Stroet, Martin; Poger, David; Nair, Pramod C.; Oostenbrink, Chris; Mark, Alan F.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">4026-4037</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The Automated force field Topol. Builder (ATB, http://compbio.biosci.uq.edu.au/atb) is a Web-accessible server that can provide topologies and parameters for a wide range of mols. appropriate for use in mol. simulations, computational drug design, and X-ray refinement.  The ATB has three primary functions: (1) to act as a repository for mols. that have been parametrized as part of the GROMOS family of force fields, (2) to act as a repository for pre-equilibrated systems for use as starting configurations in mol. dynamics simulations (solvent mixts., lipid systems pre-equilibrated to adopt a specific phase, etc.), and (3) to generate force field descriptions of novel mols. compatible with the GROMOS family of force fields in a variety of formats (GROMOS, GROMACS, and CNS).  Force field descriptions of novel mols. are derived using a multistep process in which results from quantum mech. (QM) calcns. are combined with a knowledge-based approach to ensure compatibility (as far as possible) with a specific parameter set of the GROMOS force field.  The ATB has several unique features: (1) It requires that the user stipulate the protonation and tautomeric states of the mol. (2) The symmetry of the mol. is analyzed to ensure that equiv. atoms are assigned identical parameters. (3) Charge groups are assigned automatically. (4) Where the assignment of a given parameter is ambiguous, a range of possible alternatives is provided.  The ATB also provides several validation tools to assist the user to assess the degree to which the topol. generated may be appropriate for a given task.  In addn. to detailing the steps involved in generating a force field topol. compatible with a specific GROMOS parameter set (GROMOS 53A6), the challenges involved in the automatic generation of force field parameters for at. simulations in general are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpGdRdHvU2-PLVg90H21EOLACvtfcHk0lgdj0-d1uPbiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlOksrfO&md5=172110d6b41762f68c5238e7ea127e1d</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Fct200196m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct200196m%26sid%3Dliteratum%253Aachs%26aulast%3DMalde%26aufirst%3DA.%2BK.%26aulast%3DZuo%26aufirst%3DL.%26aulast%3DBreeze%26aufirst%3DM.%26aulast%3DStroet%26aufirst%3DM.%26aulast%3DPoger%26aufirst%3DD.%26aulast%3DNair%26aufirst%3DP.%2BC.%26aulast%3DOostenbrink%26aufirst%3DC.%26aulast%3DMark%26aufirst%3DA.%2BE.%26atitle%3DAn%2520automated%2520force%2520field%2520topology%2520builder%2520%2528ATB%2529%2520and%2520repository%253A%2520version%25201.0%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2011%26volume%3D7%26spage%3D4026%26epage%3D4037%26doi%3D10.1021%2Fct200196m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Koziara, K. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stroet, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Malde, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mark, A. E.</span><span> </span><span class="NLM_article-title">Testing and validation of the automated topology builder (ATB) version 2.0: prediction of hydration free enthalpies</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">221</span><span class="NLM_x">–</span> <span class="NLM_lpage">233</span><span class="refDoi"> DOI: 10.1007/s10822-014-9713-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1007%2Fs10822-014-9713-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=24477799" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1WqtbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2014&pages=221-233&author=K.+B.+Koziaraauthor=M.+Stroetauthor=A.+K.+Maldeauthor=A.+E.+Mark&title=Testing+and+validation+of+the+automated+topology+builder+%28ATB%29+version+2.0%3A+prediction+of+hydration+free+enthalpies&doi=10.1007%2Fs10822-014-9713-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Testing and validation of the Automated Topology Builder (ATB) version 2.0: prediction of hydration free enthalpies</span></div><div class="casAuthors">Koziara, Katarzyna B.; Stroet, Martin; Malde, Alpeshkumar K.; Mark, Alan E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">221-233</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">To test and validate the Automated force field Topol. Builder and Repository (ATB; http://compbio.biosci.uq.edu.au/atb/) the hydration free enthalpies for a set of 214 drug-like mols., including 47 mols. that form part of the SAMPL4 challenge have been estd. using thermodn. integration and compared to expt.  The calcns. were performed using a fully automated protocol that incorporated a dynamic anal. of the convergence and integration error in the selection of intermediate points.  The system has been designed and implemented such that hydration free enthalpies can be obtained without manual intervention following the submission of a mol. to the ATB.  The overall av. unsigned error (AUE) using ATB 2.0 topologies for the complete set of 214 mols. was 6.7 kJ/mol and for mols. within the SAMPL4 7.5 kJ/mol.  The root mean square error (RMSE) was 9.5 and 10.0 kJ/mol resp.  However, for mols. contg. functional groups that form part of the main GROMOS force field the AUE was 3.4 kJ/mol and the RMSE was 4.0 kJ/mol.  This suggests it will be possible to further refine the parameters provided by the ATB based on hydration free enthalpies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpoglyAPJdgQbVg90H21EOLACvtfcHk0lgdj0-d1uPbiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1WqtbY%253D&md5=a3ebbc9492be9454290a9a64706640b1</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1007%2Fs10822-014-9713-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-014-9713-7%26sid%3Dliteratum%253Aachs%26aulast%3DKoziara%26aufirst%3DK.%2BB.%26aulast%3DStroet%26aufirst%3DM.%26aulast%3DMalde%26aufirst%3DA.%2BK.%26aulast%3DMark%26aufirst%3DA.%2BE.%26atitle%3DTesting%2520and%2520validation%2520of%2520the%2520automated%2520topology%2520builder%2520%2528ATB%2529%2520version%25202.0%253A%2520prediction%2520of%2520hydration%2520free%2520enthalpies%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2014%26volume%3D28%26spage%3D221%26epage%3D233%26doi%3D10.1007%2Fs10822-014-9713-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Wang, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kollman, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Case, D. A.</span><span> </span><span class="NLM_article-title">Automatic atom type and bond type perception in molecular mechanical calculations</span> <span class="citation_source-journal">J. Mol. Graphics Modell.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">247</span><span class="NLM_x">–</span> <span class="NLM_lpage">260</span><span class="refDoi"> DOI: 10.1016/j.jmgm.2005.12.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1016%2Fj.jmgm.2005.12.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=16458552" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BD28Xps1Gis7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2006&pages=247-260&author=J.+M.+Wangauthor=W.+Wangauthor=P.+A.+Kollmanauthor=D.+A.+Case&title=Automatic+atom+type+and+bond+type+perception+in+molecular+mechanical+calculations&doi=10.1016%2Fj.jmgm.2005.12.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Automatic atom type and bond type perception in molecular mechanical calculations</span></div><div class="casAuthors">Wang, Junmei; Wang, Wei; Kollman, Peter A.; Case, David A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Graphics & Modelling</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">247-260</span>CODEN:
                <span class="NLM_cas:coden">JMGMFI</span>;
        ISSN:<span class="NLM_cas:issn">1093-3263</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">In mol. mechanics (MM) studies, atom types and/or bond types of mols. are needed to det. prior to energy calcns.  The authors present here an automatic algorithm of perceiving atom types that are defined in a description table, and an automatic algorithm of assigning bond types just based on at. connectivity.  The algorithms have been implemented in a new module of the AMBER packages.  This auxiliary module, antechamber (roughly meaning "before AMBER"), can be applied to generate necessary inputs of leap-the AMBER program to generate topologies for minimization, mol. dynamics, etc., for most org. mols.  The algorithms behind the manipulations may be useful for other mol. mech. packages as well as applications that need to designate atom types and bond types.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqizj6F18nBYbVg90H21EOLACvtfcHk0lgdj0-d1uPbiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xps1Gis7g%253D&md5=8031a21d2784d5dea12e70868522aa61</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fj.jmgm.2005.12.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmgm.2005.12.005%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%2BM.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DKollman%26aufirst%3DP.%2BA.%26aulast%3DCase%26aufirst%3DD.%2BA.%26atitle%3DAutomatic%2520atom%2520type%2520and%2520bond%2520type%2520perception%2520in%2520molecular%2520mechanical%2520calculations%26jtitle%3DJ.%2520Mol.%2520Graphics%2520Modell.%26date%3D2006%26volume%3D25%26spage%3D247%26epage%3D260%26doi%3D10.1016%2Fj.jmgm.2005.12.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wolf, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caldwell, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kollman, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Case, D. A.</span><span> </span><span class="NLM_article-title">Development and testing of a general amber force field</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">1157</span><span class="NLM_x">–</span> <span class="NLM_lpage">1174</span><span class="refDoi"> DOI: 10.1002/jcc.20035</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1002%2Fjcc.20035" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=15116359" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BD2cXksFakurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2004&pages=1157-1174&author=J.+Wangauthor=R.+M.+Wolfauthor=J.+W.+Caldwellauthor=P.+A.+Kollmanauthor=D.+A.+Case&title=Development+and+testing+of+a+general+amber+force+field&doi=10.1002%2Fjcc.20035"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Development and testing of a general Amber force field</span></div><div class="casAuthors">Wang, Junmei; Wolf, Romain M.; Caldwell, James W.; Kollman, Peter A.; Case, David A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1157-1174</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">We describe here a general Amber force field (GAFF) for org. mols.  GAFF is designed to be compatible with existing Amber force fields for proteins and nucleic acids, and has parameters for most org. and pharmaceutical mols. that are composed of H, C, N, O, S, P, and halogens.  It uses a simple functional form and a limited no. of atom types, but incorporates both empirical and heuristic models to est. force consts. and partial at. charges.  The performance of GAFF in test cases is encouraging.  In test I, 74 crystallog. structures were compared to GAFF minimized structures, with a root-mean-square displacement of 0.26 Å, which is comparable to that of the Tripos 5.2 force field (0.25 Å) and better than those of MMFF 94 and CHARMm (0.47 and 0.44 Å, resp.).  In test II, gas phase minimizations were performed on 22 nucleic acid base pairs, and the minimized structures and intermol. energies were compared to MP2/6-31G* results.  The RMS of displacements and relative energies were 0.25 Å and 1.2 kcal/mol, resp.  These data are comparable to results from Parm99/RESP (0.16 Å and 1.18 kcal/mol, resp.), which were parameterized to these base pairs.  Test III looked at the relative energies of 71 conformational pairs that were used in development of the Parm99 force field.  The RMS error in relative energies (compared to expt.) is about 0.5 kcal/mol.  GAFF can be applied to wide range of mols. in an automatic fashion, making it suitable for rational drug design and database searching.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfW8Gsf8IiW7Vg90H21EOLACvtfcHk0lhAFeYV2OJW1Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXksFakurc%253D&md5=2992017a8cf51f89290ae2562403b115</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1002%2Fjcc.20035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.20035%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DWolf%26aufirst%3DR.%2BM.%26aulast%3DCaldwell%26aufirst%3DJ.%2BW.%26aulast%3DKollman%26aufirst%3DP.%2BA.%26aulast%3DCase%26aufirst%3DD.%2BA.%26atitle%3DDevelopment%2520and%2520testing%2520of%2520a%2520general%2520amber%2520force%2520field%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2004%26volume%3D25%26spage%3D1157%26epage%3D1174%26doi%3D10.1002%2Fjcc.20035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Vanommeslaeghe, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">MacKerell, A. D.,  Jr.</span><span> </span><span class="NLM_article-title">Automation of the CHARMM general force field (CGenFF) I: bond perception and atom typing</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">3144</span><span class="NLM_x">–</span> <span class="NLM_lpage">3154</span><span class="refDoi"> DOI: 10.1021/ci300363c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci300363c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs1Gns7fL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=3144-3154&author=K.+Vanommeslaegheauthor=A.+D.+MacKerell&title=Automation+of+the+CHARMM+general+force+field+%28CGenFF%29+I%3A+bond+perception+and+atom+typing&doi=10.1021%2Fci300363c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Automation of the CHARMM General Force Field (CGenFF) I: Bond Perception and Atom Typing</span></div><div class="casAuthors">Vanommeslaeghe, K.; MacKerell, A. D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3144-3154</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mol. mechanics force fields are widely used in computer-aided drug design for the study of drug-like mols. alone or interacting with biol. systems.  In simulations involving biol. macromols., the biol. part is typically represented by a specialized biomol. force field, while the drug is represented by a matching general (org.) force field.  In order to apply these general force fields to an arbitrary drug-like mol., functionality for assignment of atom types, parameters, and charges is required.  In the present article, which is part I of a series of two, we present the algorithms for bond perception and atom typing for the CHARMM General Force Field (CGenFF).  The CGenFF atom typer first assocs. attributes to the atoms and bonds in a mol., such as valence, bond order, and ring membership among others.  Of note are a no. of features that are specifically required for CGenFF.  This information is then used by the atom typing routine to assign CGenFF atom types based on a programmable decision tree.  This allows for straight-forward implementation of CGenFF's complicated atom typing rules and for equally straight-forward updating of the atom typing scheme as the force field grows.  The presented atom typer was validated by assigning correct atom types on 477 model compds. including in the training set as well as 126 test-set mols. that were constructed to specifically verify its different components.  The program may be utilized via an online implementation at https://www.paramchem.org/.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvkdOrvB3NELVg90H21EOLACvtfcHk0lhAFeYV2OJW1Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs1Gns7fL&md5=c6679293f4a2501f2bcadf2020ca1473</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Fci300363c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci300363c%26sid%3Dliteratum%253Aachs%26aulast%3DVanommeslaeghe%26aufirst%3DK.%26aulast%3DMacKerell%26aufirst%3DA.%2BD.%26atitle%3DAutomation%2520of%2520the%2520CHARMM%2520general%2520force%2520field%2520%2528CGenFF%2529%2520I%253A%2520bond%2520perception%2520and%2520atom%2520typing%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26spage%3D3144%26epage%3D3154%26doi%3D10.1021%2Fci300363c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Vanommeslaeghe, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raman, E. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">MacKerell, A. D.,  Jr.</span><span> </span><span class="NLM_article-title">Automation of the CHARMM general force field (CGenFF) II: assignment of bonded parameters and partial atomic charges</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">3155</span><span class="NLM_x">–</span> <span class="NLM_lpage">3168</span><span class="refDoi"> DOI: 10.1021/ci3003649</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci3003649" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs1Gns7fF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=3155-3168&author=K.+Vanommeslaegheauthor=E.+P.+Ramanauthor=A.+D.+MacKerell&title=Automation+of+the+CHARMM+general+force+field+%28CGenFF%29+II%3A+assignment+of+bonded+parameters+and+partial+atomic+charges&doi=10.1021%2Fci3003649"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Automation of the CHARMM General Force Field (CGenFF) II: Assignment of Bonded Parameters and Partial Atomic Charges</span></div><div class="casAuthors">Vanommeslaeghe, K.; Raman, E. Prabhu; MacKerell, A. D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3155-3168</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mol. mechanics force fields are widely used in computer-aided drug design for the study of drug candidates interacting with biol. systems.  In these simulations, the biol. part is typically represented by a specialized biomol. force field, while the drug is represented by a matching general (org.) force field.  In order to apply these general force fields to an arbitrary drug-like mol., functionality for assignment of atom types, parameters, and partial at. charges is required.  In the present article, algorithms for the assignment of parameters and charges for the CHARMM General Force Field (CGenFF) are presented.  These algorithms rely on the existing parameters and charges that were detd. as part of the parametrization of the force field.  Bonded parameters are assigned based on the similarity between the atom types that define said parameters, while charges are detd. using an extended bond-charge increment scheme.  Charge increments were optimized to reproduce the charges on model compds. that were part of the parametrization of the force field.  Case studies are presented to clarify the functioning of the algorithms and the significance of their output data.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0zgm66CLdn7Vg90H21EOLACvtfcHk0lhAFeYV2OJW1Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs1Gns7fF&md5=e676ad1f42cb1e98dd353d4d285e8d13</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Fci3003649&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci3003649%26sid%3Dliteratum%253Aachs%26aulast%3DVanommeslaeghe%26aufirst%3DK.%26aulast%3DRaman%26aufirst%3DE.%2BP.%26aulast%3DMacKerell%26aufirst%3DA.%2BD.%26atitle%3DAutomation%2520of%2520the%2520CHARMM%2520general%2520force%2520field%2520%2528CGenFF%2529%2520II%253A%2520assignment%2520of%2520bonded%2520parameters%2520and%2520partial%2520atomic%2520charges%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26spage%3D3155%26epage%3D3168%26doi%3D10.1021%2Fci3003649" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">Makov, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Payne, M. C.</span><span> </span><span class="NLM_article-title">Periodic boundary conditions in ab initio calculations</span> <span class="citation_source-journal">Phys. Rev. B: Condens. Matter Mater. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">4014</span><span class="NLM_x">–</span> <span class="NLM_lpage">4022</span><span class="refDoi"> DOI: 10.1103/PhysRevB.51.4014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1103%2FPhysRevB.51.4014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=9979237" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADyaK2MXjvVGksbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=1995&pages=4014-4022&author=G.+Makovauthor=M.+C.+Payne&title=Periodic+boundary+conditions+in+ab+initio+calculations&doi=10.1103%2FPhysRevB.51.4014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Periodic boundary conditions in ab initio calculations</span></div><div class="casAuthors">Makov, G.; Payne, M. C.</div><div class="citationInfo"><span class="NLM_cas:title">Physical Review B: Condensed Matter</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">4014-22</span>CODEN:
                <span class="NLM_cas:coden">PRBMDO</span>;
        ISSN:<span class="NLM_cas:issn">0163-1829</span>.
    
            (<span class="NLM_cas:orgname">American Physical Society</span>)
        </div><div class="casAbstract">The convergence of the electrostatic energy in calcns. using periodic boundary conditions is considered in the context of periodic solids and localized aperiodic systems in the gas and condensed phases.  Conditions for the abs. convergence of the total energy in periodic boundary conditions are obtained, and their implications for calcns. of the properties of polarized solids under the zero-field assumption are discussed.  For aperiodic systems the exact electrostatic energy functional in periodic boundary conditions is obtained.  The convergence in such systems is considered in the limit of large supercells, where, in the gas phase, the computational effort is proportional to the vol.  It is shown that for neutral localized aperiodic systems in either the gas or condensed phases, the energy can always be made to converge as O (L-5) where L is the linear dimension of the supercell.  For charged systems, convergence at this rate can be achieved after adding correction terms to the energy to account for spurious interactions induced by the periodic boundary conditions.  These terms are derived exactly for the gas phase and heuristically for the condensed phase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo13RcOpxda2bVg90H21EOLACvtfcHk0lgyPvJ8RG7aVg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXjvVGksbk%253D&md5=e43f50b6b2ab0d05c4f834f98f531bba</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1103%2FPhysRevB.51.4014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1103%252FPhysRevB.51.4014%26sid%3Dliteratum%253Aachs%26aulast%3DMakov%26aufirst%3DG.%26aulast%3DPayne%26aufirst%3DM.%2BC.%26atitle%3DPeriodic%2520boundary%2520conditions%2520in%2520ab%2520initio%2520calculations%26jtitle%3DPhys.%2520Rev.%2520B%253A%2520Condens.%2520Matter%2520Mater.%2520Phys.%26date%3D1995%26volume%3D51%26spage%3D4014%26epage%3D4022%26doi%3D10.1103%2FPhysRevB.51.4014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Martyna, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tobias, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klein, M. L.</span><span> </span><span class="NLM_article-title">Constant-pressure molecular-dynamics algorithms</span> <span class="citation_source-journal">J. Chem. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">101</span><span class="NLM_x">, </span> <span class="NLM_fpage">4177</span><span class="NLM_x">–</span> <span class="NLM_lpage">4189</span><span class="refDoi"> DOI: 10.1063/1.467468</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1063%2F1.467468" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADyaK2cXmtFeht7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=101&publication_year=1994&pages=4177-4189&author=G.+J.+Martynaauthor=D.+J.+Tobiasauthor=M.+L.+Klein&title=Constant-pressure+molecular-dynamics+algorithms&doi=10.1063%2F1.467468"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Constant pressure molecular dynamics algorithms</span></div><div class="casAuthors">Martyna, Glenn J.; Tobias, Douglas J.; Klein, Michael L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">101</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">4177-89</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    </div><div class="casAbstract">Modularly invariant equations of motion are derived that generate the isothermal-isobaric ensemble as their phase space avs.  Isotropic vol. fluctuations and fully flexible simulation cells as well as a hybrid scheme that naturally combines the two motions are considered.  The resulting methods are tested on two problems, a particle in a one-dimensional periodic potential and a spherical model of C60 in the solid/fluid phase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3YN7wMRJpjrVg90H21EOLACvtfcHk0lgyPvJ8RG7aVg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXmtFeht7o%253D&md5=c14bd79c6398b0b30541e3cbe92851b0</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1063%2F1.467468&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.467468%26sid%3Dliteratum%253Aachs%26aulast%3DMartyna%26aufirst%3DG.%2BJ.%26aulast%3DTobias%26aufirst%3DD.%2BJ.%26aulast%3DKlein%26aufirst%3DM.%2BL.%26atitle%3DConstant-pressure%2520molecular-dynamics%2520algorithms%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1994%26volume%3D101%26spage%3D4177%26epage%3D4189%26doi%3D10.1063%2F1.467468" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group">Hünenberger, P. H.</span><span> </span><span class="NLM_article-title">Thermostat Algorithms for Molecular Dynamics Simulations</span>. In  <span class="citation_source-book">Advanced Computer Simulation</span>; Advances in Polymer Science, Vol. 173; <span class="NLM_publisher-name">Springer</span>,<span class="NLM_x"> </span><span class="NLM_year">2005</span><span class="NLM_x">; </span>pp  <span class="NLM_fpage">105</span><span class="NLM_x">–</span> <span class="NLM_lpage">149</span>; DOI: <span class="refDoi"> DOI: 10.1007/b99427</span> .</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1007%2Fb99427" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2005&pages=105-149&author=P.+H.+H%C3%BCnenberger&title=Advanced+Computer+Simulation"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1007%2Fb99427&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252Fb99427%26sid%3Dliteratum%253Aachs%26aulast%3DH%25C3%25BCnenberger%26aufirst%3DP.%2BH.%26atitle%3DThermostat%2520Algorithms%2520for%2520Molecular%2520Dynamics%2520Simulations%26btitle%3DAdvanced%2520Computer%2520Simulation%26pub%3DSpringer%26date%3D2005%26spage%3D105%26epage%3D149%26doi%3D10.1007%2Fb99427" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group">Buch, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giorgino, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Fabritiis, G.</span><span> </span><span class="NLM_article-title">Complete reconstruction of an enzyme-inhibitor binding process by molecular dynamics simulations</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">108</span><span class="NLM_x">, </span> <span class="NLM_fpage">10184</span><span class="NLM_x">–</span> <span class="NLM_lpage">10189</span><span class="refDoi"> DOI: 10.1073/pnas.1103547108</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1073%2Fpnas.1103547108" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=21646537" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC3MXot1egu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=108&publication_year=2011&pages=10184-10189&author=I.+Buchauthor=T.+Giorginoauthor=G.+De+Fabritiis&title=Complete+reconstruction+of+an+enzyme-inhibitor+binding+process+by+molecular+dynamics+simulations&doi=10.1073%2Fpnas.1103547108"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Complete reconstruction of an enzyme-inhibitor binding process by molecular dynamics simulations</span></div><div class="casAuthors">Buch, Ignasi; Giorgino, Toni; De Fabritiis, Gianni</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">108</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">10184-10189, S10184/1-S10184/6</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The understanding of protein-ligand binding is of crit. importance for biomedical research, yet the process itself has been very difficult to study because of its intrinsically dynamic character.  Here, we have been able to quant. reconstruct the complete binding process of the enzyme-inhibitor complex trypsin-benzamidine by performing 495 mol. dynamics simulations of free ligand binding of 100 ns each, 187 of which produced binding events with an rmsd less than 2 Å compared to the crystal structure.  The binding paths obtained are able to capture the kinetic pathway of the inhibitor diffusing from solvent (SO) to the bound (S4) state passing through two metastable intermediate states S2 and S3.  Rather than directly entering the binding pocket the inhibitor appears to roll on the surface of the protein in its transition between S3 and the final binding pocket, whereas the transition between S2 and the bound pose requires rediffusion to S3.  An estn. of the std. free energy of binding gives ΔG° = -5.2 0.4 kcal/mol (cf. the exptl. value -6.2 kcal/mol), and a two-states kinetic model kon = (1.5 ± 0.2) × 108 M-1 s-1 and koff = (9.5 ± 3.3) × 104 s-1 for unbound to bound transitions.  The ability to reconstruct by simple diffusion the binding pathway of an enzyme-inhibitor binding process demonstrates the predictive power of unconventional high-throughput mol. simulations.  Moreover, the methodol. is directly applicable to other mol. systems and thus of general interest in biomedical and pharmaceutical research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrEgdoM19tFbbVg90H21EOLACvtfcHk0lgyPvJ8RG7aVg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXot1egu78%253D&md5=40fdd0ec62da71c0f11d57731ee518a1</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1103547108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1103547108%26sid%3Dliteratum%253Aachs%26aulast%3DBuch%26aufirst%3DI.%26aulast%3DGiorgino%26aufirst%3DT.%26aulast%3DDe%2BFabritiis%26aufirst%3DG.%26atitle%3DComplete%2520reconstruction%2520of%2520an%2520enzyme-inhibitor%2520binding%2520process%2520by%2520molecular%2520dynamics%2520simulations%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2011%26volume%3D108%26spage%3D10184%26epage%3D10189%26doi%3D10.1073%2Fpnas.1103547108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group">Thomas, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yuriev, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chalmers, D. K.</span><span> </span><span class="NLM_article-title">Ligand binding pathways of clozapine and haloperidol in the dopamine D-2 and D-3 receptors</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">308</span><span class="NLM_x">–</span> <span class="NLM_lpage">321</span><span class="refDoi"> DOI: 10.1021/acs.jcim.5b00457</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.5b00457" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitVGiurzO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2016&pages=308-321&author=T.+Thomasauthor=Y.+Fangauthor=E.+Yurievauthor=D.+K.+Chalmers&title=Ligand+binding+pathways+of+clozapine+and+haloperidol+in+the+dopamine+D-2+and+D-3+receptors&doi=10.1021%2Facs.jcim.5b00457"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand Binding Pathways of Clozapine and Haloperidol in the Dopamine D2 and D3 Receptors</span></div><div class="casAuthors">Thomas, Trayder; Fang, Yu; Yuriev, Elizabeth; Chalmers, David K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">308-321</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The binding of a small mol. ligand to its protein target is most often characterized by binding affinity and is typically viewed as an on/off switch.  The more complex reality is that binding involves the ligand passing through a series of intermediate states between the soln. phase and the fully bound pose.  We have performed a set of 29 unbiased mol. dynamics simulations to model the binding pathways of the dopamine receptor antagonists clozapine and haloperidol binding to the D2 and D3 dopamine receptors.  Through these simulations we have captured the binding pathways of clozapine and haloperidol from the extracellular vestibule to the orthosteric binding site and thereby, we also predict the bound pose of each ligand.  These are the first long time scale simulations of haloperidol or clozapine binding to dopamine receptors.  From these simulations, we have identified several important stages in the binding pathway, including the involvement of Tyr7.35 in a "handover" mechanism that transfers the ligand between the extracellular vestibule and Asp3.32.  We have also performed interaction and cluster analyses to det. differences in binding pathways between the D2 and D3 receptors and identified metastable states that may be of use in drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpimYd-xEW9a7Vg90H21EOLACvtfcHk0lhaP1fmjV6kpg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitVGiurzO&md5=b5faf13609b11b8efcfebd3eb5f0a77e</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.5b00457&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.5b00457%26sid%3Dliteratum%253Aachs%26aulast%3DThomas%26aufirst%3DT.%26aulast%3DFang%26aufirst%3DY.%26aulast%3DYuriev%26aufirst%3DE.%26aulast%3DChalmers%26aufirst%3DD.%2BK.%26atitle%3DLigand%2520binding%2520pathways%2520of%2520clozapine%2520and%2520haloperidol%2520in%2520the%2520dopamine%2520D-2%2520and%2520D-3%2520receptors%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2016%26volume%3D56%26spage%3D308%26epage%3D321%26doi%3D10.1021%2Facs.jcim.5b00457" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group">Torrie, G. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Valleau, J. P.</span><span> </span><span class="NLM_article-title">Non-physical sampling distributions in Monte-Carlo free-energy estimation - umbrella sampling</span> <span class="citation_source-journal">J. Comput. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1977</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">187</span><span class="NLM_x">–</span> <span class="NLM_lpage">199</span><span class="refDoi"> DOI: 10.1016/0021-9991(77)90121-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1016%2F0021-9991%2877%2990121-8" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=1977&pages=187-199&author=G.+M.+Torrieauthor=J.+P.+Valleau&title=Non-physical+sampling+distributions+in+Monte-Carlo+free-energy+estimation+-+umbrella+sampling&doi=10.1016%2F0021-9991%2877%2990121-8"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2F0021-9991%2877%2990121-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0021-9991%252877%252990121-8%26sid%3Dliteratum%253Aachs%26aulast%3DTorrie%26aufirst%3DG.%2BM.%26aulast%3DValleau%26aufirst%3DJ.%2BP.%26atitle%3DNon-physical%2520sampling%2520distributions%2520in%2520Monte-Carlo%2520free-energy%2520estimation%2520-%2520umbrella%2520sampling%26jtitle%3DJ.%2520Comput.%2520Phys.%26date%3D1977%26volume%3D23%26spage%3D187%26epage%3D199%26doi%3D10.1016%2F0021-9991%2877%2990121-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group">Woo, H. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roux, B.</span><span> </span><span class="NLM_article-title">Calculation of absolute protein-ligand binding free energy from computer simulations</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">102</span><span class="NLM_x">, </span> <span class="NLM_fpage">6825</span><span class="NLM_x">–</span> <span class="NLM_lpage">6830</span><span class="refDoi"> DOI: 10.1073/pnas.0409005102</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1073%2Fpnas.0409005102" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=15867154" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BD2MXksVKgurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=2005&pages=6825-6830&author=H.+J.+Wooauthor=B.+Roux&title=Calculation+of+absolute+protein-ligand+binding+free+energy+from+computer+simulations&doi=10.1073%2Fpnas.0409005102"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Calculation of absolute protein-ligand binding free energy from computer simulations</span></div><div class="casAuthors">Woo, Hyung-June; Roux, Benoit</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">102</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">6825-6830</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">A general methodol. for calcg. the equil. binding const. of a flexible ligand to a protein receptor is formulated on the basis of potentials of mean force.  The overall process is decompd. into several stages that can be computed sep.: the free ligand in the bulk is first restrained into the conformation it adopts in the bound state, position, and orientation by applying biasing potentials, then it is translated into the binding site, where it is released completely.  The conformational restraining potential is based on the root-mean-square deviation of the peptide coordinates relative to its av. conformation in the bound complex.  Free energy contributions from each stage are calcd. by free energy perturbation potential of mean force techniques by using appropriate order parameters.  The present approach avoids the need to decouple the ligand from its surrounding (bulk solvent and receptor protein) as is traditionally performed in the double-decoupling scheme.  It is believed that the present formulation will be particularly useful when the solvation free energy of the ligand is very large.  As an application, the equil. binding const. of the phosphotyrosine peptide pYEEI to the Src homol. 2 domain of human Lck has been calcd.  The results are in good agreement with exptl. values.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoktN0FgqFW_LVg90H21EOLACvtfcHk0lhaP1fmjV6kpg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXksVKgurw%253D&md5=23be06c80a875efaff0207ea8dc2b08d</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0409005102&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0409005102%26sid%3Dliteratum%253Aachs%26aulast%3DWoo%26aufirst%3DH.%2BJ.%26aulast%3DRoux%26aufirst%3DB.%26atitle%3DCalculation%2520of%2520absolute%2520protein-ligand%2520binding%2520free%2520energy%2520from%2520computer%2520simulations%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2005%26volume%3D102%26spage%3D6825%26epage%3D6830%26doi%3D10.1073%2Fpnas.0409005102" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group">Lee, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olson, M. A.</span><span> </span><span class="NLM_article-title">Calculation of absolute protein-ligand binding affinity using path and endpoint approaches</span> <span class="citation_source-journal">Biophys. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">90</span><span class="NLM_x">, </span> <span class="NLM_fpage">864</span><span class="NLM_x">–</span> <span class="NLM_lpage">877</span><span class="refDoi"> DOI: 10.1529/biophysj.105.071589</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1529%2Fbiophysj.105.071589" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=16284269" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtFGhs78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=90&publication_year=2006&pages=864-877&author=M.+S.+Leeauthor=M.+A.+Olson&title=Calculation+of+absolute+protein-ligand+binding+affinity+using+path+and+endpoint+approaches&doi=10.1529%2Fbiophysj.105.071589"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Calculation of absolute protein-ligand binding affinity using path and endpoint approaches</span></div><div class="casAuthors">Lee, Michael S.; Olson, Mark A.</div><div class="citationInfo"><span class="NLM_cas:title">Biophysical Journal</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">90</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">864-877</span>CODEN:
                <span class="NLM_cas:coden">BIOJAU</span>;
        ISSN:<span class="NLM_cas:issn">0006-3495</span>.
    
            (<span class="NLM_cas:orgname">Biophysical Society</span>)
        </div><div class="casAbstract">A comparative anal. is provided of rigorous and approx. methods for calcg. abs. binding affinities of two protein-ligand complexes: the FKBP protein bound with small mols. 4-hydroxy-2-butanone and FK506.  The authors' rigorous approach is an umbrella sampling technique where a potential of mean force is detd. by pulling the ligand out of the protein active site over several simulation windows.  The results of this approach agree well with exptl. obsd. binding affinities.  Also assessed is a commonly used approx. endpoint approach, which sep. ests. enthalpy, solvation free energy, and entropy.  The authors show that this endpoint approach has numerous variations, all of which are prone to crit. shortcomings.  For example, conventional harmonic and quasiharmonic entropy estn. procedures produce disparate results for the relatively simple protein-ligand systems studied in this work.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoAEk5VeXijpLVg90H21EOLACvtfcHk0lhaP1fmjV6kpg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtFGhs78%253D&md5=3d25be5ba65e123249743538bbbf9e01</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1529%2Fbiophysj.105.071589&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1529%252Fbiophysj.105.071589%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DM.%2BS.%26aulast%3DOlson%26aufirst%3DM.%2BA.%26atitle%3DCalculation%2520of%2520absolute%2520protein-ligand%2520binding%2520affinity%2520using%2520path%2520and%2520endpoint%2520approaches%26jtitle%3DBiophys.%2520J.%26date%3D2006%26volume%3D90%26spage%3D864%26epage%3D877%26doi%3D10.1529%2Fbiophysj.105.071589" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group">Kumar, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bouzida, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Swendsen, R. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kollman, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosenberg, J. M.</span><span> </span><span class="NLM_article-title">The weighted histogram analysis method for free-energy calculations on biomolecules 0.1. The method</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1992</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">1011</span><span class="NLM_x">–</span> <span class="NLM_lpage">1021</span><span class="refDoi"> DOI: 10.1002/jcc.540130812</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1002%2Fjcc.540130812" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADyaK38XmtVynsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=1992&pages=1011-1021&author=S.+Kumarauthor=D.+Bouzidaauthor=R.+H.+Swendsenauthor=P.+A.+Kollmanauthor=J.+M.+Rosenberg&title=The+weighted+histogram+analysis+method+for+free-energy+calculations+on+biomolecules+0.1.+The+method&doi=10.1002%2Fjcc.540130812"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">The weighted histogram analysis method for free-energy calculations on biomolecules.  I.  The method</span></div><div class="casAuthors">Kumar, Shankar; Bouzida, Djamal; Swendsen, Robert H.; Kollman, Peter A.; Rosenberg, John M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">1992</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1011-21</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    </div><div class="casAbstract">The Weighted Histogram Anal. Method (WHAM), an extension of Ferrenberg and Swendsen's Multiple Histogram Technique, has been applied for the first time on complex biomol. Hamiltonians.  The method is presented here as an extension of the Umbrella Sampling method for free-energy and Potential of Mean Force calcns.  This algorithm possesses the following advantages over methods that are currently employed: (1) it provides a built-in est. of sampling errors thereby yielding objective ests. of the optimal location and length of addnl. simulations needed to achieve a desired level of precision; (2) it yields the "best" value of free energies by taking into account all the simulations so as to minimize the statistical errors; (3) in addn. to optimizing the links between simulations, it also allows multiple overlaps of probability distributions for obtaining better ests. of the free-energy differences.  By recasting the Ferrenberg-Swendsen Multiple Histogram equations in a form suitable for mol. mechanics type Hamiltonians, we have demonstrated the feasibility and robustness of this method by applying it to a test problem of the generation of the Potential of Mean Force profile of the pseudorotation phase angle of the sugar ring in deoxyadenosine.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLayYt_QpVO7Vg90H21EOLACvtfcHk0lhYd4WkvFQMnw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK38XmtVynsrs%253D&md5=5b2ad7410198f03025708a37c0fbe89d</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1002%2Fjcc.540130812&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.540130812%26sid%3Dliteratum%253Aachs%26aulast%3DKumar%26aufirst%3DS.%26aulast%3DBouzida%26aufirst%3DD.%26aulast%3DSwendsen%26aufirst%3DR.%2BH.%26aulast%3DKollman%26aufirst%3DP.%2BA.%26aulast%3DRosenberg%26aufirst%3DJ.%2BM.%26atitle%3DThe%2520weighted%2520histogram%2520analysis%2520method%2520for%2520free-energy%2520calculations%2520on%2520biomolecules%25200.1.%2520The%2520method%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D1992%26volume%3D13%26spage%3D1011%26epage%3D1021%26doi%3D10.1002%2Fjcc.540130812" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group">Mobley, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Graves, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chodera, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McReynolds, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dill, K. A.</span><span> </span><span class="NLM_article-title">Predicting absolute ligand binding free energies to a simple model site</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">371</span><span class="NLM_x">, </span> <span class="NLM_fpage">1118</span><span class="NLM_x">–</span> <span class="NLM_lpage">1134</span><span class="refDoi"> DOI: 10.1016/j.jmb.2007.06.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1016%2Fj.jmb.2007.06.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=17599350" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BD2sXot12rt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=371&publication_year=2007&pages=1118-1134&author=D.+L.+Mobleyauthor=A.+P.+Gravesauthor=J.+D.+Choderaauthor=A.+C.+McReynoldsauthor=B.+K.+Shoichetauthor=K.+A.+Dill&title=Predicting+absolute+ligand+binding+free+energies+to+a+simple+model+site&doi=10.1016%2Fj.jmb.2007.06.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Predicting Absolute Ligand Binding Free Energies to a Simple Model Site</span></div><div class="casAuthors">Mobley, David L.; Graves, Alan P.; Chodera, John D.; McReynolds, Andrea C.; Shoichet, Brian K.; Dill, Ken A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">371</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1118-1134</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A central challenge in structure-based ligand design is the accurate prediction of binding free energies.  Here the authors apply alchem. free energy calcns. in explicit solvent to predict ligand binding in a model cavity in T4 lysozyme.  Even in this simple site, there are challenges.  The authors made systematic improvements, beginning with single poses from docking, then including multiple poses, addnl. protein conformational changes, and using an improved charge model.  Computed abs. binding free energies had an RMS error of 1.9 kcal/mol relative to previously detd. exptl. values.  In blind prospective tests, the methods correctly discriminated between several true ligands and decoys in a set of putative binders identified by docking.  In these prospective tests, the RMS error in predicted binding free energies relative to those subsequently detd. exptl. was only 0.6 kcal/mol.  X-ray crystal structures of the new ligands bound in the cavity corresponded closely to predictions from the free energy calcns., but sometimes differed from those predicted by docking.  Finally, the authors examd. the impact of holding the protein rigid, as in docking, with a view to learning how approxns. made in docking affect accuracy and how they may be improved.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqD1-EYEODQ47Vg90H21EOLACvtfcHk0lhYd4WkvFQMnw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXot12rt7o%253D&md5=2480139daa5a0857c59e821d2ecbce18</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2007.06.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2007.06.002%26sid%3Dliteratum%253Aachs%26aulast%3DMobley%26aufirst%3DD.%2BL.%26aulast%3DGraves%26aufirst%3DA.%2BP.%26aulast%3DChodera%26aufirst%3DJ.%2BD.%26aulast%3DMcReynolds%26aufirst%3DA.%2BC.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DDill%26aufirst%3DK.%2BA.%26atitle%3DPredicting%2520absolute%2520ligand%2520binding%2520free%2520energies%2520to%2520a%2520simple%2520model%2520site%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2007%26volume%3D371%26spage%3D1118%26epage%3D1134%26doi%3D10.1016%2Fj.jmb.2007.06.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group">Mobley, D. L.</span><span> </span><span class="NLM_article-title">Let’s get honest about sampling</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">93</span><span class="NLM_x">–</span> <span class="NLM_lpage">95</span><span class="refDoi"> DOI: 10.1007/s10822-011-9497-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1007%2Fs10822-011-9497-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=22113833" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsV2ltr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2012&pages=93-95&author=D.+L.+Mobley&title=Let%E2%80%99s+get+honest+about+sampling&doi=10.1007%2Fs10822-011-9497-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Let's get honest about sampling</span></div><div class="casAuthors">Mobley, David L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">93-95</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Mol. simulations see widespread and increasing use in computation and mol. design, esp. within the area of mol. simulations applied to biomol. binding and interactions, our focus here.  However, force field accuracy remains a concern for many practitioners, and it is often not clear what level of accuracy is really needed for payoffs in a discovery setting.  Here, I argue that despite limitations of today's force fields, current simulation tools and force fields now provide the potential for real benefits in a variety of applications.  However, these same tools also provide irreproducible results which are often poorly interpreted.  Continued progress in the field requires more honesty in assessment and care in evaluation of simulation results, esp. with respect to convergence.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoqdpLWxgFypLVg90H21EOLACvtfcHk0lhYd4WkvFQMnw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsV2ltr0%253D&md5=9cbb5eb62d7f8c7d448d58b861135897</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1007%2Fs10822-011-9497-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-011-9497-y%26sid%3Dliteratum%253Aachs%26aulast%3DMobley%26aufirst%3DD.%2BL.%26atitle%3DLet%25E2%2580%2599s%2520get%2520honest%2520about%2520sampling%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2012%26volume%3D26%26spage%3D93%26epage%3D95%26doi%3D10.1007%2Fs10822-011-9497-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group">Zhu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hummer, G.</span><span> </span><span class="NLM_article-title">Convergence and error estimation in free energy calculations using the weighted histogram analysis method</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">33</span><span class="NLM_x">, </span> <span class="NLM_fpage">453</span><span class="NLM_x">–</span> <span class="NLM_lpage">465</span><span class="refDoi"> DOI: 10.1002/jcc.21989</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1002%2Fjcc.21989" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=22109354" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsV2rurjM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2012&pages=453-465&author=F.+Zhuauthor=G.+Hummer&title=Convergence+and+error+estimation+in+free+energy+calculations+using+the+weighted+histogram+analysis+method&doi=10.1002%2Fjcc.21989"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Convergence and error estimation in free energy calculations using the weighted histogram analysis method</span></div><div class="casAuthors">Zhu, Fangqiang; Hummer, Gerhard</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">453-465</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">The weighted histogram anal. method (WHAM) has become the std. technique for the anal. of umbrella sampling simulations.  The authors address the challenges: (1) of obtaining fast and accurate solns. of the coupled nonlinear WHAM equations, (2) of quantifying the statistical errors of the resulting free energies, (3) of diagnosing possible systematic errors, and (4) of optimally allocating of the computational resources.  Traditionally, the WHAM equations are solved by a fixed-point direct iteration method, despite poor convergence and possible numerical inaccuracies in the solns.  The authors instead solve the math. equiv. problem of maximizing a target likelihood function, by using superlinear numerical optimization algorithms with a significantly faster convergence rate.  To est. the statistical errors in one-dimensional free energy profiles obtained from WHAM, the authors note that for densely spaced umbrella windows with harmonic biasing potentials, the WHAM free energy profile can be approximated by a coarse-grained free energy obtained by integrating the mean restraining forces.  The statistical errors of the coarse-grained free energies can be estd. straightforwardly and then used for the WHAM results.  A generalization to multidimensional WHAM is described.  They also propose two simple statistical criteria to test the consistency between the histograms of adjacent umbrella windows, which help identify inadequate sampling and hysteresis in the degrees of freedom orthogonal to the reaction coordinate.  Together, the ests. of the statistical errors and the diagnostics of inconsistencies in the potentials of mean force provide a basis for the efficient allocation of computational resources in free energy simulations.  © 2011 Wiley Periodicals, Inc.  J Comput Chem, 2011.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7m-OnlpQeSrVg90H21EOLACvtfcHk0lhYd4WkvFQMnw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsV2rurjM&md5=e96ef6d56ab6cb619e1f7d6b2ba1fb32</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1002%2Fjcc.21989&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.21989%26sid%3Dliteratum%253Aachs%26aulast%3DZhu%26aufirst%3DF.%26aulast%3DHummer%26aufirst%3DG.%26atitle%3DConvergence%2520and%2520error%2520estimation%2520in%2520free%2520energy%2520calculations%2520using%2520the%2520weighted%2520histogram%2520analysis%2520method%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2012%26volume%3D33%26spage%3D453%26epage%3D465%26doi%3D10.1002%2Fjcc.21989" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group">Lu, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kofke, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woolf, T. B.</span><span> </span><span class="NLM_article-title">Improving the efficiency and reliability of free energy perturbation calculations using overlap sampling methods</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">28</span><span class="NLM_x">–</span> <span class="NLM_lpage">39</span><span class="refDoi"> DOI: 10.1002/jcc.10369</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1002%2Fjcc.10369" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=14634991" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A280%3ADC%252BD3srlsVCqtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2004&pages=28-39&author=N.+Luauthor=D.+A.+Kofkeauthor=T.+B.+Woolf&title=Improving+the+efficiency+and+reliability+of+free+energy+perturbation+calculations+using+overlap+sampling+methods&doi=10.1002%2Fjcc.10369"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Improving the efficiency and reliability of free energy perturbation calculations using overlap sampling methods</span></div><div class="casAuthors">Lu Nandou; Kofke David A; Woolf Thomas B</div><div class="citationInfo"><span class="NLM_cas:title">Journal of computational chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">28-39</span>
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    </div><div class="casAbstract">A challenge in free energy calculation for complex molecular systems by computer simulation is to obtain a reliable estimate within feasible computational time.  In this study, we suggest an answer to this challenge by exploring a simple method, overlap sampling (OS), for producing reliable free-energy results in an efficient way.  The formalism of the OS method is based on ensuring sampling of important overlapping phase space during perturbation calculations.  This technique samples both forward and reverse free energy perturbation (FEP) to improve the free-energy calculation.  It considers the asymmetry of the FEP calculation and features an ability to optimize both the precision and the accuracy of the measurement without affecting the simulation process itself.  The OS method is tested at two optimization levels: no optimization (simple OS), and full optimization (equivalent to Bennett's method), and compared to conventional FEP techniques, including the widely used direct FEP averaging method, on three alchemical mutation systems: (a) an anion transformation in water solution, (b) mutation between methanol and ethane, and (c) alchemical change of an adenosine molecule.  It is consistently shown that the reliability of free-energy estimates can be greatly improved using the OS techniques at both optimization levels, while the performance of Bennett's method is particularly striking.  In addition, the efficiency of a calculation can be significantly improved because the method is able to (a) converge to the right answer quickly, and (b) work for large perturbations.  The basic two-stage OS method can be extended to admit additional stages, if needed.  We suggest that the OS method can be used as a general perturbation technique for computing free energy differences in molecular simulations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQZ3pOA1ItyLkloaUqgkFHkfW6udTcc2eZcO-ETtWQUH7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD3srlsVCqtw%253D%253D&md5=44b61ab8523f535d4d29ab4ba355ec91</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1002%2Fjcc.10369&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.10369%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DN.%26aulast%3DKofke%26aufirst%3DD.%2BA.%26aulast%3DWoolf%26aufirst%3DT.%2BB.%26atitle%3DImproving%2520the%2520efficiency%2520and%2520reliability%2520of%2520free%2520energy%2520perturbation%2520calculations%2520using%2520overlap%2520sampling%2520methods%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2004%26volume%3D25%26spage%3D28%26epage%3D39%26doi%3D10.1002%2Fjcc.10369" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group">Zwanzig, R. W.</span><span> </span><span class="NLM_article-title">High-temperature equation of state by a perturbation method. I. nonpolar gases</span> <span class="citation_source-journal">J. Chem. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1954</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">1420</span><span class="NLM_x">–</span> <span class="NLM_lpage">1426</span><span class="refDoi"> DOI: 10.1063/1.1740409</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1063%2F1.1740409" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADyaG2cXnsFCgsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=1954&pages=1420-1426&author=R.+W.+Zwanzig&title=High-temperature+equation+of+state+by+a+perturbation+method.+I.+nonpolar+gases&doi=10.1063%2F1.1740409"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">High-temperature equation of state by a perturbation method. I. Nonpolar gases</span></div><div class="casAuthors">Zwanzig, Robert W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">1954</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1420-6</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    </div><div class="casAbstract">A theoretical study was made of the equations of state of A and N at high temp. and ds.  The intermol. potential was of the Lennard-Jones form, with an adjustable rigid sphere cutoff.  A perturbation theory was developed, by which the thermodynamic properties of 1 system could be related to those of a slightly different system and to the difference in the intermol. potentials of the 2 systems.  The unperturbed system was a rigid-sphere fluid, and the Lennard-Jones potential was the perturbation.  The results were in fair agreement with expt. and can be used as an exptl. test of the theoretical rigid-sphere equation of state.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqUK-2Wdu4YdrVg90H21EOLACvtfcHk0lgFTYz5Eg24vw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaG2cXnsFCgsw%253D%253D&md5=8b0bce75afc2097b1c178dab250389a9</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1063%2F1.1740409&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.1740409%26sid%3Dliteratum%253Aachs%26aulast%3DZwanzig%26aufirst%3DR.%2BW.%26atitle%3DHigh-temperature%2520equation%2520of%2520state%2520by%2520a%2520perturbation%2520method.%2520I.%2520nonpolar%2520gases%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1954%26volume%3D22%26spage%3D1420%26epage%3D1426%26doi%3D10.1063%2F1.1740409" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group">Paliwal, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shirts, M. R.</span><span> </span><span class="NLM_article-title">A benchmark test set for alchemical free energy transformations and its use to quantify error in common free energy methods</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">4115</span><span class="NLM_x">–</span> <span class="NLM_lpage">4134</span><span class="refDoi"> DOI: 10.1021/ct2003995</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct2003995" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlaqtrvM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=4115-4134&author=H.+Paliwalauthor=M.+R.+Shirts&title=A+benchmark+test+set+for+alchemical+free+energy+transformations+and+its+use+to+quantify+error+in+common+free+energy+methods&doi=10.1021%2Fct2003995"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">A Benchmark Test Set for Alchemical Free Energy Transformations and Its Use to Quantify Error in Common Free Energy Methods</span></div><div class="casAuthors">Paliwal, Himanshu; Shirts, Michael R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">4115-4134</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">There is a significant need for improved tools to validate thermophys. quantities computed via mol. simulation.  In this paper we present the initial version of a benchmark set of testing methods for calcg. free energies of mol. transformation in soln.  This set is based on mol. changes common to many mol. design problems, such as insertion and deletion of at. sites and changing at. partial charges.  We use this benchmark set to compare the statistical efficiency, reliability, and quality of uncertainty ests. for a no. of published free energy methods, including thermodn. integration, free energy perturbation, the Bennett acceptance ratio (BAR) and its multistate equiv. MBAR.  We identify MBAR as the consistently best performing method, though other methods are frequently comparable in reliability and accuracy in many cases.  We demonstrate that assumptions of Gaussian distributed errors in free energies are usually valid for most methods studied.  We demonstrate that bootstrap error estn. is a robust and useful technique for estg. statistical variance for all free energy methods studied.  This benchmark set is provided in a no. of different file formats with the hope of becoming a useful and general tool for method comparisons.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxxPO-qPI-gLVg90H21EOLACvtfcHk0lj0Bb0XyVZuwg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlaqtrvM&md5=41383373300216165a7f1724ad535451</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Fct2003995&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct2003995%26sid%3Dliteratum%253Aachs%26aulast%3DPaliwal%26aufirst%3DH.%26aulast%3DShirts%26aufirst%3DM.%2BR.%26atitle%3DA%2520benchmark%2520test%2520set%2520for%2520alchemical%2520free%2520energy%2520transformations%2520and%2520its%2520use%2520to%2520quantify%2520error%2520in%2520common%2520free%2520energy%2520methods%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2011%26volume%3D7%26spage%3D4115%26epage%3D4134%26doi%3D10.1021%2Fct2003995" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group">Chodera, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mobley, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shirts, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dixon, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Branson, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pande, V. S.</span><span> </span><span class="NLM_article-title">Alchemical free energy methods for drug discovery: progress and challenges</span> <span class="citation_source-journal">Curr. Opin. Struct. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">150</span><span class="NLM_x">–</span> <span class="NLM_lpage">160</span><span class="refDoi"> DOI: 10.1016/j.sbi.2011.01.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1016%2Fj.sbi.2011.01.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=21349700" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC3MXkt1Kisrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2011&pages=150-160&author=J.+D.+Choderaauthor=D.+L.+Mobleyauthor=M.+R.+Shirtsauthor=R.+W.+Dixonauthor=K.+Bransonauthor=V.+S.+Pande&title=Alchemical+free+energy+methods+for+drug+discovery%3A+progress+and+challenges&doi=10.1016%2Fj.sbi.2011.01.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Alchemical free energy methods for drug discovery: Progress and challenges</span></div><div class="casAuthors">Chodera, John D.; Mobley, David L.; Shirts, Michael R.; Dixon, Richard W.; Branson, Kim; Pande, Vijay S.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Structural Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">150-160</span>CODEN:
                <span class="NLM_cas:coden">COSBEF</span>;
        ISSN:<span class="NLM_cas:issn">0959-440X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Improved rational drug design methods are needed to lower the cost and increase the success rate of drug discovery and development.  Alchem. binding free energy calcns., one potential tool for rational design, have progressed rapidly over the past decade, but still fall short of providing robust tools for pharmaceutical engineering.  Recent studies, esp. on model receptor systems, have clarified many of the challenges that must be overcome for robust predictions of binding affinity to be useful in rational design.  In this review, inspired by a recent joint academic/industry meeting organized by the authors, we discuss these challenges and suggest a no. of promising approaches for overcoming them.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobUBHsGUCIWLVg90H21EOLACvtfcHk0lj0Bb0XyVZuwg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXkt1Kisrk%253D&md5=fa17c0982d921cd1c32cd095fd34420e</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1016%2Fj.sbi.2011.01.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.sbi.2011.01.011%26sid%3Dliteratum%253Aachs%26aulast%3DChodera%26aufirst%3DJ.%2BD.%26aulast%3DMobley%26aufirst%3DD.%2BL.%26aulast%3DShirts%26aufirst%3DM.%2BR.%26aulast%3DDixon%26aufirst%3DR.%2BW.%26aulast%3DBranson%26aufirst%3DK.%26aulast%3DPande%26aufirst%3DV.%2BS.%26atitle%3DAlchemical%2520free%2520energy%2520methods%2520for%2520drug%2520discovery%253A%2520progress%2520and%2520challenges%26jtitle%3DCurr.%2520Opin.%2520Struct.%2520Biol.%26date%3D2011%26volume%3D21%26spage%3D150%26epage%3D160%26doi%3D10.1016%2Fj.sbi.2011.01.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group">De Vivo, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Masetti, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bottegoni, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cavalli, A.</span><span> </span><span class="NLM_article-title">Role of molecular dynamics and related methods in drug discovery</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">59</span><span class="NLM_x">, </span> <span class="NLM_fpage">4035</span><span class="NLM_x">–</span> <span class="NLM_lpage">4061</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01684</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01684" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtlWksLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=4035-4061&author=M.+De+Vivoauthor=M.+Masettiauthor=G.+Bottegoniauthor=A.+Cavalli&title=Role+of+molecular+dynamics+and+related+methods+in+drug+discovery&doi=10.1021%2Facs.jmedchem.5b01684"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Role of Molecular Dynamics and Related Methods in Drug Discovery</span></div><div class="casAuthors">De Vivo, Marco; Masetti, Matteo; Bottegoni, Giovanni; Cavalli, Andrea</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4035-4061</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mol. dynamics (MD) and related methods are close to becoming routine computational tools for drug discovery.  Their main advantage is in explicitly treating structural flexibility and entropic effects.  This allows a more accurate est. of the thermodn. and kinetics assocd. with drug-target recognition and binding, as better algorithms and hardware architectures increase their use.  Here, we review the theor. background of MD and enhanced sampling methods, focusing on free-energy perturbation, metadynamics, steered MD, and other methods most consistently used to study drug-target binding.  We discuss unbiased MD simulations that nowadays allow the observation of unsupervised ligand-target binding, assessing how these approaches help optimizing target affinity and drug residence time toward improved drug efficacy.  Further issues discussed include allosteric modulation and the role of water mols. in ligand binding and optimization.  We conclude by calling for more prospective studies to attest to these methods' utility in discovering novel drug candidates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQb_02a7Nmh7Vg90H21EOLACvtfcHk0lj0Bb0XyVZuwg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtlWksLs%253D&md5=b3d8a4fb5705a3555c2bb2a962420396</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01684%26sid%3Dliteratum%253Aachs%26aulast%3DDe%2BVivo%26aufirst%3DM.%26aulast%3DMasetti%26aufirst%3DM.%26aulast%3DBottegoni%26aufirst%3DG.%26aulast%3DCavalli%26aufirst%3DA.%26atitle%3DRole%2520of%2520molecular%2520dynamics%2520and%2520related%2520methods%2520in%2520drug%2520discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D4035%26epage%3D4061%26doi%3D10.1021%2Facs.jmedchem.5b01684" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deng, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roux, B.</span><span> </span><span class="NLM_article-title">Absolute binding free energy calculations using molecular dynamics simulations with restraining potentials</span> <span class="citation_source-journal">Biophys. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">91</span><span class="NLM_x">, </span> <span class="NLM_fpage">2798</span><span class="NLM_x">–</span> <span class="NLM_lpage">2814</span><span class="refDoi"> DOI: 10.1529/biophysj.106.084301</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1529%2Fbiophysj.106.084301" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=16844742" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtVGls7vP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=91&publication_year=2006&pages=2798-2814&author=J.+Wangauthor=Y.+Dengauthor=B.+Roux&title=Absolute+binding+free+energy+calculations+using+molecular+dynamics+simulations+with+restraining+potentials&doi=10.1529%2Fbiophysj.106.084301"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Absolute binding free energy calculations using molecular dynamics simulations with restraining potentials</span></div><div class="casAuthors">Wang, Jiyao; Deng, Yuqing; Roux, Benoit</div><div class="citationInfo"><span class="NLM_cas:title">Biophysical Journal</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">91</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2798-2814</span>CODEN:
                <span class="NLM_cas:coden">BIOJAU</span>;
        ISSN:<span class="NLM_cas:issn">0006-3495</span>.
    
            (<span class="NLM_cas:orgname">Biophysical Society</span>)
        </div><div class="casAbstract">The abs. (std.) binding free energy of eight FK506-related ligands to FKBP12 is calcd. using free energy perturbation mol. dynamics (FEP/MD) simulations with explicit solvent.  A no. of features are implemented to improve the accuracy and enhance the convergence of the calcns.  First, the abs. binding free energy is decompd. into sequential steps during which the ligand-surrounding interactions as well as various biasing potentials restraining the translation, orientation, and conformation of the ligand are turned "on" and "off.".  Second, sampling of the ligand conformation is enforced by a restraining potential based on the root mean-square deviation relative to the bound state conformation.  The effect of all the restraining potentials is rigorously unbiased, and it is shown explicitly that the final results are independent of all artificial restraints.  Third, the repulsive and dispersive free energy contribution arising from the Lennard-Jones interactions of the ligand with its surrounding (protein and solvent) is calcd. using the Weeks-Chandler-Andersen sepn.  This sepn. also improves convergence of the FEP/MD calcns.  Fourth, to decrease the computational cost, only a small no. of atoms in the vicinity of the binding site are simulated explicitly, while all the influence of the remaining atoms is incorporated implicitly using the generalized solvent boundary potential (GSBP) method.  With GSBP, the size of the simulated FKBP12/ligand systems is significantly reduced, from ∼25,000 to 2500.  The computations are very efficient and the statistical error is small (∼1 kcal/mol).  The calcd. binding free energies are generally in good agreement with available exptl. data and previous calcns. (within ∼2 kcal/mol).  The present results indicate that a strategy based on FEP/MD simulations of a reduced GSBP at. model sampled with conformational, translational, and orientational restraining potentials can be computationally inexpensive and accurate.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqsDftyP7DXobVg90H21EOLACvtfcHk0liVtjxebPaevA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtVGls7vP&md5=010f74f61048d03fee5dce41172dac70</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1529%2Fbiophysj.106.084301&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1529%252Fbiophysj.106.084301%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DDeng%26aufirst%3DY.%26aulast%3DRoux%26aufirst%3DB.%26atitle%3DAbsolute%2520binding%2520free%2520energy%2520calculations%2520using%2520molecular%2520dynamics%2520simulations%2520with%2520restraining%2520potentials%26jtitle%3DBiophys.%2520J.%26date%3D2006%26volume%3D91%26spage%3D2798%26epage%3D2814%26doi%3D10.1529%2Fbiophysj.106.084301" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group">Jayachandran, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shirts, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Park, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pande, V. S.</span><span> </span><span class="NLM_article-title">Parallelized-over-parts computation of absolute binding free energy with docking and molecular dynamics</span> <span class="citation_source-journal">J. Chem. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">125</span><span class="NLM_x">, </span> <span class="NLM_fpage">084901</span><span class="refDoi"> DOI: 10.1063/1.2221680</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1063%2F1.2221680" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=16965051" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BD28XptlCms7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=125&publication_year=2006&pages=084901&author=G.+Jayachandranauthor=M.+R.+Shirtsauthor=S.+Parkauthor=V.+S.+Pande&title=Parallelized-over-parts+computation+of+absolute+binding+free+energy+with+docking+and+molecular+dynamics&doi=10.1063%2F1.2221680"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Parallelized-over-parts computation of absolute binding free energy with docking and molecular dynamics</span></div><div class="casAuthors">Jayachandran, Guha; Shirts, Michael R.; Park, Sanghyun; Pande, Vijay S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">125</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">084901/1-084901/12</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    
            (<span class="NLM_cas:orgname">American Institute of Physics</span>)
        </div><div class="casAbstract">The authors present a technique for biomol. free energy calcns. that exploits highly parallelized sampling to significantly reduce the time to results.  The technique combines free energies for multiple, nonoverlapping configurational macrostates and is naturally suited to distributed computing.  The authors describe a methodol. that uses this technique with docking, mol. dynamics, and free energy perturbation to compute abs. free energies of binding quickly compared to previous methods.  The method does not require a priori knowledge of the binding pose as long as the docking technique used can generate reasonable binding modes.  The authors demonstrate the method on the protein FKBP12 and eight of its inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr1nPidX9GbT7Vg90H21EOLACvtfcHk0liVtjxebPaevA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XptlCms7c%253D&md5=130cf0d8baef6d086d86d8c4b4fa673a</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1063%2F1.2221680&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.2221680%26sid%3Dliteratum%253Aachs%26aulast%3DJayachandran%26aufirst%3DG.%26aulast%3DShirts%26aufirst%3DM.%2BR.%26aulast%3DPark%26aufirst%3DS.%26aulast%3DPande%26aufirst%3DV.%2BS.%26atitle%3DParallelized-over-parts%2520computation%2520of%2520absolute%2520binding%2520free%2520energy%2520with%2520docking%2520and%2520molecular%2520dynamics%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D2006%26volume%3D125%26spage%3D084901%26doi%3D10.1063%2F1.2221680" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group">Mobley, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chodera, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dill, K. A.</span><span> </span><span class="NLM_article-title">On the use of orientational restraints and symmetry corrections in alchemical free energy calculations</span> <span class="citation_source-journal">J. Chem. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">125</span><span class="NLM_x">, </span> <span class="NLM_fpage">084902</span><span class="refDoi"> DOI: 10.1063/1.2221683</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1063%2F1.2221683" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=16965052" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BD28XptlCmsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=125&publication_year=2006&pages=084902&author=D.+L.+Mobleyauthor=J.+D.+Choderaauthor=K.+A.+Dill&title=On+the+use+of+orientational+restraints+and+symmetry+corrections+in+alchemical+free+energy+calculations&doi=10.1063%2F1.2221683"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">On the use of orientational restraints and symmetry corrections in alchemical free energy calculations</span></div><div class="casAuthors">Mobley, David L.; Chodera, John D.; Dill, Ken A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">125</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">084902/1-084902/16</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    
            (<span class="NLM_cas:orgname">American Institute of Physics</span>)
        </div><div class="casAbstract">Alchem. free energy calcns. are becoming a useful tool for calcg. abs. binding free energies of small mol. ligands to proteins.  Here, we find that the presence of multiple metastable ligand orientations can cause convergence problems when distance restraints alone are used.  We demonstrate that the use of orientational restraints can greatly accelerate the convergence of these calcns.  However, even with this acceleration, we find that sufficient sampling requires substantially longer simulations than are used in many published protocols.  To further accelerate convergence, we introduce a new method of configuration space decompn. by orientation which reduces required simulation lengths by at least a factor of 5 in the cases examd.  Our method is easily parallelizable, well suited for cases where a ligand co-crystal structure is not available, and can utilize initial orientations generated by docking packages.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXKw8APQNZDbVg90H21EOLACvtfcHk0liVtjxebPaevA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XptlCmsL4%253D&md5=8be7d1d066ed42f067182227e952dfe1</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1063%2F1.2221683&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.2221683%26sid%3Dliteratum%253Aachs%26aulast%3DMobley%26aufirst%3DD.%2BL.%26aulast%3DChodera%26aufirst%3DJ.%2BD.%26aulast%3DDill%26aufirst%3DK.%2BA.%26atitle%3DOn%2520the%2520use%2520of%2520orientational%2520restraints%2520and%2520symmetry%2520corrections%2520in%2520alchemical%2520free%2520energy%2520calculations%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D2006%26volume%3D125%26spage%3D084902%26doi%3D10.1063%2F1.2221683" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group">Clark, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guarnieri, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shkurko, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wiseman, J.</span><span> </span><span class="NLM_article-title">Grand canonical Monte Carlo simulation of ligand-protein binding</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">231</span><span class="NLM_x">–</span> <span class="NLM_lpage">242</span><span class="refDoi"> DOI: 10.1021/ci050268f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci050268f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BD2MXht1Gju7zJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2006&pages=231-242&author=M.+Clarkauthor=F.+Guarnieriauthor=I.+Shkurkoauthor=J.+Wiseman&title=Grand+canonical+Monte+Carlo+simulation+of+ligand-protein+binding&doi=10.1021%2Fci050268f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Grand Canonical Monte Carlo Simulation of Ligand-Protein Binding</span></div><div class="casAuthors">Clark, Matthew; Guarnieri, Frank; Shkurko, Igor; Wiseman, Jeff</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">231-242</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A new application of the grand canonical thermodn. ensemble to compute ligand-protein binding is described.  The described method is sufficiently rapid that it is practical to compute ligand-protein binding free energies for a large no. of poses over the entire protein surface, thus identifying multiple putative ligand binding sites.  In addn., the method computes binding free energies for a large no. of poses.  The method is demonstrated by the simulation of two protein-ligand systems, thermolysin and T4 lysozyme, for which there is extensive thermodn. and crystallog. data for the binding of small, rigid ligands.  These low-mol.-wt. ligands correspond to the mol. fragments used in computational fragment-based drug design.  The simulations correctly identified the exptl. binding poses and rank ordered the affinities of ligands in each of these systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqD_W87xCkI47Vg90H21EOLACvtfcHk0liAJrOeJT-tQg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXht1Gju7zJ&md5=6ff1670d21b2650f0eb0a113ab38a1c5</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1021%2Fci050268f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci050268f%26sid%3Dliteratum%253Aachs%26aulast%3DClark%26aufirst%3DM.%26aulast%3DGuarnieri%26aufirst%3DF.%26aulast%3DShkurko%26aufirst%3DI.%26aulast%3DWiseman%26aufirst%3DJ.%26atitle%3DGrand%2520canonical%2520Monte%2520Carlo%2520simulation%2520of%2520ligand-protein%2520binding%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2006%26volume%3D46%26spage%3D231%26epage%3D242%26doi%3D10.1021%2Fci050268f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group">Shirts, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mobley, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chodera, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pande, V. S.</span><span> </span><span class="NLM_article-title">Accurate and efficient corrections for missing dispersion interactions in molecular simulations</span> <span class="citation_source-journal">J. Phys. Chem. B</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">111</span><span class="NLM_x">, </span> <span class="NLM_fpage">13052</span><span class="NLM_x">–</span> <span class="NLM_lpage">13063</span><span class="refDoi"> DOI: 10.1021/jp0735987</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jp0735987" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFyiur3E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2007&pages=13052-13063&author=M.+R.+Shirtsauthor=D.+L.+Mobleyauthor=J.+D.+Choderaauthor=V.+S.+Pande&title=Accurate+and+efficient+corrections+for+missing+dispersion+interactions+in+molecular+simulations&doi=10.1021%2Fjp0735987"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Accurate and Efficient Corrections for Missing Dispersion Interactions in Molecular Simulations</span></div><div class="casAuthors">Shirts, Michael R.; Mobley, David L.; Chodera, John D.; Pande, Vijay S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry B</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">111</span>
        (<span class="NLM_cas:issue">45</span>),
    <span class="NLM_cas:pages">13052-13063</span>CODEN:
                <span class="NLM_cas:coden">JPCBFK</span>;
        ISSN:<span class="NLM_cas:issn">1520-6106</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  In simulations, mol. dispersion interactions are frequently neglected beyond a cutoff of around 1 nm.  In some cases, anal. corrections appropriate for isotropic systems are applied to the pressure and/or the potential energy.  Here, we show that in systems contg. macromols., either of these approaches introduce statistically significant errors in some obsd. properties; for example, the choice of cutoff can affect computed free energies of ligand binding to proteins by 1 to 2 kcal/mol.  We review current methods for eliminating this cutoff-dependent behavior of the dispersion energy and identify some situations where they fail.  We introduce two new formalisms, appropriate for binding free energy calcns., which overcome these failings, requiring minimal computational effort beyond the time required to run the original simulation.  When these cutoff approxns. are applied, which can be done after all simulations are completed, results are consistent across simulations run with different cutoffs.  In many situations, simulations can be run with even shorter cutoffs than typically used, resulting in increased computational efficiency.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo64pX9IFyCi7Vg90H21EOLACvtfcHk0liAJrOeJT-tQg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFyiur3E&md5=1f45b806643cdc5265e94cbfa290669e</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1021%2Fjp0735987&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp0735987%26sid%3Dliteratum%253Aachs%26aulast%3DShirts%26aufirst%3DM.%2BR.%26aulast%3DMobley%26aufirst%3DD.%2BL.%26aulast%3DChodera%26aufirst%3DJ.%2BD.%26aulast%3DPande%26aufirst%3DV.%2BS.%26atitle%3DAccurate%2520and%2520efficient%2520corrections%2520for%2520missing%2520dispersion%2520interactions%2520in%2520molecular%2520simulations%26jtitle%3DJ.%2520Phys.%2520Chem.%2520B%26date%3D2007%26volume%3D111%26spage%3D13052%26epage%3D13063%26doi%3D10.1021%2Fjp0735987" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group">Boyce, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mobley, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rocklin, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Graves, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dill, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Predicting ligand binding affinity with alchemical free energy methods in a polar model binding site</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">394</span><span class="NLM_x">, </span> <span class="NLM_fpage">747</span><span class="NLM_x">–</span> <span class="NLM_lpage">763</span><span class="refDoi"> DOI: 10.1016/j.jmb.2009.09.049</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1016%2Fj.jmb.2009.09.049" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=19782087" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsVKnsbrF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=394&publication_year=2009&pages=747-763&author=S.+E.+Boyceauthor=D.+L.+Mobleyauthor=G.+J.+Rocklinauthor=A.+P.+Gravesauthor=K.+A.+Dillauthor=B.+K.+Shoichet&title=Predicting+ligand+binding+affinity+with+alchemical+free+energy+methods+in+a+polar+model+binding+site&doi=10.1016%2Fj.jmb.2009.09.049"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Predicting Ligand Binding Affinity with Alchemical Free Energy Methods in a Polar Model Binding Site</span></div><div class="casAuthors">Boyce, Sarah E.; Mobley, David L.; Rocklin, Gabriel J.; Graves, Alan P.; Dill, Ken A.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">394</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">747-763</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The authors present a combined exptl. and modeling study of org. ligand mols. binding to a slightly polar engineered cavity site in T4 lysozyme (L99A/M102Q).  For modeling, the authors computed alchem. abs. binding free energies.  These were blind tests performed prospectively on 13 diverse, previously untested candidate ligand mols.  The authors predicted that eight compds. would bind to the cavity and five would not; 11 of 13 predictions were correct at this level.  The RMS error to the measurable abs. binding energies was 1.8 kcal/mol.  In addn., the authors computed "relative" binding free energies for six phenol derivs. starting from two known ligands: phenol and catechol.  The av. RMS error in the relative free energy prediction was 2.5 kcal/mol (phenol) and 1.1 kcal/mol (catechol).  To understand these results at at. resoln., the authors obtained x-ray co-complex structures for nine of the diverse ligands and for all six phenol analogs.  The av. RMSD of the predicted pose to the expt. was 2.0 Å (diverse set), 1.8 Å (phenol-derived predictions), and 1.2 Å (catechol-derived predictions).  The authors found that predicting accurate affinities and rank-orderings required near-native starting orientations of the ligand in the binding site.  Unanticipated binding modes, multiple ligand binding, and protein conformational change all proved challenging for the free energy methods.  The authors believe that these results can help guide future improvements in physics-based abs. binding free energy methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgv5STaggJVrVg90H21EOLACvtfcHk0liAJrOeJT-tQg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsVKnsbrF&md5=5cefcd3613993fed46fc328a905c79c9</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2009.09.049&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2009.09.049%26sid%3Dliteratum%253Aachs%26aulast%3DBoyce%26aufirst%3DS.%2BE.%26aulast%3DMobley%26aufirst%3DD.%2BL.%26aulast%3DRocklin%26aufirst%3DG.%2BJ.%26aulast%3DGraves%26aufirst%3DA.%2BP.%26aulast%3DDill%26aufirst%3DK.%2BA.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DPredicting%2520ligand%2520binding%2520affinity%2520with%2520alchemical%2520free%2520energy%2520methods%2520in%2520a%2520polar%2520model%2520binding%2520site%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2009%26volume%3D394%26spage%3D747%26epage%3D763%26doi%3D10.1016%2Fj.jmb.2009.09.049" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group">Gallicchio, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lapelosa, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Levy, R. M.</span><span> </span><span class="NLM_article-title">The binding energy distribution analysis method (BEDAM) for the estimation of protein-ligand binding affinities</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">2961</span><span class="NLM_x">–</span> <span class="NLM_lpage">2977</span><span class="refDoi"> DOI: 10.1021/ct1002913</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct1002913" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC3cXpvVGgtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2010&pages=2961-2977&author=E.+Gallicchioauthor=M.+Lapelosaauthor=R.+M.+Levy&title=The+binding+energy+distribution+analysis+method+%28BEDAM%29+for+the+estimation+of+protein-ligand+binding+affinities&doi=10.1021%2Fct1002913"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Binding Energy Distribution Analysis Method (BEDAM) for Estimation of Protein-Ligand Binding Affinities</span></div><div class="casAuthors">Gallicchio, Emilio; Lapelosa, Mauro; Levy, Ronald M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2961-2977</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The binding energy distribution anal. method (BEDAM) for the computation of receptor-ligand std. binding free energies with implicit solvation is presented.  The method is based on a well-established statistical mechanics theory of mol. assocn.  In the context of implicit solvation, the theory is homologous to the test particle method of solvation thermodn. with the solute-solvent potential represented by the effective binding energy of the protein-ligand complex.  Accordingly, in BEDAM the binding const. is computed by a weighted integral of the probability distribution of the binding energy obtained in the canonical ensemble in which the ligand is positioned in the binding site but the receptor and the ligand interact only with the solvent continuum.  The binding energy distribution encodes all of the phys. effects of binding.  The balance between binding enthalpy and entropy is seen in the authors' formalism as a balance between favorable and unfavorable binding modes which are coupled through the normalization of the binding energy distribution function.  An efficient computational protocol for the binding energy distribution based on the AGBNP2 implicit solvent model, parallel Hamiltonian replica exchange sampling, and histogram reweighting is developed.  Applications of the method to a set of known binders and nonbinders of the L99A and L99A/M102Q mutants of T4 lysozyme receptor are illustrated.  The method is able to discriminate without error binders from nonbinders, and the computed std. binding free energies of the binders are in good agreement with exptl. measurements.  Anal. of the binding affinities of these systems reflect the contributions from multiple conformations spanning a wide range of binding energies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqe_4e1f7O1XLVg90H21EOLACvtfcHk0liAJrOeJT-tQg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXpvVGgtr0%253D&md5=190e552ee54264185012f053e42488b2</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1021%2Fct1002913&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct1002913%26sid%3Dliteratum%253Aachs%26aulast%3DGallicchio%26aufirst%3DE.%26aulast%3DLapelosa%26aufirst%3DM.%26aulast%3DLevy%26aufirst%3DR.%2BM.%26atitle%3DThe%2520binding%2520energy%2520distribution%2520analysis%2520method%2520%2528BEDAM%2529%2520for%2520the%2520estimation%2520of%2520protein-ligand%2520binding%2520affinities%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2010%26volume%3D6%26spage%3D2961%26epage%3D2977%26doi%3D10.1021%2Fct1002913" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group">Fujitani, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tanida, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matsuura, A.</span><span> </span><span class="NLM_article-title">Massively parallel computation of absolute binding free energy with well-equilibrated states</span> <span class="citation_source-journal">Phys. Rev. E: Stat., Nonlinear, Soft Matter Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">79</span><span class="NLM_x">, </span> <span class="NLM_fpage">021914</span><span class="refDoi"> DOI: 10.1103/PhysRevE.79.021914</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1103%2FPhysRevE.79.021914" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=19391785" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BD1MXisFOjsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=2009&pages=021914&author=H.+Fujitaniauthor=Y.+Tanidaauthor=A.+Matsuura&title=Massively+parallel+computation+of+absolute+binding+free+energy+with+well-equilibrated+states&doi=10.1103%2FPhysRevE.79.021914"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Massively parallel computation of absolute binding free energy with well-equilibrated states</span></div><div class="casAuthors">Fujitani, Hideaki; Tanida, Yoshiaki; Matsuura, Azuma</div><div class="citationInfo"><span class="NLM_cas:title">Physical Review E: Statistical, Nonlinear, and Soft Matter Physics</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">79</span>
        (<span class="NLM_cas:issue">2-1</span>),
    <span class="NLM_cas:pages">021914/1-021914/12</span>CODEN:
                <span class="NLM_cas:coden">PRESCM</span>;
        ISSN:<span class="NLM_cas:issn">1539-3755</span>.
    
            (<span class="NLM_cas:orgname">American Physical Society</span>)
        </div><div class="casAbstract">A force field formulator for org. mols. (FF-FOM) was developed to assign bond, angle, and dihedral parameters to arbitrary org. mols. in a unified manner including proteins and nucleic acids.  With the unified force field parametrization we performed massively parallel computations of abs. binding free energies for pharmaceutical target proteins and ligands.  Compared with the previous calcn. with the ff99 force field in the Amber simulation package (Amber99) and the ligand charges produced by the Austin Model 1 bond charge correction (AM1-BCC), the unified parametrization gave better abs. binding energies for the FK506 binding protein (FKBP) and ligand system.  Our method is based on extensive work measurement between thermodn. states to calc. the free energy difference and it is also the same as the traditional free energy perturbation.  There are important requirements for accurate calcns.  The first is a well-equilibrated bound structure including the conformational change of the protein induced by the binding of the ligand.  The second requirement is the convergence of the work distribution with a sufficient no. of trajectories and dense spacing of the coupling const. between the ligand and the rest of the system.  Finally, the most important requirement is the force field parametrization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrl-MtVRv_gpbVg90H21EOLACvtfcHk0lirKWWCVy6VOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXisFOjsbY%253D&md5=4aca64ba22cff055f1cd39619f798fda</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1103%2FPhysRevE.79.021914&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1103%252FPhysRevE.79.021914%26sid%3Dliteratum%253Aachs%26aulast%3DFujitani%26aufirst%3DH.%26aulast%3DTanida%26aufirst%3DY.%26aulast%3DMatsuura%26aufirst%3DA.%26atitle%3DMassively%2520parallel%2520computation%2520of%2520absolute%2520binding%2520free%2520energy%2520with%2520well-equilibrated%2520states%26jtitle%3DPhys.%2520Rev.%2520E%253A%2520Stat.%252C%2520Nonlinear%252C%2520Soft%2520Matter%2520Phys.%26date%3D2009%26volume%3D79%26spage%3D021914%26doi%3D10.1103%2FPhysRevE.79.021914" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group">Ytreberg, F. M.</span><span> </span><span class="NLM_article-title">Absolute FKBP binding affinities obtained via nonequilibrium unbinding simulations</span> <span class="citation_source-journal">J. Chem. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">130</span><span class="NLM_x">, </span> <span class="NLM_fpage">164906</span><span class="refDoi"> DOI: 10.1063/1.3119261</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1063%2F1.3119261" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=19405629" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BD1MXltV2nsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=130&publication_year=2009&pages=164906&author=F.+M.+Ytreberg&title=Absolute+FKBP+binding+affinities+obtained+via+nonequilibrium+unbinding+simulations&doi=10.1063%2F1.3119261"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Absolute FKBP binding affinities obtained via nonequilibrium unbinding simulations</span></div><div class="casAuthors">Ytreberg, F. Marty</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">130</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">164906/1-164906/8</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    
            (<span class="NLM_cas:orgname">American Institute of Physics</span>)
        </div><div class="casAbstract">We compute the abs. binding affinities for two ligands bound to the FKBP protein using nonequil. unbinding simulations.  The methodol. is straightforward, requiring little or no modification to many modern mol. simulation packages.  The approach makes use of a phys. pathway, eliminating the need for complicated alchem. decoupling schemes.  We compare our nonequil. results to those obtained via a fully equil. approach and to expt.  The results of this study suggest that to obtain accurate results using nonequil. approaches one should use the stiff-spring approxn. with the second cumulant expansion.  From this study we conclude that nonequil. simulation could provide a simple means to est. protein-ligand binding affinities.  (c) 2009 American Institute of Physics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbsbHuKp3bD7Vg90H21EOLACvtfcHk0lirKWWCVy6VOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXltV2nsbY%253D&md5=570ec255ee04e2d7884da4448eb5bbc4</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1063%2F1.3119261&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.3119261%26sid%3Dliteratum%253Aachs%26aulast%3DYtreberg%26aufirst%3DF.%2BM.%26atitle%3DAbsolute%2520FKBP%2520binding%2520affinities%2520obtained%2520via%2520nonequilibrium%2520unbinding%2520simulations%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D2009%26volume%3D130%26spage%3D164906%26doi%3D10.1063%2F1.3119261" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group">Skouteris, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Calderini, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barone, V.</span><span> </span><span class="NLM_article-title">Methods for calculating partition functions of molecules involving large amplitude and/or anharmonic motions</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">1011</span><span class="NLM_x">–</span> <span class="NLM_lpage">1018</span><span class="refDoi"> DOI: 10.1021/acs.jctc.5b01094</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jctc.5b01094" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC28XnvFCkug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=1011-1018&author=D.+Skouterisauthor=D.+Calderiniauthor=V.+Barone&title=Methods+for+calculating+partition+functions+of+molecules+involving+large+amplitude+and%2For+anharmonic+motions&doi=10.1021%2Facs.jctc.5b01094"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Methods for Calculating Partition Functions of Molecules Involving Large Amplitude and/or Anharmonic Motions</span></div><div class="casAuthors">Skouteris, D.; Calderini, D.; Barone, V.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1011-1018</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We present a method for calcg. partition functions taking into account anharmonic contributions for systems involving both small-amplitude vibrations and hindered rotations.  The Wang-Landau scheme is used in the first case, while two alternative schemes are used for hindered rotation based on imaginary time propagation and fitting of the exact energy levels as a function of quantum no.  These two schemes are shown to be complementary in their ranges of applicability (in terms of the torsional rotational const. and the relevant potential).  Partition functions for four different mols. are calcd. and compared to simpler ones obtained using a harmonic model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpgUPwtr0e_lrVg90H21EOLACvtfcHk0lirKWWCVy6VOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XnvFCkug%253D%253D&md5=6471c27f9d9d2baf764e20a1439eef8c</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1021%2Facs.jctc.5b01094&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jctc.5b01094%26sid%3Dliteratum%253Aachs%26aulast%3DSkouteris%26aufirst%3DD.%26aulast%3DCalderini%26aufirst%3DD.%26aulast%3DBarone%26aufirst%3DV.%26atitle%3DMethods%2520for%2520calculating%2520partition%2520functions%2520of%2520molecules%2520involving%2520large%2520amplitude%2520and%252For%2520anharmonic%2520motions%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2016%26volume%3D12%26spage%3D1011%26epage%3D1018%26doi%3D10.1021%2Facs.jctc.5b01094" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group">Boresch, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tettinger, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leitgeb, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karplus, M.</span><span> </span><span class="NLM_article-title">Absolute binding free energies: A quantitative approach for their calculation</span> <span class="citation_source-journal">J. Phys. Chem. B</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">107</span><span class="NLM_x">, </span> <span class="NLM_fpage">9535</span><span class="NLM_x">–</span> <span class="NLM_lpage">9551</span><span class="refDoi"> DOI: 10.1021/jp0217839</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jp0217839" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BD3sXmsVegsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=107&publication_year=2003&pages=9535-9551&author=S.+Boreschauthor=F.+Tettingerauthor=M.+Leitgebauthor=M.+Karplus&title=Absolute+binding+free+energies%3A+A+quantitative+approach+for+their+calculation&doi=10.1021%2Fjp0217839"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Absolute Binding Free Energies: A Quantitative Approach for Their Calculation</span></div><div class="casAuthors">Boresch, Stefan; Tettinger, Franz; Leitgeb, Martin; Karplus, Martin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry B</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">107</span>
        (<span class="NLM_cas:issue">35</span>),
    <span class="NLM_cas:pages">9535-9551</span>CODEN:
                <span class="NLM_cas:coden">JPCBFK</span>;
        ISSN:<span class="NLM_cas:issn">1520-6106</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The computation of abs. binding affinities by mol. dynamics (MD) based free energy simulations is analyzed, and an exact method to carry out such a computation is presented.  The key to obtaining converged results is the introduction of suitable, auxiliary restraints to prevent the ligand from leaving the binding site when the native ligand-receptor interactions are turned off alchem.  The authors describe a versatile set of restraints that (i) can be used in MD simulations, that (ii) restricts both the position and the orientation of the ligand, and that (iii) is defined relative to the receptor rather than relative to a fixed point in space.  The free energy cost, ΔAr, for this set of restraints can be evaluated anal.  Although the techniques were originally developed for the gas phase, the resulting expression is exact, since all contributions from solute-solvent interactions cancel from the final result.  The value of ΔAr depends only on the equil. values and force consts. of the chosen harmonic restraint terms and, therefore, can be easily calcd.  The std. state dependence of binding free energies is also investigated, and it is shown that the present approach takes this into account correctly.  The anal. expression for ΔAr is verified numerically by calcns. on the complex formed by benzene with the L99A mutant of T4 lysozyme.  The overall approach is illustrated by a complete binding free energy calcn. for a complex based on a simplified model for tyrosine bound to tyrosyl-tRNA-synthetase.  The results demonstrate the usefulness of the proposed set of restraints and confirm that the calcd. binding free energy is independent of the details of the restraints.  Comparisons are made with earlier formulations for the calcn. of binding free energies, and certain limitations of that work are described.  The relationship between ΔAr and the loss of translational and rotational entropy during a binding process is analyzed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDzHeD7X8aRbVg90H21EOLACvtfcHk0lj4cqwpYFajuw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXmsVegsr4%253D&md5=a6b92d5349e6a9bd5912f528dc948ce0</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1021%2Fjp0217839&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp0217839%26sid%3Dliteratum%253Aachs%26aulast%3DBoresch%26aufirst%3DS.%26aulast%3DTettinger%26aufirst%3DF.%26aulast%3DLeitgeb%26aufirst%3DM.%26aulast%3DKarplus%26aufirst%3DM.%26atitle%3DAbsolute%2520binding%2520free%2520energies%253A%2520A%2520quantitative%2520approach%2520for%2520their%2520calculation%26jtitle%3DJ.%2520Phys.%2520Chem.%2520B%26date%3D2003%26volume%3D107%26spage%3D9535%26epage%3D9551%26doi%3D10.1021%2Fjp0217839" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group">Brown, S. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muchmore, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hajduk, P. J.</span><span> </span><span class="NLM_article-title">Healthy skepticism: assessing realistic model performance</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">420</span><span class="NLM_x">–</span> <span class="NLM_lpage">427</span><span class="refDoi"> DOI: 10.1016/j.drudis.2009.01.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1016%2Fj.drudis.2009.01.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=19340931" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A280%3ADC%252BD1M3ltlaqsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2009&pages=420-427&author=S.+P.+Brownauthor=S.+W.+Muchmoreauthor=P.+J.+Hajduk&title=Healthy+skepticism%3A+assessing+realistic+model+performance&doi=10.1016%2Fj.drudis.2009.01.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Healthy skepticism: assessing realistic model performance</span></div><div class="casAuthors">Brown Scott P; Muchmore Steven W; Hajduk Philip J</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">7-8</span>),
    <span class="NLM_cas:pages">420-7</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Although the development of computational models to aid drug discovery has become an integral part of pharmaceutical research, the application of these models often fails to produce the expected impact on productivity.  One reason for this may be that the expected performance of many models is simply not supported by the underlying data, because of often neglected effects of assay and prediction errors on the reliability of the predicted outcome.  Another significant challenge to realizing the full potential of computational models is their integration into prospective medicinal chemistry campaigns.  This article will analyze the impact of assay and prediction error on model quality, and explore scenarios where computational models can expect to have a significant influence on drug discovery research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRCGY9Wj1ldXiCEBeYuoi7AfW6udTcc2eZK25b6Lmqqwrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1M3ltlaqsQ%253D%253D&md5=6ca35f48ea556a47040f91e4d84e4b7a</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2009.01.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2009.01.012%26sid%3Dliteratum%253Aachs%26aulast%3DBrown%26aufirst%3DS.%2BP.%26aulast%3DMuchmore%26aufirst%3DS.%2BW.%26aulast%3DHajduk%26aufirst%3DP.%2BJ.%26atitle%3DHealthy%2520skepticism%253A%2520assessing%2520realistic%2520model%2520performance%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2009%26volume%3D14%26spage%3D420%26epage%3D427%26doi%3D10.1016%2Fj.drudis.2009.01.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group">Reddy, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Erion, M. D.</span><span> </span><span class="NLM_article-title">Calculation of relative binding free energy differences for fructose 1,6-bisphosphatase inhibitors using the thermodynamic cycle perturbation approach</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">123</span><span class="NLM_x">, </span> <span class="NLM_fpage">6246</span><span class="NLM_x">–</span> <span class="NLM_lpage">6252</span><span class="refDoi"> DOI: 10.1021/ja0103288</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja0103288" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BD3MXktFamu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=123&publication_year=2001&pages=6246-6252&author=M.+R.+Reddyauthor=M.+D.+Erion&title=Calculation+of+relative+binding+free+energy+differences+for+fructose+1%2C6-bisphosphatase+inhibitors+using+the+thermodynamic+cycle+perturbation+approach&doi=10.1021%2Fja0103288"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Calculation of Relative Binding Free Energy Differences for Fructose 1,6-Bisphosphatase Inhibitors Using the Thermodynamic Cycle Perturbation Approach</span></div><div class="casAuthors">Reddy, M. Rami; Erion, Mark D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">123</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">6246-6252</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">An iterative, computer-assisted, drug design strategy that combines mol. design, mol. mechanics, mol. dynamics (MD), and free energy perturbation (FEP) calcns. with compd. synthesis, biochem. testing of inhibitors, and crystallog. structure detn. of protein-inhibitor complexes was successfully used to predict the rank order of a series of nucleoside monophosphate analogs as fructose 1,6-bisphosphatase (FBPase) inhibitors.  The x-ray structure of FBPase complexed with 5-aminoimidazole-4-carboxamide-1-β-D-ribofuranosyl 5'-monophosphate (ZMP) provided structural information used for subsequent analog design and free energy calcns.  The FEP protocol was validated by calcg. the free energy differences for the mutation of ZMP to AMP.  The calcd. results showed a net gain of 1.7 kcal/mol, which agreed with the exptl. result of 1.3 kcal/mol. FEP calcns. were performed for 18 other AMP analogs.  Inhibition consts. were detd. for over half of these analogs, usually after completion of the calcn., and were consistent with the predictions.  Solvation free energy differences between AMP and various AMP analogs proved to be an important factor in binding free energies, suggesting that increased desolvation costs assocd. with the addn. of polar groups to an inhibitor must be overcome by stronger ligand-protein interactions if the structural modification is to enhance inhibitor potency.  The results indicate that FEP calcns. predict relative binding affinities with high accuracy and provide valuable insight into the factors that influence inhibitor binding and therefore should greatly aid efforts to optimize initial lead compds. and reduce the time required for the discovery of new drug candidates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9zsaQP6gdLrVg90H21EOLACvtfcHk0liobq-aQsvuFw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXktFamu70%253D&md5=976a8de912c52216e26fd84a106dbca4</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1021%2Fja0103288&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja0103288%26sid%3Dliteratum%253Aachs%26aulast%3DReddy%26aufirst%3DM.%2BR.%26aulast%3DErion%26aufirst%3DM.%2BD.%26atitle%3DCalculation%2520of%2520relative%2520binding%2520free%2520energy%2520differences%2520for%2520fructose%25201%252C6-bisphosphatase%2520inhibitors%2520using%2520the%2520thermodynamic%2520cycle%2520perturbation%2520approach%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2001%26volume%3D123%26spage%3D6246%26epage%3D6252%26doi%3D10.1021%2Fja0103288" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group">Michel, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verdonk, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Essex, J. W.</span><span> </span><span class="NLM_article-title">Protein-ligand complexes: computation of the relative free energy of different scaffolds and binding modes</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">1645</span><span class="NLM_x">–</span> <span class="NLM_lpage">1655</span><span class="refDoi"> DOI: 10.1021/ct700081t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct700081t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BD2sXos1Srsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2007&pages=1645-1655&author=J.+Michelauthor=M.+L.+Verdonkauthor=J.+W.+Essex&title=Protein-ligand+complexes%3A+computation+of+the+relative+free+energy+of+different+scaffolds+and+binding+modes&doi=10.1021%2Fct700081t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Protein-Ligand Complexes: Computation of the Relative Free Energy of Different Scaffolds and Binding Modes</span></div><div class="casAuthors">Michel, Julien; Verdonk, Marcel L.; Essex, Jonathan W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1645-1655</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A methodol. for the calcn. of the free energy difference between a pair of mols. of arbitrary topol. is proposed.  The protocol relies on a dual-topol. paradigm, a softening of the intermol. interactions, and a constraint that prevents the perturbed mols. from drifting away from each other at the end states.  The equivalence and the performance of the methodol. against a single-topol. approach are demonstrated on a pair of harmonic oscillators, the calcn. of the relative solvation free energy of ethane and methanol, and the relative binding free energy of two congeneric inhibitors of cyclooxygenase 2.  The stability of two alternative binding modes of an inhibitor of cyclin-dependent kinase 2 is then investigated.  Finally, the relative binding free energy of two structurally different inhibitors of cyclin-dependent kinase 2 is calcd.  The proposed methodol. allows the study of a range of problems that are beyond the reach of traditional relative free energy calcn. protocols and should prove useful in drug design studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7vFYftcaZaLVg90H21EOLACvtfcHk0liobq-aQsvuFw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXos1Srsbw%253D&md5=ae0a799b4119091b801471cd8073b6db</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1021%2Fct700081t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct700081t%26sid%3Dliteratum%253Aachs%26aulast%3DMichel%26aufirst%3DJ.%26aulast%3DVerdonk%26aufirst%3DM.%2BL.%26aulast%3DEssex%26aufirst%3DJ.%2BW.%26atitle%3DProtein-ligand%2520complexes%253A%2520computation%2520of%2520the%2520relative%2520free%2520energy%2520of%2520different%2520scaffolds%2520and%2520binding%2520modes%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2007%26volume%3D3%26spage%3D1645%26epage%3D1655%26doi%3D10.1021%2Fct700081t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group">Mutyala, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reddy, R. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sumakanth, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reddanna, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reddy, M. R.</span><span> </span><span class="NLM_article-title">Calculation of relative binding affinities of fructose 1,6-bisphosphatase mutants with adenosine monophosphate using free energy perturbation method</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">932</span><span class="NLM_x">–</span> <span class="NLM_lpage">937</span><span class="refDoi"> DOI: 10.1002/jcc.20617</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1002%2Fjcc.20617" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=17253638" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BD2sXjtFCit7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2007&pages=932-937&author=R.+Mutyalaauthor=R.+N.+Reddyauthor=M.+Sumakanthauthor=P.+Reddannaauthor=M.+R.+Reddy&title=Calculation+of+relative+binding+affinities+of+fructose+1%2C6-bisphosphatase+mutants+with+adenosine+monophosphate+using+free+energy+perturbation+method&doi=10.1002%2Fjcc.20617"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Calculation of relative binding affinities of fructose 1,6-bisphosphatase mutants with adenosine monophosphate using free energy perturbation method</span></div><div class="casAuthors">Mutyala, Ravichandra; Reddy, R. N.; Sumakanth, M.; Reddanna, P.; Reddy, M. Rami</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">932-937</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">The free energy perturbation (FEP) methodol. is the most accurate means of estg. relative binding affinities between inhibitors and protein variants.  In this article, the importance of hydrophobic and hydrophilic residues to the binding of adenosine monophosphate (AMP) to the fructose 1,6-bisphosphatase (FBPase), a target enzyme for type-II diabetes, was examd. by FEP method.  Five mutations were made to the FBPase enzyme with AMP inhibitor bound: 113Tyr → 113Phe, 31Thr → 31Ala, 31Thr → 31Ser, 177Met → 177Ala, and 30Leu → 30Phe.  These mutations test the strength of hydrogen bonds and Van der Waals interactions between the ligand and enzyme.  The calcd. relative free energies indicated that: 113Tyr and 31Thr play an important role, each via two hydrogen bonds affecting the binding affinity of inhibitor AMP to FBPase, and any changes in these hydrogen bonds due to mutations on the protein will have significant effect on the binding affinity of AMP to FBPase, consistent to exptl. results.  Also, the free energy calcns. clearly show that the hydrophilic interactions are more important than the hydrophobic interactions of the binding pocket of FBPase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6IknNoSsHVbVg90H21EOLACvtfcHk0liobq-aQsvuFw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXjtFCit7o%253D&md5=a04352752c6bcbe5f23820b7e48152ac</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1002%2Fjcc.20617&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.20617%26sid%3Dliteratum%253Aachs%26aulast%3DMutyala%26aufirst%3DR.%26aulast%3DReddy%26aufirst%3DR.%2BN.%26aulast%3DSumakanth%26aufirst%3DM.%26aulast%3DReddanna%26aufirst%3DP.%26aulast%3DReddy%26aufirst%3DM.%2BR.%26atitle%3DCalculation%2520of%2520relative%2520binding%2520affinities%2520of%2520fructose%25201%252C6-bisphosphatase%2520mutants%2520with%2520adenosine%2520monophosphate%2520using%2520free%2520energy%2520perturbation%2520method%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2007%26volume%3D28%26spage%3D932%26epage%3D937%26doi%3D10.1002%2Fjcc.20617" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group">Reddy, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Erion, M. D.</span><span> </span><span class="NLM_article-title">Relative binding affinities of fructose-1,6-bisphosphatase inhibitors calculated using a quantum mechanics-based free energy perturbation method</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">129</span><span class="NLM_x">, </span> <span class="NLM_fpage">9296</span><span class="NLM_x">–</span> <span class="NLM_lpage">9297</span><span class="refDoi"> DOI: 10.1021/ja072905j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja072905j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=129&publication_year=2007&pages=9296-9297&author=M.+R.+Reddyauthor=M.+D.+Erion&title=Relative+binding+affinities+of+fructose-1%2C6-bisphosphatase+inhibitors+calculated+using+a+quantum+mechanics-based+free+energy+perturbation+method&doi=10.1021%2Fja072905j"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1021%2Fja072905j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja072905j%26sid%3Dliteratum%253Aachs%26aulast%3DReddy%26aufirst%3DM.%2BR.%26aulast%3DErion%26aufirst%3DM.%2BD.%26atitle%3DRelative%2520binding%2520affinities%2520of%2520fructose-1%252C6-bisphosphatase%2520inhibitors%2520calculated%2520using%2520a%2520quantum%2520mechanics-based%2520free%2520energy%2520perturbation%2520method%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2007%26volume%3D129%26spage%3D9296%26epage%3D9297%26doi%3D10.1021%2Fja072905j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group">Lee, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Park, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, S.</span><span> </span><span class="NLM_article-title">Molecular dynamics free energy simulation study to rationalize the relative activities of PPAR delta agonists</span> <span class="citation_source-journal">Bull. Korean Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x">, </span> <span class="NLM_fpage">363</span><span class="NLM_x">–</span> <span class="NLM_lpage">371</span><span class="refDoi"> DOI: 10.5012/bkcs.2008.29.2.363</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.5012%2Fbkcs.2008.29.2.363" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BD1cXjvVaktrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2008&pages=363-371&author=W.+Leeauthor=H.+Parkauthor=S.+Lee&title=Molecular+dynamics+free+energy+simulation+study+to+rationalize+the+relative+activities+of+PPAR+delta+agonists&doi=10.5012%2Fbkcs.2008.29.2.363"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular dynamics free energy simulation study to rationalize the relative activities of PPARδ agonists</span></div><div class="casAuthors">Lee, Woojin; Park, Hwangseo; Lee, Sangyoub</div><div class="citationInfo"><span class="NLM_cas:title">Bulletin of the Korean Chemical Society</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">363-371</span>CODEN:
                <span class="NLM_cas:coden">BKCSDE</span>;
        ISSN:<span class="NLM_cas:issn">0253-2964</span>.
    
            (<span class="NLM_cas:orgname">Korean Chemical Society</span>)
        </div><div class="casAbstract">As a computational method for the discovery of the effective agonists for PPARδ, we address the usefulness of mol. dynamics free energy (MDFE) simulation with explicit solvent in terms of the accuracy and the computing cost.  For this purpose, we establish an efficient computational protocol of thermodn. integration (TI) that is superior to free energy perturbation (FEP) method in parallel computing environment.  Using this protocol, the relative binding affinities of GW501516 and its derivs. for PPARδ are calcd.  The accuracy of our protocol was evaluated in two steps.  First, we devise a thermodn. cycle to calc. the abs. and relative hydration free energies of test mols.  This allows a self-consistent check for the accuracy of the calcn. protocol.  Second, the calcd. relative binding affinities of the selected ligands are compared with exptl. IC50 values.  The av. deviation of the calcd. binding free energies from the exptl. results amts. at the most to 1 kcal/mol.  The computational efficiency of current protocol is also assessed by comparing its execution times with those of the sequential version of the TI protocol.  The results show that the calcn. can be accelerated by 4 times when compared to the sequential run.  Based on the calcns. with the parallel computational protocol, a new potential agonist of GW501516 deriv. is proposed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqewQSkpQBhkLVg90H21EOLACvtfcHk0lgAFRRtM-YsjQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXjvVaktrw%253D&md5=e1efc86bd9c5eca3c1c25e6950db9add</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.5012%2Fbkcs.2008.29.2.363&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.5012%252Fbkcs.2008.29.2.363%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DW.%26aulast%3DPark%26aufirst%3DH.%26aulast%3DLee%26aufirst%3DS.%26atitle%3DMolecular%2520dynamics%2520free%2520energy%2520simulation%2520study%2520to%2520rationalize%2520the%2520relative%2520activities%2520of%2520PPAR%2520delta%2520agonists%26jtitle%3DBull.%2520Korean%2520Chem.%2520Soc.%26date%3D2008%26volume%3D29%26spage%3D363%26epage%3D371%26doi%3D10.5012%2Fbkcs.2008.29.2.363" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit63"><span><span class="NLM_contrib-group">Shine, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kikuchi, T.</span><span> </span><span class="NLM_article-title">Estimation of relative binding free energy based on a free energy variational principle for quantitative structure activity relationship analyses</span> <span class="citation_source-journal">Chem. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">365</span><span class="NLM_x">, </span> <span class="NLM_fpage">53</span><span class="NLM_x">–</span> <span class="NLM_lpage">59</span><span class="refDoi"> DOI: 10.1016/j.chemphys.2009.09.023</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1016%2Fj.chemphys.2009.09.023" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=365&publication_year=2009&pages=53-59&author=Y.+Shineauthor=T.+Kikuchi&title=Estimation+of+relative+binding+free+energy+based+on+a+free+energy+variational+principle+for+quantitative+structure+activity+relationship+analyses&doi=10.1016%2Fj.chemphys.2009.09.023"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1016%2Fj.chemphys.2009.09.023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chemphys.2009.09.023%26sid%3Dliteratum%253Aachs%26aulast%3DShine%26aufirst%3DY.%26aulast%3DKikuchi%26aufirst%3DT.%26atitle%3DEstimation%2520of%2520relative%2520binding%2520free%2520energy%2520based%2520on%2520a%2520free%2520energy%2520variational%2520principle%2520for%2520quantitative%2520structure%2520activity%2520relationship%2520analyses%26jtitle%3DChem.%2520Phys.%26date%3D2009%26volume%3D365%26spage%3D53%26epage%3D59%26doi%3D10.1016%2Fj.chemphys.2009.09.023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref64"><div class="reference"><strong class="refLabel"><a href="#ref64" class="rightTabRefNumLink">64</a></strong><div class="NLM_citation" id="rightTab-cit64"><span><span class="NLM_contrib-group">Zhao, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caplan, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Noskov, S. Y.</span><span> </span><span class="NLM_article-title">Evaluations of the absolute and relative free energies for antidepressant binding to the amino acid membrane transporter LeuT with free energy simulations</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">1900</span><span class="NLM_x">–</span> <span class="NLM_lpage">1914</span><span class="refDoi"> DOI: 10.1021/ct9006597</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct9006597" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC3cXltlygtL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2010&pages=1900-1914&author=C.+Zhaoauthor=D.+A.+Caplanauthor=S.+Y.+Noskov&title=Evaluations+of+the+absolute+and+relative+free+energies+for+antidepressant+binding+to+the+amino+acid+membrane+transporter+LeuT+with+free+energy+simulations&doi=10.1021%2Fct9006597"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Evaluations of the Absolute and Relative Free Energies for Antidepressant Binding to the Amino Acid Membrane Transporter LeuT with Free Energy Simulations</span></div><div class="casAuthors">Zhao, Chunfeng; Caplan, David A.; Noskov, Sergei Yu.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1900-1914</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The binding of ligands to protein receptors with high affinity and specificity is central to many cellular processes.  The quest for the development of computational models capable of accurately evaluating binding affinity remains one of the main goals of modern computational biophysics.  In this work, free energy perturbation/mol. dynamics simulations were used to evaluate abs. and relative binding affinity for three different antidepressants to a sodium-dependent membrane transporter, LeuT, a bacterial homolog of human serotonin and dopamine transporters.  Dysfunction of these membrane transporters in mammals has been implicated in multiple diseases of the nervous system, including bipolar disorder and depression.  Furthermore, these proteins are key targets for antidepressants including fluoxetine (aka Prozac) and tricyclic antidepressants known to block transport activity.  In addn. to being clin. relevant, this system, where multiple crystal structures are readily available, represents an ideal testing ground for methods used to study the mol. mechanisms of ligand binding to membrane proteins.  We discuss possible pitfalls and different levels of approxn. required to evaluate binding affinity, such as the dependence of the computed affinities on the strength of constraints and the sensitivity of the computed affinities to the particular partial charges derived from restrained electrostatic potential fitting of quantum mechanics electrostatic potential.  Finally, we compare the effects of different constraint schemes on the abs. and relative binding affinities obtained from free energy simulations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrIGzu_SqTUlLVg90H21EOLACvtfcHk0lizZe9dXk5ntA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXltlygtL4%253D&md5=bf8f2709f5b82ebb685ac5da0a748c56</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1021%2Fct9006597&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct9006597%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DC.%26aulast%3DCaplan%26aufirst%3DD.%2BA.%26aulast%3DNoskov%26aufirst%3DS.%2BY.%26atitle%3DEvaluations%2520of%2520the%2520absolute%2520and%2520relative%2520free%2520energies%2520for%2520antidepressant%2520binding%2520to%2520the%2520amino%2520acid%2520membrane%2520transporter%2520LeuT%2520with%2520free%2520energy%2520simulations%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2010%26volume%3D6%26spage%3D1900%26epage%3D1914%26doi%3D10.1021%2Fct9006597" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref65"><div class="reference"><strong class="refLabel"><a href="#ref65" class="rightTabRefNumLink">65</a></strong><div class="NLM_citation" id="rightTab-cit65"><span><span class="NLM_contrib-group">Palma, P. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bonifacio, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Loureiro, A. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soares-da-Silva, P.</span><span> </span><span class="NLM_article-title">Computation of the binding affinities of catechol-O-methyltransferase inhibitors: multisubstate relative free energy calculations</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">33</span><span class="NLM_x">, </span> <span class="NLM_fpage">970</span><span class="NLM_x">–</span> <span class="NLM_lpage">986</span><span class="refDoi"> DOI: 10.1002/jcc.22926</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1002%2Fjcc.22926" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=22278964" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1ygtbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2012&pages=970-986&author=P.+N.+Palmaauthor=M.+J.+Bonifacioauthor=A.+I.+Loureiroauthor=P.+Soares-da-Silva&title=Computation+of+the+binding+affinities+of+catechol-O-methyltransferase+inhibitors%3A+multisubstate+relative+free+energy+calculations&doi=10.1002%2Fjcc.22926"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Computation of the binding affinities of catechol-O-methyltransferase inhibitors: Multisubstate relative free energy calculations</span></div><div class="casAuthors">Palma, P. Nuno; Bonifacio, Maria Joao; Loureiro, Ana Isabel; Soares-da-Silva, Patricio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">970-986</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">Alchem. free energy simulations are amongst the most accurate techniques for the computation of the free energy changes assocd. with noncovalent protein-ligand interactions.  A procedure is presented to est. the relative binding free energies of several ligands to the same protein target where multiple, low-energy configurational substates might coexist, as opposed to one unique structure.  The contributions of all individual substates were estd., explicitly, with the free energy perturbation method, and combined in a rigorous fashion to compute the overall relative binding free energies and dissocn. consts.  It is shown that, unless the most stable bound forms are known a priori, inaccurate results may be obtained if the contributions of multiple substates are ignored.  The method was applied to study the complex formed between human catechol-O-methyltransferase and BIA 9-1067, a newly developed tight-binding inhibitor that is currently under clin. evaluation for the therapy of Parkinson's disease.  Our results reveal an exceptionally high-binding affinity (Kd in subpicomolar range) and provide insightful clues on the interactions and mechanism of inhibition.  The inhibitor is, itself, a slowly reacting substrate of the target enzyme and is released from the complex in the form of O-methylated product.  By comparing the exptl. catalytic rate (kcat) and the estd. dissocn. rate (koff) consts. of the enzyme-inhibitor complex, one can conclude that the obsd. inhibition potency (Ki) is primarily dependent on the catalytic rate const. of the inhibitor's O-methylation, rather than the rate const. of dissocn. of the complex. © 2012 Wiley Periodicals, Inc.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTjDjuZ-BtVbVg90H21EOLACvtfcHk0lhfIZP28_nCwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1ygtbk%253D&md5=548b34974096e60eefb9ad605de2d8ba</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1002%2Fjcc.22926&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.22926%26sid%3Dliteratum%253Aachs%26aulast%3DPalma%26aufirst%3DP.%2BN.%26aulast%3DBonifacio%26aufirst%3DM.%2BJ.%26aulast%3DLoureiro%26aufirst%3DA.%2BI.%26aulast%3DSoares-da-Silva%26aufirst%3DP.%26atitle%3DComputation%2520of%2520the%2520binding%2520affinities%2520of%2520catechol-O-methyltransferase%2520inhibitors%253A%2520multisubstate%2520relative%2520free%2520energy%2520calculations%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2012%26volume%3D33%26spage%3D970%26epage%3D986%26doi%3D10.1002%2Fjcc.22926" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref66"><div class="reference"><strong class="refLabel"><a href="#ref66" class="rightTabRefNumLink">66</a></strong><div class="NLM_citation" id="rightTab-cit66"><span><span class="NLM_contrib-group">Klimovich, P. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mobley, D. L.</span><span> </span><span class="NLM_article-title">A python tool to set up relative free energy calculations in GROMACS</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x">, </span> <span class="NLM_fpage">1007</span><span class="NLM_x">–</span> <span class="NLM_lpage">1014</span><span class="refDoi"> DOI: 10.1007/s10822-015-9873-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1007%2Fs10822-015-9873-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=26487189" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs1yktL3E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2015&pages=1007-1014&author=P.+V.+Klimovichauthor=D.+L.+Mobley&title=A+python+tool+to+set+up+relative+free+energy+calculations+in+GROMACS&doi=10.1007%2Fs10822-015-9873-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">A Python tool to set up relative free energy calculations in GROMACS</span></div><div class="casAuthors">Klimovich, Pavel V.; Mobley, David L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1007-1014</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Free energy calcns. based on mol. dynamics (MD) simulations have seen a tremendous growth in the last decade.  However, it is still difficult and tedious to set them up in an automated manner, as the majority of the present-day MD simulation packages lack that functionality.  Relative free energy calcns. are a particular challenge for several reasons, including the problem of finding a common substructure and mapping the transformation to be applied.  Here the authors present a tool, alchem.-setup.py, that automatically generates all the input files needed to perform relative solvation and binding free energy calcns. with the MD package GROMACS.  When combined with Lead Optimization Mapper (LOMAP; Liu et al. in J Comput Aided Mol Des 27(9):755-770, 2013), recently developed in the authors' group, alchem.-setup.py allows fully automated setup of relative free energy calcns. in GROMACS.  Taking a graph of the planned calcns. and a mapping, both computed by LOMAP, the authors' tool generates the topol. and coordinate files needed to perform relative free energy calcns. for a given set of mols., and provides a set of simulation input parameters.  The tool was validated by performing relative hydration free energy calcns. for a handful of mols. from the SAMPL4 challenge (Mobley et al. in J Comput Aided Mol Des 28(4):135-150, 2014).  Good agreement with previously published results and the straightforward way in which free energy calcns. can be conducted make alchem.-setup.py a promising tool for automated setup of relative solvation and binding free energy calcns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZlJT4bLywf7Vg90H21EOLACvtfcHk0lhfIZP28_nCwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs1yktL3E&md5=79a8c43be9b37ba611fda7a689951254</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1007%2Fs10822-015-9873-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-015-9873-0%26sid%3Dliteratum%253Aachs%26aulast%3DKlimovich%26aufirst%3DP.%2BV.%26aulast%3DMobley%26aufirst%3DD.%2BL.%26atitle%3DA%2520python%2520tool%2520to%2520set%2520up%2520relative%2520free%2520energy%2520calculations%2520in%2520GROMACS%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2015%26volume%3D29%26spage%3D1007%26epage%3D1014%26doi%3D10.1007%2Fs10822-015-9873-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref67"><div class="reference"><strong class="refLabel"><a href="#ref67" class="rightTabRefNumLink">67</a></strong><div class="NLM_citation" id="rightTab-cit67"><span><span class="NLM_contrib-group">Wang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deng, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pierce, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krilov, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lupyan, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robinson, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dahlgren, M. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greenwood, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Romero, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Masse, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knight, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Steinbrecher, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beuming, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Damm, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harder, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sherman, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brewer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wester, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murcko, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Farid, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mobley, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jorgensen, W. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berne, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friesner, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abel, R.</span><span> </span><span class="NLM_article-title">Accurate and reliable prediction of relative ligand binding potency in prospective drug discovery by way of a modern free-energy calculation protocol and force field</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">137</span><span class="NLM_x">, </span> <span class="NLM_fpage">2695</span><span class="NLM_x">–</span> <span class="NLM_lpage">2703</span><span class="refDoi"> DOI: 10.1021/ja512751q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja512751q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsF2iuro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=137&publication_year=2015&pages=2695-2703&author=L.+Wangauthor=Y.+Wuauthor=Y.+Dengauthor=B.+Kimauthor=L.+Pierceauthor=G.+Krilovauthor=D.+Lupyanauthor=S.+Robinsonauthor=M.+K.+Dahlgrenauthor=J.+Greenwoodauthor=D.+L.+Romeroauthor=C.+Masseauthor=J.+L.+Knightauthor=T.+Steinbrecherauthor=T.+Beumingauthor=W.+Dammauthor=E.+Harderauthor=W.+Shermanauthor=M.+Brewerauthor=R.+Westerauthor=M.+Murckoauthor=L.+Fryeauthor=R.+Faridauthor=T.+Linauthor=D.+L.+Mobleyauthor=W.+L.+Jorgensenauthor=B.+J.+Berneauthor=R.+A.+Friesnerauthor=R.+Abel&title=Accurate+and+reliable+prediction+of+relative+ligand+binding+potency+in+prospective+drug+discovery+by+way+of+a+modern+free-energy+calculation+protocol+and+force+field&doi=10.1021%2Fja512751q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">Accurate and Reliable Prediction of Relative Ligand Binding Potency in Prospective Drug Discovery by Way of a Modern Free-Energy Calculation Protocol and Force Field</span></div><div class="casAuthors">Wang, Lingle; Wu, Yujie; Deng, Yuqing; Kim, Byungchan; Pierce, Levi; Krilov, Goran; Lupyan, Dmitry; Robinson, Shaughnessy; Dahlgren, Markus K.; Greenwood, Jeremy; Romero, Donna L.; Masse, Craig; Knight, Jennifer L.; Steinbrecher, Thomas; Beuming, Thijs; Damm, Wolfgang; Harder, Ed; Sherman, Woody; Brewer, Mark; Wester, Ron; Murcko, Mark; Frye, Leah; Farid, Ramy; Lin, Teng; Mobley, David L.; Jorgensen, William L.; Berne, Bruce J.; Friesner, Richard A.; Abel, Robert</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">137</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2695-2703</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Designing tight-binding ligands is a primary objective of small-mol. drug discovery.  Over the past few decades, free-energy calcns. have benefited from improved force fields and sampling algorithms, as well as the advent of low-cost parallel computing.  However, it has proven to be challenging to reliably achieve the level of accuracy that would be needed to guide lead optimization (∼5× in binding affinity) for a wide range of ligands and protein targets.  Not surprisingly, widespread com. application of free-energy simulations has been limited due to the lack of large-scale validation coupled with the tech. challenges traditionally assocd. with running these types of calcns.  Here, we report an approach that achieves an unprecedented level of accuracy across a broad range of target classes and ligands, with retrospective results encompassing 200 ligands and a wide variety of chem. perturbations, many of which involve significant changes in ligand chem. structures.  In addn., we have applied the method in prospective drug discovery projects and found a significant improvement in the quality of the compds. synthesized that have been predicted to be potent.  Compds. predicted to be potent by this approach have a substantial redn. in false positives relative to compds. synthesized on the basis of other computational or medicinal chem. approaches.  Furthermore, the results are consistent with those obtained from our retrospective studies, demonstrating the robustness and broad range of applicability of this approach, which can be used to drive decisions in lead optimization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGor9UtXEyLPpLVg90H21EOLACvtfcHk0ljT99_h_LSGdw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsF2iuro%253D&md5=37a4f4a6c085f47ed531342643b6c33b</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1021%2Fja512751q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja512751q%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DL.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DDeng%26aufirst%3DY.%26aulast%3DKim%26aufirst%3DB.%26aulast%3DPierce%26aufirst%3DL.%26aulast%3DKrilov%26aufirst%3DG.%26aulast%3DLupyan%26aufirst%3DD.%26aulast%3DRobinson%26aufirst%3DS.%26aulast%3DDahlgren%26aufirst%3DM.%2BK.%26aulast%3DGreenwood%26aufirst%3DJ.%26aulast%3DRomero%26aufirst%3DD.%2BL.%26aulast%3DMasse%26aufirst%3DC.%26aulast%3DKnight%26aufirst%3DJ.%2BL.%26aulast%3DSteinbrecher%26aufirst%3DT.%26aulast%3DBeuming%26aufirst%3DT.%26aulast%3DDamm%26aufirst%3DW.%26aulast%3DHarder%26aufirst%3DE.%26aulast%3DSherman%26aufirst%3DW.%26aulast%3DBrewer%26aufirst%3DM.%26aulast%3DWester%26aufirst%3DR.%26aulast%3DMurcko%26aufirst%3DM.%26aulast%3DFrye%26aufirst%3DL.%26aulast%3DFarid%26aufirst%3DR.%26aulast%3DLin%26aufirst%3DT.%26aulast%3DMobley%26aufirst%3DD.%2BL.%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26aulast%3DBerne%26aufirst%3DB.%2BJ.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DAbel%26aufirst%3DR.%26atitle%3DAccurate%2520and%2520reliable%2520prediction%2520of%2520relative%2520ligand%2520binding%2520potency%2520in%2520prospective%2520drug%2520discovery%2520by%2520way%2520of%2520a%2520modern%2520free-energy%2520calculation%2520protocol%2520and%2520force%2520field%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2015%26volume%3D137%26spage%3D2695%26epage%3D2703%26doi%3D10.1021%2Fja512751q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref68"><div class="reference"><strong class="refLabel"><a href="#ref68" class="rightTabRefNumLink">68</a></strong><div class="NLM_citation" id="rightTab-cit68"><span><span class="NLM_contrib-group">Rocklin, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mobley, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dill, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hunenberger, P. H.</span><span> </span><span class="NLM_article-title">Calculating the binding free energies of charged species based on explicit-solvent simulations employing lattice-sum methods: An accurate correction scheme for electrostatic finite-size effects</span> <span class="citation_source-journal">J. Chem. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">139</span><span class="NLM_x">, </span> <span class="NLM_fpage">184103</span><span class="refDoi"> DOI: 10.1063/1.4826261</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1063%2F1.4826261" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=24320250" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslGjsLbP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=139&publication_year=2013&pages=184103&author=G.+J.+Rocklinauthor=D.+L.+Mobleyauthor=K.+A.+Dillauthor=P.+H.+Hunenberger&title=Calculating+the+binding+free+energies+of+charged+species+based+on+explicit-solvent+simulations+employing+lattice-sum+methods%3A+An+accurate+correction+scheme+for+electrostatic+finite-size+effects&doi=10.1063%2F1.4826261"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">Calculating the binding free energies of charged species based on explicit-solvent simulations employing lattice-sum methods: An accurate correction scheme for electrostatic finite-size effects</span></div><div class="casAuthors">Rocklin, Gabriel J.; Mobley, David L.; Dill, Ken A.; Huenenberger, Philippe H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">139</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">184103/1-184103/32</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    
            (<span class="NLM_cas:orgname">American Institute of Physics</span>)
        </div><div class="casAbstract">The calcn. of a protein-ligand binding free energy based on mol. dynamics (MD) simulations generally relies on a thermodn. cycle in which the ligand is alchem. inserted into the system, both in the solvated protein and free in soln.  The corresponding ligand-insertion free energies are typically calcd. in nanoscale computational boxes simulated under periodic boundary conditions and considering electrostatic interactions defined by a periodic lattice-sum.  This is distinct from the ideal bulk situation of a system of macroscopic size simulated under non-periodic boundary conditions with Coulombic electrostatic interactions.  This discrepancy results in finite-size effects, which affect primarily the charging component of the insertion free energy, are dependent on the box size, and can be large when the ligand bears a net charge, esp. if the protein is charged as well.  This article studies finite-size effects on calcd. charging free energies using as a test case: the binding of the ligand 2-amino-5-methylthiazole (net charge +1 e) to a mutant form of yeast cytochrome c peroxidase in water.  Considering different charge isoforms of the protein (net charges -5, 0, +3, or +9 e), either in the absence or the presence of neutralizing counterions, and sizes of the cubic computational box (edges ranging from 7.42 to 11.02 nm), the potentially large magnitude of finite-size effects on the raw charging free energies (up to 17.1 kJ mol-1) is demonstrated.  Two correction schemes are then proposed to eliminate these effects, a numerical and an anal. one.  Both schemes are based on a continuum-electrostatics anal. and require performing Poisson-Boltzmann (PB) calcns. on the protein-ligand system.  While the numerical scheme requires PB calcns. under both non-periodic and periodic boundary conditions, the latter at the box size considered in the MD simulations, the anal. scheme only requires three non-periodic PB calcns. for a given system, its dependence on the box size being anal.  The latter scheme also provides insight into the phys. origin of the finite-size effects.  These two schemes also encompass a correction for discrete solvent effects that persists even in the limit of infinite box sizes.  Application of either scheme essentially eliminates the size dependence of the cor. charging free energies (maximal deviation of 1.5 kJ mol-1).  Because it is simple to apply, the anal. correction scheme offers a general soln. to the problem of finite-size effects in free-energy calcns. involving charged solutes, as encountered in calcns. concerning, e.g., protein-ligand binding, biomol. assocn., residue mutation, pKa and redox potential estn., substrate transformation, solvation, and solvent-solvent partitioning.  (c) 2013 American Institute of Physics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoIZLSmKR_DYrVg90H21EOLACvtfcHk0ljT99_h_LSGdw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslGjsLbP&md5=4114ffb1d127091ee7421d146911b381</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1063%2F1.4826261&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.4826261%26sid%3Dliteratum%253Aachs%26aulast%3DRocklin%26aufirst%3DG.%2BJ.%26aulast%3DMobley%26aufirst%3DD.%2BL.%26aulast%3DDill%26aufirst%3DK.%2BA.%26aulast%3DHunenberger%26aufirst%3DP.%2BH.%26atitle%3DCalculating%2520the%2520binding%2520free%2520energies%2520of%2520charged%2520species%2520based%2520on%2520explicit-solvent%2520simulations%2520employing%2520lattice-sum%2520methods%253A%2520An%2520accurate%2520correction%2520scheme%2520for%2520electrostatic%2520finite-size%2520effects%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D2013%26volume%3D139%26spage%3D184103%26doi%3D10.1063%2F1.4826261" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref69"><div class="reference"><strong class="refLabel"><a href="#ref69" class="rightTabRefNumLink">69</a></strong><div class="NLM_citation" id="rightTab-cit69"><span><span class="NLM_contrib-group">Reif, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oostenbrink, C.</span><span> </span><span class="NLM_article-title">Net charge changes in the calculation of relative ligand-binding free energies via classical atomistic molecular dynamics simulation</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">35</span><span class="NLM_x">, </span> <span class="NLM_fpage">227</span><span class="NLM_x">–</span> <span class="NLM_lpage">243</span><span class="refDoi"> DOI: 10.1002/jcc.23490</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1002%2Fjcc.23490" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=24249099" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvVSgsLjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2014&pages=227-243&author=M.+M.+Reifauthor=C.+Oostenbrink&title=Net+charge+changes+in+the+calculation+of+relative+ligand-binding+free+energies+via+classical+atomistic+molecular+dynamics+simulation&doi=10.1002%2Fjcc.23490"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">Net charge changes in the calculation of relative ligand-binding free energies via classical atomistic molecular dynamics simulation</span></div><div class="casAuthors">Reif, Maria M.; Oostenbrink, Chris</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">227-243</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">The calcn. of binding free energies of charged species to a target mol. is a frequently encountered problem in mol. dynamics studies of (bio-)chem. thermodn.  Many important endogenous receptor-binding mols., enzyme substrates, or drug mols. have a nonzero net charge.  Abs. binding free energies, as well as binding free energies relative to another mol. with a different net charge will be affected by artifacts due to the used effective electrostatic interaction function and assocd. parameters (e.g., size of the computational box).  In the present study, charging contributions to binding free energies of small oligoat. ions to a series of model host cavities functionalized with different chem. groups are calcd. with classical atomistic mol. dynamics simulation.  Electrostatic interactions are treated using a lattice-summation scheme or a cutoff-truncation scheme with Barker-Watts reaction-field correction, and the simulations are conducted in boxes of different edge lengths.  It is illustrated that the charging free energies of the guest mols. in water and in the host strongly depend on the applied methodol. and that neglect of correction terms for the artifacts introduced by the finite size of the simulated system and the use of an effective electrostatic interaction function considerably impairs the thermodn. interpretation of guest-host interactions.  Application of correction terms for the various artifacts yields consistent results for the charging contribution to binding free energies and is thus a prerequisite for the valid interpretation or prediction of exptl. data via mol. dynamics simulation.  Anal. and correction of electrostatic artifacts according to the scheme proposed in the present study should therefore be considered an integral part of careful free-energy calcn. studies if changes in the net charge are involved.  © 2013 Wiley Periodicals, Inc.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1lirsSd7shLVg90H21EOLACvtfcHk0ljT99_h_LSGdw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvVSgsLjO&md5=fc7ffb060ff30bcda792ffc68f832906</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1002%2Fjcc.23490&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.23490%26sid%3Dliteratum%253Aachs%26aulast%3DReif%26aufirst%3DM.%2BM.%26aulast%3DOostenbrink%26aufirst%3DC.%26atitle%3DNet%2520charge%2520changes%2520in%2520the%2520calculation%2520of%2520relative%2520ligand-binding%2520free%2520energies%2520via%2520classical%2520atomistic%2520molecular%2520dynamics%2520simulation%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2014%26volume%3D35%26spage%3D227%26epage%3D243%26doi%3D10.1002%2Fjcc.23490" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref70"><div class="reference"><strong class="refLabel"><a href="#ref70" class="rightTabRefNumLink">70</a></strong><div class="NLM_citation" id="rightTab-cit70"><span><span class="NLM_contrib-group">Lazaridis, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Masunov, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gandolfo, F.</span><span> </span><span class="NLM_article-title">Contributions to the binding free energy of ligands to avidin and streptavidin</span> <span class="citation_source-journal">Proteins: Struct., Funct., Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">194</span><span class="NLM_x">–</span> <span class="NLM_lpage">208</span><span class="refDoi"> DOI: 10.1002/prot.10086</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1002%2Fprot.10086" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=11933066" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BD38XislGhurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2002&pages=194-208&author=T.+Lazaridisauthor=A.+Masunovauthor=F.+Gandolfo&title=Contributions+to+the+binding+free+energy+of+ligands+to+avidin+and+streptavidin&doi=10.1002%2Fprot.10086"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">Contributions to the binding free energy of ligands to avidin and streptavidin</span></div><div class="casAuthors">Lazaridis, Themis; Masunov, Artem; Gandolfo, Francois</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Genetics</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">194-208</span>CODEN:
                <span class="NLM_cas:coden">PSFGEY</span>;
        ISSN:<span class="NLM_cas:issn">0887-3585</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">The free energy of binding of a ligand to a macromol. is here formally decompd. into the (effective) energy of interaction, reorganization energy of the ligand and the macro-mol., conformational entropy change of the ligand and the macromol., and translational and rotational entropy loss of the ligand.  Mol. dynamics simulations with implicit solvation are used to evaluate these contributions in the binding of biotin, biotin analogs, and two peptides to avidin and streptavidin.  We find that the largest contribution opposing binding is the protein reorganization energy, which is calcd. to be from 10 to 30 kcal/mol for the ligands considered here.  The ligand reorganization energy is also significant for flexible ligands.  The translational/rotational entropy is 4.5-6 kcal/mol at 1 M std. state and room temp.  The calcd. binding free energies are in the correct range, but the large statistical uncertainty in the protein reorganization energy precludes precise predictions.  For some complexes, the simulations show multiple binding modes, different from the one obsd. in the crystal structure.  This finding is probably due to deficiencies in the force field but may also reflect considerable ligand flexibility.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrL-1EkQp_DCbVg90H21EOLACvtfcHk0ljyBSol5-Wb7A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XislGhurk%253D&md5=df50261e0738b8d62173157ad321fe3c</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1002%2Fprot.10086&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.10086%26sid%3Dliteratum%253Aachs%26aulast%3DLazaridis%26aufirst%3DT.%26aulast%3DMasunov%26aufirst%3DA.%26aulast%3DGandolfo%26aufirst%3DF.%26atitle%3DContributions%2520to%2520the%2520binding%2520free%2520energy%2520of%2520ligands%2520to%2520avidin%2520and%2520streptavidin%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D2002%26volume%3D47%26spage%3D194%26epage%3D208%26doi%3D10.1002%2Fprot.10086" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref71"><div class="reference"><strong class="refLabel"><a href="#ref71" class="rightTabRefNumLink">71</a></strong><div class="NLM_citation" id="rightTab-cit71"><span><span class="NLM_contrib-group">Oostenbrink, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Gunsteren, W. F.</span><span> </span><span class="NLM_article-title">Free energies of binding of polychlorinated biphenyls to the estrogen receptor from a single simulation</span> <span class="citation_source-journal">Proteins: Struct., Funct., Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">237</span><span class="NLM_x">–</span> <span class="NLM_lpage">246</span><span class="refDoi"> DOI: 10.1002/prot.10558</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1002%2Fprot.10558" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=14696186" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BD2cXlsFWjsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2004&pages=237-246&author=C.+Oostenbrinkauthor=W.+F.+van+Gunsteren&title=Free+energies+of+binding+of+polychlorinated+biphenyls+to+the+estrogen+receptor+from+a+single+simulation&doi=10.1002%2Fprot.10558"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">Free energies of binding of polychlorinated biphenyls to the estrogen receptor from a single simulation</span></div><div class="casAuthors">Oostenbrink, Chris; van Gunsteren, Wilfred F.</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Bioinformatics</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">237-246</span>CODEN:
                <span class="NLM_cas:coden">PSFBAF</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">Relative free energies of binding to the ligand-binding domain of the estrogen receptor have been calcd. for a series of 17 hydroxylated polychlorinated biphenyls.  Because traditional thermodn. integration or perturbation approaches are hardly feasible for these nos. of compds., the one-step perturbation approach is applied and is shown to yield accurate results based on only two 2-ns mol. dynamics simulations of an unphys., judiciously chosen, ref. state.  The mean abs. difference between the calcd. and exptl. binding free energies for the 17 compds. is 3.4 kJ/mol, which illustrates the accuracy of the GROMOS biomol. force field used.  Excluding the three largest ligands from the comparison reduces the deviation to 2.0 kJ/mol (i.e., < kBT).  Apart from the relative free energy, structural information about the binding mode and binding orientation for every compd. can also be extd. from the simulation, showing that a ligand bound to its receptor cannot be represented by a single conformation, but it samples an ensemble of different orientations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqdRCeKfjW367Vg90H21EOLACvtfcHk0ljyBSol5-Wb7A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXlsFWjsg%253D%253D&md5=6dd0c013b944251c11bbd90d567c8f8e</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1002%2Fprot.10558&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.10558%26sid%3Dliteratum%253Aachs%26aulast%3DOostenbrink%26aufirst%3DC.%26aulast%3Dvan%2BGunsteren%26aufirst%3DW.%2BF.%26atitle%3DFree%2520energies%2520of%2520binding%2520of%2520polychlorinated%2520biphenyls%2520to%2520the%2520estrogen%2520receptor%2520from%2520a%2520single%2520simulation%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D2004%26volume%3D54%26spage%3D237%26epage%3D246%26doi%3D10.1002%2Fprot.10558" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref72"><div class="reference"><strong class="refLabel"><a href="#ref72" class="rightTabRefNumLink">72</a></strong><div class="NLM_citation" id="rightTab-cit72"><span><span class="NLM_contrib-group">Steinbrecher, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Case, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Labahn, A.</span><span> </span><span class="NLM_article-title">A multistep approach to structure-based drug design: studying ligand binding at the human neutrophil elastase</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">1837</span><span class="NLM_x">–</span> <span class="NLM_lpage">1844</span><span class="refDoi"> DOI: 10.1021/jm0505720</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0505720" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BD28XhsVSqtbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=1837-1844&author=T.+Steinbrecherauthor=D.+A.+Caseauthor=A.+Labahn&title=A+multistep+approach+to+structure-based+drug+design%3A+studying+ligand+binding+at+the+human+neutrophil+elastase&doi=10.1021%2Fjm0505720"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">A Multistep Approach to Structure-Based Drug Design: Studying Ligand Binding at the Human Neutrophil Elastase</span></div><div class="casAuthors">Steinbrecher, Thomas; Case, David A.; Labahn, Andreas</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1837-1844</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In this study we show that a combination of different theor. methods is a viable approach to calc. the binding affinities of new ligands for the human neutrophile elastase.  This protease degrades elastin and likely aids neutrophils in fulfilling their immunol. functions.  Abnormally high human neutrophil elastase (HNE) levels are involved in several diseases; therefore, inhibitors of HNE are of interest as targets for drug design.  A recent study has revealed that cinnamic acid and bornyl ester derivs. bind to HNE, but ΔG0 values from ligand docking results exhibited no correlation with those calcd. from the IC50 values.  To accurately compute binding affinities, we generated possible protein ligand complex structures by ligand docking calcns.  For each of the ligands, the 30 most likely placements were used as starting points of nanosecond length mol. dynamics simulations.  The binding free energies for these complex structures were estd. using a continuum solvent (MM-PBSA) approach.  These results, along with structural data from the mol. dynamics runs, allowed the identification of a group of similar placements that serve as a model for the natural protein ligand complex structure.  This structural model was used to perform thermodn. integration (TI) calcns. to obtain the relative binding free energies of similar ligands to HNE.  The TI results were in quant. agreement with the measured binding affinities.  Thus, the presented approach can be used to generate a probable complex structure for known ligands to HNE and to use such a structure to calc. the effects of small ligand modifications on ligand binding, possibly leading to new inhibitors with improved binding affinities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpReq57qu60nLVg90H21EOLACvtfcHk0ljyBSol5-Wb7A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhsVSqtbc%253D&md5=af09a49b14c1f84df850f5e68cfac3bc</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1021%2Fjm0505720&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0505720%26sid%3Dliteratum%253Aachs%26aulast%3DSteinbrecher%26aufirst%3DT.%26aulast%3DCase%26aufirst%3DD.%2BA.%26aulast%3DLabahn%26aufirst%3DA.%26atitle%3DA%2520multistep%2520approach%2520to%2520structure-based%2520drug%2520design%253A%2520studying%2520ligand%2520binding%2520at%2520the%2520human%2520neutrophil%2520elastase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D1837%26epage%3D1844%26doi%3D10.1021%2Fjm0505720" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref73"><div class="reference"><strong class="refLabel"><a href="#ref73" class="rightTabRefNumLink">73</a></strong><div class="NLM_citation" id="rightTab-cit73"><span><span class="NLM_contrib-group">Mobley, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chodera, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dill, K. A.</span><span> </span><span class="NLM_article-title">The confine-and-release method: obtaining correct binding free energies in the presence of protein conformational change</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">1231</span><span class="NLM_x">–</span> <span class="NLM_lpage">1235</span><span class="refDoi"> DOI: 10.1021/ct700032n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct700032n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BD2sXlsFenu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2007&pages=1231-1235&author=D.+L.+Mobleyauthor=J.+D.+Choderaauthor=K.+A.+Dill&title=The+confine-and-release+method%3A+obtaining+correct+binding+free+energies+in+the+presence+of+protein+conformational+change&doi=10.1021%2Fct700032n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">Confine-and-Release Method: Obtaining Correct Binding Free Energies in the Presence of Protein Conformational Change</span></div><div class="casAuthors">Mobley, David L.; Chodera, John D.; Dill, Ken A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1231-1235</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Free energy calcns. are increasingly being used to est. abs. and relative binding free energies of ligands to proteins.  However, computed free energies often appear to depend on the initial protein conformation, indicating incomplete sampling.  This is esp. true when proteins can change conformation on ligand binding, as free energies assocd. with these conformational changes are either ignored or assumed to be included by virtue of the sampling performed in the calcn.  Here, the authors show that, in a model protein system (a designed binding site in T4 lysozyme), conformational changes can make a difference of several kcal/mol in computed binding free energies and that they are neglected in computed binding free energies if the system remains kinetically trapped in a particular metastable state on simulation timescales.  The authors introduce a general "confine-and-release" framework for free energy calcns. that accounts for these free energies of conformational change.  The authors illustrate its use in this model system by demonstrating that an umbrella sampling protocol can obtain converged binding free energies that are independent of the starting protein structure and include these conformational change free energies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHdRFLUwdd7LVg90H21EOLACvtfcHk0lgR6Pcl4Thsyw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXlsFenu7g%253D&md5=fa953a85593d81ee192ff6c5d7f525c2</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1021%2Fct700032n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct700032n%26sid%3Dliteratum%253Aachs%26aulast%3DMobley%26aufirst%3DD.%2BL.%26aulast%3DChodera%26aufirst%3DJ.%2BD.%26aulast%3DDill%26aufirst%3DK.%2BA.%26atitle%3DThe%2520confine-and-release%2520method%253A%2520obtaining%2520correct%2520binding%2520free%2520energies%2520in%2520the%2520presence%2520of%2520protein%2520conformational%2520change%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2007%26volume%3D3%26spage%3D1231%26epage%3D1235%26doi%3D10.1021%2Fct700032n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref74"><div class="reference"><strong class="refLabel"><a href="#ref74" class="rightTabRefNumLink">74</a></strong><div class="NLM_citation" id="rightTab-cit74"><span><span class="NLM_contrib-group">Montfort, W. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perry, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fauman, E. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Finer-Moore, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maley, G. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hardy, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maley, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stroud, R. M.</span><span> </span><span class="NLM_article-title">Structure, multiple site binding, and segmental accommodation in thymidylate synthase on binding dUMP and an anti-folate</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1990</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x">, </span> <span class="NLM_fpage">6964</span><span class="NLM_x">–</span> <span class="NLM_lpage">6977</span><span class="refDoi"> DOI: 10.1021/bi00482a004</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi00482a004" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADyaK3cXksFCqtbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=1990&pages=6964-6977&author=W.+R.+Montfortauthor=K.+M.+Perryauthor=E.+B.+Faumanauthor=J.+S.+Finer-Mooreauthor=G.+F.+Maleyauthor=L.+Hardyauthor=F.+Maleyauthor=R.+M.+Stroud&title=Structure%2C+multiple+site+binding%2C+and+segmental+accommodation+in+thymidylate+synthase+on+binding+dUMP+and+an+anti-folate&doi=10.1021%2Fbi00482a004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit74R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">Structure, multiple site binding, and segmental accommodation in thymidylate synthase on binding dUMP and an anti-folate</span></div><div class="casAuthors">Montfort, William R.; Perry, Kathy M.; Fauman, Eric B.; Finer-Moore, Janet S.; Maley, Gladys F.; Hardy, Larry; Maley, Frank; Stroud, Robert M.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">1990</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">30</span>),
    <span class="NLM_cas:pages">6964-77</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    </div><div class="casAbstract">The structure of Escherichia coli thymidylate synthase (TS) complexed with the substrate, dUMP, and CB 3717, an analog of the cofactor, methylenetetrahydrofolate, was solved by multiple isomorphous replacement and refined at 1.97-Å resoln. to a residual of 18% for all data (16% for data >2σ) for a highly constrained structure.  All residues in the structure were clearly resolved and gave a very high confidence in total correctness of the structure.  The ternary complex directly suggested how methylation of dUMP takes place.  C-6 of dUMP was covalently bound to γS of cysteine-198(146) during catalysis, and the reactants were surrounded by specific H-bonds and hydrophobic interactions from conserved residues.  Comparison with the independently solved structure of unliganded TS revealed a large conformation change in the enzyme, which closes down to sequester the reactants and several highly ordered water mols. within a cavernous active center, away from bulk solvent.  A 2nd binding site for the quinazoline ring of the cofactor analog was discovered by withholding addn. of reducing agent during crystal storage.  The chem. change in the protein was slight, and from difference d. maps modification of SH groups was not directly responsible for blockade of the primary site.  The site, only partially overlapping with the primary site, was also surrounded by conserved residues and thus may play a functional role.  The ligand-induced conformational change was not a domain shift but involved the segmental accommodation of several helixes, β-strands, and loops that moved as units against the β-sheet interface between monomers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoaidrVNYJuorVg90H21EOLACvtfcHk0lgR6Pcl4Thsyw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3cXksFCqtbY%253D&md5=a4a99e1360917b52f32615aa29b862b8</span></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1021%2Fbi00482a004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi00482a004%26sid%3Dliteratum%253Aachs%26aulast%3DMontfort%26aufirst%3DW.%2BR.%26aulast%3DPerry%26aufirst%3DK.%2BM.%26aulast%3DFauman%26aufirst%3DE.%2BB.%26aulast%3DFiner-Moore%26aufirst%3DJ.%2BS.%26aulast%3DMaley%26aufirst%3DG.%2BF.%26aulast%3DHardy%26aufirst%3DL.%26aulast%3DMaley%26aufirst%3DF.%26aulast%3DStroud%26aufirst%3DR.%2BM.%26atitle%3DStructure%252C%2520multiple%2520site%2520binding%252C%2520and%2520segmental%2520accommodation%2520in%2520thymidylate%2520synthase%2520on%2520binding%2520dUMP%2520and%2520an%2520anti-folate%26jtitle%3DBiochemistry%26date%3D1990%26volume%3D29%26spage%3D6964%26epage%3D6977%26doi%3D10.1021%2Fbi00482a004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref75"><div class="reference"><strong class="refLabel"><a href="#ref75" class="rightTabRefNumLink">75</a></strong><div class="NLM_citation" id="rightTab-cit75"><span><span class="NLM_contrib-group">Mobley, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dill, K. A.</span><span> </span><span class="NLM_article-title">Binding of small-molecule ligands to proteins: “what you see” is not always “what you get”</span> <span class="citation_source-journal">Structure</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">489</span><span class="NLM_x">–</span> <span class="NLM_lpage">498</span><span class="refDoi"> DOI: 10.1016/j.str.2009.02.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1016%2Fj.str.2009.02.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=19368882" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BD1MXksVCmsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2009&pages=489-498&author=D.+L.+Mobleyauthor=K.+A.+Dill&title=Binding+of+small-molecule+ligands+to+proteins%3A+%E2%80%9Cwhat+you+see%E2%80%9D+is+not+always+%E2%80%9Cwhat+you+get%E2%80%9D&doi=10.1016%2Fj.str.2009.02.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit75R"><div class="casContent"><span class="casTitleNuber">75</span><div class="casTitle"><span class="NLM_cas:atitle">Binding of Small-Molecule Ligands to Proteins: "What You See" Is Not Always "What You Get"</span></div><div class="casAuthors">Mobley, David L.; Dill, Ken A.</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">489-498</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  We review insights from computational studies of affinities of ligands binding to proteins.  The power of structural biol. is in translating knowledge of protein structures into insights about their forces, binding, and mechanisms.  However, the complementary power of computer modeling is in showing "the rest of the story" (i.e., how motions and ensembles and alternative conformers and the entropies and forces that cannot be seen in single mol. structures also contribute to binding affinities).  Upon binding to a protein, a ligand can bind in multiple orientations; the protein or ligand can be deformed by the binding event; waters, ions, or cofactors can have unexpected involvement; and conformational or solvation entropies can sometimes play large and otherwise unpredictable roles.  Computer modeling is helping to elucidate these factors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnlrwNwfLm4rVg90H21EOLACvtfcHk0lgR6Pcl4Thsyw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXksVCmsrk%253D&md5=993623b1444e33a45b2fe8b904dec9c1</span></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2009.02.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2009.02.010%26sid%3Dliteratum%253Aachs%26aulast%3DMobley%26aufirst%3DD.%2BL.%26aulast%3DDill%26aufirst%3DK.%2BA.%26atitle%3DBinding%2520of%2520small-molecule%2520ligands%2520to%2520proteins%253A%2520%25E2%2580%259Cwhat%2520you%2520see%25E2%2580%259D%2520is%2520not%2520always%2520%25E2%2580%259Cwhat%2520you%2520get%25E2%2580%259D%26jtitle%3DStructure%26date%3D2009%26volume%3D17%26spage%3D489%26epage%3D498%26doi%3D10.1016%2Fj.str.2009.02.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref76"><div class="reference"><strong class="refLabel"><a href="#ref76" class="rightTabRefNumLink">76</a></strong><div class="NLM_citation" id="rightTab-cit76"><span><span class="NLM_contrib-group">Wang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deng, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knight, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sherman, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shelley, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abel, R.</span><span> </span><span class="NLM_article-title">Modeling local structural rearrangements using FEP/REST: application to relative binding affinity predictions of CDK2 inhibitors</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">1282</span><span class="NLM_x">–</span> <span class="NLM_lpage">1293</span><span class="refDoi"> DOI: 10.1021/ct300911a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct300911a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC3sXoslOjuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=1282-1293&author=L.+Wangauthor=Y.+Dengauthor=J.+L.+Knightauthor=Y.+Wuauthor=B.+Kimauthor=W.+Shermanauthor=J.+C.+Shelleyauthor=T.+Linauthor=R.+Abel&title=Modeling+local+structural+rearrangements+using+FEP%2FREST%3A+application+to+relative+binding+affinity+predictions+of+CDK2+inhibitors&doi=10.1021%2Fct300911a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit76R"><div class="casContent"><span class="casTitleNuber">76</span><div class="casTitle"><span class="NLM_cas:atitle">Modeling Local Structural Rearrangements Using FEP/REST: Application to Relative Binding Affinity Predictions of CDK2 Inhibitors</span></div><div class="casAuthors">Wang, Lingle; Deng, Yuqing; Knight, Jennifer L.; Wu, Yujie; Kim, Byungchan; Sherman, Woody; Shelley, John C.; Lin, Teng; Abel, Robert</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">1282-1293</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Accurate and reliable calcn. of protein-ligand binding affinities remains a hotbed of computer-aided drug design research.  Despite the potentially large impact FEP (free energy perturbation) may have in drug design projects, practical applications of FEP in industrial contexts have been limited.  In this work, we use a recently developed method, FEP/REST (free energy perturbation/replica exchange with solute tempering), to calc. the relative binding affinities for a set of congeneric ligands binding to the CDK2 receptor.  We compare the FEP/REST results with traditional FEP/MD (mol. dynamics) results and MM/GBSA (mol. mechanics/Generalized Born Surface Area model) results and examine why FEP/REST performed notably better than these other methods, as well as why certain ligand mutations lead to large increases of the binding affinity while others do not.  We also introduce a math. framework for assessing the consistency and reliability of the calcns. using cycle closures in FEP mutation paths.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrorAEUJEbEYbVg90H21EOLACvtfcHk0liYXAIohgOQbg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXoslOjuw%253D%253D&md5=cec7d69a0010904a55fcda5a277e8a1a</span></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.1021%2Fct300911a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct300911a%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DL.%26aulast%3DDeng%26aufirst%3DY.%26aulast%3DKnight%26aufirst%3DJ.%2BL.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DKim%26aufirst%3DB.%26aulast%3DSherman%26aufirst%3DW.%26aulast%3DShelley%26aufirst%3DJ.%2BC.%26aulast%3DLin%26aufirst%3DT.%26aulast%3DAbel%26aufirst%3DR.%26atitle%3DModeling%2520local%2520structural%2520rearrangements%2520using%2520FEP%252FREST%253A%2520application%2520to%2520relative%2520binding%2520affinity%2520predictions%2520of%2520CDK2%2520inhibitors%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2013%26volume%3D9%26spage%3D1282%26epage%3D1293%26doi%3D10.1021%2Fct300911a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref77"><div class="reference"><strong class="refLabel"><a href="#ref77" class="rightTabRefNumLink">77</a></strong><div class="NLM_citation" id="rightTab-cit77"><span><span class="NLM_contrib-group">Dziedzic, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cisneros, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robertson, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hare, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Danford, N. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baxter, R. H. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jorgensen, W. L.</span><span> </span><span class="NLM_article-title">Design, synthesis, and protein crystallography of biaryltriazoles as potent tautomerase inhibitors of macrophage migration inhibitory factor</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">137</span><span class="NLM_x">, </span> <span class="NLM_fpage">2996</span><span class="NLM_x">–</span> <span class="NLM_lpage">3003</span><span class="refDoi"> DOI: 10.1021/ja512112j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja512112j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjtV2ntLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=137&publication_year=2015&pages=2996-3003&author=P.+Dziedzicauthor=J.+A.+Cisnerosauthor=M.+J.+Robertsonauthor=A.+A.+Hareauthor=N.+E.+Danfordauthor=R.+H.+G.+Baxterauthor=W.+L.+Jorgensen&title=Design%2C+synthesis%2C+and+protein+crystallography+of+biaryltriazoles+as+potent+tautomerase+inhibitors+of+macrophage+migration+inhibitory+factor&doi=10.1021%2Fja512112j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit77R"><div class="casContent"><span class="casTitleNuber">77</span><div class="casTitle"><span class="NLM_cas:atitle">Design, synthesis, and protein crystallography of biaryltriazoles as potent tautomerase inhibitors of macrophage migration inhibitory factor</span></div><div class="casAuthors">Dziedzic, Pawel; Cisneros, Jose A.; Robertson, Michael J.; Hare, Alissa A.; Danford, Nadia E.; Baxter, Richard H. G.; Jorgensen, William L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">137</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2996-3003</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Optimization is reported for biaryltriazoles as inhibitors of the tautomerase activity of human macrophage migration inhibitory factor (MIF), a proinflammatory cytokine assocd. with numerous inflammatory diseases and cancer.  A combined approach was taken featuring org. synthesis, enzymic assaying, crystallog., and modeling including free-energy perturbation (FEP) calcns.  X-ray crystal structures for two inhibitors bound to MIF are reported and provided a basis for the modeling efforts.  The accommodation of the inhibitors in the binding site is striking with multiple hydrogen bonds and aryl-aryl interactions.  Addnl. modeling encouraged pursuit of 5-phenoxyquinolinyl analogs, which led to the very potent compd.  Activity was further enhanced by addn. of a fluorine atom adjacent to the phenolic hydroxyl group to strengthen a key hydrogen bond.  It is also shown that phys. properties of the compds. can be modulated by variation of solvent-exposed substituents.  Several of the compds. are likely the most potent known MIF tautomerase inhibitors; the most active ones are more than 1000-fold more active than the well-studied (R)-ISO-1 and more than 200-fold more active than the chromen-4-one Orita-13.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqYFcj_0uhyz7Vg90H21EOLACvtfcHk0liYXAIohgOQbg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjtV2ntLo%253D&md5=2b5fea70b6db3cb39f7217c39995bd06</span></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=10.1021%2Fja512112j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja512112j%26sid%3Dliteratum%253Aachs%26aulast%3DDziedzic%26aufirst%3DP.%26aulast%3DCisneros%26aufirst%3DJ.%2BA.%26aulast%3DRobertson%26aufirst%3DM.%2BJ.%26aulast%3DHare%26aufirst%3DA.%2BA.%26aulast%3DDanford%26aufirst%3DN.%2BE.%26aulast%3DBaxter%26aufirst%3DR.%2BH.%2BG.%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26atitle%3DDesign%252C%2520synthesis%252C%2520and%2520protein%2520crystallography%2520of%2520biaryltriazoles%2520as%2520potent%2520tautomerase%2520inhibitors%2520of%2520macrophage%2520migration%2520inhibitory%2520factor%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2015%26volume%3D137%26spage%3D2996%26epage%3D3003%26doi%3D10.1021%2Fja512112j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref78"><div class="reference"><strong class="refLabel"><a href="#ref78" class="rightTabRefNumLink">78</a></strong><div class="NLM_citation" id="rightTab-cit78"><span><span class="NLM_contrib-group">Mutyala, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reddy, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sumakanth, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reddanna, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reddy, M. R.</span><span> </span><span class="NLM_article-title">Calculation of relative binding affinities of fructose 1, 6-bisphosphatase mutants with adenosine monophosphate using free energy perturbation method</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">932</span><span class="NLM_x">–</span> <span class="NLM_lpage">937</span><span class="refDoi"> DOI: 10.1002/jcc.20617</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1002%2Fjcc.20617" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2007&pages=932-937&author=R.+Mutyalaauthor=R.+Reddyauthor=M.+Sumakanthauthor=P.+Reddannaauthor=M.+R.+Reddy&title=Calculation+of+relative+binding+affinities+of+fructose+1%2C+6-bisphosphatase+mutants+with+adenosine+monophosphate+using+free+energy+perturbation+method&doi=10.1002%2Fjcc.20617"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit78&amp;dbid=16384&amp;doi=10.1002%2Fjcc.20617&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.20617%26sid%3Dliteratum%253Aachs%26aulast%3DMutyala%26aufirst%3DR.%26aulast%3DReddy%26aufirst%3DR.%26aulast%3DSumakanth%26aufirst%3DM.%26aulast%3DReddanna%26aufirst%3DP.%26aulast%3DReddy%26aufirst%3DM.%2BR.%26atitle%3DCalculation%2520of%2520relative%2520binding%2520affinities%2520of%2520fructose%25201%252C%25206-bisphosphatase%2520mutants%2520with%2520adenosine%2520monophosphate%2520using%2520free%2520energy%2520perturbation%2520method%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2007%26volume%3D28%26spage%3D932%26epage%3D937%26doi%3D10.1002%2Fjcc.20617" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref79"><div class="reference"><strong class="refLabel"><a href="#ref79" class="rightTabRefNumLink">79</a></strong><div class="NLM_citation" id="rightTab-cit79"><span><span class="NLM_contrib-group">Vergara-Jaque, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poblete, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, E. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schulten, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gonzalez-Nilo, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chipot, C.</span><span> </span><span class="NLM_article-title">Molecular basis of drug resistance in A/H1N1 virus</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">2650</span><span class="NLM_x">–</span> <span class="NLM_lpage">2656</span><span class="refDoi"> DOI: 10.1021/ci300343w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci300343w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtlGjsbjL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=2650-2656&author=A.+Vergara-Jaqueauthor=H.+Pobleteauthor=E.+H.+Leeauthor=K.+Schultenauthor=F.+Gonzalez-Niloauthor=C.+Chipot&title=Molecular+basis+of+drug+resistance+in+A%2FH1N1+virus&doi=10.1021%2Fci300343w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit79R"><div class="casContent"><span class="casTitleNuber">79</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Basis of Drug Resistance in A/H1N1 Virus</span></div><div class="casAuthors">Vergara-Jaque, Ariela; Poblete, Horacio; Lee, Eric H.; Schulten, Klaus; Gonzalez-Nilo, Fernando; Chipot, Christophe</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2650-2656</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">New mutants of human influenza virus (A/H1N1) exhibit resistance to antiviral drugs.  The mechanism whereby they develop insensitivity to these medications is, however, not yet completely understood.  A crystallog. structure of A/H1N1 neuraminidase has been published recently.  Using mol. dynamic simulations, it is now possible to characterize at the at. level the mechanism that underlies the loss of binding affinity of the drugs.  In this study, free-energy perturbation was used to evaluate the relative binding free energies of Tamiflu and Relenza with H274Y, N294S, and Y252H neuraminidase mutants.  Our results demonstrate a remarkable correlation between theor. and exptl. data, which quant. confirms that the mutants are resistant to Tamiflu but are still strongly inhibited by Relenza.  The simulations further reveal the key interactions that govern the affinity of the two drugs for each mutant.  This information is envisioned to prove useful for the design of novel neuraminidase inhibitors and for the characterization of new potential mutants.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGraFM3nu-VENLVg90H21EOLACvtfcHk0liYXAIohgOQbg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtlGjsbjL&md5=e2f4548450cbe16c8acc0aef73ed1c8c</span></div><a href="/servlet/linkout?suffix=cit79&amp;dbid=16384&amp;doi=10.1021%2Fci300343w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci300343w%26sid%3Dliteratum%253Aachs%26aulast%3DVergara-Jaque%26aufirst%3DA.%26aulast%3DPoblete%26aufirst%3DH.%26aulast%3DLee%26aufirst%3DE.%2BH.%26aulast%3DSchulten%26aufirst%3DK.%26aulast%3DGonzalez-Nilo%26aufirst%3DF.%26aulast%3DChipot%26aufirst%3DC.%26atitle%3DMolecular%2520basis%2520of%2520drug%2520resistance%2520in%2520A%252FH1N1%2520virus%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26spage%3D2650%26epage%3D2656%26doi%3D10.1021%2Fci300343w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref80"><div class="reference"><strong class="refLabel"><a href="#ref80" class="rightTabRefNumLink">80</a></strong><div class="NLM_citation" id="rightTab-cit80"><span><span class="NLM_contrib-group">Boukharta, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gutierrez-de-Teran, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aqvist, J.</span><span> </span><span class="NLM_article-title">Computational prediction of alanine scanning and ligand binding energetics in G-protein coupled receptors</span> <span class="citation_source-journal">PLoS Comput. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">e1003585</span><span class="refDoi"> DOI: 10.1371/journal.pcbi.1003585</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1371%2Fjournal.pcbi.1003585" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=24743773" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVGls7bF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=e1003585&author=L.+Boukhartaauthor=H.+Gutierrez-de-Teranauthor=J.+Aqvist&title=Computational+prediction+of+alanine+scanning+and+ligand+binding+energetics+in+G-protein+coupled+receptors&doi=10.1371%2Fjournal.pcbi.1003585"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit80R"><div class="casContent"><span class="casTitleNuber">80</span><div class="casTitle"><span class="NLM_cas:atitle">Computational prediction of alanine scanning and ligand binding energetics in G-protein coupled receptors</span></div><div class="casAuthors">Boukharta, Lars; Gutieerrez-de-Teraen, Hugo; Aaqvist, Johan</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Computational Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">e1003585/1-e1003585/11, 11 pp.</span>CODEN:
                <span class="NLM_cas:coden">PCBLBG</span>;
        ISSN:<span class="NLM_cas:issn">1553-7358</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Site-directed mutagenesis combined with binding affinity measurements is widely used to probe the nature of ligand interactions with GPCRs.  Such expts., as well as structure-activity relationships for series of ligands, are usually interpreted with computationally derived models of ligand binding modes.  However, systematic approaches for accurate calcns. of the corresponding binding free energies are still lacking.  Here, we report a computational strategy to quant. predict the effects of alanine scanning and ligand modifications based on mol. dynamics free energy simulations.  A smooth stepwise scheme for free energy perturbation calcns. is derived and applied to a series of thirteen alanine mutations of the human neuropeptide Y1 receptor and series of eight analogous antagonists.  The robustness and accuracy of the method enables univocal interpretation of existing mutagenesis and binding data.  We show how these calcns. can be used to validate structural models and demonstrate their ability to discriminate against suboptimal ones.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoqb5_eZx503LVg90H21EOLACvtfcHk0lgOn5kc4pWZaQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVGls7bF&md5=478097bb0b20727daf628037457a7bd7</span></div><a href="/servlet/linkout?suffix=cit80&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pcbi.1003585&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pcbi.1003585%26sid%3Dliteratum%253Aachs%26aulast%3DBoukharta%26aufirst%3DL.%26aulast%3DGutierrez-de-Teran%26aufirst%3DH.%26aulast%3DAqvist%26aufirst%3DJ.%26atitle%3DComputational%2520prediction%2520of%2520alanine%2520scanning%2520and%2520ligand%2520binding%2520energetics%2520in%2520G-protein%2520coupled%2520receptors%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2014%26volume%3D10%26spage%3De1003585%26doi%3D10.1371%2Fjournal.pcbi.1003585" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref81"><div class="reference"><strong class="refLabel"><a href="#ref81" class="rightTabRefNumLink">81</a></strong><div class="NLM_citation" id="rightTab-cit81"><span><span class="NLM_contrib-group">Maurer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Beer, S. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oostenbrink, C.</span><span> </span><span class="NLM_article-title">Calculation of relative binding free energy in the water-filled active site of oligopeptide-binding protein A</span> <span class="citation_source-journal">Molecules</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">499</span><span class="refDoi"> DOI: 10.3390/molecules21040499</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.3390%2Fmolecules21040499" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=27092480" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVCktLfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2016&pages=499&author=M.+Maurerauthor=S.+B.+de+Beerauthor=C.+Oostenbrink&title=Calculation+of+relative+binding+free+energy+in+the+water-filled+active+site+of+oligopeptide-binding+protein+A&doi=10.3390%2Fmolecules21040499"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit81R"><div class="casContent"><span class="casTitleNuber">81</span><div class="casTitle"><span class="NLM_cas:atitle">Calculation of relative binding free energy in the water-filled active site of oligopeptide-binding protein A</span></div><div class="casAuthors">Maurer, Manuela; de Beer, Stephanie B. A.; Oostenbrink, Chris</div><div class="citationInfo"><span class="NLM_cas:title">Molecules</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">499/1-499/15</span>CODEN:
                <span class="NLM_cas:coden">MOLEFW</span>;
        ISSN:<span class="NLM_cas:issn">1420-3049</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">The periplasmic oligopeptide binding protein A (OppA) represents a well-known example of water-mediated protein-ligand interactions.  Here, we perform free-energy calcns. for three different ligands binding to OppA, using a thermodn. integration approach.  The tripeptide ligands share a high structural similarity (all have the sequence KXK), but their exptl.-detd. binding free energies differ remarkably.  Thermodn. cycles were constructed for the ligands, and simulations conducted in the bound and (freely solvated) unbound states.  In the unbound state, it was obsd. that the difference in conformational freedom between alanine and glycine leads to a surprisingly slow convergence, despite their chem. similarity.  This could be overcome by increasing the softness parameter during alchem. transformations.  Discrepancies remained in the bound state however, when comparing independent simulations of the three ligands.  These difficulties could be traced to a slow relaxation of the water network within the active site.  Fluctuations in the no. of water mols. residing in the binding cavity occur mostly on a timescale larger than the simulation time along the alchem. path.  After extensive simulations, relative binding free energies that were converged to within thermal noise could be obtained, which agree well with available exptl. data.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojG9n-hQ3wDLVg90H21EOLACvtfcHk0lgOn5kc4pWZaQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVCktLfJ&md5=5179bbcb1baa17bef2da25d34a245951</span></div><a href="/servlet/linkout?suffix=cit81&amp;dbid=16384&amp;doi=10.3390%2Fmolecules21040499&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fmolecules21040499%26sid%3Dliteratum%253Aachs%26aulast%3DMaurer%26aufirst%3DM.%26aulast%3Dde%2BBeer%26aufirst%3DS.%2BB.%26aulast%3DOostenbrink%26aufirst%3DC.%26atitle%3DCalculation%2520of%2520relative%2520binding%2520free%2520energy%2520in%2520the%2520water-filled%2520active%2520site%2520of%2520oligopeptide-binding%2520protein%2520A%26jtitle%3DMolecules%26date%3D2016%26volume%3D21%26spage%3D499%26doi%3D10.3390%2Fmolecules21040499" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref82"><div class="reference"><strong class="refLabel"><a href="#ref82" class="rightTabRefNumLink">82</a></strong><div class="NLM_citation" id="rightTab-cit82"><span><span class="NLM_contrib-group">Lai, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nagy, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garate, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oostenbrink, C.</span><span> </span><span class="NLM_article-title">Entropic and enthalpic contributions to stereospecific ligand binding from enhanced sampling methods</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">151</span><span class="NLM_x">–</span> <span class="NLM_lpage">158</span><span class="refDoi"> DOI: 10.1021/ci4006657</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci4006657" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC3sXitVWnsbrE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2014&pages=151-158&author=B.+Laiauthor=G.+Nagyauthor=J.+A.+Garateauthor=C.+Oostenbrink&title=Entropic+and+enthalpic+contributions+to+stereospecific+ligand+binding+from+enhanced+sampling+methods&doi=10.1021%2Fci4006657"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit82R"><div class="casContent"><span class="casTitleNuber">82</span><div class="casTitle"><span class="NLM_cas:atitle">Entropic and enthalpic contributions to stereospecific ligand binding from enhanced sampling methods</span></div><div class="casAuthors">Lai, Balder; Nagy, Gabor; Garate, Jose Antonio; Oostenbrink, Chris</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">151-158</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The stereoselective binding of R- and S-propranolol to the metabolic enzyme cytochrome P 450 2D6 and its mutant F483A was studied using various computational approaches.  Previously reported free-energy differences from Hamiltonian replica exchange simulations, combined with thermodn. integration, were compared to the one-step perturbation approach, combined with local-elevation enhanced sampling, and an excellent agreement between methods was obtained.  Further, the free-energy differences were interpreted in terms of enthalpic and entropic contributions where it was shown that exactly compensating contributions obscured a mol. interpretation of differences in the affinity while various reduced terms allowed a more detailed anal., which agreed with heuristic observations on the interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrU4EblLCkBJrVg90H21EOLACvtfcHk0lgOn5kc4pWZaQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXitVWnsbrE&md5=441379a9078f425cb709ee1f31cdfc0a</span></div><a href="/servlet/linkout?suffix=cit82&amp;dbid=16384&amp;doi=10.1021%2Fci4006657&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci4006657%26sid%3Dliteratum%253Aachs%26aulast%3DLai%26aufirst%3DB.%26aulast%3DNagy%26aufirst%3DG.%26aulast%3DGarate%26aufirst%3DJ.%2BA.%26aulast%3DOostenbrink%26aufirst%3DC.%26atitle%3DEntropic%2520and%2520enthalpic%2520contributions%2520to%2520stereospecific%2520ligand%2520binding%2520from%2520enhanced%2520sampling%2520methods%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2014%26volume%3D54%26spage%3D151%26epage%3D158%26doi%3D10.1021%2Fci4006657" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref83"><div class="reference"><strong class="refLabel"><a href="#ref83" class="rightTabRefNumLink">83</a></strong><div class="NLM_citation" id="rightTab-cit83"><span><span class="NLM_contrib-group">de Ruiter, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oostenbrink, C.</span><span> </span><span class="NLM_article-title">Efficient and accurate free energy calculations on trypsin inhibitors</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">3686</span><span class="NLM_x">–</span> <span class="NLM_lpage">3695</span><span class="refDoi"> DOI: 10.1021/ct200750p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct200750p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtFensbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2012&pages=3686-3695&author=A.+de+Ruiterauthor=C.+Oostenbrink&title=Efficient+and+accurate+free+energy+calculations+on+trypsin+inhibitors&doi=10.1021%2Fct200750p"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit83R"><div class="casContent"><span class="casTitleNuber">83</span><div class="casTitle"><span class="NLM_cas:atitle">Efficient and Accurate Free Energy Calculations on Trypsin Inhibitors</span></div><div class="casAuthors">de Ruiter, Anita; Oostenbrink, Chris</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3686-3695</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Several combinations of free energy calcn. methods have been applied to det. the relative free energies of binding between eight para-substituted benzamidines in complex with the serine protease trypsin.  With the aim to improve efficiency and maintain accuracy, the linear response approxn. (LRA), linear interaction energy (LIE) and third power fitting (TPF) are combined with the one-step perturbation (OSP) to det. the polar and apolar contributions to the free energy, resp.  The combination TPF/OSP gives the most accurate results and is 4.5 times more efficient than the rigorous thermodn. integration (TI).  By projecting the electrostatic preorganization energy from the OSP simulations, an increase in efficiency of a factor 7.5 can even be achieved.  Loss of accuracy with respect to the TI data is limited to 3.9 and 5.6 kJ/mol, resp., making it an attractive approach for lead optimization programs in drug research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbsob2EXdDIbVg90H21EOLACvtfcHk0ljcUp97XcupCA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtFensbk%253D&md5=f43a25a8fd2ffc594c53f45c4a55b2ff</span></div><a href="/servlet/linkout?suffix=cit83&amp;dbid=16384&amp;doi=10.1021%2Fct200750p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct200750p%26sid%3Dliteratum%253Aachs%26aulast%3Dde%2BRuiter%26aufirst%3DA.%26aulast%3DOostenbrink%26aufirst%3DC.%26atitle%3DEfficient%2520and%2520accurate%2520free%2520energy%2520calculations%2520on%2520trypsin%2520inhibitors%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2012%26volume%3D8%26spage%3D3686%26epage%3D3695%26doi%3D10.1021%2Fct200750p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref84"><div class="reference"><strong class="refLabel"><a href="#ref84" class="rightTabRefNumLink">84</a></strong><div class="NLM_citation" id="rightTab-cit84"><span><span class="NLM_contrib-group">De Beer, S. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Glattli, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hutzler, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vermeulen, N. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oostenbrink, C.</span><span> </span><span class="NLM_article-title">Molecular dynamics simulations and free energy calculations on the enzyme 4-hydroxyphenylpyruvate dioxygenase</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">32</span><span class="NLM_x">, </span> <span class="NLM_fpage">2160</span><span class="NLM_x">–</span> <span class="NLM_lpage">2169</span><span class="refDoi"> DOI: 10.1002/jcc.21798</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1002%2Fjcc.21798" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2011&pages=2160-2169&author=S.+B.+De+Beerauthor=A.+Glattliauthor=J.+Hutzlerauthor=N.+P.+Vermeulenauthor=C.+Oostenbrink&title=Molecular+dynamics+simulations+and+free+energy+calculations+on+the+enzyme+4-hydroxyphenylpyruvate+dioxygenase&doi=10.1002%2Fjcc.21798"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit84&amp;dbid=16384&amp;doi=10.1002%2Fjcc.21798&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.21798%26sid%3Dliteratum%253Aachs%26aulast%3DDe%2BBeer%26aufirst%3DS.%2BB.%26aulast%3DGlattli%26aufirst%3DA.%26aulast%3DHutzler%26aufirst%3DJ.%26aulast%3DVermeulen%26aufirst%3DN.%2BP.%26aulast%3DOostenbrink%26aufirst%3DC.%26atitle%3DMolecular%2520dynamics%2520simulations%2520and%2520free%2520energy%2520calculations%2520on%2520the%2520enzyme%25204-hydroxyphenylpyruvate%2520dioxygenase%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2011%26volume%3D32%26spage%3D2160%26epage%3D2169%26doi%3D10.1002%2Fjcc.21798" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref85"><div class="reference"><strong class="refLabel"><a href="#ref85" class="rightTabRefNumLink">85</a></strong><div class="NLM_citation" id="rightTab-cit85"><span><span class="NLM_contrib-group">Chiang, Y. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y.</span><span> </span><span class="NLM_article-title">Virtual substitution scan via single-step free energy perturbation</span> <span class="citation_source-journal">Biopolymers</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">105</span><span class="NLM_x">, </span> <span class="NLM_fpage">324</span><span class="NLM_x">–</span> <span class="NLM_lpage">336</span><span class="refDoi"> DOI: 10.1002/bip.22820</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1002%2Fbip.22820" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=105&publication_year=2016&pages=324-336&author=Y.+C.+Chiangauthor=Y.+Wang&title=Virtual+substitution+scan+via+single-step+free+energy+perturbation&doi=10.1002%2Fbip.22820"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit85&amp;dbid=16384&amp;doi=10.1002%2Fbip.22820&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fbip.22820%26sid%3Dliteratum%253Aachs%26aulast%3DChiang%26aufirst%3DY.%2BC.%26aulast%3DWang%26aufirst%3DY.%26atitle%3DVirtual%2520substitution%2520scan%2520via%2520single-step%2520free%2520energy%2520perturbation%26jtitle%3DBiopolymers%26date%3D2016%26volume%3D105%26spage%3D324%26epage%3D336%26doi%3D10.1002%2Fbip.22820" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref86"><div class="reference"><strong class="refLabel"><a href="#ref86" class="rightTabRefNumLink">86</a></strong><div class="NLM_citation" id="rightTab-cit86"><span><span class="NLM_contrib-group">Zeevaart, J. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, L. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thakur, V. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leung, C. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tirado-Rives, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bailey, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Domaoal, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Anderson, K. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jorgensen, W. L.</span><span> </span><span class="NLM_article-title">Optimization of azoles as anti-human immunodeficiency virus agents guided by free-energy calculations</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">130</span><span class="NLM_x">, </span> <span class="NLM_fpage">9492</span><span class="NLM_x">–</span> <span class="NLM_lpage">9499</span><span class="refDoi"> DOI: 10.1021/ja8019214</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja8019214" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BD1cXnvF2hs7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=130&publication_year=2008&pages=9492-9499&author=J.+G.+Zeevaartauthor=L.+G.+Wangauthor=V.+V.+Thakurauthor=C.+S.+Leungauthor=J.+Tirado-Rivesauthor=C.+M.+Baileyauthor=R.+A.+Domaoalauthor=K.+S.+Andersonauthor=W.+L.+Jorgensen&title=Optimization+of+azoles+as+anti-human+immunodeficiency+virus+agents+guided+by+free-energy+calculations&doi=10.1021%2Fja8019214"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit86R"><div class="casContent"><span class="casTitleNuber">86</span><div class="casTitle"><span class="NLM_cas:atitle">Optimization of Azoles as Anti-Human Immunodeficiency Virus Agents Guided by Free-Energy Calculations</span></div><div class="casAuthors">Zeevaart, Jacob G.; Wang, Ligong; Thakur, Vinay V.; Leung, Cheryl S.; Tirado-Rives, Julian; Bailey, Christopher M.; Domaoal, Robert A.; Anderson, Karen S.; Jorgensen, William L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">130</span>
        (<span class="NLM_cas:issue">29</span>),
    <span class="NLM_cas:pages">9492-9499</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Efficient optimization of an inactive 2-anilinyl-5-benzyloxadiazole core has been guided by free energy perturbation (FEP) calcns. to provide potent nonnucleoside inhibitors of human immunodeficiency virus (HIV) reverse transcriptase (NNRTIs).  An FEP "chlorine scan" was performed to identify the most promising sites for substitution of aryl hydrogens.  This yielded NNRTIs (I and II) with activities (EC50) of 820 and 310 nM for protection of human T-cells from infection by wild-type HIV-1.  FEP calcns. for addnl. substituent modifications and change of the core heterocycle readily led to oxazoles (III and IV), which were confirmed as highly potent anti-HIV agents with activities in the 10-20 nM range.  The designed compds. were also monitored for possession of desirable pharmacol. properties by use of addnl. computational tools.  Overall, the trends predicted by the FEP calcns. were well borne out by the assay results.  FEP-guided lead optimization is confirmed as a valuable tool for mol. design including drug discovery; chlorine scans are particularly attractive since they are both straightforward to perform and highly informative.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZtMlzatLNhLVg90H21EOLACvtfcHk0ljcUp97XcupCA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXnvF2hs7s%253D&md5=24df481811465307254908323aa470b3</span></div><a href="/servlet/linkout?suffix=cit86&amp;dbid=16384&amp;doi=10.1021%2Fja8019214&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja8019214%26sid%3Dliteratum%253Aachs%26aulast%3DZeevaart%26aufirst%3DJ.%2BG.%26aulast%3DWang%26aufirst%3DL.%2BG.%26aulast%3DThakur%26aufirst%3DV.%2BV.%26aulast%3DLeung%26aufirst%3DC.%2BS.%26aulast%3DTirado-Rives%26aufirst%3DJ.%26aulast%3DBailey%26aufirst%3DC.%2BM.%26aulast%3DDomaoal%26aufirst%3DR.%2BA.%26aulast%3DAnderson%26aufirst%3DK.%2BS.%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26atitle%3DOptimization%2520of%2520azoles%2520as%2520anti-human%2520immunodeficiency%2520virus%2520agents%2520guided%2520by%2520free-energy%2520calculations%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2008%26volume%3D130%26spage%3D9492%26epage%3D9499%26doi%3D10.1021%2Fja8019214" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref87"><div class="reference"><strong class="refLabel"><a href="#ref87" class="rightTabRefNumLink">87</a></strong><div class="NLM_citation" id="rightTab-cit87"><span><span class="NLM_contrib-group">Jorgensen, W. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bollini, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thakur, V. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Domaoal, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spasov, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Anderson, K. S.</span><span> </span><span class="NLM_article-title">Efficient discovery of potent anti-HIV agents targeting the Tyr181Cys variant of HIV reverse transcriptase</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">133</span><span class="NLM_x">, </span> <span class="NLM_fpage">15686</span><span class="NLM_x">–</span> <span class="NLM_lpage">15696</span><span class="refDoi"> DOI: 10.1021/ja2058583</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja2058583" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFCgs73I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=133&publication_year=2011&pages=15686-15696&author=W.+L.+Jorgensenauthor=M.+Bolliniauthor=V.+V.+Thakurauthor=R.+A.+Domaoalauthor=K.+A.+Spasovauthor=K.+S.+Anderson&title=Efficient+discovery+of+potent+anti-HIV+agents+targeting+the+Tyr181Cys+variant+of+HIV+reverse+transcriptase&doi=10.1021%2Fja2058583"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit87R"><div class="casContent"><span class="casTitleNuber">87</span><div class="casTitle"><span class="NLM_cas:atitle">Efficient Discovery of Potent Anti-HIV Agents Targeting the Tyr181Cys Variant of HIV Reverse Transcriptase</span></div><div class="casAuthors">Jorgensen, William L.; Bollini, Mariela; Thakur, Vinay V.; Domaoal, Robert A.; Spasov, Krasimir A.; Anderson, Karen S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">133</span>
        (<span class="NLM_cas:issue">39</span>),
    <span class="NLM_cas:pages">15686-15696</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Non-nucleoside reverse transcriptase inhibitors (NNRTIs) that interfere with the replication of human immunodeficiency virus (HIV) are being pursued with guidance from mol. modeling including free-energy perturbation (FEP) calcns. for protein-inhibitor binding affinities.  The previously reported pyrimidinylphenylamine 1 (I) and its chloro analog 2 are potent anti-HIV agents; they inhibit replication of wild-type HIV-1 in infected human T-cells with EC50 values of 2 and 10 nM, resp.  However, they show no activity against viral strains contg. the Tyr181Cys (Y181C) mutation in HIV-RT.  Modeling indicates that the problem is likely assocd. with extensive interaction between the dimethylallyloxy substituent and Tyr181.  As an alternative, a phenoxy group is computed to be oriented in a manner diminishing the contact with Tyr181.  However, this replacement leads to a roughly 1000-fold loss of activity for 3 (2.5 μM).  The present report details the efficient, computationally driven evolution of 3 to novel NNRTIs with sub-10 nM potency toward both wild-type HIV-1 and Y181C-contg. variants.  The crit. contributors were FEP substituent scans for the phenoxy and pyrimidine rings and recognition of potential benefits of addn. of a cyanovinyl group to the phenoxy ring.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXO8wk_lbYPrVg90H21EOLACvtfcHk0liGZIuaBgmgFg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFCgs73I&md5=e75db60a08ad0c22ef6abcb5e1ea4891</span></div><a href="/servlet/linkout?suffix=cit87&amp;dbid=16384&amp;doi=10.1021%2Fja2058583&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja2058583%26sid%3Dliteratum%253Aachs%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26aulast%3DBollini%26aufirst%3DM.%26aulast%3DThakur%26aufirst%3DV.%2BV.%26aulast%3DDomaoal%26aufirst%3DR.%2BA.%26aulast%3DSpasov%26aufirst%3DK.%2BA.%26aulast%3DAnderson%26aufirst%3DK.%2BS.%26atitle%3DEfficient%2520discovery%2520of%2520potent%2520anti-HIV%2520agents%2520targeting%2520the%2520Tyr181Cys%2520variant%2520of%2520HIV%2520reverse%2520transcriptase%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2011%26volume%3D133%26spage%3D15686%26epage%3D15696%26doi%3D10.1021%2Fja2058583" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref88"><div class="reference"><strong class="refLabel"><a href="#ref88" class="rightTabRefNumLink">88</a></strong><div class="NLM_citation" id="rightTab-cit88"><span><span class="NLM_contrib-group">Leung, C. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leung, S. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tirado-Rives, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jorgensen, W. L.</span><span> </span><span class="NLM_article-title">Methyl effects on protein-ligand binding</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">4489</span><span class="NLM_x">–</span> <span class="NLM_lpage">4500</span><span class="refDoi"> DOI: 10.1021/jm3003697</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm3003697" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC38XlsV2iu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=4489-4500&author=C.+S.+Leungauthor=S.+S.+Leungauthor=J.+Tirado-Rivesauthor=W.+L.+Jorgensen&title=Methyl+effects+on+protein-ligand+binding&doi=10.1021%2Fjm3003697"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit88R"><div class="casContent"><span class="casTitleNuber">88</span><div class="casTitle"><span class="NLM_cas:atitle">Methyl Effects on Protein-Ligand Binding</span></div><div class="casAuthors">Leung, Cheryl S.; Leung, Siegfried S. F.; Tirado-Rives, Julian; Jorgensen, William L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4489-4500</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The effects of addn. of a Me group to a lead compd. on biol. activity are examd.  A literature anal. of >2000 cases reveals that an activity boost of a factor of 10 or more is found with an 8% frequency, and a 100-fold boost is a 1 in 200 event.  Four cases in the latter category are analyzed in depth to elucidate any unusual aspects of the protein-ligand binding, distribution of water mols., and changes in conformational energetics.  The analyses include Monte Carlo/free-energy perturbation (MC/FEP) calcns. for Me replacements in inhibitor series for p38α MAP kinase, ACK1, PTP1B, and thrombin.  Me substitutions ortho to an aryl ring can be particularly effective at improving activity by inducing a propitious conformational change.  The greatest improvements in activity arise from coupling the conformational gain with the burial of the Me group in a hydrophobic region of the protein.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoX0Czyhgy6KbVg90H21EOLACvtfcHk0liGZIuaBgmgFg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XlsV2iu7Y%253D&md5=0ea58d9c1bf845d383931b6a017f9b37</span></div><a href="/servlet/linkout?suffix=cit88&amp;dbid=16384&amp;doi=10.1021%2Fjm3003697&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm3003697%26sid%3Dliteratum%253Aachs%26aulast%3DLeung%26aufirst%3DC.%2BS.%26aulast%3DLeung%26aufirst%3DS.%2BS.%26aulast%3DTirado-Rives%26aufirst%3DJ.%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26atitle%3DMethyl%2520effects%2520on%2520protein-ligand%2520binding%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D4489%26epage%3D4500%26doi%3D10.1021%2Fjm3003697" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref89"><div class="reference"><strong class="refLabel"><a href="#ref89" class="rightTabRefNumLink">89</a></strong><div class="NLM_citation" id="rightTab-cit89"><span><span class="NLM_contrib-group">Cole, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tirado-Rives, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jorgensen, W. L.</span><span> </span><span class="NLM_article-title">Molecular dynamics and Monte Carlo simulations for protein-ligand binding and inhibitor design</span> <span class="citation_source-journal">Biochim. Biophys. Acta, Gen. Subj.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">1850</span><span class="NLM_x">, </span> <span class="NLM_fpage">966</span><span class="NLM_x">–</span> <span class="NLM_lpage">971</span><span class="refDoi"> DOI: 10.1016/j.bbagen.2014.08.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1016%2Fj.bbagen.2014.08.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=25196360" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsFWltrvK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1850&publication_year=2015&pages=966-971&author=D.+J.+Coleauthor=J.+Tirado-Rivesauthor=W.+L.+Jorgensen&title=Molecular+dynamics+and+Monte+Carlo+simulations+for+protein-ligand+binding+and+inhibitor+design&doi=10.1016%2Fj.bbagen.2014.08.018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit89R"><div class="casContent"><span class="casTitleNuber">89</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular dynamics and Monte Carlo simulations for protein-ligand binding and inhibitor design</span></div><div class="casAuthors">Cole, Daniel J.; Tirado-Rives, Julian; Jorgensen, William L.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, General Subjects</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">1850</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">966-971</span>CODEN:
                <span class="NLM_cas:coden">BBGSB3</span>;
        ISSN:<span class="NLM_cas:issn">0304-4165</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Non-nucleoside inhibitors of HIV reverse transcriptase are an important component of treatment against HIV infection.  Novel inhibitors are sought that increase potency against variants that contain the Tyr181Cys mutation.  Mol. dynamics based free energy perturbation simulations have been run to study factors that contribute to protein-ligand binding, and the results are compared with those from previous Monte Carlo based simulations and activity data.  Predictions of protein-ligand binding modes are very consistent for the two simulation methods; the accord is attributed to the use of an enhanced sampling protocol.  The Tyr181Cys binding pocket supports large, hydrophobic substituents, which is in good agreement with expt.  Although some discrepancies exist between the results of the two simulation methods and expt., free energy perturbation simulations can be used to rapidly test small mols. for gains in binding affinity.  Free energy perturbation methods show promise in providing fast, reliable and accurate data that can be used to complement expt. in lead optimization projects.  This article is part of a Special Issue entitled "Recent developments of mol. dynamics".</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1gmI-9_FKtrVg90H21EOLACvtfcHk0liGZIuaBgmgFg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsFWltrvK&md5=2fd7025d2402b77ba6fb2e091733bc12</span></div><a href="/servlet/linkout?suffix=cit89&amp;dbid=16384&amp;doi=10.1016%2Fj.bbagen.2014.08.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbagen.2014.08.018%26sid%3Dliteratum%253Aachs%26aulast%3DCole%26aufirst%3DD.%2BJ.%26aulast%3DTirado-Rives%26aufirst%3DJ.%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26atitle%3DMolecular%2520dynamics%2520and%2520Monte%2520Carlo%2520simulations%2520for%2520protein-ligand%2520binding%2520and%2520inhibitor%2520design%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Gen.%2520Subj.%26date%3D2015%26volume%3D1850%26spage%3D966%26epage%3D971%26doi%3D10.1016%2Fj.bbagen.2014.08.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref90"><div class="reference"><strong class="refLabel"><a href="#ref90" class="rightTabRefNumLink">90</a></strong><div class="NLM_citation" id="rightTab-cit90"><span><span class="NLM_contrib-group">Lovering, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aevazelis, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dehnhardt, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fitz, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Han, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janz, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaila, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McDonald, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moore, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moretto, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Papaioannou, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richard, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ryan, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wan, Z. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thorarensen, A.</span><span> </span><span class="NLM_article-title">Imidazotriazines: spleen tyrosine kinase (Syk) inhibitors identified by free-energy perturbation (FEP)</span> <span class="citation_source-journal">ChemMedChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">217</span><span class="NLM_x">–</span> <span class="NLM_lpage">233</span><span class="refDoi"> DOI: 10.1002/cmdc.201500333</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref90/cit90&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1002%2Fcmdc.201500333" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref90/cit90&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=26381330" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref90/cit90&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFWlsLnO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=217-233&author=F.+Loveringauthor=C.+Aevazelisauthor=J.+Changauthor=C.+Dehnhardtauthor=L.+Fitzauthor=S.+Hanauthor=K.+Janzauthor=J.+Leeauthor=N.+Kailaauthor=J.+McDonaldauthor=W.+Mooreauthor=A.+Morettoauthor=N.+Papaioannouauthor=D.+Richardauthor=M.+S.+Ryanauthor=Z.+K.+Wanauthor=A.+Thorarensen&title=Imidazotriazines%3A+spleen+tyrosine+kinase+%28Syk%29+inhibitors+identified+by+free-energy+perturbation+%28FEP%29&doi=10.1002%2Fcmdc.201500333"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit90R"><div class="casContent"><span class="casTitleNuber">90</span><div class="casTitle"><span class="NLM_cas:atitle">Imidazotriazines: Spleen Tyrosine Kinase (Syk) Inhibitors Identified by Free-Energy Perturbation (FEP)</span></div><div class="casAuthors">Lovering, Frank; Aevazelis, Cristina; Chang, Jeanne; Dehnhardt, Christoph; Fitz, Lori; Han, Seungil; Janz, Kristin; Lee, Julie; Kaila, Neelu; McDonald, Joseph; Moore, William; Moretto, Alessandro; Papaioannou, Nikolaos; Richard, David; Ryan, Mark S.; Wan, Zhao-Kui; Thorarensen, Atli</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">217-233</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">There has been significant interest in spleen tyrosine kinase (Syk) owing to its role in a no. of disease states, including autoimmunity, inflammation, and cancer.  Ongoing therapeutic programs have resulted in several compds. that are now in clin. use.  Herein the authors report the optimization of the imidazopyrazine core scaffold of Syk inhibitors through the use of empirical and computational approaches.  Free-energy perturbation (FEP) methods with MCPRO+ were undertaken to calc. the relative binding free energies for several alternate scaffolds.  FEP was first applied retrospectively to det. if there is any predictive value; this resulted in 12 of 13 transformations being predicted in a directionally correct manner.  FEP was then applied in a prospective manner to evaluate 17 potential targets, resulting in the realization of imidazotriazine I, which shows a tenfold improvement in activity relative to the parent compd. and no increase in atom count.  Optimization of I led to compds. with nanomolar cellular activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpNV2-OpOenz7Vg90H21EOLACvtfcHk0lim7mg9r1mYnw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFWlsLnO&md5=2797465d9360299ecc5d65499e338bbe</span></div><a href="/servlet/linkout?suffix=cit90&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201500333&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201500333%26sid%3Dliteratum%253Aachs%26aulast%3DLovering%26aufirst%3DF.%26aulast%3DAevazelis%26aufirst%3DC.%26aulast%3DChang%26aufirst%3DJ.%26aulast%3DDehnhardt%26aufirst%3DC.%26aulast%3DFitz%26aufirst%3DL.%26aulast%3DHan%26aufirst%3DS.%26aulast%3DJanz%26aufirst%3DK.%26aulast%3DLee%26aufirst%3DJ.%26aulast%3DKaila%26aufirst%3DN.%26aulast%3DMcDonald%26aufirst%3DJ.%26aulast%3DMoore%26aufirst%3DW.%26aulast%3DMoretto%26aufirst%3DA.%26aulast%3DPapaioannou%26aufirst%3DN.%26aulast%3DRichard%26aufirst%3DD.%26aulast%3DRyan%26aufirst%3DM.%2BS.%26aulast%3DWan%26aufirst%3DZ.%2BK.%26aulast%3DThorarensen%26aufirst%3DA.%26atitle%3DImidazotriazines%253A%2520spleen%2520tyrosine%2520kinase%2520%2528Syk%2529%2520inhibitors%2520identified%2520by%2520free-energy%2520perturbation%2520%2528FEP%2529%26jtitle%3DChemMedChem%26date%3D2016%26volume%3D11%26spage%3D217%26epage%3D233%26doi%3D10.1002%2Fcmdc.201500333" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref91"><div class="reference"><strong class="refLabel"><a href="#ref91" class="rightTabRefNumLink">91</a></strong><div class="NLM_citation" id="rightTab-cit91"><span><span class="NLM_contrib-group">Rathore, R. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sumakanth, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reddy, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reddanna, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rao, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Erion, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reddy, M. R.</span><span> </span><span class="NLM_article-title">Advances in binding free energies calculations: QM/MM-based free energy perturbation method for drug design</span> <span class="citation_source-journal">Curr. Pharm. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">4674</span><span class="NLM_x">–</span> <span class="NLM_lpage">4686</span><span class="refDoi"> DOI: 10.2174/1381612811319260002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.2174%2F1381612811319260002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=23260025" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1Wns7rO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2013&pages=4674-4686&author=R.+S.+Rathoreauthor=M.+Sumakanthauthor=M.+S.+Reddyauthor=P.+Reddannaauthor=A.+A.+Raoauthor=M.+D.+Erionauthor=M.+R.+Reddy&title=Advances+in+binding+free+energies+calculations%3A+QM%2FMM-based+free+energy+perturbation+method+for+drug+design&doi=10.2174%2F1381612811319260002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit91R"><div class="casContent"><span class="casTitleNuber">91</span><div class="casTitle"><span class="NLM_cas:atitle">Advances in Binding Free Energies Calculations: QM/MM-Based Free Energy Perturbation Method for Drug Design</span></div><div class="casAuthors">Rathore, R. S.; Sumakanth, M.; Reddy, M. Siva; Reddanna, P.; Rao, Allam Appa; Erion, Mark D.; Reddy, M. R.</div><div class="citationInfo"><span class="NLM_cas:title">Current Pharmaceutical Design</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">4674-4686</span>CODEN:
                <span class="NLM_cas:coden">CPDEFP</span>;
        ISSN:<span class="NLM_cas:issn">1381-6128</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Multiple approaches have been devised and evaluated to computationally est. binding free energies.  Results using a recently developed Quantum Mechanics (QM)/Mol. Mechanics (MM) based Free Energy Perturbation (FEP) method suggest that this method has the potential to provide the most accurate estn. of binding affinities to date.  The method treats ligands/inhibitors using QM while using MM for the rest of the system.  The method has been applied and validated for a structurally diverse set of fructose 1,6- bisphosphatase (FBPase) inhibitors suggesting that the approach has the potential to be used as an integral part of drug discovery for both lead identification lead optimization, where there is a structure available.  In addn., this QM/MM-based FEP method was shown to accurately replicate the anomalous hydration behavior exhibited by simple amines and amides suggesting that the method may also prove useful in predicting phys. properties of mols.  While the method is about 5-fold more computationally demanding than conventional FEP, it has the potential to be less demanding on the end user since it avoids development of MM force field parameters for novel ligands and thereby eliminates this time-consuming step that often contributes significantly to the inaccuracy of binding affinity predictions using conventional FEP methods.  The QM/MM-based FEP method has been extensively tested with respect to important considerations such as the length of the simulation required to obtain satisfactory convergence in the calcd. relative solvation and binding free energies for both small and large structural changes between ligands.  Future automation of the method and parallelization of the code is expected to enhance the speed and increase its use for drug design and lead optimization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8cjSLqWli5bVg90H21EOLACvtfcHk0lim7mg9r1mYnw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1Wns7rO&md5=04862c0dd7413ff5aeaf598459d234dc</span></div><a href="/servlet/linkout?suffix=cit91&amp;dbid=16384&amp;doi=10.2174%2F1381612811319260002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1381612811319260002%26sid%3Dliteratum%253Aachs%26aulast%3DRathore%26aufirst%3DR.%2BS.%26aulast%3DSumakanth%26aufirst%3DM.%26aulast%3DReddy%26aufirst%3DM.%2BS.%26aulast%3DReddanna%26aufirst%3DP.%26aulast%3DRao%26aufirst%3DA.%2BA.%26aulast%3DErion%26aufirst%3DM.%2BD.%26aulast%3DReddy%26aufirst%3DM.%2BR.%26atitle%3DAdvances%2520in%2520binding%2520free%2520energies%2520calculations%253A%2520QM%252FMM-based%2520free%2520energy%2520perturbation%2520method%2520for%2520drug%2520design%26jtitle%3DCurr.%2520Pharm.%2520Des.%26date%3D2013%26volume%3D19%26spage%3D4674%26epage%3D4686%26doi%3D10.2174%2F1381612811319260002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref92"><div class="reference"><strong class="refLabel"><a href="#ref92" class="rightTabRefNumLink">92</a></strong><div class="NLM_citation" id="rightTab-cit92"><span><span class="NLM_contrib-group">Genheden, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ryde, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soderhjelm, P.</span><span> </span><span class="NLM_article-title">Binding affinities by alchemical perturbation using QM/MM with a large QM system and polarizable MM model</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">36</span><span class="NLM_x">, </span> <span class="NLM_fpage">2114</span><span class="NLM_x">–</span> <span class="NLM_lpage">2124</span><span class="refDoi"> DOI: 10.1002/jcc.24048</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1002%2Fjcc.24048" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=26280564" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtlCisbjI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2015&pages=2114-2124&author=S.+Genhedenauthor=U.+Rydeauthor=P.+Soderhjelm&title=Binding+affinities+by+alchemical+perturbation+using+QM%2FMM+with+a+large+QM+system+and+polarizable+MM+model&doi=10.1002%2Fjcc.24048"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit92R"><div class="casContent"><span class="casTitleNuber">92</span><div class="casTitle"><span class="NLM_cas:atitle">Binding affinities by alchemical perturbation using QM/MM with a large QM system and polarizable MM model</span></div><div class="casAuthors">Genheden, Samuel; Ryde, Ulf; Soederhjelm, Paer</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">28</span>),
    <span class="NLM_cas:pages">2114-2124</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">The most general way to improve the accuracy of binding-affinity calcns. for protein-ligand systems is to use quantum-mech. (QM) methods together with rigorous alchem.-perturbation (AP) methods.  We explore this approach by calcg. the relative binding free energy of two synthetic disaccharides binding to galectin-3 at a reasonably high QM level (dispersion-cor. d. functional theory with a triple-zeta basis set) and with a sufficiently large QM system to include all short-range interactions with the ligand (744-748 atoms).  The rest of the protein is treated as a collection of at. multipoles (up to quadrupoles) and polarizabilities.  Several methods for evaluating the binding free energy from the 3600 QM calcns. are investigated in terms of stability and accuracy.  In particular, methods using QM calcns. only at the endpoints of the transformation are compared with the recently proposed non-Boltzmann Bennett acceptance ratio (NBB) method that uses QM calcns. at several stages of the transformation.  Unfortunately, none of the rigorous approaches give sufficient statistical precision.  However, a novel approx. method, involving the direct use of QM energies in the Bennett acceptance ratio method, gives similar results as NBB but with better precision, ∼3 kJ/mol.  The statistical error can be further reduced by performing a greater no. of QM calcns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqd0jf7UhCPvLVg90H21EOLACvtfcHk0lim7mg9r1mYnw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtlCisbjI&md5=8e02ea35846d8335dfa2259d303b2e0c</span></div><a href="/servlet/linkout?suffix=cit92&amp;dbid=16384&amp;doi=10.1002%2Fjcc.24048&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.24048%26sid%3Dliteratum%253Aachs%26aulast%3DGenheden%26aufirst%3DS.%26aulast%3DRyde%26aufirst%3DU.%26aulast%3DSoderhjelm%26aufirst%3DP.%26atitle%3DBinding%2520affinities%2520by%2520alchemical%2520perturbation%2520using%2520QM%252FMM%2520with%2520a%2520large%2520QM%2520system%2520and%2520polarizable%2520MM%2520model%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2015%26volume%3D36%26spage%3D2114%26epage%3D2124%26doi%3D10.1002%2Fjcc.24048" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref93"><div class="reference"><strong class="refLabel"><a href="#ref93" class="rightTabRefNumLink">93</a></strong><div class="NLM_citation" id="rightTab-cit93"><span><span class="NLM_contrib-group">Ignjatovic, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caldararu, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Munoz-Gutierrez, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adasme-Carreno, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ryde, U.</span><span> </span><span class="NLM_article-title">Binding-affinity predictions of HSP90 in the D3R Grand Challenge 2015 with docking, MM/GBSA, QM/MM, and free-energy simulations</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">30</span><span class="NLM_x">, </span> <span class="NLM_fpage">707</span><span class="NLM_x">–</span> <span class="NLM_lpage">730</span><span class="refDoi"> DOI: 10.1007/s10822-016-9942-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1007%2Fs10822-016-9942-z" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2016&pages=707-730&author=M.+M.+Ignjatovicauthor=O.+Caldararuauthor=G.+Dongauthor=C.+Munoz-Gutierrezauthor=F.+Adasme-Carrenoauthor=U.+Ryde&title=Binding-affinity+predictions+of+HSP90+in+the+D3R+Grand+Challenge+2015+with+docking%2C+MM%2FGBSA%2C+QM%2FMM%2C+and+free-energy+simulations&doi=10.1007%2Fs10822-016-9942-z"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit93&amp;dbid=16384&amp;doi=10.1007%2Fs10822-016-9942-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-016-9942-z%26sid%3Dliteratum%253Aachs%26aulast%3DIgnjatovic%26aufirst%3DM.%2BM.%26aulast%3DCaldararu%26aufirst%3DO.%26aulast%3DDong%26aufirst%3DG.%26aulast%3DMunoz-Gutierrez%26aufirst%3DC.%26aulast%3DAdasme-Carreno%26aufirst%3DF.%26aulast%3DRyde%26aufirst%3DU.%26atitle%3DBinding-affinity%2520predictions%2520of%2520HSP90%2520in%2520the%2520D3R%2520Grand%2520Challenge%25202015%2520with%2520docking%252C%2520MM%252FGBSA%252C%2520QM%252FMM%252C%2520and%2520free-energy%2520simulations%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2016%26volume%3D30%26spage%3D707%26epage%3D730%26doi%3D10.1007%2Fs10822-016-9942-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref94"><div class="reference"><strong class="refLabel"><a href="#ref94" class="rightTabRefNumLink">94</a></strong><div class="NLM_citation" id="rightTab-cit94"><span><span class="NLM_contrib-group">Fujitani, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tanida, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ito, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jayachandran, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Snow, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shirts, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sorin, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pande, V. S.</span><span> </span><span class="NLM_article-title">Direct calculation of the binding free energies of FKBP ligands</span> <span class="citation_source-journal">J. Chem. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">123</span><span class="NLM_x">, </span> <span class="NLM_fpage">084108</span><span class="refDoi"> DOI: 10.1063/1.1999637</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1063%2F1.1999637" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=16164283" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BD2MXpvFCksb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=123&publication_year=2005&pages=084108&author=H.+Fujitaniauthor=Y.+Tanidaauthor=M.+Itoauthor=G.+Jayachandranauthor=C.+D.+Snowauthor=M.+R.+Shirtsauthor=E.+J.+Sorinauthor=V.+S.+Pande&title=Direct+calculation+of+the+binding+free+energies+of+FKBP+ligands&doi=10.1063%2F1.1999637"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit94R"><div class="casContent"><span class="casTitleNuber">94</span><div class="casTitle"><span class="NLM_cas:atitle">Direct calculation of the binding free energies of FKBP ligands</span></div><div class="casAuthors">Fujitani, Hideaki; Tanida, Yoshiaki; Ito, Masakatsu; Jayachandran, Guha; Snow, Christopher D.; Shirts, Michael R.; Sorin, Eric J.; Pande, Vijay S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">123</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">084108/1-084108/5</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    
            (<span class="NLM_cas:orgname">American Institute of Physics</span>)
        </div><div class="casAbstract">Direct calcns. of the abs. free energies of binding for eight ligands to FKBP protein were performed using the Fujitsu BioServer massively parallel computer.  Using the latest version of the general assisted model building with energy refinement (AMBER) force field for ligand model parameters and the Bennett acceptance ratio for computing free-energy differences, we obtained an excellent linear fit between the calcd. and exptl. binding free energies.  The rms error from a linear fit is 0.4 kcal/mol for eight ligand complexes.  In comparison with a previous study of the binding energies of these same eight ligand complexes, these results suggest that the use of improved model parameters can lead to more predictive binding ests., and that these ests. can be obtained with significantly less computer time than previously thought.  These findings make such direct methods more attractive for use in rational drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKfcZ36Ys0aLVg90H21EOLACvtfcHk0ljoDBE_dSzg3w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXpvFCksb8%253D&md5=4ee04c498d1f0053c760d93287aa408b</span></div><a href="/servlet/linkout?suffix=cit94&amp;dbid=16384&amp;doi=10.1063%2F1.1999637&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.1999637%26sid%3Dliteratum%253Aachs%26aulast%3DFujitani%26aufirst%3DH.%26aulast%3DTanida%26aufirst%3DY.%26aulast%3DIto%26aufirst%3DM.%26aulast%3DJayachandran%26aufirst%3DG.%26aulast%3DSnow%26aufirst%3DC.%2BD.%26aulast%3DShirts%26aufirst%3DM.%2BR.%26aulast%3DSorin%26aufirst%3DE.%2BJ.%26aulast%3DPande%26aufirst%3DV.%2BS.%26atitle%3DDirect%2520calculation%2520of%2520the%2520binding%2520free%2520energies%2520of%2520FKBP%2520ligands%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D2005%26volume%3D123%26spage%3D084108%26doi%3D10.1063%2F1.1999637" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref95"><div class="reference"><strong class="refLabel"><a href="#ref95" class="rightTabRefNumLink">95</a></strong><div class="NLM_citation" id="rightTab-cit95"><span><span class="NLM_contrib-group">Archontis, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Watson, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Andreou, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chrysina, E. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zographos, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oikonomakos, N. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karplus, M.</span><span> </span><span class="NLM_article-title">Glycogen phosphorylase inhibitors: a free energy perturbation analysis of glucopyranose spirohydantoin analogues</span> <span class="citation_source-journal">Proteins: Struct., Funct., Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">61</span><span class="NLM_x">, </span> <span class="NLM_fpage">984</span><span class="NLM_x">–</span> <span class="NLM_lpage">998</span><span class="refDoi"> DOI: 10.1002/prot.20641</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref95/cit95&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1002%2Fprot.20641" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2005&pages=984-998&author=G.+Archontisauthor=K.+A.+Watsonauthor=Q.+Xieauthor=G.+Andreouauthor=E.+D.+Chrysinaauthor=S.+E.+Zographosauthor=N.+G.+Oikonomakosauthor=M.+Karplus&title=Glycogen+phosphorylase+inhibitors%3A+a+free+energy+perturbation+analysis+of+glucopyranose+spirohydantoin+analogues&doi=10.1002%2Fprot.20641"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit95&amp;dbid=16384&amp;doi=10.1002%2Fprot.20641&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.20641%26sid%3Dliteratum%253Aachs%26aulast%3DArchontis%26aufirst%3DG.%26aulast%3DWatson%26aufirst%3DK.%2BA.%26aulast%3DXie%26aufirst%3DQ.%26aulast%3DAndreou%26aufirst%3DG.%26aulast%3DChrysina%26aufirst%3DE.%2BD.%26aulast%3DZographos%26aufirst%3DS.%2BE.%26aulast%3DOikonomakos%26aufirst%3DN.%2BG.%26aulast%3DKarplus%26aufirst%3DM.%26atitle%3DGlycogen%2520phosphorylase%2520inhibitors%253A%2520a%2520free%2520energy%2520perturbation%2520analysis%2520of%2520glucopyranose%2520spirohydantoin%2520analogues%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D2005%26volume%3D61%26spage%3D984%26epage%3D998%26doi%3D10.1002%2Fprot.20641" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref96"><div class="reference"><strong class="refLabel"><a href="#ref96" class="rightTabRefNumLink">96</a></strong><div class="NLM_citation" id="rightTab-cit96"><span><span class="NLM_contrib-group">Steinbrecher, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hrenn, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dormann, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Merfort, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Labahn, A.</span><span> </span><span class="NLM_article-title">Bornyl (3, 4, 5-trihydroxy)-cinnamate-an optimized human neutrophil elastase inhibitor designed by free energy calculations</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">2385</span><span class="NLM_x">–</span> <span class="NLM_lpage">2390</span><span class="refDoi"> DOI: 10.1016/j.bmc.2007.11.070</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1016%2Fj.bmc.2007.11.070" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2008&pages=2385-2390&author=T.+Steinbrecherauthor=A.+Hrennauthor=K.+L.+Dormannauthor=I.+Merfortauthor=A.+Labahn&title=Bornyl+%283%2C+4%2C+5-trihydroxy%29-cinnamate-an+optimized+human+neutrophil+elastase+inhibitor+designed+by+free+energy+calculations&doi=10.1016%2Fj.bmc.2007.11.070"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit96&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2007.11.070&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2007.11.070%26sid%3Dliteratum%253Aachs%26aulast%3DSteinbrecher%26aufirst%3DT.%26aulast%3DHrenn%26aufirst%3DA.%26aulast%3DDormann%26aufirst%3DK.%2BL.%26aulast%3DMerfort%26aufirst%3DI.%26aulast%3DLabahn%26aufirst%3DA.%26atitle%3DBornyl%2520%25283%252C%25204%252C%25205-trihydroxy%2529-cinnamate-an%2520optimized%2520human%2520neutrophil%2520elastase%2520inhibitor%2520designed%2520by%2520free%2520energy%2520calculations%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2008%26volume%3D16%26spage%3D2385%26epage%3D2390%26doi%3D10.1016%2Fj.bmc.2007.11.070" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref97"><div class="reference"><strong class="refLabel"><a href="#ref97" class="rightTabRefNumLink">97</a></strong><div class="NLM_citation" id="rightTab-cit97"><span><span class="NLM_contrib-group">Rogers, I. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gammon, D. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Naidoo, K. J.</span><span> </span><span class="NLM_article-title">Conformational preferences of plumbagin with phenyl-1-thioglucoside conjugates in solution and bound to MshB determined by aromatic association</span> <span class="citation_source-journal">Carbohydr. Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">371</span><span class="NLM_x">, </span> <span class="NLM_fpage">52</span><span class="NLM_x">–</span> <span class="NLM_lpage">60</span><span class="refDoi"> DOI: 10.1016/j.carres.2013.02.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1016%2Fj.carres.2013.02.001" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=371&publication_year=2013&pages=52-60&author=I.+L.+Rogersauthor=D.+W.+Gammonauthor=K.+J.+Naidoo&title=Conformational+preferences+of+plumbagin+with+phenyl-1-thioglucoside+conjugates+in+solution+and+bound+to+MshB+determined+by+aromatic+association&doi=10.1016%2Fj.carres.2013.02.001"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit97&amp;dbid=16384&amp;doi=10.1016%2Fj.carres.2013.02.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.carres.2013.02.001%26sid%3Dliteratum%253Aachs%26aulast%3DRogers%26aufirst%3DI.%2BL.%26aulast%3DGammon%26aufirst%3DD.%2BW.%26aulast%3DNaidoo%26aufirst%3DK.%2BJ.%26atitle%3DConformational%2520preferences%2520of%2520plumbagin%2520with%2520phenyl-1-thioglucoside%2520conjugates%2520in%2520solution%2520and%2520bound%2520to%2520MshB%2520determined%2520by%2520aromatic%2520association%26jtitle%3DCarbohydr.%2520Res.%26date%3D2013%26volume%3D371%26spage%3D52%26epage%3D60%26doi%3D10.1016%2Fj.carres.2013.02.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref98"><div class="reference"><strong class="refLabel"><a href="#ref98" class="rightTabRefNumLink">98</a></strong><div class="NLM_citation" id="rightTab-cit98"><span><span class="NLM_contrib-group">Lee, H. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hsu, W. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hsu, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, Y. C.</span><span> </span><span class="NLM_article-title">Using thermodynamic integration MD simulation to compute relative protein-ligand binding free energy of a GSK3beta kinase inhibitor and its analogs</span> <span class="citation_source-journal">J. Mol. Graphics Modell.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">37</span><span class="NLM_x">–</span> <span class="NLM_lpage">49</span><span class="refDoi"> DOI: 10.1016/j.jmgm.2014.04.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1016%2Fj.jmgm.2014.04.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=24858254" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtVCisbvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2014&pages=37-49&author=H.+C.+Leeauthor=W.+C.+Hsuauthor=A.+L.+Liuauthor=C.+J.+Hsuauthor=Y.+C.+Sun&title=Using+thermodynamic+integration+MD+simulation+to+compute+relative+protein-ligand+binding+free+energy+of+a+GSK3beta+kinase+inhibitor+and+its+analogs&doi=10.1016%2Fj.jmgm.2014.04.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit98R"><div class="casContent"><span class="casTitleNuber">98</span><div class="casTitle"><span class="NLM_cas:atitle">Using thermodynamic integration MD simulation to compute relative protein-ligand binding free energy of a GSK3β kinase inhibitor and its analogs</span></div><div class="casAuthors">Lee, Hsing-Chou; Hsu, Wen-Chi; Liu, An-Lun; Hsu, Chia-Jen; Sun, Ying-Chieh</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Graphics & Modelling</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">37-49</span>CODEN:
                <span class="NLM_cas:coden">JMGMFI</span>;
        ISSN:<span class="NLM_cas:issn">1093-3263</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Thermodn. integration mol. dynamics simulation was used to investigate how TI-MD simulation preforms in reproducing relative protein-ligand binding free energy of a pair of analogous GSK3β kinase inhibitors of available exptl. data (see Fig. 1), and to predict the affinity for other analogs.  The computation for the pair gave a ΔΔG of 1.0 kcal/mol, which was in reasonably good agreement with the exptl. value of -0.1 kcal/mol.  The error bar was estd. at 0.5 kcal/mol.  Subsequently, we employed the same protocol to proceed with simulations to find analogous inhibitors with a stronger affinity.  Four analogs with a substitution at one site inside the binding pocket were the first to be tried, but no significant enhancement in affinity was found.  Subsequent simulations for another 7 analogs was focused on substitutions at the benzene ring of another site, which gave two analogs (analogs 9 and 10) with ΔΔG values of -0.6 and -0.8 kcal/mol, resp.  Both analogs had a -OH group at the meta position and another -OH group at the ortho position at the other side of the benzene ring, as shown in Table 3.  To explore further, another 4 analogs with this characteristic were investigated.  Three analogs with ΔΔG values of -2.2, -1.7 and -1.2 kcal/mol, resp., were found.  Hydrogen bond anal. suggested that the addnl. hydrogen bonds of the added -OH groups with Gln185 and/or Asn64, which did not appear in the ref. inhibitor or as an analog with one substitution only in the examd. cases, were the main contributors to an enhanced affinity.  A prediction for better inhibitors should interest experimentalists of enzyme and/or cell assays.  Anal. of the interactions between GSK3β kinase and the investigated analogs will be useful in the design of GSK3β kinase inhibitors for compds. of this class.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxtb6DM4P2-LVg90H21EOLACvtfcHk0ljoDBE_dSzg3w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtVCisbvI&md5=6fea8120488533812c0544f26bc0e1b9</span></div><a href="/servlet/linkout?suffix=cit98&amp;dbid=16384&amp;doi=10.1016%2Fj.jmgm.2014.04.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmgm.2014.04.010%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DH.%2BC.%26aulast%3DHsu%26aufirst%3DW.%2BC.%26aulast%3DLiu%26aufirst%3DA.%2BL.%26aulast%3DHsu%26aufirst%3DC.%2BJ.%26aulast%3DSun%26aufirst%3DY.%2BC.%26atitle%3DUsing%2520thermodynamic%2520integration%2520MD%2520simulation%2520to%2520compute%2520relative%2520protein-ligand%2520binding%2520free%2520energy%2520of%2520a%2520GSK3beta%2520kinase%2520inhibitor%2520and%2520its%2520analogs%26jtitle%3DJ.%2520Mol.%2520Graphics%2520Modell.%26date%3D2014%26volume%3D51%26spage%3D37%26epage%3D49%26doi%3D10.1016%2Fj.jmgm.2014.04.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref99"><div class="reference"><strong class="refLabel"><a href="#ref99" class="rightTabRefNumLink">99</a></strong><div class="NLM_citation" id="rightTab-cit99"><span><span class="NLM_contrib-group">Steinbrecher, T. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dahlgren, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cappel, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krilov, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abel, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friesner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sherman, W.</span><span> </span><span class="NLM_article-title">Accurate binding free energy predictions in fragment optimization</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">2411</span><span class="NLM_x">–</span> <span class="NLM_lpage">2420</span><span class="refDoi"> DOI: 10.1021/acs.jcim.5b00538</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.5b00538" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs1eku7%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2015&pages=2411-2420&author=T.+B.+Steinbrecherauthor=M.+Dahlgrenauthor=D.+Cappelauthor=T.+Linauthor=L.+Wangauthor=G.+Krilovauthor=R.+Abelauthor=R.+Friesnerauthor=W.+Sherman&title=Accurate+binding+free+energy+predictions+in+fragment+optimization&doi=10.1021%2Facs.jcim.5b00538"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit99R"><div class="casContent"><span class="casTitleNuber">99</span><div class="casTitle"><span class="NLM_cas:atitle">Accurate Binding Free Energy Predictions in Fragment Optimization</span></div><div class="casAuthors">Steinbrecher, Thomas B.; Dahlgren, Markus; Cappel, Daniel; Lin, Teng; Wang, Lingle; Krilov, Goran; Abel, Robert; Friesner, Richard; Sherman, Woody</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2411-2420</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Predicting protein-ligand binding free energies is a central aim of computational structure-based drug design (SBDD) - improved accuracy in binding free energy predictions could significantly reduce costs and accelerate project timelines in lead discovery and optimization.  The recent development and validation of advanced free energy calcn. methods represents a major step toward this goal.  Accurately predicting the relative binding free energy changes of modifications to ligands is esp. valuable in the field of fragment-based drug design, since fragment screens tend to deliver initial hits of low binding affinity that require multiple rounds of synthesis to gain the requisite potency for a project.  A free energy perturbation protocol, FEP+, which was previously validated on drug-like lead compds., is suitable for the calcn. of relative binding strengths of fragment-sized compds. as well.  The authors study several pharmaceutically relevant targets with a total of >90 fragments and find that the FEP+ methodol., which uses explicit solvent mol. dynamics and physics-based scoring with no parameters adjusted, can accurately predict relative fragment binding affinities.  The calcns. afford R2-values on av. >0.5 compared to exptl. data and root-mean-square errors of ∼1.1 kcal/mol overall, demonstrating significant improvements over the docking and MM-GBSA methods tested in this work and indicating that FEP+ has the requisite predictive power to impact fragment-based affinity optimization projects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2OM1KfoQE4bVg90H21EOLACvtfcHk0li5CONc4IFifQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs1eku7%252FI&md5=116425dcc073d2907c888555e9061aa8</span></div><a href="/servlet/linkout?suffix=cit99&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.5b00538&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.5b00538%26sid%3Dliteratum%253Aachs%26aulast%3DSteinbrecher%26aufirst%3DT.%2BB.%26aulast%3DDahlgren%26aufirst%3DM.%26aulast%3DCappel%26aufirst%3DD.%26aulast%3DLin%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DKrilov%26aufirst%3DG.%26aulast%3DAbel%26aufirst%3DR.%26aulast%3DFriesner%26aufirst%3DR.%26aulast%3DSherman%26aufirst%3DW.%26atitle%3DAccurate%2520binding%2520free%2520energy%2520predictions%2520in%2520fragment%2520optimization%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2015%26volume%3D55%26spage%3D2411%26epage%3D2420%26doi%3D10.1021%2Facs.jcim.5b00538" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref100"><div class="reference"><strong class="refLabel"><a href="#ref100" class="rightTabRefNumLink">100</a></strong><div class="NLM_citation" id="rightTab-cit100"><span><span class="NLM_contrib-group">Aldeghi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heifetz, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bodkin, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Biggin, P. C.</span><span> </span><span class="NLM_article-title">Accurate calculation of the absolute free energy of binding for drug molecules</span> <span class="citation_source-journal">Chem. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">207</span><span class="NLM_x">–</span> <span class="NLM_lpage">218</span><span class="refDoi"> DOI: 10.1039/C5SC02678D</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1039%2FC5SC02678D" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=26798447" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFGku7%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=207-218&author=M.+Aldeghiauthor=A.+Heifetzauthor=M.+J.+Bodkinauthor=S.+Knappauthor=P.+C.+Biggin&title=Accurate+calculation+of+the+absolute+free+energy+of+binding+for+drug+molecules&doi=10.1039%2FC5SC02678D"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit100R"><div class="casContent"><span class="casTitleNuber">100</span><div class="casTitle"><span class="NLM_cas:atitle">Accurate calculation of the absolute free energy of binding for drug molecules</span></div><div class="casAuthors">Aldeghi, Matteo; Heifetz, Alexander; Bodkin, Michael J.; Knapp, Stefan; Biggin, Philip C.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Science</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">207-218</span>CODEN:
                <span class="NLM_cas:coden">CSHCCN</span>;
        ISSN:<span class="NLM_cas:issn">2041-6520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Accurate prediction of binding affinities has been a central goal of computational chem. for decades, yet remains elusive.  Despite good progress, the required accuracy for use in a drug-discovery context has not been consistently achieved for drug-like mols.  Here, we perform abs. free energy calcns. based on a thermodn. cycle for a set of diverse inhibitors binding to bromodomain-contg. protein 4 (BRD4) and demonstrate that a mean abs. error of 0.6 kcal mol-1 can be achieved.  We also show a similar level of accuracy (1.0 kcal mol-1) can be achieved in pseudo prospective approach.  Bromodomains are epigenetic mark readers that recognize acetylation motifs and regulate gene transcription, and are currently being investigated as therapeutic targets for cancer and inflammation.  The unprecedented accuracy offers the exciting prospect that the binding free energy of drug-like compds. can be predicted for pharmacol. relevant targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrrNjJ8SvAg3rVg90H21EOLACvtfcHk0li5CONc4IFifQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFGku7%252FI&md5=d00cac3b87f1bd11837d6f292bd8c6e5</span></div><a href="/servlet/linkout?suffix=cit100&amp;dbid=16384&amp;doi=10.1039%2FC5SC02678D&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC5SC02678D%26sid%3Dliteratum%253Aachs%26aulast%3DAldeghi%26aufirst%3DM.%26aulast%3DHeifetz%26aufirst%3DA.%26aulast%3DBodkin%26aufirst%3DM.%2BJ.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBiggin%26aufirst%3DP.%2BC.%26atitle%3DAccurate%2520calculation%2520of%2520the%2520absolute%2520free%2520energy%2520of%2520binding%2520for%2520drug%2520molecules%26jtitle%3DChem.%2520Sci.%26date%3D2016%26volume%3D7%26spage%3D207%26epage%3D218%26doi%3D10.1039%2FC5SC02678D" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref101"><div class="reference"><strong class="refLabel"><a href="#ref101" class="rightTabRefNumLink">101</a></strong><div class="NLM_citation" id="rightTab-cit101"><span><span class="NLM_contrib-group">Ciordia, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perez-Benito, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Delgado, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trabanco, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tresadern, G.</span><span> </span><span class="NLM_article-title">Application of free energy perturbation for the design of BACE1 inhibitors</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">1856</span><span class="NLM_x">–</span> <span class="NLM_lpage">1871</span><span class="refDoi"> DOI: 10.1021/acs.jcim.6b00220</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.6b00220" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht12mu7%252FK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2016&pages=1856-1871&author=M.+Ciordiaauthor=L.+Perez-Benitoauthor=F.+Delgadoauthor=A.+A.+Trabancoauthor=G.+Tresadern&title=Application+of+free+energy+perturbation+for+the+design+of+BACE1+inhibitors&doi=10.1021%2Facs.jcim.6b00220"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit101R"><div class="casContent"><span class="casTitleNuber">101</span><div class="casTitle"><span class="NLM_cas:atitle">Application of Free Energy Perturbation for the Design of BACE1 Inhibitors</span></div><div class="casAuthors">Ciordia, Myriam; Perez-Benito, Laura; Delgado, Francisca; Trabanco, Andres A.; Tresadern, Gary</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1856-1871</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Novel spiroaminodihydropyrroles probing for optimized interactions at the P3 pocket of β-secretase 1 (BACE1) were designed with the use of free energy perturbation (FEP) calcns.  The resulting mols. showed pIC50 potencies in enzymic BACE1 inhibition assays ranging from approx. 5 to 7.  Good correlation was obsd. between the predicted activity from the FEP calcns. and exptl. activity.  Simulations run with a default 5 ns approach delivered a mean unsigned error (MUE) between prediction and expt. of 0.58 and 0.91 kcal/mol for retrospective and prospective applications, resp.  With longer simulations of 10 and 20 ns, the MUE was in both cases 0.57 kcal/mol for the retrospective application, and 0.69 and 0.59 kcal/mol for the prospective application.  Other considerations that impact the quality of the calcns. are discussed.  This work provides an example of the value of FEP as a computational tool for drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpqpi-OiII-FrVg90H21EOLACvtfcHk0li5CONc4IFifQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht12mu7%252FK&md5=b7577a89492867fc334cb1ec937a4eb6</span></div><a href="/servlet/linkout?suffix=cit101&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.6b00220&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.6b00220%26sid%3Dliteratum%253Aachs%26aulast%3DCiordia%26aufirst%3DM.%26aulast%3DPerez-Benito%26aufirst%3DL.%26aulast%3DDelgado%26aufirst%3DF.%26aulast%3DTrabanco%26aufirst%3DA.%2BA.%26aulast%3DTresadern%26aufirst%3DG.%26atitle%3DApplication%2520of%2520free%2520energy%2520perturbation%2520for%2520the%2520design%2520of%2520BACE1%2520inhibitors%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2016%26volume%3D56%26spage%3D1856%26epage%3D1871%26doi%3D10.1021%2Facs.jcim.6b00220" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref102"><div class="reference"><strong class="refLabel"><a href="#ref102" class="rightTabRefNumLink">102</a></strong><div class="NLM_citation" id="rightTab-cit102"><span><span class="NLM_contrib-group">Lenselink, E. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Louvel, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Forti, A. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Veldhoven, J. P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Vries, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mulder-Krieger, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McRobb, F. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Negri, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goose, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abel, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Vlijmen, H. W. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, L. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harder, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sherman, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">IJzerman, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beuming, T.</span><span> </span><span class="NLM_article-title">Predicting binding bffinities for GPCR ligands using free-energy perturbation</span> <span class="citation_source-journal">ACS Omega</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">293</span><span class="NLM_x">–</span> <span class="NLM_lpage">304</span><span class="refDoi"> DOI: 10.1021/acsomega.6b00086</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsomega.6b00086" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2016&pages=293-304&author=E.+B.+Lenselinkauthor=J.+Louvelauthor=A.+F.+Fortiauthor=J.+P.+D.+van+Veldhovenauthor=H.+de+Vriesauthor=T.+Mulder-Kriegerauthor=F.+M.+McRobbauthor=A.+Negriauthor=J.+Gooseauthor=R.+Abelauthor=H.+W.+T.+van+Vlijmenauthor=L.+L.+Wangauthor=E.+Harderauthor=W.+Shermanauthor=A.+P.+IJzermanauthor=T.+Beuming&title=Predicting+binding+bffinities+for+GPCR+ligands+using+free-energy+perturbation&doi=10.1021%2Facsomega.6b00086"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit102&amp;dbid=16384&amp;doi=10.1021%2Facsomega.6b00086&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsomega.6b00086%26sid%3Dliteratum%253Aachs%26aulast%3DLenselink%26aufirst%3DE.%2BB.%26aulast%3DLouvel%26aufirst%3DJ.%26aulast%3DForti%26aufirst%3DA.%2BF.%26aulast%3Dvan%2BVeldhoven%26aufirst%3DJ.%2BP.%2BD.%26aulast%3Dde%2BVries%26aufirst%3DH.%26aulast%3DMulder-Krieger%26aufirst%3DT.%26aulast%3DMcRobb%26aufirst%3DF.%2BM.%26aulast%3DNegri%26aufirst%3DA.%26aulast%3DGoose%26aufirst%3DJ.%26aulast%3DAbel%26aufirst%3DR.%26aulast%3Dvan%2BVlijmen%26aufirst%3DH.%2BW.%2BT.%26aulast%3DWang%26aufirst%3DL.%2BL.%26aulast%3DHarder%26aufirst%3DE.%26aulast%3DSherman%26aufirst%3DW.%26aulast%3DIJzerman%26aufirst%3DA.%2BP.%26aulast%3DBeuming%26aufirst%3DT.%26atitle%3DPredicting%2520binding%2520bffinities%2520for%2520GPCR%2520ligands%2520using%2520free-energy%2520perturbation%26jtitle%3DACS%2520Omega%26date%3D2016%26volume%3D1%26spage%3D293%26epage%3D304%26doi%3D10.1021%2Facsomega.6b00086" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref103"><div class="reference"><strong class="refLabel"><a href="#ref103" class="rightTabRefNumLink">103</a></strong><div class="NLM_citation" id="rightTab-cit103"><span><span class="NLM_contrib-group">Ioannidis, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Drakopoulos, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tzitzoglaki, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Homeyer, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kolarov, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gkeka, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Freudenberger, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liolios, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gauglitz, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cournia, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gohlke, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kolocouris, A.</span><span> </span><span class="NLM_article-title">Alchemical free energy calculations and isothermal titration calorimetry measurements of aminoadamantanes bound to the closed state of influenza A/M2TM</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">862</span><span class="NLM_x">–</span> <span class="NLM_lpage">876</span><span class="refDoi"> DOI: 10.1021/acs.jcim.6b00079</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.6b00079" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC28XmsFOmtrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2016&pages=862-876&author=H.+Ioannidisauthor=A.+Drakopoulosauthor=C.+Tzitzoglakiauthor=N.+Homeyerauthor=F.+Kolarovauthor=P.+Gkekaauthor=K.+Freudenbergerauthor=C.+Lioliosauthor=G.+Gauglitzauthor=Z.+Courniaauthor=H.+Gohlkeauthor=A.+Kolocouris&title=Alchemical+free+energy+calculations+and+isothermal+titration+calorimetry+measurements+of+aminoadamantanes+bound+to+the+closed+state+of+influenza+A%2FM2TM&doi=10.1021%2Facs.jcim.6b00079"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit103R"><div class="casContent"><span class="casTitleNuber">103</span><div class="casTitle"><span class="NLM_cas:atitle">Alchemical Free Energy Calculations and Isothermal Titration Calorimetry Measurements of Aminoadamantanes Bound to the Closed State of Influenza A/M2TM</span></div><div class="casAuthors">Ioannidis, Harris; Drakopoulos, Antonios; Tzitzoglaki, Christina; Homeyer, Nadine; Kolarov, Felix; Gkeka, Paraskevi; Freudenberger, Kathrin; Liolios, Christos; Gauglitz, Gunter; Cournia, Zoe; Gohlke, Holger; Kolocouris, Antonios</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">862-876</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Adamantane derivs., such as amantadine and rimantadine, have been reported to block the transmembrane domain (TM) of the M2 protein of influenza A virus (A/M2) but their clin. use has been discontinued due to evolved resistance in humans.  Although expts. and simulations have provided adequate information about the binding interaction of amantadine or rimantadine to the M2 protein, methods for predicting binding affinities of whole series of M2 inhibitors have so far been scarcely applied.  Such methods could assist in the development of novel potent inhibitors that overcome A/M2 resistance.  Here we show that alchem. free energy calcns. of ligand binding using the Bennett acceptance ratio (BAR) method are valuable for detg. the relative binding potency of A/M2 inhibitors of the aminoadamantane type covering a binding affinity range of only ∼2 kcal mol-1.  Their binding affinities measured by isothermal titrn. calorimetry (ITC) against the A/M2TM tetramer from the Udorn strain in its closed form at pH 8 were used as exptl. probes.  The binding consts. of rimantadine enantiomers against M2TMUdorn were measured for the first time and found to be equal.  Two series of alchem. free energy calcns. were performed using 1,2-dipalmitoyl-sn-glycero-3-phosphocholine (DPPC) and 1,2-dimyristoyl-sn-glycero-3-phosphocholine (DMPC) lipids to mimic the membrane environment.  A fair correlation was found for DPPC that was significantly improved using DMPC, which resembles more closely the DPC lipids used in the ITC expts.  This demonstrates that binding free energy calcns. by the BAR approach can be used to predict relative binding affinities of aminoadamantane derivs. toward M2TM with good accuracy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYKIuWLX1aP7Vg90H21EOLACvtfcHk0lhMw-ToUgqB-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XmsFOmtrg%253D&md5=c9ad995dcf016bc19c81fa4753827f5f</span></div><a href="/servlet/linkout?suffix=cit103&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.6b00079&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.6b00079%26sid%3Dliteratum%253Aachs%26aulast%3DIoannidis%26aufirst%3DH.%26aulast%3DDrakopoulos%26aufirst%3DA.%26aulast%3DTzitzoglaki%26aufirst%3DC.%26aulast%3DHomeyer%26aufirst%3DN.%26aulast%3DKolarov%26aufirst%3DF.%26aulast%3DGkeka%26aufirst%3DP.%26aulast%3DFreudenberger%26aufirst%3DK.%26aulast%3DLiolios%26aufirst%3DC.%26aulast%3DGauglitz%26aufirst%3DG.%26aulast%3DCournia%26aufirst%3DZ.%26aulast%3DGohlke%26aufirst%3DH.%26aulast%3DKolocouris%26aufirst%3DA.%26atitle%3DAlchemical%2520free%2520energy%2520calculations%2520and%2520isothermal%2520titration%2520calorimetry%2520measurements%2520of%2520aminoadamantanes%2520bound%2520to%2520the%2520closed%2520state%2520of%2520influenza%2520A%252FM2TM%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2016%26volume%3D56%26spage%3D862%26epage%3D876%26doi%3D10.1021%2Facs.jcim.6b00079" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref104"><div class="reference"><strong class="refLabel"><a href="#ref104" class="rightTabRefNumLink">104</a></strong><div class="NLM_citation" id="rightTab-cit104"><span><span class="NLM_contrib-group">Bhati, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wright, D. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coveney, P. V.</span><span> </span><span class="NLM_article-title">Rapid, accurate, precise, and reliable relative free energy prediction using ensemble based thermodynamic integration</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">210</span><span class="NLM_x">–</span> <span class="NLM_lpage">222</span><span class="refDoi"> DOI: 10.1021/acs.jctc.6b00979</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jctc.6b00979" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref104/cit104&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC28XitVGntrrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&pages=210-222&author=A.+P.+Bhatiauthor=S.+Wanauthor=D.+W.+Wrightauthor=P.+V.+Coveney&title=Rapid%2C+accurate%2C+precise%2C+and+reliable+relative+free+energy+prediction+using+ensemble+based+thermodynamic+integration&doi=10.1021%2Facs.jctc.6b00979"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit104R"><div class="casContent"><span class="casTitleNuber">104</span><div class="casTitle"><span class="NLM_cas:atitle">Rapid, Accurate, Precise, and Reliable Relative Free Energy Prediction Using Ensemble Based Thermodynamic Integration</span></div><div class="casAuthors">Bhati, Agastya P.; Wan, Shunzhou; Wright, David W.; Coveney, Peter V.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">210-222</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The accurate prediction of the binding affinities of ligands to proteins is a major goal in drug discovery and personalized medicine.  The time taken to make such predictions is of similar importance to their accuracy, precision and reliability.  In the last few years, an ensemble based mol. dynamics approach has been proposed that provides a route to reliable predictions of free energies based on the mol. mechanics Poisson-Boltzmann surface area method which meets the requirements of accuracy, precision and reliability.  Here, we describe an equiv. methodol. based on thermodn. integration to substantially improve the accuracy, precision and reliability of calcd. relative binding free energies.  We report the performance of the method when applied to a diverse set of protein targets and ligands.  The results are in very good agreement with exptl. data (90% of calcns. agree to within 1 kcal/mol) while the method is reproducible by construction.  Statistical uncertainties of the order of 0.5 kcal/mol or less are achieved.  We present a systematic account of how the uncertainty in the predictions may be estd.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhu9wYHCJpUbVg90H21EOLACvtfcHk0lhMw-ToUgqB-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitVGntrrJ&md5=510b70188112a3578030e291ce19b127</span></div><a href="/servlet/linkout?suffix=cit104&amp;dbid=16384&amp;doi=10.1021%2Facs.jctc.6b00979&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jctc.6b00979%26sid%3Dliteratum%253Aachs%26aulast%3DBhati%26aufirst%3DA.%2BP.%26aulast%3DWan%26aufirst%3DS.%26aulast%3DWright%26aufirst%3DD.%2BW.%26aulast%3DCoveney%26aufirst%3DP.%2BV.%26atitle%3DRapid%252C%2520accurate%252C%2520precise%252C%2520and%2520reliable%2520relative%2520free%2520energy%2520prediction%2520using%2520ensemble%2520based%2520thermodynamic%2520integration%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2017%26volume%3D13%26spage%3D210%26epage%3D222%26doi%3D10.1021%2Facs.jctc.6b00979" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref105"><div class="reference"><strong class="refLabel"><a href="#ref105" class="rightTabRefNumLink">105</a></strong><div class="NLM_citation" id="rightTab-cit105"><span><span class="NLM_contrib-group">Kiss, L. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soares-da-Silva, P.</span><span> </span><span class="NLM_article-title">Medicinal chemistry of catechol O-methyltransferase (COMT) inhibitors and their therapeutic utility</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">8692</span><span class="NLM_x">–</span> <span class="NLM_lpage">8717</span><span class="refDoi"> DOI: 10.1021/jm500572b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm500572b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref105/cit105&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht1Git7rI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=8692-8717&author=L.+E.+Kissauthor=P.+Soares-da-Silva&title=Medicinal+chemistry+of+catechol+O-methyltransferase+%28COMT%29+inhibitors+and+their+therapeutic+utility&doi=10.1021%2Fjm500572b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit105R"><div class="casContent"><span class="casTitleNuber">105</span><div class="casTitle"><span class="NLM_cas:atitle">Medicinal Chemistry of Catechol O-Methyltransferase (COMT) Inhibitors and Their Therapeutic Utility</span></div><div class="casAuthors">Kiss, Laszlo E.; Soares-da-Silva, Patricio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">8692-8717</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Catechol O-methyltransferase (COMT) is the enzyme responsible for the O-methylation of endogenous neurotransmitters and of xenobiotic substances and hormones incorporating catecholic structures.  COMT is a druggable biol. target for the treatment of various central and peripheral nervous system disorders, including Parkinson's disease, depression, schizophrenia, and other dopamine deficiency-related diseases.  The purpose of this perspective is fourfold: (i) to summarize the physiol. role of COMT inhibitors in central and peripheral nervous system disorders; (ii) to provide the history and perspective of the medicinal chem. behind the discovery and development of COMT inhibitors; (iii) to discuss how the physicochem. properties of recognized COMT inhibitors are understood to exert influence over their pharmacol. properties; and (iv) to evaluate the clin. benefits of the most relevant COMT inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_JOcVUbOM4LVg90H21EOLACvtfcHk0lhvTziScyHrFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht1Git7rI&md5=60ddd0ce0b752ddb1bd8898805a8935b</span></div><a href="/servlet/linkout?suffix=cit105&amp;dbid=16384&amp;doi=10.1021%2Fjm500572b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm500572b%26sid%3Dliteratum%253Aachs%26aulast%3DKiss%26aufirst%3DL.%2BE.%26aulast%3DSoares-da-Silva%26aufirst%3DP.%26atitle%3DMedicinal%2520chemistry%2520of%2520catechol%2520O-methyltransferase%2520%2528COMT%2529%2520inhibitors%2520and%2520their%2520therapeutic%2520utility%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D8692%26epage%3D8717%26doi%3D10.1021%2Fjm500572b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref106"><div class="reference"><strong class="refLabel"><a href="#ref106" class="rightTabRefNumLink">106</a></strong><div class="NLM_citation" id="rightTab-cit106"><span><span class="NLM_contrib-group">Loeffler, H. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Michel, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woods, C.</span><span> </span><span class="NLM_article-title">FESetup: automating setup for alchemical free energy simulations</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">2485</span><span class="NLM_x">–</span> <span class="NLM_lpage">2490</span><span class="refDoi"> DOI: 10.1021/acs.jcim.5b00368</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.5b00368" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref106/cit106&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslyktb7O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2015&pages=2485-2490&author=H.+H.+Loefflerauthor=J.+Michelauthor=C.+Woods&title=FESetup%3A+automating+setup+for+alchemical+free+energy+simulations&doi=10.1021%2Facs.jcim.5b00368"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit106R"><div class="casContent"><span class="casTitleNuber">106</span><div class="casTitle"><span class="NLM_cas:atitle">FESetup: Automating Setup for Alchemical Free Energy Simulations</span></div><div class="casAuthors">Loeffler, Hannes H.; Michel, Julien; Woods, Christopher</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2485-2490</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">FESetup is a new pipeline tool which can be used flexibly within larger workflows.  The tool aims to support fast and easy setup of alchem. free energy simulations for mol. simulation packages such as AMBER, GROMACS, Sire, or NAMD.  Post-processing methods like MM-PBSA and LIE can be set up as well.  Ligands are automatically parametrized with AM1-BCC, and atom mappings for a single topol. description are computed with a max. common substructure search (MCSS) algorithm.  An abstr. mol. dynamics (MD) engine can be used for equilibration prior to free energy setup or standalone.  Currently, all modern AMBER force fields are supported.  Ease of use, robustness of the code, and automation where it is feasible are the main development goals.  The project follows an open development model, and we welcome contributions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpaHPd8afIINLVg90H21EOLACvtfcHk0lhvTziScyHrFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslyktb7O&md5=14de057bac2999778efed9b040b3368b</span></div><a href="/servlet/linkout?suffix=cit106&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.5b00368&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.5b00368%26sid%3Dliteratum%253Aachs%26aulast%3DLoeffler%26aufirst%3DH.%2BH.%26aulast%3DMichel%26aufirst%3DJ.%26aulast%3DWoods%26aufirst%3DC.%26atitle%3DFESetup%253A%2520automating%2520setup%2520for%2520alchemical%2520free%2520energy%2520simulations%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2015%26volume%3D55%26spage%3D2485%26epage%3D2490%26doi%3D10.1021%2Facs.jcim.5b00368" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref107"><div class="reference"><strong class="refLabel"><a href="#ref107" class="rightTabRefNumLink">107</a></strong><div class="NLM_citation" id="rightTab-cit107"><span><span class="NLM_contrib-group">Liu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abel, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Redmann, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Summa, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jaber, V. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lim, N. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mobley, D. L.</span><span> </span><span class="NLM_article-title">Lead optimization mapper: automating free energy calculations for lead optimization</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">27</span><span class="NLM_x">, </span> <span class="NLM_fpage">755</span><span class="NLM_x">–</span> <span class="NLM_lpage">770</span><span class="refDoi"> DOI: 10.1007/s10822-013-9678-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1007%2Fs10822-013-9678-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=24072356" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsFaqtLbP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2013&pages=755-770&author=S.+Liuauthor=Y.+Wuauthor=T.+Linauthor=R.+Abelauthor=J.+P.+Redmannauthor=C.+M.+Summaauthor=V.+R.+Jaberauthor=N.+M.+Limauthor=D.+L.+Mobley&title=Lead+optimization+mapper%3A+automating+free+energy+calculations+for+lead+optimization&doi=10.1007%2Fs10822-013-9678-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit107R"><div class="casContent"><span class="casTitleNuber">107</span><div class="casTitle"><span class="NLM_cas:atitle">Lead optimization mapper: automating free energy calculations for lead optimization</span></div><div class="casAuthors">Liu, Shuai; Wu, Yujie; Lin, Teng; Abel, Robert; Redmann, Jonathan P.; Summa, Christopher M.; Jaber, Vivian R.; Lim, Nathan M.; Mobley, David L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">755-770</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Alchem. free energy calcns. hold increasing promise as an aid to drug discovery efforts.  However, applications of these techniques in discovery projects have been relatively few, partly because of the difficulty of planning and setting up calcns.  Here, we introduce lead optimization mapper, LOMAP, an automated algorithm to plan efficient relative free energy calcns. between potential ligands within a substantial library of perhaps hundreds of compds.  In this approach, ligands are first grouped by structural similarity primarily based on the size of a (loosely defined) maximal common substructure, and then calcns. are planned within and between sets of structurally related compds.  An emphasis is placed on ensuring that relative free energies can be obtained between any pair of compds. without combining the results of too many different relative free energy calcns. (to avoid accumulation of error) and by providing some redundancy to allow for the possibility of error and consistency checking and provide some insight into when results can be expected to be unreliable.  The algorithm is discussed in detail and a Python implementation, based on both Schroedinger's and OpenEye's APIs, has been made available freely under the BSD license.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGruuGmSu4sZV7Vg90H21EOLACvtfcHk0lhvTziScyHrFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsFaqtLbP&md5=ab9ba58e2572aa3564a52a04352f70d7</span></div><a href="/servlet/linkout?suffix=cit107&amp;dbid=16384&amp;doi=10.1007%2Fs10822-013-9678-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-013-9678-y%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DLin%26aufirst%3DT.%26aulast%3DAbel%26aufirst%3DR.%26aulast%3DRedmann%26aufirst%3DJ.%2BP.%26aulast%3DSumma%26aufirst%3DC.%2BM.%26aulast%3DJaber%26aufirst%3DV.%2BR.%26aulast%3DLim%26aufirst%3DN.%2BM.%26aulast%3DMobley%26aufirst%3DD.%2BL.%26atitle%3DLead%2520optimization%2520mapper%253A%2520automating%2520free%2520energy%2520calculations%2520for%2520lead%2520optimization%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2013%26volume%3D27%26spage%3D755%26epage%3D770%26doi%3D10.1007%2Fs10822-013-9678-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref108"><div class="reference"><strong class="refLabel"><a href="#ref108" class="rightTabRefNumLink">108</a></strong><div class="NLM_citation" id="rightTab-cit108"><span><span class="NLM_contrib-group">Filippakopoulos, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span> </span><span class="NLM_article-title">Targeting bromodomains: epigenetic readers of lysine acetylation</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">337</span><span class="NLM_x">–</span> <span class="NLM_lpage">356</span><span class="refDoi"> DOI: 10.1038/nrd4286</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref108/cit108&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=10.1038%2Fnrd4286" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref108/cit108&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=24751816" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref108/cit108&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmsFalsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=337-356&author=P.+Filippakopoulosauthor=S.+Knapp&title=Targeting+bromodomains%3A+epigenetic+readers+of+lysine+acetylation&doi=10.1038%2Fnrd4286"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit108R"><div class="casContent"><span class="casTitleNuber">108</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting bromodomains: epigenetic readers of lysine acetylation</span></div><div class="casAuthors">Filippakopoulos, Panagis; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">337-356</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Lysine acetylation is a key mechanism that regulates chromatin structure; aberrant acetylation levels have been linked to the development of several diseases.  Acetyl-lysine modifications create docking sites for bromodomains, which are small interaction modules found on diverse proteins, some of which have a key role in the acetylation-dependent assembly of transcriptional regulator complexes.  These complexes can then initiate transcriptional programs that result in phenotypic changes.  The recent discovery of potent and highly specific inhibitors for the BET (bromodomain and extra-terminal) family of bromodomains has stimulated intensive research activity in diverse therapeutic areas, particularly in oncol., where BET proteins regulate the expression of key oncogenes and anti-apoptotic proteins.  In addn., targeting BET bromodomains could hold potential for the treatment of inflammation and viral infection.  Here, we highlight recent progress in the development of bromodomain inhibitors, and their potential applications in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOSIcwBJKj4LVg90H21EOLACvtfcHk0lhvTziScyHrFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmsFalsr4%253D&md5=77049c82ce6a19b4685e3b7950399a07</span></div><a href="/servlet/linkout?suffix=cit108&amp;dbid=16384&amp;doi=10.1038%2Fnrd4286&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4286%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DTargeting%2520bromodomains%253A%2520epigenetic%2520readers%2520of%2520lysine%2520acetylation%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2014%26volume%3D13%26spage%3D337%26epage%3D356%26doi%3D10.1038%2Fnrd4286" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref109"><div class="reference"><strong class="refLabel"><a href="#ref109" class="rightTabRefNumLink">109</a></strong><div class="NLM_citation" id="rightTab-cit109"><span><span class="NLM_contrib-group">Christ, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fox, T.</span><span> </span><span class="NLM_article-title">Accuracy assessment and automation of free energy calculations for drug design</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">108</span><span class="NLM_x">–</span> <span class="NLM_lpage">120</span><span class="refDoi"> DOI: 10.1021/ci4004199</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci4004199" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref109/cit109&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvVSnur7M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2014&pages=108-120&author=C.+D.+Christauthor=T.+Fox&title=Accuracy+assessment+and+automation+of+free+energy+calculations+for+drug+design&doi=10.1021%2Fci4004199"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit109R"><div class="casContent"><span class="casTitleNuber">109</span><div class="casTitle"><span class="NLM_cas:atitle">Accuracy Assessment and Automation of Free Energy Calculations for Drug Design</span></div><div class="casAuthors">Christ, Clara D.; Fox, Thomas</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">108-120</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">As the free energy of binding of a ligand to its target is one of the crucial optimization parameters in drug design, its accurate prediction is highly desirable.  In the present study we have assessed the av. accuracy of free energy calcns. for a total of 92 ligands binding to five different targets.  To make this study and future larger scale applications possible we automated the setup procedure.  Starting from user defined binding modes, the procedure decides which ligands to connect via a perturbation based on max. common substructure criteria and produces all necessary parameter files for free energy calcns. in AMBER 11.  For the systems investigated, errors due to insufficient sampling were found to be substantial in some cases whereas differences in estimators (thermodn. integration (TI) vs. multistate Bennett acceptance ratio (MBAR)) were found to be negligible.  Anal. uncertainty ests. calcd. from a single free energy calcn. were found to be much smaller than the sample std. deviation obtained from two independent free energy calcns.  Agreement with expt. was found to be system dependent ranging from excellent to mediocre (RMSE = [0.9, 8.2, 4.7, 5.7, 8.7] kJ/mol).  When restricting analyses to free energy calcns. with sample std. deviations below 1 kJ/mol agreement with expt. improved (RMSE = [0.8, 6.9, 1.8, 3.9, 5.6] kJ/mol).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGri8r-6now_RrVg90H21EOLACvtfcHk0linT3C_QttKBQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvVSnur7M&md5=e36c5ee86688acfa1bce48f9a413ddc8</span></div><a href="/servlet/linkout?suffix=cit109&amp;dbid=16384&amp;doi=10.1021%2Fci4004199&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci4004199%26sid%3Dliteratum%253Aachs%26aulast%3DChrist%26aufirst%3DC.%2BD.%26aulast%3DFox%26aufirst%3DT.%26atitle%3DAccuracy%2520assessment%2520and%2520automation%2520of%2520free%2520energy%2520calculations%2520for%2520drug%2520design%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2014%26volume%3D54%26spage%3D108%26epage%3D120%26doi%3D10.1021%2Fci4004199" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref110"><div class="reference"><strong class="refLabel"><a href="#ref110" class="rightTabRefNumLink">110</a></strong><div class="NLM_citation" id="rightTab-cit110"><span><span class="NLM_contrib-group">Kollman, P.</span><span> </span><span class="NLM_article-title">Free energy calculations: applications to chemical and biochemical phenomena</span> <span class="citation_source-journal">Chem. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1993</span><span class="NLM_x">, </span> <span class="NLM_volume">93</span><span class="NLM_x">, </span> <span class="NLM_fpage">2395</span><span class="NLM_x">–</span> <span class="NLM_lpage">2417</span><span class="refDoi"> DOI: 10.1021/cr00023a004</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr00023a004" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref110/cit110&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;key=1%3ACAS%3A528%3ADyaK3sXmt1Sktr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=93&publication_year=1993&pages=2395-2417&author=P.+Kollman&title=Free+energy+calculations%3A+applications+to+chemical+and+biochemical+phenomena&doi=10.1021%2Fcr00023a004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit110R"><div class="casContent"><span class="casTitleNuber">110</span><div class="casTitle"><span class="NLM_cas:atitle">Free energy calculations: Applications to chemical and biochemical phenomena</span></div><div class="casAuthors">Kollman, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">93</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2395-417</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    </div><div class="casAbstract">A review with 252 refs. about applications of free energy calcns. employing mol. dynamics or Monte Carlo methods to a variety of chem. and biochem. phenomena.  The focus is on applications of such calcns. to mol. solvation, mol. assocn., macromol. stability, and enzyme catalysis.  The mols. discussed range from monovalent ions and small mols. to proteins and nucleic acids.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrN1w4SyFDqCLVg90H21EOLACvtfcHk0linT3C_QttKBQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXmt1Sktr0%253D&md5=f12326b24734ea092996cf22efb6ebd6</span></div><a href="/servlet/linkout?suffix=cit110&amp;dbid=16384&amp;doi=10.1021%2Fcr00023a004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr00023a004%26sid%3Dliteratum%253Aachs%26aulast%3DKollman%26aufirst%3DP.%26atitle%3DFree%2520energy%2520calculations%253A%2520applications%2520to%2520chemical%2520and%2520biochemical%2520phenomena%26jtitle%3DChem.%2520Rev.%26date%3D1993%26volume%3D93%26spage%3D2395%26epage%3D2417%26doi%3D10.1021%2Fcr00023a004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.7b00681&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2018.61.issue-3%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.7b00681%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.7b00681" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                3MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-b425991822a79ada59b3.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-fb88834b824b41b135ad.js"></script>

<script type="text/javascript" src="/wro/ke1q~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"66798731aede1955","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.6.0","si":10}'></script>
</body>
</html>
