
<html lang="en"     class="pb-page"  data-request-id="5a37bf60-f1d9-48fa-a904-c9c6d9128e53"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;issue:issue:10.1021/jmcmar.2021.64.issue-11;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/acs.jmedchem.1c00018;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Identification of a Novel TAR RNA-Binding Protein 2 Modulator with Potential Therapeutic Activity against Hepatocellular Carcinoma" /></meta><meta name="dc.Creator" content="Zongyuan  Zhou" /></meta><meta name="dc.Creator" content="Yiming  Li" /></meta><meta name="dc.Creator" content="Xiaofang  Ma" /></meta><meta name="dc.Creator" content="Biyun  Cao" /></meta><meta name="dc.Creator" content="Ting  Peng" /></meta><meta name="dc.Creator" content="Yuwen  Sheng" /></meta><meta name="dc.Creator" content="Huipan  Peng" /></meta><meta name="dc.Creator" content="Runze  Li" /></meta><meta name="dc.Creator" content="Yu  Cao" /></meta><meta name="dc.Creator" content="Ruiying  Xi" /></meta><meta name="dc.Creator" content="Fu  Li" /></meta><meta name="dc.Creator" content="Mengru  Wang" /></meta><meta name="dc.Creator" content="Handong  Sun" /></meta><meta name="dc.Creator" content="Guolin  Zhang" /></meta><meta name="dc.Creator" content="Hongbin  Zhang" /></meta><meta name="dc.Creator" content="Kaifeng  Hu" /></meta><meta name="dc.Creator" content="Weilie  Xiao" /></meta><meta name="dc.Creator" content="Fei  Wang" /></meta><meta name="dc.Description" content="Imbalance miRNAs contribute to tumor formation; therefore, the development of small-molecule compounds that regulate miRNA biogenesis is an important strategy in oncotherapy. Here, (−)-Gomisin M1 (..." /></meta><meta name="Description" content="Imbalance miRNAs contribute to tumor formation; therefore, the development of small-molecule compounds that regulate miRNA biogenesis is an important strategy in oncotherapy. Here, (−)-Gomisin M1 (..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="May 26, 2021" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.1c00018" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2021 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.1c00018" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.1c00018" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.1c00018" /></link>
        
    
    

<title>Identification of a Novel TAR RNA-Binding Protein 2 Modulator with Potential Therapeutic Activity against Hepatocellular Carcinoma | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.1c00018" /></meta><meta property="og:title" content="Identification of a Novel TAR RNA-Binding Protein 2 Modulator with Potential Therapeutic Activity against Hepatocellular Carcinoma" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00018/20210602/images/large/jm1c00018_0010.jpeg" /></meta><meta property="og:description" content="Imbalance miRNAs contribute to tumor formation; therefore, the development of small-molecule compounds that regulate miRNA biogenesis is an important strategy in oncotherapy. Here, (−)-Gomisin M1 (GM) was found to modulate miRNA biogenesis to inhibit the proliferation, migration, and invasion of hepatocellular carcinoma (HCC) cells. GM modulated expression profiles of miRNA and protein in HCC cells and suppressed tumor growth in a mouse model. Mechanistically, GM affected miRNA maturation by targeting TAR RNA-binding protein 2 (TRBP), with an efficacy higher than that of enoxacin, and promoted the binding of TRBP with Dicer. Structural simplification and a preliminary structure–activity relationship study via the synthesis of 20 GM derivatives showed that compound 9 exhibited more potent inhibitory activity in HCC cell proliferation and affinity for TRBP than did GM. These results suggest that TRBP may be a novel potential therapeutic target in HCC and compound 9 may be a potential drug candidate for the treatment of HCC." /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta><meta name="twitter:description" content="Imbalance miRNAs contribute to tumor formation; therefore, the development of small-molecule compounds that regulate miRNA biogenesis is an important strategy in oncotherapy. Here, (−)-Gomisin M1 (GM) was found to modulate miRNA biogenesis to inhibit the proliferation, migration, and invasion of hepatocellular carcinoma (HCC) cells. GM modulated expression profiles of miRNA and protein in HCC cells and suppressed tumor growth in a mouse model. Mechanistically, GM affected miRNA maturation by targeting TAR RNA-binding protein 2 (TRBP), with an efficacy higher than that of enoxacin, and promoted the binding of TRBP with Dicer. Structural simplification and a preliminary structure–activity relationship study via the synthesis of 20 GM derivatives showed that compound 9 exhibited more potent inhibitory activity in HCC cell proliferation and affinity for TRBP than did GM. These results suggest that TRBP may be a novel potential therapeutic target in HCC and compound 9 may be a potential drug candidate for the treatment of HCC." /></meta><meta name="twitter:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00018/20210602/images/large/jm1c00018_0010.jpeg" /></meta><meta name="twitter:title" content="Identification of a Novel TAR RNA-Binding Protein 2 Modulator with Potential Therapeutic Activity against Hepatocellular Carcinoma" /></meta>
        
            <meta property="og:type" content="Article" /></meta>
        

        
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.1c00018"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/ke1q~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-6ee29527b3d8ff00a106.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00018">
            
        





































    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-12/achre4.2021.54.issue-12/20210615/achre4.2021.54.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-6/amrcda.2021.2.issue-6/20210625/amrcda.2021.2.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-6/aabmcb.2021.4.issue-6/20210621/aabmcb.2021.4.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-6/aaembp.2021.3.issue-6/20210622/aaembp.2021.3.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-6/aaemcq.2021.4.issue-6/20210628/aaemcq.2021.4.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-25/aamick.2021.13.issue-25/20210630/aamick.2021.13.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-6/aanmf6.2021.4.issue-6/20210625/aanmf6.2021.4.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-6/aapmcd.2021.3.issue-6/20210611/aapmcd.2021.3.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-6/abseba.2021.7.issue-6/20210614/abseba.2021.7.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-12/accacs.2021.11.issue-12/20210618/accacs.2021.11.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-6/acscii.2021.7.issue-6/20210623/acscii.2021.7.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-6/acbcct.2021.16.issue-6/20210618/acbcct.2021.16.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-3/achsc5.2021.28.issue-3/20210524/achsc5.2021.28.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-12/acncdm.2021.12.issue-12/20210616/acncdm.2021.12.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-6/aesccq.2021.5.issue-6/20210617/aesccq.2021.5.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-6/aelccp.2021.6.issue-6/20210611/aelccp.2021.6.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-6/aeecco.2021.1.issue-6/20210611/aeecco.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-6/aewcaa.2021.1.issue-6/20210611/aewcaa.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-5/afsthl.2021.1.issue-5/20210618/afsthl.2021.1.issue-5.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-6/aidcbc.2021.7.issue-6/20210611/aidcbc.2021.7.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-6/amlccd.2021.10.issue-6/20210615/amlccd.2021.10.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-6/amlcef.2021.3.issue-6/20210607/amlcef.2021.3.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-6/amclct.2021.12.issue-6/20210610/amclct.2021.12.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-6/ancac3.2021.15.issue-6/20210622/ancac3.2021.15.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-25/acsodf.2021.6.issue-25/20210629/acsodf.2021.6.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-6/apchd5.2021.8.issue-6/20210616/apchd5.2021.8.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-6/ascefj.2021.6.issue-6/20210625/ascefj.2021.6.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-25/ascecg.2021.9.issue-25/20210628/ascecg.2021.9.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-6/asbcd6.2021.10.issue-6/20210618/asbcd6.2021.10.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-25/ancham.2021.93.issue-25/20210629/ancham.2021.93.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-25/bichaw.2021.60.issue-25/20210629/bichaw.2021.60.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-6/bcches.2021.32.issue-6/20210616/bcches.2021.32.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-6/bomaf6.2021.22.issue-6/20210614/bomaf6.2021.22.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-24/cgeabj.2021.99.issue-24/20210628-01/cgeabj.2021.99.issue-24.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-6/crtoec.2021.34.issue-6/20210621/crtoec.2021.34.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-12/chreay.2021.121.issue-12/20210623/chreay.2021.121.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-12/cmatex.2021.33.issue-12/20210622/cmatex.2021.33.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-6/cgdefu.2021.21.issue-6/20210602/cgdefu.2021.21.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-12/enfuem.2021.35.issue-12/20210617/enfuem.2021.35.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-12/esthag.2021.55.issue-12/20210615/esthag.2021.55.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-6/estlcu.2021.8.issue-6/20210608/estlcu.2021.8.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-25/iecred.2021.60.issue-25/20210630/iecred.2021.60.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-12/inocaj.2021.60.issue-12/20210621/inocaj.2021.60.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-6/jaaucr.2021.1.issue-6/20210628/jaaucr.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-25/jafcau.2021.69.issue-25/20210630/jafcau.2021.69.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-6/jceaax.2021.66.issue-6/20210610/jceaax.2021.66.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-6/jceda8.2021.98.issue-6/20210608/jceda8.2021.98.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-6/jcisd8.2021.61.issue-6/20210628/jcisd8.2021.61.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-6/jctcce.2021.17.issue-6/20210608/jctcce.2021.17.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/jmcmar.2021.64.issue-12/20210624/jmcmar.2021.64.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-6/jnprdf.2021.84.issue-6/20210625/jnprdf.2021.84.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-12/joceah.2021.86.issue-12/20210618/joceah.2021.86.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-24/jpcafh.2021.125.issue-24/20210624/jpcafh.2021.125.issue-24.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-24/jpcbfk.2021.125.issue-24/20210624/jpcbfk.2021.125.issue-24.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-24/jpccck.2021.125.issue-24/20210624/jpccck.2021.125.issue-24.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-24/jpclcd.2021.12.issue-24/20210624/jpclcd.2021.12.issue-24.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-6/jprobs.2021.20.issue-6/20210604/jprobs.2021.20.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-25/jacsat.2021.143.issue-25/20210630/jacsat.2021.143.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-6/jamsef.2021.32.issue-6/20210602/jamsef.2021.32.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-25/langd5.2021.37.issue-25/20210629/langd5.2021.37.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-12/mamobx.2021.54.issue-12/20210622/mamobx.2021.54.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-6/mpohbp.2021.18.issue-6/20210607/mpohbp.2021.18.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-12/nalefd.2021.21.issue-12/20210623/nalefd.2021.21.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-12/orlef7.2021.23.issue-12/20210618/orlef7.2021.23.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-6/oprdfk.2021.25.issue-6/20210618/oprdfk.2021.25.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-12/orgnd7.2021.40.issue-12/20210628/orgnd7.2021.40.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.1c00018&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.1c00018&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.1c00018&amp;href=/doi/10.1021/acs.jmedchem.1c00018" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2021</span><span class="cit-fg-volume">, 64</span><span class="cit-fg-issue">, 11</span><span class="cit-fg-pageRange">, 7404-7421</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/64/11" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.1c00010" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.1c00033" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Identification of a Novel TAR RNA-Binding Protein 2 Modulator with Potential Therapeutic Activity against Hepatocellular Carcinoma</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Zongyuan Zhou</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Zongyuan Zhou</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Center for Natural Products Research, Chengdu Institute of Biology, Chinese Academy of Sciences, Chengdu 610041, China</div><div class="loa-info-affiliations-info">University of Chinese Academy of Sciences, Beijing 100049, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Zongyuan++Zhou">Zongyuan Zhou</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0002-8403-6531" title="Orcid link">https://orcid.org/0000-0002-8403-6531</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yiming Li</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yiming Li</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Key Laboratory of Medicinal Chemistry for Natural Resource, Ministry of Education, Yunnan Research & Development Center for Natural Products, School of Chemical Science and Technology, Yunnan University, Kunming 650091, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yiming++Li">Yiming Li</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Xiaofang Ma</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xiaofang Ma</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Innovative Institute of Chinese Medicine and Pharmacy, Chengdu University of Traditional Chinese Medicine, Chengdu 611137, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xiaofang++Ma">Xiaofang Ma</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Biyun Cao</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Biyun Cao</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Key Laboratory of Medicinal Chemistry for Natural Resource, Ministry of Education, Yunnan Research & Development Center for Natural Products, School of Chemical Science and Technology, Yunnan University, Kunming 650091, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Biyun++Cao">Biyun Cao</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ting Peng</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ting Peng</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Center for Natural Products Research, Chengdu Institute of Biology, Chinese Academy of Sciences, Chengdu 610041, China</div><div class="loa-info-affiliations-info">University of Chinese Academy of Sciences, Beijing 100049, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ting++Peng">Ting Peng</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yuwen Sheng</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yuwen Sheng</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Center for Natural Products Research, Chengdu Institute of Biology, Chinese Academy of Sciences, Chengdu 610041, China</div><div class="loa-info-affiliations-info">University of Chinese Academy of Sciences, Beijing 100049, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yuwen++Sheng">Yuwen Sheng</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Huipan Peng</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Huipan Peng</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Center for Natural Products Research, Chengdu Institute of Biology, Chinese Academy of Sciences, Chengdu 610041, China</div><div class="loa-info-affiliations-info">University of Chinese Academy of Sciences, Beijing 100049, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Huipan++Peng">Huipan Peng</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Runze Li</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Runze Li</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Center for Natural Products Research, Chengdu Institute of Biology, Chinese Academy of Sciences, Chengdu 610041, China</div><div class="loa-info-affiliations-info">University of Chinese Academy of Sciences, Beijing 100049, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Runze++Li">Runze Li</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yu Cao</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yu Cao</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Center for Natural Products Research, Chengdu Institute of Biology, Chinese Academy of Sciences, Chengdu 610041, China</div><div class="loa-info-affiliations-info">University of Chinese Academy of Sciences, Beijing 100049, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yu++Cao">Yu Cao</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ruiying Xi</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ruiying Xi</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Center for Natural Products Research, Chengdu Institute of Biology, Chinese Academy of Sciences, Chengdu 610041, China</div><div class="loa-info-affiliations-info">University of Chinese Academy of Sciences, Beijing 100049, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ruiying++Xi">Ruiying Xi</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Fu Li</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Fu Li</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Center for Natural Products Research, Chengdu Institute of Biology, Chinese Academy of Sciences, Chengdu 610041, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Fu++Li">Fu Li</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Mengru Wang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Mengru Wang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Key Laboratory of Medicinal Chemistry for Natural Resource, Ministry of Education, Yunnan Research & Development Center for Natural Products, School of Chemical Science and Technology, Yunnan University, Kunming 650091, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Mengru++Wang">Mengru Wang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Handong Sun</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Handong Sun</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Phytochemistry and Plant Resources in West China, Kunming Institute of Botany, Chinese Academy of Sciences, Kunming 650201, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Handong++Sun">Handong Sun</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Guolin Zhang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Guolin Zhang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Center for Natural Products Research, Chengdu Institute of Biology, Chinese Academy of Sciences, Chengdu 610041, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Guolin++Zhang">Guolin Zhang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Hongbin Zhang</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Hongbin Zhang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Key Laboratory of Medicinal Chemistry for Natural Resource, Ministry of Education, Yunnan Research & Development Center for Natural Products, School of Chemical Science and Technology, Yunnan University, Kunming 650091, China</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#572d3f3639303f35172e392279323322793439"><span class="__cf_email__" data-cfemail="d6acbeb7b8b1beb496afb8a3f8b3b2a3f8b5b8">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Hongbin++Zhang">Hongbin Zhang</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0002-2516-2634" title="Orcid link">https://orcid.org/0000-0002-2516-2634</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Kaifeng Hu</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Kaifeng Hu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Innovative Institute of Chinese Medicine and Pharmacy, Chengdu University of Traditional Chinese Medicine, Chengdu 611137, China</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#385359515e5d565f504d785b5c4d4c5b55165d5c4d165b56"><span class="__cf_email__" data-cfemail="deb5bfb7b8bbb0b9b6ab9ebdbaabaabdb3f0bbbaabf0bdb0">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Kaifeng++Hu">Kaifeng Hu</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0002-2343-2012" title="Orcid link">https://orcid.org/0000-0002-2343-2012</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Weilie Xiao</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Weilie Xiao</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Key Laboratory of Medicinal Chemistry for Natural Resource, Ministry of Education, Yunnan Research & Development Center for Natural Products, School of Chemical Science and Technology, Yunnan University, Kunming 650091, China</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#255d4c444a52404c494c40655c4b500b4041500b464b"><span class="__cf_email__" data-cfemail="8cf4e5ede3fbe9e5e0e5e9ccf5e2f9a2e9e8f9a2efe2">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Weilie++Xiao">Weilie Xiao</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0001-6826-1993" title="Orcid link">https://orcid.org/0000-0001-6826-1993</a></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Fei Wang</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Fei Wang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Center for Natural Products Research, Chengdu Institute of Biology, Chinese Academy of Sciences, Chengdu 610041, China</div><div class="loa-info-affiliations-info">Xiongan Institute of Innovation, Chinese Academy of Sciences, Hebei 071700, China</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#c9bea8a7aeafaca089aaa0abe7a8aae7aaa7"><span class="__cf_email__" data-cfemail="6017010e07060509200309024e01034e030e">[email protected]</span></a></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Fei++Wang">Fei Wang</a></span></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.1c00018&amp;href=/doi/10.1021%2Facs.jmedchem.1c00018" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2021</span></span><span class="cit-volume">, 64</span><span class="cit-issue">, 11</span><span class="cit-pageRange">, 7404–7421</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">May 26, 2021</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>6 January 2021</li><li><span class="item_label"><b>Published</b> online</span>26 May 2021</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 10 June 2021</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.1c00018" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.1c00018</a></div><div class="article_header-article-copyright"><strong>Copyright © 2021 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D7404%26pageCount%3D18%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DZongyuan%2BZhou%252C%2BYiming%2BLi%252C%2BXiaofang%2BMa%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D64%26issueNum%3D11%26contentID%3Dacs.jmedchem.1c00018%26title%3DIdentification%2Bof%2Ba%2BNovel%2BTAR%2BRNA-Binding%2BProtein%2B2%2BModulator%2Bwith%2BPotential%2BTherapeutic%2BActivity%2Bagainst%2BHepatocellular%2BCarcinoma%26numPages%3D18%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D7421%26publicationDate%3DJune%2B2021">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.1c00018"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">531</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val">-</div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.1c00018" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Identification of a Novel TAR RNA-Binding Protein 2 Modulator with Potential Therapeutic Activity against Hepatocellular Carcinoma&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Zongyuan&quot;,&quot;last_name&quot;:&quot;Zhou&quot;},{&quot;first_name&quot;:&quot;Yiming&quot;,&quot;last_name&quot;:&quot;Li&quot;},{&quot;first_name&quot;:&quot;Xiaofang&quot;,&quot;last_name&quot;:&quot;Ma&quot;},{&quot;first_name&quot;:&quot;Biyun&quot;,&quot;last_name&quot;:&quot;Cao&quot;},{&quot;first_name&quot;:&quot;Ting&quot;,&quot;last_name&quot;:&quot;Peng&quot;},{&quot;first_name&quot;:&quot;Yuwen&quot;,&quot;last_name&quot;:&quot;Sheng&quot;},{&quot;first_name&quot;:&quot;Huipan&quot;,&quot;last_name&quot;:&quot;Peng&quot;},{&quot;first_name&quot;:&quot;Runze&quot;,&quot;last_name&quot;:&quot;Li&quot;},{&quot;first_name&quot;:&quot;Yu&quot;,&quot;last_name&quot;:&quot;Cao&quot;},{&quot;first_name&quot;:&quot;Ruiying&quot;,&quot;last_name&quot;:&quot;Xi&quot;},{&quot;first_name&quot;:&quot;Fu&quot;,&quot;last_name&quot;:&quot;Li&quot;},{&quot;first_name&quot;:&quot;Mengru&quot;,&quot;last_name&quot;:&quot;Wang&quot;},{&quot;first_name&quot;:&quot;Handong&quot;,&quot;last_name&quot;:&quot;Sun&quot;},{&quot;first_name&quot;:&quot;Guolin&quot;,&quot;last_name&quot;:&quot;Zhang&quot;},{&quot;first_name&quot;:&quot;Hongbin&quot;,&quot;last_name&quot;:&quot;Zhang&quot;},{&quot;first_name&quot;:&quot;Kaifeng&quot;,&quot;last_name&quot;:&quot;Hu&quot;},{&quot;first_name&quot;:&quot;Weilie&quot;,&quot;last_name&quot;:&quot;Xiao&quot;},{&quot;first_name&quot;:&quot;Fei&quot;,&quot;last_name&quot;:&quot;Wang&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2021&quot;,&quot;month&quot;:&quot;05&quot;,&quot;day&quot;:&quot;26&quot;,&quot;issue&quot;:&quot;11&quot;,&quot;volume&quot;:&quot;64&quot;,&quot;pages&quot;:&quot;7404-7421&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.1c00018&quot;},&quot;abstract&quot;:&quot;Imbalance miRNAs contribute to tumor formation; therefore, the development of small-molecule compounds that regulate miRNA biogenesis is an important strategy in oncotherapy. Here, (−)-Gomisin M1 (GM) was found to modulate miRNA biogenesis to inhibit the proliferation, migration, and invasion of hepatocellular carcinoma (HCC) cells. GM modulated expression profiles of miRNA and protein in HCC cells and suppressed tumor growth in a mouse model. Mechanistically, GM affected miRNA maturation by targeting TAR RNA-binding protein 2 (TRBP), with an efficacy higher than that of enoxacin, and promoted the binding of TRBP with Dicer. Structural simplification and a preliminary structure–activity relationship study via the synthesis of 20 GM derivatives showed that compound 9 exhibited more potent inhibitory activity in HCC cell proliferation and affinity for TRBP than did GM. These results suggest that TRBP may be a novel potential therapeutic target in HCC and compound 9 may be a potential drug candidate for the trea&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00018&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00018" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00018&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00018" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00018&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00018" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.1c00018&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00018&amp;href=/doi/10.1021/acs.jmedchem.1c00018" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.1c00018" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.1c00018" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (8 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.1c00018%26sid%3Dliteratum%253Aachs%26pmid%3D34038111%26genre%3Darticle%26aulast%3DZhou%26date%3D2021%26atitle%3DIdentification%2Bof%2Ba%2BNovel%2BTAR%2BRNA-Binding%2BProtein%2B2%2BModulator%2Bwith%2BPotential%2BTherapeutic%2BActivity%2Bagainst%2BHepatocellular%2BCarcinoma%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D64%26issue%3D11%26spage%3D7404%26epage%3D7421%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=290896" title="Cells">Cells</a>,</li><li><a href="/action/doSearch?ConceptID=292258" title="Cancer">Cancer</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291439" title="Genetics">Genetics</a>,</li><li><a href="/action/doSearch?ConceptID=292586" title="Protein dynamics">Protein dynamics</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/64/11" title="Go to Volume 64, Issue 11"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/jmcmar.2021.64.issue-11/20210610/jmcmar.2021.64.issue-11.largecover.jpg" alt="Go to Volume 64, Issue 11"/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00018/20210602/images/medium/jm1c00018_0010.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00018/20210602/images/large/jm1c00018_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00018&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Imbalance miRNAs contribute to tumor formation; therefore, the development of small-molecule compounds that regulate miRNA biogenesis is an important strategy in oncotherapy. Here, (−)-Gomisin M1 (GM) was found to modulate miRNA biogenesis to inhibit the proliferation, migration, and invasion of hepatocellular carcinoma (HCC) cells. GM modulated expression profiles of miRNA and protein in HCC cells and suppressed tumor growth in a mouse model. Mechanistically, GM affected miRNA maturation by targeting TAR RNA-binding protein 2 (TRBP), with an efficacy higher than that of enoxacin, and promoted the binding of TRBP with Dicer. Structural simplification and a preliminary structure–activity relationship study via the synthesis of 20 GM derivatives showed that compound <b>9</b> exhibited more potent inhibitory activity in HCC cell proliferation and affinity for TRBP than did GM. These results suggest that TRBP may be a novel potential therapeutic target in HCC and compound <b>9</b> may be a potential drug candidate for the treatment of HCC.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_34304" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_34304" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">HCC is the fourth most common cause of cancer-related death and ranks sixth in terms of incidence among all cancers in the world.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> Patients with liver disease are vulnerable to the development of HCC, mostly as a result of hepatitis B or C virus infection or bibulosity. Under these conditions, DNA alterations occur frequently, including mutations and chromosomal aberrations in the <i>TERT</i> promoter, <i>TP53</i>, <i>CTNNB1</i>, and <i>AXIN1</i>.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> However, systemic small-molecule drugs inhibit receptor tyrosine kinases and do not target these mutated genes or proteins directly. Multi-kinase inhibitors have been approved for the treatment of HCC, including the first-line therapies sorafenib and lenvatinib. However, the one-year median survival time remains short.<a onclick="showRef(event, 'ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref2 ref3">(2,3)</a> Therapies currently in development include angiogenesis inhibitors and immune checkpoint inhibitors, which show similar median survival times compared with that of sorafenib.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> Therefore, it is imperative to explore more innovative drugs and effective targets for HCC therapy. MicroRNAs (miRNAs) are involved in a variety of physiological processes and are associated with multifarious diseases through coordination of gene expression at the post-transcriptional level. Accumulating evidence suggests that the expression profiles of microRNAs differ significantly between patients with HCC and healthy individuals, and this aberrant expression promotes the proliferation, invasion, and metastasis of HCC.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> Furthermore, clinical studies have indicated that different miRNAs could be used as prognostic biomarkers for the development of liver cancer or for predicting treatment outcomes.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> Thus, treatment strategies for HCC that modulate intracellular miRNA levels are of interest. Indeed, miRNAs as novel targets for nucleotide-based drugs have great appeal in the development of HCC therapies.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> However, exogenous RNAs can induce off-target effects, are difficult to deliver, induce immunogenicity, and are associated with serious safety issues in clinical settings.<a onclick="showRef(event, 'ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref8 ref9">(8,9)</a> In addition, the functions of various miRNAs remain unclear because changes in a single miRNA, unlike a protein-coding gene, rarely lead to a specific phenotype.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> In fact, some small molecules have been reported to regulate cancer growth and metastasis in multifarious cancers by inhibiting the expression of miR-21 or miR-122 and others. Therefore, regulating global miRNA homeostasis by chemical small molecules may represent a potential novel therapeutic strategy in oncotherapy.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a></div><div class="NLM_p">TRBP, a key component of the miRNA-induced silencing complex (RISC), affects miRNA biogenesis by ensuring the efficient and accurate Dicer processing of precursor microRNA in RNA-crowded environments<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> or by modulating RISC assembly and Ago2-induced miRNA stability.<a onclick="showRef(event, 'ref13 ref14'); return false;" href="javascript:void(0);" class="ref ref13 ref14">(13,14)</a> In addition, miRNA expression can be influenced by extracellular regulated protein kinases (ERK) or S6 kinase 2 (S6K2) kinase-mediated TRBP phosphorylation.<a onclick="showRef(event, 'ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref15 ref16">(15,16)</a> Furthermore, TRBP is involved in the development and progression of various cancers.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> The level of expression of TRBP is reduced in cancer stem cells of Ewing sarcoma family tumors (ESFTs), resulting in the mRNA profile becoming disordered.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> However, TRBP can function as a tumor suppressor or promoter, with different expression levels reported in diverse cancers. For example, TRBP was found to be upregulated in prostate cancer and adrenocortical carcinoma and downregulated in colorectal cancer, ESFT, and upper urinary tract urothelial carcinoma.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> Therefore, the function of TRBP in cancer, particularly in HCC, remains unclear. Further research is needed to confirm whether the role of TRBP in tumorigenesis is cell context-dependent and whether TRBP function is dependent on different regulatory patterns.</div><div class="NLM_p last">Enoxacin, a fluoroquinolone antibacterial drug, is the only known TRBP agonist that enhances miRNA biogenesis and inhibits cancer growth.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> However, enoxacin was reported to suppress the proliferation of colon cancer cells with low potency (IC<sub>50</sub> > 100 μM) and low affinity for TRBP (affinity constant; <i>K</i><sub>D</sub> ∼ 12.56 μM).<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> Therefore, it is necessary to develop new small molecules targeting TRBP to better understand the homeostasis of miRNA biosynthesis in HCC. GM, a natural dibenzocyclooctadiene lignan isolated from <i>Schisandra chinensis</i>,<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> has been shown to exhibit anti-inflammatory,<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> antioxidant,<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> advanced glycation end-product inhibitor,<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> and anti-human immunodeficiency virus (HIV)<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> activity. However, its role in the suppression of HCC tumorigenesis remains unclear. GM analogues such as Gomisin A and Gomisin N have been found to exhibit inhibitory effects on proliferation and carcinogenesis of HCC,<a onclick="showRef(event, 'ref25 ref26 ref27 ref28'); return false;" href="javascript:void(0);" class="ref ref25 ref26 ref27 ref28">(25−28)</a> but the effects of these gomisins on miRNAs expression and their molecular targets remain unclear. Here, we found that GM modulates TRBP-mediated miRNA biogenesis and exhibits potent anti-hepatoma effects by reconstituting tumor-related proteins and signaling pathways <i>in vitro</i> and <i>in vivo</i>. Analysis of the structure–activity relationship identified GM analogues that target TRBP with increased specificity versus that of enoxacin. This study supports the role of TRBP as a potential therapeutic target as well as further development of GM or its analogues as novel drug candidates for the modulation of miRNA biogenesis in HCC.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_30001" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_30001" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> GM Modulates miRNA Biogenesis to Suppress HCC Proliferation and Metastasis In Vitro</h3><div class="NLM_p">We established a screening assay based on HeLa-miR-21-luciferase cells, in which the luciferase reporter was used to evaluate variation in endogenous miR-21 (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A). A compound library consisting of 1431 diverse natural and synthetic compounds<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> was screened (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00018/suppl_file/jm1c00018_si_001.pdf" class="ext-link">Figure S1A</a>), and GM was found to significantly decrease the intensity of the luciferase signal with better efficacy than enoxacin (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00018/suppl_file/jm1c00018_si_001.pdf" class="ext-link">Figure S1B</a>). The chemical structure of GM is shown in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B. GM increased the level of miR-21 biogenesis with a median effective concentration (IC<sub>50</sub>) of 17.15 μM (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>C). Compared with that of GM, the IC<sub>50</sub> of enoxacin was 25.77 μM (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>D). To determine the specificity of GM for miRNA biogenesis, miRNA production in SK-Hep-1 cells was evaluated following treatment with GM or enoxacin. The level of expression of tumor suppressors, including miR-122-5p,<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> miR-26a-5p,<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> and miR-214-5p,<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> was increased, along with the level of expression of oncomir miR-21-5p<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> and miR-106b-5p<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>E). Additionally, the levels of oncomiRs miR-224-5p<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> and miR-452-5p<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> decreased, along with the levels of tumor suppressor miR-30a-5p<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>E). There was no significant effect on the production of miR-222-5p, miR-10a-5p, miR-135a-5p, and miR-22-5p (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>E). However, in contrast to GM, enoxacin increased the levels of miR-106b-5p, miR-21-5p, miR-26a-5p, and miR-22-5p and decreased those of miR135a-5p and miR-30a-5p (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>F). To verify whether changes in mature miRNAs were due to the influence of upstream miRNA biogenesis, the level of expression of corresponding primary miRNAs (pri-miRNAs) and precursor miRNAs (pre-miRNAs) was measured. Expression of these pri-miRNAs and pre-miRNAs remained largely unchanged (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00018/suppl_file/jm1c00018_si_001.pdf" class="ext-link">Figure S2A,B</a>), suggesting that GM modulated the expression of a subset of miRNAs. GM was then examined for its antineoplastic activity against liver cancer cells (SK-Hep-1, MHCC97, HepG2, and Huh7), breast cancer cells (MDA-MB-231), colorectal cancer cells (HCT116), cervical cancer cells (HeLa), and immortalized hepatic normal cells (QSG-7701). GM dose-dependently inhibited the proliferation of SK-Hep-1 cells with a half-maximal inhibitory concentration (IC<sub>50</sub>) of 20.56 μM (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00018/suppl_file/jm1c00018_si_001.pdf" class="ext-link">Figure S3A</a>) compared with enoxacin, which had an IC<sub>50</sub> of 25.03 μM (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00018/suppl_file/jm1c00018_si_001.pdf" class="ext-link">Figure S3B</a>). GM significantly suppressed the proliferation of MHCC97, HepG2, Huh7, and HeLa cells but not MDA-MB-231 and HCT116 cells at 25 μM (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00018/suppl_file/jm1c00018_si_001.pdf" class="ext-link">Figure S3C</a>). Consistent with this, the IC<sub>50</sub> values of GM on these cell lines were calculated as 24.75 μM (MHCC97), 27.26 μM (HepG2), 32.23 μM (Huh7), 32.83 μM (HeLa), 56.61 μM (MDA-MB-231), 55.38 μM (HCT116), and 68.86 μM (QSG-7701) (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00018/suppl_file/jm1c00018_si_001.pdf" class="ext-link">Figure S3D–J</a>). GM inhibited colony formation in SK-Hep-1 cells (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>G). Furthermore, GM significantly inhibited the clonogenic capacity of the other six cancer cell lines compared with the cells treated with DMSO (dimethyl sulfoxide) and more potently than enoxacin, especially in SK-Hep-1 and MHCC97 cells (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>G and <a href="/doi/suppl/10.1021/acs.jmedchem.1c00018/suppl_file/jm1c00018_si_001.pdf" class="ext-link">Figure S4A</a>). GM induced significant G0/G1-phase arrest, and the percentage of cells in the G2/M phase decreased significantly compared with that of the DMSO-treated cells (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>H); however, it did not affect the number of apoptotic cells (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00018/suppl_file/jm1c00018_si_001.pdf" class="ext-link">Figure S5</a>). Compared with DMSO treatment, GM markedly suppressed the migration capability of SK-Hep-1 and MHCC97 cells as determined using the Transwell assay (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>I and <a href="/doi/suppl/10.1021/acs.jmedchem.1c00018/suppl_file/jm1c00018_si_001.pdf" class="ext-link">Figure S4B</a>). GM decreased the invasiveness of HCC cells and inhibited the rate of artificial basement membrane penetration in SK-Hep-1 cells (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>J). However, GM had no effect on the expression of vimentin, N-cadherin, E-cadherin, and β-catenin (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00018/suppl_file/jm1c00018_si_001.pdf" class="ext-link">Figure S6</a>), which are markers for epithelial mesenchymal transition (EMT), thus excluding the possibility that GM suppresses HCC cell migration by inhibiting EMT.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00018/20210602/images/medium/jm1c00018_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00018/20210602/images/large/jm1c00018_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. GM modulates miRNA biogenesis to suppress HCC proliferation and metastasis. (A) Model of high-throughput screening by HeLa-miR-21-luciferase cells. (B) Chemical structure of GM. (C and D) Luciferase activity in HeLa-miR-21-luciferase cells determined after treatment with GM and enoxacin, respectively, at the indicated concentrations. (E and F) qRT-PCR of the relative miRNA levels in SK-Hep-1 cells treated with GM and enoxacin, respectively, at 20 μM. (G) Representative images showing the effect of GM and enoxacin on colony formation 14 days after treatment at the noted concentrations in SK-Hep-1 cells. (H) Flow cytometry quantitation of the proportion of SK-Hep-1 cells treated with or without GM at 20 μM. (I and J) Representative images showing the effect of 30 μM GM or enoxacin on the migration and invasion, respectively, of SK-Hep-1 cells. Statistical significance was determined by a two-tailed Student’s <i>t</i> test. *<i>p</i> < 0.05, **<i>p</i> < 0.01, and ***<i>p</i> < 0.001 compared with the control. N.S, not significant.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00018/20210602/images/large/jm1c00018_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00018&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> GM Suppresses HCC Formation <i>In Vivo</i></h3><div class="NLM_p">To examine the antitumor effect of GM <i>in vivo</i>, an animal experiment was performed by orthotopically implanting MHCC97-luciferase cells into severe combined immunodeficient (SCID) mice. After the injection of MHCC97-luciferase cells into the left outer lobe of the liver, mice were treated with phosphate-buffered saline (PBS), 20 mg of enoxacin/kg, 20 mg of sorafenib/kg, or 20 or 40 mg of GM/kg. Imaging and quantitative bioluminescence analysis of the tumor revealed a marked decrease in the luminescence area in all treatment groups compared with the control (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A,B). Additionally, 20 mg of GM/kg showed greater anti-HCC efficacy compared with enoxacin and reduced anti-HCC efficacy compared with sorafenib. However, 40 mg of GM/kg demonstrated efficacy comparable to that of sorafenib. Consistent with the results of the bioluminescence analysis, enoxacin, sorafenib, and GM markedly reduced tumor size (black arrow in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>C). Beyond that, intrahepatic metastasis in control, enoxacin, and 20 mg of GM/kg groups was observed from a liver image (red arrow in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>C); simultaneously, 40 mg of GM and sorafenib per kilogram treatment suppressed intrahepatic metastasis (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>C). While the body weight of mice increased in both groups, there were no obvious changes between the body weight of mice that received drugs and that of control mice at the end of the experiment (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>D). Hematoxylin and eosin (H&E) staining revealed that GM markedly attenuated the infiltration of inflammatory cells in the left outer lobe of the liver (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>E). Furthermore, we examined the status of inflammation infiltration in the middle lobe of the liver and found less infiltration in the enoxacin, sorafenib, and GM (40 mg/kg) (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>F). Collectively, these results showing that GM exerts a suppressive effect on tumor burden provide <i>in vivo</i> confirmation of the <i>in vitro</i> findings that GM inhibits HCC proliferation and migration.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00018/20210602/images/medium/jm1c00018_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00018/20210602/images/large/jm1c00018_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. GM suppresses HCC tumor formation <i>in vivo</i>. (A) Representative luminescence images of mice one month after the first administration, showing anticancer activity of GM in the SK-Hep-1 orthotopic xenograft nude mouse model. (B) Tumors were quantified on the basis of the degree of luminescence (<i>n</i> = 6). (C) Representative images of liver tumors from mice one month after the first administration. (D) Mice weight over 30 days (<i>n</i> = 6). (E and F) H&E-stained sections of the left outer lobe and middle lobe, respectively, of the liver collected on day 30 (<i>n</i> = 6). A two-tailed unpaired <i>t</i> test was used for the statistical analysis in panels B and C, with means ± standard errors of the mean. *<i>p</i> < 0.05, **<i>p</i> < 0.01, and ***<i>p</i> < 0.001 compared with the control. N.S, not significant. Black and red arrow: tumor. Blue arrow: immune cell infiltration.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00018/20210602/images/large/jm1c00018_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00018&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> GM Modulates miRNA Expression and the Proteome in HCC Cells</h3><div class="NLM_p">To examine the profile of miRNAs in response to GM, we analyzed global miRNA expression profiles using a comprehensive miRNA microarray platform following the treatment of SK-Hep-1 cells. Small RNA libraries of the control and GM-treated SK-Hep-1 cells were constructed as described in the legend of <a href="/doi/suppl/10.1021/acs.jmedchem.1c00018/suppl_file/jm1c00018_si_001.pdf" class="ext-link">Figure S7</a>. All control and GM treatments generated approximately 10 million clean tags, which were then searched against the miRBase database (release 21) to identify known human miRNAs (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00018/suppl_file/jm1c00018_si_001.pdf" class="ext-link">Figure S8A</a>). Consequently, approximately 700 miRNAs were identified in each group (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00018/suppl_file/jm1c00018_si_001.pdf" class="ext-link">Figure S8B</a>). On the basis of the expression in each sample, miRNA expression was calculated and normalized to transcripts per million. A total of 69 differentially expressed miRNAs (<i>p</i> < 0.05, >2-fold change) were identified between the two groups, including 36 upregulated and 33 downregulated miRNAs (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A and <a href="/doi/suppl/10.1021/acs.jmedchem.1c00018/suppl_file/jm1c00018_si_001.pdf" class="ext-link">Table S1</a>), suggesting that GM regulated only a subset of miRNAs. Overall, the differentially expressed miRNAs (57 of 69) have been demonstrated to participate in cancer development and prognosis (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00018/suppl_file/jm1c00018_si_001.pdf" class="ext-link">Table S1</a>). Considered previous reports that miR-10b-5p,<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> miR-497-5p,<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> miR-146a-5p,<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> and miR-224-5p<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> affected the cancer cell proliferation and metastasis in liver cancer, all of them were chosen for quantitative analysis with quantitative real-time polymerase chain reaction (qRT-PCR). Consistent with miRNAomics, the qRT-PCR results showed that GM treatment enhanced the expression of miR-497-5p and miR-146a-5p in a concentration-dependent manner and decreased that of miR-10b-5p and miR-224-5p (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B). Functional enrichment analysis of these miRNA target genes revealed that the downstream regulation of GM involves nucleic acid-binding transcription factor activity, transmembrane transporter activity, DNA binding, transporter activity, and signal transducer activity (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00018/suppl_file/jm1c00018_si_001.pdf" class="ext-link">Figure S9</a>). Kyoto Encyclopedia of Genes and Genomes (KEGG) pathway enrichment analysis of miRNA target genes identified significantly enriched signal transduction pathways, such as phosphatidyl inositide 3-kinases protein kinase B (PI3K-Akt), Ras, mitogen-activated protein kinase (MAPK), and chemokine signaling pathways (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>C). GM inhibited the phosphorylation of IκB, p65, MEK, AKT, and ERK1/2 but without a significant effect on the phosphorylation of p38, STAT1, and STAT3 (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00018/suppl_file/jm1c00018_si_001.pdf" class="ext-link">Figure S10</a>).</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00018/20210602/images/medium/jm1c00018_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00018/20210602/images/large/jm1c00018_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. GM modulates the miRNAome and proteome in HCC cells. (A) Heat map representing all differentially expressed miRNAs with a <i>p</i> value of <0.05 and a >2-fold change (<i>n</i> = 3). (B) qRT-PCR of the relative miRNA levels of the indicated genes. (C) Bubble diagram showing the pathway enrichment analysis of miRNA target genes. (D) Heat map representing all differentially expressed proteins with a <i>p</i> value of <0.05 and a >1.3-fold change (<i>n</i> = 3). (E) Western blotting analysis of protein expression in SK-Hep-1 cells treated with GM at a series of concentrations. (F) Histogram showing the functional enrichment analysis of differentially expressed proteins. (G) Interactions among differentially expressed proteins shown by a network graph. Statistical significance was determined by a two-tailed Student’s <i>t</i> test. *<i>p</i> < 0.05, **<i>p</i> < 0.01, and ***<i>p</i> < 0.001 compared with the control.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00018/20210602/images/large/jm1c00018_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00018&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Isobaric tags for relative and absolute quantification (iTRAQ) and liquid chromatography-tandem mass spectrometry (LC-MS/MS) were used to analyze protein expression profiles (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00018/suppl_file/jm1c00018_si_001.pdf" class="ext-link">Figure S11</a>). Three separate iTRAQ data sets were acquired to analyze GM and DMSO treatments each. The Pearson’s correlation coefficient revealed the high repeatability of protein expression between the two groups (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00018/suppl_file/jm1c00018_si_001.pdf" class="ext-link">Table S2</a>). Using this approach, 93 differentially expressed proteins were identified (<i>p</i> < 0.05, >1.3-fold change), which included 37 downregulated and 56 upregulated proteins (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>D and <a href="/doi/suppl/10.1021/acs.jmedchem.1c00018/suppl_file/jm1c00018_si_001.pdf" class="ext-link">Table S3</a>). Four of them affect cell growth and migration inhibition, including kinesin family member 14 (KIF14),<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> targeting protein for Xklp2 (TPX2),<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> ubiquitin conjugating enzyme E2 S (UBE2S),<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> and cell division cycle 20 (CDC20).<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> A Western blot assay was used to verify the expression levels of those proteins, which were consistent with the proteomics results (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>E). Functional enrichment analysis of 93 proteins revealed that the significance of the gene ontology (GO) biological process changed after GM treatment and was related to antigen processing and presentation as well as a mitotic cell cycle (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>F). The interactions among these 93 proteins were analyzed by a direct interaction algorithm, which revealed a network of 35 protein interactions that were mainly focused on cell division and antigen processing and presentation. Among them, CDC20 connected other up- and downregulated proteins (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>G), which was consistent with our observation that GM suppressed the proliferation of HCC (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>G,H). KEGG pathway enrichment analysis revealed that ubiquitin-mediated proteolysis, p53 signaling, antigen processing and presentation, glutathione metabolism, and primary immunodeficiency were enriched in differentially expressed proteins (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00018/suppl_file/jm1c00018_si_001.pdf" class="ext-link">Figure S12</a>). Of the differentially expressed proteins, four (KIF14, TPX2, CDC20, and UBE2S) were selected and examined in tumor specimens from mice and the level of expression of all four proteins was decreased by GM (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00018/suppl_file/jm1c00018_si_001.pdf" class="ext-link">Figure S13</a>).</div><div class="NLM_p last">To identify the relationship between differentiated miRNAs and proteins, we performed an integrated analysis of differentially expressed proteins and miRNAs from proteomes and miRNAomes. Integrative analyses revealed that 27 miRNAs were directly associated with 24 proteins (25.8% of the total proteins) (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00018/suppl_file/jm1c00018_si_001.pdf" class="ext-link">Figure S14</a>). Most proteins and miRNAs in those integrative relationships were associated with the occurrence and development of cancer. Furthermore, some miRNAs targeted more than one protein, such as miR-143-3p, suggesting that the modulation of miRNA biogenesis may exhibit greater efficiency in post-transcriptional regulation networks (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00018/suppl_file/jm1c00018_si_001.pdf" class="ext-link">Figure S14</a>).</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> TRBP Modulates the Anticancer Effect of GM</h3><div class="NLM_p">To assess how GM regulates miRNA maturation, we first examined the protein levels of RISC components, including Dicer, TRBP, and Ago2. We found that the expression of RISC components was not altered in GM-treated cells (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00018/suppl_file/jm1c00018_si_001.pdf" class="ext-link">Figure S15</a>). Furthermore, the cellular thermal shift assay was used to detect interactions between the components and GM. The results showed that GM decreased the thermostability of TRBP (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A,B) rather than that of Dicer and Ago2 (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00018/suppl_file/jm1c00018_si_001.pdf" class="ext-link">Figure S16</a>) following co-incubation with cell lysis at a series of concentrations and temperatures. To further examine the role of TRBP in GM-induced suppression of HCC cells, TRBP was transiently overexpressed or knocked in SK-Hep-1 cells (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00018/suppl_file/jm1c00018_si_001.pdf" class="ext-link">Figure S17</a>). As shown in panels C and D of <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>, the level of cell proliferation was significantly decreased following the overexpression or knockdown of TRBP, indicating that TRBP plays a key role in tumorigenesis. Furthermore, overexpression of TRBP enhanced GM-induced suppression of SK-Hep-1 cells (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>C). Meanwhile, the antiproliferative effect of GM was greatly attenuated when knocking down TRBP (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>D), indicating GM had minimal side effects. Additionally, TRBP overexpression significantly enhanced GM-mediated suppression of cell migration (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>E), whereas knockdown of TRBP attenuated the antimigration activity of GM in SK-Hep-1 cells (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>F). These results suggested that TRBP mediates the anticancer effect of GM in HCC cells.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00018/20210602/images/medium/jm1c00018_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00018/20210602/images/large/jm1c00018_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. TRBP modulates the anticancer effect of GM. (A and B) Western blotting and melt curve analysis showing the results of the cellular thermal shift assay in SK-Hep-1 cellular lysates treated with 50 μM GM at different (A) temperatures or (B) concentrations at 55 °C. (C and D) Representative images of a colony formation assay in SK-Hep-1 cells performed to assess the antiproliferative effect of GM (20 μM) after overexpressing and silencing, respectively, the <i>TRBP</i> gene. (E and F) Transwell assay of SK-Hep-1 cellular migration to assess the effect of GM (20 μM) after overexpressing and silencing, respectively, of the <i>TRBP</i> gene. All of the numerical data are presented as means ± standard deviations.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00018/20210602/images/large/jm1c00018_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00018&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> GM Physically Interacts with TRBP</h3><div class="NLM_p">The binding of GM to TRBP was first examined by chemical shift perturbation (CSP) and saturation-transfer difference (STD) nuclear magnetic resonance (NMR) experiments (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A–C). In the CSP spectra, the signals for C-5-OMe and C-8-OMe (3.788 and 3.877 ppm, respectively) in GM shifted to a high field and broadened with increasing concentrations of TRBP (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B). Conversely, the peak height for C-7-OMe (3.756 ppm) in GM decreased, although with a negligible shift, and the line width broadened as the concentration of TRBP increased (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>C). The interaction between GM and TRBP was also supported by the STD spectra. Saturation pulses were applied to cover the range of approximately 8.29–8.79 ppm, in which the amide groups of TRBP are excited. In the STD spectra, signals of the same three methoxyl groups in GM in the presence of TRBP were observed (red row in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A–C), whereas no obvious STD signal was observed in the absence of TRBP (black row in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A–C). Second, the physical interaction between GM and TRBP protein was examined using surface plasmon resonance (SPR). Binding between GM and recombinant expressed full-length TRBP was measured using a CM5 sensor chip, which translated the binding signal to resonance units. As shown in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>D, a <i>K</i><sub>D</sub> of 8.06 μM was calculated when determining the affinity of GM for TRBP. In addition, stable interaction binding kinetics was observed, whereby the binding response unit decreased as the GM concentration decreased (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>E). The effect of GM on the interaction between TRBP and Dicer was examined by immunoprecipitation. The addition of GM strengthened the interaction between TRBP and Dicer but had no effect on Ago2 (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>F). These results confirm an interaction between GM and TRBP.</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00018/20210602/images/medium/jm1c00018_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00018/20210602/images/large/jm1c00018_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. GM physically interacts with TRBP. (A–C) Binding of GM and TRBP was analyzed by CSP and STD-NMR. In CSP, <sup>1</sup>H spectra of ∼20 μM GM with 0 (black), 7.5 (yellow), 15 (green), 30 (red), and 60 (blue) μM TRBP were recorded and overlapped. In STD, spectra of ∼20 μM GM without (black, as a control) and with 30 μM TRBP (red) were recorded. (A) Full CSP and STD spectra of the GM with TRBP show the H<sub>2</sub>O resonance. Peaks C-5-OMe, C-7-OMe, and C-8-OMe denoted the NMR resonance of methoxyl groups in GM. Magnified CSP and STD signals of (B) C-5-OMe or (C) C-7-OMe and C-8-OMe. (D and E) SPR assay showing the interaction between GM and TRBP. (D) Steady-state kinetics and (E) affinity constant of GM at different concentrations were calculated by a CM5 Biacore chip-immobilized TRBP protein. (F) Co-immunoprecipitation analysis of the interaction between TRBP and Dicer or Ago2 in SK-Hep-1 cells following treatment with 50 μM GM.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00018/20210602/images/large/jm1c00018_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00018&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Structure–Activity Relationship (SAR) of GM in the Modulation of miRNA Biogenesis</h3><div class="NLM_p">GM is a natural product obtained from <i>S. chinensis</i>, but it is difficult to obtain or synthesize enough of this compound for drug development. Therefore, there needs to be structural simplification to pave the way for further structural optimization; therefore, we simplified the structure of GM by removing ring C, and active groups were introduced at C-8 of ring A and at C-4 and N of ring B in the new skeleton while retaining the main skeleton (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>). On the basis of our previous synthesis of compound <b>1</b> and intermediate <b>1</b>-<b>1</b>,<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> 20 analogues of GM were synthesized to study the SAR of GM. As shown in <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>, the acyl chloride was obtained by reflux of intermediate <b>1</b>-<b>1</b> in dichloro sulfoxide, and the amide compounds (<b>2–6</b>) was obtained by adding the corresponding primary amine. Started with compound <b>1</b>, compound <b>7</b> was obtained by selective demethylation, which could be further chlorinated to form compound <b>8</b> or compound <b>9</b> with benzoyl chloride. In addition, compound <b>7</b> not only formed sulfonamide (<b>10–14</b>) with sulfonyl chloride but also corresponded to amides (<b>15–17</b>) with alkyl diamides protected by di-<i>tert</i>-butyl carbonate that could be removed in the presence of trifluoroacetic acid to obtain aminoamides (<b>18–20</b>).</div><figure id="sch1" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00018/20210602/images/medium/jm1c00018_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00018/20210602/images/large/jm1c00018_0007.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Structural Simplification of GM</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00018/20210602/images/large/jm1c00018_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00018&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="sch2" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00018/20210602/images/medium/jm1c00018_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00018/20210602/images/large/jm1c00018_0008.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of the GM Analogues (<b>1–20</b>)<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00018/20210602/images/large/jm1c00018_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00018&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) (i) NaOH, CH<sub>3</sub>OH/H<sub>2</sub>O, rt; (ii) NH<sub>4</sub>Cl; (b) (i) SOCl<sub>2</sub>, CH<sub>2</sub>Cl<sub>2</sub>, 45 °C, reflux; (ii) Et<sub>3</sub>N, pyridine, R<sub>1</sub>NH<sub>2</sub>, rt; (c) (i) 1-chloroethyl chloroformate, CHCl<sub>3</sub>, 100 °C, reflux; (ii) EtOH, 100 °C, reflux; (d) NCS, CH<sub>2</sub>Cl<sub>2</sub>, rt; (e) benzoyl chloride, Et<sub>3</sub>N, CH<sub>2</sub>Cl<sub>2</sub>, 0 °C to rt; (f) R<sub>2</sub>SO<sub>2</sub>Cl, Et<sub>3</sub>N, CH<sub>2</sub>Cl<sub>2</sub>, 0 °C to rt; (g) R<sub>3</sub>COOH, 1-ethyl-3-[3-(dimethylamino)propyl]carbodiimide hydrochloride (EDCI), 1-hydroxybenzotriazole (HOBt), DMAP, rt; (h) trifluoroacetic acid (TFA), CH<sub>2</sub>Cl<sub>2</sub>, 0 °C to rt.</p></p></figure><div class="NLM_p">Their effects on miRNA expression were examined, and all GM analogues significantly affected miR-21 biogenesis (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A). Three analogues (<b>3</b>, <b>7</b>, and <b>10</b>) had a potency similar to that of GM, and compound <b>9</b> was more potent than GM. All of the analogues did not show any chemical features of pan assay interference compounds (PAINS) by ZINC filter. Compounds <b>3</b>, <b>7</b>, <b>9</b>, and <b>10</b> decreased the intensity of the luciferase signal with IC<sub>50</sub> values of 21.91, 15.61, 12.96, and 19.48 μM, respectively (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00018/suppl_file/jm1c00018_si_001.pdf" class="ext-link">Figure S18</a>). Furthermore, the <i>K</i><sub>D</sub> values of compounds <b>3</b>, <b>7</b>, <b>9</b>, and <b>10</b> for TRBP by SPR were determined to be 17.97, 18.31, 6.79, and 24.08 μM, respectively (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00018/suppl_file/jm1c00018_si_001.pdf" class="ext-link">Figure S19</a>). GM analogues were used to examine their antiproliferative activity in SK-Hep-1 cells and QSG 7701 cells. Compounds <b>3</b>, <b>7</b>, <b>9</b>, and <b>10</b> had a similar or more potent inhibitory effect on cell proliferation than GM (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00018/suppl_file/jm1c00018_si_001.pdf" class="ext-link">Figure S20</a> and <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). The antiproliferative potency of compound <b>9</b> and GM at 20 μM was examined in MHCC97, HepG2, Huh7, and HeLa cells. As shown in <a href="/doi/suppl/10.1021/acs.jmedchem.1c00018/suppl_file/jm1c00018_si_001.pdf" class="ext-link">Figure S21</a>, compound <b>9</b> was more potent than GM in SK-Hep-1 and MHCC97 cells but with the same efficacy in HepG2, HeLa, and Huh7 cells. However, compounds exhibited low selectivity against tumor cells versus nontumor cells (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>).</div><figure id="fig6" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00018/20210602/images/medium/jm1c00018_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00018/20210602/images/large/jm1c00018_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Roles of GM analogues in anti-HCC. (A) Histogram showing the effect of 10 μM GM analogues on miR-21 biogenesis in HeLa-miR-21-luciferase cells. (B and C) Representative image showing the effect of <b>7</b> and <b>21</b> on the (B) proliferation, (C, top) migration, and (C, bottom) invasion of SK-Hep-1 cells at 20 μM. (D) Pull down assay showing the binding between <b>21</b> and TRBP protein.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00018/20210602/images/large/jm1c00018_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00018&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. IC<sub>50</sub> Values of Compounds against SK-Hep-1 and QSG-7701 Cells</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compound</th><th class="colsep0 rowsep0" align="center" char=".">SK-Hep-1 (μM)</th><th class="colsep0 rowsep0" align="center" char=".">QSG-7701 (μM)</th><th class="colsep0 rowsep0" align="center">compound</th><th class="colsep0 rowsep0" align="center" char=".">SK-Hep-1 (μM)</th><th class="colsep0 rowsep0" align="center" char=".">QSG-7701 (μM)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">enoxacin</td><td class="colsep0 rowsep0" align="char" char=".">25.03</td><td class="colsep0 rowsep0" align="char" char=".">72.35</td><td class="colsep0 rowsep0" align="left"><b>10</b></td><td class="colsep0 rowsep0" align="char" char=".">26.67</td><td class="colsep0 rowsep0" align="char" char=".">61.35</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">GM</td><td class="colsep0 rowsep0" align="char" char=".">20.56</td><td class="colsep0 rowsep0" align="char" char=".">52.98</td><td class="colsep0 rowsep0" align="left"><b>11</b></td><td class="colsep0 rowsep0" align="char" char=".">31.68</td><td class="colsep0 rowsep0" align="char" char=".">67.89</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1</b></td><td class="colsep0 rowsep0" align="char" char=".">29.89</td><td class="colsep0 rowsep0" align="char" char=".">68.54</td><td class="colsep0 rowsep0" align="left"><b>12</b></td><td class="colsep0 rowsep0" align="char" char=".">34.25</td><td class="colsep0 rowsep0" align="char" char=".">69.32</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>2</b></td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="char" char=".">>100.00</td><td class="colsep0 rowsep0" align="left"><b>13</b></td><td class="colsep0 rowsep0" align="char" char=".">28.65</td><td class="colsep0 rowsep0" align="char" char=".">62.35</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3</b></td><td class="colsep0 rowsep0" align="char" char=".">23.57</td><td class="colsep0 rowsep0" align="char" char=".">56.35</td><td class="colsep0 rowsep0" align="left"><b>14</b></td><td class="colsep0 rowsep0" align="char" char=".">46.98</td><td class="colsep0 rowsep0" align="char" char=".">75.86</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4</b></td><td class="colsep0 rowsep0" align="char" char=".">38.43</td><td class="colsep0 rowsep0" align="char" char=".">58.45</td><td class="colsep0 rowsep0" align="left"><b>15</b></td><td class="colsep0 rowsep0" align="char" char=".">39.65</td><td class="colsep0 rowsep0" align="char" char=".">63.25</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5</b></td><td class="colsep0 rowsep0" align="char" char=".">31.56</td><td class="colsep0 rowsep0" align="char" char=".">64.38</td><td class="colsep0 rowsep0" align="left"><b>16</b></td><td class="colsep0 rowsep0" align="char" char=".">89.65</td><td class="colsep0 rowsep0" align="char" char=".">>100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6</b></td><td class="colsep0 rowsep0" align="char" char=".">35.78</td><td class="colsep0 rowsep0" align="char" char=".">56.89</td><td class="colsep0 rowsep0" align="left"><b>17</b></td><td class="colsep0 rowsep0" align="char" char=".">40.91</td><td class="colsep0 rowsep0" align="char" char=".">59.47</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7</b></td><td class="colsep0 rowsep0" align="char" char=".">24.45</td><td class="colsep0 rowsep0" align="char" char=".">67.23</td><td class="colsep0 rowsep0" align="left"><b>18</b></td><td class="colsep0 rowsep0" align="char" char=".">>100.00</td><td class="colsep0 rowsep0" align="char" char=".">>100.00</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8</b></td><td class="colsep0 rowsep0" align="char" char=".">22.68</td><td class="colsep0 rowsep0" align="char" char=".">50.32</td><td class="colsep0 rowsep0" align="left"><b>19</b></td><td class="colsep0 rowsep0" align="char" char=".">>100.00</td><td class="colsep0 rowsep0" align="char" char=".">>100.00</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>9</b></td><td class="colsep0 rowsep0" align="char" char=".">19.43</td><td class="colsep0 rowsep0" align="char" char=".">48.35</td><td class="colsep0 rowsep0" align="left"><b>20</b></td><td class="colsep0 rowsep0" align="char" char=".">64.65</td><td class="colsep0 rowsep0" align="char" char=".">74.15</td></tr></tbody></table></div></div><div class="NLM_p">Upon comparing the biological activity of <b>1</b> with those of <b>2–6</b>, we found that the introduction of an amide at C-8 of ring A caused a significant reduction in the antiproliferative activity. Introduction of a chain amide group at C-8 of ring A caused a potency lower than that of <b>1</b>. Moreover, replacing the substituent at C-8 of ring A with a benzylcarbamyl resulted in a potency higher than that of <b>1</b>, reaching a level comparable to that observed for GM. In addition, introduction of phenylcarbamyl could increase the activities (<b>1</b> vs <b>4</b>), and the substituent position on the phenyl ring had no effect on the activity (<b>1</b> vs <b>5</b> and <b>6</b>). By comparing the effect of ring B, we found that removing a methyl from a tertiary amine increased antiproliferative activity as demonstrated by <b>7</b> versus <b>1</b>. The activity was further increased after chlorination at C-4 of ring B (<b>7</b> vs <b>8</b>). In addition, it was also favorable to improve the activity when the NH group of <b>7</b> was protected with benzoyl (<b>7</b> vs <b>9</b>). When benzoyl was replaced with similar sulfonic groups (<b>10–14</b>), the biological activity was reduced compared with that of <b>7</b>. Amino side chains were also introduced at the NH group of <b>7</b>, and deletion of the butyloxy carbonyl group resulted in decreased activity, as demonstrated by <b>15–17</b> compared with those of <b>18–20</b>. However, the activities of <b>15–20</b> remained lower than those of <b>7</b> with regard to antiproliferation.</div><div class="NLM_p">On the basis of the SAR of these analogues, compound <b>7</b> was chosen to label the biotin probe to obtain <b>21</b> (<a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>). The effect of <b>21</b> on miR-21 biogenesis was similar to that of compound <b>7</b> and GM (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00018/suppl_file/jm1c00018_si_001.pdf" class="ext-link">Figure S22</a>). The ability of the biotin-labeled compound to inhibit proliferation and metastasis was examined in SK-Hep-1 cells. Both <b>7</b> and <b>21</b> were found to effectively suppress the proliferation (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>B), invasion (top panel in <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>C), and migration (bottom panel in <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>C) of hepatoma cells. To further investigate the interaction between <b>21</b> and TRBP, the compound was used to pull down TRBP. As shown in <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>D, <b>21</b> was found to interact with TRBP and this binding could be competitively inhibited by <b>7</b> or GM, with the latter resulting in the strongest inhibition (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>D). Considering that enoxacin has been reported to target piwi-like protein 3 (PIWIL3) and topoisomerase 2β (TOP2β),<a onclick="showRef(event, 'ref44 ref45'); return false;" href="javascript:void(0);" class="ref ref44 ref45">(44,45)</a> compound <b>21</b> was used to pull down PIWIL3 and TOP2β. Our results showed that <b>21</b> did not bind those proteins (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00018/suppl_file/jm1c00018_si_001.pdf" class="ext-link">Figure S23</a>).</div><figure id="sch3" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00018/20210602/images/medium/jm1c00018_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00018/20210602/images/large/jm1c00018_0009.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthetic Route for Compound <b>21</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00018/20210602/images/large/jm1c00018_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00018&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i14">Discussion and Conclusion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_12073" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_12073" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">TRBP phosphorylation is mediated by the MAPK pathway.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> Interestingly, Gomisin A ameliorated metastatic melanoma by increasing the level of ERK activation. Supported by our observation that GM regulates the MAPK signaling pathway in miRNAome analysis, it is possible that GM may regulate the ERK/MAPK pathway and indirectly modulate TRBP activity for microRNA biogenesis; however, this requires further investigation. The function of TRBP can also be affected by its interacting proteins, including double-stranded RNA-specific adenosine deaminase (ADAR1),<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> PKR-associated protein X (PACT), and protein kinase R (PKR).<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> In addition, TRBP affects translation by binding to the structured 3′ untranslated region (UTR) of mRNAs,<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> and destabilization of metastasis suppressor transcripts could be affected by the binding of TRBP to mRNA hairpins.<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> Therefore, further research is needed to determine whether GM regulates the interaction between TRBP and other proteins, or a group of specific mRNAs, to jointly suppress HCC.</div><div class="NLM_p">Sorafenib is used widely in the treatment of advanced HCC; however, it is associated with adverse events and the development of acquired resistance.<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> Here, we found that GM could inhibit the growth of human liver cancer cells in orthotopically implanted mice with greater efficacy than enoxacin at the same dosage (20 mg/kg) and with an efficacy similar to that of sorafenib at 40 mg/kg. Enoxacin inhibits the growth of various cancers, including colorectal and breast cancers.<a onclick="showRef(event, 'ref11 ref44'); return false;" href="javascript:void(0);" class="ref ref11 ref44">(11,44)</a> In this study, we found that enoxacin also exhibits anticancer effects in HCC. In addition, TRBP overexpression suppressed the proliferation and migration of HCC cells. This was supported by the observation that enoxacin suppressed HCC tumorigenesis <i>in vivo</i>, consistent with the report that enoxacin activated TRBP in colorectal cancer.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> GM was found to suppress the proliferation of xenografted liver cancer cells <i>in vivo</i>, which was also consistent with the observation that GM inhibited hepatocarcinogenesis induced by 3′-methyl-4-dimethylamino-azobenzene in rats.<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> Our data for the SAR of GM indicated that dibenzocyclooctadiene lignans may be valuable for further development as novel TRBP modulators in the treatment of HCC. Consistent with proteomics analysis results, we found that GM also suppressed the expression of KIF, TPX2, UBE2S, and CDC20 <i>in vivo</i>. Downregulation of Dicer by hypoxia is an essential process in HCC progression.<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> In contrast, staphylococcal nuclease domain containing 1 confers significantly higher RISC activity in HCC by interacting with Ago2.<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> Here, we found that GM promoted the interaction between TRBP and Dicer but had no effect on the interaction between TRBP and Ago2, suggesting that GM regulates the function of Dicer to exert its anticancer effect in HCC. A limitation of targeted therapies is drug resistance, which may be obviated by the development of drugs that act on multiple targets.<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> Considering the efficacy of GM in the remodeling of a subset of miRNAs and tumor-related proteins, targeting TRBP by GM or its analogues may be a novel strategy for inhibiting HCC tumorigenesis and progression.</div><div class="NLM_p">We observed a significant alteration with only a small subset of miRNAs, including 36 that were upregulated and 33 that were downregulated. This is consistent with previous studies showing that the knockdown or mutation of TRBP resulted in a subset, rather than a global change, of differentially expressed miRNA.<a onclick="showRef(event, 'ref47 ref55'); return false;" href="javascript:void(0);" class="ref ref47 ref55">(47,55)</a> S6K2-mediated regulation of TRBP affects miRNA expression with different effects in human primary lymphatic endothelial cells.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> Furthermore, enoxacin was also observed to differently affect miRNAs by targeting TRBP in HEK293 cells, colorectal cancer cells, and prostate cancer cells.<a onclick="showRef(event, 'ref11 ref56 ref57'); return false;" href="javascript:void(0);" class="ref ref11 ref56 ref57">(11,56,57)</a> Therefore, it is possible that the expression level and variety of pre-miRNAs, their secondary structures, and post-translational modification of the RISC element may contribute to the effect of TRBP modulated by small molecules on the expression of specific miRNAs expression. The pathways enriched by the 69 miRNAs included the Ras, tumor necrosis factor (TNF), PI3K-Akt, Janus kinase/signal transducer and activator of tran-ions (JAK/STAT), MAPK, etc., indicating that these are involved in the anticancer mechanisms of GM in HCC. This is consistent with the observation that these pathways contribute to HCC tumorigenesis.<a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(58)</a> Proteomics analysis revealed that proteins related to cell division were enriched among those downregulated by GM, which is supported by our observation that GM suppresses HCC cell proliferation and migration <i>in vitro</i> and <i>in vivo.</i> This is because most proteins were involved in the proliferation and metastasis in HCC or other cancers, especially the kinesin superfamily, including KIF4A, KIF14, and KIF23. Moreover, many proteins associated with antigen processing and presentation were also enriched, including human leukocyte antigen A (HLA-A), human leukocyte antigen C (HLA-C), transporter 1 (TAP1), and TAP-binding protein (TAPBP). These proteins participate in the lysis of target cells by cytotoxic T-lymphocytes following recognition of peptide and MHC class I molecule complexes on cell surfaces. Therefore, GM may enhance tumor immunity in HCC, and this warrants further investigation. Recently, 628 therapeutic targets with potential for anticancer drug development were identified by genome-scale CRISPR-Cas9 screens in 324 human cancer models,<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(59)</a> in which 10 proteins were found to be regulated by GM in HCC cells, including TPX2, sperm-associated antigen 5 (SPAG5), solute carrier family 38 member 2 (SLC38A2), KIF14, F-box and WD repeat domain containing 11 (FBXW11), DNA methyltransferase 1 (DNMT1), carboxypeptidase D (CPD), UBE2S, cyclin-dependent kinase 2 (CDK2), and alanyl-TRNA synthetase 1 (AARS), further explaining the anticancer mechanism of GM and promoting the development of GM as a novel therapeutic candidate for cancer treatment.</div><div class="NLM_p">TRBP was initially found to bind the stem–loop transactivation response RNA region of long terminal repeats on both 5′- and 3′-ends to promote the translation of HIV-1.<a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a> GM, Gomisin G, and Gomisin J have been found to exhibit anti-HIV activity.<a onclick="showRef(event, 'ref24 ref61 ref62'); return false;" href="javascript:void(0);" class="ref ref24 ref61 ref62">(24,61,62)</a> However, the underlying mechanism remains unknown. Here, we found that GM physically interacts with TRBP and modulates its function, suggesting that GM and its analogues exert their anti-HIV activity by targeting TRBP. This is consistent with previous reports that enoxacin, a validated TRBP agonist, also protected infected cells from HIV-1-mediated cytotoxicity.<a onclick="showRef(event, 'ref63'); return false;" href="javascript:void(0);" class="ref ref63">(63)</a> In this study, we found that GM potently suppressed HCC cell proliferation, consistent with our observation that GM binds more potently to TRBP than enoxacin. We also found that GM promoted the binding of TRBP to Dicer, consistent with reports that TRBP modulates the activity of Dicer.<a onclick="showRef(event, 'ref11 ref57'); return false;" href="javascript:void(0);" class="ref ref11 ref57">(11,57)</a> Enoxacin interacts with TRBP,<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> PIWIL3,<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> and topoisomerase 2β;<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> however, in our study, <b>21</b> could not pull down PIWIL3, indicating that <b>21</b> exhibited specificity greater than that of enoxacin. This is supported by the observation that GM did not further suppress the proliferation or migration of SK-Hep-1 cells with TRBP knockdown, suggesting that GM has minimal off-target effects.</div><div class="NLM_p last">In summary, we found that GM and its analogues target TRBP to modulate miRNA biogenesis and alter the expression of a subset of miRNAs and cancer-related pathways, resulting in inhibition of HCC proliferation and metastasis. The findings of this study highlight a previously unrecognized role of TRBP in HCC. TRBP may be a potential novel therapeutic target in HCC, and dibenzocyclooctadiene lignans warrant further investigation as potential drug candidates for the treatment of patients with HCC.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i15">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_43347" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_43347" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Chemical Information</h3><div class="NLM_p last">All reagents and solvents were obtained from commercial suppliers and used as received unless otherwise stated. DMSO, dimethylformamide (DMF), dichloromethane (DCM), 1,2-dichloroethane (DCE), and chloroform were distilled with molecular sieves (4 Å). All air sensitive reactions were carried out under an inert atmosphere (nitrogen gas). Purification was carried out by column chromatography on silica gel. Thin-layer chromatography (TLC) was carried out using commercially available aluminum sheets precoated with silica gel with a fluorescence indicator and visualized under ultraviolet light at 254 nm. Mass spectra were recorded on an LTQOrbitrapXL mass spectrometer (ThermoFisher). <sup>1</sup>H and <sup>13</sup>C NMR spectra were recorded on either 300 or 400 MHz instruments. <sup>13</sup>C NMR data are reported with two decimals when recorded on an instrument with a cryoprobe. Chemical shift values are quoted in parts per million, and coupling constants (<i>J</i>) in hertz. <sup>1</sup>H and <sup>13</sup>C NMR spectra were referenced against residual solvent peaks. Samples for electrospray ionization (ESI) mass spectrometry were dissolved in MeOH, which was used as the carrier solvent. A high-performance liquid chromatography system (HPLC; Agilent 1260 Infinity II) assembled with a diode array detector was used to separate, identify, and quantify the compounds, and their purities were ≥95%. An AichromBond-AQ C18, 4.6 mm × 250 mm RP-C18 column was used as a stationary phase for chromatographic separation. Active compounds were checked for chemical features responsible for possible unreliable activity with patterns defined as PAINS by online filter ZINC (<a href="http://zinc15.docking.org/patterns/home/" class="extLink">http://zinc15.docking.org/patterns/home/</a>).</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> General Procedure A. General Procedure for Amidation by 6-[4-(Dimethylamino)phenyl]-7-methoxybenzo[<i>d</i>][1,3]dioxole-5-carboxylic acid (<b>1-1</b>) and R<sub>1</sub>NH<sub>2</sub></h3><div class="NLM_p last">Solid EDCI (1 equiv) was added in parts to the solution of 6-[4-(dimethylamino)phenyl]-7-methoxybenzo[<i>d</i>][1,3]dioxole-5-carboxylic acid (<b>1</b>-<b>1</b>) (63.00 mg, 0.2 mmol) in DMF (8 mL) at room temperature. The mixture was further stirred for 45 min, and then the R<sub>1</sub>NH<sub>2</sub> (1 equiv) and DMAP (0.1 equiv) were added. After 8 h, the mixture was then poured into water (10 mL) and extracted with EtOAc (3 × 5 mL). The combined organics were washed with brine (3 × 5 mL), dried with MgSO<sub>4</sub>, and filtered, and the solvents were removed <i>in vacuo</i>. A simple column wash could give pure product.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> General Procedure B. General Procedure for Amidation by Methyl 7-Methoxy-6-[4-(methylamino)phenyl]benzo[<i>d</i>][1,3]dioxole-5-carboxylate (<b>7</b>) and Sulfochlorides</h3><div class="NLM_p last">Et<sub>3</sub>N (2.5 equiv) was added in parts to the solution of methyl 7-methoxy-6-[4-(methylamino)phenyl]benzo[<i>d</i>][1,3]dioxole-5-carboxylate (<b>7</b>) (63.00 mg, 0.2 mmol) in DCM (1 mL) at 0 °C. The mixture was further stirred for 10 min, and then R<sub>2</sub>SO<sub>2</sub>Cl (1.1 equiv) was added. Then the mixture was further stirred overnight at room temperature. After completion of the reaction, the mixture was neutralized with saturated aqueous NH<sub>4</sub>Cl, poured into water (10 mL), and extracted with DCM (3 × 5 mL). The combined organics were washed with brine (3 × 5 mL), dried with MgSO<sub>4</sub>, and filtered, and the solvents were removed <i>in vacuo</i>. A simple column wash could give pure product.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> General Procedure C. General Procedure for Amidation by Methyl 7-Methoxy-6-[4-(methylamino)phenyl]benzo[<i>d</i>][1,3]dioxole-5-carboxylate (<b>7</b>) and R<sub>3</sub>COOH</h3><div class="NLM_p last">Solid EDCI (1.5 equiv) was added in parts to the solution of R<sub>3</sub>COOH (1 equiv) in DMF (2 mL) at room temperature. The mixture was further stirred for 45 min, and then the 7-methoxy-6-[4-(methylamino)phenyl]benzo[<i>d</i>][1,3]dioxole-5-carboxylate (<b>7</b>) (63.00 mg, 0.2 mmol) and DMAP (0.1 equiv) were added. After 8 h, the mixture was then poured into water (10 mL) and extracted with EtOAc (3 × 5 mL). The combined organics were washed with brine (3 × 5 mL), dried with MgSO<sub>4</sub>, and filtered, and the solvents were removed <i>in vacuo</i>. A simple column wash could give pure product.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> General Procedure D. General Procedure for Removal of the Boc Protective Group</h3><div class="NLM_p last">CF<sub>3</sub>COOH (1.5 equiv) was added drop by drop to the solution of compounds <b>18–20</b> (0.1 mmol) in DCM (1 mL) at room temperature. The mixture was further stirred for 3 h, then neutralized with saturated aqueous NaHCO<sub>3</sub>, poured into water (10 mL), and extracted with DCM (3 × 5 mL). The combined organics were washed with brine (3 × 5 mL), dried with MgSO<sub>4</sub>, and filtered, and the solvents were removed <i>in vacuo</i>. A simple column wash could give pure product.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> Methyl 6-[4-(Dimethylamino)phenyl]-7-methoxybenzo[<i>d</i>][1,3]dioxole-5-carboxylate (<b>1</b>)</h3><div class="NLM_p last">Compound <b>1</b> was prepared according to previous research. Briefly, the methyl 7-hydroxybenzo[<i>d</i>][1,3]dioxole-5-carboxylate was prepared by the Williamson method in the presence of potassium carbonate from commercially available methyl 3,4,5-trihydroxybenzoate. The bromination reaction took place with the addition of dibromohydantoin to yield methyl 6-bromo-7-hydroxybenzo[<i>d</i>][1,3]dioxole-5-carboxylate, which was protected by methylation to provide methyl 6-bromo-7-methoxybenzo[<i>d</i>][1,3]dioxole-5-carboxylate.<a onclick="showRef(event, 'ref64 ref65'); return false;" href="javascript:void(0);" class="ref ref64 ref65">(64,65)</a> Then the methyl 6-[4-(dimethylamino)phenyl]-7-methoxybenzo[<i>d</i>][1,3]dioxole-5-carboxylate (<b>1</b>) (60% yield) could be obtained by a Suzuki reaction.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a><sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>COCD<sub>3</sub>): δ 7.02–7.00 (2H, d, <i>J</i> = 8.8 Hz), 6.92 (1H, s), 6.73–6.71 (2H, d, <i>J</i> = 8.8 Hz), 6.10 (2H, s), 3.73 (3H, s), 3.48 (3H, s), 2.95 (6H, s). <sup>13</sup>C NMR (100 MHz, CD<sub>3</sub>COCD<sub>3</sub>): δ 167.92, 149.70, 147.82, 141.35, 140.05, 130.55, 130.27, 126.51, 124.16, 111.50, 103.66, 102.06, 59.23, 51.04, 39.7. HRMS-ESI: <i>m</i>/<i>z</i> calcd for C<sub>18</sub>H<sub>20</sub>O<sub>5</sub> [(M + H)<sup>+</sup>] 330.1336, found 330.1334.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> 6-[4-(Dimethylamino)phenyl]-7-methoxybenzo[<i>d</i>][1,3]dioxole-5-carboxylic acid (<b>1-1</b>)</h3><div class="NLM_p last">According to previous research, compound <b>1</b> (650.0 mg, 2 mmol) was subjected to hydrolysis in the presence of sodium hydroxide to afford key intermediate 6-[4-(dimethylamino)phenyl]-7-methoxybenzo[<i>d</i>][1,3]dioxole-5-carboxylic acid <b>1</b>-<b>1</b> (540.0 mg, 85%) as a yellow solid.<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a><sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.34 (1H, s), 7.00–6.97 (2H, d, <i>J</i> = 8.4 Hz), 6.94 (1H, s), 6.70–6.67 (2H, d, <i>J</i> = 8.5 Hz), 6.09 (2H, s), 3.68 (3H, s), 2.91 (6H, s). <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>): δ 168.71, 149.17, 147.24, 140.76, 139.27, 130.17, 129.37, 127.29, 123.70, 111.44, 103.44, 101.81, 59.52, 39.44. HRMS-ESI: <i>m</i>/<i>z</i> calcd for C<sub>17</sub>H<sub>18</sub>NO<sub>5</sub> [(M + H)<sup>+</sup>] 316.1179, found 387.1176.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> <i>N</i>-[2-(Diethylamino)ethyl]-6-[4-(dimethylamino)phenyl]-7-methoxybenzo[<i>d</i>][1,3]dioxole-5-carboxamide (<b>2</b>)</h3><div class="NLM_p last">Using general procedure A, compound <b>2</b> (60.00 mg) was obtained from compound <b>1</b>-<b>1</b> in 72% yield as a yellow solid. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 7.18–7.15 (2H, d, <i>J</i> = 8.7 Hz), 6.97 (1H, s), 6.74–6.71 (2H, d, <i>J</i> = 8.7 Hz), 6.00 (2H, s), 5.60 (1H, s), 3.76 (3H, s), 3.13–3.10 (2H, q, <i>J</i> = 5.1 Hz), 2.97 (6H, s), 2.35–2.28 (4H, q, <i>J</i> = 7.2 Hz), 2.18–2.14 (2H, t, <i>J</i> = 6.5 Hz), 0.88–0.83 (6H, t, <i>J</i> = 7.2 Hz). <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>): δ 168.94, 149.91, 148.18, 141.02, 138.94, 131.22, 130.87, 127.46, 123.18, 112.26, 103.97, 101.64, 60.09, 51.12, 46.23, 40.52, 37.53, 11.35. HRMS-ESI: <i>m</i>/<i>z</i> calcd for C<sub>23</sub>H<sub>32</sub>N<sub>3</sub>O<sub>4</sub> [(M + H)<sup>+</sup>] 414.2387, found 414.2387.</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> <i>N</i>-Benzyl-6-[4-(dimethylamino)phenyl]-7-methoxybenzo[<i>d</i>][1,3]dioxole-5-carboxamide (<b>3</b>)</h3><div class="NLM_p last">Using general procedure A, compound <b>3</b> (69.00 mg) was obtained from compound <b>1</b>-<b>1</b> in 85% yield as a white solid. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 7.18–7.16 (3H, m), 7.12 (1H, s), 7.09–7.08 (1H, m), 6.86–6.85 (2H, m), 6.67–6.64 (2H, d, <i>J</i> = 8.7 Hz), 5.99 (2H, s), 5.43 (1H, s), 4.24–4.22 (2H, d, <i>J</i> = 5.1 Hz), 3.74 (3H, s), 3.97 (6H, s). <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>): δ 168.43, 149.99, 148.29, 141.06, 139.13, 137.64, 130.79, 130.32, 128.42, 127.94, 127.69, 127.28, 122.83, 112.32, 104.25, 101.70, 60.08, 44.44, 40.45. HRMS-ESI: <i>m</i>/<i>z</i> calcd for C<sub>24</sub>H<sub>25</sub>N<sub>2</sub>O<sub>4</sub> [(M + H)<sup>+</sup>] 405.1809, found 405.1809.</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> 6-[4-(Dimethylamino)phenyl]-7-methoxy-<i>N</i>-phenylbenzo[<i>d</i>][1,3]dioxole-5-carboxamide (<b>4</b>)</h3><div class="NLM_p last">Using general procedure A, compound <b>4</b> (58.00 mg) was obtained from compound <b>5</b> in 74% yield as a white solid. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 7.24–7.15 (5H, m), 7.03–6.99 (4H, m), 7.80–7.77 (2H, d, <i>J</i> = 8.7 Hz), 6.03 (2H, s), 3.79 (3H, s), 2.98 (6H, s). <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>): δ 166.10, 150.46, 148.52, 141.02, 139.64, 137.99, 131.06, 130.03, 128.70, 127.89, 124.05, 122.55, 120.11, 112.90, 104.91, 101.85, 60.14, 40.61. HRMS-ESI: <i>m</i>/<i>z</i> calcd for C<sub>23</sub>H<sub>23</sub>N<sub>2</sub>O<sub>4</sub> [(M + H)<sup>+</sup>] 391.1652, found 391.1650.</div></div><div id="sec4_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> 6-[4-(Dimethylamino)phenyl]-7-methoxy-<i>N</i>-(3-methoxyphenyl)benzo[<i>d</i>][1,3]dioxole-5-carboxamide (<b>5</b>)</h3><div class="NLM_p last">Using general procedure A, compound <b>5</b> (67.00 mg) was obtained from compound <b>1</b>-<b>1</b> in 80% yield as a yellow solid. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 7.24–7.21 (2H, d, <i>J</i> = 8.7 Hz), 7.06–7.01 (2H, m), 6.81–6.78 (2H, d, <i>J</i> = 8.7 Hz), 6.68 (1H, s), 6.57–6.54 (2H, m), 6.04 (2H, s), 3.80 (3H, s), 3.71 (3H, s), 2.98 (6H, s). <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>): δ 165.97, 159.94, 150.40, 148.57, 141.03, 139.70, 139.16, 131.12, 129.90, 129.37, 127.93, 122.50, 112.83, 112.36, 110.49, 105.23, 104.97, 101.88, 60.16, 55.19, 40.52. HRMS-ESI: <i>m</i>/<i>z</i> calcd for C<sub>24</sub>H<sub>25</sub>N<sub>2</sub>O<sub>5</sub> [(M + H)<sup>+</sup>] 421.1758, found 421.1757.</div></div><div id="sec4_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> 6-[4-(Dimethylamino)phenyl]-<i>N</i>-(3-iodophenyl)-7-methoxybenzo[<i>d</i>][1,3]dioxole-5-carboxamide (<b>6</b>)</h3><div class="NLM_p last">Using general procedure A, compound <b>6</b> (46.00 mg) was obtained from compound <b>5</b> in 45% yield as a yellow solid. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 8.14–8.11 (1H, d, <i>J</i> = 6.9 Hz), 7.65–7.63 (1H, d, <i>J</i> = 7.8 Hz), 7.29–7.24 (4H, m), 7.05 (1H, s), 6.78–6.73 (1H, t, <i>J</i> = 7.5 Hz), 6.71–6.68 (2H, d, <i>J</i> = 8.4 Hz), 6.04 (2H, s), 3.80 (3H, s), 2.91 (6H, s). <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>): δ 167.39, 150.33, 148.33, 141.19, 139.65, 138.94, 131.31, 130.63, 129.02, 127.98, 125.79, 122.18, 122.00, 112.59, 104.21, 101.86, 89.85, 60.13, 40.58. HRMS-ESI: <i>m</i>/<i>z</i> calcd for C<sub>23</sub>H<sub>22</sub>IN<sub>2</sub>O<sub>4</sub> [(M + H)<sup>+</sup>] 517.0619, found 517.0618.</div></div><div id="sec4_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> Methyl 7-Methoxy-6-[4-(methylamino)phenyl]benzo[<i>d</i>][1,3]dioxole-5-carboxylate (<b>7</b>)</h3><div class="NLM_p last">To a solution of methyl 6-[4-(dimethylamino)phenyl]-7-methoxybenzo[<i>d</i>][1,3]dioxole-5-carboxylate (<b>1</b>) (330.0 mg, 1 mmol) in DCE (6 mL) was added α-chloroethyl chloroformate (ACE-Cl) (0.65 mL, 6 mmol) in small portions at 0 °C with stirring. The mixture was further stirred for 15 min, then heated to 100 °C, and refluxed for 36 h. After the mixture had cooled, the solvents were evaporated <i>in vacuo</i>. The residue was dissolved in 3 mL of DCE; to the solution was then added 6 mL of EtOH, and the mixture was heated to 100 °C and refluxed overnight. After completion of the reaction, the mixture was allowed to cool to room temperature, then neutralized with 2 N NaOH, poured into water (150 mL), and extracted with DCM (3 × 50 mL). The combined organics were washed with brine (3 × 50 mL), dried with MgSO<sub>4</sub>, and filtered, and the solvents were removed <i>in vacuo</i>. The residue was purified by flash column chromatography (EtOAc/<i>n</i>-hexane = 1/10) to afford compound <b>7</b> (140.0 mg, 45%) as a yellow solid. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 6.98–6.95 (2H, d, <i>J</i> = 8.4 Hz), 6.95 (1H, s), 6.56–6.54 (2H, d, <i>J</i> = 8.7 Hz), 5.96 (2H, s), 3.69 (3H, s), 3.49 (3H, s), 2.79 (3H, s). <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>): δ 168.36, 148.28, 147.68, 141.40, 140.13, 131.19, 130.32, 125.80, 125.02, 111.76, 104.39, 101.73, 59.99, 51.89, 30.68. HRMS-ESI: <i>m</i>/<i>z</i> calcd for C<sub>17</sub>H<sub>18</sub>NO<sub>5</sub> [(M + H)<sup>+</sup>] 316.1179, found 316.1179.</div></div><div id="sec4_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> Methyl 6-[3-Chloro-4-(methylamino)phenyl]-7-methoxybenzo[<i>d</i>][1,3]dioxole-5-carboxylate (<b>8</b>)</h3><div class="NLM_p last">To a solution of methyl 7-methoxy-6-[4-(methylamino)phenyl]benzo[<i>d</i>][1,3]dioxole-5-carboxylate (<b>7</b>) (63.00 mg, 0.2 mmol) in DCM (1 mL) was added <i>N</i>-chlorosuccinimide (NCS) (41.00 mg, 0.3 mmol) under a N<sub>2</sub> atmosphere at room temperature. After 1 h, the reaction was quenched by NaHCO<sub>3</sub>, and the mixture poured into water (10 mL) and extracted with DCM (3 × 5 mL). The combined organics were washed with brine (3 × 5 mL), dried with MgSO<sub>4</sub>, and filtered, and the solvents were removed <i>in vacuo</i>. The residue was purified by flash column chromatography (EtOAc/<i>n</i>-hexane = 1/10) to afford compound <b>8</b> (44.10 mg, 70%) as a yellow solid. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 7.13 (1H, s), 7.03–6.99 (2H, m), 6.66–6.63 (1H, d, <i>J</i> = 8.1 Hz), 6.02 (2H, s), 4.37 (1H, s), 3.79 (3H, s), 3.79 (3H, s), 2.92 (3H, s). <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>): δ 168.08, 148.16, 144.06, 141.54, 140.22, 130.10, 130.00, 129.25, 125.87, 129.25, 118.48, 109.98, 104.60, 101.95, 60.12, 52.04, 30.50. HRMS-ESI: <i>m</i>/<i>z</i> calcd for C<sub>17</sub>H<sub>17</sub>ClNO<sub>5</sub> [(M + H)<sup>+</sup>] 350.0790, found 350.0790.</div></div><div id="sec4_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> Methyl 7-Methoxy-6-[4-(<i>N</i>-methylbenzamido)phenyl]benzo[<i>d</i>][1,3]dioxole-5-carboxylate (<b>9</b>)</h3><div class="NLM_p last">Et<sub>3</sub>N (0.068 mL, 0.5 mmol) was added in parts to the solution of methyl 7-methoxy-6-[4-(methylamino)phenyl]benzo[<i>d</i>][1,3]dioxole-5-carboxylate (<b>7</b>) (63.00 mg, 0.2 mmol) in DCM (1 mL) at 0 °C. The mixture was further stirred for 10 min, and then benzoyl chloride (0.026 mL, 0.22 mmol) was added. Then the mixture was further stirred overnight at room temperature. After completion of the reaction, the mixture was neutralized with saturated aqueous NH<sub>4</sub>Cl, poured into water (10 mL), and extracted with DCM (3 × 5 mL). The combined organics were washed with brine (3 × 5 mL), dried with MgSO<sub>4</sub>, and filtered, and the solvents were removed <i>in vacuo</i>. The residue was purified by flash column chromatography (EtOAc/<i>n</i>-hexane = 1/10) to afford compound <b>9</b> (59.00 mg, 70%) as a yellow solid. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 7.29–7.27 (2H, d, <i>J</i> = 6.3 Hz), 7.26 (1H, s), 7.16–7.11 (3H, m), 6.98–6.97 (5H, m), 5.95 (2H, s), 3.6 (3H, s), 3.47 (3H, s), 3.35 (3H, s). <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>): δ 170.76, 167.48, 148.47, 143.55, 141.01, 140.36, 135.20, 130.35, 129.92, 129.52, 128.87, 127.75, 126.18, 125.24, 104.77, 102.07, 59.98, 51.84, 38.43. HRMS-ESI: <i>m</i>/<i>z</i> calcd for C<sub>24</sub>H<sub>22</sub>NO<sub>6</sub> [(M + H)<sup>+</sup>] 420.1442, found 420.1443.</div></div><div id="sec4_16" class="NLM_sec NLM_sec_level_2"><div id="ac_i31" class="anchor-spacer"></div><h3 class="article-section__title" id="_i31"> Methyl 7-Methoxy-6-[4-(<i>N</i>-methylphenylsulfonamido)phenyl]benzo[<i>d</i>][1,3]dioxole-5-carboxylate (<b>10</b>)</h3><div class="NLM_p last">Using general procedure B, compound <b>10</b> (69.00 mg) was obtained from compound <b>7</b> in 78% yield as a white solid. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 7.58–7.55 (3H, m), 7.48–7.46 (2H, d, <i>J</i> = 7.2 Hz), 7.11–7.10 (5H, m), 6.06 (2H, s), 3.78 (3H, s), 3.56 (3H, s), 3.22 (3H, s). <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>): δ 148.58, 140.21, 136.06, 132.89, 130.10, 128.79, 128.04, 125.69, 125.45, 104.90, 102.13, 60.14, 51.95, 38.11. HRMS-ESI: <i>m</i>/<i>z</i> calcd for C<sub>23</sub>H<sub>22</sub>NO<sub>7</sub>S [(M + H)<sup>+</sup>] 456.1111, found 456.1110.</div></div><div id="sec4_17" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> Methyl-6-{4-[(4-fluoro-<i>N</i>-methylphenyl)sulfonamido]phenyl}-7-methoxybenzo[<i>d</i>][1,3]dioxole-5-carboxylate (<b>11</b>)</h3><div class="NLM_p last">Using general procedure B, compound <b>11</b> (63.00 mg) was obtained from compound <b>7</b> in 77% yield as a yellow solid. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 7.58–7.53 (2H, m), 7.16–7.10 (7H, m), 6.06 (2H, s), 3.79 (3H, s), 3.58 (3H, s), 3.22 (3H, s). <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>): δ 167.32, 148.60, 141.11, 140.31, 139.99, 136.27, 132.37, 130.76, 130.64, 130.17, 125.60, 125.29, 116.17, 115.87, 104.89, 102.13, 60.09, 51.90, 38.03. HRMS-ESI: <i>m</i>/<i>z</i> calcd for C<sub>23</sub>H<sub>21</sub>FNO<sub>7</sub>S [(M + H)<sup>+</sup>] 474.1017, found 474.1017.</div></div><div id="sec4_18" class="NLM_sec NLM_sec_level_2"><div id="ac_i33" class="anchor-spacer"></div><h3 class="article-section__title" id="_i33"> Methyl 7-Methoxy-6-{4-[(<i>N</i>-methyl-4-nitrophenyl)sulfonamido]phenyl}benzo[<i>d</i>][1,3]dioxole-5-carboxylate (<b>12</b>)</h3><div class="NLM_p last">Using general procedure B, compound <b>12</b> (63.00 mg) was obtained from compound <b>7</b> in 63% yield as a yellow solid. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 8.31–8.28 (2H, d, <i>J</i> = 8.7 Hz), 7.71–6.68 (2H, d, <i>J</i> = 8.7 Hz), 7.13–7.09 (5H, m), 6.07 (2H, s), 3.81 (3H, s), 3.63 (3H, s), 3.27 (3H, s). <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>): δ 166.97, 150.23, 148.68, 141.82, 141.09, 140.29, 139.27, 136.87, 130.41, 129.25, 125,47, 124.03, 104.96, 102.16, 60.10, 51.98, 38.16. HRMS-ESI: <i>m</i>/<i>z</i> calcd for C<sub>23</sub>H<sub>21</sub>N<sub>2</sub>O<sub>9</sub>S [(M + H)<sup>+</sup>] 501.0962, found 501.0960.</div></div><div id="sec4_19" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34"> Methyl 6-{4-[(4-Bromo-<i>N</i>-methylphenyl)sulfonamido]phenyl}-7-methoxybenzo[<i>d</i>][1,3]dioxole-5-carboxylate (<b>13</b>)</h3><div class="NLM_p last">Using general procedure B, compound <b>7</b> (93.00 mg) was obtained from compound <b>7</b> in 87% yield as a yellow solid. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 7.60–7.58 (2H, d, <i>J</i> = 8.7 Hz), 7.40–7.38 (1H, d, <i>J</i> = 8.4 Hz), 7.12–7.10 (5H, m), 6.07 (2H, s), 3.79 (3H, s), 3.58 (3H, s), 3.22 (3H, s). <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>): δ 167.35, 148.63, 141.13, 140.32, 139.88, 136.36, 135.29, 132.09, 130.23, 129.56, 127.95, 125.61, 125.33, 104.92, 102.14, 60.13, 51.96, 38.08. HRMS-ESI: <i>m</i>/<i>z</i> calcd for C<sub>23</sub>H<sub>21</sub>BrNO<sub>7</sub>S [(M + H)<sup>+</sup>] 534.0217, found 534.0216.</div></div><div id="sec4_20" class="NLM_sec NLM_sec_level_2"><div id="ac_i35" class="anchor-spacer"></div><h3 class="article-section__title" id="_i35"> Methyl 7-Methoxy-6-[4-(<i>N</i>-methylmethylsulfonamido)phenyl]benzo[<i>d</i>][1,3]dioxole-5-carboxylate (<b>14</b>)</h3><div class="NLM_p last">Using general procedure B, compound <b>14</b> (75.00 mg) was obtained from compound <b>7</b> in 95% yield as a white solid. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 7.40–7.37 (2H, d, <i>J</i> = 8.4 Hz), 7.23–7.20 (2H, d, <i>J</i> = 8.4 Hz), 7.11 (1H, s), 6.06 (2H, s), 3.80 (3H, s), 3.52 (3H, s), 3.37 (3H, s), 2.87 (3H, s). <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>): δ 167.46, 148.52, 141.04, 140.08, 135.97, 130.45, 129.77, 125.31, 125.06, 104.72, 102.00, 60.02, 51.86, 38.05, 35.20. HRMS-ESI: <i>m</i>/<i>z</i> calcd for C<sub>18</sub>H<sub>20</sub>NO<sub>7</sub>S [(M + H)<sup>+</sup>] 394.0955, found 394.0954.</div></div><div id="sec4_21" class="NLM_sec NLM_sec_level_2"><div id="ac_i36" class="anchor-spacer"></div><h3 class="article-section__title" id="_i36"> Methyl 6-(4-{2-[(<i>tert</i>-Butoxycarbonyl)amino]-<i>N</i>-methylacetamido}phenyl)-7-methoxybenzo[<i>d</i>][1,3]dioxole-5-carboxylate (<b>15</b>)</h3><div class="NLM_p last">Using general procedure C, compound <b>15</b> (63.00 mg) was obtained from compound <b>7</b> in 83% yield as a white solid. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 7.24–7.22 (4H, m), 7.12 (1H, s), 6.07 (2H, s), 5.43 (1H, s), 3.83 (3H, s), 3.72–3.71 (2H, d, <i>J</i> = 3.9 Hz), 3.54 (3H, s), 3.34 (3H, s), 1.42 (9H, s). <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>): δ 168.75, 167.54, 155.79, 148.67, 140.99, 140.61, 140.06, 137.48, 131.28, 129.39, 126.35, 125.41, 104.78, 102.04, 79.43, 59.98, 51.97, 43.07, 37.54, 28.35. HRMS-ESI: <i>m</i>/<i>z</i> calcd for C<sub>24</sub>H<sub>29</sub>N<sub>2</sub>O<sub>8</sub> [(M + H)<sup>+</sup>] 473.1918, found 473.1916.</div></div><div id="sec4_22" class="NLM_sec NLM_sec_level_2"><div id="ac_i37" class="anchor-spacer"></div><h3 class="article-section__title" id="_i37"> Methyl 6-(4-{2-[(<i>tert</i>-Butoxycarbonyl)(methyl)amino]-<i>N</i>-methylacetamido}phenyl)-7-methoxybenzo[<i>d</i>][1,3]dioxole-5-carboxylate (<b>16</b>)</h3><div class="NLM_p last">Using general procedure C, compound <b>16</b> (83.00 mg) was obtained from compound <b>7</b> in 57% yield as a yellow solid. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 7.28–7.19 (4H, m), 7.13 (1H, s), 6.07 (2H, s), 3.81 (4H, s), 3.72 (1H, s), 3.53 (3H, s), 3.32 (3H, s), 2.90 (3H, s), 1.47 (6H, s), 1.45 (3H, s). <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>): δ 168.88, 168.58, 167.43, 156.57, 148.72, 141.45, 141.15, 140.26, 137.19, 131.26, 129.84, 126.62, 126.37, 125.37, 104.92, 102.13, 79.80, 60.09, 51.91, 51.56, 51.06, 37.57, 36.00, 28.51. HRMS-ESI: <i>m</i>/<i>z</i> calcd for C<sub>25</sub>H<sub>31</sub>N<sub>2</sub>O<sub>8</sub> [(M + H)<sup>+</sup>] 487.2075, found 487.2075.</div></div><div id="sec4_23" class="NLM_sec NLM_sec_level_2"><div id="ac_i38" class="anchor-spacer"></div><h3 class="article-section__title" id="_i38"> Methyl (<i>S</i>)-6-(4-{2-[(<i>tert</i>-Butoxycarbonyl)amino]-<i>N</i>-methylpropanamido}phenyl)-7-methoxybenzo[<i>d</i>][1,3]dioxole-5-carboxylate (<b>17</b>)</h3><div class="NLM_p last">Using general procedure C, compound <b>17</b> (71.00 mg) was obtained from compound <b>7</b> in 73% yield as a yellow oil. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 7.28 (4H, s), 7.12 (1H, s), 6.07 (2H, s), 5.43–5.41 (1H, d, <i>J</i> = 8.1 Hz), 4.44–4.40 (1H, t, <i>J</i> = 6.9 Hz), 3.82 (3H, s), 3.52 (3H, s), 3.33 (3H, s), 1.45 (3H, s), 1.42 (6H, s), 1.15–1.13 (3H, d, <i>J</i> = 6.6 Hz). <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>): δ 173.51, 167.82, 155.09, 148.74, 141.41, 141.07, 140.23, 137.22, 131.25, 129.55, 126.65, 125.62, 104.94, 102.12, 60.10, 51.96, 47.11, 38.06, 28.48, 18.96. HRMS-ESI: <i>m</i>/<i>z</i> calcd for C<sub>25</sub>H<sub>31</sub>N<sub>2</sub>O<sub>8</sub> [(M + H)<sup>+</sup>] 487.2075, found 487.2075.</div></div><div id="sec4_24" class="NLM_sec NLM_sec_level_2"><div id="ac_i39" class="anchor-spacer"></div><h3 class="article-section__title" id="_i39"> Methyl 6-[4-(2-Amino-<i>N</i>-methylacetamido)phenyl]-7-methoxybenzo[<i>d</i>][1,3]dioxole-5-carboxylate (<b>18</b>)</h3><div class="NLM_p last">Using general procedure D, compound <b>18</b> (39.00 mg) was obtained from compound <b>15</b> in 83% yield as a white solid. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 7.26–7.24 (4H, m), 7.11 (1H, s), 6.06 (2H, s), 5.43 (1H, s), 3.78 (3H, s), 3.63 (2H, s), 3.51 (3H, s), 3.29 (3H, s). <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>): δ 167.52, 166.21, 148.93, 141.18, 140.44, 139.26, 138.55, 131.82, 129.68, 126.07, 124.90, 105.04, 102.24, 59.99, 51.99, 41.07, 37.69. HRMS-ESI: <i>m</i>/<i>z</i> calcd for C<sub>19</sub>H<sub>21</sub>N<sub>2</sub>O<sub>6</sub> [(M + H)<sup>+</sup>] 373.1394, found 373.1393.</div></div><div id="sec4_25" class="NLM_sec NLM_sec_level_2"><div id="ac_i40" class="anchor-spacer"></div><h3 class="article-section__title" id="_i40"> Methyl 7-Methoxy-6-{4-[<i>N</i>-methyl-2-(methylamino)acetamido]phenyl}benzo[<i>d</i>][1,3]dioxole-5-carboxylate (<b>19</b>)</h3><div class="NLM_p last">Using general procedure D, compound <b>19</b> (32.00 mg) was obtained from compound <b>16</b> in 59% yield as a yellow solid. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 7.28–7.17 (4H, m), 7.13 (1H, s), 6.08 (2H, s), 3.82 (3H, s), 3.53 (3H, s), 3.35 (3H, s), 3.15 (2H, s), 2.36 (3H, s), 2.23 (1H, s). <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>): δ 171.33, 167.49, 148.72, 141.51, 141.08, 140.27, 137.10, 131.11, 129.75, 126.42, 125.35, 104.92, 102.13, 60.09, 52.89, 51.88, 37.35, 36.41. HRMS-ESI: <i>m</i>/<i>z</i> calcd for C<sub>20</sub>H<sub>23</sub>N<sub>2</sub>O<sub>6</sub> [(M + H)<sup>+</sup>] 387.1551, found 387.1551.</div></div><div id="sec4_26" class="NLM_sec NLM_sec_level_2"><div id="ac_i41" class="anchor-spacer"></div><h3 class="article-section__title" id="_i41"> Methyl (<i>S</i>)-6-[4-(2-Amino-<i>N</i>-methylpropanamido)phenyl]-7-methoxybenzo[<i>d</i>][1,3]dioxole-5-carboxylate (<b>20</b>)</h3><div class="NLM_p last">Using general procedure D, compound <b>20</b> (15.00 mg) was obtained from compound <b>17</b> in 49% yield as a yellow oil. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 7.28–7.19 (4H, m), 7.13 (1H, s), 6.08 (2H, s), 3.81 (3H, s), 3.58 (1H, s), 3.52 (3H, s), 3.33 (3H, s), 1.77 (2H, s), 1.16–1.14 (3H, d, <i>J</i> = 6.6 Hz). <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>): δ 176.84, 167.53, 148.74, 142.00, 141.08, 140.27, 137.05, 131.13, 129.69, 126.56, 125.39, 104.94, 102.14, 60.10, 51.88, 47.39, 37.85, 21.71. HRMS-ESI: <i>m</i>/<i>z</i> calcd for C<sub>20</sub>H<sub>23</sub>N<sub>2</sub>O<sub>6</sub> [(M + H)<sup>+</sup>] 387.1551, found 387.1551.</div></div><div id="sec4_27" class="NLM_sec NLM_sec_level_2"><div id="ac_i42" class="anchor-spacer"></div><h3 class="article-section__title" id="_i42"> Methyl 7-Methoxy-6-[4-(<i>N</i>-methyl-6-{5-[(3a<i>S</i>,4<i>S</i>,6a<i>R</i>)-2-oxohexahydro-1<i>H</i>-thieno[3,4-<i>d</i>]imidazol-4-yl]pentanamido}hexanamido)phenyl]benzo[<i>d</i>][1,3]dioxole-5-carboxylate (<b>21</b>)</h3><div class="NLM_p last">Solid EDCI (58.00 mg, 0.3 mmol) was added in parts to the solution of (+)-biotin-ε-aminocaproic acid (71 mg, 0.2 mmol) in DMF (2 mL) at room temperature. The mixture was further stirred for 45 min, and then the 7-methoxy-6-[4-(methylamino)phenyl]benzo[<i>d</i>][1,3]dioxole-5-carboxylate (<b>7</b>) (63.00 mg, 0.2 mmol) and DMAP (0.1 equiv) were added. After 8 h, the mixture was then poured into water (10 mL) and extracted with EtOAc (3 × 5 mL). The combined organics were washed with brine (3 × 5 mL), dried with MgSO<sub>4</sub>, and filtered, and the solvents were removed <i>in vacuo</i>. The residue was purified by flash column chromatography (H<sub>2</sub>O/CH<sub>3</sub>OH/EtOAc = 1/1/20) to afford compound <b>21</b> (97.00 mg, 70%) as a white solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.22–7.20 (2H, d, <i>J</i> = 7.6 Hz), 7.16–7.14 (2H, d, <i>J</i> = 8.0 Hz), 7.10 (1H, s), 6.06 (2H, s), 4.49 (1H, s), 4.28 (1H, s), 3.79 (3H, s), 3.51 (3H, s), 3.28 (3H, s), 3.16–3.11 (3H, m), 2.90–2.86 (1H, dd, <i>J</i><sub>1</sub> = 12.4 Hz, <i>J</i><sub>2</sub> = 4.4 Hz), 2.74–2.71 (1H, d, <i>J</i> = 12.8 Hz), 2.18–2.13 (4H, m), 1.69–1.57 (5H, m), 1.41 (4H, s). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>): δ 173.32, 173.29, 167.43, 167.43, 148.60, 142.70, 141.01, 140.22, 136.58, 130.89, 129.77, 126.43, 125.25, 104.80, 102.06, 61.76, 60.28, 60.02, 55.75, 51.79, 40.49, 39.10, 37.31, 36.05, 33.84, 29.03, 28.30, 28.11, 26.50, 25.83, 25.00. HRMS-ESI: <i>m</i>/<i>z</i> calcd for C<sub>33</sub>H<sub>42</sub>N<sub>4</sub>O<sub>8</sub>SNa (M + Na)<sup>+</sup> 677.2616, found 677.2618.</div></div><div id="sec4_28" class="NLM_sec NLM_sec_level_2"><div id="ac_i43" class="anchor-spacer"></div><h3 class="article-section__title" id="_i43"> Cells and Reagents</h3><div class="NLM_p last">HeLa, SK-Hep-1, MHCC97, MHCC97-luciferase, HepG2, HCT116, MDA-MB-231, Huh7, and QSG 7701 cells were obtained from the Cell Resource Center, Shanghai Institutes for Biological Sciences, Chinese Academy of Sciences, and grown in Dulbecco’s modified Eagle’s medium (DMEM) high-glucose or RPMI 1640 medium (Hyclone) and supplemented with 10% fetal bovine serum (FBS, Gibco, Grand Island, NY) and penicillin/streptomycin (Hyclone). GM was isolated from <i>Schisandra wilsoniana</i>as previously reported.<a onclick="showRef(event, 'ref66'); return false;" href="javascript:void(0);" class="ref ref66">(66)</a> Enoxacin was purchased from Tokyo Chemical Industry (Shanghai, China), and sorafinib from Solarbio (Beijing, China). Plasmid pGL4.26, plasmid pCI-neo, Geneticin (G418) coenzyme, luciferin, and Oligo (dT) 15 Primer were purchased from Promega (Madison, WI). Superscript II was purchased from Invitrogen (Carlsbad, CA). Matrigel was purchased from BD Biosciences. Alamar blue reagent, propidium iodide (PI), annexin V-FITC, RIPA lysis buffer, phenylmethanesulfonyl fluoride (PMSF), IgG (rabbit IgG), protein A/G agarose, and the Fast Silver Stain Kit were purchased from Beyotime biotechnology (Beijing, China). All primers were purchased from Sangong biotech (Shanghai, China). Antibodies specific for CDC20, CDK2, and UBE2SE were purchased from ABGENT (San Diego, CA). Antibodies specific for TRBP, Dicer, Ago2, E-cadherin, β-catenin, vimentin, <i>N</i>-cadherin, TOP2β, STAT1, P-STAT1, STAT3, P-STAT3, p38, P-p38, IκB, P-IκB, p65, P-p65, MEK, P-MEK, AKT, P-AKT, ERK1/2, and P-ERK1/2 were purchased from Proteintech (Wuhan, China). The KIF14 antibody was purchased from Signalway Antibody (Nanjing, China). The TPX2 antibody and HRP-linked secondary antibody were purchased from ZEN-BIOSCIENCE (Chengdu, China). SYBR Green Master Mix reagents and prestained protein markers were purchased from Transgen (Beijing, China). pCMV-TRBP and TRBP siRNA (5′-GGAGTATGGGACCAGAATA-3′) were purchased from Sino Biological (Beijing, China) and RiboBio (Guangzhou, China), respectively.</div></div><div id="sec4_29" class="NLM_sec NLM_sec_level_2"><div id="ac_i44" class="anchor-spacer"></div><h3 class="article-section__title" id="_i44"> Establishing Stably Transfected Cell Lines and Luciferase Reporter Assay</h3><div class="NLM_p last">A luciferase-coding fragment was obtained from pGL4.26 and inserted into pCI-neo between the NheI and XbaI restriction sites to establish pCIneo-luciferase. The miR-21-5p target sequence (TCAACATCAGTCTGATAAGCTA) was annealed and inserted into the pCI-neo-luciferase plasmid between the SalI and NotI restriction sites to establish pCIneo-luciferase-miR-21. Then, the pCIneo-luciferase-miR-21 plasmid was transfected into HeLa cells, and G418 was used to screen stable cell lines influenced by changes in the expression of endogenic miR-21-5p. For the luciferase reporter assay and high-throughput screening, cells were seeded in 96-well plates and treated with small-molecule compounds for approximately 24 h. Following treatment, cells were lysed for 15 min using 100 μL of cell lysis buffer. Then, 50 μL of lysate was pipetted into black 96-well plates with 50 μL of luciferin substrate containing 0.1 M coenzyme and 0.48 mM luciferin following the instructions of the Promega luciferase assay systems. Bioluminescence was measured using a microplate reader. The activity of the compounds was determined on the basis of the levels of relative luciferase expression.</div></div><div id="sec4_30" class="NLM_sec NLM_sec_level_2"><div id="ac_i45" class="anchor-spacer"></div><h3 class="article-section__title" id="_i45"> Proliferation and Colony-Forming Assays</h3><div class="NLM_p last">Cells were seeded in 96-well plates at 45% confluence, washed twice with PBS after 12–16 h, and then treated with compounds or DMSO (vehicle control). After 24 h, 10% (v/v) Alamar blue reagent was added to each well, and cells were incubated for 1 h. Fluorescence was measured using a Varioskan Flash Multimode Reader (Thermo, Shanghai, China) at 450 nm absorbance. The colony-forming assay was performed in a 12-well plate containing 1000 cells/well. Cells were then treated with different compounds or concentrations, and the culture medium was changed weekly. After being cultured for 14 days, colonies were stained with crystal violet (0.1%) for 0.5 h and imaged.</div></div><div id="sec4_30_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i46" class="anchor-spacer"></div><h3 class="article-section__title" id="_i46"> qRT-PCR</h3><div class="NLM_p last">TRIzol was used to isolate total RNA from cells according to the manufacturer’s instructions. The quality of extracted RNA was assessed by measuring the absorbance at 260 nm relative to 280 nm after resuspension in 30 μL of RNase-free water. First-strand cDNA was then synthesized with reverse transcription using Superscript II and different specific primers listed in <a href="/doi/suppl/10.1021/acs.jmedchem.1c00018/suppl_file/jm1c00018_si_001.pdf" class="ext-link">Table S4</a> for miRNA. Oligo (dT) 15 Primer was used for pri-miRNA and pre-miRNA. SYBR Green Master Mix reagents and primer (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00018/suppl_file/jm1c00018_si_001.pdf" class="ext-link">Table S5</a>) mixtures were used for qRT-PCR. Small RNA RNU6 and GAPDH were used as endogenous controls to normalize miRNA, pri-miRNA, and pre-miRNA expression. The threshold cycle number (Ct) was used for quantitative analyses, and relative RNA expression was analyzed by the 2<sup>–ΔΔC(t)</sup> method.</div></div><div id="sec4_31" class="NLM_sec NLM_sec_level_2"><div id="ac_i47" class="anchor-spacer"></div><h3 class="article-section__title" id="_i47"> Flow Cytometry Analysis</h3><div class="NLM_p last">Following treatment with 0.1% DMSO (negative control) or 20 μM GM for 24 h, SK-Hep-1 cells were fixed with 70% (v/v) alcohol overnight at −20 °C. The cells were then washed with PBS and stained with 50 μg/mL propidium iodide for 30 min while being shielded from light. Approximately 20000 events were collected for flow cytometric analysis of the cell cycle distribution using a BD FACScan flow cytometer (DOTmed, Inc.). To analyze apoptosis, SK-Hep-1 cells were stained with annexin V-FITC and PI, followed by flow cytometry using a BD FACScan flow cytometer. All data files were analyzed using ModFit LT 5.0.</div></div><div id="sec4_32" class="NLM_sec NLM_sec_level_2"><div id="ac_i48" class="anchor-spacer"></div><h3 class="article-section__title" id="_i48"> Transwell Assays</h3><div class="NLM_p last">Transwell assays were performed according to the manufacturer’s instructions to evaluate the migration and invasiveness of cells. Briefly, SK-Hep-1 and MHCC97 cells in serum-free DMEM were seeded (5 × 10<sup>6</sup> cells/mL, 200 μL/well) in the Transwell plates (Corning, Corning, NY), and DMEM containing 10% fetal bovine serum was added to the bottom chamber. The cells were allowed to migrate under different treatments for approximately 12 h at 37 °C. After that, the upper surface of the Transwell cells was removed by cotton swabs, and migratory cells were fixed with ethyl alcohol (95%) and stained with crystal violet (0.1%). The number of migrating cells (three different fields per well) was counted using an inverted microscope. To evaluate invasion, a similar method was used as described for migration, except that the upper surface of the Transwells was preincubated with 12.5% (v/v) Matrigel before the cells were seeded. All images were obtained after 24 h using an IX51 microscope (Olympus, Tokyo, Japan).</div></div><div id="sec4_33" class="NLM_sec NLM_sec_level_2"><div id="ac_i49" class="anchor-spacer"></div><h3 class="article-section__title" id="_i49"> <i>In Vivo</i> Animal Studies</h3><div class="NLM_p last">SCID mice were obtained from Chengdu Dossy Experimentals Co., Ltd. (Chengdu, China), and fed using specific-pathogen-free equipment in a temperature-controlled sterile room at 25 °C. All animal experiments were approved by the Committee of Animal Ethics of Chengdu Institute of Biology, Chinese Academy of Sciences (CIBCAS-2018-01), and performed in strict accordance with the Guide for the Care and Use of Laboratory Animals of the Ministry of Health, China. Fifty male mice (5–6 weeks of age) were fed adaptively for 1 week following arrival. MHCC97-luciferase cells were harvested overnight with free serum and then digested with cold PBS. The cells (1 × 10<sup>5</sup>) in a total volume of 0.1 mL were injected into the left external lobe liver of SCID mice. Ten days after the injection, the bioluminescent signal was observed to examine liver tumor formation and 30 SCID mice with similarly sized tumors were then randomly divided five groups: model, 20 mg of enoxacin/kg, 20 mg of sorafenib/kg, 20 mg of GM/kg, and 40 mg of GM/kg (six mice per group). Mice were intraperitoneally administered PBS, enoxacin, sorafenib, and GM once every 3 days at the required dose. Enoxacin, sorafenib, and GM were dissolved in PBS with 5% (v/v) DMSO. The first day of administration was recorded as day 1. After the mice had been sacrificed by cervical vertebral dislocation, their liver tissue was collected, fixed with 10% formalin for immunohistochemistry, and stained with H&E 30 days after the first administration.</div></div><div id="sec4_34" class="NLM_sec NLM_sec_level_2"><div id="ac_i50" class="anchor-spacer"></div><h3 class="article-section__title" id="_i50"> H&E and Immunohistochemical Staining</h3><div class="NLM_p last">The mouse tissues were rapidly fixed in 10% neutral buffered formalin and embedded in paraffin. The samples were cut into 4 μm serial sections and stained with H&E. For immunohistochemical staining, paraffin sections were dehydrated and then deparaffinized. After nonspecific sites of antigens had been blocked with 5% (v/v) BAS, slides were incubated with the primary antibody overnight at 4 °C and then with secondary antibodies for 2 h at 25 °C. Finally, the slides were visualized with a 3,3-diaminobenzidine solution (Solarbio) and counterstained with hematoxylin.</div></div><div id="sec4_35" class="NLM_sec NLM_sec_level_2"><div id="ac_i51" class="anchor-spacer"></div><h3 class="article-section__title" id="_i51"> MiRNAomics</h3><div class="NLM_p">Following the treatment with or without GM for 24 h, TRIzol was used to extract total RNA, and a size range of 18–30 nucleotides was enriched by polyacrylamide gel electrophoresis (PAGE). The 5′ adapters were then added to the RNAs after enriching 36–44-nucleotide 3′ adapter RNAs. Then, the ligated products were reverse-transcribed by PCR amplification to enrich 140–160 bp products and generate a cDNA library and sequenced using Illumina HiSeqTM 2500 by Gene Denovo Biotechnology Co. (Guangzhou, China).</div><div class="NLM_p">Bioinformatics analysis was implemented on the basis of the cDNA library and sequence. Raw reads were further filtered and then aligned with small RNA in the GenBank database (release 209.0), Rfam database (11.0), and reference genome. To identify miRNAs, clean tags were searched against the miRBase database (release 21) to identify known human miRNAs. On the basis of their expression in each sample, miRNA levels were calculated and normalized to transcripts per million as follows: transcripts per million = actual miRNA counts/total counts of clean tags ×10<sup>6</sup>.</div><div class="NLM_p">Differentially expressed miRNAs (≥2-fold change, <i>P</i> < 0.05) across groups were identified using the following formula:<span class="NLM_disp-formula" id="ueq1"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00018/20210602/images/medium/jm1c00018_m001.gif" alt="" id="_i52" /></img></span></div><div class="NLM_p last">Target genes were predicted using the software RNAhybrid(v2.1.2)+svm_light(v6.01), Miranda(v3.3a), and TargetScan (version 7.0). GO enrichment and pathway enrichment analyses were performed using the GO database (<a href="http://www.geneontology.org/" class="extLink">http://www.geneontology.org/</a>) and pathway-related database KEGG (<a href="https://www.genome.jp/kegg/pathway.html" class="extLink">https://www.genome.jp/kegg/pathway.html</a>), respectively. The formula for calculating the <i>P</i> value was as follows:<span class="NLM_disp-formula" id="ueq2"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00018/20210602/images/medium/jm1c00018_m002.gif" alt="" id="_i53" /></img></span></div></div><div id="sec4_35_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i54" class="anchor-spacer"></div><h3 class="article-section__title" id="_i54"> Proteomics</h3><div class="NLM_p">Proteomics were analyzed quantitively by means of SILAC labeling. Briefly, SK-Hep-1 cells grown to 80% confluence were labeled with either “medium isotopic amino acids” (<span class="smallcaps smallerCapital">l</span>-<i>d</i><sub>4</sub>-lysine/<span class="smallcaps smallerCapital">l</span>-<sup>13</sup>C<sub>6</sub>-arginine) or “heavy isotopic amino acids” (<span class="smallcaps smallerCapital">l</span>-<sup>13</sup>C/<sup>15</sup>N<sub>2</sub>-lysine/<span class="smallcaps smallerCapital">l</span>-<sup>13</sup>C<sub>6</sub><sup>15</sup>N<sub>4</sub>-arginine) using a SILAC Protein Quantitation Kit (Pierce, Thermo). To attain a labeling efficiency of >97%, cells were grown for at least six passages before being harvested. Subsequently, the cells were grown in SILAC medium to obtain the desired cell number.</div><div class="NLM_p">SILAC-labeled cells were treated with GM and DMSO for 24 h. Each sample was tattered using a high-intensity ultrasonic processor (Scientz, Ningbo, China) at 4 °C, and the supernatant was collected and used to determine the protein concentration with a BCA kit after centrifugation. Protein extracts derived from the “medium”- and “heavy”-labeled cell cultures were mixed in a 1/1 ratio on the basis of the concentration of the measured protein. The mixture was treated with 5 mM dithiothreitol for 30 min at 56 °C and alkylated with 11 mM iodoacetamide for 15 min at room temperature in the dark. Next, the concentration of urea in the mixture was diluted to a maximum of 2 M by adding 100 mM NH<sub>4</sub>HCO<sub>3</sub>. Finally, the mixture was digested with trypsin (1/50 protein mass ratio) overnight and then for 4 h with a lower concentration of trypsin.</div><div class="NLM_p last">The trypsinized peptides were fractionated by high-pH reverse-phase HPLC using an Agilent 300Extend C18 column (5 μm particle size, 4.6 mm internal diameter, 250 mm length). The peptides were combined into 18 fractions, which were analyzed by MS/MS using Q ExactiveTM Plus (Thermo) coupled online to UPLC. A Maxquant search engine was used to process the MS/MS data. Significant differences were determined by analysis of variance with a >1.5-fold change in DMSO versus GM. The GO annotation of the proteome was derived from the UniProt-GOA database (<a href="http://www.ebi.ac.uk/GOA/" class="extLink">http://www.ebi.ac.uk/GOA/</a>). The KEGG database was used to annotate protein pathways and identify functional enrichment.</div></div><div id="sec4_36" class="NLM_sec NLM_sec_level_2"><div id="ac_i55" class="anchor-spacer"></div><h3 class="article-section__title" id="_i55"> Western Blotting</h3><div class="NLM_p last">Cells were incubated in RIPA lysis buffer with the addition of phenylmethanesulfonyl (PMSF) for 10 min at 4 °C, followed by centrifugation at 18000<i>g</i> for 10 min. Sodium dodecyl sulfate–PAGE (10%) was used to separate proteins following transfer to NC membranes (Merck, Darmstadt, Germany). Prestained protein markers were loaded in parallel. The membranes were incubated with BSA (PBS, 5% BSA) for 1 h at 25 °C and then incubated with primary antibodies at a 1/1000 dilution with Tris-buffered saline: Tween (TBST; TBS, 0.1% Tween 20, and 5% BSA) for approximately 14 h at 4 °C. Finally, after incubation with the secondary antibody at a dilution of 1/5000 with TBST for 1.5 h at 25 °C, the bands were visualized and analyzed using ImageQuant LAS 500 (GE Healthcare, Pittsburgh, PA).</div></div><div id="sec4_37" class="NLM_sec NLM_sec_level_2"><div id="ac_i56" class="anchor-spacer"></div><h3 class="article-section__title" id="_i56"> Cellular Thermal Shift Assay</h3><div class="NLM_p last">Sk-Hep-1 cells were lysed in PBS-containing PMSF by freezing and thawing in liquid nitrogen three times and then centrifuged (20000<i>g</i>) for 10 min at 4 °C. The supernatant was collected and treated with DMSO or 50 μM GM for 30 min at 37 °C. Then, 60 μL samples were treated at a series of temperatures for 3 min each. The supernatant of the cell lysate was also divided into 60 μL samples, which were subsequently treated with different concentrations for 30 min at 37 °C, and the incubated samples were heated to 55 °C for 3 min. Then, the heated samples were cooled at room temperature for 3 min and centrifuged (20000<i>g</i>) for 10 min at 4 °C. The supernatant was used for Western blotting to evaluate the interaction between GM and proteins.</div></div><div id="sec4_38" class="NLM_sec NLM_sec_level_2"><div id="ac_i57" class="anchor-spacer"></div><h3 class="article-section__title" id="_i57"> Overexpression and Knockdown of TRBP</h3><div class="NLM_p last">pcmv-TRBP and TRBP siRNA (5′-GGAGTATGGGACCAGAATA-3′) were mixed with Lipofectamine 3000 (Thermo) and then incubated with SK-Hep-1 cells (2.5 × 10<sup>6</sup> cells per transfection). After 24 h, protein expression was detected by Western blotting.</div></div><div id="sec4_38_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i58" class="anchor-spacer"></div><h3 class="article-section__title" id="_i58"> NMR Assay</h3><div class="NLM_p last">TRBP-His6 was cloned in a pCold I vector, expressed in <i>Escherichia coli</i> Rosetta, and purified using a prepacked HisTrapTM column (GE Healthcare) on ÄKTA prime plus. Then, 9 μL of a 1 mM GM stock solution was added to 441 μL of a TRBP-His6 solution at each concentration, and the samples were mixed well. The final NMR samples comprised 20 μM GM with 0, 7.5, 15, 30, and 60 μM TRBP-His6, each in 50 mM sodium phosphate, 150 mM NaCl, 2 mM β-Me, 2% DMSO-<i>d</i><sub>6</sub>, 10% D<sub>2</sub>O, and 0.2% (w/v) NaN<sub>3</sub> at pH 7.6. With regard to the stability of the purified TRBP-His6 and the efficiency of saturation diffusion in the STD NMR experiments, the NMR spectra were recorded at 279 K using a Bruker Avance 800 MHz spectrometer equipped with a QCI-cryoprobe. CSP experiments were performed using the 1H RD pulse program, with a series of samples. In the 1H RD pulse program, a “perfect echo” CPMG module (τ–180−τ–90−τ–180−τ, where τ is the delay between pulses) was incorporated to suppress artifacts from homonuclear scalar couplings and protein signals, and two W5 sequences were used to suppress the water signal. The total CPMG time was set to 60 ms. The STD spectra were recorded using the NMR sample of 20 μM GM with and without 30 μM TRBP-His6. Because of the low water solubility and relatively low concentration of GM (20 μM) compared to that of the added protein (30 μM), a modified STD pulse program with the “perfect echo” CPMG and two W5 modules (instead of the spin-lock and the excitation sculpting pulses in the original Bruker pulse program STDDIFFESGP.3) were used to suppress protein and water signals. The off- and on-resonance saturation frequencies were 40000 and 2866.9 Hz, respectively, from the water frequency. Then, 20 ms Gauss 1.1000 pulses were applied for saturation, with a total saturation time of 4 s. The carrier frequency on H<sub>2</sub>O was set to 4.954 ppm, as the reference for chemical shifts.</div></div><div id="sec4_38_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i59" class="anchor-spacer"></div><h3 class="article-section__title" id="_i59"> BIAcore Binding Assay</h3><div class="NLM_p last">The kinetics of binding of GM to TRBP were analyzed using SPR using a BIAcore 2000 instrument with a CM5 sensor chip (GE Healthcare). TRBP protein at a concentration of 4 μg/mL was immobilized via amine coupling on flow cells 1 and 2 of the CM5 sensor, which was activated by injecting a mixture of <i>N</i>-hydroxysuccinimide and 1-ethyl-3-[3-(dimethylamino)propyl]carbodiimide hydrochloride. Consequently, TRBP was captured at a density of ∼2000 RU. HBS-P [10 mM Hepes (pH 7.4), 0.15 M NaCl, and 0.005% surfactant] was used to equilibrate the sensor surface with 5% DMSO for at least 5 h. A GM concentration series was diluted in HBS-EP buffer [0.01 M Hepes (pH 7.4) containing 0.15 M NaCl, 3 mM EDTA, and 0.005% surfactant p20] with 5% DMSO and then with flow cell 1 at the maximum flow rate of 30 μL/min. As a control, flow cell 2 flowed into HBS-EP buffer. To regenerate the chip surface, a 2 min pulse of 1 M NaCl was injected to remove the bound GM and return the sensorgram to baseline. SPR data were analyzed to measure the association rate constant, dissociation rate constant, and <i>K</i><sub>D</sub> of the compound using BIA evaluation version 3.2.</div></div><div id="sec4_38_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i60" class="anchor-spacer"></div><h3 class="article-section__title" id="_i60"> Co-immunoprecipitation Assay</h3><div class="NLM_p last">SK-Hep-1 cells were pretreated with GM and DMSO for 24 h, and then cytoplasmic proteins were extracted and incubated with the TRBP antibody or IGg overnight at 4 °C. Then, samples were further incubated with protein A/G agarose beads for 2 h at 4 °C. The mixture was centrifuged for 5 min and washed five times. Finally, the immunoprecipitates were resuspended in PBS and detected by Western blotting.</div></div><div id="sec4_38_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i61" class="anchor-spacer"></div><h3 class="article-section__title" id="_i61"> Pull Down Assay</h3><div class="NLM_p last">Biotin-labeled compounds (30 μM) were bound to 80 μL of streptavidin agarose beads (Sigma) in 200 μL of 1× TE buffer overnight at 4 °C. The beads were washed three times with 500 μL of 1× TE and centrifuged three times at 2000 rpm. Then, TRBP in 500 μL of binding buffer [20 mM Tris-HCl (pH 7.6), 10% glycerol, 150 mM KCl, 0.1% NP-40, 1 mM DTT, 50 ng μL<sup>–1</sup> BSA, and protease inhibitors] was added, and samples were incubated for 2 h at 25 °C. After binding, the mixture was washed six times with binding buffer to remove redundant and nonspecific binding proteins. Following the final wash, beads were resuspended in 40 μL of cold PBS and 10 μL of 5× sodium dodecyl sulfate sample buffer for silver staining with the stain kit.</div></div><div id="sec4_39" class="NLM_sec NLM_sec_level_2"><div id="ac_i62" class="anchor-spacer"></div><h3 class="article-section__title" id="_i62"> Statistical Analysis</h3><div class="NLM_p last">All experiments were performed using at least three independent repeated experiments. The statistical significance of variations among experimental groups was evaluated using a Student’s <i>t</i> test and one-way analysis of variance, and data were presented as means ± standard errors of means. Pearson correlation coefficients were used to evaluate the relationships of differentially expressed genes. <i>P</i> values of <0.05 were considered statistically significant.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i63"><a href="/doi/suppl/10.1021/acs.jmedchem.1c00018" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_46205" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_46205" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.1c00018?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00018</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Additional figures illustrating the screening (Figure S1), the activities of GM (Figures S2–S5), miRNAomics and proteomics analysis (Figures S7–S14), the effect of GM on the expression of RISC elements (Figure S15), evidence that GM did not target Dicer or AGO2 (Figure S16), TRBP expression after transfection (Figure S17), activities of compounds <b>3</b>, <b>7</b>, <b>9</b>, and <b>10</b> in HeLa-miR-21-luciferase cells (Figure S18), the <i>K</i><sub>D</sub> values of compounds <b>3</b>, <b>7</b>, <b>9</b>, and <b>10</b> (Figure S19), the activity of GM analogues against SK-Hep-1 cells (Figure S20), a comparison of the activity of GM and compound <b>9</b> (Figure S21), and the effect of compound <b>21</b> in HeLa-miR-21-luciferase cells (Figure S22), evidence that compound <b>21</b> did not target PIWIL3 or TPO2β, miRNAomics and proteomics data (Tables S1–S3), sequences of primers (Tables S4 and S5), NMR spectra, and HPLC data of (−)-Gomisin M1 and <b>7</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00018/suppl_file/jm1c00018_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings of GM and <b>1–21</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00018/suppl_file/jm1c00018_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00018/suppl_file/jm1c00018_si_001.pdf">jm1c00018_si_001.pdf (3.58 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00018/suppl_file/jm1c00018_si_002.csv">jm1c00018_si_002.csv (2.4 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.1c00018" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_30546" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_30546" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hongbin Zhang</span> - <span class="hlFld-Affiliation affiliation">Key
Laboratory of Medicinal Chemistry for Natural Resource, Ministry of
Education, Yunnan Research & Development Center for Natural Products,
School of Chemical Science and Technology, Yunnan University, Kunming 650091, China</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0002-2516-2634" title="Orcid link">https://orcid.org/0000-0002-2516-2634</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#770d1f1619101f15370e190259121302591419"><span class="__cf_email__" data-cfemail="aed4c6cfc0c9c6cceed7c0db80cbcadb">[email protected]</span>.cn</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kaifeng Hu</span> - <span class="hlFld-Affiliation affiliation">Innovative
Institute of Chinese Medicine and Pharmacy, Chengdu University of Traditional Chinese Medicine, Chengdu 611137, China</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0002-2343-2012" title="Orcid link">https://orcid.org/0000-0002-2343-2012</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#5b303a323d3e353c332e1b383f2e2f3836753e3f2e753835"><span class="__cf_email__" data-cfemail="0a616b636c6f646d627f4a696e7f7e6967246f6e7f246964">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Weilie Xiao</span> - <span class="hlFld-Affiliation affiliation">Key
Laboratory of Medicinal Chemistry for Natural Resource, Ministry of
Education, Yunnan Research & Development Center for Natural Products,
School of Chemical Science and Technology, Yunnan University, Kunming 650091, China</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0001-6826-1993" title="Orcid link">https://orcid.org/0000-0001-6826-1993</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#a2dacbc3cdd5c7cbcecbc7e2dbccd78cc7c6d78cc1cc"><span class="__cf_email__" data-cfemail="1f67767e70687a7673767a5f66716a317a7b6a317c71">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Fei Wang</span> - <span class="hlFld-Affiliation affiliation">Center
for Natural Products Research, Chengdu Institute
of Biology, Chinese Academy of Sciences, Chengdu 610041, China</span>; 
    <span class="hlFld-Affiliation affiliation">Xiongan
Institute of Innovation, Chinese Academy of Sciences, Hebei 071700, China</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#6314020d0405060a23000a014d02004d000d"><span class="__cf_email__" data-cfemail="4c3b2d222b2a29250c2f252e622d2f622f22">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Zongyuan Zhou</span> - <span class="hlFld-Affiliation affiliation">Center
for Natural Products Research, Chengdu Institute
of Biology, Chinese Academy of Sciences, Chengdu 610041, China</span>; 
    <span class="hlFld-Affiliation affiliation">University
of Chinese Academy of Sciences, Beijing 100049, China</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0002-8403-6531" title="Orcid link">https://orcid.org/0000-0002-8403-6531</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yiming Li</span> - <span class="hlFld-Affiliation affiliation">Key
Laboratory of Medicinal Chemistry for Natural Resource, Ministry of
Education, Yunnan Research & Development Center for Natural Products,
School of Chemical Science and Technology, Yunnan University, Kunming 650091, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xiaofang Ma</span> - <span class="hlFld-Affiliation affiliation">Innovative
Institute of Chinese Medicine and Pharmacy, Chengdu University of Traditional Chinese Medicine, Chengdu 611137, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Biyun Cao</span> - <span class="hlFld-Affiliation affiliation">Key
Laboratory of Medicinal Chemistry for Natural Resource, Ministry of
Education, Yunnan Research & Development Center for Natural Products,
School of Chemical Science and Technology, Yunnan University, Kunming 650091, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ting Peng</span> - <span class="hlFld-Affiliation affiliation">Center
for Natural Products Research, Chengdu Institute
of Biology, Chinese Academy of Sciences, Chengdu 610041, China</span>; 
    <span class="hlFld-Affiliation affiliation">University
of Chinese Academy of Sciences, Beijing 100049, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yuwen Sheng</span> - <span class="hlFld-Affiliation affiliation">Center
for Natural Products Research, Chengdu Institute
of Biology, Chinese Academy of Sciences, Chengdu 610041, China</span>; 
    <span class="hlFld-Affiliation affiliation">University
of Chinese Academy of Sciences, Beijing 100049, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Huipan Peng</span> - <span class="hlFld-Affiliation affiliation">Center
for Natural Products Research, Chengdu Institute
of Biology, Chinese Academy of Sciences, Chengdu 610041, China</span>; 
    <span class="hlFld-Affiliation affiliation">University
of Chinese Academy of Sciences, Beijing 100049, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Runze Li</span> - <span class="hlFld-Affiliation affiliation">Center
for Natural Products Research, Chengdu Institute
of Biology, Chinese Academy of Sciences, Chengdu 610041, China</span>; 
    <span class="hlFld-Affiliation affiliation">University
of Chinese Academy of Sciences, Beijing 100049, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yu Cao</span> - <span class="hlFld-Affiliation affiliation">Center
for Natural Products Research, Chengdu Institute
of Biology, Chinese Academy of Sciences, Chengdu 610041, China</span>; 
    <span class="hlFld-Affiliation affiliation">University
of Chinese Academy of Sciences, Beijing 100049, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ruiying Xi</span> - <span class="hlFld-Affiliation affiliation">Center
for Natural Products Research, Chengdu Institute
of Biology, Chinese Academy of Sciences, Chengdu 610041, China</span>; 
    <span class="hlFld-Affiliation affiliation">University
of Chinese Academy of Sciences, Beijing 100049, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Fu Li</span> - <span class="hlFld-Affiliation affiliation">Center
for Natural Products Research, Chengdu Institute
of Biology, Chinese Academy of Sciences, Chengdu 610041, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mengru Wang</span> - <span class="hlFld-Affiliation affiliation">Key
Laboratory of Medicinal Chemistry for Natural Resource, Ministry of
Education, Yunnan Research & Development Center for Natural Products,
School of Chemical Science and Technology, Yunnan University, Kunming 650091, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Handong Sun</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Phytochemistry and Plant Resources in West China, Kunming Institute of Botany, Chinese Academy of Sciences, Kunming 650201, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Guolin Zhang</span> - <span class="hlFld-Affiliation affiliation">Center
for Natural Products Research, Chengdu Institute
of Biology, Chinese Academy of Sciences, Chengdu 610041, China</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>Z.Z., Y.L., and X.M. contributed equally to this work. Z.Z., T.P., Y.S., H.P., R.L., Y.C., and R.X. performed the biological experiments. Y.L., F.L., B.C., and M.W. performed the chemical experiments. X.M. and K.H. performed the NMR experiment. Z.Z., Y.L., H.S., G.Z., F.W., and W.X. analyzed the data. Z.Z., Y.L., F.W., and W.X. wrote the manuscript. F.W., W.X., H.Z., and G.Z. designed and coordinated the experiments and provided funding.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><p>The authors declare no competing financial interest.<br /></br></p></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i65">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_77843" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_77843" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">National Natural Science Foundation of China (21672207, 21861142007, 81903541, 81860615, 21762048, and 81422046), Biological Resources Programme, Chinese Academy of Sciences (KFJ-BRP-008), Yunnan Applicative and Basic Research Program (2018FY001 and 2018FA048), the Natural Science Foundation of Yunnan University (2017YDQN03), the Key Laboratory of Medicinal Chemistry for Natural Resource, Ministry of Education (2017KF02), the Program for Changjiang Scholars and Innovative Research Team in University (IRT_17R94), and the Project of Innovative Research Team of Yunnan Province and the Yun Ling Scholar Project to W.X.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">AARS</td><td class="NLM_def"><p class="first last">alanyl-TRNA synthetase 1</p></td></tr><tr><td class="NLM_term">ADAR1</td><td class="NLM_def"><p class="first last">double-stranded RNA-specific adenosine deaminase</p></td></tr><tr><td class="NLM_term">CDC20</td><td class="NLM_def"><p class="first last">cell division cycle 20</p></td></tr><tr><td class="NLM_term">CDK2</td><td class="NLM_def"><p class="first last">cyclin-dependent kinase 2</p></td></tr><tr><td class="NLM_term">CPD</td><td class="NLM_def"><p class="first last">carboxypeptidase D</p></td></tr><tr><td class="NLM_term">CSP</td><td class="NLM_def"><p class="first last">chemical shift perturbation</p></td></tr><tr><td class="NLM_term">DCE</td><td class="NLM_def"><p class="first last">1,2-dichloroethane</p></td></tr><tr><td class="NLM_term">DCM</td><td class="NLM_def"><p class="first last">dichloromethane</p></td></tr><tr><td class="NLM_term">DMF</td><td class="NLM_def"><p class="first last">dimethylformamide</p></td></tr><tr><td class="NLM_term">DMSO</td><td class="NLM_def"><p class="first last">dimethyl sulfoxide</p></td></tr><tr><td class="NLM_term">DNMT1</td><td class="NLM_def"><p class="first last">DNA methyltransferase 1</p></td></tr><tr><td class="NLM_term">EC<sub>50</sub></td><td class="NLM_def"><p class="first last">median effective concentration</p></td></tr><tr><td class="NLM_term">EMT</td><td class="NLM_def"><p class="first last">epithelial mesenchymal transition</p></td></tr><tr><td class="NLM_term">ERK</td><td class="NLM_def"><p class="first last">extracellular regulated protein kinases</p></td></tr><tr><td class="NLM_term">ESFTs</td><td class="NLM_def"><p class="first last">Ewing sarcoma family tumors</p></td></tr><tr><td class="NLM_term">ESI</td><td class="NLM_def"><p class="first last">electrospray ionization</p></td></tr><tr><td class="NLM_term">FBXW11</td><td class="NLM_def"><p class="first last">F-box and WD repeat domain containing 11</p></td></tr><tr><td class="NLM_term">GM</td><td class="NLM_def"><p class="first last">(−)-Gomisin M1</p></td></tr><tr><td class="NLM_term">GO</td><td class="NLM_def"><p class="first last">gene ontology</p></td></tr><tr><td class="NLM_term">H&E</td><td class="NLM_def"><p class="first last">hematoxylin and eosin</p></td></tr><tr><td class="NLM_term">HCC</td><td class="NLM_def"><p class="first last">hepatocellular carcinoma</p></td></tr><tr><td class="NLM_term">HIV</td><td class="NLM_def"><p class="first last">human immunodeficiency virus</p></td></tr><tr><td class="NLM_term">HLA</td><td class="NLM_def"><p class="first last">human leukocyte antigen</p></td></tr><tr><td class="NLM_term">HPLC</td><td class="NLM_def"><p class="first last">high-performance liquid chromatography</p></td></tr><tr><td class="NLM_term">IC<sub>50</sub></td><td class="NLM_def"><p class="first last">half-maximal inhibitory concentration</p></td></tr><tr><td class="NLM_term">iTRAQ</td><td class="NLM_def"><p class="first last">isobaric tags for relative and absolute quantification</p></td></tr><tr><td class="NLM_term">JAK/STAT</td><td class="NLM_def"><p class="first last">Janus kinase/signal transducer and activator of transcription</p></td></tr><tr><td class="NLM_term">KEGG</td><td class="NLM_def"><p class="first last">Kyoto Encyclopedia of Genes and Genomes</p></td></tr><tr><td class="NLM_term">KIF14</td><td class="NLM_def"><p class="first last">kinesin family member 14</p></td></tr><tr><td class="NLM_term">KIF23</td><td class="NLM_def"><p class="first last">kinesin family member 23</p></td></tr><tr><td class="NLM_term">LC-MS/MS</td><td class="NLM_def"><p class="first last">liquid chromatography-tandem mass spectrometry</p></td></tr><tr><td class="NLM_term">MAPK</td><td class="NLM_def"><p class="first last">mitogen-activated protein kinase</p></td></tr><tr><td class="NLM_term">PI3K-Akt</td><td class="NLM_def"><p class="first last">phosphatidylinositide 3-kinases protein kinase B</p></td></tr><tr><td class="NLM_term">miRISC</td><td class="NLM_def"><p class="first last">miRNA-induced silencing complex</p></td></tr><tr><td class="NLM_term">miRNAs</td><td class="NLM_def"><p class="first last">microRNAs</p></td></tr><tr><td class="NLM_term">PACT</td><td class="NLM_def"><p class="first last">PKR-associated protein X</p></td></tr><tr><td class="NLM_term">PAINS</td><td class="NLM_def"><p class="first last">pan assay interference compounds</p></td></tr><tr><td class="NLM_term">PBS</td><td class="NLM_def"><p class="first last">phosphate-buffered saline</p></td></tr><tr><td class="NLM_term">PIWIL3</td><td class="NLM_def"><p class="first last">piwi-like protein 3</p></td></tr><tr><td class="NLM_term">PKR</td><td class="NLM_def"><p class="first last">protein kinase R</p></td></tr><tr><td class="NLM_term">PMSF</td><td class="NLM_def"><p class="first last">phenylmethanesulfonyl fluoride</p></td></tr><tr><td class="NLM_term">pre-miRNA</td><td class="NLM_def"><p class="first last">precursor miRNAs</p></td></tr><tr><td class="NLM_term">pri-miRNA</td><td class="NLM_def"><p class="first last">primary miRNAs</p></td></tr><tr><td class="NLM_term">PRR11</td><td class="NLM_def"><p class="first last">proline rich 11</p></td></tr><tr><td class="NLM_term">qRT-PCR</td><td class="NLM_def"><p class="first last">quantitative real-time polymerase chain reaction</p></td></tr><tr><td class="NLM_term">S6K2</td><td class="NLM_def"><p class="first last">S6 kinase 2</p></td></tr><tr><td class="NLM_term">SCID</td><td class="NLM_def"><p class="first last">severe combined immunodeficient</p></td></tr><tr><td class="NLM_term">SLC38A2</td><td class="NLM_def"><p class="first last">solute carrier family 38 member 2</p></td></tr><tr><td class="NLM_term">SPAG5</td><td class="NLM_def"><p class="first last">sperm-associated antigen 5</p></td></tr><tr><td class="NLM_term">STD</td><td class="NLM_def"><p class="first last">saturation-transfer difference</p></td></tr><tr><td class="NLM_term">TAP1</td><td class="NLM_def"><p class="first last">transporter 1</p></td></tr><tr><td class="NLM_term">TAPBP</td><td class="NLM_def"><p class="first last">TAP-binding protein</p></td></tr><tr><td class="NLM_term">TLC</td><td class="NLM_def"><p class="first last">thin-layer chromatography</p></td></tr><tr><td class="NLM_term">TNF</td><td class="NLM_def"><p class="first last">tumor necrosis factor</p></td></tr><tr><td class="NLM_term">TOP2β</td><td class="NLM_def"><p class="first last">topoisomerase 2β</p></td></tr><tr><td class="NLM_term">TPX2</td><td class="NLM_def"><p class="first last">targeting protein for Xklp2</p></td></tr><tr><td class="NLM_term">TRBP</td><td class="NLM_def"><p class="first last">TAR RNA-binding protein 2</p></td></tr><tr><td class="NLM_term">UBE2S</td><td class="NLM_def"><p class="first last">ubiquitin-conjugating enzyme E2 S</p></td></tr><tr><td class="NLM_term">UPLC</td><td class="NLM_def"><p class="first last">ultraperformance liquid chromatography</p></td></tr><tr><td class="NLM_term">UTR</td><td class="NLM_def"><p class="first last">untranslated region.</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i67">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_72074" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_72074" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 66 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Villanueva, A.</span></span> <span> </span><span class="NLM_article-title">Hepatocellular carcinoma</span>. <i>N. Engl. J. Med.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>380</i></span>,  <span class="NLM_fpage">1450</span>– <span class="NLM_lpage">1462</span>, <span class="refDoi"> DOI: 10.1056/NEJMra1713263</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1056%2FNEJMra1713263" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=30970190" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BC1MXotFehsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=380&publication_year=2019&pages=1450-1462&author=A.+Villanueva&title=Hepatocellular+carcinoma&doi=10.1056%2FNEJMra1713263"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Hepatocellular carcinoma</span></div><div class="casAuthors">Villanueva, Augusto</div><div class="citationInfo"><span class="NLM_cas:title">New England Journal of Medicine</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">380</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">1450-1462</span>CODEN:
                <span class="NLM_cas:coden">NEJMAG</span>;
        ISSN:<span class="NLM_cas:issn">1533-4406</span>.
    
            (<span class="NLM_cas:orgname">Massachusetts Medical Society</span>)
        </div><div class="casAbstract">A review summarizing the main genetic alterations in hepatocellular carcinoma, key epidemiol. features, and evidence-based approaches to its clin. management.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_ukXAOkaaW7Vg90H21EOLACvtfcHk0ljPMq6pIiJuoQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXotFehsb4%253D&md5=d0237fa3302cf7a48a6439023d1452be</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1056%2FNEJMra1713263&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMra1713263%26sid%3Dliteratum%253Aachs%26aulast%3DVillanueva%26aufirst%3DA.%26atitle%3DHepatocellular%2520carcinoma%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2019%26volume%3D380%26spage%3D1450%26epage%3D1462%26doi%3D10.1056%2FNEJMra1713263" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, Y. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsao, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tak, W. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burock, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Voliotis, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guan, Z.</span></span> <span> </span><span class="NLM_article-title">Efficacy and safety of sorafenib in patients in the Asia-Pacific region with advanced hepatocellular carcinoma: a phase III randomised, double-blind, placebo-controlled trial</span>. <i>Lancet Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">25</span>– <span class="NLM_lpage">34</span>, <span class="refDoi"> DOI: 10.1016/S1470-2045(08)70285-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1016%2FS1470-2045%2808%2970285-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=19095497" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsFChtbnM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2009&pages=25-34&author=A.+L.+Chengauthor=Y.+K.+Kangauthor=Z.+Chenauthor=C.+J.+Tsaoauthor=S.+Qinauthor=J.+S.+Kimauthor=R.+Luoauthor=J.+Fengauthor=S.+Yeauthor=T.+S.+Yangauthor=J.+Xuauthor=Y.+Sunauthor=H.+Liangauthor=J.+Liuauthor=J.+Wangauthor=W.+Y.+Takauthor=H.+Panauthor=K.+Burockauthor=J.+Zouauthor=D.+Voliotisauthor=Z.+Guan&title=Efficacy+and+safety+of+sorafenib+in+patients+in+the+Asia-Pacific+region+with+advanced+hepatocellular+carcinoma%3A+a+phase+III+randomised%2C+double-blind%2C+placebo-controlled+trial&doi=10.1016%2FS1470-2045%2808%2970285-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Efficacy and safety of sorafenib in patients in the Asia-Pacific region with advanced hepatocellular carcinoma: a phase III randomized, double-blind, placebo-controlled trial</span></div><div class="casAuthors">Cheng, Ann-Lii; Kang, Yoon-Koo; Chen, Zhendong; Tsao, Chao-Jung; Qin, Shukui; Kim, Jun Suk; Luo, Rongcheng; Feng, Jifeng; Ye, Shenglong; Yang, Tsai-Sheng; Xu, Jianming; Sun, Yan; Liang, Houjie; Liu, Jiwei; Wang, Jiejun; Tak, Won Young; Pan, Hongming; Burock, Karin; Zou, Jessie; Voliotis, Dimitris; Guan, Zhongzhen</div><div class="citationInfo"><span class="NLM_cas:title">Lancet Oncology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">25-34</span>CODEN:
                <span class="NLM_cas:coden">LOANBN</span>;
        ISSN:<span class="NLM_cas:issn">1470-2045</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Most cases of hepatocellular carcinoma occur in the Asia-Pacific region, where chronic hepatitis B infection is an important etiol. factor.  Assessing the efficacy and safety of new therapeutic options in an Asia-Pacific population is thus important.  We did a multinational phase III, randomized, double-blind, placebo-controlled trial to assess the efficacy and safety of sorafenib in patients from the Asia-Pacific region with advanced (unresectable or metastatic) hepatocellular carcinoma.  Between Sept 20, 2005, and Jan 31, 2007, patients with hepatocellular carcinoma who had not received previous systemic therapy and had Child-Pugh liver function class A, were randomly assigned to receive either oral sorafenib (400 mg) or placebo twice daily in 6-wk cycles, with efficacy measured at the end of each 6-wk period.  Eligible patients were stratified by the presence or absence of macroscopic vascular invasion or extrahepatic spread (or both), Eastern Cooperative Oncol. Group performance status, and geog. region.  Randomization was done centrally and in a 2:1 ratio by means of an interactive voice-response system.  There was no predefined primary endpoint; overall survival, time to progression (TTP), time to symptomatic progression (TTSP), disease control rate (DCR), and safety were assessed.  Efficacy analyses were done by intention to treat.  This trial is registered with, no.  Two hundred and seventy-one 271 patients from 23 centers in China, South Korea, and Taiwan were enrolled in the study.  Of these, 226 patients were randomly assigned to the exptl. group (n=150) or to the placebo group (n=76).  Median overall survival was 6.5 mo (95% CI 5.56-7.56) in patients treated with sorafenib, compared with 4.2 mo (3.75-5.46) in those who received placebo (hazard ratio [HR] 0.68 [95% CI 0.50-0.93]; p=0.014).  Median TTP was 2.8 mo (2.63-3.58) in the sorafenib group compared with 1.4 mo (1.35-1.55) in the placebo group (HR 0.57 [0.42-0.79]; p=0.0005).  The most frequently reported grade 3/4 drug-related adverse events in the 149 assessable patients treated with sorafenib were hand-foot skin reaction (HFSR; 16 patients [10.7%]), diarrhea (nine patients [6.0%]), and fatigue (five patients [3.4%]).  The most common adverse events resulting in dose redns. were HFSR (17 patients [11.4%]) and diarrhea (11 patients [7.4%]); these adverse events rarely led to discontinuation.  Sorafenib is effective for the treatment of advanced hepatocellular carcinoma in patients from the Asia-Pacific region, and is well tolerated.  Taken together with data from the Sorafenib Hepatocellular Carcinoma Assessment Randomized Protocol (SHARP) trial, sorafenib seems to be an appropriate option for the treatment of advanced hepatocellular carcinoma.  Funding: Bayer HealthCare Pharmaceuticals and Onyx Pharmaceuticals, Inc.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo9BFS2SjaE8LVg90H21EOLACvtfcHk0ljPMq6pIiJuoQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsFChtbnM&md5=9ec357bcb2990914da1bc41e31532902</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2FS1470-2045%2808%2970285-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1470-2045%252808%252970285-7%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DA.%2BL.%26aulast%3DKang%26aufirst%3DY.%2BK.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DTsao%26aufirst%3DC.%2BJ.%26aulast%3DQin%26aufirst%3DS.%26aulast%3DKim%26aufirst%3DJ.%2BS.%26aulast%3DLuo%26aufirst%3DR.%26aulast%3DFeng%26aufirst%3DJ.%26aulast%3DYe%26aufirst%3DS.%26aulast%3DYang%26aufirst%3DT.%2BS.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DSun%26aufirst%3DY.%26aulast%3DLiang%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DTak%26aufirst%3DW.%2BY.%26aulast%3DPan%26aufirst%3DH.%26aulast%3DBurock%26aufirst%3DK.%26aulast%3DZou%26aufirst%3DJ.%26aulast%3DVoliotis%26aufirst%3DD.%26aulast%3DGuan%26aufirst%3DZ.%26atitle%3DEfficacy%2520and%2520safety%2520of%2520sorafenib%2520in%2520patients%2520in%2520the%2520Asia-Pacific%2520region%2520with%2520advanced%2520hepatocellular%2520carcinoma%253A%2520a%2520phase%2520III%2520randomised%252C%2520double-blind%252C%2520placebo-controlled%2520trial%26jtitle%3DLancet%2520Oncol.%26date%3D2009%26volume%3D10%26spage%3D25%26epage%3D34%26doi%3D10.1016%2FS1470-2045%2808%2970285-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kudo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finn, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ikeda, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piscaglia, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baron, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jassem, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blanc, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vogel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Komov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, T. R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dutcus, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saito, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kraljevic, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tamai, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, A. L.</span></span> <span> </span><span class="NLM_article-title">Lenvatinib versus sorafenib in first-line treatment of patients with unresectable hepatocellular carcinoma: a randomised phase 3 non-inferiority trial</span>. <i>Lancet</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>391</i></span>,  <span class="NLM_fpage">1163</span>– <span class="NLM_lpage">1173</span>, <span class="refDoi"> DOI: 10.1016/S0140-6736(18)30207-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1016%2FS0140-6736%2818%2930207-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=29433850" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisFOjtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=391&publication_year=2018&pages=1163-1173&author=M.+Kudoauthor=R.+S.+Finnauthor=S.+Qinauthor=K.+H.+Hanauthor=K.+Ikedaauthor=F.+Piscagliaauthor=A.+Baronauthor=J.+W.+Parkauthor=G.+Hanauthor=J.+Jassemauthor=J.+F.+Blancauthor=A.+Vogelauthor=D.+Komovauthor=T.+R.+J.+Evansauthor=C.+Lopezauthor=C.+Dutcusauthor=M.+Guoauthor=K.+Saitoauthor=S.+Kraljevicauthor=T.+Tamaiauthor=M.+Renauthor=A.+L.+Cheng&title=Lenvatinib+versus+sorafenib+in+first-line+treatment+of+patients+with+unresectable+hepatocellular+carcinoma%3A+a+randomised+phase+3+non-inferiority+trial&doi=10.1016%2FS0140-6736%2818%2930207-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Lenvatinib versus sorafenib in first-line treatment of patients with unresectable hepatocellular carcinoma: a randomised phase 3 non-inferiority trial</span></div><div class="casAuthors">Kudo, Masatoshi; Finn, Richard S.; Qin, Shukui; Han, Kwang-Hyub; Ikeda, Kenji; Piscaglia, Fabio; Baron, Ari; Park, Joong-Won; Han, Guohong; Jassem, Jacek; Blanc, Jean Frederic; Vogel, Arndt; Komov, Dmitry; Evans, T. R. Jeffry; Lopez, Carlos; Dutcus, Corina; Guo, Matthew; Saito, Kenichi; Kraljevic, Silvija; Tamai, Toshiyuki; Ren, Min; Cheng, Ann-Lii</div><div class="citationInfo"><span class="NLM_cas:title">Lancet</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">391</span>
        (<span class="NLM_cas:issue">10126</span>),
    <span class="NLM_cas:pages">1163-1173</span>CODEN:
                <span class="NLM_cas:coden">LANCAO</span>;
        ISSN:<span class="NLM_cas:issn">0140-6736</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Background: In a phase 2 trial, lenvatinib, an inhibitor of VEGF receptors 1-3, FGF receptors 1-4, PDGF receptor α, RET, and KIT, showed activity in hepatocellular carcinoma.  We aimed to compare overall survival in patients treated with lenvatinib vs. sorafenib as a first-line treatment for unresectable hepatocellular carcinoma.  Methods: This was an open-label, phase 3, multicentre, non-inferiority trial that recruited patients with unresectable hepatocellular carcinoma, who had not received treatment for advanced disease, at 154 sites in 20 countries throughout the Asia-Pacific, European, and North American regions.  Patients were randomly assigned (1:1) via an interactive voice-web response system-with region; macroscopic portal vein invasion, extrahepatic spread, or both; Eastern Cooperative Oncol. Group performance status; and bodyweight as stratification factors-to receive oral lenvatinib (12 mg/day for bodyweight ≥60 kg or 8 mg/day for bodyweight <60 kg) or sorafenib 400 mg twice-daily in 28-day cycles.  The primary endpoint was overall survival, measured from the date of randomization until the date of death from any cause.  The efficacy anal. followed the intention-to-treat principle, and only patients who received treatment were included in the safety anal.  The non-inferiority margin was set at 1·08.  The trial is registered with ClinicalTrials.gov, no. NCT01761266.  Findings: Between March 1, 2013 and July 30, 2015, 1492 patients were recruited. 954 eligible patients were randomly assigned to lenvatinib (n=478) or sorafenib (n=476).  Median survival time for lenvatinib of 13·6 mo (95% CI 12·1-14·9) was non-inferior to sorafenib (12·3 mo, 10·4-13·9; hazard ratio 0·92, 95% CI 0·79-1·06), meeting criteria for non-inferiority.  The most common any-grade adverse events were hypertension (201 [42%]), diarrhea (184 [39%]), decreased appetite (162 [34%]), and decreased wt. (147 [31%]) for lenvatinib, and palmar-plantar erythrodysaesthesia (249 [52%]), diarrhea (220 [46%]), hypertension (144 [30%]), and decreased appetite (127 [27%]) for sorafenib.  Interoretation: Lenvatinib was non-inferior to sorafenib in overall survival in untreated advanced hepatocellular carcinoma.  The safety and tolerability profiles of lenvatinib were consistent with those previously obsd.Eisai Inc.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpK8z3Hv8KQH7Vg90H21EOLACvtfcHk0lgFjjK5oYYKnQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisFOjtr4%253D&md5=96f10bec400e857fe6d6d6504bc22ac9</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2FS0140-6736%2818%2930207-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0140-6736%252818%252930207-1%26sid%3Dliteratum%253Aachs%26aulast%3DKudo%26aufirst%3DM.%26aulast%3DFinn%26aufirst%3DR.%2BS.%26aulast%3DQin%26aufirst%3DS.%26aulast%3DHan%26aufirst%3DK.%2BH.%26aulast%3DIkeda%26aufirst%3DK.%26aulast%3DPiscaglia%26aufirst%3DF.%26aulast%3DBaron%26aufirst%3DA.%26aulast%3DPark%26aufirst%3DJ.%2BW.%26aulast%3DHan%26aufirst%3DG.%26aulast%3DJassem%26aufirst%3DJ.%26aulast%3DBlanc%26aufirst%3DJ.%2BF.%26aulast%3DVogel%26aufirst%3DA.%26aulast%3DKomov%26aufirst%3DD.%26aulast%3DEvans%26aufirst%3DT.%2BR.%2BJ.%26aulast%3DLopez%26aufirst%3DC.%26aulast%3DDutcus%26aufirst%3DC.%26aulast%3DGuo%26aufirst%3DM.%26aulast%3DSaito%26aufirst%3DK.%26aulast%3DKraljevic%26aufirst%3DS.%26aulast%3DTamai%26aufirst%3DT.%26aulast%3DRen%26aufirst%3DM.%26aulast%3DCheng%26aufirst%3DA.%2BL.%26atitle%3DLenvatinib%2520versus%2520sorafenib%2520in%2520first-line%2520treatment%2520of%2520patients%2520with%2520unresectable%2520hepatocellular%2520carcinoma%253A%2520a%2520randomised%2520phase%25203%2520non-inferiority%2520trial%26jtitle%3DLancet%26date%3D2018%26volume%3D391%26spage%3D1163%26epage%3D1173%26doi%3D10.1016%2FS0140-6736%2818%2930207-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dawkins, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Webster, R. M.</span></span> <span> </span><span class="NLM_article-title">The hepatocellular carcinoma market</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">13</span>– <span class="NLM_lpage">14</span>, <span class="refDoi"> DOI: 10.1038/nrd.2018.146</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1038%2Fnrd.2018.146" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=30168534" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhs1altr%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=13-14&author=J.+Dawkinsauthor=R.+M.+Webster&title=The+hepatocellular+carcinoma+market&doi=10.1038%2Fnrd.2018.146"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">The hepatocellular carcinoma market</span></div><div class="casAuthors">Dawkins, Joshua; Webster, Rachel M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">13-14</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">The market for hepatocellular carcinoma is set to change with the recent approvals of the kinase inhibitor lenvatinib and the immune checkpoint inhibitor nivolumab, as well as the anticipated approvals of further immunotherapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8qfEQT4N-2LVg90H21EOLACvtfcHk0lgFjjK5oYYKnQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhs1altr%252FP&md5=3dccfe676500b0395ef4a9f631ef2d9b</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2018.146&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2018.146%26sid%3Dliteratum%253Aachs%26aulast%3DDawkins%26aufirst%3DJ.%26aulast%3DWebster%26aufirst%3DR.%2BM.%26atitle%3DThe%2520hepatocellular%2520carcinoma%2520market%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2019%26volume%3D18%26spage%3D13%26epage%3D14%26doi%3D10.1038%2Fnrd.2018.146" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ji, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Budhu, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, Z. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forgues, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roessler, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ambs, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meltzer, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Croce, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, L. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Man, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lo, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ng, I. O. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, Z. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X. W.</span></span> <span> </span><span class="NLM_article-title">MicroRNA expression, survival, and response to interferon in liver cancer</span>. <i>N. Engl. J. Med.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>361</i></span>,  <span class="NLM_fpage">1437</span>– <span class="NLM_lpage">1447</span>, <span class="refDoi"> DOI: 10.1056/NEJMoa0901282</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1056%2FNEJMoa0901282" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=19812400" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht1GiurnE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=361&publication_year=2009&pages=1437-1447&author=J.+F.+Jiauthor=J.+Shiauthor=A.+Budhuauthor=Z.+P.+Yuauthor=M.+Forguesauthor=S.+Roesslerauthor=S.+Ambsauthor=Y.+D.+Chenauthor=P.+S.+Meltzerauthor=C.+M.+Croceauthor=L.+X.+Qinauthor=K.+Manauthor=C.+M.+Loauthor=J.+Leeauthor=I.+O.+L.+Ngauthor=J.+Fanauthor=Z.+Y.+Tangauthor=H.+C.+Sunauthor=X.+W.+Wang&title=MicroRNA+expression%2C+survival%2C+and+response+to+interferon+in+liver+cancer&doi=10.1056%2FNEJMoa0901282"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">MicroRNA expression, survival, and response to interferon in liver cancer</span></div><div class="casAuthors">Ji, Junfang; Shi, Jiong; Budhu, Anuradha; Yu, Zhipeng; Forgues, Marshonna; Roessler, Stephanie; Ambs, Stefan; Chen, Yidong; Meltzer, Paul S.; Croce, Carlo M.; Qin, Lun-Xiu; Man, Kwan; Lo, Chung-Mau; Lee, Joyce; Ng, Irene O. L.; Fan, Jia; Tang, Zhao-You; Sun, Hui-Chuan; Wang, Xin Wei</div><div class="citationInfo"><span class="NLM_cas:title">New England Journal of Medicine</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">361</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">1437-1447</span>CODEN:
                <span class="NLM_cas:coden">NEJMAG</span>;
        ISSN:<span class="NLM_cas:issn">0028-4793</span>.
    
            (<span class="NLM_cas:orgname">Massachusetts Medical Society</span>)
        </div><div class="casAbstract">BACKGROUND. Hepatocellular carcinoma is a common and aggressive cancer that occurs mainly in men.  We examd. microRNA expression patterns, survival, and response to interferon alfa in both men and women with the disease.  METHODS. We analyzed three independent cohorts that included a total of 455 patients with hepatocellular carcinoma who had undergone radical tumor resection between 1999 and 2003.  MicroRNA-expression profiling was performed in a cohort of 241 patients with hepatocellular carcinoma to identify tumor-related microRNAs and det. their assocn. with survival in men and women.  In addn., to validate our findings, we used quant. reverse-transcriptase-polymerase-chain-reaction assays to measure microRNAs and assess their assocn. with survival and response to therapy with interferon alfa in 214 patients from two independent, prospective, randomized, controlled trials of adjuvant interferon therapy.  RESULTS. In patients with hepatocellular carcinoma, the expression of miR-26a and miR-26b in nontumor liver tissue was higher in women than in men.  Tumors had reduced levels of miR-26 expression, as compared with paired noncancerous tissues, which indicated that the level of miR-26 expression was also assocd. with hepatocellular carcinoma.  Moreover, tumors with reduced miR-26 expression had a distinct transcriptomic pattern, and analyses of gene networks revealed that activation of signaling pathways between nuclear factor κB and interleukin-6 might play a role in tumor development.  Patients whose tumors had low miR-26 expression had shorter overall survival but a better response to interferon therapy than did patients whose tumors had high expression of the microRNA.  CONCLUSIONS. The expression patterns of microRNAs in liver tissue differ between men and women with hepatocellular carcinoma.  The miR-26 expression status of such patients is assocd. with survival and response to adjuvant therapy with interferon alfa.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpV1JB_Owz1cbVg90H21EOLACvtfcHk0lg_sHQIWQ7UJg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXht1GiurnE&md5=ae9cf6d4f975d121eef2d7104775cc49</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1056%2FNEJMoa0901282&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMoa0901282%26sid%3Dliteratum%253Aachs%26aulast%3DJi%26aufirst%3DJ.%2BF.%26aulast%3DShi%26aufirst%3DJ.%26aulast%3DBudhu%26aufirst%3DA.%26aulast%3DYu%26aufirst%3DZ.%2BP.%26aulast%3DForgues%26aufirst%3DM.%26aulast%3DRoessler%26aufirst%3DS.%26aulast%3DAmbs%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DY.%2BD.%26aulast%3DMeltzer%26aufirst%3DP.%2BS.%26aulast%3DCroce%26aufirst%3DC.%2BM.%26aulast%3DQin%26aufirst%3DL.%2BX.%26aulast%3DMan%26aufirst%3DK.%26aulast%3DLo%26aufirst%3DC.%2BM.%26aulast%3DLee%26aufirst%3DJ.%26aulast%3DNg%26aufirst%3DI.%2BO.%2BL.%26aulast%3DFan%26aufirst%3DJ.%26aulast%3DTang%26aufirst%3DZ.%2BY.%26aulast%3DSun%26aufirst%3DH.%2BC.%26aulast%3DWang%26aufirst%3DX.%2BW.%26atitle%3DMicroRNA%2520expression%252C%2520survival%252C%2520and%2520response%2520to%2520interferon%2520in%2520liver%2520cancer%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2009%26volume%3D361%26spage%3D1437%26epage%3D1447%26doi%3D10.1056%2FNEJMoa0901282" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Klingenberg, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsuda, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diederichs, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, T.</span></span> <span> </span><span class="NLM_article-title">Non-coding RNA in hepatocellular carcinoma: Mechanisms, biomarkers and therapeutic targets</span>. <i>J. Hepatol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">603</span>– <span class="NLM_lpage">618</span>, <span class="refDoi"> DOI: 10.1016/j.jhep.2017.04.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1016%2Fj.jhep.2017.04.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=28438689" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BC2sXotlWqsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2017&pages=603-618&author=M.+Klingenbergauthor=A.+Matsudaauthor=S.+Diederichsauthor=T.+Patel&title=Non-coding+RNA+in+hepatocellular+carcinoma%3A+Mechanisms%2C+biomarkers+and+therapeutic+targets&doi=10.1016%2Fj.jhep.2017.04.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Non-coding RNA in hepatocellular carcinoma: Mechanisms, biomarkers and therapeutic targets</span></div><div class="casAuthors">Klingenberg, Marcel; Matsuda, Akiko; Diederichs, Sven; Patel, Tushar</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Hepatology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">603-618</span>CODEN:
                <span class="NLM_cas:coden">JOHEEC</span>;
        ISSN:<span class="NLM_cas:issn">0168-8278</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The majority of the human genome is not translated into proteins but can be transcribed into RNA.  Even though the resulting non-coding RNAs (ncRNAs) do not encode for proteins, they contribute to diseases such as cancer.  Here, we review examples of the functions of ncRNAs in liver cancer and their potential use for the detection and treatment of liver cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJTPs3pykqf7Vg90H21EOLACvtfcHk0lg_sHQIWQ7UJg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXotlWqsrY%253D&md5=31a039f1450fbf28743012fbe71c08db</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.jhep.2017.04.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jhep.2017.04.009%26sid%3Dliteratum%253Aachs%26aulast%3DKlingenberg%26aufirst%3DM.%26aulast%3DMatsuda%26aufirst%3DA.%26aulast%3DDiederichs%26aufirst%3DS.%26aulast%3DPatel%26aufirst%3DT.%26atitle%3DNon-coding%2520RNA%2520in%2520hepatocellular%2520carcinoma%253A%2520Mechanisms%252C%2520biomarkers%2520and%2520therapeutic%2520targets%26jtitle%3DJ.%2520Hepatol.%26date%3D2017%26volume%3D67%26spage%3D603%26epage%3D618%26doi%3D10.1016%2Fj.jhep.2017.04.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kota, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chivukula, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Donnell, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wentzel, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montgomery, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hwang, H. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vivekanandan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Torbenson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mendell, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mendell, J. T.</span></span> <span> </span><span class="NLM_article-title">Therapeutic microRNA delivery suppresses tumorigenesis in a murine liver cancer model</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>137</i></span>,  <span class="NLM_fpage">1005</span>– <span class="NLM_lpage">1017</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2009.04.021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1016%2Fj.cell.2009.04.021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=19524505" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BD1MXps1eiurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=137&publication_year=2009&pages=1005-1017&author=J.+Kotaauthor=R.+R.+Chivukulaauthor=K.+A.+O%E2%80%99Donnellauthor=E.+A.+Wentzelauthor=C.+L.+Montgomeryauthor=H.+W.+Hwangauthor=T.+C.+Changauthor=P.+Vivekanandanauthor=M.+Torbensonauthor=K.+R.+Clarkauthor=J.+R.+Mendellauthor=J.+T.+Mendell&title=Therapeutic+microRNA+delivery+suppresses+tumorigenesis+in+a+murine+liver+cancer+model&doi=10.1016%2Fj.cell.2009.04.021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Therapeutic microRNA delivery suppresses tumorigenesis in a murine liver cancer model</span></div><div class="casAuthors">Kota, Janaiah; Chivukula, Raghu R.; O'Donnell, Kathryn A.; Wentzel, Erik A.; Montgomery, Chrystal L.; Hwang, Hun-Way; Chang, Tsung-Cheng; Vivekanandan, Perumal; Torbenson, Michael; Clark, K. Reed; Mendell, Jerry R.; Mendell, Joshua T.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">137</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1005-1017</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Therapeutic strategies based on modulation of microRNA (miRNA) activity hold great promise due to the ability of these small RNAs to potently influence cellular behavior.  In this study, we investigated the efficacy of a miRNA replacement therapy for liver cancer.  We demonstrate that hepatocellular carcinoma (HCC) cells exhibit reduced expression of miR-26a, a miRNA that is normally expressed at high levels in diverse tissues.  Expression of this miRNA in liver cancer cells in vitro induces cell-cycle arrest assocd. with direct targeting of cyclins D2 and E2.  Systemic administration of this miRNA in a mouse model of HCC using adeno-assocd. virus (AAV) results in inhibition of cancer cell proliferation, induction of tumor-specific apoptosis, and dramatic protection from disease progression without toxicity.  These findings suggest that delivery of miRNAs that are highly expressed and therefore tolerated in normal tissues but lost in disease cells may provide a general strategy for miRNA replacement therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhf39hbMXufLVg90H21EOLACvtfcHk0lj0uXmmfYamkw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXps1eiurg%253D&md5=a70e4b5eb29d926b03861a5613cdc3e4</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2009.04.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2009.04.021%26sid%3Dliteratum%253Aachs%26aulast%3DKota%26aufirst%3DJ.%26aulast%3DChivukula%26aufirst%3DR.%2BR.%26aulast%3DO%25E2%2580%2599Donnell%26aufirst%3DK.%2BA.%26aulast%3DWentzel%26aufirst%3DE.%2BA.%26aulast%3DMontgomery%26aufirst%3DC.%2BL.%26aulast%3DHwang%26aufirst%3DH.%2BW.%26aulast%3DChang%26aufirst%3DT.%2BC.%26aulast%3DVivekanandan%26aufirst%3DP.%26aulast%3DTorbenson%26aufirst%3DM.%26aulast%3DClark%26aufirst%3DK.%2BR.%26aulast%3DMendell%26aufirst%3DJ.%2BR.%26aulast%3DMendell%26aufirst%3DJ.%2BT.%26atitle%3DTherapeutic%2520microRNA%2520delivery%2520suppresses%2520tumorigenesis%2520in%2520a%2520murine%2520liver%2520cancer%2520model%26jtitle%3DCell%26date%3D2009%26volume%3D137%26spage%3D1005%26epage%3D1017%26doi%3D10.1016%2Fj.cell.2009.04.021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sehgal, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vaishnaw, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fitzgerald, K.</span></span> <span> </span><span class="NLM_article-title">Liver as a target for oligonucleotide therapeutics</span>. <i>J. Hepatol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">1354</span>– <span class="NLM_lpage">1359</span>, <span class="refDoi"> DOI: 10.1016/j.jhep.2013.05.045</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1016%2Fj.jhep.2013.05.045" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=23770039" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVylt7%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2013&pages=1354-1359&author=A.+Sehgalauthor=A.+Vaishnawauthor=K.+Fitzgerald&title=Liver+as+a+target+for+oligonucleotide+therapeutics&doi=10.1016%2Fj.jhep.2013.05.045"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Liver as a target for oligonucleotide therapeutics</span></div><div class="casAuthors">Sehgal, Alfica; Vaishnaw, Akshay; Fitzgerald, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Hepatology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1354-1359</span>CODEN:
                <span class="NLM_cas:coden">JOHEEC</span>;
        ISSN:<span class="NLM_cas:issn">0168-8278</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">This article focused on liver-targeted for oligonucleotide therapeutics that have shown promise at the preclin. and/or clin. level.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPIwSCPukrEbVg90H21EOLACvtfcHk0lj0uXmmfYamkw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVylt7%252FN&md5=ac42e6bb3652d09900cdecd06470f554</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.jhep.2013.05.045&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jhep.2013.05.045%26sid%3Dliteratum%253Aachs%26aulast%3DSehgal%26aufirst%3DA.%26aulast%3DVaishnaw%26aufirst%3DA.%26aulast%3DFitzgerald%26aufirst%3DK.%26atitle%3DLiver%2520as%2520a%2520target%2520for%2520oligonucleotide%2520therapeutics%26jtitle%3DJ.%2520Hepatol.%26date%3D2013%26volume%3D59%26spage%3D1354%26epage%3D1359%26doi%3D10.1016%2Fj.jhep.2013.05.045" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Manjunath, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Subramanya, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shankar, P.</span></span> <span> </span><span class="NLM_article-title">Lentiviral delivery of short hairpin RNAs</span>. <i>Adv. Drug Delivery Rev.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">732</span>– <span class="NLM_lpage">745</span>, <span class="refDoi"> DOI: 10.1016/j.addr.2009.03.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1016%2Fj.addr.2009.03.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=19341774" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BD1MXmsFyns7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2009&pages=732-745&author=N.+Manjunathauthor=H.+Wuauthor=S.+Subramanyaauthor=P.+Shankar&title=Lentiviral+delivery+of+short+hairpin+RNAs&doi=10.1016%2Fj.addr.2009.03.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Lentiviral delivery of short hairpin RNAs</span></div><div class="casAuthors">Manjunath, N.; Wu, Haoquan; Subramanya, Sandesh; Shankar, Premlata</div><div class="citationInfo"><span class="NLM_cas:title">Advanced Drug Delivery Reviews</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">732-745</span>CODEN:
                <span class="NLM_cas:coden">ADDREP</span>;
        ISSN:<span class="NLM_cas:issn">0169-409X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  In less than a decade after discovery, RNA interference-mediated gene silencing is already being tested as potential therapy in clin. trials for a no. of diseases.  Lentiviral vectors provide a means to express short hairpin RNA (shRNA) to induce stable and long-term gene silencing in both dividing and non-dividing cells and thus, are being intensively investigated for this purpose.  However, induction of long-term shRNA expression can also cause toxicities by inducing off-target effects and interference with the endogenous micro-RNA (miRNA) pathway that regulates cellular gene expression.  Recently, several advances have been made in the shRNA vector design to mimic cellular miRNA processing and to express multiplex siRNAs in a tightly regulated and reversible manner to overcome toxicities.  In this review we describe some of these advances, focusing on the progress made in the development of lentiviral shRNA delivery strategies to combat viral infections.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0x5xsKZljBbVg90H21EOLACvtfcHk0li6QDmlGFu2bg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXmsFyns7c%253D&md5=5c1bddec6d6a9377edc11978fc6616fc</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.addr.2009.03.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.addr.2009.03.004%26sid%3Dliteratum%253Aachs%26aulast%3DManjunath%26aufirst%3DN.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DSubramanya%26aufirst%3DS.%26aulast%3DShankar%26aufirst%3DP.%26atitle%3DLentiviral%2520delivery%2520of%2520short%2520hairpin%2520RNAs%26jtitle%3DAdv.%2520Drug%2520Delivery%2520Rev.%26date%3D2009%26volume%3D61%26spage%3D732%26epage%3D745%26doi%3D10.1016%2Fj.addr.2009.03.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Cancer
Genome Atlas Research Network.</span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ally, A.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Comprehensive and integrative genomic characterization of hepatocellular carcinoma</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>169</i></span>,  <span class="NLM_fpage">1327</span>– <span class="NLM_lpage">1341.e23</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2017.05.046</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1016%2Fj.cell.2017.05.046" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=28622513" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=169&publication_year=2017&pages=1327-1341.e23&author=Cancer%0AGenome+Atlas+Research+Network.author=A.+Ally&title=Comprehensive+and+integrative+genomic+characterization+of+hepatocellular+carcinoma&doi=10.1016%2Fj.cell.2017.05.046"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2017.05.046&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2017.05.046%26sid%3Dliteratum%253Aachs%26aulast%3D%26aulast%3DAlly%26aufirst%3DA.%26atitle%3DComprehensive%2520and%2520integrative%2520genomic%2520characterization%2520of%2520hepatocellular%2520carcinoma%26jtitle%3DCell%26date%3D2017%26volume%3D169%26spage%3D1327%26epage%3D1341.e23%26doi%3D10.1016%2Fj.cell.2017.05.046" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Melo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villanueva, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moutinho, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davalos, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spizzo, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ivan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Setien, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casanovas, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simo-Riudalbas, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carmona, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carrere, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vidal, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aytes, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Puertas, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ropero, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalluri, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Croce, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calin, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Esteller, M.</span></span> <span> </span><span class="NLM_article-title">Small molecule enoxacin is a cancer-specific growth inhibitor that acts by enhancing TAR RNA-binding protein 2-mediated microRNA processing</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>108</i></span>,  <span class="NLM_fpage">4394</span>– <span class="NLM_lpage">4399</span>, <span class="refDoi"> DOI: 10.1073/pnas.1014720108</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1073%2Fpnas.1014720108" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=21368194" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BC3MXjvV2jsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=108&publication_year=2011&pages=4394-4399&author=S.+Meloauthor=A.+Villanuevaauthor=C.+Moutinhoauthor=V.+Davalosauthor=R.+Spizzoauthor=C.+Ivanauthor=S.+Rossiauthor=F.+Setienauthor=O.+Casanovasauthor=L.+Simo-Riudalbasauthor=J.+Carmonaauthor=J.+Carrereauthor=A.+Vidalauthor=A.+Aytesauthor=S.+Puertasauthor=S.+Roperoauthor=R.+Kalluriauthor=C.+M.+Croceauthor=G.+A.+Calinauthor=M.+Esteller&title=Small+molecule+enoxacin+is+a+cancer-specific+growth+inhibitor+that+acts+by+enhancing+TAR+RNA-binding+protein+2-mediated+microRNA+processing&doi=10.1073%2Fpnas.1014720108"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Small molecule enoxacin is a cancer-specific growth inhibitor that acts by enhancing TAR RNA-binding protein 2-mediated microRNA processing</span></div><div class="casAuthors">Melo, Sonia; Villaneuva, Alberto; Moutinho, Catia; Davalos, Veronica; Spizzo, Riccardo; Ivan, Cristina; Rossi, Simona; Setien, Fernando; Casanovas, Oriol; Simo-Riudalbas, Laia; Carmona, Javier; Carrere, Jordi; Vidal, August; Aytes, Alvaro; Puertas, Sara; Ropero, Santiago; Kalluri, Raghu; Croce, Carlo M.; Calin, George A.; Esteller, Manel</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">108</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4394-4399, S4394/1-S4394/10</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">MicroRNAs (miRNAs) are small RNA mols. that regulate gene expression at the posttranscriptional level and are crit. for many cellular pathways.  The disruption of miRNAs and their processing machineries also contributes to the development of human tumors.  A common scenario for miRNA expression in carcinogenesis is emerging that shows that impaired miRNA prodn. and/or down-regulation of these transcripts occurs in many neoplasms.  Several of these lost miRNAs have tumor-suppressor features; so strategies to restore their expression globally in malignancies would be a welcome addn. to the current therapeutic arsenal against cancer.  Herein, we show that the small mol. enoxacin, a fluoroquinolone used as an antibacterial compd., enhances the prodn. of miRNAs with tumor suppressor functions by binding to the miRNA biosynthesis protein TAR RNA-binding protein 2 (TRBP).  The use of enoxacin in human cell cultures and xenografted, orthotopic, and metastatic mouse models reveals a TRBP- dependent and cancer-specific growth-inhibitory effect of the drug.  These results highlight the key role of disrupted miRNA expression patterns in tumorigenesis, and suggest a unique strategy for restoring the distorted microRNAome of cancer cells to a more physiol. setting.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr87Y1kq9LPYrVg90H21EOLACvtfcHk0lgjmBbHZSZF8w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXjvV2jsLk%253D&md5=bef55782f8265ad8676d5b627cbc8523</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1014720108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1014720108%26sid%3Dliteratum%253Aachs%26aulast%3DMelo%26aufirst%3DS.%26aulast%3DVillanueva%26aufirst%3DA.%26aulast%3DMoutinho%26aufirst%3DC.%26aulast%3DDavalos%26aufirst%3DV.%26aulast%3DSpizzo%26aufirst%3DR.%26aulast%3DIvan%26aufirst%3DC.%26aulast%3DRossi%26aufirst%3DS.%26aulast%3DSetien%26aufirst%3DF.%26aulast%3DCasanovas%26aufirst%3DO.%26aulast%3DSimo-Riudalbas%26aufirst%3DL.%26aulast%3DCarmona%26aufirst%3DJ.%26aulast%3DCarrere%26aufirst%3DJ.%26aulast%3DVidal%26aufirst%3DA.%26aulast%3DAytes%26aufirst%3DA.%26aulast%3DPuertas%26aufirst%3DS.%26aulast%3DRopero%26aufirst%3DS.%26aulast%3DKalluri%26aufirst%3DR.%26aulast%3DCroce%26aufirst%3DC.%2BM.%26aulast%3DCalin%26aufirst%3DG.%2BA.%26aulast%3DEsteller%26aufirst%3DM.%26atitle%3DSmall%2520molecule%2520enoxacin%2520is%2520a%2520cancer-specific%2520growth%2520inhibitor%2520that%2520acts%2520by%2520enhancing%2520TAR%2520RNA-binding%2520protein%25202-mediated%2520microRNA%2520processing%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2011%26volume%3D108%26spage%3D4394%26epage%3D4399%26doi%3D10.1073%2Fpnas.1014720108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fareh, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeom, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haagsma, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chauhan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heo, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joo, C.</span></span> <span> </span><span class="NLM_article-title">TRBP ensures efficient Dicer processing of precursor microRNA in RNA-crowded environments</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">13694</span>, <span class="refDoi"> DOI: 10.1038/ncomms13694</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1038%2Fncomms13694" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=27934859" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BC28XitFamsLfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=13694&author=M.+Farehauthor=K.+H.+Yeomauthor=A.+C.+Haagsmaauthor=S.+Chauhanauthor=I.+Heoauthor=C.+Joo&title=TRBP+ensures+efficient+Dicer+processing+of+precursor+microRNA+in+RNA-crowded+environments&doi=10.1038%2Fncomms13694"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">TRBP ensures efficient Dicer processing of precursor microRNA in RNA-crowded environments</span></div><div class="casAuthors">Fareh, Mohamed; Yeom, Kyu-Hyeon; Haagsma, Anna C.; Chauhan, Sweeny; Heo, Inha; Joo, Chirlmin</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">13694</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The RNA-binding protein TRBP is a central component of the Dicer complex.  Despite a decade of biochem. and structural studies, the essential functionality of TRBP in microRNA (miRNA) biogenesis remains unknown.  Here we show that TRBP is an integral cofactor for time-efficient Dicer processing in RNA-crowded environments.  We competed for Dicer processing of pre-miRNA with a large amt. of cellular RNA species and found that Dicer-TRBP, but not Dicer alone, remains resilient.  To apprehend the mechanism of this substrate selectivity, we use single-mol. fluorescence.  The real-time observation reveals that TRBP acts as a gatekeeper, precluding Dicer from engaging with pre-miRNA-like substrates.  TRBP acquires the selectivity using the PAZ domain of Dicer, whereas Dicer moderates the RNA-binding affinity of TRBP for fast turnover.  This coordinated action between TRBP and Dicer accomplishes an efficient way of discarding pre-miRNA-like substrates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoe2wKT09Q-zbVg90H21EOLACvtfcHk0lgjmBbHZSZF8w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitFamsLfN&md5=31ab2d8222bfe62ce80680aff52d5d99</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fncomms13694&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms13694%26sid%3Dliteratum%253Aachs%26aulast%3DFareh%26aufirst%3DM.%26aulast%3DYeom%26aufirst%3DK.%2BH.%26aulast%3DHaagsma%26aufirst%3DA.%2BC.%26aulast%3DChauhan%26aufirst%3DS.%26aulast%3DHeo%26aufirst%3DI.%26aulast%3DJoo%26aufirst%3DC.%26atitle%3DTRBP%2520ensures%2520efficient%2520Dicer%2520processing%2520of%2520precursor%2520microRNA%2520in%2520RNA-crowded%2520environments%26jtitle%3DNat.%2520Commun.%26date%3D2016%26volume%3D7%26spage%3D13694%26doi%3D10.1038%2Fncomms13694" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chendrimada, T. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gregory, R. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumaraswamy, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norman, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooch, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nishikura, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shiekhattar, R.</span></span> <span> </span><span class="NLM_article-title">TRBP recruits the Dicer complex to Ago2 for microRNA processing and gene silencing</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>436</i></span>,  <span class="NLM_fpage">740</span>– <span class="NLM_lpage">744</span>, <span class="refDoi"> DOI: 10.1038/nature03868</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1038%2Fnature03868" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=15973356" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BD2MXmvFenur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=436&publication_year=2005&pages=740-744&author=T.+P.+Chendrimadaauthor=R.+I.+Gregoryauthor=E.+Kumaraswamyauthor=J.+Normanauthor=N.+Coochauthor=K.+Nishikuraauthor=R.+Shiekhattar&title=TRBP+recruits+the+Dicer+complex+to+Ago2+for+microRNA+processing+and+gene+silencing&doi=10.1038%2Fnature03868"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">TRBP recruits the Dicer complex to Ago2 for microRNA processing and gene silencing</span></div><div class="casAuthors">Chendrimada, Thimmaiah P.; Gregory, Richard I.; Kumaraswamy, Easwari; Norman, Jessica; Cooch, Neil; Nishikura, Kazuko; Shiekhattar, Ramin</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">436</span>
        (<span class="NLM_cas:issue">7051</span>),
    <span class="NLM_cas:pages">740-744</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">MicroRNAs (miRNAs) are generated by a two-step processing pathway to yield RNA mols. of approx. 22 nucleotides that neg. regulate target gene expression at the post-transcriptional level.  Primary miRNAs are processed to precursor miRNAs (pre-miRNAs) by the Microprocessor complex.  These pre-miRNAs are cleaved by the RNase III Dicer to generate mature miRNAs that direct the RNA-induced silencing complex (RISC) to mRNAs with complementary sequence.  Here we show that TRBP (the human immunodeficiency virus transactivating response RNA-binding protein), which contains three double-stranded, RNA-binding domains, is an integral component of a Dicer-contg. complex.  Biochem. anal. of TRBP-contg. complexes revealed the assocn. of Dicer-TRBP with Argonaute 2 (Ago2), the catalytic engine of RISC.  The phys. assocn. of Dicer-TRBP and Ago2 was confirmed after the isolation of the ternary complex using Flag-tagged Ago2 cell lines.  In vitro reconstitution assays demonstrated that TRBP is required for the recruitment of Ago2 to the small interfering RNA (siRNA) bound by Dicer.  Knockdown of TRBP results in destabilization of Dicer and a consequent loss of miRNA biogenesis.  Finally, depletion of the Dicer-TRBP complex via exogenously introduced siRNAs diminished RISC-mediated reporter gene silencing.  These results support a role of the Dicer-TRBP complex not only in miRNA processing but also as a platform for RISC assembly.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2woUlrto1u7Vg90H21EOLACvtfcHk0lgNnOkIEi4pdg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXmvFenur0%253D&md5=18ce99ba3ca840a2f63d42ff21c811cb</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2Fnature03868&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature03868%26sid%3Dliteratum%253Aachs%26aulast%3DChendrimada%26aufirst%3DT.%2BP.%26aulast%3DGregory%26aufirst%3DR.%2BI.%26aulast%3DKumaraswamy%26aufirst%3DE.%26aulast%3DNorman%26aufirst%3DJ.%26aulast%3DCooch%26aufirst%3DN.%26aulast%3DNishikura%26aufirst%3DK.%26aulast%3DShiekhattar%26aufirst%3DR.%26atitle%3DTRBP%2520recruits%2520the%2520Dicer%2520complex%2520to%2520Ago2%2520for%2520microRNA%2520processing%2520and%2520gene%2520silencing%26jtitle%3DNature%26date%3D2005%26volume%3D436%26spage%3D740%26epage%3D744%26doi%3D10.1038%2Fnature03868" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winter, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diederichs, S.</span></span> <span> </span><span class="NLM_article-title">Argonaute proteins regulate microRNA stability: Increased microRNA abundance by Argonaute proteins is due to microRNA stabilization</span>. <i>RNA Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1149</span>– <span class="NLM_lpage">1157</span>, <span class="refDoi"> DOI: 10.4161/rna.8.6.17665</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.4161%2Frna.8.6.17665" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=21941127" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BC38XkslCjtL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2011&pages=1149-1157&author=J.+Winterauthor=S.+Diederichs&title=Argonaute+proteins+regulate+microRNA+stability%3A+Increased+microRNA+abundance+by+Argonaute+proteins+is+due+to+microRNA+stabilization&doi=10.4161%2Frna.8.6.17665"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Argonaute proteins regulate microRNA stability</span></div><div class="casAuthors">Winter, Julia; Diederichs, Sven</div><div class="citationInfo"><span class="NLM_cas:title">RNA Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1149-1157</span>CODEN:
                <span class="NLM_cas:coden">RBNIBE</span>;
        ISSN:<span class="NLM_cas:issn">1547-6286</span>.
    
            (<span class="NLM_cas:orgname">Landes Bioscience</span>)
        </div><div class="casAbstract">Argonaute proteins are key players in microRNA (miRNA) processing and function.  Next to their role as RISC effector proteins mediating target silencing, they actively participate in miRNA biogenesis and increase miRNA abundance by a yet unidentified mechanism.  It has been hypothesized that this increase in steady-state miRNA levels might be due to a stabilizing effect of Argonaute proteins, but this has not been analyzed so far due to a lack of test systems.  Here, we provide two approaches to est. miRNA stability and factors affecting it: in cells lacking Ago2, endogenous miRNA guide strand degrdn. and half-lives can be assessed using actinomycin D or α-amanitin.  In turn, miRNA passenger strands can serve as a model in wildtype cells to ascertain the impact of miRNA stability factors.  We provide evidence that Argonaute proteins stabilize mature miRNAs in a slicing-independent manner.  Transcriptional inhibition reveals reduced half-lives of multiple endogenous miRNA guide strands in cells lacking Ago2.  This effect is reversible upon the reconstitution of Argonaute expression.  Correspondingly, overexpression of Argonaute proteins decelerates miRNA degrdn. and increases miRNA half-life.  Taken together, this study employs two model systems to identify factors altering miRNA stability and provides evidence how Argonaute proteins post-transcriptionally elevate mature miRNA levels via increasing miRNA stability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpchZzDOzAubrVg90H21EOLACvtfcHk0lgNnOkIEi4pdg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XkslCjtL0%253D&md5=a46f2eb9ed471256ce101ad4ff77fcec</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.4161%2Frna.8.6.17665&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252Frna.8.6.17665%26sid%3Dliteratum%253Aachs%26aulast%3DWinter%26aufirst%3DJ.%26aulast%3DDiederichs%26aufirst%3DS.%26atitle%3DArgonaute%2520proteins%2520regulate%2520microRNA%2520stability%253A%2520Increased%2520microRNA%2520abundance%2520by%2520Argonaute%2520proteins%2520is%2520due%2520to%2520microRNA%2520stabilization%26jtitle%3DRNA%2520Biol.%26date%3D2011%26volume%3D8%26spage%3D1149%26epage%3D1157%26doi%3D10.4161%2Frna.8.6.17665" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Warner, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bridge, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hewitson, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hodgkinson, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heyam, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Massa, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haslam, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chatzifrangkeskou, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plevin, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharp, T. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lagos, D.</span></span> <span> </span><span class="NLM_article-title">S6K2-mediated regulation of TRBP as a determinant of miRNA expression in human primary lymphatic endothelial cells</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>44</i></span>,  <span class="NLM_fpage">9942</span>– <span class="NLM_lpage">9955</span>, <span class="refDoi"> DOI: 10.1093/nar/gkw631</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1093%2Fnar%2Fgkw631" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=27407113" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvVCltr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&pages=9942-9955&author=M.+J.+Warnerauthor=K.+S.+Bridgeauthor=J.+P.+Hewitsonauthor=M.+R.+Hodgkinsonauthor=A.+Heyamauthor=B.+C.+Massaauthor=J.+C.+Haslamauthor=M.+Chatzifrangkeskouauthor=G.+J.+Evansauthor=M.+J.+Plevinauthor=T.+V.+Sharpauthor=D.+Lagos&title=S6K2-mediated+regulation+of+TRBP+as+a+determinant+of+miRNA+expression+in+human+primary+lymphatic+endothelial+cells&doi=10.1093%2Fnar%2Fgkw631"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">S6K2-mediated regulation of TRBP as a determinant of miRNA expression in human primary lymphatic endothelial cells</span></div><div class="casAuthors">Warner, Matthew J.; Bridge, Katherine S.; Hewitson, James P.; Hodgkinson, Michael R.; Heyam, Alex; Massa, Bailey C.; Haslam, Jessica C.; Chatzifrangkeskou, Maria; Evans, Gareth J. O.; Plevin, Michael J.; Sharp, Tyson V.; Lagos, Dimitris</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">9942-9955</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">MicroRNAs (miRNAs) are short non-coding RNAs that silence mRNAs.  They are generated following transcription and cleavage by the DROSHA/DGCR8 and DICER/TRBP/PACT complexes.  Although it is known that components of the miRNA biogenesis machinery can be phosphorylated, it remains poorly understood how these events become engaged during physiol. cellular activation.  We demonstrate that S6 kinases can phosphorylate the extended C-terminal domain of TRBP and interact with TRBP in situ in primary cells.  TRBP serines 283/286 are essential for S6K-mediated TRBP phosphorylation, optimal expression of TRBP, and the S6K-TRBP interaction in human primary cells.  We demonstrate the functional relevance of this interaction in primary human dermal lymphatic endothelial cells (HDLECs).  Angiopoietin-1 (ANG1) can augment miRNA biogenesis in HDLECs through enhancing TRBP phosphorylation and expression in an S6K2-dependent manner.  We propose that the S6K2/TRBP node controls miRNA biogenesis in HDLECs and provides a mol. link between the mTOR pathway and the miRNA biogenesis machinery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoOqwWieJEx7rVg90H21EOLACvtfcHk0lj0VwL0gwL-Ug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvVCltr8%253D&md5=548bd1b1067891398081068e7891713f</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkw631&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkw631%26sid%3Dliteratum%253Aachs%26aulast%3DWarner%26aufirst%3DM.%2BJ.%26aulast%3DBridge%26aufirst%3DK.%2BS.%26aulast%3DHewitson%26aufirst%3DJ.%2BP.%26aulast%3DHodgkinson%26aufirst%3DM.%2BR.%26aulast%3DHeyam%26aufirst%3DA.%26aulast%3DMassa%26aufirst%3DB.%2BC.%26aulast%3DHaslam%26aufirst%3DJ.%2BC.%26aulast%3DChatzifrangkeskou%26aufirst%3DM.%26aulast%3DEvans%26aufirst%3DG.%2BJ.%26aulast%3DPlevin%26aufirst%3DM.%2BJ.%26aulast%3DSharp%26aufirst%3DT.%2BV.%26aulast%3DLagos%26aufirst%3DD.%26atitle%3DS6K2-mediated%2520regulation%2520of%2520TRBP%2520as%2520a%2520determinant%2520of%2520miRNA%2520expression%2520in%2520human%2520primary%2520lymphatic%2520endothelial%2520cells%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2016%26volume%3D44%26spage%3D9942%26epage%3D9955%26doi%3D10.1093%2Fnar%2Fgkw631" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Paroo, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Q.</span></span> <span> </span><span class="NLM_article-title">Phosphorylation of the human microRNA-generating complex mediates MAPK/Erk signaling</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>139</i></span>,  <span class="NLM_fpage">112</span>– <span class="NLM_lpage">122</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2009.06.044</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1016%2Fj.cell.2009.06.044" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=19804757" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsVCjs7jF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=139&publication_year=2009&pages=112-122&author=Z.+Parooauthor=X.+Yeauthor=S.+Chenauthor=Q.+Liu&title=Phosphorylation+of+the+human+microRNA-generating+complex+mediates+MAPK%2FErk+signaling&doi=10.1016%2Fj.cell.2009.06.044"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Phosphorylation of the human microRNA-generating complex mediates MAPK/Erk signaling</span></div><div class="casAuthors">Paroo, Zain; Ye, Xuecheng; Chen, She; Liu, Qinghua</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">139</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">112-122</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">MicroRNAs (miRNAs) govern an expanding no. of biol. and disease processes.  Understanding the mechanisms by which the miRNA pathway is regulated, therefore, represents an important area of investigation.  The authors detd. that the human miRNA-generating complex is comprised of Dicer and phospho-TAR RNA binding protein (TRBP) isoforms.  Phosphorylation of TRBP is mediated by the MAP kinase (MAPK) Erk1/2.  Expression of phospho-mimic TRBP and TRBP phosphorylation enhanced miRNA prodn. by increasing the stability of the miRNA-generating complex.  Mitogenic signaling in response to serum and the tumor promoter, phorbol-12-myristate-13-acetate (PMA), was dependent on TRBP phosphorylation.  These effects were accompanied by a coordinated increase in levels of growth-promoting miRNA and reduced expression of let-7 tumor suppressor miRNA.  Conversely, pharmacol. inhibition of MAPK/Erk1/2 resulted in an anti-growth miRNA profile.  Taken together, these studies indicate that the MAPK/Erk1/2 pathway regulates the miRNA machinery and suggest a general principle, wherein signaling systems target the miRNA pathway to achieve biol. responses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCm24qq1BiC7Vg90H21EOLACvtfcHk0lj0VwL0gwL-Ug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsVCjs7jF&md5=dd298ee5b41196f93ed166c1595fb756</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2009.06.044&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2009.06.044%26sid%3Dliteratum%253Aachs%26aulast%3DParoo%26aufirst%3DZ.%26aulast%3DYe%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DQ.%26atitle%3DPhosphorylation%2520of%2520the%2520human%2520microRNA-generating%2520complex%2520mediates%2520MAPK%252FErk%2520signaling%26jtitle%3DCell%26date%3D2009%26volume%3D139%26spage%3D112%26epage%3D122%26doi%3D10.1016%2Fj.cell.2009.06.044" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">De
Vito, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riggi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cornaz, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suva, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baumer, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Provero, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stamenkovic, I.</span></span> <span> </span><span class="NLM_article-title">A TARBP2-dependent miRNA expression profile underlies cancer stem cell properties and provides candlidate therapeutic reagents in ewing sarcoma</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">807</span>– <span class="NLM_lpage">821</span>, <span class="refDoi"> DOI: 10.1016/j.ccr.2012.04.023</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1016%2Fj.ccr.2012.04.023" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=22698405" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BC38XosVeht7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2012&pages=807-821&author=C.+De%0AVitoauthor=N.+Riggiauthor=S.+Cornazauthor=M.+L.+Suvaauthor=K.+Baumerauthor=P.+Proveroauthor=I.+Stamenkovic&title=A+TARBP2-dependent+miRNA+expression+profile+underlies+cancer+stem+cell+properties+and+provides+candlidate+therapeutic+reagents+in+ewing+sarcoma&doi=10.1016%2Fj.ccr.2012.04.023"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">A TARBP2-Dependent miRNA Expression Profile Underlies Cancer Stem Cell Properties and Provides Candidate Therapeutic Reagents in Ewing Sarcoma</span></div><div class="casAuthors">De Vito, Claudio; Riggi, Nicolo; Cornaz, Sandrine; Suva, Mario-Luca; Baumer, Karine; Provero, Paolo; Stamenkovic, Ivan</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">807-821</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">We have recently demonstrated that human pediatric mesenchymal stem cells can be reprogrammed toward a Ewing sarcoma family tumor (ESFT) cancer stem cell (CSC) phenotype by mechanisms that implicate microRNAs (miRNAs).  Here, we show that the miRNA profile of ESFT CSCs is shared by embryonic stem cells and CSCs from divergent tumor types.  We also provide evidence that the miRNA profile of ESFT CSCs is the result of reversible disruption of TARBP2-dependent miRNA maturation.  Restoration of TARBP2 activity and systemic delivery of synthetic forms of either of two of its targets, miRNA-143 or miRNA-145, inhibited ESFT CSC clonogenicity and tumor growth in vivo.  Our observations suggest that CSC self-renewal and tumor maintenance may depend on deregulation of TARBP2-dependent miRNA expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZC0RNr45QabVg90H21EOLACvtfcHk0lgB82Y5X6AEjA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XosVeht7c%253D&md5=8b77858eeb1cf052a1ec9c0d39040932</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2012.04.023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2012.04.023%26sid%3Dliteratum%253Aachs%26aulast%3DDe%2BVito%26aufirst%3DC.%26aulast%3DRiggi%26aufirst%3DN.%26aulast%3DCornaz%26aufirst%3DS.%26aulast%3DSuva%26aufirst%3DM.%2BL.%26aulast%3DBaumer%26aufirst%3DK.%26aulast%3DProvero%26aufirst%3DP.%26aulast%3DStamenkovic%26aufirst%3DI.%26atitle%3DA%2520TARBP2-dependent%2520miRNA%2520expression%2520profile%2520underlies%2520cancer%2520stem%2520cell%2520properties%2520and%2520provides%2520candlidate%2520therapeutic%2520reagents%2520in%2520ewing%2520sarcoma%26jtitle%3DCancer%2520Cell%26date%3D2012%26volume%3D21%26spage%3D807%26epage%3D821%26doi%3D10.1016%2Fj.ccr.2012.04.023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span> <span> </span><span class="NLM_article-title">The role of TARBP2 in the development and progression of cancers</span>. <i>Tumor Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>37</i></span>,  <span class="NLM_fpage">57</span>– <span class="NLM_lpage">60</span>, <span class="refDoi"> DOI: 10.1007/s13277-015-4273-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1007%2Fs13277-015-4273-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslekurvE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2016&pages=57-60&author=X.+Yuauthor=Z.+Li&title=The+role+of+TARBP2+in+the+development+and+progression+of+cancers&doi=10.1007%2Fs13277-015-4273-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">The role of TARBP2 in the development and progression of cancers</span></div><div class="casAuthors">Yu, Xin; Li, Zheng</div><div class="citationInfo"><span class="NLM_cas:title">Tumor Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">57-60</span>CODEN:
                <span class="NLM_cas:coden">TUMBEA</span>;
        ISSN:<span class="NLM_cas:issn">1010-4283</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">TARBP2 is a RNA-binding protein (RBP) involved in miRNA processing and maturation.  TARBP2 plays significant roles in many biol. and pathol. conditions, including viral expression of HIV-1, microsatellite instability, cancer stem cell properties, and tumor progression.  Overexpression of TARBP2 was obsd. in many cancers such as prostate cancer, cutaneous malignant melanoma, and adrenocortical carcinoma.  In addn., TARBP2 was also found to be downregulated in some cancers including colorectal cancer, gastric cancer, Ewing sarcoma, and upper urinary tract urothelial carcinoma.  Therefore, whether TARBP2 functions as the tumor suppressor or tumor promoter is conflicting.  In the present review, we provide an overview of current knowledge concerning the role of TARBP2 in tumor development and progression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpDzi0qOWRxhbVg90H21EOLACvtfcHk0lgB82Y5X6AEjA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslekurvE&md5=d30827a7a0b48c0d7d50582ca30e13ea</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1007%2Fs13277-015-4273-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs13277-015-4273-6%26sid%3Dliteratum%253Aachs%26aulast%3DYu%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DZ.%26atitle%3DThe%2520role%2520of%2520TARBP2%2520in%2520the%2520development%2520and%2520progression%2520of%2520cancers%26jtitle%3DTumor%2520Biol.%26date%3D2016%26volume%3D37%26spage%3D57%26epage%3D60%26doi%3D10.1007%2Fs13277-015-4273-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Felicetti, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cecchetti, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manfroni, G.</span></span> <span> </span><span class="NLM_article-title">Modulating microRNA Processing: Enoxacin, the Progenitor of a New Class of Drugs</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">12275</span>– <span class="NLM_lpage">12289</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c00510</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00510" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtl2rsrnL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=12275-12289&author=T.+Felicettiauthor=V.+Cecchettiauthor=G.+Manfroni&title=Modulating+microRNA+Processing%3A+Enoxacin%2C+the+Progenitor+of+a+New+Class+of+Drugs&doi=10.1021%2Facs.jmedchem.0c00510"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Modulating microRNA Processing: Enoxacin, the Progenitor of a New Class of Drugs</span></div><div class="casAuthors">Felicetti, Tommaso; Cecchetti, Violetta; Manfroni, Giuseppe</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">12275-12289</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The RNA interference (RNAi) process encompasses the cellular mechanisms by which short-noncoding RNAs posttranscriptionally modulate gene expression.  First discovered in 1998, today RNAi represents the foundation underlying complex biol. mechanisms that are dysregulated in many diseases.  MicroRNAs are effector mols. of gene silencing in RNAi, and their modulation can lead to a wide response in cells.  Enoxacin was reported as the first and unique small-mol. enhancer of microRNA (SMER) maturation.  Herein, the biol. activity of enoxacin as SMER is discussed to shed light on its innovative mode of action, its potential in treating different diseases, and the feasibility of using enoxacin as a chem. template for inspiring medicinal chemists.  We debate its mechanism of action at the mol. level and the possible impact on future ligand and/or structure-guided chem. optimizations, as well as opportunities and drawbacks assocd. with the development of quinolones such as SMERs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrCQcmft_rGlLVg90H21EOLACvtfcHk0lgR3YSZOaATKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtl2rsrnL&md5=81a379dd43abf95a4b23366c006db178</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00510&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00510%26sid%3Dliteratum%253Aachs%26aulast%3DFelicetti%26aufirst%3DT.%26aulast%3DCecchetti%26aufirst%3DV.%26aulast%3DManfroni%26aufirst%3DG.%26atitle%3DModulating%2520microRNA%2520Processing%253A%2520Enoxacin%252C%2520the%2520Progenitor%2520of%2520a%2520New%2520Class%2520of%2520Drugs%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D12275%26epage%3D12289%26doi%3D10.1021%2Facs.jmedchem.0c00510" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ikeya, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taguchi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yosioka, I.</span></span> <span> </span><span class="NLM_article-title">The constituents of Schizandra-chinensis baill.10. The structures of gamma-schizandrin and 4 New Lignans, (−)-Gomisins L1 and L2, (+1)-Gomisin M1 and (+)-Gomisin M2</span>. <i>Chem. Pharm. Bull.</i> <span class="NLM_year" style="font-weight: bold;">1982</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">132</span>– <span class="NLM_lpage">139</span>, <span class="refDoi"> DOI: 10.1248/cpb.30.132</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1248%2Fcpb.30.132" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADyaL38XksFGmt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=1982&pages=132-139&author=Y.+Ikeyaauthor=H.+Taguchiauthor=I.+Yosioka&title=The+constituents+of+Schizandra-chinensis+baill.10.+The+structures+of+gamma-schizandrin+and+4+New+Lignans%2C+%28%E2%88%92%29-Gomisins+L1+and+L2%2C+%28%2B1%29-Gomisin+M1+and+%28%2B%29-Gomisin+M2&doi=10.1248%2Fcpb.30.132"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">The constituents of Schizandra chinensis Baill.  X.  The structures of γ-schizandrin and four new lignans, (-)-gomisins L1 and L2 (±)-gomisin M1 and (+)-gomisin M2</span></div><div class="casAuthors">Ikeya, Yukinobu; Taguchi, Heihachiro; Yosioka, Itiro</div><div class="citationInfo"><span class="NLM_cas:title">Chemical & Pharmaceutical Bulletin</span>
        (<span class="NLM_cas:date">1982</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">132-9</span>CODEN:
                <span class="NLM_cas:coden">CPBTAL</span>;
        ISSN:<span class="NLM_cas:issn">0009-2363</span>.
    </div><div class="casAbstract">Four new title lignans,(-)-gomisin L1 (I, R = H, R1 = Me) and L2 (I, R = Me, R1 = H), (±)-gomisin M1 (II, R2 = H, R3 = Me) and (+)-gomisin (II, R2 = Me, R3 = H), were isolated from the fruits of Schizandra chinensis and their structures detd. by chem. and spectral studies.  The stereostructure of γ-schizandrin (III, dl-form), the plane structure of which had already been suggested (N. K. Kochetkov, et al. 1964 and C. S. Liu, et al. 1978) was also elucidated on the basis of chem. and spectral studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPz7LUyIN4OrVg90H21EOLACvtfcHk0lgR3YSZOaATKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL38XksFGmt7w%253D&md5=2a1b6c69669f058b837cfa4b2ac46319</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1248%2Fcpb.30.132&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1248%252Fcpb.30.132%26sid%3Dliteratum%253Aachs%26aulast%3DIkeya%26aufirst%3DY.%26aulast%3DTaguchi%26aufirst%3DH.%26aulast%3DYosioka%26aufirst%3DI.%26atitle%3DThe%2520constituents%2520of%2520Schizandra-chinensis%2520baill.10.%2520The%2520structures%2520of%2520gamma-schizandrin%2520and%25204%2520New%2520Lignans%252C%2520%2528%25E2%2588%2592%2529-Gomisins%2520L1%2520and%2520L2%252C%2520%2528%252B1%2529-Gomisin%2520M1%2520and%2520%2528%252B%2529-Gomisin%2520M2%26jtitle%3DChem.%2520Pharm.%2520Bull.%26date%3D1982%26volume%3D30%26spage%3D132%26epage%3D139%26doi%3D10.1248%2Fcpb.30.132" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Z. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, X. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, J.</span></span> <span> </span><span class="NLM_article-title">Dibenzocyclooctadiene lignans from Schisandra chinensis and their inhibitory activity on NO production in lipopolysaccharide-activated microglia cells</span>. <i>Phytochemistry</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>104</i></span>,  <span class="NLM_fpage">72</span>– <span class="NLM_lpage">78</span>, <span class="refDoi"> DOI: 10.1016/j.phytochem.2014.04.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1016%2Fj.phytochem.2014.04.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=24864016" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BC2cXoslCmsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=104&publication_year=2014&pages=72-78&author=D.+Huauthor=Z.+Y.+Yangauthor=X.+C.+Yaoauthor=H.+Wangauthor=N.+Hanauthor=Z.+H.+Liuauthor=Y.+Wangauthor=J.+Y.+Yangauthor=J.+Yin&title=Dibenzocyclooctadiene+lignans+from+Schisandra+chinensis+and+their+inhibitory+activity+on+NO+production+in+lipopolysaccharide-activated+microglia+cells&doi=10.1016%2Fj.phytochem.2014.04.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Dibenzocyclooctadiene lignans from Schisandra chinensis and their inhibitory activity on NO production in lipopolysaccharide-activated microglia cells</span></div><div class="casAuthors">Hu, Di; Yang, Zhiyou; Yao, Xuechun; Wang, Hua; Han, Na; Liu, Zhihui; Wang, Yu; Yang, Jingyu; Yin, Jun</div><div class="citationInfo"><span class="NLM_cas:title">Phytochemistry (Elsevier)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">104</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">72-78</span>CODEN:
                <span class="NLM_cas:coden">PYTCAS</span>;
        ISSN:<span class="NLM_cas:issn">0031-9422</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Four dibenzocyclooctadiene lignans, schisanchinins A-D, and 10 known compds. were isolated from the EtOAc ext. of fruits of Schisandra chinensis (Turcz.) Baill.  Structures of compds. 1-4 were elucidated using a combination of spectroscopic techniques, including MS, UV and IR, NMR (1H NMR, 13C NMR, HMQC, HMBC).  The stereochem. of the chiral centers and the biphenyl configuration were detd. using NOESY, as well as anal. of CD spectra.  In vitro activity assays showed that 11 of the 14 compds. exhibited inhibitory activity on lipopolysaccharide (LPS)-induced NO release in primary murine BV2 microglia cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzoyq4inTCY7Vg90H21EOLACvtfcHk0liSIdwgC3kFSA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXoslCmsr4%253D&md5=d547bb4e00d23b5d21102f645f889524</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.phytochem.2014.04.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.phytochem.2014.04.014%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DD.%26aulast%3DYang%26aufirst%3DZ.%2BY.%26aulast%3DYao%26aufirst%3DX.%2BC.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DHan%26aufirst%3DN.%26aulast%3DLiu%26aufirst%3DZ.%2BH.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DJ.%2BY.%26aulast%3DYin%26aufirst%3DJ.%26atitle%3DDibenzocyclooctadiene%2520lignans%2520from%2520Schisandra%2520chinensis%2520and%2520their%2520inhibitory%2520activity%2520on%2520NO%2520production%2520in%2520lipopolysaccharide-activated%2520microglia%2520cells%26jtitle%3DPhytochemistry%26date%3D2014%26volume%3D104%26spage%3D72%26epage%3D78%26doi%3D10.1016%2Fj.phytochem.2014.04.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, C. L.</span></span> <span> </span><span class="NLM_article-title">High throughput screening and antioxidant assay of dibenzo[a, c]cyclooctadiene lignans in modified-ultrasonic and supercritical fluid extracts of Schisandra chinensis Baill by liquid chromatography-mass spectrometry and a free radical-scavenging method</span>. <i>J. Sep. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">1322</span>– <span class="NLM_lpage">1332</span>, <span class="refDoi"> DOI: 10.1002/jssc.200700443</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1002%2Fjssc.200700443" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=18446887" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BD1cXmsFOis78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2008&pages=1322-1332&author=M.+C.+Wangauthor=Y.+C.+Laiauthor=C.+L.+Chang&title=High+throughput+screening+and+antioxidant+assay+of+dibenzo%5Ba%2C+c%5Dcyclooctadiene+lignans+in+modified-ultrasonic+and+supercritical+fluid+extracts+of+Schisandra+chinensis+Baill+by+liquid+chromatography-mass+spectrometry+and+a+free+radical-scavenging+method&doi=10.1002%2Fjssc.200700443"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">High throughput screening and antioxidant assay of dibenzo[a,c]cyclooctadiene lignans in modified-ultrasonic and supercritical fluid extracts of Schisandra chinensis Baill by liquid chromatography-mass spectrometry and a free radical-scavenging method</span></div><div class="casAuthors">Wang, Ming-Chih; Lai, Yih-Cherng; Chang, Chia-Lin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Separation Science</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1322-1332</span>CODEN:
                <span class="NLM_cas:coden">JSSCCJ</span>;
        ISSN:<span class="NLM_cas:issn">1615-9306</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Dibenzo[a,c]cyclooctadiene lignans of Schisandra chinensis are well known because of their hepatoprotective activity, antioxidant activity, and anticancer effect.  For the isolation of the dibenzo[a,c]cyclooctadiene lignans of Schisandra chinensis two extn. methods were used: modified-ultrasonic extn. and supercrit. fluid extn.  A specific and fast anal. method for structure identification is established for quality control because structure elucidation could be accomplished by means of liq. chromatog.-mass spectrometry (LC-MS) technologies.  The sepn. and identification of the compds. were completed by: (i) a water-acetonitrile gradient system using a C18 reversed-phase column; (ii) UV detection at 225 nm; (iii) MS/MS expts. with electrospray ionization interface (ESI) ion trap mass spectrometry in the pos. mode.  Normalized collision energy was used to obtain fragment ions of structural relevance in the LC-MS/MS.  These results provided a reliable LC-MS/MS method for the detn. of the dibenzo[a,c]cyclooctadiene lignans from Schisandra chinensis.  Finally, we also detected 1,1-diphenyl-2-picrylhydrazyl (DPPH) scavenging effects (%) of the modified-ultrasonic and supercrit. fluid exts. of Schisandra chinensis compared with 6-hydroxy-2,5,7,8-tetramethylchroman-2-carboxylic acid (Trolox).  The antioxidant activities of the modified-ultrasonic and supercrit. fluid exts. were lower than that of Trolox.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrvYEP231BrBLVg90H21EOLACvtfcHk0liSIdwgC3kFSA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXmsFOis78%253D&md5=fefe69afa3599afffb30530b6637a059</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1002%2Fjssc.200700443&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjssc.200700443%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DM.%2BC.%26aulast%3DLai%26aufirst%3DY.%2BC.%26aulast%3DChang%26aufirst%3DC.%2BL.%26atitle%3DHigh%2520throughput%2520screening%2520and%2520antioxidant%2520assay%2520of%2520dibenzo%255Ba%252C%2520c%255Dcyclooctadiene%2520lignans%2520in%2520modified-ultrasonic%2520and%2520supercritical%2520fluid%2520extracts%2520of%2520Schisandra%2520chinensis%2520Baill%2520by%2520liquid%2520chromatography-mass%2520spectrometry%2520and%2520a%2520free%2520radical-scavenging%2520method%26jtitle%3DJ.%2520Sep.%2520Sci.%26date%3D2008%26volume%3D31%26spage%3D1322%26epage%3D1332%26doi%3D10.1002%2Fjssc.200700443" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Poornima, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anand Kumar, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siva, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Venkanna, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vadaparthi, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tiwari, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suresh Babu, K.</span></span> <span> </span><span class="NLM_article-title">Advanced glycation end-products inhibitors isolated from Schisandra grandiflora</span>. <i>Nat. Prod. Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">493</span>– <span class="NLM_lpage">496</span>, <span class="refDoi"> DOI: 10.1080/14786419.2015.1024117</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1080%2F14786419.2015.1024117" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=25812677" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BC2MXlsFGlt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2016&pages=493-496&author=B.+Poornimaauthor=D.+Anand+Kumarauthor=B.+Sivaauthor=A.+Venkannaauthor=P.+R.+Vadaparthiauthor=K.+Kumarauthor=A.+K.+Tiwariauthor=K.+Suresh+Babu&title=Advanced+glycation+end-products+inhibitors+isolated+from+Schisandra+grandiflora&doi=10.1080%2F14786419.2015.1024117"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Advanced glycation end-products inhibitors isolated from Schisandra grandiflora</span></div><div class="casAuthors">Poornima, B.; Anand Kumar, D.; Siva, Bandi; Venkanna, A.; Vadaparthi, P. R. Rao; Kumar, K.; Tiwari, Ashok K.; Suresh Babu, K.</div><div class="citationInfo"><span class="NLM_cas:title">Natural Product Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">493-496</span>CODEN:
                <span class="NLM_cas:coden">NPRAAT</span>;
        ISSN:<span class="NLM_cas:issn">1478-6419</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">Free radicals scavenging and advanced glycation end-products (AGEs) inhibitory potentials in crude chloroform ext. of Schisandra grandiflora were evaluated.  Bioassay-guided isolation of the chloroform ext. led to the identification of 24 compds.  Among the isolates, (±)-gomisin M1, arisantetralone C and D, macelignan, saurulignan B and SZ-MO displayed potent-free radical scavenging as well as AGEs inhibitory potentials.  This is the first report identifying the presence of AGEs inhibitory activity and assigning AGEs inhibitory activity to these compds.  Therefore, our research finds new application of traditional medicinal plant S. grandiflora having capacity to reduce formation and accumulation of AGEs in diabetes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpQpv_FcubLfbVg90H21EOLACvtfcHk0lgCeQXT6zk38g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXlsFGlt7w%253D&md5=b5822177bf0ed1f9b089738875c98e0f</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1080%2F14786419.2015.1024117&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F14786419.2015.1024117%26sid%3Dliteratum%253Aachs%26aulast%3DPoornima%26aufirst%3DB.%26aulast%3DAnand%2BKumar%26aufirst%3DD.%26aulast%3DSiva%26aufirst%3DB.%26aulast%3DVenkanna%26aufirst%3DA.%26aulast%3DVadaparthi%26aufirst%3DP.%2BR.%26aulast%3DKumar%26aufirst%3DK.%26aulast%3DTiwari%26aufirst%3DA.%2BK.%26aulast%3DSuresh%2BBabu%26aufirst%3DK.%26atitle%3DAdvanced%2520glycation%2520end-products%2520inhibitors%2520isolated%2520from%2520Schisandra%2520grandiflora%26jtitle%3DNat.%2520Prod.%2520Res.%26date%3D2016%26volume%3D30%26spage%3D493%26epage%3D496%26doi%3D10.1080%2F14786419.2015.1024117" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fujihashi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hara, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakata, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mori, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Higuchi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaji, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaji, A.</span></span> <span> </span><span class="NLM_article-title">Anti-human immunodeficiency virus (HIV) activities of halogenated Gomisin J derivatives, new nonnucleoside inhibitors of HIV type 1 reverse transcriptase</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">2000</span>– <span class="NLM_lpage">2007</span>, <span class="refDoi"> DOI: 10.1128/AAC.39.9.2000</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1128%2FAAC.39.9.2000" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=8540706" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADyaK2MXnvVygsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=1995&pages=2000-2007&author=T.+Fujihashiauthor=H.+Haraauthor=T.+Sakataauthor=K.+Moriauthor=H.+Higuchiauthor=A.+Tanakaauthor=H.+Kajiauthor=A.+Kaji&title=Anti-human+immunodeficiency+virus+%28HIV%29+activities+of+halogenated+Gomisin+J+derivatives%2C+new+nonnucleoside+inhibitors+of+HIV+type+1+reverse+transcriptase&doi=10.1128%2FAAC.39.9.2000"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Anti-human immunodeficiency virus (HIV) activities of halogenated gomisin J derivatives, new nonnucleoside inhibitors of HIV type 1 reverse transcriptase</span></div><div class="casAuthors">Fujihashi, Toshiaki; Hara, Hiroto; Sakata, Toshiya; Mori, Kazuya; Higuchi, Hirotaka; Tanaka, Akio; Kaji, Hideko; Kaji, Akira</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2000-7</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">0066-4804</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Halogenated gomisin J (a deriv. of lignan compd.), represented by the bromine deriv. 1506 {(6R,7S,S-biar)-4,9-dibromo-3,10-dihydroxy-1,2,11,12-tetramethoxy-6,7-dimethyl-5,6,7.8-tetrahydrodibenzo[a,c]cyclooctene}, was a potent inhibitor of the cytopathic effects of human immunodeficiency virus type 1 (HIV-1) on MT-4 human T cells (50% ED, 0.1 to 0.5 μM).  Gomisin J derivs. were active in preventing p24 prodn. from acutely HIV-1-infected H9 cells.  The selective indexes (toxic dose/ED) of these compds. were as high as >300 in some systems.  1506 Was active against 3'-azido-3'-deoxythymidine-resistant HIV-1 and acted synergistically with AZT and 2',3'-ddC.  1506 Inhibited HIV-1 reverse transcriptase (RT) in vitro but not HIV-1 protease.  From the time-of-addn. expt., 1506 was found to inhibit the early phase of the HIV life cycle.  A 1506-resistant HIV mutant was selected and shown to possess a mutation within the RT-coding region (at position 188 [Tyr to Leu]).  The mutant RT expressed in Escherichia coli was resistant to 1506 in the in vitro RT assay.  Some of the HIV strains resistant to other nonnucleoside HIV-1 RT inhibitors were also resistant to 1506.  Comparison of various gomisin J derivs. with gomisin J showed that iodine, bromine, and chlorine in the fourth and ninth positions increased RT inhibitory activity as well as cytoprotective activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoy4YdCT_tsTbVg90H21EOLACvtfcHk0lgCeQXT6zk38g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXnvVygsbg%253D&md5=ec3f5e4b2f8f62ac91c9074436ffd615</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1128%2FAAC.39.9.2000&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.39.9.2000%26sid%3Dliteratum%253Aachs%26aulast%3DFujihashi%26aufirst%3DT.%26aulast%3DHara%26aufirst%3DH.%26aulast%3DSakata%26aufirst%3DT.%26aulast%3DMori%26aufirst%3DK.%26aulast%3DHiguchi%26aufirst%3DH.%26aulast%3DTanaka%26aufirst%3DA.%26aulast%3DKaji%26aufirst%3DH.%26aulast%3DKaji%26aufirst%3DA.%26atitle%3DAnti-human%2520immunodeficiency%2520virus%2520%2528HIV%2529%2520activities%2520of%2520halogenated%2520Gomisin%2520J%2520derivatives%252C%2520new%2520nonnucleoside%2520inhibitors%2520of%2520HIV%2520type%25201%2520reverse%2520transcriptase%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D1995%26volume%3D39%26spage%3D2000%26epage%3D2007%26doi%3D10.1128%2FAAC.39.9.2000" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Miyamoto, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wakusawa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nomura, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanae, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakai, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sudo, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohtaki, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takeda, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujii, Y.</span></span> <span> </span><span class="NLM_article-title">Effects of Gomisin A on hepatocarcinogenesis by 3′-methyl-4-dimethylaminoazobenzene in rats</span>. <i>Jpn. J. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">1991</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">71</span>– <span class="NLM_lpage">77</span>, <span class="refDoi"> DOI: 10.1254/jjp.57.71</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1254%2Fjjp.57.71" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1800800" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADyaK3MXlvFOqsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=1991&pages=71-77&author=K.+Miyamotoauthor=S.+Wakusawaauthor=M.+Nomuraauthor=F.+Sanaeauthor=R.+Sakaiauthor=K.+Sudoauthor=Y.+Ohtakiauthor=S.+Takedaauthor=Y.+Fujii&title=Effects+of+Gomisin+A+on+hepatocarcinogenesis+by+3%E2%80%B2-methyl-4-dimethylaminoazobenzene+in+rats&doi=10.1254%2Fjjp.57.71"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Effects of gomisin A on hepatocarcinogenesis by 3'-methyl-4-dimethylaminoazobenzene in rats</span></div><div class="casAuthors">Miyamoto, Kenichi; Wakusawa, Shinya; Nomura, Masaaki; Sanae, Fujiko; Sakai, Ryosuke; Sudo, Kazuhiko; Ohtaki, Yumiko; Takeda, Shigefumi; Fujii, Yuichi</div><div class="citationInfo"><span class="NLM_cas:title">Japanese Journal of Pharmacology</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">71-7</span>CODEN:
                <span class="NLM_cas:coden">JJPAAZ</span>;
        ISSN:<span class="NLM_cas:issn">0021-5198</span>.
    </div><div class="casAbstract">The effects of gomisin A (I) on tumor promotion in the liver after a short-term feeding of 3'-methyl-4-dimethylaminoazobenzene (3'-MeDAB) to rats was examd., compared with the effects of phenobarbital.  Male Donryu rats were fed ad libitum a diet contg. 0.06% 3'-MeDAB and 0.03% or 0.01% I or water contg. 0.05% phenobarbital.  I and phenobarbital did not cause any proliferative and neoplastic lesions by themselves in 40 wk of feeding.  Altered foci in the liver increased with a peak at 12 wk after the rats were fed 3'-MeDAB.  I decreased the no. of hepatic altered foci such as the clear cell and basophilic cell type foci in the early stages.  Phenobarbital enhanced neoplastic alterations so that the no. and size of the foci were much larger in the phenobarbital-combined group than in the 3'-MeDAB-control group.  Thus, phenobarbital acted as a promoter of cells initiated by 3'-MeDAB; on the other hand, I showed a weak suppressive effect on tumor promotion.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0_zPRkFFyvbVg90H21EOLACvtfcHk0li8I_DunQtl7A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3MXlvFOqsbw%253D&md5=30d6ab7a7986923a5deef12fb676ede5</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1254%2Fjjp.57.71&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1254%252Fjjp.57.71%26sid%3Dliteratum%253Aachs%26aulast%3DMiyamoto%26aufirst%3DK.%26aulast%3DWakusawa%26aufirst%3DS.%26aulast%3DNomura%26aufirst%3DM.%26aulast%3DSanae%26aufirst%3DF.%26aulast%3DSakai%26aufirst%3DR.%26aulast%3DSudo%26aufirst%3DK.%26aulast%3DOhtaki%26aufirst%3DY.%26aulast%3DTakeda%26aufirst%3DS.%26aulast%3DFujii%26aufirst%3DY.%26atitle%3DEffects%2520of%2520Gomisin%2520A%2520on%2520hepatocarcinogenesis%2520by%25203%25E2%2580%25B2-methyl-4-dimethylaminoazobenzene%2520in%2520rats%26jtitle%3DJpn.%2520J.%2520Pharmacol.%26date%3D1991%26volume%3D57%26spage%3D71%26epage%3D77%26doi%3D10.1254%2Fjjp.57.71" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Miyamoto, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hiramatsu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohtaki, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kanitani, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nomura, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aburada, M.</span></span> <span> </span><span class="NLM_article-title">Effects of Gomisin A on the promotor action and serum bile acid concentration in hepatocarcinogenesis induced by 3′-methyl-4-dimethylamino-azobenzene</span>. <i>Biol. Pharm. Bull.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">1443</span>– <span class="NLM_lpage">1445</span>, <span class="refDoi"> DOI: 10.1248/bpb.18.1443</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1248%2Fbpb.18.1443" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=8593453" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADyaK2MXpsFCnt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=1995&pages=1443-1445&author=K.+Miyamotoauthor=K.+Hiramatsuauthor=Y.+Ohtakiauthor=M.+Kanitaniauthor=M.+Nomuraauthor=M.+Aburada&title=Effects+of+Gomisin+A+on+the+promotor+action+and+serum+bile+acid+concentration+in+hepatocarcinogenesis+induced+by+3%E2%80%B2-methyl-4-dimethylamino-azobenzene&doi=10.1248%2Fbpb.18.1443"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Effects of gomisin A on the promoter action and serum bile acid concentration in hepatocarcinogenesis induced by 3'-methyl-4-dimethylamino-azobenzene</span></div><div class="casAuthors">Miyamoto, Ken-ichi; Hiramatsu, Kazuyoshi; Ohtaki, Yumiko; Kanitani, Masanao; Nomura, Masaaki; Aburada, Masaki</div><div class="citationInfo"><span class="NLM_cas:title">Biological & Pharmaceutical Bulletin</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1443-5</span>CODEN:
                <span class="NLM_cas:coden">BPBLEO</span>;
        ISSN:<span class="NLM_cas:issn">0918-6158</span>.
    
            (<span class="NLM_cas:orgname">Pharmaceutical Society of Japan</span>)
        </div><div class="casAbstract">The effects of gomisin A, a lignan component of Schizandra fruits, on the promotion stage of hepatocarcinogenesis initiated by 3'-methyl-4-dimethylamino-azobenzene (3'-MeDAB) in male Donryu rats were investigated.  When different types of tumor promoters, phenobarbital (PB) and deoxycholic acid (DCA), were administered for 5 wk after initiation by 3'-MeDAB, preneoplastic alterations in the liver, detd. by glutathione S-transferase placental form (GST-P), were markedly increased.  Gomisin A significantly inhibited the increase in no. and size of GST-P pos. foci, regardless of the promoter.  This lignan inhibited the increase in serum bile acid concn. by administration of DCA, but hardly influenced the serum bile acids in the PB-combined group.  These results suggest that the inhibitory effect of gomisin A on the promotive action of DCA is based on improving bile acid metab., but regarding the action of PB, the effect could not be elucidated from the metab. of bile acids.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrsxYCKwPleC7Vg90H21EOLACvtfcHk0li8I_DunQtl7A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXpsFCnt7Y%253D&md5=385d7f3394612fd862fec49428192b8c</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1248%2Fbpb.18.1443&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1248%252Fbpb.18.1443%26sid%3Dliteratum%253Aachs%26aulast%3DMiyamoto%26aufirst%3DK.%26aulast%3DHiramatsu%26aufirst%3DK.%26aulast%3DOhtaki%26aufirst%3DY.%26aulast%3DKanitani%26aufirst%3DM.%26aulast%3DNomura%26aufirst%3DM.%26aulast%3DAburada%26aufirst%3DM.%26atitle%3DEffects%2520of%2520Gomisin%2520A%2520on%2520the%2520promotor%2520action%2520and%2520serum%2520bile%2520acid%2520concentration%2520in%2520hepatocarcinogenesis%2520induced%2520by%25203%25E2%2580%25B2-methyl-4-dimethylamino-azobenzene%26jtitle%3DBiol.%2520Pharm.%2520Bull.%26date%3D1995%26volume%3D18%26spage%3D1443%26epage%3D1445%26doi%3D10.1248%2Fbpb.18.1443" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, X. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chou, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, J. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, Z. L.</span></span> <span> </span><span class="NLM_article-title">Gomisin N exerts anti-liver cancer effects and regulates PI3K-Akt and mTOR-ULK1 pathways in Vitro</span>. <i>Biol. Pharm. Bull.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>43</i></span>,  <span class="NLM_fpage">1267</span>– <span class="NLM_lpage">1271</span>, <span class="refDoi"> DOI: 10.1248/bpb.b20-00030</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1248%2Fbpb.b20-00030" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=32741948" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BB3cXis1equrjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2020&pages=1267-1271&author=P.+L.+Zhuauthor=D.+F.+Lamauthor=J.+K.+Liauthor=X.+Q.+Fuauthor=C.+L.+Yinauthor=J.+Y.+Chouauthor=Y.+P.+Wangauthor=Y.+X.+Liuauthor=Y.+J.+Chenauthor=J.+Y.+Wuauthor=Y.+Wuauthor=J.+X.+Baiauthor=C.+Liangauthor=Z.+L.+Yu&title=Gomisin+N+exerts+anti-liver+cancer+effects+and+regulates+PI3K-Akt+and+mTOR-ULK1+pathways+in+Vitro&doi=10.1248%2Fbpb.b20-00030"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Gomisin N exerts anti-liver cancer effects and regulates PI3K-Akt and mTOR-ULK1 pathways in vitro</span></div><div class="casAuthors">Zhu, Pei-Li; Lam, Dick Fai; Li, Jun-Kui; Fu, Xiu-Qiong; Yin, Cheng-Le; Chou, Ji-Yao; Wang, Ya-Ping; Liu, Yu-Xi; Chen, Ying-Jie; Wu, Jia-Ying; Wu, Ying; Bai, Jing-Xuan; Liang, Chun; Yu, Zhi-Ling</div><div class="citationInfo"><span class="NLM_cas:title">Biological & Pharmaceutical Bulletin</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1267-1271</span>CODEN:
                <span class="NLM_cas:coden">BPBLEO</span>;
        ISSN:<span class="NLM_cas:issn">0918-6158</span>.
    
            (<span class="NLM_cas:orgname">Pharmaceutical Society of Japan</span>)
        </div><div class="casAbstract">Primary liver cancer is a lethal cancer.  The phosphatidylinositol 3-kinase (PI3K)-Akt-mammalian target of rapamycin (mTOR) pathway has been implicated in the pathogenesis of liver cancer.  Gomisin N (GN), a lignan isolated from the dried fruits of Schisandra chinensis (Turca.) Baill., has been reported to reduce viability of, and induce apoptosis in, HepG2 liver cancer cells.  In preadipocytes, GN was found to inhibit Akt activity.  In the present study, Akt signaling-related anti-liver cancer mechanisms of GN were investigated.  We confirmed that GN reduces cell viability of, and triggers apoptosis in, more liver cancer cell lines.  Mechanistic studies revealed that GN lowers protein levels of phospho-PI3K (p85 tyrosine (Tyr)458), phospho-Akt (serine (Ser)473), and Akt downstream mols. Mcl-1 in HepG2 and HCCLM3 cells.  Meanwhile, GN activates mTOR and inhibits ULK1 (a neg. downstream effector of mTOR) activities.  Activation of mTOR has been reported to suppress ULK1 activity and repress autophagy.  Indeed, we obsd. that GN inhibits autophagy in liver cancer cells.  In summary, we for the first time demonstrated that GN inhibits the PI3K-Akt pathway and regulates the mTOR-ULK1 pathway in liver cancer cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpry92uaZ42gbVg90H21EOLACvtfcHk0lim1aAdQFbGww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXis1equrjO&md5=edc739e6983564feee24ec43b81e018f</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1248%2Fbpb.b20-00030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1248%252Fbpb.b20-00030%26sid%3Dliteratum%253Aachs%26aulast%3DZhu%26aufirst%3DP.%2BL.%26aulast%3DLam%26aufirst%3DD.%2BF.%26aulast%3DLi%26aufirst%3DJ.%2BK.%26aulast%3DFu%26aufirst%3DX.%2BQ.%26aulast%3DYin%26aufirst%3DC.%2BL.%26aulast%3DChou%26aufirst%3DJ.%2BY.%26aulast%3DWang%26aufirst%3DY.%2BP.%26aulast%3DLiu%26aufirst%3DY.%2BX.%26aulast%3DChen%26aufirst%3DY.%2BJ.%26aulast%3DWu%26aufirst%3DJ.%2BY.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DBai%26aufirst%3DJ.%2BX.%26aulast%3DLiang%26aufirst%3DC.%26aulast%3DYu%26aufirst%3DZ.%2BL.%26atitle%3DGomisin%2520N%2520exerts%2520anti-liver%2520cancer%2520effects%2520and%2520regulates%2520PI3K-Akt%2520and%2520mTOR-ULK1%2520pathways%2520in%2520Vitro%26jtitle%3DBiol.%2520Pharm.%2520Bull.%26date%3D2020%26volume%3D43%26spage%3D1267%26epage%3D1271%26doi%3D10.1248%2Fbpb.b20-00030" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yim, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, Y. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, Y. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jung, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Son, B. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, Y. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, Y. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, Y. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hwang, D. Y.</span></span> <span> </span><span class="NLM_article-title">Gomisin N isolated from Schisandra Chinensis significantly induces anti-proliferative and pro-apoptotic effects in hepatic carcinoma</span>. <i>Mol. Med. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">725</span>– <span class="NLM_lpage">732</span>, <span class="refDoi"> DOI: 10.3892/mmr_00000163</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.3892%2Fmmr_00000163" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=21475892" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BD1MXpvFGnsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2009&pages=725-732&author=S.+Y.+Yimauthor=Y.+J.+Leeauthor=Y.+K.+Leeauthor=S.+E.+Jungauthor=J.+H.+Kimauthor=H.+J.+Kimauthor=B.+G.+Sonauthor=Y.+H.+Parkauthor=Y.+G.+Leeauthor=Y.+W.+Choiauthor=D.+Y.+Hwang&title=Gomisin+N+isolated+from+Schisandra+Chinensis+significantly+induces+anti-proliferative+and+pro-apoptotic+effects+in+hepatic+carcinoma&doi=10.3892%2Fmmr_00000163"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Gomisin N isolated from Schisandra chinensis significantly induces anti-proliferative and pro-apoptotic effects in hepatic carcinoma</span></div><div class="casAuthors">Yim, Su Youn; Lee, You Jin; Lee, Yoen Kyung; Jung, Seung Eun; Kim, Ji Ha; Kim, Ji Eun; Kim, Hak Jin; Son, Beung Gu; Park, Young Hoon; Lee, Young Guen; Choi, Young Whan; Hwang, Dae Youn</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Medicine Reports</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">725-732</span>CODEN:
                <span class="NLM_cas:coden">MMROA5</span>;
        ISSN:<span class="NLM_cas:issn">1791-2997</span>.
    
            (<span class="NLM_cas:orgname">Spandidos Publications Ltd.</span>)
        </div><div class="casAbstract">Lignans isolated from Schisandria chinensis have been prescribed as anti-cancer and anti-hepatitis treatments in Chinese medicine.  To investigate the applications of lignans isolated from Schisandria chinensis in hepatic carcinoma therapy, their apoptotic ability was screened using a cell proliferation assay.  Compared to the other lignans, gomisin N induced high apoptotic levels in hepatic carcinoma.  Cell morphol. and flow cytometric anal. demonstrated that this lignan induced cell death at high concns., but did not induce any changes at low concns.  In addn., the expression levels of Bcl-2 and Bax proteins, which are involved in the apoptotic pathway, were markedly increased in only the 320 μM-treated group compared to the vehicle and other concn. groups, while the expression level of p53 protein remained unchanged in this group.  These results suggest that gomisin N is an anti-cancer drug candidate capable of inhibiting the proliferation and inducing the apoptosis of human hepatic carcinomas.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpFaeLzz2_DQrVg90H21EOLACvtfcHk0lim1aAdQFbGww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXpvFGnsro%253D&md5=4ee7484292fc20a1ebc6d5711228caac</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.3892%2Fmmr_00000163&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3892%252Fmmr_00000163%26sid%3Dliteratum%253Aachs%26aulast%3DYim%26aufirst%3DS.%2BY.%26aulast%3DLee%26aufirst%3DY.%2BJ.%26aulast%3DLee%26aufirst%3DY.%2BK.%26aulast%3DJung%26aufirst%3DS.%2BE.%26aulast%3DKim%26aufirst%3DJ.%2BH.%26aulast%3DKim%26aufirst%3DH.%2BJ.%26aulast%3DSon%26aufirst%3DB.%2BG.%26aulast%3DPark%26aufirst%3DY.%2BH.%26aulast%3DLee%26aufirst%3DY.%2BG.%26aulast%3DChoi%26aufirst%3DY.%2BW.%26aulast%3DHwang%26aufirst%3DD.%2BY.%26atitle%3DGomisin%2520N%2520isolated%2520from%2520Schisandra%2520Chinensis%2520significantly%2520induces%2520anti-proliferative%2520and%2520pro-apoptotic%2520effects%2520in%2520hepatic%2520carcinoma%26jtitle%3DMol.%2520Med.%2520Rep.%26date%3D2009%26volume%3D2%26spage%3D725%26epage%3D732%26doi%3D10.3892%2Fmmr_00000163" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tai, Z. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, F.</span></span> <span> </span><span class="NLM_article-title">Identification of small molecule activators of the janus kinase/signal transducer and activator of transcription pathway using a cell-based screen</span>. <i>Biol. Pharm. Bull.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">65</span>– <span class="NLM_lpage">71</span>, <span class="refDoi"> DOI: 10.1248/bpb.35.65</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1248%2Fbpb.35.65" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=22223339" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BC38XivFWhur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2012&pages=65-71&author=Z.+F.+Taiauthor=G.+L.+Zhangauthor=F.+Wang&title=Identification+of+small+molecule+activators+of+the+janus+kinase%2Fsignal+transducer+and+activator+of+transcription+pathway+using+a+cell-based+screen&doi=10.1248%2Fbpb.35.65"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of small molecule activators of the Janus kinase/signal transducer and activator of transcription pathway using a cell-based screen</span></div><div class="casAuthors">Tai, Zheng Fu; Zhang, Guo Lin; Wang, Fei</div><div class="citationInfo"><span class="NLM_cas:title">Biological & Pharmaceutical Bulletin</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">65-71</span>CODEN:
                <span class="NLM_cas:coden">BPBLEO</span>;
        ISSN:<span class="NLM_cas:issn">0918-6158</span>.
    
            (<span class="NLM_cas:orgname">Pharmaceutical Society of Japan</span>)
        </div><div class="casAbstract">Type I interferons (IFN-α/β) have been widely used in the treatment of many viral and malignant diseases by activation of the Janus kinase/signal transducer and activator of transcription (JAK/STAT) signaling pathway, but the side effects of protein-based IFN therapy severely limit their clin. use.  Discovering small mols. to activate the JAK/STAT pathway will greatly facilitate the development of new drugs which have similar pharmacol. function to IFNs but with fewer side effects.  To screen a natural products-based library, the authors established a cell-based screening assay using human hepatoma HepG2 cells stably transfected with a plasmid where the luciferase reporter activity is driven by interferon α-stimulated response element (ISRE), the motif specifically recognized by type I IFN-induced activation of JAK/STAT pathway.  Among 1431 natural product compds. screened, four compds. (emodin, quercetin, apigenin and luteolin) were identified as activators of the JAK/STAT pathway.  Further studies demonstrated that these four compds. could increase the endogenous antiviral gene expression regulated by the IFN-activated JAK/STAT pathway.  The identified small mol. activators are valuable for structural modification and warrant further investigation for use in new antiviral drugs as IFN mimics or adjuvants.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrLV-4Xq_79WrVg90H21EOLACvtfcHk0lhMTcgT_QKFsw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XivFWhur8%253D&md5=b67dd1467ee491480c67bf9a08e398b9</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1248%2Fbpb.35.65&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1248%252Fbpb.35.65%26sid%3Dliteratum%253Aachs%26aulast%3DTai%26aufirst%3DZ.%2BF.%26aulast%3DZhang%26aufirst%3DG.%2BL.%26aulast%3DWang%26aufirst%3DF.%26atitle%3DIdentification%2520of%2520small%2520molecule%2520activators%2520of%2520the%2520janus%2520kinase%252Fsignal%2520transducer%2520and%2520activator%2520of%2520transcription%2520pathway%2520using%2520a%2520cell-based%2520screen%26jtitle%3DBiol.%2520Pharm.%2520Bull.%26date%3D2012%26volume%3D35%26spage%3D65%26epage%3D71%26doi%3D10.1248%2Fbpb.35.65" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hui, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span> <span> </span><span class="NLM_article-title">lncRNA-SOX2OT promotes hepatocellular carcinoma invasion and metastasis through miR-122–5p-mediated activation of PKM2</span>. <i>Oncogenesis</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">54</span>– <span class="NLM_lpage">65</span>, <span class="refDoi"> DOI: 10.1038/s41389-020-0242-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1038%2Fs41389-020-0242-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=32467565" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtVyqtbzP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2020&pages=54-65&author=Y.+Liangauthor=D.+Zhangauthor=T.+Zhengauthor=G.+Yangauthor=J.+Wangauthor=F.+Mengauthor=Y.+Liuauthor=G.+Zhangauthor=L.+Zhangauthor=J.+Hanauthor=P.+Huiauthor=Z.+Chenauthor=Y.+Liuauthor=M.+Wangauthor=H.+Jiangauthor=L.+Liu&title=lncRNA-SOX2OT+promotes+hepatocellular+carcinoma+invasion+and+metastasis+through+miR-122%E2%80%935p-mediated+activation+of+PKM2&doi=10.1038%2Fs41389-020-0242-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">lncRNA-SOX2OT promotes hepatocellular carcinoma invasion and metastasis through miR-122-5p-mediated activation of PKM2</span></div><div class="casAuthors">Liang, Yingjian; Zhang, Dandan; Zheng, Tongsen; Yang, Guangchao; Wang, Jiabei; Meng, Fanzheng; Liu, Yao; Zhang, Guoli; Zhang, Linhan; Han, Jihua; Hui, Peng; Chen, Zhengliang; Liu, Yu; Wang, Mingyu; Jiang, Hongchi; Liu, Lianxin</div><div class="citationInfo"><span class="NLM_cas:title">Oncogenesis</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">54</span>CODEN:
                <span class="NLM_cas:coden">ONCOCT</span>;
        ISSN:<span class="NLM_cas:issn">2157-9024</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Abstr.: Tumor cells primarily utilize aerobic glycolysis for energy prodn., a phenomenon known as the Warburg effect, but the involvement of Warburg effect in liver cancer cell metastasis is not well understood.  In present study, our results indicate a pos. correlation between glucose metab. level and metastatic potential of hepatocellular carcinoma (HCC).  We also obsd. that a long noncoding RNA-SOX2OT (lncRNA-SOX2OT) can not only increase the metastatic potential of HCC but also promote a pyruvate kinase M2 (PKM2)-mediated activation of glucose metab.  Inhibition of PKM2 in HCC cells greatly compromises lncRNA-SOX2OT in promoting Warburg effect and metastasis.  Furthermore, miR-122-5p was found being a direct target of lncRNA-SOX2OT in regulating PKM2 expression.  Thus, our findings reveal that lncRNA-SOX2OT, a regulator of PKM2, could predispose HCC patients to metastases and may serve as a candidate for metastatic prediction and therapies in HCC patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHZu-8Nq0s_LVg90H21EOLACvtfcHk0lhMTcgT_QKFsw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtVyqtbzP&md5=c8df62ffe1784cf2147e8c1cbabca252</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1038%2Fs41389-020-0242-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41389-020-0242-z%26sid%3Dliteratum%253Aachs%26aulast%3DLiang%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DD.%26aulast%3DZheng%26aufirst%3DT.%26aulast%3DYang%26aufirst%3DG.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DMeng%26aufirst%3DF.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DHan%26aufirst%3DJ.%26aulast%3DHui%26aufirst%3DP.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DL.%26atitle%3DlncRNA-SOX2OT%2520promotes%2520hepatocellular%2520carcinoma%2520invasion%2520and%2520metastasis%2520through%2520miR-122%25E2%2580%25935p-mediated%2520activation%2520of%2520PKM2%26jtitle%3DOncogenesis%26date%3D2020%26volume%3D9%26spage%3D54%26epage%3D65%26doi%3D10.1038%2Fs41389-020-0242-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, X.</span></span> <span> </span><span class="NLM_article-title">Long non-coding RNA SNHG5 promotes human hepatocellular carcinoma progression by regulating miR-26a-5p/GSK3beta signal pathway</span>. <i>Cell Death Dis.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">888</span>, <span class="refDoi"> DOI: 10.1038/s41419-018-0882-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1038%2Fs41419-018-0882-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=30166525" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A280%3ADC%252BB3c3jvFGlsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=888&author=Y.+Liauthor=D.+Guoauthor=Y.+Zhaoauthor=M.+Renauthor=G.+Luauthor=Y.+Wangauthor=J.+Zhangauthor=C.+Miauthor=S.+Heauthor=X.+Lu&title=Long+non-coding+RNA+SNHG5+promotes+human+hepatocellular+carcinoma+progression+by+regulating+miR-26a-5p%2FGSK3beta+signal+pathway&doi=10.1038%2Fs41419-018-0882-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Long non-coding RNA SNHG5 promotes human hepatocellular carcinoma progression by regulating miR-26a-5p/GSK3β signal pathway</span></div><div class="casAuthors">Li Yarui; Guo Dan; Zhao Yan; Ren Mudan; Lu Guifang; Wang Yun; Zhang Juan; Mi Chen; He Shuixiang; Lu Xinlan</div><div class="citationInfo"><span class="NLM_cas:title">Cell death & disease</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">888</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Accumulating evidence have suggested that long non-coding RNAs (lncRNAs) had malfunctioning roles in the development of human cancers.  The present study aimed to investigate the role of lncRNA small nucleolar RNA host gene 5 (SNHG5) in hepatocellular carcinoma (HCC) progression using human tissues and cell lines.  The quantitative real-time PCR results showed that SNHG5 was up-regulated in both HCC tissues and hepatoma cell lines and was closely associated with tumor size, hepatitis B virus infection, histologic grade, TNM stage, and portal vein tumor thrombus (PVTT) in HCC patients.  Knockdown of SNHG5 induced apoptosis and repressed cell cycle progression, cell growth, and metastasis in hepatoma cell lines, whereas overexpression of SNHG5 had the opposite effects.  In vivo functional assay, xenograft tumors grown from SNHG5-knockdown cells had smaller mean volumes than the tumors grown from negative control cells.  Further investigations showed that SNHG5 may act as a competing endogenous RNA by competitively binding miR-26a-5p and thereby modulating the derepression of downstream target GSK3β, which were further confirmed by luciferase reporter assay.  Functionally, SNHG5 promotes tumor growth and metastasis by activating Wnt/β-catenin pathway and inducing epithelial to mesenchymal transition (EMT).  Taken together, SNHG5 promotes HCC progression by competitively binding miR-26a-5p and regulating GSK3β and Wnt/β-catenin signal pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTimSCl3vJjCO3MPKNEWB5RfW6udTcc2eYsw7cGjaugXrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3c3jvFGlsw%253D%253D&md5=6cf876288e32a7dcbef3ac5758b9f20e</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1038%2Fs41419-018-0882-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41419-018-0882-5%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DY.%26aulast%3DGuo%26aufirst%3DD.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DRen%26aufirst%3DM.%26aulast%3DLu%26aufirst%3DG.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DMi%26aufirst%3DC.%26aulast%3DHe%26aufirst%3DS.%26aulast%3DLu%26aufirst%3DX.%26atitle%3DLong%2520non-coding%2520RNA%2520SNHG5%2520promotes%2520human%2520hepatocellular%2520carcinoma%2520progression%2520by%2520regulating%2520miR-26a-5p%252FGSK3beta%2520signal%2520pathway%26jtitle%3DCell%2520Death%2520Dis.%26date%3D2018%26volume%3D9%26spage%3D888%26doi%3D10.1038%2Fs41419-018-0882-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shih, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tien, Y. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeh, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yen, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsieh, S. Y.</span></span> <span> </span><span class="NLM_article-title">MicroRNA-214 downregulation contributes to tumor angiogenesis by inducing secretion of the hepatoma-derived growth factor in human hepatoma</span>. <i>J. Hepatol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">584</span>– <span class="NLM_lpage">591</span>, <span class="refDoi"> DOI: 10.1016/j.jhep.2012.04.031</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1016%2Fj.jhep.2012.04.031" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=22613005" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1ensr3F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2012&pages=584-591&author=T.+C.+Shihauthor=Y.+J.+Tienauthor=C.+J.+Wenauthor=T.+S.+Yehauthor=M.+C.+Yuauthor=C.+H.+Huangauthor=Y.+S.+Leeauthor=T.+C.+Yenauthor=S.+Y.+Hsieh&title=MicroRNA-214+downregulation+contributes+to+tumor+angiogenesis+by+inducing+secretion+of+the+hepatoma-derived+growth+factor+in+human+hepatoma&doi=10.1016%2Fj.jhep.2012.04.031"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">MicroRNA-214 downregulation contributes to tumor angiogenesis by inducing secretion of the hepatoma-derived growth factor in human hepatoma</span></div><div class="casAuthors">Shih, Tsung-Chieh; Tien, Yin-Jing; Wen, Chih-Jen; Yeh, Ta-Sen; Yu, Ming-Chin; Huang, Chia-Hao; Lee, Yun-Shien; Yen, Tzu-Chen; Hsieh, Sen-Yung</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Hepatology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">584-591</span>CODEN:
                <span class="NLM_cas:coden">JOHEEC</span>;
        ISSN:<span class="NLM_cas:issn">0168-8278</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Background & Aims: Unusual hypervascularity is a hallmark of human hepatocellular carcinoma (HCC).  Although microRNA-214 (miR-214) is upregulated in other human cancers, it is downregulated in HCC.  We elucidated the biol. and clin. significance of miR-214 downregulation in HCC.  Methods: MicroRNAs deregulated in HCC were identified using array-based microRNA profiling.  A luciferase reporter assay confirmed target assocn. between miR-214 and the hepatoma-derived growth factor (HDGF).  Tube formation and in vivo angiogenesis assays validated the roles of miR-214/HDGF in angiogenesis.  Results: miR-214 downregulation was assocd. with higher tumor recurrence and worse clin. outcomes.  Ectopic expression of miR-214 suppressed xenograft tumor growth and microvascularity of the tumors and their surrounding tissues.  The genes downregulated by ectopic expression of miR-214 were involved in the regulation of apoptosis, cell cycle, and angiogenesis.  Integrated anal. disclosed HDGF as a downstream target of miR-214.  Conditioned medium of HCC cells contained bioactivity to stimulate tube formation of human umbilical vein endothelial cells, which was abolished by pretreatment of the conditioned media with HDGF antibodies, suppression of HDGF expression or ectopic expression of miR-214 in the donor HCC cells.  The angiogenic activity of the conditioned media, lost by ectopic expression of miR-214 in the donor cells, was restored by supplementation with recombinant HDGF.  In vivo tumor angiogenesis assays showed significant suppression of tumor vascularity by ectopic expression of miR-214.  Conclusions: A novel role of microRNA in tumorigenesis is identified.  Downregulation of miR-214 contributes to the unusual hypervascularity of HCC via activation of the HDGF paracrine pathway for tumor angiogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr9BJn4AWzRhLVg90H21EOLACvtfcHk0lj7dKMoFR-YSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1ensr3F&md5=43a43fe1b4a0d42ce377c1682eae6686</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2Fj.jhep.2012.04.031&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jhep.2012.04.031%26sid%3Dliteratum%253Aachs%26aulast%3DShih%26aufirst%3DT.%2BC.%26aulast%3DTien%26aufirst%3DY.%2BJ.%26aulast%3DWen%26aufirst%3DC.%2BJ.%26aulast%3DYeh%26aufirst%3DT.%2BS.%26aulast%3DYu%26aufirst%3DM.%2BC.%26aulast%3DHuang%26aufirst%3DC.%2BH.%26aulast%3DLee%26aufirst%3DY.%2BS.%26aulast%3DYen%26aufirst%3DT.%2BC.%26aulast%3DHsieh%26aufirst%3DS.%2BY.%26atitle%3DMicroRNA-214%2520downregulation%2520contributes%2520to%2520tumor%2520angiogenesis%2520by%2520inducing%2520secretion%2520of%2520the%2520hepatoma-derived%2520growth%2520factor%2520in%2520human%2520hepatoma%26jtitle%3DJ.%2520Hepatol.%26date%3D2012%26volume%3D57%26spage%3D584%26epage%3D591%26doi%3D10.1016%2Fj.jhep.2012.04.031" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tian, X. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, X. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, F. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yun, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, R. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, Q. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, D.</span></span> <span> </span><span class="NLM_article-title">Acidic microenvironment Up-regulates exosomal miR-21 and miR-10b in early-stage hepatocellular carcinoma to promote cancer cell proliferation and metastasis</span>. <i>Theranostics</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1965</span>– <span class="NLM_lpage">1979</span>, <span class="refDoi"> DOI: 10.7150/thno.30958</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.7150%2Fthno.30958" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=31037150" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtlGru7bN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2019&pages=1965-1979&author=X.+P.+Tianauthor=C.+Y.+Wangauthor=X.+H.+Jinauthor=M.+Liauthor=F.+W.+Wangauthor=W.+J.+Huangauthor=J.+P.+Yunauthor=R.+H.+Xuauthor=Q.+Q.+Caiauthor=D.+Xie&title=Acidic+microenvironment+Up-regulates+exosomal+miR-21+and+miR-10b+in+early-stage+hepatocellular+carcinoma+to+promote+cancer+cell+proliferation+and+metastasis&doi=10.7150%2Fthno.30958"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Acidic Microenvironment Up-Regulates Exosomal miR-21 and miR-10b in Early-Stage Hepatocellular Carcinoma to Promote Cancer Cell Proliferation and Metastasis</span></div><div class="casAuthors">Tian, Xiao-Peng; Wang, Chen-Yuan; Jin, Xiao-Han; Li, Mei; Wang, Feng-Wei; Huang, Wei-Juan; Yun, Jing-Ping; Xu, Rui-Hua; Cai, Qing-Qing; Xie, Dan</div><div class="citationInfo"><span class="NLM_cas:title">Theranostics</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1965-1979</span>CODEN:
                <span class="NLM_cas:coden">THERDS</span>;
        ISSN:<span class="NLM_cas:issn">1838-7640</span>.
    
            (<span class="NLM_cas:orgname">Ivyspring International Publisher</span>)
        </div><div class="casAbstract">The incidence of hepatocellular carcinoma is rising worldwide.  It is predicted that nearly half of the early-stage hepatocellular carcinoma (E-HCC) patients will develop recurrence.  Dysregulated pH, a hallmark of E-HCC, is correlated with poor prognosis.  The acidic microenvironment has been shown to promote the release of exosomes, the membrane vesicles recognized as intercellular communicators assocd. with tumor progression, recurrence, and metastasis.  To identify exosomes induced by acidic microenvironment that may regulate E-HCC progression and to explore their mechanisms and clin. significance in E-HCCs.  The miRNA microarray anal. and LASSO logistic statistic model were used to identify the main functional exosomal miRNAs.  Invasion and scratch assays were performed to examine the migration and invasion of HCC cells.  Immunoblotting and immunofluorescence were employed to detect the epithelial-to-mesenchymal transition (EMT) in HCC cells.  Chromatin immunopptn. (ChIP) was used to analyze the binding of HIF-1a and HIF-2a to promoter regions of miR-21 and miR-10b.  The acidic microenvironment in HCC was correlated with poor prognosis of patients.  Exosomes from HCC cells cultured in the acidic medium could promote cell proliferation, migration, and invasion of recipient HCC cells.  The miR-21 and miR-10b as the most important functional miRNAs in acidic HCC-derived exosomes.  Also, the acidic microenvironment triggered the activation of HIF-1α and HIF-2α and stimulated exosomal miR-21 and miR-10b expression substantially promoting HCC cell proliferation, migration, and invasion both in vivo and in vitro.  In E-HCC patients, serum exosomal miR-21 and miR-10b levels were assocd. with advanced tumor stage and HIF-1a and HIF-2a expression and were independent prognostic factors for disease-free survival of E-HCC patients.  Most importantly, we developed a nano-drug to target exosomal miR-21 and/or miR-10b and examd. its therapeutic effects against HCC in vivo.  The findings suggested that the exosomal miR-21 and miR-10b induced by acidic microenvironment in HCC promote cancer cell proliferation and metastasis and may serve as prognostic mol. markers and therapeutic targets for HCC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVgyh3Piatf7Vg90H21EOLACvtfcHk0lhip1nfcQ69fw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtlGru7bN&md5=3d0156b7944e2f5063f43ca531c7341b</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.7150%2Fthno.30958&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7150%252Fthno.30958%26sid%3Dliteratum%253Aachs%26aulast%3DTian%26aufirst%3DX.%2BP.%26aulast%3DWang%26aufirst%3DC.%2BY.%26aulast%3DJin%26aufirst%3DX.%2BH.%26aulast%3DLi%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DF.%2BW.%26aulast%3DHuang%26aufirst%3DW.%2BJ.%26aulast%3DYun%26aufirst%3DJ.%2BP.%26aulast%3DXu%26aufirst%3DR.%2BH.%26aulast%3DCai%26aufirst%3DQ.%2BQ.%26aulast%3DXie%26aufirst%3DD.%26atitle%3DAcidic%2520microenvironment%2520Up-regulates%2520exosomal%2520miR-21%2520and%2520miR-10b%2520in%2520early-stage%2520hepatocellular%2520carcinoma%2520to%2520promote%2520cancer%2520cell%2520proliferation%2520and%2520metastasis%26jtitle%3DTheranostics%26date%3D2019%26volume%3D9%26spage%3D1965%26epage%3D1979%26doi%3D10.7150%2Fthno.30958" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hasengbayi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, W.</span></span> <span> </span><span class="NLM_article-title">Mir-106b-5p Promotes Aggressive Progression of Hepatocellular Carcinoma Via Targeting Runx3</span>. <i>Cancer Med.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">6756</span>– <span class="NLM_lpage">6767</span>, <span class="refDoi"> DOI: 10.1002/cam4.2511</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1002%2Fcam4.2511" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=31503422" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitFGgsb%252FJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2019&pages=6756-6767&author=H.+Guauthor=S.+Guauthor=X.+Zhangauthor=S.+Zhangauthor=D.+Zhangauthor=J.+Linauthor=S.+Hasengbayiauthor=W.+Han&title=Mir-106b-5p+Promotes+Aggressive+Progression+of+Hepatocellular+Carcinoma+Via+Targeting+Runx3&doi=10.1002%2Fcam4.2511"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">MiR-106b-5p promotes aggressive progression of hepatocellular carcinoma via targeting RUNX3</span></div><div class="casAuthors">Gu, Hao; Gu, Shensen; Zhang, Xinlong; Zhang, Songjiang; Zhang, Dongming; Lin, Junsheng; Hasengbayi, Saiken; Han, Wei</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Medicine</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">6756-6767</span>CODEN:
                <span class="NLM_cas:coden">CMAEDL</span>;
        ISSN:<span class="NLM_cas:issn">2045-7634</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons Ltd.</span>)
        </div><div class="casAbstract">Background and Objectives : The roles of microRNA(miR)-106b-5p in hepatocellular carcinoma (HCC) remain unclear.  We aimed here to investigate the clin. significance of miR-106b-5p expression in HCC and its underlying mechanisms.  Methods : Expression levels of miR-106b-5p in 108 HCC clin. samples by quant. real-time reverse transcription PCR.  Assocns. of miR-106b-5p expression with various clinicopathol. features and patients' prognosis were evaluated by Chi-square test, Kaplan-Meier, and Cox proportional regression analyses, resp.  The target gene of miR-106b-5p and their functions in HCC cells were investigated by luciferase reporter, CCK-8, and Transwell Matrigel invasion assays.  Results : miR-106b-5p expression was markedly higher in HCC tissues than in noncancerous adjacent liver tissues (P < .001). miR-106b-5p upregulation was significantly assocd. with advanced TNM stage (P = .02), short recurrence-free (P = .005), and overall (P = .001) survivals.  Importantly, miR-106b-5p expression was an independent predictor of poor prognosis (P < .05).  RUNX3 was identified as a direct target gene of miR-106b-5p in HCC cells.  Functionally, miR-106b-5p upregulation promoted the viability and invasion of HCC cells, while enforced RUNX3 expression reversed the oncogenic effects of miR-106b-5p overexpression.  Conclusions : miR-106b-5p may serve as a potent prognostic marker for tumor recurrence and survival of HCC patients. miR-106b-5p may exert an oncogenic role in HCC via regulating its target gene RUNX3.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp8Ry_4fNQyu7Vg90H21EOLACvtfcHk0lht4g_ceS0j7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitFGgsb%252FJ&md5=89e217855a7f0d581ecb08a8f6e2765d</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1002%2Fcam4.2511&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcam4.2511%26sid%3Dliteratum%253Aachs%26aulast%3DGu%26aufirst%3DH.%26aulast%3DGu%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DD.%26aulast%3DLin%26aufirst%3DJ.%26aulast%3DHasengbayi%26aufirst%3DS.%26aulast%3DHan%26aufirst%3DW.%26atitle%3DMir-106b-5p%2520Promotes%2520Aggressive%2520Progression%2520of%2520Hepatocellular%2520Carcinoma%2520Via%2520Targeting%2520Runx3%26jtitle%3DCancer%2520Med.%26date%3D2019%26volume%3D8%26spage%3D6756%26epage%3D6767%26doi%3D10.1002%2Fcam4.2511" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Scisciani, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vossio, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guerrieri, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schinzari, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Iaco, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">D’Onorio
de Meo, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cervello, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montalto, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollicino, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raimondo, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levrero, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pediconi, N.</span></span> <span> </span><span class="NLM_article-title">Transcriptional regulation of miR-224 upregulated in human HCCs by NFkappaB inflammatory pathways</span>. <i>J. Hepatol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">855</span>– <span class="NLM_lpage">861</span>, <span class="refDoi"> DOI: 10.1016/j.jhep.2011.11.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1016%2Fj.jhep.2011.11.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=22178270" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BC38Xktlaqt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2012&pages=855-861&author=C.+Sciscianiauthor=S.+Vossioauthor=F.+Guerrieriauthor=V.+Schinzariauthor=R.+De+Iacoauthor=P.+D%E2%80%99Onorio%0Ade+Meoauthor=M.+Cervelloauthor=G.+Montaltoauthor=T.+Pollicinoauthor=G.+Raimondoauthor=M.+Levreroauthor=N.+Pediconi&title=Transcriptional+regulation+of+miR-224+upregulated+in+human+HCCs+by+NFkappaB+inflammatory+pathways&doi=10.1016%2Fj.jhep.2011.11.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Transcriptional regulation of miR-224 upregulated in human HCCs by NFκB inflammatory pathways</span></div><div class="casAuthors">Scisciani, Cecilia; Vossio, Stefania; Guerrieri, Francesca; Schinzari, Valeria; De Iaco, Rossana; D'Onorio de Meo, Paolo; Cervello, Melchiorre; Montalto, Giuseppe; Pollicino, Teresa; Raimondo, Giovanni; Levrero, Massimo; Pediconi, Natalia</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Hepatology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">855-861</span>CODEN:
                <span class="NLM_cas:coden">JOHEEC</span>;
        ISSN:<span class="NLM_cas:issn">0168-8278</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Background & Aims: miR-224 is up-regulated in human HCCs as compared to both paired peri-tumoral cirrhotic tissues and cirrhotic livers without HCC.  Here, we have cloned the miR-224 regulatory region and characterized its transcriptional regulation by the NFκB-dependent inflammatory pathways.  Methods: Mature miRNA expression was evaluated by a 2 step stem-loop real-time RT-PCR.  The recruitment of polymerase II and transcription factors on the pre-miR-224 promoter has been assessed by ChIPSeq and ChIP.  Results: We found miR-224 levels strongly up-regulated in both peri-tumoral cirrhotic livers and HCC samples as compared to normal livers.  In silico anal. of the putative miR-224 promoter revealed multiple NFκB sites.  We showed that LTα and TNFα activate transcription from the miR-224 promoter and of endogenous miR-224 expression in HCC cell lines, whereas the expression of miR-224 target API5 was reduced.  Exogenously expressed p65/RelA activates the miR-224 promoter and a dominant neg. form of IκBα (IκBSR) represses it.  ChIP anal. showed that p65/NFκB is recruited on the miR-224 promoter and that its binding sharply increases after exposure to LPS, TNFα, and LTα.  Altogether these findings link the inflammatory signals to NFκB-mediated activation of miR-224 expression.  An antago-miR specific for miR-224 blocked LPS and LTα stimulated HCC cells migration and invasion.  Conversely, the IKK inhibitor BMS-345541 blocks pre-miR-224-induced cellular migration and invasion.  Conclusions: Our results identify p65/NFκB as a direct transcriptional regulator of miR-224 expression and link miR-224 up-regulation with the activation of the LPS, LTα, and TNFα inflammatory pathways and cell migration/invasion in HCC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZufINPVyE0bVg90H21EOLACvtfcHk0lht4g_ceS0j7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xktlaqt70%253D&md5=0e9842de1df58d1a7b0a9671ee1a7b8b</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2Fj.jhep.2011.11.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jhep.2011.11.017%26sid%3Dliteratum%253Aachs%26aulast%3DScisciani%26aufirst%3DC.%26aulast%3DVossio%26aufirst%3DS.%26aulast%3DGuerrieri%26aufirst%3DF.%26aulast%3DSchinzari%26aufirst%3DV.%26aulast%3DDe%2BIaco%26aufirst%3DR.%26aulast%3DD%25E2%2580%2599Onorio%2Bde%2BMeo%26aufirst%3DP.%26aulast%3DCervello%26aufirst%3DM.%26aulast%3DMontalto%26aufirst%3DG.%26aulast%3DPollicino%26aufirst%3DT.%26aulast%3DRaimondo%26aufirst%3DG.%26aulast%3DLevrero%26aufirst%3DM.%26aulast%3DPediconi%26aufirst%3DN.%26atitle%3DTranscriptional%2520regulation%2520of%2520miR-224%2520upregulated%2520in%2520human%2520HCCs%2520by%2520NFkappaB%2520inflammatory%2520pathways%26jtitle%3DJ.%2520Hepatol.%26date%3D2012%26volume%3D56%26spage%3D855%26epage%3D861%26doi%3D10.1016%2Fj.jhep.2011.11.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fu, X. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, Y. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, Z. B.</span></span> <span> </span><span class="NLM_article-title">MicroRNA-30a suppresses autophagy-mediated anoikis resistance and metastasis in hepatocellular carcinoma</span>. <i>Cancer Lett.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>412</i></span>,  <span class="NLM_fpage">108</span>– <span class="NLM_lpage">117</span>, <span class="refDoi"> DOI: 10.1016/j.canlet.2017.10.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1016%2Fj.canlet.2017.10.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=29061507" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhslagtbfE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=412&publication_year=2018&pages=108-117&author=X.+T.+Fuauthor=Y.+H.+Shiauthor=J.+Zhouauthor=Y.+F.+Pengauthor=W.+R.+Liuauthor=G.+M.+Shiauthor=Q.+Gaoauthor=X.+Y.+Wangauthor=K.+Songauthor=J.+Fanauthor=Z.+B.+Ding&title=MicroRNA-30a+suppresses+autophagy-mediated+anoikis+resistance+and+metastasis+in+hepatocellular+carcinoma&doi=10.1016%2Fj.canlet.2017.10.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">MicroRNA-30a suppresses autophagy-mediated anoikis resistance and metastasis in hepatocellular carcinoma</span></div><div class="casAuthors">Fu, Xiu-Tao; Shi, Ying-Hong; Zhou, Jian; Peng, Yuan-Fei; Liu, Wei-Ren; Shi, Guo-Ming; Gao, Qiang; Wang, Xiao-Ying; Song, Kang; Fan, Jia; Ding, Zhen-Bin</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Letters (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">412</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">108-117</span>CODEN:
                <span class="NLM_cas:coden">CALEDQ</span>;
        ISSN:<span class="NLM_cas:issn">0304-3835</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">MiRNA-30a (miR-30a) was previously reported as one of metastatic hepatocellular carcinoma (HCC)-related microRNAs.  However, the function of miR-30a on enhancing our biol. understanding of HCC metastasis is not clear.  This study demonstrated that miR-30a was significantly down-regulated in HCC tissues and cell lines, and was assocd. with vascular invasion, metastasis potential and recurrent disease in HCC.  Functional studies confirmed that miR-30a could inhibit the metastasis of HCC in a well-established nude mouse model of lung metastasis.  Moreover, miR-30a was proved to prevent anoikis inhibition of HCC cells in vivo and in vitro.  Mech., autophagy related protein Beclin 1 and Atg5 were direct downstream targets of miR-30a, and mediated autophagy activity influence of miR-30a in HCC.  Taken together, downregulated miR-30a in metastatic HCC mediates Beclin 1 and Atg5-dependent autophagy, which confers anoikis resistance in HCC cells.  The mol. basis of autophagy action during this process partly contributes to the HCC metastasis, suggesting that targeting autophagy via miR-30a may have therapeutic implications for the prevention of HCC recurrence/metastasis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBP1M-OheQVrVg90H21EOLACvtfcHk0lhhAdfesD9jEg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhslagtbfE&md5=633717b5e6c40b158dbb6ef1f7db7307</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2Fj.canlet.2017.10.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.canlet.2017.10.012%26sid%3Dliteratum%253Aachs%26aulast%3DFu%26aufirst%3DX.%2BT.%26aulast%3DShi%26aufirst%3DY.%2BH.%26aulast%3DZhou%26aufirst%3DJ.%26aulast%3DPeng%26aufirst%3DY.%2BF.%26aulast%3DLiu%26aufirst%3DW.%2BR.%26aulast%3DShi%26aufirst%3DG.%2BM.%26aulast%3DGao%26aufirst%3DQ.%26aulast%3DWang%26aufirst%3DX.%2BY.%26aulast%3DSong%26aufirst%3DK.%26aulast%3DFan%26aufirst%3DJ.%26aulast%3DDing%26aufirst%3DZ.%2BB.%26atitle%3DMicroRNA-30a%2520suppresses%2520autophagy-mediated%2520anoikis%2520resistance%2520and%2520metastasis%2520in%2520hepatocellular%2520carcinoma%26jtitle%3DCancer%2520Lett.%26date%3D2018%26volume%3D412%26spage%3D108%26epage%3D117%26doi%3D10.1016%2Fj.canlet.2017.10.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ding, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duan, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xia, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, G.</span></span> <span> </span><span class="NLM_article-title">SSRP1 contributes to the malignancy of hepatocellular carcinoma and is negatively regulated by miR-497</span>. <i>Mol. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">903</span>– <span class="NLM_lpage">914</span>, <span class="refDoi"> DOI: 10.1038/mt.2016.9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1038%2Fmt.2016.9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=26755331" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BC28Xit1altrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2016&pages=903-914&author=Q.+Dingauthor=K.+Heauthor=T.+Luoauthor=Y.+Dengauthor=H.+Wangauthor=H.+Liuauthor=J.+Zhangauthor=K.+Chenauthor=J.+Xiaoauthor=X.+Duanauthor=R.+Huangauthor=Z.+Xiaauthor=W.+Zhouauthor=J.+Heauthor=H.+Yuauthor=X.+Jiaoauthor=G.+Xiang&title=SSRP1+contributes+to+the+malignancy+of+hepatocellular+carcinoma+and+is+negatively+regulated+by+miR-497&doi=10.1038%2Fmt.2016.9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">SSRP1 Contributes to the Malignancy of Hepatocellular Carcinoma and Is Negatively Regulated by miR-497</span></div><div class="casAuthors">Ding, Qianshan; He, Ke; Luo, Tao; Deng, Yunchao; Wang, Hanning; Liu, Hao; Zhang, Jinqian; Chen, Kaiyun; Xiao, Jinfeng; Duan, Xiaopeng; Huang, Rui; Xia, Zhenglin; Zhou, Wenjie; He, Jinliang; Yu, Honggang; Jiao, Xingyuan; Xiang, Guoan</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Therapy</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">903-914</span>CODEN:
                <span class="NLM_cas:coden">MTOHCK</span>;
        ISSN:<span class="NLM_cas:issn">1525-0016</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The aim of this study is to clarify the clin. implication and functional role of structure specific recognition protein 1 (SSRP1) in hepatocellular carcinoma (HCC) and explore the underlying mechanism of aberrant high expression of SSRP1 in cancers.  In the present investigation, we validated that SSRP1 was upregulated in HCC samples.  We also demonstrated that its upregulation was assocd. with several clinicopathol. features such as higher serum AFP level, larger tumor size, and higher T stage of HCC patients; and its high expression indicated shorter overall survival and faster recurrence.  To investigate the role of SSRP1 in HCC progression, both loss- and gain-function models were established.  We demonstrated that SSPR1 modulated both proliferation and metastasis of HCC cells in vitro and vivo.  Furthermore, we demonstrated that SSRP1-modulated apoptosis process and its knockdown increased the sensitivity of HCC cells to doxorubicin, 5-Fluorouracil, and cisplatin.  We also identified microRNA-497 (miR-497) as a posttranscriptional regulator of SSRP1.  Ectopic expression of miR-497 inhibited 3'-untranslated-region-coupled luciferase activity and suppressed endogenous SSRP1 expression at both mRNA and protein levels.  For the first time, we proved that SSRP1 upregulation contributed to HCC development and the tumor-suppressive miR-497 served as its neg. regulator.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbZ9DcAIqU0LVg90H21EOLACvtfcHk0lhhAdfesD9jEg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xit1altrg%253D&md5=e73f7b7ab114825692a5fdd44bd46855</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1038%2Fmt.2016.9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fmt.2016.9%26sid%3Dliteratum%253Aachs%26aulast%3DDing%26aufirst%3DQ.%26aulast%3DHe%26aufirst%3DK.%26aulast%3DLuo%26aufirst%3DT.%26aulast%3DDeng%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DK.%26aulast%3DXiao%26aufirst%3DJ.%26aulast%3DDuan%26aufirst%3DX.%26aulast%3DHuang%26aufirst%3DR.%26aulast%3DXia%26aufirst%3DZ.%26aulast%3DZhou%26aufirst%3DW.%26aulast%3DHe%26aufirst%3DJ.%26aulast%3DYu%26aufirst%3DH.%26aulast%3DJiao%26aufirst%3DX.%26aulast%3DXiang%26aufirst%3DG.%26atitle%3DSSRP1%2520contributes%2520to%2520the%2520malignancy%2520of%2520hepatocellular%2520carcinoma%2520and%2520is%2520negatively%2520regulated%2520by%2520miR-497%26jtitle%3DMol.%2520Ther.%26date%3D2016%26volume%3D24%26spage%3D903%26epage%3D914%26doi%3D10.1038%2Fmt.2016.9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, X. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z. N.</span></span> <span> </span><span class="NLM_article-title">microRNA-146a inhibits cancer metastasis by downregulating VEGF through dual pathways in hepatocellular carcinoma</span>. <i>Mol. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">5</span>– <span class="NLM_lpage">19</span>, <span class="refDoi"> DOI: 10.1186/1476-4598-14-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1186%2F1476-4598-14-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=25608619" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BC2MXkt1ejtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2015&pages=5-19&author=Z.+Zhangauthor=Y.+Zhangauthor=X.+X.+Sunauthor=X.+Maauthor=Z.+N.+Chen&title=microRNA-146a+inhibits+cancer+metastasis+by+downregulating+VEGF+through+dual+pathways+in+hepatocellular+carcinoma&doi=10.1186%2F1476-4598-14-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">microRNA-146a inhibits cancer metastasis by downregulating VEGF through dual pathways in hepatocellular carcinoma</span></div><div class="casAuthors">Zhang, Zheng; Zhang, Yang; Sun, Xiu-Xuan; Ma, Xi; Chen, Zhi-Nan</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">5</span>CODEN:
                <span class="NLM_cas:coden">MCOACG</span>;
        ISSN:<span class="NLM_cas:issn">1476-4598</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">Growing evidence indicates that miR-146a is involved in carcinogenesis and tumor progression in several human malignancies.  However, the mol. details underlying miR-146a mediated regulation of its target genes and its precise biol. function in cancer, esp. in hepatocellular carcinoma (HCC) remains unclear.  Methods: The expression levels of genes including miR-146a, APC, VEGF and HAb18G were examd. in HCC cell lines and patient specimens were compared with control levels using quant. reverse transcription-PCR.  The functions of miR-146a and HAb18G in migration/invasion and liver metastasis formation were detd. by transwell and spleen injection assays, resp. miR-146a related genes were detd. by PCR array.  The potential regulatory targets of miR-146a were detd. by bioinformatics and prediction tools, correlation with target protein expression, and luciferase reporter assay.  DNA methylation status of miR-146a promoter were performed by PCR anal. of bisulfite-modified genomic DNA.  Results: We demonstrated that miR-146a expression was markedly downregulated in hepatoma cells and hepatoma tissues compared to immortalized normal liver epithelial cells and normal hepatic tissues.  DNA methylation of miR-146a promoter correlated with its downexpression and with liver cancer metastasis.  The restoration of miR-146a dramatically suppressed HCC cell invasion and metastasis by repressing VEGF expression through upregulating APC, which inhibits β-catenin accumulation in nucleus, and downregulating NF-κB p65 by targeting HAb18G.  In human HCC, miR-146a expression was neg. correlated with increased HAb18G, VEGF, NF-κB p65 and beneficial prognosis.  Conclusion: This study identified a novel target of miR-146a and defined miR-146a as a crucial tumor suppressor in human HCC that acts through multiple pathways and mechanisms to suppress HCC invasion or metastasis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGopbG7clEzdIbVg90H21EOLACvtfcHk0ljo6UfkgZwOJQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXkt1ejtLs%253D&md5=1042a1f24b5cb5af954b8658d66f1e0c</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1186%2F1476-4598-14-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1476-4598-14-5%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DSun%26aufirst%3DX.%2BX.%26aulast%3DMa%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DZ.%2BN.%26atitle%3DmicroRNA-146a%2520inhibits%2520cancer%2520metastasis%2520by%2520downregulating%2520VEGF%2520through%2520dual%2520pathways%2520in%2520hepatocellular%2520carcinoma%26jtitle%3DMol.%2520Cancer%26date%3D2015%26volume%3D14%26spage%3D5%26epage%3D19%26doi%3D10.1186%2F1476-4598-14-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Z. M.</span></span> <span> </span><span class="NLM_article-title">Suppression of KIF14 expression inhibits hepatocellular carcinoma progression and predicts favorable outcome</span>. <i>Cancer Sci.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>104</i></span>,  <span class="NLM_fpage">552</span>– <span class="NLM_lpage">557</span>, <span class="refDoi"> DOI: 10.1111/cas.12128</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1111%2Fcas.12128" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=23414349" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BC3sXlvVCjsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=104&publication_year=2013&pages=552-557&author=T.+Yangauthor=X.+B.+Zhangauthor=Z.+M.+Zheng&title=Suppression+of+KIF14+expression+inhibits+hepatocellular+carcinoma+progression+and+predicts+favorable+outcome&doi=10.1111%2Fcas.12128"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Suppression of KIF14 expression inhibits hepatocellular carcinoma progression and predicts favorable outcome</span></div><div class="casAuthors">Yang, Tao; Zhang, Xian-Bo; Zheng, Zhi-Min</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Science</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">104</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">552-557</span>CODEN:
                <span class="NLM_cas:coden">CSACCM</span>;
        ISSN:<span class="NLM_cas:issn">1349-7006</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">The mitotic kinesin superfamily protein KIF14 is essential for cytokinesis and chromosome segregation and increased KIF14 expression is related to a variety of human cancers.  In this study, we investigate KIF14 expression in assocn. with clin. variables and the role of KIF14 during tumorigenesis.  We found that KIF14 is overexpressed in most primary hepatocellular carcinoma (HCC) tissues compared with the adjacent normal liver tissues and KIF14 overexpression is assocd. with tumor grade (P = 0.002), stage (P = 0.013) and poor survival (P < 0.001).  Downregulation of KIF14 decreased the capacity of proliferation both in vitro and in vivo.  Furthermore, suppression of KIF14 not only decreases cancer cell migration but also induces apoptosis of cells with inactivation of the phosphatidylinositol 3-kinase-Akt signaling pathway.  Therefore, our current study indicates that KIF14 promotes HCC carcinogenesis and may serve as a potential therapeutic target for human HCC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXbA_SnAQJHrVg90H21EOLACvtfcHk0ljo6UfkgZwOJQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXlvVCjsr8%253D&md5=0aab5cf484a0ee193fe972885808b10d</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1111%2Fcas.12128&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fcas.12128%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DT.%26aulast%3DZhang%26aufirst%3DX.%2BB.%26aulast%3DZheng%26aufirst%3DZ.%2BM.%26atitle%3DSuppression%2520of%2520KIF14%2520expression%2520inhibits%2520hepatocellular%2520carcinoma%2520progression%2520and%2520predicts%2520favorable%2520outcome%26jtitle%3DCancer%2520Sci.%26date%3D2013%26volume%3D104%26spage%3D552%26epage%3D557%26doi%3D10.1111%2Fcas.12128" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span> <span> </span><span class="NLM_article-title">CDK5-mediated phosphorylation and stabilization of TPX2 promotes hepatocellular tumorigenesis</span>. <i>J. Exp. Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">286</span>– <span class="NLM_lpage">300</span>, <span class="refDoi"> DOI: 10.1186/s13046-019-1297-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1186%2Fs13046-019-1297-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=31272499" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A280%3ADC%252BB3MzktVGqtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2019&pages=286-300&author=F.+Wangauthor=W.+Zhaoauthor=Y.+Gaoauthor=J.+Zhouauthor=H.+Liauthor=G.+Zhangauthor=D.+Guoauthor=C.+Xieauthor=J.+Liauthor=Z.+Yinauthor=J.+Zhang&title=CDK5-mediated+phosphorylation+and+stabilization+of+TPX2+promotes+hepatocellular+tumorigenesis&doi=10.1186%2Fs13046-019-1297-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">CDK5-mediated phosphorylation and stabilization of TPX2 promotes hepatocellular tumorigenesis</span></div><div class="casAuthors">Wang Fuqiang; Xie Chengrong; Li Jie; Yin Zhenyu; Zhao Wenxing; Gao Yuehong; Li Huifang; Zhang Guanyun; Guo Dong; Zhang Jie; Zhou Jiechao</div><div class="citationInfo"><span class="NLM_cas:title">Journal of experimental & clinical cancer research : CR</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">286</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  CDK5, an atypical member of the CDK family, play a significant role in the tumorigenesis of multiple organ, but CDK5 and its substrates in genesis and development of HCC is still unclear.  METHODS:  Expression of CDK5 in HCC tumor and paired adjacent noncancerous tissues from 90 patients were measured by Western blotting, immunohistochemistry, and real-time PCR.  The role of CDK5 in cell function and tumorigenesis was explored in HCC cell lines, ex vivo xenografts and diethylnitrosamine induced HCC model.  Furthermore, comparative phosphoproteomic screening identified the oncoprotein TPX2 as a new substrate of CDK5.  We also identified the effect of CDK5/P25 interaction blocker tamoxifen on HCC cell growth and migration.  RESULTS:  CDK5 was increased in HCC tisues and the level of CDK5 was correlated with the severity of HCC based on patient recurrence and 5-year fatality rate.  Exogenously expressed CDK5 but not kinase-dead CDK5 promoted proliferation, migration, and invasion of HCC cells.  Functional ablation of CDK5 significantly inhibited the exacerbation of HCC cells.  Xenograft implantation of HCC cells overexpressing CDK5 promoted tumorigenesis, and genetic knockdown of CDK5 reduced HCC growth and metastasis in vivo.  More importantly, heterozygous knockout CDK5 (Cdk5+/-) attenuated HCC tumorigenesis induced by diethylnitrosamine.  CDK5-mediated phosphorylation of TPX2 at serine 486 promoted its protein stability.  TPX2 silence could restore HCC cell migration capability with overexpression CDK5.  Treatment with tamoxifen inhibited cell growth and migration of HCC, demonstrating the role of active CDK5 in HCC.  CONCLUSIONS:  Our results suggest activation of CDK5 is associated with HCC tumorigenesis.  CDK5-mediated phosphorylation and stabilization of TPX2 promotes hepatocellular proliferation and tumorigenicity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSH7rKwqk-g40bxh97K6Vt6fW6udTcc2eZtggQdBf_Ut7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3MzktVGqtw%253D%253D&md5=63eb69a3b25860dc1d59fb962e9b74b8</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1186%2Fs13046-019-1297-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs13046-019-1297-6%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DF.%26aulast%3DZhao%26aufirst%3DW.%26aulast%3DGao%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DGuo%26aufirst%3DD.%26aulast%3DXie%26aufirst%3DC.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DYin%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DJ.%26atitle%3DCDK5-mediated%2520phosphorylation%2520and%2520stabilization%2520of%2520TPX2%2520promotes%2520hepatocellular%2520tumorigenesis%26jtitle%3DJ.%2520Exp.%2520Clin.%2520Cancer%2520Res.%26date%3D2019%26volume%3D38%26spage%3D286%26epage%3D300%26doi%3D10.1186%2Fs13046-019-1297-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, R. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yong, Y. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, C. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bian, H.</span></span> <span> </span><span class="NLM_article-title">UBE2S interacting with TRIM28 in the nucleus accelerates cell cycle by ubiquitination of p27 to promote hepatocellular carcinoma development</span>. <i>Signal Transduction Targeted Ther.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">64</span>– <span class="NLM_lpage">75</span>, <span class="refDoi"> DOI: 10.1038/s41392-020-00432-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1038%2Fs41392-020-00432-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=33589597" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BB3MXhtVOmsbzL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2021&pages=64-75&author=R.+Y.+Zhangauthor=Z.+K.+Liuauthor=D.+Weiauthor=Y.+L.+Yongauthor=P.+Linauthor=H.+Liauthor=M.+Liuauthor=N.+S.+Zhengauthor=K.+Liuauthor=C.+X.+Huauthor=X.+Z.+Yangauthor=Z.+N.+Chenauthor=H.+Bian&title=UBE2S+interacting+with+TRIM28+in+the+nucleus+accelerates+cell+cycle+by+ubiquitination+of+p27+to+promote+hepatocellular+carcinoma+development&doi=10.1038%2Fs41392-020-00432-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">UBE2S interacting with TRIM28 in the nucleus accelerates cell cycle by ubiquitination of p27 to promote hepatocellular carcinoma development</span></div><div class="casAuthors">Zhang, Ren-Yu; Liu, Ze-Kun; Wei, Ding; Yong, Yu-Le; Lin, Peng; Li, Hao; Liu, Man; Zheng, Nai-Shan; Liu, Ke; Hu, Cai-Xia; Yang, Xiao-Zhen; Chen, Zhi-Nan; Bian, Huijie</div><div class="citationInfo"><span class="NLM_cas:title">Signal Transduction and Targeted Therapy</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">64</span>CODEN:
                <span class="NLM_cas:coden">STTTCB</span>;
        ISSN:<span class="NLM_cas:issn">2059-3635</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Abstr.: Genomic sequencing anal. of tumors provides potential mol. therapeutic targets for precision medicine.  However, identifying a key driver gene or mutation that can be used for hepatocellular carcinoma (HCC) treatment remains difficult.  Here, we performed whole-exome sequencing on genomic DNA obtained from six pairs of HCC and adjacent tissues and identified two novel somatic mutations of UBE2S (p. Gly57Ala and p. Lys63Asn).  Predictions of the functional effects of the mutations showed that two amino-acid substitutions were potentially deleterious.  Further, we obsd. that wild-type UBE2S, esp. in the nucleus, was significantly higher in HCC tissues than that in adjacent tissues and closely related to the clinicopathol. features of patients with HCC.  Functional assays revealed that overexpression of UBE2S promoted the proliferation, invasion, metastasis, and G1/S phase transition of HCC cells in vitro, and promoted the tumor growth significantly in vivo.  Mechanistically, UBE2S interacted with TRIM28 in the nucleus, both together enhanced the ubiquitination of p27 to facilitate its degrdn. and cell cycle progression.  Most importantly, the small-mol. cephalomannine was found by a luciferase-based sensitive high-throughput screen (HTS) to inhibit UBE2S expression and significantly attenuate HCC progression in vitro and in vivo, which may represent a promising strategy for HCC therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr6vLVvO6SN5rVg90H21EOLACvtfcHk0lhtVgi2Vzrw9A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXhtVOmsbzL&md5=0f2c9d1ab510ea6c8c6e450968b8071f</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1038%2Fs41392-020-00432-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41392-020-00432-z%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DR.%2BY.%26aulast%3DLiu%26aufirst%3DZ.%2BK.%26aulast%3DWei%26aufirst%3DD.%26aulast%3DYong%26aufirst%3DY.%2BL.%26aulast%3DLin%26aufirst%3DP.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DM.%26aulast%3DZheng%26aufirst%3DN.%2BS.%26aulast%3DLiu%26aufirst%3DK.%26aulast%3DHu%26aufirst%3DC.%2BX.%26aulast%3DYang%26aufirst%3DX.%2BZ.%26aulast%3DChen%26aufirst%3DZ.%2BN.%26aulast%3DBian%26aufirst%3DH.%26atitle%3DUBE2S%2520interacting%2520with%2520TRIM28%2520in%2520the%2520nucleus%2520accelerates%2520cell%2520cycle%2520by%2520ubiquitination%2520of%2520p27%2520to%2520promote%2520hepatocellular%2520carcinoma%2520development%26jtitle%3DSignal%2520Transduction%2520Targeted%2520Ther.%26date%3D2021%26volume%3D6%26spage%3D64%26epage%3D75%26doi%3D10.1038%2Fs41392-020-00432-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, W. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jia, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Q. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, L.</span></span> <span> </span><span class="NLM_article-title">APC(CDC20)-mediated degradation of PHD3 stabilizes HIF-1a and promotes tumorigenesis in hepatocellular carcinoma</span>. <i>Cancer Lett.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>496</i></span>,  <span class="NLM_fpage">144</span>– <span class="NLM_lpage">155</span>, <span class="refDoi"> DOI: 10.1016/j.canlet.2020.10.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1016%2Fj.canlet.2020.10.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=33039559" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BB3cXitFOisbfI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=496&publication_year=2021&pages=144-155&author=M.+Shiauthor=W.+Q.+Daiauthor=R.+R.+Jiaauthor=Q.+H.+Zhangauthor=J.+Weiauthor=Y.+G.+Wangauthor=S.+H.+Xiangauthor=B.+Liuauthor=L.+Xu&title=APC%28CDC20%29-mediated+degradation+of+PHD3+stabilizes+HIF-1a+and+promotes+tumorigenesis+in+hepatocellular+carcinoma&doi=10.1016%2Fj.canlet.2020.10.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">APCCDC20-mediated degradation of PHD3 stabilizes HIF-1a and promotes tumorigenesis in hepatocellular carcinoma</span></div><div class="casAuthors">Shi, Min; Dai, Wei-Qi; Jia, Rong-Rong; Zhang, Qing-Hui; Wei, Jue; Wang, Yu-Gang; Xiang, Shi-Hao; Liu, Bin; Xu, Ling</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Letters (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">496</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">144-155</span>CODEN:
                <span class="NLM_cas:coden">CALEDQ</span>;
        ISSN:<span class="NLM_cas:issn">0304-3835</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">CDC20 regulates cell cycle progression by targeting key substrates for destruction, but its role in hepatocellular carcinoma (HCC) tumorigenesis remains to be explored.  Here, by using weighted gene co-expression network anal. (WGCNA), we identified CDC20 as a hub gene in HCC.  We demonstrated that CDC20 expression is correlated with HIF-1 activity and overall survival (OS) of clinic HCC patients.  The activity of HIF-1 is regulated by the stability of HIF-1a subunit, which is hydroxylated by oxygen-dependent prolyl hydroxylase enzymes, the PHDs.  In addn., we show that genetic ablation or pharmacol. inhibition of CDC20 can accelerate the degrdn. of HIF-1a and impair VEGF secretion in HCC cells.  Mechanistically, we found that CDC20 binds to the destruction-box (D-box) motif present in the PHD3 protein to promote its polyubiquitination and degrdn.  The depletion of endogenous PHD3 in CDC20 knockdown HCC cells greatly attenuated the decline of HIF-1a protein and restored the secretion of VEGF.  In contrast, overexpression of a non-degradable PHD3 mutant significantly inhibited the proliferation of HCC cells both in vitro and in vivo.  Collectively, our findings indicate that CDC20 plays a crucial role in the development of HCC by governing PHD3 protein.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqLCvffuGn_jLVg90H21EOLACvtfcHk0lhEFGV9m9BQmQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXitFOisbfI&md5=4d27f17964d10161717851256dd29714</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1016%2Fj.canlet.2020.10.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.canlet.2020.10.011%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DM.%26aulast%3DDai%26aufirst%3DW.%2BQ.%26aulast%3DJia%26aufirst%3DR.%2BR.%26aulast%3DZhang%26aufirst%3DQ.%2BH.%26aulast%3DWei%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DY.%2BG.%26aulast%3DXiang%26aufirst%3DS.%2BH.%26aulast%3DLiu%26aufirst%3DB.%26aulast%3DXu%26aufirst%3DL.%26atitle%3DAPC%2528CDC20%2529-mediated%2520degradation%2520of%2520PHD3%2520stabilizes%2520HIF-1a%2520and%2520promotes%2520tumorigenesis%2520in%2520hepatocellular%2520carcinoma%26jtitle%3DCancer%2520Lett.%26date%3D2021%26volume%3D496%26spage%3D144%26epage%3D155%26doi%3D10.1016%2Fj.canlet.2020.10.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J. P.</span>; <span class="NLM_string-name">Sun, H. D.</span>; <span class="NLM_string-name">Zhang, H. B.</span>; <span class="NLM_string-name">Zheng, Y. T.</span>; <span class="NLM_string-name">Xiao, W. L.</span>; <span class="NLM_string-name">Pu, J. X.</span>; <span class="NLM_string-name">Wang, R. R.</span>; <span class="NLM_string-name">Yang, L. M.</span></span> <span> </span><span class="NLM_article-title">Nitrogen-containing biphenyl compounds as anti-Aids agents and their preparation, pharmaceutical compositions and use in the treatment of HIV-1 infection</span>. U.S. Patent US <span class="NLM_patent">20130331390</span>, <span class="NLM_year">2013</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&author=J.+P.+Liu&author=H.+D.+Sun&author=H.+B.+Zhang&author=Y.+T.+Zheng&author=W.+L.+Xiao&author=J.+X.+Pu&author=R.+R.+Wang&author=L.+M.+Yang&title=Nitrogen-containing+biphenyl+compounds+as+anti-Aids+agents+and+their+preparation%2C+pharmaceutical+compositions+and+use+in+the+treatment+of+HIV-1+infection"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DJ.%2BP.%26atitle%3DNitrogen-containing%2520biphenyl%2520compounds%2520as%2520anti-Aids%2520agents%2520and%2520their%2520preparation%252C%2520pharmaceutical%2520compositions%2520and%2520use%2520in%2520the%2520treatment%2520of%2520HIV-1%2520infection%26date%3D2013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abell, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mercado, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caneque, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguez, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xhemalce, B.</span></span> <span> </span><span class="NLM_article-title">Click quantitative mass spectrometry identifies PIWIL3 as a mechanistic target of RNA interference activator enoxacin in cancer cells</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>139</i></span>,  <span class="NLM_fpage">1400</span>– <span class="NLM_lpage">1403</span>, <span class="refDoi"> DOI: 10.1021/jacs.6b11751</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.6b11751" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVKhsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=139&publication_year=2017&pages=1400-1403&author=N.+S.+Abellauthor=M.+Mercadoauthor=T.+Canequeauthor=R.+Rodriguezauthor=B.+Xhemalce&title=Click+quantitative+mass+spectrometry+identifies+PIWIL3+as+a+mechanistic+target+of+RNA+interference+activator+enoxacin+in+cancer+cells&doi=10.1021%2Fjacs.6b11751"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Click Quantitative Mass Spectrometry Identifies PIWIL3 as a Mechanistic Target of RNA Interference Activator Enoxacin in Cancer Cells</span></div><div class="casAuthors">Abell, Nathan S.; Mercado, Marvin; Caneque, Tatiana; Rodriguez, Raphael; Xhemalce, Blerta</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">139</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1400-1403</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Enoxacin is a small mol. that stimulates RNA interference (RNAi) and acts as a growth inhibitor selectively in cancer but not in untransformed cells.  Here, we used alkenox, a clickable enoxacin surrogate, coupled with quant. mass spectrometry, to identify PIWIL3 as a mechanistic target of enoxacin.  PIWIL3 is an Argonaute protein of the PIWI subfamily that is mainly expressed in the germline and that mediates RNAi through piRNAs.  Our results suggest that cancer cells re-express PIWIL3 to repress RNAi through miRNAs and thus open a new opportunity for cancer-specific targeting.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJ5Cpc09nUzLVg90H21EOLACvtfcHk0lhEFGV9m9BQmQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVKhsrY%253D&md5=7dc812390e460f651676384bee9e8fbe</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Fjacs.6b11751&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.6b11751%26sid%3Dliteratum%253Aachs%26aulast%3DAbell%26aufirst%3DN.%2BS.%26aulast%3DMercado%26aufirst%3DM.%26aulast%3DCaneque%26aufirst%3DT.%26aulast%3DRodriguez%26aufirst%3DR.%26aulast%3DXhemalce%26aufirst%3DB.%26atitle%3DClick%2520quantitative%2520mass%2520spectrometry%2520identifies%2520PIWIL3%2520as%2520a%2520mechanistic%2520target%2520of%2520RNA%2520interference%2520activator%2520enoxacin%2520in%2520cancer%2520cells%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2017%26volume%3D139%26spage%3D1400%26epage%3D1403%26doi%3D10.1021%2Fjacs.6b11751" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, Z. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y. Z.</span></span> <span> </span><span class="NLM_article-title">TTD: Therapeutic Target Database</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">412</span>– <span class="NLM_lpage">415</span>, <span class="refDoi"> DOI: 10.1093/nar/30.1.412</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1093%2Fnar%2F30.1.412" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=11752352" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BD38Xht12rtrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2002&pages=412-415&author=X.+Chenauthor=Z.+L.+Jiauthor=Y.+Z.+Chen&title=TTD%3A+Therapeutic+Target+Database&doi=10.1093%2Fnar%2F30.1.412"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">TTD: Therapeutic Target Database</span></div><div class="casAuthors">Chen, X.; Ji, Z. L.; Chen, Y. Z.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">412-415</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">A no. of proteins and nucleic acids have been explored as therapeutic targets.  These targets are subjects of interest in different areas of biomedical and pharmaceutical research and in the development and evaluation of bioinformatics, mol. modeling, computer-aided drug design and anal. tools.  A publicly accessible database that provides comprehensive information about these targets is therefore helpful to the relevant communities.  The Therapeutic Target Database (TTD) is designed to provide information about the known therapeutic protein and nucleic acid targets described in the literature, the targeted disease conditions, the pathway information and the corresponding drugs/ligands directed at each of these targets.  Cross-links to other databases are also introduced to facilitate the access of information about the sequence, 3D structure, function, nomenclature, drug/ligand binding properties, drug usage and effects, and related literature for each target.  This database can be accessed at http://xin.cz3.nus.edu.sg/group/ttd/ttd.asp and it currently contains entries for 433 targets covering 125 disease conditions along with 809 drugs/ligands directed at each of these targets.  Each entry can be retrieved through multiple methods including target name, disease name, drug/ligand name, drug/ligand function and drug therapeutic classification.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr2ZySWqC-s2bVg90H21EOLACvtfcHk0ljUb4Ai0beMZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xht12rtrw%253D&md5=549be6ff77be558f04ef2d3a2a79642e</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1093%2Fnar%2F30.1.412&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252F30.1.412%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DX.%26aulast%3DJi%26aufirst%3DZ.%2BL.%26aulast%3DChen%26aufirst%3DY.%2BZ.%26atitle%3DTTD%253A%2520Therapeutic%2520Target%2520Database%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2002%26volume%3D30%26spage%3D412%26epage%3D415%26doi%3D10.1093%2Fnar%2F30.1.412" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ota, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakurai, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gupta, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valente, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wulff, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ariyoshi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iizasa, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davuluri, R. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nishikura, K.</span></span> <span> </span><span class="NLM_article-title">ADAR1 forms a complex with Dicer to promote microRNA processing and RNA-induced gene silencing</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>153</i></span>,  <span class="NLM_fpage">575</span>– <span class="NLM_lpage">589</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2013.03.024</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1016%2Fj.cell.2013.03.024" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=23622242" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BC3sXmslegtrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=153&publication_year=2013&pages=575-589&author=H.+Otaauthor=M.+Sakuraiauthor=R.+Guptaauthor=L.+Valenteauthor=B.+E.+Wulffauthor=K.+Ariyoshiauthor=H.+Iizasaauthor=R.+V.+Davuluriauthor=K.+Nishikura&title=ADAR1+forms+a+complex+with+Dicer+to+promote+microRNA+processing+and+RNA-induced+gene+silencing&doi=10.1016%2Fj.cell.2013.03.024"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">ADAR1 forms a complex with Dicer to promote microRNA processing and RNA-induced gene silencing</span></div><div class="casAuthors">Ota, Hiromitsu; Sakurai, Masayuki; Gupta, Ravi; Valente, Louis; Wulff, Bjorn-Erik; Ariyoshi, Kentaro; Iizasa, Hisashi; Davuluri, Ramana V.; Nishikura, Kazuko</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">153</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">575-589</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Adenosine deaminases acting on RNA (ADARs) are involved in RNA editing that converts adenosine residues to inosine specifically in double-stranded RNAs.  In this study, we investigated the interaction of the RNA editing mechanism with the RNA interference (RNAi) machinery and found that ADAR1 forms a complex with Dicer through direct protein-protein interaction.  Most importantly, ADAR1 increases the max. rate (Vmax) of pre-microRNA (miRNA) cleavage by Dicer and facilitates loading of miRNA onto RNA-induced silencing complexes, identifying a new role of ADAR1 in miRNA processing and RNAi mechanisms.  ADAR1 differentiates its functions in RNA editing and RNAi by the formation of either ADAR1/ADAR1 homodimer or Dicer/ADAR1 heterodimer complexes, resp.  As expected, the expression of miRNAs is globally inhibited in ADAR1-/- mouse embryos, which, in turn, alters the expression of their target genes and might contribute to their embryonic lethal phenotype.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJLya0xKC2i7Vg90H21EOLACvtfcHk0ljUb4Ai0beMZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXmslegtrc%253D&md5=92bac637be6db8cb4f92cf66d7166de3</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2013.03.024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2013.03.024%26sid%3Dliteratum%253Aachs%26aulast%3DOta%26aufirst%3DH.%26aulast%3DSakurai%26aufirst%3DM.%26aulast%3DGupta%26aufirst%3DR.%26aulast%3DValente%26aufirst%3DL.%26aulast%3DWulff%26aufirst%3DB.%2BE.%26aulast%3DAriyoshi%26aufirst%3DK.%26aulast%3DIizasa%26aufirst%3DH.%26aulast%3DDavuluri%26aufirst%3DR.%2BV.%26aulast%3DNishikura%26aufirst%3DK.%26atitle%3DADAR1%2520forms%2520a%2520complex%2520with%2520Dicer%2520to%2520promote%2520microRNA%2520processing%2520and%2520RNA-induced%2520gene%2520silencing%26jtitle%3DCell%26date%3D2013%26volume%3D153%26spage%3D575%26epage%3D589%26doi%3D10.1016%2Fj.cell.2013.03.024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seo, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, V. N.</span></span> <span> </span><span class="NLM_article-title">Deletion of human tarbp2 reveals cellular microRNA targets and cell-cycle function of TRBP</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1061</span>– <span class="NLM_lpage">1074</span>, <span class="refDoi"> DOI: 10.1016/j.celrep.2014.09.039</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1016%2Fj.celrep.2014.09.039" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=25437560" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVantbbL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=1061-1074&author=Y.+Kimauthor=J.+Yeoauthor=J.+H.+Leeauthor=J.+Choauthor=D.+Seoauthor=J.+S.+Kimauthor=V.+N.+Kim&title=Deletion+of+human+tarbp2+reveals+cellular+microRNA+targets+and+cell-cycle+function+of+TRBP&doi=10.1016%2Fj.celrep.2014.09.039"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Deletion of Human tarbp2 Reveals Cellular MicroRNA Targets and Cell-Cycle Function of TRBP</span></div><div class="casAuthors">Kim, Yoosik; Yeo, Jinah; Lee, Jung Hyun; Cho, Jun; Seo, Daekwan; Kim, Jong-Seo; Kim, V. Narry</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1061-1074</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">TRBP functions as both a Dicer cofactor and a PKR inhibitor.  However, the role of TRBP in microRNA (miRNA) biogenesis is controversial and its regulation of PKR in mitosis remains unexplored.  Here, we generate TRBP knockout cells and find altered Dicer-processing sites in a subset of miRNAs but no effect on Dicer stability, miRNA abundance, or Argonaute loading.  By generating PACT, another Dicer interactor, and TRBP/PACT double knockout (KO) cells, we further show that TRBP and PACT do not functionally compensate for one another and that only TRBP contributes to Dicer processing.  We also report that TRBP is hyperphosphorylated by JNK in M phase when PKR is activated by cellular double-stranded RNAs (dsRNAs).  Hyperphosphorylation potentiates the inhibitory activity of TRBP on PKR, suppressing PKR in M-G1 transition.  By generating human TRBP KO cells, our study clarifies the role of TRBP and unveils neg. feedback regulation of PKR through TRBP phosphorylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdd5hwHvIYs7Vg90H21EOLACvtfcHk0liwbTfCgqT2Rw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVantbbL&md5=9a5cf49eb26b6b8bf15483da4087db6b</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2014.09.039&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2014.09.039%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DY.%26aulast%3DYeo%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DJ.%2BH.%26aulast%3DCho%26aufirst%3DJ.%26aulast%3DSeo%26aufirst%3DD.%26aulast%3DKim%26aufirst%3DJ.%2BS.%26aulast%3DKim%26aufirst%3DV.%2BN.%26atitle%3DDeletion%2520of%2520human%2520tarbp2%2520reveals%2520cellular%2520microRNA%2520targets%2520and%2520cell-cycle%2520function%2520of%2520TRBP%26jtitle%3DCell%2520Rep.%26date%3D2014%26volume%3D9%26spage%3D1061%26epage%3D1074%26doi%3D10.1016%2Fj.celrep.2014.09.039" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peters, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Braun, R. E.</span></span> <span> </span><span class="NLM_article-title">A double-stranded RNA binding protein required for activation of repressed messages in mammalian germ cells</span>. <i>Nat. Genet.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">171</span>– <span class="NLM_lpage">174</span>, <span class="refDoi"> DOI: 10.1038/9684</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1038%2F9684" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10369260" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADyaK1MXjvVelsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=1999&pages=171-174&author=J.+Zhongauthor=A.+H.+Petersauthor=K.+Leeauthor=R.+E.+Braun&title=A+double-stranded+RNA+binding+protein+required+for+activation+of+repressed+messages+in+mammalian+germ+cells&doi=10.1038%2F9684"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">A double-stranded RNA binding protein required for activation of repressed messages in mammalian germ cells</span></div><div class="casAuthors">Zhong, Jun; Peters, Antoine H. F. M.; Lee, Keesook; Braun, Robert E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">171-174</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature America</span>)
        </div><div class="casAbstract">Chromatin packaging in mammalian spermatozoa requires an ordered replacement of the somatic histones by two classes of spermatid-specific basic proteins, the transition proteins and the protamines.  Temporal expression of transition proteins and protamines during spermatid differentiation is under translational control, and premature translation of protamine 1 leads to precocious nuclear condensation and sterility.  The authors have previously suggested that the double-stranded (ds) RNA binding protein Prbp (encoded by the gene Tarbp2) functions as a translational regulator during mouse spermatogenesis.  Here the authors show that Prbp is required for proper translational activation of the mRNAs encoding the protamines.  The authors generated mice that carry a targeted disruption of Tarbp2 and detd. that they were sterile and severely oligospermic.  Using immunohistol. anal., they detd. that the endogenous Prm2 mRNA and a reporter mRNA carrying protamine 1 translational-control elements were translated in a mosaic pattern.  The authors showed that failure to synthesize the protamines resulted in delayed replacement of the transition proteins and subsequent failure of spermiation.  The timing of Prbp expression suggests that it may function as a chaperone in the assembly of specific translationally regulated ribonucleoprotein particles.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrvyhVt7hShNbVg90H21EOLACvtfcHk0liwbTfCgqT2Rw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXjvVelsbk%253D&md5=9367584833d42e6b1294f420c57a3777</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1038%2F9684&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F9684%26sid%3Dliteratum%253Aachs%26aulast%3DZhong%26aufirst%3DJ.%26aulast%3DPeters%26aufirst%3DA.%2BH.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DBraun%26aufirst%3DR.%2BE.%26atitle%3DA%2520double-stranded%2520RNA%2520binding%2520protein%2520required%2520for%2520activation%2520of%2520repressed%2520messages%2520in%2520mammalian%2520germ%2520cells%26jtitle%3DNat.%2520Genet.%26date%3D1999%26volume%3D22%26spage%3D171%26epage%3D174%26doi%3D10.1038%2F9684" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Goodarzi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buss, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fish, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tavazoie, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tavazoie, S. F.</span></span> <span> </span><span class="NLM_article-title">Metastasis-suppressor transcript destabilization through TARBP2 binding of mRNA hairpins</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>513</i></span>,  <span class="NLM_fpage">256</span>– <span class="NLM_lpage">260</span>, <span class="refDoi"> DOI: 10.1038/nature13466</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1038%2Fnature13466" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=25043050" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsFOltrjP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=513&publication_year=2014&pages=256-260&author=H.+Goodarziauthor=S.+Zhangauthor=C.+G.+Bussauthor=L.+Fishauthor=S.+Tavazoieauthor=S.+F.+Tavazoie&title=Metastasis-suppressor+transcript+destabilization+through+TARBP2+binding+of+mRNA+hairpins&doi=10.1038%2Fnature13466"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Metastasis-suppressor transcript destabilization through TARBP2 binding of mRNA hairpins</span></div><div class="casAuthors">Goodarzi, Hani; Zhang, Steven; Buss, Colin G.; Fish, Lisa; Tavazoie, Saeed; Tavazoie, Sohail F.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">513</span>
        (<span class="NLM_cas:issue">7517</span>),
    <span class="NLM_cas:pages">256-260</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Aberrant regulation of RNA stability has an important role in many disease states.  Deregulated post-transcriptional modulation, such as that governed by microRNAs targeting linear sequence elements in mRNAs, has been implicated in the progression of many cancer types.  A defining feature of RNA is its ability to fold into structures.  However, the roles of structural mRNA elements in cancer progression remain unexplored.  Here we performed an unbiased search for post-transcriptional modulators of mRNA stability in breast cancer by conducting whole-genome transcript stability measurements in poorly and highly metastatic isogenic human breast cancer lines.  Using a computational framework that searches RNA sequence and structure space, we discovered a family of GC-rich structural cis-regulatory RNA elements, termed sRSEs for structural RNA stability elements, which are significantly overrepresented in transcripts displaying reduced stability in highly metastatic cells.  By integrating computational and biochem. approaches, we identified TARBP2, a double-stranded RNA-binding protein implicated in microRNA processing, as the trans factor that binds the sRSE family and similar structural elements-collectively termed TARBP2-binding structural elements (TBSEs)-in transcripts.  TARBP2 is overexpressed in metastatic cells and metastatic human breast tumors and destabilizes transcripts contg. TBSEs.  Endogenous TARBP2 promotes metastatic cell invasion and colonization by destabilizing amyloid precursor protein (APP) and ZNF395 transcripts, two genes previously assocd. with Alzheimer's and Huntington's disease, resp.  We reveal these genes to be novel metastasis suppressor genes in breast cancer.  The cleavage product of APP, extracellular amyloid-α peptide, directly suppresses invasion while ZNF395 transcriptionally represses a pro-metastatic gene expression program.  The expression levels of TARBP2, APP and ZNF395 in human breast carcinomas support their exptl. uncovered roles in metastasis.  Our findings establish a non-canonical and direct role for TARBP2 in mammalian gene expression regulation and reveal that regulated RNA destabilization through protein-mediated binding of mRNA structural elements can govern cancer progression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGre2AI_7HekqLVg90H21EOLACvtfcHk0liwbTfCgqT2Rw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsFOltrjP&md5=fdf3b64ca02bb25ad16dd9bca9500a83</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1038%2Fnature13466&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature13466%26sid%3Dliteratum%253Aachs%26aulast%3DGoodarzi%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DBuss%26aufirst%3DC.%2BG.%26aulast%3DFish%26aufirst%3DL.%26aulast%3DTavazoie%26aufirst%3DS.%26aulast%3DTavazoie%26aufirst%3DS.%2BF.%26atitle%3DMetastasis-suppressor%2520transcript%2520destabilization%2520through%2520TARBP2%2520binding%2520of%2520mRNA%2520hairpins%26jtitle%3DNature%26date%3D2014%26volume%3D513%26spage%3D256%26epage%3D260%26doi%3D10.1038%2Fnature13466" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Granito, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marinelli, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Negrini, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Menetti, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benevento, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bolondi, L.</span></span> <span> </span><span class="NLM_article-title">Prognostic significance of adverse events in patients with hepatocellular carcinoma treated with sorafenib</span>. <i>Ther. Adv. Gastroenterol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">240</span>– <span class="NLM_lpage">249</span>, <span class="refDoi"> DOI: 10.1177/1756283X15618129</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1177%2F1756283X15618129" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=26929785" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BC2sXntlenur4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2016&pages=240-249&author=A.+Granitoauthor=S.+Marinelliauthor=G.+Negriniauthor=S.+Menettiauthor=F.+Beneventoauthor=L.+Bolondi&title=Prognostic+significance+of+adverse+events+in+patients+with+hepatocellular+carcinoma+treated+with+sorafenib&doi=10.1177%2F1756283X15618129"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Prognostic significance of adverse events in patients with hepatocellular carcinoma treated with sorafenib</span></div><div class="casAuthors">Granito, Alessandro; Marinelli, Sara; Negrini, Giulia; Menetti, Saverio; Benevento, Francesca; Bolondi, Luigi</div><div class="citationInfo"><span class="NLM_cas:title">Therapeutic Advances in Gastroenterology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">240-249</span>CODEN:
                <span class="NLM_cas:coden">TAGHAO</span>;
        ISSN:<span class="NLM_cas:issn">1756-283X</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications Ltd.</span>)
        </div><div class="casAbstract">Sorafenib is the std. treatment for patients with hepatocellular carcinoma (HCC) with advanced stage disease.  Although its effectiveness has been demonstrated by randomized clin. trials and confirmed by field practice studies, reliable markers predicting therapeutic response have not yet been identified.  Like other tyrosine kinase inhibitors, treatment with sorafenib is burdened by the development of adverse effects, the most frequent being cutaneous toxicity, diarrhea, arterial hypertension and fatigue.  In recent years, several studies have analyzed the correlation between off-target effects and sorafenib efficacy in patients with HCC.  In this review, an overview of the studies assessing the prognostic significance of sorafenib-related adverse events is provided.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpU-IurkEqIA7Vg90H21EOLACvtfcHk0ljy4sRLJ8qAmQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXntlenur4%253D&md5=ce7bc70fcb6b7aab8ed9b33324688eba</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1177%2F1756283X15618129&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1756283X15618129%26sid%3Dliteratum%253Aachs%26aulast%3DGranito%26aufirst%3DA.%26aulast%3DMarinelli%26aufirst%3DS.%26aulast%3DNegrini%26aufirst%3DG.%26aulast%3DMenetti%26aufirst%3DS.%26aulast%3DBenevento%26aufirst%3DF.%26aulast%3DBolondi%26aufirst%3DL.%26atitle%3DPrognostic%2520significance%2520of%2520adverse%2520events%2520in%2520patients%2520with%2520hepatocellular%2520carcinoma%2520treated%2520with%2520sorafenib%26jtitle%3DTher.%2520Adv.%2520Gastroenterol.%26date%3D2016%26volume%3D9%26spage%3D240%26epage%3D249%26doi%3D10.1177%2F1756283X15618129" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nomura, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakachiyama, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hida, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohtaki, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sudo, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aizawa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aburada, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miyamoto, K. I.</span></span> <span> </span><span class="NLM_article-title">Gomisin A, a lignan component of Schizandora fruits, inhibits development of preneoplastic lesions in rat liver by 3′-methyl-4-dimethylamino-azobenzene</span>. <i>Cancer Lett.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>76</i></span>,  <span class="NLM_fpage">11</span>– <span class="NLM_lpage">18</span>, <span class="refDoi"> DOI: 10.1016/0304-3835(94)90128-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1016%2F0304-3835%2894%2990128-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=8124661" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADyaK2cXitlGls7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=76&publication_year=1994&pages=11-18&author=M.+Nomuraauthor=M.+Nakachiyamaauthor=T.+Hidaauthor=Y.+Ohtakiauthor=K.+Sudoauthor=T.+Aizawaauthor=M.+Aburadaauthor=K.+I.+Miyamoto&title=Gomisin+A%2C+a+lignan+component+of+Schizandora+fruits%2C+inhibits+development+of+preneoplastic+lesions+in+rat+liver+by+3%E2%80%B2-methyl-4-dimethylamino-azobenzene&doi=10.1016%2F0304-3835%2894%2990128-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Gomisin A, a lignan component of Schizandra fruits, inhibits development of preneoplastic lesions in rat liver by 3'-methyl-4-dimethylamino-azobenzene</span></div><div class="casAuthors">Nomura, M.; Nakachiyama, M.; Hida, T.; Ohtaki, Y.; Sudo, K.; Aizawa, T.; Aburada, M.; Miyamoto, K. I.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Letters (Shannon, Ireland)</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">76</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">11-18</span>CODEN:
                <span class="NLM_cas:coden">CALEDQ</span>;
        ISSN:<span class="NLM_cas:issn">0304-3835</span>.
    </div><div class="casAbstract">The effects of gomisin A, a lignan component of Schizandra fruits, on development of preneoplastic lesions in the liver after a short-term (3 wk) feeding of 3'-methyl-4-dimethylaminoazobenzene (3'-MeDAB) to male Donryu rats were investigated, and compared with the effects of phenobarbital.  Gomisin A inhibited both increases of the level of glutathione-S-transferases placental form (GST-P) and the no. and size of GST-P pos. foci in the liver increased after treatment with 3'-MeDAB.  Moreover, although the population of diploid nuclei was increased and that of tetraploid nuclei was decreased by pretreatment with 3'-MeDAB, gomisin A returned this to near the normal ploidy pattern.  But phenobarbital increased the level of GST-P and the no. and size of GST-P pos. foci with little affect on the ploidy population changed by 3'-MeDAB.  Thus, the effect of gomisin A on hepatocarcinogenesis was inhibitory in contrast with that of phenobarbital.  This study suggests that gomisin A is a candidate for a chemopreventive drug inhibiting the promotion process in hepatocarcinogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7ke46q2lcoLVg90H21EOLACvtfcHk0ljy4sRLJ8qAmQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXitlGls7w%253D&md5=402ce9f28057daa8b7f2230415439e50</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1016%2F0304-3835%2894%2990128-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0304-3835%252894%252990128-7%26sid%3Dliteratum%253Aachs%26aulast%3DNomura%26aufirst%3DM.%26aulast%3DNakachiyama%26aufirst%3DM.%26aulast%3DHida%26aufirst%3DT.%26aulast%3DOhtaki%26aufirst%3DY.%26aulast%3DSudo%26aufirst%3DK.%26aulast%3DAizawa%26aufirst%3DT.%26aulast%3DAburada%26aufirst%3DM.%26aulast%3DMiyamoto%26aufirst%3DK.%2BI.%26atitle%3DGomisin%2520A%252C%2520a%2520lignan%2520component%2520of%2520Schizandora%2520fruits%252C%2520inhibits%2520development%2520of%2520preneoplastic%2520lesions%2520in%2520rat%2520liver%2520by%25203%25E2%2580%25B2-methyl-4-dimethylamino-azobenzene%26jtitle%3DCancer%2520Lett.%26date%3D1994%26volume%3D76%26spage%3D11%26epage%3D18%26doi%3D10.1016%2F0304-3835%2894%2990128-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ibrahim, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmithals, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kowarz, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koberle, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kakoschky, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pleli, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kollmar, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nitsch, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waidmann, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finkelmeier, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeuzem, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korf, H. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmid, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weigert, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kronenberger, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marschalek, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piiper, A.</span></span> <span> </span><span class="NLM_article-title">Hypoxia causes downregulation of Dicer in hepatocellular carcinoma, which is required for upregulation of hypoxia-inducible factor 1alpha and epithelial-mesenchymal transition</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">3896</span>– <span class="NLM_lpage">3905</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.CCR-16-1762</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1158%2F1078-0432.CCR-16-1762" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=28167508" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtFKiurbN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2017&pages=3896-3905&author=A.+A.+Ibrahimauthor=C.+Schmithalsauthor=E.+Kowarzauthor=V.+Koberleauthor=B.+Kakoschkyauthor=T.+Pleliauthor=O.+Kollmarauthor=S.+Nitschauthor=O.+Waidmannauthor=F.+Finkelmeierauthor=S.+Zeuzemauthor=H.+W.+Korfauthor=T.+Schmidauthor=A.+Weigertauthor=B.+Kronenbergerauthor=R.+Marschalekauthor=A.+Piiper&title=Hypoxia+causes+downregulation+of+Dicer+in+hepatocellular+carcinoma%2C+which+is+required+for+upregulation+of+hypoxia-inducible+factor+1alpha+and+epithelial-mesenchymal+transition&doi=10.1158%2F1078-0432.CCR-16-1762"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Hypoxia Causes Downregulation of Dicer in Hepatocellular Carcinoma, Which Is Required for Upregulation of Hypoxia-Inducible Factor 1α and Epithelial-Mesenchymal Transition</span></div><div class="casAuthors">Ibrahim, Ahmed Atef; Schmithals, Christian; Kowarz, Erik; Koberle, Verena; Kakoschky, Bianca; Pleli, Thomas; Kollmar, Otto; Nitsch, Scarlett; Waidmann, Oliver; Finkelmeier, Fabian; Zeuzem, Stefan; Korf, Horst-Werner; Schmid, Tobias; Weigert, Andreas; Kronenberger, Bernd; Marschalek, Rolf; Piiper, Albrecht</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">3896-3905</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Purpose: A role of Dicer, which converts precursor miRNAs to mature miRNAs, in the tumor-promoting effect of hypoxia is currently emerging in some tumor entities.  Its role in hepatocellular carcinoma (HCC) is unknown.  Exptl. Design: HepG2 and Huh-7 cells were stably transfected with an inducible Dicer expression vector and were exposed to hypoxia/normoxia.  HepG2-Dicer xenografts were established in nude mice; hypoxic areas and Dicer were detected in HCC xenografts and HCCs from mice with endogenous hepatocarcinogenesis; and epithelial-mesenchymal transition (EMT) markers were analyzed by immunohistochem. or by immunoblotting.  The correlation between Dicer and carbonic anhydrase 9 (CA9), a marker of hypoxia, was investigated in resected human HCCs.  Results: Hypoxia increased EMT markers in vitro and in vivo and led to a downregulation of Dicer in HCC cells.  The levels of Dicer were downregulated in hypoxic tumor regions in mice with endogenous hepatocarcinogenesis and in HepG2 xenografts.  In human HCCs, the levels of Dicer correlated inversely with those of CA9, indicating that the neg. regulation of Dicer by hypoxia also applies to HCC patients.  Forced expression of Dicer prevented the hypoxia-induced increase in hypoxia-inducible factor 1α (HIF1α), HIF2α, hypoxia-inducible genes (CA9, glucose transporter 1), EMT markers, and cell migration.  Conclusions: We here identify downmodulation of Dicer as novel essential process in hypoxia-induced EMT in HCC and demonstrate that induced expression of Dicer counteracted hypoxia-induced EMT.  Thus, targeting hypoxia-induced downmodulation of Dicer is a promising novel strategy to reduce HCC progression.  Clin Cancer Res; 23(14); 3896-905. ©2017 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqiCM5iMZ0KMLVg90H21EOLACvtfcHk0liqBwGDrutYpQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtFKiurbN&md5=8c8e45c5641e60b848b9bec0901472a9</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-16-1762&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-16-1762%26sid%3Dliteratum%253Aachs%26aulast%3DIbrahim%26aufirst%3DA.%2BA.%26aulast%3DSchmithals%26aufirst%3DC.%26aulast%3DKowarz%26aufirst%3DE.%26aulast%3DKoberle%26aufirst%3DV.%26aulast%3DKakoschky%26aufirst%3DB.%26aulast%3DPleli%26aufirst%3DT.%26aulast%3DKollmar%26aufirst%3DO.%26aulast%3DNitsch%26aufirst%3DS.%26aulast%3DWaidmann%26aufirst%3DO.%26aulast%3DFinkelmeier%26aufirst%3DF.%26aulast%3DZeuzem%26aufirst%3DS.%26aulast%3DKorf%26aufirst%3DH.%2BW.%26aulast%3DSchmid%26aufirst%3DT.%26aulast%3DWeigert%26aufirst%3DA.%26aulast%3DKronenberger%26aufirst%3DB.%26aulast%3DMarschalek%26aufirst%3DR.%26aulast%3DPiiper%26aufirst%3DA.%26atitle%3DHypoxia%2520causes%2520downregulation%2520of%2520Dicer%2520in%2520hepatocellular%2520carcinoma%252C%2520which%2520is%2520required%2520for%2520upregulation%2520of%2520hypoxia-inducible%2520factor%25201alpha%2520and%2520epithelial-mesenchymal%2520transition%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2017%26volume%3D23%26spage%3D3896%26epage%3D3905%26doi%3D10.1158%2F1078-0432.CCR-16-1762" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yoo, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santhekadur, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gredler, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emdad, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhutia, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pannell, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fisher, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sarkar, D.</span></span> <span> </span><span class="NLM_article-title">Increased RNA-induced silencing complex (RISC) activity contributes to hepatocellular carcinoma</span>. <i>Hepatology</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">1538</span>– <span class="NLM_lpage">1548</span>, <span class="refDoi"> DOI: 10.1002/hep.24216</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1002%2Fhep.24216" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=21520169" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BC3MXltVygu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2011&pages=1538-1548&author=B.+K.+Yooauthor=P.+K.+Santhekadurauthor=R.+Gredlerauthor=D.+Chenauthor=L.+Emdadauthor=S.+Bhutiaauthor=L.+Pannellauthor=P.+B.+Fisherauthor=D.+Sarkar&title=Increased+RNA-induced+silencing+complex+%28RISC%29+activity+contributes+to+hepatocellular+carcinoma&doi=10.1002%2Fhep.24216"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Increased RNA-induced silencing complex (RISC) activity contributes to hepatocellular carcinoma</span></div><div class="casAuthors">Yoo, Byoung Kwon; Santhekadur, Prasanna K.; Gredler, Rachel; Chen, Dong; Emdad, Luni; Bhutia, Sujit; Pannell, Lewis; Fisher, Paul B.; Sarkar, Devanand</div><div class="citationInfo"><span class="NLM_cas:title">Hepatology (Hoboken, NJ, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1538-1548</span>CODEN:
                <span class="NLM_cas:coden">HPTLD9</span>;
        ISSN:<span class="NLM_cas:issn">0270-9139</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">There is virtually no effective treatment for advanced hepatocellular carcinoma (HCC) and novel targets need to be identified to develop effective treatment.  We recently documented that the oncogene Astrocyte elevated gene-1 (AEG-1) plays a seminal role in hepatocarcinogenesis.  Employing yeast two-hybrid assay and coimmunopptn. followed by mass spectrometry, we identified staphylococcal nuclease domain contg. 1 (SND1), a nuclease in the RNA-induced silencing complex (RISC) facilitating RNAi-mediated gene silencing, as an AEG-1 interacting protein.  Coimmunopptn. and colocalization studies confirmed that AEG-1 is also a component of RISC and both AEG-1 and SND1 are required for optimum RISC activity facilitating small interfering RNA (siRNA) and micro RNA (miRNA)-mediated silencing of luciferase reporter gene.  In 109 human HCC samples SND1 was overexpressed in ≈74% cases compared to normal liver.  Correspondingly, significantly higher RISC activity was obsd. in human HCC cells compared to immortal normal hepatocytes.  Increased RISC activity, conferred by AEG-1 or SND1, resulted in increased degrdn. of tumor suppressor mRNAs that are target of oncomiRs.  Inhibition of enzymic activity of SND1 significantly inhibited proliferation of human HCC cells.  As a corollary, stable overexpression of SND1 augmented and siRNA-mediated inhibition of SND1 abrogated growth of human HCC cells in vitro and in vivo, thus revealing a potential role of SND1 in hepatocarcinogenesis.  Conclusion: We unravel a novel mechanism that overexpression of AEG-1 and SND1 leading to increased RISC activity might contribute to hepatocarcinogenesis.  Targeted inhibition of SND1 enzymic activity might be developed as an effective therapy for HCC. (HEPATOl. 2011;).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrY2Bz39GT-77Vg90H21EOLACvtfcHk0liqBwGDrutYpQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXltVygu7o%253D&md5=86da7288e46fe36e5ae805fae8ea2f12</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1002%2Fhep.24216&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fhep.24216%26sid%3Dliteratum%253Aachs%26aulast%3DYoo%26aufirst%3DB.%2BK.%26aulast%3DSanthekadur%26aufirst%3DP.%2BK.%26aulast%3DGredler%26aufirst%3DR.%26aulast%3DChen%26aufirst%3DD.%26aulast%3DEmdad%26aufirst%3DL.%26aulast%3DBhutia%26aufirst%3DS.%26aulast%3DPannell%26aufirst%3DL.%26aulast%3DFisher%26aufirst%3DP.%2BB.%26aulast%3DSarkar%26aufirst%3DD.%26atitle%3DIncreased%2520RNA-induced%2520silencing%2520complex%2520%2528RISC%2529%2520activity%2520contributes%2520to%2520hepatocellular%2520carcinoma%26jtitle%3DHepatology%26date%3D2011%26volume%3D53%26spage%3D1538%26epage%3D1548%26doi%3D10.1002%2Fhep.24216" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, F. Y.</span></span> <span> </span><span class="NLM_article-title">Multi-targeted anticancer agents</span>. <i>Curr. Top. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">3084</span>– <span class="NLM_lpage">3098</span>, <span class="refDoi"> DOI: 10.2174/1568026617666170707124126</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.2174%2F1568026617666170707124126" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=28685693" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFens7fJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2017&pages=3084-3098&author=W.+Zhengauthor=Y.+Zhaoauthor=Q.+Luoauthor=Y.+Zhangauthor=K.+Wuauthor=F.+Y.+Wang&title=Multi-targeted+anticancer+agents&doi=10.2174%2F1568026617666170707124126"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Multi-Targeted Anticancer Agents</span></div><div class="casAuthors">Zheng, Wei; Zhao, Yao; Luo, Qun; Zhang, Yang; Wu, Kui; Wang, Fuyi</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">28</span>),
    <span class="NLM_cas:pages">3084-3098</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">There is a great demand for the development of novel anticancer drugs, due to the increasing morbidity and high mortality of cancer.  To date, chemotherapy still plays a central role in the clin. treatment of cancer, but this role is being reduced by targeted therapies.  Since cancer is a complicated and multiple genes involved disease, drugs that act at multiple targets can enhance efficacy and lower drug resistance, and are thought to be the future of novel anticancer drug development.  In this paper, we discuss the recent development of "single mol., multi-target" anticancer agents that combine two or more pharmacophores in a single mol., including org. multitargeted anticancer agents and metal based complexes such as platinum, ruthenium, iridium and rhodium complexes.  These efforts will contribute to clin. cancer therapy and benefit patients in the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoj7unFBNhn3rVg90H21EOLACvtfcHk0liADDpc8SbvRQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvFens7fJ&md5=b9f6834bf2af9c3e9460badd9ca133c8</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.2174%2F1568026617666170707124126&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1568026617666170707124126%26sid%3Dliteratum%253Aachs%26aulast%3DZheng%26aufirst%3DW.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DLuo%26aufirst%3DQ.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DWu%26aufirst%3DK.%26aulast%3DWang%26aufirst%3DF.%2BY.%26atitle%3DMulti-targeted%2520anticancer%2520agents%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2017%26volume%3D17%26spage%3D3084%26epage%3D3098%26doi%3D10.2174%2F1568026617666170707124126" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ding, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kataoka, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nie, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, Z. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pu, W. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, D. Z.</span></span> <span> </span><span class="NLM_article-title">Trbp regulates heart function through microRNA-mediated Sox6 repression</span>. <i>Nat. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">776</span>– <span class="NLM_lpage">783</span>, <span class="refDoi"> DOI: 10.1038/ng.3324</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1038%2Fng.3324" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=26029872" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFemt7vI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2015&pages=776-783&author=J.+Dingauthor=J.+Chenauthor=Y.+Wangauthor=M.+Kataokaauthor=L.+Maauthor=P.+Zhouauthor=X.+Huauthor=Z.+Linauthor=M.+Nieauthor=Z.+L.+Dengauthor=W.+T.+Puauthor=D.+Z.+Wang&title=Trbp+regulates+heart+function+through+microRNA-mediated+Sox6+repression&doi=10.1038%2Fng.3324"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Trbp regulates heart function through microRNA-mediated Sox6 repression</span></div><div class="casAuthors">Ding, Jian; Chen, Jinghai; Wang, Yanqun; Kataoka, Masaharu; Ma, Lixin; Zhou, Pingzhu; Hu, Xiaoyun; Lin, Zhiqiang; Nie, Mao; Deng, Zhong-Liang; Pu, William T.; Wang, Da-Zhi</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">776-783</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Cardiomyopathy is assocd. with altered expression of genes encoding contractile proteins.  Here we show that Trbp (Tarbp2), an RNA-binding protein, is required for normal heart function.  Cardiac-specific inactivation in mice of Trbp (TrbpcKO) caused progressive cardiomyopathy and lethal heart failure.  Loss of Trbp function resulted in upregulation of Sox6, repression of genes encoding normal cardiac slow-twitch myofiber proteins and pathol. increased expression of genes encoding skeletal fast-twitch myofiber proteins.  Remarkably, knockdown of Sox6 fully rescued the Trbp-mutant phenotype, whereas mice overexpressing Sox6 phenocopied TrbpcKO mice.  Trbp inactivation was mechanistically linked to Sox6 upregulation through altered processing of miR-208a, which is a direct inhibitor of Sox6.  Transgenic overexpression of Mir208a sufficiently repressed Sox6, restored the balance in gene expression for fast- and slow-twitch myofiber proteins, and rescued cardiac function in TrbpcKO mice.  Together, our studies identify a new Trbp-mediated microRNA-processing mechanism in the regulation of a linear genetic cascade essential for normal heart function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_b3TtPlJgtrVg90H21EOLACvtfcHk0liADDpc8SbvRQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFemt7vI&md5=8de59059910c041985ba0f11c716191a</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1038%2Fng.3324&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fng.3324%26sid%3Dliteratum%253Aachs%26aulast%3DDing%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DKataoka%26aufirst%3DM.%26aulast%3DMa%26aufirst%3DL.%26aulast%3DZhou%26aufirst%3DP.%26aulast%3DHu%26aufirst%3DX.%26aulast%3DLin%26aufirst%3DZ.%26aulast%3DNie%26aufirst%3DM.%26aulast%3DDeng%26aufirst%3DZ.%2BL.%26aulast%3DPu%26aufirst%3DW.%2BT.%26aulast%3DWang%26aufirst%3DD.%2BZ.%26atitle%3DTrbp%2520regulates%2520heart%2520function%2520through%2520microRNA-mediated%2520Sox6%2520repression%26jtitle%3DNat.%2520Genet.%26date%3D2015%26volume%3D47%26spage%3D776%26epage%3D783%26doi%3D10.1038%2Fng.3324" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shan, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szulwach, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faghihi, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khalil, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paroo, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wahlestedt, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, P.</span></span> <span> </span><span class="NLM_article-title">A small molecule enhances RNA interference and promotes microRNA processing</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">933</span>– <span class="NLM_lpage">940</span>, <span class="refDoi"> DOI: 10.1038/nbt.1481</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1038%2Fnbt.1481" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=18641635" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BD1cXps1Wls7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2008&pages=933-940&author=G.+Shanauthor=Y.+Liauthor=J.+Zhangauthor=W.+Liauthor=K.+E.+Szulwachauthor=R.+Duanauthor=M.+A.+Faghihiauthor=A.+M.+Khalilauthor=L.+Luauthor=Z.+Parooauthor=A.+W.+Chanauthor=Z.+Shiauthor=Q.+Liuauthor=C.+Wahlestedtauthor=C.+Heauthor=P.+Jin&title=A+small+molecule+enhances+RNA+interference+and+promotes+microRNA+processing&doi=10.1038%2Fnbt.1481"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">A small molecule enhances RNA interference and promotes microRNA processing</span></div><div class="casAuthors">Shan, Ge; Li, Yujing; Zhang, Junliang; Li, Wendi; Szulwach, Keith E.; Duan, Ranhui; Faghihi, Mohammad A.; Khalil, Ahmad M.; Lu, Lianghua; Paroo, Zain; Chan, Anthony W. S.; Shi, Zhangjie; Liu, Qinghua; Wahlestedt, Claes; He, Chuan; Jin, Peng</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">933-940</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Little is known about the regulation of RNA interference (RNAi).  Shan et al. constructed a reporter system to monitor RNAi activity and identified a small mol. that increases RNAi by facilitating the interaction between small RNAs and a protein involved in small RNA loading and processing.  Little is known about the regulation of RNA interference (RNAi).  Shan et al. constructed a reporter system to monitor RNAi activity and identified a small mol. that increases RNAi by facilitating the interaction between small RNAs and a protein involved in small RNA loading and processing.  Small interfering RNAs (siRNAs) and microRNAs (miRNAs) are sequence-specific post-transcriptional regulators of gene expression.  Although major components of the RNA interference (RNAi) pathway have been identified, regulatory mechanisms for this pathway remain largely unknown.  Here we demonstrate that the RNAi pathway can be modulated intracellularly by small mols.  We have developed a cell-based assay to monitor the activity of the RNAi pathway and find that the small-mol. enoxacin (Penetrex) enhances siRNA-mediated mRNA degrdn. and promotes the biogenesis of endogenous miRNAs.  We show that this RNAi-enhancing activity depends on the trans-activation-responsive region RNA-binding protein.  Our results provide a proof-of-principle demonstration that small mols. can be used to modulate the activity of the RNAi pathway.  RNAi enhancers may be useful in the development of research tools and therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmMvLQLjUDALVg90H21EOLACvtfcHk0lgD_SUTd29TZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXps1Wls7c%253D&md5=15210fd18afcb865b6d2e218449c42b4</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1038%2Fnbt.1481&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt.1481%26sid%3Dliteratum%253Aachs%26aulast%3DShan%26aufirst%3DG.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DSzulwach%26aufirst%3DK.%2BE.%26aulast%3DDuan%26aufirst%3DR.%26aulast%3DFaghihi%26aufirst%3DM.%2BA.%26aulast%3DKhalil%26aufirst%3DA.%2BM.%26aulast%3DLu%26aufirst%3DL.%26aulast%3DParoo%26aufirst%3DZ.%26aulast%3DChan%26aufirst%3DA.%2BW.%26aulast%3DShi%26aufirst%3DZ.%26aulast%3DLiu%26aufirst%3DQ.%26aulast%3DWahlestedt%26aufirst%3DC.%26aulast%3DHe%26aufirst%3DC.%26aulast%3DJin%26aufirst%3DP.%26atitle%3DA%2520small%2520molecule%2520enhances%2520RNA%2520interference%2520and%2520promotes%2520microRNA%2520processing%26jtitle%3DNat.%2520Biotechnol.%26date%3D2008%26volume%3D26%26spage%3D933%26epage%3D940%26doi%3D10.1038%2Fnbt.1481" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sousa, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graca, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baptista, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vieira, F. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palmeira, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henrique, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeronimo, C.</span></span> <span> </span><span class="NLM_article-title">Enoxacin inhibits growth of prostate cancer cells and effectively restores microRNA processing</span>. <i>Epigenetics</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">548</span>– <span class="NLM_lpage">558</span>, <span class="refDoi"> DOI: 10.4161/epi.24519</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.4161%2Fepi.24519" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=23644875" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjs1yitr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=548-558&author=E.+Sousaauthor=I.+Gracaauthor=T.+Baptistaauthor=F.+Q.+Vieiraauthor=C.+Palmeiraauthor=R.+Henriqueauthor=C.+Jeronimo&title=Enoxacin+inhibits+growth+of+prostate+cancer+cells+and+effectively+restores+microRNA+processing&doi=10.4161%2Fepi.24519"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Enoxacin inhibits growth of prostate cancer cells and effectively restores microRNA processing</span></div><div class="casAuthors">Sousa, Elsa J.; Graca, Ines; Baptista, Tiago; Vieira, Filipa Q.; Palmeira, Carlos; Henrique, Rui; Jeronimo, Carmen</div><div class="citationInfo"><span class="NLM_cas:title">Epigenetics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">548-558</span>CODEN:
                <span class="NLM_cas:coden">EPIGB6</span>;
        ISSN:<span class="NLM_cas:issn">1559-2294</span>.
    
            (<span class="NLM_cas:orgname">Landes Bioscience</span>)
        </div><div class="casAbstract">Prostate cancer (PCa) is one of the most incident malignancies worldwide.  Although efficient therapy is available for early-stage PCa, treatment of advanced disease is mainly ineffective and remains a clin. challenge.  MicroRNA (miRNA) dysregulation is assocd. with PCa development and progression.  In fact, several studies have reported a widespread downregulation of miRNAs in PCa, which highlights the importance of studying compds. capable of restoring the global miRNA expression.  The main aim of this study was to define the usefulness of enoxacin as an anti-tumoral agent in PCa, due to its ability to induce miRNA biogenesis in a TRBP-mediated manner.  Using a panel of five PCa cell lines, we obsd. that all of them were wild type for the TARBP2 gene and expressed TRBP protein.  Furthermore, primary prostate carcinomas displayed normal levels of TRBP protein.  Remarkably, enoxacin was able to decrease cell viability, induce apoptosis, cause cell cycle arrest, and inhibit the invasiveness of cell lines.  Enoxacin was also effective in restoring the global expression of miRNAs.  This study is the first to show that PCa cells are highly responsive to the anti-tumoral effects of enoxacin.  Therefore, enoxacin constitutes a promising therapeutic agent for PCa.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUsYbG1rODRbVg90H21EOLACvtfcHk0lgD_SUTd29TZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjs1yitr4%253D&md5=88b039ce63ee7c0e682bbd184f6d7da3</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.4161%2Fepi.24519&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252Fepi.24519%26sid%3Dliteratum%253Aachs%26aulast%3DSousa%26aufirst%3DE.%26aulast%3DGraca%26aufirst%3DI.%26aulast%3DBaptista%26aufirst%3DT.%26aulast%3DVieira%26aufirst%3DF.%2BQ.%26aulast%3DPalmeira%26aufirst%3DC.%26aulast%3DHenrique%26aufirst%3DR.%26aulast%3DJeronimo%26aufirst%3DC.%26atitle%3DEnoxacin%2520inhibits%2520growth%2520of%2520prostate%2520cancer%2520cells%2520and%2520effectively%2520restores%2520microRNA%2520processing%26jtitle%3DEpigenetics%26date%3D2013%26volume%3D8%26spage%3D548%26epage%3D558%26doi%3D10.4161%2Fepi.24519" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schulze, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nault, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villanueva, A.</span></span> <span> </span><span class="NLM_article-title">Genetic profiling of hepatocellular carcinoma using next-generation sequencing</span>. <i>J. Hepatol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>65</i></span>,  <span class="NLM_fpage">1031</span>– <span class="NLM_lpage">1042</span>, <span class="refDoi"> DOI: 10.1016/j.jhep.2016.05.035</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1016%2Fj.jhep.2016.05.035" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=27262756" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVantLjN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=2016&pages=1031-1042&author=K.+Schulzeauthor=J.+C.+Naultauthor=A.+Villanueva&title=Genetic+profiling+of+hepatocellular+carcinoma+using+next-generation+sequencing&doi=10.1016%2Fj.jhep.2016.05.035"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Genetic profiling of hepatocellular carcinoma using next-generation sequencing</span></div><div class="casAuthors">Schulze, Kornelius; Nault, Jean-Charles; Villanueva, Augusto</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Hepatology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">65</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1031-1042</span>CODEN:
                <span class="NLM_cas:coden">JOHEEC</span>;
        ISSN:<span class="NLM_cas:issn">0168-8278</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Hepatocellular carcinoma (HCC) is a highly heterogeneous disease, both clin. and from a mol. standpoint.  The advent of next-generation sequencing technologies has provided new opportunities to extensively analyze mol. defects in HCC samples.  This has uncovered major cancer driver genes and assocd. oncogenic pathways operating in HCC.  More sophisticated analyses of sequencing data have linked specific nucleotide patterns to external toxic agents and defined so-called 'mutational signatures' in HCC.  Mol. signatures, taking into account intra- and inter-tumor heterogeneity, and their functional validation could provide useful data to predict treatment response to mol. therapies.  In this review we will focus on the current knowledge of deep sequencing in HCC and its foreseeable clin. impact.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpI8a2Sa6XEYbVg90H21EOLACvtfcHk0lhuCWlWeIfKOQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVantLjN&md5=049570fd840487e7db157e39760e6ccb</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1016%2Fj.jhep.2016.05.035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jhep.2016.05.035%26sid%3Dliteratum%253Aachs%26aulast%3DSchulze%26aufirst%3DK.%26aulast%3DNault%26aufirst%3DJ.%2BC.%26aulast%3DVillanueva%26aufirst%3DA.%26atitle%3DGenetic%2520profiling%2520of%2520hepatocellular%2520carcinoma%2520using%2520next-generation%2520sequencing%26jtitle%3DJ.%2520Hepatol.%26date%3D2016%26volume%3D65%26spage%3D1031%26epage%3D1042%26doi%3D10.1016%2Fj.jhep.2016.05.035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Behan, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iorio, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picco, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goncalves, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beaver, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Migliardi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santos, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sassi, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pinnelli, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ansari, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harper, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jackson, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McRae, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pooley, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilkinson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der Meer, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dow, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buser-Doepner, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertotti, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trusolino, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stronach, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saez-Rodriguez, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yusa, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garnett, M. J.</span></span> <span> </span><span class="NLM_article-title">Prioritization of cancer therapeutic targets using crispr-Cas9 screens</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>568</i></span>,  <span class="NLM_fpage">511</span>– <span class="NLM_lpage">516</span>, <span class="refDoi"> DOI: 10.1038/s41586-019-1103-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1038%2Fs41586-019-1103-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=30971826" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BC1MXosVSgtrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=568&publication_year=2019&pages=511-516&author=F.+M.+Behanauthor=F.+Iorioauthor=G.+Piccoauthor=E.+Goncalvesauthor=C.+M.+Beaverauthor=G.+Migliardiauthor=R.+Santosauthor=Y.+Raoauthor=F.+Sassiauthor=M.+Pinnelliauthor=R.+Ansariauthor=S.+Harperauthor=D.+A.+Jacksonauthor=R.+McRaeauthor=R.+Pooleyauthor=P.+Wilkinsonauthor=D.+van+der+Meerauthor=D.+Dowauthor=C.+Buser-Doepnerauthor=A.+Bertottiauthor=L.+Trusolinoauthor=E.+A.+Stronachauthor=J.+Saez-Rodriguezauthor=K.+Yusaauthor=M.+J.+Garnett&title=Prioritization+of+cancer+therapeutic+targets+using+crispr-Cas9+screens&doi=10.1038%2Fs41586-019-1103-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Prioritization of cancer therapeutic targets using CRISPR-Cas9 screens</span></div><div class="casAuthors">Behan, Fiona M.; Iorio, Francesco; Picco, Gabriele; Goncalves, Emanuel; Beaver, Charlotte M.; Migliardi, Giorgia; Santos, Rita; Rao, Yanhua; Sassi, Francesco; Pinnelli, Marika; Ansari, Rizwan; Harper, Sarah; Jackson, David Adam; McRae, Rebecca; Pooley, Rachel; Wilkinson, Piers; van der Meer, Dieudonne; Dow, David; Buser-Doepner, Carolyn; Bertotti, Andrea; Trusolino, Livio; Stronach, Euan A.; Saez-Rodriguez, Julio; Yusa, Kosuke; Garnett, Mathew J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">568</span>
        (<span class="NLM_cas:issue">7753</span>),
    <span class="NLM_cas:pages">511-516</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Functional genomics approaches can overcome limitations-such as the lack of identification of robust targets and poor clin. efficacy-that hamper cancer drug development.  Here we performed genome-scale CRISPR-Cas9 screens in 324 human cancer cell lines from 30 cancer types and developed a data-driven framework to prioritize candidates for cancer therapeutics.  We integrated cell fitness effects with genomic biomarkers and target tractability for drug development to systematically prioritize new targets in defined tissues and genotypes.  We verified one of our most promising dependencies, the Werner syndrome ATP-dependent helicase, as a synthetic lethal target in tumors from multiple cancer types with microsatellite instability.  Our anal. provides a resource of cancer dependencies, generates a framework to prioritize cancer drug targets and suggests specific new targets.  The principles described in this study can inform the initial stages of drug development by contributing to a new, diverse and more effective portfolio of cancer drug targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHnWQcCFsy-LVg90H21EOLACvtfcHk0lhuCWlWeIfKOQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXosVSgtrw%253D&md5=3cac7e1a9ee5c34c8599154e8fc30573</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1038%2Fs41586-019-1103-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-019-1103-9%26sid%3Dliteratum%253Aachs%26aulast%3DBehan%26aufirst%3DF.%2BM.%26aulast%3DIorio%26aufirst%3DF.%26aulast%3DPicco%26aufirst%3DG.%26aulast%3DGoncalves%26aufirst%3DE.%26aulast%3DBeaver%26aufirst%3DC.%2BM.%26aulast%3DMigliardi%26aufirst%3DG.%26aulast%3DSantos%26aufirst%3DR.%26aulast%3DRao%26aufirst%3DY.%26aulast%3DSassi%26aufirst%3DF.%26aulast%3DPinnelli%26aufirst%3DM.%26aulast%3DAnsari%26aufirst%3DR.%26aulast%3DHarper%26aufirst%3DS.%26aulast%3DJackson%26aufirst%3DD.%2BA.%26aulast%3DMcRae%26aufirst%3DR.%26aulast%3DPooley%26aufirst%3DR.%26aulast%3DWilkinson%26aufirst%3DP.%26aulast%3Dvan%2Bder%2BMeer%26aufirst%3DD.%26aulast%3DDow%26aufirst%3DD.%26aulast%3DBuser-Doepner%26aufirst%3DC.%26aulast%3DBertotti%26aufirst%3DA.%26aulast%3DTrusolino%26aufirst%3DL.%26aulast%3DStronach%26aufirst%3DE.%2BA.%26aulast%3DSaez-Rodriguez%26aufirst%3DJ.%26aulast%3DYusa%26aufirst%3DK.%26aulast%3DGarnett%26aufirst%3DM.%2BJ.%26atitle%3DPrioritization%2520of%2520cancer%2520therapeutic%2520targets%2520using%2520crispr-Cas9%2520screens%26jtitle%3DNature%26date%3D2019%26volume%3D568%26spage%3D511%26epage%3D516%26doi%3D10.1038%2Fs41586-019-1103-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gatignol, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckler-White, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berkhout, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeang, K. T.</span></span> <span> </span><span class="NLM_article-title">Characterization of a human TAR RNA-binding protein that activates the HIV-1 LTR</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">1991</span>,  <span class="NLM_volume"><i>251</i></span>,  <span class="NLM_fpage">1597</span>– <span class="NLM_lpage">1600</span>, <span class="refDoi"> DOI: 10.1126/science.2011739</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1126%2Fscience.2011739" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=2011739" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADyaK3MXlslKgs7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=251&publication_year=1991&pages=1597-1600&author=A.+Gatignolauthor=A.+Buckler-Whiteauthor=B.+Berkhoutauthor=K.+T.+Jeang&title=Characterization+of+a+human+TAR+RNA-binding+protein+that+activates+the+HIV-1+LTR&doi=10.1126%2Fscience.2011739"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of a human TAR RNA-binding protein that activates the HIV-1 LTR</span></div><div class="casAuthors">Gatignol, Anne; Buckler-White, Alicia; Berkhout, Ben; Jeang, Kuan Teh</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">251</span>
        (<span class="NLM_cas:issue">5001</span>),
    <span class="NLM_cas:pages">1597-600</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    </div><div class="casAbstract">Human immunodeficiency virus type 1 (HIV-1) gene expression is activated by Tat, a virally encoded protein.  Tat trans-activation requires viral (trans-activation-responsive; TAR) RNA sequences located in the R region of the long terminal repeat (LTR).  Existing evidence suggests that Tat probably cooperates with cellular factors that bind to TAR RNA in the overall trans-activation process.  A HeLa cDNA was isolated and characterized that encodes a TAR RNA-binding protein (TRBP).  TRBP activated the HIV-1 LTR and was synergistic with Tat function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoqs0r0DyFmSLVg90H21EOLACvtfcHk0ljPZJV7YVM42Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3MXlslKgs7w%253D&md5=19ff68923d636e815072329827f3ecd8</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1126%2Fscience.2011739&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.2011739%26sid%3Dliteratum%253Aachs%26aulast%3DGatignol%26aufirst%3DA.%26aulast%3DBuckler-White%26aufirst%3DA.%26aulast%3DBerkhout%26aufirst%3DB.%26aulast%3DJeang%26aufirst%3DK.%2BT.%26atitle%3DCharacterization%2520of%2520a%2520human%2520TAR%2520RNA-binding%2520protein%2520that%2520activates%2520the%2520HIV-1%2520LTR%26jtitle%3DScience%26date%3D1991%26volume%3D251%26spage%3D1597%26epage%3D1600%26doi%3D10.1126%2Fscience.2011739" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kilgore, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, D. F.</span></span> <span> </span><span class="NLM_article-title">Rubrisandrins A and B, lignans and related anti-HIV compounds from Schisandra rubriflora</span>. <i>J. Nat. Prod.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>69</i></span>,  <span class="NLM_fpage">1697</span>– <span class="NLM_lpage">1701</span>, <span class="refDoi"> DOI: 10.1021/np060239e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/np060239e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BD28Xht1WhsLzE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2006&pages=1697-1701&author=M.+Chenauthor=N.+Kilgoreauthor=K.+H.+Leeauthor=D.+F.+Chen&title=Rubrisandrins+A+and+B%2C+lignans+and+related+anti-HIV+compounds+from+Schisandra+rubriflora&doi=10.1021%2Fnp060239e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Rubrisandrins A and B, Lignans and Related Anti-HIV Compounds from Schisandra rubriflora</span></div><div class="casAuthors">Chen, Min; Kilgore, Nicole; Lee, Kuo-Hsiung; Chen, Dao-Feng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Natural Products</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1697-1701</span>CODEN:
                <span class="NLM_cas:coden">JNPRDF</span>;
        ISSN:<span class="NLM_cas:issn">0163-3864</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society-American Society of Pharmacognosy</span>)
        </div><div class="casAbstract">Bioactivity-directed fractionation of an ethanolic ext. of the fruits of Schisandra rubriflora led to the isolation and identification of dibenzocyclooctadiene lignans including the new lignans rubrisandrins A (1a + 1b) and B (2) and the known lignans gomisin J (3), (±)-gomisin M1 (4), (+)-gomisin M2 (5), schisanhenol (6), deoxyschisandrin, schisantherin B, schisandrin, tigloylgomisin P, gomisin O, angeloylgomisin P, and epigomisin O.  Their structure and stereochem. were detd. by spectroscopic methods, including 2D-NMR techniques.  Compds. 1 and 3-6 were active as anti-HIV agents.  (±)-Gomisin M1 (4) exhibited the most potent anti-HIV activity, with EC50 and therapeutic index (TI) values of <0.65 μM and >68, resp.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTBEiiLbO66bVg90H21EOLACvtfcHk0ljPZJV7YVM42Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xht1WhsLzE&md5=048953e15712f9e93593d6489e2add33</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1021%2Fnp060239e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fnp060239e%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DM.%26aulast%3DKilgore%26aufirst%3DN.%26aulast%3DLee%26aufirst%3DK.%2BH.%26aulast%3DChen%26aufirst%3DD.%2BF.%26atitle%3DRubrisandrins%2520A%2520and%2520B%252C%2520lignans%2520and%2520related%2520anti-HIV%2520compounds%2520from%2520Schisandra%2520rubriflora%26jtitle%3DJ.%2520Nat.%2520Prod.%26date%3D2006%26volume%3D69%26spage%3D1697%26epage%3D1701%26doi%3D10.1021%2Fnp060239e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, J. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kashiwada, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, B. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cosentino, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K. H.</span></span> <span> </span><span class="NLM_article-title">Anti-AIDS agents--XXVI. Structure-activity correlations of Gomisin-G-related anti-HIV lignans from Kadsura interior and of related synthetic analogues</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">1715</span>– <span class="NLM_lpage">1723</span>, <span class="refDoi"> DOI: 10.1016/S0968-0896(97)00118-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1016%2FS0968-0896%2897%2900118-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=9313872" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADyaK2sXmsVGrs70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=1997&pages=1715-1723&author=D.+F.+Chenauthor=S.+X.+Zhangauthor=L.+Xieauthor=J.+X.+Xieauthor=K.+Chenauthor=Y.+Kashiwadaauthor=B.+N.+Zhouauthor=P.+Wangauthor=L.+M.+Cosentinoauthor=K.+H.+Lee&title=Anti-AIDS+agents%2D%2DXXVI.+Structure-activity+correlations+of+Gomisin-G-related+anti-HIV+lignans+from+Kadsura+interior+and+of+related+synthetic+analogues&doi=10.1016%2FS0968-0896%2897%2900118-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Anti-AIDS agents-XXVI. Structure-activity correlations of gomisin-G-related anti-HIV lignans from Kadsura interior and of related synthetic analogs</span></div><div class="casAuthors">Chen, Dao-Feng; Zhang, Shun-Xiang; Xie, Lan; Xie, Jing-Xi; Chen, Ke; Kashiwada, Yoshiki; Zhou, Bing-Nan; Wang, Pei; Cosentino, L. Mark; Lee, Kuo-Hsiung</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1715-1723</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Bioactivity-directed fractionation of an ethanolic ext. of the stems of Kadsura interior led to the isolation and identification of 12 known lignans.  Seven of these compds. were active as anti-HIV agents.  Gomisin-G exhibited the most potent anti-HIV activity with EC50 and therapeutic index (TI) values of 0.006 μg/mL and 300, resp.  Schisantherin-D, kadsuranin, and schisandrin-C showed good activity with EC50 values of 0.5, 0.8, and 1.2 μg/mL, and TI values of 110, 56, and 33.3, resp.  Ten related synthetic biphenyl compds., five variously substituted bismethylenedioxy, dimethoxy, and dimethoxycarbonyl isomers and five brominated derivs. also were evaluated for inhibitory activity against HIV-1 replication in acutely infected H9 cells.  The total syntheses of two new isomers are reported for the first time.  The anti-HIV data indicated that the relative position and types of substituents on the phenolic hydroxy groups of either the natural lignans or the synthetic biphenyl compds. rather than the nos. of bromine(s) on the arom. rings are of primary importance.  In the cyclooctane ring of the natural lignans, the position and substitution of hydroxy groups are also important to enhanced anti-HIV activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJsMP2Q4CfULVg90H21EOLACvtfcHk0lidtE7lcaAm4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXmsVGrs70%253D&md5=132467a2f62b457e709c311cc49e3897</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1016%2FS0968-0896%2897%2900118-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0968-0896%252897%252900118-1%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DD.%2BF.%26aulast%3DZhang%26aufirst%3DS.%2BX.%26aulast%3DXie%26aufirst%3DL.%26aulast%3DXie%26aufirst%3DJ.%2BX.%26aulast%3DChen%26aufirst%3DK.%26aulast%3DKashiwada%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DB.%2BN.%26aulast%3DWang%26aufirst%3DP.%26aulast%3DCosentino%26aufirst%3DL.%2BM.%26aulast%3DLee%26aufirst%3DK.%2BH.%26atitle%3DAnti-AIDS%2520agents--XXVI.%2520Structure-activity%2520correlations%2520of%2520Gomisin-G-related%2520anti-HIV%2520lignans%2520from%2520Kadsura%2520interior%2520and%2520of%2520related%2520synthetic%2520analogues%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D1997%26volume%3D5%26spage%3D1715%26epage%3D1723%26doi%3D10.1016%2FS0968-0896%2897%2900118-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nozaki-Renard, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iino, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sato, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marumoto, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohta, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furusawa, M.</span></span> <span> </span><span class="NLM_article-title">Fluoroquinolones protect the human lymphocyte CEM cell line from HIV-1-mediated cytotoxicity</span>. <i>Cell Struct. Funct.</i> <span class="NLM_year" style="font-weight: bold;">1990</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">295</span>– <span class="NLM_lpage">299</span>, <span class="refDoi"> DOI: 10.1247/csf.15.295</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1247%2Fcsf.15.295" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=2085844" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADyaK3MXhvF2isrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=1990&pages=295-299&author=J.+Nozaki-Renardauthor=T.+Iinoauthor=Y.+Satoauthor=Y.+Marumotoauthor=G.+Ohtaauthor=M.+Furusawa&title=Fluoroquinolones+protect+the+human+lymphocyte+CEM+cell+line+from+HIV-1-mediated+cytotoxicity&doi=10.1247%2Fcsf.15.295"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Fluoroquinolones protect the human lymphocyte CEM cell line from HIV-1-mediated cytotoxicity</span></div><div class="casAuthors">Nozaki-Renard, Junko; Iino, Takashi; Sato, Yoshinari; Marumoto, Yasumasa; Ohta, Genkichi; Furusawa, Mitsuru</div><div class="citationInfo"><span class="NLM_cas:title">Cell Structure and Function</span>
        (<span class="NLM_cas:date">1990</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">295-9</span>CODEN:
                <span class="NLM_cas:coden">CSFUDY</span>;
        ISSN:<span class="NLM_cas:issn">0386-7196</span>.
    </div><div class="casAbstract">Infection of the human lymphocyte CEM cell line with the HIV-1 (human immunodeficiency virus type-1, LAV-1 strain) results in cell death.  A fluoroquinolone antibiotic, ofloxacin, protected the infected cells from HIV-1-mediated cytolysis.  Other fluoroquinolones, e.g. ciprofloxacin, norfloxacin, and enoxacin, also protected the infected cells from HIV-1-mediated cytolysis.  The d-isomer of ofloxacin (DR-3354) was about 50-fold less effective than the l-isomer (DR-3355).  Almost none of the rescued cells had detectable HIV-antigens and they could be maintained for long periods in vitro without drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpInRxyWd8OybVg90H21EOLACvtfcHk0lidtE7lcaAm4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3MXhvF2isrg%253D&md5=5acc04da45143d08e6fcbd9c947abdfa</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1247%2Fcsf.15.295&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1247%252Fcsf.15.295%26sid%3Dliteratum%253Aachs%26aulast%3DNozaki-Renard%26aufirst%3DJ.%26aulast%3DIino%26aufirst%3DT.%26aulast%3DSato%26aufirst%3DY.%26aulast%3DMarumoto%26aufirst%3DY.%26aulast%3DOhta%26aufirst%3DG.%26aulast%3DFurusawa%26aufirst%3DM.%26atitle%3DFluoroquinolones%2520protect%2520the%2520human%2520lymphocyte%2520CEM%2520cell%2520line%2520from%2520HIV-1-mediated%2520cytotoxicity%26jtitle%3DCell%2520Struct.%2520Funct.%26date%3D1990%26volume%3D15%26spage%3D295%26epage%3D299%26doi%3D10.1247%2Fcsf.15.295" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref64"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref64'); return false;" data-citation="" class="refNumLink">64</a></strong><div class="NLM_citation" id="cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Venkanna, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poornima, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siva, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Babu, B. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Babu, K. S.</span></span> <span> </span><span class="NLM_article-title">Towards the total synthesis of Schisandrene: stereoselective synthesis of the dibenzocyclooctadiene lignan core</span>. <i>Synlett</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">908</span>– <span class="NLM_lpage">911</span>, <span class="refDoi"> DOI: 10.1055/s-0036-1591539</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1055%2Fs-0036-1591539" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BC1cXivFGrs7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2018&pages=908-911&author=A.+Venkannaauthor=B.+Poornimaauthor=B.+Sivaauthor=B.+H.+Babuauthor=K.+S.+Babu&title=Towards+the+total+synthesis+of+Schisandrene%3A+stereoselective+synthesis+of+the+dibenzocyclooctadiene+lignan+core&doi=10.1055%2Fs-0036-1591539"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Towards the Total Synthesis of Schisandrene: Stereoselective Synthesis of the Dibenzocyclooctadiene Lignan Core</span></div><div class="casAuthors">Venkanna, Arramshetti; Poornima, Borra; Siva, Bandi; Babu, B. Hari; Babu, K. Suresh</div><div class="citationInfo"><span class="NLM_cas:title">Synlett</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">908-911</span>CODEN:
                <span class="NLM_cas:coden">SYNLES</span>;
        ISSN:<span class="NLM_cas:issn">0936-5214</span>.
    
            (<span class="NLM_cas:orgname">Georg Thieme Verlag</span>)
        </div><div class="casAbstract">A stereoselective synthesis of the dibenzocyclooctadiene lignan core, I, of the natural product schisandrene (II) is described.  Starting from readily available gallic acid, the synthetic strategy involves Suzuki-Miyaura cross-coupling, Stille reaction, and ring-closing metathesis (RCM) in the reaction sequence.  The required asym. center at C-7' was established by an asym. redn. of a keto compd. using the Corey-Bakshi-Shibata (CBS) catalyst.  In our approach, the eight-membered ring was achieved by RCM for the first time.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgMb9gXgSr-7Vg90H21EOLACvtfcHk0lhlEge80SEscg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXivFGrs7k%253D&md5=5d43e9211c044c53fb752dce4c5bd969</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1055%2Fs-0036-1591539&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1055%252Fs-0036-1591539%26sid%3Dliteratum%253Aachs%26aulast%3DVenkanna%26aufirst%3DA.%26aulast%3DPoornima%26aufirst%3DB.%26aulast%3DSiva%26aufirst%3DB.%26aulast%3DBabu%26aufirst%3DB.%2BH.%26aulast%3DBabu%26aufirst%3DK.%2BS.%26atitle%3DTowards%2520the%2520total%2520synthesis%2520of%2520Schisandrene%253A%2520stereoselective%2520synthesis%2520of%2520the%2520dibenzocyclooctadiene%2520lignan%2520core%26jtitle%3DSynlett%26date%3D2018%26volume%3D29%26spage%3D908%26epage%3D911%26doi%3D10.1055%2Fs-0036-1591539" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref65"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref65'); return false;" data-citation="" class="refNumLink">65</a></strong><div class="NLM_citation" id="cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Alam, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takaguchi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ito, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshida, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsuboi, S.</span></span> <span> </span><span class="NLM_article-title">Multi-functionalization of gallic acid towards improved synthesis of α- and β-DDB</span>. <i>Tetrahedron</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">1909</span>– <span class="NLM_lpage">1918</span>, <span class="refDoi"> DOI: 10.1016/j.tet.2004.11.083</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1016%2Fj.tet.2004.11.083" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BD2MXoslGquw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2005&pages=1909-1918&author=A.+Alamauthor=Y.+Takaguchiauthor=H.+Itoauthor=T.+Yoshidaauthor=S.+Tsuboi&title=Multi-functionalization+of+gallic+acid+towards+improved+synthesis+of+%CE%B1-+and+%CE%B2-DDB&doi=10.1016%2Fj.tet.2004.11.083"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Multi-functionalization of gallic acid towards improved synthesis of α- and β-DDB</span></div><div class="casAuthors">Alam, Ashraful; Takaguchi, Yutaka; Ito, Hideyuki; Yoshida, Takashi; Tsuboi, Sadao</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1909-1918</span>CODEN:
                <span class="NLM_cas:coden">TETRAB</span>;
        ISSN:<span class="NLM_cas:issn">0040-4020</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The synthesis of mono-, di- and trisubstituted gallic acids and their ester with similar or different groups including different acetal and ketals is described.  Regioselective bromination on two ortho-positions of Me gallate, which is very crucial for many org. syntheses, was achieved in high yield and purity.  The α- (I) and β-DDB (II) were synthesized in high overall yield and purity from the regioselective bromo-derivs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8dD8akD52YrVg90H21EOLACvtfcHk0lhlEge80SEscg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXoslGquw%253D%253D&md5=39d96d563db8b5c6cbce0ea22619c7ec</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1016%2Fj.tet.2004.11.083&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tet.2004.11.083%26sid%3Dliteratum%253Aachs%26aulast%3DAlam%26aufirst%3DA.%26aulast%3DTakaguchi%26aufirst%3DY.%26aulast%3DIto%26aufirst%3DH.%26aulast%3DYoshida%26aufirst%3DT.%26aulast%3DTsuboi%26aufirst%3DS.%26atitle%3DMulti-functionalization%2520of%2520gallic%2520acid%2520towards%2520improved%2520synthesis%2520of%2520%25CE%25B1-%2520and%2520%25CE%25B2-DDB%26jtitle%3DTetrahedron%26date%3D2005%26volume%3D61%26spage%3D1909%26epage%3D1918%26doi%3D10.1016%2Fj.tet.2004.11.083" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref66"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref66'); return false;" data-citation="" class="refNumLink">66</a></strong><div class="NLM_citation" id="cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, G. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pu, J. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, H. D.</span></span> <span> </span><span class="NLM_article-title">Dibenzocyclooctadiene lignans from Schisandra wilsoniana and their anti-HIV-1 activities</span>. <i>J. Nat. Prod.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>73</i></span>,  <span class="NLM_fpage">915</span>– <span class="NLM_lpage">919</span>, <span class="refDoi"> DOI: 10.1021/np100067w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/np100067w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BC3cXlvFKqtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2010&pages=915-919&author=G.+Y.+Yangauthor=Y.+K.+Liauthor=R.+R.+Wangauthor=X.+N.+Liauthor=W.+L.+Xiaoauthor=L.+M.+Yangauthor=J.+X.+Puauthor=Y.+T.+Zhengauthor=H.+D.+Sun&title=Dibenzocyclooctadiene+lignans+from+Schisandra+wilsoniana+and+their+anti-HIV-1+activities&doi=10.1021%2Fnp100067w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">Dibenzocyclooctadiene Lignans from Schisandra wilsoniana and Their Anti-HIV-1 Activities</span></div><div class="casAuthors">Yang, Guang-Yu; Li, Yin-Ke; Wang, Rui-Rui; Li, Xiao-Nian; Xiao, Wei-Lie; Yang, Liu-Meng; Pu, Jian-Xin; Zheng, Yong-Tang; Sun, Han-Dong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Natural Products</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">73</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">915-919</span>CODEN:
                <span class="NLM_cas:coden">JNPRDF</span>;
        ISSN:<span class="NLM_cas:issn">0163-3864</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society-American Society of Pharmacognosy</span>)
        </div><div class="casAbstract">Twelve new dibenzocyclooctadiene lignans, marlignans A-L (1-12), together with 16 known compds., were isolated from the leaves and stems of Schisandra wilsoniana.  The structures of 1-12 were elucidated by spectroscopic methods including 1D- and 2D-NMR techniques.  Compds. 1-12 were evaluated for their anti-HIV activities, of which compds. 3, 6, 8, and 12 showed modest activities with therapeutic index values of 13.2, 15.6, 17.6, and 16.4, resp.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxspLjw3yVFrVg90H21EOLACvtfcHk0ljVu66v8Zgdhg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXlvFKqtrY%253D&md5=f35c4b77e0a3ae4e9fa2f98aa5429a38</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1021%2Fnp100067w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fnp100067w%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DG.%2BY.%26aulast%3DLi%26aufirst%3DY.%2BK.%26aulast%3DWang%26aufirst%3DR.%2BR.%26aulast%3DLi%26aufirst%3DX.%2BN.%26aulast%3DXiao%26aufirst%3DW.%2BL.%26aulast%3DYang%26aufirst%3DL.%2BM.%26aulast%3DPu%26aufirst%3DJ.%2BX.%26aulast%3DZheng%26aufirst%3DY.%2BT.%26aulast%3DSun%26aufirst%3DH.%2BD.%26atitle%3DDibenzocyclooctadiene%2520lignans%2520from%2520Schisandra%2520wilsoniana%2520and%2520their%2520anti-HIV-1%2520activities%26jtitle%3DJ.%2520Nat.%2520Prod.%26date%3D2010%26volume%3D73%26spage%3D915%26epage%3D919%26doi%3D10.1021%2Fnp100067w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62'],'ref63':['cit63'],'ref64':['cit64'],'ref65':['cit65'],'ref66':['cit66']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article has not yet been cited by other publications.</p></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00018/20210602/images/medium/jm1c00018_0010.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00018/20210602/images/large/jm1c00018_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00018&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00018/20210602/images/medium/jm1c00018_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00018/20210602/images/large/jm1c00018_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. GM modulates miRNA biogenesis to suppress HCC proliferation and metastasis. (A) Model of high-throughput screening by HeLa-miR-21-luciferase cells. (B) Chemical structure of GM. (C and D) Luciferase activity in HeLa-miR-21-luciferase cells determined after treatment with GM and enoxacin, respectively, at the indicated concentrations. (E and F) qRT-PCR of the relative miRNA levels in SK-Hep-1 cells treated with GM and enoxacin, respectively, at 20 μM. (G) Representative images showing the effect of GM and enoxacin on colony formation 14 days after treatment at the noted concentrations in SK-Hep-1 cells. (H) Flow cytometry quantitation of the proportion of SK-Hep-1 cells treated with or without GM at 20 μM. (I and J) Representative images showing the effect of 30 μM GM or enoxacin on the migration and invasion, respectively, of SK-Hep-1 cells. Statistical significance was determined by a two-tailed Student’s <i>t</i> test. *<i>p</i> < 0.05, **<i>p</i> < 0.01, and ***<i>p</i> < 0.001 compared with the control. N.S, not significant.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00018/20210602/images/large/jm1c00018_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00018&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00018/20210602/images/medium/jm1c00018_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00018/20210602/images/large/jm1c00018_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. GM suppresses HCC tumor formation <i>in vivo</i>. (A) Representative luminescence images of mice one month after the first administration, showing anticancer activity of GM in the SK-Hep-1 orthotopic xenograft nude mouse model. (B) Tumors were quantified on the basis of the degree of luminescence (<i>n</i> = 6). (C) Representative images of liver tumors from mice one month after the first administration. (D) Mice weight over 30 days (<i>n</i> = 6). (E and F) H&E-stained sections of the left outer lobe and middle lobe, respectively, of the liver collected on day 30 (<i>n</i> = 6). A two-tailed unpaired <i>t</i> test was used for the statistical analysis in panels B and C, with means ± standard errors of the mean. *<i>p</i> < 0.05, **<i>p</i> < 0.01, and ***<i>p</i> < 0.001 compared with the control. N.S, not significant. Black and red arrow: tumor. Blue arrow: immune cell infiltration.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00018/20210602/images/large/jm1c00018_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00018&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00018/20210602/images/medium/jm1c00018_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00018/20210602/images/large/jm1c00018_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. GM modulates the miRNAome and proteome in HCC cells. (A) Heat map representing all differentially expressed miRNAs with a <i>p</i> value of <0.05 and a >2-fold change (<i>n</i> = 3). (B) qRT-PCR of the relative miRNA levels of the indicated genes. (C) Bubble diagram showing the pathway enrichment analysis of miRNA target genes. (D) Heat map representing all differentially expressed proteins with a <i>p</i> value of <0.05 and a >1.3-fold change (<i>n</i> = 3). (E) Western blotting analysis of protein expression in SK-Hep-1 cells treated with GM at a series of concentrations. (F) Histogram showing the functional enrichment analysis of differentially expressed proteins. (G) Interactions among differentially expressed proteins shown by a network graph. Statistical significance was determined by a two-tailed Student’s <i>t</i> test. *<i>p</i> < 0.05, **<i>p</i> < 0.01, and ***<i>p</i> < 0.001 compared with the control.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00018/20210602/images/large/jm1c00018_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00018&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00018/20210602/images/medium/jm1c00018_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00018/20210602/images/large/jm1c00018_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. TRBP modulates the anticancer effect of GM. (A and B) Western blotting and melt curve analysis showing the results of the cellular thermal shift assay in SK-Hep-1 cellular lysates treated with 50 μM GM at different (A) temperatures or (B) concentrations at 55 °C. (C and D) Representative images of a colony formation assay in SK-Hep-1 cells performed to assess the antiproliferative effect of GM (20 μM) after overexpressing and silencing, respectively, the <i>TRBP</i> gene. (E and F) Transwell assay of SK-Hep-1 cellular migration to assess the effect of GM (20 μM) after overexpressing and silencing, respectively, of the <i>TRBP</i> gene. All of the numerical data are presented as means ± standard deviations.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00018/20210602/images/large/jm1c00018_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00018&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00018/20210602/images/medium/jm1c00018_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00018/20210602/images/large/jm1c00018_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. GM physically interacts with TRBP. (A–C) Binding of GM and TRBP was analyzed by CSP and STD-NMR. In CSP, <sup>1</sup>H spectra of ∼20 μM GM with 0 (black), 7.5 (yellow), 15 (green), 30 (red), and 60 (blue) μM TRBP were recorded and overlapped. In STD, spectra of ∼20 μM GM without (black, as a control) and with 30 μM TRBP (red) were recorded. (A) Full CSP and STD spectra of the GM with TRBP show the H<sub>2</sub>O resonance. Peaks C-5-OMe, C-7-OMe, and C-8-OMe denoted the NMR resonance of methoxyl groups in GM. Magnified CSP and STD signals of (B) C-5-OMe or (C) C-7-OMe and C-8-OMe. (D and E) SPR assay showing the interaction between GM and TRBP. (D) Steady-state kinetics and (E) affinity constant of GM at different concentrations were calculated by a CM5 Biacore chip-immobilized TRBP protein. (F) Co-immunoprecipitation analysis of the interaction between TRBP and Dicer or Ago2 in SK-Hep-1 cells following treatment with 50 μM GM.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00018/20210602/images/large/jm1c00018_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00018&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00018/20210602/images/medium/jm1c00018_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00018/20210602/images/large/jm1c00018_0007.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Structural Simplification of GM</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00018/20210602/images/large/jm1c00018_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00018&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch2" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00018/20210602/images/medium/jm1c00018_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00018/20210602/images/large/jm1c00018_0008.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of the GM Analogues (<b>1–20</b>)<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00018/20210602/images/large/jm1c00018_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00018&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) (i) NaOH, CH<sub>3</sub>OH/H<sub>2</sub>O, rt; (ii) NH<sub>4</sub>Cl; (b) (i) SOCl<sub>2</sub>, CH<sub>2</sub>Cl<sub>2</sub>, 45 °C, reflux; (ii) Et<sub>3</sub>N, pyridine, R<sub>1</sub>NH<sub>2</sub>, rt; (c) (i) 1-chloroethyl chloroformate, CHCl<sub>3</sub>, 100 °C, reflux; (ii) EtOH, 100 °C, reflux; (d) NCS, CH<sub>2</sub>Cl<sub>2</sub>, rt; (e) benzoyl chloride, Et<sub>3</sub>N, CH<sub>2</sub>Cl<sub>2</sub>, 0 °C to rt; (f) R<sub>2</sub>SO<sub>2</sub>Cl, Et<sub>3</sub>N, CH<sub>2</sub>Cl<sub>2</sub>, 0 °C to rt; (g) R<sub>3</sub>COOH, 1-ethyl-3-[3-(dimethylamino)propyl]carbodiimide hydrochloride (EDCI), 1-hydroxybenzotriazole (HOBt), DMAP, rt; (h) trifluoroacetic acid (TFA), CH<sub>2</sub>Cl<sub>2</sub>, 0 °C to rt.</p></p></figure><figure data-id="fig6" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00018/20210602/images/medium/jm1c00018_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00018/20210602/images/large/jm1c00018_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Roles of GM analogues in anti-HCC. (A) Histogram showing the effect of 10 μM GM analogues on miR-21 biogenesis in HeLa-miR-21-luciferase cells. (B and C) Representative image showing the effect of <b>7</b> and <b>21</b> on the (B) proliferation, (C, top) migration, and (C, bottom) invasion of SK-Hep-1 cells at 20 μM. (D) Pull down assay showing the binding between <b>21</b> and TRBP protein.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00018/20210602/images/large/jm1c00018_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00018&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch3" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00018/20210602/images/medium/jm1c00018_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00018/20210602/images/large/jm1c00018_0009.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthetic Route for Compound <b>21</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00018/20210602/images/large/jm1c00018_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00018&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i67">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_56111" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_56111" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 66 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Villanueva, A.</span></span> <span> </span><span class="NLM_article-title">Hepatocellular carcinoma</span>. <i>N. Engl. J. Med.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>380</i></span>,  <span class="NLM_fpage">1450</span>– <span class="NLM_lpage">1462</span>, <span class="refDoi"> DOI: 10.1056/NEJMra1713263</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1056%2FNEJMra1713263" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=30970190" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BC1MXotFehsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=380&publication_year=2019&pages=1450-1462&author=A.+Villanueva&title=Hepatocellular+carcinoma&doi=10.1056%2FNEJMra1713263"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Hepatocellular carcinoma</span></div><div class="casAuthors">Villanueva, Augusto</div><div class="citationInfo"><span class="NLM_cas:title">New England Journal of Medicine</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">380</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">1450-1462</span>CODEN:
                <span class="NLM_cas:coden">NEJMAG</span>;
        ISSN:<span class="NLM_cas:issn">1533-4406</span>.
    
            (<span class="NLM_cas:orgname">Massachusetts Medical Society</span>)
        </div><div class="casAbstract">A review summarizing the main genetic alterations in hepatocellular carcinoma, key epidemiol. features, and evidence-based approaches to its clin. management.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_ukXAOkaaW7Vg90H21EOLACvtfcHk0liMSxue1_Y2cw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXotFehsb4%253D&md5=d0237fa3302cf7a48a6439023d1452be</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1056%2FNEJMra1713263&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMra1713263%26sid%3Dliteratum%253Aachs%26aulast%3DVillanueva%26aufirst%3DA.%26atitle%3DHepatocellular%2520carcinoma%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2019%26volume%3D380%26spage%3D1450%26epage%3D1462%26doi%3D10.1056%2FNEJMra1713263" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, Y. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsao, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tak, W. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burock, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Voliotis, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guan, Z.</span></span> <span> </span><span class="NLM_article-title">Efficacy and safety of sorafenib in patients in the Asia-Pacific region with advanced hepatocellular carcinoma: a phase III randomised, double-blind, placebo-controlled trial</span>. <i>Lancet Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">25</span>– <span class="NLM_lpage">34</span>, <span class="refDoi"> DOI: 10.1016/S1470-2045(08)70285-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1016%2FS1470-2045%2808%2970285-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=19095497" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsFChtbnM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2009&pages=25-34&author=A.+L.+Chengauthor=Y.+K.+Kangauthor=Z.+Chenauthor=C.+J.+Tsaoauthor=S.+Qinauthor=J.+S.+Kimauthor=R.+Luoauthor=J.+Fengauthor=S.+Yeauthor=T.+S.+Yangauthor=J.+Xuauthor=Y.+Sunauthor=H.+Liangauthor=J.+Liuauthor=J.+Wangauthor=W.+Y.+Takauthor=H.+Panauthor=K.+Burockauthor=J.+Zouauthor=D.+Voliotisauthor=Z.+Guan&title=Efficacy+and+safety+of+sorafenib+in+patients+in+the+Asia-Pacific+region+with+advanced+hepatocellular+carcinoma%3A+a+phase+III+randomised%2C+double-blind%2C+placebo-controlled+trial&doi=10.1016%2FS1470-2045%2808%2970285-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Efficacy and safety of sorafenib in patients in the Asia-Pacific region with advanced hepatocellular carcinoma: a phase III randomized, double-blind, placebo-controlled trial</span></div><div class="casAuthors">Cheng, Ann-Lii; Kang, Yoon-Koo; Chen, Zhendong; Tsao, Chao-Jung; Qin, Shukui; Kim, Jun Suk; Luo, Rongcheng; Feng, Jifeng; Ye, Shenglong; Yang, Tsai-Sheng; Xu, Jianming; Sun, Yan; Liang, Houjie; Liu, Jiwei; Wang, Jiejun; Tak, Won Young; Pan, Hongming; Burock, Karin; Zou, Jessie; Voliotis, Dimitris; Guan, Zhongzhen</div><div class="citationInfo"><span class="NLM_cas:title">Lancet Oncology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">25-34</span>CODEN:
                <span class="NLM_cas:coden">LOANBN</span>;
        ISSN:<span class="NLM_cas:issn">1470-2045</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Most cases of hepatocellular carcinoma occur in the Asia-Pacific region, where chronic hepatitis B infection is an important etiol. factor.  Assessing the efficacy and safety of new therapeutic options in an Asia-Pacific population is thus important.  We did a multinational phase III, randomized, double-blind, placebo-controlled trial to assess the efficacy and safety of sorafenib in patients from the Asia-Pacific region with advanced (unresectable or metastatic) hepatocellular carcinoma.  Between Sept 20, 2005, and Jan 31, 2007, patients with hepatocellular carcinoma who had not received previous systemic therapy and had Child-Pugh liver function class A, were randomly assigned to receive either oral sorafenib (400 mg) or placebo twice daily in 6-wk cycles, with efficacy measured at the end of each 6-wk period.  Eligible patients were stratified by the presence or absence of macroscopic vascular invasion or extrahepatic spread (or both), Eastern Cooperative Oncol. Group performance status, and geog. region.  Randomization was done centrally and in a 2:1 ratio by means of an interactive voice-response system.  There was no predefined primary endpoint; overall survival, time to progression (TTP), time to symptomatic progression (TTSP), disease control rate (DCR), and safety were assessed.  Efficacy analyses were done by intention to treat.  This trial is registered with, no.  Two hundred and seventy-one 271 patients from 23 centers in China, South Korea, and Taiwan were enrolled in the study.  Of these, 226 patients were randomly assigned to the exptl. group (n=150) or to the placebo group (n=76).  Median overall survival was 6.5 mo (95% CI 5.56-7.56) in patients treated with sorafenib, compared with 4.2 mo (3.75-5.46) in those who received placebo (hazard ratio [HR] 0.68 [95% CI 0.50-0.93]; p=0.014).  Median TTP was 2.8 mo (2.63-3.58) in the sorafenib group compared with 1.4 mo (1.35-1.55) in the placebo group (HR 0.57 [0.42-0.79]; p=0.0005).  The most frequently reported grade 3/4 drug-related adverse events in the 149 assessable patients treated with sorafenib were hand-foot skin reaction (HFSR; 16 patients [10.7%]), diarrhea (nine patients [6.0%]), and fatigue (five patients [3.4%]).  The most common adverse events resulting in dose redns. were HFSR (17 patients [11.4%]) and diarrhea (11 patients [7.4%]); these adverse events rarely led to discontinuation.  Sorafenib is effective for the treatment of advanced hepatocellular carcinoma in patients from the Asia-Pacific region, and is well tolerated.  Taken together with data from the Sorafenib Hepatocellular Carcinoma Assessment Randomized Protocol (SHARP) trial, sorafenib seems to be an appropriate option for the treatment of advanced hepatocellular carcinoma.  Funding: Bayer HealthCare Pharmaceuticals and Onyx Pharmaceuticals, Inc.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo9BFS2SjaE8LVg90H21EOLACvtfcHk0liMSxue1_Y2cw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsFChtbnM&md5=9ec357bcb2990914da1bc41e31532902</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2FS1470-2045%2808%2970285-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1470-2045%252808%252970285-7%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DA.%2BL.%26aulast%3DKang%26aufirst%3DY.%2BK.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DTsao%26aufirst%3DC.%2BJ.%26aulast%3DQin%26aufirst%3DS.%26aulast%3DKim%26aufirst%3DJ.%2BS.%26aulast%3DLuo%26aufirst%3DR.%26aulast%3DFeng%26aufirst%3DJ.%26aulast%3DYe%26aufirst%3DS.%26aulast%3DYang%26aufirst%3DT.%2BS.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DSun%26aufirst%3DY.%26aulast%3DLiang%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DTak%26aufirst%3DW.%2BY.%26aulast%3DPan%26aufirst%3DH.%26aulast%3DBurock%26aufirst%3DK.%26aulast%3DZou%26aufirst%3DJ.%26aulast%3DVoliotis%26aufirst%3DD.%26aulast%3DGuan%26aufirst%3DZ.%26atitle%3DEfficacy%2520and%2520safety%2520of%2520sorafenib%2520in%2520patients%2520in%2520the%2520Asia-Pacific%2520region%2520with%2520advanced%2520hepatocellular%2520carcinoma%253A%2520a%2520phase%2520III%2520randomised%252C%2520double-blind%252C%2520placebo-controlled%2520trial%26jtitle%3DLancet%2520Oncol.%26date%3D2009%26volume%3D10%26spage%3D25%26epage%3D34%26doi%3D10.1016%2FS1470-2045%2808%2970285-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kudo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finn, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ikeda, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piscaglia, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baron, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jassem, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blanc, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vogel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Komov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, T. R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dutcus, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saito, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kraljevic, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tamai, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, A. L.</span></span> <span> </span><span class="NLM_article-title">Lenvatinib versus sorafenib in first-line treatment of patients with unresectable hepatocellular carcinoma: a randomised phase 3 non-inferiority trial</span>. <i>Lancet</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>391</i></span>,  <span class="NLM_fpage">1163</span>– <span class="NLM_lpage">1173</span>, <span class="refDoi"> DOI: 10.1016/S0140-6736(18)30207-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1016%2FS0140-6736%2818%2930207-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=29433850" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisFOjtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=391&publication_year=2018&pages=1163-1173&author=M.+Kudoauthor=R.+S.+Finnauthor=S.+Qinauthor=K.+H.+Hanauthor=K.+Ikedaauthor=F.+Piscagliaauthor=A.+Baronauthor=J.+W.+Parkauthor=G.+Hanauthor=J.+Jassemauthor=J.+F.+Blancauthor=A.+Vogelauthor=D.+Komovauthor=T.+R.+J.+Evansauthor=C.+Lopezauthor=C.+Dutcusauthor=M.+Guoauthor=K.+Saitoauthor=S.+Kraljevicauthor=T.+Tamaiauthor=M.+Renauthor=A.+L.+Cheng&title=Lenvatinib+versus+sorafenib+in+first-line+treatment+of+patients+with+unresectable+hepatocellular+carcinoma%3A+a+randomised+phase+3+non-inferiority+trial&doi=10.1016%2FS0140-6736%2818%2930207-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Lenvatinib versus sorafenib in first-line treatment of patients with unresectable hepatocellular carcinoma: a randomised phase 3 non-inferiority trial</span></div><div class="casAuthors">Kudo, Masatoshi; Finn, Richard S.; Qin, Shukui; Han, Kwang-Hyub; Ikeda, Kenji; Piscaglia, Fabio; Baron, Ari; Park, Joong-Won; Han, Guohong; Jassem, Jacek; Blanc, Jean Frederic; Vogel, Arndt; Komov, Dmitry; Evans, T. R. Jeffry; Lopez, Carlos; Dutcus, Corina; Guo, Matthew; Saito, Kenichi; Kraljevic, Silvija; Tamai, Toshiyuki; Ren, Min; Cheng, Ann-Lii</div><div class="citationInfo"><span class="NLM_cas:title">Lancet</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">391</span>
        (<span class="NLM_cas:issue">10126</span>),
    <span class="NLM_cas:pages">1163-1173</span>CODEN:
                <span class="NLM_cas:coden">LANCAO</span>;
        ISSN:<span class="NLM_cas:issn">0140-6736</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Background: In a phase 2 trial, lenvatinib, an inhibitor of VEGF receptors 1-3, FGF receptors 1-4, PDGF receptor α, RET, and KIT, showed activity in hepatocellular carcinoma.  We aimed to compare overall survival in patients treated with lenvatinib vs. sorafenib as a first-line treatment for unresectable hepatocellular carcinoma.  Methods: This was an open-label, phase 3, multicentre, non-inferiority trial that recruited patients with unresectable hepatocellular carcinoma, who had not received treatment for advanced disease, at 154 sites in 20 countries throughout the Asia-Pacific, European, and North American regions.  Patients were randomly assigned (1:1) via an interactive voice-web response system-with region; macroscopic portal vein invasion, extrahepatic spread, or both; Eastern Cooperative Oncol. Group performance status; and bodyweight as stratification factors-to receive oral lenvatinib (12 mg/day for bodyweight ≥60 kg or 8 mg/day for bodyweight <60 kg) or sorafenib 400 mg twice-daily in 28-day cycles.  The primary endpoint was overall survival, measured from the date of randomization until the date of death from any cause.  The efficacy anal. followed the intention-to-treat principle, and only patients who received treatment were included in the safety anal.  The non-inferiority margin was set at 1·08.  The trial is registered with ClinicalTrials.gov, no. NCT01761266.  Findings: Between March 1, 2013 and July 30, 2015, 1492 patients were recruited. 954 eligible patients were randomly assigned to lenvatinib (n=478) or sorafenib (n=476).  Median survival time for lenvatinib of 13·6 mo (95% CI 12·1-14·9) was non-inferior to sorafenib (12·3 mo, 10·4-13·9; hazard ratio 0·92, 95% CI 0·79-1·06), meeting criteria for non-inferiority.  The most common any-grade adverse events were hypertension (201 [42%]), diarrhea (184 [39%]), decreased appetite (162 [34%]), and decreased wt. (147 [31%]) for lenvatinib, and palmar-plantar erythrodysaesthesia (249 [52%]), diarrhea (220 [46%]), hypertension (144 [30%]), and decreased appetite (127 [27%]) for sorafenib.  Interoretation: Lenvatinib was non-inferior to sorafenib in overall survival in untreated advanced hepatocellular carcinoma.  The safety and tolerability profiles of lenvatinib were consistent with those previously obsd.Eisai Inc.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpK8z3Hv8KQH7Vg90H21EOLACvtfcHk0lg3X5742I0sQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisFOjtr4%253D&md5=96f10bec400e857fe6d6d6504bc22ac9</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2FS0140-6736%2818%2930207-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0140-6736%252818%252930207-1%26sid%3Dliteratum%253Aachs%26aulast%3DKudo%26aufirst%3DM.%26aulast%3DFinn%26aufirst%3DR.%2BS.%26aulast%3DQin%26aufirst%3DS.%26aulast%3DHan%26aufirst%3DK.%2BH.%26aulast%3DIkeda%26aufirst%3DK.%26aulast%3DPiscaglia%26aufirst%3DF.%26aulast%3DBaron%26aufirst%3DA.%26aulast%3DPark%26aufirst%3DJ.%2BW.%26aulast%3DHan%26aufirst%3DG.%26aulast%3DJassem%26aufirst%3DJ.%26aulast%3DBlanc%26aufirst%3DJ.%2BF.%26aulast%3DVogel%26aufirst%3DA.%26aulast%3DKomov%26aufirst%3DD.%26aulast%3DEvans%26aufirst%3DT.%2BR.%2BJ.%26aulast%3DLopez%26aufirst%3DC.%26aulast%3DDutcus%26aufirst%3DC.%26aulast%3DGuo%26aufirst%3DM.%26aulast%3DSaito%26aufirst%3DK.%26aulast%3DKraljevic%26aufirst%3DS.%26aulast%3DTamai%26aufirst%3DT.%26aulast%3DRen%26aufirst%3DM.%26aulast%3DCheng%26aufirst%3DA.%2BL.%26atitle%3DLenvatinib%2520versus%2520sorafenib%2520in%2520first-line%2520treatment%2520of%2520patients%2520with%2520unresectable%2520hepatocellular%2520carcinoma%253A%2520a%2520randomised%2520phase%25203%2520non-inferiority%2520trial%26jtitle%3DLancet%26date%3D2018%26volume%3D391%26spage%3D1163%26epage%3D1173%26doi%3D10.1016%2FS0140-6736%2818%2930207-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dawkins, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Webster, R. M.</span></span> <span> </span><span class="NLM_article-title">The hepatocellular carcinoma market</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">13</span>– <span class="NLM_lpage">14</span>, <span class="refDoi"> DOI: 10.1038/nrd.2018.146</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1038%2Fnrd.2018.146" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=30168534" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhs1altr%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=13-14&author=J.+Dawkinsauthor=R.+M.+Webster&title=The+hepatocellular+carcinoma+market&doi=10.1038%2Fnrd.2018.146"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">The hepatocellular carcinoma market</span></div><div class="casAuthors">Dawkins, Joshua; Webster, Rachel M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">13-14</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">The market for hepatocellular carcinoma is set to change with the recent approvals of the kinase inhibitor lenvatinib and the immune checkpoint inhibitor nivolumab, as well as the anticipated approvals of further immunotherapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8qfEQT4N-2LVg90H21EOLACvtfcHk0lh3K8IO8sIOqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhs1altr%252FP&md5=3dccfe676500b0395ef4a9f631ef2d9b</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2018.146&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2018.146%26sid%3Dliteratum%253Aachs%26aulast%3DDawkins%26aufirst%3DJ.%26aulast%3DWebster%26aufirst%3DR.%2BM.%26atitle%3DThe%2520hepatocellular%2520carcinoma%2520market%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2019%26volume%3D18%26spage%3D13%26epage%3D14%26doi%3D10.1038%2Fnrd.2018.146" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ji, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Budhu, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, Z. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forgues, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roessler, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ambs, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meltzer, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Croce, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, L. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Man, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lo, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ng, I. O. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, Z. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X. W.</span></span> <span> </span><span class="NLM_article-title">MicroRNA expression, survival, and response to interferon in liver cancer</span>. <i>N. Engl. J. Med.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>361</i></span>,  <span class="NLM_fpage">1437</span>– <span class="NLM_lpage">1447</span>, <span class="refDoi"> DOI: 10.1056/NEJMoa0901282</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1056%2FNEJMoa0901282" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=19812400" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht1GiurnE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=361&publication_year=2009&pages=1437-1447&author=J.+F.+Jiauthor=J.+Shiauthor=A.+Budhuauthor=Z.+P.+Yuauthor=M.+Forguesauthor=S.+Roesslerauthor=S.+Ambsauthor=Y.+D.+Chenauthor=P.+S.+Meltzerauthor=C.+M.+Croceauthor=L.+X.+Qinauthor=K.+Manauthor=C.+M.+Loauthor=J.+Leeauthor=I.+O.+L.+Ngauthor=J.+Fanauthor=Z.+Y.+Tangauthor=H.+C.+Sunauthor=X.+W.+Wang&title=MicroRNA+expression%2C+survival%2C+and+response+to+interferon+in+liver+cancer&doi=10.1056%2FNEJMoa0901282"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">MicroRNA expression, survival, and response to interferon in liver cancer</span></div><div class="casAuthors">Ji, Junfang; Shi, Jiong; Budhu, Anuradha; Yu, Zhipeng; Forgues, Marshonna; Roessler, Stephanie; Ambs, Stefan; Chen, Yidong; Meltzer, Paul S.; Croce, Carlo M.; Qin, Lun-Xiu; Man, Kwan; Lo, Chung-Mau; Lee, Joyce; Ng, Irene O. L.; Fan, Jia; Tang, Zhao-You; Sun, Hui-Chuan; Wang, Xin Wei</div><div class="citationInfo"><span class="NLM_cas:title">New England Journal of Medicine</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">361</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">1437-1447</span>CODEN:
                <span class="NLM_cas:coden">NEJMAG</span>;
        ISSN:<span class="NLM_cas:issn">0028-4793</span>.
    
            (<span class="NLM_cas:orgname">Massachusetts Medical Society</span>)
        </div><div class="casAbstract">BACKGROUND. Hepatocellular carcinoma is a common and aggressive cancer that occurs mainly in men.  We examd. microRNA expression patterns, survival, and response to interferon alfa in both men and women with the disease.  METHODS. We analyzed three independent cohorts that included a total of 455 patients with hepatocellular carcinoma who had undergone radical tumor resection between 1999 and 2003.  MicroRNA-expression profiling was performed in a cohort of 241 patients with hepatocellular carcinoma to identify tumor-related microRNAs and det. their assocn. with survival in men and women.  In addn., to validate our findings, we used quant. reverse-transcriptase-polymerase-chain-reaction assays to measure microRNAs and assess their assocn. with survival and response to therapy with interferon alfa in 214 patients from two independent, prospective, randomized, controlled trials of adjuvant interferon therapy.  RESULTS. In patients with hepatocellular carcinoma, the expression of miR-26a and miR-26b in nontumor liver tissue was higher in women than in men.  Tumors had reduced levels of miR-26 expression, as compared with paired noncancerous tissues, which indicated that the level of miR-26 expression was also assocd. with hepatocellular carcinoma.  Moreover, tumors with reduced miR-26 expression had a distinct transcriptomic pattern, and analyses of gene networks revealed that activation of signaling pathways between nuclear factor κB and interleukin-6 might play a role in tumor development.  Patients whose tumors had low miR-26 expression had shorter overall survival but a better response to interferon therapy than did patients whose tumors had high expression of the microRNA.  CONCLUSIONS. The expression patterns of microRNAs in liver tissue differ between men and women with hepatocellular carcinoma.  The miR-26 expression status of such patients is assocd. with survival and response to adjuvant therapy with interferon alfa.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpV1JB_Owz1cbVg90H21EOLACvtfcHk0lh3K8IO8sIOqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXht1GiurnE&md5=ae9cf6d4f975d121eef2d7104775cc49</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1056%2FNEJMoa0901282&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMoa0901282%26sid%3Dliteratum%253Aachs%26aulast%3DJi%26aufirst%3DJ.%2BF.%26aulast%3DShi%26aufirst%3DJ.%26aulast%3DBudhu%26aufirst%3DA.%26aulast%3DYu%26aufirst%3DZ.%2BP.%26aulast%3DForgues%26aufirst%3DM.%26aulast%3DRoessler%26aufirst%3DS.%26aulast%3DAmbs%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DY.%2BD.%26aulast%3DMeltzer%26aufirst%3DP.%2BS.%26aulast%3DCroce%26aufirst%3DC.%2BM.%26aulast%3DQin%26aufirst%3DL.%2BX.%26aulast%3DMan%26aufirst%3DK.%26aulast%3DLo%26aufirst%3DC.%2BM.%26aulast%3DLee%26aufirst%3DJ.%26aulast%3DNg%26aufirst%3DI.%2BO.%2BL.%26aulast%3DFan%26aufirst%3DJ.%26aulast%3DTang%26aufirst%3DZ.%2BY.%26aulast%3DSun%26aufirst%3DH.%2BC.%26aulast%3DWang%26aufirst%3DX.%2BW.%26atitle%3DMicroRNA%2520expression%252C%2520survival%252C%2520and%2520response%2520to%2520interferon%2520in%2520liver%2520cancer%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2009%26volume%3D361%26spage%3D1437%26epage%3D1447%26doi%3D10.1056%2FNEJMoa0901282" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Klingenberg, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsuda, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diederichs, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, T.</span></span> <span> </span><span class="NLM_article-title">Non-coding RNA in hepatocellular carcinoma: Mechanisms, biomarkers and therapeutic targets</span>. <i>J. Hepatol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">603</span>– <span class="NLM_lpage">618</span>, <span class="refDoi"> DOI: 10.1016/j.jhep.2017.04.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1016%2Fj.jhep.2017.04.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=28438689" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BC2sXotlWqsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2017&pages=603-618&author=M.+Klingenbergauthor=A.+Matsudaauthor=S.+Diederichsauthor=T.+Patel&title=Non-coding+RNA+in+hepatocellular+carcinoma%3A+Mechanisms%2C+biomarkers+and+therapeutic+targets&doi=10.1016%2Fj.jhep.2017.04.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Non-coding RNA in hepatocellular carcinoma: Mechanisms, biomarkers and therapeutic targets</span></div><div class="casAuthors">Klingenberg, Marcel; Matsuda, Akiko; Diederichs, Sven; Patel, Tushar</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Hepatology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">603-618</span>CODEN:
                <span class="NLM_cas:coden">JOHEEC</span>;
        ISSN:<span class="NLM_cas:issn">0168-8278</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The majority of the human genome is not translated into proteins but can be transcribed into RNA.  Even though the resulting non-coding RNAs (ncRNAs) do not encode for proteins, they contribute to diseases such as cancer.  Here, we review examples of the functions of ncRNAs in liver cancer and their potential use for the detection and treatment of liver cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJTPs3pykqf7Vg90H21EOLACvtfcHk0lh3K8IO8sIOqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXotlWqsrY%253D&md5=31a039f1450fbf28743012fbe71c08db</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.jhep.2017.04.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jhep.2017.04.009%26sid%3Dliteratum%253Aachs%26aulast%3DKlingenberg%26aufirst%3DM.%26aulast%3DMatsuda%26aufirst%3DA.%26aulast%3DDiederichs%26aufirst%3DS.%26aulast%3DPatel%26aufirst%3DT.%26atitle%3DNon-coding%2520RNA%2520in%2520hepatocellular%2520carcinoma%253A%2520Mechanisms%252C%2520biomarkers%2520and%2520therapeutic%2520targets%26jtitle%3DJ.%2520Hepatol.%26date%3D2017%26volume%3D67%26spage%3D603%26epage%3D618%26doi%3D10.1016%2Fj.jhep.2017.04.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kota, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chivukula, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Donnell, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wentzel, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montgomery, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hwang, H. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vivekanandan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Torbenson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mendell, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mendell, J. T.</span></span> <span> </span><span class="NLM_article-title">Therapeutic microRNA delivery suppresses tumorigenesis in a murine liver cancer model</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>137</i></span>,  <span class="NLM_fpage">1005</span>– <span class="NLM_lpage">1017</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2009.04.021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1016%2Fj.cell.2009.04.021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=19524505" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BD1MXps1eiurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=137&publication_year=2009&pages=1005-1017&author=J.+Kotaauthor=R.+R.+Chivukulaauthor=K.+A.+O%E2%80%99Donnellauthor=E.+A.+Wentzelauthor=C.+L.+Montgomeryauthor=H.+W.+Hwangauthor=T.+C.+Changauthor=P.+Vivekanandanauthor=M.+Torbensonauthor=K.+R.+Clarkauthor=J.+R.+Mendellauthor=J.+T.+Mendell&title=Therapeutic+microRNA+delivery+suppresses+tumorigenesis+in+a+murine+liver+cancer+model&doi=10.1016%2Fj.cell.2009.04.021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Therapeutic microRNA delivery suppresses tumorigenesis in a murine liver cancer model</span></div><div class="casAuthors">Kota, Janaiah; Chivukula, Raghu R.; O'Donnell, Kathryn A.; Wentzel, Erik A.; Montgomery, Chrystal L.; Hwang, Hun-Way; Chang, Tsung-Cheng; Vivekanandan, Perumal; Torbenson, Michael; Clark, K. Reed; Mendell, Jerry R.; Mendell, Joshua T.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">137</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1005-1017</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Therapeutic strategies based on modulation of microRNA (miRNA) activity hold great promise due to the ability of these small RNAs to potently influence cellular behavior.  In this study, we investigated the efficacy of a miRNA replacement therapy for liver cancer.  We demonstrate that hepatocellular carcinoma (HCC) cells exhibit reduced expression of miR-26a, a miRNA that is normally expressed at high levels in diverse tissues.  Expression of this miRNA in liver cancer cells in vitro induces cell-cycle arrest assocd. with direct targeting of cyclins D2 and E2.  Systemic administration of this miRNA in a mouse model of HCC using adeno-assocd. virus (AAV) results in inhibition of cancer cell proliferation, induction of tumor-specific apoptosis, and dramatic protection from disease progression without toxicity.  These findings suggest that delivery of miRNAs that are highly expressed and therefore tolerated in normal tissues but lost in disease cells may provide a general strategy for miRNA replacement therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhf39hbMXufLVg90H21EOLACvtfcHk0lguV6yXGUTDwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXps1eiurg%253D&md5=a70e4b5eb29d926b03861a5613cdc3e4</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2009.04.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2009.04.021%26sid%3Dliteratum%253Aachs%26aulast%3DKota%26aufirst%3DJ.%26aulast%3DChivukula%26aufirst%3DR.%2BR.%26aulast%3DO%25E2%2580%2599Donnell%26aufirst%3DK.%2BA.%26aulast%3DWentzel%26aufirst%3DE.%2BA.%26aulast%3DMontgomery%26aufirst%3DC.%2BL.%26aulast%3DHwang%26aufirst%3DH.%2BW.%26aulast%3DChang%26aufirst%3DT.%2BC.%26aulast%3DVivekanandan%26aufirst%3DP.%26aulast%3DTorbenson%26aufirst%3DM.%26aulast%3DClark%26aufirst%3DK.%2BR.%26aulast%3DMendell%26aufirst%3DJ.%2BR.%26aulast%3DMendell%26aufirst%3DJ.%2BT.%26atitle%3DTherapeutic%2520microRNA%2520delivery%2520suppresses%2520tumorigenesis%2520in%2520a%2520murine%2520liver%2520cancer%2520model%26jtitle%3DCell%26date%3D2009%26volume%3D137%26spage%3D1005%26epage%3D1017%26doi%3D10.1016%2Fj.cell.2009.04.021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sehgal, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vaishnaw, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fitzgerald, K.</span></span> <span> </span><span class="NLM_article-title">Liver as a target for oligonucleotide therapeutics</span>. <i>J. Hepatol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">1354</span>– <span class="NLM_lpage">1359</span>, <span class="refDoi"> DOI: 10.1016/j.jhep.2013.05.045</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1016%2Fj.jhep.2013.05.045" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=23770039" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVylt7%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2013&pages=1354-1359&author=A.+Sehgalauthor=A.+Vaishnawauthor=K.+Fitzgerald&title=Liver+as+a+target+for+oligonucleotide+therapeutics&doi=10.1016%2Fj.jhep.2013.05.045"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Liver as a target for oligonucleotide therapeutics</span></div><div class="casAuthors">Sehgal, Alfica; Vaishnaw, Akshay; Fitzgerald, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Hepatology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1354-1359</span>CODEN:
                <span class="NLM_cas:coden">JOHEEC</span>;
        ISSN:<span class="NLM_cas:issn">0168-8278</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">This article focused on liver-targeted for oligonucleotide therapeutics that have shown promise at the preclin. and/or clin. level.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPIwSCPukrEbVg90H21EOLACvtfcHk0lguV6yXGUTDwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVylt7%252FN&md5=ac42e6bb3652d09900cdecd06470f554</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.jhep.2013.05.045&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jhep.2013.05.045%26sid%3Dliteratum%253Aachs%26aulast%3DSehgal%26aufirst%3DA.%26aulast%3DVaishnaw%26aufirst%3DA.%26aulast%3DFitzgerald%26aufirst%3DK.%26atitle%3DLiver%2520as%2520a%2520target%2520for%2520oligonucleotide%2520therapeutics%26jtitle%3DJ.%2520Hepatol.%26date%3D2013%26volume%3D59%26spage%3D1354%26epage%3D1359%26doi%3D10.1016%2Fj.jhep.2013.05.045" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Manjunath, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Subramanya, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shankar, P.</span></span> <span> </span><span class="NLM_article-title">Lentiviral delivery of short hairpin RNAs</span>. <i>Adv. Drug Delivery Rev.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">732</span>– <span class="NLM_lpage">745</span>, <span class="refDoi"> DOI: 10.1016/j.addr.2009.03.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1016%2Fj.addr.2009.03.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=19341774" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BD1MXmsFyns7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2009&pages=732-745&author=N.+Manjunathauthor=H.+Wuauthor=S.+Subramanyaauthor=P.+Shankar&title=Lentiviral+delivery+of+short+hairpin+RNAs&doi=10.1016%2Fj.addr.2009.03.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Lentiviral delivery of short hairpin RNAs</span></div><div class="casAuthors">Manjunath, N.; Wu, Haoquan; Subramanya, Sandesh; Shankar, Premlata</div><div class="citationInfo"><span class="NLM_cas:title">Advanced Drug Delivery Reviews</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">732-745</span>CODEN:
                <span class="NLM_cas:coden">ADDREP</span>;
        ISSN:<span class="NLM_cas:issn">0169-409X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  In less than a decade after discovery, RNA interference-mediated gene silencing is already being tested as potential therapy in clin. trials for a no. of diseases.  Lentiviral vectors provide a means to express short hairpin RNA (shRNA) to induce stable and long-term gene silencing in both dividing and non-dividing cells and thus, are being intensively investigated for this purpose.  However, induction of long-term shRNA expression can also cause toxicities by inducing off-target effects and interference with the endogenous micro-RNA (miRNA) pathway that regulates cellular gene expression.  Recently, several advances have been made in the shRNA vector design to mimic cellular miRNA processing and to express multiplex siRNAs in a tightly regulated and reversible manner to overcome toxicities.  In this review we describe some of these advances, focusing on the progress made in the development of lentiviral shRNA delivery strategies to combat viral infections.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0x5xsKZljBbVg90H21EOLACvtfcHk0lguV6yXGUTDwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXmsFyns7c%253D&md5=5c1bddec6d6a9377edc11978fc6616fc</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.addr.2009.03.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.addr.2009.03.004%26sid%3Dliteratum%253Aachs%26aulast%3DManjunath%26aufirst%3DN.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DSubramanya%26aufirst%3DS.%26aulast%3DShankar%26aufirst%3DP.%26atitle%3DLentiviral%2520delivery%2520of%2520short%2520hairpin%2520RNAs%26jtitle%3DAdv.%2520Drug%2520Delivery%2520Rev.%26date%3D2009%26volume%3D61%26spage%3D732%26epage%3D745%26doi%3D10.1016%2Fj.addr.2009.03.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Cancer
Genome Atlas Research Network.</span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ally, A.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Comprehensive and integrative genomic characterization of hepatocellular carcinoma</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>169</i></span>,  <span class="NLM_fpage">1327</span>– <span class="NLM_lpage">1341.e23</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2017.05.046</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1016%2Fj.cell.2017.05.046" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=28622513" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=169&publication_year=2017&pages=1327-1341.e23&author=Cancer%0AGenome+Atlas+Research+Network.author=A.+Ally&title=Comprehensive+and+integrative+genomic+characterization+of+hepatocellular+carcinoma&doi=10.1016%2Fj.cell.2017.05.046"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2017.05.046&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2017.05.046%26sid%3Dliteratum%253Aachs%26aulast%3D%26aulast%3DAlly%26aufirst%3DA.%26atitle%3DComprehensive%2520and%2520integrative%2520genomic%2520characterization%2520of%2520hepatocellular%2520carcinoma%26jtitle%3DCell%26date%3D2017%26volume%3D169%26spage%3D1327%26epage%3D1341.e23%26doi%3D10.1016%2Fj.cell.2017.05.046" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Melo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villanueva, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moutinho, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davalos, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spizzo, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ivan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Setien, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casanovas, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simo-Riudalbas, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carmona, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carrere, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vidal, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aytes, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Puertas, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ropero, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalluri, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Croce, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calin, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Esteller, M.</span></span> <span> </span><span class="NLM_article-title">Small molecule enoxacin is a cancer-specific growth inhibitor that acts by enhancing TAR RNA-binding protein 2-mediated microRNA processing</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>108</i></span>,  <span class="NLM_fpage">4394</span>– <span class="NLM_lpage">4399</span>, <span class="refDoi"> DOI: 10.1073/pnas.1014720108</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1073%2Fpnas.1014720108" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=21368194" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BC3MXjvV2jsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=108&publication_year=2011&pages=4394-4399&author=S.+Meloauthor=A.+Villanuevaauthor=C.+Moutinhoauthor=V.+Davalosauthor=R.+Spizzoauthor=C.+Ivanauthor=S.+Rossiauthor=F.+Setienauthor=O.+Casanovasauthor=L.+Simo-Riudalbasauthor=J.+Carmonaauthor=J.+Carrereauthor=A.+Vidalauthor=A.+Aytesauthor=S.+Puertasauthor=S.+Roperoauthor=R.+Kalluriauthor=C.+M.+Croceauthor=G.+A.+Calinauthor=M.+Esteller&title=Small+molecule+enoxacin+is+a+cancer-specific+growth+inhibitor+that+acts+by+enhancing+TAR+RNA-binding+protein+2-mediated+microRNA+processing&doi=10.1073%2Fpnas.1014720108"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Small molecule enoxacin is a cancer-specific growth inhibitor that acts by enhancing TAR RNA-binding protein 2-mediated microRNA processing</span></div><div class="casAuthors">Melo, Sonia; Villaneuva, Alberto; Moutinho, Catia; Davalos, Veronica; Spizzo, Riccardo; Ivan, Cristina; Rossi, Simona; Setien, Fernando; Casanovas, Oriol; Simo-Riudalbas, Laia; Carmona, Javier; Carrere, Jordi; Vidal, August; Aytes, Alvaro; Puertas, Sara; Ropero, Santiago; Kalluri, Raghu; Croce, Carlo M.; Calin, George A.; Esteller, Manel</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">108</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4394-4399, S4394/1-S4394/10</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">MicroRNAs (miRNAs) are small RNA mols. that regulate gene expression at the posttranscriptional level and are crit. for many cellular pathways.  The disruption of miRNAs and their processing machineries also contributes to the development of human tumors.  A common scenario for miRNA expression in carcinogenesis is emerging that shows that impaired miRNA prodn. and/or down-regulation of these transcripts occurs in many neoplasms.  Several of these lost miRNAs have tumor-suppressor features; so strategies to restore their expression globally in malignancies would be a welcome addn. to the current therapeutic arsenal against cancer.  Herein, we show that the small mol. enoxacin, a fluoroquinolone used as an antibacterial compd., enhances the prodn. of miRNAs with tumor suppressor functions by binding to the miRNA biosynthesis protein TAR RNA-binding protein 2 (TRBP).  The use of enoxacin in human cell cultures and xenografted, orthotopic, and metastatic mouse models reveals a TRBP- dependent and cancer-specific growth-inhibitory effect of the drug.  These results highlight the key role of disrupted miRNA expression patterns in tumorigenesis, and suggest a unique strategy for restoring the distorted microRNAome of cancer cells to a more physiol. setting.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr87Y1kq9LPYrVg90H21EOLACvtfcHk0li8K5MFqaBtaw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXjvV2jsLk%253D&md5=bef55782f8265ad8676d5b627cbc8523</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1014720108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1014720108%26sid%3Dliteratum%253Aachs%26aulast%3DMelo%26aufirst%3DS.%26aulast%3DVillanueva%26aufirst%3DA.%26aulast%3DMoutinho%26aufirst%3DC.%26aulast%3DDavalos%26aufirst%3DV.%26aulast%3DSpizzo%26aufirst%3DR.%26aulast%3DIvan%26aufirst%3DC.%26aulast%3DRossi%26aufirst%3DS.%26aulast%3DSetien%26aufirst%3DF.%26aulast%3DCasanovas%26aufirst%3DO.%26aulast%3DSimo-Riudalbas%26aufirst%3DL.%26aulast%3DCarmona%26aufirst%3DJ.%26aulast%3DCarrere%26aufirst%3DJ.%26aulast%3DVidal%26aufirst%3DA.%26aulast%3DAytes%26aufirst%3DA.%26aulast%3DPuertas%26aufirst%3DS.%26aulast%3DRopero%26aufirst%3DS.%26aulast%3DKalluri%26aufirst%3DR.%26aulast%3DCroce%26aufirst%3DC.%2BM.%26aulast%3DCalin%26aufirst%3DG.%2BA.%26aulast%3DEsteller%26aufirst%3DM.%26atitle%3DSmall%2520molecule%2520enoxacin%2520is%2520a%2520cancer-specific%2520growth%2520inhibitor%2520that%2520acts%2520by%2520enhancing%2520TAR%2520RNA-binding%2520protein%25202-mediated%2520microRNA%2520processing%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2011%26volume%3D108%26spage%3D4394%26epage%3D4399%26doi%3D10.1073%2Fpnas.1014720108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fareh, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeom, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haagsma, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chauhan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heo, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joo, C.</span></span> <span> </span><span class="NLM_article-title">TRBP ensures efficient Dicer processing of precursor microRNA in RNA-crowded environments</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">13694</span>, <span class="refDoi"> DOI: 10.1038/ncomms13694</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1038%2Fncomms13694" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=27934859" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BC28XitFamsLfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=13694&author=M.+Farehauthor=K.+H.+Yeomauthor=A.+C.+Haagsmaauthor=S.+Chauhanauthor=I.+Heoauthor=C.+Joo&title=TRBP+ensures+efficient+Dicer+processing+of+precursor+microRNA+in+RNA-crowded+environments&doi=10.1038%2Fncomms13694"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">TRBP ensures efficient Dicer processing of precursor microRNA in RNA-crowded environments</span></div><div class="casAuthors">Fareh, Mohamed; Yeom, Kyu-Hyeon; Haagsma, Anna C.; Chauhan, Sweeny; Heo, Inha; Joo, Chirlmin</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">13694</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The RNA-binding protein TRBP is a central component of the Dicer complex.  Despite a decade of biochem. and structural studies, the essential functionality of TRBP in microRNA (miRNA) biogenesis remains unknown.  Here we show that TRBP is an integral cofactor for time-efficient Dicer processing in RNA-crowded environments.  We competed for Dicer processing of pre-miRNA with a large amt. of cellular RNA species and found that Dicer-TRBP, but not Dicer alone, remains resilient.  To apprehend the mechanism of this substrate selectivity, we use single-mol. fluorescence.  The real-time observation reveals that TRBP acts as a gatekeeper, precluding Dicer from engaging with pre-miRNA-like substrates.  TRBP acquires the selectivity using the PAZ domain of Dicer, whereas Dicer moderates the RNA-binding affinity of TRBP for fast turnover.  This coordinated action between TRBP and Dicer accomplishes an efficient way of discarding pre-miRNA-like substrates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoe2wKT09Q-zbVg90H21EOLACvtfcHk0li8K5MFqaBtaw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitFamsLfN&md5=31ab2d8222bfe62ce80680aff52d5d99</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fncomms13694&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms13694%26sid%3Dliteratum%253Aachs%26aulast%3DFareh%26aufirst%3DM.%26aulast%3DYeom%26aufirst%3DK.%2BH.%26aulast%3DHaagsma%26aufirst%3DA.%2BC.%26aulast%3DChauhan%26aufirst%3DS.%26aulast%3DHeo%26aufirst%3DI.%26aulast%3DJoo%26aufirst%3DC.%26atitle%3DTRBP%2520ensures%2520efficient%2520Dicer%2520processing%2520of%2520precursor%2520microRNA%2520in%2520RNA-crowded%2520environments%26jtitle%3DNat.%2520Commun.%26date%3D2016%26volume%3D7%26spage%3D13694%26doi%3D10.1038%2Fncomms13694" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chendrimada, T. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gregory, R. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumaraswamy, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norman, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooch, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nishikura, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shiekhattar, R.</span></span> <span> </span><span class="NLM_article-title">TRBP recruits the Dicer complex to Ago2 for microRNA processing and gene silencing</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>436</i></span>,  <span class="NLM_fpage">740</span>– <span class="NLM_lpage">744</span>, <span class="refDoi"> DOI: 10.1038/nature03868</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1038%2Fnature03868" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=15973356" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BD2MXmvFenur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=436&publication_year=2005&pages=740-744&author=T.+P.+Chendrimadaauthor=R.+I.+Gregoryauthor=E.+Kumaraswamyauthor=J.+Normanauthor=N.+Coochauthor=K.+Nishikuraauthor=R.+Shiekhattar&title=TRBP+recruits+the+Dicer+complex+to+Ago2+for+microRNA+processing+and+gene+silencing&doi=10.1038%2Fnature03868"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">TRBP recruits the Dicer complex to Ago2 for microRNA processing and gene silencing</span></div><div class="casAuthors">Chendrimada, Thimmaiah P.; Gregory, Richard I.; Kumaraswamy, Easwari; Norman, Jessica; Cooch, Neil; Nishikura, Kazuko; Shiekhattar, Ramin</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">436</span>
        (<span class="NLM_cas:issue">7051</span>),
    <span class="NLM_cas:pages">740-744</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">MicroRNAs (miRNAs) are generated by a two-step processing pathway to yield RNA mols. of approx. 22 nucleotides that neg. regulate target gene expression at the post-transcriptional level.  Primary miRNAs are processed to precursor miRNAs (pre-miRNAs) by the Microprocessor complex.  These pre-miRNAs are cleaved by the RNase III Dicer to generate mature miRNAs that direct the RNA-induced silencing complex (RISC) to mRNAs with complementary sequence.  Here we show that TRBP (the human immunodeficiency virus transactivating response RNA-binding protein), which contains three double-stranded, RNA-binding domains, is an integral component of a Dicer-contg. complex.  Biochem. anal. of TRBP-contg. complexes revealed the assocn. of Dicer-TRBP with Argonaute 2 (Ago2), the catalytic engine of RISC.  The phys. assocn. of Dicer-TRBP and Ago2 was confirmed after the isolation of the ternary complex using Flag-tagged Ago2 cell lines.  In vitro reconstitution assays demonstrated that TRBP is required for the recruitment of Ago2 to the small interfering RNA (siRNA) bound by Dicer.  Knockdown of TRBP results in destabilization of Dicer and a consequent loss of miRNA biogenesis.  Finally, depletion of the Dicer-TRBP complex via exogenously introduced siRNAs diminished RISC-mediated reporter gene silencing.  These results support a role of the Dicer-TRBP complex not only in miRNA processing but also as a platform for RISC assembly.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2woUlrto1u7Vg90H21EOLACvtfcHk0liqLcfXdpK0BQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXmvFenur0%253D&md5=18ce99ba3ca840a2f63d42ff21c811cb</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2Fnature03868&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature03868%26sid%3Dliteratum%253Aachs%26aulast%3DChendrimada%26aufirst%3DT.%2BP.%26aulast%3DGregory%26aufirst%3DR.%2BI.%26aulast%3DKumaraswamy%26aufirst%3DE.%26aulast%3DNorman%26aufirst%3DJ.%26aulast%3DCooch%26aufirst%3DN.%26aulast%3DNishikura%26aufirst%3DK.%26aulast%3DShiekhattar%26aufirst%3DR.%26atitle%3DTRBP%2520recruits%2520the%2520Dicer%2520complex%2520to%2520Ago2%2520for%2520microRNA%2520processing%2520and%2520gene%2520silencing%26jtitle%3DNature%26date%3D2005%26volume%3D436%26spage%3D740%26epage%3D744%26doi%3D10.1038%2Fnature03868" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winter, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diederichs, S.</span></span> <span> </span><span class="NLM_article-title">Argonaute proteins regulate microRNA stability: Increased microRNA abundance by Argonaute proteins is due to microRNA stabilization</span>. <i>RNA Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1149</span>– <span class="NLM_lpage">1157</span>, <span class="refDoi"> DOI: 10.4161/rna.8.6.17665</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.4161%2Frna.8.6.17665" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=21941127" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BC38XkslCjtL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2011&pages=1149-1157&author=J.+Winterauthor=S.+Diederichs&title=Argonaute+proteins+regulate+microRNA+stability%3A+Increased+microRNA+abundance+by+Argonaute+proteins+is+due+to+microRNA+stabilization&doi=10.4161%2Frna.8.6.17665"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Argonaute proteins regulate microRNA stability</span></div><div class="casAuthors">Winter, Julia; Diederichs, Sven</div><div class="citationInfo"><span class="NLM_cas:title">RNA Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1149-1157</span>CODEN:
                <span class="NLM_cas:coden">RBNIBE</span>;
        ISSN:<span class="NLM_cas:issn">1547-6286</span>.
    
            (<span class="NLM_cas:orgname">Landes Bioscience</span>)
        </div><div class="casAbstract">Argonaute proteins are key players in microRNA (miRNA) processing and function.  Next to their role as RISC effector proteins mediating target silencing, they actively participate in miRNA biogenesis and increase miRNA abundance by a yet unidentified mechanism.  It has been hypothesized that this increase in steady-state miRNA levels might be due to a stabilizing effect of Argonaute proteins, but this has not been analyzed so far due to a lack of test systems.  Here, we provide two approaches to est. miRNA stability and factors affecting it: in cells lacking Ago2, endogenous miRNA guide strand degrdn. and half-lives can be assessed using actinomycin D or α-amanitin.  In turn, miRNA passenger strands can serve as a model in wildtype cells to ascertain the impact of miRNA stability factors.  We provide evidence that Argonaute proteins stabilize mature miRNAs in a slicing-independent manner.  Transcriptional inhibition reveals reduced half-lives of multiple endogenous miRNA guide strands in cells lacking Ago2.  This effect is reversible upon the reconstitution of Argonaute expression.  Correspondingly, overexpression of Argonaute proteins decelerates miRNA degrdn. and increases miRNA half-life.  Taken together, this study employs two model systems to identify factors altering miRNA stability and provides evidence how Argonaute proteins post-transcriptionally elevate mature miRNA levels via increasing miRNA stability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpchZzDOzAubrVg90H21EOLACvtfcHk0liqLcfXdpK0BQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XkslCjtL0%253D&md5=a46f2eb9ed471256ce101ad4ff77fcec</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.4161%2Frna.8.6.17665&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252Frna.8.6.17665%26sid%3Dliteratum%253Aachs%26aulast%3DWinter%26aufirst%3DJ.%26aulast%3DDiederichs%26aufirst%3DS.%26atitle%3DArgonaute%2520proteins%2520regulate%2520microRNA%2520stability%253A%2520Increased%2520microRNA%2520abundance%2520by%2520Argonaute%2520proteins%2520is%2520due%2520to%2520microRNA%2520stabilization%26jtitle%3DRNA%2520Biol.%26date%3D2011%26volume%3D8%26spage%3D1149%26epage%3D1157%26doi%3D10.4161%2Frna.8.6.17665" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Warner, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bridge, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hewitson, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hodgkinson, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heyam, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Massa, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haslam, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chatzifrangkeskou, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plevin, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharp, T. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lagos, D.</span></span> <span> </span><span class="NLM_article-title">S6K2-mediated regulation of TRBP as a determinant of miRNA expression in human primary lymphatic endothelial cells</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>44</i></span>,  <span class="NLM_fpage">9942</span>– <span class="NLM_lpage">9955</span>, <span class="refDoi"> DOI: 10.1093/nar/gkw631</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1093%2Fnar%2Fgkw631" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=27407113" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvVCltr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&pages=9942-9955&author=M.+J.+Warnerauthor=K.+S.+Bridgeauthor=J.+P.+Hewitsonauthor=M.+R.+Hodgkinsonauthor=A.+Heyamauthor=B.+C.+Massaauthor=J.+C.+Haslamauthor=M.+Chatzifrangkeskouauthor=G.+J.+Evansauthor=M.+J.+Plevinauthor=T.+V.+Sharpauthor=D.+Lagos&title=S6K2-mediated+regulation+of+TRBP+as+a+determinant+of+miRNA+expression+in+human+primary+lymphatic+endothelial+cells&doi=10.1093%2Fnar%2Fgkw631"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">S6K2-mediated regulation of TRBP as a determinant of miRNA expression in human primary lymphatic endothelial cells</span></div><div class="casAuthors">Warner, Matthew J.; Bridge, Katherine S.; Hewitson, James P.; Hodgkinson, Michael R.; Heyam, Alex; Massa, Bailey C.; Haslam, Jessica C.; Chatzifrangkeskou, Maria; Evans, Gareth J. O.; Plevin, Michael J.; Sharp, Tyson V.; Lagos, Dimitris</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">9942-9955</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">MicroRNAs (miRNAs) are short non-coding RNAs that silence mRNAs.  They are generated following transcription and cleavage by the DROSHA/DGCR8 and DICER/TRBP/PACT complexes.  Although it is known that components of the miRNA biogenesis machinery can be phosphorylated, it remains poorly understood how these events become engaged during physiol. cellular activation.  We demonstrate that S6 kinases can phosphorylate the extended C-terminal domain of TRBP and interact with TRBP in situ in primary cells.  TRBP serines 283/286 are essential for S6K-mediated TRBP phosphorylation, optimal expression of TRBP, and the S6K-TRBP interaction in human primary cells.  We demonstrate the functional relevance of this interaction in primary human dermal lymphatic endothelial cells (HDLECs).  Angiopoietin-1 (ANG1) can augment miRNA biogenesis in HDLECs through enhancing TRBP phosphorylation and expression in an S6K2-dependent manner.  We propose that the S6K2/TRBP node controls miRNA biogenesis in HDLECs and provides a mol. link between the mTOR pathway and the miRNA biogenesis machinery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoOqwWieJEx7rVg90H21EOLACvtfcHk0liqLcfXdpK0BQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvVCltr8%253D&md5=548bd1b1067891398081068e7891713f</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkw631&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkw631%26sid%3Dliteratum%253Aachs%26aulast%3DWarner%26aufirst%3DM.%2BJ.%26aulast%3DBridge%26aufirst%3DK.%2BS.%26aulast%3DHewitson%26aufirst%3DJ.%2BP.%26aulast%3DHodgkinson%26aufirst%3DM.%2BR.%26aulast%3DHeyam%26aufirst%3DA.%26aulast%3DMassa%26aufirst%3DB.%2BC.%26aulast%3DHaslam%26aufirst%3DJ.%2BC.%26aulast%3DChatzifrangkeskou%26aufirst%3DM.%26aulast%3DEvans%26aufirst%3DG.%2BJ.%26aulast%3DPlevin%26aufirst%3DM.%2BJ.%26aulast%3DSharp%26aufirst%3DT.%2BV.%26aulast%3DLagos%26aufirst%3DD.%26atitle%3DS6K2-mediated%2520regulation%2520of%2520TRBP%2520as%2520a%2520determinant%2520of%2520miRNA%2520expression%2520in%2520human%2520primary%2520lymphatic%2520endothelial%2520cells%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2016%26volume%3D44%26spage%3D9942%26epage%3D9955%26doi%3D10.1093%2Fnar%2Fgkw631" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Paroo, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Q.</span></span> <span> </span><span class="NLM_article-title">Phosphorylation of the human microRNA-generating complex mediates MAPK/Erk signaling</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>139</i></span>,  <span class="NLM_fpage">112</span>– <span class="NLM_lpage">122</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2009.06.044</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1016%2Fj.cell.2009.06.044" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=19804757" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsVCjs7jF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=139&publication_year=2009&pages=112-122&author=Z.+Parooauthor=X.+Yeauthor=S.+Chenauthor=Q.+Liu&title=Phosphorylation+of+the+human+microRNA-generating+complex+mediates+MAPK%2FErk+signaling&doi=10.1016%2Fj.cell.2009.06.044"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Phosphorylation of the human microRNA-generating complex mediates MAPK/Erk signaling</span></div><div class="casAuthors">Paroo, Zain; Ye, Xuecheng; Chen, She; Liu, Qinghua</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">139</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">112-122</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">MicroRNAs (miRNAs) govern an expanding no. of biol. and disease processes.  Understanding the mechanisms by which the miRNA pathway is regulated, therefore, represents an important area of investigation.  The authors detd. that the human miRNA-generating complex is comprised of Dicer and phospho-TAR RNA binding protein (TRBP) isoforms.  Phosphorylation of TRBP is mediated by the MAP kinase (MAPK) Erk1/2.  Expression of phospho-mimic TRBP and TRBP phosphorylation enhanced miRNA prodn. by increasing the stability of the miRNA-generating complex.  Mitogenic signaling in response to serum and the tumor promoter, phorbol-12-myristate-13-acetate (PMA), was dependent on TRBP phosphorylation.  These effects were accompanied by a coordinated increase in levels of growth-promoting miRNA and reduced expression of let-7 tumor suppressor miRNA.  Conversely, pharmacol. inhibition of MAPK/Erk1/2 resulted in an anti-growth miRNA profile.  Taken together, these studies indicate that the MAPK/Erk1/2 pathway regulates the miRNA machinery and suggest a general principle, wherein signaling systems target the miRNA pathway to achieve biol. responses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCm24qq1BiC7Vg90H21EOLACvtfcHk0lie4sdOwOEfcg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsVCjs7jF&md5=dd298ee5b41196f93ed166c1595fb756</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2009.06.044&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2009.06.044%26sid%3Dliteratum%253Aachs%26aulast%3DParoo%26aufirst%3DZ.%26aulast%3DYe%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DQ.%26atitle%3DPhosphorylation%2520of%2520the%2520human%2520microRNA-generating%2520complex%2520mediates%2520MAPK%252FErk%2520signaling%26jtitle%3DCell%26date%3D2009%26volume%3D139%26spage%3D112%26epage%3D122%26doi%3D10.1016%2Fj.cell.2009.06.044" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">De
Vito, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riggi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cornaz, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suva, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baumer, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Provero, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stamenkovic, I.</span></span> <span> </span><span class="NLM_article-title">A TARBP2-dependent miRNA expression profile underlies cancer stem cell properties and provides candlidate therapeutic reagents in ewing sarcoma</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">807</span>– <span class="NLM_lpage">821</span>, <span class="refDoi"> DOI: 10.1016/j.ccr.2012.04.023</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1016%2Fj.ccr.2012.04.023" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=22698405" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BC38XosVeht7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2012&pages=807-821&author=C.+De%0AVitoauthor=N.+Riggiauthor=S.+Cornazauthor=M.+L.+Suvaauthor=K.+Baumerauthor=P.+Proveroauthor=I.+Stamenkovic&title=A+TARBP2-dependent+miRNA+expression+profile+underlies+cancer+stem+cell+properties+and+provides+candlidate+therapeutic+reagents+in+ewing+sarcoma&doi=10.1016%2Fj.ccr.2012.04.023"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">A TARBP2-Dependent miRNA Expression Profile Underlies Cancer Stem Cell Properties and Provides Candidate Therapeutic Reagents in Ewing Sarcoma</span></div><div class="casAuthors">De Vito, Claudio; Riggi, Nicolo; Cornaz, Sandrine; Suva, Mario-Luca; Baumer, Karine; Provero, Paolo; Stamenkovic, Ivan</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">807-821</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">We have recently demonstrated that human pediatric mesenchymal stem cells can be reprogrammed toward a Ewing sarcoma family tumor (ESFT) cancer stem cell (CSC) phenotype by mechanisms that implicate microRNAs (miRNAs).  Here, we show that the miRNA profile of ESFT CSCs is shared by embryonic stem cells and CSCs from divergent tumor types.  We also provide evidence that the miRNA profile of ESFT CSCs is the result of reversible disruption of TARBP2-dependent miRNA maturation.  Restoration of TARBP2 activity and systemic delivery of synthetic forms of either of two of its targets, miRNA-143 or miRNA-145, inhibited ESFT CSC clonogenicity and tumor growth in vivo.  Our observations suggest that CSC self-renewal and tumor maintenance may depend on deregulation of TARBP2-dependent miRNA expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZC0RNr45QabVg90H21EOLACvtfcHk0lie4sdOwOEfcg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XosVeht7c%253D&md5=8b77858eeb1cf052a1ec9c0d39040932</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2012.04.023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2012.04.023%26sid%3Dliteratum%253Aachs%26aulast%3DDe%2BVito%26aufirst%3DC.%26aulast%3DRiggi%26aufirst%3DN.%26aulast%3DCornaz%26aufirst%3DS.%26aulast%3DSuva%26aufirst%3DM.%2BL.%26aulast%3DBaumer%26aufirst%3DK.%26aulast%3DProvero%26aufirst%3DP.%26aulast%3DStamenkovic%26aufirst%3DI.%26atitle%3DA%2520TARBP2-dependent%2520miRNA%2520expression%2520profile%2520underlies%2520cancer%2520stem%2520cell%2520properties%2520and%2520provides%2520candlidate%2520therapeutic%2520reagents%2520in%2520ewing%2520sarcoma%26jtitle%3DCancer%2520Cell%26date%3D2012%26volume%3D21%26spage%3D807%26epage%3D821%26doi%3D10.1016%2Fj.ccr.2012.04.023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span> <span> </span><span class="NLM_article-title">The role of TARBP2 in the development and progression of cancers</span>. <i>Tumor Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>37</i></span>,  <span class="NLM_fpage">57</span>– <span class="NLM_lpage">60</span>, <span class="refDoi"> DOI: 10.1007/s13277-015-4273-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1007%2Fs13277-015-4273-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslekurvE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2016&pages=57-60&author=X.+Yuauthor=Z.+Li&title=The+role+of+TARBP2+in+the+development+and+progression+of+cancers&doi=10.1007%2Fs13277-015-4273-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">The role of TARBP2 in the development and progression of cancers</span></div><div class="casAuthors">Yu, Xin; Li, Zheng</div><div class="citationInfo"><span class="NLM_cas:title">Tumor Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">57-60</span>CODEN:
                <span class="NLM_cas:coden">TUMBEA</span>;
        ISSN:<span class="NLM_cas:issn">1010-4283</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">TARBP2 is a RNA-binding protein (RBP) involved in miRNA processing and maturation.  TARBP2 plays significant roles in many biol. and pathol. conditions, including viral expression of HIV-1, microsatellite instability, cancer stem cell properties, and tumor progression.  Overexpression of TARBP2 was obsd. in many cancers such as prostate cancer, cutaneous malignant melanoma, and adrenocortical carcinoma.  In addn., TARBP2 was also found to be downregulated in some cancers including colorectal cancer, gastric cancer, Ewing sarcoma, and upper urinary tract urothelial carcinoma.  Therefore, whether TARBP2 functions as the tumor suppressor or tumor promoter is conflicting.  In the present review, we provide an overview of current knowledge concerning the role of TARBP2 in tumor development and progression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpDzi0qOWRxhbVg90H21EOLACvtfcHk0liT0jwPN8oDuw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslekurvE&md5=d30827a7a0b48c0d7d50582ca30e13ea</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1007%2Fs13277-015-4273-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs13277-015-4273-6%26sid%3Dliteratum%253Aachs%26aulast%3DYu%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DZ.%26atitle%3DThe%2520role%2520of%2520TARBP2%2520in%2520the%2520development%2520and%2520progression%2520of%2520cancers%26jtitle%3DTumor%2520Biol.%26date%3D2016%26volume%3D37%26spage%3D57%26epage%3D60%26doi%3D10.1007%2Fs13277-015-4273-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Felicetti, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cecchetti, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manfroni, G.</span></span> <span> </span><span class="NLM_article-title">Modulating microRNA Processing: Enoxacin, the Progenitor of a New Class of Drugs</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">12275</span>– <span class="NLM_lpage">12289</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c00510</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00510" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtl2rsrnL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=12275-12289&author=T.+Felicettiauthor=V.+Cecchettiauthor=G.+Manfroni&title=Modulating+microRNA+Processing%3A+Enoxacin%2C+the+Progenitor+of+a+New+Class+of+Drugs&doi=10.1021%2Facs.jmedchem.0c00510"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Modulating microRNA Processing: Enoxacin, the Progenitor of a New Class of Drugs</span></div><div class="casAuthors">Felicetti, Tommaso; Cecchetti, Violetta; Manfroni, Giuseppe</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">12275-12289</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The RNA interference (RNAi) process encompasses the cellular mechanisms by which short-noncoding RNAs posttranscriptionally modulate gene expression.  First discovered in 1998, today RNAi represents the foundation underlying complex biol. mechanisms that are dysregulated in many diseases.  MicroRNAs are effector mols. of gene silencing in RNAi, and their modulation can lead to a wide response in cells.  Enoxacin was reported as the first and unique small-mol. enhancer of microRNA (SMER) maturation.  Herein, the biol. activity of enoxacin as SMER is discussed to shed light on its innovative mode of action, its potential in treating different diseases, and the feasibility of using enoxacin as a chem. template for inspiring medicinal chemists.  We debate its mechanism of action at the mol. level and the possible impact on future ligand and/or structure-guided chem. optimizations, as well as opportunities and drawbacks assocd. with the development of quinolones such as SMERs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrCQcmft_rGlLVg90H21EOLACvtfcHk0liT0jwPN8oDuw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtl2rsrnL&md5=81a379dd43abf95a4b23366c006db178</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00510&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00510%26sid%3Dliteratum%253Aachs%26aulast%3DFelicetti%26aufirst%3DT.%26aulast%3DCecchetti%26aufirst%3DV.%26aulast%3DManfroni%26aufirst%3DG.%26atitle%3DModulating%2520microRNA%2520Processing%253A%2520Enoxacin%252C%2520the%2520Progenitor%2520of%2520a%2520New%2520Class%2520of%2520Drugs%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D12275%26epage%3D12289%26doi%3D10.1021%2Facs.jmedchem.0c00510" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ikeya, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taguchi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yosioka, I.</span></span> <span> </span><span class="NLM_article-title">The constituents of Schizandra-chinensis baill.10. The structures of gamma-schizandrin and 4 New Lignans, (−)-Gomisins L1 and L2, (+1)-Gomisin M1 and (+)-Gomisin M2</span>. <i>Chem. Pharm. Bull.</i> <span class="NLM_year" style="font-weight: bold;">1982</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">132</span>– <span class="NLM_lpage">139</span>, <span class="refDoi"> DOI: 10.1248/cpb.30.132</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1248%2Fcpb.30.132" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADyaL38XksFGmt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=1982&pages=132-139&author=Y.+Ikeyaauthor=H.+Taguchiauthor=I.+Yosioka&title=The+constituents+of+Schizandra-chinensis+baill.10.+The+structures+of+gamma-schizandrin+and+4+New+Lignans%2C+%28%E2%88%92%29-Gomisins+L1+and+L2%2C+%28%2B1%29-Gomisin+M1+and+%28%2B%29-Gomisin+M2&doi=10.1248%2Fcpb.30.132"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">The constituents of Schizandra chinensis Baill.  X.  The structures of γ-schizandrin and four new lignans, (-)-gomisins L1 and L2 (±)-gomisin M1 and (+)-gomisin M2</span></div><div class="casAuthors">Ikeya, Yukinobu; Taguchi, Heihachiro; Yosioka, Itiro</div><div class="citationInfo"><span class="NLM_cas:title">Chemical & Pharmaceutical Bulletin</span>
        (<span class="NLM_cas:date">1982</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">132-9</span>CODEN:
                <span class="NLM_cas:coden">CPBTAL</span>;
        ISSN:<span class="NLM_cas:issn">0009-2363</span>.
    </div><div class="casAbstract">Four new title lignans,(-)-gomisin L1 (I, R = H, R1 = Me) and L2 (I, R = Me, R1 = H), (±)-gomisin M1 (II, R2 = H, R3 = Me) and (+)-gomisin (II, R2 = Me, R3 = H), were isolated from the fruits of Schizandra chinensis and their structures detd. by chem. and spectral studies.  The stereostructure of γ-schizandrin (III, dl-form), the plane structure of which had already been suggested (N. K. Kochetkov, et al. 1964 and C. S. Liu, et al. 1978) was also elucidated on the basis of chem. and spectral studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPz7LUyIN4OrVg90H21EOLACvtfcHk0liT0jwPN8oDuw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL38XksFGmt7w%253D&md5=2a1b6c69669f058b837cfa4b2ac46319</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1248%2Fcpb.30.132&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1248%252Fcpb.30.132%26sid%3Dliteratum%253Aachs%26aulast%3DIkeya%26aufirst%3DY.%26aulast%3DTaguchi%26aufirst%3DH.%26aulast%3DYosioka%26aufirst%3DI.%26atitle%3DThe%2520constituents%2520of%2520Schizandra-chinensis%2520baill.10.%2520The%2520structures%2520of%2520gamma-schizandrin%2520and%25204%2520New%2520Lignans%252C%2520%2528%25E2%2588%2592%2529-Gomisins%2520L1%2520and%2520L2%252C%2520%2528%252B1%2529-Gomisin%2520M1%2520and%2520%2528%252B%2529-Gomisin%2520M2%26jtitle%3DChem.%2520Pharm.%2520Bull.%26date%3D1982%26volume%3D30%26spage%3D132%26epage%3D139%26doi%3D10.1248%2Fcpb.30.132" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Z. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, X. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, J.</span></span> <span> </span><span class="NLM_article-title">Dibenzocyclooctadiene lignans from Schisandra chinensis and their inhibitory activity on NO production in lipopolysaccharide-activated microglia cells</span>. <i>Phytochemistry</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>104</i></span>,  <span class="NLM_fpage">72</span>– <span class="NLM_lpage">78</span>, <span class="refDoi"> DOI: 10.1016/j.phytochem.2014.04.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1016%2Fj.phytochem.2014.04.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=24864016" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BC2cXoslCmsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=104&publication_year=2014&pages=72-78&author=D.+Huauthor=Z.+Y.+Yangauthor=X.+C.+Yaoauthor=H.+Wangauthor=N.+Hanauthor=Z.+H.+Liuauthor=Y.+Wangauthor=J.+Y.+Yangauthor=J.+Yin&title=Dibenzocyclooctadiene+lignans+from+Schisandra+chinensis+and+their+inhibitory+activity+on+NO+production+in+lipopolysaccharide-activated+microglia+cells&doi=10.1016%2Fj.phytochem.2014.04.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Dibenzocyclooctadiene lignans from Schisandra chinensis and their inhibitory activity on NO production in lipopolysaccharide-activated microglia cells</span></div><div class="casAuthors">Hu, Di; Yang, Zhiyou; Yao, Xuechun; Wang, Hua; Han, Na; Liu, Zhihui; Wang, Yu; Yang, Jingyu; Yin, Jun</div><div class="citationInfo"><span class="NLM_cas:title">Phytochemistry (Elsevier)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">104</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">72-78</span>CODEN:
                <span class="NLM_cas:coden">PYTCAS</span>;
        ISSN:<span class="NLM_cas:issn">0031-9422</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Four dibenzocyclooctadiene lignans, schisanchinins A-D, and 10 known compds. were isolated from the EtOAc ext. of fruits of Schisandra chinensis (Turcz.) Baill.  Structures of compds. 1-4 were elucidated using a combination of spectroscopic techniques, including MS, UV and IR, NMR (1H NMR, 13C NMR, HMQC, HMBC).  The stereochem. of the chiral centers and the biphenyl configuration were detd. using NOESY, as well as anal. of CD spectra.  In vitro activity assays showed that 11 of the 14 compds. exhibited inhibitory activity on lipopolysaccharide (LPS)-induced NO release in primary murine BV2 microglia cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzoyq4inTCY7Vg90H21EOLACvtfcHk0lgphWAkG1hiuQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXoslCmsr4%253D&md5=d547bb4e00d23b5d21102f645f889524</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.phytochem.2014.04.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.phytochem.2014.04.014%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DD.%26aulast%3DYang%26aufirst%3DZ.%2BY.%26aulast%3DYao%26aufirst%3DX.%2BC.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DHan%26aufirst%3DN.%26aulast%3DLiu%26aufirst%3DZ.%2BH.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DJ.%2BY.%26aulast%3DYin%26aufirst%3DJ.%26atitle%3DDibenzocyclooctadiene%2520lignans%2520from%2520Schisandra%2520chinensis%2520and%2520their%2520inhibitory%2520activity%2520on%2520NO%2520production%2520in%2520lipopolysaccharide-activated%2520microglia%2520cells%26jtitle%3DPhytochemistry%26date%3D2014%26volume%3D104%26spage%3D72%26epage%3D78%26doi%3D10.1016%2Fj.phytochem.2014.04.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, C. L.</span></span> <span> </span><span class="NLM_article-title">High throughput screening and antioxidant assay of dibenzo[a, c]cyclooctadiene lignans in modified-ultrasonic and supercritical fluid extracts of Schisandra chinensis Baill by liquid chromatography-mass spectrometry and a free radical-scavenging method</span>. <i>J. Sep. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">1322</span>– <span class="NLM_lpage">1332</span>, <span class="refDoi"> DOI: 10.1002/jssc.200700443</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1002%2Fjssc.200700443" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=18446887" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BD1cXmsFOis78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2008&pages=1322-1332&author=M.+C.+Wangauthor=Y.+C.+Laiauthor=C.+L.+Chang&title=High+throughput+screening+and+antioxidant+assay+of+dibenzo%5Ba%2C+c%5Dcyclooctadiene+lignans+in+modified-ultrasonic+and+supercritical+fluid+extracts+of+Schisandra+chinensis+Baill+by+liquid+chromatography-mass+spectrometry+and+a+free+radical-scavenging+method&doi=10.1002%2Fjssc.200700443"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">High throughput screening and antioxidant assay of dibenzo[a,c]cyclooctadiene lignans in modified-ultrasonic and supercritical fluid extracts of Schisandra chinensis Baill by liquid chromatography-mass spectrometry and a free radical-scavenging method</span></div><div class="casAuthors">Wang, Ming-Chih; Lai, Yih-Cherng; Chang, Chia-Lin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Separation Science</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1322-1332</span>CODEN:
                <span class="NLM_cas:coden">JSSCCJ</span>;
        ISSN:<span class="NLM_cas:issn">1615-9306</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Dibenzo[a,c]cyclooctadiene lignans of Schisandra chinensis are well known because of their hepatoprotective activity, antioxidant activity, and anticancer effect.  For the isolation of the dibenzo[a,c]cyclooctadiene lignans of Schisandra chinensis two extn. methods were used: modified-ultrasonic extn. and supercrit. fluid extn.  A specific and fast anal. method for structure identification is established for quality control because structure elucidation could be accomplished by means of liq. chromatog.-mass spectrometry (LC-MS) technologies.  The sepn. and identification of the compds. were completed by: (i) a water-acetonitrile gradient system using a C18 reversed-phase column; (ii) UV detection at 225 nm; (iii) MS/MS expts. with electrospray ionization interface (ESI) ion trap mass spectrometry in the pos. mode.  Normalized collision energy was used to obtain fragment ions of structural relevance in the LC-MS/MS.  These results provided a reliable LC-MS/MS method for the detn. of the dibenzo[a,c]cyclooctadiene lignans from Schisandra chinensis.  Finally, we also detected 1,1-diphenyl-2-picrylhydrazyl (DPPH) scavenging effects (%) of the modified-ultrasonic and supercrit. fluid exts. of Schisandra chinensis compared with 6-hydroxy-2,5,7,8-tetramethylchroman-2-carboxylic acid (Trolox).  The antioxidant activities of the modified-ultrasonic and supercrit. fluid exts. were lower than that of Trolox.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrvYEP231BrBLVg90H21EOLACvtfcHk0lgphWAkG1hiuQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXmsFOis78%253D&md5=fefe69afa3599afffb30530b6637a059</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1002%2Fjssc.200700443&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjssc.200700443%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DM.%2BC.%26aulast%3DLai%26aufirst%3DY.%2BC.%26aulast%3DChang%26aufirst%3DC.%2BL.%26atitle%3DHigh%2520throughput%2520screening%2520and%2520antioxidant%2520assay%2520of%2520dibenzo%255Ba%252C%2520c%255Dcyclooctadiene%2520lignans%2520in%2520modified-ultrasonic%2520and%2520supercritical%2520fluid%2520extracts%2520of%2520Schisandra%2520chinensis%2520Baill%2520by%2520liquid%2520chromatography-mass%2520spectrometry%2520and%2520a%2520free%2520radical-scavenging%2520method%26jtitle%3DJ.%2520Sep.%2520Sci.%26date%3D2008%26volume%3D31%26spage%3D1322%26epage%3D1332%26doi%3D10.1002%2Fjssc.200700443" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Poornima, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anand Kumar, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siva, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Venkanna, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vadaparthi, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tiwari, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suresh Babu, K.</span></span> <span> </span><span class="NLM_article-title">Advanced glycation end-products inhibitors isolated from Schisandra grandiflora</span>. <i>Nat. Prod. Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">493</span>– <span class="NLM_lpage">496</span>, <span class="refDoi"> DOI: 10.1080/14786419.2015.1024117</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1080%2F14786419.2015.1024117" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=25812677" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BC2MXlsFGlt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2016&pages=493-496&author=B.+Poornimaauthor=D.+Anand+Kumarauthor=B.+Sivaauthor=A.+Venkannaauthor=P.+R.+Vadaparthiauthor=K.+Kumarauthor=A.+K.+Tiwariauthor=K.+Suresh+Babu&title=Advanced+glycation+end-products+inhibitors+isolated+from+Schisandra+grandiflora&doi=10.1080%2F14786419.2015.1024117"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Advanced glycation end-products inhibitors isolated from Schisandra grandiflora</span></div><div class="casAuthors">Poornima, B.; Anand Kumar, D.; Siva, Bandi; Venkanna, A.; Vadaparthi, P. R. Rao; Kumar, K.; Tiwari, Ashok K.; Suresh Babu, K.</div><div class="citationInfo"><span class="NLM_cas:title">Natural Product Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">493-496</span>CODEN:
                <span class="NLM_cas:coden">NPRAAT</span>;
        ISSN:<span class="NLM_cas:issn">1478-6419</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">Free radicals scavenging and advanced glycation end-products (AGEs) inhibitory potentials in crude chloroform ext. of Schisandra grandiflora were evaluated.  Bioassay-guided isolation of the chloroform ext. led to the identification of 24 compds.  Among the isolates, (±)-gomisin M1, arisantetralone C and D, macelignan, saurulignan B and SZ-MO displayed potent-free radical scavenging as well as AGEs inhibitory potentials.  This is the first report identifying the presence of AGEs inhibitory activity and assigning AGEs inhibitory activity to these compds.  Therefore, our research finds new application of traditional medicinal plant S. grandiflora having capacity to reduce formation and accumulation of AGEs in diabetes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpQpv_FcubLfbVg90H21EOLACvtfcHk0lgphWAkG1hiuQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXlsFGlt7w%253D&md5=b5822177bf0ed1f9b089738875c98e0f</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1080%2F14786419.2015.1024117&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F14786419.2015.1024117%26sid%3Dliteratum%253Aachs%26aulast%3DPoornima%26aufirst%3DB.%26aulast%3DAnand%2BKumar%26aufirst%3DD.%26aulast%3DSiva%26aufirst%3DB.%26aulast%3DVenkanna%26aufirst%3DA.%26aulast%3DVadaparthi%26aufirst%3DP.%2BR.%26aulast%3DKumar%26aufirst%3DK.%26aulast%3DTiwari%26aufirst%3DA.%2BK.%26aulast%3DSuresh%2BBabu%26aufirst%3DK.%26atitle%3DAdvanced%2520glycation%2520end-products%2520inhibitors%2520isolated%2520from%2520Schisandra%2520grandiflora%26jtitle%3DNat.%2520Prod.%2520Res.%26date%3D2016%26volume%3D30%26spage%3D493%26epage%3D496%26doi%3D10.1080%2F14786419.2015.1024117" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fujihashi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hara, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakata, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mori, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Higuchi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaji, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaji, A.</span></span> <span> </span><span class="NLM_article-title">Anti-human immunodeficiency virus (HIV) activities of halogenated Gomisin J derivatives, new nonnucleoside inhibitors of HIV type 1 reverse transcriptase</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">2000</span>– <span class="NLM_lpage">2007</span>, <span class="refDoi"> DOI: 10.1128/AAC.39.9.2000</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1128%2FAAC.39.9.2000" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=8540706" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADyaK2MXnvVygsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=1995&pages=2000-2007&author=T.+Fujihashiauthor=H.+Haraauthor=T.+Sakataauthor=K.+Moriauthor=H.+Higuchiauthor=A.+Tanakaauthor=H.+Kajiauthor=A.+Kaji&title=Anti-human+immunodeficiency+virus+%28HIV%29+activities+of+halogenated+Gomisin+J+derivatives%2C+new+nonnucleoside+inhibitors+of+HIV+type+1+reverse+transcriptase&doi=10.1128%2FAAC.39.9.2000"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Anti-human immunodeficiency virus (HIV) activities of halogenated gomisin J derivatives, new nonnucleoside inhibitors of HIV type 1 reverse transcriptase</span></div><div class="casAuthors">Fujihashi, Toshiaki; Hara, Hiroto; Sakata, Toshiya; Mori, Kazuya; Higuchi, Hirotaka; Tanaka, Akio; Kaji, Hideko; Kaji, Akira</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2000-7</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">0066-4804</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Halogenated gomisin J (a deriv. of lignan compd.), represented by the bromine deriv. 1506 {(6R,7S,S-biar)-4,9-dibromo-3,10-dihydroxy-1,2,11,12-tetramethoxy-6,7-dimethyl-5,6,7.8-tetrahydrodibenzo[a,c]cyclooctene}, was a potent inhibitor of the cytopathic effects of human immunodeficiency virus type 1 (HIV-1) on MT-4 human T cells (50% ED, 0.1 to 0.5 μM).  Gomisin J derivs. were active in preventing p24 prodn. from acutely HIV-1-infected H9 cells.  The selective indexes (toxic dose/ED) of these compds. were as high as >300 in some systems.  1506 Was active against 3'-azido-3'-deoxythymidine-resistant HIV-1 and acted synergistically with AZT and 2',3'-ddC.  1506 Inhibited HIV-1 reverse transcriptase (RT) in vitro but not HIV-1 protease.  From the time-of-addn. expt., 1506 was found to inhibit the early phase of the HIV life cycle.  A 1506-resistant HIV mutant was selected and shown to possess a mutation within the RT-coding region (at position 188 [Tyr to Leu]).  The mutant RT expressed in Escherichia coli was resistant to 1506 in the in vitro RT assay.  Some of the HIV strains resistant to other nonnucleoside HIV-1 RT inhibitors were also resistant to 1506.  Comparison of various gomisin J derivs. with gomisin J showed that iodine, bromine, and chlorine in the fourth and ninth positions increased RT inhibitory activity as well as cytoprotective activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoy4YdCT_tsTbVg90H21EOLACvtfcHk0lhdArDOqJfxfA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXnvVygsbg%253D&md5=ec3f5e4b2f8f62ac91c9074436ffd615</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1128%2FAAC.39.9.2000&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.39.9.2000%26sid%3Dliteratum%253Aachs%26aulast%3DFujihashi%26aufirst%3DT.%26aulast%3DHara%26aufirst%3DH.%26aulast%3DSakata%26aufirst%3DT.%26aulast%3DMori%26aufirst%3DK.%26aulast%3DHiguchi%26aufirst%3DH.%26aulast%3DTanaka%26aufirst%3DA.%26aulast%3DKaji%26aufirst%3DH.%26aulast%3DKaji%26aufirst%3DA.%26atitle%3DAnti-human%2520immunodeficiency%2520virus%2520%2528HIV%2529%2520activities%2520of%2520halogenated%2520Gomisin%2520J%2520derivatives%252C%2520new%2520nonnucleoside%2520inhibitors%2520of%2520HIV%2520type%25201%2520reverse%2520transcriptase%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D1995%26volume%3D39%26spage%3D2000%26epage%3D2007%26doi%3D10.1128%2FAAC.39.9.2000" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Miyamoto, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wakusawa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nomura, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanae, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakai, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sudo, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohtaki, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takeda, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujii, Y.</span></span> <span> </span><span class="NLM_article-title">Effects of Gomisin A on hepatocarcinogenesis by 3′-methyl-4-dimethylaminoazobenzene in rats</span>. <i>Jpn. J. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">1991</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">71</span>– <span class="NLM_lpage">77</span>, <span class="refDoi"> DOI: 10.1254/jjp.57.71</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1254%2Fjjp.57.71" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1800800" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADyaK3MXlvFOqsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=1991&pages=71-77&author=K.+Miyamotoauthor=S.+Wakusawaauthor=M.+Nomuraauthor=F.+Sanaeauthor=R.+Sakaiauthor=K.+Sudoauthor=Y.+Ohtakiauthor=S.+Takedaauthor=Y.+Fujii&title=Effects+of+Gomisin+A+on+hepatocarcinogenesis+by+3%E2%80%B2-methyl-4-dimethylaminoazobenzene+in+rats&doi=10.1254%2Fjjp.57.71"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Effects of gomisin A on hepatocarcinogenesis by 3'-methyl-4-dimethylaminoazobenzene in rats</span></div><div class="casAuthors">Miyamoto, Kenichi; Wakusawa, Shinya; Nomura, Masaaki; Sanae, Fujiko; Sakai, Ryosuke; Sudo, Kazuhiko; Ohtaki, Yumiko; Takeda, Shigefumi; Fujii, Yuichi</div><div class="citationInfo"><span class="NLM_cas:title">Japanese Journal of Pharmacology</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">71-7</span>CODEN:
                <span class="NLM_cas:coden">JJPAAZ</span>;
        ISSN:<span class="NLM_cas:issn">0021-5198</span>.
    </div><div class="casAbstract">The effects of gomisin A (I) on tumor promotion in the liver after a short-term feeding of 3'-methyl-4-dimethylaminoazobenzene (3'-MeDAB) to rats was examd., compared with the effects of phenobarbital.  Male Donryu rats were fed ad libitum a diet contg. 0.06% 3'-MeDAB and 0.03% or 0.01% I or water contg. 0.05% phenobarbital.  I and phenobarbital did not cause any proliferative and neoplastic lesions by themselves in 40 wk of feeding.  Altered foci in the liver increased with a peak at 12 wk after the rats were fed 3'-MeDAB.  I decreased the no. of hepatic altered foci such as the clear cell and basophilic cell type foci in the early stages.  Phenobarbital enhanced neoplastic alterations so that the no. and size of the foci were much larger in the phenobarbital-combined group than in the 3'-MeDAB-control group.  Thus, phenobarbital acted as a promoter of cells initiated by 3'-MeDAB; on the other hand, I showed a weak suppressive effect on tumor promotion.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0_zPRkFFyvbVg90H21EOLACvtfcHk0lhdArDOqJfxfA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3MXlvFOqsbw%253D&md5=30d6ab7a7986923a5deef12fb676ede5</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1254%2Fjjp.57.71&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1254%252Fjjp.57.71%26sid%3Dliteratum%253Aachs%26aulast%3DMiyamoto%26aufirst%3DK.%26aulast%3DWakusawa%26aufirst%3DS.%26aulast%3DNomura%26aufirst%3DM.%26aulast%3DSanae%26aufirst%3DF.%26aulast%3DSakai%26aufirst%3DR.%26aulast%3DSudo%26aufirst%3DK.%26aulast%3DOhtaki%26aufirst%3DY.%26aulast%3DTakeda%26aufirst%3DS.%26aulast%3DFujii%26aufirst%3DY.%26atitle%3DEffects%2520of%2520Gomisin%2520A%2520on%2520hepatocarcinogenesis%2520by%25203%25E2%2580%25B2-methyl-4-dimethylaminoazobenzene%2520in%2520rats%26jtitle%3DJpn.%2520J.%2520Pharmacol.%26date%3D1991%26volume%3D57%26spage%3D71%26epage%3D77%26doi%3D10.1254%2Fjjp.57.71" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Miyamoto, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hiramatsu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohtaki, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kanitani, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nomura, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aburada, M.</span></span> <span> </span><span class="NLM_article-title">Effects of Gomisin A on the promotor action and serum bile acid concentration in hepatocarcinogenesis induced by 3′-methyl-4-dimethylamino-azobenzene</span>. <i>Biol. Pharm. Bull.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">1443</span>– <span class="NLM_lpage">1445</span>, <span class="refDoi"> DOI: 10.1248/bpb.18.1443</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1248%2Fbpb.18.1443" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=8593453" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADyaK2MXpsFCnt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=1995&pages=1443-1445&author=K.+Miyamotoauthor=K.+Hiramatsuauthor=Y.+Ohtakiauthor=M.+Kanitaniauthor=M.+Nomuraauthor=M.+Aburada&title=Effects+of+Gomisin+A+on+the+promotor+action+and+serum+bile+acid+concentration+in+hepatocarcinogenesis+induced+by+3%E2%80%B2-methyl-4-dimethylamino-azobenzene&doi=10.1248%2Fbpb.18.1443"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Effects of gomisin A on the promoter action and serum bile acid concentration in hepatocarcinogenesis induced by 3'-methyl-4-dimethylamino-azobenzene</span></div><div class="casAuthors">Miyamoto, Ken-ichi; Hiramatsu, Kazuyoshi; Ohtaki, Yumiko; Kanitani, Masanao; Nomura, Masaaki; Aburada, Masaki</div><div class="citationInfo"><span class="NLM_cas:title">Biological & Pharmaceutical Bulletin</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1443-5</span>CODEN:
                <span class="NLM_cas:coden">BPBLEO</span>;
        ISSN:<span class="NLM_cas:issn">0918-6158</span>.
    
            (<span class="NLM_cas:orgname">Pharmaceutical Society of Japan</span>)
        </div><div class="casAbstract">The effects of gomisin A, a lignan component of Schizandra fruits, on the promotion stage of hepatocarcinogenesis initiated by 3'-methyl-4-dimethylamino-azobenzene (3'-MeDAB) in male Donryu rats were investigated.  When different types of tumor promoters, phenobarbital (PB) and deoxycholic acid (DCA), were administered for 5 wk after initiation by 3'-MeDAB, preneoplastic alterations in the liver, detd. by glutathione S-transferase placental form (GST-P), were markedly increased.  Gomisin A significantly inhibited the increase in no. and size of GST-P pos. foci, regardless of the promoter.  This lignan inhibited the increase in serum bile acid concn. by administration of DCA, but hardly influenced the serum bile acids in the PB-combined group.  These results suggest that the inhibitory effect of gomisin A on the promotive action of DCA is based on improving bile acid metab., but regarding the action of PB, the effect could not be elucidated from the metab. of bile acids.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrsxYCKwPleC7Vg90H21EOLACvtfcHk0lhdArDOqJfxfA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXpsFCnt7Y%253D&md5=385d7f3394612fd862fec49428192b8c</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1248%2Fbpb.18.1443&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1248%252Fbpb.18.1443%26sid%3Dliteratum%253Aachs%26aulast%3DMiyamoto%26aufirst%3DK.%26aulast%3DHiramatsu%26aufirst%3DK.%26aulast%3DOhtaki%26aufirst%3DY.%26aulast%3DKanitani%26aufirst%3DM.%26aulast%3DNomura%26aufirst%3DM.%26aulast%3DAburada%26aufirst%3DM.%26atitle%3DEffects%2520of%2520Gomisin%2520A%2520on%2520the%2520promotor%2520action%2520and%2520serum%2520bile%2520acid%2520concentration%2520in%2520hepatocarcinogenesis%2520induced%2520by%25203%25E2%2580%25B2-methyl-4-dimethylamino-azobenzene%26jtitle%3DBiol.%2520Pharm.%2520Bull.%26date%3D1995%26volume%3D18%26spage%3D1443%26epage%3D1445%26doi%3D10.1248%2Fbpb.18.1443" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, X. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chou, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, J. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, Z. L.</span></span> <span> </span><span class="NLM_article-title">Gomisin N exerts anti-liver cancer effects and regulates PI3K-Akt and mTOR-ULK1 pathways in Vitro</span>. <i>Biol. Pharm. Bull.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>43</i></span>,  <span class="NLM_fpage">1267</span>– <span class="NLM_lpage">1271</span>, <span class="refDoi"> DOI: 10.1248/bpb.b20-00030</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1248%2Fbpb.b20-00030" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=32741948" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BB3cXis1equrjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2020&pages=1267-1271&author=P.+L.+Zhuauthor=D.+F.+Lamauthor=J.+K.+Liauthor=X.+Q.+Fuauthor=C.+L.+Yinauthor=J.+Y.+Chouauthor=Y.+P.+Wangauthor=Y.+X.+Liuauthor=Y.+J.+Chenauthor=J.+Y.+Wuauthor=Y.+Wuauthor=J.+X.+Baiauthor=C.+Liangauthor=Z.+L.+Yu&title=Gomisin+N+exerts+anti-liver+cancer+effects+and+regulates+PI3K-Akt+and+mTOR-ULK1+pathways+in+Vitro&doi=10.1248%2Fbpb.b20-00030"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Gomisin N exerts anti-liver cancer effects and regulates PI3K-Akt and mTOR-ULK1 pathways in vitro</span></div><div class="casAuthors">Zhu, Pei-Li; Lam, Dick Fai; Li, Jun-Kui; Fu, Xiu-Qiong; Yin, Cheng-Le; Chou, Ji-Yao; Wang, Ya-Ping; Liu, Yu-Xi; Chen, Ying-Jie; Wu, Jia-Ying; Wu, Ying; Bai, Jing-Xuan; Liang, Chun; Yu, Zhi-Ling</div><div class="citationInfo"><span class="NLM_cas:title">Biological & Pharmaceutical Bulletin</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1267-1271</span>CODEN:
                <span class="NLM_cas:coden">BPBLEO</span>;
        ISSN:<span class="NLM_cas:issn">0918-6158</span>.
    
            (<span class="NLM_cas:orgname">Pharmaceutical Society of Japan</span>)
        </div><div class="casAbstract">Primary liver cancer is a lethal cancer.  The phosphatidylinositol 3-kinase (PI3K)-Akt-mammalian target of rapamycin (mTOR) pathway has been implicated in the pathogenesis of liver cancer.  Gomisin N (GN), a lignan isolated from the dried fruits of Schisandra chinensis (Turca.) Baill., has been reported to reduce viability of, and induce apoptosis in, HepG2 liver cancer cells.  In preadipocytes, GN was found to inhibit Akt activity.  In the present study, Akt signaling-related anti-liver cancer mechanisms of GN were investigated.  We confirmed that GN reduces cell viability of, and triggers apoptosis in, more liver cancer cell lines.  Mechanistic studies revealed that GN lowers protein levels of phospho-PI3K (p85 tyrosine (Tyr)458), phospho-Akt (serine (Ser)473), and Akt downstream mols. Mcl-1 in HepG2 and HCCLM3 cells.  Meanwhile, GN activates mTOR and inhibits ULK1 (a neg. downstream effector of mTOR) activities.  Activation of mTOR has been reported to suppress ULK1 activity and repress autophagy.  Indeed, we obsd. that GN inhibits autophagy in liver cancer cells.  In summary, we for the first time demonstrated that GN inhibits the PI3K-Akt pathway and regulates the mTOR-ULK1 pathway in liver cancer cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpry92uaZ42gbVg90H21EOLACvtfcHk0lj7bPMVCECgsQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXis1equrjO&md5=edc739e6983564feee24ec43b81e018f</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1248%2Fbpb.b20-00030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1248%252Fbpb.b20-00030%26sid%3Dliteratum%253Aachs%26aulast%3DZhu%26aufirst%3DP.%2BL.%26aulast%3DLam%26aufirst%3DD.%2BF.%26aulast%3DLi%26aufirst%3DJ.%2BK.%26aulast%3DFu%26aufirst%3DX.%2BQ.%26aulast%3DYin%26aufirst%3DC.%2BL.%26aulast%3DChou%26aufirst%3DJ.%2BY.%26aulast%3DWang%26aufirst%3DY.%2BP.%26aulast%3DLiu%26aufirst%3DY.%2BX.%26aulast%3DChen%26aufirst%3DY.%2BJ.%26aulast%3DWu%26aufirst%3DJ.%2BY.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DBai%26aufirst%3DJ.%2BX.%26aulast%3DLiang%26aufirst%3DC.%26aulast%3DYu%26aufirst%3DZ.%2BL.%26atitle%3DGomisin%2520N%2520exerts%2520anti-liver%2520cancer%2520effects%2520and%2520regulates%2520PI3K-Akt%2520and%2520mTOR-ULK1%2520pathways%2520in%2520Vitro%26jtitle%3DBiol.%2520Pharm.%2520Bull.%26date%3D2020%26volume%3D43%26spage%3D1267%26epage%3D1271%26doi%3D10.1248%2Fbpb.b20-00030" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yim, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, Y. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, Y. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jung, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Son, B. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, Y. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, Y. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, Y. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hwang, D. Y.</span></span> <span> </span><span class="NLM_article-title">Gomisin N isolated from Schisandra Chinensis significantly induces anti-proliferative and pro-apoptotic effects in hepatic carcinoma</span>. <i>Mol. Med. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">725</span>– <span class="NLM_lpage">732</span>, <span class="refDoi"> DOI: 10.3892/mmr_00000163</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.3892%2Fmmr_00000163" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=21475892" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BD1MXpvFGnsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2009&pages=725-732&author=S.+Y.+Yimauthor=Y.+J.+Leeauthor=Y.+K.+Leeauthor=S.+E.+Jungauthor=J.+H.+Kimauthor=H.+J.+Kimauthor=B.+G.+Sonauthor=Y.+H.+Parkauthor=Y.+G.+Leeauthor=Y.+W.+Choiauthor=D.+Y.+Hwang&title=Gomisin+N+isolated+from+Schisandra+Chinensis+significantly+induces+anti-proliferative+and+pro-apoptotic+effects+in+hepatic+carcinoma&doi=10.3892%2Fmmr_00000163"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Gomisin N isolated from Schisandra chinensis significantly induces anti-proliferative and pro-apoptotic effects in hepatic carcinoma</span></div><div class="casAuthors">Yim, Su Youn; Lee, You Jin; Lee, Yoen Kyung; Jung, Seung Eun; Kim, Ji Ha; Kim, Ji Eun; Kim, Hak Jin; Son, Beung Gu; Park, Young Hoon; Lee, Young Guen; Choi, Young Whan; Hwang, Dae Youn</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Medicine Reports</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">725-732</span>CODEN:
                <span class="NLM_cas:coden">MMROA5</span>;
        ISSN:<span class="NLM_cas:issn">1791-2997</span>.
    
            (<span class="NLM_cas:orgname">Spandidos Publications Ltd.</span>)
        </div><div class="casAbstract">Lignans isolated from Schisandria chinensis have been prescribed as anti-cancer and anti-hepatitis treatments in Chinese medicine.  To investigate the applications of lignans isolated from Schisandria chinensis in hepatic carcinoma therapy, their apoptotic ability was screened using a cell proliferation assay.  Compared to the other lignans, gomisin N induced high apoptotic levels in hepatic carcinoma.  Cell morphol. and flow cytometric anal. demonstrated that this lignan induced cell death at high concns., but did not induce any changes at low concns.  In addn., the expression levels of Bcl-2 and Bax proteins, which are involved in the apoptotic pathway, were markedly increased in only the 320 μM-treated group compared to the vehicle and other concn. groups, while the expression level of p53 protein remained unchanged in this group.  These results suggest that gomisin N is an anti-cancer drug candidate capable of inhibiting the proliferation and inducing the apoptosis of human hepatic carcinomas.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpFaeLzz2_DQrVg90H21EOLACvtfcHk0lj7bPMVCECgsQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXpvFGnsro%253D&md5=4ee7484292fc20a1ebc6d5711228caac</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.3892%2Fmmr_00000163&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3892%252Fmmr_00000163%26sid%3Dliteratum%253Aachs%26aulast%3DYim%26aufirst%3DS.%2BY.%26aulast%3DLee%26aufirst%3DY.%2BJ.%26aulast%3DLee%26aufirst%3DY.%2BK.%26aulast%3DJung%26aufirst%3DS.%2BE.%26aulast%3DKim%26aufirst%3DJ.%2BH.%26aulast%3DKim%26aufirst%3DH.%2BJ.%26aulast%3DSon%26aufirst%3DB.%2BG.%26aulast%3DPark%26aufirst%3DY.%2BH.%26aulast%3DLee%26aufirst%3DY.%2BG.%26aulast%3DChoi%26aufirst%3DY.%2BW.%26aulast%3DHwang%26aufirst%3DD.%2BY.%26atitle%3DGomisin%2520N%2520isolated%2520from%2520Schisandra%2520Chinensis%2520significantly%2520induces%2520anti-proliferative%2520and%2520pro-apoptotic%2520effects%2520in%2520hepatic%2520carcinoma%26jtitle%3DMol.%2520Med.%2520Rep.%26date%3D2009%26volume%3D2%26spage%3D725%26epage%3D732%26doi%3D10.3892%2Fmmr_00000163" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tai, Z. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, F.</span></span> <span> </span><span class="NLM_article-title">Identification of small molecule activators of the janus kinase/signal transducer and activator of transcription pathway using a cell-based screen</span>. <i>Biol. Pharm. Bull.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">65</span>– <span class="NLM_lpage">71</span>, <span class="refDoi"> DOI: 10.1248/bpb.35.65</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1248%2Fbpb.35.65" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=22223339" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BC38XivFWhur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2012&pages=65-71&author=Z.+F.+Taiauthor=G.+L.+Zhangauthor=F.+Wang&title=Identification+of+small+molecule+activators+of+the+janus+kinase%2Fsignal+transducer+and+activator+of+transcription+pathway+using+a+cell-based+screen&doi=10.1248%2Fbpb.35.65"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of small molecule activators of the Janus kinase/signal transducer and activator of transcription pathway using a cell-based screen</span></div><div class="casAuthors">Tai, Zheng Fu; Zhang, Guo Lin; Wang, Fei</div><div class="citationInfo"><span class="NLM_cas:title">Biological & Pharmaceutical Bulletin</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">65-71</span>CODEN:
                <span class="NLM_cas:coden">BPBLEO</span>;
        ISSN:<span class="NLM_cas:issn">0918-6158</span>.
    
            (<span class="NLM_cas:orgname">Pharmaceutical Society of Japan</span>)
        </div><div class="casAbstract">Type I interferons (IFN-α/β) have been widely used in the treatment of many viral and malignant diseases by activation of the Janus kinase/signal transducer and activator of transcription (JAK/STAT) signaling pathway, but the side effects of protein-based IFN therapy severely limit their clin. use.  Discovering small mols. to activate the JAK/STAT pathway will greatly facilitate the development of new drugs which have similar pharmacol. function to IFNs but with fewer side effects.  To screen a natural products-based library, the authors established a cell-based screening assay using human hepatoma HepG2 cells stably transfected with a plasmid where the luciferase reporter activity is driven by interferon α-stimulated response element (ISRE), the motif specifically recognized by type I IFN-induced activation of JAK/STAT pathway.  Among 1431 natural product compds. screened, four compds. (emodin, quercetin, apigenin and luteolin) were identified as activators of the JAK/STAT pathway.  Further studies demonstrated that these four compds. could increase the endogenous antiviral gene expression regulated by the IFN-activated JAK/STAT pathway.  The identified small mol. activators are valuable for structural modification and warrant further investigation for use in new antiviral drugs as IFN mimics or adjuvants.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrLV-4Xq_79WrVg90H21EOLACvtfcHk0lj7bPMVCECgsQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XivFWhur8%253D&md5=b67dd1467ee491480c67bf9a08e398b9</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1248%2Fbpb.35.65&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1248%252Fbpb.35.65%26sid%3Dliteratum%253Aachs%26aulast%3DTai%26aufirst%3DZ.%2BF.%26aulast%3DZhang%26aufirst%3DG.%2BL.%26aulast%3DWang%26aufirst%3DF.%26atitle%3DIdentification%2520of%2520small%2520molecule%2520activators%2520of%2520the%2520janus%2520kinase%252Fsignal%2520transducer%2520and%2520activator%2520of%2520transcription%2520pathway%2520using%2520a%2520cell-based%2520screen%26jtitle%3DBiol.%2520Pharm.%2520Bull.%26date%3D2012%26volume%3D35%26spage%3D65%26epage%3D71%26doi%3D10.1248%2Fbpb.35.65" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hui, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span> <span> </span><span class="NLM_article-title">lncRNA-SOX2OT promotes hepatocellular carcinoma invasion and metastasis through miR-122–5p-mediated activation of PKM2</span>. <i>Oncogenesis</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">54</span>– <span class="NLM_lpage">65</span>, <span class="refDoi"> DOI: 10.1038/s41389-020-0242-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1038%2Fs41389-020-0242-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=32467565" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtVyqtbzP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2020&pages=54-65&author=Y.+Liangauthor=D.+Zhangauthor=T.+Zhengauthor=G.+Yangauthor=J.+Wangauthor=F.+Mengauthor=Y.+Liuauthor=G.+Zhangauthor=L.+Zhangauthor=J.+Hanauthor=P.+Huiauthor=Z.+Chenauthor=Y.+Liuauthor=M.+Wangauthor=H.+Jiangauthor=L.+Liu&title=lncRNA-SOX2OT+promotes+hepatocellular+carcinoma+invasion+and+metastasis+through+miR-122%E2%80%935p-mediated+activation+of+PKM2&doi=10.1038%2Fs41389-020-0242-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">lncRNA-SOX2OT promotes hepatocellular carcinoma invasion and metastasis through miR-122-5p-mediated activation of PKM2</span></div><div class="casAuthors">Liang, Yingjian; Zhang, Dandan; Zheng, Tongsen; Yang, Guangchao; Wang, Jiabei; Meng, Fanzheng; Liu, Yao; Zhang, Guoli; Zhang, Linhan; Han, Jihua; Hui, Peng; Chen, Zhengliang; Liu, Yu; Wang, Mingyu; Jiang, Hongchi; Liu, Lianxin</div><div class="citationInfo"><span class="NLM_cas:title">Oncogenesis</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">54</span>CODEN:
                <span class="NLM_cas:coden">ONCOCT</span>;
        ISSN:<span class="NLM_cas:issn">2157-9024</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Abstr.: Tumor cells primarily utilize aerobic glycolysis for energy prodn., a phenomenon known as the Warburg effect, but the involvement of Warburg effect in liver cancer cell metastasis is not well understood.  In present study, our results indicate a pos. correlation between glucose metab. level and metastatic potential of hepatocellular carcinoma (HCC).  We also obsd. that a long noncoding RNA-SOX2OT (lncRNA-SOX2OT) can not only increase the metastatic potential of HCC but also promote a pyruvate kinase M2 (PKM2)-mediated activation of glucose metab.  Inhibition of PKM2 in HCC cells greatly compromises lncRNA-SOX2OT in promoting Warburg effect and metastasis.  Furthermore, miR-122-5p was found being a direct target of lncRNA-SOX2OT in regulating PKM2 expression.  Thus, our findings reveal that lncRNA-SOX2OT, a regulator of PKM2, could predispose HCC patients to metastases and may serve as a candidate for metastatic prediction and therapies in HCC patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHZu-8Nq0s_LVg90H21EOLACvtfcHk0lipBJg0Hydzyw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtVyqtbzP&md5=c8df62ffe1784cf2147e8c1cbabca252</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1038%2Fs41389-020-0242-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41389-020-0242-z%26sid%3Dliteratum%253Aachs%26aulast%3DLiang%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DD.%26aulast%3DZheng%26aufirst%3DT.%26aulast%3DYang%26aufirst%3DG.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DMeng%26aufirst%3DF.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DHan%26aufirst%3DJ.%26aulast%3DHui%26aufirst%3DP.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DL.%26atitle%3DlncRNA-SOX2OT%2520promotes%2520hepatocellular%2520carcinoma%2520invasion%2520and%2520metastasis%2520through%2520miR-122%25E2%2580%25935p-mediated%2520activation%2520of%2520PKM2%26jtitle%3DOncogenesis%26date%3D2020%26volume%3D9%26spage%3D54%26epage%3D65%26doi%3D10.1038%2Fs41389-020-0242-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, X.</span></span> <span> </span><span class="NLM_article-title">Long non-coding RNA SNHG5 promotes human hepatocellular carcinoma progression by regulating miR-26a-5p/GSK3beta signal pathway</span>. <i>Cell Death Dis.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">888</span>, <span class="refDoi"> DOI: 10.1038/s41419-018-0882-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1038%2Fs41419-018-0882-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=30166525" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A280%3ADC%252BB3c3jvFGlsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=888&author=Y.+Liauthor=D.+Guoauthor=Y.+Zhaoauthor=M.+Renauthor=G.+Luauthor=Y.+Wangauthor=J.+Zhangauthor=C.+Miauthor=S.+Heauthor=X.+Lu&title=Long+non-coding+RNA+SNHG5+promotes+human+hepatocellular+carcinoma+progression+by+regulating+miR-26a-5p%2FGSK3beta+signal+pathway&doi=10.1038%2Fs41419-018-0882-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Long non-coding RNA SNHG5 promotes human hepatocellular carcinoma progression by regulating miR-26a-5p/GSK3β signal pathway</span></div><div class="casAuthors">Li Yarui; Guo Dan; Zhao Yan; Ren Mudan; Lu Guifang; Wang Yun; Zhang Juan; Mi Chen; He Shuixiang; Lu Xinlan</div><div class="citationInfo"><span class="NLM_cas:title">Cell death & disease</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">888</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Accumulating evidence have suggested that long non-coding RNAs (lncRNAs) had malfunctioning roles in the development of human cancers.  The present study aimed to investigate the role of lncRNA small nucleolar RNA host gene 5 (SNHG5) in hepatocellular carcinoma (HCC) progression using human tissues and cell lines.  The quantitative real-time PCR results showed that SNHG5 was up-regulated in both HCC tissues and hepatoma cell lines and was closely associated with tumor size, hepatitis B virus infection, histologic grade, TNM stage, and portal vein tumor thrombus (PVTT) in HCC patients.  Knockdown of SNHG5 induced apoptosis and repressed cell cycle progression, cell growth, and metastasis in hepatoma cell lines, whereas overexpression of SNHG5 had the opposite effects.  In vivo functional assay, xenograft tumors grown from SNHG5-knockdown cells had smaller mean volumes than the tumors grown from negative control cells.  Further investigations showed that SNHG5 may act as a competing endogenous RNA by competitively binding miR-26a-5p and thereby modulating the derepression of downstream target GSK3β, which were further confirmed by luciferase reporter assay.  Functionally, SNHG5 promotes tumor growth and metastasis by activating Wnt/β-catenin pathway and inducing epithelial to mesenchymal transition (EMT).  Taken together, SNHG5 promotes HCC progression by competitively binding miR-26a-5p and regulating GSK3β and Wnt/β-catenin signal pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTimSCl3vJjCO3MPKNEWB5RfW6udTcc2eaBL96zKKjUwbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3c3jvFGlsw%253D%253D&md5=6cf876288e32a7dcbef3ac5758b9f20e</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1038%2Fs41419-018-0882-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41419-018-0882-5%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DY.%26aulast%3DGuo%26aufirst%3DD.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DRen%26aufirst%3DM.%26aulast%3DLu%26aufirst%3DG.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DMi%26aufirst%3DC.%26aulast%3DHe%26aufirst%3DS.%26aulast%3DLu%26aufirst%3DX.%26atitle%3DLong%2520non-coding%2520RNA%2520SNHG5%2520promotes%2520human%2520hepatocellular%2520carcinoma%2520progression%2520by%2520regulating%2520miR-26a-5p%252FGSK3beta%2520signal%2520pathway%26jtitle%3DCell%2520Death%2520Dis.%26date%3D2018%26volume%3D9%26spage%3D888%26doi%3D10.1038%2Fs41419-018-0882-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shih, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tien, Y. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeh, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yen, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsieh, S. Y.</span></span> <span> </span><span class="NLM_article-title">MicroRNA-214 downregulation contributes to tumor angiogenesis by inducing secretion of the hepatoma-derived growth factor in human hepatoma</span>. <i>J. Hepatol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">584</span>– <span class="NLM_lpage">591</span>, <span class="refDoi"> DOI: 10.1016/j.jhep.2012.04.031</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1016%2Fj.jhep.2012.04.031" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=22613005" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1ensr3F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2012&pages=584-591&author=T.+C.+Shihauthor=Y.+J.+Tienauthor=C.+J.+Wenauthor=T.+S.+Yehauthor=M.+C.+Yuauthor=C.+H.+Huangauthor=Y.+S.+Leeauthor=T.+C.+Yenauthor=S.+Y.+Hsieh&title=MicroRNA-214+downregulation+contributes+to+tumor+angiogenesis+by+inducing+secretion+of+the+hepatoma-derived+growth+factor+in+human+hepatoma&doi=10.1016%2Fj.jhep.2012.04.031"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">MicroRNA-214 downregulation contributes to tumor angiogenesis by inducing secretion of the hepatoma-derived growth factor in human hepatoma</span></div><div class="casAuthors">Shih, Tsung-Chieh; Tien, Yin-Jing; Wen, Chih-Jen; Yeh, Ta-Sen; Yu, Ming-Chin; Huang, Chia-Hao; Lee, Yun-Shien; Yen, Tzu-Chen; Hsieh, Sen-Yung</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Hepatology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">584-591</span>CODEN:
                <span class="NLM_cas:coden">JOHEEC</span>;
        ISSN:<span class="NLM_cas:issn">0168-8278</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Background & Aims: Unusual hypervascularity is a hallmark of human hepatocellular carcinoma (HCC).  Although microRNA-214 (miR-214) is upregulated in other human cancers, it is downregulated in HCC.  We elucidated the biol. and clin. significance of miR-214 downregulation in HCC.  Methods: MicroRNAs deregulated in HCC were identified using array-based microRNA profiling.  A luciferase reporter assay confirmed target assocn. between miR-214 and the hepatoma-derived growth factor (HDGF).  Tube formation and in vivo angiogenesis assays validated the roles of miR-214/HDGF in angiogenesis.  Results: miR-214 downregulation was assocd. with higher tumor recurrence and worse clin. outcomes.  Ectopic expression of miR-214 suppressed xenograft tumor growth and microvascularity of the tumors and their surrounding tissues.  The genes downregulated by ectopic expression of miR-214 were involved in the regulation of apoptosis, cell cycle, and angiogenesis.  Integrated anal. disclosed HDGF as a downstream target of miR-214.  Conditioned medium of HCC cells contained bioactivity to stimulate tube formation of human umbilical vein endothelial cells, which was abolished by pretreatment of the conditioned media with HDGF antibodies, suppression of HDGF expression or ectopic expression of miR-214 in the donor HCC cells.  The angiogenic activity of the conditioned media, lost by ectopic expression of miR-214 in the donor cells, was restored by supplementation with recombinant HDGF.  In vivo tumor angiogenesis assays showed significant suppression of tumor vascularity by ectopic expression of miR-214.  Conclusions: A novel role of microRNA in tumorigenesis is identified.  Downregulation of miR-214 contributes to the unusual hypervascularity of HCC via activation of the HDGF paracrine pathway for tumor angiogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr9BJn4AWzRhLVg90H21EOLACvtfcHk0lgkNtxu--2QHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1ensr3F&md5=43a43fe1b4a0d42ce377c1682eae6686</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2Fj.jhep.2012.04.031&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jhep.2012.04.031%26sid%3Dliteratum%253Aachs%26aulast%3DShih%26aufirst%3DT.%2BC.%26aulast%3DTien%26aufirst%3DY.%2BJ.%26aulast%3DWen%26aufirst%3DC.%2BJ.%26aulast%3DYeh%26aufirst%3DT.%2BS.%26aulast%3DYu%26aufirst%3DM.%2BC.%26aulast%3DHuang%26aufirst%3DC.%2BH.%26aulast%3DLee%26aufirst%3DY.%2BS.%26aulast%3DYen%26aufirst%3DT.%2BC.%26aulast%3DHsieh%26aufirst%3DS.%2BY.%26atitle%3DMicroRNA-214%2520downregulation%2520contributes%2520to%2520tumor%2520angiogenesis%2520by%2520inducing%2520secretion%2520of%2520the%2520hepatoma-derived%2520growth%2520factor%2520in%2520human%2520hepatoma%26jtitle%3DJ.%2520Hepatol.%26date%3D2012%26volume%3D57%26spage%3D584%26epage%3D591%26doi%3D10.1016%2Fj.jhep.2012.04.031" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tian, X. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, X. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, F. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yun, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, R. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, Q. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, D.</span></span> <span> </span><span class="NLM_article-title">Acidic microenvironment Up-regulates exosomal miR-21 and miR-10b in early-stage hepatocellular carcinoma to promote cancer cell proliferation and metastasis</span>. <i>Theranostics</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1965</span>– <span class="NLM_lpage">1979</span>, <span class="refDoi"> DOI: 10.7150/thno.30958</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.7150%2Fthno.30958" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=31037150" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtlGru7bN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2019&pages=1965-1979&author=X.+P.+Tianauthor=C.+Y.+Wangauthor=X.+H.+Jinauthor=M.+Liauthor=F.+W.+Wangauthor=W.+J.+Huangauthor=J.+P.+Yunauthor=R.+H.+Xuauthor=Q.+Q.+Caiauthor=D.+Xie&title=Acidic+microenvironment+Up-regulates+exosomal+miR-21+and+miR-10b+in+early-stage+hepatocellular+carcinoma+to+promote+cancer+cell+proliferation+and+metastasis&doi=10.7150%2Fthno.30958"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Acidic Microenvironment Up-Regulates Exosomal miR-21 and miR-10b in Early-Stage Hepatocellular Carcinoma to Promote Cancer Cell Proliferation and Metastasis</span></div><div class="casAuthors">Tian, Xiao-Peng; Wang, Chen-Yuan; Jin, Xiao-Han; Li, Mei; Wang, Feng-Wei; Huang, Wei-Juan; Yun, Jing-Ping; Xu, Rui-Hua; Cai, Qing-Qing; Xie, Dan</div><div class="citationInfo"><span class="NLM_cas:title">Theranostics</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1965-1979</span>CODEN:
                <span class="NLM_cas:coden">THERDS</span>;
        ISSN:<span class="NLM_cas:issn">1838-7640</span>.
    
            (<span class="NLM_cas:orgname">Ivyspring International Publisher</span>)
        </div><div class="casAbstract">The incidence of hepatocellular carcinoma is rising worldwide.  It is predicted that nearly half of the early-stage hepatocellular carcinoma (E-HCC) patients will develop recurrence.  Dysregulated pH, a hallmark of E-HCC, is correlated with poor prognosis.  The acidic microenvironment has been shown to promote the release of exosomes, the membrane vesicles recognized as intercellular communicators assocd. with tumor progression, recurrence, and metastasis.  To identify exosomes induced by acidic microenvironment that may regulate E-HCC progression and to explore their mechanisms and clin. significance in E-HCCs.  The miRNA microarray anal. and LASSO logistic statistic model were used to identify the main functional exosomal miRNAs.  Invasion and scratch assays were performed to examine the migration and invasion of HCC cells.  Immunoblotting and immunofluorescence were employed to detect the epithelial-to-mesenchymal transition (EMT) in HCC cells.  Chromatin immunopptn. (ChIP) was used to analyze the binding of HIF-1a and HIF-2a to promoter regions of miR-21 and miR-10b.  The acidic microenvironment in HCC was correlated with poor prognosis of patients.  Exosomes from HCC cells cultured in the acidic medium could promote cell proliferation, migration, and invasion of recipient HCC cells.  The miR-21 and miR-10b as the most important functional miRNAs in acidic HCC-derived exosomes.  Also, the acidic microenvironment triggered the activation of HIF-1α and HIF-2α and stimulated exosomal miR-21 and miR-10b expression substantially promoting HCC cell proliferation, migration, and invasion both in vivo and in vitro.  In E-HCC patients, serum exosomal miR-21 and miR-10b levels were assocd. with advanced tumor stage and HIF-1a and HIF-2a expression and were independent prognostic factors for disease-free survival of E-HCC patients.  Most importantly, we developed a nano-drug to target exosomal miR-21 and/or miR-10b and examd. its therapeutic effects against HCC in vivo.  The findings suggested that the exosomal miR-21 and miR-10b induced by acidic microenvironment in HCC promote cancer cell proliferation and metastasis and may serve as prognostic mol. markers and therapeutic targets for HCC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVgyh3Piatf7Vg90H21EOLACvtfcHk0lgkNtxu--2QHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtlGru7bN&md5=3d0156b7944e2f5063f43ca531c7341b</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.7150%2Fthno.30958&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7150%252Fthno.30958%26sid%3Dliteratum%253Aachs%26aulast%3DTian%26aufirst%3DX.%2BP.%26aulast%3DWang%26aufirst%3DC.%2BY.%26aulast%3DJin%26aufirst%3DX.%2BH.%26aulast%3DLi%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DF.%2BW.%26aulast%3DHuang%26aufirst%3DW.%2BJ.%26aulast%3DYun%26aufirst%3DJ.%2BP.%26aulast%3DXu%26aufirst%3DR.%2BH.%26aulast%3DCai%26aufirst%3DQ.%2BQ.%26aulast%3DXie%26aufirst%3DD.%26atitle%3DAcidic%2520microenvironment%2520Up-regulates%2520exosomal%2520miR-21%2520and%2520miR-10b%2520in%2520early-stage%2520hepatocellular%2520carcinoma%2520to%2520promote%2520cancer%2520cell%2520proliferation%2520and%2520metastasis%26jtitle%3DTheranostics%26date%3D2019%26volume%3D9%26spage%3D1965%26epage%3D1979%26doi%3D10.7150%2Fthno.30958" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hasengbayi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, W.</span></span> <span> </span><span class="NLM_article-title">Mir-106b-5p Promotes Aggressive Progression of Hepatocellular Carcinoma Via Targeting Runx3</span>. <i>Cancer Med.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">6756</span>– <span class="NLM_lpage">6767</span>, <span class="refDoi"> DOI: 10.1002/cam4.2511</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1002%2Fcam4.2511" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=31503422" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitFGgsb%252FJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2019&pages=6756-6767&author=H.+Guauthor=S.+Guauthor=X.+Zhangauthor=S.+Zhangauthor=D.+Zhangauthor=J.+Linauthor=S.+Hasengbayiauthor=W.+Han&title=Mir-106b-5p+Promotes+Aggressive+Progression+of+Hepatocellular+Carcinoma+Via+Targeting+Runx3&doi=10.1002%2Fcam4.2511"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">MiR-106b-5p promotes aggressive progression of hepatocellular carcinoma via targeting RUNX3</span></div><div class="casAuthors">Gu, Hao; Gu, Shensen; Zhang, Xinlong; Zhang, Songjiang; Zhang, Dongming; Lin, Junsheng; Hasengbayi, Saiken; Han, Wei</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Medicine</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">6756-6767</span>CODEN:
                <span class="NLM_cas:coden">CMAEDL</span>;
        ISSN:<span class="NLM_cas:issn">2045-7634</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons Ltd.</span>)
        </div><div class="casAbstract">Background and Objectives : The roles of microRNA(miR)-106b-5p in hepatocellular carcinoma (HCC) remain unclear.  We aimed here to investigate the clin. significance of miR-106b-5p expression in HCC and its underlying mechanisms.  Methods : Expression levels of miR-106b-5p in 108 HCC clin. samples by quant. real-time reverse transcription PCR.  Assocns. of miR-106b-5p expression with various clinicopathol. features and patients' prognosis were evaluated by Chi-square test, Kaplan-Meier, and Cox proportional regression analyses, resp.  The target gene of miR-106b-5p and their functions in HCC cells were investigated by luciferase reporter, CCK-8, and Transwell Matrigel invasion assays.  Results : miR-106b-5p expression was markedly higher in HCC tissues than in noncancerous adjacent liver tissues (P < .001). miR-106b-5p upregulation was significantly assocd. with advanced TNM stage (P = .02), short recurrence-free (P = .005), and overall (P = .001) survivals.  Importantly, miR-106b-5p expression was an independent predictor of poor prognosis (P < .05).  RUNX3 was identified as a direct target gene of miR-106b-5p in HCC cells.  Functionally, miR-106b-5p upregulation promoted the viability and invasion of HCC cells, while enforced RUNX3 expression reversed the oncogenic effects of miR-106b-5p overexpression.  Conclusions : miR-106b-5p may serve as a potent prognostic marker for tumor recurrence and survival of HCC patients. miR-106b-5p may exert an oncogenic role in HCC via regulating its target gene RUNX3.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp8Ry_4fNQyu7Vg90H21EOLACvtfcHk0lgkNtxu--2QHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitFGgsb%252FJ&md5=89e217855a7f0d581ecb08a8f6e2765d</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1002%2Fcam4.2511&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcam4.2511%26sid%3Dliteratum%253Aachs%26aulast%3DGu%26aufirst%3DH.%26aulast%3DGu%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DD.%26aulast%3DLin%26aufirst%3DJ.%26aulast%3DHasengbayi%26aufirst%3DS.%26aulast%3DHan%26aufirst%3DW.%26atitle%3DMir-106b-5p%2520Promotes%2520Aggressive%2520Progression%2520of%2520Hepatocellular%2520Carcinoma%2520Via%2520Targeting%2520Runx3%26jtitle%3DCancer%2520Med.%26date%3D2019%26volume%3D8%26spage%3D6756%26epage%3D6767%26doi%3D10.1002%2Fcam4.2511" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Scisciani, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vossio, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guerrieri, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schinzari, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Iaco, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">D’Onorio
de Meo, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cervello, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montalto, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollicino, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raimondo, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levrero, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pediconi, N.</span></span> <span> </span><span class="NLM_article-title">Transcriptional regulation of miR-224 upregulated in human HCCs by NFkappaB inflammatory pathways</span>. <i>J. Hepatol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">855</span>– <span class="NLM_lpage">861</span>, <span class="refDoi"> DOI: 10.1016/j.jhep.2011.11.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1016%2Fj.jhep.2011.11.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=22178270" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BC38Xktlaqt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2012&pages=855-861&author=C.+Sciscianiauthor=S.+Vossioauthor=F.+Guerrieriauthor=V.+Schinzariauthor=R.+De+Iacoauthor=P.+D%E2%80%99Onorio%0Ade+Meoauthor=M.+Cervelloauthor=G.+Montaltoauthor=T.+Pollicinoauthor=G.+Raimondoauthor=M.+Levreroauthor=N.+Pediconi&title=Transcriptional+regulation+of+miR-224+upregulated+in+human+HCCs+by+NFkappaB+inflammatory+pathways&doi=10.1016%2Fj.jhep.2011.11.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Transcriptional regulation of miR-224 upregulated in human HCCs by NFκB inflammatory pathways</span></div><div class="casAuthors">Scisciani, Cecilia; Vossio, Stefania; Guerrieri, Francesca; Schinzari, Valeria; De Iaco, Rossana; D'Onorio de Meo, Paolo; Cervello, Melchiorre; Montalto, Giuseppe; Pollicino, Teresa; Raimondo, Giovanni; Levrero, Massimo; Pediconi, Natalia</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Hepatology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">855-861</span>CODEN:
                <span class="NLM_cas:coden">JOHEEC</span>;
        ISSN:<span class="NLM_cas:issn">0168-8278</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Background & Aims: miR-224 is up-regulated in human HCCs as compared to both paired peri-tumoral cirrhotic tissues and cirrhotic livers without HCC.  Here, we have cloned the miR-224 regulatory region and characterized its transcriptional regulation by the NFκB-dependent inflammatory pathways.  Methods: Mature miRNA expression was evaluated by a 2 step stem-loop real-time RT-PCR.  The recruitment of polymerase II and transcription factors on the pre-miR-224 promoter has been assessed by ChIPSeq and ChIP.  Results: We found miR-224 levels strongly up-regulated in both peri-tumoral cirrhotic livers and HCC samples as compared to normal livers.  In silico anal. of the putative miR-224 promoter revealed multiple NFκB sites.  We showed that LTα and TNFα activate transcription from the miR-224 promoter and of endogenous miR-224 expression in HCC cell lines, whereas the expression of miR-224 target API5 was reduced.  Exogenously expressed p65/RelA activates the miR-224 promoter and a dominant neg. form of IκBα (IκBSR) represses it.  ChIP anal. showed that p65/NFκB is recruited on the miR-224 promoter and that its binding sharply increases after exposure to LPS, TNFα, and LTα.  Altogether these findings link the inflammatory signals to NFκB-mediated activation of miR-224 expression.  An antago-miR specific for miR-224 blocked LPS and LTα stimulated HCC cells migration and invasion.  Conversely, the IKK inhibitor BMS-345541 blocks pre-miR-224-induced cellular migration and invasion.  Conclusions: Our results identify p65/NFκB as a direct transcriptional regulator of miR-224 expression and link miR-224 up-regulation with the activation of the LPS, LTα, and TNFα inflammatory pathways and cell migration/invasion in HCC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZufINPVyE0bVg90H21EOLACvtfcHk0lhzWrJV-2pFIw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xktlaqt70%253D&md5=0e9842de1df58d1a7b0a9671ee1a7b8b</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2Fj.jhep.2011.11.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jhep.2011.11.017%26sid%3Dliteratum%253Aachs%26aulast%3DScisciani%26aufirst%3DC.%26aulast%3DVossio%26aufirst%3DS.%26aulast%3DGuerrieri%26aufirst%3DF.%26aulast%3DSchinzari%26aufirst%3DV.%26aulast%3DDe%2BIaco%26aufirst%3DR.%26aulast%3DD%25E2%2580%2599Onorio%2Bde%2BMeo%26aufirst%3DP.%26aulast%3DCervello%26aufirst%3DM.%26aulast%3DMontalto%26aufirst%3DG.%26aulast%3DPollicino%26aufirst%3DT.%26aulast%3DRaimondo%26aufirst%3DG.%26aulast%3DLevrero%26aufirst%3DM.%26aulast%3DPediconi%26aufirst%3DN.%26atitle%3DTranscriptional%2520regulation%2520of%2520miR-224%2520upregulated%2520in%2520human%2520HCCs%2520by%2520NFkappaB%2520inflammatory%2520pathways%26jtitle%3DJ.%2520Hepatol.%26date%3D2012%26volume%3D56%26spage%3D855%26epage%3D861%26doi%3D10.1016%2Fj.jhep.2011.11.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fu, X. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, Y. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, Z. B.</span></span> <span> </span><span class="NLM_article-title">MicroRNA-30a suppresses autophagy-mediated anoikis resistance and metastasis in hepatocellular carcinoma</span>. <i>Cancer Lett.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>412</i></span>,  <span class="NLM_fpage">108</span>– <span class="NLM_lpage">117</span>, <span class="refDoi"> DOI: 10.1016/j.canlet.2017.10.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1016%2Fj.canlet.2017.10.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=29061507" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhslagtbfE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=412&publication_year=2018&pages=108-117&author=X.+T.+Fuauthor=Y.+H.+Shiauthor=J.+Zhouauthor=Y.+F.+Pengauthor=W.+R.+Liuauthor=G.+M.+Shiauthor=Q.+Gaoauthor=X.+Y.+Wangauthor=K.+Songauthor=J.+Fanauthor=Z.+B.+Ding&title=MicroRNA-30a+suppresses+autophagy-mediated+anoikis+resistance+and+metastasis+in+hepatocellular+carcinoma&doi=10.1016%2Fj.canlet.2017.10.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">MicroRNA-30a suppresses autophagy-mediated anoikis resistance and metastasis in hepatocellular carcinoma</span></div><div class="casAuthors">Fu, Xiu-Tao; Shi, Ying-Hong; Zhou, Jian; Peng, Yuan-Fei; Liu, Wei-Ren; Shi, Guo-Ming; Gao, Qiang; Wang, Xiao-Ying; Song, Kang; Fan, Jia; Ding, Zhen-Bin</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Letters (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">412</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">108-117</span>CODEN:
                <span class="NLM_cas:coden">CALEDQ</span>;
        ISSN:<span class="NLM_cas:issn">0304-3835</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">MiRNA-30a (miR-30a) was previously reported as one of metastatic hepatocellular carcinoma (HCC)-related microRNAs.  However, the function of miR-30a on enhancing our biol. understanding of HCC metastasis is not clear.  This study demonstrated that miR-30a was significantly down-regulated in HCC tissues and cell lines, and was assocd. with vascular invasion, metastasis potential and recurrent disease in HCC.  Functional studies confirmed that miR-30a could inhibit the metastasis of HCC in a well-established nude mouse model of lung metastasis.  Moreover, miR-30a was proved to prevent anoikis inhibition of HCC cells in vivo and in vitro.  Mech., autophagy related protein Beclin 1 and Atg5 were direct downstream targets of miR-30a, and mediated autophagy activity influence of miR-30a in HCC.  Taken together, downregulated miR-30a in metastatic HCC mediates Beclin 1 and Atg5-dependent autophagy, which confers anoikis resistance in HCC cells.  The mol. basis of autophagy action during this process partly contributes to the HCC metastasis, suggesting that targeting autophagy via miR-30a may have therapeutic implications for the prevention of HCC recurrence/metastasis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBP1M-OheQVrVg90H21EOLACvtfcHk0lhzWrJV-2pFIw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhslagtbfE&md5=633717b5e6c40b158dbb6ef1f7db7307</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2Fj.canlet.2017.10.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.canlet.2017.10.012%26sid%3Dliteratum%253Aachs%26aulast%3DFu%26aufirst%3DX.%2BT.%26aulast%3DShi%26aufirst%3DY.%2BH.%26aulast%3DZhou%26aufirst%3DJ.%26aulast%3DPeng%26aufirst%3DY.%2BF.%26aulast%3DLiu%26aufirst%3DW.%2BR.%26aulast%3DShi%26aufirst%3DG.%2BM.%26aulast%3DGao%26aufirst%3DQ.%26aulast%3DWang%26aufirst%3DX.%2BY.%26aulast%3DSong%26aufirst%3DK.%26aulast%3DFan%26aufirst%3DJ.%26aulast%3DDing%26aufirst%3DZ.%2BB.%26atitle%3DMicroRNA-30a%2520suppresses%2520autophagy-mediated%2520anoikis%2520resistance%2520and%2520metastasis%2520in%2520hepatocellular%2520carcinoma%26jtitle%3DCancer%2520Lett.%26date%3D2018%26volume%3D412%26spage%3D108%26epage%3D117%26doi%3D10.1016%2Fj.canlet.2017.10.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ding, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duan, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xia, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, G.</span></span> <span> </span><span class="NLM_article-title">SSRP1 contributes to the malignancy of hepatocellular carcinoma and is negatively regulated by miR-497</span>. <i>Mol. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">903</span>– <span class="NLM_lpage">914</span>, <span class="refDoi"> DOI: 10.1038/mt.2016.9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1038%2Fmt.2016.9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=26755331" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BC28Xit1altrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2016&pages=903-914&author=Q.+Dingauthor=K.+Heauthor=T.+Luoauthor=Y.+Dengauthor=H.+Wangauthor=H.+Liuauthor=J.+Zhangauthor=K.+Chenauthor=J.+Xiaoauthor=X.+Duanauthor=R.+Huangauthor=Z.+Xiaauthor=W.+Zhouauthor=J.+Heauthor=H.+Yuauthor=X.+Jiaoauthor=G.+Xiang&title=SSRP1+contributes+to+the+malignancy+of+hepatocellular+carcinoma+and+is+negatively+regulated+by+miR-497&doi=10.1038%2Fmt.2016.9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">SSRP1 Contributes to the Malignancy of Hepatocellular Carcinoma and Is Negatively Regulated by miR-497</span></div><div class="casAuthors">Ding, Qianshan; He, Ke; Luo, Tao; Deng, Yunchao; Wang, Hanning; Liu, Hao; Zhang, Jinqian; Chen, Kaiyun; Xiao, Jinfeng; Duan, Xiaopeng; Huang, Rui; Xia, Zhenglin; Zhou, Wenjie; He, Jinliang; Yu, Honggang; Jiao, Xingyuan; Xiang, Guoan</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Therapy</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">903-914</span>CODEN:
                <span class="NLM_cas:coden">MTOHCK</span>;
        ISSN:<span class="NLM_cas:issn">1525-0016</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The aim of this study is to clarify the clin. implication and functional role of structure specific recognition protein 1 (SSRP1) in hepatocellular carcinoma (HCC) and explore the underlying mechanism of aberrant high expression of SSRP1 in cancers.  In the present investigation, we validated that SSRP1 was upregulated in HCC samples.  We also demonstrated that its upregulation was assocd. with several clinicopathol. features such as higher serum AFP level, larger tumor size, and higher T stage of HCC patients; and its high expression indicated shorter overall survival and faster recurrence.  To investigate the role of SSRP1 in HCC progression, both loss- and gain-function models were established.  We demonstrated that SSPR1 modulated both proliferation and metastasis of HCC cells in vitro and vivo.  Furthermore, we demonstrated that SSRP1-modulated apoptosis process and its knockdown increased the sensitivity of HCC cells to doxorubicin, 5-Fluorouracil, and cisplatin.  We also identified microRNA-497 (miR-497) as a posttranscriptional regulator of SSRP1.  Ectopic expression of miR-497 inhibited 3'-untranslated-region-coupled luciferase activity and suppressed endogenous SSRP1 expression at both mRNA and protein levels.  For the first time, we proved that SSRP1 upregulation contributed to HCC development and the tumor-suppressive miR-497 served as its neg. regulator.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbZ9DcAIqU0LVg90H21EOLACvtfcHk0li5Bbm44Hu_zg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xit1altrg%253D&md5=e73f7b7ab114825692a5fdd44bd46855</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1038%2Fmt.2016.9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fmt.2016.9%26sid%3Dliteratum%253Aachs%26aulast%3DDing%26aufirst%3DQ.%26aulast%3DHe%26aufirst%3DK.%26aulast%3DLuo%26aufirst%3DT.%26aulast%3DDeng%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DK.%26aulast%3DXiao%26aufirst%3DJ.%26aulast%3DDuan%26aufirst%3DX.%26aulast%3DHuang%26aufirst%3DR.%26aulast%3DXia%26aufirst%3DZ.%26aulast%3DZhou%26aufirst%3DW.%26aulast%3DHe%26aufirst%3DJ.%26aulast%3DYu%26aufirst%3DH.%26aulast%3DJiao%26aufirst%3DX.%26aulast%3DXiang%26aufirst%3DG.%26atitle%3DSSRP1%2520contributes%2520to%2520the%2520malignancy%2520of%2520hepatocellular%2520carcinoma%2520and%2520is%2520negatively%2520regulated%2520by%2520miR-497%26jtitle%3DMol.%2520Ther.%26date%3D2016%26volume%3D24%26spage%3D903%26epage%3D914%26doi%3D10.1038%2Fmt.2016.9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, X. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z. N.</span></span> <span> </span><span class="NLM_article-title">microRNA-146a inhibits cancer metastasis by downregulating VEGF through dual pathways in hepatocellular carcinoma</span>. <i>Mol. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">5</span>– <span class="NLM_lpage">19</span>, <span class="refDoi"> DOI: 10.1186/1476-4598-14-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1186%2F1476-4598-14-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=25608619" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BC2MXkt1ejtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2015&pages=5-19&author=Z.+Zhangauthor=Y.+Zhangauthor=X.+X.+Sunauthor=X.+Maauthor=Z.+N.+Chen&title=microRNA-146a+inhibits+cancer+metastasis+by+downregulating+VEGF+through+dual+pathways+in+hepatocellular+carcinoma&doi=10.1186%2F1476-4598-14-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">microRNA-146a inhibits cancer metastasis by downregulating VEGF through dual pathways in hepatocellular carcinoma</span></div><div class="casAuthors">Zhang, Zheng; Zhang, Yang; Sun, Xiu-Xuan; Ma, Xi; Chen, Zhi-Nan</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">5</span>CODEN:
                <span class="NLM_cas:coden">MCOACG</span>;
        ISSN:<span class="NLM_cas:issn">1476-4598</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">Growing evidence indicates that miR-146a is involved in carcinogenesis and tumor progression in several human malignancies.  However, the mol. details underlying miR-146a mediated regulation of its target genes and its precise biol. function in cancer, esp. in hepatocellular carcinoma (HCC) remains unclear.  Methods: The expression levels of genes including miR-146a, APC, VEGF and HAb18G were examd. in HCC cell lines and patient specimens were compared with control levels using quant. reverse transcription-PCR.  The functions of miR-146a and HAb18G in migration/invasion and liver metastasis formation were detd. by transwell and spleen injection assays, resp. miR-146a related genes were detd. by PCR array.  The potential regulatory targets of miR-146a were detd. by bioinformatics and prediction tools, correlation with target protein expression, and luciferase reporter assay.  DNA methylation status of miR-146a promoter were performed by PCR anal. of bisulfite-modified genomic DNA.  Results: We demonstrated that miR-146a expression was markedly downregulated in hepatoma cells and hepatoma tissues compared to immortalized normal liver epithelial cells and normal hepatic tissues.  DNA methylation of miR-146a promoter correlated with its downexpression and with liver cancer metastasis.  The restoration of miR-146a dramatically suppressed HCC cell invasion and metastasis by repressing VEGF expression through upregulating APC, which inhibits β-catenin accumulation in nucleus, and downregulating NF-κB p65 by targeting HAb18G.  In human HCC, miR-146a expression was neg. correlated with increased HAb18G, VEGF, NF-κB p65 and beneficial prognosis.  Conclusion: This study identified a novel target of miR-146a and defined miR-146a as a crucial tumor suppressor in human HCC that acts through multiple pathways and mechanisms to suppress HCC invasion or metastasis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGopbG7clEzdIbVg90H21EOLACvtfcHk0li5Bbm44Hu_zg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXkt1ejtLs%253D&md5=1042a1f24b5cb5af954b8658d66f1e0c</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1186%2F1476-4598-14-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1476-4598-14-5%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DSun%26aufirst%3DX.%2BX.%26aulast%3DMa%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DZ.%2BN.%26atitle%3DmicroRNA-146a%2520inhibits%2520cancer%2520metastasis%2520by%2520downregulating%2520VEGF%2520through%2520dual%2520pathways%2520in%2520hepatocellular%2520carcinoma%26jtitle%3DMol.%2520Cancer%26date%3D2015%26volume%3D14%26spage%3D5%26epage%3D19%26doi%3D10.1186%2F1476-4598-14-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Z. M.</span></span> <span> </span><span class="NLM_article-title">Suppression of KIF14 expression inhibits hepatocellular carcinoma progression and predicts favorable outcome</span>. <i>Cancer Sci.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>104</i></span>,  <span class="NLM_fpage">552</span>– <span class="NLM_lpage">557</span>, <span class="refDoi"> DOI: 10.1111/cas.12128</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1111%2Fcas.12128" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=23414349" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BC3sXlvVCjsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=104&publication_year=2013&pages=552-557&author=T.+Yangauthor=X.+B.+Zhangauthor=Z.+M.+Zheng&title=Suppression+of+KIF14+expression+inhibits+hepatocellular+carcinoma+progression+and+predicts+favorable+outcome&doi=10.1111%2Fcas.12128"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Suppression of KIF14 expression inhibits hepatocellular carcinoma progression and predicts favorable outcome</span></div><div class="casAuthors">Yang, Tao; Zhang, Xian-Bo; Zheng, Zhi-Min</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Science</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">104</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">552-557</span>CODEN:
                <span class="NLM_cas:coden">CSACCM</span>;
        ISSN:<span class="NLM_cas:issn">1349-7006</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">The mitotic kinesin superfamily protein KIF14 is essential for cytokinesis and chromosome segregation and increased KIF14 expression is related to a variety of human cancers.  In this study, we investigate KIF14 expression in assocn. with clin. variables and the role of KIF14 during tumorigenesis.  We found that KIF14 is overexpressed in most primary hepatocellular carcinoma (HCC) tissues compared with the adjacent normal liver tissues and KIF14 overexpression is assocd. with tumor grade (P = 0.002), stage (P = 0.013) and poor survival (P < 0.001).  Downregulation of KIF14 decreased the capacity of proliferation both in vitro and in vivo.  Furthermore, suppression of KIF14 not only decreases cancer cell migration but also induces apoptosis of cells with inactivation of the phosphatidylinositol 3-kinase-Akt signaling pathway.  Therefore, our current study indicates that KIF14 promotes HCC carcinogenesis and may serve as a potential therapeutic target for human HCC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXbA_SnAQJHrVg90H21EOLACvtfcHk0li5Bbm44Hu_zg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXlvVCjsr8%253D&md5=0aab5cf484a0ee193fe972885808b10d</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1111%2Fcas.12128&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fcas.12128%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DT.%26aulast%3DZhang%26aufirst%3DX.%2BB.%26aulast%3DZheng%26aufirst%3DZ.%2BM.%26atitle%3DSuppression%2520of%2520KIF14%2520expression%2520inhibits%2520hepatocellular%2520carcinoma%2520progression%2520and%2520predicts%2520favorable%2520outcome%26jtitle%3DCancer%2520Sci.%26date%3D2013%26volume%3D104%26spage%3D552%26epage%3D557%26doi%3D10.1111%2Fcas.12128" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span> <span> </span><span class="NLM_article-title">CDK5-mediated phosphorylation and stabilization of TPX2 promotes hepatocellular tumorigenesis</span>. <i>J. Exp. Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">286</span>– <span class="NLM_lpage">300</span>, <span class="refDoi"> DOI: 10.1186/s13046-019-1297-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1186%2Fs13046-019-1297-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=31272499" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A280%3ADC%252BB3MzktVGqtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2019&pages=286-300&author=F.+Wangauthor=W.+Zhaoauthor=Y.+Gaoauthor=J.+Zhouauthor=H.+Liauthor=G.+Zhangauthor=D.+Guoauthor=C.+Xieauthor=J.+Liauthor=Z.+Yinauthor=J.+Zhang&title=CDK5-mediated+phosphorylation+and+stabilization+of+TPX2+promotes+hepatocellular+tumorigenesis&doi=10.1186%2Fs13046-019-1297-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">CDK5-mediated phosphorylation and stabilization of TPX2 promotes hepatocellular tumorigenesis</span></div><div class="casAuthors">Wang Fuqiang; Xie Chengrong; Li Jie; Yin Zhenyu; Zhao Wenxing; Gao Yuehong; Li Huifang; Zhang Guanyun; Guo Dong; Zhang Jie; Zhou Jiechao</div><div class="citationInfo"><span class="NLM_cas:title">Journal of experimental & clinical cancer research : CR</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">286</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  CDK5, an atypical member of the CDK family, play a significant role in the tumorigenesis of multiple organ, but CDK5 and its substrates in genesis and development of HCC is still unclear.  METHODS:  Expression of CDK5 in HCC tumor and paired adjacent noncancerous tissues from 90 patients were measured by Western blotting, immunohistochemistry, and real-time PCR.  The role of CDK5 in cell function and tumorigenesis was explored in HCC cell lines, ex vivo xenografts and diethylnitrosamine induced HCC model.  Furthermore, comparative phosphoproteomic screening identified the oncoprotein TPX2 as a new substrate of CDK5.  We also identified the effect of CDK5/P25 interaction blocker tamoxifen on HCC cell growth and migration.  RESULTS:  CDK5 was increased in HCC tisues and the level of CDK5 was correlated with the severity of HCC based on patient recurrence and 5-year fatality rate.  Exogenously expressed CDK5 but not kinase-dead CDK5 promoted proliferation, migration, and invasion of HCC cells.  Functional ablation of CDK5 significantly inhibited the exacerbation of HCC cells.  Xenograft implantation of HCC cells overexpressing CDK5 promoted tumorigenesis, and genetic knockdown of CDK5 reduced HCC growth and metastasis in vivo.  More importantly, heterozygous knockout CDK5 (Cdk5+/-) attenuated HCC tumorigenesis induced by diethylnitrosamine.  CDK5-mediated phosphorylation of TPX2 at serine 486 promoted its protein stability.  TPX2 silence could restore HCC cell migration capability with overexpression CDK5.  Treatment with tamoxifen inhibited cell growth and migration of HCC, demonstrating the role of active CDK5 in HCC.  CONCLUSIONS:  Our results suggest activation of CDK5 is associated with HCC tumorigenesis.  CDK5-mediated phosphorylation and stabilization of TPX2 promotes hepatocellular proliferation and tumorigenicity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSH7rKwqk-g40bxh97K6Vt6fW6udTcc2eZxbwK6_fPkMrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3MzktVGqtw%253D%253D&md5=63eb69a3b25860dc1d59fb962e9b74b8</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1186%2Fs13046-019-1297-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs13046-019-1297-6%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DF.%26aulast%3DZhao%26aufirst%3DW.%26aulast%3DGao%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DGuo%26aufirst%3DD.%26aulast%3DXie%26aufirst%3DC.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DYin%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DJ.%26atitle%3DCDK5-mediated%2520phosphorylation%2520and%2520stabilization%2520of%2520TPX2%2520promotes%2520hepatocellular%2520tumorigenesis%26jtitle%3DJ.%2520Exp.%2520Clin.%2520Cancer%2520Res.%26date%3D2019%26volume%3D38%26spage%3D286%26epage%3D300%26doi%3D10.1186%2Fs13046-019-1297-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, R. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yong, Y. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, C. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bian, H.</span></span> <span> </span><span class="NLM_article-title">UBE2S interacting with TRIM28 in the nucleus accelerates cell cycle by ubiquitination of p27 to promote hepatocellular carcinoma development</span>. <i>Signal Transduction Targeted Ther.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">64</span>– <span class="NLM_lpage">75</span>, <span class="refDoi"> DOI: 10.1038/s41392-020-00432-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1038%2Fs41392-020-00432-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=33589597" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BB3MXhtVOmsbzL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2021&pages=64-75&author=R.+Y.+Zhangauthor=Z.+K.+Liuauthor=D.+Weiauthor=Y.+L.+Yongauthor=P.+Linauthor=H.+Liauthor=M.+Liuauthor=N.+S.+Zhengauthor=K.+Liuauthor=C.+X.+Huauthor=X.+Z.+Yangauthor=Z.+N.+Chenauthor=H.+Bian&title=UBE2S+interacting+with+TRIM28+in+the+nucleus+accelerates+cell+cycle+by+ubiquitination+of+p27+to+promote+hepatocellular+carcinoma+development&doi=10.1038%2Fs41392-020-00432-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">UBE2S interacting with TRIM28 in the nucleus accelerates cell cycle by ubiquitination of p27 to promote hepatocellular carcinoma development</span></div><div class="casAuthors">Zhang, Ren-Yu; Liu, Ze-Kun; Wei, Ding; Yong, Yu-Le; Lin, Peng; Li, Hao; Liu, Man; Zheng, Nai-Shan; Liu, Ke; Hu, Cai-Xia; Yang, Xiao-Zhen; Chen, Zhi-Nan; Bian, Huijie</div><div class="citationInfo"><span class="NLM_cas:title">Signal Transduction and Targeted Therapy</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">64</span>CODEN:
                <span class="NLM_cas:coden">STTTCB</span>;
        ISSN:<span class="NLM_cas:issn">2059-3635</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Abstr.: Genomic sequencing anal. of tumors provides potential mol. therapeutic targets for precision medicine.  However, identifying a key driver gene or mutation that can be used for hepatocellular carcinoma (HCC) treatment remains difficult.  Here, we performed whole-exome sequencing on genomic DNA obtained from six pairs of HCC and adjacent tissues and identified two novel somatic mutations of UBE2S (p. Gly57Ala and p. Lys63Asn).  Predictions of the functional effects of the mutations showed that two amino-acid substitutions were potentially deleterious.  Further, we obsd. that wild-type UBE2S, esp. in the nucleus, was significantly higher in HCC tissues than that in adjacent tissues and closely related to the clinicopathol. features of patients with HCC.  Functional assays revealed that overexpression of UBE2S promoted the proliferation, invasion, metastasis, and G1/S phase transition of HCC cells in vitro, and promoted the tumor growth significantly in vivo.  Mechanistically, UBE2S interacted with TRIM28 in the nucleus, both together enhanced the ubiquitination of p27 to facilitate its degrdn. and cell cycle progression.  Most importantly, the small-mol. cephalomannine was found by a luciferase-based sensitive high-throughput screen (HTS) to inhibit UBE2S expression and significantly attenuate HCC progression in vitro and in vivo, which may represent a promising strategy for HCC therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr6vLVvO6SN5rVg90H21EOLACvtfcHk0lgQO7RheByRJA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXhtVOmsbzL&md5=0f2c9d1ab510ea6c8c6e450968b8071f</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1038%2Fs41392-020-00432-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41392-020-00432-z%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DR.%2BY.%26aulast%3DLiu%26aufirst%3DZ.%2BK.%26aulast%3DWei%26aufirst%3DD.%26aulast%3DYong%26aufirst%3DY.%2BL.%26aulast%3DLin%26aufirst%3DP.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DM.%26aulast%3DZheng%26aufirst%3DN.%2BS.%26aulast%3DLiu%26aufirst%3DK.%26aulast%3DHu%26aufirst%3DC.%2BX.%26aulast%3DYang%26aufirst%3DX.%2BZ.%26aulast%3DChen%26aufirst%3DZ.%2BN.%26aulast%3DBian%26aufirst%3DH.%26atitle%3DUBE2S%2520interacting%2520with%2520TRIM28%2520in%2520the%2520nucleus%2520accelerates%2520cell%2520cycle%2520by%2520ubiquitination%2520of%2520p27%2520to%2520promote%2520hepatocellular%2520carcinoma%2520development%26jtitle%3DSignal%2520Transduction%2520Targeted%2520Ther.%26date%3D2021%26volume%3D6%26spage%3D64%26epage%3D75%26doi%3D10.1038%2Fs41392-020-00432-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, W. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jia, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Q. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, L.</span></span> <span> </span><span class="NLM_article-title">APC(CDC20)-mediated degradation of PHD3 stabilizes HIF-1a and promotes tumorigenesis in hepatocellular carcinoma</span>. <i>Cancer Lett.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>496</i></span>,  <span class="NLM_fpage">144</span>– <span class="NLM_lpage">155</span>, <span class="refDoi"> DOI: 10.1016/j.canlet.2020.10.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1016%2Fj.canlet.2020.10.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=33039559" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BB3cXitFOisbfI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=496&publication_year=2021&pages=144-155&author=M.+Shiauthor=W.+Q.+Daiauthor=R.+R.+Jiaauthor=Q.+H.+Zhangauthor=J.+Weiauthor=Y.+G.+Wangauthor=S.+H.+Xiangauthor=B.+Liuauthor=L.+Xu&title=APC%28CDC20%29-mediated+degradation+of+PHD3+stabilizes+HIF-1a+and+promotes+tumorigenesis+in+hepatocellular+carcinoma&doi=10.1016%2Fj.canlet.2020.10.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">APCCDC20-mediated degradation of PHD3 stabilizes HIF-1a and promotes tumorigenesis in hepatocellular carcinoma</span></div><div class="casAuthors">Shi, Min; Dai, Wei-Qi; Jia, Rong-Rong; Zhang, Qing-Hui; Wei, Jue; Wang, Yu-Gang; Xiang, Shi-Hao; Liu, Bin; Xu, Ling</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Letters (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">496</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">144-155</span>CODEN:
                <span class="NLM_cas:coden">CALEDQ</span>;
        ISSN:<span class="NLM_cas:issn">0304-3835</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">CDC20 regulates cell cycle progression by targeting key substrates for destruction, but its role in hepatocellular carcinoma (HCC) tumorigenesis remains to be explored.  Here, by using weighted gene co-expression network anal. (WGCNA), we identified CDC20 as a hub gene in HCC.  We demonstrated that CDC20 expression is correlated with HIF-1 activity and overall survival (OS) of clinic HCC patients.  The activity of HIF-1 is regulated by the stability of HIF-1a subunit, which is hydroxylated by oxygen-dependent prolyl hydroxylase enzymes, the PHDs.  In addn., we show that genetic ablation or pharmacol. inhibition of CDC20 can accelerate the degrdn. of HIF-1a and impair VEGF secretion in HCC cells.  Mechanistically, we found that CDC20 binds to the destruction-box (D-box) motif present in the PHD3 protein to promote its polyubiquitination and degrdn.  The depletion of endogenous PHD3 in CDC20 knockdown HCC cells greatly attenuated the decline of HIF-1a protein and restored the secretion of VEGF.  In contrast, overexpression of a non-degradable PHD3 mutant significantly inhibited the proliferation of HCC cells both in vitro and in vivo.  Collectively, our findings indicate that CDC20 plays a crucial role in the development of HCC by governing PHD3 protein.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqLCvffuGn_jLVg90H21EOLACvtfcHk0lgQO7RheByRJA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXitFOisbfI&md5=4d27f17964d10161717851256dd29714</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1016%2Fj.canlet.2020.10.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.canlet.2020.10.011%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DM.%26aulast%3DDai%26aufirst%3DW.%2BQ.%26aulast%3DJia%26aufirst%3DR.%2BR.%26aulast%3DZhang%26aufirst%3DQ.%2BH.%26aulast%3DWei%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DY.%2BG.%26aulast%3DXiang%26aufirst%3DS.%2BH.%26aulast%3DLiu%26aufirst%3DB.%26aulast%3DXu%26aufirst%3DL.%26atitle%3DAPC%2528CDC20%2529-mediated%2520degradation%2520of%2520PHD3%2520stabilizes%2520HIF-1a%2520and%2520promotes%2520tumorigenesis%2520in%2520hepatocellular%2520carcinoma%26jtitle%3DCancer%2520Lett.%26date%3D2021%26volume%3D496%26spage%3D144%26epage%3D155%26doi%3D10.1016%2Fj.canlet.2020.10.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J. P.</span>; <span class="NLM_string-name">Sun, H. D.</span>; <span class="NLM_string-name">Zhang, H. B.</span>; <span class="NLM_string-name">Zheng, Y. T.</span>; <span class="NLM_string-name">Xiao, W. L.</span>; <span class="NLM_string-name">Pu, J. X.</span>; <span class="NLM_string-name">Wang, R. R.</span>; <span class="NLM_string-name">Yang, L. M.</span></span> <span> </span><span class="NLM_article-title">Nitrogen-containing biphenyl compounds as anti-Aids agents and their preparation, pharmaceutical compositions and use in the treatment of HIV-1 infection</span>. U.S. Patent US <span class="NLM_patent">20130331390</span>, <span class="NLM_year">2013</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&author=J.+P.+Liu&author=H.+D.+Sun&author=H.+B.+Zhang&author=Y.+T.+Zheng&author=W.+L.+Xiao&author=J.+X.+Pu&author=R.+R.+Wang&author=L.+M.+Yang&title=Nitrogen-containing+biphenyl+compounds+as+anti-Aids+agents+and+their+preparation%2C+pharmaceutical+compositions+and+use+in+the+treatment+of+HIV-1+infection"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DJ.%2BP.%26atitle%3DNitrogen-containing%2520biphenyl%2520compounds%2520as%2520anti-Aids%2520agents%2520and%2520their%2520preparation%252C%2520pharmaceutical%2520compositions%2520and%2520use%2520in%2520the%2520treatment%2520of%2520HIV-1%2520infection%26date%3D2013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abell, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mercado, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caneque, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguez, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xhemalce, B.</span></span> <span> </span><span class="NLM_article-title">Click quantitative mass spectrometry identifies PIWIL3 as a mechanistic target of RNA interference activator enoxacin in cancer cells</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>139</i></span>,  <span class="NLM_fpage">1400</span>– <span class="NLM_lpage">1403</span>, <span class="refDoi"> DOI: 10.1021/jacs.6b11751</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.6b11751" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVKhsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=139&publication_year=2017&pages=1400-1403&author=N.+S.+Abellauthor=M.+Mercadoauthor=T.+Canequeauthor=R.+Rodriguezauthor=B.+Xhemalce&title=Click+quantitative+mass+spectrometry+identifies+PIWIL3+as+a+mechanistic+target+of+RNA+interference+activator+enoxacin+in+cancer+cells&doi=10.1021%2Fjacs.6b11751"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Click Quantitative Mass Spectrometry Identifies PIWIL3 as a Mechanistic Target of RNA Interference Activator Enoxacin in Cancer Cells</span></div><div class="casAuthors">Abell, Nathan S.; Mercado, Marvin; Caneque, Tatiana; Rodriguez, Raphael; Xhemalce, Blerta</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">139</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1400-1403</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Enoxacin is a small mol. that stimulates RNA interference (RNAi) and acts as a growth inhibitor selectively in cancer but not in untransformed cells.  Here, we used alkenox, a clickable enoxacin surrogate, coupled with quant. mass spectrometry, to identify PIWIL3 as a mechanistic target of enoxacin.  PIWIL3 is an Argonaute protein of the PIWI subfamily that is mainly expressed in the germline and that mediates RNAi through piRNAs.  Our results suggest that cancer cells re-express PIWIL3 to repress RNAi through miRNAs and thus open a new opportunity for cancer-specific targeting.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJ5Cpc09nUzLVg90H21EOLACvtfcHk0lgAaZklt1JUZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVKhsrY%253D&md5=7dc812390e460f651676384bee9e8fbe</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Fjacs.6b11751&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.6b11751%26sid%3Dliteratum%253Aachs%26aulast%3DAbell%26aufirst%3DN.%2BS.%26aulast%3DMercado%26aufirst%3DM.%26aulast%3DCaneque%26aufirst%3DT.%26aulast%3DRodriguez%26aufirst%3DR.%26aulast%3DXhemalce%26aufirst%3DB.%26atitle%3DClick%2520quantitative%2520mass%2520spectrometry%2520identifies%2520PIWIL3%2520as%2520a%2520mechanistic%2520target%2520of%2520RNA%2520interference%2520activator%2520enoxacin%2520in%2520cancer%2520cells%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2017%26volume%3D139%26spage%3D1400%26epage%3D1403%26doi%3D10.1021%2Fjacs.6b11751" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, Z. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y. Z.</span></span> <span> </span><span class="NLM_article-title">TTD: Therapeutic Target Database</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">412</span>– <span class="NLM_lpage">415</span>, <span class="refDoi"> DOI: 10.1093/nar/30.1.412</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1093%2Fnar%2F30.1.412" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=11752352" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BD38Xht12rtrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2002&pages=412-415&author=X.+Chenauthor=Z.+L.+Jiauthor=Y.+Z.+Chen&title=TTD%3A+Therapeutic+Target+Database&doi=10.1093%2Fnar%2F30.1.412"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">TTD: Therapeutic Target Database</span></div><div class="casAuthors">Chen, X.; Ji, Z. L.; Chen, Y. Z.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">412-415</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">A no. of proteins and nucleic acids have been explored as therapeutic targets.  These targets are subjects of interest in different areas of biomedical and pharmaceutical research and in the development and evaluation of bioinformatics, mol. modeling, computer-aided drug design and anal. tools.  A publicly accessible database that provides comprehensive information about these targets is therefore helpful to the relevant communities.  The Therapeutic Target Database (TTD) is designed to provide information about the known therapeutic protein and nucleic acid targets described in the literature, the targeted disease conditions, the pathway information and the corresponding drugs/ligands directed at each of these targets.  Cross-links to other databases are also introduced to facilitate the access of information about the sequence, 3D structure, function, nomenclature, drug/ligand binding properties, drug usage and effects, and related literature for each target.  This database can be accessed at http://xin.cz3.nus.edu.sg/group/ttd/ttd.asp and it currently contains entries for 433 targets covering 125 disease conditions along with 809 drugs/ligands directed at each of these targets.  Each entry can be retrieved through multiple methods including target name, disease name, drug/ligand name, drug/ligand function and drug therapeutic classification.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr2ZySWqC-s2bVg90H21EOLACvtfcHk0liHPI5FxCjBWg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xht12rtrw%253D&md5=549be6ff77be558f04ef2d3a2a79642e</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1093%2Fnar%2F30.1.412&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252F30.1.412%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DX.%26aulast%3DJi%26aufirst%3DZ.%2BL.%26aulast%3DChen%26aufirst%3DY.%2BZ.%26atitle%3DTTD%253A%2520Therapeutic%2520Target%2520Database%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2002%26volume%3D30%26spage%3D412%26epage%3D415%26doi%3D10.1093%2Fnar%2F30.1.412" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ota, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakurai, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gupta, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valente, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wulff, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ariyoshi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iizasa, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davuluri, R. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nishikura, K.</span></span> <span> </span><span class="NLM_article-title">ADAR1 forms a complex with Dicer to promote microRNA processing and RNA-induced gene silencing</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>153</i></span>,  <span class="NLM_fpage">575</span>– <span class="NLM_lpage">589</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2013.03.024</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1016%2Fj.cell.2013.03.024" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=23622242" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BC3sXmslegtrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=153&publication_year=2013&pages=575-589&author=H.+Otaauthor=M.+Sakuraiauthor=R.+Guptaauthor=L.+Valenteauthor=B.+E.+Wulffauthor=K.+Ariyoshiauthor=H.+Iizasaauthor=R.+V.+Davuluriauthor=K.+Nishikura&title=ADAR1+forms+a+complex+with+Dicer+to+promote+microRNA+processing+and+RNA-induced+gene+silencing&doi=10.1016%2Fj.cell.2013.03.024"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">ADAR1 forms a complex with Dicer to promote microRNA processing and RNA-induced gene silencing</span></div><div class="casAuthors">Ota, Hiromitsu; Sakurai, Masayuki; Gupta, Ravi; Valente, Louis; Wulff, Bjorn-Erik; Ariyoshi, Kentaro; Iizasa, Hisashi; Davuluri, Ramana V.; Nishikura, Kazuko</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">153</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">575-589</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Adenosine deaminases acting on RNA (ADARs) are involved in RNA editing that converts adenosine residues to inosine specifically in double-stranded RNAs.  In this study, we investigated the interaction of the RNA editing mechanism with the RNA interference (RNAi) machinery and found that ADAR1 forms a complex with Dicer through direct protein-protein interaction.  Most importantly, ADAR1 increases the max. rate (Vmax) of pre-microRNA (miRNA) cleavage by Dicer and facilitates loading of miRNA onto RNA-induced silencing complexes, identifying a new role of ADAR1 in miRNA processing and RNAi mechanisms.  ADAR1 differentiates its functions in RNA editing and RNAi by the formation of either ADAR1/ADAR1 homodimer or Dicer/ADAR1 heterodimer complexes, resp.  As expected, the expression of miRNAs is globally inhibited in ADAR1-/- mouse embryos, which, in turn, alters the expression of their target genes and might contribute to their embryonic lethal phenotype.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJLya0xKC2i7Vg90H21EOLACvtfcHk0liHPI5FxCjBWg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXmslegtrc%253D&md5=92bac637be6db8cb4f92cf66d7166de3</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2013.03.024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2013.03.024%26sid%3Dliteratum%253Aachs%26aulast%3DOta%26aufirst%3DH.%26aulast%3DSakurai%26aufirst%3DM.%26aulast%3DGupta%26aufirst%3DR.%26aulast%3DValente%26aufirst%3DL.%26aulast%3DWulff%26aufirst%3DB.%2BE.%26aulast%3DAriyoshi%26aufirst%3DK.%26aulast%3DIizasa%26aufirst%3DH.%26aulast%3DDavuluri%26aufirst%3DR.%2BV.%26aulast%3DNishikura%26aufirst%3DK.%26atitle%3DADAR1%2520forms%2520a%2520complex%2520with%2520Dicer%2520to%2520promote%2520microRNA%2520processing%2520and%2520RNA-induced%2520gene%2520silencing%26jtitle%3DCell%26date%3D2013%26volume%3D153%26spage%3D575%26epage%3D589%26doi%3D10.1016%2Fj.cell.2013.03.024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seo, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, V. N.</span></span> <span> </span><span class="NLM_article-title">Deletion of human tarbp2 reveals cellular microRNA targets and cell-cycle function of TRBP</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1061</span>– <span class="NLM_lpage">1074</span>, <span class="refDoi"> DOI: 10.1016/j.celrep.2014.09.039</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1016%2Fj.celrep.2014.09.039" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=25437560" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVantbbL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=1061-1074&author=Y.+Kimauthor=J.+Yeoauthor=J.+H.+Leeauthor=J.+Choauthor=D.+Seoauthor=J.+S.+Kimauthor=V.+N.+Kim&title=Deletion+of+human+tarbp2+reveals+cellular+microRNA+targets+and+cell-cycle+function+of+TRBP&doi=10.1016%2Fj.celrep.2014.09.039"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Deletion of Human tarbp2 Reveals Cellular MicroRNA Targets and Cell-Cycle Function of TRBP</span></div><div class="casAuthors">Kim, Yoosik; Yeo, Jinah; Lee, Jung Hyun; Cho, Jun; Seo, Daekwan; Kim, Jong-Seo; Kim, V. Narry</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1061-1074</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">TRBP functions as both a Dicer cofactor and a PKR inhibitor.  However, the role of TRBP in microRNA (miRNA) biogenesis is controversial and its regulation of PKR in mitosis remains unexplored.  Here, we generate TRBP knockout cells and find altered Dicer-processing sites in a subset of miRNAs but no effect on Dicer stability, miRNA abundance, or Argonaute loading.  By generating PACT, another Dicer interactor, and TRBP/PACT double knockout (KO) cells, we further show that TRBP and PACT do not functionally compensate for one another and that only TRBP contributes to Dicer processing.  We also report that TRBP is hyperphosphorylated by JNK in M phase when PKR is activated by cellular double-stranded RNAs (dsRNAs).  Hyperphosphorylation potentiates the inhibitory activity of TRBP on PKR, suppressing PKR in M-G1 transition.  By generating human TRBP KO cells, our study clarifies the role of TRBP and unveils neg. feedback regulation of PKR through TRBP phosphorylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdd5hwHvIYs7Vg90H21EOLACvtfcHk0liHPI5FxCjBWg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVantbbL&md5=9a5cf49eb26b6b8bf15483da4087db6b</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2014.09.039&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2014.09.039%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DY.%26aulast%3DYeo%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DJ.%2BH.%26aulast%3DCho%26aufirst%3DJ.%26aulast%3DSeo%26aufirst%3DD.%26aulast%3DKim%26aufirst%3DJ.%2BS.%26aulast%3DKim%26aufirst%3DV.%2BN.%26atitle%3DDeletion%2520of%2520human%2520tarbp2%2520reveals%2520cellular%2520microRNA%2520targets%2520and%2520cell-cycle%2520function%2520of%2520TRBP%26jtitle%3DCell%2520Rep.%26date%3D2014%26volume%3D9%26spage%3D1061%26epage%3D1074%26doi%3D10.1016%2Fj.celrep.2014.09.039" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peters, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Braun, R. E.</span></span> <span> </span><span class="NLM_article-title">A double-stranded RNA binding protein required for activation of repressed messages in mammalian germ cells</span>. <i>Nat. Genet.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">171</span>– <span class="NLM_lpage">174</span>, <span class="refDoi"> DOI: 10.1038/9684</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1038%2F9684" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10369260" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADyaK1MXjvVelsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=1999&pages=171-174&author=J.+Zhongauthor=A.+H.+Petersauthor=K.+Leeauthor=R.+E.+Braun&title=A+double-stranded+RNA+binding+protein+required+for+activation+of+repressed+messages+in+mammalian+germ+cells&doi=10.1038%2F9684"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">A double-stranded RNA binding protein required for activation of repressed messages in mammalian germ cells</span></div><div class="casAuthors">Zhong, Jun; Peters, Antoine H. F. M.; Lee, Keesook; Braun, Robert E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">171-174</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature America</span>)
        </div><div class="casAbstract">Chromatin packaging in mammalian spermatozoa requires an ordered replacement of the somatic histones by two classes of spermatid-specific basic proteins, the transition proteins and the protamines.  Temporal expression of transition proteins and protamines during spermatid differentiation is under translational control, and premature translation of protamine 1 leads to precocious nuclear condensation and sterility.  The authors have previously suggested that the double-stranded (ds) RNA binding protein Prbp (encoded by the gene Tarbp2) functions as a translational regulator during mouse spermatogenesis.  Here the authors show that Prbp is required for proper translational activation of the mRNAs encoding the protamines.  The authors generated mice that carry a targeted disruption of Tarbp2 and detd. that they were sterile and severely oligospermic.  Using immunohistol. anal., they detd. that the endogenous Prm2 mRNA and a reporter mRNA carrying protamine 1 translational-control elements were translated in a mosaic pattern.  The authors showed that failure to synthesize the protamines resulted in delayed replacement of the transition proteins and subsequent failure of spermiation.  The timing of Prbp expression suggests that it may function as a chaperone in the assembly of specific translationally regulated ribonucleoprotein particles.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrvyhVt7hShNbVg90H21EOLACvtfcHk0li5Gyuenpmy9w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXjvVelsbk%253D&md5=9367584833d42e6b1294f420c57a3777</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1038%2F9684&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F9684%26sid%3Dliteratum%253Aachs%26aulast%3DZhong%26aufirst%3DJ.%26aulast%3DPeters%26aufirst%3DA.%2BH.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DBraun%26aufirst%3DR.%2BE.%26atitle%3DA%2520double-stranded%2520RNA%2520binding%2520protein%2520required%2520for%2520activation%2520of%2520repressed%2520messages%2520in%2520mammalian%2520germ%2520cells%26jtitle%3DNat.%2520Genet.%26date%3D1999%26volume%3D22%26spage%3D171%26epage%3D174%26doi%3D10.1038%2F9684" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Goodarzi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buss, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fish, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tavazoie, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tavazoie, S. F.</span></span> <span> </span><span class="NLM_article-title">Metastasis-suppressor transcript destabilization through TARBP2 binding of mRNA hairpins</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>513</i></span>,  <span class="NLM_fpage">256</span>– <span class="NLM_lpage">260</span>, <span class="refDoi"> DOI: 10.1038/nature13466</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1038%2Fnature13466" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=25043050" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsFOltrjP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=513&publication_year=2014&pages=256-260&author=H.+Goodarziauthor=S.+Zhangauthor=C.+G.+Bussauthor=L.+Fishauthor=S.+Tavazoieauthor=S.+F.+Tavazoie&title=Metastasis-suppressor+transcript+destabilization+through+TARBP2+binding+of+mRNA+hairpins&doi=10.1038%2Fnature13466"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Metastasis-suppressor transcript destabilization through TARBP2 binding of mRNA hairpins</span></div><div class="casAuthors">Goodarzi, Hani; Zhang, Steven; Buss, Colin G.; Fish, Lisa; Tavazoie, Saeed; Tavazoie, Sohail F.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">513</span>
        (<span class="NLM_cas:issue">7517</span>),
    <span class="NLM_cas:pages">256-260</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Aberrant regulation of RNA stability has an important role in many disease states.  Deregulated post-transcriptional modulation, such as that governed by microRNAs targeting linear sequence elements in mRNAs, has been implicated in the progression of many cancer types.  A defining feature of RNA is its ability to fold into structures.  However, the roles of structural mRNA elements in cancer progression remain unexplored.  Here we performed an unbiased search for post-transcriptional modulators of mRNA stability in breast cancer by conducting whole-genome transcript stability measurements in poorly and highly metastatic isogenic human breast cancer lines.  Using a computational framework that searches RNA sequence and structure space, we discovered a family of GC-rich structural cis-regulatory RNA elements, termed sRSEs for structural RNA stability elements, which are significantly overrepresented in transcripts displaying reduced stability in highly metastatic cells.  By integrating computational and biochem. approaches, we identified TARBP2, a double-stranded RNA-binding protein implicated in microRNA processing, as the trans factor that binds the sRSE family and similar structural elements-collectively termed TARBP2-binding structural elements (TBSEs)-in transcripts.  TARBP2 is overexpressed in metastatic cells and metastatic human breast tumors and destabilizes transcripts contg. TBSEs.  Endogenous TARBP2 promotes metastatic cell invasion and colonization by destabilizing amyloid precursor protein (APP) and ZNF395 transcripts, two genes previously assocd. with Alzheimer's and Huntington's disease, resp.  We reveal these genes to be novel metastasis suppressor genes in breast cancer.  The cleavage product of APP, extracellular amyloid-α peptide, directly suppresses invasion while ZNF395 transcriptionally represses a pro-metastatic gene expression program.  The expression levels of TARBP2, APP and ZNF395 in human breast carcinomas support their exptl. uncovered roles in metastasis.  Our findings establish a non-canonical and direct role for TARBP2 in mammalian gene expression regulation and reveal that regulated RNA destabilization through protein-mediated binding of mRNA structural elements can govern cancer progression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGre2AI_7HekqLVg90H21EOLACvtfcHk0li5Gyuenpmy9w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsFOltrjP&md5=fdf3b64ca02bb25ad16dd9bca9500a83</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1038%2Fnature13466&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature13466%26sid%3Dliteratum%253Aachs%26aulast%3DGoodarzi%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DBuss%26aufirst%3DC.%2BG.%26aulast%3DFish%26aufirst%3DL.%26aulast%3DTavazoie%26aufirst%3DS.%26aulast%3DTavazoie%26aufirst%3DS.%2BF.%26atitle%3DMetastasis-suppressor%2520transcript%2520destabilization%2520through%2520TARBP2%2520binding%2520of%2520mRNA%2520hairpins%26jtitle%3DNature%26date%3D2014%26volume%3D513%26spage%3D256%26epage%3D260%26doi%3D10.1038%2Fnature13466" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Granito, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marinelli, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Negrini, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Menetti, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benevento, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bolondi, L.</span></span> <span> </span><span class="NLM_article-title">Prognostic significance of adverse events in patients with hepatocellular carcinoma treated with sorafenib</span>. <i>Ther. Adv. Gastroenterol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">240</span>– <span class="NLM_lpage">249</span>, <span class="refDoi"> DOI: 10.1177/1756283X15618129</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1177%2F1756283X15618129" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=26929785" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BC2sXntlenur4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2016&pages=240-249&author=A.+Granitoauthor=S.+Marinelliauthor=G.+Negriniauthor=S.+Menettiauthor=F.+Beneventoauthor=L.+Bolondi&title=Prognostic+significance+of+adverse+events+in+patients+with+hepatocellular+carcinoma+treated+with+sorafenib&doi=10.1177%2F1756283X15618129"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Prognostic significance of adverse events in patients with hepatocellular carcinoma treated with sorafenib</span></div><div class="casAuthors">Granito, Alessandro; Marinelli, Sara; Negrini, Giulia; Menetti, Saverio; Benevento, Francesca; Bolondi, Luigi</div><div class="citationInfo"><span class="NLM_cas:title">Therapeutic Advances in Gastroenterology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">240-249</span>CODEN:
                <span class="NLM_cas:coden">TAGHAO</span>;
        ISSN:<span class="NLM_cas:issn">1756-283X</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications Ltd.</span>)
        </div><div class="casAbstract">Sorafenib is the std. treatment for patients with hepatocellular carcinoma (HCC) with advanced stage disease.  Although its effectiveness has been demonstrated by randomized clin. trials and confirmed by field practice studies, reliable markers predicting therapeutic response have not yet been identified.  Like other tyrosine kinase inhibitors, treatment with sorafenib is burdened by the development of adverse effects, the most frequent being cutaneous toxicity, diarrhea, arterial hypertension and fatigue.  In recent years, several studies have analyzed the correlation between off-target effects and sorafenib efficacy in patients with HCC.  In this review, an overview of the studies assessing the prognostic significance of sorafenib-related adverse events is provided.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpU-IurkEqIA7Vg90H21EOLACvtfcHk0lhz5Jj02sO9Ng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXntlenur4%253D&md5=ce7bc70fcb6b7aab8ed9b33324688eba</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1177%2F1756283X15618129&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1756283X15618129%26sid%3Dliteratum%253Aachs%26aulast%3DGranito%26aufirst%3DA.%26aulast%3DMarinelli%26aufirst%3DS.%26aulast%3DNegrini%26aufirst%3DG.%26aulast%3DMenetti%26aufirst%3DS.%26aulast%3DBenevento%26aufirst%3DF.%26aulast%3DBolondi%26aufirst%3DL.%26atitle%3DPrognostic%2520significance%2520of%2520adverse%2520events%2520in%2520patients%2520with%2520hepatocellular%2520carcinoma%2520treated%2520with%2520sorafenib%26jtitle%3DTher.%2520Adv.%2520Gastroenterol.%26date%3D2016%26volume%3D9%26spage%3D240%26epage%3D249%26doi%3D10.1177%2F1756283X15618129" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nomura, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakachiyama, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hida, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohtaki, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sudo, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aizawa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aburada, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miyamoto, K. I.</span></span> <span> </span><span class="NLM_article-title">Gomisin A, a lignan component of Schizandora fruits, inhibits development of preneoplastic lesions in rat liver by 3′-methyl-4-dimethylamino-azobenzene</span>. <i>Cancer Lett.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>76</i></span>,  <span class="NLM_fpage">11</span>– <span class="NLM_lpage">18</span>, <span class="refDoi"> DOI: 10.1016/0304-3835(94)90128-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1016%2F0304-3835%2894%2990128-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=8124661" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADyaK2cXitlGls7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=76&publication_year=1994&pages=11-18&author=M.+Nomuraauthor=M.+Nakachiyamaauthor=T.+Hidaauthor=Y.+Ohtakiauthor=K.+Sudoauthor=T.+Aizawaauthor=M.+Aburadaauthor=K.+I.+Miyamoto&title=Gomisin+A%2C+a+lignan+component+of+Schizandora+fruits%2C+inhibits+development+of+preneoplastic+lesions+in+rat+liver+by+3%E2%80%B2-methyl-4-dimethylamino-azobenzene&doi=10.1016%2F0304-3835%2894%2990128-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Gomisin A, a lignan component of Schizandra fruits, inhibits development of preneoplastic lesions in rat liver by 3'-methyl-4-dimethylamino-azobenzene</span></div><div class="casAuthors">Nomura, M.; Nakachiyama, M.; Hida, T.; Ohtaki, Y.; Sudo, K.; Aizawa, T.; Aburada, M.; Miyamoto, K. I.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Letters (Shannon, Ireland)</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">76</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">11-18</span>CODEN:
                <span class="NLM_cas:coden">CALEDQ</span>;
        ISSN:<span class="NLM_cas:issn">0304-3835</span>.
    </div><div class="casAbstract">The effects of gomisin A, a lignan component of Schizandra fruits, on development of preneoplastic lesions in the liver after a short-term (3 wk) feeding of 3'-methyl-4-dimethylaminoazobenzene (3'-MeDAB) to male Donryu rats were investigated, and compared with the effects of phenobarbital.  Gomisin A inhibited both increases of the level of glutathione-S-transferases placental form (GST-P) and the no. and size of GST-P pos. foci in the liver increased after treatment with 3'-MeDAB.  Moreover, although the population of diploid nuclei was increased and that of tetraploid nuclei was decreased by pretreatment with 3'-MeDAB, gomisin A returned this to near the normal ploidy pattern.  But phenobarbital increased the level of GST-P and the no. and size of GST-P pos. foci with little affect on the ploidy population changed by 3'-MeDAB.  Thus, the effect of gomisin A on hepatocarcinogenesis was inhibitory in contrast with that of phenobarbital.  This study suggests that gomisin A is a candidate for a chemopreventive drug inhibiting the promotion process in hepatocarcinogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7ke46q2lcoLVg90H21EOLACvtfcHk0lhz5Jj02sO9Ng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXitlGls7w%253D&md5=402ce9f28057daa8b7f2230415439e50</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1016%2F0304-3835%2894%2990128-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0304-3835%252894%252990128-7%26sid%3Dliteratum%253Aachs%26aulast%3DNomura%26aufirst%3DM.%26aulast%3DNakachiyama%26aufirst%3DM.%26aulast%3DHida%26aufirst%3DT.%26aulast%3DOhtaki%26aufirst%3DY.%26aulast%3DSudo%26aufirst%3DK.%26aulast%3DAizawa%26aufirst%3DT.%26aulast%3DAburada%26aufirst%3DM.%26aulast%3DMiyamoto%26aufirst%3DK.%2BI.%26atitle%3DGomisin%2520A%252C%2520a%2520lignan%2520component%2520of%2520Schizandora%2520fruits%252C%2520inhibits%2520development%2520of%2520preneoplastic%2520lesions%2520in%2520rat%2520liver%2520by%25203%25E2%2580%25B2-methyl-4-dimethylamino-azobenzene%26jtitle%3DCancer%2520Lett.%26date%3D1994%26volume%3D76%26spage%3D11%26epage%3D18%26doi%3D10.1016%2F0304-3835%2894%2990128-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ibrahim, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmithals, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kowarz, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koberle, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kakoschky, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pleli, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kollmar, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nitsch, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waidmann, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finkelmeier, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeuzem, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korf, H. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmid, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weigert, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kronenberger, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marschalek, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piiper, A.</span></span> <span> </span><span class="NLM_article-title">Hypoxia causes downregulation of Dicer in hepatocellular carcinoma, which is required for upregulation of hypoxia-inducible factor 1alpha and epithelial-mesenchymal transition</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">3896</span>– <span class="NLM_lpage">3905</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.CCR-16-1762</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1158%2F1078-0432.CCR-16-1762" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=28167508" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtFKiurbN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2017&pages=3896-3905&author=A.+A.+Ibrahimauthor=C.+Schmithalsauthor=E.+Kowarzauthor=V.+Koberleauthor=B.+Kakoschkyauthor=T.+Pleliauthor=O.+Kollmarauthor=S.+Nitschauthor=O.+Waidmannauthor=F.+Finkelmeierauthor=S.+Zeuzemauthor=H.+W.+Korfauthor=T.+Schmidauthor=A.+Weigertauthor=B.+Kronenbergerauthor=R.+Marschalekauthor=A.+Piiper&title=Hypoxia+causes+downregulation+of+Dicer+in+hepatocellular+carcinoma%2C+which+is+required+for+upregulation+of+hypoxia-inducible+factor+1alpha+and+epithelial-mesenchymal+transition&doi=10.1158%2F1078-0432.CCR-16-1762"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Hypoxia Causes Downregulation of Dicer in Hepatocellular Carcinoma, Which Is Required for Upregulation of Hypoxia-Inducible Factor 1α and Epithelial-Mesenchymal Transition</span></div><div class="casAuthors">Ibrahim, Ahmed Atef; Schmithals, Christian; Kowarz, Erik; Koberle, Verena; Kakoschky, Bianca; Pleli, Thomas; Kollmar, Otto; Nitsch, Scarlett; Waidmann, Oliver; Finkelmeier, Fabian; Zeuzem, Stefan; Korf, Horst-Werner; Schmid, Tobias; Weigert, Andreas; Kronenberger, Bernd; Marschalek, Rolf; Piiper, Albrecht</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">3896-3905</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Purpose: A role of Dicer, which converts precursor miRNAs to mature miRNAs, in the tumor-promoting effect of hypoxia is currently emerging in some tumor entities.  Its role in hepatocellular carcinoma (HCC) is unknown.  Exptl. Design: HepG2 and Huh-7 cells were stably transfected with an inducible Dicer expression vector and were exposed to hypoxia/normoxia.  HepG2-Dicer xenografts were established in nude mice; hypoxic areas and Dicer were detected in HCC xenografts and HCCs from mice with endogenous hepatocarcinogenesis; and epithelial-mesenchymal transition (EMT) markers were analyzed by immunohistochem. or by immunoblotting.  The correlation between Dicer and carbonic anhydrase 9 (CA9), a marker of hypoxia, was investigated in resected human HCCs.  Results: Hypoxia increased EMT markers in vitro and in vivo and led to a downregulation of Dicer in HCC cells.  The levels of Dicer were downregulated in hypoxic tumor regions in mice with endogenous hepatocarcinogenesis and in HepG2 xenografts.  In human HCCs, the levels of Dicer correlated inversely with those of CA9, indicating that the neg. regulation of Dicer by hypoxia also applies to HCC patients.  Forced expression of Dicer prevented the hypoxia-induced increase in hypoxia-inducible factor 1α (HIF1α), HIF2α, hypoxia-inducible genes (CA9, glucose transporter 1), EMT markers, and cell migration.  Conclusions: We here identify downmodulation of Dicer as novel essential process in hypoxia-induced EMT in HCC and demonstrate that induced expression of Dicer counteracted hypoxia-induced EMT.  Thus, targeting hypoxia-induced downmodulation of Dicer is a promising novel strategy to reduce HCC progression.  Clin Cancer Res; 23(14); 3896-905. ©2017 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqiCM5iMZ0KMLVg90H21EOLACvtfcHk0lgu5LPaBjDZYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtFKiurbN&md5=8c8e45c5641e60b848b9bec0901472a9</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-16-1762&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-16-1762%26sid%3Dliteratum%253Aachs%26aulast%3DIbrahim%26aufirst%3DA.%2BA.%26aulast%3DSchmithals%26aufirst%3DC.%26aulast%3DKowarz%26aufirst%3DE.%26aulast%3DKoberle%26aufirst%3DV.%26aulast%3DKakoschky%26aufirst%3DB.%26aulast%3DPleli%26aufirst%3DT.%26aulast%3DKollmar%26aufirst%3DO.%26aulast%3DNitsch%26aufirst%3DS.%26aulast%3DWaidmann%26aufirst%3DO.%26aulast%3DFinkelmeier%26aufirst%3DF.%26aulast%3DZeuzem%26aufirst%3DS.%26aulast%3DKorf%26aufirst%3DH.%2BW.%26aulast%3DSchmid%26aufirst%3DT.%26aulast%3DWeigert%26aufirst%3DA.%26aulast%3DKronenberger%26aufirst%3DB.%26aulast%3DMarschalek%26aufirst%3DR.%26aulast%3DPiiper%26aufirst%3DA.%26atitle%3DHypoxia%2520causes%2520downregulation%2520of%2520Dicer%2520in%2520hepatocellular%2520carcinoma%252C%2520which%2520is%2520required%2520for%2520upregulation%2520of%2520hypoxia-inducible%2520factor%25201alpha%2520and%2520epithelial-mesenchymal%2520transition%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2017%26volume%3D23%26spage%3D3896%26epage%3D3905%26doi%3D10.1158%2F1078-0432.CCR-16-1762" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yoo, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santhekadur, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gredler, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emdad, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhutia, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pannell, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fisher, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sarkar, D.</span></span> <span> </span><span class="NLM_article-title">Increased RNA-induced silencing complex (RISC) activity contributes to hepatocellular carcinoma</span>. <i>Hepatology</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">1538</span>– <span class="NLM_lpage">1548</span>, <span class="refDoi"> DOI: 10.1002/hep.24216</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1002%2Fhep.24216" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=21520169" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BC3MXltVygu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2011&pages=1538-1548&author=B.+K.+Yooauthor=P.+K.+Santhekadurauthor=R.+Gredlerauthor=D.+Chenauthor=L.+Emdadauthor=S.+Bhutiaauthor=L.+Pannellauthor=P.+B.+Fisherauthor=D.+Sarkar&title=Increased+RNA-induced+silencing+complex+%28RISC%29+activity+contributes+to+hepatocellular+carcinoma&doi=10.1002%2Fhep.24216"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Increased RNA-induced silencing complex (RISC) activity contributes to hepatocellular carcinoma</span></div><div class="casAuthors">Yoo, Byoung Kwon; Santhekadur, Prasanna K.; Gredler, Rachel; Chen, Dong; Emdad, Luni; Bhutia, Sujit; Pannell, Lewis; Fisher, Paul B.; Sarkar, Devanand</div><div class="citationInfo"><span class="NLM_cas:title">Hepatology (Hoboken, NJ, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1538-1548</span>CODEN:
                <span class="NLM_cas:coden">HPTLD9</span>;
        ISSN:<span class="NLM_cas:issn">0270-9139</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">There is virtually no effective treatment for advanced hepatocellular carcinoma (HCC) and novel targets need to be identified to develop effective treatment.  We recently documented that the oncogene Astrocyte elevated gene-1 (AEG-1) plays a seminal role in hepatocarcinogenesis.  Employing yeast two-hybrid assay and coimmunopptn. followed by mass spectrometry, we identified staphylococcal nuclease domain contg. 1 (SND1), a nuclease in the RNA-induced silencing complex (RISC) facilitating RNAi-mediated gene silencing, as an AEG-1 interacting protein.  Coimmunopptn. and colocalization studies confirmed that AEG-1 is also a component of RISC and both AEG-1 and SND1 are required for optimum RISC activity facilitating small interfering RNA (siRNA) and micro RNA (miRNA)-mediated silencing of luciferase reporter gene.  In 109 human HCC samples SND1 was overexpressed in ≈74% cases compared to normal liver.  Correspondingly, significantly higher RISC activity was obsd. in human HCC cells compared to immortal normal hepatocytes.  Increased RISC activity, conferred by AEG-1 or SND1, resulted in increased degrdn. of tumor suppressor mRNAs that are target of oncomiRs.  Inhibition of enzymic activity of SND1 significantly inhibited proliferation of human HCC cells.  As a corollary, stable overexpression of SND1 augmented and siRNA-mediated inhibition of SND1 abrogated growth of human HCC cells in vitro and in vivo, thus revealing a potential role of SND1 in hepatocarcinogenesis.  Conclusion: We unravel a novel mechanism that overexpression of AEG-1 and SND1 leading to increased RISC activity might contribute to hepatocarcinogenesis.  Targeted inhibition of SND1 enzymic activity might be developed as an effective therapy for HCC. (HEPATOl. 2011;).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrY2Bz39GT-77Vg90H21EOLACvtfcHk0lgu5LPaBjDZYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXltVygu7o%253D&md5=86da7288e46fe36e5ae805fae8ea2f12</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1002%2Fhep.24216&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fhep.24216%26sid%3Dliteratum%253Aachs%26aulast%3DYoo%26aufirst%3DB.%2BK.%26aulast%3DSanthekadur%26aufirst%3DP.%2BK.%26aulast%3DGredler%26aufirst%3DR.%26aulast%3DChen%26aufirst%3DD.%26aulast%3DEmdad%26aufirst%3DL.%26aulast%3DBhutia%26aufirst%3DS.%26aulast%3DPannell%26aufirst%3DL.%26aulast%3DFisher%26aufirst%3DP.%2BB.%26aulast%3DSarkar%26aufirst%3DD.%26atitle%3DIncreased%2520RNA-induced%2520silencing%2520complex%2520%2528RISC%2529%2520activity%2520contributes%2520to%2520hepatocellular%2520carcinoma%26jtitle%3DHepatology%26date%3D2011%26volume%3D53%26spage%3D1538%26epage%3D1548%26doi%3D10.1002%2Fhep.24216" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, F. Y.</span></span> <span> </span><span class="NLM_article-title">Multi-targeted anticancer agents</span>. <i>Curr. Top. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">3084</span>– <span class="NLM_lpage">3098</span>, <span class="refDoi"> DOI: 10.2174/1568026617666170707124126</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.2174%2F1568026617666170707124126" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=28685693" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFens7fJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2017&pages=3084-3098&author=W.+Zhengauthor=Y.+Zhaoauthor=Q.+Luoauthor=Y.+Zhangauthor=K.+Wuauthor=F.+Y.+Wang&title=Multi-targeted+anticancer+agents&doi=10.2174%2F1568026617666170707124126"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Multi-Targeted Anticancer Agents</span></div><div class="casAuthors">Zheng, Wei; Zhao, Yao; Luo, Qun; Zhang, Yang; Wu, Kui; Wang, Fuyi</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">28</span>),
    <span class="NLM_cas:pages">3084-3098</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">There is a great demand for the development of novel anticancer drugs, due to the increasing morbidity and high mortality of cancer.  To date, chemotherapy still plays a central role in the clin. treatment of cancer, but this role is being reduced by targeted therapies.  Since cancer is a complicated and multiple genes involved disease, drugs that act at multiple targets can enhance efficacy and lower drug resistance, and are thought to be the future of novel anticancer drug development.  In this paper, we discuss the recent development of "single mol., multi-target" anticancer agents that combine two or more pharmacophores in a single mol., including org. multitargeted anticancer agents and metal based complexes such as platinum, ruthenium, iridium and rhodium complexes.  These efforts will contribute to clin. cancer therapy and benefit patients in the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoj7unFBNhn3rVg90H21EOLACvtfcHk0lg1oYuFbEaXpA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvFens7fJ&md5=b9f6834bf2af9c3e9460badd9ca133c8</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.2174%2F1568026617666170707124126&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1568026617666170707124126%26sid%3Dliteratum%253Aachs%26aulast%3DZheng%26aufirst%3DW.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DLuo%26aufirst%3DQ.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DWu%26aufirst%3DK.%26aulast%3DWang%26aufirst%3DF.%2BY.%26atitle%3DMulti-targeted%2520anticancer%2520agents%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2017%26volume%3D17%26spage%3D3084%26epage%3D3098%26doi%3D10.2174%2F1568026617666170707124126" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ding, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kataoka, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nie, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, Z. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pu, W. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, D. Z.</span></span> <span> </span><span class="NLM_article-title">Trbp regulates heart function through microRNA-mediated Sox6 repression</span>. <i>Nat. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">776</span>– <span class="NLM_lpage">783</span>, <span class="refDoi"> DOI: 10.1038/ng.3324</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1038%2Fng.3324" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=26029872" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFemt7vI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2015&pages=776-783&author=J.+Dingauthor=J.+Chenauthor=Y.+Wangauthor=M.+Kataokaauthor=L.+Maauthor=P.+Zhouauthor=X.+Huauthor=Z.+Linauthor=M.+Nieauthor=Z.+L.+Dengauthor=W.+T.+Puauthor=D.+Z.+Wang&title=Trbp+regulates+heart+function+through+microRNA-mediated+Sox6+repression&doi=10.1038%2Fng.3324"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Trbp regulates heart function through microRNA-mediated Sox6 repression</span></div><div class="casAuthors">Ding, Jian; Chen, Jinghai; Wang, Yanqun; Kataoka, Masaharu; Ma, Lixin; Zhou, Pingzhu; Hu, Xiaoyun; Lin, Zhiqiang; Nie, Mao; Deng, Zhong-Liang; Pu, William T.; Wang, Da-Zhi</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">776-783</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Cardiomyopathy is assocd. with altered expression of genes encoding contractile proteins.  Here we show that Trbp (Tarbp2), an RNA-binding protein, is required for normal heart function.  Cardiac-specific inactivation in mice of Trbp (TrbpcKO) caused progressive cardiomyopathy and lethal heart failure.  Loss of Trbp function resulted in upregulation of Sox6, repression of genes encoding normal cardiac slow-twitch myofiber proteins and pathol. increased expression of genes encoding skeletal fast-twitch myofiber proteins.  Remarkably, knockdown of Sox6 fully rescued the Trbp-mutant phenotype, whereas mice overexpressing Sox6 phenocopied TrbpcKO mice.  Trbp inactivation was mechanistically linked to Sox6 upregulation through altered processing of miR-208a, which is a direct inhibitor of Sox6.  Transgenic overexpression of Mir208a sufficiently repressed Sox6, restored the balance in gene expression for fast- and slow-twitch myofiber proteins, and rescued cardiac function in TrbpcKO mice.  Together, our studies identify a new Trbp-mediated microRNA-processing mechanism in the regulation of a linear genetic cascade essential for normal heart function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_b3TtPlJgtrVg90H21EOLACvtfcHk0lg1oYuFbEaXpA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFemt7vI&md5=8de59059910c041985ba0f11c716191a</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1038%2Fng.3324&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fng.3324%26sid%3Dliteratum%253Aachs%26aulast%3DDing%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DKataoka%26aufirst%3DM.%26aulast%3DMa%26aufirst%3DL.%26aulast%3DZhou%26aufirst%3DP.%26aulast%3DHu%26aufirst%3DX.%26aulast%3DLin%26aufirst%3DZ.%26aulast%3DNie%26aufirst%3DM.%26aulast%3DDeng%26aufirst%3DZ.%2BL.%26aulast%3DPu%26aufirst%3DW.%2BT.%26aulast%3DWang%26aufirst%3DD.%2BZ.%26atitle%3DTrbp%2520regulates%2520heart%2520function%2520through%2520microRNA-mediated%2520Sox6%2520repression%26jtitle%3DNat.%2520Genet.%26date%3D2015%26volume%3D47%26spage%3D776%26epage%3D783%26doi%3D10.1038%2Fng.3324" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shan, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szulwach, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faghihi, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khalil, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paroo, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wahlestedt, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, P.</span></span> <span> </span><span class="NLM_article-title">A small molecule enhances RNA interference and promotes microRNA processing</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">933</span>– <span class="NLM_lpage">940</span>, <span class="refDoi"> DOI: 10.1038/nbt.1481</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1038%2Fnbt.1481" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=18641635" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BD1cXps1Wls7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2008&pages=933-940&author=G.+Shanauthor=Y.+Liauthor=J.+Zhangauthor=W.+Liauthor=K.+E.+Szulwachauthor=R.+Duanauthor=M.+A.+Faghihiauthor=A.+M.+Khalilauthor=L.+Luauthor=Z.+Parooauthor=A.+W.+Chanauthor=Z.+Shiauthor=Q.+Liuauthor=C.+Wahlestedtauthor=C.+Heauthor=P.+Jin&title=A+small+molecule+enhances+RNA+interference+and+promotes+microRNA+processing&doi=10.1038%2Fnbt.1481"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">A small molecule enhances RNA interference and promotes microRNA processing</span></div><div class="casAuthors">Shan, Ge; Li, Yujing; Zhang, Junliang; Li, Wendi; Szulwach, Keith E.; Duan, Ranhui; Faghihi, Mohammad A.; Khalil, Ahmad M.; Lu, Lianghua; Paroo, Zain; Chan, Anthony W. S.; Shi, Zhangjie; Liu, Qinghua; Wahlestedt, Claes; He, Chuan; Jin, Peng</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">933-940</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Little is known about the regulation of RNA interference (RNAi).  Shan et al. constructed a reporter system to monitor RNAi activity and identified a small mol. that increases RNAi by facilitating the interaction between small RNAs and a protein involved in small RNA loading and processing.  Little is known about the regulation of RNA interference (RNAi).  Shan et al. constructed a reporter system to monitor RNAi activity and identified a small mol. that increases RNAi by facilitating the interaction between small RNAs and a protein involved in small RNA loading and processing.  Small interfering RNAs (siRNAs) and microRNAs (miRNAs) are sequence-specific post-transcriptional regulators of gene expression.  Although major components of the RNA interference (RNAi) pathway have been identified, regulatory mechanisms for this pathway remain largely unknown.  Here we demonstrate that the RNAi pathway can be modulated intracellularly by small mols.  We have developed a cell-based assay to monitor the activity of the RNAi pathway and find that the small-mol. enoxacin (Penetrex) enhances siRNA-mediated mRNA degrdn. and promotes the biogenesis of endogenous miRNAs.  We show that this RNAi-enhancing activity depends on the trans-activation-responsive region RNA-binding protein.  Our results provide a proof-of-principle demonstration that small mols. can be used to modulate the activity of the RNAi pathway.  RNAi enhancers may be useful in the development of research tools and therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmMvLQLjUDALVg90H21EOLACvtfcHk0lgm086bglGy3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXps1Wls7c%253D&md5=15210fd18afcb865b6d2e218449c42b4</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1038%2Fnbt.1481&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt.1481%26sid%3Dliteratum%253Aachs%26aulast%3DShan%26aufirst%3DG.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DSzulwach%26aufirst%3DK.%2BE.%26aulast%3DDuan%26aufirst%3DR.%26aulast%3DFaghihi%26aufirst%3DM.%2BA.%26aulast%3DKhalil%26aufirst%3DA.%2BM.%26aulast%3DLu%26aufirst%3DL.%26aulast%3DParoo%26aufirst%3DZ.%26aulast%3DChan%26aufirst%3DA.%2BW.%26aulast%3DShi%26aufirst%3DZ.%26aulast%3DLiu%26aufirst%3DQ.%26aulast%3DWahlestedt%26aufirst%3DC.%26aulast%3DHe%26aufirst%3DC.%26aulast%3DJin%26aufirst%3DP.%26atitle%3DA%2520small%2520molecule%2520enhances%2520RNA%2520interference%2520and%2520promotes%2520microRNA%2520processing%26jtitle%3DNat.%2520Biotechnol.%26date%3D2008%26volume%3D26%26spage%3D933%26epage%3D940%26doi%3D10.1038%2Fnbt.1481" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sousa, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graca, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baptista, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vieira, F. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palmeira, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henrique, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeronimo, C.</span></span> <span> </span><span class="NLM_article-title">Enoxacin inhibits growth of prostate cancer cells and effectively restores microRNA processing</span>. <i>Epigenetics</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">548</span>– <span class="NLM_lpage">558</span>, <span class="refDoi"> DOI: 10.4161/epi.24519</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.4161%2Fepi.24519" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=23644875" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjs1yitr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=548-558&author=E.+Sousaauthor=I.+Gracaauthor=T.+Baptistaauthor=F.+Q.+Vieiraauthor=C.+Palmeiraauthor=R.+Henriqueauthor=C.+Jeronimo&title=Enoxacin+inhibits+growth+of+prostate+cancer+cells+and+effectively+restores+microRNA+processing&doi=10.4161%2Fepi.24519"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Enoxacin inhibits growth of prostate cancer cells and effectively restores microRNA processing</span></div><div class="casAuthors">Sousa, Elsa J.; Graca, Ines; Baptista, Tiago; Vieira, Filipa Q.; Palmeira, Carlos; Henrique, Rui; Jeronimo, Carmen</div><div class="citationInfo"><span class="NLM_cas:title">Epigenetics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">548-558</span>CODEN:
                <span class="NLM_cas:coden">EPIGB6</span>;
        ISSN:<span class="NLM_cas:issn">1559-2294</span>.
    
            (<span class="NLM_cas:orgname">Landes Bioscience</span>)
        </div><div class="casAbstract">Prostate cancer (PCa) is one of the most incident malignancies worldwide.  Although efficient therapy is available for early-stage PCa, treatment of advanced disease is mainly ineffective and remains a clin. challenge.  MicroRNA (miRNA) dysregulation is assocd. with PCa development and progression.  In fact, several studies have reported a widespread downregulation of miRNAs in PCa, which highlights the importance of studying compds. capable of restoring the global miRNA expression.  The main aim of this study was to define the usefulness of enoxacin as an anti-tumoral agent in PCa, due to its ability to induce miRNA biogenesis in a TRBP-mediated manner.  Using a panel of five PCa cell lines, we obsd. that all of them were wild type for the TARBP2 gene and expressed TRBP protein.  Furthermore, primary prostate carcinomas displayed normal levels of TRBP protein.  Remarkably, enoxacin was able to decrease cell viability, induce apoptosis, cause cell cycle arrest, and inhibit the invasiveness of cell lines.  Enoxacin was also effective in restoring the global expression of miRNAs.  This study is the first to show that PCa cells are highly responsive to the anti-tumoral effects of enoxacin.  Therefore, enoxacin constitutes a promising therapeutic agent for PCa.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUsYbG1rODRbVg90H21EOLACvtfcHk0lgm086bglGy3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjs1yitr4%253D&md5=88b039ce63ee7c0e682bbd184f6d7da3</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.4161%2Fepi.24519&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252Fepi.24519%26sid%3Dliteratum%253Aachs%26aulast%3DSousa%26aufirst%3DE.%26aulast%3DGraca%26aufirst%3DI.%26aulast%3DBaptista%26aufirst%3DT.%26aulast%3DVieira%26aufirst%3DF.%2BQ.%26aulast%3DPalmeira%26aufirst%3DC.%26aulast%3DHenrique%26aufirst%3DR.%26aulast%3DJeronimo%26aufirst%3DC.%26atitle%3DEnoxacin%2520inhibits%2520growth%2520of%2520prostate%2520cancer%2520cells%2520and%2520effectively%2520restores%2520microRNA%2520processing%26jtitle%3DEpigenetics%26date%3D2013%26volume%3D8%26spage%3D548%26epage%3D558%26doi%3D10.4161%2Fepi.24519" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schulze, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nault, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villanueva, A.</span></span> <span> </span><span class="NLM_article-title">Genetic profiling of hepatocellular carcinoma using next-generation sequencing</span>. <i>J. Hepatol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>65</i></span>,  <span class="NLM_fpage">1031</span>– <span class="NLM_lpage">1042</span>, <span class="refDoi"> DOI: 10.1016/j.jhep.2016.05.035</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1016%2Fj.jhep.2016.05.035" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=27262756" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVantLjN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=2016&pages=1031-1042&author=K.+Schulzeauthor=J.+C.+Naultauthor=A.+Villanueva&title=Genetic+profiling+of+hepatocellular+carcinoma+using+next-generation+sequencing&doi=10.1016%2Fj.jhep.2016.05.035"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Genetic profiling of hepatocellular carcinoma using next-generation sequencing</span></div><div class="casAuthors">Schulze, Kornelius; Nault, Jean-Charles; Villanueva, Augusto</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Hepatology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">65</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1031-1042</span>CODEN:
                <span class="NLM_cas:coden">JOHEEC</span>;
        ISSN:<span class="NLM_cas:issn">0168-8278</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Hepatocellular carcinoma (HCC) is a highly heterogeneous disease, both clin. and from a mol. standpoint.  The advent of next-generation sequencing technologies has provided new opportunities to extensively analyze mol. defects in HCC samples.  This has uncovered major cancer driver genes and assocd. oncogenic pathways operating in HCC.  More sophisticated analyses of sequencing data have linked specific nucleotide patterns to external toxic agents and defined so-called 'mutational signatures' in HCC.  Mol. signatures, taking into account intra- and inter-tumor heterogeneity, and their functional validation could provide useful data to predict treatment response to mol. therapies.  In this review we will focus on the current knowledge of deep sequencing in HCC and its foreseeable clin. impact.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpI8a2Sa6XEYbVg90H21EOLACvtfcHk0lgm086bglGy3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVantLjN&md5=049570fd840487e7db157e39760e6ccb</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1016%2Fj.jhep.2016.05.035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jhep.2016.05.035%26sid%3Dliteratum%253Aachs%26aulast%3DSchulze%26aufirst%3DK.%26aulast%3DNault%26aufirst%3DJ.%2BC.%26aulast%3DVillanueva%26aufirst%3DA.%26atitle%3DGenetic%2520profiling%2520of%2520hepatocellular%2520carcinoma%2520using%2520next-generation%2520sequencing%26jtitle%3DJ.%2520Hepatol.%26date%3D2016%26volume%3D65%26spage%3D1031%26epage%3D1042%26doi%3D10.1016%2Fj.jhep.2016.05.035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Behan, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iorio, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picco, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goncalves, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beaver, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Migliardi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santos, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sassi, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pinnelli, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ansari, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harper, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jackson, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McRae, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pooley, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilkinson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der Meer, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dow, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buser-Doepner, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertotti, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trusolino, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stronach, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saez-Rodriguez, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yusa, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garnett, M. J.</span></span> <span> </span><span class="NLM_article-title">Prioritization of cancer therapeutic targets using crispr-Cas9 screens</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>568</i></span>,  <span class="NLM_fpage">511</span>– <span class="NLM_lpage">516</span>, <span class="refDoi"> DOI: 10.1038/s41586-019-1103-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1038%2Fs41586-019-1103-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=30971826" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BC1MXosVSgtrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=568&publication_year=2019&pages=511-516&author=F.+M.+Behanauthor=F.+Iorioauthor=G.+Piccoauthor=E.+Goncalvesauthor=C.+M.+Beaverauthor=G.+Migliardiauthor=R.+Santosauthor=Y.+Raoauthor=F.+Sassiauthor=M.+Pinnelliauthor=R.+Ansariauthor=S.+Harperauthor=D.+A.+Jacksonauthor=R.+McRaeauthor=R.+Pooleyauthor=P.+Wilkinsonauthor=D.+van+der+Meerauthor=D.+Dowauthor=C.+Buser-Doepnerauthor=A.+Bertottiauthor=L.+Trusolinoauthor=E.+A.+Stronachauthor=J.+Saez-Rodriguezauthor=K.+Yusaauthor=M.+J.+Garnett&title=Prioritization+of+cancer+therapeutic+targets+using+crispr-Cas9+screens&doi=10.1038%2Fs41586-019-1103-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Prioritization of cancer therapeutic targets using CRISPR-Cas9 screens</span></div><div class="casAuthors">Behan, Fiona M.; Iorio, Francesco; Picco, Gabriele; Goncalves, Emanuel; Beaver, Charlotte M.; Migliardi, Giorgia; Santos, Rita; Rao, Yanhua; Sassi, Francesco; Pinnelli, Marika; Ansari, Rizwan; Harper, Sarah; Jackson, David Adam; McRae, Rebecca; Pooley, Rachel; Wilkinson, Piers; van der Meer, Dieudonne; Dow, David; Buser-Doepner, Carolyn; Bertotti, Andrea; Trusolino, Livio; Stronach, Euan A.; Saez-Rodriguez, Julio; Yusa, Kosuke; Garnett, Mathew J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">568</span>
        (<span class="NLM_cas:issue">7753</span>),
    <span class="NLM_cas:pages">511-516</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Functional genomics approaches can overcome limitations-such as the lack of identification of robust targets and poor clin. efficacy-that hamper cancer drug development.  Here we performed genome-scale CRISPR-Cas9 screens in 324 human cancer cell lines from 30 cancer types and developed a data-driven framework to prioritize candidates for cancer therapeutics.  We integrated cell fitness effects with genomic biomarkers and target tractability for drug development to systematically prioritize new targets in defined tissues and genotypes.  We verified one of our most promising dependencies, the Werner syndrome ATP-dependent helicase, as a synthetic lethal target in tumors from multiple cancer types with microsatellite instability.  Our anal. provides a resource of cancer dependencies, generates a framework to prioritize cancer drug targets and suggests specific new targets.  The principles described in this study can inform the initial stages of drug development by contributing to a new, diverse and more effective portfolio of cancer drug targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHnWQcCFsy-LVg90H21EOLACvtfcHk0ljwGhMsoazAuQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXosVSgtrw%253D&md5=3cac7e1a9ee5c34c8599154e8fc30573</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1038%2Fs41586-019-1103-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-019-1103-9%26sid%3Dliteratum%253Aachs%26aulast%3DBehan%26aufirst%3DF.%2BM.%26aulast%3DIorio%26aufirst%3DF.%26aulast%3DPicco%26aufirst%3DG.%26aulast%3DGoncalves%26aufirst%3DE.%26aulast%3DBeaver%26aufirst%3DC.%2BM.%26aulast%3DMigliardi%26aufirst%3DG.%26aulast%3DSantos%26aufirst%3DR.%26aulast%3DRao%26aufirst%3DY.%26aulast%3DSassi%26aufirst%3DF.%26aulast%3DPinnelli%26aufirst%3DM.%26aulast%3DAnsari%26aufirst%3DR.%26aulast%3DHarper%26aufirst%3DS.%26aulast%3DJackson%26aufirst%3DD.%2BA.%26aulast%3DMcRae%26aufirst%3DR.%26aulast%3DPooley%26aufirst%3DR.%26aulast%3DWilkinson%26aufirst%3DP.%26aulast%3Dvan%2Bder%2BMeer%26aufirst%3DD.%26aulast%3DDow%26aufirst%3DD.%26aulast%3DBuser-Doepner%26aufirst%3DC.%26aulast%3DBertotti%26aufirst%3DA.%26aulast%3DTrusolino%26aufirst%3DL.%26aulast%3DStronach%26aufirst%3DE.%2BA.%26aulast%3DSaez-Rodriguez%26aufirst%3DJ.%26aulast%3DYusa%26aufirst%3DK.%26aulast%3DGarnett%26aufirst%3DM.%2BJ.%26atitle%3DPrioritization%2520of%2520cancer%2520therapeutic%2520targets%2520using%2520crispr-Cas9%2520screens%26jtitle%3DNature%26date%3D2019%26volume%3D568%26spage%3D511%26epage%3D516%26doi%3D10.1038%2Fs41586-019-1103-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gatignol, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckler-White, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berkhout, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeang, K. T.</span></span> <span> </span><span class="NLM_article-title">Characterization of a human TAR RNA-binding protein that activates the HIV-1 LTR</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">1991</span>,  <span class="NLM_volume"><i>251</i></span>,  <span class="NLM_fpage">1597</span>– <span class="NLM_lpage">1600</span>, <span class="refDoi"> DOI: 10.1126/science.2011739</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1126%2Fscience.2011739" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=2011739" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADyaK3MXlslKgs7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=251&publication_year=1991&pages=1597-1600&author=A.+Gatignolauthor=A.+Buckler-Whiteauthor=B.+Berkhoutauthor=K.+T.+Jeang&title=Characterization+of+a+human+TAR+RNA-binding+protein+that+activates+the+HIV-1+LTR&doi=10.1126%2Fscience.2011739"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of a human TAR RNA-binding protein that activates the HIV-1 LTR</span></div><div class="casAuthors">Gatignol, Anne; Buckler-White, Alicia; Berkhout, Ben; Jeang, Kuan Teh</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">251</span>
        (<span class="NLM_cas:issue">5001</span>),
    <span class="NLM_cas:pages">1597-600</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    </div><div class="casAbstract">Human immunodeficiency virus type 1 (HIV-1) gene expression is activated by Tat, a virally encoded protein.  Tat trans-activation requires viral (trans-activation-responsive; TAR) RNA sequences located in the R region of the long terminal repeat (LTR).  Existing evidence suggests that Tat probably cooperates with cellular factors that bind to TAR RNA in the overall trans-activation process.  A HeLa cDNA was isolated and characterized that encodes a TAR RNA-binding protein (TRBP).  TRBP activated the HIV-1 LTR and was synergistic with Tat function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoqs0r0DyFmSLVg90H21EOLACvtfcHk0ljwGhMsoazAuQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3MXlslKgs7w%253D&md5=19ff68923d636e815072329827f3ecd8</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1126%2Fscience.2011739&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.2011739%26sid%3Dliteratum%253Aachs%26aulast%3DGatignol%26aufirst%3DA.%26aulast%3DBuckler-White%26aufirst%3DA.%26aulast%3DBerkhout%26aufirst%3DB.%26aulast%3DJeang%26aufirst%3DK.%2BT.%26atitle%3DCharacterization%2520of%2520a%2520human%2520TAR%2520RNA-binding%2520protein%2520that%2520activates%2520the%2520HIV-1%2520LTR%26jtitle%3DScience%26date%3D1991%26volume%3D251%26spage%3D1597%26epage%3D1600%26doi%3D10.1126%2Fscience.2011739" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kilgore, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, D. F.</span></span> <span> </span><span class="NLM_article-title">Rubrisandrins A and B, lignans and related anti-HIV compounds from Schisandra rubriflora</span>. <i>J. Nat. Prod.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>69</i></span>,  <span class="NLM_fpage">1697</span>– <span class="NLM_lpage">1701</span>, <span class="refDoi"> DOI: 10.1021/np060239e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/np060239e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BD28Xht1WhsLzE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2006&pages=1697-1701&author=M.+Chenauthor=N.+Kilgoreauthor=K.+H.+Leeauthor=D.+F.+Chen&title=Rubrisandrins+A+and+B%2C+lignans+and+related+anti-HIV+compounds+from+Schisandra+rubriflora&doi=10.1021%2Fnp060239e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Rubrisandrins A and B, Lignans and Related Anti-HIV Compounds from Schisandra rubriflora</span></div><div class="casAuthors">Chen, Min; Kilgore, Nicole; Lee, Kuo-Hsiung; Chen, Dao-Feng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Natural Products</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1697-1701</span>CODEN:
                <span class="NLM_cas:coden">JNPRDF</span>;
        ISSN:<span class="NLM_cas:issn">0163-3864</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society-American Society of Pharmacognosy</span>)
        </div><div class="casAbstract">Bioactivity-directed fractionation of an ethanolic ext. of the fruits of Schisandra rubriflora led to the isolation and identification of dibenzocyclooctadiene lignans including the new lignans rubrisandrins A (1a + 1b) and B (2) and the known lignans gomisin J (3), (±)-gomisin M1 (4), (+)-gomisin M2 (5), schisanhenol (6), deoxyschisandrin, schisantherin B, schisandrin, tigloylgomisin P, gomisin O, angeloylgomisin P, and epigomisin O.  Their structure and stereochem. were detd. by spectroscopic methods, including 2D-NMR techniques.  Compds. 1 and 3-6 were active as anti-HIV agents.  (±)-Gomisin M1 (4) exhibited the most potent anti-HIV activity, with EC50 and therapeutic index (TI) values of <0.65 μM and >68, resp.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTBEiiLbO66bVg90H21EOLACvtfcHk0liNBOvuObJKtQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xht1WhsLzE&md5=048953e15712f9e93593d6489e2add33</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1021%2Fnp060239e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fnp060239e%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DM.%26aulast%3DKilgore%26aufirst%3DN.%26aulast%3DLee%26aufirst%3DK.%2BH.%26aulast%3DChen%26aufirst%3DD.%2BF.%26atitle%3DRubrisandrins%2520A%2520and%2520B%252C%2520lignans%2520and%2520related%2520anti-HIV%2520compounds%2520from%2520Schisandra%2520rubriflora%26jtitle%3DJ.%2520Nat.%2520Prod.%26date%3D2006%26volume%3D69%26spage%3D1697%26epage%3D1701%26doi%3D10.1021%2Fnp060239e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, J. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kashiwada, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, B. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cosentino, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K. H.</span></span> <span> </span><span class="NLM_article-title">Anti-AIDS agents--XXVI. Structure-activity correlations of Gomisin-G-related anti-HIV lignans from Kadsura interior and of related synthetic analogues</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">1715</span>– <span class="NLM_lpage">1723</span>, <span class="refDoi"> DOI: 10.1016/S0968-0896(97)00118-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1016%2FS0968-0896%2897%2900118-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=9313872" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADyaK2sXmsVGrs70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=1997&pages=1715-1723&author=D.+F.+Chenauthor=S.+X.+Zhangauthor=L.+Xieauthor=J.+X.+Xieauthor=K.+Chenauthor=Y.+Kashiwadaauthor=B.+N.+Zhouauthor=P.+Wangauthor=L.+M.+Cosentinoauthor=K.+H.+Lee&title=Anti-AIDS+agents%2D%2DXXVI.+Structure-activity+correlations+of+Gomisin-G-related+anti-HIV+lignans+from+Kadsura+interior+and+of+related+synthetic+analogues&doi=10.1016%2FS0968-0896%2897%2900118-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Anti-AIDS agents-XXVI. Structure-activity correlations of gomisin-G-related anti-HIV lignans from Kadsura interior and of related synthetic analogs</span></div><div class="casAuthors">Chen, Dao-Feng; Zhang, Shun-Xiang; Xie, Lan; Xie, Jing-Xi; Chen, Ke; Kashiwada, Yoshiki; Zhou, Bing-Nan; Wang, Pei; Cosentino, L. Mark; Lee, Kuo-Hsiung</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1715-1723</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Bioactivity-directed fractionation of an ethanolic ext. of the stems of Kadsura interior led to the isolation and identification of 12 known lignans.  Seven of these compds. were active as anti-HIV agents.  Gomisin-G exhibited the most potent anti-HIV activity with EC50 and therapeutic index (TI) values of 0.006 μg/mL and 300, resp.  Schisantherin-D, kadsuranin, and schisandrin-C showed good activity with EC50 values of 0.5, 0.8, and 1.2 μg/mL, and TI values of 110, 56, and 33.3, resp.  Ten related synthetic biphenyl compds., five variously substituted bismethylenedioxy, dimethoxy, and dimethoxycarbonyl isomers and five brominated derivs. also were evaluated for inhibitory activity against HIV-1 replication in acutely infected H9 cells.  The total syntheses of two new isomers are reported for the first time.  The anti-HIV data indicated that the relative position and types of substituents on the phenolic hydroxy groups of either the natural lignans or the synthetic biphenyl compds. rather than the nos. of bromine(s) on the arom. rings are of primary importance.  In the cyclooctane ring of the natural lignans, the position and substitution of hydroxy groups are also important to enhanced anti-HIV activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJsMP2Q4CfULVg90H21EOLACvtfcHk0liNBOvuObJKtQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXmsVGrs70%253D&md5=132467a2f62b457e709c311cc49e3897</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1016%2FS0968-0896%2897%2900118-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0968-0896%252897%252900118-1%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DD.%2BF.%26aulast%3DZhang%26aufirst%3DS.%2BX.%26aulast%3DXie%26aufirst%3DL.%26aulast%3DXie%26aufirst%3DJ.%2BX.%26aulast%3DChen%26aufirst%3DK.%26aulast%3DKashiwada%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DB.%2BN.%26aulast%3DWang%26aufirst%3DP.%26aulast%3DCosentino%26aufirst%3DL.%2BM.%26aulast%3DLee%26aufirst%3DK.%2BH.%26atitle%3DAnti-AIDS%2520agents--XXVI.%2520Structure-activity%2520correlations%2520of%2520Gomisin-G-related%2520anti-HIV%2520lignans%2520from%2520Kadsura%2520interior%2520and%2520of%2520related%2520synthetic%2520analogues%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D1997%26volume%3D5%26spage%3D1715%26epage%3D1723%26doi%3D10.1016%2FS0968-0896%2897%2900118-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nozaki-Renard, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iino, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sato, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marumoto, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohta, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furusawa, M.</span></span> <span> </span><span class="NLM_article-title">Fluoroquinolones protect the human lymphocyte CEM cell line from HIV-1-mediated cytotoxicity</span>. <i>Cell Struct. Funct.</i> <span class="NLM_year" style="font-weight: bold;">1990</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">295</span>– <span class="NLM_lpage">299</span>, <span class="refDoi"> DOI: 10.1247/csf.15.295</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1247%2Fcsf.15.295" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=2085844" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADyaK3MXhvF2isrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=1990&pages=295-299&author=J.+Nozaki-Renardauthor=T.+Iinoauthor=Y.+Satoauthor=Y.+Marumotoauthor=G.+Ohtaauthor=M.+Furusawa&title=Fluoroquinolones+protect+the+human+lymphocyte+CEM+cell+line+from+HIV-1-mediated+cytotoxicity&doi=10.1247%2Fcsf.15.295"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Fluoroquinolones protect the human lymphocyte CEM cell line from HIV-1-mediated cytotoxicity</span></div><div class="casAuthors">Nozaki-Renard, Junko; Iino, Takashi; Sato, Yoshinari; Marumoto, Yasumasa; Ohta, Genkichi; Furusawa, Mitsuru</div><div class="citationInfo"><span class="NLM_cas:title">Cell Structure and Function</span>
        (<span class="NLM_cas:date">1990</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">295-9</span>CODEN:
                <span class="NLM_cas:coden">CSFUDY</span>;
        ISSN:<span class="NLM_cas:issn">0386-7196</span>.
    </div><div class="casAbstract">Infection of the human lymphocyte CEM cell line with the HIV-1 (human immunodeficiency virus type-1, LAV-1 strain) results in cell death.  A fluoroquinolone antibiotic, ofloxacin, protected the infected cells from HIV-1-mediated cytolysis.  Other fluoroquinolones, e.g. ciprofloxacin, norfloxacin, and enoxacin, also protected the infected cells from HIV-1-mediated cytolysis.  The d-isomer of ofloxacin (DR-3354) was about 50-fold less effective than the l-isomer (DR-3355).  Almost none of the rescued cells had detectable HIV-antigens and they could be maintained for long periods in vitro without drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpInRxyWd8OybVg90H21EOLACvtfcHk0lh8UMkBcW6xhg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3MXhvF2isrg%253D&md5=5acc04da45143d08e6fcbd9c947abdfa</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1247%2Fcsf.15.295&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1247%252Fcsf.15.295%26sid%3Dliteratum%253Aachs%26aulast%3DNozaki-Renard%26aufirst%3DJ.%26aulast%3DIino%26aufirst%3DT.%26aulast%3DSato%26aufirst%3DY.%26aulast%3DMarumoto%26aufirst%3DY.%26aulast%3DOhta%26aufirst%3DG.%26aulast%3DFurusawa%26aufirst%3DM.%26atitle%3DFluoroquinolones%2520protect%2520the%2520human%2520lymphocyte%2520CEM%2520cell%2520line%2520from%2520HIV-1-mediated%2520cytotoxicity%26jtitle%3DCell%2520Struct.%2520Funct.%26date%3D1990%26volume%3D15%26spage%3D295%26epage%3D299%26doi%3D10.1247%2Fcsf.15.295" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref64"><div class="reference"><strong class="refLabel"><a href="#ref64" class="rightTabRefNumLink">64</a></strong><div class="NLM_citation" id="rightTab-cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Venkanna, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poornima, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siva, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Babu, B. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Babu, K. S.</span></span> <span> </span><span class="NLM_article-title">Towards the total synthesis of Schisandrene: stereoselective synthesis of the dibenzocyclooctadiene lignan core</span>. <i>Synlett</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">908</span>– <span class="NLM_lpage">911</span>, <span class="refDoi"> DOI: 10.1055/s-0036-1591539</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1055%2Fs-0036-1591539" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BC1cXivFGrs7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2018&pages=908-911&author=A.+Venkannaauthor=B.+Poornimaauthor=B.+Sivaauthor=B.+H.+Babuauthor=K.+S.+Babu&title=Towards+the+total+synthesis+of+Schisandrene%3A+stereoselective+synthesis+of+the+dibenzocyclooctadiene+lignan+core&doi=10.1055%2Fs-0036-1591539"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Towards the Total Synthesis of Schisandrene: Stereoselective Synthesis of the Dibenzocyclooctadiene Lignan Core</span></div><div class="casAuthors">Venkanna, Arramshetti; Poornima, Borra; Siva, Bandi; Babu, B. Hari; Babu, K. Suresh</div><div class="citationInfo"><span class="NLM_cas:title">Synlett</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">908-911</span>CODEN:
                <span class="NLM_cas:coden">SYNLES</span>;
        ISSN:<span class="NLM_cas:issn">0936-5214</span>.
    
            (<span class="NLM_cas:orgname">Georg Thieme Verlag</span>)
        </div><div class="casAbstract">A stereoselective synthesis of the dibenzocyclooctadiene lignan core, I, of the natural product schisandrene (II) is described.  Starting from readily available gallic acid, the synthetic strategy involves Suzuki-Miyaura cross-coupling, Stille reaction, and ring-closing metathesis (RCM) in the reaction sequence.  The required asym. center at C-7' was established by an asym. redn. of a keto compd. using the Corey-Bakshi-Shibata (CBS) catalyst.  In our approach, the eight-membered ring was achieved by RCM for the first time.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgMb9gXgSr-7Vg90H21EOLACvtfcHk0lh8UMkBcW6xhg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXivFGrs7k%253D&md5=5d43e9211c044c53fb752dce4c5bd969</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1055%2Fs-0036-1591539&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1055%252Fs-0036-1591539%26sid%3Dliteratum%253Aachs%26aulast%3DVenkanna%26aufirst%3DA.%26aulast%3DPoornima%26aufirst%3DB.%26aulast%3DSiva%26aufirst%3DB.%26aulast%3DBabu%26aufirst%3DB.%2BH.%26aulast%3DBabu%26aufirst%3DK.%2BS.%26atitle%3DTowards%2520the%2520total%2520synthesis%2520of%2520Schisandrene%253A%2520stereoselective%2520synthesis%2520of%2520the%2520dibenzocyclooctadiene%2520lignan%2520core%26jtitle%3DSynlett%26date%3D2018%26volume%3D29%26spage%3D908%26epage%3D911%26doi%3D10.1055%2Fs-0036-1591539" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref65"><div class="reference"><strong class="refLabel"><a href="#ref65" class="rightTabRefNumLink">65</a></strong><div class="NLM_citation" id="rightTab-cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Alam, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takaguchi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ito, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshida, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsuboi, S.</span></span> <span> </span><span class="NLM_article-title">Multi-functionalization of gallic acid towards improved synthesis of α- and β-DDB</span>. <i>Tetrahedron</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">1909</span>– <span class="NLM_lpage">1918</span>, <span class="refDoi"> DOI: 10.1016/j.tet.2004.11.083</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=10.1016%2Fj.tet.2004.11.083" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BD2MXoslGquw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2005&pages=1909-1918&author=A.+Alamauthor=Y.+Takaguchiauthor=H.+Itoauthor=T.+Yoshidaauthor=S.+Tsuboi&title=Multi-functionalization+of+gallic+acid+towards+improved+synthesis+of+%CE%B1-+and+%CE%B2-DDB&doi=10.1016%2Fj.tet.2004.11.083"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Multi-functionalization of gallic acid towards improved synthesis of α- and β-DDB</span></div><div class="casAuthors">Alam, Ashraful; Takaguchi, Yutaka; Ito, Hideyuki; Yoshida, Takashi; Tsuboi, Sadao</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1909-1918</span>CODEN:
                <span class="NLM_cas:coden">TETRAB</span>;
        ISSN:<span class="NLM_cas:issn">0040-4020</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The synthesis of mono-, di- and trisubstituted gallic acids and their ester with similar or different groups including different acetal and ketals is described.  Regioselective bromination on two ortho-positions of Me gallate, which is very crucial for many org. syntheses, was achieved in high yield and purity.  The α- (I) and β-DDB (II) were synthesized in high overall yield and purity from the regioselective bromo-derivs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8dD8akD52YrVg90H21EOLACvtfcHk0lh8UMkBcW6xhg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXoslGquw%253D%253D&md5=39d96d563db8b5c6cbce0ea22619c7ec</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1016%2Fj.tet.2004.11.083&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tet.2004.11.083%26sid%3Dliteratum%253Aachs%26aulast%3DAlam%26aufirst%3DA.%26aulast%3DTakaguchi%26aufirst%3DY.%26aulast%3DIto%26aufirst%3DH.%26aulast%3DYoshida%26aufirst%3DT.%26aulast%3DTsuboi%26aufirst%3DS.%26atitle%3DMulti-functionalization%2520of%2520gallic%2520acid%2520towards%2520improved%2520synthesis%2520of%2520%25CE%25B1-%2520and%2520%25CE%25B2-DDB%26jtitle%3DTetrahedron%26date%3D2005%26volume%3D61%26spage%3D1909%26epage%3D1918%26doi%3D10.1016%2Fj.tet.2004.11.083" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref66"><div class="reference"><strong class="refLabel"><a href="#ref66" class="rightTabRefNumLink">66</a></strong><div class="NLM_citation" id="rightTab-cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, G. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pu, J. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, H. D.</span></span> <span> </span><span class="NLM_article-title">Dibenzocyclooctadiene lignans from Schisandra wilsoniana and their anti-HIV-1 activities</span>. <i>J. Nat. Prod.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>73</i></span>,  <span class="NLM_fpage">915</span>– <span class="NLM_lpage">919</span>, <span class="refDoi"> DOI: 10.1021/np100067w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/np100067w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;key=1%3ACAS%3A528%3ADC%252BC3cXlvFKqtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2010&pages=915-919&author=G.+Y.+Yangauthor=Y.+K.+Liauthor=R.+R.+Wangauthor=X.+N.+Liauthor=W.+L.+Xiaoauthor=L.+M.+Yangauthor=J.+X.+Puauthor=Y.+T.+Zhengauthor=H.+D.+Sun&title=Dibenzocyclooctadiene+lignans+from+Schisandra+wilsoniana+and+their+anti-HIV-1+activities&doi=10.1021%2Fnp100067w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">Dibenzocyclooctadiene Lignans from Schisandra wilsoniana and Their Anti-HIV-1 Activities</span></div><div class="casAuthors">Yang, Guang-Yu; Li, Yin-Ke; Wang, Rui-Rui; Li, Xiao-Nian; Xiao, Wei-Lie; Yang, Liu-Meng; Pu, Jian-Xin; Zheng, Yong-Tang; Sun, Han-Dong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Natural Products</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">73</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">915-919</span>CODEN:
                <span class="NLM_cas:coden">JNPRDF</span>;
        ISSN:<span class="NLM_cas:issn">0163-3864</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society-American Society of Pharmacognosy</span>)
        </div><div class="casAbstract">Twelve new dibenzocyclooctadiene lignans, marlignans A-L (1-12), together with 16 known compds., were isolated from the leaves and stems of Schisandra wilsoniana.  The structures of 1-12 were elucidated by spectroscopic methods including 1D- and 2D-NMR techniques.  Compds. 1-12 were evaluated for their anti-HIV activities, of which compds. 3, 6, 8, and 12 showed modest activities with therapeutic index values of 13.2, 15.6, 17.6, and 16.4, resp.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxspLjw3yVFrVg90H21EOLACvtfcHk0lg6x9uKWppaKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXlvFKqtrY%253D&md5=f35c4b77e0a3ae4e9fa2f98aa5429a38</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1021%2Fnp100067w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fnp100067w%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DG.%2BY.%26aulast%3DLi%26aufirst%3DY.%2BK.%26aulast%3DWang%26aufirst%3DR.%2BR.%26aulast%3DLi%26aufirst%3DX.%2BN.%26aulast%3DXiao%26aufirst%3DW.%2BL.%26aulast%3DYang%26aufirst%3DL.%2BM.%26aulast%3DPu%26aufirst%3DJ.%2BX.%26aulast%3DZheng%26aufirst%3DY.%2BT.%26aulast%3DSun%26aufirst%3DH.%2BD.%26atitle%3DDibenzocyclooctadiene%2520lignans%2520from%2520Schisandra%2520wilsoniana%2520and%2520their%2520anti-HIV-1%2520activities%26jtitle%3DJ.%2520Nat.%2520Prod.%26date%3D2010%26volume%3D73%26spage%3D915%26epage%3D919%26doi%3D10.1021%2Fnp100067w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i63"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00018">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_55473"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.1c00018?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00018</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Additional figures illustrating the screening (Figure S1), the activities of GM (Figures S2–S5), miRNAomics and proteomics analysis (Figures S7–S14), the effect of GM on the expression of RISC elements (Figure S15), evidence that GM did not target Dicer or AGO2 (Figure S16), TRBP expression after transfection (Figure S17), activities of compounds <b>3</b>, <b>7</b>, <b>9</b>, and <b>10</b> in HeLa-miR-21-luciferase cells (Figure S18), the <i>K</i><sub>D</sub> values of compounds <b>3</b>, <b>7</b>, <b>9</b>, and <b>10</b> (Figure S19), the activity of GM analogues against SK-Hep-1 cells (Figure S20), a comparison of the activity of GM and compound <b>9</b> (Figure S21), and the effect of compound <b>21</b> in HeLa-miR-21-luciferase cells (Figure S22), evidence that compound <b>21</b> did not target PIWIL3 or TPO2β, miRNAomics and proteomics data (Tables S1–S3), sequences of primers (Tables S4 and S5), NMR spectra, and HPLC data of (−)-Gomisin M1 and <b>7</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00018/suppl_file/jm1c00018_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings of GM and <b>1–21</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00018/suppl_file/jm1c00018_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00018/suppl_file/jm1c00018_si_001.pdf">jm1c00018_si_001.pdf (3.58 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00018/suppl_file/jm1c00018_si_002.csv">jm1c00018_si_002.csv (2.4 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.1c00018&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2021.64.issue-11%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.1c00018%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.1c00018" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                8MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-b425991822a79ada59b3.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-fb88834b824b41b135ad.js"></script>

<script type="text/javascript" src="/wro/ke1q~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"66797ffa6b8b196f","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.6.0","si":10}'></script>
</body>
</html>
