
<html lang="en"     class="pb-page"  data-request-id="58211472-2dcf-4ba6-b2fa-bbf0d635823a"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;issue:issue:10.1021/jmcmar.2021.64.issue-7;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text;article:article:10.1021/acs.jmedchem.0c01689"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Targeting Lysosomal Degradation Pathways: New Strategies and Techniques for Drug Discovery" /></meta><meta name="dc.Creator" content="Junping  Pei" /></meta><meta name="dc.Creator" content="Guan  Wang" /></meta><meta name="dc.Creator" content="Lu  Feng" /></meta><meta name="dc.Creator" content="Jifa  Zhang" /></meta><meta name="dc.Creator" content="Tingting  Jiang" /></meta><meta name="dc.Creator" content="Qiu  Sun" /></meta><meta name="dc.Creator" content="Liang  Ouyang" /></meta><meta name="dc.Description" content="A series of tools for targeted protein degradation are inspiring scientists to develop new drugs with advantages over traditional small-molecule drugs. Among these tools, proteolysis-targeting chim..." /></meta><meta name="Description" content="A series of tools for targeted protein degradation are inspiring scientists to develop new drugs with advantages over traditional small-molecule drugs. Among these tools, proteolysis-targeting chim..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="March 25, 2021" /></meta><meta name="dc.Type" content="review-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.0c01689" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2021 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c01689" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.0c01689" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c01689" /></link>
        
    
    

<title>Targeting Lysosomal Degradation Pathways: New Strategies and Techniques for Drug Discovery | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c01689" /></meta><meta property="og:title" content="Targeting Lysosomal Degradation Pathways: New Strategies and Techniques for Drug Discovery" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01689/20210331/images/large/jm0c01689_0008.jpeg" /></meta><meta property="og:description" content="A series of tools for targeted protein degradation are inspiring scientists to develop new drugs with advantages over traditional small-molecule drugs. Among these tools, proteolysis-targeting chimeras (PROTACs) are most representative of the technology based on proteasomes. However, the proteasome has little degradation effect on certain macromolecular proteins or aggregates, extracellular proteins, and organelles, which limits the application of PROTACs. Additionally, lysosomes play an important role in protein degradation. Therefore, lysosome-induced protein degradation drugs can directly regulate protein levels in vivo, achieve the goal of treating diseases, and provide new strategies for drug discovery. Lysosome-based degradation technology has the potential for clinical translation. In this review, strategies targeting lysosomal pathways and lysosome-based degradation techniques are summarized. In addition, lysosome-based degrading drugs are described, and the advantages and challenges are listed. Our efforts will certainly promote the design, discovery, and clinical application of drugs associated with this technology." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.0c01689"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01689">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.0c01689&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.0c01689&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.0c01689&amp;href=/doi/10.1021/acs.jmedchem.0c01689" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2021</span><span class="cit-fg-volume">, 64</span><span class="cit-fg-issue">, 7</span><span class="cit-fg-pageRange">, 3493-3507</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/64/7" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><span class="content-navigation__contentType">Perspective</span><a href="/doi/10.1021/acs.jmedchem.0c01808" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Targeting Lysosomal Degradation Pathways: New Strategies and Techniques for Drug Discovery</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Junping Pei</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Junping Pei</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Biotherapy and Cancer Center, National Clinical Research Center for Geriatrics, Innovation Center of Nursing Research, Nursing Key Laboratory of Sichuan Province, West China Hospital, and Collaborative Innovation Center of Biotherapy, Sichuan University, Chengdu 610041, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Junping++Pei">Junping Pei</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath1">View Biography</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Guan Wang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Guan Wang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Biotherapy and Cancer Center, National Clinical Research Center for Geriatrics, Innovation Center of Nursing Research, Nursing Key Laboratory of Sichuan Province, West China Hospital, and Collaborative Innovation Center of Biotherapy, Sichuan University, Chengdu 610041, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Guan++Wang">Guan Wang</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath2">View Biography</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Lu Feng</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Lu Feng</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Biotherapy and Cancer Center, National Clinical Research Center for Geriatrics, Innovation Center of Nursing Research, Nursing Key Laboratory of Sichuan Province, West China Hospital, and Collaborative Innovation Center of Biotherapy, Sichuan University, Chengdu 610041, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Lu++Feng">Lu Feng</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath3">View Biography</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jifa Zhang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jifa Zhang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Biotherapy and Cancer Center, National Clinical Research Center for Geriatrics, Innovation Center of Nursing Research, Nursing Key Laboratory of Sichuan Province, West China Hospital, and Collaborative Innovation Center of Biotherapy, Sichuan University, Chengdu 610041, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jifa++Zhang">Jifa Zhang</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath4">View Biography</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Tingting Jiang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Tingting Jiang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Biotherapy and Cancer Center, National Clinical Research Center for Geriatrics, Innovation Center of Nursing Research, Nursing Key Laboratory of Sichuan Province, West China Hospital, and Collaborative Innovation Center of Biotherapy, Sichuan University, Chengdu 610041, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Tingting++Jiang">Tingting Jiang</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath5">View Biography</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Qiu Sun</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Qiu Sun</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Biotherapy and Cancer Center, National Clinical Research Center for Geriatrics, Innovation Center of Nursing Research, Nursing Key Laboratory of Sichuan Province, West China Hospital, and Collaborative Innovation Center of Biotherapy, Sichuan University, Chengdu 610041, China</div></div><span class="conrtib-corresp"><strong>*</strong>Q.S.: email, <a href="/cdn-cgi/l/email-protection#790a0c1708100c390a1a0c571c1d0c571a17"><span class="__cf_email__" data-cfemail="deadabb0afb7ab9eadbdabf0bbbaabf0bdb0">[email protected]</span></a></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Qiu++Sun">Qiu Sun</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath6">View Biography</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Liang Ouyang</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Liang Ouyang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Biotherapy and Cancer Center, National Clinical Research Center for Geriatrics, Innovation Center of Nursing Research, Nursing Key Laboratory of Sichuan Province, West China Hospital, and Collaborative Innovation Center of Biotherapy, Sichuan University, Chengdu 610041, China</div></div><span class="conrtib-corresp"><strong>*</strong>L.O.: email, <a href="/cdn-cgi/l/email-protection#79160c0018171e151018171e390a1a0c571c1d0c571a17"><span class="__cf_email__" data-cfemail="056a707c646b62696c646b62457666702b6061702b666b">[email protected]</span></a></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Liang++Ouyang">Liang Ouyang</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath7">View Biography</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-5537-8834" title="Orcid link">http://orcid.org/0000-0001-5537-8834</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c01689&amp;href=/doi/10.1021%2Facs.jmedchem.0c01689" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2021</span></span><span class="cit-volume">, 64</span><span class="cit-issue">, 7</span><span class="cit-pageRange">, 3493–3507</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">March 25, 2021</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>26 September 2020</li><li><span class="item_label"><b>Published</b> online</span>25 March 2021</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 8 April 2021</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.0c01689" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01689</a></div><div class="article_header-article-copyright"><strong>Copyright © 2021 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D3493%26pageCount%3D15%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DJunping%2BPei%252C%2BGuan%2BWang%252C%2BLu%2BFeng%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D64%26issueNum%3D7%26contentID%3Dacs.jmedchem.0c01689%26title%3DTargeting%2BLysosomal%2BDegradation%2BPathways%253A%2BNew%2BStrategies%2Band%2BTechniques%2Bfor%2BDrug%2BDiscovery%26numPages%3D15%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D3507%26publicationDate%3DApril%2B2021">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.0c01689"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">4193</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val">-</div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.0c01689" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Targeting Lysosomal Degradation Pathways: New Strategies and Techniques for Drug Discovery&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Junping&quot;,&quot;last_name&quot;:&quot;Pei&quot;},{&quot;first_name&quot;:&quot;Guan&quot;,&quot;last_name&quot;:&quot;Wang&quot;},{&quot;first_name&quot;:&quot;Lu&quot;,&quot;last_name&quot;:&quot;Feng&quot;},{&quot;first_name&quot;:&quot;Jifa&quot;,&quot;last_name&quot;:&quot;Zhang&quot;},{&quot;first_name&quot;:&quot;Tingting&quot;,&quot;last_name&quot;:&quot;Jiang&quot;},{&quot;first_name&quot;:&quot;Qiu&quot;,&quot;last_name&quot;:&quot;Sun&quot;},{&quot;first_name&quot;:&quot;Liang&quot;,&quot;last_name&quot;:&quot;Ouyang&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2021&quot;,&quot;month&quot;:&quot;03&quot;,&quot;day&quot;:&quot;25&quot;,&quot;issue&quot;:&quot;7&quot;,&quot;volume&quot;:&quot;64&quot;,&quot;pages&quot;:&quot;3493-3507&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.0c01689&quot;},&quot;abstract&quot;:&quot;A series of tools for targeted protein degradation are inspiring scientists to develop new drugs with advantages over traditional small-molecule drugs. Among these tools, proteolysis-targeting chimeras (PROTACs) are most representative of the technology based on proteasomes. However, the proteasome has little degradation effect on certain macromolecular proteins or aggregates, extracellular proteins, and organelles, which limits the application of PROTACs. Additionally, lysosomes play an important role in protein degradation. Therefore, lysosome-induced protein degradation drugs can directly regulate protein levels in vivo, achieve the goal of treating diseases, and provide new strategies for drug discovery. Lysosome-based degradation technology has the potential for clinical translation. In this review, strategies targeting lysosomal pathways and lysosome-based degradation techniques are summarized. In addition, lysosome-based degrading drugs are described, and the advantages and challenges are listed. Our e&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01689&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01689" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01689&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01689" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01689&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01689" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c01689&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01689&amp;href=/doi/10.1021/acs.jmedchem.0c01689" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.0c01689" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.0c01689" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (3 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01689&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.0c01689%26sid%3Dliteratum%253Aachs%26pmid%3D33764774%26genre%3Darticle%26aulast%3DPei%26date%3D2021%26atitle%3DTargeting%2BLysosomal%2BDegradation%2BPathways%253A%2BNew%2BStrategies%2Band%2BTechniques%2Bfor%2BDrug%2BDiscovery%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D64%26issue%3D7%26spage%3D3493%26epage%3D3507%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291781" title="Degradation">Degradation</a>,</li><li><a href="/action/doSearch?ConceptID=292544" title="Targeting">Targeting</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=292527" title="Autophagy">Autophagy</a>,</li><li><a href="/action/doSearch?ConceptID=291754" title="Molecules">Molecules</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/64/7" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/jmcmar.2021.64.issue-7/20210408/jmcmar.2021.64.issue-7.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01689/20210331/images/medium/jm0c01689_0008.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01689/20210331/images/large/jm0c01689_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01689&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">A series of tools for targeted protein degradation are inspiring scientists to develop new drugs with advantages over traditional small-molecule drugs. Among these tools, proteolysis-targeting chimeras (PROTACs) are most representative of the technology based on proteasomes. However, the proteasome has little degradation effect on certain macromolecular proteins or aggregates, extracellular proteins, and organelles, which limits the application of PROTACs. Additionally, lysosomes play an important role in protein degradation. Therefore, lysosome-induced protein degradation drugs can directly regulate protein levels <i>in vivo</i>, achieve the goal of treating diseases, and provide new strategies for drug discovery. Lysosome-based degradation technology has the potential for clinical translation. In this review, strategies targeting lysosomal pathways and lysosome-based degradation techniques are summarized. In addition, lysosome-based degrading drugs are described, and the advantages and challenges are listed. Our efforts will certainly promote the design, discovery, and clinical application of drugs associated with this technology.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_77726" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_77726" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Lysosomes are ubiquitous organelles that can degrade proteins, nucleic acids, polysaccharides, and other biomaterials.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> Extracellular substances and cytoplasmic membrane proteins (such as receptors or channels) can enter the cell through endocytosis and are then degraded in the lysosome, which contribute to plasma membrane repair, the immune response, bone resorption, and pathogen elimination in stores. Intracellular damaged proteins or organelles are wrapped by autophagosomes in autophagy. Then, autophagosomes fuse with lysosomes to form autolysosomes, which can degrade these cargoes and release the resulting short products back into the cytosol for reuse (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>).<a onclick="showRef(event, 'ref2 ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref2 ref3 ref4">(2−4)</a> The lysosomal membrane contains proteins that are involved in the import and export of substances to the lysosome.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> Lysosomal exocytosis not only is critical for the secretion of lysosomal content but also contributes to plasma membrane repair, the immune response, bone resorption, and pathogen elimination.<a onclick="showRef(event, 'ref6 ref7'); return false;" href="javascript:void(0);" class="ref ref6 ref7">(6,7)</a> The regulation of intracellular substance levels through lysosomes is of great significance for maintaining the normal metabolic activities of cells and defending against microbial infections.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> The lysosomal system and proteasome pathway are the two most significant degradation pathways in cells.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> Two pathways not only interact with each other through a certain mechanism to achieve compensatory or synergistic outcomes but also play an independent role to ensure the specificity of the degradation of biomaterials in cells.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> In particular, the lysosomal pathway plays an important role in regulating intracellular and extracellular protein homeostasis.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01689/20210331/images/medium/jm0c01689_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01689/20210331/images/large/jm0c01689_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Lysosomes participate in the degradation and recycling of intracellular and extracellular material through autophagy and endocytosis to provide energy and source molecules. Exocytosis contributes to plasma membrane repair, the immune response, bone resorption, and pathogen elimination in stores. Lysosomes can sense nutrients and activate metabolic signal transduction.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01689/20210331/images/large/jm0c01689_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01689&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Many diseases are caused by the mutation or accumulation of specific proteins. The specific degradation of these pathogenic proteins can fundamentally intervene in or treat diseases, which has important scientific and clinical value and is revolutionizing the biomedical field.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> In recent years, scientists have been inspired by the natural regulation of protein levels, which is the normal physiological process of protein metabolism in organismal cells.<a onclick="showRef(event, 'ref12 ref13 ref14'); return false;" href="javascript:void(0);" class="ref ref12 ref13 ref14">(12−14)</a> This result has led to the rapid emergence of protein-silencing tools, such as proteolysis-targeting chimeras (PROTACs)<a onclick="showRef(event, 'ref15 ref16 ref17'); return false;" href="javascript:void(0);" class="ref ref15 ref16 ref17">(15−17)</a> at the posttranslational level, biological tools (e.g., RNAi<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> and CRISPR<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a>), and other methods (e.g., Trim-Away,<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> dTAGs,<a onclick="showRef(event, 'ref21 ref22'); return false;" href="javascript:void(0);" class="ref ref21 ref22">(21,22)</a> and SNIPERs<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a>). Among these tools, PROTAC is the most potent method for drug discovery. Currently, three PROTAC molecules (ARV-110 in NCT03888612, ARV471 in NCT04072952, and CC-94676) have entered clinical trials. Some clinical trial data revealed that ARV110 has good oral bioavailability, a high safety profile, and adequate tolerance for use in cancer treatment. However, PROTACs can only degrade certain intracellular proteins. Lysosomes can effectively remove certain large proteins or aggregates, damaged or excess organelles and extracellular proteins (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>),<a onclick="showRef(event, 'ref1 ref24 ref25'); return false;" href="javascript:void(0);" class="ref ref1 ref24 ref25">(1,24,25)</a> that are closely related to cancer,<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> autoimmune disorders,<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> and/or aging-related diseases.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> Therefore, the strategy of lysosome-based selective degradation of biomaterials has the potential to improve human health. The future development and clinical transformation of lysosome-based degraders will lead to exciting and important developments in the treatment of diseases through protein degradation.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01689/20210331/images/medium/jm0c01689_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01689/20210331/images/large/jm0c01689_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Techniques for the targeted degradation of biomaterials. (A) PROTACs can degrade certain intracellular proteins through the proteasome. (B) New techniques harnessing the lysosome degradation pathway have the potential to degrade viruses, damaged or excess organelles, protein aggregates, DNA lesions, transmembrane proteins, extracellular proteins, intracellular proteins, and so on.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01689/20210331/images/large/jm0c01689_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01689&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Therefore, lysosome-based degradation of disease-related proteins can greatly enrich the techniques for inducing protein degradation and is complementary to proteasomal degradation.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> Autophagy agonists cannot specifically degrade abnormal proteins through lysosomes. Therefore, the development of a technique to specifically degrade disease-related proteins through lysosomes without affecting normal metabolized proteins, which represents a new strategy for the discovery of innovative drugs, is of great practical value. Here, we review the main ways to degrade biomaterials, including biological molecules and diseased organelles through lysosomes and the recent emergence of new techniques associated with this approach. Finally, we outline the achievements and challenges associated with targeted lysosomal degradation. In summary, new strategies for targeting protein and organelle degradation through the lysosomal pathway provide new ideas for drug discovery.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Degradation of Proteins and Organelles through the Lysosomal Pathway</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_67571" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_67571" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The balance of intracellular substances, the maintenance of cell homeostasis, and the normal cellular physiological functions of the body are inseparable from the lysosomal degradation pathway and proteasome system. These two pathways are independent but exhibit cross talk.<a onclick="showRef(event, 'ref30 ref31 ref32 ref33 ref34'); return false;" href="javascript:void(0);" class="ref ref30 ref31 ref32 ref33 ref34">(30−34)</a> On the one hand, the ubiquitin–proteasome system mainly degrades soluble,<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> short-lived, and monomeric misfolded proteins, while the lysosomal system mainly degrades long-lived proteins, protein aggregates, and damaged organelles.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> Both autophagy and ubiquitin–proteasome systems can degrade substrates through ubiquitination. On the other hand, the two pathways can synergistically degrade the same substrate in the cell. Additionally, because interference by one pathway affects the activity of the other, these pathways are interdependent.<a onclick="showRef(event, 'ref37 ref38 ref39'); return false;" href="javascript:void(0);" class="ref ref37 ref38 ref39">(37−39)</a> Therefore, research on lysosomal degradation <i>in vivo</i> broadens the understanding of the mechanism underlying cell degradation-related diseases. Protein degradation through lysosomal pathways has greatly expanded the range of molecules known to act as substrates, providing a new strategy for targeting many traditionally nontargeted proteins, such as backbone proteins and transcription factors, bacteria, and viruses. On the basis of lysosome-targeted receptors, endocytosis, and autophagy, more novel strategies can be developed to selectively reduce the degradation of biological materials <i>in vivo</i> through the lysosomal system.</div><div class="NLM_p">The transport of proteins to lysosomes can be hijacked by lysosome-targeting receptors (LTRs). The formation of lysosomes is a complex process involving multiple pathways, and different types of cells and even the same type of cell may adopt different pathways. Lysosomal hydrolases are synthesized in the endoplasmic reticulum and enter the lysosome in different ways.<a onclick="showRef(event, 'ref5 ref24 ref40'); return false;" href="javascript:void(0);" class="ref ref5 ref24 ref40">(5,24,40)</a> Currently, two main approaches are used: in one mechanism, most hydrolytic enzymes are modified with mannose 6-phosphate receptor (M6PR) residues, which are recognized by the Golgi apparatus, and finally transported to the lysosomal system, which is the most well-developed method in current research.<a onclick="showRef(event, 'ref5 ref41 ref42'); return false;" href="javascript:void(0);" class="ref ref5 ref41 ref42">(5,41,42)</a> In the other mechanism, lysosomal integral membrane protein 2 (LIMP-2) participates in the transport of glucocerebrosidase to lysosomes (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>).<a onclick="showRef(event, 'ref43 ref44 ref45'); return false;" href="javascript:void(0);" class="ref ref43 ref44 ref45">(43−45)</a> M6PR can effectively transport substrates to lysosomes and is widely expressed in tissues. Recently, it has been reported that receptors can be used to target lysosomes to successfully degrade extracellular proteins. Therefore, molecules that bind with high affinity for M6PR, LIMP-2, or other LTRs can be screened and linked to the target protein to achieve target protein degradation.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01689/20210331/images/medium/jm0c01689_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01689/20210331/images/large/jm0c01689_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Pathways for the targeted degradation of biological materials by lysosomes. The lysosome is an acidic organelle that can degrade intracellular and extracellular substructures through the activity of acidic hydrolases. Most of the newly synthesized lysosomal enzymes are transported into the lysosome through the M6PR in mammalian cells. However, β-glucocerebrosidase binds to LIMP-2 to target lysosomes. There are two main degradation pathways that involve lysosomes: the degradation of extracellular substances by endocytosis and the degradation of cytoplasmic proteins and organelles by autophagy including macroautophagy, microautophagy, and CMA.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01689/20210331/images/large/jm0c01689_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01689&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">There are two ways by which lysosomes clear protein aggregates, pathogens, and superfluous or damaged organelles in a selective manner. These two pathways are endocytosis and autophagy,<a onclick="showRef(event, 'ref46 ref47'); return false;" href="javascript:void(0);" class="ref ref46 ref47">(46,47)</a> both of which lead to the degradation of substances from different sources in the cell.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> Endocytosis is an important way to internalize cellular nutrients, access external information, and renew the plasma membrane.<a onclick="showRef(event, 'ref48 ref49'); return false;" href="javascript:void(0);" class="ref ref48 ref49">(48,49)</a> This pathway begins at the plasma membrane and ends at the lysosome. Autophagy is a highly conserved cell degradation process that can be divided into three major types: macroautophagy,<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> microautophagy, and CMA. Macroautophagy is a process in which aggregated proteins are detected by autophagy receptors such as p62/SQSTM1 and are selectively blocked in autophagosomes and then fused with lysosomes through lysosomal hydrolases.<a onclick="showRef(event, 'ref51 ref52 ref53'); return false;" href="javascript:void(0);" class="ref ref51 ref52 ref53">(51−53)</a> In macroautophagy, LC3, located on the autophagosomal membrane and in the autophagosome, can connect with a particular target protein and autophagosome. When we design a small-molecule drug targeting lysosomes, we can make full use of the bridge produced by LC3, which connects the target protein and the autophagosome. Microautophagy is a phagocytic process in which the lysosomal membrane directly engulfs intradermal cytoplasmic components. CMA is a unique highly selective autophagy mechanism that can only degrade certain specific proteins but not organelles. The most significant feature of CMA is the degradation of protein substrates with the KFERQ sequence.<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> To date, this specific CMA-targeting motif (CTM) has been found in all CMA substrates. Autophagy proceeds in all eukaryotic cells and is abnormally expressed in many human diseases, such as cancer, neurodegenerative diseases, and metabolic diseases.<a onclick="showRef(event, 'ref55 ref56'); return false;" href="javascript:void(0);" class="ref ref55 ref56">(55,56)</a> Therefore, the use of autophagy to degrade certain target proteins may have potential applications to drug discovery.</div><div class="NLM_p last">By studying how lysosomal biogenesis and lysosome-related components involved in this process mediate the different pathways of lysosome internalization and how the mechanisms of these pathways can be regulated, we can find new ways to target lysosomes. New techniques have been developed to target proteins and organelles for degradation through the lysosomal pathway. The development of lysosome-based degradation technique has good prospects for the treatment of diseases, especially neurodegenerative diseases.<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i4">Lysosome-Based Degradation Techniques</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_82387" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_82387" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Using the human lysosomal system to target protein degradation can lead to the degradation of various extracellular proteins, transmembrane proteins, protein aggregates, and organelles, thus greatly expanding the range of potential drug targets.<a onclick="showRef(event, 'ref58 ref59 ref60 ref61 ref62 ref63 ref64 ref65'); return false;" href="javascript:void(0);" class="ref ref58 ref59 ref60 ref61 ref62 ref63 ref64 ref65">(58−65)</a> Therefore, increasing attention has been given to lysosome-based degradation technique. The lysosomal degradation pathway includes the endocytosis/lysosome pathway and autophagy pathway.<a onclick="showRef(event, 'ref66 ref67'); return false;" href="javascript:void(0);" class="ref ref66 ref67">(66,67)</a> On the basis of endocytosis and different forms of autophagy, the techniques that target lysosomal degradation pathways can be divided into the following four types.</div><div id="sec3_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Lysosome-Targeting Chimeras (LYTACs) for Drug Discovery</h3><div class="NLM_p">The cation-independent mannose 6-phosphate receptor (CI-M6PR) is a representative receptor in the LTR family that can transport proteins to lysosomes.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> CI-M6PR uses its constitutive circulation in the Golgi complex and endosomes and at the cell surface to induce the endocytosis and lysosomal degradation of <i>N</i>-glycan proteins capped with mannose 6-phosphate residues.<a onclick="showRef(event, 'ref68'); return false;" href="javascript:void(0);" class="ref ref68">(68)</a> Banik et al. developed a lysosome-targeting chimera (LYTAC) by using CI-M6PR, which successfully led to the transport and degradation of target proteins (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>).<a onclick="showRef(event, 'ref58 ref69'); return false;" href="javascript:void(0);" class="ref ref58 ref69">(58,69)</a> These LYTACs are made by the fusion of antibodies to the target protein with chemically synthesized <i>N</i>-carboxyanhydride (NCA)-derived glycopolypeptides, which can target CI-M6PR.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01689/20210331/images/medium/jm0c01689_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01689/20210331/images/large/jm0c01689_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Extracellular proteins and transmembrane proteins can be degraded by LYTACs, which are composed of target protein antibodies and ligands for target LTRs. In addition, the asialoglycoprotein receptor (ASGPR) has tissue specificity; therefore, LYTACs targeting ASGPR can degrade transmembrane proteins in specific tissues.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01689/20210331/images/large/jm0c01689_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01689&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">First, an NCA glycan peptide with an M6P branch and a specific antibody target protein binding to an LTR were designed and optimized. Different LYTACs were synthesized, in which the binding part of a targeted LTR consisted of NCA glycan peptides with different sizes and linkers, and the part with the targeting protein was NeutrAvidin-647 (NA-647) with an Alexa Fluor-647-labeled protein linked to biotin. Biotinylated LYTAC, created by optimizing NCA glycan peptides, can facilitate NA-647 uptake in various cell lines, such as breast cancer, cervical cancer, lymphoma, and leukemia cell lines. These outcomes show that the optimized LTR-targeting ligands can localize the target protein to the lysosome. Next, the effectiveness of the targeted protein was verified, and the first LYTAC molecule Ab-1 was synthesized by covalently coupling poly-serine-<i>O</i>-mannose-6-phosphonate (M6Pn) and polyclonal anti-mouse IgG. When Ab-1 was cultured with K562 cells and mouse IgG labeled with Alexa Fluor 488, the Alexa Fluor 488 fluorescence signal in the lysosome increased 40-fold compared with that in the control group. In addition, Ab-1 transported the mCherry protein and its antibody into the cell. Therefore, LYTAC can relocate the antibody from its normal site of immune function and achieve targeted extracellular protein degradation through the lysosome pathway.</div><div class="NLM_p">To determine whether LYTAC can induce degradation of the target protein via lysosomes, the LYTAC Ab-1 was found to transport ApoE4<a onclick="showRef(event, 'ref70'); return false;" href="javascript:void(0);" class="ref ref70">(70)</a> (a protein involved in neurodegenerative diseases) into lysosomes for further degradation. In addition, when mouse anti-CD71 antibody and Ab-1 are incubated in the cells, CD71<a onclick="showRef(event, 'ref71 ref72'); return false;" href="javascript:void(0);" class="ref ref71 ref72">(71,72)</a> is internalized and degraded by lysosomes. Eighty-one percent of CD71 was degraded after treatment with 50 nM anti-CD71 and Ab-1 in Jurkat cells for 24 h. Given these conclusions, LYTAC Ab-2 and LYTAC Ab-3 were designed, and these agents achieved the degradation of EGFR and PD-L1, respectively. Seventy-six percent of EGFR was degraded after treatment with 10 nM Ab-2 for 48 h in dCas9-KRAB HeLa cells. PD-L1 was downregulated with 73% after treatment with 25 nM atz-LYTAC for 48 h in HDLM-2 cells. Moreover, LYTAC Ab-2 constructed by covalently coupling poly(M6Pn) and the EGFR antibody cetuximab<a onclick="showRef(event, 'ref73 ref74'); return false;" href="javascript:void(0);" class="ref ref73 ref74">(73,74)</a> can specifically degrade EGFR in a variety of cell lines, including HeLa cells and in breast cancer (BT474 and MDA-MB-361) and lung cancer (A549) cell lines, with no off-target effects. Notably, PD-L1 is the first type of transmembrane protein, with a size of 40 kDa, that helps cancer cells evade the immune system. When using anti-PD-L1-M6Pn LYTAC Ab-3 to treat PD-L1-positive MDA-MB-231 cells, a significant decrease in the level of PD-L1 on the cell surface was observed.</div><div class="NLM_p">CI-M6PR is one of many LTRs and is expressed in most tissues. The expression of some receptors in the LTR is tissue-specific. The design of molecules to target tissue-specific LTRs can lead to the degradation of target proteins in specific tissues and achieve precise treatment. Inspired by M6Pn-LYTAC, a research group developed an LYTAC targeting ASGPR, which is an LTR with tissue specificity. LYTAC is composed of an antibody that recognizes a target protein and glycoproteins bearing <i>N</i>-acetylgalactosamine (GalNAc) that target ASGPR.<a onclick="showRef(event, 'ref59 ref75 ref76'); return false;" href="javascript:void(0);" class="ref ref59 ref75 ref76">(59,75,76)</a> Since the expression of ASGPR is concentrated in the liver, GalNAc-LYTAC can achieve liver-specific target protein degradation.<a onclick="showRef(event, 'ref77'); return false;" href="javascript:void(0);" class="ref ref77">(77)</a> Researchers cocultured liver cancer cells with other cells that do not express ASGPR and confirmed that GalNAc-LYTAC has cell specificity. GalNAc-LYTAC successfully induces the degradation of EGFR and HER2 in hepatocarcinoma cell lines, and this process depends on the lysosomal system of the cell and the internalization of ASGPR. Although a large number of target proteins are transported to lysosomes, lysosomal homeostasis is not destroyed, and other cell functions are not affected. In addition, because the expression of ASGPR on the surface of liver cells is higher than that of CI-M6PR, the degradation ability of GalNAc-LYTAC is stronger than that of M6Pn-LYTAC. In summary, the LYTAC technique targeting ASGPR is feasible, effective, and specific.</div><div class="NLM_p last">In conclusion, the LYTAC technique can directly target extracellular and membrane-associated proteins for degradation via the lysosomal pathway. LYTAC is a good supplement to PROTACs, which use the ubiquitin degradation system and can only degrade intracellular proteins. Extracellular and membrane-associated proteins account for 40% of protein-coding genes<a onclick="showRef(event, 'ref78'); return false;" href="javascript:void(0);" class="ref ref78">(78)</a> and are key factors in aging-related diseases, autoimmune diseases, and cancer; therefore, strategies for targeted degradation of these proteins have the potential to treat human diseases and improve human health. Although the aforementioned LYTAC makes full use of CI-M6PR, other recyclable receptors can also be used in principle. It is reasonable and necessary to find and study other recoverable ligands. In addition, the antibody has strong specificity and a low mistargeting rate, but this method also requires an antibody to maintain its characteristics when binding with recyclable ligands with effective targeting.</div></div><div id="sec3_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Autophagy-Targeting Chimeras (AUTACs) for Drug Discovery</h3><div class="NLM_p">Yoshimori et al. found that cells can activate autophagy to specifically remove pathogens when they are invaded by cytoplasmic group A streptococci.<a onclick="showRef(event, 'ref79'); return false;" href="javascript:void(0);" class="ref ref79">(79)</a> Dr. Arimoto later discovered that <i>S</i>-guanylation is involved in the recognition events leading to the above-mentioned antibacterial autophagy.<a onclick="showRef(event, 'ref80'); return false;" href="javascript:void(0);" class="ref ref80">(80)</a> The endogenous nucleotide 8-nitroguanosine 3′,5′-cyclic monophosphate (8-nitro-cGMP) is a signal for cells to recruit autophagosomes;<a onclick="showRef(event, 'ref80 ref81 ref82'); return false;" href="javascript:void(0);" class="ref ref80 ref81 ref82">(80−82)</a> therefore, Takahashi et al. designed a small molecule named autophagy-targeting chimera (AUTAC) that recruits autophagosomes specifically based on endogenous cGMP modification (<i>S</i>-guanylation); this molecule consists of a tag for recruiting autophagosomes (guanine derivatives), a linker, and a ligand for targeting proteins or organelles for binding. Using this method, they successfully degraded cytoplasmic proteins and fragmented mitochondria (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>).<a onclick="showRef(event, 'ref60 ref83'); return false;" href="javascript:void(0);" class="ref ref60 ref83">(60,83)</a></div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01689/20210331/images/medium/jm0c01689_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01689/20210331/images/large/jm0c01689_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. AUTAC degrades cytoplasmic proteins and damaged mitochondria by autophagy. The combination of AUTAC with cytoplasmic proteins or mitochondria can induce K63 polyubiquitination. The ubiquitinated substrates are subsequently recognized by the autophagy receptor SQSTM1/p62, interact with LC3, and are recruited into autophagosomes to be degraded through the selective autophagy pathway. The design and synthesis of four AUTAC molecules that lead to substrate degradation are shown.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01689/20210331/images/large/jm0c01689_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01689&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The authors originally designed a cGMP-containing ligand probe to combine guanosine 3′,5′-cyclic monophosphate (cGMP) with a Halo tag. The application of this probe to EGFP indicated that it can induce EGFP <i>S</i>-guanylation and transport the protein to autophagosomes for degradation. However, due to the slow process of EGFP degradation by this probe, an analog probe, the <i>p</i>-fluorobenzylguanine ligand, was developed through optimization. The new probe has strong binding ability and undergoes rapid reaction.</div><div class="NLM_p">To verify the effectiveness of AUTACs in degrading proteins, researchers have used new probes to develop three small molecules, <b>1</b>, <b>2</b>, and <b>3</b>, that can target degraded proteins. These researchers successfully designed and synthesized compound <b>1</b> to degrade methionine aminopeptidase 2 (MetAP-2).<a onclick="showRef(event, 'ref84 ref85'); return false;" href="javascript:void(0);" class="ref ref84 ref85">(84,85)</a> As a bifunctional small molecule, compound <b>1</b> is mainly composed of three parts: <i>p</i>-fluorobenzylguanine (FBnG), a polyethylene glycol linker, and the hydrolysis product of fumagillol.<a onclick="showRef(event, 'ref86 ref87 ref88'); return false;" href="javascript:void(0);" class="ref ref86 ref87 ref88">(86−88)</a> The degradation tag of <b>1</b> is <i>p</i>-fluorobenzylguanine, which can specifically recruit autophagosomes. The covalently bound part of <b>1</b> and MetAP-2 is a hydrolysate of fumagillol. The result demonstrated that more than 80% of extant MetAP-2 was successfully degraded in HeLa cells after treatment with 10 μM AUTAC. When the lysosomal inhibitor bafilomycin A1 was added, the silencing of MetAP-2 was inhibited, which indicated that the degradation mechanism was accomplished by autophagy. FK506-binding protein<a onclick="showRef(event, 'ref89 ref90'); return false;" href="javascript:void(0);" class="ref ref89 ref90">(89,90)</a> (FKBP12) participates in the release of intracellular Ca<sup>2+</sup>, which is related to immune suppression and nerve cell regeneration.<a onclick="showRef(event, 'ref91'); return false;" href="javascript:void(0);" class="ref ref91">(91)</a> In HeLa cells, compound <b>2</b> at a concentration of 10 μM successfully degraded FKBP12. In the design of compound <b>2</b>, SLF (noncovalently targeting FKBP) was selected as the warhead for FKBP12, and it was connected to the <i>p</i>-fluorobenzylguanine ligand recruiting the autophagosome through a polyethylene glycol linker. Since the upregulation of bromodomain-containing protein 4<a onclick="showRef(event, 'ref92'); return false;" href="javascript:void(0);" class="ref ref92">(92)</a> (BRD4) expression is closely related to the occurrence and development of lung cancer, breast cancer, acute myeloid lymphoma, liver cancer, and so on,<a onclick="showRef(event, 'ref93 ref94 ref95 ref96'); return false;" href="javascript:void(0);" class="ref ref93 ref94 ref95 ref96">(93−96)</a> the authors synthesized compound <b>3</b>, which was joined by the BRD4 inhibitors JQ1 and FBnG and named <b>3</b>, to study its degradation effect. However, the results showed that the levels of BRD4 were only slightly reduced in A549 cells treated with 10 μM compound <b>3</b>.</div><div class="NLM_p">In addition, compound <b>4</b>, which can deliver a guanine tag to the mitochondrial membrane, was prepared. It functions with a 2-phenylindole derivative as the binding agent for targeting mitochondria, and FBnG is connected to the other end by a polyethylene glycol linker unit. 2-Phenylindole 3-glyoxyamide is a ligand for translocator proteins on the outer mitochondrial membrane.<a onclick="showRef(event, 'ref97 ref98'); return false;" href="javascript:void(0);" class="ref ref97 ref98">(97,98)</a> From the perspective of the autophagy substrate range, mitochondria were degraded after treatment with 10 μM <b>4</b>. Mitochondrial dysfunction is associated with many aging-related diseases; therefore, removing broken mitochondria may ameliorate these disease conditions. Targeted mitochondrial <b>4</b> (mito-AUTAC) treatment can significantly improve the mitochondrial activity of fibroblasts in Down’s syndrome. Compound <b>4</b> targeting mitochondria not only can accelerate the specific clearance of dysfunctional mitochondria in fibroblasts derived from patients but also can stimulate cells to assemble new mitochondria. At the same time, a cytoprotective effect against acute mitochondrial injury was also observed.</div><div class="NLM_p last">These favorable results prove the feasibility of using AUTACs; therefore, AUTACs may have a wide range of applications in the discovery of autophagy-related drugs in the future. Despite these achievements, there are still some major challenges to be addressed. For example, the application of AUTACs to the removal of protein aggregates has not been tested. Although similar to PROTAC action, the AUTAC-mediated autophagy mechanism may be more complex than that of a PROTAC. Obviously, we need to have an in-depth understanding of the working mechanism of AUTACs. The modes of action of AUTACs differ from those of similar PROTACs that induce ubiquitin ligases to approach a substrate. In selective autophagy, substrate selectivity is usually achieved by ubiquitination of the substrate or direct recognition of the autophagy receptor. The mechanism of action of AUTAC degradation tags also needs to be clarified in detail. Furthermore, targeted protein degradation by PROTACs is completed within 1 h,<a onclick="showRef(event, 'ref99'); return false;" href="javascript:void(0);" class="ref ref99">(99)</a> while targeted protein degradation by AUTACs takes several hours; therefore, the degradation rates of AUTACs need to be further optimized. Finally, AUTACs have been successfully applied to several cytoplasmic proteins, and the effect of nuclear proteins generally needs to be further studied and confirmed to better guide the wide application of AUTACs.</div></div><div id="sec3_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> Autophagosome-Tethering Compounds (ATTECs) for Drug Discovery</h3><div class="NLM_p">To search for a small-molecule agent that degrades mutant huntingtin protein (mHTT) through autophagy, Lu and Fei’s group screened nearly 4000 small-molecule compounds through a new high-throughput compound screening platform based on small-molecule microarrays (SMMs) and oblique-incidence reflectivity difference (OI-RD) technology<a onclick="showRef(event, 'ref100 ref101 ref102 ref103 ref104 ref105'); return false;" href="javascript:void(0);" class="ref ref100 ref101 ref102 ref103 ref104 ref105">(100−105)</a> to identify <b>5</b> (3-3-((3,5-dibromo-4-hydroxyphenyl)methylidene)-5-iodo-1<i>H</i>-indol-2-one) and <b>6</b> (<i>N-</i>(3-aminopropyl)-<i>N</i>-((1<i>R</i>)-1-(7-chloro-4-oxo-3-(phenylmethyl)-2-quinazolinyl)-2-methylpropyl)-4-methylbenzamide) (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>).<a onclick="showRef(event, 'ref61'); return false;" href="javascript:void(0);" class="ref ref61">(61)</a><b>5</b> and <b>6</b> selectively degraded mHTT but spared the wild-type HTT protein. These two compounds can bind both the LC3 protein and mutant huntingtin protein but not wild-type huntingtin protein. Through the detection of small-molecule compounds with similar features, the authors obtained two other ATTECs, <b>7</b> and <b>8</b>, which can be referred to as small-molecule glue. ATTECs can specifically bind the key autophagy protein LC3 and the pathogenic protein without binding to the normal protein and specifically package the pathogenic material into the autophagosome for degradation. This research has finally achieved selective degradation of the pathogenic mutant huntingtin protein and can be applied to other polyQ diseases. This method is expected to provide new opportunities for the clinical treatment of Huntingtin’s disease.</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01689/20210331/images/medium/jm0c01689_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01689/20210331/images/large/jm0c01689_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. ATTEC small molecules can effectively bind LC3 and autophagosomes to degrade pathogenic proteins by autophagy. The researchers first obtained four mHTT–LC3 linker compounds through high-throughput screening and then performed targeted mHTT degradation tests <i>in vivo</i> and <i>in vitro</i>. Fluorescence labeling verified that ATTECs tether mHTT to autophagosomes. Finally, through rotating tripods, balance beams, and gripping power tests, ATTEC can ameliorate the HD-related phenotype.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01689/20210331/images/large/jm0c01689_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01689&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The mechanism of action of these small molecules was verified. Small molecules can indeed reduce the level of mHTT in fibroblasts from HD patients and neurons derived from induced pluripotent stem cells (iPS cells), as shown by the homogeneous time-resolved fluorescence (HTRF) assay,<a onclick="showRef(event, 'ref106 ref107'); return false;" href="javascript:void(0);" class="ref ref106 ref107">(106,107)</a> but have no effect on the wild-type HTT protein in fibroblasts obtained from healthy people or patients with Parkinson’s disease. The authors demonstrated that small molecules mediate mHTT degradation through autophagy by knocking out the key gene ATG5.<a onclick="showRef(event, 'ref108'); return false;" href="javascript:void(0);" class="ref ref108">(108)</a> In addition, through the detection of the autophagy-selective substrate protein SQSTM1 (p62) and the control protein level, it was determined that small-molecule glue had no effect on the autophagy process. Phenotypic experiments were carried out in an HD transgenic fly model and a mouse model expressing mHTT. In transgenic flies expressing full-length HTT (Q128) driven by elav-GAL4, feeding the indicated compounds at 10 μM for 6 days led to significantly reduced mHTT levels. The cortices from Hdh<sup><i>Q7/Q140</i></sup>-knock-in mice intracerebroventricularly treated with one dose of <b>5</b>, <b>7</b>, or <b>8</b> (2 μL at 25 μM for each mouse) per day for 10 days showed significant degradation of mHTT. <b>5</b> and <b>8</b> can cross the blood–brain barrier and enter the brain. They can directly reduce mutant huntingtin protein levels in the cerebral cortex and striatum of mice with Huntingtin’s disease upon intraperitoneal injection at 0.5 mg kg<sup>–1</sup> without affecting the wild-type huntingtin protein levels in brain tissue. In addition, ATTECs can ameliorate HD-relevant phenotypes. ATTECs provide new strategies for the development of oral therapeutic drugs for Huntingtin’s disease.</div><div class="NLM_p">The internal mechanism of mHTT–LC3 linker compounds that can distinguish between mutant and wild-type HTT proteins was investigated. The mutant and wild-type HTT proteins are almost identical except for the difference in the length of an expanded polyglutamine (polyQ) tract. The results of this specificity study indicated that these linking compounds can recognize the conformation of the expanded polyQ stretch in the mHTT protein to distinguish between these two proteins. Consistent with this prediction, the researchers used the spinocerebellar ataxia type 3 cell line harboring the mutant ataxin-3 with an expanded polyQ region containing 74 glutamines to find that these linking compounds can effectively degrade the ataxin-3 protein mutants without affecting the concentration of wild-type ataxin-3.</div><div class="NLM_p last">Lu et al.<a onclick="showRef(event, 'ref61'); return false;" href="javascript:void(0);" class="ref ref61">(61)</a> demonstrated the idea of autophagic degradation of proteins (such as mHTT) by binding ATTEC compounds with LC3. The next key step in this new drug development strategy is to study in detail the interface between the compound and LC3 through medicinal chemistry and structural chemistry and to develop chemical molecules that match the interface without affecting the function. In addition, according to the existing common core structure, more effective small-molecule glues can be created to optimize the structure of the compound using strategies such as bioelectronics and skeleton transitions in pharmaceutical chemistry. Furthermore, in the absence of a known structure for these compounds, it is still possible to design compounds that specifically interact with mHTT and LC3 through coupled linkers identified by high-throughput screening of SMMs and OI-RD technology. However, there remain some questions requiring further study. It is not clear whether the ATTEC molecule damages normal brain proteins due to the increase in autophagy it induces or whether there are off-target effects, which may cause secondary damage to the nervous system.<a onclick="showRef(event, 'ref109 ref110'); return false;" href="javascript:void(0);" class="ref ref109 ref110">(109,110)</a></div></div><div id="sec3_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Chaperone-Mediated Autophagy (CMA) Targets Protein Degradation through Lysosomes for Drug Discovery</h3><div class="NLM_p">Molecular chaperone-mediated autophagy is the molecular chaperone HSC70 that recognizes a soluble cytoplasmic protein substrate with a KFERQ sequence (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>). After the chaperone–substrate complex binds to lysosome-associated membrane protein 2a on the lysosomal membrane, the substrate is transported into the lysosome via the HSC70 located in the lysosomal cavity, which is decomposed by a hydrolase and then reused by the cell.<a onclick="showRef(event, 'ref111 ref112 ref113'); return false;" href="javascript:void(0);" class="ref ref111 ref112 ref113">(111−113)</a> Bauer et al. showed the expression of a 46 amino acid-peptide chain containing two different HSC70 binding motifs and two copies of the polyglutamine-binding peptide 1 sequence (QBP1),<a onclick="showRef(event, 'ref114 ref115'); return false;" href="javascript:void(0);" class="ref ref114 ref115">(114,115)</a> which can recognize the expanded polyQ sequence in the mutant huntingtin protein without binding to the normal mutant huntingtin protein polyQ motif.<a onclick="showRef(event, 'ref62'); return false;" href="javascript:void(0);" class="ref ref62">(62)</a> This peptide can specifically degrade mHTT with extended polyQ fragments via CMA. CMA promotes the specific degradation of HTT with expanded polyQ in Neuro2a cells expressing this construct. The results from experiments with R6/2 and HD190Q-EGFP mouse HD models clearly show that expressing a fusion molecule effectively reduces the expansion of HTT aggregation <i>in vivo</i>, and the reduction in HTT aggregation is accompanied by a decrease in the solubility level of polyQ. An <i>in vivo</i> study revealed that this molecule can ameliorate the pathology and phenotype of the HD model mice. To explain this effect, it is proposed that a peptide-like molecule with the HSC70-binding motif fused with a specific structural binding domain that can target the protein of interest (POI) may have the potential to treat diseases caused by a mutant protein.</div><figure id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01689/20210331/images/medium/jm0c01689_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01689/20210331/images/large/jm0c01689_0007.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Targeting proteins for the internalization and degradation by chaperone-mediated autophagy (CMA). The amino acid–peptide adaptor molecule fused with a tag for the target protein of interest to a CMA sorting signal containing KFERQ.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01689/20210331/images/large/jm0c01689_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01689&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Wang’s group used CMA to degrade endogenous proteins to design a peptide system.<a onclick="showRef(event, 'ref63'); return false;" href="javascript:void(0);" class="ref ref63">(63)</a> The peptide consists of three parts: a cell membrane penetrating domain (CMPD) that can be delivered across the blood–brain barrier and plasma membrane, a protein-binding domain (PBD) that specifically binds to the target protein, and CTM that can target lysosomes to hydrolyze proteins. When the targeting peptide is applied to the cell environment, it enters the cell through its CMPD and then specifically recognizes and forms a stable complex with its target protein through the interaction between PBD and CTM. Finally, the complex is translocated to the lysosome through the CMA pathway for rapid degradation. The team designed three of these targeted peptides that can degrade the scaffolding protein PSD-95, death-associated protein kinase 1 (DAPK1), and α-synuclein. Pretreatment of H<sub>2</sub>O<sub>2</sub>-induced neurons with DAPK1-targeting peptide (100 μM) resulted in a 63.46% reduction in DAPK1 levels. In a well-characterized rat model of focal ischemia, treatment with DAPK1-targeting peptide at 10 mg kg<sup>–1</sup> reduced DAPK1 levels compared with the control. The results confirmed that the target peptide can knock out the protein of interest not only in primary neurons but also in rat brains and can cross the blood–brain barrier, as expected. In addition, Zhou et al. synthesized a peptide that can induce the degradation of cyclin-dependent kinase 5 (CDK5) through the specific binding of the partner-mediated autophagy-targeting motif (Tat-CDK5-CTM) with CDK5.<a onclick="showRef(event, 'ref64'); return false;" href="javascript:void(0);" class="ref ref64">(64)</a> Tests confirmed that this peptide not only can prevent calcium overload and the death of cultured neurons but also can reduce infarct size and neuronal loss in mice with middle cerebral artery occlusion at 1 mg/kg and thus improve neurological function. This type of targeted peptide-directed lysosomal degradation provides a new method for regulating endogenous protein levels.</div><div class="NLM_p">Xu’s team discovered the PD-L1 lysosomal degradation pathway and its regulatory factor HIP1R.<a onclick="showRef(event, 'ref65'); return false;" href="javascript:void(0);" class="ref ref65">(65)</a> HIP1R can directly interact with PD-L1 and deliver it for lysosomal degradation through the lysosomal targeted signaling pathway. They also found that the interaction between HIP1R and PD-L1 depends on the structural motif of PD-L1 and the signal peptide directed to the lysosome. Using this principle, this research team first designed peptides containing an epitope tag obtained for PD-L1 binding from HIP1R and a CMA sorting signal with the sequence KFERQQKILDQRFFE. The fusion peptides significantly reduced the level of PD-L1. Then, the researchers designed the PD-LYSO polypeptide. This synthetic peptide contains the region of HIP1R that binds to PD-L1 and the sequence that mediates the transport of PD-L1 to the lysosome and can target PD-L1 expressed by tumor cells and carry it to lysosomes for degradation.</div><div class="NLM_p last">Peptide-targeting chimeras provide powerful tools for scientific research through the CMA degradation of proteins and offer new strategies for the development and treatment of diseases caused by misfolded proteins. However, there are still some shortcomings associated with this method: (i) designing these peptides requires a domain with a high affinity for the target protein, and (ii) the delivery and stability of these peptides as therapeutic agents remain problems to be resolved. Fortunately, researchers are constantly trying to improve this method and hope to treat various human diseases by this method in the future.</div></div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i9">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_43714" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_43714" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Currently, PROTACs that induce targeted protein degradation represent a brand-new technique that has attracted much attention. Compared with traditional drug inhibitors, PROTACs have potential advantages in terms of drug resistance, side effects, dosage, and the regulation of “undruggable” targets. Lysosome-based degradation technique has more extensive substrates than PROTACs (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>) because it can degrade not only cytoplasmic proteins but also extracellular proteins, protein aggregates, organelles, and non-protein molecules, which are important functions not exhibited by proteasomes. Several technical platforms have been developed to degrade extracellular proteins, large proteins, and damaged organelles (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). In addition, the role of a PROTAC depends on the specific E3 ligase subunit. Due to challenges with their expression, the application of PROTACs in specific cell types is limited. However, lysosomes exist in almost all animal cells (except mature red blood cells of mammals), and there are similar organelles in plant cells; therefore, in theory, lysosome-based degradation can be achieved in nearly all cell types.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Comparison of PROTACs and Lysosome-Based Degradation Technique</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">degrader technique</th><th class="colsep0 rowsep0" align="center">degradation application</th><th class="colsep0 rowsep0" align="center">ref</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" rowspan="3" align="left">PROTAC</td><td class="colsep0 rowsep0" align="left">Intracellular proteins</td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref116'); return false;" href="javascript:void(0);" class="ref ref116">(116)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Pathogenic bacteria</td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref117'); return false;" href="javascript:void(0);" class="ref ref117">(117)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Nuclear protein</td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref118'); return false;" href="javascript:void(0);" class="ref ref118">(118)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" rowspan="5" align="left">Lysosome-based degradation technique</td><td class="colsep0 rowsep0" align="left">Intracellular proteins</td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Extracellular substances</td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(58)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Transmembrane proteins</td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref63'); return false;" href="javascript:void(0);" class="ref ref63">(63)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Organelles</td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Protein aggregate</td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref62'); return false;" href="javascript:void(0);" class="ref ref62">(62)</a></td></tr></tbody></table></div></div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Summary of the Advantages and Drawbacks of Various Degradation Techniques</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">technology</th><th class="colsep0 rowsep0" align="center">degradation pathway</th><th class="colsep0 rowsep0" align="center">advantages</th><th class="colsep0 rowsep0" align="center">drawbacks</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PROTACs</td><td class="colsep0 rowsep0" align="left">Proteasome pathway</td><td class="colsep0 rowsep0" align="left">Catalytic; clear degradation mechanism; potent activity; spatiotemporal control.</td><td class="colsep0 rowsep0" align="left">Depends on E3 ligase; unable to degrade extracellular proteins or organelles.</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">LYTACs</td><td class="colsep0 rowsep0" align="left">Endocytosis/lysosome pathway</td><td class="colsep0 rowsep0" align="left">Degradation of extracellular and transmembrane proteins; tissue specificity.</td><td class="colsep0 rowsep0" align="left">High molecular weight; slow degradation rate.</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">AUTACs</td><td class="colsep0 rowsep0" align="left">Macroautophagy pathway</td><td class="colsep0 rowsep0" align="left">Degrades organelles and intracellular proteins.</td><td class="colsep0 rowsep0" align="left">Unclear mechanisms of action.</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ATTECs</td><td class="colsep0 rowsep0" align="left">Macroautophagy pathway</td><td class="colsep0 rowsep0" align="left">Easily crosses the blood–brain barrier.</td><td class="colsep0 rowsep0" align="left">Lack of detailed interface between LC3 and the compound.</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Targeted CMA degraders</td><td class="colsep0 rowsep0" align="left">CMA pathway</td><td class="colsep0 rowsep0" align="left">Degradation of transmembrane proteins and protein aggregates.</td><td class="colsep0 rowsep0" align="left">Limited stability within the cell; poor cell permeability; weak activity.</td></tr></tbody></table></div></div><div class="NLM_p">Lysosome-based degradation technique is still in an early stage, but some breakthroughs have been made. ATTECs, which are small-molecule binding compounds, can bind LC3 and pathogenic proteins (or other pathogenic substances) together and then encapsulate pathogenic proteins in autophagosomes for degradation by lysosomes. The small-molecule compound that can specifically reduce the pathogenic protein associated with Huntingtin’s disease reaches the nmol level in HD cells. In addition, in HD-knock-in mice, ATTECs at 0.5 mg kg<sup>–1</sup> reduced mHTT and reached the brain through the blood–brain barrier. These compounds have no effect on wild-type HTT. In addition, Lu et al. assessed protein levels in tissue samples. They found that only a small percentage of the protein levels changed in mice treated with the compound compared with the control animals. Currently, it is not clear whether the level changes of some proteins affect the function of normal proteins. The LYTAC technique platform has been used to successfully degrade disease-related proteins (including apolipoprotein E4, EGFR, CD71, and PD-L1) <i>in vitro</i>, which fully proves that the LYTAC technique platform can directly target extracellular and membrane-associated proteins for lysosomal degradation. The proteome-wide characterization of EGFR degradation specificity was assessed by quantitative mass spectrometry analysis of cells treated with LYTACs. Decreased levels of certain proteins may be associated with molecules regulated by EGFR. Targeted peptides were designed using specific recognition of molecular chaperones and autophagy substrate proteins to achieve selective degradation of specific proteins. These targeted peptide-directed lysosomes can rapidly and reversibly knock out endogenous proteins <i>in situ</i>. DAPK1-targeting peptide at 10 mg kg<sup>–1</sup> successfully reduced DAPK1 levels <i>in vivo</i>. AUTAC technique utilizing macroautophagy not only successfully degrades cytoplasmic proteins but also degrades damaged mitochondria selectively in cells, which provides new inspiration for the development of a system for degrading organelles. It is believed that targeting protein and organelle degradation through the lysosomal pathway is a promising therapeutic strategy.</div><div class="NLM_p">Obtaining the appropriate physicochemical properties and the absorption, distribution, metabolism, and excretion properties that are sufficient for oral administration and for the drugs to reach the central nervous system is extremely important. Currently, there are many methods to optimize PK, such as optimizing permeability, which can greatly improve bioavailability. Enhancing the solubility of the compound facilitates drug exposure <i>in vivo</i> and <i>in vitro</i>. However, bioavailability optimization of macromolecules such as AUTACs is still in its infancy. ATTECs are small molecules; therefore, PK can be optimized according to structure–activity relationship studies. In addition, ATTECs and AUTACs may have catalytic activity and can be reused after the target protein is degraded by lysosomes, but this supposition requires further experimental validation. LYTACs and targeting peptides may be noncatalytic because their biomolecules may be degraded by lysosomes. In theory, the size of the AUTAC and targeting peptide affects the ability of the molecule to pass through the cell membrane. However, for LYTACs, molecular size is not a problem because they can cross the cell membrane by endocytosis.</div><div class="NLM_p">For the degradation of extracellular substances and plasma membrane proteins, lysosome-based degradation molecules can be designed by targeting LTRs that undergo endocytosis. For intracellular proteins and organelles, lysosome-based degradation molecules can be designed by targeting LTRs that undergo autophagy. Peptide molecules targeting CMA, AUTACs and ATTECs targeting autophagy, and LYTACs targeting LTRs have been developed. Lysosome-based degradation technique and PROTACs can enable the binding of two proteins to form a trimeric complex for which administration at high concentrations does not yield better results. The phenomenon associated with this characteristic is called the HOOK effect. This effect features PK/PD relationships and drug delivery patterns. The formation of all three components in a stable and long-lived trimer can be optimized to accelerate the degradation of the target and improve the efficacy of the degraders. Lysosome-based degradation molecules have a greater molecular weight than traditional small-molecule drugs, except for ATTECs, and may not effectively penetrate the biomembrane. Therefore, there is an urgent need to develop degraders targeting lysosomes with lower molecular weight to take effect after passing through the biomembrane.</div><div class="NLM_p">Compared with PROTAC, the development of targeted lysosomal degradation technique is relatively lagging behind, but this degradation technique has been validated preliminarily at the cellular and animal levels, which greatly expands the range of targets for protein degradation agent technique. The critical challenge of the targeted lysosome-based degrader is how to rationally design and effectively promote the lysosome-based degrader into the clinic. First, the discovery of more ligands targeting lysosomes will promote development in the field of protein degradation. So far, only CI-M6PR and ASGPR have been developed for target protein degradation in LYTAC technique. The design of different small molecule ligands targeting lysosomal receptors can promote the diversification of LYTAC, and targeting the distribution of lysosomal receptors in different tissues can achieve the purpose of treating tissue-specific diseases. Targeting lysosomes through the autophagy pathway is also a strategy for developing targeted lysosome therapy. Autophagy can also transport intracellular waste products into lysosomes for degradation. Therefore, exploring new recruiting autophagy ligands will greatly expand the application of targeted lysosomal degradation. Whether the existing autophagy inhibitors or agonists can be used to design and recruit autophagy ligands needs further investigation. It is also possible to optimize the recruitment of autophagy ligands of AUTAC and ATEEC to make them universal targeted lysosomal ligands. For example, the specific mechanism of the interaction between ATTEC molecule and LC3 has not been clarified, therefore, the characterization of complex structure of ATTEC molecule and LC3 or the systematic structure–activity relationship study will help in-depth understanding of the mode of action and clarify the mechanism and provide a basis for the rational design of high-affinity ligands targeting LC3. The design of peptides targeting lysosomes is relatively simple, so peptides coupled with small molecules can be considered to degrade target proteins. Structure-based drug design is helpful for the discovery and optimization of lead compounds for targeted lysosome-based degrader. High-throughput screening, especially DNA coding libraries (DELS), can rapidly screen a large number of molecules and is expected to obtain highly efficient ligands targeting lysosomes. These studies will facilitate the discovery of ligands with high affinity for lysosome and then greatly improve the degradation efficiency of lysosome degradation.</div><div class="NLM_p">Second, although some degradation agents have been studied <i>in vivo</i> in animals, the PK and PD of the drug need to be further studied. The ADME of degraders needs to be thoroughly investigated. Lysosome-based degraders are mainly large molecules, which may cause problems such as low permeability, poor tissue access, and reduced bioavailability; thus structural optimization is required and the PK and PD of the degradation agents need to be further studied. So far, LYTAC has no <i>in vivo</i> data disclosure. The PK properties of AUTAC molecules can be improved by chemical modification, such as ring formation, pharmacophore bioisostere replacement, and salt formation. ATTEC has relatively good ADME properties due to its smaller molecular weight; however, molecular glue currently lacks systematic design and discovery. In the future, we believe that scientists will develop more small molecular glue compounds based on experience and computer-aided drug design to improve their PK properties. On the other hand, there have been relatively few studies on peptide-targeted chimeras in recent years, which is mainly due to the rapid clearance of peptides from the kidneys and limited oral bioavailability. These inherent flaws can be overcome by chemical modification, physical technology, and nanomaterials, which have been proven to improve the ADME properties of polypeptides by improving cell permeability, enhancing proteolytic stability, and reducing renal clearance, thereby prolonging the half-life. This provides a potential for the druggability and clinical application of peptide-targeted chimeras.</div><div class="NLM_p">Lastly, drug-like factors of lysosome-based degraders need to be systematically investigated, including applicability, selectivity, and potential for undesired effects induced by off-target effects <i>in vivo</i>. Lysosome-based degraders should be tested in multiple cell types to determine their application and suitability. Additionally, it is necessary to study their biodistribution and biostability <i>in vivo</i> to avoid accumulation in healthy organs or tissues. The technique of degrading proteins of interest and organelles through the lysosomal pathway may also cause off-target effects and result in damage to normal proteins and organelles. Therefore, the safety and toxicity of degrading agents need to be fully explored including the dosage and the frequency to determine the potential for undesired effects and toxicity of the drug.</div><div class="NLM_p last">Conclusively, the lysosome-based degradation technique has greatly expanded the “undruggable” targets, especially in the treatment of neurodegenerative, metabolic, and aging diseases. It is believed that the development of a lysosome-based degradation techniques will lead to a wide range of therapeutic applications, providing new drug therapies for previously unconquerable diseases.</div></div><div class="NLM_back"><div class="article_supporting-info" id=""></div><div class="testing" data-doi="10.1021/acs.jmedchem.0c01689" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_64050" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_64050" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Qiu Sun</span> - <span class="hlFld-Affiliation affiliation">State Key Laboratory of
Biotherapy and Cancer Center, National Clinical Research Center for
Geriatrics, Innovation Center of Nursing Research, Nursing Key Laboratory
of Sichuan Province, West China Hospital, and Collaborative Innovation
Center of Biotherapy, Sichuan University, Chengdu 610041, China</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#4d3e38233c24380d3e2e3863282938632e23"><span class="__cf_email__" data-cfemail="a4d7d1cad5cdd1e4d7c7d18ac1c0d18ac7ca">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Liang Ouyang</span> - <span class="hlFld-Affiliation affiliation">State Key Laboratory of
Biotherapy and Cancer Center, National Clinical Research Center for
Geriatrics, Innovation Center of Nursing Research, Nursing Key Laboratory
of Sichuan Province, West China Hospital, and Collaborative Innovation
Center of Biotherapy, Sichuan University, Chengdu 610041, China</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-5537-8834" title="Orcid link">http://orcid.org/0000-0001-5537-8834</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#036c767a626d646f6a626d64437060762d6667762d606d"><span class="__cf_email__" data-cfemail="127d676b737c757e7b737c75526171673c7776673c717c">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Junping Pei</span> - <span class="hlFld-Affiliation affiliation">State Key Laboratory of
Biotherapy and Cancer Center, National Clinical Research Center for
Geriatrics, Innovation Center of Nursing Research, Nursing Key Laboratory
of Sichuan Province, West China Hospital, and Collaborative Innovation
Center of Biotherapy, Sichuan University, Chengdu 610041, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Guan Wang</span> - <span class="hlFld-Affiliation affiliation">State Key Laboratory of
Biotherapy and Cancer Center, National Clinical Research Center for
Geriatrics, Innovation Center of Nursing Research, Nursing Key Laboratory
of Sichuan Province, West China Hospital, and Collaborative Innovation
Center of Biotherapy, Sichuan University, Chengdu 610041, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lu Feng</span> - <span class="hlFld-Affiliation affiliation">State Key Laboratory of
Biotherapy and Cancer Center, National Clinical Research Center for
Geriatrics, Innovation Center of Nursing Research, Nursing Key Laboratory
of Sichuan Province, West China Hospital, and Collaborative Innovation
Center of Biotherapy, Sichuan University, Chengdu 610041, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jifa Zhang</span> - <span class="hlFld-Affiliation affiliation">State Key Laboratory of
Biotherapy and Cancer Center, National Clinical Research Center for
Geriatrics, Innovation Center of Nursing Research, Nursing Key Laboratory
of Sichuan Province, West China Hospital, and Collaborative Innovation
Center of Biotherapy, Sichuan University, Chengdu 610041, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Tingting Jiang</span> - <span class="hlFld-Affiliation affiliation">State Key Laboratory of
Biotherapy and Cancer Center, National Clinical Research Center for
Geriatrics, Innovation Center of Nursing Research, Nursing Key Laboratory
of Sichuan Province, West China Hospital, and Collaborative Innovation
Center of Biotherapy, Sichuan University, Chengdu 610041, China</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>J.P. and G.W. contributed equally to this work.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="bio" rid="ath1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="biographies">Biographies</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_86225" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_86225" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><figure id="ath1" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Junping Pei</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01689&amp;id=BIO-d7e1175-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Junping Pei</b> received his Master’s degree in Medicinal Chemistry from Ocean University of China in 2019 and is currently a Ph.D. candidate at the State Key Laboratory of Biotherapy and Cancer Center, National Clinical Research Center for Geriatrics, Innovation Center of Nursing Research, Nursing Key Laboratory of Sichuan Province, West China Hospital, and Collaborative Innovation Center of Biotherapy, Sichuan University. His research interests are the design, synthesis, and biological evaluation of protein degraders and small-molecule inhibitors with potential anticancer activity.</p></figure></div><div class="bio" rid="ath2"><figure id="ath2" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Guan Wang</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01689&amp;id=BIO-d7e1180-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Guan Wang</b> received his Ph.D. from School of Traditional Chinese Materia Medica, Wuya College of Innovation, Key Laboratory of Structure-Based Drug Design & Discovery, Ministry of Education, Shenyang Pharmaceutical University in 2017. Currently, he is an Assistant Professor at the State Key Laboratory of Biotherapy and Cancer Center, National Clinical Research Center for Geriatrics, Innovation Center of Nursing Research, Nursing Key Laboratory of Sichuan Province, West China Hospital, and Collaborative Innovation Center of Biotherapy, Sichuan University. His research interests are bioinformatics, pharmaceutical chemistry, structure-based design of drugs and bioactive compounds.</p></figure></div><div class="bio" rid="ath3"><figure id="ath3" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Lu Feng</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01689&amp;id=BIO-d7e1185-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Lu Feng</b> received her Bachelor’s degree in pharmaceutical engineering from Southwest Jiaotong University in 2019 and is a Master’s degree candidate at the State Key Laboratory of Biotherapy and Cancer Center, West China Hospital, and Collaborative Innovation Center of Biotherapy, Sichuan University. Her research interest focuses on pharmaceutical chemistry.</p></figure></div><div class="bio" rid="ath4"><figure id="ath4" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Jifa Zhang</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01689&amp;id=BIO-d7e1190-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Jifa Zhang</b> received his Ph.D. degree from Southwest Jiaotong University in 2018 and is currently a postdoctoral student at the State Key Laboratory of Biotherapy and Cancer Center, West China Hospital, and Collaborative Innovation Center of Biotherapy, Sichuan University. His research interests focus on the discovery, optimization, and molecular mechanisms of candidate small-molecule compounds with antitumor activity.</p></figure></div><div class="bio" rid="ath5"><figure id="ath5" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Tingting Jiang</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01689&amp;id=BIO-d7e1195-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Tingting Jiang</b> received her Bachelor’s degree in Pharmacy from Sichuan Agricultural University in 2018 and is currently pursuing a Master’s degree at the State Key Laboratory of Biotherapy and Cancer Center, West China Hospital, and Collaborative Innovation Center of Biotherapy, Sichuan University. Her research interests are the design, synthesis, and biological evaluation of small-molecule inhibitors with potential anticancer activity.</p></figure></div><div class="bio" rid="ath6"><figure id="ath6" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Qiu Sun</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01689&amp;id=BIO-d7e1200-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Qiu Sun</b> received her Ph.D. degree in 2015 from Universität Regensburg, Germany, under the direction of Prof. Burkhard Koenig. She continued her scientific education as a postdoctoral fellow with Prof. Yuquan Wei, Sichuan University. In 2020, she began her academic career at the State Key Laboratory of Biotherapy and Cancer Center, National Clinical Research Center for Geriatrics, Innovation Center of Nursing Research, Nursing Key Laboratory of Sichuan Province, West China Hospital, and Collaborative Innovation Center of Biotherapy, Sichuan University as an Associate Professor. Her current research interests focus on the characterization of new biological target structures and the structure-based discovery of small molecules.</p></figure></div><div class="bio" rid="ath7"><figure id="ath7" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Liang Ouyang</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01689&amp;id=BIO-d7e1207-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Liang Ouyang</b> received his Ph.D. degree in Medicinal Chemistry from West China School of Pharmacy, Sichuan University, in 2010. He began his academic career in 2012 at the State Key Laboratory of Biotherapy and Cancer Center, National Clinical Research Center for Geriatrics, Innovation Center of Nursing Research, Nursing Key Laboratory of Sichuan Province, West China Hospital, and Collaborative Innovation Center of Biotherapy, Sichuan University as a Professor. His research interests include the identification of novel drug targets in cell death and the structure-based discovery of small-molecule and peptide drugs.</p></figure></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i20">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_48137" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_48137" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This work was supported by grants from the National Natural Science Foundation of China (Grants 81922064, 81874290, 81803755, and 81903502), Project of Science and Technology Department of Sichuan Province (Grant 2019JDRC0091), and National Clinical Research Center for Geriatrics, West China Hospital, Sichuan University (Grant Z20201004). Figures were created with <a href="http://BioRender.com" class="extLink">BioRender.com</a>.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">ADME</td><td class="NLM_def"><p class="first last">absorption, distribution, metabolism, excretion</p></td></tr><tr><td class="NLM_term">ASGPR</td><td class="NLM_def"><p class="first last">asialoglycoprotein receptor</p></td></tr><tr><td class="NLM_term">ATTEC</td><td class="NLM_def"><p class="first last">autophagosome-tethering compound</p></td></tr><tr><td class="NLM_term">AUTAC</td><td class="NLM_def"><p class="first last">autophagy-targeting chimera</p></td></tr><tr><td class="NLM_term">BRD4</td><td class="NLM_def"><p class="first last">bromodomain-containing protein 4</p></td></tr><tr><td class="NLM_term">CDK5</td><td class="NLM_def"><p class="first last">cyclin-dependent kinase 5</p></td></tr><tr><td class="NLM_term">CI-M6PR</td><td class="NLM_def"><p class="first last">cation-independent mannose 6-phosphate receptor</p></td></tr><tr><td class="NLM_term">CMA</td><td class="NLM_def"><p class="first last">chaperone-mediated autophagy</p></td></tr><tr><td class="NLM_term">CMPD</td><td class="NLM_def"><p class="first last">cell membrane penetrating domain</p></td></tr><tr><td class="NLM_term">CTM</td><td class="NLM_def"><p class="first last">CMA-targeting motif</p></td></tr><tr><td class="NLM_term">DAPK1</td><td class="NLM_def"><p class="first last">death-associated protein kinase 1</p></td></tr><tr><td class="NLM_term">DELS</td><td class="NLM_def"><p class="first last">DNA coding libraries</p></td></tr><tr><td class="NLM_term">FBnG</td><td class="NLM_def"><p class="first last"><i>p</i>-fluorobenzylguanine</p></td></tr><tr><td class="NLM_term">GalNAc</td><td class="NLM_def"><p class="first last"><i>N</i>-acetylgalactosamine</p></td></tr><tr><td class="NLM_term">HTRF</td><td class="NLM_def"><p class="first last">homogeneous time-resolved fluorescence</p></td></tr><tr><td class="NLM_term">LIMP-2</td><td class="NLM_def"><p class="first last">lysosomal integral membrane protein 2</p></td></tr><tr><td class="NLM_term">LTR</td><td class="NLM_def"><p class="first last">lysosome-targeting receptor</p></td></tr><tr><td class="NLM_term">LYTAC</td><td class="NLM_def"><p class="first last">lysosome-targeting chimera</p></td></tr><tr><td class="NLM_term">M6Pn</td><td class="NLM_def"><p class="first last">serine-<i>O</i>-mannose 6-phosphonate</p></td></tr><tr><td class="NLM_term">M6PR</td><td class="NLM_def"><p class="first last">mannose 6-phosphate receptor</p></td></tr><tr><td class="NLM_term">MetAP-2</td><td class="NLM_def"><p class="first last">methionine aminopeptidase 2</p></td></tr><tr><td class="NLM_term">mHTT</td><td class="NLM_def"><p class="first last">mutant huntingtin protein</p></td></tr><tr><td class="NLM_term">NA-647</td><td class="NLM_def"><p class="first last">NeutrAvidin-647</p></td></tr><tr><td class="NLM_term">NCA</td><td class="NLM_def"><p class="first last"><i>N</i>-carboxyanhydride</p></td></tr><tr><td class="NLM_term">OI-RD</td><td class="NLM_def"><p class="first last">oblique-incidence reflectivity difference</p></td></tr><tr><td class="NLM_term">PBD</td><td class="NLM_def"><p class="first last">protein-binding domain</p></td></tr><tr><td class="NLM_term">POI</td><td class="NLM_def"><p class="first last">protein of interest</p></td></tr><tr><td class="NLM_term">polyQ</td><td class="NLM_def"><p class="first last">expanded polyglutamine</p></td></tr><tr><td class="NLM_term">PROTAC</td><td class="NLM_def"><p class="first last">proteolysis-targeting chimera</p></td></tr><tr><td class="NLM_term">QBP1</td><td class="NLM_def"><p class="first last">polyglutamine-binding peptide 1 sequence</p></td></tr><tr><td class="NLM_term">SMM</td><td class="NLM_def"><p class="first last">small-molecule microarray</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i22">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_88514" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_88514" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 118 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gómez-Sintes, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boya, P.</span></span> <span> </span><span class="NLM_article-title">Lysosomal membrane permeabilization and cell death</span>. <i>Traffic</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">918</span>– <span class="NLM_lpage">931</span>, <span class="refDoi"> DOI: 10.1111/tra.12613</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1111%2Ftra.12613" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=30125440" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslSrsL3M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2018&pages=918-931&author=F.+Wangauthor=R.+G%C3%B3mez-Sintesauthor=P.+Boya&title=Lysosomal+membrane+permeabilization+and+cell+death&doi=10.1111%2Ftra.12613"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Lysosomal membrane permeabilization and cell death</span></div><div class="casAuthors">Wang, Fengjuan; Gomez-Sintes, Raquel; Boya, Patricia</div><div class="citationInfo"><span class="NLM_cas:title">Traffic (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">918-931</span>CODEN:
                <span class="NLM_cas:coden">TRAFFA</span>;
        ISSN:<span class="NLM_cas:issn">1398-9219</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  Lysosomes are membrane-enclosed organelles that mediate the intracellular degrdn. of macromols.  They play an essential role in calcium regulation and have emerged as key signaling hubs in controlling the nutrient response.  Maintaining lysosomal integrity and function is therefore crucial for cellular homeostasis.  Different forms of stress can induce lysosomal membrane permeabilization (LMP), resulting in the translocation to the cytoplasm of intralysosomal components, such as cathepsins, inducing lysosomal-dependent cell death (LDCD).  Here, we review recent advances that have furthered our understanding of the mol. mechanisms of LMP and the methods used to detect it.  We discuss several endolysosomal damage-response mechanisms that mediate the repair or elimination of compromised lysosomes and summarize the role of LMP and cathepsins in LDCD and other cell death pathways.  Finally, with the emergence of lysosomes as promising therapeutic targets for several human diseases, we review a variety of therapeutic strategies that seek to either destabilize lysosomes or to maintain, enhance or restore lysosomal function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7BHvqj7V7ULVg90H21EOLACvtfcHk0ljY3Clod-0enQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslSrsL3M&md5=f20ef531226f238f550ddf5b3cf91e8a</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1111%2Ftra.12613&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Ftra.12613%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DF.%26aulast%3DG%25C3%25B3mez-Sintes%26aufirst%3DR.%26aulast%3DBoya%26aufirst%3DP.%26atitle%3DLysosomal%2520membrane%2520permeabilization%2520and%2520cell%2520death%26jtitle%3DTraffic%26date%3D2018%26volume%3D19%26spage%3D918%26epage%3D931%26doi%3D10.1111%2Ftra.12613" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yim, W. W.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mizushima, N.</span></span> <span> </span><span class="NLM_article-title">Lysosome biology in autophagy</span>. <i>Cell Discovery</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">6</span>, <span class="refDoi"> DOI: 10.1038/s41421-020-0141-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1038%2Fs41421-020-0141-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=32047650" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BB3cXktlegtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2020&pages=6&author=W.+W.-Y.+Yimauthor=N.+Mizushima&title=Lysosome+biology+in+autophagy&doi=10.1038%2Fs41421-020-0141-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Lysosome biology in autophagy</span></div><div class="casAuthors">Yim, Willa Wen-You; Mizushima, Noboru</div><div class="citationInfo"><span class="NLM_cas:title">Cell Discovery</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">6</span>CODEN:
                <span class="NLM_cas:coden">CDEIBM</span>;
        ISSN:<span class="NLM_cas:issn">2056-5968</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Abstr.: Autophagy is a major intracellular degrdn. system that derives its degradative abilities from the lysosome.  The most well-studied form of autophagy is macroautophagy, which delivers cytoplasmic material to lysosomes via the double-membraned autophagosome.  Other forms of autophagy, namely chaperone-mediated autophagy and microautophagy, occur directly on the lysosome.  Besides providing the means for degrdn., lysosomes are also involved in autophagy regulation and can become substrates of autophagy when damaged.  During autophagy, they exhibit notable changes, including increased acidification, enhanced enzymic activity, and perinuclear localization.  Despite their importance to autophagy, details on autophagy-specific regulation of lysosomes remain relatively scarce.  This review aims to provide a summary of current understanding on the behavior of lysosomes during autophagy and outline unexplored areas of autophagy-specific lysosome research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqpU2zO_7Pq8bVg90H21EOLACvtfcHk0ljY3Clod-0enQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXktlegtLs%253D&md5=bd5096a008d31a55449e6980b2bd9471</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fs41421-020-0141-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41421-020-0141-7%26sid%3Dliteratum%253Aachs%26aulast%3DYim%26aufirst%3DW.%2BW.-Y.%26aulast%3DMizushima%26aufirst%3DN.%26atitle%3DLysosome%2520biology%2520in%2520autophagy%26jtitle%3DCell%2520Discovery%26date%3D2020%26volume%3D6%26spage%3D6%26doi%3D10.1038%2Fs41421-020-0141-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Deus, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yambire, K. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliveira, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raimundo, N.</span></span> <span> </span><span class="NLM_article-title">Mitochondria-lysosome crosstalk: from physiology to neurodegeneration</span>. <i>Trends Mol. Med.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">71</span>– <span class="NLM_lpage">88</span>, <span class="refDoi"> DOI: 10.1016/j.molmed.2019.10.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1016%2Fj.molmed.2019.10.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=31791731" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitF2murvJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2020&pages=71-88&author=C.+M.+Deusauthor=K.+F.+Yambireauthor=P.+J.+Oliveiraauthor=N.+Raimundo&title=Mitochondria-lysosome+crosstalk%3A+from+physiology+to+neurodegeneration&doi=10.1016%2Fj.molmed.2019.10.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Mitochondria-Lysosome Crosstalk: From Physiology to Neurodegeneration</span></div><div class="casAuthors">Deus, Claudia M.; Yambire, King Faisal; Oliveira, Paulo J.; Raimundo, Nuno</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Molecular Medicine</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">71-88</span>CODEN:
                <span class="NLM_cas:coden">TMMRCY</span>;
        ISSN:<span class="NLM_cas:issn">1471-4914</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Cellular function requires coordination between different organelles and metabolic cues.  Mitochondria and lysosomes are essential for cellular metab. as major contributors of chem. energy and building blocks.  It is therefore pivotal for cellular function to coordinate the metabolic roles of mitochondria and lysosomes.  However, these organelles do more than metab., given their function as fundamental signaling platforms in the cell that regulate many key processes such as autophagy, proliferation, and cell death.  Mechanisms of crosstalk between mitochondria and lysosomes are discussed, both under physiol. conditions and in diseases that affect these organelles.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpiRMg_NRB2FrVg90H21EOLACvtfcHk0lig3EZ-DodKFw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitF2murvJ&md5=3fd733dfde23717db9bc09bb1ccff10c</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.molmed.2019.10.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molmed.2019.10.009%26sid%3Dliteratum%253Aachs%26aulast%3DDeus%26aufirst%3DC.%2BM.%26aulast%3DYambire%26aufirst%3DK.%2BF.%26aulast%3DOliveira%26aufirst%3DP.%2BJ.%26aulast%3DRaimundo%26aufirst%3DN.%26atitle%3DMitochondria-lysosome%2520crosstalk%253A%2520from%2520physiology%2520to%2520neurodegeneration%26jtitle%3DTrends%2520Mol.%2520Med.%26date%3D2020%26volume%3D26%26spage%3D71%26epage%3D88%26doi%3D10.1016%2Fj.molmed.2019.10.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tooze, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abada, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elazar, Z.</span></span> <span> </span><span class="NLM_article-title">Endocytosis and autophagy: exploitation or cooperation?</span>. <i>Cold Spring Harbor Perspect. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>6</i></span>, <span class="NLM_elocation-id">a018358</span> <span class="refDoi"> DOI: 10.1101/cshperspect.a018358</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1101%2Fcshperspect.a018358" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=24789822" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2014&author=S.+A.+Toozeauthor=A.+Abadaauthor=Z.+Elazar&title=Endocytosis+and+autophagy%3A+exploitation+or+cooperation%3F&doi=10.1101%2Fcshperspect.a018358"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1101%2Fcshperspect.a018358&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fcshperspect.a018358%26sid%3Dliteratum%253Aachs%26aulast%3DTooze%26aufirst%3DS.%2BA.%26aulast%3DAbada%26aufirst%3DA.%26aulast%3DElazar%26aufirst%3DZ.%26atitle%3DEndocytosis%2520and%2520autophagy%253A%2520exploitation%2520or%2520cooperation%253F%26jtitle%3DCold%2520Spring%2520Harbor%2520Perspect.%2520Biol.%26date%3D2014%26volume%3D6%26doi%3D10.1101%2Fcshperspect.a018358" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Saftig, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klumperman, J.</span></span> <span> </span><span class="NLM_article-title">Lysosome biogenesis and lysosomal membrane proteins: trafficking meets function</span>. <i>Nat. Rev. Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">623</span>– <span class="NLM_lpage">635</span>, <span class="refDoi"> DOI: 10.1038/nrm2745</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1038%2Fnrm2745" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=19672277" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BD1MXpsleqsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2009&pages=623-635&author=P.+Saftigauthor=J.+Klumperman&title=Lysosome+biogenesis+and+lysosomal+membrane+proteins%3A+trafficking+meets+function&doi=10.1038%2Fnrm2745"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Lysosome biogenesis and lysosomal membrane proteins: trafficking meets function</span></div><div class="casAuthors">Saftig, Paul; Klumperman, Judith</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">623-635</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Lysosomes are the primary catabolic compartments of eukaryotic cells.  They degrade extracellular material that has been internalized by endocytosis and intracellular components that have been sequestered by autophagy.  In addn., specialized cells contain lysosome-related organelles that store and secrete proteins for cell-type-specific functions.  The functioning of a healthy cell is dependent on the proper targeting of newly synthesized lysosomal proteins.  Accumulating evidence suggests that there are multiple lysosomal delivery pathways that together allow the regulated and sequential deposition of lysosomal components.  The importance of lysosomal trafficking pathways is emphasized by recent findings that reveal new roles for lysosomal membrane proteins in cellular physiol. and in an increasing no. of diseases that are characterized by defects in lysosome biogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpby5TZ_C2_I7Vg90H21EOLACvtfcHk0lig3EZ-DodKFw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXpsleqsrc%253D&md5=e45f191f35eba89f8936a7a9a9f76cdf</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnrm2745&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm2745%26sid%3Dliteratum%253Aachs%26aulast%3DSaftig%26aufirst%3DP.%26aulast%3DKlumperman%26aufirst%3DJ.%26atitle%3DLysosome%2520biogenesis%2520and%2520lysosomal%2520membrane%2520proteins%253A%2520trafficking%2520meets%2520function%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2009%26volume%3D10%26spage%3D623%26epage%3D635%26doi%3D10.1038%2Fnrm2745" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chieregatti, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meldolesi, J.</span></span> <span> </span><span class="NLM_article-title">Regulated exocytosis: new organelles for non-secretory purposes</span>. <i>Nat. Rev. Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">181</span>– <span class="NLM_lpage">187</span>, <span class="refDoi"> DOI: 10.1038/nrm1572</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1038%2Fnrm1572" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=15688003" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BD2MXoslOnuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2005&pages=181-187&author=E.+Chieregattiauthor=J.+Meldolesi&title=Regulated+exocytosis%3A+new+organelles+for+non-secretory+purposes&doi=10.1038%2Fnrm1572"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Opinion: Regulated exocytosis: new organelles for non-secretory purposes</span></div><div class="casAuthors">Chieregatti, Evelina; Meldolesi, Jacopo</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">181-187</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A discussion and review.  Regulated exocytosis is a process in which the membranes of cytoplasmic organelles fuse with the plasma membrane in response to stimulation.  In many cases (secretory exocytoses), the process functions to secrete specific products that are segregated in the organelle lumen (for example, neurotransmitters, hormones and enzymes) to the extracellular space.  In other cases ('non-secretory exocytoses'), it functions to transfer the organelle membrane and its components to the cell surface.  Here, the general properties of non-secretory exocytoses are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqMb4F1DBfmsrVg90H21EOLACvtfcHk0lig3EZ-DodKFw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXoslOnuw%253D%253D&md5=8d0d6f4b2f5d713ae30691a64de91794</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnrm1572&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm1572%26sid%3Dliteratum%253Aachs%26aulast%3DChieregatti%26aufirst%3DE.%26aulast%3DMeldolesi%26aufirst%3DJ.%26atitle%3DRegulated%2520exocytosis%253A%2520new%2520organelles%2520for%2520non-secretory%2520purposes%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2005%26volume%3D6%26spage%3D181%26epage%3D187%26doi%3D10.1038%2Fnrm1572" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M. C.</span></span> <span> </span><span class="NLM_article-title">Molecular mechanisms of lysosome and nucleus communication</span>. <i>Trends Biochem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">978</span>– <span class="NLM_lpage">991</span>, <span class="refDoi"> DOI: 10.1016/j.tibs.2020.06.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1016%2Fj.tibs.2020.06.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=32624271" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BB3cXht1WlsLjJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2020&pages=978-991&author=Q.+Zhaoauthor=S.+M.+Gaoauthor=M.+C.+Wang&title=Molecular+mechanisms+of+lysosome+and+nucleus+communication&doi=10.1016%2Fj.tibs.2020.06.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Mechanisms of Lysosome and Nucleus Communication</span></div><div class="casAuthors">Zhao, Qian; Gao, Shihong Max; Wang, Meng C.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Biochemical Sciences</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">978-991</span>CODEN:
                <span class="NLM_cas:coden">TBSCDB</span>;
        ISSN:<span class="NLM_cas:issn">0968-0004</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Lysosomes transcend the role of degrdn. stations, acting as key nodes for interorganelle crosstalk and signal transduction.  Lysosomes communicate with the nucleus through phys. proximity and functional interaction.  In response to external and internal stimuli, lysosomes actively adjust their distribution between peripheral and perinuclear regions and modulate lysosome-nucleus signaling pathways; in turn, the nucleus fine-tunes lysosomal biogenesis and functions through transcriptional controls.  Changes in coordination between these two essential organelles are assocd. with metabolic disorders, neurodegenerative diseases, and aging.  In this review, we address recent advances in lysosome-nucleus communication by multi-tiered regulatory mechanisms and discuss how these regulations couple metabolic inputs with organellar motility, cellular signaling, and transcriptional network.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohlX7bTRTVW7Vg90H21EOLACvtfcHk0ljfPd0YClH1eQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXht1WlsLjJ&md5=efef671e93a93e45e44f00199ea06dcb</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.tibs.2020.06.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tibs.2020.06.004%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DQ.%26aulast%3DGao%26aufirst%3DS.%2BM.%26aulast%3DWang%26aufirst%3DM.%2BC.%26atitle%3DMolecular%2520mechanisms%2520of%2520lysosome%2520and%2520nucleus%2520communication%26jtitle%3DTrends%2520Biochem.%2520Sci.%26date%3D2020%26volume%3D45%26spage%3D978%26epage%3D991%26doi%3D10.1016%2Fj.tibs.2020.06.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bonam, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, S.</span></span> <span> </span><span class="NLM_article-title">Lysosomes as a therapeutic target</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">923</span>– <span class="NLM_lpage">948</span>, <span class="refDoi"> DOI: 10.1038/s41573-019-0036-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1038%2Fs41573-019-0036-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=31477883" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhs12gurrM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=923-948&author=S.+R.+Bonamauthor=F.+Wangauthor=S.+Muller&title=Lysosomes+as+a+therapeutic+target&doi=10.1038%2Fs41573-019-0036-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Lysosomes as a therapeutic target</span></div><div class="casAuthors">Bonam, Srinivasa Reddy; Wang, Fengjuan; Muller, Sylviane</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">923-948</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Lysosomes are membrane-bound organelles with roles in processes involved in degrading and recycling cellular waste, cellular signalling and energy metab.  Defects in genes encoding lysosomal proteins cause lysosomal storage disorders, in which enzyme replacement therapy has proved successful.  Growing evidence also implicates roles for lysosomal dysfunction in more common diseases including inflammatory and autoimmune disorders, neurodegenerative diseases, cancer and metabolic disorders.  With a focus on lysosomal dysfunction in autoimmune disorders and neurodegenerative diseases - including lupus, rheumatoid arthritis, multiple sclerosis, Alzheimer disease and Parkinson disease - this Review critically analyses progress and opportunities for therapeutically targeting lysosomal proteins and processes, particularly with small mols. and peptide drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-J0ZL8ZiXtrVg90H21EOLACvtfcHk0ljfPd0YClH1eQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhs12gurrM&md5=705fc139e1d00bf2e1fb4267585ed9c7</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fs41573-019-0036-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41573-019-0036-1%26sid%3Dliteratum%253Aachs%26aulast%3DBonam%26aufirst%3DS.%2BR.%26aulast%3DWang%26aufirst%3DF.%26aulast%3DMuller%26aufirst%3DS.%26atitle%3DLysosomes%2520as%2520a%2520therapeutic%2520target%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2019%26volume%3D18%26spage%3D923%26epage%3D948%26doi%3D10.1038%2Fs41573-019-0036-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horak, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Molkentin, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span> <span> </span><span class="NLM_article-title">Proteasome functional insufficiency activates the calcineurin-NFAT pathway in cardiomyocytes and promotes maladaptive remodelling of stressed mouse hearts</span>. <i>Cardiovasc. Res.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>88</i></span>,  <span class="NLM_fpage">424</span>– <span class="NLM_lpage">433</span>, <span class="refDoi"> DOI: 10.1093/cvr/cvq217</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1093%2Fcvr%2Fcvq217" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=20601385" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVSltLvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=88&publication_year=2010&pages=424-433&author=M.+Tangauthor=J.+Liauthor=W.+Huangauthor=H.+Suauthor=Q.+Liangauthor=Z.+Tianauthor=K.+M.+Horakauthor=J.+D.+Molkentinauthor=X.+Wang&title=Proteasome+functional+insufficiency+activates+the+calcineurin-NFAT+pathway+in+cardiomyocytes+and+promotes+maladaptive+remodelling+of+stressed+mouse+hearts&doi=10.1093%2Fcvr%2Fcvq217"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Proteasome functional insufficiency activates the calcineurin-NFAT pathway in cardiomyocytes and promotes maladaptive remodelling of stressed mouse hearts</span></div><div class="casAuthors">Tang, Mingxin; Li, Jie; Huang, Wei; Su, Huabo; Liang, Qiangrong; Tian, Zongwen; Horak, Kathleen M.; Molkentin, Jeffery D.; Wang, Xuejun</div><div class="citationInfo"><span class="NLM_cas:title">Cardiovascular Research</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">88</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">424-433</span>CODEN:
                <span class="NLM_cas:coden">CVREAU</span>;
        ISSN:<span class="NLM_cas:issn">0008-6363</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Aims Proteasome functional insufficiency (PFI) may play an important role in the progression of congestive heart failure but the underlying mol. mechanism is poorly understood.  Calcineurin and nuclear factor of activated T-cells (NFAT) are degraded by the proteasome, and the calcineurin-NFAT pathway mediates cardiac remodelling.  The present study examd. the hypothesis that PFI activates the calcineurin-NFAT pathway and promotes maladaptive remodelling of the heart.  Methods and results Using a reporter gene assay, we found that pharmacol. inhibition of 20S proteasomes stimulated NFAT transactivation in both mouse hearts and cultured adult mouse cardiomyocytes.  Proteasome inhibition stimulated NFAT nuclear translocation in a calcineurin-dependent manner and led to a maladaptive cell shape change in cultured neonatal rat ventricular myocytes.  Proteasome inhibition facilitated left ventricular dilatation and functional decompensation and increased fatality in mice with aortic constriction while causing cardiac hypertrophy in the sham surgery group.  It was further revealed that both calcineurin protein levels and NFAT transactivation were markedly increased in the mouse hearts with desmin-related cardiomyopathy and severe PFI.  Expression of an aggregation-prone mutant desmin also directly increased calcineurin protein levels in cultured cardiomyocytes.  Conclusions The calcineurin-NFAT pathway in the heart can be activated by proteasome inhibition and is activated in the heart of a mouse model of desmin-related cardiomyopathy that is characterized by severe PFI.  The interplay between PFI and the calcineurin-NFAT pathway may contribute to the pathol. remodelling of cardiomyocytes characteristic of congestive heart failure.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqcCPCW33B9I7Vg90H21EOLACvtfcHk0ljfPd0YClH1eQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsVSltLvI&md5=fe920fc0f006dd0c98fa7308063954b7</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1093%2Fcvr%2Fcvq217&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fcvr%252Fcvq217%26sid%3Dliteratum%253Aachs%26aulast%3DTang%26aufirst%3DM.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DHuang%26aufirst%3DW.%26aulast%3DSu%26aufirst%3DH.%26aulast%3DLiang%26aufirst%3DQ.%26aulast%3DTian%26aufirst%3DZ.%26aulast%3DHorak%26aufirst%3DK.%2BM.%26aulast%3DMolkentin%26aufirst%3DJ.%2BD.%26aulast%3DWang%26aufirst%3DX.%26atitle%3DProteasome%2520functional%2520insufficiency%2520activates%2520the%2520calcineurin-NFAT%2520pathway%2520in%2520cardiomyocytes%2520and%2520promotes%2520maladaptive%2520remodelling%2520of%2520stressed%2520mouse%2520hearts%26jtitle%3DCardiovasc.%2520Res.%26date%3D2010%26volume%3D88%26spage%3D424%26epage%3D433%26doi%3D10.1093%2Fcvr%2Fcvq217" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parajuli, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewno, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bedford, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mayer, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cui, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span> <span> </span><span class="NLM_article-title">The calcineurin-TFEB-p62 pathway mediates the activation of cardiac macroautophagy by proteasomal malfunction</span>. <i>Circ. Res.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>127</i></span>,  <span class="NLM_fpage">502</span>– <span class="NLM_lpage">518</span>, <span class="refDoi"> DOI: 10.1161/CIRCRESAHA.119.316007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1161%2FCIRCRESAHA.119.316007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=32366200" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhsV2mt73N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=127&publication_year=2020&pages=502-518&author=B.+Panauthor=J.+Liauthor=N.+Parajuliauthor=Z.+Tianauthor=P.+Wuauthor=M.+T.+Lewnoauthor=J.+Zouauthor=W.+Wangauthor=L.+Bedfordauthor=R.+J.+Mayerauthor=J.+Fangauthor=J.+Liuauthor=T.+Cuiauthor=H.+Suauthor=X.+Wang&title=The+calcineurin-TFEB-p62+pathway+mediates+the+activation+of+cardiac+macroautophagy+by+proteasomal+malfunction&doi=10.1161%2FCIRCRESAHA.119.316007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">The Calcineurin-TFEB-p62 Pathway Mediates the Activation of Cardiac Macroautophagy by Proteasomal Malfunction</span></div><div class="casAuthors">Pan, Bo; Li, Jie; Parajuli, Nirmal; Tian, Zongwen; Wu, Penglong; Lewno, Megan T.; Zou, Jianqiu; Wang, Wenjuan; Bedford, Lynn; Mayer, R. John; Fang, Jing; Liu, Jinbao; Cui, Taixing; Su, Huabo; Wang, Xuejun</div><div class="citationInfo"><span class="NLM_cas:title">Circulation Research</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">127</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">502-518</span>CODEN:
                <span class="NLM_cas:coden">CIRUAL</span>;
        ISSN:<span class="NLM_cas:issn">0009-7330</span>.
    
            (<span class="NLM_cas:orgname">Lippincott Williams & Wilkins</span>)
        </div><div class="casAbstract">Objective: Systemic proteasome inhibition (PSMI) was shown to increase p62/SQSTM1 expression and induce myocardial macroautophagy.  Methods and results: Myocardial macroautophagy, TFEB (transcription factor EB) expression and activity, and p62 expression were markedly increased in mice with either cardiomyocyte-restricted ablation of Psmc1 (an essential proteasome subunit gene) or pharmacol. PSMI.  In cultured cardiomyocytes, PSMI-induced increases in TFEB activation and p62 expression were blunted by pharmacol. and genetic calcineurin inhibition and by siRNA-mediated Molcn1 silencing.  Bortezomib-induced left ventricular wall thickening and diastolic malfunction was exacerbated by p62 deficiency.  In cultured cardiomyocytes from wild type mice but not p62-KO mice, PSMI induced increases in LC3-II flux and the lysosomal removal of ubiquitinated proteins.  Myocardial TFEB activation by PSMI as reflected by TFEB nuclear localization and target gene expression was strikingly less in p62-KO mice compared with wild type mice.  Conclusions: (1) The activation of cardiac macroautophagy by proteasomal malfunction is mediated by the Mocln1-calcineurin-TFEB-p62 pathway; (2) p62 unexpectedly exerts a feed-forward effect on TFEB activation by proteasome malfunction; and (3) targeting the Mcoln1-calcineurin-TFEB-p62 pathway provide new means to intervene cardiac autophagic-lysosomal pathway activation during proteasome malfunction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_tFYmiHwBnrVg90H21EOLACvtfcHk0ljqJ5TJbY-4IQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhsV2mt73N&md5=7340faa0f31afcdd32cb4ee83d4fa5ea</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1161%2FCIRCRESAHA.119.316007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1161%252FCIRCRESAHA.119.316007%26sid%3Dliteratum%253Aachs%26aulast%3DPan%26aufirst%3DB.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DParajuli%26aufirst%3DN.%26aulast%3DTian%26aufirst%3DZ.%26aulast%3DWu%26aufirst%3DP.%26aulast%3DLewno%26aufirst%3DM.%2BT.%26aulast%3DZou%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DBedford%26aufirst%3DL.%26aulast%3DMayer%26aufirst%3DR.%2BJ.%26aulast%3DFang%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DCui%26aufirst%3DT.%26aulast%3DSu%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DX.%26atitle%3DThe%2520calcineurin-TFEB-p62%2520pathway%2520mediates%2520the%2520activation%2520of%2520cardiac%2520macroautophagy%2520by%2520proteasomal%2520malfunction%26jtitle%3DCirc.%2520Res.%26date%3D2020%26volume%3D127%26spage%3D502%26epage%3D518%26doi%3D10.1161%2FCIRCRESAHA.119.316007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ding, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fei, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, B.</span></span> <span> </span><span class="NLM_article-title">Emerging new concepts of degrader technologies</span>. <i>Trends Pharmacol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">464</span>– <span class="NLM_lpage">474</span>, <span class="refDoi"> DOI: 10.1016/j.tips.2020.04.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1016%2Fj.tips.2020.04.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=32416934" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BB3cXovVChurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2020&pages=464-474&author=Y.+Dingauthor=Y.+Feiauthor=B.+Lu&title=Emerging+new+concepts+of+degrader+technologies&doi=10.1016%2Fj.tips.2020.04.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Emerging New Concepts of Degrader Technologies</span></div><div class="casAuthors">Ding, Yu; Fei, Yiyan; Lu, Boxun</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Pharmacological Sciences</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">464-474</span>CODEN:
                <span class="NLM_cas:coden">TPHSDY</span>;
        ISSN:<span class="NLM_cas:issn">0165-6147</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Traditional drug discovery focuses on identifying direct inhibitors of target proteins.  This typically relies on a measurable biochem. readout and accessible binding sites whose occupancy influences the function of the target protein.  These requirements preclude many disease-causing proteins from being 'druggable' targets, and these proteins are categorized as 'undruggable'.  The proteolysis-targeting chimera (PROTAC) technol. provides powerful tools to degrade these undruggable targets and has become a promising approach for drug discovery.  However, the PROTAC technol. has some limitations, and emerging new degrader technologies may greatly broaden the spectrum of targets that could be selectively degraded by harnessing a second major degrdn. pathway in cells.  We review key emerging technologies that exploit the lysosomal degrdn. pathway and discuss their potential applications and limitations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrlDwQwefE2UrVg90H21EOLACvtfcHk0ljqJ5TJbY-4IQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXovVChurk%253D&md5=8614b50f72ee30c80d965db683cfd194</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.tips.2020.04.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tips.2020.04.005%26sid%3Dliteratum%253Aachs%26aulast%3DDing%26aufirst%3DY.%26aulast%3DFei%26aufirst%3DY.%26aulast%3DLu%26aufirst%3DB.%26atitle%3DEmerging%2520new%2520concepts%2520of%2520degrader%2520technologies%26jtitle%3DTrends%2520Pharmacol.%2520Sci.%26date%3D2020%26volume%3D41%26spage%3D464%26epage%3D474%26doi%3D10.1016%2Fj.tips.2020.04.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dybas, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herrmann, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weitzman, M. D.</span></span> <span> </span><span class="NLM_article-title">Ubiquitination at the interface of tumor viruses and DNA damage responses</span>. <i>Curr. Opin. Virol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">40</span>– <span class="NLM_lpage">47</span>, <span class="refDoi"> DOI: 10.1016/j.coviro.2018.08.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1016%2Fj.coviro.2018.08.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=30261451" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhs12nt7jO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2018&pages=40-47&author=J.+M.+Dybasauthor=C.+Herrmannauthor=M.+D.+Weitzman&title=Ubiquitination+at+the+interface+of+tumor+viruses+and+DNA+damage+responses&doi=10.1016%2Fj.coviro.2018.08.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Ubiquitination at the interface of tumor viruses and DNA damage responses</span></div><div class="casAuthors">Dybas, Joseph M.; Herrmann, Christin; Weitzman, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Virology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">40-47</span>CODEN:
                <span class="NLM_cas:coden">COVUAF</span>;
        ISSN:<span class="NLM_cas:issn">1879-6257</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Viruses exploit cellular ubiquitination machinery to shape the host proteome and promote productive infection.  Among the cellular processes influenced by viral manipulation of ubiquitination is the DNA damage response (DDR), a network of cellular signaling pathways that sense and respond to genomic damage.  This host-pathogen interaction is particularly important during virus replication and transformation by DNA tumor viruses.  Manipulating DDR pathways can promote virus replication but also impacts host genomic instability, potentially leading to cellular transformation and tumor formation.  We review ways in which viruses are known to hijack the cellular ubiquitin system to reshape host DDR pathways.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr1tDS_SCm3WrVg90H21EOLACvtfcHk0ljqJ5TJbY-4IQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhs12nt7jO&md5=a09b76664af51715fc0d65e661c5189d</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.coviro.2018.08.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.coviro.2018.08.017%26sid%3Dliteratum%253Aachs%26aulast%3DDybas%26aufirst%3DJ.%2BM.%26aulast%3DHerrmann%26aufirst%3DC.%26aulast%3DWeitzman%26aufirst%3DM.%2BD.%26atitle%3DUbiquitination%2520at%2520the%2520interface%2520of%2520tumor%2520viruses%2520and%2520DNA%2520damage%2520responses%26jtitle%3DCurr.%2520Opin.%2520Virol.%26date%3D2018%26volume%3D32%26spage%3D40%26epage%3D47%26doi%3D10.1016%2Fj.coviro.2018.08.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mahon, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krogan, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craik, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pick, E.</span></span> <span> </span><span class="NLM_article-title">Cullin E3 ligases and their rewiring by viral factors</span>. <i>Biomolecules</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">897</span>– <span class="NLM_lpage">930</span>, <span class="refDoi"> DOI: 10.3390/biom4040897</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.3390%2Fbiom4040897" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=25314029" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvV2isrzI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2014&pages=897-930&author=C.+Mahonauthor=N.+J.+Kroganauthor=C.+S.+Craikauthor=E.+Pick&title=Cullin+E3+ligases+and+their+rewiring+by+viral+factors&doi=10.3390%2Fbiom4040897"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Cullin E3 ligases and their rewiring by viral factors</span></div><div class="casAuthors">Mahon, Cathal; Krogan, Nevan J.; Craik, Charles S.; Pick, Elah</div><div class="citationInfo"><span class="NLM_cas:title">Biomolecules</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">897-930, 34 pp.</span>CODEN:
                <span class="NLM_cas:coden">BIOMHC</span>;
        ISSN:<span class="NLM_cas:issn">2218-273X</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">A review.  The ability of viruses to subvert host pathways is central in disease pathogenesis.  Over the past decade, a crit. role for the Ubiquitin Proteasome System (UPS) in counteracting host immune factors during viral infection has emerged.  This counteraction is commonly achieved by the expression of viral proteins capable of sequestering host ubiquitin E3 ligases and their regulators.  In particular, many viruses hijack members of the Cullin-RING E3 Ligase (CRL) family.  Viruses interact in many ways with CRLs in order to impact their ligase activity; one key recurring interaction involves re-directing CRL complexes to degrade host targets that are otherwise not degraded within host cells.  Removal of host immune factors by this mechanism creates a more amenable cellular environment for viral propagation.  To date, a small no. of target host factors have been identified, many of which are degraded via a CRL-proteasome pathway.  Substantial effort within the field is ongoing to uncover the identities of further host proteins targeted in this fashion and the underlying mechanisms driving their turnover by the UPS.  Elucidation of these targets and mechanisms will provide appealing anti-viral therapeutic opportunities.  This review is focused on the many methods used by viruses to perturb host CRLs, focusing on substrate sequestration and viral regulation of E3 activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGowiATFlVQnE7Vg90H21EOLACvtfcHk0lgGpGhQQGLNBA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvV2isrzI&md5=dec804942b4aec1f00a77a3570c95b47</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.3390%2Fbiom4040897&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fbiom4040897%26sid%3Dliteratum%253Aachs%26aulast%3DMahon%26aufirst%3DC.%26aulast%3DKrogan%26aufirst%3DN.%2BJ.%26aulast%3DCraik%26aufirst%3DC.%2BS.%26aulast%3DPick%26aufirst%3DE.%26atitle%3DCullin%2520E3%2520ligases%2520and%2520their%2520rewiring%2520by%2520viral%2520factors%26jtitle%3DBiomolecules%26date%3D2014%26volume%3D4%26spage%3D897%26epage%3D930%26doi%3D10.3390%2Fbiom4040897" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dharmasiri, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dharmasiri, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Estelle, M.</span></span> <span> </span><span class="NLM_article-title">The F-box protein TIR1 is an auxin receptor</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>435</i></span>,  <span class="NLM_fpage">441</span>– <span class="NLM_lpage">445</span>, <span class="refDoi"> DOI: 10.1038/nature03543</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1038%2Fnature03543" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=15917797" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BD2MXksVeisrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=435&publication_year=2005&pages=441-445&author=N.+Dharmasiriauthor=S.+Dharmasiriauthor=M.+Estelle&title=The+F-box+protein+TIR1+is+an+auxin+receptor&doi=10.1038%2Fnature03543"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">The F-box protein TIR1 is an auxin receptor</span></div><div class="casAuthors">Dharmasiri, Nihal; Dharmasiri, Sunethra; Estelle, Mark</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">435</span>
        (<span class="NLM_cas:issue">7041</span>),
    <span class="NLM_cas:pages">441-445</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The plant hormone auxin regulates diverse aspects of plant growth and development.  Recent studies indicate that auxin acts by promoting the degrdn. of the Aux/IAA transcriptional repressors through the action of the ubiquitin protein ligase SCFTIR1.  The nature of the signalling cascade that leads to this effect is not known.  However, recent studies indicate that the auxin receptor and other signalling components involved in this response are sol. factors.  Using an in vitro pull-down assay, we demonstrate that the interaction between transport inhibitor response 1 (TIR1) and Aux/IAA proteins does not require stable modification of either protein.  Instead auxin promotes the Aux/IAA-SCFTIR1 interaction by binding directly to SCFTIR1.  We further show that the loss of TIR1 and three related F-box proteins eliminates saturable auxin binding in plant exts.  Finally, TIR1 synthesized in insect cells binds Aux/IAA proteins in an auxin-dependent manner.  Together, these results indicate that TIR1 is an auxin receptor that mediates Aux/IAA degrdn. and auxin-regulated transcription.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqz0XlTe0QR7Vg90H21EOLACvtfcHk0lgGpGhQQGLNBA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXksVeisrs%253D&md5=db87d9696f0bdb7003b79a7a95ab32d8</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1038%2Fnature03543&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature03543%26sid%3Dliteratum%253Aachs%26aulast%3DDharmasiri%26aufirst%3DN.%26aulast%3DDharmasiri%26aufirst%3DS.%26aulast%3DEstelle%26aufirst%3DM.%26atitle%3DThe%2520F-box%2520protein%2520TIR1%2520is%2520an%2520auxin%2520receptor%26jtitle%3DNature%26date%3D2005%26volume%3D435%26spage%3D441%26epage%3D445%26doi%3D10.1038%2Fnature03543" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Burslem, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Proteolysis-targeting chimeras as therapeutics and tools for biological discovery</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>181</i></span>,  <span class="NLM_fpage">102</span>– <span class="NLM_lpage">114</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2019.11.031</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1016%2Fj.cell.2019.11.031" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=31955850" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BB3cXht1Sitrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=181&publication_year=2020&pages=102-114&author=G.+M.+Burslemauthor=C.+M.+Crews&title=Proteolysis-targeting+chimeras+as+therapeutics+and+tools+for+biological+discovery&doi=10.1016%2Fj.cell.2019.11.031"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Proteolysis-Targeting Chimeras as Therapeutics and Tools for Biological Discovery</span></div><div class="casAuthors">Burslem, George M.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">181</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">102-114</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  New biol. tools provide new techniques to probe fundamental biol. processes.  Here we describe the burgeoning field of proteolysis-targeting chimeras (PROTACs), which are capable of modulating protein concns. at a post-translational level by co-opting the ubiquitin-proteasome system.  We describe the PROTAC technol. and its application to drug discovery and provide examples where PROTACs have enabled novel biol. insights.  Furthermore, we provide a workflow for PROTAC development and use and discuss the benefits and issues assocd. with PROTACs.  Finally, we compare PROTAC-mediated protein-level modulation with other technologies, such as RNAi and genome editing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXgumsEakVWrVg90H21EOLACvtfcHk0lgGpGhQQGLNBA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXht1Sitrk%253D&md5=3010929d2979cf04dc6003e66575f1b0</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2019.11.031&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2019.11.031%26sid%3Dliteratum%253Aachs%26aulast%3DBurslem%26aufirst%3DG.%2BM.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DProteolysis-targeting%2520chimeras%2520as%2520therapeutics%2520and%2520tools%2520for%2520biological%2520discovery%26jtitle%3DCell%26date%3D2020%26volume%3D181%26spage%3D102%26epage%3D114%26doi%3D10.1016%2Fj.cell.2019.11.031" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lai, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Induced protein degradation: an emerging drug discovery paradigm</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">101</span>– <span class="NLM_lpage">114</span>, <span class="refDoi"> DOI: 10.1038/nrd.2016.211</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1038%2Fnrd.2016.211" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=27885283" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFCit7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=101-114&author=A.+C.+Laiauthor=C.+M.+Crews&title=Induced+protein+degradation%3A+an+emerging+drug+discovery+paradigm&doi=10.1038%2Fnrd.2016.211"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Induced protein degradation: an emerging drug discovery paradigm</span></div><div class="casAuthors">Lai, Ashton C.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">101-114</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Small-mol. drug discovery has traditionally focused on occupancy of a binding site that directly affects protein function, and this approach typically precludes targeting proteins that lack such amenable sites.  Furthermore, high systemic drug exposures may be needed to maintain sufficient target inhibition in vivo, increasing the risk of undesirable off-target effects.  Induced protein degrdn. is an alternative approach that is event-driven: upon drug binding, the target protein is tagged for elimination.  Emerging technologies based on proteolysis-targeting chimaeras (PROTACs) that exploit cellular quality control machinery to selectively degrade target proteins are attracting considerable attention in the pharmaceutical industry owing to the advantages they could offer over traditional small-mol. strategies.  These advantages include the potential to reduce systemic drug exposure, the ability to counteract increased target protein expression that often accompanies inhibition of protein function and the potential ability to target proteins that are not currently therapeutically tractable, such as transcription factors, scaffolding and regulatory proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoj-O7zXGOpXLVg90H21EOLACvtfcHk0lgGpGhQQGLNBA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFCit7rP&md5=a01ff43e9b696fe4dc03f0815158a98f</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.211&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.211%26sid%3Dliteratum%253Aachs%26aulast%3DLai%26aufirst%3DA.%2BC.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DInduced%2520protein%2520degradation%253A%2520an%2520emerging%2520drug%2520discovery%2520paradigm%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2017%26volume%3D16%26spage%3D101%26epage%3D114%26doi%3D10.1038%2Fnrd.2016.211" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calabrese, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bullock, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Targeted protein degradation: expanding the toolbox</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">949</span>– <span class="NLM_lpage">963</span>, <span class="refDoi"> DOI: 10.1038/s41573-019-0047-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1038%2Fs41573-019-0047-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=31666732" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitVKrtLrM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=949-963&author=M.+Schapiraauthor=M.+F.+Calabreseauthor=A.+N.+Bullockauthor=C.+M.+Crews&title=Targeted+protein+degradation%3A+expanding+the+toolbox&doi=10.1038%2Fs41573-019-0047-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted protein degradation: expanding the toolbox</span></div><div class="casAuthors">Schapira, Matthieu; Calabrese, Matthew F.; Bullock, Alex N.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">949-963</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review. Proteolysis-targeting chimeras (PROTACs) and related mols. that induce targeted protein degrdn. by the ubiquitin-proteasome system represent a new therapeutic modality and are the focus of great interest, owing to potential advantages over traditional occupancy-based inhibitors with respect to dosing, side effects, drug resistance and modulating 'undruggable' targets.  However, the technol. is still maturing, and the design elements for successful PROTAC-based drugs are currently being elucidated.  Importantly, fewer than 10 of the more than 600 E3 ubiquitin ligases have so far been exploited for targeted protein degrdn., and expansion of knowledge in this area is a key opportunity.  Here, we briefly discuss lessons learned about targeted protein degrdn. in chem. biol. and drug discovery and systematically review the expression profile, domain architecture and chem. tractability of human E3 ligases that could expand the toolbox for PROTAC discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwUvM27MU3WrVg90H21EOLACvtfcHk0lgY_4gq3UTcBg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitVKrtLrM&md5=9bfd48c6c47fc1cce15129495fea7079</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1038%2Fs41573-019-0047-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41573-019-0047-y%26sid%3Dliteratum%253Aachs%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DCalabrese%26aufirst%3DM.%2BF.%26aulast%3DBullock%26aufirst%3DA.%2BN.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DTargeted%2520protein%2520degradation%253A%2520expanding%2520the%2520toolbox%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2019%26volume%3D18%26spage%3D949%26epage%3D963%26doi%3D10.1038%2Fs41573-019-0047-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Setten, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossi, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, S.-p.</span></span> <span> </span><span class="NLM_article-title">The current state and future directions of RNAi-based therapeutics</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">421</span>– <span class="NLM_lpage">446</span>, <span class="refDoi"> DOI: 10.1038/s41573-019-0017-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1038%2Fs41573-019-0017-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=30846871" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC1MXms1GjtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=421-446&author=R.+L.+Settenauthor=J.+J.+Rossiauthor=S.-p.+Han&title=The+current+state+and+future+directions+of+RNAi-based+therapeutics&doi=10.1038%2Fs41573-019-0017-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">The current state and future directions of RNAi-based therapeutics</span></div><div class="casAuthors">Setten, Ryan L.; Rossi, John J.; Han, Si-ping</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">421-446</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review.  The RNA interference (RNAi) pathway regulates mRNA stability and translation in nearly all human cells.  Small double-stranded RNA mols. can efficiently trigger RNAi silencing of specific genes, but their therapeutic use has faced numerous challenges involving safety and potency.  However, August 2018 marked a new era for the field, with the US Food and Drug Administration approving ONPATTRO, the first RNAi-based drug.  In this Review, we discuss key advances in the design and development of RNAi drugs leading up to this landmark achievement, the state of the current clin. pipeline and prospects for future advances, including novel RNAi pathway agents utilizing mechanisms beyond post-translational RNAi silencing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr2SSDt9nwcmLVg90H21EOLACvtfcHk0lgY_4gq3UTcBg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXms1GjtL8%253D&md5=679994c46a9e0e4868a97001b742ce90</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1038%2Fs41573-019-0017-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41573-019-0017-4%26sid%3Dliteratum%253Aachs%26aulast%3DSetten%26aufirst%3DR.%2BL.%26aulast%3DRossi%26aufirst%3DJ.%2BJ.%26aulast%3DHan%26aufirst%3DS.-p.%26atitle%3DThe%2520current%2520state%2520and%2520future%2520directions%2520of%2520RNAi-based%2520therapeutics%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2019%26volume%3D18%26spage%3D421%26epage%3D446%26doi%3D10.1038%2Fs41573-019-0017-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brunetti, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gundry, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sorcini, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guzman, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Y.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramabadran, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gionfriddo, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mezzasoma, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milano, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nabet, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kornblau, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sportoletti, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martelli, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Falini, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goodell, M. A.</span></span> <span> </span><span class="NLM_article-title">Mutant NPM1 maintains the leukemic state through HOX expression</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">499</span>– <span class="NLM_lpage">512</span>, <span class="refDoi"> DOI: 10.1016/j.ccell.2018.08.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1016%2Fj.ccell.2018.08.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=30205049" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhs12ntLbM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2018&pages=499-512&author=L.+Brunettiauthor=M.+C.+Gundryauthor=D.+Sorciniauthor=A.+G.+Guzmanauthor=Y.-H.+Huangauthor=R.+Ramabadranauthor=I.+Gionfriddoauthor=F.+Mezzasomaauthor=F.+Milanoauthor=B.+Nabetauthor=D.+L.+Buckleyauthor=S.+M.+Kornblauauthor=C.+Y.+Linauthor=P.+Sportolettiauthor=M.+P.+Martelliauthor=B.+Faliniauthor=M.+A.+Goodell&title=Mutant+NPM1+maintains+the+leukemic+state+through+HOX+expression&doi=10.1016%2Fj.ccell.2018.08.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Mutant NPM1 Maintains the Leukemic State through HOX Expression</span></div><div class="casAuthors">Brunetti, Lorenzo; Gundry, Michael C.; Sorcini, Daniele; Guzman, Anna G.; Huang, Yung-Hsin; Ramabadran, Raghav; Gionfriddo, Ilaria; Mezzasoma, Federica; Milano, Francesca; Nabet, Behnam; Buckley, Dennis L.; Kornblau, Steven M.; Lin, Charles Y.; Sportoletti, Paolo; Martelli, Maria Paola; Falini, Brunangelo; Goodell, Margaret A.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">499-512.e9</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">NPM1 is the most frequently mutated gene in cytogenetically normal acute myeloid leukemia (AML).  In AML cells, NPM1 mutations result in abnormal cytoplasmic localization of the mutant protein (NPM1c); however, it is unknown whether NPM1c is required to maintain the leukemic state.  Here, we show that loss of NPM1c from the cytoplasm, either through nuclear relocalization or targeted degrdn., results in immediate downregulation of homeobox (HOX) genes followed by differentiation.  Finally, we show that XPO1 inhibition relocalizes NPM1c to the nucleus, promotes differentiation of AML cells, and prolongs survival of Npm1-mutated leukemic mice.  We describe an exquisite dependency of NPM1-mutant AML cells on NPM1c, providing the rationale for the use of nuclear export inhibitors in AML with mutated NPM1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrwmZdVS4yl8LVg90H21EOLACvtfcHk0lgY_4gq3UTcBg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhs12ntLbM&md5=909916554bb4861ed8b2f9f39d108cf4</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2018.08.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2018.08.005%26sid%3Dliteratum%253Aachs%26aulast%3DBrunetti%26aufirst%3DL.%26aulast%3DGundry%26aufirst%3DM.%2BC.%26aulast%3DSorcini%26aufirst%3DD.%26aulast%3DGuzman%26aufirst%3DA.%2BG.%26aulast%3DHuang%26aufirst%3DY.-H.%26aulast%3DRamabadran%26aufirst%3DR.%26aulast%3DGionfriddo%26aufirst%3DI.%26aulast%3DMezzasoma%26aufirst%3DF.%26aulast%3DMilano%26aufirst%3DF.%26aulast%3DNabet%26aufirst%3DB.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DKornblau%26aufirst%3DS.%2BM.%26aulast%3DLin%26aufirst%3DC.%2BY.%26aulast%3DSportoletti%26aufirst%3DP.%26aulast%3DMartelli%26aufirst%3DM.%2BP.%26aulast%3DFalini%26aufirst%3DB.%26aulast%3DGoodell%26aufirst%3DM.%2BA.%26atitle%3DMutant%2520NPM1%2520maintains%2520the%2520leukemic%2520state%2520through%2520HOX%2520expression%26jtitle%3DCancer%2520Cell%26date%3D2018%26volume%3D34%26spage%3D499%26epage%3D512%26doi%3D10.1016%2Fj.ccell.2018.08.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Clift, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mcewan, W. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Labzin, L. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Konieczny, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mogessie, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">James, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuh, M. J. C.</span></span> <span> </span><span class="NLM_article-title">A Method for the acute and rapid degradation of endogenous proteins</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>171</i></span>,  <span class="NLM_fpage">1692</span>– <span class="NLM_lpage">1706</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2017.10.033</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1016%2Fj.cell.2017.10.033" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=29153837" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvValur%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=171&publication_year=2017&pages=1692-1706&author=D.+Cliftauthor=W.+A.+Mcewanauthor=L.+I.+Labzinauthor=V.+Koniecznyauthor=B.+Mogessieauthor=L.+C.+Jamesauthor=M.+J.+C.+Schuh&title=A+Method+for+the+acute+and+rapid+degradation+of+endogenous+proteins&doi=10.1016%2Fj.cell.2017.10.033"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">A Method for the Acute and Rapid Degradation of Endogenous Proteins</span></div><div class="casAuthors">Clift, Dean; McEwan, William A.; Labzin, Larisa I.; Konieczny, Vera; Mogessie, Binyam; James, Leo C.; Schuh, Melina</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">171</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1692-1706.e18</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Methods for the targeted disruption of protein function have revolutionized science and greatly expedited the systematic characterization of genes.  Two main approaches are currently used to disrupt protein function: DNA knockout and RNA interference, which act at the genome and mRNA level, resp.  A method that directly alters endogenous protein levels is currently not available.  Here, the authors present Trim-Away, a technique to degrade endogenous proteins acutely in mammalian cells without prior modification of the genome or mRNA.  Trim-Away harnesses the cellular protein degrdn. machinery to remove unmodified native proteins within minutes of application.  This rapidity minimizes the risk that phenotypes are compensated and that secondary, non-specific defects accumulate over time.  Because Trim-Away uses antibodies, it can be applied to a wide range of target proteins using off-the-shelf reagents.  Trim-Away allows the study of protein function in diverse cell types, including non-dividing primary cells where genome- and RNA-targeting methods are limited.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZqwRirT2Kn7Vg90H21EOLACvtfcHk0li9P7BuwwF4Fw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvValur%252FE&md5=255d6ce0a6a66720f57e4c3d3071a3b4</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2017.10.033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2017.10.033%26sid%3Dliteratum%253Aachs%26aulast%3DClift%26aufirst%3DD.%26aulast%3DMcewan%26aufirst%3DW.%2BA.%26aulast%3DLabzin%26aufirst%3DL.%2BI.%26aulast%3DKonieczny%26aufirst%3DV.%26aulast%3DMogessie%26aufirst%3DB.%26aulast%3DJames%26aufirst%3DL.%2BC.%26aulast%3DSchuh%26aufirst%3DM.%2BJ.%2BC.%26atitle%3DA%2520Method%2520for%2520the%2520acute%2520and%2520rapid%2520degradation%2520of%2520endogenous%2520proteins%26jtitle%3DCell%26date%3D2017%26volume%3D171%26spage%3D1692%26epage%3D1706%26doi%3D10.1016%2Fj.cell.2017.10.033" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brand, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G. E.</span></span> <span> </span><span class="NLM_article-title">Locus-specific knock-in of a degradable tag for target validation studies</span>. <i>Methods Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>1953</i></span>,  <span class="NLM_fpage">105</span>– <span class="NLM_lpage">119</span>, <span class="refDoi"> DOI: 10.1007/978-1-4939-9145-7_7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1007%2F978-1-4939-9145-7_7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=30912018" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtFOju7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1953&publication_year=2019&pages=105-119&author=M.+Brandauthor=G.+E.+Winter&title=Locus-specific+knock-in+of+a+degradable+tag+for+target+validation+studies&doi=10.1007%2F978-1-4939-9145-7_7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Locus-specific knock-in of a degradable tag for target validation studies.</span></div><div class="casAuthors">Brand, Matthias; Winter, Georg E.</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Molecular Biology (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">1953</span>
        (<span class="NLM_cas:issue">Target Identification and Validation in Drug Discovery</span>),
    <span class="NLM_cas:pages">105-119</span>CODEN:
                <span class="NLM_cas:coden">MMBIED</span>;
        ISSN:<span class="NLM_cas:issn">1940-6029</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Small mol.-induced targeted protein degrdn. is a powerful approach for drug target validation given its selectivity, high kinetic resoln., dose dependency, and reversibility.  Out of the several methods that have been reported so far, the 12 kDa degrdn. tag (dTAG) system has the advantage of hijacking a degrdn. machinery that is ubiquitously expressed in all human tissues.  Therefore it is independent of expressing addnl., exogenous factors and addnl. permits target validation in vivo.  Here, we describe the protocol for generation and validation of clones harboring knock-in of a selectable dTAG cassette at the endogenous locus of proteins of interest using the near-haploid cell line KBM7.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJ07a4wMM3mLVg90H21EOLACvtfcHk0li9P7BuwwF4Fw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtFOju7k%253D&md5=bdc2cdcd6fba21a1eb4263a5c3d0c137</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1007%2F978-1-4939-9145-7_7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-1-4939-9145-7_7%26sid%3Dliteratum%253Aachs%26aulast%3DBrand%26aufirst%3DM.%26aulast%3DWinter%26aufirst%3DG.%2BE.%26atitle%3DLocus-specific%2520knock-in%2520of%2520a%2520degradable%2520tag%2520for%2520target%2520validation%2520studies%26jtitle%3DMethods%2520Mol.%2520Biol.%26date%3D2019%26volume%3D1953%26spage%3D105%26epage%3D119%26doi%3D10.1007%2F978-1-4939-9145-7_7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nabet, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulk, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dastjerdi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leggett, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erb, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawlor, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Souza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, T. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vittori, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perry, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, K.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">The dTAG system for immediate and target-specific protein degradation</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">431</span>– <span class="NLM_lpage">441</span>, <span class="refDoi"> DOI: 10.1038/s41589-018-0021-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1038%2Fs41589-018-0021-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=29581585" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC1cXosFGmsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2018&pages=431-441&author=B.+Nabetauthor=J.+M.+Robertsauthor=D.+L.+Buckleyauthor=J.+Paulkauthor=S.+Dastjerdiauthor=A.+Yangauthor=A.+L.+Leggettauthor=M.+A.+Erbauthor=M.+A.+Lawlorauthor=A.+Souzaauthor=T.+G.+Scottauthor=S.+Vittoriauthor=J.+A.+Perryauthor=J.+Qiauthor=G.+E.+Winterauthor=K.-K.+Wongauthor=N.+S.+Grayauthor=J.+E.+Bradner&title=The+dTAG+system+for+immediate+and+target-specific+protein+degradation&doi=10.1038%2Fs41589-018-0021-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">The dTAG system for immediate and target-specific protein degradation</span></div><div class="casAuthors">Nabet, Behnam; Roberts, Justin M.; Buckley, Dennis L.; Paulk, Joshiawa; Dastjerdi, Shiva; Yang, Annan; Leggett, Alan L.; Erb, Michael A.; Lawlor, Matthew A.; Souza, Amanda; Scott, Thomas G.; Vittori, Sarah; Perry, Jennifer A.; Qi, Jun; Winter, Georg E.; Wong, Kwok-Kin; Gray, Nathanael S.; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">431-441</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Dissection of complex biol. systems requires target-specific control of the function or abundance of proteins.  Genetic perturbations are limited by off-target effects, multicomponent complexity, and irreversibility.  Most limiting is the requisite delay between modulation to exptl. measurement.  To enable the immediate and selective control of single protein abundance, we created a chem. biol. system that leverages the potency of cell-permeable heterobifunctional degraders.  The dTAG system pairs a novel degrader of FKBP12F36V with expression of FKBP12F36V in-frame with a protein of interest.  By transgene expression or CRISPR-mediated locus-specific knock-in, we exemplify a generalizable strategy to study the immediate consequence of protein loss.  Using dTAG, we observe an unexpected superior antiproliferative effect of pan-BET bromodomain degrdn. over selective BRD4 degrdn., characterize immediate effects of KRASG12V loss on proteomic signaling, and demonstrate rapid degrdn. in vivo.  This technol. platform will confer kinetic resoln. to biol. investigation and provide target validation in the context of drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqwQvCDe-2I7Vg90H21EOLACvtfcHk0li9P7BuwwF4Fw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXosFGmsbs%253D&md5=d7d9c111c68c84bc3aa9c2aad3f591a6</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1038%2Fs41589-018-0021-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41589-018-0021-8%26sid%3Dliteratum%253Aachs%26aulast%3DNabet%26aufirst%3DB.%26aulast%3DRoberts%26aufirst%3DJ.%2BM.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DPaulk%26aufirst%3DJ.%26aulast%3DDastjerdi%26aufirst%3DS.%26aulast%3DYang%26aufirst%3DA.%26aulast%3DLeggett%26aufirst%3DA.%2BL.%26aulast%3DErb%26aufirst%3DM.%2BA.%26aulast%3DLawlor%26aufirst%3DM.%2BA.%26aulast%3DSouza%26aufirst%3DA.%26aulast%3DScott%26aufirst%3DT.%2BG.%26aulast%3DVittori%26aufirst%3DS.%26aulast%3DPerry%26aufirst%3DJ.%2BA.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DWinter%26aufirst%3DG.%2BE.%26aulast%3DWong%26aufirst%3DK.-K.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DThe%2520dTAG%2520system%2520for%2520immediate%2520and%2520target-specific%2520protein%2520degradation%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2018%26volume%3D14%26spage%3D431%26epage%3D441%26doi%3D10.1038%2Fs41589-018-0021-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ohoka, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okuhira, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ito, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagai, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shibata, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hattori, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ujikawa, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shimokawa, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sano, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koyama, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujita, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teratani, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsumoto, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imaeda, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nara, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naito, M.</span></span> <span> </span><span class="NLM_article-title">In vivo knockdown of pathogenic proteins via specific and nongenetic inhibitor of apoptosis protein (IAP)-dependent protein erasers (SNIPERs)</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>292</i></span>,  <span class="NLM_fpage">4556</span>– <span class="NLM_lpage">4570</span>, <span class="refDoi"> DOI: 10.1074/jbc.M116.768853</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1074%2Fjbc.M116.768853" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=28154167" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC2sXksVClur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=292&publication_year=2017&pages=4556-4570&author=N.+Ohokaauthor=K.+Okuhiraauthor=M.+Itoauthor=K.+Nagaiauthor=N.+Shibataauthor=T.+Hattoriauthor=O.+Ujikawaauthor=K.+Shimokawaauthor=O.+Sanoauthor=R.+Koyamaauthor=H.+Fujitaauthor=M.+Terataniauthor=H.+Matsumotoauthor=Y.+Imaedaauthor=H.+Naraauthor=N.+Choauthor=M.+Naito&title=In+vivo+knockdown+of+pathogenic+proteins+via+specific+and+nongenetic+inhibitor+of+apoptosis+protein+%28IAP%29-dependent+protein+erasers+%28SNIPERs%29&doi=10.1074%2Fjbc.M116.768853"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">In Vivo Knockdown of Pathogenic Proteins via Specific and Nongenetic Inhibitor of Apoptosis Protein (IAP)-dependent Protein Erasers (SNIPERs)</span></div><div class="casAuthors">Ohoka, Nobumichi; Okuhira, Keiichiro; Ito, Masahiro; Nagai, Katsunori; Shibata, Norihito; Hattori, Takayuki; Ujikawa, Osamu; Shimokawa, Kenichiro; Sano, Osamu; Koyama, Ryokichi; Fujita, Hisashi; Teratani, Mika; Matsumoto, Hirokazu; Imaeda, Yasuhiro; Nara, Hiroshi; Cho, Nobuo; Naito, Mikihiko</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">292</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4556-4570</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Many diseases, esp. cancers, result from aberrant or overexpression of pathogenic proteins.  Specific inhibitors against these proteins have shown remarkable therapeutic effects, but these are limited mainly to enzymes.  An alternative approach that may have utility in drug development relies on selective degrdn. of pathogenic proteins via small chimeric mols. linking an E3 ubiquitin ligase to the targeted protein for proteasomal degrdn.  To this end, the authors recently developed a protein knockdown system based on hybrid small mol. SNIPERs (Specific and Nongenetic IAP-dependent Protein Erasers) that recruit inhibitor of the apoptosis protein (IAP) ubiquitin ligases to specifically degrade targeted proteins.  Here, the authors extend the authors' previous study to show a proof of concept of the SNIPER technol. in vivo.  By incorporating a high affinity IAP ligand, the authors developed a novel SNIPER against estrogen receptor α (ERα), SNIPER(ER)-87, that has a potent protein knockdown activity.  The SNIPER(ER) reduced ERα levels in tumor xenografts and suppressed the growth of ERα-pos. breast tumors in mice.  Mechanistically, it preferentially recruits X-linked IAP (XIAP) rather than cellular IAP1, to degrade ERα via the ubiquitin-proteasome pathway.  With this IAP ligand, potent SNIPERs against other pathogenic proteins, BCR-ABL, bromodomain-contg. protein 4 (BRD4), and phosphodiesterase-4 (PDE4) could also be developed.  These results indicate that forced ubiquitylation by SNIPERs is a useful method to achieve efficient protein knockdown with potential therapeutic activities and could also be applied to study the role of ubiquitylation in many cellular processes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqEzH5uNeAFabVg90H21EOLACvtfcHk0ljz7Yzn-I1erQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXksVClur8%253D&md5=b09264d53c47c54343ed92d5a6d0eb79</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M116.768853&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M116.768853%26sid%3Dliteratum%253Aachs%26aulast%3DOhoka%26aufirst%3DN.%26aulast%3DOkuhira%26aufirst%3DK.%26aulast%3DIto%26aufirst%3DM.%26aulast%3DNagai%26aufirst%3DK.%26aulast%3DShibata%26aufirst%3DN.%26aulast%3DHattori%26aufirst%3DT.%26aulast%3DUjikawa%26aufirst%3DO.%26aulast%3DShimokawa%26aufirst%3DK.%26aulast%3DSano%26aufirst%3DO.%26aulast%3DKoyama%26aufirst%3DR.%26aulast%3DFujita%26aufirst%3DH.%26aulast%3DTeratani%26aufirst%3DM.%26aulast%3DMatsumoto%26aufirst%3DH.%26aulast%3DImaeda%26aufirst%3DY.%26aulast%3DNara%26aufirst%3DH.%26aulast%3DCho%26aufirst%3DN.%26aulast%3DNaito%26aufirst%3DM.%26atitle%3DIn%2520vivo%2520knockdown%2520of%2520pathogenic%2520proteins%2520via%2520specific%2520and%2520nongenetic%2520inhibitor%2520of%2520apoptosis%2520protein%2520%2528IAP%2529-dependent%2520protein%2520erasers%2520%2528SNIPERs%2529%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2017%26volume%3D292%26spage%3D4556%26epage%3D4570%26doi%3D10.1074%2Fjbc.M116.768853" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Perera, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zoncu, R.</span></span> <span> </span><span class="NLM_article-title">The lysosome as a regulatory hub</span>. <i>Annu. Rev. Cell Dev. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">223</span>– <span class="NLM_lpage">253</span>, <span class="refDoi"> DOI: 10.1146/annurev-cellbio-111315-125125</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1146%2Fannurev-cellbio-111315-125125" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=27501449" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtlaitrnO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2016&pages=223-253&author=R.+M.+Pereraauthor=R.+Zoncu&title=The+lysosome+as+a+regulatory+hub&doi=10.1146%2Fannurev-cellbio-111315-125125"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">The Lysosome as a Regulatory Hub</span></div><div class="casAuthors">Perera, Rushika M.; Zoncu, Roberto</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Cell and Developmental Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">223-253</span>CODEN:
                <span class="NLM_cas:coden">ARDBF8</span>;
        ISSN:<span class="NLM_cas:issn">1081-0706</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews</span>)
        </div><div class="casAbstract">The lysosome has long been viewed as the recycling center of the cell.  However, recent discoveries have challenged this simple view and have established a central role of the lysosome in nutrient-dependent signal transduction.  The degradative role of the lysosome and its newly discovered signaling functions are not in conflict but rather cooperate extensively to mediate fundamental cellular activities such as nutrient sensing, metabolic adaptation, and quality control of proteins and organelles.  Moreover, lysosome-based signaling and degrdn. are subject to reciprocal regulation.  Transcriptional programs of increasing complexity control the biogenesis, compn., and abundance of lysosomes and fine-tune their activity to match the evolving needs of the cell.  Alterations in these essential activities are, not surprisingly, central to the pathophysiol. of an ever-expanding spectrum of conditions, including storage disorders, neurodegenerative diseases, and cancer.  Thus, unraveling the functions of this fascinating organelle will contribute to our understanding of the fundamental logic of metabolic organization and will point to novel therapeutic avenues in several human diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpH74ZfQKHDrrVg90H21EOLACvtfcHk0ljz7Yzn-I1erQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtlaitrnO&md5=73d1ab71b801facb20ae036cdfae74a3</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1146%2Fannurev-cellbio-111315-125125&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-cellbio-111315-125125%26sid%3Dliteratum%253Aachs%26aulast%3DPerera%26aufirst%3DR.%2BM.%26aulast%3DZoncu%26aufirst%3DR.%26atitle%3DThe%2520lysosome%2520as%2520a%2520regulatory%2520hub%26jtitle%3DAnnu.%2520Rev.%2520Cell%2520Dev.%2520Biol.%26date%3D2016%26volume%3D32%26spage%3D223%26epage%3D253%26doi%3D10.1146%2Fannurev-cellbio-111315-125125" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Settembre, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fraldi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Medina, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ballabio, A.</span></span> <span> </span><span class="NLM_article-title">Signals from the lysosome: a control centre for cellular clearance and energy metabolism</span>. <i>Nat. Rev. Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">283</span>– <span class="NLM_lpage">296</span>, <span class="refDoi"> DOI: 10.1038/nrm3565</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1038%2Fnrm3565" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=23609508" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC3sXmt1Oksrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2013&pages=283-296&author=C.+Settembreauthor=A.+Fraldiauthor=D.+L.+Medinaauthor=A.+Ballabio&title=Signals+from+the+lysosome%3A+a+control+centre+for+cellular+clearance+and+energy+metabolism&doi=10.1038%2Fnrm3565"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Signals from the lysosome: a control centre for cellular clearance and energy metabolism</span></div><div class="casAuthors">Settembre, Carmine; Fraldi, Alessandro; Medina, Diego L.; Ballabio, Andrea</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">283-296</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  For a long time, lysosomes were considered merely to be cellular 'incinerators' involved in the degrdn. and recycling of cellular waste.  However, now there is compelling evidence indicating that lysosomes have a much broader function and that they are involved in fundamental processes such as secretion, plasma membrane repair, signaling and energy metab.  Furthermore, the essential role of lysosomes in autophagic pathways puts these organelles at the crossroads of several cellular processes, with significant implications for health and disease.  The identification of a master regulator, transcription factor EB (TFEB), that regulates lysosomal biogenesis and autophagy has revealed how the lysosome adapts to environmental cues, such as starvation, and targeting TFEB may provide a novel therapeutic strategy for modulating lysosomal function in human disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZpiDHfiHRzLVg90H21EOLACvtfcHk0ljz7Yzn-I1erQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXmt1Oksrc%253D&md5=6ef324f69939d95c37ef334e3d4c2c78</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1038%2Fnrm3565&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm3565%26sid%3Dliteratum%253Aachs%26aulast%3DSettembre%26aufirst%3DC.%26aulast%3DFraldi%26aufirst%3DA.%26aulast%3DMedina%26aufirst%3DD.%2BL.%26aulast%3DBallabio%26aufirst%3DA.%26atitle%3DSignals%2520from%2520the%2520lysosome%253A%2520a%2520control%2520centre%2520for%2520cellular%2520clearance%2520and%2520energy%2520metabolism%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2013%26volume%3D14%26spage%3D283%26epage%3D296%26doi%3D10.1038%2Fnrm3565" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Davidson, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vander Heiden, M. G.</span></span> <span> </span><span class="NLM_article-title">Critical functions of the lysosome in cancer biology</span>. <i>Annu. Rev. Pharmacol. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">481</span>– <span class="NLM_lpage">507</span>, <span class="refDoi"> DOI: 10.1146/annurev-pharmtox-010715-103101</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1146%2Fannurev-pharmtox-010715-103101" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=27732799" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhs1ykt7rO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2017&pages=481-507&author=S.+M.+Davidsonauthor=M.+G.+Vander+Heiden&title=Critical+functions+of+the+lysosome+in+cancer+biology&doi=10.1146%2Fannurev-pharmtox-010715-103101"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Critical Functions of the Lysosome in Cancer Biology</span></div><div class="casAuthors">Davidson, Shawn M.; Vander Heiden, Matthew G.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Pharmacology and Toxicology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">481-507</span>CODEN:
                <span class="NLM_cas:coden">ARPTDI</span>;
        ISSN:<span class="NLM_cas:issn">0362-1642</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews</span>)
        </div><div class="casAbstract">Lysosomes (or lytic bodies) were so named because they contain high levels of hydrolytic enzymes.  Lysosome function and dysfunction have been found to play important roles in human disease, including cancer; however, the ways in which lysosomes contribute to tumorigenesis and cancer progression are still being uncovered.  Beyond serving as a cellular recycling center, recent evidence suggests that the lysosome is involved in energy homeostasis, generating building blocks for cell growth, mitogenic signaling, priming tissues for angiogenesis and metastasis formation, and activating transcriptional programs.  This review examines emerging knowledge of how lysosomal processes contribute to the hallmarks of cancer and highlights vulnerabilities that might be exploited for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpekDKNs3_ogrVg90H21EOLACvtfcHk0lj1Fe4kyzv5eQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhs1ykt7rO&md5=a104126535284709504690b4d0463b1b</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1146%2Fannurev-pharmtox-010715-103101&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-pharmtox-010715-103101%26sid%3Dliteratum%253Aachs%26aulast%3DDavidson%26aufirst%3DS.%2BM.%26aulast%3DVander%2BHeiden%26aufirst%3DM.%2BG.%26atitle%3DCritical%2520functions%2520of%2520the%2520lysosome%2520in%2520cancer%2520biology%26jtitle%3DAnnu.%2520Rev.%2520Pharmacol.%2520Toxicol.%26date%3D2017%26volume%3D57%26spage%3D481%26epage%3D507%26doi%3D10.1146%2Fannurev-pharmtox-010715-103101" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bonam, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, S.</span></span> <span> </span><span class="NLM_article-title">Autophagy: a new concept in autoimmunity regulation and a novel therapeutic option</span>. <i>J. Autoimmun.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>94</i></span>,  <span class="NLM_fpage">16</span>– <span class="NLM_lpage">32</span>, <span class="refDoi"> DOI: 10.1016/j.jaut.2018.08.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1016%2Fj.jaut.2018.08.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=30219390" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslSkt7rJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=94&publication_year=2018&pages=16-32&author=S.+R.+Bonamauthor=F.+Wangauthor=S.+Muller&title=Autophagy%3A+a+new+concept+in+autoimmunity+regulation+and+a+novel+therapeutic+option&doi=10.1016%2Fj.jaut.2018.08.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Autophagy: A new concept in autoimmunity regulation and a novel therapeutic option</span></div><div class="casAuthors">Bonam, Srinivasa Reddy; Wang, Fengjuan; Muller, Sylviane</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Autoimmunity</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">94</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">16-32</span>CODEN:
                <span class="NLM_cas:coden">JOAUEP</span>;
        ISSN:<span class="NLM_cas:issn">0896-8411</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Nowadays, pharmacol. treatments of autoinflammatory diseases are largely palliative rather than curative.  Most of them result in non-specific immunosuppression, which can be assocd. with broad disruption of natural and induced immunity with significant and sometimes serious unwanted injuries.  Peptide therapeutics, class of potent drugs, the phosphopeptide P140 is very promising for treating patients with lupus, and likely also patients with other chronic inflammatory diseases.  We discovered that P140 targets autophagy, a finely orchestrated catabolic process, involved in the regulation of inflammation and in the biol. of immune cells.  In vitro, P140 acts directly on a particular form of autophagy called chaperone-mediated autophagy, which seems to be hyperactivated in certain subsets of lymphocytes in lupus and in other autoinflammatory settings.  In lupus, the "correcting" effect of P140 on autophagy results in a weaker signaling of autoreactive T cells, leading to a significant improvement of pathophysiol. status of treated mice.  These findings also demonstrated ex vivo in human cells, open novel avenues of therapeutic intervention in pathol. conditions, in which specific and not general targeting is highly pursued in the context of the new action plans for personalized medicines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGroze5ifIgxJbVg90H21EOLACvtfcHk0lj1Fe4kyzv5eQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslSkt7rJ&md5=c96a3a37777c98cc81b230b405b96087</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2Fj.jaut.2018.08.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jaut.2018.08.009%26sid%3Dliteratum%253Aachs%26aulast%3DBonam%26aufirst%3DS.%2BR.%26aulast%3DWang%26aufirst%3DF.%26aulast%3DMuller%26aufirst%3DS.%26atitle%3DAutophagy%253A%2520a%2520new%2520concept%2520in%2520autoimmunity%2520regulation%2520and%2520a%2520novel%2520therapeutic%2520option%26jtitle%3DJ.%2520Autoimmun.%26date%3D2018%26volume%3D94%26spage%3D16%26epage%3D32%26doi%3D10.1016%2Fj.jaut.2018.08.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boland, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, W. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corti, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mollereau, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henriques, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bezard, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pastores, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rubinsztein, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nixon, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duchen, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mallucci, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kroemer, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levine, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eskelinen, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mochel, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spedding, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Louis, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, O. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Millan, M. J.</span></span> <span> </span><span class="NLM_article-title">Promoting the clearance of neurotoxic proteins in neurodegenerative disorders of ageing</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">660</span>– <span class="NLM_lpage">688</span>, <span class="refDoi"> DOI: 10.1038/nrd.2018.109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1038%2Fnrd.2018.109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=30116051" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsFans73J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2018&pages=660-688&author=B.+Bolandauthor=W.+H.+Yuauthor=O.+Cortiauthor=B.+Mollereauauthor=A.+Henriquesauthor=E.+Bezardauthor=G.+M.+Pastoresauthor=D.+C.+Rubinszteinauthor=R.+A.+Nixonauthor=M.+R.+Duchenauthor=G.+R.+Mallucciauthor=G.+Kroemerauthor=B.+Levineauthor=E.+L.+Eskelinenauthor=F.+Mochelauthor=M.+Speddingauthor=C.+Louisauthor=O.+R.+Martinauthor=M.+J.+Millan&title=Promoting+the+clearance+of+neurotoxic+proteins+in+neurodegenerative+disorders+of+ageing&doi=10.1038%2Fnrd.2018.109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Promoting the clearance of neurotoxic proteins in neurodegenerative disorders of ageing</span></div><div class="casAuthors">Boland, Barry; Yu, Wai Haung; Corti, Olga; Mollereau, Bertrand; Henriques, Alexandre; Bezard, Erwan; Pastores, Greg M.; Rubinsztein, David C.; Nixon, Ralph A.; Duchen, Michael R.; Mallucci, Giovanna R.; Kroemer, Guido; Levine, Beth; Eskelinen, Eeva-Liisa; Mochel, Fanny; Spedding, Michael; Louis, Caroline; Martin, Olivier R.; Millan, Mark J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">660-688</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review.  Neurodegenerative disorders of ageing (NDAs) such as Alzheimer disease, Parkinson disease, frontotemporal dementia, Huntington disease and amyotrophic lateral sclerosis represent a major socio-economic challenge in view of their high prevalence yet poor treatment.  They are often called 'proteinopathies' owing to the presence of misfolded and aggregated proteins that lose their physiol. roles and acquire neurotoxic properties.  One reason underlying the accumulation and spread of oligomeric forms of neurotoxic proteins is insufficient clearance by the autophagic-lysosomal network.  Several other clearance pathways are also compromised in NDAs: chaperone-mediated autophagy, the ubiquitin-proteasome system, extracellular clearance by proteases and extrusion into the circulation via the blood-brain barrier and glymphatic system.  This article focuses on emerging mechanisms for promoting the clearance of neurotoxic proteins, a strategy that may curtail the onset and slow the progression of NDAs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGph_pFdlqvN4LVg90H21EOLACvtfcHk0lj1Fe4kyzv5eQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsFans73J&md5=3ae9f39bde30d40bb5349419d6d1edd2</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2018.109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2018.109%26sid%3Dliteratum%253Aachs%26aulast%3DBoland%26aufirst%3DB.%26aulast%3DYu%26aufirst%3DW.%2BH.%26aulast%3DCorti%26aufirst%3DO.%26aulast%3DMollereau%26aufirst%3DB.%26aulast%3DHenriques%26aufirst%3DA.%26aulast%3DBezard%26aufirst%3DE.%26aulast%3DPastores%26aufirst%3DG.%2BM.%26aulast%3DRubinsztein%26aufirst%3DD.%2BC.%26aulast%3DNixon%26aufirst%3DR.%2BA.%26aulast%3DDuchen%26aufirst%3DM.%2BR.%26aulast%3DMallucci%26aufirst%3DG.%2BR.%26aulast%3DKroemer%26aufirst%3DG.%26aulast%3DLevine%26aufirst%3DB.%26aulast%3DEskelinen%26aufirst%3DE.%2BL.%26aulast%3DMochel%26aufirst%3DF.%26aulast%3DSpedding%26aufirst%3DM.%26aulast%3DLouis%26aufirst%3DC.%26aulast%3DMartin%26aufirst%3DO.%2BR.%26aulast%3DMillan%26aufirst%3DM.%2BJ.%26atitle%3DPromoting%2520the%2520clearance%2520of%2520neurotoxic%2520proteins%2520in%2520neurodegenerative%2520disorders%2520of%2520ageing%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2018%26volume%3D17%26spage%3D660%26epage%3D688%26doi%3D10.1038%2Fnrd.2018.109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tomoshige, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishikawa, M.</span></span> <span> </span><span class="NLM_article-title">PROTACs and other chemical protein degradation technologies for the treatment of neurodegenerative disorders</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">3346</span>– <span class="NLM_lpage">3354</span>, <span class="refDoi"> DOI: 10.1002/anie.202004746</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1002%2Fanie.202004746" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhs1ersbbM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2021&pages=3346-3354&author=S.+Tomoshigeauthor=M.+Ishikawa&title=PROTACs+and+other+chemical+protein+degradation+technologies+for+the+treatment+of+neurodegenerative+disorders&doi=10.1002%2Fanie.202004746"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">PROTACs and Other Chemical Protein Degradation Technologies for the Treatment of Neurodegenerative Disorders</span></div><div class="casAuthors">Tomoshige, Shusuke; Ishikawa, Minoru</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3346-3354</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  Neurodegenerative disorders (NDs) are a group of diseases that cause neural cell damage, leading to motility and/or cognitive dysfunctions.  One of the causative agents is misfolded protein aggregates, which are considered as undruggable in terms of conventional tools, such as inhibitors and agonists/antagonists.  Indeed, there is currently no FDA-approved drug for the causal treatment of NDs.  However, emerging technologies for chem. protein degrdn. are opening up the possibility of selective elimination of target proteins through physiol. protein degrdn. machineries, which do not depend on the functions of the target proteins.  Here, we review recent efforts towards the treatment of NDs using chem. protein degrdn. technologies, and we briefly discuss the challenges and prospects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptTufLqJjYuLVg90H21EOLACvtfcHk0lj1Fe4kyzv5eQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhs1ersbbM&md5=0438ca3689960eac5a722b755c5cbdd5</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1002%2Fanie.202004746&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.202004746%26sid%3Dliteratum%253Aachs%26aulast%3DTomoshige%26aufirst%3DS.%26aulast%3DIshikawa%26aufirst%3DM.%26atitle%3DPROTACs%2520and%2520other%2520chemical%2520protein%2520degradation%2520technologies%2520for%2520the%2520treatment%2520of%2520neurodegenerative%2520disorders%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2021%26volume%3D60%26spage%3D3346%26epage%3D3354%26doi%3D10.1002%2Fanie.202004746" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Clague, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urbé, S.</span></span> <span> </span><span class="NLM_article-title">Ubiquitin: same molecule, different degradation pathways</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>143</i></span>,  <span class="NLM_fpage">682</span>– <span class="NLM_lpage">685</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2010.11.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1016%2Fj.cell.2010.11.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=21111229" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVygsLbP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=143&publication_year=2010&pages=682-685&author=M.+J.+Clagueauthor=S.+Urb%C3%A9&title=Ubiquitin%3A+same+molecule%2C+different+degradation+pathways&doi=10.1016%2Fj.cell.2010.11.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Ubiquitin: Same molecule, different degradation pathways</span></div><div class="casAuthors">Clague, Michael J.; Urbe, Sylvie</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">143</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">682-685</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Ubiquitin is a common denominator in the targeting of substrates to all 3 major protein degrdn. pathways in mammalian cells: the proteasome, the lysosome, and the autophagosome.  The factors that direct a substrate toward a particular route of degrdn. likely include ubiquitin chain length and linkage type, which may favor interaction with particular receptors or confer differential susceptibility to deubiquitinase activities assocd. with each pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGol6eqfdT1hI7Vg90H21EOLACvtfcHk0lhGeGfkob-k2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsVygsLbP&md5=72f5984164b176a89c5fffbafe157d1c</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2010.11.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2010.11.012%26sid%3Dliteratum%253Aachs%26aulast%3DClague%26aufirst%3DM.%2BJ.%26aulast%3DUrb%25C3%25A9%26aufirst%3DS.%26atitle%3DUbiquitin%253A%2520same%2520molecule%252C%2520different%2520degradation%2520pathways%26jtitle%3DCell%26date%3D2010%26volume%3D143%26spage%3D682%26epage%3D685%26doi%3D10.1016%2Fj.cell.2010.11.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kirkin, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McEwan, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Novak, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dikic, I.</span></span> <span> </span><span class="NLM_article-title">A role for ubiquitin in selective autophagy</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">259</span>– <span class="NLM_lpage">269</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2009.04.026</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1016%2Fj.molcel.2009.04.026" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=19450525" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BD1MXmtlGlt7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2009&pages=259-269&author=V.+Kirkinauthor=D.+G.+McEwanauthor=I.+Novakauthor=I.+Dikic&title=A+role+for+ubiquitin+in+selective+autophagy&doi=10.1016%2Fj.molcel.2009.04.026"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">A role for ubiquitin in selective autophagy</span></div><div class="casAuthors">Kirkin, Vladimir; McEwan, David G.; Novak, Ivana; Dikic, Ivan</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">259-269</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Ubiquitination is the hallmark of protein degrdn. by the 26S proteasome.  However, the proteasome is limited its capacity to degrade oligomeric and aggregated proteins.  Removal of harmful protein aggregates is mediated by autophagy, a mechanism by which the cell sequesters cytosolic cargo and delivers it for degrdn. by the lysosome.  Identification of autophagy receptors, such as p62/SQSTM1 and NBR1, which simultaneously bind both ubiquitin and autophagy-specific ubiquitin-like modifiers, LC3/GABARAP, has provided a mol. link between ubiquitination and autophagy.  Here, the authors explore the hypothesis that ubiquitin represents a selective degrdn. signal suitable for targeting various types of cargo, ranging from protein aggregates to membrane-bound organelles and microbes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpVkQ7Iie0DJrVg90H21EOLACvtfcHk0lhGeGfkob-k2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXmtlGlt7c%253D&md5=5ecbfc53c64ec238a31863456f0da650</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2009.04.026&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2009.04.026%26sid%3Dliteratum%253Aachs%26aulast%3DKirkin%26aufirst%3DV.%26aulast%3DMcEwan%26aufirst%3DD.%2BG.%26aulast%3DNovak%26aufirst%3DI.%26aulast%3DDikic%26aufirst%3DI.%26atitle%3DA%2520role%2520for%2520ubiquitin%2520in%2520selective%2520autophagy%26jtitle%3DMol.%2520Cell%26date%3D2009%26volume%3D34%26spage%3D259%26epage%3D269%26doi%3D10.1016%2Fj.molcel.2009.04.026" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kraft, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peter, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hofmann, K.</span></span> <span> </span><span class="NLM_article-title">Selective autophagy: ubiquitin-mediated recognition and beyond</span>. <i>Nat. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">836</span>– <span class="NLM_lpage">841</span>, <span class="refDoi"> DOI: 10.1038/ncb0910-836</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1038%2Fncb0910-836" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=20811356" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtFSju7zI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2010&pages=836-841&author=C.+Kraftauthor=M.+Peterauthor=K.+Hofmann&title=Selective+autophagy%3A+ubiquitin-mediated+recognition+and+beyond&doi=10.1038%2Fncb0910-836"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Selective autophagy: ubiquitin-mediated recognition and beyond</span></div><div class="casAuthors">Kraft, Claudine; Peter, Matthias; Hofmann, Kay</div><div class="citationInfo"><span class="NLM_cas:title">Nature Cell Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">836-841</span>CODEN:
                <span class="NLM_cas:coden">NCBIFN</span>;
        ISSN:<span class="NLM_cas:issn">1465-7392</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Eukaryotic cells use autophagy and the ubiquitin-proteasome system as their major protein degrdn. pathways.  Whereas the ubiquitin-proteasome system is involved in the rapid degrdn. of proteins, autophagy pathways can selectively remove protein aggregates and damaged or excess organelles.  Proteasome-mediated degrdn. requires previous ubiquitylation of the cargo, which is then recognized by ubiquitin receptors directing it to 26S proteasomes.  Although autophagy has long been viewed as a random cytoplasmic degrdn. system, the involvement of ubiquitin as a specificity factor for selective autophagy is rapidly emerging.  Recent evidence also suggests active crosstalk between proteasome-mediated degrdn. and selective autophagy.  Here, we discuss the mol. mechanisms that link autophagy and the proteasome system, as well as the emerging roles of ubiquitin and ubiquitin-binding proteins in selective autophagy.  On the basis of the evolutionary history of autophagic ubiquitin receptors, we propose a common origin for metazoan ubiquitin-dependent autophagy and the cytoplasm-to-vacuole targeting pathway of yeast.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoRPRAkcgcWKrVg90H21EOLACvtfcHk0lhGeGfkob-k2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtFSju7zI&md5=53177b341b1165bf3497e81f22d7c30e</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1038%2Fncb0910-836&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncb0910-836%26sid%3Dliteratum%253Aachs%26aulast%3DKraft%26aufirst%3DC.%26aulast%3DPeter%26aufirst%3DM.%26aulast%3DHofmann%26aufirst%3DK.%26atitle%3DSelective%2520autophagy%253A%2520ubiquitin-mediated%2520recognition%2520and%2520beyond%26jtitle%3DNat.%2520Cell%2520Biol.%26date%3D2010%26volume%3D12%26spage%3D836%26epage%3D841%26doi%3D10.1038%2Fncb0910-836" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lamark, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johansen, T.</span></span> <span> </span><span class="NLM_article-title">Autophagy: links with the proteasome</span>. <i>Curr. Opin. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">192</span>– <span class="NLM_lpage">198</span>, <span class="refDoi"> DOI: 10.1016/j.ceb.2009.11.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1016%2Fj.ceb.2009.11.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=19962293" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC3cXks1SrtbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2010&pages=192-198&author=T.+Lamarkauthor=T.+Johansen&title=Autophagy%3A+links+with+the+proteasome&doi=10.1016%2Fj.ceb.2009.11.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Autophagy: links with the proteasome</span></div><div class="casAuthors">Lamark, Trond; Johansen, Terje</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Cell Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">192-198</span>CODEN:
                <span class="NLM_cas:coden">COCBE3</span>;
        ISSN:<span class="NLM_cas:issn">0955-0674</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The two main protein degrdn. systems of eukaryotic cells, the ubiquitin-proteasome system and autophagy, have been thought of as quite sep. systems.  However, recent findings strongly suggest that there is crosstalk and even cooperation between these two degrdn. pathways.  Ubiquitination and degrdn. of misfolded proteins by the ubiquitin-proteasome system have been investigated for some time, but much less is known about autophagic degrdn. of misfolded proteins.  We will here discuss recent findings that shed some light on the cellular processes deciding when and how misfolded proteins are specifically selected for autophagic degrdn. in favor of proteasomal degrdn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpgE1TVJTSLNrVg90H21EOLACvtfcHk0lijtlD1w9461A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXks1SrtbY%253D&md5=e0f73ff73f2d0943b020ed656f7bdc67</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2Fj.ceb.2009.11.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ceb.2009.11.002%26sid%3Dliteratum%253Aachs%26aulast%3DLamark%26aufirst%3DT.%26aulast%3DJohansen%26aufirst%3DT.%26atitle%3DAutophagy%253A%2520links%2520with%2520the%2520proteasome%26jtitle%3DCurr.%2520Opin.%2520Cell%2520Biol.%26date%3D2010%26volume%3D22%26spage%3D192%26epage%3D198%26doi%3D10.1016%2Fj.ceb.2009.11.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Paul, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, A.</span></span> <span> </span><span class="NLM_article-title">TRAF6 coordinates the activation of autophagy and ubiquitin-proteasome systems in atrophying skeletal muscle</span>. <i>Autophagy</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">555</span>– <span class="NLM_lpage">556</span>, <span class="refDoi"> DOI: 10.4161/auto.7.5.15102</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.4161%2Fauto.7.5.15102" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=21412053" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC38XitF2itg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=555-556&author=P.+K.+Paulauthor=A.+Kumar&title=TRAF6+coordinates+the+activation+of+autophagy+and+ubiquitin-proteasome+systems+in+atrophying+skeletal+muscle&doi=10.4161%2Fauto.7.5.15102"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">TRAF6 coordinates the activation of autophagy and ubiquitin-proteasome systems in atrophying skeletal muscle</span></div><div class="casAuthors">Paul, Pradyut K.; Kumar, Ashok</div><div class="citationInfo"><span class="NLM_cas:title">Autophagy</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">555-556</span>CODEN:
                <span class="NLM_cas:coden">AUTOC9</span>;
        ISSN:<span class="NLM_cas:issn">1554-8627</span>.
    
            (<span class="NLM_cas:orgname">Landes Bioscience</span>)
        </div><div class="casAbstract">Skeletal muscle wasting is a major reason for morbidity and mortality in many chronic disease states, disuse conditions and aging.  The ubiquitin-proteasome and autophagy-lysosomal systems are the two major proteolytic pathways involved in regulation of both physiol. and pathol. muscle wasting.  Tumor necrosis factor receptor (TNFR)-assocd. factor 6 (TRAF6) is an important adaptor protein involved in receptor-mediated activation of various signaling pathways in response to cytokines and bacterial products.  TRAF6 also possesses E3 ubiquitin ligase activity causing lysine-63-linked polyubiquitination of target proteins.  We have uncovered a novel role of TRAF6 in regulation of skeletal muscle mass.  Muscle-wasting stimuli upregulate the expression, as well as the auto-ubiquitination, of TRAF6 leading to downstream activation of major catabolic pathways in skeletal muscle.  Muscle-specific depletion of TRAF6 preserves skeletal muscle mass in a mouse model of cancer cachexia or denervation.  Inhibition of TRAF6 also blocks the expression of the components of the ubiquitin-proteasome system (UPS) and autophagosome formation in atrophying skeletal muscle.  While more investigations are required to understand its mechanisms of action in skeletal muscle, our results indicate that blocking TRAF6 activity can be used as a therapeutic approach to preserve skeletal muscle mass and function in different disease states and conditions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqrAeK7825ZorVg90H21EOLACvtfcHk0lijtlD1w9461A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XitF2itg%253D%253D&md5=58f3bfc0cba102b1e10817f8a4cfaf41</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.4161%2Fauto.7.5.15102&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252Fauto.7.5.15102%26sid%3Dliteratum%253Aachs%26aulast%3DPaul%26aufirst%3DP.%2BK.%26aulast%3DKumar%26aufirst%3DA.%26atitle%3DTRAF6%2520coordinates%2520the%2520activation%2520of%2520autophagy%2520and%2520ubiquitin-proteasome%2520systems%2520in%2520atrophying%2520skeletal%2520muscle%26jtitle%3DAutophagy%26date%3D2011%26volume%3D7%26spage%3D555%26epage%3D556%26doi%3D10.4161%2Fauto.7.5.15102" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Veggiani, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerpe, M. C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sidhu, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span> <span> </span><span class="NLM_article-title">Emerging drug development technologies targeting ubiquitination for cancer therapeutics</span>. <i>Pharmacol. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>199</i></span>,  <span class="NLM_fpage">139</span>– <span class="NLM_lpage">154</span>, <span class="refDoi"> DOI: 10.1016/j.pharmthera.2019.03.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1016%2Fj.pharmthera.2019.03.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=30851297" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC1MXlvVCjtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=199&publication_year=2019&pages=139-154&author=G.+Veggianiauthor=M.+C.+R.+Gerpeauthor=S.+S.+Sidhuauthor=W.+Zhang&title=Emerging+drug+development+technologies+targeting+ubiquitination+for+cancer+therapeutics&doi=10.1016%2Fj.pharmthera.2019.03.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Emerging drug development technologies targeting ubiquitination for cancer therapeutics</span></div><div class="casAuthors">Veggiani, Gianluca; Gerpe, Maria Carla Rosales; Sidhu, Sachdev S.; Zhang, Wei</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacology & Therapeutics</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">199</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">139-154</span>CODEN:
                <span class="NLM_cas:coden">PHTHDT</span>;
        ISSN:<span class="NLM_cas:issn">0163-7258</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review.  Development of effective cancer therapeutic strategies relies on our ability to interfere with cellular processes that are dysregulated in tumors.  Given the essential role of the ubiquitin proteasome system (UPS) in regulating a myriad of cellular processes, it is not surprising that malfunction of UPS components is implicated in numerous human diseases, including many types of cancer.  The clin. success of proteasome inhibitors in treating multiple myeloma has further stimulated enthusiasm for targeting UPS proteins for pharmacol. intervention in cancer treatment, particularly in the precision medicine era.  Unfortunately, despite tremendous efforts, the paucity of potent and selective UPS inhibitors has severely hampered attempts to exploit the UPS for therapeutic benefits.  To tackle this problem, many groups have been working on technol. advancement to rapidly and effectively screen for potent and specific UPS modulators as intracellular probes or early-phase therapeutic agents.  Here, we review several emerging technologies for developing chem.- and protein-based mols. to manipulate UPS enzymic activity, with the aim of providing an overview of strategies available to target ubiquitination for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqnpUXmOrEplLVg90H21EOLACvtfcHk0lijtlD1w9461A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXlvVCjtr8%253D&md5=fcf119fca745b463e67cc841c402ccdd</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2Fj.pharmthera.2019.03.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pharmthera.2019.03.003%26sid%3Dliteratum%253Aachs%26aulast%3DVeggiani%26aufirst%3DG.%26aulast%3DGerpe%26aufirst%3DM.%2BC.%2BR.%26aulast%3DSidhu%26aufirst%3DS.%2BS.%26aulast%3DZhang%26aufirst%3DW.%26atitle%3DEmerging%2520drug%2520development%2520technologies%2520targeting%2520ubiquitination%2520for%2520cancer%2520therapeutics%26jtitle%3DPharmacol.%2520Ther.%26date%3D2019%26volume%3D199%26spage%3D139%26epage%3D154%26doi%3D10.1016%2Fj.pharmthera.2019.03.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cohen-Kaplan, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Livneh, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Avni, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cohen-Rosenzweig, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciechanover, A.</span></span> <span> </span><span class="NLM_article-title">The ubiquitin-proteasome system and autophagy: coordinated and independent activities</span>. <i>Int. J. Biochem. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>79</i></span>,  <span class="NLM_fpage">403</span>– <span class="NLM_lpage">418</span>, <span class="refDoi"> DOI: 10.1016/j.biocel.2016.07.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1016%2Fj.biocel.2016.07.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=27448843" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1eisrzF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=2016&pages=403-418&author=V.+Cohen-Kaplanauthor=I.+Livnehauthor=N.+Avniauthor=C.+Cohen-Rosenzweigauthor=A.+Ciechanover&title=The+ubiquitin-proteasome+system+and+autophagy%3A+coordinated+and+independent+activities&doi=10.1016%2Fj.biocel.2016.07.019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">The ubiquitin-proteasome system and autophagy: Coordinated and independent activities</span></div><div class="casAuthors">Cohen-Kaplan, Victoria; Livneh, Ido; Avni, Noa; Cohen-Rosenzweig, Chen; Ciechanover, Aaron</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Biochemistry & Cell Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">79</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">403-418</span>CODEN:
                <span class="NLM_cas:coden">IJBBFU</span>;
        ISSN:<span class="NLM_cas:issn">1357-2725</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The living cell is an ever changing, responsive, and adaptive environment where proteins play key roles in all processes and functions.  While the scientific community focused for a long time on the decoding of the information required for protein synthesis, little attention was paid to the mechanisms by which proteins are removed from the cell.  We now realize that the timely and proper activity of proteins is regulated to a large extent by their degrdn.; that cellular coping with different physiol. cues and stress conditions depends on different catabolic pathways; and that many pathol. states result from improper protein breakdown.  There are two major protein degrdn. systems in all eukaryotic cells-the ubiquitin- proteasome and the autophagy-lysosome.  The two systems are highly regulated, and-via degrdn. of a broad array of proteins-are responsible for maintenance of protein homeostasis and adaptation to environmental changes.  Each is comprised of numerous components responsible for its coordinated function, and together they encompass a considerable fraction of the entire genome.  In this review, we shall discuss the common and diverse characteristics of the ubiquitin-proteasome system (UPS) and autophagy-their substructure, mechanisms of action, function and concerted regulation under varying pathophysiol. conditions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqrcq6cLaANorVg90H21EOLACvtfcHk0ljUU3h6xpmkpg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1eisrzF&md5=5ec871a4cfa9e9e2ffc69beab08bfd6c</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2Fj.biocel.2016.07.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.biocel.2016.07.019%26sid%3Dliteratum%253Aachs%26aulast%3DCohen-Kaplan%26aufirst%3DV.%26aulast%3DLivneh%26aufirst%3DI.%26aulast%3DAvni%26aufirst%3DN.%26aulast%3DCohen-Rosenzweig%26aufirst%3DC.%26aulast%3DCiechanover%26aufirst%3DA.%26atitle%3DThe%2520ubiquitin-proteasome%2520system%2520and%2520autophagy%253A%2520coordinated%2520and%2520independent%2520activities%26jtitle%3DInt.%2520J.%2520Biochem.%2520Cell%2520Biol.%26date%3D2016%26volume%3D79%26spage%3D403%26epage%3D418%26doi%3D10.1016%2Fj.biocel.2016.07.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cho, Y. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, Y. M.</span></span> <span> </span><span class="NLM_article-title">Autophagy regulates homeostasis of pluripotency-associated proteins in hESCs</span>. <i>Stem Cells</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">424</span>– <span class="NLM_lpage">435</span>, <span class="refDoi"> DOI: 10.1002/stem.1589</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1002%2Fstem.1589" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=24170349" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC2cXlsFOrtLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2014&pages=424-435&author=Y.+H.+Choauthor=K.+M.+Hanauthor=D.+Kimauthor=J.+Leeauthor=S.+H.+Leeauthor=K.+W.+Choiauthor=J.+Kimauthor=Y.+M.+Han&title=Autophagy+regulates+homeostasis+of+pluripotency-associated+proteins+in+hESCs&doi=10.1002%2Fstem.1589"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Autophagy regulates homeostasis of pluripotency-associated proteins in hESCs</span></div><div class="casAuthors">Cho, Yun-Hee; Han, Kyu-Min; Kim, Dongkyu; Lee, Joonsun; Lee, Sang-Hee; Choi, Kyeng-Won; Kim, Jungho; Han, Yong-Mahn</div><div class="citationInfo"><span class="NLM_cas:title">Stem Cells (Durham, NC, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">424-435</span>CODEN:
                <span class="NLM_cas:coden">STCEEJ</span>;
        ISSN:<span class="NLM_cas:issn">1066-5099</span>.
    
            (<span class="NLM_cas:orgname">AlphaMed Press</span>)
        </div><div class="casAbstract">The pluripotency of embryonic stem cells (ESCs) is maintained by intracellular networks of many pluripotency-assocd. (PA) proteins such as OCT4, SOX2, and NANOG.  However, the mechanisms underlying the regulation of protein homeostasis for pluripotency remain elusive.  Here, we first demonstrate that autophagy acts together with the ubiquitin-proteasome system (UPS) to modulate the levels of PA proteins in human ESCs (hESCs).  Autophagy inhibition impaired the pluripotency despite increment of PA proteins in hESCs.  Immunogold-electron microscopy confirmed localization of OCT4 mols. within autophagosomes.  Also, knockdown of LC3 expression led to accumulation of PA proteins and redn. of pluripotency in hESCs.  Interestingly, autophagy and the UPS showed differential kinetics in the degrdn. of PA proteins.  Autophagy inhibition caused enhanced accumulation of both cytoplasmic and nuclear PA proteins, whereas the UPS inhibition led to preferentially degrade nuclear PA proteins.  Our findings suggest that autophagy modulates homeostasis of PA proteins, providing a new insight in the regulation of pluripotency in hESCs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJTsLmH6KAlLVg90H21EOLACvtfcHk0ljUU3h6xpmkpg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXlsFOrtLc%253D&md5=459ec78b2a2c988e67d6d902f123ffe6</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1002%2Fstem.1589&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fstem.1589%26sid%3Dliteratum%253Aachs%26aulast%3DCho%26aufirst%3DY.%2BH.%26aulast%3DHan%26aufirst%3DK.%2BM.%26aulast%3DKim%26aufirst%3DD.%26aulast%3DLee%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DS.%2BH.%26aulast%3DChoi%26aufirst%3DK.%2BW.%26aulast%3DKim%26aufirst%3DJ.%26aulast%3DHan%26aufirst%3DY.%2BM.%26atitle%3DAutophagy%2520regulates%2520homeostasis%2520of%2520pluripotency-associated%2520proteins%2520in%2520hESCs%26jtitle%3DStem%2520Cells%26date%3D2014%26volume%3D32%26spage%3D424%26epage%3D435%26doi%3D10.1002%2Fstem.1589" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ding, W. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ni, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshimori, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stolz, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ron, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, X. M.</span></span> <span> </span><span class="NLM_article-title">Linking of autophagy to ubiquitin-proteasome system is important for the regulation of endoplasmic reticulum stress and cell viability</span>. <i>Am. J. Pathol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>171</i></span>,  <span class="NLM_fpage">513</span>– <span class="NLM_lpage">524</span>, <span class="refDoi"> DOI: 10.2353/ajpath.2007.070188</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.2353%2Fajpath.2007.070188" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=17620365" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BD2sXpsFaiu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=171&publication_year=2007&pages=513-524&author=W.+X.+Dingauthor=H.+M.+Niauthor=W.+Gaoauthor=T.+Yoshimoriauthor=D.+B.+Stolzauthor=D.+Ronauthor=X.+M.+Yin&title=Linking+of+autophagy+to+ubiquitin-proteasome+system+is+important+for+the+regulation+of+endoplasmic+reticulum+stress+and+cell+viability&doi=10.2353%2Fajpath.2007.070188"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Linking of autophagy to ubiquitin-proteasome system is important for the regulation of endoplasmic reticulum stress and cell viability</span></div><div class="casAuthors">Ding, Wen-Xing; Ni, Hong-Min; Gao, Wentao; Yoshimori, Tamotsu; Stolz, Donna B.; Ron, David; Yin, Xiao-Ming</div><div class="citationInfo"><span class="NLM_cas:title">American Journal of Pathology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">171</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">513-524</span>CODEN:
                <span class="NLM_cas:coden">AJPAA4</span>;
        ISSN:<span class="NLM_cas:issn">0002-9440</span>.
    
            (<span class="NLM_cas:orgname">American Society for Investigative Pathology</span>)
        </div><div class="casAbstract">Two major protein degrdn. systems exist in cells, the ubiquitin proteasome system and the autophagy machinery.  Here, we investigated the functional relationship of the two systems and the underlying mechanisms.  Proteasome inhibition activated autophagy, suggesting that the two are functionally coupled.  Autophagy played a compensatory role as suppression of autophagy promoted the accumulation of polyubiquitinated protein aggregates.  Autophagy was likely activated in response to endoplasmic reticulum stress caused by misfolded proteins during proteasome inhibition.  Suppression of a major unfolded protein response pathway mediated by IRE1 by either gene deletion or RNA interference dramatically suppressed the activation of autophagy by proteasome inhibitors.  Interestingly, c-Jun NH2-terminal kinase (JNK) but not XBP-1, both of which are the known downstream targets of IRE1, seemed to participate in autophagy induction by proteasome inhibitors.  Finally, proteasome inhibitor-induced autophagy was important for controlling endoplasmic reticulum stress and reducing cell death in cancer cells.  Our studies thus provide a mechanistic view and elucidate the functional significance of the link between the two protein degrdn. systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrwdXb333fNrLVg90H21EOLACvtfcHk0ljUU3h6xpmkpg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXpsFaiu7w%253D&md5=00ac3988a17c61c7ac929baa5cf5f8f5</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.2353%2Fajpath.2007.070188&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2353%252Fajpath.2007.070188%26sid%3Dliteratum%253Aachs%26aulast%3DDing%26aufirst%3DW.%2BX.%26aulast%3DNi%26aufirst%3DH.%2BM.%26aulast%3DGao%26aufirst%3DW.%26aulast%3DYoshimori%26aufirst%3DT.%26aulast%3DStolz%26aufirst%3DD.%2BB.%26aulast%3DRon%26aufirst%3DD.%26aulast%3DYin%26aufirst%3DX.%2BM.%26atitle%3DLinking%2520of%2520autophagy%2520to%2520ubiquitin-proteasome%2520system%2520is%2520important%2520for%2520the%2520regulation%2520of%2520endoplasmic%2520reticulum%2520stress%2520and%2520cell%2520viability%26jtitle%3DAm.%2520J.%2520Pathol.%26date%3D2007%26volume%3D171%26spage%3D513%26epage%3D524%26doi%3D10.2353%2Fajpath.2007.070188" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pandey, U. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nie, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Batlevi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCray, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ritson, G. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nedelsky, N. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwartz, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DiProspero, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knight, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuldiner, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Padmanabhan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hild, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berry, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garza, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hubbert, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, T. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baehrecke, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, J. P.</span></span> <span> </span><span class="NLM_article-title">HDAC6 rescues neurodegeneration and provides an essential link between autophagy and the UPS</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>447</i></span>,  <span class="NLM_fpage">859</span>– <span class="NLM_lpage">863</span>, <span class="refDoi"> DOI: 10.1038/nature05853</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1038%2Fnature05853" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=17568747" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BD2sXms1WjsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=447&publication_year=2007&pages=859-863&author=U.+B.+Pandeyauthor=Z.+Nieauthor=Y.+Batleviauthor=B.+A.+McCrayauthor=G.+P.+Ritsonauthor=N.+B.+Nedelskyauthor=S.+L.+Schwartzauthor=N.+A.+DiProsperoauthor=M.+A.+Knightauthor=O.+Schuldinerauthor=R.+Padmanabhanauthor=M.+Hildauthor=D.+L.+Berryauthor=D.+Garzaauthor=C.+C.+Hubbertauthor=T.+P.+Yaoauthor=E.+H.+Baehreckeauthor=J.+P.+Taylor&title=HDAC6+rescues+neurodegeneration+and+provides+an+essential+link+between+autophagy+and+the+UPS&doi=10.1038%2Fnature05853"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">HDAC6 rescues neurodegeneration and provides an essential link between autophagy and the UPS</span></div><div class="casAuthors">Pandey, Udai Bhan; Nie, Zhiping; Batlevi, Yakup; McCray, Brett A.; Ritson, Gillian P.; Nedelsky, Natalia B.; Schwartz, Stephanie L.; DiProspero, Nicholas A.; Knight, Melanie A.; Schuldiner, Oren; Padmanabhan, Ranjani; Hild, Marc; Berry, Deborah L.; Garza, Dan; Hubbert, Charlotte C.; Yao, Tso-Pang; Baehrecke, Eric H.; Taylor, J. Paul</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">447</span>
        (<span class="NLM_cas:issue">7146</span>),
    <span class="NLM_cas:pages">859-863</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A prominent feature of late-onset neurodegenerative diseases is accumulation of misfolded protein in vulnerable neurons.  When levels of misfolded protein overwhelm degradative pathways, the result is cellular toxicity and neurodegeneration.  Cellular mechanisms for degrading misfolded protein include the ubiquitin-proteasome system (UPS), the main non-lysosomal degradative pathway for ubiquitinated proteins, and autophagy, a lysosome-mediated degradative pathway.  The UPS and autophagy have long been viewed as complementary degrdn. systems with no point of intersection.  This view was challenged by 2 observations suggesting an apparent interaction: impairment of the UPS induces autophagy in vitro, and conditional knockout of autophagy in the mouse brain leads to neurodegeneration with ubiquitin-pos. pathol.  It is not known whether autophagy is strictly a parallel degrdn. system, or whether it is a compensatory degrdn. system when the UPS is impaired; furthermore, if there is a compensatory interaction between these systems, the mol. link is not known.  Here the authors show that autophagy acts as a compensatory degrdn. system when the UPS is impaired in Drosophila melanogaster, and that histone deacetylase 6 (HDAC6), a microtubule-assocd. deacetylase that interacts with polyubiquitinated proteins, is an essential mechanistic link in this compensatory interaction.  The authors found that compensatory autophagy was induced in response to mutations affecting the proteasome and in response to UPS impairment in a fly model of the neurodegenerative disease spinobulbar muscular atrophy.  Autophagy compensated for impaired UPS function in an HDAC6-dependent manner.  Furthermore, expression of HDAC6 was sufficient to rescue degeneration assocd. with UPS dysfunction in vivo in an autophagy-dependent manner.  This study suggests that impairment of autophagy (for example, assocd. with aging or genetic variation) might predispose to neurodegeneration.  Morover, these findings suggest that it may be possible to intervene in neurodegeneration by augmenting HDAC6 to enhance autophagy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqncfliMTrJvbVg90H21EOLACvtfcHk0lj3FayJzXf6ZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXms1WjsL8%253D&md5=75925c1b12fc4bd012332f4a9b07ad19</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1038%2Fnature05853&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature05853%26sid%3Dliteratum%253Aachs%26aulast%3DPandey%26aufirst%3DU.%2BB.%26aulast%3DNie%26aufirst%3DZ.%26aulast%3DBatlevi%26aufirst%3DY.%26aulast%3DMcCray%26aufirst%3DB.%2BA.%26aulast%3DRitson%26aufirst%3DG.%2BP.%26aulast%3DNedelsky%26aufirst%3DN.%2BB.%26aulast%3DSchwartz%26aufirst%3DS.%2BL.%26aulast%3DDiProspero%26aufirst%3DN.%2BA.%26aulast%3DKnight%26aufirst%3DM.%2BA.%26aulast%3DSchuldiner%26aufirst%3DO.%26aulast%3DPadmanabhan%26aufirst%3DR.%26aulast%3DHild%26aufirst%3DM.%26aulast%3DBerry%26aufirst%3DD.%2BL.%26aulast%3DGarza%26aufirst%3DD.%26aulast%3DHubbert%26aufirst%3DC.%2BC.%26aulast%3DYao%26aufirst%3DT.%2BP.%26aulast%3DBaehrecke%26aufirst%3DE.%2BH.%26aulast%3DTaylor%26aufirst%3DJ.%2BP.%26atitle%3DHDAC6%2520rescues%2520neurodegeneration%2520and%2520provides%2520an%2520essential%2520link%2520between%2520autophagy%2520and%2520the%2520UPS%26jtitle%3DNature%26date%3D2007%26volume%3D447%26spage%3D859%26epage%3D863%26doi%3D10.1038%2Fnature05853" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">di
Ronza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bajaj, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanagasetti, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lotfi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adamski, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collette, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palmieri, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amawi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Popp, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, K. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meschini, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leung, H. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Segatori, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simonati, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sifers, R. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santorelli, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sardiello, M.</span></span> <span> </span><span class="NLM_article-title">CLN8 is an endoplasmic reticulum cargo receptor that regulates lysosome biogenesis</span>. <i>Nat. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">1370</span>– <span class="NLM_lpage">1377</span>, <span class="refDoi"> DOI: 10.1038/s41556-018-0228-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1038%2Fs41556-018-0228-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=30397314" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitV2qu7bJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2018&pages=1370-1377&author=A.+di%0ARonzaauthor=L.+Bajajauthor=J.+Sharmaauthor=D.+Sanagasettiauthor=P.+Lotfiauthor=C.+J.+Adamskiauthor=J.+Colletteauthor=M.+Palmieriauthor=A.+Amawiauthor=L.+Poppauthor=K.+T.+Changauthor=M.+C.+Meschiniauthor=H.+E.+Leungauthor=L.+Segatoriauthor=A.+Simonatiauthor=R.+N.+Sifersauthor=F.+M.+Santorelliauthor=M.+Sardiello&title=CLN8+is+an+endoplasmic+reticulum+cargo+receptor+that+regulates+lysosome+biogenesis&doi=10.1038%2Fs41556-018-0228-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">CLN8 is an endoplasmic reticulum cargo receptor that regulates lysosome biogenesis</span></div><div class="casAuthors">di Ronza, Alberto; Bajaj, Lakshya; Sharma, Jaiprakash; Sanagasetti, Deepthi; Lotfi, Parisa; Adamski, Carolyn Joy; Collette, John; Palmieri, Michela; Amawi, Abdallah; Popp, Lauren; Chang, Kevin Tommy; Meschini, Maria Chiara; Leung, Hon-Chiu Eastwood; Segatori, Laura; Simonati, Alessandro; Sifers, Richard Norman; Santorelli, Filippo Maria; Sardiello, Marco</div><div class="citationInfo"><span class="NLM_cas:title">Nature Cell Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1370-1377</span>CODEN:
                <span class="NLM_cas:coden">NCBIFN</span>;
        ISSN:<span class="NLM_cas:issn">1465-7392</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Organelle biogenesis requires proper transport of proteins from their site of synthesis to their target subcellular compartment.  Lysosomal enzymes are synthesized in the endoplasmic reticulum (ER) and traffic through the Golgi complex before being transferred to the endolysosomal system, but how they are transferred from the ER to the Golgi is unknown.  Here, we show that ER-to-Golgi transfer of lysosomal enzymes requires CLN8, an ER-assocd. membrane protein whose loss of function leads to the lysosomal storage disorder, neuronal ceroid lipofuscinosis 8 (a type of Batten disease).  ER-to-Golgi trafficking of CLN8 requires interaction with the COPII and COPI machineries via specific export and retrieval signals localized in the cytosolic carboxy terminus of CLN8.  CLN8 deficiency leads to depletion of sol. enzymes in the lysosome, thus impairing lysosome biogenesis.  Binding to lysosomal enzymes requires the second luminal loop of CLN8 and is abolished by some disease-causing mutations within this region.  Our data establish an unanticipated example of an ER receptor serving the biogenesis of an organelle and indicate that impaired transport of lysosomal enzymes underlies Batten disease caused by mutations in CLN8.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzFrRJaP5Hq7Vg90H21EOLACvtfcHk0lj3FayJzXf6ZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitV2qu7bJ&md5=321fcc86732777bce171e5594e0e02b2</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1038%2Fs41556-018-0228-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41556-018-0228-7%26sid%3Dliteratum%253Aachs%26aulast%3Ddi%2BRonza%26aufirst%3DA.%26aulast%3DBajaj%26aufirst%3DL.%26aulast%3DSharma%26aufirst%3DJ.%26aulast%3DSanagasetti%26aufirst%3DD.%26aulast%3DLotfi%26aufirst%3DP.%26aulast%3DAdamski%26aufirst%3DC.%2BJ.%26aulast%3DCollette%26aufirst%3DJ.%26aulast%3DPalmieri%26aufirst%3DM.%26aulast%3DAmawi%26aufirst%3DA.%26aulast%3DPopp%26aufirst%3DL.%26aulast%3DChang%26aufirst%3DK.%2BT.%26aulast%3DMeschini%26aufirst%3DM.%2BC.%26aulast%3DLeung%26aufirst%3DH.%2BE.%26aulast%3DSegatori%26aufirst%3DL.%26aulast%3DSimonati%26aufirst%3DA.%26aulast%3DSifers%26aufirst%3DR.%2BN.%26aulast%3DSantorelli%26aufirst%3DF.%2BM.%26aulast%3DSardiello%26aufirst%3DM.%26atitle%3DCLN8%2520is%2520an%2520endoplasmic%2520reticulum%2520cargo%2520receptor%2520that%2520regulates%2520lysosome%2520biogenesis%26jtitle%3DNat.%2520Cell%2520Biol.%26date%3D2018%26volume%3D20%26spage%3D1370%26epage%3D1377%26doi%3D10.1038%2Fs41556-018-0228-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Griffiths, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoflack, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simons, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mellman, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kornfeld, S.</span></span> <span> </span><span class="NLM_article-title">The mannose 6-phosphate receptor and the biogenesis of lysosomes</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">1988</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">329</span>– <span class="NLM_lpage">341</span>, <span class="refDoi"> DOI: 10.1016/S0092-8674(88)80026-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1016%2FS0092-8674%2888%2980026-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=2964276" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADyaL1cXhs1Ggtrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=1988&pages=329-341&author=G.+Griffithsauthor=B.+Hoflackauthor=K.+Simonsauthor=I.+Mellmanauthor=S.+Kornfeld&title=The+mannose+6-phosphate+receptor+and+the+biogenesis+of+lysosomes&doi=10.1016%2FS0092-8674%2888%2980026-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">The mannose 6-phosphate receptor and the biogenesis of lysosomes</span></div><div class="casAuthors">Griffiths, Gareth; Hoflack, Bernard; Simons, Kai; Mellman, Ira; Kornfeld, Stuart</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">1988</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">329-41</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    </div><div class="casAbstract">Localization of the 215-kilodalton mannose 6-phosphate receptor (MPR) was studied in normal rat kidney cells.  Low levels of receptor were detected in the trans Golgi network, Golgi stack, plasma membrane, and peripheral endosomes.  The bulk of the receptor was localized to an acidic, reticular-vesicular structure adjacent to the Golgi complex.  The structure also labeled with antibodies to lysosomal enzymes and a lysosomal membrane glycoprotein (Igp120).  Although lysosome-like, this structure is not a typical lysosome that is devoid of MPRs.  The endocytic marker α2-macroglobulin-Au entered the structure at 37°, but not at 20°.  With prolonged chase, most of the marker was transported from the structure into lysosomes.  It is proposed that the MPR/Ipg-enriched structure is a specialized endosome (prelysosome) that serves as an intermediate compartment into which endocytic vesicles discharge their contents, and where lysosomal enzymes are released from the MPR and packaged along with newly synthesized lysosomal glycoproteins into lysosomes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbMt6dilOlvbVg90H21EOLACvtfcHk0lj3FayJzXf6ZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1cXhs1Ggtrc%253D&md5=1e662f6d51a65732b2641a015ffff0ac</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1016%2FS0092-8674%2888%2980026-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0092-8674%252888%252980026-6%26sid%3Dliteratum%253Aachs%26aulast%3DGriffiths%26aufirst%3DG.%26aulast%3DHoflack%26aufirst%3DB.%26aulast%3DSimons%26aufirst%3DK.%26aulast%3DMellman%26aufirst%3DI.%26aulast%3DKornfeld%26aufirst%3DS.%26atitle%3DThe%2520mannose%25206-phosphate%2520receptor%2520and%2520the%2520biogenesis%2520of%2520lysosomes%26jtitle%3DCell%26date%3D1988%26volume%3D52%26spage%3D329%26epage%3D341%26doi%3D10.1016%2FS0092-8674%2888%2980026-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ni, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Canuel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morales, C. R.</span></span> <span> </span><span class="NLM_article-title">The sorting and trafficking of lysosomal proteins</span>. <i>Histol. Histopathol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">899</span>– <span class="NLM_lpage">913</span>, <span class="refDoi"> DOI: 10.14670/HH-21.899</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.14670%2FHH-21.899" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=16691542" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtF2gsLfE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2006&pages=899-913&author=X.+Niauthor=M.+Canuelauthor=C.+R.+Morales&title=The+sorting+and+trafficking+of+lysosomal+proteins&doi=10.14670%2FHH-21.899"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">The sorting and trafficking of lysosomal proteins</span></div><div class="casAuthors">Ni, X.; Canuel, M.; Morales, C. R.</div><div class="citationInfo"><span class="NLM_cas:title">Histology and Histopathology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">7,8 and 9</span>),
    <span class="NLM_cas:pages">899-913</span>CODEN:
                <span class="NLM_cas:coden">HIHIES</span>;
        ISSN:<span class="NLM_cas:issn">0213-3911</span>.
    
            (<span class="NLM_cas:orgname">Jimenez Godoy, S.A.</span>)
        </div><div class="casAbstract">A review.  For a long time lysosomes were considered terminal organelles involved in the degrdn. of different substrates.  However, this view is rapidly changing by evidence demonstrating that these organelles and their content display specialized functions in addn. to the degrdn. of substances.  Many lysosomal proteins have been implicated in specialized cellular functions and disorders such as antigen processing, targeting of surfactant proteins, and most lysosomal storage disorders.  To date, about 50 lysosomal hydrolases have been identified, and the majority of them are targeted to the lysosomes via the mannose 6-phosphate receptor (M6P-Rc).  However, recent studies on the intracellular trafficking of the nonenzymic lysosomal proteins, prosaposin and GM2 activator (GM2AP), demonstrated that they use an alternative receptor termed "sortilin".  Existing evidence suggests that some hydrolases traffic to the lysosomes in a mannose 6-phosphate-independent manner.  The possibility that sortilin is implicated in the targeting of some sol. hydrolases, as well as the consequences of this process, is addressed in the present review.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoyRGG79S6lg7Vg90H21EOLACvtfcHk0lhFx26wusb1kg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtF2gsLfE&md5=537893c9550c41eeb77235fbfcc9165b</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.14670%2FHH-21.899&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.14670%252FHH-21.899%26sid%3Dliteratum%253Aachs%26aulast%3DNi%26aufirst%3DX.%26aulast%3DCanuel%26aufirst%3DM.%26aulast%3DMorales%26aufirst%3DC.%2BR.%26atitle%3DThe%2520sorting%2520and%2520trafficking%2520of%2520lysosomal%2520proteins%26jtitle%3DHistol.%2520Histopathol.%26date%3D2006%26volume%3D21%26spage%3D899%26epage%3D913%26doi%3D10.14670%2FHH-21.899" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Braulke, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonifacino, J. S.</span></span> <span> </span><span class="NLM_article-title">Sorting of lysosomal proteins</span>. <i>Biochim. Biophys. Acta, Mol. Cell Res.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>1793</i></span>,  <span class="NLM_fpage">605</span>– <span class="NLM_lpage">614</span>, <span class="refDoi"> DOI: 10.1016/j.bbamcr.2008.10.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1016%2Fj.bbamcr.2008.10.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=19046998" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BD1MXjslOlsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1793&publication_year=2009&pages=605-614&author=T.+Braulkeauthor=J.+S.+Bonifacino&title=Sorting+of+lysosomal+proteins&doi=10.1016%2Fj.bbamcr.2008.10.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Sorting of lysosomal proteins</span></div><div class="casAuthors">Braulke, Thomas; Bonifacino, Juan S.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Molecular Cell Research</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">1793</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">605-614</span>CODEN:
                <span class="NLM_cas:coden">BBAMCO</span>;
        ISSN:<span class="NLM_cas:issn">0167-4889</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Lysosomes are composed of sol. and transmembrane proteins that are targeted to lysosomes in a signal-dependent manner.  The majority of sol. acid hydrolases are modified with mannose 6-phosphate (M6P) residues, allowing their recognition by M6P receptors in the Golgi complex and ensuing transport to the endosomal/lysosomal system.  Other sol. enzymes and nonenzymic proteins are transported to lysosomes in an M6P-independent manner mediated by alternative receptors such as lysosomal integral membrane protein LIMP-2 or sortilin.  The sorting of cargo receptors and lysosomal transmembrane proteins requires sorting signals present in their cytosolic domains.  These signals include di-leucine-based motifs, DXXLL or [DE]XXXL[LI], and tyrosine-based motifs, YXXO, which interact with components of clathrin coats such as GGAs or adaptor protein complexes.  In addn., phosphorylation and lipid modifications regulate signal recognition and trafficking of lysosomal membrane proteins.  The complex interaction of both luminal and cytosolic signals with recognition proteins guarantees the specific and directed transport of proteins to lysosomes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpGvqmTon5cC7Vg90H21EOLACvtfcHk0lhFx26wusb1kg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXjslOlsb4%253D&md5=ffcb5e646608aec952b6d6cc9e3c6837</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1016%2Fj.bbamcr.2008.10.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbamcr.2008.10.016%26sid%3Dliteratum%253Aachs%26aulast%3DBraulke%26aufirst%3DT.%26aulast%3DBonifacino%26aufirst%3DJ.%2BS.%26atitle%3DSorting%2520of%2520lysosomal%2520proteins%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Mol.%2520Cell%2520Res.%26date%3D2009%26volume%3D1793%26spage%3D605%26epage%3D614%26doi%3D10.1016%2Fj.bbamcr.2008.10.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Coutinho, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prata, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alves, S.</span></span> <span> </span><span class="NLM_article-title">A shortcut to the lysosome: the mannose-6-phosphate-independent pathway</span>. <i>Mol. Genet. Metab.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>107</i></span>,  <span class="NLM_fpage">257</span>– <span class="NLM_lpage">266</span>, <span class="refDoi"> DOI: 10.1016/j.ymgme.2012.07.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1016%2Fj.ymgme.2012.07.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=22884962" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtF2it7jL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=107&publication_year=2012&pages=257-266&author=M.+F.+Coutinhoauthor=M.+J.+Prataauthor=S.+Alves&title=A+shortcut+to+the+lysosome%3A+the+mannose-6-phosphate-independent+pathway&doi=10.1016%2Fj.ymgme.2012.07.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">A shortcut to the lysosome: The mannose-6-phosphate-independent pathway</span></div><div class="casAuthors">Coutinho, Maria Francisca; Prata, Maria Joao; Alves, Sandra</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Genetics and Metabolism</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">107</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">257-266</span>CODEN:
                <span class="NLM_cas:coden">MGMEFF</span>;
        ISSN:<span class="NLM_cas:issn">1096-7192</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Lysosomal hydrolases have long been known to be responsible for the degrdn. of different substrates in the cell.  These acid hydrolases are synthesized in the rough endoplasmic reticulum and transported through the Golgi app. to the trans-Golgi network (TGN).  From there, they are delivered to endosomal/lysosomal compartments, where they finally become active due to the acidic pH characteristic of the lysosomal compartment.  The majority of the enzymes leave the TGN after modification with mannose-6-phosphate (M6P) residues, which are specifically recognized by M6P receptors (MPRs), ensuring their transport to the endosomal/lysosomal system.  Although M6P receptors play a major role in the intracellular transport of newly synthesized lysosomal enzymes in mammalian cells, several lines of evidence suggest the existence of alternative processes of lysosomal targeting.  Among them, the 2 that are mediated by the M6P alternative receptors, lysosomal integral membrane protein (LIMP-2) and sortilin, have gained unequivocal support.  LIMP-2 was shown to be implicated in the delivery of beta-glucocerebrosidase (GCase) to the lysosomes, whereas sortilin was suggested to be a multifunctional receptor capable of binding several different ligands, including neurotensin and receptor-assocd. protein (RAP), and of targeting several proteins to the lysosome, including sphingolipid activator proteins (prosaposin and GM2 activator protein), acid sphingomyelinase and cathepsins D and H.  Here, the authors review the current knowledge on these 2 proteins: their discovery, study, structural features and cellular function, with special attention to their role as alternative receptors to lysosomal trafficking.  Recent studies assocg. both LIMP2 and sortilin to disease are also extensively reviewed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqcyx4mje1htrVg90H21EOLACvtfcHk0lhFx26wusb1kg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtF2it7jL&md5=690f6719cd28e370bf09a74cdf2ffdc6</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1016%2Fj.ymgme.2012.07.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ymgme.2012.07.012%26sid%3Dliteratum%253Aachs%26aulast%3DCoutinho%26aufirst%3DM.%2BF.%26aulast%3DPrata%26aufirst%3DM.%2BJ.%26aulast%3DAlves%26aufirst%3DS.%26atitle%3DA%2520shortcut%2520to%2520the%2520lysosome%253A%2520the%2520mannose-6-phosphate-independent%2520pathway%26jtitle%3DMol.%2520Genet.%2520Metab.%26date%3D2012%26volume%3D107%26spage%3D257%26epage%3D266%26doi%3D10.1016%2Fj.ymgme.2012.07.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Malini, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zampieri, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deganuto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romanello, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sechi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bembi, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dardis, A.</span></span> <span> </span><span class="NLM_article-title">Role of LIMP-2 in the intracellular trafficking of β-glucosidase in different human cellular models</span>. <i>FASEB J.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">3839</span>– <span class="NLM_lpage">3852</span>, <span class="refDoi"> DOI: 10.1096/fj.15-271148</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1096%2Ffj.15-271148" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=26018676" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC28XnsFegsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2015&pages=3839-3852&author=E.+Maliniauthor=S.+Zampieriauthor=M.+Deganutoauthor=M.+Romanelloauthor=A.+Sechiauthor=B.+Bembiauthor=A.+Dardis&title=Role+of+LIMP-2+in+the+intracellular+trafficking+of+%CE%B2-glucosidase+in+different+human+cellular+models&doi=10.1096%2Ffj.15-271148"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Role of LIMP-2 in the intracellular trafficking of β-glucosidase in different human cellular models</span></div><div class="casAuthors">Malini, Erika; Zampieri, Stefania; Deganuto, Marta; Romanello, Milena; Sechi, Annalisa; Bembi, Bruno; Dardis, Andrea</div><div class="citationInfo"><span class="NLM_cas:title">FASEB Journal</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3839-3852</span>CODEN:
                <span class="NLM_cas:coden">FAJOEC</span>;
        ISSN:<span class="NLM_cas:issn">0892-6638</span>.
    
            (<span class="NLM_cas:orgname">Federation of American Societies for Experimental Biology</span>)
        </div><div class="casAbstract">Acid β-glucosidase (GCase), the enzyme deficient in Gaucher disease (GD), is transported to lysosomes by the lysosomal integral membrane protein (LIMP)-2.  In humans, LIMP-2 deficiency leads to action myoclonus-renal failure (AMRF) syndrome.  GD and AMRF syndrome share some clin. features.  However, they are different from clin. and biochem. points of view, suggesting that the role of LIMP-2 in the targeting of GCase would be different in different tissues.  Besides, the role of LIMP-2 in the uptake and trafficking of the human recombinant (hr)GCase used in the treatment of GD is unknown.  Thus, we compared GCase activity and intracellular localization in immortalized lymphocytes, fibroblasts, and a neuronal model derived from multipotent adult stem cells, from a patient with AMRF syndrome, patients with GD, and control subjects.  In fibroblasts and neuronlike cells, GCase targeting to the lysosomes is completely dependent on LIMP-2, whereas in blood cells, GCase is partially targeted to lysosomes by a LIMP-2-independent mechanism.  Although hrGCase cellular uptake is independent of LIMP-2, its trafficking to the lysosomes is mediated by this receptor.  These data provide new insights into the mechanisms involved in the intracellular trafficking of GCase and in the pathogeneses of GD and AMRF syndrome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpk_1m_RE750LVg90H21EOLACvtfcHk0lhFx26wusb1kg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XnsFegsLc%253D&md5=402baba943ebc321d779633afab66dd8</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1096%2Ffj.15-271148&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1096%252Ffj.15-271148%26sid%3Dliteratum%253Aachs%26aulast%3DMalini%26aufirst%3DE.%26aulast%3DZampieri%26aufirst%3DS.%26aulast%3DDeganuto%26aufirst%3DM.%26aulast%3DRomanello%26aufirst%3DM.%26aulast%3DSechi%26aufirst%3DA.%26aulast%3DBembi%26aufirst%3DB.%26aulast%3DDardis%26aufirst%3DA.%26atitle%3DRole%2520of%2520LIMP-2%2520in%2520the%2520intracellular%2520trafficking%2520of%2520%25CE%25B2-glucosidase%2520in%2520different%2520human%2520cellular%2520models%26jtitle%3DFASEB%2520J.%26date%3D2015%26volume%3D29%26spage%3D3839%26epage%3D3852%26doi%3D10.1096%2Ffj.15-271148" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ouyang, L.</span></span> <span> </span><span class="NLM_article-title">Targeting autophagy-related protein kinases for potential therapeutic purpose</span>. <i>Acta Pharm. Sin. B</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">569</span>– <span class="NLM_lpage">581</span>, <span class="refDoi"> DOI: 10.1016/j.apsb.2019.10.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1016%2Fj.apsb.2019.10.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=32322463" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BB3MXnslWjug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2020&pages=569-581&author=H.+Xiangauthor=J.+Zhangauthor=C.+Linauthor=L.+Zhangauthor=B.+Liuauthor=L.+Ouyang&title=Targeting+autophagy-related+protein+kinases+for+potential+therapeutic+purpose&doi=10.1016%2Fj.apsb.2019.10.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting autophagy-related protein kinases for potential therapeutic purpose</span></div><div class="casAuthors">Xiang, Honggang; Zhang, Jifa; Lin, Congcong; Zhang, Lan; Liu, Bo; Ouyang, Liang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Pharmaceutica Sinica B</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">569-581</span>CODEN:
                <span class="NLM_cas:coden">APSBCW</span>;
        ISSN:<span class="NLM_cas:issn">2211-3835</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Autophagy, defined as a scavenging process of protein aggregates and damaged organelles mediated by lysosomes, plays a significant role in the quality control of macromols. and organelles.  Since protein kinases are integral to the autophagy process, it is critically important to understand the role of kinases in autophagic regulation.  At present, intervention of autophagic processes by small-mol. modulators targeting specific kinases has becoming a reasonable and prevalent strategy for treating several varieties of human disease, esp. cancer.  In this review, we describe the role of some autophagy-related kinase targets and kinase-mediated phosphorylation mechanisms in autophagy regulation.  We also summarize the small-mol. kinase inhibitors/activators of these targets, high-lighting the opportunities of these new therapeutic agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUZ5eGWVZ5RbVg90H21EOLACvtfcHk0ljFsmphmZb_bA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXnslWjug%253D%253D&md5=277da77a9145fb63c521c5babf333e9b</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1016%2Fj.apsb.2019.10.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.apsb.2019.10.003%26sid%3Dliteratum%253Aachs%26aulast%3DXiang%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DLin%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DLiu%26aufirst%3DB.%26aulast%3DOuyang%26aufirst%3DL.%26atitle%3DTargeting%2520autophagy-related%2520protein%2520kinases%2520for%2520potential%2520therapeutic%2520purpose%26jtitle%3DActa%2520Pharm.%2520Sin.%2520B%26date%3D2020%26volume%3D10%26spage%3D569%26epage%3D581%26doi%3D10.1016%2Fj.apsb.2019.10.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ouyang, L.</span></span> <span> </span><span class="NLM_article-title">Discovery of a novel dual-Target inhibitor of ERK1 and ERK5 that induces regulated cell death to overcome compensatory mechanism in specific tumor types</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">3976</span>– <span class="NLM_lpage">3995</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b01896</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01896" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BB3cXjsFOku7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=3976-3995&author=G.+Wangauthor=Y.+Zhaoauthor=Y.+Liuauthor=D.+Sunauthor=Y.+Zhenauthor=J.+Liuauthor=L.+Fuauthor=L.+Zhangauthor=L.+Ouyang&title=Discovery+of+a+novel+dual-Target+inhibitor+of+ERK1+and+ERK5+that+induces+regulated+cell+death+to+overcome+compensatory+mechanism+in+specific+tumor+types&doi=10.1021%2Facs.jmedchem.9b01896"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Novel Dual-Target Inhibitor of ERK1 and ERK5 That Induces Regulated Cell Death to Overcome Compensatory Mechanism in Specific Tumor Types</span></div><div class="casAuthors">Wang, Guan; Zhao, Yuqian; Liu, Yao; Sun, Dejuan; Zhen, Yongqi; Liu, Jie; Fu, Leilei; Zhang, Lan; Ouyang, Liang</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3976-3995</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">ERK1 and ERK5 are proposed to have pivotal roles in several types of cancer.  Under some circumstance, ERK5 may provide a common bypass route, which rescues proliferation upon abrogation of ERK1 signaling.  Thus, we accurately classified the tumor types from The Cancer Genome Atlas (TCGA) based on the expression levels of ERK1 and ERK5.  We proposed a novel therapeutic strategy to overcome the above-mentioned compensatory mechanism in specific tumor types by co-targeting both ERK1 and ERK5.  On the basis of the idea of overcoming ERK5 compensation mechanism, 22ac (ADTL-EI1712) as the first selective dual-target inhibitor of ERK1 and ERK5 was discovered to have potent antitumor effects in vitro and in vivo.  Interestingly, this compd. was found to induce regulated cell death accompanied by autophagy in MKN-74 cells.  Taken together, our results warrant the potential of this dual-target inhibitor as a new candidate drug that conquers compensatory mechanism in certain tumor types.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlxVBmkLH7mrVg90H21EOLACvtfcHk0ljFsmphmZb_bA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXjsFOku7c%253D&md5=a39e2b19b4e97ba739e78f2453335714</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01896&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01896%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DG.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DSun%26aufirst%3DD.%26aulast%3DZhen%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DFu%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DOuyang%26aufirst%3DL.%26atitle%3DDiscovery%2520of%2520a%2520novel%2520dual-Target%2520inhibitor%2520of%2520ERK1%2520and%2520ERK5%2520that%2520induces%2520regulated%2520cell%2520death%2520to%2520overcome%2520compensatory%2520mechanism%2520in%2520specific%2520tumor%2520types%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D3976%26epage%3D3995%26doi%3D10.1021%2Facs.jmedchem.9b01896" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Doherty, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McMahon, H. T.</span></span> <span> </span><span class="NLM_article-title">Mechanisms of endocytosis</span>. <i>Annu. Rev. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>78</i></span>,  <span class="NLM_fpage">857</span>– <span class="NLM_lpage">902</span>, <span class="refDoi"> DOI: 10.1146/annurev.biochem.78.081307.110540</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1146%2Fannurev.biochem.78.081307.110540" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=19317650" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BD1MXos1Ghu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=78&publication_year=2009&pages=857-902&author=G.+J.+Dohertyauthor=H.+T.+McMahon&title=Mechanisms+of+endocytosis&doi=10.1146%2Fannurev.biochem.78.081307.110540"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanisms of endocytosis</span></div><div class="casAuthors">Doherty, Gary J.; McMahon, Harvey T.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Biochemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">78</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">857-902</span>CODEN:
                <span class="NLM_cas:coden">ARBOAW</span>;
        ISSN:<span class="NLM_cas:issn">0066-4154</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review.  Endocytic mechanisms control the lipid and protein compn. of the plasma membrane, thereby regulating how cells interact with their environments.  Here, the authors review what is known about mammalian endocytic mechanisms, with a focus on the cellular proteins that control these events.  The authors discuss the well-studied clathrin-mediated endocytic mechanisms and dissect endocytic pathways that proceed independently of clathrin.  These clathrin-independent pathways include the CLIC/GEEC endocytic pathway, arf6-dependent endocytosis, flotillin-dependent endocytosis, macropinocytosis, circular doral ruffles, phagocytosis, and trans-endocytosis.  The authors also critically review the role of caveolae and caveolin 1 in endocytosis.  The authors highlight the roles of lipids, membrane curvature-modulating proteins, small G proteins, actin, and dynamin in endocytic pathways.  The authors discuss the functional relevance of distinct endocytic pathways and emphasize the importance of studying these pathways to understanding human disease processes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphxXOCrv5TH7Vg90H21EOLACvtfcHk0lgeRATkRhkkhg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXos1Ghu7s%253D&md5=da59d8e8c3c629f548b91e0899693a4f</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1146%2Fannurev.biochem.78.081307.110540&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.biochem.78.081307.110540%26sid%3Dliteratum%253Aachs%26aulast%3DDoherty%26aufirst%3DG.%2BJ.%26aulast%3DMcMahon%26aufirst%3DH.%2BT.%26atitle%3DMechanisms%2520of%2520endocytosis%26jtitle%3DAnnu.%2520Rev.%2520Biochem.%26date%3D2009%26volume%3D78%26spage%3D857%26epage%3D902%26doi%3D10.1146%2Fannurev.biochem.78.081307.110540" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mayor, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pagano, R. E.</span></span> <span> </span><span class="NLM_article-title">Pathways of clathrin-independent endocytosis</span>. <i>Nat. Rev. Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">603</span>– <span class="NLM_lpage">612</span>, <span class="refDoi"> DOI: 10.1038/nrm2216</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1038%2Fnrm2216" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=17609668" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BD2sXotVahtrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2007&pages=603-612&author=S.+Mayorauthor=R.+E.+Pagano&title=Pathways+of+clathrin-independent+endocytosis&doi=10.1038%2Fnrm2216"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Pathways of clathrin-independent endocytosis</span></div><div class="casAuthors">Mayor, Satyajit; Pagano, Richard E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">603-612</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  There are numerous ways that endocytic cargo mols. may be internalized from the surface of eukaryotic cells.  In addn. to the classical clathrin-dependent mechanism of endocytosis, several pathways that do not use a clathrin coat are emerging.  These pathways transport a diverse array of cargoes and are sometimes hijacked by bacteria and viruses to gain access to the host cell.  Here, the authors review the current understanding of various clathrin-independent mechanisms of endocytosis and propose a classification scheme to help organize the data in this complex and evolving field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqwyq4_P7rbMrVg90H21EOLACvtfcHk0lgeRATkRhkkhg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXotVahtrg%253D&md5=94d628b1b0f415086bcab5113bfdd801</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1038%2Fnrm2216&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm2216%26sid%3Dliteratum%253Aachs%26aulast%3DMayor%26aufirst%3DS.%26aulast%3DPagano%26aufirst%3DR.%2BE.%26atitle%3DPathways%2520of%2520clathrin-independent%2520endocytosis%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2007%26volume%3D8%26spage%3D603%26epage%3D612%26doi%3D10.1038%2Fnrm2216" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klionsky, D. J.</span></span> <span> </span><span class="NLM_article-title">Eaten alive: a history of macroautophagy</span>. <i>Nat. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">814</span>– <span class="NLM_lpage">822</span>, <span class="refDoi"> DOI: 10.1038/ncb0910-814</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1038%2Fncb0910-814" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=20811353" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtFSju7zE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2010&pages=814-822&author=Z.+Yangauthor=D.+J.+Klionsky&title=Eaten+alive%3A+a+history+of+macroautophagy&doi=10.1038%2Fncb0910-814"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Eaten alive: a history of macroautophagy</span></div><div class="casAuthors">Yang, Zhifen; Klionsky, Daniel J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Cell Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">814-822</span>CODEN:
                <span class="NLM_cas:coden">NCBIFN</span>;
        ISSN:<span class="NLM_cas:issn">1465-7392</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Macroautophagy (hereafter autophagy), or 'self-eating', is a conserved cellular pathway that controls protein and organelle degrdn., and has essential roles in survival, development and homeostasis.  Autophagy is also integral to human health and is involved in physiol., development, lifespan and a wide range of diseases, including cancer, neurodegeneration and microbial infection.  Although research on this topic began in the late 1950s, substantial progress in the mol. study of autophagy has taken place during only the past 15 years.  This review traces the key findings that led to our current mol. understanding of this complex process.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJ_iZJ3YUoHrVg90H21EOLACvtfcHk0lgeRATkRhkkhg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtFSju7zE&md5=a20abac7db3f20eac668c41052413397</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1038%2Fncb0910-814&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncb0910-814%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DZ.%26aulast%3DKlionsky%26aufirst%3DD.%2BJ.%26atitle%3DEaten%2520alive%253A%2520a%2520history%2520of%2520macroautophagy%26jtitle%3DNat.%2520Cell%2520Biol.%26date%3D2010%26volume%3D12%26spage%3D814%26epage%3D822%26doi%3D10.1038%2Fncb0910-814" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Komatsu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ichimura, Y.</span></span> <span> </span><span class="NLM_article-title">Selective autophagy regulates various cellular functions</span>. <i>Genes Cells.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">923</span>– <span class="NLM_lpage">933</span>, <span class="refDoi"> DOI: 10.1111/j.1365-2443.2010.01433.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1111%2Fj.1365-2443.2010.01433.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=20670274" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtFeqsrvF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2010&pages=923-933&author=M.+Komatsuauthor=Y.+Ichimura&title=Selective+autophagy+regulates+various+cellular+functions&doi=10.1111%2Fj.1365-2443.2010.01433.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Selective autophagy regulates various cellular functions</span></div><div class="casAuthors">Komatsu, Masaaki; Ichimura, Yoshinobu</div><div class="citationInfo"><span class="NLM_cas:title">Genes to Cells</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">923-933</span>CODEN:
                <span class="NLM_cas:coden">GECEFL</span>;
        ISSN:<span class="NLM_cas:issn">1356-9597</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  Autophagy is a self-eating system conserved among eukaryotes, in which cellular components including organelles are entrapped into a double membrane structure called the autophagosome and then degraded by lysosomal hydrolases.  In addn. to its role in supplying amino acids in response to nutrient starvation, autophagy is involved in quality control to maintain cell health.  Thus, inactivation of autophagy causes the formation of cytoplasmic protein inclusions, which comprise misfolded proteins and the accumulation of many degenerated organelles, resulting in liver injury, diabetes, myopathy and neurodegeneration.  Furthermore, although autophagy has been considered nonselective, increasing evidence points to the selectivity of autophagy in sorting vacuolar enzymes and removal of aggregate-prone proteins and unwanted organelles.  Such selectivity allows diverse cellular regulation, similar to the ubiquitin proteasome pathway.  In this review, we discuss the physiol. roles of selective autophagy and their mol. mechanisms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq1HKtcqB-IHLVg90H21EOLACvtfcHk0lgeRATkRhkkhg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtFeqsrvF&md5=5ff745e51923a3ca69aa792fcfc62cc2</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1111%2Fj.1365-2443.2010.01433.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1365-2443.2010.01433.x%26sid%3Dliteratum%253Aachs%26aulast%3DKomatsu%26aufirst%3DM.%26aulast%3DIchimura%26aufirst%3DY.%26atitle%3DSelective%2520autophagy%2520regulates%2520various%2520cellular%2520functions%26jtitle%3DGenes%2520Cells.%26date%3D2010%26volume%3D15%26spage%3D923%26epage%3D933%26doi%3D10.1111%2Fj.1365-2443.2010.01433.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lamark, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Svenning, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johansen, T.</span></span> <span> </span><span class="NLM_article-title">Regulation of selective autophagy: the p62/SQSTM1 paradigm</span>. <i>Essays Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">609</span>– <span class="NLM_lpage">624</span>, <span class="refDoi"> DOI: 10.1042/EBC20170035</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1042%2FEBC20170035" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=29233872" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A280%3ADC%252BC1MzgvVSktQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2017&pages=609-624&author=T.+Lamarkauthor=S.+Svenningauthor=T.+Johansen&title=Regulation+of+selective+autophagy%3A+the+p62%2FSQSTM1+paradigm&doi=10.1042%2FEBC20170035"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of selective autophagy: the p62/SQSTM1 paradigm</span></div><div class="casAuthors">Lamark Trond; Svenning Steingrim; Johansen Terje</div><div class="citationInfo"><span class="NLM_cas:title">Essays in biochemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">609-624</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">In selective autophagy, cytoplasmic components are selected and tagged before being sequestered into an autophagosome by means of selective autophagy receptors such as p62/SQSTM1.  In this review, we discuss how selective autophagy is regulated.  An important level of regulation is the selection of proteins or organelles for degradation.  Components selected for degradation are tagged, often with ubiquitin, to facilitate recognition by autophagy receptors.  Another level of regulation is represented by the autophagy receptors themselves.  For p62, its ability to co-aggregate with ubiquitinated substrates is strongly induced by post-translational modifications (PTMs).  The transcription of p62 is also markedly increased during conditions in which selective autophagy substrates accumulate.  For other autophagy receptors, the LC3-interacting region (LIR) motif is regulated by PTMs, inhibiting or stimulating the interaction with ATG8 family proteins.  ATG8 proteins are also regulated by PTMs.  Regulation of the capacity of the core autophagy machinery also affects selective autophagy.  Importantly, autophagy receptors can induce local recruitment and activation of ULK1/2 and PI3KC3 complexes at the site of cargo sequestration.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSothEGF92qRPxQvRxYU81tfW6udTcc2ebM6iS2V4EKzbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1MzgvVSktQ%253D%253D&md5=024cf98b5e452cb2dde8c4516cffcf7a</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1042%2FEBC20170035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FEBC20170035%26sid%3Dliteratum%253Aachs%26aulast%3DLamark%26aufirst%3DT.%26aulast%3DSvenning%26aufirst%3DS.%26aulast%3DJohansen%26aufirst%3DT.%26atitle%3DRegulation%2520of%2520selective%2520autophagy%253A%2520the%2520p62%252FSQSTM1%2520paradigm%26jtitle%3DEssays%2520Biochem.%26date%3D2017%26volume%3D61%26spage%3D609%26epage%3D624%26doi%3D10.1042%2FEBC20170035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stolz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ernst, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dikic, I.</span></span> <span> </span><span class="NLM_article-title">Cargo recognition and trafficking in selective autophagy</span>. <i>Nat. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">495</span>– <span class="NLM_lpage">501</span>, <span class="refDoi"> DOI: 10.1038/ncb2979</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1038%2Fncb2979" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=24875736" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC2cXovVaktrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2014&pages=495-501&author=A.+Stolzauthor=A.+Ernstauthor=I.+Dikic&title=Cargo+recognition+and+trafficking+in+selective+autophagy&doi=10.1038%2Fncb2979"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Cargo recognition and trafficking in selective autophagy</span></div><div class="casAuthors">Stolz, Alexandra; Ernst, Andreas; Dikic, Ivan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Cell Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">495-501</span>CODEN:
                <span class="NLM_cas:coden">NCBIFN</span>;
        ISSN:<span class="NLM_cas:issn">1465-7392</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Selective autophagy is a quality control pathway through which cellular components are sequestered into double-membrane vesicles and delivered to specific intracellular compartments.  This process requires autophagy receptors that link cargo to growing autophagosomal membranes.  Selective autophagy is also implicated in various membrane trafficking events.  Here we discuss the current view on how cargo selection and transport are achieved during selective autophagy, and point out mol. mechanisms that are congruent between autophagy and vesicle trafficking pathways.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoP4vnK9jGLP7Vg90H21EOLACvtfcHk0lguARtIW_resw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXovVaktrg%253D&md5=3f58139fdf955faabff12cbe94ad865c</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1038%2Fncb2979&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncb2979%26sid%3Dliteratum%253Aachs%26aulast%3DStolz%26aufirst%3DA.%26aulast%3DErnst%26aufirst%3DA.%26aulast%3DDikic%26aufirst%3DI.%26atitle%3DCargo%2520recognition%2520and%2520trafficking%2520in%2520selective%2520autophagy%26jtitle%3DNat.%2520Cell%2520Biol.%26date%3D2014%26volume%3D16%26spage%3D495%26epage%3D501%26doi%3D10.1038%2Fncb2979" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bonam, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruff, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, S.</span></span> <span> </span><span class="NLM_article-title">HSPA8/HSC70 in immune disorders: a molecular rheostat that adjusts chaperone-mediated autophagy substrates</span>. <i>Cells</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">849</span>, <span class="refDoi"> DOI: 10.3390/cells8080849</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.3390%2Fcells8080849" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BB3cXlsVGjsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2019&pages=849&author=S.+R.+Bonamauthor=M.+Ruffauthor=S.+Muller&title=HSPA8%2FHSC70+in+immune+disorders%3A+a+molecular+rheostat+that+adjusts+chaperone-mediated+autophagy+substrates&doi=10.3390%2Fcells8080849"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">HSPA8/HSC70 in immune disorders: a molecular rheostat that adjusts chaperone-mediated autophagy substrates</span></div><div class="casAuthors">Bonam, Srinivasa Reddy; Ruff, Marc; Muller, Sylviane</div><div class="citationInfo"><span class="NLM_cas:title">Cells</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">849</span>CODEN:
                <span class="NLM_cas:coden">CELLC6</span>;
        ISSN:<span class="NLM_cas:issn">2073-4409</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">HSPA8/HSC70 is a mol. chaperone involved in a wide variety of cellular processes.  It plays a crucial role in protein quality control, ensuring the correct folding and re-folding of selected proteins, and controlling the elimination of abnormally-folded conformers and of proteins daily produced in excess in our cells.  HSPA8 is a crucial mol. regulator of chaperone-mediated autophagy, as a detector of substrates that will be processed by this specialized autophagy pathway.  In this review, we shortly summarize its structure and overall functions, dissect its implication in immune disorders, and list the known pharmacol. tools that modulate its functions.  We also exemplify the interest of targeting HSPA8 to regulate pathol. immune dysfunctions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUjQS9gBwqhbVg90H21EOLACvtfcHk0lguARtIW_resw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXlsVGjsbc%253D&md5=4053930b5aa4d2ecb63344af4c59e479</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.3390%2Fcells8080849&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fcells8080849%26sid%3Dliteratum%253Aachs%26aulast%3DBonam%26aufirst%3DS.%2BR.%26aulast%3DRuff%26aufirst%3DM.%26aulast%3DMuller%26aufirst%3DS.%26atitle%3DHSPA8%252FHSC70%2520in%2520immune%2520disorders%253A%2520a%2520molecular%2520rheostat%2520that%2520adjusts%2520chaperone-mediated%2520autophagy%2520substrates%26jtitle%3DCells%26date%3D2019%26volume%3D8%26spage%3D849%26doi%3D10.3390%2Fcells8080849" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dikic, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elazar, Z.</span></span> <span> </span><span class="NLM_article-title">Mechanism and medical implications of mammalian autophagy</span>. <i>Nat. Rev. Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">349</span>– <span class="NLM_lpage">364</span>, <span class="refDoi"> DOI: 10.1038/s41580-018-0003-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1038%2Fs41580-018-0003-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=29618831" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC1cXosVCns7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2018&pages=349-364&author=I.+Dikicauthor=Z.+Elazar&title=Mechanism+and+medical+implications+of+mammalian+autophagy&doi=10.1038%2Fs41580-018-0003-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanism and medical implications of mammalian autophagy</span></div><div class="casAuthors">Dikic, Ivan; Elazar, Zvulun</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">349-364</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review.  Autophagy is a highly conserved catabolic process induced under various conditions of cellular stress, which prevents cell damage and promotes survival in the event of energy or nutrient shortage and responds to various cytotoxic insults.  Thus, autophagy has primarily cytoprotective functions and needs to be tightly regulated to respond correctly to the different stimuli that cells experience, thereby conferring adaptation to the ever-changing environment.  It is now apparent that autophagy is deregulated in the context of various human pathologies, including cancer and neurodegeneration, and its modulation has considerable potential as a therapeutic approach.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHijrOpvofErVg90H21EOLACvtfcHk0ljDwY6QgI6yuQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXosVCns7s%253D&md5=7cbbf53eca81386639df68c0f4a42339</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1038%2Fs41580-018-0003-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41580-018-0003-4%26sid%3Dliteratum%253Aachs%26aulast%3DDikic%26aufirst%3DI.%26aulast%3DElazar%26aufirst%3DZ.%26atitle%3DMechanism%2520and%2520medical%2520implications%2520of%2520mammalian%2520autophagy%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2018%26volume%3D19%26spage%3D349%26epage%3D364%26doi%3D10.1038%2Fs41580-018-0003-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mizushima, N.</span></span> <span> </span><span class="NLM_article-title">Autophagy and human diseases</span>. <i>Cell Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">69</span>– <span class="NLM_lpage">79</span>, <span class="refDoi"> DOI: 10.1038/cr.2013.161</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1038%2Fcr.2013.161" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=24323045" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFCru7fK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2014&pages=69-79&author=P.+Jiangauthor=N.+Mizushima&title=Autophagy+and+human+diseases&doi=10.1038%2Fcr.2013.161"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Autophagy and human diseases</span></div><div class="casAuthors">Jiang, Peidu; Mizushima, Noboru</div><div class="citationInfo"><span class="NLM_cas:title">Cell Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">69-79</span>CODEN:
                <span class="NLM_cas:coden">CREEB6</span>;
        ISSN:<span class="NLM_cas:issn">1001-0602</span>.
    
            (<span class="NLM_cas:orgname">NPG Nature Asia-Pacific</span>)
        </div><div class="casAbstract">A review.  Autophagy is a major intracellular degradative process that delivers cytoplasmic materials to the lysosome for degrdn.  Since the discovery of autophagy-related (Atg) genes in the 1990s, there has been a proliferation of studies on the physiol. and pathol. roles of autophagy in a variety of autophagy knockout models.  However, direct evidence of the connections between ATG gene dysfunction and human diseases has emerged only recently.  There are an increasing no. of reports showing that mutations in the ATG genes were identified in various human diseases such as neurodegenerative diseases, infectious diseases, and cancers.  Here, we review the major advances in identification of mutations or polymorphisms of the ATG genes in human diseases.  Current autophagy-modulating compds. in clin. trials are also summarized.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoTFjg9nyx11bVg90H21EOLACvtfcHk0ljDwY6QgI6yuQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFCru7fK&md5=4155f89f1cc9957e607a2955299fc5da</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1038%2Fcr.2013.161&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fcr.2013.161%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DP.%26aulast%3DMizushima%26aufirst%3DN.%26atitle%3DAutophagy%2520and%2520human%2520diseases%26jtitle%3DCell%2520Res.%26date%3D2014%26volume%3D24%26spage%3D69%26epage%3D79%26doi%3D10.1038%2Fcr.2013.161" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shacka, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span> <span> </span><span class="NLM_article-title">The autophagy-lysosomal degradation pathway: role in neurodegenerative disease and therapy</span>. <i>Front. Biosci., Landmark Ed.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">718</span>– <span class="NLM_lpage">736</span>, <span class="refDoi"> DOI: 10.2741/2714</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.2741%2F2714" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=17981582" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhs1Cht7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2008&pages=718-736&author=J.+J.+Shackaauthor=K.+A.+Rothauthor=J.+Zhang&title=The+autophagy-lysosomal+degradation+pathway%3A+role+in+neurodegenerative+disease+and+therapy&doi=10.2741%2F2714"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">The autophagy-lysosomal degradation pathway: role in neurodegenerative disease and therapy</span></div><div class="casAuthors">Shacka, John J.; Roth, Kevin A.; Zhang, Jianhua</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in Bioscience</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">718-736</span>CODEN:
                <span class="NLM_cas:coden">FRBIF6</span>;
        ISSN:<span class="NLM_cas:issn">1093-4715</span>.
    
            (<span class="NLM_cas:orgname">Frontiers in Bioscience</span>)
        </div><div class="casAbstract">A review.  Alterations in the autophagy-lysosomal degrdn. pathway have been described in normal brain aging and in age-related neurodegenerative diseases including Alzheimer's (AD) and Parkinson's (PD).  An improper clearance of proteins in AD and PD may result either from a compromise in the autophagy-lysosomal degrdn. pathway or induce alterations in this pathway, and may result in neuron dysfunction and neuron loss.  This review provides an overview of AD and PD with a specific focus on macroautophagy, chaperone-mediated autophagy and lysosome function in human and exptl. models of AD and PD.  Potential therapies for AD and PD are also discussed that may promote survival by regulating the autophagy and lysosomal degrdn. pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo332V8qqFPC7Vg90H21EOLACvtfcHk0ljDwY6QgI6yuQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhs1Cht7c%253D&md5=65657dccac0dca90f5eaaddf8530e54d</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.2741%2F2714&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2741%252F2714%26sid%3Dliteratum%253Aachs%26aulast%3DShacka%26aufirst%3DJ.%2BJ.%26aulast%3DRoth%26aufirst%3DK.%2BA.%26aulast%3DZhang%26aufirst%3DJ.%26atitle%3DThe%2520autophagy-lysosomal%2520degradation%2520pathway%253A%2520role%2520in%2520neurodegenerative%2520disease%2520and%2520therapy%26jtitle%3DFront.%2520Biosci.%252C%2520Landmark%2520Ed.%26date%3D2008%26volume%3D13%26spage%3D718%26epage%3D736%26doi%3D10.2741%2F2714" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Banik, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pedram, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wisnovsky, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahn, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riley, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertozzi, C. R.</span></span> <span> </span><span class="NLM_article-title">Lysosome-targeting chimaeras for degradation of extracellular proteins</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>584</i></span>,  <span class="NLM_fpage">291</span>– <span class="NLM_lpage">297</span>, <span class="refDoi"> DOI: 10.1038/s41586-020-2545-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1038%2Fs41586-020-2545-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=32728216" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhsVygtLrP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=584&publication_year=2020&pages=291-297&author=S.+M.+Banikauthor=K.+Pedramauthor=S.+Wisnovskyauthor=G.+Ahnauthor=N.+M.+Rileyauthor=C.+R.+Bertozzi&title=Lysosome-targeting+chimaeras+for+degradation+of+extracellular+proteins&doi=10.1038%2Fs41586-020-2545-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Lysosome-targeting chimaeras for degradation of extracellular proteins</span></div><div class="casAuthors">Banik, Steven M.; Pedram, Kayvon; Wisnovsky, Simon; Ahn, Green; Riley, Nicholas M.; Bertozzi, Carolyn R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">584</span>
        (<span class="NLM_cas:issue">7820</span>),
    <span class="NLM_cas:pages">291-297</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">The majority of therapies that target individual proteins rely on specific activity-modulating interactions with the target protein-for example, enzyme inhibition or ligand blocking.  However, several major classes of therapeutically relevant proteins have unknown or inaccessible activity profiles and so cannot be targeted by such strategies.  Protein-degrdn. platforms such as proteolysis-targeting chimeras (PROTACs) and others (for example, dTAGs, Trim-Away, chaperone-mediated autophagy targeting and SNIPERs) have been developed for proteins that are typically difficult to target; however, these methods involve the manipulation of intracellular protein degrdn. machinery and are therefore fundamentally limited to proteins that contain cytosolic domains to which ligands can bind and recruit the requisite cellular components.  Extracellular and membrane-assocd. proteins-the products of 40% of all protein-encoding genes-are key agents in cancer, ageing-related diseases and autoimmune disorders, and so a general strategy to selectively degrade these proteins has the potential to improve human health.  Here we establish the targeted degrdn. of extracellular and membrane-assocd. proteins using conjugates that bind both a cell-surface lysosome-shuttling receptor and the extracellular domain of a target protein.  These initial lysosome-targeting chimeras, which we term LYTACs, consist of a small mol. or antibody fused to chem. synthesized glycopeptide ligands that are agonists of the cation-independent mannose-6-phosphate receptor (CI-M6PR).  We use LYTACs to develop a CRISPR interference screen that reveals the biochem. pathway for CI-M6PR-mediated cargo internalization in cell lines, and uncover the exocyst complex as a previously unidentified-but essential-component of this pathway.  We demonstrate the scope of this platform through the degrdn. of therapeutically relevant proteins, including apolipoprotein E4, epidermal growth factor receptor, CD71 and programmed death-ligand 1.  Our results establish a modular strategy for directing secreted and membrane proteins for lysosomal degrdn., with broad implications for biochem. research and for therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmZfj1oJhbx7Vg90H21EOLACvtfcHk0ljDwY6QgI6yuQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhsVygtLrP&md5=1109c68b56a3ee93427ae89f194c0a16</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1038%2Fs41586-020-2545-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-020-2545-9%26sid%3Dliteratum%253Aachs%26aulast%3DBanik%26aufirst%3DS.%2BM.%26aulast%3DPedram%26aufirst%3DK.%26aulast%3DWisnovsky%26aufirst%3DS.%26aulast%3DAhn%26aufirst%3DG.%26aulast%3DRiley%26aufirst%3DN.%2BM.%26aulast%3DBertozzi%26aufirst%3DC.%2BR.%26atitle%3DLysosome-targeting%2520chimaeras%2520for%2520degradation%2520of%2520extracellular%2520proteins%26jtitle%3DNature%26date%3D2020%26volume%3D584%26spage%3D291%26epage%3D297%26doi%3D10.1038%2Fs41586-020-2545-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ahn, G.</span>; <span class="NLM_string-name">Banik, S.</span>; <span class="NLM_string-name">Miller, C. M.</span>; <span class="NLM_string-name">Riley, N.</span>; <span class="NLM_string-name">Cochran, J. R.</span>; <span class="NLM_string-name">Bertozzi, C.</span></span> <span> </span><span class="NLM_article-title">Lysosome targeting chimeras (LYTACs) that engage a liver-specific asialoglycoprotein receptor for targeted protein degradation</span>.  <i>ChemRxiv</i> <span class="NLM_year" style="font-weight: bold;">2020</span>, <span class="refDoi"> DOI: 10.26434/chemrxiv.12736778.v1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.26434%2Fchemrxiv.12736778.v1" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2020&author=G.+Ahn&author=S.+Banik&author=C.+M.+Miller&author=N.+Riley&author=J.+R.+Cochran&author=C.+Bertozzi&title=Lysosome+targeting+chimeras+%28LYTACs%29+that+engage+a+liver-specific+asialoglycoprotein+receptor+for+targeted+protein+degradation&doi=10.26434%2Fchemrxiv.12736778.v1"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.26434%2Fchemrxiv.12736778.v1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.26434%252Fchemrxiv.12736778.v1%26sid%3Dliteratum%253Aachs%26aulast%3DAhn%26aufirst%3DG.%26atitle%3DLysosome%2520targeting%2520chimeras%2520%2528LYTACs%2529%2520that%2520engage%2520a%2520liver-specific%2520asialoglycoprotein%2520receptor%2520for%2520targeted%2520protein%2520degradation%26jtitle%3DChemRxiv%26date%3D2020%26doi%3D10.26434%2Fchemrxiv.12736778.v1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Takahashi, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriyama, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakamura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miki, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takahashi, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sato, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akaike, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Itto-Nakama, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arimoto, H.</span></span> <span> </span><span class="NLM_article-title">AUTACs: cargo-specific degraders using selective autophagy</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>76</i></span>,  <span class="NLM_fpage">797</span>– <span class="NLM_lpage">810</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2019.09.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1016%2Fj.molcel.2019.09.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=31606272" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhvFKjsr%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=76&publication_year=2019&pages=797-810&author=D.+Takahashiauthor=J.+Moriyamaauthor=T.+Nakamuraauthor=E.+Mikiauthor=E.+Takahashiauthor=A.+Satoauthor=T.+Akaikeauthor=K.+Itto-Nakamaauthor=H.+Arimoto&title=AUTACs%3A+cargo-specific+degraders+using+selective+autophagy&doi=10.1016%2Fj.molcel.2019.09.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">AUTACs: Cargo-Specific Degraders Using Selective Autophagy</span></div><div class="casAuthors">Takahashi, Daiki; Moriyama, Jun; Nakamura, Tomoe; Miki, Erika; Takahashi, Eriko; Sato, Ayami; Akaike, Takaaki; Itto-Nakama, Kaori; Arimoto, Hirokazu</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">76</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">797-810.e10</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Protein silencing represents an essential tool in biomedical research.  Targeted protein degrdn. (TPD) strategies exemplified by PROTACs are rapidly emerging as modalities in drug discovery.  However, the scope of current TPD techniques is limited because many intracellular materials are not substrates of proteasomal clearance.  Here, we described a novel targeted-clearance strategy (autophagy-targeting chimera [AUTAC]) that contains a degrdn. tag (guanine derivs.) and a warhead to provide target specificity.  As expected from the substrate scope of autophagy, AUTAC degraded fragmented mitochondria as well as proteins.  Mitochondria-targeted AUTAC accelerated both the removal of dysfunctional fragmented mitochondria and the biogenesis of functionally normal mitochondria in patient-derived fibroblast cells.  Cytoprotective effects against acute mitochondrial injuries were also seen.  Canonical autophagy is viewed as a nonselective bulk decompn. system, and none of the available autophagy-inducing agents exhibit useful cargo selectivity.  With its target specificity, AUTAC provides a new modality for research on autophagy-based drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqEB3iYVVOYJbVg90H21EOLACvtfcHk0lhC3fxdRojkuw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhvFKjsr%252FE&md5=2eb738f4a51d83e7aee6d69e47bf2e00</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2019.09.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2019.09.009%26sid%3Dliteratum%253Aachs%26aulast%3DTakahashi%26aufirst%3DD.%26aulast%3DMoriyama%26aufirst%3DJ.%26aulast%3DNakamura%26aufirst%3DT.%26aulast%3DMiki%26aufirst%3DE.%26aulast%3DTakahashi%26aufirst%3DE.%26aulast%3DSato%26aufirst%3DA.%26aulast%3DAkaike%26aufirst%3DT.%26aulast%3DItto-Nakama%26aufirst%3DK.%26aulast%3DArimoto%26aufirst%3DH.%26atitle%3DAUTACs%253A%2520cargo-specific%2520degraders%2520using%2520selective%2520autophagy%26jtitle%3DMol.%2520Cell%26date%3D2019%26volume%3D76%26spage%3D797%26epage%3D810%26doi%3D10.1016%2Fj.molcel.2019.09.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sha, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Difiglia, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mei, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fei, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, B.</span></span> <span> </span><span class="NLM_article-title">Allele-selective lowering of mutant HTT protein by HTT-LC3 linker compounds</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>575</i></span>,  <span class="NLM_fpage">203</span>– <span class="NLM_lpage">209</span>, <span class="refDoi"> DOI: 10.1038/s41586-019-1722-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1038%2Fs41586-019-1722-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=31666698" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitV2hsb3O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=575&publication_year=2019&pages=203-209&author=Z.+Liauthor=C.+Wangauthor=Z.+Wangauthor=C.+Zhuauthor=J.+Liauthor=T.+Shaauthor=L.+Maauthor=C.+Gaoauthor=Y.+Yangauthor=Y.+Sunauthor=J.+Wangauthor=X.+Sunauthor=C.+Luauthor=M.+Difigliaauthor=Y.+Meiauthor=C.+Dingauthor=S.+Luoauthor=Y.+Dangauthor=Y.+Dingauthor=Y.+Feiauthor=B.+Lu&title=Allele-selective+lowering+of+mutant+HTT+protein+by+HTT-LC3+linker+compounds&doi=10.1038%2Fs41586-019-1722-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Allele-selective lowering of mutant HTT protein by HTT-LC3 linker compounds</span></div><div class="casAuthors">Li, Zhaoyang; Wang, Cen; Wang, Ziying; Zhu, Chenggang; Li, Jie; Sha, Tian; Ma, Lixiang; Gao, Chao; Yang, Yi; Sun, Yimin; Wang, Jian; Sun, Xiaoli; Lu, Chenqi; Difiglia, Marian; Mei, Yanai; Ding, Chen; Luo, Shouqing; Dang, Yongjun; Ding, Yu; Fei, Yiyan; Lu, Boxun</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">575</span>
        (<span class="NLM_cas:issue">7781</span>),
    <span class="NLM_cas:pages">203-209</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Accumulation of mutant proteins is a major cause of many diseases (collectively called proteopathies), and lowering the level of these proteins can be useful for treatment of these diseases.  We hypothesized that compds. that interact with both the autophagosome protein microtubule-assocd. protein 1A/1B light chain 3 (LC3)1 and the disease-causing protein may target the latter for autophagic clearance.  Mutant huntingtin protein (mHTT) contains an expanded polyglutamine (polyQ) tract and causes Huntington's disease, an incurable neurodegenerative disorder2.  Here, using small-mol.-microarray-based screening, we identified four compds. that interact with both LC3 and mHTT, but not with the wild-type HTT protein.  Some of these compds. targeted mHTT to autophagosomes, reduced mHTT levels in an allele-selective manner, and rescued disease-relevant phenotypes in cells and in vivo in fly and mouse models of Huntington's disease.  We further show that these compds. interact with the expanded polyQ stretch and could lower the level of mutant ataxin-3 (ATXN3), another disease-causing protein with an expanded polyQ tract3.  This study presents candidate compds. for lowering mHTT and potentially other disease-causing proteins with polyQ expansions, demonstrating the concept of lowering levels of disease-causing proteins using autophagosome-tethering compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7rSfxy_Wl7rVg90H21EOLACvtfcHk0lhC3fxdRojkuw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitV2hsb3O&md5=98cba22501f8157c83c0181b74ca15e2</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1038%2Fs41586-019-1722-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-019-1722-1%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DZhu%26aufirst%3DC.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DSha%26aufirst%3DT.%26aulast%3DMa%26aufirst%3DL.%26aulast%3DGao%26aufirst%3DC.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DSun%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DSun%26aufirst%3DX.%26aulast%3DLu%26aufirst%3DC.%26aulast%3DDifiglia%26aufirst%3DM.%26aulast%3DMei%26aufirst%3DY.%26aulast%3DDing%26aufirst%3DC.%26aulast%3DLuo%26aufirst%3DS.%26aulast%3DDang%26aufirst%3DY.%26aulast%3DDing%26aufirst%3DY.%26aulast%3DFei%26aufirst%3DY.%26aulast%3DLu%26aufirst%3DB.%26atitle%3DAllele-selective%2520lowering%2520of%2520mutant%2520HTT%2520protein%2520by%2520HTT-LC3%2520linker%2520compounds%26jtitle%3DNature%26date%3D2019%26volume%3D575%26spage%3D203%26epage%3D209%26doi%3D10.1038%2Fs41586-019-1722-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bauer, P. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goswami, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, H. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okuno, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kurosawa, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamada, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miyazaki, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsumoto, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kino, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nukina, N.</span></span> <span> </span><span class="NLM_article-title">Harnessing chaperone-mediated autophagy for the selective degradation of mutant huntingtin protein</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">256</span>– <span class="NLM_lpage">263</span>, <span class="refDoi"> DOI: 10.1038/nbt.1608</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1038%2Fnbt.1608" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=20190739" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC3cXisFeksb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2010&pages=256-263&author=P.+O.+Bauerauthor=A.+Goswamiauthor=H.+K.+Wongauthor=M.+Okunoauthor=M.+Kurosawaauthor=M.+Yamadaauthor=H.+Miyazakiauthor=G.+Matsumotoauthor=Y.+Kinoauthor=Y.+Nagaiauthor=N.+Nukina&title=Harnessing+chaperone-mediated+autophagy+for+the+selective+degradation+of+mutant+huntingtin+protein&doi=10.1038%2Fnbt.1608"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Harnessing chaperone-mediated autophagy for the selective degradation of mutant huntingtin protein</span></div><div class="casAuthors">Bauer, Peter O.; Goswami, Anand; Wong, Hon Kit; Okuno, Misako; Kurosawa, Masaru; Yamada, Mizuki; Miyazaki, Haruko; Matsumoto, Gen; Kino, Yoshihiro; Nagai, Yoshitaka; Nukina, Nobuyuki</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">256-263</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Huntington's Disease (HD) is a dominantly inherited pathol. caused by the accumulation of mutant huntingtin protein (HTT) contg. an expanded polyglutamine (polyQ) tract.  As the polyglutamine binding peptide 1 (QBP1) is known to bind an expanded polyQ tract but not the polyQ motif found in normal HTT, we selectively targeted mutant HTT for degrdn. by expressing a fusion mol. comprising two copies of QBP1 and copies of two different heat shock cognate protein 70 (HSC70)-binding motifs in cellular and mouse models of HD.  Chaperone-mediated autophagy contributed to the specific degrdn. of mutant HTT in cultured cells expressing the construct.  Intrastriatal delivery of a virus expressing the fusion mol. ameliorated the disease phenotype in the R6/2 mouse model of HD.  Similar adaptor mols. comprising HSC70-binding motifs fused to an appropriate structure-specific binding agent(s) may have therapeutic potential for treating diseases caused by misfolded proteins other than those with expanded polyQ tracts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoOHevKvAH5LrVg90H21EOLACvtfcHk0lhC3fxdRojkuw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXisFeksb8%253D&md5=57caf1a71777ff31ab423541db69ad75</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1038%2Fnbt.1608&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt.1608%26sid%3Dliteratum%253Aachs%26aulast%3DBauer%26aufirst%3DP.%2BO.%26aulast%3DGoswami%26aufirst%3DA.%26aulast%3DWong%26aufirst%3DH.%2BK.%26aulast%3DOkuno%26aufirst%3DM.%26aulast%3DKurosawa%26aufirst%3DM.%26aulast%3DYamada%26aufirst%3DM.%26aulast%3DMiyazaki%26aufirst%3DH.%26aulast%3DMatsumoto%26aufirst%3DG.%26aulast%3DKino%26aufirst%3DY.%26aulast%3DNagai%26aufirst%3DY.%26aulast%3DNukina%26aufirst%3DN.%26atitle%3DHarnessing%2520chaperone-mediated%2520autophagy%2520for%2520the%2520selective%2520degradation%2520of%2520mutant%2520huntingtin%2520protein%26jtitle%3DNat.%2520Biotechnol.%26date%3D2010%26volume%3D28%26spage%3D256%26epage%3D263%26doi%3D10.1038%2Fnbt.1608" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fan, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, W. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y. T.</span></span> <span> </span><span class="NLM_article-title">Rapid and reversible knockdown of endogenous proteins by peptide-directed lysosomal degradation</span>. <i>Nat. Neurosci.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">471</span>– <span class="NLM_lpage">480</span>, <span class="refDoi"> DOI: 10.1038/nn.3637</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1038%2Fnn.3637" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=24464042" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht1ynsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2014&pages=471-480&author=X.+Fanauthor=W.+Y.+Jinauthor=J.+Luauthor=J.+Wangauthor=Y.+T.+Wang&title=Rapid+and+reversible+knockdown+of+endogenous+proteins+by+peptide-directed+lysosomal+degradation&doi=10.1038%2Fnn.3637"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Rapid and reversible knockdown of endogenous proteins by peptide-directed lysosomal degradation</span></div><div class="casAuthors">Fan, Xuelai; Jin, Wu Yang; Lu, Jie; Wang, Jin; Wang, Yu Tian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Neuroscience</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">471-480</span>CODEN:
                <span class="NLM_cas:coden">NANEFN</span>;
        ISSN:<span class="NLM_cas:issn">1097-6256</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Rapid and reversible methods for altering the levels of endogenous proteins are critically important for studying biol. systems and developing therapeutics.  Here we describe a membrane-permeant targeting peptide-based method that rapidly and reversibly knocks down endogenous proteins through chaperone-mediated autophagy in vitro and in vivo.  We demonstrate the specificity, efficacy and generalizability of the method by showing efficient knockdown of various proteins, including death assocd. protein kinase 1 (160 kDa), scaffolding protein PSD-95 (95 kDa) and α-synuclein (18 kDa), with their resp. targeting peptides in a dose-, time- and lysosomal activity-dependent manner in rat neuronal cultures.  Moreover, we show that, when given systemically, the peptide system efficiently knocked down the targeted protein in the brains of intact rats.  Our study provides a robust and convenient research tool for manipulating endogenous protein levels and may also lead to the development of protein knockdown-based therapeutics for treating human diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpVBXNfrBcGa7Vg90H21EOLACvtfcHk0lhXaqR_g57R0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht1ynsr8%253D&md5=6891b206bf70a86dbbf410c8e08346bd</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1038%2Fnn.3637&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnn.3637%26sid%3Dliteratum%253Aachs%26aulast%3DFan%26aufirst%3DX.%26aulast%3DJin%26aufirst%3DW.%2BY.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DY.%2BT.%26atitle%3DRapid%2520and%2520reversible%2520knockdown%2520of%2520endogenous%2520proteins%2520by%2520peptide-directed%2520lysosomal%2520degradation%26jtitle%3DNat.%2520Neurosci.%26date%3D2014%26volume%3D17%26spage%3D471%26epage%3D480%26doi%3D10.1038%2Fnn.3637" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref64"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref64'); return false;" data-citation="" class="refNumLink">64</a></strong><div class="NLM_citation" id="cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Y. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chi, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, L. Q.</span></span> <span> </span><span class="NLM_article-title">The Peptide-directed lysosomal degradation of CDK5 exerts therapeutic effects against stroke</span>. <i>Aging Dis.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1140</span>– <span class="NLM_lpage">1145</span>, <span class="refDoi"> DOI: 10.14336/AD.2018.1225</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.14336%2FAD.2018.1225" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=31595208" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A280%3ADC%252BB3MnmtFCktQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=1140-1145&author=Y.+F.+Zhouauthor=J.+Wangauthor=M.+F.+Dengauthor=B.+Chiauthor=N.+Weiauthor=J.+G.+Chenauthor=D.+Liuauthor=X.+Yinauthor=Y.+Luauthor=L.+Q.+Zhu&title=The+Peptide-directed+lysosomal+degradation+of+CDK5+exerts+therapeutic+effects+against+stroke&doi=10.14336%2FAD.2018.1225"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">The Peptide-Directed Lysosomal Degradation of CDK5 Exerts Therapeutic Effects against Stroke</span></div><div class="casAuthors">Zhou Ya-Fan; Deng Man-Fei; Liu Dan; Zhu Ling-Qiang; Wang Jing; Chi Bin; Wei Na; Chen Jian-Guo; Chen Jian-Guo; Liu Dan; Lu Youming; Zhu Ling-Qiang; Yin Xiaoping</div><div class="citationInfo"><span class="NLM_cas:title">Aging and disease</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1140-1145</span>
        ISSN:<span class="NLM_cas:issn">2152-5250</span>.
    </div><div class="casAbstract">The aberrant activation of CDK5 has been implicated in neuronal death in stroke.  The goal of this study is to determine whether knocking down CDK5 by a peptide-directed lysosomal degradation approach is therapeutically effective against stroke.  We synthesized a membrane-permeable peptide that specifically binds to CDK5 with a chaperone-mediated autophagy targeting motif (Tat-CDK5-CTM) and tested its therapeutic effects on a mouse model of ischemic stroke.  Our results showed that Tat-CDK5-CTM blocked the CDK5-NR2B interaction, resulting in the degradation of CDK5, which in turn prevented calcium overload and neuronal death in cultured neurons.  Tat-CDK5-CTM also reduced the infarction area and neuronal loss and improved the neurological functions in MCAO (Middle cerebral artery occlusion) mice.  The peptide-directed lysosomal degradation of CDK5 is a promising therapeutic intervention for stroke.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQ9EnaSakFg7AxyojvcjibhfW6udTcc2ea0QZe_OVbI3bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3MnmtFCktQ%253D%253D&md5=64be3cec620c41c197ad7a45c8f8234e</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.14336%2FAD.2018.1225&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.14336%252FAD.2018.1225%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DY.%2BF.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DDeng%26aufirst%3DM.%2BF.%26aulast%3DChi%26aufirst%3DB.%26aulast%3DWei%26aufirst%3DN.%26aulast%3DChen%26aufirst%3DJ.%2BG.%26aulast%3DLiu%26aufirst%3DD.%26aulast%3DYin%26aufirst%3DX.%26aulast%3DLu%26aufirst%3DY.%26aulast%3DZhu%26aufirst%3DL.%2BQ.%26atitle%3DThe%2520Peptide-directed%2520lysosomal%2520degradation%2520of%2520CDK5%2520exerts%2520therapeutic%2520effects%2520against%2520stroke%26jtitle%3DAging%2520Dis.%26date%3D2019%26volume%3D10%26spage%3D1140%26epage%3D1145%26doi%3D10.14336%2FAD.2018.1225" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref65"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref65'); return false;" data-citation="" class="refNumLink">65</a></strong><div class="NLM_citation" id="cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, J.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J.</span></span> <span> </span><span class="NLM_article-title">HIP1R targets PD-L1 to lysosomal degradation to alter T cell-mediated cytotoxicity</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">42</span>– <span class="NLM_lpage">50</span>, <span class="refDoi"> DOI: 10.1038/s41589-018-0161-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1038%2Fs41589-018-0161-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=30397328" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitV2qtbbN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2019&pages=42-50&author=H.+Wangauthor=H.+Yaoauthor=C.+Liauthor=H.+Shiauthor=J.+Lanauthor=Z.+Liauthor=Y.+Zhangauthor=L.+Liangauthor=J.-Y.+Fangauthor=J.+Xu&title=HIP1R+targets+PD-L1+to+lysosomal+degradation+to+alter+T+cell-mediated+cytotoxicity&doi=10.1038%2Fs41589-018-0161-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">HIP1R targets PD-L1 to lysosomal degradation to alter T cell-mediated cytotoxicity</span></div><div class="casAuthors">Wang, Huanbin; Yao, Han; Li, Chushu; Shi, Hubing; Lan, Jiang; Li, Zhaoli; Zhang, Yao; Liang, Lunxi; Fang, Jing-Yuan; Xu, Jie</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">42-50</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Expression of programmed cell death 1 (PD-1) ligand 1 (PD-L1) protects tumor cells from T cell-mediated immune surveillance, and immune checkpoint blockade (ICB) therapies targeting PD-1 and PD-L1 have exhibited significant clin. benefits.  However, the relatively low response rate and obsd. ICB resistance highlight the need to understand the mol. regulation of PD-L1.  Here we show that HIP1R targets PD-L1 to lysosomal degrdn. to alter T cell-mediated cytotoxicity.  HIP1R phys. interacts with PD-L1 and delivers PD-L1 to the lysosome through a lysosomal targeting signal.  Depletion of HIP1R in tumor cells caused PD-L1 accumulation and suppressed T cell-mediated cytotoxicity.  A rationally designed peptide (PD-LYSO) incorporating the lysosome-sorting signal and the PD-L1-binding sequence of HIP1R successfully depleted PD-L1 expression in tumor cells.  Our results identify the mol. machineries governing the lysosomal degrdn. of PD-L1 and exemplify the development of a chimeric peptide for targeted degrdn. of PD-L1 as a crucial anticancer target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreVHjIVSWPmLVg90H21EOLACvtfcHk0lhXaqR_g57R0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitV2qtbbN&md5=eac9da24ac6ff6e17cf10cfc6ee2c08a</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1038%2Fs41589-018-0161-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41589-018-0161-x%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DH.%26aulast%3DYao%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DShi%26aufirst%3DH.%26aulast%3DLan%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DLiang%26aufirst%3DL.%26aulast%3DFang%26aufirst%3DJ.-Y.%26aulast%3DXu%26aufirst%3DJ.%26atitle%3DHIP1R%2520targets%2520PD-L1%2520to%2520lysosomal%2520degradation%2520to%2520alter%2520T%2520cell-mediated%2520cytotoxicity%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2019%26volume%3D15%26spage%3D42%26epage%3D50%26doi%3D10.1038%2Fs41589-018-0161-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref66"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref66'); return false;" data-citation="" class="refNumLink">66</a></strong><div class="NLM_citation" id="cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pillay, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elliott, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dennison, C.</span></span> <span> </span><span class="NLM_article-title">Endolysosomal proteolysis and its regulation</span>. <i>Biochem. J.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>363</i></span>,  <span class="NLM_fpage">417</span>– <span class="NLM_lpage">429</span>, <span class="refDoi"> DOI: 10.1042/bj3630417</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1042%2Fbj3630417" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=11964142" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BD38XjvFenurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=363&publication_year=2002&pages=417-429&author=C.+S.+Pillayauthor=E.+Elliottauthor=C.+Dennison&title=Endolysosomal+proteolysis+and+its+regulation&doi=10.1042%2Fbj3630417"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">Endolysosomal proteolysis and its regulation</span></div><div class="casAuthors">Pillay, Che S.; Elliott, Edith; Dennison, Clive</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Journal</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">363</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">417-429</span>CODEN:
                <span class="NLM_cas:coden">BIJOAK</span>;
        ISSN:<span class="NLM_cas:issn">0264-6021</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">A review.  The endolysosomal system comprises a unique environment for proteolysis, which is regulated in a manner that apparently does not involve protease inhibitors.  The system comprises a series of membrane-bound intracellular compartments, within which endocytosed material and redundant cellular components are hydrolyzed.  Endocytosed material tends to flow vectorially through the system, proceeding through the early endosome, the endosome carrier vesicle, the late endosome and the lysosome.  Phagocytosis and autophagy provide alternative entry points into the system.  Late endosomes, lysosome/late endosome hybrid organelles, phagosomes and autophagosomes are the principal sites for proteolysis.  In each case, hydrolytic competence is due to components of the endolysosomal system, i.e. proteases, lysosome-assocd. membrane proteins, H+-ATPases and possibly cysteine transporters.  The view is emerging that lysosomes are organelles for the storage of hydrolases, perhaps in an inactivated form.  Once a substrate has entered a proteolytically competent environment, the rate-limiting proteolytic steps are probably effected by cysteine endoproteinases.  As these are affected by pH and possibly redox potential, they may be regulated by the organelle luminal environment.  Regulation is probably also affected, among other factors, by organelle fusion reactions, whereby the meeting of enzyme and substrate may be controlled.  Such systems would permit simultaneous regulation of a no. of unrelated hydrolases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHet05kxwVELVg90H21EOLACvtfcHk0lhmRMldUyXJJA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XjvFenurg%253D&md5=41f045e53f3e1bc80fa44020bc950cef</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1042%2Fbj3630417&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252Fbj3630417%26sid%3Dliteratum%253Aachs%26aulast%3DPillay%26aufirst%3DC.%2BS.%26aulast%3DElliott%26aufirst%3DE.%26aulast%3DDennison%26aufirst%3DC.%26atitle%3DEndolysosomal%2520proteolysis%2520and%2520its%2520regulation%26jtitle%3DBiochem.%2520J.%26date%3D2002%26volume%3D363%26spage%3D417%26epage%3D429%26doi%3D10.1042%2Fbj3630417" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref67"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref67'); return false;" data-citation="" class="refNumLink">67</a></strong><div class="NLM_citation" id="cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kundu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, C. B.</span></span> <span> </span><span class="NLM_article-title">Autophagy: basic principles and relevance to disease</span>. <i>Annu. Rev. Pathol.: Mech. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">427</span>– <span class="NLM_lpage">455</span>, <span class="refDoi"> DOI: 10.1146/annurev.pathmechdis.2.010506.091842</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1146%2Fannurev.pathmechdis.2.010506.091842" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=18039129" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BD1cXjsFersLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2008&pages=427-455&author=M.+Kunduauthor=C.+B.+Thompson&title=Autophagy%3A+basic+principles+and+relevance+to+disease&doi=10.1146%2Fannurev.pathmechdis.2.010506.091842"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">Autophagy: basic principles and relevance to disease</span></div><div class="casAuthors">Kundu, Mondira; Thompson, Craig B.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Pathology: Mechanisms of Disease</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">427-455</span>CODEN:
                <span class="NLM_cas:coden">ARPMCU</span>;
        ISSN:<span class="NLM_cas:issn">1553-4006</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review.  Autophagy is a process by which cytoplasmic components are sequestered in double membrane vesicles and degraded upon fusion with lysosomal compartments.  In yeast, autophagy is activated in response to changes in the extracellular milieu.  Depending upon the stimulus, autophagy can degrade cytoplasmic contents nonspecifically or can target the degrdn. of specific cellular components.  Both of these have been adopted in higher eukaryotes and account for the expanding role of autophagy in various cellular processes, as well as contribute to the variation in cellular outcomes after induction of autophagy.  In some cases, autophagy appears to be an adaptive response, whereas under other circumstances it is involved in cell death.  In mammals, autophagy has been implicated in either the pathogenesis or response to a wide variety of diseases, including neurodegenerative disease, chronic bacterial and viral infections, atherosclerosis, and cancer.  As the basic mol. pathways that regulate autophagy are elucidated, the relationship of autophagy to the pathogenesis of various disease states emerges.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGraj2W1uAZS3rVg90H21EOLACvtfcHk0lhmRMldUyXJJA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXjsFersLk%253D&md5=57ef3d3e75a3d83a0dfcf9f3d9c9f7a0</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1146%2Fannurev.pathmechdis.2.010506.091842&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.pathmechdis.2.010506.091842%26sid%3Dliteratum%253Aachs%26aulast%3DKundu%26aufirst%3DM.%26aulast%3DThompson%26aufirst%3DC.%2BB.%26atitle%3DAutophagy%253A%2520basic%2520principles%2520and%2520relevance%2520to%2520disease%26jtitle%3DAnnu.%2520Rev.%2520Pathol.%253A%2520Mech.%2520Dis.%26date%3D2008%26volume%3D3%26spage%3D427%26epage%3D455%26doi%3D10.1146%2Fannurev.pathmechdis.2.010506.091842" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref68"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref68'); return false;" data-citation="" class="refNumLink">68</a></strong><div class="NLM_citation" id="cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dahms, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kornfeld, S.</span></span> <span> </span><span class="NLM_article-title">Mannose 6-phosphate receptors: new twists in the tale</span>. <i>Nat. Rev. Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">202</span>– <span class="NLM_lpage">213</span>, <span class="refDoi"> DOI: 10.1038/nrm1050</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1038%2Fnrm1050" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=12612639" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BD3sXhsFajsL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2003&pages=202-213&author=P.+Ghoshauthor=N.+M.+Dahmsauthor=S.+Kornfeld&title=Mannose+6-phosphate+receptors%3A+new+twists+in+the+tale&doi=10.1038%2Fnrm1050"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">Mannose 6-phosphate receptors: new twists in the tale</span></div><div class="casAuthors">Ghosh, Pradipta; Dahms, Nancy M.; Kornfeld, Stuart</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">202-213</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The two mannose 6-phosphate (M6P) receptors were identified because of their ability to bind M6P-contg. sol. acid hydrolases in the Golgi and transport them to the endosomal-lysosomal system.  During the past decade, we have started to understand the structural features of these receptors that allow them to do this job, and how the receptors themselves are sorted as they pass through various membrane-bound compartments.  But trafficking of acid hydrolases is only part of the story.  Evidence is emerging that one of the receptors can regulate cell growth and motility, and that it functions as a tumor suppressor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqKvbnA0U5O-bVg90H21EOLACvtfcHk0lhmRMldUyXJJA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXhsFajsL0%253D&md5=c079ad8e76fb1d7ee11c2ca554956961</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1038%2Fnrm1050&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm1050%26sid%3Dliteratum%253Aachs%26aulast%3DGhosh%26aufirst%3DP.%26aulast%3DDahms%26aufirst%3DN.%2BM.%26aulast%3DKornfeld%26aufirst%3DS.%26atitle%3DMannose%25206-phosphate%2520receptors%253A%2520new%2520twists%2520in%2520the%2520tale%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2003%26volume%3D4%26spage%3D202%26epage%3D213%26doi%3D10.1038%2Fnrm1050" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref69"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref69'); return false;" data-citation="" class="refNumLink">69</a></strong><div class="NLM_citation" id="cit69"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Whitworth, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">New class of molecule targets proteins outside cells for degradation</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>584</i></span>,  <span class="NLM_fpage">193</span>– <span class="NLM_lpage">194</span>, <span class="refDoi"> DOI: 10.1038/d41586-020-02211-w</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1038%2Fd41586-020-02211-w" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=32728153" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhsFKlu7zP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=584&publication_year=2020&pages=193-194&author=C.+Whitworthauthor=A.+Ciulli&title=New+class+of+molecule+targets+proteins+outside+cells+for+degradation&doi=10.1038%2Fd41586-020-02211-w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">New class of molecule targets proteins outside cells for degradation</span></div><div class="casAuthors">Whitworth, Claire; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">584</span>
        (<span class="NLM_cas:issue">7820</span>),
    <span class="NLM_cas:pages">193-194</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Mols. have previously been made that induce protein destruction inside cells.  A new class of mol. now induces the degrdn. of membrane and extracellular proteins - opening up avenues for drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2WZkkfZYTdLVg90H21EOLACvtfcHk0lgN6l1CUCt5Qg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhsFKlu7zP&md5=b19b685e8e7fb29fbcdc7b83c6fdf6cb</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1038%2Fd41586-020-02211-w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fd41586-020-02211-w%26sid%3Dliteratum%253Aachs%26aulast%3DWhitworth%26aufirst%3DC.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DNew%2520class%2520of%2520molecule%2520targets%2520proteins%2520outside%2520cells%2520for%2520degradation%26jtitle%3DNature%26date%3D2020%26volume%3D584%26spage%3D193%26epage%3D194%26doi%3D10.1038%2Fd41586-020-02211-w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref70"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref70'); return false;" data-citation="" class="refNumLink">70</a></strong><div class="NLM_citation" id="cit70"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yamazaki, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Painter, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kanekiyo, T.</span></span> <span> </span><span class="NLM_article-title">Apolipoprotein e as a therapeutic target in alzheimer’s disease: a review of basic research and clinical evidence</span>. <i>CNS Drugs</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">773</span>– <span class="NLM_lpage">789</span>, <span class="refDoi"> DOI: 10.1007/s40263-016-0361-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1007%2Fs40263-016-0361-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=27328687" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtVyiurfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2016&pages=773-789&author=Y.+Yamazakiauthor=M.+M.+Painterauthor=G.+Buauthor=T.+Kanekiyo&title=Apolipoprotein+e+as+a+therapeutic+target+in+alzheimer%E2%80%99s+disease%3A+a+review+of+basic+research+and+clinical+evidence&doi=10.1007%2Fs40263-016-0361-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">Apolipoprotein E as a Therapeutic Target in Alzheimer's Disease: A Review of Basic Research and Clinical Evidence</span></div><div class="casAuthors">Yamazaki, Yu; Painter, Meghan M.; Bu, Guojun; Kanekiyo, Takahisa</div><div class="citationInfo"><span class="NLM_cas:title">CNS Drugs</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">773-789</span>CODEN:
                <span class="NLM_cas:coden">CNDREF</span>;
        ISSN:<span class="NLM_cas:issn">1172-7047</span>.
    
            (<span class="NLM_cas:orgname">Springer International Publishing AG</span>)
        </div><div class="casAbstract">Alzheimer's disease (AD) is a devastating neurodegenerative disorder that causes progressive cognitive decline.  The majority of AD cases are sporadic and late-onset (>65 years old) making it the leading cause of dementia in the elderly.  While both genetic and environmental factors contribute to the development of late-onset AD (LOAD), APOE polymorphism is a major genetic risk determinant for LOAD.  In humans, the APOE gene has three major allelic variants: ε2, ε3, and ε4, of which APOE ε4 is the strongest genetic risk factor for LOAD, whereas APOE ε2 is protective.  Mounting evidence suggests that APOE ε4 contributes to AD pathogenesis through multiple pathways including facilitated amyloid-β deposition, increased tangle formation, synaptic dysfunction, exacerbated neuroinflammation, and cerebrovascular defects.  Since APOE modulates multiple biol. processes through its corresponding protein apolipoprotein E (apoE), APOE gene and apoE properties have been a promising target for therapy and drug development against AD.  In this review, we summarize the current evidence regarding how the APOE ε4 allele contributes to the pathogenesis of AD and how relevant therapeutic approaches can be developed to target apoE-mediated pathways in AD.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoU4kbr0W368rVg90H21EOLACvtfcHk0lgN6l1CUCt5Qg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtVyiurfJ&md5=f1fa4329f1de8914d1b8e8db41ce5580</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1007%2Fs40263-016-0361-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs40263-016-0361-4%26sid%3Dliteratum%253Aachs%26aulast%3DYamazaki%26aufirst%3DY.%26aulast%3DPainter%26aufirst%3DM.%2BM.%26aulast%3DBu%26aufirst%3DG.%26aulast%3DKanekiyo%26aufirst%3DT.%26atitle%3DApolipoprotein%2520e%2520as%2520a%2520therapeutic%2520target%2520in%2520alzheimer%25E2%2580%2599s%2520disease%253A%2520a%2520review%2520of%2520basic%2520research%2520and%2520clinical%2520evidence%26jtitle%3DCNS%2520Drugs%26date%3D2016%26volume%3D30%26spage%3D773%26epage%3D789%26doi%3D10.1007%2Fs40263-016-0361-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref71"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref71'); return false;" data-citation="" class="refNumLink">71</a></strong><div class="NLM_citation" id="cit71"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xue, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shang, P.</span></span> <span> </span><span class="NLM_article-title">Transferrin receptor 1 in cancer: a new sight for cancer therapy</span>. <i>Am. J. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">916</span>– <span class="NLM_lpage">931</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=30034931" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsFOjs7rF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2018&pages=916-931&author=Y.+Shenauthor=X.+Liauthor=D.+Dongauthor=B.+Zhangauthor=Y.+Xueauthor=P.+Shang&title=Transferrin+receptor+1+in+cancer%3A+a+new+sight+for+cancer+therapy"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">Transferrin receptor 1 in cancer: a new sight for cancer therapy</span></div><div class="casAuthors">Shen, Ying; Li, Xin; Dong, Dandan; Zhang, Bin; Xue, Yanru; Shang, Peng</div><div class="citationInfo"><span class="NLM_cas:title">American Journal of Cancer Research</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">916-931</span>CODEN:
                <span class="NLM_cas:coden">AJCRFT</span>;
        ISSN:<span class="NLM_cas:issn">2156-6976</span>.
    
            (<span class="NLM_cas:orgname">e-Century Publishing Corp.</span>)
        </div><div class="casAbstract">Iron as an important element plays crucial roles in various physiol. and pathol. processes.  Iron metab. behaves in systemic and cellular two levels that usually are in balance conditions.  The disorders of the iron metab. balances relate with many kinds of diseases including Alzheimer's disease, osteoporosis and various cancers.  In systemic iron metab. that is regulated by hepcidin-ferroportin axis, plasma iron is bound with transferrin (TF) which has two high-affinity binding sites for ferric iron.  The generic cellular iron metab. consists of iron intake, utilization and efflux.  During the iron intake process in generic cells, transferrin receptors (TFRs) act as the most important receptor mediated controls.  TFR1 and TFR2 are two subtypes of TFRs those bind with iron-transferrin complex to facilitate iron into cells.  TFR1 is ubiquitously expressed on the surfaces of generic cells, whereas TFR2 is specially expressed in liver cells.  TFR1 has attracted more attention than TFR2 by having diverse functions in both invertebrates and vertebrates.  Recently reports showed that TFR1 involved in many kinds of diseases including anemia, neurodegenerative diseases and cancers.  Most importantly, TFR1 has been verified to be abnormally expressed in various cancers.  Some exptl. and clin. drugs and antibodies targeting TFR1 have showed strong anti-tumor effects, herein TFR1 probably become a potential mol. target for diagnosis and treatment for cancer therapy.  This paper reviewed the research progresses of the roles of TFR1 in the tumorigenesis and cancer progression, the regulations of TFR1, and the therapeutic effects of targeting TFR1 on many kinds of cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrDe5E49dNFeLVg90H21EOLACvtfcHk0lgN6l1CUCt5Qg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsFOjs7rF&md5=f44799696f18eb50941fb7783d5377f0</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DShen%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DDong%26aufirst%3DD.%26aulast%3DZhang%26aufirst%3DB.%26aulast%3DXue%26aufirst%3DY.%26aulast%3DShang%26aufirst%3DP.%26atitle%3DTransferrin%2520receptor%25201%2520in%2520cancer%253A%2520a%2520new%2520sight%2520for%2520cancer%2520therapy%26jtitle%3DAm.%2520J.%2520Cancer%2520Res.%26date%3D2018%26volume%3D8%26spage%3D916%26epage%3D931" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref72"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref72'); return false;" data-citation="" class="refNumLink">72</a></strong><div class="NLM_citation" id="cit72"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cui, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, W.</span></span> <span> </span><span class="NLM_article-title">Elucidation and structural modeling of CD71 as a molecular target for cell-specific aptamer binding</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>141</i></span>,  <span class="NLM_fpage">10760</span>– <span class="NLM_lpage">10769</span>, <span class="refDoi"> DOI: 10.1021/jacs.9b03720</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.9b03720" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtFGqsLfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=141&publication_year=2019&pages=10760-10769&author=X.+Wuauthor=H.+Liuauthor=D.+Hanauthor=B.+Pengauthor=H.+Zhangauthor=L.+Zhangauthor=J.+Liauthor=J.+Liuauthor=C.+Cuiauthor=S.+Fangauthor=M.+Liauthor=M.+Yeauthor=W.+Tan&title=Elucidation+and+structural+modeling+of+CD71+as+a+molecular+target+for+cell-specific+aptamer+binding&doi=10.1021%2Fjacs.9b03720"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">Elucidation and Structural Modeling of CD71 as a Molecular Target for Cell-Specific Aptamer Binding</span></div><div class="casAuthors">Wu, Xiaoqiu; Liu, Honglin; Han, Dongmei; Peng, Bo; Zhang, Hui; Zhang, Lin; Li, Jianglin; Liu, Jing; Cui, Cheng; Fang, Senbiao; Li, Min; Ye, Mao; Tan, Weihong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">141</span>
        (<span class="NLM_cas:issue">27</span>),
    <span class="NLM_cas:pages">10760-10769</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Pancreatic cancer is a highly lethal malignancy assocd. with tissues of the pancreas.  Early diagnosis and effective treatment are crucial to improving the survival rate of patients with pancreatic cancer.  In a previous study, we employed the cell-SELEX strategy to obtain an ssDNA aptamer termed XQ-2d with high binding affinity for pancreatic cancer.  Here, we first identify CD71 as the XQ-2d-binding target.  We found that knockdown of CD71 abolished the binding of XQ-2d and that the binding affinity of XQ-2d is assocd. with membrane-bound CD71, rather than total CD71 levels.  Competitive anal. revealed that XQ-2d shares the same binding site on CD71 with transferrin (Tf), but not anti-CD71 antibody.  We then used a surface energy transfer (SET) nanoruler to measure the distance between the binding sites of XQ-2d and anti-CD71 antibody, and it was about 15 nm.  Furthermore, we did mol. dynamics simulation to clarify that the spatial structure of XQ-2d and Tf competitively binding to CD71.  We also engineered XQ-2d-mediated targeted therapy for pancreatic cancer, using an XQ-2d-based complex for loading doxorubicin (Dox).  Because CD71 is overexpressed not only in pancreatic cancer but also in a variety of tumors, our work provides a systematic novel way of studying a potential biomarker and also promising tools for cancer diagnosis and therapy, opening new doors for effective cancer theranostics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGofAAoZt5oLlrVg90H21EOLACvtfcHk0lgN6l1CUCt5Qg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtFGqsLfN&md5=04ce70f5db10f5459f633b073ca3f9df</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1021%2Fjacs.9b03720&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.9b03720%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DHan%26aufirst%3DD.%26aulast%3DPeng%26aufirst%3DB.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DCui%26aufirst%3DC.%26aulast%3DFang%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DM.%26aulast%3DYe%26aufirst%3DM.%26aulast%3DTan%26aufirst%3DW.%26atitle%3DElucidation%2520and%2520structural%2520modeling%2520of%2520CD71%2520as%2520a%2520molecular%2520target%2520for%2520cell-specific%2520aptamer%2520binding%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2019%26volume%3D141%26spage%3D10760%26epage%3D10769%26doi%3D10.1021%2Fjacs.9b03720" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref73"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref73'); return false;" data-citation="" class="refNumLink">73</a></strong><div class="NLM_citation" id="cit73"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perry, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muniz-Medina, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wetzel, L. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rebelatto, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hinrichs, M. J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bezabeh, B. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fleming, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dimasi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toader, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, A. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dixit, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osbourn, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coats, S. R.</span></span> <span> </span><span class="NLM_article-title">A biparatopic HER2-targeting antibody-drug conjugate induces tumor regression in primary models refractory to or ineligible for HER2-targeted therapy</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">117</span>– <span class="NLM_lpage">129</span>, <span class="refDoi"> DOI: 10.1016/j.ccell.2015.12.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1016%2Fj.ccell.2015.12.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=26766593" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC28Xmsl2lsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2016&pages=117-129&author=J.+Y.+Liauthor=S.+R.+Perryauthor=V.+Muniz-Medinaauthor=X.+Wangauthor=L.+K.+Wetzelauthor=M.+C.+Rebelattoauthor=M.+J.+M.+Hinrichsauthor=B.+Z.+Bezabehauthor=R.+L.+Flemingauthor=N.+Dimasiauthor=H.+Fengauthor=D.+Toaderauthor=A.+Q.+Yuanauthor=L.+Xuauthor=J.+Linauthor=C.+Gaoauthor=H.+Wuauthor=R.+Dixitauthor=J.+K.+Osbournauthor=S.+R.+Coats&title=A+biparatopic+HER2-targeting+antibody-drug+conjugate+induces+tumor+regression+in+primary+models+refractory+to+or+ineligible+for+HER2-targeted+therapy&doi=10.1016%2Fj.ccell.2015.12.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">A Biparatopic HER2-Targeting Antibody-Drug Conjugate Induces Tumor Regression in Primary Models Refractory to or Ineligible for HER2-Targeted Therapy</span></div><div class="casAuthors">Li, John Y.; Perry, Samuel R.; Muniz-Medina, Vanessa; Wang, Xinzhong; Wetzel, Leslie K.; Rebelatto, Marlon C.; Hinrichs, Mary Jane Masson; Bezabeh, Binyam Z.; Fleming, Ryan L.; Dimasi, Nazzareno; Feng, Hui; Toader, Dorin; Yuan, Andy Q.; Xu, Lan; Lin, Jia; Gao, Changshou; Wu, Herren; Dixit, Rakesh; Osbourn, Jane K.; Coats, Steven R.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">117-129</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Antibody-drug conjugate (ADC) which delivers cytotoxic drugs specifically into targeted cells through internalization and lysosomal trafficking has emerged as an effective cancer therapy.  We show that a bivalent biparatopic antibody targeting two non-overlapping epitopes on HER2 can induce HER2 receptor clustering, which in turn promotes robust internalization, lysosomal trafficking, and degrdn.  When conjugated with a tubulysin-based microtubule inhibitor, the biparatopic ADC demonstrates superior anti-tumor activity over ado-trastuzumab emtansine (T-DM1) in tumor models representing various patient subpopulations, including T-DM1 eligible, T-DM1 ineligible, and T-DM1 relapsed/refractory.  Our findings indicate that this biparatopic ADC has promising potential as an effective therapy for metastatic breast cancer and a broader patient population may benefit from this unique HER2-targeting ADC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp18b-9TGTzUbVg90H21EOLACvtfcHk0ljc3vE4-9x_-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xmsl2lsA%253D%253D&md5=bbd94b53427de32f8d636b63507dd139</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2015.12.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2015.12.008%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DJ.%2BY.%26aulast%3DPerry%26aufirst%3DS.%2BR.%26aulast%3DMuniz-Medina%26aufirst%3DV.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DWetzel%26aufirst%3DL.%2BK.%26aulast%3DRebelatto%26aufirst%3DM.%2BC.%26aulast%3DHinrichs%26aufirst%3DM.%2BJ.%2BM.%26aulast%3DBezabeh%26aufirst%3DB.%2BZ.%26aulast%3DFleming%26aufirst%3DR.%2BL.%26aulast%3DDimasi%26aufirst%3DN.%26aulast%3DFeng%26aufirst%3DH.%26aulast%3DToader%26aufirst%3DD.%26aulast%3DYuan%26aufirst%3DA.%2BQ.%26aulast%3DXu%26aufirst%3DL.%26aulast%3DLin%26aufirst%3DJ.%26aulast%3DGao%26aufirst%3DC.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DDixit%26aufirst%3DR.%26aulast%3DOsbourn%26aufirst%3DJ.%2BK.%26aulast%3DCoats%26aufirst%3DS.%2BR.%26atitle%3DA%2520biparatopic%2520HER2-targeting%2520antibody-drug%2520conjugate%2520induces%2520tumor%2520regression%2520in%2520primary%2520models%2520refractory%2520to%2520or%2520ineligible%2520for%2520HER2-targeted%2520therapy%26jtitle%3DCancer%2520Cell%26date%3D2016%26volume%3D29%26spage%3D117%26epage%3D129%26doi%3D10.1016%2Fj.ccell.2015.12.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref74"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref74'); return false;" data-citation="" class="refNumLink">74</a></strong><div class="NLM_citation" id="cit74"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Spangler, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neil, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abramovitch, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yarden, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lauffenburger, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wittrup, K. D.</span></span> <span> </span><span class="NLM_article-title">Combination antibody treatment down-regulates epidermal growth factor receptor by inhibiting endosomal recycling</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>107</i></span>,  <span class="NLM_fpage">13252</span>– <span class="NLM_lpage">13257</span>, <span class="refDoi"> DOI: 10.1073/pnas.0913476107</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1073%2Fpnas.0913476107" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=20616078" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht1GrurvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=107&publication_year=2010&pages=13252-13257&author=J.+B.+Spanglerauthor=J.+R.+Neilauthor=S.+Abramovitchauthor=Y.+Yardenauthor=F.+M.+Whiteauthor=D.+A.+Lauffenburgerauthor=K.+D.+Wittrup&title=Combination+antibody+treatment+down-regulates+epidermal+growth+factor+receptor+by+inhibiting+endosomal+recycling&doi=10.1073%2Fpnas.0913476107"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit74R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">Combination antibody treatment down-regulates epidermal growth factor receptor by inhibiting endosomal recycling</span></div><div class="casAuthors">Spangler, Jamie B.; Neil, Jason R.; Abramovitch, Sivan; Yarden, Yosef; White, Forest M.; Lauffenburger, Douglas A.; Wittrup, K. Dane</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">107</span>
        (<span class="NLM_cas:issue">30</span>),
    <span class="NLM_cas:pages">13252-13257, S13252/1-S13252/8</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Due to its common dysregulation in epithelial-based cancers and extensive characterization of its role in tumor growth, epidermal growth factor receptor (EGFR) is a highly validated target for anticancer therapies.  There has been particular interest in the development of monoclonal antibodies (mAbs) targeting EGFR, resulting in two approved mAb-based drugs and several others in clin. trials.  It has recently been reported that treatment with combinations of noncompetitive mAbs can induce receptor clustering, leading to synergistic receptor down-regulation.  We elucidate three key aspects of this phenomenon.  First, we show that highly potent combinations consisting of two noncompetitive mAbs that target EGFR domain 3 reduce surface receptor levels by up to 80% with a half-time of 0.5-5 h in both normal and transformed human cell lines to an extent inversely proportional to receptor d.  Second, we find the mechanism underlying down-regulation to be consistent with recycling inhibition.  Third, in contrast to the agonism assocd. with ligand-induced down-regulation, we demonstrate that mAb-induced down-regulation does not activate EGFR or its downstream effectors and it leads to synergistic redn. in migration and proliferation of cells that secrete autocrine ligand.  These new insights will aid in ongoing rational design of EGFR-targeted antibody therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoTrRDTQQgtl7Vg90H21EOLACvtfcHk0ljc3vE4-9x_-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht1GrurvI&md5=5da987aa7b91ebf421103a58232839d1</span></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0913476107&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0913476107%26sid%3Dliteratum%253Aachs%26aulast%3DSpangler%26aufirst%3DJ.%2BB.%26aulast%3DNeil%26aufirst%3DJ.%2BR.%26aulast%3DAbramovitch%26aufirst%3DS.%26aulast%3DYarden%26aufirst%3DY.%26aulast%3DWhite%26aufirst%3DF.%2BM.%26aulast%3DLauffenburger%26aufirst%3DD.%2BA.%26aulast%3DWittrup%26aufirst%3DK.%2BD.%26atitle%3DCombination%2520antibody%2520treatment%2520down-regulates%2520epidermal%2520growth%2520factor%2520receptor%2520by%2520inhibiting%2520endosomal%2520recycling%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2010%26volume%3D107%26spage%3D13252%26epage%3D13257%26doi%3D10.1073%2Fpnas.0913476107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref75"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref75'); return false;" data-citation="" class="refNumLink">75</a></strong><div class="NLM_citation" id="cit75"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Spiess, M.</span></span> <span> </span><span class="NLM_article-title">The asialoglycoprotein receptor: a model for endocytic transport receptors</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">1990</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">10009</span>– <span class="NLM_lpage">10018</span>, <span class="refDoi"> DOI: 10.1021/bi00495a001</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi00495a001" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADyaK3cXlvVymurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=1990&pages=10009-10018&author=M.+Spiess&title=The+asialoglycoprotein+receptor%3A+a+model+for+endocytic+transport+receptors&doi=10.1021%2Fbi00495a001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit75R"><div class="casContent"><span class="casTitleNuber">75</span><div class="casTitle"><span class="NLM_cas:atitle">The asialoglycoprotein receptor:  a model for endocytic transport receptors</span></div><div class="casAuthors">Spiess, Martin</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">1990</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">43</span>),
    <span class="NLM_cas:pages">10009-18</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    </div><div class="casAbstract">A review, with 129 refs., of the structure and function of the asialoglycoprotein receptor of hepatocytes as a model for endocytic transport receptors in general.  The topics included are: its role as an hepatic lectin; the existence of 2 genes and 2 proteins for the receptor; the organization of the genes for the receptor; and the carbohydrate-binding domain, the transmembrane signal-anchor domain, the cytoplasmic domain, and the connecting domain(s) of the receptor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAiexeMoNXGbVg90H21EOLACvtfcHk0ljc3vE4-9x_-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3cXlvVymurs%253D&md5=480a20b64d2c561fe9de0fb4b8528940</span></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1021%2Fbi00495a001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi00495a001%26sid%3Dliteratum%253Aachs%26aulast%3DSpiess%26aufirst%3DM.%26atitle%3DThe%2520asialoglycoprotein%2520receptor%253A%2520a%2520model%2520for%2520endocytic%2520transport%2520receptors%26jtitle%3DBiochemistry%26date%3D1990%26volume%3D29%26spage%3D10009%26epage%3D10018%26doi%3D10.1021%2Fbi00495a001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref76"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref76'); return false;" data-citation="" class="refNumLink">76</a></strong><div class="NLM_citation" id="cit76"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Park, E. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Unverzagt, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gabius, H.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baenziger, J. U.</span></span> <span> </span><span class="NLM_article-title">The asialoglycoprotein receptor clears glycoconjugates terminating with sialic acidα2,6GalNAc</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>102</i></span>,  <span class="NLM_fpage">17125</span>, <span class="refDoi"> DOI: 10.1073/pnas.0508537102</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1073%2Fpnas.0508537102" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=16286643" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BD2MXht1yksLbI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=2005&pages=17125&author=E.+I.+Parkauthor=Y.+Miauthor=C.+Unverzagtauthor=H.-J.+Gabiusauthor=J.+U.+Baenziger&title=The+asialoglycoprotein+receptor+clears+glycoconjugates+terminating+with+sialic+acid%CE%B12%2C6GalNAc&doi=10.1073%2Fpnas.0508537102"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit76R"><div class="casContent"><span class="casTitleNuber">76</span><div class="casTitle"><span class="NLM_cas:atitle">The asialoglycoprotein receptor clears glycoconjugates terminating with sialic acidα2,6GalNAc</span></div><div class="casAuthors">Park, Eric I.; Mi, Yiling; Unverzagt, Carlo; Gabius, Hans-Joachim; Baenziger, Jacques U.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">102</span>
        (<span class="NLM_cas:issue">47</span>),
    <span class="NLM_cas:pages">17125-17129</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Endogenous ligands have not, to date, been identified for the asialoglycoprotein receptor (ASGP-R), which is abundantly expressed by parenchymal cells in the liver of mammals.  On the basis of the rapid clearance of BSA bearing multiple chem. coupled sialic acid (Sia)α2,6GalNAcβ1,4GlcNAcβ1,2Man tetrasaccharides (SiaGGnM-BSA) from the circulation, and the ability of the ASGP-R hepatic lectin-1 subunit to bind SiaGGnM-BSA, we previously proposed that glycoproteins modified with structures terminating with Siaα2,6GalNAc may represent previously unrecognized examples of endogenous ligands for this receptor.  Here, we have taken a genetic approach using wild-type and ASGP-R-deficient mice to det. that the ASGP-R in vivo does indeed account for the rapid clearance of glycoconjugates terminating with Siaα2,6GalNAc.  We have also detd. that the ASGP-R is able to bind core-substituted oligosaccharides with the terminal sequence Siaα2,6Galβ1,4GlcNAc but not those with the terminal Siaα2,3Galβ1,4GlcNAc.  We propose that glycoproteins bearing terminals Siaα2,6GalNAc and Siaα2,6Gal are endogenous ligands for the ASGP-R, and that the ASGP-R helps to regulate the relative concn. of serum glycoproteins bearing α2,6-linked Sia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqP8PVdAuhz6bVg90H21EOLACvtfcHk0liaO8ko8mGzjQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXht1yksLbI&md5=739e0ef26daa7d1cb6675c1ab002e37a</span></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0508537102&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0508537102%26sid%3Dliteratum%253Aachs%26aulast%3DPark%26aufirst%3DE.%2BI.%26aulast%3DMi%26aufirst%3DY.%26aulast%3DUnverzagt%26aufirst%3DC.%26aulast%3DGabius%26aufirst%3DH.-J.%26aulast%3DBaenziger%26aufirst%3DJ.%2BU.%26atitle%3DThe%2520asialoglycoprotein%2520receptor%2520clears%2520glycoconjugates%2520terminating%2520with%2520sialic%2520acid%25CE%25B12%252C6GalNAc%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2005%26volume%3D102%26spage%3D17125%26doi%3D10.1073%2Fpnas.0508537102" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref77"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref77'); return false;" data-citation="" class="refNumLink">77</a></strong><div class="NLM_citation" id="cit77"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zimmermann, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karsten, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiesa, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boyce, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bettencourt, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hutabarat, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nochur, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vaishnaw, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gollob, J.</span></span> <span> </span><span class="NLM_article-title">Clinical proof of concept for a novel hepatocyte-targeting GalNAc-siRNA conjugate</span>. <i>Mol. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">71</span>– <span class="NLM_lpage">78</span>, <span class="refDoi"> DOI: 10.1016/j.ymthe.2016.10.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1016%2Fj.ymthe.2016.10.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=28129130" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmvFCmtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2017&pages=71-78&author=T.+S.+Zimmermannauthor=V.+Karstenauthor=A.+Chanauthor=J.+Chiesaauthor=M.+Boyceauthor=B.+R.+Bettencourtauthor=R.+Hutabaratauthor=S.+Nochurauthor=A.+Vaishnawauthor=J.+Gollob&title=Clinical+proof+of+concept+for+a+novel+hepatocyte-targeting+GalNAc-siRNA+conjugate&doi=10.1016%2Fj.ymthe.2016.10.019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit77R"><div class="casContent"><span class="casTitleNuber">77</span><div class="casTitle"><span class="NLM_cas:atitle">Clinical Proof of Concept for a Novel Hepatocyte-Targeting GalNAc-siRNA Conjugate</span></div><div class="casAuthors">Zimmermann, Tracy S.; Karsten, Verena; Chan, Amy; Chiesa, Joseph; Boyce, Malcolm; Bettencourt, Brian R.; Hutabarat, Renta; Nochur, Saraswathy; Vaishnaw, Akshay; Gollob, Jared</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Therapy</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">71-78</span>CODEN:
                <span class="NLM_cas:coden">MTOHCK</span>;
        ISSN:<span class="NLM_cas:issn">1525-0024</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Advancement of RNAi-based therapeutics depends on effective delivery to the site of protein synthesis.  Although i.v. administered, multi-component delivery vehicles have enabled small interfering RNA (siRNA) delivery and progression into clin. development, advances of single-component, systemic siRNA delivery have been challenging.  In pre-clin. models, attachment of a triantennary N-acetylgalactosamine (GalNAc) ligand to an siRNA mediates hepatocyte uptake via the asialoglycoprotein receptor enabling RNAi-mediated gene silencing.  In this phase 1 study, we assessed translation of this delivery approach by evaluating the safety, tolerability, pharmacokinetics, and pharmacodynamics of a GalNAc-siRNA conjugate, revusiran, targeting transthyretin (TTR).  Subjects received a placebo or ascending doses of revusiran s.c. ranging from 1.25-10 mg/kg in the single and 2.5-10 mg/kg in the multiple ascending dose phases.  Revusiran was generally well tolerated, with transient, mild to moderate injection site reactions the most common treatment-emergent adverse events.  Doses of 2.5-10 mg/kg revusiran elicited a significant redn. of serum TTR vs. the placebo (p < 0.01), with mean TTR redns. of approx. 90% obsd. with multiple dosing.  These results demonstrate translation of this novel delivery platform, enabling clin. development of s.c. administered GalNAc-siRNAs for liver-based diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoC6JF_TMrMX7Vg90H21EOLACvtfcHk0liaO8ko8mGzjQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmvFCmtb8%253D&md5=76ea4ba1335793017d436d8a3f78f643</span></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=10.1016%2Fj.ymthe.2016.10.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ymthe.2016.10.019%26sid%3Dliteratum%253Aachs%26aulast%3DZimmermann%26aufirst%3DT.%2BS.%26aulast%3DKarsten%26aufirst%3DV.%26aulast%3DChan%26aufirst%3DA.%26aulast%3DChiesa%26aufirst%3DJ.%26aulast%3DBoyce%26aufirst%3DM.%26aulast%3DBettencourt%26aufirst%3DB.%2BR.%26aulast%3DHutabarat%26aufirst%3DR.%26aulast%3DNochur%26aufirst%3DS.%26aulast%3DVaishnaw%26aufirst%3DA.%26aulast%3DGollob%26aufirst%3DJ.%26atitle%3DClinical%2520proof%2520of%2520concept%2520for%2520a%2520novel%2520hepatocyte-targeting%2520GalNAc-siRNA%2520conjugate%26jtitle%3DMol.%2520Ther.%26date%3D2017%26volume%3D25%26spage%3D71%26epage%3D78%26doi%3D10.1016%2Fj.ymthe.2016.10.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref78"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref78'); return false;" data-citation="" class="refNumLink">78</a></strong><div class="NLM_citation" id="cit78"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Uhlén, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fagerberg, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hallström, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindskog, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oksvold, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mardinoglu, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sivertsson, Å.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kampf, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sjöstedt, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asplund, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olsson, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edlund, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lundberg, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Navani, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szigyarto, C. A.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Odeberg, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Djureinovic, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takanen, J. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hober, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alm, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edqvist, P.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berling, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tegel, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulder, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rockberg, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nilsson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwenk, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamsten, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">von Feilitzen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forsberg, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Persson, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johansson, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwahlen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">von Heijne, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nielsen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pontén, F.</span></span> <span> </span><span class="NLM_article-title">Tissue-based map of the human proteome</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>347</i></span>,  <span class="NLM_fpage">1260419</span>, <span class="refDoi"> DOI: 10.1126/science.1260419</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1126%2Fscience.1260419" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=25613900" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A280%3ADC%252BC2MvptFakug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=347&publication_year=2015&pages=1260419&author=M.+Uhl%C3%A9nauthor=L.+Fagerbergauthor=B.+M.+Hallstr%C3%B6mauthor=C.+Lindskogauthor=P.+Oksvoldauthor=A.+Mardinogluauthor=%C3%85.+Sivertssonauthor=C.+Kampfauthor=E.+Sj%C3%B6stedtauthor=A.+Asplundauthor=I.+Olssonauthor=K.+Edlundauthor=E.+Lundbergauthor=S.+Navaniauthor=C.+A.-K.+Szigyartoauthor=J.+Odebergauthor=D.+Djureinovicauthor=J.+O.+Takanenauthor=S.+Hoberauthor=T.+Almauthor=P.-H.+Edqvistauthor=H.+Berlingauthor=H.+Tegelauthor=J.+Mulderauthor=J.+Rockbergauthor=P.+Nilssonauthor=J.+M.+Schwenkauthor=M.+Hamstenauthor=K.+von+Feilitzenauthor=M.+Forsbergauthor=L.+Perssonauthor=F.+Johanssonauthor=M.+Zwahlenauthor=G.+von+Heijneauthor=J.+Nielsenauthor=F.+Pont%C3%A9n&title=Tissue-based+map+of+the+human+proteome&doi=10.1126%2Fscience.1260419"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit78R"><div class="casContent"><span class="casTitleNuber">78</span><div class="casTitle"><span class="NLM_cas:atitle">Proteomics. Tissue-based map of the human proteome</span></div><div class="casAuthors">Uhlen Mathias; Fagerberg Linn; Oksvold Per; Sivertsson ¡ÑÜAsa; Lundberg Emma; Odeberg Jacob; Alm Tove; Nilsson Peter; Schwenk Jochen M; von Feilitzen Kalle; Forsberg Mattias; Persson Lukas; Johansson Fredric; Zwahlen Martin; Hallstrom Bjorn M; Lindskog Cecilia; Kampf Caroline; Asplund Anna; Olsson IngMarie; Djureinovic Dijana; Edqvist Per-Henrik; Ponten Fredrik; Mardinoglu Adil; Sjostedt Evelina; Edlund Karolina; Navani Sanjay; Szigyarto Cristina Al-Khalili; Takanen Jenny Ottosson; Hober Sophia; Berling Holger; Tegel Hanna; Rockberg Johan; Hamsten Marica; Mulder Jan; von Heijne Gunnar; Nielsen Jens</div><div class="citationInfo"><span class="NLM_cas:title">Science (New York, N.Y.)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">347</span>
        (<span class="NLM_cas:issue">6220</span>),
    <span class="NLM_cas:pages">1260419</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Resolving the molecular details of proteome variation in the different tissues and organs of the human body will greatly increase our knowledge of human biology and disease.  Here, we present a map of the human tissue proteome based on an integrated omics approach that involves quantitative transcriptomics at the tissue and organ level, combined with tissue microarray-based immunohistochemistry, to achieve spatial localization of proteins down to the single-cell level.  Our tissue-based analysis detected more than 90% of the putative protein-coding genes.  We used this approach to explore the human secretome, the membrane proteome, the druggable proteome, the cancer proteome, and the metabolic functions in 32 different tissues and organs.  All the data are integrated in an interactive Web-based database that allows exploration of individual proteins, as well as navigation of global expression patterns, in all major tissues and organs in the human body.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQT9YWouQZh5DEC-75aqa5AfW6udTcc2eabC-ZI9qDYhrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2MvptFakug%253D%253D&md5=95510888806e5feb6b8024ee2a417d1c</span></div><a href="/servlet/linkout?suffix=cit78&amp;dbid=16384&amp;doi=10.1126%2Fscience.1260419&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1260419%26sid%3Dliteratum%253Aachs%26aulast%3DUhl%25C3%25A9n%26aufirst%3DM.%26aulast%3DFagerberg%26aufirst%3DL.%26aulast%3DHallstr%25C3%25B6m%26aufirst%3DB.%2BM.%26aulast%3DLindskog%26aufirst%3DC.%26aulast%3DOksvold%26aufirst%3DP.%26aulast%3DMardinoglu%26aufirst%3DA.%26aulast%3DSivertsson%26aufirst%3D%25C3%2585.%26aulast%3DKampf%26aufirst%3DC.%26aulast%3DSj%25C3%25B6stedt%26aufirst%3DE.%26aulast%3DAsplund%26aufirst%3DA.%26aulast%3DOlsson%26aufirst%3DI.%26aulast%3DEdlund%26aufirst%3DK.%26aulast%3DLundberg%26aufirst%3DE.%26aulast%3DNavani%26aufirst%3DS.%26aulast%3DSzigyarto%26aufirst%3DC.%2BA.-K.%26aulast%3DOdeberg%26aufirst%3DJ.%26aulast%3DDjureinovic%26aufirst%3DD.%26aulast%3DTakanen%26aufirst%3DJ.%2BO.%26aulast%3DHober%26aufirst%3DS.%26aulast%3DAlm%26aufirst%3DT.%26aulast%3DEdqvist%26aufirst%3DP.-H.%26aulast%3DBerling%26aufirst%3DH.%26aulast%3DTegel%26aufirst%3DH.%26aulast%3DMulder%26aufirst%3DJ.%26aulast%3DRockberg%26aufirst%3DJ.%26aulast%3DNilsson%26aufirst%3DP.%26aulast%3DSchwenk%26aufirst%3DJ.%2BM.%26aulast%3DHamsten%26aufirst%3DM.%26aulast%3Dvon%2BFeilitzen%26aufirst%3DK.%26aulast%3DForsberg%26aufirst%3DM.%26aulast%3DPersson%26aufirst%3DL.%26aulast%3DJohansson%26aufirst%3DF.%26aulast%3DZwahlen%26aufirst%3DM.%26aulast%3Dvon%2BHeijne%26aufirst%3DG.%26aulast%3DNielsen%26aufirst%3DJ.%26aulast%3DPont%25C3%25A9n%26aufirst%3DF.%26atitle%3DTissue-based%2520map%2520of%2520the%2520human%2520proteome%26jtitle%3DScience%26date%3D2015%26volume%3D347%26spage%3D1260419%26doi%3D10.1126%2Fscience.1260419" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref79"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref79'); return false;" data-citation="" class="refNumLink">79</a></strong><div class="NLM_citation" id="cit79"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nakagawa, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amano, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mizushima, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamaguchi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamimoto, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nara, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Funao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakata, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsuda, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamada, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshimori, T.</span></span> <span> </span><span class="NLM_article-title">Autophagy defends cells against invading group a streptococcus</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>306</i></span>,  <span class="NLM_fpage">1037</span>– <span class="NLM_lpage">1040</span>, <span class="refDoi"> DOI: 10.1126/science.1103966</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1126%2Fscience.1103966" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=15528445" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BD2cXptFyjurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=306&publication_year=2004&pages=1037-1040&author=I.+Nakagawaauthor=A.+Amanoauthor=N.+Mizushimaauthor=A.+Yamamotoauthor=H.+Yamaguchiauthor=T.+Kamimotoauthor=A.+Naraauthor=J.+Funaoauthor=M.+Nakataauthor=K.+Tsudaauthor=S.+Hamadaauthor=T.+Yoshimori&title=Autophagy+defends+cells+against+invading+group+a+streptococcus&doi=10.1126%2Fscience.1103966"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit79R"><div class="casContent"><span class="casTitleNuber">79</span><div class="casTitle"><span class="NLM_cas:atitle">Autophagy Defends Cells Against Invading Group A Streptococcus</span></div><div class="casAuthors">Nakagawa, Ichiro; Amano, Atsuo; Mizushima, Noboru; Yamamoto, Akitsugu; Yamaguchi, Hitomi; Kamimoto, Takahiro; Nara, Atsuki; Funao, Junko; Nakata, Masanobu; Tsuda, Kayoko; Hamada, Shigeyuki; Yoshimori, Tamotsu</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">306</span>
        (<span class="NLM_cas:issue">5698</span>),
    <span class="NLM_cas:pages">1037-1040</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">We found that the autophagic machinery could effectively eliminate pathogenic group A Streptococcus (GAS) within nonphagocytic cells.  After escaping from endosomes into the cytoplasm, GAS became enveloped by autophagosome-like compartments and were killed upon fusion of these compartments with lysosomes.  In autophagy-deficient Atg5-/- cells, GAS survived, multiplied, and were released from the cells.  Thus, the autophagic machinery can act as an innate defense system against invading pathogens.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqtqiNA9WkovbVg90H21EOLACvtfcHk0lgZPXMWNvOuwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXptFyjurc%253D&md5=59cdd8514ac2752c83c2a2aa2d0ef8f5</span></div><a href="/servlet/linkout?suffix=cit79&amp;dbid=16384&amp;doi=10.1126%2Fscience.1103966&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1103966%26sid%3Dliteratum%253Aachs%26aulast%3DNakagawa%26aufirst%3DI.%26aulast%3DAmano%26aufirst%3DA.%26aulast%3DMizushima%26aufirst%3DN.%26aulast%3DYamamoto%26aufirst%3DA.%26aulast%3DYamaguchi%26aufirst%3DH.%26aulast%3DKamimoto%26aufirst%3DT.%26aulast%3DNara%26aufirst%3DA.%26aulast%3DFunao%26aufirst%3DJ.%26aulast%3DNakata%26aufirst%3DM.%26aulast%3DTsuda%26aufirst%3DK.%26aulast%3DHamada%26aufirst%3DS.%26aulast%3DYoshimori%26aufirst%3DT.%26atitle%3DAutophagy%2520defends%2520cells%2520against%2520invading%2520group%2520a%2520streptococcus%26jtitle%3DScience%26date%3D2004%26volume%3D306%26spage%3D1037%26epage%3D1040%26doi%3D10.1126%2Fscience.1103966" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref80"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref80'); return false;" data-citation="" class="refNumLink">80</a></strong><div class="NLM_citation" id="cit80"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ito, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saito, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nozawa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujii, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sawa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inoue, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsunaga, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akashi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hashimoto, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aikawa, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takahashi, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sagara, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Komatsu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akaike, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakagawa, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arimoto, H.</span></span> <span> </span><span class="NLM_article-title">Endogenous nitrated nucleotide is a key mediator of autophagy and innate defense against bacteria</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">794</span>– <span class="NLM_lpage">804</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2013.10.024</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1016%2Fj.molcel.2013.10.024" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=24268578" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvVKgsb3L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2013&pages=794-804&author=C.+Itoauthor=Y.+Saitoauthor=T.+Nozawaauthor=S.+Fujiiauthor=T.+Sawaauthor=H.+Inoueauthor=T.+Matsunagaauthor=S.+Khanauthor=S.+Akashiauthor=R.+Hashimotoauthor=C.+Aikawaauthor=E.+Takahashiauthor=H.+Sagaraauthor=M.+Komatsuauthor=K.+Tanakaauthor=T.+Akaikeauthor=I.+Nakagawaauthor=H.+Arimoto&title=Endogenous+nitrated+nucleotide+is+a+key+mediator+of+autophagy+and+innate+defense+against+bacteria&doi=10.1016%2Fj.molcel.2013.10.024"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit80R"><div class="casContent"><span class="casTitleNuber">80</span><div class="casTitle"><span class="NLM_cas:atitle">Endogenous Nitrated Nucleotide Is a Key Mediator of Autophagy and Innate Defense against Bacteria</span></div><div class="casAuthors">Ito, Chiaki; Saito, Yohei; Nozawa, Takashi; Fujii, Shigemoto; Sawa, Tomohiro; Inoue, Hirofumi; Matsunaga, Tetsuro; Khan, Shahzada; Akashi, Soichiro; Hashimoto, Ryota; Aikawa, Chihiro; Takahashi, Eriko; Sagara, Hiroshi; Komatsu, Masaaki; Tanaka, Keiji; Akaike, Takaaki; Nakagawa, Ichiro; Arimoto, Hirokazu</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">794-804</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Autophagy is a cellular self-catabolic process wherein organelles, macromols., and invading microbes are sequestered in autophagosomes that fuse with lysosomes.  In this study, we uncover the role of nitric oxide (NO) as a signaling mol. for autophagy induction via its downstream mediator, 8-nitroguanosine 3',5'-cyclic monophosphate (8-nitro-cGMP).  We found that 8-nitro-cGMP-induced autophagy is mediated by Lys63-linked polyubiquitination and that endogenous 8-nitro-cGMP promotes autophagic exclusion of invading group A Streptococcus (GAS) from cells.  8-Nitro-cGMP can modify Cys residues by S-guanylation of proteins.  We showed that intracellular GAS is modified with S-guanylation extensively in autophagosomes-like vacuoles, suggesting the role of S-guanylation as a marker for selective autophagic degrdn.  This finding is supported by the fact that S-guanylated bacteria were selectively marked with polyubiquitin, a known mol. tag for selective transport to autophagosomes.  These results collectively indicate that 8-nitro-cGMP plays a crucial role in cytoprotection during bacterial infections or inflammations via autophagy upregulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrnl0efbVmHOrVg90H21EOLACvtfcHk0lgZPXMWNvOuwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvVKgsb3L&md5=713814689a0ea39dceb41f5767545fcb</span></div><a href="/servlet/linkout?suffix=cit80&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2013.10.024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2013.10.024%26sid%3Dliteratum%253Aachs%26aulast%3DIto%26aufirst%3DC.%26aulast%3DSaito%26aufirst%3DY.%26aulast%3DNozawa%26aufirst%3DT.%26aulast%3DFujii%26aufirst%3DS.%26aulast%3DSawa%26aufirst%3DT.%26aulast%3DInoue%26aufirst%3DH.%26aulast%3DMatsunaga%26aufirst%3DT.%26aulast%3DKhan%26aufirst%3DS.%26aulast%3DAkashi%26aufirst%3DS.%26aulast%3DHashimoto%26aufirst%3DR.%26aulast%3DAikawa%26aufirst%3DC.%26aulast%3DTakahashi%26aufirst%3DE.%26aulast%3DSagara%26aufirst%3DH.%26aulast%3DKomatsu%26aufirst%3DM.%26aulast%3DTanaka%26aufirst%3DK.%26aulast%3DAkaike%26aufirst%3DT.%26aulast%3DNakagawa%26aufirst%3DI.%26aulast%3DArimoto%26aufirst%3DH.%26atitle%3DEndogenous%2520nitrated%2520nucleotide%2520is%2520a%2520key%2520mediator%2520of%2520autophagy%2520and%2520innate%2520defense%2520against%2520bacteria%26jtitle%3DMol.%2520Cell%26date%3D2013%26volume%3D52%26spage%3D794%26epage%3D804%26doi%3D10.1016%2Fj.molcel.2013.10.024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref81"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref81'); return false;" data-citation="" class="refNumLink">81</a></strong><div class="NLM_citation" id="cit81"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, S.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawabata, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, Y.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Omori, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamasaki, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kusaba, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwamoto, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arimoto, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noda, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Y.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshimori, T.</span></span> <span> </span><span class="NLM_article-title">Endothelial cells are intrinsically defective in xenophagy of Streptococcus pyogenes</span>. <i>PLoS Pathog.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">e1006444</span>, <span class="refDoi"> DOI: 10.1371/journal.ppat.1006444</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1371%2Fjournal.ppat.1006444" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=28683091" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC1cXis1eksbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&pages=e1006444&author=S.-L.+Luauthor=T.+Kawabataauthor=Y.-L.+Chengauthor=H.+Omoriauthor=M.+Hamasakiauthor=T.+Kusabaauthor=R.+Iwamotoauthor=H.+Arimotoauthor=T.+Nodaauthor=Y.-S.+Linauthor=T.+Yoshimori&title=Endothelial+cells+are+intrinsically+defective+in+xenophagy+of+Streptococcus+pyogenes&doi=10.1371%2Fjournal.ppat.1006444"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit81R"><div class="casContent"><span class="casTitleNuber">81</span><div class="casTitle"><span class="NLM_cas:atitle">Endothelial cells are intrinsically defective in xenophagy of Streptococcus pyogenes</span></div><div class="casAuthors">Lu, Shiou-Ling; Kawabata, Tsuyoshi; Cheng, Yi-Lin; Omori, Hiroko; Hamasaki, Maho; Kusaba, Tatsuya; Iwamoto, Ryo; Arimoto, Hirokazu; Noda, Takeshi; Lin, Yee-Shin; Yoshimori, Tamotsu</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Pathogens</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">e1006444/1-e1006444/19</span>CODEN:
                <span class="NLM_cas:coden">PPLACN</span>;
        ISSN:<span class="NLM_cas:issn">1553-7374</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Group A Streptococcus (GAS) is deleterious pathogenic bacteria whose interaction with blood vessels leads to life-threatening bacteremia.  Although xenophagy, a special form of autophagy, eliminates invading GAS in epithelial cells, we found that GAS could survive and multiply in endothelial cells.  Endothelial cells were competent in starvation-induced autophagy, but failed to form double-membrane structures surrounding GAS, an essential step in xenophagy.  This deficiency stemmed from reduced recruitment of ubiquitin and several core autophagy proteins in endothelial cells, as demonstrated by the fact that it could be rescued by exogenous coating of GAS with ubiquitin.  The defect was assocd. with reduced NO-mediated ubiquitin signaling.  Therefore, we propose that the lack of efficient clearance of GAS in endothelial cells is caused by their intrinsic inability to target GAS with ubiquitin to promote autophagosome biogenesis for xenophagy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPO60rYg1GYLVg90H21EOLACvtfcHk0lhajW09wwwHRg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXis1eksbg%253D&md5=86844f09ff46cd74b91836f37d97693f</span></div><a href="/servlet/linkout?suffix=cit81&amp;dbid=16384&amp;doi=10.1371%2Fjournal.ppat.1006444&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.ppat.1006444%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DS.-L.%26aulast%3DKawabata%26aufirst%3DT.%26aulast%3DCheng%26aufirst%3DY.-L.%26aulast%3DOmori%26aufirst%3DH.%26aulast%3DHamasaki%26aufirst%3DM.%26aulast%3DKusaba%26aufirst%3DT.%26aulast%3DIwamoto%26aufirst%3DR.%26aulast%3DArimoto%26aufirst%3DH.%26aulast%3DNoda%26aufirst%3DT.%26aulast%3DLin%26aufirst%3DY.-S.%26aulast%3DYoshimori%26aufirst%3DT.%26atitle%3DEndothelial%2520cells%2520are%2520intrinsically%2520defective%2520in%2520xenophagy%2520of%2520Streptococcus%2520pyogenes%26jtitle%3DPLoS%2520Pathog.%26date%3D2017%26volume%3D13%26spage%3De1006444%26doi%3D10.1371%2Fjournal.ppat.1006444" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref82"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref82'); return false;" data-citation="" class="refNumLink">82</a></strong><div class="NLM_citation" id="cit82"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sawa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zaki, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okamoto, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akuta, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tokutomi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim-Mitsuyama, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ihara, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobayashi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujii, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arimoto, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akaike, T.</span></span> <span> </span><span class="NLM_article-title">Protein <i>S</i>-guanylation by the biological signal 8-nitroguanosine 3′,5′-cyclic monophosphate</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">727</span>– <span class="NLM_lpage">735</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2007.33</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1038%2Fnchembio.2007.33" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=17906641" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFygs73F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2007&pages=727-735&author=T.+Sawaauthor=M.+H.+Zakiauthor=T.+Okamotoauthor=T.+Akutaauthor=Y.+Tokutomiauthor=S.+Kim-Mitsuyamaauthor=H.+Iharaauthor=A.+Kobayashiauthor=M.+Yamamotoauthor=S.+Fujiiauthor=H.+Arimotoauthor=T.+Akaike&title=Protein+S-guanylation+by+the+biological+signal+8-nitroguanosine+3%E2%80%B2%2C5%E2%80%B2-cyclic+monophosphate&doi=10.1038%2Fnchembio.2007.33"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit82R"><div class="casContent"><span class="casTitleNuber">82</span><div class="casTitle"><span class="NLM_cas:atitle">Protein S-guanylation by the biological signal 8-nitroguanosine 3',5'-cyclic monophosphate</span></div><div class="casAuthors">Sawa, Tomohiro; Zaki, Mohammad Hasan; Okamoto, Tatsuya; Akuta, Teruo; Tokutomi, Yoshiko; Kim-Mitsuyama, Shokei; Ihara, Hideshi; Kobayashi, Akira; Yamamoto, Masayuki; Fujii, Shigemoto; Arimoto, Hirokazu; Akaike, Takaaki</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">727-735</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The signaling pathway of nitric oxide (NO) depends mainly on guanosine 3',5'-cyclic monophosphate (cGMP).  Here we report the formation and chem. biol. of a nitrated deriv. of cGMP, 8-nitroguanosine 3',5'-cyclic monophosphate (8-nitro-cGMP), in NO-mediated signal transduction.  Immunocytochem. demonstrated marked 8-nitro-cGMP prodn. in various cultured cells in an NO-dependent manner.  This finding was confirmed by HPLC plus electrochem. detection and tandem mass spectrometry.  8-Nitro-cGMP activated cGMP-dependent protein kinase (PKG) and showed unique redox-active properties independent of cGMP activity.  Formation of protein Cys-cGMP adducts by 8-nitro-cGMP was identified as a new post-translational modification, which we call protein S-guanylation.  8-Nitro-cGMP seems to regulate the redox-sensor signaling protein Keap1, via S-guanylation of the highly nucleophilic cysteine sulfhydryls of Keap1.  This study reveals 8-nitro-cGMP to be a second messenger of NO and sheds light on new areas of the physiol. and chem. biol. of signal transduction by NO.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGruENgL603fArVg90H21EOLACvtfcHk0lhajW09wwwHRg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFygs73F&md5=1915b89720a9bf353c63ebe621ac58bd</span></div><a href="/servlet/linkout?suffix=cit82&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2007.33&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2007.33%26sid%3Dliteratum%253Aachs%26aulast%3DSawa%26aufirst%3DT.%26aulast%3DZaki%26aufirst%3DM.%2BH.%26aulast%3DOkamoto%26aufirst%3DT.%26aulast%3DAkuta%26aufirst%3DT.%26aulast%3DTokutomi%26aufirst%3DY.%26aulast%3DKim-Mitsuyama%26aufirst%3DS.%26aulast%3DIhara%26aufirst%3DH.%26aulast%3DKobayashi%26aufirst%3DA.%26aulast%3DYamamoto%26aufirst%3DM.%26aulast%3DFujii%26aufirst%3DS.%26aulast%3DArimoto%26aufirst%3DH.%26aulast%3DAkaike%26aufirst%3DT.%26atitle%3DProtein%2520S-guanylation%2520by%2520the%2520biological%2520signal%25208-nitroguanosine%25203%25E2%2580%25B2%252C5%25E2%2580%25B2-cyclic%2520monophosphate%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2007%26volume%3D3%26spage%3D727%26epage%3D735%26doi%3D10.1038%2Fnchembio.2007.33" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref83"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref83'); return false;" data-citation="" class="refNumLink">83</a></strong><div class="NLM_citation" id="cit83"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Takahashi, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arimoto, H.</span></span> <span> </span><span class="NLM_article-title">Targeting selective autophagy by AUTAC degraders</span>. <i>Autophagy</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">765</span>– <span class="NLM_lpage">766</span>, <span class="refDoi"> DOI: 10.1080/15548627.2020.1718362</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1080%2F15548627.2020.1718362" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=31958028" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BB3cXht1Kjurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2020&pages=765-766&author=D.+Takahashiauthor=H.+Arimoto&title=Targeting+selective+autophagy+by+AUTAC+degraders&doi=10.1080%2F15548627.2020.1718362"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit83R"><div class="casContent"><span class="casTitleNuber">83</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting selective autophagy by AUTAC degraders</span></div><div class="casAuthors">Takahashi, Daiki; Arimoto, Hirokazu</div><div class="citationInfo"><span class="NLM_cas:title">Autophagy</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">765-766</span>CODEN:
                <span class="NLM_cas:coden">AUTOC9</span>;
        ISSN:<span class="NLM_cas:issn">1554-8635</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">Targeted degrdn. is a promising new modality in drug discovery that makes it possible to reduce intracellular protein levels with small mols.  It is a complementary approach to the conventional protein knockdown typically used in labs. and may offer a way to approach the currently undruggable human proteome.  Recently, the first autophagy-mediated degraders, called AUTACs, were developed based on observations in a xenophagy study.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpRAzgVZeo_9bVg90H21EOLACvtfcHk0ljDwZuoRhTOyQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXht1Kjurg%253D&md5=b859e2173c1da6072a0662e58c8af80d</span></div><a href="/servlet/linkout?suffix=cit83&amp;dbid=16384&amp;doi=10.1080%2F15548627.2020.1718362&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F15548627.2020.1718362%26sid%3Dliteratum%253Aachs%26aulast%3DTakahashi%26aufirst%3DD.%26aulast%3DArimoto%26aufirst%3DH.%26atitle%3DTargeting%2520selective%2520autophagy%2520by%2520AUTAC%2520degraders%26jtitle%3DAutophagy%26date%3D2020%26volume%3D16%26spage%3D765%26epage%3D766%26doi%3D10.1080%2F15548627.2020.1718362" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref84"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref84'); return false;" data-citation="" class="refNumLink">84</a></strong><div class="NLM_citation" id="cit84"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sin, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M. Q. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bornmann, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">The anti-angiogenic agent fumagillin covalently binds and inhibits the methionine aminopeptidase, MetAP-2</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>94</i></span>,  <span class="NLM_fpage">6099</span>– <span class="NLM_lpage">6103</span>, <span class="refDoi"> DOI: 10.1073/pnas.94.12.6099</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1073%2Fpnas.94.12.6099" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=9177176" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADyaK2sXjvFCnsb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=94&publication_year=1997&pages=6099-6103&author=N.+Sinauthor=L.+Mengauthor=M.+Q.+W.+Wangauthor=J.+J.+Wenauthor=W.+G.+Bornmannauthor=C.+M.+Crews&title=The+anti-angiogenic+agent+fumagillin+covalently+binds+and+inhibits+the+methionine+aminopeptidase%2C+MetAP-2&doi=10.1073%2Fpnas.94.12.6099"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit84R"><div class="casContent"><span class="casTitleNuber">84</span><div class="casTitle"><span class="NLM_cas:atitle">The anti-angiogenic agent fumagillin covalently binds and inhibits the methionine aminopeptidase, MetAP-2</span></div><div class="casAuthors">Sin, Ny; Meng, Lihao; Wang, Margaret Q. W.; Wen, James J.; Bornmann, William G.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">94</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">6099-6103</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The inhibition of new blood vessel formation (angiogenesis) is an effective means of limiting both the size and metastasis of solid tumors.  The leading antiangiogenic compd., TNP-470, has proven to be effective in in vitro and in animal model studies, and is currently being tested in phase III antitumor clin. trials.  Despite many detailed pharmacol. studies, little is known of the mol. mode of action of TNP-470.  Using a deriv. of the TNP-470 parent compd., the fungal metabolite, fumagillin, we have purified a mammalian protein that is selectively and covalently bound by this natural product.  This fumagillin binding protein was found to be a metalloprotease, methionine aminopeptidase (MetAP-2), that is highly conserved between human and Saccharomyces cerevisiae.  In the absence of MetAP-1, a distantly related methionine aminopeptidase, MetAP-2 function is essential for vegetative growth in yeast.  We demonstrate that fumagillin selectively inhibits the S. cerevisiae MetAP-2 protein in vivo.  The binding is highly specific as judged by the failure of fumagillin to inhibit MetAP-1 in vivo.  Hence, these results identify MetAP-2 as an important target of study in the anal. of the potent biol. activities of fumagillin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHTZpt2wUOq7Vg90H21EOLACvtfcHk0ljDwZuoRhTOyQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXjvFCnsb0%253D&md5=511f1751897559a60ccd9f814315c355</span></div><a href="/servlet/linkout?suffix=cit84&amp;dbid=16384&amp;doi=10.1073%2Fpnas.94.12.6099&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.94.12.6099%26sid%3Dliteratum%253Aachs%26aulast%3DSin%26aufirst%3DN.%26aulast%3DMeng%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DM.%2BQ.%2BW.%26aulast%3DWen%26aufirst%3DJ.%2BJ.%26aulast%3DBornmann%26aufirst%3DW.%2BG.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DThe%2520anti-angiogenic%2520agent%2520fumagillin%2520covalently%2520binds%2520and%2520inhibits%2520the%2520methionine%2520aminopeptidase%252C%2520MetAP-2%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1997%26volume%3D94%26spage%3D6099%26epage%3D6103%26doi%3D10.1073%2Fpnas.94.12.6099" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref85"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref85'); return false;" data-citation="" class="refNumLink">85</a></strong><div class="NLM_citation" id="cit85"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tucker, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheppard, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lou, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKeegan, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lesniewski, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davidsen, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bell, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span> <span> </span><span class="NLM_article-title">Ectopic expression of methionine aminopeptidase-2 causes cell transformation and stimulates proliferation</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">3967</span>– <span class="NLM_lpage">3976</span>, <span class="refDoi"> DOI: 10.1038/onc.2008.14</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1038%2Fonc.2008.14" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=18264137" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BD1cXns1Onsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2008&pages=3967-3976&author=L.+A.+Tuckerauthor=Q.+Zhangauthor=G.+S.+Sheppardauthor=P.+Louauthor=F.+Jiangauthor=E.+McKeeganauthor=R.+Lesniewskiauthor=S.+K.+Davidsenauthor=R.+L.+Bellauthor=J.+Wang&title=Ectopic+expression+of+methionine+aminopeptidase-2+causes+cell+transformation+and+stimulates+proliferation&doi=10.1038%2Fonc.2008.14"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit85R"><div class="casContent"><span class="casTitleNuber">85</span><div class="casTitle"><span class="NLM_cas:atitle">Ectopic expression of methionine aminopeptidase-2 causes cell transformation and stimulates proliferation</span></div><div class="casAuthors">Tucker, L. A.; Zhang, Q.; Sheppard, G. S.; Lou, P.; Jiang, F.; McKeegan, E.; Lesniewski, R.; Davidsen, S. K.; Bell, R. L.; Wang, J.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">28</span>),
    <span class="NLM_cas:pages">3967-3976</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Methionine aminopeptidase-2 (MetAP2) processes N-terminal methionine from nascent cellular proteins.  Inhibition of MetAP2 has been shown to block angiogenesis and suppress tumor growth in preclin. tumor models.  However, the biol. role of MetAP2 in cancer is not well understood.  We examd. the effect of three distinct chem. classes of MetAP2 inhibitors on the growth of a panel of human cancer cells in vitro.  All MetAP2 inhibitors caused inhibition of tumor cell growth in both anchorage-dependent and, particularly, in anchorage-independent manner.  These data prompted us to examine the possible roles of MetAP2 in cancers.  Ectopic expression of MetAP2 in NIH-3T3 cells caused transformation, evidenced by the formation of foci in monolayer culture and growth of large colonies in soft agar.  Overexpression of MetAP2 in an immortalized bronchial epithelial cell line NL20 accelerated growth.  These phenotypes induced by the overexpression of MetAP2 were reversed by the treatment with MetAP2 inhibitors, indicating that the catalytic function of MetAP2 was essential.  Accordingly, overexpression of a catalytically inactive MetAP2 resulted in growth retardation of HT1080 tumor cells, suggesting a dominant-neg. role of the inactive MetAP2 mutant.  Finally, we analyzed the expression of MetAP2 in patient cancer samples by immunohistochem.  Moderate-to-high staining was identified in the majority of breast, colon, lung, ovarian and prostate carcinomas examd.  These data suggest that MetAP2 plays an important role in tumor cell growth and may contribute to tumorigenesis.  Oncogene (2008) 27, 3967-3976; doi:10.1038/onc.2008.14; published online 11 Feb. 2008.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobZn65gMTSUrVg90H21EOLACvtfcHk0ljDwZuoRhTOyQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXns1Onsr8%253D&md5=564ea8d498c043534100cf0a72136f4d</span></div><a href="/servlet/linkout?suffix=cit85&amp;dbid=16384&amp;doi=10.1038%2Fonc.2008.14&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fonc.2008.14%26sid%3Dliteratum%253Aachs%26aulast%3DTucker%26aufirst%3DL.%2BA.%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DSheppard%26aufirst%3DG.%2BS.%26aulast%3DLou%26aufirst%3DP.%26aulast%3DJiang%26aufirst%3DF.%26aulast%3DMcKeegan%26aufirst%3DE.%26aulast%3DLesniewski%26aufirst%3DR.%26aulast%3DDavidsen%26aufirst%3DS.%2BK.%26aulast%3DBell%26aufirst%3DR.%2BL.%26aulast%3DWang%26aufirst%3DJ.%26atitle%3DEctopic%2520expression%2520of%2520methionine%2520aminopeptidase-2%2520causes%2520cell%2520transformation%2520and%2520stimulates%2520proliferation%26jtitle%3DOncogene%26date%3D2008%26volume%3D27%26spage%3D3967%26epage%3D3976%26doi%3D10.1038%2Fonc.2008.14" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref86"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref86'); return false;" data-citation="" class="refNumLink">86</a></strong><div class="NLM_citation" id="cit86"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Guruceaga, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perez-Cuesta, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abad-Diaz de Cerio, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gonzalez, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alonso, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hernando, F. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramirez-Garcia, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rementeria, A.</span></span> <span> </span><span class="NLM_article-title">Fumagillin, a mycotoxin of aspergillus fumigatus: biosynthesis, biological activities, detection, and applications</span>. <i>Toxins</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">7</span>, <span class="refDoi"> DOI: 10.3390/toxins12010007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.3390%2Ftoxins12010007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhslCgtb%252FK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2020&pages=7&author=X.+Guruceagaauthor=U.+Perez-Cuestaauthor=A.+Abad-Diaz+de+Cerioauthor=O.+Gonzalezauthor=R.+M.+Alonsoauthor=F.+L.+Hernandoauthor=A.+Ramirez-Garciaauthor=A.+Rementeria&title=Fumagillin%2C+a+mycotoxin+of+aspergillus+fumigatus%3A+biosynthesis%2C+biological+activities%2C+detection%2C+and+applications&doi=10.3390%2Ftoxins12010007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit86R"><div class="casContent"><span class="casTitleNuber">86</span><div class="casTitle"><span class="NLM_cas:atitle">Fumagillin, a mycotoxin of Aspergillus fumigatus: biosynthesis, biological activities, detection, and applications</span></div><div class="casAuthors">Guruceaga, Xabier; Perez-Cuesta, Uxue; de Cerio, Ana Abad-Diaz; Gonzalez, Oskar; Alonso, Rosa M.; Hernando, Fernando Luis; Ramirez-Garcia, Andoni; Rementeria, Aitor</div><div class="citationInfo"><span class="NLM_cas:title">Toxins</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">7</span>CODEN:
                <span class="NLM_cas:coden">TOXIB7</span>;
        ISSN:<span class="NLM_cas:issn">2072-6651</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Fumagillin is a mycotoxin produced, above all, by the saprophytic filamentous fungus Aspergillus fumigatus.  This mold is an opportunistic pathogen that can cause invasive aspergillosis, a disease that has high mortality rates linked to it.  Its ability to adapt to environmental stresses through the prodn. of secondary metabolites, including several mycotoxins (gliotoxin, fumagillin, pseurotin A, etc.) also seem to play an important role in causing these infections.  Since the discovery of the A. fumigatus fumagillin in 1949, many studies have focused on this toxin and in this review we gather all the information currently available.  First of all, the structural characteristics of this mycotoxin and the different methods developed for its detn. are given in detail.  Then, the biosynthetic gene cluster and the metabolic pathway involved in its prodn. and regulation are explained.  The activity of fumagillin on its target, the methionine aminopeptidase type 2 (MetAP2) enzyme, and the effects of blocking this enzyme in the host are also described.  Finally, the applications that this toxin and its derivs. have in different fields, such as the treatment of cancer and its microsporicidal activity in the treatment of honeybee hive infections with Nosema spp., are reviewed.  Therefore, this work offers a complete review of all the information currently related to the fumagillin mycotoxin secreted by A. fumigatus, important because of its role in the fungal infection process but also because it has many other applications, notably in beekeeping, the treatment of infectious diseases, and in oncol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkKqz-kSI757Vg90H21EOLACvtfcHk0lh-7iT_VfQvpw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhslCgtb%252FK&md5=51fd0841db089c313716194ef926c8de</span></div><a href="/servlet/linkout?suffix=cit86&amp;dbid=16384&amp;doi=10.3390%2Ftoxins12010007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Ftoxins12010007%26sid%3Dliteratum%253Aachs%26aulast%3DGuruceaga%26aufirst%3DX.%26aulast%3DPerez-Cuesta%26aufirst%3DU.%26aulast%3DAbad-Diaz%2Bde%2BCerio%26aufirst%3DA.%26aulast%3DGonzalez%26aufirst%3DO.%26aulast%3DAlonso%26aufirst%3DR.%2BM.%26aulast%3DHernando%26aufirst%3DF.%2BL.%26aulast%3DRamirez-Garcia%26aufirst%3DA.%26aulast%3DRementeria%26aufirst%3DA.%26atitle%3DFumagillin%252C%2520a%2520mycotoxin%2520of%2520aspergillus%2520fumigatus%253A%2520biosynthesis%252C%2520biological%2520activities%252C%2520detection%252C%2520and%2520applications%26jtitle%3DToxins%26date%3D2020%26volume%3D12%26spage%3D7%26doi%3D10.3390%2Ftoxins12010007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref87"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref87'); return false;" data-citation="" class="refNumLink">87</a></strong><div class="NLM_citation" id="cit87"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Heinrich, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seenisamy, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becker, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blume, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bomke, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dietz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eckert, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friese-Hamim, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gunera, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leuthner, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Musil, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pfalzgraf, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rohdich, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siegl, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spuck, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wegener, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zenke, F. T.</span></span> <span> </span><span class="NLM_article-title">Identification of methionine aminopeptidase-2 (MetAP-2) inhibitor M8891: a clinical compound for the treatment of cancer</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">11119</span>– <span class="NLM_lpage">11134</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b01070</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01070" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitFKksLbL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=11119-11134&author=T.+Heinrichauthor=J.+Seenisamyauthor=F.+Beckerauthor=B.+Blumeauthor=J.+Bomkeauthor=M.+Dietzauthor=U.+Eckertauthor=M.+Friese-Hamimauthor=J.+Guneraauthor=K.+Hansenauthor=B.+Leuthnerauthor=D.+Musilauthor=J.+Pfalzgrafauthor=F.+Rohdichauthor=C.+Sieglauthor=D.+Spuckauthor=A.+Wegenerauthor=F.+T.+Zenke&title=Identification+of+methionine+aminopeptidase-2+%28MetAP-2%29+inhibitor+M8891%3A+a+clinical+compound+for+the+treatment+of+cancer&doi=10.1021%2Facs.jmedchem.9b01070"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit87R"><div class="casContent"><span class="casTitleNuber">87</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of Methionine Aminopeptidase-2 (MetAP-2) Inhibitor M8891: A Clinical Compound for the Treatment of Cancer</span></div><div class="casAuthors">Heinrich, Timo; Seenisamy, Jeyaprakashnarayanan; Becker, Frank; Blume, Beatrix; Bomke, Joerg; Dietz, Melanie; Eckert, Uwe; Friese-Hamim, Manja; Gunera, Jakub; Hansen, Kerrin; Leuthner, Birgitta; Musil, Djordje; Pfalzgraf, Jens; Rohdich, Felix; Siegl, Christian; Spuck, Dieter; Wegener, Ansgar; Zenke, Frank T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">11119-11134</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The recently disclosed next generation of reversible, selective, and potent MetAP-2 inhibitors introduced a cyclic tartronic diamide scaffold.  However, the lead compd. 1a suffered from enterohepatic circulation, preventing further development.  Nevertheless, 1a served as a starting point for further optimization.  Maintaining potent antiproliferation activity, while improving other compd. properties, enabled the generation of an attractive array of new MetAP-2 inhibitors.  The most promising derivs. were identified by a multiparameter anal. of the compd. properties.  Essential for the efficient selection of candidates with in vivo activity was the identification of mols. with a long residence time on the target protein, high permeability, and low efflux ratio not only in Caco-2 but also in the MDR-MDCK cell line.  Orally bioavailable, potent, and reversible MetAP-2 inhibitors impede the growth of primary endothelial cells and demonstrated antitumoral activity in mouse models.  This assessment led to the nomination of the clin. development compd. M8891, which is currently in phase I clin. testing in oncol. patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoiUCw42raiq7Vg90H21EOLACvtfcHk0lh-7iT_VfQvpw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitFKksLbL&md5=8a452bfe7918123546658355ff30a2e1</span></div><a href="/servlet/linkout?suffix=cit87&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01070&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01070%26sid%3Dliteratum%253Aachs%26aulast%3DHeinrich%26aufirst%3DT.%26aulast%3DSeenisamy%26aufirst%3DJ.%26aulast%3DBecker%26aufirst%3DF.%26aulast%3DBlume%26aufirst%3DB.%26aulast%3DBomke%26aufirst%3DJ.%26aulast%3DDietz%26aufirst%3DM.%26aulast%3DEckert%26aufirst%3DU.%26aulast%3DFriese-Hamim%26aufirst%3DM.%26aulast%3DGunera%26aufirst%3DJ.%26aulast%3DHansen%26aufirst%3DK.%26aulast%3DLeuthner%26aufirst%3DB.%26aulast%3DMusil%26aufirst%3DD.%26aulast%3DPfalzgraf%26aufirst%3DJ.%26aulast%3DRohdich%26aufirst%3DF.%26aulast%3DSiegl%26aufirst%3DC.%26aulast%3DSpuck%26aufirst%3DD.%26aulast%3DWegener%26aufirst%3DA.%26aulast%3DZenke%26aufirst%3DF.%2BT.%26atitle%3DIdentification%2520of%2520methionine%2520aminopeptidase-2%2520%2528MetAP-2%2529%2520inhibitor%2520M8891%253A%2520a%2520clinical%2520compound%2520for%2520the%2520treatment%2520of%2520cancer%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D11119%26epage%3D11134%26doi%3D10.1021%2Facs.jmedchem.9b01070" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref88"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref88'); return false;" data-citation="" class="refNumLink">88</a></strong><div class="NLM_citation" id="cit88"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Heinrich, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seenisamy, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blume, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bomke, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calderini, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eckert, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friese-Hamim, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kohl, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lehmann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leuthner, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Musil, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rohdich, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zenke, F. T.</span></span> <span> </span><span class="NLM_article-title">Discovery and structure-based optimization of next-generation reversible methionine aminopeptidase-2 (MetAP-2) inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">5025</span>– <span class="NLM_lpage">5039</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b00041</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b00041" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC1MXms1aiu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=5025-5039&author=T.+Heinrichauthor=J.+Seenisamyauthor=B.+Blumeauthor=J.+Bomkeauthor=M.+Calderiniauthor=U.+Eckertauthor=M.+Friese-Hamimauthor=R.+Kohlauthor=M.+Lehmannauthor=B.+Leuthnerauthor=D.+Musilauthor=F.+Rohdichauthor=F.+T.+Zenke&title=Discovery+and+structure-based+optimization+of+next-generation+reversible+methionine+aminopeptidase-2+%28MetAP-2%29+inhibitors&doi=10.1021%2Facs.jmedchem.9b00041"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit88R"><div class="casContent"><span class="casTitleNuber">88</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and Structure-Based Optimization of Next-Generation Reversible Methionine Aminopeptidase-2 (MetAP-2) Inhibitors</span></div><div class="casAuthors">Heinrich, Timo; Seenisamy, Jeyaprakashnarayanan; Blume, Beatrix; Bomke, Joerg; Calderini, Michel; Eckert, Uwe; Friese-Hamim, Manja; Kohl, Rainer; Lehmann, Martin; Leuthner, Birgitta; Musil, Djordje; Rohdich, Felix; Zenke, Frank T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">5025-5039</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Co- and posttranslational processing are crucial maturation steps to generate functional proteins.  MetAP-2 plays an important role in this process and inhibition of its proteolytic activity has been shown to be important for angiogenesis and tumor growth suggesting that small mol. inhibitors of MetAP-2 may be promising options for the treatment of cancer.  This work describes the discovery and structure-based hit optimization of a novel MetAP-2 inhibitory scaffold.  Of crit. importance, a cyclic tartronic diamide coordinates the MetAP-2 metal ion in the active site while addnl. side chains of the mol. were designed to occupy the liphophilic methionine side chain recognition pocket as well as the shallow cavity at the opening of the active site.  The racemic screening hit from a HTS campaign I was discovered with an enzymic IC50 of 150 nM.  The resynthesized eutomer confirmed this activity and inhibited HUVEC proliferation with an IC50 of 1.9 μM.  Its structural anal. revealed a sophisticated interaction pattern of polar and lipophilic contacts that were used to improve cellular potency to an IC50 of 15 nM.  In parallel, the mol. properties were optimized on plasma exposure and antitumor efficacy which led to the identification of advanced lead II.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrsd8oNNAZALLVg90H21EOLACvtfcHk0lh-7iT_VfQvpw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXms1aiu7c%253D&md5=d3841fb334e4d75cc5050936c882fa2b</span></div><a href="/servlet/linkout?suffix=cit88&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b00041&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b00041%26sid%3Dliteratum%253Aachs%26aulast%3DHeinrich%26aufirst%3DT.%26aulast%3DSeenisamy%26aufirst%3DJ.%26aulast%3DBlume%26aufirst%3DB.%26aulast%3DBomke%26aufirst%3DJ.%26aulast%3DCalderini%26aufirst%3DM.%26aulast%3DEckert%26aufirst%3DU.%26aulast%3DFriese-Hamim%26aufirst%3DM.%26aulast%3DKohl%26aufirst%3DR.%26aulast%3DLehmann%26aufirst%3DM.%26aulast%3DLeuthner%26aufirst%3DB.%26aulast%3DMusil%26aufirst%3DD.%26aulast%3DRohdich%26aufirst%3DF.%26aulast%3DZenke%26aufirst%3DF.%2BT.%26atitle%3DDiscovery%2520and%2520structure-based%2520optimization%2520of%2520next-generation%2520reversible%2520methionine%2520aminopeptidase-2%2520%2528MetAP-2%2529%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D5025%26epage%3D5039%26doi%3D10.1021%2Facs.jmedchem.9b00041" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref89"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref89'); return false;" data-citation="" class="refNumLink">89</a></strong><div class="NLM_citation" id="cit89"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Braun, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barglow, K. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Y. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akompong, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Briesewitz, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ray, G. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haldar, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wandless, T. J.</span></span> <span> </span><span class="NLM_article-title">A bifunctional molecule that displays context-dependent cellular activity</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>125</i></span>,  <span class="NLM_fpage">7575</span>– <span class="NLM_lpage">7580</span>, <span class="refDoi"> DOI: 10.1021/ja035176q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja035176q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BD3sXkt1Wgsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=125&publication_year=2003&pages=7575-7580&author=P.+D.+Braunauthor=K.+T.+Barglowauthor=Y.+M.+Linauthor=T.+Akompongauthor=R.+Briesewitzauthor=G.+T.+Rayauthor=K.+Haldarauthor=T.+J.+Wandless&title=A+bifunctional+molecule+that+displays+context-dependent+cellular+activity&doi=10.1021%2Fja035176q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit89R"><div class="casContent"><span class="casTitleNuber">89</span><div class="casTitle"><span class="NLM_cas:atitle">A Bifunctional Molecule That Displays Context-Dependent Cellular Activity</span></div><div class="casAuthors">Braun, Patrick D.; Barglow, Catherine T.; Lin, Yun-Ming; Akompong, Thomas; Briesewitz, Roger; Ray, Gregory T.; Haldar, Kasturi; Wandless, Thomas J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">125</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">7575-7580</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The cell-permeable dihydrofolate reductase inhibitor methotrexate was covalently linked to a ligand for the protein FKBP to create a bifunctional mol. called MTXSLF.  The covalent tether between the two ligands was designed to be prohibitively short, so that unfavorable protein-protein interactions between dihydrofolate reductase (DHFR) and FKBP preclude formation of a trimeric complex.  In vitro and in vivo expts. demonstrate that MTXSLF is an effective inhibitor of human DHFR, but that efficacy is decreased in the presence of human FKBP due to the high concn. of FKBP and its tight affinity for MTXSLF.  MTXSLF also inhibits Plasmodium falciparum DHFR in vitro, but a low concn. of the weaker binding Plasmodium FKBP has no effect on the inhibitory potency of MTXSLF in vivo.  These studies illustrate a potentially general strategy for modulating the biol. activity of synthetic mols. that depends on the ligand-binding properties of a nontarget protein.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHTfky2FXq0bVg90H21EOLACvtfcHk0ljoKpeEJ9-xCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXkt1Wgsbw%253D&md5=625076f0fc225f79bdeead5a3fd02293</span></div><a href="/servlet/linkout?suffix=cit89&amp;dbid=16384&amp;doi=10.1021%2Fja035176q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja035176q%26sid%3Dliteratum%253Aachs%26aulast%3DBraun%26aufirst%3DP.%2BD.%26aulast%3DBarglow%26aufirst%3DK.%2BT.%26aulast%3DLin%26aufirst%3DY.%2BM.%26aulast%3DAkompong%26aufirst%3DT.%26aulast%3DBriesewitz%26aufirst%3DR.%26aulast%3DRay%26aufirst%3DG.%2BT.%26aulast%3DHaldar%26aufirst%3DK.%26aulast%3DWandless%26aufirst%3DT.%2BJ.%26atitle%3DA%2520bifunctional%2520molecule%2520that%2520displays%2520context-dependent%2520cellular%2520activity%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2003%26volume%3D125%26spage%3D7575%26epage%3D7580%26doi%3D10.1021%2Fja035176q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref90"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref90'); return false;" data-citation="" class="refNumLink">90</a></strong><div class="NLM_citation" id="cit90"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Somarelli, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herrera, R. J.</span></span> <span> </span><span class="NLM_article-title">Evolution of the 12 kDa FK506-binding protein gene</span>. <i>Biol. Cell.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>99</i></span>,  <span class="NLM_fpage">311</span>– <span class="NLM_lpage">321</span>, <span class="refDoi"> DOI: 10.1042/BC20060125</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref90/cit90&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1042%2FBC20060125" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref90/cit90&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=17309448" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref90/cit90&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BD2sXltl2ntrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=99&publication_year=2007&pages=311-321&author=J.+A.+Somarelliauthor=R.+J.+Herrera&title=Evolution+of+the+12+kDa+FK506-binding+protein+gene&doi=10.1042%2FBC20060125"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit90R"><div class="casContent"><span class="casTitleNuber">90</span><div class="casTitle"><span class="NLM_cas:atitle">Evolution of the 12kDa FK506-binding protein gene</span></div><div class="casAuthors">Somarelli, Jason A.; Herrera, Rene J.</div><div class="citationInfo"><span class="NLM_cas:title">Biology of the Cell</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">99</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">311-321</span>CODEN:
                <span class="NLM_cas:coden">BCELDF</span>;
        ISSN:<span class="NLM_cas:issn">0248-4900</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">The FKBPs (FK506-binding proteins) belong to a ubiquitous family of proteins that are found in a wide range of taxonomic groups.  These proteins participate in a variety of pathways, including protein folding, down-regulation of T-cell activation and inhibition of cell-cycle progression.  A cDNA for the 12kDa FKBP gene ortholog (FKBP12) in Bombyx mori was isolated from both Bm-5 cultured cells and silk-gland tissue.  Using the FKBP12 cDNA in combination with the B. mori 6× whole-genome shotgun database, we were able to identify the FKBP12 gene, as well as the positions of its intron-exon junctions.  FKBP12 exon sizes and intronic positions are highly conserved among FKBP12 orthologs in 24 diverse genomes.  Comparison of 41 FKBP12 genes revealed several intronic insertion and deletion events throughout evolution.  In addn., paralogous FKBP12 isoforms were identified in all 12 vertebrate genomes.  Both structural and phylogenetics analyses suggest that the isoforms may be evolving independently, possibly due to the distinct functional roles played by each paralog.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpzUWY5Ln2Iz7Vg90H21EOLACvtfcHk0ljoKpeEJ9-xCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXltl2ntrk%253D&md5=e586b381f607ec33a27b8a3aa018ec42</span></div><a href="/servlet/linkout?suffix=cit90&amp;dbid=16384&amp;doi=10.1042%2FBC20060125&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBC20060125%26sid%3Dliteratum%253Aachs%26aulast%3DSomarelli%26aufirst%3DJ.%2BA.%26aulast%3DHerrera%26aufirst%3DR.%2BJ.%26atitle%3DEvolution%2520of%2520the%252012%2520kDa%2520FK506-binding%2520protein%2520gene%26jtitle%3DBiol.%2520Cell.%26date%3D2007%26volume%3D99%26spage%3D311%26epage%3D321%26doi%3D10.1042%2FBC20060125" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref91"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref91'); return false;" data-citation="" class="refNumLink">91</a></strong><div class="NLM_citation" id="cit91"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Galfre, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pitt, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Venturi, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sitsapesan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zaccai, N. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsaneva-Atanasova, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Neill, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sitsapesan, R.</span></span> <span> </span><span class="NLM_article-title">FKBP12 activates the cardiac ryanodine receptor Ca<sup>2+</sup>-release channel and is antagonised by FKBP12.6</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">e31956</span>, <span class="refDoi"> DOI: 10.1371/journal.pone.0031956</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1371%2Fjournal.pone.0031956" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=22363773" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC38XjsVaqsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&pages=e31956&author=E.+Galfreauthor=S.+J.+Pittauthor=E.+Venturiauthor=M.+Sitsapesanauthor=N.+R.+Zaccaiauthor=K.+Tsaneva-Atanasovaauthor=S.+O%E2%80%99Neillauthor=R.+Sitsapesan&title=FKBP12+activates+the+cardiac+ryanodine+receptor+Ca2%2B-release+channel+and+is+antagonised+by+FKBP12.6&doi=10.1371%2Fjournal.pone.0031956"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit91R"><div class="casContent"><span class="casTitleNuber">91</span><div class="casTitle"><span class="NLM_cas:atitle">FKBP12 activates the cardiac ryanodine receptor Ca2+-release channel and is antagonised by FKBP12.6</span></div><div class="casAuthors">Galfre, Elena; Pitt, Samantha J.; Venturi, Elisa; Sitsapesan, Mano; Zaccai, Nathan R.; Tsaneva-Atanasova, Krasimira; O'Neill, Stephen; Sitsapesan, Rebecca</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">e31956</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Changes in FKBP12.6 binding to cardiac ryanodine receptors (RyR2) are implicated in mediating disturbances in Ca2+-homeostasis in heart failure but there is controversy over the functional effects of FKBP12.6 on RyR2 channel gating.  We have therefore investigated the effects of FKBP12.6 and another structurally similar mol., FKBP12, which is far more abundant in heart, on the gating of single sheep RyR2 channels incorporated into planar phospholipid bilayers and on spontaneous waves of Ca2+-induced Ca2+-release in rat isolated permeabilized cardiac cells.  We demonstrate that FKBP12 is a high affinity activator of RyR2, sensitizing the channel to cytosolic Ca2+, whereas FKBP12.6 has very low efficacy, but can antagonize the effects of FKBP12.  Math. modeling of the data shows the importance of the relative concns. of FKBP12 and FKBP12.6 in detg. RyR2 activity.  Consistent with the single-channel results, physiol. concns. of FKBP12 (3 μM) increased Ca2+-wave frequency and decreased the SR Ca2+-content in cardiac cells.  FKBP12.6, itself, had no effect on wave frequency but antagonized the effects of FKBP12.  We provide a biophys. anal. of the mechanisms by which FK-binding proteins can regulate RyR2 single-channel gating.  Our data indicate that FKBP12, in addn. to FKBP12.6, may be important in regulating RyR2 function in the heart.  In heart failure, it is possible that an alteration in the dual regulation of RyR2 by FKBP12 and FKBP12.6 may occur.  This could contribute towards a higher RyR2 open probability, 'leaky' RyR2 channels and Ca2+-dependent arrhythmias.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpt_6Q8HmWQ3bVg90H21EOLACvtfcHk0ljoKpeEJ9-xCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XjsVaqsLk%253D&md5=800d4fdfd59b6399a453569720a25bc6</span></div><a href="/servlet/linkout?suffix=cit91&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0031956&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0031956%26sid%3Dliteratum%253Aachs%26aulast%3DGalfre%26aufirst%3DE.%26aulast%3DPitt%26aufirst%3DS.%2BJ.%26aulast%3DVenturi%26aufirst%3DE.%26aulast%3DSitsapesan%26aufirst%3DM.%26aulast%3DZaccai%26aufirst%3DN.%2BR.%26aulast%3DTsaneva-Atanasova%26aufirst%3DK.%26aulast%3DO%25E2%2580%2599Neill%26aufirst%3DS.%26aulast%3DSitsapesan%26aufirst%3DR.%26atitle%3DFKBP12%2520activates%2520the%2520cardiac%2520ryanodine%2520receptor%2520Ca2%252B-release%2520channel%2520and%2520is%2520antagonised%2520by%2520FKBP12.6%26jtitle%3DPLoS%2520One%26date%3D2012%26volume%3D7%26spage%3De31956%26doi%3D10.1371%2Fjournal.pone.0031956" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref92"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref92'); return false;" data-citation="" class="refNumLink">92</a></strong><div class="NLM_citation" id="cit92"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Small-molecule PROTAC degraders of the bromodomain and extra terminal (BET) proteins — a review</span>. <i>Drug Discovery Today: Technol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">43</span>– <span class="NLM_lpage">51</span>, <span class="refDoi"> DOI: 10.1016/j.ddtec.2019.04.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1016%2Fj.ddtec.2019.04.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=31200858" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A280%3ADC%252BB3M3mvFWrsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2019&pages=43-51&author=C.-Y.+Yangauthor=C.+Qinauthor=L.+Baiauthor=S.+Wang&title=Small-molecule+PROTAC+degraders+of+the+bromodomain+and+extra+terminal+%28BET%29+proteins+%E2%80%94+a+review&doi=10.1016%2Fj.ddtec.2019.04.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit92R"><div class="casContent"><span class="casTitleNuber">92</span><div class="casTitle"><span class="NLM_cas:atitle">Small-molecule PROTAC degraders of the Bromodomain and Extra Terminal (BET) proteins - A review</span></div><div class="casAuthors">Yang Chao-Yie; Qin Chong; Bai Longchuan; Wang Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today. Technologies</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">43-51</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The PROteolysis TArgeting Chimeric (PROTAC) concept has provided an opportunity for the discovery and development of a completely new type of therapy involving induction of protein degradation.  The BET proteins, comprised of BRD2, BRD3, BRD4 and the testis-specific BRDT protein, are epigenetic readers and master transcription coactivators.  Extremely potent and efficacious small-molecule PROTAC degraders of the BET proteins, based on available, potent and selective BET inhibitors, have been reported.  BET degraders differ from BET inhibitors in their cellular potency, phenotypic effects, pharmacokinetic properties and toxicity profiles.  Herein, we provide a review of BET degraders and the differential outcome observed in the cellular and animal models for BET degraders in comparison to BET inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQCRHissLLXoQvupmecNw_lfW6udTcc2eYj23-2uOkw2rntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3M3mvFWrsA%253D%253D&md5=de5790c1db28d56717ff76374072ad81</span></div><a href="/servlet/linkout?suffix=cit92&amp;dbid=16384&amp;doi=10.1016%2Fj.ddtec.2019.04.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ddtec.2019.04.001%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DC.-Y.%26aulast%3DQin%26aufirst%3DC.%26aulast%3DBai%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DSmall-molecule%2520PROTAC%2520degraders%2520of%2520the%2520bromodomain%2520and%2520extra%2520terminal%2520%2528BET%2529%2520proteins%2520%25E2%2580%2594%2520a%2520review%26jtitle%3DDrug%2520Discovery%2520Today%253A%2520Technol.%26date%3D2019%26volume%3D31%26spage%3D43%26epage%3D51%26doi%3D10.1016%2Fj.ddtec.2019.04.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref93"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref93'); return false;" data-citation="" class="refNumLink">93</a></strong><div class="NLM_citation" id="cit93"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Saraswat, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patki, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barot, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dukhande, V. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, K.</span></span> <span> </span><span class="NLM_article-title">Nanoformulation of proteolysis targeting chimera targeting ‘undruggable’ c-myc for the treatment of pancreatic cancer</span>. <i>Nanomedicine (London, U. K.)</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">1761</span>– <span class="NLM_lpage">1777</span>, <span class="refDoi"> DOI: 10.2217/nnm-2020-0156</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.2217%2Fnnm-2020-0156" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=32698663" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhsFOjurvM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2020&pages=1761-1777&author=A.+Saraswatauthor=M.+Patkiauthor=Y.+Fuauthor=S.+Barotauthor=V.+V.+Dukhandeauthor=K.+Patel&title=Nanoformulation+of+proteolysis+targeting+chimera+targeting+%E2%80%98undruggable%E2%80%99+c-myc+for+the+treatment+of+pancreatic+cancer&doi=10.2217%2Fnnm-2020-0156"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit93R"><div class="casContent"><span class="casTitleNuber">93</span><div class="casTitle"><span class="NLM_cas:atitle">Nanoformulation of PROteolysis TArgeting Chimera targeting 'undruggable' c-Myc for the treatment of pancreatic cancer</span></div><div class="casAuthors">Saraswat, Aishwarya; Patki, Manali; Fu, Yige; Barot, Shrikant; Dukhande, Vikas V.; Patel, Ketan</div><div class="citationInfo"><span class="NLM_cas:title">Nanomedicine (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">1761-1777</span>CODEN:
                <span class="NLM_cas:coden">NLUKAC</span>;
        ISSN:<span class="NLM_cas:issn">1748-6963</span>.
    
            (<span class="NLM_cas:orgname">Future Medicine Ltd.</span>)
        </div><div class="casAbstract">To explore the anticancer activity of a novel BRD4 protein degrader ARV-825 (ARV) and its nanoformulation development (ARV-NP) for treatment of pancreatic cancer.  ARV-NP were prepd. using nanopptn. method and characterized for their physicochem. properties and various anticancer cell culture assays.  ARV-NP (89.63 ± 16.39 nm) demonstrated good phys. stability, negligible hemolysis and improved half-life of ARV.  ARV-NP showed significant cytotoxicity, apoptosis and anticlonogenic effect in pancreatic cancer cells.  Significant downregulation of target proteins BRD4, c-Myc, Bcl-2 and upregulation of apoptotic marker cleaved caspase-3 was obsd.  Most importantly, ARV-NP treatment significantly inhibited the cell viability of 3D tumor spheroids of pancreatic cancer.  ARV-NP represents a novel therapeutic strategy for pancreatic cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpsLd0yKZiKT7Vg90H21EOLACvtfcHk0lhEJoOu13ACKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhsFOjurvM&md5=209481e90fb02516b1f88756adce02ff</span></div><a href="/servlet/linkout?suffix=cit93&amp;dbid=16384&amp;doi=10.2217%2Fnnm-2020-0156&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2217%252Fnnm-2020-0156%26sid%3Dliteratum%253Aachs%26aulast%3DSaraswat%26aufirst%3DA.%26aulast%3DPatki%26aufirst%3DM.%26aulast%3DFu%26aufirst%3DY.%26aulast%3DBarot%26aufirst%3DS.%26aulast%3DDukhande%26aufirst%3DV.%2BV.%26aulast%3DPatel%26aufirst%3DK.%26atitle%3DNanoformulation%2520of%2520proteolysis%2520targeting%2520chimera%2520targeting%2520%25E2%2580%2598undruggable%25E2%2580%2599%2520c-myc%2520for%2520the%2520treatment%2520of%2520pancreatic%2520cancer%26jtitle%3DNanomedicine%2520%2528London%252C%2520U.%2520K.%2529%26date%3D2020%26volume%3D15%26spage%3D1761%26epage%3D1777%26doi%3D10.2217%2Fnnm-2020-0156" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref94"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref94'); return false;" data-citation="" class="refNumLink">94</a></strong><div class="NLM_citation" id="cit94"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shafran, J. S.</span>; <span class="NLM_string-name">Jafari, N.</span>; <span class="NLM_string-name">Casey, A. N.</span>; <span class="NLM_string-name">Gyorffy, B.</span>; <span class="NLM_string-name">Denis, G. V.</span></span> <span> </span><span class="NLM_article-title">BRD4 regulates key transcription factors that drive epithelial-mesenchymal transition in castration-resistant prostate cancer</span>.  <i>Prostate Cancer Prostatic Dis.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>, <span class="refDoi"> DOI: 10.1038/s41391-020-0246-y</span> .</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1038%2Fs41391-020-0246-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=32690869" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2020&author=J.+S.+Shafran&author=N.+Jafari&author=A.+N.+Casey&author=B.+Gyorffy&author=G.+V.+Denis&title=BRD4+regulates+key+transcription+factors+that+drive+epithelial-mesenchymal+transition+in+castration-resistant+prostate+cancer&doi=10.1038%2Fs41391-020-0246-y"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit94&amp;dbid=16384&amp;doi=10.1038%2Fs41391-020-0246-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41391-020-0246-y%26sid%3Dliteratum%253Aachs%26aulast%3DShafran%26aufirst%3DJ.%2BS.%26atitle%3DBRD4%2520regulates%2520key%2520transcription%2520factors%2520that%2520drive%2520epithelial-mesenchymal%2520transition%2520in%2520castration-resistant%2520prostate%2520cancer%26jtitle%3DProstate%2520Cancer%2520Prostatic%2520Dis.%26date%3D2020%26doi%3D10.1038%2Fs41391-020-0246-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref95"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref95'); return false;" data-citation="" class="refNumLink">95</a></strong><div class="NLM_citation" id="cit95"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Spriano, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stathis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertoni, F.</span></span> <span> </span><span class="NLM_article-title">Targeting BET bromodomain proteins in cancer: The example of lymphomas</span>. <i>Pharmacol. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>215</i></span>,  <span class="NLM_fpage">107631</span>, <span class="refDoi"> DOI: 10.1016/j.pharmthera.2020.107631</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref95/cit95&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1016%2Fj.pharmthera.2020.107631" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref95/cit95&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=32693114" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref95/cit95&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhsVyisrbI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=215&publication_year=2020&pages=107631&author=F.+Sprianoauthor=A.+Stathisauthor=F.+Bertoni&title=Targeting+BET+bromodomain+proteins+in+cancer%3A+The+example+of+lymphomas&doi=10.1016%2Fj.pharmthera.2020.107631"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit95R"><div class="casContent"><span class="casTitleNuber">95</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting BET bromodomain proteins in cancer: The example of lymphomas</span></div><div class="casAuthors">Spriano, Filippo; Stathis, Anastasios; Bertoni, Francesco</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacology & Therapeutics</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">215</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">107631</span>CODEN:
                <span class="NLM_cas:coden">PHTHDT</span>;
        ISSN:<span class="NLM_cas:issn">0163-7258</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">A review.  The Bromo- and Extra-Terminal domain (BET) family proteins act as ''readers'' of acetylated histones and they are important transcription regulators.  BRD2, BRD3, BRD4 and BRDT, part of the BET family, are important in different tumors, where upregulation or translocation often occurs.  The potential of targeting BET proteins as anti-cancer treatment originated with data obtained with a first series of compds., and there are now several data supporting BET inhibition in both solid tumors and hematol. malignancies.  Despite very pos. preclin. data in different tumor types, the clin. results have been so far moderate.  Using lymphoma as an example to review the data produced in the lab. and in the context of the early clin. trials, we discuss the modalities to make BET targeting more efficient both generating novel generation of compds. and by exploring the combination with small mols. affecting various signaling pathways, BCL2, or DNA damage response signaling, but also with addnl. epigenetic agents and with immunotherapy.  We also discuss the mechanisms of resistance and the toxicity profiles so far reported.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3Jn61Q24_wrVg90H21EOLACvtfcHk0lhEJoOu13ACKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhsVyisrbI&md5=cdb034e619849cb681f48c5b03f7cde8</span></div><a href="/servlet/linkout?suffix=cit95&amp;dbid=16384&amp;doi=10.1016%2Fj.pharmthera.2020.107631&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pharmthera.2020.107631%26sid%3Dliteratum%253Aachs%26aulast%3DSpriano%26aufirst%3DF.%26aulast%3DStathis%26aufirst%3DA.%26aulast%3DBertoni%26aufirst%3DF.%26atitle%3DTargeting%2520BET%2520bromodomain%2520proteins%2520in%2520cancer%253A%2520The%2520example%2520of%2520lymphomas%26jtitle%3DPharmacol.%2520Ther.%26date%3D2020%26volume%3D215%26spage%3D107631%26doi%3D10.1016%2Fj.pharmthera.2020.107631" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref96"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref96'); return false;" data-citation="" class="refNumLink">96</a></strong><div class="NLM_citation" id="cit96"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roeder, R. G.</span></span> <span> </span><span class="NLM_article-title">The long and the short of BRD4: two tales in breast cancer</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>78</i></span>,  <span class="NLM_fpage">993</span>– <span class="NLM_lpage">995</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2020.05.033</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1016%2Fj.molcel.2020.05.033" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=32559427" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BB3cXht1alurrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=78&publication_year=2020&pages=993-995&author=S.+Zhangauthor=R.+G.+Roeder&title=The+long+and+the+short+of+BRD4%3A+two+tales+in+breast+cancer&doi=10.1016%2Fj.molcel.2020.05.033"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit96R"><div class="casContent"><span class="casTitleNuber">96</span><div class="casTitle"><span class="NLM_cas:atitle">The Long and the Short of BRD4: Two Tales in Breast Cancer</span></div><div class="casAuthors">Zhang, Sicong; Roeder, Robert G.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">78</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">993-995</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">A review.  In this issue of Mol. Cell, Wu et al. (2020) describe studies that establish oncogenic vs. tumor-suppressive functions of two BRD4 isoforms in the regulation of gene expression and breast cancer development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGotDF1hISm9V7Vg90H21EOLACvtfcHk0lg_245xymEuJA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXht1alurrK&md5=9c06e499e9d1b87ca0491675a13b18df</span></div><a href="/servlet/linkout?suffix=cit96&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2020.05.033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2020.05.033%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DRoeder%26aufirst%3DR.%2BG.%26atitle%3DThe%2520long%2520and%2520the%2520short%2520of%2520BRD4%253A%2520two%2520tales%2520in%2520breast%2520cancer%26jtitle%3DMol.%2520Cell%26date%3D2020%26volume%3D78%26spage%3D993%26epage%3D995%26doi%3D10.1016%2Fj.molcel.2020.05.033" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref97"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref97'); return false;" data-citation="" class="refNumLink">97</a></strong><div class="NLM_citation" id="cit97"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rupprecht, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Papadopoulos, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rammes, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baghai, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akula, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groyer, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schumacher, M.</span></span> <span> </span><span class="NLM_article-title">Translocator protein (18 kDa) (TSPO) as a therapeutic target for neurological and psychiatric disorders</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">971</span>– <span class="NLM_lpage">988</span>, <span class="refDoi"> DOI: 10.1038/nrd3295</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1038%2Fnrd3295" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=21119734" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsV2ju73E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2010&pages=971-988&author=R.+Rupprechtauthor=V.+Papadopoulosauthor=G.+Rammesauthor=T.+C.+Baghaiauthor=J.+Fanauthor=N.+Akulaauthor=G.+Groyerauthor=D.+Adamsauthor=M.+Schumacher&title=Translocator+protein+%2818+kDa%29+%28TSPO%29+as+a+therapeutic+target+for+neurological+and+psychiatric+disorders&doi=10.1038%2Fnrd3295"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit97R"><div class="casContent"><span class="casTitleNuber">97</span><div class="casTitle"><span class="NLM_cas:atitle">Translocator protein (18 kDa) (TSPO) as a therapeutic target for neurological and psychiatric disorders</span></div><div class="casAuthors">Rupprecht, Rainer; Papadopoulos, Vassilios; Rammes, Gerhard; Baghai, Thomas C.; Fan, Jinjiang; Akula, Nagaraju; Groyer, Ghislaine; Adams, David; Schumacher, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">971-988</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The translocator protein (18 kDa) (TSPO) is localized primarily in the outer mitochondrial membrane of steroid-synthesizing cells, including those in the central and peripheral nervous system.  One of its main functions is the transport of the substrate cholesterol into mitochondria, a prerequisite for steroid synthesis.  TSPO expression may constitute a biomarker of brain inflammation and reactive gliosis that could be monitored by using TSPO ligands as neuroimaging agents.  Moreover, initial clin. trials have indicated that TSPO ligands might be valuable in the treatment of neurol. and psychiatric disorders.  This Review focuses on the biol. and pathophysiol. of TSPO and the potential of currently available TSPO ligands for the diagnosis and treatment of neurol. and psychiatric disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqOS50xtO7oprVg90H21EOLACvtfcHk0lg_245xymEuJA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsV2ju73E&md5=d2376d74a6ddd0eb795aa34d9e4df3f3</span></div><a href="/servlet/linkout?suffix=cit97&amp;dbid=16384&amp;doi=10.1038%2Fnrd3295&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3295%26sid%3Dliteratum%253Aachs%26aulast%3DRupprecht%26aufirst%3DR.%26aulast%3DPapadopoulos%26aufirst%3DV.%26aulast%3DRammes%26aufirst%3DG.%26aulast%3DBaghai%26aufirst%3DT.%2BC.%26aulast%3DFan%26aufirst%3DJ.%26aulast%3DAkula%26aufirst%3DN.%26aulast%3DGroyer%26aufirst%3DG.%26aulast%3DAdams%26aufirst%3DD.%26aulast%3DSchumacher%26aufirst%3DM.%26atitle%3DTranslocator%2520protein%2520%252818%2520kDa%2529%2520%2528TSPO%2529%2520as%2520a%2520therapeutic%2520target%2520for%2520neurological%2520and%2520psychiatric%2520disorders%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2010%26volume%3D9%26spage%3D971%26epage%3D988%26doi%3D10.1038%2Fnrd3295" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref98"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref98'); return false;" data-citation="" class="refNumLink">98</a></strong><div class="NLM_citation" id="cit98"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xie, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yui, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hatori, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamasaki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumata, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wakizaka, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshida, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujinaga, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawamura, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, M.-R.</span></span> <span> </span><span class="NLM_article-title">Translocator protein (18kDa), a potential molecular imaging biomarker for non-invasively distinguishing non-alcoholic fatty liver disease</span>. <i>J. Hepatol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">1076</span>– <span class="NLM_lpage">1082</span>, <span class="refDoi"> DOI: 10.1016/j.jhep.2012.07.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1016%2Fj.jhep.2012.07.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=22796156" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFKlurvE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2012&pages=1076-1082&author=L.+Xieauthor=J.+Yuiauthor=A.+Hatoriauthor=T.+Yamasakiauthor=K.+Kumataauthor=H.+Wakizakaauthor=Y.+Yoshidaauthor=M.+Fujinagaauthor=K.+Kawamuraauthor=M.-R.+Zhang&title=Translocator+protein+%2818kDa%29%2C+a+potential+molecular+imaging+biomarker+for+non-invasively+distinguishing+non-alcoholic+fatty+liver+disease&doi=10.1016%2Fj.jhep.2012.07.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit98R"><div class="casContent"><span class="casTitleNuber">98</span><div class="casTitle"><span class="NLM_cas:atitle">Translocator protein (18 kDa), a potential molecular imaging biomarker for non-invasively distinguishing non-alcoholic fatty liver disease</span></div><div class="casAuthors">Xie, Lin; Yui, Joji; Hatori, Akiko; Yamasaki, Tomoteru; Kumata, Katsushi; Wakizaka, Hidekatsu; Yoshida, Yuichiro; Fujinaga, Masayuki; Kawamura, Kazunori; Zhang, Ming-Rong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Hepatology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1076-1082</span>CODEN:
                <span class="NLM_cas:coden">JOHEEC</span>;
        ISSN:<span class="NLM_cas:issn">0168-8278</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Background & Aims: Mitochondrial dysfunction is responsible for liver damage and disease progression in non-alc. fatty liver disease (NAFLD).  Translocator protein (18 kDa) (TSPO), a mitochondrial transmembrane protein, plays important roles in modulating mitochondrial function.  This study explored whether TSPO can be used as an imaging biomarker of non-invasive diagnosis and staging of NAFLD, monitored using positron emission tomog. (PET) with a TSPO radioligand [18F]FEDAC.  Methods: PET with [18F]FEDAC, non-enhanced computerized tomog. (CT), autoradiog., histopathol., and gene anal. were performed to evaluate and quantify TSPO levels and NAFLD progression in methionine and choline-deficient diet-fed mice.  Correlations were analyzed between uptake ratio of radioactivity and NAFLD activity score (NAS) in the liver.  Results: Uptake of [18F]FEDAC obviously increased with disease progression from simple steatosis to non-alc. steatohepatitis (NASH) (p <0.01).  A close correlation was identified between [18F]FEDAC uptake ratio and NAS in the liver (Pearson's r = 0.922, p = 0.000).  Specific binding of [18F]FEDAC to TSPO in the NAFLD livers was assessed in competition studies with the unlabeled TSPO-selective ligand PK11195.  Autoradiog. and histopathol. confirmed the PET imaging results.  Further, the mRNA levels of the functional macromol. signaling complex composed of TSPO were obviously higher compared to controls.  Conclusions: TSPO expression increases in NAFLD and closely correlates with NAFLD progression.  TSPO as a specific mol. imaging biomarker may open a novel avenue for non-invasive, reliable, and quant. diagnosis and staging of NAFLD.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoESltKjDYFkbVg90H21EOLACvtfcHk0lg_245xymEuJA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFKlurvE&md5=9f99a7b3ef20b3e107260b7bc59703e0</span></div><a href="/servlet/linkout?suffix=cit98&amp;dbid=16384&amp;doi=10.1016%2Fj.jhep.2012.07.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jhep.2012.07.002%26sid%3Dliteratum%253Aachs%26aulast%3DXie%26aufirst%3DL.%26aulast%3DYui%26aufirst%3DJ.%26aulast%3DHatori%26aufirst%3DA.%26aulast%3DYamasaki%26aufirst%3DT.%26aulast%3DKumata%26aufirst%3DK.%26aulast%3DWakizaka%26aufirst%3DH.%26aulast%3DYoshida%26aufirst%3DY.%26aulast%3DFujinaga%26aufirst%3DM.%26aulast%3DKawamura%26aufirst%3DK.%26aulast%3DZhang%26aufirst%3DM.-R.%26atitle%3DTranslocator%2520protein%2520%252818kDa%2529%252C%2520a%2520potential%2520molecular%2520imaging%2520biomarker%2520for%2520non-invasively%2520distinguishing%2520non-alcoholic%2520fatty%2520liver%2520disease%26jtitle%3DJ.%2520Hepatol.%26date%3D2012%26volume%3D57%26spage%3D1076%26epage%3D1082%26doi%3D10.1016%2Fj.jhep.2012.07.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref99"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref99'); return false;" data-citation="" class="refNumLink">99</a></strong><div class="NLM_citation" id="cit99"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulk, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Souza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhe-Paganon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">Phthalimide conjugation as a strategy for in vivo target protein degradation</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>348</i></span>,  <span class="NLM_fpage">1376</span>, <span class="refDoi"> DOI: 10.1126/science.aab1433</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1126%2Fscience.aab1433" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=25999370" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC2MXpvFansLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=348&publication_year=2015&pages=1376&author=G.+E.+Winterauthor=D.+L.+Buckleyauthor=J.+Paulkauthor=J.+M.+Robertsauthor=A.+Souzaauthor=S.+Dhe-Paganonauthor=J.+E.+Bradner&title=Phthalimide+conjugation+as+a+strategy+for+in+vivo+target+protein+degradation&doi=10.1126%2Fscience.aab1433"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit99R"><div class="casContent"><span class="casTitleNuber">99</span><div class="casTitle"><span class="NLM_cas:atitle">Phthalimide conjugation as a strategy for in vivo target protein degradation</span></div><div class="casAuthors">Winter, Georg E.; Buckley, Dennis L.; Paulk, Joshiawa; Roberts, Justin M.; Souza, Amanda; Dhe-Paganon, Sirano; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">348</span>
        (<span class="NLM_cas:issue">6241</span>),
    <span class="NLM_cas:pages">1376-1381</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The development of effective pharmacol. inhibitors of multidomain scaffold proteins, notably transcription factors, is a particularly challenging problem.  In part, this is because many small-mol. antagonists disrupt the activity of only one domain in the target protein.  The authors devised a chem. strategy that promotes ligand-dependent target protein degrdn. using as an example the transcriptional coactivator BRD4, a protein crit. for cancer cell growth and survival.  The authors appended a competitive antagonist of BET bromodomains to a phthalimide moiety to hijack the cereblon E3 ubiquitin ligase complex.  The resultant compd., dBET1, induced highly selective cereblon-dependent BET protein degrdn. in vitro and in vivo and delayed leukemia progression in mice.  A second series of probes resulted in selective degrdn. of the cytosolic protein FKBP12.  This chem. strategy for controlling target protein stability may have implications for therapeutically targeting previously intractable proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1ZW4zjRjXi7Vg90H21EOLACvtfcHk0lg_245xymEuJA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXpvFansLk%253D&md5=b0277cc95d318d926b96e7a119066e3b</span></div><a href="/servlet/linkout?suffix=cit99&amp;dbid=16384&amp;doi=10.1126%2Fscience.aab1433&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aab1433%26sid%3Dliteratum%253Aachs%26aulast%3DWinter%26aufirst%3DG.%2BE.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DPaulk%26aufirst%3DJ.%26aulast%3DRoberts%26aufirst%3DJ.%2BM.%26aulast%3DSouza%26aufirst%3DA.%26aulast%3DDhe-Paganon%26aufirst%3DS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DPhthalimide%2520conjugation%2520as%2520a%2520strategy%2520for%2520in%2520vivo%2520target%2520protein%2520degradation%26jtitle%3DScience%26date%3D2015%26volume%3D348%26spage%3D1376%26doi%3D10.1126%2Fscience.aab1433" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref100"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref100'); return false;" data-citation="" class="refNumLink">100</a></strong><div class="NLM_citation" id="cit100"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fei, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lau, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Landry, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baumgarth, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, X.</span></span> <span> </span><span class="NLM_article-title">Characterization of receptor binding profiles of influenza a viruses using an ellipsometry-based label-free glycan microarray assay platform</span>. <i>Biomolecules</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">1480</span>– <span class="NLM_lpage">1498</span>, <span class="refDoi"> DOI: 10.3390/biom5031480</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.3390%2Fbiom5031480" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=26193329" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1egs7vE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2015&pages=1480-1498&author=Y.+Feiauthor=Y.+S.+Sunauthor=Y.+Liauthor=H.+Yuauthor=K.+Lauauthor=J.+P.+Landryauthor=Z.+Luoauthor=N.+Baumgarthauthor=X.+Chenauthor=X.+Zhu&title=Characterization+of+receptor+binding+profiles+of+influenza+a+viruses+using+an+ellipsometry-based+label-free+glycan+microarray+assay+platform&doi=10.3390%2Fbiom5031480"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit100R"><div class="casContent"><span class="casTitleNuber">100</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of receptor binding profiles of influenza a viruses using an ellipsometry-based label-free glycan microarray assay platform</span></div><div class="casAuthors">Fei, Yiyan; Sun, Yung-Shin; Li, Yanhong; Yu, Hai; Lau, Kam; Landry, James P.; Luo, Zeng; Baumgarth, Nicole; Chen, Xi; Zhu, Xiangdong</div><div class="citationInfo"><span class="NLM_cas:title">Biomolecules</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1480-1498</span>CODEN:
                <span class="NLM_cas:coden">BIOMHC</span>;
        ISSN:<span class="NLM_cas:issn">2218-273X</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">A key step leading to influenza viral infection is the highly specific binding of a viral spike protein, hemagglutinin (HA), with an extracellular glycan receptor of a host cell.  Detailed and timely characterization of virus-receptor binding profiles may be used to evaluate and track the pandemic potential of an influenza virus strain.  We demonstrate a label-free glycan microarray assay platform for acquiring influenza virus binding profiles against a wide variety of glycan receptors.  By immobilizing biotinylated receptors on a streptavidin-functionalized solid surface, we measured binding curves of five influenza A virus strains with 24 glycans of diverse structures and used the apparent equil. dissocn. consts. (avidity consts., 10-100 pM) as characterizing parameters of viral receptor profiles.  Furthermore by measuring binding kinetic consts. of soln.-phase glycans to immobilized viruses, we confirmed that the glycan-HA affinity const. is in the range of 10 mM and the reaction is enthalpy-driven.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbRqRPEu0OirVg90H21EOLACvtfcHk0lhLVrA8VPmZhg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1egs7vE&md5=2b9a40222f727794475275c4cd8ecf7b</span></div><a href="/servlet/linkout?suffix=cit100&amp;dbid=16384&amp;doi=10.3390%2Fbiom5031480&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fbiom5031480%26sid%3Dliteratum%253Aachs%26aulast%3DFei%26aufirst%3DY.%26aulast%3DSun%26aufirst%3DY.%2BS.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DYu%26aufirst%3DH.%26aulast%3DLau%26aufirst%3DK.%26aulast%3DLandry%26aufirst%3DJ.%2BP.%26aulast%3DLuo%26aufirst%3DZ.%26aulast%3DBaumgarth%26aufirst%3DN.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DZhu%26aufirst%3DX.%26atitle%3DCharacterization%2520of%2520receptor%2520binding%2520profiles%2520of%2520influenza%2520a%2520viruses%2520using%2520an%2520ellipsometry-based%2520label-free%2520glycan%2520microarray%2520assay%2520platform%26jtitle%3DBiomolecules%26date%3D2015%26volume%3D5%26spage%3D1480%26epage%3D1498%26doi%3D10.3390%2Fbiom5031480" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref101"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref101'); return false;" data-citation="" class="refNumLink">101</a></strong><div class="NLM_citation" id="cit101"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Landry, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fei, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ke, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, P.</span></span> <span> </span><span class="NLM_article-title">Discovering small molecule ligands of vascular endothelial growth factor that block VEGF-KDR binding using label-free microarray-based assays</span>. <i>Assay Drug Dev. Technol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">326</span>– <span class="NLM_lpage">332</span>, <span class="refDoi"> DOI: 10.1089/adt.2012.485</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1089%2Fadt.2012.485" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=23772553" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC3sXpsVOku7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2013&pages=326-332&author=J.+P.+Landryauthor=Y.+Feiauthor=X.+Zhuauthor=Y.+Keauthor=G.+Yuauthor=P.+Lee&title=Discovering+small+molecule+ligands+of+vascular+endothelial+growth+factor+that+block+VEGF-KDR+binding+using+label-free+microarray-based+assays&doi=10.1089%2Fadt.2012.485"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit101R"><div class="casContent"><span class="casTitleNuber">101</span><div class="casTitle"><span class="NLM_cas:atitle">Discovering Small Molecule Ligands of Vascular Endothelial Growth Factor That Block VEGF-KDR Binding Using Label-Free Microarray-Based Assays</span></div><div class="casAuthors">Landry, James P.; Fei, Yiyan; Zhu, Xiangdong; Ke, Yaohuang; Yu, Guoliang; Lee, Pierre</div><div class="citationInfo"><span class="NLM_cas:title">Assay and Drug Development Technologies</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">326-332</span>CODEN:
                <span class="NLM_cas:coden">ADDTAR</span>;
        ISSN:<span class="NLM_cas:issn">1540-658X</span>.
    
            (<span class="NLM_cas:orgname">Mary Ann Liebert, Inc.</span>)
        </div><div class="casAbstract">We present here a label-free microarray-based assay platform that we used to identify inhibitors of vascular endothelial growth factor (VEGF)-kinase-insertion domain receptor (KDR) binding.  Supported by a combination of special ellipsometry-based optical detection and small mol. microarrays (SMM), this platform consists of three assays: (1) the first assay detects binding of a target protein with SMM and identifies ligands to the protein as inhibitor candidates; (2) the second assay detects binding of a receptor protein with identical SMM and subsequent binding of the target protein (a sandwich assay) to identify the ligands to the receptor protein that do not interfere with the target-receptor binding; (3) the third assay detects binding of the target protein to the receptor protein in the presence of the ligands of the target protein identified from the first assay, with the receptor protein immobilized to a solid surface through the ligands identified in the second assay, to yield dose-response curves.  Using this platform, we screened 7,961 compds. from the National Cancer Institute and found 12 inhibitors to VEGF-KDR (VEGFR2) interactions with IC50 ranging from 0.3 to 60μM.  The inhibitory potency of these inhibitors found in the microarray-based assay was confirmed by their inhibition of VEGF-induced VEGFR2 phosphorylation in a cell-based assay.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr08LF_ToYfVrVg90H21EOLACvtfcHk0lhLVrA8VPmZhg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXpsVOku7Y%253D&md5=f6c946fb0e05d4e500e9111af0b6f4e6</span></div><a href="/servlet/linkout?suffix=cit101&amp;dbid=16384&amp;doi=10.1089%2Fadt.2012.485&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1089%252Fadt.2012.485%26sid%3Dliteratum%253Aachs%26aulast%3DLandry%26aufirst%3DJ.%2BP.%26aulast%3DFei%26aufirst%3DY.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DKe%26aufirst%3DY.%26aulast%3DYu%26aufirst%3DG.%26aulast%3DLee%26aufirst%3DP.%26atitle%3DDiscovering%2520small%2520molecule%2520ligands%2520of%2520vascular%2520endothelial%2520growth%2520factor%2520that%2520block%2520VEGF-KDR%2520binding%2520using%2520label-free%2520microarray-based%2520assays%26jtitle%3DAssay%2520Drug%2520Dev.%2520Technol.%26date%3D2013%26volume%3D11%26spage%3D326%26epage%3D332%26doi%3D10.1089%2Fadt.2012.485" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref102"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref102'); return false;" data-citation="" class="refNumLink">102</a></strong><div class="NLM_citation" id="cit102"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Landry, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, X. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gregg, J. P.</span></span> <span> </span><span class="NLM_article-title">Label-free detection of microarrays of biomolecules by oblique-incidence reflectivity difference microscopy</span>. <i>Opt. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">581</span>– <span class="NLM_lpage">583</span>, <span class="refDoi"> DOI: 10.1364/OL.29.000581</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1364%2FOL.29.000581" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=15035477" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BD2cXislKrsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2004&pages=581-583&author=J.+P.+Landryauthor=X.+D.+Zhuauthor=J.+P.+Gregg&title=Label-free+detection+of+microarrays+of+biomolecules+by+oblique-incidence+reflectivity+difference+microscopy&doi=10.1364%2FOL.29.000581"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit102R"><div class="casContent"><span class="casTitleNuber">102</span><div class="casTitle"><span class="NLM_cas:atitle">Label-free detection of microarrays of biomolecules by oblique-incidence reflectivity difference microscopy</span></div><div class="casAuthors">Landry, J. P.; Zhu, X. D.; Gregg, J. P.</div><div class="citationInfo"><span class="NLM_cas:title">Optics Letters</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">581-583</span>CODEN:
                <span class="NLM_cas:coden">OPLEDP</span>;
        ISSN:<span class="NLM_cas:issn">0146-9592</span>.
    
            (<span class="NLM_cas:orgname">Optical Society of America</span>)
        </div><div class="casAbstract">We developed an oblique-incidence reflectivity difference (OI-RD) scanning microscope for label-free imaging of microarrays of biomols. upon solid substrates.  We demonstrate that hybridization reactions in an oligonucleotide microarray fabricated upon a glass slide can be detected by such an OI-RD microscope.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfxixOfT4iqrVg90H21EOLACvtfcHk0lhLVrA8VPmZhg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXislKrsrs%253D&md5=7c1bbadf6e7240e37803eb531761285f</span></div><a href="/servlet/linkout?suffix=cit102&amp;dbid=16384&amp;doi=10.1364%2FOL.29.000581&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1364%252FOL.29.000581%26sid%3Dliteratum%253Aachs%26aulast%3DLandry%26aufirst%3DJ.%2BP.%26aulast%3DZhu%26aufirst%3DX.%2BD.%26aulast%3DGregg%26aufirst%3DJ.%2BP.%26atitle%3DLabel-free%2520detection%2520of%2520microarrays%2520of%2520biomolecules%2520by%2520oblique-incidence%2520reflectivity%2520difference%2520microscopy%26jtitle%3DOpt.%2520Lett.%26date%3D2004%26volume%3D29%26spage%3D581%26epage%3D583%26doi%3D10.1364%2FOL.29.000581" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref103"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref103'); return false;" data-citation="" class="refNumLink">103</a></strong><div class="NLM_citation" id="cit103"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fei, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jungblut, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pei, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dorhoi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaufmann, S. H. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ge, B.</span></span> <span> </span><span class="NLM_article-title">Nuclear cGAS suppresses DNA repair and promotes tumorigenesis</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>563</i></span>,  <span class="NLM_fpage">131</span>– <span class="NLM_lpage">136</span>, <span class="refDoi"> DOI: 10.1038/s41586-018-0629-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1038%2Fs41586-018-0629-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=30356214" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitVSitLrP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=563&publication_year=2018&pages=131-136&author=H.+Liuauthor=H.+Zhangauthor=X.+Wuauthor=D.+Maauthor=J.+Wuauthor=L.+Wangauthor=Y.+Jiangauthor=Y.+Feiauthor=C.+Zhuauthor=R.+Tanauthor=P.+Jungblutauthor=G.+Peiauthor=A.+Dorhoiauthor=Q.+Yanauthor=F.+Zhangauthor=R.+Zhengauthor=S.+Liuauthor=H.+Liangauthor=Z.+Liuauthor=H.+Yangauthor=J.+Chenauthor=P.+Wangauthor=T.+Tangauthor=W.+Pengauthor=Z.+Huauthor=Z.+Xuauthor=X.+Huangauthor=J.+Wangauthor=H.+Liauthor=Y.+Zhouauthor=F.+Liuauthor=D.+Yanauthor=S.+H.+E.+Kaufmannauthor=C.+Chenauthor=Z.+Maoauthor=B.+Ge&title=Nuclear+cGAS+suppresses+DNA+repair+and+promotes+tumorigenesis&doi=10.1038%2Fs41586-018-0629-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit103R"><div class="casContent"><span class="casTitleNuber">103</span><div class="casTitle"><span class="NLM_cas:atitle">Nuclear cGAS suppresses DNA repair and promotes tumorigenesis</span></div><div class="casAuthors">Liu, Haipeng; Zhang, Haiping; Wu, Xiangyang; Ma, Dapeng; Wu, Juehui; Wang, Lin; Jiang, Yan; Fei, Yiyan; Zhu, Chenggang; Tan, Rong; Jungblut, Peter; Pei, Gang; Dorhoi, Anca; Yan, Qiaoling; Zhang, Fan; Zheng, Ruijuan; Liu, Siyu; Liang, Haijiao; Liu, Zhonghua; Yang, Hua; Chen, Jianxia; Wang, Peng; Tang, Tianqi; Peng, Wenxia; Hu, Zhangsen; Xu, Zhu; Huang, Xiaochen; Wang, Jie; Li, Haohao; Zhou, Yilong; Liu, Feng; Yan, Dapeng; Kaufmann, Stefan H. E.; Chen, Chang; Mao, Zhiyong; Ge, Baoxue</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">563</span>
        (<span class="NLM_cas:issue">7729</span>),
    <span class="NLM_cas:pages">131-136</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Accurate repair of DNA double-stranded breaks by homologous recombination preserves genome integrity and inhibits tumorigenesis.  CGMP-AMP synthase (cGAS) is a cytosolic DNA sensor that activates innate immunity by initiating the STING-IRF3-type I IFN signaling cascade1,2.  Recognition of ruptured micronuclei by cGAS links genome instability to the innate immune response3,4, but the potential involvement of cGAS in DNA repair remains unknown.  Here we demonstrate that cGAS inhibits homologous recombination in mouse and human models.  DNA damage induces nuclear translocation of cGAS in a manner that is dependent on importin-α, and the phosphorylation of cGAS at tyrosine 215-mediated by B-lymphoid tyrosine kinase-facilitates the cytosolic retention of cGAS.  In the nucleus, cGAS is recruited to double-stranded breaks and interacts with PARP1 via poly(ADP-ribose).  The cGAS-PARP1 interaction impedes the formation of the PARP1-Timeless complex, and thereby suppresses homologous recombination.  We show that knockdown of cGAS suppresses DNA damage and inhibits tumor growth both in vitro and in vivo.  We conclude that nuclear cGAS suppresses homologous-recombination-mediated repair and promotes tumor growth, and that cGAS therefore represents a potential target for cancer prevention and therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_FFWfXc3dP7Vg90H21EOLACvtfcHk0ljfomGD8BIu9g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitVSitLrP&md5=d47b3675923877e00882c9cf3dcabae9</span></div><a href="/servlet/linkout?suffix=cit103&amp;dbid=16384&amp;doi=10.1038%2Fs41586-018-0629-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-018-0629-6%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DWu%26aufirst%3DX.%26aulast%3DMa%26aufirst%3DD.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DFei%26aufirst%3DY.%26aulast%3DZhu%26aufirst%3DC.%26aulast%3DTan%26aufirst%3DR.%26aulast%3DJungblut%26aufirst%3DP.%26aulast%3DPei%26aufirst%3DG.%26aulast%3DDorhoi%26aufirst%3DA.%26aulast%3DYan%26aufirst%3DQ.%26aulast%3DZhang%26aufirst%3DF.%26aulast%3DZheng%26aufirst%3DR.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DLiang%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DP.%26aulast%3DTang%26aufirst%3DT.%26aulast%3DPeng%26aufirst%3DW.%26aulast%3DHu%26aufirst%3DZ.%26aulast%3DXu%26aufirst%3DZ.%26aulast%3DHuang%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DYan%26aufirst%3DD.%26aulast%3DKaufmann%26aufirst%3DS.%2BH.%2BE.%26aulast%3DChen%26aufirst%3DC.%26aulast%3DMao%26aufirst%3DZ.%26aulast%3DGe%26aufirst%3DB.%26atitle%3DNuclear%2520cGAS%2520suppresses%2520DNA%2520repair%2520and%2520promotes%2520tumorigenesis%26jtitle%3DNature%26date%3D2018%26volume%3D563%26spage%3D131%26epage%3D136%26doi%3D10.1038%2Fs41586-018-0629-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref104"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref104'); return false;" data-citation="" class="refNumLink">104</a></strong><div class="NLM_citation" id="cit104"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ge, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mi, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fei, Y.</span></span> <span> </span><span class="NLM_article-title">Fast focal point correction in prism-coupled total internal reflection scanning imager using an electronically tunable lens</span>. <i>Sensors</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">524</span>, <span class="refDoi"> DOI: 10.3390/s18020524</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref104/cit104&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.3390%2Fs18020524" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref104/cit104&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC1cXit1agtLbE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2018&pages=524&author=C.+Zhuauthor=B.+Geauthor=R.+Chenauthor=X.+Zhuauthor=L.+Miauthor=J.+Maauthor=X.+Wangauthor=F.+Zhengauthor=Y.+Fei&title=Fast+focal+point+correction+in+prism-coupled+total+internal+reflection+scanning+imager+using+an+electronically+tunable+lens&doi=10.3390%2Fs18020524"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit104R"><div class="casContent"><span class="casTitleNuber">104</span><div class="casTitle"><span class="NLM_cas:atitle">Fast focal point correction in prism-coupled total internal reflection scanning imager using an electronically tunable lens</span></div><div class="casAuthors">Zhu, Chenggang; Ge, Bilin; Chen, Ru; Zhu, Xiangdong; Mi, Lan; Ma, Jiong; Wang, Xu; Zheng, Fengyun; Fei, Yiyan</div><div class="citationInfo"><span class="NLM_cas:title">Sensors</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">524/1-524/10</span>CODEN:
                <span class="NLM_cas:coden">SENSC9</span>;
        ISSN:<span class="NLM_cas:issn">1424-8220</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Total internal reflection (TIR) is useful for interrogating phys. and chem. processes that occur at the interface between two transparent media.  Yet prism-coupled TIR imaging microscopes suffer from limited sensing areas due to the fact that the interface (the object plane) is not perpendicular to the optical axis of the microscope.  In this paper, we show that an elec. tunable lens can be used to rapidly and reproducibly correct the focal length of an oblique-incidence scanning microscope (OI-RD) in a prism-coupled TIR geometry.  We demonstrate the performance of such a correction by acquiring an image of a protein microarray over a scan area of 4 cm2 with an effective resoln. of less than 20 μ.  The electronic focal length tuning eliminates the mech. movement of the illumination lens in the scanning microscope and in turn the noise and background drift assocd. with the motion.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2FdcCHwpOXLVg90H21EOLACvtfcHk0ljfomGD8BIu9g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXit1agtLbE&md5=6c3c73884972ed2b1a306244b904b710</span></div><a href="/servlet/linkout?suffix=cit104&amp;dbid=16384&amp;doi=10.3390%2Fs18020524&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fs18020524%26sid%3Dliteratum%253Aachs%26aulast%3DZhu%26aufirst%3DC.%26aulast%3DGe%26aufirst%3DB.%26aulast%3DChen%26aufirst%3DR.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DMi%26aufirst%3DL.%26aulast%3DMa%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DZheng%26aufirst%3DF.%26aulast%3DFei%26aufirst%3DY.%26atitle%3DFast%2520focal%2520point%2520correction%2520in%2520prism-coupled%2520total%2520internal%2520reflection%2520scanning%2520imager%2520using%2520an%2520electronically%2520tunable%2520lens%26jtitle%3DSensors%26date%3D2018%26volume%3D18%26spage%3D524%26doi%3D10.3390%2Fs18020524" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref105"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref105'); return false;" data-citation="" class="refNumLink">105</a></strong><div class="NLM_citation" id="cit105"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Landry, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gregg, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, X.</span></span> <span> </span><span class="NLM_article-title">Oblique-incidence reflectivity difference microscope for label-free high-throughput detection of biochemical reactions in a microarray format</span>. <i>Appl. Opt.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">1890</span>– <span class="NLM_lpage">1895</span>, <span class="refDoi"> DOI: 10.1364/AO.46.001890</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref105/cit105&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1364%2FAO.46.001890" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref105/cit105&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=17356635" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref105/cit105&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BD2sXks1alu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2007&pages=1890-1895&author=X.+Zhuauthor=J.+P.+Landryauthor=Y.-S.+Sunauthor=J.+P.+Greggauthor=K.+S.+Lamauthor=X.+Guo&title=Oblique-incidence+reflectivity+difference+microscope+for+label-free+high-throughput+detection+of+biochemical+reactions+in+a+microarray+format&doi=10.1364%2FAO.46.001890"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit105R"><div class="casContent"><span class="casTitleNuber">105</span><div class="casTitle"><span class="NLM_cas:atitle">Oblique-incidence reflectivity difference microscope for label-free high-throughput detection of biochemical reactions in a microarray format</span></div><div class="casAuthors">Zhu, Xiangdong; Landry, James P.; Sun, Yung-Shin; Gregg, Jeff P.; Lam, Kit S.; Guo, Xiaowen</div><div class="citationInfo"><span class="NLM_cas:title">Applied Optics</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1890-1895</span>CODEN:
                <span class="NLM_cas:coden">APOPAI</span>;
        ISSN:<span class="NLM_cas:issn">0003-6935</span>.
    
            (<span class="NLM_cas:orgname">Optical Society of America</span>)
        </div><div class="casAbstract">We describe a recently developed oblique-incidence reflectivity difference (OI-RD) microscope, a form of polarization-modulated imaging ellipsometer, for label-free-high-throughput detection of biomol. reactions on DNA and protein microarrays.  We present examples of application of this technique to end-point and real-time investigations of DNA-DNA hybridization, antibody-antigen capture, and protein-small-mol. binding reactions.  Compared to a conventional imaging ellipsometer based on the polarizer-compensator-sample-analyzer scheme and under the off-null condition, a polarization-modulated OI-RD microscope is inherently more sensitive by at least 1 order of magnitude to thickness changes on a solid surface.  Compared with imaging surface plasmon resonance microscopes based on reflectance change on falling or rising slopes of the surface plasmon resonance, the OI-RD microscope (1) has a comparable sensitivity, (2) is applicable to conventional microscope glass slides, and (3) easily covers a field of view as large as the entire surface of a 1 in. × 3 in. (2.54 cm × 7.62 cm) microscope slide.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr2VZofLGnTFrVg90H21EOLACvtfcHk0ljfomGD8BIu9g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXks1alu7w%253D&md5=c8de9e695ee3a79ba7d9c5027b93df5f</span></div><a href="/servlet/linkout?suffix=cit105&amp;dbid=16384&amp;doi=10.1364%2FAO.46.001890&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1364%252FAO.46.001890%26sid%3Dliteratum%253Aachs%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DLandry%26aufirst%3DJ.%2BP.%26aulast%3DSun%26aufirst%3DY.-S.%26aulast%3DGregg%26aufirst%3DJ.%2BP.%26aulast%3DLam%26aufirst%3DK.%2BS.%26aulast%3DGuo%26aufirst%3DX.%26atitle%3DOblique-incidence%2520reflectivity%2520difference%2520microscope%2520for%2520label-free%2520high-throughput%2520detection%2520of%2520biochemical%2520reactions%2520in%2520a%2520microarray%2520format%26jtitle%3DAppl.%2520Opt.%26date%3D2007%26volume%3D46%26spage%3D1890%26epage%3D1895%26doi%3D10.1364%2FAO.46.001890" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref106"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref106'); return false;" data-citation="" class="refNumLink">106</a></strong><div class="NLM_citation" id="cit106"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baldo, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paganetti, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grueninger, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marcellin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaltenbach, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lo, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Semmelroth, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zivanovic, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abramowski, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lotz, G. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bates, G. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, A.</span></span> <span> </span><span class="NLM_article-title">TR-FRET-based duplex immunoassay reveals an inverse correlation of soluble and aggregated mutant huntingtin in huntington’s disease</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">264</span>– <span class="NLM_lpage">275</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2011.12.020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref106/cit106&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1016%2Fj.chembiol.2011.12.020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref106/cit106&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=22365609" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref106/cit106&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC38XjtFWhtLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2012&pages=264-275&author=B.+Baldoauthor=P.+Paganettiauthor=S.+Grueningerauthor=D.+Marcellinauthor=L.+S.+Kaltenbachauthor=D.+C.+Loauthor=M.+Semmelrothauthor=A.+Zivanovicauthor=D.+Abramowskiauthor=D.+Smithauthor=G.+P.+Lotzauthor=G.+P.+Batesauthor=A.+Weiss&title=TR-FRET-based+duplex+immunoassay+reveals+an+inverse+correlation+of+soluble+and+aggregated+mutant+huntingtin+in+huntington%E2%80%99s+disease&doi=10.1016%2Fj.chembiol.2011.12.020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit106R"><div class="casContent"><span class="casTitleNuber">106</span><div class="casTitle"><span class="NLM_cas:atitle">TR-FRET-Based Duplex Immunoassay Reveals an Inverse Correlation of Soluble and Aggregated Mutant huntingtin in Huntington's Disease</span></div><div class="casAuthors">Baldo, Barbara; Paganetti, Paolo; Grueninger, Stephan; Marcellin, David; Kaltenbach, Linda S.; Lo, Donald C.; Semmelroth, Martin; Zivanovic, Andjelija; Abramowski, Dorothee; Smith, Donna; Lotz, Gregor P.; Bates, Gillian P.; Weiss, Andreas</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">264-275</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Huntington's disease (HD) is an inherited neurodegenerative disorder caused by the amplification of a polyglutamine stretch at the N terminus of the huntingtin protein.  N-terminal fragments of the mutant huntingtin (mHtt) aggregate and form intracellular inclusions in brain and peripheral tissues.  Aggregates are an important hallmark of the disease, translating into a high need to quantify them in vitro and in vivo.  We developed a one-step TR-FRET-based immunoassay to quantify sol. and aggregated mHtt in cell and tissue homogenates.  Strikingly, quantification revealed a decrease of sol. mHtt correlating with an increase of aggregated protein in primary neuronal cell cultures, transgenic R6/2, and HdhQ150 knock-in HD mice.  These results emphasize the assay's efficiency for highly sensitive and quant. detection of sol. and aggregated mHtt and its application in high-throughput screening and characterization of HD models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHjXLeXbarcrVg90H21EOLACvtfcHk0lhgFJ1N9xqrCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XjtFWhtLo%253D&md5=ff1f0f82383793c05895bd86ec9cec3a</span></div><a href="/servlet/linkout?suffix=cit106&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2011.12.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2011.12.020%26sid%3Dliteratum%253Aachs%26aulast%3DBaldo%26aufirst%3DB.%26aulast%3DPaganetti%26aufirst%3DP.%26aulast%3DGrueninger%26aufirst%3DS.%26aulast%3DMarcellin%26aufirst%3DD.%26aulast%3DKaltenbach%26aufirst%3DL.%2BS.%26aulast%3DLo%26aufirst%3DD.%2BC.%26aulast%3DSemmelroth%26aufirst%3DM.%26aulast%3DZivanovic%26aufirst%3DA.%26aulast%3DAbramowski%26aufirst%3DD.%26aulast%3DSmith%26aufirst%3DD.%26aulast%3DLotz%26aufirst%3DG.%2BP.%26aulast%3DBates%26aufirst%3DG.%2BP.%26aulast%3DWeiss%26aufirst%3DA.%26atitle%3DTR-FRET-based%2520duplex%2520immunoassay%2520reveals%2520an%2520inverse%2520correlation%2520of%2520soluble%2520and%2520aggregated%2520mutant%2520huntingtin%2520in%2520huntington%25E2%2580%2599s%2520disease%26jtitle%3DChem.%2520Biol.%26date%3D2012%26volume%3D19%26spage%3D264%26epage%3D275%26doi%3D10.1016%2Fj.chembiol.2011.12.020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref107"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref107'); return false;" data-citation="" class="refNumLink">107</a></strong><div class="NLM_citation" id="cit107"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abramowski, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bibel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bodner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chopra, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DiFiglia, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fox, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kegel, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grueninger, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hersch, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Housman, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Régulier, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosas, H. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stefani, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeitlin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bilbe, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paganetti, P.</span></span> <span> </span><span class="NLM_article-title">Single-step detection of mutant huntingtin in animal and human tissues: a bioassay for huntington’s disease</span>. <i>Anal. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>395</i></span>,  <span class="NLM_fpage">8</span>– <span class="NLM_lpage">15</span>, <span class="refDoi"> DOI: 10.1016/j.ab.2009.08.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1016%2Fj.ab.2009.08.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=19664996" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtFKnsb7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=395&publication_year=2009&pages=8-15&author=A.+Weissauthor=D.+Abramowskiauthor=M.+Bibelauthor=R.+Bodnerauthor=V.+Chopraauthor=M.+DiFigliaauthor=J.+Foxauthor=K.+Kegelauthor=C.+Kleinauthor=S.+Grueningerauthor=S.+Herschauthor=D.+Housmanauthor=E.+R%C3%A9gulierauthor=H.+D.+Rosasauthor=M.+Stefaniauthor=S.+Zeitlinauthor=G.+Bilbeauthor=P.+Paganetti&title=Single-step+detection+of+mutant+huntingtin+in+animal+and+human+tissues%3A+a+bioassay+for+huntington%E2%80%99s+disease&doi=10.1016%2Fj.ab.2009.08.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit107R"><div class="casContent"><span class="casTitleNuber">107</span><div class="casTitle"><span class="NLM_cas:atitle">Single-step detection of mutant huntingtin in animal and human tissues: A bioassay for Huntington's disease</span></div><div class="casAuthors">Weiss, Andreas; Abramowski, Dorothee; Bibel, Miriam; Bodner, Ruth; Chopra, Vanita; DiFiglia, Marian; Fox, Jonathan; Kegel, Kimberly; Klein, Corinna; Grueninger, Stephan; Hersch, Steven; Housman, David; Regulier, Etienne; Rosas, H. Diana; Stefani, Muriel; Zeitlin, Scott; Bilbe, Graeme; Paganetti, Paolo</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Biochemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">395</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">8-15</span>CODEN:
                <span class="NLM_cas:coden">ANBCA2</span>;
        ISSN:<span class="NLM_cas:issn">0003-2697</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The genetic mutation causing Huntington's disease is a polyglutamine expansion in the huntingtin protein where more than 37 glutamines cause disease by formation of toxic intracellular fragments, aggregates, and cell death.  Despite a clear pathogenic role for mutant huntingtin, understanding huntingtin expression during the presymptomatic phase of the disease or during disease progression has remained obscure.  Central to clarifying the role in the pathomechanism of disease is the ability to easily and accurately measure mutant huntingtin in accessible human tissue samples as well as cell and animal models.  Here the authors describe a highly sensitive time-resolved Forster resonance energy transfer (FRET) assay for quantification of sol. mutant huntingtin in brain, plasma, and cerebrospinal fluid.  Surprisingly, in mice, sol. huntingtin levels decrease during disease progression, inversely correlating with brain aggregate load.  Mutant huntingtin is easily detected in human brain and blood-derived fractions, providing a utility to assess mutant huntingtin expression during disease course as well as a pharmacodynamic marker for disease-modifying therapeutics targeting expression, cleavage, or degrdn. of mutant huntingtin.  The design of the homogeneous one-step method for huntingtin detection is such that it can be easily applied to measure other proteins of interest.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrla9646FajzrVg90H21EOLACvtfcHk0lhgFJ1N9xqrCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtFKnsb7I&md5=2fe224880e4761ec0566994b18353c8c</span></div><a href="/servlet/linkout?suffix=cit107&amp;dbid=16384&amp;doi=10.1016%2Fj.ab.2009.08.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ab.2009.08.001%26sid%3Dliteratum%253Aachs%26aulast%3DWeiss%26aufirst%3DA.%26aulast%3DAbramowski%26aufirst%3DD.%26aulast%3DBibel%26aufirst%3DM.%26aulast%3DBodner%26aufirst%3DR.%26aulast%3DChopra%26aufirst%3DV.%26aulast%3DDiFiglia%26aufirst%3DM.%26aulast%3DFox%26aufirst%3DJ.%26aulast%3DKegel%26aufirst%3DK.%26aulast%3DKlein%26aufirst%3DC.%26aulast%3DGrueninger%26aufirst%3DS.%26aulast%3DHersch%26aufirst%3DS.%26aulast%3DHousman%26aufirst%3DD.%26aulast%3DR%25C3%25A9gulier%26aufirst%3DE.%26aulast%3DRosas%26aufirst%3DH.%2BD.%26aulast%3DStefani%26aufirst%3DM.%26aulast%3DZeitlin%26aufirst%3DS.%26aulast%3DBilbe%26aufirst%3DG.%26aulast%3DPaganetti%26aufirst%3DP.%26atitle%3DSingle-step%2520detection%2520of%2520mutant%2520huntingtin%2520in%2520animal%2520and%2520human%2520tissues%253A%2520a%2520bioassay%2520for%2520huntington%25E2%2580%2599s%2520disease%26jtitle%3DAnal.%2520Biochem.%26date%3D2009%26volume%3D395%26spage%3D8%26epage%3D15%26doi%3D10.1016%2Fj.ab.2009.08.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref108"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref108'); return false;" data-citation="" class="refNumLink">108</a></strong><div class="NLM_citation" id="cit108"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mizushima, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hatano, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobayashi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kabeya, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suzuki, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tokuhisa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohsumi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshimori, T.</span></span> <span> </span><span class="NLM_article-title">Dissection of autophagosome formation using Apg5-deficient mouse embryonic stem cells</span>. <i>J. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>152</i></span>,  <span class="NLM_fpage">657</span>– <span class="NLM_lpage">668</span>, <span class="refDoi"> DOI: 10.1083/jcb.152.4.657</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref108/cit108&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1083%2Fjcb.152.4.657" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref108/cit108&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=11266458" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref108/cit108&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BD3MXhtlGmtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=152&publication_year=2001&pages=657-668&author=N.+Mizushimaauthor=A.+Yamamotoauthor=M.+Hatanoauthor=Y.+Kobayashiauthor=Y.+Kabeyaauthor=K.+Suzukiauthor=T.+Tokuhisaauthor=Y.+Ohsumiauthor=T.+Yoshimori&title=Dissection+of+autophagosome+formation+using+Apg5-deficient+mouse+embryonic+stem+cells&doi=10.1083%2Fjcb.152.4.657"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit108R"><div class="casContent"><span class="casTitleNuber">108</span><div class="casTitle"><span class="NLM_cas:atitle">Dissection of autophagosome formation using Apg5-deficient mouse embryonic stem cells</span></div><div class="casAuthors">Mizushima, Noboru; Yamamoto, Akitsugu; Hatano, Masahiko; Kobayashi, Yoshinori; Kabeya, Yukiko; Suzuki, Kuninori; Tokuhisa, Takeshi; Ohsumi, Yoshinori; Yoshimori, Tamotsu</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cell Biology</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">152</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">657-667</span>CODEN:
                <span class="NLM_cas:coden">JCLBA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9525</span>.
    
            (<span class="NLM_cas:orgname">Rockefeller University Press</span>)
        </div><div class="casAbstract">In macroautophagy, cytoplasmic components are delivered to lysosomes for degrdn. via autophagosomes that are formed by closure of cupshaped isolation membranes.  However, how the isolation membranes are formed is poorly understood.  We recently found in yeast that a novel ubiquitin-like system, the Apg12-Apg5 conjugation system, is essential for autophagy.  Here we show that mouse Apg12-Apg5 conjugate localizes to the isolation membranes in mouse embryonic stem cells.  Using green fluorescent protein-tagged Apg5, we revealed that the cup-shaped isolation membrane is developed from a small crescent-shaped compartment.  Apg5 localizes on the isolation membrane throughout its elongation process.  To examine the role of Apg5, we generated Apg5-deficient embryonic stem cells, which showed defects in autophagosome formation.  The covalent modification of Apg5 with Apg12 is not required for its membrane targeting, but is essential for involvement of Apg5 in elongation of the isolation membranes.  We also show that Apg12-Apg5 is required for targeting of a mammalian Aut7/Apg8 homolog, LC3, to the isolation membranes.  These results suggest that the Apg12-Apg5 conjugate plays essential roles in isolation membrane development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoskuGRbi_Y5bVg90H21EOLACvtfcHk0lhqZB8mAbf9WQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXhtlGmtrY%253D&md5=99359e2ebf2fcd4b9476368362468247</span></div><a href="/servlet/linkout?suffix=cit108&amp;dbid=16384&amp;doi=10.1083%2Fjcb.152.4.657&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1083%252Fjcb.152.4.657%26sid%3Dliteratum%253Aachs%26aulast%3DMizushima%26aufirst%3DN.%26aulast%3DYamamoto%26aufirst%3DA.%26aulast%3DHatano%26aufirst%3DM.%26aulast%3DKobayashi%26aufirst%3DY.%26aulast%3DKabeya%26aufirst%3DY.%26aulast%3DSuzuki%26aufirst%3DK.%26aulast%3DTokuhisa%26aufirst%3DT.%26aulast%3DOhsumi%26aufirst%3DY.%26aulast%3DYoshimori%26aufirst%3DT.%26atitle%3DDissection%2520of%2520autophagosome%2520formation%2520using%2520Apg5-deficient%2520mouse%2520embryonic%2520stem%2520cells%26jtitle%3DJ.%2520Cell%2520Biol.%26date%3D2001%26volume%3D152%26spage%3D657%26epage%3D668%26doi%3D10.1083%2Fjcb.152.4.657" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref109"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref109'); return false;" data-citation="" class="refNumLink">109</a></strong><div class="NLM_citation" id="cit109"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zoghbi, H. Y.</span></span> <span> </span><span class="NLM_article-title">Strategy to selectively remove mutant proteins could combat neurodegeneration</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>575</i></span>,  <span class="NLM_fpage">57</span>– <span class="NLM_lpage">58</span>, <span class="refDoi"> DOI: 10.1038/d41586-019-03243-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref109/cit109&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1038%2Fd41586-019-03243-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref109/cit109&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=31690850" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref109/cit109&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitFWntL%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=575&publication_year=2019&pages=57-58&author=H.+Y.+Zoghbi&title=Strategy+to+selectively+remove+mutant+proteins+could+combat+neurodegeneration&doi=10.1038%2Fd41586-019-03243-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit109R"><div class="casContent"><span class="casTitleNuber">109</span><div class="casTitle"><span class="NLM_cas:atitle">Strategy to selectively remove mutant proteins could combat neurodegeneration</span></div><div class="casAuthors">Zoghbi, Huda Y.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">575</span>
        (<span class="NLM_cas:issue">7781</span>),
    <span class="NLM_cas:pages">57-58</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Compds. have been found that reduce levels of the harmful protein present in Huntington's disease, without affecting the normal version.  The compds. interact with the mutated protein and the cell's protein-clearance machinery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTDIA6v3B88bVg90H21EOLACvtfcHk0lhqZB8mAbf9WQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitFWntL%252FE&md5=e61d603989cfad6519b0aa9edde5157f</span></div><a href="/servlet/linkout?suffix=cit109&amp;dbid=16384&amp;doi=10.1038%2Fd41586-019-03243-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fd41586-019-03243-7%26sid%3Dliteratum%253Aachs%26aulast%3DZoghbi%26aufirst%3DH.%2BY.%26atitle%3DStrategy%2520to%2520selectively%2520remove%2520mutant%2520proteins%2520could%2520combat%2520neurodegeneration%26jtitle%3DNature%26date%3D2019%26volume%3D575%26spage%3D57%26epage%3D58%26doi%3D10.1038%2Fd41586-019-03243-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref110"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref110'); return false;" data-citation="" class="refNumLink">110</a></strong><div class="NLM_citation" id="cit110"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gennarino, V. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Maio, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jafar-Nejad, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">di Ronza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sayegh, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooper, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orr, H. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sillitoe, R. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zoghbi, H. Y.</span></span> <span> </span><span class="NLM_article-title">Pumilio1 haploinsufficiency leads to SCA1-like neurodegeneration by increasing wild-type ataxin1 levels</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>160</i></span>,  <span class="NLM_fpage">1087</span>– <span class="NLM_lpage">1098</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2015.02.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref110/cit110&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1016%2Fj.cell.2015.02.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref110/cit110&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=25768905" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref110/cit110&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC2MXks1agtbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=160&publication_year=2015&pages=1087-1098&author=V.+A.+Gennarinoauthor=R.+K.+Singhauthor=J.+J.+Whiteauthor=A.+De+Maioauthor=K.+Hanauthor=J.-Y.+Kimauthor=P.+Jafar-Nejadauthor=A.+di+Ronzaauthor=H.+Kangauthor=L.+S.+Sayeghauthor=T.+A.+Cooperauthor=H.+T.+Orrauthor=R.+V.+Sillitoeauthor=H.+Y.+Zoghbi&title=Pumilio1+haploinsufficiency+leads+to+SCA1-like+neurodegeneration+by+increasing+wild-type+ataxin1+levels&doi=10.1016%2Fj.cell.2015.02.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit110R"><div class="casContent"><span class="casTitleNuber">110</span><div class="casTitle"><span class="NLM_cas:atitle">Pumilio1 Haploinsufficiency Leads to SCA1-like Neurodegeneration by Increasing Wild-Type Ataxin1 Levels</span></div><div class="casAuthors">Gennarino, Vincenzo A.; Singh, Ravi K.; White, Joshua J.; De Maio, Antonia; Han, Kihoon; Kim, Ji-Yoen; Jafar-Nejad, Paymaan; di Ronza, Alberto; Kang, Hyojin; Sayegh, Layal S.; Cooper, Thomas A.; Orr, Harry T.; Sillitoe, Roy V.; Zoghbi, Huda Y.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">160</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1087-1098</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Spinocerebellar ataxia type 1 (SCA1) is a paradigmatic neurodegenerative proteinopathy, in which a mutant protein (in this case, ATAXIN1) accumulates in neurons and exerts toxicity; in SCA1, this process causes progressive deterioration of motor coordination.  Seeking to understand how post-translational modification of ATAXIN1 levels influences disease, we discovered that the RNA-binding protein PUMILIO1 (PUM1) not only directly regulates ATAXIN1 but also plays an unexpectedly important role in neuronal function.  Loss of Pum1 caused progressive motor dysfunction and SCA1-like neurodegeneration with motor impairment, primarily by increasing Ataxin1 levels.  Breeding Pum1+/- mice to SCA1 mice (Atxn1154Q/+) exacerbated disease progression, whereas breeding them to Atxn1+/- mice normalized Ataxin1 levels and largely rescued the Pum1+/- phenotype.  Thus, both increased wild-type ATAXIN1 levels and PUM1 haploinsufficiency could contribute to human neurodegeneration.  These results demonstrate the importance of studying post-transcriptional regulation of disease-driving proteins to reveal factors underlying neurodegenerative disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQKCmGA_iyjbVg90H21EOLACvtfcHk0lhqZB8mAbf9WQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXks1agtbc%253D&md5=45678b51683bd54ae255619f9d4b9f51</span></div><a href="/servlet/linkout?suffix=cit110&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2015.02.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2015.02.012%26sid%3Dliteratum%253Aachs%26aulast%3DGennarino%26aufirst%3DV.%2BA.%26aulast%3DSingh%26aufirst%3DR.%2BK.%26aulast%3DWhite%26aufirst%3DJ.%2BJ.%26aulast%3DDe%2BMaio%26aufirst%3DA.%26aulast%3DHan%26aufirst%3DK.%26aulast%3DKim%26aufirst%3DJ.-Y.%26aulast%3DJafar-Nejad%26aufirst%3DP.%26aulast%3Ddi%2BRonza%26aufirst%3DA.%26aulast%3DKang%26aufirst%3DH.%26aulast%3DSayegh%26aufirst%3DL.%2BS.%26aulast%3DCooper%26aufirst%3DT.%2BA.%26aulast%3DOrr%26aufirst%3DH.%2BT.%26aulast%3DSillitoe%26aufirst%3DR.%2BV.%26aulast%3DZoghbi%26aufirst%3DH.%2BY.%26atitle%3DPumilio1%2520haploinsufficiency%2520leads%2520to%2520SCA1-like%2520neurodegeneration%2520by%2520increasing%2520wild-type%2520ataxin1%2520levels%26jtitle%3DCell%26date%3D2015%26volume%3D160%26spage%3D1087%26epage%3D1098%26doi%3D10.1016%2Fj.cell.2015.02.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref111"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref111'); return false;" data-citation="" class="refNumLink">111</a></strong><div class="NLM_citation" id="cit111"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Agarraberes, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dice, J. F.</span></span> <span> </span><span class="NLM_article-title">A molecular chaperone complex at the lysosomal membrane is required for protein translocation</span>. <i>J. Cell Sci.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>114</i></span>,  <span class="NLM_fpage">2491</span>– <span class="NLM_lpage">2499</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref111/cit111&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=11559757" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref111/cit111&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BD3MXlsFamtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=2001&pages=2491-2499&author=F.+A.+Agarraberesauthor=J.+F.+Dice&title=A+molecular+chaperone+complex+at+the+lysosomal+membrane+is+required+for+protein+translocation"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit111R"><div class="casContent"><span class="casTitleNuber">111</span><div class="casTitle"><span class="NLM_cas:atitle">A molecular chaperone complex at the lysosomal membrane is required for protein translocation</span></div><div class="casAuthors">Agarraberes, Fernando A.; Dice, J. Fred</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cell Science</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">114</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">2491-2499</span>CODEN:
                <span class="NLM_cas:coden">JNCSAI</span>;
        ISSN:<span class="NLM_cas:issn">0021-9533</span>.
    
            (<span class="NLM_cas:orgname">Company of Biologists Ltd.</span>)
        </div><div class="casAbstract">A group of cytosolic proteins are targeted to lysosomes for degrdn. in response to serum withdrawal or prolonged starvation by a process termed chaperone-mediated autophagy.  In this proteolytic pathway little is known about how proteins are translocated across lysosomal membranes.  We now show that an isoform of the constitutively expressed protein of the heat shock family of 70 kDa (Hsc70) is assocd. with the cytosolic side of the lysosomal membrane where it binds to substrates of this proteolytic pathway.  Results from coimmunopptn. and colocalization studies indicate that this mol. chaperone forms complexes with other mol. chaperones and cochaperones, including Hsp90, Hsp40, the Hsp70-Hsp90 organizing protein (Hop), the Hsp70-interacting protein (Hip), and the Bcl2-assocd. athanogene 1 protein (BAG-1).  Antibodies against Hip, Hop, Hsp40 and Hsc70 block transport of protein substrates into purified lysosomes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvrsExYbAmeLVg90H21EOLACvtfcHk0lhqZB8mAbf9WQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXlsFamtb8%253D&md5=77407aede1ac38f1c02d716684c99397</span></div><a href="/servlet/linkout?suffix=cit111&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DAgarraberes%26aufirst%3DF.%2BA.%26aulast%3DDice%26aufirst%3DJ.%2BF.%26atitle%3DA%2520molecular%2520chaperone%2520complex%2520at%2520the%2520lysosomal%2520membrane%2520is%2520required%2520for%2520protein%2520translocation%26jtitle%3DJ.%2520Cell%2520Sci.%26date%3D2001%26volume%3D114%26spage%3D2491%26epage%3D2499" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref112"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref112'); return false;" data-citation="" class="refNumLink">112</a></strong><div class="NLM_citation" id="cit112"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cuervo, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dice, J. F.</span></span> <span> </span><span class="NLM_article-title">A Receptor for the selective uptake and degradation of proteins by lysosomes</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>273</i></span>,  <span class="NLM_fpage">501</span>, <span class="refDoi"> DOI: 10.1126/science.273.5274.501</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref112/cit112&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1126%2Fscience.273.5274.501" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref112/cit112&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=8662539" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref112/cit112&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADyaK28Xks1Wmu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=273&publication_year=1996&pages=501&author=A.+M.+Cuervoauthor=J.+F.+Dice&title=A+Receptor+for+the+selective+uptake+and+degradation+of+proteins+by+lysosomes&doi=10.1126%2Fscience.273.5274.501"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit112R"><div class="casContent"><span class="casTitleNuber">112</span><div class="casTitle"><span class="NLM_cas:atitle">A receptor for the selective uptake and degradation of proteins by lysosomes</span></div><div class="casAuthors">Cuervo, Ana Maria; Dice, J. Fred</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, D. C.)</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">273</span>
        (<span class="NLM_cas:issue">5274</span>),
    <span class="NLM_cas:pages">501-503</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Multiple pathways of protein degrdn. operate within cells.  A selective protein import pathway exists for the uptake and degrdn. of particular cytosolic proteins by lysosomes.  Here, the lysosomal membrane glycoprotein LGP96 was identified as a receptor for the selective import and degrdn. of proteins within lysosomes.  Specific substrates of this proteolytic pathway bound to the cytosolic tail of a 96-kilodalton lysosomal membrane protein in two different binding assays.  Overexpression of human LGP96 in Chinese hamster ovary cells increased the activity of the selective lysosomal proteolytic pathway in vivo and in vitro.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpN-TFxOJ9p17Vg90H21EOLACvtfcHk0lhqv3ivjuZ3BQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28Xks1Wmu7o%253D&md5=45dd167e6aedfc1143633026fef176c9</span></div><a href="/servlet/linkout?suffix=cit112&amp;dbid=16384&amp;doi=10.1126%2Fscience.273.5274.501&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.273.5274.501%26sid%3Dliteratum%253Aachs%26aulast%3DCuervo%26aufirst%3DA.%2BM.%26aulast%3DDice%26aufirst%3DJ.%2BF.%26atitle%3DA%2520Receptor%2520for%2520the%2520selective%2520uptake%2520and%2520degradation%2520of%2520proteins%2520by%2520lysosomes%26jtitle%3DScience%26date%3D1996%26volume%3D273%26spage%3D501%26doi%3D10.1126%2Fscience.273.5274.501" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref113"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref113'); return false;" data-citation="" class="refNumLink">113</a></strong><div class="NLM_citation" id="cit113"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kaushik, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuervo, A. M.</span></span> <span> </span><span class="NLM_article-title">Chaperone-mediated autophagy: a unique way to enter the lysosome world</span>. <i>Trends Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">407</span>– <span class="NLM_lpage">417</span>, <span class="refDoi"> DOI: 10.1016/j.tcb.2012.05.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref113/cit113&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1016%2Fj.tcb.2012.05.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref113/cit113&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=22748206" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref113/cit113&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtFaks7%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2012&pages=407-417&author=S.+Kaushikauthor=A.+M.+Cuervo&title=Chaperone-mediated+autophagy%3A+a+unique+way+to+enter+the+lysosome+world&doi=10.1016%2Fj.tcb.2012.05.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit113R"><div class="casContent"><span class="casTitleNuber">113</span><div class="casTitle"><span class="NLM_cas:atitle">Chaperone-mediated autophagy: a unique way to enter the lysosome world</span></div><div class="casAuthors">Kaushik, Susmita; Cuervo, Ana Maria</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Cell Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">407-417</span>CODEN:
                <span class="NLM_cas:coden">TCBIEK</span>;
        ISSN:<span class="NLM_cas:issn">0962-8924</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  All cellular proteins undergo continuous synthesis and degrdn.  This permanent renewal is necessary to maintain a functional proteome and to allow rapid changes in levels of specific proteins with regulatory purposes.  Although for a long time lysosomes were considered unable to contribute to the selective degrdn. of individual proteins, the discovery of chaperone-mediated autophagy (CMA) changed this notion.  Here, we review the characteristics that set CMA apart from other types of lysosomal degrdn. and the subset of mols. that confer cells the capability to identify individual cytosolic proteins and direct them across the lysosomal membrane for degrdn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2-Hs13dlknrVg90H21EOLACvtfcHk0lhqv3ivjuZ3BQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtFaks7%252FI&md5=49b2a4e8b0bc6fe127396b8c451af649</span></div><a href="/servlet/linkout?suffix=cit113&amp;dbid=16384&amp;doi=10.1016%2Fj.tcb.2012.05.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tcb.2012.05.006%26sid%3Dliteratum%253Aachs%26aulast%3DKaushik%26aufirst%3DS.%26aulast%3DCuervo%26aufirst%3DA.%2BM.%26atitle%3DChaperone-mediated%2520autophagy%253A%2520a%2520unique%2520way%2520to%2520enter%2520the%2520lysosome%2520world%26jtitle%3DTrends%2520Cell%2520Biol.%26date%3D2012%26volume%3D22%26spage%3D407%26epage%3D417%26doi%3D10.1016%2Fj.tcb.2012.05.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref114"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref114'); return false;" data-citation="" class="refNumLink">114</a></strong><div class="NLM_citation" id="cit114"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nagai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujikake, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohno, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Higashiyama, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Popiel, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rahadian, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamaguchi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strittmatter, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burke, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toda, T.</span></span> <span> </span><span class="NLM_article-title">Prevention of polyglutamine oligomerization and neurodegeneration by the peptide inhibitor QBP1 in drosophila</span>. <i>Hum. Mol. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">1253</span>– <span class="NLM_lpage">1259</span>, <span class="refDoi"> DOI: 10.1093/hmg/ddg144</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref114/cit114&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1093%2Fhmg%2Fddg144" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref114/cit114&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=12761040" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref114/cit114&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BD3sXkt1Wru7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2003&pages=1253-1259&author=Y.+Nagaiauthor=N.+Fujikakeauthor=K.+Ohnoauthor=H.+Higashiyamaauthor=H.+A.+Popielauthor=J.+Rahadianauthor=M.+Yamaguchiauthor=W.+J.+Strittmatterauthor=J.+R.+Burkeauthor=T.+Toda&title=Prevention+of+polyglutamine+oligomerization+and+neurodegeneration+by+the+peptide+inhibitor+QBP1+in+drosophila&doi=10.1093%2Fhmg%2Fddg144"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit114R"><div class="casContent"><span class="casTitleNuber">114</span><div class="casTitle"><span class="NLM_cas:atitle">Prevention of polyglutamine oligomerization and neurodegeneration by the peptide inhibitor QBP1 in Drosophila</span></div><div class="casAuthors">Nagai, Yoshitaka; Fujikake, Nobuhiro; Ohno, Katsuhito; Higashiyama, Hiroyuki; Popiel, Helena A.; Rahadian, Julia; Yamaguchi, Masamitsu; Strittmatter, Warren J.; Burke, James R.; Toda, Tatsushi</div><div class="citationInfo"><span class="NLM_cas:title">Human Molecular Genetics</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1253-1259</span>CODEN:
                <span class="NLM_cas:coden">HMGEE5</span>;
        ISSN:<span class="NLM_cas:issn">0964-6906</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Polyglutamine (polyQ) diseases are a growing class of inherited neurodegenerative diseases including Huntington's disease, which are caused by abnormal expansions of the polyQ stretch in each unrelated disease protein.  The expanded polyQ stretch is thought to confer toxic properties on the disease proteins through alteration of their conformation leading to pathogenic protein-protein interactions including oligomerization and/or aggregation.  Hypothesizing that mols. with selective binding affinity to the expanded polyQ stretch may interfere with the pathogenic properties, we previously identified Polyglutamine Binding Peptide 1 (QBP1) from combinatorial peptide phage display libraries.  We show here that a tandem repeat of the inhibitor peptide QBP1, (QBP1)2, significantly suppresses polyQ aggregation and polyQ-induced neurodegeneration in the compd. eye of Drosophila polyQ disease models, which express the expanded polyQ protein under the eye specific promoter.  Most importantly, (QBP1)2 expression dramatically rescues premature death of flies expressing the expanded polyQ protein in the nervous system, resulting in the dramatic increase of the median life span from 5.5 to 52 days.  These results suggest that QBP1 can prevent polyQ-induced neurodegeneration in vivo.  We propose that QBP1 prevents polyQ oligomerization and/or aggregation either by altering the toxic conformation of the expanded polyQ stretch, or by simply competing with the expanded polyQ stretches for binding to other expanded polyQ proteins.  The peptide inhibitor QBP1 is a promising candidate with great potential as a therapeutic mol. against the currently untreatable polyQ diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp0TkGk7B7Wv7Vg90H21EOLACvtfcHk0lhqv3ivjuZ3BQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXkt1Wru7s%253D&md5=cfd7e7b12698f6d083868190e22ccb42</span></div><a href="/servlet/linkout?suffix=cit114&amp;dbid=16384&amp;doi=10.1093%2Fhmg%2Fddg144&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fhmg%252Fddg144%26sid%3Dliteratum%253Aachs%26aulast%3DNagai%26aufirst%3DY.%26aulast%3DFujikake%26aufirst%3DN.%26aulast%3DOhno%26aufirst%3DK.%26aulast%3DHigashiyama%26aufirst%3DH.%26aulast%3DPopiel%26aufirst%3DH.%2BA.%26aulast%3DRahadian%26aufirst%3DJ.%26aulast%3DYamaguchi%26aufirst%3DM.%26aulast%3DStrittmatter%26aufirst%3DW.%2BJ.%26aulast%3DBurke%26aufirst%3DJ.%2BR.%26aulast%3DToda%26aufirst%3DT.%26atitle%3DPrevention%2520of%2520polyglutamine%2520oligomerization%2520and%2520neurodegeneration%2520by%2520the%2520peptide%2520inhibitor%2520QBP1%2520in%2520drosophila%26jtitle%3DHum.%2520Mol.%2520Genet.%26date%3D2003%26volume%3D12%26spage%3D1253%26epage%3D1259%26doi%3D10.1093%2Fhmg%2Fddg144" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref115"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref115'); return false;" data-citation="" class="refNumLink">115</a></strong><div class="NLM_citation" id="cit115"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nagai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tucker, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kenan, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henderson, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keene, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strittmatter, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burke, J. R.</span></span> <span> </span><span class="NLM_article-title">Inhibition of polyglutamine protein aggregation and cell death by novel peptides identified by phage display screening</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>275</i></span>,  <span class="NLM_fpage">10437</span>– <span class="NLM_lpage">10442</span>, <span class="refDoi"> DOI: 10.1074/jbc.275.14.10437</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref115/cit115&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1074%2Fjbc.275.14.10437" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref115/cit115&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10744733" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref115/cit115&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BD3cXisFWjt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=275&publication_year=2000&pages=10437-10442&author=Y.+Nagaiauthor=T.+Tuckerauthor=H.+Renauthor=D.+J.+Kenanauthor=B.+S.+Hendersonauthor=J.+D.+Keeneauthor=W.+J.+Strittmatterauthor=J.+R.+Burke&title=Inhibition+of+polyglutamine+protein+aggregation+and+cell+death+by+novel+peptides+identified+by+phage+display+screening&doi=10.1074%2Fjbc.275.14.10437"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit115R"><div class="casContent"><span class="casTitleNuber">115</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of polyglutamine protein aggregation and cell death by novel peptides identified by phage display screening</span></div><div class="casAuthors">Nagai, Yoshitaka; Tucker, Timothy; Ren, Hongzu; Kenan, Daniel J.; Henderson, Barry S.; Keene, Jack D.; Strittmatter, Warren J.; Burke, James R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">275</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">10437-10442</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Proteins with expanded polyglutamine domains cause eight inherited neurodegenerative diseases, including Huntington's, but the mol. mechanism(s) responsible for neuronal degeneration are not yet established.  Expanded polyglutamine domain proteins possess properties that distinguish them from the same proteins with shorter glutamine repeats.  Unlike proteins with short polyglutamine domains, proteins with expanded polyglutamine domains display unique protein interactions, form intracellular aggregates, and adopt a novel conformation that can be recognized by monoclonal antibodies.  Any of these polyglutamine length-dependent properties could be responsible for the pathogenic effects of expanded polyglutamine proteins.  To identify peptides that interfere with pathogenic polyglutamine interactions, the authors screened a combinatorial peptide library expressed on M13 phage pIII protein to identify peptides that preferentially bind pathol.-length polyglutamine domains.  The authors identified six tryptophan-rich peptides that preferentially bind pathol.-length polyglutamine domain proteins.  Polyglutamine-binding peptide 1 (QBP1) potently inhibits polyglutamine protein aggregation in an in vitro assay, while a scrambled sequence has no effect on aggregation.  QBP1 and a tandem repeat of QBP1 also inhibit aggregation of polyglutamine-yellow fluorescent fusion protein in transfected COS-7 cells.  Expression of QBP1 potently inhibits polyglutamine-induced cell death.  Selective inhibition of pathol. interactions of expanded polyglutamine domains with themselves or other proteins may be a useful strategy for preventing disease onset or for slowing progression of the polyglutamine repeat diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrRg3sI3AkjgLVg90H21EOLACvtfcHk0lh7b7Jo6apHpw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXisFWjt7k%253D&md5=c479bed94a9e2fbd695ddecfc6d0f29c</span></div><a href="/servlet/linkout?suffix=cit115&amp;dbid=16384&amp;doi=10.1074%2Fjbc.275.14.10437&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.275.14.10437%26sid%3Dliteratum%253Aachs%26aulast%3DNagai%26aufirst%3DY.%26aulast%3DTucker%26aufirst%3DT.%26aulast%3DRen%26aufirst%3DH.%26aulast%3DKenan%26aufirst%3DD.%2BJ.%26aulast%3DHenderson%26aufirst%3DB.%2BS.%26aulast%3DKeene%26aufirst%3DJ.%2BD.%26aulast%3DStrittmatter%26aufirst%3DW.%2BJ.%26aulast%3DBurke%26aufirst%3DJ.%2BR.%26atitle%3DInhibition%2520of%2520polyglutamine%2520protein%2520aggregation%2520and%2520cell%2520death%2520by%2520novel%2520peptides%2520identified%2520by%2520phage%2520display%2520screening%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2000%26volume%3D275%26spage%3D10437%26epage%3D10442%26doi%3D10.1074%2Fjbc.275.14.10437" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref116"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref116'); return false;" data-citation="" class="refNumLink">116</a></strong><div class="NLM_citation" id="cit116"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sakamoto, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, K. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumagai, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mercurio, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deshaies, R. J.</span></span> <span> </span><span class="NLM_article-title">Protacs: Chimeric molecules that target proteins to the Skp1-Cullin-F box complex for ubiquitination and degradation</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>98</i></span>,  <span class="NLM_fpage">8554</span>, <span class="refDoi"> DOI: 10.1073/pnas.141230798</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref116/cit116&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1073%2Fpnas.141230798" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref116/cit116&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=11438690" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref116/cit116&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BD3MXls1Wisbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=2001&pages=8554&author=K.+M.+Sakamotoauthor=K.+B.+Kimauthor=A.+Kumagaiauthor=F.+Mercurioauthor=C.+M.+Crewsauthor=R.+J.+Deshaies&title=Protacs%3A+Chimeric+molecules+that+target+proteins+to+the+Skp1-Cullin-F+box+complex+for+ubiquitination+and+degradation&doi=10.1073%2Fpnas.141230798"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit116R"><div class="casContent"><span class="casTitleNuber">116</span><div class="casTitle"><span class="NLM_cas:atitle">Protacs: chimeric molecules that target proteins to the Skp1-Cullin-F box complex for ubiquitination and degradation</span></div><div class="casAuthors">Sakamoto, Kathleen M.; Kim, Kyung B.; Kumagai, Akiko; Mercurio, Frank; Crews, Craig M.; Deshaies, Raymond J.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">98</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">8554-8559</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The intracellular levels of many proteins are regulated by ubiquitin-dependent proteolysis.  One of the best-characterized enzymes that catalyzes the attachment of ubiquitin to proteins is a ubiquitin ligase complex, Skp1-Cullin-F box complex contg. Hrt1 (SCF).  We sought to artificially target a protein to the SCF complex for ubiquitination and degrdn.  To this end, we tested methionine aminopeptidase-2 (MetAP-2), which covalently binds the angiogenesis inhibitor ovalicin.  A chimeric compd., protein-targeting chimeric mol. 1 (Protac-1), was synthesized to recruit MetAP-2 to SCF.  One domain of Protac-1 contains the IκBα phosphopeptide that is recognized by the F-box protein β-TRCP, whereas the other domain is composed of ovalicin.  We show that MetAP-2 can be tethered to SCFβ-TRCP, ubiquitinated, and degraded in a Protac-1-dependent manner.  In the future, this approach may be useful for conditional inactivation of proteins, and for targeting disease-causing proteins for destruction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp49fYtuEVlJrVg90H21EOLACvtfcHk0lh7b7Jo6apHpw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXls1Wisbk%253D&md5=540358035222f745f7b6367b38781a21</span></div><a href="/servlet/linkout?suffix=cit116&amp;dbid=16384&amp;doi=10.1073%2Fpnas.141230798&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.141230798%26sid%3Dliteratum%253Aachs%26aulast%3DSakamoto%26aufirst%3DK.%2BM.%26aulast%3DKim%26aufirst%3DK.%2BB.%26aulast%3DKumagai%26aufirst%3DA.%26aulast%3DMercurio%26aufirst%3DF.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26aulast%3DDeshaies%26aufirst%3DR.%2BJ.%26atitle%3DProtacs%253A%2520Chimeric%2520molecules%2520that%2520target%2520proteins%2520to%2520the%2520Skp1-Cullin-F%2520box%2520complex%2520for%2520ubiquitination%2520and%2520degradation%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2001%26volume%3D98%26spage%3D8554%26doi%3D10.1073%2Fpnas.141230798" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref117"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref117'); return false;" data-citation="" class="refNumLink">117</a></strong><div class="NLM_citation" id="cit117"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gopal, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dick, T.</span></span> <span> </span><span class="NLM_article-title">Targeted protein degradation in antibacterial drug discovery?</span>. <i>Prog. Biophys. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>152</i></span>,  <span class="NLM_fpage">10</span>– <span class="NLM_lpage">14</span>, <span class="refDoi"> DOI: 10.1016/j.pbiomolbio.2019.11.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref117/cit117&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1016%2Fj.pbiomolbio.2019.11.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref117/cit117&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=31738980" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref117/cit117&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitF2ltrnM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=152&publication_year=2020&pages=10-14&author=P.+Gopalauthor=T.+Dick&title=Targeted+protein+degradation+in+antibacterial+drug+discovery%3F&doi=10.1016%2Fj.pbiomolbio.2019.11.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit117R"><div class="casContent"><span class="casTitleNuber">117</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted protein degradation in antibacterial drug discovery?</span></div><div class="casAuthors">Gopal, Pooja; Dick, Thomas</div><div class="citationInfo"><span class="NLM_cas:title">Progress in Biophysics & Molecular Biology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">152</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">10-14</span>CODEN:
                <span class="NLM_cas:coden">PBIMAC</span>;
        ISSN:<span class="NLM_cas:issn">0079-6107</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Drug induced degrdn. of a target protein is a novel concept in drug discovery.  Traditionally drugs modulate activity, as opposed to abundance, of their targets.  Degrdn. inducing ligands act catalytically.  Thus, one advantage of target degrdn. over the classical on-target mechanism is that lower drug concn. may be sufficient to cause the desired cellular effects.  The first promoters of target degrdn. were discovered unintentionally: it turned out that some drugs 'accidently' promote degrdn. of their target by the cellular proteolytic machinery.  Elegant methods were developed to target specific proteins of interest for degrdn., thus enabling the rational discovery of degrdn. inducers.  The application of targeted degrdn. has so far been limited to human cells.  Recently, we discovered that an antibacterial drug, the anti-tuberculosis antibiotic pyrazinamide, functions as a promotor of degrdn. of its bacterial target.  Increasing antimicrobial resistance makes the discovery of novel antibiotics more urgent than ever.  Can rational target degrdn. be applied for the discovery of anti-bacterials.  Here, we first discuss briefly some historic examples and then recent approaches in rational target degrdn. for human diseases.  Then, we describe how the first anti-bacterial target degrdn. promoter pyrazinamide triggers removal of its target.  Efforts are under way to exploit this specific mechanistic knowledge for the discovery of next generation pyrazinamide.  We end with the big - and open - question whether targeted protein degrdn. as an approach to anti-bacterial drug discovery can be generalized, similar to what has been achieved in the area of drug discovery for human diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqA8HSCHcOWrVg90H21EOLACvtfcHk0lh7b7Jo6apHpw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitF2ltrnM&md5=12650628c6b916ff1585e417af52c705</span></div><a href="/servlet/linkout?suffix=cit117&amp;dbid=16384&amp;doi=10.1016%2Fj.pbiomolbio.2019.11.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pbiomolbio.2019.11.005%26sid%3Dliteratum%253Aachs%26aulast%3DGopal%26aufirst%3DP.%26aulast%3DDick%26aufirst%3DT.%26atitle%3DTargeted%2520protein%2520degradation%2520in%2520antibacterial%2520drug%2520discovery%253F%26jtitle%3DProg.%2520Biophys.%2520Mol.%2520Biol.%26date%3D2020%26volume%3D152%26spage%3D10%26epage%3D14%26doi%3D10.1016%2Fj.pbiomolbio.2019.11.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref118"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref118'); return false;" data-citation="" class="refNumLink">118</a></strong><div class="NLM_citation" id="cit118"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulk, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Souza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhe-Paganon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">Phthalimide conjugation as a strategy for in vivo target protein degradation</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>348</i></span>,  <span class="NLM_fpage">1376</span>– <span class="NLM_lpage">1381</span>, <span class="refDoi"> DOI: 10.1126/science.aab1433</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref118/cit118&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1126%2Fscience.aab1433" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref118/cit118&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=25999370" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref118/cit118&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC2MXpvFansLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=348&publication_year=2015&pages=1376-1381&author=G.+E.+Winterauthor=D.+L.+Buckleyauthor=J.+Paulkauthor=J.+M.+Robertsauthor=A.+Souzaauthor=S.+Dhe-Paganonauthor=J.+E.+Bradner&title=Phthalimide+conjugation+as+a+strategy+for+in+vivo+target+protein+degradation&doi=10.1126%2Fscience.aab1433"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit118R"><div class="casContent"><span class="casTitleNuber">118</span><div class="casTitle"><span class="NLM_cas:atitle">Phthalimide conjugation as a strategy for in vivo target protein degradation</span></div><div class="casAuthors">Winter, Georg E.; Buckley, Dennis L.; Paulk, Joshiawa; Roberts, Justin M.; Souza, Amanda; Dhe-Paganon, Sirano; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">348</span>
        (<span class="NLM_cas:issue">6241</span>),
    <span class="NLM_cas:pages">1376-1381</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The development of effective pharmacol. inhibitors of multidomain scaffold proteins, notably transcription factors, is a particularly challenging problem.  In part, this is because many small-mol. antagonists disrupt the activity of only one domain in the target protein.  The authors devised a chem. strategy that promotes ligand-dependent target protein degrdn. using as an example the transcriptional coactivator BRD4, a protein crit. for cancer cell growth and survival.  The authors appended a competitive antagonist of BET bromodomains to a phthalimide moiety to hijack the cereblon E3 ubiquitin ligase complex.  The resultant compd., dBET1, induced highly selective cereblon-dependent BET protein degrdn. in vitro and in vivo and delayed leukemia progression in mice.  A second series of probes resulted in selective degrdn. of the cytosolic protein FKBP12.  This chem. strategy for controlling target protein stability may have implications for therapeutically targeting previously intractable proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1ZW4zjRjXi7Vg90H21EOLACvtfcHk0lh7b7Jo6apHpw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXpvFansLk%253D&md5=b0277cc95d318d926b96e7a119066e3b</span></div><a href="/servlet/linkout?suffix=cit118&amp;dbid=16384&amp;doi=10.1126%2Fscience.aab1433&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aab1433%26sid%3Dliteratum%253Aachs%26aulast%3DWinter%26aufirst%3DG.%2BE.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DPaulk%26aufirst%3DJ.%26aulast%3DRoberts%26aufirst%3DJ.%2BM.%26aulast%3DSouza%26aufirst%3DA.%26aulast%3DDhe-Paganon%26aufirst%3DS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DPhthalimide%2520conjugation%2520as%2520a%2520strategy%2520for%2520in%2520vivo%2520target%2520protein%2520degradation%26jtitle%3DScience%26date%3D2015%26volume%3D348%26spage%3D1376%26epage%3D1381%26doi%3D10.1126%2Fscience.aab1433" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62'],'ref63':['cit63'],'ref64':['cit64'],'ref65':['cit65'],'ref66':['cit66'],'ref67':['cit67'],'ref68':['cit68'],'ref69':['cit69'],'ref70':['cit70'],'ref71':['cit71'],'ref72':['cit72'],'ref73':['cit73'],'ref74':['cit74'],'ref75':['cit75'],'ref76':['cit76'],'ref77':['cit77'],'ref78':['cit78'],'ref79':['cit79'],'ref80':['cit80'],'ref81':['cit81'],'ref82':['cit82'],'ref83':['cit83'],'ref84':['cit84'],'ref85':['cit85'],'ref86':['cit86'],'ref87':['cit87'],'ref88':['cit88'],'ref89':['cit89'],'ref90':['cit90'],'ref91':['cit91'],'ref92':['cit92'],'ref93':['cit93'],'ref94':['cit94'],'ref95':['cit95'],'ref96':['cit96'],'ref97':['cit97'],'ref98':['cit98'],'ref99':['cit99'],'ref100':['cit100'],'ref101':['cit101'],'ref102':['cit102'],'ref103':['cit103'],'ref104':['cit104'],'ref105':['cit105'],'ref106':['cit106'],'ref107':['cit107'],'ref108':['cit108'],'ref109':['cit109'],'ref110':['cit110'],'ref111':['cit111'],'ref112':['cit112'],'ref113':['cit113'],'ref114':['cit114'],'ref115':['cit115'],'ref116':['cit116'],'ref117':['cit117'],'ref118':['cit118']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article has not yet been cited by other publications.</p></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01689/20210331/images/medium/jm0c01689_0008.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01689/20210331/images/large/jm0c01689_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01689&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01689/20210331/images/medium/jm0c01689_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01689/20210331/images/large/jm0c01689_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Lysosomes participate in the degradation and recycling of intracellular and extracellular material through autophagy and endocytosis to provide energy and source molecules. Exocytosis contributes to plasma membrane repair, the immune response, bone resorption, and pathogen elimination in stores. Lysosomes can sense nutrients and activate metabolic signal transduction.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01689/20210331/images/large/jm0c01689_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01689&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01689/20210331/images/medium/jm0c01689_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01689/20210331/images/large/jm0c01689_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Techniques for the targeted degradation of biomaterials. (A) PROTACs can degrade certain intracellular proteins through the proteasome. (B) New techniques harnessing the lysosome degradation pathway have the potential to degrade viruses, damaged or excess organelles, protein aggregates, DNA lesions, transmembrane proteins, extracellular proteins, intracellular proteins, and so on.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01689/20210331/images/large/jm0c01689_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01689&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01689/20210331/images/medium/jm0c01689_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01689/20210331/images/large/jm0c01689_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Pathways for the targeted degradation of biological materials by lysosomes. The lysosome is an acidic organelle that can degrade intracellular and extracellular substructures through the activity of acidic hydrolases. Most of the newly synthesized lysosomal enzymes are transported into the lysosome through the M6PR in mammalian cells. However, β-glucocerebrosidase binds to LIMP-2 to target lysosomes. There are two main degradation pathways that involve lysosomes: the degradation of extracellular substances by endocytosis and the degradation of cytoplasmic proteins and organelles by autophagy including macroautophagy, microautophagy, and CMA.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01689/20210331/images/large/jm0c01689_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01689&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01689/20210331/images/medium/jm0c01689_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01689/20210331/images/large/jm0c01689_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Extracellular proteins and transmembrane proteins can be degraded by LYTACs, which are composed of target protein antibodies and ligands for target LTRs. In addition, the asialoglycoprotein receptor (ASGPR) has tissue specificity; therefore, LYTACs targeting ASGPR can degrade transmembrane proteins in specific tissues.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01689/20210331/images/large/jm0c01689_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01689&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01689/20210331/images/medium/jm0c01689_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01689/20210331/images/large/jm0c01689_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. AUTAC degrades cytoplasmic proteins and damaged mitochondria by autophagy. The combination of AUTAC with cytoplasmic proteins or mitochondria can induce K63 polyubiquitination. The ubiquitinated substrates are subsequently recognized by the autophagy receptor SQSTM1/p62, interact with LC3, and are recruited into autophagosomes to be degraded through the selective autophagy pathway. The design and synthesis of four AUTAC molecules that lead to substrate degradation are shown.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01689/20210331/images/large/jm0c01689_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01689&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01689/20210331/images/medium/jm0c01689_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01689/20210331/images/large/jm0c01689_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. ATTEC small molecules can effectively bind LC3 and autophagosomes to degrade pathogenic proteins by autophagy. The researchers first obtained four mHTT–LC3 linker compounds through high-throughput screening and then performed targeted mHTT degradation tests <i>in vivo</i> and <i>in vitro</i>. Fluorescence labeling verified that ATTECs tether mHTT to autophagosomes. Finally, through rotating tripods, balance beams, and gripping power tests, ATTEC can ameliorate the HD-related phenotype.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01689/20210331/images/large/jm0c01689_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01689&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01689/20210331/images/medium/jm0c01689_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01689/20210331/images/large/jm0c01689_0007.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Targeting proteins for the internalization and degradation by chaperone-mediated autophagy (CMA). The amino acid–peptide adaptor molecule fused with a tag for the target protein of interest to a CMA sorting signal containing KFERQ.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01689/20210331/images/large/jm0c01689_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01689&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i22">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_27702" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_27702" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 118 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gómez-Sintes, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boya, P.</span></span> <span> </span><span class="NLM_article-title">Lysosomal membrane permeabilization and cell death</span>. <i>Traffic</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">918</span>– <span class="NLM_lpage">931</span>, <span class="refDoi"> DOI: 10.1111/tra.12613</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1111%2Ftra.12613" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=30125440" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslSrsL3M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2018&pages=918-931&author=F.+Wangauthor=R.+G%C3%B3mez-Sintesauthor=P.+Boya&title=Lysosomal+membrane+permeabilization+and+cell+death&doi=10.1111%2Ftra.12613"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Lysosomal membrane permeabilization and cell death</span></div><div class="casAuthors">Wang, Fengjuan; Gomez-Sintes, Raquel; Boya, Patricia</div><div class="citationInfo"><span class="NLM_cas:title">Traffic (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">918-931</span>CODEN:
                <span class="NLM_cas:coden">TRAFFA</span>;
        ISSN:<span class="NLM_cas:issn">1398-9219</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  Lysosomes are membrane-enclosed organelles that mediate the intracellular degrdn. of macromols.  They play an essential role in calcium regulation and have emerged as key signaling hubs in controlling the nutrient response.  Maintaining lysosomal integrity and function is therefore crucial for cellular homeostasis.  Different forms of stress can induce lysosomal membrane permeabilization (LMP), resulting in the translocation to the cytoplasm of intralysosomal components, such as cathepsins, inducing lysosomal-dependent cell death (LDCD).  Here, we review recent advances that have furthered our understanding of the mol. mechanisms of LMP and the methods used to detect it.  We discuss several endolysosomal damage-response mechanisms that mediate the repair or elimination of compromised lysosomes and summarize the role of LMP and cathepsins in LDCD and other cell death pathways.  Finally, with the emergence of lysosomes as promising therapeutic targets for several human diseases, we review a variety of therapeutic strategies that seek to either destabilize lysosomes or to maintain, enhance or restore lysosomal function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7BHvqj7V7ULVg90H21EOLACvtfcHk0liEGB0zU8It9A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslSrsL3M&md5=f20ef531226f238f550ddf5b3cf91e8a</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1111%2Ftra.12613&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Ftra.12613%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DF.%26aulast%3DG%25C3%25B3mez-Sintes%26aufirst%3DR.%26aulast%3DBoya%26aufirst%3DP.%26atitle%3DLysosomal%2520membrane%2520permeabilization%2520and%2520cell%2520death%26jtitle%3DTraffic%26date%3D2018%26volume%3D19%26spage%3D918%26epage%3D931%26doi%3D10.1111%2Ftra.12613" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yim, W. W.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mizushima, N.</span></span> <span> </span><span class="NLM_article-title">Lysosome biology in autophagy</span>. <i>Cell Discovery</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">6</span>, <span class="refDoi"> DOI: 10.1038/s41421-020-0141-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1038%2Fs41421-020-0141-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=32047650" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BB3cXktlegtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2020&pages=6&author=W.+W.-Y.+Yimauthor=N.+Mizushima&title=Lysosome+biology+in+autophagy&doi=10.1038%2Fs41421-020-0141-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Lysosome biology in autophagy</span></div><div class="casAuthors">Yim, Willa Wen-You; Mizushima, Noboru</div><div class="citationInfo"><span class="NLM_cas:title">Cell Discovery</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">6</span>CODEN:
                <span class="NLM_cas:coden">CDEIBM</span>;
        ISSN:<span class="NLM_cas:issn">2056-5968</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Abstr.: Autophagy is a major intracellular degrdn. system that derives its degradative abilities from the lysosome.  The most well-studied form of autophagy is macroautophagy, which delivers cytoplasmic material to lysosomes via the double-membraned autophagosome.  Other forms of autophagy, namely chaperone-mediated autophagy and microautophagy, occur directly on the lysosome.  Besides providing the means for degrdn., lysosomes are also involved in autophagy regulation and can become substrates of autophagy when damaged.  During autophagy, they exhibit notable changes, including increased acidification, enhanced enzymic activity, and perinuclear localization.  Despite their importance to autophagy, details on autophagy-specific regulation of lysosomes remain relatively scarce.  This review aims to provide a summary of current understanding on the behavior of lysosomes during autophagy and outline unexplored areas of autophagy-specific lysosome research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqpU2zO_7Pq8bVg90H21EOLACvtfcHk0liEGB0zU8It9A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXktlegtLs%253D&md5=bd5096a008d31a55449e6980b2bd9471</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fs41421-020-0141-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41421-020-0141-7%26sid%3Dliteratum%253Aachs%26aulast%3DYim%26aufirst%3DW.%2BW.-Y.%26aulast%3DMizushima%26aufirst%3DN.%26atitle%3DLysosome%2520biology%2520in%2520autophagy%26jtitle%3DCell%2520Discovery%26date%3D2020%26volume%3D6%26spage%3D6%26doi%3D10.1038%2Fs41421-020-0141-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Deus, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yambire, K. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliveira, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raimundo, N.</span></span> <span> </span><span class="NLM_article-title">Mitochondria-lysosome crosstalk: from physiology to neurodegeneration</span>. <i>Trends Mol. Med.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">71</span>– <span class="NLM_lpage">88</span>, <span class="refDoi"> DOI: 10.1016/j.molmed.2019.10.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1016%2Fj.molmed.2019.10.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=31791731" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitF2murvJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2020&pages=71-88&author=C.+M.+Deusauthor=K.+F.+Yambireauthor=P.+J.+Oliveiraauthor=N.+Raimundo&title=Mitochondria-lysosome+crosstalk%3A+from+physiology+to+neurodegeneration&doi=10.1016%2Fj.molmed.2019.10.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Mitochondria-Lysosome Crosstalk: From Physiology to Neurodegeneration</span></div><div class="casAuthors">Deus, Claudia M.; Yambire, King Faisal; Oliveira, Paulo J.; Raimundo, Nuno</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Molecular Medicine</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">71-88</span>CODEN:
                <span class="NLM_cas:coden">TMMRCY</span>;
        ISSN:<span class="NLM_cas:issn">1471-4914</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Cellular function requires coordination between different organelles and metabolic cues.  Mitochondria and lysosomes are essential for cellular metab. as major contributors of chem. energy and building blocks.  It is therefore pivotal for cellular function to coordinate the metabolic roles of mitochondria and lysosomes.  However, these organelles do more than metab., given their function as fundamental signaling platforms in the cell that regulate many key processes such as autophagy, proliferation, and cell death.  Mechanisms of crosstalk between mitochondria and lysosomes are discussed, both under physiol. conditions and in diseases that affect these organelles.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpiRMg_NRB2FrVg90H21EOLACvtfcHk0liEGB0zU8It9A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitF2murvJ&md5=3fd733dfde23717db9bc09bb1ccff10c</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.molmed.2019.10.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molmed.2019.10.009%26sid%3Dliteratum%253Aachs%26aulast%3DDeus%26aufirst%3DC.%2BM.%26aulast%3DYambire%26aufirst%3DK.%2BF.%26aulast%3DOliveira%26aufirst%3DP.%2BJ.%26aulast%3DRaimundo%26aufirst%3DN.%26atitle%3DMitochondria-lysosome%2520crosstalk%253A%2520from%2520physiology%2520to%2520neurodegeneration%26jtitle%3DTrends%2520Mol.%2520Med.%26date%3D2020%26volume%3D26%26spage%3D71%26epage%3D88%26doi%3D10.1016%2Fj.molmed.2019.10.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tooze, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abada, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elazar, Z.</span></span> <span> </span><span class="NLM_article-title">Endocytosis and autophagy: exploitation or cooperation?</span>. <i>Cold Spring Harbor Perspect. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>6</i></span>, <span class="NLM_elocation-id">a018358</span> <span class="refDoi"> DOI: 10.1101/cshperspect.a018358</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1101%2Fcshperspect.a018358" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=24789822" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2014&author=S.+A.+Toozeauthor=A.+Abadaauthor=Z.+Elazar&title=Endocytosis+and+autophagy%3A+exploitation+or+cooperation%3F&doi=10.1101%2Fcshperspect.a018358"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1101%2Fcshperspect.a018358&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fcshperspect.a018358%26sid%3Dliteratum%253Aachs%26aulast%3DTooze%26aufirst%3DS.%2BA.%26aulast%3DAbada%26aufirst%3DA.%26aulast%3DElazar%26aufirst%3DZ.%26atitle%3DEndocytosis%2520and%2520autophagy%253A%2520exploitation%2520or%2520cooperation%253F%26jtitle%3DCold%2520Spring%2520Harbor%2520Perspect.%2520Biol.%26date%3D2014%26volume%3D6%26doi%3D10.1101%2Fcshperspect.a018358" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Saftig, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klumperman, J.</span></span> <span> </span><span class="NLM_article-title">Lysosome biogenesis and lysosomal membrane proteins: trafficking meets function</span>. <i>Nat. Rev. Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">623</span>– <span class="NLM_lpage">635</span>, <span class="refDoi"> DOI: 10.1038/nrm2745</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1038%2Fnrm2745" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=19672277" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BD1MXpsleqsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2009&pages=623-635&author=P.+Saftigauthor=J.+Klumperman&title=Lysosome+biogenesis+and+lysosomal+membrane+proteins%3A+trafficking+meets+function&doi=10.1038%2Fnrm2745"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Lysosome biogenesis and lysosomal membrane proteins: trafficking meets function</span></div><div class="casAuthors">Saftig, Paul; Klumperman, Judith</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">623-635</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Lysosomes are the primary catabolic compartments of eukaryotic cells.  They degrade extracellular material that has been internalized by endocytosis and intracellular components that have been sequestered by autophagy.  In addn., specialized cells contain lysosome-related organelles that store and secrete proteins for cell-type-specific functions.  The functioning of a healthy cell is dependent on the proper targeting of newly synthesized lysosomal proteins.  Accumulating evidence suggests that there are multiple lysosomal delivery pathways that together allow the regulated and sequential deposition of lysosomal components.  The importance of lysosomal trafficking pathways is emphasized by recent findings that reveal new roles for lysosomal membrane proteins in cellular physiol. and in an increasing no. of diseases that are characterized by defects in lysosome biogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpby5TZ_C2_I7Vg90H21EOLACvtfcHk0ljk1Ztot5r9ZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXpsleqsrc%253D&md5=e45f191f35eba89f8936a7a9a9f76cdf</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnrm2745&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm2745%26sid%3Dliteratum%253Aachs%26aulast%3DSaftig%26aufirst%3DP.%26aulast%3DKlumperman%26aufirst%3DJ.%26atitle%3DLysosome%2520biogenesis%2520and%2520lysosomal%2520membrane%2520proteins%253A%2520trafficking%2520meets%2520function%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2009%26volume%3D10%26spage%3D623%26epage%3D635%26doi%3D10.1038%2Fnrm2745" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chieregatti, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meldolesi, J.</span></span> <span> </span><span class="NLM_article-title">Regulated exocytosis: new organelles for non-secretory purposes</span>. <i>Nat. Rev. Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">181</span>– <span class="NLM_lpage">187</span>, <span class="refDoi"> DOI: 10.1038/nrm1572</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1038%2Fnrm1572" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=15688003" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BD2MXoslOnuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2005&pages=181-187&author=E.+Chieregattiauthor=J.+Meldolesi&title=Regulated+exocytosis%3A+new+organelles+for+non-secretory+purposes&doi=10.1038%2Fnrm1572"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Opinion: Regulated exocytosis: new organelles for non-secretory purposes</span></div><div class="casAuthors">Chieregatti, Evelina; Meldolesi, Jacopo</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">181-187</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A discussion and review.  Regulated exocytosis is a process in which the membranes of cytoplasmic organelles fuse with the plasma membrane in response to stimulation.  In many cases (secretory exocytoses), the process functions to secrete specific products that are segregated in the organelle lumen (for example, neurotransmitters, hormones and enzymes) to the extracellular space.  In other cases ('non-secretory exocytoses'), it functions to transfer the organelle membrane and its components to the cell surface.  Here, the general properties of non-secretory exocytoses are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqMb4F1DBfmsrVg90H21EOLACvtfcHk0ljk1Ztot5r9ZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXoslOnuw%253D%253D&md5=8d0d6f4b2f5d713ae30691a64de91794</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnrm1572&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm1572%26sid%3Dliteratum%253Aachs%26aulast%3DChieregatti%26aufirst%3DE.%26aulast%3DMeldolesi%26aufirst%3DJ.%26atitle%3DRegulated%2520exocytosis%253A%2520new%2520organelles%2520for%2520non-secretory%2520purposes%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2005%26volume%3D6%26spage%3D181%26epage%3D187%26doi%3D10.1038%2Fnrm1572" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M. C.</span></span> <span> </span><span class="NLM_article-title">Molecular mechanisms of lysosome and nucleus communication</span>. <i>Trends Biochem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">978</span>– <span class="NLM_lpage">991</span>, <span class="refDoi"> DOI: 10.1016/j.tibs.2020.06.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1016%2Fj.tibs.2020.06.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=32624271" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BB3cXht1WlsLjJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2020&pages=978-991&author=Q.+Zhaoauthor=S.+M.+Gaoauthor=M.+C.+Wang&title=Molecular+mechanisms+of+lysosome+and+nucleus+communication&doi=10.1016%2Fj.tibs.2020.06.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Mechanisms of Lysosome and Nucleus Communication</span></div><div class="casAuthors">Zhao, Qian; Gao, Shihong Max; Wang, Meng C.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Biochemical Sciences</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">978-991</span>CODEN:
                <span class="NLM_cas:coden">TBSCDB</span>;
        ISSN:<span class="NLM_cas:issn">0968-0004</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Lysosomes transcend the role of degrdn. stations, acting as key nodes for interorganelle crosstalk and signal transduction.  Lysosomes communicate with the nucleus through phys. proximity and functional interaction.  In response to external and internal stimuli, lysosomes actively adjust their distribution between peripheral and perinuclear regions and modulate lysosome-nucleus signaling pathways; in turn, the nucleus fine-tunes lysosomal biogenesis and functions through transcriptional controls.  Changes in coordination between these two essential organelles are assocd. with metabolic disorders, neurodegenerative diseases, and aging.  In this review, we address recent advances in lysosome-nucleus communication by multi-tiered regulatory mechanisms and discuss how these regulations couple metabolic inputs with organellar motility, cellular signaling, and transcriptional network.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohlX7bTRTVW7Vg90H21EOLACvtfcHk0ljk1Ztot5r9ZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXht1WlsLjJ&md5=efef671e93a93e45e44f00199ea06dcb</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.tibs.2020.06.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tibs.2020.06.004%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DQ.%26aulast%3DGao%26aufirst%3DS.%2BM.%26aulast%3DWang%26aufirst%3DM.%2BC.%26atitle%3DMolecular%2520mechanisms%2520of%2520lysosome%2520and%2520nucleus%2520communication%26jtitle%3DTrends%2520Biochem.%2520Sci.%26date%3D2020%26volume%3D45%26spage%3D978%26epage%3D991%26doi%3D10.1016%2Fj.tibs.2020.06.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bonam, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, S.</span></span> <span> </span><span class="NLM_article-title">Lysosomes as a therapeutic target</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">923</span>– <span class="NLM_lpage">948</span>, <span class="refDoi"> DOI: 10.1038/s41573-019-0036-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1038%2Fs41573-019-0036-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=31477883" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhs12gurrM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=923-948&author=S.+R.+Bonamauthor=F.+Wangauthor=S.+Muller&title=Lysosomes+as+a+therapeutic+target&doi=10.1038%2Fs41573-019-0036-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Lysosomes as a therapeutic target</span></div><div class="casAuthors">Bonam, Srinivasa Reddy; Wang, Fengjuan; Muller, Sylviane</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">923-948</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Lysosomes are membrane-bound organelles with roles in processes involved in degrading and recycling cellular waste, cellular signalling and energy metab.  Defects in genes encoding lysosomal proteins cause lysosomal storage disorders, in which enzyme replacement therapy has proved successful.  Growing evidence also implicates roles for lysosomal dysfunction in more common diseases including inflammatory and autoimmune disorders, neurodegenerative diseases, cancer and metabolic disorders.  With a focus on lysosomal dysfunction in autoimmune disorders and neurodegenerative diseases - including lupus, rheumatoid arthritis, multiple sclerosis, Alzheimer disease and Parkinson disease - this Review critically analyses progress and opportunities for therapeutically targeting lysosomal proteins and processes, particularly with small mols. and peptide drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-J0ZL8ZiXtrVg90H21EOLACvtfcHk0lhzFtZ_SowLUA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhs12gurrM&md5=705fc139e1d00bf2e1fb4267585ed9c7</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fs41573-019-0036-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41573-019-0036-1%26sid%3Dliteratum%253Aachs%26aulast%3DBonam%26aufirst%3DS.%2BR.%26aulast%3DWang%26aufirst%3DF.%26aulast%3DMuller%26aufirst%3DS.%26atitle%3DLysosomes%2520as%2520a%2520therapeutic%2520target%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2019%26volume%3D18%26spage%3D923%26epage%3D948%26doi%3D10.1038%2Fs41573-019-0036-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horak, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Molkentin, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span> <span> </span><span class="NLM_article-title">Proteasome functional insufficiency activates the calcineurin-NFAT pathway in cardiomyocytes and promotes maladaptive remodelling of stressed mouse hearts</span>. <i>Cardiovasc. Res.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>88</i></span>,  <span class="NLM_fpage">424</span>– <span class="NLM_lpage">433</span>, <span class="refDoi"> DOI: 10.1093/cvr/cvq217</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1093%2Fcvr%2Fcvq217" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=20601385" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVSltLvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=88&publication_year=2010&pages=424-433&author=M.+Tangauthor=J.+Liauthor=W.+Huangauthor=H.+Suauthor=Q.+Liangauthor=Z.+Tianauthor=K.+M.+Horakauthor=J.+D.+Molkentinauthor=X.+Wang&title=Proteasome+functional+insufficiency+activates+the+calcineurin-NFAT+pathway+in+cardiomyocytes+and+promotes+maladaptive+remodelling+of+stressed+mouse+hearts&doi=10.1093%2Fcvr%2Fcvq217"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Proteasome functional insufficiency activates the calcineurin-NFAT pathway in cardiomyocytes and promotes maladaptive remodelling of stressed mouse hearts</span></div><div class="casAuthors">Tang, Mingxin; Li, Jie; Huang, Wei; Su, Huabo; Liang, Qiangrong; Tian, Zongwen; Horak, Kathleen M.; Molkentin, Jeffery D.; Wang, Xuejun</div><div class="citationInfo"><span class="NLM_cas:title">Cardiovascular Research</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">88</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">424-433</span>CODEN:
                <span class="NLM_cas:coden">CVREAU</span>;
        ISSN:<span class="NLM_cas:issn">0008-6363</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Aims Proteasome functional insufficiency (PFI) may play an important role in the progression of congestive heart failure but the underlying mol. mechanism is poorly understood.  Calcineurin and nuclear factor of activated T-cells (NFAT) are degraded by the proteasome, and the calcineurin-NFAT pathway mediates cardiac remodelling.  The present study examd. the hypothesis that PFI activates the calcineurin-NFAT pathway and promotes maladaptive remodelling of the heart.  Methods and results Using a reporter gene assay, we found that pharmacol. inhibition of 20S proteasomes stimulated NFAT transactivation in both mouse hearts and cultured adult mouse cardiomyocytes.  Proteasome inhibition stimulated NFAT nuclear translocation in a calcineurin-dependent manner and led to a maladaptive cell shape change in cultured neonatal rat ventricular myocytes.  Proteasome inhibition facilitated left ventricular dilatation and functional decompensation and increased fatality in mice with aortic constriction while causing cardiac hypertrophy in the sham surgery group.  It was further revealed that both calcineurin protein levels and NFAT transactivation were markedly increased in the mouse hearts with desmin-related cardiomyopathy and severe PFI.  Expression of an aggregation-prone mutant desmin also directly increased calcineurin protein levels in cultured cardiomyocytes.  Conclusions The calcineurin-NFAT pathway in the heart can be activated by proteasome inhibition and is activated in the heart of a mouse model of desmin-related cardiomyopathy that is characterized by severe PFI.  The interplay between PFI and the calcineurin-NFAT pathway may contribute to the pathol. remodelling of cardiomyocytes characteristic of congestive heart failure.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqcCPCW33B9I7Vg90H21EOLACvtfcHk0lhzFtZ_SowLUA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsVSltLvI&md5=fe920fc0f006dd0c98fa7308063954b7</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1093%2Fcvr%2Fcvq217&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fcvr%252Fcvq217%26sid%3Dliteratum%253Aachs%26aulast%3DTang%26aufirst%3DM.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DHuang%26aufirst%3DW.%26aulast%3DSu%26aufirst%3DH.%26aulast%3DLiang%26aufirst%3DQ.%26aulast%3DTian%26aufirst%3DZ.%26aulast%3DHorak%26aufirst%3DK.%2BM.%26aulast%3DMolkentin%26aufirst%3DJ.%2BD.%26aulast%3DWang%26aufirst%3DX.%26atitle%3DProteasome%2520functional%2520insufficiency%2520activates%2520the%2520calcineurin-NFAT%2520pathway%2520in%2520cardiomyocytes%2520and%2520promotes%2520maladaptive%2520remodelling%2520of%2520stressed%2520mouse%2520hearts%26jtitle%3DCardiovasc.%2520Res.%26date%3D2010%26volume%3D88%26spage%3D424%26epage%3D433%26doi%3D10.1093%2Fcvr%2Fcvq217" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parajuli, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewno, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bedford, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mayer, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cui, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span> <span> </span><span class="NLM_article-title">The calcineurin-TFEB-p62 pathway mediates the activation of cardiac macroautophagy by proteasomal malfunction</span>. <i>Circ. Res.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>127</i></span>,  <span class="NLM_fpage">502</span>– <span class="NLM_lpage">518</span>, <span class="refDoi"> DOI: 10.1161/CIRCRESAHA.119.316007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1161%2FCIRCRESAHA.119.316007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=32366200" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhsV2mt73N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=127&publication_year=2020&pages=502-518&author=B.+Panauthor=J.+Liauthor=N.+Parajuliauthor=Z.+Tianauthor=P.+Wuauthor=M.+T.+Lewnoauthor=J.+Zouauthor=W.+Wangauthor=L.+Bedfordauthor=R.+J.+Mayerauthor=J.+Fangauthor=J.+Liuauthor=T.+Cuiauthor=H.+Suauthor=X.+Wang&title=The+calcineurin-TFEB-p62+pathway+mediates+the+activation+of+cardiac+macroautophagy+by+proteasomal+malfunction&doi=10.1161%2FCIRCRESAHA.119.316007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">The Calcineurin-TFEB-p62 Pathway Mediates the Activation of Cardiac Macroautophagy by Proteasomal Malfunction</span></div><div class="casAuthors">Pan, Bo; Li, Jie; Parajuli, Nirmal; Tian, Zongwen; Wu, Penglong; Lewno, Megan T.; Zou, Jianqiu; Wang, Wenjuan; Bedford, Lynn; Mayer, R. John; Fang, Jing; Liu, Jinbao; Cui, Taixing; Su, Huabo; Wang, Xuejun</div><div class="citationInfo"><span class="NLM_cas:title">Circulation Research</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">127</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">502-518</span>CODEN:
                <span class="NLM_cas:coden">CIRUAL</span>;
        ISSN:<span class="NLM_cas:issn">0009-7330</span>.
    
            (<span class="NLM_cas:orgname">Lippincott Williams & Wilkins</span>)
        </div><div class="casAbstract">Objective: Systemic proteasome inhibition (PSMI) was shown to increase p62/SQSTM1 expression and induce myocardial macroautophagy.  Methods and results: Myocardial macroautophagy, TFEB (transcription factor EB) expression and activity, and p62 expression were markedly increased in mice with either cardiomyocyte-restricted ablation of Psmc1 (an essential proteasome subunit gene) or pharmacol. PSMI.  In cultured cardiomyocytes, PSMI-induced increases in TFEB activation and p62 expression were blunted by pharmacol. and genetic calcineurin inhibition and by siRNA-mediated Molcn1 silencing.  Bortezomib-induced left ventricular wall thickening and diastolic malfunction was exacerbated by p62 deficiency.  In cultured cardiomyocytes from wild type mice but not p62-KO mice, PSMI induced increases in LC3-II flux and the lysosomal removal of ubiquitinated proteins.  Myocardial TFEB activation by PSMI as reflected by TFEB nuclear localization and target gene expression was strikingly less in p62-KO mice compared with wild type mice.  Conclusions: (1) The activation of cardiac macroautophagy by proteasomal malfunction is mediated by the Mocln1-calcineurin-TFEB-p62 pathway; (2) p62 unexpectedly exerts a feed-forward effect on TFEB activation by proteasome malfunction; and (3) targeting the Mcoln1-calcineurin-TFEB-p62 pathway provide new means to intervene cardiac autophagic-lysosomal pathway activation during proteasome malfunction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_tFYmiHwBnrVg90H21EOLACvtfcHk0limEahfeiqEHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhsV2mt73N&md5=7340faa0f31afcdd32cb4ee83d4fa5ea</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1161%2FCIRCRESAHA.119.316007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1161%252FCIRCRESAHA.119.316007%26sid%3Dliteratum%253Aachs%26aulast%3DPan%26aufirst%3DB.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DParajuli%26aufirst%3DN.%26aulast%3DTian%26aufirst%3DZ.%26aulast%3DWu%26aufirst%3DP.%26aulast%3DLewno%26aufirst%3DM.%2BT.%26aulast%3DZou%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DBedford%26aufirst%3DL.%26aulast%3DMayer%26aufirst%3DR.%2BJ.%26aulast%3DFang%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DCui%26aufirst%3DT.%26aulast%3DSu%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DX.%26atitle%3DThe%2520calcineurin-TFEB-p62%2520pathway%2520mediates%2520the%2520activation%2520of%2520cardiac%2520macroautophagy%2520by%2520proteasomal%2520malfunction%26jtitle%3DCirc.%2520Res.%26date%3D2020%26volume%3D127%26spage%3D502%26epage%3D518%26doi%3D10.1161%2FCIRCRESAHA.119.316007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ding, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fei, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, B.</span></span> <span> </span><span class="NLM_article-title">Emerging new concepts of degrader technologies</span>. <i>Trends Pharmacol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">464</span>– <span class="NLM_lpage">474</span>, <span class="refDoi"> DOI: 10.1016/j.tips.2020.04.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1016%2Fj.tips.2020.04.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=32416934" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BB3cXovVChurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2020&pages=464-474&author=Y.+Dingauthor=Y.+Feiauthor=B.+Lu&title=Emerging+new+concepts+of+degrader+technologies&doi=10.1016%2Fj.tips.2020.04.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Emerging New Concepts of Degrader Technologies</span></div><div class="casAuthors">Ding, Yu; Fei, Yiyan; Lu, Boxun</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Pharmacological Sciences</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">464-474</span>CODEN:
                <span class="NLM_cas:coden">TPHSDY</span>;
        ISSN:<span class="NLM_cas:issn">0165-6147</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Traditional drug discovery focuses on identifying direct inhibitors of target proteins.  This typically relies on a measurable biochem. readout and accessible binding sites whose occupancy influences the function of the target protein.  These requirements preclude many disease-causing proteins from being 'druggable' targets, and these proteins are categorized as 'undruggable'.  The proteolysis-targeting chimera (PROTAC) technol. provides powerful tools to degrade these undruggable targets and has become a promising approach for drug discovery.  However, the PROTAC technol. has some limitations, and emerging new degrader technologies may greatly broaden the spectrum of targets that could be selectively degraded by harnessing a second major degrdn. pathway in cells.  We review key emerging technologies that exploit the lysosomal degrdn. pathway and discuss their potential applications and limitations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrlDwQwefE2UrVg90H21EOLACvtfcHk0limEahfeiqEHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXovVChurk%253D&md5=8614b50f72ee30c80d965db683cfd194</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.tips.2020.04.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tips.2020.04.005%26sid%3Dliteratum%253Aachs%26aulast%3DDing%26aufirst%3DY.%26aulast%3DFei%26aufirst%3DY.%26aulast%3DLu%26aufirst%3DB.%26atitle%3DEmerging%2520new%2520concepts%2520of%2520degrader%2520technologies%26jtitle%3DTrends%2520Pharmacol.%2520Sci.%26date%3D2020%26volume%3D41%26spage%3D464%26epage%3D474%26doi%3D10.1016%2Fj.tips.2020.04.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dybas, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herrmann, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weitzman, M. D.</span></span> <span> </span><span class="NLM_article-title">Ubiquitination at the interface of tumor viruses and DNA damage responses</span>. <i>Curr. Opin. Virol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">40</span>– <span class="NLM_lpage">47</span>, <span class="refDoi"> DOI: 10.1016/j.coviro.2018.08.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1016%2Fj.coviro.2018.08.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=30261451" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhs12nt7jO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2018&pages=40-47&author=J.+M.+Dybasauthor=C.+Herrmannauthor=M.+D.+Weitzman&title=Ubiquitination+at+the+interface+of+tumor+viruses+and+DNA+damage+responses&doi=10.1016%2Fj.coviro.2018.08.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Ubiquitination at the interface of tumor viruses and DNA damage responses</span></div><div class="casAuthors">Dybas, Joseph M.; Herrmann, Christin; Weitzman, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Virology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">40-47</span>CODEN:
                <span class="NLM_cas:coden">COVUAF</span>;
        ISSN:<span class="NLM_cas:issn">1879-6257</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Viruses exploit cellular ubiquitination machinery to shape the host proteome and promote productive infection.  Among the cellular processes influenced by viral manipulation of ubiquitination is the DNA damage response (DDR), a network of cellular signaling pathways that sense and respond to genomic damage.  This host-pathogen interaction is particularly important during virus replication and transformation by DNA tumor viruses.  Manipulating DDR pathways can promote virus replication but also impacts host genomic instability, potentially leading to cellular transformation and tumor formation.  We review ways in which viruses are known to hijack the cellular ubiquitin system to reshape host DDR pathways.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr1tDS_SCm3WrVg90H21EOLACvtfcHk0limEahfeiqEHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhs12nt7jO&md5=a09b76664af51715fc0d65e661c5189d</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.coviro.2018.08.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.coviro.2018.08.017%26sid%3Dliteratum%253Aachs%26aulast%3DDybas%26aufirst%3DJ.%2BM.%26aulast%3DHerrmann%26aufirst%3DC.%26aulast%3DWeitzman%26aufirst%3DM.%2BD.%26atitle%3DUbiquitination%2520at%2520the%2520interface%2520of%2520tumor%2520viruses%2520and%2520DNA%2520damage%2520responses%26jtitle%3DCurr.%2520Opin.%2520Virol.%26date%3D2018%26volume%3D32%26spage%3D40%26epage%3D47%26doi%3D10.1016%2Fj.coviro.2018.08.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mahon, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krogan, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craik, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pick, E.</span></span> <span> </span><span class="NLM_article-title">Cullin E3 ligases and their rewiring by viral factors</span>. <i>Biomolecules</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">897</span>– <span class="NLM_lpage">930</span>, <span class="refDoi"> DOI: 10.3390/biom4040897</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.3390%2Fbiom4040897" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=25314029" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvV2isrzI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2014&pages=897-930&author=C.+Mahonauthor=N.+J.+Kroganauthor=C.+S.+Craikauthor=E.+Pick&title=Cullin+E3+ligases+and+their+rewiring+by+viral+factors&doi=10.3390%2Fbiom4040897"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Cullin E3 ligases and their rewiring by viral factors</span></div><div class="casAuthors">Mahon, Cathal; Krogan, Nevan J.; Craik, Charles S.; Pick, Elah</div><div class="citationInfo"><span class="NLM_cas:title">Biomolecules</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">897-930, 34 pp.</span>CODEN:
                <span class="NLM_cas:coden">BIOMHC</span>;
        ISSN:<span class="NLM_cas:issn">2218-273X</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">A review.  The ability of viruses to subvert host pathways is central in disease pathogenesis.  Over the past decade, a crit. role for the Ubiquitin Proteasome System (UPS) in counteracting host immune factors during viral infection has emerged.  This counteraction is commonly achieved by the expression of viral proteins capable of sequestering host ubiquitin E3 ligases and their regulators.  In particular, many viruses hijack members of the Cullin-RING E3 Ligase (CRL) family.  Viruses interact in many ways with CRLs in order to impact their ligase activity; one key recurring interaction involves re-directing CRL complexes to degrade host targets that are otherwise not degraded within host cells.  Removal of host immune factors by this mechanism creates a more amenable cellular environment for viral propagation.  To date, a small no. of target host factors have been identified, many of which are degraded via a CRL-proteasome pathway.  Substantial effort within the field is ongoing to uncover the identities of further host proteins targeted in this fashion and the underlying mechanisms driving their turnover by the UPS.  Elucidation of these targets and mechanisms will provide appealing anti-viral therapeutic opportunities.  This review is focused on the many methods used by viruses to perturb host CRLs, focusing on substrate sequestration and viral regulation of E3 activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGowiATFlVQnE7Vg90H21EOLACvtfcHk0limEahfeiqEHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvV2isrzI&md5=dec804942b4aec1f00a77a3570c95b47</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.3390%2Fbiom4040897&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fbiom4040897%26sid%3Dliteratum%253Aachs%26aulast%3DMahon%26aufirst%3DC.%26aulast%3DKrogan%26aufirst%3DN.%2BJ.%26aulast%3DCraik%26aufirst%3DC.%2BS.%26aulast%3DPick%26aufirst%3DE.%26atitle%3DCullin%2520E3%2520ligases%2520and%2520their%2520rewiring%2520by%2520viral%2520factors%26jtitle%3DBiomolecules%26date%3D2014%26volume%3D4%26spage%3D897%26epage%3D930%26doi%3D10.3390%2Fbiom4040897" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dharmasiri, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dharmasiri, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Estelle, M.</span></span> <span> </span><span class="NLM_article-title">The F-box protein TIR1 is an auxin receptor</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>435</i></span>,  <span class="NLM_fpage">441</span>– <span class="NLM_lpage">445</span>, <span class="refDoi"> DOI: 10.1038/nature03543</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1038%2Fnature03543" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=15917797" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BD2MXksVeisrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=435&publication_year=2005&pages=441-445&author=N.+Dharmasiriauthor=S.+Dharmasiriauthor=M.+Estelle&title=The+F-box+protein+TIR1+is+an+auxin+receptor&doi=10.1038%2Fnature03543"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">The F-box protein TIR1 is an auxin receptor</span></div><div class="casAuthors">Dharmasiri, Nihal; Dharmasiri, Sunethra; Estelle, Mark</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">435</span>
        (<span class="NLM_cas:issue">7041</span>),
    <span class="NLM_cas:pages">441-445</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The plant hormone auxin regulates diverse aspects of plant growth and development.  Recent studies indicate that auxin acts by promoting the degrdn. of the Aux/IAA transcriptional repressors through the action of the ubiquitin protein ligase SCFTIR1.  The nature of the signalling cascade that leads to this effect is not known.  However, recent studies indicate that the auxin receptor and other signalling components involved in this response are sol. factors.  Using an in vitro pull-down assay, we demonstrate that the interaction between transport inhibitor response 1 (TIR1) and Aux/IAA proteins does not require stable modification of either protein.  Instead auxin promotes the Aux/IAA-SCFTIR1 interaction by binding directly to SCFTIR1.  We further show that the loss of TIR1 and three related F-box proteins eliminates saturable auxin binding in plant exts.  Finally, TIR1 synthesized in insect cells binds Aux/IAA proteins in an auxin-dependent manner.  Together, these results indicate that TIR1 is an auxin receptor that mediates Aux/IAA degrdn. and auxin-regulated transcription.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqz0XlTe0QR7Vg90H21EOLACvtfcHk0lhQFlzv-8hjHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXksVeisrs%253D&md5=db87d9696f0bdb7003b79a7a95ab32d8</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1038%2Fnature03543&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature03543%26sid%3Dliteratum%253Aachs%26aulast%3DDharmasiri%26aufirst%3DN.%26aulast%3DDharmasiri%26aufirst%3DS.%26aulast%3DEstelle%26aufirst%3DM.%26atitle%3DThe%2520F-box%2520protein%2520TIR1%2520is%2520an%2520auxin%2520receptor%26jtitle%3DNature%26date%3D2005%26volume%3D435%26spage%3D441%26epage%3D445%26doi%3D10.1038%2Fnature03543" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Burslem, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Proteolysis-targeting chimeras as therapeutics and tools for biological discovery</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>181</i></span>,  <span class="NLM_fpage">102</span>– <span class="NLM_lpage">114</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2019.11.031</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1016%2Fj.cell.2019.11.031" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=31955850" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BB3cXht1Sitrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=181&publication_year=2020&pages=102-114&author=G.+M.+Burslemauthor=C.+M.+Crews&title=Proteolysis-targeting+chimeras+as+therapeutics+and+tools+for+biological+discovery&doi=10.1016%2Fj.cell.2019.11.031"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Proteolysis-Targeting Chimeras as Therapeutics and Tools for Biological Discovery</span></div><div class="casAuthors">Burslem, George M.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">181</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">102-114</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  New biol. tools provide new techniques to probe fundamental biol. processes.  Here we describe the burgeoning field of proteolysis-targeting chimeras (PROTACs), which are capable of modulating protein concns. at a post-translational level by co-opting the ubiquitin-proteasome system.  We describe the PROTAC technol. and its application to drug discovery and provide examples where PROTACs have enabled novel biol. insights.  Furthermore, we provide a workflow for PROTAC development and use and discuss the benefits and issues assocd. with PROTACs.  Finally, we compare PROTAC-mediated protein-level modulation with other technologies, such as RNAi and genome editing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXgumsEakVWrVg90H21EOLACvtfcHk0lhQFlzv-8hjHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXht1Sitrk%253D&md5=3010929d2979cf04dc6003e66575f1b0</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2019.11.031&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2019.11.031%26sid%3Dliteratum%253Aachs%26aulast%3DBurslem%26aufirst%3DG.%2BM.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DProteolysis-targeting%2520chimeras%2520as%2520therapeutics%2520and%2520tools%2520for%2520biological%2520discovery%26jtitle%3DCell%26date%3D2020%26volume%3D181%26spage%3D102%26epage%3D114%26doi%3D10.1016%2Fj.cell.2019.11.031" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lai, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Induced protein degradation: an emerging drug discovery paradigm</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">101</span>– <span class="NLM_lpage">114</span>, <span class="refDoi"> DOI: 10.1038/nrd.2016.211</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1038%2Fnrd.2016.211" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=27885283" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFCit7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=101-114&author=A.+C.+Laiauthor=C.+M.+Crews&title=Induced+protein+degradation%3A+an+emerging+drug+discovery+paradigm&doi=10.1038%2Fnrd.2016.211"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Induced protein degradation: an emerging drug discovery paradigm</span></div><div class="casAuthors">Lai, Ashton C.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">101-114</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Small-mol. drug discovery has traditionally focused on occupancy of a binding site that directly affects protein function, and this approach typically precludes targeting proteins that lack such amenable sites.  Furthermore, high systemic drug exposures may be needed to maintain sufficient target inhibition in vivo, increasing the risk of undesirable off-target effects.  Induced protein degrdn. is an alternative approach that is event-driven: upon drug binding, the target protein is tagged for elimination.  Emerging technologies based on proteolysis-targeting chimaeras (PROTACs) that exploit cellular quality control machinery to selectively degrade target proteins are attracting considerable attention in the pharmaceutical industry owing to the advantages they could offer over traditional small-mol. strategies.  These advantages include the potential to reduce systemic drug exposure, the ability to counteract increased target protein expression that often accompanies inhibition of protein function and the potential ability to target proteins that are not currently therapeutically tractable, such as transcription factors, scaffolding and regulatory proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoj-O7zXGOpXLVg90H21EOLACvtfcHk0lhQFlzv-8hjHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFCit7rP&md5=a01ff43e9b696fe4dc03f0815158a98f</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.211&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.211%26sid%3Dliteratum%253Aachs%26aulast%3DLai%26aufirst%3DA.%2BC.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DInduced%2520protein%2520degradation%253A%2520an%2520emerging%2520drug%2520discovery%2520paradigm%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2017%26volume%3D16%26spage%3D101%26epage%3D114%26doi%3D10.1038%2Fnrd.2016.211" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calabrese, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bullock, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Targeted protein degradation: expanding the toolbox</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">949</span>– <span class="NLM_lpage">963</span>, <span class="refDoi"> DOI: 10.1038/s41573-019-0047-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1038%2Fs41573-019-0047-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=31666732" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitVKrtLrM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=949-963&author=M.+Schapiraauthor=M.+F.+Calabreseauthor=A.+N.+Bullockauthor=C.+M.+Crews&title=Targeted+protein+degradation%3A+expanding+the+toolbox&doi=10.1038%2Fs41573-019-0047-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted protein degradation: expanding the toolbox</span></div><div class="casAuthors">Schapira, Matthieu; Calabrese, Matthew F.; Bullock, Alex N.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">949-963</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review. Proteolysis-targeting chimeras (PROTACs) and related mols. that induce targeted protein degrdn. by the ubiquitin-proteasome system represent a new therapeutic modality and are the focus of great interest, owing to potential advantages over traditional occupancy-based inhibitors with respect to dosing, side effects, drug resistance and modulating 'undruggable' targets.  However, the technol. is still maturing, and the design elements for successful PROTAC-based drugs are currently being elucidated.  Importantly, fewer than 10 of the more than 600 E3 ubiquitin ligases have so far been exploited for targeted protein degrdn., and expansion of knowledge in this area is a key opportunity.  Here, we briefly discuss lessons learned about targeted protein degrdn. in chem. biol. and drug discovery and systematically review the expression profile, domain architecture and chem. tractability of human E3 ligases that could expand the toolbox for PROTAC discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwUvM27MU3WrVg90H21EOLACvtfcHk0lhMuUq729hn8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitVKrtLrM&md5=9bfd48c6c47fc1cce15129495fea7079</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1038%2Fs41573-019-0047-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41573-019-0047-y%26sid%3Dliteratum%253Aachs%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DCalabrese%26aufirst%3DM.%2BF.%26aulast%3DBullock%26aufirst%3DA.%2BN.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DTargeted%2520protein%2520degradation%253A%2520expanding%2520the%2520toolbox%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2019%26volume%3D18%26spage%3D949%26epage%3D963%26doi%3D10.1038%2Fs41573-019-0047-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Setten, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossi, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, S.-p.</span></span> <span> </span><span class="NLM_article-title">The current state and future directions of RNAi-based therapeutics</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">421</span>– <span class="NLM_lpage">446</span>, <span class="refDoi"> DOI: 10.1038/s41573-019-0017-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1038%2Fs41573-019-0017-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=30846871" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC1MXms1GjtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=421-446&author=R.+L.+Settenauthor=J.+J.+Rossiauthor=S.-p.+Han&title=The+current+state+and+future+directions+of+RNAi-based+therapeutics&doi=10.1038%2Fs41573-019-0017-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">The current state and future directions of RNAi-based therapeutics</span></div><div class="casAuthors">Setten, Ryan L.; Rossi, John J.; Han, Si-ping</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">421-446</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review.  The RNA interference (RNAi) pathway regulates mRNA stability and translation in nearly all human cells.  Small double-stranded RNA mols. can efficiently trigger RNAi silencing of specific genes, but their therapeutic use has faced numerous challenges involving safety and potency.  However, August 2018 marked a new era for the field, with the US Food and Drug Administration approving ONPATTRO, the first RNAi-based drug.  In this Review, we discuss key advances in the design and development of RNAi drugs leading up to this landmark achievement, the state of the current clin. pipeline and prospects for future advances, including novel RNAi pathway agents utilizing mechanisms beyond post-translational RNAi silencing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr2SSDt9nwcmLVg90H21EOLACvtfcHk0lhMuUq729hn8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXms1GjtL8%253D&md5=679994c46a9e0e4868a97001b742ce90</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1038%2Fs41573-019-0017-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41573-019-0017-4%26sid%3Dliteratum%253Aachs%26aulast%3DSetten%26aufirst%3DR.%2BL.%26aulast%3DRossi%26aufirst%3DJ.%2BJ.%26aulast%3DHan%26aufirst%3DS.-p.%26atitle%3DThe%2520current%2520state%2520and%2520future%2520directions%2520of%2520RNAi-based%2520therapeutics%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2019%26volume%3D18%26spage%3D421%26epage%3D446%26doi%3D10.1038%2Fs41573-019-0017-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brunetti, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gundry, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sorcini, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guzman, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Y.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramabadran, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gionfriddo, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mezzasoma, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milano, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nabet, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kornblau, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sportoletti, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martelli, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Falini, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goodell, M. A.</span></span> <span> </span><span class="NLM_article-title">Mutant NPM1 maintains the leukemic state through HOX expression</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">499</span>– <span class="NLM_lpage">512</span>, <span class="refDoi"> DOI: 10.1016/j.ccell.2018.08.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1016%2Fj.ccell.2018.08.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=30205049" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhs12ntLbM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2018&pages=499-512&author=L.+Brunettiauthor=M.+C.+Gundryauthor=D.+Sorciniauthor=A.+G.+Guzmanauthor=Y.-H.+Huangauthor=R.+Ramabadranauthor=I.+Gionfriddoauthor=F.+Mezzasomaauthor=F.+Milanoauthor=B.+Nabetauthor=D.+L.+Buckleyauthor=S.+M.+Kornblauauthor=C.+Y.+Linauthor=P.+Sportolettiauthor=M.+P.+Martelliauthor=B.+Faliniauthor=M.+A.+Goodell&title=Mutant+NPM1+maintains+the+leukemic+state+through+HOX+expression&doi=10.1016%2Fj.ccell.2018.08.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Mutant NPM1 Maintains the Leukemic State through HOX Expression</span></div><div class="casAuthors">Brunetti, Lorenzo; Gundry, Michael C.; Sorcini, Daniele; Guzman, Anna G.; Huang, Yung-Hsin; Ramabadran, Raghav; Gionfriddo, Ilaria; Mezzasoma, Federica; Milano, Francesca; Nabet, Behnam; Buckley, Dennis L.; Kornblau, Steven M.; Lin, Charles Y.; Sportoletti, Paolo; Martelli, Maria Paola; Falini, Brunangelo; Goodell, Margaret A.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">499-512.e9</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">NPM1 is the most frequently mutated gene in cytogenetically normal acute myeloid leukemia (AML).  In AML cells, NPM1 mutations result in abnormal cytoplasmic localization of the mutant protein (NPM1c); however, it is unknown whether NPM1c is required to maintain the leukemic state.  Here, we show that loss of NPM1c from the cytoplasm, either through nuclear relocalization or targeted degrdn., results in immediate downregulation of homeobox (HOX) genes followed by differentiation.  Finally, we show that XPO1 inhibition relocalizes NPM1c to the nucleus, promotes differentiation of AML cells, and prolongs survival of Npm1-mutated leukemic mice.  We describe an exquisite dependency of NPM1-mutant AML cells on NPM1c, providing the rationale for the use of nuclear export inhibitors in AML with mutated NPM1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrwmZdVS4yl8LVg90H21EOLACvtfcHk0ljNQs-k1MUGDA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhs12ntLbM&md5=909916554bb4861ed8b2f9f39d108cf4</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2018.08.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2018.08.005%26sid%3Dliteratum%253Aachs%26aulast%3DBrunetti%26aufirst%3DL.%26aulast%3DGundry%26aufirst%3DM.%2BC.%26aulast%3DSorcini%26aufirst%3DD.%26aulast%3DGuzman%26aufirst%3DA.%2BG.%26aulast%3DHuang%26aufirst%3DY.-H.%26aulast%3DRamabadran%26aufirst%3DR.%26aulast%3DGionfriddo%26aufirst%3DI.%26aulast%3DMezzasoma%26aufirst%3DF.%26aulast%3DMilano%26aufirst%3DF.%26aulast%3DNabet%26aufirst%3DB.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DKornblau%26aufirst%3DS.%2BM.%26aulast%3DLin%26aufirst%3DC.%2BY.%26aulast%3DSportoletti%26aufirst%3DP.%26aulast%3DMartelli%26aufirst%3DM.%2BP.%26aulast%3DFalini%26aufirst%3DB.%26aulast%3DGoodell%26aufirst%3DM.%2BA.%26atitle%3DMutant%2520NPM1%2520maintains%2520the%2520leukemic%2520state%2520through%2520HOX%2520expression%26jtitle%3DCancer%2520Cell%26date%3D2018%26volume%3D34%26spage%3D499%26epage%3D512%26doi%3D10.1016%2Fj.ccell.2018.08.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Clift, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mcewan, W. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Labzin, L. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Konieczny, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mogessie, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">James, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuh, M. J. C.</span></span> <span> </span><span class="NLM_article-title">A Method for the acute and rapid degradation of endogenous proteins</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>171</i></span>,  <span class="NLM_fpage">1692</span>– <span class="NLM_lpage">1706</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2017.10.033</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1016%2Fj.cell.2017.10.033" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=29153837" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvValur%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=171&publication_year=2017&pages=1692-1706&author=D.+Cliftauthor=W.+A.+Mcewanauthor=L.+I.+Labzinauthor=V.+Koniecznyauthor=B.+Mogessieauthor=L.+C.+Jamesauthor=M.+J.+C.+Schuh&title=A+Method+for+the+acute+and+rapid+degradation+of+endogenous+proteins&doi=10.1016%2Fj.cell.2017.10.033"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">A Method for the Acute and Rapid Degradation of Endogenous Proteins</span></div><div class="casAuthors">Clift, Dean; McEwan, William A.; Labzin, Larisa I.; Konieczny, Vera; Mogessie, Binyam; James, Leo C.; Schuh, Melina</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">171</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1692-1706.e18</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Methods for the targeted disruption of protein function have revolutionized science and greatly expedited the systematic characterization of genes.  Two main approaches are currently used to disrupt protein function: DNA knockout and RNA interference, which act at the genome and mRNA level, resp.  A method that directly alters endogenous protein levels is currently not available.  Here, the authors present Trim-Away, a technique to degrade endogenous proteins acutely in mammalian cells without prior modification of the genome or mRNA.  Trim-Away harnesses the cellular protein degrdn. machinery to remove unmodified native proteins within minutes of application.  This rapidity minimizes the risk that phenotypes are compensated and that secondary, non-specific defects accumulate over time.  Because Trim-Away uses antibodies, it can be applied to a wide range of target proteins using off-the-shelf reagents.  Trim-Away allows the study of protein function in diverse cell types, including non-dividing primary cells where genome- and RNA-targeting methods are limited.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZqwRirT2Kn7Vg90H21EOLACvtfcHk0ljNQs-k1MUGDA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvValur%252FE&md5=255d6ce0a6a66720f57e4c3d3071a3b4</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2017.10.033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2017.10.033%26sid%3Dliteratum%253Aachs%26aulast%3DClift%26aufirst%3DD.%26aulast%3DMcewan%26aufirst%3DW.%2BA.%26aulast%3DLabzin%26aufirst%3DL.%2BI.%26aulast%3DKonieczny%26aufirst%3DV.%26aulast%3DMogessie%26aufirst%3DB.%26aulast%3DJames%26aufirst%3DL.%2BC.%26aulast%3DSchuh%26aufirst%3DM.%2BJ.%2BC.%26atitle%3DA%2520Method%2520for%2520the%2520acute%2520and%2520rapid%2520degradation%2520of%2520endogenous%2520proteins%26jtitle%3DCell%26date%3D2017%26volume%3D171%26spage%3D1692%26epage%3D1706%26doi%3D10.1016%2Fj.cell.2017.10.033" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brand, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G. E.</span></span> <span> </span><span class="NLM_article-title">Locus-specific knock-in of a degradable tag for target validation studies</span>. <i>Methods Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>1953</i></span>,  <span class="NLM_fpage">105</span>– <span class="NLM_lpage">119</span>, <span class="refDoi"> DOI: 10.1007/978-1-4939-9145-7_7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1007%2F978-1-4939-9145-7_7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=30912018" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtFOju7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1953&publication_year=2019&pages=105-119&author=M.+Brandauthor=G.+E.+Winter&title=Locus-specific+knock-in+of+a+degradable+tag+for+target+validation+studies&doi=10.1007%2F978-1-4939-9145-7_7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Locus-specific knock-in of a degradable tag for target validation studies.</span></div><div class="casAuthors">Brand, Matthias; Winter, Georg E.</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Molecular Biology (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">1953</span>
        (<span class="NLM_cas:issue">Target Identification and Validation in Drug Discovery</span>),
    <span class="NLM_cas:pages">105-119</span>CODEN:
                <span class="NLM_cas:coden">MMBIED</span>;
        ISSN:<span class="NLM_cas:issn">1940-6029</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Small mol.-induced targeted protein degrdn. is a powerful approach for drug target validation given its selectivity, high kinetic resoln., dose dependency, and reversibility.  Out of the several methods that have been reported so far, the 12 kDa degrdn. tag (dTAG) system has the advantage of hijacking a degrdn. machinery that is ubiquitously expressed in all human tissues.  Therefore it is independent of expressing addnl., exogenous factors and addnl. permits target validation in vivo.  Here, we describe the protocol for generation and validation of clones harboring knock-in of a selectable dTAG cassette at the endogenous locus of proteins of interest using the near-haploid cell line KBM7.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJ07a4wMM3mLVg90H21EOLACvtfcHk0liYXWYFYkL5mQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtFOju7k%253D&md5=bdc2cdcd6fba21a1eb4263a5c3d0c137</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1007%2F978-1-4939-9145-7_7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-1-4939-9145-7_7%26sid%3Dliteratum%253Aachs%26aulast%3DBrand%26aufirst%3DM.%26aulast%3DWinter%26aufirst%3DG.%2BE.%26atitle%3DLocus-specific%2520knock-in%2520of%2520a%2520degradable%2520tag%2520for%2520target%2520validation%2520studies%26jtitle%3DMethods%2520Mol.%2520Biol.%26date%3D2019%26volume%3D1953%26spage%3D105%26epage%3D119%26doi%3D10.1007%2F978-1-4939-9145-7_7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nabet, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulk, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dastjerdi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leggett, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erb, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawlor, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Souza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, T. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vittori, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perry, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, K.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">The dTAG system for immediate and target-specific protein degradation</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">431</span>– <span class="NLM_lpage">441</span>, <span class="refDoi"> DOI: 10.1038/s41589-018-0021-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1038%2Fs41589-018-0021-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=29581585" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC1cXosFGmsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2018&pages=431-441&author=B.+Nabetauthor=J.+M.+Robertsauthor=D.+L.+Buckleyauthor=J.+Paulkauthor=S.+Dastjerdiauthor=A.+Yangauthor=A.+L.+Leggettauthor=M.+A.+Erbauthor=M.+A.+Lawlorauthor=A.+Souzaauthor=T.+G.+Scottauthor=S.+Vittoriauthor=J.+A.+Perryauthor=J.+Qiauthor=G.+E.+Winterauthor=K.-K.+Wongauthor=N.+S.+Grayauthor=J.+E.+Bradner&title=The+dTAG+system+for+immediate+and+target-specific+protein+degradation&doi=10.1038%2Fs41589-018-0021-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">The dTAG system for immediate and target-specific protein degradation</span></div><div class="casAuthors">Nabet, Behnam; Roberts, Justin M.; Buckley, Dennis L.; Paulk, Joshiawa; Dastjerdi, Shiva; Yang, Annan; Leggett, Alan L.; Erb, Michael A.; Lawlor, Matthew A.; Souza, Amanda; Scott, Thomas G.; Vittori, Sarah; Perry, Jennifer A.; Qi, Jun; Winter, Georg E.; Wong, Kwok-Kin; Gray, Nathanael S.; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">431-441</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Dissection of complex biol. systems requires target-specific control of the function or abundance of proteins.  Genetic perturbations are limited by off-target effects, multicomponent complexity, and irreversibility.  Most limiting is the requisite delay between modulation to exptl. measurement.  To enable the immediate and selective control of single protein abundance, we created a chem. biol. system that leverages the potency of cell-permeable heterobifunctional degraders.  The dTAG system pairs a novel degrader of FKBP12F36V with expression of FKBP12F36V in-frame with a protein of interest.  By transgene expression or CRISPR-mediated locus-specific knock-in, we exemplify a generalizable strategy to study the immediate consequence of protein loss.  Using dTAG, we observe an unexpected superior antiproliferative effect of pan-BET bromodomain degrdn. over selective BRD4 degrdn., characterize immediate effects of KRASG12V loss on proteomic signaling, and demonstrate rapid degrdn. in vivo.  This technol. platform will confer kinetic resoln. to biol. investigation and provide target validation in the context of drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqwQvCDe-2I7Vg90H21EOLACvtfcHk0liYXWYFYkL5mQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXosFGmsbs%253D&md5=d7d9c111c68c84bc3aa9c2aad3f591a6</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1038%2Fs41589-018-0021-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41589-018-0021-8%26sid%3Dliteratum%253Aachs%26aulast%3DNabet%26aufirst%3DB.%26aulast%3DRoberts%26aufirst%3DJ.%2BM.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DPaulk%26aufirst%3DJ.%26aulast%3DDastjerdi%26aufirst%3DS.%26aulast%3DYang%26aufirst%3DA.%26aulast%3DLeggett%26aufirst%3DA.%2BL.%26aulast%3DErb%26aufirst%3DM.%2BA.%26aulast%3DLawlor%26aufirst%3DM.%2BA.%26aulast%3DSouza%26aufirst%3DA.%26aulast%3DScott%26aufirst%3DT.%2BG.%26aulast%3DVittori%26aufirst%3DS.%26aulast%3DPerry%26aufirst%3DJ.%2BA.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DWinter%26aufirst%3DG.%2BE.%26aulast%3DWong%26aufirst%3DK.-K.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DThe%2520dTAG%2520system%2520for%2520immediate%2520and%2520target-specific%2520protein%2520degradation%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2018%26volume%3D14%26spage%3D431%26epage%3D441%26doi%3D10.1038%2Fs41589-018-0021-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ohoka, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okuhira, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ito, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagai, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shibata, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hattori, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ujikawa, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shimokawa, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sano, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koyama, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujita, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teratani, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsumoto, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imaeda, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nara, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naito, M.</span></span> <span> </span><span class="NLM_article-title">In vivo knockdown of pathogenic proteins via specific and nongenetic inhibitor of apoptosis protein (IAP)-dependent protein erasers (SNIPERs)</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>292</i></span>,  <span class="NLM_fpage">4556</span>– <span class="NLM_lpage">4570</span>, <span class="refDoi"> DOI: 10.1074/jbc.M116.768853</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1074%2Fjbc.M116.768853" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=28154167" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC2sXksVClur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=292&publication_year=2017&pages=4556-4570&author=N.+Ohokaauthor=K.+Okuhiraauthor=M.+Itoauthor=K.+Nagaiauthor=N.+Shibataauthor=T.+Hattoriauthor=O.+Ujikawaauthor=K.+Shimokawaauthor=O.+Sanoauthor=R.+Koyamaauthor=H.+Fujitaauthor=M.+Terataniauthor=H.+Matsumotoauthor=Y.+Imaedaauthor=H.+Naraauthor=N.+Choauthor=M.+Naito&title=In+vivo+knockdown+of+pathogenic+proteins+via+specific+and+nongenetic+inhibitor+of+apoptosis+protein+%28IAP%29-dependent+protein+erasers+%28SNIPERs%29&doi=10.1074%2Fjbc.M116.768853"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">In Vivo Knockdown of Pathogenic Proteins via Specific and Nongenetic Inhibitor of Apoptosis Protein (IAP)-dependent Protein Erasers (SNIPERs)</span></div><div class="casAuthors">Ohoka, Nobumichi; Okuhira, Keiichiro; Ito, Masahiro; Nagai, Katsunori; Shibata, Norihito; Hattori, Takayuki; Ujikawa, Osamu; Shimokawa, Kenichiro; Sano, Osamu; Koyama, Ryokichi; Fujita, Hisashi; Teratani, Mika; Matsumoto, Hirokazu; Imaeda, Yasuhiro; Nara, Hiroshi; Cho, Nobuo; Naito, Mikihiko</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">292</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4556-4570</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Many diseases, esp. cancers, result from aberrant or overexpression of pathogenic proteins.  Specific inhibitors against these proteins have shown remarkable therapeutic effects, but these are limited mainly to enzymes.  An alternative approach that may have utility in drug development relies on selective degrdn. of pathogenic proteins via small chimeric mols. linking an E3 ubiquitin ligase to the targeted protein for proteasomal degrdn.  To this end, the authors recently developed a protein knockdown system based on hybrid small mol. SNIPERs (Specific and Nongenetic IAP-dependent Protein Erasers) that recruit inhibitor of the apoptosis protein (IAP) ubiquitin ligases to specifically degrade targeted proteins.  Here, the authors extend the authors' previous study to show a proof of concept of the SNIPER technol. in vivo.  By incorporating a high affinity IAP ligand, the authors developed a novel SNIPER against estrogen receptor α (ERα), SNIPER(ER)-87, that has a potent protein knockdown activity.  The SNIPER(ER) reduced ERα levels in tumor xenografts and suppressed the growth of ERα-pos. breast tumors in mice.  Mechanistically, it preferentially recruits X-linked IAP (XIAP) rather than cellular IAP1, to degrade ERα via the ubiquitin-proteasome pathway.  With this IAP ligand, potent SNIPERs against other pathogenic proteins, BCR-ABL, bromodomain-contg. protein 4 (BRD4), and phosphodiesterase-4 (PDE4) could also be developed.  These results indicate that forced ubiquitylation by SNIPERs is a useful method to achieve efficient protein knockdown with potential therapeutic activities and could also be applied to study the role of ubiquitylation in many cellular processes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqEzH5uNeAFabVg90H21EOLACvtfcHk0lhmws2cgSDZKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXksVClur8%253D&md5=b09264d53c47c54343ed92d5a6d0eb79</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M116.768853&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M116.768853%26sid%3Dliteratum%253Aachs%26aulast%3DOhoka%26aufirst%3DN.%26aulast%3DOkuhira%26aufirst%3DK.%26aulast%3DIto%26aufirst%3DM.%26aulast%3DNagai%26aufirst%3DK.%26aulast%3DShibata%26aufirst%3DN.%26aulast%3DHattori%26aufirst%3DT.%26aulast%3DUjikawa%26aufirst%3DO.%26aulast%3DShimokawa%26aufirst%3DK.%26aulast%3DSano%26aufirst%3DO.%26aulast%3DKoyama%26aufirst%3DR.%26aulast%3DFujita%26aufirst%3DH.%26aulast%3DTeratani%26aufirst%3DM.%26aulast%3DMatsumoto%26aufirst%3DH.%26aulast%3DImaeda%26aufirst%3DY.%26aulast%3DNara%26aufirst%3DH.%26aulast%3DCho%26aufirst%3DN.%26aulast%3DNaito%26aufirst%3DM.%26atitle%3DIn%2520vivo%2520knockdown%2520of%2520pathogenic%2520proteins%2520via%2520specific%2520and%2520nongenetic%2520inhibitor%2520of%2520apoptosis%2520protein%2520%2528IAP%2529-dependent%2520protein%2520erasers%2520%2528SNIPERs%2529%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2017%26volume%3D292%26spage%3D4556%26epage%3D4570%26doi%3D10.1074%2Fjbc.M116.768853" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Perera, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zoncu, R.</span></span> <span> </span><span class="NLM_article-title">The lysosome as a regulatory hub</span>. <i>Annu. Rev. Cell Dev. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">223</span>– <span class="NLM_lpage">253</span>, <span class="refDoi"> DOI: 10.1146/annurev-cellbio-111315-125125</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1146%2Fannurev-cellbio-111315-125125" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=27501449" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtlaitrnO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2016&pages=223-253&author=R.+M.+Pereraauthor=R.+Zoncu&title=The+lysosome+as+a+regulatory+hub&doi=10.1146%2Fannurev-cellbio-111315-125125"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">The Lysosome as a Regulatory Hub</span></div><div class="casAuthors">Perera, Rushika M.; Zoncu, Roberto</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Cell and Developmental Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">223-253</span>CODEN:
                <span class="NLM_cas:coden">ARDBF8</span>;
        ISSN:<span class="NLM_cas:issn">1081-0706</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews</span>)
        </div><div class="casAbstract">The lysosome has long been viewed as the recycling center of the cell.  However, recent discoveries have challenged this simple view and have established a central role of the lysosome in nutrient-dependent signal transduction.  The degradative role of the lysosome and its newly discovered signaling functions are not in conflict but rather cooperate extensively to mediate fundamental cellular activities such as nutrient sensing, metabolic adaptation, and quality control of proteins and organelles.  Moreover, lysosome-based signaling and degrdn. are subject to reciprocal regulation.  Transcriptional programs of increasing complexity control the biogenesis, compn., and abundance of lysosomes and fine-tune their activity to match the evolving needs of the cell.  Alterations in these essential activities are, not surprisingly, central to the pathophysiol. of an ever-expanding spectrum of conditions, including storage disorders, neurodegenerative diseases, and cancer.  Thus, unraveling the functions of this fascinating organelle will contribute to our understanding of the fundamental logic of metabolic organization and will point to novel therapeutic avenues in several human diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpH74ZfQKHDrrVg90H21EOLACvtfcHk0lhmws2cgSDZKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtlaitrnO&md5=73d1ab71b801facb20ae036cdfae74a3</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1146%2Fannurev-cellbio-111315-125125&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-cellbio-111315-125125%26sid%3Dliteratum%253Aachs%26aulast%3DPerera%26aufirst%3DR.%2BM.%26aulast%3DZoncu%26aufirst%3DR.%26atitle%3DThe%2520lysosome%2520as%2520a%2520regulatory%2520hub%26jtitle%3DAnnu.%2520Rev.%2520Cell%2520Dev.%2520Biol.%26date%3D2016%26volume%3D32%26spage%3D223%26epage%3D253%26doi%3D10.1146%2Fannurev-cellbio-111315-125125" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Settembre, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fraldi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Medina, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ballabio, A.</span></span> <span> </span><span class="NLM_article-title">Signals from the lysosome: a control centre for cellular clearance and energy metabolism</span>. <i>Nat. Rev. Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">283</span>– <span class="NLM_lpage">296</span>, <span class="refDoi"> DOI: 10.1038/nrm3565</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1038%2Fnrm3565" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=23609508" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC3sXmt1Oksrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2013&pages=283-296&author=C.+Settembreauthor=A.+Fraldiauthor=D.+L.+Medinaauthor=A.+Ballabio&title=Signals+from+the+lysosome%3A+a+control+centre+for+cellular+clearance+and+energy+metabolism&doi=10.1038%2Fnrm3565"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Signals from the lysosome: a control centre for cellular clearance and energy metabolism</span></div><div class="casAuthors">Settembre, Carmine; Fraldi, Alessandro; Medina, Diego L.; Ballabio, Andrea</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">283-296</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  For a long time, lysosomes were considered merely to be cellular 'incinerators' involved in the degrdn. and recycling of cellular waste.  However, now there is compelling evidence indicating that lysosomes have a much broader function and that they are involved in fundamental processes such as secretion, plasma membrane repair, signaling and energy metab.  Furthermore, the essential role of lysosomes in autophagic pathways puts these organelles at the crossroads of several cellular processes, with significant implications for health and disease.  The identification of a master regulator, transcription factor EB (TFEB), that regulates lysosomal biogenesis and autophagy has revealed how the lysosome adapts to environmental cues, such as starvation, and targeting TFEB may provide a novel therapeutic strategy for modulating lysosomal function in human disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZpiDHfiHRzLVg90H21EOLACvtfcHk0lhmws2cgSDZKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXmt1Oksrc%253D&md5=6ef324f69939d95c37ef334e3d4c2c78</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1038%2Fnrm3565&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm3565%26sid%3Dliteratum%253Aachs%26aulast%3DSettembre%26aufirst%3DC.%26aulast%3DFraldi%26aufirst%3DA.%26aulast%3DMedina%26aufirst%3DD.%2BL.%26aulast%3DBallabio%26aufirst%3DA.%26atitle%3DSignals%2520from%2520the%2520lysosome%253A%2520a%2520control%2520centre%2520for%2520cellular%2520clearance%2520and%2520energy%2520metabolism%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2013%26volume%3D14%26spage%3D283%26epage%3D296%26doi%3D10.1038%2Fnrm3565" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Davidson, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vander Heiden, M. G.</span></span> <span> </span><span class="NLM_article-title">Critical functions of the lysosome in cancer biology</span>. <i>Annu. Rev. Pharmacol. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">481</span>– <span class="NLM_lpage">507</span>, <span class="refDoi"> DOI: 10.1146/annurev-pharmtox-010715-103101</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1146%2Fannurev-pharmtox-010715-103101" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=27732799" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhs1ykt7rO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2017&pages=481-507&author=S.+M.+Davidsonauthor=M.+G.+Vander+Heiden&title=Critical+functions+of+the+lysosome+in+cancer+biology&doi=10.1146%2Fannurev-pharmtox-010715-103101"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Critical Functions of the Lysosome in Cancer Biology</span></div><div class="casAuthors">Davidson, Shawn M.; Vander Heiden, Matthew G.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Pharmacology and Toxicology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">481-507</span>CODEN:
                <span class="NLM_cas:coden">ARPTDI</span>;
        ISSN:<span class="NLM_cas:issn">0362-1642</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews</span>)
        </div><div class="casAbstract">Lysosomes (or lytic bodies) were so named because they contain high levels of hydrolytic enzymes.  Lysosome function and dysfunction have been found to play important roles in human disease, including cancer; however, the ways in which lysosomes contribute to tumorigenesis and cancer progression are still being uncovered.  Beyond serving as a cellular recycling center, recent evidence suggests that the lysosome is involved in energy homeostasis, generating building blocks for cell growth, mitogenic signaling, priming tissues for angiogenesis and metastasis formation, and activating transcriptional programs.  This review examines emerging knowledge of how lysosomal processes contribute to the hallmarks of cancer and highlights vulnerabilities that might be exploited for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpekDKNs3_ogrVg90H21EOLACvtfcHk0ljw6VEIoekWCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhs1ykt7rO&md5=a104126535284709504690b4d0463b1b</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1146%2Fannurev-pharmtox-010715-103101&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-pharmtox-010715-103101%26sid%3Dliteratum%253Aachs%26aulast%3DDavidson%26aufirst%3DS.%2BM.%26aulast%3DVander%2BHeiden%26aufirst%3DM.%2BG.%26atitle%3DCritical%2520functions%2520of%2520the%2520lysosome%2520in%2520cancer%2520biology%26jtitle%3DAnnu.%2520Rev.%2520Pharmacol.%2520Toxicol.%26date%3D2017%26volume%3D57%26spage%3D481%26epage%3D507%26doi%3D10.1146%2Fannurev-pharmtox-010715-103101" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bonam, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, S.</span></span> <span> </span><span class="NLM_article-title">Autophagy: a new concept in autoimmunity regulation and a novel therapeutic option</span>. <i>J. Autoimmun.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>94</i></span>,  <span class="NLM_fpage">16</span>– <span class="NLM_lpage">32</span>, <span class="refDoi"> DOI: 10.1016/j.jaut.2018.08.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1016%2Fj.jaut.2018.08.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=30219390" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslSkt7rJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=94&publication_year=2018&pages=16-32&author=S.+R.+Bonamauthor=F.+Wangauthor=S.+Muller&title=Autophagy%3A+a+new+concept+in+autoimmunity+regulation+and+a+novel+therapeutic+option&doi=10.1016%2Fj.jaut.2018.08.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Autophagy: A new concept in autoimmunity regulation and a novel therapeutic option</span></div><div class="casAuthors">Bonam, Srinivasa Reddy; Wang, Fengjuan; Muller, Sylviane</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Autoimmunity</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">94</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">16-32</span>CODEN:
                <span class="NLM_cas:coden">JOAUEP</span>;
        ISSN:<span class="NLM_cas:issn">0896-8411</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Nowadays, pharmacol. treatments of autoinflammatory diseases are largely palliative rather than curative.  Most of them result in non-specific immunosuppression, which can be assocd. with broad disruption of natural and induced immunity with significant and sometimes serious unwanted injuries.  Peptide therapeutics, class of potent drugs, the phosphopeptide P140 is very promising for treating patients with lupus, and likely also patients with other chronic inflammatory diseases.  We discovered that P140 targets autophagy, a finely orchestrated catabolic process, involved in the regulation of inflammation and in the biol. of immune cells.  In vitro, P140 acts directly on a particular form of autophagy called chaperone-mediated autophagy, which seems to be hyperactivated in certain subsets of lymphocytes in lupus and in other autoinflammatory settings.  In lupus, the "correcting" effect of P140 on autophagy results in a weaker signaling of autoreactive T cells, leading to a significant improvement of pathophysiol. status of treated mice.  These findings also demonstrated ex vivo in human cells, open novel avenues of therapeutic intervention in pathol. conditions, in which specific and not general targeting is highly pursued in the context of the new action plans for personalized medicines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGroze5ifIgxJbVg90H21EOLACvtfcHk0ljw6VEIoekWCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslSkt7rJ&md5=c96a3a37777c98cc81b230b405b96087</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2Fj.jaut.2018.08.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jaut.2018.08.009%26sid%3Dliteratum%253Aachs%26aulast%3DBonam%26aufirst%3DS.%2BR.%26aulast%3DWang%26aufirst%3DF.%26aulast%3DMuller%26aufirst%3DS.%26atitle%3DAutophagy%253A%2520a%2520new%2520concept%2520in%2520autoimmunity%2520regulation%2520and%2520a%2520novel%2520therapeutic%2520option%26jtitle%3DJ.%2520Autoimmun.%26date%3D2018%26volume%3D94%26spage%3D16%26epage%3D32%26doi%3D10.1016%2Fj.jaut.2018.08.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boland, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, W. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corti, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mollereau, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henriques, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bezard, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pastores, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rubinsztein, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nixon, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duchen, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mallucci, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kroemer, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levine, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eskelinen, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mochel, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spedding, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Louis, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, O. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Millan, M. J.</span></span> <span> </span><span class="NLM_article-title">Promoting the clearance of neurotoxic proteins in neurodegenerative disorders of ageing</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">660</span>– <span class="NLM_lpage">688</span>, <span class="refDoi"> DOI: 10.1038/nrd.2018.109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1038%2Fnrd.2018.109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=30116051" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsFans73J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2018&pages=660-688&author=B.+Bolandauthor=W.+H.+Yuauthor=O.+Cortiauthor=B.+Mollereauauthor=A.+Henriquesauthor=E.+Bezardauthor=G.+M.+Pastoresauthor=D.+C.+Rubinszteinauthor=R.+A.+Nixonauthor=M.+R.+Duchenauthor=G.+R.+Mallucciauthor=G.+Kroemerauthor=B.+Levineauthor=E.+L.+Eskelinenauthor=F.+Mochelauthor=M.+Speddingauthor=C.+Louisauthor=O.+R.+Martinauthor=M.+J.+Millan&title=Promoting+the+clearance+of+neurotoxic+proteins+in+neurodegenerative+disorders+of+ageing&doi=10.1038%2Fnrd.2018.109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Promoting the clearance of neurotoxic proteins in neurodegenerative disorders of ageing</span></div><div class="casAuthors">Boland, Barry; Yu, Wai Haung; Corti, Olga; Mollereau, Bertrand; Henriques, Alexandre; Bezard, Erwan; Pastores, Greg M.; Rubinsztein, David C.; Nixon, Ralph A.; Duchen, Michael R.; Mallucci, Giovanna R.; Kroemer, Guido; Levine, Beth; Eskelinen, Eeva-Liisa; Mochel, Fanny; Spedding, Michael; Louis, Caroline; Martin, Olivier R.; Millan, Mark J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">660-688</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review.  Neurodegenerative disorders of ageing (NDAs) such as Alzheimer disease, Parkinson disease, frontotemporal dementia, Huntington disease and amyotrophic lateral sclerosis represent a major socio-economic challenge in view of their high prevalence yet poor treatment.  They are often called 'proteinopathies' owing to the presence of misfolded and aggregated proteins that lose their physiol. roles and acquire neurotoxic properties.  One reason underlying the accumulation and spread of oligomeric forms of neurotoxic proteins is insufficient clearance by the autophagic-lysosomal network.  Several other clearance pathways are also compromised in NDAs: chaperone-mediated autophagy, the ubiquitin-proteasome system, extracellular clearance by proteases and extrusion into the circulation via the blood-brain barrier and glymphatic system.  This article focuses on emerging mechanisms for promoting the clearance of neurotoxic proteins, a strategy that may curtail the onset and slow the progression of NDAs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGph_pFdlqvN4LVg90H21EOLACvtfcHk0ljw6VEIoekWCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsFans73J&md5=3ae9f39bde30d40bb5349419d6d1edd2</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2018.109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2018.109%26sid%3Dliteratum%253Aachs%26aulast%3DBoland%26aufirst%3DB.%26aulast%3DYu%26aufirst%3DW.%2BH.%26aulast%3DCorti%26aufirst%3DO.%26aulast%3DMollereau%26aufirst%3DB.%26aulast%3DHenriques%26aufirst%3DA.%26aulast%3DBezard%26aufirst%3DE.%26aulast%3DPastores%26aufirst%3DG.%2BM.%26aulast%3DRubinsztein%26aufirst%3DD.%2BC.%26aulast%3DNixon%26aufirst%3DR.%2BA.%26aulast%3DDuchen%26aufirst%3DM.%2BR.%26aulast%3DMallucci%26aufirst%3DG.%2BR.%26aulast%3DKroemer%26aufirst%3DG.%26aulast%3DLevine%26aufirst%3DB.%26aulast%3DEskelinen%26aufirst%3DE.%2BL.%26aulast%3DMochel%26aufirst%3DF.%26aulast%3DSpedding%26aufirst%3DM.%26aulast%3DLouis%26aufirst%3DC.%26aulast%3DMartin%26aufirst%3DO.%2BR.%26aulast%3DMillan%26aufirst%3DM.%2BJ.%26atitle%3DPromoting%2520the%2520clearance%2520of%2520neurotoxic%2520proteins%2520in%2520neurodegenerative%2520disorders%2520of%2520ageing%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2018%26volume%3D17%26spage%3D660%26epage%3D688%26doi%3D10.1038%2Fnrd.2018.109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tomoshige, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishikawa, M.</span></span> <span> </span><span class="NLM_article-title">PROTACs and other chemical protein degradation technologies for the treatment of neurodegenerative disorders</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">3346</span>– <span class="NLM_lpage">3354</span>, <span class="refDoi"> DOI: 10.1002/anie.202004746</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1002%2Fanie.202004746" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhs1ersbbM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2021&pages=3346-3354&author=S.+Tomoshigeauthor=M.+Ishikawa&title=PROTACs+and+other+chemical+protein+degradation+technologies+for+the+treatment+of+neurodegenerative+disorders&doi=10.1002%2Fanie.202004746"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">PROTACs and Other Chemical Protein Degradation Technologies for the Treatment of Neurodegenerative Disorders</span></div><div class="casAuthors">Tomoshige, Shusuke; Ishikawa, Minoru</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3346-3354</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  Neurodegenerative disorders (NDs) are a group of diseases that cause neural cell damage, leading to motility and/or cognitive dysfunctions.  One of the causative agents is misfolded protein aggregates, which are considered as undruggable in terms of conventional tools, such as inhibitors and agonists/antagonists.  Indeed, there is currently no FDA-approved drug for the causal treatment of NDs.  However, emerging technologies for chem. protein degrdn. are opening up the possibility of selective elimination of target proteins through physiol. protein degrdn. machineries, which do not depend on the functions of the target proteins.  Here, we review recent efforts towards the treatment of NDs using chem. protein degrdn. technologies, and we briefly discuss the challenges and prospects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptTufLqJjYuLVg90H21EOLACvtfcHk0lhkMyNdqBpLhw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhs1ersbbM&md5=0438ca3689960eac5a722b755c5cbdd5</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1002%2Fanie.202004746&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.202004746%26sid%3Dliteratum%253Aachs%26aulast%3DTomoshige%26aufirst%3DS.%26aulast%3DIshikawa%26aufirst%3DM.%26atitle%3DPROTACs%2520and%2520other%2520chemical%2520protein%2520degradation%2520technologies%2520for%2520the%2520treatment%2520of%2520neurodegenerative%2520disorders%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2021%26volume%3D60%26spage%3D3346%26epage%3D3354%26doi%3D10.1002%2Fanie.202004746" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Clague, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urbé, S.</span></span> <span> </span><span class="NLM_article-title">Ubiquitin: same molecule, different degradation pathways</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>143</i></span>,  <span class="NLM_fpage">682</span>– <span class="NLM_lpage">685</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2010.11.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1016%2Fj.cell.2010.11.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=21111229" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVygsLbP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=143&publication_year=2010&pages=682-685&author=M.+J.+Clagueauthor=S.+Urb%C3%A9&title=Ubiquitin%3A+same+molecule%2C+different+degradation+pathways&doi=10.1016%2Fj.cell.2010.11.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Ubiquitin: Same molecule, different degradation pathways</span></div><div class="casAuthors">Clague, Michael J.; Urbe, Sylvie</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">143</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">682-685</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Ubiquitin is a common denominator in the targeting of substrates to all 3 major protein degrdn. pathways in mammalian cells: the proteasome, the lysosome, and the autophagosome.  The factors that direct a substrate toward a particular route of degrdn. likely include ubiquitin chain length and linkage type, which may favor interaction with particular receptors or confer differential susceptibility to deubiquitinase activities assocd. with each pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGol6eqfdT1hI7Vg90H21EOLACvtfcHk0lhkMyNdqBpLhw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsVygsLbP&md5=72f5984164b176a89c5fffbafe157d1c</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2010.11.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2010.11.012%26sid%3Dliteratum%253Aachs%26aulast%3DClague%26aufirst%3DM.%2BJ.%26aulast%3DUrb%25C3%25A9%26aufirst%3DS.%26atitle%3DUbiquitin%253A%2520same%2520molecule%252C%2520different%2520degradation%2520pathways%26jtitle%3DCell%26date%3D2010%26volume%3D143%26spage%3D682%26epage%3D685%26doi%3D10.1016%2Fj.cell.2010.11.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kirkin, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McEwan, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Novak, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dikic, I.</span></span> <span> </span><span class="NLM_article-title">A role for ubiquitin in selective autophagy</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">259</span>– <span class="NLM_lpage">269</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2009.04.026</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1016%2Fj.molcel.2009.04.026" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=19450525" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BD1MXmtlGlt7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2009&pages=259-269&author=V.+Kirkinauthor=D.+G.+McEwanauthor=I.+Novakauthor=I.+Dikic&title=A+role+for+ubiquitin+in+selective+autophagy&doi=10.1016%2Fj.molcel.2009.04.026"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">A role for ubiquitin in selective autophagy</span></div><div class="casAuthors">Kirkin, Vladimir; McEwan, David G.; Novak, Ivana; Dikic, Ivan</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">259-269</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Ubiquitination is the hallmark of protein degrdn. by the 26S proteasome.  However, the proteasome is limited its capacity to degrade oligomeric and aggregated proteins.  Removal of harmful protein aggregates is mediated by autophagy, a mechanism by which the cell sequesters cytosolic cargo and delivers it for degrdn. by the lysosome.  Identification of autophagy receptors, such as p62/SQSTM1 and NBR1, which simultaneously bind both ubiquitin and autophagy-specific ubiquitin-like modifiers, LC3/GABARAP, has provided a mol. link between ubiquitination and autophagy.  Here, the authors explore the hypothesis that ubiquitin represents a selective degrdn. signal suitable for targeting various types of cargo, ranging from protein aggregates to membrane-bound organelles and microbes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpVkQ7Iie0DJrVg90H21EOLACvtfcHk0lhkMyNdqBpLhw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXmtlGlt7c%253D&md5=5ecbfc53c64ec238a31863456f0da650</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2009.04.026&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2009.04.026%26sid%3Dliteratum%253Aachs%26aulast%3DKirkin%26aufirst%3DV.%26aulast%3DMcEwan%26aufirst%3DD.%2BG.%26aulast%3DNovak%26aufirst%3DI.%26aulast%3DDikic%26aufirst%3DI.%26atitle%3DA%2520role%2520for%2520ubiquitin%2520in%2520selective%2520autophagy%26jtitle%3DMol.%2520Cell%26date%3D2009%26volume%3D34%26spage%3D259%26epage%3D269%26doi%3D10.1016%2Fj.molcel.2009.04.026" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kraft, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peter, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hofmann, K.</span></span> <span> </span><span class="NLM_article-title">Selective autophagy: ubiquitin-mediated recognition and beyond</span>. <i>Nat. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">836</span>– <span class="NLM_lpage">841</span>, <span class="refDoi"> DOI: 10.1038/ncb0910-836</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1038%2Fncb0910-836" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=20811356" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtFSju7zI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2010&pages=836-841&author=C.+Kraftauthor=M.+Peterauthor=K.+Hofmann&title=Selective+autophagy%3A+ubiquitin-mediated+recognition+and+beyond&doi=10.1038%2Fncb0910-836"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Selective autophagy: ubiquitin-mediated recognition and beyond</span></div><div class="casAuthors">Kraft, Claudine; Peter, Matthias; Hofmann, Kay</div><div class="citationInfo"><span class="NLM_cas:title">Nature Cell Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">836-841</span>CODEN:
                <span class="NLM_cas:coden">NCBIFN</span>;
        ISSN:<span class="NLM_cas:issn">1465-7392</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Eukaryotic cells use autophagy and the ubiquitin-proteasome system as their major protein degrdn. pathways.  Whereas the ubiquitin-proteasome system is involved in the rapid degrdn. of proteins, autophagy pathways can selectively remove protein aggregates and damaged or excess organelles.  Proteasome-mediated degrdn. requires previous ubiquitylation of the cargo, which is then recognized by ubiquitin receptors directing it to 26S proteasomes.  Although autophagy has long been viewed as a random cytoplasmic degrdn. system, the involvement of ubiquitin as a specificity factor for selective autophagy is rapidly emerging.  Recent evidence also suggests active crosstalk between proteasome-mediated degrdn. and selective autophagy.  Here, we discuss the mol. mechanisms that link autophagy and the proteasome system, as well as the emerging roles of ubiquitin and ubiquitin-binding proteins in selective autophagy.  On the basis of the evolutionary history of autophagic ubiquitin receptors, we propose a common origin for metazoan ubiquitin-dependent autophagy and the cytoplasm-to-vacuole targeting pathway of yeast.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoRPRAkcgcWKrVg90H21EOLACvtfcHk0ljBtPj2zSVngg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtFSju7zI&md5=53177b341b1165bf3497e81f22d7c30e</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1038%2Fncb0910-836&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncb0910-836%26sid%3Dliteratum%253Aachs%26aulast%3DKraft%26aufirst%3DC.%26aulast%3DPeter%26aufirst%3DM.%26aulast%3DHofmann%26aufirst%3DK.%26atitle%3DSelective%2520autophagy%253A%2520ubiquitin-mediated%2520recognition%2520and%2520beyond%26jtitle%3DNat.%2520Cell%2520Biol.%26date%3D2010%26volume%3D12%26spage%3D836%26epage%3D841%26doi%3D10.1038%2Fncb0910-836" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lamark, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johansen, T.</span></span> <span> </span><span class="NLM_article-title">Autophagy: links with the proteasome</span>. <i>Curr. Opin. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">192</span>– <span class="NLM_lpage">198</span>, <span class="refDoi"> DOI: 10.1016/j.ceb.2009.11.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1016%2Fj.ceb.2009.11.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=19962293" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC3cXks1SrtbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2010&pages=192-198&author=T.+Lamarkauthor=T.+Johansen&title=Autophagy%3A+links+with+the+proteasome&doi=10.1016%2Fj.ceb.2009.11.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Autophagy: links with the proteasome</span></div><div class="casAuthors">Lamark, Trond; Johansen, Terje</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Cell Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">192-198</span>CODEN:
                <span class="NLM_cas:coden">COCBE3</span>;
        ISSN:<span class="NLM_cas:issn">0955-0674</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The two main protein degrdn. systems of eukaryotic cells, the ubiquitin-proteasome system and autophagy, have been thought of as quite sep. systems.  However, recent findings strongly suggest that there is crosstalk and even cooperation between these two degrdn. pathways.  Ubiquitination and degrdn. of misfolded proteins by the ubiquitin-proteasome system have been investigated for some time, but much less is known about autophagic degrdn. of misfolded proteins.  We will here discuss recent findings that shed some light on the cellular processes deciding when and how misfolded proteins are specifically selected for autophagic degrdn. in favor of proteasomal degrdn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpgE1TVJTSLNrVg90H21EOLACvtfcHk0ljBtPj2zSVngg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXks1SrtbY%253D&md5=e0f73ff73f2d0943b020ed656f7bdc67</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2Fj.ceb.2009.11.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ceb.2009.11.002%26sid%3Dliteratum%253Aachs%26aulast%3DLamark%26aufirst%3DT.%26aulast%3DJohansen%26aufirst%3DT.%26atitle%3DAutophagy%253A%2520links%2520with%2520the%2520proteasome%26jtitle%3DCurr.%2520Opin.%2520Cell%2520Biol.%26date%3D2010%26volume%3D22%26spage%3D192%26epage%3D198%26doi%3D10.1016%2Fj.ceb.2009.11.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Paul, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, A.</span></span> <span> </span><span class="NLM_article-title">TRAF6 coordinates the activation of autophagy and ubiquitin-proteasome systems in atrophying skeletal muscle</span>. <i>Autophagy</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">555</span>– <span class="NLM_lpage">556</span>, <span class="refDoi"> DOI: 10.4161/auto.7.5.15102</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.4161%2Fauto.7.5.15102" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=21412053" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC38XitF2itg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=555-556&author=P.+K.+Paulauthor=A.+Kumar&title=TRAF6+coordinates+the+activation+of+autophagy+and+ubiquitin-proteasome+systems+in+atrophying+skeletal+muscle&doi=10.4161%2Fauto.7.5.15102"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">TRAF6 coordinates the activation of autophagy and ubiquitin-proteasome systems in atrophying skeletal muscle</span></div><div class="casAuthors">Paul, Pradyut K.; Kumar, Ashok</div><div class="citationInfo"><span class="NLM_cas:title">Autophagy</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">555-556</span>CODEN:
                <span class="NLM_cas:coden">AUTOC9</span>;
        ISSN:<span class="NLM_cas:issn">1554-8627</span>.
    
            (<span class="NLM_cas:orgname">Landes Bioscience</span>)
        </div><div class="casAbstract">Skeletal muscle wasting is a major reason for morbidity and mortality in many chronic disease states, disuse conditions and aging.  The ubiquitin-proteasome and autophagy-lysosomal systems are the two major proteolytic pathways involved in regulation of both physiol. and pathol. muscle wasting.  Tumor necrosis factor receptor (TNFR)-assocd. factor 6 (TRAF6) is an important adaptor protein involved in receptor-mediated activation of various signaling pathways in response to cytokines and bacterial products.  TRAF6 also possesses E3 ubiquitin ligase activity causing lysine-63-linked polyubiquitination of target proteins.  We have uncovered a novel role of TRAF6 in regulation of skeletal muscle mass.  Muscle-wasting stimuli upregulate the expression, as well as the auto-ubiquitination, of TRAF6 leading to downstream activation of major catabolic pathways in skeletal muscle.  Muscle-specific depletion of TRAF6 preserves skeletal muscle mass in a mouse model of cancer cachexia or denervation.  Inhibition of TRAF6 also blocks the expression of the components of the ubiquitin-proteasome system (UPS) and autophagosome formation in atrophying skeletal muscle.  While more investigations are required to understand its mechanisms of action in skeletal muscle, our results indicate that blocking TRAF6 activity can be used as a therapeutic approach to preserve skeletal muscle mass and function in different disease states and conditions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqrAeK7825ZorVg90H21EOLACvtfcHk0lgt858yLEIjFw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XitF2itg%253D%253D&md5=58f3bfc0cba102b1e10817f8a4cfaf41</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.4161%2Fauto.7.5.15102&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252Fauto.7.5.15102%26sid%3Dliteratum%253Aachs%26aulast%3DPaul%26aufirst%3DP.%2BK.%26aulast%3DKumar%26aufirst%3DA.%26atitle%3DTRAF6%2520coordinates%2520the%2520activation%2520of%2520autophagy%2520and%2520ubiquitin-proteasome%2520systems%2520in%2520atrophying%2520skeletal%2520muscle%26jtitle%3DAutophagy%26date%3D2011%26volume%3D7%26spage%3D555%26epage%3D556%26doi%3D10.4161%2Fauto.7.5.15102" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Veggiani, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerpe, M. C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sidhu, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span> <span> </span><span class="NLM_article-title">Emerging drug development technologies targeting ubiquitination for cancer therapeutics</span>. <i>Pharmacol. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>199</i></span>,  <span class="NLM_fpage">139</span>– <span class="NLM_lpage">154</span>, <span class="refDoi"> DOI: 10.1016/j.pharmthera.2019.03.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1016%2Fj.pharmthera.2019.03.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=30851297" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC1MXlvVCjtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=199&publication_year=2019&pages=139-154&author=G.+Veggianiauthor=M.+C.+R.+Gerpeauthor=S.+S.+Sidhuauthor=W.+Zhang&title=Emerging+drug+development+technologies+targeting+ubiquitination+for+cancer+therapeutics&doi=10.1016%2Fj.pharmthera.2019.03.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Emerging drug development technologies targeting ubiquitination for cancer therapeutics</span></div><div class="casAuthors">Veggiani, Gianluca; Gerpe, Maria Carla Rosales; Sidhu, Sachdev S.; Zhang, Wei</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacology & Therapeutics</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">199</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">139-154</span>CODEN:
                <span class="NLM_cas:coden">PHTHDT</span>;
        ISSN:<span class="NLM_cas:issn">0163-7258</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review.  Development of effective cancer therapeutic strategies relies on our ability to interfere with cellular processes that are dysregulated in tumors.  Given the essential role of the ubiquitin proteasome system (UPS) in regulating a myriad of cellular processes, it is not surprising that malfunction of UPS components is implicated in numerous human diseases, including many types of cancer.  The clin. success of proteasome inhibitors in treating multiple myeloma has further stimulated enthusiasm for targeting UPS proteins for pharmacol. intervention in cancer treatment, particularly in the precision medicine era.  Unfortunately, despite tremendous efforts, the paucity of potent and selective UPS inhibitors has severely hampered attempts to exploit the UPS for therapeutic benefits.  To tackle this problem, many groups have been working on technol. advancement to rapidly and effectively screen for potent and specific UPS modulators as intracellular probes or early-phase therapeutic agents.  Here, we review several emerging technologies for developing chem.- and protein-based mols. to manipulate UPS enzymic activity, with the aim of providing an overview of strategies available to target ubiquitination for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqnpUXmOrEplLVg90H21EOLACvtfcHk0lgt858yLEIjFw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXlvVCjtr8%253D&md5=fcf119fca745b463e67cc841c402ccdd</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2Fj.pharmthera.2019.03.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pharmthera.2019.03.003%26sid%3Dliteratum%253Aachs%26aulast%3DVeggiani%26aufirst%3DG.%26aulast%3DGerpe%26aufirst%3DM.%2BC.%2BR.%26aulast%3DSidhu%26aufirst%3DS.%2BS.%26aulast%3DZhang%26aufirst%3DW.%26atitle%3DEmerging%2520drug%2520development%2520technologies%2520targeting%2520ubiquitination%2520for%2520cancer%2520therapeutics%26jtitle%3DPharmacol.%2520Ther.%26date%3D2019%26volume%3D199%26spage%3D139%26epage%3D154%26doi%3D10.1016%2Fj.pharmthera.2019.03.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cohen-Kaplan, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Livneh, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Avni, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cohen-Rosenzweig, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciechanover, A.</span></span> <span> </span><span class="NLM_article-title">The ubiquitin-proteasome system and autophagy: coordinated and independent activities</span>. <i>Int. J. Biochem. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>79</i></span>,  <span class="NLM_fpage">403</span>– <span class="NLM_lpage">418</span>, <span class="refDoi"> DOI: 10.1016/j.biocel.2016.07.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1016%2Fj.biocel.2016.07.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=27448843" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1eisrzF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=2016&pages=403-418&author=V.+Cohen-Kaplanauthor=I.+Livnehauthor=N.+Avniauthor=C.+Cohen-Rosenzweigauthor=A.+Ciechanover&title=The+ubiquitin-proteasome+system+and+autophagy%3A+coordinated+and+independent+activities&doi=10.1016%2Fj.biocel.2016.07.019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">The ubiquitin-proteasome system and autophagy: Coordinated and independent activities</span></div><div class="casAuthors">Cohen-Kaplan, Victoria; Livneh, Ido; Avni, Noa; Cohen-Rosenzweig, Chen; Ciechanover, Aaron</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Biochemistry & Cell Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">79</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">403-418</span>CODEN:
                <span class="NLM_cas:coden">IJBBFU</span>;
        ISSN:<span class="NLM_cas:issn">1357-2725</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The living cell is an ever changing, responsive, and adaptive environment where proteins play key roles in all processes and functions.  While the scientific community focused for a long time on the decoding of the information required for protein synthesis, little attention was paid to the mechanisms by which proteins are removed from the cell.  We now realize that the timely and proper activity of proteins is regulated to a large extent by their degrdn.; that cellular coping with different physiol. cues and stress conditions depends on different catabolic pathways; and that many pathol. states result from improper protein breakdown.  There are two major protein degrdn. systems in all eukaryotic cells-the ubiquitin- proteasome and the autophagy-lysosome.  The two systems are highly regulated, and-via degrdn. of a broad array of proteins-are responsible for maintenance of protein homeostasis and adaptation to environmental changes.  Each is comprised of numerous components responsible for its coordinated function, and together they encompass a considerable fraction of the entire genome.  In this review, we shall discuss the common and diverse characteristics of the ubiquitin-proteasome system (UPS) and autophagy-their substructure, mechanisms of action, function and concerted regulation under varying pathophysiol. conditions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqrcq6cLaANorVg90H21EOLACvtfcHk0lgt858yLEIjFw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1eisrzF&md5=5ec871a4cfa9e9e2ffc69beab08bfd6c</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2Fj.biocel.2016.07.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.biocel.2016.07.019%26sid%3Dliteratum%253Aachs%26aulast%3DCohen-Kaplan%26aufirst%3DV.%26aulast%3DLivneh%26aufirst%3DI.%26aulast%3DAvni%26aufirst%3DN.%26aulast%3DCohen-Rosenzweig%26aufirst%3DC.%26aulast%3DCiechanover%26aufirst%3DA.%26atitle%3DThe%2520ubiquitin-proteasome%2520system%2520and%2520autophagy%253A%2520coordinated%2520and%2520independent%2520activities%26jtitle%3DInt.%2520J.%2520Biochem.%2520Cell%2520Biol.%26date%3D2016%26volume%3D79%26spage%3D403%26epage%3D418%26doi%3D10.1016%2Fj.biocel.2016.07.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cho, Y. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, Y. M.</span></span> <span> </span><span class="NLM_article-title">Autophagy regulates homeostasis of pluripotency-associated proteins in hESCs</span>. <i>Stem Cells</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">424</span>– <span class="NLM_lpage">435</span>, <span class="refDoi"> DOI: 10.1002/stem.1589</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1002%2Fstem.1589" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=24170349" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC2cXlsFOrtLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2014&pages=424-435&author=Y.+H.+Choauthor=K.+M.+Hanauthor=D.+Kimauthor=J.+Leeauthor=S.+H.+Leeauthor=K.+W.+Choiauthor=J.+Kimauthor=Y.+M.+Han&title=Autophagy+regulates+homeostasis+of+pluripotency-associated+proteins+in+hESCs&doi=10.1002%2Fstem.1589"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Autophagy regulates homeostasis of pluripotency-associated proteins in hESCs</span></div><div class="casAuthors">Cho, Yun-Hee; Han, Kyu-Min; Kim, Dongkyu; Lee, Joonsun; Lee, Sang-Hee; Choi, Kyeng-Won; Kim, Jungho; Han, Yong-Mahn</div><div class="citationInfo"><span class="NLM_cas:title">Stem Cells (Durham, NC, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">424-435</span>CODEN:
                <span class="NLM_cas:coden">STCEEJ</span>;
        ISSN:<span class="NLM_cas:issn">1066-5099</span>.
    
            (<span class="NLM_cas:orgname">AlphaMed Press</span>)
        </div><div class="casAbstract">The pluripotency of embryonic stem cells (ESCs) is maintained by intracellular networks of many pluripotency-assocd. (PA) proteins such as OCT4, SOX2, and NANOG.  However, the mechanisms underlying the regulation of protein homeostasis for pluripotency remain elusive.  Here, we first demonstrate that autophagy acts together with the ubiquitin-proteasome system (UPS) to modulate the levels of PA proteins in human ESCs (hESCs).  Autophagy inhibition impaired the pluripotency despite increment of PA proteins in hESCs.  Immunogold-electron microscopy confirmed localization of OCT4 mols. within autophagosomes.  Also, knockdown of LC3 expression led to accumulation of PA proteins and redn. of pluripotency in hESCs.  Interestingly, autophagy and the UPS showed differential kinetics in the degrdn. of PA proteins.  Autophagy inhibition caused enhanced accumulation of both cytoplasmic and nuclear PA proteins, whereas the UPS inhibition led to preferentially degrade nuclear PA proteins.  Our findings suggest that autophagy modulates homeostasis of PA proteins, providing a new insight in the regulation of pluripotency in hESCs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJTsLmH6KAlLVg90H21EOLACvtfcHk0ljSJE-by2XWZw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXlsFOrtLc%253D&md5=459ec78b2a2c988e67d6d902f123ffe6</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1002%2Fstem.1589&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fstem.1589%26sid%3Dliteratum%253Aachs%26aulast%3DCho%26aufirst%3DY.%2BH.%26aulast%3DHan%26aufirst%3DK.%2BM.%26aulast%3DKim%26aufirst%3DD.%26aulast%3DLee%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DS.%2BH.%26aulast%3DChoi%26aufirst%3DK.%2BW.%26aulast%3DKim%26aufirst%3DJ.%26aulast%3DHan%26aufirst%3DY.%2BM.%26atitle%3DAutophagy%2520regulates%2520homeostasis%2520of%2520pluripotency-associated%2520proteins%2520in%2520hESCs%26jtitle%3DStem%2520Cells%26date%3D2014%26volume%3D32%26spage%3D424%26epage%3D435%26doi%3D10.1002%2Fstem.1589" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ding, W. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ni, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshimori, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stolz, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ron, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, X. M.</span></span> <span> </span><span class="NLM_article-title">Linking of autophagy to ubiquitin-proteasome system is important for the regulation of endoplasmic reticulum stress and cell viability</span>. <i>Am. J. Pathol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>171</i></span>,  <span class="NLM_fpage">513</span>– <span class="NLM_lpage">524</span>, <span class="refDoi"> DOI: 10.2353/ajpath.2007.070188</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.2353%2Fajpath.2007.070188" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=17620365" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BD2sXpsFaiu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=171&publication_year=2007&pages=513-524&author=W.+X.+Dingauthor=H.+M.+Niauthor=W.+Gaoauthor=T.+Yoshimoriauthor=D.+B.+Stolzauthor=D.+Ronauthor=X.+M.+Yin&title=Linking+of+autophagy+to+ubiquitin-proteasome+system+is+important+for+the+regulation+of+endoplasmic+reticulum+stress+and+cell+viability&doi=10.2353%2Fajpath.2007.070188"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Linking of autophagy to ubiquitin-proteasome system is important for the regulation of endoplasmic reticulum stress and cell viability</span></div><div class="casAuthors">Ding, Wen-Xing; Ni, Hong-Min; Gao, Wentao; Yoshimori, Tamotsu; Stolz, Donna B.; Ron, David; Yin, Xiao-Ming</div><div class="citationInfo"><span class="NLM_cas:title">American Journal of Pathology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">171</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">513-524</span>CODEN:
                <span class="NLM_cas:coden">AJPAA4</span>;
        ISSN:<span class="NLM_cas:issn">0002-9440</span>.
    
            (<span class="NLM_cas:orgname">American Society for Investigative Pathology</span>)
        </div><div class="casAbstract">Two major protein degrdn. systems exist in cells, the ubiquitin proteasome system and the autophagy machinery.  Here, we investigated the functional relationship of the two systems and the underlying mechanisms.  Proteasome inhibition activated autophagy, suggesting that the two are functionally coupled.  Autophagy played a compensatory role as suppression of autophagy promoted the accumulation of polyubiquitinated protein aggregates.  Autophagy was likely activated in response to endoplasmic reticulum stress caused by misfolded proteins during proteasome inhibition.  Suppression of a major unfolded protein response pathway mediated by IRE1 by either gene deletion or RNA interference dramatically suppressed the activation of autophagy by proteasome inhibitors.  Interestingly, c-Jun NH2-terminal kinase (JNK) but not XBP-1, both of which are the known downstream targets of IRE1, seemed to participate in autophagy induction by proteasome inhibitors.  Finally, proteasome inhibitor-induced autophagy was important for controlling endoplasmic reticulum stress and reducing cell death in cancer cells.  Our studies thus provide a mechanistic view and elucidate the functional significance of the link between the two protein degrdn. systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrwdXb333fNrLVg90H21EOLACvtfcHk0ljSJE-by2XWZw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXpsFaiu7w%253D&md5=00ac3988a17c61c7ac929baa5cf5f8f5</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.2353%2Fajpath.2007.070188&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2353%252Fajpath.2007.070188%26sid%3Dliteratum%253Aachs%26aulast%3DDing%26aufirst%3DW.%2BX.%26aulast%3DNi%26aufirst%3DH.%2BM.%26aulast%3DGao%26aufirst%3DW.%26aulast%3DYoshimori%26aufirst%3DT.%26aulast%3DStolz%26aufirst%3DD.%2BB.%26aulast%3DRon%26aufirst%3DD.%26aulast%3DYin%26aufirst%3DX.%2BM.%26atitle%3DLinking%2520of%2520autophagy%2520to%2520ubiquitin-proteasome%2520system%2520is%2520important%2520for%2520the%2520regulation%2520of%2520endoplasmic%2520reticulum%2520stress%2520and%2520cell%2520viability%26jtitle%3DAm.%2520J.%2520Pathol.%26date%3D2007%26volume%3D171%26spage%3D513%26epage%3D524%26doi%3D10.2353%2Fajpath.2007.070188" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pandey, U. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nie, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Batlevi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCray, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ritson, G. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nedelsky, N. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwartz, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DiProspero, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knight, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuldiner, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Padmanabhan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hild, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berry, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garza, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hubbert, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, T. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baehrecke, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, J. P.</span></span> <span> </span><span class="NLM_article-title">HDAC6 rescues neurodegeneration and provides an essential link between autophagy and the UPS</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>447</i></span>,  <span class="NLM_fpage">859</span>– <span class="NLM_lpage">863</span>, <span class="refDoi"> DOI: 10.1038/nature05853</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1038%2Fnature05853" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=17568747" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BD2sXms1WjsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=447&publication_year=2007&pages=859-863&author=U.+B.+Pandeyauthor=Z.+Nieauthor=Y.+Batleviauthor=B.+A.+McCrayauthor=G.+P.+Ritsonauthor=N.+B.+Nedelskyauthor=S.+L.+Schwartzauthor=N.+A.+DiProsperoauthor=M.+A.+Knightauthor=O.+Schuldinerauthor=R.+Padmanabhanauthor=M.+Hildauthor=D.+L.+Berryauthor=D.+Garzaauthor=C.+C.+Hubbertauthor=T.+P.+Yaoauthor=E.+H.+Baehreckeauthor=J.+P.+Taylor&title=HDAC6+rescues+neurodegeneration+and+provides+an+essential+link+between+autophagy+and+the+UPS&doi=10.1038%2Fnature05853"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">HDAC6 rescues neurodegeneration and provides an essential link between autophagy and the UPS</span></div><div class="casAuthors">Pandey, Udai Bhan; Nie, Zhiping; Batlevi, Yakup; McCray, Brett A.; Ritson, Gillian P.; Nedelsky, Natalia B.; Schwartz, Stephanie L.; DiProspero, Nicholas A.; Knight, Melanie A.; Schuldiner, Oren; Padmanabhan, Ranjani; Hild, Marc; Berry, Deborah L.; Garza, Dan; Hubbert, Charlotte C.; Yao, Tso-Pang; Baehrecke, Eric H.; Taylor, J. Paul</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">447</span>
        (<span class="NLM_cas:issue">7146</span>),
    <span class="NLM_cas:pages">859-863</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A prominent feature of late-onset neurodegenerative diseases is accumulation of misfolded protein in vulnerable neurons.  When levels of misfolded protein overwhelm degradative pathways, the result is cellular toxicity and neurodegeneration.  Cellular mechanisms for degrading misfolded protein include the ubiquitin-proteasome system (UPS), the main non-lysosomal degradative pathway for ubiquitinated proteins, and autophagy, a lysosome-mediated degradative pathway.  The UPS and autophagy have long been viewed as complementary degrdn. systems with no point of intersection.  This view was challenged by 2 observations suggesting an apparent interaction: impairment of the UPS induces autophagy in vitro, and conditional knockout of autophagy in the mouse brain leads to neurodegeneration with ubiquitin-pos. pathol.  It is not known whether autophagy is strictly a parallel degrdn. system, or whether it is a compensatory degrdn. system when the UPS is impaired; furthermore, if there is a compensatory interaction between these systems, the mol. link is not known.  Here the authors show that autophagy acts as a compensatory degrdn. system when the UPS is impaired in Drosophila melanogaster, and that histone deacetylase 6 (HDAC6), a microtubule-assocd. deacetylase that interacts with polyubiquitinated proteins, is an essential mechanistic link in this compensatory interaction.  The authors found that compensatory autophagy was induced in response to mutations affecting the proteasome and in response to UPS impairment in a fly model of the neurodegenerative disease spinobulbar muscular atrophy.  Autophagy compensated for impaired UPS function in an HDAC6-dependent manner.  Furthermore, expression of HDAC6 was sufficient to rescue degeneration assocd. with UPS dysfunction in vivo in an autophagy-dependent manner.  This study suggests that impairment of autophagy (for example, assocd. with aging or genetic variation) might predispose to neurodegeneration.  Morover, these findings suggest that it may be possible to intervene in neurodegeneration by augmenting HDAC6 to enhance autophagy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqncfliMTrJvbVg90H21EOLACvtfcHk0ljSJE-by2XWZw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXms1WjsL8%253D&md5=75925c1b12fc4bd012332f4a9b07ad19</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1038%2Fnature05853&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature05853%26sid%3Dliteratum%253Aachs%26aulast%3DPandey%26aufirst%3DU.%2BB.%26aulast%3DNie%26aufirst%3DZ.%26aulast%3DBatlevi%26aufirst%3DY.%26aulast%3DMcCray%26aufirst%3DB.%2BA.%26aulast%3DRitson%26aufirst%3DG.%2BP.%26aulast%3DNedelsky%26aufirst%3DN.%2BB.%26aulast%3DSchwartz%26aufirst%3DS.%2BL.%26aulast%3DDiProspero%26aufirst%3DN.%2BA.%26aulast%3DKnight%26aufirst%3DM.%2BA.%26aulast%3DSchuldiner%26aufirst%3DO.%26aulast%3DPadmanabhan%26aufirst%3DR.%26aulast%3DHild%26aufirst%3DM.%26aulast%3DBerry%26aufirst%3DD.%2BL.%26aulast%3DGarza%26aufirst%3DD.%26aulast%3DHubbert%26aufirst%3DC.%2BC.%26aulast%3DYao%26aufirst%3DT.%2BP.%26aulast%3DBaehrecke%26aufirst%3DE.%2BH.%26aulast%3DTaylor%26aufirst%3DJ.%2BP.%26atitle%3DHDAC6%2520rescues%2520neurodegeneration%2520and%2520provides%2520an%2520essential%2520link%2520between%2520autophagy%2520and%2520the%2520UPS%26jtitle%3DNature%26date%3D2007%26volume%3D447%26spage%3D859%26epage%3D863%26doi%3D10.1038%2Fnature05853" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">di
Ronza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bajaj, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanagasetti, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lotfi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adamski, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collette, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palmieri, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amawi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Popp, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, K. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meschini, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leung, H. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Segatori, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simonati, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sifers, R. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santorelli, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sardiello, M.</span></span> <span> </span><span class="NLM_article-title">CLN8 is an endoplasmic reticulum cargo receptor that regulates lysosome biogenesis</span>. <i>Nat. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">1370</span>– <span class="NLM_lpage">1377</span>, <span class="refDoi"> DOI: 10.1038/s41556-018-0228-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1038%2Fs41556-018-0228-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=30397314" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitV2qu7bJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2018&pages=1370-1377&author=A.+di%0ARonzaauthor=L.+Bajajauthor=J.+Sharmaauthor=D.+Sanagasettiauthor=P.+Lotfiauthor=C.+J.+Adamskiauthor=J.+Colletteauthor=M.+Palmieriauthor=A.+Amawiauthor=L.+Poppauthor=K.+T.+Changauthor=M.+C.+Meschiniauthor=H.+E.+Leungauthor=L.+Segatoriauthor=A.+Simonatiauthor=R.+N.+Sifersauthor=F.+M.+Santorelliauthor=M.+Sardiello&title=CLN8+is+an+endoplasmic+reticulum+cargo+receptor+that+regulates+lysosome+biogenesis&doi=10.1038%2Fs41556-018-0228-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">CLN8 is an endoplasmic reticulum cargo receptor that regulates lysosome biogenesis</span></div><div class="casAuthors">di Ronza, Alberto; Bajaj, Lakshya; Sharma, Jaiprakash; Sanagasetti, Deepthi; Lotfi, Parisa; Adamski, Carolyn Joy; Collette, John; Palmieri, Michela; Amawi, Abdallah; Popp, Lauren; Chang, Kevin Tommy; Meschini, Maria Chiara; Leung, Hon-Chiu Eastwood; Segatori, Laura; Simonati, Alessandro; Sifers, Richard Norman; Santorelli, Filippo Maria; Sardiello, Marco</div><div class="citationInfo"><span class="NLM_cas:title">Nature Cell Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1370-1377</span>CODEN:
                <span class="NLM_cas:coden">NCBIFN</span>;
        ISSN:<span class="NLM_cas:issn">1465-7392</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Organelle biogenesis requires proper transport of proteins from their site of synthesis to their target subcellular compartment.  Lysosomal enzymes are synthesized in the endoplasmic reticulum (ER) and traffic through the Golgi complex before being transferred to the endolysosomal system, but how they are transferred from the ER to the Golgi is unknown.  Here, we show that ER-to-Golgi transfer of lysosomal enzymes requires CLN8, an ER-assocd. membrane protein whose loss of function leads to the lysosomal storage disorder, neuronal ceroid lipofuscinosis 8 (a type of Batten disease).  ER-to-Golgi trafficking of CLN8 requires interaction with the COPII and COPI machineries via specific export and retrieval signals localized in the cytosolic carboxy terminus of CLN8.  CLN8 deficiency leads to depletion of sol. enzymes in the lysosome, thus impairing lysosome biogenesis.  Binding to lysosomal enzymes requires the second luminal loop of CLN8 and is abolished by some disease-causing mutations within this region.  Our data establish an unanticipated example of an ER receptor serving the biogenesis of an organelle and indicate that impaired transport of lysosomal enzymes underlies Batten disease caused by mutations in CLN8.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzFrRJaP5Hq7Vg90H21EOLACvtfcHk0ljQnfX_XAOAKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitV2qu7bJ&md5=321fcc86732777bce171e5594e0e02b2</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1038%2Fs41556-018-0228-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41556-018-0228-7%26sid%3Dliteratum%253Aachs%26aulast%3Ddi%2BRonza%26aufirst%3DA.%26aulast%3DBajaj%26aufirst%3DL.%26aulast%3DSharma%26aufirst%3DJ.%26aulast%3DSanagasetti%26aufirst%3DD.%26aulast%3DLotfi%26aufirst%3DP.%26aulast%3DAdamski%26aufirst%3DC.%2BJ.%26aulast%3DCollette%26aufirst%3DJ.%26aulast%3DPalmieri%26aufirst%3DM.%26aulast%3DAmawi%26aufirst%3DA.%26aulast%3DPopp%26aufirst%3DL.%26aulast%3DChang%26aufirst%3DK.%2BT.%26aulast%3DMeschini%26aufirst%3DM.%2BC.%26aulast%3DLeung%26aufirst%3DH.%2BE.%26aulast%3DSegatori%26aufirst%3DL.%26aulast%3DSimonati%26aufirst%3DA.%26aulast%3DSifers%26aufirst%3DR.%2BN.%26aulast%3DSantorelli%26aufirst%3DF.%2BM.%26aulast%3DSardiello%26aufirst%3DM.%26atitle%3DCLN8%2520is%2520an%2520endoplasmic%2520reticulum%2520cargo%2520receptor%2520that%2520regulates%2520lysosome%2520biogenesis%26jtitle%3DNat.%2520Cell%2520Biol.%26date%3D2018%26volume%3D20%26spage%3D1370%26epage%3D1377%26doi%3D10.1038%2Fs41556-018-0228-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Griffiths, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoflack, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simons, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mellman, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kornfeld, S.</span></span> <span> </span><span class="NLM_article-title">The mannose 6-phosphate receptor and the biogenesis of lysosomes</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">1988</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">329</span>– <span class="NLM_lpage">341</span>, <span class="refDoi"> DOI: 10.1016/S0092-8674(88)80026-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1016%2FS0092-8674%2888%2980026-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=2964276" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADyaL1cXhs1Ggtrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=1988&pages=329-341&author=G.+Griffithsauthor=B.+Hoflackauthor=K.+Simonsauthor=I.+Mellmanauthor=S.+Kornfeld&title=The+mannose+6-phosphate+receptor+and+the+biogenesis+of+lysosomes&doi=10.1016%2FS0092-8674%2888%2980026-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">The mannose 6-phosphate receptor and the biogenesis of lysosomes</span></div><div class="casAuthors">Griffiths, Gareth; Hoflack, Bernard; Simons, Kai; Mellman, Ira; Kornfeld, Stuart</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">1988</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">329-41</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    </div><div class="casAbstract">Localization of the 215-kilodalton mannose 6-phosphate receptor (MPR) was studied in normal rat kidney cells.  Low levels of receptor were detected in the trans Golgi network, Golgi stack, plasma membrane, and peripheral endosomes.  The bulk of the receptor was localized to an acidic, reticular-vesicular structure adjacent to the Golgi complex.  The structure also labeled with antibodies to lysosomal enzymes and a lysosomal membrane glycoprotein (Igp120).  Although lysosome-like, this structure is not a typical lysosome that is devoid of MPRs.  The endocytic marker α2-macroglobulin-Au entered the structure at 37°, but not at 20°.  With prolonged chase, most of the marker was transported from the structure into lysosomes.  It is proposed that the MPR/Ipg-enriched structure is a specialized endosome (prelysosome) that serves as an intermediate compartment into which endocytic vesicles discharge their contents, and where lysosomal enzymes are released from the MPR and packaged along with newly synthesized lysosomal glycoproteins into lysosomes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbMt6dilOlvbVg90H21EOLACvtfcHk0ljQnfX_XAOAKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1cXhs1Ggtrc%253D&md5=1e662f6d51a65732b2641a015ffff0ac</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1016%2FS0092-8674%2888%2980026-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0092-8674%252888%252980026-6%26sid%3Dliteratum%253Aachs%26aulast%3DGriffiths%26aufirst%3DG.%26aulast%3DHoflack%26aufirst%3DB.%26aulast%3DSimons%26aufirst%3DK.%26aulast%3DMellman%26aufirst%3DI.%26aulast%3DKornfeld%26aufirst%3DS.%26atitle%3DThe%2520mannose%25206-phosphate%2520receptor%2520and%2520the%2520biogenesis%2520of%2520lysosomes%26jtitle%3DCell%26date%3D1988%26volume%3D52%26spage%3D329%26epage%3D341%26doi%3D10.1016%2FS0092-8674%2888%2980026-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ni, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Canuel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morales, C. R.</span></span> <span> </span><span class="NLM_article-title">The sorting and trafficking of lysosomal proteins</span>. <i>Histol. Histopathol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">899</span>– <span class="NLM_lpage">913</span>, <span class="refDoi"> DOI: 10.14670/HH-21.899</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.14670%2FHH-21.899" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=16691542" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtF2gsLfE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2006&pages=899-913&author=X.+Niauthor=M.+Canuelauthor=C.+R.+Morales&title=The+sorting+and+trafficking+of+lysosomal+proteins&doi=10.14670%2FHH-21.899"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">The sorting and trafficking of lysosomal proteins</span></div><div class="casAuthors">Ni, X.; Canuel, M.; Morales, C. R.</div><div class="citationInfo"><span class="NLM_cas:title">Histology and Histopathology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">7,8 and 9</span>),
    <span class="NLM_cas:pages">899-913</span>CODEN:
                <span class="NLM_cas:coden">HIHIES</span>;
        ISSN:<span class="NLM_cas:issn">0213-3911</span>.
    
            (<span class="NLM_cas:orgname">Jimenez Godoy, S.A.</span>)
        </div><div class="casAbstract">A review.  For a long time lysosomes were considered terminal organelles involved in the degrdn. of different substrates.  However, this view is rapidly changing by evidence demonstrating that these organelles and their content display specialized functions in addn. to the degrdn. of substances.  Many lysosomal proteins have been implicated in specialized cellular functions and disorders such as antigen processing, targeting of surfactant proteins, and most lysosomal storage disorders.  To date, about 50 lysosomal hydrolases have been identified, and the majority of them are targeted to the lysosomes via the mannose 6-phosphate receptor (M6P-Rc).  However, recent studies on the intracellular trafficking of the nonenzymic lysosomal proteins, prosaposin and GM2 activator (GM2AP), demonstrated that they use an alternative receptor termed "sortilin".  Existing evidence suggests that some hydrolases traffic to the lysosomes in a mannose 6-phosphate-independent manner.  The possibility that sortilin is implicated in the targeting of some sol. hydrolases, as well as the consequences of this process, is addressed in the present review.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoyRGG79S6lg7Vg90H21EOLACvtfcHk0ljQnfX_XAOAKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtF2gsLfE&md5=537893c9550c41eeb77235fbfcc9165b</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.14670%2FHH-21.899&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.14670%252FHH-21.899%26sid%3Dliteratum%253Aachs%26aulast%3DNi%26aufirst%3DX.%26aulast%3DCanuel%26aufirst%3DM.%26aulast%3DMorales%26aufirst%3DC.%2BR.%26atitle%3DThe%2520sorting%2520and%2520trafficking%2520of%2520lysosomal%2520proteins%26jtitle%3DHistol.%2520Histopathol.%26date%3D2006%26volume%3D21%26spage%3D899%26epage%3D913%26doi%3D10.14670%2FHH-21.899" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Braulke, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonifacino, J. S.</span></span> <span> </span><span class="NLM_article-title">Sorting of lysosomal proteins</span>. <i>Biochim. Biophys. Acta, Mol. Cell Res.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>1793</i></span>,  <span class="NLM_fpage">605</span>– <span class="NLM_lpage">614</span>, <span class="refDoi"> DOI: 10.1016/j.bbamcr.2008.10.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1016%2Fj.bbamcr.2008.10.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=19046998" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BD1MXjslOlsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1793&publication_year=2009&pages=605-614&author=T.+Braulkeauthor=J.+S.+Bonifacino&title=Sorting+of+lysosomal+proteins&doi=10.1016%2Fj.bbamcr.2008.10.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Sorting of lysosomal proteins</span></div><div class="casAuthors">Braulke, Thomas; Bonifacino, Juan S.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Molecular Cell Research</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">1793</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">605-614</span>CODEN:
                <span class="NLM_cas:coden">BBAMCO</span>;
        ISSN:<span class="NLM_cas:issn">0167-4889</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Lysosomes are composed of sol. and transmembrane proteins that are targeted to lysosomes in a signal-dependent manner.  The majority of sol. acid hydrolases are modified with mannose 6-phosphate (M6P) residues, allowing their recognition by M6P receptors in the Golgi complex and ensuing transport to the endosomal/lysosomal system.  Other sol. enzymes and nonenzymic proteins are transported to lysosomes in an M6P-independent manner mediated by alternative receptors such as lysosomal integral membrane protein LIMP-2 or sortilin.  The sorting of cargo receptors and lysosomal transmembrane proteins requires sorting signals present in their cytosolic domains.  These signals include di-leucine-based motifs, DXXLL or [DE]XXXL[LI], and tyrosine-based motifs, YXXO, which interact with components of clathrin coats such as GGAs or adaptor protein complexes.  In addn., phosphorylation and lipid modifications regulate signal recognition and trafficking of lysosomal membrane proteins.  The complex interaction of both luminal and cytosolic signals with recognition proteins guarantees the specific and directed transport of proteins to lysosomes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpGvqmTon5cC7Vg90H21EOLACvtfcHk0lgOdnCqNlHVbw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXjslOlsb4%253D&md5=ffcb5e646608aec952b6d6cc9e3c6837</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1016%2Fj.bbamcr.2008.10.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbamcr.2008.10.016%26sid%3Dliteratum%253Aachs%26aulast%3DBraulke%26aufirst%3DT.%26aulast%3DBonifacino%26aufirst%3DJ.%2BS.%26atitle%3DSorting%2520of%2520lysosomal%2520proteins%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Mol.%2520Cell%2520Res.%26date%3D2009%26volume%3D1793%26spage%3D605%26epage%3D614%26doi%3D10.1016%2Fj.bbamcr.2008.10.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Coutinho, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prata, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alves, S.</span></span> <span> </span><span class="NLM_article-title">A shortcut to the lysosome: the mannose-6-phosphate-independent pathway</span>. <i>Mol. Genet. Metab.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>107</i></span>,  <span class="NLM_fpage">257</span>– <span class="NLM_lpage">266</span>, <span class="refDoi"> DOI: 10.1016/j.ymgme.2012.07.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1016%2Fj.ymgme.2012.07.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=22884962" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtF2it7jL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=107&publication_year=2012&pages=257-266&author=M.+F.+Coutinhoauthor=M.+J.+Prataauthor=S.+Alves&title=A+shortcut+to+the+lysosome%3A+the+mannose-6-phosphate-independent+pathway&doi=10.1016%2Fj.ymgme.2012.07.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">A shortcut to the lysosome: The mannose-6-phosphate-independent pathway</span></div><div class="casAuthors">Coutinho, Maria Francisca; Prata, Maria Joao; Alves, Sandra</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Genetics and Metabolism</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">107</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">257-266</span>CODEN:
                <span class="NLM_cas:coden">MGMEFF</span>;
        ISSN:<span class="NLM_cas:issn">1096-7192</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Lysosomal hydrolases have long been known to be responsible for the degrdn. of different substrates in the cell.  These acid hydrolases are synthesized in the rough endoplasmic reticulum and transported through the Golgi app. to the trans-Golgi network (TGN).  From there, they are delivered to endosomal/lysosomal compartments, where they finally become active due to the acidic pH characteristic of the lysosomal compartment.  The majority of the enzymes leave the TGN after modification with mannose-6-phosphate (M6P) residues, which are specifically recognized by M6P receptors (MPRs), ensuring their transport to the endosomal/lysosomal system.  Although M6P receptors play a major role in the intracellular transport of newly synthesized lysosomal enzymes in mammalian cells, several lines of evidence suggest the existence of alternative processes of lysosomal targeting.  Among them, the 2 that are mediated by the M6P alternative receptors, lysosomal integral membrane protein (LIMP-2) and sortilin, have gained unequivocal support.  LIMP-2 was shown to be implicated in the delivery of beta-glucocerebrosidase (GCase) to the lysosomes, whereas sortilin was suggested to be a multifunctional receptor capable of binding several different ligands, including neurotensin and receptor-assocd. protein (RAP), and of targeting several proteins to the lysosome, including sphingolipid activator proteins (prosaposin and GM2 activator protein), acid sphingomyelinase and cathepsins D and H.  Here, the authors review the current knowledge on these 2 proteins: their discovery, study, structural features and cellular function, with special attention to their role as alternative receptors to lysosomal trafficking.  Recent studies assocg. both LIMP2 and sortilin to disease are also extensively reviewed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqcyx4mje1htrVg90H21EOLACvtfcHk0lgOdnCqNlHVbw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtF2it7jL&md5=690f6719cd28e370bf09a74cdf2ffdc6</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1016%2Fj.ymgme.2012.07.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ymgme.2012.07.012%26sid%3Dliteratum%253Aachs%26aulast%3DCoutinho%26aufirst%3DM.%2BF.%26aulast%3DPrata%26aufirst%3DM.%2BJ.%26aulast%3DAlves%26aufirst%3DS.%26atitle%3DA%2520shortcut%2520to%2520the%2520lysosome%253A%2520the%2520mannose-6-phosphate-independent%2520pathway%26jtitle%3DMol.%2520Genet.%2520Metab.%26date%3D2012%26volume%3D107%26spage%3D257%26epage%3D266%26doi%3D10.1016%2Fj.ymgme.2012.07.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Malini, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zampieri, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deganuto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romanello, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sechi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bembi, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dardis, A.</span></span> <span> </span><span class="NLM_article-title">Role of LIMP-2 in the intracellular trafficking of β-glucosidase in different human cellular models</span>. <i>FASEB J.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">3839</span>– <span class="NLM_lpage">3852</span>, <span class="refDoi"> DOI: 10.1096/fj.15-271148</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1096%2Ffj.15-271148" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=26018676" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC28XnsFegsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2015&pages=3839-3852&author=E.+Maliniauthor=S.+Zampieriauthor=M.+Deganutoauthor=M.+Romanelloauthor=A.+Sechiauthor=B.+Bembiauthor=A.+Dardis&title=Role+of+LIMP-2+in+the+intracellular+trafficking+of+%CE%B2-glucosidase+in+different+human+cellular+models&doi=10.1096%2Ffj.15-271148"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Role of LIMP-2 in the intracellular trafficking of β-glucosidase in different human cellular models</span></div><div class="casAuthors">Malini, Erika; Zampieri, Stefania; Deganuto, Marta; Romanello, Milena; Sechi, Annalisa; Bembi, Bruno; Dardis, Andrea</div><div class="citationInfo"><span class="NLM_cas:title">FASEB Journal</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3839-3852</span>CODEN:
                <span class="NLM_cas:coden">FAJOEC</span>;
        ISSN:<span class="NLM_cas:issn">0892-6638</span>.
    
            (<span class="NLM_cas:orgname">Federation of American Societies for Experimental Biology</span>)
        </div><div class="casAbstract">Acid β-glucosidase (GCase), the enzyme deficient in Gaucher disease (GD), is transported to lysosomes by the lysosomal integral membrane protein (LIMP)-2.  In humans, LIMP-2 deficiency leads to action myoclonus-renal failure (AMRF) syndrome.  GD and AMRF syndrome share some clin. features.  However, they are different from clin. and biochem. points of view, suggesting that the role of LIMP-2 in the targeting of GCase would be different in different tissues.  Besides, the role of LIMP-2 in the uptake and trafficking of the human recombinant (hr)GCase used in the treatment of GD is unknown.  Thus, we compared GCase activity and intracellular localization in immortalized lymphocytes, fibroblasts, and a neuronal model derived from multipotent adult stem cells, from a patient with AMRF syndrome, patients with GD, and control subjects.  In fibroblasts and neuronlike cells, GCase targeting to the lysosomes is completely dependent on LIMP-2, whereas in blood cells, GCase is partially targeted to lysosomes by a LIMP-2-independent mechanism.  Although hrGCase cellular uptake is independent of LIMP-2, its trafficking to the lysosomes is mediated by this receptor.  These data provide new insights into the mechanisms involved in the intracellular trafficking of GCase and in the pathogeneses of GD and AMRF syndrome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpk_1m_RE750LVg90H21EOLACvtfcHk0lgOdnCqNlHVbw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XnsFegsLc%253D&md5=402baba943ebc321d779633afab66dd8</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1096%2Ffj.15-271148&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1096%252Ffj.15-271148%26sid%3Dliteratum%253Aachs%26aulast%3DMalini%26aufirst%3DE.%26aulast%3DZampieri%26aufirst%3DS.%26aulast%3DDeganuto%26aufirst%3DM.%26aulast%3DRomanello%26aufirst%3DM.%26aulast%3DSechi%26aufirst%3DA.%26aulast%3DBembi%26aufirst%3DB.%26aulast%3DDardis%26aufirst%3DA.%26atitle%3DRole%2520of%2520LIMP-2%2520in%2520the%2520intracellular%2520trafficking%2520of%2520%25CE%25B2-glucosidase%2520in%2520different%2520human%2520cellular%2520models%26jtitle%3DFASEB%2520J.%26date%3D2015%26volume%3D29%26spage%3D3839%26epage%3D3852%26doi%3D10.1096%2Ffj.15-271148" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ouyang, L.</span></span> <span> </span><span class="NLM_article-title">Targeting autophagy-related protein kinases for potential therapeutic purpose</span>. <i>Acta Pharm. Sin. B</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">569</span>– <span class="NLM_lpage">581</span>, <span class="refDoi"> DOI: 10.1016/j.apsb.2019.10.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1016%2Fj.apsb.2019.10.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=32322463" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BB3MXnslWjug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2020&pages=569-581&author=H.+Xiangauthor=J.+Zhangauthor=C.+Linauthor=L.+Zhangauthor=B.+Liuauthor=L.+Ouyang&title=Targeting+autophagy-related+protein+kinases+for+potential+therapeutic+purpose&doi=10.1016%2Fj.apsb.2019.10.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting autophagy-related protein kinases for potential therapeutic purpose</span></div><div class="casAuthors">Xiang, Honggang; Zhang, Jifa; Lin, Congcong; Zhang, Lan; Liu, Bo; Ouyang, Liang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Pharmaceutica Sinica B</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">569-581</span>CODEN:
                <span class="NLM_cas:coden">APSBCW</span>;
        ISSN:<span class="NLM_cas:issn">2211-3835</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Autophagy, defined as a scavenging process of protein aggregates and damaged organelles mediated by lysosomes, plays a significant role in the quality control of macromols. and organelles.  Since protein kinases are integral to the autophagy process, it is critically important to understand the role of kinases in autophagic regulation.  At present, intervention of autophagic processes by small-mol. modulators targeting specific kinases has becoming a reasonable and prevalent strategy for treating several varieties of human disease, esp. cancer.  In this review, we describe the role of some autophagy-related kinase targets and kinase-mediated phosphorylation mechanisms in autophagy regulation.  We also summarize the small-mol. kinase inhibitors/activators of these targets, high-lighting the opportunities of these new therapeutic agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUZ5eGWVZ5RbVg90H21EOLACvtfcHk0liZb0yfuMLKjQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXnslWjug%253D%253D&md5=277da77a9145fb63c521c5babf333e9b</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1016%2Fj.apsb.2019.10.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.apsb.2019.10.003%26sid%3Dliteratum%253Aachs%26aulast%3DXiang%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DLin%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DLiu%26aufirst%3DB.%26aulast%3DOuyang%26aufirst%3DL.%26atitle%3DTargeting%2520autophagy-related%2520protein%2520kinases%2520for%2520potential%2520therapeutic%2520purpose%26jtitle%3DActa%2520Pharm.%2520Sin.%2520B%26date%3D2020%26volume%3D10%26spage%3D569%26epage%3D581%26doi%3D10.1016%2Fj.apsb.2019.10.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ouyang, L.</span></span> <span> </span><span class="NLM_article-title">Discovery of a novel dual-Target inhibitor of ERK1 and ERK5 that induces regulated cell death to overcome compensatory mechanism in specific tumor types</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">3976</span>– <span class="NLM_lpage">3995</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b01896</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01896" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BB3cXjsFOku7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=3976-3995&author=G.+Wangauthor=Y.+Zhaoauthor=Y.+Liuauthor=D.+Sunauthor=Y.+Zhenauthor=J.+Liuauthor=L.+Fuauthor=L.+Zhangauthor=L.+Ouyang&title=Discovery+of+a+novel+dual-Target+inhibitor+of+ERK1+and+ERK5+that+induces+regulated+cell+death+to+overcome+compensatory+mechanism+in+specific+tumor+types&doi=10.1021%2Facs.jmedchem.9b01896"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Novel Dual-Target Inhibitor of ERK1 and ERK5 That Induces Regulated Cell Death to Overcome Compensatory Mechanism in Specific Tumor Types</span></div><div class="casAuthors">Wang, Guan; Zhao, Yuqian; Liu, Yao; Sun, Dejuan; Zhen, Yongqi; Liu, Jie; Fu, Leilei; Zhang, Lan; Ouyang, Liang</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3976-3995</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">ERK1 and ERK5 are proposed to have pivotal roles in several types of cancer.  Under some circumstance, ERK5 may provide a common bypass route, which rescues proliferation upon abrogation of ERK1 signaling.  Thus, we accurately classified the tumor types from The Cancer Genome Atlas (TCGA) based on the expression levels of ERK1 and ERK5.  We proposed a novel therapeutic strategy to overcome the above-mentioned compensatory mechanism in specific tumor types by co-targeting both ERK1 and ERK5.  On the basis of the idea of overcoming ERK5 compensation mechanism, 22ac (ADTL-EI1712) as the first selective dual-target inhibitor of ERK1 and ERK5 was discovered to have potent antitumor effects in vitro and in vivo.  Interestingly, this compd. was found to induce regulated cell death accompanied by autophagy in MKN-74 cells.  Taken together, our results warrant the potential of this dual-target inhibitor as a new candidate drug that conquers compensatory mechanism in certain tumor types.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlxVBmkLH7mrVg90H21EOLACvtfcHk0liZb0yfuMLKjQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXjsFOku7c%253D&md5=a39e2b19b4e97ba739e78f2453335714</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01896&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01896%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DG.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DSun%26aufirst%3DD.%26aulast%3DZhen%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DFu%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DOuyang%26aufirst%3DL.%26atitle%3DDiscovery%2520of%2520a%2520novel%2520dual-Target%2520inhibitor%2520of%2520ERK1%2520and%2520ERK5%2520that%2520induces%2520regulated%2520cell%2520death%2520to%2520overcome%2520compensatory%2520mechanism%2520in%2520specific%2520tumor%2520types%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D3976%26epage%3D3995%26doi%3D10.1021%2Facs.jmedchem.9b01896" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Doherty, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McMahon, H. T.</span></span> <span> </span><span class="NLM_article-title">Mechanisms of endocytosis</span>. <i>Annu. Rev. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>78</i></span>,  <span class="NLM_fpage">857</span>– <span class="NLM_lpage">902</span>, <span class="refDoi"> DOI: 10.1146/annurev.biochem.78.081307.110540</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1146%2Fannurev.biochem.78.081307.110540" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=19317650" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BD1MXos1Ghu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=78&publication_year=2009&pages=857-902&author=G.+J.+Dohertyauthor=H.+T.+McMahon&title=Mechanisms+of+endocytosis&doi=10.1146%2Fannurev.biochem.78.081307.110540"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanisms of endocytosis</span></div><div class="casAuthors">Doherty, Gary J.; McMahon, Harvey T.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Biochemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">78</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">857-902</span>CODEN:
                <span class="NLM_cas:coden">ARBOAW</span>;
        ISSN:<span class="NLM_cas:issn">0066-4154</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review.  Endocytic mechanisms control the lipid and protein compn. of the plasma membrane, thereby regulating how cells interact with their environments.  Here, the authors review what is known about mammalian endocytic mechanisms, with a focus on the cellular proteins that control these events.  The authors discuss the well-studied clathrin-mediated endocytic mechanisms and dissect endocytic pathways that proceed independently of clathrin.  These clathrin-independent pathways include the CLIC/GEEC endocytic pathway, arf6-dependent endocytosis, flotillin-dependent endocytosis, macropinocytosis, circular doral ruffles, phagocytosis, and trans-endocytosis.  The authors also critically review the role of caveolae and caveolin 1 in endocytosis.  The authors highlight the roles of lipids, membrane curvature-modulating proteins, small G proteins, actin, and dynamin in endocytic pathways.  The authors discuss the functional relevance of distinct endocytic pathways and emphasize the importance of studying these pathways to understanding human disease processes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphxXOCrv5TH7Vg90H21EOLACvtfcHk0liZb0yfuMLKjQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXos1Ghu7s%253D&md5=da59d8e8c3c629f548b91e0899693a4f</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1146%2Fannurev.biochem.78.081307.110540&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.biochem.78.081307.110540%26sid%3Dliteratum%253Aachs%26aulast%3DDoherty%26aufirst%3DG.%2BJ.%26aulast%3DMcMahon%26aufirst%3DH.%2BT.%26atitle%3DMechanisms%2520of%2520endocytosis%26jtitle%3DAnnu.%2520Rev.%2520Biochem.%26date%3D2009%26volume%3D78%26spage%3D857%26epage%3D902%26doi%3D10.1146%2Fannurev.biochem.78.081307.110540" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mayor, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pagano, R. E.</span></span> <span> </span><span class="NLM_article-title">Pathways of clathrin-independent endocytosis</span>. <i>Nat. Rev. Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">603</span>– <span class="NLM_lpage">612</span>, <span class="refDoi"> DOI: 10.1038/nrm2216</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1038%2Fnrm2216" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=17609668" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BD2sXotVahtrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2007&pages=603-612&author=S.+Mayorauthor=R.+E.+Pagano&title=Pathways+of+clathrin-independent+endocytosis&doi=10.1038%2Fnrm2216"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Pathways of clathrin-independent endocytosis</span></div><div class="casAuthors">Mayor, Satyajit; Pagano, Richard E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">603-612</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  There are numerous ways that endocytic cargo mols. may be internalized from the surface of eukaryotic cells.  In addn. to the classical clathrin-dependent mechanism of endocytosis, several pathways that do not use a clathrin coat are emerging.  These pathways transport a diverse array of cargoes and are sometimes hijacked by bacteria and viruses to gain access to the host cell.  Here, the authors review the current understanding of various clathrin-independent mechanisms of endocytosis and propose a classification scheme to help organize the data in this complex and evolving field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqwyq4_P7rbMrVg90H21EOLACvtfcHk0lgU6fOV053mSg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXotVahtrg%253D&md5=94d628b1b0f415086bcab5113bfdd801</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1038%2Fnrm2216&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm2216%26sid%3Dliteratum%253Aachs%26aulast%3DMayor%26aufirst%3DS.%26aulast%3DPagano%26aufirst%3DR.%2BE.%26atitle%3DPathways%2520of%2520clathrin-independent%2520endocytosis%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2007%26volume%3D8%26spage%3D603%26epage%3D612%26doi%3D10.1038%2Fnrm2216" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klionsky, D. J.</span></span> <span> </span><span class="NLM_article-title">Eaten alive: a history of macroautophagy</span>. <i>Nat. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">814</span>– <span class="NLM_lpage">822</span>, <span class="refDoi"> DOI: 10.1038/ncb0910-814</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1038%2Fncb0910-814" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=20811353" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtFSju7zE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2010&pages=814-822&author=Z.+Yangauthor=D.+J.+Klionsky&title=Eaten+alive%3A+a+history+of+macroautophagy&doi=10.1038%2Fncb0910-814"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Eaten alive: a history of macroautophagy</span></div><div class="casAuthors">Yang, Zhifen; Klionsky, Daniel J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Cell Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">814-822</span>CODEN:
                <span class="NLM_cas:coden">NCBIFN</span>;
        ISSN:<span class="NLM_cas:issn">1465-7392</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Macroautophagy (hereafter autophagy), or 'self-eating', is a conserved cellular pathway that controls protein and organelle degrdn., and has essential roles in survival, development and homeostasis.  Autophagy is also integral to human health and is involved in physiol., development, lifespan and a wide range of diseases, including cancer, neurodegeneration and microbial infection.  Although research on this topic began in the late 1950s, substantial progress in the mol. study of autophagy has taken place during only the past 15 years.  This review traces the key findings that led to our current mol. understanding of this complex process.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJ_iZJ3YUoHrVg90H21EOLACvtfcHk0lgU6fOV053mSg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtFSju7zE&md5=a20abac7db3f20eac668c41052413397</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1038%2Fncb0910-814&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncb0910-814%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DZ.%26aulast%3DKlionsky%26aufirst%3DD.%2BJ.%26atitle%3DEaten%2520alive%253A%2520a%2520history%2520of%2520macroautophagy%26jtitle%3DNat.%2520Cell%2520Biol.%26date%3D2010%26volume%3D12%26spage%3D814%26epage%3D822%26doi%3D10.1038%2Fncb0910-814" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Komatsu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ichimura, Y.</span></span> <span> </span><span class="NLM_article-title">Selective autophagy regulates various cellular functions</span>. <i>Genes Cells.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">923</span>– <span class="NLM_lpage">933</span>, <span class="refDoi"> DOI: 10.1111/j.1365-2443.2010.01433.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1111%2Fj.1365-2443.2010.01433.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=20670274" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtFeqsrvF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2010&pages=923-933&author=M.+Komatsuauthor=Y.+Ichimura&title=Selective+autophagy+regulates+various+cellular+functions&doi=10.1111%2Fj.1365-2443.2010.01433.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Selective autophagy regulates various cellular functions</span></div><div class="casAuthors">Komatsu, Masaaki; Ichimura, Yoshinobu</div><div class="citationInfo"><span class="NLM_cas:title">Genes to Cells</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">923-933</span>CODEN:
                <span class="NLM_cas:coden">GECEFL</span>;
        ISSN:<span class="NLM_cas:issn">1356-9597</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  Autophagy is a self-eating system conserved among eukaryotes, in which cellular components including organelles are entrapped into a double membrane structure called the autophagosome and then degraded by lysosomal hydrolases.  In addn. to its role in supplying amino acids in response to nutrient starvation, autophagy is involved in quality control to maintain cell health.  Thus, inactivation of autophagy causes the formation of cytoplasmic protein inclusions, which comprise misfolded proteins and the accumulation of many degenerated organelles, resulting in liver injury, diabetes, myopathy and neurodegeneration.  Furthermore, although autophagy has been considered nonselective, increasing evidence points to the selectivity of autophagy in sorting vacuolar enzymes and removal of aggregate-prone proteins and unwanted organelles.  Such selectivity allows diverse cellular regulation, similar to the ubiquitin proteasome pathway.  In this review, we discuss the physiol. roles of selective autophagy and their mol. mechanisms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq1HKtcqB-IHLVg90H21EOLACvtfcHk0lgU6fOV053mSg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtFeqsrvF&md5=5ff745e51923a3ca69aa792fcfc62cc2</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1111%2Fj.1365-2443.2010.01433.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1365-2443.2010.01433.x%26sid%3Dliteratum%253Aachs%26aulast%3DKomatsu%26aufirst%3DM.%26aulast%3DIchimura%26aufirst%3DY.%26atitle%3DSelective%2520autophagy%2520regulates%2520various%2520cellular%2520functions%26jtitle%3DGenes%2520Cells.%26date%3D2010%26volume%3D15%26spage%3D923%26epage%3D933%26doi%3D10.1111%2Fj.1365-2443.2010.01433.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lamark, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Svenning, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johansen, T.</span></span> <span> </span><span class="NLM_article-title">Regulation of selective autophagy: the p62/SQSTM1 paradigm</span>. <i>Essays Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">609</span>– <span class="NLM_lpage">624</span>, <span class="refDoi"> DOI: 10.1042/EBC20170035</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1042%2FEBC20170035" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=29233872" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A280%3ADC%252BC1MzgvVSktQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2017&pages=609-624&author=T.+Lamarkauthor=S.+Svenningauthor=T.+Johansen&title=Regulation+of+selective+autophagy%3A+the+p62%2FSQSTM1+paradigm&doi=10.1042%2FEBC20170035"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of selective autophagy: the p62/SQSTM1 paradigm</span></div><div class="casAuthors">Lamark Trond; Svenning Steingrim; Johansen Terje</div><div class="citationInfo"><span class="NLM_cas:title">Essays in biochemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">609-624</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">In selective autophagy, cytoplasmic components are selected and tagged before being sequestered into an autophagosome by means of selective autophagy receptors such as p62/SQSTM1.  In this review, we discuss how selective autophagy is regulated.  An important level of regulation is the selection of proteins or organelles for degradation.  Components selected for degradation are tagged, often with ubiquitin, to facilitate recognition by autophagy receptors.  Another level of regulation is represented by the autophagy receptors themselves.  For p62, its ability to co-aggregate with ubiquitinated substrates is strongly induced by post-translational modifications (PTMs).  The transcription of p62 is also markedly increased during conditions in which selective autophagy substrates accumulate.  For other autophagy receptors, the LC3-interacting region (LIR) motif is regulated by PTMs, inhibiting or stimulating the interaction with ATG8 family proteins.  ATG8 proteins are also regulated by PTMs.  Regulation of the capacity of the core autophagy machinery also affects selective autophagy.  Importantly, autophagy receptors can induce local recruitment and activation of ULK1/2 and PI3KC3 complexes at the site of cargo sequestration.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSothEGF92qRPxQvRxYU81tfW6udTcc2eb2YdCnyVbUsLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1MzgvVSktQ%253D%253D&md5=024cf98b5e452cb2dde8c4516cffcf7a</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1042%2FEBC20170035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FEBC20170035%26sid%3Dliteratum%253Aachs%26aulast%3DLamark%26aufirst%3DT.%26aulast%3DSvenning%26aufirst%3DS.%26aulast%3DJohansen%26aufirst%3DT.%26atitle%3DRegulation%2520of%2520selective%2520autophagy%253A%2520the%2520p62%252FSQSTM1%2520paradigm%26jtitle%3DEssays%2520Biochem.%26date%3D2017%26volume%3D61%26spage%3D609%26epage%3D624%26doi%3D10.1042%2FEBC20170035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stolz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ernst, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dikic, I.</span></span> <span> </span><span class="NLM_article-title">Cargo recognition and trafficking in selective autophagy</span>. <i>Nat. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">495</span>– <span class="NLM_lpage">501</span>, <span class="refDoi"> DOI: 10.1038/ncb2979</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1038%2Fncb2979" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=24875736" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC2cXovVaktrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2014&pages=495-501&author=A.+Stolzauthor=A.+Ernstauthor=I.+Dikic&title=Cargo+recognition+and+trafficking+in+selective+autophagy&doi=10.1038%2Fncb2979"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Cargo recognition and trafficking in selective autophagy</span></div><div class="casAuthors">Stolz, Alexandra; Ernst, Andreas; Dikic, Ivan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Cell Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">495-501</span>CODEN:
                <span class="NLM_cas:coden">NCBIFN</span>;
        ISSN:<span class="NLM_cas:issn">1465-7392</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Selective autophagy is a quality control pathway through which cellular components are sequestered into double-membrane vesicles and delivered to specific intracellular compartments.  This process requires autophagy receptors that link cargo to growing autophagosomal membranes.  Selective autophagy is also implicated in various membrane trafficking events.  Here we discuss the current view on how cargo selection and transport are achieved during selective autophagy, and point out mol. mechanisms that are congruent between autophagy and vesicle trafficking pathways.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoP4vnK9jGLP7Vg90H21EOLACvtfcHk0lj_S9e8Do5PXQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXovVaktrg%253D&md5=3f58139fdf955faabff12cbe94ad865c</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1038%2Fncb2979&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncb2979%26sid%3Dliteratum%253Aachs%26aulast%3DStolz%26aufirst%3DA.%26aulast%3DErnst%26aufirst%3DA.%26aulast%3DDikic%26aufirst%3DI.%26atitle%3DCargo%2520recognition%2520and%2520trafficking%2520in%2520selective%2520autophagy%26jtitle%3DNat.%2520Cell%2520Biol.%26date%3D2014%26volume%3D16%26spage%3D495%26epage%3D501%26doi%3D10.1038%2Fncb2979" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bonam, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruff, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, S.</span></span> <span> </span><span class="NLM_article-title">HSPA8/HSC70 in immune disorders: a molecular rheostat that adjusts chaperone-mediated autophagy substrates</span>. <i>Cells</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">849</span>, <span class="refDoi"> DOI: 10.3390/cells8080849</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.3390%2Fcells8080849" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BB3cXlsVGjsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2019&pages=849&author=S.+R.+Bonamauthor=M.+Ruffauthor=S.+Muller&title=HSPA8%2FHSC70+in+immune+disorders%3A+a+molecular+rheostat+that+adjusts+chaperone-mediated+autophagy+substrates&doi=10.3390%2Fcells8080849"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">HSPA8/HSC70 in immune disorders: a molecular rheostat that adjusts chaperone-mediated autophagy substrates</span></div><div class="casAuthors">Bonam, Srinivasa Reddy; Ruff, Marc; Muller, Sylviane</div><div class="citationInfo"><span class="NLM_cas:title">Cells</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">849</span>CODEN:
                <span class="NLM_cas:coden">CELLC6</span>;
        ISSN:<span class="NLM_cas:issn">2073-4409</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">HSPA8/HSC70 is a mol. chaperone involved in a wide variety of cellular processes.  It plays a crucial role in protein quality control, ensuring the correct folding and re-folding of selected proteins, and controlling the elimination of abnormally-folded conformers and of proteins daily produced in excess in our cells.  HSPA8 is a crucial mol. regulator of chaperone-mediated autophagy, as a detector of substrates that will be processed by this specialized autophagy pathway.  In this review, we shortly summarize its structure and overall functions, dissect its implication in immune disorders, and list the known pharmacol. tools that modulate its functions.  We also exemplify the interest of targeting HSPA8 to regulate pathol. immune dysfunctions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUjQS9gBwqhbVg90H21EOLACvtfcHk0lj_S9e8Do5PXQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXlsVGjsbc%253D&md5=4053930b5aa4d2ecb63344af4c59e479</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.3390%2Fcells8080849&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fcells8080849%26sid%3Dliteratum%253Aachs%26aulast%3DBonam%26aufirst%3DS.%2BR.%26aulast%3DRuff%26aufirst%3DM.%26aulast%3DMuller%26aufirst%3DS.%26atitle%3DHSPA8%252FHSC70%2520in%2520immune%2520disorders%253A%2520a%2520molecular%2520rheostat%2520that%2520adjusts%2520chaperone-mediated%2520autophagy%2520substrates%26jtitle%3DCells%26date%3D2019%26volume%3D8%26spage%3D849%26doi%3D10.3390%2Fcells8080849" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dikic, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elazar, Z.</span></span> <span> </span><span class="NLM_article-title">Mechanism and medical implications of mammalian autophagy</span>. <i>Nat. Rev. Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">349</span>– <span class="NLM_lpage">364</span>, <span class="refDoi"> DOI: 10.1038/s41580-018-0003-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1038%2Fs41580-018-0003-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=29618831" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC1cXosVCns7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2018&pages=349-364&author=I.+Dikicauthor=Z.+Elazar&title=Mechanism+and+medical+implications+of+mammalian+autophagy&doi=10.1038%2Fs41580-018-0003-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanism and medical implications of mammalian autophagy</span></div><div class="casAuthors">Dikic, Ivan; Elazar, Zvulun</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">349-364</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review.  Autophagy is a highly conserved catabolic process induced under various conditions of cellular stress, which prevents cell damage and promotes survival in the event of energy or nutrient shortage and responds to various cytotoxic insults.  Thus, autophagy has primarily cytoprotective functions and needs to be tightly regulated to respond correctly to the different stimuli that cells experience, thereby conferring adaptation to the ever-changing environment.  It is now apparent that autophagy is deregulated in the context of various human pathologies, including cancer and neurodegeneration, and its modulation has considerable potential as a therapeutic approach.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHijrOpvofErVg90H21EOLACvtfcHk0liTXlGFYNd7CA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXosVCns7s%253D&md5=7cbbf53eca81386639df68c0f4a42339</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1038%2Fs41580-018-0003-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41580-018-0003-4%26sid%3Dliteratum%253Aachs%26aulast%3DDikic%26aufirst%3DI.%26aulast%3DElazar%26aufirst%3DZ.%26atitle%3DMechanism%2520and%2520medical%2520implications%2520of%2520mammalian%2520autophagy%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2018%26volume%3D19%26spage%3D349%26epage%3D364%26doi%3D10.1038%2Fs41580-018-0003-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mizushima, N.</span></span> <span> </span><span class="NLM_article-title">Autophagy and human diseases</span>. <i>Cell Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">69</span>– <span class="NLM_lpage">79</span>, <span class="refDoi"> DOI: 10.1038/cr.2013.161</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1038%2Fcr.2013.161" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=24323045" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFCru7fK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2014&pages=69-79&author=P.+Jiangauthor=N.+Mizushima&title=Autophagy+and+human+diseases&doi=10.1038%2Fcr.2013.161"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Autophagy and human diseases</span></div><div class="casAuthors">Jiang, Peidu; Mizushima, Noboru</div><div class="citationInfo"><span class="NLM_cas:title">Cell Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">69-79</span>CODEN:
                <span class="NLM_cas:coden">CREEB6</span>;
        ISSN:<span class="NLM_cas:issn">1001-0602</span>.
    
            (<span class="NLM_cas:orgname">NPG Nature Asia-Pacific</span>)
        </div><div class="casAbstract">A review.  Autophagy is a major intracellular degradative process that delivers cytoplasmic materials to the lysosome for degrdn.  Since the discovery of autophagy-related (Atg) genes in the 1990s, there has been a proliferation of studies on the physiol. and pathol. roles of autophagy in a variety of autophagy knockout models.  However, direct evidence of the connections between ATG gene dysfunction and human diseases has emerged only recently.  There are an increasing no. of reports showing that mutations in the ATG genes were identified in various human diseases such as neurodegenerative diseases, infectious diseases, and cancers.  Here, we review the major advances in identification of mutations or polymorphisms of the ATG genes in human diseases.  Current autophagy-modulating compds. in clin. trials are also summarized.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoTFjg9nyx11bVg90H21EOLACvtfcHk0liTXlGFYNd7CA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFCru7fK&md5=4155f89f1cc9957e607a2955299fc5da</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1038%2Fcr.2013.161&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fcr.2013.161%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DP.%26aulast%3DMizushima%26aufirst%3DN.%26atitle%3DAutophagy%2520and%2520human%2520diseases%26jtitle%3DCell%2520Res.%26date%3D2014%26volume%3D24%26spage%3D69%26epage%3D79%26doi%3D10.1038%2Fcr.2013.161" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shacka, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span> <span> </span><span class="NLM_article-title">The autophagy-lysosomal degradation pathway: role in neurodegenerative disease and therapy</span>. <i>Front. Biosci., Landmark Ed.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">718</span>– <span class="NLM_lpage">736</span>, <span class="refDoi"> DOI: 10.2741/2714</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.2741%2F2714" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=17981582" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhs1Cht7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2008&pages=718-736&author=J.+J.+Shackaauthor=K.+A.+Rothauthor=J.+Zhang&title=The+autophagy-lysosomal+degradation+pathway%3A+role+in+neurodegenerative+disease+and+therapy&doi=10.2741%2F2714"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">The autophagy-lysosomal degradation pathway: role in neurodegenerative disease and therapy</span></div><div class="casAuthors">Shacka, John J.; Roth, Kevin A.; Zhang, Jianhua</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in Bioscience</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">718-736</span>CODEN:
                <span class="NLM_cas:coden">FRBIF6</span>;
        ISSN:<span class="NLM_cas:issn">1093-4715</span>.
    
            (<span class="NLM_cas:orgname">Frontiers in Bioscience</span>)
        </div><div class="casAbstract">A review.  Alterations in the autophagy-lysosomal degrdn. pathway have been described in normal brain aging and in age-related neurodegenerative diseases including Alzheimer's (AD) and Parkinson's (PD).  An improper clearance of proteins in AD and PD may result either from a compromise in the autophagy-lysosomal degrdn. pathway or induce alterations in this pathway, and may result in neuron dysfunction and neuron loss.  This review provides an overview of AD and PD with a specific focus on macroautophagy, chaperone-mediated autophagy and lysosome function in human and exptl. models of AD and PD.  Potential therapies for AD and PD are also discussed that may promote survival by regulating the autophagy and lysosomal degrdn. pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo332V8qqFPC7Vg90H21EOLACvtfcHk0liTXlGFYNd7CA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhs1Cht7c%253D&md5=65657dccac0dca90f5eaaddf8530e54d</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.2741%2F2714&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2741%252F2714%26sid%3Dliteratum%253Aachs%26aulast%3DShacka%26aufirst%3DJ.%2BJ.%26aulast%3DRoth%26aufirst%3DK.%2BA.%26aulast%3DZhang%26aufirst%3DJ.%26atitle%3DThe%2520autophagy-lysosomal%2520degradation%2520pathway%253A%2520role%2520in%2520neurodegenerative%2520disease%2520and%2520therapy%26jtitle%3DFront.%2520Biosci.%252C%2520Landmark%2520Ed.%26date%3D2008%26volume%3D13%26spage%3D718%26epage%3D736%26doi%3D10.2741%2F2714" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Banik, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pedram, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wisnovsky, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahn, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riley, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertozzi, C. R.</span></span> <span> </span><span class="NLM_article-title">Lysosome-targeting chimaeras for degradation of extracellular proteins</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>584</i></span>,  <span class="NLM_fpage">291</span>– <span class="NLM_lpage">297</span>, <span class="refDoi"> DOI: 10.1038/s41586-020-2545-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1038%2Fs41586-020-2545-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=32728216" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhsVygtLrP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=584&publication_year=2020&pages=291-297&author=S.+M.+Banikauthor=K.+Pedramauthor=S.+Wisnovskyauthor=G.+Ahnauthor=N.+M.+Rileyauthor=C.+R.+Bertozzi&title=Lysosome-targeting+chimaeras+for+degradation+of+extracellular+proteins&doi=10.1038%2Fs41586-020-2545-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Lysosome-targeting chimaeras for degradation of extracellular proteins</span></div><div class="casAuthors">Banik, Steven M.; Pedram, Kayvon; Wisnovsky, Simon; Ahn, Green; Riley, Nicholas M.; Bertozzi, Carolyn R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">584</span>
        (<span class="NLM_cas:issue">7820</span>),
    <span class="NLM_cas:pages">291-297</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">The majority of therapies that target individual proteins rely on specific activity-modulating interactions with the target protein-for example, enzyme inhibition or ligand blocking.  However, several major classes of therapeutically relevant proteins have unknown or inaccessible activity profiles and so cannot be targeted by such strategies.  Protein-degrdn. platforms such as proteolysis-targeting chimeras (PROTACs) and others (for example, dTAGs, Trim-Away, chaperone-mediated autophagy targeting and SNIPERs) have been developed for proteins that are typically difficult to target; however, these methods involve the manipulation of intracellular protein degrdn. machinery and are therefore fundamentally limited to proteins that contain cytosolic domains to which ligands can bind and recruit the requisite cellular components.  Extracellular and membrane-assocd. proteins-the products of 40% of all protein-encoding genes-are key agents in cancer, ageing-related diseases and autoimmune disorders, and so a general strategy to selectively degrade these proteins has the potential to improve human health.  Here we establish the targeted degrdn. of extracellular and membrane-assocd. proteins using conjugates that bind both a cell-surface lysosome-shuttling receptor and the extracellular domain of a target protein.  These initial lysosome-targeting chimeras, which we term LYTACs, consist of a small mol. or antibody fused to chem. synthesized glycopeptide ligands that are agonists of the cation-independent mannose-6-phosphate receptor (CI-M6PR).  We use LYTACs to develop a CRISPR interference screen that reveals the biochem. pathway for CI-M6PR-mediated cargo internalization in cell lines, and uncover the exocyst complex as a previously unidentified-but essential-component of this pathway.  We demonstrate the scope of this platform through the degrdn. of therapeutically relevant proteins, including apolipoprotein E4, epidermal growth factor receptor, CD71 and programmed death-ligand 1.  Our results establish a modular strategy for directing secreted and membrane proteins for lysosomal degrdn., with broad implications for biochem. research and for therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmZfj1oJhbx7Vg90H21EOLACvtfcHk0lj4DWpvQAfCVQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhsVygtLrP&md5=1109c68b56a3ee93427ae89f194c0a16</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1038%2Fs41586-020-2545-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-020-2545-9%26sid%3Dliteratum%253Aachs%26aulast%3DBanik%26aufirst%3DS.%2BM.%26aulast%3DPedram%26aufirst%3DK.%26aulast%3DWisnovsky%26aufirst%3DS.%26aulast%3DAhn%26aufirst%3DG.%26aulast%3DRiley%26aufirst%3DN.%2BM.%26aulast%3DBertozzi%26aufirst%3DC.%2BR.%26atitle%3DLysosome-targeting%2520chimaeras%2520for%2520degradation%2520of%2520extracellular%2520proteins%26jtitle%3DNature%26date%3D2020%26volume%3D584%26spage%3D291%26epage%3D297%26doi%3D10.1038%2Fs41586-020-2545-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ahn, G.</span>; <span class="NLM_string-name">Banik, S.</span>; <span class="NLM_string-name">Miller, C. M.</span>; <span class="NLM_string-name">Riley, N.</span>; <span class="NLM_string-name">Cochran, J. R.</span>; <span class="NLM_string-name">Bertozzi, C.</span></span> <span> </span><span class="NLM_article-title">Lysosome targeting chimeras (LYTACs) that engage a liver-specific asialoglycoprotein receptor for targeted protein degradation</span>.  <i>ChemRxiv</i> <span class="NLM_year" style="font-weight: bold;">2020</span>, <span class="refDoi"> DOI: 10.26434/chemrxiv.12736778.v1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.26434%2Fchemrxiv.12736778.v1" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2020&author=G.+Ahn&author=S.+Banik&author=C.+M.+Miller&author=N.+Riley&author=J.+R.+Cochran&author=C.+Bertozzi&title=Lysosome+targeting+chimeras+%28LYTACs%29+that+engage+a+liver-specific+asialoglycoprotein+receptor+for+targeted+protein+degradation&doi=10.26434%2Fchemrxiv.12736778.v1"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.26434%2Fchemrxiv.12736778.v1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.26434%252Fchemrxiv.12736778.v1%26sid%3Dliteratum%253Aachs%26aulast%3DAhn%26aufirst%3DG.%26atitle%3DLysosome%2520targeting%2520chimeras%2520%2528LYTACs%2529%2520that%2520engage%2520a%2520liver-specific%2520asialoglycoprotein%2520receptor%2520for%2520targeted%2520protein%2520degradation%26jtitle%3DChemRxiv%26date%3D2020%26doi%3D10.26434%2Fchemrxiv.12736778.v1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Takahashi, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriyama, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakamura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miki, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takahashi, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sato, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akaike, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Itto-Nakama, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arimoto, H.</span></span> <span> </span><span class="NLM_article-title">AUTACs: cargo-specific degraders using selective autophagy</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>76</i></span>,  <span class="NLM_fpage">797</span>– <span class="NLM_lpage">810</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2019.09.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1016%2Fj.molcel.2019.09.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=31606272" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhvFKjsr%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=76&publication_year=2019&pages=797-810&author=D.+Takahashiauthor=J.+Moriyamaauthor=T.+Nakamuraauthor=E.+Mikiauthor=E.+Takahashiauthor=A.+Satoauthor=T.+Akaikeauthor=K.+Itto-Nakamaauthor=H.+Arimoto&title=AUTACs%3A+cargo-specific+degraders+using+selective+autophagy&doi=10.1016%2Fj.molcel.2019.09.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">AUTACs: Cargo-Specific Degraders Using Selective Autophagy</span></div><div class="casAuthors">Takahashi, Daiki; Moriyama, Jun; Nakamura, Tomoe; Miki, Erika; Takahashi, Eriko; Sato, Ayami; Akaike, Takaaki; Itto-Nakama, Kaori; Arimoto, Hirokazu</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">76</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">797-810.e10</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Protein silencing represents an essential tool in biomedical research.  Targeted protein degrdn. (TPD) strategies exemplified by PROTACs are rapidly emerging as modalities in drug discovery.  However, the scope of current TPD techniques is limited because many intracellular materials are not substrates of proteasomal clearance.  Here, we described a novel targeted-clearance strategy (autophagy-targeting chimera [AUTAC]) that contains a degrdn. tag (guanine derivs.) and a warhead to provide target specificity.  As expected from the substrate scope of autophagy, AUTAC degraded fragmented mitochondria as well as proteins.  Mitochondria-targeted AUTAC accelerated both the removal of dysfunctional fragmented mitochondria and the biogenesis of functionally normal mitochondria in patient-derived fibroblast cells.  Cytoprotective effects against acute mitochondrial injuries were also seen.  Canonical autophagy is viewed as a nonselective bulk decompn. system, and none of the available autophagy-inducing agents exhibit useful cargo selectivity.  With its target specificity, AUTAC provides a new modality for research on autophagy-based drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqEB3iYVVOYJbVg90H21EOLACvtfcHk0lj4DWpvQAfCVQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhvFKjsr%252FE&md5=2eb738f4a51d83e7aee6d69e47bf2e00</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2019.09.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2019.09.009%26sid%3Dliteratum%253Aachs%26aulast%3DTakahashi%26aufirst%3DD.%26aulast%3DMoriyama%26aufirst%3DJ.%26aulast%3DNakamura%26aufirst%3DT.%26aulast%3DMiki%26aufirst%3DE.%26aulast%3DTakahashi%26aufirst%3DE.%26aulast%3DSato%26aufirst%3DA.%26aulast%3DAkaike%26aufirst%3DT.%26aulast%3DItto-Nakama%26aufirst%3DK.%26aulast%3DArimoto%26aufirst%3DH.%26atitle%3DAUTACs%253A%2520cargo-specific%2520degraders%2520using%2520selective%2520autophagy%26jtitle%3DMol.%2520Cell%26date%3D2019%26volume%3D76%26spage%3D797%26epage%3D810%26doi%3D10.1016%2Fj.molcel.2019.09.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sha, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Difiglia, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mei, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fei, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, B.</span></span> <span> </span><span class="NLM_article-title">Allele-selective lowering of mutant HTT protein by HTT-LC3 linker compounds</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>575</i></span>,  <span class="NLM_fpage">203</span>– <span class="NLM_lpage">209</span>, <span class="refDoi"> DOI: 10.1038/s41586-019-1722-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1038%2Fs41586-019-1722-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=31666698" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitV2hsb3O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=575&publication_year=2019&pages=203-209&author=Z.+Liauthor=C.+Wangauthor=Z.+Wangauthor=C.+Zhuauthor=J.+Liauthor=T.+Shaauthor=L.+Maauthor=C.+Gaoauthor=Y.+Yangauthor=Y.+Sunauthor=J.+Wangauthor=X.+Sunauthor=C.+Luauthor=M.+Difigliaauthor=Y.+Meiauthor=C.+Dingauthor=S.+Luoauthor=Y.+Dangauthor=Y.+Dingauthor=Y.+Feiauthor=B.+Lu&title=Allele-selective+lowering+of+mutant+HTT+protein+by+HTT-LC3+linker+compounds&doi=10.1038%2Fs41586-019-1722-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Allele-selective lowering of mutant HTT protein by HTT-LC3 linker compounds</span></div><div class="casAuthors">Li, Zhaoyang; Wang, Cen; Wang, Ziying; Zhu, Chenggang; Li, Jie; Sha, Tian; Ma, Lixiang; Gao, Chao; Yang, Yi; Sun, Yimin; Wang, Jian; Sun, Xiaoli; Lu, Chenqi; Difiglia, Marian; Mei, Yanai; Ding, Chen; Luo, Shouqing; Dang, Yongjun; Ding, Yu; Fei, Yiyan; Lu, Boxun</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">575</span>
        (<span class="NLM_cas:issue">7781</span>),
    <span class="NLM_cas:pages">203-209</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Accumulation of mutant proteins is a major cause of many diseases (collectively called proteopathies), and lowering the level of these proteins can be useful for treatment of these diseases.  We hypothesized that compds. that interact with both the autophagosome protein microtubule-assocd. protein 1A/1B light chain 3 (LC3)1 and the disease-causing protein may target the latter for autophagic clearance.  Mutant huntingtin protein (mHTT) contains an expanded polyglutamine (polyQ) tract and causes Huntington's disease, an incurable neurodegenerative disorder2.  Here, using small-mol.-microarray-based screening, we identified four compds. that interact with both LC3 and mHTT, but not with the wild-type HTT protein.  Some of these compds. targeted mHTT to autophagosomes, reduced mHTT levels in an allele-selective manner, and rescued disease-relevant phenotypes in cells and in vivo in fly and mouse models of Huntington's disease.  We further show that these compds. interact with the expanded polyQ stretch and could lower the level of mutant ataxin-3 (ATXN3), another disease-causing protein with an expanded polyQ tract3.  This study presents candidate compds. for lowering mHTT and potentially other disease-causing proteins with polyQ expansions, demonstrating the concept of lowering levels of disease-causing proteins using autophagosome-tethering compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7rSfxy_Wl7rVg90H21EOLACvtfcHk0lj4DWpvQAfCVQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitV2hsb3O&md5=98cba22501f8157c83c0181b74ca15e2</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1038%2Fs41586-019-1722-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-019-1722-1%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DZhu%26aufirst%3DC.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DSha%26aufirst%3DT.%26aulast%3DMa%26aufirst%3DL.%26aulast%3DGao%26aufirst%3DC.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DSun%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DSun%26aufirst%3DX.%26aulast%3DLu%26aufirst%3DC.%26aulast%3DDifiglia%26aufirst%3DM.%26aulast%3DMei%26aufirst%3DY.%26aulast%3DDing%26aufirst%3DC.%26aulast%3DLuo%26aufirst%3DS.%26aulast%3DDang%26aufirst%3DY.%26aulast%3DDing%26aufirst%3DY.%26aulast%3DFei%26aufirst%3DY.%26aulast%3DLu%26aufirst%3DB.%26atitle%3DAllele-selective%2520lowering%2520of%2520mutant%2520HTT%2520protein%2520by%2520HTT-LC3%2520linker%2520compounds%26jtitle%3DNature%26date%3D2019%26volume%3D575%26spage%3D203%26epage%3D209%26doi%3D10.1038%2Fs41586-019-1722-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bauer, P. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goswami, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, H. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okuno, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kurosawa, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamada, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miyazaki, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsumoto, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kino, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nukina, N.</span></span> <span> </span><span class="NLM_article-title">Harnessing chaperone-mediated autophagy for the selective degradation of mutant huntingtin protein</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">256</span>– <span class="NLM_lpage">263</span>, <span class="refDoi"> DOI: 10.1038/nbt.1608</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1038%2Fnbt.1608" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=20190739" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC3cXisFeksb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2010&pages=256-263&author=P.+O.+Bauerauthor=A.+Goswamiauthor=H.+K.+Wongauthor=M.+Okunoauthor=M.+Kurosawaauthor=M.+Yamadaauthor=H.+Miyazakiauthor=G.+Matsumotoauthor=Y.+Kinoauthor=Y.+Nagaiauthor=N.+Nukina&title=Harnessing+chaperone-mediated+autophagy+for+the+selective+degradation+of+mutant+huntingtin+protein&doi=10.1038%2Fnbt.1608"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Harnessing chaperone-mediated autophagy for the selective degradation of mutant huntingtin protein</span></div><div class="casAuthors">Bauer, Peter O.; Goswami, Anand; Wong, Hon Kit; Okuno, Misako; Kurosawa, Masaru; Yamada, Mizuki; Miyazaki, Haruko; Matsumoto, Gen; Kino, Yoshihiro; Nagai, Yoshitaka; Nukina, Nobuyuki</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">256-263</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Huntington's Disease (HD) is a dominantly inherited pathol. caused by the accumulation of mutant huntingtin protein (HTT) contg. an expanded polyglutamine (polyQ) tract.  As the polyglutamine binding peptide 1 (QBP1) is known to bind an expanded polyQ tract but not the polyQ motif found in normal HTT, we selectively targeted mutant HTT for degrdn. by expressing a fusion mol. comprising two copies of QBP1 and copies of two different heat shock cognate protein 70 (HSC70)-binding motifs in cellular and mouse models of HD.  Chaperone-mediated autophagy contributed to the specific degrdn. of mutant HTT in cultured cells expressing the construct.  Intrastriatal delivery of a virus expressing the fusion mol. ameliorated the disease phenotype in the R6/2 mouse model of HD.  Similar adaptor mols. comprising HSC70-binding motifs fused to an appropriate structure-specific binding agent(s) may have therapeutic potential for treating diseases caused by misfolded proteins other than those with expanded polyQ tracts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoOHevKvAH5LrVg90H21EOLACvtfcHk0lh4cognGAjeVg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXisFeksb8%253D&md5=57caf1a71777ff31ab423541db69ad75</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1038%2Fnbt.1608&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt.1608%26sid%3Dliteratum%253Aachs%26aulast%3DBauer%26aufirst%3DP.%2BO.%26aulast%3DGoswami%26aufirst%3DA.%26aulast%3DWong%26aufirst%3DH.%2BK.%26aulast%3DOkuno%26aufirst%3DM.%26aulast%3DKurosawa%26aufirst%3DM.%26aulast%3DYamada%26aufirst%3DM.%26aulast%3DMiyazaki%26aufirst%3DH.%26aulast%3DMatsumoto%26aufirst%3DG.%26aulast%3DKino%26aufirst%3DY.%26aulast%3DNagai%26aufirst%3DY.%26aulast%3DNukina%26aufirst%3DN.%26atitle%3DHarnessing%2520chaperone-mediated%2520autophagy%2520for%2520the%2520selective%2520degradation%2520of%2520mutant%2520huntingtin%2520protein%26jtitle%3DNat.%2520Biotechnol.%26date%3D2010%26volume%3D28%26spage%3D256%26epage%3D263%26doi%3D10.1038%2Fnbt.1608" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fan, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, W. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y. T.</span></span> <span> </span><span class="NLM_article-title">Rapid and reversible knockdown of endogenous proteins by peptide-directed lysosomal degradation</span>. <i>Nat. Neurosci.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">471</span>– <span class="NLM_lpage">480</span>, <span class="refDoi"> DOI: 10.1038/nn.3637</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1038%2Fnn.3637" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=24464042" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht1ynsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2014&pages=471-480&author=X.+Fanauthor=W.+Y.+Jinauthor=J.+Luauthor=J.+Wangauthor=Y.+T.+Wang&title=Rapid+and+reversible+knockdown+of+endogenous+proteins+by+peptide-directed+lysosomal+degradation&doi=10.1038%2Fnn.3637"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Rapid and reversible knockdown of endogenous proteins by peptide-directed lysosomal degradation</span></div><div class="casAuthors">Fan, Xuelai; Jin, Wu Yang; Lu, Jie; Wang, Jin; Wang, Yu Tian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Neuroscience</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">471-480</span>CODEN:
                <span class="NLM_cas:coden">NANEFN</span>;
        ISSN:<span class="NLM_cas:issn">1097-6256</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Rapid and reversible methods for altering the levels of endogenous proteins are critically important for studying biol. systems and developing therapeutics.  Here we describe a membrane-permeant targeting peptide-based method that rapidly and reversibly knocks down endogenous proteins through chaperone-mediated autophagy in vitro and in vivo.  We demonstrate the specificity, efficacy and generalizability of the method by showing efficient knockdown of various proteins, including death assocd. protein kinase 1 (160 kDa), scaffolding protein PSD-95 (95 kDa) and α-synuclein (18 kDa), with their resp. targeting peptides in a dose-, time- and lysosomal activity-dependent manner in rat neuronal cultures.  Moreover, we show that, when given systemically, the peptide system efficiently knocked down the targeted protein in the brains of intact rats.  Our study provides a robust and convenient research tool for manipulating endogenous protein levels and may also lead to the development of protein knockdown-based therapeutics for treating human diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpVBXNfrBcGa7Vg90H21EOLACvtfcHk0ljotYdaxqgniw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht1ynsr8%253D&md5=6891b206bf70a86dbbf410c8e08346bd</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1038%2Fnn.3637&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnn.3637%26sid%3Dliteratum%253Aachs%26aulast%3DFan%26aufirst%3DX.%26aulast%3DJin%26aufirst%3DW.%2BY.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DY.%2BT.%26atitle%3DRapid%2520and%2520reversible%2520knockdown%2520of%2520endogenous%2520proteins%2520by%2520peptide-directed%2520lysosomal%2520degradation%26jtitle%3DNat.%2520Neurosci.%26date%3D2014%26volume%3D17%26spage%3D471%26epage%3D480%26doi%3D10.1038%2Fnn.3637" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref64"><div class="reference"><strong class="refLabel"><a href="#ref64" class="rightTabRefNumLink">64</a></strong><div class="NLM_citation" id="rightTab-cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Y. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chi, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, L. Q.</span></span> <span> </span><span class="NLM_article-title">The Peptide-directed lysosomal degradation of CDK5 exerts therapeutic effects against stroke</span>. <i>Aging Dis.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1140</span>– <span class="NLM_lpage">1145</span>, <span class="refDoi"> DOI: 10.14336/AD.2018.1225</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.14336%2FAD.2018.1225" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=31595208" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A280%3ADC%252BB3MnmtFCktQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=1140-1145&author=Y.+F.+Zhouauthor=J.+Wangauthor=M.+F.+Dengauthor=B.+Chiauthor=N.+Weiauthor=J.+G.+Chenauthor=D.+Liuauthor=X.+Yinauthor=Y.+Luauthor=L.+Q.+Zhu&title=The+Peptide-directed+lysosomal+degradation+of+CDK5+exerts+therapeutic+effects+against+stroke&doi=10.14336%2FAD.2018.1225"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">The Peptide-Directed Lysosomal Degradation of CDK5 Exerts Therapeutic Effects against Stroke</span></div><div class="casAuthors">Zhou Ya-Fan; Deng Man-Fei; Liu Dan; Zhu Ling-Qiang; Wang Jing; Chi Bin; Wei Na; Chen Jian-Guo; Chen Jian-Guo; Liu Dan; Lu Youming; Zhu Ling-Qiang; Yin Xiaoping</div><div class="citationInfo"><span class="NLM_cas:title">Aging and disease</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1140-1145</span>
        ISSN:<span class="NLM_cas:issn">2152-5250</span>.
    </div><div class="casAbstract">The aberrant activation of CDK5 has been implicated in neuronal death in stroke.  The goal of this study is to determine whether knocking down CDK5 by a peptide-directed lysosomal degradation approach is therapeutically effective against stroke.  We synthesized a membrane-permeable peptide that specifically binds to CDK5 with a chaperone-mediated autophagy targeting motif (Tat-CDK5-CTM) and tested its therapeutic effects on a mouse model of ischemic stroke.  Our results showed that Tat-CDK5-CTM blocked the CDK5-NR2B interaction, resulting in the degradation of CDK5, which in turn prevented calcium overload and neuronal death in cultured neurons.  Tat-CDK5-CTM also reduced the infarction area and neuronal loss and improved the neurological functions in MCAO (Middle cerebral artery occlusion) mice.  The peptide-directed lysosomal degradation of CDK5 is a promising therapeutic intervention for stroke.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQ9EnaSakFg7AxyojvcjibhfW6udTcc2eY-jk1qjQ6QbLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3MnmtFCktQ%253D%253D&md5=64be3cec620c41c197ad7a45c8f8234e</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.14336%2FAD.2018.1225&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.14336%252FAD.2018.1225%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DY.%2BF.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DDeng%26aufirst%3DM.%2BF.%26aulast%3DChi%26aufirst%3DB.%26aulast%3DWei%26aufirst%3DN.%26aulast%3DChen%26aufirst%3DJ.%2BG.%26aulast%3DLiu%26aufirst%3DD.%26aulast%3DYin%26aufirst%3DX.%26aulast%3DLu%26aufirst%3DY.%26aulast%3DZhu%26aufirst%3DL.%2BQ.%26atitle%3DThe%2520Peptide-directed%2520lysosomal%2520degradation%2520of%2520CDK5%2520exerts%2520therapeutic%2520effects%2520against%2520stroke%26jtitle%3DAging%2520Dis.%26date%3D2019%26volume%3D10%26spage%3D1140%26epage%3D1145%26doi%3D10.14336%2FAD.2018.1225" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref65"><div class="reference"><strong class="refLabel"><a href="#ref65" class="rightTabRefNumLink">65</a></strong><div class="NLM_citation" id="rightTab-cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, J.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J.</span></span> <span> </span><span class="NLM_article-title">HIP1R targets PD-L1 to lysosomal degradation to alter T cell-mediated cytotoxicity</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">42</span>– <span class="NLM_lpage">50</span>, <span class="refDoi"> DOI: 10.1038/s41589-018-0161-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1038%2Fs41589-018-0161-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=30397328" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitV2qtbbN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2019&pages=42-50&author=H.+Wangauthor=H.+Yaoauthor=C.+Liauthor=H.+Shiauthor=J.+Lanauthor=Z.+Liauthor=Y.+Zhangauthor=L.+Liangauthor=J.-Y.+Fangauthor=J.+Xu&title=HIP1R+targets+PD-L1+to+lysosomal+degradation+to+alter+T+cell-mediated+cytotoxicity&doi=10.1038%2Fs41589-018-0161-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">HIP1R targets PD-L1 to lysosomal degradation to alter T cell-mediated cytotoxicity</span></div><div class="casAuthors">Wang, Huanbin; Yao, Han; Li, Chushu; Shi, Hubing; Lan, Jiang; Li, Zhaoli; Zhang, Yao; Liang, Lunxi; Fang, Jing-Yuan; Xu, Jie</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">42-50</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Expression of programmed cell death 1 (PD-1) ligand 1 (PD-L1) protects tumor cells from T cell-mediated immune surveillance, and immune checkpoint blockade (ICB) therapies targeting PD-1 and PD-L1 have exhibited significant clin. benefits.  However, the relatively low response rate and obsd. ICB resistance highlight the need to understand the mol. regulation of PD-L1.  Here we show that HIP1R targets PD-L1 to lysosomal degrdn. to alter T cell-mediated cytotoxicity.  HIP1R phys. interacts with PD-L1 and delivers PD-L1 to the lysosome through a lysosomal targeting signal.  Depletion of HIP1R in tumor cells caused PD-L1 accumulation and suppressed T cell-mediated cytotoxicity.  A rationally designed peptide (PD-LYSO) incorporating the lysosome-sorting signal and the PD-L1-binding sequence of HIP1R successfully depleted PD-L1 expression in tumor cells.  Our results identify the mol. machineries governing the lysosomal degrdn. of PD-L1 and exemplify the development of a chimeric peptide for targeted degrdn. of PD-L1 as a crucial anticancer target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreVHjIVSWPmLVg90H21EOLACvtfcHk0lhi7lQ4bgty0w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitV2qtbbN&md5=eac9da24ac6ff6e17cf10cfc6ee2c08a</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1038%2Fs41589-018-0161-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41589-018-0161-x%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DH.%26aulast%3DYao%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DShi%26aufirst%3DH.%26aulast%3DLan%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DLiang%26aufirst%3DL.%26aulast%3DFang%26aufirst%3DJ.-Y.%26aulast%3DXu%26aufirst%3DJ.%26atitle%3DHIP1R%2520targets%2520PD-L1%2520to%2520lysosomal%2520degradation%2520to%2520alter%2520T%2520cell-mediated%2520cytotoxicity%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2019%26volume%3D15%26spage%3D42%26epage%3D50%26doi%3D10.1038%2Fs41589-018-0161-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref66"><div class="reference"><strong class="refLabel"><a href="#ref66" class="rightTabRefNumLink">66</a></strong><div class="NLM_citation" id="rightTab-cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pillay, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elliott, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dennison, C.</span></span> <span> </span><span class="NLM_article-title">Endolysosomal proteolysis and its regulation</span>. <i>Biochem. J.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>363</i></span>,  <span class="NLM_fpage">417</span>– <span class="NLM_lpage">429</span>, <span class="refDoi"> DOI: 10.1042/bj3630417</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1042%2Fbj3630417" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=11964142" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BD38XjvFenurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=363&publication_year=2002&pages=417-429&author=C.+S.+Pillayauthor=E.+Elliottauthor=C.+Dennison&title=Endolysosomal+proteolysis+and+its+regulation&doi=10.1042%2Fbj3630417"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">Endolysosomal proteolysis and its regulation</span></div><div class="casAuthors">Pillay, Che S.; Elliott, Edith; Dennison, Clive</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Journal</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">363</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">417-429</span>CODEN:
                <span class="NLM_cas:coden">BIJOAK</span>;
        ISSN:<span class="NLM_cas:issn">0264-6021</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">A review.  The endolysosomal system comprises a unique environment for proteolysis, which is regulated in a manner that apparently does not involve protease inhibitors.  The system comprises a series of membrane-bound intracellular compartments, within which endocytosed material and redundant cellular components are hydrolyzed.  Endocytosed material tends to flow vectorially through the system, proceeding through the early endosome, the endosome carrier vesicle, the late endosome and the lysosome.  Phagocytosis and autophagy provide alternative entry points into the system.  Late endosomes, lysosome/late endosome hybrid organelles, phagosomes and autophagosomes are the principal sites for proteolysis.  In each case, hydrolytic competence is due to components of the endolysosomal system, i.e. proteases, lysosome-assocd. membrane proteins, H+-ATPases and possibly cysteine transporters.  The view is emerging that lysosomes are organelles for the storage of hydrolases, perhaps in an inactivated form.  Once a substrate has entered a proteolytically competent environment, the rate-limiting proteolytic steps are probably effected by cysteine endoproteinases.  As these are affected by pH and possibly redox potential, they may be regulated by the organelle luminal environment.  Regulation is probably also affected, among other factors, by organelle fusion reactions, whereby the meeting of enzyme and substrate may be controlled.  Such systems would permit simultaneous regulation of a no. of unrelated hydrolases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHet05kxwVELVg90H21EOLACvtfcHk0liqmuSEARhrpA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XjvFenurg%253D&md5=41f045e53f3e1bc80fa44020bc950cef</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1042%2Fbj3630417&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252Fbj3630417%26sid%3Dliteratum%253Aachs%26aulast%3DPillay%26aufirst%3DC.%2BS.%26aulast%3DElliott%26aufirst%3DE.%26aulast%3DDennison%26aufirst%3DC.%26atitle%3DEndolysosomal%2520proteolysis%2520and%2520its%2520regulation%26jtitle%3DBiochem.%2520J.%26date%3D2002%26volume%3D363%26spage%3D417%26epage%3D429%26doi%3D10.1042%2Fbj3630417" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref67"><div class="reference"><strong class="refLabel"><a href="#ref67" class="rightTabRefNumLink">67</a></strong><div class="NLM_citation" id="rightTab-cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kundu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, C. B.</span></span> <span> </span><span class="NLM_article-title">Autophagy: basic principles and relevance to disease</span>. <i>Annu. Rev. Pathol.: Mech. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">427</span>– <span class="NLM_lpage">455</span>, <span class="refDoi"> DOI: 10.1146/annurev.pathmechdis.2.010506.091842</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1146%2Fannurev.pathmechdis.2.010506.091842" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=18039129" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BD1cXjsFersLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2008&pages=427-455&author=M.+Kunduauthor=C.+B.+Thompson&title=Autophagy%3A+basic+principles+and+relevance+to+disease&doi=10.1146%2Fannurev.pathmechdis.2.010506.091842"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">Autophagy: basic principles and relevance to disease</span></div><div class="casAuthors">Kundu, Mondira; Thompson, Craig B.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Pathology: Mechanisms of Disease</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">427-455</span>CODEN:
                <span class="NLM_cas:coden">ARPMCU</span>;
        ISSN:<span class="NLM_cas:issn">1553-4006</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review.  Autophagy is a process by which cytoplasmic components are sequestered in double membrane vesicles and degraded upon fusion with lysosomal compartments.  In yeast, autophagy is activated in response to changes in the extracellular milieu.  Depending upon the stimulus, autophagy can degrade cytoplasmic contents nonspecifically or can target the degrdn. of specific cellular components.  Both of these have been adopted in higher eukaryotes and account for the expanding role of autophagy in various cellular processes, as well as contribute to the variation in cellular outcomes after induction of autophagy.  In some cases, autophagy appears to be an adaptive response, whereas under other circumstances it is involved in cell death.  In mammals, autophagy has been implicated in either the pathogenesis or response to a wide variety of diseases, including neurodegenerative disease, chronic bacterial and viral infections, atherosclerosis, and cancer.  As the basic mol. pathways that regulate autophagy are elucidated, the relationship of autophagy to the pathogenesis of various disease states emerges.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGraj2W1uAZS3rVg90H21EOLACvtfcHk0liqmuSEARhrpA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXjsFersLk%253D&md5=57ef3d3e75a3d83a0dfcf9f3d9c9f7a0</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1146%2Fannurev.pathmechdis.2.010506.091842&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.pathmechdis.2.010506.091842%26sid%3Dliteratum%253Aachs%26aulast%3DKundu%26aufirst%3DM.%26aulast%3DThompson%26aufirst%3DC.%2BB.%26atitle%3DAutophagy%253A%2520basic%2520principles%2520and%2520relevance%2520to%2520disease%26jtitle%3DAnnu.%2520Rev.%2520Pathol.%253A%2520Mech.%2520Dis.%26date%3D2008%26volume%3D3%26spage%3D427%26epage%3D455%26doi%3D10.1146%2Fannurev.pathmechdis.2.010506.091842" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref68"><div class="reference"><strong class="refLabel"><a href="#ref68" class="rightTabRefNumLink">68</a></strong><div class="NLM_citation" id="rightTab-cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dahms, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kornfeld, S.</span></span> <span> </span><span class="NLM_article-title">Mannose 6-phosphate receptors: new twists in the tale</span>. <i>Nat. Rev. Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">202</span>– <span class="NLM_lpage">213</span>, <span class="refDoi"> DOI: 10.1038/nrm1050</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1038%2Fnrm1050" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=12612639" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BD3sXhsFajsL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2003&pages=202-213&author=P.+Ghoshauthor=N.+M.+Dahmsauthor=S.+Kornfeld&title=Mannose+6-phosphate+receptors%3A+new+twists+in+the+tale&doi=10.1038%2Fnrm1050"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">Mannose 6-phosphate receptors: new twists in the tale</span></div><div class="casAuthors">Ghosh, Pradipta; Dahms, Nancy M.; Kornfeld, Stuart</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">202-213</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The two mannose 6-phosphate (M6P) receptors were identified because of their ability to bind M6P-contg. sol. acid hydrolases in the Golgi and transport them to the endosomal-lysosomal system.  During the past decade, we have started to understand the structural features of these receptors that allow them to do this job, and how the receptors themselves are sorted as they pass through various membrane-bound compartments.  But trafficking of acid hydrolases is only part of the story.  Evidence is emerging that one of the receptors can regulate cell growth and motility, and that it functions as a tumor suppressor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqKvbnA0U5O-bVg90H21EOLACvtfcHk0liqmuSEARhrpA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXhsFajsL0%253D&md5=c079ad8e76fb1d7ee11c2ca554956961</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1038%2Fnrm1050&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm1050%26sid%3Dliteratum%253Aachs%26aulast%3DGhosh%26aufirst%3DP.%26aulast%3DDahms%26aufirst%3DN.%2BM.%26aulast%3DKornfeld%26aufirst%3DS.%26atitle%3DMannose%25206-phosphate%2520receptors%253A%2520new%2520twists%2520in%2520the%2520tale%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2003%26volume%3D4%26spage%3D202%26epage%3D213%26doi%3D10.1038%2Fnrm1050" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref69"><div class="reference"><strong class="refLabel"><a href="#ref69" class="rightTabRefNumLink">69</a></strong><div class="NLM_citation" id="rightTab-cit69"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Whitworth, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">New class of molecule targets proteins outside cells for degradation</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>584</i></span>,  <span class="NLM_fpage">193</span>– <span class="NLM_lpage">194</span>, <span class="refDoi"> DOI: 10.1038/d41586-020-02211-w</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1038%2Fd41586-020-02211-w" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=32728153" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhsFKlu7zP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=584&publication_year=2020&pages=193-194&author=C.+Whitworthauthor=A.+Ciulli&title=New+class+of+molecule+targets+proteins+outside+cells+for+degradation&doi=10.1038%2Fd41586-020-02211-w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">New class of molecule targets proteins outside cells for degradation</span></div><div class="casAuthors">Whitworth, Claire; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">584</span>
        (<span class="NLM_cas:issue">7820</span>),
    <span class="NLM_cas:pages">193-194</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Mols. have previously been made that induce protein destruction inside cells.  A new class of mol. now induces the degrdn. of membrane and extracellular proteins - opening up avenues for drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2WZkkfZYTdLVg90H21EOLACvtfcHk0liqmuSEARhrpA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhsFKlu7zP&md5=b19b685e8e7fb29fbcdc7b83c6fdf6cb</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1038%2Fd41586-020-02211-w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fd41586-020-02211-w%26sid%3Dliteratum%253Aachs%26aulast%3DWhitworth%26aufirst%3DC.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DNew%2520class%2520of%2520molecule%2520targets%2520proteins%2520outside%2520cells%2520for%2520degradation%26jtitle%3DNature%26date%3D2020%26volume%3D584%26spage%3D193%26epage%3D194%26doi%3D10.1038%2Fd41586-020-02211-w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref70"><div class="reference"><strong class="refLabel"><a href="#ref70" class="rightTabRefNumLink">70</a></strong><div class="NLM_citation" id="rightTab-cit70"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yamazaki, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Painter, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kanekiyo, T.</span></span> <span> </span><span class="NLM_article-title">Apolipoprotein e as a therapeutic target in alzheimer’s disease: a review of basic research and clinical evidence</span>. <i>CNS Drugs</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">773</span>– <span class="NLM_lpage">789</span>, <span class="refDoi"> DOI: 10.1007/s40263-016-0361-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1007%2Fs40263-016-0361-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=27328687" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtVyiurfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2016&pages=773-789&author=Y.+Yamazakiauthor=M.+M.+Painterauthor=G.+Buauthor=T.+Kanekiyo&title=Apolipoprotein+e+as+a+therapeutic+target+in+alzheimer%E2%80%99s+disease%3A+a+review+of+basic+research+and+clinical+evidence&doi=10.1007%2Fs40263-016-0361-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">Apolipoprotein E as a Therapeutic Target in Alzheimer's Disease: A Review of Basic Research and Clinical Evidence</span></div><div class="casAuthors">Yamazaki, Yu; Painter, Meghan M.; Bu, Guojun; Kanekiyo, Takahisa</div><div class="citationInfo"><span class="NLM_cas:title">CNS Drugs</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">773-789</span>CODEN:
                <span class="NLM_cas:coden">CNDREF</span>;
        ISSN:<span class="NLM_cas:issn">1172-7047</span>.
    
            (<span class="NLM_cas:orgname">Springer International Publishing AG</span>)
        </div><div class="casAbstract">Alzheimer's disease (AD) is a devastating neurodegenerative disorder that causes progressive cognitive decline.  The majority of AD cases are sporadic and late-onset (>65 years old) making it the leading cause of dementia in the elderly.  While both genetic and environmental factors contribute to the development of late-onset AD (LOAD), APOE polymorphism is a major genetic risk determinant for LOAD.  In humans, the APOE gene has three major allelic variants: ε2, ε3, and ε4, of which APOE ε4 is the strongest genetic risk factor for LOAD, whereas APOE ε2 is protective.  Mounting evidence suggests that APOE ε4 contributes to AD pathogenesis through multiple pathways including facilitated amyloid-β deposition, increased tangle formation, synaptic dysfunction, exacerbated neuroinflammation, and cerebrovascular defects.  Since APOE modulates multiple biol. processes through its corresponding protein apolipoprotein E (apoE), APOE gene and apoE properties have been a promising target for therapy and drug development against AD.  In this review, we summarize the current evidence regarding how the APOE ε4 allele contributes to the pathogenesis of AD and how relevant therapeutic approaches can be developed to target apoE-mediated pathways in AD.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoU4kbr0W368rVg90H21EOLACvtfcHk0lgpuHz1n_uilA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtVyiurfJ&md5=f1fa4329f1de8914d1b8e8db41ce5580</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1007%2Fs40263-016-0361-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs40263-016-0361-4%26sid%3Dliteratum%253Aachs%26aulast%3DYamazaki%26aufirst%3DY.%26aulast%3DPainter%26aufirst%3DM.%2BM.%26aulast%3DBu%26aufirst%3DG.%26aulast%3DKanekiyo%26aufirst%3DT.%26atitle%3DApolipoprotein%2520e%2520as%2520a%2520therapeutic%2520target%2520in%2520alzheimer%25E2%2580%2599s%2520disease%253A%2520a%2520review%2520of%2520basic%2520research%2520and%2520clinical%2520evidence%26jtitle%3DCNS%2520Drugs%26date%3D2016%26volume%3D30%26spage%3D773%26epage%3D789%26doi%3D10.1007%2Fs40263-016-0361-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref71"><div class="reference"><strong class="refLabel"><a href="#ref71" class="rightTabRefNumLink">71</a></strong><div class="NLM_citation" id="rightTab-cit71"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xue, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shang, P.</span></span> <span> </span><span class="NLM_article-title">Transferrin receptor 1 in cancer: a new sight for cancer therapy</span>. <i>Am. J. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">916</span>– <span class="NLM_lpage">931</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=30034931" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsFOjs7rF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2018&pages=916-931&author=Y.+Shenauthor=X.+Liauthor=D.+Dongauthor=B.+Zhangauthor=Y.+Xueauthor=P.+Shang&title=Transferrin+receptor+1+in+cancer%3A+a+new+sight+for+cancer+therapy"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">Transferrin receptor 1 in cancer: a new sight for cancer therapy</span></div><div class="casAuthors">Shen, Ying; Li, Xin; Dong, Dandan; Zhang, Bin; Xue, Yanru; Shang, Peng</div><div class="citationInfo"><span class="NLM_cas:title">American Journal of Cancer Research</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">916-931</span>CODEN:
                <span class="NLM_cas:coden">AJCRFT</span>;
        ISSN:<span class="NLM_cas:issn">2156-6976</span>.
    
            (<span class="NLM_cas:orgname">e-Century Publishing Corp.</span>)
        </div><div class="casAbstract">Iron as an important element plays crucial roles in various physiol. and pathol. processes.  Iron metab. behaves in systemic and cellular two levels that usually are in balance conditions.  The disorders of the iron metab. balances relate with many kinds of diseases including Alzheimer's disease, osteoporosis and various cancers.  In systemic iron metab. that is regulated by hepcidin-ferroportin axis, plasma iron is bound with transferrin (TF) which has two high-affinity binding sites for ferric iron.  The generic cellular iron metab. consists of iron intake, utilization and efflux.  During the iron intake process in generic cells, transferrin receptors (TFRs) act as the most important receptor mediated controls.  TFR1 and TFR2 are two subtypes of TFRs those bind with iron-transferrin complex to facilitate iron into cells.  TFR1 is ubiquitously expressed on the surfaces of generic cells, whereas TFR2 is specially expressed in liver cells.  TFR1 has attracted more attention than TFR2 by having diverse functions in both invertebrates and vertebrates.  Recently reports showed that TFR1 involved in many kinds of diseases including anemia, neurodegenerative diseases and cancers.  Most importantly, TFR1 has been verified to be abnormally expressed in various cancers.  Some exptl. and clin. drugs and antibodies targeting TFR1 have showed strong anti-tumor effects, herein TFR1 probably become a potential mol. target for diagnosis and treatment for cancer therapy.  This paper reviewed the research progresses of the roles of TFR1 in the tumorigenesis and cancer progression, the regulations of TFR1, and the therapeutic effects of targeting TFR1 on many kinds of cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrDe5E49dNFeLVg90H21EOLACvtfcHk0lgpuHz1n_uilA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsFOjs7rF&md5=f44799696f18eb50941fb7783d5377f0</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DShen%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DDong%26aufirst%3DD.%26aulast%3DZhang%26aufirst%3DB.%26aulast%3DXue%26aufirst%3DY.%26aulast%3DShang%26aufirst%3DP.%26atitle%3DTransferrin%2520receptor%25201%2520in%2520cancer%253A%2520a%2520new%2520sight%2520for%2520cancer%2520therapy%26jtitle%3DAm.%2520J.%2520Cancer%2520Res.%26date%3D2018%26volume%3D8%26spage%3D916%26epage%3D931" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref72"><div class="reference"><strong class="refLabel"><a href="#ref72" class="rightTabRefNumLink">72</a></strong><div class="NLM_citation" id="rightTab-cit72"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cui, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, W.</span></span> <span> </span><span class="NLM_article-title">Elucidation and structural modeling of CD71 as a molecular target for cell-specific aptamer binding</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>141</i></span>,  <span class="NLM_fpage">10760</span>– <span class="NLM_lpage">10769</span>, <span class="refDoi"> DOI: 10.1021/jacs.9b03720</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.9b03720" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtFGqsLfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=141&publication_year=2019&pages=10760-10769&author=X.+Wuauthor=H.+Liuauthor=D.+Hanauthor=B.+Pengauthor=H.+Zhangauthor=L.+Zhangauthor=J.+Liauthor=J.+Liuauthor=C.+Cuiauthor=S.+Fangauthor=M.+Liauthor=M.+Yeauthor=W.+Tan&title=Elucidation+and+structural+modeling+of+CD71+as+a+molecular+target+for+cell-specific+aptamer+binding&doi=10.1021%2Fjacs.9b03720"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">Elucidation and Structural Modeling of CD71 as a Molecular Target for Cell-Specific Aptamer Binding</span></div><div class="casAuthors">Wu, Xiaoqiu; Liu, Honglin; Han, Dongmei; Peng, Bo; Zhang, Hui; Zhang, Lin; Li, Jianglin; Liu, Jing; Cui, Cheng; Fang, Senbiao; Li, Min; Ye, Mao; Tan, Weihong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">141</span>
        (<span class="NLM_cas:issue">27</span>),
    <span class="NLM_cas:pages">10760-10769</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Pancreatic cancer is a highly lethal malignancy assocd. with tissues of the pancreas.  Early diagnosis and effective treatment are crucial to improving the survival rate of patients with pancreatic cancer.  In a previous study, we employed the cell-SELEX strategy to obtain an ssDNA aptamer termed XQ-2d with high binding affinity for pancreatic cancer.  Here, we first identify CD71 as the XQ-2d-binding target.  We found that knockdown of CD71 abolished the binding of XQ-2d and that the binding affinity of XQ-2d is assocd. with membrane-bound CD71, rather than total CD71 levels.  Competitive anal. revealed that XQ-2d shares the same binding site on CD71 with transferrin (Tf), but not anti-CD71 antibody.  We then used a surface energy transfer (SET) nanoruler to measure the distance between the binding sites of XQ-2d and anti-CD71 antibody, and it was about 15 nm.  Furthermore, we did mol. dynamics simulation to clarify that the spatial structure of XQ-2d and Tf competitively binding to CD71.  We also engineered XQ-2d-mediated targeted therapy for pancreatic cancer, using an XQ-2d-based complex for loading doxorubicin (Dox).  Because CD71 is overexpressed not only in pancreatic cancer but also in a variety of tumors, our work provides a systematic novel way of studying a potential biomarker and also promising tools for cancer diagnosis and therapy, opening new doors for effective cancer theranostics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGofAAoZt5oLlrVg90H21EOLACvtfcHk0lgpuHz1n_uilA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtFGqsLfN&md5=04ce70f5db10f5459f633b073ca3f9df</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1021%2Fjacs.9b03720&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.9b03720%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DHan%26aufirst%3DD.%26aulast%3DPeng%26aufirst%3DB.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DCui%26aufirst%3DC.%26aulast%3DFang%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DM.%26aulast%3DYe%26aufirst%3DM.%26aulast%3DTan%26aufirst%3DW.%26atitle%3DElucidation%2520and%2520structural%2520modeling%2520of%2520CD71%2520as%2520a%2520molecular%2520target%2520for%2520cell-specific%2520aptamer%2520binding%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2019%26volume%3D141%26spage%3D10760%26epage%3D10769%26doi%3D10.1021%2Fjacs.9b03720" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref73"><div class="reference"><strong class="refLabel"><a href="#ref73" class="rightTabRefNumLink">73</a></strong><div class="NLM_citation" id="rightTab-cit73"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perry, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muniz-Medina, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wetzel, L. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rebelatto, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hinrichs, M. J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bezabeh, B. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fleming, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dimasi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toader, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, A. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dixit, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osbourn, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coats, S. R.</span></span> <span> </span><span class="NLM_article-title">A biparatopic HER2-targeting antibody-drug conjugate induces tumor regression in primary models refractory to or ineligible for HER2-targeted therapy</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">117</span>– <span class="NLM_lpage">129</span>, <span class="refDoi"> DOI: 10.1016/j.ccell.2015.12.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1016%2Fj.ccell.2015.12.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=26766593" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC28Xmsl2lsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2016&pages=117-129&author=J.+Y.+Liauthor=S.+R.+Perryauthor=V.+Muniz-Medinaauthor=X.+Wangauthor=L.+K.+Wetzelauthor=M.+C.+Rebelattoauthor=M.+J.+M.+Hinrichsauthor=B.+Z.+Bezabehauthor=R.+L.+Flemingauthor=N.+Dimasiauthor=H.+Fengauthor=D.+Toaderauthor=A.+Q.+Yuanauthor=L.+Xuauthor=J.+Linauthor=C.+Gaoauthor=H.+Wuauthor=R.+Dixitauthor=J.+K.+Osbournauthor=S.+R.+Coats&title=A+biparatopic+HER2-targeting+antibody-drug+conjugate+induces+tumor+regression+in+primary+models+refractory+to+or+ineligible+for+HER2-targeted+therapy&doi=10.1016%2Fj.ccell.2015.12.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">A Biparatopic HER2-Targeting Antibody-Drug Conjugate Induces Tumor Regression in Primary Models Refractory to or Ineligible for HER2-Targeted Therapy</span></div><div class="casAuthors">Li, John Y.; Perry, Samuel R.; Muniz-Medina, Vanessa; Wang, Xinzhong; Wetzel, Leslie K.; Rebelatto, Marlon C.; Hinrichs, Mary Jane Masson; Bezabeh, Binyam Z.; Fleming, Ryan L.; Dimasi, Nazzareno; Feng, Hui; Toader, Dorin; Yuan, Andy Q.; Xu, Lan; Lin, Jia; Gao, Changshou; Wu, Herren; Dixit, Rakesh; Osbourn, Jane K.; Coats, Steven R.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">117-129</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Antibody-drug conjugate (ADC) which delivers cytotoxic drugs specifically into targeted cells through internalization and lysosomal trafficking has emerged as an effective cancer therapy.  We show that a bivalent biparatopic antibody targeting two non-overlapping epitopes on HER2 can induce HER2 receptor clustering, which in turn promotes robust internalization, lysosomal trafficking, and degrdn.  When conjugated with a tubulysin-based microtubule inhibitor, the biparatopic ADC demonstrates superior anti-tumor activity over ado-trastuzumab emtansine (T-DM1) in tumor models representing various patient subpopulations, including T-DM1 eligible, T-DM1 ineligible, and T-DM1 relapsed/refractory.  Our findings indicate that this biparatopic ADC has promising potential as an effective therapy for metastatic breast cancer and a broader patient population may benefit from this unique HER2-targeting ADC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp18b-9TGTzUbVg90H21EOLACvtfcHk0li2E7iayL6-UA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xmsl2lsA%253D%253D&md5=bbd94b53427de32f8d636b63507dd139</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2015.12.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2015.12.008%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DJ.%2BY.%26aulast%3DPerry%26aufirst%3DS.%2BR.%26aulast%3DMuniz-Medina%26aufirst%3DV.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DWetzel%26aufirst%3DL.%2BK.%26aulast%3DRebelatto%26aufirst%3DM.%2BC.%26aulast%3DHinrichs%26aufirst%3DM.%2BJ.%2BM.%26aulast%3DBezabeh%26aufirst%3DB.%2BZ.%26aulast%3DFleming%26aufirst%3DR.%2BL.%26aulast%3DDimasi%26aufirst%3DN.%26aulast%3DFeng%26aufirst%3DH.%26aulast%3DToader%26aufirst%3DD.%26aulast%3DYuan%26aufirst%3DA.%2BQ.%26aulast%3DXu%26aufirst%3DL.%26aulast%3DLin%26aufirst%3DJ.%26aulast%3DGao%26aufirst%3DC.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DDixit%26aufirst%3DR.%26aulast%3DOsbourn%26aufirst%3DJ.%2BK.%26aulast%3DCoats%26aufirst%3DS.%2BR.%26atitle%3DA%2520biparatopic%2520HER2-targeting%2520antibody-drug%2520conjugate%2520induces%2520tumor%2520regression%2520in%2520primary%2520models%2520refractory%2520to%2520or%2520ineligible%2520for%2520HER2-targeted%2520therapy%26jtitle%3DCancer%2520Cell%26date%3D2016%26volume%3D29%26spage%3D117%26epage%3D129%26doi%3D10.1016%2Fj.ccell.2015.12.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref74"><div class="reference"><strong class="refLabel"><a href="#ref74" class="rightTabRefNumLink">74</a></strong><div class="NLM_citation" id="rightTab-cit74"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Spangler, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neil, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abramovitch, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yarden, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lauffenburger, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wittrup, K. D.</span></span> <span> </span><span class="NLM_article-title">Combination antibody treatment down-regulates epidermal growth factor receptor by inhibiting endosomal recycling</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>107</i></span>,  <span class="NLM_fpage">13252</span>– <span class="NLM_lpage">13257</span>, <span class="refDoi"> DOI: 10.1073/pnas.0913476107</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1073%2Fpnas.0913476107" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=20616078" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht1GrurvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=107&publication_year=2010&pages=13252-13257&author=J.+B.+Spanglerauthor=J.+R.+Neilauthor=S.+Abramovitchauthor=Y.+Yardenauthor=F.+M.+Whiteauthor=D.+A.+Lauffenburgerauthor=K.+D.+Wittrup&title=Combination+antibody+treatment+down-regulates+epidermal+growth+factor+receptor+by+inhibiting+endosomal+recycling&doi=10.1073%2Fpnas.0913476107"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit74R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">Combination antibody treatment down-regulates epidermal growth factor receptor by inhibiting endosomal recycling</span></div><div class="casAuthors">Spangler, Jamie B.; Neil, Jason R.; Abramovitch, Sivan; Yarden, Yosef; White, Forest M.; Lauffenburger, Douglas A.; Wittrup, K. Dane</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">107</span>
        (<span class="NLM_cas:issue">30</span>),
    <span class="NLM_cas:pages">13252-13257, S13252/1-S13252/8</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Due to its common dysregulation in epithelial-based cancers and extensive characterization of its role in tumor growth, epidermal growth factor receptor (EGFR) is a highly validated target for anticancer therapies.  There has been particular interest in the development of monoclonal antibodies (mAbs) targeting EGFR, resulting in two approved mAb-based drugs and several others in clin. trials.  It has recently been reported that treatment with combinations of noncompetitive mAbs can induce receptor clustering, leading to synergistic receptor down-regulation.  We elucidate three key aspects of this phenomenon.  First, we show that highly potent combinations consisting of two noncompetitive mAbs that target EGFR domain 3 reduce surface receptor levels by up to 80% with a half-time of 0.5-5 h in both normal and transformed human cell lines to an extent inversely proportional to receptor d.  Second, we find the mechanism underlying down-regulation to be consistent with recycling inhibition.  Third, in contrast to the agonism assocd. with ligand-induced down-regulation, we demonstrate that mAb-induced down-regulation does not activate EGFR or its downstream effectors and it leads to synergistic redn. in migration and proliferation of cells that secrete autocrine ligand.  These new insights will aid in ongoing rational design of EGFR-targeted antibody therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoTrRDTQQgtl7Vg90H21EOLACvtfcHk0li2E7iayL6-UA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht1GrurvI&md5=5da987aa7b91ebf421103a58232839d1</span></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0913476107&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0913476107%26sid%3Dliteratum%253Aachs%26aulast%3DSpangler%26aufirst%3DJ.%2BB.%26aulast%3DNeil%26aufirst%3DJ.%2BR.%26aulast%3DAbramovitch%26aufirst%3DS.%26aulast%3DYarden%26aufirst%3DY.%26aulast%3DWhite%26aufirst%3DF.%2BM.%26aulast%3DLauffenburger%26aufirst%3DD.%2BA.%26aulast%3DWittrup%26aufirst%3DK.%2BD.%26atitle%3DCombination%2520antibody%2520treatment%2520down-regulates%2520epidermal%2520growth%2520factor%2520receptor%2520by%2520inhibiting%2520endosomal%2520recycling%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2010%26volume%3D107%26spage%3D13252%26epage%3D13257%26doi%3D10.1073%2Fpnas.0913476107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref75"><div class="reference"><strong class="refLabel"><a href="#ref75" class="rightTabRefNumLink">75</a></strong><div class="NLM_citation" id="rightTab-cit75"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Spiess, M.</span></span> <span> </span><span class="NLM_article-title">The asialoglycoprotein receptor: a model for endocytic transport receptors</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">1990</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">10009</span>– <span class="NLM_lpage">10018</span>, <span class="refDoi"> DOI: 10.1021/bi00495a001</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi00495a001" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADyaK3cXlvVymurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=1990&pages=10009-10018&author=M.+Spiess&title=The+asialoglycoprotein+receptor%3A+a+model+for+endocytic+transport+receptors&doi=10.1021%2Fbi00495a001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit75R"><div class="casContent"><span class="casTitleNuber">75</span><div class="casTitle"><span class="NLM_cas:atitle">The asialoglycoprotein receptor:  a model for endocytic transport receptors</span></div><div class="casAuthors">Spiess, Martin</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">1990</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">43</span>),
    <span class="NLM_cas:pages">10009-18</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    </div><div class="casAbstract">A review, with 129 refs., of the structure and function of the asialoglycoprotein receptor of hepatocytes as a model for endocytic transport receptors in general.  The topics included are: its role as an hepatic lectin; the existence of 2 genes and 2 proteins for the receptor; the organization of the genes for the receptor; and the carbohydrate-binding domain, the transmembrane signal-anchor domain, the cytoplasmic domain, and the connecting domain(s) of the receptor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAiexeMoNXGbVg90H21EOLACvtfcHk0liR-h6l9oqP7Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3cXlvVymurs%253D&md5=480a20b64d2c561fe9de0fb4b8528940</span></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1021%2Fbi00495a001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi00495a001%26sid%3Dliteratum%253Aachs%26aulast%3DSpiess%26aufirst%3DM.%26atitle%3DThe%2520asialoglycoprotein%2520receptor%253A%2520a%2520model%2520for%2520endocytic%2520transport%2520receptors%26jtitle%3DBiochemistry%26date%3D1990%26volume%3D29%26spage%3D10009%26epage%3D10018%26doi%3D10.1021%2Fbi00495a001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref76"><div class="reference"><strong class="refLabel"><a href="#ref76" class="rightTabRefNumLink">76</a></strong><div class="NLM_citation" id="rightTab-cit76"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Park, E. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Unverzagt, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gabius, H.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baenziger, J. U.</span></span> <span> </span><span class="NLM_article-title">The asialoglycoprotein receptor clears glycoconjugates terminating with sialic acidα2,6GalNAc</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>102</i></span>,  <span class="NLM_fpage">17125</span>, <span class="refDoi"> DOI: 10.1073/pnas.0508537102</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1073%2Fpnas.0508537102" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=16286643" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BD2MXht1yksLbI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=2005&pages=17125&author=E.+I.+Parkauthor=Y.+Miauthor=C.+Unverzagtauthor=H.-J.+Gabiusauthor=J.+U.+Baenziger&title=The+asialoglycoprotein+receptor+clears+glycoconjugates+terminating+with+sialic+acid%CE%B12%2C6GalNAc&doi=10.1073%2Fpnas.0508537102"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit76R"><div class="casContent"><span class="casTitleNuber">76</span><div class="casTitle"><span class="NLM_cas:atitle">The asialoglycoprotein receptor clears glycoconjugates terminating with sialic acidα2,6GalNAc</span></div><div class="casAuthors">Park, Eric I.; Mi, Yiling; Unverzagt, Carlo; Gabius, Hans-Joachim; Baenziger, Jacques U.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">102</span>
        (<span class="NLM_cas:issue">47</span>),
    <span class="NLM_cas:pages">17125-17129</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Endogenous ligands have not, to date, been identified for the asialoglycoprotein receptor (ASGP-R), which is abundantly expressed by parenchymal cells in the liver of mammals.  On the basis of the rapid clearance of BSA bearing multiple chem. coupled sialic acid (Sia)α2,6GalNAcβ1,4GlcNAcβ1,2Man tetrasaccharides (SiaGGnM-BSA) from the circulation, and the ability of the ASGP-R hepatic lectin-1 subunit to bind SiaGGnM-BSA, we previously proposed that glycoproteins modified with structures terminating with Siaα2,6GalNAc may represent previously unrecognized examples of endogenous ligands for this receptor.  Here, we have taken a genetic approach using wild-type and ASGP-R-deficient mice to det. that the ASGP-R in vivo does indeed account for the rapid clearance of glycoconjugates terminating with Siaα2,6GalNAc.  We have also detd. that the ASGP-R is able to bind core-substituted oligosaccharides with the terminal sequence Siaα2,6Galβ1,4GlcNAc but not those with the terminal Siaα2,3Galβ1,4GlcNAc.  We propose that glycoproteins bearing terminals Siaα2,6GalNAc and Siaα2,6Gal are endogenous ligands for the ASGP-R, and that the ASGP-R helps to regulate the relative concn. of serum glycoproteins bearing α2,6-linked Sia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqP8PVdAuhz6bVg90H21EOLACvtfcHk0liR-h6l9oqP7Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXht1yksLbI&md5=739e0ef26daa7d1cb6675c1ab002e37a</span></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0508537102&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0508537102%26sid%3Dliteratum%253Aachs%26aulast%3DPark%26aufirst%3DE.%2BI.%26aulast%3DMi%26aufirst%3DY.%26aulast%3DUnverzagt%26aufirst%3DC.%26aulast%3DGabius%26aufirst%3DH.-J.%26aulast%3DBaenziger%26aufirst%3DJ.%2BU.%26atitle%3DThe%2520asialoglycoprotein%2520receptor%2520clears%2520glycoconjugates%2520terminating%2520with%2520sialic%2520acid%25CE%25B12%252C6GalNAc%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2005%26volume%3D102%26spage%3D17125%26doi%3D10.1073%2Fpnas.0508537102" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref77"><div class="reference"><strong class="refLabel"><a href="#ref77" class="rightTabRefNumLink">77</a></strong><div class="NLM_citation" id="rightTab-cit77"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zimmermann, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karsten, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiesa, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boyce, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bettencourt, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hutabarat, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nochur, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vaishnaw, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gollob, J.</span></span> <span> </span><span class="NLM_article-title">Clinical proof of concept for a novel hepatocyte-targeting GalNAc-siRNA conjugate</span>. <i>Mol. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">71</span>– <span class="NLM_lpage">78</span>, <span class="refDoi"> DOI: 10.1016/j.ymthe.2016.10.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1016%2Fj.ymthe.2016.10.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=28129130" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmvFCmtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2017&pages=71-78&author=T.+S.+Zimmermannauthor=V.+Karstenauthor=A.+Chanauthor=J.+Chiesaauthor=M.+Boyceauthor=B.+R.+Bettencourtauthor=R.+Hutabaratauthor=S.+Nochurauthor=A.+Vaishnawauthor=J.+Gollob&title=Clinical+proof+of+concept+for+a+novel+hepatocyte-targeting+GalNAc-siRNA+conjugate&doi=10.1016%2Fj.ymthe.2016.10.019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit77R"><div class="casContent"><span class="casTitleNuber">77</span><div class="casTitle"><span class="NLM_cas:atitle">Clinical Proof of Concept for a Novel Hepatocyte-Targeting GalNAc-siRNA Conjugate</span></div><div class="casAuthors">Zimmermann, Tracy S.; Karsten, Verena; Chan, Amy; Chiesa, Joseph; Boyce, Malcolm; Bettencourt, Brian R.; Hutabarat, Renta; Nochur, Saraswathy; Vaishnaw, Akshay; Gollob, Jared</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Therapy</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">71-78</span>CODEN:
                <span class="NLM_cas:coden">MTOHCK</span>;
        ISSN:<span class="NLM_cas:issn">1525-0024</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Advancement of RNAi-based therapeutics depends on effective delivery to the site of protein synthesis.  Although i.v. administered, multi-component delivery vehicles have enabled small interfering RNA (siRNA) delivery and progression into clin. development, advances of single-component, systemic siRNA delivery have been challenging.  In pre-clin. models, attachment of a triantennary N-acetylgalactosamine (GalNAc) ligand to an siRNA mediates hepatocyte uptake via the asialoglycoprotein receptor enabling RNAi-mediated gene silencing.  In this phase 1 study, we assessed translation of this delivery approach by evaluating the safety, tolerability, pharmacokinetics, and pharmacodynamics of a GalNAc-siRNA conjugate, revusiran, targeting transthyretin (TTR).  Subjects received a placebo or ascending doses of revusiran s.c. ranging from 1.25-10 mg/kg in the single and 2.5-10 mg/kg in the multiple ascending dose phases.  Revusiran was generally well tolerated, with transient, mild to moderate injection site reactions the most common treatment-emergent adverse events.  Doses of 2.5-10 mg/kg revusiran elicited a significant redn. of serum TTR vs. the placebo (p < 0.01), with mean TTR redns. of approx. 90% obsd. with multiple dosing.  These results demonstrate translation of this novel delivery platform, enabling clin. development of s.c. administered GalNAc-siRNAs for liver-based diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoC6JF_TMrMX7Vg90H21EOLACvtfcHk0liR-h6l9oqP7Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmvFCmtb8%253D&md5=76ea4ba1335793017d436d8a3f78f643</span></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=10.1016%2Fj.ymthe.2016.10.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ymthe.2016.10.019%26sid%3Dliteratum%253Aachs%26aulast%3DZimmermann%26aufirst%3DT.%2BS.%26aulast%3DKarsten%26aufirst%3DV.%26aulast%3DChan%26aufirst%3DA.%26aulast%3DChiesa%26aufirst%3DJ.%26aulast%3DBoyce%26aufirst%3DM.%26aulast%3DBettencourt%26aufirst%3DB.%2BR.%26aulast%3DHutabarat%26aufirst%3DR.%26aulast%3DNochur%26aufirst%3DS.%26aulast%3DVaishnaw%26aufirst%3DA.%26aulast%3DGollob%26aufirst%3DJ.%26atitle%3DClinical%2520proof%2520of%2520concept%2520for%2520a%2520novel%2520hepatocyte-targeting%2520GalNAc-siRNA%2520conjugate%26jtitle%3DMol.%2520Ther.%26date%3D2017%26volume%3D25%26spage%3D71%26epage%3D78%26doi%3D10.1016%2Fj.ymthe.2016.10.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref78"><div class="reference"><strong class="refLabel"><a href="#ref78" class="rightTabRefNumLink">78</a></strong><div class="NLM_citation" id="rightTab-cit78"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Uhlén, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fagerberg, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hallström, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindskog, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oksvold, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mardinoglu, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sivertsson, Å.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kampf, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sjöstedt, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asplund, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olsson, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edlund, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lundberg, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Navani, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szigyarto, C. A.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Odeberg, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Djureinovic, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takanen, J. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hober, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alm, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edqvist, P.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berling, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tegel, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulder, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rockberg, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nilsson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwenk, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamsten, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">von Feilitzen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forsberg, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Persson, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johansson, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwahlen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">von Heijne, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nielsen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pontén, F.</span></span> <span> </span><span class="NLM_article-title">Tissue-based map of the human proteome</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>347</i></span>,  <span class="NLM_fpage">1260419</span>, <span class="refDoi"> DOI: 10.1126/science.1260419</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1126%2Fscience.1260419" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=25613900" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A280%3ADC%252BC2MvptFakug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=347&publication_year=2015&pages=1260419&author=M.+Uhl%C3%A9nauthor=L.+Fagerbergauthor=B.+M.+Hallstr%C3%B6mauthor=C.+Lindskogauthor=P.+Oksvoldauthor=A.+Mardinogluauthor=%C3%85.+Sivertssonauthor=C.+Kampfauthor=E.+Sj%C3%B6stedtauthor=A.+Asplundauthor=I.+Olssonauthor=K.+Edlundauthor=E.+Lundbergauthor=S.+Navaniauthor=C.+A.-K.+Szigyartoauthor=J.+Odebergauthor=D.+Djureinovicauthor=J.+O.+Takanenauthor=S.+Hoberauthor=T.+Almauthor=P.-H.+Edqvistauthor=H.+Berlingauthor=H.+Tegelauthor=J.+Mulderauthor=J.+Rockbergauthor=P.+Nilssonauthor=J.+M.+Schwenkauthor=M.+Hamstenauthor=K.+von+Feilitzenauthor=M.+Forsbergauthor=L.+Perssonauthor=F.+Johanssonauthor=M.+Zwahlenauthor=G.+von+Heijneauthor=J.+Nielsenauthor=F.+Pont%C3%A9n&title=Tissue-based+map+of+the+human+proteome&doi=10.1126%2Fscience.1260419"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit78R"><div class="casContent"><span class="casTitleNuber">78</span><div class="casTitle"><span class="NLM_cas:atitle">Proteomics. Tissue-based map of the human proteome</span></div><div class="casAuthors">Uhlen Mathias; Fagerberg Linn; Oksvold Per; Sivertsson ¡ÑÜAsa; Lundberg Emma; Odeberg Jacob; Alm Tove; Nilsson Peter; Schwenk Jochen M; von Feilitzen Kalle; Forsberg Mattias; Persson Lukas; Johansson Fredric; Zwahlen Martin; Hallstrom Bjorn M; Lindskog Cecilia; Kampf Caroline; Asplund Anna; Olsson IngMarie; Djureinovic Dijana; Edqvist Per-Henrik; Ponten Fredrik; Mardinoglu Adil; Sjostedt Evelina; Edlund Karolina; Navani Sanjay; Szigyarto Cristina Al-Khalili; Takanen Jenny Ottosson; Hober Sophia; Berling Holger; Tegel Hanna; Rockberg Johan; Hamsten Marica; Mulder Jan; von Heijne Gunnar; Nielsen Jens</div><div class="citationInfo"><span class="NLM_cas:title">Science (New York, N.Y.)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">347</span>
        (<span class="NLM_cas:issue">6220</span>),
    <span class="NLM_cas:pages">1260419</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Resolving the molecular details of proteome variation in the different tissues and organs of the human body will greatly increase our knowledge of human biology and disease.  Here, we present a map of the human tissue proteome based on an integrated omics approach that involves quantitative transcriptomics at the tissue and organ level, combined with tissue microarray-based immunohistochemistry, to achieve spatial localization of proteins down to the single-cell level.  Our tissue-based analysis detected more than 90% of the putative protein-coding genes.  We used this approach to explore the human secretome, the membrane proteome, the druggable proteome, the cancer proteome, and the metabolic functions in 32 different tissues and organs.  All the data are integrated in an interactive Web-based database that allows exploration of individual proteins, as well as navigation of global expression patterns, in all major tissues and organs in the human body.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQT9YWouQZh5DEC-75aqa5AfW6udTcc2eYU_dQ86Irn6Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2MvptFakug%253D%253D&md5=95510888806e5feb6b8024ee2a417d1c</span></div><a href="/servlet/linkout?suffix=cit78&amp;dbid=16384&amp;doi=10.1126%2Fscience.1260419&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1260419%26sid%3Dliteratum%253Aachs%26aulast%3DUhl%25C3%25A9n%26aufirst%3DM.%26aulast%3DFagerberg%26aufirst%3DL.%26aulast%3DHallstr%25C3%25B6m%26aufirst%3DB.%2BM.%26aulast%3DLindskog%26aufirst%3DC.%26aulast%3DOksvold%26aufirst%3DP.%26aulast%3DMardinoglu%26aufirst%3DA.%26aulast%3DSivertsson%26aufirst%3D%25C3%2585.%26aulast%3DKampf%26aufirst%3DC.%26aulast%3DSj%25C3%25B6stedt%26aufirst%3DE.%26aulast%3DAsplund%26aufirst%3DA.%26aulast%3DOlsson%26aufirst%3DI.%26aulast%3DEdlund%26aufirst%3DK.%26aulast%3DLundberg%26aufirst%3DE.%26aulast%3DNavani%26aufirst%3DS.%26aulast%3DSzigyarto%26aufirst%3DC.%2BA.-K.%26aulast%3DOdeberg%26aufirst%3DJ.%26aulast%3DDjureinovic%26aufirst%3DD.%26aulast%3DTakanen%26aufirst%3DJ.%2BO.%26aulast%3DHober%26aufirst%3DS.%26aulast%3DAlm%26aufirst%3DT.%26aulast%3DEdqvist%26aufirst%3DP.-H.%26aulast%3DBerling%26aufirst%3DH.%26aulast%3DTegel%26aufirst%3DH.%26aulast%3DMulder%26aufirst%3DJ.%26aulast%3DRockberg%26aufirst%3DJ.%26aulast%3DNilsson%26aufirst%3DP.%26aulast%3DSchwenk%26aufirst%3DJ.%2BM.%26aulast%3DHamsten%26aufirst%3DM.%26aulast%3Dvon%2BFeilitzen%26aufirst%3DK.%26aulast%3DForsberg%26aufirst%3DM.%26aulast%3DPersson%26aufirst%3DL.%26aulast%3DJohansson%26aufirst%3DF.%26aulast%3DZwahlen%26aufirst%3DM.%26aulast%3Dvon%2BHeijne%26aufirst%3DG.%26aulast%3DNielsen%26aufirst%3DJ.%26aulast%3DPont%25C3%25A9n%26aufirst%3DF.%26atitle%3DTissue-based%2520map%2520of%2520the%2520human%2520proteome%26jtitle%3DScience%26date%3D2015%26volume%3D347%26spage%3D1260419%26doi%3D10.1126%2Fscience.1260419" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref79"><div class="reference"><strong class="refLabel"><a href="#ref79" class="rightTabRefNumLink">79</a></strong><div class="NLM_citation" id="rightTab-cit79"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nakagawa, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amano, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mizushima, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamaguchi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamimoto, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nara, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Funao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakata, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsuda, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamada, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshimori, T.</span></span> <span> </span><span class="NLM_article-title">Autophagy defends cells against invading group a streptococcus</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>306</i></span>,  <span class="NLM_fpage">1037</span>– <span class="NLM_lpage">1040</span>, <span class="refDoi"> DOI: 10.1126/science.1103966</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1126%2Fscience.1103966" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=15528445" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BD2cXptFyjurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=306&publication_year=2004&pages=1037-1040&author=I.+Nakagawaauthor=A.+Amanoauthor=N.+Mizushimaauthor=A.+Yamamotoauthor=H.+Yamaguchiauthor=T.+Kamimotoauthor=A.+Naraauthor=J.+Funaoauthor=M.+Nakataauthor=K.+Tsudaauthor=S.+Hamadaauthor=T.+Yoshimori&title=Autophagy+defends+cells+against+invading+group+a+streptococcus&doi=10.1126%2Fscience.1103966"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit79R"><div class="casContent"><span class="casTitleNuber">79</span><div class="casTitle"><span class="NLM_cas:atitle">Autophagy Defends Cells Against Invading Group A Streptococcus</span></div><div class="casAuthors">Nakagawa, Ichiro; Amano, Atsuo; Mizushima, Noboru; Yamamoto, Akitsugu; Yamaguchi, Hitomi; Kamimoto, Takahiro; Nara, Atsuki; Funao, Junko; Nakata, Masanobu; Tsuda, Kayoko; Hamada, Shigeyuki; Yoshimori, Tamotsu</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">306</span>
        (<span class="NLM_cas:issue">5698</span>),
    <span class="NLM_cas:pages">1037-1040</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">We found that the autophagic machinery could effectively eliminate pathogenic group A Streptococcus (GAS) within nonphagocytic cells.  After escaping from endosomes into the cytoplasm, GAS became enveloped by autophagosome-like compartments and were killed upon fusion of these compartments with lysosomes.  In autophagy-deficient Atg5-/- cells, GAS survived, multiplied, and were released from the cells.  Thus, the autophagic machinery can act as an innate defense system against invading pathogens.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqtqiNA9WkovbVg90H21EOLACvtfcHk0lg2ppvhBJyknw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXptFyjurc%253D&md5=59cdd8514ac2752c83c2a2aa2d0ef8f5</span></div><a href="/servlet/linkout?suffix=cit79&amp;dbid=16384&amp;doi=10.1126%2Fscience.1103966&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1103966%26sid%3Dliteratum%253Aachs%26aulast%3DNakagawa%26aufirst%3DI.%26aulast%3DAmano%26aufirst%3DA.%26aulast%3DMizushima%26aufirst%3DN.%26aulast%3DYamamoto%26aufirst%3DA.%26aulast%3DYamaguchi%26aufirst%3DH.%26aulast%3DKamimoto%26aufirst%3DT.%26aulast%3DNara%26aufirst%3DA.%26aulast%3DFunao%26aufirst%3DJ.%26aulast%3DNakata%26aufirst%3DM.%26aulast%3DTsuda%26aufirst%3DK.%26aulast%3DHamada%26aufirst%3DS.%26aulast%3DYoshimori%26aufirst%3DT.%26atitle%3DAutophagy%2520defends%2520cells%2520against%2520invading%2520group%2520a%2520streptococcus%26jtitle%3DScience%26date%3D2004%26volume%3D306%26spage%3D1037%26epage%3D1040%26doi%3D10.1126%2Fscience.1103966" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref80"><div class="reference"><strong class="refLabel"><a href="#ref80" class="rightTabRefNumLink">80</a></strong><div class="NLM_citation" id="rightTab-cit80"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ito, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saito, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nozawa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujii, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sawa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inoue, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsunaga, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akashi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hashimoto, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aikawa, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takahashi, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sagara, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Komatsu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akaike, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakagawa, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arimoto, H.</span></span> <span> </span><span class="NLM_article-title">Endogenous nitrated nucleotide is a key mediator of autophagy and innate defense against bacteria</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">794</span>– <span class="NLM_lpage">804</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2013.10.024</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1016%2Fj.molcel.2013.10.024" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=24268578" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvVKgsb3L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2013&pages=794-804&author=C.+Itoauthor=Y.+Saitoauthor=T.+Nozawaauthor=S.+Fujiiauthor=T.+Sawaauthor=H.+Inoueauthor=T.+Matsunagaauthor=S.+Khanauthor=S.+Akashiauthor=R.+Hashimotoauthor=C.+Aikawaauthor=E.+Takahashiauthor=H.+Sagaraauthor=M.+Komatsuauthor=K.+Tanakaauthor=T.+Akaikeauthor=I.+Nakagawaauthor=H.+Arimoto&title=Endogenous+nitrated+nucleotide+is+a+key+mediator+of+autophagy+and+innate+defense+against+bacteria&doi=10.1016%2Fj.molcel.2013.10.024"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit80R"><div class="casContent"><span class="casTitleNuber">80</span><div class="casTitle"><span class="NLM_cas:atitle">Endogenous Nitrated Nucleotide Is a Key Mediator of Autophagy and Innate Defense against Bacteria</span></div><div class="casAuthors">Ito, Chiaki; Saito, Yohei; Nozawa, Takashi; Fujii, Shigemoto; Sawa, Tomohiro; Inoue, Hirofumi; Matsunaga, Tetsuro; Khan, Shahzada; Akashi, Soichiro; Hashimoto, Ryota; Aikawa, Chihiro; Takahashi, Eriko; Sagara, Hiroshi; Komatsu, Masaaki; Tanaka, Keiji; Akaike, Takaaki; Nakagawa, Ichiro; Arimoto, Hirokazu</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">794-804</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Autophagy is a cellular self-catabolic process wherein organelles, macromols., and invading microbes are sequestered in autophagosomes that fuse with lysosomes.  In this study, we uncover the role of nitric oxide (NO) as a signaling mol. for autophagy induction via its downstream mediator, 8-nitroguanosine 3',5'-cyclic monophosphate (8-nitro-cGMP).  We found that 8-nitro-cGMP-induced autophagy is mediated by Lys63-linked polyubiquitination and that endogenous 8-nitro-cGMP promotes autophagic exclusion of invading group A Streptococcus (GAS) from cells.  8-Nitro-cGMP can modify Cys residues by S-guanylation of proteins.  We showed that intracellular GAS is modified with S-guanylation extensively in autophagosomes-like vacuoles, suggesting the role of S-guanylation as a marker for selective autophagic degrdn.  This finding is supported by the fact that S-guanylated bacteria were selectively marked with polyubiquitin, a known mol. tag for selective transport to autophagosomes.  These results collectively indicate that 8-nitro-cGMP plays a crucial role in cytoprotection during bacterial infections or inflammations via autophagy upregulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrnl0efbVmHOrVg90H21EOLACvtfcHk0lj7J-VdIx00qA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvVKgsb3L&md5=713814689a0ea39dceb41f5767545fcb</span></div><a href="/servlet/linkout?suffix=cit80&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2013.10.024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2013.10.024%26sid%3Dliteratum%253Aachs%26aulast%3DIto%26aufirst%3DC.%26aulast%3DSaito%26aufirst%3DY.%26aulast%3DNozawa%26aufirst%3DT.%26aulast%3DFujii%26aufirst%3DS.%26aulast%3DSawa%26aufirst%3DT.%26aulast%3DInoue%26aufirst%3DH.%26aulast%3DMatsunaga%26aufirst%3DT.%26aulast%3DKhan%26aufirst%3DS.%26aulast%3DAkashi%26aufirst%3DS.%26aulast%3DHashimoto%26aufirst%3DR.%26aulast%3DAikawa%26aufirst%3DC.%26aulast%3DTakahashi%26aufirst%3DE.%26aulast%3DSagara%26aufirst%3DH.%26aulast%3DKomatsu%26aufirst%3DM.%26aulast%3DTanaka%26aufirst%3DK.%26aulast%3DAkaike%26aufirst%3DT.%26aulast%3DNakagawa%26aufirst%3DI.%26aulast%3DArimoto%26aufirst%3DH.%26atitle%3DEndogenous%2520nitrated%2520nucleotide%2520is%2520a%2520key%2520mediator%2520of%2520autophagy%2520and%2520innate%2520defense%2520against%2520bacteria%26jtitle%3DMol.%2520Cell%26date%3D2013%26volume%3D52%26spage%3D794%26epage%3D804%26doi%3D10.1016%2Fj.molcel.2013.10.024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref81"><div class="reference"><strong class="refLabel"><a href="#ref81" class="rightTabRefNumLink">81</a></strong><div class="NLM_citation" id="rightTab-cit81"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, S.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawabata, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, Y.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Omori, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamasaki, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kusaba, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwamoto, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arimoto, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noda, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Y.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshimori, T.</span></span> <span> </span><span class="NLM_article-title">Endothelial cells are intrinsically defective in xenophagy of Streptococcus pyogenes</span>. <i>PLoS Pathog.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">e1006444</span>, <span class="refDoi"> DOI: 10.1371/journal.ppat.1006444</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1371%2Fjournal.ppat.1006444" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=28683091" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC1cXis1eksbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&pages=e1006444&author=S.-L.+Luauthor=T.+Kawabataauthor=Y.-L.+Chengauthor=H.+Omoriauthor=M.+Hamasakiauthor=T.+Kusabaauthor=R.+Iwamotoauthor=H.+Arimotoauthor=T.+Nodaauthor=Y.-S.+Linauthor=T.+Yoshimori&title=Endothelial+cells+are+intrinsically+defective+in+xenophagy+of+Streptococcus+pyogenes&doi=10.1371%2Fjournal.ppat.1006444"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit81R"><div class="casContent"><span class="casTitleNuber">81</span><div class="casTitle"><span class="NLM_cas:atitle">Endothelial cells are intrinsically defective in xenophagy of Streptococcus pyogenes</span></div><div class="casAuthors">Lu, Shiou-Ling; Kawabata, Tsuyoshi; Cheng, Yi-Lin; Omori, Hiroko; Hamasaki, Maho; Kusaba, Tatsuya; Iwamoto, Ryo; Arimoto, Hirokazu; Noda, Takeshi; Lin, Yee-Shin; Yoshimori, Tamotsu</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Pathogens</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">e1006444/1-e1006444/19</span>CODEN:
                <span class="NLM_cas:coden">PPLACN</span>;
        ISSN:<span class="NLM_cas:issn">1553-7374</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Group A Streptococcus (GAS) is deleterious pathogenic bacteria whose interaction with blood vessels leads to life-threatening bacteremia.  Although xenophagy, a special form of autophagy, eliminates invading GAS in epithelial cells, we found that GAS could survive and multiply in endothelial cells.  Endothelial cells were competent in starvation-induced autophagy, but failed to form double-membrane structures surrounding GAS, an essential step in xenophagy.  This deficiency stemmed from reduced recruitment of ubiquitin and several core autophagy proteins in endothelial cells, as demonstrated by the fact that it could be rescued by exogenous coating of GAS with ubiquitin.  The defect was assocd. with reduced NO-mediated ubiquitin signaling.  Therefore, we propose that the lack of efficient clearance of GAS in endothelial cells is caused by their intrinsic inability to target GAS with ubiquitin to promote autophagosome biogenesis for xenophagy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPO60rYg1GYLVg90H21EOLACvtfcHk0lj7J-VdIx00qA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXis1eksbg%253D&md5=86844f09ff46cd74b91836f37d97693f</span></div><a href="/servlet/linkout?suffix=cit81&amp;dbid=16384&amp;doi=10.1371%2Fjournal.ppat.1006444&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.ppat.1006444%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DS.-L.%26aulast%3DKawabata%26aufirst%3DT.%26aulast%3DCheng%26aufirst%3DY.-L.%26aulast%3DOmori%26aufirst%3DH.%26aulast%3DHamasaki%26aufirst%3DM.%26aulast%3DKusaba%26aufirst%3DT.%26aulast%3DIwamoto%26aufirst%3DR.%26aulast%3DArimoto%26aufirst%3DH.%26aulast%3DNoda%26aufirst%3DT.%26aulast%3DLin%26aufirst%3DY.-S.%26aulast%3DYoshimori%26aufirst%3DT.%26atitle%3DEndothelial%2520cells%2520are%2520intrinsically%2520defective%2520in%2520xenophagy%2520of%2520Streptococcus%2520pyogenes%26jtitle%3DPLoS%2520Pathog.%26date%3D2017%26volume%3D13%26spage%3De1006444%26doi%3D10.1371%2Fjournal.ppat.1006444" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref82"><div class="reference"><strong class="refLabel"><a href="#ref82" class="rightTabRefNumLink">82</a></strong><div class="NLM_citation" id="rightTab-cit82"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sawa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zaki, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okamoto, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akuta, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tokutomi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim-Mitsuyama, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ihara, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobayashi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujii, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arimoto, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akaike, T.</span></span> <span> </span><span class="NLM_article-title">Protein <i>S</i>-guanylation by the biological signal 8-nitroguanosine 3′,5′-cyclic monophosphate</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">727</span>– <span class="NLM_lpage">735</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2007.33</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1038%2Fnchembio.2007.33" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=17906641" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFygs73F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2007&pages=727-735&author=T.+Sawaauthor=M.+H.+Zakiauthor=T.+Okamotoauthor=T.+Akutaauthor=Y.+Tokutomiauthor=S.+Kim-Mitsuyamaauthor=H.+Iharaauthor=A.+Kobayashiauthor=M.+Yamamotoauthor=S.+Fujiiauthor=H.+Arimotoauthor=T.+Akaike&title=Protein+S-guanylation+by+the+biological+signal+8-nitroguanosine+3%E2%80%B2%2C5%E2%80%B2-cyclic+monophosphate&doi=10.1038%2Fnchembio.2007.33"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit82R"><div class="casContent"><span class="casTitleNuber">82</span><div class="casTitle"><span class="NLM_cas:atitle">Protein S-guanylation by the biological signal 8-nitroguanosine 3',5'-cyclic monophosphate</span></div><div class="casAuthors">Sawa, Tomohiro; Zaki, Mohammad Hasan; Okamoto, Tatsuya; Akuta, Teruo; Tokutomi, Yoshiko; Kim-Mitsuyama, Shokei; Ihara, Hideshi; Kobayashi, Akira; Yamamoto, Masayuki; Fujii, Shigemoto; Arimoto, Hirokazu; Akaike, Takaaki</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">727-735</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The signaling pathway of nitric oxide (NO) depends mainly on guanosine 3',5'-cyclic monophosphate (cGMP).  Here we report the formation and chem. biol. of a nitrated deriv. of cGMP, 8-nitroguanosine 3',5'-cyclic monophosphate (8-nitro-cGMP), in NO-mediated signal transduction.  Immunocytochem. demonstrated marked 8-nitro-cGMP prodn. in various cultured cells in an NO-dependent manner.  This finding was confirmed by HPLC plus electrochem. detection and tandem mass spectrometry.  8-Nitro-cGMP activated cGMP-dependent protein kinase (PKG) and showed unique redox-active properties independent of cGMP activity.  Formation of protein Cys-cGMP adducts by 8-nitro-cGMP was identified as a new post-translational modification, which we call protein S-guanylation.  8-Nitro-cGMP seems to regulate the redox-sensor signaling protein Keap1, via S-guanylation of the highly nucleophilic cysteine sulfhydryls of Keap1.  This study reveals 8-nitro-cGMP to be a second messenger of NO and sheds light on new areas of the physiol. and chem. biol. of signal transduction by NO.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGruENgL603fArVg90H21EOLACvtfcHk0lj7J-VdIx00qA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFygs73F&md5=1915b89720a9bf353c63ebe621ac58bd</span></div><a href="/servlet/linkout?suffix=cit82&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2007.33&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2007.33%26sid%3Dliteratum%253Aachs%26aulast%3DSawa%26aufirst%3DT.%26aulast%3DZaki%26aufirst%3DM.%2BH.%26aulast%3DOkamoto%26aufirst%3DT.%26aulast%3DAkuta%26aufirst%3DT.%26aulast%3DTokutomi%26aufirst%3DY.%26aulast%3DKim-Mitsuyama%26aufirst%3DS.%26aulast%3DIhara%26aufirst%3DH.%26aulast%3DKobayashi%26aufirst%3DA.%26aulast%3DYamamoto%26aufirst%3DM.%26aulast%3DFujii%26aufirst%3DS.%26aulast%3DArimoto%26aufirst%3DH.%26aulast%3DAkaike%26aufirst%3DT.%26atitle%3DProtein%2520S-guanylation%2520by%2520the%2520biological%2520signal%25208-nitroguanosine%25203%25E2%2580%25B2%252C5%25E2%2580%25B2-cyclic%2520monophosphate%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2007%26volume%3D3%26spage%3D727%26epage%3D735%26doi%3D10.1038%2Fnchembio.2007.33" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref83"><div class="reference"><strong class="refLabel"><a href="#ref83" class="rightTabRefNumLink">83</a></strong><div class="NLM_citation" id="rightTab-cit83"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Takahashi, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arimoto, H.</span></span> <span> </span><span class="NLM_article-title">Targeting selective autophagy by AUTAC degraders</span>. <i>Autophagy</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">765</span>– <span class="NLM_lpage">766</span>, <span class="refDoi"> DOI: 10.1080/15548627.2020.1718362</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1080%2F15548627.2020.1718362" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=31958028" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BB3cXht1Kjurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2020&pages=765-766&author=D.+Takahashiauthor=H.+Arimoto&title=Targeting+selective+autophagy+by+AUTAC+degraders&doi=10.1080%2F15548627.2020.1718362"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit83R"><div class="casContent"><span class="casTitleNuber">83</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting selective autophagy by AUTAC degraders</span></div><div class="casAuthors">Takahashi, Daiki; Arimoto, Hirokazu</div><div class="citationInfo"><span class="NLM_cas:title">Autophagy</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">765-766</span>CODEN:
                <span class="NLM_cas:coden">AUTOC9</span>;
        ISSN:<span class="NLM_cas:issn">1554-8635</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">Targeted degrdn. is a promising new modality in drug discovery that makes it possible to reduce intracellular protein levels with small mols.  It is a complementary approach to the conventional protein knockdown typically used in labs. and may offer a way to approach the currently undruggable human proteome.  Recently, the first autophagy-mediated degraders, called AUTACs, were developed based on observations in a xenophagy study.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpRAzgVZeo_9bVg90H21EOLACvtfcHk0lhJDGxOrGhWRg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXht1Kjurg%253D&md5=b859e2173c1da6072a0662e58c8af80d</span></div><a href="/servlet/linkout?suffix=cit83&amp;dbid=16384&amp;doi=10.1080%2F15548627.2020.1718362&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F15548627.2020.1718362%26sid%3Dliteratum%253Aachs%26aulast%3DTakahashi%26aufirst%3DD.%26aulast%3DArimoto%26aufirst%3DH.%26atitle%3DTargeting%2520selective%2520autophagy%2520by%2520AUTAC%2520degraders%26jtitle%3DAutophagy%26date%3D2020%26volume%3D16%26spage%3D765%26epage%3D766%26doi%3D10.1080%2F15548627.2020.1718362" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref84"><div class="reference"><strong class="refLabel"><a href="#ref84" class="rightTabRefNumLink">84</a></strong><div class="NLM_citation" id="rightTab-cit84"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sin, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M. Q. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bornmann, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">The anti-angiogenic agent fumagillin covalently binds and inhibits the methionine aminopeptidase, MetAP-2</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>94</i></span>,  <span class="NLM_fpage">6099</span>– <span class="NLM_lpage">6103</span>, <span class="refDoi"> DOI: 10.1073/pnas.94.12.6099</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1073%2Fpnas.94.12.6099" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=9177176" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADyaK2sXjvFCnsb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=94&publication_year=1997&pages=6099-6103&author=N.+Sinauthor=L.+Mengauthor=M.+Q.+W.+Wangauthor=J.+J.+Wenauthor=W.+G.+Bornmannauthor=C.+M.+Crews&title=The+anti-angiogenic+agent+fumagillin+covalently+binds+and+inhibits+the+methionine+aminopeptidase%2C+MetAP-2&doi=10.1073%2Fpnas.94.12.6099"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit84R"><div class="casContent"><span class="casTitleNuber">84</span><div class="casTitle"><span class="NLM_cas:atitle">The anti-angiogenic agent fumagillin covalently binds and inhibits the methionine aminopeptidase, MetAP-2</span></div><div class="casAuthors">Sin, Ny; Meng, Lihao; Wang, Margaret Q. W.; Wen, James J.; Bornmann, William G.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">94</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">6099-6103</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The inhibition of new blood vessel formation (angiogenesis) is an effective means of limiting both the size and metastasis of solid tumors.  The leading antiangiogenic compd., TNP-470, has proven to be effective in in vitro and in animal model studies, and is currently being tested in phase III antitumor clin. trials.  Despite many detailed pharmacol. studies, little is known of the mol. mode of action of TNP-470.  Using a deriv. of the TNP-470 parent compd., the fungal metabolite, fumagillin, we have purified a mammalian protein that is selectively and covalently bound by this natural product.  This fumagillin binding protein was found to be a metalloprotease, methionine aminopeptidase (MetAP-2), that is highly conserved between human and Saccharomyces cerevisiae.  In the absence of MetAP-1, a distantly related methionine aminopeptidase, MetAP-2 function is essential for vegetative growth in yeast.  We demonstrate that fumagillin selectively inhibits the S. cerevisiae MetAP-2 protein in vivo.  The binding is highly specific as judged by the failure of fumagillin to inhibit MetAP-1 in vivo.  Hence, these results identify MetAP-2 as an important target of study in the anal. of the potent biol. activities of fumagillin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHTZpt2wUOq7Vg90H21EOLACvtfcHk0lhJDGxOrGhWRg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXjvFCnsb0%253D&md5=511f1751897559a60ccd9f814315c355</span></div><a href="/servlet/linkout?suffix=cit84&amp;dbid=16384&amp;doi=10.1073%2Fpnas.94.12.6099&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.94.12.6099%26sid%3Dliteratum%253Aachs%26aulast%3DSin%26aufirst%3DN.%26aulast%3DMeng%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DM.%2BQ.%2BW.%26aulast%3DWen%26aufirst%3DJ.%2BJ.%26aulast%3DBornmann%26aufirst%3DW.%2BG.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DThe%2520anti-angiogenic%2520agent%2520fumagillin%2520covalently%2520binds%2520and%2520inhibits%2520the%2520methionine%2520aminopeptidase%252C%2520MetAP-2%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1997%26volume%3D94%26spage%3D6099%26epage%3D6103%26doi%3D10.1073%2Fpnas.94.12.6099" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref85"><div class="reference"><strong class="refLabel"><a href="#ref85" class="rightTabRefNumLink">85</a></strong><div class="NLM_citation" id="rightTab-cit85"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tucker, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheppard, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lou, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKeegan, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lesniewski, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davidsen, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bell, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span> <span> </span><span class="NLM_article-title">Ectopic expression of methionine aminopeptidase-2 causes cell transformation and stimulates proliferation</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">3967</span>– <span class="NLM_lpage">3976</span>, <span class="refDoi"> DOI: 10.1038/onc.2008.14</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1038%2Fonc.2008.14" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=18264137" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BD1cXns1Onsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2008&pages=3967-3976&author=L.+A.+Tuckerauthor=Q.+Zhangauthor=G.+S.+Sheppardauthor=P.+Louauthor=F.+Jiangauthor=E.+McKeeganauthor=R.+Lesniewskiauthor=S.+K.+Davidsenauthor=R.+L.+Bellauthor=J.+Wang&title=Ectopic+expression+of+methionine+aminopeptidase-2+causes+cell+transformation+and+stimulates+proliferation&doi=10.1038%2Fonc.2008.14"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit85R"><div class="casContent"><span class="casTitleNuber">85</span><div class="casTitle"><span class="NLM_cas:atitle">Ectopic expression of methionine aminopeptidase-2 causes cell transformation and stimulates proliferation</span></div><div class="casAuthors">Tucker, L. A.; Zhang, Q.; Sheppard, G. S.; Lou, P.; Jiang, F.; McKeegan, E.; Lesniewski, R.; Davidsen, S. K.; Bell, R. L.; Wang, J.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">28</span>),
    <span class="NLM_cas:pages">3967-3976</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Methionine aminopeptidase-2 (MetAP2) processes N-terminal methionine from nascent cellular proteins.  Inhibition of MetAP2 has been shown to block angiogenesis and suppress tumor growth in preclin. tumor models.  However, the biol. role of MetAP2 in cancer is not well understood.  We examd. the effect of three distinct chem. classes of MetAP2 inhibitors on the growth of a panel of human cancer cells in vitro.  All MetAP2 inhibitors caused inhibition of tumor cell growth in both anchorage-dependent and, particularly, in anchorage-independent manner.  These data prompted us to examine the possible roles of MetAP2 in cancers.  Ectopic expression of MetAP2 in NIH-3T3 cells caused transformation, evidenced by the formation of foci in monolayer culture and growth of large colonies in soft agar.  Overexpression of MetAP2 in an immortalized bronchial epithelial cell line NL20 accelerated growth.  These phenotypes induced by the overexpression of MetAP2 were reversed by the treatment with MetAP2 inhibitors, indicating that the catalytic function of MetAP2 was essential.  Accordingly, overexpression of a catalytically inactive MetAP2 resulted in growth retardation of HT1080 tumor cells, suggesting a dominant-neg. role of the inactive MetAP2 mutant.  Finally, we analyzed the expression of MetAP2 in patient cancer samples by immunohistochem.  Moderate-to-high staining was identified in the majority of breast, colon, lung, ovarian and prostate carcinomas examd.  These data suggest that MetAP2 plays an important role in tumor cell growth and may contribute to tumorigenesis.  Oncogene (2008) 27, 3967-3976; doi:10.1038/onc.2008.14; published online 11 Feb. 2008.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobZn65gMTSUrVg90H21EOLACvtfcHk0lhJDGxOrGhWRg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXns1Onsr8%253D&md5=564ea8d498c043534100cf0a72136f4d</span></div><a href="/servlet/linkout?suffix=cit85&amp;dbid=16384&amp;doi=10.1038%2Fonc.2008.14&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fonc.2008.14%26sid%3Dliteratum%253Aachs%26aulast%3DTucker%26aufirst%3DL.%2BA.%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DSheppard%26aufirst%3DG.%2BS.%26aulast%3DLou%26aufirst%3DP.%26aulast%3DJiang%26aufirst%3DF.%26aulast%3DMcKeegan%26aufirst%3DE.%26aulast%3DLesniewski%26aufirst%3DR.%26aulast%3DDavidsen%26aufirst%3DS.%2BK.%26aulast%3DBell%26aufirst%3DR.%2BL.%26aulast%3DWang%26aufirst%3DJ.%26atitle%3DEctopic%2520expression%2520of%2520methionine%2520aminopeptidase-2%2520causes%2520cell%2520transformation%2520and%2520stimulates%2520proliferation%26jtitle%3DOncogene%26date%3D2008%26volume%3D27%26spage%3D3967%26epage%3D3976%26doi%3D10.1038%2Fonc.2008.14" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref86"><div class="reference"><strong class="refLabel"><a href="#ref86" class="rightTabRefNumLink">86</a></strong><div class="NLM_citation" id="rightTab-cit86"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Guruceaga, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perez-Cuesta, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abad-Diaz de Cerio, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gonzalez, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alonso, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hernando, F. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramirez-Garcia, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rementeria, A.</span></span> <span> </span><span class="NLM_article-title">Fumagillin, a mycotoxin of aspergillus fumigatus: biosynthesis, biological activities, detection, and applications</span>. <i>Toxins</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">7</span>, <span class="refDoi"> DOI: 10.3390/toxins12010007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.3390%2Ftoxins12010007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhslCgtb%252FK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2020&pages=7&author=X.+Guruceagaauthor=U.+Perez-Cuestaauthor=A.+Abad-Diaz+de+Cerioauthor=O.+Gonzalezauthor=R.+M.+Alonsoauthor=F.+L.+Hernandoauthor=A.+Ramirez-Garciaauthor=A.+Rementeria&title=Fumagillin%2C+a+mycotoxin+of+aspergillus+fumigatus%3A+biosynthesis%2C+biological+activities%2C+detection%2C+and+applications&doi=10.3390%2Ftoxins12010007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit86R"><div class="casContent"><span class="casTitleNuber">86</span><div class="casTitle"><span class="NLM_cas:atitle">Fumagillin, a mycotoxin of Aspergillus fumigatus: biosynthesis, biological activities, detection, and applications</span></div><div class="casAuthors">Guruceaga, Xabier; Perez-Cuesta, Uxue; de Cerio, Ana Abad-Diaz; Gonzalez, Oskar; Alonso, Rosa M.; Hernando, Fernando Luis; Ramirez-Garcia, Andoni; Rementeria, Aitor</div><div class="citationInfo"><span class="NLM_cas:title">Toxins</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">7</span>CODEN:
                <span class="NLM_cas:coden">TOXIB7</span>;
        ISSN:<span class="NLM_cas:issn">2072-6651</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Fumagillin is a mycotoxin produced, above all, by the saprophytic filamentous fungus Aspergillus fumigatus.  This mold is an opportunistic pathogen that can cause invasive aspergillosis, a disease that has high mortality rates linked to it.  Its ability to adapt to environmental stresses through the prodn. of secondary metabolites, including several mycotoxins (gliotoxin, fumagillin, pseurotin A, etc.) also seem to play an important role in causing these infections.  Since the discovery of the A. fumigatus fumagillin in 1949, many studies have focused on this toxin and in this review we gather all the information currently available.  First of all, the structural characteristics of this mycotoxin and the different methods developed for its detn. are given in detail.  Then, the biosynthetic gene cluster and the metabolic pathway involved in its prodn. and regulation are explained.  The activity of fumagillin on its target, the methionine aminopeptidase type 2 (MetAP2) enzyme, and the effects of blocking this enzyme in the host are also described.  Finally, the applications that this toxin and its derivs. have in different fields, such as the treatment of cancer and its microsporicidal activity in the treatment of honeybee hive infections with Nosema spp., are reviewed.  Therefore, this work offers a complete review of all the information currently related to the fumagillin mycotoxin secreted by A. fumigatus, important because of its role in the fungal infection process but also because it has many other applications, notably in beekeeping, the treatment of infectious diseases, and in oncol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkKqz-kSI757Vg90H21EOLACvtfcHk0liXB36nG7Y9wA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhslCgtb%252FK&md5=51fd0841db089c313716194ef926c8de</span></div><a href="/servlet/linkout?suffix=cit86&amp;dbid=16384&amp;doi=10.3390%2Ftoxins12010007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Ftoxins12010007%26sid%3Dliteratum%253Aachs%26aulast%3DGuruceaga%26aufirst%3DX.%26aulast%3DPerez-Cuesta%26aufirst%3DU.%26aulast%3DAbad-Diaz%2Bde%2BCerio%26aufirst%3DA.%26aulast%3DGonzalez%26aufirst%3DO.%26aulast%3DAlonso%26aufirst%3DR.%2BM.%26aulast%3DHernando%26aufirst%3DF.%2BL.%26aulast%3DRamirez-Garcia%26aufirst%3DA.%26aulast%3DRementeria%26aufirst%3DA.%26atitle%3DFumagillin%252C%2520a%2520mycotoxin%2520of%2520aspergillus%2520fumigatus%253A%2520biosynthesis%252C%2520biological%2520activities%252C%2520detection%252C%2520and%2520applications%26jtitle%3DToxins%26date%3D2020%26volume%3D12%26spage%3D7%26doi%3D10.3390%2Ftoxins12010007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref87"><div class="reference"><strong class="refLabel"><a href="#ref87" class="rightTabRefNumLink">87</a></strong><div class="NLM_citation" id="rightTab-cit87"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Heinrich, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seenisamy, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becker, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blume, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bomke, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dietz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eckert, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friese-Hamim, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gunera, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leuthner, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Musil, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pfalzgraf, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rohdich, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siegl, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spuck, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wegener, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zenke, F. T.</span></span> <span> </span><span class="NLM_article-title">Identification of methionine aminopeptidase-2 (MetAP-2) inhibitor M8891: a clinical compound for the treatment of cancer</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">11119</span>– <span class="NLM_lpage">11134</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b01070</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01070" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitFKksLbL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=11119-11134&author=T.+Heinrichauthor=J.+Seenisamyauthor=F.+Beckerauthor=B.+Blumeauthor=J.+Bomkeauthor=M.+Dietzauthor=U.+Eckertauthor=M.+Friese-Hamimauthor=J.+Guneraauthor=K.+Hansenauthor=B.+Leuthnerauthor=D.+Musilauthor=J.+Pfalzgrafauthor=F.+Rohdichauthor=C.+Sieglauthor=D.+Spuckauthor=A.+Wegenerauthor=F.+T.+Zenke&title=Identification+of+methionine+aminopeptidase-2+%28MetAP-2%29+inhibitor+M8891%3A+a+clinical+compound+for+the+treatment+of+cancer&doi=10.1021%2Facs.jmedchem.9b01070"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit87R"><div class="casContent"><span class="casTitleNuber">87</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of Methionine Aminopeptidase-2 (MetAP-2) Inhibitor M8891: A Clinical Compound for the Treatment of Cancer</span></div><div class="casAuthors">Heinrich, Timo; Seenisamy, Jeyaprakashnarayanan; Becker, Frank; Blume, Beatrix; Bomke, Joerg; Dietz, Melanie; Eckert, Uwe; Friese-Hamim, Manja; Gunera, Jakub; Hansen, Kerrin; Leuthner, Birgitta; Musil, Djordje; Pfalzgraf, Jens; Rohdich, Felix; Siegl, Christian; Spuck, Dieter; Wegener, Ansgar; Zenke, Frank T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">11119-11134</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The recently disclosed next generation of reversible, selective, and potent MetAP-2 inhibitors introduced a cyclic tartronic diamide scaffold.  However, the lead compd. 1a suffered from enterohepatic circulation, preventing further development.  Nevertheless, 1a served as a starting point for further optimization.  Maintaining potent antiproliferation activity, while improving other compd. properties, enabled the generation of an attractive array of new MetAP-2 inhibitors.  The most promising derivs. were identified by a multiparameter anal. of the compd. properties.  Essential for the efficient selection of candidates with in vivo activity was the identification of mols. with a long residence time on the target protein, high permeability, and low efflux ratio not only in Caco-2 but also in the MDR-MDCK cell line.  Orally bioavailable, potent, and reversible MetAP-2 inhibitors impede the growth of primary endothelial cells and demonstrated antitumoral activity in mouse models.  This assessment led to the nomination of the clin. development compd. M8891, which is currently in phase I clin. testing in oncol. patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoiUCw42raiq7Vg90H21EOLACvtfcHk0liXB36nG7Y9wA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitFKksLbL&md5=8a452bfe7918123546658355ff30a2e1</span></div><a href="/servlet/linkout?suffix=cit87&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01070&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01070%26sid%3Dliteratum%253Aachs%26aulast%3DHeinrich%26aufirst%3DT.%26aulast%3DSeenisamy%26aufirst%3DJ.%26aulast%3DBecker%26aufirst%3DF.%26aulast%3DBlume%26aufirst%3DB.%26aulast%3DBomke%26aufirst%3DJ.%26aulast%3DDietz%26aufirst%3DM.%26aulast%3DEckert%26aufirst%3DU.%26aulast%3DFriese-Hamim%26aufirst%3DM.%26aulast%3DGunera%26aufirst%3DJ.%26aulast%3DHansen%26aufirst%3DK.%26aulast%3DLeuthner%26aufirst%3DB.%26aulast%3DMusil%26aufirst%3DD.%26aulast%3DPfalzgraf%26aufirst%3DJ.%26aulast%3DRohdich%26aufirst%3DF.%26aulast%3DSiegl%26aufirst%3DC.%26aulast%3DSpuck%26aufirst%3DD.%26aulast%3DWegener%26aufirst%3DA.%26aulast%3DZenke%26aufirst%3DF.%2BT.%26atitle%3DIdentification%2520of%2520methionine%2520aminopeptidase-2%2520%2528MetAP-2%2529%2520inhibitor%2520M8891%253A%2520a%2520clinical%2520compound%2520for%2520the%2520treatment%2520of%2520cancer%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D11119%26epage%3D11134%26doi%3D10.1021%2Facs.jmedchem.9b01070" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref88"><div class="reference"><strong class="refLabel"><a href="#ref88" class="rightTabRefNumLink">88</a></strong><div class="NLM_citation" id="rightTab-cit88"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Heinrich, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seenisamy, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blume, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bomke, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calderini, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eckert, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friese-Hamim, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kohl, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lehmann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leuthner, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Musil, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rohdich, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zenke, F. T.</span></span> <span> </span><span class="NLM_article-title">Discovery and structure-based optimization of next-generation reversible methionine aminopeptidase-2 (MetAP-2) inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">5025</span>– <span class="NLM_lpage">5039</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b00041</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b00041" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC1MXms1aiu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=5025-5039&author=T.+Heinrichauthor=J.+Seenisamyauthor=B.+Blumeauthor=J.+Bomkeauthor=M.+Calderiniauthor=U.+Eckertauthor=M.+Friese-Hamimauthor=R.+Kohlauthor=M.+Lehmannauthor=B.+Leuthnerauthor=D.+Musilauthor=F.+Rohdichauthor=F.+T.+Zenke&title=Discovery+and+structure-based+optimization+of+next-generation+reversible+methionine+aminopeptidase-2+%28MetAP-2%29+inhibitors&doi=10.1021%2Facs.jmedchem.9b00041"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit88R"><div class="casContent"><span class="casTitleNuber">88</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and Structure-Based Optimization of Next-Generation Reversible Methionine Aminopeptidase-2 (MetAP-2) Inhibitors</span></div><div class="casAuthors">Heinrich, Timo; Seenisamy, Jeyaprakashnarayanan; Blume, Beatrix; Bomke, Joerg; Calderini, Michel; Eckert, Uwe; Friese-Hamim, Manja; Kohl, Rainer; Lehmann, Martin; Leuthner, Birgitta; Musil, Djordje; Rohdich, Felix; Zenke, Frank T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">5025-5039</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Co- and posttranslational processing are crucial maturation steps to generate functional proteins.  MetAP-2 plays an important role in this process and inhibition of its proteolytic activity has been shown to be important for angiogenesis and tumor growth suggesting that small mol. inhibitors of MetAP-2 may be promising options for the treatment of cancer.  This work describes the discovery and structure-based hit optimization of a novel MetAP-2 inhibitory scaffold.  Of crit. importance, a cyclic tartronic diamide coordinates the MetAP-2 metal ion in the active site while addnl. side chains of the mol. were designed to occupy the liphophilic methionine side chain recognition pocket as well as the shallow cavity at the opening of the active site.  The racemic screening hit from a HTS campaign I was discovered with an enzymic IC50 of 150 nM.  The resynthesized eutomer confirmed this activity and inhibited HUVEC proliferation with an IC50 of 1.9 μM.  Its structural anal. revealed a sophisticated interaction pattern of polar and lipophilic contacts that were used to improve cellular potency to an IC50 of 15 nM.  In parallel, the mol. properties were optimized on plasma exposure and antitumor efficacy which led to the identification of advanced lead II.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrsd8oNNAZALLVg90H21EOLACvtfcHk0liysOSnn5hsGw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXms1aiu7c%253D&md5=d3841fb334e4d75cc5050936c882fa2b</span></div><a href="/servlet/linkout?suffix=cit88&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b00041&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b00041%26sid%3Dliteratum%253Aachs%26aulast%3DHeinrich%26aufirst%3DT.%26aulast%3DSeenisamy%26aufirst%3DJ.%26aulast%3DBlume%26aufirst%3DB.%26aulast%3DBomke%26aufirst%3DJ.%26aulast%3DCalderini%26aufirst%3DM.%26aulast%3DEckert%26aufirst%3DU.%26aulast%3DFriese-Hamim%26aufirst%3DM.%26aulast%3DKohl%26aufirst%3DR.%26aulast%3DLehmann%26aufirst%3DM.%26aulast%3DLeuthner%26aufirst%3DB.%26aulast%3DMusil%26aufirst%3DD.%26aulast%3DRohdich%26aufirst%3DF.%26aulast%3DZenke%26aufirst%3DF.%2BT.%26atitle%3DDiscovery%2520and%2520structure-based%2520optimization%2520of%2520next-generation%2520reversible%2520methionine%2520aminopeptidase-2%2520%2528MetAP-2%2529%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D5025%26epage%3D5039%26doi%3D10.1021%2Facs.jmedchem.9b00041" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref89"><div class="reference"><strong class="refLabel"><a href="#ref89" class="rightTabRefNumLink">89</a></strong><div class="NLM_citation" id="rightTab-cit89"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Braun, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barglow, K. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Y. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akompong, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Briesewitz, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ray, G. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haldar, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wandless, T. J.</span></span> <span> </span><span class="NLM_article-title">A bifunctional molecule that displays context-dependent cellular activity</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>125</i></span>,  <span class="NLM_fpage">7575</span>– <span class="NLM_lpage">7580</span>, <span class="refDoi"> DOI: 10.1021/ja035176q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja035176q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BD3sXkt1Wgsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=125&publication_year=2003&pages=7575-7580&author=P.+D.+Braunauthor=K.+T.+Barglowauthor=Y.+M.+Linauthor=T.+Akompongauthor=R.+Briesewitzauthor=G.+T.+Rayauthor=K.+Haldarauthor=T.+J.+Wandless&title=A+bifunctional+molecule+that+displays+context-dependent+cellular+activity&doi=10.1021%2Fja035176q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit89R"><div class="casContent"><span class="casTitleNuber">89</span><div class="casTitle"><span class="NLM_cas:atitle">A Bifunctional Molecule That Displays Context-Dependent Cellular Activity</span></div><div class="casAuthors">Braun, Patrick D.; Barglow, Catherine T.; Lin, Yun-Ming; Akompong, Thomas; Briesewitz, Roger; Ray, Gregory T.; Haldar, Kasturi; Wandless, Thomas J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">125</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">7575-7580</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The cell-permeable dihydrofolate reductase inhibitor methotrexate was covalently linked to a ligand for the protein FKBP to create a bifunctional mol. called MTXSLF.  The covalent tether between the two ligands was designed to be prohibitively short, so that unfavorable protein-protein interactions between dihydrofolate reductase (DHFR) and FKBP preclude formation of a trimeric complex.  In vitro and in vivo expts. demonstrate that MTXSLF is an effective inhibitor of human DHFR, but that efficacy is decreased in the presence of human FKBP due to the high concn. of FKBP and its tight affinity for MTXSLF.  MTXSLF also inhibits Plasmodium falciparum DHFR in vitro, but a low concn. of the weaker binding Plasmodium FKBP has no effect on the inhibitory potency of MTXSLF in vivo.  These studies illustrate a potentially general strategy for modulating the biol. activity of synthetic mols. that depends on the ligand-binding properties of a nontarget protein.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHTfky2FXq0bVg90H21EOLACvtfcHk0liysOSnn5hsGw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXkt1Wgsbw%253D&md5=625076f0fc225f79bdeead5a3fd02293</span></div><a href="/servlet/linkout?suffix=cit89&amp;dbid=16384&amp;doi=10.1021%2Fja035176q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja035176q%26sid%3Dliteratum%253Aachs%26aulast%3DBraun%26aufirst%3DP.%2BD.%26aulast%3DBarglow%26aufirst%3DK.%2BT.%26aulast%3DLin%26aufirst%3DY.%2BM.%26aulast%3DAkompong%26aufirst%3DT.%26aulast%3DBriesewitz%26aufirst%3DR.%26aulast%3DRay%26aufirst%3DG.%2BT.%26aulast%3DHaldar%26aufirst%3DK.%26aulast%3DWandless%26aufirst%3DT.%2BJ.%26atitle%3DA%2520bifunctional%2520molecule%2520that%2520displays%2520context-dependent%2520cellular%2520activity%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2003%26volume%3D125%26spage%3D7575%26epage%3D7580%26doi%3D10.1021%2Fja035176q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref90"><div class="reference"><strong class="refLabel"><a href="#ref90" class="rightTabRefNumLink">90</a></strong><div class="NLM_citation" id="rightTab-cit90"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Somarelli, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herrera, R. J.</span></span> <span> </span><span class="NLM_article-title">Evolution of the 12 kDa FK506-binding protein gene</span>. <i>Biol. Cell.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>99</i></span>,  <span class="NLM_fpage">311</span>– <span class="NLM_lpage">321</span>, <span class="refDoi"> DOI: 10.1042/BC20060125</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref90/cit90&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1042%2FBC20060125" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref90/cit90&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=17309448" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref90/cit90&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BD2sXltl2ntrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=99&publication_year=2007&pages=311-321&author=J.+A.+Somarelliauthor=R.+J.+Herrera&title=Evolution+of+the+12+kDa+FK506-binding+protein+gene&doi=10.1042%2FBC20060125"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit90R"><div class="casContent"><span class="casTitleNuber">90</span><div class="casTitle"><span class="NLM_cas:atitle">Evolution of the 12kDa FK506-binding protein gene</span></div><div class="casAuthors">Somarelli, Jason A.; Herrera, Rene J.</div><div class="citationInfo"><span class="NLM_cas:title">Biology of the Cell</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">99</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">311-321</span>CODEN:
                <span class="NLM_cas:coden">BCELDF</span>;
        ISSN:<span class="NLM_cas:issn">0248-4900</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">The FKBPs (FK506-binding proteins) belong to a ubiquitous family of proteins that are found in a wide range of taxonomic groups.  These proteins participate in a variety of pathways, including protein folding, down-regulation of T-cell activation and inhibition of cell-cycle progression.  A cDNA for the 12kDa FKBP gene ortholog (FKBP12) in Bombyx mori was isolated from both Bm-5 cultured cells and silk-gland tissue.  Using the FKBP12 cDNA in combination with the B. mori 6× whole-genome shotgun database, we were able to identify the FKBP12 gene, as well as the positions of its intron-exon junctions.  FKBP12 exon sizes and intronic positions are highly conserved among FKBP12 orthologs in 24 diverse genomes.  Comparison of 41 FKBP12 genes revealed several intronic insertion and deletion events throughout evolution.  In addn., paralogous FKBP12 isoforms were identified in all 12 vertebrate genomes.  Both structural and phylogenetics analyses suggest that the isoforms may be evolving independently, possibly due to the distinct functional roles played by each paralog.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpzUWY5Ln2Iz7Vg90H21EOLACvtfcHk0liysOSnn5hsGw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXltl2ntrk%253D&md5=e586b381f607ec33a27b8a3aa018ec42</span></div><a href="/servlet/linkout?suffix=cit90&amp;dbid=16384&amp;doi=10.1042%2FBC20060125&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBC20060125%26sid%3Dliteratum%253Aachs%26aulast%3DSomarelli%26aufirst%3DJ.%2BA.%26aulast%3DHerrera%26aufirst%3DR.%2BJ.%26atitle%3DEvolution%2520of%2520the%252012%2520kDa%2520FK506-binding%2520protein%2520gene%26jtitle%3DBiol.%2520Cell.%26date%3D2007%26volume%3D99%26spage%3D311%26epage%3D321%26doi%3D10.1042%2FBC20060125" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref91"><div class="reference"><strong class="refLabel"><a href="#ref91" class="rightTabRefNumLink">91</a></strong><div class="NLM_citation" id="rightTab-cit91"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Galfre, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pitt, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Venturi, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sitsapesan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zaccai, N. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsaneva-Atanasova, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Neill, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sitsapesan, R.</span></span> <span> </span><span class="NLM_article-title">FKBP12 activates the cardiac ryanodine receptor Ca<sup>2+</sup>-release channel and is antagonised by FKBP12.6</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">e31956</span>, <span class="refDoi"> DOI: 10.1371/journal.pone.0031956</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1371%2Fjournal.pone.0031956" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=22363773" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC38XjsVaqsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&pages=e31956&author=E.+Galfreauthor=S.+J.+Pittauthor=E.+Venturiauthor=M.+Sitsapesanauthor=N.+R.+Zaccaiauthor=K.+Tsaneva-Atanasovaauthor=S.+O%E2%80%99Neillauthor=R.+Sitsapesan&title=FKBP12+activates+the+cardiac+ryanodine+receptor+Ca2%2B-release+channel+and+is+antagonised+by+FKBP12.6&doi=10.1371%2Fjournal.pone.0031956"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit91R"><div class="casContent"><span class="casTitleNuber">91</span><div class="casTitle"><span class="NLM_cas:atitle">FKBP12 activates the cardiac ryanodine receptor Ca2+-release channel and is antagonised by FKBP12.6</span></div><div class="casAuthors">Galfre, Elena; Pitt, Samantha J.; Venturi, Elisa; Sitsapesan, Mano; Zaccai, Nathan R.; Tsaneva-Atanasova, Krasimira; O'Neill, Stephen; Sitsapesan, Rebecca</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">e31956</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Changes in FKBP12.6 binding to cardiac ryanodine receptors (RyR2) are implicated in mediating disturbances in Ca2+-homeostasis in heart failure but there is controversy over the functional effects of FKBP12.6 on RyR2 channel gating.  We have therefore investigated the effects of FKBP12.6 and another structurally similar mol., FKBP12, which is far more abundant in heart, on the gating of single sheep RyR2 channels incorporated into planar phospholipid bilayers and on spontaneous waves of Ca2+-induced Ca2+-release in rat isolated permeabilized cardiac cells.  We demonstrate that FKBP12 is a high affinity activator of RyR2, sensitizing the channel to cytosolic Ca2+, whereas FKBP12.6 has very low efficacy, but can antagonize the effects of FKBP12.  Math. modeling of the data shows the importance of the relative concns. of FKBP12 and FKBP12.6 in detg. RyR2 activity.  Consistent with the single-channel results, physiol. concns. of FKBP12 (3 μM) increased Ca2+-wave frequency and decreased the SR Ca2+-content in cardiac cells.  FKBP12.6, itself, had no effect on wave frequency but antagonized the effects of FKBP12.  We provide a biophys. anal. of the mechanisms by which FK-binding proteins can regulate RyR2 single-channel gating.  Our data indicate that FKBP12, in addn. to FKBP12.6, may be important in regulating RyR2 function in the heart.  In heart failure, it is possible that an alteration in the dual regulation of RyR2 by FKBP12 and FKBP12.6 may occur.  This could contribute towards a higher RyR2 open probability, 'leaky' RyR2 channels and Ca2+-dependent arrhythmias.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpt_6Q8HmWQ3bVg90H21EOLACvtfcHk0lgZ1APReB09Yg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XjsVaqsLk%253D&md5=800d4fdfd59b6399a453569720a25bc6</span></div><a href="/servlet/linkout?suffix=cit91&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0031956&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0031956%26sid%3Dliteratum%253Aachs%26aulast%3DGalfre%26aufirst%3DE.%26aulast%3DPitt%26aufirst%3DS.%2BJ.%26aulast%3DVenturi%26aufirst%3DE.%26aulast%3DSitsapesan%26aufirst%3DM.%26aulast%3DZaccai%26aufirst%3DN.%2BR.%26aulast%3DTsaneva-Atanasova%26aufirst%3DK.%26aulast%3DO%25E2%2580%2599Neill%26aufirst%3DS.%26aulast%3DSitsapesan%26aufirst%3DR.%26atitle%3DFKBP12%2520activates%2520the%2520cardiac%2520ryanodine%2520receptor%2520Ca2%252B-release%2520channel%2520and%2520is%2520antagonised%2520by%2520FKBP12.6%26jtitle%3DPLoS%2520One%26date%3D2012%26volume%3D7%26spage%3De31956%26doi%3D10.1371%2Fjournal.pone.0031956" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref92"><div class="reference"><strong class="refLabel"><a href="#ref92" class="rightTabRefNumLink">92</a></strong><div class="NLM_citation" id="rightTab-cit92"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Small-molecule PROTAC degraders of the bromodomain and extra terminal (BET) proteins — a review</span>. <i>Drug Discovery Today: Technol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">43</span>– <span class="NLM_lpage">51</span>, <span class="refDoi"> DOI: 10.1016/j.ddtec.2019.04.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1016%2Fj.ddtec.2019.04.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=31200858" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A280%3ADC%252BB3M3mvFWrsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2019&pages=43-51&author=C.-Y.+Yangauthor=C.+Qinauthor=L.+Baiauthor=S.+Wang&title=Small-molecule+PROTAC+degraders+of+the+bromodomain+and+extra+terminal+%28BET%29+proteins+%E2%80%94+a+review&doi=10.1016%2Fj.ddtec.2019.04.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit92R"><div class="casContent"><span class="casTitleNuber">92</span><div class="casTitle"><span class="NLM_cas:atitle">Small-molecule PROTAC degraders of the Bromodomain and Extra Terminal (BET) proteins - A review</span></div><div class="casAuthors">Yang Chao-Yie; Qin Chong; Bai Longchuan; Wang Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today. Technologies</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">43-51</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The PROteolysis TArgeting Chimeric (PROTAC) concept has provided an opportunity for the discovery and development of a completely new type of therapy involving induction of protein degradation.  The BET proteins, comprised of BRD2, BRD3, BRD4 and the testis-specific BRDT protein, are epigenetic readers and master transcription coactivators.  Extremely potent and efficacious small-molecule PROTAC degraders of the BET proteins, based on available, potent and selective BET inhibitors, have been reported.  BET degraders differ from BET inhibitors in their cellular potency, phenotypic effects, pharmacokinetic properties and toxicity profiles.  Herein, we provide a review of BET degraders and the differential outcome observed in the cellular and animal models for BET degraders in comparison to BET inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQCRHissLLXoQvupmecNw_lfW6udTcc2ebhzoTltGSSOrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3M3mvFWrsA%253D%253D&md5=de5790c1db28d56717ff76374072ad81</span></div><a href="/servlet/linkout?suffix=cit92&amp;dbid=16384&amp;doi=10.1016%2Fj.ddtec.2019.04.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ddtec.2019.04.001%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DC.-Y.%26aulast%3DQin%26aufirst%3DC.%26aulast%3DBai%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DSmall-molecule%2520PROTAC%2520degraders%2520of%2520the%2520bromodomain%2520and%2520extra%2520terminal%2520%2528BET%2529%2520proteins%2520%25E2%2580%2594%2520a%2520review%26jtitle%3DDrug%2520Discovery%2520Today%253A%2520Technol.%26date%3D2019%26volume%3D31%26spage%3D43%26epage%3D51%26doi%3D10.1016%2Fj.ddtec.2019.04.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref93"><div class="reference"><strong class="refLabel"><a href="#ref93" class="rightTabRefNumLink">93</a></strong><div class="NLM_citation" id="rightTab-cit93"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Saraswat, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patki, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barot, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dukhande, V. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, K.</span></span> <span> </span><span class="NLM_article-title">Nanoformulation of proteolysis targeting chimera targeting ‘undruggable’ c-myc for the treatment of pancreatic cancer</span>. <i>Nanomedicine (London, U. K.)</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">1761</span>– <span class="NLM_lpage">1777</span>, <span class="refDoi"> DOI: 10.2217/nnm-2020-0156</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.2217%2Fnnm-2020-0156" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=32698663" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhsFOjurvM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2020&pages=1761-1777&author=A.+Saraswatauthor=M.+Patkiauthor=Y.+Fuauthor=S.+Barotauthor=V.+V.+Dukhandeauthor=K.+Patel&title=Nanoformulation+of+proteolysis+targeting+chimera+targeting+%E2%80%98undruggable%E2%80%99+c-myc+for+the+treatment+of+pancreatic+cancer&doi=10.2217%2Fnnm-2020-0156"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit93R"><div class="casContent"><span class="casTitleNuber">93</span><div class="casTitle"><span class="NLM_cas:atitle">Nanoformulation of PROteolysis TArgeting Chimera targeting 'undruggable' c-Myc for the treatment of pancreatic cancer</span></div><div class="casAuthors">Saraswat, Aishwarya; Patki, Manali; Fu, Yige; Barot, Shrikant; Dukhande, Vikas V.; Patel, Ketan</div><div class="citationInfo"><span class="NLM_cas:title">Nanomedicine (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">1761-1777</span>CODEN:
                <span class="NLM_cas:coden">NLUKAC</span>;
        ISSN:<span class="NLM_cas:issn">1748-6963</span>.
    
            (<span class="NLM_cas:orgname">Future Medicine Ltd.</span>)
        </div><div class="casAbstract">To explore the anticancer activity of a novel BRD4 protein degrader ARV-825 (ARV) and its nanoformulation development (ARV-NP) for treatment of pancreatic cancer.  ARV-NP were prepd. using nanopptn. method and characterized for their physicochem. properties and various anticancer cell culture assays.  ARV-NP (89.63 ± 16.39 nm) demonstrated good phys. stability, negligible hemolysis and improved half-life of ARV.  ARV-NP showed significant cytotoxicity, apoptosis and anticlonogenic effect in pancreatic cancer cells.  Significant downregulation of target proteins BRD4, c-Myc, Bcl-2 and upregulation of apoptotic marker cleaved caspase-3 was obsd.  Most importantly, ARV-NP treatment significantly inhibited the cell viability of 3D tumor spheroids of pancreatic cancer.  ARV-NP represents a novel therapeutic strategy for pancreatic cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpsLd0yKZiKT7Vg90H21EOLACvtfcHk0lgZ1APReB09Yg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhsFOjurvM&md5=209481e90fb02516b1f88756adce02ff</span></div><a href="/servlet/linkout?suffix=cit93&amp;dbid=16384&amp;doi=10.2217%2Fnnm-2020-0156&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2217%252Fnnm-2020-0156%26sid%3Dliteratum%253Aachs%26aulast%3DSaraswat%26aufirst%3DA.%26aulast%3DPatki%26aufirst%3DM.%26aulast%3DFu%26aufirst%3DY.%26aulast%3DBarot%26aufirst%3DS.%26aulast%3DDukhande%26aufirst%3DV.%2BV.%26aulast%3DPatel%26aufirst%3DK.%26atitle%3DNanoformulation%2520of%2520proteolysis%2520targeting%2520chimera%2520targeting%2520%25E2%2580%2598undruggable%25E2%2580%2599%2520c-myc%2520for%2520the%2520treatment%2520of%2520pancreatic%2520cancer%26jtitle%3DNanomedicine%2520%2528London%252C%2520U.%2520K.%2529%26date%3D2020%26volume%3D15%26spage%3D1761%26epage%3D1777%26doi%3D10.2217%2Fnnm-2020-0156" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref94"><div class="reference"><strong class="refLabel"><a href="#ref94" class="rightTabRefNumLink">94</a></strong><div class="NLM_citation" id="rightTab-cit94"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shafran, J. S.</span>; <span class="NLM_string-name">Jafari, N.</span>; <span class="NLM_string-name">Casey, A. N.</span>; <span class="NLM_string-name">Gyorffy, B.</span>; <span class="NLM_string-name">Denis, G. V.</span></span> <span> </span><span class="NLM_article-title">BRD4 regulates key transcription factors that drive epithelial-mesenchymal transition in castration-resistant prostate cancer</span>.  <i>Prostate Cancer Prostatic Dis.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>, <span class="refDoi"> DOI: 10.1038/s41391-020-0246-y</span> .</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1038%2Fs41391-020-0246-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=32690869" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2020&author=J.+S.+Shafran&author=N.+Jafari&author=A.+N.+Casey&author=B.+Gyorffy&author=G.+V.+Denis&title=BRD4+regulates+key+transcription+factors+that+drive+epithelial-mesenchymal+transition+in+castration-resistant+prostate+cancer&doi=10.1038%2Fs41391-020-0246-y"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit94&amp;dbid=16384&amp;doi=10.1038%2Fs41391-020-0246-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41391-020-0246-y%26sid%3Dliteratum%253Aachs%26aulast%3DShafran%26aufirst%3DJ.%2BS.%26atitle%3DBRD4%2520regulates%2520key%2520transcription%2520factors%2520that%2520drive%2520epithelial-mesenchymal%2520transition%2520in%2520castration-resistant%2520prostate%2520cancer%26jtitle%3DProstate%2520Cancer%2520Prostatic%2520Dis.%26date%3D2020%26doi%3D10.1038%2Fs41391-020-0246-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref95"><div class="reference"><strong class="refLabel"><a href="#ref95" class="rightTabRefNumLink">95</a></strong><div class="NLM_citation" id="rightTab-cit95"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Spriano, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stathis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertoni, F.</span></span> <span> </span><span class="NLM_article-title">Targeting BET bromodomain proteins in cancer: The example of lymphomas</span>. <i>Pharmacol. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>215</i></span>,  <span class="NLM_fpage">107631</span>, <span class="refDoi"> DOI: 10.1016/j.pharmthera.2020.107631</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref95/cit95&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1016%2Fj.pharmthera.2020.107631" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref95/cit95&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=32693114" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref95/cit95&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhsVyisrbI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=215&publication_year=2020&pages=107631&author=F.+Sprianoauthor=A.+Stathisauthor=F.+Bertoni&title=Targeting+BET+bromodomain+proteins+in+cancer%3A+The+example+of+lymphomas&doi=10.1016%2Fj.pharmthera.2020.107631"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit95R"><div class="casContent"><span class="casTitleNuber">95</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting BET bromodomain proteins in cancer: The example of lymphomas</span></div><div class="casAuthors">Spriano, Filippo; Stathis, Anastasios; Bertoni, Francesco</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacology & Therapeutics</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">215</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">107631</span>CODEN:
                <span class="NLM_cas:coden">PHTHDT</span>;
        ISSN:<span class="NLM_cas:issn">0163-7258</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">A review.  The Bromo- and Extra-Terminal domain (BET) family proteins act as ''readers'' of acetylated histones and they are important transcription regulators.  BRD2, BRD3, BRD4 and BRDT, part of the BET family, are important in different tumors, where upregulation or translocation often occurs.  The potential of targeting BET proteins as anti-cancer treatment originated with data obtained with a first series of compds., and there are now several data supporting BET inhibition in both solid tumors and hematol. malignancies.  Despite very pos. preclin. data in different tumor types, the clin. results have been so far moderate.  Using lymphoma as an example to review the data produced in the lab. and in the context of the early clin. trials, we discuss the modalities to make BET targeting more efficient both generating novel generation of compds. and by exploring the combination with small mols. affecting various signaling pathways, BCL2, or DNA damage response signaling, but also with addnl. epigenetic agents and with immunotherapy.  We also discuss the mechanisms of resistance and the toxicity profiles so far reported.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3Jn61Q24_wrVg90H21EOLACvtfcHk0liCpg57lnRb6w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhsVyisrbI&md5=cdb034e619849cb681f48c5b03f7cde8</span></div><a href="/servlet/linkout?suffix=cit95&amp;dbid=16384&amp;doi=10.1016%2Fj.pharmthera.2020.107631&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pharmthera.2020.107631%26sid%3Dliteratum%253Aachs%26aulast%3DSpriano%26aufirst%3DF.%26aulast%3DStathis%26aufirst%3DA.%26aulast%3DBertoni%26aufirst%3DF.%26atitle%3DTargeting%2520BET%2520bromodomain%2520proteins%2520in%2520cancer%253A%2520The%2520example%2520of%2520lymphomas%26jtitle%3DPharmacol.%2520Ther.%26date%3D2020%26volume%3D215%26spage%3D107631%26doi%3D10.1016%2Fj.pharmthera.2020.107631" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref96"><div class="reference"><strong class="refLabel"><a href="#ref96" class="rightTabRefNumLink">96</a></strong><div class="NLM_citation" id="rightTab-cit96"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roeder, R. G.</span></span> <span> </span><span class="NLM_article-title">The long and the short of BRD4: two tales in breast cancer</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>78</i></span>,  <span class="NLM_fpage">993</span>– <span class="NLM_lpage">995</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2020.05.033</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1016%2Fj.molcel.2020.05.033" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=32559427" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BB3cXht1alurrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=78&publication_year=2020&pages=993-995&author=S.+Zhangauthor=R.+G.+Roeder&title=The+long+and+the+short+of+BRD4%3A+two+tales+in+breast+cancer&doi=10.1016%2Fj.molcel.2020.05.033"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit96R"><div class="casContent"><span class="casTitleNuber">96</span><div class="casTitle"><span class="NLM_cas:atitle">The Long and the Short of BRD4: Two Tales in Breast Cancer</span></div><div class="casAuthors">Zhang, Sicong; Roeder, Robert G.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">78</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">993-995</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">A review.  In this issue of Mol. Cell, Wu et al. (2020) describe studies that establish oncogenic vs. tumor-suppressive functions of two BRD4 isoforms in the regulation of gene expression and breast cancer development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGotDF1hISm9V7Vg90H21EOLACvtfcHk0liCpg57lnRb6w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXht1alurrK&md5=9c06e499e9d1b87ca0491675a13b18df</span></div><a href="/servlet/linkout?suffix=cit96&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2020.05.033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2020.05.033%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DRoeder%26aufirst%3DR.%2BG.%26atitle%3DThe%2520long%2520and%2520the%2520short%2520of%2520BRD4%253A%2520two%2520tales%2520in%2520breast%2520cancer%26jtitle%3DMol.%2520Cell%26date%3D2020%26volume%3D78%26spage%3D993%26epage%3D995%26doi%3D10.1016%2Fj.molcel.2020.05.033" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref97"><div class="reference"><strong class="refLabel"><a href="#ref97" class="rightTabRefNumLink">97</a></strong><div class="NLM_citation" id="rightTab-cit97"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rupprecht, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Papadopoulos, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rammes, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baghai, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akula, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groyer, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schumacher, M.</span></span> <span> </span><span class="NLM_article-title">Translocator protein (18 kDa) (TSPO) as a therapeutic target for neurological and psychiatric disorders</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">971</span>– <span class="NLM_lpage">988</span>, <span class="refDoi"> DOI: 10.1038/nrd3295</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1038%2Fnrd3295" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=21119734" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsV2ju73E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2010&pages=971-988&author=R.+Rupprechtauthor=V.+Papadopoulosauthor=G.+Rammesauthor=T.+C.+Baghaiauthor=J.+Fanauthor=N.+Akulaauthor=G.+Groyerauthor=D.+Adamsauthor=M.+Schumacher&title=Translocator+protein+%2818+kDa%29+%28TSPO%29+as+a+therapeutic+target+for+neurological+and+psychiatric+disorders&doi=10.1038%2Fnrd3295"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit97R"><div class="casContent"><span class="casTitleNuber">97</span><div class="casTitle"><span class="NLM_cas:atitle">Translocator protein (18 kDa) (TSPO) as a therapeutic target for neurological and psychiatric disorders</span></div><div class="casAuthors">Rupprecht, Rainer; Papadopoulos, Vassilios; Rammes, Gerhard; Baghai, Thomas C.; Fan, Jinjiang; Akula, Nagaraju; Groyer, Ghislaine; Adams, David; Schumacher, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">971-988</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The translocator protein (18 kDa) (TSPO) is localized primarily in the outer mitochondrial membrane of steroid-synthesizing cells, including those in the central and peripheral nervous system.  One of its main functions is the transport of the substrate cholesterol into mitochondria, a prerequisite for steroid synthesis.  TSPO expression may constitute a biomarker of brain inflammation and reactive gliosis that could be monitored by using TSPO ligands as neuroimaging agents.  Moreover, initial clin. trials have indicated that TSPO ligands might be valuable in the treatment of neurol. and psychiatric disorders.  This Review focuses on the biol. and pathophysiol. of TSPO and the potential of currently available TSPO ligands for the diagnosis and treatment of neurol. and psychiatric disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqOS50xtO7oprVg90H21EOLACvtfcHk0liCpg57lnRb6w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsV2ju73E&md5=d2376d74a6ddd0eb795aa34d9e4df3f3</span></div><a href="/servlet/linkout?suffix=cit97&amp;dbid=16384&amp;doi=10.1038%2Fnrd3295&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3295%26sid%3Dliteratum%253Aachs%26aulast%3DRupprecht%26aufirst%3DR.%26aulast%3DPapadopoulos%26aufirst%3DV.%26aulast%3DRammes%26aufirst%3DG.%26aulast%3DBaghai%26aufirst%3DT.%2BC.%26aulast%3DFan%26aufirst%3DJ.%26aulast%3DAkula%26aufirst%3DN.%26aulast%3DGroyer%26aufirst%3DG.%26aulast%3DAdams%26aufirst%3DD.%26aulast%3DSchumacher%26aufirst%3DM.%26atitle%3DTranslocator%2520protein%2520%252818%2520kDa%2529%2520%2528TSPO%2529%2520as%2520a%2520therapeutic%2520target%2520for%2520neurological%2520and%2520psychiatric%2520disorders%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2010%26volume%3D9%26spage%3D971%26epage%3D988%26doi%3D10.1038%2Fnrd3295" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref98"><div class="reference"><strong class="refLabel"><a href="#ref98" class="rightTabRefNumLink">98</a></strong><div class="NLM_citation" id="rightTab-cit98"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xie, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yui, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hatori, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamasaki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumata, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wakizaka, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshida, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujinaga, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawamura, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, M.-R.</span></span> <span> </span><span class="NLM_article-title">Translocator protein (18kDa), a potential molecular imaging biomarker for non-invasively distinguishing non-alcoholic fatty liver disease</span>. <i>J. Hepatol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">1076</span>– <span class="NLM_lpage">1082</span>, <span class="refDoi"> DOI: 10.1016/j.jhep.2012.07.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1016%2Fj.jhep.2012.07.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=22796156" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFKlurvE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2012&pages=1076-1082&author=L.+Xieauthor=J.+Yuiauthor=A.+Hatoriauthor=T.+Yamasakiauthor=K.+Kumataauthor=H.+Wakizakaauthor=Y.+Yoshidaauthor=M.+Fujinagaauthor=K.+Kawamuraauthor=M.-R.+Zhang&title=Translocator+protein+%2818kDa%29%2C+a+potential+molecular+imaging+biomarker+for+non-invasively+distinguishing+non-alcoholic+fatty+liver+disease&doi=10.1016%2Fj.jhep.2012.07.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit98R"><div class="casContent"><span class="casTitleNuber">98</span><div class="casTitle"><span class="NLM_cas:atitle">Translocator protein (18 kDa), a potential molecular imaging biomarker for non-invasively distinguishing non-alcoholic fatty liver disease</span></div><div class="casAuthors">Xie, Lin; Yui, Joji; Hatori, Akiko; Yamasaki, Tomoteru; Kumata, Katsushi; Wakizaka, Hidekatsu; Yoshida, Yuichiro; Fujinaga, Masayuki; Kawamura, Kazunori; Zhang, Ming-Rong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Hepatology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1076-1082</span>CODEN:
                <span class="NLM_cas:coden">JOHEEC</span>;
        ISSN:<span class="NLM_cas:issn">0168-8278</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Background & Aims: Mitochondrial dysfunction is responsible for liver damage and disease progression in non-alc. fatty liver disease (NAFLD).  Translocator protein (18 kDa) (TSPO), a mitochondrial transmembrane protein, plays important roles in modulating mitochondrial function.  This study explored whether TSPO can be used as an imaging biomarker of non-invasive diagnosis and staging of NAFLD, monitored using positron emission tomog. (PET) with a TSPO radioligand [18F]FEDAC.  Methods: PET with [18F]FEDAC, non-enhanced computerized tomog. (CT), autoradiog., histopathol., and gene anal. were performed to evaluate and quantify TSPO levels and NAFLD progression in methionine and choline-deficient diet-fed mice.  Correlations were analyzed between uptake ratio of radioactivity and NAFLD activity score (NAS) in the liver.  Results: Uptake of [18F]FEDAC obviously increased with disease progression from simple steatosis to non-alc. steatohepatitis (NASH) (p <0.01).  A close correlation was identified between [18F]FEDAC uptake ratio and NAS in the liver (Pearson's r = 0.922, p = 0.000).  Specific binding of [18F]FEDAC to TSPO in the NAFLD livers was assessed in competition studies with the unlabeled TSPO-selective ligand PK11195.  Autoradiog. and histopathol. confirmed the PET imaging results.  Further, the mRNA levels of the functional macromol. signaling complex composed of TSPO were obviously higher compared to controls.  Conclusions: TSPO expression increases in NAFLD and closely correlates with NAFLD progression.  TSPO as a specific mol. imaging biomarker may open a novel avenue for non-invasive, reliable, and quant. diagnosis and staging of NAFLD.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoESltKjDYFkbVg90H21EOLACvtfcHk0lgx8hYiz9ACRA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFKlurvE&md5=9f99a7b3ef20b3e107260b7bc59703e0</span></div><a href="/servlet/linkout?suffix=cit98&amp;dbid=16384&amp;doi=10.1016%2Fj.jhep.2012.07.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jhep.2012.07.002%26sid%3Dliteratum%253Aachs%26aulast%3DXie%26aufirst%3DL.%26aulast%3DYui%26aufirst%3DJ.%26aulast%3DHatori%26aufirst%3DA.%26aulast%3DYamasaki%26aufirst%3DT.%26aulast%3DKumata%26aufirst%3DK.%26aulast%3DWakizaka%26aufirst%3DH.%26aulast%3DYoshida%26aufirst%3DY.%26aulast%3DFujinaga%26aufirst%3DM.%26aulast%3DKawamura%26aufirst%3DK.%26aulast%3DZhang%26aufirst%3DM.-R.%26atitle%3DTranslocator%2520protein%2520%252818kDa%2529%252C%2520a%2520potential%2520molecular%2520imaging%2520biomarker%2520for%2520non-invasively%2520distinguishing%2520non-alcoholic%2520fatty%2520liver%2520disease%26jtitle%3DJ.%2520Hepatol.%26date%3D2012%26volume%3D57%26spage%3D1076%26epage%3D1082%26doi%3D10.1016%2Fj.jhep.2012.07.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref99"><div class="reference"><strong class="refLabel"><a href="#ref99" class="rightTabRefNumLink">99</a></strong><div class="NLM_citation" id="rightTab-cit99"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulk, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Souza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhe-Paganon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">Phthalimide conjugation as a strategy for in vivo target protein degradation</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>348</i></span>,  <span class="NLM_fpage">1376</span>, <span class="refDoi"> DOI: 10.1126/science.aab1433</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1126%2Fscience.aab1433" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=25999370" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC2MXpvFansLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=348&publication_year=2015&pages=1376&author=G.+E.+Winterauthor=D.+L.+Buckleyauthor=J.+Paulkauthor=J.+M.+Robertsauthor=A.+Souzaauthor=S.+Dhe-Paganonauthor=J.+E.+Bradner&title=Phthalimide+conjugation+as+a+strategy+for+in+vivo+target+protein+degradation&doi=10.1126%2Fscience.aab1433"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit99R"><div class="casContent"><span class="casTitleNuber">99</span><div class="casTitle"><span class="NLM_cas:atitle">Phthalimide conjugation as a strategy for in vivo target protein degradation</span></div><div class="casAuthors">Winter, Georg E.; Buckley, Dennis L.; Paulk, Joshiawa; Roberts, Justin M.; Souza, Amanda; Dhe-Paganon, Sirano; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">348</span>
        (<span class="NLM_cas:issue">6241</span>),
    <span class="NLM_cas:pages">1376-1381</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The development of effective pharmacol. inhibitors of multidomain scaffold proteins, notably transcription factors, is a particularly challenging problem.  In part, this is because many small-mol. antagonists disrupt the activity of only one domain in the target protein.  The authors devised a chem. strategy that promotes ligand-dependent target protein degrdn. using as an example the transcriptional coactivator BRD4, a protein crit. for cancer cell growth and survival.  The authors appended a competitive antagonist of BET bromodomains to a phthalimide moiety to hijack the cereblon E3 ubiquitin ligase complex.  The resultant compd., dBET1, induced highly selective cereblon-dependent BET protein degrdn. in vitro and in vivo and delayed leukemia progression in mice.  A second series of probes resulted in selective degrdn. of the cytosolic protein FKBP12.  This chem. strategy for controlling target protein stability may have implications for therapeutically targeting previously intractable proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1ZW4zjRjXi7Vg90H21EOLACvtfcHk0lgx8hYiz9ACRA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXpvFansLk%253D&md5=b0277cc95d318d926b96e7a119066e3b</span></div><a href="/servlet/linkout?suffix=cit99&amp;dbid=16384&amp;doi=10.1126%2Fscience.aab1433&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aab1433%26sid%3Dliteratum%253Aachs%26aulast%3DWinter%26aufirst%3DG.%2BE.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DPaulk%26aufirst%3DJ.%26aulast%3DRoberts%26aufirst%3DJ.%2BM.%26aulast%3DSouza%26aufirst%3DA.%26aulast%3DDhe-Paganon%26aufirst%3DS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DPhthalimide%2520conjugation%2520as%2520a%2520strategy%2520for%2520in%2520vivo%2520target%2520protein%2520degradation%26jtitle%3DScience%26date%3D2015%26volume%3D348%26spage%3D1376%26doi%3D10.1126%2Fscience.aab1433" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref100"><div class="reference"><strong class="refLabel"><a href="#ref100" class="rightTabRefNumLink">100</a></strong><div class="NLM_citation" id="rightTab-cit100"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fei, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lau, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Landry, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baumgarth, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, X.</span></span> <span> </span><span class="NLM_article-title">Characterization of receptor binding profiles of influenza a viruses using an ellipsometry-based label-free glycan microarray assay platform</span>. <i>Biomolecules</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">1480</span>– <span class="NLM_lpage">1498</span>, <span class="refDoi"> DOI: 10.3390/biom5031480</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.3390%2Fbiom5031480" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=26193329" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1egs7vE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2015&pages=1480-1498&author=Y.+Feiauthor=Y.+S.+Sunauthor=Y.+Liauthor=H.+Yuauthor=K.+Lauauthor=J.+P.+Landryauthor=Z.+Luoauthor=N.+Baumgarthauthor=X.+Chenauthor=X.+Zhu&title=Characterization+of+receptor+binding+profiles+of+influenza+a+viruses+using+an+ellipsometry-based+label-free+glycan+microarray+assay+platform&doi=10.3390%2Fbiom5031480"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit100R"><div class="casContent"><span class="casTitleNuber">100</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of receptor binding profiles of influenza a viruses using an ellipsometry-based label-free glycan microarray assay platform</span></div><div class="casAuthors">Fei, Yiyan; Sun, Yung-Shin; Li, Yanhong; Yu, Hai; Lau, Kam; Landry, James P.; Luo, Zeng; Baumgarth, Nicole; Chen, Xi; Zhu, Xiangdong</div><div class="citationInfo"><span class="NLM_cas:title">Biomolecules</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1480-1498</span>CODEN:
                <span class="NLM_cas:coden">BIOMHC</span>;
        ISSN:<span class="NLM_cas:issn">2218-273X</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">A key step leading to influenza viral infection is the highly specific binding of a viral spike protein, hemagglutinin (HA), with an extracellular glycan receptor of a host cell.  Detailed and timely characterization of virus-receptor binding profiles may be used to evaluate and track the pandemic potential of an influenza virus strain.  We demonstrate a label-free glycan microarray assay platform for acquiring influenza virus binding profiles against a wide variety of glycan receptors.  By immobilizing biotinylated receptors on a streptavidin-functionalized solid surface, we measured binding curves of five influenza A virus strains with 24 glycans of diverse structures and used the apparent equil. dissocn. consts. (avidity consts., 10-100 pM) as characterizing parameters of viral receptor profiles.  Furthermore by measuring binding kinetic consts. of soln.-phase glycans to immobilized viruses, we confirmed that the glycan-HA affinity const. is in the range of 10 mM and the reaction is enthalpy-driven.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbRqRPEu0OirVg90H21EOLACvtfcHk0lgx8hYiz9ACRA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1egs7vE&md5=2b9a40222f727794475275c4cd8ecf7b</span></div><a href="/servlet/linkout?suffix=cit100&amp;dbid=16384&amp;doi=10.3390%2Fbiom5031480&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fbiom5031480%26sid%3Dliteratum%253Aachs%26aulast%3DFei%26aufirst%3DY.%26aulast%3DSun%26aufirst%3DY.%2BS.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DYu%26aufirst%3DH.%26aulast%3DLau%26aufirst%3DK.%26aulast%3DLandry%26aufirst%3DJ.%2BP.%26aulast%3DLuo%26aufirst%3DZ.%26aulast%3DBaumgarth%26aufirst%3DN.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DZhu%26aufirst%3DX.%26atitle%3DCharacterization%2520of%2520receptor%2520binding%2520profiles%2520of%2520influenza%2520a%2520viruses%2520using%2520an%2520ellipsometry-based%2520label-free%2520glycan%2520microarray%2520assay%2520platform%26jtitle%3DBiomolecules%26date%3D2015%26volume%3D5%26spage%3D1480%26epage%3D1498%26doi%3D10.3390%2Fbiom5031480" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref101"><div class="reference"><strong class="refLabel"><a href="#ref101" class="rightTabRefNumLink">101</a></strong><div class="NLM_citation" id="rightTab-cit101"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Landry, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fei, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ke, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, P.</span></span> <span> </span><span class="NLM_article-title">Discovering small molecule ligands of vascular endothelial growth factor that block VEGF-KDR binding using label-free microarray-based assays</span>. <i>Assay Drug Dev. Technol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">326</span>– <span class="NLM_lpage">332</span>, <span class="refDoi"> DOI: 10.1089/adt.2012.485</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1089%2Fadt.2012.485" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=23772553" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC3sXpsVOku7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2013&pages=326-332&author=J.+P.+Landryauthor=Y.+Feiauthor=X.+Zhuauthor=Y.+Keauthor=G.+Yuauthor=P.+Lee&title=Discovering+small+molecule+ligands+of+vascular+endothelial+growth+factor+that+block+VEGF-KDR+binding+using+label-free+microarray-based+assays&doi=10.1089%2Fadt.2012.485"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit101R"><div class="casContent"><span class="casTitleNuber">101</span><div class="casTitle"><span class="NLM_cas:atitle">Discovering Small Molecule Ligands of Vascular Endothelial Growth Factor That Block VEGF-KDR Binding Using Label-Free Microarray-Based Assays</span></div><div class="casAuthors">Landry, James P.; Fei, Yiyan; Zhu, Xiangdong; Ke, Yaohuang; Yu, Guoliang; Lee, Pierre</div><div class="citationInfo"><span class="NLM_cas:title">Assay and Drug Development Technologies</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">326-332</span>CODEN:
                <span class="NLM_cas:coden">ADDTAR</span>;
        ISSN:<span class="NLM_cas:issn">1540-658X</span>.
    
            (<span class="NLM_cas:orgname">Mary Ann Liebert, Inc.</span>)
        </div><div class="casAbstract">We present here a label-free microarray-based assay platform that we used to identify inhibitors of vascular endothelial growth factor (VEGF)-kinase-insertion domain receptor (KDR) binding.  Supported by a combination of special ellipsometry-based optical detection and small mol. microarrays (SMM), this platform consists of three assays: (1) the first assay detects binding of a target protein with SMM and identifies ligands to the protein as inhibitor candidates; (2) the second assay detects binding of a receptor protein with identical SMM and subsequent binding of the target protein (a sandwich assay) to identify the ligands to the receptor protein that do not interfere with the target-receptor binding; (3) the third assay detects binding of the target protein to the receptor protein in the presence of the ligands of the target protein identified from the first assay, with the receptor protein immobilized to a solid surface through the ligands identified in the second assay, to yield dose-response curves.  Using this platform, we screened 7,961 compds. from the National Cancer Institute and found 12 inhibitors to VEGF-KDR (VEGFR2) interactions with IC50 ranging from 0.3 to 60μM.  The inhibitory potency of these inhibitors found in the microarray-based assay was confirmed by their inhibition of VEGF-induced VEGFR2 phosphorylation in a cell-based assay.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr08LF_ToYfVrVg90H21EOLACvtfcHk0licufXRMHl1Tg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXpsVOku7Y%253D&md5=f6c946fb0e05d4e500e9111af0b6f4e6</span></div><a href="/servlet/linkout?suffix=cit101&amp;dbid=16384&amp;doi=10.1089%2Fadt.2012.485&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1089%252Fadt.2012.485%26sid%3Dliteratum%253Aachs%26aulast%3DLandry%26aufirst%3DJ.%2BP.%26aulast%3DFei%26aufirst%3DY.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DKe%26aufirst%3DY.%26aulast%3DYu%26aufirst%3DG.%26aulast%3DLee%26aufirst%3DP.%26atitle%3DDiscovering%2520small%2520molecule%2520ligands%2520of%2520vascular%2520endothelial%2520growth%2520factor%2520that%2520block%2520VEGF-KDR%2520binding%2520using%2520label-free%2520microarray-based%2520assays%26jtitle%3DAssay%2520Drug%2520Dev.%2520Technol.%26date%3D2013%26volume%3D11%26spage%3D326%26epage%3D332%26doi%3D10.1089%2Fadt.2012.485" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref102"><div class="reference"><strong class="refLabel"><a href="#ref102" class="rightTabRefNumLink">102</a></strong><div class="NLM_citation" id="rightTab-cit102"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Landry, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, X. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gregg, J. P.</span></span> <span> </span><span class="NLM_article-title">Label-free detection of microarrays of biomolecules by oblique-incidence reflectivity difference microscopy</span>. <i>Opt. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">581</span>– <span class="NLM_lpage">583</span>, <span class="refDoi"> DOI: 10.1364/OL.29.000581</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1364%2FOL.29.000581" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=15035477" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BD2cXislKrsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2004&pages=581-583&author=J.+P.+Landryauthor=X.+D.+Zhuauthor=J.+P.+Gregg&title=Label-free+detection+of+microarrays+of+biomolecules+by+oblique-incidence+reflectivity+difference+microscopy&doi=10.1364%2FOL.29.000581"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit102R"><div class="casContent"><span class="casTitleNuber">102</span><div class="casTitle"><span class="NLM_cas:atitle">Label-free detection of microarrays of biomolecules by oblique-incidence reflectivity difference microscopy</span></div><div class="casAuthors">Landry, J. P.; Zhu, X. D.; Gregg, J. P.</div><div class="citationInfo"><span class="NLM_cas:title">Optics Letters</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">581-583</span>CODEN:
                <span class="NLM_cas:coden">OPLEDP</span>;
        ISSN:<span class="NLM_cas:issn">0146-9592</span>.
    
            (<span class="NLM_cas:orgname">Optical Society of America</span>)
        </div><div class="casAbstract">We developed an oblique-incidence reflectivity difference (OI-RD) scanning microscope for label-free imaging of microarrays of biomols. upon solid substrates.  We demonstrate that hybridization reactions in an oligonucleotide microarray fabricated upon a glass slide can be detected by such an OI-RD microscope.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfxixOfT4iqrVg90H21EOLACvtfcHk0licufXRMHl1Tg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXislKrsrs%253D&md5=7c1bbadf6e7240e37803eb531761285f</span></div><a href="/servlet/linkout?suffix=cit102&amp;dbid=16384&amp;doi=10.1364%2FOL.29.000581&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1364%252FOL.29.000581%26sid%3Dliteratum%253Aachs%26aulast%3DLandry%26aufirst%3DJ.%2BP.%26aulast%3DZhu%26aufirst%3DX.%2BD.%26aulast%3DGregg%26aufirst%3DJ.%2BP.%26atitle%3DLabel-free%2520detection%2520of%2520microarrays%2520of%2520biomolecules%2520by%2520oblique-incidence%2520reflectivity%2520difference%2520microscopy%26jtitle%3DOpt.%2520Lett.%26date%3D2004%26volume%3D29%26spage%3D581%26epage%3D583%26doi%3D10.1364%2FOL.29.000581" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref103"><div class="reference"><strong class="refLabel"><a href="#ref103" class="rightTabRefNumLink">103</a></strong><div class="NLM_citation" id="rightTab-cit103"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fei, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jungblut, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pei, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dorhoi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaufmann, S. H. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ge, B.</span></span> <span> </span><span class="NLM_article-title">Nuclear cGAS suppresses DNA repair and promotes tumorigenesis</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>563</i></span>,  <span class="NLM_fpage">131</span>– <span class="NLM_lpage">136</span>, <span class="refDoi"> DOI: 10.1038/s41586-018-0629-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1038%2Fs41586-018-0629-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=30356214" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitVSitLrP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=563&publication_year=2018&pages=131-136&author=H.+Liuauthor=H.+Zhangauthor=X.+Wuauthor=D.+Maauthor=J.+Wuauthor=L.+Wangauthor=Y.+Jiangauthor=Y.+Feiauthor=C.+Zhuauthor=R.+Tanauthor=P.+Jungblutauthor=G.+Peiauthor=A.+Dorhoiauthor=Q.+Yanauthor=F.+Zhangauthor=R.+Zhengauthor=S.+Liuauthor=H.+Liangauthor=Z.+Liuauthor=H.+Yangauthor=J.+Chenauthor=P.+Wangauthor=T.+Tangauthor=W.+Pengauthor=Z.+Huauthor=Z.+Xuauthor=X.+Huangauthor=J.+Wangauthor=H.+Liauthor=Y.+Zhouauthor=F.+Liuauthor=D.+Yanauthor=S.+H.+E.+Kaufmannauthor=C.+Chenauthor=Z.+Maoauthor=B.+Ge&title=Nuclear+cGAS+suppresses+DNA+repair+and+promotes+tumorigenesis&doi=10.1038%2Fs41586-018-0629-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit103R"><div class="casContent"><span class="casTitleNuber">103</span><div class="casTitle"><span class="NLM_cas:atitle">Nuclear cGAS suppresses DNA repair and promotes tumorigenesis</span></div><div class="casAuthors">Liu, Haipeng; Zhang, Haiping; Wu, Xiangyang; Ma, Dapeng; Wu, Juehui; Wang, Lin; Jiang, Yan; Fei, Yiyan; Zhu, Chenggang; Tan, Rong; Jungblut, Peter; Pei, Gang; Dorhoi, Anca; Yan, Qiaoling; Zhang, Fan; Zheng, Ruijuan; Liu, Siyu; Liang, Haijiao; Liu, Zhonghua; Yang, Hua; Chen, Jianxia; Wang, Peng; Tang, Tianqi; Peng, Wenxia; Hu, Zhangsen; Xu, Zhu; Huang, Xiaochen; Wang, Jie; Li, Haohao; Zhou, Yilong; Liu, Feng; Yan, Dapeng; Kaufmann, Stefan H. E.; Chen, Chang; Mao, Zhiyong; Ge, Baoxue</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">563</span>
        (<span class="NLM_cas:issue">7729</span>),
    <span class="NLM_cas:pages">131-136</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Accurate repair of DNA double-stranded breaks by homologous recombination preserves genome integrity and inhibits tumorigenesis.  CGMP-AMP synthase (cGAS) is a cytosolic DNA sensor that activates innate immunity by initiating the STING-IRF3-type I IFN signaling cascade1,2.  Recognition of ruptured micronuclei by cGAS links genome instability to the innate immune response3,4, but the potential involvement of cGAS in DNA repair remains unknown.  Here we demonstrate that cGAS inhibits homologous recombination in mouse and human models.  DNA damage induces nuclear translocation of cGAS in a manner that is dependent on importin-α, and the phosphorylation of cGAS at tyrosine 215-mediated by B-lymphoid tyrosine kinase-facilitates the cytosolic retention of cGAS.  In the nucleus, cGAS is recruited to double-stranded breaks and interacts with PARP1 via poly(ADP-ribose).  The cGAS-PARP1 interaction impedes the formation of the PARP1-Timeless complex, and thereby suppresses homologous recombination.  We show that knockdown of cGAS suppresses DNA damage and inhibits tumor growth both in vitro and in vivo.  We conclude that nuclear cGAS suppresses homologous-recombination-mediated repair and promotes tumor growth, and that cGAS therefore represents a potential target for cancer prevention and therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_FFWfXc3dP7Vg90H21EOLACvtfcHk0ljCsvBs0ROi4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitVSitLrP&md5=d47b3675923877e00882c9cf3dcabae9</span></div><a href="/servlet/linkout?suffix=cit103&amp;dbid=16384&amp;doi=10.1038%2Fs41586-018-0629-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-018-0629-6%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DWu%26aufirst%3DX.%26aulast%3DMa%26aufirst%3DD.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DFei%26aufirst%3DY.%26aulast%3DZhu%26aufirst%3DC.%26aulast%3DTan%26aufirst%3DR.%26aulast%3DJungblut%26aufirst%3DP.%26aulast%3DPei%26aufirst%3DG.%26aulast%3DDorhoi%26aufirst%3DA.%26aulast%3DYan%26aufirst%3DQ.%26aulast%3DZhang%26aufirst%3DF.%26aulast%3DZheng%26aufirst%3DR.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DLiang%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DP.%26aulast%3DTang%26aufirst%3DT.%26aulast%3DPeng%26aufirst%3DW.%26aulast%3DHu%26aufirst%3DZ.%26aulast%3DXu%26aufirst%3DZ.%26aulast%3DHuang%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DYan%26aufirst%3DD.%26aulast%3DKaufmann%26aufirst%3DS.%2BH.%2BE.%26aulast%3DChen%26aufirst%3DC.%26aulast%3DMao%26aufirst%3DZ.%26aulast%3DGe%26aufirst%3DB.%26atitle%3DNuclear%2520cGAS%2520suppresses%2520DNA%2520repair%2520and%2520promotes%2520tumorigenesis%26jtitle%3DNature%26date%3D2018%26volume%3D563%26spage%3D131%26epage%3D136%26doi%3D10.1038%2Fs41586-018-0629-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref104"><div class="reference"><strong class="refLabel"><a href="#ref104" class="rightTabRefNumLink">104</a></strong><div class="NLM_citation" id="rightTab-cit104"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ge, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mi, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fei, Y.</span></span> <span> </span><span class="NLM_article-title">Fast focal point correction in prism-coupled total internal reflection scanning imager using an electronically tunable lens</span>. <i>Sensors</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">524</span>, <span class="refDoi"> DOI: 10.3390/s18020524</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref104/cit104&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.3390%2Fs18020524" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref104/cit104&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC1cXit1agtLbE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2018&pages=524&author=C.+Zhuauthor=B.+Geauthor=R.+Chenauthor=X.+Zhuauthor=L.+Miauthor=J.+Maauthor=X.+Wangauthor=F.+Zhengauthor=Y.+Fei&title=Fast+focal+point+correction+in+prism-coupled+total+internal+reflection+scanning+imager+using+an+electronically+tunable+lens&doi=10.3390%2Fs18020524"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit104R"><div class="casContent"><span class="casTitleNuber">104</span><div class="casTitle"><span class="NLM_cas:atitle">Fast focal point correction in prism-coupled total internal reflection scanning imager using an electronically tunable lens</span></div><div class="casAuthors">Zhu, Chenggang; Ge, Bilin; Chen, Ru; Zhu, Xiangdong; Mi, Lan; Ma, Jiong; Wang, Xu; Zheng, Fengyun; Fei, Yiyan</div><div class="citationInfo"><span class="NLM_cas:title">Sensors</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">524/1-524/10</span>CODEN:
                <span class="NLM_cas:coden">SENSC9</span>;
        ISSN:<span class="NLM_cas:issn">1424-8220</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Total internal reflection (TIR) is useful for interrogating phys. and chem. processes that occur at the interface between two transparent media.  Yet prism-coupled TIR imaging microscopes suffer from limited sensing areas due to the fact that the interface (the object plane) is not perpendicular to the optical axis of the microscope.  In this paper, we show that an elec. tunable lens can be used to rapidly and reproducibly correct the focal length of an oblique-incidence scanning microscope (OI-RD) in a prism-coupled TIR geometry.  We demonstrate the performance of such a correction by acquiring an image of a protein microarray over a scan area of 4 cm2 with an effective resoln. of less than 20 μ.  The electronic focal length tuning eliminates the mech. movement of the illumination lens in the scanning microscope and in turn the noise and background drift assocd. with the motion.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2FdcCHwpOXLVg90H21EOLACvtfcHk0ljCsvBs0ROi4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXit1agtLbE&md5=6c3c73884972ed2b1a306244b904b710</span></div><a href="/servlet/linkout?suffix=cit104&amp;dbid=16384&amp;doi=10.3390%2Fs18020524&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fs18020524%26sid%3Dliteratum%253Aachs%26aulast%3DZhu%26aufirst%3DC.%26aulast%3DGe%26aufirst%3DB.%26aulast%3DChen%26aufirst%3DR.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DMi%26aufirst%3DL.%26aulast%3DMa%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DZheng%26aufirst%3DF.%26aulast%3DFei%26aufirst%3DY.%26atitle%3DFast%2520focal%2520point%2520correction%2520in%2520prism-coupled%2520total%2520internal%2520reflection%2520scanning%2520imager%2520using%2520an%2520electronically%2520tunable%2520lens%26jtitle%3DSensors%26date%3D2018%26volume%3D18%26spage%3D524%26doi%3D10.3390%2Fs18020524" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref105"><div class="reference"><strong class="refLabel"><a href="#ref105" class="rightTabRefNumLink">105</a></strong><div class="NLM_citation" id="rightTab-cit105"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Landry, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gregg, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, X.</span></span> <span> </span><span class="NLM_article-title">Oblique-incidence reflectivity difference microscope for label-free high-throughput detection of biochemical reactions in a microarray format</span>. <i>Appl. Opt.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">1890</span>– <span class="NLM_lpage">1895</span>, <span class="refDoi"> DOI: 10.1364/AO.46.001890</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref105/cit105&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1364%2FAO.46.001890" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref105/cit105&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=17356635" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref105/cit105&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BD2sXks1alu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2007&pages=1890-1895&author=X.+Zhuauthor=J.+P.+Landryauthor=Y.-S.+Sunauthor=J.+P.+Greggauthor=K.+S.+Lamauthor=X.+Guo&title=Oblique-incidence+reflectivity+difference+microscope+for+label-free+high-throughput+detection+of+biochemical+reactions+in+a+microarray+format&doi=10.1364%2FAO.46.001890"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit105R"><div class="casContent"><span class="casTitleNuber">105</span><div class="casTitle"><span class="NLM_cas:atitle">Oblique-incidence reflectivity difference microscope for label-free high-throughput detection of biochemical reactions in a microarray format</span></div><div class="casAuthors">Zhu, Xiangdong; Landry, James P.; Sun, Yung-Shin; Gregg, Jeff P.; Lam, Kit S.; Guo, Xiaowen</div><div class="citationInfo"><span class="NLM_cas:title">Applied Optics</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1890-1895</span>CODEN:
                <span class="NLM_cas:coden">APOPAI</span>;
        ISSN:<span class="NLM_cas:issn">0003-6935</span>.
    
            (<span class="NLM_cas:orgname">Optical Society of America</span>)
        </div><div class="casAbstract">We describe a recently developed oblique-incidence reflectivity difference (OI-RD) microscope, a form of polarization-modulated imaging ellipsometer, for label-free-high-throughput detection of biomol. reactions on DNA and protein microarrays.  We present examples of application of this technique to end-point and real-time investigations of DNA-DNA hybridization, antibody-antigen capture, and protein-small-mol. binding reactions.  Compared to a conventional imaging ellipsometer based on the polarizer-compensator-sample-analyzer scheme and under the off-null condition, a polarization-modulated OI-RD microscope is inherently more sensitive by at least 1 order of magnitude to thickness changes on a solid surface.  Compared with imaging surface plasmon resonance microscopes based on reflectance change on falling or rising slopes of the surface plasmon resonance, the OI-RD microscope (1) has a comparable sensitivity, (2) is applicable to conventional microscope glass slides, and (3) easily covers a field of view as large as the entire surface of a 1 in. × 3 in. (2.54 cm × 7.62 cm) microscope slide.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr2VZofLGnTFrVg90H21EOLACvtfcHk0ljCsvBs0ROi4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXks1alu7w%253D&md5=c8de9e695ee3a79ba7d9c5027b93df5f</span></div><a href="/servlet/linkout?suffix=cit105&amp;dbid=16384&amp;doi=10.1364%2FAO.46.001890&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1364%252FAO.46.001890%26sid%3Dliteratum%253Aachs%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DLandry%26aufirst%3DJ.%2BP.%26aulast%3DSun%26aufirst%3DY.-S.%26aulast%3DGregg%26aufirst%3DJ.%2BP.%26aulast%3DLam%26aufirst%3DK.%2BS.%26aulast%3DGuo%26aufirst%3DX.%26atitle%3DOblique-incidence%2520reflectivity%2520difference%2520microscope%2520for%2520label-free%2520high-throughput%2520detection%2520of%2520biochemical%2520reactions%2520in%2520a%2520microarray%2520format%26jtitle%3DAppl.%2520Opt.%26date%3D2007%26volume%3D46%26spage%3D1890%26epage%3D1895%26doi%3D10.1364%2FAO.46.001890" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref106"><div class="reference"><strong class="refLabel"><a href="#ref106" class="rightTabRefNumLink">106</a></strong><div class="NLM_citation" id="rightTab-cit106"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baldo, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paganetti, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grueninger, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marcellin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaltenbach, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lo, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Semmelroth, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zivanovic, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abramowski, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lotz, G. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bates, G. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, A.</span></span> <span> </span><span class="NLM_article-title">TR-FRET-based duplex immunoassay reveals an inverse correlation of soluble and aggregated mutant huntingtin in huntington’s disease</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">264</span>– <span class="NLM_lpage">275</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2011.12.020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref106/cit106&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1016%2Fj.chembiol.2011.12.020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref106/cit106&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=22365609" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref106/cit106&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC38XjtFWhtLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2012&pages=264-275&author=B.+Baldoauthor=P.+Paganettiauthor=S.+Grueningerauthor=D.+Marcellinauthor=L.+S.+Kaltenbachauthor=D.+C.+Loauthor=M.+Semmelrothauthor=A.+Zivanovicauthor=D.+Abramowskiauthor=D.+Smithauthor=G.+P.+Lotzauthor=G.+P.+Batesauthor=A.+Weiss&title=TR-FRET-based+duplex+immunoassay+reveals+an+inverse+correlation+of+soluble+and+aggregated+mutant+huntingtin+in+huntington%E2%80%99s+disease&doi=10.1016%2Fj.chembiol.2011.12.020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit106R"><div class="casContent"><span class="casTitleNuber">106</span><div class="casTitle"><span class="NLM_cas:atitle">TR-FRET-Based Duplex Immunoassay Reveals an Inverse Correlation of Soluble and Aggregated Mutant huntingtin in Huntington's Disease</span></div><div class="casAuthors">Baldo, Barbara; Paganetti, Paolo; Grueninger, Stephan; Marcellin, David; Kaltenbach, Linda S.; Lo, Donald C.; Semmelroth, Martin; Zivanovic, Andjelija; Abramowski, Dorothee; Smith, Donna; Lotz, Gregor P.; Bates, Gillian P.; Weiss, Andreas</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">264-275</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Huntington's disease (HD) is an inherited neurodegenerative disorder caused by the amplification of a polyglutamine stretch at the N terminus of the huntingtin protein.  N-terminal fragments of the mutant huntingtin (mHtt) aggregate and form intracellular inclusions in brain and peripheral tissues.  Aggregates are an important hallmark of the disease, translating into a high need to quantify them in vitro and in vivo.  We developed a one-step TR-FRET-based immunoassay to quantify sol. and aggregated mHtt in cell and tissue homogenates.  Strikingly, quantification revealed a decrease of sol. mHtt correlating with an increase of aggregated protein in primary neuronal cell cultures, transgenic R6/2, and HdhQ150 knock-in HD mice.  These results emphasize the assay's efficiency for highly sensitive and quant. detection of sol. and aggregated mHtt and its application in high-throughput screening and characterization of HD models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHjXLeXbarcrVg90H21EOLACvtfcHk0ljqqsw3DBUhmg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XjtFWhtLo%253D&md5=ff1f0f82383793c05895bd86ec9cec3a</span></div><a href="/servlet/linkout?suffix=cit106&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2011.12.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2011.12.020%26sid%3Dliteratum%253Aachs%26aulast%3DBaldo%26aufirst%3DB.%26aulast%3DPaganetti%26aufirst%3DP.%26aulast%3DGrueninger%26aufirst%3DS.%26aulast%3DMarcellin%26aufirst%3DD.%26aulast%3DKaltenbach%26aufirst%3DL.%2BS.%26aulast%3DLo%26aufirst%3DD.%2BC.%26aulast%3DSemmelroth%26aufirst%3DM.%26aulast%3DZivanovic%26aufirst%3DA.%26aulast%3DAbramowski%26aufirst%3DD.%26aulast%3DSmith%26aufirst%3DD.%26aulast%3DLotz%26aufirst%3DG.%2BP.%26aulast%3DBates%26aufirst%3DG.%2BP.%26aulast%3DWeiss%26aufirst%3DA.%26atitle%3DTR-FRET-based%2520duplex%2520immunoassay%2520reveals%2520an%2520inverse%2520correlation%2520of%2520soluble%2520and%2520aggregated%2520mutant%2520huntingtin%2520in%2520huntington%25E2%2580%2599s%2520disease%26jtitle%3DChem.%2520Biol.%26date%3D2012%26volume%3D19%26spage%3D264%26epage%3D275%26doi%3D10.1016%2Fj.chembiol.2011.12.020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref107"><div class="reference"><strong class="refLabel"><a href="#ref107" class="rightTabRefNumLink">107</a></strong><div class="NLM_citation" id="rightTab-cit107"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abramowski, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bibel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bodner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chopra, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DiFiglia, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fox, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kegel, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grueninger, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hersch, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Housman, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Régulier, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosas, H. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stefani, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeitlin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bilbe, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paganetti, P.</span></span> <span> </span><span class="NLM_article-title">Single-step detection of mutant huntingtin in animal and human tissues: a bioassay for huntington’s disease</span>. <i>Anal. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>395</i></span>,  <span class="NLM_fpage">8</span>– <span class="NLM_lpage">15</span>, <span class="refDoi"> DOI: 10.1016/j.ab.2009.08.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1016%2Fj.ab.2009.08.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=19664996" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtFKnsb7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=395&publication_year=2009&pages=8-15&author=A.+Weissauthor=D.+Abramowskiauthor=M.+Bibelauthor=R.+Bodnerauthor=V.+Chopraauthor=M.+DiFigliaauthor=J.+Foxauthor=K.+Kegelauthor=C.+Kleinauthor=S.+Grueningerauthor=S.+Herschauthor=D.+Housmanauthor=E.+R%C3%A9gulierauthor=H.+D.+Rosasauthor=M.+Stefaniauthor=S.+Zeitlinauthor=G.+Bilbeauthor=P.+Paganetti&title=Single-step+detection+of+mutant+huntingtin+in+animal+and+human+tissues%3A+a+bioassay+for+huntington%E2%80%99s+disease&doi=10.1016%2Fj.ab.2009.08.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit107R"><div class="casContent"><span class="casTitleNuber">107</span><div class="casTitle"><span class="NLM_cas:atitle">Single-step detection of mutant huntingtin in animal and human tissues: A bioassay for Huntington's disease</span></div><div class="casAuthors">Weiss, Andreas; Abramowski, Dorothee; Bibel, Miriam; Bodner, Ruth; Chopra, Vanita; DiFiglia, Marian; Fox, Jonathan; Kegel, Kimberly; Klein, Corinna; Grueninger, Stephan; Hersch, Steven; Housman, David; Regulier, Etienne; Rosas, H. Diana; Stefani, Muriel; Zeitlin, Scott; Bilbe, Graeme; Paganetti, Paolo</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Biochemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">395</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">8-15</span>CODEN:
                <span class="NLM_cas:coden">ANBCA2</span>;
        ISSN:<span class="NLM_cas:issn">0003-2697</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The genetic mutation causing Huntington's disease is a polyglutamine expansion in the huntingtin protein where more than 37 glutamines cause disease by formation of toxic intracellular fragments, aggregates, and cell death.  Despite a clear pathogenic role for mutant huntingtin, understanding huntingtin expression during the presymptomatic phase of the disease or during disease progression has remained obscure.  Central to clarifying the role in the pathomechanism of disease is the ability to easily and accurately measure mutant huntingtin in accessible human tissue samples as well as cell and animal models.  Here the authors describe a highly sensitive time-resolved Forster resonance energy transfer (FRET) assay for quantification of sol. mutant huntingtin in brain, plasma, and cerebrospinal fluid.  Surprisingly, in mice, sol. huntingtin levels decrease during disease progression, inversely correlating with brain aggregate load.  Mutant huntingtin is easily detected in human brain and blood-derived fractions, providing a utility to assess mutant huntingtin expression during disease course as well as a pharmacodynamic marker for disease-modifying therapeutics targeting expression, cleavage, or degrdn. of mutant huntingtin.  The design of the homogeneous one-step method for huntingtin detection is such that it can be easily applied to measure other proteins of interest.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrla9646FajzrVg90H21EOLACvtfcHk0ljqqsw3DBUhmg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtFKnsb7I&md5=2fe224880e4761ec0566994b18353c8c</span></div><a href="/servlet/linkout?suffix=cit107&amp;dbid=16384&amp;doi=10.1016%2Fj.ab.2009.08.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ab.2009.08.001%26sid%3Dliteratum%253Aachs%26aulast%3DWeiss%26aufirst%3DA.%26aulast%3DAbramowski%26aufirst%3DD.%26aulast%3DBibel%26aufirst%3DM.%26aulast%3DBodner%26aufirst%3DR.%26aulast%3DChopra%26aufirst%3DV.%26aulast%3DDiFiglia%26aufirst%3DM.%26aulast%3DFox%26aufirst%3DJ.%26aulast%3DKegel%26aufirst%3DK.%26aulast%3DKlein%26aufirst%3DC.%26aulast%3DGrueninger%26aufirst%3DS.%26aulast%3DHersch%26aufirst%3DS.%26aulast%3DHousman%26aufirst%3DD.%26aulast%3DR%25C3%25A9gulier%26aufirst%3DE.%26aulast%3DRosas%26aufirst%3DH.%2BD.%26aulast%3DStefani%26aufirst%3DM.%26aulast%3DZeitlin%26aufirst%3DS.%26aulast%3DBilbe%26aufirst%3DG.%26aulast%3DPaganetti%26aufirst%3DP.%26atitle%3DSingle-step%2520detection%2520of%2520mutant%2520huntingtin%2520in%2520animal%2520and%2520human%2520tissues%253A%2520a%2520bioassay%2520for%2520huntington%25E2%2580%2599s%2520disease%26jtitle%3DAnal.%2520Biochem.%26date%3D2009%26volume%3D395%26spage%3D8%26epage%3D15%26doi%3D10.1016%2Fj.ab.2009.08.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref108"><div class="reference"><strong class="refLabel"><a href="#ref108" class="rightTabRefNumLink">108</a></strong><div class="NLM_citation" id="rightTab-cit108"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mizushima, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hatano, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobayashi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kabeya, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suzuki, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tokuhisa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohsumi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshimori, T.</span></span> <span> </span><span class="NLM_article-title">Dissection of autophagosome formation using Apg5-deficient mouse embryonic stem cells</span>. <i>J. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>152</i></span>,  <span class="NLM_fpage">657</span>– <span class="NLM_lpage">668</span>, <span class="refDoi"> DOI: 10.1083/jcb.152.4.657</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref108/cit108&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1083%2Fjcb.152.4.657" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref108/cit108&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=11266458" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref108/cit108&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BD3MXhtlGmtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=152&publication_year=2001&pages=657-668&author=N.+Mizushimaauthor=A.+Yamamotoauthor=M.+Hatanoauthor=Y.+Kobayashiauthor=Y.+Kabeyaauthor=K.+Suzukiauthor=T.+Tokuhisaauthor=Y.+Ohsumiauthor=T.+Yoshimori&title=Dissection+of+autophagosome+formation+using+Apg5-deficient+mouse+embryonic+stem+cells&doi=10.1083%2Fjcb.152.4.657"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit108R"><div class="casContent"><span class="casTitleNuber">108</span><div class="casTitle"><span class="NLM_cas:atitle">Dissection of autophagosome formation using Apg5-deficient mouse embryonic stem cells</span></div><div class="casAuthors">Mizushima, Noboru; Yamamoto, Akitsugu; Hatano, Masahiko; Kobayashi, Yoshinori; Kabeya, Yukiko; Suzuki, Kuninori; Tokuhisa, Takeshi; Ohsumi, Yoshinori; Yoshimori, Tamotsu</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cell Biology</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">152</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">657-667</span>CODEN:
                <span class="NLM_cas:coden">JCLBA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9525</span>.
    
            (<span class="NLM_cas:orgname">Rockefeller University Press</span>)
        </div><div class="casAbstract">In macroautophagy, cytoplasmic components are delivered to lysosomes for degrdn. via autophagosomes that are formed by closure of cupshaped isolation membranes.  However, how the isolation membranes are formed is poorly understood.  We recently found in yeast that a novel ubiquitin-like system, the Apg12-Apg5 conjugation system, is essential for autophagy.  Here we show that mouse Apg12-Apg5 conjugate localizes to the isolation membranes in mouse embryonic stem cells.  Using green fluorescent protein-tagged Apg5, we revealed that the cup-shaped isolation membrane is developed from a small crescent-shaped compartment.  Apg5 localizes on the isolation membrane throughout its elongation process.  To examine the role of Apg5, we generated Apg5-deficient embryonic stem cells, which showed defects in autophagosome formation.  The covalent modification of Apg5 with Apg12 is not required for its membrane targeting, but is essential for involvement of Apg5 in elongation of the isolation membranes.  We also show that Apg12-Apg5 is required for targeting of a mammalian Aut7/Apg8 homolog, LC3, to the isolation membranes.  These results suggest that the Apg12-Apg5 conjugate plays essential roles in isolation membrane development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoskuGRbi_Y5bVg90H21EOLACvtfcHk0ljevty1awHd8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXhtlGmtrY%253D&md5=99359e2ebf2fcd4b9476368362468247</span></div><a href="/servlet/linkout?suffix=cit108&amp;dbid=16384&amp;doi=10.1083%2Fjcb.152.4.657&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1083%252Fjcb.152.4.657%26sid%3Dliteratum%253Aachs%26aulast%3DMizushima%26aufirst%3DN.%26aulast%3DYamamoto%26aufirst%3DA.%26aulast%3DHatano%26aufirst%3DM.%26aulast%3DKobayashi%26aufirst%3DY.%26aulast%3DKabeya%26aufirst%3DY.%26aulast%3DSuzuki%26aufirst%3DK.%26aulast%3DTokuhisa%26aufirst%3DT.%26aulast%3DOhsumi%26aufirst%3DY.%26aulast%3DYoshimori%26aufirst%3DT.%26atitle%3DDissection%2520of%2520autophagosome%2520formation%2520using%2520Apg5-deficient%2520mouse%2520embryonic%2520stem%2520cells%26jtitle%3DJ.%2520Cell%2520Biol.%26date%3D2001%26volume%3D152%26spage%3D657%26epage%3D668%26doi%3D10.1083%2Fjcb.152.4.657" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref109"><div class="reference"><strong class="refLabel"><a href="#ref109" class="rightTabRefNumLink">109</a></strong><div class="NLM_citation" id="rightTab-cit109"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zoghbi, H. Y.</span></span> <span> </span><span class="NLM_article-title">Strategy to selectively remove mutant proteins could combat neurodegeneration</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>575</i></span>,  <span class="NLM_fpage">57</span>– <span class="NLM_lpage">58</span>, <span class="refDoi"> DOI: 10.1038/d41586-019-03243-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref109/cit109&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1038%2Fd41586-019-03243-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref109/cit109&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=31690850" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref109/cit109&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitFWntL%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=575&publication_year=2019&pages=57-58&author=H.+Y.+Zoghbi&title=Strategy+to+selectively+remove+mutant+proteins+could+combat+neurodegeneration&doi=10.1038%2Fd41586-019-03243-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit109R"><div class="casContent"><span class="casTitleNuber">109</span><div class="casTitle"><span class="NLM_cas:atitle">Strategy to selectively remove mutant proteins could combat neurodegeneration</span></div><div class="casAuthors">Zoghbi, Huda Y.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">575</span>
        (<span class="NLM_cas:issue">7781</span>),
    <span class="NLM_cas:pages">57-58</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Compds. have been found that reduce levels of the harmful protein present in Huntington's disease, without affecting the normal version.  The compds. interact with the mutated protein and the cell's protein-clearance machinery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTDIA6v3B88bVg90H21EOLACvtfcHk0ljevty1awHd8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitFWntL%252FE&md5=e61d603989cfad6519b0aa9edde5157f</span></div><a href="/servlet/linkout?suffix=cit109&amp;dbid=16384&amp;doi=10.1038%2Fd41586-019-03243-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fd41586-019-03243-7%26sid%3Dliteratum%253Aachs%26aulast%3DZoghbi%26aufirst%3DH.%2BY.%26atitle%3DStrategy%2520to%2520selectively%2520remove%2520mutant%2520proteins%2520could%2520combat%2520neurodegeneration%26jtitle%3DNature%26date%3D2019%26volume%3D575%26spage%3D57%26epage%3D58%26doi%3D10.1038%2Fd41586-019-03243-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref110"><div class="reference"><strong class="refLabel"><a href="#ref110" class="rightTabRefNumLink">110</a></strong><div class="NLM_citation" id="rightTab-cit110"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gennarino, V. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Maio, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jafar-Nejad, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">di Ronza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sayegh, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooper, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orr, H. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sillitoe, R. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zoghbi, H. Y.</span></span> <span> </span><span class="NLM_article-title">Pumilio1 haploinsufficiency leads to SCA1-like neurodegeneration by increasing wild-type ataxin1 levels</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>160</i></span>,  <span class="NLM_fpage">1087</span>– <span class="NLM_lpage">1098</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2015.02.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref110/cit110&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1016%2Fj.cell.2015.02.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref110/cit110&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=25768905" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref110/cit110&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC2MXks1agtbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=160&publication_year=2015&pages=1087-1098&author=V.+A.+Gennarinoauthor=R.+K.+Singhauthor=J.+J.+Whiteauthor=A.+De+Maioauthor=K.+Hanauthor=J.-Y.+Kimauthor=P.+Jafar-Nejadauthor=A.+di+Ronzaauthor=H.+Kangauthor=L.+S.+Sayeghauthor=T.+A.+Cooperauthor=H.+T.+Orrauthor=R.+V.+Sillitoeauthor=H.+Y.+Zoghbi&title=Pumilio1+haploinsufficiency+leads+to+SCA1-like+neurodegeneration+by+increasing+wild-type+ataxin1+levels&doi=10.1016%2Fj.cell.2015.02.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit110R"><div class="casContent"><span class="casTitleNuber">110</span><div class="casTitle"><span class="NLM_cas:atitle">Pumilio1 Haploinsufficiency Leads to SCA1-like Neurodegeneration by Increasing Wild-Type Ataxin1 Levels</span></div><div class="casAuthors">Gennarino, Vincenzo A.; Singh, Ravi K.; White, Joshua J.; De Maio, Antonia; Han, Kihoon; Kim, Ji-Yoen; Jafar-Nejad, Paymaan; di Ronza, Alberto; Kang, Hyojin; Sayegh, Layal S.; Cooper, Thomas A.; Orr, Harry T.; Sillitoe, Roy V.; Zoghbi, Huda Y.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">160</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1087-1098</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Spinocerebellar ataxia type 1 (SCA1) is a paradigmatic neurodegenerative proteinopathy, in which a mutant protein (in this case, ATAXIN1) accumulates in neurons and exerts toxicity; in SCA1, this process causes progressive deterioration of motor coordination.  Seeking to understand how post-translational modification of ATAXIN1 levels influences disease, we discovered that the RNA-binding protein PUMILIO1 (PUM1) not only directly regulates ATAXIN1 but also plays an unexpectedly important role in neuronal function.  Loss of Pum1 caused progressive motor dysfunction and SCA1-like neurodegeneration with motor impairment, primarily by increasing Ataxin1 levels.  Breeding Pum1+/- mice to SCA1 mice (Atxn1154Q/+) exacerbated disease progression, whereas breeding them to Atxn1+/- mice normalized Ataxin1 levels and largely rescued the Pum1+/- phenotype.  Thus, both increased wild-type ATAXIN1 levels and PUM1 haploinsufficiency could contribute to human neurodegeneration.  These results demonstrate the importance of studying post-transcriptional regulation of disease-driving proteins to reveal factors underlying neurodegenerative disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQKCmGA_iyjbVg90H21EOLACvtfcHk0ljevty1awHd8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXks1agtbc%253D&md5=45678b51683bd54ae255619f9d4b9f51</span></div><a href="/servlet/linkout?suffix=cit110&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2015.02.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2015.02.012%26sid%3Dliteratum%253Aachs%26aulast%3DGennarino%26aufirst%3DV.%2BA.%26aulast%3DSingh%26aufirst%3DR.%2BK.%26aulast%3DWhite%26aufirst%3DJ.%2BJ.%26aulast%3DDe%2BMaio%26aufirst%3DA.%26aulast%3DHan%26aufirst%3DK.%26aulast%3DKim%26aufirst%3DJ.-Y.%26aulast%3DJafar-Nejad%26aufirst%3DP.%26aulast%3Ddi%2BRonza%26aufirst%3DA.%26aulast%3DKang%26aufirst%3DH.%26aulast%3DSayegh%26aufirst%3DL.%2BS.%26aulast%3DCooper%26aufirst%3DT.%2BA.%26aulast%3DOrr%26aufirst%3DH.%2BT.%26aulast%3DSillitoe%26aufirst%3DR.%2BV.%26aulast%3DZoghbi%26aufirst%3DH.%2BY.%26atitle%3DPumilio1%2520haploinsufficiency%2520leads%2520to%2520SCA1-like%2520neurodegeneration%2520by%2520increasing%2520wild-type%2520ataxin1%2520levels%26jtitle%3DCell%26date%3D2015%26volume%3D160%26spage%3D1087%26epage%3D1098%26doi%3D10.1016%2Fj.cell.2015.02.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref111"><div class="reference"><strong class="refLabel"><a href="#ref111" class="rightTabRefNumLink">111</a></strong><div class="NLM_citation" id="rightTab-cit111"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Agarraberes, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dice, J. F.</span></span> <span> </span><span class="NLM_article-title">A molecular chaperone complex at the lysosomal membrane is required for protein translocation</span>. <i>J. Cell Sci.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>114</i></span>,  <span class="NLM_fpage">2491</span>– <span class="NLM_lpage">2499</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref111/cit111&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=11559757" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref111/cit111&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BD3MXlsFamtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=2001&pages=2491-2499&author=F.+A.+Agarraberesauthor=J.+F.+Dice&title=A+molecular+chaperone+complex+at+the+lysosomal+membrane+is+required+for+protein+translocation"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit111R"><div class="casContent"><span class="casTitleNuber">111</span><div class="casTitle"><span class="NLM_cas:atitle">A molecular chaperone complex at the lysosomal membrane is required for protein translocation</span></div><div class="casAuthors">Agarraberes, Fernando A.; Dice, J. Fred</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cell Science</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">114</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">2491-2499</span>CODEN:
                <span class="NLM_cas:coden">JNCSAI</span>;
        ISSN:<span class="NLM_cas:issn">0021-9533</span>.
    
            (<span class="NLM_cas:orgname">Company of Biologists Ltd.</span>)
        </div><div class="casAbstract">A group of cytosolic proteins are targeted to lysosomes for degrdn. in response to serum withdrawal or prolonged starvation by a process termed chaperone-mediated autophagy.  In this proteolytic pathway little is known about how proteins are translocated across lysosomal membranes.  We now show that an isoform of the constitutively expressed protein of the heat shock family of 70 kDa (Hsc70) is assocd. with the cytosolic side of the lysosomal membrane where it binds to substrates of this proteolytic pathway.  Results from coimmunopptn. and colocalization studies indicate that this mol. chaperone forms complexes with other mol. chaperones and cochaperones, including Hsp90, Hsp40, the Hsp70-Hsp90 organizing protein (Hop), the Hsp70-interacting protein (Hip), and the Bcl2-assocd. athanogene 1 protein (BAG-1).  Antibodies against Hip, Hop, Hsp40 and Hsc70 block transport of protein substrates into purified lysosomes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvrsExYbAmeLVg90H21EOLACvtfcHk0lg3-gy3xF3lVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXlsFamtb8%253D&md5=77407aede1ac38f1c02d716684c99397</span></div><a href="/servlet/linkout?suffix=cit111&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DAgarraberes%26aufirst%3DF.%2BA.%26aulast%3DDice%26aufirst%3DJ.%2BF.%26atitle%3DA%2520molecular%2520chaperone%2520complex%2520at%2520the%2520lysosomal%2520membrane%2520is%2520required%2520for%2520protein%2520translocation%26jtitle%3DJ.%2520Cell%2520Sci.%26date%3D2001%26volume%3D114%26spage%3D2491%26epage%3D2499" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref112"><div class="reference"><strong class="refLabel"><a href="#ref112" class="rightTabRefNumLink">112</a></strong><div class="NLM_citation" id="rightTab-cit112"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cuervo, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dice, J. F.</span></span> <span> </span><span class="NLM_article-title">A Receptor for the selective uptake and degradation of proteins by lysosomes</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>273</i></span>,  <span class="NLM_fpage">501</span>, <span class="refDoi"> DOI: 10.1126/science.273.5274.501</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref112/cit112&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1126%2Fscience.273.5274.501" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref112/cit112&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=8662539" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref112/cit112&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADyaK28Xks1Wmu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=273&publication_year=1996&pages=501&author=A.+M.+Cuervoauthor=J.+F.+Dice&title=A+Receptor+for+the+selective+uptake+and+degradation+of+proteins+by+lysosomes&doi=10.1126%2Fscience.273.5274.501"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit112R"><div class="casContent"><span class="casTitleNuber">112</span><div class="casTitle"><span class="NLM_cas:atitle">A receptor for the selective uptake and degradation of proteins by lysosomes</span></div><div class="casAuthors">Cuervo, Ana Maria; Dice, J. Fred</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, D. C.)</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">273</span>
        (<span class="NLM_cas:issue">5274</span>),
    <span class="NLM_cas:pages">501-503</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Multiple pathways of protein degrdn. operate within cells.  A selective protein import pathway exists for the uptake and degrdn. of particular cytosolic proteins by lysosomes.  Here, the lysosomal membrane glycoprotein LGP96 was identified as a receptor for the selective import and degrdn. of proteins within lysosomes.  Specific substrates of this proteolytic pathway bound to the cytosolic tail of a 96-kilodalton lysosomal membrane protein in two different binding assays.  Overexpression of human LGP96 in Chinese hamster ovary cells increased the activity of the selective lysosomal proteolytic pathway in vivo and in vitro.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpN-TFxOJ9p17Vg90H21EOLACvtfcHk0lg3-gy3xF3lVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28Xks1Wmu7o%253D&md5=45dd167e6aedfc1143633026fef176c9</span></div><a href="/servlet/linkout?suffix=cit112&amp;dbid=16384&amp;doi=10.1126%2Fscience.273.5274.501&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.273.5274.501%26sid%3Dliteratum%253Aachs%26aulast%3DCuervo%26aufirst%3DA.%2BM.%26aulast%3DDice%26aufirst%3DJ.%2BF.%26atitle%3DA%2520Receptor%2520for%2520the%2520selective%2520uptake%2520and%2520degradation%2520of%2520proteins%2520by%2520lysosomes%26jtitle%3DScience%26date%3D1996%26volume%3D273%26spage%3D501%26doi%3D10.1126%2Fscience.273.5274.501" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref113"><div class="reference"><strong class="refLabel"><a href="#ref113" class="rightTabRefNumLink">113</a></strong><div class="NLM_citation" id="rightTab-cit113"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kaushik, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuervo, A. M.</span></span> <span> </span><span class="NLM_article-title">Chaperone-mediated autophagy: a unique way to enter the lysosome world</span>. <i>Trends Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">407</span>– <span class="NLM_lpage">417</span>, <span class="refDoi"> DOI: 10.1016/j.tcb.2012.05.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref113/cit113&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1016%2Fj.tcb.2012.05.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref113/cit113&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=22748206" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref113/cit113&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtFaks7%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2012&pages=407-417&author=S.+Kaushikauthor=A.+M.+Cuervo&title=Chaperone-mediated+autophagy%3A+a+unique+way+to+enter+the+lysosome+world&doi=10.1016%2Fj.tcb.2012.05.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit113R"><div class="casContent"><span class="casTitleNuber">113</span><div class="casTitle"><span class="NLM_cas:atitle">Chaperone-mediated autophagy: a unique way to enter the lysosome world</span></div><div class="casAuthors">Kaushik, Susmita; Cuervo, Ana Maria</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Cell Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">407-417</span>CODEN:
                <span class="NLM_cas:coden">TCBIEK</span>;
        ISSN:<span class="NLM_cas:issn">0962-8924</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  All cellular proteins undergo continuous synthesis and degrdn.  This permanent renewal is necessary to maintain a functional proteome and to allow rapid changes in levels of specific proteins with regulatory purposes.  Although for a long time lysosomes were considered unable to contribute to the selective degrdn. of individual proteins, the discovery of chaperone-mediated autophagy (CMA) changed this notion.  Here, we review the characteristics that set CMA apart from other types of lysosomal degrdn. and the subset of mols. that confer cells the capability to identify individual cytosolic proteins and direct them across the lysosomal membrane for degrdn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2-Hs13dlknrVg90H21EOLACvtfcHk0lg3-gy3xF3lVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtFaks7%252FI&md5=49b2a4e8b0bc6fe127396b8c451af649</span></div><a href="/servlet/linkout?suffix=cit113&amp;dbid=16384&amp;doi=10.1016%2Fj.tcb.2012.05.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tcb.2012.05.006%26sid%3Dliteratum%253Aachs%26aulast%3DKaushik%26aufirst%3DS.%26aulast%3DCuervo%26aufirst%3DA.%2BM.%26atitle%3DChaperone-mediated%2520autophagy%253A%2520a%2520unique%2520way%2520to%2520enter%2520the%2520lysosome%2520world%26jtitle%3DTrends%2520Cell%2520Biol.%26date%3D2012%26volume%3D22%26spage%3D407%26epage%3D417%26doi%3D10.1016%2Fj.tcb.2012.05.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref114"><div class="reference"><strong class="refLabel"><a href="#ref114" class="rightTabRefNumLink">114</a></strong><div class="NLM_citation" id="rightTab-cit114"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nagai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujikake, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohno, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Higashiyama, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Popiel, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rahadian, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamaguchi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strittmatter, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burke, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toda, T.</span></span> <span> </span><span class="NLM_article-title">Prevention of polyglutamine oligomerization and neurodegeneration by the peptide inhibitor QBP1 in drosophila</span>. <i>Hum. Mol. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">1253</span>– <span class="NLM_lpage">1259</span>, <span class="refDoi"> DOI: 10.1093/hmg/ddg144</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref114/cit114&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1093%2Fhmg%2Fddg144" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref114/cit114&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=12761040" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref114/cit114&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BD3sXkt1Wru7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2003&pages=1253-1259&author=Y.+Nagaiauthor=N.+Fujikakeauthor=K.+Ohnoauthor=H.+Higashiyamaauthor=H.+A.+Popielauthor=J.+Rahadianauthor=M.+Yamaguchiauthor=W.+J.+Strittmatterauthor=J.+R.+Burkeauthor=T.+Toda&title=Prevention+of+polyglutamine+oligomerization+and+neurodegeneration+by+the+peptide+inhibitor+QBP1+in+drosophila&doi=10.1093%2Fhmg%2Fddg144"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit114R"><div class="casContent"><span class="casTitleNuber">114</span><div class="casTitle"><span class="NLM_cas:atitle">Prevention of polyglutamine oligomerization and neurodegeneration by the peptide inhibitor QBP1 in Drosophila</span></div><div class="casAuthors">Nagai, Yoshitaka; Fujikake, Nobuhiro; Ohno, Katsuhito; Higashiyama, Hiroyuki; Popiel, Helena A.; Rahadian, Julia; Yamaguchi, Masamitsu; Strittmatter, Warren J.; Burke, James R.; Toda, Tatsushi</div><div class="citationInfo"><span class="NLM_cas:title">Human Molecular Genetics</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1253-1259</span>CODEN:
                <span class="NLM_cas:coden">HMGEE5</span>;
        ISSN:<span class="NLM_cas:issn">0964-6906</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Polyglutamine (polyQ) diseases are a growing class of inherited neurodegenerative diseases including Huntington's disease, which are caused by abnormal expansions of the polyQ stretch in each unrelated disease protein.  The expanded polyQ stretch is thought to confer toxic properties on the disease proteins through alteration of their conformation leading to pathogenic protein-protein interactions including oligomerization and/or aggregation.  Hypothesizing that mols. with selective binding affinity to the expanded polyQ stretch may interfere with the pathogenic properties, we previously identified Polyglutamine Binding Peptide 1 (QBP1) from combinatorial peptide phage display libraries.  We show here that a tandem repeat of the inhibitor peptide QBP1, (QBP1)2, significantly suppresses polyQ aggregation and polyQ-induced neurodegeneration in the compd. eye of Drosophila polyQ disease models, which express the expanded polyQ protein under the eye specific promoter.  Most importantly, (QBP1)2 expression dramatically rescues premature death of flies expressing the expanded polyQ protein in the nervous system, resulting in the dramatic increase of the median life span from 5.5 to 52 days.  These results suggest that QBP1 can prevent polyQ-induced neurodegeneration in vivo.  We propose that QBP1 prevents polyQ oligomerization and/or aggregation either by altering the toxic conformation of the expanded polyQ stretch, or by simply competing with the expanded polyQ stretches for binding to other expanded polyQ proteins.  The peptide inhibitor QBP1 is a promising candidate with great potential as a therapeutic mol. against the currently untreatable polyQ diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp0TkGk7B7Wv7Vg90H21EOLACvtfcHk0ljgGvv0TzKBww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXkt1Wru7s%253D&md5=cfd7e7b12698f6d083868190e22ccb42</span></div><a href="/servlet/linkout?suffix=cit114&amp;dbid=16384&amp;doi=10.1093%2Fhmg%2Fddg144&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fhmg%252Fddg144%26sid%3Dliteratum%253Aachs%26aulast%3DNagai%26aufirst%3DY.%26aulast%3DFujikake%26aufirst%3DN.%26aulast%3DOhno%26aufirst%3DK.%26aulast%3DHigashiyama%26aufirst%3DH.%26aulast%3DPopiel%26aufirst%3DH.%2BA.%26aulast%3DRahadian%26aufirst%3DJ.%26aulast%3DYamaguchi%26aufirst%3DM.%26aulast%3DStrittmatter%26aufirst%3DW.%2BJ.%26aulast%3DBurke%26aufirst%3DJ.%2BR.%26aulast%3DToda%26aufirst%3DT.%26atitle%3DPrevention%2520of%2520polyglutamine%2520oligomerization%2520and%2520neurodegeneration%2520by%2520the%2520peptide%2520inhibitor%2520QBP1%2520in%2520drosophila%26jtitle%3DHum.%2520Mol.%2520Genet.%26date%3D2003%26volume%3D12%26spage%3D1253%26epage%3D1259%26doi%3D10.1093%2Fhmg%2Fddg144" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref115"><div class="reference"><strong class="refLabel"><a href="#ref115" class="rightTabRefNumLink">115</a></strong><div class="NLM_citation" id="rightTab-cit115"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nagai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tucker, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kenan, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henderson, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keene, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strittmatter, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burke, J. R.</span></span> <span> </span><span class="NLM_article-title">Inhibition of polyglutamine protein aggregation and cell death by novel peptides identified by phage display screening</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>275</i></span>,  <span class="NLM_fpage">10437</span>– <span class="NLM_lpage">10442</span>, <span class="refDoi"> DOI: 10.1074/jbc.275.14.10437</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref115/cit115&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1074%2Fjbc.275.14.10437" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref115/cit115&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10744733" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref115/cit115&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BD3cXisFWjt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=275&publication_year=2000&pages=10437-10442&author=Y.+Nagaiauthor=T.+Tuckerauthor=H.+Renauthor=D.+J.+Kenanauthor=B.+S.+Hendersonauthor=J.+D.+Keeneauthor=W.+J.+Strittmatterauthor=J.+R.+Burke&title=Inhibition+of+polyglutamine+protein+aggregation+and+cell+death+by+novel+peptides+identified+by+phage+display+screening&doi=10.1074%2Fjbc.275.14.10437"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit115R"><div class="casContent"><span class="casTitleNuber">115</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of polyglutamine protein aggregation and cell death by novel peptides identified by phage display screening</span></div><div class="casAuthors">Nagai, Yoshitaka; Tucker, Timothy; Ren, Hongzu; Kenan, Daniel J.; Henderson, Barry S.; Keene, Jack D.; Strittmatter, Warren J.; Burke, James R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">275</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">10437-10442</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Proteins with expanded polyglutamine domains cause eight inherited neurodegenerative diseases, including Huntington's, but the mol. mechanism(s) responsible for neuronal degeneration are not yet established.  Expanded polyglutamine domain proteins possess properties that distinguish them from the same proteins with shorter glutamine repeats.  Unlike proteins with short polyglutamine domains, proteins with expanded polyglutamine domains display unique protein interactions, form intracellular aggregates, and adopt a novel conformation that can be recognized by monoclonal antibodies.  Any of these polyglutamine length-dependent properties could be responsible for the pathogenic effects of expanded polyglutamine proteins.  To identify peptides that interfere with pathogenic polyglutamine interactions, the authors screened a combinatorial peptide library expressed on M13 phage pIII protein to identify peptides that preferentially bind pathol.-length polyglutamine domains.  The authors identified six tryptophan-rich peptides that preferentially bind pathol.-length polyglutamine domain proteins.  Polyglutamine-binding peptide 1 (QBP1) potently inhibits polyglutamine protein aggregation in an in vitro assay, while a scrambled sequence has no effect on aggregation.  QBP1 and a tandem repeat of QBP1 also inhibit aggregation of polyglutamine-yellow fluorescent fusion protein in transfected COS-7 cells.  Expression of QBP1 potently inhibits polyglutamine-induced cell death.  Selective inhibition of pathol. interactions of expanded polyglutamine domains with themselves or other proteins may be a useful strategy for preventing disease onset or for slowing progression of the polyglutamine repeat diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrRg3sI3AkjgLVg90H21EOLACvtfcHk0ljgGvv0TzKBww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXisFWjt7k%253D&md5=c479bed94a9e2fbd695ddecfc6d0f29c</span></div><a href="/servlet/linkout?suffix=cit115&amp;dbid=16384&amp;doi=10.1074%2Fjbc.275.14.10437&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.275.14.10437%26sid%3Dliteratum%253Aachs%26aulast%3DNagai%26aufirst%3DY.%26aulast%3DTucker%26aufirst%3DT.%26aulast%3DRen%26aufirst%3DH.%26aulast%3DKenan%26aufirst%3DD.%2BJ.%26aulast%3DHenderson%26aufirst%3DB.%2BS.%26aulast%3DKeene%26aufirst%3DJ.%2BD.%26aulast%3DStrittmatter%26aufirst%3DW.%2BJ.%26aulast%3DBurke%26aufirst%3DJ.%2BR.%26atitle%3DInhibition%2520of%2520polyglutamine%2520protein%2520aggregation%2520and%2520cell%2520death%2520by%2520novel%2520peptides%2520identified%2520by%2520phage%2520display%2520screening%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2000%26volume%3D275%26spage%3D10437%26epage%3D10442%26doi%3D10.1074%2Fjbc.275.14.10437" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref116"><div class="reference"><strong class="refLabel"><a href="#ref116" class="rightTabRefNumLink">116</a></strong><div class="NLM_citation" id="rightTab-cit116"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sakamoto, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, K. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumagai, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mercurio, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deshaies, R. J.</span></span> <span> </span><span class="NLM_article-title">Protacs: Chimeric molecules that target proteins to the Skp1-Cullin-F box complex for ubiquitination and degradation</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>98</i></span>,  <span class="NLM_fpage">8554</span>, <span class="refDoi"> DOI: 10.1073/pnas.141230798</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref116/cit116&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1073%2Fpnas.141230798" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref116/cit116&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=11438690" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref116/cit116&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BD3MXls1Wisbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=2001&pages=8554&author=K.+M.+Sakamotoauthor=K.+B.+Kimauthor=A.+Kumagaiauthor=F.+Mercurioauthor=C.+M.+Crewsauthor=R.+J.+Deshaies&title=Protacs%3A+Chimeric+molecules+that+target+proteins+to+the+Skp1-Cullin-F+box+complex+for+ubiquitination+and+degradation&doi=10.1073%2Fpnas.141230798"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit116R"><div class="casContent"><span class="casTitleNuber">116</span><div class="casTitle"><span class="NLM_cas:atitle">Protacs: chimeric molecules that target proteins to the Skp1-Cullin-F box complex for ubiquitination and degradation</span></div><div class="casAuthors">Sakamoto, Kathleen M.; Kim, Kyung B.; Kumagai, Akiko; Mercurio, Frank; Crews, Craig M.; Deshaies, Raymond J.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">98</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">8554-8559</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The intracellular levels of many proteins are regulated by ubiquitin-dependent proteolysis.  One of the best-characterized enzymes that catalyzes the attachment of ubiquitin to proteins is a ubiquitin ligase complex, Skp1-Cullin-F box complex contg. Hrt1 (SCF).  We sought to artificially target a protein to the SCF complex for ubiquitination and degrdn.  To this end, we tested methionine aminopeptidase-2 (MetAP-2), which covalently binds the angiogenesis inhibitor ovalicin.  A chimeric compd., protein-targeting chimeric mol. 1 (Protac-1), was synthesized to recruit MetAP-2 to SCF.  One domain of Protac-1 contains the IκBα phosphopeptide that is recognized by the F-box protein β-TRCP, whereas the other domain is composed of ovalicin.  We show that MetAP-2 can be tethered to SCFβ-TRCP, ubiquitinated, and degraded in a Protac-1-dependent manner.  In the future, this approach may be useful for conditional inactivation of proteins, and for targeting disease-causing proteins for destruction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp49fYtuEVlJrVg90H21EOLACvtfcHk0ljgGvv0TzKBww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXls1Wisbk%253D&md5=540358035222f745f7b6367b38781a21</span></div><a href="/servlet/linkout?suffix=cit116&amp;dbid=16384&amp;doi=10.1073%2Fpnas.141230798&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.141230798%26sid%3Dliteratum%253Aachs%26aulast%3DSakamoto%26aufirst%3DK.%2BM.%26aulast%3DKim%26aufirst%3DK.%2BB.%26aulast%3DKumagai%26aufirst%3DA.%26aulast%3DMercurio%26aufirst%3DF.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26aulast%3DDeshaies%26aufirst%3DR.%2BJ.%26atitle%3DProtacs%253A%2520Chimeric%2520molecules%2520that%2520target%2520proteins%2520to%2520the%2520Skp1-Cullin-F%2520box%2520complex%2520for%2520ubiquitination%2520and%2520degradation%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2001%26volume%3D98%26spage%3D8554%26doi%3D10.1073%2Fpnas.141230798" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref117"><div class="reference"><strong class="refLabel"><a href="#ref117" class="rightTabRefNumLink">117</a></strong><div class="NLM_citation" id="rightTab-cit117"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gopal, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dick, T.</span></span> <span> </span><span class="NLM_article-title">Targeted protein degradation in antibacterial drug discovery?</span>. <i>Prog. Biophys. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>152</i></span>,  <span class="NLM_fpage">10</span>– <span class="NLM_lpage">14</span>, <span class="refDoi"> DOI: 10.1016/j.pbiomolbio.2019.11.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref117/cit117&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1016%2Fj.pbiomolbio.2019.11.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref117/cit117&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=31738980" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref117/cit117&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitF2ltrnM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=152&publication_year=2020&pages=10-14&author=P.+Gopalauthor=T.+Dick&title=Targeted+protein+degradation+in+antibacterial+drug+discovery%3F&doi=10.1016%2Fj.pbiomolbio.2019.11.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit117R"><div class="casContent"><span class="casTitleNuber">117</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted protein degradation in antibacterial drug discovery?</span></div><div class="casAuthors">Gopal, Pooja; Dick, Thomas</div><div class="citationInfo"><span class="NLM_cas:title">Progress in Biophysics & Molecular Biology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">152</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">10-14</span>CODEN:
                <span class="NLM_cas:coden">PBIMAC</span>;
        ISSN:<span class="NLM_cas:issn">0079-6107</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Drug induced degrdn. of a target protein is a novel concept in drug discovery.  Traditionally drugs modulate activity, as opposed to abundance, of their targets.  Degrdn. inducing ligands act catalytically.  Thus, one advantage of target degrdn. over the classical on-target mechanism is that lower drug concn. may be sufficient to cause the desired cellular effects.  The first promoters of target degrdn. were discovered unintentionally: it turned out that some drugs 'accidently' promote degrdn. of their target by the cellular proteolytic machinery.  Elegant methods were developed to target specific proteins of interest for degrdn., thus enabling the rational discovery of degrdn. inducers.  The application of targeted degrdn. has so far been limited to human cells.  Recently, we discovered that an antibacterial drug, the anti-tuberculosis antibiotic pyrazinamide, functions as a promotor of degrdn. of its bacterial target.  Increasing antimicrobial resistance makes the discovery of novel antibiotics more urgent than ever.  Can rational target degrdn. be applied for the discovery of anti-bacterials.  Here, we first discuss briefly some historic examples and then recent approaches in rational target degrdn. for human diseases.  Then, we describe how the first anti-bacterial target degrdn. promoter pyrazinamide triggers removal of its target.  Efforts are under way to exploit this specific mechanistic knowledge for the discovery of next generation pyrazinamide.  We end with the big - and open - question whether targeted protein degrdn. as an approach to anti-bacterial drug discovery can be generalized, similar to what has been achieved in the area of drug discovery for human diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqA8HSCHcOWrVg90H21EOLACvtfcHk0lgk1Qy8vGtyLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitF2ltrnM&md5=12650628c6b916ff1585e417af52c705</span></div><a href="/servlet/linkout?suffix=cit117&amp;dbid=16384&amp;doi=10.1016%2Fj.pbiomolbio.2019.11.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pbiomolbio.2019.11.005%26sid%3Dliteratum%253Aachs%26aulast%3DGopal%26aufirst%3DP.%26aulast%3DDick%26aufirst%3DT.%26atitle%3DTargeted%2520protein%2520degradation%2520in%2520antibacterial%2520drug%2520discovery%253F%26jtitle%3DProg.%2520Biophys.%2520Mol.%2520Biol.%26date%3D2020%26volume%3D152%26spage%3D10%26epage%3D14%26doi%3D10.1016%2Fj.pbiomolbio.2019.11.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref118"><div class="reference"><strong class="refLabel"><a href="#ref118" class="rightTabRefNumLink">118</a></strong><div class="NLM_citation" id="rightTab-cit118"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulk, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Souza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhe-Paganon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">Phthalimide conjugation as a strategy for in vivo target protein degradation</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>348</i></span>,  <span class="NLM_fpage">1376</span>– <span class="NLM_lpage">1381</span>, <span class="refDoi"> DOI: 10.1126/science.aab1433</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref118/cit118&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=10.1126%2Fscience.aab1433" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref118/cit118&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=25999370" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref118/cit118&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;key=1%3ACAS%3A528%3ADC%252BC2MXpvFansLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=348&publication_year=2015&pages=1376-1381&author=G.+E.+Winterauthor=D.+L.+Buckleyauthor=J.+Paulkauthor=J.+M.+Robertsauthor=A.+Souzaauthor=S.+Dhe-Paganonauthor=J.+E.+Bradner&title=Phthalimide+conjugation+as+a+strategy+for+in+vivo+target+protein+degradation&doi=10.1126%2Fscience.aab1433"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit118R"><div class="casContent"><span class="casTitleNuber">118</span><div class="casTitle"><span class="NLM_cas:atitle">Phthalimide conjugation as a strategy for in vivo target protein degradation</span></div><div class="casAuthors">Winter, Georg E.; Buckley, Dennis L.; Paulk, Joshiawa; Roberts, Justin M.; Souza, Amanda; Dhe-Paganon, Sirano; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">348</span>
        (<span class="NLM_cas:issue">6241</span>),
    <span class="NLM_cas:pages">1376-1381</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The development of effective pharmacol. inhibitors of multidomain scaffold proteins, notably transcription factors, is a particularly challenging problem.  In part, this is because many small-mol. antagonists disrupt the activity of only one domain in the target protein.  The authors devised a chem. strategy that promotes ligand-dependent target protein degrdn. using as an example the transcriptional coactivator BRD4, a protein crit. for cancer cell growth and survival.  The authors appended a competitive antagonist of BET bromodomains to a phthalimide moiety to hijack the cereblon E3 ubiquitin ligase complex.  The resultant compd., dBET1, induced highly selective cereblon-dependent BET protein degrdn. in vitro and in vivo and delayed leukemia progression in mice.  A second series of probes resulted in selective degrdn. of the cytosolic protein FKBP12.  This chem. strategy for controlling target protein stability may have implications for therapeutically targeting previously intractable proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1ZW4zjRjXi7Vg90H21EOLACvtfcHk0lgk1Qy8vGtyLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXpvFansLk%253D&md5=b0277cc95d318d926b96e7a119066e3b</span></div><a href="/servlet/linkout?suffix=cit118&amp;dbid=16384&amp;doi=10.1126%2Fscience.aab1433&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aab1433%26sid%3Dliteratum%253Aachs%26aulast%3DWinter%26aufirst%3DG.%2BE.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DPaulk%26aufirst%3DJ.%26aulast%3DRoberts%26aufirst%3DJ.%2BM.%26aulast%3DSouza%26aufirst%3DA.%26aulast%3DDhe-Paganon%26aufirst%3DS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DPhthalimide%2520conjugation%2520as%2520a%2520strategy%2520for%2520in%2520vivo%2520target%2520protein%2520degradation%26jtitle%3DScience%26date%3D2015%26volume%3D348%26spage%3D1376%26epage%3D1381%26doi%3D10.1126%2Fscience.aab1433" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.0c01689&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2021.64.issue-7%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.0c01689" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.0c01689" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                3MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679953d04b9c248d","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
