
<html lang="en"     class="pb-page"  data-request-id="1e02eef0-dc00-41a6-b838-9138d6817321"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";issue:issue:10.1021/jmcmar.2020.63.issue-23;requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;article:article:10.1021/acs.jmedchem.0c00873;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Design, Synthesis, and Characterization of an Orally Active Dual-Specific ULK1/2 Autophagy Inhibitor that Synergizes with the PARP Inhibitor Olaparib for the Treatment of Triple-Negative Breast Cancer" /></meta><meta name="dc.Creator" content="Huiyu  Ren" /></meta><meta name="dc.Creator" content="Nicole A.  Bakas" /></meta><meta name="dc.Creator" content="Mitchell  Vamos" /></meta><meta name="dc.Creator" content="Apirat  Chaikuad" /></meta><meta name="dc.Creator" content="Allison S.  Limpert" /></meta><meta name="dc.Creator" content="Carina D.  Wimer" /></meta><meta name="dc.Creator" content="Sonja N.  Brun" /></meta><meta name="dc.Creator" content="Lester J.  Lambert" /></meta><meta name="dc.Creator" content="Lutz  Tautz" /></meta><meta name="dc.Creator" content="Maria  Celeridad" /></meta><meta name="dc.Creator" content="Douglas J.  Sheffler" /></meta><meta name="dc.Creator" content="Stefan  Knapp" /></meta><meta name="dc.Creator" content="Reuben J.  Shaw" /></meta><meta name="dc.Creator" content="Nicholas D. P.  Cosford" /></meta><meta name="dc.Description" content="Inhibition of autophagy, the major cellular recycling pathway in mammalian cells, is a promising strategy for the treatment of triple-negative breast cancer (TNBC). We previously reported SBI-02069..." /></meta><meta name="Description" content="Inhibition of autophagy, the major cellular recycling pathway in mammalian cells, is a promising strategy for the treatment of triple-negative breast cancer (TNBC). We previously reported SBI-02069..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="November 17, 2020" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.0c00873" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2020 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c00873" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.0c00873" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c00873" /></link>
        
    
    

<title>Design, Synthesis, and Characterization of an Orally Active Dual-Specific ULK1/2 Autophagy Inhibitor that Synergizes with the PARP Inhibitor Olaparib for the Treatment of Triple-Negative Breast Cancer | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c00873" /></meta><meta property="og:title" content="Design, Synthesis, and Characterization of an Orally Active Dual-Specific ULK1/2 Autophagy Inhibitor that Synergizes with the PARP Inhibitor Olaparib for the Treatment of Triple-Negative Breast Cancer" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00873/20201203/images/large/jm0c00873_0013.jpeg" /></meta><meta property="og:description" content="Inhibition of autophagy, the major cellular recycling pathway in mammalian cells, is a promising strategy for the treatment of triple-negative breast cancer (TNBC). We previously reported SBI-0206965, a small molecule inhibitor of unc-51-like autophagy activating kinase 1 (ULK1), which is a key regulator of autophagy initiation. Herein, we describe the design, synthesis, and characterization of new dual inhibitors of ULK1 and ULK2 (ULK1/2). One inhibitor, SBP-7455 (compound 26), displayed improved binding affinity for ULK1/2 compared with SBI-0206965, potently inhibited ULK1/2 enzymatic activity in vitro and in cells, reduced the viability of TNBC cells and had oral bioavailability in mice. SBP-7455 inhibited starvation-induced autophagic flux in TNBC cells that were dependent on autophagy for survival and displayed synergistic cytotoxicity with the poly (ADP-ribose) polymerase (PARP) inhibitor olaparib against TNBC cells. These data suggest that combining ULK1/2 and PARP inhibition may have clinical utility for the treatment of TNBC." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.0c00873"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00873">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.0c00873&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.0c00873&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.0c00873&amp;href=/doi/10.1021/acs.jmedchem.0c00873" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2020</span><span class="cit-fg-volume">, 63</span><span class="cit-fg-issue">, 23</span><span class="cit-fg-pageRange">, 14609-14625</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/63/23" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.0c01246" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.0c01174" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Design, Synthesis, and Characterization of an Orally Active Dual-Specific ULK1/2 Autophagy Inhibitor that Synergizes with the PARP Inhibitor Olaparib for the Treatment of Triple-Negative Breast Cancer</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Huiyu Ren</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Huiyu Ren</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Molecules & Structures Program, NCI-Designated Cancer Center, Sanford Burnham Prebys Medical Discovery Institute, La Jolla, California 92037, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Huiyu++Ren">Huiyu Ren</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Nicole A. Bakas</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Nicole A. Bakas</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Molecules & Structures Program, NCI-Designated Cancer Center, Sanford Burnham Prebys Medical Discovery Institute, La Jolla, California 92037, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Nicole+A.++Bakas">Nicole A. Bakas</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Mitchell Vamos</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Mitchell Vamos</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Molecules & Structures Program, NCI-Designated Cancer Center, Sanford Burnham Prebys Medical Discovery Institute, La Jolla, California 92037, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Mitchell++Vamos">Mitchell Vamos</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Apirat Chaikuad</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Apirat Chaikuad</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium, Buchmann Institute for Molecular Life Sciences, Goethe-University Frankfurt, Frankfurt 60438, Germany</div><div class="loa-info-affiliations-info">Institute of Pharmaceutical Chemistry, Goethe-University Frankfurt, Frankfurt 60438, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Apirat++Chaikuad">Apirat Chaikuad</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-1120-2209" title="Orcid link">http://orcid.org/0000-0003-1120-2209</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Allison S. Limpert</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Allison S. Limpert</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Molecules & Structures Program, NCI-Designated Cancer Center, Sanford Burnham Prebys Medical Discovery Institute, La Jolla, California 92037, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Allison+S.++Limpert">Allison S. Limpert</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Carina D. Wimer</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Carina D. Wimer</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Molecules & Structures Program, NCI-Designated Cancer Center, Sanford Burnham Prebys Medical Discovery Institute, La Jolla, California 92037, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Carina+D.++Wimer">Carina D. Wimer</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Sonja N. Brun</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Sonja N. Brun</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Molecular and Cell Biology Laboratory, The Salk Institute for Biological Studies, La Jolla, San Diego, California 92037, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Sonja+N.++Brun">Sonja N. Brun</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Lester J. Lambert</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Lester J. Lambert</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Molecules & Structures Program, NCI-Designated Cancer Center, Sanford Burnham Prebys Medical Discovery Institute, La Jolla, California 92037, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Lester+J.++Lambert">Lester J. Lambert</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Lutz Tautz</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Lutz Tautz</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Molecules & Structures Program, NCI-Designated Cancer Center, Sanford Burnham Prebys Medical Discovery Institute, La Jolla, California 92037, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Lutz++Tautz">Lutz Tautz</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Maria Celeridad</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Maria Celeridad</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Molecules & Structures Program, NCI-Designated Cancer Center, Sanford Burnham Prebys Medical Discovery Institute, La Jolla, California 92037, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Maria++Celeridad">Maria Celeridad</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Douglas J. Sheffler</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Douglas J. Sheffler</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Molecules & Structures Program, NCI-Designated Cancer Center, Sanford Burnham Prebys Medical Discovery Institute, La Jolla, California 92037, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Douglas+J.++Sheffler">Douglas J. Sheffler</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Stefan Knapp</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Stefan Knapp</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium, Buchmann Institute for Molecular Life Sciences, Goethe-University Frankfurt, Frankfurt 60438, Germany</div><div class="loa-info-affiliations-info">Institute of Pharmaceutical Chemistry, Goethe-University Frankfurt, Frankfurt 60438, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Stefan++Knapp">Stefan Knapp</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-5995-6494" title="Orcid link">http://orcid.org/0000-0001-5995-6494</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Reuben J. Shaw</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Reuben J. Shaw</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Molecular and Cell Biology Laboratory, The Salk Institute for Biological Studies, La Jolla, San Diego, California 92037, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Reuben+J.++Shaw">Reuben J. Shaw</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Nicholas D. P. Cosford</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Nicholas D. P. Cosford</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Molecules & Structures Program, NCI-Designated Cancer Center, Sanford Burnham Prebys Medical Discovery Institute, La Jolla, California 92037, United States</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#f69895998590998492b6859486929f8595998093848fd8998491"><span class="__cf_email__" data-cfemail="acc2cfc3dfcac3dec8ecdfcedcc8c5dfcfc3dac9ded582c3decb">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Nicholas+D.+P.++Cosford">Nicholas D. P. Cosford</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-7228-4709" title="Orcid link">http://orcid.org/0000-0001-7228-4709</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c00873&amp;href=/doi/10.1021%2Facs.jmedchem.0c00873" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2020</span></span><span class="cit-volume">, 63</span><span class="cit-issue">, 23</span><span class="cit-pageRange">, 14609–14625</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">November 17, 2020</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>22 May 2020</li><li><span class="item_label"><b>Published</b> online</span>17 November 2020</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 10 December 2020</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.0c00873" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00873</a></div><div class="article_header-article-copyright"><strong>Copyright © 2020 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D14609%26pageCount%3D17%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DHuiyu%2BRen%252C%2BNicole%2BA.%2BBakas%252C%2BMitchell%2BVamos%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D63%26issueNum%3D23%26contentID%3Dacs.jmedchem.0c00873%26title%3DDesign%252C%2BSynthesis%252C%2Band%2BCharacterization%2Bof%2Ban%2BOrally%2BActive%2BDual-Specific%2BULK1%252F2%2BAutophagy%2BInhibitor%2Bthat%2BSynergizes%2Bwith%2Bthe%2BPARP%2BInhibitor%2BOlaparib%2Bfor%2Bthe%2BTreatment%2Bof%2BTriple-Negative%2BBreast%2BCancer%26numPages%3D17%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D14625%26publicationDate%3DDecember%2B2020">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.0c00873"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1989</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">2</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.0c00873" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Design, Synthesis, and Characterization of an Orally Active Dual-Specific ULK1/2 Autophagy Inhibitor that Synergizes with the PARP Inhibitor Olaparib for the Treatment of Triple-Negative Breast Cancer&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Huiyu&quot;,&quot;last_name&quot;:&quot;Ren&quot;},{&quot;first_name&quot;:&quot;Nicole&quot;,&quot;last_name&quot;:&quot;A. Bakas&quot;},{&quot;first_name&quot;:&quot;Mitchell&quot;,&quot;last_name&quot;:&quot;Vamos&quot;},{&quot;first_name&quot;:&quot;Apirat&quot;,&quot;last_name&quot;:&quot;Chaikuad&quot;},{&quot;first_name&quot;:&quot;Allison&quot;,&quot;last_name&quot;:&quot;S. Limpert&quot;},{&quot;first_name&quot;:&quot;Carina&quot;,&quot;last_name&quot;:&quot;D. Wimer&quot;},{&quot;first_name&quot;:&quot;Sonja&quot;,&quot;last_name&quot;:&quot;N. Brun&quot;},{&quot;first_name&quot;:&quot;Lester&quot;,&quot;last_name&quot;:&quot;J. Lambert&quot;},{&quot;first_name&quot;:&quot;Lutz&quot;,&quot;last_name&quot;:&quot;Tautz&quot;},{&quot;first_name&quot;:&quot;Maria&quot;,&quot;last_name&quot;:&quot;Celeridad&quot;},{&quot;first_name&quot;:&quot;Douglas&quot;,&quot;last_name&quot;:&quot;J. Sheffler&quot;},{&quot;first_name&quot;:&quot;Stefan&quot;,&quot;last_name&quot;:&quot;Knapp&quot;},{&quot;first_name&quot;:&quot;Reuben&quot;,&quot;last_name&quot;:&quot;J. Shaw&quot;},{&quot;first_name&quot;:&quot;Nicholas&quot;,&quot;last_name&quot;:&quot;D. P. Cosford&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2020&quot;,&quot;month&quot;:&quot;11&quot;,&quot;day&quot;:&quot;17&quot;,&quot;issue&quot;:&quot;23&quot;,&quot;volume&quot;:&quot;63&quot;,&quot;pages&quot;:&quot;14609-14625&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.0c00873&quot;},&quot;abstract&quot;:&quot;Inhibition of autophagy, the major cellular recycling pathway in mammalian cells, is a promising strategy for the treatment of triple-negative breast cancer (TNBC). We previously reported SBI-0206965, a small molecule inhibitor of unc-51-like autophagy activating kinase 1 (ULK1), which is a key regulator of autophagy initiation. Herein, we describe the design, synthesis, and characterization of new dual inhibitors of ULK1 and ULK2 (ULK1/2). One inhibitor, SBP-7455 (compound 26), displayed improved binding affinity for ULK1/2 compared with SBI-0206965, potently inhibited ULK1/2 enzymatic activity in vitro and in cells, reduced the viability of TNBC cells and had oral bioavailability in mice. SBP-7455 inhibited starvation-induced autophagic flux in TNBC cells that were dependent on autophagy for survival and displayed synergistic cytotoxicity with the poly (ADP-ribose) polymerase (PARP) inhibitor olaparib against TNBC cells. These data suggest that combining ULK1/2 and PARP inhibition may have clinical utility &quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00873&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00873" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00873&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00873" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00873&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00873" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c00873&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00873&amp;href=/doi/10.1021/acs.jmedchem.0c00873" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.0c00873" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.0c00873" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (5 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00873&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.0c00873%26sid%3Dliteratum%253Aachs%26pmid%3D33200929%26genre%3Darticle%26aulast%3DRen%26date%3D2020%26atitle%3DDesign%252C%2BSynthesis%252C%2Band%2BCharacterization%2Bof%2Ban%2BOrally%2BActive%2BDual-Specific%2BULK1%252F2%2BAutophagy%2BInhibitor%2Bthat%2BSynergizes%2Bwith%2Bthe%2BPARP%2BInhibitor%2BOlaparib%2Bfor%2Bthe%2BTreatment%2Bof%2BTriple-Negative%2BBreast%2BCancer%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D63%26issue%3D23%26spage%3D14609%26epage%3D14625%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (1)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291350" title="Assays">Assays</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291793" title="Inhibition">Inhibition</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=292527" title="Autophagy">Autophagy</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/63/23" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/jmcmar.2020.63.issue-23/20201210/jmcmar.2020.63.issue-23.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00873/20201203/images/medium/jm0c00873_0013.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00873/20201203/images/large/jm0c00873_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00873&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Inhibition of autophagy, the major cellular recycling pathway in mammalian cells, is a promising strategy for the treatment of triple-negative breast cancer (TNBC). We previously reported SBI-0206965, a small molecule inhibitor of unc-51-like autophagy activating kinase 1 (ULK1), which is a key regulator of autophagy initiation. Herein, we describe the design, synthesis, and characterization of new dual inhibitors of ULK1 and ULK2 (ULK1/2). One inhibitor, SBP-7455 (compound <b>26</b>), displayed improved binding affinity for ULK1/2 compared with SBI-0206965, potently inhibited ULK1/2 enzymatic activity in vitro and in cells, reduced the viability of TNBC cells and had oral bioavailability in mice. SBP-7455 inhibited starvation-induced autophagic flux in TNBC cells that were dependent on autophagy for survival and displayed synergistic cytotoxicity with the poly (ADP-ribose) polymerase (PARP) inhibitor olaparib against TNBC cells. These data suggest that combining ULK1/2 and PARP inhibition may have clinical utility for the treatment of TNBC.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_61020" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_61020" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Macroautophagy (hereafter referred to as autophagy) is a highly conserved pathway in eukaryotic cells that mediates the coordinated lysosome-dependent disassembly and recycling of cellular components. By degrading damaged organelles, misfolded or toxic proteins, and pathogens, autophagy maintains cellular homoeostasis and promotes cell survival under stress.<a onclick="showRef(event, 'ref1 ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3">(1−3)</a> Autophagic flux is maintained at a basal level in all cells, but it can be further increased by various stressors, such as elevated levels of reactive oxygen species (ROS), deprivation of nutrients, and exposure to chemotherapeutic agents.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a></div><div class="NLM_p">Autophagy is a multistep pathway initiated through the ULK preinitiation complex, which consists of unc-51-like autophagy activating kinase 1 (ULK1) or its homologue ULK2, autophagy related protein 13 (ATG13), ATG101 and focal adhesion kinase (FAK) family-interacting protein (FIP200) (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00873/suppl_file/jm0c00873_si_001.pdf" class="ext-link">Figure S1</a>).<a onclick="showRef(event, 'ref5 ref6'); return false;" href="javascript:void(0);" class="ref ref5 ref6">(5,6)</a> Autophagy is stimulated under stress conditions by activation of AMP-activated protein kinase (AMPK) and inhibited under nonstressed conditions by the mechanistic target of rapamycin (mTOR).<a onclick="showRef(event, 'ref7 ref8'); return false;" href="javascript:void(0);" class="ref ref7 ref8">(7,8)</a> AMPK activates autophagy by two mechanisms: first, by inhibiting mTOR activity via phosphorylation of the mTOR complex components raptor and tuberous sclerosis complex 2 (TSC2); and second, by directly phosphorylating ULK1/2 at multiple sites <i>N-</i>terminal to the mTOR-phosphorylation site, triggering ULK1/2 activation. Conversely, mTOR suppresses autophagy by directly phosphorylating and inhibiting ULK1/2 function (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00873/suppl_file/jm0c00873_si_001.pdf" class="ext-link">Figure S1</a>).</div><div class="NLM_p">ULK1/2 are the only serine/threonine (S/T) kinases in the core autophagy pathway, and their involvement at the first committed step in the autophagy cascade makes them attractive druggable targets for inhibiting autophagy. ULK1/2 play crucial roles in autophagy by phosphorylating the other three proteins in the ULK1/2 preinitiation complex (ATG13, ATG101, and FIP200)<a onclick="showRef(event, 'ref9 ref10 ref11'); return false;" href="javascript:void(0);" class="ref ref9 ref10 ref11">(9−11)</a> and proteins in the downstream Beclin1 initiation complex: Beclin1 (encoded by the gene BECN1), VPS34, ATG9, and ATG16L1.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a></div><div class="NLM_p">As a major cellular recycling process, autophagy plays important roles in numerous diseases, including cancer.<a onclick="showRef(event, 'ref13 ref14'); return false;" href="javascript:void(0);" class="ref ref13 ref14">(13,14)</a> Autophagy promotes tumor growth in several ways: by providing nutrients and small molecule building blocks to support the elevated metabolic demands of tumor cells, regulating oxidative stress,<a onclick="showRef(event, 'ref15 ref16 ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref15 ref16 ref17 ref18">(15−18)</a> and contributing to the development of drug resistance.<a onclick="showRef(event, 'ref19 ref20 ref21 ref22 ref23'); return false;" href="javascript:void(0);" class="ref ref19 ref20 ref21 ref22 ref23">(19−23)</a> Some tumor types require a higher basal level of autophagy than normal cells, which suggests that autophagy inhibition may be a feasible therapeutic approach for certain cancers.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> One example is triple-negative breast cancer (TNBC), which accounts for about 15–20% of all breast cancer cases.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> TNBC cells do not express clinically significant levels of estrogen receptors, progesterone receptors, or of the human epidermal growth factor receptor 2 (HER2), thus rendering them insensitive to endocrine or anti-HER2-targeted treatments.<a onclick="showRef(event, 'ref25 ref26'); return false;" href="javascript:void(0);" class="ref ref25 ref26">(25,26)</a> Both basic research and clinical studies support the hypothesis that autophagy inhibition is a promising therapeutic strategy for TNBC.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> However, the only drugs available for clinical studies are the weak autophagy inhibitors chloroquine (CQ) and hydroxychloroquine (HCQ),<a onclick="showRef(event, 'ref28 ref29'); return false;" href="javascript:void(0);" class="ref ref28 ref29">(28,29)</a> which are not autophagy-specific and have poor toxicity profiles resulting in serious adverse side effects. Thus, we considered that small molecule ULK1/2 inhibitors that block autophagy initiation may hold promise for the treatment of TNBC.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a></div><div class="NLM_p">We previously reported SBI-0206965 (compound <b>1</b> in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, <a href="/doi/suppl/10.1021/acs.jmedchem.0c00873/suppl_file/jm0c00873_si_001.pdf" class="ext-link">Figure S2</a>), a first-generation chemical probe that inhibits ULK1 enzymatic activity in vitro and in cells, as demonstrated by inhibition of phosphorylation of the downstream substrates VPS34 and Beclin1.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> While SBI-0206965 has proven to be a valuable autophagy probe,<a onclick="showRef(event, 'ref31 ref32 ref33'); return false;" href="javascript:void(0);" class="ref ref31 ref32 ref33">(31−33)</a> we were keen to improve its on-target potency and drug-like properties, including oral bioavailability. Herein, we describe new dual ULK1/2 inhibitors that resulted from a combination of a structure-based and rational design. We demonstrate that the optimized autophagy inhibitor SBP-7455 (<b>26</b>) selectively blocks autophagic flux in human TNBC cell lines and exhibits favorable PK and PD properties in vivo. In addition, we show that compound <b>26</b> acts synergistically with poly(ADP-ribose)polymerase (PARP) inhibitors to kill TNBC cells, suggesting that combination therapy with ULK1/2 inhibitors may have utility for the treatment of drug-resistant TNBC.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Structures of ULK1/2 Inhibitors and IC<sub>50</sub> Values Measured by ADP-Glo and NanoBRET Assays<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00873/20201203/images/medium/jm0c00873_0011.gif" alt="" id="gr10" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00873/20201203/images/medium/jm0c00873_0012.gif" alt="" id="gr11" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">(*)Compounds 1 and 26 are SBI-0206965 and SBP-7455, respectively. ADP-Glo and NanoBRET assay results are presented as the means ± SD of three independent experiments performed in triplicate (ADP-Glo) or duplicate (NanoBRET).</p></div></div><div></div></div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i4">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_07604" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_07604" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> SAR Studies</h3><div class="NLM_p">To obtain a structure-based pharmacophore for improving the ULK1/2 potency of SBI-0206965 (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00873/suppl_file/jm0c00873_si_001.pdf" class="ext-link">Figure S2</a>), we obtained crystals of the ULK2 kinase domain (1-277) bound to SBI-0206965.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> The kinase domains of ULK1 and ULK2 share high sequence identity (78%) with near-complete structural correspondence of residues in the interlobe region. We recently reported that several small molecule kinase inhibitors exhibit comparable potencies against recombinant ULK1 and ULK2.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> By utilizing a similar strategy, crystals of the T150D ULK2 kinase domain bound to SBI-0206965 were obtained at a resolution of 2.7 Å (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A–C and <a href="/doi/suppl/10.1021/acs.jmedchem.0c00873/suppl_file/jm0c00873_si_001.pdf" class="ext-link">Table S1</a>). The T150D mutant of ULK2 was used to mimic the active form of the kinase domain because it more readily forms high-quality crystals than the wild-type (WT) protein. The crystallized form of ULK2 bound to SBI-0206965 adopted face-to-face interactions with a symmetrical protomer counterpart in which the activation loop and C helix of the two protomers interdigitated.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a></div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00873/20201203/images/medium/jm0c00873_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00873/20201203/images/large/jm0c00873_0002.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Crystal structure of the kinase domain of ULK2 bound to chemical probe SBI-0206965 (<b>1</b>). (A) Crystal structure of SBI-0206965 bound to the interlobe ATP site in the kinase domain of ULK2 at 2.7 Å (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YID">6YID</a>). (B) SBI-0206965 forms hydrogen bonds to Cys88 in the hinge region. (C) Detailed interactions with hinge residue atoms and Lys39 are indicated. (D) In silico docking of compound <b>3</b> to the crystal structure of ULK2.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00873/20201203/images/large/jm0c00873_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00873&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Our results show that SBI-0206965 bound to the ATP binding site of ULK2 adopts an orientation that formed a specific bidentate hydrogen bond between the trisubstituted pyrimidine scaffold and backbone residue Cys88 in the hinge region. The compact binding mode of SBI-0206965 conformed to the dimensions of the ATP binding site with a high degree of competitive occupancy. The amide substituent of the <i>N</i>-methylbenzamide ring extended toward the catalytic residues of the DFG loop. The trimethoxyphenyl ring was located along the hinge toward the solvent-exposed region of the binding pocket. The bromine of the pyrimidine scaffold filled a malleable “back pocket” cavity proximal to the gatekeeper residue Met85. The trimethoxyphenyl and pyrimidinyl rings of SBI-0206965 adopted nearly coplanar geometries, while the <i>N</i>-methylbenzamide ring was rotated ∼6° out of plane. Overall, SBI-0206965 adopted a Type I binding mode that targeted the ULK1/2-active conformation.<a onclick="showRef(event, 'ref35 ref36 ref37'); return false;" href="javascript:void(0);" class="ref ref35 ref36 ref37">(35−37)</a></div><div class="NLM_p last">The currently reported structures of inhibitors bound to ULK1 or ULK2 suggest the presence of common specificity-determining features that belie the observed binding pocket plasticity.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> In all compounds with high potency, the cyclopropyl and trifluoromethyl groups, or the halogen atoms located at the 5-position of the pyrimidine are placed into the back pocket of ULK1/2. This is evident in the structures of MRT67307 or MRT68921 bound to ULK2 (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6QAU">6QAU</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6QAV">6QAV</a>, respectively).<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> ULK1/2 accepts diverse Type I scaffolds; the reported pyrazolylquinazoline (Shokat compound, <a href="/doi/suppl/10.1021/acs.jmedchem.0c00873/suppl_file/jm0c00873_si_001.pdf" class="ext-link">Figure S2</a>) even binds to ULK1 in a flipped orientation (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4WNO">4WNO</a>).<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> However, the structure of SBI-0206965 bound to ULK2 revealed multiple adjacent sites for expanded interaction, providing the opportunity to construct second-generation autophagy inhibitors using structure-based design.</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Optimization and Characterization of ULK Inhibitors</h3><div class="NLM_p">Using the hinge-binding pyrimidine scaffold of SBI-0206965 as the basis for next-generation ULK inhibitors, we conducted a focused biochemical screen on a small number of 2,4,5-trisubsitituted pyrimidine compounds and identified the benzo[d][1,3]dioxole <b>2</b> and the related <i>N</i><sup>4</sup>-cyclopropyl derivative <b>3</b> as promising candidates for further optimization (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>A).<named-content content-type="anchor" rid="tbl1" type="simple"></named-content> Because compound <b>3</b> possessed a 5-trifluoromethyl to insert in the hydrophobic back pocket and convenient 2,4-diamine substituents to explore space in and around the active site, we used the crystal structure of ULK2 to model the interactions of <b>3</b> within the ligand binding site (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>D). Flexible ligand docking with compound <b>3</b> was performed using the ICM docking algorithm.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> The data obtained from these docking studies supported a binding mode in which nitrogen in the 1-position of the pyrimidine and the amino group in the 2-position of the pyrimidine ring form hydrogen-bonding interactions with the backbone atoms of Cys88 in the hinge region. These data are in agreement with the co-crystal structure of ULK2 bound to SBI-0206965, which showed the corresponding 2-aminopyrimidine moiety of SBI-0206965 in a similar position, undergoing similar interactions. The docking data indicated that the trifluoromethyl group in the 5-position of the pyrimidine ring in <b>3</b> occupied the hydrophobic back pocket formed by the side chains of Val23, Ala37, Lys39, Val69, Met85, and Leu138, thus providing additional favorable interactions compared with the bromo group at the corresponding position in SBI-0206965. The docking data also suggested that the cyclopropylamine moiety in the 4-position of the pyrimidine ring of <b>3</b> formed hydrophobic interactions with the sidechains of Val15 and Val23 and with backbone atoms of Gly16 within the β1 and β2 sheets, respectively. Finally, the benzo[d][1,3]dioxole moiety was found to undergo hydrophobic interactions with the side chains of Val15, Tyr87, and Leu138 and with backbone atoms of Asp89, Gly90, and Gly91. Notably, the benzo[d][1,3]dioxole moiety was not involved in any specific hydrogen bonding interactions, suggesting that it could be replaced with other moieties that might add additional polar interactions or could be truncated to improve ligand efficiency.</div><figure id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00873/20201203/images/medium/jm0c00873_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00873/20201203/images/large/jm0c00873_0010.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. SAR Strategy Around ULK1/2 Fragments<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00873/20201203/images/large/jm0c00873_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00873&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>(A) Compounds <b>2</b> and <b>3</b> identified in a screen for ULK1 inhibition, with IC<sub>50</sub> values (ADP-Glo assay) and (B) a synthetic scheme for <i>N</i><sup>4</sup>-cyclopropyl-5-(trifluoromethyl)pyrimidine ULK inhibitors.</p></p></figure><div class="NLM_p">To fully characterize new ULK1/2 inhibitors and facilitate expansion of the SAR, we established a versatile testing platform. We used an in vitro end-point ADP-Glo kinase assay<a onclick="showRef(event, 'ref34 ref40'); return false;" href="javascript:void(0);" class="ref ref34 ref40">(34,40)</a> as the primary assay to measure the biochemical inhibitory potency of compounds (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.0c00873/suppl_file/jm0c00873_si_001.pdf" class="ext-link">Figure S3</a>). To determine on-target kinase engagement in live cells, we employed a bioluminescence energy transfer NanoBRET assay that we optimized for ULK1 (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.0c00873/suppl_file/jm0c00873_si_001.pdf" class="ext-link">Figure S4</a>).<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> This quantitative technique relies on a proximity-based energy transfer between a protein tagged with a 19 kDa luciferase domain and cell-permeable bivalent fluorescent energy transfer (FRET) probe that emits a signal. A kinase inhibitor that is a cell penetrant competes with the fluorescent probe and reduces the signal.<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> This assay was compatible with a high-throughput workflow and offered a reliable metric to compare the cellular potencies of compounds.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> In addition, protein thermal shift (PTS) assays were employed to confirm the biochemical potency and determine the relative ability of compounds to stabilize ULK1 or ULK2, as measured by a shift in protein-melting temperatures (Tm).<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a></div><div class="NLM_p">The co-crystal structure of SBI-0206965 bound to ULK2, coupled with our in silico modeling data and previous structural studies<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> around ULK1 and ULK2, suggested that improved potency could be elicited by systematic evaluation of aryl-substituents at the <b>R</b><sup>2</sup> position (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Analogues were synthesized in a modular approach by two sequential substitutions on the pyrimidine scaffold (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>B). Thus, 2,4-dichloro-5-(trifluoromethyl)pyrimidine was reacted with various alkyl amines under basic conditions to afford key intermediate <b>A</b>. Compound <b>A</b> was then reacted with a panel of aryl amines in acetic acid utilizing microwave irradiation to provide the target inhibitors (<b>B</b>) with diverse electronic and polar characteristics. The analogues were tested in dose–response mode in the ADP-Glo kinase assay (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00873/suppl_file/jm0c00873_si_001.pdf" class="ext-link">Figure S3</a>), and the data for compounds with IC<sub>50</sub> values better than 50 nM are shown in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>.</div><div class="NLM_p">Since our docking studies revealed that the aniline moiety (<b>R<sup>2</sup></b>) is directed toward solvent-exposed space, it is perhaps unsurprising that a range of aryl substituents were tolerated. For example, systematic expansion of the dioxole ring to the dioxine <b>5</b> or dioxepine <b>6</b> improved the IC<sub>50</sub> compared with <b>3</b> (21 nM) to 14 and 7 nM, respectively, in the ADP-Glo assay (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). On the other hand, the difluorodioxole derivative <b>4</b> (42 nM) was less potent than <b>3</b>. The series of derivatives in which the dioxole moiety was replaced with a 5-membered heteroaromatic ring (<b>7</b>, <b>8</b>, <b>9</b>, <b>10</b>, <b>11</b>, and <b>15</b>) showed subtle variations in potency. Thus, the indole (<b>8</b>), pyrrolopyridine (<b>11</b>), and benzoxazole (<b>15</b>) were approximately equipotent with <b>3</b>, while the benzimidazole (<b>7</b>) and benzotriazole (<b>10</b>) derivatives were about 4-fold more potent (IC<sub>50</sub> values 5 and 6 nM, respectively). We also tested analogues of <b>11</b> in which the cyclopropyl moiety (<b>R<sup>1</sup></b>) was replaced with a cyclobutyl (<b>12</b>) or a methyl (<b>13</b>) group, resulting in no significant improvement in potency. Interestingly, in the series of 6,6-fused ring analogues (<b>14</b>, <b>16–19</b>), all except the pyrido[1,4]oxazine derivative (<b>14</b>) were more potent than <b>3</b>. Notably, the tetralone derivative (<b>19</b>) was the most potent analogue with an IC<sub>50</sub> value of 3 nM in the ADP-Glo assay. Finally, in the series of six-membered ring aryl and heteroaryl derivatives (<b>20</b>–<b>26</b>), the potencies ranged from 13 to 50 nM in the ADP-Glo assay, with <b>21</b>, <b>24</b>, and <b>26</b> being the most potent. To verify the binding affinities and benchmark, the relative stabilization of protein compared with SBI-0206965, fifteen of the most potent analogues were tested in the PTS assay against the kinase domain of ULK1 (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00873/suppl_file/jm0c00873_si_001.pdf" class="ext-link">Table S2</a>). The largest thermal shift observed was ΔTm = 9.5 °C for compound <b>26,</b> compared with ΔTm = 8.3 °C for SBI-0206965. Interestingly, the PTS data followed a similar trend to the ADP-Glo data, with the more potent compounds in the ADP-Glo assay generally exhibiting a larger ΔTm in the PTS assay (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00873/suppl_file/jm0c00873_si_001.pdf" class="ext-link">Table S2</a>).</div><div class="NLM_p">We next utilized the ULK1 NanoBRET intracellular kinase assay to quantify the target engagement of compounds in live cells, marking the first report of a NanoBRET assay for ULK1 (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.0c00873/suppl_file/jm0c00873_si_001.pdf" class="ext-link">Figure S4</a>). The IC<sub>50</sub> values, shown in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, indicate the capacity of the inhibitors to both enter the cell and bind to full-length ULK1 kinase at intracellular physiological levels of ATP. Compared with the ADP-Glo kinase assay, the IC<sub>50</sub> values for compounds were right-shifted in the ULK1 NanoBRET cellular assay. The results in the NanoBRET assay proved to be critical to the selection of compounds for further characterization. Thus, our first-generation chemical probe SBI-0206965 is active in this assay, with a benchmark IC<sub>50</sub> value of 785 nM. Remarkably, only five of the new inhibitors (<b>7</b>, <b>16</b>, <b>21</b>, <b>23</b>, and <b>26</b>) exhibited IC<sub>50</sub> values in the sub-micromolar range, with the potencies of the other analogues ranging from 1.0 to >10 μM (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>).</div><div class="NLM_p">The most potent ULK1 inhibitors, as well as SBI-0206965, were next tested for their ability to inhibit ULK2 in vitro and in cells. Thus, ULK2 biochemical inhibition (using the ADP-Glo assay) and cellular target engagement (using the NanoBRET assay) were measured for compounds <b>1</b>, <b>7</b>, <b>15</b>, <b>16</b>, <b>21, 23</b>, and <b>26</b> (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). As shown in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>, in general, the compounds were much less potent against ULK2 compared with ULK1 in both assays, except compound <b>7</b>, which was similarly potent against ULK1 and ULK2 in the NanoBRET assay, with IC<sub>50</sub> values 585 and 703 nM, respectively. As demonstrated with ULK1, compound IC<sub>50</sub> values were generally right-shifted in the ULK2 NanoBRET assay relative to the ULK2 ADP-Glo assay. Notably, however, compound <b>26</b> was significantly more potent than SBI-0206965 in both ULK2 assays (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>).</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Inhibitory Activities of Selected Compounds against ULK2</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col></colgroup><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">compound</td><td class="colsep0 rowsep0" align="center" char="±">ULK2 ADP-Glo IC<sub>50</sub> (nM)</td><td class="colsep0 rowsep0" align="center" char="±">ULK2 NanoBRET IC<sub>50</sub> (nM)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1</b><a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="char" char="±">2448 ± 298</td><td class="colsep0 rowsep0" align="char" char="±">4578 ± 234</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7</b></td><td class="colsep0 rowsep0" align="char" char="±">720 ± 60</td><td class="colsep0 rowsep0" align="char" char="±">703 ± 84</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>15</b></td><td class="colsep0 rowsep0" align="char" char="±">1193 ± 88</td><td class="colsep0 rowsep0" align="char" char="±">4256 ± 513</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>16</b></td><td class="colsep0 rowsep0" align="char" char="±">714 ± 99</td><td class="colsep0 rowsep0" align="char" char="±">1647 ± 200</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>21</b></td><td class="colsep0 rowsep0" align="char" char="±">695 ± 244</td><td class="colsep0 rowsep0" align="char" char="±">1885 ± 141</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>23</b></td><td class="colsep0 rowsep0" align="char" char="±">1329 ± 94</td><td class="colsep0 rowsep0" align="char" char="±">1687 ± 75</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>26</b><a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="char" char="±">476 ± 21</td><td class="colsep0 rowsep0" align="char" char="±">1158 ± 116</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">Compounds <b>1</b> and <b>26</b> are SBI-0206965 and SBP-7455, respectively. Mean ± SD, representative of three independent experiments performed in triplicate (ADP-Glo) or in duplicate (NanoBRET).</p></div></div></div><div class="NLM_p">The right-shifts in potency for the NanoBRET versus ADP-Glo assays for ULK1 and ULK2 could be due to a combination of effects, including membrane permeability, ATP competition, influence of the activation state and cellular protein interactions. These differences are due to the cellular nature of the NanoBRET assay, which requires compounds to cross the cytoplasmic membrane to interact with the target and to then compete with higher levels of ATP present in the NanoBRET assays, which are performed under cellular levels of ATP as opposed to <i>K</i><sub>m</sub> levels of ATP utilized in the ADP-Glo assays.</div><div class="NLM_p">Since we were keen to identify ULK1/2 inhibitors with improved drug-like properties, we next evaluated aqueous solubility, membrane permeability, and whether compounds could act as a substrate for drug transporters such as P-glycoprotein (PgP). We selected a subset of compounds and determined kinetic solubility in three media: PBS pH 7.4 buffer, fasted simulated intestinal fluid (FaSSIF), or fasted simulated gastric fluid (FaSSGF). The results, shown in <a href="/doi/suppl/10.1021/acs.jmedchem.0c00873/suppl_file/jm0c00873_si_001.pdf" class="ext-link">Table S4</a>, indicated that several compounds had reasonable solubility (>5 μM) in PBS, increased solubility in FaSSIF, and greatly increased solubility in FaSSGF. We also assessed the bidirectional permeability of the same subset of compounds in Caco-2 cells (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00873/suppl_file/jm0c00873_si_001.pdf" class="ext-link">Table S5</a>). All the compounds tested, with the exception of <b>11</b> and <b>19</b>, which also have limited solubility in PBS (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00873/suppl_file/jm0c00873_si_001.pdf" class="ext-link">Table S4</a>), had high levels of permeability (Papp >10 × 10<sup>–6</sup> cm/s), and the majority had limited potential as a PgP substrate (efflux ratio < 2).</div><div class="NLM_p last">Given these membrane permeability data, the right-shifts observed in the NanoBRET assays relative to the ADP-Glo assays are more likely due to competition from cellular levels of ATP and not due to differences in membrane permeability. Based on the data shown in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Tables <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> and <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a> and <a href="/doi/suppl/10.1021/acs.jmedchem.0c00873/suppl_file/jm0c00873_si_001.pdf" class="ext-link">Tables S4 and S5</a>, compounds <b>7</b>, <b>15</b>, <b>16</b>, <b>21</b>, <b>23</b>, and <b>26</b> were selected for further characterization.</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Evaluation of Downstream Target Engagement by ULK1/2 Inhibitors in Mammalian Cells</h3><div class="NLM_p">We previously reported a method for quantifying the phosphorylation of VPS34 at Ser249 when ULK1 and VPS34 are overexpressed in HEK293T cells.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> We have established a similar assay to detect the phosphorylation of Beclin1 at Ser15. We therefore performed Western blot analysis to determine whether the new ULK1/2 inhibitors inhibit phosphorylation of Beclin1 or VPS34 (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00873/suppl_file/jm0c00873_si_001.pdf" class="ext-link">Figure S1</a> and <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>).<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a></div><figure id="fig2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00873/20201203/images/medium/jm0c00873_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00873/20201203/images/large/jm0c00873_0003.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. ULK1/2 inhibitors block phosphorylation of downstream autophagy target proteins. Western blot analysis (upper panels) and quantification (lower panels) of Beclin1 and VPS34 phosphorylation. HEK293T cells were transfected with Myc-tagged kinase inactive (KI) or wildtype (WT) ULK1 and WT Flag-tagged (A) Beclin1 or (B) VPS34 for 24 h. Cells were then treated with DMSO or the indicated ULK1/2 inhibitors at 10 μM for 1 h, lysed, and subjected to Western blotting with the indicated antibodies. Blots were quantified by densitometry. Data are expressed as the ratio of pBeclin1/Beclin1 or pVPS34/VPS34 normalized to DMSO-treated samples. Mean ± SD of three independent experiments. *<i>P</i> < 0.05, **<i>P</i> < 0.01 vs SBI-0206965 by two-sided unpaired test.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00873/20201203/images/large/jm0c00873_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00873&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">For these experiments, Flag-tagged Beclin1 or VPS34 and Myc-tagged ULK1 were overexpressed in HEK293T cells, and the cells were incubated with DMSO or ULK1/2 inhibitors (10 μM, 1 h). Cell lysates were then subjected to Western blot analysis with antibodies to Ser15-phosphorylated and total Beclin1 or Ser249-phosphorylated and total VPS34. Importantly, the ULK1 specificity of the inhibitors was verified by examining cells overexpressing a Myc-tagged kinase-inactive (KI) mutant in place of the WT enzyme. As expected, control (DMSO-treated) cells overexpressing WT ULK1 showed a dramatic increase in Beclin1 phosphorylation compared with untransfected cells or ULK1-KI-expressing cells (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A). Treatment with SBI-0206965 reduced Beclin1 phosphorylation by ∼30% compared with control (DMSO), and a similar level of inhibition was observed for compounds <b>7</b>, <b>15</b>, <b>16</b>, <b>21</b>, and <b>23</b>. The most pronounced effect was displayed by compound <b>26</b>, which inhibited Beclin1 phosphorylation by ∼60%. We observed the same pattern of activity for our compounds when phosphorylation of VPS34 Ser249 by ULK1 was measured (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B). All of the ULK inhibitors greatly reduced VPS34 phosphorylation (by ∼70–80%), with compound <b>26</b> again having the greatest effect (∼90% inhibition) (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B). These data confirm that our ULK1/2 inhibitors effectively engage their intended targets in the cellular environment and reveal the superior potency of <b>26</b>, which was selected for further evaluation and characterization.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> PK Analysis of Compound <b>26</b></h3><div class="NLM_p">As noted previously, SBI-0206965 has a limited value as an in vivo probe due to poor systemic exposure following ip dosing and no measurable oral bioavailability in mice. To determine whether compound <b>26</b> had improved PK properties compared with SBI-0206965, a single dose of <b>26</b> (30 mg/kg) was orally administered to mice and its plasma concentration was measured over the following 24 h using LC–MS/MS. This analysis showed that the time to peak plasma concentration (<i>T</i><sub>max</sub>) for <b>26</b> was approximately 1 h and the maximum plasma concentration (<i>C</i><sub>max</sub>) was 990 nM, which was about 3-fold higher than the IC<sub>50</sub> value for <b>26</b> (328 nM) against ULK1 in the NanoBRET assay (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A). The plasma concentration of compound <b>26</b> remained above the ULK1 IC<sub>50</sub> for almost 4 h after oral dosing. These data indicate that <b>26</b> has improved PK properties compared with SBI-0206965, which will enable analysis of its PD properties in vivo.</div><figure id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00873/20201203/images/medium/jm0c00873_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00873/20201203/images/large/jm0c00873_0004.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. PK and PD analyses of compound <b>26</b> (SBP-7455). (A) Mice were dosed orally with compound <b>26</b> (30 mg/kg in 5% DMSO, 10% Tween 80, and 85% H<sub>2</sub>O). Plasma samples were taken at the indicated time points, and the concentration of compound was determined by LC–MS/MS analysis. The data were analyzed using PKSolver software. Mean ± SD of <i>N</i> = 3. Dotted line indicates the IC<sub>50</sub> of compound <b>26</b> in the ULK1 NanoBRET assay. (B) Mice were dosed with compound <b>26</b> (10 mg/kg) or vehicle (5% DMSO, 10% Tween 80, and 85% H<sub>2</sub>O) via oral gavage. Liver samples were harvested and lysed 2 h post dosing, and in vivo target engagement was assessed by Western blot. <i>N</i> = 3. All the lanes were on the same gel; the lanes between vehicle and SBP-7455 were cropped out.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00873/20201203/images/large/jm0c00873_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00873&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> In Vivo Target Engagement of SBP-7455</h3><div class="NLM_p last">As noted above, ATG13 resides with ULK1 in the autophagy preinitiation complex in cells. When autophagy is initiated, it is phosphorylated by ULK1 at Ser318. ATG13 is therefore a reliable PD biomarker for ULK1 activity. To evaluate the PD profile and in vivo target engagement, mice were dosed with compound <b>26</b> (10 mg/kg) or vehicle by oral gavage, and liver samples were collected after 2 h. The effects of compound <b>26</b> on ULK1 and ATG13 were assessed in the liver lysates. Western blot analysis revealed robust inhibition of pATG13 (Ser318), as well as downregulation of total ATG13 and ULK1 levels by compound <b>26</b> (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B). Based on the PK and PD profiles, our data show that compound <b>26</b> is an efficacious tool that can be used to modulate autophagy in vivo.</div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> TNBC Cells Require ULK1 Activity and Autophagy Initiation for Survival and Proliferation</h3><div class="NLM_p">A growing body of evidence supports a role for autophagy in the survival and proliferation of TNBC cells, suggesting that autophagy inhibition may be an effective strategy for the treatment of TNBC.<a onclick="showRef(event, 'ref26 ref28 ref46 ref47'); return false;" href="javascript:void(0);" class="ref ref26 ref28 ref46 ref47">(26,28,46,47)</a> For example, Maycotte and co-workers used short hairpin RNAs (shRNAs) to target the core autophagy genes ATG5, ATG7, and BECN1 in a panel of breast tumor cells.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> Their data showed that TNBC cell lines were sensitive to autophagy inhibition by gene knockdown (KD) of ATG5, ATG7, or BECN1 and pharmacologically with the lysosomotropic autophagy inhibitor CQ. Interestingly, the TNBC cells were dependent on autophagy for survival even in complete media and thus in the absence of stressors known to stimulate autophagy. To examine whether this was true for other core autophagy genes, we performed shRNA-mediated KD of ULK1 or BECN1 in the human TNBC cell lines MDA-MB-468, MDA-MB-231, and BT549. Western blot analysis confirmed that the respective target proteins were effectively and specifically silenced compared with control cells expressing scramble shRNA (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00873/suppl_file/jm0c00873_si_001.pdf" class="ext-link">Figure S5</a>). Analysis of cell survival and proliferation revealed that ULK1 or BECN1 KD markedly reduced the proliferation of all three cell lines over 8 days, with near-complete abrogation of growth in ULK1-KD and BECN1-KD MDA-MB-468 cells and in ULK1-KD BT549 cells (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>). These data demonstrate that all three TNBC cells require autophagy initiation for survival and growth, with the most pronounced effects observed for MDA-MB-468 and BT549 cells.</div><figure id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00873/20201203/images/medium/jm0c00873_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00873/20201203/images/large/jm0c00873_0005.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Ablation of ULK1 or BECN1 by shRNA-mediated knockdown reduces the viability of TNBC cells. MDA-MB-468 (A), BT549 (B), and MDA-MB-231 (C) cells transduced with scramble, ULK1-targeting, or BECN1-targeting shRNAs and selected in puromycin for 3 days before analysis. Cell viability was measured with the Promega CellTiter-Glo assay on the indicated days. Mean ± SD of three independent experiments, each performed in quadruplicate. Data are normalized to the viability of scramble shRNA control cells. *<i>P</i> < 0.05, **<i>P</i> < 0.01, ***<i>P</i> < 0.001 vs scramble control by two-sided unpaired t-test.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00873/20201203/images/large/jm0c00873_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00873&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> ULK1/2 Inhibitors Reduce TNBC Cell Viability</h3><div class="NLM_p">Because our KD results showed that MDA-MB-468 cells were the most dependent on autophagy initiation, we next examined the effects of our ULK1/2 inhibitors in this cell line. MDA-MB-468 cells were incubated for 72 h with varying concentrations of SBI-0206965, compound <b>26</b> (SBP-7455), or CQ as a positive control. As shown in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A, MDA-MB-468 cells were relatively insensitive to CQ (IC<sub>50</sub> = 10.6 μM) compared with <b>26</b> (IC<sub>50</sub> = 0.3 μM) or SBI-0206965 (IC<sub>50</sub> = 2.1 μM). These data are consistent with the results of the in vitro (ADP-Glo) and cellular (NanoBRET) ULK1/2 kinase assays, which revealed more potent inhibition by compound <b>26</b> compared with SBI-0206965. Mechanistically, the ULK1/2 inhibitors regulate autophagy initiation, whereas the pleiotropic drug CQ inhibits end-stage lysosomal degradation in the autophagy pathway. Accordingly, our data suggest that TNBC cell survival and growth may be more sensitive to inhibition early in the autophagy pathway as opposed to downstream.</div><figure id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00873/20201203/images/medium/jm0c00873_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00873/20201203/images/large/jm0c00873_0006.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. ULK1/2 inhibitors are cytotoxic as single agents against TNBC cells. (A) MDA-MB-468 cells were treated with the indicated concentrations of each ULK1/2 inhibitor or chloroquine (CQ) and incubated for 72 h. Cell viability was then measured using CellTiter-Glo. Compound treatment started at 30 μM for the ULK1/2 inhibitors and 90 μM for CQ with serial dilutions of 1:3. IC<sub>50</sub> values were determined using GraphPad Prism 8. The dose–response curves are representative of three independent experiments performed in quadruplicate; mean ± SD. (B) Compound <b>26</b> (SBP-7455) promotes apoptosis in TNBC cells under nutrient-deprived conditions. MDA-MB-468 cells were treated with DMSO or compound <b>26</b> (10 μM) in normal growth media (RPMI with 10% FBS, upper panel) or starvation media (EBSS, lower panel) for 18 h. The cells were then collected, and apoptosis was analyzed with PE-Annexin V/7-AAD staining by flow cytometry. Numbers indicate the percentage of cells in each gate. Shown is a representative of three independent experiments with similar results.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00873/20201203/images/large/jm0c00873_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00873&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Given the role of autophagy as a survival mechanism for cancer cells in response to nutrient deprivation, we hypothesized that our new autophagy inhibitor, compound <b>26</b>, would increase apoptosis in starved cells compared with cells provided with full nutrients. To investigate this, wild-type MDA-MB-468 cells were treated with compound <b>26</b> in normal media (RPMI with 10% FBS) or low-glucose, amino acid-free Earle’s balanced salt solution (EBSS) for 18 h. Apoptosis was measured with Annexin V and 7-AAD using flow cytometry, and the results are shown in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B. In normal media, the same percentages of cells (6.8%) were in late-stage apoptosis regardless of whether they were treated with DMSO or compound <b>26</b> (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B, panels i and ii). Under starvation conditions, however (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B, panels iii and iv), a much larger percentage of cells (18.4%) treated with compound <b>26</b> were in late-stage apoptosis compared with DMSO-treated cells (7.6%). Furthermore, a higher percentages of starved cells (42.9%) underwent early stage apoptosis compared with cells in normal media (28.9%) when treated with compound <b>26</b> (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B, panels ii and iv). The effects of compound <b>26</b> were much more pronounced than the percentages of cells (5.6 and 5.7%) in early stage apoptosis for the DMSO controls (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B, panels i and iii). Taken together, these data are consistent with enhanced cell killing by compound <b>26</b> under starved conditions that activate autophagy.</div></div><div id="sec2_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> ULK1/2 Inhibitors Block Autophagic Flux in TNBC Cells</h3><div class="NLM_p">Autophagic flux is the cellular process, whereby autophagosomes fuse with autolysosomes and subsequently degrade their cargo. To confirm that autophagic flux, and therefore autophagy, is inhibited by compound <b>26</b>, we employed a flow cytometry-based assay using MDA-MB-468 cells expressing a tandem-labeled mCherry-EGFP-LC3 chimeric reporter protein to count the number of autophagosomes and autolysosomes.<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> GFP fluorescence, but not mCherry fluorescence, is quenched in the acidic (pH ≤ 5) environment of lysosomes; therefore, inhibition of autophagic flux is reflected by a decrease in the cellular ratio of mCherry to GFP fluorescence (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A). In the absence of an autophagy stimulus, the MDA-MB-468 cells exhibited low (∼55%) or intermediate (∼40%) basal levels of autophagy with only ∼5% of the cell population in high flux (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>B and <a href="/doi/suppl/10.1021/acs.jmedchem.0c00873/suppl_file/jm0c00873_si_001.pdf" class="ext-link">Figure S6</a>). After incubation (18 h) in EBSS to starve the cells and induce autophagy, the percentage of cells with high autophagic flux increased to >90%. Treatment of starved cells with 10 μM SBI-0206965 or compound <b>26</b> significantly reduced the mCherry:GFP ratio, indicating inhibition of the starvation-induced increase in autophagic flux (by ∼10% and ∼ 60%, respectively; <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>B and <a href="/doi/suppl/10.1021/acs.jmedchem.0c00873/suppl_file/jm0c00873_si_001.pdf" class="ext-link">Figure S6</a>). Autophagosomes in the cells were visualized by transmission electron microscopy (TEM) (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>C). The amount of autophagosomes under different conditions was consistent with the data from the autophagic flux assay, with the highest number of autophagosomes identified in starved cells (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>C, ii). Again, similar to the autophagic flux assay, compound <b>26</b> (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>C, iv) decreased the number of autophagosomes to a greater extent than SBI-0206965 (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>C, iii). These data show that the compounds effectively inhibited autophagy in TNBC cells, with compound <b>26</b>, a more potent ULK1/2 inhibitor than SBI-0206965, being the most effective.</div><figure id="fig6" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00873/20201203/images/medium/jm0c00873_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00873/20201203/images/large/jm0c00873_0007.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. ULK1/2 inhibitors reduce autophagic flux. (A) MDA-MB-468 cells expressing mCherry-EGFP-LC3 fusion protein were used to monitor autophagic flux, as measured by the ratio of mCherry:GFP by flow cytometry. (B) Cells were incubated in normal growth media (control), with EBSS (starved) or EBSS plus the ULK1/2 inhibitors (10 μM) for 18 h before autophagic flux was assessed. Three independent experiments were performed; mean ± SD. *<i>P</i> < 0.001 by two-sided unpaired t-test of the high autophagic flux population. (C) Ultrastructural analysis of cells in (B) by TEM to monitor autophagosome accumulation. Red arrows indicate autophagosomes. (i) Representative untreated control cells, (ii) EBSS (starved) cells, (iii) cells treated with SBI-0206965 (10 μM), and (iv) cells treated with compound <b>26</b> (10 μM). Images are representative of 15–25 cells for each set of conditions analyzed by TEM.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00873/20201203/images/large/jm0c00873_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00873&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> SBP-7455 Exhibits Synergistic Cytotoxicity with Poly (ADP-ribose) Polymerase (PARP) Inhibitors against TNBC Cells</h3><div class="NLM_p">PARPs are a family of proteins that maintain genome stability and are responsible for DNA-damage repair in cells.<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> Since DNA damage can induce cancer cell death, PARP inhibition, which leads to the accumulation of single-stranded breaks, is a viable strategy to treat cancer.<a onclick="showRef(event, 'ref50 ref51'); return false;" href="javascript:void(0);" class="ref ref50 ref51">(50,51)</a> Several PARP inhibitors, including olaparib, niraparib, rucaparib, and talazoparib, have received regulatory approval for the treatment of cancer.<a onclick="showRef(event, 'ref50 ref51'); return false;" href="javascript:void(0);" class="ref ref50 ref51">(50,51)</a> However, the emergence of resistance has led to interest in combination therapies that can resensitize tumor cells to the effects of PARP inhibitors.<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> Recent evidence has emerged that PARP inhibitors activate autophagy in cancer cells, and that this may lead to resistance. For example, inhibition of autophagy with CQ was found to enhance the effects of niraparib in xenograft models of hepatocellular carcinoma, while autophagy inhibition either genetically (ATG5 KD) or with CQ increased the cytotoxicity of talazoparib in models of pediatric chronic myelogenous leukemia (CML).<a onclick="showRef(event, 'ref53 ref54'); return false;" href="javascript:void(0);" class="ref ref53 ref54">(53,54)</a> Because PARP inhibitors have been approved for the treatment of TNBC, we investigated whether combining compound <b>26</b> with PARP inhibitors would increase cytotoxicity against TNBC cells. Activity of the PARP inhibitors olaparib, niraparib, and rucaparib was first tested against MDA-MB-468 cells to determine their potencies as single agents (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00873/suppl_file/jm0c00873_si_001.pdf" class="ext-link">Figure S7</a>). Based on these data, we selected olaparib (IC<sub>50</sub> = 10.1 μM) and niraparib (IC<sub>50</sub> = 7.6 μM) for further studies. We tested different concentrations of compound <b>26</b> with niraparib (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00873/suppl_file/jm0c00873_si_001.pdf" class="ext-link">Figure S9</a>) and olaparib (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.0c00873/suppl_file/jm0c00873_si_001.pdf" class="ext-link">Figure S8</a>) to measure the combined effect of inhibiting both ULK1/2 and PARP. MDA-MB-468 cells were treated with compound <b>26</b> (0–15 μM) and either niraparib or olaparib (0–15 μM) in a dose-reponse matrix, and the synergistic effect was analyzed with Combenefit using the Loewe model.<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a> We observed strong antiproliferative synergy between <b>26</b> and both olaparib (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.0c00873/suppl_file/jm0c00873_si_001.pdf" class="ext-link">Figure S8</a>) and niraparib (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00873/suppl_file/jm0c00873_si_001.pdf" class="ext-link">Figure S9</a>), suggesting that autophagy inhibition enhances the cytotoxic effects of PARP inhibitors toward TNBC cells. Specifically, significant synergy was observed with concentrations of <b>26</b> at 0.021–15 μM and olaparib at 3.8–15 μM (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>), and with <b>26</b> at 0.19–1.7 μM and niraparib at 3.8–7.5 μM (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00873/suppl_file/jm0c00873_si_001.pdf" class="ext-link">Figure S9</a>). Of note, the synergistic effect was greater with olaparib than with niraparib, with the maximum synergy scores of 34 and 26, respectively. These data support the hypothesis that an ULK1/2 inhibitor in combination with a PARP inhibitor may have therapeutic utility for the treatment of TNBC.</div><figure id="fig7" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00873/20201203/images/medium/jm0c00873_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00873/20201203/images/large/jm0c00873_0008.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Compound <b>26</b> (SBP-7455) and the PARP inhibitor olaparib synergize to kill MDA-MB-468 cells. (A) MDA-MB-468 cells were treated with <b>26</b> (0.25 nM–15 μM) and olaparib (0.47–15 μM) for 72 h, and cell viability was assessed (CellTiter-Glo assay). A synergy graph was generated using the Combenefit Loewe model. Experiments were repeated three times (<i>n</i> = 4) with similar results. (B) Normalized cell viability after incubation with DMSO (vehicle control), olaparib (7.5 μM), compound <b>26</b> (0.19 μM), or a combination of both compounds. Mean ± SD. *<i>P</i> < 0.025, **<i>P</i> < 0.01, ***<i>P</i> < 0.001 <i>P</i> vs by two-sided unpaired t-test.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00873/20201203/images/large/jm0c00873_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00873&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> SBP-7455 Reverses Olaparib-Induced Upregulation of Autophagic Flux in TNBC Cells</h3><div class="NLM_p">To investigate the mechanism by which compound <b>26</b> potentiates the cytotoxic effects of PARP inhibitors, we treated MDA-MB-468 cells expressing mCherry-EGFP-LC3 with olaparib (30 μM) and/or <b>26</b> (10 μM) for 48 h and then quantified autophagic flux. Olaparib alone was a strong inducer of autophagy, as demonstrated by an approximately 30% increase in autophagic flux compared with control cells (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.0c00873/suppl_file/jm0c00873_si_001.pdf" class="ext-link">Figure S10</a>). However, concomitant treatment with compound <b>26</b> significantly suppressed the olaparib-induced increase in autophagic flux to a level similar to that observed in untreated control cells (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a> and Figure 10). These data suggest that compound <b>26</b> potentiates olaparib activity, at least in part, by abrogating the ability of olaparib to induce autophagy. Of note, the higher autophagic flux levels in untreated MDA-MB-468 cells in <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a> compared with <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>B can be attributed to the measurement of basal autophagy at different time points, specifically 18 h in <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>B versus 48 h in <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>.</div><figure id="fig8" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00873/20201203/images/medium/jm0c00873_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00873/20201203/images/large/jm0c00873_0009.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Compound <b>26</b> (SBP-7455) inhibits the increase in autophagic flux induced by the PARP inhibitor olaparib. MDA-MB-468 cells expressing mCherry-EGFP-LC3 were treated with DMSO control, compound <b>26</b> (10 μM), olaparib (30 μM), or a combination of both for 48 h. Two independent experiments were performed; mean ± SD. *<i>P</i> < 0.05, **<i>P</i> < 0.01 for the high autophagic flux population by two-sided unpaired t-test.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00873/20201203/images/large/jm0c00873_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00873&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i17">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_67350" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_67350" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">We utilized a combination of rational- and structure-based design, based on a high-resolution structure of our first-generation probe SBI-0206965 bound to ULK2, to create a series of novel small molecule ULK1/2 inhibitors. The best compounds potently inhibited ULK1 and ULK2 in vitro and in cells, as measured by ADP-Glo, PTS, and Nano-BRET assays. SBP-7455 (compound <b>26</b>) was identified as a highly effective inhibitor of phosphorylation of the ULK1 downstream substrates Beclin1 and Vps34 and exhibited promising PK and PD properties when dosed orally in mice. Importantly, inhibition of autophagic flux by shRNA-mediated KD of ULK1 was replicated with the optimized ULK1/2 inhibitor SBP-7455 in MDA-MB-468 TNBC cells.</div><div class="NLM_p">TNBC therapy is particularly challenging due to its resistance to antihormonal and HER2 targeted therapies. PARP inhibitors are effective in a subset (∼20%) of TNBC patients that harbor BRCA mutations;<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a> however, finding effective ways to sensitize WT BRCA TNBC to anti-cancer therapies is an area of ongoing research. Our results suggest that autophagy inhibition may be an attractive option. In the present study, we found that the autophagy inhibitor SBP-7455 was cytotoxic as a single agent against TNBC cells. In a clinical setting, it is likely that autophagy inhibition will be most effectively deployed as an adjunct to existing therapies. Therefore, we also showed that SBP-7455 and the PARP inhibitor olaparib synergistically kill MDA-MB-468 TNBC cells. Thus, in these cells, PARP inhibition combined with autophagy inhibition is more effective than either tumor cell killing mechanism alone.</div><div class="NLM_p last">Our results from the present study are in alignment with data from other groups on overcoming the protective effects of autophagy in advanced cancers. For example, recent findings in pancreatic ductal adenocarcinoma (PDAC) suggest that inhibition of RAF-MEK-ERK signaling activates a compensatory pro-autophagic LKB1-AMPK-ULK1 axis.<a onclick="showRef(event, 'ref57 ref58'); return false;" href="javascript:void(0);" class="ref ref57 ref58">(57,58)</a> Thus, a growing body of preclinical and clinical evidence suggests that autophagy activation is a common mechanism of drug resistance in many advanced cancers; therefore, combination therapies that block autophagy, such as those described here, could have wide therapeutic applications.<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(59)</a> We focused our efforts on the development of inhibitors of ULK1/2 as initiating kinases in the autophagy signaling pathway. It is likely that autophagy inhibitors acting at later steps in the pathway would have distinct toxicity profiles compared with inhibitors of autophagy initiation. Blockade of lysosomal degradation leads to the accumulation of unprocessed autophagosomes, whereas blockade of the upstream stages causes a buildup of the autophagy cargo; namely, protein aggregates and damaged organelles.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> Chemical probes, such as the one described here, that act at different stages in the autophagy pathway could serve as tools to assess the relative importance of each component of the autophagic machinery from a therapeutic perspective.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i18">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_47316" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_47316" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> General Chemistry Information</h3><div class="NLM_p last">All reactions were performed with oven-dried glassware under a nitrogen atmosphere with magnetic stirring. All solvents and chemicals were purchased from commercial sources and used without further purification unless otherwise stated. Reactions conducted under microwave irradiation were performed in a CEM Discover microwave reactor using 10 mL reaction vessels. Reaction progress was monitored by reverse-phase HPLC and/or thin-layer chromatography (TLC). Chromatographic purification was carried out using prepacked silica or C18 cartridges (from RediSep and Luknova) and eluted using an ISCO companion system. Reverse-phase purifications were conducted using water and acetonitrile doped with 0.1% formic acid. Purity and characterization of compounds were established by a combination of LC–MS and NMR. HPLC–MS analyses were performed on a Shimadzu 2010EV LC–MS using the following conditions: a Kromasil C18 column (reverse phase, 4.6 mm _ 50 mm), a linear gradient from 10% acetonitrile and 90% water to 95% acetonitrile and 5% water over 4.5 min, a flow rate of 1 mL/min, and UV photodiode array detection from 200 to 300 nm. High-resolution ESI-TOF mass spectra were acquired from the Scripps Research Institute Center for Metabolomics and Mass Spectrometry (La Jolla, California). Proton (<sup>1</sup>H) and carbon (<sup>13</sup>C) NMR spectra were obtained on a JEOL400 spectrometer at 400 and 101 MHz, respectively. Chemical shifts are reported in δ (ppm) and were internally referenced to deuterated solvent signals. The data for <sup>1</sup>H NMRs are reported in terms of chemical shift (δ ppm), multiplicity, coupling constant (Hz), and proton integration. The data for <sup>13</sup>C NMR are reported in terms of chemical shift (δ ppm) and coupling constant (Hz). The purity of final products submitted for biological testing was determined by analytical HPLC–MS to be ≥95%. For NMR spectrum and LCMS traces of compounds <b>7</b>, <b>15</b>, <b>16</b>, <b>21</b>, <b>23</b>, and <b>26</b>, see the <a href="/doi/suppl/10.1021/acs.jmedchem.0c00873/suppl_file/jm0c00873_si_001.pdf" class="ext-link">Supporting Information</a>.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> 2-Chloro-<i>N</i>-cyclopropyl-5-(trifluoromethyl)pyrimidin-4-amine</h3><div class="NLM_p last">A solution of 2,4-dichloro-5-(trifluoromethyl)pyrimidine (1.00 g, 4.61 mmol), cyclopropylamine (0.32 mL, 4.61 mmol), and <i>N</i>,<i>N</i>-diisopropylethylamine (0.80 mL, 4.61 mmol) in acetonitrile (15 mL) was microwaved at 70 °C for 10 min in a 38 mL pressure vessel. The reaction mixture was then concentrated in vacuo and purified by automated reverse-phase chromatography (water-acetonitrile eluent). Tan solid (0.349 g, 32% yield). LC–MS (ESI) calcd for C<sub>8</sub>H<sub>8</sub>ClF<sub>3</sub>N<sub>3</sub> [M + H]<sup>+</sup>: 238.04; found 238.30. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.39 (s, 1H), 7.93 (d, <i>J</i> = 3.6 Hz, 1H, 1H), 2.88 (dq, <i>J</i> = 7.2, 3.6 Hz, 1H), 0.79–0.72 (m, 2H), 0.70–0.64 (m, 2H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 162.66, 159.73, 155.42, 123.34 (q, <i>J</i> = 271 Hz), 105.31 (q, <i>J</i> = 32 Hz), 24.72, 6.30.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> General Method <b>1</b></h3><div class="NLM_p last">A solution of commercially purchased 2-chloro-<i>N</i>-cyclopropyl-5-(trifluoromethyl)pyrimidin-4-amine (1 equiv) and the appropriate aniline (1 equiv) in acetic acid (2 mL) was microwaved at 110 °C for 10 min and then concentrated in vacuo. The product was purified by automated reverse-phase chromatography (water-acetonitrile eluent).</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> <i>N</i><sup>2</sup>-(Benzo[d][1,3]dioxol-5-yl)-5-(trifluoromethyl)pyrimidine-2,4-diamine (<b>2</b>)</h3><div class="NLM_p last">A solution of 2,4-dichloro-5-(trifluoromethyl)pyrimidine (85 mg, 0.392 mmol), benzo[d][1,3]dioxol-5-amine (54 mg, 0.394 mmol), and <i>N</i>,<i>N</i>-diisopropylethylamine (0.068 mL) in acetonitrile (3 mL) was microwaved at 100 °C for 10 min. Ammonia (0.22 mL of a 7 M solution in MeOH) and <i>N</i>,<i>N-</i>diisopropylethylamine (0.07 mL) were added, and the reaction mixture was microwaved at 100 °C for 10 min, then concentrated in vacuo. Product was purified by automated reverse-phase chromatography. Brown solid (15 mg, 13%). LC–MS (ESI) calcd for C<sub>12</sub>H<sub>10</sub>F<sub>3</sub>N<sub>4</sub>O<sub>2</sub> [M + H]<sup>+</sup>: 299.08, found 299.28. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.39 (s, 1H), 8.15 (s, 1H), 7.54 (s, 2H), 7.11 (d, <i>J</i> = 10.5 Hz, 1H), 6.76 (d, <i>J</i> = 8.4 Hz, 1H), 5.93 (s, 2H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 161.2, 159.8, 155.1, 147.1, 142.2, 134.8, 125.3 (q, <i>J</i> = 269 Hz), 112.5, 107.9, 102.3, 100.9, 96.5 (q, <i>J</i> = 31 Hz).</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> <i>N</i><sup>2</sup>-(Benzo[d][1,3]dioxol-5-yl)-<i>N</i><sup>4</sup>-cyclopropyl-5-(tri-fluoromethyl)pyrimidine-2,4-diamine (<b>3</b>)</h3><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_3" aria-label="General Method 1">General Method 1</a> using 2-chloro-<i>N</i>-cyclopropyl-5-(trifluoromethyl)pyrimidin-4-amine (60 mg, 0.253 mmol) and benzo[d][1,3]dioxol-5-amine (35 mg, 0.253 mmol). Purple solid (56 mg, 66% yield). LC–MS (ESI) calcd for C<sub>15</sub>H<sub>14</sub>F<sub>3</sub>N<sub>4</sub>O<sub>2</sub> [M + H]<sup>+</sup>: 339.11, found 339.00. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.61 (s, 1H), 8.16 (s, 1H), 7.82 (br. s, 1H), 7.18 (dd, <i>J</i> = 8.5, 2.2 Hz, 1H), 7.15 (br. s, 1H), 6.83 (d, <i>J</i> = 8.6 Hz, 1H), 5.95 (s, 2H), 2.79 (dq, <i>J</i> = 6.2, 3.4 Hz, 1H), 0.78 (td, <i>J</i> = 7.2, 4.9 Hz, 2H), 0.67 (dt, <i>J</i> = 7.3, 4.6 Hz, 2H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 160.6, 159.4, 154.4, 146.5, 142.3, 134.3, 124.96 (q, <i>J</i> = 269 Hz), 111.8, 107.8, 102.3, 100.7, 24.5, 6.7.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> <i>N</i><sup>4</sup>-Cyclopropyl-<i>N</i><sup>2</sup>-(2,2-difluorobenzo[d][1,3]dioxol-5-yl)-5-(trifluoromethyl)pyrimidine-2,4-diamine (<b>4</b>)</h3><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_3" aria-label="General Method 1">General Method 1</a> using 2-chloro-<i>N</i>-cyclopropyl-5-(trifluoromethyl)pyrimidin-4-amine (50 mg, 0.210 mmol) and 2,2-difluorobenzo[d][1,3]dioxol-5-amine (36 mg, 0.210 mmol). The crude product was purified by automated normal phase chromatography on silica gel (DCM eluent). White solid (35 mg, 44% yield). LC–MS (ESI) calcd for C<sub>15</sub>H<sub>12</sub>F<sub>5</sub>N<sub>4</sub>O<sub>2</sub> [M + H]<sup>+</sup>: 375.09, found 375.20. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.93 (s, 1H), 8.30 (br. s, 1H), 8.21 (s, 1H), 7.46 (dd, <i>J</i> = 8.8, 2.2 Hz, 1H), 7.30 (d, <i>J</i> = 8.8 Hz, 1H), 2.81 (tq, <i>J</i> = 6.7, 3.1 Hz, 1H), 0.84–0.75 (m, 2H), 0.72–0.66 (m, 2H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 160.5, 159.4, 154.6, 154.5, 142.6, 137.3, 137.2, 131.4 (t, <i>J</i> = 252 Hz), 124.8 (q, <i>J</i> = 269 Hz) 114.2, 109.7, 101.6, 24.5, 6.7.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> <i>N</i><sup>4</sup>-Cyclopropyl-<i>N</i><sup>2</sup>-(2,3-dihydrobenzo[b][1,4]dioxin-6-yl)-5-(trifluoromethyl)pyrimidine-2,4-diamine (<b>5</b>)</h3><div class="NLM_p last">A solution of 2,4-dichloro-5-(trifluoromethyl)pyrimidine (85 mg, 0.392 mmol), cyclopropanamine (22 mg, 0.385 mmol), and <i>N</i>,<i>N-</i>diisopropylethylamine (0.068 mL) in acetonitrile (3 mL) was microwaved at 100 °C for 10 min, then concentrated in vacuo. 2,3-Dihydrobenzo[b][1,4]dioxin-6-amine (59 mg, 0.390 mmol) was added, and the reaction mixture was processed according to <a class="ref internalNav" href="#sec4_3" aria-label="General Method 1">General Method 1</a>. The mixture was microwaved at 120 °C for 10 min. White solid (15 mg, 11%). LC–MS (ESI) calcd for C<sub>16</sub>H<sub>16</sub>F<sub>3</sub>N<sub>4</sub>O<sub>2</sub> [M + H]<sup>+</sup>: 353.12, found 353.31. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.98 (s, 1H), 9.53 (s, 1H), 8.15 (s, 1H), 7.79 (s, 1H), 7.18 (dd, <i>J</i> = 8.8, 2.6 Hz, 1H), 6.75 (d, <i>J</i> = 8.7 Hz, 1H), 4.23–4.15 (m, 4H), 2.79 (br. s, 1H), 0.82 (dt, <i>J</i> = 6.8, 3.4 Hz, 2H), 0.70–0.65 (m, 2H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 160.6, 159.5, 154.5, 154.4, 142.9, 138.3, 134.1, 125.0 (q, <i>J</i> = 269 Hz), 116.5, 112.5, 108.4, 64.3, 64.0, 24.6, 6.7. HRMS (ESI-TOF) calcd for C<sub>16</sub>H<sub>16</sub>F<sub>3</sub>N<sub>4</sub>O<sub>2</sub> [M + H]<sup>+</sup>: 353.1225; found 353.1216.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> <i>N</i><sup>4</sup>-Cyclopropyl-<i>N</i><sup>2</sup>-(3,4-dihydro-2<i>H</i>-benzo[b][1,4] dioxepin-7-yl)-5-(trifluoromethyl)pyrimidine-2,4-diamine (<b>6</b>)</h3><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_3" aria-label="General Method 1">General Method 1</a> using 2-chloro<i>-N</i>-cyclopropyl-5-(trifluoromethyl)pyrimidin-4-amine (80 mg, 0.337 mmol) and 3,4-dihydro-2<i>H</i>-benzo[b][1,4]dioxepin-7-amine (56 mg, 0.337 mmol). White solid (103 mg, 84% yield). LC–MS (ESI) calcd for C<sub>17</sub>H<sub>18</sub>F<sub>3</sub>N<sub>4</sub>O<sub>2</sub> [M + H]<sup>+</sup>: 367.14, found 367.30. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.61 (s, 1H), 8.17 (s, 1H), 7.77 (br. s, 1H), 7.36 (dd, <i>J</i> = 8.8, 2.6 Hz, 1H), 7.16 (br. s, 1H), 6.87 (d, <i>J</i> = 8.6 Hz, 1H), 4.07 (t, <i>J</i> = 5.3 Hz, 2H), 4.03 (t, <i>J</i> = 5.3 Hz, 2H), 2.83–2.76 (m, 1H), 2.05 (q, <i>J</i> = 6.0, 5.4z Hz, 1H), 0.82 (dt, <i>J</i> = 7.1, 3.4 Hz, 2H), 0.67 (td, <i>J</i> = 7.0, 3.5 Hz, 2H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 160.6, 159.4, 154.5, 154.4, 150.9, 145.9, 135.9, 124.9 (q, <i>J</i> = 270 Hz), 121.1, 114.2, 112.6, 70.6, 32.1, 24.6, 6.7. HRMS (ESI-TOF) calcd for C<sub>17</sub>H<sub>18</sub>F<sub>3</sub>N<sub>4</sub>O<sub>2</sub> [M + H]<sup>+</sup>: 367.1382; found 367.1368.</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> <i>N</i><sup>2</sup>-(1<i>H</i>-Benzo[d]imidazol-6-yl)<i>-N</i><sup>4</sup>-cyclopropyl-5-(trifluoromethyl)pyrimidine-2,4-diamine (<b>7</b>)</h3><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_3" aria-label="General Method 1">General Method 1</a> using 2-chloro<i>-N</i>-cyclopropyl-5-(trifluoromethyl)pyrimidin-4-amine (65 mg, 0.274 mmol) and 1<i>H</i>-benzo[d]imidazol-6-amine (36 mg, 0.274 mmol). The mixture was microwaved at 130 °C for 20 min. White solid (41 mg, 45% yield) LC–MS (ESI) calcd for C<sub>15</sub>H<sub>13</sub>F<sub>3</sub>N<sub>6</sub> [M + H]<sup>+</sup>: 335.13; found 335.00. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.70 (s, 1H), 8.49 (br. s, 1H), 8.19 (s, 1H), 8.12 (s, 1H), 7.48 (s, 2H), 7.13 (br. s, 1H), 2.89 (dq, <i>J</i> = 6.9, 3.3 Hz, 1H), 0.93–0.80 (m, 2H), 0.73–0.64 (m, 2H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 160.8, 159.5, 154.5, 154.4, 141.6, 136.7, 135.2, 125.0 (q, <i>J</i> = 269 Hz), 115.8, 115.3, 104.5, 24.6, 6.8. HRMS (ESI-TOF) calcd for C<sub>15</sub>H<sub>14</sub>F<sub>3</sub>N<sub>6</sub> [M + H]<sup>+</sup>: 335.1232; found 335.1223.</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> <i>N</i><sup>4</sup>-Cyclopropyl-<i>N</i><sup>2</sup>-(1<i>H</i>-indol-5-yl)-5-(trifluoromethyl)pyrimidine-2,4-diamine (<b>8</b>)</h3><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_3" aria-label="General Method 1">General Method 1</a> using 2-chloro<i>-N</i>-cyclopropyl-5-(trifluoromethyl)pyrimidin-4-amine (70 mg, 0.295 mmol) and 1<i>H</i>-indol-5-amine (39 mg, 0.295 mmol). Black solid (72 mg, 73% yield). LC–MS (ESI) calcd for C<sub>16</sub>H<sub>15</sub>F<sub>3</sub>N<sub>5</sub> [M + H]<sup>+</sup>: 334.13, found 334.15. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.92 (s, 1H), 9.47 (s, 1H), 8.25 (s, 1H), 8.15 (s, 1H), 7.43 (d, <i>J</i> = 8.8 Hz, 1H), 7.33–7.24 (m, 2H), 7.02 (s, 1H), 6.32 (t, <i>J</i> = 2.6 Hz, 1H), 2.87 (s, 1H), 0.85–0.76 (m, 2H), 0.74–0.64 (m, 2H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 166.2, 161.0, 159.4, 154.5, 154.4, 132.1, 127.5, 125.6, 125.2 (q, <i>J</i> = 269 Hz), 115.5, 110.8, 100.9, 24.5, 6.7. HRMS (ESI-TOF) calcd for C<sub>16</sub>H<sub>15</sub>F<sub>3</sub>N<sub>5</sub> [M + H]<sup>+</sup>: 334.1280; found 334.1267.</div></div><div id="sec4_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> <i>N</i><sup>4</sup>-Cyclopropyl-<i>N</i><sup>2</sup>-(1<i>H</i>-indazol-5-yl)-5-(trifluoro-methyl)pyrimidine-2,4-diamine (<b>9</b>)</h3><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_3" aria-label="General Method 1">General Method 1</a> using 2-chloro<i>-N</i>-cyclopropyl-5-(trifluoromethyl)pyrimidin-4-amine (80 mg, 0.337 mmol) and 1<i>H-</i>indazol-5-amine (45 mg, 0.337 mmol). The product was isolated by precipitation and filtration with acetone. Pink solid (70 mg, 62% yield). LC–MS (ESI) calcd for C<sub>15</sub>H<sub>14</sub>F<sub>3</sub>N<sub>6</sub> [M + H]<sup>+</sup>: 335.13, found 335.15. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.00 (s, 1H), 8.07 (s, 1H), 7.57 (s, 2H), 2.97–2.88 (m, 1H), 0.89–0.82 (m, 2H), 0.78 (dt, <i>J</i> = 7.8, 3.9 Hz, 2H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 172.1, 158.9, 152.6, 145.3, 137.5, 133.5, 133.0, 129.9, 122.8, 122.7 (d, <i>J</i> = 270 Hz), 121.4, 110.6, 25.5, 6.6. HRMS (ESI-TOF) calcd for C<sub>15</sub>H<sub>14</sub>F<sub>3</sub>N<sub>6</sub> [M + H]<sup>+</sup>: 335.1232; found 335.1219.</div></div><div id="sec4_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> <i>N</i><sup>2</sup>-(1<i>H</i>-Benzo[d][1,2,3]triazol-6-yl)-<i>N</i><sup>4</sup>-cyclopropyl-5-(trifluoromethyl)pyrimidine-2,4-diamine (<b>10</b>)</h3><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_3" aria-label="General Method 1">General Method 1</a> using 2-chloro<i>-N</i>-cyclopropyl-5-(trifluoromethyl)pyrimidin-4-amine (90 mg, 0.379 mmol) and 1<i>H-</i>benzo [d][l,2,3]triazol-5-amine (51 mg, 0.379 mmol). The product was isolated by precipitation and filtration with acetone. White solid (97 mg, 76% yield). LC–MS (ESI) calcd for C<sub>14</sub>H<sub>13</sub>F<sub>3</sub>N<sub>7</sub> [M + H]<sup>+</sup>: 336.12, found 336.20. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.27 (br. s, 1H), 8.63 (s, 1H), 8.46 (s, 1H), 7.91 (d, <i>J</i> = 8.8 Hz, 1H), 7.50 (dd, <i>J</i> = 9.0, 1.9 Hz, 1H), 2.93 (dq, <i>J</i> = 7.2, 3.4 Hz, 1H), 0.95 (td, <i>J</i> = 7.2, 4.9 Hz, 2H), 0.75 (dt, <i>J</i> = 7.0, 4.8 Hz, 2H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 159.1, 154.0, 147.0, 138.0, 136.8, 136.0, 119.4, 116.9, 123.0 (q, <i>J</i> = 270 Hz), 102.8, 98.8 (q, <i>J</i> = 34 Hz), 25.5, 6.7. HRMS (ESI-TOF) calcd for C<sub>14</sub>H<sub>13</sub>F<sub>3</sub>N<sub>7</sub> [M + H]<sup>+</sup>: 336.1185; found 336.1186.</div></div><div id="sec4_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i31" class="anchor-spacer"></div><h3 class="article-section__title" id="_i31"> <i>N</i><sup>4</sup>-Cyclopropyl-<i>N</i><sup>2</sup>-(1<i>H</i>-pyrrolo[2,3-b]pyridin-5-yl)-5-(trifluoromethyl)pyrimidine-2,4-diamine (<b>11</b>)</h3><div class="NLM_p last">A solution of 2,4-dichloro-5-(trifluoromethyl)pyrimidine (85 mg, 0.392 mmol), cyclopropanamine (22 mg, 0.385 mmol), and <i>N,N-</i>diisopropylethylamine (0.07 mL) in acetonitrile (3 mL) was microwaved at 100 °C for 10 min. The mixture was concentrated in vacuo and processed according to <a class="ref internalNav" href="#sec4_3" aria-label="General Method 1">General Method 1</a> with the addition of 1<i>H-</i>pyrrolo[2,3-b]pyridin-5-amine (52 mg, 0.391 mmol) in acetic acid (2 mL). The mixture was microwaved at 120 °C for 10 min. Brown solid (13 mg, 10%). LC–MS (ESI) calcd for C<sub>15</sub>H<sub>14</sub>F<sub>3</sub>N<sub>6</sub> [M + H]<sup>+</sup>: 335.12, found 335.25. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.48 (s, 1H), 9.67 (s, 1H), 8.61 (s, 1H), 8.51 (s, 1H), 8.19 (s, 1H), 7.42–7.38 (m, 1H), 7.15–7.07 (m, 1H), 6.37 (dd, <i>J</i> = 3.5, 1.8 Hz, 1H), 2.84 (br. s, 1H), 0.79 (td, <i>J</i> = 7.1, 4.7 Hz, 2H), 0.72–0.65 (m, 2H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 161.2, 159.5, 154.6, 154.6, 144.8, 136.8, 130.0, 126.5, 125.1 (q, <i>J</i> = 269 Hz), 119.0, 99.6, 24.5, 6.7.</div></div><div id="sec4_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> <i>N</i><sup>4</sup>-Cyclobutyl-<i>N</i><sup>2</sup>-(1<i>H</i>-pyrrolo[2,3-b]pyridin-5-yl)-5-(trifluoromethyl)pyrimidine-2,4-diamine (<b>12</b>)</h3><div class="NLM_p last">A solution of 2,4-dichloro-5-(trifluoromethyl)pyrimidine (120 mg, 0.553 mmol), cyclobutanamine (49 mg, 0.689 mmol), and <i>N</i>,<i>N-</i>diisopropylethylamine (0.12 mL) in acetonitrile (3 mL) was microwaved at 100 °C for 10 min. 1<i>H-</i>Pyrrolo[2,3-b]pyridin-5-amine (92 mg, 0.691 mmol) and acetic acid (3 mL) were added, and the reaction mixture was processed according to <a class="ref internalNav" href="#sec4_3" aria-label="General Method 1">General Method 1</a> and microwaved at 100 °C for 20 min. Tan solid (50 mg, 42% yield). LC–MS (ESI) calcd for C<sub>16</sub>H<sub>16</sub>F<sub>3</sub>N<sub>6</sub> [M + H]<sup>+</sup>: 349.14, found 349.25. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.52 (s, 1H), 9.54 (br. s, 1H), 8.42 (s, 1H), 8.31 (s, 1H), 8.18 (s, 1H), 7.42 (t, <i>J</i> = 3.0 Hz, 1H), 6.94 (d, <i>J</i> = 6.9 Hz, 1H), 6.38 (dd, <i>J</i> = 3.3, 1.9 Hz, 1H), 4.60 (br. s, 1H), 2.29–2.08 (m, 4H), 1.74–1.53 (m, 2H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 161.3, 157.2, 154.9, 154.8, 145.0, 137.3, 129.6, 126.6, 125.0 (q, <i>J</i> = 269 Hz), 119.8, 119.0, 99.5, 45.9, 29.9, 14.8.</div></div><div id="sec4_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i33" class="anchor-spacer"></div><h3 class="article-section__title" id="_i33"> <i>N</i><sup>4</sup>-Methyl-<i>N</i><sup>2</sup>-(1<i>H</i>-pyrrolo[2,3-b]pyridin-5-yl)-5-(trifluoromethyl)pyrimidine-2,4-diamine (<b>13</b>)</h3><div class="NLM_p">To a solution of 2,4-dichloro-5-(trifluoromethyl)pyrimidine (400 mg, 1.844 mmol) in dichloromethane and <i>tert</i>-butanol (10 mL, 1:1) at −10 °C under nitrogen was added zinc(II) chloride (502 mg, 3.69 mmol) and stirred at −10 to 0 °C. After 1 h, 1<i>H-</i>pyrrolo[2,3-b]pyridin-5-amine (245 mg, 0.844 mmol) and triethylamine (0.28 mL, 2.03 mmol) were added. The reaction mixture was warmed to room temperature. The dichloromethane was removed in vacuo, and the solid precipitate was filtered and washed with water. The crude product (620 mg) was isolated as a 70:30 mix of isomers, with <i>N-</i>(4-chloro-5-(trifluoromethyl)pyrimidin-2-yl)-l<i>H-</i>pyrrolo[2,3-b]pyridin-5-amine as the major isomer. LC–MS (ESI) calcd for C<sub>12</sub>H<sub>8</sub>ClF<sub>3</sub>N<sub>5</sub> [M + H]<sup>+</sup>: 314.04, found 313.80.</div><div class="NLM_p last">A solution of <i>N-</i>(4-chloro-5-(trifluoromethyl)pyrimidin-2-yl)-l<i>H-</i>pyrrolo[2,3-b]pyridin-5-amine (100 mg, 0.319 mmol, containing 30% of the regioisomer) and methanamine (0.19 mL, 0.96 mmol) in DMF (2 mL) was microwaved at 100 °C for 20 min and concentrated in vacuo. The product was purified by automated reverse-phase chromatography. Tan solid (20 mg, 20% yield). LC–MS (ESI) calcd for C<sub>13</sub>H<sub>12</sub>F<sub>3</sub>N<sub>6</sub> [M + H]<sup>+</sup>: 309.11, found 308.95. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.47 (s, 1H), 9.51 (br. s, 1H), 8.43 (s, 1H), 8.31 (s, 1H), 8.14 (s, 1H), 7.43–7.37 (m, 1H), 7.07 (br. s, 1H), 6.38 (dd, <i>J</i> = 5.0, 2.0 Hz, 1H), 2.91 (d, <i>J</i> = 4.2 Hz, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 161.6, 158.6, 154.5, 154.4, 145.0, 137.4, 129.8, 126.7, 125.3 (q, <i>J</i> = 269 Hz), 119.2, 99.8, 28.1.</div></div><div id="sec4_16" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34"> <i>N</i><sup>4</sup>-Cyclopropyl-<i>N</i><sup>2</sup>-(4-ethyl-3,4-dihydro-2<i>H</i>-pyrido[3,2-b][1,4]oxazin-7-yl)-5-(trifluoromethyl)pyrimidine-2,4-diamine (<b>14</b>)</h3><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_3" aria-label="General Method 1">General Method 1</a> using 2-chloro<i>-N</i>-cyclopropyl-5-(trifluoromethyl)pyrimidin-4-amine (120 mg, 0.505 mmol) and 4-ethyl-3,4-dihydro-2<i>H</i>-pyrido[3,2-b][1,4]oxazin-7-amine (91 mg, 0.505 mmol). Pink solid (76 mg, 40% yield). LC–MS (ESI) calcd for C<sub>17</sub>H<sub>20</sub>F<sub>3</sub>N<sub>6</sub>O [M + H]<sup>+</sup>: 381.17, found 381.33. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.45 (s, 1H), 8.13 (s, 1H), 7.99 (s, 1H), 4.18 (t, <i>J</i> = 4.5 Hz, 2H), 3.53 (q, <i>J</i> = 7.0 Hz, 2H), 3.37 (t, <i>J</i> = 4.4 Hz, 2H), 2.82–2.69 (m, 1H), 1.05 (t, <i>J</i> = 7.0 Hz, 3H), 0.77 (dt, <i>J</i> = 6.9, 3.3 Hz, 2H), 0.65 (m, 2H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 160.8, 159.4, 154.5, 142.5, 138.6, 130.5, 127.7, 125.0 (q, <i>J</i> = 269 Hz), 114.4, 64.3, 44.4, 41.6, 24.4, 11.4, 6.7.</div></div><div id="sec4_17" class="NLM_sec NLM_sec_level_2"><div id="ac_i35" class="anchor-spacer"></div><h3 class="article-section__title" id="_i35"> <i>N</i><sup>2</sup>-(Benzo[d]oxazol-6-yl)-<i>N</i><sup>4</sup>-cyclopropyl-5-(trifluoromethyl)pyrimidine-2,4-diamine (15)</h3><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_3" aria-label="General Method 1">General Method 1</a> using 2-chloro<i>-N</i>-cyclopropyl-5-(trifluoromethyl)pyrimidin-4-amine (65 mg, 0.274 mmol) and benzo[d]oxazol-6-amine (37 mg, 0.274 mmol). The mixture was microwaved at 130 °C for 20 min. Brown solid (4 mg, 4% yield). LC–MS (ESI) calcd for C<sub>15</sub>H<sub>13</sub>F<sub>3</sub>N<sub>5</sub>O [M + H]<sup>+</sup>: 336.11, found 335.95. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.02 (s, 1H), 8.79 (s, 1H), 8.58 (s, 1H), 8.23 (s, 1H), 7.77–7.55 (m, 2H), 7.28 (s, 1H), 2.87 (dq, <i>J</i> = 6.9, 3.4 Hz, 1H), 0.85 (dt, <i>J</i> = 6.8, 3.3 Hz, 2H), 0.76–0.66 (m, 2H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 160.7, 159.5, 154.6, 154.5, 153.3, 149.9, 138.6, 134.0, 124.8 (q, <i>J</i> = 269 Hz), 119.4, 116.8, 101.3, 24.6, 6.7. HRMS (ESI-TOF) calcd for C<sub>15</sub>H<sub>13</sub>F<sub>3</sub>N<sub>5</sub>O [M + H]<sup>+</sup>: 336.1072; found 336.1064.</div></div><div id="sec4_18" class="NLM_sec NLM_sec_level_2"><div id="ac_i36" class="anchor-spacer"></div><h3 class="article-section__title" id="_i36"> <i>N</i><sup>4</sup>-Cyclopropyl-<i>N</i><sup>2</sup>-(quinolin-6-yl)-5-(trifluoromethyl)pyrimidine-2,4-diamine (<b>16</b>)</h3><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_3" aria-label="General Method b">General Method b</a> using 2-chloro<i>-N</i>-cyclopropyl-5-(trifluoromethyl)pyrimidin-4-amine (60 mg, 0.253 mmol), copper (1.6 mg, 0.025 mmol) and quinolin-6-amine (36 mg, 0.253 mmol). The reaction mixture was microwaved at 120 °C for 10 min. The product was isolated by precipitation with acetone and further purified by automated reverse-phase chromatography of the filtrate. White solid (7 mg, 8% yield). LC–MS (ESI) calcd for C<sub>17</sub>H<sub>15</sub>F<sub>3</sub>N<sub>5</sub> [M + H]<sup>+</sup>: 346.13 found 345.95. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i>6): δ 10.09 (s, 1H), 8.76 (d, <i>J</i> = 2.4 Hz, 1H), 8.72 (dd, <i>J</i> = 4.1, 1.7 Hz, 1H), 8.27 (s, 1H), 8.13 (dd, <i>J</i> = 8.3, 1.6 Hz, 1H), 8.04 (dd, <i>J</i> = 9.1, 2.4 Hz, 1H), 7.93 (d, <i>J</i> = 9.1 Hz, 1H), 7.43 (dd, <i>J</i> = 8.3, 4.2 Hz, 1H), 7.29 (s, 1H), 2.99 (dq, <i>J</i> = 7.1, 3.4 Hz, 1H), 0.88 (td, <i>J</i> = 7.1, 4.8 Hz, 2H), 0.78–0.69 (m, 2H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i>6): δ 160.8, 159.5, 154.6 (q, <i>J</i> = 5 Hz), 148.2, 144.2, 138.3, 135.1, 129.0, 128.53, 124.8 (q, <i>J</i> = 270 Hz), 124.1, 121.7, 113.9, 40.2, 39.9, 39.7, 39.5, 39.3, 39.1, 38.9, 24.6, 6.8. HRMS (ESI-TOF) calcd for C<sub>17</sub>H<sub>15</sub>F<sub>3</sub>N<sub>5</sub> [M + H]<sup>+</sup>: 346.1280; found 346.1284.</div></div><div id="sec4_19" class="NLM_sec NLM_sec_level_2"><div id="ac_i37" class="anchor-spacer"></div><h3 class="article-section__title" id="_i37"> <i>N</i><sup>4</sup>-Cyclopropyl-<i>N</i><sup>2</sup>-(quinolin-3-yl)-5-(trifluoromethyl)pyrimidine-2,4-diamine (<b>17</b>)</h3><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_3" aria-label="General Method 1">General Method 1</a> using 2-chloro<i>-N</i>-cyclopropyl-5-(trifiuoromethyl)pyrimidin-4-amine (60 mg, 0.253 mmol) and quinolin-3-amine (36 mg, 0.253 mmol). The mixture was microwaved at 120 °C for 20 min. The product was isolated by precipitation with acetone and further purified by automated reverse-phase chromatography of the filtrate. White solid (8 mg, 9% yield). LC–MS (ESI) calcd for C<sub>17</sub>H<sub>15</sub>F<sub>3</sub>N<sub>5</sub> [M + H]<sup>+</sup>: 346.13 found 345.80. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.20 (s, 1H), 9.12 (d, <i>J</i> = 2.7 Hz, 1H), 9.06 (br. s, 1H), 8.29 (s, 1H), 7.93 (dd, <i>J</i> = 8.5, 1.5 Hz, 1H), 7.79 (dd, <i>J</i> = 7.8, 1.7 Hz, 1H), 7.62–7.49 (m, 2H), 7.35 (br. s, 1H), 2.96 (tq, <i>J</i> = 7.1, 3.7 Hz, 1H), 0.88 (td, <i>J</i> = 7.0, 4.8 Hz, 2H), 0.74 (dt, <i>J</i> = 6.9, 4.9 Hz, 2H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 160.9, 159.5, 154.6, 145.3, 143.4, 134.1, 128.6, 128.1, 127.3, 127.0, 127.0, 124.7 (q, <i>J</i> = 270 Hz), 120.7, 24.6, 6.8. HRMS (ESI-TOF) calcd for C<sub>17</sub>H<sub>15</sub>F<sub>3</sub>N<sub>5</sub> [M + H]<sup>+</sup>: 346.1280; found 346.1281.</div></div><div id="sec4_20" class="NLM_sec NLM_sec_level_2"><div id="ac_i38" class="anchor-spacer"></div><h3 class="article-section__title" id="_i38"> <i>N</i><sup>4</sup>-Cyclopropyl-<i>N</i><sup>2</sup>-(quinoxalin-6-yl)-5-(trifluoromethyl)pyrimidine-2,4-diamine (<b>18</b>)</h3><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_3" aria-label="General Method 1">General Method 1</a> using 2-chloro<i>-N</i>-cyclopropyl-5-(trifluoromethyl)pyrimidin-4-amine (60 mg, 0.253 mmol) and quinoxalin-6-amine (37 mg, 0.253 mmol). The mixture was microwaved at 120 °C for 20 min. White solid (17 mg, 19% yield). LC–MS (ESI) calcd for C<sub>16</sub>H<sub>14</sub>F<sub>3</sub>N<sub>6</sub> [M + H]<sup>+</sup>: 347.13, found 347.10. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.31 (s, 1H), 9.15 (s, 1H), 8.83 (d, <i>J</i> = 1.8 Hz, 1H), 8.74 (d, <i>J</i> = 1.9 Hz, 1H), 8.31 (s, 1H), 8.09 (dd, <i>J</i> = 9.2, 2.3 Hz, 1H), 7.99 (d, <i>J</i> = 9.1 Hz, 1H), 7.45 (s, 1H), 2.91 (dq, <i>J</i> = 7.1, 3.2 Hz, 1H), 1.03 (dt, <i>J</i> = 7.2, 3.6 Hz, 2H), 0.74 (td, <i>J</i> = 7.1, 3.6 Hz, 2H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 160.7, 159.6, 154.7, 154.6, 145.8, 143.7, 143.1, 141.6, 138.6, 129.0, 124.7 (q, <i>J</i> = 273 Hz), 124.5, 114.5, 24.8, 6.8. HRMS (ESI-TOF) calcd for C<sub>16</sub>H<sub>14</sub>F<sub>3</sub>N<sub>6</sub> [M + H]<sup>+</sup>: 347.1232; found 347.1216.</div></div><div id="sec4_21" class="NLM_sec NLM_sec_level_2"><div id="ac_i39" class="anchor-spacer"></div><h3 class="article-section__title" id="_i39"> 6-((4-(Cyclopropylamino)-5-(trifluoromethyl)pyrimidin-2-yl)amino)-3,4<b>-</b>dihydronaphthalen-1(2<i>H</i>)-one (<b>19</b>)</h3><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_3" aria-label="General Method 1">General Method 1</a> using 2-chloro<i>-N</i>-cyclopropyl-5-(trifluoromethyl)pyrimidin-4-amine (60 mg, 0.253 mmol) and 6-amino-3,4-dihydro-naphthalen-1(2<i>H</i>)-one (41 mg, 0.253 mmol). The mixture was microwaved at 110 °C for 20 min and then concentrated in vacuo. Ethyl acetate-dichloromethane was added, and the precipitate filtered to obtain the desired product. Tan solid (61 mg, 67% yield). LC–MS (ESI) calcd for C<sub>18</sub>H<sub>18</sub>F<sub>3</sub>N<sub>4</sub>O [M + H]<sup>+</sup>: 363.15, found 363.15. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.10 (s, 1H), 8.47 (s, 1H), 8.27 (br. s, 1H), 7.92 (br. s, 1H), 7.85 (d, <i>J</i> = 8.7 Hz, 1H), 7.71 (dd, <i>J</i> = 8.7, 2.2 Hz, 1H), 2.95 (tq, <i>J</i> = 7.2, 3.6 Hz, 1H), 2.90 (t, <i>J</i> = 6.0 Hz, 2H), 2.54 (t, <i>J</i> = 6.4 Hz, 2H), 2.02 (p, <i>J</i> = 6.3 Hz, 2H), 0.92–0.80 (m, 2H), 0.84–0.72 (m, 2H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 196.2, 172.1, 159.2, 155.1, 148.5, 145.9, 142.8, 127.6, 127.4, 123.2 (q, <i>J</i> = 270 Hz), 118.4, 117.8, 99.01 (q, <i>J</i> = 34 Hz), 38.0, 29.8, 25.3, 22.9, 7.1. HRMS (ESI-TOF) calcd for C<sub>18</sub>H<sub>18</sub>F<sub>3</sub>N<sub>4</sub>O [M + H]<sup>+</sup>: 363.1433; found 363.1439.</div></div><div id="sec4_22" class="NLM_sec NLM_sec_level_2"><div id="ac_i40" class="anchor-spacer"></div><h3 class="article-section__title" id="_i40"> <i>N</i><sup>4</sup>-Cyclopropyl-<i>N</i><sup>2</sup>-(pyridazin-4-yl)-5-(trifluoromethyl)pyrimidine-2,4-diamine (<b>20</b>)</h3><div class="NLM_p last">Glassware and the stir bar were flame dried, and nitrogen was bubbled through reagents and solvents prior to use. 2-Chloro<i>-N</i>-cyclopropyl-5-(trifluoromethyl)pyrimidin-4-amine (70 mg, 0.295 mmol), pyridazin-4-amine (28 mg, 0.295 mmol), 9,9-dimethyl-(9<i>H-</i>xanthene-4,5-diyl)bis(diphenyl)phosphane (17 mg, 0.029 mmol), diacetoxypalladium (3.3 mg, 0.015 mmol), and cesium carbonate (120 mg, 0.589 mmol) were mixed in 1,4-dioxane (1 mL). The mixture was microwaved at 140 °C for 20 min, filtered through celite with methanol, and concentrated in vacuo. The crude product was purified by automated reverse-phase chromatography. White solid (5 mg, 6% yield). LC–MS (ESI) calcd for C<sub>12</sub>H<sub>12</sub>F<sub>3</sub>N<sub>6</sub> [M + H]<sup>+</sup>: 297.11, found 296.75. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.39 (s, 1H), 9.51 (d, <i>J</i> = 2.1 Hz, 1H), 8.96 (d, <i>J</i> = 5.9 Hz, 1H), 8.35–8.28 (m, 2H), 7.54 (d, <i>J</i> = 2.4 Hz, 1H), 2.87 (dq, <i>J</i> = 7.2, 3.7 Hz, 1H), 0.86 (td, <i>J</i> = 7.0, 4.8 Hz, 2H), 0.70 (dt, <i>J</i> = 7.2, 4.6 Hz, 2H).</div></div><div id="sec4_23" class="NLM_sec NLM_sec_level_2"><div id="ac_i41" class="anchor-spacer"></div><h3 class="article-section__title" id="_i41"> <i>N</i><sup>4</sup>-Cyclopropyl-<i>N</i><sup>2</sup>-(5-methoxypyridin-3-yl)-5-(trifluoromethyl)pyrimidine-2,4-diamine (<b>21</b>)</h3><div class="NLM_p last">Glassware and the stir bar were flame dried, and nitrogen was bubbled through reagents and solvents prior to use. 2-Chloro<i>-N</i>-cyclopropyl-5-(trifluoromethyl)pyrimidin-4-amine (65 mg, 0.274 mmol), 5-methoxypyridin-3-amine (34 mg, 0.274 mmol), (9,9-dimethyl-9<i>H-</i>xanthene-4,5-diyl)bis(diphenylphosphane) (16 mg, 0.027 mmol), diacetoxypalladium (3.1 mg, 0.014 mmol), and cesium carbonate (134 mg, 0.410 mmol) were mixed in 1,4-dioxane (1 mL). The mixture was microwaved at 140 °C for 20 min, filtered through celite with methanol, and concentrated in vacuo. The crude product was purified by automated reverse-phase chromatography. White solid (9 mg, 10% yield). LC–MS (ESI) calcd for C<sub>14</sub>H<sub>15</sub>F<sub>3</sub>N<sub>5</sub>O [M + H]<sup>+</sup>: 326.13, found 325.90. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.85 (s, 1H), 8.56 (d, <i>J</i> = 2.1 Hz, 1H), 8.20 (s, 1H), 8.06 (t, <i>J</i> = 2.3 Hz, 1H), 7.88 (d, <i>J</i> = 2.5 Hz, 1H), 7.25 (br. s, 1H), 3.76 (s, 3H), 2.82 (dq, <i>J</i> = 7.2, 3.4 Hz, 1H), 0.76 (td, <i>J</i> = 7.2, 4.7 Hz, 2H), 0.65 (dt, <i>J</i> = 7.1, 4.8 Hz, 2H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 160.8, 159.4, 155.3, 154.7, 154.6, 137.7, 133.7, 129.2, 124.7 (q, <i>J</i> = 270 Hz), 111.4, 55.5, 24.6, 6.6.</div></div><div id="sec4_24" class="NLM_sec NLM_sec_level_2"><div id="ac_i42" class="anchor-spacer"></div><h3 class="article-section__title" id="_i42"> 2-((4-(Cyclopropylamino)-5-(trifluoromethyl)pyrimidin-2-yl)amino)phenol (<b>22</b>)</h3><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_3" aria-label="General Method 1">General Method 1</a> using 2-chloro<i>-N</i>-cyclopropyl-5-(trifluoromethyl)pyrimidin-4-amine (60 mg, 0.253 mmol) and 2-aminophenol (28 mg, 0.253 mmol). The mixture was microwaved at 110 °C for 20 min and then concentrated in vacuo. Ethyl acetate was added, and the precipitate filtered to obtain the desired product. Brown solid (57 mg, 73% yield). LC–MS (ESI) calcd for C<sub>14</sub>H<sub>14</sub>F<sub>3</sub>N<sub>4</sub>O [M + H]<sup>+</sup>: 31z1.11, found 311.15. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.07 (s, 1H), 8.26 (s, 1H), 8.17 (d, <i>J</i> = 11.6 Hz, 2H), 7.20 (s, 1H), 6.90–6.84 (m, 2H), 6.84–6.74 (m, 1H), 2.84 (dq, <i>J</i> = 7.0, 3.5 Hz, 1H), 0.77 (td, <i>J</i> = 7.1, 4.6 Hz, 2H), 0.73–0.62 (m, 2H).<sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 160.6, 159.5, 154.6, 154.5, 146.8, 127.6, 124.8 (q, <i>J</i> = 270 Hz), 123.0, 120.6, 119.1, 115.2, 24.5, 6.6.</div></div><div id="sec4_25" class="NLM_sec NLM_sec_level_2"><div id="ac_i43" class="anchor-spacer"></div><h3 class="article-section__title" id="_i43"> <i>N</i><sup>2</sup>-(4-Aminophenyl)-<i>N</i><sup>4</sup>-cyclopropyl-5-(trifluoro-methyl)pyrimidine-2,4-diamine (<b>23</b>)</h3><div class="NLM_p last">A solution of 2-chloro<i>-N</i>-cyclopropyl-5-(trifluoromethyl)pyrimidin-4-amine (60 mg, 0.253 mmol) and benzene-1,4-diamine (55 mg, 0.505 mmol) in butan-l-ol (1 mL) was microwaved at 120 °C for 20 min and concentrated in vacuo. Acetone was added, and the mixture was filtered. The filtrate was concentrated to give the desired product. Brown solid (92 mg, 98% yield). LC–MS (ESI) calcd for C<sub>14</sub>H<sub>15</sub>F<sub>3</sub>N<sub>5</sub> [M + H]<sup>+</sup>: 310.13, found 310.00. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.24 (s, 1H), 8.09 (s, 1H), 7.49 (br. s, 1H), 6.95 (s, 1H), 6.49 (d, <i>J</i> = 8.5 Hz, 2H), 4.75 (br. s, 2H), 2.85–2.77 (m, 1H), 0.78–0.68 (m, 2H), 0.67–0.61 (m, 2H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 160.8, 159.4, 154.4 (q, <i>J</i> = 4.6 Hz), 143.8, 129.4, 125.2 (q, <i>J</i> = 269 Hz), 121.1, 113.8, 24.4, 6.6.</div></div><div id="sec4_26" class="NLM_sec NLM_sec_level_2"><div id="ac_i44" class="anchor-spacer"></div><h3 class="article-section__title" id="_i44"> <i>N</i>-(4-((4-(Cyclopropylamino)-5-(trifluoromethyl)pyrimidin-2-yl)amino)phenyl)cyclopropanecarbox-amide (<b>24</b>)</h3><div class="NLM_p last">A solution of <i>N</i><sup>2</sup>-(4-aminophenyl)<i>-N</i><sup>4</sup>-cyclopropyl-5-(trifluoromethyl)pyrimidine-2,4-diamine (20 mg, 0.065 mmol), cyclopropanecarbonyl chloride (6.2 μL, 0.068 mmol), and triethylamine (0.012 mL, 0.084 mmol) in DMF (1 mL) was microwaved at 60 °C for 10 min and concentrated in vacuo. The crude product was purified by automated reverse phase chromatography. White solid (3 mg, 12% yield). LC–MS (ESI) calcd for C<sub>18</sub>H<sub>19</sub>F<sub>3</sub>N<sub>5</sub>O [M + H]<sup>+</sup>: 378.16, found 378.00. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.06 (s, 1H), 9.61 (br. s, 1H), 8.17 (s, 1H), 7.80 (d, <i>J</i> = 8.6 Hz, 2H), 7.48 (d, <i>J</i> = 8.9 Hz, 2H), 7.11 (br. s, 1H), 2.84 (td, <i>J</i> = 7.2, 3.6 Hz, 1H), 1.80–1.69 (m, 1H), 0.84–0.72 (m, 6H), 0.71–0.60 (m, 2H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 171.1, 160.7, 159.4, 154.5, 154.5, 135.5, 133.7, 124.97 (q, <i>J</i> = 269 Hz), 119.6, 119.3, 24.5, 14.0, 6.9, 6.6. HRMS (ESI-TOF) calcd for C<sub>18</sub>H<sub>19</sub>F<sub>3</sub>N<sub>5</sub>O [M + H]<sup>+</sup>: 378.1542; found 378.1530.</div></div><div id="sec4_27" class="NLM_sec NLM_sec_level_2"><div id="ac_i45" class="anchor-spacer"></div><h3 class="article-section__title" id="_i45"> <i>N</i>-(Cyclopropanecarbonyl)-<i>N</i>-(4-((4-(cyclopropyl amino)-5-(trifluoromethyl)pyrimidin-2-yl)amino)phenyl)cyclopropane-carboxamide (<b>25</b>)</h3><div class="NLM_p last">A solution of <i>N</i><sup>2</sup>-(4-aminophenyl)<i>-N</i><sup>4</sup>-cyclopropyl-5-(trifluoromethyl)pyrimidine-2,4-diamine (43 mg, 0.139 mmol), cyclopropanecarbonyl chloride (16 mg, 0.153 mmol) and <i>N-</i>ethyl<i>-N</i>-isopropyl propan-2-amine (0.03 mL, 0.167 mmol) in DMF (1 mL) was microwaved at 60 °C for 10 min. Additional cyclopropanecarbonyl chloride (16 mg, 0.153 mmol) was added and microwaved at 60 °C for an additional 10 min. The crude product was purified by automated reverse-phase chromatography. White solid (10 mg, 16% yield). LC–MS (ESI) calcd for C<sub>22</sub>H<sub>23</sub>F<sub>3</sub>N<sub>5</sub>O<sub>2</sub> [M + H]<sup>+</sup>: 446.18, found 446.35. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.25 (s, 1H), 8.36 (s, 1H), 7.79 (d, <i>J</i> = 8.6 Hz, 1H), 7.57 (d, <i>J</i> = 8.7 Hz, 2H), 7.47 (d, <i>J</i> = 9.1 Hz, 1H), 7.07 (d, <i>J</i> = 8.7 Hz, 2H), 2.76 (dq, <i>J</i> = 7.1, 3.3 Hz, 1H), 2.45–2.35 (m, 1H), 1.82–1.68 (m, 1H), 1.10–0.82 (m, 4H), 0.82–0.73 (m, 4H), 0.70–0.47 (m, 4H).</div></div><div id="sec4_28" class="NLM_sec NLM_sec_level_2"><div id="ac_i46" class="anchor-spacer"></div><h3 class="article-section__title" id="_i46"> <i>N</i><sup>4</sup>-Cyclopropyl-<i>N</i><sup>2</sup>-(3,4-dimethoxyphenyl)-5-(trifluoromethyl)pyrimidine-2,4-diamine (<b>26</b>, SBP-7455)</h3><div class="NLM_p">A solution of 4-dichloro-5-(trifluoromethyl)pyrimidine (503 mg, 2.32 mmol) and zinc chloride (379 mg, 2.78 mmol) in 1,2-dichloroethane and <i>tert</i>-butanol (1:1, 20 mL) was set to stir at 0 °C. After 30 min, a solution of 3,4-dimethoxyaniline (355 mg, 2.32 mmol) and triethylamine (0.40 mL, 2.78 mmol) in 1,2-dichloroethane and <i>tert</i>-butanol (1:1, 2 mL) was added dropwise to the reaction mixture. The reaction mixture was stirred at room temperature for 4 h then concentrated in vacuo. The crude product was triturated with methanol, filtered, and dried to yield 4-chloro<i>-N</i>-(3,4-dimethoxyphenyl)-5-(trifluoromethyl)pyrimidin-2-amine. White solid (702 mg, 91% yield) used crude in the next step. LC–MS (ESI) calcd for C<sub>13</sub>H<sub>12</sub>ClF<sub>3</sub>N<sub>3</sub>O<sub>2</sub> [M + H]<sup>+</sup>: 334.06 found 334.00. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.49 (s, 1H), 8.74 (s, 1H), 7.35 (d, <i>J</i> = 2.6 Hz, 1H), 7.19 (d, <i>J</i> = 7.8 Hz, 1H), 6.93 (d, <i>J</i> = 8.7 Hz, 1H), 3.74 (s, 3H), 3.74 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 160.6, 158.0, 148.6, 145.5, 131.6, 123.1 (q, <i>J</i> = 270 Hz), 112.9, 112.0, 106.1, 55.7, 55.5.</div><div class="NLM_p last">A solution of 4-chloro<i>-N</i>-(3,4-dimethoxyphenyl)-5-(trifluoromethyl)pyrimidin-2-amine (500 mg, 1.50 mmol), cyclopropanamine (125 μL, 1.80 mmol) and <i>N,N-</i>diisopropyl ethylamine (0.31 mL, 1.80 mmol) in DMF (2 mL) was microwaved at 120 °C for 10 min. The crude product was concentrated in vacuo and purified by automated reverse-phase chromatography. White solid (291 mg, 55% yield). LC–MS (ESI) calcd for C<sub>16</sub>H<sub>18</sub>F<sub>3</sub>N<sub>4</sub>O<sub>2</sub> [M + H]<sup>+</sup>: 355.14 found 355.50. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.46 (s, 1H), 8.16 (s, 1H), 7.55 (br. s, 1H), 7.42 (d, <i>J</i> = 2.4 Hz, 1H), 7.05 (br. s, 1H), 6.88 (d, <i>J</i> = 8.8 Hz, 1H), 3.71 (s, 3H), 3.70 (s, 3H), 2.91 (dq, <i>J</i> = 6.9, 3.3 Hz, 1H), 0.77 (td, <i>J</i> = 7.2, 4.9 Hz, 2H), 0.67 (dt, <i>J</i> = 7.3, 4.4 Hz, 2H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 160.8, 159.4, 154.5, 154.4, 148.5, 144.2, 133.8, 125.0 (q, <i>J</i> = 269 Hz), 112.2, 111.4, 105.2, 55.8, 55.6, 24.5, 6.6. HRMS (ESI-TOF) calcd for C<sub>16</sub>H<sub>18</sub>F<sub>3</sub>N<sub>4</sub>O<sub>2</sub> [M + H]<sup>+</sup>: 355.1382; found 355.1388.</div></div><div id="sec4_29" class="NLM_sec NLM_sec_level_2"><div id="ac_i47" class="anchor-spacer"></div><h3 class="article-section__title" id="_i47"> Crystallization and Structure Determination</h3><div class="NLM_p last">ULK2 kinase domain was purified and crystallized as previously described.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> Data collection was performed at an SLS X06SA, and data were processed and scaled with XDS and aimless, respectively.<a onclick="showRef(event, 'ref60 ref61'); return false;" href="javascript:void(0);" class="ref ref60 ref61">(60,61)</a> Molecular replacement was performed using a Phaser and the published ULK2 structure (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6QAV">6QAV</a>).<a onclick="showRef(event, 'ref62'); return false;" href="javascript:void(0);" class="ref ref62">(62)</a> The structure was rebuilt in COOT and refined using REFMAC.<a onclick="showRef(event, 'ref63 ref64'); return false;" href="javascript:void(0);" class="ref ref63 ref64">(63,64)</a> Data collection and refinement statistics are summarized in <a href="/doi/suppl/10.1021/acs.jmedchem.0c00873/suppl_file/jm0c00873_si_001.pdf" class="ext-link">Table S1</a>.</div></div><div id="sec4_30" class="NLM_sec NLM_sec_level_2"><div id="ac_i48" class="anchor-spacer"></div><h3 class="article-section__title" id="_i48"> In Silico Docking</h3><div class="NLM_p last">Fully continuously flexible small-molecule ligand docking to the ULK2 protein, represented by grid interaction potentials, was performed using the ICM docking algorithm [<sup>i</sup>] as implemented in the ICM-Pro program (v3.8, Molsoft, LLC.). The coordinates of the three-dimensional structure of ULK2 (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YID">6YID</a>) reported here were converted into an ICM object, charges were assigned, orientations of sidechain amides were corrected, and hydrogen atoms added and their positions optimized by energy minimization using the MMFF force field. The docking receptor site was defined as an 8 Å radius around the SBI-0206965 ligand. An energy-minimized three-dimensional molecular model of the compound was generated and docking performed using the implemented routine in ICM.</div></div><div id="sec4_31" class="NLM_sec NLM_sec_level_2"><div id="ac_i49" class="anchor-spacer"></div><h3 class="article-section__title" id="_i49"> ULK1/2 ADP-Glo Assay</h3><div class="NLM_p last">The kinase reaction was performed in 5 μL total volume containing 25 μM ATP (Sigma-Aldrich #A7699). For the ULK1 ADP-Glo assay, the reaction employed 2 μg/mL recombinant human ULK1 protein (1-649, SignalChem #U01-11G) and 80 μg/mL myelin basic protein (MBP, Sigma-Aldrich #M1891). The ULK2 assay included 4 μg/mL recombinant human ULK2 protein (1-478, SignalChem #U02-11G) and 80 μg/mL MBP. IC<sub>50</sub> values were determined in 16-dose assay mode with threefold serial dilutions starting at 30 μM compound, performed in triplicate. Staurosporine, a nonselective protein kinase inhibitor, was used as a positive control.</div></div><div id="sec4_32" class="NLM_sec NLM_sec_level_2"><div id="ac_i50" class="anchor-spacer"></div><h3 class="article-section__title" id="_i50"> ULK1/2 NanoBRET Assay</h3><div class="NLM_p last">Human embryonic kidney cells (HEK293T) were transfected with a NanoLuc-ULK1 Fusion Vector (Promega #NV2211) or a NanoLuc-ULK2 Fusion Vector (Promega #NV2221) using the jetPRIME transfection reagent (Polyplus Transfection #114-15) for 24 h. Cells were trypsinized and resuspended in Opti-MEM I (1X) Reduced Serum Medium (Gibco, #11058–021) and plated at approximately 7 × 10<sup>3</sup> cells/34 μL/well total volume in nonbinding surface 384 well plates (Corning #3574). Complete NanoBRET 20× Tracer K-5 reagent, prepared according to the manufacturer’s directions, was added at 2 μL/well, and the plate was mixed on an orbital shaker for 15 s at 700 rpm. Compounds were serially diluted at 10× the final concentration in an assay medium (Opti-MEM I, Reduced Serum Medium) and added to the cells at 4 μL/well, and the assay plate was mixed for 15 s at 700 rpm. The assay plate was incubated for 2 h in a 37 °C incubator with 5% CO<sub>2</sub> and then equilibrated to rt for 15 min. 3X Complete Substrate plus Inhibitor Solution was prepared according to the manufacturer’s directions, with Extracellular NanoLuc Inhibitor (60 μM), and added to the cells at a final working concentration of 20 μM/well. The assay plate and incubated at rt for 2–3 min, and the signals at the donor (450 nm) and acceptor (610 nm) emission wavelengths were measured using a Spark multimode microplate reader (Tecan).</div></div><div id="sec4_33" class="NLM_sec NLM_sec_level_2"><div id="ac_i51" class="anchor-spacer"></div><h3 class="article-section__title" id="_i51"> Thermal Shift Assay</h3><div class="NLM_p last">Recombinant kinase domains of ULK1 and ULK2 (2 μM) were mixed with each ULK1/2 inhibitor (10 μM), and the assay was performed as previously described.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a></div></div><div id="sec4_34" class="NLM_sec NLM_sec_level_2"><div id="ac_i52" class="anchor-spacer"></div><h3 class="article-section__title" id="_i52"> pBeclin1 Assay</h3><div class="NLM_p last">HEK293T cells were incubated in 6-well dishes at 6 × 10<sup>5</sup> cells/well in normal growth media (DMEM containing 10% fetal bovine serum [FBS] and 1% penicilin/streptomycin) for 24 h and then transfected with WT or KI Myc-tagged ULK1 plus WT Flag-tagged Beclin1 using a jetPRIME transfection reagent (Polyplus Transfection #114-15). At 24 h posttransfection, the cells were incubated in fresh media containing DMSO or ULK1/2 inhibitor (10 μM) for 1 h. The cells were lysed in RIPA buffer (Sigma #R0278) containing protease inhibitors (Roche #5892970001) and phosphatase inhibitors (Roche #4906837001). The lysates were immunoblotted with a Beclin1 (pSer15) antibody (Abbiotec #254515) or glyceraldehyde 3-phosphate dehydrogenase (GAPDH; 14C10 Cell Signaling Technology [CST] #2118S) and then stripped with Restore Western Blot Stripping Buffer (Thermo Scientific #21059) and incubated with Beclin1 antibody (CST #3738S) to probe total Beclin1. Densitometry analysis was performed with Image Lab 5.2.1 (Bio-Rad).</div></div><div id="sec4_35" class="NLM_sec NLM_sec_level_2"><div id="ac_i53" class="anchor-spacer"></div><h3 class="article-section__title" id="_i53"> pVPS34 Assay</h3><div class="NLM_p last">HEK293T cells (6 × 10<sup>5</sup>) cells were incubated for 24 h as described for pBeclin1 and then transfected with WT or KI Myc-tagged ULK1 plus WT Flag-tagged Vps34 (WT Vps34) using a standard Lipofectamine 2000 transfection protocol (Invitrogen #11668500). At 24 h posttransfection, the cells were incubated in fresh media containing DMSO or ULK1/2 inhibitor (10 μM) for 1 h, and the cells were lysed with cell lysis buffer (CST #9803). The cell lysates were immunoblotted with anti-Myc tag (CST #2278), anti-FLAG tag polyclonal (Sigma #F7425), and anti-pVPS34 (CST #13857) antibodies. Bands were quantified as described for pBeclin1.</div></div><div id="sec4_36" class="NLM_sec NLM_sec_level_2"><div id="ac_i54" class="anchor-spacer"></div><h3 class="article-section__title" id="_i54"> PK and PD Studies</h3><div class="NLM_p last">All animal care and handling was in accordance with the NIH guidelines for the Care and Use of Laboratory Animals, and the procedures were approved by the Sanford Burnham Prebys Medical Discovery Institute Institutional Animal Care and Use Committee. Female C57BL/6J mice (8 weeks of age) were purchased from the Jackson Laboratory, housed in pathogen-free conditions on a 12 h light/dark cycle, and given free access to food and water. Mice were orally administered compound <b>26</b> (30 mg/kg) formulated in vehicle (5% DMSO, 10% Tween-80, and 85% distilled and deionized H<sub>2</sub>0). At the indicated times of postdosing, blood samples were collected retro-orbitally and plasma was separated by centrifugation. Plasma samples were extracted with acetonitrile: water (4:1) with 0.1% formic acid-containing indomethacin as an internal standard. Samples were centrifuged, and supernatants were diluted with acetonitrile: water and analyzed by LC–MS/MS on a Shimadzu Nexera X2 HPLC coupled to an AB Sciex 6500 QTRAP. For the PD studies, mice were orally administered compound <b>26</b> (10 mg/kg) in a vehicle (5% DMSO, 10% Tween-80, and 85% distilled and deionized H2O) or vehicle alone. The mice were sacrificed 2 h post-dose by cervical dislocation, and liver samples flash frozen in liquid nitrogen then homogenized on ice in CST lysis buffer. The protein lysates were equilibrated for protein levels using a BCA protein assay kit (Pierce) and resolved on 8% gels. The lysates were immunoblotted with pATG13 S318 (Rockland #600-401-C49S), ATG13 (CST #13273), ULK1 (CST #8054), and β-actin (Sigma A5441) antibodies to evaluate on-target inhibition in the livers.</div></div><div id="sec4_37" class="NLM_sec NLM_sec_level_2"><div id="ac_i55" class="anchor-spacer"></div><h3 class="article-section__title" id="_i55"> Lentiviral Preparation and Viral Infection of Cells</h3><div class="NLM_p last">Lentiviruses containing shRNA-encoding pLKO.1 vectors were prepared by the Sanford Burnham Prebys Viral Vector Core Facility following a previously described protocol.<a onclick="showRef(event, 'ref65'); return false;" href="javascript:void(0);" class="ref ref65">(65)</a> The ULK1 and BECN1 target sequences are listed in <a href="/doi/suppl/10.1021/acs.jmedchem.0c00873/suppl_file/jm0c00873_si_001.pdf" class="ext-link">Table S3</a>. For the viral infection, MDA-MB-468 cells were plated at 1 × 10<sup>5</sup>/well of 6-well plates and incubated for 24 h. The viruses were mixed with 8 μg/mL Polybrene (Santa Cruz Biotechnology #sc-134220) and added to the cells. Three days later, infected cells were selected by the addition of 1 μg/mL puromycin (Gibco #A11138-03) for an additional 3 days, harvested by trypsinization, and then dispensed into 96-well plates for experiments.</div></div><div id="sec4_38" class="NLM_sec NLM_sec_level_2"><div id="ac_i56" class="anchor-spacer"></div><h3 class="article-section__title" id="_i56"> Apoptosis Assay</h3><div class="NLM_p last">MDA-MB-468 cells were treated with compound <b>26</b> or DMSO for 18 h after which cells the were trypsinized and washed twice with cold PBS. The cells were then stained using a PE Annexin V Apoptosis Detection Kit I (BD Pharmingen #559763). Specifically, the cells were resuspended in 1× binding buffer at a concentration of 1 × 10<sup>6</sup> cells/mL, and 100 μL of the solution was transferred to a 5 mL culture tube. PE Annexin V (5 μL) and 7-AAD (5 μL) were added to the tube with cells. The mix was incubated for 15 min at room temperature in the dark after gentle vortexing. Next, 1× binding buffer (400 μL) was added to each tube, and flow cytometry to detect PE-Annexin V and 7-AAD was performed within 1 h using a LSRFortessa 14-color (BD Biosciences) to analyze apoptosis. Events of 10,000 were collected for each sample.</div></div><div id="sec4_39" class="NLM_sec NLM_sec_level_2"><div id="ac_i57" class="anchor-spacer"></div><h3 class="article-section__title" id="_i57"> mCherry-GFP-LC3 FACS Assay</h3><div class="NLM_p last">MDA-MB-468 cells stably expressing the mCherry-GFP-LC3 fusion protein were plated in 6-well plates at a density of 6 × 10<sup>5</sup> cells/well in growth media DMEM/F12 + GlutaMAX-1 (Gibco) supplemented with 10% FBS (Gibco) and 1× Anti-Anti (Gibco) and allowed to grow overnight. Cells were treated, trypsinized, and then resuspended in PBS, and analytical cytometry was performed in the Sanford Burnham Prebys flow cytometry core using an LSRFortessa 14-color (BD Biosciences) with a 488 nm and 610 nM laser. For flow cytometry, WT MDA-MB-468 cells were used as the unstained control, and MDA-MB-468 cells expressing either mCherry or GFP were used for compensation. Events of 10,000 were collected for each sample.</div></div><div id="sec4_40" class="NLM_sec NLM_sec_level_2"><div id="ac_i58" class="anchor-spacer"></div><h3 class="article-section__title" id="_i58"> TEM Imaging of Autophagy Vacuoles</h3><div class="NLM_p last">Cells were seeded on 10 cm acrylic culture dishes (Corning) until 70% confluence, fixed in a cold solution containing 4% formaldehyde (EMS), 2.5% glutaraldehyde (EMS), 100 mM cacocylate buffer (Sigma) for 1 min using a microwave oven (at 150 watts, Ted Pella), and additional 40 min at 4 °C. The samples were washed in the same buffer (3 × 15 min), post-fixed with 1% osmium tetroxide (EMS) and 1.2% potassium ferrocyanide in cacodylate buffer (100 mM) for 40 min, washed with buffer (3 × 15 min), stained with 1% uranyl acetate (EMS) for 40 min, and washed with Milliq water (3 × 15 min). The cells were scraped off the plates, transferred to 1.5 mL eppendorfs, and pelleted for 15 min at 15,000 rpm (Eppendorf Centrifuge model 5415R). The cells were dehydrated in an acetone series until 100% (3×) and embedded in an Epon resin. Ultrathin sections of 50 nm were obtained with a diamond knife (Diatome), collected on 300 mesh copper grids, and post-stained with UranyLess (EMS) for 5 min. Sections were observed in a Zeiss Libra 120 operated at 80 kV, and high-resolution images (4K × 4K) obtained with a CCD camera Gatan US4000.</div></div><div id="sec4_41" class="NLM_sec NLM_sec_level_2"><div id="ac_i59" class="anchor-spacer"></div><h3 class="article-section__title" id="_i59"> Statistics</h3><div class="NLM_p last">The 50% inhibitory concentrations (IC<sub>50</sub> values) were determined by analyzing the log of the concentration-response curves by nonlinear regression analysis using Prism 8 (GraphPad).</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes3"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i60"><a href="/doi/suppl/10.1021/acs.jmedchem.0c00873" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_57145" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_57145" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.0c00873?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00873</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Overview of the autophagy pathway, structures of ULK1/2 inhibitors: SBI-0206965 and Shokat compound, data collection and refinement statistics for the ULK2 structure, representative dose–response curves of compounds in the ADP-Glo assay, representative dose–response curves of compounds in the NanoBRET assay, protein thermal shift data of selected compounds, shRNA target sequences for knockdown studies, Western blot analysis of shRNA-mediated ULK1 and BECN1 knockdown in three TNBC cell lines, data showing the downregulation of starvation-induced autophagic flux in MDA-MB-468 cells by ULK inhibitors, data showing that PARP inhibitors kill MDA-MB-468 cells with low potency, data showing that SBP-7455 and olaparib synergize to inhibit the viability of MDA-MB-468 cells, data showing that SBP-7455 and niraparib synergize to inhibit the viability of MDA-MB-468 cells, data showing that induction of autophagic flux in MDA-MB-468 cells by olaparib is inhibited by concomitant treatment with SBP-7455, kinetic solubility of selected compounds, bidirectional permeability of selected compounds in Caco-2 cells, NMR and LC–MS spectra of selected compounds, and molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00873/suppl_file/jm0c00873_si_001.pdf" class="ext-link">PDF</a>).</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00873/suppl_file/jm0c00873_si_001.pdf">jm0c00873_si_001.pdf (2.53 MB)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">X-ray crystal structure of SBI-0206965 in complex with ULK2, PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YID">6YID</a>. Authors will release the atomic coordinates and experimental data upon article publication.</p><div class="testing" data-doi="10.1021/acs.jmedchem.0c00873" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_14117" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_14117" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Nicholas D. P. Cosford</span> - <span class="hlFld-Affiliation affiliation">Cancer
Molecules & Structures Program, NCI-Designated Cancer Center, Sanford Burnham Prebys Medical Discovery Institute, La Jolla, California 92037, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-7228-4709" title="Orcid link">http://orcid.org/0000-0001-7228-4709</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#48262b273b2e273a2c083b2a382c213b2b273e2d3a3166273a2f"><span class="__cf_email__" data-cfemail="c9a7aaa6baafa6bbad89baabb9ada0baaaa6bfacbbb0e7a6bbae">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Huiyu Ren</span> - <span class="hlFld-Affiliation affiliation">Cancer
Molecules & Structures Program, NCI-Designated Cancer Center, Sanford Burnham Prebys Medical Discovery Institute, La Jolla, California 92037, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Nicole A. Bakas</span> - <span class="hlFld-Affiliation affiliation">Cancer
Molecules & Structures Program, NCI-Designated Cancer Center, Sanford Burnham Prebys Medical Discovery Institute, La Jolla, California 92037, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mitchell Vamos</span> - <span class="hlFld-Affiliation affiliation">Cancer
Molecules & Structures Program, NCI-Designated Cancer Center, Sanford Burnham Prebys Medical Discovery Institute, La Jolla, California 92037, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Apirat Chaikuad</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, Buchmann Institute for Molecular Life Sciences, Goethe-University Frankfurt, Frankfurt 60438, Germany</span>; 
    <span class="hlFld-Affiliation affiliation">Institute
of Pharmaceutical Chemistry, Goethe-University
Frankfurt, Frankfurt 60438, Germany</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-1120-2209" title="Orcid link">http://orcid.org/0000-0003-1120-2209</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Allison S. Limpert</span> - <span class="hlFld-Affiliation affiliation">Cancer
Molecules & Structures Program, NCI-Designated Cancer Center, Sanford Burnham Prebys Medical Discovery Institute, La Jolla, California 92037, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Carina D. Wimer</span> - <span class="hlFld-Affiliation affiliation">Cancer
Molecules & Structures Program, NCI-Designated Cancer Center, Sanford Burnham Prebys Medical Discovery Institute, La Jolla, California 92037, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sonja N. Brun</span> - <span class="hlFld-Affiliation affiliation">Molecular
and Cell Biology Laboratory, The Salk Institute
for Biological Studies, La Jolla, San Diego, California 92037, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lester J. Lambert</span> - <span class="hlFld-Affiliation affiliation">Cancer
Molecules & Structures Program, NCI-Designated Cancer Center, Sanford Burnham Prebys Medical Discovery Institute, La Jolla, California 92037, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lutz Tautz</span> - <span class="hlFld-Affiliation affiliation">Cancer
Molecules & Structures Program, NCI-Designated Cancer Center, Sanford Burnham Prebys Medical Discovery Institute, La Jolla, California 92037, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Maria Celeridad</span> - <span class="hlFld-Affiliation affiliation">Cancer
Molecules & Structures Program, NCI-Designated Cancer Center, Sanford Burnham Prebys Medical Discovery Institute, La Jolla, California 92037, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Douglas J. Sheffler</span> - <span class="hlFld-Affiliation affiliation">Cancer
Molecules & Structures Program, NCI-Designated Cancer Center, Sanford Burnham Prebys Medical Discovery Institute, La Jolla, California 92037, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Stefan Knapp</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, Buchmann Institute for Molecular Life Sciences, Goethe-University Frankfurt, Frankfurt 60438, Germany</span>; 
    <span class="hlFld-Affiliation affiliation">Institute
of Pharmaceutical Chemistry, Goethe-University
Frankfurt, Frankfurt 60438, Germany</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-5995-6494" title="Orcid link">http://orcid.org/0000-0001-5995-6494</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Reuben J. Shaw</span> - <span class="hlFld-Affiliation affiliation">Molecular
and Cell Biology Laboratory, The Salk Institute
for Biological Studies, La Jolla, San Diego, California 92037, United States</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>The manuscript was written through contributions of all authors. All authors have given approval to the final version of the manuscript.</p></li><li><span class="author-information-subsection-header">Funding</span><p>This work was supported by the Epstein Family Foundation, a Sanford Burnham Prebys National Cancer Institute Cancer Center Support Grant P30 CA030199, a Larry L. Hillblom Foundation Grant 2019-A-005-NET, and a 2019 Pancreatic Cancer Action Network Translational Research Grant 19-65-COSF to NDPC. NAB was supported by the National Cancer Institute of the National Institutes of Health under award number T32CA211036. S.K. and A.C. are grateful for support from the SGC, a registered charity (no. 1097737) that receives funds from AbbVie, Bayer Pharma AG, Boehringer Ingelheim, Canada Foundation for Innovation, Eshelman Institute for Innovation, Genome Canada through Ontario Genomics Institute [OGI-196], EU/EFPIA/OICR/McGill/KTH/Diamond, Innovative Medicines Initiative 2 Joint Undertaking (EUbOPEN grant 875510), Janssen, Merck KGaA (a.k.a. EMD in Canada and US), Merck & Co (aka MSD outside Canada and US), Pfizer, São Paulo Research Foundation-FAPESP, Takeda, and Wellcome. S.K. and A.C. are also grateful for funding by the DKTK and FCI cancer network as well as the DFG funded SFB1177 (selective autophagy). The content is solely the responsibility of the authors and does not necessarily represent the official views of the National Institutes of Health.</p></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i63">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_60585" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_60585" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We thank Dr. Andrew Thorburn (U.C. Denver) for providing the MDA-MB-468 cells expressing mCherry-GFP-LC3, Dr. Do-Hyung Kim (University of Minnesota) for sharing the ULK1 shRNA construct, the Viral Vector Core Facility at Sanford Burnham Prebys Medical Discovery Institute (SBP) for generating lentiviruses, the Flow Cytometry Core at SBP for help generating the data in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B, and the Waitt Advanced Biophotonics Core at the Salk Institute for Biological Studies for taking the TEM images. The authors thank the staff at Swiss Light Source for their assistance with data collection, which was also supported by the funding from the European Union’s Horizon 2020 research and innovation program under grant agreement number 730872, project CALIPSOplus.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">ULK</td><td class="NLM_def"><p class="first last">Unc-51-like autophagy activating kinase</p></td></tr><tr><td class="NLM_term">TNBC</td><td class="NLM_def"><p class="first last">triple-negative breast cancer</p></td></tr><tr><td class="NLM_term">PARP</td><td class="NLM_def"><p class="first last">poly (ADP-ribose) polymerase</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structure–activity relationship</p></td></tr><tr><td class="NLM_term">PTS</td><td class="NLM_def"><p class="first last">protein thermal shift</p></td></tr><tr><td class="NLM_term">GFP</td><td class="NLM_def"><p class="first last">green fluorescent protein</p></td></tr><tr><td class="NLM_term">LC3</td><td class="NLM_def"><p class="first last">MAP1LC3B</p></td></tr><tr><td class="NLM_term">TEM</td><td class="NLM_def"><p class="first last">transmission electron microscope</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i65">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_51730" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_51730" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 65 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bento, C. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renna, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghislat, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Puri, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ashkenazi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vicinanza, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Menzies, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rubinsztein, D. C.</span></span> <span> </span><span class="NLM_article-title">Mammalian autophagy: how does it work?</span>. <i>Annu. Rev. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>85</i></span>,  <span class="NLM_fpage">685</span>– <span class="NLM_lpage">713</span>, <span class="refDoi"> DOI: 10.1146/annurev-biochem-060815-014556</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1146%2Fannurev-biochem-060815-014556" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=26865532" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BC28XisVGitL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=85&publication_year=2016&pages=685-713&author=C.+F.+Bentoauthor=M.+Rennaauthor=G.+Ghislatauthor=C.+Puriauthor=A.+Ashkenaziauthor=M.+Vicinanzaauthor=F.+M.+Menziesauthor=D.+C.+Rubinsztein&title=Mammalian+autophagy%3A+how+does+it+work%3F&doi=10.1146%2Fannurev-biochem-060815-014556"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Mammalian Autophagy: How Does It Work?</span></div><div class="casAuthors">Bento, Carla F.; Renna, Maurizio; Ghislat, Ghita; Puri, Claudia; Ashkenazi, Avraham; Vicinanza, Mariella; Menzies, Fiona M.; Rubinsztein, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Biochemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">85</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">685-713</span>CODEN:
                <span class="NLM_cas:coden">ARBOAW</span>;
        ISSN:<span class="NLM_cas:issn">0066-4154</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews</span>)
        </div><div class="casAbstract">Autophagy is a conserved intracellular pathway that delivers cytoplasmic contents to lysosomes for degrdn. via double-membrane autophagosomes.  Autophagy substrates include organelles such as mitochondria, aggregate-prone proteins that cause neurodegeneration and various pathogens.  Thus, this pathway appears to be relevant to the pathogenesis of diverse diseases, and its modulation may have therapeutic value.  Here, we focus on the cell and mol. biol. of mammalian autophagy and review the key proteins that regulate the process by discussing their roles and how these may be modulated by posttranslational modifications.  We consider the membrane-trafficking events that impact autophagy and the questions relating to the sources of autophagosome membrane(s).  Finally, we discuss data from structural studies and some of the insights these have provided.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqp1Jf2PY-Uv7Vg90H21EOLACvtfcHk0ljkwPFcoSZ4PA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XisVGitL0%253D&md5=f77b5661956ce2b101bc7ff32a02908e</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1146%2Fannurev-biochem-060815-014556&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-biochem-060815-014556%26sid%3Dliteratum%253Aachs%26aulast%3DBento%26aufirst%3DC.%2BF.%26aulast%3DRenna%26aufirst%3DM.%26aulast%3DGhislat%26aufirst%3DG.%26aulast%3DPuri%26aufirst%3DC.%26aulast%3DAshkenazi%26aufirst%3DA.%26aulast%3DVicinanza%26aufirst%3DM.%26aulast%3DMenzies%26aufirst%3DF.%2BM.%26aulast%3DRubinsztein%26aufirst%3DD.%2BC.%26atitle%3DMammalian%2520autophagy%253A%2520how%2520does%2520it%2520work%253F%26jtitle%3DAnnu.%2520Rev.%2520Biochem.%26date%3D2016%26volume%3D85%26spage%3D685%26epage%3D713%26doi%3D10.1146%2Fannurev-biochem-060815-014556" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kimmelman, A. C.</span></span> <span> </span><span class="NLM_article-title">The dynamic nature of autophagy in cancer</span>. <i>Genes Dev.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">1999</span>– <span class="NLM_lpage">2010</span>, <span class="refDoi"> DOI: 10.1101/gad.17558811</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1101%2Fgad.17558811" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=21979913" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlyhtr%252FK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2011&pages=1999-2010&author=A.+C.+Kimmelman&title=The+dynamic+nature+of+autophagy+in+cancer&doi=10.1101%2Fgad.17558811"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">The dynamic nature of autophagy in cancer</span></div><div class="casAuthors">Kimmelman, Alec C.</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">1999-2010</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">A review.  Macroautophagy (referred to hereafter as autophagy) is a highly regulated cellular process that serves to remove damaged proteins and organelles from the cell.  Autophagy contributes to an array of normal and pathol. processes, and has recently emerged as a key regulator of multiple aspects of cancer biol.  The role of autophagy in cancer is complex and is likely dependent on tumor type, stage, and genetic context.  This complexity is illustrated by the identification of settings where autophagy acts potently to either promote or inhibit tumorigenesis.  This review discusses the underlying basis for these opposing functions and propose a model suggesting a dynamic role for autophagy in malignancy.  Collectively, the data point to autophagy as serving as a barrier to limit tumor initiation.  Once neoplastic lesions are established, it appears that adaptive changes occur that now result in pos. roles for autophagy in malignant progression and in subsequent tumor maintenance.  Remarkably, constitutive activation of autophagy is crit. for continued growth of some tumors, serving to both reduce oxidative stress and provide key intermediates to sustain cell metab.  Autophagy is also induced in response to cancer therapies where it can function as a survival mechanism that limits drug efficacy.  These findings have inspired significant interest in applying anti-autophagy therapies as an entirely new approach to cancer treatment.  It is now apparent that aberrant control of autophagy is among the key hallmarks of cancer.  While much needs to be learned about the regulation and context-dependent biol. functions of autophagy, it seems clear that modulation of this process will be an attractive avenue for future cancer therapeutic approaches.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq4HmXPQn60FbVg90H21EOLACvtfcHk0ljkwPFcoSZ4PA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlyhtr%252FK&md5=cde8efa9551e55503d5ace828906c068</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1101%2Fgad.17558811&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.17558811%26sid%3Dliteratum%253Aachs%26aulast%3DKimmelman%26aufirst%3DA.%2BC.%26atitle%3DThe%2520dynamic%2520nature%2520of%2520autophagy%2520in%2520cancer%26jtitle%3DGenes%2520Dev.%26date%3D2011%26volume%3D25%26spage%3D1999%26epage%3D2010%26doi%3D10.1101%2Fgad.17558811" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klionsky, D. J.</span></span> <span> </span><span class="NLM_article-title">Eaten alive: a history of macroautophagy</span>. <i>Nat. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">814</span>, <span class="refDoi"> DOI: 10.1038/ncb0910-814</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1038%2Fncb0910-814" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=20811353" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtFSju7zE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2010&pages=814&author=Z.+Yangauthor=D.+J.+Klionsky&title=Eaten+alive%3A+a+history+of+macroautophagy&doi=10.1038%2Fncb0910-814"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Eaten alive: a history of macroautophagy</span></div><div class="casAuthors">Yang, Zhifen; Klionsky, Daniel J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Cell Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">814-822</span>CODEN:
                <span class="NLM_cas:coden">NCBIFN</span>;
        ISSN:<span class="NLM_cas:issn">1465-7392</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Macroautophagy (hereafter autophagy), or 'self-eating', is a conserved cellular pathway that controls protein and organelle degrdn., and has essential roles in survival, development and homeostasis.  Autophagy is also integral to human health and is involved in physiol., development, lifespan and a wide range of diseases, including cancer, neurodegeneration and microbial infection.  Although research on this topic began in the late 1950s, substantial progress in the mol. study of autophagy has taken place during only the past 15 years.  This review traces the key findings that led to our current mol. understanding of this complex process.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJ_iZJ3YUoHrVg90H21EOLACvtfcHk0ljW1wbPXoFJ9Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtFSju7zE&md5=a20abac7db3f20eac668c41052413397</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fncb0910-814&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncb0910-814%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DZ.%26aulast%3DKlionsky%26aufirst%3DD.%2BJ.%26atitle%3DEaten%2520alive%253A%2520a%2520history%2520of%2520macroautophagy%26jtitle%3DNat.%2520Cell%2520Biol.%26date%3D2010%26volume%3D12%26spage%3D814%26doi%3D10.1038%2Fncb0910-814" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Amaravadi, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lum, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bui, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christophorou, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evan, G. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas-Tikhonenko, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, C. B.</span></span> <span> </span><span class="NLM_article-title">Autophagy inhibition enhances therapy-induced apoptosis in a Myc-induced model of lymphoma</span>. <i>J. Clin. Invest.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>117</i></span>,  <span class="NLM_fpage">326</span>– <span class="NLM_lpage">336</span>, <span class="refDoi"> DOI: 10.1172/JCI28833</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1172%2FJCI28833" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=17235397" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhs1Oqsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=117&publication_year=2007&pages=326-336&author=R.+K.+Amaravadiauthor=D.+Yuauthor=J.+J.+Lumauthor=T.+Buiauthor=M.+A.+Christophorouauthor=G.+I.+Evanauthor=A.+Thomas-Tikhonenkoauthor=C.+B.+Thompson&title=Autophagy+inhibition+enhances+therapy-induced+apoptosis+in+a+Myc-induced+model+of+lymphoma&doi=10.1172%2FJCI28833"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Autophagy inhibition enhances therapy-induced apoptosis in a Myc-induced model of lymphoma</span></div><div class="casAuthors">Amaravadi, Ravi K.; Yu, Duonan; Lum, Julian J.; Bui, Thi; Christophorou, Maria A.; Evan, Gerard I.; Thomas-Tikhonenko, Andrei; Thompson, Craig B.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Clinical Investigation</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">117</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">326-336</span>CODEN:
                <span class="NLM_cas:coden">JCINAO</span>;
        ISSN:<span class="NLM_cas:issn">0021-9738</span>.
    
            (<span class="NLM_cas:orgname">American Society for Clinical Investigation</span>)
        </div><div class="casAbstract">Autophagy is a lysosome-dependent degradative pathway frequently activated in tumor cells treated with chemotherapy or radiation.  Whether autophagy obsd. in treated cancer cells represents a mechanism that allows tumor cells to survive therapy or a mechanism for initiating a nonapoptotic form of programmed cell death remains controversial.  To address this issue, the role of autophagy in a Myc-induced model of lymphoma generated from cells derived from p53ERTAM/p53ERTAM mice (with ER denoting estrogen receptor) was examd.  Such tumors are resistant to apoptosis due to a lack of nuclear p53.  Systemic administration of tamoxifen led to p53 activation and tumor regression followed by tumor recurrence.  Activation of p53 was assocd. with the rapid appearance of apoptotic cells and the induction of autophagy in surviving cells.  Inhibition of autophagy with either chloroquine or ATG5 short hairpin RNA (shRNA) enhanced the ability of either p53 activation or alkylating drug therapy to induce tumor cell death.  These studies provide evidence that autophagy serves as a survival pathway in tumor cells treated with apoptosis activators and a rationale for the use of autophagy inhibitors such as chloroquine in combination with therapies designed to induce apoptosis in human cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKpX84jI9L8rVg90H21EOLACvtfcHk0ljW1wbPXoFJ9Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhs1Oqsbs%253D&md5=bad463df500a94f489629685badb59e8</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1172%2FJCI28833&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1172%252FJCI28833%26sid%3Dliteratum%253Aachs%26aulast%3DAmaravadi%26aufirst%3DR.%2BK.%26aulast%3DYu%26aufirst%3DD.%26aulast%3DLum%26aufirst%3DJ.%2BJ.%26aulast%3DBui%26aufirst%3DT.%26aulast%3DChristophorou%26aufirst%3DM.%2BA.%26aulast%3DEvan%26aufirst%3DG.%2BI.%26aulast%3DThomas-Tikhonenko%26aufirst%3DA.%26aulast%3DThompson%26aufirst%3DC.%2BB.%26atitle%3DAutophagy%2520inhibition%2520enhances%2520therapy-induced%2520apoptosis%2520in%2520a%2520Myc-induced%2520model%2520of%2520lymphoma%26jtitle%3DJ.%2520Clin.%2520Invest.%26date%3D2007%26volume%3D117%26spage%3D326%26epage%3D336%26doi%3D10.1172%2FJCI28833" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hosokawa, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sasaki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iemura, S.-i.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Natsume, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hara, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mizushima, N.</span></span> <span> </span><span class="NLM_article-title">Atg101, a novel mammalian autophagy protein interacting with Atg13</span>. <i>Autophagy</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">973</span>– <span class="NLM_lpage">979</span>, <span class="refDoi"> DOI: 10.4161/auto.5.7.9296</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.4161%2Fauto.5.7.9296" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=19597335" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BC3cXmt1ChtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2009&pages=973-979&author=N.+Hosokawaauthor=T.+Sasakiauthor=S.-i.+Iemuraauthor=T.+Natsumeauthor=T.+Haraauthor=N.+Mizushima&title=Atg101%2C+a+novel+mammalian+autophagy+protein+interacting+with+Atg13&doi=10.4161%2Fauto.5.7.9296"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Atg101, a novel mammalian autophagy protein interacting with Atg13</span></div><div class="casAuthors">Hosokawa, Nao; Sasaki, Takahiro; Lemura, Shun-ichiro; Natsume, Tohru; Hara, Taichi; Mizushima, Noboru</div><div class="citationInfo"><span class="NLM_cas:title">Autophagy</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">973-979</span>CODEN:
                <span class="NLM_cas:coden">AUTOC9</span>;
        ISSN:<span class="NLM_cas:issn">1554-8627</span>.
    
            (<span class="NLM_cas:orgname">Landes Bioscience</span>)
        </div><div class="casAbstract">Autophagy is a major route by which cytoplasmic contents are delivered to the lysosome for degrdn.  Many autophagy-related (ATG) genes have been identified in yeast.  Although most of them are conserved in human, the mol. compn. of the Atg1 complex appears to differ between yeast and mammals.  In yeast, Atg1 forms a complex with Atg11, Atg13, Atg17, Atg29 and Atg31, whereas mammalian Atg1 (ULK1/2) interacts with Atg13 and FIP200.  Here, we identify a novel mammalian Atg13 binding protein, named Atg101.  Atg101 shows no homol. with other Atg proteins, and is conserved in various eukaryotes, but not in Saccharomyces cerevisiae.  Atg101 assocs. with the ULK-Atg13-FIP200 complex, most likely through direct interaction with Atg13.  In Atg13 siRNA-treated cells, Atg101 is present solely as a monomer.  Interaction between Atg101 and the ULK-Atg13-FIP200 complex is stable, and is not regulated by nutrient conditions.  GFP-Atg101 localizes to the isolation membrane/phagophore.  GFP-LC3 dot formation is suppressed and endogenous LC3-I accumulates in Atg101 siRNA-treated cells, suggesting that Atg101 is a crit. factor for autophagy.  Furthermore, Atg101 is important for the stability and basal phosphorylation of Atg13 and ULK1.  These data suggest that Atg101 is a novel Atg protein that functions together with ULK, Atg13 and FIP200.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXscuMfT-o27Vg90H21EOLACvtfcHk0ljW1wbPXoFJ9Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXmt1ChtQ%253D%253D&md5=f73ad092947c3835c9ffdf05921a2118</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.4161%2Fauto.5.7.9296&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252Fauto.5.7.9296%26sid%3Dliteratum%253Aachs%26aulast%3DHosokawa%26aufirst%3DN.%26aulast%3DSasaki%26aufirst%3DT.%26aulast%3DIemura%26aufirst%3DS.-i.%26aulast%3DNatsume%26aufirst%3DT.%26aulast%3DHara%26aufirst%3DT.%26aulast%3DMizushima%26aufirst%3DN.%26atitle%3DAtg101%252C%2520a%2520novel%2520mammalian%2520autophagy%2520protein%2520interacting%2520with%2520Atg13%26jtitle%3DAutophagy%26date%3D2009%26volume%3D5%26spage%3D973%26epage%3D979%26doi%3D10.4161%2Fauto.5.7.9296" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ganley, I. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, X.</span></span> <span> </span><span class="NLM_article-title">ULK1· ATG13 FIP200 complex mediates mTOR signaling and is essential for autophagy</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>284</i></span>,  <span class="NLM_fpage">12297</span>– <span class="NLM_lpage">12305</span>, <span class="refDoi"> DOI: 10.1074/jbc.M900573200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1074%2Fjbc.M900573200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=19258318" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BD1MXltVCmu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=284&publication_year=2009&pages=12297-12305&author=I.+G.+Ganleyauthor=D.+H.+Lamauthor=J.+Wangauthor=X.+Dingauthor=S.+Chenauthor=X.+Jiang&title=ULK1%C2%B7+ATG13+FIP200+complex+mediates+mTOR+signaling+and+is+essential+for+autophagy&doi=10.1074%2Fjbc.M900573200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">ULK1·ATG13·FIP200 Complex Mediates mTOR Signaling and Is Essential for Autophagy</span></div><div class="casAuthors">Ganley, Ian G.; Lam, Du H.; Wang, Junru; Ding, Xiaojun; Chen, She; Jiang, Xuejun</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">284</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">12297-12305</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Autophagy is a degradative process that recycles long-lived and faulty cellular components.  It is linked to many diseases and is required for normal development.  ULK1, a mammalian serine/threonine protein kinase, plays a key role in the initial stages of autophagy, though the exact mol. mechanism is unknown.  Here we report identification of a novel protein complex contg. ULK1 and two addnl. protein factors, FIP200 and ATG13, all of which are essential for starvation-induced autophagy.  Both FIP200 and ATG13 are crit. for correct localization of ULK1 to the pre-autophagosome and stability of ULK1 protein.  Addnl., using both cellular expts. and a de novo in vitro reconstituted reaction we demonstrate that FIP200 and ATG13 can enhance ULK1 kinase activity individually, but both are required for maximal stimulation.  Further, we show that ATG13 and ULK1 are phosphorylated by the mTOR pathway in a nutrient starvation-regulated manner, indicating that the ULK1·ATG13·FIP200 complex acts as a node for integrating incoming autophagy signals into autophagosome biogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEId7bV69EoLVg90H21EOLACvtfcHk0lgYeCH3VuVmGw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXltVCmu7c%253D&md5=6443c68a54f1e604d097ad6f9ec10998</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M900573200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M900573200%26sid%3Dliteratum%253Aachs%26aulast%3DGanley%26aufirst%3DI.%2BG.%26aulast%3DLam%26aufirst%3DD.%2BH.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DDing%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DS.%26aulast%3DJiang%26aufirst%3DX.%26atitle%3DULK1%25C2%25B7%2520ATG13%2520FIP200%2520complex%2520mediates%2520mTOR%2520signaling%2520and%2520is%2520essential%2520for%2520autophagy%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2009%26volume%3D284%26spage%3D12297%26epage%3D12305%26doi%3D10.1074%2Fjbc.M900573200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Laplante, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sabatini, D. M.</span></span> <span> </span><span class="NLM_article-title">mTOR signaling in growth control and disease</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>149</i></span>,  <span class="NLM_fpage">274</span>– <span class="NLM_lpage">293</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2012.03.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1016%2Fj.cell.2012.03.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=22500797" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BC38Xls1eguro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=149&publication_year=2012&pages=274-293&author=M.+Laplanteauthor=D.+M.+Sabatini&title=mTOR+signaling+in+growth+control+and+disease&doi=10.1016%2Fj.cell.2012.03.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">mTOR signaling in growth control and disease</span></div><div class="casAuthors">Laplante, Mathieu; Sabatini, David M.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">149</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">274-293</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  The mechanistic target of rapamycin (mTOR) signaling pathway senses and integrates a variety of environmental cues to regulate organismal growth and homeostasis.  The pathway regulates many major cellular processes and is implicated in an increasing no. of pathol. conditions, including cancer, obesity, type 2 diabetes, and neurodegeneration.  Here, we review recent advances in our understanding of the mTOR pathway and its role in health, disease, and aging.  We further discuss pharmacol. approaches to treat human pathologies linked to mTOR deregulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqjbKBX-sScfrVg90H21EOLACvtfcHk0lgYeCH3VuVmGw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xls1eguro%253D&md5=d1adac8ec64da63358e0af26a17ceb4e</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2012.03.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2012.03.017%26sid%3Dliteratum%253Aachs%26aulast%3DLaplante%26aufirst%3DM.%26aulast%3DSabatini%26aufirst%3DD.%2BM.%26atitle%3DmTOR%2520signaling%2520in%2520growth%2520control%2520and%2520disease%26jtitle%3DCell%26date%3D2012%26volume%3D149%26spage%3D274%26epage%3D293%26doi%3D10.1016%2Fj.cell.2012.03.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Galluzzi, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pietrocola, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levine, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kroemer, G.</span></span> <span> </span><span class="NLM_article-title">Metabolic control of autophagy</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>159</i></span>,  <span class="NLM_fpage">1263</span>– <span class="NLM_lpage">1276</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2014.11.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1016%2Fj.cell.2014.11.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=25480292" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitFOrt7bN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=159&publication_year=2014&pages=1263-1276&author=L.+Galluzziauthor=F.+Pietrocolaauthor=B.+Levineauthor=G.+Kroemer&title=Metabolic+control+of+autophagy&doi=10.1016%2Fj.cell.2014.11.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Metabolic Control of Autophagy</span></div><div class="casAuthors">Galluzzi, Lorenzo; Pietrocola, Federico; Levine, Beth; Kroemer, Guido</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">159</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1263-1276</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Macroautophagy (herein referred to as autophagy) is an evolutionarily conserved mechanism of adaptation to adverse microenvironmental conditions, including limited nutrient supplies.  Several sensors interacting with the autophagic machinery have evolved to detect fluctuations in key metabolic parameters.  The signal transduction cascades operating downstream of these sensors are highly interconnected to control a spatially and chronol. coordinated autophagic response that maintains the health and function of individual cells while preserving organismal homeostasis.  Here, we discuss the physiol. regulation of autophagy by metabolic circuitries, as well as alterations of such control in disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohrPb03zHBxbVg90H21EOLACvtfcHk0lgYeCH3VuVmGw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitFOrt7bN&md5=f701e43e0183f880fc061ad38b3ef392</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2014.11.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2014.11.006%26sid%3Dliteratum%253Aachs%26aulast%3DGalluzzi%26aufirst%3DL.%26aulast%3DPietrocola%26aufirst%3DF.%26aulast%3DLevine%26aufirst%3DB.%26aulast%3DKroemer%26aufirst%3DG.%26atitle%3DMetabolic%2520control%2520of%2520autophagy%26jtitle%3DCell%26date%3D2014%26volume%3D159%26spage%3D1263%26epage%3D1276%26doi%3D10.1016%2Fj.cell.2014.11.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Papinski, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuschnig, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reiter, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilhelm, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barnes, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maiolica, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansmann, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pfaffenwimmer, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kijanska, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stoffel, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brezovich, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lou, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turk, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aebersold, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ammerer, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peter, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kraft, C.</span></span> <span> </span><span class="NLM_article-title">Early steps in autophagy depend on direct phosphorylation of Atg9 by the Atg1 kinase</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">471</span>– <span class="NLM_lpage">483</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2013.12.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1016%2Fj.molcel.2013.12.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=24440502" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitVyqu7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2014&pages=471-483&author=D.+Papinskiauthor=M.+Schuschnigauthor=W.+Reiterauthor=L.+Wilhelmauthor=C.+A.+Barnesauthor=A.+Maiolicaauthor=I.+Hansmannauthor=T.+Pfaffenwimmerauthor=M.+Kijanskaauthor=I.+Stoffelauthor=S.+S.+Leeauthor=A.+Brezovichauthor=J.+H.+Louauthor=B.+E.+Turkauthor=R.+Aebersoldauthor=G.+Ammererauthor=M.+Peterauthor=C.+Kraft&title=Early+steps+in+autophagy+depend+on+direct+phosphorylation+of+Atg9+by+the+Atg1+kinase&doi=10.1016%2Fj.molcel.2013.12.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Early Steps in Autophagy Depend on Direct Phosphorylation of Atg9 by the Atg1 Kinase</span></div><div class="casAuthors">Papinski, Daniel; Schuschnig, Martina; Reiter, Wolfgang; Wilhelm, Larissa; Barnes, Christopher A.; Maiolica, Alessio; Hansmann, Isabella; Pfaffenwimmer, Thaddaeus; Kijanska, Monika; Stoffel, Ingrid; Lee, Sung Sik; Brezovich, Andrea; Lou, Jane Hua; Turk, Benjamin E.; Aebersold, Ruedi; Ammerer, Gustav; Peter, Matthias; Kraft, Claudine</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">471-483</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Bulk degrdn. of cytoplasmic material is mediated by a highly conserved intracellular trafficking pathway termed autophagy.  This pathway is characterized by the formation of double-membrane vesicles termed autophagosomes engulfing the substrate and transporting it to the vacuole/lysosome for breakdown and recycling.  The Atg1/ULK1 kinase is essential for this process; however, little is known about its targets and the means by which it controls autophagy.  Here we have screened for Atg1 kinase substrates using consensus peptide arrays and identified three components of the autophagy machinery.  The multimembrane-spanning protein Atg9 is a direct target of this kinase essential for autophagy.  Phosphorylated Atg9 is then required for the efficient recruitment of Atg8 and Atg18 to the site of autophagosome formation and subsequent expansion of the isolation membrane, a prerequisite for a functioning autophagy pathway.  These findings show that the Atg1 kinase acts early in autophagy by regulating the outgrowth of autophagosomal membranes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBlZrV_9D_krVg90H21EOLACvtfcHk0lgYeCH3VuVmGw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitVyqu7k%253D&md5=1a99d4b1d79554eb8d7ca1ea735af1be</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2013.12.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2013.12.011%26sid%3Dliteratum%253Aachs%26aulast%3DPapinski%26aufirst%3DD.%26aulast%3DSchuschnig%26aufirst%3DM.%26aulast%3DReiter%26aufirst%3DW.%26aulast%3DWilhelm%26aufirst%3DL.%26aulast%3DBarnes%26aufirst%3DC.%2BA.%26aulast%3DMaiolica%26aufirst%3DA.%26aulast%3DHansmann%26aufirst%3DI.%26aulast%3DPfaffenwimmer%26aufirst%3DT.%26aulast%3DKijanska%26aufirst%3DM.%26aulast%3DStoffel%26aufirst%3DI.%26aulast%3DLee%26aufirst%3DS.%2BS.%26aulast%3DBrezovich%26aufirst%3DA.%26aulast%3DLou%26aufirst%3DJ.%2BH.%26aulast%3DTurk%26aufirst%3DB.%2BE.%26aulast%3DAebersold%26aufirst%3DR.%26aulast%3DAmmerer%26aufirst%3DG.%26aulast%3DPeter%26aufirst%3DM.%26aulast%3DKraft%26aufirst%3DC.%26atitle%3DEarly%2520steps%2520in%2520autophagy%2520depend%2520on%2520direct%2520phosphorylation%2520of%2520Atg9%2520by%2520the%2520Atg1%2520kinase%26jtitle%3DMol.%2520Cell%26date%3D2014%26volume%3D53%26spage%3D471%26epage%3D483%26doi%3D10.1016%2Fj.molcel.2013.12.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Papinski, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kraft, C.</span></span> <span> </span><span class="NLM_article-title">Regulation of autophagy by signaling through the Atg1/ULK1 complex</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>428</i></span>,  <span class="NLM_fpage">1725</span>– <span class="NLM_lpage">1741</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2016.03.030</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1016%2Fj.jmb.2016.03.030" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=27059781" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BC28Xls1Ojtrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=428&publication_year=2016&pages=1725-1741&author=D.+Papinskiauthor=C.+Kraft&title=Regulation+of+autophagy+by+signaling+through+the+Atg1%2FULK1+complex&doi=10.1016%2Fj.jmb.2016.03.030"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of autophagy by signaling through the Atg1/ULK1 complex</span></div><div class="casAuthors">Papinski, Daniel; Kraft, Claudine</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">428</span>
        (<span class="NLM_cas:issue">9_Part_A</span>),
    <span class="NLM_cas:pages">1725-1741</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Autophagy is an intracellular degrdn. pathway highly conserved in eukaryotic species.  It is characterized by selective or bulk trafficking of cytosolic structures-ranging from single proteins to cell organelles, the vacuole or a lysosome, in which the autophagic cargo is degraded.  Autophagy fulfils a large set of roles, including nutrient mobilization in starvation conditions, clearance of protein aggregates, and host defense against intracellular pathogens.  Not surprisingly, autophagy has been linked to several human diseases, among them neurodegenerative disorders and cancer.  Autophagy is coordinated by the action of Atg1/ULK1 kinase, which is the target of several important stress signaling cascades.  Here, the authors discuss the available information on both upstream regulation and downstream effectors of Atg1/ULK1 kinase, with special focus on reported and proposed kinase substrates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo74qOboWXq2rVg90H21EOLACvtfcHk0ljYxPEr7OMeOg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xls1Ojtrs%253D&md5=4583bf3c25d9da1154bfd1a78ce5a465</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2016.03.030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2016.03.030%26sid%3Dliteratum%253Aachs%26aulast%3DPapinski%26aufirst%3DD.%26aulast%3DKraft%26aufirst%3DC.%26atitle%3DRegulation%2520of%2520autophagy%2520by%2520signaling%2520through%2520the%2520Atg1%252FULK1%2520complex%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2016%26volume%3D428%26spage%3D1725%26epage%3D1741%26doi%3D10.1016%2Fj.jmb.2016.03.030" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Egan, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chun, M. G. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vamos, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rong, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lou, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raveendra-Panickar, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheffler, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teriete, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asara, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turk, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cosford, N. D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, R. J.</span></span> <span> </span><span class="NLM_article-title">Small molecule inhibition of the autophagy kinase ULK1 and identification of ULK1 substrates</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">285</span>– <span class="NLM_lpage">297</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2015.05.031</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1016%2Fj.molcel.2015.05.031" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=26118643" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtV2itbvM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2015&pages=285-297&author=D.+F.+Eganauthor=M.+G.+H.+Chunauthor=M.+Vamosauthor=H.+Zouauthor=J.+Rongauthor=C.+J.+Millerauthor=H.+J.+Louauthor=D.+Raveendra-Panickarauthor=C.-C.+Yangauthor=D.+J.+Shefflerauthor=P.+Terieteauthor=J.+M.+Asaraauthor=B.+E.+Turkauthor=N.+D.+P.+Cosfordauthor=R.+J.+Shaw&title=Small+molecule+inhibition+of+the+autophagy+kinase+ULK1+and+identification+of+ULK1+substrates&doi=10.1016%2Fj.molcel.2015.05.031"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Small Molecule Inhibition of the Autophagy Kinase ULK1 and Identification of ULK1 Substrates</span></div><div class="casAuthors">Egan, Daniel F.; Chun, Matthew G. H.; Vamos, Mitchell; Zou, Haixia; Rong, Juan; Miller, Chad J.; Lou, Hua Jane; Raveendra-Panickar, Dhanya; Yang, Chih-Cheng; Sheffler, Douglas J.; Teriete, Peter; Asara, John M.; Turk, Benjamin E.; Cosford, Nicholas D. P.; Shaw, Reuben J.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">285-297</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Many tumors become addicted to autophagy for survival, suggesting inhibition of autophagy as a potential broadly applicable cancer therapy.  ULK1/Atg1 is the only serine/threonine kinase in the core autophagy pathway and thus represents an excellent drug target.  Despite recent advances in the understanding of ULK1 activation by nutrient deprivation, how ULK1 promotes autophagy remains poorly understood.  Here, we screened degenerate peptide libraries to deduce the optimal ULK1 substrate motif and discovered 15 phosphorylation sites in core autophagy proteins that were verified as in vivo ULK1 targets.  We utilized these ULK1 substrates to perform a cell-based screen to identify and characterize a potent ULK1 small mol. inhibitor.  The compd. SBI-0206965 is a highly selective ULK1 kinase inhibitor in vitro and suppressed ULK1-mediated phosphorylation events in cells, regulating autophagy and cell survival.  SBI-0206965 greatly synergized with mechanistic target of rapamycin (mTOR) inhibitors to kill tumor cells, providing a strong rationale for their combined use in the clinic.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdE4ITU-IqIbVg90H21EOLACvtfcHk0ljYxPEr7OMeOg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtV2itbvM&md5=02ede4bf37aa3aca8360d518bec5525c</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2015.05.031&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2015.05.031%26sid%3Dliteratum%253Aachs%26aulast%3DEgan%26aufirst%3DD.%2BF.%26aulast%3DChun%26aufirst%3DM.%2BG.%2BH.%26aulast%3DVamos%26aufirst%3DM.%26aulast%3DZou%26aufirst%3DH.%26aulast%3DRong%26aufirst%3DJ.%26aulast%3DMiller%26aufirst%3DC.%2BJ.%26aulast%3DLou%26aufirst%3DH.%2BJ.%26aulast%3DRaveendra-Panickar%26aufirst%3DD.%26aulast%3DYang%26aufirst%3DC.-C.%26aulast%3DSheffler%26aufirst%3DD.%2BJ.%26aulast%3DTeriete%26aufirst%3DP.%26aulast%3DAsara%26aufirst%3DJ.%2BM.%26aulast%3DTurk%26aufirst%3DB.%2BE.%26aulast%3DCosford%26aufirst%3DN.%2BD.%2BP.%26aulast%3DShaw%26aufirst%3DR.%2BJ.%26atitle%3DSmall%2520molecule%2520inhibition%2520of%2520the%2520autophagy%2520kinase%2520ULK1%2520and%2520identification%2520of%2520ULK1%2520substrates%26jtitle%3DMol.%2520Cell%26date%3D2015%26volume%3D59%26spage%3D285%26epage%3D297%26doi%3D10.1016%2Fj.molcel.2015.05.031" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Alsaadi, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Losier, T. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jackson, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rubinsztein, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Russell, R. C.</span></span> <span> </span><span class="NLM_article-title">ULK1-mediated phosphorylation of ATG16L1 promotes xenophagy, but destabilizes the ATG16L1 Crohn’s mutant</span>. <i>EMBO Rep.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>20</i></span>, <span class="NLM_elocation-id">e46885</span> <span class="refDoi"> DOI: 10.15252/embr.201846885</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.15252%2Fembr.201846885" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=31267703" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2019&author=R.+M.+Alsaadiauthor=T.+T.+Losierauthor=W.+Tianauthor=A.+Jacksonauthor=Z.+Guoauthor=D.+C.+Rubinszteinauthor=R.+C.+Russell&title=ULK1-mediated+phosphorylation+of+ATG16L1+promotes+xenophagy%2C+but+destabilizes+the+ATG16L1+Crohn%E2%80%99s+mutant&doi=10.15252%2Fembr.201846885"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.15252%2Fembr.201846885&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.15252%252Fembr.201846885%26sid%3Dliteratum%253Aachs%26aulast%3DAlsaadi%26aufirst%3DR.%2BM.%26aulast%3DLosier%26aufirst%3DT.%2BT.%26aulast%3DTian%26aufirst%3DW.%26aulast%3DJackson%26aufirst%3DA.%26aulast%3DGuo%26aufirst%3DZ.%26aulast%3DRubinsztein%26aufirst%3DD.%2BC.%26aulast%3DRussell%26aufirst%3DR.%2BC.%26atitle%3DULK1-mediated%2520phosphorylation%2520of%2520ATG16L1%2520promotes%2520xenophagy%252C%2520but%2520destabilizes%2520the%2520ATG16L1%2520Crohn%25E2%2580%2599s%2520mutant%26jtitle%3DEMBO%2520Rep.%26date%3D2019%26volume%3D20%26doi%3D10.15252%2Fembr.201846885" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Saha, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Panigrahi, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patil, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhutia, S. K.</span></span> <span> </span><span class="NLM_article-title">Autophagy in health and disease: A comprehensive review</span>. <i>Biomed. Pharmacother.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>104</i></span>,  <span class="NLM_fpage">485</span>– <span class="NLM_lpage">495</span>, <span class="refDoi"> DOI: 10.1016/j.biopha.2018.05.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1016%2Fj.biopha.2018.05.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=29800913" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BC1cXpvFKgtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=104&publication_year=2018&pages=485-495&author=S.+Sahaauthor=D.+P.+Panigrahiauthor=S.+Patilauthor=S.+K.+Bhutia&title=Autophagy+in+health+and+disease%3A+A+comprehensive+review&doi=10.1016%2Fj.biopha.2018.05.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Autophagy in health and disease: A comprehensive review</span></div><div class="casAuthors">Saha, Sarbari; Panigrahi, Debasna P.; Patil, Shankargouda; Bhutia, Sujit K.</div><div class="citationInfo"><span class="NLM_cas:title">Biomedicine & Pharmacotherapy</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">104</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">485-495</span>CODEN:
                <span class="NLM_cas:coden">BIPHEX</span>;
        ISSN:<span class="NLM_cas:issn">0753-3322</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Autophagy, a conserved catabolic process, plays an immensely significant role in a variety of diseases.  However, whether it imparts a protective function in diseases remains debatable.  During aging, autophagy gradually subsides, manifested by the reduced formation of autophagic vacuoles and improper fusion of these vacuoles with the lysosomes.  Similarly, in neurodegenerative disorders, accumulation of tau and synuclein proteins has been attributed to the decline in the autophagic removal of proteins.  Equivalently, lysosomal disorders show an impairment of the autophagic process leading to the accumulation of lipid mols. within lysosomes.  On the other hand, activation of the autophagic pathway has also proved beneficial in evading various foreign pathogens, thereby contributing to the innate immunity.  In the context of cancer, autophagy has shown to play a puzzling role where it serves as a tumor suppressor during initial stages but later protects the tumor cells from the immune system defense mechanisms.  Similarly, muscular and heart disorders have been shown to be pos. and neg. regulated by autophagy, resp.  In the present review, we, therefore, present a comprehensive review on the role of autophagy in various diseases and their corresponding outcomes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqo7G7CF_NtgbVg90H21EOLACvtfcHk0lhQRkt7fyQPzg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXpvFKgtrY%253D&md5=ade9f524674ce9da482dde10abc3eb89</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.biopha.2018.05.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.biopha.2018.05.007%26sid%3Dliteratum%253Aachs%26aulast%3DSaha%26aufirst%3DS.%26aulast%3DPanigrahi%26aufirst%3DD.%2BP.%26aulast%3DPatil%26aufirst%3DS.%26aulast%3DBhutia%26aufirst%3DS.%2BK.%26atitle%3DAutophagy%2520in%2520health%2520and%2520disease%253A%2520A%2520comprehensive%2520review%26jtitle%3DBiomed.%2520Pharmacother.%26date%3D2018%26volume%3D104%26spage%3D485%26epage%3D495%26doi%3D10.1016%2Fj.biopha.2018.05.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Galluzzi, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bravo-San Pedro, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levine, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kroemer, G.</span></span> <span> </span><span class="NLM_article-title">Pharmacological modulation of autophagy: therapeutic potential and persisting obstacles</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">487</span>, <span class="refDoi"> DOI: 10.1038/nrd.2017.22</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1038%2Fnrd.2017.22" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=28529316" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BC2sXotVKntr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=487&author=L.+Galluzziauthor=J.+M.+Bravo-San+Pedroauthor=B.+Levineauthor=D.+R.+Greenauthor=G.+Kroemer&title=Pharmacological+modulation+of+autophagy%3A+therapeutic+potential+and+persisting+obstacles&doi=10.1038%2Fnrd.2017.22"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacological modulation of autophagy: therapeutic potential and persisting obstacles</span></div><div class="casAuthors">Galluzzi, Lorenzo; Bravo-San Pedro, Jose Manuel; Levine, Beth; Green, Douglas R.; Kroemer, Guido</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">487-511</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Autophagy is central to the maintenance of organismal homeostasis in both physiol. and pathol. situations.  Accordingly, alterations in autophagy have been linked to clin. relevant conditions as diverse as cancer, neurodegeneration and cardiac disorders.  Throughout the past decade, autophagy has attracted considerable attention as a target for the development of novel therapeutics.  However, such efforts have not yet generated clin. viable interventions.  In this review, we discuss the therapeutic potential of autophagy modulators, analyze the obstacles that have limited their development and propose strategies that may unlock the full therapeutic potential of autophagy modulation in the clinic.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr1YAuK4nNrTbVg90H21EOLACvtfcHk0lhQRkt7fyQPzg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXotVKntr4%253D&md5=0cf36178d5fd9b4d2493649d44aa1051</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2017.22&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2017.22%26sid%3Dliteratum%253Aachs%26aulast%3DGalluzzi%26aufirst%3DL.%26aulast%3DBravo-San%2BPedro%26aufirst%3DJ.%2BM.%26aulast%3DLevine%26aufirst%3DB.%26aulast%3DGreen%26aufirst%3DD.%2BR.%26aulast%3DKroemer%26aufirst%3DG.%26atitle%3DPharmacological%2520modulation%2520of%2520autophagy%253A%2520therapeutic%2520potential%2520and%2520persisting%2520obstacles%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2017%26volume%3D16%26spage%3D487%26doi%3D10.1038%2Fnrd.2017.22" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rao, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tortola, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perlot, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wirnsberger, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Novatchkova, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nitsch, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sykacek, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frank, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schramek, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Komnenovic, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sigl, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aumayr, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmauss, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fellner, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Handschuh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glösmann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pasierbek, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlederer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Resch, G. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Popper, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kenner, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kroemer, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Penninger, J. M.</span></span> <span> </span><span class="NLM_article-title">A dual role for autophagy in a murine model of lung cancer</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">3056</span>, <span class="refDoi"> DOI: 10.1038/ncomms4056</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1038%2Fncomms4056" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=24445999" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A280%3ADC%252BC2cznt1aguw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=3056&author=S.+Raoauthor=L.+Tortolaauthor=T.+Perlotauthor=G.+Wirnsbergerauthor=M.+Novatchkovaauthor=R.+Nitschauthor=P.+Sykacekauthor=L.+Frankauthor=D.+Schramekauthor=V.+Komnenovicauthor=V.+Siglauthor=K.+Aumayrauthor=G.+Schmaussauthor=N.+Fellnerauthor=S.+Handschuhauthor=M.+Gl%C3%B6smannauthor=P.+Pasierbekauthor=M.+Schledererauthor=G.+P.+Reschauthor=Y.+Maauthor=H.+Yangauthor=H.+Popperauthor=L.+Kennerauthor=G.+Kroemerauthor=J.+M.+Penninger&title=A+dual+role+for+autophagy+in+a+murine+model+of+lung+cancer&doi=10.1038%2Fncomms4056"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">A dual role for autophagy in a murine model of lung cancer</span></div><div class="casAuthors">Rao Shuan; Tortola Luigi; Perlot Thomas; Wirnsberger Gerald; Novatchkova Maria; Nitsch Roberto; Schramek Daniel; Komnenovic Vukoslav; Sigl Verena; Aumayr Karin; Schmauss Gerald; Pasierbek Pawel; Penninger Josef M; Sykacek Peter; Frank Lukas; Fellner Nicole; Resch Guenter P; Handschuh Stephan; Glosmann Martin; Schlederer Michaela; Kenner Lukas; Ma Yuting; Yang Heng; Popper Helmuth; Kroemer Guido</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">3056</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Autophagy is a mechanism by which starving cells can control their energy requirements and metabolic states, thus facilitating the survival of cells in stressful environments, in particular in the pathogenesis of cancer.  Here we report that tissue-specific inactivation of Atg5, essential for the formation of autophagosomes, markedly impairs the progression of KRas(G12D)-driven lung cancer, resulting in a significant survival advantage of tumour-bearing mice.  Autophagy-defective lung cancers exhibit impaired mitochondrial energy homoeostasis, oxidative stress and a constitutively active DNA damage response.  Genetic deletion of the tumour suppressor p53 reinstates cancer progression of autophagy-deficient tumours.  Although there is improved survival, the onset of Atg5-mutant KRas(G12D)-driven lung tumours is markedly accelerated.  Mechanistically, increased oncogenesis maps to regulatory T cells.  These results demonstrate that, in KRas(G12D)-driven lung cancer, Atg5-regulated autophagy accelerates tumour progression; however, autophagy also represses early oncogenesis, suggesting a link between deregulated autophagy and regulatory T cell controlled anticancer immunity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRd1A4UrEIWovEeosoOnpOLfW6udTcc2eanh2JKprH4wbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2cznt1aguw%253D%253D&md5=e3b8cdd4aee24a82da1ad94b48886c02</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2Fncomms4056&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms4056%26sid%3Dliteratum%253Aachs%26aulast%3DRao%26aufirst%3DS.%26aulast%3DTortola%26aufirst%3DL.%26aulast%3DPerlot%26aufirst%3DT.%26aulast%3DWirnsberger%26aufirst%3DG.%26aulast%3DNovatchkova%26aufirst%3DM.%26aulast%3DNitsch%26aufirst%3DR.%26aulast%3DSykacek%26aufirst%3DP.%26aulast%3DFrank%26aufirst%3DL.%26aulast%3DSchramek%26aufirst%3DD.%26aulast%3DKomnenovic%26aufirst%3DV.%26aulast%3DSigl%26aufirst%3DV.%26aulast%3DAumayr%26aufirst%3DK.%26aulast%3DSchmauss%26aufirst%3DG.%26aulast%3DFellner%26aufirst%3DN.%26aulast%3DHandschuh%26aufirst%3DS.%26aulast%3DGl%25C3%25B6smann%26aufirst%3DM.%26aulast%3DPasierbek%26aufirst%3DP.%26aulast%3DSchlederer%26aufirst%3DM.%26aulast%3DResch%26aufirst%3DG.%2BP.%26aulast%3DMa%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DPopper%26aufirst%3DH.%26aulast%3DKenner%26aufirst%3DL.%26aulast%3DKroemer%26aufirst%3DG.%26aulast%3DPenninger%26aufirst%3DJ.%2BM.%26atitle%3DA%2520dual%2520role%2520for%2520autophagy%2520in%2520a%2520murine%2520model%2520of%2520lung%2520cancer%26jtitle%3DNat.%2520Commun.%26date%3D2014%26volume%3D5%26spage%3D3056%26doi%3D10.1038%2Fncomms4056" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rosenfeldt, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Prey, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morton, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nixon, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacKay, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mrowinska, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Au, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rai, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ridgway, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, K. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gottlieb, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sansom, O. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryan, K. M.</span></span> <span> </span><span class="NLM_article-title">p53 status determines the role of autophagy in pancreatic tumour development</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>504</i></span>,  <span class="NLM_fpage">296</span>, <span class="refDoi"> DOI: 10.1038/nature12865</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1038%2Fnature12865" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=24305049" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFart7rL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=504&publication_year=2013&pages=296&author=M.+T.+Rosenfeldtauthor=J.+O%E2%80%99Preyauthor=J.+P.+Mortonauthor=C.+Nixonauthor=G.+MacKayauthor=A.+Mrowinskaauthor=A.+Auauthor=T.+S.+Raiauthor=L.+Zhengauthor=R.+Ridgwayauthor=P.+D.+Adamsauthor=K.+I.+Andersonauthor=E.+Gottliebauthor=O.+W.+Sansomauthor=K.+M.+Ryan&title=p53+status+determines+the+role+of+autophagy+in+pancreatic+tumour+development&doi=10.1038%2Fnature12865"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">p53 status determines the role of autophagy in pancreatic tumor development</span></div><div class="casAuthors">Rosenfeldt, Mathias T.; O'Prey, Jim; Morton, Jennifer P.; Nixon, Colin; MacKay, Gillian; Mrowinska, Agata; Au, Amy; Rai, Taranjit Singh; Zheng, Liang; Ridgway, Rachel; Adams, Peter D.; Anderson, Kurt I.; Gottlieb, Eyal; Sansom, Owen J.; Ryan, Kevin M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">504</span>
        (<span class="NLM_cas:issue">7479</span>),
    <span class="NLM_cas:pages">296-300</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Macroautophagy (hereafter referred to as autophagy) is a process in which organelles termed autophagosomes deliver cytoplasmic constituents to lysosomes for degrdn.  Autophagy has a major role in cellular homeostasis and has been implicated in various forms of human disease.  The role of autophagy in cancer seems to be complex, with reports indicating both pro-tumorigenic and tumor-suppressive roles.  Here we show, in a humanized genetically-modified mouse model of pancreatic ductal adenocarcinoma (PDAC), that autophagy's role in tumor development is intrinsically connected to the status of the tumor suppressor p53.  Mice with pancreases contg. an activated oncogenic allele of Kras (also called Ki-Ras)-the most common mutational event in PDAC-develop a small no. of pre-cancerous lesions that stochastically develop into PDAC over time.  However, mice also lacking the essential autophagy genes Atg5 or Atg7 accumulate low-grade, pre-malignant pancreatic intraepithelial neoplasia lesions, but progression to high-grade pancreatic intraepithelial neoplasias and PDAC is blocked.  In marked contrast, in mice contg. oncogenic Kras and lacking p53, loss of autophagy no longer blocks tumor progression, but actually accelerates tumor onset, with metabolic anal. revealing enhanced glucose uptake and enrichment of anabolic pathways, which can fuel tumor growth.  These findings provide considerable insight into the role of autophagy in cancer and have important implications for autophagy inhibition in cancer therapy.  In this regard, we also show that treatment of mice with the autophagy inhibitor hydroxychloroquine, which is currently being used in several clin. trials, significantly accelerates tumor formation in mice contg. oncogenic Kras but lacking p53.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9_zf9BrV1ErVg90H21EOLACvtfcHk0ljG1tWRk_tWCw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFart7rL&md5=86c9d7ec9ea35e1f65fa4ad660d377fe</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1038%2Fnature12865&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature12865%26sid%3Dliteratum%253Aachs%26aulast%3DRosenfeldt%26aufirst%3DM.%2BT.%26aulast%3DO%25E2%2580%2599Prey%26aufirst%3DJ.%26aulast%3DMorton%26aufirst%3DJ.%2BP.%26aulast%3DNixon%26aufirst%3DC.%26aulast%3DMacKay%26aufirst%3DG.%26aulast%3DMrowinska%26aufirst%3DA.%26aulast%3DAu%26aufirst%3DA.%26aulast%3DRai%26aufirst%3DT.%2BS.%26aulast%3DZheng%26aufirst%3DL.%26aulast%3DRidgway%26aufirst%3DR.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DAnderson%26aufirst%3DK.%2BI.%26aulast%3DGottlieb%26aufirst%3DE.%26aulast%3DSansom%26aufirst%3DO.%2BW.%26aulast%3DRyan%26aufirst%3DK.%2BM.%26atitle%3Dp53%2520status%2520determines%2520the%2520role%2520of%2520autophagy%2520in%2520pancreatic%2520tumour%2520development%26jtitle%3DNature%26date%3D2013%26volume%3D504%26spage%3D296%26doi%3D10.1038%2Fnature12865" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">White, E.</span></span> <span> </span><span class="NLM_article-title">The role for autophagy in cancer</span>. <i>J. Clin. Invest.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>125</i></span>,  <span class="NLM_fpage">42</span>– <span class="NLM_lpage">46</span>, <span class="refDoi"> DOI: 10.1172/JCI73941</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1172%2FJCI73941" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=25654549" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A280%3ADC%252BC2Mrjt1ensQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=125&publication_year=2015&pages=42-46&author=E.+White&title=The+role+for+autophagy+in+cancer&doi=10.1172%2FJCI73941"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">The role for autophagy in cancer</span></div><div class="casAuthors">White Eileen</div><div class="citationInfo"><span class="NLM_cas:title">The Journal of clinical investigation</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">125</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">42-6</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Autophagy is a survival-promoting pathway that captures, degrades, and recycles intracellular proteins and organelles in lysosomes.  Autophagy preserves organelle function, prevents the toxic buildup of cellular waste products, and provides substrates to sustain metabolism in starvation.  Although in some contexts autophagy suppresses tumorigenesis, in most contexts autophagy facilitates tumorigenesis.  Cancers can upregulate autophagy to survive microenvironmental stress and to increase growth and aggressiveness.  Mechanisms by which autophagy promotes cancer include suppressing induction of the p53 tumor suppressor protein and maintaining metabolic function of mitochondria.  Efforts to inhibit autophagy to improve cancer therapy have thereby attracted great interest.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQsFLRpnBdsgcSzjBFCN0BJfW6udTcc2ean_MfLoSyaxrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2Mrjt1ensQ%253D%253D&md5=9c662eaf1f0bfe280982046d15e316c1</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1172%2FJCI73941&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1172%252FJCI73941%26sid%3Dliteratum%253Aachs%26aulast%3DWhite%26aufirst%3DE.%26atitle%3DThe%2520role%2520for%2520autophagy%2520in%2520cancer%26jtitle%3DJ.%2520Clin.%2520Invest.%26date%3D2015%26volume%3D125%26spage%3D42%26epage%3D46%26doi%3D10.1172%2FJCI73941" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Guo, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xia, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, E.</span></span> <span> </span><span class="NLM_article-title">Autophagy-mediated tumor promotion</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>155</i></span>,  <span class="NLM_fpage">1216</span>– <span class="NLM_lpage">1219</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2013.11.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1016%2Fj.cell.2013.11.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=24315093" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFyksLnO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=155&publication_year=2013&pages=1216-1219&author=J.+Y.+Guoauthor=B.+Xiaauthor=E.+White&title=Autophagy-mediated+tumor+promotion&doi=10.1016%2Fj.cell.2013.11.019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Autophagy-mediated tumor promotion</span></div><div class="casAuthors">Guo, Jessie Yanxiang; Xia, Bing; White, Eileen</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">155</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1216-1219</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Mouse models for cancer are revealing novel cancer-promoting roles for autophagy.  Autophagy promotes tumor growth by suppressing the p53 response, maintaining mitochondrial function, sustaining metabolic homeostasis and survival in stress, and preventing diversion of tumor progression to benign oncocytomas.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphypvWit2Wg7Vg90H21EOLACvtfcHk0ljG1tWRk_tWCw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFyksLnO&md5=05e06234fc4ebb7727e37e21b59dbd21</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2013.11.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2013.11.019%26sid%3Dliteratum%253Aachs%26aulast%3DGuo%26aufirst%3DJ.%2BY.%26aulast%3DXia%26aufirst%3DB.%26aulast%3DWhite%26aufirst%3DE.%26atitle%3DAutophagy-mediated%2520tumor%2520promotion%26jtitle%3DCell%26date%3D2013%26volume%3D155%26spage%3D1216%26epage%3D1219%26doi%3D10.1016%2Fj.cell.2013.11.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">O’Donovan, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Sullivan, G. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKenna, S. L.</span></span> <span> </span><span class="NLM_article-title">Induction of autophagy by drug-resistant esophageal cancer cells promotes their survival and recovery following treatment with chemotherapeutics</span>. <i>Autophagy</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">509</span>– <span class="NLM_lpage">524</span>, <span class="refDoi"> DOI: 10.4161/auto.7.5.15066</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.4161%2Fauto.7.5.15066" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=21325880" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BC38XitF2jtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=509-524&author=T.+R.+O%E2%80%99Donovanauthor=G.+C.+O%E2%80%99Sullivanauthor=S.+L.+McKenna&title=Induction+of+autophagy+by+drug-resistant+esophageal+cancer+cells+promotes+their+survival+and+recovery+following+treatment+with+chemotherapeutics&doi=10.4161%2Fauto.7.5.15066"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Induction of autophagy by drug-resistant esophageal cancer cells promotes their survival and recovery following treatment with chemotherapeutics</span></div><div class="casAuthors">O'Donovan, Tracey R.; O'Sullivan, Gerald C.; McKenna, Sharon L.</div><div class="citationInfo"><span class="NLM_cas:title">Autophagy</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">509-524</span>CODEN:
                <span class="NLM_cas:coden">AUTOC9</span>;
        ISSN:<span class="NLM_cas:issn">1554-8627</span>.
    
            (<span class="NLM_cas:orgname">Landes Bioscience</span>)
        </div><div class="casAbstract">We investigated the cell-death mechanisms induced in esophageal cancer cells in response to the chemotherapeutic drugs, 5-fluorouracil (5-FU) and cisplatin.  Chemosensitive cell lines exhibited apoptosis whereas chemoresistant populations exhibited autophagy and a morphol. resembling type II programmed cell death (PCD).  Cell populations that respond with autophagy are more resistant and will recover following withdrawal of the chemotherapeutic agents.  Specific inhibition of early autophagy induction with siRNA targeted to Beclin 1 and ATG7 significantly enhanced the effect of 5-FU and reduced the recovery of drug-treated cells.  Pharmacol. inhibitors of autophagy were evaluated for their ability to improve chemotherapeutic effect.  The PtdIns 3-kinase inhibitor 3-methyladenine did not enhance the cytotoxicity of 5-FU.  Disruption of lysosomal activity with bafilomycin A1 or chloroquine caused extensive vesicular accumulation but did not improve chemotherapeutic effect.  These observations suggest that an autophagic response to chemotherapy is a survival mechanism that promotes chemoresistance and recovery and that selective inhibition of autophagy regulators has the potential to improve chemotherapeutic regimes.  Currently available indirect inhibitors of autophagy are, however, ineffective at modulating chemosensitivity in these esophageal cancer cell lines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqX4Hfa6svG1LVg90H21EOLACvtfcHk0lhoYUPZrMiRhw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XitF2jtw%253D%253D&md5=bbf8ffa02c8fa302787a7ced6f40175a</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.4161%2Fauto.7.5.15066&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252Fauto.7.5.15066%26sid%3Dliteratum%253Aachs%26aulast%3DO%25E2%2580%2599Donovan%26aufirst%3DT.%2BR.%26aulast%3DO%25E2%2580%2599Sullivan%26aufirst%3DG.%2BC.%26aulast%3DMcKenna%26aufirst%3DS.%2BL.%26atitle%3DInduction%2520of%2520autophagy%2520by%2520drug-resistant%2520esophageal%2520cancer%2520cells%2520promotes%2520their%2520survival%2520and%2520recovery%2520following%2520treatment%2520with%2520chemotherapeutics%26jtitle%3DAutophagy%26date%3D2011%26volume%3D7%26spage%3D509%26epage%3D524%26doi%3D10.4161%2Fauto.7.5.15066" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rehman, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span> <span> </span><span class="NLM_article-title">Autophagy is a therapeutic target in anticancer drug resistance</span>. <i>Biochim. Biophys. Acta, (BBA)-Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>1806</i></span>,  <span class="NLM_fpage">220</span>– <span class="NLM_lpage">229</span>, <span class="refDoi"> DOI: 10.1016/j.bbcan.2010.07.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1016%2Fj.bbcan.2010.07.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=20637264" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVyhurrO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1806&publication_year=2010&pages=220-229&author=S.+Chenauthor=S.+K.+Rehmanauthor=W.+Zhangauthor=A.+Wenauthor=L.+Yaoauthor=J.+Zhang&title=Autophagy+is+a+therapeutic+target+in+anticancer+drug+resistance&doi=10.1016%2Fj.bbcan.2010.07.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Autophagy is a therapeutic target in anticancer drug resistance</span></div><div class="casAuthors">Chen, Suning; Rehman, Sumaiyah K.; Zhang, Wei; Wen, Aidong; Yao, Libo; Zhang, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Reviews on Cancer</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">1806</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">220-229</span>CODEN:
                <span class="NLM_cas:coden">BBACEU</span>;
        ISSN:<span class="NLM_cas:issn">0304-419X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Autophagy is a type of cellular catabolic degrdn. response to nutrient starvation or metabolic stress.  The main function of autophagy is to maintain intracellular metabolic homeostasis through degrdn. of unfolded or aggregated proteins and organelles.  Although autophagic regulation is a complicated process, solid evidence demonstrates that the PI3K-Akt-mTOR, LKB1-AMPK-mTOR and p53 are the main upstream regulators of the autophagic pathway.  Currently, there is a bulk of data indicating the important function of autophagy in cancer.  It is noteworthy that autophagy facilitates the cancer cells' resistance to chemotherapy and radiation treatment.  The abrogation of autophagy potentiates the re-sensitization of therapeutic resistant cancer cells to the anticancer treatment via autophagy inhibitors, such as 3-MA, CQ and BA, or knockdown of the autophagy related mols.  In this review, we summarize the accumulation of evidence for autophagy's involvement in mediating resistance of cancer cells to anticancer therapy and suggest that autophagy might be a potential therapeutic target in anticancer drug resistance in the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojU_TzkfkqsrVg90H21EOLACvtfcHk0lhoYUPZrMiRhw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsVyhurrO&md5=c3783db09a386f8dbe14cc929d0cf7d9</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.bbcan.2010.07.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbcan.2010.07.003%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DS.%26aulast%3DRehman%26aufirst%3DS.%2BK.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DWen%26aufirst%3DA.%26aulast%3DYao%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DJ.%26atitle%3DAutophagy%2520is%2520a%2520therapeutic%2520target%2520in%2520anticancer%2520drug%2520resistance%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520%2528BBA%2529-Rev.%2520Cancer%26date%3D2010%26volume%3D1806%26spage%3D220%26epage%3D229%26doi%3D10.1016%2Fj.bbcan.2010.07.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sui, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kong, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lou, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, H.</span></span> <span> </span><span class="NLM_article-title">Autophagy and chemotherapy resistance: a promising therapeutic target for cancer treatment</span>. <i>Cell death dis.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>4</i></span>, <span class="NLM_elocation-id">e838</span> <span class="refDoi"> DOI: 10.1038/cddis.2013.350</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1038%2Fcddis.2013.350" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=24113172" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1SksbjL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&author=X.+Suiauthor=R.+Chenauthor=Z.+Wangauthor=Z.+Huangauthor=N.+Kongauthor=M.+Zhangauthor=W.+Hanauthor=F.+Louauthor=J.+Yangauthor=Q.+Zhangauthor=X.+Wangauthor=C.+Heauthor=H.+Pan&title=Autophagy+and+chemotherapy+resistance%3A+a+promising+therapeutic+target+for+cancer+treatment&doi=10.1038%2Fcddis.2013.350"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Autophagy and chemotherapy resistance: a promising therapeutic target for cancer treatment</span></div><div class="casAuthors">Sui, X.; Chen, R.; Wang, Z.; Huang, Z.; Kong, N.; Zhang, M.; Han, W.; Lou, F.; Yang, J.; Zhang, Q.; Wang, X.; He, C.; Pan, H.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Death & Disease</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">Oct.</span>),
    <span class="NLM_cas:pages">e838</span>CODEN:
                <span class="NLM_cas:coden">CDDEA4</span>;
        ISSN:<span class="NLM_cas:issn">2041-4889</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Induction of cell death and inhibition of cell survival are the main principles of cancer therapy.  Resistance to chemotherapeutic agents is a major problem in oncol., which limits the effectiveness of anticancer drugs.  A variety of factors contribute to drug resistance, including host factors, specific genetic or epigenetic alterations in the cancer cells and so on.  Although various mechanisms by which cancer cells become resistant to anticancer drugs in the microenvironment have been well elucidated, how to circumvent this resistance to improve anticancer efficacy remains to be defined.  Autophagy, an important homeostatic cellular recycling mechanism, is now emerging as a crucial player in response to metabolic and therapeutic stresses, which attempts to maintain/restore metabolic homeostasis through the catabolic lysis of excessive or unnecessary proteins and injured or aged organelles.  Recently, several studies have shown that autophagy constitutes a potential target for cancer therapy and the induction of autophagy in response to therapeutics can be viewed as having a prodeath or a prosurvival role, which contributes to the anticancer efficacy of these drugs as well as drug resistance.  Thus, understanding the novel function of autophagy may allow us to develop a promising therapeutic strategy to enhance the effects of chemotherapy and improve clin. outcomes in the treatment of cancer patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoe48kz8t23vLVg90H21EOLACvtfcHk0lie9e8JfFMcSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1SksbjL&md5=d92762eb7c4bbc041161c1656e6e17b9</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Fcddis.2013.350&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fcddis.2013.350%26sid%3Dliteratum%253Aachs%26aulast%3DSui%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DR.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DHuang%26aufirst%3DZ.%26aulast%3DKong%26aufirst%3DN.%26aulast%3DZhang%26aufirst%3DM.%26aulast%3DHan%26aufirst%3DW.%26aulast%3DLou%26aufirst%3DF.%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DHe%26aufirst%3DC.%26aulast%3DPan%26aufirst%3DH.%26atitle%3DAutophagy%2520and%2520chemotherapy%2520resistance%253A%2520a%2520promising%2520therapeutic%2520target%2520for%2520cancer%2520treatment%26jtitle%3DCell%2520death%2520dis.%26date%3D2013%26volume%3D4%26doi%3D10.1038%2Fcddis.2013.350" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Holohan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Schaeybroeck, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Longley, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnston, P. G.</span></span> <span> </span><span class="NLM_article-title">Cancer drug resistance: an evolving paradigm</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">714</span>, <span class="refDoi"> DOI: 10.1038/nrc3599</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1038%2Fnrc3599" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=24060863" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVyqurnE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2013&pages=714&author=C.+Holohanauthor=S.+Van+Schaeybroeckauthor=D.+B.+Longleyauthor=P.+G.+Johnston&title=Cancer+drug+resistance%3A+an+evolving+paradigm&doi=10.1038%2Fnrc3599"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Cancer drug resistance: an evolving paradigm</span></div><div class="casAuthors">Holohan, Caitriona; Van Schaeybroeck, Sandra; Longley, Daniel B.; Johnston, Patrick G.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">714-726</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Resistance to chemotherapy and molecularly targeted therapies is a major problem facing current cancer research.  The mechanisms of resistance to 'classical' cytotoxic chemotherapeutics and to therapies that are designed to be selective for specific mol. targets share many features, such as alterations in the drug target, activation of prosurvival pathways and ineffective induction of cell death.  With the increasing arsenal of anticancer agents, improving preclin. models and the advent of powerful high-throughput screening techniques, there are now unprecedented opportunities to understand and overcome drug resistance through the clin. assessment of rational therapeutic drug combinations and the use of predictive biomarkers to enable patient stratification.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqrykBEUSgvv7Vg90H21EOLACvtfcHk0lie9e8JfFMcSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVyqurnE&md5=b497a823bbd968904ce1591b918d0679</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1038%2Fnrc3599&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc3599%26sid%3Dliteratum%253Aachs%26aulast%3DHolohan%26aufirst%3DC.%26aulast%3DVan%2BSchaeybroeck%26aufirst%3DS.%26aulast%3DLongley%26aufirst%3DD.%2BB.%26aulast%3DJohnston%26aufirst%3DP.%2BG.%26atitle%3DCancer%2520drug%2520resistance%253A%2520an%2520evolving%2520paradigm%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2013%26volume%3D13%26spage%3D714%26doi%3D10.1038%2Fnrc3599" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Limpert, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lambert, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bakas, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bata, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brun, S. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cosford, N. D. P.</span></span> <span> </span><span class="NLM_article-title">Autophagy in cancer: regulation by small molecules</span>. <i>Trends Pharmacol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">1021</span>– <span class="NLM_lpage">1032</span>, <span class="refDoi"> DOI: 10.1016/j.tips.2018.10.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1016%2Fj.tips.2018.10.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=30454769" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvFejt7jP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2018&pages=1021-1032&author=A.+S.+Limpertauthor=L.+J.+Lambertauthor=N.+A.+Bakasauthor=N.+Bataauthor=S.+N.+Brunauthor=R.+J.+Shawauthor=N.+D.+P.+Cosford&title=Autophagy+in+cancer%3A+regulation+by+small+molecules&doi=10.1016%2Fj.tips.2018.10.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Autophagy in Cancer: Regulation by Small Molecules</span></div><div class="casAuthors">Limpert, Allison S.; Lambert, Lester J.; Bakas, Nicole A.; Bata, Nicole; Brun, Sonja N.; Shaw, Reuben J.; Cosford, Nicholas D. P.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Pharmacological Sciences</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1021-1032</span>CODEN:
                <span class="NLM_cas:coden">TPHSDY</span>;
        ISSN:<span class="NLM_cas:issn">0165-6147</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  During times of stress, autophagy is a cellular process that enables cells to reclaim damaged components by a controlled recycling pathway.  This mechanism for cellular catabolism is dysregulated in cancer, with evidence indicating that cancer cells rely on autophagy in the hypoxic and nutrient-poor microenvironment of solid tumors.  Mounting evidence suggests that autophagy has a role in the resistance of tumors to std.-of-care (SOC) therapies.  Therefore, there is significant interest in the discovery of small mols. that can safely modulate autophagy.  In this review, we describe recent advances in the identification of new pharmacol. compds. that modulate autophagy, with a focus on their mode of action, value as probe compds., and validation as potential therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgGylHAF7HCLVg90H21EOLACvtfcHk0liMqdn0uTeHyg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvFejt7jP&md5=4f0d411699ded0d57a3211015fccc83d</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fj.tips.2018.10.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tips.2018.10.004%26sid%3Dliteratum%253Aachs%26aulast%3DLimpert%26aufirst%3DA.%2BS.%26aulast%3DLambert%26aufirst%3DL.%2BJ.%26aulast%3DBakas%26aufirst%3DN.%2BA.%26aulast%3DBata%26aufirst%3DN.%26aulast%3DBrun%26aufirst%3DS.%2BN.%26aulast%3DShaw%26aufirst%3DR.%2BJ.%26aulast%3DCosford%26aufirst%3DN.%2BD.%2BP.%26atitle%3DAutophagy%2520in%2520cancer%253A%2520regulation%2520by%2520small%2520molecules%26jtitle%3DTrends%2520Pharmacol.%2520Sci.%26date%3D2018%26volume%3D39%26spage%3D1021%26epage%3D1032%26doi%3D10.1016%2Fj.tips.2018.10.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Maycotte, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gearheart, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barnard, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aryal, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levy, J. M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fosmire, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morgan, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porter, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gustafson, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thorburn, A.</span></span> <span> </span><span class="NLM_article-title">STAT3-mediated autophagy dependence identifies subtypes of breast cancer where autophagy inhibition can be efficacious</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>74</i></span>,  <span class="NLM_fpage">2579</span>– <span class="NLM_lpage">2590</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-13-3470</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1158%2F0008-5472.CAN-13-3470" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=24590058" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BC2cXnt1arsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=74&publication_year=2014&pages=2579-2590&author=P.+Maycotteauthor=C.+M.+Gearheartauthor=R.+Barnardauthor=S.+Aryalauthor=J.+M.+M.+Levyauthor=S.+P.+Fosmireauthor=R.+J.+Hansenauthor=M.+J.+Morganauthor=C.+C.+Porterauthor=D.+L.+Gustafsonauthor=A.+Thorburn&title=STAT3-mediated+autophagy+dependence+identifies+subtypes+of+breast+cancer+where+autophagy+inhibition+can+be+efficacious&doi=10.1158%2F0008-5472.CAN-13-3470"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">STAT3-Mediated Autophagy Dependence Identifies Subtypes of Breast Cancer Where Autophagy Inhibition Can Be Efficacious</span></div><div class="casAuthors">Maycotte, Paola; Gearheart, Christy M.; Barnard, Rebecca; Aryal, Suraj; Mulcahy Levy, Jean M.; Fosmire, Susan P.; Hansen, Ryan J.; Morgan, Michael J.; Porter, Christopher C.; Gustafson, Daniel L.; Thorburn, Andrew</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">74</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2579-2590</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Autophagy is a protein and organelle degrdn. pathway that is involved in diverse diseases, including cancer.  Recent evidence suggests that autophagy is a cell survival mechanism in tumor cells and that its inhibition, esp. in combination with other therapy, could be beneficial but it remains unclear if all cancer cells behave the same way when autophagy is inhibited.  We inhibited autophagy in a panel of breast cancer cell lines and found that some of them are dependent on autophagy for survival even in nutrient rich conditions without any addnl. stress, whereas others need autophagy only when stressed.  Survival under unstressed conditions is due to cell type-specific autophagy regulation of STAT3 activity and this phenotype is enriched in triple-neg. cell lines.  This autophagy-dependency affects response to therapy because autophagy inhibition reduced tumor growth in vivo in autophagy-dependent but not in autophagy-independent breast tumors, whereas combination treatment with autophagy inhibitors and other agent was preferentially synergistic in autophagy-dependent cells.  These results imply that autophagy-dependence represents a tumor cell-specific characteristic where autophagy inhibition will be more effective.  Moreover, our results suggest that autophagy inhibition might be a potential therapeutic strategy for triple-neg. breast cancers, which currently lack an effective targeted treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUScxQ9Siw3rVg90H21EOLACvtfcHk0liMqdn0uTeHyg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXnt1arsrg%253D&md5=21db68cf66514add33588020bece688e</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-13-3470&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-13-3470%26sid%3Dliteratum%253Aachs%26aulast%3DMaycotte%26aufirst%3DP.%26aulast%3DGearheart%26aufirst%3DC.%2BM.%26aulast%3DBarnard%26aufirst%3DR.%26aulast%3DAryal%26aufirst%3DS.%26aulast%3DLevy%26aufirst%3DJ.%2BM.%2BM.%26aulast%3DFosmire%26aufirst%3DS.%2BP.%26aulast%3DHansen%26aufirst%3DR.%2BJ.%26aulast%3DMorgan%26aufirst%3DM.%2BJ.%26aulast%3DPorter%26aufirst%3DC.%2BC.%26aulast%3DGustafson%26aufirst%3DD.%2BL.%26aulast%3DThorburn%26aufirst%3DA.%26atitle%3DSTAT3-mediated%2520autophagy%2520dependence%2520identifies%2520subtypes%2520of%2520breast%2520cancer%2520where%2520autophagy%2520inhibition%2520can%2520be%2520efficacious%26jtitle%3DCancer%2520Res.%26date%3D2014%26volume%3D74%26spage%3D2579%26epage%3D2590%26doi%3D10.1158%2F0008-5472.CAN-13-3470" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosol, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, X.</span></span> <span> </span><span class="NLM_article-title">Triple-negative breast cancer: is there a treatment on the horizon?</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1913</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.12284</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.18632%2Foncotarget.12284" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=27765921" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A280%3ADC%252BC2srks1aksw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=1913&author=H.+Yaoauthor=G.+Heauthor=S.+Yanauthor=C.+Chenauthor=L.+Songauthor=T.+J.+Rosolauthor=X.+Deng&title=Triple-negative+breast+cancer%3A+is+there+a+treatment+on+the+horizon%3F&doi=10.18632%2Foncotarget.12284"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Triple-negative breast cancer: is there a treatment on the horizon?</span></div><div class="casAuthors">Yao Hui; He Guangchun; Yan Shichao; Chen Chao; Deng Xiyun; Song Liujiang; Rosol Thomas J</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1913-1924</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Triple-negative breast cancer (TNBC), which accounts for 15-20% of all breast cancers, does not express estrogen receptor (ER) or progesterone receptor (PR) and lacks human epidermal growth factor receptor 2 (HER2) overexpression or amplification.  These tumors have a more aggressive phenotype and a poorer prognosis due to the high propensity for metastatic progression and absence of specific targeted treatments.  Patients with TNBC do not benefit from hormonal or trastuzumab-based targeted therapies because of the loss of target receptors.  Although these patients respond to chemotherapeutic agents such as taxanes and anthracyclines better than other subtypes of breast cancer, prognosis remains poor.  A group of targeted therapies under investigation showed favorable results in TNBC, especially in cancers with BRCA mutation.  The lipid-lowering statins (3-hydroxy-3-methyl-glutaryl coenzyme A reductase inhibitors), including lovastatin and simvastatin, have been shown to preferentially target TNBC compared with non-TNBC.  These statins hold great promise for the management of TNBC.  Only with the understanding of the molecular basis for the preference of statins for TNBC and more investigations in clinical trials can they be reformulated into a clinically approved drug against TNBC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSj55CS-5D4KA5nADWVYHg1fW6udTcc2easR1ZddJBVeLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2srks1aksw%253D%253D&md5=d78d07246b042a1087a643d9eb6761d4</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.12284&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.12284%26sid%3Dliteratum%253Aachs%26aulast%3DYao%26aufirst%3DH.%26aulast%3DHe%26aufirst%3DG.%26aulast%3DYan%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DC.%26aulast%3DSong%26aufirst%3DL.%26aulast%3DRosol%26aufirst%3DT.%2BJ.%26aulast%3DDeng%26aufirst%3DX.%26atitle%3DTriple-negative%2520breast%2520cancer%253A%2520is%2520there%2520a%2520treatment%2520on%2520the%2520horizon%253F%26jtitle%3DOncotarget%26date%3D2017%26volume%3D8%26spage%3D1913%26doi%3D10.18632%2Foncotarget.12284" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Prat, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perou, C. M.</span></span> <span> </span><span class="NLM_article-title">Deconstructing the molecular portraits of breast cancer</span>. <i>Mol. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">5</span>– <span class="NLM_lpage">23</span>, <span class="refDoi"> DOI: 10.1016/j.molonc.2010.11.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1016%2Fj.molonc.2010.11.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=21147047" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1Cjs7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2011&pages=5-23&author=A.+Pratauthor=C.+M.+Perou&title=Deconstructing+the+molecular+portraits+of+breast+cancer&doi=10.1016%2Fj.molonc.2010.11.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Deconstructing the molecular portraits of breast cancer</span></div><div class="casAuthors">Prat, Aleix; Perou, Charles M.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Oncology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">5-23</span>CODEN:
                <span class="NLM_cas:coden">MOONC3</span>;
        ISSN:<span class="NLM_cas:issn">1574-7891</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Breast cancer is a heterogeneous disease in terms of histol., therapeutic response, dissemination patterns to distant sites, and patient outcomes.  Global gene expression analyses using high-throughput technologies have helped to explain much of this heterogeneity and provided important new classifications of cancer patients.  In the last decade, genomic studies have established five breast cancer intrinsic subtypes (Luminal A, Luminal B, HER2-enriched, Claudin-low, Basal-like) and a Normal Breast-like group.  In this review, we dissect the most recent data on this genomic classification of breast cancer with a special focus on the Claudin-low subtype, which appears enriched for mesenchymal and stem cell features.  In addn., we discuss how the combination of std. clin.-pathol. markers with the information provided by these genomic entities might help further understand the biol. complexity of this disease, increase the efficacy of current and novel therapies, and ultimately improve outcomes for breast cancer patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrIk8X02ivwybVg90H21EOLACvtfcHk0lgw06gjpP8HpQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1Cjs7c%253D&md5=b320e1836b5f0e7e9fbbedf8bacec152</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2Fj.molonc.2010.11.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molonc.2010.11.003%26sid%3Dliteratum%253Aachs%26aulast%3DPrat%26aufirst%3DA.%26aulast%3DPerou%26aufirst%3DC.%2BM.%26atitle%3DDeconstructing%2520the%2520molecular%2520portraits%2520of%2520breast%2520cancer%26jtitle%3DMol.%2520Oncol.%26date%3D2011%26volume%3D5%26spage%3D5%26epage%3D23%26doi%3D10.1016%2Fj.molonc.2010.11.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">O’Reilly, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gubbins, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tully, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guang, M. H. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiner-Gorzel, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCaffrey, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harrison, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furlong, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kell, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCann, A.</span></span> <span> </span><span class="NLM_article-title">The fate of chemoresistance in triple negative breast cancer (TNBC)</span>. <i>BBA clin.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">257</span>– <span class="NLM_lpage">275</span>, <span class="refDoi"> DOI: 10.1016/j.bbacli.2015.03.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1016%2Fj.bbacli.2015.03.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=26676166" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A280%3ADC%252BC28risFagsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2015&pages=257-275&author=E.+A.+O%E2%80%99Reillyauthor=L.+Gubbinsauthor=S.+Sharmaauthor=R.+Tullyauthor=M.+H.+Z.+Guangauthor=K.+Weiner-Gorzelauthor=J.+McCaffreyauthor=M.+Harrisonauthor=F.+Furlongauthor=M.+Kellauthor=A.+McCann&title=The+fate+of+chemoresistance+in+triple+negative+breast+cancer+%28TNBC%29&doi=10.1016%2Fj.bbacli.2015.03.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">The fate of chemoresistance in triple negative breast cancer (TNBC)</span></div><div class="casAuthors">O'Reilly Elma A; Sharma Shiva; Gubbins Luke; Tully Riona; Guang Matthew Ho Zhing; Weiner-Gorzel Karolina; McCann Amanda; McCaffrey John; Harrison Michele; Furlong Fiona; Kell Malcolm</div><div class="citationInfo"><span class="NLM_cas:title">BBA clinical</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">257-75</span>
        ISSN:<span class="NLM_cas:issn">2214-6474</span>.
    </div><div class="casAbstract">BACKGROUND:  Treatment options for women presenting with triple negative breast cancer (TNBC) are limited due to the lack of a therapeutic target and as a result, are managed with standard chemotherapy such as paclitaxel (Taxol®).  Following chemotherapy, the ideal tumour response is apoptotic cell death.  Post-chemotherapy, cells can maintain viability by undergoing viable cellular responses such as cellular senescence, generating secretomes which can directly enhance the malignant phenotype.  SCOPE OF REVIEW:  How tumour cells retain viability in response to chemotherapeutic engagement is discussed.  In addition we discuss the implications of this retained tumour cell viability in the context of the development of recurrent and metastatic TNBC disease.  Current adjuvant and neo-adjuvant treatments available and the novel potential therapies that are being researched are also reviewed.  MAJOR CONCLUSIONS:  Cellular senescence and cytoprotective autophagy are potential mechanisms of chemoresistance in TNBC.  These two non-apoptotic outcomes in response to chemotherapy are inextricably linked and are neglected outcomes of investigation in the chemotherapeutic arena.  Cellular fate assessments may therefore have the potential to predict TNBC patient outcome.  GENERAL SIGNIFICANCE:  Focusing on the fact that cancer cells can bypass the desired cellular apoptotic response to chemotherapy through cellular senescence and cytoprotective autophagy will highlight the importance of targeting non-apoptotic survival pathways to enhance chemotherapeutic efficacy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSDPBSDA-sOPf9pSBfyDvp7fW6udTcc2eZfPKetitgNa7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC28risFagsg%253D%253D&md5=af77680801122c14cdc0a8b714f4796b</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2Fj.bbacli.2015.03.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbacli.2015.03.003%26sid%3Dliteratum%253Aachs%26aulast%3DO%25E2%2580%2599Reilly%26aufirst%3DE.%2BA.%26aulast%3DGubbins%26aufirst%3DL.%26aulast%3DSharma%26aufirst%3DS.%26aulast%3DTully%26aufirst%3DR.%26aulast%3DGuang%26aufirst%3DM.%2BH.%2BZ.%26aulast%3DWeiner-Gorzel%26aufirst%3DK.%26aulast%3DMcCaffrey%26aufirst%3DJ.%26aulast%3DHarrison%26aufirst%3DM.%26aulast%3DFurlong%26aufirst%3DF.%26aulast%3DKell%26aufirst%3DM.%26aulast%3DMcCann%26aufirst%3DA.%26atitle%3DThe%2520fate%2520of%2520chemoresistance%2520in%2520triple%2520negative%2520breast%2520cancer%2520%2528TNBC%2529%26jtitle%3DBBA%2520clin.%26date%3D2015%26volume%3D3%26spage%3D257%26epage%3D275%26doi%3D10.1016%2Fj.bbacli.2015.03.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chittaranjan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bortnik, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dragowska, W. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abeysundara, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leung, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Go, N. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeVorkin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weppler, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gelmon, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yapp, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bally, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gorski, S. M.</span></span> <span> </span><span class="NLM_article-title">Autophagy inhibition augments the anticancer effects of epirubicin treatment in anthracycline-sensitive and-resistant triple-negative breast cancer</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">3159</span>– <span class="NLM_lpage">3173</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.CCR-13-2060</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1158%2F1078-0432.CCR-13-2060" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=24721646" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BC2cXps1Cmsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2014&pages=3159-3173&author=S.+Chittaranjanauthor=S.+Bortnikauthor=W.+H.+Dragowskaauthor=J.+Xuauthor=N.+Abeysundaraauthor=A.+Leungauthor=N.+E.+Goauthor=L.+DeVorkinauthor=S.+A.+Wepplerauthor=K.+Gelmonauthor=D.+T.+Yappauthor=M.+B.+Ballyauthor=S.+M.+Gorski&title=Autophagy+inhibition+augments+the+anticancer+effects+of+epirubicin+treatment+in+anthracycline-sensitive+and-resistant+triple-negative+breast+cancer&doi=10.1158%2F1078-0432.CCR-13-2060"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Autophagy Inhibition Augments the Anticancer Effects of Epirubicin Treatment in Anthracycline-Sensitive and -Resistant Triple-Negative Breast Cancer</span></div><div class="casAuthors">Chittaranjan, Suganthi; Bortnik, Svetlana; Dragowska, Wieslawa H.; Xu, Jing; Abeysundara, Namal; Leung, Amy; Go, Nancy E.; De Vorkin, Lindsay; Weppler, Sherry A.; Gelmon, Karen; Yapp, Donald T.; Bally, Marcel B.; Gorski, Sharon M.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3159-3173</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Purpose: Triple-neg. breast cancers (TNBC) are defined by a lack of expression of estrogen receptor (ER), progesterone receptor (PR), and human epidermal growth factor receptor 2 (ERBB2/HER2).  Although initially responsive to chemotherapy, most recurrent TNBCs develop resistance, resulting in disease progression.  Autophagy is a lysosome-mediated degrdn. and recycling process that can function as an adaptive survival response during chemotherapy and contribute to chemoresistance.  Our goal was to det. whether autophagy inhibition improves treatment efficacy in TNBC cells in tumors either sensitive or refractory to anthracyclines.  Exptl. Design: We used in vitro and in vivo models of TNBC using cell lines sensitive to epirubicin and other anthracyclines, as well as deriv. lines, resistant to the same drugs.  We assessed basal autophagy levels and the effects of chemotherapy on autophagy in parental and resistant cells.  Applying various approaches to inhibit autophagy alone and in combination with chemotherapy, we assessed the effects on cell viability in vitro and tumor growth rates in vivo.  Results: We demonstrated that epirubicin induced autophagic flux in TNBC cells.  Epirubicin-resistant lines exhibited at least 1.5-fold increased basal autophagy levels and, when treated with autophagy inhibitors, showed a significant loss in viability, indicating dependence of resistant cells on autophagy for survival.  Combination of epirubicin with the autophagy inhibitor hydroxychloroquine resulted in a significant redn. in tumor growth compared with monotherapy with epirubicin.  Conclusion: Autophagy inhibition enhances therapeutic response in both anthracycline-sensitive and -resistant TNBC and may be an effective new treatment strategy for this disease.  Clin Cancer Res; 20(12); 3159-73. ©2014 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXG-XFIhGb77Vg90H21EOLACvtfcHk0ljbluxZp49VvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXps1Cmsrc%253D&md5=3469e5a5e9c1f7e9ead384c8959f1602</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-13-2060&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-13-2060%26sid%3Dliteratum%253Aachs%26aulast%3DChittaranjan%26aufirst%3DS.%26aulast%3DBortnik%26aufirst%3DS.%26aulast%3DDragowska%26aufirst%3DW.%2BH.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DAbeysundara%26aufirst%3DN.%26aulast%3DLeung%26aufirst%3DA.%26aulast%3DGo%26aufirst%3DN.%2BE.%26aulast%3DDeVorkin%26aufirst%3DL.%26aulast%3DWeppler%26aufirst%3DS.%2BA.%26aulast%3DGelmon%26aufirst%3DK.%26aulast%3DYapp%26aufirst%3DD.%2BT.%26aulast%3DBally%26aufirst%3DM.%2BB.%26aulast%3DGorski%26aufirst%3DS.%2BM.%26atitle%3DAutophagy%2520inhibition%2520augments%2520the%2520anticancer%2520effects%2520of%2520epirubicin%2520treatment%2520in%2520anthracycline-sensitive%2520and-resistant%2520triple-negative%2520breast%2520cancer%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2014%26volume%3D20%26spage%3D3159%26epage%3D3173%26doi%3D10.1158%2F1078-0432.CCR-13-2060" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vogel, R. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coughlin, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scotti, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iizuka, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anchoori, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roden, R. B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marastoni, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bazzaro, M.</span></span> <span> </span><span class="NLM_article-title">Simultaneous inhibition of deubiquitinating enzymes (DUBs) and autophagy synergistically kills breast cancer cells</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">4159</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.2904</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.18632%2Foncotarget.2904" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=25784654" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A280%3ADC%252BC2Mnls1Sksg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=4159&author=R.+I.+Vogelauthor=K.+Coughlinauthor=A.+Scottiauthor=Y.+Iizukaauthor=R.+Anchooriauthor=R.+B.+S.+Rodenauthor=M.+Marastoniauthor=M.+Bazzaro&title=Simultaneous+inhibition+of+deubiquitinating+enzymes+%28DUBs%29+and+autophagy+synergistically+kills+breast+cancer+cells&doi=10.18632%2Foncotarget.2904"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Simultaneous inhibition of deubiquitinating enzymes (DUBs) and autophagy synergistically kills breast cancer cells</span></div><div class="casAuthors">Vogel Rachel Isaksson; Bazzaro Martina; Coughlin Kathleen; Iizuka Yoshie; Bazzaro Martina; Scotti Alessandra; Marastoni Mauro; Anchoori Ravi; Roden Richard B S; Anchoori Ravi; Roden Richard B S; Roden Richard B S</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">4159-70</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Breast cancer is one of the leading causes of cancer death among women in the United States.  Patients expressing the estrogen and progesterone receptor (ER and PR) and human epidermal growth factor 2 (HER-2) tumor markers have favorable prognosis and efficacious therapeutic options.  In contrast, tumors that are negative for these markers (triple-negative) have a disproportionate share of morbidity and mortality due to lack of a validated molecular target.  Deubiquitinating enzymes (DUBs) are a critical component of ubiquitin-proteasome-system degradation and have been shown to be differentially expressed and activated in a number of cancers, including breast, with their aberrant activity linked to cancer prognosis and clinical outcome.  We evaluated the effect of the DUB inhibitors b-AP15 and RA-9 alone and in combination with early- and late-stage lysosomal inhibitors on cell viability in a panel of triple negative breast cancer (TNBC) cell lines.  Our results indicate small-molecule DUB inhibitors have a profound effect on TNBC viability and lead to activation of autophagy as a cellular mechanism to compensate for ubiquitin-proteasome-system stress.  Treatment with sub-optimal doses of DUB and lysosome inhibitors synergistically kills TNBC cells.  This supports the evaluation of DUB inhibition, in combination with lysosomal inhibition, as a therapeutic approach for the treatment of TNBC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQvDoMDseirLGhHGZei4OW9fW6udTcc2eZz-g_uJDbpH7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2Mnls1Sksg%253D%253D&md5=4cae506139805133d1a3776a41149aa6</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.2904&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.2904%26sid%3Dliteratum%253Aachs%26aulast%3DVogel%26aufirst%3DR.%2BI.%26aulast%3DCoughlin%26aufirst%3DK.%26aulast%3DScotti%26aufirst%3DA.%26aulast%3DIizuka%26aufirst%3DY.%26aulast%3DAnchoori%26aufirst%3DR.%26aulast%3DRoden%26aufirst%3DR.%2BB.%2BS.%26aulast%3DMarastoni%26aufirst%3DM.%26aulast%3DBazzaro%26aufirst%3DM.%26atitle%3DSimultaneous%2520inhibition%2520of%2520deubiquitinating%2520enzymes%2520%2528DUBs%2529%2520and%2520autophagy%2520synergistically%2520kills%2520breast%2520cancer%2520cells%26jtitle%3DOncotarget%26date%3D2015%26volume%3D6%26spage%3D4159%26doi%3D10.18632%2Foncotarget.2904" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Amaravadi, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kimmelman, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Debnath, J.</span></span> <span> </span><span class="NLM_article-title">Targeting Autophagy in Cancer: Recent Advances and Future Directions</span>. <i>Cancer discovery</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1167</span>– <span class="NLM_lpage">1181</span>, <span class="refDoi"> DOI: 10.1158/2159-8290.CD-19-0292</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1158%2F2159-8290.CD-19-0292" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=31434711" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhvFGjs7zN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2019&pages=1167-1181&author=R.+K.+Amaravadiauthor=A.+C.+Kimmelmanauthor=J.+Debnath&title=Targeting+Autophagy+in+Cancer%3A+Recent+Advances+and+Future+Directions&doi=10.1158%2F2159-8290.CD-19-0292"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting autophagy in cancer: recent advances and future directions</span></div><div class="casAuthors">Amaravadi, Ravi K.; Kimmelman, Alec C.; Debnath, Jayanta</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Discovery</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1167-1181</span>CODEN:
                <span class="NLM_cas:coden">CDAIB2</span>;
        ISSN:<span class="NLM_cas:issn">2159-8274</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">A review.  Autophagy, a multistep lysosomal degrdn. pathway that supports nutrient recycling and metabolic adaptation, has been implicated as a process that regulates cancer.  Although autophagy induction may limit the development of tumors, evidence in mouse models demonstrates that autophagy inhibition can limit the growth of established tumors and improve response to cancer therapeutics.  Certain cancer genotypes may be esp. prone to autophagy inhibition.  Different strategies for autophagy modulation may be needed depending on the cancer context.  Here, we review new advances in the mol. control of autophagy, the role of selective autophagy in cancer, and the role of autophagy within the tumor microenvironment and tumor immunity.  We also highlight clin. efforts to repurpose lysosomal inhibitors, such as hydroxychloroquine, as anticancer agents that block autophagy, as well as the development of more potent and specific autophagy inhibitors for cancer treatment, and review future directions for autophagy research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpuQskWwQTM5rVg90H21EOLACvtfcHk0lhsgsWhop4I9g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhvFGjs7zN&md5=cb8453bd893e3dbfbfccf2b2116a9e6b</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1158%2F2159-8290.CD-19-0292&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F2159-8290.CD-19-0292%26sid%3Dliteratum%253Aachs%26aulast%3DAmaravadi%26aufirst%3DR.%2BK.%26aulast%3DKimmelman%26aufirst%3DA.%2BC.%26aulast%3DDebnath%26aufirst%3DJ.%26atitle%3DTargeting%2520Autophagy%2520in%2520Cancer%253A%2520Recent%2520Advances%2520and%2520Future%2520Directions%26jtitle%3DCancer%2520discovery%26date%3D2019%26volume%3D9%26spage%3D1167%26epage%3D1181%26doi%3D10.1158%2F2159-8290.CD-19-0292" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hwang, D. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eom, J.-I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jang, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeung, H.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheong, J.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Min, Y. H.</span></span> <span> </span><span class="NLM_article-title">ULK1 inhibition as a targeted therapeutic strategy for FLT3-ITD-mutated acute myeloid leukemia</span>. <i>J. Exp. Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">14</span>, <span class="refDoi"> DOI: 10.1186/s13046-020-01580-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1186%2Fs13046-020-01580-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=31928527" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2020&pages=1-14&author=D.+Y.+Hwangauthor=J.-I.+Eomauthor=J.+E.+Jangauthor=H.-K.+Jeungauthor=H.+Chungauthor=J.+S.+Kimauthor=J.-W.+Cheongauthor=Y.+H.+Min&title=ULK1+inhibition+as+a+targeted+therapeutic+strategy+for+FLT3-ITD-mutated+acute+myeloid+leukemia&doi=10.1186%2Fs13046-020-01580-4"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1186%2Fs13046-020-01580-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs13046-020-01580-4%26sid%3Dliteratum%253Aachs%26aulast%3DHwang%26aufirst%3DD.%2BY.%26aulast%3DEom%26aufirst%3DJ.-I.%26aulast%3DJang%26aufirst%3DJ.%2BE.%26aulast%3DJeung%26aufirst%3DH.-K.%26aulast%3DChung%26aufirst%3DH.%26aulast%3DKim%26aufirst%3DJ.%2BS.%26aulast%3DCheong%26aufirst%3DJ.-W.%26aulast%3DMin%26aufirst%3DY.%2BH.%26atitle%3DULK1%2520inhibition%2520as%2520a%2520targeted%2520therapeutic%2520strategy%2520for%2520FLT3-ITD-mutated%2520acute%2520myeloid%2520leukemia%26jtitle%3DJ.%2520Exp.%2520Clin.%2520Cancer%2520Res.%26date%3D2020%26volume%3D39%26spage%3D1%26epage%3D14%26doi%3D10.1186%2Fs13046-020-01580-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Qiu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, S.</span></span> <span> </span><span class="NLM_article-title">Targeted inhibition of ULK1 enhances daunorubicin sensitivity in acute myeloid leukemia</span>. <i>Life Sci.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>243</i></span>,  <span class="NLM_fpage">117234</span>, <span class="refDoi"> DOI: 10.1016/j.lfs.2019.117234</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1016%2Fj.lfs.2019.117234" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=31887299" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtFCrt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=243&publication_year=2020&pages=117234&author=L.+Qiuauthor=G.+Zhouauthor=S.+Cao&title=Targeted+inhibition+of+ULK1+enhances+daunorubicin+sensitivity+in+acute+myeloid+leukemia&doi=10.1016%2Fj.lfs.2019.117234"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted inhibition of ULK1 enhances daunorubicin sensitivity in acute myeloid leukemia</span></div><div class="casAuthors">Qiu, Li; Zhou, Gan; Cao, Shan</div><div class="citationInfo"><span class="NLM_cas:title">Life Sciences</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">243</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">117234</span>CODEN:
                <span class="NLM_cas:coden">LIFSAK</span>;
        ISSN:<span class="NLM_cas:issn">0024-3205</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">In acute myeloid leukemia (AML), complete remission can be achieved in parts of patients using cytarabine/anthracycline combination-based chemotherapy, however, drug resistance-related recurrence is still a common cause of treatment failure, leading to high mortality among patients.  In our research, we revealed the mol. mechanisms that were sufficient to improve sensitivity of AML cells to the anthracycline daunorubicin (DNR).  We evaluated the effects of autophagy and apoptosis induced by DNR using two AML cell lines HL60 and U937.  Western blot was preformed to analyze the apoptotic pathway protein expression and flow cytometric anal. was used to detect the level of apoptosis in AML cells.  The levels of autophagy-related proteins were detected by western blotting and autophagic vesicles were obsd. by electron microscopy.  Furthermore, ULK1 inhibition by a highly selective kinase inhibitor SBI-0206965 and shRNA enhanced cytotoxicity of DNR against AML cells.  Independent of mTOR -ULK1 signaling pathway, activation of autophagy of DNR was proved to be mediated by AMPK (pThr172)/ULK1 pathway.  These results revealed that pro-survival autophagy induced by ULK1 activation was one of the potential mechanisms of AML resistance to DNR.  Targeting ULK1 selectively could be a promising therapeutic strategy to enhance sensitivity of DNR for AML therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhTKvNP6OQUbVg90H21EOLACvtfcHk0li0t8ST2K7_vw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtFCrt7w%253D&md5=293767bee9d9835e1d669d922ed1bf37</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2Fj.lfs.2019.117234&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.lfs.2019.117234%26sid%3Dliteratum%253Aachs%26aulast%3DQiu%26aufirst%3DL.%26aulast%3DZhou%26aufirst%3DG.%26aulast%3DCao%26aufirst%3DS.%26atitle%3DTargeted%2520inhibition%2520of%2520ULK1%2520enhances%2520daunorubicin%2520sensitivity%2520in%2520acute%2520myeloid%2520leukemia%26jtitle%3DLife%2520Sci.%26date%3D2020%26volume%3D243%26spage%3D117234%26doi%3D10.1016%2Fj.lfs.2019.117234" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xue, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gong, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, C.</span></span> <span> </span><span class="NLM_article-title">SBI0206965, a novel inhibitor of Ulk1, suppresses non-small cell lung cancer cell growth by modulating both autophagy and apoptosis pathways</span>. <i>Oncol. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>37</i></span>,  <span class="NLM_fpage">3449</span>– <span class="NLM_lpage">3458</span>, <span class="refDoi"> DOI: 10.3892/or.2017.5635</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.3892%2For.2017.5635" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=28498429" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsVOiu7zJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2017&pages=3449-3458&author=F.+Tangauthor=P.+Huauthor=Z.+Yangauthor=C.+Xueauthor=J.+Gongauthor=S.+Sunauthor=L.+Shiauthor=S.+Zhangauthor=Z.+Liauthor=C.+Yangauthor=C.+Zhangauthor=C.+Xie&title=SBI0206965%2C+a+novel+inhibitor+of+Ulk1%2C+suppresses+non-small+cell+lung+cancer+cell+growth+by+modulating+both+autophagy+and+apoptosis+pathways&doi=10.3892%2For.2017.5635"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">SBI0206965, a novel inhibitor of Ulk1, suppresses non-small cell lung cancer cell growth by modulating both autophagy and apoptosis pathways</span></div><div class="casAuthors">Tang, Fang; Hu, Pengchao; Yang, Zetian; Xue, Chao; Gong, Jun; Sun, Shaoxing; Shi, Liu; Zhang, Shimin; Li, Zhenzhen; Yang, Chunxu; Zhang, Junhong; Xie, Conghua</div><div class="citationInfo"><span class="NLM_cas:title">Oncology Reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">3449-3458</span>CODEN:
                <span class="NLM_cas:coden">OCRPEW</span>;
        ISSN:<span class="NLM_cas:issn">1791-2431</span>.
    
            (<span class="NLM_cas:orgname">Spandidos Publications Ltd.</span>)
        </div><div class="casAbstract">Lung cancer is a major public health problem worldwide.  Non-small cell lung cancer (NSCLC) accounts for 85% of lung cancer cases.  Autophagy has recently sparked great interest, and it is thought to participate in a variety of diseases, including lung cancer.  Uncoordinated (Unc) 51-like kinase 1 (Ulk1), a serine/threonine kinase, plays a central role in the autophagy pathway.  However, the role of Ulk1 in NSCLC remains unclear.  We report that NSCLC cell lines exhibited high expression of Ulk1 and that Ulk1 was neg. correlated with prognosis in lung cancer patients.  Knockdown of Ulk1 or the inhibition of Ulk1 by the selective inhibitor SBI0206965, inhibited cell proliferation, induced cell apoptosis and enhanced the sensitivity of cisplatin against NSCLC cells.  Moreover, we demonstrated that Ulk1 exerted oncogenic activity in NSCLC by modulating both autophagy and apoptosis pathways.  Inhibition of autophagy by SBI0206965 sensitized NSCLC cells to cisplatin by inhibiting cisplatin induced cell-protective autophagy to promote apoptosis.  Furthermore, SBI0206965 promoted apoptosis in NSCLC cells independent of autophagy, which was partly mediated by destabilization of Bcl2/Bclxl.  In summary, our results show that inhibition of Ulk1 suppresses NSCLC cell growth and sensitizes NSCLC cells to cisplatin by modulating both autophagy and apoptosis pathways, and that Ulk1 might be a promising target for NSCLC treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGSAExMnHCRLVg90H21EOLACvtfcHk0li0t8ST2K7_vw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsVOiu7zJ&md5=00bfa972d3f82ae5439ccc97cd8221d4</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.3892%2For.2017.5635&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3892%252For.2017.5635%26sid%3Dliteratum%253Aachs%26aulast%3DTang%26aufirst%3DF.%26aulast%3DHu%26aufirst%3DP.%26aulast%3DYang%26aufirst%3DZ.%26aulast%3DXue%26aufirst%3DC.%26aulast%3DGong%26aufirst%3DJ.%26aulast%3DSun%26aufirst%3DS.%26aulast%3DShi%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DYang%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DXie%26aufirst%3DC.%26atitle%3DSBI0206965%252C%2520a%2520novel%2520inhibitor%2520of%2520Ulk1%252C%2520suppresses%2520non-small%2520cell%2520lung%2520cancer%2520cell%2520growth%2520by%2520modulating%2520both%2520autophagy%2520and%2520apoptosis%2520pathways%26jtitle%3DOncol.%2520Rep.%26date%3D2017%26volume%3D37%26spage%3D3449%26epage%3D3458%26doi%3D10.3892%2For.2017.5635" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chaikuad, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koschade, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stolz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zivkovic, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pohl, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaid, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lambert, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cosford, N. D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brandts, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">Conservation of structure, function and inhibitor binding in UNC-51-like kinase 1 and 2 (ULK1/2)</span>. <i>Biochem. J.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>476</i></span>,  <span class="NLM_fpage">875</span>– <span class="NLM_lpage">887</span>, <span class="refDoi"> DOI: 10.1042/BCJ20190038</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1042%2FBCJ20190038" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=30782972" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BC1MXpt1aitbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=476&publication_year=2019&pages=875-887&author=A.+Chaikuadauthor=S.+E.+Koschadeauthor=A.+Stolzauthor=K.+Zivkovicauthor=C.+Pohlauthor=S.+Shaidauthor=H.+Renauthor=L.+J.+Lambertauthor=N.+D.+P.+Cosfordauthor=C.+H.+Brandtsauthor=S.+Knapp&title=Conservation+of+structure%2C+function+and+inhibitor+binding+in+UNC-51-like+kinase+1+and+2+%28ULK1%2F2%29&doi=10.1042%2FBCJ20190038"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Conservation of structure, function and inhibitor binding in UNC-51-like kinase 1 and 2 (ULK1/2)</span></div><div class="casAuthors">Chaikuad, Apirat; Koschade, Sebastian E.; Stolz, Alexandra; Zivkovic, Katarina; Pohl, Christian; Shaid, Shabnam; Ren, Huiyu; Lambert, Lester J.; Cosford, Nicholas D. P.; Brandts, Christian H.; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Journal</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">476</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">875-887</span>CODEN:
                <span class="NLM_cas:coden">BIJOAK</span>;
        ISSN:<span class="NLM_cas:issn">0264-6021</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">Autophagy is essential for cellular homeostasis and when deregulated this survival mechanism has been assocd. with disease development.  Inhibition of autophagy initiation by inhibiting the kinase ULK1 (Unc-51-like autophagy activating kinase 1) has been proposed as a potential cancer therapy.  While inhibitors and crystal structures of ULK1 have been reported, little is known about the other closely related kinase ULK2 (Unc-51-like autophagy activating kinase 2).  Here, we present the crystal structure of ULK2 in complex with ATP competitive inhibitors.  Surprisingly, the ULK2 structure revealed a dimeric assembly reminiscent of dimeric arrangements of auto-activating kinases suggesting a role for this assocn. in ULK activation.  Screening of a kinase focused library of pre-clin. and clin. compds. revealed several potent ULK1/2 inhibitors and good correlation of inhibitor-binding behavior with both ULK kinases.  Aurora A was identified as a major off-target of currently used ULK1 inhibitors.  Autophagic flux assays demonstrated that this off-target activity by strongly inducing autophagy in different cellular systems conferred an addnl. layer of complexity in the interpretation of cellular data.  The data presented here provide structural models and chem. starting points for the development of ULK1/2 dual inhibitors with improved selectivity for future exploitation of autophagy inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq-GpxTacuIQLVg90H21EOLACvtfcHk0lgJpqR0oEJuJQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXpt1aitbo%253D&md5=4c2fbe0d3a3180ffb58b28fcb377a5a2</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1042%2FBCJ20190038&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBCJ20190038%26sid%3Dliteratum%253Aachs%26aulast%3DChaikuad%26aufirst%3DA.%26aulast%3DKoschade%26aufirst%3DS.%2BE.%26aulast%3DStolz%26aufirst%3DA.%26aulast%3DZivkovic%26aufirst%3DK.%26aulast%3DPohl%26aufirst%3DC.%26aulast%3DShaid%26aufirst%3DS.%26aulast%3DRen%26aufirst%3DH.%26aulast%3DLambert%26aufirst%3DL.%2BJ.%26aulast%3DCosford%26aufirst%3DN.%2BD.%2BP.%26aulast%3DBrandts%26aufirst%3DC.%2BH.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DConservation%2520of%2520structure%252C%2520function%2520and%2520inhibitor%2520binding%2520in%2520UNC-51-like%2520kinase%25201%2520and%25202%2520%2528ULK1%252F2%2529%26jtitle%3DBiochem.%2520J.%26date%3D2019%26volume%3D476%26spage%3D875%26epage%3D887%26doi%3D10.1042%2FBCJ20190038" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roskoski, R.,  Jr.</span></span> <span> </span><span class="NLM_article-title">Classification of small molecule protein kinase inhibitors based upon the structures of their drug-enzyme complexes</span>. <i>Pharmacol. Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>103</i></span>,  <span class="NLM_fpage">26</span>– <span class="NLM_lpage">48</span>, <span class="refDoi"> DOI: 10.1016/j.phrs.2015.10.021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1016%2Fj.phrs.2015.10.021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=26529477" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslKksL%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=2016&pages=26-48&author=R.+Roskoski&title=Classification+of+small+molecule+protein+kinase+inhibitors+based+upon+the+structures+of+their+drug-enzyme+complexes&doi=10.1016%2Fj.phrs.2015.10.021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Classification of small molecule protein kinase inhibitors based upon the structures of their drug-enzyme complexes</span></div><div class="casAuthors">Roskoski, Robert</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacological Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">103</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">26-48</span>CODEN:
                <span class="NLM_cas:coden">PHMREP</span>;
        ISSN:<span class="NLM_cas:issn">1043-6618</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Because dysregulation and mutations of protein kinases play causal roles in human disease, this family of enzymes has become one of the most important drug targets over the past two decades.  The X-ray crystal structures of 21 of the 27 FDA-approved small mol. inhibitors bound to their target protein kinases are depicted in this paper.  The structure of the enzyme-bound antagonist complex is used in the classification of these inhibitors.  Type I inhibitors bind to the active protein kinase conformation (DFG-Asp in, αC-helix in).  Type I1/2 inhibitors bind to a DFG-Asp in inactive conformation while Type II inhibitors bind to a DFG-Asp out inactive conformation.  Type I, I1/2, and type II inhibitors occupy part of the adenine binding pocket and form hydrogen bonds with the hinge region connecting the small and large lobes of the enzyme.  Type III inhibitors bind next to the ATP-binding pocket and type IV inhibitors do not bind to the ATP or peptide substrate binding sites.  Type III and IV inhibitors are allosteric in nature.  Type V inhibitors bind to two different regions of the protein kinase domain and are therefore bivalent inhibitors.  The type I-V inhibitors are reversible.  In contrast, type VI inhibitors bind covalently to their target enzyme.  Type I, I1/2, and II inhibitors are divided into A and B subtypes.  The type A inhibitors bind in the front cleft, the back cleft, and near the gatekeeper residue, all of which occur within the region sepg. the small and large lobes of the protein kinase.  The type B inhibitors bind in the front cleft and gate area but do not extend into the back cleft.  An anal. of the limited available data indicates that type A inhibitors have a long residence time (minutes to hours) while the type B inhibitors have a short residence time (seconds to minutes).  The catalytic spine includes residues from the small and large lobes and interacts with the adenine ring of ATP.  Nearly all of the approved protein kinase inhibitors occupy the adenine-binding pocket; thus it is not surprising that these inhibitors interact with nearby catalytic spine (CS) residues.  Moreover, a significant no. of approved drugs also interact with regulatory spine (RS) residues.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpH0COwqC0CabVg90H21EOLACvtfcHk0lgJpqR0oEJuJQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslKksL%252FN&md5=e6b45c8644e322f30f898e763780c64b</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2Fj.phrs.2015.10.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.phrs.2015.10.021%26sid%3Dliteratum%253Aachs%26aulast%3DRoskoski%26aufirst%3DR.%26atitle%3DClassification%2520of%2520small%2520molecule%2520protein%2520kinase%2520inhibitors%2520based%2520upon%2520the%2520structures%2520of%2520their%2520drug-enzyme%2520complexes%26jtitle%3DPharmacol.%2520Res.%26date%3D2016%26volume%3D103%26spage%3D26%26epage%3D48%26doi%3D10.1016%2Fj.phrs.2015.10.021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">de
Oliveira, P. S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferraz, F. A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pena, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pramio, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morais, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schechtman, D.</span></span> <span> </span><span class="NLM_article-title">Revisiting protein kinase–substrate interactions: Toward therapeutic development</span>. <i>Sci. Signaling</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">re3</span>– <span class="NLM_lpage">re3</span>, <span class="refDoi"> DOI: 10.1126/scisignal.aad4016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1126%2Fscisignal.aad4016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=27016527" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2016&pages=re3-re3&author=P.+S.+L.+de%0AOliveiraauthor=F.+A.+N.+Ferrazauthor=D.+A.+Penaauthor=D.+T.+Pramioauthor=F.+A.+Moraisauthor=D.+Schechtman&title=Revisiting+protein+kinase%E2%80%93substrate+interactions%3A+Toward+therapeutic+development&doi=10.1126%2Fscisignal.aad4016"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1126%2Fscisignal.aad4016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscisignal.aad4016%26sid%3Dliteratum%253Aachs%26aulast%3Dde%2BOliveira%26aufirst%3DP.%2BS.%2BL.%26aulast%3DFerraz%26aufirst%3DF.%2BA.%2BN.%26aulast%3DPena%26aufirst%3DD.%2BA.%26aulast%3DPramio%26aufirst%3DD.%2BT.%26aulast%3DMorais%26aufirst%3DF.%2BA.%26aulast%3DSchechtman%26aufirst%3DD.%26atitle%3DRevisiting%2520protein%2520kinase%25E2%2580%2593substrate%2520interactions%253A%2520Toward%2520therapeutic%2520development%26jtitle%3DSci.%2520Signaling%26date%3D2016%26volume%3D9%26spage%3Dre3%26epage%3Dre3%26doi%3D10.1126%2Fscisignal.aad4016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dar, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shokat, K. M.</span></span> <span> </span><span class="NLM_article-title">The evolution of protein kinase inhibitors from antagonists to agonists of cellular signaling</span>. <i>Annu. Rev. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>80</i></span>,  <span class="NLM_fpage">769</span>– <span class="NLM_lpage">795</span>, <span class="refDoi"> DOI: 10.1146/annurev-biochem-090308-173656</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1146%2Fannurev-biochem-090308-173656" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=21548788" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BC3MXptVCnsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=80&publication_year=2011&pages=769-795&author=A.+C.+Darauthor=K.+M.+Shokat&title=The+evolution+of+protein+kinase+inhibitors+from+antagonists+to+agonists+of+cellular+signaling&doi=10.1146%2Fannurev-biochem-090308-173656"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">The evolution of protein kinase inhibitors from antagonists to agonists of cellular signaling</span></div><div class="casAuthors">Dar, Arvin C.; Shokat, Kevan M.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Biochemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">80</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">769-795</span>CODEN:
                <span class="NLM_cas:coden">ARBOAW</span>;
        ISSN:<span class="NLM_cas:issn">0066-4154</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review.  Kinases are highly regulated enzymes with diverse mechanisms controlling their catalytic output.  Over time, chem. discovery efforts for kinases have produced ATP-competitive compds., allosteric regulators, irreversible binders, and highly specific inhibitors.  These distinct classes of small mols. have revealed many novel aspects about kinase-mediated signaling, and some have progressed from simple tool compds. into clin. validated therapeutics.  This review explores several small-mol. inhibitors for kinases high-lighting elaborate mechanisms by which kinase function is modulated.  A complete surprise of targeted kinase drug discovery has been the finding of ATP-competitive inhibitors that behave as agonists, rather than antagonists, of their direct kinase target.  These studies hint at a connection between ATP-binding site occupancy and networks of communication that are independent of kinase catalysis.  Indeed, kinase inhibitors that induce changes in protein localization, protein-protein interactions, and even enhancement of catalytic activity of the target kinase have been found.  The relevance of these findings to the therapeutic efficacy of kinase inhibitors and to the future identification of new classes of drug targets is discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVWDRwiciMXrVg90H21EOLACvtfcHk0lg-D0hgRUpTYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXptVCnsbY%253D&md5=1bb989c45f767784e4ee777359ae4870</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1146%2Fannurev-biochem-090308-173656&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-biochem-090308-173656%26sid%3Dliteratum%253Aachs%26aulast%3DDar%26aufirst%3DA.%2BC.%26aulast%3DShokat%26aufirst%3DK.%2BM.%26atitle%3DThe%2520evolution%2520of%2520protein%2520kinase%2520inhibitors%2520from%2520antagonists%2520to%2520agonists%2520of%2520cellular%2520signaling%26jtitle%3DAnnu.%2520Rev.%2520Biochem.%26date%3D2011%26volume%3D80%26spage%3D769%26epage%3D795%26doi%3D10.1146%2Fannurev-biochem-090308-173656" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lazarus, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Novotny, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shokat, K. M.</span></span> <span> </span><span class="NLM_article-title">Structure of the human autophagy initiating kinase ULK1 in complex with potent inhibitors</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">257</span>– <span class="NLM_lpage">261</span>, <span class="refDoi"> DOI: 10.1021/cb500835z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb500835z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFGqsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=257-261&author=M.+B.+Lazarusauthor=C.+J.+Novotnyauthor=K.+M.+Shokat&title=Structure+of+the+human+autophagy+initiating+kinase+ULK1+in+complex+with+potent+inhibitors&doi=10.1021%2Fcb500835z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of the Human Autophagy Initiating Kinase ULK1 in Complex with Potent Inhibitors</span></div><div class="casAuthors">Lazarus, Michael B.; Novotny, Chris J.; Shokat, Kevan M.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">257-261</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Autophagy is a conserved cellular process that involves the degrdn. of cellular components for energy maintenance and cytoplasmic quality control that has recently gained interest as a novel target for a variety of human diseases, including cancer.  A prime candidate to det. the potential therapeutic benefit of targeting autophagy is the kinase ULK1, whose activation initiates autophagy.  Here, we report the first structures of ULK1, in complex with multiple potent inhibitors.  These structures show features unique to the enzyme and will provide a path for the rational design of selective compds. as cellular probes and potential therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGryGDGKTxwZ3bVg90H21EOLACvtfcHk0lihMEKJxEkMyQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFGqsQ%253D%253D&md5=3439432fc00cc6954ab664027bedf9e7</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fcb500835z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb500835z%26sid%3Dliteratum%253Aachs%26aulast%3DLazarus%26aufirst%3DM.%2BB.%26aulast%3DNovotny%26aufirst%3DC.%2BJ.%26aulast%3DShokat%26aufirst%3DK.%2BM.%26atitle%3DStructure%2520of%2520the%2520human%2520autophagy%2520initiating%2520kinase%2520ULK1%2520in%2520complex%2520with%2520potent%2520inhibitors%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2015%26volume%3D10%26spage%3D257%26epage%3D261%26doi%3D10.1021%2Fcb500835z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abagyan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Totrov, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuznetsov, D.</span></span> <span> </span><span class="NLM_article-title">ICM—a new method for protein modeling and design: applications to docking and structure prediction from the distorted native conformation</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">488</span>– <span class="NLM_lpage">506</span>, <span class="refDoi"> DOI: 10.1002/jcc.540150503</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1002%2Fjcc.540150503" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADyaK2MXhtVWisA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=1994&pages=488-506&author=R.+Abagyanauthor=M.+Totrovauthor=D.+Kuznetsov&title=ICM%E2%80%94a+new+method+for+protein+modeling+and+design%3A+applications+to+docking+and+structure+prediction+from+the+distorted+native+conformation&doi=10.1002%2Fjcc.540150503"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">ICM - a new method for protein modeling and design: applications to docking and structure prediction from the distorted native conformation</span></div><div class="casAuthors">Abagyan, Ruben; Totrov, Maxim; Kuznetsov, Dmitry</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">488-506</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    </div><div class="casAbstract">An efficient methodol., further referred to as internal coordinate modeling (ICM), for versatile modeling operations and global energy optimization on arbitrary fixed multimol. systems is described.  It is aimed at protein structure prediction, homol. modeling, mol. docking, NMR structure detn., and protein design.  The method uses and further develops a previously introduced approach to model biomol. structures in which bond lengths, bond angles, and torsion angles are considered as independent variables, any subset of them being fixed.  Here the authors simplify and generalize the basic description of the system, introduce the variable dihedral phase angle, and allow arbitrary connections of the mols. and conventional definition of the torsion angles.  Algorithms for calcn. of energy derivs. with respect to internal variables in the topol. tree of the system and for rapid evaluation of accessible surface are presented.  Multidimensional variables restraints are proposed to represent the statistical information about the torsion angle distributions in proteins.  To incorporate complex energy terms as solvation energy and electrostatics into a structure prediction procedure, a "double-energy" Monte Carlo minimization procedure in which these terms are omitted during the minimization state of the random step and included for the comparison with the previous conformation in a Markov chain is proposed and justified.  The ICM method is applied successfully to a mol. docking problem.  The procedure finds the correct parallel arrangement of 2 rigid helixes from a leucine zipper domain as the lowest-energy conformation (0.5 Å root mean square, RMS, deviation from the native structure) starting from completely random configuration.  Structures with antiparallel helixes or helixes staggered by one helix turn had energies higher by about 7 or 9 kcal/mol, resp.  Soft docking was also attempted.  A docking procedure allowing side-chain flexibility also converged to the parallel configuration starting from the helixes optimized individually.  To justify an internal coordinate approach to the structure prediction as opposed to a Cartesian one, energy hypersurfaces around the native structure of the squash seeds trypsin inhibitor were studied.  Torsion angle minimization from the optimal conformation randomly distorted up to the RMS deviation of 2.2 Å or angular RMS deviation of 10° restored the native conformation in most cases.  In contrast, Cartesian coordinate minimization did not reach the min. from deviations as small as 0.3 Å or 2°.  The authors conclude that the most promising detailed approach to the protein-folding problem would consist of some coarse global sampling strategy combined with the local energy minimization in the torsion coordinate space.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6kscl2--38LVg90H21EOLACvtfcHk0lihMEKJxEkMyQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXhtVWisA%253D%253D&md5=5f1a1836429860ded9012dab112c9c47</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1002%2Fjcc.540150503&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.540150503%26sid%3Dliteratum%253Aachs%26aulast%3DAbagyan%26aufirst%3DR.%26aulast%3DTotrov%26aufirst%3DM.%26aulast%3DKuznetsov%26aufirst%3DD.%26atitle%3DICM%25E2%2580%2594a%2520new%2520method%2520for%2520protein%2520modeling%2520and%2520design%253A%2520applications%2520to%2520docking%2520and%2520structure%2520prediction%2520from%2520the%2520distorted%2520native%2520conformation%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D1994%26volume%3D15%26spage%3D488%26epage%3D506%26doi%3D10.1002%2Fjcc.540150503" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zegzouti, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zdanovskaia, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsiao, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goueli, S. A.</span></span> <span> </span><span class="NLM_article-title">ADP-Glo: A Bioluminescent and homogeneous ADP monitoring assay for kinases</span>. <i>Assay Drug Dev. Technol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">560</span>– <span class="NLM_lpage">572</span>, <span class="refDoi"> DOI: 10.1089/adt.2009.0222</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1089%2Fadt.2009.0222" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=20105026" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsFejtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2009&pages=560-572&author=H.+Zegzoutiauthor=M.+Zdanovskaiaauthor=K.+Hsiaoauthor=S.+A.+Goueli&title=ADP-Glo%3A+A+Bioluminescent+and+homogeneous+ADP+monitoring+assay+for+kinases&doi=10.1089%2Fadt.2009.0222"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">ADP-Glo: a bioluminescent and homogeneous ADP monitoring assay for kinases</span></div><div class="casAuthors">Zegzouti, Hicham; Zdanovskaia, Marina; Hsiao, Kevin; Goueli, Said A.</div><div class="citationInfo"><span class="NLM_cas:title">Assay and Drug Development Technologies</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">560-572</span>CODEN:
                <span class="NLM_cas:coden">ADDTAR</span>;
        ISSN:<span class="NLM_cas:issn">1540-658X</span>.
    
            (<span class="NLM_cas:orgname">Mary Ann Liebert, Inc.</span>)
        </div><div class="casAbstract">ADP-Glo is a novel bioluminescent, homogeneous assay for monitoring ADP producing biochem. reactions and thus it is an ideal assay for detecting enzyme activity using a wide variety of substrates.  It is a universal assay that can be used with protein kinases, lipid kinases, sugar kinases, and many more kinases as well as ATPases.  Because of its high sensitivity, it is suitable for monitoring enzyme activities at very early substrate conversions requiring very low amt. of enzymes.  Furthermore, as the assay is applicable to a broad range of ATP and substrate concns., it is optimal for enzymes that require high ATP and substrate concns.  This is crit. since inhibitor potency has to be demonstrated at the cellular level where ATP is present at millimolar concns.  ADP-Glo is performed in 2 steps upon completion of kinase reaction: a combined termination of kinase reaction and depletion of remaining ATP in the first step, and conversion of generated ADP to ATP and the newly produced ATP to light output using luciferase/luciferin reaction in the second step.  The luminescent signal generated is proportional to the ADP concn. produced and is correlated with the kinase activity.  Due to its high signal to background and luminescent readout, this assay is less susceptible to generation of false hits and thus it is applicable to not only primary and secondary screening but also kinase profiling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGocYSq13vSVnrVg90H21EOLACvtfcHk0lj_1rP4mWU82w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsFejtL8%253D&md5=b2bba87e66707b7e7015220c1c3a414a</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1089%2Fadt.2009.0222&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1089%252Fadt.2009.0222%26sid%3Dliteratum%253Aachs%26aulast%3DZegzouti%26aufirst%3DH.%26aulast%3DZdanovskaia%26aufirst%3DM.%26aulast%3DHsiao%26aufirst%3DK.%26aulast%3DGoueli%26aufirst%3DS.%2BA.%26atitle%3DADP-Glo%253A%2520A%2520Bioluminescent%2520and%2520homogeneous%2520ADP%2520monitoring%2520assay%2520for%2520kinases%26jtitle%3DAssay%2520Drug%2520Dev.%2520Technol.%26date%3D2009%26volume%3D7%26spage%3D560%26epage%3D572%26doi%3D10.1089%2Fadt.2009.0222" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Robers, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dart, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woodroofe, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimprich, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirkland, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Machleidt, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kupcho, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartnett, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimmerman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimmerman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niles, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohana, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daniels, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slater, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cong, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, Y.-Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, K. V.</span></span> <span> </span><span class="NLM_article-title">Target engagement and drug residence time can be observed in living cells with BRET</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">10</span>, <span class="refDoi"> DOI: 10.1038/ncomms10091</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1038%2Fncomms10091" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=1-10&author=M.+B.+Robersauthor=M.+L.+Dartauthor=C.+C.+Woodroofeauthor=C.+A.+Zimprichauthor=T.+A.+Kirklandauthor=T.+Machleidtauthor=K.+R.+Kupchoauthor=S.+Levinauthor=J.+R.+Hartnettauthor=K.+Zimmermanauthor=K.+Zimmermanauthor=A.+L.+Nilesauthor=R.+F.+Ohanaauthor=D.+L.+Danielsauthor=M.+Slaterauthor=M.+G.+Woodauthor=M.+Congauthor=Y.-Q.+Chengauthor=K.+V.+Wood&title=Target+engagement+and+drug+residence+time+can+be+observed+in+living+cells+with+BRET&doi=10.1038%2Fncomms10091"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1038%2Fncomms10091&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms10091%26sid%3Dliteratum%253Aachs%26aulast%3DRobers%26aufirst%3DM.%2BB.%26aulast%3DDart%26aufirst%3DM.%2BL.%26aulast%3DWoodroofe%26aufirst%3DC.%2BC.%26aulast%3DZimprich%26aufirst%3DC.%2BA.%26aulast%3DKirkland%26aufirst%3DT.%2BA.%26aulast%3DMachleidt%26aufirst%3DT.%26aulast%3DKupcho%26aufirst%3DK.%2BR.%26aulast%3DLevin%26aufirst%3DS.%26aulast%3DHartnett%26aufirst%3DJ.%2BR.%26aulast%3DZimmerman%26aufirst%3DK.%26aulast%3DZimmerman%26aufirst%3DK.%26aulast%3DNiles%26aufirst%3DA.%2BL.%26aulast%3DOhana%26aufirst%3DR.%2BF.%26aulast%3DDaniels%26aufirst%3DD.%2BL.%26aulast%3DSlater%26aufirst%3DM.%26aulast%3DWood%26aufirst%3DM.%2BG.%26aulast%3DCong%26aufirst%3DM.%26aulast%3DCheng%26aufirst%3DY.-Q.%26aulast%3DWood%26aufirst%3DK.%2BV.%26atitle%3DTarget%2520engagement%2520and%2520drug%2520residence%2520time%2520can%2520be%2520observed%2520in%2520living%2520cells%2520with%2520BRET%26jtitle%3DNat.%2520Commun.%26date%3D2015%26volume%3D6%26spage%3D1%26epage%3D10%26doi%3D10.1038%2Fncomms10091" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vasta, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corona, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilkinson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimprich, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartnett, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ingold, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimmerman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Machleidt, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirkland, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huwiler, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohana, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slater, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Otto, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cong, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, C. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, B.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanke, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, C. G. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drewry, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, K. V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willson, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meisenheimer, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, K. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robers, M. B.</span></span> <span> </span><span class="NLM_article-title">Quantitative, wide-spectrum kinase profiling in live cells for assessing the effect of cellular ATP on target engagement</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">206</span>– <span class="NLM_lpage">214.e11</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2017.10.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1016%2Fj.chembiol.2017.10.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=29174542" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvVOitbfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=206-214.e11&author=J.+D.+Vastaauthor=C.+R.+Coronaauthor=J.+Wilkinsonauthor=C.+A.+Zimprichauthor=J.+R.+Hartnettauthor=M.+R.+Ingoldauthor=K.+Zimmermanauthor=T.+Machleidtauthor=T.+A.+Kirklandauthor=K.+G.+Huwilerauthor=R.+F.+Ohanaauthor=M.+Slaterauthor=P.+Ottoauthor=M.+Congauthor=C.+I.+Wellsauthor=B.-T.+Bergerauthor=T.+Hankeauthor=C.+G.+K.+Dingauthor=D.+H.+Drewryauthor=K.+V.+M.+Huberauthor=T.+M.+Willsonauthor=S.+Knappauthor=S.+M%C3%BCllerauthor=P.+L.+Meisenheimerauthor=F.+Fanauthor=K.+V.+Woodauthor=M.+B.+Robers&title=Quantitative%2C+wide-spectrum+kinase+profiling+in+live+cells+for+assessing+the+effect+of+cellular+ATP+on+target+engagement&doi=10.1016%2Fj.chembiol.2017.10.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Quantitative, Wide-Spectrum Kinase Profiling in Live Cells for Assessing the Effect of Cellular ATP on Target Engagement</span></div><div class="casAuthors">Vasta, James D.; Corona, Cesear R.; Wilkinson, Jennifer; Zimprich, Chad A.; Hartnett, James R.; Ingold, Morgan R.; Zimmerman, Kristopher; Machleidt, Thomas; Kirkland, Thomas A.; Huwiler, Kristin G.; Ohana, Rachel Friedman; Slater, Michael; Otto, Paul; Cong, Mei; Wells, Carrow I.; Berger, Benedict-Tilman; Hanke, Thomas; Glas, Carina; Ding, Ke; Drewry, David H.; Huber, Kilian V. M.; Willson, Timothy M.; Knapp, Stefan; Muller, Susanne; Meisenheimer, Poncho L.; Fan, Frank; Wood, Keith V.; Robers, Matthew B.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">206-214.e11</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">For kinase inhibitors, intracellular target selectivity is fundamental to pharmacol. mechanism.  Although a no. of acellular techniques have been developed to measure kinase binding or enzymic inhibition, such approaches can fail to accurately predict engagement in cells.  Here we report the application of an energy transfer technique that enabled the first broad-spectrum, equil.-based approach to quant. profile target occupancy and compd. affinity in live cells.  Using this method, we performed a selectivity profiling for clin. relevant kinase inhibitors against 178 full-length kinases, and a mechanistic interrogation of the potency offsets obsd. between cellular and biochem. anal.  For the multikinase inhibitor crizotinib, our approach accurately predicted cellular potency and revealed improved target selectivity compared with biochem. measurements.  Due to cellular ATP, a no. of putative crizotinib targets are unexpectedly disengaged in live cells at a clin. relevant drug dose.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoaPVDw5pU6zrVg90H21EOLACvtfcHk0ljJP9BJeugDhw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvVOitbfN&md5=bc3a1f618ab0d2ad6355f31af66d7ef4</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2017.10.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2017.10.010%26sid%3Dliteratum%253Aachs%26aulast%3DVasta%26aufirst%3DJ.%2BD.%26aulast%3DCorona%26aufirst%3DC.%2BR.%26aulast%3DWilkinson%26aufirst%3DJ.%26aulast%3DZimprich%26aufirst%3DC.%2BA.%26aulast%3DHartnett%26aufirst%3DJ.%2BR.%26aulast%3DIngold%26aufirst%3DM.%2BR.%26aulast%3DZimmerman%26aufirst%3DK.%26aulast%3DMachleidt%26aufirst%3DT.%26aulast%3DKirkland%26aufirst%3DT.%2BA.%26aulast%3DHuwiler%26aufirst%3DK.%2BG.%26aulast%3DOhana%26aufirst%3DR.%2BF.%26aulast%3DSlater%26aufirst%3DM.%26aulast%3DOtto%26aufirst%3DP.%26aulast%3DCong%26aufirst%3DM.%26aulast%3DWells%26aufirst%3DC.%2BI.%26aulast%3DBerger%26aufirst%3DB.-T.%26aulast%3DHanke%26aufirst%3DT.%26aulast%3DDing%26aufirst%3DC.%2BG.%2BK.%26aulast%3DDrewry%26aufirst%3DD.%2BH.%26aulast%3DHuber%26aufirst%3DK.%2BV.%2BM.%26aulast%3DWillson%26aufirst%3DT.%2BM.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DMeisenheimer%26aufirst%3DP.%2BL.%26aulast%3DFan%26aufirst%3DF.%26aulast%3DWood%26aufirst%3DK.%2BV.%26aulast%3DRobers%26aufirst%3DM.%2BB.%26atitle%3DQuantitative%252C%2520wide-spectrum%2520kinase%2520profiling%2520in%2520live%2520cells%2520for%2520assessing%2520the%2520effect%2520of%2520cellular%2520ATP%2520on%2520target%2520engagement%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2018%26volume%3D25%26spage%3D206%26epage%3D214.e11%26doi%3D10.1016%2Fj.chembiol.2017.10.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Robers, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friedman-Ohana, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, K. V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kilpatrick, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vasta, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, B.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaudhry, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, K. V.</span></span> <span> </span><span class="NLM_article-title">Quantifying target occupancy of small molecules within living cells</span>. <i>Annu. Rev. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>89</i></span>,  <span class="NLM_fpage">557</span>– <span class="NLM_lpage">581</span>, <span class="refDoi"> DOI: 10.1146/annurev-biochem-011420-092302</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1146%2Fannurev-biochem-011420-092302" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=32208767" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BB3cXlsFCgsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=89&publication_year=2020&pages=557-581&author=M.+B.+Robersauthor=R.+Friedman-Ohanaauthor=K.+V.+M.+Huberauthor=L.+Kilpatrickauthor=J.+D.+Vastaauthor=B.-T.+Bergerauthor=C.+Chaudhryauthor=S.+Hillauthor=S.+M%C3%BCllerauthor=S.+Knappauthor=K.+V.+Wood&title=Quantifying+target+occupancy+of+small+molecules+within+living+cells&doi=10.1146%2Fannurev-biochem-011420-092302"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Quantifying Target Occupancy of Small Molecules Within Living Cells</span></div><div class="casAuthors">Robers, M. B.; Friedman-Ohana, R.; Huber, K. V. M.; Kilpatrick, L.; Vasta, J. D.; Berger, B.-T.; Chaudhry, C.; Hill, S.; Muller, S.; Knapp, S.; Wood, K. V.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Biochemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">89</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">557-581</span>CODEN:
                <span class="NLM_cas:coden">ARBOAW</span>;
        ISSN:<span class="NLM_cas:issn">0066-4154</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews</span>)
        </div><div class="casAbstract">The binding affinity and kinetics of target engagement are fundamental to establishing structure-activity relationships (SARs) for prospective therapeutic agents.  Enhancing these binding parameters for operative targets, while minimizing binding to off-target sites, can translate to improved drug efficacy and a widened therapeutic window.  Compd. activity is typically assessed through modulation of an obsd. phenotype in cultured cells.  Quantifying the corresponding binding properties under common cellular conditions can provide more meaningful interpretation of the cellular SAR anal.  Consequently, methods for assessing drug binding in living cells have advanced and are now integral to medicinal chem. workflows.  In this review, we survey key technol. advancements that support quant. assessments of target occupancy in cultured cells, emphasizing generalizable methodologies able to deliver anal. precision that heretofore required reductionist biochem. approaches.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq1fWsTEZUETbVg90H21EOLACvtfcHk0ljJP9BJeugDhw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXlsFCgsrg%253D&md5=38876614c21a3137d1003fcf6744895c</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1146%2Fannurev-biochem-011420-092302&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-biochem-011420-092302%26sid%3Dliteratum%253Aachs%26aulast%3DRobers%26aufirst%3DM.%2BB.%26aulast%3DFriedman-Ohana%26aufirst%3DR.%26aulast%3DHuber%26aufirst%3DK.%2BV.%2BM.%26aulast%3DKilpatrick%26aufirst%3DL.%26aulast%3DVasta%26aufirst%3DJ.%2BD.%26aulast%3DBerger%26aufirst%3DB.-T.%26aulast%3DChaudhry%26aufirst%3DC.%26aulast%3DHill%26aufirst%3DS.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DWood%26aufirst%3DK.%2BV.%26atitle%3DQuantifying%2520target%2520occupancy%2520of%2520small%2520molecules%2520within%2520living%2520cells%26jtitle%3DAnnu.%2520Rev.%2520Biochem.%26date%3D2020%26volume%3D89%26spage%3D557%26epage%3D581%26doi%3D10.1146%2Fannurev-biochem-011420-092302" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Niesen, F. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berglund, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span> <span> </span><span class="NLM_article-title">The use of differential scanning fluorimetry to detect ligand interactions that promote protein stability</span>. <i>Nat. Protoc.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">2212</span>, <span class="refDoi"> DOI: 10.1038/nprot.2007.321</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1038%2Fnprot.2007.321" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=17853878" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFagsb7N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2007&pages=2212&author=F.+H.+Niesenauthor=H.+Berglundauthor=M.+Vedadi&title=The+use+of+differential+scanning+fluorimetry+to+detect+ligand+interactions+that+promote+protein+stability&doi=10.1038%2Fnprot.2007.321"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">The use of differential scanning fluorimetry to detect ligand interactions that promote protein stability</span></div><div class="casAuthors">Niesen, Frank H.; Berglund, Helena; Vedadi, Masoud</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2212-2221</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Differential scanning fluorometry (DSF) is a rapid and inexpensive screening method to identify low-mol.-wt. ligands that bind and stabilize purified proteins.  The temp. at which a protein unfolds is measured by an increase in the fluorescence of a dye with affinity for hydrophobic parts of the protein, which are exposed as the protein unfolds.  A simple fitting procedure allows quick calcn. of the transition midpoint; the difference in the temp. of this midpoint in the presence and absence of ligand is related to the binding affinity of the small mol., which can be a low-mol.-wt. compd., a peptide or a nucleic acid.  DSF is best performed using a conventional real-time PCR instrument.  Ligand solns. from a storage plate are added to a soln. of protein and dye, distributed into the wells of the PCR plate and fluorescence intensity measured as the temp. is raised gradually.  Results can be obtained in a single day.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomViqp6Tq7krVg90H21EOLACvtfcHk0lhBIW0aZnDXlA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFagsb7N&md5=8a74dbb348ab81179614ab4ae5797d94</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2007.321&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2007.321%26sid%3Dliteratum%253Aachs%26aulast%3DNiesen%26aufirst%3DF.%2BH.%26aulast%3DBerglund%26aufirst%3DH.%26aulast%3DVedadi%26aufirst%3DM.%26atitle%3DThe%2520use%2520of%2520differential%2520scanning%2520fluorimetry%2520to%2520detect%2520ligand%2520interactions%2520that%2520promote%2520protein%2520stability%26jtitle%3DNat.%2520Protoc.%26date%3D2007%26volume%3D2%26spage%3D2212%26doi%3D10.1038%2Fnprot.2007.321" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Russell, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, H. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, Y.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neufeld, T. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dillin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guan, K.-L.</span></span> <span> </span><span class="NLM_article-title">ULK1 induces autophagy by phosphorylating Beclin-1 and activating VPS34 lipid kinase</span>. <i>Nat. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">741</span>, <span class="refDoi"> DOI: 10.1038/ncb2757</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1038%2Fncb2757" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=23685627" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BC3sXnvVSitb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2013&pages=741&author=R.+C.+Russellauthor=Y.+Tianauthor=H.+Yuanauthor=H.+W.+Parkauthor=Y.-Y.+Changauthor=J.+Kimauthor=H.+Kimauthor=T.+P.+Neufeldauthor=A.+Dillinauthor=K.-L.+Guan&title=ULK1+induces+autophagy+by+phosphorylating+Beclin-1+and+activating+VPS34+lipid+kinase&doi=10.1038%2Fncb2757"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">ULK1 induces autophagy by phosphorylating Beclin-1 and activating VPS34 lipid kinase</span></div><div class="casAuthors">Russell, Ryan C.; Tian, Ye; Yuan, Haixin; Park, Hyun Woo; Chang, Yu-Yun; Kim, Joungmok; Kim, Haerin; Neufeld, Thomas P.; Dillin, Andrew; Guan, Kun-Liang</div><div class="citationInfo"><span class="NLM_cas:title">Nature Cell Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">741-750</span>CODEN:
                <span class="NLM_cas:coden">NCBIFN</span>;
        ISSN:<span class="NLM_cas:issn">1465-7392</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Autophagy is the primary cellular catabolic program activated in response to nutrient starvation.  Initiation of autophagy, particularly by amino acid withdrawal, requires the ULK kinases.  Despite its pivotal role in autophagy initiation, little is known about the mechanisms by which ULK promotes autophagy.  Here we describe a mol. mechanism linking ULK to the pro-autophagic lipid kinase VPS34.  Following amino acid starvation or mTOR inhibition, the activated ULK1 phosphorylates Beclin-1 on Ser14, thereby enhancing the activity of the ATG14L-contg. VPS34 complexes.  The Beclin-1 Ser14 phosphorylation by ULK is required for full autophagic induction in mammals and this requirement is conserved in Caenorhabditis elegans.  Our study reveals a mol. link from ULK1 to activation of the autophagy-specific VPS34 complex and autophagy induction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqf7cYRCSx3VbVg90H21EOLACvtfcHk0lhBIW0aZnDXlA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXnvVSitb0%253D&md5=380158d2f07518eae293a932cb90a4d1</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1038%2Fncb2757&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncb2757%26sid%3Dliteratum%253Aachs%26aulast%3DRussell%26aufirst%3DR.%2BC.%26aulast%3DTian%26aufirst%3DY.%26aulast%3DYuan%26aufirst%3DH.%26aulast%3DPark%26aufirst%3DH.%2BW.%26aulast%3DChang%26aufirst%3DY.-Y.%26aulast%3DKim%26aufirst%3DJ.%26aulast%3DKim%26aufirst%3DH.%26aulast%3DNeufeld%26aufirst%3DT.%2BP.%26aulast%3DDillin%26aufirst%3DA.%26aulast%3DGuan%26aufirst%3DK.-L.%26atitle%3DULK1%2520induces%2520autophagy%2520by%2520phosphorylating%2520Beclin-1%2520and%2520activating%2520VPS34%2520lipid%2520kinase%26jtitle%3DNat.%2520Cell%2520Biol.%26date%3D2013%26volume%3D15%26spage%3D741%26doi%3D10.1038%2Fncb2757" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lefort, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joffre, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kieffer, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Givel, A.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bourachot, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zago, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bieche, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dubois, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meseure, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vincent-Salomon, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Camonis, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mechta-Grigoriou, F.</span></span> <span> </span><span class="NLM_article-title">Inhibition of autophagy as a new means of improving chemotherapy efficiency in high-LC3B triple-negative breast cancers</span>. <i>Autophagy</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">2122</span>– <span class="NLM_lpage">2142</span>, <span class="refDoi"> DOI: 10.4161/15548627.2014.981788</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.4161%2F15548627.2014.981788" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=25427136" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmtlSns7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=2122-2142&author=S.+Lefortauthor=C.+Joffreauthor=Y.+Kiefferauthor=A.-M.+Givelauthor=B.+Bourachotauthor=G.+Zagoauthor=I.+Biecheauthor=T.+Duboisauthor=D.+Meseureauthor=A.+Vincent-Salomonauthor=J.+Camonisauthor=F.+Mechta-Grigoriou&title=Inhibition+of+autophagy+as+a+new+means+of+improving+chemotherapy+efficiency+in+high-LC3B+triple-negative+breast+cancers&doi=10.4161%2F15548627.2014.981788"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of autophagy as a new means of improving chemotherapy efficiency in high-LC3B triple-negative breast cancers</span></div><div class="casAuthors">Lefort, Sylvain; Joffre, Carine; Kieffer, Yann; Givel, Anne-Marie; Bourachot, Brigitte; Zago, Giulia; Bieche, Ivan; Dubois, Thierry; Meseure, Didier; Vincent-Salomon, Anne; Camonis, Jacques; Mechta-Grigoriou, Fatima</div><div class="citationInfo"><span class="NLM_cas:title">Autophagy</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2122-2142</span>CODEN:
                <span class="NLM_cas:coden">AUTOC9</span>;
        ISSN:<span class="NLM_cas:issn">1554-8627</span>.
    
            (<span class="NLM_cas:orgname">Landes Bioscience</span>)
        </div><div class="casAbstract">The triple-neg. breast cancer (TN BC) subtype is the most aggressive form of invasive BC.  Despite intensive efforts to improve BC treatments, patients with TN BC continue to exhibit poor survival, with half developing resistance to chemotherapy.  Here we identify autophagy as a key mechanism in the progression and chemoresistance of a subset of TN tumors.  We demonstrate that LC3B, a protein involved in autophagosome formation, is a reliable marker of poor prognosis in TN BC, validating this prognostic value at both the mRNA and protein levels in several independent cohorts.  We also show that LC3B has no prognostic value for other BC subtypes (Luminal or HER2 BC), thus revealing a specific impact of autophagy on TN tumors.  Autophagy is essential for the proliferative and invasive properties in 3D of TN BC cells characterized by high LC3B levels.  Interestingly, the activity of the transcriptional co-activator YAP1 (Yes-assocd. protein 1) is regulated by the autophagy process and we identify YAP1 as a new actor in the autophagy-dependent profilerative and invasive properties of high-LC3B TN BC.  Finally, inhibiting autophagy by silencing ATG5 or ATG7 significantly impaired high-LC3B TN tumor growth in vivo.  Moreover, using a patient-derived TN tumor transplanted into mice, we show that an autophagy inhibitor, chloroquine, potentiates the effects of chemotherapeutic agents.  Overall, our data identify LC3B as a new prognostic marker for TN BC and the inhibition of autophagy as a promising therapeutic strategy for TN BC patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoQ2g6Fni8bOrVg90H21EOLACvtfcHk0lhJASM7msEq5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXmtlSns7o%253D&md5=b482b5542af1ae309084b248aa63bb4b</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.4161%2F15548627.2014.981788&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252F15548627.2014.981788%26sid%3Dliteratum%253Aachs%26aulast%3DLefort%26aufirst%3DS.%26aulast%3DJoffre%26aufirst%3DC.%26aulast%3DKieffer%26aufirst%3DY.%26aulast%3DGivel%26aufirst%3DA.-M.%26aulast%3DBourachot%26aufirst%3DB.%26aulast%3DZago%26aufirst%3DG.%26aulast%3DBieche%26aufirst%3DI.%26aulast%3DDubois%26aufirst%3DT.%26aulast%3DMeseure%26aufirst%3DD.%26aulast%3DVincent-Salomon%26aufirst%3DA.%26aulast%3DCamonis%26aufirst%3DJ.%26aulast%3DMechta-Grigoriou%26aufirst%3DF.%26atitle%3DInhibition%2520of%2520autophagy%2520as%2520a%2520new%2520means%2520of%2520improving%2520chemotherapy%2520efficiency%2520in%2520high-LC3B%2520triple-negative%2520breast%2520cancers%26jtitle%3DAutophagy%26date%3D2014%26volume%3D10%26spage%3D2122%26epage%3D2142%26doi%3D10.4161%2F15548627.2014.981788" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liang, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ensor, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaipparettu, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bass, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, J. C.</span></span> <span> </span><span class="NLM_article-title">The autophagy inhibitor chloroquine targets cancer stem cells in triple negative breast cancer by inducing mitochondrial damage and impairing DNA break repair</span>. <i>Cancer Lett.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>376</i></span>,  <span class="NLM_fpage">249</span>– <span class="NLM_lpage">258</span>, <span class="refDoi"> DOI: 10.1016/j.canlet.2016.04.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1016%2Fj.canlet.2016.04.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=27060208" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BC28XlsF2msL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=376&publication_year=2016&pages=249-258&author=D.+H.+Liangauthor=D.+S.+Choiauthor=J.+E.+Ensorauthor=B.+A.+Kaipparettuauthor=B.+L.+Bassauthor=J.+C.+Chang&title=The+autophagy+inhibitor+chloroquine+targets+cancer+stem+cells+in+triple+negative+breast+cancer+by+inducing+mitochondrial+damage+and+impairing+DNA+break+repair&doi=10.1016%2Fj.canlet.2016.04.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">The autophagy inhibitor chloroquine targets cancer stem cells in triple negative breast cancer by inducing mitochondrial damage and impairing DNA break repair</span></div><div class="casAuthors">Liang, Diana H.; Choi, Dong Soon; Ensor, Joe E.; Kaipparettu, Benny A.; Bass, Barbara L.; Chang, Jenny C.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Letters (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">376</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">249-258</span>CODEN:
                <span class="NLM_cas:coden">CALEDQ</span>;
        ISSN:<span class="NLM_cas:issn">0304-3835</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Triple neg. breast cancer (TNBC), characterized by an abundance of treatment-resistant breast cancer stem cells (CSCs), has a poorer prognosis than other types of breast cancers.  Despite its aggressiveness, no effective targeted therapy exists for TNBC.  Here, we demonstrate that CQ effectively targets CSCs via autophagy inhibition, mitochondrial structural damage, and impairment of double-stranded DNA break repair.  Electron microscopy demonstrates CQ-induced mitochondrial cristae damage, which leads to mitochondrial membrane depolarization with a significant redn. in the activity of cytochrome c oxidase and accumulation of superoxide and double-stranded DNA breaks.  CQ effectively diminishes the TNBC cells' ability to metastasize in vitro and in a TNBC xenograft model.  When administered in combination with carboplatin, CQ effectively inhibits carboplatin-induced autophagy.  This combination treatment significantly diminishes the expression of DNA repair proteins in CSC subpopulations, resulting in tumor growth redn. in carboplatin-resistant BRCA1 wild-type TNBC orthotopic xenografts.  As TNBC's high treatment failure rate has been attributed to enrichment of CSCs, CQ, an autophagy inhibitor with anti-CSC effects, may be an effective adjunct to current TNBC chemotherapy regimens with carboplatin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrCa5QjbkmSabVg90H21EOLACvtfcHk0lhJASM7msEq5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XlsF2msL8%253D&md5=1f5df69ea174bfc767ed5e8441528582</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1016%2Fj.canlet.2016.04.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.canlet.2016.04.002%26sid%3Dliteratum%253Aachs%26aulast%3DLiang%26aufirst%3DD.%2BH.%26aulast%3DChoi%26aufirst%3DD.%2BS.%26aulast%3DEnsor%26aufirst%3DJ.%2BE.%26aulast%3DKaipparettu%26aufirst%3DB.%2BA.%26aulast%3DBass%26aufirst%3DB.%2BL.%26aulast%3DChang%26aufirst%3DJ.%2BC.%26atitle%3DThe%2520autophagy%2520inhibitor%2520chloroquine%2520targets%2520cancer%2520stem%2520cells%2520in%2520triple%2520negative%2520breast%2520cancer%2520by%2520inducing%2520mitochondrial%2520damage%2520and%2520impairing%2520DNA%2520break%2520repair%26jtitle%3DCancer%2520Lett.%26date%3D2016%26volume%3D376%26spage%3D249%26epage%3D258%26doi%3D10.1016%2Fj.canlet.2016.04.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gump, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thorburn, A.</span></span> <span> </span><span class="NLM_article-title">Sorting cells for basal and induced autophagic flux by quantitative ratiometric flow cytometry</span>. <i>Autophagy</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1327</span>– <span class="NLM_lpage">1334</span>, <span class="refDoi"> DOI: 10.4161/auto.29394</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.4161%2Fauto.29394" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=24915460" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1Sgu74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=1327-1334&author=J.+M.+Gumpauthor=A.+Thorburn&title=Sorting+cells+for+basal+and+induced+autophagic+flux+by+quantitative+ratiometric+flow+cytometry&doi=10.4161%2Fauto.29394"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Sorting cells for basal and induced autophagic flux by quantitative ratiometric flow cytometry</span></div><div class="casAuthors">Gump, Jacob M.; Thorburn, Andrew</div><div class="citationInfo"><span class="NLM_cas:title">Autophagy</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1327-1334, 8 pp.</span>CODEN:
                <span class="NLM_cas:coden">AUTOC9</span>;
        ISSN:<span class="NLM_cas:issn">1554-8627</span>.
    
            (<span class="NLM_cas:orgname">Landes Bioscience</span>)
        </div><div class="casAbstract">We detail here a protocol using tandem-tagged mCherry-EGFP-LC3 C-G-LC3 to quantify autophagic flux in single cells by ratiometric flow cytometry and to isolate subpopulations of cells based on their relative levels of autophagic flux.  This robust and sensitive method measures autophagic flux rather than autophagosome no. and is an important addn. to the autophagy researcher's array of tools for measuring autophagy.  Two crucial steps in this protocol are (i) generate cells constitutively expressing C-G-LC3 with low to medium fluorescence and low fluorescence variability, and (ii) correctly set up gates and voltage/gain on a properly equipped flow cytometer.  We have used this method to measure autophagic flux in a variety of cell types and exptl. systems using many different autophagy stimuli.  On a sorting flow cytometer, this technique can be used to isolate cells with different levels of basal autophagic flux, or cells with variable induction of flux in response to a given stimulus for further anal. or experimentation.  We have also combined quantification of autophagic flux with methods to measure apoptosis and cell surface proteins, demonstrating the usefulness of this protocol in combination with other flow cytometry labels and markers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrSuu4-b_9wvLVg90H21EOLACvtfcHk0lgMfz19W7hhHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1Sgu74%253D&md5=8426adc1181ac63cb49dd8decab71656</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.4161%2Fauto.29394&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252Fauto.29394%26sid%3Dliteratum%253Aachs%26aulast%3DGump%26aufirst%3DJ.%2BM.%26aulast%3DThorburn%26aufirst%3DA.%26atitle%3DSorting%2520cells%2520for%2520basal%2520and%2520induced%2520autophagic%2520flux%2520by%2520quantitative%2520ratiometric%2520flow%2520cytometry%26jtitle%3DAutophagy%26date%3D2014%26volume%3D10%26spage%3D1327%26epage%3D1334%26doi%3D10.4161%2Fauto.29394" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bai, P.</span></span> <span> </span><span class="NLM_article-title">Biology of poly (ADP-ribose) polymerases: the factotums of cell maintenance</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">947</span>– <span class="NLM_lpage">958</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2015.01.034</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1016%2Fj.molcel.2015.01.034" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=26091343" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1SqsbbI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=947-958&author=P.+Bai&title=Biology+of+poly+%28ADP-ribose%29+polymerases%3A+the+factotums+of+cell+maintenance&doi=10.1016%2Fj.molcel.2015.01.034"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Biology of poly(ADP-Ribose) polymerases: The factotums of cell maintenance</span></div><div class="casAuthors">Bai, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">947-958</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">The protein family of poly(ADP-ribose) polymerases (PARPs) or diphtheria toxin-type ADP-ribose transferases (ARTDs) are multidomain proteins originally identified as DNA repair factors.  There are 17 PARP enzymes in humans, and it is now evident that PARPs undertake more tasks than DNA repair.  The aim of this review is to give a comprehensive view of the biol. roles of the PARP family starting from the simplest biochem. reactions to complex regulatory circuits.  Special attention will be laid on discussing linkage of PARP enzymes with tumor biol., oxidative stress, inflammatory, and metabolic diseases.  A better understanding of PARP-mediated processes and pathologies may help in identifying new pathways and, by these, new targets to combat diseases that affect large populations and seriously shorten life expectancy and the quality of life, such as cancer, metabolic, or inflammatory diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOMvLRwxttfbVg90H21EOLACvtfcHk0lgMfz19W7hhHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1SqsbbI&md5=5c03dcea525dd8add3b4555bf6f8a144</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2015.01.034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2015.01.034%26sid%3Dliteratum%253Aachs%26aulast%3DBai%26aufirst%3DP.%26atitle%3DBiology%2520of%2520poly%2520%2528ADP-ribose%2529%2520polymerases%253A%2520the%2520factotums%2520of%2520cell%2520maintenance%26jtitle%3DMol.%2520Cell%26date%3D2015%26volume%3D58%26spage%3D947%26epage%3D958%26doi%3D10.1016%2Fj.molcel.2015.01.034" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Krishnakumar, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kraus, W. L.</span></span> <span> </span><span class="NLM_article-title">The PARP side of the nucleus: molecular actions, physiological outcomes, and clinical targets</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">8</span>– <span class="NLM_lpage">24</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2010.06.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1016%2Fj.molcel.2010.06.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=20603072" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BC3cXpsFejsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2010&pages=8-24&author=R.+Krishnakumarauthor=W.+L.+Kraus&title=The+PARP+side+of+the+nucleus%3A+molecular+actions%2C+physiological+outcomes%2C+and+clinical+targets&doi=10.1016%2Fj.molcel.2010.06.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">The PARP side of the nucleus: molecular actions, physiological outcomes, and clinical targets</span></div><div class="casAuthors">Krishnakumar, Raga; Kraus, W. Lee</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">8-24</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  The abundant nuclear enzyme PARP-1, a multifunctional regulator of chromatin structure, transcription, and genomic integrity, plays key roles in a wide variety of processes in the nucleus.  Recent studies have begun to connect the mol. functions of PARP-1 to specific physiol. and pathol. outcomes, many of which can be altered by an expanding array of chem. inhibitors of PARP enzymic activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoy5Mj-wajemLVg90H21EOLACvtfcHk0lj7Zabb1F0-FQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXpsFejsr4%253D&md5=afb9423e09ed68d2bfa589ed0d7b9f13</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2010.06.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2010.06.017%26sid%3Dliteratum%253Aachs%26aulast%3DKrishnakumar%26aufirst%3DR.%26aulast%3DKraus%26aufirst%3DW.%2BL.%26atitle%3DThe%2520PARP%2520side%2520of%2520the%2520nucleus%253A%2520molecular%2520actions%252C%2520physiological%2520outcomes%252C%2520and%2520clinical%2520targets%26jtitle%3DMol.%2520Cell%26date%3D2010%26volume%3D39%26spage%3D8%26epage%3D24%26doi%3D10.1016%2Fj.molcel.2010.06.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rouleau, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hendzel, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaufmann, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poirier, G. G.</span></span> <span> </span><span class="NLM_article-title">PARP inhibition: PARP1 and beyond</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">293</span>– <span class="NLM_lpage">301</span>, <span class="refDoi"> DOI: 10.1038/nrc2812</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1038%2Fnrc2812" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=20200537" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BC3cXis1yms7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2010&pages=293-301&author=M.+Rouleauauthor=A.+Patelauthor=M.+J.+Hendzelauthor=S.+H.+Kaufmannauthor=G.+G.+Poirier&title=PARP+inhibition%3A+PARP1+and+beyond&doi=10.1038%2Fnrc2812"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">PARP inhibition: PARP1 and beyond</span></div><div class="casAuthors">Rouleau, Michele; Patel, Anand; Hendzel, Michael J.; Kaufmann, Scott H.; Poirier, Guy G.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">293-301</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Recent findings have thrust poly(ADP-ribose) polymerases (PARPs) into the limelight as potential chemotherapeutic targets.  To provide a framework for understanding these recent observations, the authors have reviewed what is known about the structures and functions of the family of PARP enzymes, and then outlined a series of questions that should be addressed to guide the rational development of PARP inhibitors as anticancer agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqOiY_n1IClmbVg90H21EOLACvtfcHk0lj7Zabb1F0-FQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXis1yms7w%253D&md5=fa3e3964e84229317dadbd505f11ad86</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1038%2Fnrc2812&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc2812%26sid%3Dliteratum%253Aachs%26aulast%3DRouleau%26aufirst%3DM.%26aulast%3DPatel%26aufirst%3DA.%26aulast%3DHendzel%26aufirst%3DM.%2BJ.%26aulast%3DKaufmann%26aufirst%3DS.%2BH.%26aulast%3DPoirier%26aufirst%3DG.%2BG.%26atitle%3DPARP%2520inhibition%253A%2520PARP1%2520and%2520beyond%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2010%26volume%3D10%26spage%3D293%26epage%3D301%26doi%3D10.1038%2Fnrc2812" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dréan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lord, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ashworth, A.</span></span> <span> </span><span class="NLM_article-title">PARP inhibitor combination therapy</span>. <i>Crit. Rev. Oncol. Hematol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>108</i></span>,  <span class="NLM_fpage">73</span>– <span class="NLM_lpage">85</span>, <span class="refDoi"> DOI: 10.1016/j.critrevonc.2016.10.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1016%2Fj.critrevonc.2016.10.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=27931843" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A280%3ADC%252BC2sjkvFCgtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=108&publication_year=2016&pages=73-85&author=A.+Dr%C3%A9anauthor=C.+J.+Lordauthor=A.+Ashworth&title=PARP+inhibitor+combination+therapy&doi=10.1016%2Fj.critrevonc.2016.10.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">PARP inhibitor combination therapy</span></div><div class="casAuthors">Drean Amy; Lord Christopher J; Ashworth Alan</div><div class="citationInfo"><span class="NLM_cas:title">Critical reviews in oncology/hematology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">108</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">73-85</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">In 2014, olaparib (Lynparza) became the first PARP (Poly(ADP-ribose) polymerase) inhibitor to be approved for the treatment of cancer.  When used as single agents, PARP inhibitors can selectively target tumour cells with BRCA1 or BRCA2 tumour suppressor gene mutations through synthetic lethality.  However, PARP inhibition also shows considerable promise when used together with other therapeutic agents.  Here, we summarise both the pre-clinical and clinical evidence for the utility of such combinations and discuss the future prospects and challenges for PARP inhibitor combinatorial therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRrc1DGTVYh3rBItaL1WaVAfW6udTcc2eYxXerqZahHhrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2sjkvFCgtA%253D%253D&md5=77ba6fe10daf60936688cf320259e996</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1016%2Fj.critrevonc.2016.10.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.critrevonc.2016.10.010%26sid%3Dliteratum%253Aachs%26aulast%3DDr%25C3%25A9an%26aufirst%3DA.%26aulast%3DLord%26aufirst%3DC.%2BJ.%26aulast%3DAshworth%26aufirst%3DA.%26atitle%3DPARP%2520inhibitor%2520combination%2520therapy%26jtitle%3DCrit.%2520Rev.%2520Oncol.%2520Hematol.%26date%3D2016%26volume%3D108%26spage%3D73%26epage%3D85%26doi%3D10.1016%2Fj.critrevonc.2016.10.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zai, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ju, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span> <span> </span><span class="NLM_article-title">Targeting PARP and autophagy evoked synergistic lethality in hepatocellular carcinoma</span>. <i>Carcinogenesis</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">345</span>– <span class="NLM_lpage">357</span>, <span class="refDoi"> DOI: 10.1093/carcin/bgz104</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1093%2Fcarcin%2Fbgz104" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2019&pages=345-357&author=W.+Zaiauthor=W.+Chenauthor=Y.+Hanauthor=Z.+Wuauthor=J.+Fanauthor=X.+Zhangauthor=J.+Luanauthor=S.+Tangauthor=X.+Jinauthor=X.+Fuauthor=H.+Gaoauthor=D.+Juauthor=H.+Liu&title=Targeting+PARP+and+autophagy+evoked+synergistic+lethality+in+hepatocellular+carcinoma&doi=10.1093%2Fcarcin%2Fbgz104"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1093%2Fcarcin%2Fbgz104&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fcarcin%252Fbgz104%26sid%3Dliteratum%253Aachs%26aulast%3DZai%26aufirst%3DW.%26aulast%3DChen%26aufirst%3DW.%26aulast%3DHan%26aufirst%3DY.%26aulast%3DWu%26aufirst%3DZ.%26aulast%3DFan%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DLuan%26aufirst%3DJ.%26aulast%3DTang%26aufirst%3DS.%26aulast%3DJin%26aufirst%3DX.%26aulast%3DFu%26aufirst%3DX.%26aulast%3DGao%26aufirst%3DH.%26aulast%3DJu%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DH.%26atitle%3DTargeting%2520PARP%2520and%2520autophagy%2520evoked%2520synergistic%2520lethality%2520in%2520hepatocellular%2520carcinoma%26jtitle%3DCarcinogenesis%26date%3D2019%26volume%3D41%26spage%3D345%26epage%3D357%26doi%3D10.1093%2Fcarcin%2Fbgz104" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huo, Z.</span></span> <span> </span><span class="NLM_article-title">Targeting autophagy potentiates the anti-tumor effect of PARP inhibitor in pediatric chronic myeloid leukemia</span>. <i>AMB Express</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">108</span>, <span class="refDoi"> DOI: 10.1186/s13568-019-0836-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1186%2Fs13568-019-0836-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=31309361" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A280%3ADC%252BB3MznslensQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2019&pages=108&author=Y.+Liuauthor=H.+Songauthor=H.+Songauthor=X.+Fengauthor=C.+Zhouauthor=Z.+Huo&title=Targeting+autophagy+potentiates+the+anti-tumor+effect+of+PARP+inhibitor+in+pediatric+chronic+myeloid+leukemia&doi=10.1186%2Fs13568-019-0836-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting autophagy potentiates the anti-tumor effect of PARP inhibitor in pediatric chronic myeloid leukemia</span></div><div class="casAuthors">Liu Yuanyuan; Song Hong; Song Huanqing; Feng Xiaoxia; Zhou Chuan; Huo Zhifang</div><div class="citationInfo"><span class="NLM_cas:title">AMB Express</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">108</span>
        ISSN:<span class="NLM_cas:issn">2191-0855</span>.
    </div><div class="casAbstract">Due to its potent cytotoxicity in BRCA-mutated tumors, synthetic lethality elicited by poly (ADP-ribose) polymerase (PARP) inhibitor gives renewed enthusiasm to researching and developing anti-cancer therapies.  Chronic myeloid leukemia (CML) is a type of cancers that starts in certain blood-forming cells of the bone marrow.  Here, we showed that poly (ADP-ribose) polymerase (PARP) inhibitor talazoparib could induce a concentration-dependent cytotoxicity in CML cells derived from pediatric patients.  During talazoparib treatment, autophagy was markedly activated, which was confirmed by the accumulation of autophagosomes, decrease of SQSTM1 and up-regulation of LC3-II.  Inhibition of autophagy by pharmaceutical inhibitor chloroquine or small-interfering RNA siATG5 significantly increased the cytotoxicity of talazoparib in pediatric CML cells and elicited synergistic anti-tumor effect in patient-derived xenograft model.  Our data demonstrated that autophagy played a cyto-protective role in talazoparib-treated pediatric CML and co-treatment with talazoparib and autophagy inhibitor could induce synergetic anti-tumor effect, providing novel insights for pediatric CML treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRc23_CSyS503T4ugvCDgnZfW6udTcc2eZUcBbAEXfqdbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3MznslensQ%253D%253D&md5=9252afe6c9793e6e23a35ba9da35ad5c</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1186%2Fs13568-019-0836-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs13568-019-0836-z%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DSong%26aufirst%3DH.%26aulast%3DSong%26aufirst%3DH.%26aulast%3DFeng%26aufirst%3DX.%26aulast%3DZhou%26aufirst%3DC.%26aulast%3DHuo%26aufirst%3DZ.%26atitle%3DTargeting%2520autophagy%2520potentiates%2520the%2520anti-tumor%2520effect%2520of%2520PARP%2520inhibitor%2520in%2520pediatric%2520chronic%2520myeloid%2520leukemia%26jtitle%3DAMB%2520Express%26date%3D2019%26volume%3D9%26spage%3D108%26doi%3D10.1186%2Fs13568-019-0836-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Di
Veroli, G. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fornari, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mollard, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bramhall, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richards, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jodrell, D. I.</span></span> <span> </span><span class="NLM_article-title">Combenefit: an interactive platform for the analysis and visualization of drug combinations</span>. <i>Bioinformatics</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">2866</span>– <span class="NLM_lpage">2868</span>, <span class="refDoi"> DOI: 10.1093/bioinformatics/btw230</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1093%2Fbioinformatics%2Fbtw230" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=27153664" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtFGlurbO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2016&pages=2866-2868&author=G.+Y.+Di%0AVeroliauthor=C.+Fornariauthor=D.+Wangauthor=S.+Mollardauthor=J.+L.+Bramhallauthor=F.+M.+Richardsauthor=D.+I.+Jodrell&title=Combenefit%3A+an+interactive+platform+for+the+analysis+and+visualization+of+drug+combinations&doi=10.1093%2Fbioinformatics%2Fbtw230"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Combenefit: an interactive platform for the analysis and visualization of drug combinations</span></div><div class="casAuthors">Di Veroli, Giovanni Y.; Fornari, Chiara; Wang, Dennis; Mollard, Severine; Bramhall, Jo L.; Richards, Frances M.; Jodrell, Duncan I.</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">2866-2868</span>CODEN:
                <span class="NLM_cas:coden">BOINFP</span>;
        ISSN:<span class="NLM_cas:issn">1367-4803</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Motivation: Many drug combinations are routinely assessed to identify synergistic interactions in the attempt to develop novel treatment strategies.  Appropriate software is required to analyze the results of these studies.  Results: We present Combenefit, new free software tool that enables the visualization, anal. and quantification of drug combination effects in terms of synergy and/or antagonism.  Data from combinations assays can be processed using classical Synergy models (Loewe, Bliss, HSA), as single expts. or in batch for High Throughput Screens.  This user-friendly tool provides lab. scientists with an easy and systematic way to analyze their data.  The companion package provides bioinformaticians with crit. implementations of routines enabling the processing of combination data.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-sh5U3Gp4wrVg90H21EOLACvtfcHk0lhnXnU4BVxrIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtFGlurbO&md5=97422dbe25c99eda4f7a8a61f775dd1e</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtw230&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtw230%26sid%3Dliteratum%253Aachs%26aulast%3DDi%2BVeroli%26aufirst%3DG.%2BY.%26aulast%3DFornari%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DD.%26aulast%3DMollard%26aufirst%3DS.%26aulast%3DBramhall%26aufirst%3DJ.%2BL.%26aulast%3DRichards%26aufirst%3DF.%2BM.%26aulast%3DJodrell%26aufirst%3DD.%2BI.%26atitle%3DCombenefit%253A%2520an%2520interactive%2520platform%2520for%2520the%2520analysis%2520and%2520visualization%2520of%2520drug%2520combinations%26jtitle%3DBioinformatics%26date%3D2016%26volume%3D32%26spage%3D2866%26epage%3D2868%26doi%3D10.1093%2Fbioinformatics%2Fbtw230" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Papadimitriou, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mountzios, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Papadimitriou, C. A.</span></span> <span> </span><span class="NLM_article-title">The role of PARP inhibition in triple-negative breast cancer: unraveling the wide spectrum of synthetic lethality</span>. <i>Cancer Treat. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">34</span>– <span class="NLM_lpage">44</span>, <span class="refDoi"> DOI: 10.1016/j.ctrv.2018.04.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1016%2Fj.ctrv.2018.04.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=29753961" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BC1cXptVCrtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2018&pages=34-44&author=M.+Papadimitriouauthor=G.+Mountziosauthor=C.+A.+Papadimitriou&title=The+role+of+PARP+inhibition+in+triple-negative+breast+cancer%3A+unraveling+the+wide+spectrum+of+synthetic+lethality&doi=10.1016%2Fj.ctrv.2018.04.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">The role of PARP inhibition in triple-negative breast cancer: Unraveling the wide spectrum of synthetic lethality</span></div><div class="casAuthors">Papadimitriou, Marios; Mountzios, Giannis; Papadimitriou, Christos A.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Treatment Reviews</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">34-44</span>CODEN:
                <span class="NLM_cas:coden">CTREDJ</span>;
        ISSN:<span class="NLM_cas:issn">0305-7372</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Triple-neg. breast cancer (TNBC) accounts for approx. 15-20% of all breast cancers and is characterized by a lack of immunohistochem. expression of estrogen receptors (ER), progesterone receptors (PR) and HER2.  TNBC is assocd. with poor long-term outcomes compared with other breast cancer subtypes.  Many of these tumors are also basal-like cancers which are characterized by an aggressive biol. behavior with a distant recurrence peak obsd. early at 3 years following diagnosis.  Furthermore, metastatic TNBC bears a dismal prognosis with an av. survival of 12 mo.  Although the prevalence of genetic alterations among women with TNBC differs significantly by ethnicity, race and age, BRCA mutations (including both germline mutations and somatic genetic aberrations) are found in up to 20-25% of unselected patients and esp. in those of the basal-like immunophenotype.  Therefore, defects in the DNA repair pathway could represent a promising therapeutic target for this subgroup of TNBC patients.  Poly(ADP-ribose) polymerase (PARP) inhibitors exploit this deficiency through synthetic lethality and have emerged as promising anticancer therapies, esp. in BRCA1 or BRCA2 mutation carriers.  Several PARP inhibitors are currently being evaluated in the adjuvant, neo-adjuvant, and metastatic setting for the treatment of breast cancer patients with a deficient homologous recombination pathway.  In this article, we review the major mol. characteristics of TNBC, the mechanisms of homologous recombination, and the role of PARP inhibition as an emerging therapeutic strategy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrH50CyPikNqLVg90H21EOLACvtfcHk0ljDYdb_vKWnUA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXptVCrtL8%253D&md5=e86a097ea52737257618cf18b8c05c7e</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1016%2Fj.ctrv.2018.04.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ctrv.2018.04.010%26sid%3Dliteratum%253Aachs%26aulast%3DPapadimitriou%26aufirst%3DM.%26aulast%3DMountzios%26aufirst%3DG.%26aulast%3DPapadimitriou%26aufirst%3DC.%2BA.%26atitle%3DThe%2520role%2520of%2520PARP%2520inhibition%2520in%2520triple-negative%2520breast%2520cancer%253A%2520unraveling%2520the%2520wide%2520spectrum%2520of%2520synthetic%2520lethality%26jtitle%3DCancer%2520Treat.%2520Rev.%26date%3D2018%26volume%3D67%26spage%3D34%26epage%3D44%26doi%3D10.1016%2Fj.ctrv.2018.04.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kinsey, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Camolotto, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boespflug, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guillen, K. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foth, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Truong, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuman, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shea, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seipp, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yap, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burrell, L. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lum, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whisenant, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilcrease, G. W.,  III.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cavalieri, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rehbein, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cutler, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Affolter, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Welm, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Welm, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scaife, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snyder, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McMahon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomar, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Papke, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hobbs, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayes, T. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diehl, J. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goode, G. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaika, N. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Witkiewicz, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knudsen, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petricoin, E. F.,  III</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macdonald, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tran, N. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lyssiotis, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ying, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kimmelman, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cox, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Der, C. J.</span></span> <span> </span><span class="NLM_article-title">Protective autophagy elicited by RAF→ MEK→ ERK inhibition suggests a treatment strategy for RAS-driven cancers</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">620</span>, <span class="refDoi"> DOI: 10.1038/s41591-019-0367-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1038%2Fs41591-019-0367-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=30833748" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BC1MXmsV2isbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2019&pages=620&author=C.+G.+Kinseyauthor=S.+A.+Camolottoauthor=A.+M.+Boespflugauthor=K.+P.+Guillenauthor=M.+Fothauthor=A.+Truongauthor=S.+S.+Schumanauthor=J.+E.+Sheaauthor=M.+T.+Seippauthor=J.+T.+Yapauthor=L.+D.+Burrellauthor=D.+H.+Lumauthor=J.+R.+Whisenantauthor=G.+W.+Gilcreaseauthor=C.+C.+Cavalieriauthor=K.+M.+Rehbeinauthor=S.+L.+Cutlerauthor=K.+E.+Affolterauthor=A.+L.+Welmauthor=B.+E.+Welmauthor=C.+L.+Scaifeauthor=E.+L.+Snyderauthor=M.+McMahonauthor=G.+Tomarauthor=B.+Papkeauthor=G.+A.+Hobbsauthor=L.+Yanauthor=T.+K.+Hayesauthor=J.+N.+Diehlauthor=G.+D.+Goodeauthor=N.+V.+Chaikaauthor=Y.+Wangauthor=G.-F.+Zhangauthor=A.+K.+Witkiewiczauthor=E.+S.+Knudsenauthor=E.+F.+Petricoinauthor=P.+K.+Singhauthor=J.+M.+Macdonaldauthor=N.+L.+Tranauthor=C.+A.+Lyssiotisauthor=H.+Yingauthor=A.+C.+Kimmelmanauthor=A.+D.+Coxauthor=C.+J.+Der&title=Protective+autophagy+elicited+by+RAF%E2%86%92+MEK%E2%86%92+ERK+inhibition+suggests+a+treatment+strategy+for+RAS-driven+cancers&doi=10.1038%2Fs41591-019-0367-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Protective autophagy elicited by RAF→MEK→ERK inhibition suggests a treatment strategy for RAS-driven cancers</span></div><div class="casAuthors">Kinsey, Conan G.; Camolotto, Soledad A.; Boespflug, Amelie M.; Gullien, Katrin P.; Foth, Mona; Truong, Amanda; Schuman, Sophia S.; Shea, Jill E.; Seipp, Michael T.; Yap, Jeffrey T.; Burrell, Lance D.; Lum, David H.; Whisenant, Jonathan R.; Gilcrease, G. Weldon, III; Cavalieri, Courtney C.; Rehbein, Kaitrin M.; Cutler, Stephanie L.; Affolter, Kajsa E.; Welm, Alana L.; Welm, Bryan E.; Scaife, Courtney L.; Snyder, Eric L.; McMahon, Martin</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">620-627</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Pancreatic ductal adenocarcinoma (PDA) was responsible for ∼ 44,000 deaths in the United States in 2018 and is the epitome of a recalcitrant cancer driven by a pharmacol. intractable oncoprotein, KRAS1-4.  Downstream of KRAS, the RAF→MEK→ERK signaling pathway plays a central role in pancreatic carcinogenesis5.  However, paradoxically, inhibition of this pathway has provided no clin. benefit to patients with PDA6.  Here we show that inhibition of KRAS→RAF→MEK→ERK signaling elicits autophagy, a process of cellular recycling that protects PDA cells from the cytotoxic effects of KRAS pathway inhibition.  Mechanistically, inhibition of MEK1/2 leads to activation of the LKB1→AMPK→ULK1 signaling axis, a key regulator of autophagy.  Furthermore, combined inhibition of MEK1/2 plus autophagy displays synergistic anti-proliferative effects against PDA cell lines in vitro and promotes regression of xenografted patient-derived PDA tumors in mice.  The obsd. effect of combination trametinib plus chloroquine was not restricted to PDA as other tumors, including patient-derived xenografts (PDX) of NRAS-mutated melanoma and BRAF-mutated colorectal cancer displayed similar responses.  Finally, treatment of a patient with PDA with the combination of trametinib plus hydroxychloroquine resulted in a partial, but nonetheless striking disease response.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqYfVZ8qCSszrVg90H21EOLACvtfcHk0likQrfNQa9TgQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXmsV2isbw%253D&md5=0fa2c90810c3d9bc403f2f119389adbe</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1038%2Fs41591-019-0367-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41591-019-0367-9%26sid%3Dliteratum%253Aachs%26aulast%3DKinsey%26aufirst%3DC.%2BG.%26aulast%3DCamolotto%26aufirst%3DS.%2BA.%26aulast%3DBoespflug%26aufirst%3DA.%2BM.%26aulast%3DGuillen%26aufirst%3DK.%2BP.%26aulast%3DFoth%26aufirst%3DM.%26aulast%3DTruong%26aufirst%3DA.%26aulast%3DSchuman%26aufirst%3DS.%2BS.%26aulast%3DShea%26aufirst%3DJ.%2BE.%26aulast%3DSeipp%26aufirst%3DM.%2BT.%26aulast%3DYap%26aufirst%3DJ.%2BT.%26aulast%3DBurrell%26aufirst%3DL.%2BD.%26aulast%3DLum%26aufirst%3DD.%2BH.%26aulast%3DWhisenant%26aufirst%3DJ.%2BR.%26aulast%3DGilcrease%26aufirst%3DG.%2BW.%26aulast%3DCavalieri%26aufirst%3DC.%2BC.%26aulast%3DRehbein%26aufirst%3DK.%2BM.%26aulast%3DCutler%26aufirst%3DS.%2BL.%26aulast%3DAffolter%26aufirst%3DK.%2BE.%26aulast%3DWelm%26aufirst%3DA.%2BL.%26aulast%3DWelm%26aufirst%3DB.%2BE.%26aulast%3DScaife%26aufirst%3DC.%2BL.%26aulast%3DSnyder%26aufirst%3DE.%2BL.%26aulast%3DMcMahon%26aufirst%3DM.%26aulast%3DTomar%26aufirst%3DG.%26aulast%3DPapke%26aufirst%3DB.%26aulast%3DHobbs%26aufirst%3DG.%2BA.%26aulast%3DYan%26aufirst%3DL.%26aulast%3DHayes%26aufirst%3DT.%2BK.%26aulast%3DDiehl%26aufirst%3DJ.%2BN.%26aulast%3DGoode%26aufirst%3DG.%2BD.%26aulast%3DChaika%26aufirst%3DN.%2BV.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DG.-F.%26aulast%3DWitkiewicz%26aufirst%3DA.%2BK.%26aulast%3DKnudsen%26aufirst%3DE.%2BS.%26aulast%3DPetricoin%26aufirst%3DE.%2BF.%26aulast%3DSingh%26aufirst%3DP.%2BK.%26aulast%3DMacdonald%26aufirst%3DJ.%2BM.%26aulast%3DTran%26aufirst%3DN.%2BL.%26aulast%3DLyssiotis%26aufirst%3DC.%2BA.%26aulast%3DYing%26aufirst%3DH.%26aulast%3DKimmelman%26aufirst%3DA.%2BC.%26aulast%3DCox%26aufirst%3DA.%2BD.%26aulast%3DDer%26aufirst%3DC.%2BJ.%26atitle%3DProtective%2520autophagy%2520elicited%2520by%2520RAF%25E2%2586%2592%2520MEK%25E2%2586%2592%2520ERK%2520inhibition%2520suggests%2520a%2520treatment%2520strategy%2520for%2520RAS-driven%2520cancers%26jtitle%3DNat.%2520Med.%26date%3D2019%26volume%3D25%26spage%3D620%26doi%3D10.1038%2Fs41591-019-0367-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bryant, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stalnecker, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeitouni, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klomp, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tikunov, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gunda, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pierobon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waters, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">George, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomar, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Papke, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hobbs, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayes, T. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diehl, J. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goode, G. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaika, N. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Witkiewicz, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knudsen, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petricoin, E. F.,  III</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macdonald, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tran, N. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lyssiotis, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ying, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kimmelman, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cox, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De, C. J.</span></span> <span> </span><span class="NLM_article-title">Combination of ERK and autophagy inhibition as a treatment approach for pancreatic cancer</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">628</span>– <span class="NLM_lpage">640</span>, <span class="refDoi"> DOI: 10.1038/s41591-019-0368-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1038%2Fs41591-019-0368-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=30833752" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BC1MXmsV2isbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2019&pages=628-640&author=K.+L.+Bryantauthor=C.+A.+Stalneckerauthor=D.+Zeitouniauthor=J.+E.+Klompauthor=S.+Pengauthor=A.+P.+Tikunovauthor=V.+Gundaauthor=M.+Pierobonauthor=A.+M.+Watersauthor=S.+D.+Georgeauthor=G.+Tomarauthor=B.+Papkeauthor=G.+A.+Hobbsauthor=L.+Yanauthor=T.+K.+Hayesauthor=J.+N.+Diehlauthor=G.+D.+Goodeauthor=N.+V.+Chaikaauthor=Y.+Wangauthor=G.-F.+Zhangauthor=A.+K.+Witkiewiczauthor=E.+S.+Knudsenauthor=E.+F.+Petricoinauthor=P.+K.+Singhauthor=J.+M.+Macdonaldauthor=N.+L.+Tranauthor=C.+A.+Lyssiotisauthor=H.+Yingauthor=A.+C.+Kimmelmanauthor=A.+D.+Coxauthor=C.+J.+De&title=Combination+of+ERK+and+autophagy+inhibition+as+a+treatment+approach+for+pancreatic+cancer&doi=10.1038%2Fs41591-019-0368-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Combination of ERK and autophagy inhibition as a treatment approach for pancreatic cancer</span></div><div class="casAuthors">Bryant, Kirsten L.; Stalnecker, Clint A.; Zeitouni, Daniel; Klomp, Jennifer E.; Peng, Sen; Tikunov, Andrey P.; Gunda, Venugopal; Pierobon, Mariaelena; Waters, Andrew M.; George, Samuel D.; Tomar, Garima; Papke, Bjorn; Hobbs, G. Aaron; Yan, Liang; Hayes, Tikvah K.; Diehl, J. Nathaniel; Goode, Gennifer D.; Chaika, Nina V.; Wang, Yingxue; Zhang, Guo-Fang; Witkiewicz, Agnieszka K.; Knudsen, Erik S.; Petricoin, Emanuel F., III; Singh, Pankaj K.; Macdonald, Jeffrey M.; Tran, Nhan L.; Lyssiotis, Costas A.; Ying, Haoqiang; Kimmelman, Alec C.; Cox, Adrienne D.; Der, Channing J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">628-640</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Pancreatic ductal adenocarcinoma (PDAC) is characterized by KRAS- and autophagy-dependent tumorigenic growth, but the role of KRAS in supporting autophagy has not been established.  We show that, to our surprise, suppression of KRAS increased autophagic flux, as did pharmacol. inhibition of its effector ERK MAPK.  Furthermore, we demonstrate that either KRAS suppression or ERK inhibition decreased both glycolytic and mitochondrial functions.  We speculated that ERK inhibition might thus enhance PDAC dependence on autophagy, in part by impairing other KRAS- or ERK-driven metabolic processes.  Accordingly, we found that the autophagy inhibitor chloroquine and genetic or pharmacol. inhibition of specific autophagy regulators synergistically enhanced the ability of ERK inhibitors to mediate antitumor activity in KRAS-driven PDAC.  We conclude that combinations of pharmacol. inhibitors that concurrently block both ERK MAPK and autophagic processes that are upregulated in response to ERK inhibition may be effective treatments for PDAC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkazpCczVUqLVg90H21EOLACvtfcHk0lhOo-TtwTz3kQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXmsV2isbo%253D&md5=120142384aa8e884aa694f8490cefbe6</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1038%2Fs41591-019-0368-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41591-019-0368-8%26sid%3Dliteratum%253Aachs%26aulast%3DBryant%26aufirst%3DK.%2BL.%26aulast%3DStalnecker%26aufirst%3DC.%2BA.%26aulast%3DZeitouni%26aufirst%3DD.%26aulast%3DKlomp%26aufirst%3DJ.%2BE.%26aulast%3DPeng%26aufirst%3DS.%26aulast%3DTikunov%26aufirst%3DA.%2BP.%26aulast%3DGunda%26aufirst%3DV.%26aulast%3DPierobon%26aufirst%3DM.%26aulast%3DWaters%26aufirst%3DA.%2BM.%26aulast%3DGeorge%26aufirst%3DS.%2BD.%26aulast%3DTomar%26aufirst%3DG.%26aulast%3DPapke%26aufirst%3DB.%26aulast%3DHobbs%26aufirst%3DG.%2BA.%26aulast%3DYan%26aufirst%3DL.%26aulast%3DHayes%26aufirst%3DT.%2BK.%26aulast%3DDiehl%26aufirst%3DJ.%2BN.%26aulast%3DGoode%26aufirst%3DG.%2BD.%26aulast%3DChaika%26aufirst%3DN.%2BV.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DG.-F.%26aulast%3DWitkiewicz%26aufirst%3DA.%2BK.%26aulast%3DKnudsen%26aufirst%3DE.%2BS.%26aulast%3DPetricoin%26aufirst%3DE.%2BF.%26aulast%3DSingh%26aufirst%3DP.%2BK.%26aulast%3DMacdonald%26aufirst%3DJ.%2BM.%26aulast%3DTran%26aufirst%3DN.%2BL.%26aulast%3DLyssiotis%26aufirst%3DC.%2BA.%26aulast%3DYing%26aufirst%3DH.%26aulast%3DKimmelman%26aufirst%3DA.%2BC.%26aulast%3DCox%26aufirst%3DA.%2BD.%26aulast%3DDe%26aufirst%3DC.%2BJ.%26atitle%3DCombination%2520of%2520ERK%2520and%2520autophagy%2520inhibition%2520as%2520a%2520treatment%2520approach%2520for%2520pancreatic%2520cancer%26jtitle%3DNat.%2520Med.%26date%3D2019%26volume%3D25%26spage%3D628%26epage%3D640%26doi%3D10.1038%2Fs41591-019-0368-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rebecca, V. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amaravadi, R. K.</span></span> <span> </span><span class="NLM_article-title">Emerging strategies to effectively target autophagy in cancer</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">11</span>, <span class="refDoi"> DOI: 10.1038/onc.2015.99</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1038%2Fonc.2015.99" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=25893285" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BC28XjtFKktw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2016&pages=1-11&author=V.+W.+Rebeccaauthor=R.+K.+Amaravadi&title=Emerging+strategies+to+effectively+target+autophagy+in+cancer&doi=10.1038%2Fonc.2015.99"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Emerging strategies to effectively target autophagy in cancer</span></div><div class="casAuthors">Rebecca, V. W.; Amaravadi, R. K.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-11</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Autophagy serves a dichotomous role in cancer and recent advances have helped delineate the appropriate settings where inhibiting or promoting autophagy may confer therapeutic efficacy in patients.  Our evolving understanding of the mol. machinery responsible for the tightly controlled regulation of this homeostatic mechanism has begun to bear fruit in the way of autophagy-oriented clin. trials and promising lead compds. to modulate autophagy for therapeutic benefit.  In this manuscript we review the recent preclin. and clin. therapeutic strategies that involve autophagy modulation in cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqSXqZO1qEuorVg90H21EOLACvtfcHk0lhOo-TtwTz3kQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XjtFKktw%253D%253D&md5=2d316e630be8c239e72b820b63c9a969</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1038%2Fonc.2015.99&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fonc.2015.99%26sid%3Dliteratum%253Aachs%26aulast%3DRebecca%26aufirst%3DV.%2BW.%26aulast%3DAmaravadi%26aufirst%3DR.%2BK.%26atitle%3DEmerging%2520strategies%2520to%2520effectively%2520target%2520autophagy%2520in%2520cancer%26jtitle%3DOncogene%26date%3D2016%26volume%3D35%26spage%3D1%26epage%3D11%26doi%3D10.1038%2Fonc.2015.99" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kabsch, W.</span></span> <span> </span><span class="NLM_article-title">Xds</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">125</span>– <span class="NLM_lpage">132</span>, <span class="refDoi"> DOI: 10.1107/S0907444909047337</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1107%2FS0907444909047337" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=20124692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=125-132&author=W.+Kabsch&title=Xds&doi=10.1107%2FS0907444909047337"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Software XDS for image rotation, recognition and crystal symmetry assignment</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">125-132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The usage and control of recent modifications of the program package XDS for the processing of rotation images are described in the context of previous versions.  New features include automatic detn. of spot size and reflecting range and recognition and assignment of crystal symmetry.  Moreover, the limitations of earlier package versions on the no. of correction/scaling factors and the representation of pixel contents have been removed.  Large program parts have been restructured for parallel processing so that the quality and completeness of collected data can be assessed soon after measurement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv5AyUw00DYbVg90H21EOLACvtfcHk0lhOo-TtwTz3kQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D&md5=1aa9a38aeb3ce95af4ffb7d8b8a142bd</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047337%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DXds%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D125%26epage%3D132%26doi%3D10.1107%2FS0907444909047337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Evans, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span> <span> </span><span class="NLM_article-title">How good are my data and what is the resolution?</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>69</i></span>,  <span class="NLM_fpage">1204</span>– <span class="NLM_lpage">1214</span>, <span class="refDoi"> DOI: 10.1107/S0907444913000061</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1107%2FS0907444913000061" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=23793146" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BC3sXpvVejsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2013&pages=1204-1214&author=P.+R.+Evansauthor=G.+N.+Murshudov&title=How+good+are+my+data+and+what+is+the+resolution%3F&doi=10.1107%2FS0907444913000061"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">How good are my data and what is the resolution?</span></div><div class="casAuthors">Evans, Philip R.; Murshudov, Garib N.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1204-1214</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Following integration of the obsd. diffraction spots, the process of  data redn.' initially aims to det. the point-group symmetry of the data and the likely space group.  This can be performed with the program POINTLESS.  The scaling program then puts all the measurements on a common scale, avs. measurements of symmetry-related reflections (using the symmetry detd. previously) and produces many statistics that provide the first important measures of data quality.  A new scaling program, AIMLESS, implements scaling models similar to those in SCALA but adds some addnl. analyses.  From the analyses, a no. of decisions can be made about the quality of the data and whether some measurements should be discarded.  The effective  resoln.' of a data set is a difficult and possibly contentious question (particularly with referees of papers) and this is discussed in the light of tests comparing the data-processing statistics with trials of refinement against obsd. and simulated data, and automated model-building and comparison of maps calcd. with different resoln. limits.  These trials show that adding weak high-resoln. data beyond the commonly used limits may make some improvement and does no harm.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreOyPAR8GLBrVg90H21EOLACvtfcHk0ljSP-36ob5bFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXpvVejsL8%253D&md5=3f649426c21e9d423f94ef8d7c568233</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1107%2FS0907444913000061&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444913000061%26sid%3Dliteratum%253Aachs%26aulast%3DEvans%26aufirst%3DP.%2BR.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26atitle%3DHow%2520good%2520are%2520my%2520data%2520and%2520what%2520is%2520the%2520resolution%253F%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2013%26volume%3D69%26spage%3D1204%26epage%3D1214%26doi%3D10.1107%2FS0907444913000061" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span> <span> </span><span class="NLM_article-title">Acknowledging errors: advanced molecular replacement with Phaser</span>. In  <i>Protein Crystallography</i>; <span class="NLM_publisher-name">Springer: Humana Press</span>: <span class="NLM_publisher-loc">New York, NY</span>, <span class="NLM_year">2017</span>; pp.  <span class="NLM_fpage">421</span>– <span class="NLM_lpage">453</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1007%2F978-1-4939-7000-1_18" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&pages=421-453&author=A.+J.+McCoy&title=Protein+Crystallography"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1007%2F978-1-4939-7000-1_18&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252F978-1-4939-7000-1_18%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26atitle%3DAcknowledging%2520errors%253A%2520advanced%2520molecular%2520replacement%2520with%2520Phaser%26btitle%3DProtein%2520Crystallography%26pub%3DSpringer%253A%2520Humana%2520Press%26date%3D2017%26spage%3D421%26epage%3D453" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span> <span> </span><span class="NLM_article-title">Tools for ligand validation in Coot</span>. <i>Acta Crystallogr., Sect. D: Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>73</i></span>,  <span class="NLM_fpage">203</span>– <span class="NLM_lpage">210</span>, <span class="refDoi"> DOI: 10.1107/S2059798317003382</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1107%2FS2059798317003382" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=28362283" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BC2sXktFyrsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2017&pages=203-210&author=P.+Emsley&title=Tools+for+ligand+validation+in+Coot&doi=10.1107%2FS2059798317003382"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Tools for ligand validation in Coot</span></div><div class="casAuthors">Emsley, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Structural Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">73</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">203-210</span>CODEN:
                <span class="NLM_cas:coden">ACSDAD</span>;
        ISSN:<span class="NLM_cas:issn">2059-7983</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics program primarily aimed at model building using X-ray data.  Recently, tools for the manipulation and representation of ligands have been introduced.  Here, these new tools for ligand validation and comparison are described.  Ligands in the wwPDB have been scored by d.-fit, distortion and atom-clash metrics.  The distributions of these scores can be used to assess the relative merits of the particular ligand in the protein-ligand complex of interest by means of 'sliders' akin to those now available for each accession code on the wwPDB websites.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGponrHT1Fibu7Vg90H21EOLACvtfcHk0ljSP-36ob5bFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXktFyrsLc%253D&md5=f67392ef4aada17fdffadeda97900455</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1107%2FS2059798317003382&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS2059798317003382%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26atitle%3DTools%2520for%2520ligand%2520validation%2520in%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Struct.%2520Biol.%26date%3D2017%26volume%3D73%26spage%3D203%26epage%3D210%26doi%3D10.1107%2FS2059798317003382" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref64"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref64'); return false;" data-citation="" class="refNumLink">64</a></strong><div class="NLM_citation" id="cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Skubák, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pannu, N. S.</span></span> <span> </span><span class="NLM_article-title">Direct incorporation of experimental phase information in model refinement</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">2196</span>– <span class="NLM_lpage">2201</span>, <span class="refDoi"> DOI: 10.1107/S0907444904019079</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1107%2FS0907444904019079" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=15572772" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars7rM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2196-2201&author=P.+Skub%C3%A1kauthor=G.+N.+Murshudovauthor=N.+S.+Pannu&title=Direct+incorporation+of+experimental+phase+information+in+model+refinement&doi=10.1107%2FS0907444904019079"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Direct incorporation of experimental phase information in model refinement</span></div><div class="casAuthors">Skubak, Pavol; Murshudov, Garib N.; Pannu, Navraj S.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2196-2201</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">The incorporation of prior phase information into a max.-likelihood formalism has been shown to strengthen model refinement.  However, the currently available likelihood refinement target using prior phase information has shortcomings; the 'phased' refinement target considers exptl. phase information indirectly and statically in the form of Hendrickson-Lattman coeffs.  Furthermore, the current refinement target implicitly assumes that the prior phase information is independent of the calcd. model structure factor.  This paper describes the derivation of a multivariate likelihood function that overcomes these shortcomings and directly incorporates exptl. phase information from a single-wavelength anomalous diffraction (SAD) expt.  This function, which simultaneously refines heavy-atom and model parameters, has been implemented in the refinement program REFMAC5.  The SAD function used in conjunction with the automated model-building procedures of ARP/wARP leads to a successful soln. when current likelihood functions fail in a test case shown.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUod9J97BAj7Vg90H21EOLACvtfcHk0ljSP-36ob5bFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars7rM&md5=a3d4fcf3a6c4b14357068aa16b6704d3</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1107%2FS0907444904019079&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904019079%26sid%3Dliteratum%253Aachs%26aulast%3DSkub%25C3%25A1k%26aufirst%3DP.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DPannu%26aufirst%3DN.%2BS.%26atitle%3DDirect%2520incorporation%2520of%2520experimental%2520phase%2520information%2520in%2520model%2520refinement%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26spage%3D2196%26epage%3D2201%26doi%3D10.1107%2FS0907444904019079" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref65"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref65'); return false;" data-citation="" class="refNumLink">65</a></strong><div class="NLM_citation" id="cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cunningham, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michael, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKeithan, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spiering, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carrette, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, C.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bushway, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tierney, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albini, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giacca, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mano, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Puri, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sacco, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruiz-Lozano, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riou, J.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Umbhauer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duester, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mercola, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colas, A. R.</span></span> <span> </span><span class="NLM_article-title">Id genes are essential for early heart formation</span>. <i>Genes Dev.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">1325</span>– <span class="NLM_lpage">1338</span>, <span class="refDoi"> DOI: 10.1101/gad.300400.117</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1101%2Fgad.300400.117" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=28794185" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BC1cXptlOnu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2017&pages=1325-1338&author=T.+J.+Cunninghamauthor=S.+Y.+Michaelauthor=W.+L.+McKeithanauthor=S.+Spieringauthor=F.+Carretteauthor=C.-T.+Huangauthor=P.+J.+Bushwayauthor=M.+Tierneyauthor=S.+Albiniauthor=M.+Giaccaauthor=M.+Manoauthor=P.+L.+Puriauthor=A.+Saccoauthor=P.+Ruiz-Lozanoauthor=J.-F.+Riouauthor=M.+Umbhauerauthor=G.+Duesterauthor=M.+Mercolaauthor=A.+R.+Colas&title=Id+genes+are+essential+for+early+heart+formation&doi=10.1101%2Fgad.300400.117"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Id genes are essential for early heart formation</span></div><div class="casAuthors">Cunningham, Thomas J.; Yu, Michael S.; McKeithan, Wesley L.; Spiering, Sean; Carrette, Florent; Huang, Chun-Teng; Bushway, Paul J.; Tierney, Matthew; Albini, Sonia; Giacca, Mauro; Mano, Miguel; Puri, Pier Lorenzo; Sacco, Alessandra; Ruiz-Lozano, Pilar; Riou, Jean-Francois; Umbhauer, Muriel; Duester, Gregg; Mercola, Mark; Colas, Alexandre R.</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">1325-1338</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">Deciphering the fundamental mechanisms controlling cardiac specification is crit. for our understanding of how heart formation is initiated during embryonic development and for applying stem cell biol. to regenerative medicine and disease modeling.  Using systematic and unbiased functional screening approaches, we discovered that the Id family of helix-loop-helix proteins is both necessary and sufficient to direct cardiac mesoderm formation in frog embryos and human embryonic stem cells.  Mechanistically, Id proteins specify cardiac cell fate by repressing two inhibitors of cardiogenic mesoderm formation-Tcf3 and Foxa2-and activating inducers Evx1, Grrp1, and Mesp1.  Most importantly, CRISPR/Cas9-mediated ablation of the entire Id (Id1-4) family in mouse embryos leads to failure of anterior cardiac progenitor specification and the development of heartless embryos.  Thus, Id proteins play a central and evolutionarily conserved role during heart formation and provide a novel means to efficiently produce cardiovascular progenitors for regenerative medicine and drug discovery applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqBtCAjfxPveLVg90H21EOLACvtfcHk0lhj81ez3u9kUA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXptlOnu7s%253D&md5=6d2a26f05a9356be7ff171074cb139cd</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1101%2Fgad.300400.117&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.300400.117%26sid%3Dliteratum%253Aachs%26aulast%3DCunningham%26aufirst%3DT.%2BJ.%26aulast%3DMichael%26aufirst%3DS.%2BY.%26aulast%3DMcKeithan%26aufirst%3DW.%2BL.%26aulast%3DSpiering%26aufirst%3DS.%26aulast%3DCarrette%26aufirst%3DF.%26aulast%3DHuang%26aufirst%3DC.-T.%26aulast%3DBushway%26aufirst%3DP.%2BJ.%26aulast%3DTierney%26aufirst%3DM.%26aulast%3DAlbini%26aufirst%3DS.%26aulast%3DGiacca%26aufirst%3DM.%26aulast%3DMano%26aufirst%3DM.%26aulast%3DPuri%26aufirst%3DP.%2BL.%26aulast%3DSacco%26aufirst%3DA.%26aulast%3DRuiz-Lozano%26aufirst%3DP.%26aulast%3DRiou%26aufirst%3DJ.-F.%26aulast%3DUmbhauer%26aufirst%3DM.%26aulast%3DDuester%26aufirst%3DG.%26aulast%3DMercola%26aufirst%3DM.%26aulast%3DColas%26aufirst%3DA.%2BR.%26atitle%3DId%2520genes%2520are%2520essential%2520for%2520early%2520heart%2520formation%26jtitle%3DGenes%2520Dev.%26date%3D2017%26volume%3D31%26spage%3D1325%26epage%3D1338%26doi%3D10.1101%2Fgad.300400.117" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62'],'ref63':['cit63'],'ref64':['cit64'],'ref65':['cit65']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 2 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sebastian  Mathea</span>, <span class="hlFld-ContribAuthor ">Eidarus  Salah</span>, <span class="hlFld-ContribAuthor ">Cynthia  Tallant</span>, <span class="hlFld-ContribAuthor ">Deep  Chatterjee</span>, <span class="hlFld-ContribAuthor ">Benedict-Tilman  Berger</span>, <span class="hlFld-ContribAuthor ">Rebecca  Konietzny</span>, <span class="hlFld-ContribAuthor ">Susanne  Müller</span>, <span class="hlFld-ContribAuthor ">Benedikt M.  Kessler</span>, <span class="hlFld-ContribAuthor ">Stefan  Knapp</span>. </span><span class="cited-content_cbyCitation_article-title">Conformational plasticity of the ULK3 kinase domain. </span><span class="cited-content_cbyCitation_journal-name">Biochemical Journal</span><span> <strong>2021,</strong> <em>478 </em>
                                    (14)
                                     , 2811-2823. <a href="https://doi.org/10.1042/BCJ20210257" title="DOI URL">https://doi.org/10.1042/BCJ20210257</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1042/BCJ20210257&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1042%2FBCJ20210257%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochemical%2520Journal%26atitle%3DConformational%252Bplasticity%252Bof%252Bthe%252BULK3%252Bkinase%252Bdomain%26aulast%3DMathea%26aufirst%3DSebastian%26date%3D2021%26date%3D2021%26volume%3D478%26issue%3D14%26spage%3D2811%26epage%3D2823" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yingying  Li</span>, <span class="hlFld-ContribAuthor ">Shan  Gao</span>, <span class="hlFld-ContribAuthor ">Xiyou  Du</span>, <span class="hlFld-ContribAuthor ">Jianbo  Ji</span>, <span class="hlFld-ContribAuthor ">Yanwei  Xi</span>, <span class="hlFld-ContribAuthor ">Guangxi  Zhai</span>. </span><span class="cited-content_cbyCitation_article-title">Advances in autophagy as a target in the treatment of tumors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Drug Targeting</span><span> <strong>2021,</strong> <em> </em>, 1-61. <a href="https://doi.org/10.1080/1061186X.2021.1961792" title="DOI URL">https://doi.org/10.1080/1061186X.2021.1961792</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/1061186X.2021.1961792&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F1061186X.2021.1961792%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Drug%2520Targeting%26atitle%3DAdvances%252Bin%252Bautophagy%252Bas%252Ba%252Btarget%252Bin%252Bthe%252Btreatment%252Bof%252Btumors%26aulast%3DLi%26aufirst%3DYingying%26date%3D2021%26date%3D2021%26spage%3D1%26epage%3D61" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00873/20201203/images/medium/jm0c00873_0013.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00873/20201203/images/large/jm0c00873_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00873&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00873/20201203/images/medium/jm0c00873_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00873/20201203/images/large/jm0c00873_0002.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Crystal structure of the kinase domain of ULK2 bound to chemical probe SBI-0206965 (<b>1</b>). (A) Crystal structure of SBI-0206965 bound to the interlobe ATP site in the kinase domain of ULK2 at 2.7 Å (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YID">6YID</a>). (B) SBI-0206965 forms hydrogen bonds to Cys88 in the hinge region. (C) Detailed interactions with hinge residue atoms and Lys39 are indicated. (D) In silico docking of compound <b>3</b> to the crystal structure of ULK2.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00873/20201203/images/large/jm0c00873_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00873&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00873/20201203/images/medium/jm0c00873_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00873/20201203/images/large/jm0c00873_0010.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. SAR Strategy Around ULK1/2 Fragments<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00873/20201203/images/large/jm0c00873_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00873&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>(A) Compounds <b>2</b> and <b>3</b> identified in a screen for ULK1 inhibition, with IC<sub>50</sub> values (ADP-Glo assay) and (B) a synthetic scheme for <i>N</i><sup>4</sup>-cyclopropyl-5-(trifluoromethyl)pyrimidine ULK inhibitors.</p></p></figure><figure data-id="fig2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00873/20201203/images/medium/jm0c00873_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00873/20201203/images/large/jm0c00873_0003.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. ULK1/2 inhibitors block phosphorylation of downstream autophagy target proteins. Western blot analysis (upper panels) and quantification (lower panels) of Beclin1 and VPS34 phosphorylation. HEK293T cells were transfected with Myc-tagged kinase inactive (KI) or wildtype (WT) ULK1 and WT Flag-tagged (A) Beclin1 or (B) VPS34 for 24 h. Cells were then treated with DMSO or the indicated ULK1/2 inhibitors at 10 μM for 1 h, lysed, and subjected to Western blotting with the indicated antibodies. Blots were quantified by densitometry. Data are expressed as the ratio of pBeclin1/Beclin1 or pVPS34/VPS34 normalized to DMSO-treated samples. Mean ± SD of three independent experiments. *<i>P</i> < 0.05, **<i>P</i> < 0.01 vs SBI-0206965 by two-sided unpaired test.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00873/20201203/images/large/jm0c00873_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00873&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00873/20201203/images/medium/jm0c00873_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00873/20201203/images/large/jm0c00873_0004.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. PK and PD analyses of compound <b>26</b> (SBP-7455). (A) Mice were dosed orally with compound <b>26</b> (30 mg/kg in 5% DMSO, 10% Tween 80, and 85% H<sub>2</sub>O). Plasma samples were taken at the indicated time points, and the concentration of compound was determined by LC–MS/MS analysis. The data were analyzed using PKSolver software. Mean ± SD of <i>N</i> = 3. Dotted line indicates the IC<sub>50</sub> of compound <b>26</b> in the ULK1 NanoBRET assay. (B) Mice were dosed with compound <b>26</b> (10 mg/kg) or vehicle (5% DMSO, 10% Tween 80, and 85% H<sub>2</sub>O) via oral gavage. Liver samples were harvested and lysed 2 h post dosing, and in vivo target engagement was assessed by Western blot. <i>N</i> = 3. All the lanes were on the same gel; the lanes between vehicle and SBP-7455 were cropped out.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00873/20201203/images/large/jm0c00873_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00873&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00873/20201203/images/medium/jm0c00873_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00873/20201203/images/large/jm0c00873_0005.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Ablation of ULK1 or BECN1 by shRNA-mediated knockdown reduces the viability of TNBC cells. MDA-MB-468 (A), BT549 (B), and MDA-MB-231 (C) cells transduced with scramble, ULK1-targeting, or BECN1-targeting shRNAs and selected in puromycin for 3 days before analysis. Cell viability was measured with the Promega CellTiter-Glo assay on the indicated days. Mean ± SD of three independent experiments, each performed in quadruplicate. Data are normalized to the viability of scramble shRNA control cells. *<i>P</i> < 0.05, **<i>P</i> < 0.01, ***<i>P</i> < 0.001 vs scramble control by two-sided unpaired t-test.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00873/20201203/images/large/jm0c00873_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00873&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00873/20201203/images/medium/jm0c00873_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00873/20201203/images/large/jm0c00873_0006.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. ULK1/2 inhibitors are cytotoxic as single agents against TNBC cells. (A) MDA-MB-468 cells were treated with the indicated concentrations of each ULK1/2 inhibitor or chloroquine (CQ) and incubated for 72 h. Cell viability was then measured using CellTiter-Glo. Compound treatment started at 30 μM for the ULK1/2 inhibitors and 90 μM for CQ with serial dilutions of 1:3. IC<sub>50</sub> values were determined using GraphPad Prism 8. The dose–response curves are representative of three independent experiments performed in quadruplicate; mean ± SD. (B) Compound <b>26</b> (SBP-7455) promotes apoptosis in TNBC cells under nutrient-deprived conditions. MDA-MB-468 cells were treated with DMSO or compound <b>26</b> (10 μM) in normal growth media (RPMI with 10% FBS, upper panel) or starvation media (EBSS, lower panel) for 18 h. The cells were then collected, and apoptosis was analyzed with PE-Annexin V/7-AAD staining by flow cytometry. Numbers indicate the percentage of cells in each gate. Shown is a representative of three independent experiments with similar results.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00873/20201203/images/large/jm0c00873_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00873&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00873/20201203/images/medium/jm0c00873_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00873/20201203/images/large/jm0c00873_0007.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. ULK1/2 inhibitors reduce autophagic flux. (A) MDA-MB-468 cells expressing mCherry-EGFP-LC3 fusion protein were used to monitor autophagic flux, as measured by the ratio of mCherry:GFP by flow cytometry. (B) Cells were incubated in normal growth media (control), with EBSS (starved) or EBSS plus the ULK1/2 inhibitors (10 μM) for 18 h before autophagic flux was assessed. Three independent experiments were performed; mean ± SD. *<i>P</i> < 0.001 by two-sided unpaired t-test of the high autophagic flux population. (C) Ultrastructural analysis of cells in (B) by TEM to monitor autophagosome accumulation. Red arrows indicate autophagosomes. (i) Representative untreated control cells, (ii) EBSS (starved) cells, (iii) cells treated with SBI-0206965 (10 μM), and (iv) cells treated with compound <b>26</b> (10 μM). Images are representative of 15–25 cells for each set of conditions analyzed by TEM.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00873/20201203/images/large/jm0c00873_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00873&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00873/20201203/images/medium/jm0c00873_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00873/20201203/images/large/jm0c00873_0008.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Compound <b>26</b> (SBP-7455) and the PARP inhibitor olaparib synergize to kill MDA-MB-468 cells. (A) MDA-MB-468 cells were treated with <b>26</b> (0.25 nM–15 μM) and olaparib (0.47–15 μM) for 72 h, and cell viability was assessed (CellTiter-Glo assay). A synergy graph was generated using the Combenefit Loewe model. Experiments were repeated three times (<i>n</i> = 4) with similar results. (B) Normalized cell viability after incubation with DMSO (vehicle control), olaparib (7.5 μM), compound <b>26</b> (0.19 μM), or a combination of both compounds. Mean ± SD. *<i>P</i> < 0.025, **<i>P</i> < 0.01, ***<i>P</i> < 0.001 <i>P</i> vs by two-sided unpaired t-test.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00873/20201203/images/large/jm0c00873_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00873&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00873/20201203/images/medium/jm0c00873_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00873/20201203/images/large/jm0c00873_0009.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Compound <b>26</b> (SBP-7455) inhibits the increase in autophagic flux induced by the PARP inhibitor olaparib. MDA-MB-468 cells expressing mCherry-EGFP-LC3 were treated with DMSO control, compound <b>26</b> (10 μM), olaparib (30 μM), or a combination of both for 48 h. Two independent experiments were performed; mean ± SD. *<i>P</i> < 0.05, **<i>P</i> < 0.01 for the high autophagic flux population by two-sided unpaired t-test.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c00873/20201203/images/large/jm0c00873_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00873&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i65">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_12204" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_12204" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 65 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bento, C. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renna, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghislat, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Puri, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ashkenazi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vicinanza, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Menzies, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rubinsztein, D. C.</span></span> <span> </span><span class="NLM_article-title">Mammalian autophagy: how does it work?</span>. <i>Annu. Rev. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>85</i></span>,  <span class="NLM_fpage">685</span>– <span class="NLM_lpage">713</span>, <span class="refDoi"> DOI: 10.1146/annurev-biochem-060815-014556</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1146%2Fannurev-biochem-060815-014556" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=26865532" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BC28XisVGitL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=85&publication_year=2016&pages=685-713&author=C.+F.+Bentoauthor=M.+Rennaauthor=G.+Ghislatauthor=C.+Puriauthor=A.+Ashkenaziauthor=M.+Vicinanzaauthor=F.+M.+Menziesauthor=D.+C.+Rubinsztein&title=Mammalian+autophagy%3A+how+does+it+work%3F&doi=10.1146%2Fannurev-biochem-060815-014556"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Mammalian Autophagy: How Does It Work?</span></div><div class="casAuthors">Bento, Carla F.; Renna, Maurizio; Ghislat, Ghita; Puri, Claudia; Ashkenazi, Avraham; Vicinanza, Mariella; Menzies, Fiona M.; Rubinsztein, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Biochemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">85</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">685-713</span>CODEN:
                <span class="NLM_cas:coden">ARBOAW</span>;
        ISSN:<span class="NLM_cas:issn">0066-4154</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews</span>)
        </div><div class="casAbstract">Autophagy is a conserved intracellular pathway that delivers cytoplasmic contents to lysosomes for degrdn. via double-membrane autophagosomes.  Autophagy substrates include organelles such as mitochondria, aggregate-prone proteins that cause neurodegeneration and various pathogens.  Thus, this pathway appears to be relevant to the pathogenesis of diverse diseases, and its modulation may have therapeutic value.  Here, we focus on the cell and mol. biol. of mammalian autophagy and review the key proteins that regulate the process by discussing their roles and how these may be modulated by posttranslational modifications.  We consider the membrane-trafficking events that impact autophagy and the questions relating to the sources of autophagosome membrane(s).  Finally, we discuss data from structural studies and some of the insights these have provided.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqp1Jf2PY-Uv7Vg90H21EOLACvtfcHk0lgB8YM01Q4QyQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XisVGitL0%253D&md5=f77b5661956ce2b101bc7ff32a02908e</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1146%2Fannurev-biochem-060815-014556&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-biochem-060815-014556%26sid%3Dliteratum%253Aachs%26aulast%3DBento%26aufirst%3DC.%2BF.%26aulast%3DRenna%26aufirst%3DM.%26aulast%3DGhislat%26aufirst%3DG.%26aulast%3DPuri%26aufirst%3DC.%26aulast%3DAshkenazi%26aufirst%3DA.%26aulast%3DVicinanza%26aufirst%3DM.%26aulast%3DMenzies%26aufirst%3DF.%2BM.%26aulast%3DRubinsztein%26aufirst%3DD.%2BC.%26atitle%3DMammalian%2520autophagy%253A%2520how%2520does%2520it%2520work%253F%26jtitle%3DAnnu.%2520Rev.%2520Biochem.%26date%3D2016%26volume%3D85%26spage%3D685%26epage%3D713%26doi%3D10.1146%2Fannurev-biochem-060815-014556" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kimmelman, A. C.</span></span> <span> </span><span class="NLM_article-title">The dynamic nature of autophagy in cancer</span>. <i>Genes Dev.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">1999</span>– <span class="NLM_lpage">2010</span>, <span class="refDoi"> DOI: 10.1101/gad.17558811</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1101%2Fgad.17558811" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=21979913" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlyhtr%252FK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2011&pages=1999-2010&author=A.+C.+Kimmelman&title=The+dynamic+nature+of+autophagy+in+cancer&doi=10.1101%2Fgad.17558811"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">The dynamic nature of autophagy in cancer</span></div><div class="casAuthors">Kimmelman, Alec C.</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">1999-2010</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">A review.  Macroautophagy (referred to hereafter as autophagy) is a highly regulated cellular process that serves to remove damaged proteins and organelles from the cell.  Autophagy contributes to an array of normal and pathol. processes, and has recently emerged as a key regulator of multiple aspects of cancer biol.  The role of autophagy in cancer is complex and is likely dependent on tumor type, stage, and genetic context.  This complexity is illustrated by the identification of settings where autophagy acts potently to either promote or inhibit tumorigenesis.  This review discusses the underlying basis for these opposing functions and propose a model suggesting a dynamic role for autophagy in malignancy.  Collectively, the data point to autophagy as serving as a barrier to limit tumor initiation.  Once neoplastic lesions are established, it appears that adaptive changes occur that now result in pos. roles for autophagy in malignant progression and in subsequent tumor maintenance.  Remarkably, constitutive activation of autophagy is crit. for continued growth of some tumors, serving to both reduce oxidative stress and provide key intermediates to sustain cell metab.  Autophagy is also induced in response to cancer therapies where it can function as a survival mechanism that limits drug efficacy.  These findings have inspired significant interest in applying anti-autophagy therapies as an entirely new approach to cancer treatment.  It is now apparent that aberrant control of autophagy is among the key hallmarks of cancer.  While much needs to be learned about the regulation and context-dependent biol. functions of autophagy, it seems clear that modulation of this process will be an attractive avenue for future cancer therapeutic approaches.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq4HmXPQn60FbVg90H21EOLACvtfcHk0lgB8YM01Q4QyQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlyhtr%252FK&md5=cde8efa9551e55503d5ace828906c068</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1101%2Fgad.17558811&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.17558811%26sid%3Dliteratum%253Aachs%26aulast%3DKimmelman%26aufirst%3DA.%2BC.%26atitle%3DThe%2520dynamic%2520nature%2520of%2520autophagy%2520in%2520cancer%26jtitle%3DGenes%2520Dev.%26date%3D2011%26volume%3D25%26spage%3D1999%26epage%3D2010%26doi%3D10.1101%2Fgad.17558811" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klionsky, D. J.</span></span> <span> </span><span class="NLM_article-title">Eaten alive: a history of macroautophagy</span>. <i>Nat. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">814</span>, <span class="refDoi"> DOI: 10.1038/ncb0910-814</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1038%2Fncb0910-814" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=20811353" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtFSju7zE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2010&pages=814&author=Z.+Yangauthor=D.+J.+Klionsky&title=Eaten+alive%3A+a+history+of+macroautophagy&doi=10.1038%2Fncb0910-814"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Eaten alive: a history of macroautophagy</span></div><div class="casAuthors">Yang, Zhifen; Klionsky, Daniel J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Cell Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">814-822</span>CODEN:
                <span class="NLM_cas:coden">NCBIFN</span>;
        ISSN:<span class="NLM_cas:issn">1465-7392</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Macroautophagy (hereafter autophagy), or 'self-eating', is a conserved cellular pathway that controls protein and organelle degrdn., and has essential roles in survival, development and homeostasis.  Autophagy is also integral to human health and is involved in physiol., development, lifespan and a wide range of diseases, including cancer, neurodegeneration and microbial infection.  Although research on this topic began in the late 1950s, substantial progress in the mol. study of autophagy has taken place during only the past 15 years.  This review traces the key findings that led to our current mol. understanding of this complex process.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJ_iZJ3YUoHrVg90H21EOLACvtfcHk0lgB8YM01Q4QyQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtFSju7zE&md5=a20abac7db3f20eac668c41052413397</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fncb0910-814&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncb0910-814%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DZ.%26aulast%3DKlionsky%26aufirst%3DD.%2BJ.%26atitle%3DEaten%2520alive%253A%2520a%2520history%2520of%2520macroautophagy%26jtitle%3DNat.%2520Cell%2520Biol.%26date%3D2010%26volume%3D12%26spage%3D814%26doi%3D10.1038%2Fncb0910-814" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Amaravadi, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lum, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bui, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christophorou, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evan, G. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas-Tikhonenko, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, C. B.</span></span> <span> </span><span class="NLM_article-title">Autophagy inhibition enhances therapy-induced apoptosis in a Myc-induced model of lymphoma</span>. <i>J. Clin. Invest.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>117</i></span>,  <span class="NLM_fpage">326</span>– <span class="NLM_lpage">336</span>, <span class="refDoi"> DOI: 10.1172/JCI28833</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1172%2FJCI28833" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=17235397" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhs1Oqsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=117&publication_year=2007&pages=326-336&author=R.+K.+Amaravadiauthor=D.+Yuauthor=J.+J.+Lumauthor=T.+Buiauthor=M.+A.+Christophorouauthor=G.+I.+Evanauthor=A.+Thomas-Tikhonenkoauthor=C.+B.+Thompson&title=Autophagy+inhibition+enhances+therapy-induced+apoptosis+in+a+Myc-induced+model+of+lymphoma&doi=10.1172%2FJCI28833"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Autophagy inhibition enhances therapy-induced apoptosis in a Myc-induced model of lymphoma</span></div><div class="casAuthors">Amaravadi, Ravi K.; Yu, Duonan; Lum, Julian J.; Bui, Thi; Christophorou, Maria A.; Evan, Gerard I.; Thomas-Tikhonenko, Andrei; Thompson, Craig B.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Clinical Investigation</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">117</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">326-336</span>CODEN:
                <span class="NLM_cas:coden">JCINAO</span>;
        ISSN:<span class="NLM_cas:issn">0021-9738</span>.
    
            (<span class="NLM_cas:orgname">American Society for Clinical Investigation</span>)
        </div><div class="casAbstract">Autophagy is a lysosome-dependent degradative pathway frequently activated in tumor cells treated with chemotherapy or radiation.  Whether autophagy obsd. in treated cancer cells represents a mechanism that allows tumor cells to survive therapy or a mechanism for initiating a nonapoptotic form of programmed cell death remains controversial.  To address this issue, the role of autophagy in a Myc-induced model of lymphoma generated from cells derived from p53ERTAM/p53ERTAM mice (with ER denoting estrogen receptor) was examd.  Such tumors are resistant to apoptosis due to a lack of nuclear p53.  Systemic administration of tamoxifen led to p53 activation and tumor regression followed by tumor recurrence.  Activation of p53 was assocd. with the rapid appearance of apoptotic cells and the induction of autophagy in surviving cells.  Inhibition of autophagy with either chloroquine or ATG5 short hairpin RNA (shRNA) enhanced the ability of either p53 activation or alkylating drug therapy to induce tumor cell death.  These studies provide evidence that autophagy serves as a survival pathway in tumor cells treated with apoptosis activators and a rationale for the use of autophagy inhibitors such as chloroquine in combination with therapies designed to induce apoptosis in human cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKpX84jI9L8rVg90H21EOLACvtfcHk0lgxm1m48rrh6w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhs1Oqsbs%253D&md5=bad463df500a94f489629685badb59e8</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1172%2FJCI28833&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1172%252FJCI28833%26sid%3Dliteratum%253Aachs%26aulast%3DAmaravadi%26aufirst%3DR.%2BK.%26aulast%3DYu%26aufirst%3DD.%26aulast%3DLum%26aufirst%3DJ.%2BJ.%26aulast%3DBui%26aufirst%3DT.%26aulast%3DChristophorou%26aufirst%3DM.%2BA.%26aulast%3DEvan%26aufirst%3DG.%2BI.%26aulast%3DThomas-Tikhonenko%26aufirst%3DA.%26aulast%3DThompson%26aufirst%3DC.%2BB.%26atitle%3DAutophagy%2520inhibition%2520enhances%2520therapy-induced%2520apoptosis%2520in%2520a%2520Myc-induced%2520model%2520of%2520lymphoma%26jtitle%3DJ.%2520Clin.%2520Invest.%26date%3D2007%26volume%3D117%26spage%3D326%26epage%3D336%26doi%3D10.1172%2FJCI28833" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hosokawa, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sasaki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iemura, S.-i.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Natsume, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hara, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mizushima, N.</span></span> <span> </span><span class="NLM_article-title">Atg101, a novel mammalian autophagy protein interacting with Atg13</span>. <i>Autophagy</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">973</span>– <span class="NLM_lpage">979</span>, <span class="refDoi"> DOI: 10.4161/auto.5.7.9296</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.4161%2Fauto.5.7.9296" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=19597335" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BC3cXmt1ChtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2009&pages=973-979&author=N.+Hosokawaauthor=T.+Sasakiauthor=S.-i.+Iemuraauthor=T.+Natsumeauthor=T.+Haraauthor=N.+Mizushima&title=Atg101%2C+a+novel+mammalian+autophagy+protein+interacting+with+Atg13&doi=10.4161%2Fauto.5.7.9296"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Atg101, a novel mammalian autophagy protein interacting with Atg13</span></div><div class="casAuthors">Hosokawa, Nao; Sasaki, Takahiro; Lemura, Shun-ichiro; Natsume, Tohru; Hara, Taichi; Mizushima, Noboru</div><div class="citationInfo"><span class="NLM_cas:title">Autophagy</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">973-979</span>CODEN:
                <span class="NLM_cas:coden">AUTOC9</span>;
        ISSN:<span class="NLM_cas:issn">1554-8627</span>.
    
            (<span class="NLM_cas:orgname">Landes Bioscience</span>)
        </div><div class="casAbstract">Autophagy is a major route by which cytoplasmic contents are delivered to the lysosome for degrdn.  Many autophagy-related (ATG) genes have been identified in yeast.  Although most of them are conserved in human, the mol. compn. of the Atg1 complex appears to differ between yeast and mammals.  In yeast, Atg1 forms a complex with Atg11, Atg13, Atg17, Atg29 and Atg31, whereas mammalian Atg1 (ULK1/2) interacts with Atg13 and FIP200.  Here, we identify a novel mammalian Atg13 binding protein, named Atg101.  Atg101 shows no homol. with other Atg proteins, and is conserved in various eukaryotes, but not in Saccharomyces cerevisiae.  Atg101 assocs. with the ULK-Atg13-FIP200 complex, most likely through direct interaction with Atg13.  In Atg13 siRNA-treated cells, Atg101 is present solely as a monomer.  Interaction between Atg101 and the ULK-Atg13-FIP200 complex is stable, and is not regulated by nutrient conditions.  GFP-Atg101 localizes to the isolation membrane/phagophore.  GFP-LC3 dot formation is suppressed and endogenous LC3-I accumulates in Atg101 siRNA-treated cells, suggesting that Atg101 is a crit. factor for autophagy.  Furthermore, Atg101 is important for the stability and basal phosphorylation of Atg13 and ULK1.  These data suggest that Atg101 is a novel Atg protein that functions together with ULK, Atg13 and FIP200.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXscuMfT-o27Vg90H21EOLACvtfcHk0lgxm1m48rrh6w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXmt1ChtQ%253D%253D&md5=f73ad092947c3835c9ffdf05921a2118</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.4161%2Fauto.5.7.9296&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252Fauto.5.7.9296%26sid%3Dliteratum%253Aachs%26aulast%3DHosokawa%26aufirst%3DN.%26aulast%3DSasaki%26aufirst%3DT.%26aulast%3DIemura%26aufirst%3DS.-i.%26aulast%3DNatsume%26aufirst%3DT.%26aulast%3DHara%26aufirst%3DT.%26aulast%3DMizushima%26aufirst%3DN.%26atitle%3DAtg101%252C%2520a%2520novel%2520mammalian%2520autophagy%2520protein%2520interacting%2520with%2520Atg13%26jtitle%3DAutophagy%26date%3D2009%26volume%3D5%26spage%3D973%26epage%3D979%26doi%3D10.4161%2Fauto.5.7.9296" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ganley, I. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, X.</span></span> <span> </span><span class="NLM_article-title">ULK1· ATG13 FIP200 complex mediates mTOR signaling and is essential for autophagy</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>284</i></span>,  <span class="NLM_fpage">12297</span>– <span class="NLM_lpage">12305</span>, <span class="refDoi"> DOI: 10.1074/jbc.M900573200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1074%2Fjbc.M900573200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=19258318" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BD1MXltVCmu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=284&publication_year=2009&pages=12297-12305&author=I.+G.+Ganleyauthor=D.+H.+Lamauthor=J.+Wangauthor=X.+Dingauthor=S.+Chenauthor=X.+Jiang&title=ULK1%C2%B7+ATG13+FIP200+complex+mediates+mTOR+signaling+and+is+essential+for+autophagy&doi=10.1074%2Fjbc.M900573200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">ULK1·ATG13·FIP200 Complex Mediates mTOR Signaling and Is Essential for Autophagy</span></div><div class="casAuthors">Ganley, Ian G.; Lam, Du H.; Wang, Junru; Ding, Xiaojun; Chen, She; Jiang, Xuejun</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">284</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">12297-12305</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Autophagy is a degradative process that recycles long-lived and faulty cellular components.  It is linked to many diseases and is required for normal development.  ULK1, a mammalian serine/threonine protein kinase, plays a key role in the initial stages of autophagy, though the exact mol. mechanism is unknown.  Here we report identification of a novel protein complex contg. ULK1 and two addnl. protein factors, FIP200 and ATG13, all of which are essential for starvation-induced autophagy.  Both FIP200 and ATG13 are crit. for correct localization of ULK1 to the pre-autophagosome and stability of ULK1 protein.  Addnl., using both cellular expts. and a de novo in vitro reconstituted reaction we demonstrate that FIP200 and ATG13 can enhance ULK1 kinase activity individually, but both are required for maximal stimulation.  Further, we show that ATG13 and ULK1 are phosphorylated by the mTOR pathway in a nutrient starvation-regulated manner, indicating that the ULK1·ATG13·FIP200 complex acts as a node for integrating incoming autophagy signals into autophagosome biogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEId7bV69EoLVg90H21EOLACvtfcHk0ljXrxnvmdNbCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXltVCmu7c%253D&md5=6443c68a54f1e604d097ad6f9ec10998</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M900573200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M900573200%26sid%3Dliteratum%253Aachs%26aulast%3DGanley%26aufirst%3DI.%2BG.%26aulast%3DLam%26aufirst%3DD.%2BH.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DDing%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DS.%26aulast%3DJiang%26aufirst%3DX.%26atitle%3DULK1%25C2%25B7%2520ATG13%2520FIP200%2520complex%2520mediates%2520mTOR%2520signaling%2520and%2520is%2520essential%2520for%2520autophagy%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2009%26volume%3D284%26spage%3D12297%26epage%3D12305%26doi%3D10.1074%2Fjbc.M900573200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Laplante, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sabatini, D. M.</span></span> <span> </span><span class="NLM_article-title">mTOR signaling in growth control and disease</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>149</i></span>,  <span class="NLM_fpage">274</span>– <span class="NLM_lpage">293</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2012.03.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1016%2Fj.cell.2012.03.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=22500797" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BC38Xls1eguro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=149&publication_year=2012&pages=274-293&author=M.+Laplanteauthor=D.+M.+Sabatini&title=mTOR+signaling+in+growth+control+and+disease&doi=10.1016%2Fj.cell.2012.03.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">mTOR signaling in growth control and disease</span></div><div class="casAuthors">Laplante, Mathieu; Sabatini, David M.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">149</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">274-293</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  The mechanistic target of rapamycin (mTOR) signaling pathway senses and integrates a variety of environmental cues to regulate organismal growth and homeostasis.  The pathway regulates many major cellular processes and is implicated in an increasing no. of pathol. conditions, including cancer, obesity, type 2 diabetes, and neurodegeneration.  Here, we review recent advances in our understanding of the mTOR pathway and its role in health, disease, and aging.  We further discuss pharmacol. approaches to treat human pathologies linked to mTOR deregulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqjbKBX-sScfrVg90H21EOLACvtfcHk0ljXrxnvmdNbCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xls1eguro%253D&md5=d1adac8ec64da63358e0af26a17ceb4e</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2012.03.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2012.03.017%26sid%3Dliteratum%253Aachs%26aulast%3DLaplante%26aufirst%3DM.%26aulast%3DSabatini%26aufirst%3DD.%2BM.%26atitle%3DmTOR%2520signaling%2520in%2520growth%2520control%2520and%2520disease%26jtitle%3DCell%26date%3D2012%26volume%3D149%26spage%3D274%26epage%3D293%26doi%3D10.1016%2Fj.cell.2012.03.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Galluzzi, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pietrocola, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levine, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kroemer, G.</span></span> <span> </span><span class="NLM_article-title">Metabolic control of autophagy</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>159</i></span>,  <span class="NLM_fpage">1263</span>– <span class="NLM_lpage">1276</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2014.11.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1016%2Fj.cell.2014.11.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=25480292" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitFOrt7bN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=159&publication_year=2014&pages=1263-1276&author=L.+Galluzziauthor=F.+Pietrocolaauthor=B.+Levineauthor=G.+Kroemer&title=Metabolic+control+of+autophagy&doi=10.1016%2Fj.cell.2014.11.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Metabolic Control of Autophagy</span></div><div class="casAuthors">Galluzzi, Lorenzo; Pietrocola, Federico; Levine, Beth; Kroemer, Guido</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">159</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1263-1276</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Macroautophagy (herein referred to as autophagy) is an evolutionarily conserved mechanism of adaptation to adverse microenvironmental conditions, including limited nutrient supplies.  Several sensors interacting with the autophagic machinery have evolved to detect fluctuations in key metabolic parameters.  The signal transduction cascades operating downstream of these sensors are highly interconnected to control a spatially and chronol. coordinated autophagic response that maintains the health and function of individual cells while preserving organismal homeostasis.  Here, we discuss the physiol. regulation of autophagy by metabolic circuitries, as well as alterations of such control in disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohrPb03zHBxbVg90H21EOLACvtfcHk0ljXrxnvmdNbCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitFOrt7bN&md5=f701e43e0183f880fc061ad38b3ef392</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2014.11.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2014.11.006%26sid%3Dliteratum%253Aachs%26aulast%3DGalluzzi%26aufirst%3DL.%26aulast%3DPietrocola%26aufirst%3DF.%26aulast%3DLevine%26aufirst%3DB.%26aulast%3DKroemer%26aufirst%3DG.%26atitle%3DMetabolic%2520control%2520of%2520autophagy%26jtitle%3DCell%26date%3D2014%26volume%3D159%26spage%3D1263%26epage%3D1276%26doi%3D10.1016%2Fj.cell.2014.11.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Papinski, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuschnig, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reiter, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilhelm, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barnes, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maiolica, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansmann, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pfaffenwimmer, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kijanska, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stoffel, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brezovich, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lou, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turk, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aebersold, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ammerer, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peter, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kraft, C.</span></span> <span> </span><span class="NLM_article-title">Early steps in autophagy depend on direct phosphorylation of Atg9 by the Atg1 kinase</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">471</span>– <span class="NLM_lpage">483</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2013.12.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1016%2Fj.molcel.2013.12.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=24440502" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitVyqu7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2014&pages=471-483&author=D.+Papinskiauthor=M.+Schuschnigauthor=W.+Reiterauthor=L.+Wilhelmauthor=C.+A.+Barnesauthor=A.+Maiolicaauthor=I.+Hansmannauthor=T.+Pfaffenwimmerauthor=M.+Kijanskaauthor=I.+Stoffelauthor=S.+S.+Leeauthor=A.+Brezovichauthor=J.+H.+Louauthor=B.+E.+Turkauthor=R.+Aebersoldauthor=G.+Ammererauthor=M.+Peterauthor=C.+Kraft&title=Early+steps+in+autophagy+depend+on+direct+phosphorylation+of+Atg9+by+the+Atg1+kinase&doi=10.1016%2Fj.molcel.2013.12.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Early Steps in Autophagy Depend on Direct Phosphorylation of Atg9 by the Atg1 Kinase</span></div><div class="casAuthors">Papinski, Daniel; Schuschnig, Martina; Reiter, Wolfgang; Wilhelm, Larissa; Barnes, Christopher A.; Maiolica, Alessio; Hansmann, Isabella; Pfaffenwimmer, Thaddaeus; Kijanska, Monika; Stoffel, Ingrid; Lee, Sung Sik; Brezovich, Andrea; Lou, Jane Hua; Turk, Benjamin E.; Aebersold, Ruedi; Ammerer, Gustav; Peter, Matthias; Kraft, Claudine</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">471-483</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Bulk degrdn. of cytoplasmic material is mediated by a highly conserved intracellular trafficking pathway termed autophagy.  This pathway is characterized by the formation of double-membrane vesicles termed autophagosomes engulfing the substrate and transporting it to the vacuole/lysosome for breakdown and recycling.  The Atg1/ULK1 kinase is essential for this process; however, little is known about its targets and the means by which it controls autophagy.  Here we have screened for Atg1 kinase substrates using consensus peptide arrays and identified three components of the autophagy machinery.  The multimembrane-spanning protein Atg9 is a direct target of this kinase essential for autophagy.  Phosphorylated Atg9 is then required for the efficient recruitment of Atg8 and Atg18 to the site of autophagosome formation and subsequent expansion of the isolation membrane, a prerequisite for a functioning autophagy pathway.  These findings show that the Atg1 kinase acts early in autophagy by regulating the outgrowth of autophagosomal membranes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBlZrV_9D_krVg90H21EOLACvtfcHk0ljXrxnvmdNbCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitVyqu7k%253D&md5=1a99d4b1d79554eb8d7ca1ea735af1be</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2013.12.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2013.12.011%26sid%3Dliteratum%253Aachs%26aulast%3DPapinski%26aufirst%3DD.%26aulast%3DSchuschnig%26aufirst%3DM.%26aulast%3DReiter%26aufirst%3DW.%26aulast%3DWilhelm%26aufirst%3DL.%26aulast%3DBarnes%26aufirst%3DC.%2BA.%26aulast%3DMaiolica%26aufirst%3DA.%26aulast%3DHansmann%26aufirst%3DI.%26aulast%3DPfaffenwimmer%26aufirst%3DT.%26aulast%3DKijanska%26aufirst%3DM.%26aulast%3DStoffel%26aufirst%3DI.%26aulast%3DLee%26aufirst%3DS.%2BS.%26aulast%3DBrezovich%26aufirst%3DA.%26aulast%3DLou%26aufirst%3DJ.%2BH.%26aulast%3DTurk%26aufirst%3DB.%2BE.%26aulast%3DAebersold%26aufirst%3DR.%26aulast%3DAmmerer%26aufirst%3DG.%26aulast%3DPeter%26aufirst%3DM.%26aulast%3DKraft%26aufirst%3DC.%26atitle%3DEarly%2520steps%2520in%2520autophagy%2520depend%2520on%2520direct%2520phosphorylation%2520of%2520Atg9%2520by%2520the%2520Atg1%2520kinase%26jtitle%3DMol.%2520Cell%26date%3D2014%26volume%3D53%26spage%3D471%26epage%3D483%26doi%3D10.1016%2Fj.molcel.2013.12.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Papinski, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kraft, C.</span></span> <span> </span><span class="NLM_article-title">Regulation of autophagy by signaling through the Atg1/ULK1 complex</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>428</i></span>,  <span class="NLM_fpage">1725</span>– <span class="NLM_lpage">1741</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2016.03.030</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1016%2Fj.jmb.2016.03.030" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=27059781" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BC28Xls1Ojtrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=428&publication_year=2016&pages=1725-1741&author=D.+Papinskiauthor=C.+Kraft&title=Regulation+of+autophagy+by+signaling+through+the+Atg1%2FULK1+complex&doi=10.1016%2Fj.jmb.2016.03.030"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of autophagy by signaling through the Atg1/ULK1 complex</span></div><div class="casAuthors">Papinski, Daniel; Kraft, Claudine</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">428</span>
        (<span class="NLM_cas:issue">9_Part_A</span>),
    <span class="NLM_cas:pages">1725-1741</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Autophagy is an intracellular degrdn. pathway highly conserved in eukaryotic species.  It is characterized by selective or bulk trafficking of cytosolic structures-ranging from single proteins to cell organelles, the vacuole or a lysosome, in which the autophagic cargo is degraded.  Autophagy fulfils a large set of roles, including nutrient mobilization in starvation conditions, clearance of protein aggregates, and host defense against intracellular pathogens.  Not surprisingly, autophagy has been linked to several human diseases, among them neurodegenerative disorders and cancer.  Autophagy is coordinated by the action of Atg1/ULK1 kinase, which is the target of several important stress signaling cascades.  Here, the authors discuss the available information on both upstream regulation and downstream effectors of Atg1/ULK1 kinase, with special focus on reported and proposed kinase substrates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo74qOboWXq2rVg90H21EOLACvtfcHk0ljsrdJx1ZAzdQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xls1Ojtrs%253D&md5=4583bf3c25d9da1154bfd1a78ce5a465</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2016.03.030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2016.03.030%26sid%3Dliteratum%253Aachs%26aulast%3DPapinski%26aufirst%3DD.%26aulast%3DKraft%26aufirst%3DC.%26atitle%3DRegulation%2520of%2520autophagy%2520by%2520signaling%2520through%2520the%2520Atg1%252FULK1%2520complex%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2016%26volume%3D428%26spage%3D1725%26epage%3D1741%26doi%3D10.1016%2Fj.jmb.2016.03.030" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Egan, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chun, M. G. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vamos, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rong, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lou, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raveendra-Panickar, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheffler, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teriete, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asara, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turk, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cosford, N. D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, R. J.</span></span> <span> </span><span class="NLM_article-title">Small molecule inhibition of the autophagy kinase ULK1 and identification of ULK1 substrates</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">285</span>– <span class="NLM_lpage">297</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2015.05.031</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1016%2Fj.molcel.2015.05.031" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=26118643" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtV2itbvM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2015&pages=285-297&author=D.+F.+Eganauthor=M.+G.+H.+Chunauthor=M.+Vamosauthor=H.+Zouauthor=J.+Rongauthor=C.+J.+Millerauthor=H.+J.+Louauthor=D.+Raveendra-Panickarauthor=C.-C.+Yangauthor=D.+J.+Shefflerauthor=P.+Terieteauthor=J.+M.+Asaraauthor=B.+E.+Turkauthor=N.+D.+P.+Cosfordauthor=R.+J.+Shaw&title=Small+molecule+inhibition+of+the+autophagy+kinase+ULK1+and+identification+of+ULK1+substrates&doi=10.1016%2Fj.molcel.2015.05.031"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Small Molecule Inhibition of the Autophagy Kinase ULK1 and Identification of ULK1 Substrates</span></div><div class="casAuthors">Egan, Daniel F.; Chun, Matthew G. H.; Vamos, Mitchell; Zou, Haixia; Rong, Juan; Miller, Chad J.; Lou, Hua Jane; Raveendra-Panickar, Dhanya; Yang, Chih-Cheng; Sheffler, Douglas J.; Teriete, Peter; Asara, John M.; Turk, Benjamin E.; Cosford, Nicholas D. P.; Shaw, Reuben J.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">285-297</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Many tumors become addicted to autophagy for survival, suggesting inhibition of autophagy as a potential broadly applicable cancer therapy.  ULK1/Atg1 is the only serine/threonine kinase in the core autophagy pathway and thus represents an excellent drug target.  Despite recent advances in the understanding of ULK1 activation by nutrient deprivation, how ULK1 promotes autophagy remains poorly understood.  Here, we screened degenerate peptide libraries to deduce the optimal ULK1 substrate motif and discovered 15 phosphorylation sites in core autophagy proteins that were verified as in vivo ULK1 targets.  We utilized these ULK1 substrates to perform a cell-based screen to identify and characterize a potent ULK1 small mol. inhibitor.  The compd. SBI-0206965 is a highly selective ULK1 kinase inhibitor in vitro and suppressed ULK1-mediated phosphorylation events in cells, regulating autophagy and cell survival.  SBI-0206965 greatly synergized with mechanistic target of rapamycin (mTOR) inhibitors to kill tumor cells, providing a strong rationale for their combined use in the clinic.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdE4ITU-IqIbVg90H21EOLACvtfcHk0ljsrdJx1ZAzdQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtV2itbvM&md5=02ede4bf37aa3aca8360d518bec5525c</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2015.05.031&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2015.05.031%26sid%3Dliteratum%253Aachs%26aulast%3DEgan%26aufirst%3DD.%2BF.%26aulast%3DChun%26aufirst%3DM.%2BG.%2BH.%26aulast%3DVamos%26aufirst%3DM.%26aulast%3DZou%26aufirst%3DH.%26aulast%3DRong%26aufirst%3DJ.%26aulast%3DMiller%26aufirst%3DC.%2BJ.%26aulast%3DLou%26aufirst%3DH.%2BJ.%26aulast%3DRaveendra-Panickar%26aufirst%3DD.%26aulast%3DYang%26aufirst%3DC.-C.%26aulast%3DSheffler%26aufirst%3DD.%2BJ.%26aulast%3DTeriete%26aufirst%3DP.%26aulast%3DAsara%26aufirst%3DJ.%2BM.%26aulast%3DTurk%26aufirst%3DB.%2BE.%26aulast%3DCosford%26aufirst%3DN.%2BD.%2BP.%26aulast%3DShaw%26aufirst%3DR.%2BJ.%26atitle%3DSmall%2520molecule%2520inhibition%2520of%2520the%2520autophagy%2520kinase%2520ULK1%2520and%2520identification%2520of%2520ULK1%2520substrates%26jtitle%3DMol.%2520Cell%26date%3D2015%26volume%3D59%26spage%3D285%26epage%3D297%26doi%3D10.1016%2Fj.molcel.2015.05.031" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Alsaadi, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Losier, T. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jackson, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rubinsztein, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Russell, R. C.</span></span> <span> </span><span class="NLM_article-title">ULK1-mediated phosphorylation of ATG16L1 promotes xenophagy, but destabilizes the ATG16L1 Crohn’s mutant</span>. <i>EMBO Rep.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>20</i></span>, <span class="NLM_elocation-id">e46885</span> <span class="refDoi"> DOI: 10.15252/embr.201846885</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.15252%2Fembr.201846885" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=31267703" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2019&author=R.+M.+Alsaadiauthor=T.+T.+Losierauthor=W.+Tianauthor=A.+Jacksonauthor=Z.+Guoauthor=D.+C.+Rubinszteinauthor=R.+C.+Russell&title=ULK1-mediated+phosphorylation+of+ATG16L1+promotes+xenophagy%2C+but+destabilizes+the+ATG16L1+Crohn%E2%80%99s+mutant&doi=10.15252%2Fembr.201846885"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.15252%2Fembr.201846885&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.15252%252Fembr.201846885%26sid%3Dliteratum%253Aachs%26aulast%3DAlsaadi%26aufirst%3DR.%2BM.%26aulast%3DLosier%26aufirst%3DT.%2BT.%26aulast%3DTian%26aufirst%3DW.%26aulast%3DJackson%26aufirst%3DA.%26aulast%3DGuo%26aufirst%3DZ.%26aulast%3DRubinsztein%26aufirst%3DD.%2BC.%26aulast%3DRussell%26aufirst%3DR.%2BC.%26atitle%3DULK1-mediated%2520phosphorylation%2520of%2520ATG16L1%2520promotes%2520xenophagy%252C%2520but%2520destabilizes%2520the%2520ATG16L1%2520Crohn%25E2%2580%2599s%2520mutant%26jtitle%3DEMBO%2520Rep.%26date%3D2019%26volume%3D20%26doi%3D10.15252%2Fembr.201846885" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Saha, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Panigrahi, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patil, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhutia, S. K.</span></span> <span> </span><span class="NLM_article-title">Autophagy in health and disease: A comprehensive review</span>. <i>Biomed. Pharmacother.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>104</i></span>,  <span class="NLM_fpage">485</span>– <span class="NLM_lpage">495</span>, <span class="refDoi"> DOI: 10.1016/j.biopha.2018.05.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1016%2Fj.biopha.2018.05.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=29800913" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BC1cXpvFKgtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=104&publication_year=2018&pages=485-495&author=S.+Sahaauthor=D.+P.+Panigrahiauthor=S.+Patilauthor=S.+K.+Bhutia&title=Autophagy+in+health+and+disease%3A+A+comprehensive+review&doi=10.1016%2Fj.biopha.2018.05.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Autophagy in health and disease: A comprehensive review</span></div><div class="casAuthors">Saha, Sarbari; Panigrahi, Debasna P.; Patil, Shankargouda; Bhutia, Sujit K.</div><div class="citationInfo"><span class="NLM_cas:title">Biomedicine & Pharmacotherapy</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">104</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">485-495</span>CODEN:
                <span class="NLM_cas:coden">BIPHEX</span>;
        ISSN:<span class="NLM_cas:issn">0753-3322</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Autophagy, a conserved catabolic process, plays an immensely significant role in a variety of diseases.  However, whether it imparts a protective function in diseases remains debatable.  During aging, autophagy gradually subsides, manifested by the reduced formation of autophagic vacuoles and improper fusion of these vacuoles with the lysosomes.  Similarly, in neurodegenerative disorders, accumulation of tau and synuclein proteins has been attributed to the decline in the autophagic removal of proteins.  Equivalently, lysosomal disorders show an impairment of the autophagic process leading to the accumulation of lipid mols. within lysosomes.  On the other hand, activation of the autophagic pathway has also proved beneficial in evading various foreign pathogens, thereby contributing to the innate immunity.  In the context of cancer, autophagy has shown to play a puzzling role where it serves as a tumor suppressor during initial stages but later protects the tumor cells from the immune system defense mechanisms.  Similarly, muscular and heart disorders have been shown to be pos. and neg. regulated by autophagy, resp.  In the present review, we, therefore, present a comprehensive review on the role of autophagy in various diseases and their corresponding outcomes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqo7G7CF_NtgbVg90H21EOLACvtfcHk0liiILnPIyf_ew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXpvFKgtrY%253D&md5=ade9f524674ce9da482dde10abc3eb89</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.biopha.2018.05.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.biopha.2018.05.007%26sid%3Dliteratum%253Aachs%26aulast%3DSaha%26aufirst%3DS.%26aulast%3DPanigrahi%26aufirst%3DD.%2BP.%26aulast%3DPatil%26aufirst%3DS.%26aulast%3DBhutia%26aufirst%3DS.%2BK.%26atitle%3DAutophagy%2520in%2520health%2520and%2520disease%253A%2520A%2520comprehensive%2520review%26jtitle%3DBiomed.%2520Pharmacother.%26date%3D2018%26volume%3D104%26spage%3D485%26epage%3D495%26doi%3D10.1016%2Fj.biopha.2018.05.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Galluzzi, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bravo-San Pedro, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levine, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kroemer, G.</span></span> <span> </span><span class="NLM_article-title">Pharmacological modulation of autophagy: therapeutic potential and persisting obstacles</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">487</span>, <span class="refDoi"> DOI: 10.1038/nrd.2017.22</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1038%2Fnrd.2017.22" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=28529316" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BC2sXotVKntr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=487&author=L.+Galluzziauthor=J.+M.+Bravo-San+Pedroauthor=B.+Levineauthor=D.+R.+Greenauthor=G.+Kroemer&title=Pharmacological+modulation+of+autophagy%3A+therapeutic+potential+and+persisting+obstacles&doi=10.1038%2Fnrd.2017.22"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacological modulation of autophagy: therapeutic potential and persisting obstacles</span></div><div class="casAuthors">Galluzzi, Lorenzo; Bravo-San Pedro, Jose Manuel; Levine, Beth; Green, Douglas R.; Kroemer, Guido</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">487-511</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Autophagy is central to the maintenance of organismal homeostasis in both physiol. and pathol. situations.  Accordingly, alterations in autophagy have been linked to clin. relevant conditions as diverse as cancer, neurodegeneration and cardiac disorders.  Throughout the past decade, autophagy has attracted considerable attention as a target for the development of novel therapeutics.  However, such efforts have not yet generated clin. viable interventions.  In this review, we discuss the therapeutic potential of autophagy modulators, analyze the obstacles that have limited their development and propose strategies that may unlock the full therapeutic potential of autophagy modulation in the clinic.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr1YAuK4nNrTbVg90H21EOLACvtfcHk0liiILnPIyf_ew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXotVKntr4%253D&md5=0cf36178d5fd9b4d2493649d44aa1051</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2017.22&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2017.22%26sid%3Dliteratum%253Aachs%26aulast%3DGalluzzi%26aufirst%3DL.%26aulast%3DBravo-San%2BPedro%26aufirst%3DJ.%2BM.%26aulast%3DLevine%26aufirst%3DB.%26aulast%3DGreen%26aufirst%3DD.%2BR.%26aulast%3DKroemer%26aufirst%3DG.%26atitle%3DPharmacological%2520modulation%2520of%2520autophagy%253A%2520therapeutic%2520potential%2520and%2520persisting%2520obstacles%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2017%26volume%3D16%26spage%3D487%26doi%3D10.1038%2Fnrd.2017.22" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rao, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tortola, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perlot, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wirnsberger, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Novatchkova, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nitsch, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sykacek, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frank, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schramek, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Komnenovic, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sigl, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aumayr, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmauss, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fellner, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Handschuh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glösmann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pasierbek, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlederer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Resch, G. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Popper, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kenner, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kroemer, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Penninger, J. M.</span></span> <span> </span><span class="NLM_article-title">A dual role for autophagy in a murine model of lung cancer</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">3056</span>, <span class="refDoi"> DOI: 10.1038/ncomms4056</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1038%2Fncomms4056" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=24445999" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A280%3ADC%252BC2cznt1aguw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=3056&author=S.+Raoauthor=L.+Tortolaauthor=T.+Perlotauthor=G.+Wirnsbergerauthor=M.+Novatchkovaauthor=R.+Nitschauthor=P.+Sykacekauthor=L.+Frankauthor=D.+Schramekauthor=V.+Komnenovicauthor=V.+Siglauthor=K.+Aumayrauthor=G.+Schmaussauthor=N.+Fellnerauthor=S.+Handschuhauthor=M.+Gl%C3%B6smannauthor=P.+Pasierbekauthor=M.+Schledererauthor=G.+P.+Reschauthor=Y.+Maauthor=H.+Yangauthor=H.+Popperauthor=L.+Kennerauthor=G.+Kroemerauthor=J.+M.+Penninger&title=A+dual+role+for+autophagy+in+a+murine+model+of+lung+cancer&doi=10.1038%2Fncomms4056"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">A dual role for autophagy in a murine model of lung cancer</span></div><div class="casAuthors">Rao Shuan; Tortola Luigi; Perlot Thomas; Wirnsberger Gerald; Novatchkova Maria; Nitsch Roberto; Schramek Daniel; Komnenovic Vukoslav; Sigl Verena; Aumayr Karin; Schmauss Gerald; Pasierbek Pawel; Penninger Josef M; Sykacek Peter; Frank Lukas; Fellner Nicole; Resch Guenter P; Handschuh Stephan; Glosmann Martin; Schlederer Michaela; Kenner Lukas; Ma Yuting; Yang Heng; Popper Helmuth; Kroemer Guido</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">3056</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Autophagy is a mechanism by which starving cells can control their energy requirements and metabolic states, thus facilitating the survival of cells in stressful environments, in particular in the pathogenesis of cancer.  Here we report that tissue-specific inactivation of Atg5, essential for the formation of autophagosomes, markedly impairs the progression of KRas(G12D)-driven lung cancer, resulting in a significant survival advantage of tumour-bearing mice.  Autophagy-defective lung cancers exhibit impaired mitochondrial energy homoeostasis, oxidative stress and a constitutively active DNA damage response.  Genetic deletion of the tumour suppressor p53 reinstates cancer progression of autophagy-deficient tumours.  Although there is improved survival, the onset of Atg5-mutant KRas(G12D)-driven lung tumours is markedly accelerated.  Mechanistically, increased oncogenesis maps to regulatory T cells.  These results demonstrate that, in KRas(G12D)-driven lung cancer, Atg5-regulated autophagy accelerates tumour progression; however, autophagy also represses early oncogenesis, suggesting a link between deregulated autophagy and regulatory T cell controlled anticancer immunity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRd1A4UrEIWovEeosoOnpOLfW6udTcc2eaGqaImlfRdaLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2cznt1aguw%253D%253D&md5=e3b8cdd4aee24a82da1ad94b48886c02</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2Fncomms4056&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms4056%26sid%3Dliteratum%253Aachs%26aulast%3DRao%26aufirst%3DS.%26aulast%3DTortola%26aufirst%3DL.%26aulast%3DPerlot%26aufirst%3DT.%26aulast%3DWirnsberger%26aufirst%3DG.%26aulast%3DNovatchkova%26aufirst%3DM.%26aulast%3DNitsch%26aufirst%3DR.%26aulast%3DSykacek%26aufirst%3DP.%26aulast%3DFrank%26aufirst%3DL.%26aulast%3DSchramek%26aufirst%3DD.%26aulast%3DKomnenovic%26aufirst%3DV.%26aulast%3DSigl%26aufirst%3DV.%26aulast%3DAumayr%26aufirst%3DK.%26aulast%3DSchmauss%26aufirst%3DG.%26aulast%3DFellner%26aufirst%3DN.%26aulast%3DHandschuh%26aufirst%3DS.%26aulast%3DGl%25C3%25B6smann%26aufirst%3DM.%26aulast%3DPasierbek%26aufirst%3DP.%26aulast%3DSchlederer%26aufirst%3DM.%26aulast%3DResch%26aufirst%3DG.%2BP.%26aulast%3DMa%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DPopper%26aufirst%3DH.%26aulast%3DKenner%26aufirst%3DL.%26aulast%3DKroemer%26aufirst%3DG.%26aulast%3DPenninger%26aufirst%3DJ.%2BM.%26atitle%3DA%2520dual%2520role%2520for%2520autophagy%2520in%2520a%2520murine%2520model%2520of%2520lung%2520cancer%26jtitle%3DNat.%2520Commun.%26date%3D2014%26volume%3D5%26spage%3D3056%26doi%3D10.1038%2Fncomms4056" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rosenfeldt, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Prey, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morton, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nixon, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacKay, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mrowinska, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Au, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rai, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ridgway, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, K. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gottlieb, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sansom, O. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryan, K. M.</span></span> <span> </span><span class="NLM_article-title">p53 status determines the role of autophagy in pancreatic tumour development</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>504</i></span>,  <span class="NLM_fpage">296</span>, <span class="refDoi"> DOI: 10.1038/nature12865</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1038%2Fnature12865" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=24305049" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFart7rL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=504&publication_year=2013&pages=296&author=M.+T.+Rosenfeldtauthor=J.+O%E2%80%99Preyauthor=J.+P.+Mortonauthor=C.+Nixonauthor=G.+MacKayauthor=A.+Mrowinskaauthor=A.+Auauthor=T.+S.+Raiauthor=L.+Zhengauthor=R.+Ridgwayauthor=P.+D.+Adamsauthor=K.+I.+Andersonauthor=E.+Gottliebauthor=O.+W.+Sansomauthor=K.+M.+Ryan&title=p53+status+determines+the+role+of+autophagy+in+pancreatic+tumour+development&doi=10.1038%2Fnature12865"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">p53 status determines the role of autophagy in pancreatic tumor development</span></div><div class="casAuthors">Rosenfeldt, Mathias T.; O'Prey, Jim; Morton, Jennifer P.; Nixon, Colin; MacKay, Gillian; Mrowinska, Agata; Au, Amy; Rai, Taranjit Singh; Zheng, Liang; Ridgway, Rachel; Adams, Peter D.; Anderson, Kurt I.; Gottlieb, Eyal; Sansom, Owen J.; Ryan, Kevin M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">504</span>
        (<span class="NLM_cas:issue">7479</span>),
    <span class="NLM_cas:pages">296-300</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Macroautophagy (hereafter referred to as autophagy) is a process in which organelles termed autophagosomes deliver cytoplasmic constituents to lysosomes for degrdn.  Autophagy has a major role in cellular homeostasis and has been implicated in various forms of human disease.  The role of autophagy in cancer seems to be complex, with reports indicating both pro-tumorigenic and tumor-suppressive roles.  Here we show, in a humanized genetically-modified mouse model of pancreatic ductal adenocarcinoma (PDAC), that autophagy's role in tumor development is intrinsically connected to the status of the tumor suppressor p53.  Mice with pancreases contg. an activated oncogenic allele of Kras (also called Ki-Ras)-the most common mutational event in PDAC-develop a small no. of pre-cancerous lesions that stochastically develop into PDAC over time.  However, mice also lacking the essential autophagy genes Atg5 or Atg7 accumulate low-grade, pre-malignant pancreatic intraepithelial neoplasia lesions, but progression to high-grade pancreatic intraepithelial neoplasias and PDAC is blocked.  In marked contrast, in mice contg. oncogenic Kras and lacking p53, loss of autophagy no longer blocks tumor progression, but actually accelerates tumor onset, with metabolic anal. revealing enhanced glucose uptake and enrichment of anabolic pathways, which can fuel tumor growth.  These findings provide considerable insight into the role of autophagy in cancer and have important implications for autophagy inhibition in cancer therapy.  In this regard, we also show that treatment of mice with the autophagy inhibitor hydroxychloroquine, which is currently being used in several clin. trials, significantly accelerates tumor formation in mice contg. oncogenic Kras but lacking p53.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9_zf9BrV1ErVg90H21EOLACvtfcHk0liGkPjqI0CU-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFart7rL&md5=86c9d7ec9ea35e1f65fa4ad660d377fe</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1038%2Fnature12865&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature12865%26sid%3Dliteratum%253Aachs%26aulast%3DRosenfeldt%26aufirst%3DM.%2BT.%26aulast%3DO%25E2%2580%2599Prey%26aufirst%3DJ.%26aulast%3DMorton%26aufirst%3DJ.%2BP.%26aulast%3DNixon%26aufirst%3DC.%26aulast%3DMacKay%26aufirst%3DG.%26aulast%3DMrowinska%26aufirst%3DA.%26aulast%3DAu%26aufirst%3DA.%26aulast%3DRai%26aufirst%3DT.%2BS.%26aulast%3DZheng%26aufirst%3DL.%26aulast%3DRidgway%26aufirst%3DR.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DAnderson%26aufirst%3DK.%2BI.%26aulast%3DGottlieb%26aufirst%3DE.%26aulast%3DSansom%26aufirst%3DO.%2BW.%26aulast%3DRyan%26aufirst%3DK.%2BM.%26atitle%3Dp53%2520status%2520determines%2520the%2520role%2520of%2520autophagy%2520in%2520pancreatic%2520tumour%2520development%26jtitle%3DNature%26date%3D2013%26volume%3D504%26spage%3D296%26doi%3D10.1038%2Fnature12865" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">White, E.</span></span> <span> </span><span class="NLM_article-title">The role for autophagy in cancer</span>. <i>J. Clin. Invest.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>125</i></span>,  <span class="NLM_fpage">42</span>– <span class="NLM_lpage">46</span>, <span class="refDoi"> DOI: 10.1172/JCI73941</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1172%2FJCI73941" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=25654549" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A280%3ADC%252BC2Mrjt1ensQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=125&publication_year=2015&pages=42-46&author=E.+White&title=The+role+for+autophagy+in+cancer&doi=10.1172%2FJCI73941"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">The role for autophagy in cancer</span></div><div class="casAuthors">White Eileen</div><div class="citationInfo"><span class="NLM_cas:title">The Journal of clinical investigation</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">125</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">42-6</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Autophagy is a survival-promoting pathway that captures, degrades, and recycles intracellular proteins and organelles in lysosomes.  Autophagy preserves organelle function, prevents the toxic buildup of cellular waste products, and provides substrates to sustain metabolism in starvation.  Although in some contexts autophagy suppresses tumorigenesis, in most contexts autophagy facilitates tumorigenesis.  Cancers can upregulate autophagy to survive microenvironmental stress and to increase growth and aggressiveness.  Mechanisms by which autophagy promotes cancer include suppressing induction of the p53 tumor suppressor protein and maintaining metabolic function of mitochondria.  Efforts to inhibit autophagy to improve cancer therapy have thereby attracted great interest.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQsFLRpnBdsgcSzjBFCN0BJfW6udTcc2ebCrGAiHvTXQrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2Mrjt1ensQ%253D%253D&md5=9c662eaf1f0bfe280982046d15e316c1</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1172%2FJCI73941&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1172%252FJCI73941%26sid%3Dliteratum%253Aachs%26aulast%3DWhite%26aufirst%3DE.%26atitle%3DThe%2520role%2520for%2520autophagy%2520in%2520cancer%26jtitle%3DJ.%2520Clin.%2520Invest.%26date%3D2015%26volume%3D125%26spage%3D42%26epage%3D46%26doi%3D10.1172%2FJCI73941" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Guo, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xia, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, E.</span></span> <span> </span><span class="NLM_article-title">Autophagy-mediated tumor promotion</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>155</i></span>,  <span class="NLM_fpage">1216</span>– <span class="NLM_lpage">1219</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2013.11.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1016%2Fj.cell.2013.11.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=24315093" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFyksLnO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=155&publication_year=2013&pages=1216-1219&author=J.+Y.+Guoauthor=B.+Xiaauthor=E.+White&title=Autophagy-mediated+tumor+promotion&doi=10.1016%2Fj.cell.2013.11.019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Autophagy-mediated tumor promotion</span></div><div class="casAuthors">Guo, Jessie Yanxiang; Xia, Bing; White, Eileen</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">155</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1216-1219</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Mouse models for cancer are revealing novel cancer-promoting roles for autophagy.  Autophagy promotes tumor growth by suppressing the p53 response, maintaining mitochondrial function, sustaining metabolic homeostasis and survival in stress, and preventing diversion of tumor progression to benign oncocytomas.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphypvWit2Wg7Vg90H21EOLACvtfcHk0liGkPjqI0CU-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFyksLnO&md5=05e06234fc4ebb7727e37e21b59dbd21</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2013.11.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2013.11.019%26sid%3Dliteratum%253Aachs%26aulast%3DGuo%26aufirst%3DJ.%2BY.%26aulast%3DXia%26aufirst%3DB.%26aulast%3DWhite%26aufirst%3DE.%26atitle%3DAutophagy-mediated%2520tumor%2520promotion%26jtitle%3DCell%26date%3D2013%26volume%3D155%26spage%3D1216%26epage%3D1219%26doi%3D10.1016%2Fj.cell.2013.11.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">O’Donovan, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Sullivan, G. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKenna, S. L.</span></span> <span> </span><span class="NLM_article-title">Induction of autophagy by drug-resistant esophageal cancer cells promotes their survival and recovery following treatment with chemotherapeutics</span>. <i>Autophagy</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">509</span>– <span class="NLM_lpage">524</span>, <span class="refDoi"> DOI: 10.4161/auto.7.5.15066</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.4161%2Fauto.7.5.15066" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=21325880" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BC38XitF2jtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=509-524&author=T.+R.+O%E2%80%99Donovanauthor=G.+C.+O%E2%80%99Sullivanauthor=S.+L.+McKenna&title=Induction+of+autophagy+by+drug-resistant+esophageal+cancer+cells+promotes+their+survival+and+recovery+following+treatment+with+chemotherapeutics&doi=10.4161%2Fauto.7.5.15066"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Induction of autophagy by drug-resistant esophageal cancer cells promotes their survival and recovery following treatment with chemotherapeutics</span></div><div class="casAuthors">O'Donovan, Tracey R.; O'Sullivan, Gerald C.; McKenna, Sharon L.</div><div class="citationInfo"><span class="NLM_cas:title">Autophagy</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">509-524</span>CODEN:
                <span class="NLM_cas:coden">AUTOC9</span>;
        ISSN:<span class="NLM_cas:issn">1554-8627</span>.
    
            (<span class="NLM_cas:orgname">Landes Bioscience</span>)
        </div><div class="casAbstract">We investigated the cell-death mechanisms induced in esophageal cancer cells in response to the chemotherapeutic drugs, 5-fluorouracil (5-FU) and cisplatin.  Chemosensitive cell lines exhibited apoptosis whereas chemoresistant populations exhibited autophagy and a morphol. resembling type II programmed cell death (PCD).  Cell populations that respond with autophagy are more resistant and will recover following withdrawal of the chemotherapeutic agents.  Specific inhibition of early autophagy induction with siRNA targeted to Beclin 1 and ATG7 significantly enhanced the effect of 5-FU and reduced the recovery of drug-treated cells.  Pharmacol. inhibitors of autophagy were evaluated for their ability to improve chemotherapeutic effect.  The PtdIns 3-kinase inhibitor 3-methyladenine did not enhance the cytotoxicity of 5-FU.  Disruption of lysosomal activity with bafilomycin A1 or chloroquine caused extensive vesicular accumulation but did not improve chemotherapeutic effect.  These observations suggest that an autophagic response to chemotherapy is a survival mechanism that promotes chemoresistance and recovery and that selective inhibition of autophagy regulators has the potential to improve chemotherapeutic regimes.  Currently available indirect inhibitors of autophagy are, however, ineffective at modulating chemosensitivity in these esophageal cancer cell lines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqX4Hfa6svG1LVg90H21EOLACvtfcHk0liGkPjqI0CU-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XitF2jtw%253D%253D&md5=bbf8ffa02c8fa302787a7ced6f40175a</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.4161%2Fauto.7.5.15066&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252Fauto.7.5.15066%26sid%3Dliteratum%253Aachs%26aulast%3DO%25E2%2580%2599Donovan%26aufirst%3DT.%2BR.%26aulast%3DO%25E2%2580%2599Sullivan%26aufirst%3DG.%2BC.%26aulast%3DMcKenna%26aufirst%3DS.%2BL.%26atitle%3DInduction%2520of%2520autophagy%2520by%2520drug-resistant%2520esophageal%2520cancer%2520cells%2520promotes%2520their%2520survival%2520and%2520recovery%2520following%2520treatment%2520with%2520chemotherapeutics%26jtitle%3DAutophagy%26date%3D2011%26volume%3D7%26spage%3D509%26epage%3D524%26doi%3D10.4161%2Fauto.7.5.15066" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rehman, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span> <span> </span><span class="NLM_article-title">Autophagy is a therapeutic target in anticancer drug resistance</span>. <i>Biochim. Biophys. Acta, (BBA)-Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>1806</i></span>,  <span class="NLM_fpage">220</span>– <span class="NLM_lpage">229</span>, <span class="refDoi"> DOI: 10.1016/j.bbcan.2010.07.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1016%2Fj.bbcan.2010.07.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=20637264" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVyhurrO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1806&publication_year=2010&pages=220-229&author=S.+Chenauthor=S.+K.+Rehmanauthor=W.+Zhangauthor=A.+Wenauthor=L.+Yaoauthor=J.+Zhang&title=Autophagy+is+a+therapeutic+target+in+anticancer+drug+resistance&doi=10.1016%2Fj.bbcan.2010.07.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Autophagy is a therapeutic target in anticancer drug resistance</span></div><div class="casAuthors">Chen, Suning; Rehman, Sumaiyah K.; Zhang, Wei; Wen, Aidong; Yao, Libo; Zhang, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Reviews on Cancer</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">1806</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">220-229</span>CODEN:
                <span class="NLM_cas:coden">BBACEU</span>;
        ISSN:<span class="NLM_cas:issn">0304-419X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Autophagy is a type of cellular catabolic degrdn. response to nutrient starvation or metabolic stress.  The main function of autophagy is to maintain intracellular metabolic homeostasis through degrdn. of unfolded or aggregated proteins and organelles.  Although autophagic regulation is a complicated process, solid evidence demonstrates that the PI3K-Akt-mTOR, LKB1-AMPK-mTOR and p53 are the main upstream regulators of the autophagic pathway.  Currently, there is a bulk of data indicating the important function of autophagy in cancer.  It is noteworthy that autophagy facilitates the cancer cells' resistance to chemotherapy and radiation treatment.  The abrogation of autophagy potentiates the re-sensitization of therapeutic resistant cancer cells to the anticancer treatment via autophagy inhibitors, such as 3-MA, CQ and BA, or knockdown of the autophagy related mols.  In this review, we summarize the accumulation of evidence for autophagy's involvement in mediating resistance of cancer cells to anticancer therapy and suggest that autophagy might be a potential therapeutic target in anticancer drug resistance in the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojU_TzkfkqsrVg90H21EOLACvtfcHk0ljMa5PcWaGmEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsVyhurrO&md5=c3783db09a386f8dbe14cc929d0cf7d9</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.bbcan.2010.07.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbcan.2010.07.003%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DS.%26aulast%3DRehman%26aufirst%3DS.%2BK.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DWen%26aufirst%3DA.%26aulast%3DYao%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DJ.%26atitle%3DAutophagy%2520is%2520a%2520therapeutic%2520target%2520in%2520anticancer%2520drug%2520resistance%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520%2528BBA%2529-Rev.%2520Cancer%26date%3D2010%26volume%3D1806%26spage%3D220%26epage%3D229%26doi%3D10.1016%2Fj.bbcan.2010.07.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sui, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kong, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lou, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, H.</span></span> <span> </span><span class="NLM_article-title">Autophagy and chemotherapy resistance: a promising therapeutic target for cancer treatment</span>. <i>Cell death dis.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>4</i></span>, <span class="NLM_elocation-id">e838</span> <span class="refDoi"> DOI: 10.1038/cddis.2013.350</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1038%2Fcddis.2013.350" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=24113172" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1SksbjL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&author=X.+Suiauthor=R.+Chenauthor=Z.+Wangauthor=Z.+Huangauthor=N.+Kongauthor=M.+Zhangauthor=W.+Hanauthor=F.+Louauthor=J.+Yangauthor=Q.+Zhangauthor=X.+Wangauthor=C.+Heauthor=H.+Pan&title=Autophagy+and+chemotherapy+resistance%3A+a+promising+therapeutic+target+for+cancer+treatment&doi=10.1038%2Fcddis.2013.350"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Autophagy and chemotherapy resistance: a promising therapeutic target for cancer treatment</span></div><div class="casAuthors">Sui, X.; Chen, R.; Wang, Z.; Huang, Z.; Kong, N.; Zhang, M.; Han, W.; Lou, F.; Yang, J.; Zhang, Q.; Wang, X.; He, C.; Pan, H.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Death & Disease</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">Oct.</span>),
    <span class="NLM_cas:pages">e838</span>CODEN:
                <span class="NLM_cas:coden">CDDEA4</span>;
        ISSN:<span class="NLM_cas:issn">2041-4889</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Induction of cell death and inhibition of cell survival are the main principles of cancer therapy.  Resistance to chemotherapeutic agents is a major problem in oncol., which limits the effectiveness of anticancer drugs.  A variety of factors contribute to drug resistance, including host factors, specific genetic or epigenetic alterations in the cancer cells and so on.  Although various mechanisms by which cancer cells become resistant to anticancer drugs in the microenvironment have been well elucidated, how to circumvent this resistance to improve anticancer efficacy remains to be defined.  Autophagy, an important homeostatic cellular recycling mechanism, is now emerging as a crucial player in response to metabolic and therapeutic stresses, which attempts to maintain/restore metabolic homeostasis through the catabolic lysis of excessive or unnecessary proteins and injured or aged organelles.  Recently, several studies have shown that autophagy constitutes a potential target for cancer therapy and the induction of autophagy in response to therapeutics can be viewed as having a prodeath or a prosurvival role, which contributes to the anticancer efficacy of these drugs as well as drug resistance.  Thus, understanding the novel function of autophagy may allow us to develop a promising therapeutic strategy to enhance the effects of chemotherapy and improve clin. outcomes in the treatment of cancer patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoe48kz8t23vLVg90H21EOLACvtfcHk0ljMa5PcWaGmEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1SksbjL&md5=d92762eb7c4bbc041161c1656e6e17b9</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Fcddis.2013.350&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fcddis.2013.350%26sid%3Dliteratum%253Aachs%26aulast%3DSui%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DR.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DHuang%26aufirst%3DZ.%26aulast%3DKong%26aufirst%3DN.%26aulast%3DZhang%26aufirst%3DM.%26aulast%3DHan%26aufirst%3DW.%26aulast%3DLou%26aufirst%3DF.%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DHe%26aufirst%3DC.%26aulast%3DPan%26aufirst%3DH.%26atitle%3DAutophagy%2520and%2520chemotherapy%2520resistance%253A%2520a%2520promising%2520therapeutic%2520target%2520for%2520cancer%2520treatment%26jtitle%3DCell%2520death%2520dis.%26date%3D2013%26volume%3D4%26doi%3D10.1038%2Fcddis.2013.350" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Holohan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Schaeybroeck, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Longley, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnston, P. G.</span></span> <span> </span><span class="NLM_article-title">Cancer drug resistance: an evolving paradigm</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">714</span>, <span class="refDoi"> DOI: 10.1038/nrc3599</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1038%2Fnrc3599" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=24060863" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVyqurnE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2013&pages=714&author=C.+Holohanauthor=S.+Van+Schaeybroeckauthor=D.+B.+Longleyauthor=P.+G.+Johnston&title=Cancer+drug+resistance%3A+an+evolving+paradigm&doi=10.1038%2Fnrc3599"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Cancer drug resistance: an evolving paradigm</span></div><div class="casAuthors">Holohan, Caitriona; Van Schaeybroeck, Sandra; Longley, Daniel B.; Johnston, Patrick G.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">714-726</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Resistance to chemotherapy and molecularly targeted therapies is a major problem facing current cancer research.  The mechanisms of resistance to 'classical' cytotoxic chemotherapeutics and to therapies that are designed to be selective for specific mol. targets share many features, such as alterations in the drug target, activation of prosurvival pathways and ineffective induction of cell death.  With the increasing arsenal of anticancer agents, improving preclin. models and the advent of powerful high-throughput screening techniques, there are now unprecedented opportunities to understand and overcome drug resistance through the clin. assessment of rational therapeutic drug combinations and the use of predictive biomarkers to enable patient stratification.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqrykBEUSgvv7Vg90H21EOLACvtfcHk0ljMa5PcWaGmEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVyqurnE&md5=b497a823bbd968904ce1591b918d0679</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1038%2Fnrc3599&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc3599%26sid%3Dliteratum%253Aachs%26aulast%3DHolohan%26aufirst%3DC.%26aulast%3DVan%2BSchaeybroeck%26aufirst%3DS.%26aulast%3DLongley%26aufirst%3DD.%2BB.%26aulast%3DJohnston%26aufirst%3DP.%2BG.%26atitle%3DCancer%2520drug%2520resistance%253A%2520an%2520evolving%2520paradigm%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2013%26volume%3D13%26spage%3D714%26doi%3D10.1038%2Fnrc3599" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Limpert, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lambert, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bakas, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bata, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brun, S. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cosford, N. D. P.</span></span> <span> </span><span class="NLM_article-title">Autophagy in cancer: regulation by small molecules</span>. <i>Trends Pharmacol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">1021</span>– <span class="NLM_lpage">1032</span>, <span class="refDoi"> DOI: 10.1016/j.tips.2018.10.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1016%2Fj.tips.2018.10.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=30454769" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvFejt7jP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2018&pages=1021-1032&author=A.+S.+Limpertauthor=L.+J.+Lambertauthor=N.+A.+Bakasauthor=N.+Bataauthor=S.+N.+Brunauthor=R.+J.+Shawauthor=N.+D.+P.+Cosford&title=Autophagy+in+cancer%3A+regulation+by+small+molecules&doi=10.1016%2Fj.tips.2018.10.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Autophagy in Cancer: Regulation by Small Molecules</span></div><div class="casAuthors">Limpert, Allison S.; Lambert, Lester J.; Bakas, Nicole A.; Bata, Nicole; Brun, Sonja N.; Shaw, Reuben J.; Cosford, Nicholas D. P.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Pharmacological Sciences</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1021-1032</span>CODEN:
                <span class="NLM_cas:coden">TPHSDY</span>;
        ISSN:<span class="NLM_cas:issn">0165-6147</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  During times of stress, autophagy is a cellular process that enables cells to reclaim damaged components by a controlled recycling pathway.  This mechanism for cellular catabolism is dysregulated in cancer, with evidence indicating that cancer cells rely on autophagy in the hypoxic and nutrient-poor microenvironment of solid tumors.  Mounting evidence suggests that autophagy has a role in the resistance of tumors to std.-of-care (SOC) therapies.  Therefore, there is significant interest in the discovery of small mols. that can safely modulate autophagy.  In this review, we describe recent advances in the identification of new pharmacol. compds. that modulate autophagy, with a focus on their mode of action, value as probe compds., and validation as potential therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgGylHAF7HCLVg90H21EOLACvtfcHk0liaXL5lCO1zTw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvFejt7jP&md5=4f0d411699ded0d57a3211015fccc83d</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fj.tips.2018.10.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tips.2018.10.004%26sid%3Dliteratum%253Aachs%26aulast%3DLimpert%26aufirst%3DA.%2BS.%26aulast%3DLambert%26aufirst%3DL.%2BJ.%26aulast%3DBakas%26aufirst%3DN.%2BA.%26aulast%3DBata%26aufirst%3DN.%26aulast%3DBrun%26aufirst%3DS.%2BN.%26aulast%3DShaw%26aufirst%3DR.%2BJ.%26aulast%3DCosford%26aufirst%3DN.%2BD.%2BP.%26atitle%3DAutophagy%2520in%2520cancer%253A%2520regulation%2520by%2520small%2520molecules%26jtitle%3DTrends%2520Pharmacol.%2520Sci.%26date%3D2018%26volume%3D39%26spage%3D1021%26epage%3D1032%26doi%3D10.1016%2Fj.tips.2018.10.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Maycotte, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gearheart, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barnard, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aryal, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levy, J. M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fosmire, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morgan, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porter, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gustafson, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thorburn, A.</span></span> <span> </span><span class="NLM_article-title">STAT3-mediated autophagy dependence identifies subtypes of breast cancer where autophagy inhibition can be efficacious</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>74</i></span>,  <span class="NLM_fpage">2579</span>– <span class="NLM_lpage">2590</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-13-3470</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1158%2F0008-5472.CAN-13-3470" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=24590058" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BC2cXnt1arsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=74&publication_year=2014&pages=2579-2590&author=P.+Maycotteauthor=C.+M.+Gearheartauthor=R.+Barnardauthor=S.+Aryalauthor=J.+M.+M.+Levyauthor=S.+P.+Fosmireauthor=R.+J.+Hansenauthor=M.+J.+Morganauthor=C.+C.+Porterauthor=D.+L.+Gustafsonauthor=A.+Thorburn&title=STAT3-mediated+autophagy+dependence+identifies+subtypes+of+breast+cancer+where+autophagy+inhibition+can+be+efficacious&doi=10.1158%2F0008-5472.CAN-13-3470"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">STAT3-Mediated Autophagy Dependence Identifies Subtypes of Breast Cancer Where Autophagy Inhibition Can Be Efficacious</span></div><div class="casAuthors">Maycotte, Paola; Gearheart, Christy M.; Barnard, Rebecca; Aryal, Suraj; Mulcahy Levy, Jean M.; Fosmire, Susan P.; Hansen, Ryan J.; Morgan, Michael J.; Porter, Christopher C.; Gustafson, Daniel L.; Thorburn, Andrew</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">74</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2579-2590</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Autophagy is a protein and organelle degrdn. pathway that is involved in diverse diseases, including cancer.  Recent evidence suggests that autophagy is a cell survival mechanism in tumor cells and that its inhibition, esp. in combination with other therapy, could be beneficial but it remains unclear if all cancer cells behave the same way when autophagy is inhibited.  We inhibited autophagy in a panel of breast cancer cell lines and found that some of them are dependent on autophagy for survival even in nutrient rich conditions without any addnl. stress, whereas others need autophagy only when stressed.  Survival under unstressed conditions is due to cell type-specific autophagy regulation of STAT3 activity and this phenotype is enriched in triple-neg. cell lines.  This autophagy-dependency affects response to therapy because autophagy inhibition reduced tumor growth in vivo in autophagy-dependent but not in autophagy-independent breast tumors, whereas combination treatment with autophagy inhibitors and other agent was preferentially synergistic in autophagy-dependent cells.  These results imply that autophagy-dependence represents a tumor cell-specific characteristic where autophagy inhibition will be more effective.  Moreover, our results suggest that autophagy inhibition might be a potential therapeutic strategy for triple-neg. breast cancers, which currently lack an effective targeted treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUScxQ9Siw3rVg90H21EOLACvtfcHk0liaXL5lCO1zTw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXnt1arsrg%253D&md5=21db68cf66514add33588020bece688e</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-13-3470&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-13-3470%26sid%3Dliteratum%253Aachs%26aulast%3DMaycotte%26aufirst%3DP.%26aulast%3DGearheart%26aufirst%3DC.%2BM.%26aulast%3DBarnard%26aufirst%3DR.%26aulast%3DAryal%26aufirst%3DS.%26aulast%3DLevy%26aufirst%3DJ.%2BM.%2BM.%26aulast%3DFosmire%26aufirst%3DS.%2BP.%26aulast%3DHansen%26aufirst%3DR.%2BJ.%26aulast%3DMorgan%26aufirst%3DM.%2BJ.%26aulast%3DPorter%26aufirst%3DC.%2BC.%26aulast%3DGustafson%26aufirst%3DD.%2BL.%26aulast%3DThorburn%26aufirst%3DA.%26atitle%3DSTAT3-mediated%2520autophagy%2520dependence%2520identifies%2520subtypes%2520of%2520breast%2520cancer%2520where%2520autophagy%2520inhibition%2520can%2520be%2520efficacious%26jtitle%3DCancer%2520Res.%26date%3D2014%26volume%3D74%26spage%3D2579%26epage%3D2590%26doi%3D10.1158%2F0008-5472.CAN-13-3470" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosol, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, X.</span></span> <span> </span><span class="NLM_article-title">Triple-negative breast cancer: is there a treatment on the horizon?</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1913</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.12284</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.18632%2Foncotarget.12284" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=27765921" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A280%3ADC%252BC2srks1aksw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=1913&author=H.+Yaoauthor=G.+Heauthor=S.+Yanauthor=C.+Chenauthor=L.+Songauthor=T.+J.+Rosolauthor=X.+Deng&title=Triple-negative+breast+cancer%3A+is+there+a+treatment+on+the+horizon%3F&doi=10.18632%2Foncotarget.12284"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Triple-negative breast cancer: is there a treatment on the horizon?</span></div><div class="casAuthors">Yao Hui; He Guangchun; Yan Shichao; Chen Chao; Deng Xiyun; Song Liujiang; Rosol Thomas J</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1913-1924</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Triple-negative breast cancer (TNBC), which accounts for 15-20% of all breast cancers, does not express estrogen receptor (ER) or progesterone receptor (PR) and lacks human epidermal growth factor receptor 2 (HER2) overexpression or amplification.  These tumors have a more aggressive phenotype and a poorer prognosis due to the high propensity for metastatic progression and absence of specific targeted treatments.  Patients with TNBC do not benefit from hormonal or trastuzumab-based targeted therapies because of the loss of target receptors.  Although these patients respond to chemotherapeutic agents such as taxanes and anthracyclines better than other subtypes of breast cancer, prognosis remains poor.  A group of targeted therapies under investigation showed favorable results in TNBC, especially in cancers with BRCA mutation.  The lipid-lowering statins (3-hydroxy-3-methyl-glutaryl coenzyme A reductase inhibitors), including lovastatin and simvastatin, have been shown to preferentially target TNBC compared with non-TNBC.  These statins hold great promise for the management of TNBC.  Only with the understanding of the molecular basis for the preference of statins for TNBC and more investigations in clinical trials can they be reformulated into a clinically approved drug against TNBC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSj55CS-5D4KA5nADWVYHg1fW6udTcc2eapAZZOyAF0Jrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2srks1aksw%253D%253D&md5=d78d07246b042a1087a643d9eb6761d4</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.12284&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.12284%26sid%3Dliteratum%253Aachs%26aulast%3DYao%26aufirst%3DH.%26aulast%3DHe%26aufirst%3DG.%26aulast%3DYan%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DC.%26aulast%3DSong%26aufirst%3DL.%26aulast%3DRosol%26aufirst%3DT.%2BJ.%26aulast%3DDeng%26aufirst%3DX.%26atitle%3DTriple-negative%2520breast%2520cancer%253A%2520is%2520there%2520a%2520treatment%2520on%2520the%2520horizon%253F%26jtitle%3DOncotarget%26date%3D2017%26volume%3D8%26spage%3D1913%26doi%3D10.18632%2Foncotarget.12284" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Prat, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perou, C. M.</span></span> <span> </span><span class="NLM_article-title">Deconstructing the molecular portraits of breast cancer</span>. <i>Mol. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">5</span>– <span class="NLM_lpage">23</span>, <span class="refDoi"> DOI: 10.1016/j.molonc.2010.11.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1016%2Fj.molonc.2010.11.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=21147047" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1Cjs7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2011&pages=5-23&author=A.+Pratauthor=C.+M.+Perou&title=Deconstructing+the+molecular+portraits+of+breast+cancer&doi=10.1016%2Fj.molonc.2010.11.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Deconstructing the molecular portraits of breast cancer</span></div><div class="casAuthors">Prat, Aleix; Perou, Charles M.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Oncology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">5-23</span>CODEN:
                <span class="NLM_cas:coden">MOONC3</span>;
        ISSN:<span class="NLM_cas:issn">1574-7891</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Breast cancer is a heterogeneous disease in terms of histol., therapeutic response, dissemination patterns to distant sites, and patient outcomes.  Global gene expression analyses using high-throughput technologies have helped to explain much of this heterogeneity and provided important new classifications of cancer patients.  In the last decade, genomic studies have established five breast cancer intrinsic subtypes (Luminal A, Luminal B, HER2-enriched, Claudin-low, Basal-like) and a Normal Breast-like group.  In this review, we dissect the most recent data on this genomic classification of breast cancer with a special focus on the Claudin-low subtype, which appears enriched for mesenchymal and stem cell features.  In addn., we discuss how the combination of std. clin.-pathol. markers with the information provided by these genomic entities might help further understand the biol. complexity of this disease, increase the efficacy of current and novel therapies, and ultimately improve outcomes for breast cancer patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrIk8X02ivwybVg90H21EOLACvtfcHk0liaXL5lCO1zTw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1Cjs7c%253D&md5=b320e1836b5f0e7e9fbbedf8bacec152</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2Fj.molonc.2010.11.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molonc.2010.11.003%26sid%3Dliteratum%253Aachs%26aulast%3DPrat%26aufirst%3DA.%26aulast%3DPerou%26aufirst%3DC.%2BM.%26atitle%3DDeconstructing%2520the%2520molecular%2520portraits%2520of%2520breast%2520cancer%26jtitle%3DMol.%2520Oncol.%26date%3D2011%26volume%3D5%26spage%3D5%26epage%3D23%26doi%3D10.1016%2Fj.molonc.2010.11.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">O’Reilly, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gubbins, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tully, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guang, M. H. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiner-Gorzel, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCaffrey, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harrison, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furlong, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kell, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCann, A.</span></span> <span> </span><span class="NLM_article-title">The fate of chemoresistance in triple negative breast cancer (TNBC)</span>. <i>BBA clin.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">257</span>– <span class="NLM_lpage">275</span>, <span class="refDoi"> DOI: 10.1016/j.bbacli.2015.03.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1016%2Fj.bbacli.2015.03.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=26676166" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A280%3ADC%252BC28risFagsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2015&pages=257-275&author=E.+A.+O%E2%80%99Reillyauthor=L.+Gubbinsauthor=S.+Sharmaauthor=R.+Tullyauthor=M.+H.+Z.+Guangauthor=K.+Weiner-Gorzelauthor=J.+McCaffreyauthor=M.+Harrisonauthor=F.+Furlongauthor=M.+Kellauthor=A.+McCann&title=The+fate+of+chemoresistance+in+triple+negative+breast+cancer+%28TNBC%29&doi=10.1016%2Fj.bbacli.2015.03.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">The fate of chemoresistance in triple negative breast cancer (TNBC)</span></div><div class="casAuthors">O'Reilly Elma A; Sharma Shiva; Gubbins Luke; Tully Riona; Guang Matthew Ho Zhing; Weiner-Gorzel Karolina; McCann Amanda; McCaffrey John; Harrison Michele; Furlong Fiona; Kell Malcolm</div><div class="citationInfo"><span class="NLM_cas:title">BBA clinical</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">257-75</span>
        ISSN:<span class="NLM_cas:issn">2214-6474</span>.
    </div><div class="casAbstract">BACKGROUND:  Treatment options for women presenting with triple negative breast cancer (TNBC) are limited due to the lack of a therapeutic target and as a result, are managed with standard chemotherapy such as paclitaxel (Taxol®).  Following chemotherapy, the ideal tumour response is apoptotic cell death.  Post-chemotherapy, cells can maintain viability by undergoing viable cellular responses such as cellular senescence, generating secretomes which can directly enhance the malignant phenotype.  SCOPE OF REVIEW:  How tumour cells retain viability in response to chemotherapeutic engagement is discussed.  In addition we discuss the implications of this retained tumour cell viability in the context of the development of recurrent and metastatic TNBC disease.  Current adjuvant and neo-adjuvant treatments available and the novel potential therapies that are being researched are also reviewed.  MAJOR CONCLUSIONS:  Cellular senescence and cytoprotective autophagy are potential mechanisms of chemoresistance in TNBC.  These two non-apoptotic outcomes in response to chemotherapy are inextricably linked and are neglected outcomes of investigation in the chemotherapeutic arena.  Cellular fate assessments may therefore have the potential to predict TNBC patient outcome.  GENERAL SIGNIFICANCE:  Focusing on the fact that cancer cells can bypass the desired cellular apoptotic response to chemotherapy through cellular senescence and cytoprotective autophagy will highlight the importance of targeting non-apoptotic survival pathways to enhance chemotherapeutic efficacy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSDPBSDA-sOPf9pSBfyDvp7fW6udTcc2eaysrxBr9MVArntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC28risFagsg%253D%253D&md5=af77680801122c14cdc0a8b714f4796b</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2Fj.bbacli.2015.03.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbacli.2015.03.003%26sid%3Dliteratum%253Aachs%26aulast%3DO%25E2%2580%2599Reilly%26aufirst%3DE.%2BA.%26aulast%3DGubbins%26aufirst%3DL.%26aulast%3DSharma%26aufirst%3DS.%26aulast%3DTully%26aufirst%3DR.%26aulast%3DGuang%26aufirst%3DM.%2BH.%2BZ.%26aulast%3DWeiner-Gorzel%26aufirst%3DK.%26aulast%3DMcCaffrey%26aufirst%3DJ.%26aulast%3DHarrison%26aufirst%3DM.%26aulast%3DFurlong%26aufirst%3DF.%26aulast%3DKell%26aufirst%3DM.%26aulast%3DMcCann%26aufirst%3DA.%26atitle%3DThe%2520fate%2520of%2520chemoresistance%2520in%2520triple%2520negative%2520breast%2520cancer%2520%2528TNBC%2529%26jtitle%3DBBA%2520clin.%26date%3D2015%26volume%3D3%26spage%3D257%26epage%3D275%26doi%3D10.1016%2Fj.bbacli.2015.03.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chittaranjan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bortnik, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dragowska, W. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abeysundara, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leung, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Go, N. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeVorkin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weppler, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gelmon, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yapp, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bally, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gorski, S. M.</span></span> <span> </span><span class="NLM_article-title">Autophagy inhibition augments the anticancer effects of epirubicin treatment in anthracycline-sensitive and-resistant triple-negative breast cancer</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">3159</span>– <span class="NLM_lpage">3173</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.CCR-13-2060</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1158%2F1078-0432.CCR-13-2060" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=24721646" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BC2cXps1Cmsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2014&pages=3159-3173&author=S.+Chittaranjanauthor=S.+Bortnikauthor=W.+H.+Dragowskaauthor=J.+Xuauthor=N.+Abeysundaraauthor=A.+Leungauthor=N.+E.+Goauthor=L.+DeVorkinauthor=S.+A.+Wepplerauthor=K.+Gelmonauthor=D.+T.+Yappauthor=M.+B.+Ballyauthor=S.+M.+Gorski&title=Autophagy+inhibition+augments+the+anticancer+effects+of+epirubicin+treatment+in+anthracycline-sensitive+and-resistant+triple-negative+breast+cancer&doi=10.1158%2F1078-0432.CCR-13-2060"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Autophagy Inhibition Augments the Anticancer Effects of Epirubicin Treatment in Anthracycline-Sensitive and -Resistant Triple-Negative Breast Cancer</span></div><div class="casAuthors">Chittaranjan, Suganthi; Bortnik, Svetlana; Dragowska, Wieslawa H.; Xu, Jing; Abeysundara, Namal; Leung, Amy; Go, Nancy E.; De Vorkin, Lindsay; Weppler, Sherry A.; Gelmon, Karen; Yapp, Donald T.; Bally, Marcel B.; Gorski, Sharon M.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3159-3173</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Purpose: Triple-neg. breast cancers (TNBC) are defined by a lack of expression of estrogen receptor (ER), progesterone receptor (PR), and human epidermal growth factor receptor 2 (ERBB2/HER2).  Although initially responsive to chemotherapy, most recurrent TNBCs develop resistance, resulting in disease progression.  Autophagy is a lysosome-mediated degrdn. and recycling process that can function as an adaptive survival response during chemotherapy and contribute to chemoresistance.  Our goal was to det. whether autophagy inhibition improves treatment efficacy in TNBC cells in tumors either sensitive or refractory to anthracyclines.  Exptl. Design: We used in vitro and in vivo models of TNBC using cell lines sensitive to epirubicin and other anthracyclines, as well as deriv. lines, resistant to the same drugs.  We assessed basal autophagy levels and the effects of chemotherapy on autophagy in parental and resistant cells.  Applying various approaches to inhibit autophagy alone and in combination with chemotherapy, we assessed the effects on cell viability in vitro and tumor growth rates in vivo.  Results: We demonstrated that epirubicin induced autophagic flux in TNBC cells.  Epirubicin-resistant lines exhibited at least 1.5-fold increased basal autophagy levels and, when treated with autophagy inhibitors, showed a significant loss in viability, indicating dependence of resistant cells on autophagy for survival.  Combination of epirubicin with the autophagy inhibitor hydroxychloroquine resulted in a significant redn. in tumor growth compared with monotherapy with epirubicin.  Conclusion: Autophagy inhibition enhances therapeutic response in both anthracycline-sensitive and -resistant TNBC and may be an effective new treatment strategy for this disease.  Clin Cancer Res; 20(12); 3159-73. ©2014 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXG-XFIhGb77Vg90H21EOLACvtfcHk0lhglm59g8pTHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXps1Cmsrc%253D&md5=3469e5a5e9c1f7e9ead384c8959f1602</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-13-2060&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-13-2060%26sid%3Dliteratum%253Aachs%26aulast%3DChittaranjan%26aufirst%3DS.%26aulast%3DBortnik%26aufirst%3DS.%26aulast%3DDragowska%26aufirst%3DW.%2BH.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DAbeysundara%26aufirst%3DN.%26aulast%3DLeung%26aufirst%3DA.%26aulast%3DGo%26aufirst%3DN.%2BE.%26aulast%3DDeVorkin%26aufirst%3DL.%26aulast%3DWeppler%26aufirst%3DS.%2BA.%26aulast%3DGelmon%26aufirst%3DK.%26aulast%3DYapp%26aufirst%3DD.%2BT.%26aulast%3DBally%26aufirst%3DM.%2BB.%26aulast%3DGorski%26aufirst%3DS.%2BM.%26atitle%3DAutophagy%2520inhibition%2520augments%2520the%2520anticancer%2520effects%2520of%2520epirubicin%2520treatment%2520in%2520anthracycline-sensitive%2520and-resistant%2520triple-negative%2520breast%2520cancer%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2014%26volume%3D20%26spage%3D3159%26epage%3D3173%26doi%3D10.1158%2F1078-0432.CCR-13-2060" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vogel, R. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coughlin, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scotti, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iizuka, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anchoori, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roden, R. B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marastoni, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bazzaro, M.</span></span> <span> </span><span class="NLM_article-title">Simultaneous inhibition of deubiquitinating enzymes (DUBs) and autophagy synergistically kills breast cancer cells</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">4159</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.2904</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.18632%2Foncotarget.2904" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=25784654" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A280%3ADC%252BC2Mnls1Sksg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=4159&author=R.+I.+Vogelauthor=K.+Coughlinauthor=A.+Scottiauthor=Y.+Iizukaauthor=R.+Anchooriauthor=R.+B.+S.+Rodenauthor=M.+Marastoniauthor=M.+Bazzaro&title=Simultaneous+inhibition+of+deubiquitinating+enzymes+%28DUBs%29+and+autophagy+synergistically+kills+breast+cancer+cells&doi=10.18632%2Foncotarget.2904"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Simultaneous inhibition of deubiquitinating enzymes (DUBs) and autophagy synergistically kills breast cancer cells</span></div><div class="casAuthors">Vogel Rachel Isaksson; Bazzaro Martina; Coughlin Kathleen; Iizuka Yoshie; Bazzaro Martina; Scotti Alessandra; Marastoni Mauro; Anchoori Ravi; Roden Richard B S; Anchoori Ravi; Roden Richard B S; Roden Richard B S</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">4159-70</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Breast cancer is one of the leading causes of cancer death among women in the United States.  Patients expressing the estrogen and progesterone receptor (ER and PR) and human epidermal growth factor 2 (HER-2) tumor markers have favorable prognosis and efficacious therapeutic options.  In contrast, tumors that are negative for these markers (triple-negative) have a disproportionate share of morbidity and mortality due to lack of a validated molecular target.  Deubiquitinating enzymes (DUBs) are a critical component of ubiquitin-proteasome-system degradation and have been shown to be differentially expressed and activated in a number of cancers, including breast, with their aberrant activity linked to cancer prognosis and clinical outcome.  We evaluated the effect of the DUB inhibitors b-AP15 and RA-9 alone and in combination with early- and late-stage lysosomal inhibitors on cell viability in a panel of triple negative breast cancer (TNBC) cell lines.  Our results indicate small-molecule DUB inhibitors have a profound effect on TNBC viability and lead to activation of autophagy as a cellular mechanism to compensate for ubiquitin-proteasome-system stress.  Treatment with sub-optimal doses of DUB and lysosome inhibitors synergistically kills TNBC cells.  This supports the evaluation of DUB inhibition, in combination with lysosomal inhibition, as a therapeutic approach for the treatment of TNBC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQvDoMDseirLGhHGZei4OW9fW6udTcc2eaysrxBr9MVArntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2Mnls1Sksg%253D%253D&md5=4cae506139805133d1a3776a41149aa6</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.2904&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.2904%26sid%3Dliteratum%253Aachs%26aulast%3DVogel%26aufirst%3DR.%2BI.%26aulast%3DCoughlin%26aufirst%3DK.%26aulast%3DScotti%26aufirst%3DA.%26aulast%3DIizuka%26aufirst%3DY.%26aulast%3DAnchoori%26aufirst%3DR.%26aulast%3DRoden%26aufirst%3DR.%2BB.%2BS.%26aulast%3DMarastoni%26aufirst%3DM.%26aulast%3DBazzaro%26aufirst%3DM.%26atitle%3DSimultaneous%2520inhibition%2520of%2520deubiquitinating%2520enzymes%2520%2528DUBs%2529%2520and%2520autophagy%2520synergistically%2520kills%2520breast%2520cancer%2520cells%26jtitle%3DOncotarget%26date%3D2015%26volume%3D6%26spage%3D4159%26doi%3D10.18632%2Foncotarget.2904" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Amaravadi, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kimmelman, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Debnath, J.</span></span> <span> </span><span class="NLM_article-title">Targeting Autophagy in Cancer: Recent Advances and Future Directions</span>. <i>Cancer discovery</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1167</span>– <span class="NLM_lpage">1181</span>, <span class="refDoi"> DOI: 10.1158/2159-8290.CD-19-0292</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1158%2F2159-8290.CD-19-0292" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=31434711" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhvFGjs7zN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2019&pages=1167-1181&author=R.+K.+Amaravadiauthor=A.+C.+Kimmelmanauthor=J.+Debnath&title=Targeting+Autophagy+in+Cancer%3A+Recent+Advances+and+Future+Directions&doi=10.1158%2F2159-8290.CD-19-0292"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting autophagy in cancer: recent advances and future directions</span></div><div class="casAuthors">Amaravadi, Ravi K.; Kimmelman, Alec C.; Debnath, Jayanta</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Discovery</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1167-1181</span>CODEN:
                <span class="NLM_cas:coden">CDAIB2</span>;
        ISSN:<span class="NLM_cas:issn">2159-8274</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">A review.  Autophagy, a multistep lysosomal degrdn. pathway that supports nutrient recycling and metabolic adaptation, has been implicated as a process that regulates cancer.  Although autophagy induction may limit the development of tumors, evidence in mouse models demonstrates that autophagy inhibition can limit the growth of established tumors and improve response to cancer therapeutics.  Certain cancer genotypes may be esp. prone to autophagy inhibition.  Different strategies for autophagy modulation may be needed depending on the cancer context.  Here, we review new advances in the mol. control of autophagy, the role of selective autophagy in cancer, and the role of autophagy within the tumor microenvironment and tumor immunity.  We also highlight clin. efforts to repurpose lysosomal inhibitors, such as hydroxychloroquine, as anticancer agents that block autophagy, as well as the development of more potent and specific autophagy inhibitors for cancer treatment, and review future directions for autophagy research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpuQskWwQTM5rVg90H21EOLACvtfcHk0lgYgb2WQ4sAYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhvFGjs7zN&md5=cb8453bd893e3dbfbfccf2b2116a9e6b</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1158%2F2159-8290.CD-19-0292&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F2159-8290.CD-19-0292%26sid%3Dliteratum%253Aachs%26aulast%3DAmaravadi%26aufirst%3DR.%2BK.%26aulast%3DKimmelman%26aufirst%3DA.%2BC.%26aulast%3DDebnath%26aufirst%3DJ.%26atitle%3DTargeting%2520Autophagy%2520in%2520Cancer%253A%2520Recent%2520Advances%2520and%2520Future%2520Directions%26jtitle%3DCancer%2520discovery%26date%3D2019%26volume%3D9%26spage%3D1167%26epage%3D1181%26doi%3D10.1158%2F2159-8290.CD-19-0292" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hwang, D. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eom, J.-I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jang, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeung, H.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheong, J.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Min, Y. H.</span></span> <span> </span><span class="NLM_article-title">ULK1 inhibition as a targeted therapeutic strategy for FLT3-ITD-mutated acute myeloid leukemia</span>. <i>J. Exp. Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">14</span>, <span class="refDoi"> DOI: 10.1186/s13046-020-01580-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1186%2Fs13046-020-01580-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=31928527" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2020&pages=1-14&author=D.+Y.+Hwangauthor=J.-I.+Eomauthor=J.+E.+Jangauthor=H.-K.+Jeungauthor=H.+Chungauthor=J.+S.+Kimauthor=J.-W.+Cheongauthor=Y.+H.+Min&title=ULK1+inhibition+as+a+targeted+therapeutic+strategy+for+FLT3-ITD-mutated+acute+myeloid+leukemia&doi=10.1186%2Fs13046-020-01580-4"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1186%2Fs13046-020-01580-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs13046-020-01580-4%26sid%3Dliteratum%253Aachs%26aulast%3DHwang%26aufirst%3DD.%2BY.%26aulast%3DEom%26aufirst%3DJ.-I.%26aulast%3DJang%26aufirst%3DJ.%2BE.%26aulast%3DJeung%26aufirst%3DH.-K.%26aulast%3DChung%26aufirst%3DH.%26aulast%3DKim%26aufirst%3DJ.%2BS.%26aulast%3DCheong%26aufirst%3DJ.-W.%26aulast%3DMin%26aufirst%3DY.%2BH.%26atitle%3DULK1%2520inhibition%2520as%2520a%2520targeted%2520therapeutic%2520strategy%2520for%2520FLT3-ITD-mutated%2520acute%2520myeloid%2520leukemia%26jtitle%3DJ.%2520Exp.%2520Clin.%2520Cancer%2520Res.%26date%3D2020%26volume%3D39%26spage%3D1%26epage%3D14%26doi%3D10.1186%2Fs13046-020-01580-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Qiu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, S.</span></span> <span> </span><span class="NLM_article-title">Targeted inhibition of ULK1 enhances daunorubicin sensitivity in acute myeloid leukemia</span>. <i>Life Sci.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>243</i></span>,  <span class="NLM_fpage">117234</span>, <span class="refDoi"> DOI: 10.1016/j.lfs.2019.117234</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1016%2Fj.lfs.2019.117234" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=31887299" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtFCrt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=243&publication_year=2020&pages=117234&author=L.+Qiuauthor=G.+Zhouauthor=S.+Cao&title=Targeted+inhibition+of+ULK1+enhances+daunorubicin+sensitivity+in+acute+myeloid+leukemia&doi=10.1016%2Fj.lfs.2019.117234"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted inhibition of ULK1 enhances daunorubicin sensitivity in acute myeloid leukemia</span></div><div class="casAuthors">Qiu, Li; Zhou, Gan; Cao, Shan</div><div class="citationInfo"><span class="NLM_cas:title">Life Sciences</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">243</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">117234</span>CODEN:
                <span class="NLM_cas:coden">LIFSAK</span>;
        ISSN:<span class="NLM_cas:issn">0024-3205</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">In acute myeloid leukemia (AML), complete remission can be achieved in parts of patients using cytarabine/anthracycline combination-based chemotherapy, however, drug resistance-related recurrence is still a common cause of treatment failure, leading to high mortality among patients.  In our research, we revealed the mol. mechanisms that were sufficient to improve sensitivity of AML cells to the anthracycline daunorubicin (DNR).  We evaluated the effects of autophagy and apoptosis induced by DNR using two AML cell lines HL60 and U937.  Western blot was preformed to analyze the apoptotic pathway protein expression and flow cytometric anal. was used to detect the level of apoptosis in AML cells.  The levels of autophagy-related proteins were detected by western blotting and autophagic vesicles were obsd. by electron microscopy.  Furthermore, ULK1 inhibition by a highly selective kinase inhibitor SBI-0206965 and shRNA enhanced cytotoxicity of DNR against AML cells.  Independent of mTOR -ULK1 signaling pathway, activation of autophagy of DNR was proved to be mediated by AMPK (pThr172)/ULK1 pathway.  These results revealed that pro-survival autophagy induced by ULK1 activation was one of the potential mechanisms of AML resistance to DNR.  Targeting ULK1 selectively could be a promising therapeutic strategy to enhance sensitivity of DNR for AML therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhTKvNP6OQUbVg90H21EOLACvtfcHk0lgYgb2WQ4sAYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtFCrt7w%253D&md5=293767bee9d9835e1d669d922ed1bf37</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2Fj.lfs.2019.117234&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.lfs.2019.117234%26sid%3Dliteratum%253Aachs%26aulast%3DQiu%26aufirst%3DL.%26aulast%3DZhou%26aufirst%3DG.%26aulast%3DCao%26aufirst%3DS.%26atitle%3DTargeted%2520inhibition%2520of%2520ULK1%2520enhances%2520daunorubicin%2520sensitivity%2520in%2520acute%2520myeloid%2520leukemia%26jtitle%3DLife%2520Sci.%26date%3D2020%26volume%3D243%26spage%3D117234%26doi%3D10.1016%2Fj.lfs.2019.117234" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xue, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gong, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, C.</span></span> <span> </span><span class="NLM_article-title">SBI0206965, a novel inhibitor of Ulk1, suppresses non-small cell lung cancer cell growth by modulating both autophagy and apoptosis pathways</span>. <i>Oncol. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>37</i></span>,  <span class="NLM_fpage">3449</span>– <span class="NLM_lpage">3458</span>, <span class="refDoi"> DOI: 10.3892/or.2017.5635</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.3892%2For.2017.5635" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=28498429" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsVOiu7zJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2017&pages=3449-3458&author=F.+Tangauthor=P.+Huauthor=Z.+Yangauthor=C.+Xueauthor=J.+Gongauthor=S.+Sunauthor=L.+Shiauthor=S.+Zhangauthor=Z.+Liauthor=C.+Yangauthor=C.+Zhangauthor=C.+Xie&title=SBI0206965%2C+a+novel+inhibitor+of+Ulk1%2C+suppresses+non-small+cell+lung+cancer+cell+growth+by+modulating+both+autophagy+and+apoptosis+pathways&doi=10.3892%2For.2017.5635"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">SBI0206965, a novel inhibitor of Ulk1, suppresses non-small cell lung cancer cell growth by modulating both autophagy and apoptosis pathways</span></div><div class="casAuthors">Tang, Fang; Hu, Pengchao; Yang, Zetian; Xue, Chao; Gong, Jun; Sun, Shaoxing; Shi, Liu; Zhang, Shimin; Li, Zhenzhen; Yang, Chunxu; Zhang, Junhong; Xie, Conghua</div><div class="citationInfo"><span class="NLM_cas:title">Oncology Reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">3449-3458</span>CODEN:
                <span class="NLM_cas:coden">OCRPEW</span>;
        ISSN:<span class="NLM_cas:issn">1791-2431</span>.
    
            (<span class="NLM_cas:orgname">Spandidos Publications Ltd.</span>)
        </div><div class="casAbstract">Lung cancer is a major public health problem worldwide.  Non-small cell lung cancer (NSCLC) accounts for 85% of lung cancer cases.  Autophagy has recently sparked great interest, and it is thought to participate in a variety of diseases, including lung cancer.  Uncoordinated (Unc) 51-like kinase 1 (Ulk1), a serine/threonine kinase, plays a central role in the autophagy pathway.  However, the role of Ulk1 in NSCLC remains unclear.  We report that NSCLC cell lines exhibited high expression of Ulk1 and that Ulk1 was neg. correlated with prognosis in lung cancer patients.  Knockdown of Ulk1 or the inhibition of Ulk1 by the selective inhibitor SBI0206965, inhibited cell proliferation, induced cell apoptosis and enhanced the sensitivity of cisplatin against NSCLC cells.  Moreover, we demonstrated that Ulk1 exerted oncogenic activity in NSCLC by modulating both autophagy and apoptosis pathways.  Inhibition of autophagy by SBI0206965 sensitized NSCLC cells to cisplatin by inhibiting cisplatin induced cell-protective autophagy to promote apoptosis.  Furthermore, SBI0206965 promoted apoptosis in NSCLC cells independent of autophagy, which was partly mediated by destabilization of Bcl2/Bclxl.  In summary, our results show that inhibition of Ulk1 suppresses NSCLC cell growth and sensitizes NSCLC cells to cisplatin by modulating both autophagy and apoptosis pathways, and that Ulk1 might be a promising target for NSCLC treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGSAExMnHCRLVg90H21EOLACvtfcHk0lgYgb2WQ4sAYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsVOiu7zJ&md5=00bfa972d3f82ae5439ccc97cd8221d4</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.3892%2For.2017.5635&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3892%252For.2017.5635%26sid%3Dliteratum%253Aachs%26aulast%3DTang%26aufirst%3DF.%26aulast%3DHu%26aufirst%3DP.%26aulast%3DYang%26aufirst%3DZ.%26aulast%3DXue%26aufirst%3DC.%26aulast%3DGong%26aufirst%3DJ.%26aulast%3DSun%26aufirst%3DS.%26aulast%3DShi%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DYang%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DXie%26aufirst%3DC.%26atitle%3DSBI0206965%252C%2520a%2520novel%2520inhibitor%2520of%2520Ulk1%252C%2520suppresses%2520non-small%2520cell%2520lung%2520cancer%2520cell%2520growth%2520by%2520modulating%2520both%2520autophagy%2520and%2520apoptosis%2520pathways%26jtitle%3DOncol.%2520Rep.%26date%3D2017%26volume%3D37%26spage%3D3449%26epage%3D3458%26doi%3D10.3892%2For.2017.5635" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chaikuad, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koschade, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stolz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zivkovic, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pohl, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaid, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lambert, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cosford, N. D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brandts, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">Conservation of structure, function and inhibitor binding in UNC-51-like kinase 1 and 2 (ULK1/2)</span>. <i>Biochem. J.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>476</i></span>,  <span class="NLM_fpage">875</span>– <span class="NLM_lpage">887</span>, <span class="refDoi"> DOI: 10.1042/BCJ20190038</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1042%2FBCJ20190038" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=30782972" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BC1MXpt1aitbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=476&publication_year=2019&pages=875-887&author=A.+Chaikuadauthor=S.+E.+Koschadeauthor=A.+Stolzauthor=K.+Zivkovicauthor=C.+Pohlauthor=S.+Shaidauthor=H.+Renauthor=L.+J.+Lambertauthor=N.+D.+P.+Cosfordauthor=C.+H.+Brandtsauthor=S.+Knapp&title=Conservation+of+structure%2C+function+and+inhibitor+binding+in+UNC-51-like+kinase+1+and+2+%28ULK1%2F2%29&doi=10.1042%2FBCJ20190038"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Conservation of structure, function and inhibitor binding in UNC-51-like kinase 1 and 2 (ULK1/2)</span></div><div class="casAuthors">Chaikuad, Apirat; Koschade, Sebastian E.; Stolz, Alexandra; Zivkovic, Katarina; Pohl, Christian; Shaid, Shabnam; Ren, Huiyu; Lambert, Lester J.; Cosford, Nicholas D. P.; Brandts, Christian H.; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Journal</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">476</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">875-887</span>CODEN:
                <span class="NLM_cas:coden">BIJOAK</span>;
        ISSN:<span class="NLM_cas:issn">0264-6021</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">Autophagy is essential for cellular homeostasis and when deregulated this survival mechanism has been assocd. with disease development.  Inhibition of autophagy initiation by inhibiting the kinase ULK1 (Unc-51-like autophagy activating kinase 1) has been proposed as a potential cancer therapy.  While inhibitors and crystal structures of ULK1 have been reported, little is known about the other closely related kinase ULK2 (Unc-51-like autophagy activating kinase 2).  Here, we present the crystal structure of ULK2 in complex with ATP competitive inhibitors.  Surprisingly, the ULK2 structure revealed a dimeric assembly reminiscent of dimeric arrangements of auto-activating kinases suggesting a role for this assocn. in ULK activation.  Screening of a kinase focused library of pre-clin. and clin. compds. revealed several potent ULK1/2 inhibitors and good correlation of inhibitor-binding behavior with both ULK kinases.  Aurora A was identified as a major off-target of currently used ULK1 inhibitors.  Autophagic flux assays demonstrated that this off-target activity by strongly inducing autophagy in different cellular systems conferred an addnl. layer of complexity in the interpretation of cellular data.  The data presented here provide structural models and chem. starting points for the development of ULK1/2 dual inhibitors with improved selectivity for future exploitation of autophagy inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq-GpxTacuIQLVg90H21EOLACvtfcHk0lgmOX6omrpiew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXpt1aitbo%253D&md5=4c2fbe0d3a3180ffb58b28fcb377a5a2</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1042%2FBCJ20190038&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBCJ20190038%26sid%3Dliteratum%253Aachs%26aulast%3DChaikuad%26aufirst%3DA.%26aulast%3DKoschade%26aufirst%3DS.%2BE.%26aulast%3DStolz%26aufirst%3DA.%26aulast%3DZivkovic%26aufirst%3DK.%26aulast%3DPohl%26aufirst%3DC.%26aulast%3DShaid%26aufirst%3DS.%26aulast%3DRen%26aufirst%3DH.%26aulast%3DLambert%26aufirst%3DL.%2BJ.%26aulast%3DCosford%26aufirst%3DN.%2BD.%2BP.%26aulast%3DBrandts%26aufirst%3DC.%2BH.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DConservation%2520of%2520structure%252C%2520function%2520and%2520inhibitor%2520binding%2520in%2520UNC-51-like%2520kinase%25201%2520and%25202%2520%2528ULK1%252F2%2529%26jtitle%3DBiochem.%2520J.%26date%3D2019%26volume%3D476%26spage%3D875%26epage%3D887%26doi%3D10.1042%2FBCJ20190038" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roskoski, R.,  Jr.</span></span> <span> </span><span class="NLM_article-title">Classification of small molecule protein kinase inhibitors based upon the structures of their drug-enzyme complexes</span>. <i>Pharmacol. Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>103</i></span>,  <span class="NLM_fpage">26</span>– <span class="NLM_lpage">48</span>, <span class="refDoi"> DOI: 10.1016/j.phrs.2015.10.021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1016%2Fj.phrs.2015.10.021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=26529477" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslKksL%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=2016&pages=26-48&author=R.+Roskoski&title=Classification+of+small+molecule+protein+kinase+inhibitors+based+upon+the+structures+of+their+drug-enzyme+complexes&doi=10.1016%2Fj.phrs.2015.10.021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Classification of small molecule protein kinase inhibitors based upon the structures of their drug-enzyme complexes</span></div><div class="casAuthors">Roskoski, Robert</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacological Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">103</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">26-48</span>CODEN:
                <span class="NLM_cas:coden">PHMREP</span>;
        ISSN:<span class="NLM_cas:issn">1043-6618</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Because dysregulation and mutations of protein kinases play causal roles in human disease, this family of enzymes has become one of the most important drug targets over the past two decades.  The X-ray crystal structures of 21 of the 27 FDA-approved small mol. inhibitors bound to their target protein kinases are depicted in this paper.  The structure of the enzyme-bound antagonist complex is used in the classification of these inhibitors.  Type I inhibitors bind to the active protein kinase conformation (DFG-Asp in, αC-helix in).  Type I1/2 inhibitors bind to a DFG-Asp in inactive conformation while Type II inhibitors bind to a DFG-Asp out inactive conformation.  Type I, I1/2, and type II inhibitors occupy part of the adenine binding pocket and form hydrogen bonds with the hinge region connecting the small and large lobes of the enzyme.  Type III inhibitors bind next to the ATP-binding pocket and type IV inhibitors do not bind to the ATP or peptide substrate binding sites.  Type III and IV inhibitors are allosteric in nature.  Type V inhibitors bind to two different regions of the protein kinase domain and are therefore bivalent inhibitors.  The type I-V inhibitors are reversible.  In contrast, type VI inhibitors bind covalently to their target enzyme.  Type I, I1/2, and II inhibitors are divided into A and B subtypes.  The type A inhibitors bind in the front cleft, the back cleft, and near the gatekeeper residue, all of which occur within the region sepg. the small and large lobes of the protein kinase.  The type B inhibitors bind in the front cleft and gate area but do not extend into the back cleft.  An anal. of the limited available data indicates that type A inhibitors have a long residence time (minutes to hours) while the type B inhibitors have a short residence time (seconds to minutes).  The catalytic spine includes residues from the small and large lobes and interacts with the adenine ring of ATP.  Nearly all of the approved protein kinase inhibitors occupy the adenine-binding pocket; thus it is not surprising that these inhibitors interact with nearby catalytic spine (CS) residues.  Moreover, a significant no. of approved drugs also interact with regulatory spine (RS) residues.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpH0COwqC0CabVg90H21EOLACvtfcHk0lgmOX6omrpiew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslKksL%252FN&md5=e6b45c8644e322f30f898e763780c64b</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2Fj.phrs.2015.10.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.phrs.2015.10.021%26sid%3Dliteratum%253Aachs%26aulast%3DRoskoski%26aufirst%3DR.%26atitle%3DClassification%2520of%2520small%2520molecule%2520protein%2520kinase%2520inhibitors%2520based%2520upon%2520the%2520structures%2520of%2520their%2520drug-enzyme%2520complexes%26jtitle%3DPharmacol.%2520Res.%26date%3D2016%26volume%3D103%26spage%3D26%26epage%3D48%26doi%3D10.1016%2Fj.phrs.2015.10.021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">de
Oliveira, P. S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferraz, F. A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pena, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pramio, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morais, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schechtman, D.</span></span> <span> </span><span class="NLM_article-title">Revisiting protein kinase–substrate interactions: Toward therapeutic development</span>. <i>Sci. Signaling</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">re3</span>– <span class="NLM_lpage">re3</span>, <span class="refDoi"> DOI: 10.1126/scisignal.aad4016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1126%2Fscisignal.aad4016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=27016527" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2016&pages=re3-re3&author=P.+S.+L.+de%0AOliveiraauthor=F.+A.+N.+Ferrazauthor=D.+A.+Penaauthor=D.+T.+Pramioauthor=F.+A.+Moraisauthor=D.+Schechtman&title=Revisiting+protein+kinase%E2%80%93substrate+interactions%3A+Toward+therapeutic+development&doi=10.1126%2Fscisignal.aad4016"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1126%2Fscisignal.aad4016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscisignal.aad4016%26sid%3Dliteratum%253Aachs%26aulast%3Dde%2BOliveira%26aufirst%3DP.%2BS.%2BL.%26aulast%3DFerraz%26aufirst%3DF.%2BA.%2BN.%26aulast%3DPena%26aufirst%3DD.%2BA.%26aulast%3DPramio%26aufirst%3DD.%2BT.%26aulast%3DMorais%26aufirst%3DF.%2BA.%26aulast%3DSchechtman%26aufirst%3DD.%26atitle%3DRevisiting%2520protein%2520kinase%25E2%2580%2593substrate%2520interactions%253A%2520Toward%2520therapeutic%2520development%26jtitle%3DSci.%2520Signaling%26date%3D2016%26volume%3D9%26spage%3Dre3%26epage%3Dre3%26doi%3D10.1126%2Fscisignal.aad4016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dar, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shokat, K. M.</span></span> <span> </span><span class="NLM_article-title">The evolution of protein kinase inhibitors from antagonists to agonists of cellular signaling</span>. <i>Annu. Rev. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>80</i></span>,  <span class="NLM_fpage">769</span>– <span class="NLM_lpage">795</span>, <span class="refDoi"> DOI: 10.1146/annurev-biochem-090308-173656</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1146%2Fannurev-biochem-090308-173656" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=21548788" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BC3MXptVCnsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=80&publication_year=2011&pages=769-795&author=A.+C.+Darauthor=K.+M.+Shokat&title=The+evolution+of+protein+kinase+inhibitors+from+antagonists+to+agonists+of+cellular+signaling&doi=10.1146%2Fannurev-biochem-090308-173656"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">The evolution of protein kinase inhibitors from antagonists to agonists of cellular signaling</span></div><div class="casAuthors">Dar, Arvin C.; Shokat, Kevan M.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Biochemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">80</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">769-795</span>CODEN:
                <span class="NLM_cas:coden">ARBOAW</span>;
        ISSN:<span class="NLM_cas:issn">0066-4154</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review.  Kinases are highly regulated enzymes with diverse mechanisms controlling their catalytic output.  Over time, chem. discovery efforts for kinases have produced ATP-competitive compds., allosteric regulators, irreversible binders, and highly specific inhibitors.  These distinct classes of small mols. have revealed many novel aspects about kinase-mediated signaling, and some have progressed from simple tool compds. into clin. validated therapeutics.  This review explores several small-mol. inhibitors for kinases high-lighting elaborate mechanisms by which kinase function is modulated.  A complete surprise of targeted kinase drug discovery has been the finding of ATP-competitive inhibitors that behave as agonists, rather than antagonists, of their direct kinase target.  These studies hint at a connection between ATP-binding site occupancy and networks of communication that are independent of kinase catalysis.  Indeed, kinase inhibitors that induce changes in protein localization, protein-protein interactions, and even enhancement of catalytic activity of the target kinase have been found.  The relevance of these findings to the therapeutic efficacy of kinase inhibitors and to the future identification of new classes of drug targets is discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVWDRwiciMXrVg90H21EOLACvtfcHk0lgmOX6omrpiew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXptVCnsbY%253D&md5=1bb989c45f767784e4ee777359ae4870</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1146%2Fannurev-biochem-090308-173656&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-biochem-090308-173656%26sid%3Dliteratum%253Aachs%26aulast%3DDar%26aufirst%3DA.%2BC.%26aulast%3DShokat%26aufirst%3DK.%2BM.%26atitle%3DThe%2520evolution%2520of%2520protein%2520kinase%2520inhibitors%2520from%2520antagonists%2520to%2520agonists%2520of%2520cellular%2520signaling%26jtitle%3DAnnu.%2520Rev.%2520Biochem.%26date%3D2011%26volume%3D80%26spage%3D769%26epage%3D795%26doi%3D10.1146%2Fannurev-biochem-090308-173656" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lazarus, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Novotny, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shokat, K. M.</span></span> <span> </span><span class="NLM_article-title">Structure of the human autophagy initiating kinase ULK1 in complex with potent inhibitors</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">257</span>– <span class="NLM_lpage">261</span>, <span class="refDoi"> DOI: 10.1021/cb500835z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb500835z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFGqsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=257-261&author=M.+B.+Lazarusauthor=C.+J.+Novotnyauthor=K.+M.+Shokat&title=Structure+of+the+human+autophagy+initiating+kinase+ULK1+in+complex+with+potent+inhibitors&doi=10.1021%2Fcb500835z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of the Human Autophagy Initiating Kinase ULK1 in Complex with Potent Inhibitors</span></div><div class="casAuthors">Lazarus, Michael B.; Novotny, Chris J.; Shokat, Kevan M.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">257-261</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Autophagy is a conserved cellular process that involves the degrdn. of cellular components for energy maintenance and cytoplasmic quality control that has recently gained interest as a novel target for a variety of human diseases, including cancer.  A prime candidate to det. the potential therapeutic benefit of targeting autophagy is the kinase ULK1, whose activation initiates autophagy.  Here, we report the first structures of ULK1, in complex with multiple potent inhibitors.  These structures show features unique to the enzyme and will provide a path for the rational design of selective compds. as cellular probes and potential therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGryGDGKTxwZ3bVg90H21EOLACvtfcHk0lhA_sVq7vpxmA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFGqsQ%253D%253D&md5=3439432fc00cc6954ab664027bedf9e7</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fcb500835z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb500835z%26sid%3Dliteratum%253Aachs%26aulast%3DLazarus%26aufirst%3DM.%2BB.%26aulast%3DNovotny%26aufirst%3DC.%2BJ.%26aulast%3DShokat%26aufirst%3DK.%2BM.%26atitle%3DStructure%2520of%2520the%2520human%2520autophagy%2520initiating%2520kinase%2520ULK1%2520in%2520complex%2520with%2520potent%2520inhibitors%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2015%26volume%3D10%26spage%3D257%26epage%3D261%26doi%3D10.1021%2Fcb500835z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abagyan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Totrov, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuznetsov, D.</span></span> <span> </span><span class="NLM_article-title">ICM—a new method for protein modeling and design: applications to docking and structure prediction from the distorted native conformation</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">488</span>– <span class="NLM_lpage">506</span>, <span class="refDoi"> DOI: 10.1002/jcc.540150503</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1002%2Fjcc.540150503" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADyaK2MXhtVWisA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=1994&pages=488-506&author=R.+Abagyanauthor=M.+Totrovauthor=D.+Kuznetsov&title=ICM%E2%80%94a+new+method+for+protein+modeling+and+design%3A+applications+to+docking+and+structure+prediction+from+the+distorted+native+conformation&doi=10.1002%2Fjcc.540150503"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">ICM - a new method for protein modeling and design: applications to docking and structure prediction from the distorted native conformation</span></div><div class="casAuthors">Abagyan, Ruben; Totrov, Maxim; Kuznetsov, Dmitry</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">488-506</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    </div><div class="casAbstract">An efficient methodol., further referred to as internal coordinate modeling (ICM), for versatile modeling operations and global energy optimization on arbitrary fixed multimol. systems is described.  It is aimed at protein structure prediction, homol. modeling, mol. docking, NMR structure detn., and protein design.  The method uses and further develops a previously introduced approach to model biomol. structures in which bond lengths, bond angles, and torsion angles are considered as independent variables, any subset of them being fixed.  Here the authors simplify and generalize the basic description of the system, introduce the variable dihedral phase angle, and allow arbitrary connections of the mols. and conventional definition of the torsion angles.  Algorithms for calcn. of energy derivs. with respect to internal variables in the topol. tree of the system and for rapid evaluation of accessible surface are presented.  Multidimensional variables restraints are proposed to represent the statistical information about the torsion angle distributions in proteins.  To incorporate complex energy terms as solvation energy and electrostatics into a structure prediction procedure, a "double-energy" Monte Carlo minimization procedure in which these terms are omitted during the minimization state of the random step and included for the comparison with the previous conformation in a Markov chain is proposed and justified.  The ICM method is applied successfully to a mol. docking problem.  The procedure finds the correct parallel arrangement of 2 rigid helixes from a leucine zipper domain as the lowest-energy conformation (0.5 Å root mean square, RMS, deviation from the native structure) starting from completely random configuration.  Structures with antiparallel helixes or helixes staggered by one helix turn had energies higher by about 7 or 9 kcal/mol, resp.  Soft docking was also attempted.  A docking procedure allowing side-chain flexibility also converged to the parallel configuration starting from the helixes optimized individually.  To justify an internal coordinate approach to the structure prediction as opposed to a Cartesian one, energy hypersurfaces around the native structure of the squash seeds trypsin inhibitor were studied.  Torsion angle minimization from the optimal conformation randomly distorted up to the RMS deviation of 2.2 Å or angular RMS deviation of 10° restored the native conformation in most cases.  In contrast, Cartesian coordinate minimization did not reach the min. from deviations as small as 0.3 Å or 2°.  The authors conclude that the most promising detailed approach to the protein-folding problem would consist of some coarse global sampling strategy combined with the local energy minimization in the torsion coordinate space.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6kscl2--38LVg90H21EOLACvtfcHk0lhA_sVq7vpxmA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXhtVWisA%253D%253D&md5=5f1a1836429860ded9012dab112c9c47</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1002%2Fjcc.540150503&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.540150503%26sid%3Dliteratum%253Aachs%26aulast%3DAbagyan%26aufirst%3DR.%26aulast%3DTotrov%26aufirst%3DM.%26aulast%3DKuznetsov%26aufirst%3DD.%26atitle%3DICM%25E2%2580%2594a%2520new%2520method%2520for%2520protein%2520modeling%2520and%2520design%253A%2520applications%2520to%2520docking%2520and%2520structure%2520prediction%2520from%2520the%2520distorted%2520native%2520conformation%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D1994%26volume%3D15%26spage%3D488%26epage%3D506%26doi%3D10.1002%2Fjcc.540150503" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zegzouti, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zdanovskaia, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsiao, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goueli, S. A.</span></span> <span> </span><span class="NLM_article-title">ADP-Glo: A Bioluminescent and homogeneous ADP monitoring assay for kinases</span>. <i>Assay Drug Dev. Technol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">560</span>– <span class="NLM_lpage">572</span>, <span class="refDoi"> DOI: 10.1089/adt.2009.0222</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1089%2Fadt.2009.0222" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=20105026" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsFejtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2009&pages=560-572&author=H.+Zegzoutiauthor=M.+Zdanovskaiaauthor=K.+Hsiaoauthor=S.+A.+Goueli&title=ADP-Glo%3A+A+Bioluminescent+and+homogeneous+ADP+monitoring+assay+for+kinases&doi=10.1089%2Fadt.2009.0222"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">ADP-Glo: a bioluminescent and homogeneous ADP monitoring assay for kinases</span></div><div class="casAuthors">Zegzouti, Hicham; Zdanovskaia, Marina; Hsiao, Kevin; Goueli, Said A.</div><div class="citationInfo"><span class="NLM_cas:title">Assay and Drug Development Technologies</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">560-572</span>CODEN:
                <span class="NLM_cas:coden">ADDTAR</span>;
        ISSN:<span class="NLM_cas:issn">1540-658X</span>.
    
            (<span class="NLM_cas:orgname">Mary Ann Liebert, Inc.</span>)
        </div><div class="casAbstract">ADP-Glo is a novel bioluminescent, homogeneous assay for monitoring ADP producing biochem. reactions and thus it is an ideal assay for detecting enzyme activity using a wide variety of substrates.  It is a universal assay that can be used with protein kinases, lipid kinases, sugar kinases, and many more kinases as well as ATPases.  Because of its high sensitivity, it is suitable for monitoring enzyme activities at very early substrate conversions requiring very low amt. of enzymes.  Furthermore, as the assay is applicable to a broad range of ATP and substrate concns., it is optimal for enzymes that require high ATP and substrate concns.  This is crit. since inhibitor potency has to be demonstrated at the cellular level where ATP is present at millimolar concns.  ADP-Glo is performed in 2 steps upon completion of kinase reaction: a combined termination of kinase reaction and depletion of remaining ATP in the first step, and conversion of generated ADP to ATP and the newly produced ATP to light output using luciferase/luciferin reaction in the second step.  The luminescent signal generated is proportional to the ADP concn. produced and is correlated with the kinase activity.  Due to its high signal to background and luminescent readout, this assay is less susceptible to generation of false hits and thus it is applicable to not only primary and secondary screening but also kinase profiling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGocYSq13vSVnrVg90H21EOLACvtfcHk0lhA_sVq7vpxmA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsFejtL8%253D&md5=b2bba87e66707b7e7015220c1c3a414a</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1089%2Fadt.2009.0222&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1089%252Fadt.2009.0222%26sid%3Dliteratum%253Aachs%26aulast%3DZegzouti%26aufirst%3DH.%26aulast%3DZdanovskaia%26aufirst%3DM.%26aulast%3DHsiao%26aufirst%3DK.%26aulast%3DGoueli%26aufirst%3DS.%2BA.%26atitle%3DADP-Glo%253A%2520A%2520Bioluminescent%2520and%2520homogeneous%2520ADP%2520monitoring%2520assay%2520for%2520kinases%26jtitle%3DAssay%2520Drug%2520Dev.%2520Technol.%26date%3D2009%26volume%3D7%26spage%3D560%26epage%3D572%26doi%3D10.1089%2Fadt.2009.0222" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Robers, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dart, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woodroofe, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimprich, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirkland, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Machleidt, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kupcho, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartnett, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimmerman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimmerman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niles, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohana, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daniels, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slater, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cong, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, Y.-Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, K. V.</span></span> <span> </span><span class="NLM_article-title">Target engagement and drug residence time can be observed in living cells with BRET</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">10</span>, <span class="refDoi"> DOI: 10.1038/ncomms10091</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1038%2Fncomms10091" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=1-10&author=M.+B.+Robersauthor=M.+L.+Dartauthor=C.+C.+Woodroofeauthor=C.+A.+Zimprichauthor=T.+A.+Kirklandauthor=T.+Machleidtauthor=K.+R.+Kupchoauthor=S.+Levinauthor=J.+R.+Hartnettauthor=K.+Zimmermanauthor=K.+Zimmermanauthor=A.+L.+Nilesauthor=R.+F.+Ohanaauthor=D.+L.+Danielsauthor=M.+Slaterauthor=M.+G.+Woodauthor=M.+Congauthor=Y.-Q.+Chengauthor=K.+V.+Wood&title=Target+engagement+and+drug+residence+time+can+be+observed+in+living+cells+with+BRET&doi=10.1038%2Fncomms10091"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1038%2Fncomms10091&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms10091%26sid%3Dliteratum%253Aachs%26aulast%3DRobers%26aufirst%3DM.%2BB.%26aulast%3DDart%26aufirst%3DM.%2BL.%26aulast%3DWoodroofe%26aufirst%3DC.%2BC.%26aulast%3DZimprich%26aufirst%3DC.%2BA.%26aulast%3DKirkland%26aufirst%3DT.%2BA.%26aulast%3DMachleidt%26aufirst%3DT.%26aulast%3DKupcho%26aufirst%3DK.%2BR.%26aulast%3DLevin%26aufirst%3DS.%26aulast%3DHartnett%26aufirst%3DJ.%2BR.%26aulast%3DZimmerman%26aufirst%3DK.%26aulast%3DZimmerman%26aufirst%3DK.%26aulast%3DNiles%26aufirst%3DA.%2BL.%26aulast%3DOhana%26aufirst%3DR.%2BF.%26aulast%3DDaniels%26aufirst%3DD.%2BL.%26aulast%3DSlater%26aufirst%3DM.%26aulast%3DWood%26aufirst%3DM.%2BG.%26aulast%3DCong%26aufirst%3DM.%26aulast%3DCheng%26aufirst%3DY.-Q.%26aulast%3DWood%26aufirst%3DK.%2BV.%26atitle%3DTarget%2520engagement%2520and%2520drug%2520residence%2520time%2520can%2520be%2520observed%2520in%2520living%2520cells%2520with%2520BRET%26jtitle%3DNat.%2520Commun.%26date%3D2015%26volume%3D6%26spage%3D1%26epage%3D10%26doi%3D10.1038%2Fncomms10091" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vasta, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corona, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilkinson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimprich, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartnett, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ingold, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimmerman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Machleidt, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirkland, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huwiler, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohana, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slater, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Otto, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cong, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, C. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, B.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanke, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, C. G. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drewry, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, K. V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willson, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meisenheimer, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, K. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robers, M. B.</span></span> <span> </span><span class="NLM_article-title">Quantitative, wide-spectrum kinase profiling in live cells for assessing the effect of cellular ATP on target engagement</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">206</span>– <span class="NLM_lpage">214.e11</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2017.10.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1016%2Fj.chembiol.2017.10.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=29174542" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvVOitbfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=206-214.e11&author=J.+D.+Vastaauthor=C.+R.+Coronaauthor=J.+Wilkinsonauthor=C.+A.+Zimprichauthor=J.+R.+Hartnettauthor=M.+R.+Ingoldauthor=K.+Zimmermanauthor=T.+Machleidtauthor=T.+A.+Kirklandauthor=K.+G.+Huwilerauthor=R.+F.+Ohanaauthor=M.+Slaterauthor=P.+Ottoauthor=M.+Congauthor=C.+I.+Wellsauthor=B.-T.+Bergerauthor=T.+Hankeauthor=C.+G.+K.+Dingauthor=D.+H.+Drewryauthor=K.+V.+M.+Huberauthor=T.+M.+Willsonauthor=S.+Knappauthor=S.+M%C3%BCllerauthor=P.+L.+Meisenheimerauthor=F.+Fanauthor=K.+V.+Woodauthor=M.+B.+Robers&title=Quantitative%2C+wide-spectrum+kinase+profiling+in+live+cells+for+assessing+the+effect+of+cellular+ATP+on+target+engagement&doi=10.1016%2Fj.chembiol.2017.10.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Quantitative, Wide-Spectrum Kinase Profiling in Live Cells for Assessing the Effect of Cellular ATP on Target Engagement</span></div><div class="casAuthors">Vasta, James D.; Corona, Cesear R.; Wilkinson, Jennifer; Zimprich, Chad A.; Hartnett, James R.; Ingold, Morgan R.; Zimmerman, Kristopher; Machleidt, Thomas; Kirkland, Thomas A.; Huwiler, Kristin G.; Ohana, Rachel Friedman; Slater, Michael; Otto, Paul; Cong, Mei; Wells, Carrow I.; Berger, Benedict-Tilman; Hanke, Thomas; Glas, Carina; Ding, Ke; Drewry, David H.; Huber, Kilian V. M.; Willson, Timothy M.; Knapp, Stefan; Muller, Susanne; Meisenheimer, Poncho L.; Fan, Frank; Wood, Keith V.; Robers, Matthew B.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">206-214.e11</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">For kinase inhibitors, intracellular target selectivity is fundamental to pharmacol. mechanism.  Although a no. of acellular techniques have been developed to measure kinase binding or enzymic inhibition, such approaches can fail to accurately predict engagement in cells.  Here we report the application of an energy transfer technique that enabled the first broad-spectrum, equil.-based approach to quant. profile target occupancy and compd. affinity in live cells.  Using this method, we performed a selectivity profiling for clin. relevant kinase inhibitors against 178 full-length kinases, and a mechanistic interrogation of the potency offsets obsd. between cellular and biochem. anal.  For the multikinase inhibitor crizotinib, our approach accurately predicted cellular potency and revealed improved target selectivity compared with biochem. measurements.  Due to cellular ATP, a no. of putative crizotinib targets are unexpectedly disengaged in live cells at a clin. relevant drug dose.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoaPVDw5pU6zrVg90H21EOLACvtfcHk0liLIERu4IhUrA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvVOitbfN&md5=bc3a1f618ab0d2ad6355f31af66d7ef4</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2017.10.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2017.10.010%26sid%3Dliteratum%253Aachs%26aulast%3DVasta%26aufirst%3DJ.%2BD.%26aulast%3DCorona%26aufirst%3DC.%2BR.%26aulast%3DWilkinson%26aufirst%3DJ.%26aulast%3DZimprich%26aufirst%3DC.%2BA.%26aulast%3DHartnett%26aufirst%3DJ.%2BR.%26aulast%3DIngold%26aufirst%3DM.%2BR.%26aulast%3DZimmerman%26aufirst%3DK.%26aulast%3DMachleidt%26aufirst%3DT.%26aulast%3DKirkland%26aufirst%3DT.%2BA.%26aulast%3DHuwiler%26aufirst%3DK.%2BG.%26aulast%3DOhana%26aufirst%3DR.%2BF.%26aulast%3DSlater%26aufirst%3DM.%26aulast%3DOtto%26aufirst%3DP.%26aulast%3DCong%26aufirst%3DM.%26aulast%3DWells%26aufirst%3DC.%2BI.%26aulast%3DBerger%26aufirst%3DB.-T.%26aulast%3DHanke%26aufirst%3DT.%26aulast%3DDing%26aufirst%3DC.%2BG.%2BK.%26aulast%3DDrewry%26aufirst%3DD.%2BH.%26aulast%3DHuber%26aufirst%3DK.%2BV.%2BM.%26aulast%3DWillson%26aufirst%3DT.%2BM.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DMeisenheimer%26aufirst%3DP.%2BL.%26aulast%3DFan%26aufirst%3DF.%26aulast%3DWood%26aufirst%3DK.%2BV.%26aulast%3DRobers%26aufirst%3DM.%2BB.%26atitle%3DQuantitative%252C%2520wide-spectrum%2520kinase%2520profiling%2520in%2520live%2520cells%2520for%2520assessing%2520the%2520effect%2520of%2520cellular%2520ATP%2520on%2520target%2520engagement%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2018%26volume%3D25%26spage%3D206%26epage%3D214.e11%26doi%3D10.1016%2Fj.chembiol.2017.10.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Robers, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friedman-Ohana, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, K. V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kilpatrick, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vasta, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, B.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaudhry, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, K. V.</span></span> <span> </span><span class="NLM_article-title">Quantifying target occupancy of small molecules within living cells</span>. <i>Annu. Rev. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>89</i></span>,  <span class="NLM_fpage">557</span>– <span class="NLM_lpage">581</span>, <span class="refDoi"> DOI: 10.1146/annurev-biochem-011420-092302</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1146%2Fannurev-biochem-011420-092302" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=32208767" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BB3cXlsFCgsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=89&publication_year=2020&pages=557-581&author=M.+B.+Robersauthor=R.+Friedman-Ohanaauthor=K.+V.+M.+Huberauthor=L.+Kilpatrickauthor=J.+D.+Vastaauthor=B.-T.+Bergerauthor=C.+Chaudhryauthor=S.+Hillauthor=S.+M%C3%BCllerauthor=S.+Knappauthor=K.+V.+Wood&title=Quantifying+target+occupancy+of+small+molecules+within+living+cells&doi=10.1146%2Fannurev-biochem-011420-092302"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Quantifying Target Occupancy of Small Molecules Within Living Cells</span></div><div class="casAuthors">Robers, M. B.; Friedman-Ohana, R.; Huber, K. V. M.; Kilpatrick, L.; Vasta, J. D.; Berger, B.-T.; Chaudhry, C.; Hill, S.; Muller, S.; Knapp, S.; Wood, K. V.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Biochemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">89</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">557-581</span>CODEN:
                <span class="NLM_cas:coden">ARBOAW</span>;
        ISSN:<span class="NLM_cas:issn">0066-4154</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews</span>)
        </div><div class="casAbstract">The binding affinity and kinetics of target engagement are fundamental to establishing structure-activity relationships (SARs) for prospective therapeutic agents.  Enhancing these binding parameters for operative targets, while minimizing binding to off-target sites, can translate to improved drug efficacy and a widened therapeutic window.  Compd. activity is typically assessed through modulation of an obsd. phenotype in cultured cells.  Quantifying the corresponding binding properties under common cellular conditions can provide more meaningful interpretation of the cellular SAR anal.  Consequently, methods for assessing drug binding in living cells have advanced and are now integral to medicinal chem. workflows.  In this review, we survey key technol. advancements that support quant. assessments of target occupancy in cultured cells, emphasizing generalizable methodologies able to deliver anal. precision that heretofore required reductionist biochem. approaches.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq1fWsTEZUETbVg90H21EOLACvtfcHk0liLIERu4IhUrA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXlsFCgsrg%253D&md5=38876614c21a3137d1003fcf6744895c</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1146%2Fannurev-biochem-011420-092302&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-biochem-011420-092302%26sid%3Dliteratum%253Aachs%26aulast%3DRobers%26aufirst%3DM.%2BB.%26aulast%3DFriedman-Ohana%26aufirst%3DR.%26aulast%3DHuber%26aufirst%3DK.%2BV.%2BM.%26aulast%3DKilpatrick%26aufirst%3DL.%26aulast%3DVasta%26aufirst%3DJ.%2BD.%26aulast%3DBerger%26aufirst%3DB.-T.%26aulast%3DChaudhry%26aufirst%3DC.%26aulast%3DHill%26aufirst%3DS.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DWood%26aufirst%3DK.%2BV.%26atitle%3DQuantifying%2520target%2520occupancy%2520of%2520small%2520molecules%2520within%2520living%2520cells%26jtitle%3DAnnu.%2520Rev.%2520Biochem.%26date%3D2020%26volume%3D89%26spage%3D557%26epage%3D581%26doi%3D10.1146%2Fannurev-biochem-011420-092302" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Niesen, F. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berglund, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span> <span> </span><span class="NLM_article-title">The use of differential scanning fluorimetry to detect ligand interactions that promote protein stability</span>. <i>Nat. Protoc.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">2212</span>, <span class="refDoi"> DOI: 10.1038/nprot.2007.321</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1038%2Fnprot.2007.321" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=17853878" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFagsb7N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2007&pages=2212&author=F.+H.+Niesenauthor=H.+Berglundauthor=M.+Vedadi&title=The+use+of+differential+scanning+fluorimetry+to+detect+ligand+interactions+that+promote+protein+stability&doi=10.1038%2Fnprot.2007.321"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">The use of differential scanning fluorimetry to detect ligand interactions that promote protein stability</span></div><div class="casAuthors">Niesen, Frank H.; Berglund, Helena; Vedadi, Masoud</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2212-2221</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Differential scanning fluorometry (DSF) is a rapid and inexpensive screening method to identify low-mol.-wt. ligands that bind and stabilize purified proteins.  The temp. at which a protein unfolds is measured by an increase in the fluorescence of a dye with affinity for hydrophobic parts of the protein, which are exposed as the protein unfolds.  A simple fitting procedure allows quick calcn. of the transition midpoint; the difference in the temp. of this midpoint in the presence and absence of ligand is related to the binding affinity of the small mol., which can be a low-mol.-wt. compd., a peptide or a nucleic acid.  DSF is best performed using a conventional real-time PCR instrument.  Ligand solns. from a storage plate are added to a soln. of protein and dye, distributed into the wells of the PCR plate and fluorescence intensity measured as the temp. is raised gradually.  Results can be obtained in a single day.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomViqp6Tq7krVg90H21EOLACvtfcHk0lj2z9bZJbWRvw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFagsb7N&md5=8a74dbb348ab81179614ab4ae5797d94</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2007.321&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2007.321%26sid%3Dliteratum%253Aachs%26aulast%3DNiesen%26aufirst%3DF.%2BH.%26aulast%3DBerglund%26aufirst%3DH.%26aulast%3DVedadi%26aufirst%3DM.%26atitle%3DThe%2520use%2520of%2520differential%2520scanning%2520fluorimetry%2520to%2520detect%2520ligand%2520interactions%2520that%2520promote%2520protein%2520stability%26jtitle%3DNat.%2520Protoc.%26date%3D2007%26volume%3D2%26spage%3D2212%26doi%3D10.1038%2Fnprot.2007.321" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Russell, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, H. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, Y.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neufeld, T. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dillin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guan, K.-L.</span></span> <span> </span><span class="NLM_article-title">ULK1 induces autophagy by phosphorylating Beclin-1 and activating VPS34 lipid kinase</span>. <i>Nat. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">741</span>, <span class="refDoi"> DOI: 10.1038/ncb2757</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1038%2Fncb2757" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=23685627" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BC3sXnvVSitb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2013&pages=741&author=R.+C.+Russellauthor=Y.+Tianauthor=H.+Yuanauthor=H.+W.+Parkauthor=Y.-Y.+Changauthor=J.+Kimauthor=H.+Kimauthor=T.+P.+Neufeldauthor=A.+Dillinauthor=K.-L.+Guan&title=ULK1+induces+autophagy+by+phosphorylating+Beclin-1+and+activating+VPS34+lipid+kinase&doi=10.1038%2Fncb2757"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">ULK1 induces autophagy by phosphorylating Beclin-1 and activating VPS34 lipid kinase</span></div><div class="casAuthors">Russell, Ryan C.; Tian, Ye; Yuan, Haixin; Park, Hyun Woo; Chang, Yu-Yun; Kim, Joungmok; Kim, Haerin; Neufeld, Thomas P.; Dillin, Andrew; Guan, Kun-Liang</div><div class="citationInfo"><span class="NLM_cas:title">Nature Cell Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">741-750</span>CODEN:
                <span class="NLM_cas:coden">NCBIFN</span>;
        ISSN:<span class="NLM_cas:issn">1465-7392</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Autophagy is the primary cellular catabolic program activated in response to nutrient starvation.  Initiation of autophagy, particularly by amino acid withdrawal, requires the ULK kinases.  Despite its pivotal role in autophagy initiation, little is known about the mechanisms by which ULK promotes autophagy.  Here we describe a mol. mechanism linking ULK to the pro-autophagic lipid kinase VPS34.  Following amino acid starvation or mTOR inhibition, the activated ULK1 phosphorylates Beclin-1 on Ser14, thereby enhancing the activity of the ATG14L-contg. VPS34 complexes.  The Beclin-1 Ser14 phosphorylation by ULK is required for full autophagic induction in mammals and this requirement is conserved in Caenorhabditis elegans.  Our study reveals a mol. link from ULK1 to activation of the autophagy-specific VPS34 complex and autophagy induction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqf7cYRCSx3VbVg90H21EOLACvtfcHk0lj2z9bZJbWRvw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXnvVSitb0%253D&md5=380158d2f07518eae293a932cb90a4d1</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1038%2Fncb2757&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncb2757%26sid%3Dliteratum%253Aachs%26aulast%3DRussell%26aufirst%3DR.%2BC.%26aulast%3DTian%26aufirst%3DY.%26aulast%3DYuan%26aufirst%3DH.%26aulast%3DPark%26aufirst%3DH.%2BW.%26aulast%3DChang%26aufirst%3DY.-Y.%26aulast%3DKim%26aufirst%3DJ.%26aulast%3DKim%26aufirst%3DH.%26aulast%3DNeufeld%26aufirst%3DT.%2BP.%26aulast%3DDillin%26aufirst%3DA.%26aulast%3DGuan%26aufirst%3DK.-L.%26atitle%3DULK1%2520induces%2520autophagy%2520by%2520phosphorylating%2520Beclin-1%2520and%2520activating%2520VPS34%2520lipid%2520kinase%26jtitle%3DNat.%2520Cell%2520Biol.%26date%3D2013%26volume%3D15%26spage%3D741%26doi%3D10.1038%2Fncb2757" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lefort, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joffre, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kieffer, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Givel, A.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bourachot, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zago, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bieche, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dubois, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meseure, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vincent-Salomon, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Camonis, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mechta-Grigoriou, F.</span></span> <span> </span><span class="NLM_article-title">Inhibition of autophagy as a new means of improving chemotherapy efficiency in high-LC3B triple-negative breast cancers</span>. <i>Autophagy</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">2122</span>– <span class="NLM_lpage">2142</span>, <span class="refDoi"> DOI: 10.4161/15548627.2014.981788</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.4161%2F15548627.2014.981788" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=25427136" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmtlSns7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=2122-2142&author=S.+Lefortauthor=C.+Joffreauthor=Y.+Kiefferauthor=A.-M.+Givelauthor=B.+Bourachotauthor=G.+Zagoauthor=I.+Biecheauthor=T.+Duboisauthor=D.+Meseureauthor=A.+Vincent-Salomonauthor=J.+Camonisauthor=F.+Mechta-Grigoriou&title=Inhibition+of+autophagy+as+a+new+means+of+improving+chemotherapy+efficiency+in+high-LC3B+triple-negative+breast+cancers&doi=10.4161%2F15548627.2014.981788"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of autophagy as a new means of improving chemotherapy efficiency in high-LC3B triple-negative breast cancers</span></div><div class="casAuthors">Lefort, Sylvain; Joffre, Carine; Kieffer, Yann; Givel, Anne-Marie; Bourachot, Brigitte; Zago, Giulia; Bieche, Ivan; Dubois, Thierry; Meseure, Didier; Vincent-Salomon, Anne; Camonis, Jacques; Mechta-Grigoriou, Fatima</div><div class="citationInfo"><span class="NLM_cas:title">Autophagy</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2122-2142</span>CODEN:
                <span class="NLM_cas:coden">AUTOC9</span>;
        ISSN:<span class="NLM_cas:issn">1554-8627</span>.
    
            (<span class="NLM_cas:orgname">Landes Bioscience</span>)
        </div><div class="casAbstract">The triple-neg. breast cancer (TN BC) subtype is the most aggressive form of invasive BC.  Despite intensive efforts to improve BC treatments, patients with TN BC continue to exhibit poor survival, with half developing resistance to chemotherapy.  Here we identify autophagy as a key mechanism in the progression and chemoresistance of a subset of TN tumors.  We demonstrate that LC3B, a protein involved in autophagosome formation, is a reliable marker of poor prognosis in TN BC, validating this prognostic value at both the mRNA and protein levels in several independent cohorts.  We also show that LC3B has no prognostic value for other BC subtypes (Luminal or HER2 BC), thus revealing a specific impact of autophagy on TN tumors.  Autophagy is essential for the proliferative and invasive properties in 3D of TN BC cells characterized by high LC3B levels.  Interestingly, the activity of the transcriptional co-activator YAP1 (Yes-assocd. protein 1) is regulated by the autophagy process and we identify YAP1 as a new actor in the autophagy-dependent profilerative and invasive properties of high-LC3B TN BC.  Finally, inhibiting autophagy by silencing ATG5 or ATG7 significantly impaired high-LC3B TN tumor growth in vivo.  Moreover, using a patient-derived TN tumor transplanted into mice, we show that an autophagy inhibitor, chloroquine, potentiates the effects of chemotherapeutic agents.  Overall, our data identify LC3B as a new prognostic marker for TN BC and the inhibition of autophagy as a promising therapeutic strategy for TN BC patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoQ2g6Fni8bOrVg90H21EOLACvtfcHk0lj2z9bZJbWRvw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXmtlSns7o%253D&md5=b482b5542af1ae309084b248aa63bb4b</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.4161%2F15548627.2014.981788&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252F15548627.2014.981788%26sid%3Dliteratum%253Aachs%26aulast%3DLefort%26aufirst%3DS.%26aulast%3DJoffre%26aufirst%3DC.%26aulast%3DKieffer%26aufirst%3DY.%26aulast%3DGivel%26aufirst%3DA.-M.%26aulast%3DBourachot%26aufirst%3DB.%26aulast%3DZago%26aufirst%3DG.%26aulast%3DBieche%26aufirst%3DI.%26aulast%3DDubois%26aufirst%3DT.%26aulast%3DMeseure%26aufirst%3DD.%26aulast%3DVincent-Salomon%26aufirst%3DA.%26aulast%3DCamonis%26aufirst%3DJ.%26aulast%3DMechta-Grigoriou%26aufirst%3DF.%26atitle%3DInhibition%2520of%2520autophagy%2520as%2520a%2520new%2520means%2520of%2520improving%2520chemotherapy%2520efficiency%2520in%2520high-LC3B%2520triple-negative%2520breast%2520cancers%26jtitle%3DAutophagy%26date%3D2014%26volume%3D10%26spage%3D2122%26epage%3D2142%26doi%3D10.4161%2F15548627.2014.981788" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liang, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ensor, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaipparettu, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bass, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, J. C.</span></span> <span> </span><span class="NLM_article-title">The autophagy inhibitor chloroquine targets cancer stem cells in triple negative breast cancer by inducing mitochondrial damage and impairing DNA break repair</span>. <i>Cancer Lett.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>376</i></span>,  <span class="NLM_fpage">249</span>– <span class="NLM_lpage">258</span>, <span class="refDoi"> DOI: 10.1016/j.canlet.2016.04.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1016%2Fj.canlet.2016.04.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=27060208" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BC28XlsF2msL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=376&publication_year=2016&pages=249-258&author=D.+H.+Liangauthor=D.+S.+Choiauthor=J.+E.+Ensorauthor=B.+A.+Kaipparettuauthor=B.+L.+Bassauthor=J.+C.+Chang&title=The+autophagy+inhibitor+chloroquine+targets+cancer+stem+cells+in+triple+negative+breast+cancer+by+inducing+mitochondrial+damage+and+impairing+DNA+break+repair&doi=10.1016%2Fj.canlet.2016.04.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">The autophagy inhibitor chloroquine targets cancer stem cells in triple negative breast cancer by inducing mitochondrial damage and impairing DNA break repair</span></div><div class="casAuthors">Liang, Diana H.; Choi, Dong Soon; Ensor, Joe E.; Kaipparettu, Benny A.; Bass, Barbara L.; Chang, Jenny C.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Letters (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">376</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">249-258</span>CODEN:
                <span class="NLM_cas:coden">CALEDQ</span>;
        ISSN:<span class="NLM_cas:issn">0304-3835</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Triple neg. breast cancer (TNBC), characterized by an abundance of treatment-resistant breast cancer stem cells (CSCs), has a poorer prognosis than other types of breast cancers.  Despite its aggressiveness, no effective targeted therapy exists for TNBC.  Here, we demonstrate that CQ effectively targets CSCs via autophagy inhibition, mitochondrial structural damage, and impairment of double-stranded DNA break repair.  Electron microscopy demonstrates CQ-induced mitochondrial cristae damage, which leads to mitochondrial membrane depolarization with a significant redn. in the activity of cytochrome c oxidase and accumulation of superoxide and double-stranded DNA breaks.  CQ effectively diminishes the TNBC cells' ability to metastasize in vitro and in a TNBC xenograft model.  When administered in combination with carboplatin, CQ effectively inhibits carboplatin-induced autophagy.  This combination treatment significantly diminishes the expression of DNA repair proteins in CSC subpopulations, resulting in tumor growth redn. in carboplatin-resistant BRCA1 wild-type TNBC orthotopic xenografts.  As TNBC's high treatment failure rate has been attributed to enrichment of CSCs, CQ, an autophagy inhibitor with anti-CSC effects, may be an effective adjunct to current TNBC chemotherapy regimens with carboplatin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrCa5QjbkmSabVg90H21EOLACvtfcHk0lhKoLZzmiGeRQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XlsF2msL8%253D&md5=1f5df69ea174bfc767ed5e8441528582</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1016%2Fj.canlet.2016.04.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.canlet.2016.04.002%26sid%3Dliteratum%253Aachs%26aulast%3DLiang%26aufirst%3DD.%2BH.%26aulast%3DChoi%26aufirst%3DD.%2BS.%26aulast%3DEnsor%26aufirst%3DJ.%2BE.%26aulast%3DKaipparettu%26aufirst%3DB.%2BA.%26aulast%3DBass%26aufirst%3DB.%2BL.%26aulast%3DChang%26aufirst%3DJ.%2BC.%26atitle%3DThe%2520autophagy%2520inhibitor%2520chloroquine%2520targets%2520cancer%2520stem%2520cells%2520in%2520triple%2520negative%2520breast%2520cancer%2520by%2520inducing%2520mitochondrial%2520damage%2520and%2520impairing%2520DNA%2520break%2520repair%26jtitle%3DCancer%2520Lett.%26date%3D2016%26volume%3D376%26spage%3D249%26epage%3D258%26doi%3D10.1016%2Fj.canlet.2016.04.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gump, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thorburn, A.</span></span> <span> </span><span class="NLM_article-title">Sorting cells for basal and induced autophagic flux by quantitative ratiometric flow cytometry</span>. <i>Autophagy</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1327</span>– <span class="NLM_lpage">1334</span>, <span class="refDoi"> DOI: 10.4161/auto.29394</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.4161%2Fauto.29394" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=24915460" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1Sgu74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=1327-1334&author=J.+M.+Gumpauthor=A.+Thorburn&title=Sorting+cells+for+basal+and+induced+autophagic+flux+by+quantitative+ratiometric+flow+cytometry&doi=10.4161%2Fauto.29394"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Sorting cells for basal and induced autophagic flux by quantitative ratiometric flow cytometry</span></div><div class="casAuthors">Gump, Jacob M.; Thorburn, Andrew</div><div class="citationInfo"><span class="NLM_cas:title">Autophagy</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1327-1334, 8 pp.</span>CODEN:
                <span class="NLM_cas:coden">AUTOC9</span>;
        ISSN:<span class="NLM_cas:issn">1554-8627</span>.
    
            (<span class="NLM_cas:orgname">Landes Bioscience</span>)
        </div><div class="casAbstract">We detail here a protocol using tandem-tagged mCherry-EGFP-LC3 C-G-LC3 to quantify autophagic flux in single cells by ratiometric flow cytometry and to isolate subpopulations of cells based on their relative levels of autophagic flux.  This robust and sensitive method measures autophagic flux rather than autophagosome no. and is an important addn. to the autophagy researcher's array of tools for measuring autophagy.  Two crucial steps in this protocol are (i) generate cells constitutively expressing C-G-LC3 with low to medium fluorescence and low fluorescence variability, and (ii) correctly set up gates and voltage/gain on a properly equipped flow cytometer.  We have used this method to measure autophagic flux in a variety of cell types and exptl. systems using many different autophagy stimuli.  On a sorting flow cytometer, this technique can be used to isolate cells with different levels of basal autophagic flux, or cells with variable induction of flux in response to a given stimulus for further anal. or experimentation.  We have also combined quantification of autophagic flux with methods to measure apoptosis and cell surface proteins, demonstrating the usefulness of this protocol in combination with other flow cytometry labels and markers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrSuu4-b_9wvLVg90H21EOLACvtfcHk0lhKoLZzmiGeRQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1Sgu74%253D&md5=8426adc1181ac63cb49dd8decab71656</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.4161%2Fauto.29394&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252Fauto.29394%26sid%3Dliteratum%253Aachs%26aulast%3DGump%26aufirst%3DJ.%2BM.%26aulast%3DThorburn%26aufirst%3DA.%26atitle%3DSorting%2520cells%2520for%2520basal%2520and%2520induced%2520autophagic%2520flux%2520by%2520quantitative%2520ratiometric%2520flow%2520cytometry%26jtitle%3DAutophagy%26date%3D2014%26volume%3D10%26spage%3D1327%26epage%3D1334%26doi%3D10.4161%2Fauto.29394" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bai, P.</span></span> <span> </span><span class="NLM_article-title">Biology of poly (ADP-ribose) polymerases: the factotums of cell maintenance</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">947</span>– <span class="NLM_lpage">958</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2015.01.034</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1016%2Fj.molcel.2015.01.034" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=26091343" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1SqsbbI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=947-958&author=P.+Bai&title=Biology+of+poly+%28ADP-ribose%29+polymerases%3A+the+factotums+of+cell+maintenance&doi=10.1016%2Fj.molcel.2015.01.034"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Biology of poly(ADP-Ribose) polymerases: The factotums of cell maintenance</span></div><div class="casAuthors">Bai, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">947-958</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">The protein family of poly(ADP-ribose) polymerases (PARPs) or diphtheria toxin-type ADP-ribose transferases (ARTDs) are multidomain proteins originally identified as DNA repair factors.  There are 17 PARP enzymes in humans, and it is now evident that PARPs undertake more tasks than DNA repair.  The aim of this review is to give a comprehensive view of the biol. roles of the PARP family starting from the simplest biochem. reactions to complex regulatory circuits.  Special attention will be laid on discussing linkage of PARP enzymes with tumor biol., oxidative stress, inflammatory, and metabolic diseases.  A better understanding of PARP-mediated processes and pathologies may help in identifying new pathways and, by these, new targets to combat diseases that affect large populations and seriously shorten life expectancy and the quality of life, such as cancer, metabolic, or inflammatory diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOMvLRwxttfbVg90H21EOLACvtfcHk0lhKoLZzmiGeRQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1SqsbbI&md5=5c03dcea525dd8add3b4555bf6f8a144</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2015.01.034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2015.01.034%26sid%3Dliteratum%253Aachs%26aulast%3DBai%26aufirst%3DP.%26atitle%3DBiology%2520of%2520poly%2520%2528ADP-ribose%2529%2520polymerases%253A%2520the%2520factotums%2520of%2520cell%2520maintenance%26jtitle%3DMol.%2520Cell%26date%3D2015%26volume%3D58%26spage%3D947%26epage%3D958%26doi%3D10.1016%2Fj.molcel.2015.01.034" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Krishnakumar, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kraus, W. L.</span></span> <span> </span><span class="NLM_article-title">The PARP side of the nucleus: molecular actions, physiological outcomes, and clinical targets</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">8</span>– <span class="NLM_lpage">24</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2010.06.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1016%2Fj.molcel.2010.06.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=20603072" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BC3cXpsFejsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2010&pages=8-24&author=R.+Krishnakumarauthor=W.+L.+Kraus&title=The+PARP+side+of+the+nucleus%3A+molecular+actions%2C+physiological+outcomes%2C+and+clinical+targets&doi=10.1016%2Fj.molcel.2010.06.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">The PARP side of the nucleus: molecular actions, physiological outcomes, and clinical targets</span></div><div class="casAuthors">Krishnakumar, Raga; Kraus, W. Lee</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">8-24</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  The abundant nuclear enzyme PARP-1, a multifunctional regulator of chromatin structure, transcription, and genomic integrity, plays key roles in a wide variety of processes in the nucleus.  Recent studies have begun to connect the mol. functions of PARP-1 to specific physiol. and pathol. outcomes, many of which can be altered by an expanding array of chem. inhibitors of PARP enzymic activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoy5Mj-wajemLVg90H21EOLACvtfcHk0lhKoLZzmiGeRQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXpsFejsr4%253D&md5=afb9423e09ed68d2bfa589ed0d7b9f13</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2010.06.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2010.06.017%26sid%3Dliteratum%253Aachs%26aulast%3DKrishnakumar%26aufirst%3DR.%26aulast%3DKraus%26aufirst%3DW.%2BL.%26atitle%3DThe%2520PARP%2520side%2520of%2520the%2520nucleus%253A%2520molecular%2520actions%252C%2520physiological%2520outcomes%252C%2520and%2520clinical%2520targets%26jtitle%3DMol.%2520Cell%26date%3D2010%26volume%3D39%26spage%3D8%26epage%3D24%26doi%3D10.1016%2Fj.molcel.2010.06.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rouleau, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hendzel, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaufmann, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poirier, G. G.</span></span> <span> </span><span class="NLM_article-title">PARP inhibition: PARP1 and beyond</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">293</span>– <span class="NLM_lpage">301</span>, <span class="refDoi"> DOI: 10.1038/nrc2812</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1038%2Fnrc2812" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=20200537" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BC3cXis1yms7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2010&pages=293-301&author=M.+Rouleauauthor=A.+Patelauthor=M.+J.+Hendzelauthor=S.+H.+Kaufmannauthor=G.+G.+Poirier&title=PARP+inhibition%3A+PARP1+and+beyond&doi=10.1038%2Fnrc2812"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">PARP inhibition: PARP1 and beyond</span></div><div class="casAuthors">Rouleau, Michele; Patel, Anand; Hendzel, Michael J.; Kaufmann, Scott H.; Poirier, Guy G.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">293-301</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Recent findings have thrust poly(ADP-ribose) polymerases (PARPs) into the limelight as potential chemotherapeutic targets.  To provide a framework for understanding these recent observations, the authors have reviewed what is known about the structures and functions of the family of PARP enzymes, and then outlined a series of questions that should be addressed to guide the rational development of PARP inhibitors as anticancer agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqOiY_n1IClmbVg90H21EOLACvtfcHk0lg3Hc3LlD76Mg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXis1yms7w%253D&md5=fa3e3964e84229317dadbd505f11ad86</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1038%2Fnrc2812&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc2812%26sid%3Dliteratum%253Aachs%26aulast%3DRouleau%26aufirst%3DM.%26aulast%3DPatel%26aufirst%3DA.%26aulast%3DHendzel%26aufirst%3DM.%2BJ.%26aulast%3DKaufmann%26aufirst%3DS.%2BH.%26aulast%3DPoirier%26aufirst%3DG.%2BG.%26atitle%3DPARP%2520inhibition%253A%2520PARP1%2520and%2520beyond%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2010%26volume%3D10%26spage%3D293%26epage%3D301%26doi%3D10.1038%2Fnrc2812" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dréan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lord, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ashworth, A.</span></span> <span> </span><span class="NLM_article-title">PARP inhibitor combination therapy</span>. <i>Crit. Rev. Oncol. Hematol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>108</i></span>,  <span class="NLM_fpage">73</span>– <span class="NLM_lpage">85</span>, <span class="refDoi"> DOI: 10.1016/j.critrevonc.2016.10.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1016%2Fj.critrevonc.2016.10.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=27931843" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A280%3ADC%252BC2sjkvFCgtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=108&publication_year=2016&pages=73-85&author=A.+Dr%C3%A9anauthor=C.+J.+Lordauthor=A.+Ashworth&title=PARP+inhibitor+combination+therapy&doi=10.1016%2Fj.critrevonc.2016.10.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">PARP inhibitor combination therapy</span></div><div class="casAuthors">Drean Amy; Lord Christopher J; Ashworth Alan</div><div class="citationInfo"><span class="NLM_cas:title">Critical reviews in oncology/hematology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">108</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">73-85</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">In 2014, olaparib (Lynparza) became the first PARP (Poly(ADP-ribose) polymerase) inhibitor to be approved for the treatment of cancer.  When used as single agents, PARP inhibitors can selectively target tumour cells with BRCA1 or BRCA2 tumour suppressor gene mutations through synthetic lethality.  However, PARP inhibition also shows considerable promise when used together with other therapeutic agents.  Here, we summarise both the pre-clinical and clinical evidence for the utility of such combinations and discuss the future prospects and challenges for PARP inhibitor combinatorial therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRrc1DGTVYh3rBItaL1WaVAfW6udTcc2eaXr-ePTxEjOrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2sjkvFCgtA%253D%253D&md5=77ba6fe10daf60936688cf320259e996</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1016%2Fj.critrevonc.2016.10.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.critrevonc.2016.10.010%26sid%3Dliteratum%253Aachs%26aulast%3DDr%25C3%25A9an%26aufirst%3DA.%26aulast%3DLord%26aufirst%3DC.%2BJ.%26aulast%3DAshworth%26aufirst%3DA.%26atitle%3DPARP%2520inhibitor%2520combination%2520therapy%26jtitle%3DCrit.%2520Rev.%2520Oncol.%2520Hematol.%26date%3D2016%26volume%3D108%26spage%3D73%26epage%3D85%26doi%3D10.1016%2Fj.critrevonc.2016.10.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zai, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ju, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span> <span> </span><span class="NLM_article-title">Targeting PARP and autophagy evoked synergistic lethality in hepatocellular carcinoma</span>. <i>Carcinogenesis</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">345</span>– <span class="NLM_lpage">357</span>, <span class="refDoi"> DOI: 10.1093/carcin/bgz104</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1093%2Fcarcin%2Fbgz104" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2019&pages=345-357&author=W.+Zaiauthor=W.+Chenauthor=Y.+Hanauthor=Z.+Wuauthor=J.+Fanauthor=X.+Zhangauthor=J.+Luanauthor=S.+Tangauthor=X.+Jinauthor=X.+Fuauthor=H.+Gaoauthor=D.+Juauthor=H.+Liu&title=Targeting+PARP+and+autophagy+evoked+synergistic+lethality+in+hepatocellular+carcinoma&doi=10.1093%2Fcarcin%2Fbgz104"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1093%2Fcarcin%2Fbgz104&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fcarcin%252Fbgz104%26sid%3Dliteratum%253Aachs%26aulast%3DZai%26aufirst%3DW.%26aulast%3DChen%26aufirst%3DW.%26aulast%3DHan%26aufirst%3DY.%26aulast%3DWu%26aufirst%3DZ.%26aulast%3DFan%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DLuan%26aufirst%3DJ.%26aulast%3DTang%26aufirst%3DS.%26aulast%3DJin%26aufirst%3DX.%26aulast%3DFu%26aufirst%3DX.%26aulast%3DGao%26aufirst%3DH.%26aulast%3DJu%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DH.%26atitle%3DTargeting%2520PARP%2520and%2520autophagy%2520evoked%2520synergistic%2520lethality%2520in%2520hepatocellular%2520carcinoma%26jtitle%3DCarcinogenesis%26date%3D2019%26volume%3D41%26spage%3D345%26epage%3D357%26doi%3D10.1093%2Fcarcin%2Fbgz104" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huo, Z.</span></span> <span> </span><span class="NLM_article-title">Targeting autophagy potentiates the anti-tumor effect of PARP inhibitor in pediatric chronic myeloid leukemia</span>. <i>AMB Express</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">108</span>, <span class="refDoi"> DOI: 10.1186/s13568-019-0836-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1186%2Fs13568-019-0836-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=31309361" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A280%3ADC%252BB3MznslensQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2019&pages=108&author=Y.+Liuauthor=H.+Songauthor=H.+Songauthor=X.+Fengauthor=C.+Zhouauthor=Z.+Huo&title=Targeting+autophagy+potentiates+the+anti-tumor+effect+of+PARP+inhibitor+in+pediatric+chronic+myeloid+leukemia&doi=10.1186%2Fs13568-019-0836-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting autophagy potentiates the anti-tumor effect of PARP inhibitor in pediatric chronic myeloid leukemia</span></div><div class="casAuthors">Liu Yuanyuan; Song Hong; Song Huanqing; Feng Xiaoxia; Zhou Chuan; Huo Zhifang</div><div class="citationInfo"><span class="NLM_cas:title">AMB Express</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">108</span>
        ISSN:<span class="NLM_cas:issn">2191-0855</span>.
    </div><div class="casAbstract">Due to its potent cytotoxicity in BRCA-mutated tumors, synthetic lethality elicited by poly (ADP-ribose) polymerase (PARP) inhibitor gives renewed enthusiasm to researching and developing anti-cancer therapies.  Chronic myeloid leukemia (CML) is a type of cancers that starts in certain blood-forming cells of the bone marrow.  Here, we showed that poly (ADP-ribose) polymerase (PARP) inhibitor talazoparib could induce a concentration-dependent cytotoxicity in CML cells derived from pediatric patients.  During talazoparib treatment, autophagy was markedly activated, which was confirmed by the accumulation of autophagosomes, decrease of SQSTM1 and up-regulation of LC3-II.  Inhibition of autophagy by pharmaceutical inhibitor chloroquine or small-interfering RNA siATG5 significantly increased the cytotoxicity of talazoparib in pediatric CML cells and elicited synergistic anti-tumor effect in patient-derived xenograft model.  Our data demonstrated that autophagy played a cyto-protective role in talazoparib-treated pediatric CML and co-treatment with talazoparib and autophagy inhibitor could induce synergetic anti-tumor effect, providing novel insights for pediatric CML treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRc23_CSyS503T4ugvCDgnZfW6udTcc2eaXr-ePTxEjOrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3MznslensQ%253D%253D&md5=9252afe6c9793e6e23a35ba9da35ad5c</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1186%2Fs13568-019-0836-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs13568-019-0836-z%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DSong%26aufirst%3DH.%26aulast%3DSong%26aufirst%3DH.%26aulast%3DFeng%26aufirst%3DX.%26aulast%3DZhou%26aufirst%3DC.%26aulast%3DHuo%26aufirst%3DZ.%26atitle%3DTargeting%2520autophagy%2520potentiates%2520the%2520anti-tumor%2520effect%2520of%2520PARP%2520inhibitor%2520in%2520pediatric%2520chronic%2520myeloid%2520leukemia%26jtitle%3DAMB%2520Express%26date%3D2019%26volume%3D9%26spage%3D108%26doi%3D10.1186%2Fs13568-019-0836-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Di
Veroli, G. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fornari, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mollard, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bramhall, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richards, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jodrell, D. I.</span></span> <span> </span><span class="NLM_article-title">Combenefit: an interactive platform for the analysis and visualization of drug combinations</span>. <i>Bioinformatics</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">2866</span>– <span class="NLM_lpage">2868</span>, <span class="refDoi"> DOI: 10.1093/bioinformatics/btw230</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1093%2Fbioinformatics%2Fbtw230" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=27153664" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtFGlurbO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2016&pages=2866-2868&author=G.+Y.+Di%0AVeroliauthor=C.+Fornariauthor=D.+Wangauthor=S.+Mollardauthor=J.+L.+Bramhallauthor=F.+M.+Richardsauthor=D.+I.+Jodrell&title=Combenefit%3A+an+interactive+platform+for+the+analysis+and+visualization+of+drug+combinations&doi=10.1093%2Fbioinformatics%2Fbtw230"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Combenefit: an interactive platform for the analysis and visualization of drug combinations</span></div><div class="casAuthors">Di Veroli, Giovanni Y.; Fornari, Chiara; Wang, Dennis; Mollard, Severine; Bramhall, Jo L.; Richards, Frances M.; Jodrell, Duncan I.</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">2866-2868</span>CODEN:
                <span class="NLM_cas:coden">BOINFP</span>;
        ISSN:<span class="NLM_cas:issn">1367-4803</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Motivation: Many drug combinations are routinely assessed to identify synergistic interactions in the attempt to develop novel treatment strategies.  Appropriate software is required to analyze the results of these studies.  Results: We present Combenefit, new free software tool that enables the visualization, anal. and quantification of drug combination effects in terms of synergy and/or antagonism.  Data from combinations assays can be processed using classical Synergy models (Loewe, Bliss, HSA), as single expts. or in batch for High Throughput Screens.  This user-friendly tool provides lab. scientists with an easy and systematic way to analyze their data.  The companion package provides bioinformaticians with crit. implementations of routines enabling the processing of combination data.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-sh5U3Gp4wrVg90H21EOLACvtfcHk0liL2lWnVflIvg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtFGlurbO&md5=97422dbe25c99eda4f7a8a61f775dd1e</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtw230&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtw230%26sid%3Dliteratum%253Aachs%26aulast%3DDi%2BVeroli%26aufirst%3DG.%2BY.%26aulast%3DFornari%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DD.%26aulast%3DMollard%26aufirst%3DS.%26aulast%3DBramhall%26aufirst%3DJ.%2BL.%26aulast%3DRichards%26aufirst%3DF.%2BM.%26aulast%3DJodrell%26aufirst%3DD.%2BI.%26atitle%3DCombenefit%253A%2520an%2520interactive%2520platform%2520for%2520the%2520analysis%2520and%2520visualization%2520of%2520drug%2520combinations%26jtitle%3DBioinformatics%26date%3D2016%26volume%3D32%26spage%3D2866%26epage%3D2868%26doi%3D10.1093%2Fbioinformatics%2Fbtw230" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Papadimitriou, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mountzios, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Papadimitriou, C. A.</span></span> <span> </span><span class="NLM_article-title">The role of PARP inhibition in triple-negative breast cancer: unraveling the wide spectrum of synthetic lethality</span>. <i>Cancer Treat. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">34</span>– <span class="NLM_lpage">44</span>, <span class="refDoi"> DOI: 10.1016/j.ctrv.2018.04.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1016%2Fj.ctrv.2018.04.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=29753961" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BC1cXptVCrtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2018&pages=34-44&author=M.+Papadimitriouauthor=G.+Mountziosauthor=C.+A.+Papadimitriou&title=The+role+of+PARP+inhibition+in+triple-negative+breast+cancer%3A+unraveling+the+wide+spectrum+of+synthetic+lethality&doi=10.1016%2Fj.ctrv.2018.04.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">The role of PARP inhibition in triple-negative breast cancer: Unraveling the wide spectrum of synthetic lethality</span></div><div class="casAuthors">Papadimitriou, Marios; Mountzios, Giannis; Papadimitriou, Christos A.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Treatment Reviews</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">34-44</span>CODEN:
                <span class="NLM_cas:coden">CTREDJ</span>;
        ISSN:<span class="NLM_cas:issn">0305-7372</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Triple-neg. breast cancer (TNBC) accounts for approx. 15-20% of all breast cancers and is characterized by a lack of immunohistochem. expression of estrogen receptors (ER), progesterone receptors (PR) and HER2.  TNBC is assocd. with poor long-term outcomes compared with other breast cancer subtypes.  Many of these tumors are also basal-like cancers which are characterized by an aggressive biol. behavior with a distant recurrence peak obsd. early at 3 years following diagnosis.  Furthermore, metastatic TNBC bears a dismal prognosis with an av. survival of 12 mo.  Although the prevalence of genetic alterations among women with TNBC differs significantly by ethnicity, race and age, BRCA mutations (including both germline mutations and somatic genetic aberrations) are found in up to 20-25% of unselected patients and esp. in those of the basal-like immunophenotype.  Therefore, defects in the DNA repair pathway could represent a promising therapeutic target for this subgroup of TNBC patients.  Poly(ADP-ribose) polymerase (PARP) inhibitors exploit this deficiency through synthetic lethality and have emerged as promising anticancer therapies, esp. in BRCA1 or BRCA2 mutation carriers.  Several PARP inhibitors are currently being evaluated in the adjuvant, neo-adjuvant, and metastatic setting for the treatment of breast cancer patients with a deficient homologous recombination pathway.  In this article, we review the major mol. characteristics of TNBC, the mechanisms of homologous recombination, and the role of PARP inhibition as an emerging therapeutic strategy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrH50CyPikNqLVg90H21EOLACvtfcHk0liL2lWnVflIvg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXptVCrtL8%253D&md5=e86a097ea52737257618cf18b8c05c7e</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1016%2Fj.ctrv.2018.04.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ctrv.2018.04.010%26sid%3Dliteratum%253Aachs%26aulast%3DPapadimitriou%26aufirst%3DM.%26aulast%3DMountzios%26aufirst%3DG.%26aulast%3DPapadimitriou%26aufirst%3DC.%2BA.%26atitle%3DThe%2520role%2520of%2520PARP%2520inhibition%2520in%2520triple-negative%2520breast%2520cancer%253A%2520unraveling%2520the%2520wide%2520spectrum%2520of%2520synthetic%2520lethality%26jtitle%3DCancer%2520Treat.%2520Rev.%26date%3D2018%26volume%3D67%26spage%3D34%26epage%3D44%26doi%3D10.1016%2Fj.ctrv.2018.04.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kinsey, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Camolotto, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boespflug, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guillen, K. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foth, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Truong, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuman, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shea, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seipp, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yap, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burrell, L. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lum, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whisenant, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilcrease, G. W.,  III.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cavalieri, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rehbein, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cutler, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Affolter, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Welm, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Welm, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scaife, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snyder, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McMahon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomar, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Papke, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hobbs, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayes, T. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diehl, J. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goode, G. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaika, N. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Witkiewicz, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knudsen, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petricoin, E. F.,  III</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macdonald, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tran, N. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lyssiotis, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ying, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kimmelman, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cox, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Der, C. J.</span></span> <span> </span><span class="NLM_article-title">Protective autophagy elicited by RAF→ MEK→ ERK inhibition suggests a treatment strategy for RAS-driven cancers</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">620</span>, <span class="refDoi"> DOI: 10.1038/s41591-019-0367-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1038%2Fs41591-019-0367-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=30833748" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BC1MXmsV2isbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2019&pages=620&author=C.+G.+Kinseyauthor=S.+A.+Camolottoauthor=A.+M.+Boespflugauthor=K.+P.+Guillenauthor=M.+Fothauthor=A.+Truongauthor=S.+S.+Schumanauthor=J.+E.+Sheaauthor=M.+T.+Seippauthor=J.+T.+Yapauthor=L.+D.+Burrellauthor=D.+H.+Lumauthor=J.+R.+Whisenantauthor=G.+W.+Gilcreaseauthor=C.+C.+Cavalieriauthor=K.+M.+Rehbeinauthor=S.+L.+Cutlerauthor=K.+E.+Affolterauthor=A.+L.+Welmauthor=B.+E.+Welmauthor=C.+L.+Scaifeauthor=E.+L.+Snyderauthor=M.+McMahonauthor=G.+Tomarauthor=B.+Papkeauthor=G.+A.+Hobbsauthor=L.+Yanauthor=T.+K.+Hayesauthor=J.+N.+Diehlauthor=G.+D.+Goodeauthor=N.+V.+Chaikaauthor=Y.+Wangauthor=G.-F.+Zhangauthor=A.+K.+Witkiewiczauthor=E.+S.+Knudsenauthor=E.+F.+Petricoinauthor=P.+K.+Singhauthor=J.+M.+Macdonaldauthor=N.+L.+Tranauthor=C.+A.+Lyssiotisauthor=H.+Yingauthor=A.+C.+Kimmelmanauthor=A.+D.+Coxauthor=C.+J.+Der&title=Protective+autophagy+elicited+by+RAF%E2%86%92+MEK%E2%86%92+ERK+inhibition+suggests+a+treatment+strategy+for+RAS-driven+cancers&doi=10.1038%2Fs41591-019-0367-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Protective autophagy elicited by RAF→MEK→ERK inhibition suggests a treatment strategy for RAS-driven cancers</span></div><div class="casAuthors">Kinsey, Conan G.; Camolotto, Soledad A.; Boespflug, Amelie M.; Gullien, Katrin P.; Foth, Mona; Truong, Amanda; Schuman, Sophia S.; Shea, Jill E.; Seipp, Michael T.; Yap, Jeffrey T.; Burrell, Lance D.; Lum, David H.; Whisenant, Jonathan R.; Gilcrease, G. Weldon, III; Cavalieri, Courtney C.; Rehbein, Kaitrin M.; Cutler, Stephanie L.; Affolter, Kajsa E.; Welm, Alana L.; Welm, Bryan E.; Scaife, Courtney L.; Snyder, Eric L.; McMahon, Martin</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">620-627</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Pancreatic ductal adenocarcinoma (PDA) was responsible for ∼ 44,000 deaths in the United States in 2018 and is the epitome of a recalcitrant cancer driven by a pharmacol. intractable oncoprotein, KRAS1-4.  Downstream of KRAS, the RAF→MEK→ERK signaling pathway plays a central role in pancreatic carcinogenesis5.  However, paradoxically, inhibition of this pathway has provided no clin. benefit to patients with PDA6.  Here we show that inhibition of KRAS→RAF→MEK→ERK signaling elicits autophagy, a process of cellular recycling that protects PDA cells from the cytotoxic effects of KRAS pathway inhibition.  Mechanistically, inhibition of MEK1/2 leads to activation of the LKB1→AMPK→ULK1 signaling axis, a key regulator of autophagy.  Furthermore, combined inhibition of MEK1/2 plus autophagy displays synergistic anti-proliferative effects against PDA cell lines in vitro and promotes regression of xenografted patient-derived PDA tumors in mice.  The obsd. effect of combination trametinib plus chloroquine was not restricted to PDA as other tumors, including patient-derived xenografts (PDX) of NRAS-mutated melanoma and BRAF-mutated colorectal cancer displayed similar responses.  Finally, treatment of a patient with PDA with the combination of trametinib plus hydroxychloroquine resulted in a partial, but nonetheless striking disease response.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqYfVZ8qCSszrVg90H21EOLACvtfcHk0liL2lWnVflIvg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXmsV2isbw%253D&md5=0fa2c90810c3d9bc403f2f119389adbe</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1038%2Fs41591-019-0367-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41591-019-0367-9%26sid%3Dliteratum%253Aachs%26aulast%3DKinsey%26aufirst%3DC.%2BG.%26aulast%3DCamolotto%26aufirst%3DS.%2BA.%26aulast%3DBoespflug%26aufirst%3DA.%2BM.%26aulast%3DGuillen%26aufirst%3DK.%2BP.%26aulast%3DFoth%26aufirst%3DM.%26aulast%3DTruong%26aufirst%3DA.%26aulast%3DSchuman%26aufirst%3DS.%2BS.%26aulast%3DShea%26aufirst%3DJ.%2BE.%26aulast%3DSeipp%26aufirst%3DM.%2BT.%26aulast%3DYap%26aufirst%3DJ.%2BT.%26aulast%3DBurrell%26aufirst%3DL.%2BD.%26aulast%3DLum%26aufirst%3DD.%2BH.%26aulast%3DWhisenant%26aufirst%3DJ.%2BR.%26aulast%3DGilcrease%26aufirst%3DG.%2BW.%26aulast%3DCavalieri%26aufirst%3DC.%2BC.%26aulast%3DRehbein%26aufirst%3DK.%2BM.%26aulast%3DCutler%26aufirst%3DS.%2BL.%26aulast%3DAffolter%26aufirst%3DK.%2BE.%26aulast%3DWelm%26aufirst%3DA.%2BL.%26aulast%3DWelm%26aufirst%3DB.%2BE.%26aulast%3DScaife%26aufirst%3DC.%2BL.%26aulast%3DSnyder%26aufirst%3DE.%2BL.%26aulast%3DMcMahon%26aufirst%3DM.%26aulast%3DTomar%26aufirst%3DG.%26aulast%3DPapke%26aufirst%3DB.%26aulast%3DHobbs%26aufirst%3DG.%2BA.%26aulast%3DYan%26aufirst%3DL.%26aulast%3DHayes%26aufirst%3DT.%2BK.%26aulast%3DDiehl%26aufirst%3DJ.%2BN.%26aulast%3DGoode%26aufirst%3DG.%2BD.%26aulast%3DChaika%26aufirst%3DN.%2BV.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DG.-F.%26aulast%3DWitkiewicz%26aufirst%3DA.%2BK.%26aulast%3DKnudsen%26aufirst%3DE.%2BS.%26aulast%3DPetricoin%26aufirst%3DE.%2BF.%26aulast%3DSingh%26aufirst%3DP.%2BK.%26aulast%3DMacdonald%26aufirst%3DJ.%2BM.%26aulast%3DTran%26aufirst%3DN.%2BL.%26aulast%3DLyssiotis%26aufirst%3DC.%2BA.%26aulast%3DYing%26aufirst%3DH.%26aulast%3DKimmelman%26aufirst%3DA.%2BC.%26aulast%3DCox%26aufirst%3DA.%2BD.%26aulast%3DDer%26aufirst%3DC.%2BJ.%26atitle%3DProtective%2520autophagy%2520elicited%2520by%2520RAF%25E2%2586%2592%2520MEK%25E2%2586%2592%2520ERK%2520inhibition%2520suggests%2520a%2520treatment%2520strategy%2520for%2520RAS-driven%2520cancers%26jtitle%3DNat.%2520Med.%26date%3D2019%26volume%3D25%26spage%3D620%26doi%3D10.1038%2Fs41591-019-0367-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bryant, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stalnecker, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeitouni, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klomp, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tikunov, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gunda, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pierobon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waters, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">George, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomar, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Papke, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hobbs, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayes, T. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diehl, J. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goode, G. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaika, N. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Witkiewicz, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knudsen, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petricoin, E. F.,  III</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macdonald, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tran, N. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lyssiotis, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ying, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kimmelman, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cox, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De, C. J.</span></span> <span> </span><span class="NLM_article-title">Combination of ERK and autophagy inhibition as a treatment approach for pancreatic cancer</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">628</span>– <span class="NLM_lpage">640</span>, <span class="refDoi"> DOI: 10.1038/s41591-019-0368-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1038%2Fs41591-019-0368-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=30833752" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BC1MXmsV2isbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2019&pages=628-640&author=K.+L.+Bryantauthor=C.+A.+Stalneckerauthor=D.+Zeitouniauthor=J.+E.+Klompauthor=S.+Pengauthor=A.+P.+Tikunovauthor=V.+Gundaauthor=M.+Pierobonauthor=A.+M.+Watersauthor=S.+D.+Georgeauthor=G.+Tomarauthor=B.+Papkeauthor=G.+A.+Hobbsauthor=L.+Yanauthor=T.+K.+Hayesauthor=J.+N.+Diehlauthor=G.+D.+Goodeauthor=N.+V.+Chaikaauthor=Y.+Wangauthor=G.-F.+Zhangauthor=A.+K.+Witkiewiczauthor=E.+S.+Knudsenauthor=E.+F.+Petricoinauthor=P.+K.+Singhauthor=J.+M.+Macdonaldauthor=N.+L.+Tranauthor=C.+A.+Lyssiotisauthor=H.+Yingauthor=A.+C.+Kimmelmanauthor=A.+D.+Coxauthor=C.+J.+De&title=Combination+of+ERK+and+autophagy+inhibition+as+a+treatment+approach+for+pancreatic+cancer&doi=10.1038%2Fs41591-019-0368-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Combination of ERK and autophagy inhibition as a treatment approach for pancreatic cancer</span></div><div class="casAuthors">Bryant, Kirsten L.; Stalnecker, Clint A.; Zeitouni, Daniel; Klomp, Jennifer E.; Peng, Sen; Tikunov, Andrey P.; Gunda, Venugopal; Pierobon, Mariaelena; Waters, Andrew M.; George, Samuel D.; Tomar, Garima; Papke, Bjorn; Hobbs, G. Aaron; Yan, Liang; Hayes, Tikvah K.; Diehl, J. Nathaniel; Goode, Gennifer D.; Chaika, Nina V.; Wang, Yingxue; Zhang, Guo-Fang; Witkiewicz, Agnieszka K.; Knudsen, Erik S.; Petricoin, Emanuel F., III; Singh, Pankaj K.; Macdonald, Jeffrey M.; Tran, Nhan L.; Lyssiotis, Costas A.; Ying, Haoqiang; Kimmelman, Alec C.; Cox, Adrienne D.; Der, Channing J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">628-640</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Pancreatic ductal adenocarcinoma (PDAC) is characterized by KRAS- and autophagy-dependent tumorigenic growth, but the role of KRAS in supporting autophagy has not been established.  We show that, to our surprise, suppression of KRAS increased autophagic flux, as did pharmacol. inhibition of its effector ERK MAPK.  Furthermore, we demonstrate that either KRAS suppression or ERK inhibition decreased both glycolytic and mitochondrial functions.  We speculated that ERK inhibition might thus enhance PDAC dependence on autophagy, in part by impairing other KRAS- or ERK-driven metabolic processes.  Accordingly, we found that the autophagy inhibitor chloroquine and genetic or pharmacol. inhibition of specific autophagy regulators synergistically enhanced the ability of ERK inhibitors to mediate antitumor activity in KRAS-driven PDAC.  We conclude that combinations of pharmacol. inhibitors that concurrently block both ERK MAPK and autophagic processes that are upregulated in response to ERK inhibition may be effective treatments for PDAC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkazpCczVUqLVg90H21EOLACvtfcHk0lgm5bkiImrYnA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXmsV2isbo%253D&md5=120142384aa8e884aa694f8490cefbe6</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1038%2Fs41591-019-0368-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41591-019-0368-8%26sid%3Dliteratum%253Aachs%26aulast%3DBryant%26aufirst%3DK.%2BL.%26aulast%3DStalnecker%26aufirst%3DC.%2BA.%26aulast%3DZeitouni%26aufirst%3DD.%26aulast%3DKlomp%26aufirst%3DJ.%2BE.%26aulast%3DPeng%26aufirst%3DS.%26aulast%3DTikunov%26aufirst%3DA.%2BP.%26aulast%3DGunda%26aufirst%3DV.%26aulast%3DPierobon%26aufirst%3DM.%26aulast%3DWaters%26aufirst%3DA.%2BM.%26aulast%3DGeorge%26aufirst%3DS.%2BD.%26aulast%3DTomar%26aufirst%3DG.%26aulast%3DPapke%26aufirst%3DB.%26aulast%3DHobbs%26aufirst%3DG.%2BA.%26aulast%3DYan%26aufirst%3DL.%26aulast%3DHayes%26aufirst%3DT.%2BK.%26aulast%3DDiehl%26aufirst%3DJ.%2BN.%26aulast%3DGoode%26aufirst%3DG.%2BD.%26aulast%3DChaika%26aufirst%3DN.%2BV.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DG.-F.%26aulast%3DWitkiewicz%26aufirst%3DA.%2BK.%26aulast%3DKnudsen%26aufirst%3DE.%2BS.%26aulast%3DPetricoin%26aufirst%3DE.%2BF.%26aulast%3DSingh%26aufirst%3DP.%2BK.%26aulast%3DMacdonald%26aufirst%3DJ.%2BM.%26aulast%3DTran%26aufirst%3DN.%2BL.%26aulast%3DLyssiotis%26aufirst%3DC.%2BA.%26aulast%3DYing%26aufirst%3DH.%26aulast%3DKimmelman%26aufirst%3DA.%2BC.%26aulast%3DCox%26aufirst%3DA.%2BD.%26aulast%3DDe%26aufirst%3DC.%2BJ.%26atitle%3DCombination%2520of%2520ERK%2520and%2520autophagy%2520inhibition%2520as%2520a%2520treatment%2520approach%2520for%2520pancreatic%2520cancer%26jtitle%3DNat.%2520Med.%26date%3D2019%26volume%3D25%26spage%3D628%26epage%3D640%26doi%3D10.1038%2Fs41591-019-0368-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rebecca, V. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amaravadi, R. K.</span></span> <span> </span><span class="NLM_article-title">Emerging strategies to effectively target autophagy in cancer</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">11</span>, <span class="refDoi"> DOI: 10.1038/onc.2015.99</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1038%2Fonc.2015.99" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=25893285" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BC28XjtFKktw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2016&pages=1-11&author=V.+W.+Rebeccaauthor=R.+K.+Amaravadi&title=Emerging+strategies+to+effectively+target+autophagy+in+cancer&doi=10.1038%2Fonc.2015.99"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Emerging strategies to effectively target autophagy in cancer</span></div><div class="casAuthors">Rebecca, V. W.; Amaravadi, R. K.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-11</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Autophagy serves a dichotomous role in cancer and recent advances have helped delineate the appropriate settings where inhibiting or promoting autophagy may confer therapeutic efficacy in patients.  Our evolving understanding of the mol. machinery responsible for the tightly controlled regulation of this homeostatic mechanism has begun to bear fruit in the way of autophagy-oriented clin. trials and promising lead compds. to modulate autophagy for therapeutic benefit.  In this manuscript we review the recent preclin. and clin. therapeutic strategies that involve autophagy modulation in cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqSXqZO1qEuorVg90H21EOLACvtfcHk0lgm5bkiImrYnA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XjtFKktw%253D%253D&md5=2d316e630be8c239e72b820b63c9a969</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1038%2Fonc.2015.99&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fonc.2015.99%26sid%3Dliteratum%253Aachs%26aulast%3DRebecca%26aufirst%3DV.%2BW.%26aulast%3DAmaravadi%26aufirst%3DR.%2BK.%26atitle%3DEmerging%2520strategies%2520to%2520effectively%2520target%2520autophagy%2520in%2520cancer%26jtitle%3DOncogene%26date%3D2016%26volume%3D35%26spage%3D1%26epage%3D11%26doi%3D10.1038%2Fonc.2015.99" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kabsch, W.</span></span> <span> </span><span class="NLM_article-title">Xds</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">125</span>– <span class="NLM_lpage">132</span>, <span class="refDoi"> DOI: 10.1107/S0907444909047337</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1107%2FS0907444909047337" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=20124692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=125-132&author=W.+Kabsch&title=Xds&doi=10.1107%2FS0907444909047337"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Software XDS for image rotation, recognition and crystal symmetry assignment</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">125-132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The usage and control of recent modifications of the program package XDS for the processing of rotation images are described in the context of previous versions.  New features include automatic detn. of spot size and reflecting range and recognition and assignment of crystal symmetry.  Moreover, the limitations of earlier package versions on the no. of correction/scaling factors and the representation of pixel contents have been removed.  Large program parts have been restructured for parallel processing so that the quality and completeness of collected data can be assessed soon after measurement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv5AyUw00DYbVg90H21EOLACvtfcHk0lg4GDa0Ut3jFg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D&md5=1aa9a38aeb3ce95af4ffb7d8b8a142bd</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047337%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DXds%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D125%26epage%3D132%26doi%3D10.1107%2FS0907444909047337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Evans, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span> <span> </span><span class="NLM_article-title">How good are my data and what is the resolution?</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>69</i></span>,  <span class="NLM_fpage">1204</span>– <span class="NLM_lpage">1214</span>, <span class="refDoi"> DOI: 10.1107/S0907444913000061</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1107%2FS0907444913000061" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=23793146" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BC3sXpvVejsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2013&pages=1204-1214&author=P.+R.+Evansauthor=G.+N.+Murshudov&title=How+good+are+my+data+and+what+is+the+resolution%3F&doi=10.1107%2FS0907444913000061"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">How good are my data and what is the resolution?</span></div><div class="casAuthors">Evans, Philip R.; Murshudov, Garib N.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1204-1214</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Following integration of the obsd. diffraction spots, the process of  data redn.' initially aims to det. the point-group symmetry of the data and the likely space group.  This can be performed with the program POINTLESS.  The scaling program then puts all the measurements on a common scale, avs. measurements of symmetry-related reflections (using the symmetry detd. previously) and produces many statistics that provide the first important measures of data quality.  A new scaling program, AIMLESS, implements scaling models similar to those in SCALA but adds some addnl. analyses.  From the analyses, a no. of decisions can be made about the quality of the data and whether some measurements should be discarded.  The effective  resoln.' of a data set is a difficult and possibly contentious question (particularly with referees of papers) and this is discussed in the light of tests comparing the data-processing statistics with trials of refinement against obsd. and simulated data, and automated model-building and comparison of maps calcd. with different resoln. limits.  These trials show that adding weak high-resoln. data beyond the commonly used limits may make some improvement and does no harm.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreOyPAR8GLBrVg90H21EOLACvtfcHk0lg4GDa0Ut3jFg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXpvVejsL8%253D&md5=3f649426c21e9d423f94ef8d7c568233</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1107%2FS0907444913000061&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444913000061%26sid%3Dliteratum%253Aachs%26aulast%3DEvans%26aufirst%3DP.%2BR.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26atitle%3DHow%2520good%2520are%2520my%2520data%2520and%2520what%2520is%2520the%2520resolution%253F%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2013%26volume%3D69%26spage%3D1204%26epage%3D1214%26doi%3D10.1107%2FS0907444913000061" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span> <span> </span><span class="NLM_article-title">Acknowledging errors: advanced molecular replacement with Phaser</span>. In  <i>Protein Crystallography</i>; <span class="NLM_publisher-name">Springer: Humana Press</span>: <span class="NLM_publisher-loc">New York, NY</span>, <span class="NLM_year">2017</span>; pp.  <span class="NLM_fpage">421</span>– <span class="NLM_lpage">453</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1007%2F978-1-4939-7000-1_18" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&pages=421-453&author=A.+J.+McCoy&title=Protein+Crystallography"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1007%2F978-1-4939-7000-1_18&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252F978-1-4939-7000-1_18%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26atitle%3DAcknowledging%2520errors%253A%2520advanced%2520molecular%2520replacement%2520with%2520Phaser%26btitle%3DProtein%2520Crystallography%26pub%3DSpringer%253A%2520Humana%2520Press%26date%3D2017%26spage%3D421%26epage%3D453" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span> <span> </span><span class="NLM_article-title">Tools for ligand validation in Coot</span>. <i>Acta Crystallogr., Sect. D: Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>73</i></span>,  <span class="NLM_fpage">203</span>– <span class="NLM_lpage">210</span>, <span class="refDoi"> DOI: 10.1107/S2059798317003382</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1107%2FS2059798317003382" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=28362283" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BC2sXktFyrsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2017&pages=203-210&author=P.+Emsley&title=Tools+for+ligand+validation+in+Coot&doi=10.1107%2FS2059798317003382"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Tools for ligand validation in Coot</span></div><div class="casAuthors">Emsley, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Structural Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">73</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">203-210</span>CODEN:
                <span class="NLM_cas:coden">ACSDAD</span>;
        ISSN:<span class="NLM_cas:issn">2059-7983</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics program primarily aimed at model building using X-ray data.  Recently, tools for the manipulation and representation of ligands have been introduced.  Here, these new tools for ligand validation and comparison are described.  Ligands in the wwPDB have been scored by d.-fit, distortion and atom-clash metrics.  The distributions of these scores can be used to assess the relative merits of the particular ligand in the protein-ligand complex of interest by means of 'sliders' akin to those now available for each accession code on the wwPDB websites.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGponrHT1Fibu7Vg90H21EOLACvtfcHk0lg4GDa0Ut3jFg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXktFyrsLc%253D&md5=f67392ef4aada17fdffadeda97900455</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1107%2FS2059798317003382&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS2059798317003382%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26atitle%3DTools%2520for%2520ligand%2520validation%2520in%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Struct.%2520Biol.%26date%3D2017%26volume%3D73%26spage%3D203%26epage%3D210%26doi%3D10.1107%2FS2059798317003382" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref64"><div class="reference"><strong class="refLabel"><a href="#ref64" class="rightTabRefNumLink">64</a></strong><div class="NLM_citation" id="rightTab-cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Skubák, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pannu, N. S.</span></span> <span> </span><span class="NLM_article-title">Direct incorporation of experimental phase information in model refinement</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">2196</span>– <span class="NLM_lpage">2201</span>, <span class="refDoi"> DOI: 10.1107/S0907444904019079</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1107%2FS0907444904019079" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=15572772" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars7rM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2196-2201&author=P.+Skub%C3%A1kauthor=G.+N.+Murshudovauthor=N.+S.+Pannu&title=Direct+incorporation+of+experimental+phase+information+in+model+refinement&doi=10.1107%2FS0907444904019079"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Direct incorporation of experimental phase information in model refinement</span></div><div class="casAuthors">Skubak, Pavol; Murshudov, Garib N.; Pannu, Navraj S.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2196-2201</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">The incorporation of prior phase information into a max.-likelihood formalism has been shown to strengthen model refinement.  However, the currently available likelihood refinement target using prior phase information has shortcomings; the 'phased' refinement target considers exptl. phase information indirectly and statically in the form of Hendrickson-Lattman coeffs.  Furthermore, the current refinement target implicitly assumes that the prior phase information is independent of the calcd. model structure factor.  This paper describes the derivation of a multivariate likelihood function that overcomes these shortcomings and directly incorporates exptl. phase information from a single-wavelength anomalous diffraction (SAD) expt.  This function, which simultaneously refines heavy-atom and model parameters, has been implemented in the refinement program REFMAC5.  The SAD function used in conjunction with the automated model-building procedures of ARP/wARP leads to a successful soln. when current likelihood functions fail in a test case shown.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUod9J97BAj7Vg90H21EOLACvtfcHk0lg4GDa0Ut3jFg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars7rM&md5=a3d4fcf3a6c4b14357068aa16b6704d3</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1107%2FS0907444904019079&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904019079%26sid%3Dliteratum%253Aachs%26aulast%3DSkub%25C3%25A1k%26aufirst%3DP.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DPannu%26aufirst%3DN.%2BS.%26atitle%3DDirect%2520incorporation%2520of%2520experimental%2520phase%2520information%2520in%2520model%2520refinement%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26spage%3D2196%26epage%3D2201%26doi%3D10.1107%2FS0907444904019079" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref65"><div class="reference"><strong class="refLabel"><a href="#ref65" class="rightTabRefNumLink">65</a></strong><div class="NLM_citation" id="rightTab-cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cunningham, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michael, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKeithan, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spiering, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carrette, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, C.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bushway, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tierney, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albini, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giacca, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mano, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Puri, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sacco, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruiz-Lozano, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riou, J.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Umbhauer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duester, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mercola, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colas, A. R.</span></span> <span> </span><span class="NLM_article-title">Id genes are essential for early heart formation</span>. <i>Genes Dev.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">1325</span>– <span class="NLM_lpage">1338</span>, <span class="refDoi"> DOI: 10.1101/gad.300400.117</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=10.1101%2Fgad.300400.117" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=28794185" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;key=1%3ACAS%3A528%3ADC%252BC1cXptlOnu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2017&pages=1325-1338&author=T.+J.+Cunninghamauthor=S.+Y.+Michaelauthor=W.+L.+McKeithanauthor=S.+Spieringauthor=F.+Carretteauthor=C.-T.+Huangauthor=P.+J.+Bushwayauthor=M.+Tierneyauthor=S.+Albiniauthor=M.+Giaccaauthor=M.+Manoauthor=P.+L.+Puriauthor=A.+Saccoauthor=P.+Ruiz-Lozanoauthor=J.-F.+Riouauthor=M.+Umbhauerauthor=G.+Duesterauthor=M.+Mercolaauthor=A.+R.+Colas&title=Id+genes+are+essential+for+early+heart+formation&doi=10.1101%2Fgad.300400.117"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Id genes are essential for early heart formation</span></div><div class="casAuthors">Cunningham, Thomas J.; Yu, Michael S.; McKeithan, Wesley L.; Spiering, Sean; Carrette, Florent; Huang, Chun-Teng; Bushway, Paul J.; Tierney, Matthew; Albini, Sonia; Giacca, Mauro; Mano, Miguel; Puri, Pier Lorenzo; Sacco, Alessandra; Ruiz-Lozano, Pilar; Riou, Jean-Francois; Umbhauer, Muriel; Duester, Gregg; Mercola, Mark; Colas, Alexandre R.</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">1325-1338</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">Deciphering the fundamental mechanisms controlling cardiac specification is crit. for our understanding of how heart formation is initiated during embryonic development and for applying stem cell biol. to regenerative medicine and disease modeling.  Using systematic and unbiased functional screening approaches, we discovered that the Id family of helix-loop-helix proteins is both necessary and sufficient to direct cardiac mesoderm formation in frog embryos and human embryonic stem cells.  Mechanistically, Id proteins specify cardiac cell fate by repressing two inhibitors of cardiogenic mesoderm formation-Tcf3 and Foxa2-and activating inducers Evx1, Grrp1, and Mesp1.  Most importantly, CRISPR/Cas9-mediated ablation of the entire Id (Id1-4) family in mouse embryos leads to failure of anterior cardiac progenitor specification and the development of heartless embryos.  Thus, Id proteins play a central and evolutionarily conserved role during heart formation and provide a novel means to efficiently produce cardiovascular progenitors for regenerative medicine and drug discovery applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqBtCAjfxPveLVg90H21EOLACvtfcHk0lhGRrytgDQ9pw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXptlOnu7s%253D&md5=6d2a26f05a9356be7ff171074cb139cd</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1101%2Fgad.300400.117&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.300400.117%26sid%3Dliteratum%253Aachs%26aulast%3DCunningham%26aufirst%3DT.%2BJ.%26aulast%3DMichael%26aufirst%3DS.%2BY.%26aulast%3DMcKeithan%26aufirst%3DW.%2BL.%26aulast%3DSpiering%26aufirst%3DS.%26aulast%3DCarrette%26aufirst%3DF.%26aulast%3DHuang%26aufirst%3DC.-T.%26aulast%3DBushway%26aufirst%3DP.%2BJ.%26aulast%3DTierney%26aufirst%3DM.%26aulast%3DAlbini%26aufirst%3DS.%26aulast%3DGiacca%26aufirst%3DM.%26aulast%3DMano%26aufirst%3DM.%26aulast%3DPuri%26aufirst%3DP.%2BL.%26aulast%3DSacco%26aufirst%3DA.%26aulast%3DRuiz-Lozano%26aufirst%3DP.%26aulast%3DRiou%26aufirst%3DJ.-F.%26aulast%3DUmbhauer%26aufirst%3DM.%26aulast%3DDuester%26aufirst%3DG.%26aulast%3DMercola%26aufirst%3DM.%26aulast%3DColas%26aufirst%3DA.%2BR.%26atitle%3DId%2520genes%2520are%2520essential%2520for%2520early%2520heart%2520formation%26jtitle%3DGenes%2520Dev.%26date%3D2017%26volume%3D31%26spage%3D1325%26epage%3D1338%26doi%3D10.1101%2Fgad.300400.117" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YID" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YID','PDB','6YID'); return false;">PDB: 6YID</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6QAU" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6QAU','PDB','6QAU'); return false;">PDB: 6QAU</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6QAV" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6QAV','PDB','6QAV'); return false;">PDB: 6QAV</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4WNO" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4WNO','PDB','4WNO'); return false;">PDB: 4WNO</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes3"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i60"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00873">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_10862"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.0c00873?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00873</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Overview of the autophagy pathway, structures of ULK1/2 inhibitors: SBI-0206965 and Shokat compound, data collection and refinement statistics for the ULK2 structure, representative dose–response curves of compounds in the ADP-Glo assay, representative dose–response curves of compounds in the NanoBRET assay, protein thermal shift data of selected compounds, shRNA target sequences for knockdown studies, Western blot analysis of shRNA-mediated ULK1 and BECN1 knockdown in three TNBC cell lines, data showing the downregulation of starvation-induced autophagic flux in MDA-MB-468 cells by ULK inhibitors, data showing that PARP inhibitors kill MDA-MB-468 cells with low potency, data showing that SBP-7455 and olaparib synergize to inhibit the viability of MDA-MB-468 cells, data showing that SBP-7455 and niraparib synergize to inhibit the viability of MDA-MB-468 cells, data showing that induction of autophagic flux in MDA-MB-468 cells by olaparib is inhibited by concomitant treatment with SBP-7455, kinetic solubility of selected compounds, bidirectional permeability of selected compounds in Caco-2 cells, NMR and LC–MS spectra of selected compounds, and molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00873/suppl_file/jm0c00873_si_001.pdf">PDF</a>).</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00873/suppl_file/jm0c00873_si_001.pdf">jm0c00873_si_001.pdf (2.53 MB)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">X-ray crystal structure of SBI-0206965 in complex with ULK2, PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YID">6YID</a>. Authors will release the atomic coordinates and experimental data upon article publication.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.0c00873&amp;pbContext=%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2020.63.issue-23%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.0c00873%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.0c00873" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                5MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"67993c06ec2e3d4e","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
