
<html lang="en"     class="pb-page"  data-request-id="945c8d32-c410-4b1c-b8f0-3fe0989c1979"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/acs.jmedchem.9b00763;issue:issue:10.1021/jmcmar.2019.62.issue-18;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Development of Novel Irreversible Pyruvate Kinase M2 Inhibitors" /></meta><meta name="dc.Creator" content="I-Shan  Hsieh" /></meta><meta name="dc.Creator" content="Balraj  Gopula" /></meta><meta name="dc.Creator" content="Chi-Chi  Chou" /></meta><meta name="dc.Creator" content="Hsiang-Yi  Wu" /></meta><meta name="dc.Creator" content="Geen-Dong  Chang" /></meta><meta name="dc.Creator" content="Wen-Jin  Wu" /></meta><meta name="dc.Creator" content="Chih-Shiang  Chang" /></meta><meta name="dc.Creator" content="Po-Chen  Chu" /></meta><meta name="dc.Creator" content="Ching S.  Chen" /></meta><meta name="dc.Description" content="As cancer cells undergo metabolic reprogramming in the course of tumorigenesis, targeting energy metabolism represents a promising strategy in cancer therapy. Among various metabolic enzymes examin..." /></meta><meta name="Description" content="As cancer cells undergo metabolic reprogramming in the course of tumorigenesis, targeting energy metabolism represents a promising strategy in cancer therapy. Among various metabolic enzymes examin..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="August 29, 2019" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.9b00763" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2019 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b00763" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.9b00763" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b00763" /></link>
        
    
    

<title>Development of Novel Irreversible Pyruvate Kinase M2 Inhibitors | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.9b00763" /></meta><meta property="og:title" content="Development of Novel Irreversible Pyruvate Kinase M2 Inhibitors" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b00763/20190920/images/large/jm9b00763_0009.jpeg" /></meta><meta property="og:description" content="As cancer cells undergo metabolic reprogramming in the course of tumorigenesis, targeting energy metabolism represents a promising strategy in cancer therapy. Among various metabolic enzymes examined, pyruvate kinase M2 type (PKM2) has received much attention in light of its multifaceted function in promoting tumor growth and progression. In this study, we reported the development of a novel irreversible inhibitor of PKM2, compound 1, that exhibits a differential tumor-suppressive effect among an array of cancer cell lines. We further used a clickable activity-based protein profiling (ABPP) probe and SILAC coupled with LC-MS/MS to identify the Cys-317 and Cys-326 residues of PKM2 as the covalent binding sites. Equally important, compound 1 at 10 mg/kg was effective in suppressing xenograft tumor growth in nude mice without causing acute toxicity by targeting both metabolic and oncogenic functions. Together, these data suggest its translational potential to foster new strategies for cancer therapy." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.9b00763"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b00763">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.9b00763&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.9b00763&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.9b00763&amp;href=/doi/10.1021/acs.jmedchem.9b00763" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2019</span><span class="cit-fg-volume">, 62</span><span class="cit-fg-issue">, 18</span><span class="cit-fg-pageRange">, 8497-8510</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/62/18" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.9b00728" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.9b00778" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Development of Novel Irreversible Pyruvate Kinase M2 Inhibitors</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">I-Shan Hsieh</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">I-Shan Hsieh</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Biological Chemistry, Academia Sinica, Taipei 11529, Taiwan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=I-Shan++Hsieh">I-Shan Hsieh</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Balraj Gopula</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Balraj Gopula</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Biological Chemistry, Academia Sinica, Taipei 11529, Taiwan</div><div class="loa-info-affiliations-info">Drug Development Center, China Medical University, Taichung 40402, Taiwan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Balraj++Gopula">Balraj Gopula</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Chi-Chi Chou</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Chi-Chi Chou</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Biological Chemistry, Academia Sinica, Taipei 11529, Taiwan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Chi-Chi++Chou">Chi-Chi Chou</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Hsiang-Yi Wu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Hsiang-Yi Wu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Biological Chemistry, Academia Sinica, Taipei 11529, Taiwan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Hsiang-Yi++Wu">Hsiang-Yi Wu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Geen-Dong Chang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Geen-Dong Chang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Biochemical Sciences, National Taiwan University, Taipei 10617, Taiwan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Geen-Dong++Chang">Geen-Dong Chang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Wen-Jin Wu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Wen-Jin Wu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Biological Chemistry, Academia Sinica, Taipei 11529, Taiwan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Wen-Jin++Wu">Wen-Jin Wu</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-7253-5384" title="Orcid link">http://orcid.org/0000-0002-7253-5384</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Chih-Shiang Chang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Chih-Shiang Chang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Drug Development Center, China Medical University, Taichung 40402, Taiwan</div><div class="loa-info-affiliations-info">School of Pharmacy, College of Pharmacy, China Medical University, Taichung 40402, Taiwan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Chih-Shiang++Chang">Chih-Shiang Chang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Po-Chen Chu</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Po-Chen Chu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Drug Development Center, China Medical University, Taichung 40402, Taiwan</div><div class="loa-info-affiliations-info">Department of Cosmeceutics and Graduate Institute of Cosmeceutics, China Medical University, Taichung 40402, Taiwan</div></div><span class="conrtib-corresp"><strong>*</strong>Address: Department of Cosmeceutics and Graduate Institute of Cosmeceutics, China Medical University, No. 91, Hsueh-Shih Road, North District, Taichung 40402, Taiwan. Phone: 886-4-22053366, ext. 5705. E-mail: <a href="/cdn-cgi/l/email-protection#88f8e7f8e7f8e1c8efe5e9e1e4a6ebe7e5"><span class="__cf_email__" data-cfemail="c1b1aeb1aeb1a881a6aca0a8adefa2aeac">[email protected]</span></a></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Po-Chen++Chu">Po-Chen Chu</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-9510-2269" title="Orcid link">http://orcid.org/0000-0001-9510-2269</a></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Ching S. Chen</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ching S. Chen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of New Drug Development, China Medical University, Taichung 40402, Taiwan</div><div class="loa-info-affiliations-info">Department of Medical Research, China Medical University Hospital, China Medical University, Taichung 40447, Taiwan</div></div><span class="conrtib-corresp"><strong>*</strong>Address: Department of Medical Research, China Medical University Hospital, China Medical University, No. 2, Yude Road, North District, Taichung 40447, Taiwan. Phone: 886-4-22053366 ext. 8201. E-mail: <a href="/cdn-cgi/l/email-protection#87e4f4e4efe2e9e4eaf2c7e0eae6eeeba9e4e8ea"><span class="__cf_email__" data-cfemail="d5b6a6b6bdb0bbb6b8a095b2b8b4bcb9fbb6bab8">[email protected]</span></a></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ching+S.++Chen">Ching S. Chen</a></span></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b00763&amp;href=/doi/10.1021%2Facs.jmedchem.9b00763" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2019</span></span><span class="cit-volume">, 62</span><span class="cit-issue">, 18</span><span class="cit-pageRange">, 8497–8510</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">August 29, 2019</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>10 May 2019</li><li><span class="item_label"><b>Published</b> online</span>29 August 2019</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 26 September 2019</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.9b00763" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b00763</a></div><div class="article_header-article-copyright"><strong>Copyright © 2019 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D8497%26pageCount%3D14%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DI-Shan%2BHsieh%252C%2BBalraj%2BGopula%252C%2BChi-Chi%2BChou%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D62%26issueNum%3D18%26contentID%3Dacs.jmedchem.9b00763%26title%3DDevelopment%2Bof%2BNovel%2BIrreversible%2BPyruvate%2BKinase%2BM2%2BInhibitors%26numPages%3D14%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D8510%26publicationDate%3DSeptember%2B2019">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.9b00763"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1933</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">5</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.9b00763" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Development of Novel Irreversible Pyruvate Kinase M2 Inhibitors&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;I-Shan&quot;,&quot;last_name&quot;:&quot;Hsieh&quot;},{&quot;first_name&quot;:&quot;Balraj&quot;,&quot;last_name&quot;:&quot;Gopula&quot;},{&quot;first_name&quot;:&quot;Chi-Chi&quot;,&quot;last_name&quot;:&quot;Chou&quot;},{&quot;first_name&quot;:&quot;Hsiang-Yi&quot;,&quot;last_name&quot;:&quot;Wu&quot;},{&quot;first_name&quot;:&quot;Geen-Dong&quot;,&quot;last_name&quot;:&quot;Chang&quot;},{&quot;first_name&quot;:&quot;Wen-Jin&quot;,&quot;last_name&quot;:&quot;Wu&quot;},{&quot;first_name&quot;:&quot;Chih-Shiang&quot;,&quot;last_name&quot;:&quot;Chang&quot;},{&quot;first_name&quot;:&quot;Po-Chen&quot;,&quot;last_name&quot;:&quot;Chu&quot;},{&quot;first_name&quot;:&quot;Ching&quot;,&quot;last_name&quot;:&quot;S. Chen&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2019&quot;,&quot;month&quot;:&quot;08&quot;,&quot;day&quot;:&quot;29&quot;,&quot;issue&quot;:&quot;18&quot;,&quot;volume&quot;:&quot;62&quot;,&quot;pages&quot;:&quot;8497-8510&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.9b00763&quot;},&quot;abstract&quot;:&quot;As cancer cells undergo metabolic reprogramming in the course of tumorigenesis, targeting energy metabolism represents a promising strategy in cancer therapy. Among various metabolic enzymes examined, pyruvate kinase M2 type (PKM2) has received much attention in light of its multifaceted function in promoting tumor growth and progression. In this study, we reported the development of a novel irreversible inhibitor of PKM2, compound 1, that exhibits a differential tumor-suppressive effect among an array of cancer cell lines. We further used a clickable activity-based protein profiling (ABPP) probe and SILAC coupled with LC-MS/MS to identify the Cys-317 and Cys-326 residues of PKM2 as the covalent binding sites. Equally important, compound 1 at 10 mg/kg was effective in suppressing xenograft tumor growth in nude mice without causing acute toxicity by targeting both metabolic and oncogenic functions. Together, these data suggest its translational potential to foster new strategies for cancer therapy.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00763&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00763" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00763&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00763" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00763&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00763" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b00763&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00763&amp;href=/doi/10.1021/acs.jmedchem.9b00763" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.9b00763" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.9b00763" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (6 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b00763&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.9b00763%26sid%3Dliteratum%253Aachs%26pmid%3D31465224%26genre%3Darticle%26aulast%3DHsieh%26date%3D2019%26atitle%3DDevelopment%2Bof%2BNovel%2BIrreversible%2BPyruvate%2BKinase%2BM2%2BInhibitors%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D62%26issue%3D18%26spage%3D8497%26epage%3D8510%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (3)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292258" title="Cancer">Cancer</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291439" title="Genetics">Genetics</a>,</li><li><a href="/action/doSearch?ConceptID=290831" title="Monomers">Monomers</a>,</li><li><a href="/action/doSearch?ConceptID=290896" title="Cells">Cells</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/62/18" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/jmcmar.2019.62.issue-18/20190926/jmcmar.2019.62.issue-18.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b00763/20190920/images/medium/jm9b00763_0009.gif" alt="" id="_i2" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b00763/20190920/images/large/jm9b00763_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00763&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">As cancer cells undergo metabolic reprogramming in the course of tumorigenesis, targeting energy metabolism represents a promising strategy in cancer therapy. Among various metabolic enzymes examined, pyruvate kinase M2 type (PKM2) has received much attention in light of its multifaceted function in promoting tumor growth and progression. In this study, we reported the development of a novel irreversible inhibitor of PKM2, compound <b>1</b>, that exhibits a differential tumor-suppressive effect among an array of cancer cell lines. We further used a clickable activity-based protein profiling (ABPP) probe and SILAC coupled with LC-MS/MS to identify the Cys-317 and Cys-326 residues of PKM2 as the covalent binding sites. Equally important, compound <b>1</b> at 10 mg/kg was effective in suppressing xenograft tumor growth in nude mice without causing acute toxicity by targeting both metabolic and oncogenic functions. Together, these data suggest its translational potential to foster new strategies for cancer therapy.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_55635" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_55635" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">It is well recognized that cancer cells undergo metabolic shift to a glycolytic phenotype in the course of tumorigenesis (the so-called Warburg effect), which provides tumor cells with survival advantages under unfavorable growth environments.<a onclick="showRef(event, 'ref1 ref2 ref3 ref4 ref5'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3 ref4 ref5">(1−5)</a> Evidence indicates that this metabolic reprogramming necessitates cancer cells to upregulate the expression of key regulators of the glycolytic pathway, including glucose transporter 1,<a onclick="showRef(event, 'ref6 ref7'); return false;" href="javascript:void(0);" class="ref ref6 ref7">(6,7)</a> hexokinase 2,<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> and the M2 splice form of pyruvate kinase (PKM2).<a onclick="showRef(event, 'ref9 ref10 ref11 ref12'); return false;" href="javascript:void(0);" class="ref ref9 ref10 ref11 ref12">(9−12)</a> From a therapeutic perspective, these glycolytic regulators represent promising targets for cancer drug development,<a onclick="showRef(event, 'ref13 ref14'); return false;" href="javascript:void(0);" class="ref ref13 ref14">(13,14)</a> among which PKM2 has received much attention because of its multifaceted function in promoting tumor growth and progression.<a onclick="showRef(event, 'ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref15 ref16">(15,16)</a> PKM2 catalyzes the rate-limiting step of glycolysis, that is, conversion of phosphoenolpyruvate (PEP) to pyruvate. In addition, PKM2 also regulates the transcription of various cancer-associated genes upon entering the nucleus, where it activates a number of transcription factors through physical interactions or phosphorylation.<a onclick="showRef(event, 'ref9 ref10 ref11 ref16'); return false;" href="javascript:void(0);" class="ref ref9 ref10 ref11 ref16">(9−11,16)</a> To date, multiple structurally diverse small-molecule PKM2 modulators have been developed in the past decade (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00763/suppl_file/jm9b00763_si_001.pdf" class="ext-link">Figure S1</a>).<a onclick="showRef(event, 'ref15 ref17 ref18 ref19 ref20 ref21 ref22 ref23'); return false;" href="javascript:void(0);" class="ref ref15 ref17 ref18 ref19 ref20 ref21 ref22 ref23">(15,17−23)</a> These PKM2 inhibitors (e.g., shikonin<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> and alkannin<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a>) or activators (e.g., NCGC00030335<a onclick="showRef(event, 'ref20 ref21'); return false;" href="javascript:void(0);" class="ref ref20 ref21">(20,21)</a> and micheliolide<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a>) could suppress cancer cell proliferation by interfering with energy metabolism and/or PKM2’s nuclear translocation, which underscores the crucial function of PKM2 in maintaining the malignant phenotype of cancer cells.</div><div class="NLM_p">In this study, we report the development of a novel irreversible inhibitor of PKM2, <i>N</i>-(4-(3-(3-(methylamino)-3-oxopropyl)-5-(4′-(trifluoromethyl)-[1,1′-biphenyl]-4-yl)-1<i>H</i>-pyrazol-1-yl)phenyl)propiolamide (<b>1</b>; <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A), which exhibited differential growth inhibition among a panel of cancer cell lines. The ability of compound <b>1</b> to target PKM2 was characterized by several experiments. For example, immunoblotting analysis indicates that antibodies raised against this small-molecule agent cross-reacted with compound <b>1</b>-treated PKM2. In addition, we developed a clickable activity-based protein profiling (ABPP) probe to conduct SILAC (stable isotope labeling by amino acids in cell culture)-based proteomic analysis, and via tandem mass spectrometry (MS/MS), we identified the Cys-317 and Cys-326 residues as the covalent binding sites. Furthermore, we obtained evidence that the high in vitro and in vivo antitumor efficacy of compound <b>1</b> was attributable to its ability to target both metabolic and nonmetabolic functions of cancer cells. Together, these data suggest that compound <b>1</b> represents the first irreversible PKM2 inhibitor, of which the translational value as a potential cancer therapeutic agent warrants further investigation.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b00763/20190920/images/medium/jm9b00763_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b00763/20190920/images/large/jm9b00763_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. (A) Structures of compound <b>22</b> versus compound <b>1</b> and its ABPP probe <b>2</b>. (B and C) Synthetic schemes for compounds <b>1</b> (B) and <b>2</b> (C).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b00763/20190920/images/large/jm9b00763_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00763&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div id="sec1_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Chemistry</h3><div class="NLM_p">In an effort to develop irreversible kinase inhibitors, we chose the core structures of several published kinase inhibitors to couple with a propiolyl moiety, which could act as an electrophilic warhead to irreversibly modify the cysteine residues of target proteins. Among various core structures evaluated, that of the integrin-linked kinase inhibitor compound <b>22</b><a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A), i.e., 3-(1-(4-aminophenyl)-5-[4′-(trifluoromethyl)-(1,1′-biphenyl)-4-yl]-1<i>H</i>-pyrazol-3-yl)-<i>N</i>-methyl-propanamide (<b>i</b>; <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B), was of particular interest as the resulting propargylic amide <b>1</b> exhibited differential growth inhibition in a panel of cancer cell lines (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A). When compound <b>1</b> was incubated with a stoichiometric amount of free cysteine in solution, mass spectral analysis showed the formation of a covalent <b>1</b>-cysteine adduct (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00763/suppl_file/jm9b00763_si_001.pdf" class="ext-link">Supplementary Figure S2</a>), which provided a proof-of-concept that <b>1</b> undergoes Michael addition with a nucleophile.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b00763/20190920/images/medium/jm9b00763_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b00763/20190920/images/large/jm9b00763_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Differential antiproliferative effects of compounds <b>1</b> and <b>2</b> in different cancer cell lines by MTT assays. (A) Concentration-dependent suppressive effects of compound <b>1</b> on the viability of different cancer cell lines, including those of pancreas (Panc-1, AsPC-1, and MiaPaCa-2), breast (MDA-MB-231 and MCF-7), oral (SCC2095 and SCC4), and prostate (PC-3 and LNCaP). (B) Concentration-dependent suppressive effects of compounds <b>1</b> (left) and <b>2</b> (right) on the viability of lung cancer cell lines harboring mutant EGFR (H1650, PC-9, and H1975) versus those with wild-type EGFR (H460 and H157). Value, means ± SD (<i>n</i> = 6).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b00763/20190920/images/large/jm9b00763_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00763&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">To help identify the cellular target of compound <b>1</b>, we developed a clickable ABPP probe of <b>1</b>, <i>N</i>-(4-(5-(4′-(ethynyloxy)-[1,1′-biphenyl]-4-yl)-3-(3-(methylamino)-3-oxopropyl)-1<i>H</i>-pyrazol-1-yl)phenyl)propiolamide (compound <b>2</b>; <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A), which contained a prop-2-yn-1-yloxy moiety, in lieu of CF<sub>3</sub>, as a reporter tag at the terminal phenyl ring. This ABPP probe was used to conduct SILAC-based proteomic analysis. The alkyne reporter tag allowed the biotinylation of compound <b>2</b>-labeled target proteins via Cu(I)-catalyzed biorthogonal conjugation with azide-biotin, which facilitated the subsequent streptavidin-bead pulldown to undergo proteomic analysis. The synthesis of compound <b>2</b> was depicted in the scheme shown in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>C.</div></div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i5">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_75251" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_75251" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Compound <b>1</b> Shows Differential Antiproliferative Efficacies against Cancer Cell Lines with Different Genetic Characteristics</h3><div class="NLM_p last">The antitumor activity of compound <b>1</b> was evaluated in a panel of cancer cell lines from different tissue types, including those of pancreas (Panc-1, AsPC-1, MiaPaCa-2), breast (MDA-MB-231, MCF-7), oral (SCC2095, SCC4), prostate (PC-3, LNCaP), and lung (H1650, PC-9, H1975, H460, H157). MTT assays indicate that these cell lines exhibited differential susceptibility to compound <b>1</b> with IC<sub>50</sub> values ranging from 0.1 μM (MDA-MB-231) to >2 μM (LNCaP and H157), which could be attributable to differences in their genetic characteristics (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). For example, the aggressive MDA-MB-231 and PC-3 cells were more susceptible to compound <b>1</b>-mediated growth inhibition relative to their noninvasive counterparts MCF-7 (IC<sub>50</sub>, 0.1 versus 1 μM) and LNCaP cells (IC<sub>50</sub>, 0.6 versus >2 μM), respectively. This differential sensitivity was also evident in lung cancer cell lines as mutant EGFR cell lines (IC<sub>50</sub>, H1650, 0.2 μM; PC-9, 0.5 μM; H1975, 1 μM) were more sensitive as compared to those harboring wild type EGFR (IC<sub>50</sub>, H460, 1.5 μM; H157, > 2 μM) (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B, left). This discriminative antiproliferative effect suggested a unique cellular target and/or pathway by which compound <b>1</b> mediated its antitumor activity. To shed light onto its potential target, compound <b>1</b> was submitted to a commercial vendor for kinase profiling analysis (Life Technologies’ SelectScreen Profiling Service), in which the inhibitory effects of compound <b>1</b> at 500 nM on the activity of 246 different kinases (% inhibition) as well as substrate binding to 143 different kinases (% displacement) were tested. However, these profiling data indicate that none of the kinases examined was effectively inhibited by compound <b>1</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00763/suppl_file/jm9b00763_si_001.pdf" class="ext-link">Table S1 and S2</a>), which argued against the involvement of these kinases in the tumor-suppressive effect of compound <b>1</b>.</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> Identification of PKM2 as a Target of Compound <b>1</b> via ABPP-SILAC</h3><div class="NLM_p">As part of our effort to identify compound <b>1</b>’s targets, we embarked on the use of compound <b>2</b> as an ABPP probe to conduct SILAC-based proteomic analysis. Compound <b>2</b> exhibited a pattern of growth inhibition similar to that of compound <b>1</b> among different lung cancer cell lines examined. As PC-9 cells showed the highest sensitivity to both compounds, we used this cell line to conduct target identification via ABPP-SILAC, as depicted in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b00763/20190920/images/medium/jm9b00763_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b00763/20190920/images/large/jm9b00763_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. ABPP-SILAC-based proteomic analysis to identify cellular targets of compound <b>1</b>. (A) A schematic diagram depicting the ABPP-SILAC-based quantitative proteomics strategy. (B) SDS-PAGE analysis of ABPP-labeled target proteins via streptavidin-bead pulldown, which was silver stained. The indicated gel regions were excised from SDS-PAGE gels and subjected to in-gel digestion with trypsin and subsequent target identification by LC-MS/MS.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b00763/20190920/images/large/jm9b00763_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00763&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In this experiment, PC-9 cells were grown in cultural medium supplemented with unlabeled <span class="smallcaps smallerCapital">l</span>-arginine and <span class="smallcaps smallerCapital">l</span>-lysine (light medium) versus <sup>13</sup>C/<sup>15</sup>N stable isotope-labeled <span class="smallcaps smallerCapital">l</span>-arginine and <span class="smallcaps smallerCapital">l</span>-lysine (heavy medium), generating two populations of cells (light versus heavy cells). Heavy PC-9 cells were then exposed to the ABPP (1 μM) for 1 h to allow intracellular labeling of target proteins, while light cells were treated with DMSO vehicle under the same condition. These light and heavy cells were mixed at a 1:1 ratio and lysed, and cell lysates were treated with biotin-azide to facilitate the biotinylation of compound <b>2</b>-labeled target proteins. Subsequently, the resulting biotin conjugates of ABPP-labeled target proteins were purified by streptavidin bead pulldown, followed by 10% SDS-PAGE. Silver staining of the gel showed three protein bands at approximately 45, 58, and 73 kDa, respectively (arrowhead, <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B), each of which was excised from the gel and subjected to in-gel trypsin digestion. These peptide mixtures were analyzed by liquid chromatography–tandem mass spectrometry (LC-MS/MS), and the resulting data were searched against various primary sequence databases by using the Mascot search engine. This proteomic analysis led to the identification of nine putative protein targets, including PKM2, lactate dehydrogenase A (LDHA), citrate synthase (CS), malate dehydrogenase (MDHM), ATP-citrate lyase (ACYL), α-enolase (ENOA), phosphoenolpyruvate carboxylase (PCKGM), acyl-CoA synthetase family member 3 (ACSF3), and phosphoglycerate kinase (PGK1), among which the protein with the highest matching score was the 58 kDa PKM2 (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A). Analysis using the STRING database revealed the protein association network of PKM2 with other identified target proteins (except ACSF3) through direct or indirect interactions, of which the map is depicted in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>B. Additionally, to shed light onto cellular changes in response to compound <b>2</b>, these target proteins were functionally categorized according to the Gene Ontology analysis for biological processes. As shown, these proteins were intimately involved in the metabolic processes of glucose, NAD, and citrate (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>C), indicating the ability of compound <b>2</b> to target energy metabolism in PC-9 cells.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b00763/20190920/images/medium/jm9b00763_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b00763/20190920/images/large/jm9b00763_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Identification of PKM2 as a target of compound <b>1</b>. (A) A list of kinase protein targets of compound <b>1</b> identified by LC-MS/MS in PC-9 cancer cells. (B) Protein–protein interaction pathway map of compound <b>1</b>’s target proteins. (C) GO functional annotation for related targets and biological processes of PKM2.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b00763/20190920/images/large/jm9b00763_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00763&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Evidence that Compound <b>1</b> is an Irreversible Inhibitor of PKM2</h3><div class="NLM_p">The above ABPP-SILAC analysis suggests that PKM2 might represent a primary target for compound <b>1</b>. This premise was supported by the significantly reduced pyruvate kinase activity (<i>P</i> < 0.005) in PC-9 cells treated with compound <b>1</b> (0.25 μM) relative to that of vehicle control for 6 h (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A). As this cell-based kinase assay could not rule out the involvement of PKM1, we examined the abilities of compound <b>1</b> to inhibit the kinase activity of recombinant PKM2 (SAE0021, Sigma-Aldrich) versus recombinant PKM1 (SRP0415, Sigma-Aldrich) using a commercial PKM assay kit, which was based on the colorimetric assay of H<sub>2</sub>O<sub>2</sub> generated from PEP and ATP via two consecutive enzymatic reactions, i.e., PK and pyruvate oxidase. As shown, after 20 min of exposure, compound <b>1</b> could significantly suppress the kinase activity of recombinant PKM2 (<i>P</i> < 0.001), while no appreciable inhibition was noted with recombinant PKM1 (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B), indicating the specificity of compound <b>1</b> in PKM2 inhibition. It should be noted that due to the intrinsic instability of H<sub>2</sub>O<sub>2</sub>, the incubation time was short to avoid H<sub>2</sub>O<sub>2</sub> decomposition in the course of assay. Therefore, compound <b>1</b> only achieved approximately 30% inhibition in this assay, in part, because of this short exposure time.</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b00763/20190920/images/medium/jm9b00763_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b00763/20190920/images/large/jm9b00763_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Evidence that compound <b>1</b> is an irreversible inhibitor of PKM2. (A) PC-9 cells treated with compound <b>1</b> (0.25 μM) showed significantly reduced pyruvate kinase activity relative to vehicle control. Bar, means ± SD (<i>n</i> = 3). ***<i>P</i> < 0.005. (B) Compound <b>1</b> selectively inhibited the kinase activity of recombinant PKM2, but not recombinant PKM1. Bar, means ± SD (<i>n</i> = 3). ***<i>P</i> < 0.001. (C) Compound <b>1</b> was effective in competing with the ABPP probe compound <b>2</b> for PKM2 binding. PC-9 cells were preincubated with 1 μM compound <b>1</b> for 1 h, and treated with compound <b>2</b> at indicated concentrations. Cells were lysed, and treated with biotin-azide, followed by streptavidin bead pulldown and Western blot analysis. (D) Anticompound <b>1</b> antibodies cross reacted with compound <b>1</b>-treated PKM2. Recombinant PKM2 (4 μg) was incubated with vehicle control or compound <b>1</b> at indicated concentrations at 4 °C for 6 h, followed by immunoblotting with anti-PKM2 and anti-compound-<b>1</b> antibodies.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b00763/20190920/images/large/jm9b00763_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00763&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">In addition, we demonstrated that compound <b>1</b> was able to compete with the ABPP probe compound <b>2</b> for PKM2 binding. In brief, PC-9 cells were exposed to different concentrations of compound <b>2</b> in the presence of compound <b>1</b> (1 μM) for 1 h, lysed, and the cell lysates were treated with biotin-azide, followed by streptavidin bead pulldown under the aforementioned conditions. Western blot analysis indicated that the binding of the ABPP probe to PKM2 was attenuated by compound <b>1</b> (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>C). Equally important, we raised antibodies against compound <b>1</b> to demonstrate the ability of compound <b>1</b> to form covalent adducts with recombinant PKM2. The antigen of compound <b>1</b> was prepared by coupling this small molecule with cysteine residues of ovalbumin, which was then used for immunization in guinea pigs. Consistent with the covalent mode of binding, Western blot analysis showed that these antibodies cross reacted with compound <b>1</b>-treated PKM2 (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>D).</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Mass Spectral Identification of Potential Compound <b>1</b>-Modified Sites on PKM2</h3><div class="NLM_p">As PKM2 has a total of 10 cysteine residues (NCBI reference sequence: NM_002654; <a href="/doi/suppl/10.1021/acs.jmedchem.9b00763/suppl_file/jm9b00763_si_001.pdf" class="ext-link">Supplementary Figure S3</a>), we conducted proteomic analysis to identify which cysteine residues were covalently modified by compound <b>1</b>. Recombinant PKM2, prepared in our laboratory (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00763/suppl_file/jm9b00763_si_001.pdf" class="ext-link">Supplementary Figure S4</a>), was treated with compound <b>1</b>, followed by tryptic digestion. To facilitate a more accurate and comprehensive mapping of the modified sites, in-solution digested compound <b>1</b>-modified tryptic peptides of PKM2, obtained from 6 separate experiments were pooled and subjected to 48 repeated runs of nanoLC-MS/MS analysis under a data-dependent acquisition mode. The resulting MS/MS data sets were individually searched against the PKM2 human protein sequence (UniProtKB, P14618–1) database using the Mascot search engine. The search results from 48 analyses provided a protein sequence coverage of 97.1%. The corresponding MS/MS spectra for each of the compound <b>1</b>-modified sites were then manually verified and annotated for the sequence informative b and y fragment ions. These MS/MS data afforded the [M+5H]<sup>5+</sup> quintuply protonated precursor ion at <i>m</i>/<i>z</i> 615.12323 for the Cys326-containing peptide (320–342) AGKPVIC<sup>326</sup>ATQMLESMIKKPRPTR (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A) and the [M+3H]<sup>3+</sup> triply charged precursor ion at <i>m</i>/<i>z</i> 499.55002 for the Cys317-carrying peptide (312–319) MMIGRC<sup>317</sup>NR (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>B). In general, the modification site assignment was considered reliable if the b and y ions flanking the implicated site could be detected, with mass shifts corresponding to a compound <b>1</b>-modified cysteine (+ 516.1773 Da). As expected, detected b and y ions carrying the modified Cys residue including b7–11, b14, and y19–20 for peptide AGKPVIC<sup>326</sup>ATQMLESMIKKPRPTR; b6, y3, and y5–7 for peptide MMIGRC<sup>317</sup>NR (dotted in the peptide sequence shown) were found to retain the compound <b>1</b> moiety. Together, this proteomic analysis revealed that compound <b>1</b> was covalently coupled to the two cysteine residues Cys317 and Cys326 near the substrate-binding site of PKM2 (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>C).</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b00763/20190920/images/medium/jm9b00763_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b00763/20190920/images/large/jm9b00763_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Identification of Cys326 and Cys317 as potential compound <b>1</b> modified sites on PKM2 via MS/MS analysis. MS/MS spectra derived from compound <b>1</b>-modified tryptic peptides of PKM2 afforded (A) the [M+5H]<sup>5+</sup> precursor ion at <i>m</i>/<i>z</i> 615.12323 Da for the Cys326-containing peptide AGKPVIC<sup>326</sup>ATQMLESMIKKPRPTR and (B) the [M+3H]<sup>3+</sup> precursor ion at <i>m</i>/<i>z</i> 499.55002 Da for the Cys317-carrying peptide MMIGRC<sup>317</sup>NR (B). The amino acid sequences and respective b and y ions are shown in each spectrum, with the compound <b>1</b>-modified cysteine residues underlined. All detected b and y ions carrying the modified Cys residue (b7–11, b14, and y19–20 for peptide AGKPVIC<sup>326</sup>ATQMLESMIKKPRPTR; b6, y3, and y5–7 for peptide MMIGRC<sup>317</sup>NR) dotted in the peptide sequence shown were found to retain the compound <b>1</b> moiety (+ 516.1773 Da). (C) A depiction of the Cys326 and Cys317 residues on PKM2 (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3gr4">3gr4</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b00763/20190920/images/large/jm9b00763_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00763&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">To gain insight into the mode of ligand binding, we attempted to obtain crystals of the PKM2-compound <b>1</b> complex. Although we were able to obtain crystals of the apo-form of PKM2 using the sitting drop vapor diffusion method, obtaining the cocrystal with compound <b>1</b> through either socking (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00763/suppl_file/jm9b00763_si_001.pdf" class="ext-link">Supplementary Figure S5</a>) or cocrystallization was unsuccessful, which might, in part, be attributable to ligand’s poor solubility.</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Effects of Compound <b>1</b> on Glycolysis and Oncogenic Signaling in PC-9 Cells</h3><div class="NLM_p">In light of the role of PKM2 in promoting glycolysis, we hypothesized that the antitumor activity of compound <b>1</b> might, in part, be attributable to its ability to reverse the glycolytic phenotype (i.e., Warburg effect) of cancer cells. Accordingly, we examined the effects of compound <b>1</b> on the extracellular acidification rate (ECAR) and cellular oxygen consumption rate (OCR) in PC-9 cells, which represent key bioenergetic parameters of glycolysis (lactate production) and mitochondrial respiration (oxidative phosphorylation), respectively, using a commercial kit (Agilent Seahorse XF Glycolysis Stress Test Kit).<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> In the ECAR measurement, glucose, oligomycin (an ATP synthetase inhibitor), and 2-deoxyglucose (2-DG; a hexokinase/glycolysis inhibitor) were added in tandem to glucose-starved cells at different time intervals to activate or interfere with glycolysis, thereby allowing the calculation of the glycolytic flux and glycolytic capacity.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> As shown, although compound <b>1</b> had no appreciable effect on the nonglycolytic acidification (i.e., the basal state prior to glucose injection), this PKM2 inhibitor suppressed the glycolytic acidification in a concentration-dependent manner (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>A). Meanwhile, the OCR response, a parameter for oxidative phosphorylation, was concurrently monitored. Relative to DMSO control, compound <b>1</b> was able to increase oxygen consumption in the basal state (prior to glucose supplementation), suggesting its ability to elevate mitochondrial respiration (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>B). It is noteworthy that when glucose was added to the basal medium, there was a decrease in the OCR in the control group. This phenomenon was previously referred to as the “Crabtree effect’’,<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> i.e., when glucose is added to activate glycolysis, it might be more favorable for cells to generate ATP through substrate-level phosphorylation, thereby reducing the need of oxidative phosphorylation. However, this glucose-induced drop in OCR became less apparent in the presence of compound <b>1</b>, which might be associated with the ability of compound <b>1</b> to block glycolysis.</div><figure id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b00763/20190920/images/medium/jm9b00763_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b00763/20190920/images/large/jm9b00763_0007.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Evidence that compound <b>1</b> target both metabolic and oncogenic functions in PC-9 cells. (A) Left, a representative graph of ECAR outputs in response to vehicle control (blue) or compound <b>1</b> at 0.25 μM (beige), 0.5 μM (green), and 1 μM (red). Glycolytic stress tests were performed using the Seahorse XF bioanalyzer to measure the glycolytic capacity of PC-9 cells. Right, average values of key parameters for the evaluation of glycolytic function with or without compound <b>1</b>. Bar, means ± SD (<i>n</i> = 6). *<i>P</i> < 0.05, **<i>P</i> < 0.01. (B) Left, a representative graph of OCAR outputs in response to vehicle control (blue) or compound <b>1</b> at 0.25 μM (beige), 0.5 μM (green), and 1 μM (red). Right, average values of key parameters for the evaluation of mitochondrial functions with or without compound <b>1</b>. Bar, means ± SD (<i>n</i> = 6). *<i>P</i> < 0.05. (C) Western blot analyses of the concentration-dependent suppressive effect of compound <b>1</b> on the expression and/or phosphorylation of PKM2, EGFR, Stat3, β-Catenin in PC-9 cells. (D) Effect of compound 1 at indicated concentrations on ROS production in PC-9 cells. Bar, means ± SD (<i>n</i> = 3). **<i>P</i> < 0.01.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b00763/20190920/images/large/jm9b00763_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00763&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Beyond its function as a metabolic regulator, PKM2 has also been reported to activate a number of oncogenic effectors through physical interactions in different cellular compartments,<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> leading to increased protein stability [i.e., EGFR<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a>] or phosphorylation [i.e., Stat3<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> and β-catenin<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a>]. Thus, the effect of compound <b>1</b> on the expression and/or phosphorylation of these interacting partners was examined. As shown, treatment with compound <b>1</b> downregulated the expression of PKM2 and EGFR, accompanied by parallel decreases in the phosphorylation and expression of Stat3 and β-catenin in PC-9 cells (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>C). We rationalized that covalent modifications of PKM2 by compound <b>1</b> might decrease its protein stability, leading to decreases in the observed protein expression level. Although it has been reported that PKM2 regulated EGFR protein stability,<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> the mechanism by which compound <b>1</b> decreased the expression levels of Stat3 and β-catenin remained unclear, which warrants investigation. It is interesting to note that the suppressive effect of compound <b>1</b> at 0.25 μM on Stat3 and β-catenin expression was accompanied by increased phosphorylation of these two oncoproteins. It is plausible that this increase in phosphorylation might be attributable to a compensatory mechanism in response to reduced protein expression in drug-treated cells.</div><div class="NLM_p">Moreover, as PKM2 has been reported to be negatively regulated by reactive oxygen species (ROS),<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> we examined the effect of compound <b>1</b> on ROS production in PC-9 cells. As shown, compound <b>1</b> at 0.25 μM and 0.5 μM reduced the ROS level (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>D), which refuted the possibility that compound <b>1</b> might, in part, inhibit PKM2 through ROS.</div><div class="NLM_p last">Together, these findings demonstrated the ability of compound <b>1</b> to target both metabolic and nonmetabolic functions of cancer cells, at least in part, through irreversible inhibition of PKM2, which underlies its high antiproliferative potency.</div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> In Vivo Efficacy of Compound <b>1</b> in Suppressing the Growth of PC-9 Xenograft Tumors in Nude Mice</h3><div class="NLM_p">Pursuant to the above in vitro findings, we evaluated the in vivo tumor-suppressive efficacy of compound <b>1</b> in an ectopic PC-9 xenograft tumor model. Athymic nude mice bearing established subcutaneous PC-9 tumors were randomly divided into two groups (<i>n</i> = 8 for each group; initial tumor volume: vehicle, 50 ± 11 mm<sup>3</sup>; compound <b>1</b>, 53 ± 6 mm<sup>3</sup>), and were treated once daily with compound <b>1</b> at 10 mg/kg or vehicle via intraperitoneal injection. As shown, compound <b>1</b> significantly suppressed tumor growth, as indicated by tumor volume and tumor weight, relative to the vehicle control after 25 days of treatment (***<i>P</i> < 0.001) (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>A). No appreciable weight loss was noted in the drug-treated group relative to control (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>B), suggesting that compound <b>1</b> did not show overt toxicity over the course of 25-day treatment. Moreover, Western blot analysis of tumor lysates showed that compound <b>1</b> treatment led to significant suppression of the expression of PKM2 and its binding partners EGFR, Stat3, and β-catenin, as compared to the vehicle control (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>C), indicating that tumor-suppressive activity correlated with the ability of compound <b>1</b> to target the expression of PKM2 and relevant nonmetabolic biomarkers.</div><figure id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b00763/20190920/images/medium/jm9b00763_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b00763/20190920/images/large/jm9b00763_0008.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. <i>In vivo</i> efficacy of compound <b>1</b> in suppressing the growth of PC-9 xenograft tumors in nude mice. (A) Suppressive effect of compound <b>1</b> at 10 mg/kg via daily i.p. injection on PC-9 xenograft tumor growth after 25 days of treatment. Top, representative images of vehicle- and compound <b>1</b>-treated PC-9 xenograft tumor-bearing mice and dissected tumor samples after 25 days of treatment. Bottom, effects of compound <b>1</b> versus vehicle control on tumor volumes (left), tumor weight (right), and body weight (B) in the course of treatment. Data are expressed as mean ± SD (<i>n</i> = 8). ***<i>P</i> < 0.001. (C) Left, Western blot analysis of the effects of compound <b>1</b> versus vehicle control on the expression of PKM2 and downstream targets, including PGK1, EGFR, Stat3 and β-catenin in tumor lysates. GAPDH as an internal control. Right, quantification of the ratio of protein expression level was normalized to GAPDH of tumor lysate by ImageJ tool. **<i>P</i> < 0.01, ***<i>P</i> < 0.001.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b00763/20190920/images/large/jm9b00763_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00763&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i12">Discussion and Conclusion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_82167" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_82167" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The past decade has witnessed an increasing interest in the development of irreversible inhibitors of the kinase cysteinome as the majority of kinases contains cysteine residues located near the catalytic domain. In principle, covalent kinase inhibitors offer several advantages over conventional ATP competitors, which include improved biochemical efficacy, high degree of selectivity, and favorable pharmacokinetic behaviors.<a onclick="showRef(event, 'ref30 ref31 ref32'); return false;" href="javascript:void(0);" class="ref ref30 ref31 ref32">(30−32)</a> The utility of this strategy is manifested by the FDA approval of four acrylamide-based covalent kinase inhibitors since 2013, including those targeting BTK (ibrutinib), EGFR (afatinib and osimertinib), and HER2 (neratinib), as well as many other irreversible inhibitors currently under preclinical development.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a></div><div class="NLM_p">In this study, we report the development of the first irreversible inhibitor of the glycolytic regulator PKM2. Compound <b>1</b> contains a propiolylamide electrophile at the terminus, which could selectively target Cys326 and Cys317 of PKM2. Accordingly, treatment of PC-9 cancer cells or recombinant PKM2 with compound <b>1</b> led to loss of kinase activity (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A and B). In contrast, recombinant PKM1 was not susceptible to the inhibitory effect of compound <b>1</b> on kinase activity (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B), suggesting the specificity of compound <b>1</b> in targeting PKM2 although PKM1 also contains Cys326 and Cys317. Interestingly, replacement of the propiolyl warhead with an acryloyl group substantially diminished the antiproliferative activity of the resulting compound (data not shown), suggesting the importance of the orientation of the electrophilic warhead in interacting with Cys326 and Cys317. In addition to these two cysteine residues, other neighboring cysteines have also been implicated in modulating PKM2 activity. It was reported that the natural product micheliolide could selectively activate PKM2 via covalent binding at Cys424,<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> and that elevated concentrations of reactive oxygen species caused inhibition of PKM2 though oxidation of Cys358.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> From a structural perspective, the dichotomous effect of micheliolide and compound <b>1</b> on the activity of PKM2 by selectively targeting cysteine residues (Cys424 versus Cys326/Cys317) near the catalytic domain is intriguing, which might be associated with the ability of these covalently modified cysteines to promote or destabilize PKM2. Based on our finding that exposure of PC-9 cells to compound <b>1</b> led to decreases in PKM2 expression (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>C), we hypothesize that covalent modifications of Cys326/Cys317 by compound <b>1</b> might facilitate the dissociation of the PKM2 complex, resulting in protein destabilization.</div><div class="NLM_p">Consistent with the role of PKM2 in promoting tumor growth and invasion,<a onclick="showRef(event, 'ref12 ref16'); return false;" href="javascript:void(0);" class="ref ref12 ref16">(12,16)</a> the ability of compound <b>1</b> to preferentially suppress the proliferation of cancer cell lines with more aggressive phenotype within the same tissue type is noteworthy (e.g., MDA-MB-231 versus MCF-7, PC-3 versus LNCaP, and EGFR mutant versus wild-type lung cancer cells). This differential effect was not corelated with the relative abundance of PKM2 among these cancer cell lines (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00763/suppl_file/jm9b00763_si_001.pdf" class="ext-link">Supplementary Figure S6</a>). It is well documented that aggressive cancer cells adopt glycolytic phenotype to gain growth advantage in face of adverse environments, and are more relying upon glycolysis for survival. Thus, glycolytic versus nonglycolytic phenotype, instead of the expression level of PKM2, might underlie the differential susceptibility to the antiproliferative activity of compound <b>1</b> between aggressive and noninvasive cancer cell lines.</div><div class="NLM_p">Mechanistically, the in vitro and in vivo tumor-suppressive effect of compound <b>1</b> is attributable to its ability to target both metabolic and nonmetabolic functions in cancer cells. Glycolysis stress test shows that compound <b>1</b> was effective in reversing the glycolytic phenotype, that is, concomitantly decreasing glycolysis and increasing oxidative capacities, of PC-9 cells. This metabolic reprogramming has been shown to play a critical role in the antitumor effect of other glycolysis inhibitors, including 2-deoxyglucose,<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> resveratrol,<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> and dichloroacetate.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> In addition, our data show that exposure of PC-9 cells to compound <b>1</b> led to downregulation of the expression and/or phosphorylation of many of PKM2’s interacting partners, including EGFR, Stat3, and β-catenin (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>C and <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a>C). This dual mechanism in metabolic and nonmetabolic targets might explain why the nonglycolytic MCF-7 and LNCaP cells were still susceptible, though with lower sensitivity, to the antiproliferative effect of compound <b>1</b>.</div><div class="NLM_p">Although the role of PKM2 in cancer<a onclick="showRef(event, 'ref36 ref37 ref38'); return false;" href="javascript:void(0);" class="ref ref36 ref37 ref38">(36−38)</a> or as a protein kinase<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> remains controversial, increasing evidence suggests that this discrepancy might arise from the metabolic heterogeneity in tumor cells, i.e., PKM2 might mediates its biological functions in a cancer type- or context-specific manner.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> From a mechanistic perspective, compound <b>1</b> might serve as a useful pharmacological probe to shed light onto the biological function of PKM2 in tumor cells with different genetic characteristics.</div><div class="NLM_p">In addition to PKM2, proteomic analysis of compound <b>2</b>-tagged proteins also revealed a number of other enzymes intimately involved in different energy metabolic processes as putative targets (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A). It is possible that some of these proteins might be subjected to streptavidin pulldown due to their interactions or complex formation with PKM2 inside cancer cells (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>B). However, one cannot rule out the possibility that some other proteins might also be relevant targets of compound <b>1</b> through irreversible binding, which warrants clarification. For example, LDHA, which had the second highest matching score (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A), has been reported to contain a cysteine residue (Cys165) essential for its catalytic activity.<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> In light of the pivotal role of LDHA in tumorigenesis,<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> investigation of LDHA as a potential target of compound <b>1</b> is currently under investigation.</div><div class="NLM_p last">In conclusion, compound <b>1</b> might have translational potential to foster new strategies for cancer therapy, of which the anticipated advantage over noncovalent PKM2 regulators is multifold. For example, similar to other reported irreversible kinase inhibitors, compound <b>1</b> exhibited a high degree of selectivity according to data from kinase profiling assays (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00763/suppl_file/jm9b00763_si_001.pdf" class="ext-link">Tables S1 and S2</a>). Kinase activity assays also indicated that compound <b>1</b> could selectively inhibit PKM2 without affecting PKM1. This high degree of target selectivity has been associated with less toxicity of irreversible kinase inhibitors. In addition, compound <b>1</b> exhibited high in vivo efficacy in suppressing tumor growth. Thus, further preclinical development of compound <b>1</b> as a cancer therapeutic agent is currently under way.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i13">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_26080" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_26080" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> General</h3><div class="NLM_p">3-(1-(4-Aminophenyl)-5-(4′-(trifluoromethyl)-[1,1′-biphenyl]-4-yl)-1<i>H</i>-pyrazol-3-yl)-<i>N</i>-methylpropanamide (<b>i</b>) was prepared according to a published procedure.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> All commercially available reagents were used without further purification unless otherwise stated. Anhydrous THF was obtained by distilling commercial THF over calcium hydride, and anhydrous DMF was obtained by distillation over P<sub>2</sub>O<sub>5</sub> under reduced pressure. Silica gel for column chromatography was purchased from Fisher Scientific (230–400 mesh). Routine <sup>1</sup>H and <sup>13</sup>C nuclear magnetic resonance spectra were recorded on a Bruker AV400 or AVII 500 spectrometer. Samples were dissolved in deuterated chloroform (CDCl<sub>3</sub>) or dimethyl sulfoxide (DMSO-<i>d</i><sub>6</sub>) with tetramethylsilane (TMS) as a reference. Electrospray ionization mass spectrometry analyses were performed on a Bruker maXis 4G mass spectrometer. All biologically evaluated compounds were shown in exist in greater than 95% purity by the following methods. Purity of compound <b>1</b> was confirmed by HPLC, which was measured by a reverse phase HPLC system column: Merck 50995 Lichrospher 100 RP18; column temperature (°C) 20; column length (mm), 250–4 mm end-capped; column internal diameter (mm), 5; detector, Jasco MD-910; mobile phase, MeOH: H<sub>2</sub>O = 80:20; injection volume (μL), 20; flow rate, 0.5 mL/min. Quantitative 1D <sup>1</sup>H NMR (qNMR) was conducted to confirm the purity of compound <b>2</b> by following the Journal’s guidelines (purity by absolute qNMR <a href="http://pubs.acs.org/paragonplus/submission/jmcmar/jmcmar_purity_instructions.pdf" class="extLink">http://pubs.acs.org/paragonplus/submission/jmcmar/jmcmar_purity_instructions.pdf</a>). All <sup>1</sup>H and <sup>13</sup>C NMR spectra and MS spectra of compounds <b>1</b> and <b>2</b> and all pertinent intermediates and purity determination data of compounds <b>1</b> and <b>2</b> are appended in the <a class="ref internalNav" href="#notes-5" aria-label="Supporting Information">Supporting Information</a>.</div><div id="sec4_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i15" class="anchor-spacer"></div><h4 class="article-section__title" id="_i15"> <i>N</i>-(4-(3-(3-(Methylamino)-3-oxopropyl)-5-(4′-(trifluoromethyl)-[1,1′-biphenyl]-4-yl)-1<i>H</i>-pyrazol-1-yl)phenyl)propiolamide (<b>1</b>)</h4><div class="NLM_p last">To an ice-cold solution of intermediate <b>i</b> (92.8 mg, 0.2 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (4 mL) was added <i>O</i>-(benzotriazol-1-yl)-<i>N,N,N′,N</i>′-tetramethyluronium tetrafluoroborate (TBTU) (257 mg, 0.8 mmol), followed by propynoic acid (61 μL, 1.0 mmol, 5.0 equiv). The reaction mixture was stirred at room temperature for 4–5 days under argon, and concentrated. The residue was purified by flash column chromatography to afford 69 mg (67%) of compound <b>1</b> as an off-white solid, of which the purity was determined to be of 97.97% by HPLC. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): δ 8.01 (s, 1H), 7.91 (d, <i>J</i> = 8.0 Hz, 2H), 7.73–7.62 (m, 8H), 7.47 (d, <i>J</i> = 8.5 Hz, 2H), 6.56 (s, 1H), 5.51 (s, 1H), 3.05 (t, <i>J</i> = 7.5 Hz, 2H), 2.94 (s, 1H), 2.79 (d, <i>J</i> = 5.0 Hz, 3H), 2.47 (t, <i>J</i> = 7.7 Hz, 2H). <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>): δ 171.5, 150.3, 150.2, 145.1, 144.2, 138.3, 138.1, 135.8, 133.7, 128.2, 127.8, 127.7, 126.3, 126.2, 126.1, 125.9, 123.8, 120.7, 103.4, 79.6, 78.8, 77.9, 34.4, 25.9, 22.4. HRMS (ESI): calcd. for C<sub>29</sub>H<sub>23</sub>N<sub>4</sub>O<sub>2</sub>F<sub>3</sub>Na, [M + Na]<sup>+</sup> 539.1671 Da, found 539.1672 Da.</div></div><div id="sec4_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i16" class="anchor-spacer"></div><h4 class="article-section__title" id="_i16"> <i>N</i>-(4-(3-(3-(Methylamino)-3-oxopropyl)-5-(4′-(prop-2-yn-1-yloxy)-[1,1′-biphenyl]-4-yl)-1<i>H</i>-pyrazol-1-yl)phenyl)propiolamide (<b>2</b>)</h4><div class="NLM_p"><i>N</i>-(4-(3-(3-(methylamino)-3-oxopropyl)-5-(4′-(prop-2-yn-1-yloxy)-[1,1′-biphenyl]-4-yl)-1H-pyrazol-1-yl)phenyl)propiolamide (<b>2</b>) was synthesized as follows (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>C):</div><div id="sec4_1_2_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i17" class="anchor-spacer"></div><h5 class="article-section__title" id="_i17"> 1-(4′-Methoxy-[1,1′-biphenyl]-4-yl)ethan-1-one (<b>ii</b>) (Step a)</h5><div class="NLM_p last">4-Methoxy phenylboronic acid (3.0 g, 19.7 mmol) was added to a solution of 4-bromo-acetophenone (4.3 g, 21.7 mmol), palladium(II) acetate (88 mg, 2 mol %), potassium carbonate (8.15 g, 59.1 mmol), and tetrabutylammonium bromide (TBAB; 8.25 g, 25.6 mmol). To the reaction mixture was added water (200 mL), heated to 60 °C with stirring under argon for 2 h, cooled, diluted with water, and extracted with ethyl acetate. The organic layer was dried and concentrated, and the residue was purified by flash column chromatography (ethyl acetate: hexane, 3: 7) to afford <b>ii</b> (4.0 g, 90% yield). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): δ 8.07 (d, <i>J</i> = 8.5 Hz, 2H). 7.64 (d, <i>J</i> = 8.0 Hz, 2H), 7.57 (d, <i>J</i> = 9.0 Hz, 2H), 7.00 (d, <i>J</i> = 8.5 Hz, 2H), 3.86 (s, 3H), 2.62 (s, 3H). <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>): δ 197.7, 159.9, 145.4, 135.3, 132.3, 128.9, 128.4, 126.6, 114.4, 55.4, 26.6. HRMS (EI): calcd for C<sub>15</sub>H<sub>14</sub>O<sub>2</sub>, [M]<sup>+</sup> 226.0994 Da, found 226.0992 Da.</div></div><div id="sec4_1_2_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i18" class="anchor-spacer"></div><h5 class="article-section__title" id="_i18"> Ethyl 4-(1<i>H</i>-Benzo[<i>D</i>][1,2,3]triazol-1-yl)-4-oxobutanoate (<b>iii</b>) (Step b)</h5><div class="NLM_p last">To an ice-cooled solution of benzotriazole (10 g, 84 mmol) and triethylamine (17.6 mL, 126 mmol) in dichloromethane (DCM; 200 mL) was added ethyl-4-chloro-4-oxobutyrate (13.8 g, 84 mmol) slowly. The resulting mixture was brought to room temperature and stirred overnight. Observed white salt was filtered off and washed with 2 N HCl (2 × 200 mL), followed by a wash with brine (200 mL). The organic layer was dried over Na<sub>2</sub>SO<sub>4</sub>, concentrated, and dried under vacuum to give compound <b>iii</b>, which was used directly for the next step without purification.</div></div><div id="sec4_1_2_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i19" class="anchor-spacer"></div><h5 class="article-section__title" id="_i19"> Ethyl (<i>Z</i>)-4-Hydroxy-6-(4′-methoxy-[1,1′-biphenyl]-4-yl)-6-oxohex-4-enoate (<b>iv</b>) (Step c)</h5><div class="NLM_p last">To a solution of compound <b>ii</b> (3.0 g, 13.25 mmol) in dry DCM (150 mL) was added compound <b>iii</b> (3.9 g, 15.9 mmol) and magnesium bromide ethyl etherate (6.8 g, 26.5 mmol) under argon. The resulting solution was stirred under argon for 10 min, added dropwise <i>N</i>,<i>N</i>-diisopropylethylamine (DIPEA; 4.6 mL, 26.5 mmol), stirred for 16 h, and washed, in tandem, with 10% 2 N HCl (150 mL x 1) and water (200 mL × 2). The organic phase was dried and concentrated. The residue was purified by chromatography (ethyl acetate:hexane, 9:1), followed by recrystallization in ethanol to give compound <b>iv</b> (3.2 g, 68% yield). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): δ 7.91 (d, <i>J</i> = 8.5 Hz, 2H), 7.63 (d, <i>J</i> = 8.5 Hz, 2H), 7.57 (d, <i>J</i> = 8.5 Hz, 2H), 7.00 (d, <i>J</i> = 9.0 Hz, 2H), 6.23 (s, 1H), 4.17 (q, <i>J</i> = 7.0 Hz, 2H), 3.86 (s, 3H), 2.82 (t, <i>J</i> = 7.0 Hz, 2H), 2.70 (t, <i>J</i> = 7.0 Hz, 2H), 1.27 (t, <i>J</i> = 7.0 Hz, 3H). <sup>13</sup>C NMR (125 MHz, CDCl3): δ 196.6, 180.4, 172.6, 159.9, 144.6, 132.4, 132.3, 128.4, 128.3, 127.4, 126.8, 126.7, 114.4, 96.0, 60.7, 55.4, 34.5, 29.2, 14.2. HRMS (ESI): calcd. for C<sub>21</sub>H<sub>22</sub>O<sub>5</sub>Na, [M + Na]<sup>+</sup> 377.1365 Da, found 377.1356 Da.</div></div><div id="sec4_1_2_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i20" class="anchor-spacer"></div><h5 class="article-section__title" id="_i20"> Ethyl 3-(5-(4′-Methoxy-[1,1′-biphenyl]-4-yl)-1-(4-nitrophenyl)-1<i>H</i>-pyrazol-3-yl) Propanoate (<b>v</b>) (Step d)</h5><div class="NLM_p last">To a solution of compound <b>iv</b> (3.0 g, 8.5 mmol) in ethanol (40 mL) was added 4-nitro-phenylhydrazine hydrochloride (1.92 g, 10.2 mmol) and <i>p</i>-toluenesulfonic acid (PTSA; 1.6 g, 8.5 mmol). The resulting solution was heated to reflux with stirring for overnight, cooled to room temperature, filtered and washed with ethanol to give compound <b>v</b> as yellow crystal (3.0 g, 75% yield). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): δ 8.37 (d, <i>J</i> = 9.0 Hz, 2H). 7.88 (d, <i>J</i> = 8.0 Hz, 2H), 7.78 (d, <i>J</i> = 9.0 Hz, 2H), 7.61 (d, <i>J</i> = 8.0 Hz, 2H), 7.57 (d, <i>J</i> = 8.5 Hz, 2H), 6.99 (d, <i>J</i> = 8.5 Hz, 2H), 6.34 (s, 1H), 4.15 (q, <i>J</i> = 7.0 Hz, 2H), 3.86 (s, 3H), 3.13 (t, <i>J</i> = 7.5 Hz, 2H), 2.74 (t, <i>J</i> = 7.5 Hz, 2H), 1.25 (t, <i>J</i> = 7.0 Hz, 3H). <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>): δ 171.7, 159.3, 152.7, 146.3, 143.7, 140.8, 133.1, 130.7, 128.0, 126.9, 126.2, 124.8, 114.3, 104.6, 60.9, 55.4, 33.1, 22.2, 14.2. HRMS (EI): calcd for C<sub>27</sub>H<sub>25</sub>N<sub>3</sub>O<sub>5</sub>, [M]<sup>+</sup> 471.1794 Da, found 471.1801 Da.</div></div><div id="sec4_1_2_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i21" class="anchor-spacer"></div><h5 class="article-section__title" id="_i21"> 3-(5-(4′-Methoxy-[1,1′-biphenyl]-4-yl)-1-(4-nitrophenyl)-1<i>H</i>-pyrazol-3-yl)-<i>N</i>-methyl-propanamide (<b>vi</b>) (Step e)</h5><div class="NLM_p last">To a solution of compound <b>v</b> (3.0 g) in ethanol (15 mL) was added 1 M methylamine in ethanol solution (10 mL). The resulting solution was heated to 120 °C with stirring in a sealed tube for 16 h and concentrated, followed by recrystallization in ethanol to give compound <b>vi</b> as yellow powder (2 g, 69%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.40 (d, <i>J</i> = 8.4 Hz, 2H). 8.13 (s, 1H), 7.96–7.90 (m, 4H), 7.71–7.65 (m, 4H), 7.03 (d, <i>J</i> = 8.0 Hz, 2H), 6.95 (s, 1H), 3.80 (s, 3H), 3.07 (t, <i>J</i> = 7.2 Hz, 2H), 2.56 (bs, 5H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 171.5, 159.5, 151.9, 146.2,145.8, 144.9, 140.0, 132.4, 131.2, 128.1, 126.9, 126.4, 125.3, 125.2, 114.8, 105.1, 55.7, 34.1, 25.9, 22.9. HRMS (ESI-negative): calcd. for C<sub>26</sub>H<sub>23</sub>N<sub>4</sub>O<sub>4</sub>, [M–H]<sup>−</sup> 455.1719 Da, found 455.1714 Da.</div></div><div id="sec4_1_2_6" class="NLM_sec NLM_sec_level_4"><div id="ac_i22" class="anchor-spacer"></div><h5 class="article-section__title" id="_i22"> 3-(5-(4′-Hydroxy-[1,1′-biphenyl]-4-yl)-1-(4-nitrophenyl)-1<i>H</i>-pyrazol-3-yl)-<i>N</i>-methyl Propanamide (<b>vii</b>) (Step f)</h5><div class="NLM_p last">BBr<sub>3</sub> (1.0 M in DCM], 17.52 mL, 17.52 mmol) was added dropwise into ice cold mixture of compound <b>vi</b> (2 g, 4.38 mmol) in DCM (25 mL). After the mixture was stirred at room temperature for 90 min, progress of reaction was monitored by TLC. After the complete consumption of compound <b>v</b>, the mixture was cooled to 0 °C, and then ice water was added slowly. The resulting yellow solid was filtered and washed with DCM, followed by a wash with ethyl acetate, to obtain pure compound <b>vii</b> (1.2 g, 62%). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.68 (s, 1H), 8.41 (d, <i>J</i> = 9.2 Hz, 2H). 7.95 (d, <i>J</i> = 8.8 Hz, 2H), 7.89 (d, <i>J</i> = 8.4 Hz, 2H), 7.87 (d, <i>J</i> = 8.0 Hz, 2H), 7.67 (d, <i>J</i> = 8.0 Hz, 2H), 7.55 (d, <i>J</i> = 8.4 Hz, 2H), 6.91 (s, 1H), 6.87 (d, <i>J</i> = 8.8 Hz, 2H), 3.07 (t, <i>J</i> = 7.6 Hz, 2H), 2.57 (d, <i>J</i> = 4.8 Hz, 3H), 2.53 (t, <i>J</i> = 7.6 Hz, 2H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 171.3, 157.7, 151.9, 146.1, 145.7, 144.8, 140.3, 130.7, 130.6, 127.9, 126.5, 126.2, 125.2, 125.1, 116.1, 104.9, 34.0, 25.8, 22.7. HRMS (ESI-negative): calcd for C<sub>25</sub>H<sub>21</sub>N<sub>4</sub>O<sub>4</sub>, [M–H]<sup>−</sup> 441.1563 Da, found 441.1563 Da.</div></div><div id="sec4_1_2_7" class="NLM_sec NLM_sec_level_4"><div id="ac_i23" class="anchor-spacer"></div><h5 class="article-section__title" id="_i23"> <i>N</i>-Methyl-3-(1-(4-nitrophenyl)-5-(4′-(prop-2-yn-1-yloxy)-[1,1′-biphenyl]-4-yl)-1<i>H</i>-pyrazol-3-yl)propanamide (<b>viii</b>) (Step g)</h5><div class="NLM_p last">To a solution of compound <b>vii</b> (1.2 g, 2.7 mmol) in acetone (15 mL) was added K<sub>2</sub>CO<sub>3</sub> (748 mg, 5.4 mmol), followed by propargyl bromide (80% solution in toluene; 0.86 mL, 5.4 mmol). The resulting mixture was refluxed for 24 h, cooled to room temperature, and concentrated under reduced pressure. Water was added to the residue; the mixture was sonicated, and the resulting yellow solid was filtered off, washed several times with water, and dried to obtain compound <b>viii</b> (1.2 g, 92% yield). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): δ 8.38 (d, <i>J</i> = 9.0 Hz, 2H). 7.88 (d, <i>J</i> = 8.0 Hz, 2H), 7.80 (d, <i>J</i> = 8.5 Hz, 2H), 7.61 (d, <i>J</i> = 8.0 Hz, 2H), 8.58 (d, <i>J</i> = 8.5 Hz, 2H). 7.07 (d, <i>J</i> = 8.5 Hz, 2H), 6.62 (s, 1H), 5.45 (s, 1H), 4.75 (d, <i>J</i> = 1.5 Hz, 2H), 3.19 (t, <i>J</i> = 7.5 Hz, 2H), 2.83 (d, <i>J</i> = 5.0 Hz, 3H), 2.57 (t, <i>J</i> = 7.5 Hz, 2H), 2.55 (s, 1H). HRMS (ESI-negative): calcd for C<sub>28</sub>H<sub>23</sub>N<sub>4</sub>O<sub>4</sub>, [M–H]<sup>−</sup> 479.1719 Da, found 479.1717 Da.</div></div><div id="sec4_1_2_8" class="NLM_sec NLM_sec_level_4"><div id="ac_i24" class="anchor-spacer"></div><h5 class="article-section__title" id="_i24"> 3-(1-(4-Aminophenyl)-5-(4′-(prop-2-yn-1-yloxy)-[1,1′-biphenyl]-4-yl)-1<i>H</i>-pyrazol-3-yl)-<i>N</i>-methylpropanamide (<b>ix</b>) (Step h)</h5><div class="NLM_p last">To a solution of compound <b>viii</b> (1.2 g, 2.5 mmol) in ethyl acetate (60 mL) was added SnCl<sub>2</sub>·H<sub>2</sub>O (3.38 g, 14.98 mmol) under argon. The resulting mixture was heated to reflux with stirring under argon for 6–8 h and was cooled to room temperature. The reaction mixture was diluted with ethyl acetate and washed with a saturated solution of NaHCO<sub>3</sub>. The aqueous layer was extracted with ethyl acetate and combined with the organic layer. The combined solution was dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated to get pure compound <b>ix</b> (1.1 g, 97% yield). <sup>1</sup>H NMR (400 MHz, CDCl3): δ 7.87 (d, <i>J</i> = 8.4 Hz, 2H). 7.58 (d, <i>J</i> = 8.4 Hz, 4H), 7.24 (d, <i>J</i> = 10.4 Hz, 2H), 7.05 (d, <i>J</i> = 8.8 Hz, 2H), 6.74 (d, <i>J</i> = 8.4 Hz, 2H), 6.51 (s, 1H), 5.42 (s, 1H), 4.74 (d, <i>J</i> = 2.4 Hz, 2H), 3.00 (t, <i>J</i> = 7.6 Hz, 2H), 2.79 (d, <i>J</i> = 5.2 Hz, 3H), 2.54 (t, <i>J</i> = 2.4 Hz, 1H), 2.44 (t, <i>J</i> = 7.8 Hz, 2H).</div></div><div id="sec4_1_2_9" class="NLM_sec NLM_sec_level_4"><div id="ac_i25" class="anchor-spacer"></div><h5 class="article-section__title" id="_i25"> <i>N</i>-(4-(3-(3-(Methylamino)-3-oxopropyl)-5-(4′-(prop-2-yn-1-yloxy)-[1,1′-biphenyl]-4-yl)-1<i>H</i>-pyrazol-1-yl)phenyl)propiolamide (<b>2</b>) (Step i)</h5><div class="NLM_p last">To an ice-cold solution of compound <b>ix</b> (1.1 g, 2.44 mmol) in DCM (20 mL) was added TBTU (3.1 g, 9.76 mmol), followed by the addition of propynoic acid (855 mg, 12.20 mmol). The reaction mixture was stirred at room temperature for 4–5 days under argon and concentrated under reduced pressure. The residue was purified by flash column chromatography to afford 150 mg of compound <b>2</b> as an off-white solid of which the purity was determined to be of 97.3% by qNMR. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.04 (s, 1H), 7.86 (d, <i>J</i> = 8.5 Hz, 3H), 7.77 (d, <i>J</i> = 8.5 Hz, 2H), 7.68 (t, <i>J</i> = 8.5 Hz 4H), 7.54 (d, <i>J</i> = 9.0 Hz, 2H), 7.08 (d, <i>J</i> = 8.5 Hz, 2H), 6.76 (s, 1H), 4.85 (d, <i>J</i> = 2.5 Hz, 2H), 4.48 (s, 1H), 3.59 (t, <i>J</i> = 2.0 Hz, 1H), 2.90 (t, <i>J</i> = 7.5 Hz, 2H), 2.55 (d, <i>J</i> = 4.5 Hz, 3H), 2.45 (t, <i>J</i> = 7.5 Hz, 2H). <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>): δ 171.0, 156.9, 149.9, 149.7, 144.4, 138.9, 137.7, 135.3, 132.8, 131.5, 127.5, 126.4, 125.7, 125.6, 120.1, 115.3, 102.7, 79.2, 78.3, 77.5, 55.5, 33.8, 25.5, 21.9. HRMS (ESI): calcd for C<sub>31</sub>H<sub>26</sub>N<sub>4</sub>O<sub>3</sub>Na, [M + Na]<sup>+</sup> 525.1903 Da, found 525.1902 Da.</div></div></div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> Cell Lines, Cell Culture, Biochemical Reagents, and Antibodies</h3><div class="NLM_p last">MiaPaCa-2, AsPC-1, and Panc-1 human pancreatic cancer cells, MCF-7 and MDA-MB-231 breast cancer cells, SCC4 and SCC2059 oral cancer cells, PC-3 and LNCaP prostate cancer cells, and H157, H460, H1975, PC-9, and H1650 lung cancer cells were obtained from the American Type Culture Collection (ATCC, Manassas, VA). All these cells were maintained in recommended growth medium (RPMI 1640 or DMEM), supplemented with 10% fetal bovine serum (FBS) (Invitrogen, Carlsbad, CA, USA) and antibiotics, at 37 °C in a humidified incubator containing 5% CO<sub>2</sub>. For SILAC experiments, PC-9 cells were grown in SILAC DMEM medium (Thermo Fisher) supplemented with 10% dialyzed FBS and 100 μg/mL each of unlabeled <span class="smallcaps smallerCapital">l</span>-arginine and <span class="smallcaps smallerCapital">l</span>-lysine (light medium) or <sup>13</sup>C/<sup>15</sup>N stable isotope-labeled <span class="smallcaps smallerCapital">l</span>-arginine and <span class="smallcaps smallerCapital">l</span>-lysine (heavy medium). PC-9 cells were passaged at least six times in isotope-containing DMEM medium before being used for analysis by LC-MS/MS.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> Preparation of Recombinant PKM2</h3><div class="NLM_p last">A DNA fragment encoding full-length human PKM2 was prepared by using codon-optimized human PKM2 gene and was inserted into an XhoI-BamH1 cutting site of the pET15b vector with a N-terminal 6xHis-tagged PKM2 that encompassed a DNA sequence including codons predominantly used in <i>Escherichia coli</i> but do not result in a change of the amino acid sequence. The expression plasmid for PKM2 was transformed into <i>E. coli</i> BL21(DE3). After bacterial growth in LB medium at 37 °C reaching 1.2 OD<sub>600</sub>, protein expression was induced by incubating with 0.5 mM isopropyl β-D-α-thiogalactopyranoside at 16 °C for 20 h. Cells were harvested by centrifugation and stored at −80 °C until further use. The <i>E. coli</i> cell pellets were resuspended in lysis buffer (10 mM Tris-HCl, pH 7.0, 5% glycerol, 100 mM KCl, 1 mM DTT, and 10 mM imidazole), and cells were disrupted by sonication. The crude cell extract was centrifuged at 27 216<i>g</i> at 4 °C for 120 min and applied to a pre-equilibrated Ni-NTA column. The column was washed to baseline, and the protein was eluted with elution buffer (lysis buffer containing 500 mM imidazole). The eluted fractions were pooled and purified on a Hi-Load Superdex 200 size-exclusion column in a buffer containing 10 mM HEPES (pH7.5), 100 mM KCl, 5% glycerol, 5 mM MgCl<sub>2</sub>, and 2 mM DTT. The PKM2 protein fractions were pooled and concentrated to a concentration of approximately 15 mg/mL for further analysis.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> Cell Viability Assays</h3><div class="NLM_p last">Drug effects on cell viability was assessed by 3-(4,5-dimethylthiazol-2-yl)-2,5-diphenyltetrazolium bromide (MTT) assays. Cancer cells were seeded into 96-well plates at a density of 3500 cells per well in the presence of 10% FBS. After overnight incubation, cells were exposed to test agents vis-à-vis vehicle in the presence of 5% FBS for 24 h. After treatment, cells were incubated with MTT (Biomatik, Wilmington, DE) for an additional 1 h. The medium was then removed from each well and replaced with DMSO to dissolve the reduced MTT dye for subsequent colorimetric measurement of absorbance at 560 nm. Cell viabilities are expressed as percentages of viable cells relative to the corresponding vehicle-treated control group.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> SILAC Analysis and Click Chemistry</h3><div class="NLM_p last">PC-9 cells (2 × 10<sup>6</sup> cells/plate) were seeded in SILAC DMEM growth medium containing unlabeled <span class="smallcaps smallerCapital">l</span>-arginine and <span class="smallcaps smallerCapital">l</span>-lysine (light medium) or <sup>13</sup>C/<sup>15</sup>N stable isotope-labeled <span class="smallcaps smallerCapital">l</span>-arginine and <span class="smallcaps smallerCapital">l</span>-lysine (heavy medium) in 10 cm plates. After 16 h, the SILAC DMEM medium was aspirated off, and cells were washed twice with Dulbecco’s phosphate-buffered saline (DPBS), followed by the addition of, in tandem, fresh DMEM medium and compound <b>1</b> (μM) or DMSO. After incubation at 37 °C for 1 h, cells were collected, washed three times with DPBS, suspended in cold DPBS, containing a protease inhibitor cocktail (Roche), and lysed by a probe sonicator, followed by centrifugation at 100 000<i>g</i> for 30 min. The supernatant was transferred to a 1.5 mL microfuge tube, and the protein concentration was determined by the BCA protein assay (Thermo Fisher Scientific) and then normalized to 1 mg/mL. Heavy and light cell lysates at 500 μg each were mixed, and the combined cell lysates were subjected to the copper-catalyzed Click reaction for 1 h with 40 μM biotin-azide (PEG4 carboxamide-6-azidohexanyl biotin) using the Click-IT protein reaction buffer kit (Invitrogen) according to the manufacturer’s instruction. Biotin-tagged proteins were then treated with 50 μL of streptavidin-coupled magnetic beads (Dynabeads M-280 Streptavidin, Invitrogen) for 16 h at 4 °C. After incubation, cell lysates were washed by PBS containing 0.5% SDS three times to remove nonspecific binding. After they were washed, the streptavidin-coupled magnetic beads were incubated in 30 μL of 2 × loading buffer for 10 min at 100 °C to separate out the targeted proteins from beads. The pulled-down proteins were subjected to 10% SDS-PAGE, and then stained with sliver staining before processing for LC-MS/MS.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> Proteomic Data Analysis</h3><div class="NLM_p last">Putative target proteins of compound <b>1</b> and protein–protein interaction network were identified and analyzed, respectively, through a web-based search of the STRING database (<a href="https://string-db.org/" class="extLink">https://string-db.org/</a>). The PKM2 interaction protein network was functionally characterized by using the Gene Ontology analysis for biological processes.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i31" class="anchor-spacer"></div><h3 class="article-section__title" id="_i31"> Preparation of Anticompound <b>1</b> Antiserum</h3><div class="NLM_p last">Compound <b>1</b> was coupled to the cysteine thiolate in ovalbumin (OVA) via the propiolyl moiety under alkaline conditions by using the modifications to a published method.<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> In brief, two milliliters of OVA at 2 mg/mL in phosphate-buffered saline (PBS) was treated with 50 mM 1,4-dithioerytreitol at 37 °C for 1 h, followed by the addition of, in tandem, 2 mL of 20% trichloroacetic acid and 20 mL of ice-cold acetone. The mixture was kept at −20 °C overnight, and the resulting precipitate was collected by low-speed centrifugation and was dissolved in 2 mL of 8 M urea in 0.1 M sodium carbonate buffer, pH 9.4, containing 4 mg of compound <b>1</b>. The solution was incubated at 37 °C for 4 h and buffer-exchanged into PBS by centrifugal concentration using an Amicon device with a cutoff of 10 kDa (MilliporeSigma, Burlington, MA, USA) and was then used for routine subcutaneous immunizations in guinea pigs. Following six biweekly injections, whole blood was collected from the anesthetized animals 10 days after the final injection.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> Pyruvate Kinase (PK) Activity Assays</h3><div class="NLM_p last">PC-9 cells were seeded into 6 cm dishes at a concentration of 5 × 10<sup>5</sup> cells/mL. After 24 h, cells were treated with 0.25 μM compound <b>1</b> or DMSO for 6 h. The PK activity of the lysates of compound <b>1</b>-treated PC-9 and vehicle control cells was measured by using a commercial colorimetric assay kit from BioVision (Milpitas, CA, USA) according to the manufacturer’s protocol. This PK assay was based on the measurement of H<sub>2</sub>O<sub>2</sub> produced via two consecutive enzymatic reactions, that is, PK first catalyzed the production of pyruvate and ATP from PEP and ADP, followed by pyruvate oxidase to convert pyruvate in the presence of phosphate and O<sub>2</sub> to acetyl phosphate, CO<sub>2</sub> and H<sub>2</sub>O<sub>2</sub>, and the level of H<sub>2</sub>O<sub>2</sub> was determined colorimetrically at 570 nm. In addition, to verify the specificity of compound <b>1</b> toward PKM2 versus PKM1, the inhibition of the kinase activity of recombinant PKM1 (SRP0415, Sigma-Aldrich) and recombinant PKM2 (SAE0021, Sigma-Aldrich) by compound <b>1</b> was conducted using the aforementioned assay kit. In brief, recombinant PKMs were incubated with compound <b>1</b> at room temperature for 20 min, and the remaining PK activity was measured accordingly.</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i33" class="anchor-spacer"></div><h3 class="article-section__title" id="_i33"> Gel-Based ABPP</h3><div class="NLM_p last">PC-9 cells (2 × 10<sup>6</sup>) were seeded in DMEM growth medium in 10 cm plates. After overnight incubation, the growth medium was replaced with fresh medium, followed by the addition of 1 μM compound <b>1</b> or DMSO control. After 1 h, the ABPP probe compound <b>2</b> at 0.5, 1, and 2 μM was then added, and the mixture was incubated at 37 °C for an additional 1 h. The cell lysates were subjected to the copper-catalyzed Click reaction, and the biotin-tagged proteins were treated with 50 μL of streptavidin-coupled magnetic beads for 16 h at 4 °C. After incubation, cell lysates were washed with PBS, containing 0.5% SDS, three times to remove nonspecific binding. After they were washed, the streptavidin-coupled magnetic beads were incubated in 30 μL of 2× loading buffer for 10 min at 100 °C to separate out bound proteins from beads, which were then subjected to 10% SDS-PAGE.</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34"> In-Solution Tryptic Digestion of PKM2 for Mass Spectral Analysis</h3><div class="NLM_p last">PKM2 protein was reduced by 30 mM dithiothreitol at 37 °C for 1 h and then alkylated by treating with 30 mM iodoacetamide at room temperature in the dark for 1 h. The alkylated protein was diluted 4-fold with 25 mM ammonium bicarbonate buffer (pH 8.5), and then, it was incubated overnight at 37 °C with sequencing-grade modified trypsin (Promega, Madison, WI, USA) at an enzyme to substrate ratio of 1:30 (w/w). The tryptic peptides were dried completely under vacuum. The peptide mixtures were desalted by C18 Zip-tip (Millipore) and subjected to mass spectrometric analysis.</div></div><div id="sec4_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i35" class="anchor-spacer"></div><h3 class="article-section__title" id="_i35"> NanoLC-MS/MS Analysis and MS/MS Database Searching</h3><div class="NLM_p">The tryptic peptides were analyzed on an LTQ-Orbitrap Fusion mass spectrometer (Thermo Fisher Scientific, San Jose, CA) coupled to an Agilent 1100 Series binary high-performance liquid chromatography pump (Agilent Technologies, Palo Alto, CA, USA), and a FAMOS autosampler (LC Packing, San Francisco, CA, USA). A total of 5 μL of samples were injected into a manually packed precolumn (150 μm ID × 30 mm, 5 μm, 200 Å) at a 10 μL/min flow rate. Chromatographic separation was performed over 60 min on a manually packed reversed phase C18 nanocolumn (75 μm ID × 200 mm, 3 μm, 200 Å) using 0.1% formic acid in water as mobile phase A, 0.1% formic acid in 80% acetonitrile as mobile phase B, and a split flow rate of 300 nL/min. The dynamics exclusion duration was set at 120 s, with a range in mass tolerance of ±25 ppm. The scan sequence began with an MS1 spectrum (Orbitrap analysis; resolution 120 000 at 200 <i>m</i>/<i>z</i>; mass range 200–2000 <i>m</i>/<i>z</i>; automatic gain control (AGC) was set to accumulate 2 × 10<sup>5</sup> ions, with a maximum injection time of 200 ms). The most-abundant MS1 ions of charge states 2–7 were selected and fragmented using a top-speed approach (cycle time of 3 s). MS2 analysis was composed of higher-energy C-trap dissociation (HCD) (Orbitrap analysis; AGC 5 × 10<sup>4</sup>; normalized collision energy (NCE) 28; maximum injection time 250 ms).</div><div class="NLM_p last">All MS and MS/MS raw data were processed with Proteome Discoverer version 2.1 (Thermo Scientific), and the peptides were identified from the MS/MS data searched against the target PKM2 human protein sequence (UniProtKB, P14618-1, isoform M2 of Pyruvate kinase PKM) database using the Mascot search engine 2.3.02 (Matrix Science). Search criteria used were as follows: trypsin digestion; considered variable modifications of serines, threonines, and tyrosines phosphorylation (+79.9663 Da), cysteine compound <b>1</b>-modification (peptides molecular +516.1773 Da), glutamine deamidation (+ 0.98402 Da), methionine oxidation (+15.9949 Da), and cysteine carboxyamidomethylation (+57.0214 Da); up to three missed cleavages were allowed; and mass accuracy of 5 ppm for the parent ion and 0.05 Da for the fragment ions. The significant peptide hits, defined as peptide score, must be higher than the Mascot significance threshold (*<i>P</i> < 0.05) and, therefore, are considered to be highly reliable, and manual interpretation confirmed the agreement between spectra and peptide sequence. The false discovery rate (FDR) of the peptides and protein groups was set to 1% for the MS/MS spectra automatically processed by Proteome Discoverer for statistical validation and quantification.</div></div><div id="sec4_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i36" class="anchor-spacer"></div><h3 class="article-section__title" id="_i36"> Glycolysis Stress Assay</h3><div class="NLM_p last">Extracellular acidification rate (ECAR) and oxygen consumption rate (OCR) were measured using a Seahorse Bioscience XF96 extracellular flux analyzer (Seahorse Bioscience) according to the manufacturer’s protocol. PC-9 cells (5 × 10<sup>4</sup>) were seeded in 96 Seahorse XF Cell Culture Microplate with normal growth medium 24 h before treatment with compounds. The cells were treated with DMSO or compound <b>1</b> for 4 h at 37 °C under 5% CO<sub>2</sub> atmosphere. After 4 h, the culture medium was replaced with glycolysis optimization medium and incubated at 37 °C without CO<sub>2</sub> for 1 h prior to assay. The ECAR and OCAR measurement trace during Seahorse Glycolysis Stress Assay in which the control and compound <b>1</b> treated PC-9 cells was injected with of 10 mM glucose, 1 μM oligomycin, and 50 mM 2-DG. Every point represents the average of six different wells.</div></div><div id="sec4_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i37" class="anchor-spacer"></div><h3 class="article-section__title" id="_i37"> Immunoblot Analysis</h3><div class="NLM_p last">PC-9 cells were seeded into 6 cm dish at a concentration of 5 × 10<sup>5</sup> cells/mL. After 24 h, cells were treated with 0–1 μM compound <b>1</b> or DMSO alone for 24 h. The cells were harvested at 24 h, and protein extractions were carried out. The protein lysate was subjected to 10% SDS-PAGE and transferred to a nitrocellulose membrane (GE Healthcare Life Sciences). Protein expression was analyzed by Western blotting using primary antibodies against PKM2, EGFR, p-STAT3, STAT3, p-β-catenin (Cell Signaling Technology), β-catenin (Santa Cruz Biotechnology), and β-actin (Sigma-Aldrich), followed by incubation with horseradish peroxidase-conjugated secondary antibody (Jackson ImmunoResearch Laboratories). Specific proteins were detected by chemiluminescence using ECL Plus Western Blotting Detection Reagents (GE Healthcare Biosciences).</div></div><div id="sec4_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i38" class="anchor-spacer"></div><h3 class="article-section__title" id="_i38"> ROS Assay</h3><div class="NLM_p last">ROS detection was performed by using a fluorescent assay kit from BioVision (Milpitas, CA, USA) according to the manufacturer’s instructions. Briefly, PC-9 cells were seeded into 96-well plate at a cell density of 2 × 10<sup>4</sup> cells per well. After 24 h, cells were treated with indicated concentration of compound <b>1</b> or DMSO for 6 h. After treatment, cells were washed in ROS assay buffer and then incubated with 1× ROS label diluted in ROS assay buffer for 45 min at 37 °C in the dark. The ROS label was removed, and the fluorescence at Ex/Em 495/529 nm in end point mode was measured immediately after 100 μL of ROS assay buffer was added to each well.</div></div><div id="sec4_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i39" class="anchor-spacer"></div><h3 class="article-section__title" id="_i39"> In Vivo Efficacy Study</h3><div class="NLM_p last">This animal experiment was approved by the Institutional Animal Care and Use Committee at Academia Sinica, and athymic nude mice were purchased from the National Laboratory Animal Center (Taipei, Taiwan). To assess the effect of compound <b>1</b> on tumor growth in vivo, 1 × 10<sup>6</sup> PC-9 cells were mixed with Matrigel (BD Biosciences) at a 1:1 ratio and subcutaneously implanted into nude mice (6-week-old, female; a total of 16 mice). After 7 days, mice were treated with once daily vehicle or 10 mg/kg compound <b>1</b> in sterile water containing 0.5% methylcellulose (w/v) + 0.1% Tween-80 (v/v) (<i>n</i> = 8 for each group). Tumors were measured with calipers and volumes were calculated using <i>V</i> = (width<sup>2</sup> × length) × 0.52. On day 25, tumors were excised and weighed. Tumor specimens were collected, and expression of target proteins in tumor lysates were analyzed by Western blotting, which were quantified by ImageJ according to a published online procedure (<a href="https://openwetware.org/wiki/Protein_Quantification_Using_ImageJ" class="extLink">https://openwetware.org/wiki/Protein_Quantification_Using_ImageJ</a>).</div></div><div id="sec4_16" class="NLM_sec NLM_sec_level_2"><div id="ac_i40" class="anchor-spacer"></div><h3 class="article-section__title" id="_i40"> Statistical Analysis</h3><div class="NLM_p last">In vitro experiments were performed in triplicate, and the data were presented as means ± SD. Group means were analyzed by using one-way ANOVA, followed by <i>t</i> test. For the in vivo experiments, differences in tumor volume and tumor weight were analyzed by log-rank test and Student’s <i>t</i> test, respectively. Differences were considered significant at *<i>P</i> < 0.05, **<i>P</i> < 0.01, and ***<i>P</i> < 0.001.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-5"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i41"><a href="/doi/suppl/10.1021/acs.jmedchem.9b00763" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_56334" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_56334" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.9b00763" class="ext-link">10.1021/acs.jmedchem.9b00763</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Kinase profiling analysis of the effect of compound <b>1</b> on the activities of 246 different kinases; kinase profiling analysis of the effect of compound <b>1</b> on the substrate binding to 143 different kinases; structures of small-molecule modulators of PKM2; identification of compound <b>1</b>–cysteine conjugate by mass spectral analysis; sequence of PKM2 and the distribution of 10 cysteine residues; purification of recombinant PKM2; images of apo PKM2 crystals at day 16 of the crystallization process; images of PKM2 crystals socked with compound <b>1</b> at different time intervals; and Western blot analysis of the relative PKM2 abundance in six lung cancer cell lines (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00763/suppl_file/jm9b00763_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">NMR, MS, and purity determination data (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00763/suppl_file/jm9b00763_si_002.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00763/suppl_file/jm9b00763_si_003.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00763/suppl_file/jm9b00763_si_001.pdf">jm9b00763_si_001.pdf (2.31 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00763/suppl_file/jm9b00763_si_002.pdf">jm9b00763_si_002.pdf (5.13 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00763/suppl_file/jm9b00763_si_003.csv">jm9b00763_si_003.csv (0.39 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.9b00763" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_88067" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_88067" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Po-Chen Chu</span> - <span class="hlFld-Affiliation affiliation">Drug
Development Center, China Medical University, Taichung 40402, Taiwan</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Cosmeceutics and Graduate Institute of Cosmeceutics, China Medical University, Taichung 40402, Taiwan</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-9510-2269" title="Orcid link">http://orcid.org/0000-0001-9510-2269</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#91e1fee1fee1f8d1f6fcf0f8fdbff2fefc"><span class="__cf_email__" data-cfemail="3c4c534c534c557c5b515d5550125f5351">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ching S. Chen</span> - <span class="hlFld-Affiliation affiliation">Institute
of New Drug Development, China Medical University, Taichung 40402, Taiwan</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Medical Research, China Medical University
Hospital, China Medical University, Taichung 40447, Taiwan</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#86e5f5e5eee3e8e5ebf3c6e1ebe7efeaa8e5e9eb"><span class="__cf_email__" data-cfemail="8ae9f9e9e2efe4e9e7ffcaede7ebe3e6a4e9e5e7">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">I-Shan Hsieh</span> - <span class="hlFld-Affiliation affiliation">Institute
of Biological Chemistry, Academia Sinica, Taipei 11529, Taiwan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Balraj Gopula</span> - <span class="hlFld-Affiliation affiliation">Institute
of Biological Chemistry, Academia Sinica, Taipei 11529, Taiwan</span>; 
    <span class="hlFld-Affiliation affiliation">Drug
Development Center, China Medical University, Taichung 40402, Taiwan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Chi-Chi Chou</span> - <span class="hlFld-Affiliation affiliation">Institute
of Biological Chemistry, Academia Sinica, Taipei 11529, Taiwan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hsiang-Yi Wu</span> - <span class="hlFld-Affiliation affiliation">Institute
of Biological Chemistry, Academia Sinica, Taipei 11529, Taiwan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Geen-Dong Chang</span> - <span class="hlFld-Affiliation affiliation">Institute
of Biochemical Sciences, National Taiwan
University, Taipei 10617, Taiwan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Wen-Jin Wu</span> - <span class="hlFld-Affiliation affiliation">Institute
of Biological Chemistry, Academia Sinica, Taipei 11529, Taiwan</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-7253-5384" title="Orcid link">http://orcid.org/0000-0002-7253-5384</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Chih-Shiang Chang</span> - <span class="hlFld-Affiliation affiliation">Drug
Development Center, China Medical University, Taichung 40402, Taiwan</span>; 
    <span class="hlFld-Affiliation affiliation">School
of Pharmacy, College of Pharmacy, China
Medical University, Taichung 40402, Taiwan</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>I.-S.H. and B.G. contributed equally to this work. I.-S.H. conducted most of the experiments and summarized all data. B.G. synthesized and characterized compounds, and performed structure–activity relationship analysis. C.-C.C. and W.-J.W. contributed to the mass spectral experiments and binding-site analysis. H.-Y.W. conducted the purification and crystallization of recombinant PKM2 protein. G.-D.C.prepared the anti-compound-1 antiserum. C.-S.C. was involved in the structural characterization and purity determination. P.-C.C. and C.S.C. formed the concept, designed experiments, analyzed data, and wrote the manuscript.</p></li><li><span class="author-information-subsection-header">Funding</span><p>This work was supported, in part, by the intramural grants CMU-1065918B, 1075955A, and 1085955A from China Medical University.</p></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">PKM</td><td class="NLM_def"><p class="first last">pyruvate kinase M</p></td></tr><tr><td class="NLM_term">ABPP</td><td class="NLM_def"><p class="first last">activity-based protein profiling</p></td></tr><tr><td class="NLM_term">SILAC</td><td class="NLM_def"><p class="first last">stable isotope labeling by amino acids in cell culture</p></td></tr><tr><td class="NLM_term">PEP</td><td class="NLM_def"><p class="first last">phosphoenolpyruvate</p></td></tr><tr><td class="NLM_term">LC-MS/MS</td><td class="NLM_def"><p class="first last">liquid chromatography–tandem mass spectrometry</p></td></tr><tr><td class="NLM_term">LDHA</td><td class="NLM_def"><p class="first last">lactate dehydrogenase A</p></td></tr><tr><td class="NLM_term">CS</td><td class="NLM_def"><p class="first last">citrate synthase citrate synthase</p></td></tr><tr><td class="NLM_term">MDHM</td><td class="NLM_def"><p class="first last">malate dehydrogenase</p></td></tr><tr><td class="NLM_term">ACYL</td><td class="NLM_def"><p class="first last">ATP-citrate lyase</p></td></tr><tr><td class="NLM_term">ENOA</td><td class="NLM_def"><p class="first last">α-enolase</p></td></tr><tr><td class="NLM_term">PCKGM</td><td class="NLM_def"><p class="first last">phosphoenolpyruvate carboxylase</p></td></tr><tr><td class="NLM_term">ACSF3</td><td class="NLM_def"><p class="first last">acyl-CoA synthetase family member 3</p></td></tr><tr><td class="NLM_term">PGK1</td><td class="NLM_def"><p class="first last">phosphoglycerate kinase</p></td></tr><tr><td class="NLM_term">ECAR</td><td class="NLM_def"><p class="first last">extracellular acidification rate</p></td></tr><tr><td class="NLM_term">OCR</td><td class="NLM_def"><p class="first last">cellular oxygen consumption rate</p></td></tr><tr><td class="NLM_term">2DG</td><td class="NLM_def"><p class="first last">2-deoxyglucose</p></td></tr><tr><td class="NLM_term">ROS</td><td class="NLM_def"><p class="first last">reactive oxygen species</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i44">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_71600" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_71600" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 42 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hsu, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sabatini, D. M.</span></span> <span> </span><span class="NLM_article-title">Cancer Cell Metabolism: Warburg and Beyond</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>134</i></span>,  <span class="NLM_fpage">703</span>– <span class="NLM_lpage">707</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2008.08.021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=10.1016%2Fj.cell.2008.08.021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=18775299" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtFCqs7jK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=134&publication_year=2008&pages=703-707&author=P.+P.+Hsuauthor=D.+M.+Sabatini&title=Cancer+Cell+Metabolism%3A+Warburg+and+Beyond&doi=10.1016%2Fj.cell.2008.08.021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Cancer cell metabolism: Warburg and beyond</span></div><div class="casAuthors">Hsu, Peggy P.; Sabatini, David M.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">134</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">703-707</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Described decades ago, the Warburg effect of aerobic glycolysis is a key metabolic hallmark of cancer, yet its significance remains unclear.  In this Essay, we re-examine the Warburg effect and establish a framework for understanding its contribution to the altered metab. of cancer cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqRBI3IwvZP5rVg90H21EOLACvtfcHk0lh1fzgzgIazmQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtFCqs7jK&md5=bc4e91dc941eb33a7b60572126537676</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2008.08.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2008.08.021%26sid%3Dliteratum%253Aachs%26aulast%3DHsu%26aufirst%3DP.%2BP.%26aulast%3DSabatini%26aufirst%3DD.%2BM.%26atitle%3DCancer%2520Cell%2520Metabolism%253A%2520Warburg%2520and%2520Beyond%26jtitle%3DCell%26date%3D2008%26volume%3D134%26spage%3D703%26epage%3D707%26doi%3D10.1016%2Fj.cell.2008.08.021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tennant, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duran, R. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boulahbel, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gottlieb, E.</span></span> <span> </span><span class="NLM_article-title">Metabolic Transformation in Cancer</span>. <i>Carcinogenesis</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">1269</span>– <span class="NLM_lpage">1280</span>, <span class="refDoi"> DOI: 10.1093/carcin/bgp070</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=10.1093%2Fcarcin%2Fbgp070" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=19321800" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=1%3ACAS%3A528%3ADC%252BD1MXpsVaksbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2009&pages=1269-1280&author=D.+A.+Tennantauthor=R.+V.+Duranauthor=H.+Boulahbelauthor=E.+Gottlieb&title=Metabolic+Transformation+in+Cancer&doi=10.1093%2Fcarcin%2Fbgp070"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Metabolic transformation in cancer</span></div><div class="casAuthors">Tennant, Daniel A.; Duran, Raul V.; Boulahbel, Houda; Gottlieb, Eyal</div><div class="citationInfo"><span class="NLM_cas:title">Carcinogenesis</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1269-1280</span>CODEN:
                <span class="NLM_cas:coden">CRNGDP</span>;
        ISSN:<span class="NLM_cas:issn">0143-3334</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">A review.  In 2000, Douglas Hanahan and Robert Weinberg published a review detailing the six hallmarks of cancer.  These are six phenotypes that a tumor requires in order to become a fully fledged malignancy: persistent growth signals, evasion of apoptosis, insensitivity to anti-growth signals, unlimited replicative potential, angiogenesis and invasion and metastasis.  However, it is becoming increasingly clear that these phenotypes do not portray the whole story and that other hallmarks are necessary: one of which is a shift in cellular metab.  The tumor environment creates a unique collection of stresses to which cells must adapt in order to survive.  This environment is formed by the uncontrolled proliferation of cells, which ignore the cues that would create normal tissue architecture.  As a result, the cells forming the tumor are exposed to low oxygen and nutrient levels, as well as high levels of toxic cellular waste products, which is thought to propel cells towards a more transformed phenotype, resistant to cell death and pro-metastatic.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-VBZ0PfaxO7Vg90H21EOLACvtfcHk0lh1fzgzgIazmQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXpsVaksbc%253D&md5=60e1db58d07faae083753aa604a7c1cf</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1093%2Fcarcin%2Fbgp070&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fcarcin%252Fbgp070%26sid%3Dliteratum%253Aachs%26aulast%3DTennant%26aufirst%3DD.%2BA.%26aulast%3DDuran%26aufirst%3DR.%2BV.%26aulast%3DBoulahbel%26aufirst%3DH.%26aulast%3DGottlieb%26aufirst%3DE.%26atitle%3DMetabolic%2520Transformation%2520in%2520Cancer%26jtitle%3DCarcinogenesis%26date%3D2009%26volume%3D30%26spage%3D1269%26epage%3D1280%26doi%3D10.1093%2Fcarcin%2Fbgp070" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vander
Heiden, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantley, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, C. B.</span></span> <span> </span><span class="NLM_article-title">Understanding the Warburg Effect: The Metabolic Requirements of Cell Proliferation</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>324</i></span>,  <span class="NLM_fpage">1029</span>– <span class="NLM_lpage">1033</span>, <span class="refDoi"> DOI: 10.1126/science.1160809</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=10.1126%2Fscience.1160809" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=19460998" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=1%3ACAS%3A528%3ADC%252BD1MXmtVKlsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=324&publication_year=2009&pages=1029-1033&author=M.+G.+Vander%0AHeidenauthor=L.+C.+Cantleyauthor=C.+B.+Thompson&title=Understanding+the+Warburg+Effect%3A+The+Metabolic+Requirements+of+Cell+Proliferation&doi=10.1126%2Fscience.1160809"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Understanding the Warburg Effect: The Metabolic Requirements of Cell Proliferation</span></div><div class="casAuthors">Vander Heiden, Matthew G.; Cantley, Lewis C.; Thompson, Craig B.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">324</span>
        (<span class="NLM_cas:issue">5930</span>),
    <span class="NLM_cas:pages">1029-1033</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">A review.  In contrast to normal differentiated cells, which rely primarily on mitochondrial oxidative phosphorylation to generate the energy needed for cellular processes, most cancer cells instead rely on aerobic glycolysis, a phenomenon termed "the Warburg effect".  Aerobic glycolysis is an inefficient way to generate ATP, however, and the advantage it confers to cancer cells has been unclear.  Here we propose that the metab. of cancer cells, and indeed all proliferating cells, is adapted to facilitate the uptake and incorporation of nutrients into the biomass (e.g., nucleotides, amino acids, and lipids) needed to produce a new cell.  Supporting this idea are recent studies showing that (i) several signaling pathways implicated in cell proliferation also regulate metabolic pathways that incorporate nutrients into biomass; and that (ii) certain cancer-assocd. mutations enable cancer cells to acquire and metabolize nutrients in a manner conducive to proliferation rather than efficient ATP prodn.  A better understanding of the mechanistic links between cellular metab. and growth control may ultimately lead to better treatments for human cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrGvLHGQ9TX77Vg90H21EOLACvtfcHk0lh1fzgzgIazmQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXmtVKlsbg%253D&md5=f927402a43cdca471c3f4874dcbeb943</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1126%2Fscience.1160809&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1160809%26sid%3Dliteratum%253Aachs%26aulast%3DVander%2BHeiden%26aufirst%3DM.%2BG.%26aulast%3DCantley%26aufirst%3DL.%2BC.%26aulast%3DThompson%26aufirst%3DC.%2BB.%26atitle%3DUnderstanding%2520the%2520Warburg%2520Effect%253A%2520The%2520Metabolic%2520Requirements%2520of%2520Cell%2520Proliferation%26jtitle%3DScience%26date%3D2009%26volume%3D324%26spage%3D1029%26epage%3D1033%26doi%3D10.1126%2Fscience.1160809" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lunt, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vander Heiden, M. G.</span></span> <span> </span><span class="NLM_article-title">Aerobic Glycolysis: Meeting the Metabolic Requirements of Cell Proliferation</span>. <i>Annu. Rev. Cell Dev. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">441</span>– <span class="NLM_lpage">464</span>, <span class="refDoi"> DOI: 10.1146/annurev-cellbio-092910-154237</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=10.1146%2Fannurev-cellbio-092910-154237" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=21985671" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsFyqtbfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2011&pages=441-464&author=S.+Y.+Luntauthor=M.+G.+Vander+Heiden&title=Aerobic+Glycolysis%3A+Meeting+the+Metabolic+Requirements+of+Cell+Proliferation&doi=10.1146%2Fannurev-cellbio-092910-154237"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Aerobic glycolysis: meeting the metabolic requirements of cell proliferation</span></div><div class="casAuthors">Lunt, Sophia Y.; Vander Heiden, Matthew G.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Cell and Developmental Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">441-464</span>CODEN:
                <span class="NLM_cas:coden">ARDBF8</span>;
        ISSN:<span class="NLM_cas:issn">1081-0706</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review.  Warburg's observation that cancer cells exhibit a high rate of glycolysis even in the presence of oxygen (aerobic glycolysis) sparked debate over the role of glycolysis in normal and cancer cells.  Although it has been established that defects in mitochondrial respiration are not the cause of cancer or aerobic glycolysis, the advantages of enhanced glycolysis in cancer remain controversial.  Many cells ranging from microbes to lymphocytes use aerobic glycolysis during rapid proliferation, which suggests it may play a fundamental role in supporting cell growth.  Here, we review how glycolysis contributes to the metabolic processes of dividing cells.  We provide a detailed accounting of the biosynthetic requirements to construct a new cell and illustrate the importance of glycolysis in providing carbons to generate biomass.  We argue that the major function of aerobic glycolysis is to maintain high levels of glycolytic intermediates to support anabolic reactions in cells, thus providing an explanation for why increased glucose metab. is selected for in proliferating cells throughout nature.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrDaFTmS-BapLVg90H21EOLACvtfcHk0lgRUKZ-lnwRvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsFyqtbfN&md5=85f04cff291334181cc0aae9015e0e0d</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1146%2Fannurev-cellbio-092910-154237&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-cellbio-092910-154237%26sid%3Dliteratum%253Aachs%26aulast%3DLunt%26aufirst%3DS.%2BY.%26aulast%3DVander%2BHeiden%26aufirst%3DM.%2BG.%26atitle%3DAerobic%2520Glycolysis%253A%2520Meeting%2520the%2520Metabolic%2520Requirements%2520of%2520Cell%2520Proliferation%26jtitle%3DAnnu.%2520Rev.%2520Cell%2520Dev.%2520Biol.%26date%3D2011%26volume%3D27%26spage%3D441%26epage%3D464%26doi%3D10.1146%2Fannurev-cellbio-092910-154237" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lehuede, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dupuy, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rabinovitch, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siegel, P. M.</span></span> <span> </span><span class="NLM_article-title">Metabolic Plasticity as a Determinant of Tumor Growth and Metastasis</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>76</i></span>,  <span class="NLM_fpage">5201</span>– <span class="NLM_lpage">5208</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-16-0266</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=10.1158%2F0008-5472.CAN-16-0266" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=27587539" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=1%3ACAS%3A280%3ADC%252BC2szmvVaktQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=76&publication_year=2016&pages=5201-5208&author=C.+Lehuedeauthor=F.+Dupuyauthor=R.+Rabinovitchauthor=R.+G.+Jonesauthor=P.+M.+Siegel&title=Metabolic+Plasticity+as+a+Determinant+of+Tumor+Growth+and+Metastasis&doi=10.1158%2F0008-5472.CAN-16-0266"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Metabolic Plasticity as a Determinant of Tumor Growth and Metastasis</span></div><div class="casAuthors">Lehuede Camille; Dupuy Fanny; Rabinovitch Rebecca; Jones Russell G; Siegel Peter M</div><div class="citationInfo"><span class="NLM_cas:title">Cancer research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">76</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">5201-8</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Cancer cells must adapt their metabolism to meet the energetic and biosynthetic demands that accompany rapid growth of the primary tumor and colonization of distinct metastatic sites.  Different stages of the metastatic cascade can also present distinct metabolic challenges to disseminating cancer cells.  However, little is known regarding how changes in cellular metabolism, both within the cancer cell and the metastatic microenvironment, alter the ability of tumor cells to colonize and grow in distinct secondary sites.  This review examines the concept of metabolic heterogeneity within the primary tumor, and how cancer cells are metabolically coupled with other cancer cells that comprise the tumor and cells within the tumor stroma.  We examine how metabolic strategies, which are engaged by cancer cells in the primary site, change during the metastatic process.  Finally, we discuss the metabolic adaptations that occur as cancer cells colonize foreign metastatic microenvironments and how cancer cells influence the metabolism of stromal cells at sites of metastasis.  Through a discussion of these topics, it is clear that plasticity in tumor metabolic programs, which allows cancer cells to adapt and grow in hostile microenvironments, is emerging as an important variable that may change clinical approaches to managing metastatic disease.  Cancer Res; 76(18); 5201-8. ©2016 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRAzaCt0cgSwUNHe_gFldTbfW6udTcc2eY8FWM_jb-_77ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2szmvVaktQ%253D%253D&md5=c9f4da8a4e27842634af2801334245ff</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-16-0266&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-16-0266%26sid%3Dliteratum%253Aachs%26aulast%3DLehuede%26aufirst%3DC.%26aulast%3DDupuy%26aufirst%3DF.%26aulast%3DRabinovitch%26aufirst%3DR.%26aulast%3DJones%26aufirst%3DR.%2BG.%26aulast%3DSiegel%26aufirst%3DP.%2BM.%26atitle%3DMetabolic%2520Plasticity%2520as%2520a%2520Determinant%2520of%2520Tumor%2520Growth%2520and%2520Metastasis%26jtitle%3DCancer%2520Res.%26date%3D2016%26volume%3D76%26spage%3D5201%26epage%3D5208%26doi%3D10.1158%2F0008-5472.CAN-16-0266" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chan, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sutphin, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turcotte, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, E. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banh, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reynolds, G. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chi, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Solow-Cordero, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonnet, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flanagan, J. U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bouley, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graves, E. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denny, W. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hay, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giaccia, A. J.</span></span> <span> </span><span class="NLM_article-title">Targeting Glut1 and the Warburg Effect in Renal Cell Carcinoma by Chemical Synthetic Lethality</span>. <i>Sci. Transl. Med.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">94ra70</span>, <span class="refDoi"> DOI: 10.1126/scitranslmed.3002394</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=10.1126%2Fscitranslmed.3002394" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=21813754" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFOisLzK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2011&pages=94ra70&author=D.+A.+Chanauthor=P.+D.+Sutphinauthor=P.+Nguyenauthor=S.+Turcotteauthor=E.+W.+Laiauthor=A.+Banhauthor=G.+E.+Reynoldsauthor=J.+T.+Chiauthor=J.+Wuauthor=D.+E.+Solow-Corderoauthor=M.+Bonnetauthor=J.+U.+Flanaganauthor=D.+M.+Bouleyauthor=E.+E.+Gravesauthor=W.+A.+Dennyauthor=M.+P.+Hayauthor=A.+J.+Giaccia&title=Targeting+Glut1+and+the+Warburg+Effect+in+Renal+Cell+Carcinoma+by+Chemical+Synthetic+Lethality&doi=10.1126%2Fscitranslmed.3002394"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting GLUT1 and the Warburg effect in renal cell carcinoma by chemical synthetic lethality</span></div><div class="casAuthors">Chan, Denise A.; Sutphin, Patrick D.; Nguyen, Phuong; Turcotte, Sandra; Lai, Edwin W.; Banh, Alice; Reynolds, Gloria E.; Chi, Jen-Tsan; Wu, Jason; Solow-Cordero, David E.; Bonnet, Muriel; Flanagan, Jack U.; Bouley, Donna M.; Graves, Edward E.; Denny, William A.; Hay, Michael P.; Giaccia, Amato J.</div><div class="citationInfo"><span class="NLM_cas:title">Science Translational Medicine</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">94</span>),
    <span class="NLM_cas:pages">94ra70</span>CODEN:
                <span class="NLM_cas:coden">STMCBQ</span>;
        ISSN:<span class="NLM_cas:issn">1946-6242</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Identifying new targeted therapies that kill tumor cells while sparing normal tissue is a major challenge of cancer research.  Using a high-throughput chem. synthetic lethal screen, we sought to identify compds. that exploit the loss of the von Hippel-Lindau (VHL) tumor suppressor gene, which occurs in about 80% of renal cell carcinomas (RCCs).  RCCs, like many other cancers, are dependent on aerobic glycolysis for ATP prodn., a phenomenon known as the Warburg effect.  The dependence of RCCs on glycolysis is in part a result of induction of glucose transporter 1 (GLUT1).  Here, we report the identification of a class of compds., the 3-series, exemplified by STF-31, which selectively kills RCCs by specifically targeting glucose uptake through GLUT1 and exploiting the unique dependence of these cells on GLUT1 for survival.  Treatment with these agents inhibits the growth of RCCs by binding GLUT1 directly and impeding glucose uptake in vivo without toxicity to normal tissue.  Activity of STF-31 in these exptl. renal tumors can be monitored by [18F] fluorodeoxyglucose uptake by micro-positron emission tomog. imaging, and therefore, these agents may be readily tested clin. in human tumors.  Our results show that the Warburg effect confers distinct characteristics on tumor cells that can be selectively targeted for therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZodRjh_SzxrVg90H21EOLACvtfcHk0lgRUKZ-lnwRvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFOisLzK&md5=418628e65c3a96425353b4be2d692927</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1126%2Fscitranslmed.3002394&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscitranslmed.3002394%26sid%3Dliteratum%253Aachs%26aulast%3DChan%26aufirst%3DD.%2BA.%26aulast%3DSutphin%26aufirst%3DP.%2BD.%26aulast%3DNguyen%26aufirst%3DP.%26aulast%3DTurcotte%26aufirst%3DS.%26aulast%3DLai%26aufirst%3DE.%2BW.%26aulast%3DBanh%26aufirst%3DA.%26aulast%3DReynolds%26aufirst%3DG.%2BE.%26aulast%3DChi%26aufirst%3DJ.%2BT.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DSolow-Cordero%26aufirst%3DD.%2BE.%26aulast%3DBonnet%26aufirst%3DM.%26aulast%3DFlanagan%26aufirst%3DJ.%2BU.%26aulast%3DBouley%26aufirst%3DD.%2BM.%26aulast%3DGraves%26aufirst%3DE.%2BE.%26aulast%3DDenny%26aufirst%3DW.%2BA.%26aulast%3DHay%26aufirst%3DM.%2BP.%26aulast%3DGiaccia%26aufirst%3DA.%2BJ.%26atitle%3DTargeting%2520Glut1%2520and%2520the%2520Warburg%2520Effect%2520in%2520Renal%2520Cell%2520Carcinoma%2520by%2520Chemical%2520Synthetic%2520Lethality%26jtitle%3DSci.%2520Transl.%2520Med.%26date%3D2011%26volume%3D3%26spage%3D94ra70%26doi%3D10.1126%2Fscitranslmed.3002394" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Calvo, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Figueroa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pulido, E. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campelo, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aparicio, L. A.</span></span> <span> </span><span class="NLM_article-title">Potential Role of Sugar Transporters in Cancer and Their Relationship with Anticancer Therapy</span>. <i>Int. J. Endocrinol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>2010</i></span>, <span class="NLM_elocation-id">205357</span> <span class="refDoi"> DOI: 10.1155/2010/205357</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=10.1155%2F2010%2F205357" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=20706540" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2010&publication_year=2010&author=M.+B.+Calvoauthor=A.+Figueroaauthor=E.+G.+Pulidoauthor=R.+G.+Campeloauthor=L.+A.+Aparicio&title=Potential+Role+of+Sugar+Transporters+in+Cancer+and+Their+Relationship+with+Anticancer+Therapy&doi=10.1155%2F2010%2F205357"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1155%2F2010%2F205357&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1155%252F2010%252F205357%26sid%3Dliteratum%253Aachs%26aulast%3DCalvo%26aufirst%3DM.%2BB.%26aulast%3DFigueroa%26aufirst%3DA.%26aulast%3DPulido%26aufirst%3DE.%2BG.%26aulast%3DCampelo%26aufirst%3DR.%2BG.%26aulast%3DAparicio%26aufirst%3DL.%2BA.%26atitle%3DPotential%2520Role%2520of%2520Sugar%2520Transporters%2520in%2520Cancer%2520and%2520Their%2520Relationship%2520with%2520Anticancer%2520Therapy%26jtitle%3DInt.%2520J.%2520Endocrinol.%26date%3D2010%26volume%3D2010%26doi%3D10.1155%2F2010%2F205357" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Patra, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhaskar, P. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wheaton, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chandel, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laakso, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allen, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jha, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smolen, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clasquin, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robey, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hay, N.</span></span> <span> </span><span class="NLM_article-title">Hexokinase 2 Is Required for Tumor Initiation and Maintenance and Its Systemic Deletion Is Therapeutic in Mouse Models of Cancer</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">213</span>– <span class="NLM_lpage">228</span>, <span class="refDoi"> DOI: 10.1016/j.ccr.2013.06.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=10.1016%2Fj.ccr.2013.06.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=23911236" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1Wrur%252FL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2013&pages=213-228&author=K.+C.+Patraauthor=Q.+Wangauthor=P.+T.+Bhaskarauthor=L.+Millerauthor=Z.+Wangauthor=W.+Wheatonauthor=N.+Chandelauthor=M.+Laaksoauthor=W.+J.+Mullerauthor=E.+L.+Allenauthor=A.+K.+Jhaauthor=G.+A.+Smolenauthor=M.+F.+Clasquinauthor=B.+Robeyauthor=N.+Hay&title=Hexokinase+2+Is+Required+for+Tumor+Initiation+and+Maintenance+and+Its+Systemic+Deletion+Is+Therapeutic+in+Mouse+Models+of+Cancer&doi=10.1016%2Fj.ccr.2013.06.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Hexokinase 2 Is Required for Tumor Initiation and Maintenance and Its Systemic Deletion Is Therapeutic in Mouse Models of Cancer</span></div><div class="casAuthors">Patra, Krushna C.; Wang, Qi; Bhaskar, Prashanth T.; Miller, Luke; Wang, Zebin; Wheaton, Will; Chandel, Navdeep; Laakso, Markku; Muller, William J.; Allen, Eric L.; Jha, Abhishek K.; Smolen, Gromoslaw A.; Clasquin, Michelle F.; Robey, R. Brooks; Hay, Nissim</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">213-228</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Accelerated glucose metab. is a common feature of cancer cells.  Hexokinases catalyze the first committed step of glucose metab.  Hexokinase 2 (HK2) is expressed at high level in cancer cells, but only in a limited no. of normal adult tissues.  Using Hk2 conditional knockout mice, we showed that HK2 is required for tumor initiation and maintenance in mouse models of KRas-driven lung cancer, and ErbB2-driven breast cancer, despite continued HK1 expression.  Similarly, HK2 ablation inhibits the neoplastic phenotype of human lung and breast cancer cells in vitro and in vivo.  Systemic Hk2 deletion is therapeutic in mice bearing lung tumors without adverse physiol. consequences.  Hk2 deletion in lung cancer cells suppressed glucose-derived ribonucleotides and impaired glutamine-derived carbon utilization in anaplerosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMgkDdepYxc7Vg90H21EOLACvtfcHk0lh1_FSlGyKoDQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1Wrur%252FL&md5=837dbeef1e451013754b98cbcfa61bb4</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2013.06.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2013.06.014%26sid%3Dliteratum%253Aachs%26aulast%3DPatra%26aufirst%3DK.%2BC.%26aulast%3DWang%26aufirst%3DQ.%26aulast%3DBhaskar%26aufirst%3DP.%2BT.%26aulast%3DMiller%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DWheaton%26aufirst%3DW.%26aulast%3DChandel%26aufirst%3DN.%26aulast%3DLaakso%26aufirst%3DM.%26aulast%3DMuller%26aufirst%3DW.%2BJ.%26aulast%3DAllen%26aufirst%3DE.%2BL.%26aulast%3DJha%26aufirst%3DA.%2BK.%26aulast%3DSmolen%26aufirst%3DG.%2BA.%26aulast%3DClasquin%26aufirst%3DM.%2BF.%26aulast%3DRobey%26aufirst%3DB.%26aulast%3DHay%26aufirst%3DN.%26atitle%3DHexokinase%25202%2520Is%2520Required%2520for%2520Tumor%2520Initiation%2520and%2520Maintenance%2520and%2520Its%2520Systemic%2520Deletion%2520Is%2520Therapeutic%2520in%2520Mouse%2520Models%2520of%2520Cancer%26jtitle%3DCancer%2520Cell%26date%3D2013%26volume%3D24%26spage%3D213%26epage%3D228%26doi%3D10.1016%2Fj.ccr.2013.06.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tamada, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suematsu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saya, H.</span></span> <span> </span><span class="NLM_article-title">Pyruvate Kinase M2: Multiple Faces for Conferring Benefits on Cancer Cells</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">5554</span>– <span class="NLM_lpage">5561</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.CCR-12-0859</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=10.1158%2F1078-0432.CCR-12-0859" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=23071357" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFahu73L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2012&pages=5554-5561&author=M.+Tamadaauthor=M.+Suematsuauthor=H.+Saya&title=Pyruvate+Kinase+M2%3A+Multiple+Faces+for+Conferring+Benefits+on+Cancer+Cells&doi=10.1158%2F1078-0432.CCR-12-0859"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Pyruvate Kinase M2: Multiple Faces for Conferring Benefits on Cancer Cells</span></div><div class="casAuthors">Tamada, Mayumi; Suematsu, Makoto; Saya, Hideyuki</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">5554-5561</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">A review.  The M2 splice isoform of pyruvate kinase (PKM2), an enzyme that catalyzes the later step of glycolysis, is a key regulator of aerobic glycolysis (known as the Warburg effect) in cancer cells.  Expression and low enzymic activity of PKM2 confer on cancer cells the glycolytic phenotype, which promotes rapid energy prodn. and flow of glycolytic intermediates into collateral pathways to synthesize nucleic acids, amino acids, and lipids without the accumulation of reactive oxygen species.  PKM2 enzymic activity has also been shown to be neg. regulated by the interaction with CD44 adhesion mol., which is a cell surface marker for cancer stem cells.  In addn. to the glycolytic functions, nonglycolytic functions of PKM2 in cancer cells are of particular interest.  PKM2 is induced translocation into the nucleus, where it activates transcription of various genes by interacting with and phosphorylating specific nuclear proteins, endowing cancer cells with a survival and growth advantage.  Therefore, inhibitors and activators of PKM2 are well underway to evaluate their anticancer effects and suitability for use as novel therapeutic strategies.  Clin Cancer Res; 18(20); 5554-61. ©2012 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3jTAHz_wVC7Vg90H21EOLACvtfcHk0lh1_FSlGyKoDQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFahu73L&md5=0da5d879fe53330037ccbecce75b89e4</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-12-0859&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-12-0859%26sid%3Dliteratum%253Aachs%26aulast%3DTamada%26aufirst%3DM.%26aulast%3DSuematsu%26aufirst%3DM.%26aulast%3DSaya%26aufirst%3DH.%26atitle%3DPyruvate%2520Kinase%2520M2%253A%2520Multiple%2520Faces%2520for%2520Conferring%2520Benefits%2520on%2520Cancer%2520Cells%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2012%26volume%3D18%26spage%3D5554%26epage%3D5561%26doi%3D10.1158%2F1078-0432.CCR-12-0859" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wong, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ojo, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, D.</span></span> <span> </span><span class="NLM_article-title">Pkm2 Contributes to Cancer Metabolism</span>. <i>Cancer Lett.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>356</i></span>,  <span class="NLM_fpage">184</span>– <span class="NLM_lpage">191</span>, <span class="refDoi"> DOI: 10.1016/j.canlet.2014.01.031</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=10.1016%2Fj.canlet.2014.01.031" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=24508027" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=1%3ACAS%3A528%3ADC%252BC2cXisVart7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=356&publication_year=2015&pages=184-191&author=N.+Wongauthor=D.+Ojoauthor=J.+Yanauthor=D.+Tang&title=Pkm2+Contributes+to+Cancer+Metabolism&doi=10.1016%2Fj.canlet.2014.01.031"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">PKM2 contributes to cancer metabolism</span></div><div class="casAuthors">Wong, Nicholas; Ojo, Diane; Yan, Judy; Tang, Damu</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Letters (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">356</span>
        (<span class="NLM_cas:issue">2_Part_A</span>),
    <span class="NLM_cas:pages">184-191</span>CODEN:
                <span class="NLM_cas:coden">CALEDQ</span>;
        ISSN:<span class="NLM_cas:issn">0304-3835</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review.  Reprogramming of cell metab. is essential for tumorigenesis, and is regulated by a complex network, in which PKM2 plays a crit. role.  PKM2 exists as an inactive monomer, less active dimer and active tetramer.  While dimeric PKM2 diverts glucose metab. towards anabolism through aerobic glycolysis, tetrameric PKM2 promotes the flux of glucose-derived carbons for ATP prodn. via oxidative phosphorylation.  Equil. of the PKM2 dimers and tetramers is crit. for tumorigenesis, and is controlled by multiple factors.  The PKM2 dimer also promotes aerobic glycolysis by modulating transcriptional regulation.  We will discuss the current understanding of PKM2 in regulating cancer metab.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxTIgwuY8YnLVg90H21EOLACvtfcHk0lgN50-ru55RPw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXisVart7c%253D&md5=8562ddc23e089703d976d165b3a2bb3b</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.canlet.2014.01.031&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.canlet.2014.01.031%26sid%3Dliteratum%253Aachs%26aulast%3DWong%26aufirst%3DN.%26aulast%3DOjo%26aufirst%3DD.%26aulast%3DYan%26aufirst%3DJ.%26aulast%3DTang%26aufirst%3DD.%26atitle%3DPkm2%2520Contributes%2520to%2520Cancer%2520Metabolism%26jtitle%3DCancer%2520Lett.%26date%3D2015%26volume%3D356%26spage%3D184%26epage%3D191%26doi%3D10.1016%2Fj.canlet.2014.01.031" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Iqbal, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gupta, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gopinath, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mazurek, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamezai, R. N.</span></span> <span> </span><span class="NLM_article-title">Pyruvate Kinase M2 and Cancer: An Updated Assessment</span>. <i>FEBS Lett.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>588</i></span>,  <span class="NLM_fpage">2685</span>– <span class="NLM_lpage">2692</span>, <span class="refDoi"> DOI: 10.1016/j.febslet.2014.04.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=10.1016%2Fj.febslet.2014.04.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=24747424" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=1%3ACAS%3A528%3ADC%252BC2cXntFCqur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=588&publication_year=2014&pages=2685-2692&author=M.+A.+Iqbalauthor=V.+Guptaauthor=P.+Gopinathauthor=S.+Mazurekauthor=R.+N.+Bamezai&title=Pyruvate+Kinase+M2+and+Cancer%3A+An+Updated+Assessment&doi=10.1016%2Fj.febslet.2014.04.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Pyruvate kinase M2 and cancer: an updated assessment</span></div><div class="casAuthors">Iqbal, Mohd Askandar; Gupta, Vibhor; Gopinath, Prakasam; Mazurek, Sybille; Bamezai, Rameshwar N. K.</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Letters</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">588</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">2685-2692</span>CODEN:
                <span class="NLM_cas:coden">FEBLAL</span>;
        ISSN:<span class="NLM_cas:issn">0014-5793</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Cancer cells are characterized by high glycolytic rates to support energy regeneration and anabolic metab., along with the expression of pyruvate kinase isoenzyme M2 (PKM2).  The latter catalyzes the last step of glycolysis and reprograms the glycolytic flux to feed the special metabolic demands of proliferating cells.  Besides, PKM2 has moonlight functions, such as gene transcription, favoring cancer.  Accumulating evidence suggests a crit. role played by the low-activity-dimeric PKM2 in tumor progression, supported by the identification of mutations which result in the down-regulation of its activity and tumorigenesis in a nude mouse model.  This review discusses PKM2 regulation and the benefits it confers to cancer cells.  Further, conflicting views on PKM2's role in cancer, its therapeutic relevance and future directions in the field are also discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpM-7GvwndQlbVg90H21EOLACvtfcHk0lgN50-ru55RPw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXntFCqur0%253D&md5=7552a7afcd740bdf54c28f421a321ca2</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.febslet.2014.04.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.febslet.2014.04.011%26sid%3Dliteratum%253Aachs%26aulast%3DIqbal%26aufirst%3DM.%2BA.%26aulast%3DGupta%26aufirst%3DV.%26aulast%3DGopinath%26aufirst%3DP.%26aulast%3DMazurek%26aufirst%3DS.%26aulast%3DBamezai%26aufirst%3DR.%2BN.%26atitle%3DPyruvate%2520Kinase%2520M2%2520and%2520Cancer%253A%2520An%2520Updated%2520Assessment%26jtitle%3DFEBS%2520Lett.%26date%3D2014%26volume%3D588%26spage%3D2685%26epage%3D2692%26doi%3D10.1016%2Fj.febslet.2014.04.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Christofk, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vander Heiden, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramanathan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerszten, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fleming, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schreiber, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantley, L. C.</span></span> <span> </span><span class="NLM_article-title">The M2 Splice Isoform of Pyruvate Kinase Is Important for Cancer Metabolism and Tumour Growth</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>452</i></span>,  <span class="NLM_fpage">230</span>– <span class="NLM_lpage">233</span>, <span class="refDoi"> DOI: 10.1038/nature06734</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=10.1038%2Fnature06734" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=18337823" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=1%3ACAS%3A528%3ADC%252BD1cXjt1Gnu74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=452&publication_year=2008&pages=230-233&author=H.+R.+Christofkauthor=M.+G.+Vander+Heidenauthor=M.+H.+Harrisauthor=A.+Ramanathanauthor=R.+E.+Gersztenauthor=R.+Weiauthor=M.+D.+Flemingauthor=S.+L.+Schreiberauthor=L.+C.+Cantley&title=The+M2+Splice+Isoform+of+Pyruvate+Kinase+Is+Important+for+Cancer+Metabolism+and+Tumour+Growth&doi=10.1038%2Fnature06734"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">The M2 splice isoform of pyruvate kinase is important for cancer metabolism and tumor growth</span></div><div class="casAuthors">Christofk, Heather R.; Vander Heiden, Matthew G.; Harris, Marian H.; Ramanathan, Arvind; Gerszten, Robert E.; Wei, Ru; Fleming, Mark D.; Schreiber, Stuart L.; Cantley, Lewis C.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">452</span>
        (<span class="NLM_cas:issue">7184</span>),
    <span class="NLM_cas:pages">230-233</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Many tumor cells have elevated rates of glucose uptake but reduced rates of oxidative phosphorylation.  This persistence of high lactate prodn. by tumors in the presence of oxygen, known as aerobic glycolysis, was first noted by Otto Warburg more than 75 yr ago.  How tumor cells establish this altered metabolic phenotype and whether it is essential for tumorigenesis is as yet unknown.  Here we show that a single switch in a splice isoform of the glycolytic enzyme pyruvate kinase is necessary for the shift in cellular metab. to aerobic glycolysis and that this promotes tumorigenesis.  Tumor cells have been shown to express exclusively the embryonic M2 isoform of pyruvate kinase.  Here we use short hairpin RNA to knockdown pyruvate kinase M2 expression in human cancer cell lines and replace it with pyruvate kinase M1.  Switching pyruvate kinase expression to the M1 (adult) isoform leads to reversal of the Warburg effect, as judged by reduced lactate prodn. and increased oxygen consumption, and this correlates with a reduced ability to form tumors in nude mouse xenografts.  These results demonstrate that M2 expression is necessary for aerobic glycolysis and that this metabolic phenotype provides a selective growth advantage for tumor cells in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpvmgcXNsSjpbVg90H21EOLACvtfcHk0lgN50-ru55RPw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXjt1Gnu74%253D&md5=e7e6b7cc09477e1f9d270018d1c38be4</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fnature06734&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature06734%26sid%3Dliteratum%253Aachs%26aulast%3DChristofk%26aufirst%3DH.%2BR.%26aulast%3DVander%2BHeiden%26aufirst%3DM.%2BG.%26aulast%3DHarris%26aufirst%3DM.%2BH.%26aulast%3DRamanathan%26aufirst%3DA.%26aulast%3DGerszten%26aufirst%3DR.%2BE.%26aulast%3DWei%26aufirst%3DR.%26aulast%3DFleming%26aufirst%3DM.%2BD.%26aulast%3DSchreiber%26aufirst%3DS.%2BL.%26aulast%3DCantley%26aufirst%3DL.%2BC.%26atitle%3DThe%2520M2%2520Splice%2520Isoform%2520of%2520Pyruvate%2520Kinase%2520Is%2520Important%2520for%2520Cancer%2520Metabolism%2520and%2520Tumour%2520Growth%26jtitle%3DNature%26date%3D2008%26volume%3D452%26spage%3D230%26epage%3D233%26doi%3D10.1038%2Fnature06734" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Granchi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fancelli, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minutolo, F.</span></span> <span> </span><span class="NLM_article-title">An Update on Therapeutic Opportunities Offered by Cancer Glycolytic Metabolism</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">4915</span>– <span class="NLM_lpage">4925</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2014.09.041</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=10.1016%2Fj.bmcl.2014.09.041" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=25288186" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1eitbzJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2014&pages=4915-4925&author=C.+Granchiauthor=D.+Fancelliauthor=F.+Minutolo&title=An+Update+on+Therapeutic+Opportunities+Offered+by+Cancer+Glycolytic+Metabolism&doi=10.1016%2Fj.bmcl.2014.09.041"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">An update on therapeutic opportunities offered by cancer glycolytic metabolism</span></div><div class="casAuthors">Granchi, Carlotta; Fancelli, Daniele; Minutolo, Filippo</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">4915-4925</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Almost all invasive cancers, regardless of tissue origin, are characterized by specific modifications of their cellular energy metab.  In fact, a strong predominance of aerobic glycolysis over oxidative phosphorylation (Warburg effect) is usually assocd. with aggressive tumor phenotypes.  This metabolic shift offers a survival advantage to cancer cells, since they may continue to produce energy and anabolites even when they are exposed to either transient or permanent hypoxic conditions.  Moreover, it ensures a high prodn. rate of glycolysis intermediates, useful as building blocks for fast cell proliferation of cancer cells.  This peculiar metabolic profile may constitute an ideal target for therapeutic interventions that selectively hit cancer cells with minimal residual systemic toxicity.  In this review we provide an update about some of the most recent advances in the discovery of new bioactive mols. that are able to interfere with cancer glycolysis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIdWP6wz_jRbVg90H21EOLACvtfcHk0lhve3N8kqUiNw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1eitbzJ&md5=e8b3949321270dad2dcc4757c47b5aa6</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2014.09.041&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2014.09.041%26sid%3Dliteratum%253Aachs%26aulast%3DGranchi%26aufirst%3DC.%26aulast%3DFancelli%26aufirst%3DD.%26aulast%3DMinutolo%26aufirst%3DF.%26atitle%3DAn%2520Update%2520on%2520Therapeutic%2520Opportunities%2520Offered%2520by%2520Cancer%2520Glycolytic%2520Metabolism%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2014%26volume%3D24%26spage%3D4915%26epage%3D4925%26doi%3D10.1016%2Fj.bmcl.2014.09.041" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ganapathy-Kanniappan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geschwind, J. F.</span></span> <span> </span><span class="NLM_article-title">Tumor Glycolysis as a Target for Cancer Therapy: Progress and Prospects</span>. <i>Mol. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">152</span>, <span class="refDoi"> DOI: 10.1186/1476-4598-12-152</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=10.1186%2F1476-4598-12-152" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=24298908" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmsVyjtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2013&pages=152&author=S.+Ganapathy-Kanniappanauthor=J.+F.+Geschwind&title=Tumor+Glycolysis+as+a+Target+for+Cancer+Therapy%3A+Progress+and+Prospects&doi=10.1186%2F1476-4598-12-152"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Tumor glycolysis as a target for cancer therapy: progress and prospects</span></div><div class="casAuthors">Ganapathy-Kanniappan, Shanmugasundaram; Geschwind, Jean-Francois H.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">152/1-152/11, 11 pp.</span>CODEN:
                <span class="NLM_cas:coden">MCOACG</span>;
        ISSN:<span class="NLM_cas:issn">1476-4598</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">A review.  Altered energy metab. is a biochem. fingerprint of cancer cells that represents one of the "hallmarks of cancer".  This metabolic phenotype is characterized by preferential dependence on glycolysis (the process of conversion of glucose into pyruvate followed by lactate prodn.) for energy prodn. in an oxygen-independent manner.  Although glycolysis is less efficient than oxidative phosphorylation in the net yield of ATP (ATP), cancer cells adapt to this math. disadvantage by increased glucose up-take, which in turn facilitates a higher rate of glycolysis.  Apart from providing cellular energy, the metabolic intermediates of glycolysis also play a pivotal role in macromol. biosynthesis, thus conferring selective advantage to cancer cells under diminished nutrient supply.  Accumulating data also indicate that intracellular ATP is a crit. determinant of chemoresistance.  Under hypoxic conditions where glycolysis remains the predominant energy producing pathway sensitizing cancer cells would require intracellular depletion of ATP by inhibition of glycolysis.  Together, the oncogenic regulation of glycolysis and multifaceted roles of glycolytic components underscore the biol. significance of tumor glycolysis.  Thus targeting glycolysis remains attractive for therapeutic intervention.  Several preclin. investigations have indeed demonstrated the effectiveness of this therapeutic approach thereby supporting its scientific rationale.  Recent reviews have provided a wealth of information on the biochem. targets of glycolysis and their inhibitors.  The objective of this review is to present the most recent research on the cancer-specific role of glycolytic enzymes including their non-glycolytic functions in order to explore the potential for therapeutic opportunities.  Further, we discuss the translational potential of emerging drug candidates in light of tech. advances in treatment modalities such as image-guided targeted delivery of cancer therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoeWLa8w-XyRrVg90H21EOLACvtfcHk0lhve3N8kqUiNw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmsVyjtLs%253D&md5=1cd29873d9d8b8059aa75f5343df3f28</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1186%2F1476-4598-12-152&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1476-4598-12-152%26sid%3Dliteratum%253Aachs%26aulast%3DGanapathy-Kanniappan%26aufirst%3DS.%26aulast%3DGeschwind%26aufirst%3DJ.%2BF.%26atitle%3DTumor%2520Glycolysis%2520as%2520a%2520Target%2520for%2520Cancer%2520Therapy%253A%2520Progress%2520and%2520Prospects%26jtitle%3DMol.%2520Cancer%26date%3D2013%26volume%3D12%26spage%3D152%26doi%3D10.1186%2F1476-4598-12-152" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gupta, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wellen, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mazurek, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamezai, R. N.</span></span> <span> </span><span class="NLM_article-title">Pyruvate Kinase M2: Regulatory Circuits and Potential for Therapeutic Intervention</span>. <i>Curr. Pharm. Des.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">2595</span>– <span class="NLM_lpage">2606</span>, <span class="refDoi"> DOI: 10.2174/13816128113199990484</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=10.2174%2F13816128113199990484" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=23859618" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=1%3ACAS%3A528%3ADC%252BC2cXpt1Wmurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2014&pages=2595-2606&author=V.+Guptaauthor=K.+E.+Wellenauthor=S.+Mazurekauthor=R.+N.+Bamezai&title=Pyruvate+Kinase+M2%3A+Regulatory+Circuits+and+Potential+for+Therapeutic+Intervention&doi=10.2174%2F13816128113199990484"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Pyruvate Kinase M2: Regulatory Circuits and Potential for Therapeutic Intervention</span></div><div class="casAuthors">Gupta, Vibhor; Wellen, Kathryn E.; Mazurek, Sybille; Bamezai, Rameshwar N. K.</div><div class="citationInfo"><span class="NLM_cas:title">Current Pharmaceutical Design</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">2595-2606</span>CODEN:
                <span class="NLM_cas:coden">CPDEFP</span>;
        ISSN:<span class="NLM_cas:issn">1381-6128</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Cancer cells are characterized by reprogramming of energy metab.  Over the last decade, understanding of the metabolic changes that occur in cancer has increased dramatically, with great interest in targeting metab. for cancer therapy.  Pyruvate kinase isoenzyme type M2 (abbreviations: PKM2, M2-PK) plays a key role in modulating glucose metab. to support cell proliferation.  PKM2, like other PK isoforms, catalyzes the last energy-generating step in glycolysis, but is unique in its capacity to be regulated.  PKM2 is regulated at several cellular levels, including gene expression, alternative splicing and post-translational modification.  In addn., PKM2 is regulated by key metabolic intermediates and interacts with more than twenty different proteins.  Hence, this isoenzyme is an important regulator of glycolysis, and addnl. functions in other novel roles that have recently emerged.  Recent evidence indicates that intervening with the complex regulatory network of PKM2 has severe consequences on tumor cell proliferation, indicating the potential of this enzyme as a target for tumor therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohFL-uHUY_mrVg90H21EOLACvtfcHk0lhve3N8kqUiNw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXpt1Wmurg%253D&md5=cc9d6d0bf6050acf9038f4a8c429216e</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.2174%2F13816128113199990484&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F13816128113199990484%26sid%3Dliteratum%253Aachs%26aulast%3DGupta%26aufirst%3DV.%26aulast%3DWellen%26aufirst%3DK.%2BE.%26aulast%3DMazurek%26aufirst%3DS.%26aulast%3DBamezai%26aufirst%3DR.%2BN.%26atitle%3DPyruvate%2520Kinase%2520M2%253A%2520Regulatory%2520Circuits%2520and%2520Potential%2520for%2520Therapeutic%2520Intervention%26jtitle%3DCurr.%2520Pharm.%2520Des.%26date%3D2014%26volume%3D20%26spage%3D2595%26epage%3D2606%26doi%3D10.2174%2F13816128113199990484" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Luo, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Semenza, G. L.</span></span> <span> </span><span class="NLM_article-title">Emerging Roles of Pkm2 in Cell Metabolism and Cancer Progression</span>. <i>Trends Endocrinol. Metab.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">560</span>– <span class="NLM_lpage">566</span>, <span class="refDoi"> DOI: 10.1016/j.tem.2012.06.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=10.1016%2Fj.tem.2012.06.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=22824010" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtVOqu7fI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2012&pages=560-566&author=W.+Luoauthor=G.+L.+Semenza&title=Emerging+Roles+of+Pkm2+in+Cell+Metabolism+and+Cancer+Progression&doi=10.1016%2Fj.tem.2012.06.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Emerging roles of PKM2 in cell metabolism and cancer progression</span></div><div class="casAuthors">Luo, Weibo; Semenza, Gregg L.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Endocrinology and Metabolism</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">560-566</span>CODEN:
                <span class="NLM_cas:coden">TENME4</span>;
        ISSN:<span class="NLM_cas:issn">1043-2760</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Increased conversion of glucose to lactate is a key feature of many cancer cells that promotes rapid growth.  Pyruvate kinase M2 (PKM2) expression is increased and facilitates lactate prodn. in cancer cells.  Modulation of PKM2 catalytic activity also regulates the synthesis of DNA and lipids that are required for cell proliferation, and of NADPH that is required for redox homeostasis.  In addn. to its role as a pyruvate kinase, PKM2 also functions as a protein kinase and as a transcriptional coactivator.  These biochem. activities are controlled by allosteric regulators and post-translational modifications of PKM2 that include acetylation, oxidn., phosphorylation, prolyl hydroxylation, and sumoylation.  Given its pleiotropic effects on cancer biol., PKM2 represents an attractive target for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXplJG_hQB2rVg90H21EOLACvtfcHk0lhve3N8kqUiNw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtVOqu7fI&md5=2ae54658b3b6c05490de21b9b4310764</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.tem.2012.06.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tem.2012.06.010%26sid%3Dliteratum%253Aachs%26aulast%3DLuo%26aufirst%3DW.%26aulast%3DSemenza%26aufirst%3DG.%2BL.%26atitle%3DEmerging%2520Roles%2520of%2520Pkm2%2520in%2520Cell%2520Metabolism%2520and%2520Cancer%2520Progression%26jtitle%3DTrends%2520Endocrinol.%2520Metab.%26date%3D2012%26volume%3D23%26spage%3D560%26epage%3D566%26doi%3D10.1016%2Fj.tem.2012.06.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vander
Heiden, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christofk, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuman, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Subtelny, A. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharfi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harlow, E. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xian, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantley, L. C.</span></span> <span> </span><span class="NLM_article-title">Identification of Small Molecule Inhibitors of Pyruvate Kinase M2</span>. <i>Biochem. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>79</i></span>,  <span class="NLM_fpage">1118</span>– <span class="NLM_lpage">1124</span>, <span class="refDoi"> DOI: 10.1016/j.bcp.2009.12.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=10.1016%2Fj.bcp.2009.12.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=20005212" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=1%3ACAS%3A528%3ADC%252BC3cXitVOhsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=2010&pages=1118-1124&author=M.+G.+Vander%0AHeidenauthor=H.+R.+Christofkauthor=E.+Schumanauthor=A.+O.+Subtelnyauthor=H.+Sharfiauthor=E.+E.+Harlowauthor=J.+Xianauthor=L.+C.+Cantley&title=Identification+of+Small+Molecule+Inhibitors+of+Pyruvate+Kinase+M2&doi=10.1016%2Fj.bcp.2009.12.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of small molecule inhibitors of pyruvate kinase M2</span></div><div class="casAuthors">Vander Heiden, Matthew G.; Christofk, Heather R.; Schuman, Eli; Subtelny, Alexander O.; Sharfi, Hadar; Harlow, Edward E.; Xian, Jun; Cantley, Lewis C.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Pharmacology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">79</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1118-1124</span>CODEN:
                <span class="NLM_cas:coden">BCPCA6</span>;
        ISSN:<span class="NLM_cas:issn">0006-2952</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A common feature of tumors arising from diverse tissue types is a reliance on aerobic glycolysis for glucose metab.  This metabolic difference between cancer cells and normal cells could be exploited for therapeutic benefit in patients.  Cancer cells universally express the M2 isoform of the glycolytic enzyme pyruvate kinase (PKM2), and previous work has demonstrated that PKM2 expression is necessary for aerobic glycolysis and cell proliferation in vivo.  Because most normal tissues express an isoform of pyruvate kinase other than PKM2, selective targeting of PKM2 provides an opportunity to target cell metab. for cancer therapy.  PKM2 has an identical catalytic site as the related M1 splice variant (PKM1).  However, isoform selective inhibition is possible as PKM2 contains a unique region for allosteric regulation.  We have screened a library of greater than 1,00,000 small mols. to identify such inhibitors.  The inhibitors identified for PKM2 fell primarily into three distinct structural classes.  The most potent PKM2 inhibitor resulted in decreased glycolysis and increased cell death following loss of growth factor signaling.  At least part of this effect was due to on-target PKM2 inhibition as less cell death was obsd. in cells engineered to express PKM1.  These data suggest that isoform selective inhibition of PKM2 with small mols. is feasible and support the hypothesis that inhibition of glucose metab. in cancer cells is a viable strategy to treat human malignancy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHtS-udaOUobVg90H21EOLACvtfcHk0ljh7pa9IbiYOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXitVOhsbw%253D&md5=efe6d924601716d806f50de3c231c995</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.bcp.2009.12.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bcp.2009.12.003%26sid%3Dliteratum%253Aachs%26aulast%3DVander%2BHeiden%26aufirst%3DM.%2BG.%26aulast%3DChristofk%26aufirst%3DH.%2BR.%26aulast%3DSchuman%26aufirst%3DE.%26aulast%3DSubtelny%26aufirst%3DA.%2BO.%26aulast%3DSharfi%26aufirst%3DH.%26aulast%3DHarlow%26aufirst%3DE.%2BE.%26aulast%3DXian%26aufirst%3DJ.%26aulast%3DCantley%26aufirst%3DL.%2BC.%26atitle%3DIdentification%2520of%2520Small%2520Molecule%2520Inhibitors%2520of%2520Pyruvate%2520Kinase%2520M2%26jtitle%3DBiochem.%2520Pharmacol.%26date%3D2010%26volume%3D79%26spage%3D1118%26epage%3D1124%26doi%3D10.1016%2Fj.bcp.2009.12.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, X.</span></span> <span> </span><span class="NLM_article-title">Shikonin and Its Analogs Inhibit Cancer Cell Glycolysis by Targeting Tumor Pyruvate Kinase-M2</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">4297</span>– <span class="NLM_lpage">4306</span>, <span class="refDoi"> DOI: 10.1038/onc.2011.137</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=10.1038%2Fonc.2011.137" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=21516121" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=1%3ACAS%3A528%3ADC%252BC3MXltFyqur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2011&pages=4297-4306&author=J.+Chenauthor=J.+Xieauthor=Z.+Jiangauthor=B.+Wangauthor=Y.+Wangauthor=X.+Hu&title=Shikonin+and+Its+Analogs+Inhibit+Cancer+Cell+Glycolysis+by+Targeting+Tumor+Pyruvate+Kinase-M2&doi=10.1038%2Fonc.2011.137"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Shikonin and its analogs inhibit cancer cell glycolysis by targeting tumor pyruvate kinase-M2</span></div><div class="casAuthors">Chen, J.; Xie, J.; Jiang, Z.; Wang, B.; Wang, Y.; Hu, X.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">42</span>),
    <span class="NLM_cas:pages">4297-4306</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">We recently reported that shikonin and its analogs were a class of necroptotic inducers that could bypass cancer drug resistance.  However, the mol. targets of shikonin are not known.  Here, we showed that shikonin and its analogs are inhibitors of tumor-specific pyruvate kinase-M2 (PKM2), among which shikonin and its enantiomeric isomer alkannin were the most potent and showed promising selectivity, i.e., shikonin and alkannin at concns. that resulted in over 50% inhibition of PKM2 activity did not inhibit PKM1 and pyruvate kinase-L (PKL).  Shikonin and alkannin significantly inhibited the glycolytic rate, as manifested by cellular lactate prodn. and glucose consumption in drug-sensitive and resistant cancer cell lines (MCF-7, MCF-7/Adr, MCF-7/Bcl-2, MCF-7/Bcl-xL and A549) that primarily express PKM2.  HeLa cells transfected with PKM1 showed reduced sensitivity to shikonin- or alkannin-induced cell death.  To the best of our knowledge, shikonin and alkannin are the most potent and specific inhibitors to PKM2 reported so far.  As PKM2 universally expresses in cancer cells and dictates the last rate-limiting step of glycolysis vital for cancer cell proliferation and survival, enantiomeric shikonin and alkannin may have potential in future clin. application.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdGlWk1wpLfLVg90H21EOLACvtfcHk0ljh7pa9IbiYOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXltFyqur8%253D&md5=d4fd7186fc4e0df8578dd21add883f7b</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1038%2Fonc.2011.137&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fonc.2011.137%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DXie%26aufirst%3DJ.%26aulast%3DJiang%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DB.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DHu%26aufirst%3DX.%26atitle%3DShikonin%2520and%2520Its%2520Analogs%2520Inhibit%2520Cancer%2520Cell%2520Glycolysis%2520by%2520Targeting%2520Tumor%2520Pyruvate%2520Kinase-M2%26jtitle%3DOncogene%26date%3D2011%26volume%3D30%26spage%3D4297%26epage%3D4306%26doi%3D10.1038%2Fonc.2011.137" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ning, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNutt, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, Y.</span></span> <span> </span><span class="NLM_article-title">Discovery of Novel Naphthoquinone Derivatives as Inhibitors of the Tumor Cell Specific M2 Isoform of Pyruvate Kinase</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>138</i></span>,  <span class="NLM_fpage">343</span>– <span class="NLM_lpage">352</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2017.06.064</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=10.1016%2Fj.ejmech.2017.06.064" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=28688274" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtFKku77E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=138&publication_year=2017&pages=343-352&author=X.+Ningauthor=H.+Qiauthor=R.+Liauthor=Y.+Liauthor=Y.+Jinauthor=M.+A.+McNuttauthor=J.+Liuauthor=Y.+Yin&title=Discovery+of+Novel+Naphthoquinone+Derivatives+as+Inhibitors+of+the+Tumor+Cell+Specific+M2+Isoform+of+Pyruvate+Kinase&doi=10.1016%2Fj.ejmech.2017.06.064"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of novel naphthoquinone derivatives as inhibitors of the tumor cell specific M2 isoform of pyruvate kinase</span></div><div class="casAuthors">Ning, Xianling; Qi, Hailong; Li, Ridong; Li, Yunqiao; Jin, Yan; McNutt, Michael A.; Liu, Junyi; Yin, Yuxin</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">138</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">343-352</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Pyruvate kinase M2 (PKM2) is a rate-limiting enzyme of the glycolytic pathway which is highly expressed in cancer cells.  Cancer cells rely heavily on PKM2 for anabolic and energy requirements, and specific targeting of PKM2 therefore has potential as strategy for cancer therapy.  Here, the authors report the synthesis and biol. evaluation of novel naphthoquinone derivs. as selective small mol. inhibitors of PKM2.  Some target compds., such as compd. piperidine-1-carbodithioic acid 3-methyl-1,4-dioxo-1,4-dihydro-naphthalen-2-ylmethyl ester, displayed more potent PKM2 inhibitory activity than the reported optimal PKM2 inhibitor shikonin.  The well performing compd. piperidine-1-carbodithioic acid 3-methyl-1,4-dioxo-1,4-dihydro-naphthalen-2-ylmethyl ester also showed nanomolar antiproliferative activity toward a series of cancer cell lines with high expression of PKM2 including HCT116, Hela and H1299 with IC50 values ranging from 0.18 to 1.56 μM.  Moreover, compd. piperidine-1-carbodithioic acid 3-methyl-1,4-dioxo-1,4-dihydro-naphthalen-2-ylmethyl ester exhibited more cytotoxicity on cancer cells than normal cells.  The identification of novel potent small mol. inhibitors of PKM2 not only offers candidate compds. for cancer therapy, but also provides a tool with which to evaluate the function of PKM2 in depth.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp87KANg1TeAbVg90H21EOLACvtfcHk0ljh7pa9IbiYOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtFKku77E&md5=461ba2cd59b0bb2f49da7dfc1851dadb</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2017.06.064&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2017.06.064%26sid%3Dliteratum%253Aachs%26aulast%3DNing%26aufirst%3DX.%26aulast%3DQi%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DR.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DJin%26aufirst%3DY.%26aulast%3DMcNutt%26aufirst%3DM.%2BA.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DYin%26aufirst%3DY.%26atitle%3DDiscovery%2520of%2520Novel%2520Naphthoquinone%2520Derivatives%2520as%2520Inhibitors%2520of%2520the%2520Tumor%2520Cell%2520Specific%2520M2%2520Isoform%2520of%2520Pyruvate%2520Kinase%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2017%26volume%3D138%26spage%3D343%26epage%3D352%26doi%3D10.1016%2Fj.ejmech.2017.06.064" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boxer, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vander Heiden, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skoumbourdis, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Southall, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Veith, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leister, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Austin, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, H. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inglese, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantley, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Auld, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, C. J.</span></span> <span> </span><span class="NLM_article-title">Evaluation of Substituted N,N′-Diarylsulfonamides as Activators of the Tumor Cell Specific M2 Isoform of Pyruvate Kinase</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">1048</span>– <span class="NLM_lpage">1055</span>, <span class="refDoi"> DOI: 10.1021/jm901577g</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901577g" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsFKgurnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=1048-1055&author=M.+B.+Boxerauthor=J.+K.+Jiangauthor=M.+G.+Vander+Heidenauthor=M.+Shenauthor=A.+P.+Skoumbourdisauthor=N.+Southallauthor=H.+Veithauthor=W.+Leisterauthor=C.+P.+Austinauthor=H.+W.+Parkauthor=J.+Ingleseauthor=L.+C.+Cantleyauthor=D.+S.+Auldauthor=C.+J.+Thomas&title=Evaluation+of+Substituted+N%2CN%E2%80%B2-Diarylsulfonamides+as+Activators+of+the+Tumor+Cell+Specific+M2+Isoform+of+Pyruvate+Kinase&doi=10.1021%2Fjm901577g"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Evaluation of Substituted N,N'-Diarylsulfonamides as Activators of the Tumor Cell Specific M2 Isoform of Pyruvate Kinase</span></div><div class="casAuthors">Boxer, Matthew B.; Jiang, Jian-kang; Vander Heiden, Matthew G.; Shen, Min; Skoumbourdis, Amanda P.; Southall, Noel; Veith, Henrike; Leister, William; Austin, Christopher P.; Park, Hee Won; Inglese, James; Cantley, Lewis C.; Auld, Douglas S.; Thomas, Craig J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1048-1055</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The metab. of cancer cells is altered to support rapid proliferation.  Pharmacol. activators of a tumor cell specific pyruvate kinase isoenzyme (PKM2) may be an approach for altering the classic Warburg effect characteristic of aberrant metab. in cancer cells yielding a novel antiproliferation strategy.  In this manuscript, we detail the discovery of a series of substituted N,N'-diarylsulfonamides as activators of PKM2.  The synthesis of numerous analogs and the evaluation of structure-activity relationships are presented as well as assessments of mechanism and selectivity.  Several agents are found that have good potencies and appropriate soly. for use as chem. probes of PKM2 including 55 (AC50 = 43 nM, max. response = 84%; soly. = 7.3 μg/mL), 56 (AC50 = 99 nM, max. response = 84%; soly. = 5.7 μg/mL), and 58 (AC50 = 38 nM, max. response = 82%; soly. = 51.2 μg/mL).  The small mols. described here represent first-in-class activators of PKM2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq4C0nWknRHC7Vg90H21EOLACvtfcHk0lgON8p7mWHLOQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsFKgurnF&md5=006f6860e53fa40338899076049ebbe7</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fjm901577g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901577g%26sid%3Dliteratum%253Aachs%26aulast%3DBoxer%26aufirst%3DM.%2BB.%26aulast%3DJiang%26aufirst%3DJ.%2BK.%26aulast%3DVander%2BHeiden%26aufirst%3DM.%2BG.%26aulast%3DShen%26aufirst%3DM.%26aulast%3DSkoumbourdis%26aufirst%3DA.%2BP.%26aulast%3DSouthall%26aufirst%3DN.%26aulast%3DVeith%26aufirst%3DH.%26aulast%3DLeister%26aufirst%3DW.%26aulast%3DAustin%26aufirst%3DC.%2BP.%26aulast%3DPark%26aufirst%3DH.%2BW.%26aulast%3DInglese%26aufirst%3DJ.%26aulast%3DCantley%26aufirst%3DL.%2BC.%26aulast%3DAuld%26aufirst%3DD.%2BS.%26aulast%3DThomas%26aufirst%3DC.%2BJ.%26atitle%3DEvaluation%2520of%2520Substituted%2520N%252CN%25E2%2580%25B2-Diarylsulfonamides%2520as%2520Activators%2520of%2520the%2520Tumor%2520Cell%2520Specific%2520M2%2520Isoform%2520of%2520Pyruvate%2520Kinase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D1048%26epage%3D1055%26doi%3D10.1021%2Fjm901577g" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boxer, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vander Heiden, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skoumbourdis, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Southall, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Veith, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leister, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Austin, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, H. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inglese, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantley, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Auld, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, C. J.</span></span> <span> </span><span class="NLM_article-title">Evaluation of Thieno[3,2-B]Pyrrole[3,2-D]Pyridazinones as Activators of the Tumor Cell Specific M2 Isoform of Pyruvate Kinase</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">3387</span>– <span class="NLM_lpage">3393</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2010.04.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=10.1016%2Fj.bmcl.2010.04.015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=20451379" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=1%3ACAS%3A528%3ADC%252BC3cXmsVOhur4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2010&pages=3387-3393&author=J.+K.+Jiangauthor=M.+B.+Boxerauthor=M.+G.+Vander+Heidenauthor=M.+Shenauthor=A.+P.+Skoumbourdisauthor=N.+Southallauthor=H.+Veithauthor=W.+Leisterauthor=C.+P.+Austinauthor=H.+W.+Parkauthor=J.+Ingleseauthor=L.+C.+Cantleyauthor=D.+S.+Auldauthor=C.+J.+Thomas&title=Evaluation+of+Thieno%5B3%2C2-B%5DPyrrole%5B3%2C2-D%5DPyridazinones+as+Activators+of+the+Tumor+Cell+Specific+M2+Isoform+of+Pyruvate+Kinase&doi=10.1016%2Fj.bmcl.2010.04.015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Evaluation of thieno[3,2-b]pyrrole[3,2-d]pyridazinones as activators of the tumor cell specific M2 isoform of pyruvate kinase</span></div><div class="casAuthors">Jiang, Jian-kang; Boxer, Matthew B.; Vander Heiden, Matthew G.; Shen, Min; Skoumbourdis, Amanda P.; Southall, Noel; Veith, Henrike; Leister, William; Austin, Christopher P.; Park, Hee Won; Inglese, James; Cantley, Lewis C.; Auld, Douglas S.; Thomas, Craig J.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">3387-3393</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Cancer cells have distinct metabolic needs that are different from normal cells and can be exploited for development of anti-cancer therapeutics.  Activation of the tumor specific M2 form of pyruvate kinase (PKM2) is a potential strategy for returning cancer cells to a metabolic state characteristic of normal cells.  A series of activators of PKM2, e.g., I, based upon a substituted thieno[3,2-b]pyrrole[3,2-d]pyridazinone scaffold were prepd.  The synthesis of these agents, structure-activity relationships, anal. of activity at related targets (PKM1, PKR and PKL) and examn. of aq. soly. were investigated.  These agents represent the second reported chemotype for activation of PKM2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5vYZSvemjsLVg90H21EOLACvtfcHk0lgON8p7mWHLOQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXmsVOhur4%253D&md5=99a1753511b6643d9ac2fd7ca15da5ff</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2010.04.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2010.04.015%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DJ.%2BK.%26aulast%3DBoxer%26aufirst%3DM.%2BB.%26aulast%3DVander%2BHeiden%26aufirst%3DM.%2BG.%26aulast%3DShen%26aufirst%3DM.%26aulast%3DSkoumbourdis%26aufirst%3DA.%2BP.%26aulast%3DSouthall%26aufirst%3DN.%26aulast%3DVeith%26aufirst%3DH.%26aulast%3DLeister%26aufirst%3DW.%26aulast%3DAustin%26aufirst%3DC.%2BP.%26aulast%3DPark%26aufirst%3DH.%2BW.%26aulast%3DInglese%26aufirst%3DJ.%26aulast%3DCantley%26aufirst%3DL.%2BC.%26aulast%3DAuld%26aufirst%3DD.%2BS.%26aulast%3DThomas%26aufirst%3DC.%2BJ.%26atitle%3DEvaluation%2520of%2520Thieno%255B3%252C2-B%255DPyrrole%255B3%252C2-D%255DPyridazinones%2520as%2520Activators%2520of%2520the%2520Tumor%2520Cell%2520Specific%2520M2%2520Isoform%2520of%2520Pyruvate%2520Kinase%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2010%26volume%3D20%26spage%3D3387%26epage%3D3393%26doi%3D10.1016%2Fj.bmcl.2010.04.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Walsh, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brimacombe, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Veith, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bougie, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daniel, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leister, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantley, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Israelsen, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vander Heiden, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Auld, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boxer, M. B.</span></span> <span> </span><span class="NLM_article-title">2-Oxo-N-Aryl-1,2,3,4-Tetrahydroquinoline-6-Sulfonamides as Activators of the Tumor Cell Specific M2 Isoform of Pyruvate Kinase</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">6322</span>– <span class="NLM_lpage">6327</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2011.08.114</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=10.1016%2Fj.bmcl.2011.08.114" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=21958545" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht12jtr7K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2011&pages=6322-6327&author=M.+J.+Walshauthor=K.+R.+Brimacombeauthor=H.+Veithauthor=J.+M.+Bougieauthor=T.+Danielauthor=W.+Leisterauthor=L.+C.+Cantleyauthor=W.+J.+Israelsenauthor=M.+G.+Vander+Heidenauthor=M.+Shenauthor=D.+S.+Auldauthor=C.+J.+Thomasauthor=M.+B.+Boxer&title=2-Oxo-N-Aryl-1%2C2%2C3%2C4-Tetrahydroquinoline-6-Sulfonamides+as+Activators+of+the+Tumor+Cell+Specific+M2+Isoform+of+Pyruvate+Kinase&doi=10.1016%2Fj.bmcl.2011.08.114"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">2-Oxo-N-aryl-1,2,3,4-tetrahydroquinoline-6-sulfonamides as activators of the tumor cell specific M2 isoform of pyruvate kinase</span></div><div class="casAuthors">Walsh, Martin J.; Brimacombe, Kyle R.; Veith, Henrike; Bougie, James M.; Daniel, Thomas; Leister, William; Cantley, Lewis C.; Israelsen, William J.; Vander Heiden, Matthew G.; Shen, Min; Auld, Douglas S.; Thomas, Craig J.; Boxer, Matthew B.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">6322-6327</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Compared to normal differentiated cells, cancer cells have altered metabolic regulation to support biosynthesis and the expression of the M2 isoenzyme of pyruvate kinase (PKM2) plays an important role in this anabolic metab.  While the M1 isoform is a highly active enzyme, the alternatively spliced M2 variant is considerably less active and expressed in tumors.  While the exact mechanism by which decreased pyruvate kinase activity contributes to anabolic metab. remains unclear, it is hypothesized that activation of PKM2 to levels seen with PKM1 may promote a metabolic program that is not conducive to cell proliferation.  Here we report the third chemotype in a series of PKM2 activators based on the 2-oxo-N-aryl-1,2,3,4-tetrahydroquinoline-6-sulfonamide scaffold.  The synthesis, structure activity relationships, selectivity and notable physiochem. properties are described.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv4lcw1kll2bVg90H21EOLACvtfcHk0lh0L5jVPEzmnA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht12jtr7K&md5=82971c401c538f105d4c9144837ec175</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2011.08.114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2011.08.114%26sid%3Dliteratum%253Aachs%26aulast%3DWalsh%26aufirst%3DM.%2BJ.%26aulast%3DBrimacombe%26aufirst%3DK.%2BR.%26aulast%3DVeith%26aufirst%3DH.%26aulast%3DBougie%26aufirst%3DJ.%2BM.%26aulast%3DDaniel%26aufirst%3DT.%26aulast%3DLeister%26aufirst%3DW.%26aulast%3DCantley%26aufirst%3DL.%2BC.%26aulast%3DIsraelsen%26aufirst%3DW.%2BJ.%26aulast%3DVander%2BHeiden%26aufirst%3DM.%2BG.%26aulast%3DShen%26aufirst%3DM.%26aulast%3DAuld%26aufirst%3DD.%2BS.%26aulast%3DThomas%26aufirst%3DC.%2BJ.%26aulast%3DBoxer%26aufirst%3DM.%2BB.%26atitle%3D2-Oxo-N-Aryl-1%252C2%252C3%252C4-Tetrahydroquinoline-6-Sulfonamides%2520as%2520Activators%2520of%2520the%2520Tumor%2520Cell%2520Specific%2520M2%2520Isoform%2520of%2520Pyruvate%2520Kinase%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2011%26volume%3D21%26spage%3D6322%26epage%3D6327%26doi%3D10.1016%2Fj.bmcl.2011.08.114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xue, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ivanov, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span> <span> </span><span class="NLM_article-title">Natural Product Micheliolide (Mcl) Irreversibly Activates Pyruvate Kinase M2 and Suppresses Leukemia</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">4155</span>– <span class="NLM_lpage">4164</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00241</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00241" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=1%3ACAS%3A528%3ADC%252BC1cXnsFSguro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=4155-4164&author=J.+Liauthor=S.+Liauthor=J.+Guoauthor=Q.+Liauthor=J.+Longauthor=C.+Maauthor=Y.+Dingauthor=C.+Yanauthor=L.+Liauthor=Z.+Wuauthor=H.+Zhuauthor=K.+K.+Liauthor=L.+Wenauthor=Q.+Zhangauthor=Q.+Xueauthor=C.+Zhaoauthor=N.+Liuauthor=I.+Ivanovauthor=M.+Luoauthor=R.+Xiauthor=H.+Longauthor=P.+G.+Wangauthor=Y.+Chen&title=Natural+Product+Micheliolide+%28Mcl%29+Irreversibly+Activates+Pyruvate+Kinase+M2+and+Suppresses+Leukemia&doi=10.1021%2Facs.jmedchem.8b00241"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Natural Product Micheliolide (MCL) Irreversibly Activates Pyruvate Kinase M2 and Suppresses Leukemia</span></div><div class="casAuthors">Li, Jing; Li, Shanshan; Guo, Jianshuang; Li, Qiuying; Long, Jing; Ma, Cheng; Ding, Yahui; Yan, Chunli; Li, Liangwei; Wu, Zhigang; Zhu, He; Li, Keqin Kathy; Wen, Liuqing; Zhang, Quan; Xue, Qingqing; Zhao, Caili; Liu, Ning; Ivanov, Ivaylo; Luo, Ming; Xi, Rimo; Long, Haibo; Wang, Peng George; Chen, Yue</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4155-4164</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Metabolic reprogramming of cancer cells is essential for tumorigenesis in which pyruvate kinase M2 (PKM2), the low activity isoform of pyruvate kinase, plays a crit. role.  Herein, we describe the identification of a nature-product-derived micheliolide (MCL) that selectively activates PKM2 through the covalent binding at residue cysteine424 (C424), which is not contained in PKM1.  This interaction promotes more tetramer formation, inhibits the lysine433 (K433) acetylation, and influences the translocation of PKM2 into the nucleus.  In addn., the pro-drug dimethylaminomicheliolide (DMAMCL) with similar properties as MCL significantly suppresses the growth of leukemia cells and tumorigenesis in a zebrafish xenograft model.  Cell-based assay with knock down PKM2 expression verifies that the effects of MCL are dependent on PKM2 expression.  DMAMCL is currently in clin. trials in Australia.  Our discovery may provide a valuable pharmacol. mechanism for clin. treatment and benefit the development of new anticancer agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHlWteclCMHbVg90H21EOLACvtfcHk0lh0L5jVPEzmnA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXnsFSguro%253D&md5=8742015134a65572dee296a5e88fa861</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00241&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00241%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DGuo%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DQ.%26aulast%3DLong%26aufirst%3DJ.%26aulast%3DMa%26aufirst%3DC.%26aulast%3DDing%26aufirst%3DY.%26aulast%3DYan%26aufirst%3DC.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DWu%26aufirst%3DZ.%26aulast%3DZhu%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DK.%2BK.%26aulast%3DWen%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DXue%26aufirst%3DQ.%26aulast%3DZhao%26aufirst%3DC.%26aulast%3DLiu%26aufirst%3DN.%26aulast%3DIvanov%26aufirst%3DI.%26aulast%3DLuo%26aufirst%3DM.%26aulast%3DXi%26aufirst%3DR.%26aulast%3DLong%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DP.%2BG.%26aulast%3DChen%26aufirst%3DY.%26atitle%3DNatural%2520Product%2520Micheliolide%2520%2528Mcl%2529%2520Irreversibly%2520Activates%2520Pyruvate%2520Kinase%2520M2%2520and%2520Suppresses%2520Leukemia%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D4155%26epage%3D4164%26doi%3D10.1021%2Facs.jmedchem.8b00241" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsu, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chou, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chuang, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, L. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kulp, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C. S.</span></span> <span> </span><span class="NLM_article-title">Identification and Characterization of a Novel Integrin-Linked Kinase Inhibitor</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">6364</span>– <span class="NLM_lpage">6374</span>, <span class="refDoi"> DOI: 10.1021/jm2007744</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm2007744" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtVOns7rK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=6364-6374&author=S.+L.+Leeauthor=E.+C.+Hsuauthor=C.+C.+Chouauthor=H.+C.+Chuangauthor=L.+Y.+Baiauthor=S.+K.+Kulpauthor=C.+S.+Chen&title=Identification+and+Characterization+of+a+Novel+Integrin-Linked+Kinase+Inhibitor&doi=10.1021%2Fjm2007744"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Identification and Characterization of a Novel Integrin-Linked Kinase Inhibitor</span></div><div class="casAuthors">Lee, Su-Lin; Hsu, En-Chi; Chou, Chih-Chien; Chuang, Hsiao-Ching; Bai, Li-Yuan; Kulp, Samuel K.; Chen, Ching-Shih</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">6364-6374</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Integrin-linked kinase (ILK) represents a relevant target for cancer therapy in light of its role in promoting oncogenesis and tumor progression.  Through the screening of an inhouse focused compd. library, we identified N-methyl-3-(1-(4-(piperazin-1-yl)phenyl)-5-(4'-(trifluoromethyl)-[1,1'-biphenyl]-4-yl)-1H-pyrazol-3-yl)propanamide (I) as a novel ILK inhibitor (IC50, 0.6 μM), which exhibited high in vitro potency against a panel of prostate and breast cancer cell lines (IC50, 1-2.5 μM), while normal epithelial cells were unaffected.  I facilitated the dephosphorylation of Akt at Ser-473 and other ILK targets, including glycogen synthase kinase-3β and myosin light chain.  Moreover, I suppressed the expression of the transcription/translation factor YB-1 and its targets HER2 and EGFR in PC-3 cells, which could be rescued by the stable expression of constitutively active ILK.  Evidence indicates that I induced autophagy and apoptosis, both of which were integral to its antiproliferative activity.  Together, this broad spectrum of mechanisms underlies the therapeutic potential of I in cancer treatment, which is manifested by its in vivo efficacy as a single oral agent in suppressing PC-3 xenograft tumor growth.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpenckDD_XVxbVg90H21EOLACvtfcHk0lh0L5jVPEzmnA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtVOns7rK&md5=44d060cd589c08abf58cc3e3fb6af542</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fjm2007744&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm2007744%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DS.%2BL.%26aulast%3DHsu%26aufirst%3DE.%2BC.%26aulast%3DChou%26aufirst%3DC.%2BC.%26aulast%3DChuang%26aufirst%3DH.%2BC.%26aulast%3DBai%26aufirst%3DL.%2BY.%26aulast%3DKulp%26aufirst%3DS.%2BK.%26aulast%3DChen%26aufirst%3DC.%2BS.%26atitle%3DIdentification%2520and%2520Characterization%2520of%2520a%2520Novel%2520Integrin-Linked%2520Kinase%2520Inhibitor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D6364%26epage%3D6374%26doi%3D10.1021%2Fjm2007744" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pike
Winer, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, M.</span></span> <span> </span><span class="NLM_article-title">Rapid Analysis of Glycolytic and Oxidative Substrate Flux of Cancer Cells in a Microplate</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">e109916</span>, <span class="refDoi"> DOI: 10.1371/journal.pone.0109916</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=10.1371%2Fjournal.pone.0109916" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=25360519" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFahsr%252FJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=e109916&author=L.+S.+Pike%0AWinerauthor=M.+Wu&title=Rapid+Analysis+of+Glycolytic+and+Oxidative+Substrate+Flux+of+Cancer+Cells+in+a+Microplate&doi=10.1371%2Fjournal.pone.0109916"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Rapid analysis of glycolytic and oxidative substrate flux of cancer cells in a microplate</span></div><div class="casAuthors">Pike Winer, Lisa S.; Wu, Min</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">e109916/1-e109916/14, 14 pp.</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Cancer cells exhibit remarkable alterations in cellular metab., particularly in their nutrient substrate preference.  We have devised several exptl. methods that rapidly analyze the metabolic substrate flux in cancer cells: glycolysis and the oxidn. of major fuel substrates glucose, glutamine, and fatty acids.  Using the XF Extracellular Flux analyzer, these methods measure, in real-time, the oxygen consumption rate OCR and extracellular acidification rate ECAR of living cells in a microplate as they respond to substrates and metabolic perturbation agents.  In proof-of-principle expts., we analyzed substrate flux and mitochondrial bioenergetics of two human glioblastoma cell lines, SF188s and SF188f, which were derived from the same parental cell line but proliferate at slow and fast rates, resp.  These analyses led to three interesting observations: (1) both cell lines respired effectively with substantial endogenous substrate respiration; (2) SF188f cells underwent a significant shift from glycolytic to oxidative metab., along with a high rate of glutamine oxidn. relative to SF188s cells; and (3) the mitochondrial proton leak-linked respiration of SF188f cells increased significantly compared to SF188s cells.  It is plausible that the proton leak of SF188f cells may play a role in allowing continuous glutamine-fueled anaplerotic TCA cycle flux by partially uncoupling the TCA cycle from oxidative phosphorylation.  Taken together, these rapid, sensitive and high-throughput substrate flux anal. methods introduce highly valuable approaches for developing a greater understanding of genetic and epigenetic pathways that regulate cellular metab., and the development of therapies that target cancer metab.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkMfX8T6XL47Vg90H21EOLACvtfcHk0lhCnIQ_9Q4IBA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvFahsr%252FJ&md5=5c3143d1026c5658580067a21a467763</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0109916&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0109916%26sid%3Dliteratum%253Aachs%26aulast%3DPike%2BWiner%26aufirst%3DL.%2BS.%26aulast%3DWu%26aufirst%3DM.%26atitle%3DRapid%2520Analysis%2520of%2520Glycolytic%2520and%2520Oxidative%2520Substrate%2520Flux%2520of%2520Cancer%2520Cells%2520in%2520a%2520Microplate%26jtitle%3DPLoS%2520One%26date%3D2014%26volume%3D9%26spage%3De109916%26doi%3D10.1371%2Fjournal.pone.0109916" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, T. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, P. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsiao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hua, K. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuo, M. L.</span></span> <span> </span><span class="NLM_article-title">Cytosolic Pkm2 Stabilizes Mutant Egfr Protein Expression through Regulating Hsp90-Egfr Association</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">3387</span>– <span class="NLM_lpage">3398</span>, <span class="refDoi"> DOI: 10.1038/onc.2015.397</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=10.1038%2Fonc.2015.397" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=26500058" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslant7jK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2016&pages=3387-3398&author=Y.+C.+Yangauthor=T.+Y.+Chengauthor=S.+M.+Huangauthor=C.+Y.+Suauthor=P.+W.+Yangauthor=J.+M.+Leeauthor=C.+K.+Chenauthor=M.+Hsiaoauthor=K.+T.+Huaauthor=M.+L.+Kuo&title=Cytosolic+Pkm2+Stabilizes+Mutant+Egfr+Protein+Expression+through+Regulating+Hsp90-Egfr+Association&doi=10.1038%2Fonc.2015.397"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Cytosolic PKM2 stabilizes mutant EGFR protein expression through regulating HSP90-EGFR association</span></div><div class="casAuthors">Yang, Y.-C.; Cheng, T.-Y.; Huang, S.-M.; Su, C.-Y.; Yang, P.-W.; Lee, J.-M.; Chen, C.-K.; Hsiao, M.; Hua, K.-T.; Kuo, M.-L.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">3387-3398</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Secondary mutation of epidermal growth factor receptor (EGFR) resulting in drug resistance is one of the most crit. issues in lung cancer therapy.  Several drugs are being developed to overcome EGFR tyrosine kinase inhibitor (TKI) resistance.  Here, we report that pyruvate kinase M2 (PKM2) stabilized mutant EGFR protein by direct interaction and sustained cell survival signaling in lung cancer cells.  PKM2 silencing resulted in markedly reduced mutant EGFR expression in TKI-sensitive or -resistant human lung cancer cells, and in inhibition of tumor growth in their xenografts, concomitant with downregulation of EGFR-related signaling.  Mechanistically, PKM2 directly interacted with mutant EGFR and heat-shock protein 90 (HSP90), and thus stabilized EGFR by maintaining its binding with HSP90 and co-chaperones.  Stabilization of EGFR relied on dimeric PKM2, and the protein half-life of mutant EGFR decreased when PKM2 was forced into its tetramer form.  Clin. levels of PKM2 pos. correlated with mutant EGFR expression and with patient outcome.  These results reveal a previously undescribed non-glycolysis function of PKM2 in the cytoplasm, which contribute to EGFR-dependent tumorigenesis and provide a novel strategy to overcome drug resistance to EGFR TKIs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpNRWNlV7j1NLVg90H21EOLACvtfcHk0lhCnIQ_9Q4IBA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslant7jK&md5=6ec476c07c325470d6d068b22e4e6724</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1038%2Fonc.2015.397&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fonc.2015.397%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DY.%2BC.%26aulast%3DCheng%26aufirst%3DT.%2BY.%26aulast%3DHuang%26aufirst%3DS.%2BM.%26aulast%3DSu%26aufirst%3DC.%2BY.%26aulast%3DYang%26aufirst%3DP.%2BW.%26aulast%3DLee%26aufirst%3DJ.%2BM.%26aulast%3DChen%26aufirst%3DC.%2BK.%26aulast%3DHsiao%26aufirst%3DM.%26aulast%3DHua%26aufirst%3DK.%2BT.%26aulast%3DKuo%26aufirst%3DM.%2BL.%26atitle%3DCytosolic%2520Pkm2%2520Stabilizes%2520Mutant%2520Egfr%2520Protein%2520Expression%2520through%2520Regulating%2520Hsp90-Egfr%2520Association%26jtitle%3DOncogene%26date%3D2016%26volume%3D35%26spage%3D3387%26epage%3D3398%26doi%3D10.1038%2Fonc.2015.397" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cui, S.</span></span> <span> </span><span class="NLM_article-title">Pkm2 Aggravates Palmitate-Induced Insulin Resistance in Hepg2 Cells Via Stat3 Pathway</span>. <i>Biochem. Biophys. Res. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>492</i></span>,  <span class="NLM_fpage">109</span>– <span class="NLM_lpage">115</span>, <span class="refDoi"> DOI: 10.1016/j.bbrc.2017.08.025</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=10.1016%2Fj.bbrc.2017.08.025" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=28802581" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtlWksL7L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=492&publication_year=2017&pages=109-115&author=L.+Chenauthor=Z.+Tangauthor=X.+Wangauthor=H.+Maauthor=D.+Shanauthor=S.+Cui&title=Pkm2+Aggravates+Palmitate-Induced+Insulin+Resistance+in+Hepg2+Cells+Via+Stat3+Pathway&doi=10.1016%2Fj.bbrc.2017.08.025"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">PKM2 aggravates palmitate-induced insulin resistance in HepG2 cells via STAT3 pathway</span></div><div class="casAuthors">Chen, Ling; Tang, Zhuqi; Wang, Xiaohua; Ma, Hong; Shan, Dandan; Cui, Shiwei</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical and Biophysical Research Communications</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">492</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">109-115</span>CODEN:
                <span class="NLM_cas:coden">BBRCA9</span>;
        ISSN:<span class="NLM_cas:issn">0006-291X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Studies have identified that PKM2 is related to the development of glucose intolerance and insulin resistance in rodents and humans.  However, the underlying mechanism remains largely unknown.  In the present study, we found that PKM2 expression was significantly elevated in insulin-resistant hepatic tissues and hepatocytes, implicating an assocn. between PKM2 expression and hepatic insulin resistance (IR).  In vitro study revealed that overexpression of PKM2 impaired the insulin signaling pathway by decreasing the phosphorylation of protein kinase B (Akt) and glycogen synthase kinase-3β (GSK3β).  Furthermore, PKM2 overexpression enhanced the effects of PA on the lipid accumulation, the expression of phosphoenolpyruvate carboxykinase (PEPCK) and glucose-6-phosphatase (G6Pase) and hepatic glucose uptake.  Intriguingly, PA-induced insulin resistance was suppressed following by the ablation of PKM2 in HepG2 cells.  We also found that STAT3 was significantly activated by PKM2 overexpression.  Moreover, we identified that PKM2 could interact directly with STAT3.  Taken together, these studies demonstrate that PKM2 may promote hepatic IR via STAT3 pathway and would provide a new insight into dissecting the mol. pathogenesis of hepatic insulin resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqatCr63ZliybVg90H21EOLACvtfcHk0lgJKk1lg3-X6A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtlWksL7L&md5=ceb157d27d8fb14e623f6a8f83fe34d9</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2Fj.bbrc.2017.08.025&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbrc.2017.08.025%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DL.%26aulast%3DTang%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DMa%26aufirst%3DH.%26aulast%3DShan%26aufirst%3DD.%26aulast%3DCui%26aufirst%3DS.%26atitle%3DPkm2%2520Aggravates%2520Palmitate-Induced%2520Insulin%2520Resistance%2520in%2520Hepg2%2520Cells%2520Via%2520Stat3%2520Pathway%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D2017%26volume%3D492%26spage%3D109%26epage%3D115%26doi%3D10.1016%2Fj.bbrc.2017.08.025" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xia, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aldape, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, Z.</span></span> <span> </span><span class="NLM_article-title">Nuclear Pkm2 Regulates Beta-Catenin Transactivation Upon Egfr Activation</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>480</i></span>,  <span class="NLM_fpage">118</span>– <span class="NLM_lpage">122</span>, <span class="refDoi"> DOI: 10.1038/nature10598</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=10.1038%2Fnature10598" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=22056988" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsVagtbvP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=480&publication_year=2011&pages=118-122&author=W.+Yangauthor=Y.+Xiaauthor=H.+Jiauthor=Y.+Zhengauthor=J.+Liangauthor=W.+Huangauthor=X.+Gaoauthor=K.+Aldapeauthor=Z.+Lu&title=Nuclear+Pkm2+Regulates+Beta-Catenin+Transactivation+Upon+Egfr+Activation&doi=10.1038%2Fnature10598"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Nuclear PKM2 regulates β-catenin transactivation upon EGFR activation</span></div><div class="casAuthors">Yang, Weiwei; Xia, Yan; Ji, Haitao; Zheng, Yanhua; Liang, Ji; Huang, Wenhua; Gao, Xiang; Aldape, Kenneth; Lu, Zhimin</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">480</span>
        (<span class="NLM_cas:issue">7375</span>),
    <span class="NLM_cas:pages">118-122</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The embryonic pyruvate kinase M2 (PKM2) isoform is highly expressed in human cancer.  In contrast to the established role of PKM2 in aerobic glycolysis or the Warburg effect, its non-metabolic functions remain elusive.  Here we demonstrate, in human cancer cells, that epidermal growth factor receptor (EGFR) activation induces translocation of PKM2, but not PKM1, into the nucleus, where K433 of PKM2 binds to c-Src-phosphorylated Y333 of β-catenin.  This interaction is required for both proteins to be recruited to the CCND1 promoter, leading to HDAC3 removal from the promoter, histone H3 acetylation and cyclin D1 expression.  PKM2-dependent β-catenin transactivation is instrumental in EGFR-promoted tumor cell proliferation and brain tumor development.  In addn., pos. correlations have been identified between c-Src activity, β-catenin Y333 phosphorylation and PKM2 nuclear accumulation in human glioblastoma specimens.  Furthermore, levels of β-catenin phosphorylation and nuclear PKM2 have been correlated with grades of glioma malignancy and prognosis.  These findings reveal that EGF induces β-catenin transactivation via a mechanism distinct from that induced by Wnt/Wingless and highlight the essential non-metabolic functions of PKM2 in EGFR-promoted β-catenin transactivation, cell proliferation and tumorigenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqIoPIB2i0od7Vg90H21EOLACvtfcHk0lgJKk1lg3-X6A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsVagtbvP&md5=25b4d94c581fe862d12dfe1fd213ed06</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1038%2Fnature10598&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature10598%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DW.%26aulast%3DXia%26aufirst%3DY.%26aulast%3DJi%26aufirst%3DH.%26aulast%3DZheng%26aufirst%3DY.%26aulast%3DLiang%26aufirst%3DJ.%26aulast%3DHuang%26aufirst%3DW.%26aulast%3DGao%26aufirst%3DX.%26aulast%3DAldape%26aufirst%3DK.%26aulast%3DLu%26aufirst%3DZ.%26atitle%3DNuclear%2520Pkm2%2520Regulates%2520Beta-Catenin%2520Transactivation%2520Upon%2520Egfr%2520Activation%26jtitle%3DNature%26date%3D2011%26volume%3D480%26spage%3D118%26epage%3D122%26doi%3D10.1038%2Fnature10598" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Anastasiou, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poulogiannis, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asara, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boxer, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellinger, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sasaki, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Locasale, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Auld, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vander Heiden, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantley, L. C.</span></span> <span> </span><span class="NLM_article-title">Inhibition of Pyruvate Kinase M2 by Reactive Oxygen Species Contributes to Cellular Antioxidant Responses</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>334</i></span>,  <span class="NLM_fpage">1278</span>– <span class="NLM_lpage">1283</span>, <span class="refDoi"> DOI: 10.1126/science.1211485</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=10.1126%2Fscience.1211485" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=22052977" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsFCms7jK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=334&publication_year=2011&pages=1278-1283&author=D.+Anastasiouauthor=G.+Poulogiannisauthor=J.+M.+Asaraauthor=M.+B.+Boxerauthor=J.+K.+Jiangauthor=M.+Shenauthor=G.+Bellingerauthor=A.+T.+Sasakiauthor=J.+W.+Locasaleauthor=D.+S.+Auldauthor=C.+J.+Thomasauthor=M.+G.+Vander+Heidenauthor=L.+C.+Cantley&title=Inhibition+of+Pyruvate+Kinase+M2+by+Reactive+Oxygen+Species+Contributes+to+Cellular+Antioxidant+Responses&doi=10.1126%2Fscience.1211485"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of Pyruvate Kinase M2 by Reactive Oxygen Species Contributes to Cellular Antioxidant Responses</span></div><div class="casAuthors">Anastasiou, Dimitrios; Poulogiannis, George; Asara, John M.; Boxer, Matthew B.; Jiang, Jian-kang; Shen, Min; Bellinger, Gary; Sasaki, Atsuo T.; Locasale, Jason W.; Auld, Douglas S.; Thomas, Craig J.; Vander Heiden, Matthew G.; Cantley, Lewis C.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">334</span>
        (<span class="NLM_cas:issue">6060</span>),
    <span class="NLM_cas:pages">1278-1283</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Control of intracellular reactive oxygen species (ROS) concns. is crit. for cancer cell survival.  We show that, in human lung cancer cells, acute increases in intracellular concns. of ROS caused inhibition of the glycolytic enzyme pyruvate kinase M2 (PKM2) through oxidn. of Cys358.  This inhibition of PKM2 is required to divert glucose flux into the pentose phosphate pathway and thereby generate sufficient reducing potential for detoxification of ROS.  Lung cancer cells in which endogenous PKM2 was replaced with the Cys358 to Ser358 oxidn.-resistant mutant exhibited increased sensitivity to oxidative stress and impaired tumor formation in a xenograft model.  Besides promoting metabolic changes required for proliferation, the regulatory properties of PKM2 may confer an addnl. advantage to cancer cells by allowing them to withstand oxidative stress.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9SgiOQxnKObVg90H21EOLACvtfcHk0lgJKk1lg3-X6A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsFCms7jK&md5=9c6ac78508f3c25cbad20d1f8a21b4dc</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1126%2Fscience.1211485&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1211485%26sid%3Dliteratum%253Aachs%26aulast%3DAnastasiou%26aufirst%3DD.%26aulast%3DPoulogiannis%26aufirst%3DG.%26aulast%3DAsara%26aufirst%3DJ.%2BM.%26aulast%3DBoxer%26aufirst%3DM.%2BB.%26aulast%3DJiang%26aufirst%3DJ.%2BK.%26aulast%3DShen%26aufirst%3DM.%26aulast%3DBellinger%26aufirst%3DG.%26aulast%3DSasaki%26aufirst%3DA.%2BT.%26aulast%3DLocasale%26aufirst%3DJ.%2BW.%26aulast%3DAuld%26aufirst%3DD.%2BS.%26aulast%3DThomas%26aufirst%3DC.%2BJ.%26aulast%3DVander%2BHeiden%26aufirst%3DM.%2BG.%26aulast%3DCantley%26aufirst%3DL.%2BC.%26atitle%3DInhibition%2520of%2520Pyruvate%2520Kinase%2520M2%2520by%2520Reactive%2520Oxygen%2520Species%2520Contributes%2520to%2520Cellular%2520Antioxidant%2520Responses%26jtitle%3DScience%26date%3D2011%26volume%3D334%26spage%3D1278%26epage%3D1283%26doi%3D10.1126%2Fscience.1211485" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Leproult, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barluenga, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moras, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wurtz, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winssinger, N.</span></span> <span> </span><span class="NLM_article-title">Cysteine Mapping in Conformationally Distinct Kinase Nucleotide Binding Sites: Application to the Design of Selective Covalent Inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">1347</span>– <span class="NLM_lpage">1355</span>, <span class="refDoi"> DOI: 10.1021/jm101396q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm101396q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhvFeis70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=1347-1355&author=E.+Leproultauthor=S.+Barluengaauthor=D.+Morasauthor=J.+M.+Wurtzauthor=N.+Winssinger&title=Cysteine+Mapping+in+Conformationally+Distinct+Kinase+Nucleotide+Binding+Sites%3A+Application+to+the+Design+of+Selective+Covalent+Inhibitors&doi=10.1021%2Fjm101396q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Cysteine mapping in conformationally distinct kinase nucleotide binding sites: application to the design of selective covalent inhibitors</span></div><div class="casAuthors">Leproult, Emeline; Barluenga, Sofia; Moras, Dino; Wurtz, Jean-Marie; Winssinger, Nicolas</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1347-1355</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Kinases have emerged as one of the most prolific therapeutic targets.  An important criterion in the therapeutic success of inhibitors targeting the nucleotide binding pocket of kinases is the inhibitor residence time.  Recently, covalent kinase inhibitors have attracted attention since they confer terminal inhibition and should thus be more effective than reversible inhibitors with transient inhibition.  The most robust approach to design irreversible inhibitors is to capitalize on the nucleophilicity of a cysteine thiol group present in the target protein.  Herein, the authors report a systematic anal. of cysteine residues present in the nucleotide binding site of kinases, which could be harnessed for irreversible inhibition, taking into consideration the different kinase conformations.  The authors demonstrate the predictive power of this anal. with the design and validation of an irreversible inhibitor of KIT/PDGFR kinases.  This is the first example of a covalent kinase inhibitor that combines a pharmacophore addressing the DFG-out conformation with a covalent trap.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrkzqt0gMrz3rVg90H21EOLACvtfcHk0ljx6HiXHA1Erg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhvFeis70%253D&md5=329ae5eaa6102b6dd34a0d1274631247</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fjm101396q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm101396q%26sid%3Dliteratum%253Aachs%26aulast%3DLeproult%26aufirst%3DE.%26aulast%3DBarluenga%26aufirst%3DS.%26aulast%3DMoras%26aufirst%3DD.%26aulast%3DWurtz%26aufirst%3DJ.%2BM.%26aulast%3DWinssinger%26aufirst%3DN.%26atitle%3DCysteine%2520Mapping%2520in%2520Conformationally%2520Distinct%2520Kinase%2520Nucleotide%2520Binding%2520Sites%253A%2520Application%2520to%2520the%2520Design%2520of%2520Selective%2520Covalent%2520Inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D1347%26epage%3D1355%26doi%3D10.1021%2Fjm101396q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sabnis, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buhrlage, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">Developing Irreversible Inhibitors of the Protein Kinase Cysteinome</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">146</span>– <span class="NLM_lpage">159</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2012.12.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=10.1016%2Fj.chembiol.2012.12.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=23438744" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjtFWitbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2013&pages=146-159&author=Q.+Liuauthor=Y.+Sabnisauthor=Z.+Zhaoauthor=T.+Zhangauthor=S.+J.+Buhrlageauthor=L.+H.+Jonesauthor=N.+S.+Gray&title=Developing+Irreversible+Inhibitors+of+the+Protein+Kinase+Cysteinome&doi=10.1016%2Fj.chembiol.2012.12.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Developing Irreversible Inhibitors of the Protein Kinase Cysteinome</span></div><div class="casAuthors">Liu, Qingsong; Sabnis, Yogesh; Zhao, Zheng; Zhang, Tinghu; Buhrlage, Sara J.; Jones, Lyn H.; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">146-159</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Protein kinases are a large family of approx. 530 highly conserved enzymes that transfer a γ-phosphate group from ATP to a variety of amino acid residues, such as tyrosine, serine, and threonine, that serves as a ubiquitous mechanism for cellular signal transduction.  The clin. success of a no. of kinase-directed drugs and the frequent observation of disease causing mutations in protein kinases suggest that a large no. of kinases may represent therapeutically relevant targets.  To date, the majority of clin. and preclin. kinase inhibitors are ATP competitive, noncovalent inhibitors that achieve selectivity through recognition of unique features of particular protein kinases.  Recently, there has been renewed interest in the development of irreversible inhibitors that form covalent bonds with cysteine or other nucleophilic residues in the ATP-binding pocket.  Irreversible kinase inhibitors have a no. of potential advantages including prolonged pharmacodynamics, suitability for rational design, high potency, and ability to validate pharmacol. specificity through mutation of the reactive cysteine residue.  Here, we review recent efforts to develop cysteine-targeted irreversible protein kinase inhibitors and discuss their modes of recognizing the ATP-binding pocket and their biol. activity profiles.  In addn., we provided an informatics assessment of the potential "kinase cysteinome" and discuss strategies for the efficient development of new covalent inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVGD7TX3oIeLVg90H21EOLACvtfcHk0ljx6HiXHA1Erg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjtFWitbs%253D&md5=cef80823123bae88362e816a0a638133</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2012.12.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2012.12.006%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DQ.%26aulast%3DSabnis%26aufirst%3DY.%26aulast%3DZhao%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DBuhrlage%26aufirst%3DS.%2BJ.%26aulast%3DJones%26aufirst%3DL.%2BH.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DDeveloping%2520Irreversible%2520Inhibitors%2520of%2520the%2520Protein%2520Kinase%2520Cysteinome%26jtitle%3DChem.%2520Biol.%26date%3D2013%26volume%3D20%26spage%3D146%26epage%3D159%26doi%3D10.1016%2Fj.chembiol.2012.12.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chaikuad, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koch, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laufer, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">The Cysteinome of Protein Kinases as a Target in Drug Development</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">4372</span>– <span class="NLM_lpage">4385</span>, <span class="refDoi"> DOI: 10.1002/anie.201707875</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=10.1002%2Fanie.201707875" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitFGgs7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2018&pages=4372-4385&author=A.+Chaikuadauthor=P.+Kochauthor=S.+A.+Lauferauthor=S.+Knapp&title=The+Cysteinome+of+Protein+Kinases+as+a+Target+in+Drug+Development&doi=10.1002%2Fanie.201707875"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">The Cysteinome of Protein Kinases as a Target in Drug Development</span></div><div class="casAuthors">Chaikuad, Apirat; Koch, Pierre; Laufer, Stefan A.; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">4372-4385</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  Drugs that function through covalent bond formation represent a considerable fraction of the authors' repository of effective medicines but safety concerns and the complexity of developing covalent inhibitors has rendered covalent targeting a less attractive strategy for rational drug design.  The recent approval of four covalent kinase inhibitors and the development of highly potent covalent kinase probes with exceptional selectivity has raised significant interest in industry and academic research and validated the concept of covalent kinase targeting for clin. applications.  The abundance of cysteines at diverse positions in and around the kinase active site suggests that a large fraction of kinases can be targeted by covalent inhibitors.  Herein, the authors review recent developments of this rapidly growing area in kinase drug development and highlight the unique opportunities and challenges of this strategy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphHGXsIr-dpbVg90H21EOLACvtfcHk0ljx6HiXHA1Erg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitFGgs7c%253D&md5=311ba5ab53dc4a5dc1dc92ddd4b6d4c2</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1002%2Fanie.201707875&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201707875%26sid%3Dliteratum%253Aachs%26aulast%3DChaikuad%26aufirst%3DA.%26aulast%3DKoch%26aufirst%3DP.%26aulast%3DLaufer%26aufirst%3DS.%2BA.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DThe%2520Cysteinome%2520of%2520Protein%2520Kinases%2520as%2520a%2520Target%2520in%2520Drug%2520Development%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2018%26volume%3D57%26spage%3D4372%26epage%3D4385%26doi%3D10.1002%2Fanie.201707875" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mediani, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gibellini, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertacchini, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frasson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bosco, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Accordi, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Basso, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonora, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calabro, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mattiolo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sgarbi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baracca, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pinton, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riva, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rampazzo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petrizza, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prodi, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milani, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luppi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potenza, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Pol, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cocco, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Capitani, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marmiroli, S.</span></span> <span> </span><span class="NLM_article-title">Reversal of the Glycolytic Phenotype of Primary Effusion Lymphoma Cells by Combined Targeting of Cellular Metabolism and Pi3k/Akt/ Mtor Signaling</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">5521</span>– <span class="NLM_lpage">5537</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.6315</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=10.18632%2Foncotarget.6315" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=26575168" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=1%3ACAS%3A280%3ADC%252BC28vitlymtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=5521-5537&author=L.+Medianiauthor=F.+Gibelliniauthor=J.+Bertacchiniauthor=C.+Frassonauthor=R.+Boscoauthor=B.+Accordiauthor=G.+Bassoauthor=M.+Bonoraauthor=M.+L.+Calabroauthor=A.+Mattioloauthor=G.+Sgarbiauthor=A.+Baraccaauthor=P.+Pintonauthor=G.+Rivaauthor=E.+Rampazzoauthor=L.+Petrizzaauthor=L.+Prodiauthor=D.+Milaniauthor=M.+Luppiauthor=L.+Potenzaauthor=A.+De+Polauthor=L.+Coccoauthor=S.+Capitaniauthor=S.+Marmiroli&title=Reversal+of+the+Glycolytic+Phenotype+of+Primary+Effusion+Lymphoma+Cells+by+Combined+Targeting+of+Cellular+Metabolism+and+Pi3k%2FAkt%2F+Mtor+Signaling&doi=10.18632%2Foncotarget.6315"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Reversal of the glycolytic phenotype of primary effusion lymphoma cells by combined targeting of cellular metabolism and PI3K/Akt/ mTOR signaling</span></div><div class="casAuthors">Mediani Laura; Gibellini Federica; Bertacchini Jessika; Bosco Raffaella; De Pol Anto; Marmiroli Sandra; Bertacchini Jessika; Milani Daniela; Capitani Silvano; Frasson Chiara; Accordi Benedetta; Basso Giuseppe; Bonora Massimo; Pinton Paolo; Calabro Maria Luisa; Mattiolo Adriana; Sgarbi Gianluca; Baracca Alessandra; Cocco Lucio; Riva Giovanni; Luppi Mario; Potenza Leonardo; Rampazzo Enrico; Petrizza Luca; Prodi Luca</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">5521-37</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">PEL is a B-cell non-Hodgkin lymphoma, occurring predominantly as a lymphomatous effusion in body cavities, characterized by aggressive clinical course, with no standard therapy.  Based on previous reports that PEL cells display a Warburg phenotype, we hypothesized that the highly hypoxic environment in which they grow in vivo makes them more reliant on glycolysis, and more vulnerable to drugs targeting this pathway.  We established here that indeed PEL cells in hypoxia are more sensitive to glycolysis inhibition.  Furthermore, since PI3K/Akt/mTOR has been proposed as a drug target in PEL, we ascertained that pathway-specific inhibitors, namely the dual PI3K and mTOR inhibitor, PF-04691502, and the Akt inhibitor, Akti 1/2, display improved cytotoxicity to PEL cells in hypoxic conditions.  Unexpectedly, we found that these drugs reduce lactate production/extracellular acidification rate, and, in combination with the glycolysis inhibitor 2-deoxyglucose (2-DG), they shift PEL cells metabolism from aerobic glycolysis towards oxidative respiration.  Moreover, the associations possess strong synergistic cytotoxicity towards PEL cells, and thus may reduce adverse reaction in vivo, while displaying very low toxicity to normal lymphocytes.  Finally, we showed that the association of 2-DG and PF-04691502 maintains its cytotoxic and proapoptotic effect also in PEL cells co-cultured with human primary mesothelial cells, a condition known to mimic the in vivo environment and to exert a protective and pro-survival action.  All together, these results provide a compelling rationale for the clinical development of new therapies for the treatment of PEL, based on combined targeting of glycolytic metabolism and constitutively activated signaling pathways.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQmRFIjeiXbtkg6wdEZmT2rfW6udTcc2eautmsxa4ddVrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC28vitlymtA%253D%253D&md5=6a05c8a9f616469c201dc614a07f000f</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.6315&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.6315%26sid%3Dliteratum%253Aachs%26aulast%3DMediani%26aufirst%3DL.%26aulast%3DGibellini%26aufirst%3DF.%26aulast%3DBertacchini%26aufirst%3DJ.%26aulast%3DFrasson%26aufirst%3DC.%26aulast%3DBosco%26aufirst%3DR.%26aulast%3DAccordi%26aufirst%3DB.%26aulast%3DBasso%26aufirst%3DG.%26aulast%3DBonora%26aufirst%3DM.%26aulast%3DCalabro%26aufirst%3DM.%2BL.%26aulast%3DMattiolo%26aufirst%3DA.%26aulast%3DSgarbi%26aufirst%3DG.%26aulast%3DBaracca%26aufirst%3DA.%26aulast%3DPinton%26aufirst%3DP.%26aulast%3DRiva%26aufirst%3DG.%26aulast%3DRampazzo%26aufirst%3DE.%26aulast%3DPetrizza%26aufirst%3DL.%26aulast%3DProdi%26aufirst%3DL.%26aulast%3DMilani%26aufirst%3DD.%26aulast%3DLuppi%26aufirst%3DM.%26aulast%3DPotenza%26aufirst%3DL.%26aulast%3DDe%2BPol%26aufirst%3DA.%26aulast%3DCocco%26aufirst%3DL.%26aulast%3DCapitani%26aufirst%3DS.%26aulast%3DMarmiroli%26aufirst%3DS.%26atitle%3DReversal%2520of%2520the%2520Glycolytic%2520Phenotype%2520of%2520Primary%2520Effusion%2520Lymphoma%2520Cells%2520by%2520Combined%2520Targeting%2520of%2520Cellular%2520Metabolism%2520and%2520Pi3k%252FAkt%252F%2520Mtor%2520Signaling%26jtitle%3DOncotarget%26date%3D2016%26volume%3D7%26spage%3D5521%26epage%3D5537%26doi%3D10.18632%2Foncotarget.6315" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Saunier, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Antonio, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Regazzetti, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Auzeil, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laprevote, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shay, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coumoul, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barouki, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benelli, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huc, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bortoli, S.</span></span> <span> </span><span class="NLM_article-title">Resveratrol Reverses the Warburg Effect by Targeting the Pyruvate Dehydrogenase Complex in Colon Cancer Cells</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">6945</span>, <span class="refDoi"> DOI: 10.1038/s41598-017-07006-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=10.1038%2Fs41598-017-07006-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=28761044" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=1%3ACAS%3A280%3ADC%252BC1cfivVOmtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&pages=6945&author=E.+Saunierauthor=S.+Antonioauthor=A.+Regazzettiauthor=N.+Auzeilauthor=O.+Laprevoteauthor=J.+W.+Shayauthor=X.+Coumoulauthor=R.+Baroukiauthor=C.+Benelliauthor=L.+Hucauthor=S.+Bortoli&title=Resveratrol+Reverses+the+Warburg+Effect+by+Targeting+the+Pyruvate+Dehydrogenase+Complex+in+Colon+Cancer+Cells&doi=10.1038%2Fs41598-017-07006-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Resveratrol reverses the Warburg effect by targeting the pyruvate dehydrogenase complex in colon cancer cells</span></div><div class="casAuthors">Saunier Elise; Antonio Samantha; Coumoul Xavier; Barouki Robert; Benelli Chantal; Bortoli Sylvie; Saunier Elise; Antonio Samantha; Coumoul Xavier; Barouki Robert; Benelli Chantal; Bortoli Sylvie; Regazzetti Anne; Auzeil Nicolas; Laprevote Olivier; Shay Jerry W; Huc Laurence; Huc Laurence</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">6945</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Resveratrol (RES), a polyphenol found in natural foods, displays anti-oxidant, anti-inflammatory and anti-proliferative properties potentially beneficial in cancers, in particular in the prevention of tumor growth.  However, the rapid metabolism of resveratrol strongly limits its bioavailability.  The molecular mechanisms sustaining the potential biological activity of low doses of resveratrol has not been extensively studied and, thus, needs better characterization.  Here, we show that resveratrol (10 μM, 48 hr) induces both a cell growth arrest and a metabolic reprogramming in colon cancer cells.  Resveratrol modifies the lipidomic profile, increases oxidative capacities and decreases glycolysis, in association with a decreased pentose phosphate activity and an increased ATP production.  Resveratrol targets the pyruvate dehydrogenase (PDH) complex, a key mitochondrial gatekeeper of energy metabolism, leading to an enhanced PDH activity.  Calcium chelation, as well as the blockade of the mitochondrial calcium uniport, prevents the resveratrol-induced augmentation in oxidative capacities and the increased PDH activity suggesting that calcium might play a role in the metabolic shift.  We further demonstrate that the inhibition of the CamKKB or the downstream AMPK pathway partly abolished the resveratrol-induced increase of glucose oxidation.  This suggests that resveratrol might improve the oxidative capacities of cancer cells through the CamKKB/AMPK pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRCLVgLaW5mRWHOrQ6qeX-mfW6udTcc2eautmsxa4ddVrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1cfivVOmtw%253D%253D&md5=5a87309168c53f0f7fdb43e300d32a78</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1038%2Fs41598-017-07006-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-017-07006-0%26sid%3Dliteratum%253Aachs%26aulast%3DSaunier%26aufirst%3DE.%26aulast%3DAntonio%26aufirst%3DS.%26aulast%3DRegazzetti%26aufirst%3DA.%26aulast%3DAuzeil%26aufirst%3DN.%26aulast%3DLaprevote%26aufirst%3DO.%26aulast%3DShay%26aufirst%3DJ.%2BW.%26aulast%3DCoumoul%26aufirst%3DX.%26aulast%3DBarouki%26aufirst%3DR.%26aulast%3DBenelli%26aufirst%3DC.%26aulast%3DHuc%26aufirst%3DL.%26aulast%3DBortoli%26aufirst%3DS.%26atitle%3DResveratrol%2520Reverses%2520the%2520Warburg%2520Effect%2520by%2520Targeting%2520the%2520Pyruvate%2520Dehydrogenase%2520Complex%2520in%2520Colon%2520Cancer%2520Cells%26jtitle%3DSci.%2520Rep.%26date%3D2017%26volume%3D7%26spage%3D6945%26doi%3D10.1038%2Fs41598-017-07006-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fadia, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dahlstrom, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parish, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Board, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blackburn, A. C.</span></span> <span> </span><span class="NLM_article-title">Reversal of the Glycolytic Phenotype by Dichloroacetate Inhibits Metastatic Breast Cancer Cell Growth in Vitro and in Vivo</span>. <i>Breast Cancer Res. Treat.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>120</i></span>,  <span class="NLM_fpage">253</span>– <span class="NLM_lpage">260</span>, <span class="refDoi"> DOI: 10.1007/s10549-009-0435-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=10.1007%2Fs10549-009-0435-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=19543830" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=1%3ACAS%3A528%3ADC%252BC3cXntlaruw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=120&publication_year=2010&pages=253-260&author=R.+C.+Sunauthor=M.+Fadiaauthor=J.+E.+Dahlstromauthor=C.+R.+Parishauthor=P.+G.+Boardauthor=A.+C.+Blackburn&title=Reversal+of+the+Glycolytic+Phenotype+by+Dichloroacetate+Inhibits+Metastatic+Breast+Cancer+Cell+Growth+in+Vitro+and+in+Vivo&doi=10.1007%2Fs10549-009-0435-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Reversal of the glycolytic phenotype by dichloroacetate inhibits metastatic breast cancer cell growth in vitro and in vivo</span></div><div class="casAuthors">Sun, Ramon C.; Fadia, Mitali; Dahlstrom, Jane E.; Parish, Christopher R.; Board, Philip G.; Blackburn, Anneke C.</div><div class="citationInfo"><span class="NLM_cas:title">Breast Cancer Research and Treatment</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">120</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">253-260</span>CODEN:
                <span class="NLM_cas:coden">BCTRD6</span>;
        ISSN:<span class="NLM_cas:issn">0167-6806</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">The glycolytic phenotype is a widespread phenomenon in solid cancer forms, including breast cancer.  Dichloroacetate (DCA) has recently been proposed as a novel and relatively non-toxic anti-cancer agent that can reverse the glycolytic phenotype in cancer cells through the inhibition of pyruvate dehydrogenase kinase.  We have examd. the effect of DCA against breast cancer cells, including in a highly metastatic in vivo model.  The growth of several breast cancer cell lines was found to be inhibited by DCA in vitro.  Further examn. of 13762 MAT rat mammary adenocarcinoma cells found that reversal of the glycolytic phenotype by DCA correlated with the inhibition of proliferation without any increase in cell death.  This was despite a small but significant increase in caspase 3/7 activity, which may sensitize cancer cells to other apoptotic triggers.  In vivo, DCA caused a 58% redn. in the no. of lung metastases obsd. macroscopically after injection of 13,762 MAT cells into the tail vein of rats (P = 0.0001, n ≥ 9 per group).  These results demonstrate that DCA has anti-proliferative properties in addn. to pro-apoptotic properties, and can be effective against highly metastatic disease in vivo, highlighting its potential for clin. use.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbbgVFTE8jU7Vg90H21EOLACvtfcHk0lhDLbvQvifKpg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXntlaruw%253D%253D&md5=f681498dd473ee35e2ced31f317109d0</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1007%2Fs10549-009-0435-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10549-009-0435-9%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DR.%2BC.%26aulast%3DFadia%26aufirst%3DM.%26aulast%3DDahlstrom%26aufirst%3DJ.%2BE.%26aulast%3DParish%26aufirst%3DC.%2BR.%26aulast%3DBoard%26aufirst%3DP.%2BG.%26aulast%3DBlackburn%26aufirst%3DA.%2BC.%26atitle%3DReversal%2520of%2520the%2520Glycolytic%2520Phenotype%2520by%2520Dichloroacetate%2520Inhibits%2520Metastatic%2520Breast%2520Cancer%2520Cell%2520Growth%2520in%2520Vitro%2520and%2520in%2520Vivo%26jtitle%3DBreast%2520Cancer%2520Res.%2520Treat.%26date%3D2010%26volume%3D120%26spage%3D253%26epage%3D260%26doi%3D10.1007%2Fs10549-009-0435-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Anastasiou, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Israelsen, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boxer, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tempel, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dimov, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jha, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mattaini, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Metallo, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fiske, B. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Courtney, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malstrom, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khan, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kung, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skoumbourdis, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Veith, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Southall, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walsh, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brimacombe, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leister, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lunt, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, Z. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yen, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kunii, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davidson, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christofk, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Austin, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inglese, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asara, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stephanopoulos, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salituro, F. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Auld, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, H. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantley, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vander Heiden, M. G.</span></span> <span> </span><span class="NLM_article-title">Pyruvate Kinase M2 Activators Promote Tetramer Formation and Suppress Tumorigenesis</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">839</span>– <span class="NLM_lpage">847</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1060</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=10.1038%2Fnchembio.1060" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=22922757" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1Ght7rM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2012&pages=839-847&author=D.+Anastasiouauthor=Y.+Yuauthor=W.+J.+Israelsenauthor=J.+K.+Jiangauthor=M.+B.+Boxerauthor=B.+S.+Hongauthor=W.+Tempelauthor=S.+Dimovauthor=M.+Shenauthor=A.+Jhaauthor=H.+Yangauthor=K.+R.+Mattainiauthor=C.+M.+Metalloauthor=B.+P.+Fiskeauthor=K.+D.+Courtneyauthor=S.+Malstromauthor=T.+M.+Khanauthor=C.+Kungauthor=A.+P.+Skoumbourdisauthor=H.+Veithauthor=N.+Southallauthor=M.+J.+Walshauthor=K.+R.+Brimacombeauthor=W.+Leisterauthor=S.+Y.+Luntauthor=Z.+R.+Johnsonauthor=K.+E.+Yenauthor=K.+Kuniiauthor=S.+M.+Davidsonauthor=H.+R.+Christofkauthor=C.+P.+Austinauthor=J.+Ingleseauthor=M.+H.+Harrisauthor=J.+M.+Asaraauthor=G.+Stephanopoulosauthor=F.+G.+Salituroauthor=S.+Jinauthor=L.+Dangauthor=D.+S.+Auldauthor=H.+W.+Parkauthor=L.+C.+Cantleyauthor=C.+J.+Thomasauthor=M.+G.+Vander+Heiden&title=Pyruvate+Kinase+M2+Activators+Promote+Tetramer+Formation+and+Suppress+Tumorigenesis&doi=10.1038%2Fnchembio.1060"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Pyruvate kinase M2 activators promote tetramer formation and suppress tumorigenesis</span></div><div class="casAuthors">Anastasiou, Dimitrios; Yu, Yimin; Israelsen, William J.; Jiang, Jian-Kang; Boxer, Matthew B.; Hong, Bum Soo; Tempel, Wolfram; Dimov, Svetoslav; Shen, Min; Jha, Abhishek; Yang, Hua; Mattaini, Katherine R.; Metallo, Christian M.; Fiske, Brian P.; Courtney, Kevin D.; Malstrom, Scott; Khan, Tahsin M.; Kung, Charles; Skoumbourdis, Amanda P.; Veith, Henrike; Southall, Noel; Walsh, Martin J.; Brimacombe, Kyle R.; Leister, William; Lunt, Sophia Y.; Johnson, Zachary R.; Yen, Katharine E.; Kunii, Kaiko; Davidson, Shawn M.; Christofk, Heather R.; Austin, Christopher P.; Inglese, James; Harris, Marian H.; Asara, John M.; Stephanopoulos, Gregory; Salituro, Francesco G.; Jin, Shengfang; Dang, Lenny; Auld, Douglas S.; Park, Hee-Won; Cantley, Lewis C.; Thomas, Craig J.; Vander Heiden, Matthew G.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">839-847</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Cancer cells engage in a metabolic program to enhance biosynthesis and support cell proliferation.  The regulatory properties of pyruvate kinase M2 (PKM2) influence altered glucose metab. in cancer.  The interaction of PKM2 with phosphotyrosine-contg. proteins inhibits enzyme activity and increases the availability of glycolytic metabolites to support cell proliferation.  This suggests that high pyruvate kinase activity may suppress tumor growth.  We show that expression of PKM1, the pyruvate kinase isoform with high constitutive activity, or exposure to published small-mol. PKM2 activators inhibits the growth of xenograft tumors.  Structural studies reveal that small-mol. activators bind PKM2 at the subunit interaction interface, a site that is distinct from that of the endogenous activator fructose-1,6-bisphosphate (FBP).  However, unlike FBP, binding of activators to PKM2 promotes a constitutively active enzyme state that is resistant to inhibition by tyrosine-phosphorylated proteins.  These data support the notion that small-mol. activation of PKM2 can interfere with anabolic metab.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMDIr6LPTxPbVg90H21EOLACvtfcHk0lhDLbvQvifKpg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1Ght7rM&md5=20ae4953c6afd41458cf317933fd5d9d</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1060&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1060%26sid%3Dliteratum%253Aachs%26aulast%3DAnastasiou%26aufirst%3DD.%26aulast%3DYu%26aufirst%3DY.%26aulast%3DIsraelsen%26aufirst%3DW.%2BJ.%26aulast%3DJiang%26aufirst%3DJ.%2BK.%26aulast%3DBoxer%26aufirst%3DM.%2BB.%26aulast%3DHong%26aufirst%3DB.%2BS.%26aulast%3DTempel%26aufirst%3DW.%26aulast%3DDimov%26aufirst%3DS.%26aulast%3DShen%26aufirst%3DM.%26aulast%3DJha%26aufirst%3DA.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DMattaini%26aufirst%3DK.%2BR.%26aulast%3DMetallo%26aufirst%3DC.%2BM.%26aulast%3DFiske%26aufirst%3DB.%2BP.%26aulast%3DCourtney%26aufirst%3DK.%2BD.%26aulast%3DMalstrom%26aufirst%3DS.%26aulast%3DKhan%26aufirst%3DT.%2BM.%26aulast%3DKung%26aufirst%3DC.%26aulast%3DSkoumbourdis%26aufirst%3DA.%2BP.%26aulast%3DVeith%26aufirst%3DH.%26aulast%3DSouthall%26aufirst%3DN.%26aulast%3DWalsh%26aufirst%3DM.%2BJ.%26aulast%3DBrimacombe%26aufirst%3DK.%2BR.%26aulast%3DLeister%26aufirst%3DW.%26aulast%3DLunt%26aufirst%3DS.%2BY.%26aulast%3DJohnson%26aufirst%3DZ.%2BR.%26aulast%3DYen%26aufirst%3DK.%2BE.%26aulast%3DKunii%26aufirst%3DK.%26aulast%3DDavidson%26aufirst%3DS.%2BM.%26aulast%3DChristofk%26aufirst%3DH.%2BR.%26aulast%3DAustin%26aufirst%3DC.%2BP.%26aulast%3DInglese%26aufirst%3DJ.%26aulast%3DHarris%26aufirst%3DM.%2BH.%26aulast%3DAsara%26aufirst%3DJ.%2BM.%26aulast%3DStephanopoulos%26aufirst%3DG.%26aulast%3DSalituro%26aufirst%3DF.%2BG.%26aulast%3DJin%26aufirst%3DS.%26aulast%3DDang%26aufirst%3DL.%26aulast%3DAuld%26aufirst%3DD.%2BS.%26aulast%3DPark%26aufirst%3DH.%2BW.%26aulast%3DCantley%26aufirst%3DL.%2BC.%26aulast%3DThomas%26aufirst%3DC.%2BJ.%26aulast%3DVander%2BHeiden%26aufirst%3DM.%2BG.%26atitle%3DPyruvate%2520Kinase%2520M2%2520Activators%2520Promote%2520Tetramer%2520Formation%2520and%2520Suppress%2520Tumorigenesis%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2012%26volume%3D8%26spage%3D839%26epage%3D847%26doi%3D10.1038%2Fnchembio.1060" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cortes-Cros, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hemmerlin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferretti, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gounarides, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haberkorn, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chene, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sellers, W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hofmann, F.</span></span> <span> </span><span class="NLM_article-title">M2 Isoform of Pyruvate Kinase Is Dispensable for Tumor Maintenance and Growth</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>110</i></span>,  <span class="NLM_fpage">489</span>– <span class="NLM_lpage">494</span>, <span class="refDoi"> DOI: 10.1073/pnas.1212780110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=10.1073%2Fpnas.1212780110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=23267074" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFOhtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2013&pages=489-494&author=M.+Cortes-Crosauthor=C.+Hemmerlinauthor=S.+Ferrettiauthor=J.+Zhangauthor=J.+S.+Gounaridesauthor=H.+Yinauthor=A.+Mullerauthor=A.+Haberkornauthor=P.+Cheneauthor=W.+R.+Sellersauthor=F.+Hofmann&title=M2+Isoform+of+Pyruvate+Kinase+Is+Dispensable+for+Tumor+Maintenance+and+Growth&doi=10.1073%2Fpnas.1212780110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">M2 isoform of pyruvate kinase is dispensable for tumor maintenance and growth</span></div><div class="casAuthors">Cortes-Cros, Marta; Hemmerlin, Christelle; Ferretti, Stephane; Zhang, Juan; Gounarides, John S.; Yin, Hong; Muller, Alban; Haberkorn, Anne; Chene, Patrick; Sellers, William R.; Hofmann, Francesco</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">489-494, S489/1-S489/4</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Many cancer cells have increased rates of aerobic glycolysis, a phenomenon termed the Warburg effect.  In addn., in tumors there is a predominance of expression of the M2 isoform of pyruvate kinase (PKM2).  PKM2 expression was previously shown to be necessary for aerobic glycolysis and to provide a growth advantage to tumors.  Here, the authors report that knockdown of PKM2 in tumor cells led to a decrease in the levels of pyruvate kinase activity and an increase in the pyruvate kinase substrate, phosphoenolpyruvate.  However, lactate prodn. from glucose, although reduced, was not fully inhibited.  Furthermore, the authors are unique in reporting increased serine and glycine biosynthesis from both glucose and glutamine following PKM2 knockdown.  Although PKM2 knockdown resulted in modest impairment of proliferation in vitro, the in vivo growth of established xenograft tumors was unaffected by PKM2 absence.  Thus, these findings indicate that PKM2 is dispensable for tumor maintenance and growth in vivo, suggesting that other metabolic pathways bypass its function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXJi5-fdQ7jLVg90H21EOLACvtfcHk0lgyphJ87y8KBw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFOhtLs%253D&md5=5284f54fbaa4dad9b067ee1b0394a7e1</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1212780110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1212780110%26sid%3Dliteratum%253Aachs%26aulast%3DCortes-Cros%26aufirst%3DM.%26aulast%3DHemmerlin%26aufirst%3DC.%26aulast%3DFerretti%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DGounarides%26aufirst%3DJ.%2BS.%26aulast%3DYin%26aufirst%3DH.%26aulast%3DMuller%26aufirst%3DA.%26aulast%3DHaberkorn%26aufirst%3DA.%26aulast%3DChene%26aufirst%3DP.%26aulast%3DSellers%26aufirst%3DW.%2BR.%26aulast%3DHofmann%26aufirst%3DF.%26atitle%3DM2%2520Isoform%2520of%2520Pyruvate%2520Kinase%2520Is%2520Dispensable%2520for%2520Tumor%2520Maintenance%2520and%2520Growth%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2013%26volume%3D110%26spage%3D489%26epage%3D494%26doi%3D10.1073%2Fpnas.1212780110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Israelsen, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dayton, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davidson, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fiske, B. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hosios, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellinger, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sasaki, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horner, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burga, L. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jurczak, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DePinho, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clish, C. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacks, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kibbey, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wulf, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Vizio, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mills, G. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantley, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vander Heiden, M. G.</span></span> <span> </span><span class="NLM_article-title">Pkm2 Isoform-Specific Deletion Reveals a Differential Requirement for Pyruvate Kinase in Tumor Cells</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>155</i></span>,  <span class="NLM_fpage">397</span>– <span class="NLM_lpage">409</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2013.09.025</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=10.1016%2Fj.cell.2013.09.025" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=24120138" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1Sktr%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=155&publication_year=2013&pages=397-409&author=W.+J.+Israelsenauthor=T.+L.+Daytonauthor=S.+M.+Davidsonauthor=B.+P.+Fiskeauthor=A.+M.+Hosiosauthor=G.+Bellingerauthor=J.+Liauthor=Y.+Yuauthor=M.+Sasakiauthor=J.+W.+Hornerauthor=L.+N.+Burgaauthor=J.+Xieauthor=M.+J.+Jurczakauthor=R.+A.+DePinhoauthor=C.+B.+Clishauthor=T.+Jacksauthor=R.+G.+Kibbeyauthor=G.+M.+Wulfauthor=D.+Di+Vizioauthor=G.+B.+Millsauthor=L.+C.+Cantleyauthor=M.+G.+Vander+Heiden&title=Pkm2+Isoform-Specific+Deletion+Reveals+a+Differential+Requirement+for+Pyruvate+Kinase+in+Tumor+Cells&doi=10.1016%2Fj.cell.2013.09.025"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">PKM2 Isoform-Specific Deletion Reveals a Differential Requirement for Pyruvate Kinase in Tumor Cells</span></div><div class="casAuthors">Israelsen, William J.; Dayton, Talya L.; Davidson, Shawn M.; Fiske, Brian P.; Hosios, Aaron M.; Bellinger, Gary; Li, Jie; Yu, Yimin; Sasaki, Mika; Horner, James W.; Burga, Laura N.; Xie, Jianxin; Jurczak, Michael J.; DePinho, Ronald A.; Clish, Clary B.; Jacks, Tyler; Kibbey, Richard G.; Wulf, Gerburg M.; Di Vizio, Dolores; Mills, Gordon B.; Cantley, Lewis C.; Vander Heiden, Matthew G.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">155</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">397-409</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The pyruvate kinase M2 isoform (PKM2) is expressed in cancer and plays a role in regulating anabolic metab.  To det. whether PKM2 is required for tumor formation or growth, we generated mice with a conditional allele that abolishes PKM2 expression without disrupting PKM1 expression.  PKM2 deletion accelerated mammary tumor formation in a Brca1-loss-driven model of breast cancer.  PKM2 null tumors displayed heterogeneous PKM1 expression, with PKM1 found in nonproliferating tumor cells and no detectable pyruvate kinase expression in proliferating cells.  This suggests that PKM2 is not necessary for tumor cell proliferation and implies that the inactive state of PKM2 is assocd. with the proliferating cell population within tumors, whereas nonproliferating tumor cells require active pyruvate kinase.  Consistent with these findings, variable PKM2 expression and heterozygous PKM2 mutations are found in human tumors.  These data suggest that regulation of PKM2 activity supports the different metabolic requirements of proliferating and nonproliferating tumor cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8md57TLpos7Vg90H21EOLACvtfcHk0lgyphJ87y8KBw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1Sktr%252FI&md5=b2647ac4c5fa6a1fa03664adb60288ee</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2013.09.025&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2013.09.025%26sid%3Dliteratum%253Aachs%26aulast%3DIsraelsen%26aufirst%3DW.%2BJ.%26aulast%3DDayton%26aufirst%3DT.%2BL.%26aulast%3DDavidson%26aufirst%3DS.%2BM.%26aulast%3DFiske%26aufirst%3DB.%2BP.%26aulast%3DHosios%26aufirst%3DA.%2BM.%26aulast%3DBellinger%26aufirst%3DG.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DYu%26aufirst%3DY.%26aulast%3DSasaki%26aufirst%3DM.%26aulast%3DHorner%26aufirst%3DJ.%2BW.%26aulast%3DBurga%26aufirst%3DL.%2BN.%26aulast%3DXie%26aufirst%3DJ.%26aulast%3DJurczak%26aufirst%3DM.%2BJ.%26aulast%3DDePinho%26aufirst%3DR.%2BA.%26aulast%3DClish%26aufirst%3DC.%2BB.%26aulast%3DJacks%26aufirst%3DT.%26aulast%3DKibbey%26aufirst%3DR.%2BG.%26aulast%3DWulf%26aufirst%3DG.%2BM.%26aulast%3DDi%2BVizio%26aufirst%3DD.%26aulast%3DMills%26aufirst%3DG.%2BB.%26aulast%3DCantley%26aufirst%3DL.%2BC.%26aulast%3DVander%2BHeiden%26aufirst%3DM.%2BG.%26atitle%3DPkm2%2520Isoform-Specific%2520Deletion%2520Reveals%2520a%2520Differential%2520Requirement%2520for%2520Pyruvate%2520Kinase%2520in%2520Tumor%2520Cells%26jtitle%3DCell%26date%3D2013%26volume%3D155%26spage%3D397%26epage%3D409%26doi%3D10.1016%2Fj.cell.2013.09.025" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hosios, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fiske, B. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gui, D. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vander Heiden, M. G.</span></span> <span> </span><span class="NLM_article-title">Lack of Evidence for Pkm2 Protein Kinase Activity</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">850</span>– <span class="NLM_lpage">857</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2015.07.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=10.1016%2Fj.molcel.2015.07.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=26300261" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtlOmtbrE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2015&pages=850-857&author=A.+M.+Hosiosauthor=B.+P.+Fiskeauthor=D.+Y.+Guiauthor=M.+G.+Vander+Heiden&title=Lack+of+Evidence+for+Pkm2+Protein+Kinase+Activity&doi=10.1016%2Fj.molcel.2015.07.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Lack of Evidence for PKM2 Protein Kinase Activity</span></div><div class="casAuthors">Hosios, Aaron M.; Fiske, Brian P.; Gui, Dan Y.; Vander Heiden, Matthew G.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">850-857</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">The role of pyruvate kinase M2 (PKM2) in cell proliferation is controversial.  A unique function of PKM2 proposed to be important for the proliferation of some cancer cells involves the direct activity of this enzyme as a protein kinase; however, a detailed biochem. characterization of this activity is lacking.  Using [32P]-phosphoenolpyruvate (PEP) we examine the direct substrates of PKM2 using recombinant enzyme and in vitro systems where PKM2 is genetically deleted.  Labeling of some protein species from [32P]-PEP can be obsd.; however, most were dependent on the presence of ADP, and none were dependent on the presence of PKM2.  In addn., we also failed to observe PKM2-dependent transfer of phosphate from ATP directly to protein.  These findings argue against a role for PKM2 as a protein kinase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqdvpnulo73T7Vg90H21EOLACvtfcHk0lgyphJ87y8KBw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtlOmtbrE&md5=2b34c79757ed73f0c9cf4dd426960818</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2015.07.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2015.07.013%26sid%3Dliteratum%253Aachs%26aulast%3DHosios%26aufirst%3DA.%2BM.%26aulast%3DFiske%26aufirst%3DB.%2BP.%26aulast%3DGui%26aufirst%3DD.%2BY.%26aulast%3DVander%2BHeiden%26aufirst%3DM.%2BG.%26atitle%3DLack%2520of%2520Evidence%2520for%2520Pkm2%2520Protein%2520Kinase%2520Activity%26jtitle%3DMol.%2520Cell%26date%3D2015%26volume%3D59%26spage%3D850%26epage%3D857%26doi%3D10.1016%2Fj.molcel.2015.07.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bloxham, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilton, D. C.</span></span> <span> </span><span class="NLM_article-title">A Detailed Investigation of the Properties of Lactate Dehydrogenase in Which the ’Essential’ Cysteine-165 Is Modified by Thioalkylation</span>. <i>Biochem. J.</i> <span class="NLM_year" style="font-weight: bold;">1979</span>,  <span class="NLM_volume"><i>177</i></span>,  <span class="NLM_fpage">769</span>– <span class="NLM_lpage">780</span>, <span class="refDoi"> DOI: 10.1042/bj1770769a</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=10.1042%2Fbj1770769a" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=36072" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=1%3ACAS%3A528%3ADyaE1MXkvVKns74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=177&publication_year=1979&pages=769-780&author=D.+P.+Bloxhamauthor=R.+P.+Sharmaauthor=D.+C.+Wilton&title=A+Detailed+Investigation+of+the+Properties+of+Lactate+Dehydrogenase+in+Which+the+%E2%80%99Essential%E2%80%99+Cysteine-165+Is+Modified+by+Thioalkylation&doi=10.1042%2Fbj1770769a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">A detailed investigation of the properties of lactate dehydrogenase in which the 'essential' cysteine-165 is modified by thioalkylation</span></div><div class="casAuthors">Bloxham, David P.; Sharma, Ram P.; Wilton, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Journal</span>
        (<span class="NLM_cas:date">1979</span>),
    <span class="NLM_cas:volume">177</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">769-80</span>CODEN:
                <span class="NLM_cas:coden">BIJOAK</span>;
        ISSN:<span class="NLM_cas:issn">0264-6021</span>.
    </div><div class="casAbstract">Cysteine-165 of pig heart lactate dehydrogenase (I) was modified by reaction with Me methanethiosulfonate, and although the affinity of I for carbonyl- or hydroxy-contg. substrates was decreased, thiomethylated I possessed full catalytic activity.  The nicotinamide nucleotide-binding capacity was unaffected, as was I-mediated activation of NAD.  The affinity of oxamate for the I-NADH complex was decreased 100-fold; this constituted a net increase of 10.4 kJ/mol in the activation energy for binding.  Thiomethylated I formed an abortive adduct between NAD and fluoropyruvate, but the equil. const. was too low to demonstrate this complex at reasonable pyruvate concns.  A conformational change in structure was revealed by the decreased thermostability.  Thioethylation resulted in an increase in Km for pyruvate (13.5 mM) and an 85% decrease in max. catalytic activity.  The implications of these findings for the catalytic mechanism of I are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreoDh_0_Uqi7Vg90H21EOLACvtfcHk0lgeVTQihg3oMg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE1MXkvVKns74%253D&md5=983d8cc2ebc23c1370b0c41055d3db64</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1042%2Fbj1770769a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252Fbj1770769a%26sid%3Dliteratum%253Aachs%26aulast%3DBloxham%26aufirst%3DD.%2BP.%26aulast%3DSharma%26aufirst%3DR.%2BP.%26aulast%3DWilton%26aufirst%3DD.%2BC.%26atitle%3DA%2520Detailed%2520Investigation%2520of%2520the%2520Properties%2520of%2520Lactate%2520Dehydrogenase%2520in%2520Which%2520the%2520%25E2%2580%2599Essential%25E2%2580%2599%2520Cysteine-165%2520Is%2520Modified%2520by%2520Thioalkylation%26jtitle%3DBiochem.%2520J.%26date%3D1979%26volume%3D177%26spage%3D769%26epage%3D780%26doi%3D10.1042%2Fbj1770769a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xie, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kats, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burgess, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhargava, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Signoretti, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Billiard, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duffy, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grant, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lorkiewicz, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schatzman, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bousamra, M.,  2nd</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lane, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Higashi, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, T. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pandolfi, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sukhatme, V. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seth, P.</span></span> <span> </span><span class="NLM_article-title">Targeting Lactate Dehydrogenase--a Inhibits Tumorigenesis and Tumor Progression in Mouse Models of Lung Cancer and Impacts Tumor-Initiating Cells</span>. <i>Cell Metab.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">795</span>– <span class="NLM_lpage">809</span>, <span class="refDoi"> DOI: 10.1016/j.cmet.2014.03.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=10.1016%2Fj.cmet.2014.03.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=24726384" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmtVWrtLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2014&pages=795-809&author=H.+Xieauthor=J.+Hanaiauthor=J.+G.+Renauthor=L.+Katsauthor=K.+Burgessauthor=P.+Bhargavaauthor=S.+Signorettiauthor=J.+Billiardauthor=K.+J.+Duffyauthor=A.+Grantauthor=X.+Wangauthor=P.+K.+Lorkiewiczauthor=S.+Schatzmanauthor=M.+Bousamraauthor=A.+N.+Laneauthor=R.+M.+Higashiauthor=T.+W.+Fanauthor=P.+P.+Pandolfiauthor=V.+P.+Sukhatmeauthor=P.+Seth&title=Targeting+Lactate+Dehydrogenase%2D%2Da+Inhibits+Tumorigenesis+and+Tumor+Progression+in+Mouse+Models+of+Lung+Cancer+and+Impacts+Tumor-Initiating+Cells&doi=10.1016%2Fj.cmet.2014.03.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting Lactate Dehydrogenase-A Inhibits Tumorigenesis and Tumor Progression in Mouse Models of Lung Cancer and Impacts Tumor-Initiating Cells</span></div><div class="casAuthors">Xie, Han; Hanai, Jun-ichi; Ren, Jian-Guo; Kats, Lev; Burgess, Kerri; Bhargava, Parul; Signoretti, Sabina; Billiard, Julia; Duffy, Kevin J.; Grant, Aaron; Wang, Xiaoen; Lorkiewicz, Pawel K.; Schatzman, Sabrina; Bousamra, Michael; Lane, Andrew N.; Higashi, Richard M.; Fan, Teresa W. M.; Pandolfi, Pier Paolo; Sukhatme, Vikas P.; Seth, Pankaj</div><div class="citationInfo"><span class="NLM_cas:title">Cell Metabolism</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">795-809</span>CODEN:
                <span class="NLM_cas:coden">CMEEB5</span>;
        ISSN:<span class="NLM_cas:issn">1550-4131</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">The lactate dehydrogenase-A (LDH-A) enzyme catalyzes the interconversion of pyruvate and lactate, is upregulated in human cancers, and is assocd. with aggressive tumor outcomes.  Here we use an inducible murine model and demonstrate that inactivation of LDH-A in mouse models of NSCLC driven by oncogenic K-RAS or EGFR leads to decreased tumorigenesis and disease regression in established tumors.  We also show that abrogation of LDH-A results in reprogramming of pyruvate metab., with decreased lactic fermn. in vitro, in vivo, and ex vivo.  This was accompanied by reactivation of mitochondrial function in vitro, but not in vivo or ex vivo.  Finally, using a specific small mol. LDH-A inhibitor, we demonstrated that LDH-A is essential for cancer-initiating cell survival and proliferation.  Thus, LDH-A can be a viable therapeutic target for NSCLC, including cancer stem cell-dependent drug-resistant tumors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpNiWJNOeWwmLVg90H21EOLACvtfcHk0lgeVTQihg3oMg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmtVWrtLY%253D&md5=d80624d60895f5a6c01106841ebd1af9</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1016%2Fj.cmet.2014.03.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cmet.2014.03.003%26sid%3Dliteratum%253Aachs%26aulast%3DXie%26aufirst%3DH.%26aulast%3DHanai%26aufirst%3DJ.%26aulast%3DRen%26aufirst%3DJ.%2BG.%26aulast%3DKats%26aufirst%3DL.%26aulast%3DBurgess%26aufirst%3DK.%26aulast%3DBhargava%26aufirst%3DP.%26aulast%3DSignoretti%26aufirst%3DS.%26aulast%3DBilliard%26aufirst%3DJ.%26aulast%3DDuffy%26aufirst%3DK.%2BJ.%26aulast%3DGrant%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DLorkiewicz%26aufirst%3DP.%2BK.%26aulast%3DSchatzman%26aufirst%3DS.%26aulast%3DBousamra%26aufirst%3DM.%26aulast%3DLane%26aufirst%3DA.%2BN.%26aulast%3DHigashi%26aufirst%3DR.%2BM.%26aulast%3DFan%26aufirst%3DT.%2BW.%26aulast%3DPandolfi%26aufirst%3DP.%2BP.%26aulast%3DSukhatme%26aufirst%3DV.%2BP.%26aulast%3DSeth%26aufirst%3DP.%26atitle%3DTargeting%2520Lactate%2520Dehydrogenase--a%2520Inhibits%2520Tumorigenesis%2520and%2520Tumor%2520Progression%2520in%2520Mouse%2520Models%2520of%2520Lung%2520Cancer%2520and%2520Impacts%2520Tumor-Initiating%2520Cells%26jtitle%3DCell%2520Metab.%26date%3D2014%26volume%3D19%26spage%3D795%26epage%3D809%26doi%3D10.1016%2Fj.cmet.2014.03.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yu, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chou, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tu, H. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ho, Y. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khoo, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, G. D.</span></span> <span> </span><span class="NLM_article-title">Antibody-Assisted Target Identification Reveals Afatinib, an Egfr Covalent Inhibitor, Down-Regulating Ribonucleotide Reductase</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">21512</span>– <span class="NLM_lpage">21529</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.25177</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=10.18632%2Foncotarget.25177" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=29765556" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=1%3ACAS%3A280%3ADC%252BC1Mfjs1OksQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=21512-21529&author=C.+H.+Yuauthor=C.+C.+Chouauthor=H.+F.+Tuauthor=W.+C.+Huangauthor=Y.+Y.+Hoauthor=K.+H.+Khooauthor=M.+S.+Leeauthor=G.+D.+Chang&title=Antibody-Assisted+Target+Identification+Reveals+Afatinib%2C+an+Egfr+Covalent+Inhibitor%2C+Down-Regulating+Ribonucleotide+Reductase&doi=10.18632%2Foncotarget.25177"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Antibody-assisted target identification reveals afatinib, an EGFR covalent inhibitor, down-regulating ribonucleotide reductase</span></div><div class="casAuthors">Yu Cheng-Han; Huang Wei-Chieh; Ho Ya-Yeh; Khoo Kay-Hooi; Chang Geen-Dong; Chou Chi-Chi; Khoo Kay-Hooi; Tu Hsin-Fang; Lee Ming-Shyue</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">30</span>),
    <span class="NLM_cas:pages">21512-21529</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Afatinib, used for the first-line treatment of non-small-cell lung carcinoma (NSCLC) patients with distinct epidermal growth factor receptor (EGFR) mutations, inactivates EGFR by mimicking ATP structure and forming a covalent adduct with EGFR.  We developed a method to unravel potential targets of afatinib in NSCLC cells through immunoprecipitation of afatinib-labeling proteins with anti-afatinib antiserum and mass spectrometry analysis.  Ribonucleotide reductase (RNR) is one of target proteins of afatinib revealed by this method.  Treatment of afatinib at 10-100 nM potently inhibited intracellular RNR activity in an in vitro assay using permeabilized PC-9 cells (formerly known as PC-14).  PC-9 cells treated with 10 μM afatinib displayed elevated markers of DNA damage.  Long-term treatment of therapeutic concentrations of afatinib in PC-9 cells caused significant decrease in protein levels of RNR subunit M2 at 1-10 nM and RNR subunit M1 at 100 nM.  EGFR-null Chinese hamster ovary (CHO) cells treated with afatinib also showed similar effects.  Afatinib repressed the upregulation of RNR subunit M2 induced by gemcitabine.  Covalent modification with afatinib resulting in inhibition and protein downregulation of RNR underscores the therapeutic and off-target effects of afatinib.  Afatinib may serve as a lead compound of chemotherapeutic drugs targeting RNR.  This method can be widely used in the identification of potential targets of other covalent drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTezKTz513V2Uh5eUSa2oe_fW6udTcc2eZt0L0-VNOzS7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1Mfjs1OksQ%253D%253D&md5=0eda3616bbc935ae81c8b5602322af1e</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.25177&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.25177%26sid%3Dliteratum%253Aachs%26aulast%3DYu%26aufirst%3DC.%2BH.%26aulast%3DChou%26aufirst%3DC.%2BC.%26aulast%3DTu%26aufirst%3DH.%2BF.%26aulast%3DHuang%26aufirst%3DW.%2BC.%26aulast%3DHo%26aufirst%3DY.%2BY.%26aulast%3DKhoo%26aufirst%3DK.%2BH.%26aulast%3DLee%26aufirst%3DM.%2BS.%26aulast%3DChang%26aufirst%3DG.%2BD.%26atitle%3DAntibody-Assisted%2520Target%2520Identification%2520Reveals%2520Afatinib%252C%2520an%2520Egfr%2520Covalent%2520Inhibitor%252C%2520Down-Regulating%2520Ribonucleotide%2520Reductase%26jtitle%3DOncotarget%26date%3D2018%26volume%3D9%26spage%3D21512%26epage%3D21529%26doi%3D10.18632%2Foncotarget.25177" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 5 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Bhagyashri  Rathod</span>, <span class="hlFld-ContribAuthor ">Shivam  Chak</span>, <span class="hlFld-ContribAuthor ">Sagarkumar  Patel</span>, <span class="hlFld-ContribAuthor ">Amit  Shard</span>. </span><span class="cited-content_cbyCitation_article-title">Tumor pyruvate kinase M2 modulators: a comprehensive account of activators and inhibitors as anticancer agents. </span><span class="cited-content_cbyCitation_journal-name">RSC Medicinal Chemistry</span><span> <strong>2021,</strong> <em>12 </em>
                                    (7)
                                     , 1121-1141. <a href="https://doi.org/10.1039/D1MD00045D" title="DOI URL">https://doi.org/10.1039/D1MD00045D</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D1MD00045D&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD1MD00045D%26sid%3Dliteratum%253Aachs%26jtitle%3DRSC%2520Medicinal%2520Chemistry%26atitle%3DTumor%252Bpyruvate%252Bkinase%252BM2%252Bmodulators%25253A%252Ba%252Bcomprehensive%252Baccount%252Bof%252Bactivators%252Band%252Binhibitors%252Bas%252Banticancer%252Bagents%26aulast%3DRathod%26aufirst%3DBhagyashri%26date%3D2021%26date%3D2021%26volume%3D12%26issue%3D7%26spage%3D1121%26epage%3D1141" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xueyan  Sun</span>, <span class="hlFld-ContribAuthor ">Yijiao  Peng</span>, <span class="hlFld-ContribAuthor ">Jingduo  Zhao</span>, <span class="hlFld-ContribAuthor ">Zhizhong  Xie</span>, <span class="hlFld-ContribAuthor ">Xiaoyong  Lei</span>, <span class="hlFld-ContribAuthor ">Guotao  Tang</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery and development of tumor glycolysis rate-limiting enzyme inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic Chemistry</span><span> <strong>2021,</strong> <em>112 </em>, 104891. <a href="https://doi.org/10.1016/j.bioorg.2021.104891" title="DOI URL">https://doi.org/10.1016/j.bioorg.2021.104891</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bioorg.2021.104891&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bioorg.2021.104891%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520Chemistry%26atitle%3DDiscovery%252Band%252Bdevelopment%252Bof%252Btumor%252Bglycolysis%252Brate-limiting%252Benzyme%252Binhibitors%26aulast%3DSun%26aufirst%3DXueyan%26date%3D2021%26volume%3D112%26spage%3D104891" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Rajkumar  Patle</span>, <span class="hlFld-ContribAuthor ">Shital  Shinde</span>, <span class="hlFld-ContribAuthor ">Sagarkumar  Patel</span>, <span class="hlFld-ContribAuthor ">Rahul  Maheshwari</span>, <span class="hlFld-ContribAuthor ">Heena  Jariyal</span>, <span class="hlFld-ContribAuthor ">Akshay  Srivastava</span>, <span class="hlFld-ContribAuthor ">Neelam  Chauhan</span>, <span class="hlFld-ContribAuthor ">Christoph  Globisch</span>, <span class="hlFld-ContribAuthor ">Alok  Jain</span>, <span class="hlFld-ContribAuthor ">Rakesh K.  Tekade</span>, <span class="hlFld-ContribAuthor ">Amit  Shard</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of boronic acid-based potent activators of tumor pyruvate kinase M2 and development of gastroretentive nanoformulation for oral dosing. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2021,</strong> <em>42 </em>, 128062. <a href="https://doi.org/10.1016/j.bmcl.2021.128062" title="DOI URL">https://doi.org/10.1016/j.bmcl.2021.128062</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2021.128062&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2021.128062%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DDiscovery%252Bof%252Bboronic%252Bacid-based%252Bpotent%252Bactivators%252Bof%252Btumor%252Bpyruvate%252Bkinase%252BM2%252Band%252Bdevelopment%252Bof%252Bgastroretentive%252Bnanoformulation%252Bfor%252Boral%252Bdosing%26aulast%3DPatle%26aufirst%3DRajkumar%26date%3D2021%26volume%3D42%26spage%3D128062" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jian  Wang</span>, <span class="hlFld-ContribAuthor ">Cheng  Zhang</span>, <span class="hlFld-ContribAuthor ">Xiao-Qing  Ye</span>, <span class="hlFld-ContribAuthor ">Wenting  Du</span>, <span class="hlFld-ContribAuthor ">Shenxin  Zeng</span>, <span class="hlFld-ContribAuthor ">Jian-Hong  Xu</span>, <span class="hlFld-ContribAuthor ">Hong  Yin</span>. </span><span class="cited-content_cbyCitation_article-title">An efficient and practical aerobic oxidation of benzylic methylenes by recyclable
              N
              -hydroxyimide. </span><span class="cited-content_cbyCitation_journal-name">RSC Advances</span><span> <strong>2021,</strong> <em>11 </em>
                                    (5)
                                     , 3003-3011. <a href="https://doi.org/10.1039/D0RA10475B" title="DOI URL">https://doi.org/10.1039/D0RA10475B</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D0RA10475B&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD0RA10475B%26sid%3Dliteratum%253Aachs%26jtitle%3DRSC%2520Advances%26atitle%3DAn%252Befficient%252Band%252Bpractical%252Baerobic%252Boxidation%252Bof%252Bbenzylic%252Bmethylenes%252Bby%252Brecyclable%252BN%252B-hydroxyimide%26aulast%3DWang%26aufirst%3DJian%26date%3D2021%26date%3D2021%26volume%3D11%26issue%3D5%26spage%3D3003%26epage%3D3011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ilias  Georgakopoulos-Soares</span>, <span class="hlFld-ContribAuthor ">Dionysios V.  Chartoumpekis</span>, <span class="hlFld-ContribAuthor ">Venetsana  Kyriazopoulou</span>, <span class="hlFld-ContribAuthor ">Apostolos  Zaravinos</span>. </span><span class="cited-content_cbyCitation_article-title">EMT Factors and Metabolic Pathways in Cancer. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Oncology</span><span> <strong>2020,</strong> <em>10 </em><a href="https://doi.org/10.3389/fonc.2020.00499" title="DOI URL">https://doi.org/10.3389/fonc.2020.00499</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fonc.2020.00499&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffonc.2020.00499%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Oncology%26atitle%3DEMT%252BFactors%252Band%252BMetabolic%252BPathways%252Bin%252BCancer%26aulast%3DGeorgakopoulos-Soares%26aufirst%3DIlias%26date%3D2020%26date%3D2020%26volume%3D10" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b00763/20190920/images/medium/jm9b00763_0009.gif" alt="" id="_i2" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b00763/20190920/images/large/jm9b00763_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00763&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b00763/20190920/images/medium/jm9b00763_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b00763/20190920/images/large/jm9b00763_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. (A) Structures of compound <b>22</b> versus compound <b>1</b> and its ABPP probe <b>2</b>. (B and C) Synthetic schemes for compounds <b>1</b> (B) and <b>2</b> (C).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b00763/20190920/images/large/jm9b00763_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00763&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b00763/20190920/images/medium/jm9b00763_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b00763/20190920/images/large/jm9b00763_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Differential antiproliferative effects of compounds <b>1</b> and <b>2</b> in different cancer cell lines by MTT assays. (A) Concentration-dependent suppressive effects of compound <b>1</b> on the viability of different cancer cell lines, including those of pancreas (Panc-1, AsPC-1, and MiaPaCa-2), breast (MDA-MB-231 and MCF-7), oral (SCC2095 and SCC4), and prostate (PC-3 and LNCaP). (B) Concentration-dependent suppressive effects of compounds <b>1</b> (left) and <b>2</b> (right) on the viability of lung cancer cell lines harboring mutant EGFR (H1650, PC-9, and H1975) versus those with wild-type EGFR (H460 and H157). Value, means ± SD (<i>n</i> = 6).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b00763/20190920/images/large/jm9b00763_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00763&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b00763/20190920/images/medium/jm9b00763_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b00763/20190920/images/large/jm9b00763_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. ABPP-SILAC-based proteomic analysis to identify cellular targets of compound <b>1</b>. (A) A schematic diagram depicting the ABPP-SILAC-based quantitative proteomics strategy. (B) SDS-PAGE analysis of ABPP-labeled target proteins via streptavidin-bead pulldown, which was silver stained. The indicated gel regions were excised from SDS-PAGE gels and subjected to in-gel digestion with trypsin and subsequent target identification by LC-MS/MS.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b00763/20190920/images/large/jm9b00763_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00763&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b00763/20190920/images/medium/jm9b00763_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b00763/20190920/images/large/jm9b00763_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Identification of PKM2 as a target of compound <b>1</b>. (A) A list of kinase protein targets of compound <b>1</b> identified by LC-MS/MS in PC-9 cancer cells. (B) Protein–protein interaction pathway map of compound <b>1</b>’s target proteins. (C) GO functional annotation for related targets and biological processes of PKM2.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b00763/20190920/images/large/jm9b00763_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00763&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b00763/20190920/images/medium/jm9b00763_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b00763/20190920/images/large/jm9b00763_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Evidence that compound <b>1</b> is an irreversible inhibitor of PKM2. (A) PC-9 cells treated with compound <b>1</b> (0.25 μM) showed significantly reduced pyruvate kinase activity relative to vehicle control. Bar, means ± SD (<i>n</i> = 3). ***<i>P</i> < 0.005. (B) Compound <b>1</b> selectively inhibited the kinase activity of recombinant PKM2, but not recombinant PKM1. Bar, means ± SD (<i>n</i> = 3). ***<i>P</i> < 0.001. (C) Compound <b>1</b> was effective in competing with the ABPP probe compound <b>2</b> for PKM2 binding. PC-9 cells were preincubated with 1 μM compound <b>1</b> for 1 h, and treated with compound <b>2</b> at indicated concentrations. Cells were lysed, and treated with biotin-azide, followed by streptavidin bead pulldown and Western blot analysis. (D) Anticompound <b>1</b> antibodies cross reacted with compound <b>1</b>-treated PKM2. Recombinant PKM2 (4 μg) was incubated with vehicle control or compound <b>1</b> at indicated concentrations at 4 °C for 6 h, followed by immunoblotting with anti-PKM2 and anti-compound-<b>1</b> antibodies.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b00763/20190920/images/large/jm9b00763_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00763&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b00763/20190920/images/medium/jm9b00763_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b00763/20190920/images/large/jm9b00763_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Identification of Cys326 and Cys317 as potential compound <b>1</b> modified sites on PKM2 via MS/MS analysis. MS/MS spectra derived from compound <b>1</b>-modified tryptic peptides of PKM2 afforded (A) the [M+5H]<sup>5+</sup> precursor ion at <i>m</i>/<i>z</i> 615.12323 Da for the Cys326-containing peptide AGKPVIC<sup>326</sup>ATQMLESMIKKPRPTR and (B) the [M+3H]<sup>3+</sup> precursor ion at <i>m</i>/<i>z</i> 499.55002 Da for the Cys317-carrying peptide MMIGRC<sup>317</sup>NR (B). The amino acid sequences and respective b and y ions are shown in each spectrum, with the compound <b>1</b>-modified cysteine residues underlined. All detected b and y ions carrying the modified Cys residue (b7–11, b14, and y19–20 for peptide AGKPVIC<sup>326</sup>ATQMLESMIKKPRPTR; b6, y3, and y5–7 for peptide MMIGRC<sup>317</sup>NR) dotted in the peptide sequence shown were found to retain the compound <b>1</b> moiety (+ 516.1773 Da). (C) A depiction of the Cys326 and Cys317 residues on PKM2 (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3gr4">3gr4</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b00763/20190920/images/large/jm9b00763_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00763&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b00763/20190920/images/medium/jm9b00763_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b00763/20190920/images/large/jm9b00763_0007.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Evidence that compound <b>1</b> target both metabolic and oncogenic functions in PC-9 cells. (A) Left, a representative graph of ECAR outputs in response to vehicle control (blue) or compound <b>1</b> at 0.25 μM (beige), 0.5 μM (green), and 1 μM (red). Glycolytic stress tests were performed using the Seahorse XF bioanalyzer to measure the glycolytic capacity of PC-9 cells. Right, average values of key parameters for the evaluation of glycolytic function with or without compound <b>1</b>. Bar, means ± SD (<i>n</i> = 6). *<i>P</i> < 0.05, **<i>P</i> < 0.01. (B) Left, a representative graph of OCAR outputs in response to vehicle control (blue) or compound <b>1</b> at 0.25 μM (beige), 0.5 μM (green), and 1 μM (red). Right, average values of key parameters for the evaluation of mitochondrial functions with or without compound <b>1</b>. Bar, means ± SD (<i>n</i> = 6). *<i>P</i> < 0.05. (C) Western blot analyses of the concentration-dependent suppressive effect of compound <b>1</b> on the expression and/or phosphorylation of PKM2, EGFR, Stat3, β-Catenin in PC-9 cells. (D) Effect of compound 1 at indicated concentrations on ROS production in PC-9 cells. Bar, means ± SD (<i>n</i> = 3). **<i>P</i> < 0.01.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b00763/20190920/images/large/jm9b00763_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00763&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b00763/20190920/images/medium/jm9b00763_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b00763/20190920/images/large/jm9b00763_0008.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. <i>In vivo</i> efficacy of compound <b>1</b> in suppressing the growth of PC-9 xenograft tumors in nude mice. (A) Suppressive effect of compound <b>1</b> at 10 mg/kg via daily i.p. injection on PC-9 xenograft tumor growth after 25 days of treatment. Top, representative images of vehicle- and compound <b>1</b>-treated PC-9 xenograft tumor-bearing mice and dissected tumor samples after 25 days of treatment. Bottom, effects of compound <b>1</b> versus vehicle control on tumor volumes (left), tumor weight (right), and body weight (B) in the course of treatment. Data are expressed as mean ± SD (<i>n</i> = 8). ***<i>P</i> < 0.001. (C) Left, Western blot analysis of the effects of compound <b>1</b> versus vehicle control on the expression of PKM2 and downstream targets, including PGK1, EGFR, Stat3 and β-catenin in tumor lysates. GAPDH as an internal control. Right, quantification of the ratio of protein expression level was normalized to GAPDH of tumor lysate by ImageJ tool. **<i>P</i> < 0.01, ***<i>P</i> < 0.001.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-18/acs.jmedchem.9b00763/20190920/images/large/jm9b00763_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00763&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i44">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_64015" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_64015" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 42 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hsu, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sabatini, D. M.</span></span> <span> </span><span class="NLM_article-title">Cancer Cell Metabolism: Warburg and Beyond</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>134</i></span>,  <span class="NLM_fpage">703</span>– <span class="NLM_lpage">707</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2008.08.021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=10.1016%2Fj.cell.2008.08.021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=18775299" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtFCqs7jK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=134&publication_year=2008&pages=703-707&author=P.+P.+Hsuauthor=D.+M.+Sabatini&title=Cancer+Cell+Metabolism%3A+Warburg+and+Beyond&doi=10.1016%2Fj.cell.2008.08.021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Cancer cell metabolism: Warburg and beyond</span></div><div class="casAuthors">Hsu, Peggy P.; Sabatini, David M.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">134</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">703-707</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Described decades ago, the Warburg effect of aerobic glycolysis is a key metabolic hallmark of cancer, yet its significance remains unclear.  In this Essay, we re-examine the Warburg effect and establish a framework for understanding its contribution to the altered metab. of cancer cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqRBI3IwvZP5rVg90H21EOLACvtfcHk0lhwkSs3euzyvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtFCqs7jK&md5=bc4e91dc941eb33a7b60572126537676</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2008.08.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2008.08.021%26sid%3Dliteratum%253Aachs%26aulast%3DHsu%26aufirst%3DP.%2BP.%26aulast%3DSabatini%26aufirst%3DD.%2BM.%26atitle%3DCancer%2520Cell%2520Metabolism%253A%2520Warburg%2520and%2520Beyond%26jtitle%3DCell%26date%3D2008%26volume%3D134%26spage%3D703%26epage%3D707%26doi%3D10.1016%2Fj.cell.2008.08.021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tennant, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duran, R. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boulahbel, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gottlieb, E.</span></span> <span> </span><span class="NLM_article-title">Metabolic Transformation in Cancer</span>. <i>Carcinogenesis</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">1269</span>– <span class="NLM_lpage">1280</span>, <span class="refDoi"> DOI: 10.1093/carcin/bgp070</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=10.1093%2Fcarcin%2Fbgp070" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=19321800" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=1%3ACAS%3A528%3ADC%252BD1MXpsVaksbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2009&pages=1269-1280&author=D.+A.+Tennantauthor=R.+V.+Duranauthor=H.+Boulahbelauthor=E.+Gottlieb&title=Metabolic+Transformation+in+Cancer&doi=10.1093%2Fcarcin%2Fbgp070"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Metabolic transformation in cancer</span></div><div class="casAuthors">Tennant, Daniel A.; Duran, Raul V.; Boulahbel, Houda; Gottlieb, Eyal</div><div class="citationInfo"><span class="NLM_cas:title">Carcinogenesis</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1269-1280</span>CODEN:
                <span class="NLM_cas:coden">CRNGDP</span>;
        ISSN:<span class="NLM_cas:issn">0143-3334</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">A review.  In 2000, Douglas Hanahan and Robert Weinberg published a review detailing the six hallmarks of cancer.  These are six phenotypes that a tumor requires in order to become a fully fledged malignancy: persistent growth signals, evasion of apoptosis, insensitivity to anti-growth signals, unlimited replicative potential, angiogenesis and invasion and metastasis.  However, it is becoming increasingly clear that these phenotypes do not portray the whole story and that other hallmarks are necessary: one of which is a shift in cellular metab.  The tumor environment creates a unique collection of stresses to which cells must adapt in order to survive.  This environment is formed by the uncontrolled proliferation of cells, which ignore the cues that would create normal tissue architecture.  As a result, the cells forming the tumor are exposed to low oxygen and nutrient levels, as well as high levels of toxic cellular waste products, which is thought to propel cells towards a more transformed phenotype, resistant to cell death and pro-metastatic.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-VBZ0PfaxO7Vg90H21EOLACvtfcHk0lixmOdeoFLxyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXpsVaksbc%253D&md5=60e1db58d07faae083753aa604a7c1cf</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1093%2Fcarcin%2Fbgp070&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fcarcin%252Fbgp070%26sid%3Dliteratum%253Aachs%26aulast%3DTennant%26aufirst%3DD.%2BA.%26aulast%3DDuran%26aufirst%3DR.%2BV.%26aulast%3DBoulahbel%26aufirst%3DH.%26aulast%3DGottlieb%26aufirst%3DE.%26atitle%3DMetabolic%2520Transformation%2520in%2520Cancer%26jtitle%3DCarcinogenesis%26date%3D2009%26volume%3D30%26spage%3D1269%26epage%3D1280%26doi%3D10.1093%2Fcarcin%2Fbgp070" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vander
Heiden, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantley, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, C. B.</span></span> <span> </span><span class="NLM_article-title">Understanding the Warburg Effect: The Metabolic Requirements of Cell Proliferation</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>324</i></span>,  <span class="NLM_fpage">1029</span>– <span class="NLM_lpage">1033</span>, <span class="refDoi"> DOI: 10.1126/science.1160809</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=10.1126%2Fscience.1160809" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=19460998" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=1%3ACAS%3A528%3ADC%252BD1MXmtVKlsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=324&publication_year=2009&pages=1029-1033&author=M.+G.+Vander%0AHeidenauthor=L.+C.+Cantleyauthor=C.+B.+Thompson&title=Understanding+the+Warburg+Effect%3A+The+Metabolic+Requirements+of+Cell+Proliferation&doi=10.1126%2Fscience.1160809"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Understanding the Warburg Effect: The Metabolic Requirements of Cell Proliferation</span></div><div class="casAuthors">Vander Heiden, Matthew G.; Cantley, Lewis C.; Thompson, Craig B.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">324</span>
        (<span class="NLM_cas:issue">5930</span>),
    <span class="NLM_cas:pages">1029-1033</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">A review.  In contrast to normal differentiated cells, which rely primarily on mitochondrial oxidative phosphorylation to generate the energy needed for cellular processes, most cancer cells instead rely on aerobic glycolysis, a phenomenon termed "the Warburg effect".  Aerobic glycolysis is an inefficient way to generate ATP, however, and the advantage it confers to cancer cells has been unclear.  Here we propose that the metab. of cancer cells, and indeed all proliferating cells, is adapted to facilitate the uptake and incorporation of nutrients into the biomass (e.g., nucleotides, amino acids, and lipids) needed to produce a new cell.  Supporting this idea are recent studies showing that (i) several signaling pathways implicated in cell proliferation also regulate metabolic pathways that incorporate nutrients into biomass; and that (ii) certain cancer-assocd. mutations enable cancer cells to acquire and metabolize nutrients in a manner conducive to proliferation rather than efficient ATP prodn.  A better understanding of the mechanistic links between cellular metab. and growth control may ultimately lead to better treatments for human cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrGvLHGQ9TX77Vg90H21EOLACvtfcHk0lixmOdeoFLxyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXmtVKlsbg%253D&md5=f927402a43cdca471c3f4874dcbeb943</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1126%2Fscience.1160809&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1160809%26sid%3Dliteratum%253Aachs%26aulast%3DVander%2BHeiden%26aufirst%3DM.%2BG.%26aulast%3DCantley%26aufirst%3DL.%2BC.%26aulast%3DThompson%26aufirst%3DC.%2BB.%26atitle%3DUnderstanding%2520the%2520Warburg%2520Effect%253A%2520The%2520Metabolic%2520Requirements%2520of%2520Cell%2520Proliferation%26jtitle%3DScience%26date%3D2009%26volume%3D324%26spage%3D1029%26epage%3D1033%26doi%3D10.1126%2Fscience.1160809" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lunt, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vander Heiden, M. G.</span></span> <span> </span><span class="NLM_article-title">Aerobic Glycolysis: Meeting the Metabolic Requirements of Cell Proliferation</span>. <i>Annu. Rev. Cell Dev. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">441</span>– <span class="NLM_lpage">464</span>, <span class="refDoi"> DOI: 10.1146/annurev-cellbio-092910-154237</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=10.1146%2Fannurev-cellbio-092910-154237" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=21985671" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsFyqtbfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2011&pages=441-464&author=S.+Y.+Luntauthor=M.+G.+Vander+Heiden&title=Aerobic+Glycolysis%3A+Meeting+the+Metabolic+Requirements+of+Cell+Proliferation&doi=10.1146%2Fannurev-cellbio-092910-154237"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Aerobic glycolysis: meeting the metabolic requirements of cell proliferation</span></div><div class="casAuthors">Lunt, Sophia Y.; Vander Heiden, Matthew G.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Cell and Developmental Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">441-464</span>CODEN:
                <span class="NLM_cas:coden">ARDBF8</span>;
        ISSN:<span class="NLM_cas:issn">1081-0706</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review.  Warburg's observation that cancer cells exhibit a high rate of glycolysis even in the presence of oxygen (aerobic glycolysis) sparked debate over the role of glycolysis in normal and cancer cells.  Although it has been established that defects in mitochondrial respiration are not the cause of cancer or aerobic glycolysis, the advantages of enhanced glycolysis in cancer remain controversial.  Many cells ranging from microbes to lymphocytes use aerobic glycolysis during rapid proliferation, which suggests it may play a fundamental role in supporting cell growth.  Here, we review how glycolysis contributes to the metabolic processes of dividing cells.  We provide a detailed accounting of the biosynthetic requirements to construct a new cell and illustrate the importance of glycolysis in providing carbons to generate biomass.  We argue that the major function of aerobic glycolysis is to maintain high levels of glycolytic intermediates to support anabolic reactions in cells, thus providing an explanation for why increased glucose metab. is selected for in proliferating cells throughout nature.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrDaFTmS-BapLVg90H21EOLACvtfcHk0lixmOdeoFLxyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsFyqtbfN&md5=85f04cff291334181cc0aae9015e0e0d</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1146%2Fannurev-cellbio-092910-154237&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-cellbio-092910-154237%26sid%3Dliteratum%253Aachs%26aulast%3DLunt%26aufirst%3DS.%2BY.%26aulast%3DVander%2BHeiden%26aufirst%3DM.%2BG.%26atitle%3DAerobic%2520Glycolysis%253A%2520Meeting%2520the%2520Metabolic%2520Requirements%2520of%2520Cell%2520Proliferation%26jtitle%3DAnnu.%2520Rev.%2520Cell%2520Dev.%2520Biol.%26date%3D2011%26volume%3D27%26spage%3D441%26epage%3D464%26doi%3D10.1146%2Fannurev-cellbio-092910-154237" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lehuede, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dupuy, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rabinovitch, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siegel, P. M.</span></span> <span> </span><span class="NLM_article-title">Metabolic Plasticity as a Determinant of Tumor Growth and Metastasis</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>76</i></span>,  <span class="NLM_fpage">5201</span>– <span class="NLM_lpage">5208</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-16-0266</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=10.1158%2F0008-5472.CAN-16-0266" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=27587539" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=1%3ACAS%3A280%3ADC%252BC2szmvVaktQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=76&publication_year=2016&pages=5201-5208&author=C.+Lehuedeauthor=F.+Dupuyauthor=R.+Rabinovitchauthor=R.+G.+Jonesauthor=P.+M.+Siegel&title=Metabolic+Plasticity+as+a+Determinant+of+Tumor+Growth+and+Metastasis&doi=10.1158%2F0008-5472.CAN-16-0266"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Metabolic Plasticity as a Determinant of Tumor Growth and Metastasis</span></div><div class="casAuthors">Lehuede Camille; Dupuy Fanny; Rabinovitch Rebecca; Jones Russell G; Siegel Peter M</div><div class="citationInfo"><span class="NLM_cas:title">Cancer research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">76</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">5201-8</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Cancer cells must adapt their metabolism to meet the energetic and biosynthetic demands that accompany rapid growth of the primary tumor and colonization of distinct metastatic sites.  Different stages of the metastatic cascade can also present distinct metabolic challenges to disseminating cancer cells.  However, little is known regarding how changes in cellular metabolism, both within the cancer cell and the metastatic microenvironment, alter the ability of tumor cells to colonize and grow in distinct secondary sites.  This review examines the concept of metabolic heterogeneity within the primary tumor, and how cancer cells are metabolically coupled with other cancer cells that comprise the tumor and cells within the tumor stroma.  We examine how metabolic strategies, which are engaged by cancer cells in the primary site, change during the metastatic process.  Finally, we discuss the metabolic adaptations that occur as cancer cells colonize foreign metastatic microenvironments and how cancer cells influence the metabolism of stromal cells at sites of metastasis.  Through a discussion of these topics, it is clear that plasticity in tumor metabolic programs, which allows cancer cells to adapt and grow in hostile microenvironments, is emerging as an important variable that may change clinical approaches to managing metastatic disease.  Cancer Res; 76(18); 5201-8. ©2016 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRAzaCt0cgSwUNHe_gFldTbfW6udTcc2ebG_pjaJGdDZrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2szmvVaktQ%253D%253D&md5=c9f4da8a4e27842634af2801334245ff</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-16-0266&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-16-0266%26sid%3Dliteratum%253Aachs%26aulast%3DLehuede%26aufirst%3DC.%26aulast%3DDupuy%26aufirst%3DF.%26aulast%3DRabinovitch%26aufirst%3DR.%26aulast%3DJones%26aufirst%3DR.%2BG.%26aulast%3DSiegel%26aufirst%3DP.%2BM.%26atitle%3DMetabolic%2520Plasticity%2520as%2520a%2520Determinant%2520of%2520Tumor%2520Growth%2520and%2520Metastasis%26jtitle%3DCancer%2520Res.%26date%3D2016%26volume%3D76%26spage%3D5201%26epage%3D5208%26doi%3D10.1158%2F0008-5472.CAN-16-0266" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chan, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sutphin, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turcotte, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, E. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banh, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reynolds, G. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chi, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Solow-Cordero, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonnet, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flanagan, J. U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bouley, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graves, E. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denny, W. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hay, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giaccia, A. J.</span></span> <span> </span><span class="NLM_article-title">Targeting Glut1 and the Warburg Effect in Renal Cell Carcinoma by Chemical Synthetic Lethality</span>. <i>Sci. Transl. Med.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">94ra70</span>, <span class="refDoi"> DOI: 10.1126/scitranslmed.3002394</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=10.1126%2Fscitranslmed.3002394" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=21813754" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFOisLzK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2011&pages=94ra70&author=D.+A.+Chanauthor=P.+D.+Sutphinauthor=P.+Nguyenauthor=S.+Turcotteauthor=E.+W.+Laiauthor=A.+Banhauthor=G.+E.+Reynoldsauthor=J.+T.+Chiauthor=J.+Wuauthor=D.+E.+Solow-Corderoauthor=M.+Bonnetauthor=J.+U.+Flanaganauthor=D.+M.+Bouleyauthor=E.+E.+Gravesauthor=W.+A.+Dennyauthor=M.+P.+Hayauthor=A.+J.+Giaccia&title=Targeting+Glut1+and+the+Warburg+Effect+in+Renal+Cell+Carcinoma+by+Chemical+Synthetic+Lethality&doi=10.1126%2Fscitranslmed.3002394"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting GLUT1 and the Warburg effect in renal cell carcinoma by chemical synthetic lethality</span></div><div class="casAuthors">Chan, Denise A.; Sutphin, Patrick D.; Nguyen, Phuong; Turcotte, Sandra; Lai, Edwin W.; Banh, Alice; Reynolds, Gloria E.; Chi, Jen-Tsan; Wu, Jason; Solow-Cordero, David E.; Bonnet, Muriel; Flanagan, Jack U.; Bouley, Donna M.; Graves, Edward E.; Denny, William A.; Hay, Michael P.; Giaccia, Amato J.</div><div class="citationInfo"><span class="NLM_cas:title">Science Translational Medicine</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">94</span>),
    <span class="NLM_cas:pages">94ra70</span>CODEN:
                <span class="NLM_cas:coden">STMCBQ</span>;
        ISSN:<span class="NLM_cas:issn">1946-6242</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Identifying new targeted therapies that kill tumor cells while sparing normal tissue is a major challenge of cancer research.  Using a high-throughput chem. synthetic lethal screen, we sought to identify compds. that exploit the loss of the von Hippel-Lindau (VHL) tumor suppressor gene, which occurs in about 80% of renal cell carcinomas (RCCs).  RCCs, like many other cancers, are dependent on aerobic glycolysis for ATP prodn., a phenomenon known as the Warburg effect.  The dependence of RCCs on glycolysis is in part a result of induction of glucose transporter 1 (GLUT1).  Here, we report the identification of a class of compds., the 3-series, exemplified by STF-31, which selectively kills RCCs by specifically targeting glucose uptake through GLUT1 and exploiting the unique dependence of these cells on GLUT1 for survival.  Treatment with these agents inhibits the growth of RCCs by binding GLUT1 directly and impeding glucose uptake in vivo without toxicity to normal tissue.  Activity of STF-31 in these exptl. renal tumors can be monitored by [18F] fluorodeoxyglucose uptake by micro-positron emission tomog. imaging, and therefore, these agents may be readily tested clin. in human tumors.  Our results show that the Warburg effect confers distinct characteristics on tumor cells that can be selectively targeted for therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZodRjh_SzxrVg90H21EOLACvtfcHk0lhPZjb0bfzSGA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFOisLzK&md5=418628e65c3a96425353b4be2d692927</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1126%2Fscitranslmed.3002394&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscitranslmed.3002394%26sid%3Dliteratum%253Aachs%26aulast%3DChan%26aufirst%3DD.%2BA.%26aulast%3DSutphin%26aufirst%3DP.%2BD.%26aulast%3DNguyen%26aufirst%3DP.%26aulast%3DTurcotte%26aufirst%3DS.%26aulast%3DLai%26aufirst%3DE.%2BW.%26aulast%3DBanh%26aufirst%3DA.%26aulast%3DReynolds%26aufirst%3DG.%2BE.%26aulast%3DChi%26aufirst%3DJ.%2BT.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DSolow-Cordero%26aufirst%3DD.%2BE.%26aulast%3DBonnet%26aufirst%3DM.%26aulast%3DFlanagan%26aufirst%3DJ.%2BU.%26aulast%3DBouley%26aufirst%3DD.%2BM.%26aulast%3DGraves%26aufirst%3DE.%2BE.%26aulast%3DDenny%26aufirst%3DW.%2BA.%26aulast%3DHay%26aufirst%3DM.%2BP.%26aulast%3DGiaccia%26aufirst%3DA.%2BJ.%26atitle%3DTargeting%2520Glut1%2520and%2520the%2520Warburg%2520Effect%2520in%2520Renal%2520Cell%2520Carcinoma%2520by%2520Chemical%2520Synthetic%2520Lethality%26jtitle%3DSci.%2520Transl.%2520Med.%26date%3D2011%26volume%3D3%26spage%3D94ra70%26doi%3D10.1126%2Fscitranslmed.3002394" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Calvo, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Figueroa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pulido, E. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campelo, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aparicio, L. A.</span></span> <span> </span><span class="NLM_article-title">Potential Role of Sugar Transporters in Cancer and Their Relationship with Anticancer Therapy</span>. <i>Int. J. Endocrinol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>2010</i></span>, <span class="NLM_elocation-id">205357</span> <span class="refDoi"> DOI: 10.1155/2010/205357</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=10.1155%2F2010%2F205357" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=20706540" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2010&publication_year=2010&author=M.+B.+Calvoauthor=A.+Figueroaauthor=E.+G.+Pulidoauthor=R.+G.+Campeloauthor=L.+A.+Aparicio&title=Potential+Role+of+Sugar+Transporters+in+Cancer+and+Their+Relationship+with+Anticancer+Therapy&doi=10.1155%2F2010%2F205357"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1155%2F2010%2F205357&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1155%252F2010%252F205357%26sid%3Dliteratum%253Aachs%26aulast%3DCalvo%26aufirst%3DM.%2BB.%26aulast%3DFigueroa%26aufirst%3DA.%26aulast%3DPulido%26aufirst%3DE.%2BG.%26aulast%3DCampelo%26aufirst%3DR.%2BG.%26aulast%3DAparicio%26aufirst%3DL.%2BA.%26atitle%3DPotential%2520Role%2520of%2520Sugar%2520Transporters%2520in%2520Cancer%2520and%2520Their%2520Relationship%2520with%2520Anticancer%2520Therapy%26jtitle%3DInt.%2520J.%2520Endocrinol.%26date%3D2010%26volume%3D2010%26doi%3D10.1155%2F2010%2F205357" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Patra, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhaskar, P. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wheaton, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chandel, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laakso, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allen, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jha, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smolen, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clasquin, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robey, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hay, N.</span></span> <span> </span><span class="NLM_article-title">Hexokinase 2 Is Required for Tumor Initiation and Maintenance and Its Systemic Deletion Is Therapeutic in Mouse Models of Cancer</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">213</span>– <span class="NLM_lpage">228</span>, <span class="refDoi"> DOI: 10.1016/j.ccr.2013.06.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=10.1016%2Fj.ccr.2013.06.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=23911236" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1Wrur%252FL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2013&pages=213-228&author=K.+C.+Patraauthor=Q.+Wangauthor=P.+T.+Bhaskarauthor=L.+Millerauthor=Z.+Wangauthor=W.+Wheatonauthor=N.+Chandelauthor=M.+Laaksoauthor=W.+J.+Mullerauthor=E.+L.+Allenauthor=A.+K.+Jhaauthor=G.+A.+Smolenauthor=M.+F.+Clasquinauthor=B.+Robeyauthor=N.+Hay&title=Hexokinase+2+Is+Required+for+Tumor+Initiation+and+Maintenance+and+Its+Systemic+Deletion+Is+Therapeutic+in+Mouse+Models+of+Cancer&doi=10.1016%2Fj.ccr.2013.06.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Hexokinase 2 Is Required for Tumor Initiation and Maintenance and Its Systemic Deletion Is Therapeutic in Mouse Models of Cancer</span></div><div class="casAuthors">Patra, Krushna C.; Wang, Qi; Bhaskar, Prashanth T.; Miller, Luke; Wang, Zebin; Wheaton, Will; Chandel, Navdeep; Laakso, Markku; Muller, William J.; Allen, Eric L.; Jha, Abhishek K.; Smolen, Gromoslaw A.; Clasquin, Michelle F.; Robey, R. Brooks; Hay, Nissim</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">213-228</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Accelerated glucose metab. is a common feature of cancer cells.  Hexokinases catalyze the first committed step of glucose metab.  Hexokinase 2 (HK2) is expressed at high level in cancer cells, but only in a limited no. of normal adult tissues.  Using Hk2 conditional knockout mice, we showed that HK2 is required for tumor initiation and maintenance in mouse models of KRas-driven lung cancer, and ErbB2-driven breast cancer, despite continued HK1 expression.  Similarly, HK2 ablation inhibits the neoplastic phenotype of human lung and breast cancer cells in vitro and in vivo.  Systemic Hk2 deletion is therapeutic in mice bearing lung tumors without adverse physiol. consequences.  Hk2 deletion in lung cancer cells suppressed glucose-derived ribonucleotides and impaired glutamine-derived carbon utilization in anaplerosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMgkDdepYxc7Vg90H21EOLACvtfcHk0lhPZjb0bfzSGA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1Wrur%252FL&md5=837dbeef1e451013754b98cbcfa61bb4</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2013.06.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2013.06.014%26sid%3Dliteratum%253Aachs%26aulast%3DPatra%26aufirst%3DK.%2BC.%26aulast%3DWang%26aufirst%3DQ.%26aulast%3DBhaskar%26aufirst%3DP.%2BT.%26aulast%3DMiller%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DWheaton%26aufirst%3DW.%26aulast%3DChandel%26aufirst%3DN.%26aulast%3DLaakso%26aufirst%3DM.%26aulast%3DMuller%26aufirst%3DW.%2BJ.%26aulast%3DAllen%26aufirst%3DE.%2BL.%26aulast%3DJha%26aufirst%3DA.%2BK.%26aulast%3DSmolen%26aufirst%3DG.%2BA.%26aulast%3DClasquin%26aufirst%3DM.%2BF.%26aulast%3DRobey%26aufirst%3DB.%26aulast%3DHay%26aufirst%3DN.%26atitle%3DHexokinase%25202%2520Is%2520Required%2520for%2520Tumor%2520Initiation%2520and%2520Maintenance%2520and%2520Its%2520Systemic%2520Deletion%2520Is%2520Therapeutic%2520in%2520Mouse%2520Models%2520of%2520Cancer%26jtitle%3DCancer%2520Cell%26date%3D2013%26volume%3D24%26spage%3D213%26epage%3D228%26doi%3D10.1016%2Fj.ccr.2013.06.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tamada, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suematsu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saya, H.</span></span> <span> </span><span class="NLM_article-title">Pyruvate Kinase M2: Multiple Faces for Conferring Benefits on Cancer Cells</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">5554</span>– <span class="NLM_lpage">5561</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.CCR-12-0859</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=10.1158%2F1078-0432.CCR-12-0859" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=23071357" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFahu73L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2012&pages=5554-5561&author=M.+Tamadaauthor=M.+Suematsuauthor=H.+Saya&title=Pyruvate+Kinase+M2%3A+Multiple+Faces+for+Conferring+Benefits+on+Cancer+Cells&doi=10.1158%2F1078-0432.CCR-12-0859"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Pyruvate Kinase M2: Multiple Faces for Conferring Benefits on Cancer Cells</span></div><div class="casAuthors">Tamada, Mayumi; Suematsu, Makoto; Saya, Hideyuki</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">5554-5561</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">A review.  The M2 splice isoform of pyruvate kinase (PKM2), an enzyme that catalyzes the later step of glycolysis, is a key regulator of aerobic glycolysis (known as the Warburg effect) in cancer cells.  Expression and low enzymic activity of PKM2 confer on cancer cells the glycolytic phenotype, which promotes rapid energy prodn. and flow of glycolytic intermediates into collateral pathways to synthesize nucleic acids, amino acids, and lipids without the accumulation of reactive oxygen species.  PKM2 enzymic activity has also been shown to be neg. regulated by the interaction with CD44 adhesion mol., which is a cell surface marker for cancer stem cells.  In addn. to the glycolytic functions, nonglycolytic functions of PKM2 in cancer cells are of particular interest.  PKM2 is induced translocation into the nucleus, where it activates transcription of various genes by interacting with and phosphorylating specific nuclear proteins, endowing cancer cells with a survival and growth advantage.  Therefore, inhibitors and activators of PKM2 are well underway to evaluate their anticancer effects and suitability for use as novel therapeutic strategies.  Clin Cancer Res; 18(20); 5554-61. ©2012 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3jTAHz_wVC7Vg90H21EOLACvtfcHk0ligp_3a54-dVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFahu73L&md5=0da5d879fe53330037ccbecce75b89e4</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-12-0859&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-12-0859%26sid%3Dliteratum%253Aachs%26aulast%3DTamada%26aufirst%3DM.%26aulast%3DSuematsu%26aufirst%3DM.%26aulast%3DSaya%26aufirst%3DH.%26atitle%3DPyruvate%2520Kinase%2520M2%253A%2520Multiple%2520Faces%2520for%2520Conferring%2520Benefits%2520on%2520Cancer%2520Cells%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2012%26volume%3D18%26spage%3D5554%26epage%3D5561%26doi%3D10.1158%2F1078-0432.CCR-12-0859" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wong, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ojo, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, D.</span></span> <span> </span><span class="NLM_article-title">Pkm2 Contributes to Cancer Metabolism</span>. <i>Cancer Lett.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>356</i></span>,  <span class="NLM_fpage">184</span>– <span class="NLM_lpage">191</span>, <span class="refDoi"> DOI: 10.1016/j.canlet.2014.01.031</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=10.1016%2Fj.canlet.2014.01.031" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=24508027" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=1%3ACAS%3A528%3ADC%252BC2cXisVart7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=356&publication_year=2015&pages=184-191&author=N.+Wongauthor=D.+Ojoauthor=J.+Yanauthor=D.+Tang&title=Pkm2+Contributes+to+Cancer+Metabolism&doi=10.1016%2Fj.canlet.2014.01.031"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">PKM2 contributes to cancer metabolism</span></div><div class="casAuthors">Wong, Nicholas; Ojo, Diane; Yan, Judy; Tang, Damu</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Letters (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">356</span>
        (<span class="NLM_cas:issue">2_Part_A</span>),
    <span class="NLM_cas:pages">184-191</span>CODEN:
                <span class="NLM_cas:coden">CALEDQ</span>;
        ISSN:<span class="NLM_cas:issn">0304-3835</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review.  Reprogramming of cell metab. is essential for tumorigenesis, and is regulated by a complex network, in which PKM2 plays a crit. role.  PKM2 exists as an inactive monomer, less active dimer and active tetramer.  While dimeric PKM2 diverts glucose metab. towards anabolism through aerobic glycolysis, tetrameric PKM2 promotes the flux of glucose-derived carbons for ATP prodn. via oxidative phosphorylation.  Equil. of the PKM2 dimers and tetramers is crit. for tumorigenesis, and is controlled by multiple factors.  The PKM2 dimer also promotes aerobic glycolysis by modulating transcriptional regulation.  We will discuss the current understanding of PKM2 in regulating cancer metab.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxTIgwuY8YnLVg90H21EOLACvtfcHk0ligp_3a54-dVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXisVart7c%253D&md5=8562ddc23e089703d976d165b3a2bb3b</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.canlet.2014.01.031&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.canlet.2014.01.031%26sid%3Dliteratum%253Aachs%26aulast%3DWong%26aufirst%3DN.%26aulast%3DOjo%26aufirst%3DD.%26aulast%3DYan%26aufirst%3DJ.%26aulast%3DTang%26aufirst%3DD.%26atitle%3DPkm2%2520Contributes%2520to%2520Cancer%2520Metabolism%26jtitle%3DCancer%2520Lett.%26date%3D2015%26volume%3D356%26spage%3D184%26epage%3D191%26doi%3D10.1016%2Fj.canlet.2014.01.031" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Iqbal, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gupta, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gopinath, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mazurek, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamezai, R. N.</span></span> <span> </span><span class="NLM_article-title">Pyruvate Kinase M2 and Cancer: An Updated Assessment</span>. <i>FEBS Lett.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>588</i></span>,  <span class="NLM_fpage">2685</span>– <span class="NLM_lpage">2692</span>, <span class="refDoi"> DOI: 10.1016/j.febslet.2014.04.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=10.1016%2Fj.febslet.2014.04.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=24747424" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=1%3ACAS%3A528%3ADC%252BC2cXntFCqur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=588&publication_year=2014&pages=2685-2692&author=M.+A.+Iqbalauthor=V.+Guptaauthor=P.+Gopinathauthor=S.+Mazurekauthor=R.+N.+Bamezai&title=Pyruvate+Kinase+M2+and+Cancer%3A+An+Updated+Assessment&doi=10.1016%2Fj.febslet.2014.04.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Pyruvate kinase M2 and cancer: an updated assessment</span></div><div class="casAuthors">Iqbal, Mohd Askandar; Gupta, Vibhor; Gopinath, Prakasam; Mazurek, Sybille; Bamezai, Rameshwar N. K.</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Letters</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">588</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">2685-2692</span>CODEN:
                <span class="NLM_cas:coden">FEBLAL</span>;
        ISSN:<span class="NLM_cas:issn">0014-5793</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Cancer cells are characterized by high glycolytic rates to support energy regeneration and anabolic metab., along with the expression of pyruvate kinase isoenzyme M2 (PKM2).  The latter catalyzes the last step of glycolysis and reprograms the glycolytic flux to feed the special metabolic demands of proliferating cells.  Besides, PKM2 has moonlight functions, such as gene transcription, favoring cancer.  Accumulating evidence suggests a crit. role played by the low-activity-dimeric PKM2 in tumor progression, supported by the identification of mutations which result in the down-regulation of its activity and tumorigenesis in a nude mouse model.  This review discusses PKM2 regulation and the benefits it confers to cancer cells.  Further, conflicting views on PKM2's role in cancer, its therapeutic relevance and future directions in the field are also discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpM-7GvwndQlbVg90H21EOLACvtfcHk0ligp_3a54-dVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXntFCqur0%253D&md5=7552a7afcd740bdf54c28f421a321ca2</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.febslet.2014.04.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.febslet.2014.04.011%26sid%3Dliteratum%253Aachs%26aulast%3DIqbal%26aufirst%3DM.%2BA.%26aulast%3DGupta%26aufirst%3DV.%26aulast%3DGopinath%26aufirst%3DP.%26aulast%3DMazurek%26aufirst%3DS.%26aulast%3DBamezai%26aufirst%3DR.%2BN.%26atitle%3DPyruvate%2520Kinase%2520M2%2520and%2520Cancer%253A%2520An%2520Updated%2520Assessment%26jtitle%3DFEBS%2520Lett.%26date%3D2014%26volume%3D588%26spage%3D2685%26epage%3D2692%26doi%3D10.1016%2Fj.febslet.2014.04.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Christofk, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vander Heiden, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramanathan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerszten, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fleming, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schreiber, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantley, L. C.</span></span> <span> </span><span class="NLM_article-title">The M2 Splice Isoform of Pyruvate Kinase Is Important for Cancer Metabolism and Tumour Growth</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>452</i></span>,  <span class="NLM_fpage">230</span>– <span class="NLM_lpage">233</span>, <span class="refDoi"> DOI: 10.1038/nature06734</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=10.1038%2Fnature06734" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=18337823" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=1%3ACAS%3A528%3ADC%252BD1cXjt1Gnu74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=452&publication_year=2008&pages=230-233&author=H.+R.+Christofkauthor=M.+G.+Vander+Heidenauthor=M.+H.+Harrisauthor=A.+Ramanathanauthor=R.+E.+Gersztenauthor=R.+Weiauthor=M.+D.+Flemingauthor=S.+L.+Schreiberauthor=L.+C.+Cantley&title=The+M2+Splice+Isoform+of+Pyruvate+Kinase+Is+Important+for+Cancer+Metabolism+and+Tumour+Growth&doi=10.1038%2Fnature06734"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">The M2 splice isoform of pyruvate kinase is important for cancer metabolism and tumor growth</span></div><div class="casAuthors">Christofk, Heather R.; Vander Heiden, Matthew G.; Harris, Marian H.; Ramanathan, Arvind; Gerszten, Robert E.; Wei, Ru; Fleming, Mark D.; Schreiber, Stuart L.; Cantley, Lewis C.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">452</span>
        (<span class="NLM_cas:issue">7184</span>),
    <span class="NLM_cas:pages">230-233</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Many tumor cells have elevated rates of glucose uptake but reduced rates of oxidative phosphorylation.  This persistence of high lactate prodn. by tumors in the presence of oxygen, known as aerobic glycolysis, was first noted by Otto Warburg more than 75 yr ago.  How tumor cells establish this altered metabolic phenotype and whether it is essential for tumorigenesis is as yet unknown.  Here we show that a single switch in a splice isoform of the glycolytic enzyme pyruvate kinase is necessary for the shift in cellular metab. to aerobic glycolysis and that this promotes tumorigenesis.  Tumor cells have been shown to express exclusively the embryonic M2 isoform of pyruvate kinase.  Here we use short hairpin RNA to knockdown pyruvate kinase M2 expression in human cancer cell lines and replace it with pyruvate kinase M1.  Switching pyruvate kinase expression to the M1 (adult) isoform leads to reversal of the Warburg effect, as judged by reduced lactate prodn. and increased oxygen consumption, and this correlates with a reduced ability to form tumors in nude mouse xenografts.  These results demonstrate that M2 expression is necessary for aerobic glycolysis and that this metabolic phenotype provides a selective growth advantage for tumor cells in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpvmgcXNsSjpbVg90H21EOLACvtfcHk0ljj5xgWKKvzhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXjt1Gnu74%253D&md5=e7e6b7cc09477e1f9d270018d1c38be4</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fnature06734&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature06734%26sid%3Dliteratum%253Aachs%26aulast%3DChristofk%26aufirst%3DH.%2BR.%26aulast%3DVander%2BHeiden%26aufirst%3DM.%2BG.%26aulast%3DHarris%26aufirst%3DM.%2BH.%26aulast%3DRamanathan%26aufirst%3DA.%26aulast%3DGerszten%26aufirst%3DR.%2BE.%26aulast%3DWei%26aufirst%3DR.%26aulast%3DFleming%26aufirst%3DM.%2BD.%26aulast%3DSchreiber%26aufirst%3DS.%2BL.%26aulast%3DCantley%26aufirst%3DL.%2BC.%26atitle%3DThe%2520M2%2520Splice%2520Isoform%2520of%2520Pyruvate%2520Kinase%2520Is%2520Important%2520for%2520Cancer%2520Metabolism%2520and%2520Tumour%2520Growth%26jtitle%3DNature%26date%3D2008%26volume%3D452%26spage%3D230%26epage%3D233%26doi%3D10.1038%2Fnature06734" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Granchi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fancelli, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minutolo, F.</span></span> <span> </span><span class="NLM_article-title">An Update on Therapeutic Opportunities Offered by Cancer Glycolytic Metabolism</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">4915</span>– <span class="NLM_lpage">4925</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2014.09.041</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=10.1016%2Fj.bmcl.2014.09.041" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=25288186" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1eitbzJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2014&pages=4915-4925&author=C.+Granchiauthor=D.+Fancelliauthor=F.+Minutolo&title=An+Update+on+Therapeutic+Opportunities+Offered+by+Cancer+Glycolytic+Metabolism&doi=10.1016%2Fj.bmcl.2014.09.041"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">An update on therapeutic opportunities offered by cancer glycolytic metabolism</span></div><div class="casAuthors">Granchi, Carlotta; Fancelli, Daniele; Minutolo, Filippo</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">4915-4925</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Almost all invasive cancers, regardless of tissue origin, are characterized by specific modifications of their cellular energy metab.  In fact, a strong predominance of aerobic glycolysis over oxidative phosphorylation (Warburg effect) is usually assocd. with aggressive tumor phenotypes.  This metabolic shift offers a survival advantage to cancer cells, since they may continue to produce energy and anabolites even when they are exposed to either transient or permanent hypoxic conditions.  Moreover, it ensures a high prodn. rate of glycolysis intermediates, useful as building blocks for fast cell proliferation of cancer cells.  This peculiar metabolic profile may constitute an ideal target for therapeutic interventions that selectively hit cancer cells with minimal residual systemic toxicity.  In this review we provide an update about some of the most recent advances in the discovery of new bioactive mols. that are able to interfere with cancer glycolysis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIdWP6wz_jRbVg90H21EOLACvtfcHk0ljj5xgWKKvzhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1eitbzJ&md5=e8b3949321270dad2dcc4757c47b5aa6</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2014.09.041&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2014.09.041%26sid%3Dliteratum%253Aachs%26aulast%3DGranchi%26aufirst%3DC.%26aulast%3DFancelli%26aufirst%3DD.%26aulast%3DMinutolo%26aufirst%3DF.%26atitle%3DAn%2520Update%2520on%2520Therapeutic%2520Opportunities%2520Offered%2520by%2520Cancer%2520Glycolytic%2520Metabolism%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2014%26volume%3D24%26spage%3D4915%26epage%3D4925%26doi%3D10.1016%2Fj.bmcl.2014.09.041" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ganapathy-Kanniappan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geschwind, J. F.</span></span> <span> </span><span class="NLM_article-title">Tumor Glycolysis as a Target for Cancer Therapy: Progress and Prospects</span>. <i>Mol. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">152</span>, <span class="refDoi"> DOI: 10.1186/1476-4598-12-152</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=10.1186%2F1476-4598-12-152" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=24298908" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmsVyjtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2013&pages=152&author=S.+Ganapathy-Kanniappanauthor=J.+F.+Geschwind&title=Tumor+Glycolysis+as+a+Target+for+Cancer+Therapy%3A+Progress+and+Prospects&doi=10.1186%2F1476-4598-12-152"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Tumor glycolysis as a target for cancer therapy: progress and prospects</span></div><div class="casAuthors">Ganapathy-Kanniappan, Shanmugasundaram; Geschwind, Jean-Francois H.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">152/1-152/11, 11 pp.</span>CODEN:
                <span class="NLM_cas:coden">MCOACG</span>;
        ISSN:<span class="NLM_cas:issn">1476-4598</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">A review.  Altered energy metab. is a biochem. fingerprint of cancer cells that represents one of the "hallmarks of cancer".  This metabolic phenotype is characterized by preferential dependence on glycolysis (the process of conversion of glucose into pyruvate followed by lactate prodn.) for energy prodn. in an oxygen-independent manner.  Although glycolysis is less efficient than oxidative phosphorylation in the net yield of ATP (ATP), cancer cells adapt to this math. disadvantage by increased glucose up-take, which in turn facilitates a higher rate of glycolysis.  Apart from providing cellular energy, the metabolic intermediates of glycolysis also play a pivotal role in macromol. biosynthesis, thus conferring selective advantage to cancer cells under diminished nutrient supply.  Accumulating data also indicate that intracellular ATP is a crit. determinant of chemoresistance.  Under hypoxic conditions where glycolysis remains the predominant energy producing pathway sensitizing cancer cells would require intracellular depletion of ATP by inhibition of glycolysis.  Together, the oncogenic regulation of glycolysis and multifaceted roles of glycolytic components underscore the biol. significance of tumor glycolysis.  Thus targeting glycolysis remains attractive for therapeutic intervention.  Several preclin. investigations have indeed demonstrated the effectiveness of this therapeutic approach thereby supporting its scientific rationale.  Recent reviews have provided a wealth of information on the biochem. targets of glycolysis and their inhibitors.  The objective of this review is to present the most recent research on the cancer-specific role of glycolytic enzymes including their non-glycolytic functions in order to explore the potential for therapeutic opportunities.  Further, we discuss the translational potential of emerging drug candidates in light of tech. advances in treatment modalities such as image-guided targeted delivery of cancer therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoeWLa8w-XyRrVg90H21EOLACvtfcHk0ljj5xgWKKvzhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmsVyjtLs%253D&md5=1cd29873d9d8b8059aa75f5343df3f28</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1186%2F1476-4598-12-152&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1476-4598-12-152%26sid%3Dliteratum%253Aachs%26aulast%3DGanapathy-Kanniappan%26aufirst%3DS.%26aulast%3DGeschwind%26aufirst%3DJ.%2BF.%26atitle%3DTumor%2520Glycolysis%2520as%2520a%2520Target%2520for%2520Cancer%2520Therapy%253A%2520Progress%2520and%2520Prospects%26jtitle%3DMol.%2520Cancer%26date%3D2013%26volume%3D12%26spage%3D152%26doi%3D10.1186%2F1476-4598-12-152" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gupta, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wellen, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mazurek, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamezai, R. N.</span></span> <span> </span><span class="NLM_article-title">Pyruvate Kinase M2: Regulatory Circuits and Potential for Therapeutic Intervention</span>. <i>Curr. Pharm. Des.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">2595</span>– <span class="NLM_lpage">2606</span>, <span class="refDoi"> DOI: 10.2174/13816128113199990484</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=10.2174%2F13816128113199990484" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=23859618" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=1%3ACAS%3A528%3ADC%252BC2cXpt1Wmurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2014&pages=2595-2606&author=V.+Guptaauthor=K.+E.+Wellenauthor=S.+Mazurekauthor=R.+N.+Bamezai&title=Pyruvate+Kinase+M2%3A+Regulatory+Circuits+and+Potential+for+Therapeutic+Intervention&doi=10.2174%2F13816128113199990484"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Pyruvate Kinase M2: Regulatory Circuits and Potential for Therapeutic Intervention</span></div><div class="casAuthors">Gupta, Vibhor; Wellen, Kathryn E.; Mazurek, Sybille; Bamezai, Rameshwar N. K.</div><div class="citationInfo"><span class="NLM_cas:title">Current Pharmaceutical Design</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">2595-2606</span>CODEN:
                <span class="NLM_cas:coden">CPDEFP</span>;
        ISSN:<span class="NLM_cas:issn">1381-6128</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Cancer cells are characterized by reprogramming of energy metab.  Over the last decade, understanding of the metabolic changes that occur in cancer has increased dramatically, with great interest in targeting metab. for cancer therapy.  Pyruvate kinase isoenzyme type M2 (abbreviations: PKM2, M2-PK) plays a key role in modulating glucose metab. to support cell proliferation.  PKM2, like other PK isoforms, catalyzes the last energy-generating step in glycolysis, but is unique in its capacity to be regulated.  PKM2 is regulated at several cellular levels, including gene expression, alternative splicing and post-translational modification.  In addn., PKM2 is regulated by key metabolic intermediates and interacts with more than twenty different proteins.  Hence, this isoenzyme is an important regulator of glycolysis, and addnl. functions in other novel roles that have recently emerged.  Recent evidence indicates that intervening with the complex regulatory network of PKM2 has severe consequences on tumor cell proliferation, indicating the potential of this enzyme as a target for tumor therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohFL-uHUY_mrVg90H21EOLACvtfcHk0lhmbwO9F5kFgQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXpt1Wmurg%253D&md5=cc9d6d0bf6050acf9038f4a8c429216e</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.2174%2F13816128113199990484&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F13816128113199990484%26sid%3Dliteratum%253Aachs%26aulast%3DGupta%26aufirst%3DV.%26aulast%3DWellen%26aufirst%3DK.%2BE.%26aulast%3DMazurek%26aufirst%3DS.%26aulast%3DBamezai%26aufirst%3DR.%2BN.%26atitle%3DPyruvate%2520Kinase%2520M2%253A%2520Regulatory%2520Circuits%2520and%2520Potential%2520for%2520Therapeutic%2520Intervention%26jtitle%3DCurr.%2520Pharm.%2520Des.%26date%3D2014%26volume%3D20%26spage%3D2595%26epage%3D2606%26doi%3D10.2174%2F13816128113199990484" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Luo, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Semenza, G. L.</span></span> <span> </span><span class="NLM_article-title">Emerging Roles of Pkm2 in Cell Metabolism and Cancer Progression</span>. <i>Trends Endocrinol. Metab.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">560</span>– <span class="NLM_lpage">566</span>, <span class="refDoi"> DOI: 10.1016/j.tem.2012.06.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=10.1016%2Fj.tem.2012.06.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=22824010" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtVOqu7fI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2012&pages=560-566&author=W.+Luoauthor=G.+L.+Semenza&title=Emerging+Roles+of+Pkm2+in+Cell+Metabolism+and+Cancer+Progression&doi=10.1016%2Fj.tem.2012.06.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Emerging roles of PKM2 in cell metabolism and cancer progression</span></div><div class="casAuthors">Luo, Weibo; Semenza, Gregg L.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Endocrinology and Metabolism</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">560-566</span>CODEN:
                <span class="NLM_cas:coden">TENME4</span>;
        ISSN:<span class="NLM_cas:issn">1043-2760</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Increased conversion of glucose to lactate is a key feature of many cancer cells that promotes rapid growth.  Pyruvate kinase M2 (PKM2) expression is increased and facilitates lactate prodn. in cancer cells.  Modulation of PKM2 catalytic activity also regulates the synthesis of DNA and lipids that are required for cell proliferation, and of NADPH that is required for redox homeostasis.  In addn. to its role as a pyruvate kinase, PKM2 also functions as a protein kinase and as a transcriptional coactivator.  These biochem. activities are controlled by allosteric regulators and post-translational modifications of PKM2 that include acetylation, oxidn., phosphorylation, prolyl hydroxylation, and sumoylation.  Given its pleiotropic effects on cancer biol., PKM2 represents an attractive target for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXplJG_hQB2rVg90H21EOLACvtfcHk0lhmbwO9F5kFgQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtVOqu7fI&md5=2ae54658b3b6c05490de21b9b4310764</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.tem.2012.06.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tem.2012.06.010%26sid%3Dliteratum%253Aachs%26aulast%3DLuo%26aufirst%3DW.%26aulast%3DSemenza%26aufirst%3DG.%2BL.%26atitle%3DEmerging%2520Roles%2520of%2520Pkm2%2520in%2520Cell%2520Metabolism%2520and%2520Cancer%2520Progression%26jtitle%3DTrends%2520Endocrinol.%2520Metab.%26date%3D2012%26volume%3D23%26spage%3D560%26epage%3D566%26doi%3D10.1016%2Fj.tem.2012.06.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vander
Heiden, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christofk, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuman, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Subtelny, A. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharfi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harlow, E. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xian, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantley, L. C.</span></span> <span> </span><span class="NLM_article-title">Identification of Small Molecule Inhibitors of Pyruvate Kinase M2</span>. <i>Biochem. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>79</i></span>,  <span class="NLM_fpage">1118</span>– <span class="NLM_lpage">1124</span>, <span class="refDoi"> DOI: 10.1016/j.bcp.2009.12.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=10.1016%2Fj.bcp.2009.12.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=20005212" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=1%3ACAS%3A528%3ADC%252BC3cXitVOhsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=2010&pages=1118-1124&author=M.+G.+Vander%0AHeidenauthor=H.+R.+Christofkauthor=E.+Schumanauthor=A.+O.+Subtelnyauthor=H.+Sharfiauthor=E.+E.+Harlowauthor=J.+Xianauthor=L.+C.+Cantley&title=Identification+of+Small+Molecule+Inhibitors+of+Pyruvate+Kinase+M2&doi=10.1016%2Fj.bcp.2009.12.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of small molecule inhibitors of pyruvate kinase M2</span></div><div class="casAuthors">Vander Heiden, Matthew G.; Christofk, Heather R.; Schuman, Eli; Subtelny, Alexander O.; Sharfi, Hadar; Harlow, Edward E.; Xian, Jun; Cantley, Lewis C.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Pharmacology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">79</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1118-1124</span>CODEN:
                <span class="NLM_cas:coden">BCPCA6</span>;
        ISSN:<span class="NLM_cas:issn">0006-2952</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A common feature of tumors arising from diverse tissue types is a reliance on aerobic glycolysis for glucose metab.  This metabolic difference between cancer cells and normal cells could be exploited for therapeutic benefit in patients.  Cancer cells universally express the M2 isoform of the glycolytic enzyme pyruvate kinase (PKM2), and previous work has demonstrated that PKM2 expression is necessary for aerobic glycolysis and cell proliferation in vivo.  Because most normal tissues express an isoform of pyruvate kinase other than PKM2, selective targeting of PKM2 provides an opportunity to target cell metab. for cancer therapy.  PKM2 has an identical catalytic site as the related M1 splice variant (PKM1).  However, isoform selective inhibition is possible as PKM2 contains a unique region for allosteric regulation.  We have screened a library of greater than 1,00,000 small mols. to identify such inhibitors.  The inhibitors identified for PKM2 fell primarily into three distinct structural classes.  The most potent PKM2 inhibitor resulted in decreased glycolysis and increased cell death following loss of growth factor signaling.  At least part of this effect was due to on-target PKM2 inhibition as less cell death was obsd. in cells engineered to express PKM1.  These data suggest that isoform selective inhibition of PKM2 with small mols. is feasible and support the hypothesis that inhibition of glucose metab. in cancer cells is a viable strategy to treat human malignancy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHtS-udaOUobVg90H21EOLACvtfcHk0lhmbwO9F5kFgQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXitVOhsbw%253D&md5=efe6d924601716d806f50de3c231c995</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.bcp.2009.12.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bcp.2009.12.003%26sid%3Dliteratum%253Aachs%26aulast%3DVander%2BHeiden%26aufirst%3DM.%2BG.%26aulast%3DChristofk%26aufirst%3DH.%2BR.%26aulast%3DSchuman%26aufirst%3DE.%26aulast%3DSubtelny%26aufirst%3DA.%2BO.%26aulast%3DSharfi%26aufirst%3DH.%26aulast%3DHarlow%26aufirst%3DE.%2BE.%26aulast%3DXian%26aufirst%3DJ.%26aulast%3DCantley%26aufirst%3DL.%2BC.%26atitle%3DIdentification%2520of%2520Small%2520Molecule%2520Inhibitors%2520of%2520Pyruvate%2520Kinase%2520M2%26jtitle%3DBiochem.%2520Pharmacol.%26date%3D2010%26volume%3D79%26spage%3D1118%26epage%3D1124%26doi%3D10.1016%2Fj.bcp.2009.12.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, X.</span></span> <span> </span><span class="NLM_article-title">Shikonin and Its Analogs Inhibit Cancer Cell Glycolysis by Targeting Tumor Pyruvate Kinase-M2</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">4297</span>– <span class="NLM_lpage">4306</span>, <span class="refDoi"> DOI: 10.1038/onc.2011.137</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=10.1038%2Fonc.2011.137" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=21516121" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=1%3ACAS%3A528%3ADC%252BC3MXltFyqur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2011&pages=4297-4306&author=J.+Chenauthor=J.+Xieauthor=Z.+Jiangauthor=B.+Wangauthor=Y.+Wangauthor=X.+Hu&title=Shikonin+and+Its+Analogs+Inhibit+Cancer+Cell+Glycolysis+by+Targeting+Tumor+Pyruvate+Kinase-M2&doi=10.1038%2Fonc.2011.137"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Shikonin and its analogs inhibit cancer cell glycolysis by targeting tumor pyruvate kinase-M2</span></div><div class="casAuthors">Chen, J.; Xie, J.; Jiang, Z.; Wang, B.; Wang, Y.; Hu, X.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">42</span>),
    <span class="NLM_cas:pages">4297-4306</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">We recently reported that shikonin and its analogs were a class of necroptotic inducers that could bypass cancer drug resistance.  However, the mol. targets of shikonin are not known.  Here, we showed that shikonin and its analogs are inhibitors of tumor-specific pyruvate kinase-M2 (PKM2), among which shikonin and its enantiomeric isomer alkannin were the most potent and showed promising selectivity, i.e., shikonin and alkannin at concns. that resulted in over 50% inhibition of PKM2 activity did not inhibit PKM1 and pyruvate kinase-L (PKL).  Shikonin and alkannin significantly inhibited the glycolytic rate, as manifested by cellular lactate prodn. and glucose consumption in drug-sensitive and resistant cancer cell lines (MCF-7, MCF-7/Adr, MCF-7/Bcl-2, MCF-7/Bcl-xL and A549) that primarily express PKM2.  HeLa cells transfected with PKM1 showed reduced sensitivity to shikonin- or alkannin-induced cell death.  To the best of our knowledge, shikonin and alkannin are the most potent and specific inhibitors to PKM2 reported so far.  As PKM2 universally expresses in cancer cells and dictates the last rate-limiting step of glycolysis vital for cancer cell proliferation and survival, enantiomeric shikonin and alkannin may have potential in future clin. application.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdGlWk1wpLfLVg90H21EOLACvtfcHk0lhmbwO9F5kFgQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXltFyqur8%253D&md5=d4fd7186fc4e0df8578dd21add883f7b</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1038%2Fonc.2011.137&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fonc.2011.137%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DXie%26aufirst%3DJ.%26aulast%3DJiang%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DB.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DHu%26aufirst%3DX.%26atitle%3DShikonin%2520and%2520Its%2520Analogs%2520Inhibit%2520Cancer%2520Cell%2520Glycolysis%2520by%2520Targeting%2520Tumor%2520Pyruvate%2520Kinase-M2%26jtitle%3DOncogene%26date%3D2011%26volume%3D30%26spage%3D4297%26epage%3D4306%26doi%3D10.1038%2Fonc.2011.137" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ning, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNutt, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, Y.</span></span> <span> </span><span class="NLM_article-title">Discovery of Novel Naphthoquinone Derivatives as Inhibitors of the Tumor Cell Specific M2 Isoform of Pyruvate Kinase</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>138</i></span>,  <span class="NLM_fpage">343</span>– <span class="NLM_lpage">352</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2017.06.064</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=10.1016%2Fj.ejmech.2017.06.064" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=28688274" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtFKku77E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=138&publication_year=2017&pages=343-352&author=X.+Ningauthor=H.+Qiauthor=R.+Liauthor=Y.+Liauthor=Y.+Jinauthor=M.+A.+McNuttauthor=J.+Liuauthor=Y.+Yin&title=Discovery+of+Novel+Naphthoquinone+Derivatives+as+Inhibitors+of+the+Tumor+Cell+Specific+M2+Isoform+of+Pyruvate+Kinase&doi=10.1016%2Fj.ejmech.2017.06.064"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of novel naphthoquinone derivatives as inhibitors of the tumor cell specific M2 isoform of pyruvate kinase</span></div><div class="casAuthors">Ning, Xianling; Qi, Hailong; Li, Ridong; Li, Yunqiao; Jin, Yan; McNutt, Michael A.; Liu, Junyi; Yin, Yuxin</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">138</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">343-352</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Pyruvate kinase M2 (PKM2) is a rate-limiting enzyme of the glycolytic pathway which is highly expressed in cancer cells.  Cancer cells rely heavily on PKM2 for anabolic and energy requirements, and specific targeting of PKM2 therefore has potential as strategy for cancer therapy.  Here, the authors report the synthesis and biol. evaluation of novel naphthoquinone derivs. as selective small mol. inhibitors of PKM2.  Some target compds., such as compd. piperidine-1-carbodithioic acid 3-methyl-1,4-dioxo-1,4-dihydro-naphthalen-2-ylmethyl ester, displayed more potent PKM2 inhibitory activity than the reported optimal PKM2 inhibitor shikonin.  The well performing compd. piperidine-1-carbodithioic acid 3-methyl-1,4-dioxo-1,4-dihydro-naphthalen-2-ylmethyl ester also showed nanomolar antiproliferative activity toward a series of cancer cell lines with high expression of PKM2 including HCT116, Hela and H1299 with IC50 values ranging from 0.18 to 1.56 μM.  Moreover, compd. piperidine-1-carbodithioic acid 3-methyl-1,4-dioxo-1,4-dihydro-naphthalen-2-ylmethyl ester exhibited more cytotoxicity on cancer cells than normal cells.  The identification of novel potent small mol. inhibitors of PKM2 not only offers candidate compds. for cancer therapy, but also provides a tool with which to evaluate the function of PKM2 in depth.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp87KANg1TeAbVg90H21EOLACvtfcHk0lg9Q2hhODcupw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtFKku77E&md5=461ba2cd59b0bb2f49da7dfc1851dadb</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2017.06.064&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2017.06.064%26sid%3Dliteratum%253Aachs%26aulast%3DNing%26aufirst%3DX.%26aulast%3DQi%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DR.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DJin%26aufirst%3DY.%26aulast%3DMcNutt%26aufirst%3DM.%2BA.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DYin%26aufirst%3DY.%26atitle%3DDiscovery%2520of%2520Novel%2520Naphthoquinone%2520Derivatives%2520as%2520Inhibitors%2520of%2520the%2520Tumor%2520Cell%2520Specific%2520M2%2520Isoform%2520of%2520Pyruvate%2520Kinase%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2017%26volume%3D138%26spage%3D343%26epage%3D352%26doi%3D10.1016%2Fj.ejmech.2017.06.064" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boxer, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vander Heiden, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skoumbourdis, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Southall, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Veith, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leister, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Austin, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, H. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inglese, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantley, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Auld, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, C. J.</span></span> <span> </span><span class="NLM_article-title">Evaluation of Substituted N,N′-Diarylsulfonamides as Activators of the Tumor Cell Specific M2 Isoform of Pyruvate Kinase</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">1048</span>– <span class="NLM_lpage">1055</span>, <span class="refDoi"> DOI: 10.1021/jm901577g</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901577g" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsFKgurnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=1048-1055&author=M.+B.+Boxerauthor=J.+K.+Jiangauthor=M.+G.+Vander+Heidenauthor=M.+Shenauthor=A.+P.+Skoumbourdisauthor=N.+Southallauthor=H.+Veithauthor=W.+Leisterauthor=C.+P.+Austinauthor=H.+W.+Parkauthor=J.+Ingleseauthor=L.+C.+Cantleyauthor=D.+S.+Auldauthor=C.+J.+Thomas&title=Evaluation+of+Substituted+N%2CN%E2%80%B2-Diarylsulfonamides+as+Activators+of+the+Tumor+Cell+Specific+M2+Isoform+of+Pyruvate+Kinase&doi=10.1021%2Fjm901577g"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Evaluation of Substituted N,N'-Diarylsulfonamides as Activators of the Tumor Cell Specific M2 Isoform of Pyruvate Kinase</span></div><div class="casAuthors">Boxer, Matthew B.; Jiang, Jian-kang; Vander Heiden, Matthew G.; Shen, Min; Skoumbourdis, Amanda P.; Southall, Noel; Veith, Henrike; Leister, William; Austin, Christopher P.; Park, Hee Won; Inglese, James; Cantley, Lewis C.; Auld, Douglas S.; Thomas, Craig J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1048-1055</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The metab. of cancer cells is altered to support rapid proliferation.  Pharmacol. activators of a tumor cell specific pyruvate kinase isoenzyme (PKM2) may be an approach for altering the classic Warburg effect characteristic of aberrant metab. in cancer cells yielding a novel antiproliferation strategy.  In this manuscript, we detail the discovery of a series of substituted N,N'-diarylsulfonamides as activators of PKM2.  The synthesis of numerous analogs and the evaluation of structure-activity relationships are presented as well as assessments of mechanism and selectivity.  Several agents are found that have good potencies and appropriate soly. for use as chem. probes of PKM2 including 55 (AC50 = 43 nM, max. response = 84%; soly. = 7.3 μg/mL), 56 (AC50 = 99 nM, max. response = 84%; soly. = 5.7 μg/mL), and 58 (AC50 = 38 nM, max. response = 82%; soly. = 51.2 μg/mL).  The small mols. described here represent first-in-class activators of PKM2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq4C0nWknRHC7Vg90H21EOLACvtfcHk0lg9Q2hhODcupw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsFKgurnF&md5=006f6860e53fa40338899076049ebbe7</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fjm901577g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901577g%26sid%3Dliteratum%253Aachs%26aulast%3DBoxer%26aufirst%3DM.%2BB.%26aulast%3DJiang%26aufirst%3DJ.%2BK.%26aulast%3DVander%2BHeiden%26aufirst%3DM.%2BG.%26aulast%3DShen%26aufirst%3DM.%26aulast%3DSkoumbourdis%26aufirst%3DA.%2BP.%26aulast%3DSouthall%26aufirst%3DN.%26aulast%3DVeith%26aufirst%3DH.%26aulast%3DLeister%26aufirst%3DW.%26aulast%3DAustin%26aufirst%3DC.%2BP.%26aulast%3DPark%26aufirst%3DH.%2BW.%26aulast%3DInglese%26aufirst%3DJ.%26aulast%3DCantley%26aufirst%3DL.%2BC.%26aulast%3DAuld%26aufirst%3DD.%2BS.%26aulast%3DThomas%26aufirst%3DC.%2BJ.%26atitle%3DEvaluation%2520of%2520Substituted%2520N%252CN%25E2%2580%25B2-Diarylsulfonamides%2520as%2520Activators%2520of%2520the%2520Tumor%2520Cell%2520Specific%2520M2%2520Isoform%2520of%2520Pyruvate%2520Kinase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D1048%26epage%3D1055%26doi%3D10.1021%2Fjm901577g" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boxer, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vander Heiden, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skoumbourdis, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Southall, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Veith, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leister, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Austin, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, H. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inglese, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantley, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Auld, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, C. J.</span></span> <span> </span><span class="NLM_article-title">Evaluation of Thieno[3,2-B]Pyrrole[3,2-D]Pyridazinones as Activators of the Tumor Cell Specific M2 Isoform of Pyruvate Kinase</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">3387</span>– <span class="NLM_lpage">3393</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2010.04.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=10.1016%2Fj.bmcl.2010.04.015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=20451379" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=1%3ACAS%3A528%3ADC%252BC3cXmsVOhur4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2010&pages=3387-3393&author=J.+K.+Jiangauthor=M.+B.+Boxerauthor=M.+G.+Vander+Heidenauthor=M.+Shenauthor=A.+P.+Skoumbourdisauthor=N.+Southallauthor=H.+Veithauthor=W.+Leisterauthor=C.+P.+Austinauthor=H.+W.+Parkauthor=J.+Ingleseauthor=L.+C.+Cantleyauthor=D.+S.+Auldauthor=C.+J.+Thomas&title=Evaluation+of+Thieno%5B3%2C2-B%5DPyrrole%5B3%2C2-D%5DPyridazinones+as+Activators+of+the+Tumor+Cell+Specific+M2+Isoform+of+Pyruvate+Kinase&doi=10.1016%2Fj.bmcl.2010.04.015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Evaluation of thieno[3,2-b]pyrrole[3,2-d]pyridazinones as activators of the tumor cell specific M2 isoform of pyruvate kinase</span></div><div class="casAuthors">Jiang, Jian-kang; Boxer, Matthew B.; Vander Heiden, Matthew G.; Shen, Min; Skoumbourdis, Amanda P.; Southall, Noel; Veith, Henrike; Leister, William; Austin, Christopher P.; Park, Hee Won; Inglese, James; Cantley, Lewis C.; Auld, Douglas S.; Thomas, Craig J.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">3387-3393</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Cancer cells have distinct metabolic needs that are different from normal cells and can be exploited for development of anti-cancer therapeutics.  Activation of the tumor specific M2 form of pyruvate kinase (PKM2) is a potential strategy for returning cancer cells to a metabolic state characteristic of normal cells.  A series of activators of PKM2, e.g., I, based upon a substituted thieno[3,2-b]pyrrole[3,2-d]pyridazinone scaffold were prepd.  The synthesis of these agents, structure-activity relationships, anal. of activity at related targets (PKM1, PKR and PKL) and examn. of aq. soly. were investigated.  These agents represent the second reported chemotype for activation of PKM2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5vYZSvemjsLVg90H21EOLACvtfcHk0lg6z2PHBwjW5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXmsVOhur4%253D&md5=99a1753511b6643d9ac2fd7ca15da5ff</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2010.04.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2010.04.015%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DJ.%2BK.%26aulast%3DBoxer%26aufirst%3DM.%2BB.%26aulast%3DVander%2BHeiden%26aufirst%3DM.%2BG.%26aulast%3DShen%26aufirst%3DM.%26aulast%3DSkoumbourdis%26aufirst%3DA.%2BP.%26aulast%3DSouthall%26aufirst%3DN.%26aulast%3DVeith%26aufirst%3DH.%26aulast%3DLeister%26aufirst%3DW.%26aulast%3DAustin%26aufirst%3DC.%2BP.%26aulast%3DPark%26aufirst%3DH.%2BW.%26aulast%3DInglese%26aufirst%3DJ.%26aulast%3DCantley%26aufirst%3DL.%2BC.%26aulast%3DAuld%26aufirst%3DD.%2BS.%26aulast%3DThomas%26aufirst%3DC.%2BJ.%26atitle%3DEvaluation%2520of%2520Thieno%255B3%252C2-B%255DPyrrole%255B3%252C2-D%255DPyridazinones%2520as%2520Activators%2520of%2520the%2520Tumor%2520Cell%2520Specific%2520M2%2520Isoform%2520of%2520Pyruvate%2520Kinase%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2010%26volume%3D20%26spage%3D3387%26epage%3D3393%26doi%3D10.1016%2Fj.bmcl.2010.04.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Walsh, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brimacombe, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Veith, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bougie, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daniel, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leister, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantley, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Israelsen, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vander Heiden, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Auld, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boxer, M. B.</span></span> <span> </span><span class="NLM_article-title">2-Oxo-N-Aryl-1,2,3,4-Tetrahydroquinoline-6-Sulfonamides as Activators of the Tumor Cell Specific M2 Isoform of Pyruvate Kinase</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">6322</span>– <span class="NLM_lpage">6327</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2011.08.114</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=10.1016%2Fj.bmcl.2011.08.114" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=21958545" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht12jtr7K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2011&pages=6322-6327&author=M.+J.+Walshauthor=K.+R.+Brimacombeauthor=H.+Veithauthor=J.+M.+Bougieauthor=T.+Danielauthor=W.+Leisterauthor=L.+C.+Cantleyauthor=W.+J.+Israelsenauthor=M.+G.+Vander+Heidenauthor=M.+Shenauthor=D.+S.+Auldauthor=C.+J.+Thomasauthor=M.+B.+Boxer&title=2-Oxo-N-Aryl-1%2C2%2C3%2C4-Tetrahydroquinoline-6-Sulfonamides+as+Activators+of+the+Tumor+Cell+Specific+M2+Isoform+of+Pyruvate+Kinase&doi=10.1016%2Fj.bmcl.2011.08.114"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">2-Oxo-N-aryl-1,2,3,4-tetrahydroquinoline-6-sulfonamides as activators of the tumor cell specific M2 isoform of pyruvate kinase</span></div><div class="casAuthors">Walsh, Martin J.; Brimacombe, Kyle R.; Veith, Henrike; Bougie, James M.; Daniel, Thomas; Leister, William; Cantley, Lewis C.; Israelsen, William J.; Vander Heiden, Matthew G.; Shen, Min; Auld, Douglas S.; Thomas, Craig J.; Boxer, Matthew B.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">6322-6327</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Compared to normal differentiated cells, cancer cells have altered metabolic regulation to support biosynthesis and the expression of the M2 isoenzyme of pyruvate kinase (PKM2) plays an important role in this anabolic metab.  While the M1 isoform is a highly active enzyme, the alternatively spliced M2 variant is considerably less active and expressed in tumors.  While the exact mechanism by which decreased pyruvate kinase activity contributes to anabolic metab. remains unclear, it is hypothesized that activation of PKM2 to levels seen with PKM1 may promote a metabolic program that is not conducive to cell proliferation.  Here we report the third chemotype in a series of PKM2 activators based on the 2-oxo-N-aryl-1,2,3,4-tetrahydroquinoline-6-sulfonamide scaffold.  The synthesis, structure activity relationships, selectivity and notable physiochem. properties are described.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv4lcw1kll2bVg90H21EOLACvtfcHk0lg6z2PHBwjW5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht12jtr7K&md5=82971c401c538f105d4c9144837ec175</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2011.08.114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2011.08.114%26sid%3Dliteratum%253Aachs%26aulast%3DWalsh%26aufirst%3DM.%2BJ.%26aulast%3DBrimacombe%26aufirst%3DK.%2BR.%26aulast%3DVeith%26aufirst%3DH.%26aulast%3DBougie%26aufirst%3DJ.%2BM.%26aulast%3DDaniel%26aufirst%3DT.%26aulast%3DLeister%26aufirst%3DW.%26aulast%3DCantley%26aufirst%3DL.%2BC.%26aulast%3DIsraelsen%26aufirst%3DW.%2BJ.%26aulast%3DVander%2BHeiden%26aufirst%3DM.%2BG.%26aulast%3DShen%26aufirst%3DM.%26aulast%3DAuld%26aufirst%3DD.%2BS.%26aulast%3DThomas%26aufirst%3DC.%2BJ.%26aulast%3DBoxer%26aufirst%3DM.%2BB.%26atitle%3D2-Oxo-N-Aryl-1%252C2%252C3%252C4-Tetrahydroquinoline-6-Sulfonamides%2520as%2520Activators%2520of%2520the%2520Tumor%2520Cell%2520Specific%2520M2%2520Isoform%2520of%2520Pyruvate%2520Kinase%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2011%26volume%3D21%26spage%3D6322%26epage%3D6327%26doi%3D10.1016%2Fj.bmcl.2011.08.114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xue, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ivanov, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span> <span> </span><span class="NLM_article-title">Natural Product Micheliolide (Mcl) Irreversibly Activates Pyruvate Kinase M2 and Suppresses Leukemia</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">4155</span>– <span class="NLM_lpage">4164</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00241</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00241" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=1%3ACAS%3A528%3ADC%252BC1cXnsFSguro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=4155-4164&author=J.+Liauthor=S.+Liauthor=J.+Guoauthor=Q.+Liauthor=J.+Longauthor=C.+Maauthor=Y.+Dingauthor=C.+Yanauthor=L.+Liauthor=Z.+Wuauthor=H.+Zhuauthor=K.+K.+Liauthor=L.+Wenauthor=Q.+Zhangauthor=Q.+Xueauthor=C.+Zhaoauthor=N.+Liuauthor=I.+Ivanovauthor=M.+Luoauthor=R.+Xiauthor=H.+Longauthor=P.+G.+Wangauthor=Y.+Chen&title=Natural+Product+Micheliolide+%28Mcl%29+Irreversibly+Activates+Pyruvate+Kinase+M2+and+Suppresses+Leukemia&doi=10.1021%2Facs.jmedchem.8b00241"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Natural Product Micheliolide (MCL) Irreversibly Activates Pyruvate Kinase M2 and Suppresses Leukemia</span></div><div class="casAuthors">Li, Jing; Li, Shanshan; Guo, Jianshuang; Li, Qiuying; Long, Jing; Ma, Cheng; Ding, Yahui; Yan, Chunli; Li, Liangwei; Wu, Zhigang; Zhu, He; Li, Keqin Kathy; Wen, Liuqing; Zhang, Quan; Xue, Qingqing; Zhao, Caili; Liu, Ning; Ivanov, Ivaylo; Luo, Ming; Xi, Rimo; Long, Haibo; Wang, Peng George; Chen, Yue</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4155-4164</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Metabolic reprogramming of cancer cells is essential for tumorigenesis in which pyruvate kinase M2 (PKM2), the low activity isoform of pyruvate kinase, plays a crit. role.  Herein, we describe the identification of a nature-product-derived micheliolide (MCL) that selectively activates PKM2 through the covalent binding at residue cysteine424 (C424), which is not contained in PKM1.  This interaction promotes more tetramer formation, inhibits the lysine433 (K433) acetylation, and influences the translocation of PKM2 into the nucleus.  In addn., the pro-drug dimethylaminomicheliolide (DMAMCL) with similar properties as MCL significantly suppresses the growth of leukemia cells and tumorigenesis in a zebrafish xenograft model.  Cell-based assay with knock down PKM2 expression verifies that the effects of MCL are dependent on PKM2 expression.  DMAMCL is currently in clin. trials in Australia.  Our discovery may provide a valuable pharmacol. mechanism for clin. treatment and benefit the development of new anticancer agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHlWteclCMHbVg90H21EOLACvtfcHk0lg6z2PHBwjW5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXnsFSguro%253D&md5=8742015134a65572dee296a5e88fa861</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00241&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00241%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DGuo%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DQ.%26aulast%3DLong%26aufirst%3DJ.%26aulast%3DMa%26aufirst%3DC.%26aulast%3DDing%26aufirst%3DY.%26aulast%3DYan%26aufirst%3DC.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DWu%26aufirst%3DZ.%26aulast%3DZhu%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DK.%2BK.%26aulast%3DWen%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DXue%26aufirst%3DQ.%26aulast%3DZhao%26aufirst%3DC.%26aulast%3DLiu%26aufirst%3DN.%26aulast%3DIvanov%26aufirst%3DI.%26aulast%3DLuo%26aufirst%3DM.%26aulast%3DXi%26aufirst%3DR.%26aulast%3DLong%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DP.%2BG.%26aulast%3DChen%26aufirst%3DY.%26atitle%3DNatural%2520Product%2520Micheliolide%2520%2528Mcl%2529%2520Irreversibly%2520Activates%2520Pyruvate%2520Kinase%2520M2%2520and%2520Suppresses%2520Leukemia%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D4155%26epage%3D4164%26doi%3D10.1021%2Facs.jmedchem.8b00241" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsu, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chou, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chuang, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, L. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kulp, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C. S.</span></span> <span> </span><span class="NLM_article-title">Identification and Characterization of a Novel Integrin-Linked Kinase Inhibitor</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">6364</span>– <span class="NLM_lpage">6374</span>, <span class="refDoi"> DOI: 10.1021/jm2007744</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm2007744" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtVOns7rK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=6364-6374&author=S.+L.+Leeauthor=E.+C.+Hsuauthor=C.+C.+Chouauthor=H.+C.+Chuangauthor=L.+Y.+Baiauthor=S.+K.+Kulpauthor=C.+S.+Chen&title=Identification+and+Characterization+of+a+Novel+Integrin-Linked+Kinase+Inhibitor&doi=10.1021%2Fjm2007744"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Identification and Characterization of a Novel Integrin-Linked Kinase Inhibitor</span></div><div class="casAuthors">Lee, Su-Lin; Hsu, En-Chi; Chou, Chih-Chien; Chuang, Hsiao-Ching; Bai, Li-Yuan; Kulp, Samuel K.; Chen, Ching-Shih</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">6364-6374</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Integrin-linked kinase (ILK) represents a relevant target for cancer therapy in light of its role in promoting oncogenesis and tumor progression.  Through the screening of an inhouse focused compd. library, we identified N-methyl-3-(1-(4-(piperazin-1-yl)phenyl)-5-(4'-(trifluoromethyl)-[1,1'-biphenyl]-4-yl)-1H-pyrazol-3-yl)propanamide (I) as a novel ILK inhibitor (IC50, 0.6 μM), which exhibited high in vitro potency against a panel of prostate and breast cancer cell lines (IC50, 1-2.5 μM), while normal epithelial cells were unaffected.  I facilitated the dephosphorylation of Akt at Ser-473 and other ILK targets, including glycogen synthase kinase-3β and myosin light chain.  Moreover, I suppressed the expression of the transcription/translation factor YB-1 and its targets HER2 and EGFR in PC-3 cells, which could be rescued by the stable expression of constitutively active ILK.  Evidence indicates that I induced autophagy and apoptosis, both of which were integral to its antiproliferative activity.  Together, this broad spectrum of mechanisms underlies the therapeutic potential of I in cancer treatment, which is manifested by its in vivo efficacy as a single oral agent in suppressing PC-3 xenograft tumor growth.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpenckDD_XVxbVg90H21EOLACvtfcHk0ljo2b2ihp-uSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtVOns7rK&md5=44d060cd589c08abf58cc3e3fb6af542</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fjm2007744&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm2007744%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DS.%2BL.%26aulast%3DHsu%26aufirst%3DE.%2BC.%26aulast%3DChou%26aufirst%3DC.%2BC.%26aulast%3DChuang%26aufirst%3DH.%2BC.%26aulast%3DBai%26aufirst%3DL.%2BY.%26aulast%3DKulp%26aufirst%3DS.%2BK.%26aulast%3DChen%26aufirst%3DC.%2BS.%26atitle%3DIdentification%2520and%2520Characterization%2520of%2520a%2520Novel%2520Integrin-Linked%2520Kinase%2520Inhibitor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D6364%26epage%3D6374%26doi%3D10.1021%2Fjm2007744" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pike
Winer, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, M.</span></span> <span> </span><span class="NLM_article-title">Rapid Analysis of Glycolytic and Oxidative Substrate Flux of Cancer Cells in a Microplate</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">e109916</span>, <span class="refDoi"> DOI: 10.1371/journal.pone.0109916</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=10.1371%2Fjournal.pone.0109916" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=25360519" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFahsr%252FJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=e109916&author=L.+S.+Pike%0AWinerauthor=M.+Wu&title=Rapid+Analysis+of+Glycolytic+and+Oxidative+Substrate+Flux+of+Cancer+Cells+in+a+Microplate&doi=10.1371%2Fjournal.pone.0109916"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Rapid analysis of glycolytic and oxidative substrate flux of cancer cells in a microplate</span></div><div class="casAuthors">Pike Winer, Lisa S.; Wu, Min</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">e109916/1-e109916/14, 14 pp.</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Cancer cells exhibit remarkable alterations in cellular metab., particularly in their nutrient substrate preference.  We have devised several exptl. methods that rapidly analyze the metabolic substrate flux in cancer cells: glycolysis and the oxidn. of major fuel substrates glucose, glutamine, and fatty acids.  Using the XF Extracellular Flux analyzer, these methods measure, in real-time, the oxygen consumption rate OCR and extracellular acidification rate ECAR of living cells in a microplate as they respond to substrates and metabolic perturbation agents.  In proof-of-principle expts., we analyzed substrate flux and mitochondrial bioenergetics of two human glioblastoma cell lines, SF188s and SF188f, which were derived from the same parental cell line but proliferate at slow and fast rates, resp.  These analyses led to three interesting observations: (1) both cell lines respired effectively with substantial endogenous substrate respiration; (2) SF188f cells underwent a significant shift from glycolytic to oxidative metab., along with a high rate of glutamine oxidn. relative to SF188s cells; and (3) the mitochondrial proton leak-linked respiration of SF188f cells increased significantly compared to SF188s cells.  It is plausible that the proton leak of SF188f cells may play a role in allowing continuous glutamine-fueled anaplerotic TCA cycle flux by partially uncoupling the TCA cycle from oxidative phosphorylation.  Taken together, these rapid, sensitive and high-throughput substrate flux anal. methods introduce highly valuable approaches for developing a greater understanding of genetic and epigenetic pathways that regulate cellular metab., and the development of therapies that target cancer metab.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkMfX8T6XL47Vg90H21EOLACvtfcHk0ljo2b2ihp-uSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvFahsr%252FJ&md5=5c3143d1026c5658580067a21a467763</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0109916&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0109916%26sid%3Dliteratum%253Aachs%26aulast%3DPike%2BWiner%26aufirst%3DL.%2BS.%26aulast%3DWu%26aufirst%3DM.%26atitle%3DRapid%2520Analysis%2520of%2520Glycolytic%2520and%2520Oxidative%2520Substrate%2520Flux%2520of%2520Cancer%2520Cells%2520in%2520a%2520Microplate%26jtitle%3DPLoS%2520One%26date%3D2014%26volume%3D9%26spage%3De109916%26doi%3D10.1371%2Fjournal.pone.0109916" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, T. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, P. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsiao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hua, K. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuo, M. L.</span></span> <span> </span><span class="NLM_article-title">Cytosolic Pkm2 Stabilizes Mutant Egfr Protein Expression through Regulating Hsp90-Egfr Association</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">3387</span>– <span class="NLM_lpage">3398</span>, <span class="refDoi"> DOI: 10.1038/onc.2015.397</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=10.1038%2Fonc.2015.397" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=26500058" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslant7jK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2016&pages=3387-3398&author=Y.+C.+Yangauthor=T.+Y.+Chengauthor=S.+M.+Huangauthor=C.+Y.+Suauthor=P.+W.+Yangauthor=J.+M.+Leeauthor=C.+K.+Chenauthor=M.+Hsiaoauthor=K.+T.+Huaauthor=M.+L.+Kuo&title=Cytosolic+Pkm2+Stabilizes+Mutant+Egfr+Protein+Expression+through+Regulating+Hsp90-Egfr+Association&doi=10.1038%2Fonc.2015.397"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Cytosolic PKM2 stabilizes mutant EGFR protein expression through regulating HSP90-EGFR association</span></div><div class="casAuthors">Yang, Y.-C.; Cheng, T.-Y.; Huang, S.-M.; Su, C.-Y.; Yang, P.-W.; Lee, J.-M.; Chen, C.-K.; Hsiao, M.; Hua, K.-T.; Kuo, M.-L.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">3387-3398</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Secondary mutation of epidermal growth factor receptor (EGFR) resulting in drug resistance is one of the most crit. issues in lung cancer therapy.  Several drugs are being developed to overcome EGFR tyrosine kinase inhibitor (TKI) resistance.  Here, we report that pyruvate kinase M2 (PKM2) stabilized mutant EGFR protein by direct interaction and sustained cell survival signaling in lung cancer cells.  PKM2 silencing resulted in markedly reduced mutant EGFR expression in TKI-sensitive or -resistant human lung cancer cells, and in inhibition of tumor growth in their xenografts, concomitant with downregulation of EGFR-related signaling.  Mechanistically, PKM2 directly interacted with mutant EGFR and heat-shock protein 90 (HSP90), and thus stabilized EGFR by maintaining its binding with HSP90 and co-chaperones.  Stabilization of EGFR relied on dimeric PKM2, and the protein half-life of mutant EGFR decreased when PKM2 was forced into its tetramer form.  Clin. levels of PKM2 pos. correlated with mutant EGFR expression and with patient outcome.  These results reveal a previously undescribed non-glycolysis function of PKM2 in the cytoplasm, which contribute to EGFR-dependent tumorigenesis and provide a novel strategy to overcome drug resistance to EGFR TKIs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpNRWNlV7j1NLVg90H21EOLACvtfcHk0ljo2b2ihp-uSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslant7jK&md5=6ec476c07c325470d6d068b22e4e6724</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1038%2Fonc.2015.397&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fonc.2015.397%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DY.%2BC.%26aulast%3DCheng%26aufirst%3DT.%2BY.%26aulast%3DHuang%26aufirst%3DS.%2BM.%26aulast%3DSu%26aufirst%3DC.%2BY.%26aulast%3DYang%26aufirst%3DP.%2BW.%26aulast%3DLee%26aufirst%3DJ.%2BM.%26aulast%3DChen%26aufirst%3DC.%2BK.%26aulast%3DHsiao%26aufirst%3DM.%26aulast%3DHua%26aufirst%3DK.%2BT.%26aulast%3DKuo%26aufirst%3DM.%2BL.%26atitle%3DCytosolic%2520Pkm2%2520Stabilizes%2520Mutant%2520Egfr%2520Protein%2520Expression%2520through%2520Regulating%2520Hsp90-Egfr%2520Association%26jtitle%3DOncogene%26date%3D2016%26volume%3D35%26spage%3D3387%26epage%3D3398%26doi%3D10.1038%2Fonc.2015.397" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cui, S.</span></span> <span> </span><span class="NLM_article-title">Pkm2 Aggravates Palmitate-Induced Insulin Resistance in Hepg2 Cells Via Stat3 Pathway</span>. <i>Biochem. Biophys. Res. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>492</i></span>,  <span class="NLM_fpage">109</span>– <span class="NLM_lpage">115</span>, <span class="refDoi"> DOI: 10.1016/j.bbrc.2017.08.025</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=10.1016%2Fj.bbrc.2017.08.025" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=28802581" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtlWksL7L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=492&publication_year=2017&pages=109-115&author=L.+Chenauthor=Z.+Tangauthor=X.+Wangauthor=H.+Maauthor=D.+Shanauthor=S.+Cui&title=Pkm2+Aggravates+Palmitate-Induced+Insulin+Resistance+in+Hepg2+Cells+Via+Stat3+Pathway&doi=10.1016%2Fj.bbrc.2017.08.025"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">PKM2 aggravates palmitate-induced insulin resistance in HepG2 cells via STAT3 pathway</span></div><div class="casAuthors">Chen, Ling; Tang, Zhuqi; Wang, Xiaohua; Ma, Hong; Shan, Dandan; Cui, Shiwei</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical and Biophysical Research Communications</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">492</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">109-115</span>CODEN:
                <span class="NLM_cas:coden">BBRCA9</span>;
        ISSN:<span class="NLM_cas:issn">0006-291X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Studies have identified that PKM2 is related to the development of glucose intolerance and insulin resistance in rodents and humans.  However, the underlying mechanism remains largely unknown.  In the present study, we found that PKM2 expression was significantly elevated in insulin-resistant hepatic tissues and hepatocytes, implicating an assocn. between PKM2 expression and hepatic insulin resistance (IR).  In vitro study revealed that overexpression of PKM2 impaired the insulin signaling pathway by decreasing the phosphorylation of protein kinase B (Akt) and glycogen synthase kinase-3β (GSK3β).  Furthermore, PKM2 overexpression enhanced the effects of PA on the lipid accumulation, the expression of phosphoenolpyruvate carboxykinase (PEPCK) and glucose-6-phosphatase (G6Pase) and hepatic glucose uptake.  Intriguingly, PA-induced insulin resistance was suppressed following by the ablation of PKM2 in HepG2 cells.  We also found that STAT3 was significantly activated by PKM2 overexpression.  Moreover, we identified that PKM2 could interact directly with STAT3.  Taken together, these studies demonstrate that PKM2 may promote hepatic IR via STAT3 pathway and would provide a new insight into dissecting the mol. pathogenesis of hepatic insulin resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqatCr63ZliybVg90H21EOLACvtfcHk0ljeGOFbbX0ZWg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtlWksL7L&md5=ceb157d27d8fb14e623f6a8f83fe34d9</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2Fj.bbrc.2017.08.025&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbrc.2017.08.025%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DL.%26aulast%3DTang%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DMa%26aufirst%3DH.%26aulast%3DShan%26aufirst%3DD.%26aulast%3DCui%26aufirst%3DS.%26atitle%3DPkm2%2520Aggravates%2520Palmitate-Induced%2520Insulin%2520Resistance%2520in%2520Hepg2%2520Cells%2520Via%2520Stat3%2520Pathway%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D2017%26volume%3D492%26spage%3D109%26epage%3D115%26doi%3D10.1016%2Fj.bbrc.2017.08.025" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xia, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aldape, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, Z.</span></span> <span> </span><span class="NLM_article-title">Nuclear Pkm2 Regulates Beta-Catenin Transactivation Upon Egfr Activation</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>480</i></span>,  <span class="NLM_fpage">118</span>– <span class="NLM_lpage">122</span>, <span class="refDoi"> DOI: 10.1038/nature10598</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=10.1038%2Fnature10598" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=22056988" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsVagtbvP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=480&publication_year=2011&pages=118-122&author=W.+Yangauthor=Y.+Xiaauthor=H.+Jiauthor=Y.+Zhengauthor=J.+Liangauthor=W.+Huangauthor=X.+Gaoauthor=K.+Aldapeauthor=Z.+Lu&title=Nuclear+Pkm2+Regulates+Beta-Catenin+Transactivation+Upon+Egfr+Activation&doi=10.1038%2Fnature10598"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Nuclear PKM2 regulates β-catenin transactivation upon EGFR activation</span></div><div class="casAuthors">Yang, Weiwei; Xia, Yan; Ji, Haitao; Zheng, Yanhua; Liang, Ji; Huang, Wenhua; Gao, Xiang; Aldape, Kenneth; Lu, Zhimin</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">480</span>
        (<span class="NLM_cas:issue">7375</span>),
    <span class="NLM_cas:pages">118-122</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The embryonic pyruvate kinase M2 (PKM2) isoform is highly expressed in human cancer.  In contrast to the established role of PKM2 in aerobic glycolysis or the Warburg effect, its non-metabolic functions remain elusive.  Here we demonstrate, in human cancer cells, that epidermal growth factor receptor (EGFR) activation induces translocation of PKM2, but not PKM1, into the nucleus, where K433 of PKM2 binds to c-Src-phosphorylated Y333 of β-catenin.  This interaction is required for both proteins to be recruited to the CCND1 promoter, leading to HDAC3 removal from the promoter, histone H3 acetylation and cyclin D1 expression.  PKM2-dependent β-catenin transactivation is instrumental in EGFR-promoted tumor cell proliferation and brain tumor development.  In addn., pos. correlations have been identified between c-Src activity, β-catenin Y333 phosphorylation and PKM2 nuclear accumulation in human glioblastoma specimens.  Furthermore, levels of β-catenin phosphorylation and nuclear PKM2 have been correlated with grades of glioma malignancy and prognosis.  These findings reveal that EGF induces β-catenin transactivation via a mechanism distinct from that induced by Wnt/Wingless and highlight the essential non-metabolic functions of PKM2 in EGFR-promoted β-catenin transactivation, cell proliferation and tumorigenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqIoPIB2i0od7Vg90H21EOLACvtfcHk0ljGSUG8RA_3NQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsVagtbvP&md5=25b4d94c581fe862d12dfe1fd213ed06</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1038%2Fnature10598&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature10598%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DW.%26aulast%3DXia%26aufirst%3DY.%26aulast%3DJi%26aufirst%3DH.%26aulast%3DZheng%26aufirst%3DY.%26aulast%3DLiang%26aufirst%3DJ.%26aulast%3DHuang%26aufirst%3DW.%26aulast%3DGao%26aufirst%3DX.%26aulast%3DAldape%26aufirst%3DK.%26aulast%3DLu%26aufirst%3DZ.%26atitle%3DNuclear%2520Pkm2%2520Regulates%2520Beta-Catenin%2520Transactivation%2520Upon%2520Egfr%2520Activation%26jtitle%3DNature%26date%3D2011%26volume%3D480%26spage%3D118%26epage%3D122%26doi%3D10.1038%2Fnature10598" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Anastasiou, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poulogiannis, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asara, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boxer, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellinger, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sasaki, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Locasale, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Auld, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vander Heiden, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantley, L. C.</span></span> <span> </span><span class="NLM_article-title">Inhibition of Pyruvate Kinase M2 by Reactive Oxygen Species Contributes to Cellular Antioxidant Responses</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>334</i></span>,  <span class="NLM_fpage">1278</span>– <span class="NLM_lpage">1283</span>, <span class="refDoi"> DOI: 10.1126/science.1211485</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=10.1126%2Fscience.1211485" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=22052977" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsFCms7jK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=334&publication_year=2011&pages=1278-1283&author=D.+Anastasiouauthor=G.+Poulogiannisauthor=J.+M.+Asaraauthor=M.+B.+Boxerauthor=J.+K.+Jiangauthor=M.+Shenauthor=G.+Bellingerauthor=A.+T.+Sasakiauthor=J.+W.+Locasaleauthor=D.+S.+Auldauthor=C.+J.+Thomasauthor=M.+G.+Vander+Heidenauthor=L.+C.+Cantley&title=Inhibition+of+Pyruvate+Kinase+M2+by+Reactive+Oxygen+Species+Contributes+to+Cellular+Antioxidant+Responses&doi=10.1126%2Fscience.1211485"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of Pyruvate Kinase M2 by Reactive Oxygen Species Contributes to Cellular Antioxidant Responses</span></div><div class="casAuthors">Anastasiou, Dimitrios; Poulogiannis, George; Asara, John M.; Boxer, Matthew B.; Jiang, Jian-kang; Shen, Min; Bellinger, Gary; Sasaki, Atsuo T.; Locasale, Jason W.; Auld, Douglas S.; Thomas, Craig J.; Vander Heiden, Matthew G.; Cantley, Lewis C.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">334</span>
        (<span class="NLM_cas:issue">6060</span>),
    <span class="NLM_cas:pages">1278-1283</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Control of intracellular reactive oxygen species (ROS) concns. is crit. for cancer cell survival.  We show that, in human lung cancer cells, acute increases in intracellular concns. of ROS caused inhibition of the glycolytic enzyme pyruvate kinase M2 (PKM2) through oxidn. of Cys358.  This inhibition of PKM2 is required to divert glucose flux into the pentose phosphate pathway and thereby generate sufficient reducing potential for detoxification of ROS.  Lung cancer cells in which endogenous PKM2 was replaced with the Cys358 to Ser358 oxidn.-resistant mutant exhibited increased sensitivity to oxidative stress and impaired tumor formation in a xenograft model.  Besides promoting metabolic changes required for proliferation, the regulatory properties of PKM2 may confer an addnl. advantage to cancer cells by allowing them to withstand oxidative stress.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9SgiOQxnKObVg90H21EOLACvtfcHk0ljGSUG8RA_3NQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsFCms7jK&md5=9c6ac78508f3c25cbad20d1f8a21b4dc</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1126%2Fscience.1211485&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1211485%26sid%3Dliteratum%253Aachs%26aulast%3DAnastasiou%26aufirst%3DD.%26aulast%3DPoulogiannis%26aufirst%3DG.%26aulast%3DAsara%26aufirst%3DJ.%2BM.%26aulast%3DBoxer%26aufirst%3DM.%2BB.%26aulast%3DJiang%26aufirst%3DJ.%2BK.%26aulast%3DShen%26aufirst%3DM.%26aulast%3DBellinger%26aufirst%3DG.%26aulast%3DSasaki%26aufirst%3DA.%2BT.%26aulast%3DLocasale%26aufirst%3DJ.%2BW.%26aulast%3DAuld%26aufirst%3DD.%2BS.%26aulast%3DThomas%26aufirst%3DC.%2BJ.%26aulast%3DVander%2BHeiden%26aufirst%3DM.%2BG.%26aulast%3DCantley%26aufirst%3DL.%2BC.%26atitle%3DInhibition%2520of%2520Pyruvate%2520Kinase%2520M2%2520by%2520Reactive%2520Oxygen%2520Species%2520Contributes%2520to%2520Cellular%2520Antioxidant%2520Responses%26jtitle%3DScience%26date%3D2011%26volume%3D334%26spage%3D1278%26epage%3D1283%26doi%3D10.1126%2Fscience.1211485" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Leproult, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barluenga, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moras, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wurtz, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winssinger, N.</span></span> <span> </span><span class="NLM_article-title">Cysteine Mapping in Conformationally Distinct Kinase Nucleotide Binding Sites: Application to the Design of Selective Covalent Inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">1347</span>– <span class="NLM_lpage">1355</span>, <span class="refDoi"> DOI: 10.1021/jm101396q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm101396q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhvFeis70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=1347-1355&author=E.+Leproultauthor=S.+Barluengaauthor=D.+Morasauthor=J.+M.+Wurtzauthor=N.+Winssinger&title=Cysteine+Mapping+in+Conformationally+Distinct+Kinase+Nucleotide+Binding+Sites%3A+Application+to+the+Design+of+Selective+Covalent+Inhibitors&doi=10.1021%2Fjm101396q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Cysteine mapping in conformationally distinct kinase nucleotide binding sites: application to the design of selective covalent inhibitors</span></div><div class="casAuthors">Leproult, Emeline; Barluenga, Sofia; Moras, Dino; Wurtz, Jean-Marie; Winssinger, Nicolas</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1347-1355</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Kinases have emerged as one of the most prolific therapeutic targets.  An important criterion in the therapeutic success of inhibitors targeting the nucleotide binding pocket of kinases is the inhibitor residence time.  Recently, covalent kinase inhibitors have attracted attention since they confer terminal inhibition and should thus be more effective than reversible inhibitors with transient inhibition.  The most robust approach to design irreversible inhibitors is to capitalize on the nucleophilicity of a cysteine thiol group present in the target protein.  Herein, the authors report a systematic anal. of cysteine residues present in the nucleotide binding site of kinases, which could be harnessed for irreversible inhibition, taking into consideration the different kinase conformations.  The authors demonstrate the predictive power of this anal. with the design and validation of an irreversible inhibitor of KIT/PDGFR kinases.  This is the first example of a covalent kinase inhibitor that combines a pharmacophore addressing the DFG-out conformation with a covalent trap.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrkzqt0gMrz3rVg90H21EOLACvtfcHk0ljGSUG8RA_3NQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhvFeis70%253D&md5=329ae5eaa6102b6dd34a0d1274631247</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fjm101396q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm101396q%26sid%3Dliteratum%253Aachs%26aulast%3DLeproult%26aufirst%3DE.%26aulast%3DBarluenga%26aufirst%3DS.%26aulast%3DMoras%26aufirst%3DD.%26aulast%3DWurtz%26aufirst%3DJ.%2BM.%26aulast%3DWinssinger%26aufirst%3DN.%26atitle%3DCysteine%2520Mapping%2520in%2520Conformationally%2520Distinct%2520Kinase%2520Nucleotide%2520Binding%2520Sites%253A%2520Application%2520to%2520the%2520Design%2520of%2520Selective%2520Covalent%2520Inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D1347%26epage%3D1355%26doi%3D10.1021%2Fjm101396q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sabnis, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buhrlage, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">Developing Irreversible Inhibitors of the Protein Kinase Cysteinome</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">146</span>– <span class="NLM_lpage">159</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2012.12.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=10.1016%2Fj.chembiol.2012.12.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=23438744" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjtFWitbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2013&pages=146-159&author=Q.+Liuauthor=Y.+Sabnisauthor=Z.+Zhaoauthor=T.+Zhangauthor=S.+J.+Buhrlageauthor=L.+H.+Jonesauthor=N.+S.+Gray&title=Developing+Irreversible+Inhibitors+of+the+Protein+Kinase+Cysteinome&doi=10.1016%2Fj.chembiol.2012.12.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Developing Irreversible Inhibitors of the Protein Kinase Cysteinome</span></div><div class="casAuthors">Liu, Qingsong; Sabnis, Yogesh; Zhao, Zheng; Zhang, Tinghu; Buhrlage, Sara J.; Jones, Lyn H.; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">146-159</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Protein kinases are a large family of approx. 530 highly conserved enzymes that transfer a γ-phosphate group from ATP to a variety of amino acid residues, such as tyrosine, serine, and threonine, that serves as a ubiquitous mechanism for cellular signal transduction.  The clin. success of a no. of kinase-directed drugs and the frequent observation of disease causing mutations in protein kinases suggest that a large no. of kinases may represent therapeutically relevant targets.  To date, the majority of clin. and preclin. kinase inhibitors are ATP competitive, noncovalent inhibitors that achieve selectivity through recognition of unique features of particular protein kinases.  Recently, there has been renewed interest in the development of irreversible inhibitors that form covalent bonds with cysteine or other nucleophilic residues in the ATP-binding pocket.  Irreversible kinase inhibitors have a no. of potential advantages including prolonged pharmacodynamics, suitability for rational design, high potency, and ability to validate pharmacol. specificity through mutation of the reactive cysteine residue.  Here, we review recent efforts to develop cysteine-targeted irreversible protein kinase inhibitors and discuss their modes of recognizing the ATP-binding pocket and their biol. activity profiles.  In addn., we provided an informatics assessment of the potential "kinase cysteinome" and discuss strategies for the efficient development of new covalent inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVGD7TX3oIeLVg90H21EOLACvtfcHk0lj-RNEdjUbkpQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjtFWitbs%253D&md5=cef80823123bae88362e816a0a638133</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2012.12.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2012.12.006%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DQ.%26aulast%3DSabnis%26aufirst%3DY.%26aulast%3DZhao%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DBuhrlage%26aufirst%3DS.%2BJ.%26aulast%3DJones%26aufirst%3DL.%2BH.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DDeveloping%2520Irreversible%2520Inhibitors%2520of%2520the%2520Protein%2520Kinase%2520Cysteinome%26jtitle%3DChem.%2520Biol.%26date%3D2013%26volume%3D20%26spage%3D146%26epage%3D159%26doi%3D10.1016%2Fj.chembiol.2012.12.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chaikuad, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koch, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laufer, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">The Cysteinome of Protein Kinases as a Target in Drug Development</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">4372</span>– <span class="NLM_lpage">4385</span>, <span class="refDoi"> DOI: 10.1002/anie.201707875</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=10.1002%2Fanie.201707875" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitFGgs7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2018&pages=4372-4385&author=A.+Chaikuadauthor=P.+Kochauthor=S.+A.+Lauferauthor=S.+Knapp&title=The+Cysteinome+of+Protein+Kinases+as+a+Target+in+Drug+Development&doi=10.1002%2Fanie.201707875"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">The Cysteinome of Protein Kinases as a Target in Drug Development</span></div><div class="casAuthors">Chaikuad, Apirat; Koch, Pierre; Laufer, Stefan A.; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">4372-4385</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  Drugs that function through covalent bond formation represent a considerable fraction of the authors' repository of effective medicines but safety concerns and the complexity of developing covalent inhibitors has rendered covalent targeting a less attractive strategy for rational drug design.  The recent approval of four covalent kinase inhibitors and the development of highly potent covalent kinase probes with exceptional selectivity has raised significant interest in industry and academic research and validated the concept of covalent kinase targeting for clin. applications.  The abundance of cysteines at diverse positions in and around the kinase active site suggests that a large fraction of kinases can be targeted by covalent inhibitors.  Herein, the authors review recent developments of this rapidly growing area in kinase drug development and highlight the unique opportunities and challenges of this strategy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphHGXsIr-dpbVg90H21EOLACvtfcHk0lj-RNEdjUbkpQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitFGgs7c%253D&md5=311ba5ab53dc4a5dc1dc92ddd4b6d4c2</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1002%2Fanie.201707875&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201707875%26sid%3Dliteratum%253Aachs%26aulast%3DChaikuad%26aufirst%3DA.%26aulast%3DKoch%26aufirst%3DP.%26aulast%3DLaufer%26aufirst%3DS.%2BA.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DThe%2520Cysteinome%2520of%2520Protein%2520Kinases%2520as%2520a%2520Target%2520in%2520Drug%2520Development%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2018%26volume%3D57%26spage%3D4372%26epage%3D4385%26doi%3D10.1002%2Fanie.201707875" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mediani, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gibellini, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertacchini, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frasson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bosco, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Accordi, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Basso, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonora, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calabro, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mattiolo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sgarbi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baracca, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pinton, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riva, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rampazzo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petrizza, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prodi, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milani, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luppi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potenza, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Pol, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cocco, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Capitani, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marmiroli, S.</span></span> <span> </span><span class="NLM_article-title">Reversal of the Glycolytic Phenotype of Primary Effusion Lymphoma Cells by Combined Targeting of Cellular Metabolism and Pi3k/Akt/ Mtor Signaling</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">5521</span>– <span class="NLM_lpage">5537</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.6315</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=10.18632%2Foncotarget.6315" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=26575168" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=1%3ACAS%3A280%3ADC%252BC28vitlymtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=5521-5537&author=L.+Medianiauthor=F.+Gibelliniauthor=J.+Bertacchiniauthor=C.+Frassonauthor=R.+Boscoauthor=B.+Accordiauthor=G.+Bassoauthor=M.+Bonoraauthor=M.+L.+Calabroauthor=A.+Mattioloauthor=G.+Sgarbiauthor=A.+Baraccaauthor=P.+Pintonauthor=G.+Rivaauthor=E.+Rampazzoauthor=L.+Petrizzaauthor=L.+Prodiauthor=D.+Milaniauthor=M.+Luppiauthor=L.+Potenzaauthor=A.+De+Polauthor=L.+Coccoauthor=S.+Capitaniauthor=S.+Marmiroli&title=Reversal+of+the+Glycolytic+Phenotype+of+Primary+Effusion+Lymphoma+Cells+by+Combined+Targeting+of+Cellular+Metabolism+and+Pi3k%2FAkt%2F+Mtor+Signaling&doi=10.18632%2Foncotarget.6315"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Reversal of the glycolytic phenotype of primary effusion lymphoma cells by combined targeting of cellular metabolism and PI3K/Akt/ mTOR signaling</span></div><div class="casAuthors">Mediani Laura; Gibellini Federica; Bertacchini Jessika; Bosco Raffaella; De Pol Anto; Marmiroli Sandra; Bertacchini Jessika; Milani Daniela; Capitani Silvano; Frasson Chiara; Accordi Benedetta; Basso Giuseppe; Bonora Massimo; Pinton Paolo; Calabro Maria Luisa; Mattiolo Adriana; Sgarbi Gianluca; Baracca Alessandra; Cocco Lucio; Riva Giovanni; Luppi Mario; Potenza Leonardo; Rampazzo Enrico; Petrizza Luca; Prodi Luca</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">5521-37</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">PEL is a B-cell non-Hodgkin lymphoma, occurring predominantly as a lymphomatous effusion in body cavities, characterized by aggressive clinical course, with no standard therapy.  Based on previous reports that PEL cells display a Warburg phenotype, we hypothesized that the highly hypoxic environment in which they grow in vivo makes them more reliant on glycolysis, and more vulnerable to drugs targeting this pathway.  We established here that indeed PEL cells in hypoxia are more sensitive to glycolysis inhibition.  Furthermore, since PI3K/Akt/mTOR has been proposed as a drug target in PEL, we ascertained that pathway-specific inhibitors, namely the dual PI3K and mTOR inhibitor, PF-04691502, and the Akt inhibitor, Akti 1/2, display improved cytotoxicity to PEL cells in hypoxic conditions.  Unexpectedly, we found that these drugs reduce lactate production/extracellular acidification rate, and, in combination with the glycolysis inhibitor 2-deoxyglucose (2-DG), they shift PEL cells metabolism from aerobic glycolysis towards oxidative respiration.  Moreover, the associations possess strong synergistic cytotoxicity towards PEL cells, and thus may reduce adverse reaction in vivo, while displaying very low toxicity to normal lymphocytes.  Finally, we showed that the association of 2-DG and PF-04691502 maintains its cytotoxic and proapoptotic effect also in PEL cells co-cultured with human primary mesothelial cells, a condition known to mimic the in vivo environment and to exert a protective and pro-survival action.  All together, these results provide a compelling rationale for the clinical development of new therapies for the treatment of PEL, based on combined targeting of glycolytic metabolism and constitutively activated signaling pathways.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQmRFIjeiXbtkg6wdEZmT2rfW6udTcc2ebQloWaffwwcrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC28vitlymtA%253D%253D&md5=6a05c8a9f616469c201dc614a07f000f</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.6315&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.6315%26sid%3Dliteratum%253Aachs%26aulast%3DMediani%26aufirst%3DL.%26aulast%3DGibellini%26aufirst%3DF.%26aulast%3DBertacchini%26aufirst%3DJ.%26aulast%3DFrasson%26aufirst%3DC.%26aulast%3DBosco%26aufirst%3DR.%26aulast%3DAccordi%26aufirst%3DB.%26aulast%3DBasso%26aufirst%3DG.%26aulast%3DBonora%26aufirst%3DM.%26aulast%3DCalabro%26aufirst%3DM.%2BL.%26aulast%3DMattiolo%26aufirst%3DA.%26aulast%3DSgarbi%26aufirst%3DG.%26aulast%3DBaracca%26aufirst%3DA.%26aulast%3DPinton%26aufirst%3DP.%26aulast%3DRiva%26aufirst%3DG.%26aulast%3DRampazzo%26aufirst%3DE.%26aulast%3DPetrizza%26aufirst%3DL.%26aulast%3DProdi%26aufirst%3DL.%26aulast%3DMilani%26aufirst%3DD.%26aulast%3DLuppi%26aufirst%3DM.%26aulast%3DPotenza%26aufirst%3DL.%26aulast%3DDe%2BPol%26aufirst%3DA.%26aulast%3DCocco%26aufirst%3DL.%26aulast%3DCapitani%26aufirst%3DS.%26aulast%3DMarmiroli%26aufirst%3DS.%26atitle%3DReversal%2520of%2520the%2520Glycolytic%2520Phenotype%2520of%2520Primary%2520Effusion%2520Lymphoma%2520Cells%2520by%2520Combined%2520Targeting%2520of%2520Cellular%2520Metabolism%2520and%2520Pi3k%252FAkt%252F%2520Mtor%2520Signaling%26jtitle%3DOncotarget%26date%3D2016%26volume%3D7%26spage%3D5521%26epage%3D5537%26doi%3D10.18632%2Foncotarget.6315" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Saunier, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Antonio, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Regazzetti, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Auzeil, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laprevote, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shay, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coumoul, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barouki, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benelli, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huc, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bortoli, S.</span></span> <span> </span><span class="NLM_article-title">Resveratrol Reverses the Warburg Effect by Targeting the Pyruvate Dehydrogenase Complex in Colon Cancer Cells</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">6945</span>, <span class="refDoi"> DOI: 10.1038/s41598-017-07006-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=10.1038%2Fs41598-017-07006-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=28761044" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=1%3ACAS%3A280%3ADC%252BC1cfivVOmtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&pages=6945&author=E.+Saunierauthor=S.+Antonioauthor=A.+Regazzettiauthor=N.+Auzeilauthor=O.+Laprevoteauthor=J.+W.+Shayauthor=X.+Coumoulauthor=R.+Baroukiauthor=C.+Benelliauthor=L.+Hucauthor=S.+Bortoli&title=Resveratrol+Reverses+the+Warburg+Effect+by+Targeting+the+Pyruvate+Dehydrogenase+Complex+in+Colon+Cancer+Cells&doi=10.1038%2Fs41598-017-07006-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Resveratrol reverses the Warburg effect by targeting the pyruvate dehydrogenase complex in colon cancer cells</span></div><div class="casAuthors">Saunier Elise; Antonio Samantha; Coumoul Xavier; Barouki Robert; Benelli Chantal; Bortoli Sylvie; Saunier Elise; Antonio Samantha; Coumoul Xavier; Barouki Robert; Benelli Chantal; Bortoli Sylvie; Regazzetti Anne; Auzeil Nicolas; Laprevote Olivier; Shay Jerry W; Huc Laurence; Huc Laurence</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">6945</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Resveratrol (RES), a polyphenol found in natural foods, displays anti-oxidant, anti-inflammatory and anti-proliferative properties potentially beneficial in cancers, in particular in the prevention of tumor growth.  However, the rapid metabolism of resveratrol strongly limits its bioavailability.  The molecular mechanisms sustaining the potential biological activity of low doses of resveratrol has not been extensively studied and, thus, needs better characterization.  Here, we show that resveratrol (10 μM, 48 hr) induces both a cell growth arrest and a metabolic reprogramming in colon cancer cells.  Resveratrol modifies the lipidomic profile, increases oxidative capacities and decreases glycolysis, in association with a decreased pentose phosphate activity and an increased ATP production.  Resveratrol targets the pyruvate dehydrogenase (PDH) complex, a key mitochondrial gatekeeper of energy metabolism, leading to an enhanced PDH activity.  Calcium chelation, as well as the blockade of the mitochondrial calcium uniport, prevents the resveratrol-induced augmentation in oxidative capacities and the increased PDH activity suggesting that calcium might play a role in the metabolic shift.  We further demonstrate that the inhibition of the CamKKB or the downstream AMPK pathway partly abolished the resveratrol-induced increase of glucose oxidation.  This suggests that resveratrol might improve the oxidative capacities of cancer cells through the CamKKB/AMPK pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRCLVgLaW5mRWHOrQ6qeX-mfW6udTcc2eYdmqiHUV4pX7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1cfivVOmtw%253D%253D&md5=5a87309168c53f0f7fdb43e300d32a78</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1038%2Fs41598-017-07006-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-017-07006-0%26sid%3Dliteratum%253Aachs%26aulast%3DSaunier%26aufirst%3DE.%26aulast%3DAntonio%26aufirst%3DS.%26aulast%3DRegazzetti%26aufirst%3DA.%26aulast%3DAuzeil%26aufirst%3DN.%26aulast%3DLaprevote%26aufirst%3DO.%26aulast%3DShay%26aufirst%3DJ.%2BW.%26aulast%3DCoumoul%26aufirst%3DX.%26aulast%3DBarouki%26aufirst%3DR.%26aulast%3DBenelli%26aufirst%3DC.%26aulast%3DHuc%26aufirst%3DL.%26aulast%3DBortoli%26aufirst%3DS.%26atitle%3DResveratrol%2520Reverses%2520the%2520Warburg%2520Effect%2520by%2520Targeting%2520the%2520Pyruvate%2520Dehydrogenase%2520Complex%2520in%2520Colon%2520Cancer%2520Cells%26jtitle%3DSci.%2520Rep.%26date%3D2017%26volume%3D7%26spage%3D6945%26doi%3D10.1038%2Fs41598-017-07006-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fadia, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dahlstrom, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parish, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Board, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blackburn, A. C.</span></span> <span> </span><span class="NLM_article-title">Reversal of the Glycolytic Phenotype by Dichloroacetate Inhibits Metastatic Breast Cancer Cell Growth in Vitro and in Vivo</span>. <i>Breast Cancer Res. Treat.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>120</i></span>,  <span class="NLM_fpage">253</span>– <span class="NLM_lpage">260</span>, <span class="refDoi"> DOI: 10.1007/s10549-009-0435-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=10.1007%2Fs10549-009-0435-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=19543830" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=1%3ACAS%3A528%3ADC%252BC3cXntlaruw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=120&publication_year=2010&pages=253-260&author=R.+C.+Sunauthor=M.+Fadiaauthor=J.+E.+Dahlstromauthor=C.+R.+Parishauthor=P.+G.+Boardauthor=A.+C.+Blackburn&title=Reversal+of+the+Glycolytic+Phenotype+by+Dichloroacetate+Inhibits+Metastatic+Breast+Cancer+Cell+Growth+in+Vitro+and+in+Vivo&doi=10.1007%2Fs10549-009-0435-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Reversal of the glycolytic phenotype by dichloroacetate inhibits metastatic breast cancer cell growth in vitro and in vivo</span></div><div class="casAuthors">Sun, Ramon C.; Fadia, Mitali; Dahlstrom, Jane E.; Parish, Christopher R.; Board, Philip G.; Blackburn, Anneke C.</div><div class="citationInfo"><span class="NLM_cas:title">Breast Cancer Research and Treatment</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">120</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">253-260</span>CODEN:
                <span class="NLM_cas:coden">BCTRD6</span>;
        ISSN:<span class="NLM_cas:issn">0167-6806</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">The glycolytic phenotype is a widespread phenomenon in solid cancer forms, including breast cancer.  Dichloroacetate (DCA) has recently been proposed as a novel and relatively non-toxic anti-cancer agent that can reverse the glycolytic phenotype in cancer cells through the inhibition of pyruvate dehydrogenase kinase.  We have examd. the effect of DCA against breast cancer cells, including in a highly metastatic in vivo model.  The growth of several breast cancer cell lines was found to be inhibited by DCA in vitro.  Further examn. of 13762 MAT rat mammary adenocarcinoma cells found that reversal of the glycolytic phenotype by DCA correlated with the inhibition of proliferation without any increase in cell death.  This was despite a small but significant increase in caspase 3/7 activity, which may sensitize cancer cells to other apoptotic triggers.  In vivo, DCA caused a 58% redn. in the no. of lung metastases obsd. macroscopically after injection of 13,762 MAT cells into the tail vein of rats (P = 0.0001, n ≥ 9 per group).  These results demonstrate that DCA has anti-proliferative properties in addn. to pro-apoptotic properties, and can be effective against highly metastatic disease in vivo, highlighting its potential for clin. use.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbbgVFTE8jU7Vg90H21EOLACvtfcHk0li4MsOeQ9VCgQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXntlaruw%253D%253D&md5=f681498dd473ee35e2ced31f317109d0</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1007%2Fs10549-009-0435-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10549-009-0435-9%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DR.%2BC.%26aulast%3DFadia%26aufirst%3DM.%26aulast%3DDahlstrom%26aufirst%3DJ.%2BE.%26aulast%3DParish%26aufirst%3DC.%2BR.%26aulast%3DBoard%26aufirst%3DP.%2BG.%26aulast%3DBlackburn%26aufirst%3DA.%2BC.%26atitle%3DReversal%2520of%2520the%2520Glycolytic%2520Phenotype%2520by%2520Dichloroacetate%2520Inhibits%2520Metastatic%2520Breast%2520Cancer%2520Cell%2520Growth%2520in%2520Vitro%2520and%2520in%2520Vivo%26jtitle%3DBreast%2520Cancer%2520Res.%2520Treat.%26date%3D2010%26volume%3D120%26spage%3D253%26epage%3D260%26doi%3D10.1007%2Fs10549-009-0435-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Anastasiou, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Israelsen, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boxer, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tempel, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dimov, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jha, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mattaini, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Metallo, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fiske, B. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Courtney, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malstrom, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khan, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kung, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skoumbourdis, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Veith, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Southall, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walsh, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brimacombe, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leister, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lunt, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, Z. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yen, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kunii, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davidson, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christofk, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Austin, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inglese, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asara, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stephanopoulos, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salituro, F. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Auld, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, H. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantley, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vander Heiden, M. G.</span></span> <span> </span><span class="NLM_article-title">Pyruvate Kinase M2 Activators Promote Tetramer Formation and Suppress Tumorigenesis</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">839</span>– <span class="NLM_lpage">847</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1060</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=10.1038%2Fnchembio.1060" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=22922757" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1Ght7rM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2012&pages=839-847&author=D.+Anastasiouauthor=Y.+Yuauthor=W.+J.+Israelsenauthor=J.+K.+Jiangauthor=M.+B.+Boxerauthor=B.+S.+Hongauthor=W.+Tempelauthor=S.+Dimovauthor=M.+Shenauthor=A.+Jhaauthor=H.+Yangauthor=K.+R.+Mattainiauthor=C.+M.+Metalloauthor=B.+P.+Fiskeauthor=K.+D.+Courtneyauthor=S.+Malstromauthor=T.+M.+Khanauthor=C.+Kungauthor=A.+P.+Skoumbourdisauthor=H.+Veithauthor=N.+Southallauthor=M.+J.+Walshauthor=K.+R.+Brimacombeauthor=W.+Leisterauthor=S.+Y.+Luntauthor=Z.+R.+Johnsonauthor=K.+E.+Yenauthor=K.+Kuniiauthor=S.+M.+Davidsonauthor=H.+R.+Christofkauthor=C.+P.+Austinauthor=J.+Ingleseauthor=M.+H.+Harrisauthor=J.+M.+Asaraauthor=G.+Stephanopoulosauthor=F.+G.+Salituroauthor=S.+Jinauthor=L.+Dangauthor=D.+S.+Auldauthor=H.+W.+Parkauthor=L.+C.+Cantleyauthor=C.+J.+Thomasauthor=M.+G.+Vander+Heiden&title=Pyruvate+Kinase+M2+Activators+Promote+Tetramer+Formation+and+Suppress+Tumorigenesis&doi=10.1038%2Fnchembio.1060"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Pyruvate kinase M2 activators promote tetramer formation and suppress tumorigenesis</span></div><div class="casAuthors">Anastasiou, Dimitrios; Yu, Yimin; Israelsen, William J.; Jiang, Jian-Kang; Boxer, Matthew B.; Hong, Bum Soo; Tempel, Wolfram; Dimov, Svetoslav; Shen, Min; Jha, Abhishek; Yang, Hua; Mattaini, Katherine R.; Metallo, Christian M.; Fiske, Brian P.; Courtney, Kevin D.; Malstrom, Scott; Khan, Tahsin M.; Kung, Charles; Skoumbourdis, Amanda P.; Veith, Henrike; Southall, Noel; Walsh, Martin J.; Brimacombe, Kyle R.; Leister, William; Lunt, Sophia Y.; Johnson, Zachary R.; Yen, Katharine E.; Kunii, Kaiko; Davidson, Shawn M.; Christofk, Heather R.; Austin, Christopher P.; Inglese, James; Harris, Marian H.; Asara, John M.; Stephanopoulos, Gregory; Salituro, Francesco G.; Jin, Shengfang; Dang, Lenny; Auld, Douglas S.; Park, Hee-Won; Cantley, Lewis C.; Thomas, Craig J.; Vander Heiden, Matthew G.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">839-847</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Cancer cells engage in a metabolic program to enhance biosynthesis and support cell proliferation.  The regulatory properties of pyruvate kinase M2 (PKM2) influence altered glucose metab. in cancer.  The interaction of PKM2 with phosphotyrosine-contg. proteins inhibits enzyme activity and increases the availability of glycolytic metabolites to support cell proliferation.  This suggests that high pyruvate kinase activity may suppress tumor growth.  We show that expression of PKM1, the pyruvate kinase isoform with high constitutive activity, or exposure to published small-mol. PKM2 activators inhibits the growth of xenograft tumors.  Structural studies reveal that small-mol. activators bind PKM2 at the subunit interaction interface, a site that is distinct from that of the endogenous activator fructose-1,6-bisphosphate (FBP).  However, unlike FBP, binding of activators to PKM2 promotes a constitutively active enzyme state that is resistant to inhibition by tyrosine-phosphorylated proteins.  These data support the notion that small-mol. activation of PKM2 can interfere with anabolic metab.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMDIr6LPTxPbVg90H21EOLACvtfcHk0lh3rPPlbZlA8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1Ght7rM&md5=20ae4953c6afd41458cf317933fd5d9d</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1060&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1060%26sid%3Dliteratum%253Aachs%26aulast%3DAnastasiou%26aufirst%3DD.%26aulast%3DYu%26aufirst%3DY.%26aulast%3DIsraelsen%26aufirst%3DW.%2BJ.%26aulast%3DJiang%26aufirst%3DJ.%2BK.%26aulast%3DBoxer%26aufirst%3DM.%2BB.%26aulast%3DHong%26aufirst%3DB.%2BS.%26aulast%3DTempel%26aufirst%3DW.%26aulast%3DDimov%26aufirst%3DS.%26aulast%3DShen%26aufirst%3DM.%26aulast%3DJha%26aufirst%3DA.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DMattaini%26aufirst%3DK.%2BR.%26aulast%3DMetallo%26aufirst%3DC.%2BM.%26aulast%3DFiske%26aufirst%3DB.%2BP.%26aulast%3DCourtney%26aufirst%3DK.%2BD.%26aulast%3DMalstrom%26aufirst%3DS.%26aulast%3DKhan%26aufirst%3DT.%2BM.%26aulast%3DKung%26aufirst%3DC.%26aulast%3DSkoumbourdis%26aufirst%3DA.%2BP.%26aulast%3DVeith%26aufirst%3DH.%26aulast%3DSouthall%26aufirst%3DN.%26aulast%3DWalsh%26aufirst%3DM.%2BJ.%26aulast%3DBrimacombe%26aufirst%3DK.%2BR.%26aulast%3DLeister%26aufirst%3DW.%26aulast%3DLunt%26aufirst%3DS.%2BY.%26aulast%3DJohnson%26aufirst%3DZ.%2BR.%26aulast%3DYen%26aufirst%3DK.%2BE.%26aulast%3DKunii%26aufirst%3DK.%26aulast%3DDavidson%26aufirst%3DS.%2BM.%26aulast%3DChristofk%26aufirst%3DH.%2BR.%26aulast%3DAustin%26aufirst%3DC.%2BP.%26aulast%3DInglese%26aufirst%3DJ.%26aulast%3DHarris%26aufirst%3DM.%2BH.%26aulast%3DAsara%26aufirst%3DJ.%2BM.%26aulast%3DStephanopoulos%26aufirst%3DG.%26aulast%3DSalituro%26aufirst%3DF.%2BG.%26aulast%3DJin%26aufirst%3DS.%26aulast%3DDang%26aufirst%3DL.%26aulast%3DAuld%26aufirst%3DD.%2BS.%26aulast%3DPark%26aufirst%3DH.%2BW.%26aulast%3DCantley%26aufirst%3DL.%2BC.%26aulast%3DThomas%26aufirst%3DC.%2BJ.%26aulast%3DVander%2BHeiden%26aufirst%3DM.%2BG.%26atitle%3DPyruvate%2520Kinase%2520M2%2520Activators%2520Promote%2520Tetramer%2520Formation%2520and%2520Suppress%2520Tumorigenesis%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2012%26volume%3D8%26spage%3D839%26epage%3D847%26doi%3D10.1038%2Fnchembio.1060" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cortes-Cros, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hemmerlin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferretti, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gounarides, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haberkorn, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chene, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sellers, W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hofmann, F.</span></span> <span> </span><span class="NLM_article-title">M2 Isoform of Pyruvate Kinase Is Dispensable for Tumor Maintenance and Growth</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>110</i></span>,  <span class="NLM_fpage">489</span>– <span class="NLM_lpage">494</span>, <span class="refDoi"> DOI: 10.1073/pnas.1212780110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=10.1073%2Fpnas.1212780110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=23267074" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFOhtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2013&pages=489-494&author=M.+Cortes-Crosauthor=C.+Hemmerlinauthor=S.+Ferrettiauthor=J.+Zhangauthor=J.+S.+Gounaridesauthor=H.+Yinauthor=A.+Mullerauthor=A.+Haberkornauthor=P.+Cheneauthor=W.+R.+Sellersauthor=F.+Hofmann&title=M2+Isoform+of+Pyruvate+Kinase+Is+Dispensable+for+Tumor+Maintenance+and+Growth&doi=10.1073%2Fpnas.1212780110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">M2 isoform of pyruvate kinase is dispensable for tumor maintenance and growth</span></div><div class="casAuthors">Cortes-Cros, Marta; Hemmerlin, Christelle; Ferretti, Stephane; Zhang, Juan; Gounarides, John S.; Yin, Hong; Muller, Alban; Haberkorn, Anne; Chene, Patrick; Sellers, William R.; Hofmann, Francesco</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">489-494, S489/1-S489/4</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Many cancer cells have increased rates of aerobic glycolysis, a phenomenon termed the Warburg effect.  In addn., in tumors there is a predominance of expression of the M2 isoform of pyruvate kinase (PKM2).  PKM2 expression was previously shown to be necessary for aerobic glycolysis and to provide a growth advantage to tumors.  Here, the authors report that knockdown of PKM2 in tumor cells led to a decrease in the levels of pyruvate kinase activity and an increase in the pyruvate kinase substrate, phosphoenolpyruvate.  However, lactate prodn. from glucose, although reduced, was not fully inhibited.  Furthermore, the authors are unique in reporting increased serine and glycine biosynthesis from both glucose and glutamine following PKM2 knockdown.  Although PKM2 knockdown resulted in modest impairment of proliferation in vitro, the in vivo growth of established xenograft tumors was unaffected by PKM2 absence.  Thus, these findings indicate that PKM2 is dispensable for tumor maintenance and growth in vivo, suggesting that other metabolic pathways bypass its function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXJi5-fdQ7jLVg90H21EOLACvtfcHk0lh3rPPlbZlA8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFOhtLs%253D&md5=5284f54fbaa4dad9b067ee1b0394a7e1</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1212780110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1212780110%26sid%3Dliteratum%253Aachs%26aulast%3DCortes-Cros%26aufirst%3DM.%26aulast%3DHemmerlin%26aufirst%3DC.%26aulast%3DFerretti%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DGounarides%26aufirst%3DJ.%2BS.%26aulast%3DYin%26aufirst%3DH.%26aulast%3DMuller%26aufirst%3DA.%26aulast%3DHaberkorn%26aufirst%3DA.%26aulast%3DChene%26aufirst%3DP.%26aulast%3DSellers%26aufirst%3DW.%2BR.%26aulast%3DHofmann%26aufirst%3DF.%26atitle%3DM2%2520Isoform%2520of%2520Pyruvate%2520Kinase%2520Is%2520Dispensable%2520for%2520Tumor%2520Maintenance%2520and%2520Growth%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2013%26volume%3D110%26spage%3D489%26epage%3D494%26doi%3D10.1073%2Fpnas.1212780110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Israelsen, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dayton, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davidson, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fiske, B. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hosios, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellinger, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sasaki, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horner, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burga, L. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jurczak, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DePinho, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clish, C. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacks, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kibbey, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wulf, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Vizio, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mills, G. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantley, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vander Heiden, M. G.</span></span> <span> </span><span class="NLM_article-title">Pkm2 Isoform-Specific Deletion Reveals a Differential Requirement for Pyruvate Kinase in Tumor Cells</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>155</i></span>,  <span class="NLM_fpage">397</span>– <span class="NLM_lpage">409</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2013.09.025</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=10.1016%2Fj.cell.2013.09.025" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=24120138" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1Sktr%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=155&publication_year=2013&pages=397-409&author=W.+J.+Israelsenauthor=T.+L.+Daytonauthor=S.+M.+Davidsonauthor=B.+P.+Fiskeauthor=A.+M.+Hosiosauthor=G.+Bellingerauthor=J.+Liauthor=Y.+Yuauthor=M.+Sasakiauthor=J.+W.+Hornerauthor=L.+N.+Burgaauthor=J.+Xieauthor=M.+J.+Jurczakauthor=R.+A.+DePinhoauthor=C.+B.+Clishauthor=T.+Jacksauthor=R.+G.+Kibbeyauthor=G.+M.+Wulfauthor=D.+Di+Vizioauthor=G.+B.+Millsauthor=L.+C.+Cantleyauthor=M.+G.+Vander+Heiden&title=Pkm2+Isoform-Specific+Deletion+Reveals+a+Differential+Requirement+for+Pyruvate+Kinase+in+Tumor+Cells&doi=10.1016%2Fj.cell.2013.09.025"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">PKM2 Isoform-Specific Deletion Reveals a Differential Requirement for Pyruvate Kinase in Tumor Cells</span></div><div class="casAuthors">Israelsen, William J.; Dayton, Talya L.; Davidson, Shawn M.; Fiske, Brian P.; Hosios, Aaron M.; Bellinger, Gary; Li, Jie; Yu, Yimin; Sasaki, Mika; Horner, James W.; Burga, Laura N.; Xie, Jianxin; Jurczak, Michael J.; DePinho, Ronald A.; Clish, Clary B.; Jacks, Tyler; Kibbey, Richard G.; Wulf, Gerburg M.; Di Vizio, Dolores; Mills, Gordon B.; Cantley, Lewis C.; Vander Heiden, Matthew G.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">155</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">397-409</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The pyruvate kinase M2 isoform (PKM2) is expressed in cancer and plays a role in regulating anabolic metab.  To det. whether PKM2 is required for tumor formation or growth, we generated mice with a conditional allele that abolishes PKM2 expression without disrupting PKM1 expression.  PKM2 deletion accelerated mammary tumor formation in a Brca1-loss-driven model of breast cancer.  PKM2 null tumors displayed heterogeneous PKM1 expression, with PKM1 found in nonproliferating tumor cells and no detectable pyruvate kinase expression in proliferating cells.  This suggests that PKM2 is not necessary for tumor cell proliferation and implies that the inactive state of PKM2 is assocd. with the proliferating cell population within tumors, whereas nonproliferating tumor cells require active pyruvate kinase.  Consistent with these findings, variable PKM2 expression and heterozygous PKM2 mutations are found in human tumors.  These data suggest that regulation of PKM2 activity supports the different metabolic requirements of proliferating and nonproliferating tumor cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8md57TLpos7Vg90H21EOLACvtfcHk0liy0Fpf-t1pAQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1Sktr%252FI&md5=b2647ac4c5fa6a1fa03664adb60288ee</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2013.09.025&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2013.09.025%26sid%3Dliteratum%253Aachs%26aulast%3DIsraelsen%26aufirst%3DW.%2BJ.%26aulast%3DDayton%26aufirst%3DT.%2BL.%26aulast%3DDavidson%26aufirst%3DS.%2BM.%26aulast%3DFiske%26aufirst%3DB.%2BP.%26aulast%3DHosios%26aufirst%3DA.%2BM.%26aulast%3DBellinger%26aufirst%3DG.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DYu%26aufirst%3DY.%26aulast%3DSasaki%26aufirst%3DM.%26aulast%3DHorner%26aufirst%3DJ.%2BW.%26aulast%3DBurga%26aufirst%3DL.%2BN.%26aulast%3DXie%26aufirst%3DJ.%26aulast%3DJurczak%26aufirst%3DM.%2BJ.%26aulast%3DDePinho%26aufirst%3DR.%2BA.%26aulast%3DClish%26aufirst%3DC.%2BB.%26aulast%3DJacks%26aufirst%3DT.%26aulast%3DKibbey%26aufirst%3DR.%2BG.%26aulast%3DWulf%26aufirst%3DG.%2BM.%26aulast%3DDi%2BVizio%26aufirst%3DD.%26aulast%3DMills%26aufirst%3DG.%2BB.%26aulast%3DCantley%26aufirst%3DL.%2BC.%26aulast%3DVander%2BHeiden%26aufirst%3DM.%2BG.%26atitle%3DPkm2%2520Isoform-Specific%2520Deletion%2520Reveals%2520a%2520Differential%2520Requirement%2520for%2520Pyruvate%2520Kinase%2520in%2520Tumor%2520Cells%26jtitle%3DCell%26date%3D2013%26volume%3D155%26spage%3D397%26epage%3D409%26doi%3D10.1016%2Fj.cell.2013.09.025" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hosios, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fiske, B. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gui, D. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vander Heiden, M. G.</span></span> <span> </span><span class="NLM_article-title">Lack of Evidence for Pkm2 Protein Kinase Activity</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">850</span>– <span class="NLM_lpage">857</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2015.07.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=10.1016%2Fj.molcel.2015.07.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=26300261" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtlOmtbrE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2015&pages=850-857&author=A.+M.+Hosiosauthor=B.+P.+Fiskeauthor=D.+Y.+Guiauthor=M.+G.+Vander+Heiden&title=Lack+of+Evidence+for+Pkm2+Protein+Kinase+Activity&doi=10.1016%2Fj.molcel.2015.07.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Lack of Evidence for PKM2 Protein Kinase Activity</span></div><div class="casAuthors">Hosios, Aaron M.; Fiske, Brian P.; Gui, Dan Y.; Vander Heiden, Matthew G.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">850-857</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">The role of pyruvate kinase M2 (PKM2) in cell proliferation is controversial.  A unique function of PKM2 proposed to be important for the proliferation of some cancer cells involves the direct activity of this enzyme as a protein kinase; however, a detailed biochem. characterization of this activity is lacking.  Using [32P]-phosphoenolpyruvate (PEP) we examine the direct substrates of PKM2 using recombinant enzyme and in vitro systems where PKM2 is genetically deleted.  Labeling of some protein species from [32P]-PEP can be obsd.; however, most were dependent on the presence of ADP, and none were dependent on the presence of PKM2.  In addn., we also failed to observe PKM2-dependent transfer of phosphate from ATP directly to protein.  These findings argue against a role for PKM2 as a protein kinase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqdvpnulo73T7Vg90H21EOLACvtfcHk0liy0Fpf-t1pAQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtlOmtbrE&md5=2b34c79757ed73f0c9cf4dd426960818</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2015.07.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2015.07.013%26sid%3Dliteratum%253Aachs%26aulast%3DHosios%26aufirst%3DA.%2BM.%26aulast%3DFiske%26aufirst%3DB.%2BP.%26aulast%3DGui%26aufirst%3DD.%2BY.%26aulast%3DVander%2BHeiden%26aufirst%3DM.%2BG.%26atitle%3DLack%2520of%2520Evidence%2520for%2520Pkm2%2520Protein%2520Kinase%2520Activity%26jtitle%3DMol.%2520Cell%26date%3D2015%26volume%3D59%26spage%3D850%26epage%3D857%26doi%3D10.1016%2Fj.molcel.2015.07.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bloxham, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilton, D. C.</span></span> <span> </span><span class="NLM_article-title">A Detailed Investigation of the Properties of Lactate Dehydrogenase in Which the ’Essential’ Cysteine-165 Is Modified by Thioalkylation</span>. <i>Biochem. J.</i> <span class="NLM_year" style="font-weight: bold;">1979</span>,  <span class="NLM_volume"><i>177</i></span>,  <span class="NLM_fpage">769</span>– <span class="NLM_lpage">780</span>, <span class="refDoi"> DOI: 10.1042/bj1770769a</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=10.1042%2Fbj1770769a" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=36072" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=1%3ACAS%3A528%3ADyaE1MXkvVKns74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=177&publication_year=1979&pages=769-780&author=D.+P.+Bloxhamauthor=R.+P.+Sharmaauthor=D.+C.+Wilton&title=A+Detailed+Investigation+of+the+Properties+of+Lactate+Dehydrogenase+in+Which+the+%E2%80%99Essential%E2%80%99+Cysteine-165+Is+Modified+by+Thioalkylation&doi=10.1042%2Fbj1770769a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">A detailed investigation of the properties of lactate dehydrogenase in which the 'essential' cysteine-165 is modified by thioalkylation</span></div><div class="casAuthors">Bloxham, David P.; Sharma, Ram P.; Wilton, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Journal</span>
        (<span class="NLM_cas:date">1979</span>),
    <span class="NLM_cas:volume">177</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">769-80</span>CODEN:
                <span class="NLM_cas:coden">BIJOAK</span>;
        ISSN:<span class="NLM_cas:issn">0264-6021</span>.
    </div><div class="casAbstract">Cysteine-165 of pig heart lactate dehydrogenase (I) was modified by reaction with Me methanethiosulfonate, and although the affinity of I for carbonyl- or hydroxy-contg. substrates was decreased, thiomethylated I possessed full catalytic activity.  The nicotinamide nucleotide-binding capacity was unaffected, as was I-mediated activation of NAD.  The affinity of oxamate for the I-NADH complex was decreased 100-fold; this constituted a net increase of 10.4 kJ/mol in the activation energy for binding.  Thiomethylated I formed an abortive adduct between NAD and fluoropyruvate, but the equil. const. was too low to demonstrate this complex at reasonable pyruvate concns.  A conformational change in structure was revealed by the decreased thermostability.  Thioethylation resulted in an increase in Km for pyruvate (13.5 mM) and an 85% decrease in max. catalytic activity.  The implications of these findings for the catalytic mechanism of I are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreoDh_0_Uqi7Vg90H21EOLACvtfcHk0liy0Fpf-t1pAQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE1MXkvVKns74%253D&md5=983d8cc2ebc23c1370b0c41055d3db64</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1042%2Fbj1770769a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252Fbj1770769a%26sid%3Dliteratum%253Aachs%26aulast%3DBloxham%26aufirst%3DD.%2BP.%26aulast%3DSharma%26aufirst%3DR.%2BP.%26aulast%3DWilton%26aufirst%3DD.%2BC.%26atitle%3DA%2520Detailed%2520Investigation%2520of%2520the%2520Properties%2520of%2520Lactate%2520Dehydrogenase%2520in%2520Which%2520the%2520%25E2%2580%2599Essential%25E2%2580%2599%2520Cysteine-165%2520Is%2520Modified%2520by%2520Thioalkylation%26jtitle%3DBiochem.%2520J.%26date%3D1979%26volume%3D177%26spage%3D769%26epage%3D780%26doi%3D10.1042%2Fbj1770769a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xie, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kats, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burgess, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhargava, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Signoretti, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Billiard, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duffy, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grant, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lorkiewicz, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schatzman, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bousamra, M.,  2nd</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lane, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Higashi, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, T. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pandolfi, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sukhatme, V. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seth, P.</span></span> <span> </span><span class="NLM_article-title">Targeting Lactate Dehydrogenase--a Inhibits Tumorigenesis and Tumor Progression in Mouse Models of Lung Cancer and Impacts Tumor-Initiating Cells</span>. <i>Cell Metab.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">795</span>– <span class="NLM_lpage">809</span>, <span class="refDoi"> DOI: 10.1016/j.cmet.2014.03.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=10.1016%2Fj.cmet.2014.03.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=24726384" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmtVWrtLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2014&pages=795-809&author=H.+Xieauthor=J.+Hanaiauthor=J.+G.+Renauthor=L.+Katsauthor=K.+Burgessauthor=P.+Bhargavaauthor=S.+Signorettiauthor=J.+Billiardauthor=K.+J.+Duffyauthor=A.+Grantauthor=X.+Wangauthor=P.+K.+Lorkiewiczauthor=S.+Schatzmanauthor=M.+Bousamraauthor=A.+N.+Laneauthor=R.+M.+Higashiauthor=T.+W.+Fanauthor=P.+P.+Pandolfiauthor=V.+P.+Sukhatmeauthor=P.+Seth&title=Targeting+Lactate+Dehydrogenase%2D%2Da+Inhibits+Tumorigenesis+and+Tumor+Progression+in+Mouse+Models+of+Lung+Cancer+and+Impacts+Tumor-Initiating+Cells&doi=10.1016%2Fj.cmet.2014.03.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting Lactate Dehydrogenase-A Inhibits Tumorigenesis and Tumor Progression in Mouse Models of Lung Cancer and Impacts Tumor-Initiating Cells</span></div><div class="casAuthors">Xie, Han; Hanai, Jun-ichi; Ren, Jian-Guo; Kats, Lev; Burgess, Kerri; Bhargava, Parul; Signoretti, Sabina; Billiard, Julia; Duffy, Kevin J.; Grant, Aaron; Wang, Xiaoen; Lorkiewicz, Pawel K.; Schatzman, Sabrina; Bousamra, Michael; Lane, Andrew N.; Higashi, Richard M.; Fan, Teresa W. M.; Pandolfi, Pier Paolo; Sukhatme, Vikas P.; Seth, Pankaj</div><div class="citationInfo"><span class="NLM_cas:title">Cell Metabolism</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">795-809</span>CODEN:
                <span class="NLM_cas:coden">CMEEB5</span>;
        ISSN:<span class="NLM_cas:issn">1550-4131</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">The lactate dehydrogenase-A (LDH-A) enzyme catalyzes the interconversion of pyruvate and lactate, is upregulated in human cancers, and is assocd. with aggressive tumor outcomes.  Here we use an inducible murine model and demonstrate that inactivation of LDH-A in mouse models of NSCLC driven by oncogenic K-RAS or EGFR leads to decreased tumorigenesis and disease regression in established tumors.  We also show that abrogation of LDH-A results in reprogramming of pyruvate metab., with decreased lactic fermn. in vitro, in vivo, and ex vivo.  This was accompanied by reactivation of mitochondrial function in vitro, but not in vivo or ex vivo.  Finally, using a specific small mol. LDH-A inhibitor, we demonstrated that LDH-A is essential for cancer-initiating cell survival and proliferation.  Thus, LDH-A can be a viable therapeutic target for NSCLC, including cancer stem cell-dependent drug-resistant tumors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpNiWJNOeWwmLVg90H21EOLACvtfcHk0liO78Mu_6N2ew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmtVWrtLY%253D&md5=d80624d60895f5a6c01106841ebd1af9</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1016%2Fj.cmet.2014.03.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cmet.2014.03.003%26sid%3Dliteratum%253Aachs%26aulast%3DXie%26aufirst%3DH.%26aulast%3DHanai%26aufirst%3DJ.%26aulast%3DRen%26aufirst%3DJ.%2BG.%26aulast%3DKats%26aufirst%3DL.%26aulast%3DBurgess%26aufirst%3DK.%26aulast%3DBhargava%26aufirst%3DP.%26aulast%3DSignoretti%26aufirst%3DS.%26aulast%3DBilliard%26aufirst%3DJ.%26aulast%3DDuffy%26aufirst%3DK.%2BJ.%26aulast%3DGrant%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DLorkiewicz%26aufirst%3DP.%2BK.%26aulast%3DSchatzman%26aufirst%3DS.%26aulast%3DBousamra%26aufirst%3DM.%26aulast%3DLane%26aufirst%3DA.%2BN.%26aulast%3DHigashi%26aufirst%3DR.%2BM.%26aulast%3DFan%26aufirst%3DT.%2BW.%26aulast%3DPandolfi%26aufirst%3DP.%2BP.%26aulast%3DSukhatme%26aufirst%3DV.%2BP.%26aulast%3DSeth%26aufirst%3DP.%26atitle%3DTargeting%2520Lactate%2520Dehydrogenase--a%2520Inhibits%2520Tumorigenesis%2520and%2520Tumor%2520Progression%2520in%2520Mouse%2520Models%2520of%2520Lung%2520Cancer%2520and%2520Impacts%2520Tumor-Initiating%2520Cells%26jtitle%3DCell%2520Metab.%26date%3D2014%26volume%3D19%26spage%3D795%26epage%3D809%26doi%3D10.1016%2Fj.cmet.2014.03.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yu, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chou, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tu, H. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ho, Y. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khoo, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, G. D.</span></span> <span> </span><span class="NLM_article-title">Antibody-Assisted Target Identification Reveals Afatinib, an Egfr Covalent Inhibitor, Down-Regulating Ribonucleotide Reductase</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">21512</span>– <span class="NLM_lpage">21529</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.25177</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=10.18632%2Foncotarget.25177" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=29765556" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;key=1%3ACAS%3A280%3ADC%252BC1Mfjs1OksQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=21512-21529&author=C.+H.+Yuauthor=C.+C.+Chouauthor=H.+F.+Tuauthor=W.+C.+Huangauthor=Y.+Y.+Hoauthor=K.+H.+Khooauthor=M.+S.+Leeauthor=G.+D.+Chang&title=Antibody-Assisted+Target+Identification+Reveals+Afatinib%2C+an+Egfr+Covalent+Inhibitor%2C+Down-Regulating+Ribonucleotide+Reductase&doi=10.18632%2Foncotarget.25177"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Antibody-assisted target identification reveals afatinib, an EGFR covalent inhibitor, down-regulating ribonucleotide reductase</span></div><div class="casAuthors">Yu Cheng-Han; Huang Wei-Chieh; Ho Ya-Yeh; Khoo Kay-Hooi; Chang Geen-Dong; Chou Chi-Chi; Khoo Kay-Hooi; Tu Hsin-Fang; Lee Ming-Shyue</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">30</span>),
    <span class="NLM_cas:pages">21512-21529</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Afatinib, used for the first-line treatment of non-small-cell lung carcinoma (NSCLC) patients with distinct epidermal growth factor receptor (EGFR) mutations, inactivates EGFR by mimicking ATP structure and forming a covalent adduct with EGFR.  We developed a method to unravel potential targets of afatinib in NSCLC cells through immunoprecipitation of afatinib-labeling proteins with anti-afatinib antiserum and mass spectrometry analysis.  Ribonucleotide reductase (RNR) is one of target proteins of afatinib revealed by this method.  Treatment of afatinib at 10-100 nM potently inhibited intracellular RNR activity in an in vitro assay using permeabilized PC-9 cells (formerly known as PC-14).  PC-9 cells treated with 10 μM afatinib displayed elevated markers of DNA damage.  Long-term treatment of therapeutic concentrations of afatinib in PC-9 cells caused significant decrease in protein levels of RNR subunit M2 at 1-10 nM and RNR subunit M1 at 100 nM.  EGFR-null Chinese hamster ovary (CHO) cells treated with afatinib also showed similar effects.  Afatinib repressed the upregulation of RNR subunit M2 induced by gemcitabine.  Covalent modification with afatinib resulting in inhibition and protein downregulation of RNR underscores the therapeutic and off-target effects of afatinib.  Afatinib may serve as a lead compound of chemotherapeutic drugs targeting RNR.  This method can be widely used in the identification of potential targets of other covalent drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTezKTz513V2Uh5eUSa2oe_fW6udTcc2eYwtKVmw5nqeLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1Mfjs1OksQ%253D%253D&md5=0eda3616bbc935ae81c8b5602322af1e</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.25177&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.25177%26sid%3Dliteratum%253Aachs%26aulast%3DYu%26aufirst%3DC.%2BH.%26aulast%3DChou%26aufirst%3DC.%2BC.%26aulast%3DTu%26aufirst%3DH.%2BF.%26aulast%3DHuang%26aufirst%3DW.%2BC.%26aulast%3DHo%26aufirst%3DY.%2BY.%26aulast%3DKhoo%26aufirst%3DK.%2BH.%26aulast%3DLee%26aufirst%3DM.%2BS.%26aulast%3DChang%26aufirst%3DG.%2BD.%26atitle%3DAntibody-Assisted%2520Target%2520Identification%2520Reveals%2520Afatinib%252C%2520an%2520Egfr%2520Covalent%2520Inhibitor%252C%2520Down-Regulating%2520Ribonucleotide%2520Reductase%26jtitle%3DOncotarget%26date%3D2018%26volume%3D9%26spage%3D21512%26epage%3D21529%26doi%3D10.18632%2Foncotarget.25177" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3gr4" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3gr4','PDB','3gr4'); return false;">PDB: 3gr4</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-5"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i41"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00763">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_44206"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.9b00763">10.1021/acs.jmedchem.9b00763</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Kinase profiling analysis of the effect of compound <b>1</b> on the activities of 246 different kinases; kinase profiling analysis of the effect of compound <b>1</b> on the substrate binding to 143 different kinases; structures of small-molecule modulators of PKM2; identification of compound <b>1</b>–cysteine conjugate by mass spectral analysis; sequence of PKM2 and the distribution of 10 cysteine residues; purification of recombinant PKM2; images of apo PKM2 crystals at day 16 of the crystallization process; images of PKM2 crystals socked with compound <b>1</b> at different time intervals; and Western blot analysis of the relative PKM2 abundance in six lung cancer cell lines (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00763/suppl_file/jm9b00763_si_001.pdf">PDF</a>)</p></li><li><p class="inline">NMR, MS, and purity determination data (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00763/suppl_file/jm9b00763_si_002.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00763/suppl_file/jm9b00763_si_003.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00763/suppl_file/jm9b00763_si_001.pdf">jm9b00763_si_001.pdf (2.31 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00763/suppl_file/jm9b00763_si_002.pdf">jm9b00763_si_002.pdf (5.13 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00763/suppl_file/jm9b00763_si_003.csv">jm9b00763_si_003.csv (0.39 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.9b00763&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.9b00763%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2019.62.issue-18%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.9b00763" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                6MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"67995d92ae8f3d94","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
