
<html lang="en"     class="pb-page"  data-request-id="4629f569-c55a-458c-a9a3-55f5fbf861b8"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;issue:issue:10.1021/jmcmar.2021.64.issue-11;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;article:article:10.1021/acs.jmedchem.1c00154;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Synthetic RNA Modulators in Drug Discovery" /></meta><meta name="dc.Creator" content="Farzad  Zamani" /></meta><meta name="dc.Creator" content="Takayoshi  Suzuki" /></meta><meta name="dc.Description" content="RNAs are involved in an enormous range of cellular processes, including gene regulation, protein synthesis, and cell differentiation, and dysfunctional RNAs are associated with disorders such as ca..." /></meta><meta name="Description" content="RNAs are involved in an enormous range of cellular processes, including gene regulation, protein synthesis, and cell differentiation, and dysfunctional RNAs are associated with disorders such as ca..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="June 1, 2021" /></meta><meta name="dc.Type" content="review-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.1c00154" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Â© 2021 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.1c00154" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.1c00154" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.1c00154" /></link>
        
    
    

<title>Synthetic RNA Modulators in Drug Discovery | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.1c00154" /></meta><meta property="og:title" content="Synthetic RNA Modulators in Drug Discovery" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0049.jpeg" /></meta><meta property="og:description" content="RNAs are involved in an enormous range of cellular processes, including gene regulation, protein synthesis, and cell differentiation, and dysfunctional RNAs are associated with disorders such as cancers, neurodegenerative diseases, and viral infections. Thus, the identification of compounds with the ability to bind RNAs and modulate their functions is an exciting approach for developing next-generation therapies. Numerous RNA-binding agents have been reported over the past decade, but the design of synthetic molecules with selectivity for specific RNA sequences is still in its infancy. In this perspective, we highlight recent advances in targeting RNAs with synthetic molecules, and we discuss the potential value of this approach for the development of innovative therapeutic agents." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.1c00154"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00154">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=âtrueâ></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=âtrueâ></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=âtrueâ></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=âtrueâ></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.1c00154&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.1c00154&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.1c00154&amp;href=/doi/10.1021/acs.jmedchem.1c00154" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2021</span><span class="cit-fg-volume">, 64</span><span class="cit-fg-issue">, 11</span><span class="cit-fg-pageRange">, 7110-7155</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/64/11" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.1c00115" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Perspective</span><a href="/doi/10.1021/acs.jmedchem.1c00261" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Synthetic RNA Modulators in Drug Discovery</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Farzad Zamani</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Farzad Zamani</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">The Institute of Scientific and Industrial Research, Osaka University, Mihogaoka 8-1, Ibaraki, Osaka 567-0047, Japan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Farzad++Zamani">Farzad Zamani</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath1">View Biography</a></span></div></div></span></li><span class="comma-separator">Â andÂ </span><li><span><span class="hlFld-ContribAuthor">Takayoshi Suzuki</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Takayoshi Suzuki</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">The Institute of Scientific and Industrial Research, Osaka University, Mihogaoka 8-1, Ibaraki, Osaka 567-0047, Japan</div></div><span class="conrtib-corresp"><strong>*</strong><a href="/cdn-cgi/l/email-protection#14607f6d6767616e617f7d5467757a7f717a3a7b67757f7539613a75773a7e64"><span class="__cf_email__" data-cfemail="3b4f504248484e414e50527b485a55505e551554485a505a164e155a5815514b">[emailÂ protected]</span></a></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Takayoshi++Suzuki">Takayoshi Suzuki</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath2">View Biography</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0003-2439-879X" title="Orcid link">https://orcid.org/0000-0003-2439-879X</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.1c00154&amp;href=/doi/10.1021%2Facs.jmedchem.1c00154" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2021</span></span><span class="cit-volume">, 64</span><span class="cit-issue">, 11</span><span class="cit-pageRange">, 7110â7155</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">June 1, 2021</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>27 January 2021</li><li><span class="item_label"><b>Published</b> online</span>1 June 2021</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 10 June 2021</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.1c00154" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.1c00154</a></div><div class="article_header-article-copyright"><strong>Copyright Â© 2021 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D7110%26pageCount%3D46%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DFarzad%2BZamani%252C%2BTakayoshi%2BSuzuki%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D64%26issueNum%3D11%26contentID%3Dacs.jmedchem.1c00154%26title%3DSynthetic%2BRNA%2BModulators%2Bin%2BDrug%2BDiscovery%26numPages%3D46%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D7155%26publicationDate%3DJune%2B2021">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.1c00154"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">2049</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val">-</div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.1c00154" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Synthetic RNA Modulators in Drug Discovery&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Farzad&quot;,&quot;last_name&quot;:&quot;Zamani&quot;},{&quot;first_name&quot;:&quot;Takayoshi&quot;,&quot;last_name&quot;:&quot;Suzuki&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2021&quot;,&quot;month&quot;:&quot;06&quot;,&quot;day&quot;:&quot;01&quot;,&quot;issue&quot;:&quot;11&quot;,&quot;volume&quot;:&quot;64&quot;,&quot;pages&quot;:&quot;7110-7155&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.1c00154&quot;},&quot;abstract&quot;:&quot;RNAs are involved in an enormous range of cellular processes, including gene regulation, protein synthesis, and cell differentiation, and dysfunctional RNAs are associated with disorders such as cancers, neurodegenerative diseases, and viral infections. Thus, the identification of compounds with the ability to bind RNAs and modulate their functions is an exciting approach for developing next-generation therapies. Numerous RNA-binding agents have been reported over the past decade, but the design of synthetic molecules with selectivity for specific RNA sequences is still in its infancy. In this perspective, we highlight recent advances in targeting RNAs with synthetic molecules, and we discuss the potential value of this approach for the development of innovative therapeutic agents.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00154&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00154" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00154&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00154" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00154&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00154" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.1c00154&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00154&amp;href=/doi/10.1021/acs.jmedchem.1c00154" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.1c00154" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.1c00154" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (9 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00154&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.1c00154%26sid%3Dliteratum%253Aachs%26pmid%3D34060847%26genre%3Darticle%26aulast%3DZamani%26date%3D2021%26atitle%3DSynthetic%2BRNA%2BModulators%2Bin%2BDrug%2BDiscovery%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D64%26issue%3D11%26spage%3D7110%26epage%3D7155%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291350" title="Assays">Assays</a>,</li><li><a href="/action/doSearch?ConceptID=290896" title="Cells">Cells</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=292258" title="Cancer">Cancer</a>,</li><li><a href="/action/doSearch?ConceptID=291439" title="Genetics">Genetics</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/64/11" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/jmcmar.2021.64.issue-11/20210610/jmcmar.2021.64.issue-11.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/medium/jm1c00154_0049.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0049.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00154&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">RNAs are involved in an enormous range of cellular processes, including gene regulation, protein synthesis, and cell differentiation, and dysfunctional RNAs are associated with disorders such as cancers, neurodegenerative diseases, and viral infections. Thus, the identification of compounds with the ability to bind RNAs and modulate their functions is an exciting approach for developing next-generation therapies. Numerous RNA-binding agents have been reported over the past decade, but the design of synthetic molecules with selectivity for specific RNA sequences is still in its infancy. In this perspective, we highlight recent advances in targeting RNAs with synthetic molecules, and we discuss the potential value of this approach for the development of innovative therapeutic agents.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_68353" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_68353" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Potential drug targets include the approximately 20â¯000 expressed proteins in human cells, together with their various post-translational modifications (PTMs),<a onclick="showRef(event, 'ref1 ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3">(1â3)</a> but among these proteins, only around 10â15% are known to be disease-associated.<a onclick="showRef(event, 'ref4 ref5'); return false;" href="javascript:void(0);" class="ref ref4 ref5">(4,5)</a> Moreover, many of the disease-related proteins are considered undruggable, in the sense that small molecules cannot bind to them with high specificity or affinity.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> Thus, there is considerable interest in RNAs as alternative targets. Most of the human genome is transcribed into various classes of RNAs, but protein-coding exons account for less than 2% of the total.<a onclick="showRef(event, 'ref7 ref8'); return false;" href="javascript:void(0);" class="ref ref7 ref8">(7,8)</a> However, noncoding RNAs (ncRNAs) are involved in a wide range of cellular functions, including RNA splicing and gene regulation.<a onclick="showRef(event, 'ref9 ref10 ref11 ref12 ref13'); return false;" href="javascript:void(0);" class="ref ref9 ref10 ref11 ref12 ref13">(9â13)</a> Therefore, targeting RNAs, especially ncRNAs, is emerging as a promising strategy to modulate cellular events involved in human diseases, particularly those involving undruggable proteins.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a></div><div class="NLM_p">In this context, small molecule RNA modulators have a number of potential advantages, such as easy tunability of their chemical and biophysical properties, including hydrophobicity, which is important for achieving sufficient cell and tissue permeability. Highly selective binding to secondary or tertiary structures of RNAs is also critical to effectively modulate cellular processes such as protein translation, homeostasis, and localization.<a onclick="showRef(event, 'ref14 ref15 ref16 ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref14 ref15 ref16 ref17 ref18">(14â18)</a> Several reviews dealing with various aspects of molecular recognition and clinical development of RNA-targeted small molecules have been published.<a onclick="showRef(event, 'ref6 ref15 ref19 ref20 ref21 ref22 ref23 ref24'); return false;" href="javascript:void(0);" class="ref ref6 ref15 ref19 ref20 ref21 ref22 ref23 ref24">(6,15,19â24)</a> In this perspective, we aim to provide a broad overview of recent developments in synthetic protocols, mechanistic studies, and therapeutic evaluation of synthetic modulators of microRNAs (miRNAs), long ncRNAs (lncRNAs), and mRNAs relevant to human diseases.</div><div id="sec2" class="NLM_sec NLM_sec_level_2"><div id="ac_i3" class="anchor-spacer"></div><h3 class="article-section__title" id="_i3">1.  Application of Synthetic miRNA Modulators in Drug Discovery</h3><div class="NLM_p">miRNAs are a family of ncRNAs â¼22 nucleotides in length, generated <i>via</i> processing of longer RNAs by the ribonucleases Drosha and Dicer, which serve as master regulators of miRNA processing (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>). They regulate numerous protein-coding genes by binding to partially paired sequences in mRNAs, which leads either to inhibition of translation or to cleavage of the mRNA in the presence of RNA-induced silencing complex (RISC).<a onclick="showRef(event, 'ref25 ref26'); return false;" href="javascript:void(0);" class="ref ref25 ref26">(25,26)</a> Misregulation of miRNAs has been linked to the onset and development of many cancers, including glioblastomas and breast, prostate, lung, and pancreatic tumors.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> Therefore, interfering with miRNA biogenesis could be an effective strategy to induce apoptosis of tumor cells (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>). Antisense oligonucleotides (ASO) are available for inhibiting the function of miRNAs<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> but suffer from limitations such as poor cell permeability, intrinsic off-target impacts, and a poor pharmacokinetic profile.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> Therefore, alternative methods are needed to modulate miRNAs in cancer biology. In this respect, synthetic miRNA inhibitors are promising candidates.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/medium/jm1c00154_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. miRNA biogenesis and putative inhibition of Dicer-mediated cleavage of pre-miRNA by synthetic molecules.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00154&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div id="sec2_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i4" class="anchor-spacer"></div><h4 class="article-section__title" id="_i4">1.1.  Synthetic Inhibitors of miR-21 Maturation or miR-21 Expression</h4><div class="NLM_p">Schneekloth and co-workers employed a microarray screening technique to scan a large library of drug-like small molecules for the ability to bind to pre-miR-21 hairpin.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> In this strategy, molecules were covalently attached to a glass surface by a robotic microarrayer, hybridized with a fluorescently labeled pre-miR-21 hairpin, and imaged. Molecules binding with the RNA induce an increase in fluorescence. On the basis of the screening results, they synthesized a range of carbazole analogues <b>5</b> by <i>N</i>-alkylation of carbazoles <b>1</b> and epoxide ring-opening reaction with amines <b>4</b> (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>). Various functional groups were introduced to the core structure of <b>5</b> to investigate the structureâactivity relationship (SAR). The results showed that the dibromocarbazole unit is important for the interaction of small molecules <b>5</b> with the RNAs, and the effect of aniline substitution is negligible. The biological activity of the synthesized compounds was evaluated in terms of suppression of the levels of mature miR-21 in cellular models due to inhibition of Dicer-mediated processing of pre-miR-21 hairpin. Differential scanning fluorometry, fluorescence intensity assay, and 2-aminopurine (2-AP)-labeled fluorescence titration assays indicated that compounds <b>5a</b> and <b>5b</b> were the most potent inhibitors, and further analysis showed that <b>5a</b> and <b>5b</b> exhibited 47% and 59% inhibition of Dicer processing, respectively, at the concentration of 1 Î¼M. It was suggested that binding of these molecules to miR-21 either physically restricted the accessibility of the pre-miRNA to Dicer or resulted in the formation of structurally defective pre-miRNA. This study confirmed that compounds <b>5</b> bind efficiently to pre-miR-21 hairpin, but further studies using cell-based assays or animals are needed to assess their potential as chemical probes and/or therapeutic agents.</div><figure id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/medium/jm1c00154_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Carbazole Analogues and Inhibitory Activity of <b>5a,b</b> toward Dicer Processing of pre-miR-21</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00154&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">miRNAs, especially miR-21, repress important tumor-suppressor proteins such as PDCD4 and phosphatase and tensin homologue (PTEN) in tumor cells, leading to chemoresistance.<a onclick="showRef(event, 'ref31 ref32'); return false;" href="javascript:void(0);" class="ref ref31 ref32">(31,32)</a> For example, miR-21 promotes cell transformation and metastasis of renal cell carcinoma (RCC) by blocking the expression of PTEN and PDCD4 genes. Deitersâ group used a high-throughput luciferase-based cellular assay to screen a library of small molecules.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> Their assay employed a paired miRNA-binding site introduced into the 3â²-untranslated region (UTR) of the gene of firefly luciferase as a reporter construct, which was installed in HeLa cells. In this assay, down-regulation of miR-21 expression results in a significant increase of the luminescence signal. They identified a small molecule miR-21 expression inhibitor <b>8</b> with an azobenzene core, which was synthesized by simple amide formation using propargyl amine <b>7</b> (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>). Compound <b>8</b> produced a 500% increase in the luciferase signal intensity compared with a DMSO control at 10 Î¼M concentration. The SAR study revealed that replacing the amide with a sulfonamide group in compound <b>8</b> resulted in loss of activity. Other compound <b>8</b> derivatives bearing alkyl chains, such as allyl and propyl, instead of the propargyl unit displayed 11â53% lower activities in luciferase reporter assays. The selectivity of <b>8</b> for the inhibition of miR-21 expression was also evaluated by using quantitative RT-PCR to determine the levels of intracellular miRNAs, including miR-21, miR-30, and miR-93. The miR-21 expression was selectively reduced by 78%. Further studies are needed to identify the target molecule of this class of inhibitors that regulate the expression of miR-21 in living cells.</div><figure id="sch2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/medium/jm1c00154_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis and miR-21 Maturation-Inhibitory Activity of Azobenzene-Based Compound <b>8</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00154&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">A scaffold-hopping strategy was used to develop 4-benzoylamino-<i>N</i>-(prop-2-yn-1-yl)benzamide derivatives <b>13</b> and <b>18</b> as oncogenic miR-21 expression inhibitors.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> As derivatization of <b>8</b> is difficult, limiting the ability to explore more effective analogues,<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> Luâs group aimed to replace the (<i>E</i>)-1,2-diphenylazene moiety in <b>8</b> with <i>N</i>-phenylbenzamide. Two different types of benzamide derivatives were synthesized in good to high yields from commercially available 4-nitrobenzoyl chloride <b>9</b> and 4-methoxycarbonylbenzoic acid <b>14</b> (<a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>). The activity of compounds <b>13</b> and <b>18</b> to inhibit the expression of miR-21 in human cervical epithelioid carcinoma (HeLa) and human glioblastoma U-87 MG cells was examined using flow cytometric and RT-qPCR techniques. Compounds bearing bulky groups on the phenyl ring exhibited higher activity, and <b>13a</b> showed the highest antitumor efficacy at 10 Î¼M concentration in both cancer lines. It increased apoptosis and upregulated programmed cell death protein 4 (PDCD4), inducing apoptosis in approximately 24% and 13% of HeLa and U-87 MG cells, respectively.</div><figure id="sch3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/medium/jm1c00154_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of 4-Benzoylamino-<i>N</i>-(prop-2-yn-1-yl)benzamides and Antitumor Activity toward HeLa and U-87 MG Cells</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00154&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In another study, Deiters and co-workers synthesized a large number of novel miR-21 inhibitors containing both oxadiazole and triazole rings.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> Screening with the previously reported high-throughput luciferase-based cell assay<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> identified two 1,2,3-triazolyl-1,2,4-oxadiazoles as miR-21 inhibitors. On the basis of the structure of these hit compounds, novel 1,2,3-triazolyl-1,2,4-oxadiazole derivatives <b>25</b> (<a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>) were designed and synthesized to examine the SAR. To obtain compounds <b>25</b>, azidobenzenes <b>20</b> and cyanomethylene-functionalized oxadiazoles <b>24</b> were synthesized separately from anilines <b>19</b> and nitriles <b>21</b>, respectively, and then coupled (<a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>). The SAR study showed that while modification of the phenyl ring in molecules <b>25</b> had a major impact on their activity in HeLa-miR-21-Luc cell-based assays, modification of the triazole and oxadiazole units had only minor effects on the miR-21-inhibitory activity of these derivatives. For example, the introduction of an <i>O</i>-propargyl group (R<sup>2</sup>) on the phenyl ring resulted in a substantial increase in the luciferase signal from RLU = 2.19 to RLU = 12.19 at 10 Î¼M. HeLa-miR-21-Luc cell-based assay identified compound <b>25a</b> as a promising candidate, and its antitumor activity was evaluated in several cancer cell lines, including RCC, pancreatic carcinoma, glioblastoma, breast carcinoma, and colorectal carcinoma, by means of XTT cell viability assays. Excellent inhibitory activities were observed in all cases, except for RCC, with IC<sub>50</sub> values between 3.3 and 6.7 Î¼M. Although the RCC A498 cell line was resistant to <b>25a</b> (IC<sub>50</sub> > 100 Î¼M), it was found that this compound increased the expression of PDCD4 and PTEN, which enhanced the chemosensitivity of RCC cells to the clinically approved chemotherapeutic topotecan. These findings in cellular assays highlight the potential value of the oxadiazole miR-21 inhibitors <b>25</b> as chemotherapeutic agents to treat human cancers.</div><figure id="sch4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/medium/jm1c00154_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis and Antitumor Activity of Oxadiazole-Based miR-21 Inhibitors</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00154&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Deitersâ group also applied their luciferase-based cell assay in a high-throughput screen of a library of small molecules for miR-21 inhibitors and identified a new type of ether-amide heterocyclic compounds as miR-21 inhibitors.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> They then synthesized a wide range of benzoxazoles <b>31</b>, benzothiazoles <b>34</b>, and benzimidazoles <b>37</b>. Benzazole derivatives <b>29</b>, <b>33</b>, and <b>36</b> were prepared from commercially available 2-aminophenols <b>26</b>, 2-aminothiophenols <b>32</b>, and <i>ortho</i>-phenylenediamines <b>35</b>, followed by amidation reaction and phenol alkylation to afford the desired products <b>31</b>, <b>34</b>, and <b>37</b> (<a class="ref internalNav" href="#sch5" aria-label="Scheme 5">Scheme 5</a>). The SAR study revealed that the 5-chlorobenzoxazole and the ether amide linker are crucial for miR-21 maturation-inhibitory activity. Among compounds <b>31</b>, <b>34</b>, and <b>37</b>, compound <b>31a</b> showed the highest activity in luciferase-based live cell reporter assay, generating a 4.87-fold increase in the luciferase signal at 10 Î¼M. Regarding the mechanism of action, the lead compound <b>31a</b> was shown to hinder transcription of the miR-21 gene, resulting in considerable reductions of primary and mature miR-21 levels. The miR-21-inhibitory activity of <b>31a</b> was then evaluated in cervical carcinoma HeLa cells, non-small cell lung carcinoma A549 cells, and ovarian cancer SKOV3 cells, resulting in 57%, 33%, and 31% decreases, respectively, in the levels of mature miR-21.</div><figure id="sch5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/medium/jm1c00154_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Synthesis and miR-21 Maturation-Inhibitory Activity of EtherâAmide Compounds</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00154&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Liang <i>et al</i>. reported an efficient protocol to block miRNAs maturation by using small molecule Dicer inhibitors.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> They employed a fluorescence polarization (FP) assay to identify compounds targeting pre-miR-21. In their assay, the fluorescence polarization of a fluorophore-labeled small molecule is increased upon binding to RNA, due to a reduction in the rate of fluorophore tumbling. This FP-based screening led to the identification of neomycin <b>45</b> (<a class="ref internalNav" href="#sch6" aria-label="Scheme 6">Scheme 6</a>) as a pre-miR-21 binding ligand. Compounds <b>47</b> were then designed and synthesized <i>via</i> coupling between the <i>N</i>-hydroxyimides <b>44</b> as a low-affinity Dicer-inhibiting unit and neomycin <b>45</b> as a high-affinity pre-miRNA-binding unit (<a class="ref internalNav" href="#sch6" aria-label="Scheme 6">Scheme 6</a>). Esterification, Mitsunobu reaction, and hydrolysis of 2-(carboxymethyl)-5-hydroxybenzoic acid <b>38</b> provided the key intermediate <b>42</b>, which was treated with <i>O</i>-(4-methoxybenzyl)-hydroxylamine <b>43</b> followed by acid-catalyzed deprotection to give imide <b>44</b>. Azide <b>46</b> was prepared separately from neomycin <b>45</b> in four steps and coupled with <b>44</b> to afford the final bifunctional structure <b>47</b>. The activity of these bifunctional small molecules to block miR-21 biogenesis by inhibiting pre-miR-21 cleavage was evaluated <i>in vitro</i>. The results indicated that the efficacy was critically dependent on the distance between the two units; a greater distance between these components significantly decreased the miRNAs maturation-inhibitory activity. Compound <b>47</b> with the shortest length displayed the highest Dicer-inhibitory potency, showing a 90% reduction of miR-21 formation at 5 Î¼M concentration. To explore the selectivity of the bifunctional molecule <b>47</b> for miRNA regulation, the ability to inhibit miR-34a in HEK293T cells was also investigated, and a 40% reduction of miR-34a expression was observed at 5 Î¼M concentration. The results indicate that compound <b>47</b> exhibits some degree of selectivity for the inhibition of miR-21 in this cell line. Since neomycin is well-known to target rRNA and influence translation, the effect of compound <b>47</b> on translation was also examined by transfecting CHO and HEK293T cell lines with an EGFP reporter and treating them with compound <b>47</b>. No marked change of EGFP expression and no cytotoxicity were seen at 5 Î¼M concentration of the ligand. These observations suggest that compound <b>47</b> has no effect on translation and shows no toxicity.</div><figure id="sch6" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/medium/jm1c00154_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0007.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 6. Synthesis and Dicer-Inhibitory Activity of the Neomycin-Based miR-21 Bifunctional Inhibitor</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00154&amp;id=sch6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Epithelialâmesenchymal transition (EMT), which involves the transformation of epithelial cells to a mesenchymal phenotype, occurs during embryonic development and tissue regeneration. Dysregulation of EMT leads to loss of cellular polarity, cellâcell adhesion, and defects in the cytoskeleton and is associated with a cancer phenotype, as seen in breast tumors.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> miR-21 is one of the onco-miRNAs regulating the formation of PTEN, which is involved in EMT.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> Therefore, the inhibition of miR-21 is considered a potential strategy to induce apoptosis of tumor cells. To identify miR-21 inhibitors, Maiti and co-workers designed peptidic aminosugar (PA)-kanamycin derivatives <b>52</b> (<a class="ref internalNav" href="#sch7" aria-label="Scheme 7">Scheme 7</a>).<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> They performed PALâPEG-PS resin-based synthesis of several PA-kanamycin derivatives <b>52</b>, where the peptide chain was grown on FmocâPAL-PEG-PS resin <b>48</b> and finally coupled with a kanamycin acid analogue to give the corresponding conjugate <b>51</b>, which was cleaved from the resin under acidic conditions. The products <b>52</b> were screened in a high-throughput displacement assay developed by the same research group,<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> in which a fluorescent conjugated neomycin <b>45</b> (F-<b>45</b>) (<a class="ref internalNav" href="#sch7" aria-label="Scheme 7">Scheme 7</a>) was used to screen binding specificity for pre-miR-21. In this assay, binding of F-<b>45</b> to pre-miR-21 quenches the fluorescence, so displacement of F-<b>45</b> by other binders is detected as a fluorescence increase. Among the synthesized compounds, analogue <b>52a</b> (<a class="ref internalNav" href="#sch7" aria-label="Scheme 7">Scheme 7</a>) containing cysteine and phenylalanine showed 140% higher affinity for miR-21 than kanamycin alone. Compound <b>52a</b> also displayed strong binding affinity (<i>K</i><sub>d</sub> = 4.36 nM) to miR-21 with a relatively low dissociation rate from the target miRNA, causing relatively long-lasting miR-21 inhibition, which is highly desirable to target regulation of miRNA biogenesis with low toxicity in cancer treatment. In cell-based assays, compound <b>52a</b> significantly down-regulated the mature transcript of miR-21 and increased the expression of both PTEN and PDCD4, cognate targets of miR-21, at 5 Î¼M concentration. Furthermore, compound <b>52a</b> reversed EMT in MCF-7 breast cancer cells, suggesting potential efficacy of small molecule miR-21 inhibitors for cancer therapy.</div><figure id="sch7" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/medium/jm1c00154_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0008.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 7. Synthesis and miR-21 Maturation-Inhibitory Activity of PA-Kanamycins</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00154&amp;id=sch7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Colorectal cancer (CRC), accounts for about 9% of all cancers worldwide.<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> miR-21 is upregulated in CRC cells and promotes the development of CRC by inducing resistance to the programmed cell death.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> Caoâs group demonstrated that regorafenib <b>58</b>, a broad-spectrum kinase inhibitor, directly interacts with pre-miR-21 and inhibits Dicer processing of mature miR-21.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> Regorafenib <b>58</b> was synthesized from commercially available 4-chloropyridine-2-carboxylic acid <i>tert</i>-butyl ester <b>53</b> by amide formation followed by aromatic nucleophilic substitution reaction and urea formation (<a class="ref internalNav" href="#sch8" aria-label="Scheme 8">Scheme 8</a>). Several <i>in silico</i> and <i>in vitro</i> analyses were employed to confirm the proposed molecular binding mechanism of regorafenib to pre-miR-21, including RNA-ligand docking, binding energy analysis, fluorescence-based assay, and molecular dynamics simulation. The results indicated that regorafenib binds to the stem region of pre-miR-21 <i>via</i> three hydrogen bonds with three key nucleotide residues, G6, A7, and A13, in the pocket and several hydrophobic interactions, resulting in high affinity for the complex system (<a class="ref internalNav" href="#sch8" aria-label="Scheme 8">Scheme 8</a>bc) (<i>K</i><sub>d</sub> = 0.73 Î¼M). This study offered a new potential approach for anti-CRC chemotherapy based on regorafenib, <i>i.e.</i>, inhibiting the Dicer-mediated cleavage of pre-miR-21 to reduce the level of mature miR-21, thereby inducing apoptosis of tumor cells. Further cell-based analyses are needed to evaluate the inhibitory activity of regorafenib in various cancer cell lines.</div><figure id="sch8" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/medium/jm1c00154_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0009.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 8. (a) Synthesis and miR-21 Maturation-Inhibitory Activity of Regorafenib; (b, c) Binding Hotspots (G6, A7, and A13) for Ligand Binding of Regorafenib on the Surface of the miR-21 Pre-element<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00154&amp;id=sch8"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reprinted with permission from ref <a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> [  <cite><i>J. Mol. Graphics Modell.</i></cite> <span class="NLM_year">2017</span>, <em>73</em>, 48â53]. Copyright 2017 Elsevier.</p></p></figure><div class="NLM_p">Disney <i>et al</i>. used a computational approach known as Inforna to design a non-oligonucleotide-containing structure that would simultaneously recruit RNase L and selectively target pre-miR-21 for enzymatic destruction in cancer cells.<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> Inforna is a selection-based strategy using two-dimensional combinatorial screening (2DCS) of structureâactivity relationships through sequencing (StARTS) to detect RNA motif-small molecule interactions. This technique is available to design small molecule binders to Drosha and Dicer processing sites in human miRNAs.<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> The designed heterocyclic ligand <b>69</b>, consisting of a miR-21 binder <b>63</b> and a RNase recruiter, was synthesized from synthetically accessible starting materials <i>via</i> multiple steps, including Cu-catalyzed click reaction (<a class="ref internalNav" href="#sch9" aria-label="Scheme 9">Scheme 9</a>). The binder segment showed high binding affinity for pre-miR-21, with <i>K</i><sub>d</sub> = 0.98 Î¼M. The binding affinity to miR-21 was examined with a Monolith NT.115 microscale thermophoresis instrument using Cy5-labeled model miR-21 molecules, including miR-21 WT (5â²-Cy5-GUUGACUGUUGAAUCUCAUGGCAAC-3â²) and miR-21 BP (5â²-Cy5-GUUGACUGUUGAAUCUCAAUGGUCAAC-3â²). Coupling of the binder and recruiter components resulted in enhanced inhibitory activity toward miR-21 in MDA-MB-231 cells, and <b>69</b> showed an IC<sub>50</sub> value of 0.05 Î¼M. Further analysis revealed that <b>69</b> split pre-miR-21 in MDA-MB-231 cells in a substoichiometric manner: 1 mol of <b>69</b> cleaved 26 mol of pre-miR-21. Interestingly, while the scaffold <b>68</b> with OH at the <i>meta</i> position could efficiently recruit inactive monomeric ribonuclease for dimerization to active nuclease, the <i>meta</i>-OMe substituted compound <b>68</b> was completely inactive as a recruiter. Treatment with 50 nM <b>69</b> produced an approximately 50% decrease in mature miR-21 levels in MDA-MB-231, A375, and A549 cell lines, due to the enzymatic cleavage of pre-miR-21. To investigate the ability of <b>69</b> to prevent metastasis in <i>in vivo</i> assays, a lung nodule metastasis study was performed. MDA-MB-231 breast cancer-derived cells transfected with luciferase were inoculated into female mice having diabetes/severe combined immunodeficiency. After 6 days, the mice were treated with 10 mg/kg <i>q.o.d</i>. of <b>69</b>. This treatment significantly reduced the number oflung nodules that developed. At the same time, RNA fluorescence <i>in situ</i> hybridization (FISH) data indicated that the levels of pre-miR-21 and mature miR-21 were decreased by approximately 50%. These finding highlight the potential value of these molecules in cancer therapy.</div><figure id="sch9" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Scheme 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/medium/jm1c00154_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0010.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 9. Synthesis of a Non-oligonucleotide-Containing Ligand to Promote RNase-Mediated Destruction of pre-miR-21 in Cancer Cells</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00154&amp;id=sch9"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i14" class="anchor-spacer"></div><h4 class="article-section__title" id="_i14">1.2.  Synthetic Inhibitors of miR-372/373 Maturation</h4><div class="NLM_p">miR-372 and miR-373 promote tumorigenesis and proliferation in human cells <i>via</i> direct repression of large-tumor suppressor homologue 2 (LATS2), which is associated with cell-cycle regulation.<a onclick="showRef(event, 'ref47 ref48'); return false;" href="javascript:void(0);" class="ref ref47 ref48">(47,48)</a> A series of new neomycin-based small molecules <b>74</b> were prepared by using Cu-catalyzed click reaction between two RNA-binding units, the aminoglycoside neomycin <b>72</b> and various natural and artificial nucleobase analogues <b>73</b> (<a class="ref internalNav" href="#sch10" aria-label="Scheme 10">Scheme 10</a>).<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> The aim of this synthetic design was to obtain RNA ligands with improved selectivity and affinity compared to each parent compound. Introducing the artificial nucleobases into the ligand structure allows the formation of specific Hoogsteen-type hydrogen bonds with AU base pairs of RNA (<a class="ref internalNav" href="#sch10" aria-label="Scheme 10">Scheme 10</a>).<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> The natural nucleobases in the RNA ligands are also capable of base pairing with unpaired nucleosides of loops, which could increase binding affinity for the target RNA. These bifunctional scaffolds were examined for inhibitory activity toward adenocarcinoma gastric cancer (AGS) cell growth, on the basis of their ability to reduce the levels of miR-372 and miR-373 by blocking Dicer processing of pre-miRNA. Almost all the conjugated compounds exhibited excellent Dicer-inhibitory activity in gastric cancer cells. Compound <b>74a</b> was the most potent inhibitor, affording almost 100% inhibition with an IC<sub>50</sub> value of 2.42 Î¼M. These discoveries show that the multimodal ligands <b>74</b> are promising leads for anticancer agents, although further evaluations of their mechanism of action, specificity, and biological performance are needed.</div><figure id="sch10" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Scheme 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/medium/jm1c00154_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0011.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 10. Synthesis and Antitumor Activity of Neomycin-Based Small Molecules (R = Nucleobase Analogue) against Gastric Cancer Cells<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00154&amp;id=sch10"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>These compounds inhibit Dicer-mediated processing of pre-miR-372 and pre-miR-373.</p></p></figure><div class="NLM_p">Duca <i>et al</i>. also prepared various miRNA ligands <b>81</b> by coupling artificial nucleobases <b>79</b> and aminoglycosides <b>80</b> (<a class="ref internalNav" href="#sch11" aria-label="Scheme 11">Scheme 11</a>).<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> The coupling reaction of 2-amino-4-(3-nitrophenyl)thiazole <b>75</b> withbenzoyl chloride <b>76</b> first afforded <b>77</b>, and then hydrogenolysis of the nitro group and amide formation reaction gave artificial nucleobase <b>57</b>. Finally, 1,3-dipolar cycloaddition of <b>79</b> with the azide analogue of neomycin (<b>80</b>) afforded ligands <b>81</b> in high yields. The Dicer-inhibitory activity of these synthesized compounds was examined <i>in vitro</i> using a fluorescence resonance energy transfer (FRET) method with two target miRNAs (miR-372 and -373). To gain more insight into the SAR of the synthesized ligands, systematic modifications on the artificial nucleobase, linker, and aminoglycoside units were carried out. Increasing the length of the linker or altering the neomycin moiety did not increase the inhibitory activity. However, introduction of a benzoyl group <i>via</i> an amide linkage on the nucleobase increased the potency, and compounds <b>81a</b> and <b>81b</b> bearing <i>para</i>-chlorobenzoyl and <i>para</i>-aminobenzoyl groups, respectively, were found to be the most promising inhibitors, which might reflect a more favorable matching of the compounds to the RNA sequence/structure. While compound <b>81a</b> displayed the best performance toward pre-miR-372 with IC<sub>50</sub> values of 0.98 Î¼M (pre-miR-372) and 2.19 Î¼M (pre-miR-373), compound <b>81b</b> exhibited the highest activity for pre-miR-373 with IC<sub>50</sub> values of 1.08  and 1.10 Î¼M for blocking pre-miR-372 and pre-miR-373 cleavage, respectively. Antiproliferative activity of the lead ligands was also assessed in terms of AGS cell growth-inhibitory activity; <b>81a</b> showed higher activity, reducing cell growth by 50% at 25 Î¼M concentration.</div><figure id="sch11" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Scheme 11</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/medium/jm1c00154_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0012.jpeg" id="gr12" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 11. Synthesis and miR-372/miR-373 Cleavage-Inhibitory Activity of Nucleobase-Aminoglycoside-Based Compounds</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00154&amp;id=sch11"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Duca and co-workers uncovered a polyamine with potential inhibitory activity against <i>in vitro</i> Dicer processing of pre-miR-372 by screening a 640-member chemical library with a cell-free assay.<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> In their assay, pre-miRNA was labeled with fluorescein and dabcyl as fluorophore and quencher at the 3â² and 5â² ends, respectively. Cleavage by Dicer resulted in the appearance of fluorescence. The screening identified a spermine-amidine compound <b>86</b> with excellent binding affinity (<i>K</i><sub>d</sub> = 0.15 Î¼M), which suppressed Dicer-mediated cleavage of pre-miR-372 with the IC<sub>50</sub> value of 1.06 Î¼M. Compound <b>86</b> was synthesized <i><i>via</i></i> the coupling reaction of heterocyclic amidine <b>84</b> with <i>N</i>-Boc protected spermine <b>85</b> in 69% yield (<a class="ref internalNav" href="#sch12" aria-label="Scheme 12">Scheme 12</a>). It markedly inhibited miR-372 biogenesis in gastric cancer stem cells, thereby inhibiting the expansion of tumor stem cells.</div><figure id="sch12" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Scheme 12</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/medium/jm1c00154_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0013.jpeg" id="gr13" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 12. Synthesis and miR-372 Maturation-Inhibitory Activity of Spermine-Amidine Compound <b>86</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00154&amp;id=sch12"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i18" class="anchor-spacer"></div><h4 class="article-section__title" id="_i18">1.3.  Synthetic Inhibitors of miR-29a Maturation</h4><div class="NLM_p">Nakataniâs group discovered xanthone derivatives <b>92</b> (<a class="ref internalNav" href="#sch13" aria-label="Scheme 13">Scheme 13</a>), which were expected to bind molecular loops and bulges of pre-miRNAs, as inhibitors of pre-miR-29a Dicer processing.<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> Compounds <b>92</b> were prepared by means of Mitsunobu reaction and <i>N</i>-Boc deprotection, starting from commercially available xanthone <b>87</b> (<a class="ref internalNav" href="#sch13" aria-label="Scheme 13">Scheme 13</a>). Denaturing urea polyacrylamide gel electrophoresis indicated no direct interaction of <b>92</b> with Dicer, and instead it was proposed that inhibitor binds close to a Dicer cleavage site in pre-miR-29a, thereby blocking cleavage. The presence of a bulky methyl group on the aminoalkoxy chain significantly reduced the binding affinity of the ligand (<b>92</b>, R = Me), presumably by hindering its interaction with both loop and double-stranded regions of the miRNAs. Compound <b>92a</b> was the most potent inhibitor, producing a 41% reduction of mature miR-29a at 200 Î¼M, as determined by Northern blot analysis.</div><figure id="sch13" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Scheme 13</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/medium/jm1c00154_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0014.jpeg" id="gr14" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 13. Synthesis and miR-29a Maturation-Inhibitory Activity of Xanthone Derivatives</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00154&amp;id=sch13"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20">1.4.  Synthetic Inhibitors of miR-1 Expression</h4><div class="NLM_p">miR-1 is a muscle-specific miRNA with critical roles in heart function; its dysregulation can lead to various coronary artery disorders and heart failure.<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> A wide range of cyclobutene-based small molecules <b>95</b> were efficiently synthesized as selective miR-1 expression inhibitors <i><i>via</i></i> the photocycloaddition of 2-methoxy-1,4-naphthalenequinone <b>93</b> and various aryl acetylenes <b>94</b> (<a class="ref internalNav" href="#sch14" aria-label="Scheme 14">Scheme 14</a>).<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a> These compounds were initially tested for regulatory activity toward miR-1, miR-214, miR-25, and miR-150 in a luciferase reporter assay similar to that of Deitersâ group.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> The SAR study revealed that the presence of the 8-hydroxyquinoline ring on the cyclobutene ring is critical for inhibitory activity toward miR-1. Furthermore, the presence of two substituents on the CâC bond of the cyclobutene is important, since compounds <b>95</b> with R<sup>1</sup> or R<sup>2</sup> = H displayed greatly reduced activity. Compound <b>95a</b> was a selective and potent inhibitor of miR-1 expression, producing an approximately 7-fold increase of the luciferase signal in mouse myoblast C2C12 cells at 5 Î¼M concentration. The results should be helpful for the identification of further small molecule miR-1 probes, but studies to establish their mechanism of action in cells are needed.</div><figure id="sch14" data-index="15" class="article__inlineFigure"><h2 class="fig-label">Scheme 14</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/medium/jm1c00154_0015.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0015.jpeg" id="gr15" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 14. Synthesis and miR-1 Expression-Inhibitory Activity of 1,4-Naphthalenequinone Analogues</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00154&amp;id=sch14"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22">1.5.  Synthetic Inhibitors of miR-14 Expression</h4><div class="NLM_p"><i>aza</i>-Flavanone structures are found in a wide range of biologically active natural products, and Bhadra <i>et al</i>. reported a simple synthesis and antitumor activity of a range of 2-aryl-2,3-dihydroquinolin-4(1<i>H</i>)-ones (<i>aza</i>-flavanones) <b>98</b> as selective miR-14 inhibitors.<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a> They prepared a series of benzopyranone derivatives <b>98</b><i><i>via</i></i> one-pot cyclization of <i>ortho</i>-aminoacetophenone <b>96</b> and aromatic aldehydes <b>97</b> catalyzed by <span class="smallcaps smallerCapital">l</span>-proline under mild conditions (<a class="ref internalNav" href="#sch15" aria-label="Scheme 15">Scheme 15</a>). 3-(4,5-Dimethylthiazol-2-yl)-2,5-diphenyltetrazolium bromide (MTT) assay indicated that hit compounds <b>98a</b> and <b>98b</b> induced approximately 50% cell death of <i>Drosophila</i> (fruit fly) and human breast cancer MCF-7 cells at the concentration of 4 Î¼M. The presence of bromide or a hydroxyl group on the benzopyranone scaffold was essential for cytotoxic activity. For example, replacing the hydroxyl groups on ligand <b>98a</b> with either acetoxy or tosyloxy resulted in a significant reduction of miRNA-inhibitory activity. Compounds <b>98a</b> and <b>98b</b> at 4 Î¼M reduced the levels of miR-4644 and miR-14 by 80% in luciferase reporter assays, suggesting the potential value of miR-14 inhibitors as therapeutic agents. These synthetically accessible molecules could be a powerful platform for anticancer drug discovery.</div><figure id="sch15" data-index="16" class="article__inlineFigure"><h2 class="fig-label">Scheme 15</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/medium/jm1c00154_0016.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0016.jpeg" id="gr16" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 15. Synthesis and miR-4644/miR-14 Maturation-Inhibitory Activity of Benzopyranone Derivatives (<i>aza</i>-Flavanones) in MCF-7 Cells</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00154&amp;id=sch15"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24">1.6.  Synthetic Modulators of miR-122 Expression</h4><div class="NLM_p">miR-122 is the most abundant liver-specific miRNA, playing roles in hepatitis C virus (HCV) infection and development of hepatocellular carcinoma (HCC).<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a> Binding sites for miR-122 in the viral genome generally exist close to the 5â² end. In HCV-infected Huh7 cells, inhibition of miR-122 leads to a reduction of RNA levels, causing cell death. However, miR-122 is generally down-regulated in liver cancer cells compared to healthy liver cells, which suggests that upregulation of miR-122 might induce apoptosis in HCC. Deitersâ group synthesized benzamides (<b>100</b>), tetrahydroquinolines (<b>102</b>), and isoquinoline-1,3(2<i>H</i>)-diones (<b>104</b>) as candidate miR-122 regulators (<a class="ref internalNav" href="#sch16" aria-label="Scheme 16">Scheme 16</a>)<a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(58)</a> and used a screening assay based on the psiCHECK-2 reporter plasmid to identify interaction of these molecules with miR-122. The reporter construct incorporates both <i>Renilla</i> luciferase and firefly luciferase, and the targeted miR-122 sequence was installed downstream of the <i>Renilla</i> luciferase gene. Increase or reduction in the <i>Renilla</i> luciferase signal ratio indicated inhibition or activation of miR-122 function, respectively. Benzamide (<b>100</b>) and tetrahydroquinoline (<b>102</b>) derivatives displayed potent inhibitory activity, with compounds <b>100a</b> and <b>102a</b> providing 7-fold and 12-fold signal increases, respectively, at 10 Î¼M in Huh7 cells (<a class="ref internalNav" href="#sch16" aria-label="Scheme 16">Scheme 16</a>). However, the isoquinoline-1,3(2<i>H</i>)-dione analogues <b>104</b> were almost inactive at 10 Î¼M. The SAR study showed that the presence of halogens on the benzene ring in compounds <b>100</b> is essential, as replacement of the halogens with methoxy groups resulted in complete loss of inhibitory activity. Furthermore, introduction of either piperidine or dihydroquinoline instead of the <i>trans</i>-decahydroquinoline motif in compounds <b>102</b> led to loss of inhibitory activity. Removal of either the NH<sub>2</sub> group or <i>N</i>-methyl groups in ligands <b>104</b> also significantly diminished the activity. The miR-122 expression-inhibitory activity of the compounds was also assessed in terms of their direct effect on the expression level of miR-122 in Huh7 cells, measured by quantitative RT PCR. While significant reductions of 72% and 45% in miR-122 levels were observed for <b>100a</b> and <b>102a</b>, respectively, at 10 Î¼M, the isoquinoline-1,3(2<i>H</i>)-dione <b>104a</b> caused an approximately 450% increase in miR-122 expression at 10 Î¼M. The antiviral activity of <b>100a</b> and <b>102a</b> was then investigated in a hepatitis C virus/Huh7 cell model, and these compounds caused 48% and 47% reduction in HCV RNA levels, respectively. Since upregulation of miR-122, leading to an increase of caspase-3 activity, induces apoptosis in HCC, <b>100a</b> and <b>102a</b> are not expected to be effective against HCC. In contrast, HepG2 cells treated with <b>104a</b> showed a 20-fold increase in caspase-3 activity. These synthetic miR-122 activators and inhibitors are considered to be candidate chemotherapeutics against HCV infection and liver cancer.</div><figure id="sch16" data-index="17" class="article__inlineFigure"><h2 class="fig-label">Scheme 16</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/medium/jm1c00154_0017.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0017.jpeg" id="gr17" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 16. Synthesis, miR-122 Maturation-Inhibiting or -Activating Activity, and Antiviral or Caspase-3-Increasing Activity of Benzamides, Tetrahydroquinolines, and Isoquinoline-1,3(2<i>H</i>)-diones</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0017.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00154&amp;id=sch16"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26">1.7.  Synthetic Inhibitors of miR-182 Expression</h4><div class="NLM_p">BCL-2 interacting protein 3 (BNIP3) is a bifunctional mitochondrial protein that regulates cell death or survival,<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(59)</a> and miR-182 is an oncogenic miRNA associated with upregulation of BNIP3 in various types of cancer. A series of kenpaullone derivatives <b>107</b> was prepared by acid-catalyzed Fischer indole reaction of 3,4-dihydro-1<i>H</i>-benzo[<i>b</i>]azepine-2,5-dione <b>105</b> with various phenylhydrazines <b>106</b>, as potential BNIP3-targeting miR-182 inhibitors (<a class="ref internalNav" href="#sch17" aria-label="Scheme 17">Scheme 17</a>).<a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a> The effects of these heterocyclic compounds on the expression of miR-182 and BNIP3 in hypoxic cardiomyocytes were investigated in <i>in vitro</i> assays. The nitro-substituted kenpaullone derivative <b>107a</b> showed the greatest activity, increasing miR-182 expression 1.6-fold, down-regulating BNIP3, and consequently blocking mitochondrial fission and apoptosis in cardiac cells at 10 Î¼M concentration. In an <i>in vivo</i> ischemia/reperfusion (I/R) heart injury model in mice, compound <b>107a</b> (50 Î¼M per animal delivered <i><i>via</i></i> intravenous injection) improved cardiac functionvia significant down-regulation of BNIP3. Furthermore, the expression of connexin 43, a key protein in intercellular electrical and metabolic coupling, was significantly increased by treatment with <b>107a</b>. Cardiac function, including end-systolic volume and ejection fraction, was also significantly improved by <b>107a</b> compared to the control I/R group.</div><figure id="sch17" data-index="18" class="article__inlineFigure"><h2 class="fig-label">Scheme 17</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/medium/jm1c00154_0018.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0018.jpeg" id="gr18" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 17. Synthesis of Kenpaullone Derivatives as Regulators of miR-182 Expression</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0018.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00154&amp;id=sch17"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28">1.8.  Synthetic Inhibitors of miR-18a Maturation by Binding to the miR-17-92 Cluster</h4><div class="NLM_p">The miR-17-92 cluster is a polycistronic RNA producing seven mature miRNAs: miR-17-3p, miR-17-5p, miR-18a, miR-19a, miR-19b, miR-20a, and miR-92a.<a onclick="showRef(event, 'ref61'); return false;" href="javascript:void(0);" class="ref ref61">(61)</a> Dysregulation of the cluster expression contributes to the invasion and progression of many human cancers <i><i>via</i></i> multiple pathways, and suppressing these pathways using small molecules is an exciting potential strategy for cancer treatment.<a onclick="showRef(event, 'ref61'); return false;" href="javascript:void(0);" class="ref ref61">(61)</a> Disney <i><i>et al</i></i>. synthesized several benzimidazole derivatives <b>114</b><i><i>via</i></i> Suzuki coupling, hydrogenation, benzimidazole formation, hydrolysis, and amidation reactions.<a onclick="showRef(event, 'ref62'); return false;" href="javascript:void(0);" class="ref ref62">(62)</a> Compounds <b>114</b> were screened using the high-throughput structureâactivity relationships through sequencing (HiT-StARTS) system to discover molecules targeting the oncogenic miR-17-92 cluster in prostate cancer cells (<a class="ref internalNav" href="#sch18" aria-label="Scheme 18">Scheme 18</a>). HiT-StARTS is a selection-based technique similar to Inforna/2DCS that explores a compound library by examining the enrichment of selected RNA motifs. Binding affinity to the miR-17-92 cluster was also investigated using a fluorescence labeling technique. Compound <b>114a</b> was the most selective ligand for Dicer processing sites of miR-18a among the miR-17-92 cluster, and showed high binding affinity (<i>K</i><sub>d</sub> = 32 Î¼M). Furthermore, <i>in vitro</i> Dicer processing experiments indicated that <b>114a</b> significantly inhibited production of mature miR-18a with an IC<sub>50</sub> value of â¼10 Î¼M, leading to activation of serine/threonine protein kinase 4 protein and apoptosis of prostate cancer cells.</div><figure id="sch18" data-index="19" class="article__inlineFigure"><h2 class="fig-label">Scheme 18</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/medium/jm1c00154_0019.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0019.jpeg" id="gr19" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 18. Synthesis of Benzimidazole Derivatives as Inhibitors of Mature miR-18a Expression in Prostate Cancer Cells</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0019.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00154&amp;id=sch18"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30">1.9.  Synthetic Inhibitors of miR-96 Processing</h4><div class="NLM_p">Disneyâs group reported several benzimidazole-based compounds <b>122</b> as potential regulators of oncogenic miR-96. These were designed on the basis of the secondary structure of primary miR-96 (pri-miR-96) hairpin precursor with the aid of a database of RNA motif-small molecule interactions using the Inforna technique.<a onclick="showRef(event, 'ref63'); return false;" href="javascript:void(0);" class="ref ref63">(63)</a> The synthesis of <b>122</b> started with deprotection of Fmoc-protected amide resin <b>115</b> using 20% piperidine at room temperature followed by amide formation reaction to give <b>117</b>. Successive <i>N</i>-alkylation, amide formation, <i>N</i>-alkylation, amide formation, cleavage from the resin, and Cu-catalyzed triazole formation then afforded the desired peptoid compounds <b>122</b> (<a class="ref internalNav" href="#sch19" aria-label="Scheme 19">Scheme 19</a>). The binding affinity of <b>122</b> for miR-96 was measured to evaluate the advantage of the dimeric molecules <b>122</b> over monomers <b>112b</b> and <b>114a</b>. Compound <b>122a</b> recognized both the 1 Ã 1 nt GG and 1 Ã 1 nt UU loops of the RNAandexhibited the highest binding ability with <i>K</i><sub>d</sub> = 85 nM, approximately 40-fold and 30-fold higher than monomers <b>112b</b> and <b>114a</b>, respectively. Binding selectivity of compound <b>122a</b> to RNAs over DNAs was also assessed, and ligand <b>122a</b> did not bind to a DNA hairpin (<i>K</i><sub>d</sub> > 50â¯000 nM). <b>122a</b> showed the highest activity, selectively reducing miR-96 expression at 50 nM in MDA-MB-231 breast cancer cells and inducing apoptosis without affecting healthy breast cells. It also showed antitumor activity against breast cancer cells <i>in vivo</i>. Nod/SCID mice were injected with MDA-MB-231, and the developed tumors were intraperitoneally treated with <b>122a</b> (10 mg/kg) every other day for 21 days. Tumor growth was reduced concomitantly with a 50% decrease of pri-miR-96 levels and an increase of FOXO1. A pharmacokinetics analysis of <b>122a</b> was also carried out. At 48 h after <i>i.p.</i> injection at 2 mg/kg and 7 mg/kg, the plasma levels of <b>122a</b> were 1.6 and 1.9 Î¼M, respectively, considerably higher than the 50 nM concentration required to trigger apoptosis <i>in vitro</i>.</div><figure id="sch19" data-index="20" class="article__inlineFigure"><h2 class="fig-label">Scheme 19</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/medium/jm1c00154_0020.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0020.jpeg" id="gr20" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 19. Synthesis of Resin-Based Peptoid Analogues as Inhibitors of miR-96 Processing in Breast Cancer Cells</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0020.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00154&amp;id=sch19"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32">1.10.  Synthetic Degraders of miR-210</h4><div class="NLM_p">Exposure to low oxygen levels (hypoxia) induces multiple cellular changes to promote survival and maintain function.<a onclick="showRef(event, 'ref64'); return false;" href="javascript:void(0);" class="ref ref64">(64)</a> Hypoxia-inducible factors (HIFs) are a class of helix-loop-helix-PER-ARNT-SIM proteins and play a role in regulating genes associated with angiogenesis, cell proliferation, metabolism, and apoptosis. miR-210 is a hypoxia-related ncRNA that is upregulated by HIFs in human cells under hypoxia.<a onclick="showRef(event, 'ref64'); return false;" href="javascript:void(0);" class="ref ref64">(64)</a> However, overexpression of miR-210 can have pathological consequences, such as growth of hypoxic cancers and ischemic disorders. Disney and co-workers applied the 2DCS/Inforna approach to identify a class of bioactive ligands displaying selective binding affinity to miR-210.<a onclick="showRef(event, 'ref65'); return false;" href="javascript:void(0);" class="ref ref65">(65)</a> On the basis of the results, they synthesized compounds <b>127</b> in five steps, using the same procedure as shown in <a class="ref internalNav" href="#sch18" aria-label="Scheme 18">Scheme 18</a>, starting with 4-(4-methylpiperazin-1-yl)phenylboronic acid <b>59</b>, followed by hydrogenation, bezimidazole formation, hydrolysis, and amidation with RNase recruiter oligonucleotides <b>123</b> (<a class="ref internalNav" href="#sch20" aria-label="Scheme 20">Scheme 20</a>). Among the synthesized scaffolds <b>124</b>, the longest analogue <b>124a</b> (<i>n</i> = 3) displayed the most selective binding to pre-miR-210 with <i>K</i><sub>d</sub> = 340 nM in the presence of RNase L, while showing no measurable binding to DNA. The results indicate that the hit compound promotes recruitment of RNase L to pre-miR-210, thereby promoting RNA cleavage. To confirm this idea, the physical interaction of pre-miR-210 and RNase L in cells was examined by means of coimmunoprecipitation. It was found that the RNase L immunoprecipitated fractions displayed high pre-miR-210 enrichment levels only in the presence of the ligand <b>124a</b>. The inhibitory effect of <b>124a</b> on miR-210 was also investigated in hypoxic MDA-MB-231 cells (human breast adenocarcinoma), in which 200 nM <b>124a</b> induced a significant decrease (â¼50%) of both pre-miR-210 and mature miR-210 levels, resulting in apoptosis. The results of this study suggest that the 2DCS/Inforna technique can quickly define lead compounds for targeting RNAs, and the activity of these compounds can be enhanced by focusing on and developing molecules that recruit cellular nucleases. Synthetic compounds that target RNAs often bind at nonfunctional binding sites and thus have no effect on the cellular functions of the RNAs. In this regard, the ribonuclease recruitment approach enhances both the selectivity of small molecules to cleave various RNAs, particularly noncoding RNAs, and their cellular activity, regardless of the nonfunctional nature of the binding site.</div><figure id="sch20" data-index="21" class="article__inlineFigure"><h2 class="fig-label">Scheme 20</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/medium/jm1c00154_0021.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0021.jpeg" id="gr21" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 20. Synthesis of Oligoadenylate-Based Ligands as Inhibitors of miR-210 in Hypoxic Breast Cancer Cells</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0021.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00154&amp;id=sch20"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34">1.11.  Synthetic Inhibitors of miR-515 Maturation</h4><div class="NLM_p">Human epidermal growth factor receptor (HER) is a family of tyrosine kinase receptor proteins that facilitates cell growth and differentiation.<a onclick="showRef(event, 'ref66'); return false;" href="javascript:void(0);" class="ref ref66">(66)</a> Overexpression of HERs is tumorigenic, being associated with a wide range of human cancers. miR-515, a member of the ncRNAs, plays a central role in repressing sphingosine kinase 1 (SK1), which synthesizes sphingosine 1-phosphate (S1P), a regulator of HER levels in cancer cells.<a onclick="showRef(event, 'ref67'); return false;" href="javascript:void(0);" class="ref ref67">(67)</a> Disneyâs group used Inforna to design a series of dimer scaffolds <b>126</b> that would selectively target miR-515 hairpin precursors.<a onclick="showRef(event, 'ref68'); return false;" href="javascript:void(0);" class="ref ref68">(68)</a> The desired dimer products <b>126</b> were prepared according to a synthetic route (<a class="ref internalNav" href="#sch21" aria-label="Scheme 21">Scheme 21</a>) similar to that shown in <a class="ref internalNav" href="#sch18" aria-label="Schemes 18">Schemes 18</a> and <a class="ref internalNav" href="#sch19" aria-label="19">19</a>, starting with Suzuki coupling reaction between 4-(4-methylpiperazin-1-yl)phenylboronic acid <b>59</b> and 5-chloro-2-nitroaniline <b>108</b>, followed by hydrogenation and benzimidazole formation to obtain azide intermediate <b>114a</b>. Amidation of Rink amide resin <b>61</b> followed by several alkylations and amidations and cleavage from the resin afforded alkyne intermediates <b>125</b>, which then underwent Cu-catalyzed triazole formation reaction with the azide moiety <b>114a</b> to give <b>126</b>. The binding affinity of <b>126</b> to pri-miR-885, pri-miR-515, tRNA, and DNA was examined. None of the synthesized compounds bound to DNA or tRNA, presumably due to steric hindrance of the substituted benzyl group. Compound <b>126a</b> (<i>n</i> = 4) displayed the highest selective binding capability for the Drosha site in the pri-miR-515 hairpin, with an excellent <i>K</i><sub>d</sub> value of 0.06 Î¼M. The effect of the synthesized compounds on miR-515 levels and SK1 protein expression in MCF-7 breast cancer cells was examined. As expected, ligand <b>126a</b> was a potent inhibitor of miR-515 biogenesis and increased the expression of SK1 and the formation of S1P by â¼2.5-fold at 5 Î¼M concentration, resulting in increased HER2 abundance. These discoveries open up a powerful approach to make drug-insensitive cancers vulnerable to therapeutic agents based on their RNA expression profiles.</div><figure id="sch21" data-index="22" class="article__inlineFigure"><h2 class="fig-label">Scheme 21</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/medium/jm1c00154_0022.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0022.jpeg" id="gr22" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 21. Synthesis of Dimeric Ligands as Inhibitors of pri-miR-515 in MCF-7 Breast Cancer Cells</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0022.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00154&amp;id=sch21"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36">1.12.  Synthetic Enhancers of RNAi</h4><div class="NLM_p">RNA interference (RNAi) is one of the major post-transcriptional regulatory processes in human cells, in which miRNA molecules target specific mRNAs to inhibit their translation, and dysregulation causes many cellular disorders.<a onclick="showRef(event, 'ref69'); return false;" href="javascript:void(0);" class="ref ref69">(69)</a> Thus, identification of small molecules that can regulate RNAi in cancer cells has attracted great attention over the past few years.<a onclick="showRef(event, 'ref70'); return false;" href="javascript:void(0);" class="ref ref70">(70)</a> Jin and co-workers screened 2000 FDA-approved scaffolds and natural products using a cell-based assay and identified several enoxacin derivatives <b>130</b> as RNAi enhancers.<a onclick="showRef(event, 'ref71'); return false;" href="javascript:void(0);" class="ref ref71">(71)</a> They synthesized these compounds in high yields from commercially available enoxacin <b>127</b> by <i>N</i>-Boc protection and amide formation with various amines, followed by <i>N</i>-Boc deprotection (<a class="ref internalNav" href="#sch22" aria-label="Scheme 22">Scheme 22</a>). This chemical biology approach was established to regulate the RNAi pathway by designing small molecules that can enhance RNAi and promote miRNA biogenesis <i><i>via</i></i> assisting the interaction of trans-activation-responsive region RNA-binding protein (TRBP) with RNAs. Thus, the effect of enoxacin on the processing mediated by Dicer and TRBP was investigated using an <i>in vitro</i> assay. While enoxacin had no effect on the processing of pre-<i>let-7</i> or pre-miR-30a by Dicer alone, enoxacin (30 Î¼M) improved the processing of <i>let-7</i> or pre-miR-30a by Dicer and TRBP together. This observation suggests that enoxacin may target TRBP. In general, TRBP binds to miRNA precursors and accelerates the processing and loading of miRNAs and siRNAs onto RISCs. An RNA-binding assay was also performed to assess the effect of enoxacin on the interaction between TRBP and miRNA precursor. It was found that enoxacin improved the binding affinity of TRBP to miRNA precursors; the <i>K</i><sub>d</sub> was changed from 221 nM under normal conditions to 94 nM in the presence of enoxacin. These analyses strongly indicate that enoxacin facilitates the processing and loading of miRNAs/siRNAs onto RISCs by facilitating the interaction between TRBP and RNAs in human cells. Furthermore, small interfering RNAs (siRNAs) produced from shRNAs suppress the mRNA transcripts of enhanced green fluorescent protein (EGFP) expressed in human embryonic kidney HEK293 cells infected with a lentivirus. Enhancement of RNAi with small molecules increases the formation of siRNA, resulting in a reduction of EGFP-mediated fluorescence. Enoxacin <b>127</b> decreased the fluorescence dose-dependently with EC<sub>50</sub> â 30 Î¼M, but <b>130</b> was ineffective. The carboxylic acid group of enoxacin <b>127</b> was concluded to play a key role in the regulatory activity. Furthermore, the presence of bulky groups at the C-3 carbonyl of enoxacin (NH-<i>t</i>-Bu (<b>130a</b>) and NHBn (<b>130b</b>)) significantly decreased the RNAi-enhancing activity. In HEK293 cells, 50 Î¼M enoxacin <b>127</b> produced approximately 150% increase and 100% decrease in the mature miR-125a and pri-miR-125a levels, respectively, as determined by quantitative RT-PCR analysis. The results suggest that enoxacin <b>127</b> regulates the level of Dicer-mediated processing, rather than Drosha, transforming pri-miR-125a to pre-miR-125a.</div><figure id="sch22" data-index="23" class="article__inlineFigure"><h2 class="fig-label">Scheme 22</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/medium/jm1c00154_0023.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0023.jpeg" id="gr23" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 22. Synthesis of Enoxacin Derivatives as RNAi Enhancers</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0023.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00154&amp;id=sch22"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_2"><div id="ac_i38" class="anchor-spacer"></div><h3 class="article-section__title" id="_i38">2.  Application of Synthetic Small Molecule lncRNA Modulators in Drug Discovery</h3><div class="NLM_p">lncRNAs are a large family of RNAs, longer than 200 nucleotides in length, present in both nucleus and cytoplasm,<a onclick="showRef(event, 'ref72'); return false;" href="javascript:void(0);" class="ref ref72">(72)</a> and are considered to be involved in gene expression, mRNA translation, and RNA splicing. Dysregulation of lncRNAs is associated with cancer cell proliferation and metastasis, making lncRNAs attractive targets for cancer treatment. However, finding small molecules that interact with lncRNA is challenging compared to the case of the well-studied miRNAs, due in part to the low levels of lncRNA expression in cells.<a onclick="showRef(event, 'ref73'); return false;" href="javascript:void(0);" class="ref ref73">(73)</a></div><div class="NLM_p">Metastasis-associated lung adenocarcinoma transcript 1 (MALAT1) is a lncRNA that functions in the nucleus as a molecular skeleton for ribonucleoprotein complexes.<a onclick="showRef(event, 'ref74 ref75'); return false;" href="javascript:void(0);" class="ref ref74 ref75">(74,75)</a> MALAT1 has roles in tumor cell proliferation, migration, invasion, and apoptosis and is considered a potential therapeutic target. Hargrove and co-workers reported the synthesis of furamidine derivatives <b>134</b> as RNA-binding scaffolds for MALAT1 regulation (<a class="ref internalNav" href="#sch23" aria-label="Scheme 23">Scheme 23</a>).<a onclick="showRef(event, 'ref76'); return false;" href="javascript:void(0);" class="ref ref76">(76)</a> This study was the first example of selectively targeting the MALAT1 triple helix with synthetic small molecules. Considering the well-established binding ability of diphenylfuran-based ligands to a wide range of nucleic acid structures,<a onclick="showRef(event, 'ref77'); return false;" href="javascript:void(0);" class="ref ref77">(77)</a> it was anticipated that <b>134</b> would stabilize nontriple helical conformation or bind to the stem loop structure. The diphenylfuran-based compounds <b>134</b> were prepared from commercially available substrates <i><i>via</i></i> Pd-catalyzed CâH coupling of bromocyanobenzenes <b>131</b> with furan <b>132</b>, followed by amidine formation reaction. The synthesized heterocyclic compounds were screened for binding to the MALAT1 triple helix and stem loop RNAs by means of fluorescence assays. The selectivity of the furamidines was highly dependent on molecular shape; <i>ortho</i>- and <i>meta</i>-substituted structures were predicted to show decreased intramolecular interactions, which might reduce their selectivity and binding to the MALAT1 triple helix. The <i>para</i>-benzylpiperidine furamidine analogue <b>134a</b> was found to have the highest selectivity for the triple helix over the stem loop constructs, with the triple helix EC<sub>50</sub> value of 35.8 nM. This study offers efficient chemical assay methodology for evaluating binding to the MALAT1 triple helix, but further biological evaluations are required to identify selective RNA modulators.</div><figure id="sch23" data-index="24" class="article__inlineFigure"><h2 class="fig-label">Scheme 23</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/medium/jm1c00154_0024.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0024.jpeg" id="gr24" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 23. Synthesis and MALAT1-Binding Activity of Diphenylfuran-Based Derivatives</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0024.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00154&amp;id=sch23"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In another study, an <i>in silico</i> high-throughput screening strategy was utilized to identify two imidazole-based synthetic small molecules as selective ligands for the MALAT1 element of nuclear expression (ENE) triplex (<a class="ref internalNav" href="#sch24" aria-label="Scheme 24">Scheme 24</a>).<a onclick="showRef(event, 'ref78'); return false;" href="javascript:void(0);" class="ref ref78">(78)</a> A minimal 5â²-fluorescence-labeled MALAT1 RNA structure was employed to screen a library of approximately 26â¯000 molecules. To choose the most promising scaffold, a data pipeline consisting of commercial availability, selectivity, and cell-based analysis of the top 28 structures was developed. The Z-score was calculated for each molecule on the basis of the increased fluorescence in the presence of MALAT1 ENE triplex, and those compounds with a <i>Z</i>-score of more than 3 were further studied. Selected compounds were then investigated in an organoid mammary cancer model derived from MMTV-PyMT luminal B tumors in mice at a concentration of 1 Î¼M. Compounds <b>135</b> and <b>136</b> exhibited the largest reductions in MALAT1 RNA levels, amounting to 54% and 41%, respectively. Furthermore, HEK293T cells treated with these hit compounds showed asimilar morphology to untreated cells, which suggests a lack of toxicity. A preliminary SAR study showed that both the benzimidazole unit and the OMe group on the benzene ring of <b>135</b> are important for potency. FRET and isothermal titration calorimetry experiments confirmed that <b>135</b> and <b>136</b> bind directly to the ENE triplex with <i>K</i><sub>d</sub> values in the micromolar range. Molecular docking analysis predicted that compound <b>135</b> binds within the major groove of the triplex <i><i>via</i></i> van der Waals interactions with the two U-rich strands (U37â¯0U38â¯U39 and U10â¯U11â¯C12) over the 3â² A-rich tail (<a class="ref internalNav" href="#sch24" aria-label="Scheme 24">Scheme 24</a>b). In contrast, <b>136</b> displayed a different binding mode, mostly involving hydrogen bonds with A67 and U47, and the docking sites are spread across the narrow minor groove surface of the MALAT1 ENE triplex (<a class="ref internalNav" href="#sch24" aria-label="Scheme 24">Scheme 24</a>b). This study provides a useful chemical strategy for developing potential anticancer agents based on the function of MALAT1.</div><figure id="sch24" data-index="25" class="article__inlineFigure"><h2 class="fig-label">Scheme 24</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/medium/jm1c00154_0025.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0025.jpeg" id="gr25" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 24. (a) Identification of Two Imidazole-Based Synthetic Small Molecules <b>135</b> and <b>136</b> as Selective MALAT1 ENE Triplex-Binding Ligands; (b) Binding Modes of <b>135</b> and <b>136</b> with MALAT1 ENE Triplex</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0025.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00154&amp;id=sch24"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Alkylglycerone phosphate synthase (AGPS) catalyzes the synthesis of ether lipids, which are related to cancer development in humans.<a onclick="showRef(event, 'ref79'); return false;" href="javascript:void(0);" class="ref ref79">(79)</a> Upregulation of AGPS increases the invasive potential of many types of tumor cells, leading to increased cancer cell growth and metastasis. Qian and Zhu reported the computer-aided design and synthesis of a nitrogen-based heterocyclic compound <b>141</b> as an AGPS inhibitor, which may modulate the expression of AGPS-regulated lncRNAs.<a onclick="showRef(event, 'ref80'); return false;" href="javascript:void(0);" class="ref ref80">(80)</a> Their virtual screening approach was based on a combined 2D/3D examination of the interaction of the nitrogenous heterocyclic compounds with amino acid residues of AGPS. The hit imidazotriazine compound <b>141</b> was synthesized by reaction of an imidazolamide <b>137</b> with aminoisocyanate <b>139</b>, followed by amide formation reaction, as shown in <a class="ref internalNav" href="#sch25" aria-label="Scheme 25">Scheme 25</a>. Notably, compound <b>141</b> showed high antiproliferative activity in U251 glioma and MCF-7 breast cancer cells at 50 Î¼M, simultaneously decreasing the expression of tumor-related oncogenes and increasing the expression of apoptosis-associated genes, including lncRNAs, GAS5, CASC2, ANCR, and MEG3, which may be regulated by AGPS. Western blotting analysis showed that <b>141</b> significantly inhibited AGPS expression in U251 glioma and MCF-7 cells. A significant reduction in the U251 and MCF-7 cell populations in the G1 phase and an increase in the number of cells in the G2/M phase were observed after 72 h in the presence of 50 Î¼M <b>141</b>.</div><figure id="sch25" data-index="26" class="article__inlineFigure"><h2 class="fig-label">Scheme 25</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/medium/jm1c00154_0026.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0026.jpeg" id="gr26" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 25. Synthesis and Antitumor Effect of a Nitrogen-Based Heterocycle, <b>141</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0026.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00154&amp;id=sch25"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">HOX transcript antisense intergenic RNA (HOTAIR) is one of the widely studied long noncoding RNAs that plays a fundamental role in chromatin reprogramming, as well as transcriptional and post-transcriptional modulations.<a onclick="showRef(event, 'ref81'); return false;" href="javascript:void(0);" class="ref ref81">(81)</a> It is well established that HOTAIR is overexpressed in various types of human cancers, such as gastric cancer, esophageal cancer, breast cancer, and lung cancer,<a onclick="showRef(event, 'ref82'); return false;" href="javascript:void(0);" class="ref ref82">(82)</a> and high expression levels of HOTAIR are associated with a considerably worse prognosis. Mechanistically, the 5â² domain of HOTAIR appears to be mainly responsible for tumor development and progression in cancer cells through recruiting and binding to chromatin-modifying protein enhancer of zeste homologue 2 (EZH2) to silence a target gene, nemo-like kinase (NLK).<a onclick="showRef(event, 'ref83'); return false;" href="javascript:void(0);" class="ref ref83">(83)</a> Thus, there is great interest in the development of small molecules to selectively block HOTAIR activity. Kang and co-workers established a structure-based strategy, using an <i>in silico</i> screening of the PubChem library to discover a lead molecule that selectively inhibits the HOTAIR/EZH2 interaction.<a onclick="showRef(event, 'ref84'); return false;" href="javascript:void(0);" class="ref ref84">(84)</a> 3D RNA modeling of the 212â300 nt region in the 5â² domain of HOTAIR attached to EZH2 was performed using the MC-Fold and MC-Sym programs. Among 2000 virtually screened compounds, ADQ <b>142</b> (<a class="ref internalNav" href="#sch26" aria-label="Scheme 26">Scheme 26</a>) displayed the greatest binding affinity for HOTAIR. This compound also increased NLK luciferase activity by 2â2.5 fold in four cancer cell lines: MDA-MB-231 breast cancer cells, U87, U87-EGFRvIII, and LN229 glioma cells. RNA immunoprecipitation experiments using <b>142</b> in U87 and MDA-MB-231 cells showed significant reductions of HOTAIR-binding EZH2 levels, as determined by agarose gel electrophoresis and qPCR. These observations indicate that <b>142</b> effectively inhibits the formation of the HOTAIR/EZH2 complex. More importantly, the binding affinity of <b>142</b> to other lncRNAs, including MALAT1, KCNQ1OT1, HOTAIRM1, XIST, and HOXA11, to polycomb repressive complex 2 containing EZH2 was also investigated. In these cases, <b>142</b> caused no noticeable reduction of the lncRNA signals, indicating that it has high selectivity for the HOTAIR/EZH2 complex. Molecular docking and conformational analysis indicated that the 36G46A microdomain of HOTAIR forms a specific binding pocket in which the nitrobenzene units of <b>142</b> form ÏâÏ stacking with the 36G and 46A sequences. The SAR study revealed that the nitro substitution on the quinoxaline unit was essential for the inhibitory activity of <b>142</b>, as introducing dimethyl and chloro groups into the benzene ring led to loss of inhibitory activity. The reason for this may be that the nitro substitution improves ÏâÏ stacking between ADQ and HOTAIR due to its greater conjugation effect compared with the other substituents. Antitumor activity of <b>142</b> was investigated <i>in vivo</i> using a mouse MDA-MB-231 orthotopic tumor transplantation model. Injection of MDA-MB-231 cells into the mice led to the formation of large metastatic nodules in the lungs. Treatment of the inoculated mice with 15 mg/kg ADQ <b>142</b> every 2 days for 21 days significantly decreased the number and volume of metastatic nodules, suggesting that <b>142</b> effectively suppressed HOTAIR-mediated tumor metastasis. Furthermore, a considerable increase of NLK-positive cells in the treated samples was detected by immunohistochemical staining, supporting the idea that ADQ <b>142</b> inhibits the HOTAIR/EZH2 interaction, thereby increasing the expression of NLK.</div><figure id="sch26" data-index="27" class="article__inlineFigure"><h2 class="fig-label">Scheme 26</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/medium/jm1c00154_0027.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0027.jpeg" id="gr27" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 26. Identification of ASQ <b>142</b> as a Selective Inhibitor of HOTAIR/EZH2 Interaction in Cancer Cells</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0027.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00154&amp;id=sch26"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Application of synthetic lncRNA inhibitors in cancer biology is still at a very early stage, but they have potentials for treating a wide range of human diseases including cancers.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_2"><div id="ac_i43" class="anchor-spacer"></div><h3 class="article-section__title" id="_i43">3.  Application of Synthetic Small Molecule mRNA Modulators in Drug Discovery</h3><div class="NLM_p">Small-molecule-based RNA ligands that inhibit a specific step in the gene expression pathway are versatile tools in cell biology, as well as candidate anticancer agents.<a onclick="showRef(event, 'ref85'); return false;" href="javascript:void(0);" class="ref ref85">(85)</a> For example, inhibition of splicing of mRNA precursors (pre-mRNAs), a key step in eukaryotic gene expression in which introns are removed from nascent transcripts, is a potentially interesting approach to repress tumorigenesis in human cells. In this section, we will discuss several approaches to mRNA inhibition.</div><div id="sec4_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i44" class="anchor-spacer"></div><h4 class="article-section__title" id="_i44">3.1.  Synthetic Inhibitors of mRNA Translation Processing</h4><div class="NLM_p">RNA G-quadruplexes are noncanonical secondary structures of RNA generated from cyclic planar ordering of four connected guanine units (G-quartets).<a onclick="showRef(event, 'ref86'); return false;" href="javascript:void(0);" class="ref ref86">(86)</a> These structures participate in critical cellular events such as translational regulation, transcription ending, pre-mRNA splicing, and mRNA processing. Thus, synthetic small molecules targeting RNA G-quadruplex are candidate anticancer agents. Uesugi and co-workers identified <b>149</b> as an RNA G-quadruplex stabilizer that inhibits mRNA translation, using an <i>in vitro</i> screening strategy based on quantitative real-time PCR evaluation of complementary DNA (<i>c</i>DNA) production.<a onclick="showRef(event, 'ref87'); return false;" href="javascript:void(0);" class="ref ref87">(87)</a> It is well-known that RNA G-quadruplex structures are thermodynamically stable and suppress the <i>c</i>DNA production by reverse transcriptase, so RNA G-quadruplex stabilizers are expected to decrease <i>c</i>DNA production in their assay system. The hit compound <b>149</b> was efficiently synthesized from commercially available 2-acetylfuran <b>143</b><i><i>via</i></i> Knoevenagel reaction, thiophene ring formation, pyrimidone formation, followed by intermolecular <i>N</i>-alkylation-based cyclization of two pyrimidone moieties (<a class="ref internalNav" href="#sch27" aria-label="Scheme 27">Scheme 27</a>). Telomeric repeat-containing RNA (TERRA) was chosen as a well-characterized RNA G-quadruplex substrate to start the biological analyses. The binding ability was first assessed by size exclusion chromatography, in which <b>149</b> showed selective binding to TERRA with the <i>K</i><sub>d</sub> value of 5.9 Î¼M and did not bind to other RNA or DNA structures. To test the hypothesis that stabilization of the RNA G-quadruplex by <b>149</b> would suppress mRNA translation in human breast tumor cells, followed by a decrease of NRAS proto-oncogene expression, the effect of <b>149</b> on the NRAS expression in human breast cancer MCF-7 cells was investigated. Upon treatment of the tumor cells with <b>149</b> at the concentration of 20 Î¼M, the NRAS expression level was significantly decreased, as determined by Western blot analysis. This result confirms that <b>149</b> stabilizes RNA G-quadruplex structure in NRAS mRNA in cancer cells.</div><figure id="sch27" data-index="28" class="article__inlineFigure"><h2 class="fig-label">Scheme 27</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/medium/jm1c00154_0028.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0028.jpeg" id="gr28" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 27. Synthesis and Activity of a Polyaromatic RNA G-Quadruplex Binder, <b>149</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0028.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00154&amp;id=sch27"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Programmed â1 ribosomal frame-shifting (â1PRF) is a recoding process that produces two different proteins from a single transcript by redirecting mRNA translation and is used by viruses such as coronaviruses to guarantee correct stoichiometry of protein complexes required for their propagation.<a onclick="showRef(event, 'ref88'); return false;" href="javascript:void(0);" class="ref ref88">(88)</a> In eukaryotic cells, â1PRF is controlled by an mRNA pseudoknot, in which the ribosome restarts protein decoding by uncoiling the RNA pseudoknot structure to begin the â1 frame translation. A pseudoknot is one of the most common stable three-dimensional folded structures of RNAs and is generally formed by stem-loop base pairing of a single-stranded RNA.<a onclick="showRef(event, 'ref89'); return false;" href="javascript:void(0);" class="ref ref89">(89)</a> Nakataniâs group reported a class of naphthyridine-based structures <b>158</b> as mRNA modifiers (<a class="ref internalNav" href="#sch28" aria-label="Scheme 28">Scheme 28</a>).<a onclick="showRef(event, 'ref90'); return false;" href="javascript:void(0);" class="ref ref90">(90)</a> Compounds <b>158</b> consist of a <i>N</i>-alkoxycarbonyl-1,8-naphthyridine scaffold bearing hydrogen-bonding sites that can bind to guanine units in a CGG/CGG sequence of mRNA (<a class="ref internalNav" href="#sch28" aria-label="Scheme 28">Scheme 28</a>). These dimers (<b>155</b>) also displayed some binding ability to the corresponding sequence in DNA. These naphthyridine carbamate tetramers (<b>158</b>) were synthesized from 3,3â²-iminodipropionitrile <b>150</b> through ester formation, <i>N</i>-Boc protection, hydrolysis, amidation, and reductive aminoalkylation reactions. Initially, three pseudoknot sequences were prepared by mutation of a mouse mammary tumor virus (MMTV) pseudoknot (VPK); they were designated as F-VPK (mutating G2 to C2 and C18 to G18), M1-VPK (mutating C3 to G3), and M2-VPK (mutating C5 to G5 and C19 to G19). The ability of the synthesized heterocycles <b>158</b> to induce the â1PRF process was analyzed by means of <i>in vitro</i> assays measuring the protein products generated from the mRNAs encoding <i>Renilla</i> luciferase and truncated firefly luciferase. Western blot results indicated that ligand <b>158a</b> (<i>n</i> = 8) was the most effective regulator to increase the frame-shifting efficiency (FE) of â1PRF, and an increase of FE from 13% to 20% was observed for the M1-VPK sequence in the presence of 10 Î¼M <b>158a</b>. In addition, FE of the M2-VPK sequence was significantly improved from 15% to 27% in the presence of 10 Î¼M <b>158a</b>. On the other hand, the amount of protein products obtained from mRNAs containing the F-VPK was reduced from 29% to 21%, reflecting the inhibitory effect of <b>158a</b> on translation. The effect of compound <b>158a</b> on frame-shifting in human cells was next investigated. Upon treatment of HeLa cells with different concentrations of <b>158a</b>, FE for the M1- and M2-VPK sequences increased from 0.35% to 3.19% (9.1-fold increase) and from 0.32% to 1.70% (5.3-fold increase), respectively, in the presence of 0.3 Î¼M <b>158a</b>. In contrast, since the F-VPK sequence has a more stable pseudoknot structure than the others, the FE was only slightly increased (1.5-fold) at a higher concentration of <b>158a</b> (>0.1 Î¼M). This work offers a unique approach to regulate ribosome reading frame selection using synthetically accessible ligands and might find applications in drug discovery.</div><figure id="sch28" data-index="29" class="article__inlineFigure"><h2 class="fig-label">Scheme 28</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/medium/jm1c00154_0029.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0029.jpeg" id="gr29" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 28. Synthesis of Naphthyridine-Based Structures and Their Effect on â1PRF Frame-Shifting Efficiency (FE)</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0029.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00154&amp;id=sch28"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Overexpression of the presynaptic protein Î±-synuclein (SNCA) is associated with severe neurodegenerative diseases, such as Parkinsonâs disease, dementia, and Alzheimerâs disease.<a onclick="showRef(event, 'ref91'); return false;" href="javascript:void(0);" class="ref ref91">(91)</a> Î±-Synuclein is difficult to target due to the absence of drug-binding pockets, so decreasing its expression by targeting the mRNA is a possible approach for therapeutic purposes. SNCA mRNA has an iron-responsive element (IRE) in the 5â² UTR, which is involved in the translation process. Disney and co-workers reported sequence-based design of several synucleozid derivatives (<a class="ref internalNav" href="#sch29" aria-label="Scheme 29">Scheme 29</a>) that selectively target the IRE structure of SNCA mRNA and consequently inhibit Î±-synuclein expression.<a onclick="showRef(event, 'ref92'); return false;" href="javascript:void(0);" class="ref ref92">(92)</a> Synucleozid analogues <b>165</b>, <b>169</b>, and <b>172</b> were synthesized <i><i>via</i></i> the reaction of either 4-hydroxybenzaldehyde <b>159</b> or 4-aminobenzaldehyde <b>169</b> with 4-fluorobenzonitrile <b>160</b> to form adducts <b>161</b> and <b>170</b>, respectively, followed by condensation reaction with 3,4-diaminobenzonitrile <b>166</b> to give <b>167</b> and <b>171</b> (<a class="ref internalNav" href="#sch29" aria-label="Scheme 29">Scheme 29</a>). Compound <b>161</b> was treated with 4-methyl-3-nitrobenzonitrile <b>162</b> to produce <b>163</b>, which underwent CadoganâSundberg reaction to generate dinitrile <b>164</b>. Finally, P<sub>2</sub>S<sub>5</sub>-mediated cyclization reaction of <b>164</b>, <b>167</b>, and <b>171</b> afforded the desired synucleozid derivatives <b>165</b>, <b>168</b>, and <b>172</b>. The main reason for selecting synucleozid structures was to improve bloodâbrain barrier (BBB) penetrance, based on CNS multiparameter optimization (MPO) scores. 2-AP fluorescence binding assay revealed that the replacement of the guanidyl groups of synucleozid <b>173</b> with imidazolyl groups to generate <b>165</b>, <b>168</b>, and <b>172</b> improved the CNS MPO scores of the compounds without reducing their avidity for the SNCA IRE structure. However, the ability of the synucleozid analogues to block Î±-synuclein expression in SH-SY5Y cells was reduced; approximately 67% and 40% reductions of Î±-synuclein levels were observed using synucleozid (<b>173</b>) at 1 Î¼M and its derivatives (<b>165</b>, <b>168</b>, <b>171</b>) at 5 Î¼M, respectively. Mechanistic investigations showed that synucleozid <b>173</b> affects ribosomal assembly onto the SNCA mRNA by stabilizing the IRE structure and inhibiting its unfolding process. A similar approach might be applicable to other disease-causing undruggable proteins encoded by mRNAs with IREs.</div><figure id="sch29" data-index="30" class="article__inlineFigure"><h2 class="fig-label">Scheme 29</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/medium/jm1c00154_0030.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0030.jpeg" id="gr30" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 29. Synthesis of Synucleozid-Based Small Molecules and Their Inhibitory Activity toward Î±-Synuclein Expression in SH-SY5Y Cells</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0030.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00154&amp;id=sch29"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec4_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i48" class="anchor-spacer"></div><h4 class="article-section__title" id="_i48">3.2.  Synthetic Molecules Targeting mRNA Splicing Process</h4><div class="NLM_p">Tau is a microtubule-associated protein produced by mRNAs in neurons.<a onclick="showRef(event, 'ref93'); return false;" href="javascript:void(0);" class="ref ref93">(93)</a> Aberrant aggregation of tau protein in neuronal cells and axons is a key feature of the pathology of neurodegenerative diseases such as dementia, Alzheimerâs and Parkinsonâs disease.<a onclick="showRef(event, 'ref94'); return false;" href="javascript:void(0);" class="ref ref94">(94)</a> However, tau is not easily druggable, and alternative approaches are needed.<a onclick="showRef(event, 'ref95'); return false;" href="javascript:void(0);" class="ref ref95">(95)</a> In this context, Artigas and MarchÃ¡n reported four classes of ametantrone-based small molecules, <i>i.e.</i>, ametantrone <b>176</b>, guanidinoneamine <b>186</b>, neamine <b>187</b>, and azaquinolone <b>189</b>, that target tau pre-mRNAs, and investigated their RNA-binding ability.<a onclick="showRef(event, 'ref96'); return false;" href="javascript:void(0);" class="ref ref96">(96)</a> Ametantrone itself (<b>176</b>, in which Boc is replaced by H) was identified as a promising RNA ligand with low cytotoxicity in eukaryotic cells.<a onclick="showRef(event, 'ref97 ref98'); return false;" href="javascript:void(0);" class="ref ref97 ref98">(97,98)</a> As shown in <a class="ref internalNav" href="#sch30a" aria-label="Scheme 30">Scheme 30</a>, nucleophilic aromatic substitution reaction of 1,4-difluoroanthraquinone <b>174</b> with <i>N</i>-(2-hydroxyethyl)ethylenediamine <b>175</b> gave <i>N</i>-Boc protected ametantrone <b>176</b>, followed by Mitsunobu reaction and hydrolysis to yield intermediates <b>179</b> and <b>180</b>. Treatment of thiol-containing neamine <b>181</b> with 2,2â²-dithiobis(5-nitropyridine) <b>182</b> and subsequent reaction with monothiol derivative <b>179</b> provided the corresponding disulfide intermediate <b>184</b> (<a class="ref internalNav" href="#sch28" aria-label="Scheme 28">Scheme 28</a>b). The desired compound <b>186</b> was finally obtained <i><i>via</i></i> the reaction of <b>184</b> with <i>N</i>,<i>N</i>â²-di-Boc-<i>N</i>â³-triflylguanidine <b>185</b>, followed by acidic deprotection of Boc. Similar approaches were employed for the synthesis of <b>187</b> and <b>189</b>, starting from the reaction of <b>183</b> and bis-thiol derivative <b>180</b> (<a class="ref internalNav" href="#sch30a" aria-label="Scheme 30">Scheme 30</a>). Azaquinolone and guanidinoneamine heterocycles are useful structures to increase RNA-binding affinity <i><i>via</i></i> hydrogen-bonding pairing with adenine in RNA stem-loop structures, as well as to improve cellular uptake. Guanidinylated analogue <b>186</b> displayed the highest binding affinity for tau mRNA, with the EC<sub>50</sub> value of 57.2 nM. The effect of the ligands on the thermodynamic stability of tau mRNA bearing mutated sequences seen in patients suffering from chromosome 17-related frontotemporal dementia and parkinsonism (FTDP-17) was also investigated. Intronic mutations generally reduce the stability of the tau pre-mRNA stem-loop structure, resulting in abnormal splicing. Stabilizing tau pre-mRNA with organic ligands might be helpful to regulate the splicing mechanism and might suppress the development of neurodegenerative disorders. Indeed, the synthesized heterocycles increased the thermodynamic stability of mutated sequences of tau pre-mRNA by approximately 4 Â°C and thus could be considered as regulators of the tau pre-mRNA splicing process. This study provides a new strategy to design mRNA ligands based on amentantrone with increased RNA-binding affinity. Research is needed to explore the availability of these types of heterocycles for targeting other RNA secondary structures therapeutically relevant to human diseases.</div><figure id="sch30a" data-index="31" class="article__inlineFigure"><h2 class="fig-label">Scheme 30</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/medium/jm1c00154_0031.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0031.jpeg" id="GRAPHIC-d7e2457-autogenerated" class="inline-fig internalNav" /></img><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/medium/jm1c00154_0032.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0032.jpeg" id="GRAPHIC-d7e2458-autogenerated" class="inline-fig internalNav" /></img><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/medium/jm1c00154_0033.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0033.jpeg" id="GRAPHIC-d7e2459-autogenerated" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 30. Synthesis of Ametantrone-Based Structures as Tau pre-mRNA Stabilizers</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0031.jpeg jm1c00154_0032.jpeg jm1c00154_0033.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00154&amp;id=sch30a"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Disneyâs group used a combination of 3D modeling and Inforna approaches to discover bioactive small molecules with high binding affinity for tau pre-mRNA, aiming to suppress 4 repeat (4R) tau levels in FTDP-17.<a onclick="showRef(event, 'ref99'); return false;" href="javascript:void(0);" class="ref ref99">(99)</a> They constructed an <i>in vitro</i> assay system in which the A-bulge of tau pre-mRNA was replaced with 2-AP in models of the WT and disinhibition-dementia-parkinsonism-amyotrophy complex hairpins. In this assay, binding of small molecules to the labeled sites reduces the fluorescence intensity. Screening of 200 compounds, including some commercial products, identified <b>193</b> as the most potent ligand with high binding affinity (<i>K</i><sub>d</sub> = 4.9 Î¼M) and an IC<sub>50</sub> value of 2.1 Î¼M. They synthesized <b>193</b><i><i>via</i></i> conversion of readily available ethyl 3-amino-5-methoxy-1<i>H</i>-indole-2-carboxylate <b>190</b> to imine derivative <b>191</b>, followed by pyrimidone formation (<a class="ref internalNav" href="#sch31" aria-label="Scheme 31">Scheme 31</a>). Cell-based luciferase assay revealed no toxicity of the ligand up to 100 Î¼M concentration. NMR spectroscopic analysis of the ligandâtau RNA complex supported the idea that compound <b>193</b> improves the stability of the tau pre-mRNA hairpin structure through hydrogen-bonding interactions with base pairs close to the A-bulge. There are generally six tau isoforms in human cells, resulting from alternative splicing of three exons (2, 3, 10) of the tau gene, of which exon 10 is responsible for the three and four carboxyl-terminal microtubule-binding repeats (3R and 4R) isoforms. While there is an equimolar 4R/3R ratio in normal human cells, an increased 4R/3R ratio is associated with tau aggregation in neurodegenerative diseases.<a onclick="showRef(event, 'ref100'); return false;" href="javascript:void(0);" class="ref ref100">(100)</a> The effect of compound <b>193</b> on tau in LAN5 neuroblastoma cells and humanized tau mouse primary neurons was also investigated, and â¼40% and â¼50% reductions of the 4R/3R ratios were observed for neuroblastoma cells and primary cortical neurons, respectively. Having a good CNS MPO score of 4.4, ligand <b>193</b> was predicted to display promising brain penetration and thus might have reasonable ADME characteristics.</div><figure id="sch31" data-index="32" class="article__inlineFigure"><h2 class="fig-label">Scheme 31</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/medium/jm1c00154_0034.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0034.jpeg" id="gr34" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 31. Synthesis of Pyrimidone-Based Compound <b>193</b> and Activity to Suppress 4R Tau Levels in FTDP-17</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0034.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00154&amp;id=sch31"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In general, a pre-mRNA hairpin structure close to exon 10 controls the splicing process in pre-mRNA producing tau protein. Disease-causing mutations that modify this pre-mRNA structure significantly enhance the splicing in exon 10, resulting in aberrant aggregation of tau protein in neuronal cells. Accordingly, stabilizing the stem loop of mRNA with small molecules may reduce the inclusion of exon 10, offering a new therapeutic strategy to treat various neurodegenerative diseases. On the basis of this concept, Wolfeâs group identified compound <b>204</b> (<a class="ref internalNav" href="#sch32" aria-label="Scheme 32">Scheme 32</a>) as an inhibitor of tau exon 10 pre-mRNA splicing by means of RNA-directed <i>in situ</i> HÃ¼isgen 1,3-cycloaddition reaction of mitoxantrones (MTXs) with an azide linker (<b>202</b> and its analogues, <a class="ref internalNav" href="#sch32" aria-label="Scheme 32">Scheme 32</a>) and bipartite ASOs bearing a terminal alkyne (<b>203</b> and its analogues, <a class="ref internalNav" href="#sch32" aria-label="Scheme 32">Scheme 32</a>).<a onclick="showRef(event, 'ref101'); return false;" href="javascript:void(0);" class="ref ref101">(101)</a> It was anticipated that the anthracene-dione core of MTX derivatives would assist their binding to pre-mRNA hairpin structure in a suitable orientation <i><i>via</i></i> strong interactions of the two amino alcohol substitutions and the aromatic rings with the RNA groove. Mitoxantrone <b>202a</b> itself was previously reported to display high potency in stabilizing the tau mRNA stem-loop <i><i>via</i></i> hydrogen bonding between the amine and hydroxyl groups of MTX and GC base pairs in the mRNA structure.<a onclick="showRef(event, 'ref102'); return false;" href="javascript:void(0);" class="ref ref102">(102)</a> Peptide-nucleic acid (PNA) scaffolds with short sequences were also utilized as the bipartite ASOs component due to their strong binding ability to RNAs.<a onclick="showRef(event, 'ref103'); return false;" href="javascript:void(0);" class="ref ref103">(103)</a> With these components in hand, RNA-directed <i>in situ</i> click reactions were performed using tau mRNA template, MTX azides, and bipartite PNA alkynes in water at room temperature. Interestingly, HÃ¼isgen 1,3-cycloaddition reaction between MTX azides and PNA alkynes only proceeded in the presence of the mRNA template, suggesting that concurrent binding of these components to the RNA brings the azide and alkyne groups into close proximity in appropriate orientation to facilitate the 1,3-cycloaddition reaction. They finally identified conjugate <b>204</b> as an inhibitor of tau exon 10 pre-mRNA splicing. This study was the first example of RNA template-directed synthesis of small molecule RNA inhibitors. The synthesis of <b>204</b> is illustrated in <a class="ref internalNav" href="#sch32" aria-label="Scheme 32">Scheme 32</a>. Two successive <i>N</i>-protection reactions of diethylenetriamine <b>194</b> followed by partial <i>N</i>-deprotection provided the intermediate <b>197</b>, which was treated with [2-(2-azidoethoxy)ethoxy]acetic acid <b>198</b> to give azide compound <b>199</b>. The nucleophilic aromatic substitution of commercially available anthracene-dione <b>200</b> with <b>199</b> and <i>N</i>-(2-aminoethyl)ethanolamine <b>175</b> and subsequent <i>N</i>-Boc deprotection afforded MTX azide <b>202</b>. HÃ¼isgen 1,3-cycloaddition reaction of MTX azide <b>202</b> and resin-supported bipartite PNA alkyne oligomer <b>203</b> in the presence of Cu(MeCN)<sub>4</sub>PF<sub>6</sub> catalyst gave the corresponding conjugate, which was finally cleaved from the resin under acidic conditions to afford <b>204</b>. While the well-known anticancer agent MTX displayed high cytotoxicity (â¼100 nM), the conjugate <b>204</b> was much less cytotoxic (â¼1 Î¼M). Unfortunately, the cytotoxicity of <b>204</b> is still too high for <i>in vivo</i> analysis of pre-mRNA splicing, so a cell-free <i>in vitro</i> splicing assay was employed. Compound <b>204</b> showed potent inhibitory activity (complete inhibition at 500 nM) of tau exon 10 pre-mRNA splicing compared with either PNA or MTX alone.</div><figure id="sch32" data-index="33" class="article__inlineFigure"><h2 class="fig-label">Scheme 32</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/medium/jm1c00154_0035.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0035.jpeg" id="gr35" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 32. Synthesis of a Mitoxantrone-Bipartite PNA Alkyne-Based Compound with Inhibitory Activity toward Tau Exon 10 pre-mRNA Splicing</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0035.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00154&amp;id=sch32"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Spinal muscular atrophy (SMA) is a neurodegenerative disorder in which motor neurons in the spinal cord and brain stem are progressively destroyed, resulting in muscle wasting and weakness.<a onclick="showRef(event, 'ref104'); return false;" href="javascript:void(0);" class="ref ref104">(104)</a> Survival motor neuron (SMN) protein, encoded by SMN1 and SMN2 genes, plays a fundamental role in the development and function of motor neurons in the CNS. Either homozygous deletion or mutation of the SMN1 gene and abnormal splicing of the SMN2 gene contribute to the onset of SMA. In SMA, SMN2 pre-mRNA undergoes an unusual splicing process to form SMN2Î7 mRNA, which generates an unstable SMNÎ7 protein. This phenomenon results in a reduced amount of SMN2 mRNA and consequently an insufficient level of SMN protein, leading to more severe SMA symptoms over time. In this regard, shifting the splicing process of SMN2 to the generation of SMN2 mRNAs by targeting the pre-mRNA with small molecules might be a promising strategy to increase the level of SMN protein. Risdiplam has recently been developed as a selective SMN2 gene splicing modulator for the treatment of SMA.<a onclick="showRef(event, 'ref105'); return false;" href="javascript:void(0);" class="ref ref105">(105)</a> Risdiplam <b>211</b> is an improved version of compound <b>213</b>, the first SMN2 pre-mRNA splicing modifier to enter human clinical trials. Although <b>213</b> increased the levels of SMN protein up to 2-fold in patients, retinal toxicity was observed in cynomolgus monkeys after 39 weeks of daily oral dosing. Risdiplam <b>211</b>, in which this adverse effect of <b>213</b> is reduced, has been approved for clinical use under the brand name Evrysdi. In the synthesis of <b>211</b>, 3,6-dichloro-4-methylpyridazine <b>205</b> initially undergoes a consecutive nucleophilic substitution/<i>N</i>-alkylation/intramolecular cyclization reaction in the presence of aqueous ammonia and 1-bromo-2,2-dimethoxypropane to give imidazopyridazine <b>206</b> (<a class="ref internalNav" href="#sch33" aria-label="Scheme 33">Scheme 33</a>). Treatment of <b>206</b> with bis(pinacolato)diboron affords the corresponding boronic acid pinacol ester <b>207</b>, which reacts with pyrimidin-4-one <b>209</b>, prepared from 2-amino-5-fluoropyridine <b>208</b> and then with a piperazine derivative to finally produce <b>211</b>. The SAR study suggested that the presence of the pyrazine nitrogens is critical for high potency. Risdiplam <b>211</b> was significantly more potent (EC<sub>1.5Ã</sub> = 4 nM) than ligand <b>213</b> (EC<sub>1.5Ã</sub> = 23 nM) in an <i>in vitro</i> SMN2 pre-mRNA splicing assay. Furthermore, the amount of SMN protein in SMA type I fibroblasts was increased by approximately 2-fold in the presence of <b>211</b> (1 Î¼M). In metabolism studies using human liver microsomes and human hepatocytes, <i>N</i>-hydroxylated analogue <b>212</b> was identified as the main metabolite but displayed no activity toward SMN2 splicing. The pharmacokinetic profile of risdiplam <b>211</b> was examined in rats and cynomolgus monkeys upon intravenous and oral administration. Risdiplam showed a significantly reduced volume of distribution (<i>V</i><sub>ss</sub>) of 3.1 L/kg (in rats) and 2.0 L/kg (in cynomolgus monkeys) when administered at 1.9 mg/kg intravenously, compared to compound <b>213</b>, which showed <i>V</i><sub>ss</sub> = 29 L/kg in rat and <i>V</i><sub>ss</sub> = 20 L/kg in cynomolgus monkey at the intravenous dose of 2.0 mg/kg (<a class="ref internalNav" href="#sch33" aria-label="Scheme 33">Scheme 33</a>). Risdiplam also displayed a favorable half-life range of <i>T</i><sub>1/2</sub> = 5.4â6.4 h upon oral administration of 5.5 mg/kg in these animal experiments. The effects of risdiplam <b>211</b> on the hERG channel and on phospholipidosis were also investigated in <i>in vivo</i> assays. No marked changes of hERG K<sup>+</sup> current and no phospholipidosis were detected in any tissues after administration for up to 26 and 39 weeks in rats and cynomolgus monkeys, respectively. No evidence of toxicity was observed in the <i>in vitro</i> and <i>in vivo</i> studies. Neuromuscular pathological examination of SMAÎ7 mice treated with <b>211</b> from PND3 to PND14 at doses in the range of 0.1â1 mg kg<sup>â1</sup> day<sup>â1</sup> revealed increases in the number of motor neurons, muscle size, and proportion of fully innervated neuromuscular junctions in SMNÎ7 mice given the dose of 0.3 mg/kg.</div><figure id="sch33" data-index="34" class="article__inlineFigure"><h2 class="fig-label">Scheme 33</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/medium/jm1c00154_0036.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0036.jpeg" id="gr36" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 33. Synthesis of Risdiplam, Its Activity toward SMN2 pre-mRNA Splicing in SMA Type 1 Fibroblasts, and a Comparison with the Initially Reported SMN2 pre-mRNA Splicing Modifier</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0036.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00154&amp;id=sch33"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Soon after the discovery of risdiplam <b>211</b>, Cheung and co-workers developed a synthetic small molecule as an SMN2 pre-mRNA splicing modifier to stabilize the interaction between SMN2 pre-mRNA and spliceosome.<a onclick="showRef(event, 'ref106'); return false;" href="javascript:void(0);" class="ref ref106">(106)</a> Branaplam <b>215</b> was discovered by high-throughput phenotypic screening, followed by optimization of the lead structure <b>214</b> (<a class="ref internalNav" href="#sch34" aria-label="Scheme 34">Scheme 34</a>). Compound <b>214</b> showed outstanding activation of the SMN2 reporter by 1700% with EC<sub>50</sub> = 3.5 Î¼M. In a mouse SMN ELISA analysis, <b>214</b> exhibited 2.5-fold increase in the level of SMN protein with an EC<sub>50</sub> value of 0.6 Î¼M. An <i>in vivo</i> pharmacokinetics of <b>214</b> in mice revealed a high plasma clearance of 113 mL min<sup>â1</sup> kg<sup>â1</sup> with moderate bioavailability (<i>F</i> = 18%) and poor CNS exposure. Accordingly, further efforts were focused on improving the potency, maximizing the <i>in vivo</i> exposure by decreasing the clearance and increasing the permeability, and reducing the hERG channel-binding liability. A comprehensive SAR study identified key structural parameters for improvement. The introduction of an <i>ortho</i>-hydroxyl group on the aromatic ring next to the pyridazine core resulted in a substantial increase in the potency of <b>214</b> in SMN ELISA assay. It was also found that addition of a pyrazole group to the structure retained high SMN potency with reduced hERG binding. Changing the <i>N</i>-methylamine linker in <b>214</b> to an oxygen moiety, affording compound <b>215</b>, greatly improved brain exposure in SMNÎ7 mice (<a class="ref internalNav" href="#sch34" aria-label="Scheme 34">Scheme 34</a>). Synthetically, Suzuki coupling reaction between pyridazine derivative <b>218</b> and 4-chloro-2-hydroxyphenylboronic acid pinacol ester <b>220</b> catalyzed by PdCl<sub>2</sub>(dppf)Â·DCM afforded the intermediate <b>221</b>. A second Suzuki coupling between <b>221</b> and 1-Boc-pyrazole-4-boronic acid pinacol ester <b>222</b> in the presence of BrettPhos Pd G1 (Buchwald first generation (G1) palladacycle precatalyst) provided the Boc-protected species <b>223</b>, and <i>N</i>-Boc deprotection under acidic conditions afforded branaplam <b>215</b> in 47% overall yield.</div><figure id="sch34" data-index="35" class="article__inlineFigure"><h2 class="fig-label">Scheme 34</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/medium/jm1c00154_0037.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0037.jpeg" id="gr37" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 34. Synthesis of Branaplam, Its Activity toward SMN2 pre-mRNA Splicing in SMN ELISA Assay, and a Comparison with Its Predecessor Analogue <b>214</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0037.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00154&amp;id=sch34"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec4_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i54" class="anchor-spacer"></div><h4 class="article-section__title" id="_i54">3.3.  Synthetic Molecules Targeting mRNA Repeats</h4><div class="NLM_p">Myotonic dystrophy type 1 (DM1) is a multisystem skeletal muscle disorder caused by the misregulation of pre-mRNA splicing. Expansion of r(CUG) repeats to form r(CUG)<sup>exp</sup> leads to aberrant sequestration of muscle blind-like 1 protein (MBNL1) in muscle cells, where r(CUG)<sup>exp</sup> wrinkles to form a stretched hairpin structure that provides several high-affinity binding sites for MBNL1.<a onclick="showRef(event, 'ref107'); return false;" href="javascript:void(0);" class="ref ref107">(107)</a> Excessive binding of MBNL1 to the RNA significantly decreases free MBNL1 levels and affects cellular functions, leading to DM1. Therefore, targeting the r(CUG)<sup>exp</sup> transcript in pre-mRNA with synthetic small molecules could be effective to block the binding sites for MBNL1 and restore its activity. Zimmerman and co-workers designed a new triaminotriazine-containing heterocyclic scaffold <b>230</b> (<a class="ref internalNav" href="#sch35" aria-label="Scheme 35">Scheme 35</a>) to target CUG units of pre-mRNA, on the basis of a molecular modeling approach.<a onclick="showRef(event, 'ref108'); return false;" href="javascript:void(0);" class="ref ref108">(108)</a> Groove-binding compound <b>230</b> was expected to be capable of covering three sequential CUG sites in the mRNA duplex, with the two triaminotriazine groups of the ligand forming strong base triplets with the corresponding UâU mismatches (<a class="ref internalNav" href="#sch35" aria-label="Scheme 35">Scheme 35</a>a,b). Furthermore, superposing the r(CUG)<sub>6</sub> duplex structure (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3GM7">3GM7</a>) on the stem-loop region of HIV-1 FS RNA (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2L94">2L94</a>) revealed an extraordinary similarity between the two RNA motifs (<a class="ref internalNav" href="#sch35" aria-label="Scheme 35">Scheme 35</a>b). The mechanism through which <b>230</b> binds to pre-mRNA and interrupts MBNL1N-RNA complex formation is not fully established. On the other hand, recent studies have demonstrated that small molecules that interact with CTG-rich DNA also reduce CUG RNA levels through the inhibition of transcription.<a onclick="showRef(event, 'ref109 ref110'); return false;" href="javascript:void(0);" class="ref ref109 ref110">(109,110)</a> Therefore, it seems possible that <b>230</b> binds to CTG DNA to reduce expanded CUG repeat RNA. For the synthesis of <b>230</b>, diethyl terephthalimidate <b>225</b> and <i>N</i><sup>2</sup>-(4-aminobutyl)-1,3,5-triazine-2,4,6-triamine <b>229</b> were first prepared <i>via</i> acid-mediated ethanolysis of 1,4-dicyanobenzene <b>224</b> and nucleophilic substitution reaction of 2-chloro-4,6-diamino-1,3,5-triazine <b>228</b> with 1,4-diaminobutane <b>227</b>, respectively (<a class="ref internalNav" href="#sch35" aria-label="Scheme 35">Scheme 35</a>). The intermediate <b>226</b>, which was synthesized by the reaction of <b>225</b> and <i>N</i>,<i>N</i>-dimethylpropylaminein ethanolic HCl, was then reacted with <b>229</b> in the presence of ethylene glycol and Et<sub>3</sub>N to afford <b>230</b>. The activity of <b>230</b> to inhibit MBNL1-CUG interaction was investigated by means of <i>in vitro</i> electrophoretic mobility shift assay, and an IC<sub>50</sub> value of 115 Î¼M was obtained. The binding affinity and selectivity of <b>230</b> for pre-mRNA were examined by isothermal titration calorimetry with several targets, including GST-tagged MBNL1, tRNA, pre-mRNA, r(CCUG)<sub>8</sub>, and HIV frameshift site RNA. They found that <b>230</b> displayed excellent selectivity and binding ability for r(CUG)<sub>12</sub> of pre-mRNA with <i>K</i><sub>d</sub> = 8 Î¼M, while other targets showed <i>K</i><sub>d</sub> > 200 Î¼M. An <i>in vivo</i> assessment of ligand <b>230</b> against r(CUG)<sup>exp</sup>-induced toxicity was also carried out using a DM1 transgenic <i>Drosophila</i> assay, and a significant reversal of pre-mRNA missplicing defects was detected in DM1-infected <i>Drosophila</i> flies. Compound <b>230</b> showed no cytotoxicity to human DM1 and 3T3 mouse fibroblast cells up to 100 Î¼M concentration, and the maximum tolerated dose of <b>230</b> applied as a single <i>i.p.</i> injection in mice was between 50 and 100 mg/kg. Thus, <b>230</b> appears to be a promising candidate for treatment of DM1.</div><figure id="sch35" data-index="36" class="article__inlineFigure"><h2 class="fig-label">Scheme 35</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/medium/jm1c00154_0039.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0039.jpeg" id="gr38" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 35. (a) Synthesis, Activity, and Binding Mode of Triaminotriazine-Containing Heterocyclic Scaffold Targeting CUG Units of pre-mRNA; (b) Superposition of the [r(CUG)<sub>6</sub>]<sub>2</sub> Structure (Blue; PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3GM7">3GM7</a>) and the Stem-Loop Region of HIV-1 FS RNA (Gray; PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2L94">2L94</a>) and the Proposed Binding of the Ligand <b>230</b> on the CUG Sequence<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0039.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00154&amp;id=sch35"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reprinted with permission from ref <a onclick="showRef(event, 'ref108'); return false;" href="javascript:void(0);" class="ref ref108">(108)</a> [  <cite><i>J. Am. Chem. Soc.</i></cite> <span class="NLM_year">2014</span>, <em>136</em>, 6355â6361]. Copyright 2014 American Chemical Society.</p></p></figure><div class="NLM_p">Bio-orthogonal chemical reactions are increasingly used to selectively synthesize a wide range of products.<a onclick="showRef(event, 'ref111'); return false;" href="javascript:void(0);" class="ref ref111">(111)</a> Tetrazine ligation is a bio-orthogonal cycloaddition reaction that can occur between 1,2,4,5-tetrazines and different dienophiles catalyzed by RNA templates and has been used for live-cell imaging.<a onclick="showRef(event, 'ref112'); return false;" href="javascript:void(0);" class="ref ref112">(112)</a> Disneyâs group demonstrated that r(CUG)<sup>exp</sup> was capable of catalyzing the <i>in situ</i> synthesis of an mRNA inhibitor <i>via</i> bio-orthogonal cycloaddition between a tetrazine and a cyclopropene in DM1-derived living cells.<a onclick="showRef(event, 'ref113'); return false;" href="javascript:void(0);" class="ref ref113">(113)</a> Peptide <b>239</b> with tetrazine and cyclopropane moiety was initially designed to target r(CUG)<sup>exp</sup> by means of the Inforna approach and was synthesized from readily accessible starting materials (<a class="ref internalNav" href="#sch36a" aria-label="Scheme 36">Scheme 36</a>,<a class="ref internalNav" href="#sch36a" aria-label="36">36</a>a,b). The coupling reaction of <i>tert</i>-butyl [4-(methylamino)butyl]carbamate or <i>tert</i>-butyl [2-(methylamino)ethyl]carbamate (<b>231</b>) with allyl chloroformate <b>232</b> provided allyloxycarbonyl (Alloc)-protected amines <b>233</b> (<a class="ref internalNav" href="#sch36a" aria-label="Scheme 36">Scheme 36</a>a). <i>N</i>-Boc deprotection, Rink amide resin activation, peptide chain extension on resin, Alloc deprotection, and resin removal reactions afforded <b>239</b>. This peptide oligomerized upon binding to r(CUG)<sup>exp</sup> in DM1-derived cells, resulting in the <i>in situ</i> formation of highly potent inhibitors (<b>240</b> and <b>241</b>), which were 10â¯000-fold more active than the parent tetrazine and cyclopropene monomers. Interestingly, the total level of MBNL1 mRNA was not affected by r(CUG)<sup>exp</sup>-templated oligomerized <b>239</b>, possibly due to enhancement of pre-mRNA splicing defects; approximately, 65% of MBNL1 splicing defects in DM1 cells could be rescued at the concentration of 1 Î¼M. The r(CUG)<sup>exp</sup>-templated <b>239</b> also decreased r(CUG)<sup>exp</sup>-MBNL1-containing nuclear foci by about 50% at 1 Î¼M concentration, supporting the idea that <b>239</b> can repair DM1-related splicing defects. A fluorogenic derivative of <b>239</b> was prepared using fluorescein alkyne 6-isomer (<b>242</b>, <a class="ref internalNav" href="#sch36a" aria-label="Scheme 36">Scheme 36</a>c). r(CUG)<sup>exp</sup> catalyzed <i>in situ</i> formation of its own fluorescent probe from triazine <b>242</b> and cyclopropene <b>243</b> (<a class="ref internalNav" href="#sch36a" aria-label="Scheme 36">Scheme 36</a>c), making it possible to visualize and monitor r(CUG)<sup>exp</sup> in living cells and DM1 mouse myofibers. This technique would allow us to broadly image various RNAs in cells and should be helpful to design novel therapeutic agents for selectively targeting RNAs, which remains one of the most challenging aspects of RNA-based drug discovery.</div><figure id="sch36a" data-index="37" class="article__inlineFigure"><h2 class="fig-label">Scheme 36</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/medium/jm1c00154_0040.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0040.jpeg" id="GRAPHIC-d7e3262-autogenerated" class="inline-fig internalNav" /></img><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/medium/jm1c00154_0041.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0041.jpeg" id="GRAPHIC-d7e3263-autogenerated" class="inline-fig internalNav" /></img><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/medium/jm1c00154_0042.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0042.jpeg" id="GRAPHIC-d7e3264-autogenerated" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 36. (a, b) Synthesis of a Tetrazine/Cyclopropene-Based Peptide; (c) <i>In Situ</i> Generation of Inhibitors to Target r(CUG)<sup>exp</sup> Repeats in DM1-Patient Derived Cells</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0040.jpeg jm1c00154_0041.jpeg jm1c00154_0042.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00154&amp;id=sch36a"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Similarly, Disney and co-workers reported r(CUG)<sup>exp</sup>-catalyzed <i>in situ</i> click synthesis of an mRNA inhibitor in DM1-affected cells (<a class="ref internalNav" href="#sch37" aria-label="Scheme 37">Scheme 37</a>).<a onclick="showRef(event, 'ref114'); return false;" href="javascript:void(0);" class="ref ref114">(114)</a> In their technique, a DM1-derived living cell and r(CUG)<sup>exp</sup> were utilized as a round-bottomed flask and as the catalyst, respectively, for probe synthesis. Peptide <b>245</b> with azide and alkyne moieties was designed to target r(CUG)<sup>exp</sup> by means of the Inforna method and synthesized using readily accessible starting materials (<a class="ref internalNav" href="#sch37" aria-label="Scheme 37">Scheme 37</a>). This compound <b>245</b> improved DM1-associated pre-mRNA splicing defects in myotubes and myoblasts at the dosage of 100 pM, on the basis of qPCR analysis, with no effect on non-DM1 myotubes or myoblasts. In addition, <b>245</b> had no effect on the splicing process of pre-mRNAs that were not controlled by MBNL1. To compare the developed small molecule and oligonucleotides, the efficacy of a Vivo-Morpholino oligonucleotide against DM1-associated defects was analyzed in a mouse model and compared with that of <b>245</b>. The latter exhibited approximately 25â¯000-fold greater potency than the former. Two main reasons were suggested for the high activity of <b>245</b>: recognition of r(CUG)<sup>exp</sup> structure rather than sequence, and better kinetics of binding to RNAs, compared with the oligonucleotide. An <i>in situ</i> fluorescent probe synthesis of <b>246</b> containing FRET pairs, <i>i.e.</i>, fluorescein amidites (FAM) and 5-carboxytetramethylrhodamine (TAMRA), was developed (<a class="ref internalNav" href="#sch37" aria-label="Scheme 37">Scheme 37</a>c), and the resulting FRET sensor was used to probe DM1-affected cells. Imaging analyses showed that binding to r(CUG)<sup>exp</sup> facilitated cytoplasmic translocation of the fluorescent probe <b>246</b>, allowing translation of luciferase possessing r(CUG)<sup>exp</sup> repeats in the 3â² UTR without modifying the level of luciferase mRNA. This <i>in situ</i> fluorescent probe synthetic platform is expected to be useful for imaging r(CUG)<sup>exp</sup> in living cells.</div><figure id="sch37" data-index="38" class="article__inlineFigure"><h2 class="fig-label">Scheme 37</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/medium/jm1c00154_0043.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0043.jpeg" id="gr42" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 37. (a) Synthesis of Peptide <b>245</b>; (b) r(CUG)<sup>exp</sup>-Catalyzed Click Chemistry between Azides and Alkynes; (c) <i>In Situ</i> Generation of <b>246</b> with FRET Pairs to Image r(CUG)<sup>exp</sup> Repeats in DM1-Patient-Derived Cells<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0043.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00154&amp;id=sch37"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reprinted with permission from ref <a onclick="showRef(event, 'ref114'); return false;" href="javascript:void(0);" class="ref ref114">(114)</a> [  <cite><i>Nat. Chem. Biol.</i></cite> <span class="NLM_year">2017</span>, <em>13</em>, 188â193]. Copyright 2017 Nature Publishing Group.</p></p></figure><div class="NLM_p">Myotonic dystrophy type 2 (DM2) is another skeletal muscle disorder caused by the dysfunction of pre-mRNA splicing, where expansion of r(CCUG) repeats leads to the formation of r(CCUG)<sup>exp</sup> in pre-mRNAs.<a onclick="showRef(event, 'ref115'); return false;" href="javascript:void(0);" class="ref ref115">(115)</a> This phenomenon generates high-affinity binding sites for MBNL1 in the loops, resulting in inactivation of MBNL1 upon binding and leading to DM2. Disneyâs group developed an <i>in cellulo</i> click chemistry strategy to synthesize kanamycin A-based small molecules containing azide and alkyne groups that attach to adjacent sites in r(CCUG)<sup>exp</sup> and undergo Huisgen cycloaddition reaction.<a onclick="showRef(event, 'ref116'); return false;" href="javascript:void(0);" class="ref ref116">(116)</a> Compound <b>251</b> was initially synthesized by the amide formation reaction of NBD-activated 6-aminohexynoic acid <b>249</b> with azide kanamycin A analogue <b>250</b> (<a class="ref internalNav" href="#sch38" aria-label="Scheme 38">Scheme 38</a>). This molecule, containing both azide and alkyne moieties, can generate an oligomer <b>252</b><i>in cellulo</i> that is a potent inhibitor of DM2 mRNA dysfunction. To investigate the selectivity of r-(CCUG) for oligomer formation, several RNAs, including r(AUUCU)<sub>12</sub>, r(CUG)<sub>12</sub>, r(CAG)<sub>12</sub>, r(CGG)<sub>12</sub>, an RNA hairpin with a fully paired stem, and tRNA, were studied. Among them, only r(CUG)<sub>12</sub> generated a substantial amount of products. A cellular model system expressing r(CCUG)<sub>300</sub> was utilized to test the possibility of templating multivalent compounds <i>in cellulo</i>. No oligomerization occurred in cells with no RNA expression, and the templated synthesis of the oligomer took place only in disease-affected cells, indicating that r(CCUG)<sub>300</sub> catalyzes the synthesis of its own inhibitor <i>in cellulo</i>. The oligomer <b>252</b> was >1000-fold more active than the parent monomer and rescued approximately 70% of MBNL1 splicing defects in DM2 cells at the concentration of 10 Î¼M. This study of the biological effects of the templated compounds on DM2-associated defects in living cells suggests that the approach may have therapeutic potential.</div><figure id="sch38" data-index="39" class="article__inlineFigure"><h2 class="fig-label">Scheme 38</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/medium/jm1c00154_0044.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0044.jpeg" id="gr43" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 38. Synthesis of a Kanamycin A-Based Small Molecule and <i>In Situ</i> Generation of an Oligomer to Target r(CCUG)<sup>exp</sup> Repeats in DM2-Patient-Derived Cells</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0044.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00154&amp;id=sch38"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Repeat expansion of RNA is a pathogenic process associated with many neurodegenerative diseases. Spinocerebellar ataxia type 10 (SCA10) is an autosomal major cerebellar ataxia that displays severe cerebellar dysfunctions, peripheral neuropathy, and epilepsy.<a onclick="showRef(event, 'ref117'); return false;" href="javascript:void(0);" class="ref ref117">(117)</a> SCA10 is caused by an RNA gain-of-function process due to expansion of r(AUUCU) repeats at intron 9 of the ataxin 10 mRNA, leading to the formation of defective proteins. Disneyâs group employed a two-dimensional combinatorial screening method to probe small molecules for binding affinity to GC and AU base pairs of RNA hairpin structure.<a onclick="showRef(event, 'ref118'); return false;" href="javascript:void(0);" class="ref ref118">(118)</a> Among a library of compounds including benzimidazole, benzamidine, and aniline derivatives, they identified compound <b>256</b> bearing a benzamidine core as the most selective ligand for AU base pairs (â¼40-fold selectivity over GC pairs). Pd-catalyzed cross-coupling reaction of 4-bromobenzonitrile <b>131</b> and furan <b>132</b> afforded 4,4â²-(furan-2,5-diyl)dibenzonitrile <b>133</b>, which underwent acid-mediated ethanolysis to give the corresponding diethyl 4,4â²-(furan-2,5-diyl)dibenzimidate hydrochloride <b>253</b> in quantitative yield. Neutralizing <b>253</b> with Et<sub>3</sub>N followed by reaction with azidopropylamine <b>113</b> and purging with NH<sub>3</sub> gas finally afforded <b>254</b>. The binding ability of compound <b>254</b> to the r(AUUCU) repeats in mRNAs of SCA10 cells was then evaluated. Ligand <b>254</b> exhibited high affinity for r(AUUCU)<sub>11</sub>, containing 10 AU pairs, with a <i>K</i><sub>d</sub> value of 300 nM, with a stoichiometry of approximately 11:1 (<b>254</b>:r(AUUCU)<sub>11</sub>). Encouraged by these results, they designed and synthesized dimer <b>256</b> by conjugating <b>254</b> onto a peptoid backbone <i>via</i> Cu(I)-catalyzed Huisgen cycloaddition reaction (<a class="ref internalNav" href="#sch39" aria-label="Scheme 39">Scheme 39</a>). The binding ability of <b>256</b> to r(AUUCU)<sub>11</sub> was approximately 2-fold higher than that of the monomer <b>254</b> (<i>K</i><sub>d</sub> = 185 nM). Since caspase 3 is aberrantly induced in SCA10 fibroblasts, they examined the effect of dimer <b>256</b> on the caspase-3 activity in SCA10 patient-derived fibroblasts and found that the caspase-3 activity was decreased to normal levels at 50 nM concentration. Furthermore, upon treatment of SCA10 cells with <b>256</b> for 48 h, the formation of nuclear foci was decreased by approximately 70%. Ligand <b>256</b> showed no apparent cytotoxicity at 50 nM, indicating that its bioactivity was due to binding to r(AAUCU)<sup>exp</sup> in the mRNAs. Thus, <b>256</b> and its analogues are candidate therapeutic agents for SCA10.</div><figure id="sch39" data-index="40" class="article__inlineFigure"><h2 class="fig-label">Scheme 39</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/medium/jm1c00154_0045.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0045.jpeg" id="gr44" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 39. Synthesis of Dimeric Benzamidine Derivative <b>256</b> and Its Activity in SCA10 Fibroblasts</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0045.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00154&amp;id=sch39"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec4_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i60" class="anchor-spacer"></div><h4 class="article-section__title" id="_i60">3.4.  Synthetic Molecules Targeting Viral mRNAs</h4><div class="NLM_p">Human immunodeficiency virus type-1 (HIV-1) is a lipid-enveloped retrovirus that causes progressive failure of the immune system. HIV-1 trans-acting responsive region (TAR) is positioned at the 5â² end of all HIV-1 mRNAs and binds to viral proteins such as trans-activator of transcription (Tat), a transactivator of the HIV-1 long terminal repeat, to generate ribonucleoprotein complexes.<a onclick="showRef(event, 'ref119'); return false;" href="javascript:void(0);" class="ref ref119">(119)</a> Tat is associated with several cellular events in HIV-1, including transcription and viral replication, <i>via</i> binding to the TAR RNA structure. Thus, TAR RNA has become a promising target of drug development to suppress HIV-1 viral replication. Although RNA targeting research in drug discovery has often focused on peptides and oligonucleotide-based antisense derivatives, these strategies suffer from insufficient specificity, poor cellular uptake, and intrinsic instability, and synthetic small molecules might overcome these disadvantages. Benhida and co-workers reported the synthesis of two types of conjugates with either one amino acid (<b>263</b>) or two amino acid (<b>266</b>) units (<a class="ref internalNav" href="#sch40" aria-label="Scheme 40">Scheme 40</a>) to investigate their suppressive activity against HIV-1 TAR RNAs.<a onclick="showRef(event, 'ref120'); return false;" href="javascript:void(0);" class="ref ref120">(120)</a> They selected basic amino acids to synthesize the ligands due to their potential for strong electrostatic interactions with the bulge bases of the RNAs.<a onclick="showRef(event, 'ref121'); return false;" href="javascript:void(0);" class="ref ref121">(121)</a> The key intermediate <b>261</b> was first synthesized through Î±-bromination of commercially available 3-nitroacetophenone <b>257</b> followed by thiazole formation and nitro reduction (<a class="ref internalNav" href="#sch40" aria-label="Scheme 40">Scheme 40</a>a). The preparation of conjugate <b>263</b> with one amino acid consisted of two steps: coupling of <b>261</b> with various <i>N</i>-protected amino acids and <i>N</i>-deprotection. The ligands <b>266</b> bearing two amino acids were also synthesized <i>via</i> coupling reaction of <b>261</b> with <i>N</i>-Boc-protected diaminopropanoic acid <b>264</b> as a linker, followed by deprotection, reaction with various <i>N</i>-Boc-protected amino acids, and deprotection (<a class="ref internalNav" href="#sch40" aria-label="Scheme 40">Scheme 40</a>b). The obtained compounds were examined for binding ability to HIV-1 TAR RNA by means of fluorescence measurements using a labeled TAR fragment. As expected, compounds bearing two basic amino acids (<b>266</b>) exhibited high binding affinity compared to those with one amino acid (<b>263</b>), due to the presence of four additional hydrogen-bonding interactions with C and G bases (<a class="ref internalNav" href="#sch40" aria-label="Scheme 40">Scheme 40</a>c). Among the synthesized compounds, compounds <b>263a</b> (monoarginine) and <b>266a</b> (diarginine) showed the best performance, with <i>K</i><sub>d</sub> values of 2.4 Î¼M and 0.11 Î¼M, respectively. The selectivity of active ligands was then examined using a 100-fold excess of tRNA and double-stranded DNA. Compounds <b>266b</b> (monohistidine) and <b>266b</b> (dilysine) were the most selective TAR RNA binders (<a class="ref internalNav" href="#sch40" aria-label="Scheme 40">Scheme 40</a>b). Circular dichroism analysis of the TAR hairpin conducted in the presence and absence of <b>266b</b> as a model ligand revealed that the ligand-RNA interaction caused no major change of the RNA structure and did not abolish the overall base stacking. Next, the suppressive activity of potent compounds against HIV-1 replication in MAGIC-5B cells was assessed. There was a close relationship between selectivity and antiviral activity. Indeed, ligands with low specificity were completely inactive. Compound <b>263a</b> (histidine) showed the most potent inhibitory activity, with the IC<sub>50</sub> value of 0.41 Î¼M. Furthermore, when various concentrations of <b>263a</b> (0.1â100 Î¼M) were incubated with MAGIC-5B cells for 48 h, no cytotoxicity was observed, suggesting that further <i>in vivo</i> investigations would be warranted.</div><figure id="sch40" data-index="41" class="article__inlineFigure"><h2 class="fig-label">Scheme 40</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/medium/jm1c00154_0046.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0046.jpeg" id="gr45" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 40. (a, b) Synthesis of Various Amino Acid-Based Heterocycles as Inhibitors of HIV-1 TAR mRNA Activity; (c) Binding Mode of <b>266a</b> with TAR mRNA</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0046.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00154&amp;id=sch40"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Dash <i>et al</i>. carried out <i>in situ</i> click chemistry using a large number of alkynes <b>267</b> with thiazole ring systems and azides <b>268</b> and discovered <i>anti</i>-(1,4)-triazoles <b>269</b> as potential HIV-1 TAR-Tat inhibitors (<a class="ref internalNav" href="#sch41" aria-label="Scheme 41">Scheme 41</a>).<a onclick="showRef(event, 'ref122'); return false;" href="javascript:void(0);" class="ref ref122">(122)</a> In a 5â²-biotin-tagged 29-nucleotide TAR RNA model system, the TAR RNA served as a template for the cycloaddition reaction, and the cyclized products were easily separated from the reaction mixture by using streptavidin magnetic beads. Interestingly, the Cu(I)-catalyzed cyclization reaction of alkynes and azides gave a mixture of <i>anti</i>-(1,4)- and <i>syn</i>-(1,5)-triazole regioisomers, but the <i>in situ</i> RNA-templated reaction provided exclusively the <i>anti</i>-1,4-triazole isomer. Among the triazoles, <b>269a</b> and <b>269b</b> displayed potent binding affinity for TAR RNA with <i>K</i><sub>d</sub> = 0.54 Î¼M and <i>K</i><sub>d</sub> = 6.7 Î¼M, respectively. They showed DC<sub>50</sub> values of 1.5  and 10.6 Î¼M, respectively (DC<sub>50</sub> is the concentration of a ligand causing 50% displacement of Tat from TAR RNA). Molecular modeling studies suggested that <b>269a</b> interacts with the bulge sites of the RNA, while <b>269b</b> binds to the major groove region. The flexible methylene unit of the left-hand thiazole ring in <b>269a</b> (<a class="ref internalNav" href="#sch41" aria-label="Scheme 41">Scheme 41</a>) appeared to facilitate interaction with the bulge sites. Further cell-based studies might lead to the development of new TAR RNA inhibitors as antiviral agents.</div><figure id="sch41" data-index="42" class="article__inlineFigure"><h2 class="fig-label">Scheme 41</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/medium/jm1c00154_0047.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0047.jpeg" id="gr46" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 41. <i>In Situ</i> Click Chemistry Using Alkynes <b>267</b> and Azides <b>268</b> in the Presence of TAR RNA to Discover Potential HIV-1 TAR-Tat Inhibitors</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0047.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00154&amp;id=sch41"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">In addition, Al-Hashim and co-workers reported the <i>de novo</i> discovery of some TAR RNA binders, including netilmicin and MTX <b>202a</b> (<a class="ref internalNav" href="#sch32" aria-label="Scheme 32">Scheme 32</a>), by <i>in silico</i> screening.<a onclick="showRef(event, 'ref123'); return false;" href="javascript:void(0);" class="ref ref123">(123)</a> Netilmicin selectively binds to HIV TAR and inhibits HIV-1 replication with an IC<sub>50</sub> value of 23.1 Î¼M. Thus, HIV-1 TAR RNA binders might be candidate anti-HIV agents.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i63" class="anchor-spacer"></div><h4 class="article-section__title" id="_i63">3.5.  Synthetic Molecules Targeting Bacterial mRNAs</h4><div class="NLM_p">Riboswitches are <i>cis</i> regulatory structural elements located in the UTR of mRNAs that specifically bind natural ligands to regulate gene expression in many microbial species.<a onclick="showRef(event, 'ref124'); return false;" href="javascript:void(0);" class="ref ref124">(124)</a> Riboswitches interact with metabolites such as vitamins, amino acids, <i>S</i>-adenosylmethionine, glucosamine-6-phosphate, and flavin mononucleotide (FMN), which induce a conformational alteration that modulates the expression of components of the relevant biosynthetic pathways. Riboflavin (vitamin B<sub>2</sub>) is essential for microbial growth, and FMN riboswitches modulate gene expression of transporters and enzymes involved in riboflavin biosynthesis.<a onclick="showRef(event, 'ref125'); return false;" href="javascript:void(0);" class="ref ref125">(125)</a> Therefore, disturbing riboflavin biosynthesis could be a suitable strategy for the prevention of bacterial cell growth. A group of scientists at Merck Research Laboratories discovered ribocil <b>277</b> as a highly selective synthetic modifier of bacterial riboflavin riboswitches.<a onclick="showRef(event, 'ref126'); return false;" href="javascript:void(0);" class="ref ref126">(126)</a> This synthetic mimic of FMN was designed to compete with the natural ligand in order to stop FMN riboswitch-mediated mRNA expression of genes associated with riboflavin biosynthesis. They screened a library of approximately 57â¯000 synthetic compounds with antibacterial activity to identify molecules exhibiting growth-inhibitory activity toward <i>Escherichia coli</i> strain MB5746. On the basis of this phenotypic screen, they discovered ribocil <b>277</b> (<a class="ref internalNav" href="#sch42" aria-label="Scheme 42">Scheme 42</a>), which blocks riboflavin biosynthesis with an IC<sub>50</sub> value of 0.3 Î¼M. This potent ligand was prepared in five simple synthetic steps from commercially available thiophene-2-carbonyl chloride <b>270</b> (<a class="ref internalNav" href="#sch42" aria-label="Scheme 42">Scheme 42</a>a). Nucleophilic acyl substitution reaction between <b>270</b> and ethyl acetate initially gave ethyl 3-oxo-3-(thiophen-2-yl)propanoate <b>271</b>, followed by nucleophilic addition of ammonia to provide acrylamide <b>272</b>. Oxidative annulation-aromatization reaction between <b>272</b> and 1-<i>tert</i>-butyl 3-methylpiperidine-1,3-dicarboxylate <b>273</b> generated pyrimidine derivative <b>274</b>. Finally, sequential<i>N</i>-Boc deprotection/reductive amination afforded ribocil <b>277</b> (<a class="ref internalNav" href="#sch42" aria-label="Scheme 42">Scheme 42</a>a). To clarify the cellular target of ribocil <b>277</b> and its mechanism of action, several ribocil-resistant <i>E. coli</i> mutants were independently isolated and their drug-resistant mutations were investigated <i>via</i> whole-genome sequencing (WGS). All drug-resistant mutants showed base-pair variations mapping to the FMN riboswitch. Furthermore, none of the detected mutations served as compensatory mutations. The absence of additional mutations in all ribocil-resistant isolates detected by WGS clearly implicated the FMN riboswitch as a fundamental element for ribocil resistance. On the basis of these findings, it was hypothesized that ribocil <b>277</b> prevents riboflavin biosynthesis directly through mimicking the FMN ligand and binding to the FMN riboswitch to impair mRNA expression of riboflavin. To test this idea, a plasmid-based GFP reporter gene was utilized under the control of <i>E. coli</i> riboflavin promoter and FMN riboswitch 5â² sequence of mRNA in a ribocil-resistant isolate. This technique enables a ribocil-mediated effect on the expression of GFP to be measured without inhibiting cell growth. Ribocil <b>277</b> potently inhibited GFP expression with an EC<sub>50</sub> value of 0.3 Î¼M. Ribocil <b>277 also</b> effectively repressed GFP reporter expression controlled by orthologous riboswitches from <i>Acinetobacter baumannii</i> and <i>Pseudomonas aeruginosa</i>, with approximately 5-fold higher EC<sub>50</sub> than that observed for the <i>E. coli</i> riboswitch. In contrast, GFP reporter constructs harboring <i>E. coli</i> ribocil-resistant riboswitch mutations were mostly unresponsive to inhibition by ribocil <b>277</b>, further supporting the importance of a direct interaction between ribocil <b>277</b> and the riboswitch. Following unsuccessful attempts to cocrystallize the <i>E. coli</i> FMN aptamer with ribocil <b>277</b>, the X-ray crystal structure of ribocil <b>277</b> bound to the <i>Fusobacterium nucleatum</i> impX FMN riboswitch aptamer was successfully obtained (<a class="ref internalNav" href="#sch42" aria-label="Scheme 42">Scheme 42</a>b). In the complex, ribocil <b>277</b> adopts a forced U-shaped conformation and is located inside the junctional area of the six mRNAs. The 6-thiophenyl-pyrimidonyl group of ribocil <b>277</b> stacks face-to-face between A48 and A85 and edge-to-face with A49. Hydrogen bond formation between the pyrimidonyl oxygen of ribocil <b>277</b> and A48 and A99 is critical. Furthermore, the (methylamino)pyrimidinyl group of <b>277</b> stacks face-to-face with G62, and a weak hydrogen bond is formed between the hydrogen at position 5 of the piperazine of <b>277</b> and G11. Although a racemic mixture of ribocil <b>277</b> was employed for cocrystallization, electron density mapping of the cocrystal structure showed that only the (<i>S</i>)-isomer is bound in the cocrystal, and this may be the biologically active form. <i>In vivo</i> activity of ribocil <b>277</b> in a murine systemic infection model of <i>E. coli</i> was investigated. In this experiment, female DBA/2J mice were infected with <i>E. coli</i> strain MB5746 (5.0 Ã 10<sup>4</sup> CFU per mouse) <i>via</i><i>i.p.</i> injection and subsequently treated by subcutaneous injection with ribocil-C ((<i>S</i>)-isomer of <b>277</b>) at various doses (120, 60, or 30 mg/kg) 3 times over an infection period of 24 h. Spleens were then aseptically collected from five mice per group and the reduction of log[CFU per g spleen tissue] was measured, on the basis of the bacterial burden in spleens of the vehicle-treated (10% DMSO) control group. Significant reductions were observed from 9.53 log[CFU per g spleen] in the vehicle-treated group to 7.76 log[CFU per g spleen] and 6.24 log[CFU per g spleen] in in the 60 and 120 mg/kg ribocil-C treatment groups, with no mortality or gross toxic effects. This study suggests high pharmacological potential of ribocil <b>277</b> as an antimicrobial agent.</div><figure id="sch42" data-index="43" class="article__inlineFigure"><h2 class="fig-label">Scheme 42</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/medium/jm1c00154_0048.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0048.jpeg" id="gr47" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 42. (a) Synthesis of Ribocil, a Highly Selective Modifier of Bacterial Riboflavin Riboswitches; (b) X-ray Crystal Structure of Ribocil Bound to the <i>F. nucleatum</i> FMN Riboswitch<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0048.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00154&amp;id=sch42"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reprinted with permission from ref <a onclick="showRef(event, 'ref126'); return false;" href="javascript:void(0);" class="ref ref126">(126)</a> [  <cite><i>Nature</i></cite> <span class="NLM_year">2015</span>, <em>526</em>, 672â677]. Copyright 2015 Nature Publishing Group.</p></p></figure></div></div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i65">Conclusion and Perspectives</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_57573" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_57573" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">Chemotherapy and radiotherapy are often unsuccessful in patients with advanced cancer, <i>e.g.</i>, showing invasion and metastasis, due to drug resistance.<a onclick="showRef(event, 'ref127'); return false;" href="javascript:void(0);" class="ref ref127">(127)</a> Various factors may be involved, including altered drug transport or metabolism, inactivation, and genetic and epigenetic changes in the tumor cells. In this context, synthetic RNA-targeting molecules represent a promising strategy to treat drug-resistant cancers. Regulating gene expression in cancer cells with synthetic RNA modulators can cause primary genotypic variations that affect the cell life cycle and promote apoptosis of tumor cells. Apart from cancers, many other human diseases are associated with dysfunction of cellular proteins, which may be undruggable due to the lack of binding pockets. In such cases, a fascinating new strategy to knock down the undruggable proteins is to target their encoding mRNAs and thereby inhibit their translation. For this purpose, we require high selectivity and binding affinity for a specific RNA, excellent cellular uptake, low cytotoxicity, and stability. Synthetic molecules can provide these advantages, together with great structural diversity and relatively easy tunability of their physicochemical and pharmacological characteristics. In this review, we have highlighted the current status of research on synthetic RNA modulators as chemical probes to reverse severe cellular dysfunctions. Recent advances in RNA sequencing, structural confirmation by X-ray crystallography, <i>in vitro</i> and <i>in vivo</i> chemical RNA probing, and biophysical techniques have provided powerful interdisciplinary approaches to identify druggable human-disease-causing RNAs, opening up numerous avenues for further exploration of new synthetic ligands in drug design. Nevertheless, this field is still at an early stage of development, and although synthetic RNA-targeting compounds show great promise, high potency in <i>in vitro</i> assays does not necessarily predict on-target efficacy in living cells. Thus, further research employing target engagement strategies based on <i>in vivo</i> assays will be needed to explore targeting of RNAs by synthetic molecules. Improvements in efficient screening technologies such as Inforna may lead to the establishment of rational methodologies to design synthetic molecules directly from sequence data. Furthermore, the activities of most reported RNA modulators remain at around Î¼M level, and in many cases it is not clear whether the RNA-binding activity is linked to the effect on cell phenotype. Therefore, there is a great need for further mechanistic studies to establish the modes of action of these compounds. On the basis of the current experimental data, many of these synthetic molecules may also be recognized as modulators of RNA-binding proteins (RBPs) to alter RBP function and lead to the production of new therapeutics, since RBP dysregulations are implicated in several cancers and neurodegenerative diseases.<a onclick="showRef(event, 'ref128'); return false;" href="javascript:void(0);" class="ref ref128">(128)</a> Nevertheless, the available data justify a degree of optimizm, and it seems likely that structural optimization by medicinal chemists will lead to more potent and selective modulators that show clear activity at the cellular level. We hope this review will be especially useful in helping medical researchers to better understand potential future applications of these modulators in clinical therapy for cancer and other diseases.</div></div><div class="NLM_back"><div class="article_supporting-info" id=""></div><div class="testing" data-doi="10.1021/acs.jmedchem.1c00154" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_70324" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_70324" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Takayoshi Suzuki</span> - <span class="hlFld-Affiliation affiliation">The Institute of Scientific and Industrial
Research, Osaka University, Mihogaoka 8-1, Ibaraki, Osaka 567-0047, Japan</span>;Â 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0003-2439-879X" title="Orcid link">https://orcid.org/0000-0003-2439-879X</a>;Â 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#7307180a0000060906181a3300121d18161d5d1c001218125e065d12105d1903"><span class="__cf_email__" data-cfemail="01756a787272747b746a684172606f6a646f2f6e72606a602c742f60622f6b71">[emailÂ protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Farzad Zamani</span> - <span class="hlFld-Affiliation affiliation">The Institute of Scientific and Industrial
Research, Osaka University, Mihogaoka 8-1, Ibaraki, Osaka 567-0047, Japan</span></div></li></ul></li><li></li><li><span class="author-information-subsection-header">Funding</span><p>This work was supported by âDynamic Alliance for Open Innovation Bridging Human, Environment and Materialsâ from the Ministry of Education, Culture, Sports, Science and Technology of Japan (MEXT) (T.S.), the Project for Cancer Research and Therapeutic Evolution (T.S.), the Naito Foundation (T.S.), and a JSPS fellowship for foreign researchers (F.Z.).</p></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="bio" rid="ath1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="biographies">Biographies</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_33103" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_33103" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><figure id="ath1" data-index="44" class="article__inlineFigure"><h2 class="fig-label">Farzad Zamani</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00154&amp;id=BIO-d7e4561-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Farzad Zamani</b> received his M.Sc. in organic chemistry from Islamic Azad University (Iran) in 2009. He then worked as a research assistant at Isfahan University of Technology, focusing on heterogeneous catalysis (2009â2014). He completed his PhD at the University of Wollongong (Australia) in 2019 under the supervision of Associate Professor Christopher Hyland and Professor Stephen Pyne, where he worked on transition metal-catalysed cyclization reactions and heterocycle synthesis. He is currently a JSPS postdoctoral fellow in the group led by Professor Takayoshi Suzuki at The Institute of Scientific and Industrial Research, Osaka University, focusing on the synthesis and investigation of small molecule RNA inhibitors in drug discovery.</p></figure></div><div class="bio" rid="ath2"><figure id="ath2" data-index="45" class="article__inlineFigure"><h2 class="fig-label">Takayoshi Suzuki</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00154&amp;id=BIO-d7e4566-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Takayoshi Suzuki</b> received his B.Sc. and M.Sc. from the University of Tokyo (1995 and 1997, respectively), after which he became a researcher at Japan Tobacco, Inc. (1997â2002). He subsequently joined Nagoya City University, Japan, as an assistant professor (2003â2009) and later a lecturer (2009â2011). During that time, he received his Ph.D. from the Graduate School of Pharmaceutical Sciences, the University of Tokyo. He spent one year as a visiting investigator at the Scripps Research Institute, CA (Prof. M. G. Finn) (2007â2008). He was a full-time professor at Kyoto Prefectural University of Medicine, Japan (2011 to 2019), and is currently a full-time professor at Osaka University (2019 to present). His research areas of interest are medicinal chemistry and bioorganic chemistry, including epigenetic drugs.</p></figure></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">PTM</td><td class="NLM_def"><p class="first last">post-translational modification</p></td></tr><tr><td class="NLM_term">ncRNA</td><td class="NLM_def"><p class="first last">noncoding RNA</p></td></tr><tr><td class="NLM_term">miRNA</td><td class="NLM_def"><p class="first last">microRNA</p></td></tr><tr><td class="NLM_term">lncRNA</td><td class="NLM_def"><p class="first last">long noncoding RNA</p></td></tr><tr><td class="NLM_term">pre-mRNA</td><td class="NLM_def"><p class="first last">precursor mRNA</p></td></tr><tr><td class="NLM_term">pri-mRNA</td><td class="NLM_def"><p class="first last">primary mRNA</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structureâactivity relationship</p></td></tr><tr><td class="NLM_term">RISC</td><td class="NLM_def"><p class="first last">RNA-induced silencing complex</p></td></tr><tr><td class="NLM_term">ASO</td><td class="NLM_def"><p class="first last">antisense oligonucleotides</p></td></tr><tr><td class="NLM_term">2-AP</td><td class="NLM_def"><p class="first last">2-aminopurine</p></td></tr><tr><td class="NLM_term">PCDP4</td><td class="NLM_def"><p class="first last">programmed cell death protein 4</p></td></tr><tr><td class="NLM_term">PTEN</td><td class="NLM_def"><p class="first last">phosphatase and tensin homologue</p></td></tr><tr><td class="NLM_term">RCC</td><td class="NLM_def"><p class="first last">renal cell carcinoma</p></td></tr><tr><td class="NLM_term">UTR</td><td class="NLM_def"><p class="first last">untranslated region</p></td></tr><tr><td class="NLM_term">DMSO</td><td class="NLM_def"><p class="first last">dimethyl sulfoxide</p></td></tr><tr><td class="NLM_term">RT-PCR</td><td class="NLM_def"><p class="first last">reverse transcription polymerase chain reaction</p></td></tr><tr><td class="NLM_term">RT-qPCR</td><td class="NLM_def"><p class="first last">quantitative reverse transcription polymerase chain reaction</p></td></tr><tr><td class="NLM_term">RLU</td><td class="NLM_def"><p class="first last">relative light units</p></td></tr><tr><td class="NLM_term">IC<sub>50</sub></td><td class="NLM_def"><p class="first last">half-maximal inhibitory concentration</p></td></tr><tr><td class="NLM_term">FP</td><td class="NLM_def"><p class="first last">fluorescence polarization</p></td></tr><tr><td class="NLM_term">LATS2</td><td class="NLM_def"><p class="first last">large-tumor suppressor homologue 2</p></td></tr><tr><td class="NLM_term">AGS</td><td class="NLM_def"><p class="first last">adenocarcinoma gastric cancer</p></td></tr><tr><td class="NLM_term">FRET</td><td class="NLM_def"><p class="first last">fluorescence resonance energy transfer</p></td></tr><tr><td class="NLM_term">EMT</td><td class="NLM_def"><p class="first last">epithelialâmesenchymal transition</p></td></tr><tr><td class="NLM_term">PA</td><td class="NLM_def"><p class="first last">peptidic aminosugar</p></td></tr><tr><td class="NLM_term">CRC</td><td class="NLM_def"><p class="first last">colorectal cancer</p></td></tr><tr><td class="NLM_term">MTT</td><td class="NLM_def"><p class="first last">3-(4,5-dimethylthiazol-2-yl)-2,5-diphenyltetrazolium bromide</p></td></tr><tr><td class="NLM_term">HCV</td><td class="NLM_def"><p class="first last">hepatitis C virus</p></td></tr><tr><td class="NLM_term">HCC</td><td class="NLM_def"><p class="first last">hepatocellular carcinoma</p></td></tr><tr><td class="NLM_term">BNIP3</td><td class="NLM_def"><p class="first last">BCL-2 interacting protein 3</p></td></tr><tr><td class="NLM_term">2DCS</td><td class="NLM_def"><p class="first last">two-dimensional combinatorial screening</p></td></tr><tr><td class="NLM_term">StARTS</td><td class="NLM_def"><p class="first last">structureâactivity relationships through sequencing</p></td></tr><tr><td class="NLM_term">HiT-StARTS</td><td class="NLM_def"><p class="first last">high-throughput structureâactivity relationships through sequencing</p></td></tr><tr><td class="NLM_term">FISH</td><td class="NLM_def"><p class="first last">fluorescence <i>in situ</i> hybridization</p></td></tr><tr><td class="NLM_term">HIFs</td><td class="NLM_def"><p class="first last">hypoxia-inducible factors</p></td></tr><tr><td class="NLM_term">HER</td><td class="NLM_def"><p class="first last">human epidermal growth factor receptor</p></td></tr><tr><td class="NLM_term">SK1</td><td class="NLM_def"><p class="first last">sphingosine kinase 1</p></td></tr><tr><td class="NLM_term">S1P</td><td class="NLM_def"><p class="first last">sphingosine 1-phosphate</p></td></tr><tr><td class="NLM_term">RNAi</td><td class="NLM_def"><p class="first last">RNA interference</p></td></tr><tr><td class="NLM_term">shRNA</td><td class="NLM_def"><p class="first last">short hairpin RNA</p></td></tr><tr><td class="NLM_term">siRNAs</td><td class="NLM_def"><p class="first last">small interfering RNAs</p></td></tr><tr><td class="NLM_term">EGFP</td><td class="NLM_def"><p class="first last">enhanced green fluorescent protein</p></td></tr><tr><td class="NLM_term">EC<sub>50</sub></td><td class="NLM_def"><p class="first last">half-maximal effective concentration</p></td></tr><tr><td class="NLM_term">MALAT1</td><td class="NLM_def"><p class="first last">metastasis-associated lung adenocarcinoma transcript 1</p></td></tr><tr><td class="NLM_term">ENE</td><td class="NLM_def"><p class="first last">element of nuclear expression</p></td></tr><tr><td class="NLM_term">AGPS</td><td class="NLM_def"><p class="first last">alkylglycerone phosphate synthase</p></td></tr><tr><td class="NLM_term">HOTAIR</td><td class="NLM_def"><p class="first last">HOX transcript antisense intergenic RNA</p></td></tr><tr><td class="NLM_term">EZH2</td><td class="NLM_def"><p class="first last">enhancer of zeste homologue 2</p></td></tr><tr><td class="NLM_term">NLK</td><td class="NLM_def"><p class="first last">nemo-like kinase</p></td></tr><tr><td class="NLM_term"><i>c</i>DNA</td><td class="NLM_def"><p class="first last">complementary DNA</p></td></tr><tr><td class="NLM_term">TERRA</td><td class="NLM_def"><p class="first last">telomeric repeat containing RNA</p></td></tr><tr><td class="NLM_term">â1PRF</td><td class="NLM_def"><p class="first last">programmed â1 ribosomal frameshifting</p></td></tr><tr><td class="NLM_term">MMTV</td><td class="NLM_def"><p class="first last">mouse mammary tumor virus</p></td></tr><tr><td class="NLM_term">FTDP-17</td><td class="NLM_def"><p class="first last">frontotemporal dementia and parkinsonism</p></td></tr><tr><td class="NLM_term">4R</td><td class="NLM_def"><p class="first last">four repeat</p></td></tr><tr><td class="NLM_term">ADME</td><td class="NLM_def"><p class="first last">absorption, distribution, metabolism, and elimination</p></td></tr><tr><td class="NLM_term">MTX</td><td class="NLM_def"><p class="first last">mitoxantrone</p></td></tr><tr><td class="NLM_term">PNA</td><td class="NLM_def"><p class="first last">peptide-nucleic acid</p></td></tr><tr><td class="NLM_term">SMA</td><td class="NLM_def"><p class="first last">spinal muscular atrophy</p></td></tr><tr><td class="NLM_term">SMN</td><td class="NLM_def"><p class="first last">survival motor neuron</p></td></tr><tr><td class="NLM_term">DM1</td><td class="NLM_def"><p class="first last">myotonic dystrophy type 1</p></td></tr><tr><td class="NLM_term">MBNL1</td><td class="NLM_def"><p class="first last">muscle blind-like 1 protein</p></td></tr><tr><td class="NLM_term">DM2</td><td class="NLM_def"><p class="first last">myotonic dystrophy type 2</p></td></tr><tr><td class="NLM_term">HIV-1</td><td class="NLM_def"><p class="first last">human immunodeficiency virus type-1</p></td></tr><tr><td class="NLM_term">TAR</td><td class="NLM_def"><p class="first last">trans-acting responsive region</p></td></tr><tr><td class="NLM_term">Tat</td><td class="NLM_def"><p class="first last">trans-activator of transcription</p></td></tr><tr><td class="NLM_term">CD</td><td class="NLM_def"><p class="first last">circular dichroism</p></td></tr><tr><td class="NLM_term">SCA10</td><td class="NLM_def"><p class="first last">spinocerebellar ataxia type 10</p></td></tr><tr><td class="NLM_term">BBB</td><td class="NLM_def"><p class="first last">bloodâbrain barrier</p></td></tr><tr><td class="NLM_term">CNS MPO</td><td class="NLM_def"><p class="first last">CNS multiparameter optimization</p></td></tr><tr><td class="NLM_term">IRE</td><td class="NLM_def"><p class="first last">iron responsive element</p></td></tr><tr><td class="NLM_term"><i>V</i><sub>ss</sub></td><td class="NLM_def"><p class="first last">volume of distribution</p></td></tr><tr><td class="NLM_term">FMN</td><td class="NLM_def"><p class="first last">flavin mononucleotide</p></td></tr><tr><td class="NLM_term">WGS</td><td class="NLM_def"><p class="first last">whole-genome sequencing</p></td></tr><tr><td class="NLM_term">RBPs</td><td class="NLM_def"><p class="first last">RNA-binding proteins</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i69">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_00656" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_00656" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 128 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Walsh, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jefferis, R.</span></span> <span> </span><span class="NLM_article-title">Post-translational modifications in the context of therapeutic proteins</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">1241</span>â <span class="NLM_lpage">1252</span>, <span class="refDoi">Â DOI: 10.1038/nbt1252</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1038%2Fnbt1252" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=17033665" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtVGgsrnK" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2006&pages=1241-1252&author=G.+Walshauthor=R.+Jefferis&title=Post-translational+modifications+in+the+context+of+therapeutic+proteins&doi=10.1038%2Fnbt1252"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Post-translational modifications in the context of therapeutic proteins</span></div><div class="casAuthors">Walsh, Gary; Jefferis, Roy</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1241-1252</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The majority of protein-based biopharmaceuticals approved or in clin. trials bear some form of post-translational modification (PTM), which can profoundly affect protein properties relevant to their therapeutic application.  Whereas glycosylation represents the most common modification, addnl. PTMs, including carboxylation, hydroxylation, sulfation, and amidation, are characteristic of some products.  The relation between structure and function is understood for many PTMs but remains incomplete for others, particularly in the case of complex PTMs, such as glycosylation.  A better understanding of such structure-function relations will facilitate the development of 2nd-generation products displaying a PTM profile engineered to optimize therapeutic usefulness.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJTnRzm0gY6LVg90H21EOLACvtfcHk0lhHlzzvfsVdHQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtVGgsrnK&md5=3376c8170602376c6fe0505c32533828</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnbt1252&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt1252%26sid%3Dliteratum%253Aachs%26aulast%3DWalsh%26aufirst%3DG.%26aulast%3DJefferis%26aufirst%3DR.%26atitle%3DPost-translational%2520modifications%2520in%2520the%2520context%2520of%2520therapeutic%2520proteins%26jtitle%3DNat.%2520Biotechnol.%26date%3D2006%26volume%3D24%26spage%3D1241%26epage%3D1252%26doi%3D10.1038%2Fnbt1252" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Deribe, Y. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pawson, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dikic, I.</span></span> <span> </span><span class="NLM_article-title">Post-translational modifications in signal integration</span>. <i>Nat. Struct. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">666</span>â <span class="NLM_lpage">672</span>, <span class="refDoi">Â DOI: 10.1038/nsmb.1842</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1038%2Fnsmb.1842" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=20495563" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC3cXmsVWmsLw%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2010&pages=666-672&author=Y.+L.+Deribeauthor=T.+Pawsonauthor=I.+Dikic&title=Post-translational+modifications+in+signal+integration&doi=10.1038%2Fnsmb.1842"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Post-translational modifications in signal integration</span></div><div class="casAuthors">Deribe, Yonathan Lissanu; Pawson, Tony; Dikic, Ivan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">666-672</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Post-translational modifications of proteins and the domains that recognize these modifications play central roles in creating a highly dynamic relay system that reads and responds to alterations in the cellular microenvironment.  Here, the authors review the common principles of post-translational modifications and their importance in signal integration underlying epidermal growth factor receptor signaling and endocytosis, DNA-damage responses, and immunity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3REKBFj9N9rVg90H21EOLACvtfcHk0lhHlzzvfsVdHQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXmsVWmsLw%253D&md5=4486fed28be542d6f49aae1405c85ee7</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnsmb.1842&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb.1842%26sid%3Dliteratum%253Aachs%26aulast%3DDeribe%26aufirst%3DY.%2BL.%26aulast%3DPawson%26aufirst%3DT.%26aulast%3DDikic%26aufirst%3DI.%26atitle%3DPost-translational%2520modifications%2520in%2520signal%2520integration%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2010%26volume%3D17%26spage%3D666%26epage%3D672%26doi%3D10.1038%2Fnsmb.1842" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peterson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loring, J.</span></span> <span> </span><span class="NLM_article-title">Protein post-translational modifications and regulation of pluripotency in human stem cells</span>. <i>Cell Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">143</span>â <span class="NLM_lpage">160</span>, <span class="refDoi">Â DOI: 10.1038/cr.2013.151</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1038%2Fcr.2013.151" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=24217768" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslyrtr3N" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2014&pages=143-160&author=Y.+Wangauthor=S.+Petersonauthor=J.+Loring&title=Protein+post-translational+modifications+and+regulation+of+pluripotency+in+human+stem+cells&doi=10.1038%2Fcr.2013.151"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Protein post-translational modifications and regulation of pluripotency in human stem cells</span></div><div class="casAuthors">Wang, Yu-Chieh; Peterson, Suzanne E.; Loring, Jeanne F.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">143-160</span>CODEN:
                <span class="NLM_cas:coden">CREEB6</span>;
        ISSN:<span class="NLM_cas:issn">1001-0602</span>.
    
            (<span class="NLM_cas:orgname">NPG Nature Asia-Pacific</span>)
        </div><div class="casAbstract">A review.  Post-translational modifications (PTMs) are known to be essential mechanisms used by eukaryotic cells to diversify their protein functions and dynamically coordinate their signaling networks.  Defects in PTMs have been linked to numerous developmental disorders and human diseases, highlighting the importance of PTMs in maintaining normal cellular states.  Human pluripotent stem cells (hPSCs), including embryonic stem cells (hESCs) and induced pluripotent stem cells (hiPSCs), are capable of self-renewal and differentiation into a variety of functional somatic cells; these cells hold a great promise for the advancement of biomedical research and clin. therapy.  The mechanisms underlying cellular pluripotency in human cells have been extensively explored in the past decade.  In addn. to the vast amt. of knowledge obtained from the genetic and transcriptional research in hPSCs, there is a rapidly growing interest in the stem cell biol. field to examine pluripotency at the protein and PTM level.  This review addresses recent progress toward understanding the role of PTMs (glycosylation, phosphorylation, acetylation and methylation) in the regulation of cellular pluripotency.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVfWBZsXmIn7Vg90H21EOLACvtfcHk0li01Wm4LhC_8A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslyrtr3N&md5=4d3f8b4cb9a995f25993a675ec9c4f33</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fcr.2013.151&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fcr.2013.151%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DY.%26aulast%3DPeterson%26aufirst%3DS.%26aulast%3DLoring%26aufirst%3DJ.%26atitle%3DProtein%2520post-translational%2520modifications%2520and%2520regulation%2520of%2520pluripotency%2520in%2520human%2520stem%2520cells%26jtitle%3DCell%2520Res.%26date%3D2014%26volume%3D24%26spage%3D143%26epage%3D160%26doi%3D10.1038%2Fcr.2013.151" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hopkins, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groom, C. R.</span></span> <span> </span><span class="NLM_article-title">The druggable genome</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">727</span>â <span class="NLM_lpage">730</span>, <span class="refDoi">Â DOI: 10.1038/nrd892</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1038%2Fnrd892" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=12209152" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BD38XmslamtrY%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2002&pages=727-730&author=A.+L.+Hopkinsauthor=C.+R.+Groom&title=The+druggable+genome&doi=10.1038%2Fnrd892"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Opinion: The druggable genome</span></div><div class="casAuthors">Hopkins, Andrew L.; Groom, Colin R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">727-730</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  An assessment of the no. of mol. targets that represent an opportunity for therapeutic intervention is crucial to the development of post-genomic research strategies within the pharmaceutical industry.  Now that we know the size of the human genome, it is interesting to consider just how many mol. targets this opportunity represents.  We start from the position that we understand the properties that are required for a good drug, and therefore must be able to understand what makes a good drug target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq7Uvqi8O9wzrVg90H21EOLACvtfcHk0li01Wm4LhC_8A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XmslamtrY%253D&md5=d6dc7c67822661fe03e920695def1329</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fnrd892&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd892%26sid%3Dliteratum%253Aachs%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26aulast%3DGroom%26aufirst%3DC.%2BR.%26atitle%3DThe%2520druggable%2520genome%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2002%26volume%3D1%26spage%3D727%26epage%3D730%26doi%3D10.1038%2Fnrd892" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dixon, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stockwell, B. R.</span></span> <span> </span><span class="NLM_article-title">Identifying druggable disease-modifying gene products</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">549</span>â <span class="NLM_lpage">555</span>, <span class="refDoi">Â DOI: 10.1016/j.cbpa.2009.08.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1016%2Fj.cbpa.2009.08.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=19740696" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsV2msLzJ" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2009&pages=549-555&author=S.+J.+Dixonauthor=B.+R.+Stockwell&title=Identifying+druggable+disease-modifying+gene+products&doi=10.1016%2Fj.cbpa.2009.08.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Identifying druggable disease-modifying gene products</span></div><div class="casAuthors">Dixon, Scott J.; Stockwell, Brent R.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5-6</span>),
    <span class="NLM_cas:pages">549-555</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Many disease genes encode proteins that are difficult to target directly using small mol. drugs.  Improvements in libraries based on synthetic compds., natural products, and other types of mols. may ultimately allow some challenging proteins to be successfully targeted; however, these developments alone are unlikely to be sufficient.  A complementary strategy exploits the functional interconnectivity of intracellular networks to find druggable targets lying upstream, downstream, or in parallel to a disease-causing gene, where modulation can influence the disease process indirectly.  These targets can be selected using prior knowledge of disease-assocd. pathways or identified using phenotypic chem. and genetic screens in model organisms and cells.  These approaches should facilitate the identification of effective drug targets for many genetic disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJzodncPkmoLVg90H21EOLACvtfcHk0ljwTQWswZaQDg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsV2msLzJ&md5=263d9f02416aac4fad4a7e7adfdda665</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2009.08.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2009.08.003%26sid%3Dliteratum%253Aachs%26aulast%3DDixon%26aufirst%3DS.%2BJ.%26aulast%3DStockwell%26aufirst%3DB.%2BR.%26atitle%3DIdentifying%2520druggable%2520disease-modifying%2520gene%2520products%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2009%26volume%3D13%26spage%3D549%26epage%3D555%26doi%3D10.1016%2Fj.cbpa.2009.08.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Warner, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hajdin, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weeks, K. M.</span></span> <span> </span><span class="NLM_article-title">Principles for targeting RNA with drug-like small molecules</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">547</span>â <span class="NLM_lpage">558</span>, <span class="refDoi">Â DOI: 10.1038/nrd.2018.93</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1038%2Fnrd.2018.93" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=29977051" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1OntLnL" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2018&pages=547-558&author=K.+D.+Warnerauthor=C.+E.+Hajdinauthor=K.+M.+Weeks&title=Principles+for+targeting+RNA+with+drug-like+small+molecules&doi=10.1038%2Fnrd.2018.93"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Principles for targeting RNA with drug-like small molecules</span></div><div class="casAuthors">Warner, Katherine Deigan; Hajdin, Christine E.; Weeks, Kevin M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">547-558</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">RNA mols. are essential for cellular information transfer and gene regulation, and RNAs have been implicated in many human diseases.  Messenger and non-coding RNAs contain highly structured elements, and evidence suggests that many of these structures are important for function.  Targeting these RNAs with small mols. offers opportunities to therapeutically modulate numerous cellular processes, including those linked to undruggable protein targets.  Despite this promise, there is currently only a single class of human-designed small mols. that target RNA used clin. - the linezolid antibiotics.  However, a growing no. of small-mol. RNA ligands are being identified, leading to burgeoning interest in the field.  Here, we discuss principles for discovering small-mol. drugs that target RNA and argue that the overarching challenge is to identify appropriate target structures - namely, in disease-causing RNAs that have high information content and, consequently, appropriate ligand-binding pockets.  If focus is placed on such druggable binding sites in RNA, extensive knowledge of the typical physicochem. properties of drug-like small mols. could then enable small-mol. drug discovery for RNA targets to become (only) roughly as difficult as for protein targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYl_OguWzasrVg90H21EOLACvtfcHk0ljwTQWswZaQDg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1OntLnL&md5=df01185ded43db426749b9c656a04f64</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2018.93&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2018.93%26sid%3Dliteratum%253Aachs%26aulast%3DWarner%26aufirst%3DK.%2BD.%26aulast%3DHajdin%26aufirst%3DC.%2BE.%26aulast%3DWeeks%26aufirst%3DK.%2BM.%26atitle%3DPrinciples%2520for%2520targeting%2520RNA%2520with%2520drug-like%2520small%2520molecules%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2018%26volume%3D17%26spage%3D547%26epage%3D558%26doi%3D10.1038%2Fnrd.2018.93" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Beermann, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piccoli, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Viereck, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thum, T.</span></span> <span> </span><span class="NLM_article-title">Non-coding RNAs in development and disease: Background, mechanism, and therapeutic approaches</span>. <i>Physiol. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>96</i></span>,  <span class="NLM_fpage">1297</span>â <span class="NLM_lpage">1325</span>, <span class="refDoi">Â DOI: 10.1152/physrev.00041.2015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1152%2Fphysrev.00041.2015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=27535639" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC1cXkvFGhtrc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=96&publication_year=2016&pages=1297-1325&author=J.+Beermannauthor=M.+T.+Piccoliauthor=J.+Viereckauthor=T.+Thum&title=Non-coding+RNAs+in+development+and+disease%3A+Background%2C+mechanism%2C+and+therapeutic+approaches&doi=10.1152%2Fphysrev.00041.2015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Non-coding RNAs in development and disease: background, mechanisms, and therapeutic approaches</span></div><div class="casAuthors">Beermann, Julia; Piccoli, Maria-Teresa; Viereck, Janika; Thum, Thomas</div><div class="citationInfo"><span class="NLM_cas:title">Physiological Reviews</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">96</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1297-1325</span>CODEN:
                <span class="NLM_cas:coden">PHREA7</span>;
        ISSN:<span class="NLM_cas:issn">0031-9333</span>.
    
            (<span class="NLM_cas:orgname">American Physiological Society</span>)
        </div><div class="casAbstract">Advances in RNA-sequencing techniques have led to the discovery of thousands of non-coding transcripts with unknown function.  There are several types of non-coding linearRNAs such as microRNAs (miRNA) and long non-coding RNAs (lncRNA), as well as circular RNAs(circRNA) consisting of a closed continuous loop.  This review guides the reader through important aspects of non-coding RNA biol.  This includes their biogenesis, mode of actions, physiologicalfunction, as well as their role in the disease context (such as in cancer or the cardiovascularsystem).  We specically focus on non-coding RNAs as potential therapeutic targets and diagnosticbiomarkers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpyehkZcx8nILVg90H21EOLACvtfcHk0ljwTQWswZaQDg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXkvFGhtrc%253D&md5=f9fe973309e572a492313ce98bbd4f6b</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1152%2Fphysrev.00041.2015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1152%252Fphysrev.00041.2015%26sid%3Dliteratum%253Aachs%26aulast%3DBeermann%26aufirst%3DJ.%26aulast%3DPiccoli%26aufirst%3DM.%2BT.%26aulast%3DViereck%26aufirst%3DJ.%26aulast%3DThum%26aufirst%3DT.%26atitle%3DNon-coding%2520RNAs%2520in%2520development%2520and%2520disease%253A%2520Background%252C%2520mechanism%252C%2520and%2520therapeutic%2520approaches%26jtitle%3DPhysiol.%2520Rev.%26date%3D2016%26volume%3D96%26spage%3D1297%26epage%3D1325%26doi%3D10.1152%2Fphysrev.00041.2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">TEP Consortium.</span> <span> </span><span class="NLM_article-title">An integrated encyclopedia
of DNA elements in the human genome.</span> <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume">489</span>,  <span class="NLM_fpage">57</span>â <span class="NLM_lpage">74</span> <span class="refDoi">Â DOI: 10.1038/nature11247</span> .</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1038%2Fnature11247" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=22955616" title="">PubMed</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=489&publication_year=2012&pages=57-74&author=TEP+Consortium.&title=An+integrated+encyclopedia%0Aof+DNA+elements+in+the+human+genome.&doi=10.1038%2Fnature11247"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnature11247&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature11247%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DAn%2520integrated%2520encyclopedia%250Aof%2520DNA%2520elements%2520in%2520the%2520human%2520genome.%26jtitle%3DNature%26date%3D2012%26volume%3D489%26spage%3D57%26epage%3D74%26doi%3D10.1038%2Fnature11247" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mongelli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martelli, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farsetti, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaetano, C.</span></span> <span> </span><span class="NLM_article-title">The dark that matters: Long non-coding RNAs as master regulators of cellular metabolism in non-communicable diseases</span>. <i>Front. Physiol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">369</span>â <span class="NLM_lpage">405</span>, <span class="refDoi">Â DOI: 10.3389/fphys.2019.00369</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.3389%2Ffphys.2019.00369" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=31191327" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A280%3ADC%252BB3M3lvFyntw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=369-405&author=A.+Mongelliauthor=F.+Martelliauthor=A.+Farsettiauthor=C.+Gaetano&title=The+dark+that+matters%3A+Long+non-coding+RNAs+as+master+regulators+of+cellular+metabolism+in+non-communicable+diseases&doi=10.3389%2Ffphys.2019.00369"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">The Dark That Matters: Long Non-coding RNAs as Master Regulators of Cellular Metabolism in Non-communicable Diseases</span></div><div class="casAuthors">Mongelli Alessia; Gaetano Carlo; Martelli Fabio; Farsetti Antonella</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in physiology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">369</span>
        ISSN:<span class="NLM_cas:issn">1664-042X</span>.
    </div><div class="casAbstract">Non-coding RNAs are pivotal for many cellular functions, such as splicing, gene regulation, chromosome structure, and hormone-like activity.  Here, we will report about the biology and the general molecular mechanisms associated with long non-coding RNAs (lncRNAs), a class of >200 nucleotides-long ribonucleic acid sequences, and their role in chronic non-transmissible diseases.  In particular, we will summarize knowledge about some of the best-characterized lncRNAs, such as H19 and MALAT1, and how they regulate carbohydrate and lipid metabolism as well as protein synthesis and degradation.  Evidence is discussed about how lncRNAs expression might affect cellular and organismal metabolism and whether their modulation could provide ground for the development of innovative treatments.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTi8XwXVd9vV1WTCLT8lX9SfW6udTcc2ebq9zYXteFtMLntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3M3lvFyntw%253D%253D&md5=d92f9d9c49667570046473d6d93ffb50</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.3389%2Ffphys.2019.00369&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffphys.2019.00369%26sid%3Dliteratum%253Aachs%26aulast%3DMongelli%26aufirst%3DA.%26aulast%3DMartelli%26aufirst%3DF.%26aulast%3DFarsetti%26aufirst%3DA.%26aulast%3DGaetano%26aufirst%3DC.%26atitle%3DThe%2520dark%2520that%2520matters%253A%2520Long%2520non-coding%2520RNAs%2520as%2520master%2520regulators%2520of%2520cellular%2520metabolism%2520in%2520non-communicable%2520diseases%26jtitle%3DFront.%2520Physiol.%26date%3D2019%26volume%3D10%26spage%3D369%26epage%3D405%26doi%3D10.3389%2Ffphys.2019.00369" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ling, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fabbri, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calin, G. A.</span></span> <span> </span><span class="NLM_article-title">MicroRNAs and other non-coding RNAs as targets for anticancer drug development</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">847</span>â <span class="NLM_lpage">865</span>, <span class="refDoi">Â DOI: 10.1038/nrd4140</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1038%2Fnrd4140" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=24172333" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs12ks73I" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2013&pages=847-865&author=H.+Lingauthor=M.+Fabbriauthor=G.+A.+Calin&title=MicroRNAs+and+other+non-coding+RNAs+as+targets+for+anticancer+drug+development&doi=10.1038%2Fnrd4140"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">MicroRNAs and other non-coding RNAs as targets for anticancer drug development</span></div><div class="casAuthors">Ling, Hui; Fabbri, Muller; Calin, George A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">847-865</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The first cancer-targeted microRNA (miRNA) drug - MRX34, a liposome-based miR-34 mimic - entered Phase I clin. trials in patients with advanced hepatocellular carcinoma in Apr. 2013, and miRNA therapeutics are attracting special attention from both academia and biotechnol. companies.  Although miRNAs are the most studied non-coding RNAs (ncRNAs) to date, the importance of long non-coding RNAs (lncRNAs) is increasingly being recognized.  Here, we summarize the roles of miRNAs and lncRNAs in cancer, with a focus on the recently identified novel mechanisms of action, and discuss the current strategies in designing ncRNA-targeting therapeutics, as well as the assocd. challenges.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7sEXRlR4FNLVg90H21EOLACvtfcHk0lhgpAXt20NdUw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs12ks73I&md5=5bb9c58f97adaa031aea14033150e383</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fnrd4140&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4140%26sid%3Dliteratum%253Aachs%26aulast%3DLing%26aufirst%3DH.%26aulast%3DFabbri%26aufirst%3DM.%26aulast%3DCalin%26aufirst%3DG.%2BA.%26atitle%3DMicroRNAs%2520and%2520other%2520non-coding%2520RNAs%2520as%2520targets%2520for%2520anticancer%2520drug%2520development%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2013%26volume%3D12%26spage%3D847%26epage%3D865%26doi%3D10.1038%2Fnrd4140" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mattick, J. S.</span></span> <span> </span><span class="NLM_article-title">The genetic signatures of noncoding RNAs</span>. <i>PLoS Genet.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>5</i></span>, <span class="NLM_elocation-id">e1000459</span> <span class="refDoi">Â DOI: 10.1371/journal.pgen.1000459</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1371%2Fjournal.pgen.1000459" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=19390609" title="">PubMed</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2009&author=J.+S.+Mattick&title=The+genetic+signatures+of+noncoding+RNAs&doi=10.1371%2Fjournal.pgen.1000459"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pgen.1000459&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pgen.1000459%26sid%3Dliteratum%253Aachs%26aulast%3DMattick%26aufirst%3DJ.%2BS.%26atitle%3DThe%2520genetic%2520signatures%2520of%2520noncoding%2520RNAs%26jtitle%3DPLoS%2520Genet.%26date%3D2009%26volume%3D5%26doi%3D10.1371%2Fjournal.pgen.1000459" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bartel, D. P.</span></span> <span> </span><span class="NLM_article-title">MicroRNAs: target recognition and regulatory functions</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>136</i></span>,  <span class="NLM_fpage">215</span>â <span class="NLM_lpage">233</span>, <span class="refDoi">Â DOI: 10.1016/j.cell.2009.01.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1016%2Fj.cell.2009.01.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=19167326" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhs1Kiuro%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=136&publication_year=2009&pages=215-233&author=D.+P.+Bartel&title=MicroRNAs%3A+target+recognition+and+regulatory+functions&doi=10.1016%2Fj.cell.2009.01.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">MicroRNAs: target recognition and regulatory functions</span></div><div class="casAuthors">Bartel, David P.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">136</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">215-233</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  MicroRNAs (miRNAs) are endogenous â¼23 nt RNAs that play important gene-regulatory roles in animals and plants by pairing to the mRNAs of protein-coding genes to direct their posttranscriptional repression.  This review outlines the current understanding of miRNA target recognition in animals and discusses the widespread impact of miRNAs on both the expression and evolution of protein-coding genes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodHBQuLKSQY7Vg90H21EOLACvtfcHk0lhgpAXt20NdUw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhs1Kiuro%253D&md5=8d9c406309fb79682363a27d265a0674</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2009.01.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2009.01.002%26sid%3Dliteratum%253Aachs%26aulast%3DBartel%26aufirst%3DD.%2BP.%26atitle%3DMicroRNAs%253A%2520target%2520recognition%2520and%2520regulatory%2520functions%26jtitle%3DCell%26date%3D2009%26volume%3D136%26spage%3D215%26epage%3D233%26doi%3D10.1016%2Fj.cell.2009.01.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Djebali, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merkel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dobin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lassmann, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mortazavi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanzer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lagarde, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlesinger, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xue, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marinov, G. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khatun, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zaleski, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rozowsky, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">RÃ¶der, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kokocinsk, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abdelhamid, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alioto, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Antoshechkin, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baer, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bar, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Batut, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bell, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bell, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chakrabortty, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chrast, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curado, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Derrien, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drenkow, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dumais, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dumais, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duttagupta, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Falconnet, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fastuca, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fejes-Toth, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferreira, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foissac, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fullwood, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gonzalez, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gunawardena, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howald, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jha, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapranov, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">King, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kingswood, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, O. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Persaud, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Preall, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ribeca, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Risk, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robyr, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sammeth, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaffer, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">See, L.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shahab, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skancke, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suzuki, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takahashi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tilgner, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trout, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walters, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wrobel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruan, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayashizaki, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harrow, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerstein, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hubbard, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reymond, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Antonarakis, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hannon, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giddings, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wold, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carninci, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">GuigÃ³, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gingeras, T. R.</span></span> <span> </span><span class="NLM_article-title">Landscape of transcription in human cells</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>489</i></span>,  <span class="NLM_fpage">101</span>â <span class="NLM_lpage">108</span>, <span class="refDoi">Â DOI: 10.1038/nature11233</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1038%2Fnature11233" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=22955620" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtlGnt73M" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=489&publication_year=2012&pages=101-108&author=S.+Djebaliauthor=C.+A.+Davisauthor=A.+Merkelauthor=A.+Dobinauthor=T.+Lassmannauthor=A.+Mortazaviauthor=A.+Tanzerauthor=J.+Lagardeauthor=W.+Linauthor=F.+Schlesingerauthor=C.+Xueauthor=G.+K.+Marinovauthor=J.+Khatunauthor=B.+A.+Williamsauthor=C.+Zaleskiauthor=J.+Rozowskyauthor=M.+R%C3%B6derauthor=F.+Kokocinskauthor=R.+F.+Abdelhamidauthor=T.+Aliotoauthor=I.+Antoshechkinauthor=M.+T.+Baerauthor=N.+S.+Barauthor=P.+Batutauthor=K.+Bellauthor=I.+Bellauthor=S.+Chakraborttyauthor=X.+Chenauthor=J.+Chrastauthor=J.+Curadoauthor=T.+Derrienauthor=J.+Drenkowauthor=E.+Dumaisauthor=J.+Dumaisauthor=R.+Duttaguptaauthor=E.+Falconnetauthor=M.+Fastucaauthor=K.+Fejes-Tothauthor=P.+Ferreiraauthor=S.+Foissacauthor=M.+J.+Fullwoodauthor=H.+Gaoauthor=D.+Gonzalezauthor=A.+Gordonauthor=H.+Gunawardenaauthor=C.+Howaldauthor=S.+Jhaauthor=R.+Johnsonauthor=P.+Kapranovauthor=B.+Kingauthor=C.+Kingswoodauthor=O.+J.+Luoauthor=E.+Parkauthor=K.+Persaudauthor=J.+B.+Preallauthor=P.+Ribecaauthor=B.+Riskauthor=D.+Robyrauthor=M.+Sammethauthor=L.+Schafferauthor=L.-H.+Seeauthor=A.+Shahabauthor=J.+Skanckeauthor=A.+M.+Suzukiauthor=H.+Takahashiauthor=H.+Tilgnerauthor=D.+Troutauthor=N.+Waltersauthor=H.+Wangauthor=J.+Wrobelauthor=Y.+Yuauthor=X.+Ruanauthor=Y.+Hayashizakiauthor=J.+Harrowauthor=M.+Gersteinauthor=T.+Hubbardauthor=A.+Reymondauthor=S.+E.+Antonarakisauthor=G.+Hannonauthor=M.+C.+Giddingsauthor=Y.+Ruanauthor=B.+Woldauthor=P.+Carninciauthor=R.+Guig%C3%B3author=T.+R.+Gingeras&title=Landscape+of+transcription+in+human+cells&doi=10.1038%2Fnature11233"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Landscape of transcription in human cells</span></div><div class="casAuthors">Djebali, Sarah; Davis, Carrie A.; Merkel, Angelika; Dobin, Alex; Lassmann, Timo; Mortazavi, Ali; Tanzer, Andrea; Lagarde, Julien; Lin, Wei; Schlesinger, Felix; Xue, Chenghai; Marinov, Georgi K.; Khatun, Jainab; Williams, Brian A.; Zaleski, Chris; Rozowsky, Joel; Roeder, Maik; Kokocinski, Felix; Abdelhamid, Rehab F.; Alioto, Tyler; Antoshechkin, Igor; Baer, Michael T.; Bar, Nadav S.; Batut, Philippe; Bell, Kimberly; Bell, Ian; Chakrabortty, Sudipto; Chen, Xian; Chrast, Jacqueline; Curado, Joao; Derrien, Thomas; Drenkow, Jorg; Dumais, Erica; Dumais, Jacqueline; Duttagupta, Radha; Falconnet, Emilie; Fastuca, Meagan; Fejes-Toth, Kata; Ferreira, Pedro; Foissac, Sylvain; Fullwood, Melissa J.; Gao, Hui; Gonzalez, David; Gordon, Assaf; Gunawardena, Harsha; Howald, Cedric; Jha, Sonali; Johnson, Rory; Kapranov, Philipp; King, Brandon; Kingswood, Colin; Luo, Oscar J.; Park, Eddie; Persaud, Kimberly; Preall, Jonathan B.; Ribeca, Paolo; Risk, Brian; Robyr, Daniel; Sammeth, Michael; Schaffer, Lorian; See, Lei-Hoon; Shahab, Atif; Skancke, Jorgen; Suzuki, Ana Maria; Takahashi, Hazuki; Tilgner, Hagen; Trout, Diane; Walters, Nathalie; Wang, Huaien; Wrobel, John; Yu, Yanbao; Ruan, Xiaoan; Hayashizaki, Yoshihide; Harrow, Jennifer; Gerstein, Mark; Hubbard, Tim; Reymond, Alexandre; Antonarakis, Stylianos E.; Hannon, Gregory; Giddings, Morgan C.; Ruan, Yijun; Wold, Barbara; Carninci, Piero; Guigo, Roderic; Gingeras, Thomas R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">489</span>
        (<span class="NLM_cas:issue">7414</span>),
    <span class="NLM_cas:pages">101-108</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Eukaryotic cells make many types of primary and processed RNAs that are found either in specific subcellular compartments or throughout the cells.  A complete catalog of these RNAs is not yet available and their characteristic subcellular localizations are also poorly understood.  Because RNA represents the direct output of the genetic information encoded by genomes and a significant proportion of a cell's regulatory capabilities are focused on its synthesis, processing, transport, modification and translation, the generation of such a catalog is crucial for understanding genome function.  Here we report evidence that three-quarters of the human genome is capable of being transcribed, as well as observations about the range and levels of expression, localization, processing fates, regulatory regions and modifications of almost all currently annotated and thousands of previously unannotated RNAs.  These observations, taken together, prompt a redefinition of the concept of a gene.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoQJgBaBiE8o7Vg90H21EOLACvtfcHk0li8E4HVHaToZw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtlGnt73M&md5=8f8ee2a1189ced1010dec5e45a305acf</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2Fnature11233&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature11233%26sid%3Dliteratum%253Aachs%26aulast%3DDjebali%26aufirst%3DS.%26aulast%3DDavis%26aufirst%3DC.%2BA.%26aulast%3DMerkel%26aufirst%3DA.%26aulast%3DDobin%26aufirst%3DA.%26aulast%3DLassmann%26aufirst%3DT.%26aulast%3DMortazavi%26aufirst%3DA.%26aulast%3DTanzer%26aufirst%3DA.%26aulast%3DLagarde%26aufirst%3DJ.%26aulast%3DLin%26aufirst%3DW.%26aulast%3DSchlesinger%26aufirst%3DF.%26aulast%3DXue%26aufirst%3DC.%26aulast%3DMarinov%26aufirst%3DG.%2BK.%26aulast%3DKhatun%26aufirst%3DJ.%26aulast%3DWilliams%26aufirst%3DB.%2BA.%26aulast%3DZaleski%26aufirst%3DC.%26aulast%3DRozowsky%26aufirst%3DJ.%26aulast%3DR%25C3%25B6der%26aufirst%3DM.%26aulast%3DKokocinsk%26aufirst%3DF.%26aulast%3DAbdelhamid%26aufirst%3DR.%2BF.%26aulast%3DAlioto%26aufirst%3DT.%26aulast%3DAntoshechkin%26aufirst%3DI.%26aulast%3DBaer%26aufirst%3DM.%2BT.%26aulast%3DBar%26aufirst%3DN.%2BS.%26aulast%3DBatut%26aufirst%3DP.%26aulast%3DBell%26aufirst%3DK.%26aulast%3DBell%26aufirst%3DI.%26aulast%3DChakrabortty%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DChrast%26aufirst%3DJ.%26aulast%3DCurado%26aufirst%3DJ.%26aulast%3DDerrien%26aufirst%3DT.%26aulast%3DDrenkow%26aufirst%3DJ.%26aulast%3DDumais%26aufirst%3DE.%26aulast%3DDumais%26aufirst%3DJ.%26aulast%3DDuttagupta%26aufirst%3DR.%26aulast%3DFalconnet%26aufirst%3DE.%26aulast%3DFastuca%26aufirst%3DM.%26aulast%3DFejes-Toth%26aufirst%3DK.%26aulast%3DFerreira%26aufirst%3DP.%26aulast%3DFoissac%26aufirst%3DS.%26aulast%3DFullwood%26aufirst%3DM.%2BJ.%26aulast%3DGao%26aufirst%3DH.%26aulast%3DGonzalez%26aufirst%3DD.%26aulast%3DGordon%26aufirst%3DA.%26aulast%3DGunawardena%26aufirst%3DH.%26aulast%3DHowald%26aufirst%3DC.%26aulast%3DJha%26aufirst%3DS.%26aulast%3DJohnson%26aufirst%3DR.%26aulast%3DKapranov%26aufirst%3DP.%26aulast%3DKing%26aufirst%3DB.%26aulast%3DKingswood%26aufirst%3DC.%26aulast%3DLuo%26aufirst%3DO.%2BJ.%26aulast%3DPark%26aufirst%3DE.%26aulast%3DPersaud%26aufirst%3DK.%26aulast%3DPreall%26aufirst%3DJ.%2BB.%26aulast%3DRibeca%26aufirst%3DP.%26aulast%3DRisk%26aufirst%3DB.%26aulast%3DRobyr%26aufirst%3DD.%26aulast%3DSammeth%26aufirst%3DM.%26aulast%3DSchaffer%26aufirst%3DL.%26aulast%3DSee%26aufirst%3DL.-H.%26aulast%3DShahab%26aufirst%3DA.%26aulast%3DSkancke%26aufirst%3DJ.%26aulast%3DSuzuki%26aufirst%3DA.%2BM.%26aulast%3DTakahashi%26aufirst%3DH.%26aulast%3DTilgner%26aufirst%3DH.%26aulast%3DTrout%26aufirst%3DD.%26aulast%3DWalters%26aufirst%3DN.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DWrobel%26aufirst%3DJ.%26aulast%3DYu%26aufirst%3DY.%26aulast%3DRuan%26aufirst%3DX.%26aulast%3DHayashizaki%26aufirst%3DY.%26aulast%3DHarrow%26aufirst%3DJ.%26aulast%3DGerstein%26aufirst%3DM.%26aulast%3DHubbard%26aufirst%3DT.%26aulast%3DReymond%26aufirst%3DA.%26aulast%3DAntonarakis%26aufirst%3DS.%2BE.%26aulast%3DHannon%26aufirst%3DG.%26aulast%3DGiddings%26aufirst%3DM.%2BC.%26aulast%3DRuan%26aufirst%3DY.%26aulast%3DWold%26aufirst%3DB.%26aulast%3DCarninci%26aufirst%3DP.%26aulast%3DGuig%25C3%25B3%26aufirst%3DR.%26aulast%3DGingeras%26aufirst%3DT.%2BR.%26atitle%3DLandscape%2520of%2520transcription%2520in%2520human%2520cells%26jtitle%3DNature%26date%3D2012%26volume%3D489%26spage%3D101%26epage%3D108%26doi%3D10.1038%2Fnature11233" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Walder, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patten, W. J. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ritchie, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montange, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, T. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woodside, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perkins, T. T.</span></span> <span> </span><span class="NLM_article-title">High-Precision single-molecule characterization of the folding of an HIV RNA hairpin by atomic force microscopy</span>. <i>Nano Lett.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">6318</span>â <span class="NLM_lpage">6325</span>, <span class="refDoi">Â DOI: 10.1021/acs.nanolett.8b02597</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.nanolett.8b02597" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslGqsLvE" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2018&pages=6318-6325&author=R.+Walderauthor=W.+J.+V.+Pattenauthor=D.+B.+Ritchieauthor=R.+K.+Montangeauthor=T.+W.+Millerauthor=M.+T.+Woodsideauthor=T.+T.+Perkins&title=High-Precision+single-molecule+characterization+of+the+folding+of+an+HIV+RNA+hairpin+by+atomic+force+microscopy&doi=10.1021%2Facs.nanolett.8b02597"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">High-Precision Single-Molecule Characterization of the Folding of an HIV RNA Hairpin by Atomic Force Microscopy</span></div><div class="casAuthors">Walder, Robert; Van Patten, William J.; Ritchie, Dustin B.; Montange, Rebecca K.; Miller, Ty W.; Woodside, Michael T.; Perkins, Thomas T.</div><div class="citationInfo"><span class="NLM_cas:title">Nano Letters</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">6318-6325</span>CODEN:
                <span class="NLM_cas:coden">NALEFD</span>;
        ISSN:<span class="NLM_cas:issn">1530-6984</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The folding of RNA into a wide range of structures is essential for its diverse biol. functions from enzymic catalysis to ligand binding and gene regulation.  The unfolding and refolding of individual RNA mols. can be probed by single-mol. force spectroscopy (SMFS), enabling detailed characterization of the conformational dynamics of the mol. as well as the free-energy landscape underlying folding.  Historically, high-precision SMFS studies of RNA have been limited to custom-built optical traps.  Although com. at. force microscopes (AFMs) are widely deployed and offer significant advantages in ease-of-use over custom-built optical traps, traditional AFM-based SMFS lacks the sensitivity and stability to characterize individual RNA mols. precisely.  Here, the authors developed a high-precision SMFS assay to study RNA folding using a com. AFM and applied it to characterize a small RNA hairpin from HIV that plays a key role in stimulating programmed ribosomal frameshifting.  The authors achieved rapid data acquisition in a dynamic assay, unfolding and then refolding the same individual hairpin more than 1100 times in 15 min.  In comparison to measurements using optical traps, the AFM-based assay featured a stiffer force probe and a less compliant construct, providing a complementary measurement regime that dramatically accelerated equil. folding dynamics.  Not only did kinetic anal. of equil. trajectories of the HIV RNA hairpin yield the traditional parameters used to characterize folding by SMFS (zero-force rate consts. and distances to the transition state), but the authors also reconstructed the full 1D projection of the folding free-energy landscape comparable to state-of-the-art studies using dual-beam optical traps, a first for this RNA hairpin and AFM studies of nucleic acids in general.  Looking forward, the authors anticipate that the ease-of-use of the high-precision assay implemented on a com. AFM will accelerate studying folding of diverse nucleic acid structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoaueT2gVqbirVg90H21EOLACvtfcHk0ljwlFjgVJkc2w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslGqsLvE&md5=8bf17b525c2a9c52d4489737dfa3004b</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Facs.nanolett.8b02597&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.nanolett.8b02597%26sid%3Dliteratum%253Aachs%26aulast%3DWalder%26aufirst%3DR.%26aulast%3DPatten%26aufirst%3DW.%2BJ.%2BV.%26aulast%3DRitchie%26aufirst%3DD.%2BB.%26aulast%3DMontange%26aufirst%3DR.%2BK.%26aulast%3DMiller%26aufirst%3DT.%2BW.%26aulast%3DWoodside%26aufirst%3DM.%2BT.%26aulast%3DPerkins%26aufirst%3DT.%2BT.%26atitle%3DHigh-Precision%2520single-molecule%2520characterization%2520of%2520the%2520folding%2520of%2520an%2520HIV%2520RNA%2520hairpin%2520by%2520atomic%2520force%2520microscopy%26jtitle%3DNano%2520Lett.%26date%3D2018%26volume%3D18%26spage%3D6318%26epage%3D6325%26doi%3D10.1021%2Facs.nanolett.8b02597" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sztuba-Solinska, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chavez-Calvillo, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cline, S. E.</span></span> <span> </span><span class="NLM_article-title">Unveiling the druggable RNA targets and small molecule therapeutics</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">2149</span>â <span class="NLM_lpage">2165</span>, <span class="refDoi">Â DOI: 10.1016/j.bmc.2019.03.057</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1016%2Fj.bmc.2019.03.057" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=30981606" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC1MXntFCiu74%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2019&pages=2149-2165&author=J.+Sztuba-Solinskaauthor=G.+Chavez-Calvilloauthor=S.+E.+Cline&title=Unveiling+the+druggable+RNA+targets+and+small+molecule+therapeutics&doi=10.1016%2Fj.bmc.2019.03.057"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Unveiling the druggable RNA targets and small molecule therapeutics</span></div><div class="casAuthors">Sztuba-Solinska, Joanna; Chavez-Calvillo, Gabriela; Cline, Sabrina Elizabeth</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2149-2165</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The increasing appreciation for the crucial roles of RNAs in infectious and non-infectious human diseases makes them attractive therapeutic targets.  Coding and non-coding RNAs frequently fold into complex conformations which, if effectively targeted, offer opportunities to therapeutically modulate numerous cellular processes, including those linked to undruggable protein targets.  Despite the considerable skepticism as to whether RNAs can be targeted with small mol. therapeutics, overwhelming evidence suggests the challenges we are currently facing are not outside the realm of possibility.  In this review, we highlight the most recent advances in mol. techniques that have sparked a revolution in understanding the RNA structure-to-function relationship.  We bring attention to the application of these modern techniques to identify druggable RNA targets and to assess small mol. binding specificity.  Finally, we discuss novel screening methodologies that support RNA drug discovery and present examples of therapeutically valuable RNA targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJ_ZttV3TINLVg90H21EOLACvtfcHk0ljwlFjgVJkc2w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXntFCiu74%253D&md5=5c2197acf2f0a04f6fda7711fb3cc0a9</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2019.03.057&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2019.03.057%26sid%3Dliteratum%253Aachs%26aulast%3DSztuba-Solinska%26aufirst%3DJ.%26aulast%3DChavez-Calvillo%26aufirst%3DG.%26aulast%3DCline%26aufirst%3DS.%2BE.%26atitle%3DUnveiling%2520the%2520druggable%2520RNA%2520targets%2520and%2520small%2520molecule%2520therapeutics%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2019%26volume%3D27%26spage%3D2149%26epage%3D2165%26doi%3D10.1016%2Fj.bmc.2019.03.057" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kligun, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mandel-Gutfreund, Y.</span></span> <span> </span><span class="NLM_article-title">Conformational readout of RNA by small ligands</span>. <i>RNA Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">981</span>â <span class="NLM_lpage">989</span>, <span class="refDoi">Â DOI: 10.4161/rna.24682</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.4161%2Frna.24682" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjsVWl" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2013&pages=981-989&author=E.+Kligunauthor=Y.+Mandel-Gutfreund&title=Conformational+readout+of+RNA+by+small+ligands&doi=10.4161%2Frna.24682"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Conformational readout of RNA by small ligands</span></div><div class="casAuthors">Kligun, Efrat; Mandel-Gutfreund, Yael</div><div class="citationInfo"><span class="NLM_cas:title">RNA Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">981-989</span>CODEN:
                <span class="NLM_cas:coden">RBNIBE</span>;
        ISSN:<span class="NLM_cas:issn">1547-6286</span>.
    
            (<span class="NLM_cas:orgname">Landes Bioscience</span>)
        </div><div class="casAbstract">RNA mols. have highly versatile structures that can fold into myriad conformations, providing many potential pockets for binding small mols.  The increasing no. of available RNA structures, in complex with proteins, small ligands and in free form, enables the design of new therapeutically useful RNA-binding ligands.  Here we studied RNA ligand complexes from 10 RNA groups extd. from the protein data bank (PDB), including adaptive and non-adaptive complexes.  We analyzed the chem., phys., structural and conformational properties of binding pockets around the ligand.  Comparing the properties of ligand-binding pockets to the properties of computed pockets extd. from all available RNA structures and RNA-protein interfaces, revealed that ligand-binding pockets, mainly the adaptive pockets, are characterized by unique properties, specifically enriched in rare conformations of the nucleobase and the sugar pucker.  Further, we demonstrate that nucleotides possessing the rare conformations are preferentially involved in direct interactions with the ligand.  Overall, based on our comprehensive anal. of RNA-ligand complexes, we suggest that the unique conformations adopted by RNA nucleotides play an important role in RNA recognition by small ligands.  We term the recognition of a binding site by a ligand via the unique RNA conformations "RNA conformational readout.".  We propose that "conformational readout" is a general way by which RNA binding pockets are recognized and selected from an ensemble of different RNA states.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8RXanE2P-37Vg90H21EOLACvtfcHk0ljwlFjgVJkc2w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjsVWl&md5=0e6063ef9c5e97a55113b478dedd099b</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.4161%2Frna.24682&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252Frna.24682%26sid%3Dliteratum%253Aachs%26aulast%3DKligun%26aufirst%3DE.%26aulast%3DMandel-Gutfreund%26aufirst%3DY.%26atitle%3DConformational%2520readout%2520of%2520RNA%2520by%2520small%2520ligands%26jtitle%3DRNA%2520Biol.%26date%3D2013%26volume%3D10%26spage%3D981%26epage%3D989%26doi%3D10.4161%2Frna.24682" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kondo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Westhof, E.</span></span> <span> </span><span class="NLM_article-title">Base pairs and pseudo pairs observed in RNA-ligand complexes</span>. <i>J. Mol. Recognit.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">241</span>â <span class="NLM_lpage">252</span>, <span class="refDoi">Â DOI: 10.1002/jmr.978</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1002%2Fjmr.978" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2009&pages=241-252&author=J.+Kondoauthor=E.+Westhof&title=Base+pairs+and+pseudo+pairs+observed+in+RNA-ligand+complexes&doi=10.1002%2Fjmr.978"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1002%2Fjmr.978&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjmr.978%26sid%3Dliteratum%253Aachs%26aulast%3DKondo%26aufirst%3DJ.%26aulast%3DWesthof%26aufirst%3DE.%26atitle%3DBase%2520pairs%2520and%2520pseudo%2520pairs%2520observed%2520in%2520RNA-ligand%2520complexes%26jtitle%3DJ.%2520Mol.%2520Recognit.%26date%3D2009%26volume%3D23%26spage%3D241%26epage%3D252%26doi%3D10.1002%2Fjmr.978" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Angelbello, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Childs-Disney, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span> <span> </span><span class="NLM_article-title">Using genome sequence to enable the design of medicines and chemical probes</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>118</i></span>,  <span class="NLM_fpage">1599</span>â <span class="NLM_lpage">1663</span>, <span class="refDoi">Â DOI: 10.1021/acs.chemrev.7b00504</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.7b00504" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC1cXlvFGqsQ%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=118&publication_year=2018&pages=1599-1663&author=A.+J.+Angelbelloauthor=J.+L.+Chenauthor=J.+L.+Childs-Disneyauthor=P.+Zhangauthor=Z.+F.+Wangauthor=M.+D.+Disney&title=Using+genome+sequence+to+enable+the+design+of+medicines+and+chemical+probes&doi=10.1021%2Facs.chemrev.7b00504"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Using Genome Sequence to Enable the Design of Medicines and Chemical Probes</span></div><div class="casAuthors">Angelbello, Alicia J.; Chen, Jonathan L.; Childs-Disney, Jessica L.; Zhang, Peiyuan; Wang, Zi-Fu; Disney, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">118</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1599-1663</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Rapid progress in genome sequencing technol. has put us firmly into a postgenomic era.  A key challenge in biomedical research is harnessing genome sequence to fulfill the promise of personalized medicine.  This Review describes how genome sequencing has enabled the identification of disease-causing biomols. and how these data have been converted into chem. probes of function, preclin. lead modalities, and ultimately U.S.  Food and Drug Administration (FDA)-approved drugs.  In particular, the authors focus on the use of oligonucleotide-based modalities to target disease-causing RNAs; small mols. that target DNA, RNA, or protein; the rational repurposing of known therapeutic modalities; and the advantages of pharmacogenetics.  Lastly, the authors discuss the remaining challenges and opportunities in the direct utilization of genome sequence to enable design of medicines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreyX8o6MEWwLVg90H21EOLACvtfcHk0li5rONtBlFxEg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXlvFGqsQ%253D%253D&md5=6cf4911da55d4304d7c818f0f7a4a6a0</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.7b00504&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.7b00504%26sid%3Dliteratum%253Aachs%26aulast%3DAngelbello%26aufirst%3DA.%2BJ.%26aulast%3DChen%26aufirst%3DJ.%2BL.%26aulast%3DChilds-Disney%26aufirst%3DJ.%2BL.%26aulast%3DZhang%26aufirst%3DP.%26aulast%3DWang%26aufirst%3DZ.%2BF.%26aulast%3DDisney%26aufirst%3DM.%2BD.%26atitle%3DUsing%2520genome%2520sequence%2520to%2520enable%2520the%2520design%2520of%2520medicines%2520and%2520chemical%2520probes%26jtitle%3DChem.%2520Rev.%26date%3D2018%26volume%3D118%26spage%3D1599%26epage%3D1663%26doi%3D10.1021%2Facs.chemrev.7b00504" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Di
Giorgio, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duca, M.</span></span> <span> </span><span class="NLM_article-title">Synthetic small-molecule RNA ligands: future prospects as therapeutic agents</span>. <i>MedChemComm</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1242</span>â <span class="NLM_lpage">1255</span>, <span class="refDoi">Â DOI: 10.1039/C9MD00195F</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1039%2FC9MD00195F" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=31534649" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC1MXos1ShurY%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=1242-1255&author=A.+Di%0AGiorgioauthor=M.+Duca&title=Synthetic+small-molecule+RNA+ligands%3A+future+prospects+as+therapeutic+agents&doi=10.1039%2FC9MD00195F"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Synthetic small-molecule RNA ligands: future prospects as therapeutic agents</span></div><div class="casAuthors">Di Giorgio, A.; Duca, M.</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1242-1255</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">RNA is one of the most intriguing and promising biol. targets for the discovery of innovative drugs in many pathologies and various biol. relevant RNAs that could serve as drug targets have already been identified.  Among the most important ones, one can mention prokaryotic rRNA which is the target of several marketed antibiotics, viral RNAs or oncogenic microRNAs that are tightly involved in the development and progression of various cancers.  Oligonucleotides are efficient and specific RNA targeting agents but suffer from poor pharmacodynamic and pharmacokinetic properties.  For this reason, a no. of synthetic small-mol. ligands have been identified and studied upon screening of chem. libraries or focused design of RNA binders.  In this review, we report the most relevant examples of synthetic compds. bearing sufficient selectivity to envisage clin. studies and future therapeutic applications with a particular attention for the main strategies that can be undertaken toward the improvement of selectivity and biol. activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-wwNyCsBkOLVg90H21EOLACvtfcHk0li5rONtBlFxEg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXos1ShurY%253D&md5=60f47cc3df704517d5c308ebbe2cae6b</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1039%2FC9MD00195F&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC9MD00195F%26sid%3Dliteratum%253Aachs%26aulast%3DDi%2BGiorgio%26aufirst%3DA.%26aulast%3DDuca%26aufirst%3DM.%26atitle%3DSynthetic%2520small-molecule%2520RNA%2520ligands%253A%2520future%2520prospects%2520as%2520therapeutic%2520agents%26jtitle%3DMedChemComm%26date%3D2019%26volume%3D10%26spage%3D1242%26epage%3D1255%26doi%3D10.1039%2FC9MD00195F" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Connelly, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moon, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneekloth, J. S.</span></span> <span> </span><span class="NLM_article-title">The emerging role of RNA as a therapeutic target for small molecules</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">1077</span>â <span class="NLM_lpage">1090</span>, <span class="refDoi">Â DOI: 10.1016/j.chembiol.2016.05.021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1016%2Fj.chembiol.2016.05.021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=27593111" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVOqsL3E" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2016&pages=1077-1090&author=C.+M.+Connellyauthor=M.+H.+Moonauthor=J.+S.+Schneekloth&title=The+emerging+role+of+RNA+as+a+therapeutic+target+for+small+molecules&doi=10.1016%2Fj.chembiol.2016.05.021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">The Emerging Role of RNA as a Therapeutic Target for Small Molecules</span></div><div class="casAuthors">Connelly, Colleen M.; Moon, Michelle H.; Schneekloth, John S., Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1077-1090</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Recent advances in understanding different RNAs and unique features of their biol. have revealed a wealth of information.  However, approaches to identify small mols. that target these newly discovered regulatory elements have been lacking.  The application of new biochem. screening and design-based technologies, coupled with a resurgence of interest in phenotypic screening, has resulted in several compelling successes in targeting RNA.  A no. of recent advances suggest that achieving the long-standing goal of developing drug-like, biol. active small mols. that target RNA is possible.  This review highlights advances and successes in approaches to targeting RNA with diverse small mols., and the potential for these technologies to pave the way to new types of RNA-targeted therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqdPbbvOw7jO7Vg90H21EOLACvtfcHk0li5rONtBlFxEg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVOqsL3E&md5=800ffb1879c29aa7ac34940e761b5f7e</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2016.05.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2016.05.021%26sid%3Dliteratum%253Aachs%26aulast%3DConnelly%26aufirst%3DC.%2BM.%26aulast%3DMoon%26aufirst%3DM.%2BH.%26aulast%3DSchneekloth%26aufirst%3DJ.%2BS.%26atitle%3DThe%2520emerging%2520role%2520of%2520RNA%2520as%2520a%2520therapeutic%2520target%2520for%2520small%2520molecules%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2016%26volume%3D23%26spage%3D1077%26epage%3D1090%26doi%3D10.1016%2Fj.chembiol.2016.05.021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Aboul-Ela, F.</span></span> <span> </span><span class="NLM_article-title">Strategies for the design of RNA-binding small molecules</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">93</span>â <span class="NLM_lpage">119</span>, <span class="refDoi">Â DOI: 10.4155/fmc.09.149</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.4155%2Ffmc.09.149" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=21426048" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC3cXosVWjtg%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2010&pages=93-119&author=F.+Aboul-Ela&title=Strategies+for+the+design+of+RNA-binding+small+molecules&doi=10.4155%2Ffmc.09.149"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Strategies for the design of RNA-binding small molecules</span></div><div class="casAuthors">Aboul-Ela, Fareed</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">93-119</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">A review.  Bacterial rRNA is the target of clin. important antibiotics, while biol. important RNAs in viral and eukaryotic genomes present a range of potential drug targets.  The physicochem. properties of RNA present difficulties for medicinal chem., particularly when oral availability is needed.  Peptidic ligands and anal. of their RNA-binding properties are providing insight into RNA recognition.  RNA-binding ligands include far more chem. classes than just aminoglycosides.  Chem. functionalities from known RNA-binding small mols. are being exploited in fragment- and ligand-based projects.  While targeting of RNA for drug design is very challenging, continuing advances in our understanding of the principles of RNA-ligand interaction will be necessary to realize the full potential of this class of targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWWGVYo9p2_bVg90H21EOLACvtfcHk0ljzcSC73PrHrQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXosVWjtg%253D%253D&md5=18b891dc9e0af5f84b7a2e621622d52a</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.4155%2Ffmc.09.149&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc.09.149%26sid%3Dliteratum%253Aachs%26aulast%3DAboul-Ela%26aufirst%3DF.%26atitle%3DStrategies%2520for%2520the%2520design%2520of%2520RNA-binding%2520small%2520molecules%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2010%26volume%3D2%26spage%3D93%26epage%3D119%26doi%3D10.4155%2Ffmc.09.149" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Costales, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Childs-Disney, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haniff, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span> <span> </span><span class="NLM_article-title">How we think about targeting RNA with small molecules</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">8880</span>â <span class="NLM_lpage">8900</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.9b01927</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01927" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BB3cXlsFygsr8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=8880-8900&author=M.+G.+Costalesauthor=J.+L.+Childs-Disneyauthor=H.+S.+Haniffauthor=M.+D.+Disney&title=How+we+think+about+targeting+RNA+with+small+molecules&doi=10.1021%2Facs.jmedchem.9b01927"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">How We Think about Targeting RNA with Small Molecules</span></div><div class="casAuthors">Costales, Matthew G.; Childs-Disney, Jessica L.; Haniff, Hafeez S.; Disney, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">8880-8900</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  RNA offers nearly unlimited potential as a target for small mol. chem. probes and lead medicines.  Many RNAs fold into structures that can be selectively targeted with small mols.  This Perspective discusses mol. recognition of RNA by small mols. and highlights key enabling technologies and properties of bioactive interactions.  Sequence-based design of ligands targeting RNA has established rules for affecting RNA targets and provided a potentially general platform for the discovery of bioactive small mols.  The RNA targets that contain preferred small mol. binding sites can be identified from sequence, allowing identification of off-targets and prediction of bioactive interactions by nature of ligand recognition of functional sites.  Small mol. targeted degrdn. of RNA targets (RNase-targeted chimeras, RIBOTACs) and direct cleavage by small mols. have also been developed.  These growing technologies suggest that the time is right to provide small mol. chem. probes to target functionally relevant RNAs throughout the human transcriptome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpsFk_gFhxgTbVg90H21EOLACvtfcHk0ljzcSC73PrHrQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXlsFygsr8%253D&md5=4fe34106689d2d6d38814d79357ebd1a</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01927%26sid%3Dliteratum%253Aachs%26aulast%3DCostales%26aufirst%3DM.%2BG.%26aulast%3DChilds-Disney%26aufirst%3DJ.%2BL.%26aulast%3DHaniff%26aufirst%3DH.%2BS.%26aulast%3DDisney%26aufirst%3DM.%2BD.%26atitle%3DHow%2520we%2520think%2520about%2520targeting%2520RNA%2520with%2520small%2520molecules%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D8880%26epage%3D8900%26doi%3D10.1021%2Facs.jmedchem.9b01927" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ursu, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Childs-Disney, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andrews, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">OâLeary, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Angelbello, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moss, W. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span> <span> </span><span class="NLM_article-title">Design of small molecules targeting RNA structure from sequence</span>. <i>Chem. Soc. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">7252</span>â <span class="NLM_lpage">7270</span>, <span class="refDoi">Â DOI: 10.1039/D0CS00455C</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1039%2FD0CS00455C" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=32935689" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhvVClur7M" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2020&pages=7252-7270&author=A.+Ursuauthor=J.+L.+Childs-Disneyauthor=R.+J.+Andrewsauthor=C.+A.+O%E2%80%99Learyauthor=S.+M.+Meyerauthor=A.+J.+Angelbelloauthor=W.+N.+Mossauthor=M.+D.+Disney&title=Design+of+small+molecules+targeting+RNA+structure+from+sequence&doi=10.1039%2FD0CS00455C"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Design of small molecules targeting RNA structure from sequence</span></div><div class="casAuthors">Ursu, Andrei; Childs-Disney, Jessica L.; Andrews, Ryan J.; O'Leary, Collin A.; Meyer, Samantha M.; Angelbello, Alicia J.; Moss, Walter N.; Disney, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Society Reviews</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">7252-7270</span>CODEN:
                <span class="NLM_cas:coden">CSRVBR</span>;
        ISSN:<span class="NLM_cas:issn">0306-0012</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  The design and discovery of small mol. medicines has largely been focused on a small no. of druggable protein families.  A new paradigm is emerging, however, in which small mols. exert a biol. effect by interacting with RNA, both to study human disease biol. and provide lead therapeutic modalities.  Due to this potential for expanding target pipelines and treating a larger no. of human diseases, robust platforms for the rational design and optimization of small mols. interacting with RNAs (SMIRNAs) are in high demand.  This review highlights three major pillars in this area.  First, the transcriptome-wide identification and validation of structured RNA elements, or motifs, within disease-causing RNAs directly from sequence is presented.  Second, we provide an overview of high-throughput screening approaches to identify SMIRNAs as well as discuss the lead identification strategy, Inforna, which decodes the three-dimensional (3D) conformation of RNA motifs with small mol. binding partners, directly from sequence.  An emphasis is placed on target validation methods to study the causality between modulating the RNA motif in vitro and the phenotypic outcome in cells.  Third, emergent modalities that convert occupancy-driven mode of action SMIRNAs into event-driven small mol. chem. probes, such as RNA cleavers and degraders, are presented.  Finally, the future of the small mol. RNA therapeutics field is discussed, as well as hurdles to overcome to develop potent and selective RNA-centric chem. probes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqSSSsDhiFFmLVg90H21EOLACvtfcHk0ljzcSC73PrHrQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhvVClur7M&md5=9823258172b4a8392c49c5da87a23ae6</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1039%2FD0CS00455C&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FD0CS00455C%26sid%3Dliteratum%253Aachs%26aulast%3DUrsu%26aufirst%3DA.%26aulast%3DChilds-Disney%26aufirst%3DJ.%2BL.%26aulast%3DAndrews%26aufirst%3DR.%2BJ.%26aulast%3DO%25E2%2580%2599Leary%26aufirst%3DC.%2BA.%26aulast%3DMeyer%26aufirst%3DS.%2BM.%26aulast%3DAngelbello%26aufirst%3DA.%2BJ.%26aulast%3DMoss%26aufirst%3DW.%2BN.%26aulast%3DDisney%26aufirst%3DM.%2BD.%26atitle%3DDesign%2520of%2520small%2520molecules%2520targeting%2520RNA%2520structure%2520from%2520sequence%26jtitle%3DChem.%2520Soc.%2520Rev.%26date%3D2020%26volume%3D49%26spage%3D7252%26epage%3D7270%26doi%3D10.1039%2FD0CS00455C" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Falese, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donlic, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hargrove, A. E.</span></span> <span> </span><span class="NLM_article-title">Targeting RNA with small molecules: from fundamental principles towards the clinic</span>. <i>Chem. Soc. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">2224</span>â <span class="NLM_lpage">2243</span>, <span class="refDoi">Â DOI: 10.1039/D0CS01261K</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1039%2FD0CS01261K" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=33458725" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BB3MXhsVOhu7Y%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2021&pages=2224-2243&author=J.+P.+Faleseauthor=A.+Donlicauthor=A.+E.+Hargrove&title=Targeting+RNA+with+small+molecules%3A+from+fundamental+principles+towards+the+clinic&doi=10.1039%2FD0CS01261K"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting RNA with small molecules: from fundamental principles towards the clinic</span></div><div class="casAuthors">Falese, James P.; Donlic, Anita; Hargrove, Amanda E.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Society Reviews</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">2224-2243</span>CODEN:
                <span class="NLM_cas:coden">CSRVBR</span>;
        ISSN:<span class="NLM_cas:issn">0306-0012</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  Recent advances in our understanding of RNA biol. have uncovered crucial roles for RNA in multiple disease states, ranging from viral and bacterial infections to cancer and neurol. disorders.  As a result, multiple labs. have become interested in developing drug-like small mols. to target RNA.  However, this development comes with multiple unique challenges.  For example, RNA is inherently dynamic and has limited chem. diversity.  This Tutorial Review overviews important considerations and advancements for generating RNA-targeted small mols., ranging from fundamental chem. to promising small mol. examples with demonstrated clin. efficacy.  We then discuss fundamental RNA recognition principles along with methods for small mol. screening and RNA structure detn.  Finally, we review unique challenges and emerging solns. from both the RNA and small mol. perspectives for generating RNA-targeted ligands before highlighting a selection of the "Greatest Hits" to date.  Addnl., we explore the recently FDA-approved small mol. regulator of RNA splicing, risdiplam, for treatment of spinal muscular atrophy.  Together, this Tutorial Review showcases the fundamental role of chem. and mol. recognition principles in enhancing our understanding of RNA biol. and contributing to the rapidly growing no. of RNA-targeted probes and therapeutics.  In particular, we hope this widely accessible review will serve as inspiration for aspiring small mol. and/or RNA researchers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPsmlpE72GLrVg90H21EOLACvtfcHk0lg1h6ldYnwV6w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXhsVOhu7Y%253D&md5=40f76d282a164fe9409f4fc45e81b0cf</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1039%2FD0CS01261K&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FD0CS01261K%26sid%3Dliteratum%253Aachs%26aulast%3DFalese%26aufirst%3DJ.%2BP.%26aulast%3DDonlic%26aufirst%3DA.%26aulast%3DHargrove%26aufirst%3DA.%2BE.%26atitle%3DTargeting%2520RNA%2520with%2520small%2520molecules%253A%2520from%2520fundamental%2520principles%2520towards%2520the%2520clinic%26jtitle%3DChem.%2520Soc.%2520Rev.%26date%3D2021%26volume%3D50%26spage%3D2224%26epage%3D2243%26doi%3D10.1039%2FD0CS01261K" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ameres, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zamore, P. D.</span></span> <span> </span><span class="NLM_article-title">Diversifying microRNA sequence and function</span>. <i>Nat. Rev. Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">475</span>â <span class="NLM_lpage">488</span>, <span class="refDoi">Â DOI: 10.1038/nrm3611</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1038%2Fnrm3611" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=23800994" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtVShtrnK" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2013&pages=475-488&author=S.+L.+Ameresauthor=P.+D.+Zamore&title=Diversifying+microRNA+sequence+and+function&doi=10.1038%2Fnrm3611"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Diversifying microRNA sequence and function</span></div><div class="casAuthors">Ameres, Stefan L.; Zamore, Phillip D.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">475-488</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  MicroRNAs (miRNAs) regulate the expression of most genes in animals, but we are only now beginning to understand how they are generated, assembled into functional complexes and destroyed.  Various mechanisms have now been identified that regulate miRNA stability and that diversify miRNA sequences to create distinct isoforms.  The prodn. of different isoforms of individual miRNAs in specific cells and tissues may have broader implications for miRNA-mediated gene expression control.  Rigorously testing the many discrepant models for how miRNAs function using quant. biochem. measurements made in vivo and in vitro remains a major challenge for the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgqnSfNVGvZ7Vg90H21EOLACvtfcHk0lg1h6ldYnwV6w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtVShtrnK&md5=d81c4d7a8d42eda6b1f6470f05f7b021</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1038%2Fnrm3611&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm3611%26sid%3Dliteratum%253Aachs%26aulast%3DAmeres%26aufirst%3DS.%2BL.%26aulast%3DZamore%26aufirst%3DP.%2BD.%26atitle%3DDiversifying%2520microRNA%2520sequence%2520and%2520function%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2013%26volume%3D14%26spage%3D475%26epage%3D488%26doi%3D10.1038%2Fnrm3611" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bartel, D. P.</span></span> <span> </span><span class="NLM_article-title">MicroRNAs: genomics, biogenesis, mechanism, and function</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>116</i></span>,  <span class="NLM_fpage">281</span>â <span class="NLM_lpage">297</span>, <span class="refDoi">Â DOI: 10.1016/S0092-8674(04)00045-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1016%2FS0092-8674%2804%2900045-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=14744438" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVals7o%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=2004&pages=281-297&author=D.+P.+Bartel&title=MicroRNAs%3A+genomics%2C+biogenesis%2C+mechanism%2C+and+function&doi=10.1016%2FS0092-8674%2804%2900045-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">MicroRNAs: Genomics, biogenesis, mechanism, and function</span></div><div class="casAuthors">Bartel, David P.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">116</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">281-297</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  MicroRNAs (miRNAs) are endogenous â¼22 nt RNAs that can play important regulatory roles in animals and plants by targeting mRNAs for cleavage or translational repression.  Although they escaped notice until relatively recently, miRNAs comprise one of the more abundant classes of gene regulatory mols. in multicellular organisms and likely influence the output of many protein-coding genes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqzPI7QCGHbXbVg90H21EOLACvtfcHk0lg1h6ldYnwV6w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVals7o%253D&md5=9470974cbee3731cd29c7e204490b529</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2FS0092-8674%2804%2900045-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0092-8674%252804%252900045-5%26sid%3Dliteratum%253Aachs%26aulast%3DBartel%26aufirst%3DD.%2BP.%26atitle%3DMicroRNAs%253A%2520genomics%252C%2520biogenesis%252C%2520mechanism%252C%2520and%2520function%26jtitle%3DCell%26date%3D2004%26volume%3D116%26spage%3D281%26epage%3D297%26doi%3D10.1016%2FS0092-8674%2804%2900045-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Volinia, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calin, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ambs, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cimmino, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petrocca, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Visone, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iorio, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roldo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferracin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prueitt, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yanaihara, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lanza, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scarpa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vecchione, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Negrini, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Croce, C. M.</span></span> <span> </span><span class="NLM_article-title">A microRNA expression signature of human solid tumors defines cancer gene targets</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>103</i></span>,  <span class="NLM_fpage">2257</span>â <span class="NLM_lpage">2261</span>, <span class="refDoi">Â DOI: 10.1073/pnas.0510565103</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1073%2Fpnas.0510565103" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=16461460" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BD28XhslGjsbg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=2006&pages=2257-2261&author=S.+Voliniaauthor=G.+A.+Calinauthor=C.+G.+Liuauthor=S.+Ambsauthor=A.+Cimminoauthor=F.+Petroccaauthor=R.+Visoneauthor=M.+Iorioauthor=C.+Roldoauthor=M.+Ferracinauthor=R.+L.+Prueittauthor=N.+Yanaiharaauthor=G.+Lanzaauthor=A.+Scarpaauthor=A.+Vecchioneauthor=M.+Negriniauthor=C.+C.+Harrisauthor=C.+M.+Croce&title=A+microRNA+expression+signature+of+human+solid+tumors+defines+cancer+gene+targets&doi=10.1073%2Fpnas.0510565103"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">A microRNA expression signature of human solid tumors defines cancer gene targets</span></div><div class="casAuthors">Volinia, Stefano; Calin, George A.; Liu, Chang-Gong; Ambs, Stefan; Cimmino, Amelia; Petrocca, Fabio; Visone, Rosa; Iorio, Marilena; Roldo, Claudia; Ferracin, Manuela; Prueitt, Robyn L.; Yanaihara, Nozumu; Lanza, Giovanni; Scarpa, Aldo; Vecchione, Andrea; Negrini, Massimo; Harris, Curtis C.; Croce, Carlo M.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">103</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2257-2261</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Small noncoding microRNAs (miRNAs) can contribute to cancer development and progression and are differentially expressed in normal tissues and cancers.  From a large-scale miRnome anal. on 540 samples including lung, breast, stomach, prostate, colon, and pancreatic tumors, we identified a solid cancer miRNA signature composed by a large portion of overexpressed miRNAs.  Among these miRNAs are some with well characterized cancer assocn., such as miR-17-5p, miR-20a, miR-21, miR-92, miR-106a, and miR-155.  The predicted targets for the differentially expressed miRNAs are significantly enriched for protein-coding tumor suppressors and oncogenes (P < 0.0001).  A no. of the predicted targets, including the tumor suppressors RB1 (Retinoblastoma 1) and TGFBR2 (transforming growth factor, beta receptor II) genes were confirmed exptl.  Our results indicate that miRNAs are extensively involved in cancer pathogenesis of solid tumors and support their function as either dominant or recessive cancer genes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpSPx3elVKg6LVg90H21EOLACvtfcHk0lg1h6ldYnwV6w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhslGjsbg%253D&md5=e2b19f23e04996312fa922d76a4f35bc</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0510565103&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0510565103%26sid%3Dliteratum%253Aachs%26aulast%3DVolinia%26aufirst%3DS.%26aulast%3DCalin%26aufirst%3DG.%2BA.%26aulast%3DLiu%26aufirst%3DC.%2BG.%26aulast%3DAmbs%26aufirst%3DS.%26aulast%3DCimmino%26aufirst%3DA.%26aulast%3DPetrocca%26aufirst%3DF.%26aulast%3DVisone%26aufirst%3DR.%26aulast%3DIorio%26aufirst%3DM.%26aulast%3DRoldo%26aufirst%3DC.%26aulast%3DFerracin%26aufirst%3DM.%26aulast%3DPrueitt%26aufirst%3DR.%2BL.%26aulast%3DYanaihara%26aufirst%3DN.%26aulast%3DLanza%26aufirst%3DG.%26aulast%3DScarpa%26aufirst%3DA.%26aulast%3DVecchione%26aufirst%3DA.%26aulast%3DNegrini%26aufirst%3DM.%26aulast%3DHarris%26aufirst%3DC.%2BC.%26aulast%3DCroce%26aufirst%3DC.%2BM.%26atitle%3DA%2520microRNA%2520expression%2520signature%2520of%2520human%2520solid%2520tumors%2520defines%2520cancer%2520gene%2520targets%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2006%26volume%3D103%26spage%3D2257%26epage%3D2261%26doi%3D10.1073%2Fpnas.0510565103" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stenvang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petri, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindow, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Obad, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kauppinen, S.</span></span> <span> </span><span class="NLM_article-title">Inhibition of microRNA function by antimiR oligonucleotides</span>. <i>Silence</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">1</span>, <span class="refDoi">Â DOI: 10.1186/1758-907X-3-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1186%2F1758-907X-3-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=22230293" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC38Xlt1Cit7w%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2012&pages=1&author=J.+Stenvangauthor=A.+Petriauthor=M.+Lindowauthor=S.+Obadauthor=S.+Kauppinen&title=Inhibition+of+microRNA+function+by+antimiR+oligonucleotides&doi=10.1186%2F1758-907X-3-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of microRNA function by antimiR oligonucleotides</span></div><div class="casAuthors">Stenvang, Jan; Petri, Andreas; Lindow, Morten; Obad, Susanna; Kauppinen, Sakari</div><div class="citationInfo"><span class="NLM_cas:title">Silence</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1</span>CODEN:
                <span class="NLM_cas:coden">SILEBI</span>;
        ISSN:<span class="NLM_cas:issn">1758-907X</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">A review.  MicroRNAs (miRNAs) have emerged as important post-transcriptional regulators of gene expression in many developmental and cellular processes.  Moreover, there is now ample evidence that perturbations in the levels of individual or entire families of miRNAs are strongly assocd. with the pathogenesis of a wide range of human diseases.  Indeed, disease-assocd. miRNAs represent a new class of targets for the development of miRNA-based therapeutic modalities, which may yield patient benefits unobtainable by other therapeutic approaches.  The recent explosion in miRNA research has accelerated the development of several computational and exptl. approaches for probing miRNA functions in cell culture and in vivo.  In this review, we focus on the use of antisense oligonucleotides (antimiRs) in miRNA inhibition for loss-of-function studies.  We provide an overview of the currently employed antisense chemistries and their utility in designing antimiR oligonucleotides.  Furthermore, we describe the most commonly used in vivo delivery strategies and discuss different approaches for assessment of miRNA inhibition and potential off-target effects.  Finally, we summarize recent progress in antimiR mediated pharmacol. inhibition of disease-assocd. miRNAs, which shows great promise in the development of novel miRNA-based therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxHTr8fhW5n7Vg90H21EOLACvtfcHk0lgbzAHZWdd_rQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xlt1Cit7w%253D&md5=897115eb2c4cc654fdabbdd45cc4e58d</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1186%2F1758-907X-3-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1758-907X-3-1%26sid%3Dliteratum%253Aachs%26aulast%3DStenvang%26aufirst%3DJ.%26aulast%3DPetri%26aufirst%3DA.%26aulast%3DLindow%26aufirst%3DM.%26aulast%3DObad%26aufirst%3DS.%26aulast%3DKauppinen%26aufirst%3DS.%26atitle%3DInhibition%2520of%2520microRNA%2520function%2520by%2520antimiR%2520oligonucleotides%26jtitle%3DSilence%26date%3D2012%26volume%3D3%26spage%3D1%26doi%3D10.1186%2F1758-907X-3-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Juliano, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ming, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakagawa, O.</span></span> <span> </span><span class="NLM_article-title">The chemistry and biology of oligonucleotide conjugates</span>. <i>Acc. Chem. Res.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">1067</span>â <span class="NLM_lpage">1076</span>, <span class="refDoi">Â DOI: 10.1021/ar2002123</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ar2002123" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC38XislSqu7s%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2012&pages=1067-1076&author=R.+L.+Julianoauthor=X.+Mingauthor=O.+Nakagawa&title=The+chemistry+and+biology+of+oligonucleotide+conjugates&doi=10.1021%2Far2002123"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">The Chemistry and Biology of Oligonucleotide Conjugates</span></div><div class="casAuthors">Juliano, R. L.; Ming, Xin; Nakagawa, Osamu</div><div class="citationInfo"><span class="NLM_cas:title">Accounts of Chemical Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1067-1076</span>CODEN:
                <span class="NLM_cas:coden">ACHRE4</span>;
        ISSN:<span class="NLM_cas:issn">0001-4842</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Short DNA or RNA oligonucleotides have tremendous potential as therapeutic agents.  Because of their ability to engage in Watson-Crick base pairing, they can interact with mRNA or pre-mRNA targets with high selectivity.  As a result, they could precisely manipulate gene expression.  This possibility has engendered extensive efforts to develop oligonucleotides as drugs, and many candidates are already in clin. trials.  However, a major impediment to the maturation of this field of oligonucleotide-based therapeutics remains: these relatively large and often highly charged mols. don't easily cross cellular membranes, making it difficult for them to reach their sites of action in the cytosol or nucleus.  In this Account, we summarize some basic features of the biol. of antisense and siRNA oligonucleotides.  We then discuss chem. conjugation as an approach to improving the intracellular delivery and therapeutic potential of these agents.  Instead of focusing on the details of conjugation chem., we emphasize the pharmacol. ramifications of oligonucleotide conjugates.  In one important approach to improving delivery and efficacy, researchers have conjugated oligonucleotides with ligands designed to bind to particular receptors and thus provide specific interactions with cells.  In another strategy, researchers have coupled antisense or siRNA with agents such as cell penetrating peptides that are designed to provoke escape of the conjugate from intracellular vesicular compartments.  Although both of these strategies have had some success, further research is needed before oligonucleotide conjugates can find an important place in human therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqOVDWra3YKhLVg90H21EOLACvtfcHk0lgbzAHZWdd_rQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XislSqu7s%253D&md5=e6eecf9a41ee6c31acbe1018eb655d56</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Far2002123&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Far2002123%26sid%3Dliteratum%253Aachs%26aulast%3DJuliano%26aufirst%3DR.%2BL.%26aulast%3DMing%26aufirst%3DX.%26aulast%3DNakagawa%26aufirst%3DO.%26atitle%3DThe%2520chemistry%2520and%2520biology%2520of%2520oligonucleotide%2520conjugates%26jtitle%3DAcc.%2520Chem.%2520Res.%26date%3D2012%26volume%3D45%26spage%3D1067%26epage%3D1076%26doi%3D10.1021%2Far2002123" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Connelly, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boer, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moon, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gareiss, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneekloth, J. S.</span></span> <span> </span><span class="NLM_article-title">Discovery of inhibitors of microRNA-21 processing using small molecule microarrays</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">435</span>â <span class="NLM_lpage">443</span>, <span class="refDoi">Â DOI: 10.1021/acschembio.6b00945</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.6b00945" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC28XitVKqtbjP" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=435-443&author=C.+M.+Connellyauthor=R.+E.+Boerauthor=M.+H.+Moonauthor=P.+Gareissauthor=J.+S.+Schneekloth&title=Discovery+of+inhibitors+of+microRNA-21+processing+using+small+molecule+microarrays&doi=10.1021%2Facschembio.6b00945"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Inhibitors of MicroRNA-21 Processing Using Small Molecule Microarrays</span></div><div class="casAuthors">Connelly, Colleen M.; Boer, Robert E.; Moon, Michelle H.; Gareiss, Peter; Schneekloth, John S.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">435-443</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The identification of small mols. that bind to and perturb the function of microRNAs is an attractive approach for the treatment for microRNA-assocd. pathologies.  However, there are only a few small mols. known to interact directly with microRNAs.  Here, the authors report the use of a small mol. microarray (SMM) screening approach to identify low mol. wt. compds. that directly bind to a pre-miR-21 hairpin.  Compds. identified using this approach exhibit good affinity for the RNA (ranging from 0.8-2.0 Î¼M) and are not composed of a polycationic scaffold.  Several of the highest affinity compds. inhibit Dicer-mediated processing, while in-line probing expts. indicate that the compds. bind to the apical loop of the hairpin, proximal to the Dicer site.  This work provides evidence that small mols. can be developed to bind directly to and inhibit miR-21.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq3vZ7QBA7lT7Vg90H21EOLACvtfcHk0lgbzAHZWdd_rQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitVKqtbjP&md5=56a3a74254fc26302496b05de395591a</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Facschembio.6b00945&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.6b00945%26sid%3Dliteratum%253Aachs%26aulast%3DConnelly%26aufirst%3DC.%2BM.%26aulast%3DBoer%26aufirst%3DR.%2BE.%26aulast%3DMoon%26aufirst%3DM.%2BH.%26aulast%3DGareiss%26aufirst%3DP.%26aulast%3DSchneekloth%26aufirst%3DJ.%2BS.%26atitle%3DDiscovery%2520of%2520inhibitors%2520of%2520microRNA-21%2520processing%2520using%2520small%2520molecule%2520microarrays%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2017%26volume%3D12%26spage%3D435%26epage%3D443%26doi%3D10.1021%2Facschembio.6b00945" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pei, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shu, Y.</span></span> <span> </span><span class="NLM_article-title">Alteration in miR-21/PTEN expression modulates gefitinib resistance in non-small cell lung cancer</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>9</i></span>, <span class="NLM_elocation-id">e103305</span> <span class="refDoi">Â DOI: 10.1371/journal.pone.0103305</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1371%2Fjournal.pone.0103305" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=25058005" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1ersrnM" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&author=H.+Shenauthor=F.+Zhuauthor=J.+Liuauthor=T.+Xuauthor=D.+Peiauthor=R.+Wangauthor=Y.+Qianauthor=Q.+Liauthor=L.+Wangauthor=Z.+Shiauthor=J.+Zhengauthor=Q.+Chenauthor=B.+Jiangauthor=Y.+Shu&title=Alteration+in+miR-21%2FPTEN+expression+modulates+gefitinib+resistance+in+non-small+cell+lung+cancer&doi=10.1371%2Fjournal.pone.0103305"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Alteration in miR-21/Pten expression modulates gefitinib resistance in non-small cell lung cancer</span></div><div class="casAuthors">Shen, Hua; Zhu, Fang; Liu, Jinyuan; Xu, Tongpeng; Pei, Dong; Wang, Rong; Qian, Yingying; Li, Qi; Wang, Lin; Shi, Zhumei; Zheng, Jitai; Chen, Qiudan; Jiang, Binghua; Shu, Yongqian</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">e103305/1-e103305/9, 9 pp.</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Resistance to TKI treatment is a major obstacle in effective treatment of NSCLC.  Besides EGFR mutation status, the mechanisms involved are largely unknown.  Some evidence supports a role for microRNA 21 in modulating drug sensitivity of chemotherapy but its role in NSCLC TKI resistance still remains unexplored.  This study aimed to investigate whether NSCLC miR-21 mediated resistance to TKIs also results from Pten targeting.  Here, we show miR-21 promotes cancer by neg. regulating Pten expression in human NSCLC tissues: high miR-21 expression levels were assocd. with shorter DFS in 47 NSCLC patients; high miR-21/low Pten expression levels indicated a poor TKI clin. response and shorter overall survival in another 46 NSCLC patients undergoing TKI treatment.  In vitro assays showed that miR-21 was up-regulated concomitantly to down-regulation of Pten in pc-9/GR cells in comparison with pc-9 cells.  Moreover, over-expression of miR-21 significantly decreased gefitinib sensitivity by down-regulating Pten expression and activating Akt and ERK pathways in pc-9 cells, while miR-21 knockdown dramatically restored gefitinib sensitivity of pc-9/GR cells by up-regulation of Pten expression and inactivation of AKT and ERK pathways, in vivo and in vitro.  We propose alteration of miR-21/Pten expression as a novel mechanism for TKI resistance in NSCLC cancer.  Our findings provide a new basis for using miR 21/Pten-based therapeutic strategies to reverse gefitinib resistance in NSCLC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJhlsVIgQXMrVg90H21EOLACvtfcHk0lgP3F4Nb6Mkhw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1ersrnM&md5=08dda1c3b5dcd0f5ab28777438ea6533</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0103305&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0103305%26sid%3Dliteratum%253Aachs%26aulast%3DShen%26aufirst%3DH.%26aulast%3DZhu%26aufirst%3DF.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DT.%26aulast%3DPei%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DR.%26aulast%3DQian%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DQ.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DShi%26aufirst%3DZ.%26aulast%3DZheng%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DQ.%26aulast%3DJiang%26aufirst%3DB.%26aulast%3DShu%26aufirst%3DY.%26atitle%3DAlteration%2520in%2520miR-21%252FPTEN%2520expression%2520modulates%2520gefitinib%2520resistance%2520in%2520non-small%2520cell%2520lung%2520cancer%26jtitle%3DPLoS%2520One%26date%3D2014%26volume%3D9%26doi%3D10.1371%2Fjournal.pone.0103305" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fei, J.</span></span> <span> </span><span class="NLM_article-title">Anti-miR-21 oligonucleotide enhances chemosensitivity of leukemic HL60 cells to arabinosylcytosine by inducing apoptosis</span>. <i>Hematology</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">215</span>â <span class="NLM_lpage">221</span>, <span class="refDoi">Â DOI: 10.1179/102453310X12647083620840</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1179%2F102453310X12647083620840" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=20670480" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtFajsbbL" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2010&pages=215-221&author=Y.+Liauthor=X.+Zhuauthor=J.+Guauthor=H.+Huauthor=D.+Dongauthor=J.+Yaoauthor=C.+Linauthor=J.+Fei&title=Anti-miR-21+oligonucleotide+enhances+chemosensitivity+of+leukemic+HL60+cells+to+arabinosylcytosine+by+inducing+apoptosis&doi=10.1179%2F102453310X12647083620840"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Anti-miR-21 oligonucleotide enhances chemosensitivity of leukemic HL60 cells to arabinosylcytosine by inducing apoptosis</span></div><div class="casAuthors">Li, Yumin; Zhu, Xuejiao; Gu, Jingyi; Hu, Haiyan; Dong, Dawei; Yao, Junlin; Lin, Chunyan; Fei, Jia</div><div class="citationInfo"><span class="NLM_cas:title">Hematology (Leeds, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">215-221</span>CODEN:
                <span class="NLM_cas:coden">HMATFL</span>;
        ISSN:<span class="NLM_cas:issn">1024-5332</span>.
    
            (<span class="NLM_cas:orgname">Maney Publishing</span>)
        </div><div class="casAbstract">Drug insensitivity or resistance is a major obstacle for successful treatment of acute myeloid leukemia.  MicroRNAs (miRNAs) are small non-coding RNA mols.  Increasing evidence suggests that miRNAs modulate cellular sensitivity to anticancer drugs.  We used a specific anti-miR-21 oligonucleotide (AMO-miR-21) to sensitize leukemic HL60 cells to arabinosylcytosine (Ara-C) by down-regulating miR-21.  AMO-miR-21 alone effectively inhibited HL60 cell viability as measured by MTT assays and induced apoptosis as evaluated by flow cytometry, whereas AMO-miR-21 in combination with Ara-C enhanced HL60 cells to Ara-C-sensitivity and promoted Ara-C-induced apoptosis.  Levels of miR-21 and its target PDCD4, quantified by real-time PCR, showed that expression of miR-21 was significantly decreased after AMO-miR-21 treatment.  PDCD4 as a direct target of miR-21 in leukemic HL60 cells was confirmed by the dual-luciferase reporter assay.  Our study suggests that AMO-miR-21 significantly sensitizes HL60 cells to Ara-C by inducing apoptosis and these effects of AMO-miR-21 may be partially due to its up-regulation of PDCD4.  Therefore, exploiting synergistic effects between AMO-miR-21 and Ara-C might be an effective clin. strategy for leukemia chemotherapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5RDEtManOabVg90H21EOLACvtfcHk0lgP3F4Nb6Mkhw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtFajsbbL&md5=89dc5e475cf062dc95a84f7a93362ed2</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1179%2F102453310X12647083620840&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1179%252F102453310X12647083620840%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DY.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DGu%26aufirst%3DJ.%26aulast%3DHu%26aufirst%3DH.%26aulast%3DDong%26aufirst%3DD.%26aulast%3DYao%26aufirst%3DJ.%26aulast%3DLin%26aufirst%3DC.%26aulast%3DFei%26aufirst%3DJ.%26atitle%3DAnti-miR-21%2520oligonucleotide%2520enhances%2520chemosensitivity%2520of%2520leukemic%2520HL60%2520cells%2520to%2520arabinosylcytosine%2520by%2520inducing%2520apoptosis%26jtitle%3DHematology%26date%3D2010%26volume%3D15%26spage%3D215%26epage%3D221%26doi%3D10.1179%2F102453310X12647083620840" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gumireddy, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hogenesch, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deiters, A.</span></span> <span> </span><span class="NLM_article-title">Small-molecule inhibitors of microRNA miR-21 function</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">7482</span>â <span class="NLM_lpage">7484</span>, <span class="refDoi">Â DOI: 10.1002/anie.200801555</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1002%2Fanie.200801555" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BD1cXht1Sjur7I" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2008&pages=7482-7484&author=K.+Gumireddyauthor=D.+D.+Youngauthor=X.+Xiongauthor=J.+B.+Hogeneschauthor=Q.+Huangauthor=A.+Deiters&title=Small-molecule+inhibitors+of+microRNA+miR-21+function&doi=10.1002%2Fanie.200801555"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Small-molecule inhibitors of microRNA miR-21 function</span></div><div class="casAuthors">Gumireddy, Kiranmai; Young, Douglas D.; Xiong, Xin; Hogenesch, John B.; Huang, Qihong; Deiters, Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">39</span>),
    <span class="NLM_cas:pages">7482-7484</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Short, but significant, microRNAs (miRNAs) are an important class of gene regulators.  Small-mol. modifiers of miRNA function, such as 1 (schematic representation), were identified in a cellular screen for miRNA-pathway inhibitors.  Such compds. are expected to be useful tools for the elucidation of detailed mechanisms of miRNA action and may serve as lead structures for the development of new therapeutic agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoegx2d4LaAqLVg90H21EOLACvtfcHk0lgP3F4Nb6Mkhw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXht1Sjur7I&md5=af016e4b3a6bfe73de80498ccfdc1bd6</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1002%2Fanie.200801555&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.200801555%26sid%3Dliteratum%253Aachs%26aulast%3DGumireddy%26aufirst%3DK.%26aulast%3DYoung%26aufirst%3DD.%2BD.%26aulast%3DXiong%26aufirst%3DX.%26aulast%3DHogenesch%26aufirst%3DJ.%2BB.%26aulast%3DHuang%26aufirst%3DQ.%26aulast%3DDeiters%26aufirst%3DA.%26atitle%3DSmall-molecule%2520inhibitors%2520of%2520microRNA%2520miR-21%2520function%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2008%26volume%3D47%26spage%3D7482%26epage%3D7484%26doi%3D10.1002%2Fanie.200801555" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, W. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, S. M.</span></span> <span> </span><span class="NLM_article-title">Discovery of 4-benzoylamino-<i>N</i>-(prop-2-yn-1-yl)benzamides as novel microRNA-21 inhibitors</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">6510</span>â <span class="NLM_lpage">6519</span>, <span class="refDoi">Â DOI: 10.1016/j.bmc.2015.08.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1016%2Fj.bmc.2015.08.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=26344589" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtlCru7vO" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2015&pages=6510-6519&author=C.+S.+Jiangauthor=X.+M.+Wangauthor=S.+Q.+Zhangauthor=L.+S.+Mengauthor=W.+H.+Zhuauthor=J.+Xuauthor=S.+M.+Lu&title=Discovery+of+4-benzoylamino-N-%28prop-2-yn-1-yl%29benzamides+as+novel+microRNA-21+inhibitors&doi=10.1016%2Fj.bmc.2015.08.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of 4-benzoylamino-N-(prop-2-yn-1-yl)benzamides as novel microRNA-21 inhibitors</span></div><div class="casAuthors">Jiang, Cong-shan; Wang, Xiao-meng; Zhang, San-qi; Meng, Lie-su; Zhu, Wen-hua; Xu, Jing; Lu, She-min</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">6510-6519</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">MicroRNA-21, as an oncogenic miRNA, has caught great attention for medicinal chemists to develop its novel inhibitors for cancer therapy.  In the present study, the authors designed 4-benzoylamino-N-(prop-2-yn-1-yl)benzamides as miR-21 inhibitor candidates on the basis of scaffold hopping.  Eighteen compds. were synthesized.  The inhibitory activities of synthesized compds. against the expression of miR-21 were evaluated using stem loop RT-qPCR and compd. I was discovered as the most potent compd., which displayed a time and concn. dependent inhibition manner.  In addn., various functional assays such as the expression of miR-21 target gene detected by Western blotting and the cell growth and apoptosis detected by flow cytometric anal. were checked in Hela (human epithelioid cervix carcinoma) and U-87 MG (human glioblastoma) cells to confirm its activity.  The results indicate that compd. I can enhance apoptosis, retard proliferation, and up-regulate PDCD4, a target protein of miR-21.  In addn., the compd. I does not influence the expression of multiple miRNAs and the genes that participate in miRNA universal biosynthesis pathway.  These results strongly support the assumption that title compds. can serve as a small mol. inhibitor of miR-21.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7D9ug6Mh3BrVg90H21EOLACvtfcHk0ljVvLkY9Rfr_g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtlCru7vO&md5=dffa0613735eb77533a042752224c703</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2015.08.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2015.08.007%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DC.%2BS.%26aulast%3DWang%26aufirst%3DX.%2BM.%26aulast%3DZhang%26aufirst%3DS.%2BQ.%26aulast%3DMeng%26aufirst%3DL.%2BS.%26aulast%3DZhu%26aufirst%3DW.%2BH.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DLu%26aufirst%3DS.%2BM.%26atitle%3DDiscovery%2520of%25204-benzoylamino-N-%2528prop-2-yn-1-yl%2529benzamides%2520as%2520novel%2520microRNA-21%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2015%26volume%3D23%26spage%3D6510%26epage%3D6519%26doi%3D10.1016%2Fj.bmc.2015.08.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Naro, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ankenbruck, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tivon, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Connelly, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gardner, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deiters, A.</span></span> <span> </span><span class="NLM_article-title">Small molecule inhibition of microRNA miR-21 rescues chemosensitivity of renal-cell carcinoma to topotecan</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">5900</span>â <span class="NLM_lpage">5909</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.7b01891</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01891" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1yrurrF" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=5900-5909&author=Y.+Naroauthor=N.+Ankenbruckauthor=M.+Thomasauthor=Y.+Tivonauthor=C.+M.+Connellyauthor=L.+Gardnerauthor=A.+Deiters&title=Small+molecule+inhibition+of+microRNA+miR-21+rescues+chemosensitivity+of+renal-cell+carcinoma+to+topotecan&doi=10.1021%2Facs.jmedchem.7b01891"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Small Molecule Inhibition of MicroRNA miR-21 Rescues Chemosensitivity of Renal-Cell Carcinoma to Topotecan</span></div><div class="casAuthors">Naro, Yuta; Ankenbruck, Nicholas; Thomas, Meryl; Tivon, Yaniv; Connelly, Colleen M.; Gardner, Laura; Deiters, Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">5900-5909</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Chem. probes of microRNA (miRNA) function are potential tools for understanding miRNA biol. that also provide new approaches for discovering therapeutics for miRNA-assocd. diseases.  MicroRNA-21 (miR-21) is an oncogenic miRNA that is overexpressed in most cancers and has been strongly assocd. with driving chemoresistance in cancers such as renal cell carcinoma (RCC).  Using a cell-based luciferase reporter assay to screen small mols., we identified a novel inhibitor of miR-21 function.  Following structure-activity relationship studies, an optimized lead compd. demonstrated cytotoxicity in several cancer cell lines.  In a chemoresistant-RCC cell line, inhibition of miR-21 via small mol. treatment rescued the expression of tumor-suppressor proteins and sensitized cells to topotecan-induced apoptosis.  This resulted in a >10-fold improvement in topotecan activity in cell viability and clonogenic assays.  Overall, this work reports a novel small mol. inhibitor for perturbing miR-21 function and demonstrates an approach to enhancing the potency of chemotherapeutics specifically for cancers derived from oncomiR addiction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwGmy5_whHK7Vg90H21EOLACvtfcHk0ljVvLkY9Rfr_g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1yrurrF&md5=2d752532ef82fcd3cbef8eaaa3238e2d</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01891&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01891%26sid%3Dliteratum%253Aachs%26aulast%3DNaro%26aufirst%3DY.%26aulast%3DAnkenbruck%26aufirst%3DN.%26aulast%3DThomas%26aufirst%3DM.%26aulast%3DTivon%26aufirst%3DY.%26aulast%3DConnelly%26aufirst%3DC.%2BM.%26aulast%3DGardner%26aufirst%3DL.%26aulast%3DDeiters%26aufirst%3DA.%26atitle%3DSmall%2520molecule%2520inhibition%2520of%2520microRNA%2520miR-21%2520rescues%2520chemosensitivity%2520of%2520renal-cell%2520carcinoma%2520to%2520topotecan%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D5900%26epage%3D5909%26doi%3D10.1021%2Facs.jmedchem.7b01891" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ankenbruck, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumbhare, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naro, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gardner, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emanuelson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deiters, A.</span></span> <span> </span><span class="NLM_article-title">Small molecule inhibition of microRNA-21 expression reduces cell viability and microtumor formation</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">3735</span>â <span class="NLM_lpage">3743</span>, <span class="refDoi">Â DOI: 10.1016/j.bmc.2019.05.044</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1016%2Fj.bmc.2019.05.044" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=31262664" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC1MXht1ynu7jP" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2019&pages=3735-3743&author=N.+Ankenbruckauthor=R.+Kumbhareauthor=Y.+Naroauthor=M.+Thomasauthor=L.+Gardnerauthor=C.+Emanuelsonauthor=A.+Deiters&title=Small+molecule+inhibition+of+microRNA-21+expression+reduces+cell+viability+and+microtumor+formation&doi=10.1016%2Fj.bmc.2019.05.044"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Small molecule inhibition of microRNA-21 expression reduces cell viability and microtumor formation</span></div><div class="casAuthors">Ankenbruck, Nicholas; Kumbhare, Rohan; Naro, Yuta; Thomas, Meryl; Gardner, Laura; Emanuelson, Cole; Deiters, Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">3735-3743</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">MicroRNAs (miRNAs) are short, non-coding RNA mols. estd. to regulate expression of a large no. of protein-coding genes and are implicated in a variety of biol. processes such as development, differentiation, proliferation, and cell survival.  Dysregulation of miRNAs has been attributed to the onset and progression of various human diseases, including cancer.  MicroRNA-21 (miR-21), one of the most established oncogenic miRNAs, is found to be upregulated in a wide range of cancers making it an attractive therapeutic target.  Employment of a luciferase-based live-cell reporter assay in a high-throughput screen of >300,000 small mols. led to the discovery of a new class of ether-amide miR-21 inhibitors.  Following a structure-activity relationship study, an optimized lead mol. was found to inhibit miR-21 transcription.  Furthermore, the inhibitor demonstrated cytotoxicity in a cervical cancer cell line via induction of apoptosis and was capable of reducing microtumor formation in a long-term clonogenic assay.  Altogether, this work reports the discovery of a new small mol. inhibitor of miR-21 and demonstrates its potential as an alternative approach in cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoQMXpXY6aQH7Vg90H21EOLACvtfcHk0ljiwO6vCiZUjQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXht1ynu7jP&md5=dfa9d5c537cfa01f14a34c4ff784b4ea</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2019.05.044&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2019.05.044%26sid%3Dliteratum%253Aachs%26aulast%3DAnkenbruck%26aufirst%3DN.%26aulast%3DKumbhare%26aufirst%3DR.%26aulast%3DNaro%26aufirst%3DY.%26aulast%3DThomas%26aufirst%3DM.%26aulast%3DGardner%26aufirst%3DL.%26aulast%3DEmanuelson%26aufirst%3DC.%26aulast%3DDeiters%26aufirst%3DA.%26atitle%3DSmall%2520molecule%2520inhibition%2520of%2520microRNA-21%2520expression%2520reduces%2520cell%2520viability%2520and%2520microtumor%2520formation%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2019%26volume%3D27%26spage%3D3735%26epage%3D3743%26doi%3D10.1016%2Fj.bmc.2019.05.044" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhattarai, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, Z. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, F. S.</span></span> <span> </span><span class="NLM_article-title">Regulating miRNA-21 biogenesis by bifunctional small molecules</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>139</i></span>,  <span class="NLM_fpage">4987</span>â <span class="NLM_lpage">4990</span>, <span class="refDoi">Â DOI: 10.1021/jacs.7b00610</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.7b00610" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC2sXktFOmu7k%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=139&publication_year=2017&pages=4987-4990&author=H.+Yanauthor=U.+Bhattaraiauthor=Z.+F.+Guoauthor=F.+S.+Liang&title=Regulating+miRNA-21+biogenesis+by+bifunctional+small+molecules&doi=10.1021%2Fjacs.7b00610"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Regulating miRNA-21 Biogenesis By Bifunctional Small Molecules</span></div><div class="casAuthors">Yan, Hao; Bhattarai, Umesh; Guo, Zhi-Fo; Liang, Fu-Sen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">139</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">4987-4990</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We report a new strategy to regulate microRNAs (miRNAs) biogenesis by using bifunctional small mols. that consist of a pre-miRNA binding unit connected by a linker to a Dicer inhibiting unit.  In this effort, fluorescence polarization-based screening was used to identify neomycin as a pre-miR-21 binding ligand.  Although neomycin cannot inhibit miR-21 maturation, linking it to the RNase inhibitor 1 forms the bifunctional conjugate 7A, which inhibits the prodn. of miR-21.  We expect that this strategy will be applicable to design other mols. for miRNA regulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHWiyPB0E-xLVg90H21EOLACvtfcHk0ljiwO6vCiZUjQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXktFOmu7k%253D&md5=ce221c2e953cded2e2a35c3484ee7fd4</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Fjacs.7b00610&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.7b00610%26sid%3Dliteratum%253Aachs%26aulast%3DYan%26aufirst%3DH.%26aulast%3DBhattarai%26aufirst%3DU.%26aulast%3DGuo%26aufirst%3DZ.%2BF.%26aulast%3DLiang%26aufirst%3DF.%2BS.%26atitle%3DRegulating%2520miRNA-21%2520biogenesis%2520by%2520bifunctional%2520small%2520molecules%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2017%26volume%3D139%26spage%3D4987%26epage%3D4990%26doi%3D10.1021%2Fjacs.7b00610" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nieto, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, R. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jackson, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thiery, J. P.</span></span> <span> </span><span class="NLM_article-title">EMT: 2016</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>166</i></span>,  <span class="NLM_fpage">21</span>â <span class="NLM_lpage">45</span>, <span class="refDoi">Â DOI: 10.1016/j.cell.2016.06.028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1016%2Fj.cell.2016.06.028" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=27368099" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFaitLbF" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=166&publication_year=2016&pages=21-45&author=M.+A.+Nietoauthor=R.+Y.+Huangauthor=R.+A.+Jacksonauthor=J.+P.+Thiery&title=EMT%3A+2016&doi=10.1016%2Fj.cell.2016.06.028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">EMT: 2016</span></div><div class="casAuthors">Nieto, M. Angela; Huang, Ruby Yun-Ju; Jackson, Rebecca A.; Thiery, Jean Paul</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">166</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">21-45</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  The significant parallels between cell plasticity during embryonic development and carcinoma progression have helped us understand the importance of the epithelial-mesenchymal transition (EMT) in human disease.  Our expanding knowledge of EMT has led to a clarification of the EMT program as a set of multiple and dynamic transitional states between the epithelial and mesenchymal phenotypes, as opposed to a process involving a single binary decision.  EMT and its intermediate states have recently been identified as crucial drivers of organ fibrosis and tumor progression, although there is some need for caution when interpreting its contribution to metastatic colonization.  Here, we discuss the current state-of-the-art and latest findings regarding the concept of cellular plasticity and heterogeneity in EMT.  We raise some of the questions pending and identify the challenges faced in this fast-moving field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1EILVAUm1ALVg90H21EOLACvtfcHk0ljiwO6vCiZUjQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtFaitLbF&md5=f8c2b1ff3efba49dcc63ad122fa188c8</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2016.06.028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2016.06.028%26sid%3Dliteratum%253Aachs%26aulast%3DNieto%26aufirst%3DM.%2BA.%26aulast%3DHuang%26aufirst%3DR.%2BY.%26aulast%3DJackson%26aufirst%3DR.%2BA.%26aulast%3DThiery%26aufirst%3DJ.%2BP.%26atitle%3DEMT%253A%25202016%26jtitle%3DCell%26date%3D2016%26volume%3D166%26spage%3D21%26epage%3D45%26doi%3D10.1016%2Fj.cell.2016.06.028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Qi, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bart, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Platteel, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sluis, T. V. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huitema, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harms, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hollema, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">V. D. Berg, A. V. D.</span></span> <span> </span><span class="NLM_article-title">Expression of miR-21 and its targets (PTEN, PDCD4, TM1) in flat epithelial atypia of the breast in relation to ductal carcinoma in situ and invasive carcinoma</span>. <i>BMC Cancer</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">163</span>â <span class="NLM_lpage">171</span>, <span class="refDoi">Â DOI: 10.1186/1471-2407-9-163</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1186%2F1471-2407-9-163" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=19473551" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A280%3ADC%252BD1MvhtF2hug%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2009&pages=163-171&author=L.+Qiauthor=J.+Bartauthor=L.+P.+Tanauthor=L.+Platteelauthor=T.+V.+D.+Sluisauthor=S.+Huitemaauthor=G.+Harmsauthor=L.+Fuauthor=H.+Hollemaauthor=A.+V.+D.+V.+D.+Berg&title=Expression+of+miR-21+and+its+targets+%28PTEN%2C+PDCD4%2C+TM1%29+in+flat+epithelial+atypia+of+the+breast+in+relation+to+ductal+carcinoma+in+situ+and+invasive+carcinoma&doi=10.1186%2F1471-2407-9-163"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Expression of miR-21 and its targets (PTEN, PDCD4, TM1) in flat epithelial atypia of the breast in relation to ductal carcinoma in situ and invasive carcinoma</span></div><div class="casAuthors">Qi Liqiang; Bart Joost; Tan Lu Ping; Platteel Inge; Sluis Tineke van der; Huitema Sippie; Harms Geert; Fu Li; Hollema Harry; Berg Anke van den</div><div class="citationInfo"><span class="NLM_cas:title">BMC cancer</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">163</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Flat epithelial atypia (FEA) of the breast is characterised by a few layers of mildly atypical luminal epithelial cells.  Genetic changes found in ductal carcinoma in situ (DCIS) and invasive ductal breast cancer (IDC) are also found in FEA, albeit at a lower concentration.  So far, miRNA expression changes associated with invasive breast cancer, like miR-21, have not been studied in FEA.  METHODS:  We performed miRNA in-situ hybridization (ISH) on 15 cases with simultaneous presence of normal breast tissue, FEA and/or DCIS and 17 additional cases with IDC.  Expression of the miR-21 targets PDCD4, TM1 and PTEN was investigated by immunohistochemistry.  RESULTS:  Two out of fifteen cases showed positive staining for miR-21 in normal breast ductal epithelium, seven out of fifteen cases were positive in the FEA component and nine out of twelve cases were positive in the DCIS component.  A positive staining of miR-21 was observed in 15 of 17 IDC cases.  In 12 cases all three components were present in one tissue block and an increase of miR-21 from normal breast to FEA and to DCIS was observed in five cases.  In three cases the FEA component was negative, whereas the DCIS component was positive for miR-21.  In three other cases, normal, FEA and DCIS components were negative for miR-21 and in the last case all three components were positive.  Overall we observed a gradual increase in percentage of miR-21 positive cases from normal, to FEA, DCIS and IDC.  Immunohistochemical staining for PTEN revealed no obvious changes in staining intensities in normal, FEA, DCIS and IDC.  Cytoplasmic staining of PDCD4 increased from normal to IDC, whereas, the nuclear staining decreased.  TM1 staining decreased from positive in normal breast to negative in most DCIS and IDC cases.  In FEA, the staining pattern for TM1 was similar to normal breast tissue.  CONCLUSION:  Upregulation of miR-21 from normal ductal epithelial cells of the breast to FEA, DCIS and IDC parallels morphologically defined carcinogenesis.  No clear relation was observed between the staining pattern of miR-21 and its previously reported target genes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcT7QDKBZgnzoAwxfk_AGB-EfW6udTcc2eaMS1GELv0Ebrntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1MvhtF2hug%253D%253D&md5=978c074b2fc6730464f4046f3904b37b</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1186%2F1471-2407-9-163&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1471-2407-9-163%26sid%3Dliteratum%253Aachs%26aulast%3DQi%26aufirst%3DL.%26aulast%3DBart%26aufirst%3DJ.%26aulast%3DTan%26aufirst%3DL.%2BP.%26aulast%3DPlatteel%26aufirst%3DL.%26aulast%3DSluis%26aufirst%3DT.%2BV.%2BD.%26aulast%3DHuitema%26aufirst%3DS.%26aulast%3DHarms%26aufirst%3DG.%26aulast%3DFu%26aufirst%3DL.%26aulast%3DHollema%26aufirst%3DH.%26aulast%3DV.%2BD.%2BBerg%26aufirst%3DA.%2BV.%2BD.%26atitle%3DExpression%2520of%2520miR-21%2520and%2520its%2520targets%2520%2528PTEN%252C%2520PDCD4%252C%2520TM1%2529%2520in%2520flat%2520epithelial%2520atypia%2520of%2520the%2520breast%2520in%2520relation%2520to%2520ductal%2520carcinoma%2520in%2520situ%2520and%2520invasive%2520carcinoma%26jtitle%3DBMC%2520Cancer%26date%3D2009%26volume%3D9%26spage%3D163%26epage%3D171%26doi%3D10.1186%2F1471-2407-9-163" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Degyatoreva, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kukielski, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Story, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhaduri, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maiti, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nahar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ray, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arya, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maiti, S.</span></span> <span> </span><span class="NLM_article-title">Targeting miRNA by tunable small molecule binders: peptidic aminosugar mediated interference in miR-21 biogenesis reverts epithelial to mesenchymal transition</span>. <i>MedChemComm</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1147</span>â <span class="NLM_lpage">1154</span>, <span class="refDoi">Â DOI: 10.1039/C8MD00092A</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1039%2FC8MD00092A" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=30109002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtFCmtr7L" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=1147-1154&author=A.+Ghoshauthor=N.+Degyatorevaauthor=C.+Kukielskiauthor=S.+Storyauthor=S.+Bhaduriauthor=K.+Maitiauthor=S.+Naharauthor=A.+Rayauthor=D.+P.+Aryaauthor=S.+Maiti&title=Targeting+miRNA+by+tunable+small+molecule+binders%3A+peptidic+aminosugar+mediated+interference+in+miR-21+biogenesis+reverts+epithelial+to+mesenchymal+transition&doi=10.1039%2FC8MD00092A"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting miRNA by tunable small molecule binders: peptidic aminosugar mediated interference in miR-21 biogenesis reverts epithelial to mesenchymal transition</span></div><div class="casAuthors">Ghosh, Arpita; Degyatoreva, Natalya; Kukielski, Casey; Story, Sandra; Bhaduri, Sayantan; Maiti, Krishnagopal; Nahar, Smita; Ray, Arjun; Arya, Dev P.; Maiti, Souvik</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1147-1154</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Epithelial to mesenchymal transition (EMT) is a process in which epithelial cells lose cell polarity and cell-cell adhesion and gain migratory and invasive properties to become mesenchymal cells that are very vital for development, wound healing and stem cell behavior and contribute pathol. to fibrosis and cancer progression. miR21, a potent regulator of the tumor suppressor gene PTEN, can be silenced to reverse EMT, thereby providing an attractive target for abrogating the malignant behavior of breast cancer.  Here, we report the design, synthesis and binding of a peptidic-aminoglycoside (PA) based chem. library against pre-miR21 that led to the identification of a group of small mols. that bind to pre-miR21 with high affinities and antagonize miR-21 maturation and function, thereby reversing EMT.  The approach described here offers a promising miRNA targeting platform where such aminosugar conjugates can be similarly used to target other oncogenic miRNAs.  Minor changes in the amino acid sequence allow us to tailor the binding effectiveness and downstream biol. effects, thus making this approach a potentially tunable method of regulation of miRNA function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOPwbV4QOxT7Vg90H21EOLACvtfcHk0lgR10va7nfJ_g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtFCmtr7L&md5=25dc0c6d22cedb13d6eabcc1c7298443</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1039%2FC8MD00092A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC8MD00092A%26sid%3Dliteratum%253Aachs%26aulast%3DGhosh%26aufirst%3DA.%26aulast%3DDegyatoreva%26aufirst%3DN.%26aulast%3DKukielski%26aufirst%3DC.%26aulast%3DStory%26aufirst%3DS.%26aulast%3DBhaduri%26aufirst%3DS.%26aulast%3DMaiti%26aufirst%3DK.%26aulast%3DNahar%26aufirst%3DS.%26aulast%3DRay%26aufirst%3DA.%26aulast%3DArya%26aufirst%3DD.%2BP.%26aulast%3DMaiti%26aufirst%3DS.%26atitle%3DTargeting%2520miRNA%2520by%2520tunable%2520small%2520molecule%2520binders%253A%2520peptidic%2520aminosugar%2520mediated%2520interference%2520in%2520miR-21%2520biogenesis%2520reverts%2520epithelial%2520to%2520mesenchymal%2520transition%26jtitle%3DMedChemComm%26date%3D2018%26volume%3D9%26spage%3D1147%26epage%3D1154%26doi%3D10.1039%2FC8MD00092A" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Watkins, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nahar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maiti, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arya, D. P.</span></span> <span> </span><span class="NLM_article-title">A pH sensitive high-throughput assay for miRNA binding of a peptide-aminoglycoside (PA) library</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>, <span class="NLM_elocation-id">e0144251</span> <span class="refDoi">Â DOI: 10.1371/journal.pone.0144251</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1371%2Fjournal.pone.0144251" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=26656788" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC28XksF2gsbs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&author=D.+Watkinsauthor=L.+Jiangauthor=S.+Naharauthor=S.+Maitiauthor=D.+P.+Arya&title=A+pH+sensitive+high-throughput+assay+for+miRNA+binding+of+a+peptide-aminoglycoside+%28PA%29+library&doi=10.1371%2Fjournal.pone.0144251"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">A pH sensitive high-throughput assay for miRNA binding of a peptide-aminoglycoside (PA) library</span></div><div class="casAuthors">Watkins, Derrick; Jiang, Liuwei; Nahar, Smita; Maiti, Souvik; Arya, Dev P.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">e0144251/1-e0144251/23</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">MicroRNAs (miRNA) are small RNAs that have a regulatory role in gene expression.  Because of this regulatory role, miRNAs have become a new target for therapeutic compds.  Here, we outline an approach to target specific miRNAs using a high throughput capable assay and a 215 compd. peptidic-aminosugar (PA) library.  Aminosugars have been shown in a no. of recent reports as important lead compds. that bind miRNA.  In order to screen for compds. that bind miRNA, we have developed a high throughput displacement assay using a fluorescein-neomycin conjugated mol. (F-neo) as a probe for competitive miRNA binding compds.  We have applied the F-neo assay to four different miRNA constructs and the assay is applicable to most miRNAs, at various stages of processing.  The results of the screen were validated by the detn. of the IC50 for a select group of compds. from the library.  For example, we identified eight compds. that bind to hsa-miR 504 with higher affinity than the parent neomycin.  From the F-neo displacement assay we found that the no. of binding sites differs for each miRNA, and the binding sites appear to differ both phys. and chem., with different affinity of the compds. resulting from the size of the mol. as well as the chem. structure.  Addnl., the affinity of the compds. was dependent on the identity and position of the amino acid position of conjugation and the affinity of the compds. relative to other compds. in the library was miRNA dependent with the introduction of a second amino acid.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpP7dNoUHy7zbVg90H21EOLACvtfcHk0lhMB10DLtfPmg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XksF2gsbs%253D&md5=00d931a54eb439f22e612ea587cfb1ee</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0144251&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0144251%26sid%3Dliteratum%253Aachs%26aulast%3DWatkins%26aufirst%3DD.%26aulast%3DJiang%26aufirst%3DL.%26aulast%3DNahar%26aufirst%3DS.%26aulast%3DMaiti%26aufirst%3DS.%26aulast%3DArya%26aufirst%3DD.%2BP.%26atitle%3DA%2520pH%2520sensitive%2520high-throughput%2520assay%2520for%2520miRNA%2520binding%2520of%2520a%2520peptide-aminoglycoside%2520%2528PA%2529%2520library%26jtitle%3DPLoS%2520One%26date%3D2015%26volume%3D10%26doi%3D10.1371%2Fjournal.pone.0144251" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Haggar, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boushey, R. P.</span></span> <span> </span><span class="NLM_article-title">Colorectal cancer epidemiology: incidence, mortality, survival, and risk factors</span>. <i>Clin. Colon. Rectal. Surg.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">191</span>â <span class="NLM_lpage">197</span>, <span class="refDoi">Â DOI: 10.1055/s-0029-1242458</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1055%2Fs-0029-1242458" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=21037809" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A280%3ADC%252BC3cbjsFWluw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2009&pages=191-197&author=F.+A.+Haggarauthor=R.+P.+Boushey&title=Colorectal+cancer+epidemiology%3A+incidence%2C+mortality%2C+survival%2C+and+risk+factors&doi=10.1055%2Fs-0029-1242458"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Colorectal cancer epidemiology: incidence, mortality, survival, and risk factors</span></div><div class="casAuthors">Haggar Fatima A; Boushey Robin P</div><div class="citationInfo"><span class="NLM_cas:title">Clinics in colon and rectal surgery</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">191-7</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">In this article, the incidence, mortality, and survival rates for colorectal cancer are reviewed, with attention paid to regional variations and changes over time.  A concise overview of known risk factors associated with colorectal cancer is provided, including familial and hereditary factors, as well as environmental lifestyle-related risk factors such as physical inactivity, obesity, smoking, and alcohol consumption.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSYLvT3tqMgUgaK-_ZudimnfW6udTcc2ean9t6L1t50E7ntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3cbjsFWluw%253D%253D&md5=392f9c8d22332f194121f0645405249c</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1055%2Fs-0029-1242458&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1055%252Fs-0029-1242458%26sid%3Dliteratum%253Aachs%26aulast%3DHaggar%26aufirst%3DF.%2BA.%26aulast%3DBoushey%26aufirst%3DR.%2BP.%26atitle%3DColorectal%2520cancer%2520epidemiology%253A%2520incidence%252C%2520mortality%252C%2520survival%252C%2520and%2520risk%2520factors%26jtitle%3DClin.%2520Colon.%2520Rectal.%2520Surg.%26date%3D2009%26volume%3D22%26spage%3D191%26epage%3D197%26doi%3D10.1055%2Fs-0029-1242458" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stiegelbauer, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perakis, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deutsch, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ling, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerger, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pichler, M.</span></span> <span> </span><span class="NLM_article-title">MicroRNAs as novel predictive biomarkers and therapeutic targets in colorectal cancer</span>. <i>World J. Gastroenterol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">11727</span>â <span class="NLM_lpage">11735</span>, <span class="refDoi">Â DOI: 10.3748/wjg.v20.i33.11727</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.3748%2Fwjg.v20.i33.11727" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=25206276" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvV2jt7vF" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2014&pages=11727-11735&author=V.+Stiegelbauerauthor=S.+Perakisauthor=A.+Deutschauthor=H.+Lingauthor=A.+Gergerauthor=M.+Pichler&title=MicroRNAs+as+novel+predictive+biomarkers+and+therapeutic+targets+in+colorectal+cancer&doi=10.3748%2Fwjg.v20.i33.11727"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">MicroRNAs as novel predictive biomarkers and therapeutic targets in colorectal cancer</span></div><div class="casAuthors">Stiegelbauer, Verena; Perakis, Samantha; Deutsch, Alexander; Ling, Hui; Gerger, Armin; Pichler, Martin</div><div class="citationInfo"><span class="NLM_cas:title">World Journal of Gastroenterology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">33</span>),
    <span class="NLM_cas:pages">11727-11735, 10 pp.</span>CODEN:
                <span class="NLM_cas:coden">WJGAF2</span>;
        ISSN:<span class="NLM_cas:issn">1007-9327</span>.
    
            (<span class="NLM_cas:orgname">Baishideng Publishing Group Inc.</span>)
        </div><div class="casAbstract">A review.  Colorectal cancer (CRC) is the third most common cancer in western countries.  Despite significant improvement in available treatment options, CRC still remains the second leading cause of cancer-related death.  Traditionally, 5-fluorouracil has been used as the main chemotherapy drug for treatment of metastatic CRC (mCRC).  However, during the last two decades more effective chemotherapeutic agents such as oxaliplatin, irinotecan and the monoclonal antibodies cetuximab, panitumumab and bevacizumab have been used in clin. practice.  More recently, the therapeutic armamentarium has been supplemented by the monoclonal antibodies bevacizumab, cetuximab and panitumumab as well as the protein-trap aflibercept and the small mol. multi-kinase inhibitor regorafenib.  One of the major problems for the management of CRC is the inherent or acquired resistance to therapeutic approaches.  The discovery of microRNAs (miRNAs), a class of small, endogenous, non-coding, single-stranded RNAs that play a role as post-transcriptional regulators, has added new dimensions to the diagnosis and treatment of cancer.  Because miRNAs are important regulators of carcinogenesis, progression, invasion, angiogenesis and metastases in CRC, they might serve as potential predictive and prognostic factors and even as therapeutic targets themselves.  Several miRNAs are already known to be dysregulated in CRCs and have been linked to biol. processes involved in tumor progression and response to anti-cancer therapies.  This review summarizes current therapeutic approaches for treating CRC and highlights the role of miRNAs as novel predictive biomarkers and potential drug targets in CRC patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9eb1QcDI5ZLVg90H21EOLACvtfcHk0lhMB10DLtfPmg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvV2jt7vF&md5=546238c5959835d22eeda95a213706ae</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.3748%2Fwjg.v20.i33.11727&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3748%252Fwjg.v20.i33.11727%26sid%3Dliteratum%253Aachs%26aulast%3DStiegelbauer%26aufirst%3DV.%26aulast%3DPerakis%26aufirst%3DS.%26aulast%3DDeutsch%26aufirst%3DA.%26aulast%3DLing%26aufirst%3DH.%26aulast%3DGerger%26aufirst%3DA.%26aulast%3DPichler%26aufirst%3DM.%26atitle%3DMicroRNAs%2520as%2520novel%2520predictive%2520biomarkers%2520and%2520therapeutic%2520targets%2520in%2520colorectal%2520cancer%26jtitle%3DWorld%2520J.%2520Gastroenterol.%26date%3D2014%26volume%3D20%26spage%3D11727%26epage%3D11735%26doi%3D10.3748%2Fwjg.v20.i33.11727" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lv, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bie, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, F.</span></span> <span> </span><span class="NLM_article-title">Direct binding of microRNA-21 pre-element with Regorafenib: An alternative mechanism for anti-colorectal cancer chemotherapy?</span>. <i>J. Mol. Graphics Modell.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>73</i></span>,  <span class="NLM_fpage">48</span>â <span class="NLM_lpage">53</span>, <span class="refDoi">Â DOI: 10.1016/j.jmgm.2017.02.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1016%2Fj.jmgm.2017.02.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=28236743" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjtlKjtbs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2017&pages=48-53&author=X.+Chenauthor=B.+Xieauthor=L.+Caoauthor=F.+Zhuauthor=B.+Chenauthor=H.+Lvauthor=X.+Fanauthor=L.+Hanauthor=L.+Bieauthor=X.+Caoauthor=X.+Shenauthor=F.+Cao&title=Direct+binding+of+microRNA-21+pre-element+with+Regorafenib%3A+An+alternative+mechanism+for+anti-colorectal+cancer+chemotherapy%3F&doi=10.1016%2Fj.jmgm.2017.02.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Direct binding of microRNA-21 pre-element with Regorafenib: An alternative mechanism for anti-colorectal cancer chemotherapy?</span></div><div class="casAuthors">Chen, Xiaobing; Xie, Bojian; Cao, Liang; Zhu, Feng; Chen, Beibei; Lv, Huifang; Fan, Xingxing; Han, Lili; Bie, Liangyu; Cao, Xinguang; Shen, Xiaokun; Cao, Feilin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Graphics & Modelling</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">73</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">48-53</span>CODEN:
                <span class="NLM_cas:coden">JMGMFI</span>;
        ISSN:<span class="NLM_cas:issn">1093-3263</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The Regorafenib is a broad-spectrum kinase inhibitor that has been approved to treat colorectal cancer (CRC).  However, evidences have shown that the agent is also implicated in drug interaction with microRNA-21 (miR-21), an oncogenic miRNA which plays a key role in resisting programmed cell death in CRC cells.  Here, we supposed that, instead of kinase inhibition, Regorafenib can directly bind to and then stabilize miR-21 pre-element, thus preventing RNase Dicer-meditated cleavage of the pre-element to mature miR-21.  In order to verify the notion, an in silico-in vitro integrated investigation of the direct intermol. interaction between Regorafenib and miR-21 pre-element was performed by using active pocket identification, RNA-ligand docking, mol. dynamics (MD) simulation, binding energetic anal., and fluorescence-based assay.  It was revealed that the Regorafenib can bind at the major groove-like stem region of miR-21 pre-element through three geometrically satisfactory hydrogen bonds (H-bonds) as well as a no. of hydrophobic forces and Ï-Ï stacking, conferring strong specificity and high stability to the RNA-ligand complex system (Kd = 0.73 Î¼M).  Sep. inversion mutation of two base pairs (G6C, C12G) and (A13U, U4A) that are involved in the H-bonding can considerably impair the affinity of Regorafenib to miR-21 pre-element, with Kd increase to 27 and 96 Î¼M, resp.  All these supported that Regorafenib can directly bind to miR-21 pre-element at mol. level and the binding mode can be properly modeled by using the proposed integrated strategy.  This study would provide a potential, alternative mechanism for anti-colorectal cancer chemotherapy with Regorafenib.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoRvOCg1wgVd7Vg90H21EOLACvtfcHk0lhMB10DLtfPmg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjtlKjtbs%253D&md5=b772e412cd20be6de42957b0bc0732c8</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1016%2Fj.jmgm.2017.02.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmgm.2017.02.004%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DX.%26aulast%3DXie%26aufirst%3DB.%26aulast%3DCao%26aufirst%3DL.%26aulast%3DZhu%26aufirst%3DF.%26aulast%3DChen%26aufirst%3DB.%26aulast%3DLv%26aufirst%3DH.%26aulast%3DFan%26aufirst%3DX.%26aulast%3DHan%26aufirst%3DL.%26aulast%3DBie%26aufirst%3DL.%26aulast%3DCao%26aufirst%3DX.%26aulast%3DShen%26aufirst%3DX.%26aulast%3DCao%26aufirst%3DF.%26atitle%3DDirect%2520binding%2520of%2520microRNA-21%2520pre-element%2520with%2520Regorafenib%253A%2520An%2520alternative%2520mechanism%2520for%2520anti-colorectal%2520cancer%2520chemotherapy%253F%26jtitle%3DJ.%2520Mol.%2520Graphics%2520Modell.%26date%3D2017%26volume%3D73%26spage%3D48%26epage%3D53%26doi%3D10.1016%2Fj.jmgm.2017.02.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Costales, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aikawa, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Childs-Disney, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abegg, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoch, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Velagapudi, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khan, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yildirim, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adibekian, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, E. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span> <span> </span><span class="NLM_article-title">Small-molecule targeted recruitment of a nuclease to cleave an oncogenic RNA in a mouse model of metastatic cancer</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>117</i></span>,  <span class="NLM_fpage">2406</span>â <span class="NLM_lpage">2411</span>, <span class="refDoi">Â DOI: 10.1073/pnas.1914286117</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1073%2Fpnas.1914286117" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=31964809" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BB3cXisVWisL0%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=117&publication_year=2020&pages=2406-2411&author=M.+G.+Costalesauthor=H.+Aikawaauthor=Y.+Liauthor=J.+L.+Childs-Disneyauthor=D.+Abeggauthor=D.+G.+Hochauthor=S.+P.+Velagapudiauthor=Y.+Nakaiauthor=T.+Khanauthor=K.+W.+Wangauthor=I.+Yildirimauthor=A.+Adibekianauthor=E.+T.+Wangauthor=M.+D.+Disney&title=Small-molecule+targeted+recruitment+of+a+nuclease+to+cleave+an+oncogenic+RNA+in+a+mouse+model+of+metastatic+cancer&doi=10.1073%2Fpnas.1914286117"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Small-molecule targeted recruitment of a nuclease to cleave an oncogenic RNA in a mouse model of metastatic cancer</span></div><div class="casAuthors">Costales, Matthew G.; Aikawa, Haruo; Li, Yue; Childs-Disney, Jessica L.; Abegg, Daniel; Hoch, Dominic G.; Velagapudi, Sai Pradeep; Nakai, Yoshio; Khan, Tanya; Wang, Kye Won; Yildirim, Ilyas; Adibekian, Alexander; Wang, Eric T.; Disney, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">117</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2406-2411</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">As the area of small mols. interacting with RNA advances, general routes to provide bioactive compds. are needed as ligands can bind RNA avidly to sites that will not affect function.  Small-mol. targeted RNA degrdn. will thus provide a general route to affect RNA biol.  A non-oligonucleotide-contg. compd. was designed from sequence to target the precursor to oncogenic microRNA-21 (pre-miR-21) for enzymic destruction with selectivity that can exceed that for protein-targeted medicines.  The compd. specifically binds the target and contains a heterocycle that recruits and activates a RNase to pre-miR-21 to substoichiometrically effect its cleavage and subsequently impede metastasis of breast cancer to lung in a mouse model.  Transcriptomic and proteomic analyses demonstrate that the compd. is potent and selective, specifically modulating oncogenic pathways.  Thus, small mols. can be designed from sequence to have all of the functional repertoire of oligonucleotides, including inducing enzymic degrdn., and to selectively and potently modulate RNA function in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGolzufxJ0-LLLVg90H21EOLACvtfcHk0li-LtAnvW9HGA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXisVWisL0%253D&md5=450996be5aadaa9e13acb5c2cfb03c62</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1914286117&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1914286117%26sid%3Dliteratum%253Aachs%26aulast%3DCostales%26aufirst%3DM.%2BG.%26aulast%3DAikawa%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DChilds-Disney%26aufirst%3DJ.%2BL.%26aulast%3DAbegg%26aufirst%3DD.%26aulast%3DHoch%26aufirst%3DD.%2BG.%26aulast%3DVelagapudi%26aufirst%3DS.%2BP.%26aulast%3DNakai%26aufirst%3DY.%26aulast%3DKhan%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DK.%2BW.%26aulast%3DYildirim%26aufirst%3DI.%26aulast%3DAdibekian%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DE.%2BT.%26aulast%3DDisney%26aufirst%3DM.%2BD.%26atitle%3DSmall-molecule%2520targeted%2520recruitment%2520of%2520a%2520nuclease%2520to%2520cleave%2520an%2520oncogenic%2520RNA%2520in%2520a%2520mouse%2520model%2520of%2520metastatic%2520cancer%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2020%26volume%3D117%26spage%3D2406%26epage%3D2411%26doi%3D10.1073%2Fpnas.1914286117" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Velagapudi, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gallo, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span> <span> </span><span class="NLM_article-title">Sequence-based design of bioactive small molecules that target precursor microRNAs</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">291</span>â <span class="NLM_lpage">299</span>, <span class="refDoi">Â DOI: 10.1038/nchembio.1452</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1038%2Fnchembio.1452" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=24509821" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitFaksbo%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=291-299&author=S.+P.+Velagapudiauthor=S.+M.+Galloauthor=M.+D.+Disney&title=Sequence-based+design+of+bioactive+small+molecules+that+target+precursor+microRNAs&doi=10.1038%2Fnchembio.1452"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Sequence-based design of bioactive small molecules that target precursor microRNAs</span></div><div class="casAuthors">Velagapudi, Sai Pradeep; Gallo, Steven M.; Disney, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">291-297</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Oligonucleotides are designed to target RNA using base pairing rules, but they can be hampered by poor cellular delivery and nonspecific stimulation of the immune system.  Small mols. are preferred as lead drugs or probes but cannot be designed from sequence.  Herein, we describe an approach termed Inforna that designs lead small mols. for RNA from solely sequence.  Inforna was applied to all human microRNA hairpin precursors, and it identified bioactive small mols. that inhibit biogenesis by binding nuclease-processing sites (44% hit rate).  Among 27 lead interactions, the most avid interaction is between a benzimidazole (1) and precursor microRNA-96.  Compd. 1 selectively inhibits biogenesis of microRNA-96, upregulating a protein target (FOXO1) and inducing apoptosis in cancer cells.  Apoptosis is ablated when FOXO1 mRNA expression is knocked down by an siRNA, validating compd. selectivity.  Markedly, microRNA profiling shows that 1 only affects microRNA-96 biogenesis and is at least as selective as an oligonucleotide.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqy1YQeIYltbrVg90H21EOLACvtfcHk0li-LtAnvW9HGA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitFaksbo%253D&md5=d57f6a2ca7d6784017e2adff9ddb2f55</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1452&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1452%26sid%3Dliteratum%253Aachs%26aulast%3DVelagapudi%26aufirst%3DS.%2BP.%26aulast%3DGallo%26aufirst%3DS.%2BM.%26aulast%3DDisney%26aufirst%3DM.%2BD.%26atitle%3DSequence-based%2520design%2520of%2520bioactive%2520small%2520molecules%2520that%2520target%2520precursor%2520microRNAs%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2014%26volume%3D10%26spage%3D291%26epage%3D299%26doi%3D10.1038%2Fnchembio.1452" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cho, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shin, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koo, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, K. S.</span></span> <span> </span><span class="NLM_article-title">miR-372 regulates cell cycle and apoptosis of AGS human gastric cancer cell line through direct regulation of LATS2</span>. <i>Mol. Cells</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">521</span>â <span class="NLM_lpage">527</span>, <span class="refDoi">Â DOI: 10.1007/s10059-009-0158-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1007%2Fs10059-009-0158-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=19937137" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC3cXisV2nsw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2009&pages=521-527&author=W.+J.+Choauthor=J.+M.+Shinauthor=J.+S.+Kimauthor=M.+R.+Leeauthor=K.+S.+Hongauthor=J.+H.+Leeauthor=K.+H.+Kooauthor=J.+W.+Parkauthor=K.+S.+Kim&title=miR-372+regulates+cell+cycle+and+apoptosis+of+AGS+human+gastric+cancer+cell+line+through+direct+regulation+of+LATS2&doi=10.1007%2Fs10059-009-0158-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">miR-372 regulates cell cycle and apoptosis of AGS human gastric cancer cell line through direct regulation of LATS2</span></div><div class="casAuthors">Cho, Wha Ja; Shin, Jeong Min; Kim, Jong Soo; Lee, Man Ryul; Hong, Ki Sung; Lee, Jun-Ho; Koo, Kyoung Hwa; Park, Jeong Woo; Kim, Kye-Seong</div><div class="citationInfo"><span class="NLM_cas:title">Molecules and Cells</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">521-527</span>CODEN:
                <span class="NLM_cas:coden">MOCEEK</span>;
        ISSN:<span class="NLM_cas:issn">1016-8478</span>.
    
            (<span class="NLM_cas:orgname">Korean Society for Molecular and Cellular Biology</span>)
        </div><div class="casAbstract">Previously, we have reported tissue- and stage-specific expression of miR-372 in human embryonic stem cells and so far, not many reports speculate the function of this microRNA (miRNA).  In this study, we screened various human cancer cell lines including gastric cancer cell lines and found first time that miR-372 is expressed only in AGS human gastric adenocarcinoma cell line.  Inhibition of miR-372 using antisense miR-372 oligonucleotide (AS-miR-372) suppressed proliferation, arrested the cell cycle at G2/M phase, and increased apoptosis of AGS cells.  Furthermore, AS-miR-372 treatment increased expression of LATS2, while over-expression of miR-372 decreased luciferase reporter activity driven by the 3' untranslated region (3' UTR) of LATS2 mRNA.  Over-expression of LATS2 induced changes in AGS cells similar to those in AGS cells treated with AS-miR-372.  Taken together, these findings demonstrate an oncogenic role for miR-372 in controlling cell growth, cell cycle, and apoptosis through down-regulation of a tumor suppressor gene, LATS2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxrPqqSkqmZbVg90H21EOLACvtfcHk0lhb6kbq4co1xQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXisV2nsw%253D%253D&md5=400516a007dbd0ea5756d7f59ada12c9</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1007%2Fs10059-009-0158-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10059-009-0158-0%26sid%3Dliteratum%253Aachs%26aulast%3DCho%26aufirst%3DW.%2BJ.%26aulast%3DShin%26aufirst%3DJ.%2BM.%26aulast%3DKim%26aufirst%3DJ.%2BS.%26aulast%3DLee%26aufirst%3DM.%2BR.%26aulast%3DHong%26aufirst%3DK.%2BS.%26aulast%3DLee%26aufirst%3DJ.%2BH.%26aulast%3DKoo%26aufirst%3DK.%2BH.%26aulast%3DPark%26aufirst%3DJ.%2BW.%26aulast%3DKim%26aufirst%3DK.%2BS.%26atitle%3DmiR-372%2520regulates%2520cell%2520cycle%2520and%2520apoptosis%2520of%2520AGS%2520human%2520gastric%2520cancer%2520cell%2520line%2520through%2520direct%2520regulation%2520of%2520LATS2%26jtitle%3DMol.%2520Cells%26date%3D2009%26volume%3D28%26spage%3D521%26epage%3D527%26doi%3D10.1007%2Fs10059-009-0158-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Voorhoeve, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sage, C. l.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schrier, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gillis, A. J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stoop, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duijse, J. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drost, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griekspoor, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zlotorynski, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yabuta, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vita, G. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nojima, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Looijenga, L. H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agami, R.</span></span> <span> </span><span class="NLM_article-title">A genetic screen implicates miRNA-372 and miRNA-373 as oncogenes in testicular germ cell tumors</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>124</i></span>,  <span class="NLM_fpage">1169</span>â <span class="NLM_lpage">1181</span>, <span class="refDoi">Â DOI: 10.1016/j.cell.2006.02.037</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1016%2Fj.cell.2006.02.037" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=16564011" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BD28Xjt1Khsrg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=124&publication_year=2006&pages=1169-1181&author=P.+P.+Voorhoeveauthor=C.+l.+Sageauthor=M.+Schrierauthor=A.+J.+M.+Gillisauthor=H.+Stoopauthor=R.+Nagelauthor=Y.+P.+Liuauthor=J.+V.+Duijseauthor=J.+Drostauthor=A.+Griekspoorauthor=E.+Zlotorynskiauthor=N.+Yabutaauthor=G.+D.+Vitaauthor=H.+Nojimaauthor=L.+H.+J.+Looijengaauthor=R.+Agami&title=A+genetic+screen+implicates+miRNA-372+and+miRNA-373+as+oncogenes+in+testicular+germ+cell+tumors&doi=10.1016%2Fj.cell.2006.02.037"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">A genetic screen implicates miRNA-372 and miRNA-373 as oncogenes in testicular germ cell tumors</span></div><div class="casAuthors">Voorhoeve, P. Mathijs; le Sage, Carlos; Schrier, Mariette; Gillis, Ad J. M.; Stoop, Hans; Nagel, Remco; Liu, Ying-Poi; van Duijse, Josyanne; Drost, Jarno; Griekspoor, Alexander; Zlotorynski, Eitan; Yabuta, Norikazu; De Vita, Gabriella; Nojima, Hiroshi; Looijenga, Leendert H. J.; Agami, Reuven</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">124</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1169-1181</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Endogenous small RNAs (miRNAs) regulate gene expression by mechanisms conserved across metazoans.  While the no. of verified human miRNAs is still expanding, only few have been functionally annotated.  To perform genetic screens for novel functions of miRNAs, we developed a library of vectors expressing the majority of cloned human miRNAs and created corresponding DNA barcode arrays.  In a screen for miRNAs that cooperate with oncogenes in cellular transformation, we identified miR-372 and miR-373, each permitting proliferation and tumorigenesis of primary human cells that harbor both oncogenic RAS and active wild-type p53.  These miRNAs neutralize p53-mediated CDK inhibition, possibly through direct inhibition of the expression of the tumor-suppressor LATS2.  We provide evidence that these miRNAs are potential novel oncogenes participating in the development of human testicular germ cell tumors by numbing the p53 pathway, thus allowing tumorigenic growth in the presence of wild-type p53.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprKiyR8GUInrVg90H21EOLACvtfcHk0lhb6kbq4co1xQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xjt1Khsrg%253D&md5=b1793069efba1ae573f7e0e66a879c47</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2006.02.037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2006.02.037%26sid%3Dliteratum%253Aachs%26aulast%3DVoorhoeve%26aufirst%3DP.%2BP.%26aulast%3DSage%26aufirst%3DC.%2Bl.%26aulast%3DSchrier%26aufirst%3DM.%26aulast%3DGillis%26aufirst%3DA.%2BJ.%2BM.%26aulast%3DStoop%26aufirst%3DH.%26aulast%3DNagel%26aufirst%3DR.%26aulast%3DLiu%26aufirst%3DY.%2BP.%26aulast%3DDuijse%26aufirst%3DJ.%2BV.%26aulast%3DDrost%26aufirst%3DJ.%26aulast%3DGriekspoor%26aufirst%3DA.%26aulast%3DZlotorynski%26aufirst%3DE.%26aulast%3DYabuta%26aufirst%3DN.%26aulast%3DVita%26aufirst%3DG.%2BD.%26aulast%3DNojima%26aufirst%3DH.%26aulast%3DLooijenga%26aufirst%3DL.%2BH.%2BJ.%26aulast%3DAgami%26aufirst%3DR.%26atitle%3DA%2520genetic%2520screen%2520implicates%2520miRNA-372%2520and%2520miRNA-373%2520as%2520oncogenes%2520in%2520testicular%2520germ%2520cell%2520tumors%26jtitle%3DCell%26date%3D2006%26volume%3D124%26spage%3D1169%26epage%3D1181%26doi%3D10.1016%2Fj.cell.2006.02.037" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vo, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Staedel, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zehnacker, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benhida, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Darfeuille, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duca, M.</span></span> <span> </span><span class="NLM_article-title">Targeting the production of oncogenic microRNAs with multimodal synthetic small molecules</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">711</span>â <span class="NLM_lpage">721</span>, <span class="refDoi">Â DOI: 10.1021/cb400668h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb400668h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFOiur3I" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=711-721&author=D.+D.+Voauthor=C.+Staedelauthor=L.+Zehnackerauthor=R.+Benhidaauthor=F.+Darfeuilleauthor=M.+Duca&title=Targeting+the+production+of+oncogenic+microRNAs+with+multimodal+synthetic+small+molecules&doi=10.1021%2Fcb400668h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting the Production of Oncogenic MicroRNAs with Multimodal Synthetic Small Molecules</span></div><div class="casAuthors">Vo, Duc Duy; Staedel, Cathy; Zehnacker, Laura; Benhida, Rachid; Darfeuille, Fabien; Duca, Maria</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">711-721</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">MicroRNAs (miRNAs) are a recently discovered category of small RNA mols. that regulate gene expression at the post-transcriptional level.  Accumulating evidence indicates that miRNAs are aberrantly expressed in a variety of human cancers and revealed to be oncogenic and to play a pivotal role in initiation and progression of these pathologies.  It is now clear that the inhibition of oncogenic miRNAs, defined as blocking their biosynthesis or their function, could find an application in the therapy of different types of cancer in which these miRNAs are implicated.  Here the authors report the design, synthesis, and biol. evaluation of new small-mol. RNA ligands targeting the prodn. of oncogenic microRNAs.  The authors focused the authors' attention on miR-372 and miR-373 that are implicated in the tumorigenesis of different types of cancer such as gastric cancer.  These two oncogenic miRNAs are overexpressed in gastric cancer cells starting from their precursors pre-miR-372 and pre-miR-373, two stem-loop structured RNAs that lead to mature miRNAs after cleavage by the enzyme Dicer.  The small mols. described herein consist of the conjugation of two RNA binding motives, i.e., the aminoglycoside neomycin and different natural and artificial nucleobases, to obtain RNA ligands with increased affinity and selectivity compared to that of parent compds.  After the synthesis of this new series of RNA ligands, the authors demonstrated that they are able to inhibit the prodn. of the oncogenic miRNA-372 and -373 by binding their pre-miRNAs and inhibiting the processing by Dicer.  Moreover, the authors proved that some of these compds. bear anti-proliferative activity toward gastric cancer cells and that this activity is likely linked to a decrease in the prodn. of targeted miRNAs.  To date, only few examples of small mols. targeting oncogenic miRNAs have been reported, and such inhibitors could be extremely useful for the development of new anticancer therapeutic strategies as well as useful biochem. tools for the study of miRNAs' pathways and mechanisms.  Furthermore, this is the first time that a design based on current knowledge about RNA targeting is proposed to target miRNAs' prodn. with small mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqMnXqBRxfyNrVg90H21EOLACvtfcHk0lgGU5Bmg70_Xg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFOiur3I&md5=262eeeb54b8b743aefd61318f582aed2</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1021%2Fcb400668h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb400668h%26sid%3Dliteratum%253Aachs%26aulast%3DVo%26aufirst%3DD.%2BD.%26aulast%3DStaedel%26aufirst%3DC.%26aulast%3DZehnacker%26aufirst%3DL.%26aulast%3DBenhida%26aufirst%3DR.%26aulast%3DDarfeuille%26aufirst%3DF.%26aulast%3DDuca%26aufirst%3DM.%26atitle%3DTargeting%2520the%2520production%2520of%2520oncogenic%2520microRNAs%2520with%2520multimodal%2520synthetic%2520small%2520molecules%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2014%26volume%3D9%26spage%3D711%26epage%3D721%26doi%3D10.1021%2Fcb400668h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Malnuit, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duca, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benhida, R.</span></span> <span> </span><span class="NLM_article-title">Targeting DNA base pair mismatch with artificial nucleobases. Advances and perspectives in triple helix strategy</span>. <i>Org. Biomol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">326</span>â <span class="NLM_lpage">336</span>, <span class="refDoi">Â DOI: 10.1039/C0OB00418A</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1039%2FC0OB00418A" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=21046036" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1SrsbnN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2011&pages=326-336&author=V.+Malnuitauthor=M.+Ducaauthor=R.+Benhida&title=Targeting+DNA+base+pair+mismatch+with+artificial+nucleobases.+Advances+and+perspectives+in+triple+helix+strategy&doi=10.1039%2FC0OB00418A"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting DNA base pair mismatch with artificial nucleobases: Advances and perspectives in triple helix strategy</span></div><div class="casAuthors">Malnuit, Vincent; Duca, Maria; Benhida, Rachid</div><div class="citationInfo"><span class="NLM_cas:title">Organic & Biomolecular Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">326-336</span>CODEN:
                <span class="NLM_cas:coden">OBCRAK</span>;
        ISSN:<span class="NLM_cas:issn">1477-0520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  This review, divided into three sections, describes the contribution of the chemists' community to the development and application of triple helix strategy by using artificial nucleic acids, particularly for the recognition of DNA sequences incorporating base pair inversions.  Firstly, the development of nucleobases that recognize CG inversion is surveyed followed secondly by specific recognition of TA inverted base pair.  Finally, we point out in the last section recent perspectives and applications, driven from knowledge in nucleic acids interactions, in the growing field of nanotechnol. and supramol. chem. at the border area of physics, chem. and mol. biol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGriqAimgo1nfLVg90H21EOLACvtfcHk0lgGU5Bmg70_Xg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1SrsbnN&md5=fa698f0c67f4cb1fa20dcc96a152d312</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1039%2FC0OB00418A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC0OB00418A%26sid%3Dliteratum%253Aachs%26aulast%3DMalnuit%26aufirst%3DV.%26aulast%3DDuca%26aufirst%3DM.%26aulast%3DBenhida%26aufirst%3DR.%26atitle%3DTargeting%2520DNA%2520base%2520pair%2520mismatch%2520with%2520artificial%2520nucleobases.%2520Advances%2520and%2520perspectives%2520in%2520triple%2520helix%2520strategy%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2011%26volume%3D9%26spage%3D326%26epage%3D336%26doi%3D10.1039%2FC0OB00418A" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vo, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tran, T. P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Staedel, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benhida, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Darfeuille, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Giorgio, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duca, M.</span></span> <span> </span><span class="NLM_article-title">Oncogenic microRNAs biogenesis as a drug target: Structure-activity relationship studies on new aminoglycoside conjugates</span>. <i>Chem. - Eur. J.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">5350</span>â <span class="NLM_lpage">5362</span>, <span class="refDoi">Â DOI: 10.1002/chem.201505094</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1002%2Fchem.201505094" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=26928593" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC28XjtlOit7w%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2016&pages=5350-5362&author=D.+D.+Voauthor=T.+P.+A.+Tranauthor=C.+Staedelauthor=R.+Benhidaauthor=F.+Darfeuilleauthor=A.+Di+Giorgioauthor=M.+Duca&title=Oncogenic+microRNAs+biogenesis+as+a+drug+target%3A+Structure-activity+relationship+studies+on+new+aminoglycoside+conjugates&doi=10.1002%2Fchem.201505094"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Oncogenic MicroRNAs Biogenesis as a Drug Target: Structure-Activity Relationship Studies on New Aminoglycoside Conjugates</span></div><div class="casAuthors">Vo, Duc Duy; Tran, Thi Phuong Anh; Staedel, Cathy; Benhida, Rachid; Darfeuille, Fabien; Di Giorgio, Audrey; Duca, Maria</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry - A European Journal</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">5350-5362</span>CODEN:
                <span class="NLM_cas:coden">CEUJED</span>;
        ISSN:<span class="NLM_cas:issn">0947-6539</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">MicroRNAs (miRNAs) are a recently discovered category of small RNA mols. that regulate gene expression at the posttranscriptional level.  Accumulating evidence indicates that miRNAs are aberrantly expressed in a variety of human cancers and that the inhibition of these oncogenic miRNAs could find application in the therapy of different types of cancer.  Herein, the authors describe the synthesis and biol. evaluation of new small-mol. drugs that target oncogenic miRNAs prodn.  In particular, the authors chose to target two miRNAs (i.e., miRNA-372 and -373) implicated in various types of cancer, such as gastric cancer.  Their precursors (premiRNAs) are overexpressed in cancer cells and lead to mature miRNAs after cleavage of their stem-loop structure by the enzyme Dicer in the cytoplasm.  Some of the newly synthesized conjugates can inhibit Dicer processing of the targeted premiRNAs in vitro with increased efficacy relative to the previous results (D.D. Vo et al., ACS Chem. Biol. 2014, 9, 711-721) and, more importantly, to inhibit proliferations of adenocarcinoma gastric cancer (AGS) cells overexpressing these miRNAs, thus representing promising leads for future drug development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrwxgnq0HLdIrVg90H21EOLACvtfcHk0lgGU5Bmg70_Xg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XjtlOit7w%253D&md5=cb53a7beb631a1249e0c7f0ca1dc43b7</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1002%2Fchem.201505094&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fchem.201505094%26sid%3Dliteratum%253Aachs%26aulast%3DVo%26aufirst%3DD.%2BD.%26aulast%3DTran%26aufirst%3DT.%2BP.%2BA.%26aulast%3DStaedel%26aufirst%3DC.%26aulast%3DBenhida%26aufirst%3DR.%26aulast%3DDarfeuille%26aufirst%3DF.%26aulast%3DDi%2BGiorgio%26aufirst%3DA.%26aulast%3DDuca%26aufirst%3DM.%26atitle%3DOncogenic%2520microRNAs%2520biogenesis%2520as%2520a%2520drug%2520target%253A%2520Structure-activity%2520relationship%2520studies%2520on%2520new%2520aminoglycoside%2520conjugates%26jtitle%3DChem.%2520-%2520Eur.%2520J.%26date%3D2016%26volume%3D22%26spage%3D5350%26epage%3D5362%26doi%3D10.1002%2Fchem.201505094" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Staedel, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tran, T. P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giraud, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Darfeuille, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Giorgio, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tourasse, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salin, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uriac, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duca, M.</span></span> <span> </span><span class="NLM_article-title">Modulation of oncogenic miRNA biogenesis using functionalized polyamines</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1667</span>, <span class="refDoi">Â DOI: 10.1038/s41598-018-20053-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1038%2Fs41598-018-20053-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=29374231" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A280%3ADC%252BC1MvkslehtQ%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2018&pages=1667&author=C.+Staedelauthor=T.+P.+A.+Tranauthor=J.+Giraudauthor=F.+Darfeuilleauthor=A.+Di+Giorgioauthor=N.+J.+Tourasseauthor=F.+Salinauthor=P.+Uriacauthor=M.+Duca&title=Modulation+of+oncogenic+miRNA+biogenesis+using+functionalized+polyamines&doi=10.1038%2Fs41598-018-20053-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Modulation of oncogenic miRNA biogenesis using functionalized polyamines</span></div><div class="casAuthors">Staedel Cathy; Darfeuille Fabien; Tourasse Nicolas J; Tran Thi Phuong Anh; Di Giorgio Audrey; Duca Maria; Giraud Julie; Salin Franck; Uriac Philippe</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1667</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">MicroRNAs are key factors in the regulation of gene expression and their deregulation has been directly linked to various pathologies such as cancer.  The use of small molecules to tackle the overexpression of oncogenic miRNAs has proved its efficacy and holds the promise for therapeutic applications.  Here we describe the screening of a 640-compound library and the identification of polyamine derivatives interfering with in vitro Dicer-mediated processing of the oncogenic miR-372 precursor (pre-miR-372).  The most active inhibitor is a spermine-amidine conjugate that binds to the pre-miR-372 with a KD of 0.15 Î¼M, and inhibits its in vitro processing with a IC50 of 1.06 Î¼M.  The inhibition of miR-372 biogenesis was confirmed in gastric cancer cells overexpressing miR-372 and a specific inhibition of proliferation through de-repression of the tumor suppressor LATS2 protein, a miR-372 target, was observed.  This compound modifies the expression of a small set of miRNAs and its selective biological activity has been confirmed in patient-derived ex vivo cultures of gastric carcinoma.  Polyamine derivatives are promising starting materials for future studies about the inhibition of oncogenic miRNAs and, to the best of our knowledge, this is the first report about the application of functionalized polyamines as miRNAs interfering agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQbr8gYxuA1hg5nADWVYHg1fW6udTcc2eYTAg0J7a5uHLntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1MvkslehtQ%253D%253D&md5=d000435073e20bbb43b7787de637aaf4</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1038%2Fs41598-018-20053-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-018-20053-5%26sid%3Dliteratum%253Aachs%26aulast%3DStaedel%26aufirst%3DC.%26aulast%3DTran%26aufirst%3DT.%2BP.%2BA.%26aulast%3DGiraud%26aufirst%3DJ.%26aulast%3DDarfeuille%26aufirst%3DF.%26aulast%3DDi%2BGiorgio%26aufirst%3DA.%26aulast%3DTourasse%26aufirst%3DN.%2BJ.%26aulast%3DSalin%26aufirst%3DF.%26aulast%3DUriac%26aufirst%3DP.%26aulast%3DDuca%26aufirst%3DM.%26atitle%3DModulation%2520of%2520oncogenic%2520miRNA%2520biogenesis%2520using%2520functionalized%2520polyamines%26jtitle%3DSci.%2520Rep.%26date%3D2018%26volume%3D8%26spage%3D1667%26doi%3D10.1038%2Fs41598-018-20053-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murata, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fukuzumi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Umemoto, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakatani, K.</span></span> <span> </span><span class="NLM_article-title">Xanthone derivatives as potential inhibitors of miRNA processing by human Dicer: Targeting secondary structures of pre-miRNA by small molecules</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">252</span>â <span class="NLM_lpage">255</span>, <span class="refDoi">Â DOI: 10.1016/j.bmcl.2012.10.108</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1016%2Fj.bmcl.2012.10.108" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=23164709" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC38XhslSrsLnO" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2013&pages=252-255&author=A.+Murataauthor=T.+Fukuzumiauthor=S.+Umemotoauthor=K.+Nakatani&title=Xanthone+derivatives+as+potential+inhibitors+of+miRNA+processing+by+human+Dicer%3A+Targeting+secondary+structures+of+pre-miRNA+by+small+molecules&doi=10.1016%2Fj.bmcl.2012.10.108"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Xanthone derivatives as potential inhibitors of miRNA processing by human Dicer: Targeting secondary structures of pre-miRNA by small molecules</span></div><div class="casAuthors">Murata, Asako; Fukuzumi, Takeo; Umemoto, Shiori; Nakatani, Kazuhiko</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">252-255</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">In recent years, various biol. processes have been found to be regulated by miRNA-mediated gene silencing.  A small mol. that modulate the miRNA pathway will provide the biol. tool for elucidating mechanisms of miRNA-mediated gene regulation, and can be the drug lead for miRNA related diseases.  In this study, the authors demonstrated that an aminoalkoxy-substituted thioxanthone deriv. interferes Dicer-mediated processing of pre-miRNA.  Information about the interaction between these xanthone derivs. and pre-miRNAs will enable the authors to design and develop new small mol.-based inhibitors for miRNA pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqSvmgQp8NxvLVg90H21EOLACvtfcHk0lgR6DEUID9sGw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhslSrsLnO&md5=6c75fc852fbea26d548479b2be477dc9</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2012.10.108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2012.10.108%26sid%3Dliteratum%253Aachs%26aulast%3DMurata%26aufirst%3DA.%26aulast%3DFukuzumi%26aufirst%3DT.%26aulast%3DUmemoto%26aufirst%3DS.%26aulast%3DNakatani%26aufirst%3DK.%26atitle%3DXanthone%2520derivatives%2520as%2520potential%2520inhibitors%2520of%2520miRNA%2520processing%2520by%2520human%2520Dicer%253A%2520Targeting%2520secondary%2520structures%2520of%2520pre-miRNA%2520by%2520small%2520molecules%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2013%26volume%3D23%26spage%3D252%26epage%3D255%26doi%3D10.1016%2Fj.bmcl.2012.10.108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niu, H. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, Z. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, B. F.</span></span> <span> </span><span class="NLM_article-title">Overexpression of microRNA-1 impairs cardiac contractile function by damaging sarcomere assembly</span>. <i>Cardiovasc. Res.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>95</i></span>,  <span class="NLM_fpage">385</span>â <span class="NLM_lpage">393</span>, <span class="refDoi">Â DOI: 10.1093/cvr/cvs196</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1093%2Fcvr%2Fcvs196" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=22719074" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtFSlt73I" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=95&publication_year=2012&pages=385-393&author=J.+Aiauthor=R.+Zhangauthor=X.+Gaoauthor=H.+F.+Niuauthor=N.+Wangauthor=Y.+Xuauthor=Y.+Liauthor=N.+Maauthor=L.+H.+Sunauthor=Z.+W.+Panauthor=W.+M.+Liauthor=B.+F.+Yang&title=Overexpression+of+microRNA-1+impairs+cardiac+contractile+function+by+damaging+sarcomere+assembly&doi=10.1093%2Fcvr%2Fcvs196"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Overexpression of microRNA-1 impairs cardiac contractile function by damaging sarcomere assembly</span></div><div class="casAuthors">Ai, Jing; Zhang, Rong; Gao, Xu; Niu, Hui-Fang; Wang, Ning; Xu, Yi; Li, Yue; Ma, Ning; Sun, Li-Hua; Pan, Zhen-Wei; Li, Wei-Min; Yang, Bao-Feng</div><div class="citationInfo"><span class="NLM_cas:title">Cardiovascular Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">95</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">385-393</span>CODEN:
                <span class="NLM_cas:coden">CVREAU</span>;
        ISSN:<span class="NLM_cas:issn">0008-6363</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Aims: The purpose of the present study was to evaluate the effects of overexpression of microRNA-1 (miR-1) on cardiac contractile function and the potential mol. mechanisms.  Methods and results: Transgenic (Tg) mice (C57BL/6) for cardiac-specific overexpression of miR-1 driven by the Î±-myosin heavy chain promoter were generated and identified by real-time reverse-transcription polymerase chain reaction with left ventricular samples.  We found an age-dependent decrease in the heart function in Tg mice by pressure-vol. loop anal.  Histol. anal. and electron microscopy displayed short sarcomeres with the loss of the clear zone and H-zone as well as myofibril fragmentation and deliquescence in Tg mice.  Further studies demonstrated miR-1 post-transcriptionally down-regulated the expression of calmodulin (CaM) and cardiac myosin light chain kinase (cMLCK) proteins by targeting the 3'UTRs of MYLK3, CALM1, and CALM2 genes, leading to decreased phosphorylations of myosin light chain 2v (MLC2v) and cardiac myosin binding protein-C (cMyBP-C).  Knockdown of miR-1 by locked nucleic acid-modified anti-miR-1 antisense (LNA-antimiR-1) mitigated the adverse changes of cardiac function assocd. with overexpression of miR-1.  Conclusion: miR-1 induces adverse structural remodelling to impair cardiac contractile function.  Targeting cMLCK and CaM likely underlies the detrimental effects of miR-1 on structural components of muscles related to the contractile machinery.  Our study provides the first evidence that miRNAs cause adverse structural remodelling of the heart.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo9qMFjoVpI-LVg90H21EOLACvtfcHk0lgR6DEUID9sGw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtFSlt73I&md5=3864a1332fc6d5b69410287165da30bb</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1093%2Fcvr%2Fcvs196&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fcvr%252Fcvs196%26sid%3Dliteratum%253Aachs%26aulast%3DAi%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DR.%26aulast%3DGao%26aufirst%3DX.%26aulast%3DNiu%26aufirst%3DH.%2BF.%26aulast%3DWang%26aufirst%3DN.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DMa%26aufirst%3DN.%26aulast%3DSun%26aufirst%3DL.%2BH.%26aulast%3DPan%26aufirst%3DZ.%2BW.%26aulast%3DLi%26aufirst%3DW.%2BM.%26aulast%3DYang%26aufirst%3DB.%2BF.%26atitle%3DOverexpression%2520of%2520microRNA-1%2520impairs%2520cardiac%2520contractile%2520function%2520by%2520damaging%2520sarcomere%2520assembly%26jtitle%3DCardiovasc.%2520Res.%26date%3D2012%26volume%3D95%26spage%3D385%26epage%3D393%26doi%3D10.1093%2Fcvr%2Fcvs196" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tan, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span> <span> </span><span class="NLM_article-title">Small molecular inhibitors of miR-1 identified from photocycloadducts of acetylenes with 2-methoxy-1,4-naphthalenequinone</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">6124</span>â <span class="NLM_lpage">6131</span>, <span class="refDoi">Â DOI: 10.1016/j.bmc.2013.04.058</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1016%2Fj.bmc.2013.04.058" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=23685176" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC3sXnvVGktr4%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2013&pages=6124-6131&author=S.+B.+Tanauthor=C.+Huangauthor=X.+Chenauthor=Y.+Wuauthor=M.+Zhouauthor=C.+Zhangauthor=Y.+Zhang&title=Small+molecular+inhibitors+of+miR-1+identified+from+photocycloadducts+of+acetylenes+with+2-methoxy-1%2C4-naphthalenequinone&doi=10.1016%2Fj.bmc.2013.04.058"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Small molecular inhibitors of miR-1 identified from photocycloadducts of acetylenes with 2-methoxy-1,4-naphthalenequinone</span></div><div class="casAuthors">Tan, Su-Bee; Huang, Chengmei; Chen, Xuejiao; Wu, Yihan; Zhou, Mi; Zhang, Chenyu; Zhang, Yan</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">6124-6131</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Small mols. which can modulate endogenous microRNAs are important chem. tools to study microRNA regulational network.  In this Letter the authors screened the [2+2] photocycloadducts of 2-methoxy-1,4-naphthalenequinone with a series of aryl acetylenes on their activity to modulate endogenous microRNAs.  A potent inhibitor of the muscle-specific miR-1 which is closely related with cardiac development and disease was identified.  The small mol. inhibitor was the cyclobutene type product derived from the photocycloaddn. of 2-methoxy-1,4-naphthalenequinone with tert-Bu (5-(phenylethynyl)quinolin-8-yl) carbonate.  Analogs of the small mol. inhibitor were then prepd. using similar photocycloaddn. reactions for evaluation on inhibition activity on miR-1 to provide structure-activity relationship of the miR-1 inhibitor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqO2MDHBYgICLVg90H21EOLACvtfcHk0lhc90w43fcwSQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXnvVGktr4%253D&md5=efefc7c7101b3a1d7f668cd8000eb4f8</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2013.04.058&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2013.04.058%26sid%3Dliteratum%253Aachs%26aulast%3DTan%26aufirst%3DS.%2BB.%26aulast%3DHuang%26aufirst%3DC.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DY.%26atitle%3DSmall%2520molecular%2520inhibitors%2520of%2520miR-1%2520identified%2520from%2520photocycloadducts%2520of%2520acetylenes%2520with%25202-methoxy-1%252C4-naphthalenequinone%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2013%26volume%3D21%26spage%3D6124%26epage%3D6131%26doi%3D10.1016%2Fj.bmc.2013.04.058" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chandrasekhar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pushpavalli, S. N. C. V. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chatla, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mukhopadhyay, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ganganna, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vijeender, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Srihari, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reddy, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramaiah, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhadra, U.</span></span> <span> </span><span class="NLM_article-title"><i>aza</i>-Flavanones as potent cross-species microRNA inhibitors that arrest cell cycle</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">645</span>â <span class="NLM_lpage">648</span>, <span class="refDoi">Â DOI: 10.1016/j.bmcl.2011.10.061</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1016%2Fj.bmcl.2011.10.061" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=22100313" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC38XivVyqtQ%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2012&pages=645-648&author=S.+Chandrasekharauthor=S.+N.+C.+V.+L.+Pushpavalliauthor=S.+Chatlaauthor=D.+Mukhopadhyayauthor=B.+Gangannaauthor=K.+Vijeenderauthor=P.+Srihariauthor=C.+R.+Reddyauthor=M.+J.+Ramaiahauthor=U.+Bhadra&title=aza-Flavanones+as+potent+cross-species+microRNA+inhibitors+that+arrest+cell+cycle&doi=10.1016%2Fj.bmcl.2011.10.061"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">aza-Flavanones as potent cross-species microRNA inhibitors that arrest cell cycle</span></div><div class="casAuthors">Chandrasekhar, Srivari; Pushpavalli, Sreerangam N. C. V. L.; Chatla, Srinivas; Mukhopadhyay, Debasmita; Ganganna, Bogonda; Vijeender, Kandi; Srihari, Pabbaraja; Reddy, Chada Raji; Janaki Ramaiah, M.; Bhadra, Utpal</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">645-648</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Aza-Flavanones have been identified as a new class of selective microRNA inhibitors.  These compds. were found to arrest cell cycle via a novel cross species microRNA-dependent regulatory pathway interpreting an unexpected link between cell cycle arrest and microRNA mediated control in cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAxulyPUhrXrVg90H21EOLACvtfcHk0lhc90w43fcwSQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XivVyqtQ%253D%253D&md5=e16a5cc549530d0f2f2bba6e12e533ff</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2011.10.061&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2011.10.061%26sid%3Dliteratum%253Aachs%26aulast%3DChandrasekhar%26aufirst%3DS.%26aulast%3DPushpavalli%26aufirst%3DS.%2BN.%2BC.%2BV.%2BL.%26aulast%3DChatla%26aufirst%3DS.%26aulast%3DMukhopadhyay%26aufirst%3DD.%26aulast%3DGanganna%26aufirst%3DB.%26aulast%3DVijeender%26aufirst%3DK.%26aulast%3DSrihari%26aufirst%3DP.%26aulast%3DReddy%26aufirst%3DC.%2BR.%26aulast%3DRamaiah%26aufirst%3DM.%2BJ.%26aulast%3DBhadra%26aufirst%3DU.%26atitle%3Daza-Flavanones%2520as%2520potent%2520cross-species%2520microRNA%2520inhibitors%2520that%2520arrest%2520cell%2520cycle%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2012%26volume%3D22%26spage%3D645%26epage%3D648%26doi%3D10.1016%2Fj.bmcl.2011.10.061" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicolas, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marks, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sander, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lerro, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buendia, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mason, W. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moloshok, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bort, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zaret, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, J. M.</span></span> <span> </span><span class="NLM_article-title">miR-122, a mammalian liver-specific microRNA, is processed from hcr mRNA and may downregulate the high affinity cationic amino acid transporter CAT-1</span>. <i>RNA Biol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">106</span>â <span class="NLM_lpage">113</span>, <span class="refDoi">Â DOI: 10.4161/rna.1.2.1066</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.4161%2Frna.1.2.1066" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=17179747" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BD2MXlsFSrt74%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2004&pages=106-113&author=J.+Changauthor=E.+Nicolasauthor=D.+Marksauthor=C.+Sanderauthor=A.+Lerroauthor=M.+A.+Buendiaauthor=C.+Xuauthor=W.+S.+Masonauthor=T.+Moloshokauthor=R.+Bortauthor=K.+S.+Zaretauthor=J.+M.+Taylor&title=miR-122%2C+a+mammalian+liver-specific+microRNA%2C+is+processed+from+hcr+mRNA+and+may+downregulate+the+high+affinity+cationic+amino+acid+transporter+CAT-1&doi=10.4161%2Frna.1.2.1066"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">MiR-122, a mammalian liver-specific microRNA, is processed from hcr mRNA and may downregulate the high affinity cationic amino acid transporter CAT-1</span></div><div class="casAuthors">Chang, Jinhong; Nicolas, Emmanuelle; Marks, Debora; Sander, Chris; Lerro, Anthony; Buendia, Marie Annick; Xu, Chunxiao; Mason, William S.; Moloshok, Thomas; Bort, Roque; Zaret, Kenneth S.; Taylor, John M.</div><div class="citationInfo"><span class="NLM_cas:title">RNA Biology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">106-113</span>CODEN:
                <span class="NLM_cas:coden">RBNIBE</span>;
        ISSN:<span class="NLM_cas:issn">1547-6286</span>.
    
            (<span class="NLM_cas:orgname">Landes Bioscience</span>)
        </div><div class="casAbstract">These studies show that miR-122, a 22-nucleotide microRNA, is derived from a liver-specific noncoding polyadenylated RNA transcribed from the gene hcr.  The exact sequence of miR-122 as well as the adjacent secondary structure within the hcr mRNA are conserved from mammalian species back to fish.  Levels of miR-122 in the mouse liver increase to half maximal values around day 17 of embryogenesis, and reach near maximal levels of 50,000 copies per av. cell before birth.  Lewis et al. (2003) predicted the cationic amino acid transporter (CAT-1 or SLC7A1) as a miR-122 target.  CAT-1 protein and its mRNA are expressed in all mammalian tissues but with lower levels in adult liver.  Furthermore, during mouse liver development CAT-1 mRNA decreases in an almost inverse correlation with miR-122.  Eight potential miR-122 target sites were predicted within the human CAT-1 mRNA, with six in the 3'-untranslated region.  Using a reporter construct it was found that just three of the predicted sites, linked in a 400-nucleotide sequence from human CAT-1, acted with synergy and were sufficient to strongly inhibit protein synthesis and reduce mRNA levels.  In summary, these studies followed the accumulation during development of miR-122 from its mRNA precursor, hcr, through to identification of what may be a specific mRNA target, CAT-1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGocW-mOgnLCh7Vg90H21EOLACvtfcHk0lg10mg7nK-0MA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXlsFSrt74%253D&md5=7de7e091c98feac574c4a6189408879a</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.4161%2Frna.1.2.1066&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252Frna.1.2.1066%26sid%3Dliteratum%253Aachs%26aulast%3DChang%26aufirst%3DJ.%26aulast%3DNicolas%26aufirst%3DE.%26aulast%3DMarks%26aufirst%3DD.%26aulast%3DSander%26aufirst%3DC.%26aulast%3DLerro%26aufirst%3DA.%26aulast%3DBuendia%26aufirst%3DM.%2BA.%26aulast%3DXu%26aufirst%3DC.%26aulast%3DMason%26aufirst%3DW.%2BS.%26aulast%3DMoloshok%26aufirst%3DT.%26aulast%3DBort%26aufirst%3DR.%26aulast%3DZaret%26aufirst%3DK.%2BS.%26aulast%3DTaylor%26aufirst%3DJ.%2BM.%26atitle%3DmiR-122%252C%2520a%2520mammalian%2520liver-specific%2520microRNA%252C%2520is%2520processed%2520from%2520hcr%2520mRNA%2520and%2520may%2520downregulate%2520the%2520high%2520affinity%2520cationic%2520amino%2520acid%2520transporter%2520CAT-1%26jtitle%3DRNA%2520Biol.%26date%3D2004%26volume%3D1%26spage%3D106%26epage%3D113%26doi%3D10.4161%2Frna.1.2.1066" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Young, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Connelly, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grohmann, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deiters, A.</span></span> <span> </span><span class="NLM_article-title">Small molecule modifiers of microRNA miR-122 function for the treatment of hepatitis C virus infection and hepatocellular carcinoma</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>132</i></span>,  <span class="NLM_fpage">7976</span>â <span class="NLM_lpage">7981</span>, <span class="refDoi">Â DOI: 10.1021/ja910275u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja910275u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC3cXmt12ms7w%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=132&publication_year=2010&pages=7976-7981&author=D.+D.+Youngauthor=C.+M.+Connellyauthor=C.+Grohmannauthor=A.+Deiters&title=Small+molecule+modifiers+of+microRNA+miR-122+function+for+the+treatment+of+hepatitis+C+virus+infection+and+hepatocellular+carcinoma&doi=10.1021%2Fja910275u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Small Molecule Modifiers of MicroRNA miR-122 Function for the Treatment of Hepatitis C Virus Infection and Hepatocellular Carcinoma</span></div><div class="casAuthors">Young, Douglas D.; Connelly, Colleen M.; Grohmann, Christoph; Deiters, Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">132</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">7976-7981</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">MicroRNAs are a recently discovered new class of important endogenous regulators of gene function.  Aberrant regulation of microRNAs has been linked to various human diseases, most importantly cancer.  Small mol. intervention of microRNA misregulation has the potential to provide new therapeutic approaches to such diseases.  Here, we report the first small mol. inhibitors and activators of the liver-specific microRNA miR-122.  This microRNA is the most abundant microRNA in the liver and is involved in hepatocellular carcinoma development and hepatitis C virus (HCV) infection.  Our small mol. inhibitors reduce viral replication in liver cells and represent a new approach to the treatment of HCV infections.  Moreover, small mol. activation of miR-122 in liver cancer cells selectively induced apoptosis through caspase activation, thus having implications in cancer chemotherapy.  In addn. to providing a new approach for the development of therapeutics, small mol. modifiers of miR-122 function are unique tools for exploring miR-122 biogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJ6MkKhmyG_7Vg90H21EOLACvtfcHk0lg10mg7nK-0MA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXmt12ms7w%253D&md5=3aefaa3cf2af65e5a45fc52fbeaf8494</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1021%2Fja910275u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja910275u%26sid%3Dliteratum%253Aachs%26aulast%3DYoung%26aufirst%3DD.%2BD.%26aulast%3DConnelly%26aufirst%3DC.%2BM.%26aulast%3DGrohmann%26aufirst%3DC.%26aulast%3DDeiters%26aufirst%3DA.%26atitle%3DSmall%2520molecule%2520modifiers%2520of%2520microRNA%2520miR-122%2520function%2520for%2520the%2520treatment%2520of%2520hepatitis%2520C%2520virus%2520infection%2520and%2520hepatocellular%2520carcinoma%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2010%26volume%3D132%26spage%3D7976%26epage%3D7981%26doi%3D10.1021%2Fja910275u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boyd, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malstrom, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Subramanian, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Venkatesh, L. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaeper, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elangovan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DâSa-Eipper, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chinnadurai, G.</span></span> <span> </span><span class="NLM_article-title">Adenovirus E1B 19 kDa and Bcl-2 proteins interact with a common set of cellular proteins</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>79</i></span>,  <span class="NLM_fpage">341</span>â <span class="NLM_lpage">351</span>, <span class="refDoi">Â DOI: 10.1016/0092-8674(94)90202-X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1016%2F0092-8674%2894%2990202-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=7954800" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADyaK2MXhvFWrtr4%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=1994&pages=341-351&author=J.+M.+Boydauthor=S.+Malstromauthor=T.+Subramanianauthor=L.+K.+Venkateshauthor=U.+Schaeperauthor=B.+Elangovanauthor=C.+D%E2%80%99Sa-Eipperauthor=G.+Chinnadurai&title=Adenovirus+E1B+19+kDa+and+Bcl-2+proteins+interact+with+a+common+set+of+cellular+proteins&doi=10.1016%2F0092-8674%2894%2990202-X"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Adenovirus E1B 19 kDa and Bcl-2 proteins interact with a common set of cellular proteins</span></div><div class="casAuthors">Boyd, Janice M.; Malstrom, Scott; Subramanian, T.; Venkatesh, L. K.; Schaeper, Ute; Elangovan, B.; D'Sa-Eipper, Cleta; Chinnadurai, G.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, Massachusetts)</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">79</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">341-51</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Adenovirus E1B kDa protein protects against cell death induced by viral infection and certain external stimuli.  The Bcl-2 protein can functionally substitute for the E1B 19 kDa protein.  To identify cellular targets for the 19 kDa protein, the authors used the two-hybrid screen in yeast.  The authors have isolated cDNAs for three different proteins, designated Nip1, Nip2, and Nip3, that interact with the 19 kDa protein.  Mutational anal. indicates that these proteins do not assoc. with 19 kDa mutants defective in suppression of cell death, suggesting a correlation between interaction of these proteins and suppression of cell death.  These proteins also assoc. with discrete sequence motifs in the Bcl-2 protein that are homologous to motifs of the 19 kDa protein.  These results suggest that two diverse proteins, the E1B 19 kDa and the Bcl-2 proteins, promote cell survival through interaction with a common set of cellular proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCbKglHHNc7bVg90H21EOLACvtfcHk0lgu9Km9nipwfw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXhvFWrtr4%253D&md5=208a600c230fa7df99347cc8314bbd23</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1016%2F0092-8674%2894%2990202-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0092-8674%252894%252990202-X%26sid%3Dliteratum%253Aachs%26aulast%3DBoyd%26aufirst%3DJ.%2BM.%26aulast%3DMalstrom%26aufirst%3DS.%26aulast%3DSubramanian%26aufirst%3DT.%26aulast%3DVenkatesh%26aufirst%3DL.%2BK.%26aulast%3DSchaeper%26aufirst%3DU.%26aulast%3DElangovan%26aufirst%3DB.%26aulast%3DD%25E2%2580%2599Sa-Eipper%26aufirst%3DC.%26aulast%3DChinnadurai%26aufirst%3DG.%26atitle%3DAdenovirus%2520E1B%252019%2520kDa%2520and%2520Bcl-2%2520proteins%2520interact%2520with%2520a%2520common%2520set%2520of%2520cellular%2520proteins%26jtitle%3DCell%26date%3D1994%26volume%3D79%26spage%3D341%26epage%3D351%26doi%3D10.1016%2F0092-8674%2894%2990202-X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ham, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seo, H. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cha, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mun, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoon, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hwang, K. C.</span></span> <span> </span><span class="NLM_article-title">Small molecule-mediated upregulation of microRNA targeting a key cell death modulator BNIP3 improves cardiac function following ischemic injury</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">23472</span>, <span class="refDoi">Â DOI: 10.1038/srep23472</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1038%2Fsrep23472" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=27008992" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC28XkvVCgtLw%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2016&pages=23472&author=S.+Y.+Leeauthor=S.+Leeauthor=E.+Choiauthor=O.+Hamauthor=C.+Y.+Leeauthor=J.+Leeauthor=H.+H.+Seoauthor=M.+J.+Chaauthor=B.+Munauthor=Y.+Leeauthor=C.+Yoonauthor=K.+C.+Hwang&title=Small+molecule-mediated+upregulation+of+microRNA+targeting+a+key+cell+death+modulator+BNIP3+improves+cardiac+function+following+ischemic+injury&doi=10.1038%2Fsrep23472"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Small molecule-mediated up-regulation of microRNA targeting a key cell death modulator BNIP3 improves cardiac function following ischemic injury</span></div><div class="casAuthors">Lee, Se-Yeon; Lee, Seahyoung; Choi, Eunhyun; Ham, Onju; Lee, Chang Youn; Lee, Jiyun; Seo, Hyang-Hee; Cha, Min-Ji; Mun, Bohyun; Lee, Yunmi; Yoon, Cheesoon; Hwang, Ki-Chul</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">23472</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Genetic ablation of BCL2/adenovirus E1B 19 kDa protein-interacting protein 3 (BNIP3), an essential regulator of cardiac cell death, is an effective way to prevent cardiac cell death triggered by pathol. conditions.  However, currently there exists no known means, such as inhibitors, to down-regulate BNIP3 in mature heart.  Here, we report that a small mol. inducer of microRNA-182 (miR-182) suppressed ischemia/reperfusion (I/R)-induced cardiac cell death by down-regulating BNIP3.  We first selected miR-182 as a potent BNIP3-targeting miRNA based on miRNA-target prediction databases and empirical data.  The subsequent screening of small mols. for inducing miR-182 expression identified Kenpaullone as a hit compd.  Both exogenous miR-182 and Kenpaullone significantly suppressed hypoxia-induced cardiomyocyte death in vitro.  To investigate the effect of changing substituents of Kenpaullone on miR-182 expression, we synthesized 9 derivs. of Kenpaullone.  Among these derivs., compd. 5 showed significantly improved ability to induce miR-182 expression.  The results of the in vivo study showed that compd. 5 significantly improved heart function following I/R-injury in rats.  Our study provides strong evidence that the small mol.-mediated up-regulation of miRNAs is a viable strategy to down-regulate target proteins with no known chem. inhibitor and that compd. 5 may have potential to prevent I/R-inflicted cardiac cell death.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGos9r_v2BG3CbVg90H21EOLACvtfcHk0lgu9Km9nipwfw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XkvVCgtLw%253D&md5=2d636050f6e3473ce50585b7e2805550</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1038%2Fsrep23472&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep23472%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DS.%2BY.%26aulast%3DLee%26aufirst%3DS.%26aulast%3DChoi%26aufirst%3DE.%26aulast%3DHam%26aufirst%3DO.%26aulast%3DLee%26aufirst%3DC.%2BY.%26aulast%3DLee%26aufirst%3DJ.%26aulast%3DSeo%26aufirst%3DH.%2BH.%26aulast%3DCha%26aufirst%3DM.%2BJ.%26aulast%3DMun%26aufirst%3DB.%26aulast%3DLee%26aufirst%3DY.%26aulast%3DYoon%26aufirst%3DC.%26aulast%3DHwang%26aufirst%3DK.%2BC.%26atitle%3DSmall%2520molecule-mediated%2520upregulation%2520of%2520microRNA%2520targeting%2520a%2520key%2520cell%2520death%2520modulator%2520BNIP3%2520improves%2520cardiac%2520function%2520following%2520ischemic%2520injury%26jtitle%3DSci.%2520Rep.%26date%3D2016%26volume%3D6%26spage%3D23472%26doi%3D10.1038%2Fsrep23472" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jin, H. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oda, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skalsky, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bethel, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shepherd, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, W. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sabouri-Ghomi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cullen, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rajewsky, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, C.</span></span> <span> </span><span class="NLM_article-title">MicroRNA-17 â¼ 92 plays a causative role in lymphomagenesis by coordinating multiple oncogenic pathways</span>. <i>EMBO J.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">2377</span>â <span class="NLM_lpage">2391</span>, <span class="refDoi">Â DOI: 10.1038/emboj.2013.178</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1038%2Femboj.2013.178" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=23921550" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1Skur3N" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2013&pages=2377-2391&author=H.+Y.+Jinauthor=H.+Odaauthor=M.+Laiauthor=R.+L.+Skalskyauthor=K.+Bethelauthor=J.+Shepherdauthor=S.+G.+Kangauthor=W.+H.+Liuauthor=M.+Sabouri-Ghomiauthor=B.+R.+Cullenauthor=K.+Rajewskyauthor=C.+Xiao&title=MicroRNA-17+%E2%88%BC+92+plays+a+causative+role+in+lymphomagenesis+by+coordinating+multiple+oncogenic+pathways&doi=10.1038%2Femboj.2013.178"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">MicroRNA-17-92 plays a causative role in lymphomagenesis by coordinating multiple oncogenic pathways</span></div><div class="casAuthors">Jin, Hyun Yong; Oda, Hiroyo; Lai, Maoyi; Skalsky, Rebecca L.; Bethel, Kelly; Shepherd, Jovan; Kang, Seung Goo; Liu, Wen-Hsien; Sabouri-Ghomi, Mohsen; Cullen, Bryan R.; Rajewsky, Klaus; Xiao, Changchun</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Journal</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">2377-2391</span>CODEN:
                <span class="NLM_cas:coden">EMJODG</span>;
        ISSN:<span class="NLM_cas:issn">0261-4189</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">MicroRNAs (miRNAs) have been broadly implicated in cancer, but their exact function and mechanism in carcinogenesis remain poorly understood.  Elevated miR-17-92 expression is frequently found in human cancers, mainly due to gene amplification and Myc-mediated transcriptional upregulation.  Here we show that B cell-specific miR-17â¼92 transgenic mice developed lymphomas with high penetrance and that, conversely, Myc-driven lymphomagenesis stringently requires two intact alleles of miR-17â¼92.  We exptl. identified miR-17â¼92 target genes by PAR-CLIP and validated select target genes in miR-17â¼92 transgenic mice.  These analyses demonstrate that miR-17â¼92 drives lymphomagenesis by suppressing the expression of multiple neg. regulators of the PI3K and NFÎºB pathways and by inhibiting the mitochondrial apoptosis pathway.  Accordingly, miR-17-92-driven lymphoma cells exhibited constitutive activation of the PI3K and NFÎºB pathways and chem. inhibition of either pathway reduced tumor size and prolonged the survival of lymphoma-bearing mice.  These findings establish miR-17â¼92 as a powerful cancer driver that coordinates the activation of multiple oncogenic pathways, and demonstrate for the first time that chem. inhibition of miRNA downstream pathways has therapeutic value in treating cancers caused by miRNA dysregulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpy45tQaID4JLVg90H21EOLACvtfcHk0liw-nQTWO0emQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1Skur3N&md5=08ef3d1f1f7c0fe1cb343352474a53ee</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1038%2Femboj.2013.178&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Femboj.2013.178%26sid%3Dliteratum%253Aachs%26aulast%3DJin%26aufirst%3DH.%2BY.%26aulast%3DOda%26aufirst%3DH.%26aulast%3DLai%26aufirst%3DM.%26aulast%3DSkalsky%26aufirst%3DR.%2BL.%26aulast%3DBethel%26aufirst%3DK.%26aulast%3DShepherd%26aufirst%3DJ.%26aulast%3DKang%26aufirst%3DS.%2BG.%26aulast%3DLiu%26aufirst%3DW.%2BH.%26aulast%3DSabouri-Ghomi%26aufirst%3DM.%26aulast%3DCullen%26aufirst%3DB.%2BR.%26aulast%3DRajewsky%26aufirst%3DK.%26aulast%3DXiao%26aufirst%3DC.%26atitle%3DMicroRNA-17%2520%25E2%2588%25BC%252092%2520plays%2520a%2520causative%2520role%2520in%2520lymphomagenesis%2520by%2520coordinating%2520multiple%2520oncogenic%2520pathways%26jtitle%3DEMBO%2520J.%26date%3D2013%26volume%3D32%26spage%3D2377%26epage%3D2391%26doi%3D10.1038%2Femboj.2013.178" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Velagapudi, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tran, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haniff, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fallahi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinez, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Childs-Disney, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span> <span> </span><span class="NLM_article-title">Defining RNA-small molecule affinity landscapes enables design of a small molecule inhibitor of an oncogenic noncoding RNA</span>. <i>ACS Cent. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">205</span>â <span class="NLM_lpage">216</span>, <span class="refDoi">Â DOI: 10.1021/acscentsci.7b00009</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acscentsci.7b00009" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjs1GqsLs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2017&pages=205-216&author=S.+P.+Velagapudiauthor=Y.+Luoauthor=T.+Tranauthor=H.+S.+Haniffauthor=Y.+Nakaiauthor=M.+Fallahiauthor=G.+J.+Martinezauthor=J.+L.+Childs-Disneyauthor=M.+D.+Disney&title=Defining+RNA-small+molecule+affinity+landscapes+enables+design+of+a+small+molecule+inhibitor+of+an+oncogenic+noncoding+RNA&doi=10.1021%2Facscentsci.7b00009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Defining RNA-Small Molecule Affinity Landscapes Enables Design of a Small Molecule Inhibitor of an Oncogenic Noncoding RNA</span></div><div class="casAuthors">Velagapudi, Sai Pradeep; Luo, Yiling; Tran, Tuan; Haniff, Hafeez S.; Nakai, Yoshio; Fallahi, Mohammad; Martinez, Gustavo J.; Childs-Disney, Jessica L.; Disney, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Central Science</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">205-216</span>CODEN:
                <span class="NLM_cas:coden">ACSCII</span>;
        ISSN:<span class="NLM_cas:issn">2374-7951</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">RNA drug targets are pervasive in cells but methods to design small mols. that target them are sparse.  Herein, the authors report a general approach to score the affinity and selectivity of RNA motif-small mol. interactions identified via selection.  Named High Throughput Structure-Activity Relationships Through Sequencing (HiT-StARTS), HiT-StARTS is statistical in nature and compares input nucleic acid sequences to selected library members that bind a ligand via high throughput sequencing.  The approach allowed facile definition of the fitness landscape of hundreds of thousands of RNA motif-small mol. binding partners.  These results were mined against folded RNAs in the human transcriptome and identified an avid interaction between a small mol. and the Dicer nuclease-processing site in the oncogenic microRNA (miR)-18a hairpin precursor, which is a member of the miR-17-92 cluster.  Application of the small mol., Targapremir-18a, to prostate cancer cells inhibited prodn. of miR-18a from the cluster, derepressed serine/threonine protein kinase 4 protein (STK4), and triggered apoptosis.  Profiling the cellular targets of Targapremir-18a via Chem. Cross Linking and Isolation by Pull Down (Chem-CLIP), a covalent small mol.-RNA cellular profiling approach, and other studies showed specific binding of the compd. to the miR-18a precursor, revealing broadly applicable factors that govern small mol. drugging of noncoding RNAs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPmptGpjN_4rVg90H21EOLACvtfcHk0lhHriJiMVd5yA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjs1GqsLs%253D&md5=fefd2fe6e9d637c4aa04fd1e25fb1446</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1021%2Facscentsci.7b00009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facscentsci.7b00009%26sid%3Dliteratum%253Aachs%26aulast%3DVelagapudi%26aufirst%3DS.%2BP.%26aulast%3DLuo%26aufirst%3DY.%26aulast%3DTran%26aufirst%3DT.%26aulast%3DHaniff%26aufirst%3DH.%2BS.%26aulast%3DNakai%26aufirst%3DY.%26aulast%3DFallahi%26aufirst%3DM.%26aulast%3DMartinez%26aufirst%3DG.%2BJ.%26aulast%3DChilds-Disney%26aufirst%3DJ.%2BL.%26aulast%3DDisney%26aufirst%3DM.%2BD.%26atitle%3DDefining%2520RNA-small%2520molecule%2520affinity%2520landscapes%2520enables%2520design%2520of%2520a%2520small%2520molecule%2520inhibitor%2520of%2520an%2520oncogenic%2520noncoding%2520RNA%26jtitle%3DACS%2520Cent.%2520Sci.%26date%3D2017%26volume%3D3%26spage%3D205%26epage%3D216%26doi%3D10.1021%2Facscentsci.7b00009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Velagapudi, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cameron, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haga, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenberg, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lafitte, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duckett, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phinney, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span> <span> </span><span class="NLM_article-title">Design of a small molecule against an oncogenic noncoding RNA</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>113</i></span>,  <span class="NLM_fpage">5898</span>â <span class="NLM_lpage">5903</span>, <span class="refDoi">Â DOI: 10.1073/pnas.1523975113</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1073%2Fpnas.1523975113" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=27170187" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC28Xns1ejurw%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2016&pages=5898-5903&author=S.+P.+Velagapudiauthor=M.+D.+Cameronauthor=C.+L.+Hagaauthor=L.+H.+Rosenbergauthor=M.+Lafitteauthor=D.+R.+Duckettauthor=D.+G.+Phinneyauthor=M.+D.+Disney&title=Design+of+a+small+molecule+against+an+oncogenic+noncoding+RNA&doi=10.1073%2Fpnas.1523975113"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Design of a small molecule against an oncogenic noncoding RNA</span></div><div class="casAuthors">Velagapudi, Sai Pradeep; Cameron, Michael D.; Haga, Christopher L.; Rosenberg, Laura H.; Lafitte, Marie; Duckett, Derek R.; Phinney, Donald G.; Disney, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">113</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">5898-5903</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The design of precision, preclin. therapeutics from sequence is difficult, but advances in this area, particularly those focused on rational design, could quickly transform the sequence of disease-causing gene products into lead modalities.  Herein, we describe the use of Inforna, a computational approach that enables the rational design of small mols. targeting RNA to quickly provide a potent modulator of oncogenic microRNA-96 (miR-96).  We mined the secondary structure of primary microRNA-96 (pri-miR-96) hairpin precursor against a database of RNA motif-small mol. interactions, which identified modules that bound RNA motifs nearby and in the Drosha processing site.  Precise linking of these modules together provided Targaprimir-96 (3), which selectively modulates miR-96 prodn. in cancer cells and triggers apoptosis.  Importantly, the compd. is ineffective on healthy breast cells, and exogenous overexpression of pri-miR-96 reduced compd. potency in breast cancer cells.  Chem. Crosslinking and Isolation by Pull-Down (Chem-CLIP), a small-mol. RNA target validation approach, shows that 3 directly engages pri-miR-96 in breast cancer cells.  In vivo, 3 has a favorable pharmacokinetic profile and decreases tumor burden in a mouse model of triple-neg. breast cancer.  Thus, rational design can quickly produce precision, in vivo bioactive lead small mols. against hard-to-treat cancers by targeting oncogenic noncoding RNAs, advancing a disease-to-gene-to-drug paradigm.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlNk7Nj8xTxrVg90H21EOLACvtfcHk0lhHriJiMVd5yA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xns1ejurw%253D&md5=9254bf5bef8ae21aa59ae0bca5f47f37</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1523975113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1523975113%26sid%3Dliteratum%253Aachs%26aulast%3DVelagapudi%26aufirst%3DS.%2BP.%26aulast%3DCameron%26aufirst%3DM.%2BD.%26aulast%3DHaga%26aufirst%3DC.%2BL.%26aulast%3DRosenberg%26aufirst%3DL.%2BH.%26aulast%3DLafitte%26aufirst%3DM.%26aulast%3DDuckett%26aufirst%3DD.%2BR.%26aulast%3DPhinney%26aufirst%3DD.%2BG.%26aulast%3DDisney%26aufirst%3DM.%2BD.%26atitle%3DDesign%2520of%2520a%2520small%2520molecule%2520against%2520an%2520oncogenic%2520noncoding%2520RNA%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2016%26volume%3D113%26spage%3D5898%26epage%3D5903%26doi%3D10.1073%2Fpnas.1523975113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref64"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref64'); return false;" data-citation="" class="refNumLink">64</a></strong><div class="NLM_citation" id="cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chan, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loscalzo, J.</span></span> <span> </span><span class="NLM_article-title">MicroRNA-210: A unique and pleiotropic hypoxamir</span>. <i>Cell Cycle</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1072</span>â <span class="NLM_lpage">1083</span>, <span class="refDoi">Â DOI: 10.4161/cc.9.6.11006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.4161%2Fcc.9.6.11006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=20237418" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht12itr%252FM" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2010&pages=1072-1083&author=S.+Y.+Chanauthor=J.+Loscalzo&title=MicroRNA-210%3A+A+unique+and+pleiotropic+hypoxamir&doi=10.4161%2Fcc.9.6.11006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">MicroRNA-210: a unique and pleiotropic hypoxamir</span></div><div class="casAuthors">Chan, Stephen Y.; Loscalzo, Joseph</div><div class="citationInfo"><span class="NLM_cas:title">Cell Cycle</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1072-1083</span>CODEN:
                <span class="NLM_cas:coden">CCEYAS</span>;
        ISSN:<span class="NLM_cas:issn">1538-4101</span>.
    
            (<span class="NLM_cas:orgname">Landes Bioscience</span>)
        </div><div class="casAbstract">Inadequate oxygen availability or hypoxia induces a complex and still incompletely understood set of adaptations that influence cellular survival and function.  Many of these adaptations are directly controlled by a master transcription factor, hypoxia inducible factor-alpha (HIF-Î±).  In response to hypoxia, HIF-Î± levels increase and directly induce the transcription of >100 genes, influencing functions ranging from metab., survival, proliferation, migration, to angiogenesis, among others.  Recently, it has been demonstrated that a specific set of microRNA mols. are upregulated by hypoxia, which we denote here as "hypoxamirs.".  In particular, the HIF-responsive hypoxamir microRNA-210 (miR-210) is a unique microRNA that is evolutionarily conserved and ubiquitously expressed in hypoxic cell and tissue types.  A no. of direct targets of miR-210 have been identified by in silico, transcriptional and biochem. methods, a subset of which have been extensively validated.  As a result, miR-210 has been mechanistically linked to the control of a wide range of cellular responses known to influence normal developmental physiol. as well as a no. of hypoxia-dependent disease states, including tissue ischemia, inflammation and tumorigenesis.  Thus, reflecting the pleiotropic actions of HIF-Î±, miR-210 appears to function as a "master microRNA" relevant for the control of diverse functions in the hypoxic state.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrW8sJalImMsrVg90H21EOLACvtfcHk0ljE0lXBUSUIyA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht12itr%252FM&md5=633ef0fafa94dcd816d882dc212dbed8</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.4161%2Fcc.9.6.11006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252Fcc.9.6.11006%26sid%3Dliteratum%253Aachs%26aulast%3DChan%26aufirst%3DS.%2BY.%26aulast%3DLoscalzo%26aufirst%3DJ.%26atitle%3DMicroRNA-210%253A%2520A%2520unique%2520and%2520pleiotropic%2520hypoxamir%26jtitle%3DCell%2520Cycle%26date%3D2010%26volume%3D9%26spage%3D1072%26epage%3D1083%26doi%3D10.4161%2Fcc.9.6.11006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref65"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref65'); return false;" data-citation="" class="refNumLink">65</a></strong><div class="NLM_citation" id="cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Costales, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suresh, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vishnu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span> <span> </span><span class="NLM_article-title">Targeted degradation of a hypoxia-associated non-coding RNA enhances the selectivity of a small molecule interacting with RNA</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">1180</span>â <span class="NLM_lpage">1186</span>, <span class="refDoi">Â DOI: 10.1016/j.chembiol.2019.04.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1016%2Fj.chembiol.2019.04.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=31130520" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtVegt7bN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2019&pages=1180-1186&author=M.+G.+Costalesauthor=B.+Sureshauthor=K.+Vishnuauthor=M.+D.+Disney&title=Targeted+degradation+of+a+hypoxia-associated+non-coding+RNA+enhances+the+selectivity+of+a+small+molecule+interacting+with+RNA&doi=10.1016%2Fj.chembiol.2019.04.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted Degradation of a Hypoxia-Associated Non-coding RNA Enhances the Selectivity of a Small Molecule Interacting with RNA</span></div><div class="casAuthors">Costales, Matthew G.; Suresh, Blessy; Vishnu, Kamalakannan; Disney, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1180-1186.e5</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Small-mol. targeted recruitment of nucleases to RNA is a powerful method to affect RNA biol.  Inforna, a sequence-based design approach to target RNA, enables the design of small mols. that bind to and cleave RNA in a selective and substoichiometric manner.  Here, we investigate the ability of RNA-targeted degrdn. to improve the selectivity of small mols. targeting RNA.  The microRNA-210 hairpin precursor (pre-miR-210) is overexpressed in hypoxic cancers.  Previously, a small mol. (Targapremir-210 [TGP-210]) targeted this RNA in cells, but with a 5-fold window for DNA binding.  Appendage of a nuclease recruitment module onto TGP-210 locally recruited RNase L onto pre-miR-210, triggering its degrdn.  The chimera has enhanced selectivity compared with TGP-210 with nanomolar binding to the pre-miR-210, but no DNA binding, and is broadly selective for affecting RNA function in cells.  Importantly, it cleaved pre-miR-210 substoichiometrically and induced apoptosis in breast cancer cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5E5B-t5kH2rVg90H21EOLACvtfcHk0ljE0lXBUSUIyA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtVegt7bN&md5=bd7cfbb758ea4c56c50424d4d5720195</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2019.04.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2019.04.008%26sid%3Dliteratum%253Aachs%26aulast%3DCostales%26aufirst%3DM.%2BG.%26aulast%3DSuresh%26aufirst%3DB.%26aulast%3DVishnu%26aufirst%3DK.%26aulast%3DDisney%26aufirst%3DM.%2BD.%26atitle%3DTargeted%2520degradation%2520of%2520a%2520hypoxia-associated%2520non-coding%2520RNA%2520enhances%2520the%2520selectivity%2520of%2520a%2520small%2520molecule%2520interacting%2520with%2520RNA%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2019%26volume%3D26%26spage%3D1180%26epage%3D1186%26doi%3D10.1016%2Fj.chembiol.2019.04.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref66"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref66'); return false;" data-citation="" class="refNumLink">66</a></strong><div class="NLM_citation" id="cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Iqbal, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iqbal, N.</span></span> <span> </span><span class="NLM_article-title">Human epidermal growth factor receptor 2 (HER2) in cancers: Overexpression and therapeutic implications</span>. <i>Mol. Biol. Int.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>2014</i></span>,  <span class="NLM_fpage">852748</span>, <span class="refDoi">Â DOI: 10.1155/2014/852748</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1155%2F2014%2F852748" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=25276427" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A280%3ADC%252BC2M7mvFKnsw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2014&publication_year=2014&pages=852748&author=N.+Iqbalauthor=N.+Iqbal&title=Human+epidermal+growth+factor+receptor+2+%28HER2%29+in+cancers%3A+Overexpression+and+therapeutic+implications&doi=10.1155%2F2014%2F852748"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">Human Epidermal Growth Factor Receptor 2 (HER2) in Cancers: Overexpression and Therapeutic Implications</span></div><div class="casAuthors">Iqbal Nida; Iqbal Naveed</div><div class="citationInfo"><span class="NLM_cas:title">Molecular biology international</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">2014</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">852748</span>
        ISSN:<span class="NLM_cas:issn">2090-2182</span>.
    </div><div class="casAbstract">Human epidermal growth factor receptor 2 (HER2) is a member of the epidermal growth factor receptor family having tyrosine kinase activity.  Dimerization of the receptor results in the autophosphorylation of tyrosine residues within the cytoplasmic domain of the receptors and initiates a variety of signaling pathways leading to cell proliferation and tumorigenesis.  Amplification or overexpression of HER2 occurs in approximately 15-30% of breast cancers and 10-30% of gastric/gastroesophageal cancers and serves as a prognostic and predictive biomarker.  HER2 overexpression has also been seen in other cancers like ovary, endometrium, bladder, lung, colon, and head and neck.  The introduction of HER2 directed therapies has dramatically influenced the outcome of patients with HER2 positive breast and gastric/gastroesophageal cancers; however, the results have been proved disappointing in other HER2 overexpressing cancers.  This review discusses the role of HER2 in various cancers and therapeutic modalities available targeting HER2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQjSP2nh_04ir0AGQpJZWpJfW6udTcc2eZAHzp1TkWE2rntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2M7mvFKnsw%253D%253D&md5=357ce8a0bf1856baea75f2de51110204</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1155%2F2014%2F852748&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1155%252F2014%252F852748%26sid%3Dliteratum%253Aachs%26aulast%3DIqbal%26aufirst%3DN.%26aulast%3DIqbal%26aufirst%3DN.%26atitle%3DHuman%2520epidermal%2520growth%2520factor%2520receptor%25202%2520%2528HER2%2529%2520in%2520cancers%253A%2520Overexpression%2520and%2520therapeutic%2520implications%26jtitle%3DMol.%2520Biol.%2520Int.%26date%3D2014%26volume%3D2014%26spage%3D852748%26doi%3D10.1155%2F2014%2F852748" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref67"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref67'); return false;" data-citation="" class="refNumLink">67</a></strong><div class="NLM_citation" id="cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pinho, F. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frampton, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nunes, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krell, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alshaker, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacob, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pellegrino, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roca-Alonso, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Giorgio, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harding, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waxman, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stebbing, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pchejetski, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castellano, L.</span></span> <span> </span><span class="NLM_article-title">Downregulation of microRNA-515â5p by the Estrogen Receptor Modulates Sphingosine Kinase 1 and Breast Cancer Cell Proliferation</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>73</i></span>,  <span class="NLM_fpage">5936</span>â <span class="NLM_lpage">5948</span>, <span class="refDoi">Â DOI: 10.1158/0008-5472.CAN-13-0158</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1158%2F0008-5472.CAN-13-0158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=23928990" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsFGrsrvF" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2013&pages=5936-5948&author=F.+G.+Pinhoauthor=A.+E.+Framptonauthor=J.+Nunesauthor=J.+Krellauthor=H.+Alshakerauthor=J.+Jacobauthor=L.+Pellegrinoauthor=L.+Roca-Alonsoauthor=A.+de+Giorgioauthor=V.+Hardingauthor=J.+Waxmanauthor=J.+Stebbingauthor=D.+Pchejetskiauthor=L.+Castellano&title=Downregulation+of+microRNA-515%E2%80%935p+by+the+Estrogen+Receptor+Modulates+Sphingosine+Kinase+1+and+Breast+Cancer+Cell+Proliferation&doi=10.1158%2F0008-5472.CAN-13-0158"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">Downregulation of microRNA-515-5p by the Estrogen Receptor Modulates Sphingosine Kinase 1 and Breast Cancer Cell Proliferation</span></div><div class="casAuthors">Pinho, Filipa G.; Frampton, Adam E.; Nunes, Joao; Krell, Jonathan; Alshaker, Heba; Jacob, Jimmy; Pellegrino, Loredana; Roca-Alonso, Laura; de Giorgio, Alexander; Harding, Victoria; Waxman, Jonathan; Stebbing, Justin; Pchejetski, Dmitry; Castellano, Leandro</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">73</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">5936-5948</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Sphingosine kinase 1 (SKI) plays an important role in estrogen-dependent breast tumorigenesis, but its regulation is poorly understood.  A subset of microRNAs (miRNA, miR) is regulated by estrogen and contributes to cellular proliferation and cancer progression.  Here, we describe that miR-515-5p is transcriptionally repressed by estrogen receptor Î± (ERÎ±) and functions as a tumor suppressor in breast cancer.  Its downregulation enhances cell proliferation and estrogen-dependent SK1 activity, mediated by a redn. of miR-515-5p posttranscriptional repression.  Enforced expression of miR-515-5p in breast cancer cells causes a redn. in SK1 activity, reduced cell proliferation, and the induction of caspase-dependent apoptosis.  Conversely, opposing effects occur with miR-515-5p inhibition and by SK1 silencing.  Notably, we show that estradiol (E2) treatment downregulates miR-515-5p levels, whereas the antiestrogen tamoxifen causes a decrease in SK1, which is rescued by silencing miR-515-5p.  Anal. of chromatin immunopptn. sequencing (ChIP-Seq) data reveals that miR-515-5p suppression is mediated by a direct interaction of ERÎ± within its promoter.  RNA-sequencing (RNA-Seq) anal. of breast cancer cells after overexpressing miR-515-5p indicates that it partly modulates cell proliferation by regulating the Wnt pathway.  The clin. implications of this novel regulatory system are shown as miR-515-5p is significantly downregulated in ER-pos. (n = 146) compared with ER-neg. (n = 98) breast cancers.  Overall, we identify a new link between ERÎ±, miR-515-5p, proliferation, and apoptosis in breast cancer tumorigenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6tmWk7fwlsLVg90H21EOLACvtfcHk0ljfrlsdzpgluQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsFGrsrvF&md5=ff477e1ad93278ed39733f48b84af59d</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-13-0158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-13-0158%26sid%3Dliteratum%253Aachs%26aulast%3DPinho%26aufirst%3DF.%2BG.%26aulast%3DFrampton%26aufirst%3DA.%2BE.%26aulast%3DNunes%26aufirst%3DJ.%26aulast%3DKrell%26aufirst%3DJ.%26aulast%3DAlshaker%26aufirst%3DH.%26aulast%3DJacob%26aufirst%3DJ.%26aulast%3DPellegrino%26aufirst%3DL.%26aulast%3DRoca-Alonso%26aufirst%3DL.%26aulast%3Dde%2BGiorgio%26aufirst%3DA.%26aulast%3DHarding%26aufirst%3DV.%26aulast%3DWaxman%26aufirst%3DJ.%26aulast%3DStebbing%26aufirst%3DJ.%26aulast%3DPchejetski%26aufirst%3DD.%26aulast%3DCastellano%26aufirst%3DL.%26atitle%3DDownregulation%2520of%2520microRNA-515%25E2%2580%25935p%2520by%2520the%2520Estrogen%2520Receptor%2520Modulates%2520Sphingosine%2520Kinase%25201%2520and%2520Breast%2520Cancer%2520Cell%2520Proliferation%26jtitle%3DCancer%2520Res.%26date%3D2013%26volume%3D73%26spage%3D5936%26epage%3D5948%26doi%3D10.1158%2F0008-5472.CAN-13-0158" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref68"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref68'); return false;" data-citation="" class="refNumLink">68</a></strong><div class="NLM_citation" id="cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Costales, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoch, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abegg, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Childs-Disney, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Velagapudi, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adibekian, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span> <span> </span><span class="NLM_article-title">A designed small molecule inhibitor of a non-coding RNA sensitizes HER2 negative cancers to herceptin</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>141</i></span>,  <span class="NLM_fpage">2960</span>â <span class="NLM_lpage">2974</span>, <span class="refDoi">Â DOI: 10.1021/jacs.8b10558</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.8b10558" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC1MXisVGmtrY%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=141&publication_year=2019&pages=2960-2974&author=M.+G.+Costalesauthor=D.+G.+Hochauthor=D.+Abeggauthor=J.+L.+Childs-Disneyauthor=S.+P.+Velagapudiauthor=A.+Adibekianauthor=M.+D.+Disney&title=A+designed+small+molecule+inhibitor+of+a+non-coding+RNA+sensitizes+HER2+negative+cancers+to+herceptin&doi=10.1021%2Fjacs.8b10558"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">A designed small molecule inhibitor of a non-coding RNA sensitizes HER2 negative cancers to herceptin</span></div><div class="casAuthors">Costales, Matthew G.; Hoch, Dominic G.; Abegg, Daniel; Childs-Disney, Jessica L.; Velagapudi, Sai Pradeep; Adibekian, Alexander; Disney, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">141</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2960-2974</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A small mol. (1) with overlapping affinity for two microRNA (miRNA) precursors was used to inform design of a dimeric compd. (2 (I)) selective for one of the miRNAs.  In particular, I selectively targets the microRNA(miR)-515 hairpin precursor to inhibit prodn. of miR-515 that represses sphingosine kinase 1 (SK1), a key enzyme in the biosynthesis of sphingosine 1-phosphate (S1P).  Application of I to breast cancer cells enhanced SK1 and S1P levels, triggering a migratory phenotype.  Knockout of SK1, forced overexpression of miR-515, and application of a small mol. SK1 inhibitor all ablated I's effect on phenotype, consistent with its designed mode of action.  Target profiling studies via Chem-CLIP showed that I bound selectively to the miR-515 hairpin precursor in cells.  Global neoprotein synthesis upon addn. of I to MCF-7 breast cancer cells demonstrated I's selectivity and upregulation of cancer-assocd. proteins regulated by S1P.  The most upregulated protein was human epidermal growth factor receptor 2 (ERBB2/HER2), which is regulated by the SK1/S1P pathway and is normally not expressed in MCF-7 cells.  Like triple neg. breast cancer (TNBC) cells, the lack of HER2 renders them insusceptible to Herceptin and its antibody-drug conjugate Kadcyla.  In addn. to proteomics, an RNA-seq study supports that I has limited off target effects and other studies support that I is more selective than an oligonucleotide.  We therefore hypothesized that I could sensitize MCF-7 cells to anti-HER2 therapies.  Indeed, application of I sensitized cells to Herceptin.  These results were confirmed in two other cell lines that express miR-515 and are HER2-, the hepatocellular carcinoma cell line HepG2 and the TNBC line MDA-MB-231.  Importantly, normal breast epithelial cells (MCF-10A) that do not express miR-515 are not affected by I.  These observations suggest a precision medicine approach to sensitize HER2- cancers to approved anticancer medicines.  This study has implications for broadening the therapeutic utility of known targeted cancer therapeutics by using a secondary targeted approach to render otherwise insensitive cells, sensitive to a targeted therapeutic.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoaWGyvVjeHDrVg90H21EOLACvtfcHk0ljfrlsdzpgluQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXisVGmtrY%253D&md5=1e2f57da2a12fe57d6f60fa44e3e79e2</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1021%2Fjacs.8b10558&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.8b10558%26sid%3Dliteratum%253Aachs%26aulast%3DCostales%26aufirst%3DM.%2BG.%26aulast%3DHoch%26aufirst%3DD.%2BG.%26aulast%3DAbegg%26aufirst%3DD.%26aulast%3DChilds-Disney%26aufirst%3DJ.%2BL.%26aulast%3DVelagapudi%26aufirst%3DS.%2BP.%26aulast%3DAdibekian%26aufirst%3DA.%26aulast%3DDisney%26aufirst%3DM.%2BD.%26atitle%3DA%2520designed%2520small%2520molecule%2520inhibitor%2520of%2520a%2520non-coding%2520RNA%2520sensitizes%2520HER2%2520negative%2520cancers%2520to%2520herceptin%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2019%26volume%3D141%26spage%3D2960%26epage%3D2974%26doi%3D10.1021%2Fjacs.8b10558" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref69"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref69'); return false;" data-citation="" class="refNumLink">69</a></strong><div class="NLM_citation" id="cit69"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Agrawal, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dasaradhi, P. V. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohmmed, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malhotra, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhatnagar, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mukherjee, S. K.</span></span> <span> </span><span class="NLM_article-title">RNA Interference: Biology, mechanism, and applications</span>. <i>Microbiol. Mol. Biol. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">657</span>â <span class="NLM_lpage">685</span>, <span class="refDoi">Â DOI: 10.1128/MMBR.67.4.657-685.2003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1128%2FMMBR.67.4.657-685.2003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=14665679" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BD2cXmt1Sqsw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2003&pages=657-685&author=N.+Agrawalauthor=P.+V.+N.+Dasaradhiauthor=A.+Mohmmedauthor=P.+Malhotraauthor=R.+K.+Bhatnagarauthor=S.+K.+Mukherjee&title=RNA+Interference%3A+Biology%2C+mechanism%2C+and+applications&doi=10.1128%2FMMBR.67.4.657-685.2003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">RNA interference: biology, mechanism, and applications</span></div><div class="casAuthors">Agrawal, Neema; Dasaradhi, P. V. N.; Mohmmed, Asif; Malhotra, Pawan; Bhatnagar, Raj K.; Mukherjee, Sunil K.</div><div class="citationInfo"><span class="NLM_cas:title">Microbiology and Molecular Biology Reviews</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">657-685</span>CODEN:
                <span class="NLM_cas:coden">MMBRF7</span>;
        ISSN:<span class="NLM_cas:issn">1092-2172</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">A review.  Double-stranded RNA-mediated interference (RNAi) is a simple and rapid method of silencing gene expression in a range of organisms.  The silencing of a gene is a consequence of degrdn. of RNA into short RNAs that activate RNases to target homologous mRNA.  The resulting phenotypes either are identical to those of genetic null mutants or resemble an allelic series of mutants.  Specific gene silencing has been shown to be related to two ancient processes, cosuppression in plants and quelling in fungi, and has also been assocd. with regulatory processes such as transposon silencing, antiviral defense mechanisms, gene regulation, and chromosomal modification.  Extensive genetic and biochem. anal. revealed a two-step mechanism of RNAi-induced gene silencing.  The first step involves degrdn. of dsRNA into small interfering RNAs (siRNAs), 21 to 25 nucleotides long, by an RNase III-like activity.  In the second step, the siRNAs join an RNase complex, RISC (RNA-induced silencing complex), which acts on the cognate mRNA and degrades it.  Several key components such as Dicer, RNA-dependent RNA polymerase, helicases, and dSRNA endonucleases have been identified in different organisms for their roles in RNAi.  Some of these components also control the development of many organisms by processing many noncoding RNAs, called micro-RNAs.  The biogenesis and function of micro-RNAs resemble RNAi activities to a large extent.  Recent studies indicate that in the context of RNAi, the genome also undergoes alterations in the form of DNA methylation, heterochromatin formation, and programmed DNA elimination.  As a result of these changes, the silencing effect of gene functions is exercised as tightly as possible.  Because of its exquisite specificity and efficiency, RNAi is being considered as an important tool not only for functional genomics, but also for gene-specific therapeutic activities that target the mRNAs of disease-related genes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_kUuvijBt6rVg90H21EOLACvtfcHk0ljfrlsdzpgluQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXmt1Sqsw%253D%253D&md5=94f3955a1f0ecbd9712424d2db968eb9</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1128%2FMMBR.67.4.657-685.2003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMMBR.67.4.657-685.2003%26sid%3Dliteratum%253Aachs%26aulast%3DAgrawal%26aufirst%3DN.%26aulast%3DDasaradhi%26aufirst%3DP.%2BV.%2BN.%26aulast%3DMohmmed%26aufirst%3DA.%26aulast%3DMalhotra%26aufirst%3DP.%26aulast%3DBhatnagar%26aufirst%3DR.%2BK.%26aulast%3DMukherjee%26aufirst%3DS.%2BK.%26atitle%3DRNA%2520Interference%253A%2520Biology%252C%2520mechanism%252C%2520and%2520applications%26jtitle%3DMicrobiol.%2520Mol.%2520Biol.%2520Rev.%26date%3D2003%26volume%3D67%26spage%3D657%26epage%3D685%26doi%3D10.1128%2FMMBR.67.4.657-685.2003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref70"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref70'); return false;" data-citation="" class="refNumLink">70</a></strong><div class="NLM_citation" id="cit70"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Felicetti, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cecchetti, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manfroni, G.</span></span> <span> </span><span class="NLM_article-title">Modulating microRNA processing: Enoxacin, the progenitor of a new class of drugs</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">12275</span>â <span class="NLM_lpage">12289</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.0c00510</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00510" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtl2rsrnL" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=12275-12289&author=T.+Felicettiauthor=V.+Cecchettiauthor=G.+Manfroni&title=Modulating+microRNA+processing%3A+Enoxacin%2C+the+progenitor+of+a+new+class+of+drugs&doi=10.1021%2Facs.jmedchem.0c00510"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">Modulating microRNA Processing: Enoxacin, the Progenitor of a New Class of Drugs</span></div><div class="casAuthors">Felicetti, Tommaso; Cecchetti, Violetta; Manfroni, Giuseppe</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">12275-12289</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The RNA interference (RNAi) process encompasses the cellular mechanisms by which short-noncoding RNAs posttranscriptionally modulate gene expression.  First discovered in 1998, today RNAi represents the foundation underlying complex biol. mechanisms that are dysregulated in many diseases.  MicroRNAs are effector mols. of gene silencing in RNAi, and their modulation can lead to a wide response in cells.  Enoxacin was reported as the first and unique small-mol. enhancer of microRNA (SMER) maturation.  Herein, the biol. activity of enoxacin as SMER is discussed to shed light on its innovative mode of action, its potential in treating different diseases, and the feasibility of using enoxacin as a chem. template for inspiring medicinal chemists.  We debate its mechanism of action at the mol. level and the possible impact on future ligand and/or structure-guided chem. optimizations, as well as opportunities and drawbacks assocd. with the development of quinolones such as SMERs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrCQcmft_rGlLVg90H21EOLACvtfcHk0lhePz148IfZqg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtl2rsrnL&md5=81a379dd43abf95a4b23366c006db178</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00510&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00510%26sid%3Dliteratum%253Aachs%26aulast%3DFelicetti%26aufirst%3DT.%26aulast%3DCecchetti%26aufirst%3DV.%26aulast%3DManfroni%26aufirst%3DG.%26atitle%3DModulating%2520microRNA%2520processing%253A%2520Enoxacin%252C%2520the%2520progenitor%2520of%2520a%2520new%2520class%2520of%2520drugs%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D12275%26epage%3D12289%26doi%3D10.1021%2Facs.jmedchem.0c00510" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref71"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref71'); return false;" data-citation="" class="refNumLink">71</a></strong><div class="NLM_citation" id="cit71"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shan, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szulwach, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faghihi, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khalil, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paroo, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, A. W. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wahlestedt, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, P. A.</span></span> <span> </span><span class="NLM_article-title">Small molecule enhances RNA interference and promotes microRNA processing</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">933</span>â <span class="NLM_lpage">940</span>, <span class="refDoi">Â DOI: 10.1038/nbt.1481</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1038%2Fnbt.1481" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=18641635" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BD1cXps1Wls7c%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2008&pages=933-940&author=G.+Shanauthor=Y.+Liauthor=J.+Zhangauthor=W.+Liauthor=K.+E.+Szulwachauthor=R.+Duanauthor=M.+A.+Faghihiauthor=A.+M.+Khalilauthor=L.+Luauthor=Z.+Parooauthor=A.+W.+S.+Chanauthor=Z.+Shiauthor=Q.+Liuauthor=C.+Wahlestedtauthor=C.+Heauthor=P.+A.+Jin&title=Small+molecule+enhances+RNA+interference+and+promotes+microRNA+processing&doi=10.1038%2Fnbt.1481"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">A small molecule enhances RNA interference and promotes microRNA processing</span></div><div class="casAuthors">Shan, Ge; Li, Yujing; Zhang, Junliang; Li, Wendi; Szulwach, Keith E.; Duan, Ranhui; Faghihi, Mohammad A.; Khalil, Ahmad M.; Lu, Lianghua; Paroo, Zain; Chan, Anthony W. S.; Shi, Zhangjie; Liu, Qinghua; Wahlestedt, Claes; He, Chuan; Jin, Peng</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">933-940</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Little is known about the regulation of RNA interference (RNAi).  Shan et al. constructed a reporter system to monitor RNAi activity and identified a small mol. that increases RNAi by facilitating the interaction between small RNAs and a protein involved in small RNA loading and processing.  Little is known about the regulation of RNA interference (RNAi).  Shan et al. constructed a reporter system to monitor RNAi activity and identified a small mol. that increases RNAi by facilitating the interaction between small RNAs and a protein involved in small RNA loading and processing.  Small interfering RNAs (siRNAs) and microRNAs (miRNAs) are sequence-specific post-transcriptional regulators of gene expression.  Although major components of the RNA interference (RNAi) pathway have been identified, regulatory mechanisms for this pathway remain largely unknown.  Here we demonstrate that the RNAi pathway can be modulated intracellularly by small mols.  We have developed a cell-based assay to monitor the activity of the RNAi pathway and find that the small-mol. enoxacin (Penetrex) enhances siRNA-mediated mRNA degrdn. and promotes the biogenesis of endogenous miRNAs.  We show that this RNAi-enhancing activity depends on the trans-activation-responsive region RNA-binding protein.  Our results provide a proof-of-principle demonstration that small mols. can be used to modulate the activity of the RNAi pathway.  RNAi enhancers may be useful in the development of research tools and therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmMvLQLjUDALVg90H21EOLACvtfcHk0lhePz148IfZqg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXps1Wls7c%253D&md5=15210fd18afcb865b6d2e218449c42b4</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1038%2Fnbt.1481&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt.1481%26sid%3Dliteratum%253Aachs%26aulast%3DShan%26aufirst%3DG.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DSzulwach%26aufirst%3DK.%2BE.%26aulast%3DDuan%26aufirst%3DR.%26aulast%3DFaghihi%26aufirst%3DM.%2BA.%26aulast%3DKhalil%26aufirst%3DA.%2BM.%26aulast%3DLu%26aufirst%3DL.%26aulast%3DParoo%26aufirst%3DZ.%26aulast%3DChan%26aufirst%3DA.%2BW.%2BS.%26aulast%3DShi%26aufirst%3DZ.%26aulast%3DLiu%26aufirst%3DQ.%26aulast%3DWahlestedt%26aufirst%3DC.%26aulast%3DHe%26aufirst%3DC.%26aulast%3DJin%26aufirst%3DP.%2BA.%26atitle%3DSmall%2520molecule%2520enhances%2520RNA%2520interference%2520and%2520promotes%2520microRNA%2520processing%26jtitle%3DNat.%2520Biotechnol.%26date%3D2008%26volume%3D26%26spage%3D933%26epage%3D940%26doi%3D10.1038%2Fnbt.1481" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref72"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref72'); return false;" data-citation="" class="refNumLink">72</a></strong><div class="NLM_citation" id="cit72"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kung, J. T. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colognori, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. T.</span></span> <span> </span><span class="NLM_article-title">Long noncoding RNAs: past, present, and future</span>. <i>Genetics</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>193</i></span>,  <span class="NLM_fpage">651</span>â <span class="NLM_lpage">669</span>, <span class="refDoi">Â DOI: 10.1534/genetics.112.146704</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1534%2Fgenetics.112.146704" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=23463798" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtVWnu7zF" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=193&publication_year=2013&pages=651-669&author=J.+T.+Y.+Kungauthor=D.+Colognoriauthor=J.+T.+Lee&title=Long+noncoding+RNAs%3A+past%2C+present%2C+and+future&doi=10.1534%2Fgenetics.112.146704"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">Long noncoding RNAs: past, present, and future</span></div><div class="casAuthors">Kung, Johnny T. Y.; Colognori, David; Lee, Jeannie T.</div><div class="citationInfo"><span class="NLM_cas:title">Genetics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">193</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">651-669</span>CODEN:
                <span class="NLM_cas:coden">GENTAE</span>;
        ISSN:<span class="NLM_cas:issn">0016-6731</span>.
    
            (<span class="NLM_cas:orgname">Genetics Society of America</span>)
        </div><div class="casAbstract">A review.  Long noncoding RNAs (lncRNAs) have gained widespread attention in recent years as a potentially new and crucial layer of biol. regulation.  LncRNAs of all kinds have been implicated in a range of developmental processes and diseases, but knowledge of the mechanisms by which they act is still surprisingly limited, and claims that almost the entirety of the mammalian genome is transcribed into functional noncoding transcripts remain controversial.  At the same time, a small no. of well-studied lncRNAs have given us important clues about the biol. of these mols., and a few key functional and mechanistic themes have begun to emerge, although the robustness of these models and classification schemes remains to be seen.  Here, we review the current state of knowledge of the lncRNA field, discussing what is known about the genomic contexts, biol. functions, and mechanisms of action of lncRNAs.  We also reflect on how the recent interest in lncRNAs is deeply rooted in biol.'s longstanding concern with the evolution and function of genomes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqq-FrGArfXBbVg90H21EOLACvtfcHk0lh86kL-TUoRlg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtVWnu7zF&md5=7edc7db9a40409f02a59039068b3771d</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1534%2Fgenetics.112.146704&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1534%252Fgenetics.112.146704%26sid%3Dliteratum%253Aachs%26aulast%3DKung%26aufirst%3DJ.%2BT.%2BY.%26aulast%3DColognori%26aufirst%3DD.%26aulast%3DLee%26aufirst%3DJ.%2BT.%26atitle%3DLong%2520noncoding%2520RNAs%253A%2520past%252C%2520present%252C%2520and%2520future%26jtitle%3DGenetics%26date%3D2013%26volume%3D193%26spage%3D651%26epage%3D669%26doi%3D10.1534%2Fgenetics.112.146704" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref73"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref73'); return false;" data-citation="" class="refNumLink">73</a></strong><div class="NLM_citation" id="cit73"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span> <span> </span><span class="NLM_article-title">Associating lncRNAs with small molecules <i>via</i> bilevel optimization reveals cancer-related lncRNAs</span>. <i>PLoS Comput. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>15</i></span>, <span class="NLM_elocation-id">e1007540</span> <span class="refDoi">Â DOI: 10.1371/journal.pcbi.1007540</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1371%2Fjournal.pcbi.1007540" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=31877126" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BB3cXktlGmtb4%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2019&author=Y.+Wangauthor=S.+Chenauthor=L.+Chenauthor=Y.+Wang&title=Associating+lncRNAs+with+small+molecules+via+bilevel+optimization+reveals+cancer-related+lncRNAs&doi=10.1371%2Fjournal.pcbi.1007540"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">Associating lncRNAs with small molecules via bilevel optimization reveals cancer-related lncRNAs</span></div><div class="casAuthors">Wang, Yongcui; Chen, Shilong; Chen, Luonan; Wang, Yong</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Computational Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">e1007540/1-e1007540/20</span>CODEN:
                <span class="NLM_cas:coden">PCBLBG</span>;
        ISSN:<span class="NLM_cas:issn">1553-7358</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Long noncoding RNA (lncRNA) transcripts have emerging impacts in cancer studies, which suggests their potential as novel therapeutic agents.  However, the mol. mechanism behind their treatment effects is still unclear.  Here, we designed a computational model to Assoc. LncRNAs with Anti-Cancer Drugs (ALACD) based on a bilevel optimization model, which optimized the gene signature overlap in the upper level and imputed the missing lncRNA-gene assocn. in the lower level.  ALACD predicts genes coexpressed with lncRNAs mean while matching drug's gene signatures.  This model allows us to borrow the target gene information of small mols. to understand the mechanisms of action of lncRNAs and their roles in cancer.  The ALACD model was systematically applied to the 10 cancer types in The Cancer Genome Atlas (TCGA) that had matched lncRNA and mRNA expression data.  Cancer type-specific lncRNAs and assocd. drugs were identified.  These lncRNAs show significantly different expression levels in cancer patients.  Follow-up functional and mol. pathway anal. suggest the gene signatures bridging drugs and lncRNAs are closely related to cancer development.  Importantly, patient survival information and evidence from the literature suggest that the lncRNAs and drug-lncRNA assocns. identified by the ALACD model can provide an alternative choice for cancer targeting treatment and potential cancer pognostic biomarkers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6sEZB7SUrYrVg90H21EOLACvtfcHk0lh86kL-TUoRlg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXktlGmtb4%253D&md5=208e9f72a9931bb1adc6884acb10ad10</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pcbi.1007540&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pcbi.1007540%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DY.%26atitle%3DAssociating%2520lncRNAs%2520with%2520small%2520molecules%2520via%2520bilevel%2520optimization%2520reveals%2520cancer-related%2520lncRNAs%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2019%26volume%3D15%26doi%3D10.1371%2Fjournal.pcbi.1007540" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref74"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref74'); return false;" data-citation="" class="refNumLink">74</a></strong><div class="NLM_citation" id="cit74"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ji, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diederichs, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boing, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Metzger, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tidow, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brandt, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buerger, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bulk, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berdel, W. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Serve, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller-Tidow, C.</span></span> <span> </span><span class="NLM_article-title">MALAT-1, a novel noncoding RNA, and thymosin beta4 predict metastasis and survival in early-stage non-small cell lung cancer</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">8031</span>â <span class="NLM_lpage">8041</span>, <span class="refDoi">Â DOI: 10.1038/sj.onc.1206928</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1038%2Fsj.onc.1206928" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=12970751" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A280%3ADC%252BD3svivFOgsg%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2003&pages=8031-8041&author=P.+Jiauthor=S.+Diederichsauthor=W.+Wangauthor=S.+Boingauthor=R.+Metzgerauthor=P.+M.+Schneiderauthor=N.+Tidowauthor=B.+Brandtauthor=H.+Buergerauthor=E.+Bulkauthor=M.+Thomasauthor=W.+E.+Berdelauthor=H.+Serveauthor=C.+Muller-Tidow&title=MALAT-1%2C+a+novel+noncoding+RNA%2C+and+thymosin+beta4+predict+metastasis+and+survival+in+early-stage+non-small+cell+lung+cancer&doi=10.1038%2Fsj.onc.1206928"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit74R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">MALAT-1, a novel noncoding RNA, and thymosin beta4 predict metastasis and survival in early-stage non-small cell lung cancer</span></div><div class="casAuthors">Ji Ping; Diederichs Sven; Wang Wenbing; Boing Sebastian; Metzger Ralf; Schneider Paul M; Tidow Nicola; Brandt Burkhard; Buerger Horst; Bulk Etmar; Thomas Michael; Berdel Wolfgang E; Serve Hubert; Muller-Tidow Carsten</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">39</span>),
    <span class="NLM_cas:pages">8031-41</span>
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    </div><div class="casAbstract">Early-stage non-small cell lung cancer (NSCLC) can be cured by surgical resection, but a substantial fraction of patients ultimately dies due to distant metastasis.  In this study, we used subtractive hybridization to identify gene expression differences in stage I NSCLC tumors that either did or did not metastasize in the course of disease.  Individual clones (n=225) were sequenced and quantitative RT-PCR verified overexpression in metastasizing samples.  Several of the identified genes (eIF4A1, thymosin beta4 and a novel transcript named MALAT-1) were demonstrated to be significantly associated with metastasis in NSCLC patients (n=70).  The genes' association with metastasis was stage- and histology specific.  The Kaplan-Meier analyses identified MALAT-1 and thymosin beta4 as prognostic parameters for patient survival in stage I NSCLC.  The novel MALAT-1 transcript is a noncoding RNA of more than 8000 nt expressed from chromosome 11q13.  It is highly expressed in lung, pancreas and other healthy organs as well as in NSCLC.  MALAT-1 expressed sequences are conserved across several species indicating its potentially important function.  Taken together, these data contribute to the identification of early-stage NSCLC patients that are at high risk to develop metastasis.  The identification of MALAT-1 emphasizes the potential role of noncoding RNAs in human cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcT4lotbLB7vyCA3UY9EwTNrfW6udTcc2eaBuce5n25wMrntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD3svivFOgsg%253D%253D&md5=2fce458717370f8fc36ff1b85c4a5b9c</span></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1038%2Fsj.onc.1206928&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.onc.1206928%26sid%3Dliteratum%253Aachs%26aulast%3DJi%26aufirst%3DP.%26aulast%3DDiederichs%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DBoing%26aufirst%3DS.%26aulast%3DMetzger%26aufirst%3DR.%26aulast%3DSchneider%26aufirst%3DP.%2BM.%26aulast%3DTidow%26aufirst%3DN.%26aulast%3DBrandt%26aufirst%3DB.%26aulast%3DBuerger%26aufirst%3DH.%26aulast%3DBulk%26aufirst%3DE.%26aulast%3DThomas%26aufirst%3DM.%26aulast%3DBerdel%26aufirst%3DW.%2BE.%26aulast%3DServe%26aufirst%3DH.%26aulast%3DMuller-Tidow%26aufirst%3DC.%26atitle%3DMALAT-1%252C%2520a%2520novel%2520noncoding%2520RNA%252C%2520and%2520thymosin%2520beta4%2520predict%2520metastasis%2520and%2520survival%2520in%2520early-stage%2520non-small%2520cell%2520lung%2520cancer%26jtitle%3DOncogene%26date%3D2003%26volume%3D22%26spage%3D8031%26epage%3D8041%26doi%3D10.1038%2Fsj.onc.1206928" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref75"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref75'); return false;" data-citation="" class="refNumLink">75</a></strong><div class="NLM_citation" id="cit75"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ghafouri-Fard, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hafez, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taheri, M.</span></span> <span> </span><span class="NLM_article-title">Metastasis associated lung adenocarcinoma transcript 1: An update on expression pattern and functions in carcinogenesis</span>. <i>Exp. Mol. Pathol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>112</i></span>,  <span class="NLM_fpage">104330</span>, <span class="refDoi">Â DOI: 10.1016/j.yexmp.2019.104330</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1016%2Fj.yexmp.2019.104330" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=31712117" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC1MXit1SmsrfO" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=112&publication_year=2020&pages=104330&author=S.+Ghafouri-Fardauthor=A.+A.+Hafezauthor=M.+Taheri&title=Metastasis+associated+lung+adenocarcinoma+transcript+1%3A+An+update+on+expression+pattern+and+functions+in+carcinogenesis&doi=10.1016%2Fj.yexmp.2019.104330"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit75R"><div class="casContent"><span class="casTitleNuber">75</span><div class="casTitle"><span class="NLM_cas:atitle">Metastasis Associated Lung Adenocarcinoma Transcript 1: An update on expression pattern and functions in carcinogenesis</span></div><div class="casAuthors">Ghafouri-Fard, Soudeh; Ashrafi Hafez, Asghar; Taheri, Mohammad</div><div class="citationInfo"><span class="NLM_cas:title">Experimental and Molecular Pathology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">112</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">104330</span>CODEN:
                <span class="NLM_cas:coden">EXMPA6</span>;
        ISSN:<span class="NLM_cas:issn">0014-4800</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review.  The Metastasis Assocd. Lung Adenocarcinoma Transcript 1 (MALAT1) is among long non-coding RNAs (lncRNAs) which has disapproved the old term of "junk DNA" which was used for majority of human genome which are not transcribed to proteins.  An extensive portion of literature points to the fundamental role of this lncRNA in tumorigenesis process of diverse cancers ranging from solid tumors to leukemia.  Being firstly identified in lung cancer, it has prognostic and diagnostic values in several cancer types.  Consistent with the proposed oncogenic roles for this lncRNA, most of studies have shown up-regulation of MALAT1 in malignant tissues compared with non-malignant/normal tissues of the same source.  However, few studies have shown down-regulation of MALAT1 in breast cancer, endometrial cancer, colorectal cancer and glioma.  In the current study, we have conducted a comprehensive literature search and provided an up-date on the role of MALAT1 in cancer biol.  Our investigation underscores a potential role as a diagnostic/prognostic marker and a putative therapeutic target for MALAT1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPW40Uh_BlXrVg90H21EOLACvtfcHk0lg3ZNNPDVprcw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXit1SmsrfO&md5=bfa0660f0f74b5f82f177c122ba2e17c</span></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1016%2Fj.yexmp.2019.104330&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.yexmp.2019.104330%26sid%3Dliteratum%253Aachs%26aulast%3DGhafouri-Fard%26aufirst%3DS.%26aulast%3DHafez%26aufirst%3DA.%2BA.%26aulast%3DTaheri%26aufirst%3DM.%26atitle%3DMetastasis%2520associated%2520lung%2520adenocarcinoma%2520transcript%25201%253A%2520An%2520update%2520on%2520expression%2520pattern%2520and%2520functions%2520in%2520carcinogenesis%26jtitle%3DExp.%2520Mol.%2520Pathol.%26date%3D2020%26volume%3D112%26spage%3D104330%26doi%3D10.1016%2Fj.yexmp.2019.104330" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref76"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref76'); return false;" data-citation="" class="refNumLink">76</a></strong><div class="NLM_citation" id="cit76"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Donlic, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morgan, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roble, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hargrove, A. E.</span></span> <span> </span><span class="NLM_article-title">Discovery of small molecule ligands for MALAT1 by tuning an RNA-binding scaffold</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">13242</span>â <span class="NLM_lpage">13247</span>, <span class="refDoi">Â DOI: 10.1002/anie.201808823</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1002%2Fanie.201808823" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhs1Omtb%252FM" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2018&pages=13242-13247&author=A.+Donlicauthor=B.+S.+Morganauthor=J.+L.+Xuauthor=A.+Liuauthor=C.+Robleauthor=A.+E.+Hargrove&title=Discovery+of+small+molecule+ligands+for+MALAT1+by+tuning+an+RNA-binding+scaffold&doi=10.1002%2Fanie.201808823"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit76R"><div class="casContent"><span class="casTitleNuber">76</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Small Molecule Ligands for MALAT1 by Tuning an RNA-Binding Scaffold</span></div><div class="casAuthors">Donlic, Anita; Morgan, Brittany S.; Xu, Jason L.; Liu, Anqi; Roble, Carlos Jr.; Hargrove, Amanda E.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">40</span>),
    <span class="NLM_cas:pages">13242-13247</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Structural studies of the 3'-end of the oncogenic long non-coding RNA metastasis-assocd. lung adenocarcinoma transcript 1 (MALAT1) confirmed a unique triple-helix structure.  This structure enables accumulation of the transcript, and high levels of MALAT1 are found in several cancers.  Here, the authors synthesize a small mol. library based on an RNA-binding scaffold, diphenylfuran (DPF), screen it against a variety of nucleic acid constructs, and demonstrate that the MALAT1 triple helix can be selectively targeted with small mols.  Computational anal. revealed a trend between subunit positioning and compn. on DPF shape and intramol. interactions, which in turn generally correlated with selectivity and binding strengths.  This work thus provides design strategies toward chem. probe development for the MALAT1 triple helix and suggests that comprehensive analyses of RNA-focused libraries can generate insights into selective RNA recognition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqudyg9LTHbgLVg90H21EOLACvtfcHk0lg3ZNNPDVprcw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhs1Omtb%252FM&md5=181570e797ff7aef7444441387eadf30</span></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.1002%2Fanie.201808823&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201808823%26sid%3Dliteratum%253Aachs%26aulast%3DDonlic%26aufirst%3DA.%26aulast%3DMorgan%26aufirst%3DB.%2BS.%26aulast%3DXu%26aufirst%3DJ.%2BL.%26aulast%3DLiu%26aufirst%3DA.%26aulast%3DRoble%26aufirst%3DC.%26aulast%3DHargrove%26aufirst%3DA.%2BE.%26atitle%3DDiscovery%2520of%2520small%2520molecule%2520ligands%2520for%2520MALAT1%2520by%2520tuning%2520an%2520RNA-binding%2520scaffold%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2018%26volume%3D57%26spage%3D13242%26epage%3D13247%26doi%3D10.1002%2Fanie.201808823" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref77"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref77'); return false;" data-citation="" class="refNumLink">77</a></strong><div class="NLM_citation" id="cit77"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chaires, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamelberg, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pandya, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boykin, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, W. D.</span></span> <span> </span><span class="NLM_article-title">Structural selectivity of aromatic diamidines</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">5729</span>â <span class="NLM_lpage">5742</span>, <span class="refDoi">Â DOI: 10.1021/jm049491e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm049491e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BD2cXotVCisL0%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=5729-5742&author=J.+B.+Chairesauthor=J.+Renauthor=D.+Hamelbergauthor=A.+Kumarauthor=V.+Pandyaauthor=D.+W.+Boykinauthor=W.+D.+Wilson&title=Structural+selectivity+of+aromatic+diamidines&doi=10.1021%2Fjm049491e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit77R"><div class="casContent"><span class="casTitleNuber">77</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Selectivity of Aromatic Diamidines</span></div><div class="casAuthors">Chaires, Jonathan B.; Ren, Jinsong; Hamelberg, Donald; Kumar, Arvind; Pandya, Vandna; Boykin, David W.; Wilson, W. David</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">5729-5742</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Competition dialysis was used to study the interactions of 13 substituted arom. diamidine compds. with 13 nucleic acid structures and sequences.  The results show a striking selectivity of these compds. for the triplex structure poly dA:(poly dT)2, a novel aspect of their interaction with nucleic acids not previously described.  The triplex selectivity of selected compds. was confirmed by thermal denaturation studies.  Triplex selectivity was found to be modulated by the location of amidine substituents on the core phenyl-furan-Ph ring scaffold.  Mol. models were constructed to rationalize the triplex selectivity of DB359, the most selective compd. in the series.  Its triplex selectivity was found to arise from optimal ring stacking on base triplets, along with proper positioning of its amidine substituents to occupy the minor and the major-minor grooves of the triplex.  New insights into the mol. recognition of nucleic acid structures emerged from these studies, adding to the list of available design principles for selectively targeting DNA and RNA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8-gzBJdMjo7Vg90H21EOLACvtfcHk0lg3ZNNPDVprcw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXotVCisL0%253D&md5=303d146fba1b5e650230142c955ad530</span></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=10.1021%2Fjm049491e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm049491e%26sid%3Dliteratum%253Aachs%26aulast%3DChaires%26aufirst%3DJ.%2BB.%26aulast%3DRen%26aufirst%3DJ.%26aulast%3DHamelberg%26aufirst%3DD.%26aulast%3DKumar%26aufirst%3DA.%26aulast%3DPandya%26aufirst%3DV.%26aulast%3DBoykin%26aufirst%3DD.%2BW.%26aulast%3DWilson%26aufirst%3DW.%2BD.%26atitle%3DStructural%2520selectivity%2520of%2520aromatic%2520diamidines%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D5729%26epage%3D5742%26doi%3D10.1021%2Fjm049491e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref78"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref78'); return false;" data-citation="" class="refNumLink">78</a></strong><div class="NLM_citation" id="cit78"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abulwerdi, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ageeli, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yonkunas, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arun, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nam, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneekloth, J. S.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dayie, T. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spector, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baird, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le Grice, S. F. J.</span></span> <span> </span><span class="NLM_article-title">Selective small-molecule targeting of a triple helix encoded by the long noncoding RNA, MALAT1</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">223</span>â <span class="NLM_lpage">235</span>, <span class="refDoi">Â DOI: 10.1021/acschembio.8b00807</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.8b00807" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC1MXltVWjsw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2019&pages=223-235&author=F.+A.+Abulwerdiauthor=W.+Xuauthor=A.+A.+Ageeliauthor=M.+J.+Yonkunasauthor=G.+Arunauthor=H.+Namauthor=J.+S.+Schneeklothauthor=T.+K.+Dayieauthor=D.+Spectorauthor=N.+Bairdauthor=S.+F.+J.+Le+Grice&title=Selective+small-molecule+targeting+of+a+triple+helix+encoded+by+the+long+noncoding+RNA%2C+MALAT1&doi=10.1021%2Facschembio.8b00807"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit78R"><div class="casContent"><span class="casTitleNuber">78</span><div class="casTitle"><span class="NLM_cas:atitle">Selective small-molecule targeting of a triple helix encoded by the long noncoding RNA, MALAT1</span></div><div class="casAuthors">Abulwerdi, Fardokht A.; Xu, Wenbo; Ageeli, Abeer A.; Yonkunas, Michael J.; Arun, Gayatri; Nam, Hyeyeon; Schneekloth, John S.; Dayie, Theodore Kwaku; Spector, David; Baird, Nathan; Le Grice, Stuart F. J.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">223-235</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Metastasis-assocd. lung adenocarcinoma transcript 1 (Malat1/MALAT1, mouse/human), a highly conserved long noncoding (lnc) RNA, has been linked with several physiol. processes, including the alternative splicing, nuclear organization, and epigenetic modulation of gene expression.  MALAT1 has also been implicated in metastasis and tumor proliferation in multiple cancer types.  The 3' terminal stability element for nuclear expression (ENE) assumes a triple-helical configuration that promotes its nuclear accumulation and persistent function.  Utilizing a novel small mol. microarray strategy, we identified multiple Malat1 ENE triplex-binding chemotypes, among which compds. I and II reduced Malat1 RNA levels and branching morphogenesis in a mammary tumor organoid model.  Computational modeling and F.ovrddot.orster resonance energy transfer expts. demonstrate distinct binding modes for each chemotype, conferring opposing structural changes to the triplex.  I modulates Malat1 downstream genes without affecting Neat1, a nuclear lncRNA encoded in the same chromosomal region as Malat1 with a structurally similar ENE triplex.  Supporting this observation, the specificity of I for Malat1 over Neat1 and a virus-coded ENE was demonstrated by NMR spectroscopy.  Small mols. specifically targeting the MALAT1 ENE triplex lay the foundation for new classes of anticancer therapeutics and mol. probes for the treatment and investigation of MALAT1-driven cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqfz6g3iAm4orVg90H21EOLACvtfcHk0lib2VgY59da7Q"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXltVWjsw%253D%253D&md5=ce56033fbebf0fc32991901faa24766f</span></div><a href="/servlet/linkout?suffix=cit78&amp;dbid=16384&amp;doi=10.1021%2Facschembio.8b00807&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.8b00807%26sid%3Dliteratum%253Aachs%26aulast%3DAbulwerdi%26aufirst%3DF.%2BA.%26aulast%3DXu%26aufirst%3DW.%26aulast%3DAgeeli%26aufirst%3DA.%2BA.%26aulast%3DYonkunas%26aufirst%3DM.%2BJ.%26aulast%3DArun%26aufirst%3DG.%26aulast%3DNam%26aufirst%3DH.%26aulast%3DSchneekloth%26aufirst%3DJ.%2BS.%26aulast%3DDayie%26aufirst%3DT.%2BK.%26aulast%3DSpector%26aufirst%3DD.%26aulast%3DBaird%26aufirst%3DN.%26aulast%3DLe%2BGrice%26aufirst%3DS.%2BF.%2BJ.%26atitle%3DSelective%2520small-molecule%2520targeting%2520of%2520a%2520triple%2520helix%2520encoded%2520by%2520the%2520long%2520noncoding%2520RNA%252C%2520MALAT1%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2019%26volume%3D14%26spage%3D223%26epage%3D235%26doi%3D10.1021%2Facschembio.8b00807" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref79"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref79'); return false;" data-citation="" class="refNumLink">79</a></strong><div class="NLM_citation" id="cit79"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, P.</span></span> <span> </span><span class="NLM_article-title">Role and mechanism of the alkylglycerone phosphate synthase in suppressing the invasion potential of human glioma and hepatic carcinoma cells in vitro</span>. <i>Oncol. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">431</span>â <span class="NLM_lpage">436</span>, <span class="refDoi">Â DOI: 10.3892/or.2014.3189</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.3892%2For.2014.3189" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=24841318" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtlaitL%252FL" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2014&pages=431-436&author=Y.+Zhuauthor=L.+Zhuauthor=L.+Luauthor=L.+Zhangauthor=G.+Zhangauthor=Q.+Wangauthor=P.+Yang&title=Role+and+mechanism+of+the+alkylglycerone+phosphate+synthase+in+suppressing+the+invasion+potential+of+human+glioma+and+hepatic+carcinoma+cells+in+vitro&doi=10.3892%2For.2014.3189"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit79R"><div class="casContent"><span class="casTitleNuber">79</span><div class="casTitle"><span class="NLM_cas:atitle">Role and mechanism of the alkylglycerone phosphate synthase in suppressing the invasion potential of human glioma and hepatic carcinoma cells in vitro</span></div><div class="casAuthors">Zhu, Yu; Zhu, Lielie; Lu, Ling; Zhang, Ling; Zhang, Guodong; Wang, Qin; Yang, Ping</div><div class="citationInfo"><span class="NLM_cas:title">Oncology Reports</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">431-436</span>CODEN:
                <span class="NLM_cas:coden">OCRPEW</span>;
        ISSN:<span class="NLM_cas:issn">1021-335X</span>.
    
            (<span class="NLM_cas:orgname">Spandidos Publications Ltd.</span>)
        </div><div class="casAbstract">Ether lipids have been implicated in the exacerbation of human tumors.  Accumulating evidence suggests that the alkylglycerone phosphate synthase (AGPS) is involved in the suppression of some types of tumor.  However, the role and mol. mechanism of AGPS in the invasion of human glioma and hepatic carcinoma remain unclear.  In the present study, using AGPS-knockdown human glioma U87 and hepatic carcinoma HepG2 cell lines, we explored the role of AGPS, as well as its mol. mechanism, in invasion in vitro.  It was demonstrated that silencing AGPS expression resulted in a decreased expression of cellular lipids such as LPA, LPAe and PGE2, adhesion, invasion potential and arrested cell cycle in tumor cells.  The expression of invasion-related genes such asMMP-2/9,E-cadherin and CD44 showed marked changes in AGPS-knockdown cells.  In addn., we found that AGPS regulated the activity of the MAPK pathway, as well as the transcriptional activity of Twist, AP-1, and Snail.  The results demonstrated that AGPS neg. regulated the invasion potential of glioma and hepatic carcinoma cells by modulating the expression of relevant genes and activity of the MAPK pathway.  Therefore, AGPS may be a potential glioma and hepatic carcinoma therapeutic target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptlN_wvt9D_7Vg90H21EOLACvtfcHk0lib2VgY59da7Q"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtlaitL%252FL&md5=00cadef12c58cf589894805500c09b65</span></div><a href="/servlet/linkout?suffix=cit79&amp;dbid=16384&amp;doi=10.3892%2For.2014.3189&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3892%252For.2014.3189%26sid%3Dliteratum%253Aachs%26aulast%3DZhu%26aufirst%3DY.%26aulast%3DZhu%26aufirst%3DL.%26aulast%3DLu%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DWang%26aufirst%3DQ.%26aulast%3DYang%26aufirst%3DP.%26atitle%3DRole%2520and%2520mechanism%2520of%2520the%2520alkylglycerone%2520phosphate%2520synthase%2520in%2520suppressing%2520the%2520invasion%2520potential%2520of%2520human%2520glioma%2520and%2520hepatic%2520carcinoma%2520cells%2520in%2520vitro%26jtitle%3DOncol.%2520Rep.%26date%3D2014%26volume%3D32%26spage%3D431%26epage%3D436%26doi%3D10.3892%2For.2014.3189" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref80"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref80'); return false;" data-citation="" class="refNumLink">80</a></strong><div class="NLM_citation" id="cit80"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Qian, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, Y.</span></span> <span> </span><span class="NLM_article-title">Computer-aided drug design and inhibitive effect of a novel nitrogenous heterocyclic compound and its mechanism on glioma U251 cells and breast cancer MCF-7 cells</span>. <i>Drug Des., Dev. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">1931</span>â <span class="NLM_lpage">1939</span>, <span class="refDoi">Â DOI: 10.2147/DDDT.S168130</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.2147%2FDDDT.S168130" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=29983547" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsVCrs77M" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2018&pages=1931-1939&author=L.+Qianauthor=Y.+Zhu&title=Computer-aided+drug+design+and+inhibitive+effect+of+a+novel+nitrogenous+heterocyclic+compound+and+its+mechanism+on+glioma+U251+cells+and+breast+cancer+MCF-7+cells&doi=10.2147%2FDDDT.S168130"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit80R"><div class="casContent"><span class="casTitleNuber">80</span><div class="casTitle"><span class="NLM_cas:atitle">Computer-aided drug design and inhibitive effect of a novel nitrogenous heterocyclic compound and its mechanism on glioma U251 cells and breast cancer MCF-7 cells</span></div><div class="casAuthors">Qian, Liyu; Zhu, Yu</div><div class="citationInfo"><span class="NLM_cas:title">Drug Design, Development and Therapy</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1931-1939</span>CODEN:
                <span class="NLM_cas:coden">DDDTAQ</span>;
        ISSN:<span class="NLM_cas:issn">1177-8881</span>.
    
            (<span class="NLM_cas:orgname">Dove Medical Press Ltd.</span>)
        </div><div class="casAbstract">Background: Glioma and breast cancer are severe malignant cancerous tumors that highlight the importance of developing new anti-cancer drugs.  The aim of this study was to explore the effects of a novel nitrogenous heterocyclic compd. on glioma and breast cancer cells and to det. its mechanism of action.  Methods: We designed and synthesized a novel nitrogenous heterocyclic compd., 3-(4-amino-1H-benzo[d]imidazole-2-carboxamido)-4-oxo-3,4-dihydroimidazo[5,1-d][1,2,3,5]tetrazine-8-carboxamide, based on alkylglycerone phosphate synthase (AGPS) using computer-aided drug design (CADD), and we measured its effect on the proliferation, invasion, cell cycle and apoptosis of U251 glioma and MCF-7 breast cancer cells.  In addn., the compd.'s effect on the expression of tumor-related mRNA, circular RNAs (circRNAs) and long non-coding RNAs (lncRNAs) was explored.  Results: It was found that the nitrogenous heterocyclic compd. could induce cell cycle arrest at the G2/M phase of U251/MCF-7 cells and activate apoptosis.  Real-time PCR showed that the expression levels of tumor-related mRNA, circRNAs and lncRNAs were impacted.  Conclusion: We concluded that the nitrogenous heterocyclic compd. inhibits the proliferation and invasion of U251 glioma and MCF-7 breast cancer cells through the induction of apoptosis and cell cycle arrest by regulating tumor-related genes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2LXvViUrFtrVg90H21EOLACvtfcHk0lgoVO0lrcKAag"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsVCrs77M&md5=51cf53d94b9b84d08ce9dac3f21df887</span></div><a href="/servlet/linkout?suffix=cit80&amp;dbid=16384&amp;doi=10.2147%2FDDDT.S168130&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2147%252FDDDT.S168130%26sid%3Dliteratum%253Aachs%26aulast%3DQian%26aufirst%3DL.%26aulast%3DZhu%26aufirst%3DY.%26atitle%3DComputer-aided%2520drug%2520design%2520and%2520inhibitive%2520effect%2520of%2520a%2520novel%2520nitrogenous%2520heterocyclic%2520compound%2520and%2520its%2520mechanism%2520on%2520glioma%2520U251%2520cells%2520and%2520breast%2520cancer%2520MCF-7%2520cells%26jtitle%3DDrug%2520Des.%252C%2520Dev.%2520Ther.%26date%3D2018%26volume%3D12%26spage%3D1931%26epage%3D1939%26doi%3D10.2147%2FDDDT.S168130" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref81"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref81'); return false;" data-citation="" class="refNumLink">81</a></strong><div class="NLM_citation" id="cit81"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gupta, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horlings, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsai, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Argani, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rinn, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brzoska, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kong, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">West, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van de
Vijver, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sukumar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, H. Y.</span></span> <span> </span><span class="NLM_article-title">Long non-coding RNA HOTAIR reprograms chromatin state to promote cancer metastasis</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>464</i></span>,  <span class="NLM_fpage">1071</span>â <span class="NLM_lpage">1076</span>, <span class="refDoi">Â DOI: 10.1038/nature08975</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1038%2Fnature08975" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=20393566" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC3cXkslaru7w%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=464&publication_year=2010&pages=1071-1076&author=R.+A.+Guptaauthor=N.+Shahauthor=K.+C.+Wangauthor=J.+Kimauthor=H.+M.+Horlingsauthor=D.+J.+Wongauthor=M.+C.+Tsaiauthor=T.+Hungauthor=P.+Arganiauthor=J.+L.+Rinnauthor=Y.+Wangauthor=P.+Brzoskaauthor=B.+Kongauthor=R.+Liauthor=R.+B.+Westauthor=M.+J.+van+de%0AVijverauthor=S.+Sukumarauthor=H.+Y.+Chang&title=Long+non-coding+RNA+HOTAIR+reprograms+chromatin+state+to+promote+cancer+metastasis&doi=10.1038%2Fnature08975"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit81R"><div class="casContent"><span class="casTitleNuber">81</span><div class="casTitle"><span class="NLM_cas:atitle">Long non-coding RNA HOTAIR reprograms chromatin state to promote cancer metastasis</span></div><div class="casAuthors">Gupta, Rajnish A.; Shah, Nilay; Wang, Kevin C.; Kim, Jeewon; Horlings, Hugo M.; Wong, David J.; Tsai, Miao-Chih; Hung, Tiffany; Argani, Pedram; Rinn, John L.; Wang, Yulei; Brzoska, Pius; Kong, Benjamin; Li, Rui; West, Robert B.; van de Vijver, Marc J.; Sukumar, Saraswati; Chang, Howard Y.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">464</span>
        (<span class="NLM_cas:issue">7291</span>),
    <span class="NLM_cas:pages">1071-1076</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Large intervening non-coding RNAs (lincRNAs) are pervasively transcribed in the genome yet their potential involvement in human disease is not well understood.  Recent studies of dosage compensation, imprinting, and homeotic gene expression suggest that individual lincRNAs can function as the interface between DNA and specific chromatin remodelling activities.  Here we show that lincRNAs in the HOX loci become systematically dysregulated during breast cancer progression.  The lincRNA termed HOTAIR is increased in expression in primary breast tumors and metastases, and HOTAIR expression level in primary tumors is a powerful predictor of eventual metastasis and death.  Enforced expression of HOTAIR in epithelial cancer cells induced genome-wide re-targeting of Polycomb repressive complex 2 (PRC2) to an occupancy pattern more resembling embryonic fibroblasts, leading to altered histone H3 lysine 27 methylation, gene expression, and increased cancer invasiveness and metastasis in a manner dependent on PRC2.  Conversely, loss of HOTAIR can inhibit cancer invasiveness, particularly in cells that possess excessive PRC2 activity.  These findings indicate that lincRNAs have active roles in modulating the cancer epigenome and may be important targets for cancer diagnosis and therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7vmSbfc1Wi7Vg90H21EOLACvtfcHk0lgoVO0lrcKAag"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXkslaru7w%253D&md5=26084a450b406d951f4d828e295f7f56</span></div><a href="/servlet/linkout?suffix=cit81&amp;dbid=16384&amp;doi=10.1038%2Fnature08975&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature08975%26sid%3Dliteratum%253Aachs%26aulast%3DGupta%26aufirst%3DR.%2BA.%26aulast%3DShah%26aufirst%3DN.%26aulast%3DWang%26aufirst%3DK.%2BC.%26aulast%3DKim%26aufirst%3DJ.%26aulast%3DHorlings%26aufirst%3DH.%2BM.%26aulast%3DWong%26aufirst%3DD.%2BJ.%26aulast%3DTsai%26aufirst%3DM.%2BC.%26aulast%3DHung%26aufirst%3DT.%26aulast%3DArgani%26aufirst%3DP.%26aulast%3DRinn%26aufirst%3DJ.%2BL.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DBrzoska%26aufirst%3DP.%26aulast%3DKong%26aufirst%3DB.%26aulast%3DLi%26aufirst%3DR.%26aulast%3DWest%26aufirst%3DR.%2BB.%26aulast%3Dvan%2Bde%2BVijver%26aufirst%3DM.%2BJ.%26aulast%3DSukumar%26aufirst%3DS.%26aulast%3DChang%26aufirst%3DH.%2BY.%26atitle%3DLong%2520non-coding%2520RNA%2520HOTAIR%2520reprograms%2520chromatin%2520state%2520to%2520promote%2520cancer%2520metastasis%26jtitle%3DNature%26date%3D2010%26volume%3D464%26spage%3D1071%26epage%3D1076%26doi%3D10.1038%2Fnature08975" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref82"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref82'); return false;" data-citation="" class="refNumLink">82</a></strong><div class="NLM_citation" id="cit82"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span> <span> </span><span class="NLM_article-title">Long non-coding RNA HOTAIR: A novel oncogene</span>. <i>Mol. Med. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">5611</span>â <span class="NLM_lpage">5618</span>, <span class="refDoi">Â DOI: 10.3892/mmr.2015.4161</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.3892%2Fmmr.2015.4161" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=26238267" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFWns7fO" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2015&pages=5611-5618&author=X.+Yuauthor=Z.+Li&title=Long+non-coding+RNA+HOTAIR%3A+A+novel+oncogene&doi=10.3892%2Fmmr.2015.4161"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit82R"><div class="casContent"><span class="casTitleNuber">82</span><div class="casTitle"><span class="NLM_cas:atitle">Long non-coding RNA HOTAIR: a novel oncogene (Review)</span></div><div class="casAuthors">Yu, Xin; Li, Zheng</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Medicine Reports</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">4, Pt. B</span>),
    <span class="NLM_cas:pages">5611-5618</span>CODEN:
                <span class="NLM_cas:coden">MMROA5</span>;
        ISSN:<span class="NLM_cas:issn">1791-2997</span>.
    
            (<span class="NLM_cas:orgname">Spandidos Publications Ltd.</span>)
        </div><div class="casAbstract">Long non-coding RNAs (lncRNAs) have been found to be pervasively transcribed in the genome and are crit. regulators of the epigenome.  Increasing evidence suggests that lncRNAs are aberrantly expressed in several types of human cancer and that they are important in the initiation, development and metastasis of human cancer.  Previous studies have revealed that HOX transcript antisense intergenic RNA (HOTAIR) was frequently upregulated in various types of cancer, including breast cancer, esophageal cancer, lung cancer and gastric cancer.  In addn., patients with high expression levels of HOTAIR have a significantly poorer prognosis, compared with those with low levels of expression.  HOTAIR is involved in the control of cell apoptosis, growth, metastasis, angiogenesis, DNA repair and tumor cells metab.  The present review provides an overview of the current knowledge concerning the role of HOTAIR in tumor development and progression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqKPrwBMvNoE7Vg90H21EOLACvtfcHk0lihplcsgf3vrg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFWns7fO&md5=39ca36f6d205ba645dac04d0e8b8e0a1</span></div><a href="/servlet/linkout?suffix=cit82&amp;dbid=16384&amp;doi=10.3892%2Fmmr.2015.4161&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3892%252Fmmr.2015.4161%26sid%3Dliteratum%253Aachs%26aulast%3DYu%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DZ.%26atitle%3DLong%2520non-coding%2520RNA%2520HOTAIR%253A%2520A%2520novel%2520oncogene%26jtitle%3DMol.%2520Med.%2520Rep.%26date%3D2015%26volume%3D12%26spage%3D5611%26epage%3D5618%26doi%3D10.3892%2Fmmr.2015.4161" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref83"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref83'); return false;" data-citation="" class="refNumLink">83</a></strong><div class="NLM_citation" id="cit83"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, C.</span></span> <span> </span><span class="NLM_article-title">Long non-coding RNA HOTAIR promotes glioblastoma cell cycle progression in an EZH2 dependent manner</span>. <i>Oncotarget.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">537</span>â <span class="NLM_lpage">546</span>, <span class="refDoi">Â DOI: 10.18632/oncotarget.2681</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.18632%2Foncotarget.2681" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=25428914" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A280%3ADC%252BC2MzhtFSnug%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=537-546&author=K.+Zhangauthor=X.+Sunauthor=X.+Zhouauthor=L.+Hanauthor=L.+Chenauthor=Z.+Shiauthor=A.+Zhangauthor=M.+Yeauthor=Q.+Wangauthor=C.+Liuauthor=J.+Weiauthor=Y.+Renauthor=J.+Yangauthor=J.+Zhangauthor=P.+Puauthor=M.+Liauthor=C.+Kang&title=Long+non-coding+RNA+HOTAIR+promotes+glioblastoma+cell+cycle+progression+in+an+EZH2+dependent+manner&doi=10.18632%2Foncotarget.2681"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit83R"><div class="casContent"><span class="casTitleNuber">83</span><div class="casTitle"><span class="NLM_cas:atitle">Long non-coding RNA HOTAIR promotes glioblastoma cell cycle progression in an EZH2 dependent manner</span></div><div class="casAuthors">Zhang Kailiang; Sun Xiaotian; Zhou Xuan; Han Lei; Chen Luyue; Shi Zhendong; Zhang Anling; Wang Qixue; Wei Jianwei; Zhang Jianning; Pu Peiyu; Kang Chunsheng; Ye Minhua; Liu Chaoyong; Ren Yu; Yang Jingxuan; Li Min</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">537-46</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The long non-coding RNA Hox transcript antisense intergenic RNA (HOTAIR) was recently implicated in breast cancer metastasis and is predictive of poor prognosis in colorectal and pancreatic cancers.  We recently discovered that HOTAIR is a cell cycle-related lncRNA in human glioma, and its expression is closely associated with glioma staging and poor prognosis.  Although lysine specific demethylase 1 (LSD1) and polycomb repressive complex 2 (PRC2) have been demonstrated to be functional targets of HOTAIR, how HOTAIR regulates glioma cell cycle progression remains largely unknown.  In this study, we found that EZH2 (predominant PRC2 complex component) inhibition blocked cell cycle progression in glioma cells, consistent with the effects elicited by HOTAIR siRNA.  However, the inhibition of LSD1 did not affect cell cycle progression in glioma cells.  These results suggest that HOTAIR might regulate cell cycle progression through EZH2.  Our intracranial mice model also revealed delayed tumor growth in HOTAIR siRNA- and EZH2 inhibitor-treated groups.  Moreover, in HOTAIR knock-down cell lines, the expression of the PRC2-binding domain of HOTAIR (5' domain) but not of the LSD1-binding domain of HOTAIR (3' domain) resulted in accelerated cell cycle progression.  In conclusion, HOTAIR promotes cell cycle progression in glioma as a result of the binding of its 5' domain to the PRC2 complex.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSe5I48OswLk6Nc3HjJRoFAfW6udTcc2ebqYnOA8EUR67ntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2MzhtFSnug%253D%253D&md5=d6c09a59a964632af9750a6fbf7b506b</span></div><a href="/servlet/linkout?suffix=cit83&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.2681&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.2681%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DK.%26aulast%3DSun%26aufirst%3DX.%26aulast%3DZhou%26aufirst%3DX.%26aulast%3DHan%26aufirst%3DL.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DShi%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DA.%26aulast%3DYe%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DQ.%26aulast%3DLiu%26aufirst%3DC.%26aulast%3DWei%26aufirst%3DJ.%26aulast%3DRen%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DPu%26aufirst%3DP.%26aulast%3DLi%26aufirst%3DM.%26aulast%3DKang%26aufirst%3DC.%26atitle%3DLong%2520non-coding%2520RNA%2520HOTAIR%2520promotes%2520glioblastoma%2520cell%2520cycle%2520progression%2520in%2520an%2520EZH2%2520dependent%2520manner%26jtitle%3DOncotarget.%26date%3D2015%26volume%3D6%26spage%3D537%26epage%3D546%26doi%3D10.18632%2Foncotarget.2681" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref84"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref84'); return false;" data-citation="" class="refNumLink">84</a></strong><div class="NLM_citation" id="cit84"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ren, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mei, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, C.</span></span> <span> </span><span class="NLM_article-title">Targeted design and identification of AC1NOD4Q to block activity of HOTAIR by abrogating the scaffold interaction with EZH2</span>. <i>Clin. Epigenet.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">29</span>, <span class="refDoi">Â DOI: 10.1186/s13148-019-0624-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1186%2Fs13148-019-0624-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=30764859" title="">PubMed</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2019&pages=29&author=Y.+Renauthor=Y.+Wangauthor=J.+Zhangauthor=Q.+Wangauthor=L.+Hanauthor=M.+Meiauthor=C.+Kang&title=Targeted+design+and+identification+of+AC1NOD4Q+to+block+activity+of+HOTAIR+by+abrogating+the+scaffold+interaction+with+EZH2&doi=10.1186%2Fs13148-019-0624-2"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit84&amp;dbid=16384&amp;doi=10.1186%2Fs13148-019-0624-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs13148-019-0624-2%26sid%3Dliteratum%253Aachs%26aulast%3DRen%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DQ.%26aulast%3DHan%26aufirst%3DL.%26aulast%3DMei%26aufirst%3DM.%26aulast%3DKang%26aufirst%3DC.%26atitle%3DTargeted%2520design%2520and%2520identification%2520of%2520AC1NOD4Q%2520to%2520block%2520activity%2520of%2520HOTAIR%2520by%2520abrogating%2520the%2520scaffold%2520interaction%2520with%2520EZH2%26jtitle%3DClin.%2520Epigenet.%26date%3D2019%26volume%3D11%26spage%3D29%26doi%3D10.1186%2Fs13148-019-0624-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref85"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref85'); return false;" data-citation="" class="refNumLink">85</a></strong><div class="NLM_citation" id="cit85"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Malina, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mills, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pelletier, J.</span></span> <span> </span><span class="NLM_article-title">Emerging therapeutics targeting mRNA translation</span>. <i>Cold Spring Harbor Perspect. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>4</i></span>, <span class="NLM_elocation-id">a012377</span> <span class="refDoi">Â DOI: 10.1101/cshperspect.a012377</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1101%2Fcshperspect.a012377" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=22474009" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC38XpvFyrs74%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2012&author=A.+Malinaauthor=J.+R.+Millsauthor=J.+Pelletier&title=Emerging+therapeutics+targeting+mRNA+translation&doi=10.1101%2Fcshperspect.a012377"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit85R"><div class="casContent"><span class="casTitleNuber">85</span><div class="casTitle"><span class="NLM_cas:atitle">Emerging therapeutics targeting mRNA translation</span></div><div class="casAuthors">Malina, Abba; Mills, John R.; Pelletier, Jerry</div><div class="citationInfo"><span class="NLM_cas:title">Cold Spring Harbor Perspectives in Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">a012377, 18 pp.</span>CODEN:
                <span class="NLM_cas:coden">CSHPEU</span>;
        ISSN:<span class="NLM_cas:issn">1943-0264</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">A review.  A defining feature of many cancers is deregulated translational control.  Typically, this occurs at the level of recruitment of the 40S ribosomes to the 5'-cap of cellular mRNAs (mRNAs), the rate-limiting step of protein synthesis, which is controlled by the heterotrimeric eukaryotic initiation complex eIF4F.  Thus, eIF4F in particular, and translation initiation in general, represent an exploitable vulnerability and unique opportunity for therapeutic intervention in many transformed cells.  In this article, we discuss the development, mode of action and biol. activity of a no. of small-mol. inhibitors that interrupt PI3K/mTOR signaling control of eIF4F assembly, as well as compds. that more directly block eIF4F activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqH7fO5RA3nqLVg90H21EOLACvtfcHk0lhdZrQmk-6QYw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XpvFyrs74%253D&md5=b1619198f480d1fb1c8c04e296633e2a</span></div><a href="/servlet/linkout?suffix=cit85&amp;dbid=16384&amp;doi=10.1101%2Fcshperspect.a012377&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fcshperspect.a012377%26sid%3Dliteratum%253Aachs%26aulast%3DMalina%26aufirst%3DA.%26aulast%3DMills%26aufirst%3DJ.%2BR.%26aulast%3DPelletier%26aufirst%3DJ.%26atitle%3DEmerging%2520therapeutics%2520targeting%2520mRNA%2520translation%26jtitle%3DCold%2520Spring%2520Harbor%2520Perspect.%2520Biol.%26date%3D2012%26volume%3D4%26doi%3D10.1101%2Fcshperspect.a012377" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref86"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref86'); return false;" data-citation="" class="refNumLink">86</a></strong><div class="NLM_citation" id="cit86"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Agarwala, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pandey, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maiti, S.</span></span> <span> </span><span class="NLM_article-title">The tale of RNA G-quadruplex</span>. <i>Org. Biomol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">5570</span>â <span class="NLM_lpage">5585</span>, <span class="refDoi">Â DOI: 10.1039/C4OB02681K</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1039%2FC4OB02681K" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=25879384" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC2MXlslegsb0%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2015&pages=5570-5585&author=P.+Agarwalaauthor=S.+Pandeyauthor=S.+Maiti&title=The+tale+of+RNA+G-quadruplex&doi=10.1039%2FC4OB02681K"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit86R"><div class="casContent"><span class="casTitleNuber">86</span><div class="casTitle"><span class="NLM_cas:atitle">The tale of RNA G-quadruplex</span></div><div class="casAuthors">Agarwala, Prachi; Pandey, Satyaprakash; Maiti, Souvik</div><div class="citationInfo"><span class="NLM_cas:title">Organic & Biomolecular Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">5570-5585</span>CODEN:
                <span class="NLM_cas:coden">OBCRAK</span>;
        ISSN:<span class="NLM_cas:issn">1477-0520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  G-quadruplexes are non-canonical secondary structures found in guanine rich regions of DNA and RNA.  Reports have indicated the wide occurrence of RNA G-quadruplexes across the transcriptome in various regions of mRNAs and non-coding RNAs.  RNA G-quadruplexes have been implicated in playing an important role in translational regulation, mRNA processing events and maintenance of chromosomal end integrity.  In this review, we summarize the structural and functional aspects of RNA G-quadruplexes with emphasis on recent progress to understand the protein/trans factors binding these motifs.  With the revelation of the importance of these secondary structures as regulatory modules in biol., we have also evaluated the various advancements towards targeting these structures and the challenges assocd. with them.  Apart from this, numerous potential applications of this secondary motif have also been discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq7MAm0c-9cbrVg90H21EOLACvtfcHk0lhdZrQmk-6QYw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXlslegsb0%253D&md5=e51f502c003a43d1d9b6ab4506f6d8a5</span></div><a href="/servlet/linkout?suffix=cit86&amp;dbid=16384&amp;doi=10.1039%2FC4OB02681K&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC4OB02681K%26sid%3Dliteratum%253Aachs%26aulast%3DAgarwala%26aufirst%3DP.%26aulast%3DPandey%26aufirst%3DS.%26aulast%3DMaiti%26aufirst%3DS.%26atitle%3DThe%2520tale%2520of%2520RNA%2520G-quadruplex%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2015%26volume%3D13%26spage%3D5570%26epage%3D5585%26doi%3D10.1039%2FC4OB02681K" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref87"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref87'); return false;" data-citation="" class="refNumLink">87</a></strong><div class="NLM_citation" id="cit87"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Katsuda, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sato, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asano, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morimura, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furuta, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sugiyama, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hagihara, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uesugi, M.</span></span> <span> </span><span class="NLM_article-title">A small molecule that represses translation of G-quadruplex-containing mRNA</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>138</i></span>,  <span class="NLM_fpage">9037</span>â <span class="NLM_lpage">9040</span>, <span class="refDoi">Â DOI: 10.1021/jacs.6b04506</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.6b04506" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFKms7%252FP" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=138&publication_year=2016&pages=9037-9040&author=Y.+Katsudaauthor=S.+Satoauthor=L.+Asanoauthor=Y.+Morimuraauthor=T.+Furutaauthor=H.+Sugiyamaauthor=M.+Hagiharaauthor=M.+Uesugi&title=A+small+molecule+that+represses+translation+of+G-quadruplex-containing+mRNA&doi=10.1021%2Fjacs.6b04506"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit87R"><div class="casContent"><span class="casTitleNuber">87</span><div class="casTitle"><span class="NLM_cas:atitle">A small molecule that represses translation of G-quadruplex-containing mRNA</span></div><div class="casAuthors">Katsuda, Yousuke; Sato, Shin-ichi; Asano, Lisa; Morimura, Yoshitaka; Furuta, Tomoyuki; Sugiyama, Hiroshi; Hagihara, Masaki; Uesugi, Motonari</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">138</span>
        (<span class="NLM_cas:issue">29</span>),
    <span class="NLM_cas:pages">9037-9040</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The G-quadruplexes form highly stable nucleic acid structures, which are implicated in various biol. processes in both DNA and RNA.  Although DNA G-quadruplexes have been studied in great detail, biol. roles of RNA G-quadruplexes have received less attention.  Here, a screening of a chem. library permitted identification of a small-mol. tool that binds selectively to RNA G-quadruplex structures.  The polyarom. mol., RGB-1, stabilizes RNA G-quadruplex, but not DNA versions or other RNA structures.  RGB-1 intensified the G-quadruplex-mediated inhibition of RNA translation in mammalian cells, decreased expression of the NRAS proto-oncogene in breast cancer cells, and permitted identification of a novel sequence that forms G-quadruplex in NRAS mRNA.  RGB-1 may serve as a unique tool for understanding cellular roles of RNA G-quadruplex structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5-OEuiG5IuLVg90H21EOLACvtfcHk0lhdZrQmk-6QYw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtFKms7%252FP&md5=75ad58de109aa50cc964db46bc46f69b</span></div><a href="/servlet/linkout?suffix=cit87&amp;dbid=16384&amp;doi=10.1021%2Fjacs.6b04506&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.6b04506%26sid%3Dliteratum%253Aachs%26aulast%3DKatsuda%26aufirst%3DY.%26aulast%3DSato%26aufirst%3DS.%26aulast%3DAsano%26aufirst%3DL.%26aulast%3DMorimura%26aufirst%3DY.%26aulast%3DFuruta%26aufirst%3DT.%26aulast%3DSugiyama%26aufirst%3DH.%26aulast%3DHagihara%26aufirst%3DM.%26aulast%3DUesugi%26aufirst%3DM.%26atitle%3DA%2520small%2520molecule%2520that%2520represses%2520translation%2520of%2520G-quadruplex-containing%2520mRNA%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2016%26volume%3D138%26spage%3D9037%26epage%3D9040%26doi%3D10.1021%2Fjacs.6b04506" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref88"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref88'); return false;" data-citation="" class="refNumLink">88</a></strong><div class="NLM_citation" id="cit88"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Plant, E. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dinman, J. D.</span></span> <span> </span><span class="NLM_article-title">The role of programmed-1 ribosomal frameshifting in coronavirus propagation</span>. <i>Front. Biosci., Landmark Ed.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">4873</span>â <span class="NLM_lpage">4881</span>, <span class="refDoi">Â DOI: 10.2741/3046</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.2741%2F3046" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=18508552" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BD1cXnsVOjtrw%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2008&pages=4873-4881&author=E.+P.+Plantauthor=J.+D.+Dinman&title=The+role+of+programmed-1+ribosomal+frameshifting+in+coronavirus+propagation&doi=10.2741%2F3046"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit88R"><div class="casContent"><span class="casTitleNuber">88</span><div class="casTitle"><span class="NLM_cas:atitle">The role of programmed-1 ribosomal frameshifting in coronavirus propagation</span></div><div class="casAuthors">Plant, Ewan P.; Dinman, Jonathan D.</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in Bioscience</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">4873-4881</span>CODEN:
                <span class="NLM_cas:coden">FRBIF6</span>;
        ISSN:<span class="NLM_cas:issn">1093-4715</span>.
    
            (<span class="NLM_cas:orgname">Frontiers in Bioscience</span>)
        </div><div class="casAbstract">A review.  Coronaviruses have the potential to cause significant economic, agricultural and health problems.  The severe acute respiratory syndrome (SARS) assocd. coronavirus outbreak in late 2002, early 2003 called attention to the potential damage that coronaviruses could cause in the human population.  The ensuing research has enlightened many to the mol. biol. of coronaviruses.  A programmed -1 ribosomal frameshift is required by coronaviruses for the prodn. of the RNA dependent RNA polymerase which in turn is essential for viral replication.  The frameshifting signal encoded in the viral genome has addnl. features that are not essential for frameshifting.  Elucidation of the differences between coronavirus frameshift signals and signals from other viruses may help our understanding of these features.  Here we summarize current knowledge and add addnl. insight regarding the function of the programmed -1 ribosomal frameshift signal in the coronavirus lifecycle.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZWj0x7RQKvLVg90H21EOLACvtfcHk0lhq6fbcOuWoig"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXnsVOjtrw%253D&md5=8bcadd716bed6f996bce39d1da1a0fac</span></div><a href="/servlet/linkout?suffix=cit88&amp;dbid=16384&amp;doi=10.2741%2F3046&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2741%252F3046%26sid%3Dliteratum%253Aachs%26aulast%3DPlant%26aufirst%3DE.%2BP.%26aulast%3DDinman%26aufirst%3DJ.%2BD.%26atitle%3DThe%2520role%2520of%2520programmed-1%2520ribosomal%2520frameshifting%2520in%2520coronavirus%2520propagation%26jtitle%3DFront.%2520Biosci.%252C%2520Landmark%2520Ed.%26date%3D2008%26volume%3D13%26spage%3D4873%26epage%3D4881%26doi%3D10.2741%2F3046" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref89"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref89'); return false;" data-citation="" class="refNumLink">89</a></strong><div class="NLM_citation" id="cit89"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Peselis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Serganov, A.</span></span> <span> </span><span class="NLM_article-title">Structure and function of pseudoknots involved in gene expression control</span>. <i>Wiley Interdiscip. Rev. RNA</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">803</span>â <span class="NLM_lpage">822</span>, <span class="refDoi">Â DOI: 10.1002/wrna.1247</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1002%2Fwrna.1247" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=25044223" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVSjsLvE" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=803-822&author=A.+Peselisauthor=A.+Serganov&title=Structure+and+function+of+pseudoknots+involved+in+gene+expression+control&doi=10.1002%2Fwrna.1247"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit89R"><div class="casContent"><span class="casTitleNuber">89</span><div class="casTitle"><span class="NLM_cas:atitle">Structure and function of pseudoknots involved in gene expression control</span></div><div class="casAuthors">Peselis, Alla; Serganov, Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Wiley Interdisciplinary Reviews: RNA</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">803-822</span>CODEN:
                <span class="NLM_cas:coden">WIRRCU</span>;
        ISSN:<span class="NLM_cas:issn">1757-7012</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  Natural RNA mols. can have a high degree of structural complexity but even the most complexly folded RNAs are assembled from simple structural building blocks.  Among the simplest RNA elements are double-stranded helixes that participate in the formation of different folding topologies and constitute the major fraction of RNA structures.  One common folding motif of RNA is a pseudoknot, defined as a bipartite helical structure formed by base-pairing of the apical loop in the stem-loop structure with an outside sequence.  Pseudoknots constitute integral parts of the RNA structures essential for various cellular activities.  Among many functions of pseudoknotted RNAs is feedback regulation of gene expression, carried out through specific recognition of various mols.  Pseudoknotted RNAs autoregulate ribosomal and phage protein genes in response to downstream encoded proteins, while many metabolic and transport genes are controlled by cellular metabolites interacting with pseudoknotted RNA elements from the riboswitch family.  Modulation of some genes also depends on metabolite-induced mRNA cleavage performed by pseudoknotted ribozymes.  Several regulatory pseudoknots have been characterized biochem. and structurally in great detail.  These studies have demonstrated a plethora of pseudoknot-based folds and have begun uncovering diverse mol. principles of the ligand-dependent gene expression control.  The pseudoknot-mediated mechanisms of gene control and many unexpected and interesting features of the regulatory pseudoknots have significantly advanced our understanding of the genetic circuits and laid the foundation for modulation of their outcomes.  Â© 2014 John Wiley & Sons, Ltd.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrkScYBDta3RLVg90H21EOLACvtfcHk0lhq6fbcOuWoig"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVSjsLvE&md5=d936f91eac6a6844bf2fc04b0114649a</span></div><a href="/servlet/linkout?suffix=cit89&amp;dbid=16384&amp;doi=10.1002%2Fwrna.1247&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fwrna.1247%26sid%3Dliteratum%253Aachs%26aulast%3DPeselis%26aufirst%3DA.%26aulast%3DSerganov%26aufirst%3DA.%26atitle%3DStructure%2520and%2520function%2520of%2520pseudoknots%2520involved%2520in%2520gene%2520expression%2520control%26jtitle%3DWiley%2520Interdiscip.%2520Rev.%2520RNA%26date%3D2014%26volume%3D5%26spage%3D803%26epage%3D822%26doi%3D10.1002%2Fwrna.1247" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref90"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref90'); return false;" data-citation="" class="refNumLink">90</a></strong><div class="NLM_citation" id="cit90"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Matsumoto, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caliskan, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodnina, M. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murata, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakatani, K.</span></span> <span> </span><span class="NLM_article-title">Small synthetic molecule-stabilized RNA pseudoknot as an activator for â1 ribosomal frameshifting</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">8079</span>â <span class="NLM_lpage">8089</span>, <span class="refDoi">Â DOI: 10.1093/nar/gky689</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref90/cit90&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1093%2Fnar%2Fgky689" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref90/cit90&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=30085309" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref90/cit90&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC1MXosFanu7s%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2018&pages=8079-8089&author=S.+Matsumotoauthor=N.+Caliskanauthor=M.+V.+Rodninaauthor=A.+Murataauthor=K.+Nakatani&title=Small+synthetic+molecule-stabilized+RNA+pseudoknot+as+an+activator+for+%E2%88%921+ribosomal+frameshifting&doi=10.1093%2Fnar%2Fgky689"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit90R"><div class="casContent"><span class="casTitleNuber">90</span><div class="casTitle"><span class="NLM_cas:atitle">Small synthetic molecule-stabilized RNA pseudoknot as an activator for -1 ribosomal frameshifting</span></div><div class="casAuthors">Matsumoto, Saki; Caliskan, Neva; Rodnina, Marina V.; Murata, Asako; Nakatani, Kazuhiko</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">8079-8089</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">1362-4962</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Programmed -1 ribosomal frameshifting (-1PRF) is a recoding mechanism to make alternative proteins from a single mRNA transcript.  -1PRF is stimulated by cis-acting signals in mRNA, a seven-nucleotide slippery sequence and a downstream secondary structure element, which is often a pseudoknot.  In this study we engineered the frameshifting pseudoknot from the mouse mammary tumor virus to respond to a rationally designed small mol. naphthyridine carbamate tetramer (NCTn).  We demonstrate that NCTn can stabilize the pseudoknot structure in mRNA and activate -1PRF both in vitro and in human cells.  The results illustrate how NCTn-inducible -1PRF may serve as an important component of the synthetic biol. toolbox for the precise control of gene expression using small synthetic mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_y6f3PayO7LVg90H21EOLACvtfcHk0lhq6fbcOuWoig"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXosFanu7s%253D&md5=b9d5ba7cf6605fa1b6c0d46bf12b7847</span></div><a href="/servlet/linkout?suffix=cit90&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgky689&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgky689%26sid%3Dliteratum%253Aachs%26aulast%3DMatsumoto%26aufirst%3DS.%26aulast%3DCaliskan%26aufirst%3DN.%26aulast%3DRodnina%26aufirst%3DM.%2BV.%26aulast%3DMurata%26aufirst%3DA.%26aulast%3DNakatani%26aufirst%3DK.%26atitle%3DSmall%2520synthetic%2520molecule-stabilized%2520RNA%2520pseudoknot%2520as%2520an%2520activator%2520for%2520%25E2%2588%25921%2520ribosomal%2520frameshifting%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2018%26volume%3D46%26spage%3D8079%26epage%3D8089%26doi%3D10.1093%2Fnar%2Fgky689" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref91"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref91'); return false;" data-citation="" class="refNumLink">91</a></strong><div class="NLM_citation" id="cit91"><span><span class="NLM_contrib-group"><span class="NLM_string-name">BurrÃ©, J.</span></span> <span> </span><span class="NLM_article-title">The synaptic function of Î±-synuclein</span>. <i>J. Parkinson's Dis.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">699</span>â <span class="NLM_lpage">713</span>, <span class="refDoi">Â DOI: 10.3233/JPD-150642</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.3233%2FJPD-150642" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=26407041" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitVSqs7fM" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2015&pages=699-713&author=J.+Burr%C3%A9&title=The+synaptic+function+of+%CE%B1-synuclein&doi=10.3233%2FJPD-150642"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit91R"><div class="casContent"><span class="casTitleNuber">91</span><div class="casTitle"><span class="NLM_cas:atitle">The Synaptic Function of Î±-Synuclein</span></div><div class="casAuthors">Burre, Jacqueline</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Parkinson's Disease</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">699-713</span>CODEN:
                <span class="NLM_cas:coden">JPDOAP</span>;
        ISSN:<span class="NLM_cas:issn">1877-7171</span>.
    
            (<span class="NLM_cas:orgname">IOS Press</span>)
        </div><div class="casAbstract">Î±-Synuclein is an abundant neuronal protein which localizes predominantly to presynaptic terminals, and is strongly linked genetically and pathol. to Parkinson's disease and other neurodegenerative diseases.  While the accumulation of Î±-synuclein in the form of misfolded oligomers and large aggregates defines multiple neurodegenerative diseases called "synucleinopathies", its cellular function has remained largely unclear, and is the subject of intense investigation.  In this review, I focus on the structural characteristics of Î±-synuclein, its cellular and subcellular localization, and discuss how this relates to its function in neurons, in particular at the neuronal synapse.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqdLxDreICkVbVg90H21EOLACvtfcHk0ljiqQS4UtE-6w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitVSqs7fM&md5=6199a08d8d9138b554aa25fd45279b48</span></div><a href="/servlet/linkout?suffix=cit91&amp;dbid=16384&amp;doi=10.3233%2FJPD-150642&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3233%252FJPD-150642%26sid%3Dliteratum%253Aachs%26aulast%3DBurr%25C3%25A9%26aufirst%3DJ.%26atitle%3DThe%2520synaptic%2520function%2520of%2520%25CE%25B1-synuclein%26jtitle%3DJ.%2520Parkinson%2527s%2520Dis.%26date%3D2015%26volume%3D5%26spage%3D699%26epage%3D713%26doi%3D10.3233%2FJPD-150642" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref92"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref92'); return false;" data-citation="" class="refNumLink">92</a></strong><div class="NLM_citation" id="cit92"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, H.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Junn, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andrews, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Velagapudi, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abegg, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vishnu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Costales, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Childs-Disney, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adibekian, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moss, W. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mouradian, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span> <span> </span><span class="NLM_article-title">Translation of the intrinsically disordered protein Î±-synuclein is inhibited by a small molecule targeting its structured mRNA</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>117</i></span>,  <span class="NLM_fpage">1457</span>â <span class="NLM_lpage">1467</span>, <span class="refDoi">Â DOI: 10.1073/pnas.1905057117</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1073%2Fpnas.1905057117" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=31900363" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhsFKktbc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=117&publication_year=2020&pages=1457-1467&author=P.+Zhangauthor=H.-J.+Parkauthor=J.+Zhangauthor=E.+Junnauthor=R.+J.+Andrewsauthor=S.+P.+Velagapudiauthor=D.+Abeggauthor=K.+Vishnuauthor=M.+G.+Costalesauthor=J.+L.+Childs-Disneyauthor=A.+Adibekianauthor=W.+N.+Mossauthor=M.+M.+Mouradianauthor=M.+D.+Disney&title=Translation+of+the+intrinsically+disordered+protein+%CE%B1-synuclein+is+inhibited+by+a+small+molecule+targeting+its+structured+mRNA&doi=10.1073%2Fpnas.1905057117"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit92R"><div class="casContent"><span class="casTitleNuber">92</span><div class="casTitle"><span class="NLM_cas:atitle">Translation of the intrinsically disordered protein Î±-synuclein is inhibited by a small molecule targeting its structured mRNA</span></div><div class="casAuthors">Zhang, Peiyuan; Park, Hye-Jin; Zhang, Jie; Junn, Eunsung; Andrews, Ryan J.; Velagapudi, Sai Pradeep; Abegg, Daniel; Vishnu, Kamalakannan; Costales, Matthew G.; Childs-Disney, Jessica L.; Adibekian, Alexander; Moss, Walter N.; Mouradian, M. Maral; Disney, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">117</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1457-1467</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Many proteins are refractory to targeting because they lack small-mol. binding pockets.  An alternative to drugging these proteins directly is to target the messenger (m)RNA that encodes them, thereby reducing protein levels.  We describe such an approach for the difficult-to-target protein Î±-synuclein encoded by the SNCA gene.  Multiplication of the SNCA gene locus causes dominantly inherited Parkinson's disease (PD), and Î±-synuclein protein aggregates in Lewy bodies and Lewy neurites in sporadic PD.  Thus, reducing the expression of Î±-synuclein protein is expected to have therapeutic value.  Fortuitously, the SNCA mRNA has a structured iron-responsive element (IRE) in its 5' untranslated region (5' UTR) that controls its translation.  Using sequence-based design, we discovered small mols. that target the IRE structure and inhibit SNCA translation in cells, the most potent of which is named Synucleozid.  Both in vitro and cellular profiling studies showed Synucleozid directly targets the Î±-synuclein mRNA 5' UTR at the designed site.Î±.  Mechanistic studies revealed that Synucleozid reduces Î±-synuclein protein levels by decreasing the amt. of SNCA mRNA loaded into polysomes, mechanistically providing a cytoprotective effect in cells.  Proteome- and transcriptome-wide studies showed that the compd.'s selectivity makes Synucleozid suitable for further development.  Importantly, transcriptome-wide anal. of mRNAs that encode intrinsically disordered proteins revealed that each has structured regions that could be targeted with small mols.  These findings demonstrate the potential for targeting undruggable proteins at the level of their coding mRNAs.  This approach, as applied to SNCA, is a promising disease-modifying therapeutic strategy for PD and other Î±-synucleinopathies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvPjL_9G6xS7Vg90H21EOLACvtfcHk0ljiqQS4UtE-6w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhsFKktbc%253D&md5=4f2a60d2c0fe09ea99f05291621ad4ef</span></div><a href="/servlet/linkout?suffix=cit92&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1905057117&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1905057117%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DP.%26aulast%3DPark%26aufirst%3DH.-J.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DJunn%26aufirst%3DE.%26aulast%3DAndrews%26aufirst%3DR.%2BJ.%26aulast%3DVelagapudi%26aufirst%3DS.%2BP.%26aulast%3DAbegg%26aufirst%3DD.%26aulast%3DVishnu%26aufirst%3DK.%26aulast%3DCostales%26aufirst%3DM.%2BG.%26aulast%3DChilds-Disney%26aufirst%3DJ.%2BL.%26aulast%3DAdibekian%26aufirst%3DA.%26aulast%3DMoss%26aufirst%3DW.%2BN.%26aulast%3DMouradian%26aufirst%3DM.%2BM.%26aulast%3DDisney%26aufirst%3DM.%2BD.%26atitle%3DTranslation%2520of%2520the%2520intrinsically%2520disordered%2520protein%2520%25CE%25B1-synuclein%2520is%2520inhibited%2520by%2520a%2520small%2520molecule%2520targeting%2520its%2520structured%2520mRNA%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2020%26volume%3D117%26spage%3D1457%26epage%3D1467%26doi%3D10.1073%2Fpnas.1905057117" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref93"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref93'); return false;" data-citation="" class="refNumLink">93</a></strong><div class="NLM_citation" id="cit93"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cleveland, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hwo, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirschner, M. W.</span></span> <span> </span><span class="NLM_article-title">Purification of tau, a microtubule associated protein that induces assembly of microtubules from purified tubulin</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1977</span>,  <span class="NLM_volume"><i>116</i></span>,  <span class="NLM_fpage">207</span>â <span class="NLM_lpage">225</span>, <span class="refDoi">Â DOI: 10.1016/0022-2836(77)90213-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1016%2F0022-2836%2877%2990213-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=599557" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADyaE1cXks1Kisw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=1977&pages=207-225&author=D.+W.+Clevelandauthor=S.+Y.+Hwoauthor=M.+W.+Kirschner&title=Purification+of+tau%2C+a+microtubule+associated+protein+that+induces+assembly+of+microtubules+from+purified+tubulin&doi=10.1016%2F0022-2836%2877%2990213-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit93R"><div class="casContent"><span class="casTitleNuber">93</span><div class="casTitle"><span class="NLM_cas:atitle">Purification of tau, a microtubule-associated protein that induces assembly of microtubules from purified tubulin</span></div><div class="casAuthors">Cleveland, Don W.; Hwo, Shu-Ying; Kirschner, Marc W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1977</span>),
    <span class="NLM_cas:volume">116</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">207-25</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    </div><div class="casAbstract">Tau, a microtubule-assocd. protein which copurifies with tubulin through successive cycles of polymn. and depolymn., was isolated from tubulin by phosphocellulose chromatog. and purified to near homogeneity.  The purified protein migrated during electrophoresis on acrylamide gels as 4 closely spaced bands of apparent mol. wts. between 55,000 and 62,000.  Specific activity for induction of microtubule formation from purified tubulin was assayed by quant. electron microscopy and was enhanced 3-4-fold in the purified tau when compared with the unfractionated microtubule-assocd. proteins.  Nearly 90% of available tubulin at 1 mg/mL was polymerizable into microtubules with elevated levels of tau.  Moreover, the crit. concn. for polymn. of the reconstituted tau + tubulin system was a function of tau concn. and could be lowered to as little as 30 Î¼g of tubulin/mL.  Under depolymg. conditions, 50% of the tubulin at only 1 mg/mL could be driven into ring structures.  A sep. purifn. procedure for isolation of tau directly from cell exts. was developed and data from this purifn. suggest that tau is present in the ext. in roughly the same proportion to tubulin as is found in microtubules purified by cycles of assembly and disassembly.  Tau was sufficient for both nucleation and elongation of microtubules from purified tubulin and, hence, the reconstituted tau + tubulin system defined a complete microtubule assembly system under std. buffer conditions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqYiOJMmcY9-7Vg90H21EOLACvtfcHk0lj2FClSe3-_Ag"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE1cXks1Kisw%253D%253D&md5=61ff67f90dc42e4c19e13ab536d5de15</span></div><a href="/servlet/linkout?suffix=cit93&amp;dbid=16384&amp;doi=10.1016%2F0022-2836%2877%2990213-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0022-2836%252877%252990213-3%26sid%3Dliteratum%253Aachs%26aulast%3DCleveland%26aufirst%3DD.%2BW.%26aulast%3DHwo%26aufirst%3DS.%2BY.%26aulast%3DKirschner%26aufirst%3DM.%2BW.%26atitle%3DPurification%2520of%2520tau%252C%2520a%2520microtubule%2520associated%2520protein%2520that%2520induces%2520assembly%2520of%2520microtubules%2520from%2520purified%2520tubulin%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1977%26volume%3D116%26spage%3D207%26epage%3D225%26doi%3D10.1016%2F0022-2836%2877%2990213-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref94"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref94'); return false;" data-citation="" class="refNumLink">94</a></strong><div class="NLM_citation" id="cit94"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ross, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poirier, M. A.</span></span> <span> </span><span class="NLM_article-title">Protein aggregation and neurodegenerative disease</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">S10</span>â <span class="NLM_lpage">S17</span>, <span class="refDoi">Â DOI: 10.1038/nm1066</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1038%2Fnm1066" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=15272267" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A280%3ADC%252BD2czptF2rtg%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2004&pages=S10-S17&author=C.+A.+Rossauthor=M.+A.+Poirier&title=Protein+aggregation+and+neurodegenerative+disease&doi=10.1038%2Fnm1066"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit94R"><div class="casContent"><span class="casTitleNuber">94</span><div class="casTitle"><span class="NLM_cas:atitle">Protein aggregation and neurodegenerative disease</span></div><div class="casAuthors">Ross Christopher A; Poirier Michelle A</div><div class="citationInfo"><span class="NLM_cas:title">Nature medicine</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">10 Suppl</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">S10-7</span>
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    </div><div class="casAbstract">Neurodegenerative diseases such as Alzheimer's disease (AD), Parkinson's disease (PD), Huntington's disease (HD), amyotrophic lateral sclerosis (ALS) and prion diseases are increasingly being realized to have common cellular and molecular mechanisms including protein aggregation and inclusion body formation.  The aggregates usually consist of fibers containing misfolded protein with a beta-sheet conformation, termed amyloid.  There is partial but not perfect overlap among the cells in which abnormal proteins are deposited and the cells that degenerate.  The most likely explanation is that inclusions and other visible protein aggregates represent an end stage of a molecular cascade of several steps, and that earlier steps in the cascade may be more directly tied to pathogenesis than the inclusions themselves.  For several diseases, genetic variants assist in explaining the pathogenesis of the more common sporadic forms and developing mouse and other models.  There is now increased understanding of the pathways involved in protein aggregation, and some recent clues have emerged as to the molecular mechanisms of cellular toxicity.  These are leading to approaches toward rational therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR51TEFJZTv4mx8TEN9K18QfW6udTcc2ebT2U7dS-RcmLntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2czptF2rtg%253D%253D&md5=b83d9efabba21b8dd2265f3547cdb0ad</span></div><a href="/servlet/linkout?suffix=cit94&amp;dbid=16384&amp;doi=10.1038%2Fnm1066&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm1066%26sid%3Dliteratum%253Aachs%26aulast%3DRoss%26aufirst%3DC.%2BA.%26aulast%3DPoirier%26aufirst%3DM.%2BA.%26atitle%3DProtein%2520aggregation%2520and%2520neurodegenerative%2520disease%26jtitle%3DNat.%2520Med.%26date%3D2004%26volume%3D10%26spage%3DS10%26epage%3DS17%26doi%3D10.1038%2Fnm1066" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref95"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref95'); return false;" data-citation="" class="refNumLink">95</a></strong><div class="NLM_citation" id="cit95"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wolfe, M. S.</span></span> <span> </span><span class="NLM_article-title">Targeting mRNA for Alzheimerâs and related dementias</span>. <i>Scientifica</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>2014</i></span>,  <span class="NLM_fpage">757549</span>, <span class="refDoi">Â DOI: 10.1155/2014/757549</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref95/cit95&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1155%2F2014%2F757549" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref95/cit95&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=24876993" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref95/cit95&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A280%3ADC%252BC2cflvFCqsw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2014&publication_year=2014&pages=757549&author=M.+S.+Wolfe&title=Targeting+mRNA+for+Alzheimer%E2%80%99s+and+related+dementias&doi=10.1155%2F2014%2F757549"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit95R"><div class="casContent"><span class="casTitleNuber">95</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting mRNA for Alzheimer's and related dementias</span></div><div class="casAuthors">Wolfe Michael S</div><div class="citationInfo"><span class="NLM_cas:title">Scientifica</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">2014</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">757549</span>
        ISSN:<span class="NLM_cas:issn">2090-908X</span>.
    </div><div class="casAbstract">Brain deposition of the amyloid beta-protein (A Î² ) and tau are characteristic features in Alzheimer's disease (AD).  Mutations in the A Î² precursor protein (APP) and a protease involved in A Î² production from APP strongly argue for a pathogenic role of A Î² in AD, while mutations in tau are associated with related disorders collectively called frontotemporal lobar degeneration (FTLD).  Despite intense effort, therapeutic strategies that target A Î² or tau have not yet yielded medications, suggesting that alternative approaches should be pursued.  In recent years, our laboratory has studied the role of mRNA in AD and FTLD, specifically those encoding tau and the A Î² -producing protease BACE1.  As many FTLD-causing tau mutations destabilize a hairpin structure that regulates RNA splicing, we have targeted this structure with small molecules, antisense oligonucleotides, and small molecule-antisense conjugates.  We have also discovered that microRNA interaction with the 3'-untranslated region of tau regulates tau expression.  Regarding BACE1, we found that alternative splicing leads to inactive splice isoforms and antisense oligonucleotides shift splicing toward these inactive isoforms to decrease A Î² production.  In addition, a G-quadruplex structure in the BACE1 mRNA plays a role in splice regulation.  The prospects for targeting tau and BACE1 mRNAs as therapeutic strategies will be discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQOSA4BMYD0yZPq5LRczm8ffW6udTcc2ebT2U7dS-RcmLntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2cflvFCqsw%253D%253D&md5=6e03652eb4aacf52189b1f5c5f02184a</span></div><a href="/servlet/linkout?suffix=cit95&amp;dbid=16384&amp;doi=10.1155%2F2014%2F757549&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1155%252F2014%252F757549%26sid%3Dliteratum%253Aachs%26aulast%3DWolfe%26aufirst%3DM.%2BS.%26atitle%3DTargeting%2520mRNA%2520for%2520Alzheimer%25E2%2580%2599s%2520and%2520related%2520dementias%26jtitle%3DScientifica%26date%3D2014%26volume%3D2014%26spage%3D757549%26doi%3D10.1155%2F2014%2F757549" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref96"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref96'); return false;" data-citation="" class="refNumLink">96</a></strong><div class="NLM_citation" id="cit96"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Artigas, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MarchÃ¡n, V.</span></span> <span> </span><span class="NLM_article-title">Synthesis and Tau RNA binding evaluation of ametantrone-containing ligands</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>80</i></span>,  <span class="NLM_fpage">2155</span>â <span class="NLM_lpage">2164</span>, <span class="refDoi">Â DOI: 10.1021/jo502661j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo502661j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVChtLk%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=80&publication_year=2015&pages=2155-2164&author=G.+Artigasauthor=V.+March%C3%A1n&title=Synthesis+and+Tau+RNA+binding+evaluation+of+ametantrone-containing+ligands&doi=10.1021%2Fjo502661j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit96R"><div class="casContent"><span class="casTitleNuber">96</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and Tau RNA Binding Evaluation of Ametantrone-Containing Ligands</span></div><div class="casAuthors">Artigas, Gerard; Marchan, Vicente</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">80</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">2155-2164</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We describe the synthesis and characterization of ametantrone-contg. RNA ligands based on the derivatization of this intercalator with two neamine moieties (Amt-Nea,Nea) or with one azaquinolone heterocycle and one neamine (Amt-Nea,Azq) as well as its combination with guanidinoneamine (Amt-NeaG4).  Biophys. studies revealed that guanidinylation of the parent ligand (Amt-Nea) had a pos. effect on the binding of the resulting compd. for Tau pre-mRNA target as well as on the stabilization upon complexation of some of the mutated RNA sequences assocd. with the development of tauopathies.  Further studies by NMR revealed the existence of a preferred binding site in the stem-loop structure, in which ametantrone intercalates in the characteristic bulged region.  Regarding doubly-functionalized ligands, binding affinity and stabilizing ability of Amt-Nea,Nea were similar to those of the guanidinylated ligand, but the two aminoglycoside fragments seem to interfere with its accommodation in a single binding site.  However, Amt-Nea,Azq binds at the bulged region in a similar way than Amt-NeaG4.  Overall, these results provide new insights on fine-tuning RNA binding properties of ametantrone by single or double derivatization with other RNA recognition motifs, which could help in the future design of new ligands with improved selectivity for disease-causing RNA mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGogzTuI8jMylLVg90H21EOLACvtfcHk0lj2FClSe3-_Ag"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVChtLk%253D&md5=e2048c1d0094ddc5155eb884b18ca7fa</span></div><a href="/servlet/linkout?suffix=cit96&amp;dbid=16384&amp;doi=10.1021%2Fjo502661j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo502661j%26sid%3Dliteratum%253Aachs%26aulast%3DArtigas%26aufirst%3DG.%26aulast%3DMarch%25C3%25A1n%26aufirst%3DV.%26atitle%3DSynthesis%2520and%2520Tau%2520RNA%2520binding%2520evaluation%2520of%2520ametantrone-containing%2520ligands%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2015%26volume%3D80%26spage%3D2155%26epage%3D2164%26doi%3D10.1021%2Fjo502661j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref97"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref97'); return false;" data-citation="" class="refNumLink">97</a></strong><div class="NLM_citation" id="cit97"><span><span class="NLM_contrib-group"><span class="NLM_string-name">LÃ³pez-SenÃ­n, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">GÃ³mez-Pinto, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandas, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MarchÃ¡n, V.</span></span> <span> </span><span class="NLM_article-title">Identification of ligands for the Tau exon 10 splicing regulatory element RNA by using dynamic combinatorial chemistry</span>. <i>Chem. - Eur. J.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">1946</span>â <span class="NLM_lpage">1953</span>, <span class="refDoi">Â DOI: 10.1002/chem.201002065</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1002%2Fchem.201002065" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=21274946" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFKlu74%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2011&pages=1946-1953&author=P.+L%C3%B3pez-Sen%C3%ADnauthor=I.+G%C3%B3mez-Pintoauthor=A.+Grandasauthor=V.+March%C3%A1n&title=Identification+of+ligands+for+the+Tau+exon+10+splicing+regulatory+element+RNA+by+using+dynamic+combinatorial+chemistry&doi=10.1002%2Fchem.201002065"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit97R"><div class="casContent"><span class="casTitleNuber">97</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of Ligands for the Tau Exon 10 Splicing Regulatory Element RNA by Using Dynamic Combinatorial Chemistry</span></div><div class="casAuthors">Lopez-Senin, Paula; Gomez-Pinto, Irene; Grandas, Anna; Marchan, Vicente</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry - A European Journal</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1946-1953, S1946/1-S1946/68</span>CODEN:
                <span class="NLM_cas:coden">CEUJED</span>;
        ISSN:<span class="NLM_cas:issn">0947-6539</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">We describe the use of dynamic combinatorial chem. (DCC) to identify ligands for the stem-loop structure located at the exon 10-5'-intron junction of Tau pre-mRNA, which is involved in the onset of several tauopathies including frontotemporal dementia with Parkinsonism linked to chromosome 17 (FTDP-17).  A series of ligands that combine the small aminoglycoside neamine and heteroarom. moieties (azaquinolone and two acridines) have been identified by using DCC.  These compds. effectively bind the stem-loop RNA target (the concn. required for 50 % RNA response (EC50): 2-58 Î¼M), as detd. by fluorescence titrn. expts.  Importantly, most of them are able to stabilize both the wild-type and the +3 and +14 mutated sequences assocd. with the development of FTDP-17 without producing a significant change in the overall structure of the RNA (as analyzed by CD spectroscopy), which is a key factor for recognition by the splicing regulatory machinery.  A good correlation has been found between the affinity of the ligands for the target and their ability to stabilize the RNA secondary structure.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpj2NBVOgduLrVg90H21EOLACvtfcHk0lizLBAWy-3CmQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFKlu74%253D&md5=2d9a5224455d93350dd05122f6ceddd1</span></div><a href="/servlet/linkout?suffix=cit97&amp;dbid=16384&amp;doi=10.1002%2Fchem.201002065&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fchem.201002065%26sid%3Dliteratum%253Aachs%26aulast%3DL%25C3%25B3pez-Sen%25C3%25ADn%26aufirst%3DP.%26aulast%3DG%25C3%25B3mez-Pinto%26aufirst%3DI.%26aulast%3DGrandas%26aufirst%3DA.%26aulast%3DMarch%25C3%25A1n%26aufirst%3DV.%26atitle%3DIdentification%2520of%2520ligands%2520for%2520the%2520Tau%2520exon%252010%2520splicing%2520regulatory%2520element%2520RNA%2520by%2520using%2520dynamic%2520combinatorial%2520chemistry%26jtitle%3DChem.%2520-%2520Eur.%2520J.%26date%3D2011%26volume%3D17%26spage%3D1946%26epage%3D1953%26doi%3D10.1002%2Fchem.201002065" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref98"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref98'); return false;" data-citation="" class="refNumLink">98</a></strong><div class="NLM_citation" id="cit98"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gianoncelli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Basili, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scalabrin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sosic, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moro, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zagotto, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palumbo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gresh, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gatto, B.</span></span> <span> </span><span class="NLM_article-title">Rational design, synthesis, and DNA binding properties of novel sequence-selective peptidyl congeners of ametantrone</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">1080</span>â <span class="NLM_lpage">1091</span>, <span class="refDoi">Â DOI: 10.1002/cmdc.201000106</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1002%2Fcmdc.201000106" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=20458714" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC3cXotVOmurg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2010&pages=1080-1091&author=A.+Gianoncelliauthor=S.+Basiliauthor=M.+Scalabrinauthor=A.+Sosicauthor=S.+Moroauthor=G.+Zagottoauthor=M.+Palumboauthor=N.+Greshauthor=B.+Gatto&title=Rational+design%2C+synthesis%2C+and+DNA+binding+properties+of+novel+sequence-selective+peptidyl+congeners+of+ametantrone&doi=10.1002%2Fcmdc.201000106"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit98R"><div class="casContent"><span class="casTitleNuber">98</span><div class="casTitle"><span class="NLM_cas:atitle">Rational Design, Synthesis, and DNA Binding Properties of Novel Sequence-Selective Peptidyl Congeners of Ametantrone</span></div><div class="casAuthors">Gianoncelli, Alessandra; Basili, Serena; Scalabrin, Matteo; Sosic, Alice; Moro, Stefano; Zagotto, Giuseppe; Palumbo, Manlio; Gresh, Nohad; Gatto, Barbara</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1080-1091</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Natural and synthetic compds. characterized by an anthraquinone nucleus represent an important class of anti-neoplastic agents, the mechanism of action of which is related to intercalation into DNA.  Ametantrone (AM) is a synthetic 9,10-anthracenedione bearing two (hydroxyethylamino)ethylamino residues at positions 1 and 4; along with other anthraquinones and anthracyclines, it shares a polycyclic intercalating moiety and charged side chains that stabilize DNA binding.  All these drugs elicit adverse side effects, which represent a challenge for antitumor chemotherapy.  In the present work the structure of AM was augmented with appropriate groups that target well-defined base pairs in the major groove.  These should endow AM with DNA sequence selectivity.  We describe the rationale for the synthesis and the evaluation of activity of a new series of compds. in which the planar anthraquinone is conjugated at positions 1 and 4 through the side chains of AM or other bioisosteric linkers to appropriate dipeptides.  The designed novel AM derivs. were shown to selectively stabilize two oligonucleotide duplexes that both have a palindromic GC-rich hexanucleotide core, but their stabilizing effects on a random DNA sequence was negligible.  In the case of the most effective compd., the 1,4-bis-[Gly-(L-Lys)] deriv. of AM, the exptl. results confirm the predictions of earlier theor. computations.  In contrast, AM had equal stabilizing effects on all three sequences and showed no preferential binding.  This novel peptide deriv. can be classified as a strong binder regarding the sequences that it selectively targets, possibly opening the exploitation of less cytotoxic conjugates of AM to the targeted treatment of oncol. and viral diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJASjZVc5nd7Vg90H21EOLACvtfcHk0lizLBAWy-3CmQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXotVOmurg%253D&md5=0e96bda08bba3f3ac943b7713754f7f3</span></div><a href="/servlet/linkout?suffix=cit98&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201000106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201000106%26sid%3Dliteratum%253Aachs%26aulast%3DGianoncelli%26aufirst%3DA.%26aulast%3DBasili%26aufirst%3DS.%26aulast%3DScalabrin%26aufirst%3DM.%26aulast%3DSosic%26aufirst%3DA.%26aulast%3DMoro%26aufirst%3DS.%26aulast%3DZagotto%26aufirst%3DG.%26aulast%3DPalumbo%26aufirst%3DM.%26aulast%3DGresh%26aufirst%3DN.%26aulast%3DGatto%26aufirst%3DB.%26atitle%3DRational%2520design%252C%2520synthesis%252C%2520and%2520DNA%2520binding%2520properties%2520of%2520novel%2520sequence-selective%2520peptidyl%2520congeners%2520of%2520ametantrone%26jtitle%3DChemMedChem%26date%3D2010%26volume%3D5%26spage%3D1080%26epage%3D1091%26doi%3D10.1002%2Fcmdc.201000106" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref99"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref99'); return false;" data-citation="" class="refNumLink">99</a></strong><div class="NLM_citation" id="cit99"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abe, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aikawa, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frank, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zembryski, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hubbs, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Withka, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steppan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogers, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cabral, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pettersson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wager, T. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fountain, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rumbaugh, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Childs-Disney, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span> <span> </span><span class="NLM_article-title">Design, optimization, and study of small Molecules that target tau pre-mRNA and affect splicing</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>142</i></span>,  <span class="NLM_fpage">8706</span>â <span class="NLM_lpage">8727</span>, <span class="refDoi">Â DOI: 10.1021/jacs.0c00768</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.0c00768" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A280%3ADC%252BB38vis1WrtA%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=142&publication_year=2020&pages=8706-8727&author=J.+L.+Chenauthor=P.+Zhangauthor=M.+Abeauthor=H.+Aikawaauthor=L.+Zhangauthor=A.+J.+Frankauthor=T.+Zembryskiauthor=C.+Hubbsauthor=H.+Parkauthor=J.+Withkaauthor=C.+Steppanauthor=L.+Rogersauthor=S.+Cabralauthor=M.+Petterssonauthor=T.+T.+Wagerauthor=M.+A.+Fountainauthor=G.+Rumbaughauthor=J.+L.+Childs-Disneyauthor=M.+D.+Disney&title=Design%2C+optimization%2C+and+study+of+small+Molecules+that+target+tau+pre-mRNA+and+affect+splicing&doi=10.1021%2Fjacs.0c00768"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit99R"><div class="casContent"><span class="casTitleNuber">99</span><div class="casTitle"><span class="NLM_cas:atitle">Design, Optimization, and Study of Small Molecules That Target Tau Pre-mRNA and Affect Splicing</span></div><div class="casAuthors">Chen Jonathan L; Zhang Peiyuan; Abe Masahito; Aikawa Haruo; Hubbs Christopher; Park HaJeung; Rumbaugh Gavin; Childs-Disney Jessica L; Disney Matthew D; Zhang Liying; Withka Jane; Pettersson Martin; Wager Travis T; Frank Alexander J; Zembryski Timothy; Fountain Matthew A; Steppan Claire; Rogers Lucy; Cabral Shawn</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">142</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">8706-8727</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Approximately 95% of human genes are alternatively spliced, and aberrant splicing events can cause disease.  One pre-mRNA that is alternatively spliced and linked to neurodegenerative diseases is tau (microtubule-associated protein tau), which can cause frontotemporal dementia and parkinsonism linked to chromosome 17 (FTDP-17) and can contribute to Alzheimer's disease.  Here, we describe the design of structure-specific lead small molecules that directly target tau pre-mRNA from sequence.  This was followed by hit expansion and analogue synthesis to further improve upon these initial lead molecules.  The emergent compounds were assessed for functional activity in a battery of assays, including binding assays and an assay that mimics molecular recognition of tau pre-mRNA by a U1 small nuclear ribonucleoprotein (snRNP) splicing factor.  Compounds that emerged from these studies had enhanced potency and selectivity for the target RNA relative to the initial hits, while also having significantly improved drug-like properties.  The compounds are shown to directly target tau pre-mRNA in cells, via chemical cross-linking and isolation by pull-down target profiling, and to rescue disease-relevant splicing of tau pre-mRNA in a variety of cellular systems, including primary neurons.  More broadly, this study shows that lead, structure-specific compounds can be designed from sequence and then further optimized for their physicochemical properties while at the same time enhancing their activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQ7onHoxn5nY0mhnSqMnvzDfW6udTcc2eYmBVN78lrRhrntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB38vis1WrtA%253D%253D&md5=3622b3f0ea9417db76b78b222fd75ddc</span></div><a href="/servlet/linkout?suffix=cit99&amp;dbid=16384&amp;doi=10.1021%2Fjacs.0c00768&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.0c00768%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DJ.%2BL.%26aulast%3DZhang%26aufirst%3DP.%26aulast%3DAbe%26aufirst%3DM.%26aulast%3DAikawa%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DFrank%26aufirst%3DA.%2BJ.%26aulast%3DZembryski%26aufirst%3DT.%26aulast%3DHubbs%26aufirst%3DC.%26aulast%3DPark%26aufirst%3DH.%26aulast%3DWithka%26aufirst%3DJ.%26aulast%3DSteppan%26aufirst%3DC.%26aulast%3DRogers%26aufirst%3DL.%26aulast%3DCabral%26aufirst%3DS.%26aulast%3DPettersson%26aufirst%3DM.%26aulast%3DWager%26aufirst%3DT.%2BT.%26aulast%3DFountain%26aufirst%3DM.%2BA.%26aulast%3DRumbaugh%26aufirst%3DG.%26aulast%3DChilds-Disney%26aufirst%3DJ.%2BL.%26aulast%3DDisney%26aufirst%3DM.%2BD.%26atitle%3DDesign%252C%2520optimization%252C%2520and%2520study%2520of%2520small%2520Molecules%2520that%2520target%2520tau%2520pre-mRNA%2520and%2520affect%2520splicing%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2020%26volume%3D142%26spage%3D8706%26epage%3D8727%26doi%3D10.1021%2Fjacs.0c00768" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref100"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref100'); return false;" data-citation="" class="refNumLink">100</a></strong><div class="NLM_citation" id="cit100"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brunden, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trojanowski, J. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, V. M. Y.</span></span> <span> </span><span class="NLM_article-title">Advances in tau-focused drug discovery for Alzheimerâs disease and related tauopathies</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">783</span>â <span class="NLM_lpage">793</span>, <span class="refDoi">Â DOI: 10.1038/nrd2959</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1038%2Fnrd2959" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=19794442" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtF2kurrN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2009&pages=783-793&author=K.+R.+Brundenauthor=J.+Q.+Trojanowskiauthor=V.+M.+Y.+Lee&title=Advances+in+tau-focused+drug+discovery+for+Alzheimer%E2%80%99s+disease+and+related+tauopathies&doi=10.1038%2Fnrd2959"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit100R"><div class="casContent"><span class="casTitleNuber">100</span><div class="casTitle"><span class="NLM_cas:atitle">Advances in tau-focused drug discovery for Alzheimer's disease and related tauopathies</span></div><div class="casAuthors">Brunden, Kurt R.; Trojanowski, John Q.; Lee, Virginia M.-Y.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">783-793</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Neuronal inclusions comprised of the microtubule-assocd. protein tau are found in numerous neurodegenerative diseases, commonly known as tauopathies.  In Alzheimer's disease - the most prevalent tauopathy - misfolded tau is probably a key pathol. agent.  The recent failure of amyloid-Î²-targeted therapeutics in Phase III clin. trials suggests that it is timely and prudent to consider alternative drug discovery strategies for Alzheimer's disease.  Here, we focus on strategies directed at reducing misfolded tau and compensating for the loss of normal tau function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqeWhB7oau9ALVg90H21EOLACvtfcHk0lj-3J_r4TzX2w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtF2kurrN&md5=6178c6f0e9a83c31a0e7f9dd8dae79d8</span></div><a href="/servlet/linkout?suffix=cit100&amp;dbid=16384&amp;doi=10.1038%2Fnrd2959&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2959%26sid%3Dliteratum%253Aachs%26aulast%3DBrunden%26aufirst%3DK.%2BR.%26aulast%3DTrojanowski%26aufirst%3DJ.%2BQ.%26aulast%3DLee%26aufirst%3DV.%2BM.%2BY.%26atitle%3DAdvances%2520in%2520tau-focused%2520drug%2520discovery%2520for%2520Alzheimer%25E2%2580%2599s%2520disease%2520and%2520related%2520tauopathies%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2009%26volume%3D8%26spage%3D783%26epage%3D793%26doi%3D10.1038%2Fnrd2959" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref101"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref101'); return false;" data-citation="" class="refNumLink">101</a></strong><div class="NLM_citation" id="cit101"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguez, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolfe, M. S.</span></span> <span> </span><span class="NLM_article-title">Template-directed synthesis of a small molecule-antisense conjugate targeting an mRNA structure</span>. <i>Bioorg. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">7</span>â <span class="NLM_lpage">11</span>, <span class="refDoi">Â DOI: 10.1016/j.bioorg.2014.03.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1016%2Fj.bioorg.2014.03.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=24691171" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC2cXpsl2qs7g%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2014&pages=7-11&author=Y.+Liuauthor=L.+Rodriguezauthor=M.+S.+Wolfe&title=Template-directed+synthesis+of+a+small+molecule-antisense+conjugate+targeting+an+mRNA+structure&doi=10.1016%2Fj.bioorg.2014.03.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit101R"><div class="casContent"><span class="casTitleNuber">101</span><div class="casTitle"><span class="NLM_cas:atitle">Template-directed synthesis of a small molecule-antisense conjugate targeting an mRNA structure</span></div><div class="casAuthors">Liu, Yang; Rodriguez, Lilia; Wolfe, Michael S.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">7-11</span>CODEN:
                <span class="NLM_cas:coden">BOCMBM</span>;
        ISSN:<span class="NLM_cas:issn">0045-2068</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The targeting of structural features in mRNA with specificity remains a great chem. challenge.  A hairpin structure near exon 10 in the pre-mRNA encoding the tau protein controls its splicing, and dementia-causing mutations that disrupt this structure increase exon 10 splicing.  We previously reported the discovery of small mols., mitoxantrone (MTX) and analogs, which bind to the tau RNA hairpin structure and the design of bipartite antisense oligonucleotides (ASOs) that simultaneously bind to the discontinuous sequences that flank this hairpin.  Herein we report the synthesis of a bipartite ASO conjugated to MTX using the tau RNA hairpin and flanking sequences as a template.  A set of six MTX analogs, each contg. a linker-azide, and a set of ten bipartite ASOs, each contg. a branched linker-alkyne, were synthesized and tested in combinatorial fashion for their ability to conjugate in the presence or absence of template RNA.  A single template-dependent MTX-ASO conjugate was identified from among the 60 reaction mixts., demonstrating that the MTX and ASO precursors could simultaneously bind the RNA template and allow proper positioning of azide and alkyne for 1,3-cycloaddn.  While the MTX-ASO conjugate proved too cytotoxic for cell-based assays, the conjugate inhibited tau exon 10 splicing under cell-free conditions more effectively than MTX or bipartite ASO alone.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqMQ3QTybseSLVg90H21EOLACvtfcHk0lj-3J_r4TzX2w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXpsl2qs7g%253D&md5=70e5fd0e2ed2120650af43f05be7a33b</span></div><a href="/servlet/linkout?suffix=cit101&amp;dbid=16384&amp;doi=10.1016%2Fj.bioorg.2014.03.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bioorg.2014.03.001%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DRodriguez%26aufirst%3DL.%26aulast%3DWolfe%26aufirst%3DM.%2BS.%26atitle%3DTemplate-directed%2520synthesis%2520of%2520a%2520small%2520molecule-antisense%2520conjugate%2520targeting%2520an%2520mRNA%2520structure%26jtitle%3DBioorg.%2520Chem.%26date%3D2014%26volume%3D54%26spage%3D7%26epage%3D11%26doi%3D10.1016%2Fj.bioorg.2014.03.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref102"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref102'); return false;" data-citation="" class="refNumLink">102</a></strong><div class="NLM_citation" id="cit102"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donahue, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolfe, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Varani, G.</span></span> <span> </span><span class="NLM_article-title">Structural basis for stabilization of the Tau pre-mRNA splicing regulatory element by novantrone (mitoxantrone)</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">557</span>â <span class="NLM_lpage">566</span>, <span class="refDoi">Â DOI: 10.1016/j.chembiol.2009.03.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1016%2Fj.chembiol.2009.03.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=19477420" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BD1MXms1Ogtb4%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2009&pages=557-566&author=S.+Zhengauthor=Y.+Chenauthor=C.+P.+Donahueauthor=M.+S.+Wolfeauthor=G.+Varani&title=Structural+basis+for+stabilization+of+the+Tau+pre-mRNA+splicing+regulatory+element+by+novantrone+%28mitoxantrone%29&doi=10.1016%2Fj.chembiol.2009.03.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit102R"><div class="casContent"><span class="casTitleNuber">102</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Basis for Stabilization of the Tau Pre-mRNA Splicing Regulatory Element by Novantrone (Mitoxantrone)</span></div><div class="casAuthors">Zheng, Suxin; Chen, Yu; Donahue, Christine P.; Wolfe, Michael S.; Varani, Gabriele</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">557-566</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Some familial neurodegenerative diseases are assocd. with mutations that destabilize a putative stem-loop structure within an intronic region of the tau pre-mRNA and alter the prodn. of tau protein isoforms by alternative splicing.  Because stabilization of the stem loop reverses the splicing pattern assocd. with neurodegeneration, small mols. that stabilize this stem loop would provide new ways to dissect the mechanism of neurodegeneration and treat tauopathies.  The anticancer drug mitoxantrone was recently identified in a high throughput screen to stabilize the tau pre-mRNA stem loop.  Here the authors report the soln. structure of the tau mRNA-mitoxantrone complex, validated by the structure-activity relationship of existing mitoxantrone analogs.  The structure describes the mol. basis for their interaction with RNA and provides a rational basis to optimize the activity of this new class of RNA-binding mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrNxlGdrUiUcrVg90H21EOLACvtfcHk0ljEecifNHzxbA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXms1Ogtb4%253D&md5=3d38884c150535d160fcecc5d1f03534</span></div><a href="/servlet/linkout?suffix=cit102&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2009.03.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2009.03.009%26sid%3Dliteratum%253Aachs%26aulast%3DZheng%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DDonahue%26aufirst%3DC.%2BP.%26aulast%3DWolfe%26aufirst%3DM.%2BS.%26aulast%3DVarani%26aufirst%3DG.%26atitle%3DStructural%2520basis%2520for%2520stabilization%2520of%2520the%2520Tau%2520pre-mRNA%2520splicing%2520regulatory%2520element%2520by%2520novantrone%2520%2528mitoxantrone%2529%26jtitle%3DChem.%2520Biol.%26date%3D2009%26volume%3D16%26spage%3D557%26epage%3D566%26doi%3D10.1016%2Fj.chembiol.2009.03.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref103"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref103'); return false;" data-citation="" class="refNumLink">103</a></strong><div class="NLM_citation" id="cit103"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Peacey, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguez, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolfe, M. S.</span></span> <span> </span><span class="NLM_article-title">Targeting a pre-mRNA structure with bipartite antisense molecules modulates tau alternative splicing</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">9836</span>â <span class="NLM_lpage">9849</span>, <span class="refDoi">Â DOI: 10.1093/nar/gks710</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1093%2Fnar%2Fgks710" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=22844088" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFygurnF" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2012&pages=9836-9849&author=E.+Peaceyauthor=L.+Rodriguezauthor=Y.+Liuauthor=M.+S.+Wolfe&title=Targeting+a+pre-mRNA+structure+with+bipartite+antisense+molecules+modulates+tau+alternative+splicing&doi=10.1093%2Fnar%2Fgks710"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit103R"><div class="casContent"><span class="casTitleNuber">103</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting a pre-mRNA structure with bipartite antisense molecules modulates tau alternative splicing</span></div><div class="casAuthors">Peacey, Eleanor; Rodriguez, Lilia; Liu, Yang; Wolfe, Michael S.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">9836-9849</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Approx. 15% of human genetic diseases are estd. to involve dysregulation of alternative pre-mRNA splicing.  Antisense mols. designed to alter these and other splicing events typically target continuous linear sequences of the message.  Here, we show that a structural feature in a pre-mRNA can be targeted by bipartite antisense mols. designed to hybridize with the discontinuous elements that flank the structure and thereby alter splicing.  We targeted a hairpin structure at the boundary between exon 10 and intron 10 of the pre-mRNA of tau.  Mutations in this region that are assocd. with certain forms of frontotemporal dementia, destabilize the hairpin to cause increased inclusion of exon 10.  Via electrophoretic mobility shift and RNase protection assays, we demonstrate that bipartite antisense mols. designed to simultaneously interact with the available sequences that immediately flank the tau pre-mRNA hairpin do indeed bind to this structured region.  Moreover, these agents inhibit exon 10 splicing and reverse the effect of destabilizing disease-causing mutations, in both in vitro splicing assays and cell culture.  This general bipartite antisense strategy could be employed to modulate other splicing events that are regulated by RNA secondary structure.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpC63B1SXFOE7Vg90H21EOLACvtfcHk0ljEecifNHzxbA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFygurnF&md5=63d1cc9c4956e68211bf35a3081bcd08</span></div><a href="/servlet/linkout?suffix=cit103&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgks710&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgks710%26sid%3Dliteratum%253Aachs%26aulast%3DPeacey%26aufirst%3DE.%26aulast%3DRodriguez%26aufirst%3DL.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DWolfe%26aufirst%3DM.%2BS.%26atitle%3DTargeting%2520a%2520pre-mRNA%2520structure%2520with%2520bipartite%2520antisense%2520molecules%2520modulates%2520tau%2520alternative%2520splicing%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2012%26volume%3D40%26spage%3D9836%26epage%3D9849%26doi%3D10.1093%2Fnar%2Fgks710" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref104"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref104'); return false;" data-citation="" class="refNumLink">104</a></strong><div class="NLM_citation" id="cit104"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sumner, C.</span>; <span class="NLM_string-name">Paushkin, S.</span>; <span class="NLM_string-name">Ko, C.-P.</span></span> <i>Spinal muscular atrophy: Disease mechanism and therapy.</i>; <span class="NLM_publisher-name">Academic Press</span>, <span class="NLM_publisher-loc">Cambridge</span>, <span class="NLM_year">2016</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&author=C.+Sumner&author=S.+Paushkin&author=C.-P.+Ko&title=Spinal+muscular+atrophy%3A+Disease+mechanism+and+therapy."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit104&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DSumner%26aufirst%3DC.%26jtitle%3DSpinal%2520muscular%2520atrophy%253A%2520Disease%2520mechanism%2520and%2520therapy.%26pub%3DAcademic%2520Press%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref105"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref105'); return false;" data-citation="" class="refNumLink">105</a></strong><div class="NLM_citation" id="cit105"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ratni, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ebeling, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baird, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bendels, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bylund, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denk, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guerard, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jablonski, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobsen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khwaja, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kletzl, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ko, C.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kustermann, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marquet, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Metzger, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mueller, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naryshkin, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paushkin, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pinard, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poirier, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reutlinger, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weetall, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeller, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mueller, L.</span></span> <span> </span><span class="NLM_article-title">Discovery of risdiplam, a selective survival of motor neuron-2 (SMN2) gene splicing modifier for the treatment of spinal muscular atrophy (SMA)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">6501</span>â <span class="NLM_lpage">6517</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.8b00741</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00741" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref105/cit105&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtlOgu7vF" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=6501-6517&author=H.+Ratniauthor=M.+Ebelingauthor=J.+Bairdauthor=S.+Bendelsauthor=J.+Bylundauthor=K.+S.+Chenauthor=N.+Denkauthor=Z.+Fengauthor=L.+Greenauthor=M.+Guerardauthor=P.+Jablonskiauthor=B.+Jacobsenauthor=O.+Khwajaauthor=H.+Kletzlauthor=C.-P.+Koauthor=S.+Kustermannauthor=A.+Marquetauthor=F.+Metzgerauthor=B.+Muellerauthor=N.+A.+Naryshkinauthor=S.+V.+Paushkinauthor=E.+Pinardauthor=A.+Poirierauthor=M.+Reutlingerauthor=M.+Weetallauthor=A.+Zellerauthor=X.+Zhaoauthor=L.+Mueller&title=Discovery+of+risdiplam%2C+a+selective+survival+of+motor+neuron-2+%28SMN2%29+gene+splicing+modifier+for+the+treatment+of+spinal+muscular+atrophy+%28SMA%29&doi=10.1021%2Facs.jmedchem.8b00741"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit105R"><div class="casContent"><span class="casTitleNuber">105</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Risdiplam, a Selective Survival of Motor Neuron-2 (SMN2) Gene Splicing Modifier for the Treatment of Spinal Muscular Atrophy (SMA)</span></div><div class="casAuthors">Ratni, Hasane; Ebeling, Martin; Baird, John; Bendels, Stefanie; Bylund, Johan; Chen, Karen S.; Denk, Nora; Feng, Zhihua; Green, Luke; Guerard, Melanie; Jablonski, Philippe; Jacobsen, Bjoern; Khwaja, Omar; Kletzl, Heidemarie; Ko, Chien-Ping; Kustermann, Stefan; Marquet, Anne; Metzger, Friedrich; Mueller, Barbara; Naryshkin, Nikolai A.; Paushkin, Sergey V.; Pinard, Emmanuel; Poirier, Agnes; Reutlinger, Michael; Weetall, Marla; Zeller, Andreas; Zhao, Xin; Mueller, Lutz</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">6501-6517</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">SMA is an inherited disease that leads to loss of motor function and ambulation and a reduced life expectancy.  We have been working to develop orally administrated, systemically distributed small mols. to increase levels of functional SMN protein.  Compd. 2 was the first SMN2 splicing modifier tested in clin. trials in healthy volunteers and SMA patients.  It was safe and well tolerated and increased SMN protein levels up to 2-fold in patients.  Nevertheless, its development was stopped as a precautionary measure because retinal toxicity was obsd. in cynomolgus monkeys after chronic daily oral dosing (39 wk) at exposures in excess of those investigated in patients.  Herein, we describe the discovery of 1 (risdiplam, RG7916, RO7034067) that focused on thorough pharmacol., DMPK and safety characterization and optimization.  This compd. is undergoing pivotal clin. trials and is a promising medicine for the treatment of patients in all ages and stages with SMA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCF_mNT3UFurVg90H21EOLACvtfcHk0liR3e3BJJeUYw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtlOgu7vF&md5=966fa21a9b1bbcc4d64ac5c90508a23b</span></div><a href="/servlet/linkout?suffix=cit105&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00741&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00741%26sid%3Dliteratum%253Aachs%26aulast%3DRatni%26aufirst%3DH.%26aulast%3DEbeling%26aufirst%3DM.%26aulast%3DBaird%26aufirst%3DJ.%26aulast%3DBendels%26aufirst%3DS.%26aulast%3DBylund%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DK.%2BS.%26aulast%3DDenk%26aufirst%3DN.%26aulast%3DFeng%26aufirst%3DZ.%26aulast%3DGreen%26aufirst%3DL.%26aulast%3DGuerard%26aufirst%3DM.%26aulast%3DJablonski%26aufirst%3DP.%26aulast%3DJacobsen%26aufirst%3DB.%26aulast%3DKhwaja%26aufirst%3DO.%26aulast%3DKletzl%26aufirst%3DH.%26aulast%3DKo%26aufirst%3DC.-P.%26aulast%3DKustermann%26aufirst%3DS.%26aulast%3DMarquet%26aufirst%3DA.%26aulast%3DMetzger%26aufirst%3DF.%26aulast%3DMueller%26aufirst%3DB.%26aulast%3DNaryshkin%26aufirst%3DN.%2BA.%26aulast%3DPaushkin%26aufirst%3DS.%2BV.%26aulast%3DPinard%26aufirst%3DE.%26aulast%3DPoirier%26aufirst%3DA.%26aulast%3DReutlinger%26aufirst%3DM.%26aulast%3DWeetall%26aufirst%3DM.%26aulast%3DZeller%26aufirst%3DA.%26aulast%3DZhao%26aufirst%3DX.%26aulast%3DMueller%26aufirst%3DL.%26atitle%3DDiscovery%2520of%2520risdiplam%252C%2520a%2520selective%2520survival%2520of%2520motor%2520neuron-2%2520%2528SMN2%2529%2520gene%2520splicing%2520modifier%2520for%2520the%2520treatment%2520of%2520spinal%2520muscular%2520atrophy%2520%2528SMA%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D6501%26epage%3D6517%26doi%3D10.1021%2Facs.jmedchem.8b00741" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref106"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref106'); return false;" data-citation="" class="refNumLink">106</a></strong><div class="NLM_citation" id="cit106"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheung, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurley, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerrigan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chin, D. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">OâBrien, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sung, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Axford, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cody, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fazal, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fridrich, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanchez, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomlinson, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jain, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffmaster, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoosear, M. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Servais, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Towler, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hild, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curtis, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dietrich, W. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamann, L. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Briner, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobayashi, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sivasankaran, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dales, N. A.</span></span> <span> </span><span class="NLM_article-title">Discovery of small molecule splicing modulators of survival motor neuron-2 (SMN2) for the treatment of spinal muscular atrophy (SMA)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">11021</span>â <span class="NLM_lpage">11036</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.8b01291</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01291" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref106/cit106&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitFeju7nL" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=11021-11036&author=A.+K.+Cheungauthor=B.+Hurleyauthor=R.+Kerriganauthor=L.+Shuauthor=D.+N.+Chinauthor=Y.+Shenauthor=G.+O%E2%80%99Brienauthor=M.+J.+Sungauthor=Y.+Houauthor=J.+Axfordauthor=E.+Codyauthor=R.+Sunauthor=A.+Fazalauthor=C.+Fridrichauthor=C.+C.+Sanchezauthor=R.+C.+Tomlinsonauthor=M.+Jainauthor=L.+Dengauthor=K.+Hoffmasterauthor=C.+Songauthor=M.+V.+Hoosearauthor=Y.+Shinauthor=R.+Servaisauthor=C.+Towlerauthor=M.+Hildauthor=D.+Curtisauthor=W.+F.+Dietrichauthor=L.+G.+Hamannauthor=K.+Brinerauthor=K.+S.+Chenauthor=D.+Kobayashiauthor=R.+Sivasankaranauthor=N.+A.+Dales&title=Discovery+of+small+molecule+splicing+modulators+of+survival+motor+neuron-2+%28SMN2%29+for+the+treatment+of+spinal+muscular+atrophy+%28SMA%29&doi=10.1021%2Facs.jmedchem.8b01291"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit106R"><div class="casContent"><span class="casTitleNuber">106</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Small Molecule Splicing Modulators of Survival Motor Neuron-2 (SMN2) for the Treatment of Spinal Muscular Atrophy (SMA)</span></div><div class="casAuthors">Cheung, Atwood K.; Hurley, Brian; Kerrigan, Ryan; Shu, Lei; Chin, Donovan N.; Shen, Yiping; OBrien, Gary; Sung, Moo Je; Hou, Ying; Axford, Jake; Cody, Emma; Sun, Robert; Fazal, Aleem; Tomlinson, Ronald C.; Jain, Monish; Deng, Lin; Hoffmaster, Keith; Song, Cheng; Van Hoosear, Mailin; Shin, Youngah; Servais, Rebecca; Towler, Christopher; Hild, Marc; Curtis, Daniel; Dietrich, William F.; Hamann, Lawrence G.; Briner, Karin; Chen, Karen S.; Kobayashi, Dione; Sivasankaran, Rajeev; Dales, Natalie A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">11021-11036</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Spinal muscular atrophy (SMA), a rare neuromuscular disorder, is the leading genetic cause of death in infants and toddlers.  SMA is caused by the deletion or a loss of function mutation of the survival motor neuron 1 (SMN1) gene.  In humans, a second closely related gene SMN2 exists, however it codes for a less stable SMN protein.  In recent years, significant progress has been made toward disease modifying treatments for SMA by modulating SMN2 pre-mRNA splicing.  Herein, we describe the discovery of LMI070/branaplam, a small mol. that stabilizes the interaction between the spliceosome and SMN2 pre-mRNA.  Branaplam (1) originated from a high-throughput phenotypic screening hit, pyridazine 2, and evolved via multi-parameter lead optimization.  In a severe mouse SMA model, branaplam treatment increased full-length SMN RNA and protein levels, and extended survival.  Currently, branaplam is in clin. studies for SMA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprrHPkSEhGhrVg90H21EOLACvtfcHk0liR3e3BJJeUYw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitFeju7nL&md5=c564332b680a6ef0ad06bc9e5a4db16e</span></div><a href="/servlet/linkout?suffix=cit106&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01291&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01291%26sid%3Dliteratum%253Aachs%26aulast%3DCheung%26aufirst%3DA.%2BK.%26aulast%3DHurley%26aufirst%3DB.%26aulast%3DKerrigan%26aufirst%3DR.%26aulast%3DShu%26aufirst%3DL.%26aulast%3DChin%26aufirst%3DD.%2BN.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DO%25E2%2580%2599Brien%26aufirst%3DG.%26aulast%3DSung%26aufirst%3DM.%2BJ.%26aulast%3DHou%26aufirst%3DY.%26aulast%3DAxford%26aufirst%3DJ.%26aulast%3DCody%26aufirst%3DE.%26aulast%3DSun%26aufirst%3DR.%26aulast%3DFazal%26aufirst%3DA.%26aulast%3DFridrich%26aufirst%3DC.%26aulast%3DSanchez%26aufirst%3DC.%2BC.%26aulast%3DTomlinson%26aufirst%3DR.%2BC.%26aulast%3DJain%26aufirst%3DM.%26aulast%3DDeng%26aufirst%3DL.%26aulast%3DHoffmaster%26aufirst%3DK.%26aulast%3DSong%26aufirst%3DC.%26aulast%3DHoosear%26aufirst%3DM.%2BV.%26aulast%3DShin%26aufirst%3DY.%26aulast%3DServais%26aufirst%3DR.%26aulast%3DTowler%26aufirst%3DC.%26aulast%3DHild%26aufirst%3DM.%26aulast%3DCurtis%26aufirst%3DD.%26aulast%3DDietrich%26aufirst%3DW.%2BF.%26aulast%3DHamann%26aufirst%3DL.%2BG.%26aulast%3DBriner%26aufirst%3DK.%26aulast%3DChen%26aufirst%3DK.%2BS.%26aulast%3DKobayashi%26aufirst%3DD.%26aulast%3DSivasankaran%26aufirst%3DR.%26aulast%3DDales%26aufirst%3DN.%2BA.%26atitle%3DDiscovery%2520of%2520small%2520molecule%2520splicing%2520modulators%2520of%2520survival%2520motor%2520neuron-2%2520%2528SMN2%2529%2520for%2520the%2520treatment%2520of%2520spinal%2520muscular%2520atrophy%2520%2528SMA%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D11021%26epage%3D11036%26doi%3D10.1021%2Facs.jmedchem.8b01291" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref107"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref107'); return false;" data-citation="" class="refNumLink">107</a></strong><div class="NLM_citation" id="cit107"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Orengo, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ward, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooper, T. A.</span></span> <span> </span><span class="NLM_article-title">Alternative splicing dysregulation secondary to skeletal muscle regeneration</span>. <i>Ann. Neurol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>69</i></span>,  <span class="NLM_fpage">681</span>â <span class="NLM_lpage">690</span>, <span class="refDoi">Â DOI: 10.1002/ana.22278</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1002%2Fana.22278" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=21400563" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC3MXltVygtbw%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2011&pages=681-690&author=J.+P.+Orengoauthor=A.+J.+Wardauthor=T.+A.+Cooper&title=Alternative+splicing+dysregulation+secondary+to+skeletal+muscle+regeneration&doi=10.1002%2Fana.22278"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit107R"><div class="casContent"><span class="casTitleNuber">107</span><div class="casTitle"><span class="NLM_cas:atitle">Alternative splicing dysregulation secondary to skeletal muscle regeneration</span></div><div class="casAuthors">Orengo, James P.; Ward, Amanda J.; Cooper, Thomas A.</div><div class="citationInfo"><span class="NLM_cas:title">Annals of Neurology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">681-690</span>CODEN:
                <span class="NLM_cas:coden">ANNED3</span>;
        ISSN:<span class="NLM_cas:issn">0364-5134</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">Objective : Dysregulation of alternative splicing has become a mol. hallmark of myotonic dystrophy type 1 (DM1), in which neonatal splice variants are expressed in adult skeletal muscle.  Splicing dysregulation is induced by RNA contg. expanded CUG repeats expressed from the expanded mutant allele by sequestration of muscleblindlike 1 (MBNL1) protein within nuclear RNA foci and increased CUGBP, ELAV-like family member 1 (CELF1) protein levels.  Dysregulated splicing has also been identified in other neuromuscular disorders, suggesting either that diseases with different mol. causes share a common pathogenic mechanism or that dysregulated splicing can also be a common secondary consequence of muscle degeneration and regeneration.  Methods : In this study, we examd. regulation of alternative splicing in 4 different mouse models of muscular dystrophy, including DM1, limb-girdle muscular dystrophy, congenital merosin-deficient muscular dystrophy, and Duchenne muscular dystrophy, and 2 myotoxin (cardiotoxin and notexin) muscle injury models.  Results : We show that DM1-like alternative splicing dysregulation and altered expression of MBNL1 and CELF1 occur in non-DM1 mouse models of muscular dystrophy and muscle injury, most likely due to recapitulation of neonatal splicing patterns in regenerating fibers.  In contrast, CELF1 was elevated in nuclei of mature myofibers of the DM1 model, consistent with a primary effect of pathogenic RNA expression.  Interpretation : Splicing dysregulation in DM1 is a primary effect of RNA contg. expanded CUG repeats.  However, we conclude that splicing changes can also be obsd. secondary to muscle regeneration, and this possibility must be taken into account when evaluating cause-effect relationships between dysregulated splicing and disease processes.  Ann Neurol 2010;.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomAbRWIByLm7Vg90H21EOLACvtfcHk0li8-Z4iHWRCHw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXltVygtbw%253D&md5=ba02b3ea4229090c392f9204dd1658b4</span></div><a href="/servlet/linkout?suffix=cit107&amp;dbid=16384&amp;doi=10.1002%2Fana.22278&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fana.22278%26sid%3Dliteratum%253Aachs%26aulast%3DOrengo%26aufirst%3DJ.%2BP.%26aulast%3DWard%26aufirst%3DA.%2BJ.%26aulast%3DCooper%26aufirst%3DT.%2BA.%26atitle%3DAlternative%2520splicing%2520dysregulation%2520secondary%2520to%2520skeletal%2520muscle%2520regeneration%26jtitle%3DAnn.%2520Neurol.%26date%3D2011%26volume%3D69%26spage%3D681%26epage%3D690%26doi%3D10.1002%2Fana.22278" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref108"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref108'); return false;" data-citation="" class="refNumLink">108</a></strong><div class="NLM_citation" id="cit108"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wong, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peh, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luu, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanchez, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tuccinardi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsoi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, W. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, H. Y. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baranger, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hergenrother, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimmerman, S. C.</span></span> <span> </span><span class="NLM_article-title">Targeting toxic RNAs that cause myotonic dystrophy Type 1 (DM1) with a bisamidinium inhibitor</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>136</i></span>,  <span class="NLM_fpage">6355</span>â <span class="NLM_lpage">6361</span>, <span class="refDoi">Â DOI: 10.1021/ja5012146</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja5012146" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref108/cit108&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC2cXls1Crsb8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=136&publication_year=2014&pages=6355-6361&author=C.+H.+Wongauthor=L.+Nguyenauthor=J.+Pehauthor=L.+M.+Luuauthor=J.+S.+Sanchezauthor=S.+L.+Richardsonauthor=T.+Tuccinardiauthor=H.+Tsoiauthor=W.+Y.+Chanauthor=H.+Y.+E.+Chanauthor=A.+M.+Barangerauthor=P.+J.+Hergenrotherauthor=S.+C.+Zimmerman&title=Targeting+toxic+RNAs+that+cause+myotonic+dystrophy+Type+1+%28DM1%29+with+a+bisamidinium+inhibitor&doi=10.1021%2Fja5012146"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit108R"><div class="casContent"><span class="casTitleNuber">108</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting Toxic RNAs that Cause Myotonic Dystrophy Type 1 (DM1) with a Bisamidinium Inhibitor</span></div><div class="casAuthors">Wong, Chun-Ho; Nguyen, Lien; Peh, Jessie; Luu, Long M.; Sanchez, Jeannette S.; Richardson, Stacie L.; Tuccinardi, Tiziano; Tsoi, Ho; Chan, Wood Yee; Chan, H. Y. Edwin; Baranger, Anne M.; Hergenrother, Paul J.; Zimmerman, Steven C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">136</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">6355-6361</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A working hypothesis for the pathogenesis of myotonic dystrophy type 1 (DM1) involves the aberrant sequestration of an alternative splicing regulator, MBNL1, by expanded CUG repeats, r(CUG)exp.  It has been suggested that a reversal of the myotonia and potentially other symptoms of the DM1 disease can be achieved by inhibiting the toxic MBNL1-r(CUG)exp interaction.  Using rational design, we discovered an RNA-groove binding inhibitor (ligand 3) that contains two triaminotriazine units connected by a bisamidinium linker.  Ligand 3 binds r(CUG)12 with a low micromolar affinity (Kd = 8Â±2 Î¼M) and disrupts the MBNL1-r(CUG)12 interaction in vitro (Ki = 8Â±2 Î¼M).  In addn., ligand 3 is cell and nucleus permeable, exhibits negligible toxicity to mammalian cells, dissolves MBNL1-r(CUG)exp ribonuclear foci, and restores misregulated splicing of IR and cTNT in a DM1 cell culture model.  Importantly, suppression of r(CUG)exp RNA-induced toxicity in a DM1 Drosophila model was obsd. after treatment with ligand 3.  These results suggest ligand 3 as a lead for the treatment of DM1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqfs_zU_KI7w7Vg90H21EOLACvtfcHk0li8-Z4iHWRCHw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXls1Crsb8%253D&md5=205bd661957ffd5673ae516ebd6769d4</span></div><a href="/servlet/linkout?suffix=cit108&amp;dbid=16384&amp;doi=10.1021%2Fja5012146&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja5012146%26sid%3Dliteratum%253Aachs%26aulast%3DWong%26aufirst%3DC.%2BH.%26aulast%3DNguyen%26aufirst%3DL.%26aulast%3DPeh%26aufirst%3DJ.%26aulast%3DLuu%26aufirst%3DL.%2BM.%26aulast%3DSanchez%26aufirst%3DJ.%2BS.%26aulast%3DRichardson%26aufirst%3DS.%2BL.%26aulast%3DTuccinardi%26aufirst%3DT.%26aulast%3DTsoi%26aufirst%3DH.%26aulast%3DChan%26aufirst%3DW.%2BY.%26aulast%3DChan%26aufirst%3DH.%2BY.%2BE.%26aulast%3DBaranger%26aufirst%3DA.%2BM.%26aulast%3DHergenrother%26aufirst%3DP.%2BJ.%26aulast%3DZimmerman%26aufirst%3DS.%2BC.%26atitle%3DTargeting%2520toxic%2520RNAs%2520that%2520cause%2520myotonic%2520dystrophy%2520Type%25201%2520%2528DM1%2529%2520with%2520a%2520bisamidinium%2520inhibitor%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2014%26volume%3D136%26spage%3D6355%26epage%3D6361%26doi%3D10.1021%2Fja5012146" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref109"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref109'); return false;" data-citation="" class="refNumLink">109</a></strong><div class="NLM_citation" id="cit109"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Arambula, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramisetty, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baranger, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimmerman, S. C.</span></span> <span> </span><span class="NLM_article-title">A simple ligand that selectively targets CUG trinucleotide repeats and inhibits MBNL protein binding</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>106</i></span>,  <span class="NLM_fpage">16068</span>â <span class="NLM_lpage">16073</span>, <span class="refDoi">Â DOI: 10.1073/pnas.0901824106</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref109/cit109&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1073%2Fpnas.0901824106" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref109/cit109&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=19805260" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref109/cit109&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht1OjsbjO" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2009&pages=16068-16073&author=J.+F.+Arambulaauthor=S.+R.+Ramisettyauthor=A.+M.+Barangerauthor=S.+C.+Zimmerman&title=A+simple+ligand+that+selectively+targets+CUG+trinucleotide+repeats+and+inhibits+MBNL+protein+binding&doi=10.1073%2Fpnas.0901824106"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit109R"><div class="casContent"><span class="casTitleNuber">109</span><div class="casTitle"><span class="NLM_cas:atitle">A simple ligand that selectively targets CUG trinucleotide repeats and inhibits MBNL protein binding</span></div><div class="casAuthors">Arambula, Jonathan F.; Ramisetty, Sreenivasa Rao; Baranger, Anne M.; Zimmerman, Steven C.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">106</span>
        (<span class="NLM_cas:issue">38</span>),
    <span class="NLM_cas:pages">16068-16073, S16068/1-S16068/39</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">This work describes the rational design, synthesis, and study of a ligand that selectively complexes CUG repeats in RNA (and CTG repeats in DNA) with high nanomolar affinity.  This sequence is considered a causative agent of myotonic dystrophy type 1 (DM1) because of its ability to sequester muscleblind-like (MBNL) proteins.  Ligand 1 was synthesized in two steps from com. available compds., and its binding to CTG and CUG repeats in oligonucleotides studied.  Isothermal titrn. calorimetry studies of 1 with various sequences showed a preference toward the T-T mismatch (Kd of 390 Â± 80 nM) with a 13-, 169-, and 85-fold redn. in affinity toward single C-C, A-A, and G-G mismatches, resp.  Binding and Job anal. of 1 to multiple CTG step sequences revealed high affinity binding to every other T-T mismatch with neg. cooperativity for proximal T-T mismatches.  The affinity of 1 for a (CUG)4 step provided a Kd of 430 nM with a binding stoichiometry of 1:1.  The preference for the U-U in RNA was maintained with a 6-, >143-, and >143-fold redn. in affinity toward single C-C, A-A, and G-G mismatches, resp.  Ligand 1 destabilized the complexes formed between MBNL1N and (CUG)4 and (CUG)12 with IC50 values of 52 Â± 20 Î¼IM and 46 Â± 7 Î¼M, resp., and Ki values of 6 Â± 2 Î¼M and 7 Â± 1 Î¼M, resp.  These values were only minimally altered by the addn. of competitor tRNA.  Ligand 1 does not destabilize the unrelated RNA-protein complexes the U1A-SL2 RNA complex and the Sex lethal-tra RNA complex.  Thus, ligand 1 selectively destabilizes the MBNL1N-poly(CUG) complex.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrjnqvChNuskbVg90H21EOLACvtfcHk0lj3X_h_wF-9Fw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXht1OjsbjO&md5=586a30b81ac235985c76e4fea5e9e34f</span></div><a href="/servlet/linkout?suffix=cit109&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0901824106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0901824106%26sid%3Dliteratum%253Aachs%26aulast%3DArambula%26aufirst%3DJ.%2BF.%26aulast%3DRamisetty%26aufirst%3DS.%2BR.%26aulast%3DBaranger%26aufirst%3DA.%2BM.%26aulast%3DZimmerman%26aufirst%3DS.%2BC.%26atitle%3DA%2520simple%2520ligand%2520that%2520selectively%2520targets%2520CUG%2520trinucleotide%2520repeats%2520and%2520inhibits%2520MBNL%2520protein%2520binding%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2009%26volume%3D106%26spage%3D16068%26epage%3D16073%26doi%3D10.1073%2Fpnas.0901824106" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref110"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref110'); return false;" data-citation="" class="refNumLink">110</a></strong><div class="NLM_citation" id="cit110"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Siboni, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakamori, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Struck, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coonrod, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harriott, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cass, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanner, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berglund, J. A.</span></span> <span> </span><span class="NLM_article-title">Actinomycin D specifically reduces expanded CUG repeat RNA in myotonic mystrophy models</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">2386</span>â <span class="NLM_lpage">2394</span>, <span class="refDoi">Â DOI: 10.1016/j.celrep.2015.11.028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref110/cit110&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1016%2Fj.celrep.2015.11.028" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref110/cit110&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=26686629" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref110/cit110&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitVSksb3E" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2015&pages=2386-2394&author=R.+B.+Siboniauthor=M.+Nakamoriauthor=S.+D.+Wagnerauthor=A.+J.+Struckauthor=L.+A.+Coonrodauthor=S.+A.+Harriottauthor=D.+M.+Cassauthor=M.+K.+Tannerauthor=J.+A.+Berglund&title=Actinomycin+D+specifically+reduces+expanded+CUG+repeat+RNA+in+myotonic+mystrophy+models&doi=10.1016%2Fj.celrep.2015.11.028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit110R"><div class="casContent"><span class="casTitleNuber">110</span><div class="casTitle"><span class="NLM_cas:atitle">Actinomycin D Specifically Reduces Expanded CUG Repeat RNA in Myotonic Dystrophy Models</span></div><div class="casAuthors">Siboni, Ruth B.; Nakamori, Masayuki; Wagner, Stacey D.; Struck, Adam J.; Coonrod, Leslie A.; Harriott, Shanee A.; Cass, Daniel M.; Tanner, Matthew K.; Berglund, J. Andrew</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2386-2394</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Myotonic dystrophy type 1 (DM1) is an inherited disease characterized by the inability to relax contracted muscles.  Affected individuals carry large CTG expansions that are toxic when transcribed.  One possible treatment approach is to reduce or eliminate transcription of CTG repeats.  Actinomycin D (ActD) is a potent transcription inhibitor and FDA-approved chemotherapeutic that binds GC-rich DNA with high affinity.  Here, we report that ActD decreased CUG transcript levels in a dose-dependent manner in DM1 cell and mouse models at significantly lower concns. (nanomolar) compared to its use as a general transcription inhibitor or chemotherapeutic.  ActD also significantly reversed DM1-assocd. splicing defects in a DM1 mouse model, and did so within the currently approved human treatment range.  RNA-seq analyses showed that low concns. of ActD did not globally inhibit transcription in a DM1 mouse model.  These results indicate that transcription inhibition of CTG expansions is a promising treatment approach for DM1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZVoDzjglYN7Vg90H21EOLACvtfcHk0lj3X_h_wF-9Fw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitVSksb3E&md5=f9db4c8f69720951dffda03d58b81fa8</span></div><a href="/servlet/linkout?suffix=cit110&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2015.11.028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2015.11.028%26sid%3Dliteratum%253Aachs%26aulast%3DSiboni%26aufirst%3DR.%2BB.%26aulast%3DNakamori%26aufirst%3DM.%26aulast%3DWagner%26aufirst%3DS.%2BD.%26aulast%3DStruck%26aufirst%3DA.%2BJ.%26aulast%3DCoonrod%26aufirst%3DL.%2BA.%26aulast%3DHarriott%26aufirst%3DS.%2BA.%26aulast%3DCass%26aufirst%3DD.%2BM.%26aulast%3DTanner%26aufirst%3DM.%2BK.%26aulast%3DBerglund%26aufirst%3DJ.%2BA.%26atitle%3DActinomycin%2520D%2520specifically%2520reduces%2520expanded%2520CUG%2520repeat%2520RNA%2520in%2520myotonic%2520mystrophy%2520models%26jtitle%3DCell%2520Rep.%26date%3D2015%26volume%3D13%26spage%3D2386%26epage%3D2394%26doi%3D10.1016%2Fj.celrep.2015.11.028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref111"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref111'); return false;" data-citation="" class="refNumLink">111</a></strong><div class="NLM_citation" id="cit111"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sletten, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertozzi, C. R.</span></span> <span> </span><span class="NLM_article-title">Bioorthogonal chemistry: Fishing for selectivity in a sea of functionality</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">6974</span>â <span class="NLM_lpage">6998</span>, <span class="refDoi">Â DOI: 10.1002/anie.200900942</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref111/cit111&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1002%2Fanie.200900942" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref111/cit111&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtV2ltrvK" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2009&pages=6974-6998&author=E.+M.+Slettenauthor=C.+R.+Bertozzi&title=Bioorthogonal+chemistry%3A+Fishing+for+selectivity+in+a+sea+of+functionality&doi=10.1002%2Fanie.200900942"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit111R"><div class="casContent"><span class="casTitleNuber">111</span><div class="casTitle"><span class="NLM_cas:atitle">Bioorthogonal Chemistry: Fishing for Selectivity in a Sea of Functionality</span></div><div class="casAuthors">Sletten, Ellen M.; Bertozzi, Carolyn R.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">38</span>),
    <span class="NLM_cas:pages">6974-6998</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  The study of biomols. in their native environments is a challenging task because of the vast complexity of cellular systems.  Technologies developed in the last few years for the selective modification of biol. species in living systems have yielded new insights into cellular processes.  Key to these new techniques are bioorthogonal chem. reactions, whose components must react rapidly and selectively with each other under physiol. conditions in the presence of the plethora of functionality necessary to sustain life.  Herein the authors describe the bioorthogonal chem. reactions developed to date and how they can be used to study biomols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJibJXpsfxW7Vg90H21EOLACvtfcHk0lh1z9VulRrXVw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtV2ltrvK&md5=993199cb4deabc186848bae6fe81f605</span></div><a href="/servlet/linkout?suffix=cit111&amp;dbid=16384&amp;doi=10.1002%2Fanie.200900942&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.200900942%26sid%3Dliteratum%253Aachs%26aulast%3DSletten%26aufirst%3DE.%2BM.%26aulast%3DBertozzi%26aufirst%3DC.%2BR.%26atitle%3DBioorthogonal%2520chemistry%253A%2520Fishing%2520for%2520selectivity%2520in%2520a%2520sea%2520of%2520functionality%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2009%26volume%3D48%26spage%3D6974%26epage%3D6998%26doi%3D10.1002%2Fanie.200900942" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref112"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref112'); return false;" data-citation="" class="refNumLink">112</a></strong><div class="NLM_citation" id="cit112"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Å eÄkute, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Devaraj, N. K.</span></span> <span> </span><span class="NLM_article-title">Live-cell imaging of cyclopropene tags with fluorogenic tetrazine cycloadditions</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">7476</span>â <span class="NLM_lpage">7479</span>, <span class="refDoi">Â DOI: 10.1002/anie.201202122</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref112/cit112&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1002%2Fanie.201202122" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref112/cit112&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC38Xos1egs74%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2012&pages=7476-7479&author=J.+Yangauthor=J.+%C5%A0e%C4%8Dkuteauthor=C.+M.+Coleauthor=N.+K.+Devaraj&title=Live-cell+imaging+of+cyclopropene+tags+with+fluorogenic+tetrazine+cycloadditions&doi=10.1002%2Fanie.201202122"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit112R"><div class="casContent"><span class="casTitleNuber">112</span><div class="casTitle"><span class="NLM_cas:atitle">Live-Cell Imaging of Cyclopropene Tags with Fluorogenic Tetrazine Cycloadditions</span></div><div class="casAuthors">Yang, Jun; Seckute, Jolita; Cole, Christian M.; Devaraj, Neal K.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">30</span>),
    <span class="NLM_cas:pages">7476-7479, S7476/1-S7476/26</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The authors demonstrated the applicability of methylcyclopropene tags as dienophiles for reaction with fluorogenic tetrazines.  Through systematic synthetic modifications, the stability, size, and reactivity of the cyclopropene scaffold were optimized.  Methylcyclopropene derivs. were developed that react rapidly with tetrazines while retaining their aq. stability and small size.  These cyclopropene handles elicited fluorescent responses from quenched tetrazine dyes and were suitable for cellular imaging applications, which was demonstrated by imaging cyclopropene phospholipids distributed in live human breast cancer cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpb3dQUpTbfPrVg90H21EOLACvtfcHk0lh1z9VulRrXVw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xos1egs74%253D&md5=c60edddde8754df584b0205368be1e5c</span></div><a href="/servlet/linkout?suffix=cit112&amp;dbid=16384&amp;doi=10.1002%2Fanie.201202122&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201202122%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DJ.%26aulast%3D%25C5%25A0e%25C4%258Dkute%26aufirst%3DJ.%26aulast%3DCole%26aufirst%3DC.%2BM.%26aulast%3DDevaraj%26aufirst%3DN.%2BK.%26atitle%3DLive-cell%2520imaging%2520of%2520cyclopropene%2520tags%2520with%2520fluorogenic%2520tetrazine%2520cycloadditions%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2012%26volume%3D51%26spage%3D7476%26epage%3D7479%26doi%3D10.1002%2Fanie.201202122" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref113"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref113'); return false;" data-citation="" class="refNumLink">113</a></strong><div class="NLM_citation" id="cit113"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Angelbello, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span> <span> </span><span class="NLM_article-title">A toxic RNA templates the synthesis of Its own fluorogenic inhibitor by using a bio-orthogonal tetrazine ligation in cells and tissues</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">1820</span>â <span class="NLM_lpage">1825</span>, <span class="refDoi">Â DOI: 10.1021/acschembio.0c00417</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.0c00417" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref113/cit113&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtFOnsL7I" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2020&pages=1820-1825&author=A.+J.+Angelbelloauthor=M.+D.+Disney&title=A+toxic+RNA+templates+the+synthesis+of+Its+own+fluorogenic+inhibitor+by+using+a+bio-orthogonal+tetrazine+ligation+in+cells+and+tissues&doi=10.1021%2Facschembio.0c00417"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit113R"><div class="casContent"><span class="casTitleNuber">113</span><div class="casTitle"><span class="NLM_cas:atitle">A Toxic RNA Templates the Synthesis of Its Own Fluorogenic Inhibitor by Using a Bio-orthogonal Tetrazine Ligation in Cells and Tissues</span></div><div class="casAuthors">Angelbello, Alicia J.; Disney, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1820-1825</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Expanded RNA repeats cause more than 30 incurable diseases.  One approach to mitigate their toxicity is by using small mols. that assemble into potent, oligomeric species upon binding to the disease-causing RNA in cells.  Herein, the authors show that the expanded repeat [r(CUG)exp] that causes myotonic dystrophy type 1 (DM1) catalyzes the in situ synthesis of its own inhibitor using an RNA-templated tetrazine ligation in DM1 patient-derived cells.  The compd. synthesized on-site improved DM1-assocd. defects at picomolar concns., enhancing potency by 10 000-fold, compared to its parent compds. that cannot undergo oligomerization.  A fluorogenic reaction is also described where r(CUG)exp templates the synthesis of its own imaging probe to enable visualization of the repeat in its native context in live cells and muscle tissue.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4aSzV_OLQLbVg90H21EOLACvtfcHk0lh1z9VulRrXVw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtFOnsL7I&md5=2173daafb0e4280bc0f5e9bfb0222d65</span></div><a href="/servlet/linkout?suffix=cit113&amp;dbid=16384&amp;doi=10.1021%2Facschembio.0c00417&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.0c00417%26sid%3Dliteratum%253Aachs%26aulast%3DAngelbello%26aufirst%3DA.%2BJ.%26aulast%3DDisney%26aufirst%3DM.%2BD.%26atitle%3DA%2520toxic%2520RNA%2520templates%2520the%2520synthesis%2520of%2520Its%2520own%2520fluorogenic%2520inhibitor%2520by%2520using%2520a%2520bio-orthogonal%2520tetrazine%2520ligation%2520in%2520cells%2520and%2520tissues%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2020%26volume%3D15%26spage%3D1820%26epage%3D1825%26doi%3D10.1021%2Facschembio.0c00417" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref114"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref114'); return false;" data-citation="" class="refNumLink">114</a></strong><div class="NLM_citation" id="cit114"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rzuczek, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colgan, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cameron, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furling, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yasuda, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span> <span> </span><span class="NLM_article-title">Precise small-molecule recognition of a toxic CUG RNA repeat expansion</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">188</span>â <span class="NLM_lpage">193</span>, <span class="refDoi">Â DOI: 10.1038/nchembio.2251</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref114/cit114&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1038%2Fnchembio.2251" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref114/cit114&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=27941760" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref114/cit114&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC28XitVSrurbM" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&pages=188-193&author=S.+G.+Rzuczekauthor=L.+A.+Colganauthor=Y.+Nakaiauthor=M.+D.+Cameronauthor=D.+Furlingauthor=R.+Yasudaauthor=M.+D.+Disney&title=Precise+small-molecule+recognition+of+a+toxic+CUG+RNA+repeat+expansion&doi=10.1038%2Fnchembio.2251"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit114R"><div class="casContent"><span class="casTitleNuber">114</span><div class="casTitle"><span class="NLM_cas:atitle">Precise small-molecule recognition of a toxic CUG RNA repeat expansion</span></div><div class="casAuthors">Rzuczek, Suzanne G.; Colgan, Lesley A.; Nakai, Yoshio; Cameron, Michael D.; Furling, Denis; Yasuda, Ryohei; Disney, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">188-193</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Excluding the ribosome and riboswitches, developing small mols. that selectively target RNA is a longstanding problem in chem. biol.  A typical cellular RNA is difficult to target because it has little tertiary, but abundant secondary structure.  We designed allele-selective compds. that target such an RNA, the toxic noncoding repeat expansion (r(CUG)exp) that causes myotonic dystrophy type 1 (DM1).  We developed several strategies to generate allele-selective small mols., including non-covalent binding, covalent binding, cleavage and on-site probe synthesis.  Covalent binding and cleavage enabled target profiling in cells derived from individuals with DM1, showing precise recognition of r(CUG)exp.  In the on-site probe synthesis approach, small mols. bound adjacent sites in r(CUG)exp and reacted to afford picomolar inhibitors via a proximity-based click reaction only in DM1-affected cells.  We expanded this approach to image r(CUG)exp in its natural context.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrC5kTu0Wkj9bVg90H21EOLACvtfcHk0lh1z9VulRrXVw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitVSrurbM&md5=bb30d79fe6e8d513a4956d13ee20f69c</span></div><a href="/servlet/linkout?suffix=cit114&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2251&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2251%26sid%3Dliteratum%253Aachs%26aulast%3DRzuczek%26aufirst%3DS.%2BG.%26aulast%3DColgan%26aufirst%3DL.%2BA.%26aulast%3DNakai%26aufirst%3DY.%26aulast%3DCameron%26aufirst%3DM.%2BD.%26aulast%3DFurling%26aufirst%3DD.%26aulast%3DYasuda%26aufirst%3DR.%26aulast%3DDisney%26aufirst%3DM.%2BD.%26atitle%3DPrecise%2520small-molecule%2520recognition%2520of%2520a%2520toxic%2520CUG%2520RNA%2520repeat%2520expansion%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2017%26volume%3D13%26spage%3D188%26epage%3D193%26doi%3D10.1038%2Fnchembio.2251" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref115"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref115'); return false;" data-citation="" class="refNumLink">115</a></strong><div class="NLM_citation" id="cit115"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liquori, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ricker, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moseley, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobsen, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kress, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naylor, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Day, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ranum, L. P.</span></span> <span> </span><span class="NLM_article-title">Myotonic dystrophy type 2 caused by a CCTG expansion in intron 1 of ZNF9</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>293</i></span>,  <span class="NLM_fpage">864</span>â <span class="NLM_lpage">867</span>, <span class="refDoi">Â DOI: 10.1126/science.1062125</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref115/cit115&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1126%2Fscience.1062125" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref115/cit115&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=11486088" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref115/cit115&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BD3MXlvVKktbc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=293&publication_year=2001&pages=864-867&author=C.+L.+Liquoriauthor=K.+Rickerauthor=M.+L.+Moseleyauthor=J.+F.+Jacobsenauthor=W.+Kressauthor=S.+L.+Naylorauthor=J.+W.+Dayauthor=L.+P.+Ranum&title=Myotonic+dystrophy+type+2+caused+by+a+CCTG+expansion+in+intron+1+of+ZNF9&doi=10.1126%2Fscience.1062125"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit115R"><div class="casContent"><span class="casTitleNuber">115</span><div class="casTitle"><span class="NLM_cas:atitle">Myotonic dystrophy type 2 caused by a CCTG expansion in intron 1 of ZNF9</span></div><div class="casAuthors">Liquori, Christina L.; Ricker, Kenneth; Moseley, Melinda L.; Jacobsen, Jennifer F.; Kress, Wolfram; Naylor, Susan L.; Day, John W.; Ranum, Laura P. W.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">293</span>
        (<span class="NLM_cas:issue">5531</span>),
    <span class="NLM_cas:pages">864-867</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Myotonic dystrophy (DM), the most common form of muscular dystrophy in adults, can be caused by a mutation on either chromosome 19q13 (DM1) or 3q21 (DMZ/PROMM).  DM1 is caused by a CTG expansion in the 3' untranslated region of the dystrophia myotonica-protein kinase gene (DMPK).  Several mechanisms have been invoked to explain how this mutation, which does not alter the protein-coding portion of a gene, causes the specific constellation of clin. features characteristic of DM.  We now report that DMZ is caused by a CCTG expansion (mean â¼5000 repeats) located in intron 1 of the zinc finger protein 9 (ZNF9) gene.  Parallels between these mutations indicate that microsatellite expansions in RNA can be pathogenic and cause the multisystemic features of DM1 and DM2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoeKIAHv1r8_bVg90H21EOLACvtfcHk0lhrWrKioTUg5w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXlvVKktbc%253D&md5=812d852f480461b722162114fcb906f2</span></div><a href="/servlet/linkout?suffix=cit115&amp;dbid=16384&amp;doi=10.1126%2Fscience.1062125&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1062125%26sid%3Dliteratum%253Aachs%26aulast%3DLiquori%26aufirst%3DC.%2BL.%26aulast%3DRicker%26aufirst%3DK.%26aulast%3DMoseley%26aufirst%3DM.%2BL.%26aulast%3DJacobsen%26aufirst%3DJ.%2BF.%26aulast%3DKress%26aufirst%3DW.%26aulast%3DNaylor%26aufirst%3DS.%2BL.%26aulast%3DDay%26aufirst%3DJ.%2BW.%26aulast%3DRanum%26aufirst%3DL.%2BP.%26atitle%3DMyotonic%2520dystrophy%2520type%25202%2520caused%2520by%2520a%2520CCTG%2520expansion%2520in%2520intron%25201%2520of%2520ZNF9%26jtitle%3DScience%26date%3D2001%26volume%3D293%26spage%3D864%26epage%3D867%26doi%3D10.1126%2Fscience.1062125" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref116"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref116'); return false;" data-citation="" class="refNumLink">116</a></strong><div class="NLM_citation" id="cit116"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rzuczek, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span> <span> </span><span class="NLM_article-title">A toxic RNA catalyzes the <i>in cellulo</i> synthesis of its own inhibitor</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">10956</span>â <span class="NLM_lpage">10959</span>, <span class="refDoi">Â DOI: 10.1002/anie.201406465</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref116/cit116&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1002%2Fanie.201406465" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref116/cit116&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVWqtbzE" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2014&pages=10956-10959&author=S.+G.+Rzuczekauthor=H.+Parkauthor=M.+D.+Disney&title=A+toxic+RNA+catalyzes+the+in+cellulo+synthesis+of+its+own+inhibitor&doi=10.1002%2Fanie.201406465"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit116R"><div class="casContent"><span class="casTitleNuber">116</span><div class="casTitle"><span class="NLM_cas:atitle">A Toxic RNA Catalyzes the In Cellulo Synthesis of Its Own Inhibitor</span></div><div class="casAuthors">Rzuczek, Suzanne G.; Park, HaJeung; Disney, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">41</span>),
    <span class="NLM_cas:pages">10956-10959</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Potent modulators of RNA function can be assembled in cellulo by using the cell as a reaction vessel and a disease-causing RNA as a catalyst.  When designing small mol. effectors of function, a balance between permeability and potency must be struck.  Low mol. wt. compds. are more permeable, whereas higher mol. wt. compds. are more potent.  The advantages of both types of compds. could be synergized if low mol. wt. mols. could be transformed into potent, multivalent ligands by a reaction that is catalyzed by binding to a target in cells expressing a genetic defect.  It was shown that this approach is indeed viable in cellulo.  Small mol. modules with precisely positioned alkyne and azide moieties bind adjacent internal loops in r(CCUG)exp, the causative agent of myotonic dystrophy type 2 (DM2), and are transformed into oligomeric, potent inhibitors of DM2 RNA dysfunction by a Huisgen 1,3-dipolar cycloaddn. (HDCR) reaction, a variant of click chem.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqOoWZFnjUHn7Vg90H21EOLACvtfcHk0lhrWrKioTUg5w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVWqtbzE&md5=a5fc7e597bed882f6bb76bb99185420b</span></div><a href="/servlet/linkout?suffix=cit116&amp;dbid=16384&amp;doi=10.1002%2Fanie.201406465&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201406465%26sid%3Dliteratum%253Aachs%26aulast%3DRzuczek%26aufirst%3DS.%2BG.%26aulast%3DPark%26aufirst%3DH.%26aulast%3DDisney%26aufirst%3DM.%2BD.%26atitle%3DA%2520toxic%2520RNA%2520catalyzes%2520the%2520in%2520cellulo%2520synthesis%2520of%2520its%2520own%2520inhibitor%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2014%26volume%3D53%26spage%3D10956%26epage%3D10959%26doi%3D10.1002%2Fanie.201406465" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref117"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref117'); return false;" data-citation="" class="refNumLink">117</a></strong><div class="NLM_citation" id="cit117"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ashizawa, T.</span></span> <span> </span><span class="NLM_article-title">Recent progress in spinocerebellar ataxia type-10 (SCA10)</span>. <i>Cerebellum</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">37</span>â <span class="NLM_lpage">42</span>, <span class="refDoi">Â DOI: 10.1080/14734220510007897</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref117/cit117&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1080%2F14734220510007897" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref117/cit117&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=15895557" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref117/cit117&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BD2MXkvVemur8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2005&pages=37-42&author=X.+Linauthor=T.+Ashizawa&title=Recent+progress+in+spinocerebellar+ataxia+type-10+%28SCA10%29&doi=10.1080%2F14734220510007897"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit117R"><div class="casContent"><span class="casTitleNuber">117</span><div class="casTitle"><span class="NLM_cas:atitle">Recent progress in spinocerebellar ataxia type-10 (SCA10)</span></div><div class="casAuthors">Lin, Xi; Ashizawa, Tetsuo</div><div class="citationInfo"><span class="NLM_cas:title">Cerebellum</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">37-42</span>CODEN:
                <span class="NLM_cas:coden">CERECF</span>;
        ISSN:<span class="NLM_cas:issn">1473-4222</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">A review.  Spinocerebellar ataxia type 10 (SCA10) is a dominantly inherited ataxia caused by expansion of ATTCT pentanucleotide repeat in intron 9 of a novel gene, E46L, on chromosome 22q13.3.  SCA10 is a complex neurodegenerative condition.  Initial studies characterized SCA10 as pure cerebellar ataxia assocd. with seizures.  Recent identification of new SCA10 families revealed more diverse phenotypes, including polyneuropathy, pyramidal signs, cognitive and neuropsychiatric impairment.  Moreover, several families manifest with ataxia without seizures.  Thus a complete clin. spectrum is emerging.  Progress has also been made in understanding the mol. and genetic mechanisms of pathogenesis.  The length of expanded ATTCT repeats is variable in different tissues and highly unstable during paternal transmission, revealing complex genetic and pathogenetic processes.  Under torsional stress, ATTCT repeats form unpaired DNA structure and may serve as an erroneous DNA replication origin, potentially contributing to repeat instability and aberrant cell cycle entry.  E46L is a cytoplasmic protein with unknown function.  Reduced expression of E46L in primary neuronal cultures from cerebellum and cortex by small interfering RNAs (siRNAs) caused increased apoptosis, raising the possibility that reduced expression of E46L might also play an important role in SCA10 pathogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGppTtHjEWa1irVg90H21EOLACvtfcHk0lhEuE7Rvtv9Qw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXkvVemur8%253D&md5=563e6b560bddbd9b9ba34abc614d8084</span></div><a href="/servlet/linkout?suffix=cit117&amp;dbid=16384&amp;doi=10.1080%2F14734220510007897&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F14734220510007897%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DX.%26aulast%3DAshizawa%26aufirst%3DT.%26atitle%3DRecent%2520progress%2520in%2520spinocerebellar%2520ataxia%2520type-10%2520%2528SCA10%2529%26jtitle%3DCerebellum%26date%3D2005%26volume%3D4%26spage%3D37%26epage%3D42%26doi%3D10.1080%2F14734220510007897" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref118"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref118'); return false;" data-citation="" class="refNumLink">118</a></strong><div class="NLM_citation" id="cit118"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, W. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Southern, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sarkar, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span> <span> </span><span class="NLM_article-title">Design of a bioactive small molecule that targets r(AUUCU) repeats in spinocerebellar ataxia 10</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">11647</span>, <span class="refDoi">Â DOI: 10.1038/ncomms11647</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref118/cit118&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1038%2Fncomms11647" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref118/cit118&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=27248057" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref118/cit118&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC28XptVykt7w%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=11647&author=W.+Y.+Yangauthor=R.+Gaoauthor=M.+Southernauthor=P.+S.+Sarkarauthor=M.+D.+Disney&title=Design+of+a+bioactive+small+molecule+that+targets+r%28AUUCU%29+repeats+in+spinocerebellar+ataxia+10&doi=10.1038%2Fncomms11647"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit118R"><div class="casContent"><span class="casTitleNuber">118</span><div class="casTitle"><span class="NLM_cas:atitle">Design of a bioactive small molecule that targets r(AUUCU) repeats in spinocerebellar ataxia 10</span></div><div class="casAuthors">Yang, Wang-Yong; Gao, Rui; Southern, Mark; Sarkar, Partha S.; Disney, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">11647</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">RNA is an important target for chem. probes of function and lead therapeutics; however, it is difficult to target with small mols.  One approach to tackle this problem is to identify compds. that target RNA structures and utilize them to multivalently target RNA.  Here we show that small mols. can be identified to selectively bind RNA base pairs by probing a library of RNA-focused small mols.  A small mol. that selectively binds AU base pairs informed design of a dimeric compd. (2AU-2) that targets the pathogenic RNA, expanded r(AUUCU) repeats, that causes spinocerebellar ataxia type 10 (SCA10) in patient-derived cells.  Indeed, 2AU-2 (50 nM) ameliorates various aspects of SCA10 pathol. including improvement of mitochondrial dysfunction, reduced activation of caspase 3, and redn. of nuclear foci.  These studies provide a first-in-class chem. probe to study SCA10 RNA toxicity and potentially define broadly applicable compds. targeting RNA AU base pairs in cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGorN7W1vcvMzbVg90H21EOLACvtfcHk0lhEuE7Rvtv9Qw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XptVykt7w%253D&md5=60152483dd13fcaa808a7bfb9ecdcb92</span></div><a href="/servlet/linkout?suffix=cit118&amp;dbid=16384&amp;doi=10.1038%2Fncomms11647&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms11647%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DW.%2BY.%26aulast%3DGao%26aufirst%3DR.%26aulast%3DSouthern%26aufirst%3DM.%26aulast%3DSarkar%26aufirst%3DP.%2BS.%26aulast%3DDisney%26aufirst%3DM.%2BD.%26atitle%3DDesign%2520of%2520a%2520bioactive%2520small%2520molecule%2520that%2520targets%2520r%2528AUUCU%2529%2520repeats%2520in%2520spinocerebellar%2520ataxia%252010%26jtitle%3DNat.%2520Commun.%26date%3D2016%26volume%3D7%26spage%3D11647%26doi%3D10.1038%2Fncomms11647" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref119"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref119'); return false;" data-citation="" class="refNumLink">119</a></strong><div class="NLM_citation" id="cit119"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bannwarth, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gatignol, A.</span></span> <span> </span><span class="NLM_article-title">HIV-1 TAR RNA: The target of molecular interactions between the virus and its host</span>. <i>Curr. HIV Res.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">61</span>â <span class="NLM_lpage">71</span>, <span class="refDoi">Â DOI: 10.2174/1570162052772924</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref119/cit119&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.2174%2F1570162052772924" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref119/cit119&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=15638724" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref119/cit119&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhslChsA%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2005&pages=61-71&author=S.+Bannwarthauthor=A.+Gatignol&title=HIV-1+TAR+RNA%3A+The+target+of+molecular+interactions+between+the+virus+and+its+host&doi=10.2174%2F1570162052772924"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit119R"><div class="casContent"><span class="casTitleNuber">119</span><div class="casTitle"><span class="NLM_cas:atitle">HIV-1 TAR RNA: The target of molecular interactions between the virus and its host</span></div><div class="casAuthors">Bannwarth, Sylvie; Gatignol, Anne</div><div class="citationInfo"><span class="NLM_cas:title">Current HIV Research</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">61-71</span>CODEN:
                <span class="NLM_cas:coden">CHRUBF</span>;
        ISSN:<span class="NLM_cas:issn">1570-162X</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  HIV-1 TAR RNA is the binding site of the viral protein Tat, the trans-activator of the HIV-1 LTR.  It is present at the 5' end of all HIV-1 spliced and unspliced mRNAs in the nucleus as well as in the cytoplasm.  It has a highly folded stem-bulge-loop structure, which also binds cellular proteins to form ribonucleoprotein complexes.  The Tat-Cyclin T1-CDK9 complex is the main component in the trans-activation of HIV-1 and its affinity for TAR is regulated through Tat acetylation by histone acetyl transferases.  Recent studies show that this complex is able to recruit other cellular partners to mediate efficient transcriptional elongation.  TRBP, PKR and La bind directly to the TAR RNA structure and influence translation of HIV-1 in either pos. or neg. manners.  Some mutations in TAR RNA severely impair HIV-1 trans-activation, translation and viral prodn., showing its functional importance.  The overexpression or suppression of several TAR RNA-binding proteins has a strong impact on viral replication pointing out their major role in the viral life cycle.  TAR RNA has been the target of drug development to inhibit viral replication.  Recent data using small mols. or RNA-based technologies show that acting on the TAR RNA or on its viral and cellular binding factors effectively decreases virion prodn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVzliRpSqkgLVg90H21EOLACvtfcHk0lhEuE7Rvtv9Qw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhslChsA%253D%253D&md5=f6fdda283b3e8da8dca021cff8601d58</span></div><a href="/servlet/linkout?suffix=cit119&amp;dbid=16384&amp;doi=10.2174%2F1570162052772924&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1570162052772924%26sid%3Dliteratum%253Aachs%26aulast%3DBannwarth%26aufirst%3DS.%26aulast%3DGatignol%26aufirst%3DA.%26atitle%3DHIV-1%2520TAR%2520RNA%253A%2520The%2520target%2520of%2520molecular%2520interactions%2520between%2520the%2520virus%2520and%2520its%2520host%26jtitle%3DCurr.%2520HIV%2520Res.%26date%3D2005%26volume%3D3%26spage%3D61%26epage%3D71%26doi%3D10.2174%2F1570162052772924" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref120"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref120'); return false;" data-citation="" class="refNumLink">120</a></strong><div class="NLM_citation" id="cit120"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Joly, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mata, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eldin, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Briant, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fontaine-Vive, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duca, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benhida, R.</span></span> <span> </span><span class="NLM_article-title">Artificial nucleobase-amino acid conjugates: A new class of TAR RNA binding agents</span>. <i>Chem. - Eur. J.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">2071</span>â <span class="NLM_lpage">2079</span>, <span class="refDoi">Â DOI: 10.1002/chem.201303664</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref120/cit120&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1002%2Fchem.201303664" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref120/cit120&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=24431237" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref120/cit120&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC2cXotF2nug%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2014&pages=2071-2079&author=J.+P.+Jolyauthor=G.+Mataauthor=P.+Eldinauthor=L.+Briantauthor=F.+Fontaine-Viveauthor=M.+Ducaauthor=R.+Benhida&title=Artificial+nucleobase-amino+acid+conjugates%3A+A+new+class+of+TAR+RNA+binding+agents&doi=10.1002%2Fchem.201303664"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit120R"><div class="casContent"><span class="casTitleNuber">120</span><div class="casTitle"><span class="NLM_cas:atitle">Artificial Nucleobase-Amino Acid Conjugates: A New Class of TAR RNA Binding Agents</span></div><div class="casAuthors">Joly, Jean-Patrick; Mata, Guillaume; Eldin, Patrick; Briant, Laurence; Fontaine-Vive, Fabien; Duca, Maria; Benhida, Rachid</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry - A European Journal</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2071-2079</span>CODEN:
                <span class="NLM_cas:coden">CEUJED</span>;
        ISSN:<span class="NLM_cas:issn">0947-6539</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The human immunodeficiency virus type-1 (HIV-1) Tat protein [protein TBP-1 Tat-binding protein-1] stimulates transcriptional elongation.  Tat is involved in the transcription machinery by binding to the transactivation response region (TAR) RNA stem-loop structure, which is encoded by the 5' leader sequence found in all HIV-1 mRNA.  Herein, the authors report the rational design, synthesis, and in vitro evaluation of new RNA binding agents that were conceived in order to bind strongly and selectively to the stem-loop structure of TAR RNA and, thus, inhibit the Tat/TAR interaction.  It was demonstrated that the conjugation of modified nucleobases, able to interact specifically with an RNA base pair and various amino acids allows these motifs to bind the target RNA selectively and in a cooperative manner that leads to the inhibition of viral replication in HIV-infected cells.  The cooperative binding properties of the above-mentioned conjugates toward HIV-TAR-RNA was evaluated.  The title compds. thus formed included a nucleobase-amino-acid conjugate (analog) (I) and related substances.  The synthesis of the target compds. was achieved using a nucleobase analog N-[4-(3-aminophenyl)-2-thiazolyl]acetamide and N2-[(1,1-dimethylethoxy)carbonyl]-L-lysine, N2-[(phenylmethoxy)carbonyl]-L-arginine, N-[(1,1-dimethylethoxy)carbonyl]-L-histidine, N-[(1,1-dimethylethoxy)carbonyl]-L-phenylalanine, O-(1,1-dimethylethyl)-N-[(9H-fluoren-9-ylmethoxy)carbonyl]-L-tyrosine, tryptophan, alanine, threonine, glycine derivs., (amino)butanoic acid, (amino)hexanoic acid as starting materials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOZH87veaXkrVg90H21EOLACvtfcHk0ljjOdO_M5-9Mg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXotF2nug%253D%253D&md5=07a48fa5218ab41d3ebce37ccbfa0b87</span></div><a href="/servlet/linkout?suffix=cit120&amp;dbid=16384&amp;doi=10.1002%2Fchem.201303664&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fchem.201303664%26sid%3Dliteratum%253Aachs%26aulast%3DJoly%26aufirst%3DJ.%2BP.%26aulast%3DMata%26aufirst%3DG.%26aulast%3DEldin%26aufirst%3DP.%26aulast%3DBriant%26aufirst%3DL.%26aulast%3DFontaine-Vive%26aufirst%3DF.%26aulast%3DDuca%26aufirst%3DM.%26aulast%3DBenhida%26aufirst%3DR.%26atitle%3DArtificial%2520nucleobase-amino%2520acid%2520conjugates%253A%2520A%2520new%2520class%2520of%2520TAR%2520RNA%2520binding%2520agents%26jtitle%3DChem.%2520-%2520Eur.%2520J.%26date%3D2014%26volume%3D20%26spage%3D2071%26epage%3D2079%26doi%3D10.1002%2Fchem.201303664" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref121"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref121'); return false;" data-citation="" class="refNumLink">121</a></strong><div class="NLM_citation" id="cit121"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, W. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuhrmann, C. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frankel, A. D.</span></span> <span> </span><span class="NLM_article-title">Recognition of nucleic acid bases and base-pairs by hydrogen bonding to amino acid side-chains</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>327</i></span>,  <span class="NLM_fpage">781</span>â <span class="NLM_lpage">796</span>, <span class="refDoi">Â DOI: 10.1016/S0022-2836(03)00091-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref121/cit121&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1016%2FS0022-2836%2803%2900091-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref121/cit121&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=12654263" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref121/cit121&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BD3sXitlWgs7g%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=327&publication_year=2003&pages=781-796&author=A.+C.+Chengauthor=W.+W.+Chenauthor=C.+N.+Fuhrmannauthor=A.+D.+Frankel&title=Recognition+of+nucleic+acid+bases+and+base-pairs+by+hydrogen+bonding+to+amino+acid+side-chains&doi=10.1016%2FS0022-2836%2803%2900091-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit121R"><div class="casContent"><span class="casTitleNuber">121</span><div class="casTitle"><span class="NLM_cas:atitle">Recognition of Nucleic Acid Bases and Base-pairs by Hydrogen Bonding to Amino Acid Side-chains</span></div><div class="casAuthors">Cheng, Alan C.; Chen, William W.; Fuhrmann, Cynthia N.; Frankel, Alan D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">327</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">781-796</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">Sequence-specific protein-nucleic acid recognition is detd., in part, by hydrogen bonding interactions between amino acid side-chains and nucleotide bases.  To examine the repertoire of possible interactions, we have calcd. geometrically plausible arrangements in which amino acids hydrogen bond to unpaired bases, such as those found in RNA bulges and loops, or to the 53 possible RNA base-pairs.  We find 32 possible interactions that involve two or more hydrogen bonds to the six unpaired bases (including protonated A and C), 17 of which have been obsd.  We find 186 "spanning" interactions to base-pairs in which the amino acid hydrogen bonds to both bases, in principle allowing particular base-pairs to be selectively targeted, and nine of these have been obsd.  Four calcd. interactions span the Watson-Crick pairs and 15 span the G:U wobble pair, including two interesting arrangements with three hydrogen bonds to the Arg guanidinum group that have not yet been obsd.  The inherent donor-acceptor arrangements of the bases support many possible interactions to Asn (or Gln) and Ser (or Thr or Tyr), few interactions to Asp (or Glu) even though several already have been obsd., and interactions to U (or T) only if the base is in an unpaired context, as also obsd. in several cases.  This study highlights how complementary arrangements of donors and acceptors can contribute to base-specific recognition of RNA, predicts interactions not yet obsd., and provides tools to analyze proposed contacts or design novel interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7NXoYEVrOmrVg90H21EOLACvtfcHk0ljjOdO_M5-9Mg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXitlWgs7g%253D&md5=4732411ddbe7ebafe7500776655c87f0</span></div><a href="/servlet/linkout?suffix=cit121&amp;dbid=16384&amp;doi=10.1016%2FS0022-2836%2803%2900091-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0022-2836%252803%252900091-3%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DA.%2BC.%26aulast%3DChen%26aufirst%3DW.%2BW.%26aulast%3DFuhrmann%26aufirst%3DC.%2BN.%26aulast%3DFrankel%26aufirst%3DA.%2BD.%26atitle%3DRecognition%2520of%2520nucleic%2520acid%2520bases%2520and%2520base-pairs%2520by%2520hydrogen%2520bonding%2520to%2520amino%2520acid%2520side-chains%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2003%26volume%3D327%26spage%3D781%26epage%3D796%26doi%3D10.1016%2FS0022-2836%2803%2900091-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref122"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref122'); return false;" data-citation="" class="refNumLink">122</a></strong><div class="NLM_citation" id="cit122"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Paul, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dutta, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paul, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dash, J.</span></span> <span> </span><span class="NLM_article-title">Target-directed azide-alkyne cycloaddition for assembling HIV-1 TAR RNA binding ligands</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">12407</span>â <span class="NLM_lpage">12411</span>, <span class="refDoi">Â DOI: 10.1002/anie.202003461</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref122/cit122&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1002%2Fanie.202003461" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref122/cit122&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtVaitb%252FO" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2020&pages=12407-12411&author=R.+Paulauthor=D.+Duttaauthor=R.+Paulauthor=J.+Dash&title=Target-directed+azide-alkyne+cycloaddition+for+assembling+HIV-1+TAR+RNA+binding+ligands&doi=10.1002%2Fanie.202003461"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit122R"><div class="casContent"><span class="casTitleNuber">122</span><div class="casTitle"><span class="NLM_cas:atitle">Target-directed azide-alkyne cycloaddition for assembling HIV-1 TAR RNA binding ligands</span></div><div class="casAuthors">Paul, Rakesh; Dutta, Debasish; Paul, Raj; Dash, Jyotirmayee</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">30</span>),
    <span class="NLM_cas:pages">12407-12411</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The highly conserved HIV-1 transactivation response element (TAR) binds to the trans-activator protein Tat and facilitates viral replication in its latent state.  The inhibition of Tat-TAR interactions by selectively targeting TAR RNA has been used as a strategy to develop potent antiviral agents.  Therefore, HIV-1 TAR RNA represents a paradigmatic system for therapeutic intervention.  Herein, we have employed biotin-tagged TAR RNA to assemble its own ligands from a pool of reactive azide and alkyne building blocks.  To identify the binding sites and selectivity of the ligands, the in situ cycloaddn. has been further performed using control nucleotide (TAR DNA and TAR RNA without bulge) templates.  The hit triazole-linked thiazole peptidomimetic products have been isolated from the biotin-tagged target templates using streptavidin beads.  The major triazole lead generated by the TAR RNA presumably binds in the bulge region, shows specificity for TAR RNA over TAR DNA, and inhibits Tat-TAR interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqygRarHJrIibVg90H21EOLACvtfcHk0liJ_LbGYsOoXw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtVaitb%252FO&md5=6e2a48b9291957e5f0e014e25344d43b</span></div><a href="/servlet/linkout?suffix=cit122&amp;dbid=16384&amp;doi=10.1002%2Fanie.202003461&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.202003461%26sid%3Dliteratum%253Aachs%26aulast%3DPaul%26aufirst%3DR.%26aulast%3DDutta%26aufirst%3DD.%26aulast%3DPaul%26aufirst%3DR.%26aulast%3DDash%26aufirst%3DJ.%26atitle%3DTarget-directed%2520azide-alkyne%2520cycloaddition%2520for%2520assembling%2520HIV-1%2520TAR%2520RNA%2520binding%2520ligands%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2020%26volume%3D59%26spage%3D12407%26epage%3D12411%26doi%3D10.1002%2Fanie.202003461" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref123"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref123'); return false;" data-citation="" class="refNumLink">123</a></strong><div class="NLM_citation" id="cit123"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stelzer, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frank, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kratz, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swanson, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gonzalez-Hernandez, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andricioaei, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Markovitz, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Hashim, H. M.</span></span> <span> </span><span class="NLM_article-title">Discovery of selective bioactive small molecules by targeting an RNA dynamic ensemble</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">553</span>â <span class="NLM_lpage">559</span>, <span class="refDoi">Â DOI: 10.1038/nchembio.596</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref123/cit123&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1038%2Fnchembio.596" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref123/cit123&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=21706033" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref123/cit123&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC3MXotV2htbw%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=553-559&author=A.+C.+Stelzerauthor=A.+T.+Frankauthor=J.+D.+Kratzauthor=M.+D.+Swansonauthor=M.+J.+Gonzalez-Hernandezauthor=J.+Leeauthor=I.+Andricioaeiauthor=D.+M.+Markovitzauthor=H.+M.+Al-Hashim&title=Discovery+of+selective+bioactive+small+molecules+by+targeting+an+RNA+dynamic+ensemble&doi=10.1038%2Fnchembio.596"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit123R"><div class="casContent"><span class="casTitleNuber">123</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of selective bioactive small molecules by targeting an RNA dynamic ensemble</span></div><div class="casAuthors">Stelzer, Andrew C.; Frank, Aaron T.; Kratz, Jeremy D.; Swanson, Michael D.; Gonzalez-Hernandez, Marta J.; Lee, Janghyun; Andricioaei, Ioan; Markovitz, David M.; Al-Hashimi, Hashim M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">553-559</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Current approaches used to identify protein-binding small mols. are not suited for identifying small mols. that can bind emerging RNA drug targets.  By docking small mols. onto an RNA dynamic ensemble constructed by combining NMR spectroscopy and computational mol. dynamics, the authors virtually screened small mols. that target the entire structure landscape of the transactivation response element (TAR) from HIV type 1 (HIV-1).  The authors quant. predict binding energies for small mols. that bind different RNA conformations and report the de novo discovery of six compds. that bind TAR with high affinity and inhibit its interaction with a Tat peptide in vitro (Ki values of 710 nM-169 Î¼M).  One compd. binds HIV-1 TAR with marked selectivity and inhibits Tat-mediated activation of the HIV-1 long terminal repeat by 81% in T-cell lines and HIV replication in an HIV-1 indicator cell line (IC50 â¼23.1 Î¼M).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoQ1jtoA-Z8RrVg90H21EOLACvtfcHk0liJ_LbGYsOoXw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXotV2htbw%253D&md5=cd04f57b1fececc2f4aa678fe3f1f3f2</span></div><a href="/servlet/linkout?suffix=cit123&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.596&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.596%26sid%3Dliteratum%253Aachs%26aulast%3DStelzer%26aufirst%3DA.%2BC.%26aulast%3DFrank%26aufirst%3DA.%2BT.%26aulast%3DKratz%26aufirst%3DJ.%2BD.%26aulast%3DSwanson%26aufirst%3DM.%2BD.%26aulast%3DGonzalez-Hernandez%26aufirst%3DM.%2BJ.%26aulast%3DLee%26aufirst%3DJ.%26aulast%3DAndricioaei%26aufirst%3DI.%26aulast%3DMarkovitz%26aufirst%3DD.%2BM.%26aulast%3DAl-Hashim%26aufirst%3DH.%2BM.%26atitle%3DDiscovery%2520of%2520selective%2520bioactive%2520small%2520molecules%2520by%2520targeting%2520an%2520RNA%2520dynamic%2520ensemble%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2011%26volume%3D7%26spage%3D553%26epage%3D559%26doi%3D10.1038%2Fnchembio.596" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref124"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref124'); return false;" data-citation="" class="refNumLink">124</a></strong><div class="NLM_citation" id="cit124"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Serganov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nudler, E.</span></span> <span> </span><span class="NLM_article-title">A decade of riboswitches</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>152</i></span>,  <span class="NLM_fpage">17</span>â <span class="NLM_lpage">24</span>, <span class="refDoi">Â DOI: 10.1016/j.cell.2012.12.024</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref124/cit124&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1016%2Fj.cell.2012.12.024" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref124/cit124&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=23332744" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref124/cit124&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1Oluro%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=152&publication_year=2013&pages=17-24&author=A.+Serganovauthor=E.+Nudler&title=A+decade+of+riboswitches&doi=10.1016%2Fj.cell.2012.12.024"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit124R"><div class="casContent"><span class="casTitleNuber">124</span><div class="casTitle"><span class="NLM_cas:atitle">A Decade of Riboswitches</span></div><div class="casAuthors">Serganov, Alexander; Nudler, Evgeny</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">152</span>
        (<span class="NLM_cas:issue">1-2</span>),
    <span class="NLM_cas:pages">17-24</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Riboswitches were discovered in 2002 in bacteria as RNA-based intracellular sensors of vitamin derivs.  During the last decade, naturally occurring RNA sensor elements have been found to bind a range of small metabolites and ions and to exert regulatory control of transcription, translation, splicing, and RNA stability.  Extensive biochem., structural, and genetic studies have established the basic principles underpinning riboswitch function in all three kingdoms of life with implications for developing antibiotics, designing new mol. sensors, and integrating riboswitches into synthetic circuits.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2Ru8nOcVcrbVg90H21EOLACvtfcHk0li6GBC-jDaj-g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1Oluro%253D&md5=c8bb59e0772d4ec0b9dbaf163d73ae1b</span></div><a href="/servlet/linkout?suffix=cit124&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2012.12.024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2012.12.024%26sid%3Dliteratum%253Aachs%26aulast%3DSerganov%26aufirst%3DA.%26aulast%3DNudler%26aufirst%3DE.%26atitle%3DA%2520decade%2520of%2520riboswitches%26jtitle%3DCell%26date%3D2013%26volume%3D152%26spage%3D17%26epage%3D24%26doi%3D10.1016%2Fj.cell.2012.12.024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref125"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref125'); return false;" data-citation="" class="refNumLink">125</a></strong><div class="NLM_citation" id="cit125"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gelfand, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mironov, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jomantas, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozlov, Y. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perumov, D. A.</span></span> <span> </span><span class="NLM_article-title">A conserved RNA structure element involved in the regulation of bacterial riboflavin synthesis genes</span>. <i>Trends Genet.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">439</span>â <span class="NLM_lpage">442</span>, <span class="refDoi">Â DOI: 10.1016/S0168-9525(99)01856-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref125/cit125&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1016%2FS0168-9525%2899%2901856-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref125/cit125&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10529804" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref125/cit125&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADyaK1MXmvVKhsb8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=1999&pages=439-442&author=M.+S.+Gelfandauthor=A.+A.+Mironovauthor=J.+Jomantasauthor=Y.+I.+Kozlovauthor=D.+A.+Perumov&title=A+conserved+RNA+structure+element+involved+in+the+regulation+of+bacterial+riboflavin+synthesis+genes&doi=10.1016%2FS0168-9525%2899%2901856-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit125R"><div class="casContent"><span class="casTitleNuber">125</span><div class="casTitle"><span class="NLM_cas:atitle">A conserved RNA structure element involved in the regulation of bacterial riboflavin synthesis genes</span></div><div class="casAuthors">Gelfand, M. S.; Mironov, A. A.; Jomantas, J.; Kozlov, Y. I.; Perumov, D. A.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Genetics</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">439-442</span>CODEN:
                <span class="NLM_cas:coden">TRGEE2</span>;
        ISSN:<span class="NLM_cas:issn">0168-9525</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">A review, with 19 refs., on the RFN element found in riboflavin biosynthesis operons from Gram-pos. bacteria, Thermotogales, Deinococcus, and Proteobacteria.  Alignment of the RFN element sequences and the conserved structure of the element are presented.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq95lZv7DUy5bVg90H21EOLACvtfcHk0li6GBC-jDaj-g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXmvVKhsb8%253D&md5=9b3052a294ce279f9b3f6544b43223c4</span></div><a href="/servlet/linkout?suffix=cit125&amp;dbid=16384&amp;doi=10.1016%2FS0168-9525%2899%2901856-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0168-9525%252899%252901856-9%26sid%3Dliteratum%253Aachs%26aulast%3DGelfand%26aufirst%3DM.%2BS.%26aulast%3DMironov%26aufirst%3DA.%2BA.%26aulast%3DJomantas%26aufirst%3DJ.%26aulast%3DKozlov%26aufirst%3DY.%2BI.%26aulast%3DPerumov%26aufirst%3DD.%2BA.%26atitle%3DA%2520conserved%2520RNA%2520structure%2520element%2520involved%2520in%2520the%2520regulation%2520of%2520bacterial%2520riboflavin%2520synthesis%2520genes%26jtitle%3DTrends%2520Genet.%26date%3D1999%26volume%3D15%26spage%3D439%26epage%3D442%26doi%3D10.1016%2FS0168-9525%2899%2901856-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref126"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref126'); return false;" data-citation="" class="refNumLink">126</a></strong><div class="NLM_citation" id="cit126"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Howe, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischmann, T. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balibar, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galgoci, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malinverni, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mayhood, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villafania, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nahvi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murgolo, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbieri, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mann, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carr, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xia, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zuck, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riley, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Painter, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sherborne, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jesus, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plotkin, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rindgen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cummings, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garlisi, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheth, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gill, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roemer, T.</span></span> <span> </span><span class="NLM_article-title">Selective small-molecule inhibition of an RNA structural element</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>526</i></span>,  <span class="NLM_fpage">672</span>â <span class="NLM_lpage">677</span>, <span class="refDoi">Â DOI: 10.1038/nature15542</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref126/cit126&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1038%2Fnature15542" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref126/cit126&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=26416753" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref126/cit126&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslCnu7%252FL" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=526&publication_year=2015&pages=672-677&author=J.+A.+Howeauthor=H.+Wangauthor=T.+O.+Fischmannauthor=C.+J.+Balibarauthor=L.+Xiaoauthor=A.+M.+Galgociauthor=J.+C.+Malinverniauthor=T.+Mayhoodauthor=A.+Villafaniaauthor=A.+Nahviauthor=N.+Murgoloauthor=C.+M.+Barbieriauthor=P.+A.+Mannauthor=D.+Carrauthor=E.+Xiaauthor=P.+Zuckauthor=D.+Rileyauthor=R.+E.+Painterauthor=S.+S.+Walkerauthor=B.+Sherborneauthor=R.+D.+Jesusauthor=W.+Panauthor=M.+A.+Plotkinauthor=J.+Wuauthor=D.+Rindgenauthor=J.+Cummingsauthor=C.+G.+Garlisiauthor=R.+Zhangauthor=P.+R.+Shethauthor=C.+J.+Gillauthor=H.+Tangauthor=T.+Roemer&title=Selective+small-molecule+inhibition+of+an+RNA+structural+element&doi=10.1038%2Fnature15542"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit126R"><div class="casContent"><span class="casTitleNuber">126</span><div class="casTitle"><span class="NLM_cas:atitle">Selective small-molecule inhibition of an RNA structural element</span></div><div class="casAuthors">Howe, John A.; Wang, Hao; Fischmann, Thierry O.; Balibar, Carl J.; Xiao, Li; Galgoci, Andrew M.; Malinverni, Juliana C.; Mayhood, Todd; Villafania, Artjohn; Nahvi, Ali; Murgolo, Nicholas; Barbieri, Christopher M.; Mann, Paul A.; Carr, Donna; Xia, Ellen; Zuck, Paul; Riley, Dan; Painter, Ronald E.; Walker, Scott S.; Sherborne, Brad; de Jesus, Reynalda; Pan, Weidong; Plotkin, Michael A.; Wu, Jin; Rindgen, Diane; Cummings, John; Garlisi, Charles G.; Zhang, Rumin; Sheth, Payal R.; Gill, Charles J.; Tang, Haifeng; Roemer, Terry</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">526</span>
        (<span class="NLM_cas:issue">7575</span>),
    <span class="NLM_cas:pages">672-677</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Riboswitches are non-coding RNA structures located in mRNAs that bind endogenous ligands, such as a specific metabolite or ion, to regulate gene expression.  As such, riboswitches serve as a novel, yet largely unexploited, class of emerging drug targets.  Demonstrating this potential, however, has proven difficult and is restricted to structurally similar antimetabolites and semi-synthetic analogs of their cognate ligand, thus greatly restricting the chem. space and selectivity sought for such inhibitors.  Here we report the discovery and characterization of ribocil, a highly selective chem. modulator of bacterial riboflavin riboswitches, which was identified in a phenotypic screen and acts as a structurally distinct synthetic mimic of the natural ligand, FMN, to repress riboswitch-mediated ribB gene expression and inhibit bacterial cell growth.  Our findings indicate that non-coding RNA structural elements may be more broadly targeted by synthetic small mols. than previously expected.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWptaVLK3njLVg90H21EOLACvtfcHk0ljcb9BJVmp-Eg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslCnu7%252FL&md5=c0f8443348c10f69eb47f05a2ac5e893</span></div><a href="/servlet/linkout?suffix=cit126&amp;dbid=16384&amp;doi=10.1038%2Fnature15542&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature15542%26sid%3Dliteratum%253Aachs%26aulast%3DHowe%26aufirst%3DJ.%2BA.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DFischmann%26aufirst%3DT.%2BO.%26aulast%3DBalibar%26aufirst%3DC.%2BJ.%26aulast%3DXiao%26aufirst%3DL.%26aulast%3DGalgoci%26aufirst%3DA.%2BM.%26aulast%3DMalinverni%26aufirst%3DJ.%2BC.%26aulast%3DMayhood%26aufirst%3DT.%26aulast%3DVillafania%26aufirst%3DA.%26aulast%3DNahvi%26aufirst%3DA.%26aulast%3DMurgolo%26aufirst%3DN.%26aulast%3DBarbieri%26aufirst%3DC.%2BM.%26aulast%3DMann%26aufirst%3DP.%2BA.%26aulast%3DCarr%26aufirst%3DD.%26aulast%3DXia%26aufirst%3DE.%26aulast%3DZuck%26aufirst%3DP.%26aulast%3DRiley%26aufirst%3DD.%26aulast%3DPainter%26aufirst%3DR.%2BE.%26aulast%3DWalker%26aufirst%3DS.%2BS.%26aulast%3DSherborne%26aufirst%3DB.%26aulast%3DJesus%26aufirst%3DR.%2BD.%26aulast%3DPan%26aufirst%3DW.%26aulast%3DPlotkin%26aufirst%3DM.%2BA.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DRindgen%26aufirst%3DD.%26aulast%3DCummings%26aufirst%3DJ.%26aulast%3DGarlisi%26aufirst%3DC.%2BG.%26aulast%3DZhang%26aufirst%3DR.%26aulast%3DSheth%26aufirst%3DP.%2BR.%26aulast%3DGill%26aufirst%3DC.%2BJ.%26aulast%3DTang%26aufirst%3DH.%26aulast%3DRoemer%26aufirst%3DT.%26atitle%3DSelective%2520small-molecule%2520inhibition%2520of%2520an%2520RNA%2520structural%2520element%26jtitle%3DNature%26date%3D2015%26volume%3D526%26spage%3D672%26epage%3D677%26doi%3D10.1038%2Fnature15542" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref127"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref127'); return false;" data-citation="" class="refNumLink">127</a></strong><div class="NLM_citation" id="cit127"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mansoori, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohammadi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davudian, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shirjang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baradaran, B.</span></span> <span> </span><span class="NLM_article-title">The different mechanisms of cancer drug resistance: A brief review</span>. <i>Adv. Pharm. Bull.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">339</span>â <span class="NLM_lpage">348</span>, <span class="refDoi">Â DOI: 10.15171/apb.2017.041</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref127/cit127&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.15171%2Fapb.2017.041" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref127/cit127&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=29071215" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref127/cit127&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC1MXktFCntbk%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&pages=339-348&author=B.+Mansooriauthor=A.+Mohammadiauthor=S.+Davudianauthor=S.+Shirjangauthor=B.+Baradaran&title=The+different+mechanisms+of+cancer+drug+resistance%3A+A+brief+review&doi=10.15171%2Fapb.2017.041"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit127R"><div class="casContent"><span class="casTitleNuber">127</span><div class="casTitle"><span class="NLM_cas:atitle">The different mechanisms of cancer drug resistance: a brief review</span></div><div class="casAuthors">Mansoori, Behzad; Mohammadi, Ali; Davudian, Sadaf; Shirjang, Solmaz; Baradaran, Behzad</div><div class="citationInfo"><span class="NLM_cas:title">Advanced Pharmaceutical Bulletin</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">339-348</span>CODEN:
                <span class="NLM_cas:coden">APBDCL</span>;
        ISSN:<span class="NLM_cas:issn">2251-7308</span>.
    
            (<span class="NLM_cas:orgname">Tabriz University of Medical Sciences</span>)
        </div><div class="casAbstract">Anticancer drugs resistance is a complex process that arises from altering in the drug targets.  Advances in the DNA microarray, proteomics technol. and the development of targeted therapies provide the new strategies to overcome the drug resistance.  Although a design of the new chemotherapy agents is growing quickly, effective chemotherapy agent has not been discovered against the advanced stage of cancer (such as invasion and metastasis).  The cancer cell resistance against the anticancer agents can be due to many factors such as the individual's genetic differences, esp. in tumoral somatic cells.  Also, the cancer drug resistance is acquired, the drug resistance can be occurred by different mechanisms, including multi-drug resistance, cell death inhibiting (apoptosis suppression), altering in the drug metab., epigenetic and drug targets, enhancing DNA repair and gene amplification.  In this review, we outlined the mechanisms of cancer drug resistance and in following, the treatment failures by common chemotherapy agents in the different type of cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5pNO4Qea_HLVg90H21EOLACvtfcHk0lgd6_y_5R1auQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXktFCntbk%253D&md5=889d5efbe4bca0f1c3946f55848a13ee</span></div><a href="/servlet/linkout?suffix=cit127&amp;dbid=16384&amp;doi=10.15171%2Fapb.2017.041&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.15171%252Fapb.2017.041%26sid%3Dliteratum%253Aachs%26aulast%3DMansoori%26aufirst%3DB.%26aulast%3DMohammadi%26aufirst%3DA.%26aulast%3DDavudian%26aufirst%3DS.%26aulast%3DShirjang%26aufirst%3DS.%26aulast%3DBaradaran%26aufirst%3DB.%26atitle%3DThe%2520different%2520mechanisms%2520of%2520cancer%2520drug%2520resistance%253A%2520A%2520brief%2520review%26jtitle%3DAdv.%2520Pharm.%2520Bull.%26date%3D2017%26volume%3D7%26spage%3D339%26epage%3D348%26doi%3D10.15171%2Fapb.2017.041" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref128"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref128'); return false;" data-citation="" class="refNumLink">128</a></strong><div class="NLM_citation" id="cit128"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, P.</span></span> <span> </span><span class="NLM_article-title">Inhibition of RNA-binding proteins with small molecules</span>. <i>Nature Rev. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">441</span>â <span class="NLM_lpage">458</span>, <span class="refDoi">Â DOI: 10.1038/s41570-020-0201-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref128/cit128&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1038%2Fs41570-020-0201-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref128/cit128&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BB3cXitFeru7vN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2020&pages=441-458&author=P.+Wu&title=Inhibition+of+RNA-binding+proteins+with+small+molecules&doi=10.1038%2Fs41570-020-0201-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit128R"><div class="casContent"><span class="casTitleNuber">128</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of RNA-binding proteins with small molecules</span></div><div class="casAuthors">Wu, Peng</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">441-458</span>CODEN:
                <span class="NLM_cas:coden">NRCAF7</span>;
        ISSN:<span class="NLM_cas:issn">2397-3358</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review.  Protein-RNA interactions have crucial roles in various cellular activities, which, when dysregulated, can lead to a range of human diseases.  The identification of small mols. that target the interaction between RNA-binding proteins (RBPs) and RNA is progressing rapidly and represents a novel strategy for the discovery of chem. probes that facilitate understanding of the cellular functions of RBPs and of therapeutic agents with new mechanisms of action.  In this Review, I present a current overview of targeting emerging RBPs using small-mol. inhibitors and recent progress in this burgeoning field.  Small-mol. inhibitors that were reported for three representative emerging classes of RBPs, the microRNA-binding protein LIN28, the single-stranded or double-stranded RNA-binding Toll-like receptors and the CRISPR-assocd. (Cas) proteins, are highlighted from a medicinal-chem. and chem.-biol. perspective.  However, although this field is burgeoning, challenges remain in the discovery and characterization of small-mol. inhibitors of RBPs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq4zPBh0BaIybVg90H21EOLACvtfcHk0lgd6_y_5R1auQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXitFeru7vN&md5=2cb4f106e0b5b8c92f37ef5757f07fed</span></div><a href="/servlet/linkout?suffix=cit128&amp;dbid=16384&amp;doi=10.1038%2Fs41570-020-0201-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41570-020-0201-4%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DP.%26atitle%3DInhibition%2520of%2520RNA-binding%2520proteins%2520with%2520small%2520molecules%26jtitle%3DNature%2520Rev.%2520Chem.%26date%3D2020%26volume%3D4%26spage%3D441%26epage%3D458%26doi%3D10.1038%2Fs41570-020-0201-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62'],'ref63':['cit63'],'ref64':['cit64'],'ref65':['cit65'],'ref66':['cit66'],'ref67':['cit67'],'ref68':['cit68'],'ref69':['cit69'],'ref70':['cit70'],'ref71':['cit71'],'ref72':['cit72'],'ref73':['cit73'],'ref74':['cit74'],'ref75':['cit75'],'ref76':['cit76'],'ref77':['cit77'],'ref78':['cit78'],'ref79':['cit79'],'ref80':['cit80'],'ref81':['cit81'],'ref82':['cit82'],'ref83':['cit83'],'ref84':['cit84'],'ref85':['cit85'],'ref86':['cit86'],'ref87':['cit87'],'ref88':['cit88'],'ref89':['cit89'],'ref90':['cit90'],'ref91':['cit91'],'ref92':['cit92'],'ref93':['cit93'],'ref94':['cit94'],'ref95':['cit95'],'ref96':['cit96'],'ref97':['cit97'],'ref98':['cit98'],'ref99':['cit99'],'ref100':['cit100'],'ref101':['cit101'],'ref102':['cit102'],'ref103':['cit103'],'ref104':['cit104'],'ref105':['cit105'],'ref106':['cit106'],'ref107':['cit107'],'ref108':['cit108'],'ref109':['cit109'],'ref110':['cit110'],'ref111':['cit111'],'ref112':['cit112'],'ref113':['cit113'],'ref114':['cit114'],'ref115':['cit115'],'ref116':['cit116'],'ref117':['cit117'],'ref118':['cit118'],'ref119':['cit119'],'ref120':['cit120'],'ref121':['cit121'],'ref122':['cit122'],'ref123':['cit123'],'ref124':['cit124'],'ref125':['cit125'],'ref126':['cit126'],'ref127':['cit127'],'ref128':['cit128']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article has not yet been cited by other publications.</p></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/medium/jm1c00154_0049.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0049.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00154&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/medium/jm1c00154_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. miRNA biogenesis and putative inhibition of Dicer-mediated cleavage of pre-miRNA by synthetic molecules.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00154&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/medium/jm1c00154_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Carbazole Analogues and Inhibitory Activity of <b>5a,b</b> toward Dicer Processing of pre-miR-21</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00154&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/medium/jm1c00154_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis and miR-21 Maturation-Inhibitory Activity of Azobenzene-Based Compound <b>8</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00154&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/medium/jm1c00154_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of 4-Benzoylamino-<i>N</i>-(prop-2-yn-1-yl)benzamides and Antitumor Activity toward HeLa and U-87 MG Cells</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00154&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/medium/jm1c00154_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis and Antitumor Activity of Oxadiazole-Based miR-21 Inhibitors</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00154&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/medium/jm1c00154_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Synthesis and miR-21 Maturation-Inhibitory Activity of EtherâAmide Compounds</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00154&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch6" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/medium/jm1c00154_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0007.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 6. Synthesis and Dicer-Inhibitory Activity of the Neomycin-Based miR-21 Bifunctional Inhibitor</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00154&amp;id=sch6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch7" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/medium/jm1c00154_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0008.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 7. Synthesis and miR-21 Maturation-Inhibitory Activity of PA-Kanamycins</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00154&amp;id=sch7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch8" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/medium/jm1c00154_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0009.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 8. (a) Synthesis and miR-21 Maturation-Inhibitory Activity of Regorafenib; (b, c) Binding Hotspots (G6, A7, and A13) for Ligand Binding of Regorafenib on the Surface of the miR-21 Pre-element<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00154&amp;id=sch8"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reprinted with permission from ref <a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> [  <cite><i>J. Mol. Graphics Modell.</i></cite> <span class="NLM_year">2017</span>, <em>73</em>, 48â53]. Copyright 2017 Elsevier.</p></p></figure><figure data-id="sch9" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Scheme 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/medium/jm1c00154_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0010.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 9. Synthesis of a Non-oligonucleotide-Containing Ligand to Promote RNase-Mediated Destruction of pre-miR-21 in Cancer Cells</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00154&amp;id=sch9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch10" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Scheme 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/medium/jm1c00154_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0011.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 10. Synthesis and Antitumor Activity of Neomycin-Based Small Molecules (R = Nucleobase Analogue) against Gastric Cancer Cells<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00154&amp;id=sch10"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>These compounds inhibit Dicer-mediated processing of pre-miR-372 and pre-miR-373.</p></p></figure><figure data-id="sch11" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Scheme 11</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/medium/jm1c00154_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0012.jpeg" id="rightTab-gr12" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 11. Synthesis and miR-372/miR-373 Cleavage-Inhibitory Activity of Nucleobase-Aminoglycoside-Based Compounds</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00154&amp;id=sch11"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch12" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Scheme 12</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/medium/jm1c00154_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0013.jpeg" id="rightTab-gr13" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 12. Synthesis and miR-372 Maturation-Inhibitory Activity of Spermine-Amidine Compound <b>86</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00154&amp;id=sch12"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch13" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Scheme 13</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/medium/jm1c00154_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0014.jpeg" id="rightTab-gr14" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 13. Synthesis and miR-29a Maturation-Inhibitory Activity of Xanthone Derivatives</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00154&amp;id=sch13"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch14" data-index="15" class="article__inlineFigure"><h2 class="fig-label">Scheme 14</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/medium/jm1c00154_0015.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0015.jpeg" id="rightTab-gr15" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 14. Synthesis and miR-1 Expression-Inhibitory Activity of 1,4-Naphthalenequinone Analogues</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00154&amp;id=sch14"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch15" data-index="16" class="article__inlineFigure"><h2 class="fig-label">Scheme 15</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/medium/jm1c00154_0016.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0016.jpeg" id="rightTab-gr16" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 15. Synthesis and miR-4644/miR-14 Maturation-Inhibitory Activity of Benzopyranone Derivatives (<i>aza</i>-Flavanones) in MCF-7 Cells</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00154&amp;id=sch15"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch16" data-index="17" class="article__inlineFigure"><h2 class="fig-label">Scheme 16</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/medium/jm1c00154_0017.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0017.jpeg" id="rightTab-gr17" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 16. Synthesis, miR-122 Maturation-Inhibiting or -Activating Activity, and Antiviral or Caspase-3-Increasing Activity of Benzamides, Tetrahydroquinolines, and Isoquinoline-1,3(2<i>H</i>)-diones</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0017.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00154&amp;id=sch16"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch17" data-index="18" class="article__inlineFigure"><h2 class="fig-label">Scheme 17</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/medium/jm1c00154_0018.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0018.jpeg" id="rightTab-gr18" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 17. Synthesis of Kenpaullone Derivatives as Regulators of miR-182 Expression</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0018.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00154&amp;id=sch17"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch18" data-index="19" class="article__inlineFigure"><h2 class="fig-label">Scheme 18</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/medium/jm1c00154_0019.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0019.jpeg" id="rightTab-gr19" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 18. Synthesis of Benzimidazole Derivatives as Inhibitors of Mature miR-18a Expression in Prostate Cancer Cells</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0019.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00154&amp;id=sch18"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch19" data-index="20" class="article__inlineFigure"><h2 class="fig-label">Scheme 19</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/medium/jm1c00154_0020.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0020.jpeg" id="rightTab-gr20" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 19. Synthesis of Resin-Based Peptoid Analogues as Inhibitors of miR-96 Processing in Breast Cancer Cells</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0020.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00154&amp;id=sch19"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch20" data-index="21" class="article__inlineFigure"><h2 class="fig-label">Scheme 20</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/medium/jm1c00154_0021.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0021.jpeg" id="rightTab-gr21" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 20. Synthesis of Oligoadenylate-Based Ligands as Inhibitors of miR-210 in Hypoxic Breast Cancer Cells</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0021.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00154&amp;id=sch20"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch21" data-index="22" class="article__inlineFigure"><h2 class="fig-label">Scheme 21</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/medium/jm1c00154_0022.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0022.jpeg" id="rightTab-gr22" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 21. Synthesis of Dimeric Ligands as Inhibitors of pri-miR-515 in MCF-7 Breast Cancer Cells</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0022.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00154&amp;id=sch21"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch22" data-index="23" class="article__inlineFigure"><h2 class="fig-label">Scheme 22</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/medium/jm1c00154_0023.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0023.jpeg" id="rightTab-gr23" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 22. Synthesis of Enoxacin Derivatives as RNAi Enhancers</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0023.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00154&amp;id=sch22"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch23" data-index="24" class="article__inlineFigure"><h2 class="fig-label">Scheme 23</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/medium/jm1c00154_0024.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0024.jpeg" id="rightTab-gr24" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 23. Synthesis and MALAT1-Binding Activity of Diphenylfuran-Based Derivatives</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0024.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00154&amp;id=sch23"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch24" data-index="25" class="article__inlineFigure"><h2 class="fig-label">Scheme 24</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/medium/jm1c00154_0025.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0025.jpeg" id="rightTab-gr25" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 24. (a) Identification of Two Imidazole-Based Synthetic Small Molecules <b>135</b> and <b>136</b> as Selective MALAT1 ENE Triplex-Binding Ligands; (b) Binding Modes of <b>135</b> and <b>136</b> with MALAT1 ENE Triplex</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0025.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00154&amp;id=sch24"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch25" data-index="26" class="article__inlineFigure"><h2 class="fig-label">Scheme 25</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/medium/jm1c00154_0026.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0026.jpeg" id="rightTab-gr26" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 25. Synthesis and Antitumor Effect of a Nitrogen-Based Heterocycle, <b>141</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0026.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00154&amp;id=sch25"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch26" data-index="27" class="article__inlineFigure"><h2 class="fig-label">Scheme 26</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/medium/jm1c00154_0027.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0027.jpeg" id="rightTab-gr27" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 26. Identification of ASQ <b>142</b> as a Selective Inhibitor of HOTAIR/EZH2 Interaction in Cancer Cells</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0027.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00154&amp;id=sch26"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch27" data-index="28" class="article__inlineFigure"><h2 class="fig-label">Scheme 27</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/medium/jm1c00154_0028.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0028.jpeg" id="rightTab-gr28" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 27. Synthesis and Activity of a Polyaromatic RNA G-Quadruplex Binder, <b>149</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0028.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00154&amp;id=sch27"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch28" data-index="29" class="article__inlineFigure"><h2 class="fig-label">Scheme 28</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/medium/jm1c00154_0029.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0029.jpeg" id="rightTab-gr29" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 28. Synthesis of Naphthyridine-Based Structures and Their Effect on â1PRF Frame-Shifting Efficiency (FE)</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0029.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00154&amp;id=sch28"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch29" data-index="30" class="article__inlineFigure"><h2 class="fig-label">Scheme 29</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/medium/jm1c00154_0030.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0030.jpeg" id="rightTab-gr30" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 29. Synthesis of Synucleozid-Based Small Molecules and Their Inhibitory Activity toward Î±-Synuclein Expression in SH-SY5Y Cells</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0030.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00154&amp;id=sch29"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch30a" data-index="31" class="article__inlineFigure"><h2 class="fig-label">Scheme 30</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/medium/jm1c00154_0031.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0031.jpeg" id="rightTab-GRAPHIC-d7e2457-autogenerated" class="rightTab-fig internalNav" /></img><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/medium/jm1c00154_0032.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0032.jpeg" id="rightTab-GRAPHIC-d7e2458-autogenerated" class="rightTab-fig internalNav" /></img><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/medium/jm1c00154_0033.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0033.jpeg" id="rightTab-GRAPHIC-d7e2459-autogenerated" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 30. Synthesis of Ametantrone-Based Structures as Tau pre-mRNA Stabilizers</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0031.jpeg jm1c00154_0032.jpeg jm1c00154_0033.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00154&amp;id=sch30a"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch31" data-index="32" class="article__inlineFigure"><h2 class="fig-label">Scheme 31</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/medium/jm1c00154_0034.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0034.jpeg" id="rightTab-gr34" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 31. Synthesis of Pyrimidone-Based Compound <b>193</b> and Activity to Suppress 4R Tau Levels in FTDP-17</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0034.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00154&amp;id=sch31"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch32" data-index="33" class="article__inlineFigure"><h2 class="fig-label">Scheme 32</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/medium/jm1c00154_0035.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0035.jpeg" id="rightTab-gr35" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 32. Synthesis of a Mitoxantrone-Bipartite PNA Alkyne-Based Compound with Inhibitory Activity toward Tau Exon 10 pre-mRNA Splicing</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0035.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00154&amp;id=sch32"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch33" data-index="34" class="article__inlineFigure"><h2 class="fig-label">Scheme 33</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/medium/jm1c00154_0036.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0036.jpeg" id="rightTab-gr36" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 33. Synthesis of Risdiplam, Its Activity toward SMN2 pre-mRNA Splicing in SMA Type 1 Fibroblasts, and a Comparison with the Initially Reported SMN2 pre-mRNA Splicing Modifier</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0036.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00154&amp;id=sch33"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch34" data-index="35" class="article__inlineFigure"><h2 class="fig-label">Scheme 34</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/medium/jm1c00154_0037.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0037.jpeg" id="rightTab-gr37" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 34. Synthesis of Branaplam, Its Activity toward SMN2 pre-mRNA Splicing in SMN ELISA Assay, and a Comparison with Its Predecessor Analogue <b>214</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0037.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00154&amp;id=sch34"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch35" data-index="36" class="article__inlineFigure"><h2 class="fig-label">Scheme 35</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/medium/jm1c00154_0039.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0039.jpeg" id="rightTab-gr38" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 35. (a) Synthesis, Activity, and Binding Mode of Triaminotriazine-Containing Heterocyclic Scaffold Targeting CUG Units of pre-mRNA; (b) Superposition of the [r(CUG)<sub>6</sub>]<sub>2</sub> Structure (Blue; PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3GM7">3GM7</a>) and the Stem-Loop Region of HIV-1 FS RNA (Gray; PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2L94">2L94</a>) and the Proposed Binding of the Ligand <b>230</b> on the CUG Sequence<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0039.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00154&amp;id=sch35"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reprinted with permission from ref <a onclick="showRef(event, 'ref108'); return false;" href="javascript:void(0);" class="ref ref108">(108)</a> [  <cite><i>J. Am. Chem. Soc.</i></cite> <span class="NLM_year">2014</span>, <em>136</em>, 6355â6361]. Copyright 2014 American Chemical Society.</p></p></figure><figure data-id="sch36a" data-index="37" class="article__inlineFigure"><h2 class="fig-label">Scheme 36</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/medium/jm1c00154_0040.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0040.jpeg" id="rightTab-GRAPHIC-d7e3262-autogenerated" class="rightTab-fig internalNav" /></img><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/medium/jm1c00154_0041.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0041.jpeg" id="rightTab-GRAPHIC-d7e3263-autogenerated" class="rightTab-fig internalNav" /></img><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/medium/jm1c00154_0042.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0042.jpeg" id="rightTab-GRAPHIC-d7e3264-autogenerated" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 36. (a, b) Synthesis of a Tetrazine/Cyclopropene-Based Peptide; (c) <i>In Situ</i> Generation of Inhibitors to Target r(CUG)<sup>exp</sup> Repeats in DM1-Patient Derived Cells</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0040.jpeg jm1c00154_0041.jpeg jm1c00154_0042.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00154&amp;id=sch36a"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch37" data-index="38" class="article__inlineFigure"><h2 class="fig-label">Scheme 37</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/medium/jm1c00154_0043.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0043.jpeg" id="rightTab-gr42" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 37. (a) Synthesis of Peptide <b>245</b>; (b) r(CUG)<sup>exp</sup>-Catalyzed Click Chemistry between Azides and Alkynes; (c) <i>In Situ</i> Generation of <b>246</b> with FRET Pairs to Image r(CUG)<sup>exp</sup> Repeats in DM1-Patient-Derived Cells<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0043.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00154&amp;id=sch37"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reprinted with permission from ref <a onclick="showRef(event, 'ref114'); return false;" href="javascript:void(0);" class="ref ref114">(114)</a> [  <cite><i>Nat. Chem. Biol.</i></cite> <span class="NLM_year">2017</span>, <em>13</em>, 188â193]. Copyright 2017 Nature Publishing Group.</p></p></figure><figure data-id="sch38" data-index="39" class="article__inlineFigure"><h2 class="fig-label">Scheme 38</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/medium/jm1c00154_0044.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0044.jpeg" id="rightTab-gr43" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 38. Synthesis of a Kanamycin A-Based Small Molecule and <i>In Situ</i> Generation of an Oligomer to Target r(CCUG)<sup>exp</sup> Repeats in DM2-Patient-Derived Cells</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0044.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00154&amp;id=sch38"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch39" data-index="40" class="article__inlineFigure"><h2 class="fig-label">Scheme 39</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/medium/jm1c00154_0045.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0045.jpeg" id="rightTab-gr44" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 39. Synthesis of Dimeric Benzamidine Derivative <b>256</b> and Its Activity in SCA10 Fibroblasts</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0045.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00154&amp;id=sch39"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch40" data-index="41" class="article__inlineFigure"><h2 class="fig-label">Scheme 40</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/medium/jm1c00154_0046.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0046.jpeg" id="rightTab-gr45" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 40. (a, b) Synthesis of Various Amino Acid-Based Heterocycles as Inhibitors of HIV-1 TAR mRNA Activity; (c) Binding Mode of <b>266a</b> with TAR mRNA</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0046.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00154&amp;id=sch40"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch41" data-index="42" class="article__inlineFigure"><h2 class="fig-label">Scheme 41</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/medium/jm1c00154_0047.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0047.jpeg" id="rightTab-gr46" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 41. <i>In Situ</i> Click Chemistry Using Alkynes <b>267</b> and Azides <b>268</b> in the Presence of TAR RNA to Discover Potential HIV-1 TAR-Tat Inhibitors</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0047.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00154&amp;id=sch41"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch42" data-index="43" class="article__inlineFigure"><h2 class="fig-label">Scheme 42</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/medium/jm1c00154_0048.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0048.jpeg" id="rightTab-gr47" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 42. (a) Synthesis of Ribocil, a Highly Selective Modifier of Bacterial Riboflavin Riboswitches; (b) X-ray Crystal Structure of Ribocil Bound to the <i>F. nucleatum</i> FMN Riboswitch<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00154/20210602/images/large/jm1c00154_0048.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00154&amp;id=sch42"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reprinted with permission from ref <a onclick="showRef(event, 'ref126'); return false;" href="javascript:void(0);" class="ref ref126">(126)</a> [  <cite><i>Nature</i></cite> <span class="NLM_year">2015</span>, <em>526</em>, 672â677]. Copyright 2015 Nature Publishing Group.</p></p></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i69">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_08615" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_08615" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 128 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Walsh, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jefferis, R.</span></span> <span> </span><span class="NLM_article-title">Post-translational modifications in the context of therapeutic proteins</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">1241</span>â <span class="NLM_lpage">1252</span>, <span class="refDoi">Â DOI: 10.1038/nbt1252</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1038%2Fnbt1252" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=17033665" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtVGgsrnK" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2006&pages=1241-1252&author=G.+Walshauthor=R.+Jefferis&title=Post-translational+modifications+in+the+context+of+therapeutic+proteins&doi=10.1038%2Fnbt1252"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Post-translational modifications in the context of therapeutic proteins</span></div><div class="casAuthors">Walsh, Gary; Jefferis, Roy</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1241-1252</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The majority of protein-based biopharmaceuticals approved or in clin. trials bear some form of post-translational modification (PTM), which can profoundly affect protein properties relevant to their therapeutic application.  Whereas glycosylation represents the most common modification, addnl. PTMs, including carboxylation, hydroxylation, sulfation, and amidation, are characteristic of some products.  The relation between structure and function is understood for many PTMs but remains incomplete for others, particularly in the case of complex PTMs, such as glycosylation.  A better understanding of such structure-function relations will facilitate the development of 2nd-generation products displaying a PTM profile engineered to optimize therapeutic usefulness.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJTnRzm0gY6LVg90H21EOLACvtfcHk0ljQ3a2XWFYTdQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtVGgsrnK&md5=3376c8170602376c6fe0505c32533828</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnbt1252&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt1252%26sid%3Dliteratum%253Aachs%26aulast%3DWalsh%26aufirst%3DG.%26aulast%3DJefferis%26aufirst%3DR.%26atitle%3DPost-translational%2520modifications%2520in%2520the%2520context%2520of%2520therapeutic%2520proteins%26jtitle%3DNat.%2520Biotechnol.%26date%3D2006%26volume%3D24%26spage%3D1241%26epage%3D1252%26doi%3D10.1038%2Fnbt1252" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Deribe, Y. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pawson, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dikic, I.</span></span> <span> </span><span class="NLM_article-title">Post-translational modifications in signal integration</span>. <i>Nat. Struct. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">666</span>â <span class="NLM_lpage">672</span>, <span class="refDoi">Â DOI: 10.1038/nsmb.1842</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1038%2Fnsmb.1842" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=20495563" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC3cXmsVWmsLw%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2010&pages=666-672&author=Y.+L.+Deribeauthor=T.+Pawsonauthor=I.+Dikic&title=Post-translational+modifications+in+signal+integration&doi=10.1038%2Fnsmb.1842"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Post-translational modifications in signal integration</span></div><div class="casAuthors">Deribe, Yonathan Lissanu; Pawson, Tony; Dikic, Ivan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">666-672</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Post-translational modifications of proteins and the domains that recognize these modifications play central roles in creating a highly dynamic relay system that reads and responds to alterations in the cellular microenvironment.  Here, the authors review the common principles of post-translational modifications and their importance in signal integration underlying epidermal growth factor receptor signaling and endocytosis, DNA-damage responses, and immunity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3REKBFj9N9rVg90H21EOLACvtfcHk0liB8FyT5ifGVw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXmsVWmsLw%253D&md5=4486fed28be542d6f49aae1405c85ee7</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnsmb.1842&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb.1842%26sid%3Dliteratum%253Aachs%26aulast%3DDeribe%26aufirst%3DY.%2BL.%26aulast%3DPawson%26aufirst%3DT.%26aulast%3DDikic%26aufirst%3DI.%26atitle%3DPost-translational%2520modifications%2520in%2520signal%2520integration%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2010%26volume%3D17%26spage%3D666%26epage%3D672%26doi%3D10.1038%2Fnsmb.1842" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peterson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loring, J.</span></span> <span> </span><span class="NLM_article-title">Protein post-translational modifications and regulation of pluripotency in human stem cells</span>. <i>Cell Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">143</span>â <span class="NLM_lpage">160</span>, <span class="refDoi">Â DOI: 10.1038/cr.2013.151</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1038%2Fcr.2013.151" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=24217768" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslyrtr3N" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2014&pages=143-160&author=Y.+Wangauthor=S.+Petersonauthor=J.+Loring&title=Protein+post-translational+modifications+and+regulation+of+pluripotency+in+human+stem+cells&doi=10.1038%2Fcr.2013.151"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Protein post-translational modifications and regulation of pluripotency in human stem cells</span></div><div class="casAuthors">Wang, Yu-Chieh; Peterson, Suzanne E.; Loring, Jeanne F.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">143-160</span>CODEN:
                <span class="NLM_cas:coden">CREEB6</span>;
        ISSN:<span class="NLM_cas:issn">1001-0602</span>.
    
            (<span class="NLM_cas:orgname">NPG Nature Asia-Pacific</span>)
        </div><div class="casAbstract">A review.  Post-translational modifications (PTMs) are known to be essential mechanisms used by eukaryotic cells to diversify their protein functions and dynamically coordinate their signaling networks.  Defects in PTMs have been linked to numerous developmental disorders and human diseases, highlighting the importance of PTMs in maintaining normal cellular states.  Human pluripotent stem cells (hPSCs), including embryonic stem cells (hESCs) and induced pluripotent stem cells (hiPSCs), are capable of self-renewal and differentiation into a variety of functional somatic cells; these cells hold a great promise for the advancement of biomedical research and clin. therapy.  The mechanisms underlying cellular pluripotency in human cells have been extensively explored in the past decade.  In addn. to the vast amt. of knowledge obtained from the genetic and transcriptional research in hPSCs, there is a rapidly growing interest in the stem cell biol. field to examine pluripotency at the protein and PTM level.  This review addresses recent progress toward understanding the role of PTMs (glycosylation, phosphorylation, acetylation and methylation) in the regulation of cellular pluripotency.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVfWBZsXmIn7Vg90H21EOLACvtfcHk0liB8FyT5ifGVw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslyrtr3N&md5=4d3f8b4cb9a995f25993a675ec9c4f33</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fcr.2013.151&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fcr.2013.151%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DY.%26aulast%3DPeterson%26aufirst%3DS.%26aulast%3DLoring%26aufirst%3DJ.%26atitle%3DProtein%2520post-translational%2520modifications%2520and%2520regulation%2520of%2520pluripotency%2520in%2520human%2520stem%2520cells%26jtitle%3DCell%2520Res.%26date%3D2014%26volume%3D24%26spage%3D143%26epage%3D160%26doi%3D10.1038%2Fcr.2013.151" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hopkins, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groom, C. R.</span></span> <span> </span><span class="NLM_article-title">The druggable genome</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">727</span>â <span class="NLM_lpage">730</span>, <span class="refDoi">Â DOI: 10.1038/nrd892</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1038%2Fnrd892" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=12209152" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BD38XmslamtrY%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2002&pages=727-730&author=A.+L.+Hopkinsauthor=C.+R.+Groom&title=The+druggable+genome&doi=10.1038%2Fnrd892"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Opinion: The druggable genome</span></div><div class="casAuthors">Hopkins, Andrew L.; Groom, Colin R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">727-730</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  An assessment of the no. of mol. targets that represent an opportunity for therapeutic intervention is crucial to the development of post-genomic research strategies within the pharmaceutical industry.  Now that we know the size of the human genome, it is interesting to consider just how many mol. targets this opportunity represents.  We start from the position that we understand the properties that are required for a good drug, and therefore must be able to understand what makes a good drug target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq7Uvqi8O9wzrVg90H21EOLACvtfcHk0lj6d7VV7refjQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XmslamtrY%253D&md5=d6dc7c67822661fe03e920695def1329</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fnrd892&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd892%26sid%3Dliteratum%253Aachs%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26aulast%3DGroom%26aufirst%3DC.%2BR.%26atitle%3DThe%2520druggable%2520genome%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2002%26volume%3D1%26spage%3D727%26epage%3D730%26doi%3D10.1038%2Fnrd892" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dixon, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stockwell, B. R.</span></span> <span> </span><span class="NLM_article-title">Identifying druggable disease-modifying gene products</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">549</span>â <span class="NLM_lpage">555</span>, <span class="refDoi">Â DOI: 10.1016/j.cbpa.2009.08.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1016%2Fj.cbpa.2009.08.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=19740696" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsV2msLzJ" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2009&pages=549-555&author=S.+J.+Dixonauthor=B.+R.+Stockwell&title=Identifying+druggable+disease-modifying+gene+products&doi=10.1016%2Fj.cbpa.2009.08.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Identifying druggable disease-modifying gene products</span></div><div class="casAuthors">Dixon, Scott J.; Stockwell, Brent R.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5-6</span>),
    <span class="NLM_cas:pages">549-555</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Many disease genes encode proteins that are difficult to target directly using small mol. drugs.  Improvements in libraries based on synthetic compds., natural products, and other types of mols. may ultimately allow some challenging proteins to be successfully targeted; however, these developments alone are unlikely to be sufficient.  A complementary strategy exploits the functional interconnectivity of intracellular networks to find druggable targets lying upstream, downstream, or in parallel to a disease-causing gene, where modulation can influence the disease process indirectly.  These targets can be selected using prior knowledge of disease-assocd. pathways or identified using phenotypic chem. and genetic screens in model organisms and cells.  These approaches should facilitate the identification of effective drug targets for many genetic disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJzodncPkmoLVg90H21EOLACvtfcHk0lj6d7VV7refjQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsV2msLzJ&md5=263d9f02416aac4fad4a7e7adfdda665</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2009.08.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2009.08.003%26sid%3Dliteratum%253Aachs%26aulast%3DDixon%26aufirst%3DS.%2BJ.%26aulast%3DStockwell%26aufirst%3DB.%2BR.%26atitle%3DIdentifying%2520druggable%2520disease-modifying%2520gene%2520products%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2009%26volume%3D13%26spage%3D549%26epage%3D555%26doi%3D10.1016%2Fj.cbpa.2009.08.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Warner, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hajdin, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weeks, K. M.</span></span> <span> </span><span class="NLM_article-title">Principles for targeting RNA with drug-like small molecules</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">547</span>â <span class="NLM_lpage">558</span>, <span class="refDoi">Â DOI: 10.1038/nrd.2018.93</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1038%2Fnrd.2018.93" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=29977051" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1OntLnL" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2018&pages=547-558&author=K.+D.+Warnerauthor=C.+E.+Hajdinauthor=K.+M.+Weeks&title=Principles+for+targeting+RNA+with+drug-like+small+molecules&doi=10.1038%2Fnrd.2018.93"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Principles for targeting RNA with drug-like small molecules</span></div><div class="casAuthors">Warner, Katherine Deigan; Hajdin, Christine E.; Weeks, Kevin M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">547-558</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">RNA mols. are essential for cellular information transfer and gene regulation, and RNAs have been implicated in many human diseases.  Messenger and non-coding RNAs contain highly structured elements, and evidence suggests that many of these structures are important for function.  Targeting these RNAs with small mols. offers opportunities to therapeutically modulate numerous cellular processes, including those linked to undruggable protein targets.  Despite this promise, there is currently only a single class of human-designed small mols. that target RNA used clin. - the linezolid antibiotics.  However, a growing no. of small-mol. RNA ligands are being identified, leading to burgeoning interest in the field.  Here, we discuss principles for discovering small-mol. drugs that target RNA and argue that the overarching challenge is to identify appropriate target structures - namely, in disease-causing RNAs that have high information content and, consequently, appropriate ligand-binding pockets.  If focus is placed on such druggable binding sites in RNA, extensive knowledge of the typical physicochem. properties of drug-like small mols. could then enable small-mol. drug discovery for RNA targets to become (only) roughly as difficult as for protein targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYl_OguWzasrVg90H21EOLACvtfcHk0lj6d7VV7refjQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1OntLnL&md5=df01185ded43db426749b9c656a04f64</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2018.93&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2018.93%26sid%3Dliteratum%253Aachs%26aulast%3DWarner%26aufirst%3DK.%2BD.%26aulast%3DHajdin%26aufirst%3DC.%2BE.%26aulast%3DWeeks%26aufirst%3DK.%2BM.%26atitle%3DPrinciples%2520for%2520targeting%2520RNA%2520with%2520drug-like%2520small%2520molecules%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2018%26volume%3D17%26spage%3D547%26epage%3D558%26doi%3D10.1038%2Fnrd.2018.93" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Beermann, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piccoli, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Viereck, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thum, T.</span></span> <span> </span><span class="NLM_article-title">Non-coding RNAs in development and disease: Background, mechanism, and therapeutic approaches</span>. <i>Physiol. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>96</i></span>,  <span class="NLM_fpage">1297</span>â <span class="NLM_lpage">1325</span>, <span class="refDoi">Â DOI: 10.1152/physrev.00041.2015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1152%2Fphysrev.00041.2015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=27535639" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC1cXkvFGhtrc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=96&publication_year=2016&pages=1297-1325&author=J.+Beermannauthor=M.+T.+Piccoliauthor=J.+Viereckauthor=T.+Thum&title=Non-coding+RNAs+in+development+and+disease%3A+Background%2C+mechanism%2C+and+therapeutic+approaches&doi=10.1152%2Fphysrev.00041.2015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Non-coding RNAs in development and disease: background, mechanisms, and therapeutic approaches</span></div><div class="casAuthors">Beermann, Julia; Piccoli, Maria-Teresa; Viereck, Janika; Thum, Thomas</div><div class="citationInfo"><span class="NLM_cas:title">Physiological Reviews</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">96</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1297-1325</span>CODEN:
                <span class="NLM_cas:coden">PHREA7</span>;
        ISSN:<span class="NLM_cas:issn">0031-9333</span>.
    
            (<span class="NLM_cas:orgname">American Physiological Society</span>)
        </div><div class="casAbstract">Advances in RNA-sequencing techniques have led to the discovery of thousands of non-coding transcripts with unknown function.  There are several types of non-coding linearRNAs such as microRNAs (miRNA) and long non-coding RNAs (lncRNA), as well as circular RNAs(circRNA) consisting of a closed continuous loop.  This review guides the reader through important aspects of non-coding RNA biol.  This includes their biogenesis, mode of actions, physiologicalfunction, as well as their role in the disease context (such as in cancer or the cardiovascularsystem).  We specically focus on non-coding RNAs as potential therapeutic targets and diagnosticbiomarkers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpyehkZcx8nILVg90H21EOLACvtfcHk0ljvCY7Zkm-16g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXkvFGhtrc%253D&md5=f9fe973309e572a492313ce98bbd4f6b</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1152%2Fphysrev.00041.2015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1152%252Fphysrev.00041.2015%26sid%3Dliteratum%253Aachs%26aulast%3DBeermann%26aufirst%3DJ.%26aulast%3DPiccoli%26aufirst%3DM.%2BT.%26aulast%3DViereck%26aufirst%3DJ.%26aulast%3DThum%26aufirst%3DT.%26atitle%3DNon-coding%2520RNAs%2520in%2520development%2520and%2520disease%253A%2520Background%252C%2520mechanism%252C%2520and%2520therapeutic%2520approaches%26jtitle%3DPhysiol.%2520Rev.%26date%3D2016%26volume%3D96%26spage%3D1297%26epage%3D1325%26doi%3D10.1152%2Fphysrev.00041.2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">TEP Consortium.</span> <span> </span><span class="NLM_article-title">An integrated encyclopedia
of DNA elements in the human genome.</span> <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume">489</span>,  <span class="NLM_fpage">57</span>â <span class="NLM_lpage">74</span> <span class="refDoi">Â DOI: 10.1038/nature11247</span> .</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1038%2Fnature11247" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=22955616" title="">PubMed</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=489&publication_year=2012&pages=57-74&author=TEP+Consortium.&title=An+integrated+encyclopedia%0Aof+DNA+elements+in+the+human+genome.&doi=10.1038%2Fnature11247"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnature11247&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature11247%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DAn%2520integrated%2520encyclopedia%250Aof%2520DNA%2520elements%2520in%2520the%2520human%2520genome.%26jtitle%3DNature%26date%3D2012%26volume%3D489%26spage%3D57%26epage%3D74%26doi%3D10.1038%2Fnature11247" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mongelli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martelli, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farsetti, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaetano, C.</span></span> <span> </span><span class="NLM_article-title">The dark that matters: Long non-coding RNAs as master regulators of cellular metabolism in non-communicable diseases</span>. <i>Front. Physiol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">369</span>â <span class="NLM_lpage">405</span>, <span class="refDoi">Â DOI: 10.3389/fphys.2019.00369</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.3389%2Ffphys.2019.00369" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=31191327" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A280%3ADC%252BB3M3lvFyntw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=369-405&author=A.+Mongelliauthor=F.+Martelliauthor=A.+Farsettiauthor=C.+Gaetano&title=The+dark+that+matters%3A+Long+non-coding+RNAs+as+master+regulators+of+cellular+metabolism+in+non-communicable+diseases&doi=10.3389%2Ffphys.2019.00369"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">The Dark That Matters: Long Non-coding RNAs as Master Regulators of Cellular Metabolism in Non-communicable Diseases</span></div><div class="casAuthors">Mongelli Alessia; Gaetano Carlo; Martelli Fabio; Farsetti Antonella</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in physiology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">369</span>
        ISSN:<span class="NLM_cas:issn">1664-042X</span>.
    </div><div class="casAbstract">Non-coding RNAs are pivotal for many cellular functions, such as splicing, gene regulation, chromosome structure, and hormone-like activity.  Here, we will report about the biology and the general molecular mechanisms associated with long non-coding RNAs (lncRNAs), a class of >200 nucleotides-long ribonucleic acid sequences, and their role in chronic non-transmissible diseases.  In particular, we will summarize knowledge about some of the best-characterized lncRNAs, such as H19 and MALAT1, and how they regulate carbohydrate and lipid metabolism as well as protein synthesis and degradation.  Evidence is discussed about how lncRNAs expression might affect cellular and organismal metabolism and whether their modulation could provide ground for the development of innovative treatments.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTi8XwXVd9vV1WTCLT8lX9SfW6udTcc2eafAEZaLiw3_Lntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3M3lvFyntw%253D%253D&md5=d92f9d9c49667570046473d6d93ffb50</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.3389%2Ffphys.2019.00369&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffphys.2019.00369%26sid%3Dliteratum%253Aachs%26aulast%3DMongelli%26aufirst%3DA.%26aulast%3DMartelli%26aufirst%3DF.%26aulast%3DFarsetti%26aufirst%3DA.%26aulast%3DGaetano%26aufirst%3DC.%26atitle%3DThe%2520dark%2520that%2520matters%253A%2520Long%2520non-coding%2520RNAs%2520as%2520master%2520regulators%2520of%2520cellular%2520metabolism%2520in%2520non-communicable%2520diseases%26jtitle%3DFront.%2520Physiol.%26date%3D2019%26volume%3D10%26spage%3D369%26epage%3D405%26doi%3D10.3389%2Ffphys.2019.00369" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ling, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fabbri, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calin, G. A.</span></span> <span> </span><span class="NLM_article-title">MicroRNAs and other non-coding RNAs as targets for anticancer drug development</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">847</span>â <span class="NLM_lpage">865</span>, <span class="refDoi">Â DOI: 10.1038/nrd4140</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1038%2Fnrd4140" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=24172333" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs12ks73I" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2013&pages=847-865&author=H.+Lingauthor=M.+Fabbriauthor=G.+A.+Calin&title=MicroRNAs+and+other+non-coding+RNAs+as+targets+for+anticancer+drug+development&doi=10.1038%2Fnrd4140"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">MicroRNAs and other non-coding RNAs as targets for anticancer drug development</span></div><div class="casAuthors">Ling, Hui; Fabbri, Muller; Calin, George A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">847-865</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The first cancer-targeted microRNA (miRNA) drug - MRX34, a liposome-based miR-34 mimic - entered Phase I clin. trials in patients with advanced hepatocellular carcinoma in Apr. 2013, and miRNA therapeutics are attracting special attention from both academia and biotechnol. companies.  Although miRNAs are the most studied non-coding RNAs (ncRNAs) to date, the importance of long non-coding RNAs (lncRNAs) is increasingly being recognized.  Here, we summarize the roles of miRNAs and lncRNAs in cancer, with a focus on the recently identified novel mechanisms of action, and discuss the current strategies in designing ncRNA-targeting therapeutics, as well as the assocd. challenges.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7sEXRlR4FNLVg90H21EOLACvtfcHk0lhClOnz_d0szg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs12ks73I&md5=5bb9c58f97adaa031aea14033150e383</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fnrd4140&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4140%26sid%3Dliteratum%253Aachs%26aulast%3DLing%26aufirst%3DH.%26aulast%3DFabbri%26aufirst%3DM.%26aulast%3DCalin%26aufirst%3DG.%2BA.%26atitle%3DMicroRNAs%2520and%2520other%2520non-coding%2520RNAs%2520as%2520targets%2520for%2520anticancer%2520drug%2520development%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2013%26volume%3D12%26spage%3D847%26epage%3D865%26doi%3D10.1038%2Fnrd4140" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mattick, J. S.</span></span> <span> </span><span class="NLM_article-title">The genetic signatures of noncoding RNAs</span>. <i>PLoS Genet.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>5</i></span>, <span class="NLM_elocation-id">e1000459</span> <span class="refDoi">Â DOI: 10.1371/journal.pgen.1000459</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1371%2Fjournal.pgen.1000459" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=19390609" title="">PubMed</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2009&author=J.+S.+Mattick&title=The+genetic+signatures+of+noncoding+RNAs&doi=10.1371%2Fjournal.pgen.1000459"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pgen.1000459&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pgen.1000459%26sid%3Dliteratum%253Aachs%26aulast%3DMattick%26aufirst%3DJ.%2BS.%26atitle%3DThe%2520genetic%2520signatures%2520of%2520noncoding%2520RNAs%26jtitle%3DPLoS%2520Genet.%26date%3D2009%26volume%3D5%26doi%3D10.1371%2Fjournal.pgen.1000459" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bartel, D. P.</span></span> <span> </span><span class="NLM_article-title">MicroRNAs: target recognition and regulatory functions</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>136</i></span>,  <span class="NLM_fpage">215</span>â <span class="NLM_lpage">233</span>, <span class="refDoi">Â DOI: 10.1016/j.cell.2009.01.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1016%2Fj.cell.2009.01.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=19167326" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhs1Kiuro%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=136&publication_year=2009&pages=215-233&author=D.+P.+Bartel&title=MicroRNAs%3A+target+recognition+and+regulatory+functions&doi=10.1016%2Fj.cell.2009.01.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">MicroRNAs: target recognition and regulatory functions</span></div><div class="casAuthors">Bartel, David P.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">136</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">215-233</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  MicroRNAs (miRNAs) are endogenous â¼23 nt RNAs that play important gene-regulatory roles in animals and plants by pairing to the mRNAs of protein-coding genes to direct their posttranscriptional repression.  This review outlines the current understanding of miRNA target recognition in animals and discusses the widespread impact of miRNAs on both the expression and evolution of protein-coding genes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodHBQuLKSQY7Vg90H21EOLACvtfcHk0lgpi2vleeJ76g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhs1Kiuro%253D&md5=8d9c406309fb79682363a27d265a0674</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2009.01.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2009.01.002%26sid%3Dliteratum%253Aachs%26aulast%3DBartel%26aufirst%3DD.%2BP.%26atitle%3DMicroRNAs%253A%2520target%2520recognition%2520and%2520regulatory%2520functions%26jtitle%3DCell%26date%3D2009%26volume%3D136%26spage%3D215%26epage%3D233%26doi%3D10.1016%2Fj.cell.2009.01.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Djebali, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merkel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dobin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lassmann, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mortazavi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanzer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lagarde, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlesinger, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xue, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marinov, G. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khatun, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zaleski, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rozowsky, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">RÃ¶der, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kokocinsk, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abdelhamid, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alioto, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Antoshechkin, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baer, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bar, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Batut, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bell, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bell, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chakrabortty, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chrast, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curado, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Derrien, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drenkow, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dumais, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dumais, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duttagupta, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Falconnet, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fastuca, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fejes-Toth, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferreira, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foissac, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fullwood, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gonzalez, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gunawardena, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howald, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jha, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapranov, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">King, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kingswood, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, O. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Persaud, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Preall, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ribeca, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Risk, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robyr, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sammeth, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaffer, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">See, L.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shahab, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skancke, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suzuki, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takahashi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tilgner, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trout, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walters, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wrobel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruan, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayashizaki, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harrow, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerstein, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hubbard, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reymond, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Antonarakis, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hannon, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giddings, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wold, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carninci, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">GuigÃ³, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gingeras, T. R.</span></span> <span> </span><span class="NLM_article-title">Landscape of transcription in human cells</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>489</i></span>,  <span class="NLM_fpage">101</span>â <span class="NLM_lpage">108</span>, <span class="refDoi">Â DOI: 10.1038/nature11233</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1038%2Fnature11233" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=22955620" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtlGnt73M" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=489&publication_year=2012&pages=101-108&author=S.+Djebaliauthor=C.+A.+Davisauthor=A.+Merkelauthor=A.+Dobinauthor=T.+Lassmannauthor=A.+Mortazaviauthor=A.+Tanzerauthor=J.+Lagardeauthor=W.+Linauthor=F.+Schlesingerauthor=C.+Xueauthor=G.+K.+Marinovauthor=J.+Khatunauthor=B.+A.+Williamsauthor=C.+Zaleskiauthor=J.+Rozowskyauthor=M.+R%C3%B6derauthor=F.+Kokocinskauthor=R.+F.+Abdelhamidauthor=T.+Aliotoauthor=I.+Antoshechkinauthor=M.+T.+Baerauthor=N.+S.+Barauthor=P.+Batutauthor=K.+Bellauthor=I.+Bellauthor=S.+Chakraborttyauthor=X.+Chenauthor=J.+Chrastauthor=J.+Curadoauthor=T.+Derrienauthor=J.+Drenkowauthor=E.+Dumaisauthor=J.+Dumaisauthor=R.+Duttaguptaauthor=E.+Falconnetauthor=M.+Fastucaauthor=K.+Fejes-Tothauthor=P.+Ferreiraauthor=S.+Foissacauthor=M.+J.+Fullwoodauthor=H.+Gaoauthor=D.+Gonzalezauthor=A.+Gordonauthor=H.+Gunawardenaauthor=C.+Howaldauthor=S.+Jhaauthor=R.+Johnsonauthor=P.+Kapranovauthor=B.+Kingauthor=C.+Kingswoodauthor=O.+J.+Luoauthor=E.+Parkauthor=K.+Persaudauthor=J.+B.+Preallauthor=P.+Ribecaauthor=B.+Riskauthor=D.+Robyrauthor=M.+Sammethauthor=L.+Schafferauthor=L.-H.+Seeauthor=A.+Shahabauthor=J.+Skanckeauthor=A.+M.+Suzukiauthor=H.+Takahashiauthor=H.+Tilgnerauthor=D.+Troutauthor=N.+Waltersauthor=H.+Wangauthor=J.+Wrobelauthor=Y.+Yuauthor=X.+Ruanauthor=Y.+Hayashizakiauthor=J.+Harrowauthor=M.+Gersteinauthor=T.+Hubbardauthor=A.+Reymondauthor=S.+E.+Antonarakisauthor=G.+Hannonauthor=M.+C.+Giddingsauthor=Y.+Ruanauthor=B.+Woldauthor=P.+Carninciauthor=R.+Guig%C3%B3author=T.+R.+Gingeras&title=Landscape+of+transcription+in+human+cells&doi=10.1038%2Fnature11233"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Landscape of transcription in human cells</span></div><div class="casAuthors">Djebali, Sarah; Davis, Carrie A.; Merkel, Angelika; Dobin, Alex; Lassmann, Timo; Mortazavi, Ali; Tanzer, Andrea; Lagarde, Julien; Lin, Wei; Schlesinger, Felix; Xue, Chenghai; Marinov, Georgi K.; Khatun, Jainab; Williams, Brian A.; Zaleski, Chris; Rozowsky, Joel; Roeder, Maik; Kokocinski, Felix; Abdelhamid, Rehab F.; Alioto, Tyler; Antoshechkin, Igor; Baer, Michael T.; Bar, Nadav S.; Batut, Philippe; Bell, Kimberly; Bell, Ian; Chakrabortty, Sudipto; Chen, Xian; Chrast, Jacqueline; Curado, Joao; Derrien, Thomas; Drenkow, Jorg; Dumais, Erica; Dumais, Jacqueline; Duttagupta, Radha; Falconnet, Emilie; Fastuca, Meagan; Fejes-Toth, Kata; Ferreira, Pedro; Foissac, Sylvain; Fullwood, Melissa J.; Gao, Hui; Gonzalez, David; Gordon, Assaf; Gunawardena, Harsha; Howald, Cedric; Jha, Sonali; Johnson, Rory; Kapranov, Philipp; King, Brandon; Kingswood, Colin; Luo, Oscar J.; Park, Eddie; Persaud, Kimberly; Preall, Jonathan B.; Ribeca, Paolo; Risk, Brian; Robyr, Daniel; Sammeth, Michael; Schaffer, Lorian; See, Lei-Hoon; Shahab, Atif; Skancke, Jorgen; Suzuki, Ana Maria; Takahashi, Hazuki; Tilgner, Hagen; Trout, Diane; Walters, Nathalie; Wang, Huaien; Wrobel, John; Yu, Yanbao; Ruan, Xiaoan; Hayashizaki, Yoshihide; Harrow, Jennifer; Gerstein, Mark; Hubbard, Tim; Reymond, Alexandre; Antonarakis, Stylianos E.; Hannon, Gregory; Giddings, Morgan C.; Ruan, Yijun; Wold, Barbara; Carninci, Piero; Guigo, Roderic; Gingeras, Thomas R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">489</span>
        (<span class="NLM_cas:issue">7414</span>),
    <span class="NLM_cas:pages">101-108</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Eukaryotic cells make many types of primary and processed RNAs that are found either in specific subcellular compartments or throughout the cells.  A complete catalog of these RNAs is not yet available and their characteristic subcellular localizations are also poorly understood.  Because RNA represents the direct output of the genetic information encoded by genomes and a significant proportion of a cell's regulatory capabilities are focused on its synthesis, processing, transport, modification and translation, the generation of such a catalog is crucial for understanding genome function.  Here we report evidence that three-quarters of the human genome is capable of being transcribed, as well as observations about the range and levels of expression, localization, processing fates, regulatory regions and modifications of almost all currently annotated and thousands of previously unannotated RNAs.  These observations, taken together, prompt a redefinition of the concept of a gene.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoQJgBaBiE8o7Vg90H21EOLACvtfcHk0ljj2eXj-Klp1A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtlGnt73M&md5=8f8ee2a1189ced1010dec5e45a305acf</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2Fnature11233&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature11233%26sid%3Dliteratum%253Aachs%26aulast%3DDjebali%26aufirst%3DS.%26aulast%3DDavis%26aufirst%3DC.%2BA.%26aulast%3DMerkel%26aufirst%3DA.%26aulast%3DDobin%26aufirst%3DA.%26aulast%3DLassmann%26aufirst%3DT.%26aulast%3DMortazavi%26aufirst%3DA.%26aulast%3DTanzer%26aufirst%3DA.%26aulast%3DLagarde%26aufirst%3DJ.%26aulast%3DLin%26aufirst%3DW.%26aulast%3DSchlesinger%26aufirst%3DF.%26aulast%3DXue%26aufirst%3DC.%26aulast%3DMarinov%26aufirst%3DG.%2BK.%26aulast%3DKhatun%26aufirst%3DJ.%26aulast%3DWilliams%26aufirst%3DB.%2BA.%26aulast%3DZaleski%26aufirst%3DC.%26aulast%3DRozowsky%26aufirst%3DJ.%26aulast%3DR%25C3%25B6der%26aufirst%3DM.%26aulast%3DKokocinsk%26aufirst%3DF.%26aulast%3DAbdelhamid%26aufirst%3DR.%2BF.%26aulast%3DAlioto%26aufirst%3DT.%26aulast%3DAntoshechkin%26aufirst%3DI.%26aulast%3DBaer%26aufirst%3DM.%2BT.%26aulast%3DBar%26aufirst%3DN.%2BS.%26aulast%3DBatut%26aufirst%3DP.%26aulast%3DBell%26aufirst%3DK.%26aulast%3DBell%26aufirst%3DI.%26aulast%3DChakrabortty%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DChrast%26aufirst%3DJ.%26aulast%3DCurado%26aufirst%3DJ.%26aulast%3DDerrien%26aufirst%3DT.%26aulast%3DDrenkow%26aufirst%3DJ.%26aulast%3DDumais%26aufirst%3DE.%26aulast%3DDumais%26aufirst%3DJ.%26aulast%3DDuttagupta%26aufirst%3DR.%26aulast%3DFalconnet%26aufirst%3DE.%26aulast%3DFastuca%26aufirst%3DM.%26aulast%3DFejes-Toth%26aufirst%3DK.%26aulast%3DFerreira%26aufirst%3DP.%26aulast%3DFoissac%26aufirst%3DS.%26aulast%3DFullwood%26aufirst%3DM.%2BJ.%26aulast%3DGao%26aufirst%3DH.%26aulast%3DGonzalez%26aufirst%3DD.%26aulast%3DGordon%26aufirst%3DA.%26aulast%3DGunawardena%26aufirst%3DH.%26aulast%3DHowald%26aufirst%3DC.%26aulast%3DJha%26aufirst%3DS.%26aulast%3DJohnson%26aufirst%3DR.%26aulast%3DKapranov%26aufirst%3DP.%26aulast%3DKing%26aufirst%3DB.%26aulast%3DKingswood%26aufirst%3DC.%26aulast%3DLuo%26aufirst%3DO.%2BJ.%26aulast%3DPark%26aufirst%3DE.%26aulast%3DPersaud%26aufirst%3DK.%26aulast%3DPreall%26aufirst%3DJ.%2BB.%26aulast%3DRibeca%26aufirst%3DP.%26aulast%3DRisk%26aufirst%3DB.%26aulast%3DRobyr%26aufirst%3DD.%26aulast%3DSammeth%26aufirst%3DM.%26aulast%3DSchaffer%26aufirst%3DL.%26aulast%3DSee%26aufirst%3DL.-H.%26aulast%3DShahab%26aufirst%3DA.%26aulast%3DSkancke%26aufirst%3DJ.%26aulast%3DSuzuki%26aufirst%3DA.%2BM.%26aulast%3DTakahashi%26aufirst%3DH.%26aulast%3DTilgner%26aufirst%3DH.%26aulast%3DTrout%26aufirst%3DD.%26aulast%3DWalters%26aufirst%3DN.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DWrobel%26aufirst%3DJ.%26aulast%3DYu%26aufirst%3DY.%26aulast%3DRuan%26aufirst%3DX.%26aulast%3DHayashizaki%26aufirst%3DY.%26aulast%3DHarrow%26aufirst%3DJ.%26aulast%3DGerstein%26aufirst%3DM.%26aulast%3DHubbard%26aufirst%3DT.%26aulast%3DReymond%26aufirst%3DA.%26aulast%3DAntonarakis%26aufirst%3DS.%2BE.%26aulast%3DHannon%26aufirst%3DG.%26aulast%3DGiddings%26aufirst%3DM.%2BC.%26aulast%3DRuan%26aufirst%3DY.%26aulast%3DWold%26aufirst%3DB.%26aulast%3DCarninci%26aufirst%3DP.%26aulast%3DGuig%25C3%25B3%26aufirst%3DR.%26aulast%3DGingeras%26aufirst%3DT.%2BR.%26atitle%3DLandscape%2520of%2520transcription%2520in%2520human%2520cells%26jtitle%3DNature%26date%3D2012%26volume%3D489%26spage%3D101%26epage%3D108%26doi%3D10.1038%2Fnature11233" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Walder, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patten, W. J. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ritchie, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montange, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, T. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woodside, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perkins, T. T.</span></span> <span> </span><span class="NLM_article-title">High-Precision single-molecule characterization of the folding of an HIV RNA hairpin by atomic force microscopy</span>. <i>Nano Lett.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">6318</span>â <span class="NLM_lpage">6325</span>, <span class="refDoi">Â DOI: 10.1021/acs.nanolett.8b02597</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.nanolett.8b02597" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslGqsLvE" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2018&pages=6318-6325&author=R.+Walderauthor=W.+J.+V.+Pattenauthor=D.+B.+Ritchieauthor=R.+K.+Montangeauthor=T.+W.+Millerauthor=M.+T.+Woodsideauthor=T.+T.+Perkins&title=High-Precision+single-molecule+characterization+of+the+folding+of+an+HIV+RNA+hairpin+by+atomic+force+microscopy&doi=10.1021%2Facs.nanolett.8b02597"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">High-Precision Single-Molecule Characterization of the Folding of an HIV RNA Hairpin by Atomic Force Microscopy</span></div><div class="casAuthors">Walder, Robert; Van Patten, William J.; Ritchie, Dustin B.; Montange, Rebecca K.; Miller, Ty W.; Woodside, Michael T.; Perkins, Thomas T.</div><div class="citationInfo"><span class="NLM_cas:title">Nano Letters</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">6318-6325</span>CODEN:
                <span class="NLM_cas:coden">NALEFD</span>;
        ISSN:<span class="NLM_cas:issn">1530-6984</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The folding of RNA into a wide range of structures is essential for its diverse biol. functions from enzymic catalysis to ligand binding and gene regulation.  The unfolding and refolding of individual RNA mols. can be probed by single-mol. force spectroscopy (SMFS), enabling detailed characterization of the conformational dynamics of the mol. as well as the free-energy landscape underlying folding.  Historically, high-precision SMFS studies of RNA have been limited to custom-built optical traps.  Although com. at. force microscopes (AFMs) are widely deployed and offer significant advantages in ease-of-use over custom-built optical traps, traditional AFM-based SMFS lacks the sensitivity and stability to characterize individual RNA mols. precisely.  Here, the authors developed a high-precision SMFS assay to study RNA folding using a com. AFM and applied it to characterize a small RNA hairpin from HIV that plays a key role in stimulating programmed ribosomal frameshifting.  The authors achieved rapid data acquisition in a dynamic assay, unfolding and then refolding the same individual hairpin more than 1100 times in 15 min.  In comparison to measurements using optical traps, the AFM-based assay featured a stiffer force probe and a less compliant construct, providing a complementary measurement regime that dramatically accelerated equil. folding dynamics.  Not only did kinetic anal. of equil. trajectories of the HIV RNA hairpin yield the traditional parameters used to characterize folding by SMFS (zero-force rate consts. and distances to the transition state), but the authors also reconstructed the full 1D projection of the folding free-energy landscape comparable to state-of-the-art studies using dual-beam optical traps, a first for this RNA hairpin and AFM studies of nucleic acids in general.  Looking forward, the authors anticipate that the ease-of-use of the high-precision assay implemented on a com. AFM will accelerate studying folding of diverse nucleic acid structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoaueT2gVqbirVg90H21EOLACvtfcHk0lgE0VAIf0E19g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslGqsLvE&md5=8bf17b525c2a9c52d4489737dfa3004b</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Facs.nanolett.8b02597&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.nanolett.8b02597%26sid%3Dliteratum%253Aachs%26aulast%3DWalder%26aufirst%3DR.%26aulast%3DPatten%26aufirst%3DW.%2BJ.%2BV.%26aulast%3DRitchie%26aufirst%3DD.%2BB.%26aulast%3DMontange%26aufirst%3DR.%2BK.%26aulast%3DMiller%26aufirst%3DT.%2BW.%26aulast%3DWoodside%26aufirst%3DM.%2BT.%26aulast%3DPerkins%26aufirst%3DT.%2BT.%26atitle%3DHigh-Precision%2520single-molecule%2520characterization%2520of%2520the%2520folding%2520of%2520an%2520HIV%2520RNA%2520hairpin%2520by%2520atomic%2520force%2520microscopy%26jtitle%3DNano%2520Lett.%26date%3D2018%26volume%3D18%26spage%3D6318%26epage%3D6325%26doi%3D10.1021%2Facs.nanolett.8b02597" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sztuba-Solinska, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chavez-Calvillo, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cline, S. E.</span></span> <span> </span><span class="NLM_article-title">Unveiling the druggable RNA targets and small molecule therapeutics</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">2149</span>â <span class="NLM_lpage">2165</span>, <span class="refDoi">Â DOI: 10.1016/j.bmc.2019.03.057</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1016%2Fj.bmc.2019.03.057" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=30981606" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC1MXntFCiu74%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2019&pages=2149-2165&author=J.+Sztuba-Solinskaauthor=G.+Chavez-Calvilloauthor=S.+E.+Cline&title=Unveiling+the+druggable+RNA+targets+and+small+molecule+therapeutics&doi=10.1016%2Fj.bmc.2019.03.057"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Unveiling the druggable RNA targets and small molecule therapeutics</span></div><div class="casAuthors">Sztuba-Solinska, Joanna; Chavez-Calvillo, Gabriela; Cline, Sabrina Elizabeth</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2149-2165</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The increasing appreciation for the crucial roles of RNAs in infectious and non-infectious human diseases makes them attractive therapeutic targets.  Coding and non-coding RNAs frequently fold into complex conformations which, if effectively targeted, offer opportunities to therapeutically modulate numerous cellular processes, including those linked to undruggable protein targets.  Despite the considerable skepticism as to whether RNAs can be targeted with small mol. therapeutics, overwhelming evidence suggests the challenges we are currently facing are not outside the realm of possibility.  In this review, we highlight the most recent advances in mol. techniques that have sparked a revolution in understanding the RNA structure-to-function relationship.  We bring attention to the application of these modern techniques to identify druggable RNA targets and to assess small mol. binding specificity.  Finally, we discuss novel screening methodologies that support RNA drug discovery and present examples of therapeutically valuable RNA targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJ_ZttV3TINLVg90H21EOLACvtfcHk0lgE0VAIf0E19g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXntFCiu74%253D&md5=5c2197acf2f0a04f6fda7711fb3cc0a9</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2019.03.057&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2019.03.057%26sid%3Dliteratum%253Aachs%26aulast%3DSztuba-Solinska%26aufirst%3DJ.%26aulast%3DChavez-Calvillo%26aufirst%3DG.%26aulast%3DCline%26aufirst%3DS.%2BE.%26atitle%3DUnveiling%2520the%2520druggable%2520RNA%2520targets%2520and%2520small%2520molecule%2520therapeutics%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2019%26volume%3D27%26spage%3D2149%26epage%3D2165%26doi%3D10.1016%2Fj.bmc.2019.03.057" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kligun, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mandel-Gutfreund, Y.</span></span> <span> </span><span class="NLM_article-title">Conformational readout of RNA by small ligands</span>. <i>RNA Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">981</span>â <span class="NLM_lpage">989</span>, <span class="refDoi">Â DOI: 10.4161/rna.24682</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.4161%2Frna.24682" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjsVWl" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2013&pages=981-989&author=E.+Kligunauthor=Y.+Mandel-Gutfreund&title=Conformational+readout+of+RNA+by+small+ligands&doi=10.4161%2Frna.24682"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Conformational readout of RNA by small ligands</span></div><div class="casAuthors">Kligun, Efrat; Mandel-Gutfreund, Yael</div><div class="citationInfo"><span class="NLM_cas:title">RNA Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">981-989</span>CODEN:
                <span class="NLM_cas:coden">RBNIBE</span>;
        ISSN:<span class="NLM_cas:issn">1547-6286</span>.
    
            (<span class="NLM_cas:orgname">Landes Bioscience</span>)
        </div><div class="casAbstract">RNA mols. have highly versatile structures that can fold into myriad conformations, providing many potential pockets for binding small mols.  The increasing no. of available RNA structures, in complex with proteins, small ligands and in free form, enables the design of new therapeutically useful RNA-binding ligands.  Here we studied RNA ligand complexes from 10 RNA groups extd. from the protein data bank (PDB), including adaptive and non-adaptive complexes.  We analyzed the chem., phys., structural and conformational properties of binding pockets around the ligand.  Comparing the properties of ligand-binding pockets to the properties of computed pockets extd. from all available RNA structures and RNA-protein interfaces, revealed that ligand-binding pockets, mainly the adaptive pockets, are characterized by unique properties, specifically enriched in rare conformations of the nucleobase and the sugar pucker.  Further, we demonstrate that nucleotides possessing the rare conformations are preferentially involved in direct interactions with the ligand.  Overall, based on our comprehensive anal. of RNA-ligand complexes, we suggest that the unique conformations adopted by RNA nucleotides play an important role in RNA recognition by small ligands.  We term the recognition of a binding site by a ligand via the unique RNA conformations "RNA conformational readout.".  We propose that "conformational readout" is a general way by which RNA binding pockets are recognized and selected from an ensemble of different RNA states.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8RXanE2P-37Vg90H21EOLACvtfcHk0lhx8xTT46Gcag"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjsVWl&md5=0e6063ef9c5e97a55113b478dedd099b</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.4161%2Frna.24682&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252Frna.24682%26sid%3Dliteratum%253Aachs%26aulast%3DKligun%26aufirst%3DE.%26aulast%3DMandel-Gutfreund%26aufirst%3DY.%26atitle%3DConformational%2520readout%2520of%2520RNA%2520by%2520small%2520ligands%26jtitle%3DRNA%2520Biol.%26date%3D2013%26volume%3D10%26spage%3D981%26epage%3D989%26doi%3D10.4161%2Frna.24682" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kondo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Westhof, E.</span></span> <span> </span><span class="NLM_article-title">Base pairs and pseudo pairs observed in RNA-ligand complexes</span>. <i>J. Mol. Recognit.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">241</span>â <span class="NLM_lpage">252</span>, <span class="refDoi">Â DOI: 10.1002/jmr.978</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1002%2Fjmr.978" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2009&pages=241-252&author=J.+Kondoauthor=E.+Westhof&title=Base+pairs+and+pseudo+pairs+observed+in+RNA-ligand+complexes&doi=10.1002%2Fjmr.978"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1002%2Fjmr.978&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjmr.978%26sid%3Dliteratum%253Aachs%26aulast%3DKondo%26aufirst%3DJ.%26aulast%3DWesthof%26aufirst%3DE.%26atitle%3DBase%2520pairs%2520and%2520pseudo%2520pairs%2520observed%2520in%2520RNA-ligand%2520complexes%26jtitle%3DJ.%2520Mol.%2520Recognit.%26date%3D2009%26volume%3D23%26spage%3D241%26epage%3D252%26doi%3D10.1002%2Fjmr.978" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Angelbello, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Childs-Disney, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span> <span> </span><span class="NLM_article-title">Using genome sequence to enable the design of medicines and chemical probes</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>118</i></span>,  <span class="NLM_fpage">1599</span>â <span class="NLM_lpage">1663</span>, <span class="refDoi">Â DOI: 10.1021/acs.chemrev.7b00504</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.7b00504" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC1cXlvFGqsQ%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=118&publication_year=2018&pages=1599-1663&author=A.+J.+Angelbelloauthor=J.+L.+Chenauthor=J.+L.+Childs-Disneyauthor=P.+Zhangauthor=Z.+F.+Wangauthor=M.+D.+Disney&title=Using+genome+sequence+to+enable+the+design+of+medicines+and+chemical+probes&doi=10.1021%2Facs.chemrev.7b00504"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Using Genome Sequence to Enable the Design of Medicines and Chemical Probes</span></div><div class="casAuthors">Angelbello, Alicia J.; Chen, Jonathan L.; Childs-Disney, Jessica L.; Zhang, Peiyuan; Wang, Zi-Fu; Disney, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">118</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1599-1663</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Rapid progress in genome sequencing technol. has put us firmly into a postgenomic era.  A key challenge in biomedical research is harnessing genome sequence to fulfill the promise of personalized medicine.  This Review describes how genome sequencing has enabled the identification of disease-causing biomols. and how these data have been converted into chem. probes of function, preclin. lead modalities, and ultimately U.S.  Food and Drug Administration (FDA)-approved drugs.  In particular, the authors focus on the use of oligonucleotide-based modalities to target disease-causing RNAs; small mols. that target DNA, RNA, or protein; the rational repurposing of known therapeutic modalities; and the advantages of pharmacogenetics.  Lastly, the authors discuss the remaining challenges and opportunities in the direct utilization of genome sequence to enable design of medicines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreyX8o6MEWwLVg90H21EOLACvtfcHk0lhx8xTT46Gcag"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXlvFGqsQ%253D%253D&md5=6cf4911da55d4304d7c818f0f7a4a6a0</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.7b00504&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.7b00504%26sid%3Dliteratum%253Aachs%26aulast%3DAngelbello%26aufirst%3DA.%2BJ.%26aulast%3DChen%26aufirst%3DJ.%2BL.%26aulast%3DChilds-Disney%26aufirst%3DJ.%2BL.%26aulast%3DZhang%26aufirst%3DP.%26aulast%3DWang%26aufirst%3DZ.%2BF.%26aulast%3DDisney%26aufirst%3DM.%2BD.%26atitle%3DUsing%2520genome%2520sequence%2520to%2520enable%2520the%2520design%2520of%2520medicines%2520and%2520chemical%2520probes%26jtitle%3DChem.%2520Rev.%26date%3D2018%26volume%3D118%26spage%3D1599%26epage%3D1663%26doi%3D10.1021%2Facs.chemrev.7b00504" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Di
Giorgio, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duca, M.</span></span> <span> </span><span class="NLM_article-title">Synthetic small-molecule RNA ligands: future prospects as therapeutic agents</span>. <i>MedChemComm</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1242</span>â <span class="NLM_lpage">1255</span>, <span class="refDoi">Â DOI: 10.1039/C9MD00195F</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1039%2FC9MD00195F" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=31534649" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC1MXos1ShurY%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=1242-1255&author=A.+Di%0AGiorgioauthor=M.+Duca&title=Synthetic+small-molecule+RNA+ligands%3A+future+prospects+as+therapeutic+agents&doi=10.1039%2FC9MD00195F"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Synthetic small-molecule RNA ligands: future prospects as therapeutic agents</span></div><div class="casAuthors">Di Giorgio, A.; Duca, M.</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1242-1255</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">RNA is one of the most intriguing and promising biol. targets for the discovery of innovative drugs in many pathologies and various biol. relevant RNAs that could serve as drug targets have already been identified.  Among the most important ones, one can mention prokaryotic rRNA which is the target of several marketed antibiotics, viral RNAs or oncogenic microRNAs that are tightly involved in the development and progression of various cancers.  Oligonucleotides are efficient and specific RNA targeting agents but suffer from poor pharmacodynamic and pharmacokinetic properties.  For this reason, a no. of synthetic small-mol. ligands have been identified and studied upon screening of chem. libraries or focused design of RNA binders.  In this review, we report the most relevant examples of synthetic compds. bearing sufficient selectivity to envisage clin. studies and future therapeutic applications with a particular attention for the main strategies that can be undertaken toward the improvement of selectivity and biol. activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-wwNyCsBkOLVg90H21EOLACvtfcHk0ljkpxHo4izGfA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXos1ShurY%253D&md5=60f47cc3df704517d5c308ebbe2cae6b</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1039%2FC9MD00195F&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC9MD00195F%26sid%3Dliteratum%253Aachs%26aulast%3DDi%2BGiorgio%26aufirst%3DA.%26aulast%3DDuca%26aufirst%3DM.%26atitle%3DSynthetic%2520small-molecule%2520RNA%2520ligands%253A%2520future%2520prospects%2520as%2520therapeutic%2520agents%26jtitle%3DMedChemComm%26date%3D2019%26volume%3D10%26spage%3D1242%26epage%3D1255%26doi%3D10.1039%2FC9MD00195F" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Connelly, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moon, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneekloth, J. S.</span></span> <span> </span><span class="NLM_article-title">The emerging role of RNA as a therapeutic target for small molecules</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">1077</span>â <span class="NLM_lpage">1090</span>, <span class="refDoi">Â DOI: 10.1016/j.chembiol.2016.05.021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1016%2Fj.chembiol.2016.05.021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=27593111" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVOqsL3E" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2016&pages=1077-1090&author=C.+M.+Connellyauthor=M.+H.+Moonauthor=J.+S.+Schneekloth&title=The+emerging+role+of+RNA+as+a+therapeutic+target+for+small+molecules&doi=10.1016%2Fj.chembiol.2016.05.021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">The Emerging Role of RNA as a Therapeutic Target for Small Molecules</span></div><div class="casAuthors">Connelly, Colleen M.; Moon, Michelle H.; Schneekloth, John S., Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1077-1090</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Recent advances in understanding different RNAs and unique features of their biol. have revealed a wealth of information.  However, approaches to identify small mols. that target these newly discovered regulatory elements have been lacking.  The application of new biochem. screening and design-based technologies, coupled with a resurgence of interest in phenotypic screening, has resulted in several compelling successes in targeting RNA.  A no. of recent advances suggest that achieving the long-standing goal of developing drug-like, biol. active small mols. that target RNA is possible.  This review highlights advances and successes in approaches to targeting RNA with diverse small mols., and the potential for these technologies to pave the way to new types of RNA-targeted therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqdPbbvOw7jO7Vg90H21EOLACvtfcHk0ljkpxHo4izGfA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVOqsL3E&md5=800ffb1879c29aa7ac34940e761b5f7e</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2016.05.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2016.05.021%26sid%3Dliteratum%253Aachs%26aulast%3DConnelly%26aufirst%3DC.%2BM.%26aulast%3DMoon%26aufirst%3DM.%2BH.%26aulast%3DSchneekloth%26aufirst%3DJ.%2BS.%26atitle%3DThe%2520emerging%2520role%2520of%2520RNA%2520as%2520a%2520therapeutic%2520target%2520for%2520small%2520molecules%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2016%26volume%3D23%26spage%3D1077%26epage%3D1090%26doi%3D10.1016%2Fj.chembiol.2016.05.021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Aboul-Ela, F.</span></span> <span> </span><span class="NLM_article-title">Strategies for the design of RNA-binding small molecules</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">93</span>â <span class="NLM_lpage">119</span>, <span class="refDoi">Â DOI: 10.4155/fmc.09.149</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.4155%2Ffmc.09.149" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=21426048" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC3cXosVWjtg%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2010&pages=93-119&author=F.+Aboul-Ela&title=Strategies+for+the+design+of+RNA-binding+small+molecules&doi=10.4155%2Ffmc.09.149"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Strategies for the design of RNA-binding small molecules</span></div><div class="casAuthors">Aboul-Ela, Fareed</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">93-119</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">A review.  Bacterial rRNA is the target of clin. important antibiotics, while biol. important RNAs in viral and eukaryotic genomes present a range of potential drug targets.  The physicochem. properties of RNA present difficulties for medicinal chem., particularly when oral availability is needed.  Peptidic ligands and anal. of their RNA-binding properties are providing insight into RNA recognition.  RNA-binding ligands include far more chem. classes than just aminoglycosides.  Chem. functionalities from known RNA-binding small mols. are being exploited in fragment- and ligand-based projects.  While targeting of RNA for drug design is very challenging, continuing advances in our understanding of the principles of RNA-ligand interaction will be necessary to realize the full potential of this class of targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWWGVYo9p2_bVg90H21EOLACvtfcHk0ljkpxHo4izGfA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXosVWjtg%253D%253D&md5=18b891dc9e0af5f84b7a2e621622d52a</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.4155%2Ffmc.09.149&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc.09.149%26sid%3Dliteratum%253Aachs%26aulast%3DAboul-Ela%26aufirst%3DF.%26atitle%3DStrategies%2520for%2520the%2520design%2520of%2520RNA-binding%2520small%2520molecules%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2010%26volume%3D2%26spage%3D93%26epage%3D119%26doi%3D10.4155%2Ffmc.09.149" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Costales, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Childs-Disney, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haniff, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span> <span> </span><span class="NLM_article-title">How we think about targeting RNA with small molecules</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">8880</span>â <span class="NLM_lpage">8900</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.9b01927</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01927" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BB3cXlsFygsr8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=8880-8900&author=M.+G.+Costalesauthor=J.+L.+Childs-Disneyauthor=H.+S.+Haniffauthor=M.+D.+Disney&title=How+we+think+about+targeting+RNA+with+small+molecules&doi=10.1021%2Facs.jmedchem.9b01927"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">How We Think about Targeting RNA with Small Molecules</span></div><div class="casAuthors">Costales, Matthew G.; Childs-Disney, Jessica L.; Haniff, Hafeez S.; Disney, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">8880-8900</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  RNA offers nearly unlimited potential as a target for small mol. chem. probes and lead medicines.  Many RNAs fold into structures that can be selectively targeted with small mols.  This Perspective discusses mol. recognition of RNA by small mols. and highlights key enabling technologies and properties of bioactive interactions.  Sequence-based design of ligands targeting RNA has established rules for affecting RNA targets and provided a potentially general platform for the discovery of bioactive small mols.  The RNA targets that contain preferred small mol. binding sites can be identified from sequence, allowing identification of off-targets and prediction of bioactive interactions by nature of ligand recognition of functional sites.  Small mol. targeted degrdn. of RNA targets (RNase-targeted chimeras, RIBOTACs) and direct cleavage by small mols. have also been developed.  These growing technologies suggest that the time is right to provide small mol. chem. probes to target functionally relevant RNAs throughout the human transcriptome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpsFk_gFhxgTbVg90H21EOLACvtfcHk0lj7aMhaCg3I-g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXlsFygsr8%253D&md5=4fe34106689d2d6d38814d79357ebd1a</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01927%26sid%3Dliteratum%253Aachs%26aulast%3DCostales%26aufirst%3DM.%2BG.%26aulast%3DChilds-Disney%26aufirst%3DJ.%2BL.%26aulast%3DHaniff%26aufirst%3DH.%2BS.%26aulast%3DDisney%26aufirst%3DM.%2BD.%26atitle%3DHow%2520we%2520think%2520about%2520targeting%2520RNA%2520with%2520small%2520molecules%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D8880%26epage%3D8900%26doi%3D10.1021%2Facs.jmedchem.9b01927" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ursu, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Childs-Disney, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andrews, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">OâLeary, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Angelbello, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moss, W. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span> <span> </span><span class="NLM_article-title">Design of small molecules targeting RNA structure from sequence</span>. <i>Chem. Soc. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">7252</span>â <span class="NLM_lpage">7270</span>, <span class="refDoi">Â DOI: 10.1039/D0CS00455C</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1039%2FD0CS00455C" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=32935689" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhvVClur7M" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2020&pages=7252-7270&author=A.+Ursuauthor=J.+L.+Childs-Disneyauthor=R.+J.+Andrewsauthor=C.+A.+O%E2%80%99Learyauthor=S.+M.+Meyerauthor=A.+J.+Angelbelloauthor=W.+N.+Mossauthor=M.+D.+Disney&title=Design+of+small+molecules+targeting+RNA+structure+from+sequence&doi=10.1039%2FD0CS00455C"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Design of small molecules targeting RNA structure from sequence</span></div><div class="casAuthors">Ursu, Andrei; Childs-Disney, Jessica L.; Andrews, Ryan J.; O'Leary, Collin A.; Meyer, Samantha M.; Angelbello, Alicia J.; Moss, Walter N.; Disney, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Society Reviews</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">7252-7270</span>CODEN:
                <span class="NLM_cas:coden">CSRVBR</span>;
        ISSN:<span class="NLM_cas:issn">0306-0012</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  The design and discovery of small mol. medicines has largely been focused on a small no. of druggable protein families.  A new paradigm is emerging, however, in which small mols. exert a biol. effect by interacting with RNA, both to study human disease biol. and provide lead therapeutic modalities.  Due to this potential for expanding target pipelines and treating a larger no. of human diseases, robust platforms for the rational design and optimization of small mols. interacting with RNAs (SMIRNAs) are in high demand.  This review highlights three major pillars in this area.  First, the transcriptome-wide identification and validation of structured RNA elements, or motifs, within disease-causing RNAs directly from sequence is presented.  Second, we provide an overview of high-throughput screening approaches to identify SMIRNAs as well as discuss the lead identification strategy, Inforna, which decodes the three-dimensional (3D) conformation of RNA motifs with small mol. binding partners, directly from sequence.  An emphasis is placed on target validation methods to study the causality between modulating the RNA motif in vitro and the phenotypic outcome in cells.  Third, emergent modalities that convert occupancy-driven mode of action SMIRNAs into event-driven small mol. chem. probes, such as RNA cleavers and degraders, are presented.  Finally, the future of the small mol. RNA therapeutics field is discussed, as well as hurdles to overcome to develop potent and selective RNA-centric chem. probes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqSSSsDhiFFmLVg90H21EOLACvtfcHk0lj7aMhaCg3I-g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhvVClur7M&md5=9823258172b4a8392c49c5da87a23ae6</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1039%2FD0CS00455C&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FD0CS00455C%26sid%3Dliteratum%253Aachs%26aulast%3DUrsu%26aufirst%3DA.%26aulast%3DChilds-Disney%26aufirst%3DJ.%2BL.%26aulast%3DAndrews%26aufirst%3DR.%2BJ.%26aulast%3DO%25E2%2580%2599Leary%26aufirst%3DC.%2BA.%26aulast%3DMeyer%26aufirst%3DS.%2BM.%26aulast%3DAngelbello%26aufirst%3DA.%2BJ.%26aulast%3DMoss%26aufirst%3DW.%2BN.%26aulast%3DDisney%26aufirst%3DM.%2BD.%26atitle%3DDesign%2520of%2520small%2520molecules%2520targeting%2520RNA%2520structure%2520from%2520sequence%26jtitle%3DChem.%2520Soc.%2520Rev.%26date%3D2020%26volume%3D49%26spage%3D7252%26epage%3D7270%26doi%3D10.1039%2FD0CS00455C" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Falese, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donlic, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hargrove, A. E.</span></span> <span> </span><span class="NLM_article-title">Targeting RNA with small molecules: from fundamental principles towards the clinic</span>. <i>Chem. Soc. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">2224</span>â <span class="NLM_lpage">2243</span>, <span class="refDoi">Â DOI: 10.1039/D0CS01261K</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1039%2FD0CS01261K" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=33458725" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BB3MXhsVOhu7Y%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2021&pages=2224-2243&author=J.+P.+Faleseauthor=A.+Donlicauthor=A.+E.+Hargrove&title=Targeting+RNA+with+small+molecules%3A+from+fundamental+principles+towards+the+clinic&doi=10.1039%2FD0CS01261K"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting RNA with small molecules: from fundamental principles towards the clinic</span></div><div class="casAuthors">Falese, James P.; Donlic, Anita; Hargrove, Amanda E.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Society Reviews</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">2224-2243</span>CODEN:
                <span class="NLM_cas:coden">CSRVBR</span>;
        ISSN:<span class="NLM_cas:issn">0306-0012</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  Recent advances in our understanding of RNA biol. have uncovered crucial roles for RNA in multiple disease states, ranging from viral and bacterial infections to cancer and neurol. disorders.  As a result, multiple labs. have become interested in developing drug-like small mols. to target RNA.  However, this development comes with multiple unique challenges.  For example, RNA is inherently dynamic and has limited chem. diversity.  This Tutorial Review overviews important considerations and advancements for generating RNA-targeted small mols., ranging from fundamental chem. to promising small mol. examples with demonstrated clin. efficacy.  We then discuss fundamental RNA recognition principles along with methods for small mol. screening and RNA structure detn.  Finally, we review unique challenges and emerging solns. from both the RNA and small mol. perspectives for generating RNA-targeted ligands before highlighting a selection of the "Greatest Hits" to date.  Addnl., we explore the recently FDA-approved small mol. regulator of RNA splicing, risdiplam, for treatment of spinal muscular atrophy.  Together, this Tutorial Review showcases the fundamental role of chem. and mol. recognition principles in enhancing our understanding of RNA biol. and contributing to the rapidly growing no. of RNA-targeted probes and therapeutics.  In particular, we hope this widely accessible review will serve as inspiration for aspiring small mol. and/or RNA researchers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPsmlpE72GLrVg90H21EOLACvtfcHk0lhFVhx4trl9qg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXhsVOhu7Y%253D&md5=40f76d282a164fe9409f4fc45e81b0cf</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1039%2FD0CS01261K&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FD0CS01261K%26sid%3Dliteratum%253Aachs%26aulast%3DFalese%26aufirst%3DJ.%2BP.%26aulast%3DDonlic%26aufirst%3DA.%26aulast%3DHargrove%26aufirst%3DA.%2BE.%26atitle%3DTargeting%2520RNA%2520with%2520small%2520molecules%253A%2520from%2520fundamental%2520principles%2520towards%2520the%2520clinic%26jtitle%3DChem.%2520Soc.%2520Rev.%26date%3D2021%26volume%3D50%26spage%3D2224%26epage%3D2243%26doi%3D10.1039%2FD0CS01261K" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ameres, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zamore, P. D.</span></span> <span> </span><span class="NLM_article-title">Diversifying microRNA sequence and function</span>. <i>Nat. Rev. Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">475</span>â <span class="NLM_lpage">488</span>, <span class="refDoi">Â DOI: 10.1038/nrm3611</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1038%2Fnrm3611" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=23800994" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtVShtrnK" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2013&pages=475-488&author=S.+L.+Ameresauthor=P.+D.+Zamore&title=Diversifying+microRNA+sequence+and+function&doi=10.1038%2Fnrm3611"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Diversifying microRNA sequence and function</span></div><div class="casAuthors">Ameres, Stefan L.; Zamore, Phillip D.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">475-488</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  MicroRNAs (miRNAs) regulate the expression of most genes in animals, but we are only now beginning to understand how they are generated, assembled into functional complexes and destroyed.  Various mechanisms have now been identified that regulate miRNA stability and that diversify miRNA sequences to create distinct isoforms.  The prodn. of different isoforms of individual miRNAs in specific cells and tissues may have broader implications for miRNA-mediated gene expression control.  Rigorously testing the many discrepant models for how miRNAs function using quant. biochem. measurements made in vivo and in vitro remains a major challenge for the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgqnSfNVGvZ7Vg90H21EOLACvtfcHk0lhFVhx4trl9qg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtVShtrnK&md5=d81c4d7a8d42eda6b1f6470f05f7b021</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1038%2Fnrm3611&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm3611%26sid%3Dliteratum%253Aachs%26aulast%3DAmeres%26aufirst%3DS.%2BL.%26aulast%3DZamore%26aufirst%3DP.%2BD.%26atitle%3DDiversifying%2520microRNA%2520sequence%2520and%2520function%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2013%26volume%3D14%26spage%3D475%26epage%3D488%26doi%3D10.1038%2Fnrm3611" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bartel, D. P.</span></span> <span> </span><span class="NLM_article-title">MicroRNAs: genomics, biogenesis, mechanism, and function</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>116</i></span>,  <span class="NLM_fpage">281</span>â <span class="NLM_lpage">297</span>, <span class="refDoi">Â DOI: 10.1016/S0092-8674(04)00045-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1016%2FS0092-8674%2804%2900045-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=14744438" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVals7o%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=2004&pages=281-297&author=D.+P.+Bartel&title=MicroRNAs%3A+genomics%2C+biogenesis%2C+mechanism%2C+and+function&doi=10.1016%2FS0092-8674%2804%2900045-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">MicroRNAs: Genomics, biogenesis, mechanism, and function</span></div><div class="casAuthors">Bartel, David P.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">116</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">281-297</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  MicroRNAs (miRNAs) are endogenous â¼22 nt RNAs that can play important regulatory roles in animals and plants by targeting mRNAs for cleavage or translational repression.  Although they escaped notice until relatively recently, miRNAs comprise one of the more abundant classes of gene regulatory mols. in multicellular organisms and likely influence the output of many protein-coding genes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqzPI7QCGHbXbVg90H21EOLACvtfcHk0lhFVhx4trl9qg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVals7o%253D&md5=9470974cbee3731cd29c7e204490b529</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2FS0092-8674%2804%2900045-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0092-8674%252804%252900045-5%26sid%3Dliteratum%253Aachs%26aulast%3DBartel%26aufirst%3DD.%2BP.%26atitle%3DMicroRNAs%253A%2520genomics%252C%2520biogenesis%252C%2520mechanism%252C%2520and%2520function%26jtitle%3DCell%26date%3D2004%26volume%3D116%26spage%3D281%26epage%3D297%26doi%3D10.1016%2FS0092-8674%2804%2900045-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Volinia, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calin, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ambs, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cimmino, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petrocca, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Visone, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iorio, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roldo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferracin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prueitt, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yanaihara, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lanza, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scarpa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vecchione, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Negrini, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Croce, C. M.</span></span> <span> </span><span class="NLM_article-title">A microRNA expression signature of human solid tumors defines cancer gene targets</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>103</i></span>,  <span class="NLM_fpage">2257</span>â <span class="NLM_lpage">2261</span>, <span class="refDoi">Â DOI: 10.1073/pnas.0510565103</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1073%2Fpnas.0510565103" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=16461460" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BD28XhslGjsbg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=2006&pages=2257-2261&author=S.+Voliniaauthor=G.+A.+Calinauthor=C.+G.+Liuauthor=S.+Ambsauthor=A.+Cimminoauthor=F.+Petroccaauthor=R.+Visoneauthor=M.+Iorioauthor=C.+Roldoauthor=M.+Ferracinauthor=R.+L.+Prueittauthor=N.+Yanaiharaauthor=G.+Lanzaauthor=A.+Scarpaauthor=A.+Vecchioneauthor=M.+Negriniauthor=C.+C.+Harrisauthor=C.+M.+Croce&title=A+microRNA+expression+signature+of+human+solid+tumors+defines+cancer+gene+targets&doi=10.1073%2Fpnas.0510565103"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">A microRNA expression signature of human solid tumors defines cancer gene targets</span></div><div class="casAuthors">Volinia, Stefano; Calin, George A.; Liu, Chang-Gong; Ambs, Stefan; Cimmino, Amelia; Petrocca, Fabio; Visone, Rosa; Iorio, Marilena; Roldo, Claudia; Ferracin, Manuela; Prueitt, Robyn L.; Yanaihara, Nozumu; Lanza, Giovanni; Scarpa, Aldo; Vecchione, Andrea; Negrini, Massimo; Harris, Curtis C.; Croce, Carlo M.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">103</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2257-2261</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Small noncoding microRNAs (miRNAs) can contribute to cancer development and progression and are differentially expressed in normal tissues and cancers.  From a large-scale miRnome anal. on 540 samples including lung, breast, stomach, prostate, colon, and pancreatic tumors, we identified a solid cancer miRNA signature composed by a large portion of overexpressed miRNAs.  Among these miRNAs are some with well characterized cancer assocn., such as miR-17-5p, miR-20a, miR-21, miR-92, miR-106a, and miR-155.  The predicted targets for the differentially expressed miRNAs are significantly enriched for protein-coding tumor suppressors and oncogenes (P < 0.0001).  A no. of the predicted targets, including the tumor suppressors RB1 (Retinoblastoma 1) and TGFBR2 (transforming growth factor, beta receptor II) genes were confirmed exptl.  Our results indicate that miRNAs are extensively involved in cancer pathogenesis of solid tumors and support their function as either dominant or recessive cancer genes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpSPx3elVKg6LVg90H21EOLACvtfcHk0ljKXYoG82iKPg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhslGjsbg%253D&md5=e2b19f23e04996312fa922d76a4f35bc</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0510565103&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0510565103%26sid%3Dliteratum%253Aachs%26aulast%3DVolinia%26aufirst%3DS.%26aulast%3DCalin%26aufirst%3DG.%2BA.%26aulast%3DLiu%26aufirst%3DC.%2BG.%26aulast%3DAmbs%26aufirst%3DS.%26aulast%3DCimmino%26aufirst%3DA.%26aulast%3DPetrocca%26aufirst%3DF.%26aulast%3DVisone%26aufirst%3DR.%26aulast%3DIorio%26aufirst%3DM.%26aulast%3DRoldo%26aufirst%3DC.%26aulast%3DFerracin%26aufirst%3DM.%26aulast%3DPrueitt%26aufirst%3DR.%2BL.%26aulast%3DYanaihara%26aufirst%3DN.%26aulast%3DLanza%26aufirst%3DG.%26aulast%3DScarpa%26aufirst%3DA.%26aulast%3DVecchione%26aufirst%3DA.%26aulast%3DNegrini%26aufirst%3DM.%26aulast%3DHarris%26aufirst%3DC.%2BC.%26aulast%3DCroce%26aufirst%3DC.%2BM.%26atitle%3DA%2520microRNA%2520expression%2520signature%2520of%2520human%2520solid%2520tumors%2520defines%2520cancer%2520gene%2520targets%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2006%26volume%3D103%26spage%3D2257%26epage%3D2261%26doi%3D10.1073%2Fpnas.0510565103" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stenvang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petri, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindow, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Obad, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kauppinen, S.</span></span> <span> </span><span class="NLM_article-title">Inhibition of microRNA function by antimiR oligonucleotides</span>. <i>Silence</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">1</span>, <span class="refDoi">Â DOI: 10.1186/1758-907X-3-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1186%2F1758-907X-3-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=22230293" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC38Xlt1Cit7w%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2012&pages=1&author=J.+Stenvangauthor=A.+Petriauthor=M.+Lindowauthor=S.+Obadauthor=S.+Kauppinen&title=Inhibition+of+microRNA+function+by+antimiR+oligonucleotides&doi=10.1186%2F1758-907X-3-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of microRNA function by antimiR oligonucleotides</span></div><div class="casAuthors">Stenvang, Jan; Petri, Andreas; Lindow, Morten; Obad, Susanna; Kauppinen, Sakari</div><div class="citationInfo"><span class="NLM_cas:title">Silence</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1</span>CODEN:
                <span class="NLM_cas:coden">SILEBI</span>;
        ISSN:<span class="NLM_cas:issn">1758-907X</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">A review.  MicroRNAs (miRNAs) have emerged as important post-transcriptional regulators of gene expression in many developmental and cellular processes.  Moreover, there is now ample evidence that perturbations in the levels of individual or entire families of miRNAs are strongly assocd. with the pathogenesis of a wide range of human diseases.  Indeed, disease-assocd. miRNAs represent a new class of targets for the development of miRNA-based therapeutic modalities, which may yield patient benefits unobtainable by other therapeutic approaches.  The recent explosion in miRNA research has accelerated the development of several computational and exptl. approaches for probing miRNA functions in cell culture and in vivo.  In this review, we focus on the use of antisense oligonucleotides (antimiRs) in miRNA inhibition for loss-of-function studies.  We provide an overview of the currently employed antisense chemistries and their utility in designing antimiR oligonucleotides.  Furthermore, we describe the most commonly used in vivo delivery strategies and discuss different approaches for assessment of miRNA inhibition and potential off-target effects.  Finally, we summarize recent progress in antimiR mediated pharmacol. inhibition of disease-assocd. miRNAs, which shows great promise in the development of novel miRNA-based therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxHTr8fhW5n7Vg90H21EOLACvtfcHk0ljKXYoG82iKPg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xlt1Cit7w%253D&md5=897115eb2c4cc654fdabbdd45cc4e58d</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1186%2F1758-907X-3-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1758-907X-3-1%26sid%3Dliteratum%253Aachs%26aulast%3DStenvang%26aufirst%3DJ.%26aulast%3DPetri%26aufirst%3DA.%26aulast%3DLindow%26aufirst%3DM.%26aulast%3DObad%26aufirst%3DS.%26aulast%3DKauppinen%26aufirst%3DS.%26atitle%3DInhibition%2520of%2520microRNA%2520function%2520by%2520antimiR%2520oligonucleotides%26jtitle%3DSilence%26date%3D2012%26volume%3D3%26spage%3D1%26doi%3D10.1186%2F1758-907X-3-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Juliano, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ming, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakagawa, O.</span></span> <span> </span><span class="NLM_article-title">The chemistry and biology of oligonucleotide conjugates</span>. <i>Acc. Chem. Res.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">1067</span>â <span class="NLM_lpage">1076</span>, <span class="refDoi">Â DOI: 10.1021/ar2002123</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ar2002123" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC38XislSqu7s%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2012&pages=1067-1076&author=R.+L.+Julianoauthor=X.+Mingauthor=O.+Nakagawa&title=The+chemistry+and+biology+of+oligonucleotide+conjugates&doi=10.1021%2Far2002123"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">The Chemistry and Biology of Oligonucleotide Conjugates</span></div><div class="casAuthors">Juliano, R. L.; Ming, Xin; Nakagawa, Osamu</div><div class="citationInfo"><span class="NLM_cas:title">Accounts of Chemical Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1067-1076</span>CODEN:
                <span class="NLM_cas:coden">ACHRE4</span>;
        ISSN:<span class="NLM_cas:issn">0001-4842</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Short DNA or RNA oligonucleotides have tremendous potential as therapeutic agents.  Because of their ability to engage in Watson-Crick base pairing, they can interact with mRNA or pre-mRNA targets with high selectivity.  As a result, they could precisely manipulate gene expression.  This possibility has engendered extensive efforts to develop oligonucleotides as drugs, and many candidates are already in clin. trials.  However, a major impediment to the maturation of this field of oligonucleotide-based therapeutics remains: these relatively large and often highly charged mols. don't easily cross cellular membranes, making it difficult for them to reach their sites of action in the cytosol or nucleus.  In this Account, we summarize some basic features of the biol. of antisense and siRNA oligonucleotides.  We then discuss chem. conjugation as an approach to improving the intracellular delivery and therapeutic potential of these agents.  Instead of focusing on the details of conjugation chem., we emphasize the pharmacol. ramifications of oligonucleotide conjugates.  In one important approach to improving delivery and efficacy, researchers have conjugated oligonucleotides with ligands designed to bind to particular receptors and thus provide specific interactions with cells.  In another strategy, researchers have coupled antisense or siRNA with agents such as cell penetrating peptides that are designed to provoke escape of the conjugate from intracellular vesicular compartments.  Although both of these strategies have had some success, further research is needed before oligonucleotide conjugates can find an important place in human therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqOVDWra3YKhLVg90H21EOLACvtfcHk0lj-geZC4myCRQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XislSqu7s%253D&md5=e6eecf9a41ee6c31acbe1018eb655d56</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Far2002123&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Far2002123%26sid%3Dliteratum%253Aachs%26aulast%3DJuliano%26aufirst%3DR.%2BL.%26aulast%3DMing%26aufirst%3DX.%26aulast%3DNakagawa%26aufirst%3DO.%26atitle%3DThe%2520chemistry%2520and%2520biology%2520of%2520oligonucleotide%2520conjugates%26jtitle%3DAcc.%2520Chem.%2520Res.%26date%3D2012%26volume%3D45%26spage%3D1067%26epage%3D1076%26doi%3D10.1021%2Far2002123" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Connelly, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boer, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moon, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gareiss, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneekloth, J. S.</span></span> <span> </span><span class="NLM_article-title">Discovery of inhibitors of microRNA-21 processing using small molecule microarrays</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">435</span>â <span class="NLM_lpage">443</span>, <span class="refDoi">Â DOI: 10.1021/acschembio.6b00945</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.6b00945" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC28XitVKqtbjP" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=435-443&author=C.+M.+Connellyauthor=R.+E.+Boerauthor=M.+H.+Moonauthor=P.+Gareissauthor=J.+S.+Schneekloth&title=Discovery+of+inhibitors+of+microRNA-21+processing+using+small+molecule+microarrays&doi=10.1021%2Facschembio.6b00945"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Inhibitors of MicroRNA-21 Processing Using Small Molecule Microarrays</span></div><div class="casAuthors">Connelly, Colleen M.; Boer, Robert E.; Moon, Michelle H.; Gareiss, Peter; Schneekloth, John S.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">435-443</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The identification of small mols. that bind to and perturb the function of microRNAs is an attractive approach for the treatment for microRNA-assocd. pathologies.  However, there are only a few small mols. known to interact directly with microRNAs.  Here, the authors report the use of a small mol. microarray (SMM) screening approach to identify low mol. wt. compds. that directly bind to a pre-miR-21 hairpin.  Compds. identified using this approach exhibit good affinity for the RNA (ranging from 0.8-2.0 Î¼M) and are not composed of a polycationic scaffold.  Several of the highest affinity compds. inhibit Dicer-mediated processing, while in-line probing expts. indicate that the compds. bind to the apical loop of the hairpin, proximal to the Dicer site.  This work provides evidence that small mols. can be developed to bind directly to and inhibit miR-21.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq3vZ7QBA7lT7Vg90H21EOLACvtfcHk0lj-geZC4myCRQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitVKqtbjP&md5=56a3a74254fc26302496b05de395591a</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Facschembio.6b00945&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.6b00945%26sid%3Dliteratum%253Aachs%26aulast%3DConnelly%26aufirst%3DC.%2BM.%26aulast%3DBoer%26aufirst%3DR.%2BE.%26aulast%3DMoon%26aufirst%3DM.%2BH.%26aulast%3DGareiss%26aufirst%3DP.%26aulast%3DSchneekloth%26aufirst%3DJ.%2BS.%26atitle%3DDiscovery%2520of%2520inhibitors%2520of%2520microRNA-21%2520processing%2520using%2520small%2520molecule%2520microarrays%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2017%26volume%3D12%26spage%3D435%26epage%3D443%26doi%3D10.1021%2Facschembio.6b00945" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pei, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shu, Y.</span></span> <span> </span><span class="NLM_article-title">Alteration in miR-21/PTEN expression modulates gefitinib resistance in non-small cell lung cancer</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>9</i></span>, <span class="NLM_elocation-id">e103305</span> <span class="refDoi">Â DOI: 10.1371/journal.pone.0103305</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1371%2Fjournal.pone.0103305" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=25058005" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1ersrnM" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&author=H.+Shenauthor=F.+Zhuauthor=J.+Liuauthor=T.+Xuauthor=D.+Peiauthor=R.+Wangauthor=Y.+Qianauthor=Q.+Liauthor=L.+Wangauthor=Z.+Shiauthor=J.+Zhengauthor=Q.+Chenauthor=B.+Jiangauthor=Y.+Shu&title=Alteration+in+miR-21%2FPTEN+expression+modulates+gefitinib+resistance+in+non-small+cell+lung+cancer&doi=10.1371%2Fjournal.pone.0103305"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Alteration in miR-21/Pten expression modulates gefitinib resistance in non-small cell lung cancer</span></div><div class="casAuthors">Shen, Hua; Zhu, Fang; Liu, Jinyuan; Xu, Tongpeng; Pei, Dong; Wang, Rong; Qian, Yingying; Li, Qi; Wang, Lin; Shi, Zhumei; Zheng, Jitai; Chen, Qiudan; Jiang, Binghua; Shu, Yongqian</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">e103305/1-e103305/9, 9 pp.</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Resistance to TKI treatment is a major obstacle in effective treatment of NSCLC.  Besides EGFR mutation status, the mechanisms involved are largely unknown.  Some evidence supports a role for microRNA 21 in modulating drug sensitivity of chemotherapy but its role in NSCLC TKI resistance still remains unexplored.  This study aimed to investigate whether NSCLC miR-21 mediated resistance to TKIs also results from Pten targeting.  Here, we show miR-21 promotes cancer by neg. regulating Pten expression in human NSCLC tissues: high miR-21 expression levels were assocd. with shorter DFS in 47 NSCLC patients; high miR-21/low Pten expression levels indicated a poor TKI clin. response and shorter overall survival in another 46 NSCLC patients undergoing TKI treatment.  In vitro assays showed that miR-21 was up-regulated concomitantly to down-regulation of Pten in pc-9/GR cells in comparison with pc-9 cells.  Moreover, over-expression of miR-21 significantly decreased gefitinib sensitivity by down-regulating Pten expression and activating Akt and ERK pathways in pc-9 cells, while miR-21 knockdown dramatically restored gefitinib sensitivity of pc-9/GR cells by up-regulation of Pten expression and inactivation of AKT and ERK pathways, in vivo and in vitro.  We propose alteration of miR-21/Pten expression as a novel mechanism for TKI resistance in NSCLC cancer.  Our findings provide a new basis for using miR 21/Pten-based therapeutic strategies to reverse gefitinib resistance in NSCLC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJhlsVIgQXMrVg90H21EOLACvtfcHk0ljyo9R5r5hKkw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1ersrnM&md5=08dda1c3b5dcd0f5ab28777438ea6533</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0103305&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0103305%26sid%3Dliteratum%253Aachs%26aulast%3DShen%26aufirst%3DH.%26aulast%3DZhu%26aufirst%3DF.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DT.%26aulast%3DPei%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DR.%26aulast%3DQian%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DQ.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DShi%26aufirst%3DZ.%26aulast%3DZheng%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DQ.%26aulast%3DJiang%26aufirst%3DB.%26aulast%3DShu%26aufirst%3DY.%26atitle%3DAlteration%2520in%2520miR-21%252FPTEN%2520expression%2520modulates%2520gefitinib%2520resistance%2520in%2520non-small%2520cell%2520lung%2520cancer%26jtitle%3DPLoS%2520One%26date%3D2014%26volume%3D9%26doi%3D10.1371%2Fjournal.pone.0103305" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fei, J.</span></span> <span> </span><span class="NLM_article-title">Anti-miR-21 oligonucleotide enhances chemosensitivity of leukemic HL60 cells to arabinosylcytosine by inducing apoptosis</span>. <i>Hematology</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">215</span>â <span class="NLM_lpage">221</span>, <span class="refDoi">Â DOI: 10.1179/102453310X12647083620840</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1179%2F102453310X12647083620840" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=20670480" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtFajsbbL" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2010&pages=215-221&author=Y.+Liauthor=X.+Zhuauthor=J.+Guauthor=H.+Huauthor=D.+Dongauthor=J.+Yaoauthor=C.+Linauthor=J.+Fei&title=Anti-miR-21+oligonucleotide+enhances+chemosensitivity+of+leukemic+HL60+cells+to+arabinosylcytosine+by+inducing+apoptosis&doi=10.1179%2F102453310X12647083620840"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Anti-miR-21 oligonucleotide enhances chemosensitivity of leukemic HL60 cells to arabinosylcytosine by inducing apoptosis</span></div><div class="casAuthors">Li, Yumin; Zhu, Xuejiao; Gu, Jingyi; Hu, Haiyan; Dong, Dawei; Yao, Junlin; Lin, Chunyan; Fei, Jia</div><div class="citationInfo"><span class="NLM_cas:title">Hematology (Leeds, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">215-221</span>CODEN:
                <span class="NLM_cas:coden">HMATFL</span>;
        ISSN:<span class="NLM_cas:issn">1024-5332</span>.
    
            (<span class="NLM_cas:orgname">Maney Publishing</span>)
        </div><div class="casAbstract">Drug insensitivity or resistance is a major obstacle for successful treatment of acute myeloid leukemia.  MicroRNAs (miRNAs) are small non-coding RNA mols.  Increasing evidence suggests that miRNAs modulate cellular sensitivity to anticancer drugs.  We used a specific anti-miR-21 oligonucleotide (AMO-miR-21) to sensitize leukemic HL60 cells to arabinosylcytosine (Ara-C) by down-regulating miR-21.  AMO-miR-21 alone effectively inhibited HL60 cell viability as measured by MTT assays and induced apoptosis as evaluated by flow cytometry, whereas AMO-miR-21 in combination with Ara-C enhanced HL60 cells to Ara-C-sensitivity and promoted Ara-C-induced apoptosis.  Levels of miR-21 and its target PDCD4, quantified by real-time PCR, showed that expression of miR-21 was significantly decreased after AMO-miR-21 treatment.  PDCD4 as a direct target of miR-21 in leukemic HL60 cells was confirmed by the dual-luciferase reporter assay.  Our study suggests that AMO-miR-21 significantly sensitizes HL60 cells to Ara-C by inducing apoptosis and these effects of AMO-miR-21 may be partially due to its up-regulation of PDCD4.  Therefore, exploiting synergistic effects between AMO-miR-21 and Ara-C might be an effective clin. strategy for leukemia chemotherapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5RDEtManOabVg90H21EOLACvtfcHk0ljyo9R5r5hKkw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtFajsbbL&md5=89dc5e475cf062dc95a84f7a93362ed2</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1179%2F102453310X12647083620840&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1179%252F102453310X12647083620840%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DY.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DGu%26aufirst%3DJ.%26aulast%3DHu%26aufirst%3DH.%26aulast%3DDong%26aufirst%3DD.%26aulast%3DYao%26aufirst%3DJ.%26aulast%3DLin%26aufirst%3DC.%26aulast%3DFei%26aufirst%3DJ.%26atitle%3DAnti-miR-21%2520oligonucleotide%2520enhances%2520chemosensitivity%2520of%2520leukemic%2520HL60%2520cells%2520to%2520arabinosylcytosine%2520by%2520inducing%2520apoptosis%26jtitle%3DHematology%26date%3D2010%26volume%3D15%26spage%3D215%26epage%3D221%26doi%3D10.1179%2F102453310X12647083620840" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gumireddy, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hogenesch, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deiters, A.</span></span> <span> </span><span class="NLM_article-title">Small-molecule inhibitors of microRNA miR-21 function</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">7482</span>â <span class="NLM_lpage">7484</span>, <span class="refDoi">Â DOI: 10.1002/anie.200801555</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1002%2Fanie.200801555" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BD1cXht1Sjur7I" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2008&pages=7482-7484&author=K.+Gumireddyauthor=D.+D.+Youngauthor=X.+Xiongauthor=J.+B.+Hogeneschauthor=Q.+Huangauthor=A.+Deiters&title=Small-molecule+inhibitors+of+microRNA+miR-21+function&doi=10.1002%2Fanie.200801555"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Small-molecule inhibitors of microRNA miR-21 function</span></div><div class="casAuthors">Gumireddy, Kiranmai; Young, Douglas D.; Xiong, Xin; Hogenesch, John B.; Huang, Qihong; Deiters, Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">39</span>),
    <span class="NLM_cas:pages">7482-7484</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Short, but significant, microRNAs (miRNAs) are an important class of gene regulators.  Small-mol. modifiers of miRNA function, such as 1 (schematic representation), were identified in a cellular screen for miRNA-pathway inhibitors.  Such compds. are expected to be useful tools for the elucidation of detailed mechanisms of miRNA action and may serve as lead structures for the development of new therapeutic agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoegx2d4LaAqLVg90H21EOLACvtfcHk0lg3thrUXANQeA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXht1Sjur7I&md5=af016e4b3a6bfe73de80498ccfdc1bd6</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1002%2Fanie.200801555&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.200801555%26sid%3Dliteratum%253Aachs%26aulast%3DGumireddy%26aufirst%3DK.%26aulast%3DYoung%26aufirst%3DD.%2BD.%26aulast%3DXiong%26aufirst%3DX.%26aulast%3DHogenesch%26aufirst%3DJ.%2BB.%26aulast%3DHuang%26aufirst%3DQ.%26aulast%3DDeiters%26aufirst%3DA.%26atitle%3DSmall-molecule%2520inhibitors%2520of%2520microRNA%2520miR-21%2520function%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2008%26volume%3D47%26spage%3D7482%26epage%3D7484%26doi%3D10.1002%2Fanie.200801555" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, W. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, S. M.</span></span> <span> </span><span class="NLM_article-title">Discovery of 4-benzoylamino-<i>N</i>-(prop-2-yn-1-yl)benzamides as novel microRNA-21 inhibitors</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">6510</span>â <span class="NLM_lpage">6519</span>, <span class="refDoi">Â DOI: 10.1016/j.bmc.2015.08.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1016%2Fj.bmc.2015.08.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=26344589" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtlCru7vO" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2015&pages=6510-6519&author=C.+S.+Jiangauthor=X.+M.+Wangauthor=S.+Q.+Zhangauthor=L.+S.+Mengauthor=W.+H.+Zhuauthor=J.+Xuauthor=S.+M.+Lu&title=Discovery+of+4-benzoylamino-N-%28prop-2-yn-1-yl%29benzamides+as+novel+microRNA-21+inhibitors&doi=10.1016%2Fj.bmc.2015.08.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of 4-benzoylamino-N-(prop-2-yn-1-yl)benzamides as novel microRNA-21 inhibitors</span></div><div class="casAuthors">Jiang, Cong-shan; Wang, Xiao-meng; Zhang, San-qi; Meng, Lie-su; Zhu, Wen-hua; Xu, Jing; Lu, She-min</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">6510-6519</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">MicroRNA-21, as an oncogenic miRNA, has caught great attention for medicinal chemists to develop its novel inhibitors for cancer therapy.  In the present study, the authors designed 4-benzoylamino-N-(prop-2-yn-1-yl)benzamides as miR-21 inhibitor candidates on the basis of scaffold hopping.  Eighteen compds. were synthesized.  The inhibitory activities of synthesized compds. against the expression of miR-21 were evaluated using stem loop RT-qPCR and compd. I was discovered as the most potent compd., which displayed a time and concn. dependent inhibition manner.  In addn., various functional assays such as the expression of miR-21 target gene detected by Western blotting and the cell growth and apoptosis detected by flow cytometric anal. were checked in Hela (human epithelioid cervix carcinoma) and U-87 MG (human glioblastoma) cells to confirm its activity.  The results indicate that compd. I can enhance apoptosis, retard proliferation, and up-regulate PDCD4, a target protein of miR-21.  In addn., the compd. I does not influence the expression of multiple miRNAs and the genes that participate in miRNA universal biosynthesis pathway.  These results strongly support the assumption that title compds. can serve as a small mol. inhibitor of miR-21.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7D9ug6Mh3BrVg90H21EOLACvtfcHk0lg3thrUXANQeA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtlCru7vO&md5=dffa0613735eb77533a042752224c703</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2015.08.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2015.08.007%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DC.%2BS.%26aulast%3DWang%26aufirst%3DX.%2BM.%26aulast%3DZhang%26aufirst%3DS.%2BQ.%26aulast%3DMeng%26aufirst%3DL.%2BS.%26aulast%3DZhu%26aufirst%3DW.%2BH.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DLu%26aufirst%3DS.%2BM.%26atitle%3DDiscovery%2520of%25204-benzoylamino-N-%2528prop-2-yn-1-yl%2529benzamides%2520as%2520novel%2520microRNA-21%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2015%26volume%3D23%26spage%3D6510%26epage%3D6519%26doi%3D10.1016%2Fj.bmc.2015.08.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Naro, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ankenbruck, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tivon, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Connelly, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gardner, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deiters, A.</span></span> <span> </span><span class="NLM_article-title">Small molecule inhibition of microRNA miR-21 rescues chemosensitivity of renal-cell carcinoma to topotecan</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">5900</span>â <span class="NLM_lpage">5909</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.7b01891</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01891" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1yrurrF" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=5900-5909&author=Y.+Naroauthor=N.+Ankenbruckauthor=M.+Thomasauthor=Y.+Tivonauthor=C.+M.+Connellyauthor=L.+Gardnerauthor=A.+Deiters&title=Small+molecule+inhibition+of+microRNA+miR-21+rescues+chemosensitivity+of+renal-cell+carcinoma+to+topotecan&doi=10.1021%2Facs.jmedchem.7b01891"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Small Molecule Inhibition of MicroRNA miR-21 Rescues Chemosensitivity of Renal-Cell Carcinoma to Topotecan</span></div><div class="casAuthors">Naro, Yuta; Ankenbruck, Nicholas; Thomas, Meryl; Tivon, Yaniv; Connelly, Colleen M.; Gardner, Laura; Deiters, Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">5900-5909</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Chem. probes of microRNA (miRNA) function are potential tools for understanding miRNA biol. that also provide new approaches for discovering therapeutics for miRNA-assocd. diseases.  MicroRNA-21 (miR-21) is an oncogenic miRNA that is overexpressed in most cancers and has been strongly assocd. with driving chemoresistance in cancers such as renal cell carcinoma (RCC).  Using a cell-based luciferase reporter assay to screen small mols., we identified a novel inhibitor of miR-21 function.  Following structure-activity relationship studies, an optimized lead compd. demonstrated cytotoxicity in several cancer cell lines.  In a chemoresistant-RCC cell line, inhibition of miR-21 via small mol. treatment rescued the expression of tumor-suppressor proteins and sensitized cells to topotecan-induced apoptosis.  This resulted in a >10-fold improvement in topotecan activity in cell viability and clonogenic assays.  Overall, this work reports a novel small mol. inhibitor for perturbing miR-21 function and demonstrates an approach to enhancing the potency of chemotherapeutics specifically for cancers derived from oncomiR addiction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwGmy5_whHK7Vg90H21EOLACvtfcHk0lhMZMvIVgnzng"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1yrurrF&md5=2d752532ef82fcd3cbef8eaaa3238e2d</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01891&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01891%26sid%3Dliteratum%253Aachs%26aulast%3DNaro%26aufirst%3DY.%26aulast%3DAnkenbruck%26aufirst%3DN.%26aulast%3DThomas%26aufirst%3DM.%26aulast%3DTivon%26aufirst%3DY.%26aulast%3DConnelly%26aufirst%3DC.%2BM.%26aulast%3DGardner%26aufirst%3DL.%26aulast%3DDeiters%26aufirst%3DA.%26atitle%3DSmall%2520molecule%2520inhibition%2520of%2520microRNA%2520miR-21%2520rescues%2520chemosensitivity%2520of%2520renal-cell%2520carcinoma%2520to%2520topotecan%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D5900%26epage%3D5909%26doi%3D10.1021%2Facs.jmedchem.7b01891" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ankenbruck, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumbhare, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naro, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gardner, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emanuelson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deiters, A.</span></span> <span> </span><span class="NLM_article-title">Small molecule inhibition of microRNA-21 expression reduces cell viability and microtumor formation</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">3735</span>â <span class="NLM_lpage">3743</span>, <span class="refDoi">Â DOI: 10.1016/j.bmc.2019.05.044</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1016%2Fj.bmc.2019.05.044" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=31262664" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC1MXht1ynu7jP" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2019&pages=3735-3743&author=N.+Ankenbruckauthor=R.+Kumbhareauthor=Y.+Naroauthor=M.+Thomasauthor=L.+Gardnerauthor=C.+Emanuelsonauthor=A.+Deiters&title=Small+molecule+inhibition+of+microRNA-21+expression+reduces+cell+viability+and+microtumor+formation&doi=10.1016%2Fj.bmc.2019.05.044"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Small molecule inhibition of microRNA-21 expression reduces cell viability and microtumor formation</span></div><div class="casAuthors">Ankenbruck, Nicholas; Kumbhare, Rohan; Naro, Yuta; Thomas, Meryl; Gardner, Laura; Emanuelson, Cole; Deiters, Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">3735-3743</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">MicroRNAs (miRNAs) are short, non-coding RNA mols. estd. to regulate expression of a large no. of protein-coding genes and are implicated in a variety of biol. processes such as development, differentiation, proliferation, and cell survival.  Dysregulation of miRNAs has been attributed to the onset and progression of various human diseases, including cancer.  MicroRNA-21 (miR-21), one of the most established oncogenic miRNAs, is found to be upregulated in a wide range of cancers making it an attractive therapeutic target.  Employment of a luciferase-based live-cell reporter assay in a high-throughput screen of >300,000 small mols. led to the discovery of a new class of ether-amide miR-21 inhibitors.  Following a structure-activity relationship study, an optimized lead mol. was found to inhibit miR-21 transcription.  Furthermore, the inhibitor demonstrated cytotoxicity in a cervical cancer cell line via induction of apoptosis and was capable of reducing microtumor formation in a long-term clonogenic assay.  Altogether, this work reports the discovery of a new small mol. inhibitor of miR-21 and demonstrates its potential as an alternative approach in cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoQMXpXY6aQH7Vg90H21EOLACvtfcHk0ljC20hYMRNqpw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXht1ynu7jP&md5=dfa9d5c537cfa01f14a34c4ff784b4ea</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2019.05.044&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2019.05.044%26sid%3Dliteratum%253Aachs%26aulast%3DAnkenbruck%26aufirst%3DN.%26aulast%3DKumbhare%26aufirst%3DR.%26aulast%3DNaro%26aufirst%3DY.%26aulast%3DThomas%26aufirst%3DM.%26aulast%3DGardner%26aufirst%3DL.%26aulast%3DEmanuelson%26aufirst%3DC.%26aulast%3DDeiters%26aufirst%3DA.%26atitle%3DSmall%2520molecule%2520inhibition%2520of%2520microRNA-21%2520expression%2520reduces%2520cell%2520viability%2520and%2520microtumor%2520formation%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2019%26volume%3D27%26spage%3D3735%26epage%3D3743%26doi%3D10.1016%2Fj.bmc.2019.05.044" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhattarai, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, Z. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, F. S.</span></span> <span> </span><span class="NLM_article-title">Regulating miRNA-21 biogenesis by bifunctional small molecules</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>139</i></span>,  <span class="NLM_fpage">4987</span>â <span class="NLM_lpage">4990</span>, <span class="refDoi">Â DOI: 10.1021/jacs.7b00610</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.7b00610" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC2sXktFOmu7k%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=139&publication_year=2017&pages=4987-4990&author=H.+Yanauthor=U.+Bhattaraiauthor=Z.+F.+Guoauthor=F.+S.+Liang&title=Regulating+miRNA-21+biogenesis+by+bifunctional+small+molecules&doi=10.1021%2Fjacs.7b00610"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Regulating miRNA-21 Biogenesis By Bifunctional Small Molecules</span></div><div class="casAuthors">Yan, Hao; Bhattarai, Umesh; Guo, Zhi-Fo; Liang, Fu-Sen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">139</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">4987-4990</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We report a new strategy to regulate microRNAs (miRNAs) biogenesis by using bifunctional small mols. that consist of a pre-miRNA binding unit connected by a linker to a Dicer inhibiting unit.  In this effort, fluorescence polarization-based screening was used to identify neomycin as a pre-miR-21 binding ligand.  Although neomycin cannot inhibit miR-21 maturation, linking it to the RNase inhibitor 1 forms the bifunctional conjugate 7A, which inhibits the prodn. of miR-21.  We expect that this strategy will be applicable to design other mols. for miRNA regulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHWiyPB0E-xLVg90H21EOLACvtfcHk0ljC20hYMRNqpw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXktFOmu7k%253D&md5=ce221c2e953cded2e2a35c3484ee7fd4</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Fjacs.7b00610&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.7b00610%26sid%3Dliteratum%253Aachs%26aulast%3DYan%26aufirst%3DH.%26aulast%3DBhattarai%26aufirst%3DU.%26aulast%3DGuo%26aufirst%3DZ.%2BF.%26aulast%3DLiang%26aufirst%3DF.%2BS.%26atitle%3DRegulating%2520miRNA-21%2520biogenesis%2520by%2520bifunctional%2520small%2520molecules%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2017%26volume%3D139%26spage%3D4987%26epage%3D4990%26doi%3D10.1021%2Fjacs.7b00610" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nieto, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, R. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jackson, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thiery, J. P.</span></span> <span> </span><span class="NLM_article-title">EMT: 2016</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>166</i></span>,  <span class="NLM_fpage">21</span>â <span class="NLM_lpage">45</span>, <span class="refDoi">Â DOI: 10.1016/j.cell.2016.06.028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1016%2Fj.cell.2016.06.028" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=27368099" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFaitLbF" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=166&publication_year=2016&pages=21-45&author=M.+A.+Nietoauthor=R.+Y.+Huangauthor=R.+A.+Jacksonauthor=J.+P.+Thiery&title=EMT%3A+2016&doi=10.1016%2Fj.cell.2016.06.028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">EMT: 2016</span></div><div class="casAuthors">Nieto, M. Angela; Huang, Ruby Yun-Ju; Jackson, Rebecca A.; Thiery, Jean Paul</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">166</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">21-45</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  The significant parallels between cell plasticity during embryonic development and carcinoma progression have helped us understand the importance of the epithelial-mesenchymal transition (EMT) in human disease.  Our expanding knowledge of EMT has led to a clarification of the EMT program as a set of multiple and dynamic transitional states between the epithelial and mesenchymal phenotypes, as opposed to a process involving a single binary decision.  EMT and its intermediate states have recently been identified as crucial drivers of organ fibrosis and tumor progression, although there is some need for caution when interpreting its contribution to metastatic colonization.  Here, we discuss the current state-of-the-art and latest findings regarding the concept of cellular plasticity and heterogeneity in EMT.  We raise some of the questions pending and identify the challenges faced in this fast-moving field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1EILVAUm1ALVg90H21EOLACvtfcHk0ljC20hYMRNqpw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtFaitLbF&md5=f8c2b1ff3efba49dcc63ad122fa188c8</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2016.06.028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2016.06.028%26sid%3Dliteratum%253Aachs%26aulast%3DNieto%26aufirst%3DM.%2BA.%26aulast%3DHuang%26aufirst%3DR.%2BY.%26aulast%3DJackson%26aufirst%3DR.%2BA.%26aulast%3DThiery%26aufirst%3DJ.%2BP.%26atitle%3DEMT%253A%25202016%26jtitle%3DCell%26date%3D2016%26volume%3D166%26spage%3D21%26epage%3D45%26doi%3D10.1016%2Fj.cell.2016.06.028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Qi, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bart, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Platteel, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sluis, T. V. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huitema, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harms, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hollema, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">V. D. Berg, A. V. D.</span></span> <span> </span><span class="NLM_article-title">Expression of miR-21 and its targets (PTEN, PDCD4, TM1) in flat epithelial atypia of the breast in relation to ductal carcinoma in situ and invasive carcinoma</span>. <i>BMC Cancer</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">163</span>â <span class="NLM_lpage">171</span>, <span class="refDoi">Â DOI: 10.1186/1471-2407-9-163</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1186%2F1471-2407-9-163" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=19473551" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A280%3ADC%252BD1MvhtF2hug%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2009&pages=163-171&author=L.+Qiauthor=J.+Bartauthor=L.+P.+Tanauthor=L.+Platteelauthor=T.+V.+D.+Sluisauthor=S.+Huitemaauthor=G.+Harmsauthor=L.+Fuauthor=H.+Hollemaauthor=A.+V.+D.+V.+D.+Berg&title=Expression+of+miR-21+and+its+targets+%28PTEN%2C+PDCD4%2C+TM1%29+in+flat+epithelial+atypia+of+the+breast+in+relation+to+ductal+carcinoma+in+situ+and+invasive+carcinoma&doi=10.1186%2F1471-2407-9-163"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Expression of miR-21 and its targets (PTEN, PDCD4, TM1) in flat epithelial atypia of the breast in relation to ductal carcinoma in situ and invasive carcinoma</span></div><div class="casAuthors">Qi Liqiang; Bart Joost; Tan Lu Ping; Platteel Inge; Sluis Tineke van der; Huitema Sippie; Harms Geert; Fu Li; Hollema Harry; Berg Anke van den</div><div class="citationInfo"><span class="NLM_cas:title">BMC cancer</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">163</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Flat epithelial atypia (FEA) of the breast is characterised by a few layers of mildly atypical luminal epithelial cells.  Genetic changes found in ductal carcinoma in situ (DCIS) and invasive ductal breast cancer (IDC) are also found in FEA, albeit at a lower concentration.  So far, miRNA expression changes associated with invasive breast cancer, like miR-21, have not been studied in FEA.  METHODS:  We performed miRNA in-situ hybridization (ISH) on 15 cases with simultaneous presence of normal breast tissue, FEA and/or DCIS and 17 additional cases with IDC.  Expression of the miR-21 targets PDCD4, TM1 and PTEN was investigated by immunohistochemistry.  RESULTS:  Two out of fifteen cases showed positive staining for miR-21 in normal breast ductal epithelium, seven out of fifteen cases were positive in the FEA component and nine out of twelve cases were positive in the DCIS component.  A positive staining of miR-21 was observed in 15 of 17 IDC cases.  In 12 cases all three components were present in one tissue block and an increase of miR-21 from normal breast to FEA and to DCIS was observed in five cases.  In three cases the FEA component was negative, whereas the DCIS component was positive for miR-21.  In three other cases, normal, FEA and DCIS components were negative for miR-21 and in the last case all three components were positive.  Overall we observed a gradual increase in percentage of miR-21 positive cases from normal, to FEA, DCIS and IDC.  Immunohistochemical staining for PTEN revealed no obvious changes in staining intensities in normal, FEA, DCIS and IDC.  Cytoplasmic staining of PDCD4 increased from normal to IDC, whereas, the nuclear staining decreased.  TM1 staining decreased from positive in normal breast to negative in most DCIS and IDC cases.  In FEA, the staining pattern for TM1 was similar to normal breast tissue.  CONCLUSION:  Upregulation of miR-21 from normal ductal epithelial cells of the breast to FEA, DCIS and IDC parallels morphologically defined carcinogenesis.  No clear relation was observed between the staining pattern of miR-21 and its previously reported target genes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcT7QDKBZgnzoAwxfk_AGB-EfW6udTcc2ebAFvanSUIRgrntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1MvhtF2hug%253D%253D&md5=978c074b2fc6730464f4046f3904b37b</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1186%2F1471-2407-9-163&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1471-2407-9-163%26sid%3Dliteratum%253Aachs%26aulast%3DQi%26aufirst%3DL.%26aulast%3DBart%26aufirst%3DJ.%26aulast%3DTan%26aufirst%3DL.%2BP.%26aulast%3DPlatteel%26aufirst%3DL.%26aulast%3DSluis%26aufirst%3DT.%2BV.%2BD.%26aulast%3DHuitema%26aufirst%3DS.%26aulast%3DHarms%26aufirst%3DG.%26aulast%3DFu%26aufirst%3DL.%26aulast%3DHollema%26aufirst%3DH.%26aulast%3DV.%2BD.%2BBerg%26aufirst%3DA.%2BV.%2BD.%26atitle%3DExpression%2520of%2520miR-21%2520and%2520its%2520targets%2520%2528PTEN%252C%2520PDCD4%252C%2520TM1%2529%2520in%2520flat%2520epithelial%2520atypia%2520of%2520the%2520breast%2520in%2520relation%2520to%2520ductal%2520carcinoma%2520in%2520situ%2520and%2520invasive%2520carcinoma%26jtitle%3DBMC%2520Cancer%26date%3D2009%26volume%3D9%26spage%3D163%26epage%3D171%26doi%3D10.1186%2F1471-2407-9-163" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Degyatoreva, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kukielski, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Story, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhaduri, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maiti, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nahar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ray, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arya, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maiti, S.</span></span> <span> </span><span class="NLM_article-title">Targeting miRNA by tunable small molecule binders: peptidic aminosugar mediated interference in miR-21 biogenesis reverts epithelial to mesenchymal transition</span>. <i>MedChemComm</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1147</span>â <span class="NLM_lpage">1154</span>, <span class="refDoi">Â DOI: 10.1039/C8MD00092A</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1039%2FC8MD00092A" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=30109002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtFCmtr7L" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=1147-1154&author=A.+Ghoshauthor=N.+Degyatorevaauthor=C.+Kukielskiauthor=S.+Storyauthor=S.+Bhaduriauthor=K.+Maitiauthor=S.+Naharauthor=A.+Rayauthor=D.+P.+Aryaauthor=S.+Maiti&title=Targeting+miRNA+by+tunable+small+molecule+binders%3A+peptidic+aminosugar+mediated+interference+in+miR-21+biogenesis+reverts+epithelial+to+mesenchymal+transition&doi=10.1039%2FC8MD00092A"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting miRNA by tunable small molecule binders: peptidic aminosugar mediated interference in miR-21 biogenesis reverts epithelial to mesenchymal transition</span></div><div class="casAuthors">Ghosh, Arpita; Degyatoreva, Natalya; Kukielski, Casey; Story, Sandra; Bhaduri, Sayantan; Maiti, Krishnagopal; Nahar, Smita; Ray, Arjun; Arya, Dev P.; Maiti, Souvik</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1147-1154</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Epithelial to mesenchymal transition (EMT) is a process in which epithelial cells lose cell polarity and cell-cell adhesion and gain migratory and invasive properties to become mesenchymal cells that are very vital for development, wound healing and stem cell behavior and contribute pathol. to fibrosis and cancer progression. miR21, a potent regulator of the tumor suppressor gene PTEN, can be silenced to reverse EMT, thereby providing an attractive target for abrogating the malignant behavior of breast cancer.  Here, we report the design, synthesis and binding of a peptidic-aminoglycoside (PA) based chem. library against pre-miR21 that led to the identification of a group of small mols. that bind to pre-miR21 with high affinities and antagonize miR-21 maturation and function, thereby reversing EMT.  The approach described here offers a promising miRNA targeting platform where such aminosugar conjugates can be similarly used to target other oncogenic miRNAs.  Minor changes in the amino acid sequence allow us to tailor the binding effectiveness and downstream biol. effects, thus making this approach a potentially tunable method of regulation of miRNA function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOPwbV4QOxT7Vg90H21EOLACvtfcHk0lgMP8CFPXeAow"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtFCmtr7L&md5=25dc0c6d22cedb13d6eabcc1c7298443</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1039%2FC8MD00092A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC8MD00092A%26sid%3Dliteratum%253Aachs%26aulast%3DGhosh%26aufirst%3DA.%26aulast%3DDegyatoreva%26aufirst%3DN.%26aulast%3DKukielski%26aufirst%3DC.%26aulast%3DStory%26aufirst%3DS.%26aulast%3DBhaduri%26aufirst%3DS.%26aulast%3DMaiti%26aufirst%3DK.%26aulast%3DNahar%26aufirst%3DS.%26aulast%3DRay%26aufirst%3DA.%26aulast%3DArya%26aufirst%3DD.%2BP.%26aulast%3DMaiti%26aufirst%3DS.%26atitle%3DTargeting%2520miRNA%2520by%2520tunable%2520small%2520molecule%2520binders%253A%2520peptidic%2520aminosugar%2520mediated%2520interference%2520in%2520miR-21%2520biogenesis%2520reverts%2520epithelial%2520to%2520mesenchymal%2520transition%26jtitle%3DMedChemComm%26date%3D2018%26volume%3D9%26spage%3D1147%26epage%3D1154%26doi%3D10.1039%2FC8MD00092A" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Watkins, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nahar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maiti, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arya, D. P.</span></span> <span> </span><span class="NLM_article-title">A pH sensitive high-throughput assay for miRNA binding of a peptide-aminoglycoside (PA) library</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>, <span class="NLM_elocation-id">e0144251</span> <span class="refDoi">Â DOI: 10.1371/journal.pone.0144251</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1371%2Fjournal.pone.0144251" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=26656788" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC28XksF2gsbs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&author=D.+Watkinsauthor=L.+Jiangauthor=S.+Naharauthor=S.+Maitiauthor=D.+P.+Arya&title=A+pH+sensitive+high-throughput+assay+for+miRNA+binding+of+a+peptide-aminoglycoside+%28PA%29+library&doi=10.1371%2Fjournal.pone.0144251"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">A pH sensitive high-throughput assay for miRNA binding of a peptide-aminoglycoside (PA) library</span></div><div class="casAuthors">Watkins, Derrick; Jiang, Liuwei; Nahar, Smita; Maiti, Souvik; Arya, Dev P.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">e0144251/1-e0144251/23</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">MicroRNAs (miRNA) are small RNAs that have a regulatory role in gene expression.  Because of this regulatory role, miRNAs have become a new target for therapeutic compds.  Here, we outline an approach to target specific miRNAs using a high throughput capable assay and a 215 compd. peptidic-aminosugar (PA) library.  Aminosugars have been shown in a no. of recent reports as important lead compds. that bind miRNA.  In order to screen for compds. that bind miRNA, we have developed a high throughput displacement assay using a fluorescein-neomycin conjugated mol. (F-neo) as a probe for competitive miRNA binding compds.  We have applied the F-neo assay to four different miRNA constructs and the assay is applicable to most miRNAs, at various stages of processing.  The results of the screen were validated by the detn. of the IC50 for a select group of compds. from the library.  For example, we identified eight compds. that bind to hsa-miR 504 with higher affinity than the parent neomycin.  From the F-neo displacement assay we found that the no. of binding sites differs for each miRNA, and the binding sites appear to differ both phys. and chem., with different affinity of the compds. resulting from the size of the mol. as well as the chem. structure.  Addnl., the affinity of the compds. was dependent on the identity and position of the amino acid position of conjugation and the affinity of the compds. relative to other compds. in the library was miRNA dependent with the introduction of a second amino acid.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpP7dNoUHy7zbVg90H21EOLACvtfcHk0lgnSeqalekcZw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XksF2gsbs%253D&md5=00d931a54eb439f22e612ea587cfb1ee</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0144251&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0144251%26sid%3Dliteratum%253Aachs%26aulast%3DWatkins%26aufirst%3DD.%26aulast%3DJiang%26aufirst%3DL.%26aulast%3DNahar%26aufirst%3DS.%26aulast%3DMaiti%26aufirst%3DS.%26aulast%3DArya%26aufirst%3DD.%2BP.%26atitle%3DA%2520pH%2520sensitive%2520high-throughput%2520assay%2520for%2520miRNA%2520binding%2520of%2520a%2520peptide-aminoglycoside%2520%2528PA%2529%2520library%26jtitle%3DPLoS%2520One%26date%3D2015%26volume%3D10%26doi%3D10.1371%2Fjournal.pone.0144251" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Haggar, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boushey, R. P.</span></span> <span> </span><span class="NLM_article-title">Colorectal cancer epidemiology: incidence, mortality, survival, and risk factors</span>. <i>Clin. Colon. Rectal. Surg.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">191</span>â <span class="NLM_lpage">197</span>, <span class="refDoi">Â DOI: 10.1055/s-0029-1242458</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1055%2Fs-0029-1242458" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=21037809" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A280%3ADC%252BC3cbjsFWluw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2009&pages=191-197&author=F.+A.+Haggarauthor=R.+P.+Boushey&title=Colorectal+cancer+epidemiology%3A+incidence%2C+mortality%2C+survival%2C+and+risk+factors&doi=10.1055%2Fs-0029-1242458"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Colorectal cancer epidemiology: incidence, mortality, survival, and risk factors</span></div><div class="casAuthors">Haggar Fatima A; Boushey Robin P</div><div class="citationInfo"><span class="NLM_cas:title">Clinics in colon and rectal surgery</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">191-7</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">In this article, the incidence, mortality, and survival rates for colorectal cancer are reviewed, with attention paid to regional variations and changes over time.  A concise overview of known risk factors associated with colorectal cancer is provided, including familial and hereditary factors, as well as environmental lifestyle-related risk factors such as physical inactivity, obesity, smoking, and alcohol consumption.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSYLvT3tqMgUgaK-_ZudimnfW6udTcc2ea6VykQAsVhsbntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3cbjsFWluw%253D%253D&md5=392f9c8d22332f194121f0645405249c</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1055%2Fs-0029-1242458&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1055%252Fs-0029-1242458%26sid%3Dliteratum%253Aachs%26aulast%3DHaggar%26aufirst%3DF.%2BA.%26aulast%3DBoushey%26aufirst%3DR.%2BP.%26atitle%3DColorectal%2520cancer%2520epidemiology%253A%2520incidence%252C%2520mortality%252C%2520survival%252C%2520and%2520risk%2520factors%26jtitle%3DClin.%2520Colon.%2520Rectal.%2520Surg.%26date%3D2009%26volume%3D22%26spage%3D191%26epage%3D197%26doi%3D10.1055%2Fs-0029-1242458" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stiegelbauer, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perakis, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deutsch, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ling, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerger, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pichler, M.</span></span> <span> </span><span class="NLM_article-title">MicroRNAs as novel predictive biomarkers and therapeutic targets in colorectal cancer</span>. <i>World J. Gastroenterol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">11727</span>â <span class="NLM_lpage">11735</span>, <span class="refDoi">Â DOI: 10.3748/wjg.v20.i33.11727</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.3748%2Fwjg.v20.i33.11727" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=25206276" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvV2jt7vF" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2014&pages=11727-11735&author=V.+Stiegelbauerauthor=S.+Perakisauthor=A.+Deutschauthor=H.+Lingauthor=A.+Gergerauthor=M.+Pichler&title=MicroRNAs+as+novel+predictive+biomarkers+and+therapeutic+targets+in+colorectal+cancer&doi=10.3748%2Fwjg.v20.i33.11727"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">MicroRNAs as novel predictive biomarkers and therapeutic targets in colorectal cancer</span></div><div class="casAuthors">Stiegelbauer, Verena; Perakis, Samantha; Deutsch, Alexander; Ling, Hui; Gerger, Armin; Pichler, Martin</div><div class="citationInfo"><span class="NLM_cas:title">World Journal of Gastroenterology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">33</span>),
    <span class="NLM_cas:pages">11727-11735, 10 pp.</span>CODEN:
                <span class="NLM_cas:coden">WJGAF2</span>;
        ISSN:<span class="NLM_cas:issn">1007-9327</span>.
    
            (<span class="NLM_cas:orgname">Baishideng Publishing Group Inc.</span>)
        </div><div class="casAbstract">A review.  Colorectal cancer (CRC) is the third most common cancer in western countries.  Despite significant improvement in available treatment options, CRC still remains the second leading cause of cancer-related death.  Traditionally, 5-fluorouracil has been used as the main chemotherapy drug for treatment of metastatic CRC (mCRC).  However, during the last two decades more effective chemotherapeutic agents such as oxaliplatin, irinotecan and the monoclonal antibodies cetuximab, panitumumab and bevacizumab have been used in clin. practice.  More recently, the therapeutic armamentarium has been supplemented by the monoclonal antibodies bevacizumab, cetuximab and panitumumab as well as the protein-trap aflibercept and the small mol. multi-kinase inhibitor regorafenib.  One of the major problems for the management of CRC is the inherent or acquired resistance to therapeutic approaches.  The discovery of microRNAs (miRNAs), a class of small, endogenous, non-coding, single-stranded RNAs that play a role as post-transcriptional regulators, has added new dimensions to the diagnosis and treatment of cancer.  Because miRNAs are important regulators of carcinogenesis, progression, invasion, angiogenesis and metastases in CRC, they might serve as potential predictive and prognostic factors and even as therapeutic targets themselves.  Several miRNAs are already known to be dysregulated in CRCs and have been linked to biol. processes involved in tumor progression and response to anti-cancer therapies.  This review summarizes current therapeutic approaches for treating CRC and highlights the role of miRNAs as novel predictive biomarkers and potential drug targets in CRC patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9eb1QcDI5ZLVg90H21EOLACvtfcHk0lgWm-vlZ1lrjQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvV2jt7vF&md5=546238c5959835d22eeda95a213706ae</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.3748%2Fwjg.v20.i33.11727&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3748%252Fwjg.v20.i33.11727%26sid%3Dliteratum%253Aachs%26aulast%3DStiegelbauer%26aufirst%3DV.%26aulast%3DPerakis%26aufirst%3DS.%26aulast%3DDeutsch%26aufirst%3DA.%26aulast%3DLing%26aufirst%3DH.%26aulast%3DGerger%26aufirst%3DA.%26aulast%3DPichler%26aufirst%3DM.%26atitle%3DMicroRNAs%2520as%2520novel%2520predictive%2520biomarkers%2520and%2520therapeutic%2520targets%2520in%2520colorectal%2520cancer%26jtitle%3DWorld%2520J.%2520Gastroenterol.%26date%3D2014%26volume%3D20%26spage%3D11727%26epage%3D11735%26doi%3D10.3748%2Fwjg.v20.i33.11727" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lv, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bie, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, F.</span></span> <span> </span><span class="NLM_article-title">Direct binding of microRNA-21 pre-element with Regorafenib: An alternative mechanism for anti-colorectal cancer chemotherapy?</span>. <i>J. Mol. Graphics Modell.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>73</i></span>,  <span class="NLM_fpage">48</span>â <span class="NLM_lpage">53</span>, <span class="refDoi">Â DOI: 10.1016/j.jmgm.2017.02.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1016%2Fj.jmgm.2017.02.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=28236743" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjtlKjtbs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2017&pages=48-53&author=X.+Chenauthor=B.+Xieauthor=L.+Caoauthor=F.+Zhuauthor=B.+Chenauthor=H.+Lvauthor=X.+Fanauthor=L.+Hanauthor=L.+Bieauthor=X.+Caoauthor=X.+Shenauthor=F.+Cao&title=Direct+binding+of+microRNA-21+pre-element+with+Regorafenib%3A+An+alternative+mechanism+for+anti-colorectal+cancer+chemotherapy%3F&doi=10.1016%2Fj.jmgm.2017.02.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Direct binding of microRNA-21 pre-element with Regorafenib: An alternative mechanism for anti-colorectal cancer chemotherapy?</span></div><div class="casAuthors">Chen, Xiaobing; Xie, Bojian; Cao, Liang; Zhu, Feng; Chen, Beibei; Lv, Huifang; Fan, Xingxing; Han, Lili; Bie, Liangyu; Cao, Xinguang; Shen, Xiaokun; Cao, Feilin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Graphics & Modelling</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">73</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">48-53</span>CODEN:
                <span class="NLM_cas:coden">JMGMFI</span>;
        ISSN:<span class="NLM_cas:issn">1093-3263</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The Regorafenib is a broad-spectrum kinase inhibitor that has been approved to treat colorectal cancer (CRC).  However, evidences have shown that the agent is also implicated in drug interaction with microRNA-21 (miR-21), an oncogenic miRNA which plays a key role in resisting programmed cell death in CRC cells.  Here, we supposed that, instead of kinase inhibition, Regorafenib can directly bind to and then stabilize miR-21 pre-element, thus preventing RNase Dicer-meditated cleavage of the pre-element to mature miR-21.  In order to verify the notion, an in silico-in vitro integrated investigation of the direct intermol. interaction between Regorafenib and miR-21 pre-element was performed by using active pocket identification, RNA-ligand docking, mol. dynamics (MD) simulation, binding energetic anal., and fluorescence-based assay.  It was revealed that the Regorafenib can bind at the major groove-like stem region of miR-21 pre-element through three geometrically satisfactory hydrogen bonds (H-bonds) as well as a no. of hydrophobic forces and Ï-Ï stacking, conferring strong specificity and high stability to the RNA-ligand complex system (Kd = 0.73 Î¼M).  Sep. inversion mutation of two base pairs (G6C, C12G) and (A13U, U4A) that are involved in the H-bonding can considerably impair the affinity of Regorafenib to miR-21 pre-element, with Kd increase to 27 and 96 Î¼M, resp.  All these supported that Regorafenib can directly bind to miR-21 pre-element at mol. level and the binding mode can be properly modeled by using the proposed integrated strategy.  This study would provide a potential, alternative mechanism for anti-colorectal cancer chemotherapy with Regorafenib.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoRvOCg1wgVd7Vg90H21EOLACvtfcHk0lgWm-vlZ1lrjQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjtlKjtbs%253D&md5=b772e412cd20be6de42957b0bc0732c8</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1016%2Fj.jmgm.2017.02.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmgm.2017.02.004%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DX.%26aulast%3DXie%26aufirst%3DB.%26aulast%3DCao%26aufirst%3DL.%26aulast%3DZhu%26aufirst%3DF.%26aulast%3DChen%26aufirst%3DB.%26aulast%3DLv%26aufirst%3DH.%26aulast%3DFan%26aufirst%3DX.%26aulast%3DHan%26aufirst%3DL.%26aulast%3DBie%26aufirst%3DL.%26aulast%3DCao%26aufirst%3DX.%26aulast%3DShen%26aufirst%3DX.%26aulast%3DCao%26aufirst%3DF.%26atitle%3DDirect%2520binding%2520of%2520microRNA-21%2520pre-element%2520with%2520Regorafenib%253A%2520An%2520alternative%2520mechanism%2520for%2520anti-colorectal%2520cancer%2520chemotherapy%253F%26jtitle%3DJ.%2520Mol.%2520Graphics%2520Modell.%26date%3D2017%26volume%3D73%26spage%3D48%26epage%3D53%26doi%3D10.1016%2Fj.jmgm.2017.02.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Costales, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aikawa, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Childs-Disney, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abegg, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoch, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Velagapudi, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khan, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yildirim, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adibekian, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, E. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span> <span> </span><span class="NLM_article-title">Small-molecule targeted recruitment of a nuclease to cleave an oncogenic RNA in a mouse model of metastatic cancer</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>117</i></span>,  <span class="NLM_fpage">2406</span>â <span class="NLM_lpage">2411</span>, <span class="refDoi">Â DOI: 10.1073/pnas.1914286117</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1073%2Fpnas.1914286117" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=31964809" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BB3cXisVWisL0%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=117&publication_year=2020&pages=2406-2411&author=M.+G.+Costalesauthor=H.+Aikawaauthor=Y.+Liauthor=J.+L.+Childs-Disneyauthor=D.+Abeggauthor=D.+G.+Hochauthor=S.+P.+Velagapudiauthor=Y.+Nakaiauthor=T.+Khanauthor=K.+W.+Wangauthor=I.+Yildirimauthor=A.+Adibekianauthor=E.+T.+Wangauthor=M.+D.+Disney&title=Small-molecule+targeted+recruitment+of+a+nuclease+to+cleave+an+oncogenic+RNA+in+a+mouse+model+of+metastatic+cancer&doi=10.1073%2Fpnas.1914286117"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Small-molecule targeted recruitment of a nuclease to cleave an oncogenic RNA in a mouse model of metastatic cancer</span></div><div class="casAuthors">Costales, Matthew G.; Aikawa, Haruo; Li, Yue; Childs-Disney, Jessica L.; Abegg, Daniel; Hoch, Dominic G.; Velagapudi, Sai Pradeep; Nakai, Yoshio; Khan, Tanya; Wang, Kye Won; Yildirim, Ilyas; Adibekian, Alexander; Wang, Eric T.; Disney, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">117</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2406-2411</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">As the area of small mols. interacting with RNA advances, general routes to provide bioactive compds. are needed as ligands can bind RNA avidly to sites that will not affect function.  Small-mol. targeted RNA degrdn. will thus provide a general route to affect RNA biol.  A non-oligonucleotide-contg. compd. was designed from sequence to target the precursor to oncogenic microRNA-21 (pre-miR-21) for enzymic destruction with selectivity that can exceed that for protein-targeted medicines.  The compd. specifically binds the target and contains a heterocycle that recruits and activates a RNase to pre-miR-21 to substoichiometrically effect its cleavage and subsequently impede metastasis of breast cancer to lung in a mouse model.  Transcriptomic and proteomic analyses demonstrate that the compd. is potent and selective, specifically modulating oncogenic pathways.  Thus, small mols. can be designed from sequence to have all of the functional repertoire of oligonucleotides, including inducing enzymic degrdn., and to selectively and potently modulate RNA function in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGolzufxJ0-LLLVg90H21EOLACvtfcHk0lh7DkTA2BW0tQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXisVWisL0%253D&md5=450996be5aadaa9e13acb5c2cfb03c62</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1914286117&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1914286117%26sid%3Dliteratum%253Aachs%26aulast%3DCostales%26aufirst%3DM.%2BG.%26aulast%3DAikawa%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DChilds-Disney%26aufirst%3DJ.%2BL.%26aulast%3DAbegg%26aufirst%3DD.%26aulast%3DHoch%26aufirst%3DD.%2BG.%26aulast%3DVelagapudi%26aufirst%3DS.%2BP.%26aulast%3DNakai%26aufirst%3DY.%26aulast%3DKhan%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DK.%2BW.%26aulast%3DYildirim%26aufirst%3DI.%26aulast%3DAdibekian%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DE.%2BT.%26aulast%3DDisney%26aufirst%3DM.%2BD.%26atitle%3DSmall-molecule%2520targeted%2520recruitment%2520of%2520a%2520nuclease%2520to%2520cleave%2520an%2520oncogenic%2520RNA%2520in%2520a%2520mouse%2520model%2520of%2520metastatic%2520cancer%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2020%26volume%3D117%26spage%3D2406%26epage%3D2411%26doi%3D10.1073%2Fpnas.1914286117" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Velagapudi, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gallo, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span> <span> </span><span class="NLM_article-title">Sequence-based design of bioactive small molecules that target precursor microRNAs</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">291</span>â <span class="NLM_lpage">299</span>, <span class="refDoi">Â DOI: 10.1038/nchembio.1452</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1038%2Fnchembio.1452" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=24509821" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitFaksbo%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=291-299&author=S.+P.+Velagapudiauthor=S.+M.+Galloauthor=M.+D.+Disney&title=Sequence-based+design+of+bioactive+small+molecules+that+target+precursor+microRNAs&doi=10.1038%2Fnchembio.1452"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Sequence-based design of bioactive small molecules that target precursor microRNAs</span></div><div class="casAuthors">Velagapudi, Sai Pradeep; Gallo, Steven M.; Disney, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">291-297</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Oligonucleotides are designed to target RNA using base pairing rules, but they can be hampered by poor cellular delivery and nonspecific stimulation of the immune system.  Small mols. are preferred as lead drugs or probes but cannot be designed from sequence.  Herein, we describe an approach termed Inforna that designs lead small mols. for RNA from solely sequence.  Inforna was applied to all human microRNA hairpin precursors, and it identified bioactive small mols. that inhibit biogenesis by binding nuclease-processing sites (44% hit rate).  Among 27 lead interactions, the most avid interaction is between a benzimidazole (1) and precursor microRNA-96.  Compd. 1 selectively inhibits biogenesis of microRNA-96, upregulating a protein target (FOXO1) and inducing apoptosis in cancer cells.  Apoptosis is ablated when FOXO1 mRNA expression is knocked down by an siRNA, validating compd. selectivity.  Markedly, microRNA profiling shows that 1 only affects microRNA-96 biogenesis and is at least as selective as an oligonucleotide.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqy1YQeIYltbrVg90H21EOLACvtfcHk0lh7DkTA2BW0tQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitFaksbo%253D&md5=d57f6a2ca7d6784017e2adff9ddb2f55</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1452&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1452%26sid%3Dliteratum%253Aachs%26aulast%3DVelagapudi%26aufirst%3DS.%2BP.%26aulast%3DGallo%26aufirst%3DS.%2BM.%26aulast%3DDisney%26aufirst%3DM.%2BD.%26atitle%3DSequence-based%2520design%2520of%2520bioactive%2520small%2520molecules%2520that%2520target%2520precursor%2520microRNAs%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2014%26volume%3D10%26spage%3D291%26epage%3D299%26doi%3D10.1038%2Fnchembio.1452" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cho, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shin, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koo, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, K. S.</span></span> <span> </span><span class="NLM_article-title">miR-372 regulates cell cycle and apoptosis of AGS human gastric cancer cell line through direct regulation of LATS2</span>. <i>Mol. Cells</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">521</span>â <span class="NLM_lpage">527</span>, <span class="refDoi">Â DOI: 10.1007/s10059-009-0158-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1007%2Fs10059-009-0158-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=19937137" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC3cXisV2nsw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2009&pages=521-527&author=W.+J.+Choauthor=J.+M.+Shinauthor=J.+S.+Kimauthor=M.+R.+Leeauthor=K.+S.+Hongauthor=J.+H.+Leeauthor=K.+H.+Kooauthor=J.+W.+Parkauthor=K.+S.+Kim&title=miR-372+regulates+cell+cycle+and+apoptosis+of+AGS+human+gastric+cancer+cell+line+through+direct+regulation+of+LATS2&doi=10.1007%2Fs10059-009-0158-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">miR-372 regulates cell cycle and apoptosis of AGS human gastric cancer cell line through direct regulation of LATS2</span></div><div class="casAuthors">Cho, Wha Ja; Shin, Jeong Min; Kim, Jong Soo; Lee, Man Ryul; Hong, Ki Sung; Lee, Jun-Ho; Koo, Kyoung Hwa; Park, Jeong Woo; Kim, Kye-Seong</div><div class="citationInfo"><span class="NLM_cas:title">Molecules and Cells</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">521-527</span>CODEN:
                <span class="NLM_cas:coden">MOCEEK</span>;
        ISSN:<span class="NLM_cas:issn">1016-8478</span>.
    
            (<span class="NLM_cas:orgname">Korean Society for Molecular and Cellular Biology</span>)
        </div><div class="casAbstract">Previously, we have reported tissue- and stage-specific expression of miR-372 in human embryonic stem cells and so far, not many reports speculate the function of this microRNA (miRNA).  In this study, we screened various human cancer cell lines including gastric cancer cell lines and found first time that miR-372 is expressed only in AGS human gastric adenocarcinoma cell line.  Inhibition of miR-372 using antisense miR-372 oligonucleotide (AS-miR-372) suppressed proliferation, arrested the cell cycle at G2/M phase, and increased apoptosis of AGS cells.  Furthermore, AS-miR-372 treatment increased expression of LATS2, while over-expression of miR-372 decreased luciferase reporter activity driven by the 3' untranslated region (3' UTR) of LATS2 mRNA.  Over-expression of LATS2 induced changes in AGS cells similar to those in AGS cells treated with AS-miR-372.  Taken together, these findings demonstrate an oncogenic role for miR-372 in controlling cell growth, cell cycle, and apoptosis through down-regulation of a tumor suppressor gene, LATS2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxrPqqSkqmZbVg90H21EOLACvtfcHk0lhz5ZroHdhU-w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXisV2nsw%253D%253D&md5=400516a007dbd0ea5756d7f59ada12c9</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1007%2Fs10059-009-0158-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10059-009-0158-0%26sid%3Dliteratum%253Aachs%26aulast%3DCho%26aufirst%3DW.%2BJ.%26aulast%3DShin%26aufirst%3DJ.%2BM.%26aulast%3DKim%26aufirst%3DJ.%2BS.%26aulast%3DLee%26aufirst%3DM.%2BR.%26aulast%3DHong%26aufirst%3DK.%2BS.%26aulast%3DLee%26aufirst%3DJ.%2BH.%26aulast%3DKoo%26aufirst%3DK.%2BH.%26aulast%3DPark%26aufirst%3DJ.%2BW.%26aulast%3DKim%26aufirst%3DK.%2BS.%26atitle%3DmiR-372%2520regulates%2520cell%2520cycle%2520and%2520apoptosis%2520of%2520AGS%2520human%2520gastric%2520cancer%2520cell%2520line%2520through%2520direct%2520regulation%2520of%2520LATS2%26jtitle%3DMol.%2520Cells%26date%3D2009%26volume%3D28%26spage%3D521%26epage%3D527%26doi%3D10.1007%2Fs10059-009-0158-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Voorhoeve, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sage, C. l.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schrier, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gillis, A. J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stoop, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duijse, J. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drost, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griekspoor, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zlotorynski, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yabuta, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vita, G. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nojima, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Looijenga, L. H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agami, R.</span></span> <span> </span><span class="NLM_article-title">A genetic screen implicates miRNA-372 and miRNA-373 as oncogenes in testicular germ cell tumors</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>124</i></span>,  <span class="NLM_fpage">1169</span>â <span class="NLM_lpage">1181</span>, <span class="refDoi">Â DOI: 10.1016/j.cell.2006.02.037</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1016%2Fj.cell.2006.02.037" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=16564011" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BD28Xjt1Khsrg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=124&publication_year=2006&pages=1169-1181&author=P.+P.+Voorhoeveauthor=C.+l.+Sageauthor=M.+Schrierauthor=A.+J.+M.+Gillisauthor=H.+Stoopauthor=R.+Nagelauthor=Y.+P.+Liuauthor=J.+V.+Duijseauthor=J.+Drostauthor=A.+Griekspoorauthor=E.+Zlotorynskiauthor=N.+Yabutaauthor=G.+D.+Vitaauthor=H.+Nojimaauthor=L.+H.+J.+Looijengaauthor=R.+Agami&title=A+genetic+screen+implicates+miRNA-372+and+miRNA-373+as+oncogenes+in+testicular+germ+cell+tumors&doi=10.1016%2Fj.cell.2006.02.037"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">A genetic screen implicates miRNA-372 and miRNA-373 as oncogenes in testicular germ cell tumors</span></div><div class="casAuthors">Voorhoeve, P. Mathijs; le Sage, Carlos; Schrier, Mariette; Gillis, Ad J. M.; Stoop, Hans; Nagel, Remco; Liu, Ying-Poi; van Duijse, Josyanne; Drost, Jarno; Griekspoor, Alexander; Zlotorynski, Eitan; Yabuta, Norikazu; De Vita, Gabriella; Nojima, Hiroshi; Looijenga, Leendert H. J.; Agami, Reuven</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">124</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1169-1181</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Endogenous small RNAs (miRNAs) regulate gene expression by mechanisms conserved across metazoans.  While the no. of verified human miRNAs is still expanding, only few have been functionally annotated.  To perform genetic screens for novel functions of miRNAs, we developed a library of vectors expressing the majority of cloned human miRNAs and created corresponding DNA barcode arrays.  In a screen for miRNAs that cooperate with oncogenes in cellular transformation, we identified miR-372 and miR-373, each permitting proliferation and tumorigenesis of primary human cells that harbor both oncogenic RAS and active wild-type p53.  These miRNAs neutralize p53-mediated CDK inhibition, possibly through direct inhibition of the expression of the tumor-suppressor LATS2.  We provide evidence that these miRNAs are potential novel oncogenes participating in the development of human testicular germ cell tumors by numbing the p53 pathway, thus allowing tumorigenic growth in the presence of wild-type p53.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprKiyR8GUInrVg90H21EOLACvtfcHk0lhz5ZroHdhU-w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xjt1Khsrg%253D&md5=b1793069efba1ae573f7e0e66a879c47</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2006.02.037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2006.02.037%26sid%3Dliteratum%253Aachs%26aulast%3DVoorhoeve%26aufirst%3DP.%2BP.%26aulast%3DSage%26aufirst%3DC.%2Bl.%26aulast%3DSchrier%26aufirst%3DM.%26aulast%3DGillis%26aufirst%3DA.%2BJ.%2BM.%26aulast%3DStoop%26aufirst%3DH.%26aulast%3DNagel%26aufirst%3DR.%26aulast%3DLiu%26aufirst%3DY.%2BP.%26aulast%3DDuijse%26aufirst%3DJ.%2BV.%26aulast%3DDrost%26aufirst%3DJ.%26aulast%3DGriekspoor%26aufirst%3DA.%26aulast%3DZlotorynski%26aufirst%3DE.%26aulast%3DYabuta%26aufirst%3DN.%26aulast%3DVita%26aufirst%3DG.%2BD.%26aulast%3DNojima%26aufirst%3DH.%26aulast%3DLooijenga%26aufirst%3DL.%2BH.%2BJ.%26aulast%3DAgami%26aufirst%3DR.%26atitle%3DA%2520genetic%2520screen%2520implicates%2520miRNA-372%2520and%2520miRNA-373%2520as%2520oncogenes%2520in%2520testicular%2520germ%2520cell%2520tumors%26jtitle%3DCell%26date%3D2006%26volume%3D124%26spage%3D1169%26epage%3D1181%26doi%3D10.1016%2Fj.cell.2006.02.037" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vo, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Staedel, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zehnacker, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benhida, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Darfeuille, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duca, M.</span></span> <span> </span><span class="NLM_article-title">Targeting the production of oncogenic microRNAs with multimodal synthetic small molecules</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">711</span>â <span class="NLM_lpage">721</span>, <span class="refDoi">Â DOI: 10.1021/cb400668h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb400668h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFOiur3I" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=711-721&author=D.+D.+Voauthor=C.+Staedelauthor=L.+Zehnackerauthor=R.+Benhidaauthor=F.+Darfeuilleauthor=M.+Duca&title=Targeting+the+production+of+oncogenic+microRNAs+with+multimodal+synthetic+small+molecules&doi=10.1021%2Fcb400668h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting the Production of Oncogenic MicroRNAs with Multimodal Synthetic Small Molecules</span></div><div class="casAuthors">Vo, Duc Duy; Staedel, Cathy; Zehnacker, Laura; Benhida, Rachid; Darfeuille, Fabien; Duca, Maria</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">711-721</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">MicroRNAs (miRNAs) are a recently discovered category of small RNA mols. that regulate gene expression at the post-transcriptional level.  Accumulating evidence indicates that miRNAs are aberrantly expressed in a variety of human cancers and revealed to be oncogenic and to play a pivotal role in initiation and progression of these pathologies.  It is now clear that the inhibition of oncogenic miRNAs, defined as blocking their biosynthesis or their function, could find an application in the therapy of different types of cancer in which these miRNAs are implicated.  Here the authors report the design, synthesis, and biol. evaluation of new small-mol. RNA ligands targeting the prodn. of oncogenic microRNAs.  The authors focused the authors' attention on miR-372 and miR-373 that are implicated in the tumorigenesis of different types of cancer such as gastric cancer.  These two oncogenic miRNAs are overexpressed in gastric cancer cells starting from their precursors pre-miR-372 and pre-miR-373, two stem-loop structured RNAs that lead to mature miRNAs after cleavage by the enzyme Dicer.  The small mols. described herein consist of the conjugation of two RNA binding motives, i.e., the aminoglycoside neomycin and different natural and artificial nucleobases, to obtain RNA ligands with increased affinity and selectivity compared to that of parent compds.  After the synthesis of this new series of RNA ligands, the authors demonstrated that they are able to inhibit the prodn. of the oncogenic miRNA-372 and -373 by binding their pre-miRNAs and inhibiting the processing by Dicer.  Moreover, the authors proved that some of these compds. bear anti-proliferative activity toward gastric cancer cells and that this activity is likely linked to a decrease in the prodn. of targeted miRNAs.  To date, only few examples of small mols. targeting oncogenic miRNAs have been reported, and such inhibitors could be extremely useful for the development of new anticancer therapeutic strategies as well as useful biochem. tools for the study of miRNAs' pathways and mechanisms.  Furthermore, this is the first time that a design based on current knowledge about RNA targeting is proposed to target miRNAs' prodn. with small mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqMnXqBRxfyNrVg90H21EOLACvtfcHk0lhQGpc3-L6LeQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFOiur3I&md5=262eeeb54b8b743aefd61318f582aed2</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1021%2Fcb400668h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb400668h%26sid%3Dliteratum%253Aachs%26aulast%3DVo%26aufirst%3DD.%2BD.%26aulast%3DStaedel%26aufirst%3DC.%26aulast%3DZehnacker%26aufirst%3DL.%26aulast%3DBenhida%26aufirst%3DR.%26aulast%3DDarfeuille%26aufirst%3DF.%26aulast%3DDuca%26aufirst%3DM.%26atitle%3DTargeting%2520the%2520production%2520of%2520oncogenic%2520microRNAs%2520with%2520multimodal%2520synthetic%2520small%2520molecules%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2014%26volume%3D9%26spage%3D711%26epage%3D721%26doi%3D10.1021%2Fcb400668h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Malnuit, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duca, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benhida, R.</span></span> <span> </span><span class="NLM_article-title">Targeting DNA base pair mismatch with artificial nucleobases. Advances and perspectives in triple helix strategy</span>. <i>Org. Biomol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">326</span>â <span class="NLM_lpage">336</span>, <span class="refDoi">Â DOI: 10.1039/C0OB00418A</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1039%2FC0OB00418A" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=21046036" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1SrsbnN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2011&pages=326-336&author=V.+Malnuitauthor=M.+Ducaauthor=R.+Benhida&title=Targeting+DNA+base+pair+mismatch+with+artificial+nucleobases.+Advances+and+perspectives+in+triple+helix+strategy&doi=10.1039%2FC0OB00418A"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting DNA base pair mismatch with artificial nucleobases: Advances and perspectives in triple helix strategy</span></div><div class="casAuthors">Malnuit, Vincent; Duca, Maria; Benhida, Rachid</div><div class="citationInfo"><span class="NLM_cas:title">Organic & Biomolecular Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">326-336</span>CODEN:
                <span class="NLM_cas:coden">OBCRAK</span>;
        ISSN:<span class="NLM_cas:issn">1477-0520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  This review, divided into three sections, describes the contribution of the chemists' community to the development and application of triple helix strategy by using artificial nucleic acids, particularly for the recognition of DNA sequences incorporating base pair inversions.  Firstly, the development of nucleobases that recognize CG inversion is surveyed followed secondly by specific recognition of TA inverted base pair.  Finally, we point out in the last section recent perspectives and applications, driven from knowledge in nucleic acids interactions, in the growing field of nanotechnol. and supramol. chem. at the border area of physics, chem. and mol. biol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGriqAimgo1nfLVg90H21EOLACvtfcHk0lhQGpc3-L6LeQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1SrsbnN&md5=fa698f0c67f4cb1fa20dcc96a152d312</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1039%2FC0OB00418A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC0OB00418A%26sid%3Dliteratum%253Aachs%26aulast%3DMalnuit%26aufirst%3DV.%26aulast%3DDuca%26aufirst%3DM.%26aulast%3DBenhida%26aufirst%3DR.%26atitle%3DTargeting%2520DNA%2520base%2520pair%2520mismatch%2520with%2520artificial%2520nucleobases.%2520Advances%2520and%2520perspectives%2520in%2520triple%2520helix%2520strategy%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2011%26volume%3D9%26spage%3D326%26epage%3D336%26doi%3D10.1039%2FC0OB00418A" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vo, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tran, T. P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Staedel, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benhida, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Darfeuille, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Giorgio, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duca, M.</span></span> <span> </span><span class="NLM_article-title">Oncogenic microRNAs biogenesis as a drug target: Structure-activity relationship studies on new aminoglycoside conjugates</span>. <i>Chem. - Eur. J.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">5350</span>â <span class="NLM_lpage">5362</span>, <span class="refDoi">Â DOI: 10.1002/chem.201505094</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1002%2Fchem.201505094" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=26928593" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC28XjtlOit7w%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2016&pages=5350-5362&author=D.+D.+Voauthor=T.+P.+A.+Tranauthor=C.+Staedelauthor=R.+Benhidaauthor=F.+Darfeuilleauthor=A.+Di+Giorgioauthor=M.+Duca&title=Oncogenic+microRNAs+biogenesis+as+a+drug+target%3A+Structure-activity+relationship+studies+on+new+aminoglycoside+conjugates&doi=10.1002%2Fchem.201505094"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Oncogenic MicroRNAs Biogenesis as a Drug Target: Structure-Activity Relationship Studies on New Aminoglycoside Conjugates</span></div><div class="casAuthors">Vo, Duc Duy; Tran, Thi Phuong Anh; Staedel, Cathy; Benhida, Rachid; Darfeuille, Fabien; Di Giorgio, Audrey; Duca, Maria</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry - A European Journal</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">5350-5362</span>CODEN:
                <span class="NLM_cas:coden">CEUJED</span>;
        ISSN:<span class="NLM_cas:issn">0947-6539</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">MicroRNAs (miRNAs) are a recently discovered category of small RNA mols. that regulate gene expression at the posttranscriptional level.  Accumulating evidence indicates that miRNAs are aberrantly expressed in a variety of human cancers and that the inhibition of these oncogenic miRNAs could find application in the therapy of different types of cancer.  Herein, the authors describe the synthesis and biol. evaluation of new small-mol. drugs that target oncogenic miRNAs prodn.  In particular, the authors chose to target two miRNAs (i.e., miRNA-372 and -373) implicated in various types of cancer, such as gastric cancer.  Their precursors (premiRNAs) are overexpressed in cancer cells and lead to mature miRNAs after cleavage of their stem-loop structure by the enzyme Dicer in the cytoplasm.  Some of the newly synthesized conjugates can inhibit Dicer processing of the targeted premiRNAs in vitro with increased efficacy relative to the previous results (D.D. Vo et al., ACS Chem. Biol. 2014, 9, 711-721) and, more importantly, to inhibit proliferations of adenocarcinoma gastric cancer (AGS) cells overexpressing these miRNAs, thus representing promising leads for future drug development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrwxgnq0HLdIrVg90H21EOLACvtfcHk0lhQGpc3-L6LeQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XjtlOit7w%253D&md5=cb53a7beb631a1249e0c7f0ca1dc43b7</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1002%2Fchem.201505094&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fchem.201505094%26sid%3Dliteratum%253Aachs%26aulast%3DVo%26aufirst%3DD.%2BD.%26aulast%3DTran%26aufirst%3DT.%2BP.%2BA.%26aulast%3DStaedel%26aufirst%3DC.%26aulast%3DBenhida%26aufirst%3DR.%26aulast%3DDarfeuille%26aufirst%3DF.%26aulast%3DDi%2BGiorgio%26aufirst%3DA.%26aulast%3DDuca%26aufirst%3DM.%26atitle%3DOncogenic%2520microRNAs%2520biogenesis%2520as%2520a%2520drug%2520target%253A%2520Structure-activity%2520relationship%2520studies%2520on%2520new%2520aminoglycoside%2520conjugates%26jtitle%3DChem.%2520-%2520Eur.%2520J.%26date%3D2016%26volume%3D22%26spage%3D5350%26epage%3D5362%26doi%3D10.1002%2Fchem.201505094" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Staedel, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tran, T. P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giraud, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Darfeuille, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Giorgio, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tourasse, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salin, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uriac, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duca, M.</span></span> <span> </span><span class="NLM_article-title">Modulation of oncogenic miRNA biogenesis using functionalized polyamines</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1667</span>, <span class="refDoi">Â DOI: 10.1038/s41598-018-20053-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1038%2Fs41598-018-20053-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=29374231" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A280%3ADC%252BC1MvkslehtQ%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2018&pages=1667&author=C.+Staedelauthor=T.+P.+A.+Tranauthor=J.+Giraudauthor=F.+Darfeuilleauthor=A.+Di+Giorgioauthor=N.+J.+Tourasseauthor=F.+Salinauthor=P.+Uriacauthor=M.+Duca&title=Modulation+of+oncogenic+miRNA+biogenesis+using+functionalized+polyamines&doi=10.1038%2Fs41598-018-20053-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Modulation of oncogenic miRNA biogenesis using functionalized polyamines</span></div><div class="casAuthors">Staedel Cathy; Darfeuille Fabien; Tourasse Nicolas J; Tran Thi Phuong Anh; Di Giorgio Audrey; Duca Maria; Giraud Julie; Salin Franck; Uriac Philippe</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1667</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">MicroRNAs are key factors in the regulation of gene expression and their deregulation has been directly linked to various pathologies such as cancer.  The use of small molecules to tackle the overexpression of oncogenic miRNAs has proved its efficacy and holds the promise for therapeutic applications.  Here we describe the screening of a 640-compound library and the identification of polyamine derivatives interfering with in vitro Dicer-mediated processing of the oncogenic miR-372 precursor (pre-miR-372).  The most active inhibitor is a spermine-amidine conjugate that binds to the pre-miR-372 with a KD of 0.15 Î¼M, and inhibits its in vitro processing with a IC50 of 1.06 Î¼M.  The inhibition of miR-372 biogenesis was confirmed in gastric cancer cells overexpressing miR-372 and a specific inhibition of proliferation through de-repression of the tumor suppressor LATS2 protein, a miR-372 target, was observed.  This compound modifies the expression of a small set of miRNAs and its selective biological activity has been confirmed in patient-derived ex vivo cultures of gastric carcinoma.  Polyamine derivatives are promising starting materials for future studies about the inhibition of oncogenic miRNAs and, to the best of our knowledge, this is the first report about the application of functionalized polyamines as miRNAs interfering agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQbr8gYxuA1hg5nADWVYHg1fW6udTcc2ebnlLc1AYxtFbntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1MvkslehtQ%253D%253D&md5=d000435073e20bbb43b7787de637aaf4</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1038%2Fs41598-018-20053-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-018-20053-5%26sid%3Dliteratum%253Aachs%26aulast%3DStaedel%26aufirst%3DC.%26aulast%3DTran%26aufirst%3DT.%2BP.%2BA.%26aulast%3DGiraud%26aufirst%3DJ.%26aulast%3DDarfeuille%26aufirst%3DF.%26aulast%3DDi%2BGiorgio%26aufirst%3DA.%26aulast%3DTourasse%26aufirst%3DN.%2BJ.%26aulast%3DSalin%26aufirst%3DF.%26aulast%3DUriac%26aufirst%3DP.%26aulast%3DDuca%26aufirst%3DM.%26atitle%3DModulation%2520of%2520oncogenic%2520miRNA%2520biogenesis%2520using%2520functionalized%2520polyamines%26jtitle%3DSci.%2520Rep.%26date%3D2018%26volume%3D8%26spage%3D1667%26doi%3D10.1038%2Fs41598-018-20053-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murata, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fukuzumi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Umemoto, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakatani, K.</span></span> <span> </span><span class="NLM_article-title">Xanthone derivatives as potential inhibitors of miRNA processing by human Dicer: Targeting secondary structures of pre-miRNA by small molecules</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">252</span>â <span class="NLM_lpage">255</span>, <span class="refDoi">Â DOI: 10.1016/j.bmcl.2012.10.108</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1016%2Fj.bmcl.2012.10.108" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=23164709" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC38XhslSrsLnO" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2013&pages=252-255&author=A.+Murataauthor=T.+Fukuzumiauthor=S.+Umemotoauthor=K.+Nakatani&title=Xanthone+derivatives+as+potential+inhibitors+of+miRNA+processing+by+human+Dicer%3A+Targeting+secondary+structures+of+pre-miRNA+by+small+molecules&doi=10.1016%2Fj.bmcl.2012.10.108"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Xanthone derivatives as potential inhibitors of miRNA processing by human Dicer: Targeting secondary structures of pre-miRNA by small molecules</span></div><div class="casAuthors">Murata, Asako; Fukuzumi, Takeo; Umemoto, Shiori; Nakatani, Kazuhiko</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">252-255</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">In recent years, various biol. processes have been found to be regulated by miRNA-mediated gene silencing.  A small mol. that modulate the miRNA pathway will provide the biol. tool for elucidating mechanisms of miRNA-mediated gene regulation, and can be the drug lead for miRNA related diseases.  In this study, the authors demonstrated that an aminoalkoxy-substituted thioxanthone deriv. interferes Dicer-mediated processing of pre-miRNA.  Information about the interaction between these xanthone derivs. and pre-miRNAs will enable the authors to design and develop new small mol.-based inhibitors for miRNA pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqSvmgQp8NxvLVg90H21EOLACvtfcHk0ljZ1ZQ6hzPGnw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhslSrsLnO&md5=6c75fc852fbea26d548479b2be477dc9</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2012.10.108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2012.10.108%26sid%3Dliteratum%253Aachs%26aulast%3DMurata%26aufirst%3DA.%26aulast%3DFukuzumi%26aufirst%3DT.%26aulast%3DUmemoto%26aufirst%3DS.%26aulast%3DNakatani%26aufirst%3DK.%26atitle%3DXanthone%2520derivatives%2520as%2520potential%2520inhibitors%2520of%2520miRNA%2520processing%2520by%2520human%2520Dicer%253A%2520Targeting%2520secondary%2520structures%2520of%2520pre-miRNA%2520by%2520small%2520molecules%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2013%26volume%3D23%26spage%3D252%26epage%3D255%26doi%3D10.1016%2Fj.bmcl.2012.10.108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niu, H. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, Z. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, B. F.</span></span> <span> </span><span class="NLM_article-title">Overexpression of microRNA-1 impairs cardiac contractile function by damaging sarcomere assembly</span>. <i>Cardiovasc. Res.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>95</i></span>,  <span class="NLM_fpage">385</span>â <span class="NLM_lpage">393</span>, <span class="refDoi">Â DOI: 10.1093/cvr/cvs196</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1093%2Fcvr%2Fcvs196" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=22719074" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtFSlt73I" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=95&publication_year=2012&pages=385-393&author=J.+Aiauthor=R.+Zhangauthor=X.+Gaoauthor=H.+F.+Niuauthor=N.+Wangauthor=Y.+Xuauthor=Y.+Liauthor=N.+Maauthor=L.+H.+Sunauthor=Z.+W.+Panauthor=W.+M.+Liauthor=B.+F.+Yang&title=Overexpression+of+microRNA-1+impairs+cardiac+contractile+function+by+damaging+sarcomere+assembly&doi=10.1093%2Fcvr%2Fcvs196"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Overexpression of microRNA-1 impairs cardiac contractile function by damaging sarcomere assembly</span></div><div class="casAuthors">Ai, Jing; Zhang, Rong; Gao, Xu; Niu, Hui-Fang; Wang, Ning; Xu, Yi; Li, Yue; Ma, Ning; Sun, Li-Hua; Pan, Zhen-Wei; Li, Wei-Min; Yang, Bao-Feng</div><div class="citationInfo"><span class="NLM_cas:title">Cardiovascular Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">95</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">385-393</span>CODEN:
                <span class="NLM_cas:coden">CVREAU</span>;
        ISSN:<span class="NLM_cas:issn">0008-6363</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Aims: The purpose of the present study was to evaluate the effects of overexpression of microRNA-1 (miR-1) on cardiac contractile function and the potential mol. mechanisms.  Methods and results: Transgenic (Tg) mice (C57BL/6) for cardiac-specific overexpression of miR-1 driven by the Î±-myosin heavy chain promoter were generated and identified by real-time reverse-transcription polymerase chain reaction with left ventricular samples.  We found an age-dependent decrease in the heart function in Tg mice by pressure-vol. loop anal.  Histol. anal. and electron microscopy displayed short sarcomeres with the loss of the clear zone and H-zone as well as myofibril fragmentation and deliquescence in Tg mice.  Further studies demonstrated miR-1 post-transcriptionally down-regulated the expression of calmodulin (CaM) and cardiac myosin light chain kinase (cMLCK) proteins by targeting the 3'UTRs of MYLK3, CALM1, and CALM2 genes, leading to decreased phosphorylations of myosin light chain 2v (MLC2v) and cardiac myosin binding protein-C (cMyBP-C).  Knockdown of miR-1 by locked nucleic acid-modified anti-miR-1 antisense (LNA-antimiR-1) mitigated the adverse changes of cardiac function assocd. with overexpression of miR-1.  Conclusion: miR-1 induces adverse structural remodelling to impair cardiac contractile function.  Targeting cMLCK and CaM likely underlies the detrimental effects of miR-1 on structural components of muscles related to the contractile machinery.  Our study provides the first evidence that miRNAs cause adverse structural remodelling of the heart.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo9qMFjoVpI-LVg90H21EOLACvtfcHk0ljZ1ZQ6hzPGnw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtFSlt73I&md5=3864a1332fc6d5b69410287165da30bb</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1093%2Fcvr%2Fcvs196&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fcvr%252Fcvs196%26sid%3Dliteratum%253Aachs%26aulast%3DAi%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DR.%26aulast%3DGao%26aufirst%3DX.%26aulast%3DNiu%26aufirst%3DH.%2BF.%26aulast%3DWang%26aufirst%3DN.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DMa%26aufirst%3DN.%26aulast%3DSun%26aufirst%3DL.%2BH.%26aulast%3DPan%26aufirst%3DZ.%2BW.%26aulast%3DLi%26aufirst%3DW.%2BM.%26aulast%3DYang%26aufirst%3DB.%2BF.%26atitle%3DOverexpression%2520of%2520microRNA-1%2520impairs%2520cardiac%2520contractile%2520function%2520by%2520damaging%2520sarcomere%2520assembly%26jtitle%3DCardiovasc.%2520Res.%26date%3D2012%26volume%3D95%26spage%3D385%26epage%3D393%26doi%3D10.1093%2Fcvr%2Fcvs196" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tan, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span> <span> </span><span class="NLM_article-title">Small molecular inhibitors of miR-1 identified from photocycloadducts of acetylenes with 2-methoxy-1,4-naphthalenequinone</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">6124</span>â <span class="NLM_lpage">6131</span>, <span class="refDoi">Â DOI: 10.1016/j.bmc.2013.04.058</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1016%2Fj.bmc.2013.04.058" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=23685176" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC3sXnvVGktr4%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2013&pages=6124-6131&author=S.+B.+Tanauthor=C.+Huangauthor=X.+Chenauthor=Y.+Wuauthor=M.+Zhouauthor=C.+Zhangauthor=Y.+Zhang&title=Small+molecular+inhibitors+of+miR-1+identified+from+photocycloadducts+of+acetylenes+with+2-methoxy-1%2C4-naphthalenequinone&doi=10.1016%2Fj.bmc.2013.04.058"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Small molecular inhibitors of miR-1 identified from photocycloadducts of acetylenes with 2-methoxy-1,4-naphthalenequinone</span></div><div class="casAuthors">Tan, Su-Bee; Huang, Chengmei; Chen, Xuejiao; Wu, Yihan; Zhou, Mi; Zhang, Chenyu; Zhang, Yan</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">6124-6131</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Small mols. which can modulate endogenous microRNAs are important chem. tools to study microRNA regulational network.  In this Letter the authors screened the [2+2] photocycloadducts of 2-methoxy-1,4-naphthalenequinone with a series of aryl acetylenes on their activity to modulate endogenous microRNAs.  A potent inhibitor of the muscle-specific miR-1 which is closely related with cardiac development and disease was identified.  The small mol. inhibitor was the cyclobutene type product derived from the photocycloaddn. of 2-methoxy-1,4-naphthalenequinone with tert-Bu (5-(phenylethynyl)quinolin-8-yl) carbonate.  Analogs of the small mol. inhibitor were then prepd. using similar photocycloaddn. reactions for evaluation on inhibition activity on miR-1 to provide structure-activity relationship of the miR-1 inhibitor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqO2MDHBYgICLVg90H21EOLACvtfcHk0lg7JdIdZ4kT1A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXnvVGktr4%253D&md5=efefc7c7101b3a1d7f668cd8000eb4f8</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2013.04.058&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2013.04.058%26sid%3Dliteratum%253Aachs%26aulast%3DTan%26aufirst%3DS.%2BB.%26aulast%3DHuang%26aufirst%3DC.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DY.%26atitle%3DSmall%2520molecular%2520inhibitors%2520of%2520miR-1%2520identified%2520from%2520photocycloadducts%2520of%2520acetylenes%2520with%25202-methoxy-1%252C4-naphthalenequinone%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2013%26volume%3D21%26spage%3D6124%26epage%3D6131%26doi%3D10.1016%2Fj.bmc.2013.04.058" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chandrasekhar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pushpavalli, S. N. C. V. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chatla, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mukhopadhyay, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ganganna, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vijeender, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Srihari, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reddy, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramaiah, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhadra, U.</span></span> <span> </span><span class="NLM_article-title"><i>aza</i>-Flavanones as potent cross-species microRNA inhibitors that arrest cell cycle</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">645</span>â <span class="NLM_lpage">648</span>, <span class="refDoi">Â DOI: 10.1016/j.bmcl.2011.10.061</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1016%2Fj.bmcl.2011.10.061" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=22100313" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC38XivVyqtQ%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2012&pages=645-648&author=S.+Chandrasekharauthor=S.+N.+C.+V.+L.+Pushpavalliauthor=S.+Chatlaauthor=D.+Mukhopadhyayauthor=B.+Gangannaauthor=K.+Vijeenderauthor=P.+Srihariauthor=C.+R.+Reddyauthor=M.+J.+Ramaiahauthor=U.+Bhadra&title=aza-Flavanones+as+potent+cross-species+microRNA+inhibitors+that+arrest+cell+cycle&doi=10.1016%2Fj.bmcl.2011.10.061"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">aza-Flavanones as potent cross-species microRNA inhibitors that arrest cell cycle</span></div><div class="casAuthors">Chandrasekhar, Srivari; Pushpavalli, Sreerangam N. C. V. L.; Chatla, Srinivas; Mukhopadhyay, Debasmita; Ganganna, Bogonda; Vijeender, Kandi; Srihari, Pabbaraja; Reddy, Chada Raji; Janaki Ramaiah, M.; Bhadra, Utpal</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">645-648</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Aza-Flavanones have been identified as a new class of selective microRNA inhibitors.  These compds. were found to arrest cell cycle via a novel cross species microRNA-dependent regulatory pathway interpreting an unexpected link between cell cycle arrest and microRNA mediated control in cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAxulyPUhrXrVg90H21EOLACvtfcHk0lg7JdIdZ4kT1A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XivVyqtQ%253D%253D&md5=e16a5cc549530d0f2f2bba6e12e533ff</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2011.10.061&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2011.10.061%26sid%3Dliteratum%253Aachs%26aulast%3DChandrasekhar%26aufirst%3DS.%26aulast%3DPushpavalli%26aufirst%3DS.%2BN.%2BC.%2BV.%2BL.%26aulast%3DChatla%26aufirst%3DS.%26aulast%3DMukhopadhyay%26aufirst%3DD.%26aulast%3DGanganna%26aufirst%3DB.%26aulast%3DVijeender%26aufirst%3DK.%26aulast%3DSrihari%26aufirst%3DP.%26aulast%3DReddy%26aufirst%3DC.%2BR.%26aulast%3DRamaiah%26aufirst%3DM.%2BJ.%26aulast%3DBhadra%26aufirst%3DU.%26atitle%3Daza-Flavanones%2520as%2520potent%2520cross-species%2520microRNA%2520inhibitors%2520that%2520arrest%2520cell%2520cycle%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2012%26volume%3D22%26spage%3D645%26epage%3D648%26doi%3D10.1016%2Fj.bmcl.2011.10.061" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicolas, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marks, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sander, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lerro, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buendia, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mason, W. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moloshok, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bort, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zaret, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, J. M.</span></span> <span> </span><span class="NLM_article-title">miR-122, a mammalian liver-specific microRNA, is processed from hcr mRNA and may downregulate the high affinity cationic amino acid transporter CAT-1</span>. <i>RNA Biol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">106</span>â <span class="NLM_lpage">113</span>, <span class="refDoi">Â DOI: 10.4161/rna.1.2.1066</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.4161%2Frna.1.2.1066" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=17179747" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BD2MXlsFSrt74%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2004&pages=106-113&author=J.+Changauthor=E.+Nicolasauthor=D.+Marksauthor=C.+Sanderauthor=A.+Lerroauthor=M.+A.+Buendiaauthor=C.+Xuauthor=W.+S.+Masonauthor=T.+Moloshokauthor=R.+Bortauthor=K.+S.+Zaretauthor=J.+M.+Taylor&title=miR-122%2C+a+mammalian+liver-specific+microRNA%2C+is+processed+from+hcr+mRNA+and+may+downregulate+the+high+affinity+cationic+amino+acid+transporter+CAT-1&doi=10.4161%2Frna.1.2.1066"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">MiR-122, a mammalian liver-specific microRNA, is processed from hcr mRNA and may downregulate the high affinity cationic amino acid transporter CAT-1</span></div><div class="casAuthors">Chang, Jinhong; Nicolas, Emmanuelle; Marks, Debora; Sander, Chris; Lerro, Anthony; Buendia, Marie Annick; Xu, Chunxiao; Mason, William S.; Moloshok, Thomas; Bort, Roque; Zaret, Kenneth S.; Taylor, John M.</div><div class="citationInfo"><span class="NLM_cas:title">RNA Biology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">106-113</span>CODEN:
                <span class="NLM_cas:coden">RBNIBE</span>;
        ISSN:<span class="NLM_cas:issn">1547-6286</span>.
    
            (<span class="NLM_cas:orgname">Landes Bioscience</span>)
        </div><div class="casAbstract">These studies show that miR-122, a 22-nucleotide microRNA, is derived from a liver-specific noncoding polyadenylated RNA transcribed from the gene hcr.  The exact sequence of miR-122 as well as the adjacent secondary structure within the hcr mRNA are conserved from mammalian species back to fish.  Levels of miR-122 in the mouse liver increase to half maximal values around day 17 of embryogenesis, and reach near maximal levels of 50,000 copies per av. cell before birth.  Lewis et al. (2003) predicted the cationic amino acid transporter (CAT-1 or SLC7A1) as a miR-122 target.  CAT-1 protein and its mRNA are expressed in all mammalian tissues but with lower levels in adult liver.  Furthermore, during mouse liver development CAT-1 mRNA decreases in an almost inverse correlation with miR-122.  Eight potential miR-122 target sites were predicted within the human CAT-1 mRNA, with six in the 3'-untranslated region.  Using a reporter construct it was found that just three of the predicted sites, linked in a 400-nucleotide sequence from human CAT-1, acted with synergy and were sufficient to strongly inhibit protein synthesis and reduce mRNA levels.  In summary, these studies followed the accumulation during development of miR-122 from its mRNA precursor, hcr, through to identification of what may be a specific mRNA target, CAT-1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGocW-mOgnLCh7Vg90H21EOLACvtfcHk0lg7JdIdZ4kT1A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXlsFSrt74%253D&md5=7de7e091c98feac574c4a6189408879a</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.4161%2Frna.1.2.1066&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252Frna.1.2.1066%26sid%3Dliteratum%253Aachs%26aulast%3DChang%26aufirst%3DJ.%26aulast%3DNicolas%26aufirst%3DE.%26aulast%3DMarks%26aufirst%3DD.%26aulast%3DSander%26aufirst%3DC.%26aulast%3DLerro%26aufirst%3DA.%26aulast%3DBuendia%26aufirst%3DM.%2BA.%26aulast%3DXu%26aufirst%3DC.%26aulast%3DMason%26aufirst%3DW.%2BS.%26aulast%3DMoloshok%26aufirst%3DT.%26aulast%3DBort%26aufirst%3DR.%26aulast%3DZaret%26aufirst%3DK.%2BS.%26aulast%3DTaylor%26aufirst%3DJ.%2BM.%26atitle%3DmiR-122%252C%2520a%2520mammalian%2520liver-specific%2520microRNA%252C%2520is%2520processed%2520from%2520hcr%2520mRNA%2520and%2520may%2520downregulate%2520the%2520high%2520affinity%2520cationic%2520amino%2520acid%2520transporter%2520CAT-1%26jtitle%3DRNA%2520Biol.%26date%3D2004%26volume%3D1%26spage%3D106%26epage%3D113%26doi%3D10.4161%2Frna.1.2.1066" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Young, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Connelly, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grohmann, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deiters, A.</span></span> <span> </span><span class="NLM_article-title">Small molecule modifiers of microRNA miR-122 function for the treatment of hepatitis C virus infection and hepatocellular carcinoma</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>132</i></span>,  <span class="NLM_fpage">7976</span>â <span class="NLM_lpage">7981</span>, <span class="refDoi">Â DOI: 10.1021/ja910275u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja910275u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC3cXmt12ms7w%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=132&publication_year=2010&pages=7976-7981&author=D.+D.+Youngauthor=C.+M.+Connellyauthor=C.+Grohmannauthor=A.+Deiters&title=Small+molecule+modifiers+of+microRNA+miR-122+function+for+the+treatment+of+hepatitis+C+virus+infection+and+hepatocellular+carcinoma&doi=10.1021%2Fja910275u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Small Molecule Modifiers of MicroRNA miR-122 Function for the Treatment of Hepatitis C Virus Infection and Hepatocellular Carcinoma</span></div><div class="casAuthors">Young, Douglas D.; Connelly, Colleen M.; Grohmann, Christoph; Deiters, Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">132</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">7976-7981</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">MicroRNAs are a recently discovered new class of important endogenous regulators of gene function.  Aberrant regulation of microRNAs has been linked to various human diseases, most importantly cancer.  Small mol. intervention of microRNA misregulation has the potential to provide new therapeutic approaches to such diseases.  Here, we report the first small mol. inhibitors and activators of the liver-specific microRNA miR-122.  This microRNA is the most abundant microRNA in the liver and is involved in hepatocellular carcinoma development and hepatitis C virus (HCV) infection.  Our small mol. inhibitors reduce viral replication in liver cells and represent a new approach to the treatment of HCV infections.  Moreover, small mol. activation of miR-122 in liver cancer cells selectively induced apoptosis through caspase activation, thus having implications in cancer chemotherapy.  In addn. to providing a new approach for the development of therapeutics, small mol. modifiers of miR-122 function are unique tools for exploring miR-122 biogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJ6MkKhmyG_7Vg90H21EOLACvtfcHk0lgK2KYKhwKgkw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXmt12ms7w%253D&md5=3aefaa3cf2af65e5a45fc52fbeaf8494</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1021%2Fja910275u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja910275u%26sid%3Dliteratum%253Aachs%26aulast%3DYoung%26aufirst%3DD.%2BD.%26aulast%3DConnelly%26aufirst%3DC.%2BM.%26aulast%3DGrohmann%26aufirst%3DC.%26aulast%3DDeiters%26aufirst%3DA.%26atitle%3DSmall%2520molecule%2520modifiers%2520of%2520microRNA%2520miR-122%2520function%2520for%2520the%2520treatment%2520of%2520hepatitis%2520C%2520virus%2520infection%2520and%2520hepatocellular%2520carcinoma%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2010%26volume%3D132%26spage%3D7976%26epage%3D7981%26doi%3D10.1021%2Fja910275u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boyd, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malstrom, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Subramanian, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Venkatesh, L. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaeper, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elangovan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DâSa-Eipper, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chinnadurai, G.</span></span> <span> </span><span class="NLM_article-title">Adenovirus E1B 19 kDa and Bcl-2 proteins interact with a common set of cellular proteins</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>79</i></span>,  <span class="NLM_fpage">341</span>â <span class="NLM_lpage">351</span>, <span class="refDoi">Â DOI: 10.1016/0092-8674(94)90202-X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1016%2F0092-8674%2894%2990202-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=7954800" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADyaK2MXhvFWrtr4%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=1994&pages=341-351&author=J.+M.+Boydauthor=S.+Malstromauthor=T.+Subramanianauthor=L.+K.+Venkateshauthor=U.+Schaeperauthor=B.+Elangovanauthor=C.+D%E2%80%99Sa-Eipperauthor=G.+Chinnadurai&title=Adenovirus+E1B+19+kDa+and+Bcl-2+proteins+interact+with+a+common+set+of+cellular+proteins&doi=10.1016%2F0092-8674%2894%2990202-X"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Adenovirus E1B 19 kDa and Bcl-2 proteins interact with a common set of cellular proteins</span></div><div class="casAuthors">Boyd, Janice M.; Malstrom, Scott; Subramanian, T.; Venkatesh, L. K.; Schaeper, Ute; Elangovan, B.; D'Sa-Eipper, Cleta; Chinnadurai, G.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, Massachusetts)</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">79</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">341-51</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Adenovirus E1B kDa protein protects against cell death induced by viral infection and certain external stimuli.  The Bcl-2 protein can functionally substitute for the E1B 19 kDa protein.  To identify cellular targets for the 19 kDa protein, the authors used the two-hybrid screen in yeast.  The authors have isolated cDNAs for three different proteins, designated Nip1, Nip2, and Nip3, that interact with the 19 kDa protein.  Mutational anal. indicates that these proteins do not assoc. with 19 kDa mutants defective in suppression of cell death, suggesting a correlation between interaction of these proteins and suppression of cell death.  These proteins also assoc. with discrete sequence motifs in the Bcl-2 protein that are homologous to motifs of the 19 kDa protein.  These results suggest that two diverse proteins, the E1B 19 kDa and the Bcl-2 proteins, promote cell survival through interaction with a common set of cellular proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCbKglHHNc7bVg90H21EOLACvtfcHk0lgK2KYKhwKgkw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXhvFWrtr4%253D&md5=208a600c230fa7df99347cc8314bbd23</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1016%2F0092-8674%2894%2990202-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0092-8674%252894%252990202-X%26sid%3Dliteratum%253Aachs%26aulast%3DBoyd%26aufirst%3DJ.%2BM.%26aulast%3DMalstrom%26aufirst%3DS.%26aulast%3DSubramanian%26aufirst%3DT.%26aulast%3DVenkatesh%26aufirst%3DL.%2BK.%26aulast%3DSchaeper%26aufirst%3DU.%26aulast%3DElangovan%26aufirst%3DB.%26aulast%3DD%25E2%2580%2599Sa-Eipper%26aufirst%3DC.%26aulast%3DChinnadurai%26aufirst%3DG.%26atitle%3DAdenovirus%2520E1B%252019%2520kDa%2520and%2520Bcl-2%2520proteins%2520interact%2520with%2520a%2520common%2520set%2520of%2520cellular%2520proteins%26jtitle%3DCell%26date%3D1994%26volume%3D79%26spage%3D341%26epage%3D351%26doi%3D10.1016%2F0092-8674%2894%2990202-X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ham, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seo, H. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cha, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mun, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoon, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hwang, K. C.</span></span> <span> </span><span class="NLM_article-title">Small molecule-mediated upregulation of microRNA targeting a key cell death modulator BNIP3 improves cardiac function following ischemic injury</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">23472</span>, <span class="refDoi">Â DOI: 10.1038/srep23472</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1038%2Fsrep23472" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=27008992" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC28XkvVCgtLw%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2016&pages=23472&author=S.+Y.+Leeauthor=S.+Leeauthor=E.+Choiauthor=O.+Hamauthor=C.+Y.+Leeauthor=J.+Leeauthor=H.+H.+Seoauthor=M.+J.+Chaauthor=B.+Munauthor=Y.+Leeauthor=C.+Yoonauthor=K.+C.+Hwang&title=Small+molecule-mediated+upregulation+of+microRNA+targeting+a+key+cell+death+modulator+BNIP3+improves+cardiac+function+following+ischemic+injury&doi=10.1038%2Fsrep23472"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Small molecule-mediated up-regulation of microRNA targeting a key cell death modulator BNIP3 improves cardiac function following ischemic injury</span></div><div class="casAuthors">Lee, Se-Yeon; Lee, Seahyoung; Choi, Eunhyun; Ham, Onju; Lee, Chang Youn; Lee, Jiyun; Seo, Hyang-Hee; Cha, Min-Ji; Mun, Bohyun; Lee, Yunmi; Yoon, Cheesoon; Hwang, Ki-Chul</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">23472</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Genetic ablation of BCL2/adenovirus E1B 19 kDa protein-interacting protein 3 (BNIP3), an essential regulator of cardiac cell death, is an effective way to prevent cardiac cell death triggered by pathol. conditions.  However, currently there exists no known means, such as inhibitors, to down-regulate BNIP3 in mature heart.  Here, we report that a small mol. inducer of microRNA-182 (miR-182) suppressed ischemia/reperfusion (I/R)-induced cardiac cell death by down-regulating BNIP3.  We first selected miR-182 as a potent BNIP3-targeting miRNA based on miRNA-target prediction databases and empirical data.  The subsequent screening of small mols. for inducing miR-182 expression identified Kenpaullone as a hit compd.  Both exogenous miR-182 and Kenpaullone significantly suppressed hypoxia-induced cardiomyocyte death in vitro.  To investigate the effect of changing substituents of Kenpaullone on miR-182 expression, we synthesized 9 derivs. of Kenpaullone.  Among these derivs., compd. 5 showed significantly improved ability to induce miR-182 expression.  The results of the in vivo study showed that compd. 5 significantly improved heart function following I/R-injury in rats.  Our study provides strong evidence that the small mol.-mediated up-regulation of miRNAs is a viable strategy to down-regulate target proteins with no known chem. inhibitor and that compd. 5 may have potential to prevent I/R-inflicted cardiac cell death.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGos9r_v2BG3CbVg90H21EOLACvtfcHk0lgK2KYKhwKgkw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XkvVCgtLw%253D&md5=2d636050f6e3473ce50585b7e2805550</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1038%2Fsrep23472&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep23472%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DS.%2BY.%26aulast%3DLee%26aufirst%3DS.%26aulast%3DChoi%26aufirst%3DE.%26aulast%3DHam%26aufirst%3DO.%26aulast%3DLee%26aufirst%3DC.%2BY.%26aulast%3DLee%26aufirst%3DJ.%26aulast%3DSeo%26aufirst%3DH.%2BH.%26aulast%3DCha%26aufirst%3DM.%2BJ.%26aulast%3DMun%26aufirst%3DB.%26aulast%3DLee%26aufirst%3DY.%26aulast%3DYoon%26aufirst%3DC.%26aulast%3DHwang%26aufirst%3DK.%2BC.%26atitle%3DSmall%2520molecule-mediated%2520upregulation%2520of%2520microRNA%2520targeting%2520a%2520key%2520cell%2520death%2520modulator%2520BNIP3%2520improves%2520cardiac%2520function%2520following%2520ischemic%2520injury%26jtitle%3DSci.%2520Rep.%26date%3D2016%26volume%3D6%26spage%3D23472%26doi%3D10.1038%2Fsrep23472" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jin, H. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oda, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skalsky, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bethel, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shepherd, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, W. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sabouri-Ghomi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cullen, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rajewsky, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, C.</span></span> <span> </span><span class="NLM_article-title">MicroRNA-17 â¼ 92 plays a causative role in lymphomagenesis by coordinating multiple oncogenic pathways</span>. <i>EMBO J.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">2377</span>â <span class="NLM_lpage">2391</span>, <span class="refDoi">Â DOI: 10.1038/emboj.2013.178</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1038%2Femboj.2013.178" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=23921550" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1Skur3N" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2013&pages=2377-2391&author=H.+Y.+Jinauthor=H.+Odaauthor=M.+Laiauthor=R.+L.+Skalskyauthor=K.+Bethelauthor=J.+Shepherdauthor=S.+G.+Kangauthor=W.+H.+Liuauthor=M.+Sabouri-Ghomiauthor=B.+R.+Cullenauthor=K.+Rajewskyauthor=C.+Xiao&title=MicroRNA-17+%E2%88%BC+92+plays+a+causative+role+in+lymphomagenesis+by+coordinating+multiple+oncogenic+pathways&doi=10.1038%2Femboj.2013.178"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">MicroRNA-17-92 plays a causative role in lymphomagenesis by coordinating multiple oncogenic pathways</span></div><div class="casAuthors">Jin, Hyun Yong; Oda, Hiroyo; Lai, Maoyi; Skalsky, Rebecca L.; Bethel, Kelly; Shepherd, Jovan; Kang, Seung Goo; Liu, Wen-Hsien; Sabouri-Ghomi, Mohsen; Cullen, Bryan R.; Rajewsky, Klaus; Xiao, Changchun</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Journal</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">2377-2391</span>CODEN:
                <span class="NLM_cas:coden">EMJODG</span>;
        ISSN:<span class="NLM_cas:issn">0261-4189</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">MicroRNAs (miRNAs) have been broadly implicated in cancer, but their exact function and mechanism in carcinogenesis remain poorly understood.  Elevated miR-17-92 expression is frequently found in human cancers, mainly due to gene amplification and Myc-mediated transcriptional upregulation.  Here we show that B cell-specific miR-17â¼92 transgenic mice developed lymphomas with high penetrance and that, conversely, Myc-driven lymphomagenesis stringently requires two intact alleles of miR-17â¼92.  We exptl. identified miR-17â¼92 target genes by PAR-CLIP and validated select target genes in miR-17â¼92 transgenic mice.  These analyses demonstrate that miR-17â¼92 drives lymphomagenesis by suppressing the expression of multiple neg. regulators of the PI3K and NFÎºB pathways and by inhibiting the mitochondrial apoptosis pathway.  Accordingly, miR-17-92-driven lymphoma cells exhibited constitutive activation of the PI3K and NFÎºB pathways and chem. inhibition of either pathway reduced tumor size and prolonged the survival of lymphoma-bearing mice.  These findings establish miR-17â¼92 as a powerful cancer driver that coordinates the activation of multiple oncogenic pathways, and demonstrate for the first time that chem. inhibition of miRNA downstream pathways has therapeutic value in treating cancers caused by miRNA dysregulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpy45tQaID4JLVg90H21EOLACvtfcHk0ljYpX2B0rZiQQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1Skur3N&md5=08ef3d1f1f7c0fe1cb343352474a53ee</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1038%2Femboj.2013.178&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Femboj.2013.178%26sid%3Dliteratum%253Aachs%26aulast%3DJin%26aufirst%3DH.%2BY.%26aulast%3DOda%26aufirst%3DH.%26aulast%3DLai%26aufirst%3DM.%26aulast%3DSkalsky%26aufirst%3DR.%2BL.%26aulast%3DBethel%26aufirst%3DK.%26aulast%3DShepherd%26aufirst%3DJ.%26aulast%3DKang%26aufirst%3DS.%2BG.%26aulast%3DLiu%26aufirst%3DW.%2BH.%26aulast%3DSabouri-Ghomi%26aufirst%3DM.%26aulast%3DCullen%26aufirst%3DB.%2BR.%26aulast%3DRajewsky%26aufirst%3DK.%26aulast%3DXiao%26aufirst%3DC.%26atitle%3DMicroRNA-17%2520%25E2%2588%25BC%252092%2520plays%2520a%2520causative%2520role%2520in%2520lymphomagenesis%2520by%2520coordinating%2520multiple%2520oncogenic%2520pathways%26jtitle%3DEMBO%2520J.%26date%3D2013%26volume%3D32%26spage%3D2377%26epage%3D2391%26doi%3D10.1038%2Femboj.2013.178" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Velagapudi, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tran, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haniff, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fallahi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinez, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Childs-Disney, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span> <span> </span><span class="NLM_article-title">Defining RNA-small molecule affinity landscapes enables design of a small molecule inhibitor of an oncogenic noncoding RNA</span>. <i>ACS Cent. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">205</span>â <span class="NLM_lpage">216</span>, <span class="refDoi">Â DOI: 10.1021/acscentsci.7b00009</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acscentsci.7b00009" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjs1GqsLs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2017&pages=205-216&author=S.+P.+Velagapudiauthor=Y.+Luoauthor=T.+Tranauthor=H.+S.+Haniffauthor=Y.+Nakaiauthor=M.+Fallahiauthor=G.+J.+Martinezauthor=J.+L.+Childs-Disneyauthor=M.+D.+Disney&title=Defining+RNA-small+molecule+affinity+landscapes+enables+design+of+a+small+molecule+inhibitor+of+an+oncogenic+noncoding+RNA&doi=10.1021%2Facscentsci.7b00009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Defining RNA-Small Molecule Affinity Landscapes Enables Design of a Small Molecule Inhibitor of an Oncogenic Noncoding RNA</span></div><div class="casAuthors">Velagapudi, Sai Pradeep; Luo, Yiling; Tran, Tuan; Haniff, Hafeez S.; Nakai, Yoshio; Fallahi, Mohammad; Martinez, Gustavo J.; Childs-Disney, Jessica L.; Disney, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Central Science</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">205-216</span>CODEN:
                <span class="NLM_cas:coden">ACSCII</span>;
        ISSN:<span class="NLM_cas:issn">2374-7951</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">RNA drug targets are pervasive in cells but methods to design small mols. that target them are sparse.  Herein, the authors report a general approach to score the affinity and selectivity of RNA motif-small mol. interactions identified via selection.  Named High Throughput Structure-Activity Relationships Through Sequencing (HiT-StARTS), HiT-StARTS is statistical in nature and compares input nucleic acid sequences to selected library members that bind a ligand via high throughput sequencing.  The approach allowed facile definition of the fitness landscape of hundreds of thousands of RNA motif-small mol. binding partners.  These results were mined against folded RNAs in the human transcriptome and identified an avid interaction between a small mol. and the Dicer nuclease-processing site in the oncogenic microRNA (miR)-18a hairpin precursor, which is a member of the miR-17-92 cluster.  Application of the small mol., Targapremir-18a, to prostate cancer cells inhibited prodn. of miR-18a from the cluster, derepressed serine/threonine protein kinase 4 protein (STK4), and triggered apoptosis.  Profiling the cellular targets of Targapremir-18a via Chem. Cross Linking and Isolation by Pull Down (Chem-CLIP), a covalent small mol.-RNA cellular profiling approach, and other studies showed specific binding of the compd. to the miR-18a precursor, revealing broadly applicable factors that govern small mol. drugging of noncoding RNAs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPmptGpjN_4rVg90H21EOLACvtfcHk0ljYpX2B0rZiQQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjs1GqsLs%253D&md5=fefd2fe6e9d637c4aa04fd1e25fb1446</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1021%2Facscentsci.7b00009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facscentsci.7b00009%26sid%3Dliteratum%253Aachs%26aulast%3DVelagapudi%26aufirst%3DS.%2BP.%26aulast%3DLuo%26aufirst%3DY.%26aulast%3DTran%26aufirst%3DT.%26aulast%3DHaniff%26aufirst%3DH.%2BS.%26aulast%3DNakai%26aufirst%3DY.%26aulast%3DFallahi%26aufirst%3DM.%26aulast%3DMartinez%26aufirst%3DG.%2BJ.%26aulast%3DChilds-Disney%26aufirst%3DJ.%2BL.%26aulast%3DDisney%26aufirst%3DM.%2BD.%26atitle%3DDefining%2520RNA-small%2520molecule%2520affinity%2520landscapes%2520enables%2520design%2520of%2520a%2520small%2520molecule%2520inhibitor%2520of%2520an%2520oncogenic%2520noncoding%2520RNA%26jtitle%3DACS%2520Cent.%2520Sci.%26date%3D2017%26volume%3D3%26spage%3D205%26epage%3D216%26doi%3D10.1021%2Facscentsci.7b00009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Velagapudi, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cameron, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haga, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenberg, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lafitte, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duckett, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phinney, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span> <span> </span><span class="NLM_article-title">Design of a small molecule against an oncogenic noncoding RNA</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>113</i></span>,  <span class="NLM_fpage">5898</span>â <span class="NLM_lpage">5903</span>, <span class="refDoi">Â DOI: 10.1073/pnas.1523975113</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1073%2Fpnas.1523975113" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=27170187" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC28Xns1ejurw%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2016&pages=5898-5903&author=S.+P.+Velagapudiauthor=M.+D.+Cameronauthor=C.+L.+Hagaauthor=L.+H.+Rosenbergauthor=M.+Lafitteauthor=D.+R.+Duckettauthor=D.+G.+Phinneyauthor=M.+D.+Disney&title=Design+of+a+small+molecule+against+an+oncogenic+noncoding+RNA&doi=10.1073%2Fpnas.1523975113"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Design of a small molecule against an oncogenic noncoding RNA</span></div><div class="casAuthors">Velagapudi, Sai Pradeep; Cameron, Michael D.; Haga, Christopher L.; Rosenberg, Laura H.; Lafitte, Marie; Duckett, Derek R.; Phinney, Donald G.; Disney, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">113</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">5898-5903</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The design of precision, preclin. therapeutics from sequence is difficult, but advances in this area, particularly those focused on rational design, could quickly transform the sequence of disease-causing gene products into lead modalities.  Herein, we describe the use of Inforna, a computational approach that enables the rational design of small mols. targeting RNA to quickly provide a potent modulator of oncogenic microRNA-96 (miR-96).  We mined the secondary structure of primary microRNA-96 (pri-miR-96) hairpin precursor against a database of RNA motif-small mol. interactions, which identified modules that bound RNA motifs nearby and in the Drosha processing site.  Precise linking of these modules together provided Targaprimir-96 (3), which selectively modulates miR-96 prodn. in cancer cells and triggers apoptosis.  Importantly, the compd. is ineffective on healthy breast cells, and exogenous overexpression of pri-miR-96 reduced compd. potency in breast cancer cells.  Chem. Crosslinking and Isolation by Pull-Down (Chem-CLIP), a small-mol. RNA target validation approach, shows that 3 directly engages pri-miR-96 in breast cancer cells.  In vivo, 3 has a favorable pharmacokinetic profile and decreases tumor burden in a mouse model of triple-neg. breast cancer.  Thus, rational design can quickly produce precision, in vivo bioactive lead small mols. against hard-to-treat cancers by targeting oncogenic noncoding RNAs, advancing a disease-to-gene-to-drug paradigm.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlNk7Nj8xTxrVg90H21EOLACvtfcHk0lh4NOUvqIfUtA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xns1ejurw%253D&md5=9254bf5bef8ae21aa59ae0bca5f47f37</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1523975113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1523975113%26sid%3Dliteratum%253Aachs%26aulast%3DVelagapudi%26aufirst%3DS.%2BP.%26aulast%3DCameron%26aufirst%3DM.%2BD.%26aulast%3DHaga%26aufirst%3DC.%2BL.%26aulast%3DRosenberg%26aufirst%3DL.%2BH.%26aulast%3DLafitte%26aufirst%3DM.%26aulast%3DDuckett%26aufirst%3DD.%2BR.%26aulast%3DPhinney%26aufirst%3DD.%2BG.%26aulast%3DDisney%26aufirst%3DM.%2BD.%26atitle%3DDesign%2520of%2520a%2520small%2520molecule%2520against%2520an%2520oncogenic%2520noncoding%2520RNA%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2016%26volume%3D113%26spage%3D5898%26epage%3D5903%26doi%3D10.1073%2Fpnas.1523975113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref64"><div class="reference"><strong class="refLabel"><a href="#ref64" class="rightTabRefNumLink">64</a></strong><div class="NLM_citation" id="rightTab-cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chan, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loscalzo, J.</span></span> <span> </span><span class="NLM_article-title">MicroRNA-210: A unique and pleiotropic hypoxamir</span>. <i>Cell Cycle</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1072</span>â <span class="NLM_lpage">1083</span>, <span class="refDoi">Â DOI: 10.4161/cc.9.6.11006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.4161%2Fcc.9.6.11006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=20237418" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht12itr%252FM" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2010&pages=1072-1083&author=S.+Y.+Chanauthor=J.+Loscalzo&title=MicroRNA-210%3A+A+unique+and+pleiotropic+hypoxamir&doi=10.4161%2Fcc.9.6.11006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">MicroRNA-210: a unique and pleiotropic hypoxamir</span></div><div class="casAuthors">Chan, Stephen Y.; Loscalzo, Joseph</div><div class="citationInfo"><span class="NLM_cas:title">Cell Cycle</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1072-1083</span>CODEN:
                <span class="NLM_cas:coden">CCEYAS</span>;
        ISSN:<span class="NLM_cas:issn">1538-4101</span>.
    
            (<span class="NLM_cas:orgname">Landes Bioscience</span>)
        </div><div class="casAbstract">Inadequate oxygen availability or hypoxia induces a complex and still incompletely understood set of adaptations that influence cellular survival and function.  Many of these adaptations are directly controlled by a master transcription factor, hypoxia inducible factor-alpha (HIF-Î±).  In response to hypoxia, HIF-Î± levels increase and directly induce the transcription of >100 genes, influencing functions ranging from metab., survival, proliferation, migration, to angiogenesis, among others.  Recently, it has been demonstrated that a specific set of microRNA mols. are upregulated by hypoxia, which we denote here as "hypoxamirs.".  In particular, the HIF-responsive hypoxamir microRNA-210 (miR-210) is a unique microRNA that is evolutionarily conserved and ubiquitously expressed in hypoxic cell and tissue types.  A no. of direct targets of miR-210 have been identified by in silico, transcriptional and biochem. methods, a subset of which have been extensively validated.  As a result, miR-210 has been mechanistically linked to the control of a wide range of cellular responses known to influence normal developmental physiol. as well as a no. of hypoxia-dependent disease states, including tissue ischemia, inflammation and tumorigenesis.  Thus, reflecting the pleiotropic actions of HIF-Î±, miR-210 appears to function as a "master microRNA" relevant for the control of diverse functions in the hypoxic state.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrW8sJalImMsrVg90H21EOLACvtfcHk0lh4NOUvqIfUtA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht12itr%252FM&md5=633ef0fafa94dcd816d882dc212dbed8</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.4161%2Fcc.9.6.11006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252Fcc.9.6.11006%26sid%3Dliteratum%253Aachs%26aulast%3DChan%26aufirst%3DS.%2BY.%26aulast%3DLoscalzo%26aufirst%3DJ.%26atitle%3DMicroRNA-210%253A%2520A%2520unique%2520and%2520pleiotropic%2520hypoxamir%26jtitle%3DCell%2520Cycle%26date%3D2010%26volume%3D9%26spage%3D1072%26epage%3D1083%26doi%3D10.4161%2Fcc.9.6.11006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref65"><div class="reference"><strong class="refLabel"><a href="#ref65" class="rightTabRefNumLink">65</a></strong><div class="NLM_citation" id="rightTab-cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Costales, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suresh, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vishnu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span> <span> </span><span class="NLM_article-title">Targeted degradation of a hypoxia-associated non-coding RNA enhances the selectivity of a small molecule interacting with RNA</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">1180</span>â <span class="NLM_lpage">1186</span>, <span class="refDoi">Â DOI: 10.1016/j.chembiol.2019.04.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1016%2Fj.chembiol.2019.04.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=31130520" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtVegt7bN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2019&pages=1180-1186&author=M.+G.+Costalesauthor=B.+Sureshauthor=K.+Vishnuauthor=M.+D.+Disney&title=Targeted+degradation+of+a+hypoxia-associated+non-coding+RNA+enhances+the+selectivity+of+a+small+molecule+interacting+with+RNA&doi=10.1016%2Fj.chembiol.2019.04.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted Degradation of a Hypoxia-Associated Non-coding RNA Enhances the Selectivity of a Small Molecule Interacting with RNA</span></div><div class="casAuthors">Costales, Matthew G.; Suresh, Blessy; Vishnu, Kamalakannan; Disney, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1180-1186.e5</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Small-mol. targeted recruitment of nucleases to RNA is a powerful method to affect RNA biol.  Inforna, a sequence-based design approach to target RNA, enables the design of small mols. that bind to and cleave RNA in a selective and substoichiometric manner.  Here, we investigate the ability of RNA-targeted degrdn. to improve the selectivity of small mols. targeting RNA.  The microRNA-210 hairpin precursor (pre-miR-210) is overexpressed in hypoxic cancers.  Previously, a small mol. (Targapremir-210 [TGP-210]) targeted this RNA in cells, but with a 5-fold window for DNA binding.  Appendage of a nuclease recruitment module onto TGP-210 locally recruited RNase L onto pre-miR-210, triggering its degrdn.  The chimera has enhanced selectivity compared with TGP-210 with nanomolar binding to the pre-miR-210, but no DNA binding, and is broadly selective for affecting RNA function in cells.  Importantly, it cleaved pre-miR-210 substoichiometrically and induced apoptosis in breast cancer cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5E5B-t5kH2rVg90H21EOLACvtfcHk0lh4NOUvqIfUtA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtVegt7bN&md5=bd7cfbb758ea4c56c50424d4d5720195</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2019.04.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2019.04.008%26sid%3Dliteratum%253Aachs%26aulast%3DCostales%26aufirst%3DM.%2BG.%26aulast%3DSuresh%26aufirst%3DB.%26aulast%3DVishnu%26aufirst%3DK.%26aulast%3DDisney%26aufirst%3DM.%2BD.%26atitle%3DTargeted%2520degradation%2520of%2520a%2520hypoxia-associated%2520non-coding%2520RNA%2520enhances%2520the%2520selectivity%2520of%2520a%2520small%2520molecule%2520interacting%2520with%2520RNA%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2019%26volume%3D26%26spage%3D1180%26epage%3D1186%26doi%3D10.1016%2Fj.chembiol.2019.04.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref66"><div class="reference"><strong class="refLabel"><a href="#ref66" class="rightTabRefNumLink">66</a></strong><div class="NLM_citation" id="rightTab-cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Iqbal, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iqbal, N.</span></span> <span> </span><span class="NLM_article-title">Human epidermal growth factor receptor 2 (HER2) in cancers: Overexpression and therapeutic implications</span>. <i>Mol. Biol. Int.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>2014</i></span>,  <span class="NLM_fpage">852748</span>, <span class="refDoi">Â DOI: 10.1155/2014/852748</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1155%2F2014%2F852748" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=25276427" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A280%3ADC%252BC2M7mvFKnsw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2014&publication_year=2014&pages=852748&author=N.+Iqbalauthor=N.+Iqbal&title=Human+epidermal+growth+factor+receptor+2+%28HER2%29+in+cancers%3A+Overexpression+and+therapeutic+implications&doi=10.1155%2F2014%2F852748"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">Human Epidermal Growth Factor Receptor 2 (HER2) in Cancers: Overexpression and Therapeutic Implications</span></div><div class="casAuthors">Iqbal Nida; Iqbal Naveed</div><div class="citationInfo"><span class="NLM_cas:title">Molecular biology international</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">2014</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">852748</span>
        ISSN:<span class="NLM_cas:issn">2090-2182</span>.
    </div><div class="casAbstract">Human epidermal growth factor receptor 2 (HER2) is a member of the epidermal growth factor receptor family having tyrosine kinase activity.  Dimerization of the receptor results in the autophosphorylation of tyrosine residues within the cytoplasmic domain of the receptors and initiates a variety of signaling pathways leading to cell proliferation and tumorigenesis.  Amplification or overexpression of HER2 occurs in approximately 15-30% of breast cancers and 10-30% of gastric/gastroesophageal cancers and serves as a prognostic and predictive biomarker.  HER2 overexpression has also been seen in other cancers like ovary, endometrium, bladder, lung, colon, and head and neck.  The introduction of HER2 directed therapies has dramatically influenced the outcome of patients with HER2 positive breast and gastric/gastroesophageal cancers; however, the results have been proved disappointing in other HER2 overexpressing cancers.  This review discusses the role of HER2 in various cancers and therapeutic modalities available targeting HER2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQjSP2nh_04ir0AGQpJZWpJfW6udTcc2eYxLo0KrL7Mhrntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2M7mvFKnsw%253D%253D&md5=357ce8a0bf1856baea75f2de51110204</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1155%2F2014%2F852748&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1155%252F2014%252F852748%26sid%3Dliteratum%253Aachs%26aulast%3DIqbal%26aufirst%3DN.%26aulast%3DIqbal%26aufirst%3DN.%26atitle%3DHuman%2520epidermal%2520growth%2520factor%2520receptor%25202%2520%2528HER2%2529%2520in%2520cancers%253A%2520Overexpression%2520and%2520therapeutic%2520implications%26jtitle%3DMol.%2520Biol.%2520Int.%26date%3D2014%26volume%3D2014%26spage%3D852748%26doi%3D10.1155%2F2014%2F852748" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref67"><div class="reference"><strong class="refLabel"><a href="#ref67" class="rightTabRefNumLink">67</a></strong><div class="NLM_citation" id="rightTab-cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pinho, F. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frampton, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nunes, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krell, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alshaker, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacob, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pellegrino, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roca-Alonso, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Giorgio, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harding, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waxman, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stebbing, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pchejetski, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castellano, L.</span></span> <span> </span><span class="NLM_article-title">Downregulation of microRNA-515â5p by the Estrogen Receptor Modulates Sphingosine Kinase 1 and Breast Cancer Cell Proliferation</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>73</i></span>,  <span class="NLM_fpage">5936</span>â <span class="NLM_lpage">5948</span>, <span class="refDoi">Â DOI: 10.1158/0008-5472.CAN-13-0158</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1158%2F0008-5472.CAN-13-0158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=23928990" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsFGrsrvF" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2013&pages=5936-5948&author=F.+G.+Pinhoauthor=A.+E.+Framptonauthor=J.+Nunesauthor=J.+Krellauthor=H.+Alshakerauthor=J.+Jacobauthor=L.+Pellegrinoauthor=L.+Roca-Alonsoauthor=A.+de+Giorgioauthor=V.+Hardingauthor=J.+Waxmanauthor=J.+Stebbingauthor=D.+Pchejetskiauthor=L.+Castellano&title=Downregulation+of+microRNA-515%E2%80%935p+by+the+Estrogen+Receptor+Modulates+Sphingosine+Kinase+1+and+Breast+Cancer+Cell+Proliferation&doi=10.1158%2F0008-5472.CAN-13-0158"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">Downregulation of microRNA-515-5p by the Estrogen Receptor Modulates Sphingosine Kinase 1 and Breast Cancer Cell Proliferation</span></div><div class="casAuthors">Pinho, Filipa G.; Frampton, Adam E.; Nunes, Joao; Krell, Jonathan; Alshaker, Heba; Jacob, Jimmy; Pellegrino, Loredana; Roca-Alonso, Laura; de Giorgio, Alexander; Harding, Victoria; Waxman, Jonathan; Stebbing, Justin; Pchejetski, Dmitry; Castellano, Leandro</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">73</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">5936-5948</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Sphingosine kinase 1 (SKI) plays an important role in estrogen-dependent breast tumorigenesis, but its regulation is poorly understood.  A subset of microRNAs (miRNA, miR) is regulated by estrogen and contributes to cellular proliferation and cancer progression.  Here, we describe that miR-515-5p is transcriptionally repressed by estrogen receptor Î± (ERÎ±) and functions as a tumor suppressor in breast cancer.  Its downregulation enhances cell proliferation and estrogen-dependent SK1 activity, mediated by a redn. of miR-515-5p posttranscriptional repression.  Enforced expression of miR-515-5p in breast cancer cells causes a redn. in SK1 activity, reduced cell proliferation, and the induction of caspase-dependent apoptosis.  Conversely, opposing effects occur with miR-515-5p inhibition and by SK1 silencing.  Notably, we show that estradiol (E2) treatment downregulates miR-515-5p levels, whereas the antiestrogen tamoxifen causes a decrease in SK1, which is rescued by silencing miR-515-5p.  Anal. of chromatin immunopptn. sequencing (ChIP-Seq) data reveals that miR-515-5p suppression is mediated by a direct interaction of ERÎ± within its promoter.  RNA-sequencing (RNA-Seq) anal. of breast cancer cells after overexpressing miR-515-5p indicates that it partly modulates cell proliferation by regulating the Wnt pathway.  The clin. implications of this novel regulatory system are shown as miR-515-5p is significantly downregulated in ER-pos. (n = 146) compared with ER-neg. (n = 98) breast cancers.  Overall, we identify a new link between ERÎ±, miR-515-5p, proliferation, and apoptosis in breast cancer tumorigenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6tmWk7fwlsLVg90H21EOLACvtfcHk0lhI0AVlevmN1A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsFGrsrvF&md5=ff477e1ad93278ed39733f48b84af59d</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-13-0158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-13-0158%26sid%3Dliteratum%253Aachs%26aulast%3DPinho%26aufirst%3DF.%2BG.%26aulast%3DFrampton%26aufirst%3DA.%2BE.%26aulast%3DNunes%26aufirst%3DJ.%26aulast%3DKrell%26aufirst%3DJ.%26aulast%3DAlshaker%26aufirst%3DH.%26aulast%3DJacob%26aufirst%3DJ.%26aulast%3DPellegrino%26aufirst%3DL.%26aulast%3DRoca-Alonso%26aufirst%3DL.%26aulast%3Dde%2BGiorgio%26aufirst%3DA.%26aulast%3DHarding%26aufirst%3DV.%26aulast%3DWaxman%26aufirst%3DJ.%26aulast%3DStebbing%26aufirst%3DJ.%26aulast%3DPchejetski%26aufirst%3DD.%26aulast%3DCastellano%26aufirst%3DL.%26atitle%3DDownregulation%2520of%2520microRNA-515%25E2%2580%25935p%2520by%2520the%2520Estrogen%2520Receptor%2520Modulates%2520Sphingosine%2520Kinase%25201%2520and%2520Breast%2520Cancer%2520Cell%2520Proliferation%26jtitle%3DCancer%2520Res.%26date%3D2013%26volume%3D73%26spage%3D5936%26epage%3D5948%26doi%3D10.1158%2F0008-5472.CAN-13-0158" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref68"><div class="reference"><strong class="refLabel"><a href="#ref68" class="rightTabRefNumLink">68</a></strong><div class="NLM_citation" id="rightTab-cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Costales, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoch, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abegg, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Childs-Disney, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Velagapudi, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adibekian, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span> <span> </span><span class="NLM_article-title">A designed small molecule inhibitor of a non-coding RNA sensitizes HER2 negative cancers to herceptin</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>141</i></span>,  <span class="NLM_fpage">2960</span>â <span class="NLM_lpage">2974</span>, <span class="refDoi">Â DOI: 10.1021/jacs.8b10558</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.8b10558" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC1MXisVGmtrY%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=141&publication_year=2019&pages=2960-2974&author=M.+G.+Costalesauthor=D.+G.+Hochauthor=D.+Abeggauthor=J.+L.+Childs-Disneyauthor=S.+P.+Velagapudiauthor=A.+Adibekianauthor=M.+D.+Disney&title=A+designed+small+molecule+inhibitor+of+a+non-coding+RNA+sensitizes+HER2+negative+cancers+to+herceptin&doi=10.1021%2Fjacs.8b10558"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">A designed small molecule inhibitor of a non-coding RNA sensitizes HER2 negative cancers to herceptin</span></div><div class="casAuthors">Costales, Matthew G.; Hoch, Dominic G.; Abegg, Daniel; Childs-Disney, Jessica L.; Velagapudi, Sai Pradeep; Adibekian, Alexander; Disney, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">141</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2960-2974</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A small mol. (1) with overlapping affinity for two microRNA (miRNA) precursors was used to inform design of a dimeric compd. (2 (I)) selective for one of the miRNAs.  In particular, I selectively targets the microRNA(miR)-515 hairpin precursor to inhibit prodn. of miR-515 that represses sphingosine kinase 1 (SK1), a key enzyme in the biosynthesis of sphingosine 1-phosphate (S1P).  Application of I to breast cancer cells enhanced SK1 and S1P levels, triggering a migratory phenotype.  Knockout of SK1, forced overexpression of miR-515, and application of a small mol. SK1 inhibitor all ablated I's effect on phenotype, consistent with its designed mode of action.  Target profiling studies via Chem-CLIP showed that I bound selectively to the miR-515 hairpin precursor in cells.  Global neoprotein synthesis upon addn. of I to MCF-7 breast cancer cells demonstrated I's selectivity and upregulation of cancer-assocd. proteins regulated by S1P.  The most upregulated protein was human epidermal growth factor receptor 2 (ERBB2/HER2), which is regulated by the SK1/S1P pathway and is normally not expressed in MCF-7 cells.  Like triple neg. breast cancer (TNBC) cells, the lack of HER2 renders them insusceptible to Herceptin and its antibody-drug conjugate Kadcyla.  In addn. to proteomics, an RNA-seq study supports that I has limited off target effects and other studies support that I is more selective than an oligonucleotide.  We therefore hypothesized that I could sensitize MCF-7 cells to anti-HER2 therapies.  Indeed, application of I sensitized cells to Herceptin.  These results were confirmed in two other cell lines that express miR-515 and are HER2-, the hepatocellular carcinoma cell line HepG2 and the TNBC line MDA-MB-231.  Importantly, normal breast epithelial cells (MCF-10A) that do not express miR-515 are not affected by I.  These observations suggest a precision medicine approach to sensitize HER2- cancers to approved anticancer medicines.  This study has implications for broadening the therapeutic utility of known targeted cancer therapeutics by using a secondary targeted approach to render otherwise insensitive cells, sensitive to a targeted therapeutic.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoaWGyvVjeHDrVg90H21EOLACvtfcHk0lhI0AVlevmN1A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXisVGmtrY%253D&md5=1e2f57da2a12fe57d6f60fa44e3e79e2</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1021%2Fjacs.8b10558&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.8b10558%26sid%3Dliteratum%253Aachs%26aulast%3DCostales%26aufirst%3DM.%2BG.%26aulast%3DHoch%26aufirst%3DD.%2BG.%26aulast%3DAbegg%26aufirst%3DD.%26aulast%3DChilds-Disney%26aufirst%3DJ.%2BL.%26aulast%3DVelagapudi%26aufirst%3DS.%2BP.%26aulast%3DAdibekian%26aufirst%3DA.%26aulast%3DDisney%26aufirst%3DM.%2BD.%26atitle%3DA%2520designed%2520small%2520molecule%2520inhibitor%2520of%2520a%2520non-coding%2520RNA%2520sensitizes%2520HER2%2520negative%2520cancers%2520to%2520herceptin%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2019%26volume%3D141%26spage%3D2960%26epage%3D2974%26doi%3D10.1021%2Fjacs.8b10558" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref69"><div class="reference"><strong class="refLabel"><a href="#ref69" class="rightTabRefNumLink">69</a></strong><div class="NLM_citation" id="rightTab-cit69"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Agrawal, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dasaradhi, P. V. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohmmed, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malhotra, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhatnagar, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mukherjee, S. K.</span></span> <span> </span><span class="NLM_article-title">RNA Interference: Biology, mechanism, and applications</span>. <i>Microbiol. Mol. Biol. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">657</span>â <span class="NLM_lpage">685</span>, <span class="refDoi">Â DOI: 10.1128/MMBR.67.4.657-685.2003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1128%2FMMBR.67.4.657-685.2003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=14665679" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BD2cXmt1Sqsw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2003&pages=657-685&author=N.+Agrawalauthor=P.+V.+N.+Dasaradhiauthor=A.+Mohmmedauthor=P.+Malhotraauthor=R.+K.+Bhatnagarauthor=S.+K.+Mukherjee&title=RNA+Interference%3A+Biology%2C+mechanism%2C+and+applications&doi=10.1128%2FMMBR.67.4.657-685.2003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">RNA interference: biology, mechanism, and applications</span></div><div class="casAuthors">Agrawal, Neema; Dasaradhi, P. V. N.; Mohmmed, Asif; Malhotra, Pawan; Bhatnagar, Raj K.; Mukherjee, Sunil K.</div><div class="citationInfo"><span class="NLM_cas:title">Microbiology and Molecular Biology Reviews</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">657-685</span>CODEN:
                <span class="NLM_cas:coden">MMBRF7</span>;
        ISSN:<span class="NLM_cas:issn">1092-2172</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">A review.  Double-stranded RNA-mediated interference (RNAi) is a simple and rapid method of silencing gene expression in a range of organisms.  The silencing of a gene is a consequence of degrdn. of RNA into short RNAs that activate RNases to target homologous mRNA.  The resulting phenotypes either are identical to those of genetic null mutants or resemble an allelic series of mutants.  Specific gene silencing has been shown to be related to two ancient processes, cosuppression in plants and quelling in fungi, and has also been assocd. with regulatory processes such as transposon silencing, antiviral defense mechanisms, gene regulation, and chromosomal modification.  Extensive genetic and biochem. anal. revealed a two-step mechanism of RNAi-induced gene silencing.  The first step involves degrdn. of dsRNA into small interfering RNAs (siRNAs), 21 to 25 nucleotides long, by an RNase III-like activity.  In the second step, the siRNAs join an RNase complex, RISC (RNA-induced silencing complex), which acts on the cognate mRNA and degrades it.  Several key components such as Dicer, RNA-dependent RNA polymerase, helicases, and dSRNA endonucleases have been identified in different organisms for their roles in RNAi.  Some of these components also control the development of many organisms by processing many noncoding RNAs, called micro-RNAs.  The biogenesis and function of micro-RNAs resemble RNAi activities to a large extent.  Recent studies indicate that in the context of RNAi, the genome also undergoes alterations in the form of DNA methylation, heterochromatin formation, and programmed DNA elimination.  As a result of these changes, the silencing effect of gene functions is exercised as tightly as possible.  Because of its exquisite specificity and efficiency, RNAi is being considered as an important tool not only for functional genomics, but also for gene-specific therapeutic activities that target the mRNAs of disease-related genes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_kUuvijBt6rVg90H21EOLACvtfcHk0lghLY-O8VNr7w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXmt1Sqsw%253D%253D&md5=94f3955a1f0ecbd9712424d2db968eb9</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1128%2FMMBR.67.4.657-685.2003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMMBR.67.4.657-685.2003%26sid%3Dliteratum%253Aachs%26aulast%3DAgrawal%26aufirst%3DN.%26aulast%3DDasaradhi%26aufirst%3DP.%2BV.%2BN.%26aulast%3DMohmmed%26aufirst%3DA.%26aulast%3DMalhotra%26aufirst%3DP.%26aulast%3DBhatnagar%26aufirst%3DR.%2BK.%26aulast%3DMukherjee%26aufirst%3DS.%2BK.%26atitle%3DRNA%2520Interference%253A%2520Biology%252C%2520mechanism%252C%2520and%2520applications%26jtitle%3DMicrobiol.%2520Mol.%2520Biol.%2520Rev.%26date%3D2003%26volume%3D67%26spage%3D657%26epage%3D685%26doi%3D10.1128%2FMMBR.67.4.657-685.2003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref70"><div class="reference"><strong class="refLabel"><a href="#ref70" class="rightTabRefNumLink">70</a></strong><div class="NLM_citation" id="rightTab-cit70"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Felicetti, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cecchetti, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manfroni, G.</span></span> <span> </span><span class="NLM_article-title">Modulating microRNA processing: Enoxacin, the progenitor of a new class of drugs</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">12275</span>â <span class="NLM_lpage">12289</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.0c00510</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00510" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtl2rsrnL" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=12275-12289&author=T.+Felicettiauthor=V.+Cecchettiauthor=G.+Manfroni&title=Modulating+microRNA+processing%3A+Enoxacin%2C+the+progenitor+of+a+new+class+of+drugs&doi=10.1021%2Facs.jmedchem.0c00510"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">Modulating microRNA Processing: Enoxacin, the Progenitor of a New Class of Drugs</span></div><div class="casAuthors">Felicetti, Tommaso; Cecchetti, Violetta; Manfroni, Giuseppe</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">12275-12289</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The RNA interference (RNAi) process encompasses the cellular mechanisms by which short-noncoding RNAs posttranscriptionally modulate gene expression.  First discovered in 1998, today RNAi represents the foundation underlying complex biol. mechanisms that are dysregulated in many diseases.  MicroRNAs are effector mols. of gene silencing in RNAi, and their modulation can lead to a wide response in cells.  Enoxacin was reported as the first and unique small-mol. enhancer of microRNA (SMER) maturation.  Herein, the biol. activity of enoxacin as SMER is discussed to shed light on its innovative mode of action, its potential in treating different diseases, and the feasibility of using enoxacin as a chem. template for inspiring medicinal chemists.  We debate its mechanism of action at the mol. level and the possible impact on future ligand and/or structure-guided chem. optimizations, as well as opportunities and drawbacks assocd. with the development of quinolones such as SMERs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrCQcmft_rGlLVg90H21EOLACvtfcHk0lghLY-O8VNr7w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtl2rsrnL&md5=81a379dd43abf95a4b23366c006db178</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00510&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00510%26sid%3Dliteratum%253Aachs%26aulast%3DFelicetti%26aufirst%3DT.%26aulast%3DCecchetti%26aufirst%3DV.%26aulast%3DManfroni%26aufirst%3DG.%26atitle%3DModulating%2520microRNA%2520processing%253A%2520Enoxacin%252C%2520the%2520progenitor%2520of%2520a%2520new%2520class%2520of%2520drugs%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D12275%26epage%3D12289%26doi%3D10.1021%2Facs.jmedchem.0c00510" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref71"><div class="reference"><strong class="refLabel"><a href="#ref71" class="rightTabRefNumLink">71</a></strong><div class="NLM_citation" id="rightTab-cit71"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shan, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szulwach, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faghihi, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khalil, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paroo, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, A. W. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wahlestedt, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, P. A.</span></span> <span> </span><span class="NLM_article-title">Small molecule enhances RNA interference and promotes microRNA processing</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">933</span>â <span class="NLM_lpage">940</span>, <span class="refDoi">Â DOI: 10.1038/nbt.1481</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1038%2Fnbt.1481" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=18641635" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BD1cXps1Wls7c%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2008&pages=933-940&author=G.+Shanauthor=Y.+Liauthor=J.+Zhangauthor=W.+Liauthor=K.+E.+Szulwachauthor=R.+Duanauthor=M.+A.+Faghihiauthor=A.+M.+Khalilauthor=L.+Luauthor=Z.+Parooauthor=A.+W.+S.+Chanauthor=Z.+Shiauthor=Q.+Liuauthor=C.+Wahlestedtauthor=C.+Heauthor=P.+A.+Jin&title=Small+molecule+enhances+RNA+interference+and+promotes+microRNA+processing&doi=10.1038%2Fnbt.1481"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">A small molecule enhances RNA interference and promotes microRNA processing</span></div><div class="casAuthors">Shan, Ge; Li, Yujing; Zhang, Junliang; Li, Wendi; Szulwach, Keith E.; Duan, Ranhui; Faghihi, Mohammad A.; Khalil, Ahmad M.; Lu, Lianghua; Paroo, Zain; Chan, Anthony W. S.; Shi, Zhangjie; Liu, Qinghua; Wahlestedt, Claes; He, Chuan; Jin, Peng</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">933-940</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Little is known about the regulation of RNA interference (RNAi).  Shan et al. constructed a reporter system to monitor RNAi activity and identified a small mol. that increases RNAi by facilitating the interaction between small RNAs and a protein involved in small RNA loading and processing.  Little is known about the regulation of RNA interference (RNAi).  Shan et al. constructed a reporter system to monitor RNAi activity and identified a small mol. that increases RNAi by facilitating the interaction between small RNAs and a protein involved in small RNA loading and processing.  Small interfering RNAs (siRNAs) and microRNAs (miRNAs) are sequence-specific post-transcriptional regulators of gene expression.  Although major components of the RNA interference (RNAi) pathway have been identified, regulatory mechanisms for this pathway remain largely unknown.  Here we demonstrate that the RNAi pathway can be modulated intracellularly by small mols.  We have developed a cell-based assay to monitor the activity of the RNAi pathway and find that the small-mol. enoxacin (Penetrex) enhances siRNA-mediated mRNA degrdn. and promotes the biogenesis of endogenous miRNAs.  We show that this RNAi-enhancing activity depends on the trans-activation-responsive region RNA-binding protein.  Our results provide a proof-of-principle demonstration that small mols. can be used to modulate the activity of the RNAi pathway.  RNAi enhancers may be useful in the development of research tools and therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmMvLQLjUDALVg90H21EOLACvtfcHk0lghLY-O8VNr7w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXps1Wls7c%253D&md5=15210fd18afcb865b6d2e218449c42b4</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1038%2Fnbt.1481&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt.1481%26sid%3Dliteratum%253Aachs%26aulast%3DShan%26aufirst%3DG.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DSzulwach%26aufirst%3DK.%2BE.%26aulast%3DDuan%26aufirst%3DR.%26aulast%3DFaghihi%26aufirst%3DM.%2BA.%26aulast%3DKhalil%26aufirst%3DA.%2BM.%26aulast%3DLu%26aufirst%3DL.%26aulast%3DParoo%26aufirst%3DZ.%26aulast%3DChan%26aufirst%3DA.%2BW.%2BS.%26aulast%3DShi%26aufirst%3DZ.%26aulast%3DLiu%26aufirst%3DQ.%26aulast%3DWahlestedt%26aufirst%3DC.%26aulast%3DHe%26aufirst%3DC.%26aulast%3DJin%26aufirst%3DP.%2BA.%26atitle%3DSmall%2520molecule%2520enhances%2520RNA%2520interference%2520and%2520promotes%2520microRNA%2520processing%26jtitle%3DNat.%2520Biotechnol.%26date%3D2008%26volume%3D26%26spage%3D933%26epage%3D940%26doi%3D10.1038%2Fnbt.1481" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref72"><div class="reference"><strong class="refLabel"><a href="#ref72" class="rightTabRefNumLink">72</a></strong><div class="NLM_citation" id="rightTab-cit72"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kung, J. T. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colognori, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. T.</span></span> <span> </span><span class="NLM_article-title">Long noncoding RNAs: past, present, and future</span>. <i>Genetics</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>193</i></span>,  <span class="NLM_fpage">651</span>â <span class="NLM_lpage">669</span>, <span class="refDoi">Â DOI: 10.1534/genetics.112.146704</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1534%2Fgenetics.112.146704" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=23463798" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtVWnu7zF" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=193&publication_year=2013&pages=651-669&author=J.+T.+Y.+Kungauthor=D.+Colognoriauthor=J.+T.+Lee&title=Long+noncoding+RNAs%3A+past%2C+present%2C+and+future&doi=10.1534%2Fgenetics.112.146704"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">Long noncoding RNAs: past, present, and future</span></div><div class="casAuthors">Kung, Johnny T. Y.; Colognori, David; Lee, Jeannie T.</div><div class="citationInfo"><span class="NLM_cas:title">Genetics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">193</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">651-669</span>CODEN:
                <span class="NLM_cas:coden">GENTAE</span>;
        ISSN:<span class="NLM_cas:issn">0016-6731</span>.
    
            (<span class="NLM_cas:orgname">Genetics Society of America</span>)
        </div><div class="casAbstract">A review.  Long noncoding RNAs (lncRNAs) have gained widespread attention in recent years as a potentially new and crucial layer of biol. regulation.  LncRNAs of all kinds have been implicated in a range of developmental processes and diseases, but knowledge of the mechanisms by which they act is still surprisingly limited, and claims that almost the entirety of the mammalian genome is transcribed into functional noncoding transcripts remain controversial.  At the same time, a small no. of well-studied lncRNAs have given us important clues about the biol. of these mols., and a few key functional and mechanistic themes have begun to emerge, although the robustness of these models and classification schemes remains to be seen.  Here, we review the current state of knowledge of the lncRNA field, discussing what is known about the genomic contexts, biol. functions, and mechanisms of action of lncRNAs.  We also reflect on how the recent interest in lncRNAs is deeply rooted in biol.'s longstanding concern with the evolution and function of genomes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqq-FrGArfXBbVg90H21EOLACvtfcHk0lhj2rsxnHAZPQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtVWnu7zF&md5=7edc7db9a40409f02a59039068b3771d</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1534%2Fgenetics.112.146704&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1534%252Fgenetics.112.146704%26sid%3Dliteratum%253Aachs%26aulast%3DKung%26aufirst%3DJ.%2BT.%2BY.%26aulast%3DColognori%26aufirst%3DD.%26aulast%3DLee%26aufirst%3DJ.%2BT.%26atitle%3DLong%2520noncoding%2520RNAs%253A%2520past%252C%2520present%252C%2520and%2520future%26jtitle%3DGenetics%26date%3D2013%26volume%3D193%26spage%3D651%26epage%3D669%26doi%3D10.1534%2Fgenetics.112.146704" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref73"><div class="reference"><strong class="refLabel"><a href="#ref73" class="rightTabRefNumLink">73</a></strong><div class="NLM_citation" id="rightTab-cit73"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span> <span> </span><span class="NLM_article-title">Associating lncRNAs with small molecules <i>via</i> bilevel optimization reveals cancer-related lncRNAs</span>. <i>PLoS Comput. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>15</i></span>, <span class="NLM_elocation-id">e1007540</span> <span class="refDoi">Â DOI: 10.1371/journal.pcbi.1007540</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1371%2Fjournal.pcbi.1007540" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=31877126" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BB3cXktlGmtb4%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2019&author=Y.+Wangauthor=S.+Chenauthor=L.+Chenauthor=Y.+Wang&title=Associating+lncRNAs+with+small+molecules+via+bilevel+optimization+reveals+cancer-related+lncRNAs&doi=10.1371%2Fjournal.pcbi.1007540"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">Associating lncRNAs with small molecules via bilevel optimization reveals cancer-related lncRNAs</span></div><div class="casAuthors">Wang, Yongcui; Chen, Shilong; Chen, Luonan; Wang, Yong</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Computational Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">e1007540/1-e1007540/20</span>CODEN:
                <span class="NLM_cas:coden">PCBLBG</span>;
        ISSN:<span class="NLM_cas:issn">1553-7358</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Long noncoding RNA (lncRNA) transcripts have emerging impacts in cancer studies, which suggests their potential as novel therapeutic agents.  However, the mol. mechanism behind their treatment effects is still unclear.  Here, we designed a computational model to Assoc. LncRNAs with Anti-Cancer Drugs (ALACD) based on a bilevel optimization model, which optimized the gene signature overlap in the upper level and imputed the missing lncRNA-gene assocn. in the lower level.  ALACD predicts genes coexpressed with lncRNAs mean while matching drug's gene signatures.  This model allows us to borrow the target gene information of small mols. to understand the mechanisms of action of lncRNAs and their roles in cancer.  The ALACD model was systematically applied to the 10 cancer types in The Cancer Genome Atlas (TCGA) that had matched lncRNA and mRNA expression data.  Cancer type-specific lncRNAs and assocd. drugs were identified.  These lncRNAs show significantly different expression levels in cancer patients.  Follow-up functional and mol. pathway anal. suggest the gene signatures bridging drugs and lncRNAs are closely related to cancer development.  Importantly, patient survival information and evidence from the literature suggest that the lncRNAs and drug-lncRNA assocns. identified by the ALACD model can provide an alternative choice for cancer targeting treatment and potential cancer pognostic biomarkers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6sEZB7SUrYrVg90H21EOLACvtfcHk0lhj2rsxnHAZPQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXktlGmtb4%253D&md5=208e9f72a9931bb1adc6884acb10ad10</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pcbi.1007540&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pcbi.1007540%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DY.%26atitle%3DAssociating%2520lncRNAs%2520with%2520small%2520molecules%2520via%2520bilevel%2520optimization%2520reveals%2520cancer-related%2520lncRNAs%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2019%26volume%3D15%26doi%3D10.1371%2Fjournal.pcbi.1007540" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref74"><div class="reference"><strong class="refLabel"><a href="#ref74" class="rightTabRefNumLink">74</a></strong><div class="NLM_citation" id="rightTab-cit74"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ji, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diederichs, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boing, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Metzger, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tidow, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brandt, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buerger, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bulk, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berdel, W. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Serve, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller-Tidow, C.</span></span> <span> </span><span class="NLM_article-title">MALAT-1, a novel noncoding RNA, and thymosin beta4 predict metastasis and survival in early-stage non-small cell lung cancer</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">8031</span>â <span class="NLM_lpage">8041</span>, <span class="refDoi">Â DOI: 10.1038/sj.onc.1206928</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1038%2Fsj.onc.1206928" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=12970751" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A280%3ADC%252BD3svivFOgsg%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2003&pages=8031-8041&author=P.+Jiauthor=S.+Diederichsauthor=W.+Wangauthor=S.+Boingauthor=R.+Metzgerauthor=P.+M.+Schneiderauthor=N.+Tidowauthor=B.+Brandtauthor=H.+Buergerauthor=E.+Bulkauthor=M.+Thomasauthor=W.+E.+Berdelauthor=H.+Serveauthor=C.+Muller-Tidow&title=MALAT-1%2C+a+novel+noncoding+RNA%2C+and+thymosin+beta4+predict+metastasis+and+survival+in+early-stage+non-small+cell+lung+cancer&doi=10.1038%2Fsj.onc.1206928"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit74R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">MALAT-1, a novel noncoding RNA, and thymosin beta4 predict metastasis and survival in early-stage non-small cell lung cancer</span></div><div class="casAuthors">Ji Ping; Diederichs Sven; Wang Wenbing; Boing Sebastian; Metzger Ralf; Schneider Paul M; Tidow Nicola; Brandt Burkhard; Buerger Horst; Bulk Etmar; Thomas Michael; Berdel Wolfgang E; Serve Hubert; Muller-Tidow Carsten</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">39</span>),
    <span class="NLM_cas:pages">8031-41</span>
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    </div><div class="casAbstract">Early-stage non-small cell lung cancer (NSCLC) can be cured by surgical resection, but a substantial fraction of patients ultimately dies due to distant metastasis.  In this study, we used subtractive hybridization to identify gene expression differences in stage I NSCLC tumors that either did or did not metastasize in the course of disease.  Individual clones (n=225) were sequenced and quantitative RT-PCR verified overexpression in metastasizing samples.  Several of the identified genes (eIF4A1, thymosin beta4 and a novel transcript named MALAT-1) were demonstrated to be significantly associated with metastasis in NSCLC patients (n=70).  The genes' association with metastasis was stage- and histology specific.  The Kaplan-Meier analyses identified MALAT-1 and thymosin beta4 as prognostic parameters for patient survival in stage I NSCLC.  The novel MALAT-1 transcript is a noncoding RNA of more than 8000 nt expressed from chromosome 11q13.  It is highly expressed in lung, pancreas and other healthy organs as well as in NSCLC.  MALAT-1 expressed sequences are conserved across several species indicating its potentially important function.  Taken together, these data contribute to the identification of early-stage NSCLC patients that are at high risk to develop metastasis.  The identification of MALAT-1 emphasizes the potential role of noncoding RNAs in human cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcT4lotbLB7vyCA3UY9EwTNrfW6udTcc2eYkmoDZv4Z8Prntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD3svivFOgsg%253D%253D&md5=2fce458717370f8fc36ff1b85c4a5b9c</span></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1038%2Fsj.onc.1206928&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.onc.1206928%26sid%3Dliteratum%253Aachs%26aulast%3DJi%26aufirst%3DP.%26aulast%3DDiederichs%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DBoing%26aufirst%3DS.%26aulast%3DMetzger%26aufirst%3DR.%26aulast%3DSchneider%26aufirst%3DP.%2BM.%26aulast%3DTidow%26aufirst%3DN.%26aulast%3DBrandt%26aufirst%3DB.%26aulast%3DBuerger%26aufirst%3DH.%26aulast%3DBulk%26aufirst%3DE.%26aulast%3DThomas%26aufirst%3DM.%26aulast%3DBerdel%26aufirst%3DW.%2BE.%26aulast%3DServe%26aufirst%3DH.%26aulast%3DMuller-Tidow%26aufirst%3DC.%26atitle%3DMALAT-1%252C%2520a%2520novel%2520noncoding%2520RNA%252C%2520and%2520thymosin%2520beta4%2520predict%2520metastasis%2520and%2520survival%2520in%2520early-stage%2520non-small%2520cell%2520lung%2520cancer%26jtitle%3DOncogene%26date%3D2003%26volume%3D22%26spage%3D8031%26epage%3D8041%26doi%3D10.1038%2Fsj.onc.1206928" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref75"><div class="reference"><strong class="refLabel"><a href="#ref75" class="rightTabRefNumLink">75</a></strong><div class="NLM_citation" id="rightTab-cit75"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ghafouri-Fard, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hafez, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taheri, M.</span></span> <span> </span><span class="NLM_article-title">Metastasis associated lung adenocarcinoma transcript 1: An update on expression pattern and functions in carcinogenesis</span>. <i>Exp. Mol. Pathol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>112</i></span>,  <span class="NLM_fpage">104330</span>, <span class="refDoi">Â DOI: 10.1016/j.yexmp.2019.104330</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1016%2Fj.yexmp.2019.104330" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=31712117" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC1MXit1SmsrfO" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=112&publication_year=2020&pages=104330&author=S.+Ghafouri-Fardauthor=A.+A.+Hafezauthor=M.+Taheri&title=Metastasis+associated+lung+adenocarcinoma+transcript+1%3A+An+update+on+expression+pattern+and+functions+in+carcinogenesis&doi=10.1016%2Fj.yexmp.2019.104330"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit75R"><div class="casContent"><span class="casTitleNuber">75</span><div class="casTitle"><span class="NLM_cas:atitle">Metastasis Associated Lung Adenocarcinoma Transcript 1: An update on expression pattern and functions in carcinogenesis</span></div><div class="casAuthors">Ghafouri-Fard, Soudeh; Ashrafi Hafez, Asghar; Taheri, Mohammad</div><div class="citationInfo"><span class="NLM_cas:title">Experimental and Molecular Pathology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">112</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">104330</span>CODEN:
                <span class="NLM_cas:coden">EXMPA6</span>;
        ISSN:<span class="NLM_cas:issn">0014-4800</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review.  The Metastasis Assocd. Lung Adenocarcinoma Transcript 1 (MALAT1) is among long non-coding RNAs (lncRNAs) which has disapproved the old term of "junk DNA" which was used for majority of human genome which are not transcribed to proteins.  An extensive portion of literature points to the fundamental role of this lncRNA in tumorigenesis process of diverse cancers ranging from solid tumors to leukemia.  Being firstly identified in lung cancer, it has prognostic and diagnostic values in several cancer types.  Consistent with the proposed oncogenic roles for this lncRNA, most of studies have shown up-regulation of MALAT1 in malignant tissues compared with non-malignant/normal tissues of the same source.  However, few studies have shown down-regulation of MALAT1 in breast cancer, endometrial cancer, colorectal cancer and glioma.  In the current study, we have conducted a comprehensive literature search and provided an up-date on the role of MALAT1 in cancer biol.  Our investigation underscores a potential role as a diagnostic/prognostic marker and a putative therapeutic target for MALAT1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPW40Uh_BlXrVg90H21EOLACvtfcHk0liIaBalHwf8XA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXit1SmsrfO&md5=bfa0660f0f74b5f82f177c122ba2e17c</span></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1016%2Fj.yexmp.2019.104330&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.yexmp.2019.104330%26sid%3Dliteratum%253Aachs%26aulast%3DGhafouri-Fard%26aufirst%3DS.%26aulast%3DHafez%26aufirst%3DA.%2BA.%26aulast%3DTaheri%26aufirst%3DM.%26atitle%3DMetastasis%2520associated%2520lung%2520adenocarcinoma%2520transcript%25201%253A%2520An%2520update%2520on%2520expression%2520pattern%2520and%2520functions%2520in%2520carcinogenesis%26jtitle%3DExp.%2520Mol.%2520Pathol.%26date%3D2020%26volume%3D112%26spage%3D104330%26doi%3D10.1016%2Fj.yexmp.2019.104330" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref76"><div class="reference"><strong class="refLabel"><a href="#ref76" class="rightTabRefNumLink">76</a></strong><div class="NLM_citation" id="rightTab-cit76"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Donlic, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morgan, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roble, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hargrove, A. E.</span></span> <span> </span><span class="NLM_article-title">Discovery of small molecule ligands for MALAT1 by tuning an RNA-binding scaffold</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">13242</span>â <span class="NLM_lpage">13247</span>, <span class="refDoi">Â DOI: 10.1002/anie.201808823</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1002%2Fanie.201808823" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhs1Omtb%252FM" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2018&pages=13242-13247&author=A.+Donlicauthor=B.+S.+Morganauthor=J.+L.+Xuauthor=A.+Liuauthor=C.+Robleauthor=A.+E.+Hargrove&title=Discovery+of+small+molecule+ligands+for+MALAT1+by+tuning+an+RNA-binding+scaffold&doi=10.1002%2Fanie.201808823"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit76R"><div class="casContent"><span class="casTitleNuber">76</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Small Molecule Ligands for MALAT1 by Tuning an RNA-Binding Scaffold</span></div><div class="casAuthors">Donlic, Anita; Morgan, Brittany S.; Xu, Jason L.; Liu, Anqi; Roble, Carlos Jr.; Hargrove, Amanda E.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">40</span>),
    <span class="NLM_cas:pages">13242-13247</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Structural studies of the 3'-end of the oncogenic long non-coding RNA metastasis-assocd. lung adenocarcinoma transcript 1 (MALAT1) confirmed a unique triple-helix structure.  This structure enables accumulation of the transcript, and high levels of MALAT1 are found in several cancers.  Here, the authors synthesize a small mol. library based on an RNA-binding scaffold, diphenylfuran (DPF), screen it against a variety of nucleic acid constructs, and demonstrate that the MALAT1 triple helix can be selectively targeted with small mols.  Computational anal. revealed a trend between subunit positioning and compn. on DPF shape and intramol. interactions, which in turn generally correlated with selectivity and binding strengths.  This work thus provides design strategies toward chem. probe development for the MALAT1 triple helix and suggests that comprehensive analyses of RNA-focused libraries can generate insights into selective RNA recognition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqudyg9LTHbgLVg90H21EOLACvtfcHk0liIaBalHwf8XA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhs1Omtb%252FM&md5=181570e797ff7aef7444441387eadf30</span></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.1002%2Fanie.201808823&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201808823%26sid%3Dliteratum%253Aachs%26aulast%3DDonlic%26aufirst%3DA.%26aulast%3DMorgan%26aufirst%3DB.%2BS.%26aulast%3DXu%26aufirst%3DJ.%2BL.%26aulast%3DLiu%26aufirst%3DA.%26aulast%3DRoble%26aufirst%3DC.%26aulast%3DHargrove%26aufirst%3DA.%2BE.%26atitle%3DDiscovery%2520of%2520small%2520molecule%2520ligands%2520for%2520MALAT1%2520by%2520tuning%2520an%2520RNA-binding%2520scaffold%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2018%26volume%3D57%26spage%3D13242%26epage%3D13247%26doi%3D10.1002%2Fanie.201808823" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref77"><div class="reference"><strong class="refLabel"><a href="#ref77" class="rightTabRefNumLink">77</a></strong><div class="NLM_citation" id="rightTab-cit77"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chaires, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamelberg, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pandya, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boykin, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, W. D.</span></span> <span> </span><span class="NLM_article-title">Structural selectivity of aromatic diamidines</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">5729</span>â <span class="NLM_lpage">5742</span>, <span class="refDoi">Â DOI: 10.1021/jm049491e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm049491e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BD2cXotVCisL0%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=5729-5742&author=J.+B.+Chairesauthor=J.+Renauthor=D.+Hamelbergauthor=A.+Kumarauthor=V.+Pandyaauthor=D.+W.+Boykinauthor=W.+D.+Wilson&title=Structural+selectivity+of+aromatic+diamidines&doi=10.1021%2Fjm049491e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit77R"><div class="casContent"><span class="casTitleNuber">77</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Selectivity of Aromatic Diamidines</span></div><div class="casAuthors">Chaires, Jonathan B.; Ren, Jinsong; Hamelberg, Donald; Kumar, Arvind; Pandya, Vandna; Boykin, David W.; Wilson, W. David</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">5729-5742</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Competition dialysis was used to study the interactions of 13 substituted arom. diamidine compds. with 13 nucleic acid structures and sequences.  The results show a striking selectivity of these compds. for the triplex structure poly dA:(poly dT)2, a novel aspect of their interaction with nucleic acids not previously described.  The triplex selectivity of selected compds. was confirmed by thermal denaturation studies.  Triplex selectivity was found to be modulated by the location of amidine substituents on the core phenyl-furan-Ph ring scaffold.  Mol. models were constructed to rationalize the triplex selectivity of DB359, the most selective compd. in the series.  Its triplex selectivity was found to arise from optimal ring stacking on base triplets, along with proper positioning of its amidine substituents to occupy the minor and the major-minor grooves of the triplex.  New insights into the mol. recognition of nucleic acid structures emerged from these studies, adding to the list of available design principles for selectively targeting DNA and RNA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8-gzBJdMjo7Vg90H21EOLACvtfcHk0liIaBalHwf8XA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXotVCisL0%253D&md5=303d146fba1b5e650230142c955ad530</span></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=10.1021%2Fjm049491e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm049491e%26sid%3Dliteratum%253Aachs%26aulast%3DChaires%26aufirst%3DJ.%2BB.%26aulast%3DRen%26aufirst%3DJ.%26aulast%3DHamelberg%26aufirst%3DD.%26aulast%3DKumar%26aufirst%3DA.%26aulast%3DPandya%26aufirst%3DV.%26aulast%3DBoykin%26aufirst%3DD.%2BW.%26aulast%3DWilson%26aufirst%3DW.%2BD.%26atitle%3DStructural%2520selectivity%2520of%2520aromatic%2520diamidines%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D5729%26epage%3D5742%26doi%3D10.1021%2Fjm049491e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref78"><div class="reference"><strong class="refLabel"><a href="#ref78" class="rightTabRefNumLink">78</a></strong><div class="NLM_citation" id="rightTab-cit78"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abulwerdi, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ageeli, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yonkunas, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arun, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nam, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneekloth, J. S.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dayie, T. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spector, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baird, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le Grice, S. F. J.</span></span> <span> </span><span class="NLM_article-title">Selective small-molecule targeting of a triple helix encoded by the long noncoding RNA, MALAT1</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">223</span>â <span class="NLM_lpage">235</span>, <span class="refDoi">Â DOI: 10.1021/acschembio.8b00807</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.8b00807" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC1MXltVWjsw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2019&pages=223-235&author=F.+A.+Abulwerdiauthor=W.+Xuauthor=A.+A.+Ageeliauthor=M.+J.+Yonkunasauthor=G.+Arunauthor=H.+Namauthor=J.+S.+Schneeklothauthor=T.+K.+Dayieauthor=D.+Spectorauthor=N.+Bairdauthor=S.+F.+J.+Le+Grice&title=Selective+small-molecule+targeting+of+a+triple+helix+encoded+by+the+long+noncoding+RNA%2C+MALAT1&doi=10.1021%2Facschembio.8b00807"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit78R"><div class="casContent"><span class="casTitleNuber">78</span><div class="casTitle"><span class="NLM_cas:atitle">Selective small-molecule targeting of a triple helix encoded by the long noncoding RNA, MALAT1</span></div><div class="casAuthors">Abulwerdi, Fardokht A.; Xu, Wenbo; Ageeli, Abeer A.; Yonkunas, Michael J.; Arun, Gayatri; Nam, Hyeyeon; Schneekloth, John S.; Dayie, Theodore Kwaku; Spector, David; Baird, Nathan; Le Grice, Stuart F. J.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">223-235</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Metastasis-assocd. lung adenocarcinoma transcript 1 (Malat1/MALAT1, mouse/human), a highly conserved long noncoding (lnc) RNA, has been linked with several physiol. processes, including the alternative splicing, nuclear organization, and epigenetic modulation of gene expression.  MALAT1 has also been implicated in metastasis and tumor proliferation in multiple cancer types.  The 3' terminal stability element for nuclear expression (ENE) assumes a triple-helical configuration that promotes its nuclear accumulation and persistent function.  Utilizing a novel small mol. microarray strategy, we identified multiple Malat1 ENE triplex-binding chemotypes, among which compds. I and II reduced Malat1 RNA levels and branching morphogenesis in a mammary tumor organoid model.  Computational modeling and F.ovrddot.orster resonance energy transfer expts. demonstrate distinct binding modes for each chemotype, conferring opposing structural changes to the triplex.  I modulates Malat1 downstream genes without affecting Neat1, a nuclear lncRNA encoded in the same chromosomal region as Malat1 with a structurally similar ENE triplex.  Supporting this observation, the specificity of I for Malat1 over Neat1 and a virus-coded ENE was demonstrated by NMR spectroscopy.  Small mols. specifically targeting the MALAT1 ENE triplex lay the foundation for new classes of anticancer therapeutics and mol. probes for the treatment and investigation of MALAT1-driven cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqfz6g3iAm4orVg90H21EOLACvtfcHk0lj_FwMGws3zxw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXltVWjsw%253D%253D&md5=ce56033fbebf0fc32991901faa24766f</span></div><a href="/servlet/linkout?suffix=cit78&amp;dbid=16384&amp;doi=10.1021%2Facschembio.8b00807&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.8b00807%26sid%3Dliteratum%253Aachs%26aulast%3DAbulwerdi%26aufirst%3DF.%2BA.%26aulast%3DXu%26aufirst%3DW.%26aulast%3DAgeeli%26aufirst%3DA.%2BA.%26aulast%3DYonkunas%26aufirst%3DM.%2BJ.%26aulast%3DArun%26aufirst%3DG.%26aulast%3DNam%26aufirst%3DH.%26aulast%3DSchneekloth%26aufirst%3DJ.%2BS.%26aulast%3DDayie%26aufirst%3DT.%2BK.%26aulast%3DSpector%26aufirst%3DD.%26aulast%3DBaird%26aufirst%3DN.%26aulast%3DLe%2BGrice%26aufirst%3DS.%2BF.%2BJ.%26atitle%3DSelective%2520small-molecule%2520targeting%2520of%2520a%2520triple%2520helix%2520encoded%2520by%2520the%2520long%2520noncoding%2520RNA%252C%2520MALAT1%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2019%26volume%3D14%26spage%3D223%26epage%3D235%26doi%3D10.1021%2Facschembio.8b00807" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref79"><div class="reference"><strong class="refLabel"><a href="#ref79" class="rightTabRefNumLink">79</a></strong><div class="NLM_citation" id="rightTab-cit79"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, P.</span></span> <span> </span><span class="NLM_article-title">Role and mechanism of the alkylglycerone phosphate synthase in suppressing the invasion potential of human glioma and hepatic carcinoma cells in vitro</span>. <i>Oncol. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">431</span>â <span class="NLM_lpage">436</span>, <span class="refDoi">Â DOI: 10.3892/or.2014.3189</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.3892%2For.2014.3189" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=24841318" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtlaitL%252FL" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2014&pages=431-436&author=Y.+Zhuauthor=L.+Zhuauthor=L.+Luauthor=L.+Zhangauthor=G.+Zhangauthor=Q.+Wangauthor=P.+Yang&title=Role+and+mechanism+of+the+alkylglycerone+phosphate+synthase+in+suppressing+the+invasion+potential+of+human+glioma+and+hepatic+carcinoma+cells+in+vitro&doi=10.3892%2For.2014.3189"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit79R"><div class="casContent"><span class="casTitleNuber">79</span><div class="casTitle"><span class="NLM_cas:atitle">Role and mechanism of the alkylglycerone phosphate synthase in suppressing the invasion potential of human glioma and hepatic carcinoma cells in vitro</span></div><div class="casAuthors">Zhu, Yu; Zhu, Lielie; Lu, Ling; Zhang, Ling; Zhang, Guodong; Wang, Qin; Yang, Ping</div><div class="citationInfo"><span class="NLM_cas:title">Oncology Reports</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">431-436</span>CODEN:
                <span class="NLM_cas:coden">OCRPEW</span>;
        ISSN:<span class="NLM_cas:issn">1021-335X</span>.
    
            (<span class="NLM_cas:orgname">Spandidos Publications Ltd.</span>)
        </div><div class="casAbstract">Ether lipids have been implicated in the exacerbation of human tumors.  Accumulating evidence suggests that the alkylglycerone phosphate synthase (AGPS) is involved in the suppression of some types of tumor.  However, the role and mol. mechanism of AGPS in the invasion of human glioma and hepatic carcinoma remain unclear.  In the present study, using AGPS-knockdown human glioma U87 and hepatic carcinoma HepG2 cell lines, we explored the role of AGPS, as well as its mol. mechanism, in invasion in vitro.  It was demonstrated that silencing AGPS expression resulted in a decreased expression of cellular lipids such as LPA, LPAe and PGE2, adhesion, invasion potential and arrested cell cycle in tumor cells.  The expression of invasion-related genes such asMMP-2/9,E-cadherin and CD44 showed marked changes in AGPS-knockdown cells.  In addn., we found that AGPS regulated the activity of the MAPK pathway, as well as the transcriptional activity of Twist, AP-1, and Snail.  The results demonstrated that AGPS neg. regulated the invasion potential of glioma and hepatic carcinoma cells by modulating the expression of relevant genes and activity of the MAPK pathway.  Therefore, AGPS may be a potential glioma and hepatic carcinoma therapeutic target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptlN_wvt9D_7Vg90H21EOLACvtfcHk0lj_FwMGws3zxw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtlaitL%252FL&md5=00cadef12c58cf589894805500c09b65</span></div><a href="/servlet/linkout?suffix=cit79&amp;dbid=16384&amp;doi=10.3892%2For.2014.3189&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3892%252For.2014.3189%26sid%3Dliteratum%253Aachs%26aulast%3DZhu%26aufirst%3DY.%26aulast%3DZhu%26aufirst%3DL.%26aulast%3DLu%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DWang%26aufirst%3DQ.%26aulast%3DYang%26aufirst%3DP.%26atitle%3DRole%2520and%2520mechanism%2520of%2520the%2520alkylglycerone%2520phosphate%2520synthase%2520in%2520suppressing%2520the%2520invasion%2520potential%2520of%2520human%2520glioma%2520and%2520hepatic%2520carcinoma%2520cells%2520in%2520vitro%26jtitle%3DOncol.%2520Rep.%26date%3D2014%26volume%3D32%26spage%3D431%26epage%3D436%26doi%3D10.3892%2For.2014.3189" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref80"><div class="reference"><strong class="refLabel"><a href="#ref80" class="rightTabRefNumLink">80</a></strong><div class="NLM_citation" id="rightTab-cit80"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Qian, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, Y.</span></span> <span> </span><span class="NLM_article-title">Computer-aided drug design and inhibitive effect of a novel nitrogenous heterocyclic compound and its mechanism on glioma U251 cells and breast cancer MCF-7 cells</span>. <i>Drug Des., Dev. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">1931</span>â <span class="NLM_lpage">1939</span>, <span class="refDoi">Â DOI: 10.2147/DDDT.S168130</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.2147%2FDDDT.S168130" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=29983547" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsVCrs77M" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2018&pages=1931-1939&author=L.+Qianauthor=Y.+Zhu&title=Computer-aided+drug+design+and+inhibitive+effect+of+a+novel+nitrogenous+heterocyclic+compound+and+its+mechanism+on+glioma+U251+cells+and+breast+cancer+MCF-7+cells&doi=10.2147%2FDDDT.S168130"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit80R"><div class="casContent"><span class="casTitleNuber">80</span><div class="casTitle"><span class="NLM_cas:atitle">Computer-aided drug design and inhibitive effect of a novel nitrogenous heterocyclic compound and its mechanism on glioma U251 cells and breast cancer MCF-7 cells</span></div><div class="casAuthors">Qian, Liyu; Zhu, Yu</div><div class="citationInfo"><span class="NLM_cas:title">Drug Design, Development and Therapy</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1931-1939</span>CODEN:
                <span class="NLM_cas:coden">DDDTAQ</span>;
        ISSN:<span class="NLM_cas:issn">1177-8881</span>.
    
            (<span class="NLM_cas:orgname">Dove Medical Press Ltd.</span>)
        </div><div class="casAbstract">Background: Glioma and breast cancer are severe malignant cancerous tumors that highlight the importance of developing new anti-cancer drugs.  The aim of this study was to explore the effects of a novel nitrogenous heterocyclic compd. on glioma and breast cancer cells and to det. its mechanism of action.  Methods: We designed and synthesized a novel nitrogenous heterocyclic compd., 3-(4-amino-1H-benzo[d]imidazole-2-carboxamido)-4-oxo-3,4-dihydroimidazo[5,1-d][1,2,3,5]tetrazine-8-carboxamide, based on alkylglycerone phosphate synthase (AGPS) using computer-aided drug design (CADD), and we measured its effect on the proliferation, invasion, cell cycle and apoptosis of U251 glioma and MCF-7 breast cancer cells.  In addn., the compd.'s effect on the expression of tumor-related mRNA, circular RNAs (circRNAs) and long non-coding RNAs (lncRNAs) was explored.  Results: It was found that the nitrogenous heterocyclic compd. could induce cell cycle arrest at the G2/M phase of U251/MCF-7 cells and activate apoptosis.  Real-time PCR showed that the expression levels of tumor-related mRNA, circRNAs and lncRNAs were impacted.  Conclusion: We concluded that the nitrogenous heterocyclic compd. inhibits the proliferation and invasion of U251 glioma and MCF-7 breast cancer cells through the induction of apoptosis and cell cycle arrest by regulating tumor-related genes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2LXvViUrFtrVg90H21EOLACvtfcHk0lj_FwMGws3zxw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsVCrs77M&md5=51cf53d94b9b84d08ce9dac3f21df887</span></div><a href="/servlet/linkout?suffix=cit80&amp;dbid=16384&amp;doi=10.2147%2FDDDT.S168130&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2147%252FDDDT.S168130%26sid%3Dliteratum%253Aachs%26aulast%3DQian%26aufirst%3DL.%26aulast%3DZhu%26aufirst%3DY.%26atitle%3DComputer-aided%2520drug%2520design%2520and%2520inhibitive%2520effect%2520of%2520a%2520novel%2520nitrogenous%2520heterocyclic%2520compound%2520and%2520its%2520mechanism%2520on%2520glioma%2520U251%2520cells%2520and%2520breast%2520cancer%2520MCF-7%2520cells%26jtitle%3DDrug%2520Des.%252C%2520Dev.%2520Ther.%26date%3D2018%26volume%3D12%26spage%3D1931%26epage%3D1939%26doi%3D10.2147%2FDDDT.S168130" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref81"><div class="reference"><strong class="refLabel"><a href="#ref81" class="rightTabRefNumLink">81</a></strong><div class="NLM_citation" id="rightTab-cit81"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gupta, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horlings, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsai, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Argani, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rinn, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brzoska, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kong, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">West, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van de
Vijver, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sukumar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, H. Y.</span></span> <span> </span><span class="NLM_article-title">Long non-coding RNA HOTAIR reprograms chromatin state to promote cancer metastasis</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>464</i></span>,  <span class="NLM_fpage">1071</span>â <span class="NLM_lpage">1076</span>, <span class="refDoi">Â DOI: 10.1038/nature08975</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1038%2Fnature08975" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=20393566" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC3cXkslaru7w%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=464&publication_year=2010&pages=1071-1076&author=R.+A.+Guptaauthor=N.+Shahauthor=K.+C.+Wangauthor=J.+Kimauthor=H.+M.+Horlingsauthor=D.+J.+Wongauthor=M.+C.+Tsaiauthor=T.+Hungauthor=P.+Arganiauthor=J.+L.+Rinnauthor=Y.+Wangauthor=P.+Brzoskaauthor=B.+Kongauthor=R.+Liauthor=R.+B.+Westauthor=M.+J.+van+de%0AVijverauthor=S.+Sukumarauthor=H.+Y.+Chang&title=Long+non-coding+RNA+HOTAIR+reprograms+chromatin+state+to+promote+cancer+metastasis&doi=10.1038%2Fnature08975"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit81R"><div class="casContent"><span class="casTitleNuber">81</span><div class="casTitle"><span class="NLM_cas:atitle">Long non-coding RNA HOTAIR reprograms chromatin state to promote cancer metastasis</span></div><div class="casAuthors">Gupta, Rajnish A.; Shah, Nilay; Wang, Kevin C.; Kim, Jeewon; Horlings, Hugo M.; Wong, David J.; Tsai, Miao-Chih; Hung, Tiffany; Argani, Pedram; Rinn, John L.; Wang, Yulei; Brzoska, Pius; Kong, Benjamin; Li, Rui; West, Robert B.; van de Vijver, Marc J.; Sukumar, Saraswati; Chang, Howard Y.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">464</span>
        (<span class="NLM_cas:issue">7291</span>),
    <span class="NLM_cas:pages">1071-1076</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Large intervening non-coding RNAs (lincRNAs) are pervasively transcribed in the genome yet their potential involvement in human disease is not well understood.  Recent studies of dosage compensation, imprinting, and homeotic gene expression suggest that individual lincRNAs can function as the interface between DNA and specific chromatin remodelling activities.  Here we show that lincRNAs in the HOX loci become systematically dysregulated during breast cancer progression.  The lincRNA termed HOTAIR is increased in expression in primary breast tumors and metastases, and HOTAIR expression level in primary tumors is a powerful predictor of eventual metastasis and death.  Enforced expression of HOTAIR in epithelial cancer cells induced genome-wide re-targeting of Polycomb repressive complex 2 (PRC2) to an occupancy pattern more resembling embryonic fibroblasts, leading to altered histone H3 lysine 27 methylation, gene expression, and increased cancer invasiveness and metastasis in a manner dependent on PRC2.  Conversely, loss of HOTAIR can inhibit cancer invasiveness, particularly in cells that possess excessive PRC2 activity.  These findings indicate that lincRNAs have active roles in modulating the cancer epigenome and may be important targets for cancer diagnosis and therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7vmSbfc1Wi7Vg90H21EOLACvtfcHk0lg43QYIftxKVQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXkslaru7w%253D&md5=26084a450b406d951f4d828e295f7f56</span></div><a href="/servlet/linkout?suffix=cit81&amp;dbid=16384&amp;doi=10.1038%2Fnature08975&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature08975%26sid%3Dliteratum%253Aachs%26aulast%3DGupta%26aufirst%3DR.%2BA.%26aulast%3DShah%26aufirst%3DN.%26aulast%3DWang%26aufirst%3DK.%2BC.%26aulast%3DKim%26aufirst%3DJ.%26aulast%3DHorlings%26aufirst%3DH.%2BM.%26aulast%3DWong%26aufirst%3DD.%2BJ.%26aulast%3DTsai%26aufirst%3DM.%2BC.%26aulast%3DHung%26aufirst%3DT.%26aulast%3DArgani%26aufirst%3DP.%26aulast%3DRinn%26aufirst%3DJ.%2BL.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DBrzoska%26aufirst%3DP.%26aulast%3DKong%26aufirst%3DB.%26aulast%3DLi%26aufirst%3DR.%26aulast%3DWest%26aufirst%3DR.%2BB.%26aulast%3Dvan%2Bde%2BVijver%26aufirst%3DM.%2BJ.%26aulast%3DSukumar%26aufirst%3DS.%26aulast%3DChang%26aufirst%3DH.%2BY.%26atitle%3DLong%2520non-coding%2520RNA%2520HOTAIR%2520reprograms%2520chromatin%2520state%2520to%2520promote%2520cancer%2520metastasis%26jtitle%3DNature%26date%3D2010%26volume%3D464%26spage%3D1071%26epage%3D1076%26doi%3D10.1038%2Fnature08975" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref82"><div class="reference"><strong class="refLabel"><a href="#ref82" class="rightTabRefNumLink">82</a></strong><div class="NLM_citation" id="rightTab-cit82"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span> <span> </span><span class="NLM_article-title">Long non-coding RNA HOTAIR: A novel oncogene</span>. <i>Mol. Med. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">5611</span>â <span class="NLM_lpage">5618</span>, <span class="refDoi">Â DOI: 10.3892/mmr.2015.4161</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.3892%2Fmmr.2015.4161" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=26238267" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFWns7fO" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2015&pages=5611-5618&author=X.+Yuauthor=Z.+Li&title=Long+non-coding+RNA+HOTAIR%3A+A+novel+oncogene&doi=10.3892%2Fmmr.2015.4161"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit82R"><div class="casContent"><span class="casTitleNuber">82</span><div class="casTitle"><span class="NLM_cas:atitle">Long non-coding RNA HOTAIR: a novel oncogene (Review)</span></div><div class="casAuthors">Yu, Xin; Li, Zheng</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Medicine Reports</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">4, Pt. B</span>),
    <span class="NLM_cas:pages">5611-5618</span>CODEN:
                <span class="NLM_cas:coden">MMROA5</span>;
        ISSN:<span class="NLM_cas:issn">1791-2997</span>.
    
            (<span class="NLM_cas:orgname">Spandidos Publications Ltd.</span>)
        </div><div class="casAbstract">Long non-coding RNAs (lncRNAs) have been found to be pervasively transcribed in the genome and are crit. regulators of the epigenome.  Increasing evidence suggests that lncRNAs are aberrantly expressed in several types of human cancer and that they are important in the initiation, development and metastasis of human cancer.  Previous studies have revealed that HOX transcript antisense intergenic RNA (HOTAIR) was frequently upregulated in various types of cancer, including breast cancer, esophageal cancer, lung cancer and gastric cancer.  In addn., patients with high expression levels of HOTAIR have a significantly poorer prognosis, compared with those with low levels of expression.  HOTAIR is involved in the control of cell apoptosis, growth, metastasis, angiogenesis, DNA repair and tumor cells metab.  The present review provides an overview of the current knowledge concerning the role of HOTAIR in tumor development and progression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqKPrwBMvNoE7Vg90H21EOLACvtfcHk0lg43QYIftxKVQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFWns7fO&md5=39ca36f6d205ba645dac04d0e8b8e0a1</span></div><a href="/servlet/linkout?suffix=cit82&amp;dbid=16384&amp;doi=10.3892%2Fmmr.2015.4161&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3892%252Fmmr.2015.4161%26sid%3Dliteratum%253Aachs%26aulast%3DYu%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DZ.%26atitle%3DLong%2520non-coding%2520RNA%2520HOTAIR%253A%2520A%2520novel%2520oncogene%26jtitle%3DMol.%2520Med.%2520Rep.%26date%3D2015%26volume%3D12%26spage%3D5611%26epage%3D5618%26doi%3D10.3892%2Fmmr.2015.4161" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref83"><div class="reference"><strong class="refLabel"><a href="#ref83" class="rightTabRefNumLink">83</a></strong><div class="NLM_citation" id="rightTab-cit83"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, C.</span></span> <span> </span><span class="NLM_article-title">Long non-coding RNA HOTAIR promotes glioblastoma cell cycle progression in an EZH2 dependent manner</span>. <i>Oncotarget.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">537</span>â <span class="NLM_lpage">546</span>, <span class="refDoi">Â DOI: 10.18632/oncotarget.2681</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.18632%2Foncotarget.2681" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=25428914" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A280%3ADC%252BC2MzhtFSnug%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=537-546&author=K.+Zhangauthor=X.+Sunauthor=X.+Zhouauthor=L.+Hanauthor=L.+Chenauthor=Z.+Shiauthor=A.+Zhangauthor=M.+Yeauthor=Q.+Wangauthor=C.+Liuauthor=J.+Weiauthor=Y.+Renauthor=J.+Yangauthor=J.+Zhangauthor=P.+Puauthor=M.+Liauthor=C.+Kang&title=Long+non-coding+RNA+HOTAIR+promotes+glioblastoma+cell+cycle+progression+in+an+EZH2+dependent+manner&doi=10.18632%2Foncotarget.2681"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit83R"><div class="casContent"><span class="casTitleNuber">83</span><div class="casTitle"><span class="NLM_cas:atitle">Long non-coding RNA HOTAIR promotes glioblastoma cell cycle progression in an EZH2 dependent manner</span></div><div class="casAuthors">Zhang Kailiang; Sun Xiaotian; Zhou Xuan; Han Lei; Chen Luyue; Shi Zhendong; Zhang Anling; Wang Qixue; Wei Jianwei; Zhang Jianning; Pu Peiyu; Kang Chunsheng; Ye Minhua; Liu Chaoyong; Ren Yu; Yang Jingxuan; Li Min</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">537-46</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The long non-coding RNA Hox transcript antisense intergenic RNA (HOTAIR) was recently implicated in breast cancer metastasis and is predictive of poor prognosis in colorectal and pancreatic cancers.  We recently discovered that HOTAIR is a cell cycle-related lncRNA in human glioma, and its expression is closely associated with glioma staging and poor prognosis.  Although lysine specific demethylase 1 (LSD1) and polycomb repressive complex 2 (PRC2) have been demonstrated to be functional targets of HOTAIR, how HOTAIR regulates glioma cell cycle progression remains largely unknown.  In this study, we found that EZH2 (predominant PRC2 complex component) inhibition blocked cell cycle progression in glioma cells, consistent with the effects elicited by HOTAIR siRNA.  However, the inhibition of LSD1 did not affect cell cycle progression in glioma cells.  These results suggest that HOTAIR might regulate cell cycle progression through EZH2.  Our intracranial mice model also revealed delayed tumor growth in HOTAIR siRNA- and EZH2 inhibitor-treated groups.  Moreover, in HOTAIR knock-down cell lines, the expression of the PRC2-binding domain of HOTAIR (5' domain) but not of the LSD1-binding domain of HOTAIR (3' domain) resulted in accelerated cell cycle progression.  In conclusion, HOTAIR promotes cell cycle progression in glioma as a result of the binding of its 5' domain to the PRC2 complex.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSe5I48OswLk6Nc3HjJRoFAfW6udTcc2ear0PI3Aqj42bntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2MzhtFSnug%253D%253D&md5=d6c09a59a964632af9750a6fbf7b506b</span></div><a href="/servlet/linkout?suffix=cit83&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.2681&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.2681%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DK.%26aulast%3DSun%26aufirst%3DX.%26aulast%3DZhou%26aufirst%3DX.%26aulast%3DHan%26aufirst%3DL.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DShi%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DA.%26aulast%3DYe%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DQ.%26aulast%3DLiu%26aufirst%3DC.%26aulast%3DWei%26aufirst%3DJ.%26aulast%3DRen%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DPu%26aufirst%3DP.%26aulast%3DLi%26aufirst%3DM.%26aulast%3DKang%26aufirst%3DC.%26atitle%3DLong%2520non-coding%2520RNA%2520HOTAIR%2520promotes%2520glioblastoma%2520cell%2520cycle%2520progression%2520in%2520an%2520EZH2%2520dependent%2520manner%26jtitle%3DOncotarget.%26date%3D2015%26volume%3D6%26spage%3D537%26epage%3D546%26doi%3D10.18632%2Foncotarget.2681" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref84"><div class="reference"><strong class="refLabel"><a href="#ref84" class="rightTabRefNumLink">84</a></strong><div class="NLM_citation" id="rightTab-cit84"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ren, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mei, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, C.</span></span> <span> </span><span class="NLM_article-title">Targeted design and identification of AC1NOD4Q to block activity of HOTAIR by abrogating the scaffold interaction with EZH2</span>. <i>Clin. Epigenet.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">29</span>, <span class="refDoi">Â DOI: 10.1186/s13148-019-0624-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1186%2Fs13148-019-0624-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=30764859" title="">PubMed</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2019&pages=29&author=Y.+Renauthor=Y.+Wangauthor=J.+Zhangauthor=Q.+Wangauthor=L.+Hanauthor=M.+Meiauthor=C.+Kang&title=Targeted+design+and+identification+of+AC1NOD4Q+to+block+activity+of+HOTAIR+by+abrogating+the+scaffold+interaction+with+EZH2&doi=10.1186%2Fs13148-019-0624-2"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit84&amp;dbid=16384&amp;doi=10.1186%2Fs13148-019-0624-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs13148-019-0624-2%26sid%3Dliteratum%253Aachs%26aulast%3DRen%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DQ.%26aulast%3DHan%26aufirst%3DL.%26aulast%3DMei%26aufirst%3DM.%26aulast%3DKang%26aufirst%3DC.%26atitle%3DTargeted%2520design%2520and%2520identification%2520of%2520AC1NOD4Q%2520to%2520block%2520activity%2520of%2520HOTAIR%2520by%2520abrogating%2520the%2520scaffold%2520interaction%2520with%2520EZH2%26jtitle%3DClin.%2520Epigenet.%26date%3D2019%26volume%3D11%26spage%3D29%26doi%3D10.1186%2Fs13148-019-0624-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref85"><div class="reference"><strong class="refLabel"><a href="#ref85" class="rightTabRefNumLink">85</a></strong><div class="NLM_citation" id="rightTab-cit85"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Malina, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mills, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pelletier, J.</span></span> <span> </span><span class="NLM_article-title">Emerging therapeutics targeting mRNA translation</span>. <i>Cold Spring Harbor Perspect. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>4</i></span>, <span class="NLM_elocation-id">a012377</span> <span class="refDoi">Â DOI: 10.1101/cshperspect.a012377</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1101%2Fcshperspect.a012377" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=22474009" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC38XpvFyrs74%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2012&author=A.+Malinaauthor=J.+R.+Millsauthor=J.+Pelletier&title=Emerging+therapeutics+targeting+mRNA+translation&doi=10.1101%2Fcshperspect.a012377"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit85R"><div class="casContent"><span class="casTitleNuber">85</span><div class="casTitle"><span class="NLM_cas:atitle">Emerging therapeutics targeting mRNA translation</span></div><div class="casAuthors">Malina, Abba; Mills, John R.; Pelletier, Jerry</div><div class="citationInfo"><span class="NLM_cas:title">Cold Spring Harbor Perspectives in Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">a012377, 18 pp.</span>CODEN:
                <span class="NLM_cas:coden">CSHPEU</span>;
        ISSN:<span class="NLM_cas:issn">1943-0264</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">A review.  A defining feature of many cancers is deregulated translational control.  Typically, this occurs at the level of recruitment of the 40S ribosomes to the 5'-cap of cellular mRNAs (mRNAs), the rate-limiting step of protein synthesis, which is controlled by the heterotrimeric eukaryotic initiation complex eIF4F.  Thus, eIF4F in particular, and translation initiation in general, represent an exploitable vulnerability and unique opportunity for therapeutic intervention in many transformed cells.  In this article, we discuss the development, mode of action and biol. activity of a no. of small-mol. inhibitors that interrupt PI3K/mTOR signaling control of eIF4F assembly, as well as compds. that more directly block eIF4F activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqH7fO5RA3nqLVg90H21EOLACvtfcHk0lj8gPlHCfChPg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XpvFyrs74%253D&md5=b1619198f480d1fb1c8c04e296633e2a</span></div><a href="/servlet/linkout?suffix=cit85&amp;dbid=16384&amp;doi=10.1101%2Fcshperspect.a012377&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fcshperspect.a012377%26sid%3Dliteratum%253Aachs%26aulast%3DMalina%26aufirst%3DA.%26aulast%3DMills%26aufirst%3DJ.%2BR.%26aulast%3DPelletier%26aufirst%3DJ.%26atitle%3DEmerging%2520therapeutics%2520targeting%2520mRNA%2520translation%26jtitle%3DCold%2520Spring%2520Harbor%2520Perspect.%2520Biol.%26date%3D2012%26volume%3D4%26doi%3D10.1101%2Fcshperspect.a012377" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref86"><div class="reference"><strong class="refLabel"><a href="#ref86" class="rightTabRefNumLink">86</a></strong><div class="NLM_citation" id="rightTab-cit86"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Agarwala, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pandey, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maiti, S.</span></span> <span> </span><span class="NLM_article-title">The tale of RNA G-quadruplex</span>. <i>Org. Biomol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">5570</span>â <span class="NLM_lpage">5585</span>, <span class="refDoi">Â DOI: 10.1039/C4OB02681K</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1039%2FC4OB02681K" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=25879384" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC2MXlslegsb0%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2015&pages=5570-5585&author=P.+Agarwalaauthor=S.+Pandeyauthor=S.+Maiti&title=The+tale+of+RNA+G-quadruplex&doi=10.1039%2FC4OB02681K"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit86R"><div class="casContent"><span class="casTitleNuber">86</span><div class="casTitle"><span class="NLM_cas:atitle">The tale of RNA G-quadruplex</span></div><div class="casAuthors">Agarwala, Prachi; Pandey, Satyaprakash; Maiti, Souvik</div><div class="citationInfo"><span class="NLM_cas:title">Organic & Biomolecular Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">5570-5585</span>CODEN:
                <span class="NLM_cas:coden">OBCRAK</span>;
        ISSN:<span class="NLM_cas:issn">1477-0520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  G-quadruplexes are non-canonical secondary structures found in guanine rich regions of DNA and RNA.  Reports have indicated the wide occurrence of RNA G-quadruplexes across the transcriptome in various regions of mRNAs and non-coding RNAs.  RNA G-quadruplexes have been implicated in playing an important role in translational regulation, mRNA processing events and maintenance of chromosomal end integrity.  In this review, we summarize the structural and functional aspects of RNA G-quadruplexes with emphasis on recent progress to understand the protein/trans factors binding these motifs.  With the revelation of the importance of these secondary structures as regulatory modules in biol., we have also evaluated the various advancements towards targeting these structures and the challenges assocd. with them.  Apart from this, numerous potential applications of this secondary motif have also been discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq7MAm0c-9cbrVg90H21EOLACvtfcHk0lj8gPlHCfChPg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXlslegsb0%253D&md5=e51f502c003a43d1d9b6ab4506f6d8a5</span></div><a href="/servlet/linkout?suffix=cit86&amp;dbid=16384&amp;doi=10.1039%2FC4OB02681K&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC4OB02681K%26sid%3Dliteratum%253Aachs%26aulast%3DAgarwala%26aufirst%3DP.%26aulast%3DPandey%26aufirst%3DS.%26aulast%3DMaiti%26aufirst%3DS.%26atitle%3DThe%2520tale%2520of%2520RNA%2520G-quadruplex%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2015%26volume%3D13%26spage%3D5570%26epage%3D5585%26doi%3D10.1039%2FC4OB02681K" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref87"><div class="reference"><strong class="refLabel"><a href="#ref87" class="rightTabRefNumLink">87</a></strong><div class="NLM_citation" id="rightTab-cit87"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Katsuda, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sato, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asano, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morimura, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furuta, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sugiyama, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hagihara, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uesugi, M.</span></span> <span> </span><span class="NLM_article-title">A small molecule that represses translation of G-quadruplex-containing mRNA</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>138</i></span>,  <span class="NLM_fpage">9037</span>â <span class="NLM_lpage">9040</span>, <span class="refDoi">Â DOI: 10.1021/jacs.6b04506</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.6b04506" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFKms7%252FP" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=138&publication_year=2016&pages=9037-9040&author=Y.+Katsudaauthor=S.+Satoauthor=L.+Asanoauthor=Y.+Morimuraauthor=T.+Furutaauthor=H.+Sugiyamaauthor=M.+Hagiharaauthor=M.+Uesugi&title=A+small+molecule+that+represses+translation+of+G-quadruplex-containing+mRNA&doi=10.1021%2Fjacs.6b04506"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit87R"><div class="casContent"><span class="casTitleNuber">87</span><div class="casTitle"><span class="NLM_cas:atitle">A small molecule that represses translation of G-quadruplex-containing mRNA</span></div><div class="casAuthors">Katsuda, Yousuke; Sato, Shin-ichi; Asano, Lisa; Morimura, Yoshitaka; Furuta, Tomoyuki; Sugiyama, Hiroshi; Hagihara, Masaki; Uesugi, Motonari</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">138</span>
        (<span class="NLM_cas:issue">29</span>),
    <span class="NLM_cas:pages">9037-9040</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The G-quadruplexes form highly stable nucleic acid structures, which are implicated in various biol. processes in both DNA and RNA.  Although DNA G-quadruplexes have been studied in great detail, biol. roles of RNA G-quadruplexes have received less attention.  Here, a screening of a chem. library permitted identification of a small-mol. tool that binds selectively to RNA G-quadruplex structures.  The polyarom. mol., RGB-1, stabilizes RNA G-quadruplex, but not DNA versions or other RNA structures.  RGB-1 intensified the G-quadruplex-mediated inhibition of RNA translation in mammalian cells, decreased expression of the NRAS proto-oncogene in breast cancer cells, and permitted identification of a novel sequence that forms G-quadruplex in NRAS mRNA.  RGB-1 may serve as a unique tool for understanding cellular roles of RNA G-quadruplex structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5-OEuiG5IuLVg90H21EOLACvtfcHk0lj8gPlHCfChPg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtFKms7%252FP&md5=75ad58de109aa50cc964db46bc46f69b</span></div><a href="/servlet/linkout?suffix=cit87&amp;dbid=16384&amp;doi=10.1021%2Fjacs.6b04506&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.6b04506%26sid%3Dliteratum%253Aachs%26aulast%3DKatsuda%26aufirst%3DY.%26aulast%3DSato%26aufirst%3DS.%26aulast%3DAsano%26aufirst%3DL.%26aulast%3DMorimura%26aufirst%3DY.%26aulast%3DFuruta%26aufirst%3DT.%26aulast%3DSugiyama%26aufirst%3DH.%26aulast%3DHagihara%26aufirst%3DM.%26aulast%3DUesugi%26aufirst%3DM.%26atitle%3DA%2520small%2520molecule%2520that%2520represses%2520translation%2520of%2520G-quadruplex-containing%2520mRNA%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2016%26volume%3D138%26spage%3D9037%26epage%3D9040%26doi%3D10.1021%2Fjacs.6b04506" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref88"><div class="reference"><strong class="refLabel"><a href="#ref88" class="rightTabRefNumLink">88</a></strong><div class="NLM_citation" id="rightTab-cit88"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Plant, E. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dinman, J. D.</span></span> <span> </span><span class="NLM_article-title">The role of programmed-1 ribosomal frameshifting in coronavirus propagation</span>. <i>Front. Biosci., Landmark Ed.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">4873</span>â <span class="NLM_lpage">4881</span>, <span class="refDoi">Â DOI: 10.2741/3046</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.2741%2F3046" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=18508552" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BD1cXnsVOjtrw%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2008&pages=4873-4881&author=E.+P.+Plantauthor=J.+D.+Dinman&title=The+role+of+programmed-1+ribosomal+frameshifting+in+coronavirus+propagation&doi=10.2741%2F3046"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit88R"><div class="casContent"><span class="casTitleNuber">88</span><div class="casTitle"><span class="NLM_cas:atitle">The role of programmed-1 ribosomal frameshifting in coronavirus propagation</span></div><div class="casAuthors">Plant, Ewan P.; Dinman, Jonathan D.</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in Bioscience</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">4873-4881</span>CODEN:
                <span class="NLM_cas:coden">FRBIF6</span>;
        ISSN:<span class="NLM_cas:issn">1093-4715</span>.
    
            (<span class="NLM_cas:orgname">Frontiers in Bioscience</span>)
        </div><div class="casAbstract">A review.  Coronaviruses have the potential to cause significant economic, agricultural and health problems.  The severe acute respiratory syndrome (SARS) assocd. coronavirus outbreak in late 2002, early 2003 called attention to the potential damage that coronaviruses could cause in the human population.  The ensuing research has enlightened many to the mol. biol. of coronaviruses.  A programmed -1 ribosomal frameshift is required by coronaviruses for the prodn. of the RNA dependent RNA polymerase which in turn is essential for viral replication.  The frameshifting signal encoded in the viral genome has addnl. features that are not essential for frameshifting.  Elucidation of the differences between coronavirus frameshift signals and signals from other viruses may help our understanding of these features.  Here we summarize current knowledge and add addnl. insight regarding the function of the programmed -1 ribosomal frameshift signal in the coronavirus lifecycle.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZWj0x7RQKvLVg90H21EOLACvtfcHk0lgx-QepB79onA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXnsVOjtrw%253D&md5=8bcadd716bed6f996bce39d1da1a0fac</span></div><a href="/servlet/linkout?suffix=cit88&amp;dbid=16384&amp;doi=10.2741%2F3046&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2741%252F3046%26sid%3Dliteratum%253Aachs%26aulast%3DPlant%26aufirst%3DE.%2BP.%26aulast%3DDinman%26aufirst%3DJ.%2BD.%26atitle%3DThe%2520role%2520of%2520programmed-1%2520ribosomal%2520frameshifting%2520in%2520coronavirus%2520propagation%26jtitle%3DFront.%2520Biosci.%252C%2520Landmark%2520Ed.%26date%3D2008%26volume%3D13%26spage%3D4873%26epage%3D4881%26doi%3D10.2741%2F3046" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref89"><div class="reference"><strong class="refLabel"><a href="#ref89" class="rightTabRefNumLink">89</a></strong><div class="NLM_citation" id="rightTab-cit89"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Peselis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Serganov, A.</span></span> <span> </span><span class="NLM_article-title">Structure and function of pseudoknots involved in gene expression control</span>. <i>Wiley Interdiscip. Rev. RNA</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">803</span>â <span class="NLM_lpage">822</span>, <span class="refDoi">Â DOI: 10.1002/wrna.1247</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1002%2Fwrna.1247" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=25044223" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVSjsLvE" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=803-822&author=A.+Peselisauthor=A.+Serganov&title=Structure+and+function+of+pseudoknots+involved+in+gene+expression+control&doi=10.1002%2Fwrna.1247"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit89R"><div class="casContent"><span class="casTitleNuber">89</span><div class="casTitle"><span class="NLM_cas:atitle">Structure and function of pseudoknots involved in gene expression control</span></div><div class="casAuthors">Peselis, Alla; Serganov, Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Wiley Interdisciplinary Reviews: RNA</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">803-822</span>CODEN:
                <span class="NLM_cas:coden">WIRRCU</span>;
        ISSN:<span class="NLM_cas:issn">1757-7012</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  Natural RNA mols. can have a high degree of structural complexity but even the most complexly folded RNAs are assembled from simple structural building blocks.  Among the simplest RNA elements are double-stranded helixes that participate in the formation of different folding topologies and constitute the major fraction of RNA structures.  One common folding motif of RNA is a pseudoknot, defined as a bipartite helical structure formed by base-pairing of the apical loop in the stem-loop structure with an outside sequence.  Pseudoknots constitute integral parts of the RNA structures essential for various cellular activities.  Among many functions of pseudoknotted RNAs is feedback regulation of gene expression, carried out through specific recognition of various mols.  Pseudoknotted RNAs autoregulate ribosomal and phage protein genes in response to downstream encoded proteins, while many metabolic and transport genes are controlled by cellular metabolites interacting with pseudoknotted RNA elements from the riboswitch family.  Modulation of some genes also depends on metabolite-induced mRNA cleavage performed by pseudoknotted ribozymes.  Several regulatory pseudoknots have been characterized biochem. and structurally in great detail.  These studies have demonstrated a plethora of pseudoknot-based folds and have begun uncovering diverse mol. principles of the ligand-dependent gene expression control.  The pseudoknot-mediated mechanisms of gene control and many unexpected and interesting features of the regulatory pseudoknots have significantly advanced our understanding of the genetic circuits and laid the foundation for modulation of their outcomes.  Â© 2014 John Wiley & Sons, Ltd.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrkScYBDta3RLVg90H21EOLACvtfcHk0lgx-QepB79onA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVSjsLvE&md5=d936f91eac6a6844bf2fc04b0114649a</span></div><a href="/servlet/linkout?suffix=cit89&amp;dbid=16384&amp;doi=10.1002%2Fwrna.1247&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fwrna.1247%26sid%3Dliteratum%253Aachs%26aulast%3DPeselis%26aufirst%3DA.%26aulast%3DSerganov%26aufirst%3DA.%26atitle%3DStructure%2520and%2520function%2520of%2520pseudoknots%2520involved%2520in%2520gene%2520expression%2520control%26jtitle%3DWiley%2520Interdiscip.%2520Rev.%2520RNA%26date%3D2014%26volume%3D5%26spage%3D803%26epage%3D822%26doi%3D10.1002%2Fwrna.1247" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref90"><div class="reference"><strong class="refLabel"><a href="#ref90" class="rightTabRefNumLink">90</a></strong><div class="NLM_citation" id="rightTab-cit90"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Matsumoto, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caliskan, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodnina, M. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murata, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakatani, K.</span></span> <span> </span><span class="NLM_article-title">Small synthetic molecule-stabilized RNA pseudoknot as an activator for â1 ribosomal frameshifting</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">8079</span>â <span class="NLM_lpage">8089</span>, <span class="refDoi">Â DOI: 10.1093/nar/gky689</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref90/cit90&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1093%2Fnar%2Fgky689" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref90/cit90&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=30085309" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref90/cit90&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC1MXosFanu7s%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2018&pages=8079-8089&author=S.+Matsumotoauthor=N.+Caliskanauthor=M.+V.+Rodninaauthor=A.+Murataauthor=K.+Nakatani&title=Small+synthetic+molecule-stabilized+RNA+pseudoknot+as+an+activator+for+%E2%88%921+ribosomal+frameshifting&doi=10.1093%2Fnar%2Fgky689"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit90R"><div class="casContent"><span class="casTitleNuber">90</span><div class="casTitle"><span class="NLM_cas:atitle">Small synthetic molecule-stabilized RNA pseudoknot as an activator for -1 ribosomal frameshifting</span></div><div class="casAuthors">Matsumoto, Saki; Caliskan, Neva; Rodnina, Marina V.; Murata, Asako; Nakatani, Kazuhiko</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">8079-8089</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">1362-4962</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Programmed -1 ribosomal frameshifting (-1PRF) is a recoding mechanism to make alternative proteins from a single mRNA transcript.  -1PRF is stimulated by cis-acting signals in mRNA, a seven-nucleotide slippery sequence and a downstream secondary structure element, which is often a pseudoknot.  In this study we engineered the frameshifting pseudoknot from the mouse mammary tumor virus to respond to a rationally designed small mol. naphthyridine carbamate tetramer (NCTn).  We demonstrate that NCTn can stabilize the pseudoknot structure in mRNA and activate -1PRF both in vitro and in human cells.  The results illustrate how NCTn-inducible -1PRF may serve as an important component of the synthetic biol. toolbox for the precise control of gene expression using small synthetic mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_y6f3PayO7LVg90H21EOLACvtfcHk0lgx-QepB79onA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXosFanu7s%253D&md5=b9d5ba7cf6605fa1b6c0d46bf12b7847</span></div><a href="/servlet/linkout?suffix=cit90&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgky689&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgky689%26sid%3Dliteratum%253Aachs%26aulast%3DMatsumoto%26aufirst%3DS.%26aulast%3DCaliskan%26aufirst%3DN.%26aulast%3DRodnina%26aufirst%3DM.%2BV.%26aulast%3DMurata%26aufirst%3DA.%26aulast%3DNakatani%26aufirst%3DK.%26atitle%3DSmall%2520synthetic%2520molecule-stabilized%2520RNA%2520pseudoknot%2520as%2520an%2520activator%2520for%2520%25E2%2588%25921%2520ribosomal%2520frameshifting%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2018%26volume%3D46%26spage%3D8079%26epage%3D8089%26doi%3D10.1093%2Fnar%2Fgky689" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref91"><div class="reference"><strong class="refLabel"><a href="#ref91" class="rightTabRefNumLink">91</a></strong><div class="NLM_citation" id="rightTab-cit91"><span><span class="NLM_contrib-group"><span class="NLM_string-name">BurrÃ©, J.</span></span> <span> </span><span class="NLM_article-title">The synaptic function of Î±-synuclein</span>. <i>J. Parkinson's Dis.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">699</span>â <span class="NLM_lpage">713</span>, <span class="refDoi">Â DOI: 10.3233/JPD-150642</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.3233%2FJPD-150642" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=26407041" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitVSqs7fM" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2015&pages=699-713&author=J.+Burr%C3%A9&title=The+synaptic+function+of+%CE%B1-synuclein&doi=10.3233%2FJPD-150642"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit91R"><div class="casContent"><span class="casTitleNuber">91</span><div class="casTitle"><span class="NLM_cas:atitle">The Synaptic Function of Î±-Synuclein</span></div><div class="casAuthors">Burre, Jacqueline</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Parkinson's Disease</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">699-713</span>CODEN:
                <span class="NLM_cas:coden">JPDOAP</span>;
        ISSN:<span class="NLM_cas:issn">1877-7171</span>.
    
            (<span class="NLM_cas:orgname">IOS Press</span>)
        </div><div class="casAbstract">Î±-Synuclein is an abundant neuronal protein which localizes predominantly to presynaptic terminals, and is strongly linked genetically and pathol. to Parkinson's disease and other neurodegenerative diseases.  While the accumulation of Î±-synuclein in the form of misfolded oligomers and large aggregates defines multiple neurodegenerative diseases called "synucleinopathies", its cellular function has remained largely unclear, and is the subject of intense investigation.  In this review, I focus on the structural characteristics of Î±-synuclein, its cellular and subcellular localization, and discuss how this relates to its function in neurons, in particular at the neuronal synapse.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqdLxDreICkVbVg90H21EOLACvtfcHk0lj8ErZuQOrXVQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitVSqs7fM&md5=6199a08d8d9138b554aa25fd45279b48</span></div><a href="/servlet/linkout?suffix=cit91&amp;dbid=16384&amp;doi=10.3233%2FJPD-150642&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3233%252FJPD-150642%26sid%3Dliteratum%253Aachs%26aulast%3DBurr%25C3%25A9%26aufirst%3DJ.%26atitle%3DThe%2520synaptic%2520function%2520of%2520%25CE%25B1-synuclein%26jtitle%3DJ.%2520Parkinson%2527s%2520Dis.%26date%3D2015%26volume%3D5%26spage%3D699%26epage%3D713%26doi%3D10.3233%2FJPD-150642" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref92"><div class="reference"><strong class="refLabel"><a href="#ref92" class="rightTabRefNumLink">92</a></strong><div class="NLM_citation" id="rightTab-cit92"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, H.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Junn, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andrews, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Velagapudi, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abegg, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vishnu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Costales, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Childs-Disney, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adibekian, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moss, W. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mouradian, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span> <span> </span><span class="NLM_article-title">Translation of the intrinsically disordered protein Î±-synuclein is inhibited by a small molecule targeting its structured mRNA</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>117</i></span>,  <span class="NLM_fpage">1457</span>â <span class="NLM_lpage">1467</span>, <span class="refDoi">Â DOI: 10.1073/pnas.1905057117</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1073%2Fpnas.1905057117" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=31900363" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhsFKktbc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=117&publication_year=2020&pages=1457-1467&author=P.+Zhangauthor=H.-J.+Parkauthor=J.+Zhangauthor=E.+Junnauthor=R.+J.+Andrewsauthor=S.+P.+Velagapudiauthor=D.+Abeggauthor=K.+Vishnuauthor=M.+G.+Costalesauthor=J.+L.+Childs-Disneyauthor=A.+Adibekianauthor=W.+N.+Mossauthor=M.+M.+Mouradianauthor=M.+D.+Disney&title=Translation+of+the+intrinsically+disordered+protein+%CE%B1-synuclein+is+inhibited+by+a+small+molecule+targeting+its+structured+mRNA&doi=10.1073%2Fpnas.1905057117"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit92R"><div class="casContent"><span class="casTitleNuber">92</span><div class="casTitle"><span class="NLM_cas:atitle">Translation of the intrinsically disordered protein Î±-synuclein is inhibited by a small molecule targeting its structured mRNA</span></div><div class="casAuthors">Zhang, Peiyuan; Park, Hye-Jin; Zhang, Jie; Junn, Eunsung; Andrews, Ryan J.; Velagapudi, Sai Pradeep; Abegg, Daniel; Vishnu, Kamalakannan; Costales, Matthew G.; Childs-Disney, Jessica L.; Adibekian, Alexander; Moss, Walter N.; Mouradian, M. Maral; Disney, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">117</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1457-1467</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Many proteins are refractory to targeting because they lack small-mol. binding pockets.  An alternative to drugging these proteins directly is to target the messenger (m)RNA that encodes them, thereby reducing protein levels.  We describe such an approach for the difficult-to-target protein Î±-synuclein encoded by the SNCA gene.  Multiplication of the SNCA gene locus causes dominantly inherited Parkinson's disease (PD), and Î±-synuclein protein aggregates in Lewy bodies and Lewy neurites in sporadic PD.  Thus, reducing the expression of Î±-synuclein protein is expected to have therapeutic value.  Fortuitously, the SNCA mRNA has a structured iron-responsive element (IRE) in its 5' untranslated region (5' UTR) that controls its translation.  Using sequence-based design, we discovered small mols. that target the IRE structure and inhibit SNCA translation in cells, the most potent of which is named Synucleozid.  Both in vitro and cellular profiling studies showed Synucleozid directly targets the Î±-synuclein mRNA 5' UTR at the designed site.Î±.  Mechanistic studies revealed that Synucleozid reduces Î±-synuclein protein levels by decreasing the amt. of SNCA mRNA loaded into polysomes, mechanistically providing a cytoprotective effect in cells.  Proteome- and transcriptome-wide studies showed that the compd.'s selectivity makes Synucleozid suitable for further development.  Importantly, transcriptome-wide anal. of mRNAs that encode intrinsically disordered proteins revealed that each has structured regions that could be targeted with small mols.  These findings demonstrate the potential for targeting undruggable proteins at the level of their coding mRNAs.  This approach, as applied to SNCA, is a promising disease-modifying therapeutic strategy for PD and other Î±-synucleinopathies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvPjL_9G6xS7Vg90H21EOLACvtfcHk0lj8ErZuQOrXVQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhsFKktbc%253D&md5=4f2a60d2c0fe09ea99f05291621ad4ef</span></div><a href="/servlet/linkout?suffix=cit92&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1905057117&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1905057117%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DP.%26aulast%3DPark%26aufirst%3DH.-J.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DJunn%26aufirst%3DE.%26aulast%3DAndrews%26aufirst%3DR.%2BJ.%26aulast%3DVelagapudi%26aufirst%3DS.%2BP.%26aulast%3DAbegg%26aufirst%3DD.%26aulast%3DVishnu%26aufirst%3DK.%26aulast%3DCostales%26aufirst%3DM.%2BG.%26aulast%3DChilds-Disney%26aufirst%3DJ.%2BL.%26aulast%3DAdibekian%26aufirst%3DA.%26aulast%3DMoss%26aufirst%3DW.%2BN.%26aulast%3DMouradian%26aufirst%3DM.%2BM.%26aulast%3DDisney%26aufirst%3DM.%2BD.%26atitle%3DTranslation%2520of%2520the%2520intrinsically%2520disordered%2520protein%2520%25CE%25B1-synuclein%2520is%2520inhibited%2520by%2520a%2520small%2520molecule%2520targeting%2520its%2520structured%2520mRNA%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2020%26volume%3D117%26spage%3D1457%26epage%3D1467%26doi%3D10.1073%2Fpnas.1905057117" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref93"><div class="reference"><strong class="refLabel"><a href="#ref93" class="rightTabRefNumLink">93</a></strong><div class="NLM_citation" id="rightTab-cit93"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cleveland, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hwo, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirschner, M. W.</span></span> <span> </span><span class="NLM_article-title">Purification of tau, a microtubule associated protein that induces assembly of microtubules from purified tubulin</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1977</span>,  <span class="NLM_volume"><i>116</i></span>,  <span class="NLM_fpage">207</span>â <span class="NLM_lpage">225</span>, <span class="refDoi">Â DOI: 10.1016/0022-2836(77)90213-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1016%2F0022-2836%2877%2990213-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=599557" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADyaE1cXks1Kisw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=1977&pages=207-225&author=D.+W.+Clevelandauthor=S.+Y.+Hwoauthor=M.+W.+Kirschner&title=Purification+of+tau%2C+a+microtubule+associated+protein+that+induces+assembly+of+microtubules+from+purified+tubulin&doi=10.1016%2F0022-2836%2877%2990213-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit93R"><div class="casContent"><span class="casTitleNuber">93</span><div class="casTitle"><span class="NLM_cas:atitle">Purification of tau, a microtubule-associated protein that induces assembly of microtubules from purified tubulin</span></div><div class="casAuthors">Cleveland, Don W.; Hwo, Shu-Ying; Kirschner, Marc W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1977</span>),
    <span class="NLM_cas:volume">116</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">207-25</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    </div><div class="casAbstract">Tau, a microtubule-assocd. protein which copurifies with tubulin through successive cycles of polymn. and depolymn., was isolated from tubulin by phosphocellulose chromatog. and purified to near homogeneity.  The purified protein migrated during electrophoresis on acrylamide gels as 4 closely spaced bands of apparent mol. wts. between 55,000 and 62,000.  Specific activity for induction of microtubule formation from purified tubulin was assayed by quant. electron microscopy and was enhanced 3-4-fold in the purified tau when compared with the unfractionated microtubule-assocd. proteins.  Nearly 90% of available tubulin at 1 mg/mL was polymerizable into microtubules with elevated levels of tau.  Moreover, the crit. concn. for polymn. of the reconstituted tau + tubulin system was a function of tau concn. and could be lowered to as little as 30 Î¼g of tubulin/mL.  Under depolymg. conditions, 50% of the tubulin at only 1 mg/mL could be driven into ring structures.  A sep. purifn. procedure for isolation of tau directly from cell exts. was developed and data from this purifn. suggest that tau is present in the ext. in roughly the same proportion to tubulin as is found in microtubules purified by cycles of assembly and disassembly.  Tau was sufficient for both nucleation and elongation of microtubules from purified tubulin and, hence, the reconstituted tau + tubulin system defined a complete microtubule assembly system under std. buffer conditions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqYiOJMmcY9-7Vg90H21EOLACvtfcHk0lj8ErZuQOrXVQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE1cXks1Kisw%253D%253D&md5=61ff67f90dc42e4c19e13ab536d5de15</span></div><a href="/servlet/linkout?suffix=cit93&amp;dbid=16384&amp;doi=10.1016%2F0022-2836%2877%2990213-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0022-2836%252877%252990213-3%26sid%3Dliteratum%253Aachs%26aulast%3DCleveland%26aufirst%3DD.%2BW.%26aulast%3DHwo%26aufirst%3DS.%2BY.%26aulast%3DKirschner%26aufirst%3DM.%2BW.%26atitle%3DPurification%2520of%2520tau%252C%2520a%2520microtubule%2520associated%2520protein%2520that%2520induces%2520assembly%2520of%2520microtubules%2520from%2520purified%2520tubulin%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1977%26volume%3D116%26spage%3D207%26epage%3D225%26doi%3D10.1016%2F0022-2836%2877%2990213-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref94"><div class="reference"><strong class="refLabel"><a href="#ref94" class="rightTabRefNumLink">94</a></strong><div class="NLM_citation" id="rightTab-cit94"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ross, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poirier, M. A.</span></span> <span> </span><span class="NLM_article-title">Protein aggregation and neurodegenerative disease</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">S10</span>â <span class="NLM_lpage">S17</span>, <span class="refDoi">Â DOI: 10.1038/nm1066</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1038%2Fnm1066" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=15272267" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A280%3ADC%252BD2czptF2rtg%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2004&pages=S10-S17&author=C.+A.+Rossauthor=M.+A.+Poirier&title=Protein+aggregation+and+neurodegenerative+disease&doi=10.1038%2Fnm1066"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit94R"><div class="casContent"><span class="casTitleNuber">94</span><div class="casTitle"><span class="NLM_cas:atitle">Protein aggregation and neurodegenerative disease</span></div><div class="casAuthors">Ross Christopher A; Poirier Michelle A</div><div class="citationInfo"><span class="NLM_cas:title">Nature medicine</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">10 Suppl</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">S10-7</span>
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    </div><div class="casAbstract">Neurodegenerative diseases such as Alzheimer's disease (AD), Parkinson's disease (PD), Huntington's disease (HD), amyotrophic lateral sclerosis (ALS) and prion diseases are increasingly being realized to have common cellular and molecular mechanisms including protein aggregation and inclusion body formation.  The aggregates usually consist of fibers containing misfolded protein with a beta-sheet conformation, termed amyloid.  There is partial but not perfect overlap among the cells in which abnormal proteins are deposited and the cells that degenerate.  The most likely explanation is that inclusions and other visible protein aggregates represent an end stage of a molecular cascade of several steps, and that earlier steps in the cascade may be more directly tied to pathogenesis than the inclusions themselves.  For several diseases, genetic variants assist in explaining the pathogenesis of the more common sporadic forms and developing mouse and other models.  There is now increased understanding of the pathways involved in protein aggregation, and some recent clues have emerged as to the molecular mechanisms of cellular toxicity.  These are leading to approaches toward rational therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR51TEFJZTv4mx8TEN9K18QfW6udTcc2eamXIV5HCX1yrntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2czptF2rtg%253D%253D&md5=b83d9efabba21b8dd2265f3547cdb0ad</span></div><a href="/servlet/linkout?suffix=cit94&amp;dbid=16384&amp;doi=10.1038%2Fnm1066&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm1066%26sid%3Dliteratum%253Aachs%26aulast%3DRoss%26aufirst%3DC.%2BA.%26aulast%3DPoirier%26aufirst%3DM.%2BA.%26atitle%3DProtein%2520aggregation%2520and%2520neurodegenerative%2520disease%26jtitle%3DNat.%2520Med.%26date%3D2004%26volume%3D10%26spage%3DS10%26epage%3DS17%26doi%3D10.1038%2Fnm1066" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref95"><div class="reference"><strong class="refLabel"><a href="#ref95" class="rightTabRefNumLink">95</a></strong><div class="NLM_citation" id="rightTab-cit95"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wolfe, M. S.</span></span> <span> </span><span class="NLM_article-title">Targeting mRNA for Alzheimerâs and related dementias</span>. <i>Scientifica</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>2014</i></span>,  <span class="NLM_fpage">757549</span>, <span class="refDoi">Â DOI: 10.1155/2014/757549</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref95/cit95&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1155%2F2014%2F757549" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref95/cit95&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=24876993" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref95/cit95&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A280%3ADC%252BC2cflvFCqsw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2014&publication_year=2014&pages=757549&author=M.+S.+Wolfe&title=Targeting+mRNA+for+Alzheimer%E2%80%99s+and+related+dementias&doi=10.1155%2F2014%2F757549"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit95R"><div class="casContent"><span class="casTitleNuber">95</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting mRNA for Alzheimer's and related dementias</span></div><div class="casAuthors">Wolfe Michael S</div><div class="citationInfo"><span class="NLM_cas:title">Scientifica</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">2014</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">757549</span>
        ISSN:<span class="NLM_cas:issn">2090-908X</span>.
    </div><div class="casAbstract">Brain deposition of the amyloid beta-protein (A Î² ) and tau are characteristic features in Alzheimer's disease (AD).  Mutations in the A Î² precursor protein (APP) and a protease involved in A Î² production from APP strongly argue for a pathogenic role of A Î² in AD, while mutations in tau are associated with related disorders collectively called frontotemporal lobar degeneration (FTLD).  Despite intense effort, therapeutic strategies that target A Î² or tau have not yet yielded medications, suggesting that alternative approaches should be pursued.  In recent years, our laboratory has studied the role of mRNA in AD and FTLD, specifically those encoding tau and the A Î² -producing protease BACE1.  As many FTLD-causing tau mutations destabilize a hairpin structure that regulates RNA splicing, we have targeted this structure with small molecules, antisense oligonucleotides, and small molecule-antisense conjugates.  We have also discovered that microRNA interaction with the 3'-untranslated region of tau regulates tau expression.  Regarding BACE1, we found that alternative splicing leads to inactive splice isoforms and antisense oligonucleotides shift splicing toward these inactive isoforms to decrease A Î² production.  In addition, a G-quadruplex structure in the BACE1 mRNA plays a role in splice regulation.  The prospects for targeting tau and BACE1 mRNAs as therapeutic strategies will be discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQOSA4BMYD0yZPq5LRczm8ffW6udTcc2eamXIV5HCX1yrntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2cflvFCqsw%253D%253D&md5=6e03652eb4aacf52189b1f5c5f02184a</span></div><a href="/servlet/linkout?suffix=cit95&amp;dbid=16384&amp;doi=10.1155%2F2014%2F757549&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1155%252F2014%252F757549%26sid%3Dliteratum%253Aachs%26aulast%3DWolfe%26aufirst%3DM.%2BS.%26atitle%3DTargeting%2520mRNA%2520for%2520Alzheimer%25E2%2580%2599s%2520and%2520related%2520dementias%26jtitle%3DScientifica%26date%3D2014%26volume%3D2014%26spage%3D757549%26doi%3D10.1155%2F2014%2F757549" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref96"><div class="reference"><strong class="refLabel"><a href="#ref96" class="rightTabRefNumLink">96</a></strong><div class="NLM_citation" id="rightTab-cit96"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Artigas, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MarchÃ¡n, V.</span></span> <span> </span><span class="NLM_article-title">Synthesis and Tau RNA binding evaluation of ametantrone-containing ligands</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>80</i></span>,  <span class="NLM_fpage">2155</span>â <span class="NLM_lpage">2164</span>, <span class="refDoi">Â DOI: 10.1021/jo502661j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo502661j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVChtLk%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=80&publication_year=2015&pages=2155-2164&author=G.+Artigasauthor=V.+March%C3%A1n&title=Synthesis+and+Tau+RNA+binding+evaluation+of+ametantrone-containing+ligands&doi=10.1021%2Fjo502661j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit96R"><div class="casContent"><span class="casTitleNuber">96</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and Tau RNA Binding Evaluation of Ametantrone-Containing Ligands</span></div><div class="casAuthors">Artigas, Gerard; Marchan, Vicente</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">80</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">2155-2164</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We describe the synthesis and characterization of ametantrone-contg. RNA ligands based on the derivatization of this intercalator with two neamine moieties (Amt-Nea,Nea) or with one azaquinolone heterocycle and one neamine (Amt-Nea,Azq) as well as its combination with guanidinoneamine (Amt-NeaG4).  Biophys. studies revealed that guanidinylation of the parent ligand (Amt-Nea) had a pos. effect on the binding of the resulting compd. for Tau pre-mRNA target as well as on the stabilization upon complexation of some of the mutated RNA sequences assocd. with the development of tauopathies.  Further studies by NMR revealed the existence of a preferred binding site in the stem-loop structure, in which ametantrone intercalates in the characteristic bulged region.  Regarding doubly-functionalized ligands, binding affinity and stabilizing ability of Amt-Nea,Nea were similar to those of the guanidinylated ligand, but the two aminoglycoside fragments seem to interfere with its accommodation in a single binding site.  However, Amt-Nea,Azq binds at the bulged region in a similar way than Amt-NeaG4.  Overall, these results provide new insights on fine-tuning RNA binding properties of ametantrone by single or double derivatization with other RNA recognition motifs, which could help in the future design of new ligands with improved selectivity for disease-causing RNA mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGogzTuI8jMylLVg90H21EOLACvtfcHk0lg_r7zIGj06lw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVChtLk%253D&md5=e2048c1d0094ddc5155eb884b18ca7fa</span></div><a href="/servlet/linkout?suffix=cit96&amp;dbid=16384&amp;doi=10.1021%2Fjo502661j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo502661j%26sid%3Dliteratum%253Aachs%26aulast%3DArtigas%26aufirst%3DG.%26aulast%3DMarch%25C3%25A1n%26aufirst%3DV.%26atitle%3DSynthesis%2520and%2520Tau%2520RNA%2520binding%2520evaluation%2520of%2520ametantrone-containing%2520ligands%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2015%26volume%3D80%26spage%3D2155%26epage%3D2164%26doi%3D10.1021%2Fjo502661j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref97"><div class="reference"><strong class="refLabel"><a href="#ref97" class="rightTabRefNumLink">97</a></strong><div class="NLM_citation" id="rightTab-cit97"><span><span class="NLM_contrib-group"><span class="NLM_string-name">LÃ³pez-SenÃ­n, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">GÃ³mez-Pinto, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandas, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MarchÃ¡n, V.</span></span> <span> </span><span class="NLM_article-title">Identification of ligands for the Tau exon 10 splicing regulatory element RNA by using dynamic combinatorial chemistry</span>. <i>Chem. - Eur. J.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">1946</span>â <span class="NLM_lpage">1953</span>, <span class="refDoi">Â DOI: 10.1002/chem.201002065</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1002%2Fchem.201002065" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=21274946" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFKlu74%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2011&pages=1946-1953&author=P.+L%C3%B3pez-Sen%C3%ADnauthor=I.+G%C3%B3mez-Pintoauthor=A.+Grandasauthor=V.+March%C3%A1n&title=Identification+of+ligands+for+the+Tau+exon+10+splicing+regulatory+element+RNA+by+using+dynamic+combinatorial+chemistry&doi=10.1002%2Fchem.201002065"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit97R"><div class="casContent"><span class="casTitleNuber">97</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of Ligands for the Tau Exon 10 Splicing Regulatory Element RNA by Using Dynamic Combinatorial Chemistry</span></div><div class="casAuthors">Lopez-Senin, Paula; Gomez-Pinto, Irene; Grandas, Anna; Marchan, Vicente</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry - A European Journal</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1946-1953, S1946/1-S1946/68</span>CODEN:
                <span class="NLM_cas:coden">CEUJED</span>;
        ISSN:<span class="NLM_cas:issn">0947-6539</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">We describe the use of dynamic combinatorial chem. (DCC) to identify ligands for the stem-loop structure located at the exon 10-5'-intron junction of Tau pre-mRNA, which is involved in the onset of several tauopathies including frontotemporal dementia with Parkinsonism linked to chromosome 17 (FTDP-17).  A series of ligands that combine the small aminoglycoside neamine and heteroarom. moieties (azaquinolone and two acridines) have been identified by using DCC.  These compds. effectively bind the stem-loop RNA target (the concn. required for 50 % RNA response (EC50): 2-58 Î¼M), as detd. by fluorescence titrn. expts.  Importantly, most of them are able to stabilize both the wild-type and the +3 and +14 mutated sequences assocd. with the development of FTDP-17 without producing a significant change in the overall structure of the RNA (as analyzed by CD spectroscopy), which is a key factor for recognition by the splicing regulatory machinery.  A good correlation has been found between the affinity of the ligands for the target and their ability to stabilize the RNA secondary structure.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpj2NBVOgduLrVg90H21EOLACvtfcHk0lhRwJ4H-1h0TA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFKlu74%253D&md5=2d9a5224455d93350dd05122f6ceddd1</span></div><a href="/servlet/linkout?suffix=cit97&amp;dbid=16384&amp;doi=10.1002%2Fchem.201002065&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fchem.201002065%26sid%3Dliteratum%253Aachs%26aulast%3DL%25C3%25B3pez-Sen%25C3%25ADn%26aufirst%3DP.%26aulast%3DG%25C3%25B3mez-Pinto%26aufirst%3DI.%26aulast%3DGrandas%26aufirst%3DA.%26aulast%3DMarch%25C3%25A1n%26aufirst%3DV.%26atitle%3DIdentification%2520of%2520ligands%2520for%2520the%2520Tau%2520exon%252010%2520splicing%2520regulatory%2520element%2520RNA%2520by%2520using%2520dynamic%2520combinatorial%2520chemistry%26jtitle%3DChem.%2520-%2520Eur.%2520J.%26date%3D2011%26volume%3D17%26spage%3D1946%26epage%3D1953%26doi%3D10.1002%2Fchem.201002065" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref98"><div class="reference"><strong class="refLabel"><a href="#ref98" class="rightTabRefNumLink">98</a></strong><div class="NLM_citation" id="rightTab-cit98"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gianoncelli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Basili, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scalabrin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sosic, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moro, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zagotto, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palumbo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gresh, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gatto, B.</span></span> <span> </span><span class="NLM_article-title">Rational design, synthesis, and DNA binding properties of novel sequence-selective peptidyl congeners of ametantrone</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">1080</span>â <span class="NLM_lpage">1091</span>, <span class="refDoi">Â DOI: 10.1002/cmdc.201000106</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1002%2Fcmdc.201000106" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=20458714" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC3cXotVOmurg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2010&pages=1080-1091&author=A.+Gianoncelliauthor=S.+Basiliauthor=M.+Scalabrinauthor=A.+Sosicauthor=S.+Moroauthor=G.+Zagottoauthor=M.+Palumboauthor=N.+Greshauthor=B.+Gatto&title=Rational+design%2C+synthesis%2C+and+DNA+binding+properties+of+novel+sequence-selective+peptidyl+congeners+of+ametantrone&doi=10.1002%2Fcmdc.201000106"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit98R"><div class="casContent"><span class="casTitleNuber">98</span><div class="casTitle"><span class="NLM_cas:atitle">Rational Design, Synthesis, and DNA Binding Properties of Novel Sequence-Selective Peptidyl Congeners of Ametantrone</span></div><div class="casAuthors">Gianoncelli, Alessandra; Basili, Serena; Scalabrin, Matteo; Sosic, Alice; Moro, Stefano; Zagotto, Giuseppe; Palumbo, Manlio; Gresh, Nohad; Gatto, Barbara</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1080-1091</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Natural and synthetic compds. characterized by an anthraquinone nucleus represent an important class of anti-neoplastic agents, the mechanism of action of which is related to intercalation into DNA.  Ametantrone (AM) is a synthetic 9,10-anthracenedione bearing two (hydroxyethylamino)ethylamino residues at positions 1 and 4; along with other anthraquinones and anthracyclines, it shares a polycyclic intercalating moiety and charged side chains that stabilize DNA binding.  All these drugs elicit adverse side effects, which represent a challenge for antitumor chemotherapy.  In the present work the structure of AM was augmented with appropriate groups that target well-defined base pairs in the major groove.  These should endow AM with DNA sequence selectivity.  We describe the rationale for the synthesis and the evaluation of activity of a new series of compds. in which the planar anthraquinone is conjugated at positions 1 and 4 through the side chains of AM or other bioisosteric linkers to appropriate dipeptides.  The designed novel AM derivs. were shown to selectively stabilize two oligonucleotide duplexes that both have a palindromic GC-rich hexanucleotide core, but their stabilizing effects on a random DNA sequence was negligible.  In the case of the most effective compd., the 1,4-bis-[Gly-(L-Lys)] deriv. of AM, the exptl. results confirm the predictions of earlier theor. computations.  In contrast, AM had equal stabilizing effects on all three sequences and showed no preferential binding.  This novel peptide deriv. can be classified as a strong binder regarding the sequences that it selectively targets, possibly opening the exploitation of less cytotoxic conjugates of AM to the targeted treatment of oncol. and viral diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJASjZVc5nd7Vg90H21EOLACvtfcHk0lhRwJ4H-1h0TA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXotVOmurg%253D&md5=0e96bda08bba3f3ac943b7713754f7f3</span></div><a href="/servlet/linkout?suffix=cit98&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201000106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201000106%26sid%3Dliteratum%253Aachs%26aulast%3DGianoncelli%26aufirst%3DA.%26aulast%3DBasili%26aufirst%3DS.%26aulast%3DScalabrin%26aufirst%3DM.%26aulast%3DSosic%26aufirst%3DA.%26aulast%3DMoro%26aufirst%3DS.%26aulast%3DZagotto%26aufirst%3DG.%26aulast%3DPalumbo%26aufirst%3DM.%26aulast%3DGresh%26aufirst%3DN.%26aulast%3DGatto%26aufirst%3DB.%26atitle%3DRational%2520design%252C%2520synthesis%252C%2520and%2520DNA%2520binding%2520properties%2520of%2520novel%2520sequence-selective%2520peptidyl%2520congeners%2520of%2520ametantrone%26jtitle%3DChemMedChem%26date%3D2010%26volume%3D5%26spage%3D1080%26epage%3D1091%26doi%3D10.1002%2Fcmdc.201000106" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref99"><div class="reference"><strong class="refLabel"><a href="#ref99" class="rightTabRefNumLink">99</a></strong><div class="NLM_citation" id="rightTab-cit99"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abe, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aikawa, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frank, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zembryski, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hubbs, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Withka, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steppan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogers, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cabral, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pettersson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wager, T. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fountain, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rumbaugh, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Childs-Disney, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span> <span> </span><span class="NLM_article-title">Design, optimization, and study of small Molecules that target tau pre-mRNA and affect splicing</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>142</i></span>,  <span class="NLM_fpage">8706</span>â <span class="NLM_lpage">8727</span>, <span class="refDoi">Â DOI: 10.1021/jacs.0c00768</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.0c00768" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A280%3ADC%252BB38vis1WrtA%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=142&publication_year=2020&pages=8706-8727&author=J.+L.+Chenauthor=P.+Zhangauthor=M.+Abeauthor=H.+Aikawaauthor=L.+Zhangauthor=A.+J.+Frankauthor=T.+Zembryskiauthor=C.+Hubbsauthor=H.+Parkauthor=J.+Withkaauthor=C.+Steppanauthor=L.+Rogersauthor=S.+Cabralauthor=M.+Petterssonauthor=T.+T.+Wagerauthor=M.+A.+Fountainauthor=G.+Rumbaughauthor=J.+L.+Childs-Disneyauthor=M.+D.+Disney&title=Design%2C+optimization%2C+and+study+of+small+Molecules+that+target+tau+pre-mRNA+and+affect+splicing&doi=10.1021%2Fjacs.0c00768"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit99R"><div class="casContent"><span class="casTitleNuber">99</span><div class="casTitle"><span class="NLM_cas:atitle">Design, Optimization, and Study of Small Molecules That Target Tau Pre-mRNA and Affect Splicing</span></div><div class="casAuthors">Chen Jonathan L; Zhang Peiyuan; Abe Masahito; Aikawa Haruo; Hubbs Christopher; Park HaJeung; Rumbaugh Gavin; Childs-Disney Jessica L; Disney Matthew D; Zhang Liying; Withka Jane; Pettersson Martin; Wager Travis T; Frank Alexander J; Zembryski Timothy; Fountain Matthew A; Steppan Claire; Rogers Lucy; Cabral Shawn</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">142</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">8706-8727</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Approximately 95% of human genes are alternatively spliced, and aberrant splicing events can cause disease.  One pre-mRNA that is alternatively spliced and linked to neurodegenerative diseases is tau (microtubule-associated protein tau), which can cause frontotemporal dementia and parkinsonism linked to chromosome 17 (FTDP-17) and can contribute to Alzheimer's disease.  Here, we describe the design of structure-specific lead small molecules that directly target tau pre-mRNA from sequence.  This was followed by hit expansion and analogue synthesis to further improve upon these initial lead molecules.  The emergent compounds were assessed for functional activity in a battery of assays, including binding assays and an assay that mimics molecular recognition of tau pre-mRNA by a U1 small nuclear ribonucleoprotein (snRNP) splicing factor.  Compounds that emerged from these studies had enhanced potency and selectivity for the target RNA relative to the initial hits, while also having significantly improved drug-like properties.  The compounds are shown to directly target tau pre-mRNA in cells, via chemical cross-linking and isolation by pull-down target profiling, and to rescue disease-relevant splicing of tau pre-mRNA in a variety of cellular systems, including primary neurons.  More broadly, this study shows that lead, structure-specific compounds can be designed from sequence and then further optimized for their physicochemical properties while at the same time enhancing their activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQ7onHoxn5nY0mhnSqMnvzDfW6udTcc2eZyvNY-vX3GoLntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB38vis1WrtA%253D%253D&md5=3622b3f0ea9417db76b78b222fd75ddc</span></div><a href="/servlet/linkout?suffix=cit99&amp;dbid=16384&amp;doi=10.1021%2Fjacs.0c00768&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.0c00768%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DJ.%2BL.%26aulast%3DZhang%26aufirst%3DP.%26aulast%3DAbe%26aufirst%3DM.%26aulast%3DAikawa%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DFrank%26aufirst%3DA.%2BJ.%26aulast%3DZembryski%26aufirst%3DT.%26aulast%3DHubbs%26aufirst%3DC.%26aulast%3DPark%26aufirst%3DH.%26aulast%3DWithka%26aufirst%3DJ.%26aulast%3DSteppan%26aufirst%3DC.%26aulast%3DRogers%26aufirst%3DL.%26aulast%3DCabral%26aufirst%3DS.%26aulast%3DPettersson%26aufirst%3DM.%26aulast%3DWager%26aufirst%3DT.%2BT.%26aulast%3DFountain%26aufirst%3DM.%2BA.%26aulast%3DRumbaugh%26aufirst%3DG.%26aulast%3DChilds-Disney%26aufirst%3DJ.%2BL.%26aulast%3DDisney%26aufirst%3DM.%2BD.%26atitle%3DDesign%252C%2520optimization%252C%2520and%2520study%2520of%2520small%2520Molecules%2520that%2520target%2520tau%2520pre-mRNA%2520and%2520affect%2520splicing%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2020%26volume%3D142%26spage%3D8706%26epage%3D8727%26doi%3D10.1021%2Fjacs.0c00768" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref100"><div class="reference"><strong class="refLabel"><a href="#ref100" class="rightTabRefNumLink">100</a></strong><div class="NLM_citation" id="rightTab-cit100"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brunden, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trojanowski, J. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, V. M. Y.</span></span> <span> </span><span class="NLM_article-title">Advances in tau-focused drug discovery for Alzheimerâs disease and related tauopathies</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">783</span>â <span class="NLM_lpage">793</span>, <span class="refDoi">Â DOI: 10.1038/nrd2959</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1038%2Fnrd2959" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=19794442" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtF2kurrN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2009&pages=783-793&author=K.+R.+Brundenauthor=J.+Q.+Trojanowskiauthor=V.+M.+Y.+Lee&title=Advances+in+tau-focused+drug+discovery+for+Alzheimer%E2%80%99s+disease+and+related+tauopathies&doi=10.1038%2Fnrd2959"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit100R"><div class="casContent"><span class="casTitleNuber">100</span><div class="casTitle"><span class="NLM_cas:atitle">Advances in tau-focused drug discovery for Alzheimer's disease and related tauopathies</span></div><div class="casAuthors">Brunden, Kurt R.; Trojanowski, John Q.; Lee, Virginia M.-Y.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">783-793</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Neuronal inclusions comprised of the microtubule-assocd. protein tau are found in numerous neurodegenerative diseases, commonly known as tauopathies.  In Alzheimer's disease - the most prevalent tauopathy - misfolded tau is probably a key pathol. agent.  The recent failure of amyloid-Î²-targeted therapeutics in Phase III clin. trials suggests that it is timely and prudent to consider alternative drug discovery strategies for Alzheimer's disease.  Here, we focus on strategies directed at reducing misfolded tau and compensating for the loss of normal tau function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqeWhB7oau9ALVg90H21EOLACvtfcHk0liXuMhUErl7tA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtF2kurrN&md5=6178c6f0e9a83c31a0e7f9dd8dae79d8</span></div><a href="/servlet/linkout?suffix=cit100&amp;dbid=16384&amp;doi=10.1038%2Fnrd2959&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2959%26sid%3Dliteratum%253Aachs%26aulast%3DBrunden%26aufirst%3DK.%2BR.%26aulast%3DTrojanowski%26aufirst%3DJ.%2BQ.%26aulast%3DLee%26aufirst%3DV.%2BM.%2BY.%26atitle%3DAdvances%2520in%2520tau-focused%2520drug%2520discovery%2520for%2520Alzheimer%25E2%2580%2599s%2520disease%2520and%2520related%2520tauopathies%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2009%26volume%3D8%26spage%3D783%26epage%3D793%26doi%3D10.1038%2Fnrd2959" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref101"><div class="reference"><strong class="refLabel"><a href="#ref101" class="rightTabRefNumLink">101</a></strong><div class="NLM_citation" id="rightTab-cit101"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguez, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolfe, M. S.</span></span> <span> </span><span class="NLM_article-title">Template-directed synthesis of a small molecule-antisense conjugate targeting an mRNA structure</span>. <i>Bioorg. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">7</span>â <span class="NLM_lpage">11</span>, <span class="refDoi">Â DOI: 10.1016/j.bioorg.2014.03.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1016%2Fj.bioorg.2014.03.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=24691171" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC2cXpsl2qs7g%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2014&pages=7-11&author=Y.+Liuauthor=L.+Rodriguezauthor=M.+S.+Wolfe&title=Template-directed+synthesis+of+a+small+molecule-antisense+conjugate+targeting+an+mRNA+structure&doi=10.1016%2Fj.bioorg.2014.03.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit101R"><div class="casContent"><span class="casTitleNuber">101</span><div class="casTitle"><span class="NLM_cas:atitle">Template-directed synthesis of a small molecule-antisense conjugate targeting an mRNA structure</span></div><div class="casAuthors">Liu, Yang; Rodriguez, Lilia; Wolfe, Michael S.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">7-11</span>CODEN:
                <span class="NLM_cas:coden">BOCMBM</span>;
        ISSN:<span class="NLM_cas:issn">0045-2068</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The targeting of structural features in mRNA with specificity remains a great chem. challenge.  A hairpin structure near exon 10 in the pre-mRNA encoding the tau protein controls its splicing, and dementia-causing mutations that disrupt this structure increase exon 10 splicing.  We previously reported the discovery of small mols., mitoxantrone (MTX) and analogs, which bind to the tau RNA hairpin structure and the design of bipartite antisense oligonucleotides (ASOs) that simultaneously bind to the discontinuous sequences that flank this hairpin.  Herein we report the synthesis of a bipartite ASO conjugated to MTX using the tau RNA hairpin and flanking sequences as a template.  A set of six MTX analogs, each contg. a linker-azide, and a set of ten bipartite ASOs, each contg. a branched linker-alkyne, were synthesized and tested in combinatorial fashion for their ability to conjugate in the presence or absence of template RNA.  A single template-dependent MTX-ASO conjugate was identified from among the 60 reaction mixts., demonstrating that the MTX and ASO precursors could simultaneously bind the RNA template and allow proper positioning of azide and alkyne for 1,3-cycloaddn.  While the MTX-ASO conjugate proved too cytotoxic for cell-based assays, the conjugate inhibited tau exon 10 splicing under cell-free conditions more effectively than MTX or bipartite ASO alone.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqMQ3QTybseSLVg90H21EOLACvtfcHk0liXuMhUErl7tA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXpsl2qs7g%253D&md5=70e5fd0e2ed2120650af43f05be7a33b</span></div><a href="/servlet/linkout?suffix=cit101&amp;dbid=16384&amp;doi=10.1016%2Fj.bioorg.2014.03.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bioorg.2014.03.001%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DRodriguez%26aufirst%3DL.%26aulast%3DWolfe%26aufirst%3DM.%2BS.%26atitle%3DTemplate-directed%2520synthesis%2520of%2520a%2520small%2520molecule-antisense%2520conjugate%2520targeting%2520an%2520mRNA%2520structure%26jtitle%3DBioorg.%2520Chem.%26date%3D2014%26volume%3D54%26spage%3D7%26epage%3D11%26doi%3D10.1016%2Fj.bioorg.2014.03.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref102"><div class="reference"><strong class="refLabel"><a href="#ref102" class="rightTabRefNumLink">102</a></strong><div class="NLM_citation" id="rightTab-cit102"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donahue, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolfe, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Varani, G.</span></span> <span> </span><span class="NLM_article-title">Structural basis for stabilization of the Tau pre-mRNA splicing regulatory element by novantrone (mitoxantrone)</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">557</span>â <span class="NLM_lpage">566</span>, <span class="refDoi">Â DOI: 10.1016/j.chembiol.2009.03.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1016%2Fj.chembiol.2009.03.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=19477420" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BD1MXms1Ogtb4%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2009&pages=557-566&author=S.+Zhengauthor=Y.+Chenauthor=C.+P.+Donahueauthor=M.+S.+Wolfeauthor=G.+Varani&title=Structural+basis+for+stabilization+of+the+Tau+pre-mRNA+splicing+regulatory+element+by+novantrone+%28mitoxantrone%29&doi=10.1016%2Fj.chembiol.2009.03.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit102R"><div class="casContent"><span class="casTitleNuber">102</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Basis for Stabilization of the Tau Pre-mRNA Splicing Regulatory Element by Novantrone (Mitoxantrone)</span></div><div class="casAuthors">Zheng, Suxin; Chen, Yu; Donahue, Christine P.; Wolfe, Michael S.; Varani, Gabriele</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">557-566</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Some familial neurodegenerative diseases are assocd. with mutations that destabilize a putative stem-loop structure within an intronic region of the tau pre-mRNA and alter the prodn. of tau protein isoforms by alternative splicing.  Because stabilization of the stem loop reverses the splicing pattern assocd. with neurodegeneration, small mols. that stabilize this stem loop would provide new ways to dissect the mechanism of neurodegeneration and treat tauopathies.  The anticancer drug mitoxantrone was recently identified in a high throughput screen to stabilize the tau pre-mRNA stem loop.  Here the authors report the soln. structure of the tau mRNA-mitoxantrone complex, validated by the structure-activity relationship of existing mitoxantrone analogs.  The structure describes the mol. basis for their interaction with RNA and provides a rational basis to optimize the activity of this new class of RNA-binding mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrNxlGdrUiUcrVg90H21EOLACvtfcHk0liXuMhUErl7tA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXms1Ogtb4%253D&md5=3d38884c150535d160fcecc5d1f03534</span></div><a href="/servlet/linkout?suffix=cit102&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2009.03.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2009.03.009%26sid%3Dliteratum%253Aachs%26aulast%3DZheng%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DDonahue%26aufirst%3DC.%2BP.%26aulast%3DWolfe%26aufirst%3DM.%2BS.%26aulast%3DVarani%26aufirst%3DG.%26atitle%3DStructural%2520basis%2520for%2520stabilization%2520of%2520the%2520Tau%2520pre-mRNA%2520splicing%2520regulatory%2520element%2520by%2520novantrone%2520%2528mitoxantrone%2529%26jtitle%3DChem.%2520Biol.%26date%3D2009%26volume%3D16%26spage%3D557%26epage%3D566%26doi%3D10.1016%2Fj.chembiol.2009.03.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref103"><div class="reference"><strong class="refLabel"><a href="#ref103" class="rightTabRefNumLink">103</a></strong><div class="NLM_citation" id="rightTab-cit103"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Peacey, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguez, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolfe, M. S.</span></span> <span> </span><span class="NLM_article-title">Targeting a pre-mRNA structure with bipartite antisense molecules modulates tau alternative splicing</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">9836</span>â <span class="NLM_lpage">9849</span>, <span class="refDoi">Â DOI: 10.1093/nar/gks710</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1093%2Fnar%2Fgks710" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=22844088" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFygurnF" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2012&pages=9836-9849&author=E.+Peaceyauthor=L.+Rodriguezauthor=Y.+Liuauthor=M.+S.+Wolfe&title=Targeting+a+pre-mRNA+structure+with+bipartite+antisense+molecules+modulates+tau+alternative+splicing&doi=10.1093%2Fnar%2Fgks710"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit103R"><div class="casContent"><span class="casTitleNuber">103</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting a pre-mRNA structure with bipartite antisense molecules modulates tau alternative splicing</span></div><div class="casAuthors">Peacey, Eleanor; Rodriguez, Lilia; Liu, Yang; Wolfe, Michael S.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">9836-9849</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Approx. 15% of human genetic diseases are estd. to involve dysregulation of alternative pre-mRNA splicing.  Antisense mols. designed to alter these and other splicing events typically target continuous linear sequences of the message.  Here, we show that a structural feature in a pre-mRNA can be targeted by bipartite antisense mols. designed to hybridize with the discontinuous elements that flank the structure and thereby alter splicing.  We targeted a hairpin structure at the boundary between exon 10 and intron 10 of the pre-mRNA of tau.  Mutations in this region that are assocd. with certain forms of frontotemporal dementia, destabilize the hairpin to cause increased inclusion of exon 10.  Via electrophoretic mobility shift and RNase protection assays, we demonstrate that bipartite antisense mols. designed to simultaneously interact with the available sequences that immediately flank the tau pre-mRNA hairpin do indeed bind to this structured region.  Moreover, these agents inhibit exon 10 splicing and reverse the effect of destabilizing disease-causing mutations, in both in vitro splicing assays and cell culture.  This general bipartite antisense strategy could be employed to modulate other splicing events that are regulated by RNA secondary structure.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpC63B1SXFOE7Vg90H21EOLACvtfcHk0li0tVBY-LCvCg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFygurnF&md5=63d1cc9c4956e68211bf35a3081bcd08</span></div><a href="/servlet/linkout?suffix=cit103&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgks710&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgks710%26sid%3Dliteratum%253Aachs%26aulast%3DPeacey%26aufirst%3DE.%26aulast%3DRodriguez%26aufirst%3DL.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DWolfe%26aufirst%3DM.%2BS.%26atitle%3DTargeting%2520a%2520pre-mRNA%2520structure%2520with%2520bipartite%2520antisense%2520molecules%2520modulates%2520tau%2520alternative%2520splicing%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2012%26volume%3D40%26spage%3D9836%26epage%3D9849%26doi%3D10.1093%2Fnar%2Fgks710" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref104"><div class="reference"><strong class="refLabel"><a href="#ref104" class="rightTabRefNumLink">104</a></strong><div class="NLM_citation" id="rightTab-cit104"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sumner, C.</span>; <span class="NLM_string-name">Paushkin, S.</span>; <span class="NLM_string-name">Ko, C.-P.</span></span> <i>Spinal muscular atrophy: Disease mechanism and therapy.</i>; <span class="NLM_publisher-name">Academic Press</span>, <span class="NLM_publisher-loc">Cambridge</span>, <span class="NLM_year">2016</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&author=C.+Sumner&author=S.+Paushkin&author=C.-P.+Ko&title=Spinal+muscular+atrophy%3A+Disease+mechanism+and+therapy."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit104&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DSumner%26aufirst%3DC.%26jtitle%3DSpinal%2520muscular%2520atrophy%253A%2520Disease%2520mechanism%2520and%2520therapy.%26pub%3DAcademic%2520Press%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref105"><div class="reference"><strong class="refLabel"><a href="#ref105" class="rightTabRefNumLink">105</a></strong><div class="NLM_citation" id="rightTab-cit105"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ratni, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ebeling, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baird, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bendels, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bylund, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denk, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guerard, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jablonski, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobsen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khwaja, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kletzl, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ko, C.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kustermann, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marquet, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Metzger, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mueller, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naryshkin, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paushkin, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pinard, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poirier, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reutlinger, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weetall, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeller, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mueller, L.</span></span> <span> </span><span class="NLM_article-title">Discovery of risdiplam, a selective survival of motor neuron-2 (SMN2) gene splicing modifier for the treatment of spinal muscular atrophy (SMA)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">6501</span>â <span class="NLM_lpage">6517</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.8b00741</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00741" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref105/cit105&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtlOgu7vF" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=6501-6517&author=H.+Ratniauthor=M.+Ebelingauthor=J.+Bairdauthor=S.+Bendelsauthor=J.+Bylundauthor=K.+S.+Chenauthor=N.+Denkauthor=Z.+Fengauthor=L.+Greenauthor=M.+Guerardauthor=P.+Jablonskiauthor=B.+Jacobsenauthor=O.+Khwajaauthor=H.+Kletzlauthor=C.-P.+Koauthor=S.+Kustermannauthor=A.+Marquetauthor=F.+Metzgerauthor=B.+Muellerauthor=N.+A.+Naryshkinauthor=S.+V.+Paushkinauthor=E.+Pinardauthor=A.+Poirierauthor=M.+Reutlingerauthor=M.+Weetallauthor=A.+Zellerauthor=X.+Zhaoauthor=L.+Mueller&title=Discovery+of+risdiplam%2C+a+selective+survival+of+motor+neuron-2+%28SMN2%29+gene+splicing+modifier+for+the+treatment+of+spinal+muscular+atrophy+%28SMA%29&doi=10.1021%2Facs.jmedchem.8b00741"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit105R"><div class="casContent"><span class="casTitleNuber">105</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Risdiplam, a Selective Survival of Motor Neuron-2 (SMN2) Gene Splicing Modifier for the Treatment of Spinal Muscular Atrophy (SMA)</span></div><div class="casAuthors">Ratni, Hasane; Ebeling, Martin; Baird, John; Bendels, Stefanie; Bylund, Johan; Chen, Karen S.; Denk, Nora; Feng, Zhihua; Green, Luke; Guerard, Melanie; Jablonski, Philippe; Jacobsen, Bjoern; Khwaja, Omar; Kletzl, Heidemarie; Ko, Chien-Ping; Kustermann, Stefan; Marquet, Anne; Metzger, Friedrich; Mueller, Barbara; Naryshkin, Nikolai A.; Paushkin, Sergey V.; Pinard, Emmanuel; Poirier, Agnes; Reutlinger, Michael; Weetall, Marla; Zeller, Andreas; Zhao, Xin; Mueller, Lutz</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">6501-6517</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">SMA is an inherited disease that leads to loss of motor function and ambulation and a reduced life expectancy.  We have been working to develop orally administrated, systemically distributed small mols. to increase levels of functional SMN protein.  Compd. 2 was the first SMN2 splicing modifier tested in clin. trials in healthy volunteers and SMA patients.  It was safe and well tolerated and increased SMN protein levels up to 2-fold in patients.  Nevertheless, its development was stopped as a precautionary measure because retinal toxicity was obsd. in cynomolgus monkeys after chronic daily oral dosing (39 wk) at exposures in excess of those investigated in patients.  Herein, we describe the discovery of 1 (risdiplam, RG7916, RO7034067) that focused on thorough pharmacol., DMPK and safety characterization and optimization.  This compd. is undergoing pivotal clin. trials and is a promising medicine for the treatment of patients in all ages and stages with SMA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCF_mNT3UFurVg90H21EOLACvtfcHk0li0tVBY-LCvCg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtlOgu7vF&md5=966fa21a9b1bbcc4d64ac5c90508a23b</span></div><a href="/servlet/linkout?suffix=cit105&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00741&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00741%26sid%3Dliteratum%253Aachs%26aulast%3DRatni%26aufirst%3DH.%26aulast%3DEbeling%26aufirst%3DM.%26aulast%3DBaird%26aufirst%3DJ.%26aulast%3DBendels%26aufirst%3DS.%26aulast%3DBylund%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DK.%2BS.%26aulast%3DDenk%26aufirst%3DN.%26aulast%3DFeng%26aufirst%3DZ.%26aulast%3DGreen%26aufirst%3DL.%26aulast%3DGuerard%26aufirst%3DM.%26aulast%3DJablonski%26aufirst%3DP.%26aulast%3DJacobsen%26aufirst%3DB.%26aulast%3DKhwaja%26aufirst%3DO.%26aulast%3DKletzl%26aufirst%3DH.%26aulast%3DKo%26aufirst%3DC.-P.%26aulast%3DKustermann%26aufirst%3DS.%26aulast%3DMarquet%26aufirst%3DA.%26aulast%3DMetzger%26aufirst%3DF.%26aulast%3DMueller%26aufirst%3DB.%26aulast%3DNaryshkin%26aufirst%3DN.%2BA.%26aulast%3DPaushkin%26aufirst%3DS.%2BV.%26aulast%3DPinard%26aufirst%3DE.%26aulast%3DPoirier%26aufirst%3DA.%26aulast%3DReutlinger%26aufirst%3DM.%26aulast%3DWeetall%26aufirst%3DM.%26aulast%3DZeller%26aufirst%3DA.%26aulast%3DZhao%26aufirst%3DX.%26aulast%3DMueller%26aufirst%3DL.%26atitle%3DDiscovery%2520of%2520risdiplam%252C%2520a%2520selective%2520survival%2520of%2520motor%2520neuron-2%2520%2528SMN2%2529%2520gene%2520splicing%2520modifier%2520for%2520the%2520treatment%2520of%2520spinal%2520muscular%2520atrophy%2520%2528SMA%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D6501%26epage%3D6517%26doi%3D10.1021%2Facs.jmedchem.8b00741" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref106"><div class="reference"><strong class="refLabel"><a href="#ref106" class="rightTabRefNumLink">106</a></strong><div class="NLM_citation" id="rightTab-cit106"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheung, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurley, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerrigan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chin, D. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">OâBrien, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sung, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Axford, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cody, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fazal, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fridrich, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanchez, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomlinson, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jain, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffmaster, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoosear, M. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Servais, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Towler, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hild, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curtis, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dietrich, W. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamann, L. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Briner, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobayashi, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sivasankaran, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dales, N. A.</span></span> <span> </span><span class="NLM_article-title">Discovery of small molecule splicing modulators of survival motor neuron-2 (SMN2) for the treatment of spinal muscular atrophy (SMA)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">11021</span>â <span class="NLM_lpage">11036</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.8b01291</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01291" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref106/cit106&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitFeju7nL" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=11021-11036&author=A.+K.+Cheungauthor=B.+Hurleyauthor=R.+Kerriganauthor=L.+Shuauthor=D.+N.+Chinauthor=Y.+Shenauthor=G.+O%E2%80%99Brienauthor=M.+J.+Sungauthor=Y.+Houauthor=J.+Axfordauthor=E.+Codyauthor=R.+Sunauthor=A.+Fazalauthor=C.+Fridrichauthor=C.+C.+Sanchezauthor=R.+C.+Tomlinsonauthor=M.+Jainauthor=L.+Dengauthor=K.+Hoffmasterauthor=C.+Songauthor=M.+V.+Hoosearauthor=Y.+Shinauthor=R.+Servaisauthor=C.+Towlerauthor=M.+Hildauthor=D.+Curtisauthor=W.+F.+Dietrichauthor=L.+G.+Hamannauthor=K.+Brinerauthor=K.+S.+Chenauthor=D.+Kobayashiauthor=R.+Sivasankaranauthor=N.+A.+Dales&title=Discovery+of+small+molecule+splicing+modulators+of+survival+motor+neuron-2+%28SMN2%29+for+the+treatment+of+spinal+muscular+atrophy+%28SMA%29&doi=10.1021%2Facs.jmedchem.8b01291"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit106R"><div class="casContent"><span class="casTitleNuber">106</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Small Molecule Splicing Modulators of Survival Motor Neuron-2 (SMN2) for the Treatment of Spinal Muscular Atrophy (SMA)</span></div><div class="casAuthors">Cheung, Atwood K.; Hurley, Brian; Kerrigan, Ryan; Shu, Lei; Chin, Donovan N.; Shen, Yiping; OBrien, Gary; Sung, Moo Je; Hou, Ying; Axford, Jake; Cody, Emma; Sun, Robert; Fazal, Aleem; Tomlinson, Ronald C.; Jain, Monish; Deng, Lin; Hoffmaster, Keith; Song, Cheng; Van Hoosear, Mailin; Shin, Youngah; Servais, Rebecca; Towler, Christopher; Hild, Marc; Curtis, Daniel; Dietrich, William F.; Hamann, Lawrence G.; Briner, Karin; Chen, Karen S.; Kobayashi, Dione; Sivasankaran, Rajeev; Dales, Natalie A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">11021-11036</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Spinal muscular atrophy (SMA), a rare neuromuscular disorder, is the leading genetic cause of death in infants and toddlers.  SMA is caused by the deletion or a loss of function mutation of the survival motor neuron 1 (SMN1) gene.  In humans, a second closely related gene SMN2 exists, however it codes for a less stable SMN protein.  In recent years, significant progress has been made toward disease modifying treatments for SMA by modulating SMN2 pre-mRNA splicing.  Herein, we describe the discovery of LMI070/branaplam, a small mol. that stabilizes the interaction between the spliceosome and SMN2 pre-mRNA.  Branaplam (1) originated from a high-throughput phenotypic screening hit, pyridazine 2, and evolved via multi-parameter lead optimization.  In a severe mouse SMA model, branaplam treatment increased full-length SMN RNA and protein levels, and extended survival.  Currently, branaplam is in clin. studies for SMA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprrHPkSEhGhrVg90H21EOLACvtfcHk0lgWzelFi0U0nA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitFeju7nL&md5=c564332b680a6ef0ad06bc9e5a4db16e</span></div><a href="/servlet/linkout?suffix=cit106&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01291&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01291%26sid%3Dliteratum%253Aachs%26aulast%3DCheung%26aufirst%3DA.%2BK.%26aulast%3DHurley%26aufirst%3DB.%26aulast%3DKerrigan%26aufirst%3DR.%26aulast%3DShu%26aufirst%3DL.%26aulast%3DChin%26aufirst%3DD.%2BN.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DO%25E2%2580%2599Brien%26aufirst%3DG.%26aulast%3DSung%26aufirst%3DM.%2BJ.%26aulast%3DHou%26aufirst%3DY.%26aulast%3DAxford%26aufirst%3DJ.%26aulast%3DCody%26aufirst%3DE.%26aulast%3DSun%26aufirst%3DR.%26aulast%3DFazal%26aufirst%3DA.%26aulast%3DFridrich%26aufirst%3DC.%26aulast%3DSanchez%26aufirst%3DC.%2BC.%26aulast%3DTomlinson%26aufirst%3DR.%2BC.%26aulast%3DJain%26aufirst%3DM.%26aulast%3DDeng%26aufirst%3DL.%26aulast%3DHoffmaster%26aufirst%3DK.%26aulast%3DSong%26aufirst%3DC.%26aulast%3DHoosear%26aufirst%3DM.%2BV.%26aulast%3DShin%26aufirst%3DY.%26aulast%3DServais%26aufirst%3DR.%26aulast%3DTowler%26aufirst%3DC.%26aulast%3DHild%26aufirst%3DM.%26aulast%3DCurtis%26aufirst%3DD.%26aulast%3DDietrich%26aufirst%3DW.%2BF.%26aulast%3DHamann%26aufirst%3DL.%2BG.%26aulast%3DBriner%26aufirst%3DK.%26aulast%3DChen%26aufirst%3DK.%2BS.%26aulast%3DKobayashi%26aufirst%3DD.%26aulast%3DSivasankaran%26aufirst%3DR.%26aulast%3DDales%26aufirst%3DN.%2BA.%26atitle%3DDiscovery%2520of%2520small%2520molecule%2520splicing%2520modulators%2520of%2520survival%2520motor%2520neuron-2%2520%2528SMN2%2529%2520for%2520the%2520treatment%2520of%2520spinal%2520muscular%2520atrophy%2520%2528SMA%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D11021%26epage%3D11036%26doi%3D10.1021%2Facs.jmedchem.8b01291" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref107"><div class="reference"><strong class="refLabel"><a href="#ref107" class="rightTabRefNumLink">107</a></strong><div class="NLM_citation" id="rightTab-cit107"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Orengo, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ward, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooper, T. A.</span></span> <span> </span><span class="NLM_article-title">Alternative splicing dysregulation secondary to skeletal muscle regeneration</span>. <i>Ann. Neurol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>69</i></span>,  <span class="NLM_fpage">681</span>â <span class="NLM_lpage">690</span>, <span class="refDoi">Â DOI: 10.1002/ana.22278</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1002%2Fana.22278" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=21400563" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC3MXltVygtbw%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2011&pages=681-690&author=J.+P.+Orengoauthor=A.+J.+Wardauthor=T.+A.+Cooper&title=Alternative+splicing+dysregulation+secondary+to+skeletal+muscle+regeneration&doi=10.1002%2Fana.22278"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit107R"><div class="casContent"><span class="casTitleNuber">107</span><div class="casTitle"><span class="NLM_cas:atitle">Alternative splicing dysregulation secondary to skeletal muscle regeneration</span></div><div class="casAuthors">Orengo, James P.; Ward, Amanda J.; Cooper, Thomas A.</div><div class="citationInfo"><span class="NLM_cas:title">Annals of Neurology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">681-690</span>CODEN:
                <span class="NLM_cas:coden">ANNED3</span>;
        ISSN:<span class="NLM_cas:issn">0364-5134</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">Objective : Dysregulation of alternative splicing has become a mol. hallmark of myotonic dystrophy type 1 (DM1), in which neonatal splice variants are expressed in adult skeletal muscle.  Splicing dysregulation is induced by RNA contg. expanded CUG repeats expressed from the expanded mutant allele by sequestration of muscleblindlike 1 (MBNL1) protein within nuclear RNA foci and increased CUGBP, ELAV-like family member 1 (CELF1) protein levels.  Dysregulated splicing has also been identified in other neuromuscular disorders, suggesting either that diseases with different mol. causes share a common pathogenic mechanism or that dysregulated splicing can also be a common secondary consequence of muscle degeneration and regeneration.  Methods : In this study, we examd. regulation of alternative splicing in 4 different mouse models of muscular dystrophy, including DM1, limb-girdle muscular dystrophy, congenital merosin-deficient muscular dystrophy, and Duchenne muscular dystrophy, and 2 myotoxin (cardiotoxin and notexin) muscle injury models.  Results : We show that DM1-like alternative splicing dysregulation and altered expression of MBNL1 and CELF1 occur in non-DM1 mouse models of muscular dystrophy and muscle injury, most likely due to recapitulation of neonatal splicing patterns in regenerating fibers.  In contrast, CELF1 was elevated in nuclei of mature myofibers of the DM1 model, consistent with a primary effect of pathogenic RNA expression.  Interpretation : Splicing dysregulation in DM1 is a primary effect of RNA contg. expanded CUG repeats.  However, we conclude that splicing changes can also be obsd. secondary to muscle regeneration, and this possibility must be taken into account when evaluating cause-effect relationships between dysregulated splicing and disease processes.  Ann Neurol 2010;.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomAbRWIByLm7Vg90H21EOLACvtfcHk0lgWzelFi0U0nA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXltVygtbw%253D&md5=ba02b3ea4229090c392f9204dd1658b4</span></div><a href="/servlet/linkout?suffix=cit107&amp;dbid=16384&amp;doi=10.1002%2Fana.22278&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fana.22278%26sid%3Dliteratum%253Aachs%26aulast%3DOrengo%26aufirst%3DJ.%2BP.%26aulast%3DWard%26aufirst%3DA.%2BJ.%26aulast%3DCooper%26aufirst%3DT.%2BA.%26atitle%3DAlternative%2520splicing%2520dysregulation%2520secondary%2520to%2520skeletal%2520muscle%2520regeneration%26jtitle%3DAnn.%2520Neurol.%26date%3D2011%26volume%3D69%26spage%3D681%26epage%3D690%26doi%3D10.1002%2Fana.22278" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref108"><div class="reference"><strong class="refLabel"><a href="#ref108" class="rightTabRefNumLink">108</a></strong><div class="NLM_citation" id="rightTab-cit108"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wong, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peh, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luu, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanchez, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tuccinardi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsoi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, W. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, H. Y. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baranger, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hergenrother, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimmerman, S. C.</span></span> <span> </span><span class="NLM_article-title">Targeting toxic RNAs that cause myotonic dystrophy Type 1 (DM1) with a bisamidinium inhibitor</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>136</i></span>,  <span class="NLM_fpage">6355</span>â <span class="NLM_lpage">6361</span>, <span class="refDoi">Â DOI: 10.1021/ja5012146</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja5012146" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref108/cit108&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC2cXls1Crsb8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=136&publication_year=2014&pages=6355-6361&author=C.+H.+Wongauthor=L.+Nguyenauthor=J.+Pehauthor=L.+M.+Luuauthor=J.+S.+Sanchezauthor=S.+L.+Richardsonauthor=T.+Tuccinardiauthor=H.+Tsoiauthor=W.+Y.+Chanauthor=H.+Y.+E.+Chanauthor=A.+M.+Barangerauthor=P.+J.+Hergenrotherauthor=S.+C.+Zimmerman&title=Targeting+toxic+RNAs+that+cause+myotonic+dystrophy+Type+1+%28DM1%29+with+a+bisamidinium+inhibitor&doi=10.1021%2Fja5012146"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit108R"><div class="casContent"><span class="casTitleNuber">108</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting Toxic RNAs that Cause Myotonic Dystrophy Type 1 (DM1) with a Bisamidinium Inhibitor</span></div><div class="casAuthors">Wong, Chun-Ho; Nguyen, Lien; Peh, Jessie; Luu, Long M.; Sanchez, Jeannette S.; Richardson, Stacie L.; Tuccinardi, Tiziano; Tsoi, Ho; Chan, Wood Yee; Chan, H. Y. Edwin; Baranger, Anne M.; Hergenrother, Paul J.; Zimmerman, Steven C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">136</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">6355-6361</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A working hypothesis for the pathogenesis of myotonic dystrophy type 1 (DM1) involves the aberrant sequestration of an alternative splicing regulator, MBNL1, by expanded CUG repeats, r(CUG)exp.  It has been suggested that a reversal of the myotonia and potentially other symptoms of the DM1 disease can be achieved by inhibiting the toxic MBNL1-r(CUG)exp interaction.  Using rational design, we discovered an RNA-groove binding inhibitor (ligand 3) that contains two triaminotriazine units connected by a bisamidinium linker.  Ligand 3 binds r(CUG)12 with a low micromolar affinity (Kd = 8Â±2 Î¼M) and disrupts the MBNL1-r(CUG)12 interaction in vitro (Ki = 8Â±2 Î¼M).  In addn., ligand 3 is cell and nucleus permeable, exhibits negligible toxicity to mammalian cells, dissolves MBNL1-r(CUG)exp ribonuclear foci, and restores misregulated splicing of IR and cTNT in a DM1 cell culture model.  Importantly, suppression of r(CUG)exp RNA-induced toxicity in a DM1 Drosophila model was obsd. after treatment with ligand 3.  These results suggest ligand 3 as a lead for the treatment of DM1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqfs_zU_KI7w7Vg90H21EOLACvtfcHk0lhWk53jYvPVEw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXls1Crsb8%253D&md5=205bd661957ffd5673ae516ebd6769d4</span></div><a href="/servlet/linkout?suffix=cit108&amp;dbid=16384&amp;doi=10.1021%2Fja5012146&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja5012146%26sid%3Dliteratum%253Aachs%26aulast%3DWong%26aufirst%3DC.%2BH.%26aulast%3DNguyen%26aufirst%3DL.%26aulast%3DPeh%26aufirst%3DJ.%26aulast%3DLuu%26aufirst%3DL.%2BM.%26aulast%3DSanchez%26aufirst%3DJ.%2BS.%26aulast%3DRichardson%26aufirst%3DS.%2BL.%26aulast%3DTuccinardi%26aufirst%3DT.%26aulast%3DTsoi%26aufirst%3DH.%26aulast%3DChan%26aufirst%3DW.%2BY.%26aulast%3DChan%26aufirst%3DH.%2BY.%2BE.%26aulast%3DBaranger%26aufirst%3DA.%2BM.%26aulast%3DHergenrother%26aufirst%3DP.%2BJ.%26aulast%3DZimmerman%26aufirst%3DS.%2BC.%26atitle%3DTargeting%2520toxic%2520RNAs%2520that%2520cause%2520myotonic%2520dystrophy%2520Type%25201%2520%2528DM1%2529%2520with%2520a%2520bisamidinium%2520inhibitor%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2014%26volume%3D136%26spage%3D6355%26epage%3D6361%26doi%3D10.1021%2Fja5012146" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref109"><div class="reference"><strong class="refLabel"><a href="#ref109" class="rightTabRefNumLink">109</a></strong><div class="NLM_citation" id="rightTab-cit109"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Arambula, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramisetty, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baranger, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimmerman, S. C.</span></span> <span> </span><span class="NLM_article-title">A simple ligand that selectively targets CUG trinucleotide repeats and inhibits MBNL protein binding</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>106</i></span>,  <span class="NLM_fpage">16068</span>â <span class="NLM_lpage">16073</span>, <span class="refDoi">Â DOI: 10.1073/pnas.0901824106</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref109/cit109&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1073%2Fpnas.0901824106" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref109/cit109&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=19805260" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref109/cit109&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht1OjsbjO" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2009&pages=16068-16073&author=J.+F.+Arambulaauthor=S.+R.+Ramisettyauthor=A.+M.+Barangerauthor=S.+C.+Zimmerman&title=A+simple+ligand+that+selectively+targets+CUG+trinucleotide+repeats+and+inhibits+MBNL+protein+binding&doi=10.1073%2Fpnas.0901824106"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit109R"><div class="casContent"><span class="casTitleNuber">109</span><div class="casTitle"><span class="NLM_cas:atitle">A simple ligand that selectively targets CUG trinucleotide repeats and inhibits MBNL protein binding</span></div><div class="casAuthors">Arambula, Jonathan F.; Ramisetty, Sreenivasa Rao; Baranger, Anne M.; Zimmerman, Steven C.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">106</span>
        (<span class="NLM_cas:issue">38</span>),
    <span class="NLM_cas:pages">16068-16073, S16068/1-S16068/39</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">This work describes the rational design, synthesis, and study of a ligand that selectively complexes CUG repeats in RNA (and CTG repeats in DNA) with high nanomolar affinity.  This sequence is considered a causative agent of myotonic dystrophy type 1 (DM1) because of its ability to sequester muscleblind-like (MBNL) proteins.  Ligand 1 was synthesized in two steps from com. available compds., and its binding to CTG and CUG repeats in oligonucleotides studied.  Isothermal titrn. calorimetry studies of 1 with various sequences showed a preference toward the T-T mismatch (Kd of 390 Â± 80 nM) with a 13-, 169-, and 85-fold redn. in affinity toward single C-C, A-A, and G-G mismatches, resp.  Binding and Job anal. of 1 to multiple CTG step sequences revealed high affinity binding to every other T-T mismatch with neg. cooperativity for proximal T-T mismatches.  The affinity of 1 for a (CUG)4 step provided a Kd of 430 nM with a binding stoichiometry of 1:1.  The preference for the U-U in RNA was maintained with a 6-, >143-, and >143-fold redn. in affinity toward single C-C, A-A, and G-G mismatches, resp.  Ligand 1 destabilized the complexes formed between MBNL1N and (CUG)4 and (CUG)12 with IC50 values of 52 Â± 20 Î¼IM and 46 Â± 7 Î¼M, resp., and Ki values of 6 Â± 2 Î¼M and 7 Â± 1 Î¼M, resp.  These values were only minimally altered by the addn. of competitor tRNA.  Ligand 1 does not destabilize the unrelated RNA-protein complexes the U1A-SL2 RNA complex and the Sex lethal-tra RNA complex.  Thus, ligand 1 selectively destabilizes the MBNL1N-poly(CUG) complex.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrjnqvChNuskbVg90H21EOLACvtfcHk0lhWk53jYvPVEw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXht1OjsbjO&md5=586a30b81ac235985c76e4fea5e9e34f</span></div><a href="/servlet/linkout?suffix=cit109&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0901824106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0901824106%26sid%3Dliteratum%253Aachs%26aulast%3DArambula%26aufirst%3DJ.%2BF.%26aulast%3DRamisetty%26aufirst%3DS.%2BR.%26aulast%3DBaranger%26aufirst%3DA.%2BM.%26aulast%3DZimmerman%26aufirst%3DS.%2BC.%26atitle%3DA%2520simple%2520ligand%2520that%2520selectively%2520targets%2520CUG%2520trinucleotide%2520repeats%2520and%2520inhibits%2520MBNL%2520protein%2520binding%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2009%26volume%3D106%26spage%3D16068%26epage%3D16073%26doi%3D10.1073%2Fpnas.0901824106" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref110"><div class="reference"><strong class="refLabel"><a href="#ref110" class="rightTabRefNumLink">110</a></strong><div class="NLM_citation" id="rightTab-cit110"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Siboni, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakamori, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Struck, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coonrod, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harriott, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cass, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanner, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berglund, J. A.</span></span> <span> </span><span class="NLM_article-title">Actinomycin D specifically reduces expanded CUG repeat RNA in myotonic mystrophy models</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">2386</span>â <span class="NLM_lpage">2394</span>, <span class="refDoi">Â DOI: 10.1016/j.celrep.2015.11.028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref110/cit110&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1016%2Fj.celrep.2015.11.028" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref110/cit110&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=26686629" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref110/cit110&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitVSksb3E" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2015&pages=2386-2394&author=R.+B.+Siboniauthor=M.+Nakamoriauthor=S.+D.+Wagnerauthor=A.+J.+Struckauthor=L.+A.+Coonrodauthor=S.+A.+Harriottauthor=D.+M.+Cassauthor=M.+K.+Tannerauthor=J.+A.+Berglund&title=Actinomycin+D+specifically+reduces+expanded+CUG+repeat+RNA+in+myotonic+mystrophy+models&doi=10.1016%2Fj.celrep.2015.11.028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit110R"><div class="casContent"><span class="casTitleNuber">110</span><div class="casTitle"><span class="NLM_cas:atitle">Actinomycin D Specifically Reduces Expanded CUG Repeat RNA in Myotonic Dystrophy Models</span></div><div class="casAuthors">Siboni, Ruth B.; Nakamori, Masayuki; Wagner, Stacey D.; Struck, Adam J.; Coonrod, Leslie A.; Harriott, Shanee A.; Cass, Daniel M.; Tanner, Matthew K.; Berglund, J. Andrew</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2386-2394</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Myotonic dystrophy type 1 (DM1) is an inherited disease characterized by the inability to relax contracted muscles.  Affected individuals carry large CTG expansions that are toxic when transcribed.  One possible treatment approach is to reduce or eliminate transcription of CTG repeats.  Actinomycin D (ActD) is a potent transcription inhibitor and FDA-approved chemotherapeutic that binds GC-rich DNA with high affinity.  Here, we report that ActD decreased CUG transcript levels in a dose-dependent manner in DM1 cell and mouse models at significantly lower concns. (nanomolar) compared to its use as a general transcription inhibitor or chemotherapeutic.  ActD also significantly reversed DM1-assocd. splicing defects in a DM1 mouse model, and did so within the currently approved human treatment range.  RNA-seq analyses showed that low concns. of ActD did not globally inhibit transcription in a DM1 mouse model.  These results indicate that transcription inhibition of CTG expansions is a promising treatment approach for DM1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZVoDzjglYN7Vg90H21EOLACvtfcHk0lhWk53jYvPVEw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitVSksb3E&md5=f9db4c8f69720951dffda03d58b81fa8</span></div><a href="/servlet/linkout?suffix=cit110&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2015.11.028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2015.11.028%26sid%3Dliteratum%253Aachs%26aulast%3DSiboni%26aufirst%3DR.%2BB.%26aulast%3DNakamori%26aufirst%3DM.%26aulast%3DWagner%26aufirst%3DS.%2BD.%26aulast%3DStruck%26aufirst%3DA.%2BJ.%26aulast%3DCoonrod%26aufirst%3DL.%2BA.%26aulast%3DHarriott%26aufirst%3DS.%2BA.%26aulast%3DCass%26aufirst%3DD.%2BM.%26aulast%3DTanner%26aufirst%3DM.%2BK.%26aulast%3DBerglund%26aufirst%3DJ.%2BA.%26atitle%3DActinomycin%2520D%2520specifically%2520reduces%2520expanded%2520CUG%2520repeat%2520RNA%2520in%2520myotonic%2520mystrophy%2520models%26jtitle%3DCell%2520Rep.%26date%3D2015%26volume%3D13%26spage%3D2386%26epage%3D2394%26doi%3D10.1016%2Fj.celrep.2015.11.028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref111"><div class="reference"><strong class="refLabel"><a href="#ref111" class="rightTabRefNumLink">111</a></strong><div class="NLM_citation" id="rightTab-cit111"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sletten, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertozzi, C. R.</span></span> <span> </span><span class="NLM_article-title">Bioorthogonal chemistry: Fishing for selectivity in a sea of functionality</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">6974</span>â <span class="NLM_lpage">6998</span>, <span class="refDoi">Â DOI: 10.1002/anie.200900942</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref111/cit111&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1002%2Fanie.200900942" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref111/cit111&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtV2ltrvK" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2009&pages=6974-6998&author=E.+M.+Slettenauthor=C.+R.+Bertozzi&title=Bioorthogonal+chemistry%3A+Fishing+for+selectivity+in+a+sea+of+functionality&doi=10.1002%2Fanie.200900942"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit111R"><div class="casContent"><span class="casTitleNuber">111</span><div class="casTitle"><span class="NLM_cas:atitle">Bioorthogonal Chemistry: Fishing for Selectivity in a Sea of Functionality</span></div><div class="casAuthors">Sletten, Ellen M.; Bertozzi, Carolyn R.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">38</span>),
    <span class="NLM_cas:pages">6974-6998</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  The study of biomols. in their native environments is a challenging task because of the vast complexity of cellular systems.  Technologies developed in the last few years for the selective modification of biol. species in living systems have yielded new insights into cellular processes.  Key to these new techniques are bioorthogonal chem. reactions, whose components must react rapidly and selectively with each other under physiol. conditions in the presence of the plethora of functionality necessary to sustain life.  Herein the authors describe the bioorthogonal chem. reactions developed to date and how they can be used to study biomols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJibJXpsfxW7Vg90H21EOLACvtfcHk0lgpwhF3678SlA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtV2ltrvK&md5=993199cb4deabc186848bae6fe81f605</span></div><a href="/servlet/linkout?suffix=cit111&amp;dbid=16384&amp;doi=10.1002%2Fanie.200900942&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.200900942%26sid%3Dliteratum%253Aachs%26aulast%3DSletten%26aufirst%3DE.%2BM.%26aulast%3DBertozzi%26aufirst%3DC.%2BR.%26atitle%3DBioorthogonal%2520chemistry%253A%2520Fishing%2520for%2520selectivity%2520in%2520a%2520sea%2520of%2520functionality%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2009%26volume%3D48%26spage%3D6974%26epage%3D6998%26doi%3D10.1002%2Fanie.200900942" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref112"><div class="reference"><strong class="refLabel"><a href="#ref112" class="rightTabRefNumLink">112</a></strong><div class="NLM_citation" id="rightTab-cit112"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Å eÄkute, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Devaraj, N. K.</span></span> <span> </span><span class="NLM_article-title">Live-cell imaging of cyclopropene tags with fluorogenic tetrazine cycloadditions</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">7476</span>â <span class="NLM_lpage">7479</span>, <span class="refDoi">Â DOI: 10.1002/anie.201202122</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref112/cit112&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1002%2Fanie.201202122" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref112/cit112&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC38Xos1egs74%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2012&pages=7476-7479&author=J.+Yangauthor=J.+%C5%A0e%C4%8Dkuteauthor=C.+M.+Coleauthor=N.+K.+Devaraj&title=Live-cell+imaging+of+cyclopropene+tags+with+fluorogenic+tetrazine+cycloadditions&doi=10.1002%2Fanie.201202122"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit112R"><div class="casContent"><span class="casTitleNuber">112</span><div class="casTitle"><span class="NLM_cas:atitle">Live-Cell Imaging of Cyclopropene Tags with Fluorogenic Tetrazine Cycloadditions</span></div><div class="casAuthors">Yang, Jun; Seckute, Jolita; Cole, Christian M.; Devaraj, Neal K.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">30</span>),
    <span class="NLM_cas:pages">7476-7479, S7476/1-S7476/26</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The authors demonstrated the applicability of methylcyclopropene tags as dienophiles for reaction with fluorogenic tetrazines.  Through systematic synthetic modifications, the stability, size, and reactivity of the cyclopropene scaffold were optimized.  Methylcyclopropene derivs. were developed that react rapidly with tetrazines while retaining their aq. stability and small size.  These cyclopropene handles elicited fluorescent responses from quenched tetrazine dyes and were suitable for cellular imaging applications, which was demonstrated by imaging cyclopropene phospholipids distributed in live human breast cancer cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpb3dQUpTbfPrVg90H21EOLACvtfcHk0lgpwhF3678SlA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xos1egs74%253D&md5=c60edddde8754df584b0205368be1e5c</span></div><a href="/servlet/linkout?suffix=cit112&amp;dbid=16384&amp;doi=10.1002%2Fanie.201202122&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201202122%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DJ.%26aulast%3D%25C5%25A0e%25C4%258Dkute%26aufirst%3DJ.%26aulast%3DCole%26aufirst%3DC.%2BM.%26aulast%3DDevaraj%26aufirst%3DN.%2BK.%26atitle%3DLive-cell%2520imaging%2520of%2520cyclopropene%2520tags%2520with%2520fluorogenic%2520tetrazine%2520cycloadditions%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2012%26volume%3D51%26spage%3D7476%26epage%3D7479%26doi%3D10.1002%2Fanie.201202122" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref113"><div class="reference"><strong class="refLabel"><a href="#ref113" class="rightTabRefNumLink">113</a></strong><div class="NLM_citation" id="rightTab-cit113"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Angelbello, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span> <span> </span><span class="NLM_article-title">A toxic RNA templates the synthesis of Its own fluorogenic inhibitor by using a bio-orthogonal tetrazine ligation in cells and tissues</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">1820</span>â <span class="NLM_lpage">1825</span>, <span class="refDoi">Â DOI: 10.1021/acschembio.0c00417</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.0c00417" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref113/cit113&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtFOnsL7I" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2020&pages=1820-1825&author=A.+J.+Angelbelloauthor=M.+D.+Disney&title=A+toxic+RNA+templates+the+synthesis+of+Its+own+fluorogenic+inhibitor+by+using+a+bio-orthogonal+tetrazine+ligation+in+cells+and+tissues&doi=10.1021%2Facschembio.0c00417"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit113R"><div class="casContent"><span class="casTitleNuber">113</span><div class="casTitle"><span class="NLM_cas:atitle">A Toxic RNA Templates the Synthesis of Its Own Fluorogenic Inhibitor by Using a Bio-orthogonal Tetrazine Ligation in Cells and Tissues</span></div><div class="casAuthors">Angelbello, Alicia J.; Disney, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1820-1825</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Expanded RNA repeats cause more than 30 incurable diseases.  One approach to mitigate their toxicity is by using small mols. that assemble into potent, oligomeric species upon binding to the disease-causing RNA in cells.  Herein, the authors show that the expanded repeat [r(CUG)exp] that causes myotonic dystrophy type 1 (DM1) catalyzes the in situ synthesis of its own inhibitor using an RNA-templated tetrazine ligation in DM1 patient-derived cells.  The compd. synthesized on-site improved DM1-assocd. defects at picomolar concns., enhancing potency by 10 000-fold, compared to its parent compds. that cannot undergo oligomerization.  A fluorogenic reaction is also described where r(CUG)exp templates the synthesis of its own imaging probe to enable visualization of the repeat in its native context in live cells and muscle tissue.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4aSzV_OLQLbVg90H21EOLACvtfcHk0lgpwhF3678SlA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtFOnsL7I&md5=2173daafb0e4280bc0f5e9bfb0222d65</span></div><a href="/servlet/linkout?suffix=cit113&amp;dbid=16384&amp;doi=10.1021%2Facschembio.0c00417&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.0c00417%26sid%3Dliteratum%253Aachs%26aulast%3DAngelbello%26aufirst%3DA.%2BJ.%26aulast%3DDisney%26aufirst%3DM.%2BD.%26atitle%3DA%2520toxic%2520RNA%2520templates%2520the%2520synthesis%2520of%2520Its%2520own%2520fluorogenic%2520inhibitor%2520by%2520using%2520a%2520bio-orthogonal%2520tetrazine%2520ligation%2520in%2520cells%2520and%2520tissues%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2020%26volume%3D15%26spage%3D1820%26epage%3D1825%26doi%3D10.1021%2Facschembio.0c00417" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref114"><div class="reference"><strong class="refLabel"><a href="#ref114" class="rightTabRefNumLink">114</a></strong><div class="NLM_citation" id="rightTab-cit114"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rzuczek, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colgan, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cameron, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furling, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yasuda, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span> <span> </span><span class="NLM_article-title">Precise small-molecule recognition of a toxic CUG RNA repeat expansion</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">188</span>â <span class="NLM_lpage">193</span>, <span class="refDoi">Â DOI: 10.1038/nchembio.2251</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref114/cit114&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1038%2Fnchembio.2251" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref114/cit114&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=27941760" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref114/cit114&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC28XitVSrurbM" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&pages=188-193&author=S.+G.+Rzuczekauthor=L.+A.+Colganauthor=Y.+Nakaiauthor=M.+D.+Cameronauthor=D.+Furlingauthor=R.+Yasudaauthor=M.+D.+Disney&title=Precise+small-molecule+recognition+of+a+toxic+CUG+RNA+repeat+expansion&doi=10.1038%2Fnchembio.2251"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit114R"><div class="casContent"><span class="casTitleNuber">114</span><div class="casTitle"><span class="NLM_cas:atitle">Precise small-molecule recognition of a toxic CUG RNA repeat expansion</span></div><div class="casAuthors">Rzuczek, Suzanne G.; Colgan, Lesley A.; Nakai, Yoshio; Cameron, Michael D.; Furling, Denis; Yasuda, Ryohei; Disney, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">188-193</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Excluding the ribosome and riboswitches, developing small mols. that selectively target RNA is a longstanding problem in chem. biol.  A typical cellular RNA is difficult to target because it has little tertiary, but abundant secondary structure.  We designed allele-selective compds. that target such an RNA, the toxic noncoding repeat expansion (r(CUG)exp) that causes myotonic dystrophy type 1 (DM1).  We developed several strategies to generate allele-selective small mols., including non-covalent binding, covalent binding, cleavage and on-site probe synthesis.  Covalent binding and cleavage enabled target profiling in cells derived from individuals with DM1, showing precise recognition of r(CUG)exp.  In the on-site probe synthesis approach, small mols. bound adjacent sites in r(CUG)exp and reacted to afford picomolar inhibitors via a proximity-based click reaction only in DM1-affected cells.  We expanded this approach to image r(CUG)exp in its natural context.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrC5kTu0Wkj9bVg90H21EOLACvtfcHk0lgpwhF3678SlA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitVSrurbM&md5=bb30d79fe6e8d513a4956d13ee20f69c</span></div><a href="/servlet/linkout?suffix=cit114&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2251&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2251%26sid%3Dliteratum%253Aachs%26aulast%3DRzuczek%26aufirst%3DS.%2BG.%26aulast%3DColgan%26aufirst%3DL.%2BA.%26aulast%3DNakai%26aufirst%3DY.%26aulast%3DCameron%26aufirst%3DM.%2BD.%26aulast%3DFurling%26aufirst%3DD.%26aulast%3DYasuda%26aufirst%3DR.%26aulast%3DDisney%26aufirst%3DM.%2BD.%26atitle%3DPrecise%2520small-molecule%2520recognition%2520of%2520a%2520toxic%2520CUG%2520RNA%2520repeat%2520expansion%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2017%26volume%3D13%26spage%3D188%26epage%3D193%26doi%3D10.1038%2Fnchembio.2251" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref115"><div class="reference"><strong class="refLabel"><a href="#ref115" class="rightTabRefNumLink">115</a></strong><div class="NLM_citation" id="rightTab-cit115"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liquori, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ricker, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moseley, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobsen, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kress, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naylor, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Day, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ranum, L. P.</span></span> <span> </span><span class="NLM_article-title">Myotonic dystrophy type 2 caused by a CCTG expansion in intron 1 of ZNF9</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>293</i></span>,  <span class="NLM_fpage">864</span>â <span class="NLM_lpage">867</span>, <span class="refDoi">Â DOI: 10.1126/science.1062125</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref115/cit115&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1126%2Fscience.1062125" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref115/cit115&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=11486088" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref115/cit115&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BD3MXlvVKktbc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=293&publication_year=2001&pages=864-867&author=C.+L.+Liquoriauthor=K.+Rickerauthor=M.+L.+Moseleyauthor=J.+F.+Jacobsenauthor=W.+Kressauthor=S.+L.+Naylorauthor=J.+W.+Dayauthor=L.+P.+Ranum&title=Myotonic+dystrophy+type+2+caused+by+a+CCTG+expansion+in+intron+1+of+ZNF9&doi=10.1126%2Fscience.1062125"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit115R"><div class="casContent"><span class="casTitleNuber">115</span><div class="casTitle"><span class="NLM_cas:atitle">Myotonic dystrophy type 2 caused by a CCTG expansion in intron 1 of ZNF9</span></div><div class="casAuthors">Liquori, Christina L.; Ricker, Kenneth; Moseley, Melinda L.; Jacobsen, Jennifer F.; Kress, Wolfram; Naylor, Susan L.; Day, John W.; Ranum, Laura P. W.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">293</span>
        (<span class="NLM_cas:issue">5531</span>),
    <span class="NLM_cas:pages">864-867</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Myotonic dystrophy (DM), the most common form of muscular dystrophy in adults, can be caused by a mutation on either chromosome 19q13 (DM1) or 3q21 (DMZ/PROMM).  DM1 is caused by a CTG expansion in the 3' untranslated region of the dystrophia myotonica-protein kinase gene (DMPK).  Several mechanisms have been invoked to explain how this mutation, which does not alter the protein-coding portion of a gene, causes the specific constellation of clin. features characteristic of DM.  We now report that DMZ is caused by a CCTG expansion (mean â¼5000 repeats) located in intron 1 of the zinc finger protein 9 (ZNF9) gene.  Parallels between these mutations indicate that microsatellite expansions in RNA can be pathogenic and cause the multisystemic features of DM1 and DM2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoeKIAHv1r8_bVg90H21EOLACvtfcHk0liLPeE1rNkayw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXlvVKktbc%253D&md5=812d852f480461b722162114fcb906f2</span></div><a href="/servlet/linkout?suffix=cit115&amp;dbid=16384&amp;doi=10.1126%2Fscience.1062125&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1062125%26sid%3Dliteratum%253Aachs%26aulast%3DLiquori%26aufirst%3DC.%2BL.%26aulast%3DRicker%26aufirst%3DK.%26aulast%3DMoseley%26aufirst%3DM.%2BL.%26aulast%3DJacobsen%26aufirst%3DJ.%2BF.%26aulast%3DKress%26aufirst%3DW.%26aulast%3DNaylor%26aufirst%3DS.%2BL.%26aulast%3DDay%26aufirst%3DJ.%2BW.%26aulast%3DRanum%26aufirst%3DL.%2BP.%26atitle%3DMyotonic%2520dystrophy%2520type%25202%2520caused%2520by%2520a%2520CCTG%2520expansion%2520in%2520intron%25201%2520of%2520ZNF9%26jtitle%3DScience%26date%3D2001%26volume%3D293%26spage%3D864%26epage%3D867%26doi%3D10.1126%2Fscience.1062125" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref116"><div class="reference"><strong class="refLabel"><a href="#ref116" class="rightTabRefNumLink">116</a></strong><div class="NLM_citation" id="rightTab-cit116"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rzuczek, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span> <span> </span><span class="NLM_article-title">A toxic RNA catalyzes the <i>in cellulo</i> synthesis of its own inhibitor</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">10956</span>â <span class="NLM_lpage">10959</span>, <span class="refDoi">Â DOI: 10.1002/anie.201406465</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref116/cit116&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1002%2Fanie.201406465" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref116/cit116&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVWqtbzE" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2014&pages=10956-10959&author=S.+G.+Rzuczekauthor=H.+Parkauthor=M.+D.+Disney&title=A+toxic+RNA+catalyzes+the+in+cellulo+synthesis+of+its+own+inhibitor&doi=10.1002%2Fanie.201406465"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit116R"><div class="casContent"><span class="casTitleNuber">116</span><div class="casTitle"><span class="NLM_cas:atitle">A Toxic RNA Catalyzes the In Cellulo Synthesis of Its Own Inhibitor</span></div><div class="casAuthors">Rzuczek, Suzanne G.; Park, HaJeung; Disney, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">41</span>),
    <span class="NLM_cas:pages">10956-10959</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Potent modulators of RNA function can be assembled in cellulo by using the cell as a reaction vessel and a disease-causing RNA as a catalyst.  When designing small mol. effectors of function, a balance between permeability and potency must be struck.  Low mol. wt. compds. are more permeable, whereas higher mol. wt. compds. are more potent.  The advantages of both types of compds. could be synergized if low mol. wt. mols. could be transformed into potent, multivalent ligands by a reaction that is catalyzed by binding to a target in cells expressing a genetic defect.  It was shown that this approach is indeed viable in cellulo.  Small mol. modules with precisely positioned alkyne and azide moieties bind adjacent internal loops in r(CCUG)exp, the causative agent of myotonic dystrophy type 2 (DM2), and are transformed into oligomeric, potent inhibitors of DM2 RNA dysfunction by a Huisgen 1,3-dipolar cycloaddn. (HDCR) reaction, a variant of click chem.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqOoWZFnjUHn7Vg90H21EOLACvtfcHk0liLPeE1rNkayw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVWqtbzE&md5=a5fc7e597bed882f6bb76bb99185420b</span></div><a href="/servlet/linkout?suffix=cit116&amp;dbid=16384&amp;doi=10.1002%2Fanie.201406465&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201406465%26sid%3Dliteratum%253Aachs%26aulast%3DRzuczek%26aufirst%3DS.%2BG.%26aulast%3DPark%26aufirst%3DH.%26aulast%3DDisney%26aufirst%3DM.%2BD.%26atitle%3DA%2520toxic%2520RNA%2520catalyzes%2520the%2520in%2520cellulo%2520synthesis%2520of%2520its%2520own%2520inhibitor%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2014%26volume%3D53%26spage%3D10956%26epage%3D10959%26doi%3D10.1002%2Fanie.201406465" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref117"><div class="reference"><strong class="refLabel"><a href="#ref117" class="rightTabRefNumLink">117</a></strong><div class="NLM_citation" id="rightTab-cit117"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ashizawa, T.</span></span> <span> </span><span class="NLM_article-title">Recent progress in spinocerebellar ataxia type-10 (SCA10)</span>. <i>Cerebellum</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">37</span>â <span class="NLM_lpage">42</span>, <span class="refDoi">Â DOI: 10.1080/14734220510007897</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref117/cit117&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1080%2F14734220510007897" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref117/cit117&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=15895557" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref117/cit117&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BD2MXkvVemur8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2005&pages=37-42&author=X.+Linauthor=T.+Ashizawa&title=Recent+progress+in+spinocerebellar+ataxia+type-10+%28SCA10%29&doi=10.1080%2F14734220510007897"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit117R"><div class="casContent"><span class="casTitleNuber">117</span><div class="casTitle"><span class="NLM_cas:atitle">Recent progress in spinocerebellar ataxia type-10 (SCA10)</span></div><div class="casAuthors">Lin, Xi; Ashizawa, Tetsuo</div><div class="citationInfo"><span class="NLM_cas:title">Cerebellum</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">37-42</span>CODEN:
                <span class="NLM_cas:coden">CERECF</span>;
        ISSN:<span class="NLM_cas:issn">1473-4222</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">A review.  Spinocerebellar ataxia type 10 (SCA10) is a dominantly inherited ataxia caused by expansion of ATTCT pentanucleotide repeat in intron 9 of a novel gene, E46L, on chromosome 22q13.3.  SCA10 is a complex neurodegenerative condition.  Initial studies characterized SCA10 as pure cerebellar ataxia assocd. with seizures.  Recent identification of new SCA10 families revealed more diverse phenotypes, including polyneuropathy, pyramidal signs, cognitive and neuropsychiatric impairment.  Moreover, several families manifest with ataxia without seizures.  Thus a complete clin. spectrum is emerging.  Progress has also been made in understanding the mol. and genetic mechanisms of pathogenesis.  The length of expanded ATTCT repeats is variable in different tissues and highly unstable during paternal transmission, revealing complex genetic and pathogenetic processes.  Under torsional stress, ATTCT repeats form unpaired DNA structure and may serve as an erroneous DNA replication origin, potentially contributing to repeat instability and aberrant cell cycle entry.  E46L is a cytoplasmic protein with unknown function.  Reduced expression of E46L in primary neuronal cultures from cerebellum and cortex by small interfering RNAs (siRNAs) caused increased apoptosis, raising the possibility that reduced expression of E46L might also play an important role in SCA10 pathogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGppTtHjEWa1irVg90H21EOLACvtfcHk0liLPeE1rNkayw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXkvVemur8%253D&md5=563e6b560bddbd9b9ba34abc614d8084</span></div><a href="/servlet/linkout?suffix=cit117&amp;dbid=16384&amp;doi=10.1080%2F14734220510007897&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F14734220510007897%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DX.%26aulast%3DAshizawa%26aufirst%3DT.%26atitle%3DRecent%2520progress%2520in%2520spinocerebellar%2520ataxia%2520type-10%2520%2528SCA10%2529%26jtitle%3DCerebellum%26date%3D2005%26volume%3D4%26spage%3D37%26epage%3D42%26doi%3D10.1080%2F14734220510007897" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref118"><div class="reference"><strong class="refLabel"><a href="#ref118" class="rightTabRefNumLink">118</a></strong><div class="NLM_citation" id="rightTab-cit118"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, W. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Southern, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sarkar, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span> <span> </span><span class="NLM_article-title">Design of a bioactive small molecule that targets r(AUUCU) repeats in spinocerebellar ataxia 10</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">11647</span>, <span class="refDoi">Â DOI: 10.1038/ncomms11647</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref118/cit118&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1038%2Fncomms11647" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref118/cit118&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=27248057" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref118/cit118&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC28XptVykt7w%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=11647&author=W.+Y.+Yangauthor=R.+Gaoauthor=M.+Southernauthor=P.+S.+Sarkarauthor=M.+D.+Disney&title=Design+of+a+bioactive+small+molecule+that+targets+r%28AUUCU%29+repeats+in+spinocerebellar+ataxia+10&doi=10.1038%2Fncomms11647"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit118R"><div class="casContent"><span class="casTitleNuber">118</span><div class="casTitle"><span class="NLM_cas:atitle">Design of a bioactive small molecule that targets r(AUUCU) repeats in spinocerebellar ataxia 10</span></div><div class="casAuthors">Yang, Wang-Yong; Gao, Rui; Southern, Mark; Sarkar, Partha S.; Disney, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">11647</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">RNA is an important target for chem. probes of function and lead therapeutics; however, it is difficult to target with small mols.  One approach to tackle this problem is to identify compds. that target RNA structures and utilize them to multivalently target RNA.  Here we show that small mols. can be identified to selectively bind RNA base pairs by probing a library of RNA-focused small mols.  A small mol. that selectively binds AU base pairs informed design of a dimeric compd. (2AU-2) that targets the pathogenic RNA, expanded r(AUUCU) repeats, that causes spinocerebellar ataxia type 10 (SCA10) in patient-derived cells.  Indeed, 2AU-2 (50 nM) ameliorates various aspects of SCA10 pathol. including improvement of mitochondrial dysfunction, reduced activation of caspase 3, and redn. of nuclear foci.  These studies provide a first-in-class chem. probe to study SCA10 RNA toxicity and potentially define broadly applicable compds. targeting RNA AU base pairs in cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGorN7W1vcvMzbVg90H21EOLACvtfcHk0lgy7BH0j6ndCQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XptVykt7w%253D&md5=60152483dd13fcaa808a7bfb9ecdcb92</span></div><a href="/servlet/linkout?suffix=cit118&amp;dbid=16384&amp;doi=10.1038%2Fncomms11647&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms11647%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DW.%2BY.%26aulast%3DGao%26aufirst%3DR.%26aulast%3DSouthern%26aufirst%3DM.%26aulast%3DSarkar%26aufirst%3DP.%2BS.%26aulast%3DDisney%26aufirst%3DM.%2BD.%26atitle%3DDesign%2520of%2520a%2520bioactive%2520small%2520molecule%2520that%2520targets%2520r%2528AUUCU%2529%2520repeats%2520in%2520spinocerebellar%2520ataxia%252010%26jtitle%3DNat.%2520Commun.%26date%3D2016%26volume%3D7%26spage%3D11647%26doi%3D10.1038%2Fncomms11647" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref119"><div class="reference"><strong class="refLabel"><a href="#ref119" class="rightTabRefNumLink">119</a></strong><div class="NLM_citation" id="rightTab-cit119"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bannwarth, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gatignol, A.</span></span> <span> </span><span class="NLM_article-title">HIV-1 TAR RNA: The target of molecular interactions between the virus and its host</span>. <i>Curr. HIV Res.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">61</span>â <span class="NLM_lpage">71</span>, <span class="refDoi">Â DOI: 10.2174/1570162052772924</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref119/cit119&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.2174%2F1570162052772924" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref119/cit119&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=15638724" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref119/cit119&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhslChsA%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2005&pages=61-71&author=S.+Bannwarthauthor=A.+Gatignol&title=HIV-1+TAR+RNA%3A+The+target+of+molecular+interactions+between+the+virus+and+its+host&doi=10.2174%2F1570162052772924"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit119R"><div class="casContent"><span class="casTitleNuber">119</span><div class="casTitle"><span class="NLM_cas:atitle">HIV-1 TAR RNA: The target of molecular interactions between the virus and its host</span></div><div class="casAuthors">Bannwarth, Sylvie; Gatignol, Anne</div><div class="citationInfo"><span class="NLM_cas:title">Current HIV Research</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">61-71</span>CODEN:
                <span class="NLM_cas:coden">CHRUBF</span>;
        ISSN:<span class="NLM_cas:issn">1570-162X</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  HIV-1 TAR RNA is the binding site of the viral protein Tat, the trans-activator of the HIV-1 LTR.  It is present at the 5' end of all HIV-1 spliced and unspliced mRNAs in the nucleus as well as in the cytoplasm.  It has a highly folded stem-bulge-loop structure, which also binds cellular proteins to form ribonucleoprotein complexes.  The Tat-Cyclin T1-CDK9 complex is the main component in the trans-activation of HIV-1 and its affinity for TAR is regulated through Tat acetylation by histone acetyl transferases.  Recent studies show that this complex is able to recruit other cellular partners to mediate efficient transcriptional elongation.  TRBP, PKR and La bind directly to the TAR RNA structure and influence translation of HIV-1 in either pos. or neg. manners.  Some mutations in TAR RNA severely impair HIV-1 trans-activation, translation and viral prodn., showing its functional importance.  The overexpression or suppression of several TAR RNA-binding proteins has a strong impact on viral replication pointing out their major role in the viral life cycle.  TAR RNA has been the target of drug development to inhibit viral replication.  Recent data using small mols. or RNA-based technologies show that acting on the TAR RNA or on its viral and cellular binding factors effectively decreases virion prodn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVzliRpSqkgLVg90H21EOLACvtfcHk0lgy7BH0j6ndCQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhslChsA%253D%253D&md5=f6fdda283b3e8da8dca021cff8601d58</span></div><a href="/servlet/linkout?suffix=cit119&amp;dbid=16384&amp;doi=10.2174%2F1570162052772924&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1570162052772924%26sid%3Dliteratum%253Aachs%26aulast%3DBannwarth%26aufirst%3DS.%26aulast%3DGatignol%26aufirst%3DA.%26atitle%3DHIV-1%2520TAR%2520RNA%253A%2520The%2520target%2520of%2520molecular%2520interactions%2520between%2520the%2520virus%2520and%2520its%2520host%26jtitle%3DCurr.%2520HIV%2520Res.%26date%3D2005%26volume%3D3%26spage%3D61%26epage%3D71%26doi%3D10.2174%2F1570162052772924" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref120"><div class="reference"><strong class="refLabel"><a href="#ref120" class="rightTabRefNumLink">120</a></strong><div class="NLM_citation" id="rightTab-cit120"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Joly, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mata, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eldin, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Briant, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fontaine-Vive, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duca, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benhida, R.</span></span> <span> </span><span class="NLM_article-title">Artificial nucleobase-amino acid conjugates: A new class of TAR RNA binding agents</span>. <i>Chem. - Eur. J.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">2071</span>â <span class="NLM_lpage">2079</span>, <span class="refDoi">Â DOI: 10.1002/chem.201303664</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref120/cit120&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1002%2Fchem.201303664" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref120/cit120&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=24431237" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref120/cit120&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC2cXotF2nug%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2014&pages=2071-2079&author=J.+P.+Jolyauthor=G.+Mataauthor=P.+Eldinauthor=L.+Briantauthor=F.+Fontaine-Viveauthor=M.+Ducaauthor=R.+Benhida&title=Artificial+nucleobase-amino+acid+conjugates%3A+A+new+class+of+TAR+RNA+binding+agents&doi=10.1002%2Fchem.201303664"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit120R"><div class="casContent"><span class="casTitleNuber">120</span><div class="casTitle"><span class="NLM_cas:atitle">Artificial Nucleobase-Amino Acid Conjugates: A New Class of TAR RNA Binding Agents</span></div><div class="casAuthors">Joly, Jean-Patrick; Mata, Guillaume; Eldin, Patrick; Briant, Laurence; Fontaine-Vive, Fabien; Duca, Maria; Benhida, Rachid</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry - A European Journal</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2071-2079</span>CODEN:
                <span class="NLM_cas:coden">CEUJED</span>;
        ISSN:<span class="NLM_cas:issn">0947-6539</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The human immunodeficiency virus type-1 (HIV-1) Tat protein [protein TBP-1 Tat-binding protein-1] stimulates transcriptional elongation.  Tat is involved in the transcription machinery by binding to the transactivation response region (TAR) RNA stem-loop structure, which is encoded by the 5' leader sequence found in all HIV-1 mRNA.  Herein, the authors report the rational design, synthesis, and in vitro evaluation of new RNA binding agents that were conceived in order to bind strongly and selectively to the stem-loop structure of TAR RNA and, thus, inhibit the Tat/TAR interaction.  It was demonstrated that the conjugation of modified nucleobases, able to interact specifically with an RNA base pair and various amino acids allows these motifs to bind the target RNA selectively and in a cooperative manner that leads to the inhibition of viral replication in HIV-infected cells.  The cooperative binding properties of the above-mentioned conjugates toward HIV-TAR-RNA was evaluated.  The title compds. thus formed included a nucleobase-amino-acid conjugate (analog) (I) and related substances.  The synthesis of the target compds. was achieved using a nucleobase analog N-[4-(3-aminophenyl)-2-thiazolyl]acetamide and N2-[(1,1-dimethylethoxy)carbonyl]-L-lysine, N2-[(phenylmethoxy)carbonyl]-L-arginine, N-[(1,1-dimethylethoxy)carbonyl]-L-histidine, N-[(1,1-dimethylethoxy)carbonyl]-L-phenylalanine, O-(1,1-dimethylethyl)-N-[(9H-fluoren-9-ylmethoxy)carbonyl]-L-tyrosine, tryptophan, alanine, threonine, glycine derivs., (amino)butanoic acid, (amino)hexanoic acid as starting materials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOZH87veaXkrVg90H21EOLACvtfcHk0lgy7BH0j6ndCQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXotF2nug%253D%253D&md5=07a48fa5218ab41d3ebce37ccbfa0b87</span></div><a href="/servlet/linkout?suffix=cit120&amp;dbid=16384&amp;doi=10.1002%2Fchem.201303664&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fchem.201303664%26sid%3Dliteratum%253Aachs%26aulast%3DJoly%26aufirst%3DJ.%2BP.%26aulast%3DMata%26aufirst%3DG.%26aulast%3DEldin%26aufirst%3DP.%26aulast%3DBriant%26aufirst%3DL.%26aulast%3DFontaine-Vive%26aufirst%3DF.%26aulast%3DDuca%26aufirst%3DM.%26aulast%3DBenhida%26aufirst%3DR.%26atitle%3DArtificial%2520nucleobase-amino%2520acid%2520conjugates%253A%2520A%2520new%2520class%2520of%2520TAR%2520RNA%2520binding%2520agents%26jtitle%3DChem.%2520-%2520Eur.%2520J.%26date%3D2014%26volume%3D20%26spage%3D2071%26epage%3D2079%26doi%3D10.1002%2Fchem.201303664" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref121"><div class="reference"><strong class="refLabel"><a href="#ref121" class="rightTabRefNumLink">121</a></strong><div class="NLM_citation" id="rightTab-cit121"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, W. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuhrmann, C. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frankel, A. D.</span></span> <span> </span><span class="NLM_article-title">Recognition of nucleic acid bases and base-pairs by hydrogen bonding to amino acid side-chains</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>327</i></span>,  <span class="NLM_fpage">781</span>â <span class="NLM_lpage">796</span>, <span class="refDoi">Â DOI: 10.1016/S0022-2836(03)00091-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref121/cit121&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1016%2FS0022-2836%2803%2900091-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref121/cit121&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=12654263" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref121/cit121&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BD3sXitlWgs7g%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=327&publication_year=2003&pages=781-796&author=A.+C.+Chengauthor=W.+W.+Chenauthor=C.+N.+Fuhrmannauthor=A.+D.+Frankel&title=Recognition+of+nucleic+acid+bases+and+base-pairs+by+hydrogen+bonding+to+amino+acid+side-chains&doi=10.1016%2FS0022-2836%2803%2900091-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit121R"><div class="casContent"><span class="casTitleNuber">121</span><div class="casTitle"><span class="NLM_cas:atitle">Recognition of Nucleic Acid Bases and Base-pairs by Hydrogen Bonding to Amino Acid Side-chains</span></div><div class="casAuthors">Cheng, Alan C.; Chen, William W.; Fuhrmann, Cynthia N.; Frankel, Alan D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">327</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">781-796</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">Sequence-specific protein-nucleic acid recognition is detd., in part, by hydrogen bonding interactions between amino acid side-chains and nucleotide bases.  To examine the repertoire of possible interactions, we have calcd. geometrically plausible arrangements in which amino acids hydrogen bond to unpaired bases, such as those found in RNA bulges and loops, or to the 53 possible RNA base-pairs.  We find 32 possible interactions that involve two or more hydrogen bonds to the six unpaired bases (including protonated A and C), 17 of which have been obsd.  We find 186 "spanning" interactions to base-pairs in which the amino acid hydrogen bonds to both bases, in principle allowing particular base-pairs to be selectively targeted, and nine of these have been obsd.  Four calcd. interactions span the Watson-Crick pairs and 15 span the G:U wobble pair, including two interesting arrangements with three hydrogen bonds to the Arg guanidinum group that have not yet been obsd.  The inherent donor-acceptor arrangements of the bases support many possible interactions to Asn (or Gln) and Ser (or Thr or Tyr), few interactions to Asp (or Glu) even though several already have been obsd., and interactions to U (or T) only if the base is in an unpaired context, as also obsd. in several cases.  This study highlights how complementary arrangements of donors and acceptors can contribute to base-specific recognition of RNA, predicts interactions not yet obsd., and provides tools to analyze proposed contacts or design novel interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7NXoYEVrOmrVg90H21EOLACvtfcHk0lgxf0OK4JM5Kg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXitlWgs7g%253D&md5=4732411ddbe7ebafe7500776655c87f0</span></div><a href="/servlet/linkout?suffix=cit121&amp;dbid=16384&amp;doi=10.1016%2FS0022-2836%2803%2900091-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0022-2836%252803%252900091-3%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DA.%2BC.%26aulast%3DChen%26aufirst%3DW.%2BW.%26aulast%3DFuhrmann%26aufirst%3DC.%2BN.%26aulast%3DFrankel%26aufirst%3DA.%2BD.%26atitle%3DRecognition%2520of%2520nucleic%2520acid%2520bases%2520and%2520base-pairs%2520by%2520hydrogen%2520bonding%2520to%2520amino%2520acid%2520side-chains%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2003%26volume%3D327%26spage%3D781%26epage%3D796%26doi%3D10.1016%2FS0022-2836%2803%2900091-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref122"><div class="reference"><strong class="refLabel"><a href="#ref122" class="rightTabRefNumLink">122</a></strong><div class="NLM_citation" id="rightTab-cit122"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Paul, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dutta, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paul, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dash, J.</span></span> <span> </span><span class="NLM_article-title">Target-directed azide-alkyne cycloaddition for assembling HIV-1 TAR RNA binding ligands</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">12407</span>â <span class="NLM_lpage">12411</span>, <span class="refDoi">Â DOI: 10.1002/anie.202003461</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref122/cit122&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1002%2Fanie.202003461" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref122/cit122&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtVaitb%252FO" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2020&pages=12407-12411&author=R.+Paulauthor=D.+Duttaauthor=R.+Paulauthor=J.+Dash&title=Target-directed+azide-alkyne+cycloaddition+for+assembling+HIV-1+TAR+RNA+binding+ligands&doi=10.1002%2Fanie.202003461"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit122R"><div class="casContent"><span class="casTitleNuber">122</span><div class="casTitle"><span class="NLM_cas:atitle">Target-directed azide-alkyne cycloaddition for assembling HIV-1 TAR RNA binding ligands</span></div><div class="casAuthors">Paul, Rakesh; Dutta, Debasish; Paul, Raj; Dash, Jyotirmayee</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">30</span>),
    <span class="NLM_cas:pages">12407-12411</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The highly conserved HIV-1 transactivation response element (TAR) binds to the trans-activator protein Tat and facilitates viral replication in its latent state.  The inhibition of Tat-TAR interactions by selectively targeting TAR RNA has been used as a strategy to develop potent antiviral agents.  Therefore, HIV-1 TAR RNA represents a paradigmatic system for therapeutic intervention.  Herein, we have employed biotin-tagged TAR RNA to assemble its own ligands from a pool of reactive azide and alkyne building blocks.  To identify the binding sites and selectivity of the ligands, the in situ cycloaddn. has been further performed using control nucleotide (TAR DNA and TAR RNA without bulge) templates.  The hit triazole-linked thiazole peptidomimetic products have been isolated from the biotin-tagged target templates using streptavidin beads.  The major triazole lead generated by the TAR RNA presumably binds in the bulge region, shows specificity for TAR RNA over TAR DNA, and inhibits Tat-TAR interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqygRarHJrIibVg90H21EOLACvtfcHk0lgxf0OK4JM5Kg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtVaitb%252FO&md5=6e2a48b9291957e5f0e014e25344d43b</span></div><a href="/servlet/linkout?suffix=cit122&amp;dbid=16384&amp;doi=10.1002%2Fanie.202003461&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.202003461%26sid%3Dliteratum%253Aachs%26aulast%3DPaul%26aufirst%3DR.%26aulast%3DDutta%26aufirst%3DD.%26aulast%3DPaul%26aufirst%3DR.%26aulast%3DDash%26aufirst%3DJ.%26atitle%3DTarget-directed%2520azide-alkyne%2520cycloaddition%2520for%2520assembling%2520HIV-1%2520TAR%2520RNA%2520binding%2520ligands%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2020%26volume%3D59%26spage%3D12407%26epage%3D12411%26doi%3D10.1002%2Fanie.202003461" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref123"><div class="reference"><strong class="refLabel"><a href="#ref123" class="rightTabRefNumLink">123</a></strong><div class="NLM_citation" id="rightTab-cit123"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stelzer, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frank, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kratz, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swanson, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gonzalez-Hernandez, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andricioaei, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Markovitz, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Hashim, H. M.</span></span> <span> </span><span class="NLM_article-title">Discovery of selective bioactive small molecules by targeting an RNA dynamic ensemble</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">553</span>â <span class="NLM_lpage">559</span>, <span class="refDoi">Â DOI: 10.1038/nchembio.596</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref123/cit123&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1038%2Fnchembio.596" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref123/cit123&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=21706033" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref123/cit123&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC3MXotV2htbw%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=553-559&author=A.+C.+Stelzerauthor=A.+T.+Frankauthor=J.+D.+Kratzauthor=M.+D.+Swansonauthor=M.+J.+Gonzalez-Hernandezauthor=J.+Leeauthor=I.+Andricioaeiauthor=D.+M.+Markovitzauthor=H.+M.+Al-Hashim&title=Discovery+of+selective+bioactive+small+molecules+by+targeting+an+RNA+dynamic+ensemble&doi=10.1038%2Fnchembio.596"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit123R"><div class="casContent"><span class="casTitleNuber">123</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of selective bioactive small molecules by targeting an RNA dynamic ensemble</span></div><div class="casAuthors">Stelzer, Andrew C.; Frank, Aaron T.; Kratz, Jeremy D.; Swanson, Michael D.; Gonzalez-Hernandez, Marta J.; Lee, Janghyun; Andricioaei, Ioan; Markovitz, David M.; Al-Hashimi, Hashim M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">553-559</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Current approaches used to identify protein-binding small mols. are not suited for identifying small mols. that can bind emerging RNA drug targets.  By docking small mols. onto an RNA dynamic ensemble constructed by combining NMR spectroscopy and computational mol. dynamics, the authors virtually screened small mols. that target the entire structure landscape of the transactivation response element (TAR) from HIV type 1 (HIV-1).  The authors quant. predict binding energies for small mols. that bind different RNA conformations and report the de novo discovery of six compds. that bind TAR with high affinity and inhibit its interaction with a Tat peptide in vitro (Ki values of 710 nM-169 Î¼M).  One compd. binds HIV-1 TAR with marked selectivity and inhibits Tat-mediated activation of the HIV-1 long terminal repeat by 81% in T-cell lines and HIV replication in an HIV-1 indicator cell line (IC50 â¼23.1 Î¼M).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoQ1jtoA-Z8RrVg90H21EOLACvtfcHk0lgxf0OK4JM5Kg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXotV2htbw%253D&md5=cd04f57b1fececc2f4aa678fe3f1f3f2</span></div><a href="/servlet/linkout?suffix=cit123&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.596&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.596%26sid%3Dliteratum%253Aachs%26aulast%3DStelzer%26aufirst%3DA.%2BC.%26aulast%3DFrank%26aufirst%3DA.%2BT.%26aulast%3DKratz%26aufirst%3DJ.%2BD.%26aulast%3DSwanson%26aufirst%3DM.%2BD.%26aulast%3DGonzalez-Hernandez%26aufirst%3DM.%2BJ.%26aulast%3DLee%26aufirst%3DJ.%26aulast%3DAndricioaei%26aufirst%3DI.%26aulast%3DMarkovitz%26aufirst%3DD.%2BM.%26aulast%3DAl-Hashim%26aufirst%3DH.%2BM.%26atitle%3DDiscovery%2520of%2520selective%2520bioactive%2520small%2520molecules%2520by%2520targeting%2520an%2520RNA%2520dynamic%2520ensemble%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2011%26volume%3D7%26spage%3D553%26epage%3D559%26doi%3D10.1038%2Fnchembio.596" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref124"><div class="reference"><strong class="refLabel"><a href="#ref124" class="rightTabRefNumLink">124</a></strong><div class="NLM_citation" id="rightTab-cit124"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Serganov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nudler, E.</span></span> <span> </span><span class="NLM_article-title">A decade of riboswitches</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>152</i></span>,  <span class="NLM_fpage">17</span>â <span class="NLM_lpage">24</span>, <span class="refDoi">Â DOI: 10.1016/j.cell.2012.12.024</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref124/cit124&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1016%2Fj.cell.2012.12.024" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref124/cit124&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=23332744" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref124/cit124&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1Oluro%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=152&publication_year=2013&pages=17-24&author=A.+Serganovauthor=E.+Nudler&title=A+decade+of+riboswitches&doi=10.1016%2Fj.cell.2012.12.024"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit124R"><div class="casContent"><span class="casTitleNuber">124</span><div class="casTitle"><span class="NLM_cas:atitle">A Decade of Riboswitches</span></div><div class="casAuthors">Serganov, Alexander; Nudler, Evgeny</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">152</span>
        (<span class="NLM_cas:issue">1-2</span>),
    <span class="NLM_cas:pages">17-24</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Riboswitches were discovered in 2002 in bacteria as RNA-based intracellular sensors of vitamin derivs.  During the last decade, naturally occurring RNA sensor elements have been found to bind a range of small metabolites and ions and to exert regulatory control of transcription, translation, splicing, and RNA stability.  Extensive biochem., structural, and genetic studies have established the basic principles underpinning riboswitch function in all three kingdoms of life with implications for developing antibiotics, designing new mol. sensors, and integrating riboswitches into synthetic circuits.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2Ru8nOcVcrbVg90H21EOLACvtfcHk0lgxf0OK4JM5Kg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1Oluro%253D&md5=c8bb59e0772d4ec0b9dbaf163d73ae1b</span></div><a href="/servlet/linkout?suffix=cit124&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2012.12.024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2012.12.024%26sid%3Dliteratum%253Aachs%26aulast%3DSerganov%26aufirst%3DA.%26aulast%3DNudler%26aufirst%3DE.%26atitle%3DA%2520decade%2520of%2520riboswitches%26jtitle%3DCell%26date%3D2013%26volume%3D152%26spage%3D17%26epage%3D24%26doi%3D10.1016%2Fj.cell.2012.12.024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref125"><div class="reference"><strong class="refLabel"><a href="#ref125" class="rightTabRefNumLink">125</a></strong><div class="NLM_citation" id="rightTab-cit125"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gelfand, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mironov, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jomantas, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozlov, Y. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perumov, D. A.</span></span> <span> </span><span class="NLM_article-title">A conserved RNA structure element involved in the regulation of bacterial riboflavin synthesis genes</span>. <i>Trends Genet.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">439</span>â <span class="NLM_lpage">442</span>, <span class="refDoi">Â DOI: 10.1016/S0168-9525(99)01856-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref125/cit125&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1016%2FS0168-9525%2899%2901856-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref125/cit125&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10529804" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref125/cit125&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADyaK1MXmvVKhsb8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=1999&pages=439-442&author=M.+S.+Gelfandauthor=A.+A.+Mironovauthor=J.+Jomantasauthor=Y.+I.+Kozlovauthor=D.+A.+Perumov&title=A+conserved+RNA+structure+element+involved+in+the+regulation+of+bacterial+riboflavin+synthesis+genes&doi=10.1016%2FS0168-9525%2899%2901856-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit125R"><div class="casContent"><span class="casTitleNuber">125</span><div class="casTitle"><span class="NLM_cas:atitle">A conserved RNA structure element involved in the regulation of bacterial riboflavin synthesis genes</span></div><div class="casAuthors">Gelfand, M. S.; Mironov, A. A.; Jomantas, J.; Kozlov, Y. I.; Perumov, D. A.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Genetics</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">439-442</span>CODEN:
                <span class="NLM_cas:coden">TRGEE2</span>;
        ISSN:<span class="NLM_cas:issn">0168-9525</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">A review, with 19 refs., on the RFN element found in riboflavin biosynthesis operons from Gram-pos. bacteria, Thermotogales, Deinococcus, and Proteobacteria.  Alignment of the RFN element sequences and the conserved structure of the element are presented.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq95lZv7DUy5bVg90H21EOLACvtfcHk0li7Ach1tmyB8w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXmvVKhsb8%253D&md5=9b3052a294ce279f9b3f6544b43223c4</span></div><a href="/servlet/linkout?suffix=cit125&amp;dbid=16384&amp;doi=10.1016%2FS0168-9525%2899%2901856-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0168-9525%252899%252901856-9%26sid%3Dliteratum%253Aachs%26aulast%3DGelfand%26aufirst%3DM.%2BS.%26aulast%3DMironov%26aufirst%3DA.%2BA.%26aulast%3DJomantas%26aufirst%3DJ.%26aulast%3DKozlov%26aufirst%3DY.%2BI.%26aulast%3DPerumov%26aufirst%3DD.%2BA.%26atitle%3DA%2520conserved%2520RNA%2520structure%2520element%2520involved%2520in%2520the%2520regulation%2520of%2520bacterial%2520riboflavin%2520synthesis%2520genes%26jtitle%3DTrends%2520Genet.%26date%3D1999%26volume%3D15%26spage%3D439%26epage%3D442%26doi%3D10.1016%2FS0168-9525%2899%2901856-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref126"><div class="reference"><strong class="refLabel"><a href="#ref126" class="rightTabRefNumLink">126</a></strong><div class="NLM_citation" id="rightTab-cit126"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Howe, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischmann, T. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balibar, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galgoci, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malinverni, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mayhood, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villafania, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nahvi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murgolo, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbieri, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mann, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carr, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xia, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zuck, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riley, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Painter, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sherborne, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jesus, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plotkin, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rindgen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cummings, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garlisi, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheth, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gill, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roemer, T.</span></span> <span> </span><span class="NLM_article-title">Selective small-molecule inhibition of an RNA structural element</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>526</i></span>,  <span class="NLM_fpage">672</span>â <span class="NLM_lpage">677</span>, <span class="refDoi">Â DOI: 10.1038/nature15542</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref126/cit126&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1038%2Fnature15542" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref126/cit126&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=26416753" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref126/cit126&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslCnu7%252FL" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=526&publication_year=2015&pages=672-677&author=J.+A.+Howeauthor=H.+Wangauthor=T.+O.+Fischmannauthor=C.+J.+Balibarauthor=L.+Xiaoauthor=A.+M.+Galgociauthor=J.+C.+Malinverniauthor=T.+Mayhoodauthor=A.+Villafaniaauthor=A.+Nahviauthor=N.+Murgoloauthor=C.+M.+Barbieriauthor=P.+A.+Mannauthor=D.+Carrauthor=E.+Xiaauthor=P.+Zuckauthor=D.+Rileyauthor=R.+E.+Painterauthor=S.+S.+Walkerauthor=B.+Sherborneauthor=R.+D.+Jesusauthor=W.+Panauthor=M.+A.+Plotkinauthor=J.+Wuauthor=D.+Rindgenauthor=J.+Cummingsauthor=C.+G.+Garlisiauthor=R.+Zhangauthor=P.+R.+Shethauthor=C.+J.+Gillauthor=H.+Tangauthor=T.+Roemer&title=Selective+small-molecule+inhibition+of+an+RNA+structural+element&doi=10.1038%2Fnature15542"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit126R"><div class="casContent"><span class="casTitleNuber">126</span><div class="casTitle"><span class="NLM_cas:atitle">Selective small-molecule inhibition of an RNA structural element</span></div><div class="casAuthors">Howe, John A.; Wang, Hao; Fischmann, Thierry O.; Balibar, Carl J.; Xiao, Li; Galgoci, Andrew M.; Malinverni, Juliana C.; Mayhood, Todd; Villafania, Artjohn; Nahvi, Ali; Murgolo, Nicholas; Barbieri, Christopher M.; Mann, Paul A.; Carr, Donna; Xia, Ellen; Zuck, Paul; Riley, Dan; Painter, Ronald E.; Walker, Scott S.; Sherborne, Brad; de Jesus, Reynalda; Pan, Weidong; Plotkin, Michael A.; Wu, Jin; Rindgen, Diane; Cummings, John; Garlisi, Charles G.; Zhang, Rumin; Sheth, Payal R.; Gill, Charles J.; Tang, Haifeng; Roemer, Terry</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">526</span>
        (<span class="NLM_cas:issue">7575</span>),
    <span class="NLM_cas:pages">672-677</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Riboswitches are non-coding RNA structures located in mRNAs that bind endogenous ligands, such as a specific metabolite or ion, to regulate gene expression.  As such, riboswitches serve as a novel, yet largely unexploited, class of emerging drug targets.  Demonstrating this potential, however, has proven difficult and is restricted to structurally similar antimetabolites and semi-synthetic analogs of their cognate ligand, thus greatly restricting the chem. space and selectivity sought for such inhibitors.  Here we report the discovery and characterization of ribocil, a highly selective chem. modulator of bacterial riboflavin riboswitches, which was identified in a phenotypic screen and acts as a structurally distinct synthetic mimic of the natural ligand, FMN, to repress riboswitch-mediated ribB gene expression and inhibit bacterial cell growth.  Our findings indicate that non-coding RNA structural elements may be more broadly targeted by synthetic small mols. than previously expected.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWptaVLK3njLVg90H21EOLACvtfcHk0li7Ach1tmyB8w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslCnu7%252FL&md5=c0f8443348c10f69eb47f05a2ac5e893</span></div><a href="/servlet/linkout?suffix=cit126&amp;dbid=16384&amp;doi=10.1038%2Fnature15542&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature15542%26sid%3Dliteratum%253Aachs%26aulast%3DHowe%26aufirst%3DJ.%2BA.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DFischmann%26aufirst%3DT.%2BO.%26aulast%3DBalibar%26aufirst%3DC.%2BJ.%26aulast%3DXiao%26aufirst%3DL.%26aulast%3DGalgoci%26aufirst%3DA.%2BM.%26aulast%3DMalinverni%26aufirst%3DJ.%2BC.%26aulast%3DMayhood%26aufirst%3DT.%26aulast%3DVillafania%26aufirst%3DA.%26aulast%3DNahvi%26aufirst%3DA.%26aulast%3DMurgolo%26aufirst%3DN.%26aulast%3DBarbieri%26aufirst%3DC.%2BM.%26aulast%3DMann%26aufirst%3DP.%2BA.%26aulast%3DCarr%26aufirst%3DD.%26aulast%3DXia%26aufirst%3DE.%26aulast%3DZuck%26aufirst%3DP.%26aulast%3DRiley%26aufirst%3DD.%26aulast%3DPainter%26aufirst%3DR.%2BE.%26aulast%3DWalker%26aufirst%3DS.%2BS.%26aulast%3DSherborne%26aufirst%3DB.%26aulast%3DJesus%26aufirst%3DR.%2BD.%26aulast%3DPan%26aufirst%3DW.%26aulast%3DPlotkin%26aufirst%3DM.%2BA.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DRindgen%26aufirst%3DD.%26aulast%3DCummings%26aufirst%3DJ.%26aulast%3DGarlisi%26aufirst%3DC.%2BG.%26aulast%3DZhang%26aufirst%3DR.%26aulast%3DSheth%26aufirst%3DP.%2BR.%26aulast%3DGill%26aufirst%3DC.%2BJ.%26aulast%3DTang%26aufirst%3DH.%26aulast%3DRoemer%26aufirst%3DT.%26atitle%3DSelective%2520small-molecule%2520inhibition%2520of%2520an%2520RNA%2520structural%2520element%26jtitle%3DNature%26date%3D2015%26volume%3D526%26spage%3D672%26epage%3D677%26doi%3D10.1038%2Fnature15542" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref127"><div class="reference"><strong class="refLabel"><a href="#ref127" class="rightTabRefNumLink">127</a></strong><div class="NLM_citation" id="rightTab-cit127"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mansoori, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohammadi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davudian, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shirjang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baradaran, B.</span></span> <span> </span><span class="NLM_article-title">The different mechanisms of cancer drug resistance: A brief review</span>. <i>Adv. Pharm. Bull.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">339</span>â <span class="NLM_lpage">348</span>, <span class="refDoi">Â DOI: 10.15171/apb.2017.041</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref127/cit127&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.15171%2Fapb.2017.041" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref127/cit127&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=29071215" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref127/cit127&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BC1MXktFCntbk%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&pages=339-348&author=B.+Mansooriauthor=A.+Mohammadiauthor=S.+Davudianauthor=S.+Shirjangauthor=B.+Baradaran&title=The+different+mechanisms+of+cancer+drug+resistance%3A+A+brief+review&doi=10.15171%2Fapb.2017.041"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit127R"><div class="casContent"><span class="casTitleNuber">127</span><div class="casTitle"><span class="NLM_cas:atitle">The different mechanisms of cancer drug resistance: a brief review</span></div><div class="casAuthors">Mansoori, Behzad; Mohammadi, Ali; Davudian, Sadaf; Shirjang, Solmaz; Baradaran, Behzad</div><div class="citationInfo"><span class="NLM_cas:title">Advanced Pharmaceutical Bulletin</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">339-348</span>CODEN:
                <span class="NLM_cas:coden">APBDCL</span>;
        ISSN:<span class="NLM_cas:issn">2251-7308</span>.
    
            (<span class="NLM_cas:orgname">Tabriz University of Medical Sciences</span>)
        </div><div class="casAbstract">Anticancer drugs resistance is a complex process that arises from altering in the drug targets.  Advances in the DNA microarray, proteomics technol. and the development of targeted therapies provide the new strategies to overcome the drug resistance.  Although a design of the new chemotherapy agents is growing quickly, effective chemotherapy agent has not been discovered against the advanced stage of cancer (such as invasion and metastasis).  The cancer cell resistance against the anticancer agents can be due to many factors such as the individual's genetic differences, esp. in tumoral somatic cells.  Also, the cancer drug resistance is acquired, the drug resistance can be occurred by different mechanisms, including multi-drug resistance, cell death inhibiting (apoptosis suppression), altering in the drug metab., epigenetic and drug targets, enhancing DNA repair and gene amplification.  In this review, we outlined the mechanisms of cancer drug resistance and in following, the treatment failures by common chemotherapy agents in the different type of cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5pNO4Qea_HLVg90H21EOLACvtfcHk0lglO6OVcWh4fQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXktFCntbk%253D&md5=889d5efbe4bca0f1c3946f55848a13ee</span></div><a href="/servlet/linkout?suffix=cit127&amp;dbid=16384&amp;doi=10.15171%2Fapb.2017.041&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.15171%252Fapb.2017.041%26sid%3Dliteratum%253Aachs%26aulast%3DMansoori%26aufirst%3DB.%26aulast%3DMohammadi%26aufirst%3DA.%26aulast%3DDavudian%26aufirst%3DS.%26aulast%3DShirjang%26aufirst%3DS.%26aulast%3DBaradaran%26aufirst%3DB.%26atitle%3DThe%2520different%2520mechanisms%2520of%2520cancer%2520drug%2520resistance%253A%2520A%2520brief%2520review%26jtitle%3DAdv.%2520Pharm.%2520Bull.%26date%3D2017%26volume%3D7%26spage%3D339%26epage%3D348%26doi%3D10.15171%2Fapb.2017.041" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref128"><div class="reference"><strong class="refLabel"><a href="#ref128" class="rightTabRefNumLink">128</a></strong><div class="NLM_citation" id="rightTab-cit128"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, P.</span></span> <span> </span><span class="NLM_article-title">Inhibition of RNA-binding proteins with small molecules</span>. <i>Nature Rev. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">441</span>â <span class="NLM_lpage">458</span>, <span class="refDoi">Â DOI: 10.1038/s41570-020-0201-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref128/cit128&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=10.1038%2Fs41570-020-0201-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref128/cit128&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;key=1%3ACAS%3A528%3ADC%252BB3cXitFeru7vN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2020&pages=441-458&author=P.+Wu&title=Inhibition+of+RNA-binding+proteins+with+small+molecules&doi=10.1038%2Fs41570-020-0201-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit128R"><div class="casContent"><span class="casTitleNuber">128</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of RNA-binding proteins with small molecules</span></div><div class="casAuthors">Wu, Peng</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">441-458</span>CODEN:
                <span class="NLM_cas:coden">NRCAF7</span>;
        ISSN:<span class="NLM_cas:issn">2397-3358</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review.  Protein-RNA interactions have crucial roles in various cellular activities, which, when dysregulated, can lead to a range of human diseases.  The identification of small mols. that target the interaction between RNA-binding proteins (RBPs) and RNA is progressing rapidly and represents a novel strategy for the discovery of chem. probes that facilitate understanding of the cellular functions of RBPs and of therapeutic agents with new mechanisms of action.  In this Review, I present a current overview of targeting emerging RBPs using small-mol. inhibitors and recent progress in this burgeoning field.  Small-mol. inhibitors that were reported for three representative emerging classes of RBPs, the microRNA-binding protein LIN28, the single-stranded or double-stranded RNA-binding Toll-like receptors and the CRISPR-assocd. (Cas) proteins, are highlighted from a medicinal-chem. and chem.-biol. perspective.  However, although this field is burgeoning, challenges remain in the discovery and characterization of small-mol. inhibitors of RBPs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq4zPBh0BaIybVg90H21EOLACvtfcHk0lglO6OVcWh4fQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXitFeru7vN&md5=2cb4f106e0b5b8c92f37ef5757f07fed</span></div><a href="/servlet/linkout?suffix=cit128&amp;dbid=16384&amp;doi=10.1038%2Fs41570-020-0201-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41570-020-0201-4%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DP.%26atitle%3DInhibition%2520of%2520RNA-binding%2520proteins%2520with%2520small%2520molecules%26jtitle%3DNature%2520Rev.%2520Chem.%26date%3D2020%26volume%3D4%26spage%3D441%26epage%3D458%26doi%3D10.1038%2Fs41570-020-0201-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3GM7" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3GM7','PDB','3GM7'); return false;">PDB: 3GM7</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2L94" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2L94','PDB','2L94'); return false;">PDB: 2L94</a></li></ul></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.1c00154&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2021.64.issue-11%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.1c00154%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.1c00154" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                9MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright Â© 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">Youâve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"67996f993ba53d36","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
