
<html lang="en"     class="pb-page"  data-request-id="fd11e1a6-906e-4d2b-ade7-b0826c4d554b"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/acs.jmedchem.0c01692;wgroup:string:ACHS website Group;issue:issue:10.1021/jmcmar.2021.64.issue-3;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Targeting RECQL5 Functions, by a Small Molecule, Selectively Kills Breast Cancer in Vitro and in Vivo" /></meta><meta name="dc.Creator" content="Saikat  Chakraborty" /></meta><meta name="dc.Creator" content="Kartik  Dutta" /></meta><meta name="dc.Creator" content="Pooja  Gupta" /></meta><meta name="dc.Creator" content="Anubrata  Das" /></meta><meta name="dc.Creator" content="Amit  Das" /></meta><meta name="dc.Creator" content="Sunil Kumar  Ghosh" /></meta><meta name="dc.Creator" content="Birija Sankar  Patro" /></meta><meta name="dc.Description" content="Clinical and preclinical data reveal that RECQL5 protein overexpression in breast cancer was strongly correlated with poor prognosis, survival, and therapeutic resistance. In the current investigat..." /></meta><meta name="Description" content="Clinical and preclinical data reveal that RECQL5 protein overexpression in breast cancer was strongly correlated with poor prognosis, survival, and therapeutic resistance. In the current investigat..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="February 2, 2021" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.0c01692" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2021 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c01692" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.0c01692" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c01692" /></link>
        
    
    

<title>Targeting RECQL5 Functions, by a Small Molecule, Selectively Kills Breast Cancer in Vitro and in Vivo | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c01692" /></meta><meta property="og:title" content="Targeting RECQL5 Functions, by a Small Molecule, Selectively Kills Breast Cancer in Vitro and in Vivo" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01692/20210204/images/large/jm0c01692_0029.jpeg" /></meta><meta property="og:description" content="Clinical and preclinical data reveal that RECQL5 protein overexpression in breast cancer was strongly correlated with poor prognosis, survival, and therapeutic resistance. In the current investigation, we report design, synthesis, and specificity of a small molecule, 4a, which can preferentially kill RECQL5-expressing breast cancers but not RECQL5 knockout. Our stringent analysis showed that compound 4a specifically sensitizes RECQL5-expressing cancers, while it did not have any effect on other members of DNA RECQL-helicases. Integrated approaches of organic synthesis, biochemical, in silico molecular simulation, knockouts, functional mutation, and rescue experiments showed that 4a potently inhibits RECQL5-helicase activity and stabilizes RECQL5-RAD51 physical interaction, leading to impaired HRR and preferential killing of RECQL5-expressing breast cancer. Moreover, 4a treatment led to the efficient sensitization of cisplatin-resistant breast cancers but not normal mammary epithelial cells. Pharmacologically, compound 4a was orally effective in reducing the growth of RECQL5-expressing breast tumors (human xenograft) in NUDE-mice with no appreciable toxicity to the vital organs." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.0c01692"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01692">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.0c01692&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.0c01692&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.0c01692&amp;href=/doi/10.1021/acs.jmedchem.0c01692" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2021</span><span class="cit-fg-volume">, 64</span><span class="cit-fg-issue">, 3</span><span class="cit-fg-pageRange">, 1524-1544</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/64/3" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.0c01683" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.0c01700" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Targeting RECQL5 Functions, by a Small Molecule, Selectively Kills Breast Cancer <i>in Vitr</i>o and <i>in Vivo</i></span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Saikat Chakraborty</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Saikat Chakraborty</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Bio-Organic Division, Bhabha Atomic Research Centre, Trombay, Mumbai 400085, India</div><div class="loa-info-affiliations-info">Homi Bhabha National Institute, Anushaktinagar, Mumbai 400094, India</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Saikat++Chakraborty">Saikat Chakraborty</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Kartik Dutta</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Kartik Dutta</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Bio-Organic Division, Bhabha Atomic Research Centre, Trombay, Mumbai 400085, India</div><div class="loa-info-affiliations-info">Homi Bhabha National Institute, Anushaktinagar, Mumbai 400094, India</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Kartik++Dutta">Kartik Dutta</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Pooja Gupta</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Pooja Gupta</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Bio-Organic Division, Bhabha Atomic Research Centre, Trombay, Mumbai 400085, India</div><div class="loa-info-affiliations-info">Homi Bhabha National Institute, Anushaktinagar, Mumbai 400094, India</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Pooja++Gupta">Pooja Gupta</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Anubrata Das</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Anubrata Das</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Bio-Organic Division, Bhabha Atomic Research Centre, Trombay, Mumbai 400085, India</div><div class="loa-info-affiliations-info">Homi Bhabha National Institute, Anushaktinagar, Mumbai 400094, India</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Anubrata++Das">Anubrata Das</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Amit Das</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Amit Das</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Homi Bhabha National Institute, Anushaktinagar, Mumbai 400094, India</div><div class="loa-info-affiliations-info">Radiation Biology & Health Sciences Division, Bhabha Atomic Research Centre, Trombay, Mumbai 400085, India</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Amit++Das">Amit Das</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Sunil Kumar Ghosh</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Sunil Kumar Ghosh</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Bio-Organic Division, Bhabha Atomic Research Centre, Trombay, Mumbai 400085, India</div><div class="loa-info-affiliations-info">Homi Bhabha National Institute, Anushaktinagar, Mumbai 400094, India</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#fd9f948e9c9396bd9f9c8f9ed39a928bd39493"><span class="__cf_email__" data-cfemail="a6c4cfd5c7c8cde6c4c7d4c588c1c9d088cfc8">[email protected]</span></a></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Sunil+Kumar++Ghosh">Sunil Kumar Ghosh</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-2508-6181" title="Orcid link">http://orcid.org/0000-0003-2508-6181</a></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Birija Sankar Patro</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Birija Sankar Patro</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Bio-Organic Division, Bhabha Atomic Research Centre, Trombay, Mumbai 400085, India</div><div class="loa-info-affiliations-info">Homi Bhabha National Institute, Anushaktinagar, Mumbai 400094, India</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#3e59564d4b5057527e5c5f4c5d10595148105750"><span class="__cf_email__" data-cfemail="c1a6a9b2b4afa8ad81a3a0b3a2efa6aeb7efa8af">[email protected]</span></a></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Birija+Sankar++Patro">Birija Sankar Patro</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-9161-5913" title="Orcid link">http://orcid.org/0000-0001-9161-5913</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c01692&amp;href=/doi/10.1021%2Facs.jmedchem.0c01692" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2021</span></span><span class="cit-volume">, 64</span><span class="cit-issue">, 3</span><span class="cit-pageRange">, 1524–1544</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">February 2, 2021</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>28 September 2020</li><li><span class="item_label"><b>Published</b> online</span>2 February 2021</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 11 February 2021</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.0c01692" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01692</a></div><div class="article_header-article-copyright"><strong>Copyright © 2021 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D1524%26pageCount%3D21%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DSaikat%2BChakraborty%252C%2BKartik%2BDutta%252C%2BPooja%2BGupta%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D64%26issueNum%3D3%26contentID%3Dacs.jmedchem.0c01692%26title%3DTargeting%2BRECQL5%2BFunctions%252C%2Bby%2Ba%2BSmall%2BMolecule%252C%2BSelectively%2BKills%2BBreast%2BCancer%2Bin%2BVitro%2Band%2Bin%2BVivo%26numPages%3D21%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D1544%26publicationDate%3DFebruary%2B2021">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.0c01692"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">710</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val">-</div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.0c01692" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Targeting RECQL5 Functions, by a Small Molecule, Selectively Kills Breast Cancer in Vitro and in Vivo&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Saikat&quot;,&quot;last_name&quot;:&quot;Chakraborty&quot;},{&quot;first_name&quot;:&quot;Kartik&quot;,&quot;last_name&quot;:&quot;Dutta&quot;},{&quot;first_name&quot;:&quot;Pooja&quot;,&quot;last_name&quot;:&quot;Gupta&quot;},{&quot;first_name&quot;:&quot;Anubrata&quot;,&quot;last_name&quot;:&quot;Das&quot;},{&quot;first_name&quot;:&quot;Amit&quot;,&quot;last_name&quot;:&quot;Das&quot;},{&quot;first_name&quot;:&quot;Sunil&quot;,&quot;last_name&quot;:&quot;Kumar Ghosh&quot;},{&quot;first_name&quot;:&quot;Birija&quot;,&quot;last_name&quot;:&quot;Sankar Patro&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2021&quot;,&quot;month&quot;:&quot;02&quot;,&quot;day&quot;:&quot;02&quot;,&quot;issue&quot;:&quot;3&quot;,&quot;volume&quot;:&quot;64&quot;,&quot;pages&quot;:&quot;1524-1544&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.0c01692&quot;},&quot;abstract&quot;:&quot;Clinical and preclinical data reveal that RECQL5 protein overexpression in breast cancer was strongly correlated with poor prognosis, survival, and therapeutic resistance. In the current investigation, we report design, synthesis, and specificity of a small molecule, 4a, which can preferentially kill RECQL5-expressing breast cancers but not RECQL5 knockout. Our stringent analysis showed that compound 4a specifically sensitizes RECQL5-expressing cancers, while it did not have any effect on other members of DNA RECQL-helicases. Integrated approaches of organic synthesis, biochemical, in silico molecular simulation, knockouts, functional mutation, and rescue experiments showed that 4a potently inhibits RECQL5-helicase activity and stabilizes RECQL5-RAD51 physical interaction, leading to impaired HRR and preferential killing of RECQL5-expressing breast cancer. Moreover, 4a treatment led to the efficient sensitization of cisplatin-resistant breast cancers but not normal mammary epithelial cells. Pharmacologically,&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01692&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01692" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01692&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01692" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01692&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01692" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c01692&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01692&amp;href=/doi/10.1021/acs.jmedchem.0c01692" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.0c01692" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.0c01692" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (13 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01692&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.0c01692%26sid%3Dliteratum%253Aachs%26pmid%3D33529023%26genre%3Darticle%26aulast%3DChakraborty%26date%3D2021%26atitle%3DTargeting%2BRECQL5%2BFunctions%252C%2Bby%2Ba%2BSmall%2BMolecule%252C%2BSelectively%2BKills%2BBreast%2BCancer%2Bin%2BVitro%2Band%2Bin%2BVivo%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D64%26issue%3D3%26spage%3D1524%26epage%3D1544%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (3)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292258" title="Cancer">Cancer</a>,</li><li><a href="/action/doSearch?ConceptID=291788" title="Reaction products">Reaction products</a>,</li><li><a href="/action/doSearch?ConceptID=291439" title="Genetics">Genetics</a>,</li><li><a href="/action/doSearch?ConceptID=290768" title="Antineoplastic agents">Antineoplastic agents</a>,</li><li><a href="/action/doSearch?ConceptID=290896" title="Cells">Cells</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/64/3" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/jmcmar.2021.64.issue-3/20210211/jmcmar.2021.64.issue-3.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01692/20210204/images/medium/jm0c01692_0029.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01692/20210204/images/large/jm0c01692_0029.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01692&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Clinical and preclinical data reveal that RECQL5 protein overexpression in breast cancer was strongly correlated with poor prognosis, survival, and therapeutic resistance. In the current investigation, we report design, synthesis, and specificity of a small molecule, <b>4a</b>, which can preferentially kill RECQL5-expressing breast cancers but not RECQL5 knockout. Our stringent analysis showed that compound <b>4a</b> specifically sensitizes RECQL5-expressing cancers, while it did not have any effect on other members of DNA RECQL-helicases. Integrated approaches of organic synthesis, biochemical, <i>in silico</i> molecular simulation, knockouts, functional mutation, and rescue experiments showed that <b>4a</b> potently inhibits RECQL5-helicase activity and stabilizes RECQL5-RAD51 physical interaction, leading to impaired HRR and preferential killing of RECQL5-expressing breast cancer. Moreover, <b>4a</b> treatment led to the efficient sensitization of cisplatin-resistant breast cancers but not normal mammary epithelial cells. Pharmacologically, compound <b>4a</b> was orally effective in reducing the growth of RECQL5-expressing breast tumors (human xenograft) in NUDE-mice with no appreciable toxicity to the vital organs.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_04286" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_04286" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Breast cancer has presently afflicted 2.1 million females globally<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> and considered as a leading cause of cancer death in women. A better understanding of tumor biology and genomics, development of improved and sensitive diagnostics, and effective and targeted therapeutics have been associated with a reduction of breast cancer patient mortality.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> Several neoadjuvant/adjuvant chemotherapeutics (cisplatin, doxorubicin, paclitaxel, 5-FU, gemcitabine <i>etc.</i>), which causes extensive DNA damages, show positive therapeutic outcomes for breast cancer patients. Besides, targeted chemotherapeutics, like tamoxifen and PARP inhibitors, are successfully used against hormone receptor positive and BRCA1/2 mutated breast cancer patients, respectively.<a onclick="showRef(event, 'ref3 ref4 ref5'); return false;" href="javascript:void(0);" class="ref ref3 ref4 ref5">(3−5)</a> However, chemoresistance remains a paramount challenge in the treatment of breast cancer.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> The presently available interventions are inadequate to target chemoresistance and pathogenesis of breast cancer in patients. To expand the horizon of targeted oncotherapy and clinical medicines, the development of specific drugs for targeting novel key players in breast cancer pathogenesis and <i>de novo</i> and/or acquired chemoresistance is urgently required.</div><div class="NLM_p">RECQL5 is a key member of the human RECQ helicase family. Several lines of evidence suggested that RECQL5 deficiency is associated with genomic instability and development of different types of cancers, including breast cancers.<a onclick="showRef(event, 'ref6 ref7'); return false;" href="javascript:void(0);" class="ref ref6 ref7">(6,7)</a> On the other hand, high expression of RECQL5, in established tumors, may promote proliferation and survival of cancer cells. RECQL5 polymorphism is associated with predisposition to laryngeal carcinoma, osteosarcoma, and breast carcinoma.<a onclick="showRef(event, 'ref8 ref9 ref10 ref11'); return false;" href="javascript:void(0);" class="ref ref8 ref9 ref10 ref11">(8−11)</a> Recently, Arora <i>et al.</i>, analyzed 1902 breast cancer patients [Nottingham Tenovus series (<i>n</i> = 1650) and ER-cohort (<i>n</i> = 252)] and found that high RECQL5 and low RAD51 protein expressions were significantly linked to high histopathological grade (<i>p</i> < 0.0001), higher mitotic index (<i>p</i> = 0.008), dedifferentiation (<i>p</i> = 0.025), pleomorphism (<i>p</i> = 0.027), and poor survival (<i>p</i> = 0.003).<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> It has also been found that 53.7% (644/1200) of breast tumors showed a higher expression of RECQL5 in the nucleus. RECQL5 is known to play key roles in homologous recombination, transcription, replication, and other DNA repair process. RECQL5 disrupts RAD51 presynaptic filament by interacting directly with RAD51.<a onclick="showRef(event, 'ref13 ref14'); return false;" href="javascript:void(0);" class="ref ref13 ref14">(13,14)</a> Interestingly, RECQL5 expression is linked to <i>de novo</i> resistance to cisplatin in different types of cancers,<a onclick="showRef(event, 'ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref15 ref16">(15,16)</a> suggesting a DNA repair role of RECQL5 behind the resistance. Considering its putative role in breast cancer pathogenesis and chemotherapeutics resistance, we hypothesized that targeting RECQL5 with a small molecule may be a key strategy to (1) sensitize RECQL5-expressing breast cancers and (2) abrogate the neoadjuvant/adjuvant-mediated chemoresistance in breast cancer.</div><div class="NLM_p last">Because, the exact molecular function of RECQL5 in breast cancer pathogenesis and chemoresistance is not yet known, we followed a cell-based screening approach to evaluate the potential of small molecule and, designed and synthesized 1,3,4-oxadiazole derivatives in-house, which may preferentially kill RECQL5-expressing breast cancer than RECQL5-deficient breast cancers. Following this approach, we found a small molecule (1-(3,5-bis(trifluoromethyl)phenyl)-3-((<i>S</i>)-(5-(benzylthio)-1,3,4-oxadiazol-2-yl)(phenyl)methyl)urea), which can preferentially kill RECQL5-expressing breast cancers <i>in vitro</i> and is also orally effective in the preclinical breast tumor model. Mechanistically, this small molecule targeted the nonenzymatic domain of RECQL5, stabilized RECQL5-RAD51 physical interaction, and impaired homologous recombinational repair (HRR), which led to the robust sensitization of RECQL5-expressing breast cancer cells. Interestingly, this functional inhibitor of RECQL5 abrogated <i>de novo</i> and acquired cisplatin resistance in breast cancers.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_85385" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_85385" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Chemistry</h3><div id="sec2_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i5" class="anchor-spacer"></div><h4 class="article-section__title" id="_i5"> Synthesis of 1,3,4-Oxadiazole Derivatives</h4><div class="NLM_p">The synthesis pathway for the 1,3,4 oxadiazole thiourea derivatives (compound <b>3a–3c</b>) is shown in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>. The synthesis started with <i>N</i>-Boc-protected glycine methyl ester <b>1a</b>, which was converted to oxadiazole <b>2a</b> following conventional chemical reactions. The <i>N</i>-Boc deprotection followed by the reaction with substituted phenylisothiocyanates provided thiourea-substituted oxadiazole derivatives <b>3a–c</b> in moderate to good yields. By employing similar reaction sequences, oxadiazole <b>2b,c</b> were prepared from appropriate (<i>S</i>)-phenylglycine and (<i>S</i>)-phenylalanine derivatives, respectively, as shown in <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>. The <i>N</i>-Boc deprotection followed by the reaction with substituted phenylisocyanates provided urea-substituted oxadiazole derivatives <b>4a–i</b>.</div><figure id="sch1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01692/20210204/images/medium/jm0c01692_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01692/20210204/images/large/jm0c01692_0011.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Schematic Routes for the Synthesis of 1,3,4-Oxadiazole with Thiourea Derivatives</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01692/20210204/images/large/jm0c01692_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01692&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="sch2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01692/20210204/images/medium/jm0c01692_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01692/20210204/images/large/jm0c01692_0012.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Schematic Routes for the Synthesis of 1,3,4-Oxadiazole with Urea Derivatives</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01692/20210204/images/large/jm0c01692_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01692&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Biological Evaluation</h3><div id="sec2_2_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i9" class="anchor-spacer"></div><h4 class="article-section__title" id="_i9"> Potential of 1,3,4-Oxadiazole Derivatives to Kill Cancer Cells by Targeting RECQL5</h4><div class="NLM_p">By analyzing a breast cancer database,<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> comprising 65 patients, we found that the high expression of RECQL5 was inversely associated with overall survival (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A). Moreover, the exogenous expression of RECQL5 in MCF10A normal breast epithelial cell drives proliferation.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> Together, these suggest a RECQL5-mediated oncogenic shift in breast cancer cells <i>in vitro</i> and in patients. To this end, we asked whether specific pharmacological inhibition of RECQL5 functions may sensitize the breast cancer cells for a better therapeutic outcome. Initially, we screened our in-house small molecule library to find a molecule, which can preferentially kill RECQL5-expressing cells over RECQL5 knockout cells. In order to generate RECQL5-KO cells, control and two different sets of the RECQL5 CRISPR-CAS9 double nickase system were used in MCF-7 cells (ER+/PR+/HER2-, p53 WT). Both RECQL5 CRISPR-CAS9 plasmid systems efficiently knocked out RECQL5 in MCF-7 breast cancer cells (RECQL5-KO1 and RECQL5-KO2), in comparison to control plasmids in MCF-7 cells (RECQL5-WT) (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B). In-house library (200 drugs) comprises a small collection of chemotherapeutics, antidiabetic, antiseizure, antipsychotic, antibiotic, antiviral agents, nonsteroidal anti-inflammatory drugs, and so forth. From our screening, we found that 1,3,4-Oxadiazole derivative molecule (e.g., Raltegravir) showed some potential to differentially kill RECQL5-WT cells over RECQL5-KO2 cells. 1,3,4-Oxadiazole derivatives are generally considered as privileged small molecules in medicinal chemistry because of their lower lipophilicity, enhanced water solubility, superior hydrolytic and metabolic stability, and improved cell membrane permeability and pharmacokinetics (PKs).<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> Inspired by the raltegravir structure, we synthesized three 1,3,4-Oxadiazole thiourea derivatives where phenyl group was functionalized with trifluoromethyl and nitro groups at various positions (compounds <b>3a–3c</b>; <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>). Among these three oxadiazole derivatives, compound <b>3a</b> showed some selectivity in killing RECQL5-WT cells over RECQL5-KO2 cells (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01692/suppl_file/jm0c01692_si_001.pdf" class="ext-link">Figure S1A</a>). In order to further enhance the selectivity, we introduced the following changes in 1,3,4-Oxadiazole derivative: (1) thiourea was replaced with urea; (2) a bulky phenyl or phenyl methyl was introduced in between urea and 1,3,4-Oxadiazole groups; and (3) phenyl group was functionalized with methoxy, methyl, trifluoromethyl, or nitro group at various positions (compounds <b>4a–i</b>; <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>). As shown in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>C,D, all the compounds, except <b>4b</b>, reduced cell viability of both RECQL5-WT and RECQL5-KO2 cells in a concentration-dependent manner. Interestingly, compound <b>4a</b> (1-(3,5-bis(trifluoromethyl)phenyl)-3-((<i>S</i>)-(5-(benzylthio)-1,3,4-oxadiazol-2-yl) (phenyl) methyl) urea) showed significantly higher selectivity in killing RECQL5-WT cells <i>vis-á-vis</i> RECQL5-KO cells. The IC<sub>20</sub> value (concentration at which 20% killing is achieved) of <b>4a</b> was 8.2 ± 2.6 μM for RECQL5-WT cells, which was ∼2.7 times lower than the IC<sub>20</sub> value for RECQL5-KO2 cells (22.3 ± 2.1 μM) (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>C,D). From structure activity analyses, we found that the presence of 3,5-bis(trifluoromethyl)phenyl, urea, and immobile phenyl groups in between the urea and 1,3,4-oxadiazole moiety is required for the selective killing of RECQL5-expressing cells (compounds <b>4a–4i</b>; <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>, <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>C,D). Changes in any of these chemical entities led to drastic reduction in either efficacy and/or selective killing of RECQL5-WT cells. In order to assess the cytotoxicity of <b>4a</b> toward normal mammary epithelial cells (expressing RECQL5), MCF10A cells and its counter malignant MCF-7 cells were treated with <b>4a</b>. Our results showed that although <b>4a</b> was cytotoxic toward both the cells, IC<sub>20</sub> for MCF10A (33.4 ± 3.2 μM) was ∼4 times higher than MCF-7 cells (8.4 ± 2.9 μM) (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>E). In agreement with the previous report,<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> we observed very low expression of RECQL5 in normal mammary epithelial MCF10A cells than malignant MCF-7 cells. The lower RECQL5 expression in normal mammary epithelial cells (MCF10A) may attribute to lower cytotoxicity in response to <b>4a</b> treatment (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>F).</div><figure id="fig1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01692/20210204/images/medium/jm0c01692_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01692/20210204/images/large/jm0c01692_0002.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Screening and evaluation of selective cytotoxicity of 1,3,4-Oxadiazole derivatives to RECQL5-expressing breast cancer cells. (A) Kaplan–Meier analyses of overall survival of breast cancer patients with high and low RECQL5 protein expressions. (B) CRISPR-CAS9 double nickase-mediated knockout of RECQL5 in MCF-7 cells. Control double nickase plasmid and two different RECQL5 double nickase plasmids were transfected to MCF-7 cells to generate RECQL5-WT, RECQL5-KO1, and RECQL5-KO2 cells. (C) Cell viability of RECQL5-WT and RECQL5-KO2 cells after 72 h treatment with compounds <b>4a–i</b>. Highlighted green box shows the highest preferential killing ability of <b>4a</b> for RECQL5-WT cells. Data are shown as mean ± SD (<i>n</i> = 4). *<i>p</i> < 0.05 and **<i>p</i> < 0.01. (D) Dot-plot of IC<sub>20</sub> values of compounds <b>4a–i</b> for RECQL5-WT and RECQL5-KO2 cells. (E) Cell viability of normal mammary epithelial cells (MCF10A) <i>vs</i> malignant breast cancer cells (MCF-7) in response to <b>4a</b> treatment (72 h). Data are shown as mean ± SD (<i>n</i> = 4). *<i>p</i> < 0.05 w.r.t the viability of the respective vehicle-treated cells (100%). (F) RECQL5 expression in MCF10A and MCF-7 cells. (G,H) Clonogenic potential of RECQL5-WT and RECQL5-KO2 cells after 8–10 days treatment with compound <b>4a</b>. Data are shown as mean ± SD (<i>n</i> = 5). *<i>p</i> < 0.01 in comparison to respective <b>4a</b> concentration in RECQL5-KO2 cells.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01692/20210204/images/large/jm0c01692_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01692&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">In the clonogenic assay, RECQL5-WT cells were highly sensitive to <b>4a</b> treatment (IC<sub>50</sub>: 4.8 μM), while RECQL5-KO2 cells were relatively fourfold resistant (IC<sub>50</sub>: 19.6 μM) (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>G,H). Similar results were also observed in RECQL5-KO1 cells (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01692/suppl_file/jm0c01692_si_001.pdf" class="ext-link">Figure S1B</a>), suggesting that the differential effect of <b>4a</b> was not because of nonspecific effects of CRISPR plasmids. In order to assess the differential efficacy in other cancer cells, RECQL5 was knocked out in T47D (breast cancer cells; ER+/PR+/HER2-; and p53 mutant) by using two independent CRISPR-CAS9 systems or depleted in MDA-MB-231 (triple negative breast cancer; ER-/PR-/HER2-; and p53 mutant) and U2-OS cells (osteosarcoma cells; p53 WT) by using the lentiviral-mediated expression of two independent shRNAs (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01692/suppl_file/jm0c01692_si_001.pdf" class="ext-link">Figure S2A–F</a>). In corroboration with MCF-7 results, we observed that RECQL5-expressing cancer cells (RECQL5-WT T47D, MDA-MB-231, and U2-OS) were highly sensitive to <b>4a</b> treatment, while the effect was drastically reduced when RECQL5 depleted in these cells (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01692/suppl_file/jm0c01692_si_001.pdf" class="ext-link">Figure S2A–F</a>). Together, this result indicated that <b>4a</b> targets RECQL5-expressing cancer cells irrespective of their tissue origin and p53 status. In order to characterize <b>4a</b> and understand its mode of action, further experiments were carried out in RECQL5-WT and RECQL5-KO2 (henceforth named RECQL5-KO) MCF-7 cells.</div></div><div id="sec2_2_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i10" class="anchor-spacer"></div><h4 class="article-section__title" id="_i10"> Specificity of Compound <b>4a</b> in Preferential Killing of RECQL5 Helicase-Expressing Cancer Cells</h4><div class="NLM_p">The above results showed that compound <b>4a</b> sensitizes RECQL5-expressing cells (RECQL5-WT) but not RECQL5-KO cells. In order to confirm that <b>4a</b>-induced cancer cell killing is mediated specifically through RECQL5, RECQL5-KO cells were complemented with either the ectopic expression of RECQL5-RFP or RFP control (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A). Our cell viability assay showed that RECQL5-KO cells, in contrast to RECQL5-WT cells, were resistant to <b>4a</b> treatment. Meanwhile, complementation with RECQL5-RFP but not RFP reverted RECQL5-KO resistance phenotype in response to <b>4a</b> treatment (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B). RECQL5 shares homology with other human RECQ helicases, for example, WRN, BLM, RECQL4, and RECQL1.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> In order to assess the specific interaction of <b>4a</b> with RECQL5 and not with any other RECQL helicases in cells, RECQL5, RECQL1, WRN, and BLM were systematically depleted by using two different sets of respective shRNA in MCF-7 cells. ShRNA-mediated depletion of RECQL5, RECQL1, WRN, and BLM was more than 90% in the respective knockdown cells (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>C). Our cell viability assay showed that <b>4a</b> treatment reduced the viability of control cells while the effect of <b>4a</b> was significantly abrogated in RECQL5sh cells (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>D). In contrast, the cytotoxic potential of <b>4a</b> was not affected in RECQL1sh, WRNsh, and BLMsh cells (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>D). Together, these results confirm that the cytotoxicity potential of compound <b>4a</b> is mediated through its specific interaction with RECQL5 in the cellular context.</div><figure id="fig2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01692/20210204/images/medium/jm0c01692_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01692/20210204/images/large/jm0c01692_0003.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Compound <b>4a</b>-mediated selective killing is dependent on the RECQL5 expression in the cancer cells. (A) Expression of RFP (control) and RECQL5-RFP in RECQL5-KO cells. Bar: 10 μm. (B) Cell viability of RECQL5-WT, RECQL5-KO, RECQL5-KO + RFP, and RECQL5-KO + RECQL5-RFP cells after <b>4a</b> treatment for 72 h. Data are shown as mean ± SD (<i>n</i> = 5). *<i>p</i> < 0.01. (C) Expression of RECQL5, RECQL1, WRN, and BLM in MCF-7 cells, after knocking down (KD) by using control shRNA and two different sets of shRNA for RECQL5, RECQL1, WRN, and BLM, respectively. (D) Clonogenic potential of control MCF-7 cells, RECQL5-KD2 (RECQL5 shRNA2), RECQL1-KD2 (RECQL1 shRNA2), WRN-KD1 (WRN shRNA1), and BLM-KD1 (BLM shRNA1) MCF-7 cells after 8–10 days treatment with compound <b>4a</b>. Data are shown as mean ± SD (<i>n</i> = 5). *<i>p</i> < 0.05, **<i>p</i> < 0.01, ***<i>p</i> < 0.001.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01692/20210204/images/large/jm0c01692_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01692&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_2_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i11" class="anchor-spacer"></div><h4 class="article-section__title" id="_i11"> Effect of Compound <b>4a</b> on Enzymatic and Nonenzymatic Function of RECQL5 in Killing Cancer Cells</h4><div class="NLM_p">The RECQL5 protein structure comprises (1) a conserved <i>N</i>-terminal domain, which includes RECA-like helicase domain (1–364 aa) and RQC domain (438–453 aa) and (2) C-terminal domain, which contains KIX, BRCv, and SRI domains (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A).<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> Biochemical analyses of RECQL5 revealed that the helicase domain has DNA-dependent ATPase and ATP-dependent 3′–5′ helicase and Holliday junction branch migration activities. The crystal structure of RECQL5 showed that the RQC domain has a single α-helix, which is essential for DNA binding and helicase activity of the enzyme.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> In order to assess whether compound <b>4a</b>-mediated cytotoxicity might be because of its ability to inhibit ATPase, helicase, and/or DNA binding abilities of RECQL5, full length HIS-tagged human RECQL5 was expressed and purified from <i>E. coli</i> (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B). A fluorimeter-based RECQL5 helicase assay was performed by using 30-mer replication forks. One strand of this forked duplex was labeled with fluorophore (FAM) at 5′-end while the other strand was labeled with a quencher (BHQ1) at 3′-end (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>C). Unwinding of this duplex due to helicase activity abolishes proximity-based fluorescence quenching of FAM by BHQ1, leading to the enhancement of FAM fluorescence that can be detected in real time. As shown in <a href="/doi/suppl/10.1021/acs.jmedchem.0c01692/suppl_file/jm0c01692_si_001.pdf" class="ext-link">Figure S3A</a>, wild type RECQL5 time dependently enhanced the FAM fluorescence, showing its efficient helicase activity. The point mutation of the critical lysine residue to methionine (K577M) is known to cause loss of helicase activity of the WRN protein.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> Similarly, the point mutation of the critical lysine residue to arginine (K58R)<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> or methionine (K58M) led to the loss of helicase activity of the RECQL5 protein (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01692/suppl_file/jm0c01692_si_001.pdf" class="ext-link">Figure S3A</a>). Interestingly, helicase activity of the RECQL5 (WT) protein was potently inhibited in the presence of compound <b>4a</b>, in a concentration-dependent manner (IC<sub>50</sub>: 46.3 nM) (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>D, <a href="/doi/suppl/10.1021/acs.jmedchem.0c01692/suppl_file/jm0c01692_si_001.pdf" class="ext-link">S3B</a>). Although, the compound <b>4e</b> is structurally closely related to <b>4a</b>, it showed very poor RECQL5 helicase inhibition activity in the same assay (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>D). In contrast to the presence of the phenyl group in between urea and 1,3,4-Oxadiazole groups in compound <b>4a</b>, presence of the relatively more mobile phenyl methyl group in compound <b>4e</b> drastically affects its efficacy to inhibit RECQL5 (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>). In control experiments, we found that heat inactivated helicase or <b>4a</b> alone was unable to unwind duplex DNA (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>D). In contrast, our <i>in vitro</i> biochemical assay showed that <b>4a</b> was not able to abrogate RECQL5 binding to the DNA substrate, even at higher concentrations (10 μM) (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01692/suppl_file/jm0c01692_si_001.pdf" class="ext-link">Figure S3C,D</a>). A similar result was also observed at lower concentrations of <b>4a</b>. In the ATPase assay, we found that <b>4a</b> was able to reduce DNA-dependent ATPase activity of RECQL5 in a concentration-dependent manner (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01692/suppl_file/jm0c01692_si_001.pdf" class="ext-link">Figure S3E</a>). However, <b>4a</b> could suppress ∼50% of ATPase activity at higher concentrations (IC<sub>50</sub>: 23.9 μM), suggesting a weak ATPase inhibition property of <b>4a</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01692/suppl_file/jm0c01692_si_001.pdf" class="ext-link">Figure S3E</a>). Further, we carried out molecular docking simulation of <b>4a</b> and <b>4e</b> with the ATPase domain by using the reported crystal structure of RECQL5 helicase complexed to ADP (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LB3">5LB3</a>) using LeadIT incorporated in BioSolveIT software. The most favorable binding mode of <b>4a</b> with ATPase pocket of RECQL5 is shown in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>E,F. In this docking study, compound <b>4a</b> showed (1) hydrophobic interaction with the pockets comprising Thr54, Gly55, and Ala56 and Gly57, Ser59, Phe26, and Leu60 and (2) hydrogen bonding interactions with Gly55, Gly57, Lys58, and Ser59. In contrast to binding of <b>4a</b> (Docking score: 21), the binding of <b>4e</b> (Docking score: 6) to ATPase pocket was less favorable by Δ<i>G</i>: 10 kJ/mol, because of the dynamic nature of the phenyl methyl group, leading to weaker hydrophobic and hydrogen bonding interactions (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01692/suppl_file/jm0c01692_si_001.pdf" class="ext-link">Figure S3F,G</a>). Together, these results suggest that <b>4a</b> is a potent inhibitor of RECQL5 helicase activity while affecting ATPase activity partially. A similar effect was reported for a WRN helicase specific inhibitor.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> In order to further assess whether the potent inhibition of RECQL5 helicase (enzyme) activity might be attributable to its preferential killing of RECQL5-expressing cancer cells, a RECQL5 mutant with helicase dead function<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> was generated. This was carried out by introducing a point mutation (K58M) in RECQL5 through the site directed mutagenesis protocol. Our clonogenic assay result showed that RECQL5-KO cells, in contrast to RECQL5-WT cells, were resistant to <b>4a</b> treatment. Of note, the complementation of RECQL5-KO cells with either RECQL5-RFP (WT) or RECQL5-RFP (K58M) helicase dead mutant (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>G) caused robust sensitization of cancer cells in response to <b>4a</b> treatment (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>H,I). The effect of <b>4a</b> on RECQL5-RFP (K58M) helicase dead mutant was slightly less than RECQL5-RFP (WT) expressing cells. Together, these results suggest that the compound <b>4a</b>-mediated potent inhibition of RECQL5 helicase (enzyme) may partially confer to its preferential killing of RECQL5-expressing cancer cells. In addition to helicase enzyme activities, <b>4a</b> may possibly target the nonenzymatic function of RECQL5, leading to the robust sensitization of RECQL5-expressing cancer cells.</div><figure id="fig3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01692/20210204/images/medium/jm0c01692_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01692/20210204/images/large/jm0c01692_0004.jpeg" id="gr3" class="inline-fig internalNav" /></img><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01692/20210204/images/medium/jm0c01692_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01692/20210204/images/large/jm0c01692_0005.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Effect of <b>4a</b> on the ATPase, helicase, and DNA binding abilities of RECQL5 protein. (A) Schematic representation of RECQL5 protein with different domains. (B) N-terminal HIS-tagged human RECQL5 was expressed in <i>E. coli</i> and purified. The Gel shows two different batches of preparations. (C) Schematic representation of the fluorescent-based RECQL5 helicase assay using a duplex forked DNA. One of the strands was labeled with 3′-BHQ1, which quenches fluorescence of 5′-FAM labeled on the other strand. Helicase activity driven strand-unwinding results in the separation of 5′-FAM labeled strand and enhancement of the fluorescence. (D) RECQL5 helicase activity measured by the above fluorescent-based assay in the absence and presence of <b>4a</b> or <b>4e</b>. Data are shown as mean ± SD (<i>n</i> = 5). **<i>p</i> < 0.01 and ***<i>p</i> < 0.001. (E) Predicted binding mode of <b>4a</b> with ATPase pocket of RECQL5 (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LB3">5LB3</a>). The diagram shows compound <b>4a</b> in purple and ADP in yellow. (F) Possible mode of interaction of <b>4a</b> with ADP binding pocket of RECQL5. (G) RECQL5-KO cells untreated or transfected with RFP and RECQL5 (WT)-RFP. RECQL5 (K58M)-RFP and expression of these proteins was assessed. (H,I) Clonogenic potential of the indicated cells after <b>4a</b> treatment (8–10 days). Data are shown as mean ± SD (<i>n</i> = 5). *<i>p</i> < 0.01 w.r.t RECQL5-KO cells at the respective concentration of <b>4a</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01692/20210204/images/large/jm0c01692_0004.jpeg jm0c01692_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01692&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_2_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i12" class="anchor-spacer"></div><h4 class="article-section__title" id="_i12"> Effect of Compound <b>4a</b> on Double Strand Breaks Repair and DNA Damage Response</h4><div class="NLM_p">In order to gain insights into the molecular mechanism of preferential killing of RECQL5-expressing cancer cells, we evaluated the effects of <b>4a</b> on the double strand break (DSB) repair process. In corroboration with our cell viability and clonogenic assay results (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>D,G), Compound <b>4a</b> treatment induced a significantly higher amount of apoptosis (sub-G1) in RECQL5-WT cells <i>vis-á-vis</i> RECQL5-KO cells, in a dose-dependent manner (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A,B). At 20 μM of <b>4a</b>, ∼2.5-fold higher death was observed for RECQL5-WT cells as compared to RECQL5-KO cells. Besides, <b>4a</b> treatment also induced higher death in RECQL5-WT as compared to RECQL5-depleted U2-OS and MDA-MB-231 cells, suggesting that <b>4a</b> preferentially kills RECQL5-expressing cancer cells irrespective of its cancer types (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01692/suppl_file/jm0c01692_si_001.pdf" class="ext-link">Figure S4A,B</a>; data not shown for MDA-MB-231). Of note, 1,3,4-oxadiazole derivatives have the propensity to induce DNA damage.<a onclick="showRef(event, 'ref23 ref24'); return false;" href="javascript:void(0);" class="ref ref23 ref24">(23,24)</a> Because RECQL5 is known to play a pivotal role in DNA damage response (DDR) and repair, it might be possible that compound <b>4a</b> inhibits RECQL5-mediated signaling and/or DNA repair, leading to a higher cell death. In the absence of RECQL5 in RECQL5-KO cells, DNA damage might be repaired through redundant compensatory proteins/pathways. In this regard, we assessed DSBs by analyzing γH2AX foci in cells. As shown in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>C,D, the treatment of <b>4a</b> (24 h) led to a significantly higher amount of γH2AX foci in RECQL5-WT cells than RECQL5-KO cells. The γH2AX foci amount was enhanced in a <b>4a</b> dose-dependent manner. Similar results were also observed in T47D cells (RECQL5-WT and RECQL5-KO cells) (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01692/suppl_file/jm0c01692_si_001.pdf" class="ext-link">Figure S4C</a>). Further, major DDR, for example, ATM-CHK2, ATR-CHK1, and p53 signaling was remarkably elicited with higher intensities in RECQL5-WT cells <i>vis-á-vis</i> RECQL5-KO cells in response to <b>4a</b> treatment (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>E,F). Taken together, our results suggested that compound <b>4a</b> induces DNA DSBs and targets the RECQL5-mediated processing of DSBs, leading to preferential killing of RECQL5-expressing cancer cells.</div><figure id="fig4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01692/20210204/images/medium/jm0c01692_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01692/20210204/images/large/jm0c01692_0006.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Compound <b>4a</b> caused higher accumulation of DSBs, which led to preferential death of RECQL5-expressing cells. (A,B) Apoptosis of RECQL5-WT and RECQL5-KO in response to different doses of <b>4a</b> for 72 h. Data are shown as mean ± SD (<i>n</i> = 5). **<i>p</i> < 0.01 and ***<i>p</i> < 0.001. (C,D) DSBs in the cells, treated with <b>4a</b> for 24 h, was measured by analyzing γH2AX foci by immunofluorescence microscopy. Data are shown as mean ± SEM (<i>n</i> = 5). ***<i>p</i> < 0.001. (E,F) Cells were treated with <b>4a</b> for indicated time points. DDR signaling was analyzed by western blotting. Data are shown as mean of at least three experiments.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01692/20210204/images/large/jm0c01692_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01692&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_2_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i13" class="anchor-spacer"></div><h4 class="article-section__title" id="_i13"> Role of Compound <b>4a</b> in Homologous Recombination and Stabilization of the RECQL5-RAD51 Complex</h4><div class="NLM_p">Because compound <b>4a</b> causes robust accumulation of γH2AX in RECQL5-WT cells than RECQL5-KO cells, we further systematically evaluated the effect of <b>4a</b> on NHEJ- and HR-mediated DSB repair. Initially for NHEJ and HR repair, <b>4a</b> treatment-mediated time-dependent accumulation of key NHEJ/HRR proteins on chromatin was evaluated. We found no significant difference in the rate of accumulation of KU80 and DNA-PKc on chromatin, in RECQL5-WT and RECQL5-KO cells, in response to <b>4a</b> treatment (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01692/suppl_file/jm0c01692_si_001.pdf" class="ext-link">Figure S5A</a>). This result suggested that NHEJ might not be a crucial target of compound <b>4a</b>. Interestingly, <b>4a</b> treatment led to a time-dependent accumulation of HR processing MRN proteins, for example, MRE11, RAD50, and NBS1 on the chromatin of RECQL5-WT cells. In contrast, this effect was relatively lower in the RECQL5-KO cells. In corroboration with this result, our result revealed that the phosphorylation of MRE11 (S676) and NBS1 (S343) was significantly higher in RECQL5-WT cells <i>vis-á-vis</i> RECQL5-KO cells (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A). During HRR, DSBs are resected to generate extensive ssDNA, followed by coating and phosphorylation of RPA. In this regard, we observed ssDNA-mediated robust phosphorylation of RPA2, in a time-dependent manner, in RECQL5-WT cells in response to <b>4a</b> treatment (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B). This effect of <b>4a</b> was significantly lower in RECQL5-KO cells, suggesting that compound <b>4a</b> may target RECQL5 in a HR-mediated repair process.</div><figure id="fig5" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01692/20210204/images/medium/jm0c01692_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01692/20210204/images/large/jm0c01692_0007.jpeg" id="gr6" class="inline-fig internalNav" /></img><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01692/20210204/images/medium/jm0c01692_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01692/20210204/images/large/jm0c01692_0008.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Compound <b>4a</b>-stabilized RECQL5-RAD51 physical interaction and inhibited HRR. (A,B) RECQL5-WT and RECQL5-KO cells were treated with <b>4a</b> for indicated time points and chromatin bound proteins was analyzed by western blotting. (C,D) Cells were treated with <b>4a</b> for indicated time periods and RAD51 foci were analyzed by immunofluorescence microscopy. Highlighted box in the images were zoomed. Data are shown as mean ± SD (<i>n</i> = 6). ***<i>p</i> < 0.001. (E) Scheme of I-SceI-induced DSBs at the SceGFP site and its HR-based repair to functional GFP. (F) Cells were treated with <b>4a</b> for 48 h and relative HRR was analyzed by measuring GFP positive cells by flow cytometry. Data are shown as mean ± SD (<i>n</i> = 6). ***<i>p</i> < 0.001. (G) Level of RAD51 and RECQL5 in the nuclear and chromatin extract of the cells, treated with <b>4a</b>, was measured by western blotting. Data are shown as mean (<i>n</i> = 3–4). (H) RECQL5-KO (MCF-7) cells transfected with EV or RECQL5-FLAG. These cells were untreated or treated with <b>4a</b>, immuno-precipitated with anti-FLAG tagged beads and probed for FLAG and RAD51. (I) RECQL5-KO (MCF-7) cells transfected with EV, RECQL5 (WT)-FLAG, and RECQL5 (F666A)-FLAG and expression of RECQL5-FLAG was assessed. (J,K) Clonogenic potential of indicated cells after <b>4a</b> treatment (8–10 days). Data are shown as mean ± SD (<i>n</i> = 5). *<i>p</i> < 0.05 w.r.t RECQL5-KO cells at the respective concentration of <b>4a</b>. **<i>p</i> < 0.01.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01692/20210204/images/large/jm0c01692_0007.jpeg jm0c01692_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01692&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Initially, it has been proposed that RECQL5 suppresses HR-mediated DSB repair by disrupting the interaction of RAD51 with ssDNA to form presynaptic filament.<a onclick="showRef(event, 'ref25 ref26'); return false;" href="javascript:void(0);" class="ref ref25 ref26">(25,26)</a> Meanwhile, many recent studies have shown evidence that RECQL5 can also act on the postsynaptic phase during synthesis-dependent strand annealing (SDSA) process.<a onclick="showRef(event, 'ref13 ref15 ref27'); return false;" href="javascript:void(0);" class="ref ref13 ref15 ref27">(13,15,27)</a> Hence, RECQL5 prevents the formation of aberrant RAD51 filaments on the extended invading strand and its channelization to potentially lethal cross-over pathway of HRR. Because compound <b>4a</b> has the ability to prolong MRN and ssDNA signaling in a RECQL5-dependent manner (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A,B), <b>4a</b> might be regulating RAD51 and RECQL5 physical interaction in the cells. To test this hypothesis, the kinetics of RAD51 foci formation was evaluated. Our results revealed that although the initial rate of RAD51 foci formation was similar (4 h) in both RECQL5-WT and RECQL5-KO cells (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>C,D), it was enhanced at 8 h in RECQL5-WT cells. Further, a remarkably higher number of RECQL5-WT cells showed enhanced number of RAD51 foci than RECQL5-KO cells at 24 h of <b>4a</b> treatment (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>C,D; <a href="/doi/suppl/10.1021/acs.jmedchem.0c01692/suppl_file/jm0c01692_si_001.pdf" class="ext-link">Figure S5B</a>). Similar results were also observed for T47D (RECQL5-WT and RECQL5-KO) cells (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01692/suppl_file/jm0c01692_si_001.pdf" class="ext-link">Figure S5C</a>). The difference in the RAD51 foci level in both the cells was not because of the differential effect of <b>4a</b> on the cell cycle of the RECQL5-WT and RECQL5-KO cells (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01692/suppl_file/jm0c01692_si_001.pdf" class="ext-link">Figure S6A</a>). This suggests that compound <b>4a</b> was not affecting the presynaptic complex formation of RAD51 at early time points but it might be stabilizing the RAD51-RECQL5 physical interaction later, leading to the suppression of further HRR. To validate <b>4a</b>-mediated HRR inhibition, we used a plasmid (pDR-GFP)-based HRR reporter assay. These cells harbor two mutant GFP genes (SceGFP and iGFP) (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>E). SceGFP contains an I-SceI endonuclease site while iGFP is a truncated gene. The expression of I-SceI endonucleases leads to generation of resected DSB at the SceGFP site, which is repaired further by HR to generate functional GFP by using iGFP as a homologous template (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>E). The flow cytometric measurement of GFP-expressing cells provides a measure of HRR. In this assay, we observed that HRR was significantly reduced in RECQL5-WT cells, while it was not altered in RECQL5-KO cells in response to <b>4a</b> treatment (48 h; <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>F). This result confirms that compound <b>4a</b> targets RECQL5 to inhibit HRR.</div><div class="NLM_p last">Because our results indicated that <b>4a</b> inhibits HRR (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>D) by stabilizing the RAD51-RECQL5 physical interaction in RECQL5-expressing cells (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A,B), we carried out a series of experiments to assess the role of <b>4a</b> in stabilizing the RECQL5-RAD51 complex and its role in cancer cell death. (1) Treatment of <b>4a</b> led to the enhanced accumulation of chromatin-bound RECQL5 and RAD51, in a time-dependent manner in RECQL5-WT cells (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>G). Further, RECQL5 and RAD51 levels did not alter significantly in the nuclear extracts of <b>4a</b>-treated cells (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>G), indicating that <b>4a</b>-mediated accumulation of RECQL5 and RAD51 was not because of the change in the expression levels of these proteins. (2) In order to characterize the stabilization of the RECQL5-RAD51 complex by <b>4a</b>, RECQL5-KO cells were complemented with RECQL5-FLAG (WT) and treated with <b>4a</b> for 8 h and RECQL5-FLAG was pulled/immuno-precipitated (IP) by agarose beads coated with anti-FLAG. In corroboration with a previous report,<a onclick="showRef(event, 'ref14 ref28'); return false;" href="javascript:void(0);" class="ref ref14 ref28">(14,28)</a> we found that RAD51 was co-precipitated in the RECQL5-FLAG IP sample from untreated cells (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>H). Interestingly, <b>4a</b> treatment significantly enhanced the level of RAD51 in the RECQL5-FLAG IP sample (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>H). A similar but higher pulldown of RAD51 along with RECQL5-FLAG was observed in T47D cells (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01692/suppl_file/jm0c01692_si_001.pdf" class="ext-link">Figure S6B</a>). Because this IP experiment was carried out with the chromatin extract, subjected with nuclease and EtBr treatment it overrules intermediacy of DNA in the RECQL5-RAD51 interaction. The IP result confirms that <b>4a</b> targets RECQL5 to stabilize the RECQL5-RAD51 physical interaction to suppress HRR (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>F). (3) Several recent reports showed that single amino acid mutation at the 666<sup>th</sup> position in RECQL5 abolishes its anti-recombinogenic property via abrogating its interaction with RAD51.<a onclick="showRef(event, 'ref14 ref28 ref29'); return false;" href="javascript:void(0);" class="ref ref14 ref28 ref29">(14,28,29)</a> In this regard, we ectopically expressed RECQL5 (WT)-FLAG or RECQL5-F666A-FLAG in RECQL5-KO cells (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>I) and assessed their sensitivity toward <b>4a</b> treatment. In corroboration with our previous results, the colony forming ability was severely reduced when RECQL5-KO cells were complemented with the expression of RECQL5 (WT)-FLAG protein, while transfection with empty vector (EV) had no effect (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>J,K). Interestingly, RECQL5-KO cells ectopically expressing RECQL5-F666A-FLAG were almost equally resistant as RECQL5-KO cells in response to <b>4a</b> treatment (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>J,K). Taken together, our results confirm that <b>4a</b> targets RECQL5 (WT) protein to stabilize the RECQL5-RAD51 physical interaction on chromatin (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>G,H), leading to longer postsynaptic RAD51 filaments/foci (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>C,D), impaired HRR (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>E,F), and enhanced cytotoxicity of RECQL5-expressing cells (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>J,K). In the absence of RECQL5 in KO cells, the RAD51 filament might be processed further by known RAD51 filament-disrupting factors, for example, RAD54 and BLM,<a onclick="showRef(event, 'ref13 ref15'); return false;" href="javascript:void(0);" class="ref ref13 ref15">(13,15)</a> leading to low cytotoxicity.</div></div><div id="sec2_2_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i14" class="anchor-spacer"></div><h4 class="article-section__title" id="_i14"> Effect of Compound <b>4a</b> on Targeting RECQL5 in Cisplatin-Treated Breast Cancer Cells</h4><div class="NLM_p">HRR plays a crucial role in inducing <i>de novo</i> and acquired resistance in cancer cells in response to many cancer therapeutics.<a onclick="showRef(event, 'ref29 ref30'); return false;" href="javascript:void(0);" class="ref ref29 ref30">(29,30)</a> We hypothesized that <b>4a</b> treatment may be effective to target RECQL5 in HRR to abrogate <i>de novo</i> and acquired resistance in cancer cells in response to some of the clinically used DNA damaging cancer therapeutics. To test this hypothesis, we assessed enhancement of RECQL5 expression as a DDR to a range of clinically used DNA damaging cancer therapeutics, for example, aphidicolin, cisplatin, camptothecin (CPT), hydroxyurea (HU), olaparib (PARP inhibitor), gemcitabine, and ionizing radiation (IR). As shown in <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A, an enhanced expression of RECQL5 was found in response to cisplatin and CPT. We observed that the level of chromatin-bound RECQL5 was enhanced several folds in response to aphidicolin, cisplatin, CPT, and IR treatment, suggesting a putative role of RECQL5 in <i>de novo</i> resistance to these therapeutics (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>B). Previously, Hosono <i>et al.</i> showed that RECQL5 controls HRR of DNA cross-linking and mediates cell resistance in response to cisplatin treatment.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> In another study, Røe <i>et al.</i> showed that the overexpression of RECQL5 in mesothelioma cancer is associated with cisplatin resistance.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a></div><figure id="fig6" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01692/20210204/images/medium/jm0c01692_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01692/20210204/images/large/jm0c01692_0009.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Compound <b>4a</b>-abrogated <i>de novo</i> and acquired cisplatin resistance in breast cancer cells. (A) MCF-7 cells were treated with IR (6 Gy, 24 h recovery) or with indicated therapeutics for 24 h. The level of RECQL5 in whole cell extract was analyzed by western blotting. (B) MCF-7 cells were treated with IR (6 Gy, 24 h recovery) or aphidicolin, cisplatin, and CPT for 24 h. The level of chromatin-bound RECQL5 was analyzed by western blotting. (C) Cisplatin-resistant (MCF-7<sup>CDDP</sup>) cells were generated after repeated and prolonged cisplatin treatment to MCF-7 cells. Expression of RECQL5 in MCF-7 and MCF-7<sup>CDDP</sup> cells were analyzed by western blotting. (D) MCF-7 and MCF-7<sup>CDDP</sup> cells were treated with cisplatin for 8 h, cisplatin was removed and cells were treated with <b>4a</b>. Clonogenic potential of these cells was assessed after 8–10 days. Data are shown as mean ± SD (<i>n</i> = 4). *<i>p</i> < 0.05 and **<i>p</i> < 0.01. (E) MCF-7 and MCF-7<sup>CDDP</sup> cells were treated with cisplatin (20 μM) for 8 h, cisplatin was removed and cells were treated with <b>4a</b> (20 μM) for 24 h. RAD51 foci was analyzed by immunofluorescence microscopy. Data are shown as mean ± SD (<i>n</i> = 6). **<i>p</i> < 0.01 and ***<i>p</i> < 0.001. (F) Cells were treated as mentioned in E and mitotic catastrophe was analyzed by immunofluorescence microscopy. Data are shown as mean ± SD (<i>n</i> = 6). *<i>p</i> < 0.05 w.r.t respective untreated nuclei.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01692/20210204/images/large/jm0c01692_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01692&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In order to evaluate the association of RECQL5 overexpression in acquired cisplatin resistance in breast cancer cells, we generated cisplatin-resistant MCF-7 breast cancer cells by following our previously reported protocol.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> The clonogenic assay showed that IC<sub>50</sub> of cisplatin was 6.9 and 18.2 μM for parental MCF-7 and cisplatin-resistant MCF-7 (MCF-7<sup>CDDP</sup>) cells, respectively. Interestingly, we found that the expression of RECQL5 was twofold in MCF-7<sup>CDDP</sup> cells <i>vis-á-vis</i> MCF-7 cells (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>C). Further, combination treatment of cisplatin and <b>4a</b> caused robust sensitization of both MCF-7 and MCF-7<sup>CDDP</sup> cells, suggesting the role of <b>4a</b> in abrogating <i>de novo</i> resistance in MCF-7 cells and acquired resistance in MCF-7<sup>CDDP</sup> cells. The sensitizing effect of <b>4a</b> was higher in the latter cells (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>D).</div><div class="NLM_p last">Then, we also noticed that the combination treatment of cisplatin and <b>4a</b> led to a higher number of RAD51 foci positive MCF-7 and MCF-7<sup>CDDP</sup> cells (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>E). Again, the effect of <b>4a</b> on the stabilization of RAD51 foci was higher in MCF-7<sup>CDDP</sup> cells than MCF-7 cells. Deregulated HRR is known to induce mitotic catastrophe and cell death.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> In this regard, our results showed that although individual treatment led to enhancement of mitotic catastrophe (micronuclei, nuclear bridges and fused nuclei), combination treatment of cisplatin and <b>4a</b> resulted in higher number of nuclei with mitotic catastrophe phenotype in MCF-7 and MCF-7<sup>CDDP</sup> cells (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>F). Together, our results suggested that <i>de novo</i> and acquired cisplatin resistance was mediated through higher expression of RECQL5 in breast cancer cells, which is consistent with previous reports.<a onclick="showRef(event, 'ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref15 ref16">(15,16)</a> Imperatively, compound <b>4a</b> targets RECQL5 to stabilize the RECQL5-RAD51 complex, leading to the sensitization of cisplatin-resistant breast cancer cells. Hence, compound <b>4a</b> may act as an excellent adjuvant for cisplatin therapy of breast cancer.</div></div><div id="sec2_2_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i15" class="anchor-spacer"></div><h4 class="article-section__title" id="_i15"> Pharmacodynamic (PD) Effects of Compound <b>4a</b> on Reduction of Breast Tumor in Preclinical NUDE Mouse Model</h4><div class="NLM_p">The above results demonstrated the ability of compound <b>4a</b> to target RECQL5 and preferentially kill RECQL5-expressing cancer cells <i>in vitro</i>. To assess the PD effects of <b>4a</b> on RECQL5 target in tumor, we used NUDE mice-bearing RECQL5-WT and RECQL5-KO breast xenograft tumors. In this regard, our result showed that oral gavage of compound <b>4a</b> (50 mg/kg body weight; alternate day; 30 days) (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>A) robustly reduced the growth of RECQL5-expressing tumor in a time-dependent manner (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>B–E). In contrast, the effect of compound <b>4a</b> was significantly lower against RECQL5-KO breast cancer in NUDE mice (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>B–E). Further, the oral gavage of compound <b>4a</b> was well tolerated without any sign of weight loss (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01692/suppl_file/jm0c01692_si_001.pdf" class="ext-link">Figure S7A</a>) or morphological abnormality in major organs, for example, liver, kidney, spleen, lungs, and heart. Pharmacodynamically, these results showed that the oral administration of compound <b>4a</b> was also effective in targeting RECQL5-expressing breast tumors <i>in vivo</i> preclinical models.</div><figure id="fig7" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01692/20210204/images/medium/jm0c01692_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01692/20210204/images/large/jm0c01692_0010.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Efficacy of compound <b>4a</b> against the breast tumor in the preclinical mice model. (A) Schematic representation of the breast tumor (xenograft) development in NUDE mice and treatment protocol. (B–E) NUDE mice-bearing RECQL5-WT and RECQL5-KO tumors were treated with the vehicle or <b>4a</b> for 30 days. Tumor volume (TV) was assessed on every alternate day and plotted. Representative images of mouse bearing tumors and excised tumors are shown. Data are shown as mean ± SD (<i>n</i> = 8). *<i>p</i> < 0.05 w.r.t respective the vehicle-treated mice. **<i>p</i> < 0.01. (F) Mice were given oral gavage of <b>4a</b> (50 mg/kg body wt.). Blood was collected immediately after sacrificing mice, at different time points. <b>4a</b> was extracted from the serum and subjected to HPLC analysis. The serum concentration of <b>4a</b> is shown. Data is shown as mean ± SD (<i>n</i> = 4). *<i>p</i> < 0.05, ***<i>p</i> < 0.01 w.r.t 0 h.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01692/20210204/images/large/jm0c01692_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01692&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_2_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i16" class="anchor-spacer"></div><h4 class="article-section__title" id="_i16"> PKs and Toxicity Effect of <b>4a</b> in Preclinical Mouse Models</h4><div class="NLM_p">In order to assess the PK of <b>4a</b>, mice were given the oral gavage of <b>4a</b> (one dose of 50 mg/kg body weight), and serum concentration of <b>4a</b> at different time points (0, 0.5, 4, and 24 h) was analyzed by high-performance liquid chromatography (HPLC). As shown in <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>E and <a href="/doi/suppl/10.1021/acs.jmedchem.0c01692/suppl_file/jm0c01692_si_001.pdf" class="ext-link">S7B</a>, we found that the serum concentration of <b>4a</b> was enhanced time dependently up to 4 h (∼14.8 μM), which was reduced at 24 h. This result shows that the therapeutically effective serum concentration of <b>4a</b> is achievable by oral administration. Besides, higher drug uptake by tumors may further enhance the concentration of <b>4a</b> and its antitumor efficacy. Moreover, the reduction of the serum concentration of <b>4a</b>, after 24 h of oral gavage, also suggests an efficient clearance of <b>4a</b>. For chronic toxicity, the mice were given a higher dose of <b>4a</b> (100 mg/kg body weight; oral gavage) and related changes in body weight, behavior, stool texture, food, and water uptake was monitored for one month. We observed that weight, behavior, food/water uptake, and stool texture were not changed in <b>4a</b>-treated animals <i>versus</i> vehicle-treated animals. Besides, the plasma biochemistry profile revealed no hepatic and renal toxicity in <b>4a</b>-treated <i>vis-á-vis</i> vehicle-treated mice (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). To assess the effect of <b>4a</b> on the proliferating normal cells, total spleen cells were measured. As shown in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, we found no appreciable changes in the cell counts in the spleen of vehicle- and <b>4a</b>-treated mice.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Chronic Toxicity of <b>4a</b> in the Preclinical Mouse Model<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="char" char="±" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center">plasma/tissue profile</th><th class="colsep0 rowsep0" align="center" char="±"><b>4a</b>-treated mice (100 mg/kg body wt) fold changes w.r.t vehicle-treated mice</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">creatinine</td><td class="colsep0 rowsep0" align="char" char="±">1.05 ± 0.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left">SGPT</td><td class="colsep0 rowsep0" align="char" char="±">0.92 ± 0.05</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="left">ALP</td><td class="colsep0 rowsep0" align="char" char="±">0.97 ± 0.08</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">4</td><td class="colsep0 rowsep0" align="left">spleen cell count</td><td class="colsep0 rowsep0" align="char" char="±">1.03 ± 0.06</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">Data are shown as mean ± SD (<i>n</i> = 8).</p></div></div></div></div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i18">Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_52645" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_52645" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Recently, several <i>in vitro</i>, preclinical and clinical studies demonstrated that cancers develop <i>de novo</i> and acquired resistance to many DNA damaging therapeutics because of the upregulation of HRR.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> Thus, HRR has been an attractive target to develop small-molecule inhibitors for enhancing sensitivity of cancer cells and overcoming their resistance to chemo- and radiotherapy.<a onclick="showRef(event, 'ref30 ref33'); return false;" href="javascript:void(0);" class="ref ref30 ref33">(30,33)</a> In this regard, small-molecule inhibitors, for example, CYT-0851 and cediranib, which inhibit HRR, currently reached different phases of clinical trials (I–III) for the treatment of cancer patients.<a onclick="showRef(event, 'ref34 ref35'); return false;" href="javascript:void(0);" class="ref ref34 ref35">(34,35)</a> In clinical oncology, current research is mainly focused on the discovery of different druggable targets in HRR for the sensitization and treatment of different types of cancer. Recently, Bagnolini <i>et al.</i>, discovered dihydroquinolone pyrazoline-based molecule, which inhibits HRR by disrupting RAD51-BRCA2 physical interactions and sensitizes cancers to PARP inhibitor.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> Different RECQL helicases, which play important roles in the HRR process, are also considered as attractive targets for cancer therapy. In this regard, small-molecule inhibitors were developed to specifically target WRN and BLM RECQL helicases and its potential utility as an anticancer agent was advocated.<a onclick="showRef(event, 'ref22 ref37'); return false;" href="javascript:void(0);" class="ref ref22 ref37">(22,37)</a> Interestingly, Yin <i>et al</i>. identified a class of isaindigotone derivatives, which inhibits the recruitment of BLM at the DSB sites and promotes the accumulation of unresolved RAD51 filaments/foci, leading to the death of the cancer cells.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> In the current study, we discovered 1,3,4-oxadiazole derivative (compound <b>4a</b>) as a potent RECQL5 inhibitor (targeting both enzymatic and nonenzymatic domain) through synthesis, screening, biochemical assays, and <i>in vitro</i> and <i>in vivo</i> evaluation. We demonstrated that compound <b>4a</b> abrogates HRR in RECQL5-expressing cancers, leading to their robust sensitization. Because our and others’ analyses have shown that the overexpression of RECQL5 is associated with serious clinicopathological conditions and poor survival in a set of breast cancer patients (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A), the inhibition of RECQL5-regulated HRR in RECQL5-expressing tumors may be therapeutically beneficial against RECQL5-expressing cancers. Among the series of synthesized 1,3,4-oxadiazole derivatives with structural diversities, compound <b>4a</b> emerged as a preferential and potent cytotoxic agent against RECQL5-expressing breast cancer, while its cytotoxic effect was drastically reduced in the absence of RECQL5 in RECQL5-KO breast cancer cells. Our results showed that compound <b>4a</b> had the ability to induce DSBs and inhibit HRR in RECQL5-expressing breast cancer cells. The preferential killing of RECQL5 positive cells was associated with the ability of <b>4a</b> to stabilize the RECQL5-RAD51 physical interaction at the HR sites. In the context of HRR, several findings showed that RECQL5 disrupts RAD51 filament formation by (1) abrogating the interaction of RAD51 with ssDNA, prior to D-loop formation,<a onclick="showRef(event, 'ref14 ref26'); return false;" href="javascript:void(0);" class="ref ref14 ref26">(14,26)</a> and/or (2) removing RAD51 on the two preformed RAD51 filaments.<a onclick="showRef(event, 'ref15 ref27'); return false;" href="javascript:void(0);" class="ref ref15 ref27">(15,27)</a> During the latter process, RECQL5 removes RAD51 on the extended invading strand and its twin noninvading strands to support SDSA and prevent a potentially hazardous cross-over during HRR (<a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>). Our results suggested that RAD51 foci were efficiently formed initially in both RECQL5-WT and RECQL5-KO cells in response to <b>4a</b> treatment. At a latter time point, a robust accumulation of RAD51 foci was observed in RECQL5-WT cells only, suggesting that RECQL5-mediated removal of RAD51 was inhibited by stabilizing the RECQL5-RAD51 physical interaction by compound <b>4a</b> (<a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>). Nevertheless, <b>4a</b> targeted the nonenzymatic domain of RECQL5 and enhanced RAD51 foci (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>C,D), RAD51 level on chromatin (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>G), stabilized RECQL5-RAD51 physical interaction (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>H), and suppressed HRR (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>E) in RECQL5 positive cancer cells.</div><figure id="sch3" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01692/20210204/images/medium/jm0c01692_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01692/20210204/images/large/jm0c01692_0013.jpeg" id="gr12" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Schematic Representation of Mode of Action of<b>4a</b> in Preferential Killing of RECQL5-Expressing Cells<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01692/20210204/images/large/jm0c01692_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01692&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>CO: cross-over; SDSA: synthesis-dependent strand annealing.</p></p></figure><div class="NLM_p">The ATPase and helicase activity of RECQL5 is known to influence the HRR process-mediated suppression of the sister chromatid exchange in chicken DT40 cells.<a onclick="showRef(event, 'ref28 ref39'); return false;" href="javascript:void(0);" class="ref ref28 ref39">(28,39)</a> Interestingly, it is reported that (1) different RECQL5 mutants (including F666A), defective for RAD51 binding, have an intact helicase function and (2) RECQL5 helicase mutation (K58R) has no significant impact on RECQL5 binding to RAD51.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> In this regard, breast cancer cells expressing helicase dead RECQL5 mutant (K58M) were also significantly sensitive to <b>4a</b>. This suggests that although <b>4a</b> potently inhibit RECQL5 helicase activity, this may only partially contribute to <b>4a</b>-mediated killing of RECQL5-expressing cancers. In contrast, RECQL5-KO cells ectopically expressing the RECQL5 (F666A) mutant, which poorly interacts with RAD51,<a onclick="showRef(event, 'ref14 ref26 ref27'); return false;" href="javascript:void(0);" class="ref ref14 ref26 ref27">(14,26,27)</a> were almost equally resistant as RECQL5-KO cells to <b>4a</b> treatment (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>K). Because the crystal structure of the C-terminal region of RECQL5 is unavailable, we were unable to assess the molecular interaction of <b>4a</b> with the RAD51-interacting (BRCv) domain of RECQL5 (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A). Nevertheless, our extensive investigation in cellular studies suggested that compound <b>4a</b> targets RECQL5 and stabilizes RECQL5-RAD51 physical interaction, inhibits RAD51 removal by RECQL5, leading to the accumulation of unresolved toxic RAD51 foci/filaments in RECQL5-expressing cells (<a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>). Because the RECQL5 expression is not only upregulated in breast cancer patients, it may also be upregulated in response to DNA damage therapeutics, leading to enhanced HRR-mediated <i>de novo</i> or acquired resistance and therapeutic failure. In this regard, our results showed remarkable enhancement in the accumulation of RECQL5 on the chromatin of breast cancer cells in response to clinically approved cancer therapeutics, for example, cisplatin, CPT, aphidicolin, and IR (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>B). Recently, it has also been reported that the RECQL5 expression is associated with cisplatin resistance in different cancers.<a onclick="showRef(event, 'ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref15 ref16">(15,16)</a> In the current study, we also observed an enhanced upregulation of the RECQL5 expression in cisplatin-resistant breast cancer (MCF-7<sup>CDDP</sup>) cells than its parental MCF-7 breast cancer cells (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>C). Intriguingly, combination treatment of <b>4a</b> significantly enhanced RAD51 foci and robustly sensitized both MCF-7 parental and MCF-7<sup>CDDP</sup> cells to cisplatin treatment. The effect was higher in the MCF-7<sup>CDDP</sup> cells.</div><div class="NLM_p last">In conclusion, small molecule <b>4a</b> was synthetically designed to target both enzymatic and nonenzymatic domain of RECQL5 in breast cancer. Mechanistically, it stabilized RECQL5-RAD51 physical interaction, leading to the accumulation of the RAD51 filament, HRR impairment, and cell death. Together, compound <b>4a</b> may be used as a single agent to sensitize RECQL5-expressing breast cancer. It can also be used as an adjuvant with cisplatin to abort RECQL5-mediated HRR, thereby it may reduce <i>de novo</i> and acquired cisplatin resistance in breast cancer. In the preclinical mouse model, compound <b>4a</b> was orally effective in significantly reducing growth of RECQL5-expressing breast tumors as compared to RECQL5-KO tumors (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>). High potency along with its nontoxic nature toward normal mammary epithelial cells (MCF10A) and different vital organs (liver, kidney, and spleen) and easy synthetic route in an appreciable quantity make <b>4a</b> a promising and attractive neo-adjuvant/adjuvant against RECQL5-expressing breast tumors.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i20">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_63542" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_63542" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> Chemical Synthesis</h3><div id="sec4_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> Reagents, Materials, and Instrumentation</h4><div class="NLM_p last">All reagents and solvents were of the highest reagent and analytical grades and used without further purifications, unless otherwise stated. Thin-layer chromatographic plates (0.25 mm thickness, Merck Silica Gel 60F<sub>254</sub>) and MN silica gel 60 (230–400 mesh) were used for visualization and purification. The IR (KBr), <sup>1</sup>H, and <sup>13</sup>C NMR spectra were recorded with a BRUKER Tensor II Spectrophotometer Bruker AC-200 (200 MHz) or a Varian 500 MHz NMR spectrometer, respectively. Either MestReNova Lite-11.0.4, ACD/1D NMR Processor, or Bruker TOPSPIN software was used to process NMR spectra. High-resolution mass spectra (MS) were recorded on Agilent instrument and spectral analyses was carried out by 6200 series software (TOF/6500, Version Q-TOF B.05.01 B5125). All NMR spectra are shown in <a href="/doi/suppl/10.1021/acs.jmedchem.0c01692/suppl_file/jm0c01692_si_001.pdf" class="ext-link">Figure S8</a>. The purity of both the lead compounds (<b>4a</b> and <b>4e</b>) are >98% pure, as assessed by HPLC (JASCO; C18 column) (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01692/suppl_file/jm0c01692_si_001.pdf" class="ext-link">Figure S9</a>).</div></div><div id="sec4_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> General Procedure for the Synthesis of <b>2a–2c</b> (<a class="ref internalNav" href="#sch1" aria-label="Schemes 1">Schemes 1</a> and <a class="ref internalNav" href="#sch2" aria-label="2">2</a>)</h4><div class="NLM_p">A mixture of a <i>N</i>-Boc-protected amino acid methyl ester (10 mmol) and hydrazine hydrate (2 mL) in absolute ethanol (10 mL) was heated under reflux for 2 h. The reaction mixture was cooled to room temperature and evaporated under reduced pressure to give the <i>N</i>-Boc amino acid hydrazide as a white solid. <i>N</i>-Boc amino acid hydrazide (10 mmol, 2.7 g) was added to a stirred solution of KOH (0.8 g, 20 mmol) in ethanol (50 mL) followed by the dropwise addition of carbon disulfide (6 mL) with vigorous stirring. The reaction mixture was heated under reflux for 4 h, brought to room temperature, and evaporated under reduced pressure. The residue was taken in water (20 mL), neutralized with 0.5 M HCl to pH 6, and extracted with ethyl acetate (3 × 10 mL). The extract was dried over anhy. MgSO<sub>4</sub> and evaporated under reduced pressure to give crude <b>2a–c</b>. This crude product (<b>2a</b>) in dry ethanol (15 mL) was cooled in an ice-water bath and Et<sub>3</sub>N (1.1 equiv, 1.1 mL) was added followed by the addition of benzyl bromide (1.1 equiv, 1.3 mL). The reaction mixture was stirred at room temperature for 30 min and concentrated under reduced pressure. The residue was dissolved in ethyl acetate and washed with water. The organic layer was evaporated under reduced pressure and the residue was purified by column chromatography to afford pure <b>2</b>(<b>a–c</b>).<img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01692/20210204/images/medium/jm0c01692_0014.gif" alt="" id="dgr1" /></img></div><div id="sec4_1_2_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i24" class="anchor-spacer"></div><h5 class="article-section__title" id="_i24"> <i>tert</i>-Butyl (5-(Benzylthio)-1,3,4-oxadiazol-2-yl)methylcarbamate (<b>2a</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): 7.39 (d, <i>J</i> = 7.0 Hz, 2 H), 7.32–7.25 (m, 3 H), 5.33 (br s,1 H), 4.48 (d, <i>J</i> = 4.5 Hz, 1 H), 4.42 (s, 2 H), 1.43 (s, 9 H); <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>): 164.9, 164.6, 155.4, 135.4, 129.1, 128.8, 128.1, 80.5, 36.7, 35.8, 28.3.<img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01692/20210204/images/medium/jm0c01692_0015.gif" alt="" id="dgr2" /></img></div></div><div id="sec4_1_2_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i25" class="anchor-spacer"></div><h5 class="article-section__title" id="_i25"> <i>tert</i>-Butyl (<i>S</i>)-(5-(Benzylthio)-1,3,4-oxadiazol-2-yl)(phenyl)methylcarbamate (<b>2b</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): 7.40–7.35 (m, 5 H), 7.33–7.28 (m, 5 H), 6.08 (d, <i>J</i> = 7.0 Hz, 1 H), 5.61 (br s, 1 H), 4.42 (ABq, <i>J</i> = 13 Hz, 16.0 Hz, 2 H), 1.45 (s, 9 H); <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>): 166.7, 164.7, 154.5, 136.7, 135.4, 129.1, 129.0, 128.8, 128.8, 128.0, 127.1, 80.8, 51.5, 36.9, 28.3.<img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01692/20210204/images/medium/jm0c01692_0016.gif" alt="" id="dgr3" /></img></div></div><div id="sec4_1_2_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i26" class="anchor-spacer"></div><h5 class="article-section__title" id="_i26"> <i>tert</i>-Butyl (<i>S</i>)-1-(5-(Benzylthio)-1,3,4-oxadiazol-2-yl)-2-phenylethylcarbamate (<b>2c</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): 7.40 (d, <i>J</i> = 7.0 Hz, 2 H), 7.35–7.28 (m, 4 H), 7.25–7.23 (m, 2 H), 7.08–7.07 (m, 2 H), 5.25 (d, <i>J</i> = 6.5 Hz, 1 H), 5.02 (d, <i>J</i> = 8.0 Hz, 1 H), 4.41 (s, 2H), 3.23 (dd, <i>J</i> = 6.0 Hz, 13.5 Hz, 1H), 3.15 (dd, <i>J</i> = 5.5 Hz, 12.5 Hz, 1H), 1.40 (s, 9 H); <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>): 167.1, 164.2, 154.7, 135.4, 135.3, 129.2, 128.9, 128.7, 128.6, 128.0, 127.1, 80.4, 48.3, 39.6, 36.8, 28.1.</div></div></div><div id="sec4_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> General Procedure for Synthesis of <b>3</b>(<b>a–c</b>) (<a class="ref internalNav" href="#sch1" aria-label="Schemes 1">Schemes 1</a> and <a class="ref internalNav" href="#sch2" aria-label="2">2</a>)</h4><div class="NLM_p">Trifluoroacetic acid (0.5 mL) was dropwise added to a stirred solution of <b>2a</b> (0.5 mmol) in dry dichloromethane (2 mL). After 2 h, solvents were removed under reduced pressure to afford the trifluoroacetate salt. A solution of crude trifluoroacetate salt (0.5 mmol) in dichloromethane (5 mL) was cooled at 0 °C in an ice-water bath. Et<sub>3</sub>N (0.1 mL, 0.55 mmol) was added followed by respective isothiocyanates (0.5 mmol) and stirred for 15 min at room temperature, the mixture was concentrated under reduced pressure and the residue was taken in ethyl acetate. The solution was washed with water and concentrated under reduced pressure. The crude product was purified by column chromatography to afford pure <b>3a–c</b>.<img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01692/20210204/images/medium/jm0c01692_0017.gif" alt="" id="dgr4" /></img></div><div id="sec4_1_3_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i28" class="anchor-spacer"></div><h5 class="article-section__title" id="_i28"> 1-(3,5-Bis(trifluoromethyl)phenyl)-3-((5-(benzylthio)-1,3,4-oxadiazol-2-yl)methyl)thiourea (<b>3a</b>)</h5><div class="NLM_p last">White solid; yield = 160 mg (65%); mp 190.2–191.0 °C; IR (neat) ν<sub>max</sub>, cm<sup>–1</sup>: 1690 (C═S),1655 (C═N), 3196; <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): 8.17–8.12 (m, 3 H), 7.26–7.19 (m, 5 H), 4.47 (s, 2H), 3.93 (s, 2 H); <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>): 181.1, 170.4, 149.1, 139.2, 136.8, 130.6 (q, <i>J</i> = 31.7 Hz), 130.1, 129.0, 128.7, 127.2, 123.4 (q, <i>J</i> = 271.6 Hz),121.9, 46.5, 32.5; HRMS (ESI): calcd for C<sub>19</sub>H<sub>14</sub>F<sub>6</sub>N<sub>4</sub>OS<sub>2</sub> (M + H)<sup>+</sup>, 493.0585; found, 493.0580.<img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01692/20210204/images/medium/jm0c01692_0018.gif" alt="" id="dgr5" /></img></div></div><div id="sec4_1_3_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i29" class="anchor-spacer"></div><h5 class="article-section__title" id="_i29"> 1-((5-(Benzylthio)-1,3,4-oxadiazol-2-yl)methyl)-3-(4-(trifluoromethyl)phenyl)thiourea (<b>3b</b>)</h5><div class="NLM_p last">White solid; Yield = (148 mg, 70%); mp 219.1–220.2 °C; IR (neat) ν<sub>max</sub>, cm<sup>–1</sup>: 1628 (C═N), 1689 (C═S), 3154; <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): 7.83 (d, <i>J</i> = 7.5 Hz, 2 H), 7.60 (d, <i>J</i> = 7.5 Hz, 2 H), 7.25–7.19 (m, 5 H), 4.51 (s, 2 H), 3.95 (s, 2 H); <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>): 181.4, 169.8, 150.1, 139.2, 138.4, 129.9, 128.7, 128.4, 128.2, 126.8, 125.6, 124.1 (q, <i>J</i> = 270.8 Hz), 46.2, 31.9; HRMS (ESI): calcd for C<sub>18</sub>H<sub>15</sub>F<sub>3</sub>N<sub>4</sub>OS<sub>2</sub> (M + H)<sup>+</sup>, 425.0712; found, 425.0713.<img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01692/20210204/images/medium/jm0c01692_0019.gif" alt="" id="dgr6" /></img></div></div><div id="sec4_1_3_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i30" class="anchor-spacer"></div><h5 class="article-section__title" id="_i30"> 1-((5-(Benzylthio)-1,3,4-oxadiazol-2-yl)methyl)-3-(4-nitrophenyl)thiourea (<b>3c</b>)</h5><div class="NLM_p last">White solid; yield = 136 mg (68%); mp 193.8–194.2 °C; IR (neat) ν<sub>max</sub>, cm<sup>–1</sup>: 1688 (C═S),1635 (C═N), 3154; <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): 8.30 (d, <i>J</i> = 8.5 Hz, 2 H), 7.67 (d, <i>J</i> = 8.5 Hz, 2 H), 4.50 (s, 2 H), 3.93 (s, 2 H); <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>): 181.4, 170.1, 149.9, 146.8, 140.9, 139.5, 130.7, 129.1, 128.8, 127.2, 124.1, 46.6, 32.3; HRMS (ESI): calcd for C<sub>17</sub>H<sub>15</sub>N<sub>5</sub>O<sub>3</sub>S<sub>2</sub> (M + H)<sup>+</sup>, 402.0689; found, 402.0680.</div></div></div><div id="sec4_1_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> General Procedure for Synthesis of Compounds <b>4a–4i</b> (<a class="ref internalNav" href="#sch1" aria-label="Schemes 1">Schemes 1</a> and <a class="ref internalNav" href="#sch2" aria-label="2">2</a>)</h4><div class="NLM_p">Trifluoroacetic acid (0.5 mL) was dropwise added to a stirred solution of <b>2a–c</b> (0.5 mmol) in dry dichloromethane (2 mL). After 2 h, the volatiles were removed under reduced pressure to afford the trifluoroacetate salt. A solution of the crude trifluoroacetate salt (0.5 mmol) in dichloromethane (5 mL) was cooled in an ice-water bath. To it was added Et<sub>3</sub>N (0.1 mL, 0.55 mmol), followed by the addition of respective isocyanates (0.5 mmol). After 15 min of stirring at room temperature, the mixture was concentrated under reduced pressure and the residue was taken in ethyl acetate. The solution was washed with water and concentrated under reduced pressure. The crude product was purified by column chromatography to afford compounds <b>4a–i</b>.<img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01692/20210204/images/medium/jm0c01692_0020.gif" alt="" id="dgr7" /></img></div><div id="sec4_1_4_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i32" class="anchor-spacer"></div><h5 class="article-section__title" id="_i32"> 1-(3,5-Bis(trifluoromethyl)phenyl)-3-((<i>S</i>)-(5-(benzylthio)-1,3,4-oxadiazol-2-yl)(Phenyl)Methyl)urea (<b>4a</b>)</h5><div class="NLM_p last">Yield: 207 mg (75%), white solid; mp 128–129 °C; [α]<sub class="stack">D</sub><sup class="stack">25</sup> = −2.5 (<i>c</i> 1.06, CHCl<sub>3</sub>); IR (neat) ν<sub>max</sub>, cm<sup>–1</sup>: 1622 (C═N), 1694 (C═O), 3360; <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): 8.85 (s, 1 H), 7.95–7.88 (m, 2 H), 7.58, (d, <i>J</i> = 7.5 Hz, 1 H), 7.45 (s, 1 H), 7.36–7.27 (m, 5 H), 7.26–7.24 (m, 5 H), 6.55 (d, <i>J</i> = 7.5 Hz, 1H), 4.38 (br s, 2 H); <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>): 168.6, 165.5, 153.9, 140.9, 136.6, 134.5, 132.3 (q, <i>J</i> = 33.0 Hz), 129.4, 129.1, 128.9, 128.8, 128.4, 126.8, 123.3 (q, <i>J</i> = 270.8 Hz), 118.3, 115.6, 50.3, 37.3; HRMS (ESI): calcd for C<sub>25</sub>H<sub>18</sub>F<sub>6</sub>N<sub>4</sub>O<sub>2</sub>S (M + H)<sup>+</sup>, 553.1127; found, 553.1124.<img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01692/20210204/images/medium/jm0c01692_0021.gif" alt="" id="dgr8" /></img></div></div><div id="sec4_1_4_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i33" class="anchor-spacer"></div><h5 class="article-section__title" id="_i33"> 1-((<i>S</i>)-(5-(Benzylthio)-1,3,4-oxadiazol-2-yl)(phenyl)methyl)-3-(4-methoxyphenyl)urea (<b>4b</b>)</h5><div class="NLM_p last">White solid; yield = 123 mg (55%); mp 158.3–159.1 °C; [α]<sub class="stack">D</sub><sup class="stack">25.2</sup> = +17.6 (<i>c</i> 1.05, CHCl<sub>3</sub>); IR (neat) ν<sub>max</sub>, cm<sup>–1</sup>: 1596 (C═N), 1666 (C═O), 3347; <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): 7.76 (s, 1 H), 7.29 (br s, 5 H), 7.25–7.24 (m, 3 H), 7.20 (d, <i>J</i> = 9.0 Hz, 2 H), 6.98 (br s,1 H), 6.76 (d, <i>J</i> = 9.0 Hz, 2 H), 6.46 (d, <i>J</i> = 8.5 Hz, 1 H), 4.34 (ABq, <i>J</i> = 19.0, 13.0 Hz, 2 H), 3.73 (s, 3 H): <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>): 167.9, 164.9, 156.4, 155.2, 137.2, 135.1, 131.3, 129.1, 128.9, 128.8, 128.6, 128.1, 127.1, 123.0, 114.4, 55.5, 50.7, 37.0; HRMS (ESI): calcd for C<sub>24</sub>H<sub>22</sub>N<sub>4</sub>O<sub>3</sub>S (M + H)<sup>+</sup>, 447.1485; found, 447.1485.<img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01692/20210204/images/medium/jm0c01692_0022.gif" alt="" id="dgr9" /></img></div></div><div id="sec4_1_4_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i34" class="anchor-spacer"></div><h5 class="article-section__title" id="_i34"> 1-((<i>S</i>)-(5-(Benzylthio)-1,3,4-oxadiazol-2-yl)(phenyl)methyl)-3-(4-nitrophenyl)urea (<b>4c</b>)</h5><div class="NLM_p last">White solid; yield = 160 mg (70%); mp 186–187 °C; [α]<sub class="stack">D</sub><sup class="stack">25.8</sup> = +28.2 (<i>c</i> 0.96, EtOAc); IR (neat) ν<sub>max</sub>, cm<sup>–1</sup>: 1603 (C═N), 1699 (C═O), 3363; <sup>1</sup>H NMR (200 MHz, DMSO): 9.52 (s, 1 H), 8.60 (d, <i>J</i> = 3.6 Hz, 2 H), 8.18 (d, <i>J</i> = 3.6 Hz, 2 H), 7.94 (d, <i>J</i> = 3.0 Hz, 2 H), 7.89–7.83 (m, 5 H), 7.75–7.70 (m, 2 H), 6.83–6.82 (m, 1 H), 4.94, 4.92 (ABq, <i>J</i> = 5.6 Hz, 2 H).·<sup>13</sup>C NMR (50 MHz, DMSO): 167.6, 164.1, 154.0, 146.7, 141.5, 137.2, 136.9, 129.5, 129.4, 129.1, 129.0, 128.2, 127.8, 125.6, 117.7, 50.4, 36.4; HRMS (ESI): calcd for C<sub>23</sub>H<sub>19</sub>N<sub>5</sub>O<sub>4</sub>S (M + H)<sup>+</sup>, 462.1230; found, 462.1230.<img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01692/20210204/images/medium/jm0c01692_0023.gif" alt="" id="dgr10" /></img></div></div><div id="sec4_1_4_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i35" class="anchor-spacer"></div><h5 class="article-section__title" id="_i35"> 1-((<i>S</i>)-(5-(Benzylthio)-1,3,4-oxadiazol-2-yl)(phenyl)methyl)-3-<i>p</i>-tolylurea (<b>4d</b>)</h5><div class="NLM_p last">White solid; yield = 125 mg (58%); mp 118.1–119.0 °C; [α]<sub class="stack">D</sub><sup class="stack">25</sup> = −16.5 (<i>c</i> 1.05, CHCl<sub>3</sub>); IR (neat) ν<sub>max</sub>, cm<sup>–1</sup>: 1592 (C═N), 1638 (C═O), 3300; <sup>1</sup>H NMR (200 MHz, CDCl<sub>3</sub>): 8.06 (s, 1 H), 7.31–7.29 (m, 10 H), 7.23 (d, <i>J</i> = 8.4 Hz, 2 H), 7.02 (d, <i>J</i> = 8.4 Hz, 2 H), 6.53–6.48 (m, 1 H), 4.37 (s, 2 H), 2.28 (s, 3 H); <sup>13</sup>C NMR (50 MHz, CDCl<sub>3</sub>): 168.1, 165.0, 155.0, 137.2, 136.1, 134.9, 132.8, 129.5, 129.1, 129.0, 128.9, 128.6, 128.2, 127.1, 120.3, 50.5, 37.0, 20.8; HRMS (ESI): calcd for C<sub>24</sub>H<sub>22</sub>N<sub>4</sub>O<sub>2</sub>S (M + H)<sup>+</sup>, 431.1563; found, 431.1536.<img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01692/20210204/images/medium/jm0c01692_0024.gif" alt="" id="dgr11" /></img></div></div><div id="sec4_1_4_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i36" class="anchor-spacer"></div><h5 class="article-section__title" id="_i36"> 1-(3,5-Bis(trifluoromethyl)phenyl)-3-((<i>S</i>)-1-(5-(benzylthio)-1,3,4-oxadiazol-2-yl)-2phenylethyl)urea (<b>4e</b>)</h5><div class="NLM_p last">Colorless crystal; yield = 220 mg (78%); mp 167–168 °C; [α]<sub class="stack">D</sub><sup class="stack">25</sup> = −53.0 (<i>c</i> 1.01, CHCl<sub>3</sub>); IR (neat) ν<sub>max</sub>, cm<sup>–1</sup>: 1581 (C═N), 1658 (C═O), 3304; <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): 8.48 (s, 1 H), 7.75 (s, 2 H), 7.37 (t, <i>J</i> = 7.5 Hz, 3 H), 7.34–7.26 (m, 3 H), 7.18 (br s, 3 H), 6.96 (d, <i>J</i> = 4.0 Hz, 2 H), 6.85 (d, <i>J</i> = 8.5 Hz, 1 H), 5.6–5.5 (m, 1 H), 4.41 (ABq, <i>J</i> = 17.5,13.0 Hz, 2 H), 3.05 (dd, <i>J</i> = 13.5, 6.0 Hz, 1 H), 2.87 (dd, <i>J</i> = 13.5, 6.0 Hz, 1 H); <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>): 169.1, 165.4, 153.9, 140.7, 134.8, 134.7, 131.8 (q, <i>J</i> = 29.7 Hz), 129.3, 129.0, 128.9, 128.7, 128.5, 127.5, 123.2 (q, <i>J</i> = 269.0 Hz), 118.0, 115.5, 47.8, 40.2, 37.0; HRMS (ESI): calcd for C<sub>26</sub>H<sub>20</sub>F<sub>6</sub>N<sub>4</sub>O<sub>2</sub>S (M + H)<sup>+</sup>, 567.1283; found, 567.1284.<img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01692/20210204/images/medium/jm0c01692_0025.gif" alt="" id="dgr12" /></img></div></div><div id="sec4_1_4_6" class="NLM_sec NLM_sec_level_4"><div id="ac_i37" class="anchor-spacer"></div><h5 class="article-section__title" id="_i37"> 1-((<i>S</i>)-1-(5-(Benzylthio)-1,3,4-oxadiazol-2-yl)-2-phenylethyl)-3-(4-methoxyphenyl)urea (<b>4f</b>)</h5><div class="NLM_p last">White solid; yield = 126 mg (55%); mp 163–164 °C; [α]<sub class="stack">D</sub><sup class="stack">25.1</sup> = −17.05 (<i>c</i> 1.33, CHCl<sub>3</sub>); IR (neat) ν<sub>max</sub>, cm<sup>–1</sup>: 1631 (C═N), 1660 (C═O), 3334; <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): 7.37 (d, <i>J</i> = 7 Hz, 2 H), 7.33–7.27 (m, 3 H), 7.20–7.17 (m, 4 H), 7.12 (d, <i>J</i> = 9 Hz, 2 H), 7.02–7.00 (m, 2 H), 6.77 (d, <i>J</i> = 9.0 Hz, 2 H), 6.04 (d, <i>J</i> = 6.5 Hz, 1 H), 5.53 (dd, <i>J</i> = 15.0, 6.5 Hz, 1 H), 4.4–4.36 (m, 2 H), 3.76 (s, 3 H), 3.15 (dd, <i>J</i> = 14.0, 6.5 Hz, 1 H), 3.06 (dd, <i>J</i> = 14.0, 6.5 Hz, 1 H). <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>): 168.2, 164.5, 156.8, 155.3, 135.4, 135.3, 130.9, 129.4, 129.0, 128.8, 128.6, 128.2, 127.2, 123.8, 114.5, 55.5, 47.9, 40.0, 36.9; HRMS (ESI): calcd for C<sub>25</sub>H<sub>24</sub>N<sub>4</sub>O<sub>3</sub>S (M + H)<sup>+</sup>, 461.1641; found, 461.1642.<img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01692/20210204/images/medium/jm0c01692_0026.gif" alt="" id="dgr13" /></img></div></div><div id="sec4_1_4_7" class="NLM_sec NLM_sec_level_4"><div id="ac_i38" class="anchor-spacer"></div><h5 class="article-section__title" id="_i38"> 1-((<i>S</i>)-1-(5-(Benzylthio)-1,3,4-oxadiazol-2-yl)-2-phenylethyl)-3-(4-nitrophenyl)urea (<b>4g</b>)</h5><div class="NLM_p last">White solid; yield = 171 mg (72%); mp 190–192 °C; [α]<sub class="stack">D</sub><sup class="stack">25.2</sup> = −23.3 (<i>c</i> 1.04, EtOAc); IR (neat) ν<sub>max</sub>, cm<sup>–1</sup>: 1608 (C═N), 1676 (C═O), 3350; <sup>1</sup>H NMR (200 MHz, DMSO): 9.40 (s, 1H), 8.13 (d, <i>J</i> = 9.2 Hz, 2 H), 7.58 (d, <i>J</i> = 9.2 Hz, 2 H), 7.44–7.39 (m, 2 H), 7.35–7.16 (m, 9 H), 5.33–5.25 (m, 1 H), 4.46 (s, 2 H), 3.28–3.10 (m, 2H); <sup>13</sup>C NMR (50 MHz, DMSO): 168.1, 163.6, 154.2, 146.9, 141.3, 137.0, 136.8, 129.7, 129.4, 129.0, 128.9, 128.2, 127.3, 125.6, 117.7, 47.8, 38.3, 36.3; HRMS (ESI): calcd for C<sub>24</sub>H<sub>21</sub>N<sub>5</sub>O<sub>4</sub>S (M + H)<sup>+</sup>, 476.1387; found, 476.1385.<img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01692/20210204/images/medium/jm0c01692_0027.gif" alt="" id="dgr14" /></img></div></div><div id="sec4_1_4_8" class="NLM_sec NLM_sec_level_4"><div id="ac_i39" class="anchor-spacer"></div><h5 class="article-section__title" id="_i39"> 1-((<i>S</i>)-1-(5-(Benzylthio)-1,3,4-oxadiazol-2-yl)-2-phenylethyl)-3-<i>p</i>-tolylurea (<b>4h</b>)</h5><div class="NLM_p last">White solid; yield = 133 mg (60%); mp 165–166 °C; [α]<sub class="stack">D</sub><sup class="stack">24.9</sup> = −26.1 (<i>c</i> 1.04, CHCl<sub>3</sub>); IR (neat) ν<sub>max</sub>, cm<sup>–1</sup>:1593 (C═N), 1665 (C═O), 3347; <sup>1</sup>H NMR (200 MHz, CDCl<sub>3</sub>): 7.36–7.35 (m, 3 H), 7.32–7.28 (m, 3 H), 7.19–7.18 (m, 3 H), 7.12 (d, <i>J</i> = 3.2 Hz, 2 H), 7.02–7.00 (m, 4 H), 6.21 (d, <i>J</i> = 3.6 Hz, 1 H), 5.54 (dd, <i>J</i> = 2.8 Hz, 1 H), 4.38, 4.35 (ABq, <i>J</i> = 5.2 Hz, 2 H), 3.14 (dd, <i>J</i> = 5.6, 2.8 Hz, 1 H), 3.06 (dd, <i>J</i> = 5.6, 2.8 Hz, 1 H), 2.27 (s, 3 H); <sup>13</sup>C NMR (50 MHz, CDCl<sub>3</sub>): 168.3, 164.5, 154.9, 135.8, 135.4, 135.2, 133.3, 129.6, 129.4, 129.0, 128.8, 128.6, 128.2, 127.2, 121.0, 47.9, 40.1, 36.9, 20.7; HRMS (ESI): calcd for C<sub>25</sub>H<sub>24</sub>N<sub>4</sub>O<sub>2</sub>S (M + H)<sup>+</sup>, 445.1692; found, 445.1693.<img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01692/20210204/images/medium/jm0c01692_0028.gif" alt="" id="dgr15" /></img></div></div><div id="sec4_1_4_9" class="NLM_sec NLM_sec_level_4"><div id="ac_i40" class="anchor-spacer"></div><h5 class="article-section__title" id="_i40"> 1-(3,5-Bis(trifluoromethyl)phenyl)-3-((5-(benzylthio)-1,3,4-oxadiazol-2-yl)methyl)urea (<b>4i</b>)</h5><div class="NLM_p last">White solid; yield = 161 mg (68%); mp 180.1–180.9 °C; IR (neat) ν<sub>max</sub>, cm<sup>–1</sup>: 1625 (C═N), 1658 (C═O), 3365; <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): 8.66 (s, 1 H), 7.83 (br s, 2 H), 7.38 (s, 1 H), 7.30–7.27 (m, 2 H), 7.25–7.22 (2 H), 4.69 (s, 2 H), 4.36 (s, 2 H); <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>): 166.7, 166.1, 154.7, 140.5, 134.2, 131.9 (q, <i>J</i> = 32.5 Hz), 128.9, 128.9, 128.4, 123.1 (q, <i>J</i> = 271.3 Hz), 118.1, 115.8, 36.9, 35.4; HRMS (ESI): calcd for C<sub>19</sub>H<sub>14</sub>F<sub>6</sub>N<sub>4</sub>O<sub>2</sub>S (M + H)<sup>+</sup>, 477.0814; found, 477.0812.</div></div></div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i41" class="anchor-spacer"></div><h3 class="article-section__title" id="_i41"> Biological Studies</h3><div id="sec4_2_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42"> Materials</h4><div class="NLM_p last">All chemicals were procured from Sigma chemicals (St. Louis, MO), unless specified. Dulbecco’s modified Eagle’s medium (DMEM), fetal bovine serum (FBS) from Gibco Life Technologies (Carlsbad, CA). Lipofectamine, secondary antibodies tagged with Alexa fluor, and Prolong Diamond anti-fade reagent were from Invitrogen (Carlsbad, CA). Following antibodies were used in the current study: for RECQL1 (#A300-450A, Bethyl), BLM (#sc-365753, SCBT), WRN (#A300-238A Bethyl), RECQL5 (#sc-515150, SCBT; #5847S, CST), γ-H2AX (#H-5912, Sigma), H2AX (#SAB4501369, Sigma), RPA32 (#A300-24<b>4a</b>, Bethyl), Ku 80 (#2180, CST), DNA-PKc (# 4602, CST), p-MRE11 (#8344, CST), MRE11 (# 8344, CST), p-NBS1 (#8344, CST), NBS1 (#8344, CST), RAD50 (#GTX70228, Genetex; #8344, CST), RAD51 (#sc-8349, SCBT; #PC 130, Sigma), ATM (#sc23921, SCBT), p-ATM (#5883, CST), ATR (Calbiochem, #PC 538), p-ATR (#2853, CST), CHK2 (#sc17748, SCBT), p-CHK2 (#2197, CST), CHK1 (sc-8408 SCBT), p-CHK1 (#2348, CST), p-p53 (#9286, CST), FLAG (#F3165, Sigma), β-actin (#ab228001, #ab8227, Abcam), α-tubulin (#T6199, Sigma) and anti-FLAG M2 magnetic beads (#M8823, Sigma). Following western blotting material used in the study: Lumi-Light Plus western blotting kit (Roche Applied Science, Baden-Wurttemberg, Mannheim) and nitrocellulose membrane (Pall Life Sciences, Easthills, NY).</div></div><div id="sec4_2_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i43" class="anchor-spacer"></div><h4 class="article-section__title" id="_i43"> Cell Culture</h4><div class="NLM_p last">U2-OS cell line was procured from American Type Culture Collection, MCF-7, T47D, and MDA-MB-231 cells were purchased from European Collection of Authenticated Cell Cultures. The cells were cultured in DMEM medium supplemented with FBS (10%), penicillin (100 U/mL), and streptomycin (100 μg/mL). The cells were grown in an incubator (37 °C, 5% CO<sub>2</sub>).</div></div><div id="sec4_2_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i44" class="anchor-spacer"></div><h4 class="article-section__title" id="_i44"> MTT Assay for Cell Viability</h4><div class="NLM_p last">Cells (4 × 10<sup>3</sup> cells/well) were seeded in 96-well plates and allowed to grow overnight. The cells were treated with the vehicle [<0.1% dimethyl sulfoxide (DMSO)] or various concentrations of test compounds for 72 h. 3-(4,5-Dimethylthiazol-2-yl)-2,5-diphenyltetrazolium bromide (MTT) solution (0.5 mg/mL) was added for 6 h. The formazan crystals in the viable cells were solubilized with 0.01 N HCl (100 μL) with 10% sodium dodecyl sulphate (SDS) and the absorbance was read at 550 nm in a multiplate reader.</div></div><div id="sec4_2_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i45" class="anchor-spacer"></div><h4 class="article-section__title" id="_i45"> Clonogenic Survival Assay</h4><div class="NLM_p last">The cells (500/well) were seeded in 6-well plates and allowed to grow overnight. The cells were incubated with the vehicle (0.1% DMSO) or different concentrations of test compounds for 8–10 days in the complete growth media. The colonies were fixed with methanol and stained with 0.5% crystal violet in 1:1 methanol–water. The colonies were counted, and images of the colonies scanned. The surviving fractions were determined from the colony counts after correcting for the plating efficiency of the nontreated controls.</div></div><div id="sec4_2_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i46" class="anchor-spacer"></div><h4 class="article-section__title" id="_i46"> Cell Cycle and Sub-G1 Analyses by Flow Cytometry</h4><div class="NLM_p last">The assay was carried out following a reported method with minor modifications.<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> The cells (1 × 10<sup>5</sup> cells/well), grown overnight, were treated with <b>4a</b> for the indicated time period. Cells were washed with cold phosphate-buffered saline (PBS), trypsinized, and incubated in hypotonic buffer (0.1% sodium citrate, 0.1% Triton X-100) containing propidium iodide (400 μg/mL) and RNase A (200 μg/mL) for 30 min. Further, cells (at least 2 × 10<sup>4</sup>) were acquired with a Partec CyFlo flow cytometer. Flow cytometer data were analyzed by FlowJo software.</div></div><div id="sec4_2_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47"> Cell Lysate Preparations and Immunoblot Assay</h4><div class="NLM_p last">After indicated treatments, cells were lysed in a lysis buffer [Tris (20 mM, pH 7.4), NaCl (250 mM), ethylenediaminetetraacetic acid (EDTA) (2 mM pH 8.0), Triton X-100 (0.5%), aprotinin (0.01 μg/mL), leupeptin (0.01 μg/mL), PMSF (0.4 mM), and Na<sub>3</sub>VO<sub>4</sub> (4 mM)]. The lysates were centrifuged (16,500<i>g</i>, 10 min) and the supernatants were collected to obtain the whole cell extracts. Different cell fractions were isolated after following a reported protocol.<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> Briefly, control and treated cells were lysed in buffer A [HEPES (10 mM, pH 7.9), glycerol (10%), sucrose (0.34 M) KCl, MgCl<sub>2</sub> (1.5 mM), EDTA (0.2 mM), Triton X-100 (0.1%), and protease inhibitor cock-tail. After 20 min incubation, samples were centrifuged (4 min, 1300<i>g</i>, 4 °C) to obtain the nuclear pellet. Nuclei were washed once in buffer A, and then lysed in buffer B [EDTA (3 mM), ethylene glycol tetraacetic acid (EGTA) (0.2 mM), dithiothreitol (1 mM), and protease inhibitors cock-tail. The lysate was centrifuged (4 min, 1700<i>g</i>, 4 °C). The supernatant was considered as soluble nuclear fractions (F2) while pellet contains insoluble chromatin fraction (F3). This chromatin pellet (F3) was washed once with buffer B and suspended in buffer A plus CaCl<sub>2</sub> (1 mM) and micrococcal nuclease (0.2 U, Sigma). After incubation at 37 °C for 5 min, the nuclease reaction was stopped by the addition of 1 mM EGTA, and the lysate was considered as a chromatin fraction (F4). For the total nuclear fraction, F2 and F4 were pooled together. The cell lysates were separated by 8–15% SDS-polyacrylamide gel electrophoresis and transferred to a nitrocellulose membrane. The membranes were blocked for 1 h at room temperature in TBST buffer [Tris–HCl (20 mM) pH 7.6, NaCl (137 mM), and Tween-20 (0.1%)] containing 5% (w/v) nonfat milk and incubated overnight at 4 °C with their respective specific primary antibodies. After several washes, suitable horseradish peroxidase-conjugated secondary antibodies were added, the membranes were incubated further for 2 h, and the blots were developed using a Lumi-Light Plus western blotting kit. Protein bands detected using Syngene GBox XX6 and GeneSys software and the intensity ratios of immunoblots to that of normal control, taken as 1 (arbitrary unit) were quantified after normalizing with respect to the loading controls.</div></div><div id="sec4_2_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i48" class="anchor-spacer"></div><h4 class="article-section__title" id="_i48"> Knockout, Knockdown, and Ectopic Expression</h4><div class="NLM_p last">All the CRISPR-CAS9 double nickase plasmids (Control and RECQL5) and lentivirus-expressing shRNA (control, RECQL1, RECQL5, WRN, and BLM) were purchased from Santa Cruz Biotechnology (Dallas, TX, USA). Exponentially growing MCF-7 cells were transfected using lipofectamine 3000 and CRISPR-CAS9 plasmids. For knockdown, exponentially growing MCF-7/U2-OS cells were treated with lentivirus (control, RECQL1, RECQL5, WRN, or BLM shRNA) and polybrene (20 μg/mL). Further, cells were selected in the presence of puromycin (1 μg/mL) and stable knockout and knockdown efficiency was assessed by western blotting. For ectopic expression, exponentially growing cells were transiently transfected with EVs, RECQL5 wildtype, and mutant protein expressing plasmids by the standard calcium phosphate method for 16 h, and cells were allowed to grow for another 4 h in complete growth medium. Ectopic expression was assessed by western blotting or fluorescence microscopy.</div></div><div id="sec4_2_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i49" class="anchor-spacer"></div><h4 class="article-section__title" id="_i49"> Immunoprecipitation</h4><div class="NLM_p last">Respective RECQL5-KO cells (1 × 10<sup>7</sup> cells) were transfected with EVs and RECQL5 (WT)-FLAG by calcium phosphate method as mentioned above. After treatment, chromatin fractions were prepared as mentioned above. Further, chromatin extracts (1 mg) were subjected to immunoprecipitation (IP) with anti-FLAG M2 magnetic beads, in the presence of EtBr (10 μg/mL), overnight under rotation (4 °C). Beads were extensively washed with lysis buffer. Input and IP lysates were subjected to western blotting of RECQL5, FLAG, and RAD51 proteins.</div></div><div id="sec4_2_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i50" class="anchor-spacer"></div><h4 class="article-section__title" id="_i50"> Immunofluorescence Staining and Microscopy</h4><div class="NLM_p last">Cells (7.5 × 10<sup>4</sup> cells/well) were seeded in a 6-well plate containing glass cover slips. Cells were treated for the indicated time, washed with PBS, and fixed by chilled methanol. The fixed cells were washed twice with PBS and permeabilized with PBST (0.1% Tween 20) for about 10 min. Further, the cells were blocked with 5% bovine serum albumin (BSA) for 2 h, incubated with the respective primary antibodies (1:6000, 2.5% BSA) overnight (4 °C). After washing with PBST, secondary antibodies tagged with Alexa Fluor-488/Alexa Fluor-594 with suitable isotype controls (1:2000, 2.5% BSA) were added (3 h, RT). Cells were washed twice with PBST, coverslips were dried and mounted on slides with Prolong Diamond anti-fade reagent (containing DAPI). Mounted slides were then analyzed with a confocal microscope (LSM 780, Carl Zeiss, Germany). Image analysis was performed using Zeiss Zen software. Approximately 80–100 nuclei, in duplicates, were analyzed for each sample in 3–5 experiments.</div></div><div id="sec4_2_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i51" class="anchor-spacer"></div><h4 class="article-section__title" id="_i51"> RECQL5 Cloning, Expression, and Purification</h4><div class="NLM_p last">Full length ORFs of RECQL5 (WT and K58M mutant) were obtained by PCR amplification from recombinant pCMV6-AN-His-DDK plasmid (#RC209460, Origene, USA) as the template using the primers 1 and 2 (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01692/suppl_file/jm0c01692_si_001.pdf" class="ext-link">Table S1</a>). The PCR product was inserted in pET29 in the <i>Nde</i>I and <i>XhoI</i> sites. After verifying the construct, <i>E. coli</i> Rosetta-gami-2 cells were transformed with the recombinant plasmid. A batch culture (1000 mL) of these bacteria were induced with IPTG (300 μM, 3 h, RT) and harvested. The cell pellet was resuspended in equilibration buffer A [Tris–HCl (25 mM, pH. 8), NaCl (300 mM), MgCl<sub>2</sub> (2 mM), β-mercaptoethanol (BME) (2 mM), glycerol (3%), PMSF (2 mM), and protease inhibitor cocktail (Roche). After initial treatment with lysozyme (200 μg/mL) and benzonase (1 U) for 30 min, the suspension was sonicated for 5 min (30% amplitude; 4 s pulse). The cell lysate was centrifuged (8000<i>g</i>, 30 min) and the clear lysate was mixed with the pre-equilibrated Ni-NTA resin (0.5 mL, 30 min, 4 °C; Qiagen) in a rotary mixer. The mixture was further loaded in a column and the resin was repeatedly washed equilibration buffer containing imidazole (30 mM). The bound protein was eluted from the column by repeated washing with equilibration buffer containing 300 mM imidazole (6 × 0.5 mL). Excess imidazole was removed using a PD10 desalting column. The protein was then concentrated and equilibrated with buffer containing NaCl (600 mM, pH 7.6) and passed through a superdex 200 Gel Filtration column. The fraction eluting at ∼80–120 kDa was collected, concentrated and equilibrated with buffer B [Tris–HCl (25 mM, pH. 7.6), NaCl (50 mM), MgCl<sub>2</sub> (2 mM) BME (2 mM), glycerol (3%)]. Protein was characterized by western blotting with anti-HIS and anti-RECQL5.</div></div><div id="sec4_2_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i52" class="anchor-spacer"></div><h4 class="article-section__title" id="_i52"> Generation of Mutants RECQL5 by Site-Directed Mutagenesis</h4><div class="NLM_p last">Full length ORF of RECQL5 with <i>EcoR</i>I and <i>Kpn</i>I sites was obtained by PCR amplification from the recombinant pCMV6-AN-His-DDK plasmid, by using primer 3 and 4 (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01692/suppl_file/jm0c01692_si_001.pdf" class="ext-link">Table S1</a>). The PCR product was inserted in a p3X FLAG-CMV7.1 vector. The fusion PCR protocol was used to generate the RECQL5 (F666A)-FLAG mutant. By using p3X-RECQL5-FLAG-CMV7.1 as a template, two sets of parallel PCR reactions were performed. One with primers 3 and 6 and other with primers 5 and 4 (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01692/suppl_file/jm0c01692_si_001.pdf" class="ext-link">Table S1</a>). The amplified products (approx. 2 and 1 kb) were gel purified and PCR amplified with primers 3 and 4 to obtain the RECQL5 (F666A)-FLAG mutant product. This product was cloned in the p3X FLAG-CMV7.1 vector. The F666A mutation was confirmed by sequencing by using primers 7 and 8. For RECQL5-RFP, full length RECQL5 ORF from the pCMV6-AN-His-DDK plasmid was shuffled to the pCMV-AC-RFP vector (Origene, USA). Further, the helicase dead RECQL5 (K58M)-RFP mutant was obtained with primers 9 and 10 (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01692/suppl_file/jm0c01692_si_001.pdf" class="ext-link">Table S1</a>), following QuikChange II XL Site-Directed Mutagenesis kit (#200521, Agilent, CA, USA).</div></div><div id="sec4_2_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i53" class="anchor-spacer"></div><h4 class="article-section__title" id="_i53"> RECQL5 Helicase Assay</h4><div class="NLM_p last">The forked substrate for the helicase assay was prepared by annealing 5′ FAM labeled oligo with its complimentary 3′ BHQ1 labeled oligo (equimolar 30 mer Oligo1 and 2, <a href="/doi/suppl/10.1021/acs.jmedchem.0c01692/suppl_file/jm0c01692_si_001.pdf" class="ext-link">Table S1</a>). The mixture was heated to 95 °C for 5 min and slowly cooled down to 25 °C. For the helicase assay, purified RECQL5 protein (20 nM; WT) was incubated with different concentrations of <b>4a</b> in reaction buffer [Tris–HCl (10 mM, pH 8.0), MgCl<sub>2</sub> (2 mM), glycerol (1%), and BSA (100 μg/mL)] at 4 °C for 30 min. Finally, duplex forked DNA (20 nM) and ATP (2 mM) were added and the FAM fluorescence was measured in a fluorescence multiplate reader (λ<sub>ex</sub>: 495 nm and λ<sub>em</sub>: 520 nm) at different time points. Helicase activity of RECQL5-WT and RECQL5-K58M proteins was also compared by employing the above assay.</div></div><div id="sec4_2_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i54" class="anchor-spacer"></div><h4 class="article-section__title" id="_i54"> DNA Binding Activity Assay</h4><div class="NLM_p last">Oligonucleotide strand (Oligo 3, 5 pmol) was radiolabeled with [γ<sup>32</sup>P] at its 5′ end. Labeling was performed by incubating the oligo, ATP (γ<sup>32</sup>P; 3000 Ci/mmol), and polynucleotide kinase in polynucleotide kinase buffer (New England Biolabs, Inc). Labeled oligo was purified by G25 column (GE, USA). The linear dsDNA substrate was prepared by mixing equimolar oligo 3 (radio-labelled) and oligo 4 (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01692/suppl_file/jm0c01692_si_001.pdf" class="ext-link">Table S1</a>) in TE buffer [Tris–HCl (10 mM, pH 7.6, 1 mM EDTA)], heating at 95 °C for 5 min, and allowed slowly to cool down to 25 °C. The purified RECQL5 protein (50–200 nM) was incubated with <b>4a</b> (0–10 μM) in buffer [Tris–HCl (10 mM, pH 7.6), NaCl (50 mM), KCl (25 mM), BSA (100 μg/mL), and BME (2 mM)] for 30 min. Further, the dsDNA substrate was added and incubated for additional 15 min. The products were resolved on 6% native-polyacrylamide gel made with TBE (0.5×) and KCl (50 mM) by electrophoresis in 0.5× TBE containing KCl (50 mM). The gels were then dried and the autoradiograms were developed on a X-ray film.</div></div><div id="sec4_2_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i55" class="anchor-spacer"></div><h4 class="article-section__title" id="_i55"> RECQL5 ATPase Assay</h4><div class="NLM_p last">ATP consumption by RECQL5 ATPase activity was assessed by an ATP determination kit (#A22066, Thermo Scientific). Briefly, purified RECQL5 protein (200 nM) was incubated with <b>4a</b> (0.1–10 μM) for 60 min in reaction buffer [Tris–HCl (10 mM, pH. 7.4), BSA (0.1%)] at 4 °C. Further, forked dsDNA (50 nM), luciferin, and luciferase were added to the reaction mixture for 15 min. Finally, ATP (2 mM) was added for 1 h and read at 560 nm in a multiplate fluorescence reader.</div></div><div id="sec4_2_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i56" class="anchor-spacer"></div><h4 class="article-section__title" id="_i56"> Development of Cisplatin-Resistant Breast Cancer Cells</h4><div class="NLM_p last">This was developed as per the reported protocol.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> The MCF-7 cells, seeded at 70–80% confluence for 16 h in a T-25 flask, were treated with a high CDDP concentration (10 μM) for 2 days. After 2 days, cells were washed and allowed to grow in CDDP free regular medium for another 2 days. The above procedure was repeated for 4–6 weeks and cells were analyzed for cell survival by the MTT assay. CDDP-resistant MCF-7 cells were named MCF-7<sup>CDDP</sup> cells.</div></div><div id="sec4_2_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i57" class="anchor-spacer"></div><h4 class="article-section__title" id="_i57"> <b>4a</b> Treatment in the Xenograft Tumor Model in NUDE Mice</h4><div class="NLM_p last">Female NUDE mice (approx. 8 weeks) were procured from Advanced Centre for Treatment, Research, and Education in Cancer (ACTREC, Navi Mumbai, India). The due approval for the use of animals for the current study was obtained from the Institutional Animal Ethics Committee (IAEC) of Bhabha Atomic Research Center (Approval number BAEC/16/16). Institutional guidelines were strictly adhered for housing mice in standard individually ventilated cages, diet, water, dark light cycle, and experimental protocol. Exponentially growing RECQL5-WT MCF-7 and RECQL5-KO MCF-7 cells (1 × 10<sup>7</sup> cells/100 μL DMEM) were injected into the right flank of the mice. Subcutaneous injection of β-estradiol (200 ng/0.1 mL PBS) was given on alternate days. After 15 days of injection, mice bearing palpable RECQL5-WT and RECQL5-KO tumors were randomized into four different groups (<i>n</i> = 8), for example, (1) RECQL5-WT (vehicle 100 μL), (2) RECQL5-WT (<b>4a</b> 50 mg/kg/day; 100 μL), (3) RECQL5-KO (vehicle 100 μL), and (4) RECQL5-KO (<b>4a</b> 50 mg/kg/day; 100 μL). <b>4a</b> was prepared in 0.5% DMSO in Neobee M5 oil (Sigma). The above formulation was given through oral gavage on every alternate day for 30 days. TV was evaluated by measuring the perpendicular diameter axes of the tumor with vernier calipers (axes labels-“a”, long axis, and “b”, short axis).<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> Experiments were blinded by allowing two independent groups to deliver the formulation and measure the TV of the coded groups of mice. The body weight of animals was assessed regularly. At the end of the whole experiment, mice were sacrificed by euthanizing the animals by CO<sub>2</sub> asphyxiation in compliance with the standard procedure approved by the IAEC. Briefly, CO<sub>2</sub> cages were used to steadily replace the air with CO<sub>2</sub> (5–20% cage air with CO<sub>2</sub> per minute) for 8 min. After CO<sub>2</sub> treatment, mice were returned to ambient air for 15 min to ensure no recovery. Tumors were excised, weighed, and photographed. The inspection of major morphological changes of organs was also carried out.</div></div><div id="sec4_2_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i58" class="anchor-spacer"></div><h4 class="article-section__title" id="_i58"> Toxicity Studies in the Preclinical Mouse Model</h4><div class="NLM_p last">Mice (20–25 gm, male C15BL16 strain, approval no. BAEC/10/16) were obtained from the BARC animal breeding facility. Guidelines for animal treatment and maintenance were followed as mentioned above. Mice were given a single oral gavage of the vehicle or <b>4a</b> (100 mg/kg body wt; <i>n</i> = 8 mice) and kept under observation for 30 days. Body weight, behavior, stool texture, and food and water intake was monitored every alternate days. After 30 days, animals were sacrificed, as mentioned above, and blood and spleen were collected. The blood serum was subjected to the biochemical analysis of creatinine (mg/dL), ALP (U/L), and SGPT (U/L) with an auto analyzer (Rx Daytona, Randox, crumlin county, Antrim, UK). Total cells from the spleen were removed as per the reported protocol.<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a></div></div><div id="sec4_2_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i59" class="anchor-spacer"></div><h4 class="article-section__title" id="_i59"> HPLC Analysis of <b>4a</b> Concentrations in the Serum</h4><div class="NLM_p last">Nude mice were given oral gavage of vehicle or <b>4a</b> (50 mg/kg body wt <i>n</i> = 3). The blood was collected immediately after sacrificing mice at different time points. Serum samples (100 μL) were mixed with the solvent mixture {methanol: acetonitrile (ACN) (70:30); 200 μL to precipitate proteins. The serum and solvent mixture was intermittently vortexed at the interval of 5 min for 30 min. The mixture was centrifuged (12,000<i>g</i>; 10 min). Solvent extraction was repeated once again with the above process. The clear supernatant, obtained from the above protocol, was subjected to HPLC analysis. HPLC instrument (model 2200, Jasco, Japan) with following parameters were used for the analysis: eluent (ACN/water - 70:30), flow rate (1 mL/min), Eurospher 100 C18, 7 μm column (25 mm × 4.6 mm i.d; Knaur GmbH, Berlin, Germany), and UV detection (254 nm). The <b>4a</b> concentration was measured using a standard curve of <b>4a</b>. The calibration curve was found to be linear over the concentration range of 10–2000 nM. To obtain the serum concentration of <b>4a</b>, dilution factor used during the solvent extraction was adjusted.</div></div><div id="sec4_2_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i60" class="anchor-spacer"></div><h4 class="article-section__title" id="_i60"> <i>In Silico</i> Docking Study</h4><div class="NLM_p last">The crystal structure of human RECQL5 complexed with ADP (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LB3">5LB3</a>) was downloaded from the Protein Data Bank (<a href="https://www.rcsb.org/" class="extLink">https://www.rcsb.org/</a>) and used for molecular docking. ADP was used as a reference ligand and protein hydrogens and charges were added at pH 7, using LeadIT module in BioSolveIT software suite [LeadIT version 2.3.2; BioSolveIT GmbH, Sankt Augustin, Germany, 2019, <a href="http://www.biosolveit.de/LeadIT" class="extLink">www.biosolveit.de/LeadIT</a>]. The compounds were docked into the solvent accessible ATP binding pocket of RECQL5 helicase using LeadIT with a limit of 200 poses per compound. The binding pocket was defined around the ADP molecule within 10 Å limit. The molecule ranking was done using the FlexX scoring function. Top 10 docked poses per compound were listed and scored, and the binding free energy (Δ<i>G</i>) was obtained for the best pose. The docked compounds were visually inspected for H-bonds, VDW clashes, and electrostatic and hydrophobic interactions. The co-crystallized ligand ADP lied close to the two molecules (figure of surface diagrams with ligands superposed). Important amino acid residues of helicase involved in the putative interactions with the ligands were shown in 2D interaction diagrams. The hydrogen bonding interactions were shown in dotted lines. Structural, surface, and superposition figures were made using the PyMOL Molecular Graphics System, Version 2.0 Schrödinger, LLC.</div></div><div id="sec4_2_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i61" class="anchor-spacer"></div><h4 class="article-section__title" id="_i61"> Clinical Data Analysis</h4><div class="NLM_p last">Based on breast cancer database,<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> disease outcomes, that is, tumor-free survival, were assessed by using the online database (<a href="http://www.kmplot.com" class="extLink">www.kmplot.com</a>).</div></div><div id="sec4_2_21" class="NLM_sec NLM_sec_level_3"><div id="ac_i62" class="anchor-spacer"></div><h4 class="article-section__title" id="_i62"> Statistical Analyses</h4><div class="NLM_p last">At least three independent experiments were carried out and values presented as mean ± SEM or mean ± SD. The statistical analysis was performed using GraphPad Prism 5.0. Two-tailed, unpaired, Student’s <i>t</i>-test or ANOVA with Tukey post-hoc analysis was carried out, wherever necessary, to test the statistical significance of the data presented. A value of <i>p</i> < 0.05 was considered significant. For animal experiments, all animal studies were conducted using eight animals per group for each experiment. The statistical significance was determined by using the Student’s <i>t</i>-test.</div></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i63"><a href="/doi/suppl/10.1021/acs.jmedchem.0c01692" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_87423" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_87423" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.0c01692?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01692</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Effect of compounds on cell viability and clonogenic potential; effects of <b>4a</b> on different RECQL5-WT- and RECQL5-depleted breast cancer cells, DNA binding, and ATPase activity, role of <b>4a</b> in DSB generation, apoptosis, NHEJ- and HRR-regulating proteins, stabilizing RECQL5-RAD51 complex, and cell cycle; pharmacokinetic and toxicity effects of <b>4a;</b> HPLC purity; sequence of the primers and DNA oligos;NMR spectra; and oligonucleotides used in this study (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01692/suppl_file/jm0c01692_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">(<a href="/doi/suppl/10.1021/acs.jmedchem.0c01692/suppl_file/jm0c01692_si_002.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">(<a href="/doi/suppl/10.1021/acs.jmedchem.0c01692/suppl_file/jm0c01692_si_003.pdb" class="ext-link">PDB</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01692/suppl_file/jm0c01692_si_001.pdf">jm0c01692_si_001.pdf (2.38 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01692/suppl_file/jm0c01692_si_002.pdb">jm0c01692_si_002.pdb (275.48 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01692/suppl_file/jm0c01692_si_003.pdb">jm0c01692_si_003.pdb (275.72 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.0c01692" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_33854" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_33854" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sunil Kumar Ghosh</span> - <span class="hlFld-Affiliation affiliation">Bio-Organic
Division, Bhabha Atomic Research Centre, Trombay, Mumbai 400085, India</span>; 
    <span class="hlFld-Affiliation affiliation">Homi
Bhabha National Institute, Anushaktinagar, Mumbai 400094, India</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-2508-6181" title="Orcid link">http://orcid.org/0000-0003-2508-6181</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#62000b11030c0922000310014c050d144c0b0c"><span class="__cf_email__" data-cfemail="56343f2537383d163437243578313920783f38">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Birija Sankar Patro</span> - <span class="hlFld-Affiliation affiliation">Bio-Organic
Division, Bhabha Atomic Research Centre, Trombay, Mumbai 400085, India</span>; 
    <span class="hlFld-Affiliation affiliation">Homi
Bhabha National Institute, Anushaktinagar, Mumbai 400094, India</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-9161-5913" title="Orcid link">http://orcid.org/0000-0001-9161-5913</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#76111e0503181f1a361417041558111900581f18"><span class="__cf_email__" data-cfemail="4e29263d3b2027220e2c2f3c2d60292138602720">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Saikat Chakraborty</span> - <span class="hlFld-Affiliation affiliation">Bio-Organic
Division, Bhabha Atomic Research Centre, Trombay, Mumbai 400085, India</span>; 
    <span class="hlFld-Affiliation affiliation">Homi
Bhabha National Institute, Anushaktinagar, Mumbai 400094, India</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kartik Dutta</span> - <span class="hlFld-Affiliation affiliation">Bio-Organic
Division, Bhabha Atomic Research Centre, Trombay, Mumbai 400085, India</span>; 
    <span class="hlFld-Affiliation affiliation">Homi
Bhabha National Institute, Anushaktinagar, Mumbai 400094, India</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Pooja Gupta</span> - <span class="hlFld-Affiliation affiliation">Bio-Organic
Division, Bhabha Atomic Research Centre, Trombay, Mumbai 400085, India</span>; 
    <span class="hlFld-Affiliation affiliation">Homi
Bhabha National Institute, Anushaktinagar, Mumbai 400094, India</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Anubrata Das</span> - <span class="hlFld-Affiliation affiliation">Bio-Organic
Division, Bhabha Atomic Research Centre, Trombay, Mumbai 400085, India</span>; 
    <span class="hlFld-Affiliation affiliation">Homi
Bhabha National Institute, Anushaktinagar, Mumbai 400094, India</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Amit Das</span> - <span class="hlFld-Affiliation affiliation">Homi
Bhabha National Institute, Anushaktinagar, Mumbai 400094, India</span>; 
    <span class="hlFld-Affiliation affiliation">Radiation
Biology & Health Sciences Division, Bhabha Atomic Research Centre, Trombay, Mumbai 400085, India</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>S.C. and B.S.P. designed the biology experiments while K.D. and S.K.G. designed the chemistry experiments. S.C., P.G., and A.D. performed the biology experiments. K.D. synthesized and characterized the compounds. A.D. performed docking analyses. S.C., K.D., S.K.G., and B.S.P. contributed to data analysis, manuscript writing, and editing.</p></li><li><span class="author-information-subsection-header">Funding</span><p>This work was supported financially by the internal funding of the Bhabha Atomic Research Centre, Department of Atomic Energy, India.</p></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i65">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_75775" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_75775" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The authors would like to acknowledge the central animal house facility at the Bhabha Atomic Research Centre, Mumbai. The authors also like to acknowledge Ganesh Pai, and Ananda Guha Majumdar for their help in animal experimentation and protein purification, Raghunath Chowdhury and Binita Kisley Kumar for their help in HPLC analyses, respectively.</p></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i66">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_13877" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_13877" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 42 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bray, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferlay, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soerjomataram, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siegel, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Torre, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jemal, A.</span></span> <span> </span><span class="NLM_article-title">Global Cancer Statistics 2018: GLOBOCAN Estimates of Incidence and Mortality Worldwide for 36 Cancers in 185 Countries</span>. <i>Ca-Cancer J. Clin.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>68</i></span>,  <span class="NLM_fpage">394</span>– <span class="NLM_lpage">424</span>, <span class="refDoi"> DOI: 10.3322/caac.21492</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=10.3322%2Fcaac.21492" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=30207593" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=1%3ACAS%3A280%3ADC%252BB3c3otVGlsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2018&pages=394-424&author=F.+Brayauthor=J.+Ferlayauthor=I.+Soerjomataramauthor=R.+L.+Siegelauthor=L.+A.+Torreauthor=A.+Jemal&title=Global+Cancer+Statistics+2018%3A+GLOBOCAN+Estimates+of+Incidence+and+Mortality+Worldwide+for+36+Cancers+in+185+Countries&doi=10.3322%2Fcaac.21492"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Global cancer statistics 2018: GLOBOCAN estimates of incidence and mortality worldwide for 36 cancers in 185 countries</span></div><div class="casAuthors">Bray Freddie; Ferlay Jacques; Soerjomataram Isabelle; Siegel Rebecca L; Torre Lindsey A; Jemal Ahmedin</div><div class="citationInfo"><span class="NLM_cas:title">CA: a cancer journal for clinicians</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">68</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">394-424</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">This article provides a status report on the global burden of cancer worldwide using the GLOBOCAN 2018 estimates of cancer incidence and mortality produced by the International Agency for Research on Cancer, with a focus on geographic variability across 20 world regions.  There will be an estimated 18.1 million new cancer cases (17.0 million excluding nonmelanoma skin cancer) and 9.6 million cancer deaths (9.5 million excluding nonmelanoma skin cancer) in 2018.  In both sexes combined, lung cancer is the most commonly diagnosed cancer (11.6% of the total cases) and the leading cause of cancer death (18.4% of the total cancer deaths), closely followed by female breast cancer (11.6%), prostate cancer (7.1%), and colorectal cancer (6.1%) for incidence and colorectal cancer (9.2%), stomach cancer (8.2%), and liver cancer (8.2%) for mortality.  Lung cancer is the most frequent cancer and the leading cause of cancer death among males, followed by prostate and colorectal cancer (for incidence) and liver and stomach cancer (for mortality).  Among females, breast cancer is the most commonly diagnosed cancer and the leading cause of cancer death, followed by colorectal and lung cancer (for incidence), and vice versa (for mortality); cervical cancer ranks fourth for both incidence and mortality.  The most frequently diagnosed cancer and the leading cause of cancer death, however, substantially vary across countries and within each country depending on the degree of economic development and associated social and life style factors.  It is noteworthy that high-quality cancer registry data, the basis for planning and implementing evidence-based cancer control programs, are not available in most low- and middle-income countries.  The Global Initiative for Cancer Registry Development is an international partnership that supports better estimation, as well as the collection and use of local data, to prioritize and evaluate national cancer control efforts.  CA: A Cancer Journal for Clinicians 2018;0:1-31. © 2018 American Cancer Society.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQaW5wv8ocuhLxanA7vODO6fW6udTcc2ebG6McEkMdmerntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3c3otVGlsg%253D%253D&md5=48529f8f794092779d70a27eec5c9017</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.3322%2Fcaac.21492&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3322%252Fcaac.21492%26sid%3Dliteratum%253Aachs%26aulast%3DBray%26aufirst%3DF.%26aulast%3DFerlay%26aufirst%3DJ.%26aulast%3DSoerjomataram%26aufirst%3DI.%26aulast%3DSiegel%26aufirst%3DR.%2BL.%26aulast%3DTorre%26aufirst%3DL.%2BA.%26aulast%3DJemal%26aufirst%3DA.%26atitle%3DGlobal%2520Cancer%2520Statistics%25202018%253A%2520GLOBOCAN%2520Estimates%2520of%2520Incidence%2520and%2520Mortality%2520Worldwide%2520for%252036%2520Cancers%2520in%2520185%2520Countries%26jtitle%3DCa-Cancer%2520J.%2520Clin.%26date%3D2018%26volume%3D68%26spage%3D394%26epage%3D424%26doi%3D10.3322%2Fcaac.21492" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Forouzanfar, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foreman, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delossantos, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lozano, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, C. J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naghavi, M.</span></span> <span> </span><span class="NLM_article-title">Breast and Cervical Cancer in 187 Countries between 1980 and 2010: A Systematic Analysis</span>. <i>Lancet</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>378</i></span>,  <span class="NLM_fpage">1461</span>– <span class="NLM_lpage">1484</span>, <span class="refDoi"> DOI: 10.1016/S0140-6736(11)61351-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=10.1016%2FS0140-6736%2811%2961351-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=21924486" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=1%3ACAS%3A280%3ADC%252BC3MbjtFGkuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=378&publication_year=2011&pages=1461-1484&author=M.+H.+Forouzanfarauthor=K.+J.+Foremanauthor=A.+M.+Delossantosauthor=R.+Lozanoauthor=A.+D.+Lopezauthor=C.+J.+L.+Murrayauthor=M.+Naghavi&title=Breast+and+Cervical+Cancer+in+187+Countries+between+1980+and+2010%3A+A+Systematic+Analysis&doi=10.1016%2FS0140-6736%2811%2961351-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Breast and cervical cancer in 187 countries between 1980 and 2010: a systematic analysis</span></div><div class="casAuthors">Forouzanfar Mohammad H; Foreman Kyle J; Delossantos Allyne M; Lozano Rafael; Lopez Alan D; Murray Christopher J L; Naghavi Mohsen</div><div class="citationInfo"><span class="NLM_cas:title">Lancet (London, England)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">378</span>
        (<span class="NLM_cas:issue">9801</span>),
    <span class="NLM_cas:pages">1461-84</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Breast and cervical cancer are important causes of mortality in women aged ≥15 years.  We undertook annual age-specific assessments of breast and cervical cancer in 187 countries.  METHODS:  We systematically collected cancer registry data on mortality and incidence, vital registration, and verbal autopsy data for the period 1980-2010.  We modelled the mortality-to-incidence (MI) ratio using a hierarchical model.  Vital registration and verbal autopsy were supplemented with incidence multiplied by the MI ratio to yield a comprehensive database of mortality rates.  We used Gaussian process regression to develop estimates of mortality with uncertainty by age, sex, country, and year.  We used out-of-sample predictive validity to select the final model.  Estimates of incidence with uncertainty were also generated with mortality and MI ratios.  FINDINGS:  Global breast cancer incidence increased from 641,000 (95% uncertainty intervals 610,000-750,000) cases in 1980 to 1,643,000 (1,421,000-1,782,000) cases in 2010, an annual rate of increase of 3·1%.  Global cervical cancer incidence increased from 378,000 (256,000-489,000) cases per year in 1980 to 454,000 (318,000-620,000) cases per year in 2010-a 0·6% annual rate of increase.  Breast cancer killed 425,000 (359,000-453,000) women in 2010, of whom 68,000 (62,000-74,000) were aged 15-49 years in developing countries.  Cervical cancer death rates have been decreasing but the disease still killed 200,000 (139,000-276,000) women in 2010, of whom 46,000 (33,000-64,000) were aged 15-49 years in developing countries.  We recorded pronounced variation in the trend in breast cancer mortality across regions and countries.  INTERPRETATION:  More policy attention is needed to strengthen established health-system responses to reduce breast and cervical cancer, especially in developing countries.  FUNDING:  Susan G Komen for the Cure and the Bill & Melinda Gates Foundation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTjccp-xjq9WxfKb6ELK8t0fW6udTcc2ebG6McEkMdmerntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3MbjtFGkuw%253D%253D&md5=19f693da0e637477c4014d7f3c56edc5</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2FS0140-6736%2811%2961351-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0140-6736%252811%252961351-2%26sid%3Dliteratum%253Aachs%26aulast%3DForouzanfar%26aufirst%3DM.%2BH.%26aulast%3DForeman%26aufirst%3DK.%2BJ.%26aulast%3DDelossantos%26aufirst%3DA.%2BM.%26aulast%3DLozano%26aufirst%3DR.%26aulast%3DLopez%26aufirst%3DA.%2BD.%26aulast%3DMurray%26aufirst%3DC.%2BJ.%2BL.%26aulast%3DNaghavi%26aufirst%3DM.%26atitle%3DBreast%2520and%2520Cervical%2520Cancer%2520in%2520187%2520Countries%2520between%25201980%2520and%25202010%253A%2520A%2520Systematic%2520Analysis%26jtitle%3DLancet%26date%3D2011%26volume%3D378%26spage%3D1461%26epage%3D1484%26doi%3D10.1016%2FS0140-6736%2811%2961351-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Waks, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winer, E. P.</span></span> <span> </span><span class="NLM_article-title">Breast Cancer Treatment: A Review</span>. <i>JAMA</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>321</i></span>,  <span class="NLM_fpage">288</span>, <span class="refDoi"> DOI: 10.1001/jama.2018.19323</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=10.1001%2Fjama.2018.19323" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=30667505" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhvV2qt7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=321&publication_year=2019&pages=288&author=A.+G.+Waksauthor=E.+P.+Winer&title=Breast+Cancer+Treatment%3A+A+Review&doi=10.1001%2Fjama.2018.19323"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Breast cancer treatment: a review</span></div><div class="casAuthors">Waks, Adrienne G.; Winer, Eric P.</div><div class="citationInfo"><span class="NLM_cas:title">JAMA, the Journal of the American Medical Association</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">321</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">288-300</span>CODEN:
                <span class="NLM_cas:coden">JAMAAP</span>;
        ISSN:<span class="NLM_cas:issn">1538-3598</span>.
    
            (<span class="NLM_cas:orgname">American Medical Association</span>)
        </div><div class="casAbstract">A review.  Importance Breast cancer will be diagnosed in 12% of women in the United States over the course of their lifetimes and more than 250 000 new cases of breast cancer were diagnosed in the United States in 2017.  This review focuses on current approaches and evolving strategies for local and systemic therapy of breast cancer. observations Breast cancer is categorized into 3 major subtypes based on the presence or absence of mol. markers for estrogen or progesterone receptors and human epidermal growth factor 2 (ERBB2; formerly HER2): hormone receptor pos./ERBB2 neg. (70% of patients), ERBB2 pos. (15%-20%), and triple-neg. (tumors lacking all 3 std. mol. markers; 15%).  More than 90% of breast cancers are not metastatic at the time of diagnosis.  For people presenting without metastatic disease, therapeutic goals are tumor eradication and preventing recurrence.  Triple-neg. breast cancer is more likely to recur than the other 2 subtypes, with 85% 5-yr breast cancer-specific survival for stage I triple-neg. tumors vs 94% to 99% for hormone receptor pos. and ERBB2 pos.  Systemic therapy for nonmetastatic breast cancer is detd. by subtype: patients with hormone receptor-pos. tumors receive endocrine therapy, and a minority receive chemotherapy as well; patients with ERBB2-pos. tumors receive ERBB2-targeted antibody or small-mol. inhibitor therapy combined with chemotherapy; and patients with triple-neg. tumors receive chemotherapy alone.  Local therapy for all patients with nonmetastatic breast cancer consists of surgical resection, with consideration of postoperative radiation if lumpectomy is performed.  Increasingly, some systemic therapy is delivered before surgery.  Tailoring postoperative treatment based on preoperative treatment response is under investigation.  Metastatic breast cancer is treated according to subtype, with goals of prolonging life and palliating symptoms.  Median overall survival for metastatic triple-neg. breast cancer is approx. 1 yr vs approx. 5 years for the other 2 subtypes. conclusions and relevance Breast cancer consists of 3 major tumor subtypes categorized according to estrogen or progesterone receptor expression and ERBB2 gene amplification.  The 3 subtypes have distinct risk profiles and treatment strategies.  Optimal therapy for each patient depends on tumor subtype, anat. cancer stage, and patient preferences.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGriTXwcPIcZ_LVg90H21EOLACvtfcHk0ljBiCBXW0Dlpw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhvV2qt7c%253D&md5=548a8f22dc89eb559f95d779b37f3a9d</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1001%2Fjama.2018.19323&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1001%252Fjama.2018.19323%26sid%3Dliteratum%253Aachs%26aulast%3DWaks%26aufirst%3DA.%2BG.%26aulast%3DWiner%26aufirst%3DE.%2BP.%26atitle%3DBreast%2520Cancer%2520Treatment%253A%2520A%2520Review%26jtitle%3DJAMA%26date%3D2019%26volume%3D321%26spage%3D288%26doi%3D10.1001%2Fjama.2018.19323" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guan, X.</span></span> <span> </span><span class="NLM_article-title">Emerging Therapeutic Modalities of PARP Inhibitors in Breast Cancer</span>. <i>Cancer Treat. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>68</i></span>,  <span class="NLM_fpage">62</span>– <span class="NLM_lpage">68</span>, <span class="refDoi"> DOI: 10.1016/j.ctrv.2018.05.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=10.1016%2Fj.ctrv.2018.05.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=29870916" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtV2jur7O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2018&pages=62-68&author=X.+Wangauthor=Y.+Shiauthor=D.+Huangauthor=X.+Guan&title=Emerging+Therapeutic+Modalities+of+PARP+Inhibitors+in+Breast+Cancer&doi=10.1016%2Fj.ctrv.2018.05.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Emerging therapeutic modalities of PARP inhibitors in breast cancer</span></div><div class="casAuthors">Wang, Xin; Shi, Yaqin; Huang, Doudou; Guan, Xiaoxiang</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Treatment Reviews</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">68</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">62-68</span>CODEN:
                <span class="NLM_cas:coden">CTREDJ</span>;
        ISSN:<span class="NLM_cas:issn">0305-7372</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Inhibition of Poly (ADP-ribose) polymerase (PARP) has shown marked benefit for breast cancer with homologous recombination deficiency, whether driven by defects in BRCA1, BRCA2, or other pathway components.  Since the initial approval of olaparib, a mostly investigated PARP inhibitor (PARPi), the clin. development of PARPi in breast cancer treatment has been a major emphasis.  Researches in investigating platinum-PARPi combination use compared with platinum monotherapy demonstrated promising benefit in metastatic BRCA mutated breast cancer or TNBC, while no such superiority was obsd. in the neoadjuvant setting of TNBC.  Moreover, the utility of PARP inhibition in BRCA1/2 mutated breast cancer with different platinum-free interval was investigated.  There was a clear assocn. between clin. benefit with PARPi and platinum sensitivity, whereas partial efficacy of PARPi still occurs in platinum-resistant patients.  In addn., proof-of-principle studies of immunotherapy combined with PARPi in breast cancer have obtained promising results, indicating the potential benefit of the combination therapy in patients with breast cancer.  These efforts, contributing to maximize the utility of PARPi, may drive a new era of this agent after its first routine use.  In this review, we summarized the utility of combining platinum-PARPi in BRCA mutated breast cancer or TNBC compared with platinum monotherapy and provided promising prospects of PARPi as maintenance therapy in breast cancer, as well as providing a strong rationale for testing immunotherapy combined with PARPi in breast cancer to expand the clin. utility of PARPi.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrtoy0qBmshirVg90H21EOLACvtfcHk0lg9re_THH0RUA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtV2jur7O&md5=1c28f7585b4deace459da25b2ff90d32</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.ctrv.2018.05.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ctrv.2018.05.014%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DX.%26aulast%3DShi%26aufirst%3DY.%26aulast%3DHuang%26aufirst%3DD.%26aulast%3DGuan%26aufirst%3DX.%26atitle%3DEmerging%2520Therapeutic%2520Modalities%2520of%2520PARP%2520Inhibitors%2520in%2520Breast%2520Cancer%26jtitle%3DCancer%2520Treat.%2520Rev.%26date%3D2018%26volume%3D68%26spage%3D62%26epage%3D68%26doi%3D10.1016%2Fj.ctrv.2018.05.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Exman, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barroso-Sousa, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tolaney, S. M.</span></span> <span> </span><span class="NLM_article-title">Evidence to Date: Talazoparib in the Treatment of Breast Cancer</span>. <i>OncoTargets Ther.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">5177</span>– <span class="NLM_lpage">5187</span>, <span class="refDoi"> DOI: 10.2147/OTT.S184971</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=10.2147%2FOTT.S184971" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=31303769" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=1%3ACAS%3A528%3ADC%252BB3cXjsVGkt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2019&pages=5177-5187&author=P.+Exmanauthor=R.+Barroso-Sousaauthor=S.+M.+Tolaney&title=Evidence+to+Date%3A+Talazoparib+in+the+Treatment+of+Breast+Cancer&doi=10.2147%2FOTT.S184971"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Evidence to date: talazoparib in the treatment of breast cancer</span></div><div class="casAuthors">Exman, Pedro; Barroso-Sousa, Romualdo; Tolaney, Sara M.</div><div class="citationInfo"><span class="NLM_cas:title">OncoTargets and Therapy</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">5177-5187</span>CODEN:
                <span class="NLM_cas:coden">OTNHAZ</span>;
        ISSN:<span class="NLM_cas:issn">1178-6930</span>.
    
            (<span class="NLM_cas:orgname">Dove Medical Press Ltd.</span>)
        </div><div class="casAbstract">Approx. 5-10% of all patients diagnosed with breast cancer have germline BRCA1/2 mutations, which make their disease more susceptible to DNA-damaging agents and a new class of drugs known as poly(ADP-ribose) polymerase (PARP) inhibitors.  Talazoparib is a new PARP inhibitor that has been recently approved for use in patients with metastatic breast cancer with germline BRCA mutations after a phase III trial showed superior progression-free survival when compared to std. chemotherapy.  In this review, we analyze the development of talazoparib as well as its safety profile and the potential role of the combination therapy with std. cytotoxic drugs and with novel therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqKyX-RFMagpLVg90H21EOLACvtfcHk0lg9re_THH0RUA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXjsVGkt7s%253D&md5=3c49b35879d525ecce2bf383188c835a</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.2147%2FOTT.S184971&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2147%252FOTT.S184971%26sid%3Dliteratum%253Aachs%26aulast%3DExman%26aufirst%3DP.%26aulast%3DBarroso-Sousa%26aufirst%3DR.%26aulast%3DTolaney%26aufirst%3DS.%2BM.%26atitle%3DEvidence%2520to%2520Date%253A%2520Talazoparib%2520in%2520the%2520Treatment%2520of%2520Breast%2520Cancer%26jtitle%3DOncoTargets%2520Ther.%26date%3D2019%26volume%3D12%26spage%3D5177%26epage%3D5187%26doi%3D10.2147%2FOTT.S184971" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Popuri, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tadokoro, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Croteau, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bohr, V. A.</span></span> <span> </span><span class="NLM_article-title">Human RECQL5: Guarding the Crossroads of DNA Replication and Transcription and Providing Backup Capability</span>. <i>Crit. Rev. Biochem. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">289</span>– <span class="NLM_lpage">299</span>, <span class="refDoi"> DOI: 10.3109/10409238.2013.792770</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=10.3109%2F10409238.2013.792770" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=23627586" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=1%3ACAS%3A528%3ADC%252BC3sXos12gtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2013&pages=289-299&author=V.+Popuriauthor=T.+Tadokoroauthor=D.+L.+Croteauauthor=V.+A.+Bohr&title=Human+RECQL5%3A+Guarding+the+Crossroads+of+DNA+Replication+and+Transcription+and+Providing+Backup+Capability&doi=10.3109%2F10409238.2013.792770"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Human RECQL5: Guarding the crossroads of DNA replication and transcription and providing backup capability</span></div><div class="casAuthors">Popuri, Venkateswarlu; Tadokoro, Takashi; Croteau, Deborah L.; Bohr, Vilhelm A.</div><div class="citationInfo"><span class="NLM_cas:title">Critical Reviews in Biochemistry and Molecular Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">289-299</span>CODEN:
                <span class="NLM_cas:coden">CRBBEJ</span>;
        ISSN:<span class="NLM_cas:issn">1040-9238</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  DNA helicases are ubiquitous enzymes that catalyze the unwinding of duplex DNA and function in all metabolic processes in which access to single-stranded DNA is required, including DNA replication, repair, recombination, and RNA transcription.  RecQ helicases are a conserved family of DNA helicases that display highly specialized and vital roles in the maintenance of genome stability.  Mutations in 3 of the 5 human RecQ helicases (BLM, WRN, and RECQL4) are assocd. with the genetic disorders, Bloom syndrome, Werner syndrome, and Rothmund-Thomson syndrome, that are characterized by chromosomal instability, premature aging, and predisposition to cancer.  The biol. role of human RECQL5 is only partially understood and RECQL5 has not yet been assocd. with any human disease.  Illegitimate recombination and replication stress are hallmarks of human cancers and common instigators for genomic instability and cell death.  Gene Recql5-knockout mice are cancer prone and show increased chromosomal instability.  Recql5-deficient mouse embryonic fibroblasts are sensitive to camptothecin and display elevated levels of sister chromatid exchanges.  Unlike other human RecQ helicases, RECQL5 is recruited to single-stranded DNA breaks and is also proposed to play an essential role in RNA transcription.  Here, the authors review the established roles of RECQL5 at the crossroads of DNA replication, recombination, and transcription, and propose that human RECQL5 provides important backup functions in the absence of other DNA helicases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrj_4k8QzuNo7Vg90H21EOLACvtfcHk0lg9re_THH0RUA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXos12gtL8%253D&md5=52b229f58b912e59c0e46c50f718d6e6</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.3109%2F10409238.2013.792770&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3109%252F10409238.2013.792770%26sid%3Dliteratum%253Aachs%26aulast%3DPopuri%26aufirst%3DV.%26aulast%3DTadokoro%26aufirst%3DT.%26aulast%3DCroteau%26aufirst%3DD.%2BL.%26aulast%3DBohr%26aufirst%3DV.%2BA.%26atitle%3DHuman%2520RECQL5%253A%2520Guarding%2520the%2520Crossroads%2520of%2520DNA%2520Replication%2520and%2520Transcription%2520and%2520Providing%2520Backup%2520Capability%26jtitle%3DCrit.%2520Rev.%2520Biochem.%2520Mol.%2520Biol.%26date%3D2013%26volume%3D48%26spage%3D289%26epage%3D299%26doi%3D10.3109%2F10409238.2013.792770" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brosh, R. M.</span></span> <span> </span><span class="NLM_article-title">DNA Helicases Involved in DNA Repair and Their Roles in Cancer</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">542</span>– <span class="NLM_lpage">558</span>, <span class="refDoi"> DOI: 10.1038/nrc3560</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=10.1038%2Fnrc3560" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=23842644" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtVyktLfL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2013&pages=542-558&author=R.+M.+Brosh&title=DNA+Helicases+Involved+in+DNA+Repair+and+Their+Roles+in+Cancer&doi=10.1038%2Fnrc3560"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">DNA helicases involved in DNA repair and their roles in cancer</span></div><div class="casAuthors">Brosh, Robert M., Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">542-558</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Helicases have major roles in genome maintenance by unwinding structured nucleic acids.  Their prominence is marked by various cancers and genetic disorders that are linked to helicase defects.  Although considerable effort has been made to understand the functions of DNA helicases that are important for genomic stability and cellular homeostasis, the complexity of the DNA damage response leaves the authors with unanswered questions regarding how helicase-dependent DNA repair pathways are regulated and coordinated with cell cycle checkpoints.  Further studies may open the door to targeting helicases to improve cancer treatments based on DNA-damaging chemotherapy or radiation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQPs2MtOs8VrVg90H21EOLACvtfcHk0lhmeBZINRbh5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtVyktLfL&md5=e3cbd291237aecd2f801287d0e8dbb53</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fnrc3560&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc3560%26sid%3Dliteratum%253Aachs%26aulast%3DBrosh%26aufirst%3DR.%2BM.%26atitle%3DDNA%2520Helicases%2520Involved%2520in%2520DNA%2520Repair%2520and%2520Their%2520Roles%2520in%2520Cancer%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2013%26volume%3D13%26spage%3D542%26epage%3D558%26doi%3D10.1038%2Fnrc3560" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhi, L.-Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, S.-X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, B.</span></span> <span> </span><span class="NLM_article-title">Association of RECQL5 Gene Polymorphisms and Osteosarcoma in a Chinese Han Population</span>. <i>Tumor Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">3255</span>– <span class="NLM_lpage">3259</span>, <span class="refDoi"> DOI: 10.1007/s13277-013-1425-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=10.1007%2Fs13277-013-1425-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmtlSqsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2014&pages=3255-3259&author=L.-Q.+Zhiauthor=W.+Maauthor=H.+Zhangauthor=S.-X.+Zengauthor=B.+Chen&title=Association+of+RECQL5+Gene+Polymorphisms+and+Osteosarcoma+in+a+Chinese+Han+Population&doi=10.1007%2Fs13277-013-1425-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Association of RECQL5 gene polymorphisms and osteosarcoma in a Chinese Han population</span></div><div class="casAuthors">Zhi, Li-Qiang; Ma, Wei; Zhang, Hong; Zeng, Si-Xiang; Chen, Bo</div><div class="citationInfo"><span class="NLM_cas:title">Tumor Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">3255-3259</span>CODEN:
                <span class="NLM_cas:coden">TUMBEA</span>;
        ISSN:<span class="NLM_cas:issn">1010-4283</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Despite the knowledge on many genetic variants present in osteosarcoma, the complexity of this disease precludes placing its biol. into a simple conceptual framework.  RECQL is a DNA helicase involved in DNA mismatch repair and has been reported to be assocd. with many human cancers.  We aimed to investigate the assocn. of RECQL genetic polymorphism with osteosarcoma in a Chinese population.  We selected three polymorphisms of the RECQL5 gene (rs820196, rs820200, and rs4789223) in the present study.  TaqMan method was utilized for genotyping these three SNPs in 212 patients with osteosarcoma and 240 age- and sex-matched noncancer controls.  In our study, we found that CC genotype in rs820196 (17.5 vs 8.3 %, P = 0.005) and AA genotype in rs4789223 (21.7 vs 14.2, P < 0.001) were more frequent in osteosarcoma group compared to the control group, resp.  We also found that the C allele of rs820196 (OR = 1.492, 95 % CI 1.138∼1.951; P = 0.004) and A allele of rs4789223 (OR = 1.767, 95 % CI: 1.354 ∼ 2.301; P < 0.001) were common in the osteosarcoma patients than those in the control subjects, resp.  Haplotype anal. showed that TTA (OR = 3.469, 95 % CI 1.798∼6.695; P < 0.001) was assocd. with increased risk for osteosarcoma.  However, the TTG (OR = 0.578, 95 % CI 0.442∼0.756) was assocd. with decreased risk for osteosarcoma.  Our results suggested that RECQL5 genetic polymorphisms were assocd. with osteosarcoma in a Chinese population.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9zacLjGqyV7Vg90H21EOLACvtfcHk0lhmeBZINRbh5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmtlSqsbw%253D&md5=5b1021bf0d327dea0a3fbca97ae6ca5c</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1007%2Fs13277-013-1425-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs13277-013-1425-4%26sid%3Dliteratum%253Aachs%26aulast%3DZhi%26aufirst%3DL.-Q.%26aulast%3DMa%26aufirst%3DW.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DZeng%26aufirst%3DS.-X.%26aulast%3DChen%26aufirst%3DB.%26atitle%3DAssociation%2520of%2520RECQL5%2520Gene%2520Polymorphisms%2520and%2520Osteosarcoma%2520in%2520a%2520Chinese%2520Han%2520Population%26jtitle%3DTumor%2520Biol.%26date%3D2014%26volume%3D35%26spage%3D3255%26epage%3D3259%26doi%3D10.1007%2Fs13277-013-1425-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dong, Y. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Y. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, T.</span></span> <span> </span><span class="NLM_article-title">Single Nucleotide Polymorphism in the RECQL5 Gene Increased Osteosarcoma Susceptibility in a Chinese Han Population</span>. <i>GMR, Genet. Mol. Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">1899</span>– <span class="NLM_lpage">1902</span>, <span class="refDoi"> DOI: 10.4238/2015.March.13.18</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=10.4238%2F2015.March.13.18" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=25867335" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=1%3ACAS%3A528%3ADC%252BC2MXlsVejtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2015&pages=1899-1902&author=Y.+Z.+Dongauthor=Y.+X.+Huangauthor=T.+Lu&title=Single+Nucleotide+Polymorphism+in+the+RECQL5+Gene+Increased+Osteosarcoma+Susceptibility+in+a+Chinese+Han+Population&doi=10.4238%2F2015.March.13.18"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Single nucleotide polymorphism in the RECQL5 gene increased osteosarcoma susceptibility in a Chinese Han population</span></div><div class="casAuthors">Dong, Y. Z.; Huang, Y. X.; Lu, T.</div><div class="citationInfo"><span class="NLM_cas:title">GMR, Genetics and Molecular Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1899-1902</span>CODEN:
                <span class="NLM_cas:coden">GGMRBI</span>;
        ISSN:<span class="NLM_cas:issn">1676-5680</span>.
    
            (<span class="NLM_cas:orgname">FUNPEP</span>)
        </div><div class="casAbstract">In this study, we investigated the assocn. between a RECQL genetic polymorphism and osteosarcoma in a Chinese population.  We selected rs820196 in the RECQL5 gene and genotyped 185 patients with osteosarcoma and 201 age- and gender-matched non-cancer controls.  We found that the CC genotype was more frequent in the osteosarcoma group compared to the control group (P = 0.011).  We also found that the C allele was more common in osteosarcoma patients than that in control subjects (P = 0.004).  Our results suggested that the RECQL5 genetic polymorphism was assocd. with osteosarcoma in a Chinese population.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpcY1342JNvcbVg90H21EOLACvtfcHk0lhmeBZINRbh5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXlsVejtbo%253D&md5=ab44c87a23db294690669405d3d2d1d6</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.4238%2F2015.March.13.18&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4238%252F2015.March.13.18%26sid%3Dliteratum%253Aachs%26aulast%3DDong%26aufirst%3DY.%2BZ.%26aulast%3DHuang%26aufirst%3DY.%2BX.%26aulast%3DLu%26aufirst%3DT.%26atitle%3DSingle%2520Nucleotide%2520Polymorphism%2520in%2520the%2520RECQL5%2520Gene%2520Increased%2520Osteosarcoma%2520Susceptibility%2520in%2520a%2520Chinese%2520Han%2520Population%26jtitle%3DGMR%252C%2520Genet.%2520Mol.%2520Res.%26date%3D2015%26volume%3D14%26spage%3D1899%26epage%3D1902%26doi%3D10.4238%2F2015.March.13.18" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Qi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, X.</span></span> <span> </span><span class="NLM_article-title">Haplotype Analysis of RECQL5 Gene and Laryngeal Cancer</span>. <i>Tumor Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">2669</span>– <span class="NLM_lpage">2673</span>, <span class="refDoi"> DOI: 10.1007/s13277-013-1351-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=10.1007%2Fs13277-013-1351-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=1%3ACAS%3A528%3ADC%252BC2cXltlyrtb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2014&pages=2669-2673&author=Y.+Qiauthor=X.+Zhou&title=Haplotype+Analysis+of+RECQL5+Gene+and+Laryngeal+Cancer&doi=10.1007%2Fs13277-013-1351-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Haplotype analysis of RECQL5 gene and laryngeal cancer</span></div><div class="casAuthors">Qi, Ying; Zhou, Xu</div><div class="citationInfo"><span class="NLM_cas:title">Tumor Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">2669-2673</span>CODEN:
                <span class="NLM_cas:coden">TUMBEA</span>;
        ISSN:<span class="NLM_cas:issn">1010-4283</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">RECQL is a DNA helicase involved in DNA mismatch repair.  Previous studies indicated that the RECQL gene mutation was assocd. with human cancers.  In the present study, we investigated the assocn. between polymorphisms of RECQL gene and laryngeal cancer in a Chinese population.  Four polymorphisms of the RECQL5 gene (rs820186, rs820196, rs820200, and rs4789223) were genotyped by the TaqMan method in 275 patients with larynx cancer and 300 age- and sex-matched non-cancer controls.  We found that rs820196 polymorphism of RECQL5 was assocd. with larynx cancer, the CC genotype (16.4 % vs 9.3 %, P=0.013) and C allele (42 % vs 34.2 %, P=0.006) was common in larynx cancer patients than in the control subjects, resp.  Haplotype anal. showed that GCGA (OR=18.955, 95 % confidence interval (CI) 9.479∼37.905; P<0.001) and GTTG (OR=1.684; 95 % CI 1.327∼2.137; P<0.001) were assocd. with increased risk for larynx cancer.  However, ACGA (OR=0.605; 95 % CI 0.430∼0.852; P=0.003), GCGG (OR=0.636; 95 % CI 0.411∼0.982; P=0.039), and GTGG (OR=0.194; 95 % CI 0.104∼0.361; P<0.001) were assocd. with decreased risk for larynx cancer.  The present study indicated that RECQL5 genetic polymorphism and haplotypes were assocd. with larynx cancer in a Chinese population.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoArsd34p2XXrVg90H21EOLACvtfcHk0lgkf9QaimX0hQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXltlyrtb0%253D&md5=c7ad62452eb5fd3b6099aaead36eaa19</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1007%2Fs13277-013-1351-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs13277-013-1351-5%26sid%3Dliteratum%253Aachs%26aulast%3DQi%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DX.%26atitle%3DHaplotype%2520Analysis%2520of%2520RECQL5%2520Gene%2520and%2520Laryngeal%2520Cancer%26jtitle%3DTumor%2520Biol.%26date%3D2014%26volume%3D35%26spage%3D2669%26epage%3D2673%26doi%3D10.1007%2Fs13277-013-1351-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">He, Y.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiao, Z.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, Y.-Y.</span></span> <span> </span><span class="NLM_article-title">Association between RECQL5 Genetic Polymorphisms and Susceptibility to Breast Cancer</span>. <i>Tumor Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">12201</span>– <span class="NLM_lpage">12204</span>, <span class="refDoi"> DOI: 10.1007/s13277-014-2528-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=10.1007%2Fs13277-014-2528-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFGgtrfO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2014&pages=12201-12204&author=Y.-J.+Heauthor=Z.-Y.+Qiaoauthor=B.+Gaoauthor=X.-H.+Zhangauthor=Y.-Y.+Wen&title=Association+between+RECQL5+Genetic+Polymorphisms+and+Susceptibility+to+Breast+Cancer&doi=10.1007%2Fs13277-014-2528-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Association between RECQL5 genetic polymorphisms and susceptibility to breast cancer</span></div><div class="casAuthors">He, Yu-Jun; Qiao, Zuo-Yi; Gao, Bo; Zhang, Xiao-Hua; Wen, Ya-Yuan</div><div class="citationInfo"><span class="NLM_cas:title">Tumor Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">12201-12204</span>CODEN:
                <span class="NLM_cas:coden">TUMBEA</span>;
        ISSN:<span class="NLM_cas:issn">1010-4283</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Previous studies indicated that the RECQL5 gene polymorphism was assocd. with human cancers.  However, the assocn. of RECQL5 gene polymorphism with breast cancer remains unclear.  In the present study, we investigated the assocn. between polymorphisms of the RECQL gene and breast cancer in a Chinese population.  We selected four polymorphisms of the RECQL5 gene (rs820186, rs820196, rs820200, and rs4789223) for the present study.  The genotyping was performed using the TaqMan method in 510 patients with breast cancer and 510 age- and sex-matched non-cancer controls.  We found that rs820196 and rs828200 polymorphisms of RECQL5 were assocd. with breast cancer.  For rs820196, the CC genotype (16.7 vs 9.4 %, P < 0.001) and C allele (42.5 vs 34.3 %, P < 0.001) were common in the breast cancer patients than in the control subjects, resp.  For rs828200, the GG genotype (23.7 vs 18.0 %, P < 0.001) and G allele (52.7 vs 43.8 %, P < 0.001) were common in the breast cancer patients than in the control subjects, resp.  Haplotype anal. showed that C-G (odds ratio (OR) = 2.247, 95 % confidence interval (CI) 1.854∼2.722; P < 0.001) was assocd. with increased risk for breast cancer.  However, the C-T (OR = 0.175, 95 % CI 0.110∼0.278; P < 0.001) and T-G (OR = 0.544; 95 % CI 0.428∼0.692; P < 0.001) were assocd. with decreased risk for breast cancer, resp.  The present study indicated that the RECQL5 genetic polymorphism and haplotypes were assocd. with breast cancer in a Chinese population.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrYEZFicetXTbVg90H21EOLACvtfcHk0lgkf9QaimX0hQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvFGgtrfO&md5=ce91471378a894696b28ef0ba89f5a8f</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1007%2Fs13277-014-2528-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs13277-014-2528-2%26sid%3Dliteratum%253Aachs%26aulast%3DHe%26aufirst%3DY.-J.%26aulast%3DQiao%26aufirst%3DZ.-Y.%26aulast%3DGao%26aufirst%3DB.%26aulast%3DZhang%26aufirst%3DX.-H.%26aulast%3DWen%26aufirst%3DY.-Y.%26atitle%3DAssociation%2520between%2520RECQL5%2520Genetic%2520Polymorphisms%2520and%2520Susceptibility%2520to%2520Breast%2520Cancer%26jtitle%3DTumor%2520Biol.%26date%3D2014%26volume%3D35%26spage%3D12201%26epage%3D12204%26doi%3D10.1007%2Fs13277-014-2528-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Arora, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abdel-Fatah, T. M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agarwal, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doherty, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Croteau, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moseley, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hameed, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aleskandarany, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rakha, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patterson, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ball, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, S. Y. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellis, I. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bohr, V. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bryant, H. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Madhusudan, S.</span></span> <span> </span><span class="NLM_article-title">Clinicopathological and Prognostic Significance of RECQL5 Helicase Expression in Breast Cancers</span>. <i>Carcinogenesis</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>37</i></span>,  <span class="NLM_fpage">63</span>– <span class="NLM_lpage">71</span>, <span class="refDoi"> DOI: 10.1093/carcin/bgv163</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=10.1093%2Fcarcin%2Fbgv163" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=26586793" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht12rsLnN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2016&pages=63-71&author=A.+Aroraauthor=T.+M.+A.+Abdel-Fatahauthor=D.+Agarwalauthor=R.+Dohertyauthor=D.+L.+Croteauauthor=P.+M.+Moseleyauthor=K.+Hameedauthor=A.+Greenauthor=M.+A.+Aleskandaranyauthor=E.+A.+Rakhaauthor=K.+Pattersonauthor=G.+Ballauthor=S.+Y.+T.+Chanauthor=I.+O.+Ellisauthor=V.+A.+Bohrauthor=H.+E.+Bryantauthor=S.+Madhusudan&title=Clinicopathological+and+Prognostic+Significance+of+RECQL5+Helicase+Expression+in+Breast+Cancers&doi=10.1093%2Fcarcin%2Fbgv163"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Clinicopathological and prognostic significance of RECQL5 helicase expression in breast cancers</span></div><div class="casAuthors">Arora, Arvind; Abdel-Fatah, Tarek M. A.; Agarwal, Devika; Doherty, Rachel; Croteau, Deborah L.; Moseley, Paul M.; Hameed, Khalid; Green, Andrew; Aleskandarany, Mohammed A.; Rakha, Emad A.; Patterson, Karl; Ball, Graham; Chan, Stephen Y. T.; Ellis, Ian O.; Bohr, Vilhelm A.; Bryant, Helen E.; Madhusudan, Srinivasan</div><div class="citationInfo"><span class="NLM_cas:title">Carcinogenesis</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">63-71</span>CODEN:
                <span class="NLM_cas:coden">CRNGDP</span>;
        ISSN:<span class="NLM_cas:issn">0143-3334</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">RECQL5 is a member of the RecQ family of DNA helicases and has key roles in homologous recombination, base excision repair, replication and transcription.  The clinicopathol. significance of RECQL5 expression in breast cancer is unknown.  In this study, we have evaluated RECQL5 mRNA expression in 1977 breast cancers, and RECQL5 protein level in 1902 breast cancers [Nottingham Tenovus series (n = 1650) and ER- cohort (n = 252)].  Expression levels were correlated to aggressive phenotypes and survival outcomes.  High RECQL5 mRNA expression was significantly assocd. with high histol. grade (P = 0.007), HER2 overexpression (P = 0.032), ER+/HER2-/high proliferation genefu subtype (P < 0.0001), integrative mol. clusters (intClust 1and 9) (P < 0.0001) and poor survival (P < 0.0001).  In subgroup anal., high RECQL5 mRNA level remains significantly assocd. with poor BCSS in ER+ cohort (P < 0.0001) but not in ER- cohort (P = 0.116).  At the protein level, in tumors with low RAD51, high RECQL5 level was significantly assocd. with high histol. grade (P < 0.0001), higher mitotic index (P = 0.008), dedifferentiation (P = 0.025), pleomorphism (P = 0.027) and poor survival (P = 0.003).  In subgroup anal., high RECQL5/low RAD51 remains significantly assocd. with poor BCSS in ER+ cohort (P = 0.010), but not in ER-cohort (P = 0.628).  In multivariate anal., high RECQL5 mRNA and high RECQL5/low RAD51 nuclear protein coexpression independently influenced survival (P = 0.022) in whole cohort and in the ER+ subgroup.  Preclinically, we show that exogenous expression of RECQL5 in MCF10A cells can drive proliferation supporting an oncogenic function for RECQL5 in breast cancer.  We conclude that RECQL5 is a promising biomarker in breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLbW-Nlbo5p7Vg90H21EOLACvtfcHk0lgkf9QaimX0hQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht12rsLnN&md5=688c8b4611c229def638bbd7dfd8b17d</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1093%2Fcarcin%2Fbgv163&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fcarcin%252Fbgv163%26sid%3Dliteratum%253Aachs%26aulast%3DArora%26aufirst%3DA.%26aulast%3DAbdel-Fatah%26aufirst%3DT.%2BM.%2BA.%26aulast%3DAgarwal%26aufirst%3DD.%26aulast%3DDoherty%26aufirst%3DR.%26aulast%3DCroteau%26aufirst%3DD.%2BL.%26aulast%3DMoseley%26aufirst%3DP.%2BM.%26aulast%3DHameed%26aufirst%3DK.%26aulast%3DGreen%26aufirst%3DA.%26aulast%3DAleskandarany%26aufirst%3DM.%2BA.%26aulast%3DRakha%26aufirst%3DE.%2BA.%26aulast%3DPatterson%26aufirst%3DK.%26aulast%3DBall%26aufirst%3DG.%26aulast%3DChan%26aufirst%3DS.%2BY.%2BT.%26aulast%3DEllis%26aufirst%3DI.%2BO.%26aulast%3DBohr%26aufirst%3DV.%2BA.%26aulast%3DBryant%26aufirst%3DH.%2BE.%26aulast%3DMadhusudan%26aufirst%3DS.%26atitle%3DClinicopathological%2520and%2520Prognostic%2520Significance%2520of%2520RECQL5%2520Helicase%2520Expression%2520in%2520Breast%2520Cancers%26jtitle%3DCarcinogenesis%26date%3D2016%26volume%3D37%26spage%3D63%26epage%3D71%26doi%3D10.1093%2Fcarcin%2Fbgv163" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Andrs, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hasanova, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oravetzova, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dobrovolna, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janscak, P.</span></span> <span> </span><span class="NLM_article-title">RECQ5: A Mysterious Helicase at the Interface of DNA Replication and Transcription</span>. <i>Genes</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">232</span>, <span class="refDoi"> DOI: 10.3390/genes11020232</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=10.3390%2Fgenes11020232" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=1%3ACAS%3A528%3ADC%252BB3cXotlKnt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=232&author=M.+Andrsauthor=Z.+Hasanovaauthor=A.+Oravetzovaauthor=J.+Dobrovolnaauthor=P.+Janscak&title=RECQ5%3A+A+Mysterious+Helicase+at+the+Interface+of+DNA+Replication+and+Transcription&doi=10.3390%2Fgenes11020232"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">RECQ5: a mysterious helicase at the interface of DNA replication and transcription</span></div><div class="casAuthors">Andrs, Martin; Hasanova, Zdenka; Oravetzova, Anna; Dobrovolna, Jana; Janscak, Pavel</div><div class="citationInfo"><span class="NLM_cas:title">Genes</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">232</span>CODEN:
                <span class="NLM_cas:coden">GENEG9</span>;
        ISSN:<span class="NLM_cas:issn">2073-4425</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">RECQ5 belongs to the RecQ family of DNA helicases.  It is conserved from Drosophila to humans and its deficiency results in genomic instability and cancer susceptibility in mice.  Human RECQ5 is known for its ability to regulate homologous recombination by disrupting RAD51 nucleoprotein filaments.  It also binds to RNA polymerase II (RNAPII) and neg. regulates transcript elongation by RNAPII.  Here, we summarize recent studies implicating RECQ5 in the prevention and resoln. of transcription-replication conflicts, a major intrinsic source of genomic instability during cancer development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzmhnNQYCHSLVg90H21EOLACvtfcHk0lgkf9QaimX0hQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXotlKnt7o%253D&md5=d997f26d6598328f00ba07c086634216</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.3390%2Fgenes11020232&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fgenes11020232%26sid%3Dliteratum%253Aachs%26aulast%3DAndrs%26aufirst%3DM.%26aulast%3DHasanova%26aufirst%3DZ.%26aulast%3DOravetzova%26aufirst%3DA.%26aulast%3DDobrovolna%26aufirst%3DJ.%26aulast%3DJanscak%26aufirst%3DP.%26atitle%3DRECQ5%253A%2520A%2520Mysterious%2520Helicase%2520at%2520the%2520Interface%2520of%2520DNA%2520Replication%2520and%2520Transcription%26jtitle%3DGenes%26date%3D2020%26volume%3D11%26spage%3D232%26doi%3D10.3390%2Fgenes11020232" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schwendener, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raynard, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paliwal, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kanagaraj, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shevelev, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stark, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sung, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janscak, P.</span></span> <span> </span><span class="NLM_article-title">Physical Interaction of RECQ5 Helicase with RAD51 Facilitates Its Anti-Recombinase Activity</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>285</i></span>,  <span class="NLM_fpage">15739</span>– <span class="NLM_lpage">15745</span>, <span class="refDoi"> DOI: 10.1074/jbc.M110.110478</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=10.1074%2Fjbc.M110.110478" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=20348101" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=1%3ACAS%3A528%3ADC%252BC3cXmtVyks7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=285&publication_year=2010&pages=15739-15745&author=S.+Schwendenerauthor=S.+Raynardauthor=S.+Paliwalauthor=A.+Chengauthor=R.+Kanagarajauthor=I.+Shevelevauthor=J.+M.+Starkauthor=P.+Sungauthor=P.+Janscak&title=Physical+Interaction+of+RECQ5+Helicase+with+RAD51+Facilitates+Its+Anti-Recombinase+Activity&doi=10.1074%2Fjbc.M110.110478"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Physical Interaction of RECQ5 Helicase with RAD51 Facilitates Its Anti-recombinase Activity</span></div><div class="casAuthors">Schwendener, Sybille; Raynard, Steven; Paliwal, Shreya; Cheng, Anita; Kanagaraj, Radhakrishnan; Shevelev, Igor; Stark, Jeremy M.; Sung, Patrick; Janscak, Pavel</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">285</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">15739-15745</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Homologous recombination (HR) provides an efficient mechanism for error-free repair of DNA double-strand breaks (DSBs).  However, HR can also be harmful, as inappropriate or untimely HR events can give rise to lethal recombination intermediates and chromosome rearrangements.  A crit. step of HR is the formation of a RAD51 filament on single-stranded (ss)DNA, which mediates the invasion of a homologous DNA mol.  In mammalian cells, several DNA helicases have been implicated in the regulation of this process.  RECQ5, a member of the RecQ family of DNA helicases, interacts phys. with the RAD51 recombinase and disrupts RAD51 presynaptic filaments in a reaction dependent on ATP hydrolysis.  Here, we have precisely mapped the RAD51-interacting domain of RECQ5 and generated mutants that fail to interact with RAD51.  We show that although these mutants retain normal ATPase activity, they are impaired in their ability to displace RAD51 from ssDNA.  Moreover, we show that ablation of RECQ5-RAD51 complex formation by a point mutation alleviates the inhibitory effect of RECQ5 on HR-mediated DSB repair.  These findings provide support for the proposal that interaction with RAD51 is crit. for the anti-recombinase attribute of RECQ5.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDTDB-CKfN5rVg90H21EOLACvtfcHk0lj6igpg3UHxhw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXmtVyks7Y%253D&md5=3185152ffc63f32bc0857e7c04ff2c17</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M110.110478&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M110.110478%26sid%3Dliteratum%253Aachs%26aulast%3DSchwendener%26aufirst%3DS.%26aulast%3DRaynard%26aufirst%3DS.%26aulast%3DPaliwal%26aufirst%3DS.%26aulast%3DCheng%26aufirst%3DA.%26aulast%3DKanagaraj%26aufirst%3DR.%26aulast%3DShevelev%26aufirst%3DI.%26aulast%3DStark%26aufirst%3DJ.%2BM.%26aulast%3DSung%26aufirst%3DP.%26aulast%3DJanscak%26aufirst%3DP.%26atitle%3DPhysical%2520Interaction%2520of%2520RECQ5%2520Helicase%2520with%2520RAD51%2520Facilitates%2520Its%2520Anti-Recombinase%2520Activity%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2010%26volume%3D285%26spage%3D15739%26epage%3D15745%26doi%3D10.1074%2Fjbc.M110.110478" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hosono, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abe, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishiai, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Islam, M. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arakawa, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takeda, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishii, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takata, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seki, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Enomoto, T.</span></span> <span> </span><span class="NLM_article-title">Tumor suppressor RecQL5 controls recombination induced by DNA crosslinking agents</span>. <i>Biochim. Biophys. Acta, Mol. Cell Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>1843</i></span>,  <span class="NLM_fpage">1002</span>– <span class="NLM_lpage">1012</span>, <span class="refDoi"> DOI: 10.1016/j.bbamcr.2014.01.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=10.1016%2Fj.bbamcr.2014.01.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjsF2gt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1843&publication_year=2014&pages=1002-1012&author=Y.+Hosonoauthor=T.+Abeauthor=M.+Ishiaiauthor=M.+N.+Islamauthor=H.+Arakawaauthor=W.+Wangauthor=S.+Takedaauthor=Y.+Ishiiauthor=M.+Takataauthor=M.+Sekiauthor=T.+Enomoto&title=Tumor+suppressor+RecQL5+controls+recombination+induced+by+DNA+crosslinking+agents&doi=10.1016%2Fj.bbamcr.2014.01.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Tumor suppressor RecQL5 controls recombination induced by DNA crosslinking agents</span></div><div class="casAuthors">Hosono, Yoshifumi; Abe, Takuya; Ishiai, Masamichi; Islam, M. Nurul; Arakawa, Hiroshi; Wang, Weidong; Takeda, Shunichi; Ishii, Yutaka; Takata, Minoru; Seki, Masayuki; Enomoto, Takemi</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Molecular Cell Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">1843</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1002-1012</span>CODEN:
                <span class="NLM_cas:coden">BBAMCO</span>;
        ISSN:<span class="NLM_cas:issn">0167-4889</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">RecQ family DNA helicases function in the maintenance of genome stability.  Mice deficient in RecQL5, one of five RecQ helicases, show a cancer predisposition phenotype, suggesting that RecQL5 plays a tumor suppressor role.  RecQL5 interacts with Rad51, a key factor in homologous recombination (HR), and displaces Rad51 from Rad51-single stranded DNA (ssDNA) filaments in vitro.  However, the precise roles of RecQL5 in the cell remain elusive.  Here, we present evidence suggesting that RecQL5 is involved in DNA interstrand crosslink (ICL) repair.  Chicken DT40 RECQL5 gene knockout (KO) cells showed sensitivity to ICL-inducing agents such as cisplatin (CDDP) and mitomycin C (MMC) and a higher no. of chromosome aberrations in the presence of MMC than wild-type cells.  The phenotypes of RECQL5 KO cells resembled those of Fanconi anemia gene KO cells.  Genetic anal. using corresponding gene knockout cells showed that RecQL5 is involved in the FANCD1 (BRCA2)-dependent ICL repair pathway in which Rad51-ssDNA filament formation is promoted by BRCA2.  The disappearance but not appearance of Rad51-foci was delayed in RECQL5 KO cells after MMC treatment.  Deletion of Rad54, which processes the Rad51-ssDNA filament in HR, in RECQL5 KO cells increased sensitivity to CDDP and further delayed the disappearance of Rad51-foci, suggesting that RecQL5 and Rad54 have different effects on the Rad51-ssDNA filament.  Furthermore, the frequency and variation of CDDP-induced gene conversion at the Ig locus were increased in RECQL5 KO cells.  These results suggest that RecQL5 plays a role in regulating the incidence and quality of ICL-induced recombination.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpyc66YbYJ5zrVg90H21EOLACvtfcHk0lj6igpg3UHxhw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjsF2gt7s%253D&md5=8833579d58e3977a80f479a925af7760</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.bbamcr.2014.01.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbamcr.2014.01.005%26sid%3Dliteratum%253Aachs%26aulast%3DHosono%26aufirst%3DY.%26aulast%3DAbe%26aufirst%3DT.%26aulast%3DIshiai%26aufirst%3DM.%26aulast%3DIslam%26aufirst%3DM.%2BN.%26aulast%3DArakawa%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DTakeda%26aufirst%3DS.%26aulast%3DIshii%26aufirst%3DY.%26aulast%3DTakata%26aufirst%3DM.%26aulast%3DSeki%26aufirst%3DM.%26aulast%3DEnomoto%26aufirst%3DT.%26atitle%3DTumor%2520suppressor%2520RecQL5%2520controls%2520recombination%2520induced%2520by%2520DNA%2520crosslinking%2520agents%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Mol.%2520Cell%2520Res.%26date%3D2014%26volume%3D1843%26spage%3D1002%26epage%3D1012%26doi%3D10.1016%2Fj.bbamcr.2014.01.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Røe, O. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szulkin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderssen, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flatberg, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sandeck, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amundsen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erlandsen, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dobra, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sundstrøm, S. H.</span></span> <span> </span><span class="NLM_article-title">Molecular Resistance Fingerprint of Pemetrexed and Platinum in a Long-Term Survivor of Mesothelioma</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>7</i></span>, <span class="NLM_elocation-id">e40521</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0040521</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=10.1371%2Fjournal.pone.0040521" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=22905093" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&author=O.+D.+R%C3%B8eauthor=A.+Szulkinauthor=E.+Anderssenauthor=A.+Flatbergauthor=H.+Sandeckauthor=T.+Amundsenauthor=S.+E.+Erlandsenauthor=K.+Dobraauthor=S.+H.+Sundstr%C3%B8m&title=Molecular+Resistance+Fingerprint+of+Pemetrexed+and+Platinum+in+a+Long-Term+Survivor+of+Mesothelioma&doi=10.1371%2Fjournal.pone.0040521"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0040521&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0040521%26sid%3Dliteratum%253Aachs%26aulast%3DR%25C3%25B8e%26aufirst%3DO.%2BD.%26aulast%3DSzulkin%26aufirst%3DA.%26aulast%3DAnderssen%26aufirst%3DE.%26aulast%3DFlatberg%26aufirst%3DA.%26aulast%3DSandeck%26aufirst%3DH.%26aulast%3DAmundsen%26aufirst%3DT.%26aulast%3DErlandsen%26aufirst%3DS.%2BE.%26aulast%3DDobra%26aufirst%3DK.%26aulast%3DSundstr%25C3%25B8m%26aufirst%3DS.%2BH.%26atitle%3DMolecular%2520Resistance%2520Fingerprint%2520of%2520Pemetrexed%2520and%2520Platinum%2520in%2520a%2520Long-Term%2520Survivor%2520of%2520Mesothelioma%26jtitle%3DPLoS%2520One%26date%3D2012%26volume%3D7%26doi%3D10.1371%2Fjournal.pone.0040521" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Györffy, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lanczky, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eklund, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denkert, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Budczies, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szallasi, Z.</span></span> <span> </span><span class="NLM_article-title">An Online Survival Analysis Tool to Rapidly Assess the Effect of 22,277 Genes on Breast Cancer Prognosis Using Microarray Data of 1,809 Patients</span>. <i>Breast Cancer Res. Treat.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>123</i></span>,  <span class="NLM_fpage">725</span>– <span class="NLM_lpage">731</span>, <span class="refDoi"> DOI: 10.1007/s10549-009-0674-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=10.1007%2Fs10549-009-0674-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=20020197" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=1%3ACAS%3A280%3ADC%252BC3cfit1Sksw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=123&publication_year=2010&pages=725-731&author=B.+Gy%C3%B6rffyauthor=A.+Lanczkyauthor=A.+C.+Eklundauthor=C.+Denkertauthor=J.+Budcziesauthor=Q.+Liauthor=Z.+Szallasi&title=An+Online+Survival+Analysis+Tool+to+Rapidly+Assess+the+Effect+of+22%2C277+Genes+on+Breast+Cancer+Prognosis+Using+Microarray+Data+of+1%2C809+Patients&doi=10.1007%2Fs10549-009-0674-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">An online survival analysis tool to rapidly assess the effect of 22,277 genes on breast cancer prognosis using microarray data of 1,809 patients</span></div><div class="casAuthors">Gyorffy Balazs; Lanczky Andras; Eklund Aron C; Denkert Carsten; Budczies Jan; Li Qiyuan; Szallasi Zoltan</div><div class="citationInfo"><span class="NLM_cas:title">Breast cancer research and treatment</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">123</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">725-31</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Validating prognostic or predictive candidate genes in appropriately powered breast cancer cohorts are of utmost interest.  Our aim was to develop an online tool to draw survival plots, which can be used to assess the relevance of the expression levels of various genes on the clinical outcome both in untreated and treated breast cancer patients.  A background database was established using gene expression data and survival information of 1,809 patients downloaded from GEO (Affymetrix HGU133A and HGU133+2 microarrays).  The median relapse free survival is 6.43 years, 968/1,231 patients are estrogen-receptor (ER) positive, and 190/1,369 are lymph-node positive.  After quality control and normalization only probes present on both Affymetrix platforms were retained (n = 22,277).  In order to analyze the prognostic value of a particular gene, the cohorts are divided into two groups according to the median (or upper/lower quartile) expression of the gene.  The two groups can be compared in terms of relapse free survival, overall survival, and distant metastasis free survival.  A survival curve is displayed, and the hazard ratio with 95% confidence intervals and logrank P value are calculated and displayed.  Additionally, three subgroups of patients can be assessed: systematically untreated patients, endocrine-treated ER positive patients, and patients with a distribution of clinical characteristics representative of those seen in general clinical practice in the US.  Web address: www.kmplot.com .  We used this integrative data analysis tool to confirm the prognostic power of the proliferation-related genes TOP2A and TOP2B, MKI67, CCND2, CCND3, CCNDE2, as well as CDKN1A, and TK2.  We also validated the capability of microarrays to determine estrogen receptor status in 1,231 patients.  The tool is highly valuable for the preliminary assessment of biomarkers, especially for research groups with limited bioinformatic resources.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRFYBCvpM8ROty3lHz94U-GfW6udTcc2eYHL2YleRnwY7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3cfit1Sksw%253D%253D&md5=0f2317107ba5739e3fa48aba9b965eab</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1007%2Fs10549-009-0674-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10549-009-0674-9%26sid%3Dliteratum%253Aachs%26aulast%3DGy%25C3%25B6rffy%26aufirst%3DB.%26aulast%3DLanczky%26aufirst%3DA.%26aulast%3DEklund%26aufirst%3DA.%2BC.%26aulast%3DDenkert%26aufirst%3DC.%26aulast%3DBudczies%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DQ.%26aulast%3DSzallasi%26aufirst%3DZ.%26atitle%3DAn%2520Online%2520Survival%2520Analysis%2520Tool%2520to%2520Rapidly%2520Assess%2520the%2520Effect%2520of%252022%252C277%2520Genes%2520on%2520Breast%2520Cancer%2520Prognosis%2520Using%2520Microarray%2520Data%2520of%25201%252C809%2520Patients%26jtitle%3DBreast%2520Cancer%2520Res.%2520Treat.%26date%3D2010%26volume%3D123%26spage%3D725%26epage%3D731%26doi%3D10.1007%2Fs10549-009-0674-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mgimpatsang, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Konstantinidou, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shishkina, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dömling, A.</span></span> <span> </span><span class="NLM_article-title">1,3,4-Oxadiazoles by Ugi-Tetrazole and Huisgen Reaction</span>. <i>Org. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">7320</span>– <span class="NLM_lpage">7323</span>, <span class="refDoi"> DOI: 10.1021/acs.orglett.9b02614</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.orglett.9b02614" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhs1yntrzE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2019&pages=7320-7323&author=Q.+Wangauthor=K.+C.+Mgimpatsangauthor=M.+Konstantinidouauthor=S.+V.+Shishkinaauthor=A.+D%C3%B6mling&title=1%2C3%2C4-Oxadiazoles+by+Ugi-Tetrazole+and+Huisgen+Reaction&doi=10.1021%2Facs.orglett.9b02614"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">1,3,4-Oxadiazoles by Ugi-Tetrazole and Huisgen Reaction</span></div><div class="casAuthors">Wang, Qian; Mgimpatsang, Kumchok C.; Konstantinidou, Markella; Shishkina, Svitlana V.; Doemling, Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Organic Letters</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">7320-7323</span>CODEN:
                <span class="NLM_cas:coden">ORLEF7</span>;
        ISSN:<span class="NLM_cas:issn">1523-7052</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Amines, aldehydes, Me3SiN3, and tert-octyl isonitrile t-BuCH2CMe2N+≡C- underwent Ugi reactions to yield aminoalkyltetrazoles such as I; acid cleavage of the tert-octyl group followed by Huisgen reactions with acyl chlorides in pyridine yielded oxadiazoles such as II.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp4GnJ4E_VO3LVg90H21EOLACvtfcHk0li32z6ApbYy6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhs1yntrzE&md5=a4ddb092f5245191e3bc5084ee1ac990</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Facs.orglett.9b02614&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.orglett.9b02614%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DQ.%26aulast%3DMgimpatsang%26aufirst%3DK.%2BC.%26aulast%3DKonstantinidou%26aufirst%3DM.%26aulast%3DShishkina%26aufirst%3DS.%2BV.%26aulast%3DD%25C3%25B6mling%26aufirst%3DA.%26atitle%3D1%252C3%252C4-Oxadiazoles%2520by%2520Ugi-Tetrazole%2520and%2520Huisgen%2520Reaction%26jtitle%3DOrg.%2520Lett.%26date%3D2019%26volume%3D21%26spage%3D7320%26epage%3D7323%26doi%3D10.1021%2Facs.orglett.9b02614" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Newman, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aitkenhead, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savitsky, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gileadi, O.</span></span> <span> </span><span class="NLM_article-title">Insights into the RecQ Helicase Mechanism Revealed by the Structure of the Helicase Domain of Human RECQL5</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">4231</span>– <span class="NLM_lpage">4243</span>, <span class="refDoi"> DOI: 10.1093/nar/gkw1362</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=10.1093%2Fnar%2Fgkw1362" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=28100692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsVyqsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2017&pages=4231-4243&author=J.+A.+Newmanauthor=H.+Aitkenheadauthor=P.+Savitskyauthor=O.+Gileadi&title=Insights+into+the+RecQ+Helicase+Mechanism+Revealed+by+the+Structure+of+the+Helicase+Domain+of+Human+RECQL5&doi=10.1093%2Fnar%2Fgkw1362"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Insights into the RecQ helicase mechanism revealed by the structure of the helicase domain of human RECQL5</span></div><div class="casAuthors">Newman, Joseph A.; Aitkenhead, Hazel; Savitsky, Pavel; Gileadi, Opher</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">4231-4243</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">1362-4962</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">RecQ helicases are important maintainers of genome integrity with distinct roles in almost every cellular process requiring access to DNA.  RECQL5 is one of five human RecQ proteins and is particularly versatile in this regard, forming protein complexes with a diverse set of cellular partners in order to coordinate its helicase activity to various processes including replication, recombination and DNA repair.  In this study, we have detd. crystal structures of the core helicase domain of RECQL5 both with and without the nucleotide ADP in two distinctly different ('Open' and 'Closed') conformations.  Small angle X-ray scattering studies show that the 'Open' form of the protein predominates in soln. and we discuss implications of this with regards to the RECQL5 mechanism and conformational changes.  We have measured the ATPase, helicase and DNA binding properties of various RECQL5 constructs and variants and discuss the role of these regions and residues in the various RECQL5 activities.  Finally, we have performed a systematic comparison of the RECQL5 structures with other RecQ family structures and based on these comparisons we have constructed a model for the mechano-chem. cycle of the common catalytic core of these helicases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGokJCPaZR5xoLVg90H21EOLACvtfcHk0li32z6ApbYy6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsVyqsrY%253D&md5=d2480e40ee405d59871a039759b06698</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkw1362&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkw1362%26sid%3Dliteratum%253Aachs%26aulast%3DNewman%26aufirst%3DJ.%2BA.%26aulast%3DAitkenhead%26aufirst%3DH.%26aulast%3DSavitsky%26aufirst%3DP.%26aulast%3DGileadi%26aufirst%3DO.%26atitle%3DInsights%2520into%2520the%2520RecQ%2520Helicase%2520Mechanism%2520Revealed%2520by%2520the%2520Structure%2520of%2520the%2520Helicase%2520Domain%2520of%2520Human%2520RECQL5%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2017%26volume%3D45%26spage%3D4231%26epage%3D4243%26doi%3D10.1093%2Fnar%2Fgkw1362" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gray, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, J.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamath-Loeb, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blank, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sopher, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oshima, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loeb, L. A.</span></span> <span> </span><span class="NLM_article-title">The Werner syndrome protein is a DNA helicase</span>. <i>Nat. Genet.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">100</span>– <span class="NLM_lpage">103</span>, <span class="refDoi"> DOI: 10.1038/ng0997-100</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=10.1038%2Fng0997-100" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=9288107" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=1%3ACAS%3A528%3ADyaK2sXlvVGgsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=1997&pages=100-103&author=M.+D.+Grayauthor=J.-C.+Shenauthor=A.+S.+Kamath-Loebauthor=A.+Blankauthor=B.+L.+Sopherauthor=G.+M.+Martinauthor=J.+Oshimaauthor=L.+A.+Loeb&title=The+Werner+syndrome+protein+is+a+DNA+helicase&doi=10.1038%2Fng0997-100"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">The Werner syndrome protein is a DNA helicase</span></div><div class="casAuthors">Gray, Matthew D.; Shen, Jiang-Cheng; Kamath-Loeb, Ashwini S.; Blank, A.; Sopher, Bryce L.; Martin, George M.; Oshima, Junko; Loeb, Lawrence A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">100-103</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature America</span>)
        </div><div class="casAbstract">Werner syndrome (WS) is an uncommon autosomal recessive disorder characterized by premature aging.  The clin. manifestations of WS, including atherosclerosis and osteoporosis, appear early in adulthood, and death in the fourth to sixth decade commonly ensues from myocardial infarction or cancer.  In accord with the aging phenotype, cells from WS patients have a reduced replicative life span in culture.  Genomic instability is obsd. at the cytogenetic level in the form of chromosome breaks and translocations and at the mol. level by multiple large deletions.  The Werner syndrome gene (WRN) has recently been cloned.  The predicted product is a 1,432-amino-acid protein whose central domain is homologous to members of the RecQ family of DNA helicases.  Such homol. does not necessarily mean that WRN encodes an active helicase.  For example, the Saccharomyces cerevisiae RAD26 gene protein and the human transcription-repair coupling factor CSB (Cockayne syndrome B) are highly homologous to known helicases, yet neither encodes an active helicase.  Moreover,the Bloom's syndrome gene (BLM), discovered before WRN, is also homologous to the RecQ family of DNA helicases, though we still await demonstration that it encodes an active helicase.  Here we report that the WS protein does indeed catalyze DNA unwinding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoNMwhnmtR8FrVg90H21EOLACvtfcHk0lhbsakFG3rtSA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXlvVGgsLo%253D&md5=f96ec899a353b498845fb6528e9eeb9c</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1038%2Fng0997-100&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fng0997-100%26sid%3Dliteratum%253Aachs%26aulast%3DGray%26aufirst%3DM.%2BD.%26aulast%3DShen%26aufirst%3DJ.-C.%26aulast%3DKamath-Loeb%26aufirst%3DA.%2BS.%26aulast%3DBlank%26aufirst%3DA.%26aulast%3DSopher%26aufirst%3DB.%2BL.%26aulast%3DMartin%26aufirst%3DG.%2BM.%26aulast%3DOshima%26aufirst%3DJ.%26aulast%3DLoeb%26aufirst%3DL.%2BA.%26atitle%3DThe%2520Werner%2520syndrome%2520protein%2520is%2520a%2520DNA%2520helicase%26jtitle%3DNat.%2520Genet.%26date%3D1997%26volume%3D17%26spage%3D100%26epage%3D103%26doi%3D10.1038%2Fng0997-100" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Garcia, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiricny, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">West, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janscak, P.</span></span> <span> </span><span class="NLM_article-title">Human RECQ5β, a protein with DNA helicase and strand-annealing activities in a single polypeptide</span>. <i>EMBO J.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">2882</span>– <span class="NLM_lpage">2891</span>, <span class="refDoi"> DOI: 10.1038/sj.emboj.7600301</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=10.1038%2Fsj.emboj.7600301" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=15241474" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=1%3ACAS%3A528%3ADC%252BD2cXlvValt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2004&pages=2882-2891&author=P.+L.+Garciaauthor=Y.+Liuauthor=J.+Jiricnyauthor=S.+C.+Westauthor=P.+Janscak&title=Human+RECQ5%CE%B2%2C+a+protein+with+DNA+helicase+and+strand-annealing+activities+in+a+single+polypeptide&doi=10.1038%2Fsj.emboj.7600301"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Human RECQ5β, a protein with DNA helicase and strand-annealing activities in a single polypeptide</span></div><div class="casAuthors">Garcia, Patrick L.; Liu, Yilun; Jiricny, Josef; West, Stephen C.; Janscak, Pavel</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Journal</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">2882-2891</span>CODEN:
                <span class="NLM_cas:coden">EMJODG</span>;
        ISSN:<span class="NLM_cas:issn">0261-4189</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Proteins belonging to the highly conserved RecQ helicase family are essential for the maintenance of genomic stability.  Here, we describe the biochem. properties of the human RECQ5β protein.  Like BLM and WRN, RECQ5β is an ATP-dependent 3'-5' DNA helicase that can promote migration of Holliday junctions.  However, RECQ5β required the single-stranded DNA-binding protein RPA in order to mediate the efficient unwinding of oligonucleotide-based substrates.  Surprisingly, we found that RECQ5β possesses an intrinsic DNA strand-annealing activity that is inhibited by RPA.  Anal. of deletion variants of RECQ5β revealed that the DNA helicase activity resides in the conserved N-terminal portion of the protein, whereas strand annealing is mediated by the unique C-terminal domain.  Moreover, the strand-annealing activity of RECQ5β was strongly inhibited by ATPγS, a poorly hydrolyzable analog of ATP.  This effect was alleviated by mutations in the ATP-binding motif of RECQ5β, indicating that the ATP-bound form of the protein cannot promote strand annealing.  This is the first demonstration of a DNA helicase with an intrinsic DNA strand-annealing function residing in a sep. domain.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpE9WNEn1Qrk7Vg90H21EOLACvtfcHk0lhbsakFG3rtSA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXlvValt7k%253D&md5=01afeb2481055108192a40602ce44093</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Fsj.emboj.7600301&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.emboj.7600301%26sid%3Dliteratum%253Aachs%26aulast%3DGarcia%26aufirst%3DP.%2BL.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DJiricny%26aufirst%3DJ.%26aulast%3DWest%26aufirst%3DS.%2BC.%26aulast%3DJanscak%26aufirst%3DP.%26atitle%3DHuman%2520RECQ5%25CE%25B2%252C%2520a%2520protein%2520with%2520DNA%2520helicase%2520and%2520strand-annealing%2520activities%2520in%2520a%2520single%2520polypeptide%26jtitle%3DEMBO%2520J.%26date%3D2004%26volume%3D23%26spage%3D2882%26epage%3D2891%26doi%3D10.1038%2Fsj.emboj.7600301" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Aggarwal, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sommers, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoemaker, R. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brosh, R. M.</span></span> <span> </span><span class="NLM_article-title">Inhibition of Helicase Activity by a Small Molecule Impairs Werner Syndrome Helicase (WRN) Function in the Cellular Response to DNA Damage or Replication Stress</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>108</i></span>,  <span class="NLM_fpage">1525</span>– <span class="NLM_lpage">1530</span>, <span class="refDoi"> DOI: 10.1073/pnas.1006423108</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=10.1073%2Fpnas.1006423108" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=21220316" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs1SmtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=108&publication_year=2011&pages=1525-1530&author=M.+Aggarwalauthor=J.+A.+Sommersauthor=R.+H.+Shoemakerauthor=R.+M.+Brosh&title=Inhibition+of+Helicase+Activity+by+a+Small+Molecule+Impairs+Werner+Syndrome+Helicase+%28WRN%29+Function+in+the+Cellular+Response+to+DNA+Damage+or+Replication+Stress&doi=10.1073%2Fpnas.1006423108"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of helicase activity by a small molecule impairs Werner syndrome helicase (WRN) function in the cellular response to DNA damage or replication stress</span></div><div class="casAuthors">Aggarwal, Monika; Sommers, Joshua A.; Shoemaker, Robert H.; Brosh, Robert M., Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">108</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1525-1530, S1525/1-S1525/9</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Modulation of DNA repair proteins by small mols. has attracted great interest.  An in vitro helicase activity screen was used to identify mols. that modulate DNA unwinding by Werner syndrome helicase (WRN), mutated in the premature aging disorder Werner syndrome.  A small mol. from the National Cancer Institute Diversity Set designated NSC 19630 [1-(propoxymethyl)-maleimide] was identified that inhibited WRN helicase activity but did not affect other DNA helicases [Bloom syndrome (BLM), Fanconi anemia group J (FANCJ), RECQ1, RecQ, UvrD, or DnaB].  Exposure of human cells to NSC 19630 dramatically impaired growth and proliferation, induced apoptosis in a WRN-dependent manner, and resulted in elevated γ-H2AX and proliferating cell nuclear antigen (PCNA) foci.  NSC 19630 exposure led to delayed S-phase progression, consistent with the accumulation of stalled replication forks, and to DNA damage in a WRN-dependent manner.  Exposure to NSC 19630 sensitized cancer cells to the G-quadruplex-binding compd. telomestatin or a poly(ADP ribose) polymerase (PARP) inhibitor.  Sublethal dosage of NSC 19630 and the chemotherapy drug topotecan acted synergistically to inhibit cell proliferation and induce DNA damage.  The use of this WRN helicase inhibitor mol. may provide insight into the importance of WRN-mediated pathway(s) important for DNA repair and the replicational stress response.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqRIkmkwi9yCrVg90H21EOLACvtfcHk0lhbsakFG3rtSA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhs1SmtL8%253D&md5=d01ee377da5c65cdec72b0403494fa0e</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1006423108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1006423108%26sid%3Dliteratum%253Aachs%26aulast%3DAggarwal%26aufirst%3DM.%26aulast%3DSommers%26aufirst%3DJ.%2BA.%26aulast%3DShoemaker%26aufirst%3DR.%2BH.%26aulast%3DBrosh%26aufirst%3DR.%2BM.%26atitle%3DInhibition%2520of%2520Helicase%2520Activity%2520by%2520a%2520Small%2520Molecule%2520Impairs%2520Werner%2520Syndrome%2520Helicase%2520%2528WRN%2529%2520Function%2520in%2520the%2520Cellular%2520Response%2520to%2520DNA%2520Damage%2520or%2520Replication%2520Stress%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2011%26volume%3D108%26spage%3D1525%26epage%3D1530%26doi%3D10.1073%2Fpnas.1006423108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tiwari, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gopalan Kutty, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaudhary, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vasanth Raj, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shenoy, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mallikarjuna Rao, C.</span></span> <span> </span><span class="NLM_article-title">Synthesis and Evaluation of Selected 1,3,4-Oxadiazole Derivatives for in Vitro Cytotoxicity and in Vivo Anti-Tumour Activity</span>. <i>Cytotechnology</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>68</i></span>,  <span class="NLM_fpage">2553</span>– <span class="NLM_lpage">2565</span>, <span class="refDoi"> DOI: 10.1007/s10616-016-9979-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=10.1007%2Fs10616-016-9979-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=27282155" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=1%3ACAS%3A528%3ADC%252BC28XpslWmtrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2016&pages=2553-2565&author=A.+Tiwariauthor=N.+Gopalan+Kuttyauthor=N.+Kumarauthor=A.+Chaudharyauthor=P.+Vasanth+Rajauthor=R.+Shenoyauthor=C.+Mallikarjuna+Rao&title=Synthesis+and+Evaluation+of+Selected+1%2C3%2C4-Oxadiazole+Derivatives+for+in+Vitro+Cytotoxicity+and+in+Vivo+Anti-Tumour+Activity&doi=10.1007%2Fs10616-016-9979-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and evaluation of selected 1,3,4-oxadiazole derivatives for in vitro cytotoxicity and in vivo anti-tumor activity</span></div><div class="casAuthors">Tiwari, Amit; Gopalan Kutty, N.; Kumar, Nitesh; Chaudhary, Anil; Vasanth Raj, P.; Shenoy, Rekha; Mallikarjuna Rao, C.</div><div class="citationInfo"><span class="NLM_cas:title">Cytotechnology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">68</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2553-2565</span>CODEN:
                <span class="NLM_cas:coden">CYTOER</span>;
        ISSN:<span class="NLM_cas:issn">0920-9069</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">The oxadiazole moiety is known for its anticancer activity through its antiangiogenic and mitostatic potential.  Taking this as a cue, the present study was designed to investigate the anti-cancer potential of selected oxadiazole derivs.  Twelve 1,3,4-oxadiazole derivs. (AMK OX-1 to AMK OX-12) were synthesized and were tested for IC50 values through brine shrimp lethality assay and MTT assay on HeLa and A549 cell lines.  Four compds., AMK OX-8, 9, 11 and 12 showed potential cytotoxicity activity with low IC50 value.  These compds. produced considerable cytotoxic effect on Hep-2 and A549 cancer cell lines.  However, they were found to be comparatively safer to normal cell lines, viz., V-79 cell lines than to the tested cancer cell lines, such as HeLa, A 549, and Hep2 cell lines.  The mechanism of cytotoxicity was evaluated through nuclear staining and DNA ladder assay.  Although DNA ladder assay showed DNA fragmentation (apoptotic phenomenon) in Hep-2 cells treated with only AMK OX-12, the staining procedures using acridine orange, ethidium bromide and propidium iodide showed apoptotic bodies in cells treated with AMK OX-8, 9 and 12 also.  In JCI staining on isolated mitochondria of Hep2 cells, AMK OX-8, 9-11 and 12 displayed increasing fluorescence intensity with time which confirmed involvement of mitochondrial pathway and intrinsic pathway of apoptosis.  All four compds. were found to be safe in acute oral toxicity study in Swiss albino mice.  These derivs. were effective in reducing tumor size and wt. in the in vivo DLA-induced solid tumor model.  They were found to be significantly effective in reducing tumor vol. and tumor wt.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGof2fUc8OvrebVg90H21EOLACvtfcHk0ljmST2KtCOxGA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XpslWmtrs%253D&md5=9b76d230f99b5ec927a56c5c49c52112</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1007%2Fs10616-016-9979-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10616-016-9979-9%26sid%3Dliteratum%253Aachs%26aulast%3DTiwari%26aufirst%3DA.%26aulast%3DGopalan%2BKutty%26aufirst%3DN.%26aulast%3DKumar%26aufirst%3DN.%26aulast%3DChaudhary%26aufirst%3DA.%26aulast%3DVasanth%2BRaj%26aufirst%3DP.%26aulast%3DShenoy%26aufirst%3DR.%26aulast%3DMallikarjuna%2BRao%26aufirst%3DC.%26atitle%3DSynthesis%2520and%2520Evaluation%2520of%2520Selected%25201%252C3%252C4-Oxadiazole%2520Derivatives%2520for%2520in%2520Vitro%2520Cytotoxicity%2520and%2520in%2520Vivo%2520Anti-Tumour%2520Activity%26jtitle%3DCytotechnology%26date%3D2016%26volume%3D68%26spage%3D2553%26epage%3D2565%26doi%3D10.1007%2Fs10616-016-9979-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hekal, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">El-Naggar, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abu El-Azm, F. S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">El-Sayed, W. M.</span></span> <span> </span><span class="NLM_article-title">Synthesis of New Oxadiazol-Phthalazinone Derivatives with Anti-Proliferative Activity; Molecular Docking, pro-Apoptotic, and Enzyme Inhibition Profile</span>. <i>RSC Adv.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">3675</span>– <span class="NLM_lpage">3688</span>, <span class="refDoi"> DOI: 10.1039/C9RA09016A</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=10.1039%2FC9RA09016A" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhsFWjsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2020&pages=3675-3688&author=M.+H.+Hekalauthor=A.+M.+El-Naggarauthor=F.+S.+M.+Abu+El-Azmauthor=W.+M.+El-Sayed&title=Synthesis+of+New+Oxadiazol-Phthalazinone+Derivatives+with+Anti-Proliferative+Activity%3B+Molecular+Docking%2C+pro-Apoptotic%2C+and+Enzyme+Inhibition+Profile&doi=10.1039%2FC9RA09016A"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis of new oxadiazol-phthalazinone derivatives with anti-proliferative activity; molecular docking, pro-apoptotic, and enzyme inhibition profile</span></div><div class="casAuthors">Hekal, Mohamed H.; El-Naggar, Abeer M.; Abu El-Azm, Fatma S. M.; El-Sayed, Wael M.</div><div class="citationInfo"><span class="NLM_cas:title">RSC Advances</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3675-3688</span>CODEN:
                <span class="NLM_cas:coden">RSCACL</span>;
        ISSN:<span class="NLM_cas:issn">2046-2069</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Background and aim: The current study reports the synthesis and biol. evaluation of two novel series of 4-(5-mercapto-1,3,4-oxadiazol-2-yl)phthalazin-1(2H)-one derivs.  Methods: The synthetic reactions were carried out under both conventional and ultrasonic irradn. conditions.  The anti-proliferative activity of the newly synthesized compds. against two human epithelial cell lines; liver (HepG2) and breast (MCF-7) in addn. to normal fibroblasts (WI-38) was investigated.  In addn. to mol. docking studies, the possible mechanism(s) of action were also explored.  Results: In general, an improvement in synthetic rates and yields was obsd. when reactions were carried out under sonication compared with classical conditions.  The structures of the products were established based on anal. and spectral data.  These derivs. arrested the cell cycle progression and/or induced apoptosis.  This has been manifested by the elevation in the expression of p53 and caspase 3, down-regulation of cdk1, and a redn. in the concns. of MAPK and Topo II at submicromolar concns.  The latter results confirmed the mol. docking study.  Conclusions: Compd. 1 had the best profile on the gene and protein levels (arresting cell cycle and inducing apoptosis).  The ability of compds. 1 and 2e to inhibit both MAPK and Topo II nominates these derivs. as potential candidates for further anticancer and antitumor studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPx4mdMUtvR7Vg90H21EOLACvtfcHk0ljmST2KtCOxGA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhsFWjsb8%253D&md5=d24537ae457e0d631561a4bfe2f49c70</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1039%2FC9RA09016A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC9RA09016A%26sid%3Dliteratum%253Aachs%26aulast%3DHekal%26aufirst%3DM.%2BH.%26aulast%3DEl-Naggar%26aufirst%3DA.%2BM.%26aulast%3DAbu%2BEl-Azm%26aufirst%3DF.%2BS.%2BM.%26aulast%3DEl-Sayed%26aufirst%3DW.%2BM.%26atitle%3DSynthesis%2520of%2520New%2520Oxadiazol-Phthalazinone%2520Derivatives%2520with%2520Anti-Proliferative%2520Activity%253B%2520Molecular%2520Docking%252C%2520pro-Apoptotic%252C%2520and%2520Enzyme%2520Inhibition%2520Profile%26jtitle%3DRSC%2520Adv.%26date%3D2020%26volume%3D10%26spage%3D3675%26epage%3D3688%26doi%3D10.1039%2FC9RA09016A" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barnes, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lou, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, G.</span></span> <span> </span><span class="NLM_article-title">Recql5 and Blm RecQ DNA Helicases Have Nonredundant Roles in Suppressing Crossovers</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">3431</span>– <span class="NLM_lpage">3442</span>, <span class="refDoi"> DOI: 10.1128/MCB.25.9.3431-3442.2005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=10.1128%2FMCB.25.9.3431-3442.2005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=15831450" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=1%3ACAS%3A528%3ADC%252BD2MXjvFSmsb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2005&pages=3431-3442&author=Y.+Huauthor=X.+Luauthor=E.+Barnesauthor=M.+Yanauthor=H.+Louauthor=G.+Luo&title=Recql5+and+Blm+RecQ+DNA+Helicases+Have+Nonredundant+Roles+in+Suppressing+Crossovers&doi=10.1128%2FMCB.25.9.3431-3442.2005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Recql5 and Blm RecQ DNA helicases have nonredundant roles in suppressing crossovers</span></div><div class="casAuthors">Hu, Yiduo; Lu, Xincheng; Barnes, Ellen; Yan, Min; Lou, Hua; Luo, Guangbin</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3431-3442</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">In eukaryotes, crossovers in mitotic cells can have deleterious consequences and therefore must be suppressed.  Mutations in BLM give rise to Bloom syndrome, a disease that is characterized by an elevated rate of crossovers and increased cancer susceptibility.  However, simple eukaryotes such as Saccharomyces cerevisiae have multiple pathways for suppressing crossovers, suggesting that mammals also have multiple pathways for controlling crossovers in their mitotic cells.  We show here that in mouse embryonic stem (ES) cells, mutations in either the Bloom syndrome homolog (Blm) or the Recql5 genes result in a significant increase in the frequency of sister chromatid exchange (SCE), whereas deleting both Blm and Recql5 lead to an even higher frequency of SCE.  These data indicate that Blm and Recql5 have nonredundant roles in suppressing crossovers in mouse ES cells.  Furthermore, we show that mouse embryonic fibroblasts derived from Recql5 knockout mice also exhibit a significantly increased frequency of SCE compared with the corresponding wild-type control.  Thus, this study identifies a previously unknown Recql5-dependent, Blm-independent pathway for suppressing crossovers during mitosis in mice.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq3DOZyXmRYxbVg90H21EOLACvtfcHk0ljmST2KtCOxGA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXjvFSmsb0%253D&md5=e80d835a8ad2cb3527757705a24f8681</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1128%2FMCB.25.9.3431-3442.2005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.25.9.3431-3442.2005%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DY.%26aulast%3DLu%26aufirst%3DX.%26aulast%3DBarnes%26aufirst%3DE.%26aulast%3DYan%26aufirst%3DM.%26aulast%3DLou%26aufirst%3DH.%26aulast%3DLuo%26aufirst%3DG.%26atitle%3DRecql5%2520and%2520Blm%2520RecQ%2520DNA%2520Helicases%2520Have%2520Nonredundant%2520Roles%2520in%2520Suppressing%2520Crossovers%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2005%26volume%3D25%26spage%3D3431%26epage%3D3442%26doi%3D10.1128%2FMCB.25.9.3431-3442.2005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raynard, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sehorn, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bussen, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stark, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barnes, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janscak, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jasin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vogel, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sung, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, G.</span></span> <span> </span><span class="NLM_article-title">RECQL5/Recql5 Helicase Regulates Homologous Recombination and Suppresses Tumour Formation via Disruption of Rad51 Presynaptic Filaments</span>. <i>Genes Dev.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">3073</span>– <span class="NLM_lpage">3084</span>, <span class="refDoi"> DOI: 10.1101/gad.1609107</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=10.1101%2Fgad.1609107" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=18003859" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhsVaht7vO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2007&pages=3073-3084&author=Y.+Huauthor=S.+Raynardauthor=M.+G.+Sehornauthor=X.+Luauthor=W.+Bussenauthor=L.+Zhengauthor=J.+M.+Starkauthor=E.+L.+Barnesauthor=P.+Chiauthor=P.+Janscakauthor=M.+Jasinauthor=H.+Vogelauthor=P.+Sungauthor=G.+Luo&title=RECQL5%2FRecql5+Helicase+Regulates+Homologous+Recombination+and+Suppresses+Tumour+Formation+via+Disruption+of+Rad51+Presynaptic+Filaments&doi=10.1101%2Fgad.1609107"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">RECQL5/Recq15 helicase regulates homologous recombination and suppresses tumor formation via disruption of Rad51 presynaptic filaments</span></div><div class="casAuthors">Hu, Yiduo; Raynard, Steven; Sehorn, Michael G.; Lu, Xincheng; Bussen, Wendy; Zheng, Lu; Stark, Jeremy M.; Barnes, Ellen L.; Chi, Peter; Janscak, Pavel; Jasin, Maria; Vogel, Hannes; Sung, Patrick; Luo, Guangbin</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">3073-3084</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">Members of the RecQ helicase family play crit. roles in genome maintenance.  There are five RecQ homologs in mammals, and defects in three of these (BLM, WRN, and RECQL4) give rise to cancer predisposition syndromes in humans.  RECQL and RECQL5 have not been assocd. with a human disease.  Here we show that deletion of Recq15 in mice results in cancer susceptibility.  Racq15-deficient cells exhibit elevated frequencies of spontaneous DNA double-strand breaks and homologous recombination (HR) as scored using a reporter that harbors a direct repeat, and are prone to gross chromosomal rearrangements in response to replication stress.  To understand how RECQL5 regulates HR, we use purified proteins to demonstrate that human RECQL5 binds the Rad51 recombinase and inhibits Rad51-mediated D-loop formation.  By biochem. means and electron microscopy, we show that RECQL5 displaces Rad51 from single-stranded DNA (ssDNA) in a reaction that requires ATP hydrolysis and RPA.  Together, our results identify RECQL5 as an important tumor suppressor that may act by preventing inappropriate HR events via Rad51 presynaptic filament disruption.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqKBA40NBcFrVg90H21EOLACvtfcHk0ljj7cGHTw7tvg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhsVaht7vO&md5=85ddf3085111874a24beca1b9c4a91ed</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1101%2Fgad.1609107&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.1609107%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DY.%26aulast%3DRaynard%26aufirst%3DS.%26aulast%3DSehorn%26aufirst%3DM.%2BG.%26aulast%3DLu%26aufirst%3DX.%26aulast%3DBussen%26aufirst%3DW.%26aulast%3DZheng%26aufirst%3DL.%26aulast%3DStark%26aufirst%3DJ.%2BM.%26aulast%3DBarnes%26aufirst%3DE.%2BL.%26aulast%3DChi%26aufirst%3DP.%26aulast%3DJanscak%26aufirst%3DP.%26aulast%3DJasin%26aufirst%3DM.%26aulast%3DVogel%26aufirst%3DH.%26aulast%3DSung%26aufirst%3DP.%26aulast%3DLuo%26aufirst%3DG.%26atitle%3DRECQL5%252FRecql5%2520Helicase%2520Regulates%2520Homologous%2520Recombination%2520and%2520Suppresses%2520Tumour%2520Formation%2520via%2520Disruption%2520of%2520Rad51%2520Presynaptic%2520Filaments%26jtitle%3DGenes%2520Dev.%26date%3D2007%26volume%3D21%26spage%3D3073%26epage%3D3084%26doi%3D10.1101%2Fgad.1609107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Paliwal, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kanagaraj, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sturzenegger, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burdova, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janscak, P.</span></span> <span> </span><span class="NLM_article-title">Human RECQ5 Helicase Promotes Repair of DNA Double-Strand Breaks by Synthesis-Dependent Strand Annealing</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">2380</span>– <span class="NLM_lpage">2390</span>, <span class="refDoi"> DOI: 10.1093/nar/gkt1263</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=10.1093%2Fnar%2Fgkt1263" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=24319145" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjtlOrur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2014&pages=2380-2390&author=S.+Paliwalauthor=R.+Kanagarajauthor=A.+Sturzeneggerauthor=K.+Burdovaauthor=P.+Janscak&title=Human+RECQ5+Helicase+Promotes+Repair+of+DNA+Double-Strand+Breaks+by+Synthesis-Dependent+Strand+Annealing&doi=10.1093%2Fnar%2Fgkt1263"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Human RECQ5 helicase promotes repair of DNA double-strand breaks by synthesis-dependent strand annealing</span></div><div class="casAuthors">Paliwal, Shreya; Kanagaraj, Radhakrishnan; Sturzenegger, Andreas; Burdova, Kamila; Janscak, Pavel</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">2380-2390</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Most mitotic homologous recombination (HR) events proceed via a synthesis-dependent strand annealing mechanism to avoid crossing over, which may give rise to chromosomal rearrangements and loss of heterozygosity.  The mol. mechanisms controlling HR sub-pathway choice are poorly understood.  Here, we show that human RECQ5, a DNA helicase that can disrupt RAD51 nucleoprotein filaments, promotes formation of non-crossover products during DNA double-strand break-induced HR and counteracts the inhibitory effect of RAD51 on RAD52-mediated DNA annealing in vitro and in vivo.  Moreover, we demonstrate that RECQ5 deficiency is assocd. with an increased occupancy of RAD51 at a double-strand break site, and it also causes an elevation of sister chromatid exchanges on inactivation of the Holliday junction dissoln. pathway or on induction of a high load of DNA damage in the cell.  Collectively, our findings suggest that RECQ5 acts during the post-synaptic phase of synthesis-dependent strand annealing to prevent formation of aberrant RAD51 filaments on the extended invading strand, thus limiting its channeling into potentially hazardous crossover pathway of HR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqRn3s7AR2epbVg90H21EOLACvtfcHk0ljj7cGHTw7tvg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjtlOrur0%253D&md5=92435bd89e4ae52eefdb97e8844b09fb</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkt1263&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkt1263%26sid%3Dliteratum%253Aachs%26aulast%3DPaliwal%26aufirst%3DS.%26aulast%3DKanagaraj%26aufirst%3DR.%26aulast%3DSturzenegger%26aufirst%3DA.%26aulast%3DBurdova%26aufirst%3DK.%26aulast%3DJanscak%26aufirst%3DP.%26atitle%3DHuman%2520RECQ5%2520Helicase%2520Promotes%2520Repair%2520of%2520DNA%2520Double-Strand%2520Breaks%2520by%2520Synthesis-Dependent%2520Strand%2520Annealing%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2014%26volume%3D42%26spage%3D2380%26epage%3D2390%26doi%3D10.1093%2Fnar%2Fgkt1263" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Islam, M. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paquet, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fox, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dray, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, X.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sung, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span> <span> </span><span class="NLM_article-title">A Variant of the Breast Cancer Type 2 Susceptibility Protein (BRC) Repeat Is Essential for the RECQL5 Helicase to Interact with RAD51 Recombinase for Genome Stabilization</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>287</i></span>,  <span class="NLM_fpage">23808</span>– <span class="NLM_lpage">23818</span>, <span class="refDoi"> DOI: 10.1074/jbc.M112.375014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=10.1074%2Fjbc.M112.375014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=22645136" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=1%3ACAS%3A528%3ADC%252BC38XpvVejurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=287&publication_year=2012&pages=23808-23818&author=M.+N.+Islamauthor=N.+Paquetauthor=D.+Foxauthor=E.+Drayauthor=X.-F.+Zhengauthor=H.+Kleinauthor=P.+Sungauthor=W.+Wang&title=A+Variant+of+the+Breast+Cancer+Type+2+Susceptibility+Protein+%28BRC%29+Repeat+Is+Essential+for+the+RECQL5+Helicase+to+Interact+with+RAD51+Recombinase+for+Genome+Stabilization&doi=10.1074%2Fjbc.M112.375014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">A variant of the breast cancer type 2 susceptibility protein (BRC) repeat is essential for the RECQL5 helicase to interact with RAD51 recombinase for genome stabilization</span></div><div class="casAuthors">Islam, M. Nurul; Paquet, Nicolas; Fox, David, III; Dray, Eloise; Zheng, Xiao-Feng; Klein, Hannah; Sung, Patrick; Wang, Weidong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">287</span>
        (<span class="NLM_cas:issue">28</span>),
    <span class="NLM_cas:pages">23808-23818</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The BRC repeat is a structural motif in the tumor suppressor BRCA2 (breast cancer type 2 susceptibility protein), which promotes homologous recombination (HR) by regulating RAD51 recombinase activity.  To date, the BRC repeat has not been obsd. in other proteins, so that its role in HR is inferred only in the context of BRCA2.  Here, we identified a BRC repeat variant, named BRCv, in the RECQL5 helicase, which possesses anti-recombinase activity in vitro and suppresses HR and promotes cellular resistance to camptothecin-induced replication stress in vivo.  RECQL5-BRCv interacted with RAD51 through two conserved motifs similar to those in the BRCA2-BRC repeat.  Mutations of either motif compromised functions of RECQL5, including assocn. with RAD51, inhibition of RAD51-mediated D-loop formation, suppression of sister chromatid exchange, and resistance to camptothecin-induced replication stress.  Potential BRCvs were also found in other HR regulatory proteins, including Srs2 and Sgs1, which possess anti-recombinase activities similar to that of RECQL5.  A point mutation in the predicted Srs2-BRCv disrupted the ability of the protein to bind RAD51 and to inhibit D-loop formation.  Thus, BRC is a common RAD51 interaction module that can be utilized by different proteins to either promote HR, as in the case of BRCA2, or to suppress HR, as in RECQL5.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgx_Xhss7s9bVg90H21EOLACvtfcHk0ljj7cGHTw7tvg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XpvVejurw%253D&md5=e9774611014fb47f3d7d43f70f67cd34</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M112.375014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M112.375014%26sid%3Dliteratum%253Aachs%26aulast%3DIslam%26aufirst%3DM.%2BN.%26aulast%3DPaquet%26aufirst%3DN.%26aulast%3DFox%26aufirst%3DD.%26aulast%3DDray%26aufirst%3DE.%26aulast%3DZheng%26aufirst%3DX.-F.%26aulast%3DKlein%26aufirst%3DH.%26aulast%3DSung%26aufirst%3DP.%26aulast%3DWang%26aufirst%3DW.%26atitle%3DA%2520Variant%2520of%2520the%2520Breast%2520Cancer%2520Type%25202%2520Susceptibility%2520Protein%2520%2528BRC%2529%2520Repeat%2520Is%2520Essential%2520for%2520the%2520RECQL5%2520Helicase%2520to%2520Interact%2520with%2520RAD51%2520Recombinase%2520for%2520Genome%2520Stabilization%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2012%26volume%3D287%26spage%3D23808%26epage%3D23818%26doi%3D10.1074%2Fjbc.M112.375014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Helleday, T.</span></span> <span> </span><span class="NLM_article-title">Homologous Recombination in Cancer Development, Treatment and Development of Drug Resistance</span>. <i>Carcinogenesis</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">955</span>– <span class="NLM_lpage">960</span>, <span class="refDoi"> DOI: 10.1093/carcin/bgq064</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=10.1093%2Fcarcin%2Fbgq064" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=20351092" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=1%3ACAS%3A528%3ADC%252BC3cXmslSqsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2010&pages=955-960&author=T.+Helleday&title=Homologous+Recombination+in+Cancer+Development%2C+Treatment+and+Development+of+Drug+Resistance&doi=10.1093%2Fcarcin%2Fbgq064"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Homologous recombination in cancer development, treatment and development of drug resistance</span></div><div class="casAuthors">Helleday, Thomas</div><div class="citationInfo"><span class="NLM_cas:title">Carcinogenesis</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">955-960</span>CODEN:
                <span class="NLM_cas:coden">CRNGDP</span>;
        ISSN:<span class="NLM_cas:issn">0143-3334</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Although DNA double-strand breaks (DSBs) are substrates for homologous recombination (HR) repair, it is becoming apparent that DNA lesions produced at replication forks, for instance by many anticancer drugs, are more significant substrates for HR repair.  Cells defective in HR are hypersensitive to a wide variety of anticancer drugs, including those that do not produce DSBs.  Several cancers have mutations in or epigenetically silenced HR genes, which explain the genetic instability that drives cancer development.  There are an increasing no. of reports suggesting that mutation or epigenetic silencing of HR genes explains the sensitivity of cancers to current chemotherapy treatments.  Furthermore, there are also many examples of re-expression of HR genes in tumors to explain drug resistance.  Emerging data suggest that there are several different subpathways of HR, which can compensate for each other.  Unravelling the overlapping pathways in HR showed that BRCA1- and BRCA2-defective cells rely on the PARP protein for survival.  This synthetic lethal interaction is now being exploited for selective treatment of BRCA1- and BRCA2-defective cancers with PARP inhibitors.  Here, I discuss the diversity of HR and how it impacts on cancer with a particular focus on how HR can be exploited in future anticancer strategies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGraaL2tqlRLWrVg90H21EOLACvtfcHk0ljj7cGHTw7tvg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXmslSqsbY%253D&md5=1385e375ac39dd982ff9892fc8492486</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1093%2Fcarcin%2Fbgq064&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fcarcin%252Fbgq064%26sid%3Dliteratum%253Aachs%26aulast%3DHelleday%26aufirst%3DT.%26atitle%3DHomologous%2520Recombination%2520in%2520Cancer%2520Development%252C%2520Treatment%2520and%2520Development%2520of%2520Drug%2520Resistance%26jtitle%3DCarcinogenesis%26date%3D2010%26volume%3D31%26spage%3D955%26epage%3D960%26doi%3D10.1093%2Fcarcin%2Fbgq064" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hengel, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spies, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spies, M.</span></span> <span> </span><span class="NLM_article-title">Small-Molecule Inhibitors Targeting DNA Repair and DNA Repair Deficiency in Research and Cancer Therapy</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">1101</span>– <span class="NLM_lpage">1119</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2017.08.027</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=10.1016%2Fj.chembiol.2017.08.027" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=28938088" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFGgurbE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2017&pages=1101-1119&author=S.+R.+Hengelauthor=M.+A.+Spiesauthor=M.+Spies&title=Small-Molecule+Inhibitors+Targeting+DNA+Repair+and+DNA+Repair+Deficiency+in+Research+and+Cancer+Therapy&doi=10.1016%2Fj.chembiol.2017.08.027"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Small-Molecule Inhibitors Targeting DNA Repair and DNA Repair Deficiency in Research and Cancer Therapy</span></div><div class="casAuthors">Hengel, Sarah R.; Spies, M. Ashley; Spies, Maria</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1101-1119</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">To maintain stable genomes and to avoid cancer and aging, cells need to repair a multitude of deleterious DNA lesions, which arise constantly in every cell.  Processes that support genome integrity in normal cells, however, allow cancer cells to develop resistance to radiation and DNA-damaging chemotherapeutics.  Chem. inhibition of the key DNA repair proteins and pharmacol. induced synthetic lethality have become instrumental in both dissecting the complex DNA repair networks and as promising anticancer agents.  The difficulty in capitalizing on synthetically lethal interactions in cancer cells is that many potential targets do not possess well-defined small-mol. binding determinates.  In this review, we discuss several successful campaigns to identify and leverage small-mol. inhibitors of the DNA repair proteins, from PARP1, a paradigm case for clin. successful small-mol. inhibitors, to coveted new targets, such as RAD51 recombinase, RAD52 DNA repair protein, MRE11 nuclease, and WRN DNA helicase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZweRKd9gHmLVg90H21EOLACvtfcHk0liwh021oLuFrA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFGgurbE&md5=b8b865f8df43100effd60037efdc96b5</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2017.08.027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2017.08.027%26sid%3Dliteratum%253Aachs%26aulast%3DHengel%26aufirst%3DS.%2BR.%26aulast%3DSpies%26aufirst%3DM.%2BA.%26aulast%3DSpies%26aufirst%3DM.%26atitle%3DSmall-Molecule%2520Inhibitors%2520Targeting%2520DNA%2520Repair%2520and%2520DNA%2520Repair%2520Deficiency%2520in%2520Research%2520and%2520Cancer%2520Therapy%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2017%26volume%3D24%26spage%3D1101%26epage%3D1119%26doi%3D10.1016%2Fj.chembiol.2017.08.027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tyagi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patro, B. S.</span></span> <span> </span><span class="NLM_article-title">Salinomycin Reduces Growth, Proliferation and Metastasis of Cisplatin Resistant Breast Cancer Cells via NF-KB Deregulation</span>. <i>Toxicol. In Vitro</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">125</span>– <span class="NLM_lpage">133</span>, <span class="refDoi"> DOI: 10.1016/j.tiv.2019.05.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=10.1016%2Fj.tiv.2019.05.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=31077746" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtVagtbvF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2019&pages=125-133&author=M.+Tyagiauthor=B.+S.+Patro&title=Salinomycin+Reduces+Growth%2C+Proliferation+and+Metastasis+of+Cisplatin+Resistant+Breast+Cancer+Cells+via+NF-KB+Deregulation&doi=10.1016%2Fj.tiv.2019.05.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Salinomycin reduces growth, proliferation and metastasis of cisplatin resistant breast cancer cells via NF-kB deregulation</span></div><div class="casAuthors">Tyagi, Mrityunjay; Patro, Birija Sankar</div><div class="citationInfo"><span class="NLM_cas:title">Toxicology In Vitro</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">125-133</span>CODEN:
                <span class="NLM_cas:coden">TIVIEQ</span>;
        ISSN:<span class="NLM_cas:issn">0887-2333</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Cisplatin (cis-diamminedichloro-platinum, CDDP), is a widely used platinum compd. for various solid tumors including breast cancer as first line of therapy.  However, its pos. effects are limited due to acquired drug resistance and severe side effects in non-malignant tissue, esp. due to dose-dependent nephro- and/or neuro-toxicity.  Salinomycin is an antibiotic with coccidiostat effect and has shown anticancer efficacy against various cancer cells with selectivity in targeting cancer stem cells.  In the present study, anticancer efficacy and mechanism of action of salinomycin in CDDP-resistant human breast cancer (MCF7DDP) cells has been examd.  Initially, we generated CDDP-resistant cells by a new protocol followed by checking the anticancer efficacy of salinomycin through MTT, clonogenic, annexin-V/PI and sub-G1 assay.  Our results demonstrated that salinomycin diminished both cell proliferation and metastatic migration of MCF7DDP cells.  Salinomycin also induced mitochondrial dysfunction in CDDP-resistant breast cancer cells.  The anal. of nuclear translocation of pro-survival transcription factors by western blotting showed a distinct role of p65 (NF-κB) in CDDP-mediated resistance in breast cancer.  Salinomycin abrogated nuclear translocation of NF-κB proteins and also caused a concurrent redn. in NF-κB regulated expression of pro-survival proteins e.g., survivin, XIAP and BCL-2 in CDDP-resistant cells.  These results suggest that a follow up treatment of salinomycin may be promising strategy against CDDP resistant breast cancer cells and metastasis and help in reducing CDDP-induced side effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWxauTvGowGrVg90H21EOLACvtfcHk0liwh021oLuFrA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtVagtbvF&md5=82ad43598dc72d07cd086ef1f4c7abc8</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2Fj.tiv.2019.05.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tiv.2019.05.004%26sid%3Dliteratum%253Aachs%26aulast%3DTyagi%26aufirst%3DM.%26aulast%3DPatro%26aufirst%3DB.%2BS.%26atitle%3DSalinomycin%2520Reduces%2520Growth%252C%2520Proliferation%2520and%2520Metastasis%2520of%2520Cisplatin%2520Resistant%2520Breast%2520Cancer%2520Cells%2520via%2520NF-KB%2520Deregulation%26jtitle%3DToxicol.%2520In%2520Vitro%26date%3D2019%26volume%3D60%26spage%3D125%26epage%3D133%26doi%3D10.1016%2Fj.tiv.2019.05.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sarbajna, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">West, S. C.</span></span> <span> </span><span class="NLM_article-title">Roles of SLX1-SLX4, MUS81-EME1, and GEN1 in Avoiding Genome Instability and Mitotic Catastrophe</span>. <i>Genes Dev.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">1124</span>– <span class="NLM_lpage">1136</span>, <span class="refDoi"> DOI: 10.1101/gad.238303.114</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=10.1101%2Fgad.238303.114" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=24831703" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=1%3ACAS%3A528%3ADC%252BC2cXpt12qs7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2014&pages=1124-1136&author=S.+Sarbajnaauthor=D.+Daviesauthor=S.+C.+West&title=Roles+of+SLX1-SLX4%2C+MUS81-EME1%2C+and+GEN1+in+Avoiding+Genome+Instability+and+Mitotic+Catastrophe&doi=10.1101%2Fgad.238303.114"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Roles of SLX1-SLX4, MUS81-EME1, and GEN1 in avoiding genome instability and mitotic catastrophe</span></div><div class="casAuthors">Sarbajna, Shriparna; Davies, Derek; West, Stephen C.</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1124-1136</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">The resoln. of recombination intermediates contg. Holliday junctions (HJs) is crit. for genome maintenance and proper chromosome segregation.  Three pathways for HJ processing exist in human cells and involve the following enzymes/complexes: BLM-TopoIIIα-RMI1-RMI2 (BTR complex), SLX1-SLX4-MUS81-EME1 (SLX-MUS complex), and GEN1.  Cycling cells preferentially use the BTR complex for the removal of double HJs in S phase, with SLX-MUS and GEN1 acting at temporally distinct phases of the cell cycle.  Cells lacking SLX-MUS and GEN1 exhibit chromosome missegregation, micronucleus formation, and elevated levels of 53BP1-pos. G1 nuclear bodies, suggesting that defects in chromosome segregation lead to the transmission of extensive DNA damage to daughter cells.  In addn., however, we found that the effects of SLX4, MUS81, and GEN1 depletion extend beyond mitosis, since genome instability is obsd. throughout all phases of the cell cycle.  This is exemplified in the form of impaired replication fork movement and S-phase progression, endogenous checkpoint activation, chromosome segmentation, and multinucleation.  In contrast to SLX4, SLX1, the nuclease subunit of the SLX1-SLX4 structure-selective nuclease, plays no role in the replication-related phenotypes assocd. with SLX4/MUS81 and GEN1 depletion.  These observations demonstrate that the SLX1-SLX4 nuclease and the SLX4 scaffold play divergent roles in the maintenance of genome integrity in human cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphztHqtWw8nLVg90H21EOLACvtfcHk0liwh021oLuFrA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXpt12qs7w%253D&md5=2d1beecb6ad11ac1857778539977aca2</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1101%2Fgad.238303.114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.238303.114%26sid%3Dliteratum%253Aachs%26aulast%3DSarbajna%26aufirst%3DS.%26aulast%3DDavies%26aufirst%3DD.%26aulast%3DWest%26aufirst%3DS.%2BC.%26atitle%3DRoles%2520of%2520SLX1-SLX4%252C%2520MUS81-EME1%252C%2520and%2520GEN1%2520in%2520Avoiding%2520Genome%2520Instability%2520and%2520Mitotic%2520Catastrophe%26jtitle%3DGenes%2520Dev.%26date%3D2014%26volume%3D28%26spage%3D1124%26epage%3D1136%26doi%3D10.1101%2Fgad.238303.114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Krajewska, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fehrmann, R. S. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de
Vries, E. G. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Vugt, M. A. T. M.</span></span> <span> </span><span class="NLM_article-title">Regulators of Homologous Recombination Repair as Novel Targets for Cancer Treatment</span>. <i>Front. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">96</span>, <span class="refDoi"> DOI: 10.3389/fgene.2015.00096</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=10.3389%2Ffgene.2015.00096" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=25852742" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=1%3ACAS%3A280%3ADC%252BC2MjitFynug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=96&author=M.+Krajewskaauthor=R.+S.+N.+Fehrmannauthor=E.+G.+E.+de%0AVriesauthor=M.+A.+T.+M.+van+Vugt&title=Regulators+of+Homologous+Recombination+Repair+as+Novel+Targets+for+Cancer+Treatment&doi=10.3389%2Ffgene.2015.00096"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Regulators of homologous recombination repair as novel targets for cancer treatment</span></div><div class="casAuthors">Krajewska Malgorzata; Fehrmann Rudolf S N; de Vries Elisabeth G E; van Vugt Marcel A T M</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in genetics</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">96</span>
        ISSN:<span class="NLM_cas:issn">1664-8021</span>.
    </div><div class="casAbstract">To cope with DNA damage, cells possess a complex signaling network called the 'DNA damage response', which coordinates cell cycle control with DNA repair.  The importance of this network is underscored by the cancer predisposition that frequently goes along with hereditary mutations in DNA repair genes.  One especially important DNA repair pathway in this respect is homologous recombination (HR) repair.  Defects in HR repair are observed in various cancers, including hereditary breast, and ovarian cancer.  Intriguingly, tumor cells with defective HR repair show increased sensitivity to chemotherapeutic reagents, including platinum-containing agents.  These observations suggest that HR-proficient tumor cells might be sensitized to chemotherapeutics if HR repair could be therapeutically inactivated.  HR repair is an extensively regulated process, which depends strongly on the activity of various other pathways, including cell cycle pathways, protein-control pathways, and growth factor-activated receptor signaling pathways.  In this review, we discuss how the mechanistic wiring of HR is controlled by cell-intrinsic or extracellular pathways.  Furthermore, we have performed a meta-analysis on available genome-wide RNA interference studies to identify additional pathways that control HR repair.  Finally, we discuss how these HR-regulatory pathways may provide therapeutic targets in the context of radio/chemosensitization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSOCPY1LIxia0cLvDNbOD4DfW6udTcc2eaUMKxEnuIBKbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2MjitFynug%253D%253D&md5=c7e4d6066151172de4130c8df50aec97</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.3389%2Ffgene.2015.00096&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffgene.2015.00096%26sid%3Dliteratum%253Aachs%26aulast%3DKrajewska%26aufirst%3DM.%26aulast%3DFehrmann%26aufirst%3DR.%2BS.%2BN.%26aulast%3Dde%2BVries%26aufirst%3DE.%2BG.%2BE.%26aulast%3Dvan%2BVugt%26aufirst%3DM.%2BA.%2BT.%2BM.%26atitle%3DRegulators%2520of%2520Homologous%2520Recombination%2520Repair%2520as%2520Novel%2520Targets%2520for%2520Cancer%2520Treatment%26jtitle%3DFront.%2520Genet.%26date%3D2015%26volume%3D6%26spage%3D96%26doi%3D10.3389%2Ffgene.2015.00096" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Orbegoso, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marquina, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">George, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banerjee, S.</span></span> <span> </span><span class="NLM_article-title">The Role of Cediranib in Ovarian Cancer</span>. <i>Expert Opin. Pharmacother.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">1637</span>– <span class="NLM_lpage">1648</span>, <span class="refDoi"> DOI: 10.1080/14656566.2017.1383384</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=10.1080%2F14656566.2017.1383384" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=28933580" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1GqtLfM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=1637-1648&author=C.+Orbegosoauthor=G.+Marquinaauthor=A.+Georgeauthor=S.+Banerjee&title=The+Role+of+Cediranib+in+Ovarian+Cancer&doi=10.1080%2F14656566.2017.1383384"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">The role of Cediranib in ovarian cancer</span></div><div class="casAuthors">Orbegoso, Cecilia; Marquina, Gloria; George, Angela; Banerjee, Susana</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Pharmacotherapy</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">1637-1648</span>CODEN:
                <span class="NLM_cas:coden">EOPHF7</span>;
        ISSN:<span class="NLM_cas:issn">1465-6566</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">Treatment options for relapsed ovarian cancer have increased over the decade with the addn. of targeted agents, such as PARP inhibitors and antiangiogenic agents.  Bevacizumab, a monoclonal antibody binding vascular endothelial growth factor (VEGF), was the first anti-angiogenic agent to be incorporated in the ovarian cancer treatment landscape.  Other mols. utilizing different mechanisms of action to target angiogenesis have been developed, including cediranib, an oral potent inhibitor of VEGF Tyrosine Kinase Inhibitor that has demonstrated activity in both phase II and phase III studies.  Herein we will review cediranib as well as the evidence for its use in ovarian cancer, both as monotherapy and in combination with chemotherapy, PARP inhibitors and immunotherapy.  A literature search was made in PubMed and on ClinicalTrials.gov for clin. trials with cediranib.  The addn. of cediranib for the treatment of ovarian cancer is promising, and has demonstrated a significant improvement in progression free survival in a phase III trial in combination with chemotherapy and maintenance treatment.  Cediranib is currently being explored in ovarian cancer and other gynaecol. malignancies aiming to improve patient care; further research will help define its role in std. clin. practice for patients with ovarian cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGruIr3HI2T3y7Vg90H21EOLACvtfcHk0ljpbjqQ3lFVXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1GqtLfM&md5=0f304893af3b8bdaf3d4863b29353684</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1080%2F14656566.2017.1383384&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F14656566.2017.1383384%26sid%3Dliteratum%253Aachs%26aulast%3DOrbegoso%26aufirst%3DC.%26aulast%3DMarquina%26aufirst%3DG.%26aulast%3DGeorge%26aufirst%3DA.%26aulast%3DBanerjee%26aufirst%3DS.%26atitle%3DThe%2520Role%2520of%2520Cediranib%2520in%2520Ovarian%2520Cancer%26jtitle%3DExpert%2520Opin.%2520Pharmacother.%26date%3D2017%26volume%3D18%26spage%3D1637%26epage%3D1648%26doi%3D10.1080%2F14656566.2017.1383384" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kaplan, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gueble, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oeck, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yun, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glazer, P. M.</span></span> <span> </span><span class="NLM_article-title">Cediranib Suppresses Homology-Directed DNA Repair through down-Regulation of BRCA1/2 and RAD51</span>. <i>Sci. Transl. Med.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>11</i></span>, <span class="NLM_elocation-id">eaav4508</span> <span class="refDoi"> DOI: 10.1126/scitranslmed.aav4508</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=10.1126%2Fscitranslmed.aav4508" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=31092693" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2019&author=A.+R.+Kaplanauthor=S.+E.+Guebleauthor=Y.+Liuauthor=S.+Oeckauthor=H.+Kimauthor=Z.+Yunauthor=P.+M.+Glazer&title=Cediranib+Suppresses+Homology-Directed+DNA+Repair+through+down-Regulation+of+BRCA1%2F2+and+RAD51&doi=10.1126%2Fscitranslmed.aav4508"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1126%2Fscitranslmed.aav4508&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscitranslmed.aav4508%26sid%3Dliteratum%253Aachs%26aulast%3DKaplan%26aufirst%3DA.%2BR.%26aulast%3DGueble%26aufirst%3DS.%2BE.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DOeck%26aufirst%3DS.%26aulast%3DKim%26aufirst%3DH.%26aulast%3DYun%26aufirst%3DZ.%26aulast%3DGlazer%26aufirst%3DP.%2BM.%26atitle%3DCediranib%2520Suppresses%2520Homology-Directed%2520DNA%2520Repair%2520through%2520down-Regulation%2520of%2520BRCA1%252F2%2520and%2520RAD51%26jtitle%3DSci.%2520Transl.%2520Med.%26date%3D2019%26volume%3D11%26doi%3D10.1126%2Fscitranslmed.aav4508" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bagnolini, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milano, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manerba, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schipani, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ortega, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gioia, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Falchi, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balboni, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farabegoli, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Franco, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robertson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pellicciari, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pallavicini, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peri, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minucci, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Girotto, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Stefano, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberti, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cavalli, A.</span></span> <span> </span><span class="NLM_article-title">Synthetic Lethality in Pancreatic Cancer: Discovery of a New RAD51-BRCA2 Small Molecule Disruptor That Inhibits Homologous Recombination and Synergizes with Olaparib</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">2588</span>– <span class="NLM_lpage">2619</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b01526</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01526" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=1%3ACAS%3A528%3ADC%252BB3cXisFKrurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=2588-2619&author=G.+Bagnoliniauthor=D.+Milanoauthor=M.+Manerbaauthor=F.+Schipaniauthor=J.+A.+Ortegaauthor=D.+Gioiaauthor=F.+Falchiauthor=A.+Balboniauthor=F.+Farabegoliauthor=F.+De+Francoauthor=J.+Robertsonauthor=R.+Pellicciariauthor=I.+Pallaviciniauthor=S.+Periauthor=S.+Minucciauthor=S.+Girottoauthor=G.+Di+Stefanoauthor=M.+Robertiauthor=A.+Cavalli&title=Synthetic+Lethality+in+Pancreatic+Cancer%3A+Discovery+of+a+New+RAD51-BRCA2+Small+Molecule+Disruptor+That+Inhibits+Homologous+Recombination+and+Synergizes+with+Olaparib&doi=10.1021%2Facs.jmedchem.9b01526"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Synthetic Lethality in Pancreatic Cancer: Discovery of a New RAD51-BRCA2 Small Molecule Disruptor That Inhibits Homologous Recombination and Synergizes with Olaparib</span></div><div class="casAuthors">Bagnolini, Greta; Milano, Domenico; Manerba, Marcella; Schipani, Fabrizio; Ortega, Jose Antonio; Gioia, Dario; Falchi, Federico; Balboni, Andrea; Farabegoli, Fulvia; De Franco, Francesca; Robertson, Janet; Pellicciari, Roberto; Pallavicini, Isabella; Peri, Sebastiano; Minucci, Saverio; Girotto, Stefania; Di Stefano, Giuseppina; Roberti, Marinella; Cavalli, Andrea</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2588-2619</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Synthetic lethality is an innovative framework for discovering novel anticancer drug candidates.  One example is the use of PARP inhibitors (PARPi) in oncol. patients with BRCA mutations.  Here, we exploit a new paradigm based on the possibility of triggering synthetic lethality using only small org. mols. (dubbed "fully small-mol.-induced synthetic lethality").  We exploited this paradigm to target pancreatic cancer, one of the major unmet needs in oncol.  We discovered a dihydroquinolone pyrazoline-based mol. (35d) that disrupts the RAD51-BRCA2 protein-protein interaction, thus mimicking the effect of BRCA2 mutation. 35d inhibits the homologous recombination in a human pancreatic adenocarcinoma cell line.  In addn., it synergizes with olaparib (a PARPi) to trigger synthetic lethality.  This strategy aims to widen the use of PARPi in BRCA-competent and olaparib-resistant cancers, making fully small-mol.-induced synthetic lethality an innovative approach toward unmet oncol. needs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoiTSEdA8VygbVg90H21EOLACvtfcHk0ljpbjqQ3lFVXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXisFKrurg%253D&md5=1cae61064fc0e637f9b5f00c47e0bdeb</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01526%26sid%3Dliteratum%253Aachs%26aulast%3DBagnolini%26aufirst%3DG.%26aulast%3DMilano%26aufirst%3DD.%26aulast%3DManerba%26aufirst%3DM.%26aulast%3DSchipani%26aufirst%3DF.%26aulast%3DOrtega%26aufirst%3DJ.%2BA.%26aulast%3DGioia%26aufirst%3DD.%26aulast%3DFalchi%26aufirst%3DF.%26aulast%3DBalboni%26aufirst%3DA.%26aulast%3DFarabegoli%26aufirst%3DF.%26aulast%3DDe%2BFranco%26aufirst%3DF.%26aulast%3DRobertson%26aufirst%3DJ.%26aulast%3DPellicciari%26aufirst%3DR.%26aulast%3DPallavicini%26aufirst%3DI.%26aulast%3DPeri%26aufirst%3DS.%26aulast%3DMinucci%26aufirst%3DS.%26aulast%3DGirotto%26aufirst%3DS.%26aulast%3DDi%2BStefano%26aufirst%3DG.%26aulast%3DRoberti%26aufirst%3DM.%26aulast%3DCavalli%26aufirst%3DA.%26atitle%3DSynthetic%2520Lethality%2520in%2520Pancreatic%2520Cancer%253A%2520Discovery%2520of%2520a%2520New%2520RAD51-BRCA2%2520Small%2520Molecule%2520Disruptor%2520That%2520Inhibits%2520Homologous%2520Recombination%2520and%2520Synergizes%2520with%2520Olaparib%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D2588%26epage%3D2619%26doi%3D10.1021%2Facs.jmedchem.9b01526" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, G. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dexheimer, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenthal, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chu, W. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mosedale, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bachrati, C. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schultz, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakurai, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savitsky, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McHugh, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bohr, V. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jadhav, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gileadi, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maloney, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simeonov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hickson, I. D.</span></span> <span> </span><span class="NLM_article-title">A Small Molecule Inhibitor of the BLM Helicase Modulates Chromosome Stability in Human Cells</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">55</span>– <span class="NLM_lpage">62</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2012.10.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=10.1016%2Fj.chembiol.2012.10.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=23352139" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1equr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2013&pages=55-62&author=G.+H.+Nguyenauthor=T.+S.+Dexheimerauthor=A.+S.+Rosenthalauthor=W.+K.+Chuauthor=D.+K.+Singhauthor=G.+Mosedaleauthor=C.+Z.+Bachratiauthor=L.+Schultzauthor=M.+Sakuraiauthor=P.+Savitskyauthor=M.+Abuauthor=P.+J.+McHughauthor=V.+A.+Bohrauthor=C.+C.+Harrisauthor=A.+Jadhavauthor=O.+Gileadiauthor=D.+J.+Maloneyauthor=A.+Simeonovauthor=I.+D.+Hickson&title=A+Small+Molecule+Inhibitor+of+the+BLM+Helicase+Modulates+Chromosome+Stability+in+Human+Cells&doi=10.1016%2Fj.chembiol.2012.10.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">A Small Molecule Inhibitor of the BLM Helicase Modulates Chromosome Stability in Human Cells</span></div><div class="casAuthors">Nguyen, Giang Huong; Dexheimer, Thomas S.; Rosenthal, Andrew S.; Chu, Wai Kit; Singh, Dharmendra Kumar; Mosedale, Georgina; Bachrati, Csanad Z.; Schultz, Lena; Sakurai, Masaaki; Savitsky, Pavel; Abu, Mika; McHugh, Peter J.; Bohr, Vilhelm A.; Harris, Curtis C.; Jadhav, Ajit; Gileadi, Opher; Maloney, David J.; Simeonov, Anton; Hickson, Ian D.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">55-62</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The Bloom's syndrome protein, BLM, is a member of the conserved RecQ helicase family.  Although cell lines lacking BLM exist, these exhibit progressive genomic instability that makes distinguishing primary from secondary effects of BLM loss problematic.  In order to be able to acutely disable BLM function in cells, we undertook a high throughput screen of a chem. compd. library for small mol. inhibitors of BLM.  We present ML216, a potent inhibitor of the DNA unwinding activity of BLM.  ML216 shows cell-based activity and can induce sister chromatid exchanges, enhance the toxicity of aphidicolin, and exert antiproliferative activity in cells expressing BLM, but not those lacking BLM.  These data indicate that ML216 shows strong selectivity for BLM in cultured cells.  We discuss the potential utility of such a BLM-targeting compd. as an anticancer agent.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhJ5QlteU3zLVg90H21EOLACvtfcHk0lhlK-3NAWvrfw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1equr8%253D&md5=26bfa8e45f2253aa8807d055be0689f2</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2012.10.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2012.10.016%26sid%3Dliteratum%253Aachs%26aulast%3DNguyen%26aufirst%3DG.%2BH.%26aulast%3DDexheimer%26aufirst%3DT.%2BS.%26aulast%3DRosenthal%26aufirst%3DA.%2BS.%26aulast%3DChu%26aufirst%3DW.%2BK.%26aulast%3DSingh%26aufirst%3DD.%2BK.%26aulast%3DMosedale%26aufirst%3DG.%26aulast%3DBachrati%26aufirst%3DC.%2BZ.%26aulast%3DSchultz%26aufirst%3DL.%26aulast%3DSakurai%26aufirst%3DM.%26aulast%3DSavitsky%26aufirst%3DP.%26aulast%3DAbu%26aufirst%3DM.%26aulast%3DMcHugh%26aufirst%3DP.%2BJ.%26aulast%3DBohr%26aufirst%3DV.%2BA.%26aulast%3DHarris%26aufirst%3DC.%2BC.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DGileadi%26aufirst%3DO.%26aulast%3DMaloney%26aufirst%3DD.%2BJ.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DHickson%26aufirst%3DI.%2BD.%26atitle%3DA%2520Small%2520Molecule%2520Inhibitor%2520of%2520the%2520BLM%2520Helicase%2520Modulates%2520Chromosome%2520Stability%2520in%2520Human%2520Cells%26jtitle%3DChem.%2520Biol.%26date%3D2013%26volume%3D20%26spage%3D55%26epage%3D62%26doi%3D10.1016%2Fj.chembiol.2012.10.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yin, Q.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.-X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.-Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, Q.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ou, T.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, S.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.-G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, J.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S.-B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Z.-S.</span></span> <span> </span><span class="NLM_article-title">Discovery of Isaindigotone Derivatives as Novel Bloom’s Syndrome Protein (BLM) Helicase Inhibitors That Disrupt the BLM/DNA Interactions and Regulate the Homologous Recombination Repair</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">3147</span>– <span class="NLM_lpage">3162</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b00083</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b00083" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=1%3ACAS%3A528%3ADC%252BC1MXktV2msbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=3147-3162&author=Q.-K.+Yinauthor=C.-X.+Wangauthor=Y.-Q.+Wangauthor=Q.-L.+Guoauthor=Z.-L.+Zhangauthor=T.-M.+Ouauthor=S.-L.+Huangauthor=D.+Liauthor=H.-G.+Wangauthor=J.-H.+Tanauthor=S.-B.+Chenauthor=Z.-S.+Huang&title=Discovery+of+Isaindigotone+Derivatives+as+Novel+Bloom%E2%80%99s+Syndrome+Protein+%28BLM%29+Helicase+Inhibitors+That+Disrupt+the+BLM%2FDNA+Interactions+and+Regulate+the+Homologous+Recombination+Repair&doi=10.1021%2Facs.jmedchem.9b00083"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Isaindigotone Derivatives as Novel Bloom's Syndrome Protein (BLM) Helicase Inhibitors That Disrupt the BLM/DNA Interactions and Regulate the Homologous Recombination Repair</span></div><div class="casAuthors">Yin, Qi-Kun; Wang, Chen-Xi; Wang, Yu-Qing; Guo, Qian-Liang; Zhang, Zi-Lin; Ou, Tian-Miao; Huang, Shi-Liang; Li, Ding; Wang, Hong-Gen; Tan, Jia-Heng; Chen, Shuo-Bin; Huang, Zhi-Shu</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">3147-3162</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Homologous recombination repair (HRR), a crucial approach in DNA damage repair, is an attractive target in cancer therapy and drug design.  The Bloom syndrome protein (BLM) is a 3'-5' DNA helicase that performs an important role in HRR regulation.  However, limited studies about BLM inhibitors and their biol. effects have been reported.  Here, we identified a class of isaindigotone derivs. as novel BLM inhibitors by synthesis, screening, and evaluating.  Among them, compd. 29 was found as an effective BLM inhibitor with a high binding affinity and good inhibitory effect on BLM.  Cellular evaluation indicated that 29 effectively disrupted the recruitment of BLM at DNA double-strand break sites, promoted an accumulation of RAD51, and regulated the HRR process.  Meanwhile, 29 significantly induced DNA damage responses, as well as apoptosis and proliferation arrest in cancer cells.  Our finding provides a potential anticancer strategy based on interfering with BLM via small mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_jlhubiNMYLVg90H21EOLACvtfcHk0lhlK-3NAWvrfw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXktV2msbg%253D&md5=63b56e7be57afe71d3b279e2f5fb2663</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b00083%26sid%3Dliteratum%253Aachs%26aulast%3DYin%26aufirst%3DQ.-K.%26aulast%3DWang%26aufirst%3DC.-X.%26aulast%3DWang%26aufirst%3DY.-Q.%26aulast%3DGuo%26aufirst%3DQ.-L.%26aulast%3DZhang%26aufirst%3DZ.-L.%26aulast%3DOu%26aufirst%3DT.-M.%26aulast%3DHuang%26aufirst%3DS.-L.%26aulast%3DLi%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DH.-G.%26aulast%3DTan%26aufirst%3DJ.-H.%26aulast%3DChen%26aufirst%3DS.-B.%26aulast%3DHuang%26aufirst%3DZ.-S.%26atitle%3DDiscovery%2520of%2520Isaindigotone%2520Derivatives%2520as%2520Novel%2520Bloom%25E2%2580%2599s%2520Syndrome%2520Protein%2520%2528BLM%2529%2520Helicase%2520Inhibitors%2520That%2520Disrupt%2520the%2520BLM%252FDNA%2520Interactions%2520and%2520Regulate%2520the%2520Homologous%2520Recombination%2520Repair%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D3147%26epage%3D3162%26doi%3D10.1021%2Facs.jmedchem.9b00083" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Islam, M. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fox, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Enomoto, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span> <span> </span><span class="NLM_article-title">RecQL5 Promotes Genome Stabilization through Two Parallel Mechanisms-Interacting with RNA Polymerase II and Acting as a Helicase</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">2460</span>– <span class="NLM_lpage">2472</span>, <span class="refDoi"> DOI: 10.1128/MCB.01583-09</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=10.1128%2FMCB.01583-09" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=20231364" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=1%3ACAS%3A528%3ADC%252BC3cXmtFymsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2010&pages=2460-2472&author=M.+N.+Islamauthor=D.+Foxauthor=R.+Guoauthor=T.+Enomotoauthor=W.+Wang&title=RecQL5+Promotes+Genome+Stabilization+through+Two+Parallel+Mechanisms-Interacting+with+RNA+Polymerase+II+and+Acting+as+a+Helicase&doi=10.1128%2FMCB.01583-09"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">RecQL5 promotes genome stabilization through two parallel mechanisms-interacting with RNA polymerase II and acting as a helicase</span></div><div class="casAuthors">Islam, M. Nurul; Fox, David, III; Guo, Rong; Enomoto, Takemi; Wang, Weidong</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2460-2472</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The RecQL5 helicase is essential for maintaining genome stability and reducing cancer risk.  To elucidate its mechanism of action, we purified a RecQL5-assocd. complex and identified its major component as RNA polymerase II (Pol II).  Bioinformatics and structural modeling-guided mutagenesis revealed two conserved regions in RecQL5 as KIX and SRI domains, already known in transcriptional regulators for Pol II.  The RecQL5-KIX domain binds both initiation (Pol IIa) and elongation (Pol IIo) forms of the polymerase, whereas the RecQL5-SRI domain interacts only with the elongation form.  Fully functional RecQL5 requires both helicase activity and assocns. with the initiation polymerase, because mutants lacking either activity are partially defective in the suppression of sister chromatid exchange and resistance to camptothecin-induced DNA damage, and mutants lacking both activities are completely defective.  We propose that RecQL5 promotes genome stabilization through two parallel mechanisms: by participation in homologous recombination-dependent DNA repair as a RecQ helicase and by regulating the initiation of Pol II to reduce transcription-assocd. replication impairment and recombination.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrslpjg_dVw0bVg90H21EOLACvtfcHk0lhlK-3NAWvrfw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXmtFymsb4%253D&md5=0098f5ba745b0ee230b0ee3dd1901b62</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1128%2FMCB.01583-09&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.01583-09%26sid%3Dliteratum%253Aachs%26aulast%3DIslam%26aufirst%3DM.%2BN.%26aulast%3DFox%26aufirst%3DD.%26aulast%3DGuo%26aufirst%3DR.%26aulast%3DEnomoto%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DW.%26atitle%3DRecQL5%2520Promotes%2520Genome%2520Stabilization%2520through%2520Two%2520Parallel%2520Mechanisms-Interacting%2520with%2520RNA%2520Polymerase%2520II%2520and%2520Acting%2520as%2520a%2520Helicase%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2010%26volume%3D30%26spage%3D2460%26epage%3D2472%26doi%3D10.1128%2FMCB.01583-09" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Saha, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patro, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koli, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pai, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ray, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bandyopadhyay, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chattopadhyay, S.</span></span> <span> </span><span class="NLM_article-title">trans-4,4’-Dihydroxystilbene (DHS) inhibits human neuroblastoma tumor growth and induces mitochondrial and lysosomal damages in neuroblastoma cell lines</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">73905</span>– <span class="NLM_lpage">73924</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.17879</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=10.18632%2Foncotarget.17879" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=29088756" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=1%3ACAS%3A280%3ADC%252BC1M7mtlaksw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=73905-73924&author=B.+Sahaauthor=B.+S.+Patroauthor=M.+Koliauthor=G.+Paiauthor=J.+Rayauthor=S.+K.+Bandyopadhyayauthor=S.+Chattopadhyay&title=trans-4%2C4%E2%80%99-Dihydroxystilbene+%28DHS%29+inhibits+human+neuroblastoma+tumor+growth+and+induces+mitochondrial+and+lysosomal+damages+in+neuroblastoma+cell+lines&doi=10.18632%2Foncotarget.17879"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">trans-4,4'-Dihydroxystilbene (DHS) inhibits human neuroblastoma tumor growth and induces mitochondrial and lysosomal damages in neuroblastoma cell lines</span></div><div class="casAuthors">Saha Bhaskar; Bandyopadhyay Sandip K; Saha Bhaskar; Ray Jharna; Patro Birija Sankar; Koli Mrunesh; Pai Ganesh; Chattopadhyay Subrata; Patro Birija Sankar; Chattopadhyay Subrata</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">43</span>),
    <span class="NLM_cas:pages">73905-73924</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">In view of the inadequacy of neuroblastoma treatment, five hydroxystilbenes and resveratrol (Resv) were screened for their cytotoxic property against human neuroblastoma cell lines.  The mechanism of cytotoxic action of the most potent compound, trans-4,4'-dihydroxystilbene (DHS) was investigated in vitro using human neuroblastoma cell lines.  DHS was also tested in a mouse xenograft model of human neuroblastoma tumor.  The MTT, sub-G1, annexin V and clonogenic assays as well as microscopy established higher cytotoxicity of DHS than Resv to the IMR32 cell line.  DHS (20 μM) induced mitochondrial membrane permeabilization (MMP) in the cells, as revealed from JC-1 staining, cytochrome c and ApaF1 release and caspases-9/3 activation.  DHS also induced lysosomal membrane permeabilization (LMP) to release cathepsins B, L and D, and the cathepsins inhibitors partially reduced MMP/caspase-3 activation.  The ROS, produced by DHS activated the p38 and JNK MAPKs to augment the BAX activity and BID-cleavage, and induce LMP and MMP in the cells.  DHS (100 mg/kg) also inhibited human neuroblastoma tumor growth in SCID mice by 51%.  Hence, DHS may be a potential chemotherapeutic option against neuroblastoma.  The involvement of an independent LMP as well as a partially LMP-dependent MMP by DHS is attractive as it provides options to target both mitochondria and lysosome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQRKdIxzbJTPMrRqEXsZufAfW6udTcc2eYpwGbnMMBqj7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M7mtlaksw%253D%253D&md5=b45c3118eb6a1568040792ac15e48ad9</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.17879&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.17879%26sid%3Dliteratum%253Aachs%26aulast%3DSaha%26aufirst%3DB.%26aulast%3DPatro%26aufirst%3DB.%2BS.%26aulast%3DKoli%26aufirst%3DM.%26aulast%3DPai%26aufirst%3DG.%26aulast%3DRay%26aufirst%3DJ.%26aulast%3DBandyopadhyay%26aufirst%3DS.%2BK.%26aulast%3DChattopadhyay%26aufirst%3DS.%26atitle%3Dtrans-4%252C4%25E2%2580%2599-Dihydroxystilbene%2520%2528DHS%2529%2520inhibits%2520human%2520neuroblastoma%2520tumor%2520growth%2520and%2520induces%2520mitochondrial%2520and%2520lysosomal%2520damages%2520in%2520neuroblastoma%2520cell%2520lines%26jtitle%3DOncotarget%26date%3D2017%26volume%3D8%26spage%3D73905%26epage%3D73924%26doi%3D10.18632%2Foncotarget.17879" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Patro, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frohlich, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bohr, V. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevnsner, T.</span></span> <span> </span><span class="NLM_article-title">WRN Helicase Regulates the ATR-CHK1-Induced S-Phase Checkpoint Pathway in Response to Topoisomerase-I-DNA Covalent Complexes</span>. <i>J. Cell Sci.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>124</i></span>,  <span class="NLM_fpage">3967</span>– <span class="NLM_lpage">3979</span>, <span class="refDoi"> DOI: 10.1242/jcs.081372</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=10.1242%2Fjcs.081372" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=22159421" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsV2lu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=124&publication_year=2011&pages=3967-3979&author=B.+S.+Patroauthor=R.+Frohlichauthor=V.+A.+Bohrauthor=T.+Stevnsner&title=WRN+Helicase+Regulates+the+ATR-CHK1-Induced+S-Phase+Checkpoint+Pathway+in+Response+to+Topoisomerase-I-DNA+Covalent+Complexes&doi=10.1242%2Fjcs.081372"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">WRN helicase regulates the ATR-CHK1-induced S-phase checkpoint pathway in response to topoisomerase-I-DNA covalent complexes</span></div><div class="casAuthors">Patro, Birija Sankar; Froehlich, Rikke; Bohr, Vilhelm A.; Stevnsner, Tinna</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cell Science</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">124</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">3967-3979</span>CODEN:
                <span class="NLM_cas:coden">JNCSAI</span>;
        ISSN:<span class="NLM_cas:issn">0021-9533</span>.
    
            (<span class="NLM_cas:orgname">Company of Biologists Ltd.</span>)
        </div><div class="casAbstract">Checkpoints are cellular surveillance and signaling pathways that coordinate the response to DNA damage and replicative stress.  Consequently, failure of cellular checkpoints increases susceptibility to DNA damage and can lead to profound genome instability.  This study examines the role of a human RECQ helicase, WRN, in checkpoint activation in response to DNA damage.  Mutations in WRN lead to genomic instability and the premature aging condition Werner syndrome.  Here, the role of WRN in a DNA-damage-induced checkpoint was analyzed in U-2 OS (WRN wild type) and isogenic cells stably expressing WRN-targeted shRNA (WRN knockdown).  The results of our studies suggest that WRN has a crucial role in inducing an S-phase checkpoint in cells exposed to the topoisomerase I inhibitor campthothecin (CPT), but not in cells exposed to hydroxyurea.  Intriguingly, WRN decreases the rate of replication fork elongation, increases the accumulation of ssDNA and stimulates phosphorylation of CHK1, which releases CHK1 from chromatin in CPT-treated cells.  Importantly, knockdown of WRN expression abolished or delayed all these processes in response to CPT.  Together, our results strongly suggest an essential regulatory role for WRN in controlling the ATR-CHK1-mediated S-phase checkpoint in CPT-treated cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrFgrwlOxEQ7bVg90H21EOLACvtfcHk0liDqfLYdeSIDw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsV2lu7s%253D&md5=079e1b6a1b371997113498274ab81f39</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1242%2Fjcs.081372&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1242%252Fjcs.081372%26sid%3Dliteratum%253Aachs%26aulast%3DPatro%26aufirst%3DB.%2BS.%26aulast%3DFrohlich%26aufirst%3DR.%26aulast%3DBohr%26aufirst%3DV.%2BA.%26aulast%3DStevnsner%26aufirst%3DT.%26atitle%3DWRN%2520Helicase%2520Regulates%2520the%2520ATR-CHK1-Induced%2520S-Phase%2520Checkpoint%2520Pathway%2520in%2520Response%2520to%2520Topoisomerase-I-DNA%2520Covalent%2520Complexes%26jtitle%3DJ.%2520Cell%2520Sci.%26date%3D2011%26volume%3D124%26spage%3D3967%26epage%3D3979%26doi%3D10.1242%2Fjcs.081372" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Han, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span> <span> </span><span class="NLM_article-title">Immunomodulatory Activity of a Water-Soluble Polysaccharide Obtained from Highland Barley on Immunosuppressive Mice Models</span>. <i>Food Funct.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">304</span>– <span class="NLM_lpage">314</span>, <span class="refDoi"> DOI: 10.1039/C8FO01991F</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=10.1039%2FC8FO01991F" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=30574978" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisFSmsL7L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=304-314&author=L.+Hanauthor=M.+Mengauthor=M.+Guoauthor=D.+Chengauthor=L.+Shiauthor=X.+Wangauthor=C.+Wang&title=Immunomodulatory+Activity+of+a+Water-Soluble+Polysaccharide+Obtained+from+Highland+Barley+on+Immunosuppressive+Mice+Models&doi=10.1039%2FC8FO01991F"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Immunomodulatory activity of a water-soluble polysaccharide obtained from highland barley on immunosuppression mice models</span></div><div class="casAuthors">Han, Lirong; Meng, Meng; Guo, Mingzhu; Cheng, Dai; Shi, Lei; Wang, Xu; Wang, Chunling</div><div class="citationInfo"><span class="NLM_cas:title">Food & Function</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">304-314</span>CODEN:
                <span class="NLM_cas:coden">FFOUAI</span>;
        ISSN:<span class="NLM_cas:issn">2042-6496</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A water-sol. polysaccharide (BP-1) was extd. and purified from highland barley (Hordeum vulgare L.) and its av. mol. wt. was about 6.7x104 Da.  In this study, the immunomodu-latory activity of BP-1 on the immunosuppressive BALB/c mice model and its mol. mechanism were elucidated.  It was found that the wt. indexes of spleens and thymus were significantly in-creased by BP-1 (80 mg· kg-1 and 160 mg· kg-1) treatment in immunosuppressive mice model.  The results showed that BP-1 (80 mg· kg-1 and 160 mg· kg-1) could significantly increase the no. of bone marrow cells (BMC) and peripheral blood white blood cells (WBC) in immunosuppressive mice model.  In addn., the result further confirmed that BP-1 could increase the serum levels of IL-2, TNF-α and IFN-γ, so as to improve the immune function of immunosuppressive mice.  The results showed that BP-1 (80 mg· kg-1 and 160 mg· kg-1) could promote the proliferation of spleen cells, the natural killer (NK) cells activity in vivo.  The quant. real-time polymerase chain reaction (qRT-PCR) and ELISA results revealed that BP-1 (80 mg· kg-1 and 160 mg· kg-1) could enhance the prodn. of IL-2, TNF-α, IFN-γ, IgG and IgM in the spleen of immunosuppressive mice.  The HE (hematoxylin and eosin) stained histopathol. images showed that BP-1 (80 mg· kg-1 and 160 mg· kg-1) could repair the damage inducing by CTX in the spleen cells of immunosuppressive mice.  The result of macrophages showed that BP-1 (80 mg· kg-1 and 160 mg· kg-1) could promote the pro-liferation and phagocytosis activity of macrophages in immunosuppressive mice.  Furthermore, BP-1 could activate macrophages by the TLR-4- TRAF6- TAK1- nuclear factor κB (NF-κB) p65 pathway in vivo.  These results suggested that BP-1 has a remarkable immunomodulatory activity on the immu-nosuppressive mice model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoS0HrEh7f80rVg90H21EOLACvtfcHk0liDqfLYdeSIDw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisFSmsL7L&md5=684d7c09b2ebb6daad1f924c8554aab2</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1039%2FC8FO01991F&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC8FO01991F%26sid%3Dliteratum%253Aachs%26aulast%3DHan%26aufirst%3DL.%26aulast%3DMeng%26aufirst%3DM.%26aulast%3DGuo%26aufirst%3DM.%26aulast%3DCheng%26aufirst%3DD.%26aulast%3DShi%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DC.%26atitle%3DImmunomodulatory%2520Activity%2520of%2520a%2520Water-Soluble%2520Polysaccharide%2520Obtained%2520from%2520Highland%2520Barley%2520on%2520Immunosuppressive%2520Mice%2520Models%26jtitle%3DFood%2520Funct.%26date%3D2019%26volume%3D10%26spage%3D304%26epage%3D314%26doi%3D10.1039%2FC8FO01991F" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article has not yet been cited by other publications.</p></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01692/20210204/images/medium/jm0c01692_0029.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01692/20210204/images/large/jm0c01692_0029.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01692&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01692/20210204/images/medium/jm0c01692_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01692/20210204/images/large/jm0c01692_0011.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Schematic Routes for the Synthesis of 1,3,4-Oxadiazole with Thiourea Derivatives</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01692/20210204/images/large/jm0c01692_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01692&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01692/20210204/images/medium/jm0c01692_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01692/20210204/images/large/jm0c01692_0012.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Schematic Routes for the Synthesis of 1,3,4-Oxadiazole with Urea Derivatives</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01692/20210204/images/large/jm0c01692_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01692&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01692/20210204/images/medium/jm0c01692_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01692/20210204/images/large/jm0c01692_0002.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Screening and evaluation of selective cytotoxicity of 1,3,4-Oxadiazole derivatives to RECQL5-expressing breast cancer cells. (A) Kaplan–Meier analyses of overall survival of breast cancer patients with high and low RECQL5 protein expressions. (B) CRISPR-CAS9 double nickase-mediated knockout of RECQL5 in MCF-7 cells. Control double nickase plasmid and two different RECQL5 double nickase plasmids were transfected to MCF-7 cells to generate RECQL5-WT, RECQL5-KO1, and RECQL5-KO2 cells. (C) Cell viability of RECQL5-WT and RECQL5-KO2 cells after 72 h treatment with compounds <b>4a–i</b>. Highlighted green box shows the highest preferential killing ability of <b>4a</b> for RECQL5-WT cells. Data are shown as mean ± SD (<i>n</i> = 4). *<i>p</i> < 0.05 and **<i>p</i> < 0.01. (D) Dot-plot of IC<sub>20</sub> values of compounds <b>4a–i</b> for RECQL5-WT and RECQL5-KO2 cells. (E) Cell viability of normal mammary epithelial cells (MCF10A) <i>vs</i> malignant breast cancer cells (MCF-7) in response to <b>4a</b> treatment (72 h). Data are shown as mean ± SD (<i>n</i> = 4). *<i>p</i> < 0.05 w.r.t the viability of the respective vehicle-treated cells (100%). (F) RECQL5 expression in MCF10A and MCF-7 cells. (G,H) Clonogenic potential of RECQL5-WT and RECQL5-KO2 cells after 8–10 days treatment with compound <b>4a</b>. Data are shown as mean ± SD (<i>n</i> = 5). *<i>p</i> < 0.01 in comparison to respective <b>4a</b> concentration in RECQL5-KO2 cells.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01692/20210204/images/large/jm0c01692_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01692&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01692/20210204/images/medium/jm0c01692_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01692/20210204/images/large/jm0c01692_0003.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Compound <b>4a</b>-mediated selective killing is dependent on the RECQL5 expression in the cancer cells. (A) Expression of RFP (control) and RECQL5-RFP in RECQL5-KO cells. Bar: 10 μm. (B) Cell viability of RECQL5-WT, RECQL5-KO, RECQL5-KO + RFP, and RECQL5-KO + RECQL5-RFP cells after <b>4a</b> treatment for 72 h. Data are shown as mean ± SD (<i>n</i> = 5). *<i>p</i> < 0.01. (C) Expression of RECQL5, RECQL1, WRN, and BLM in MCF-7 cells, after knocking down (KD) by using control shRNA and two different sets of shRNA for RECQL5, RECQL1, WRN, and BLM, respectively. (D) Clonogenic potential of control MCF-7 cells, RECQL5-KD2 (RECQL5 shRNA2), RECQL1-KD2 (RECQL1 shRNA2), WRN-KD1 (WRN shRNA1), and BLM-KD1 (BLM shRNA1) MCF-7 cells after 8–10 days treatment with compound <b>4a</b>. Data are shown as mean ± SD (<i>n</i> = 5). *<i>p</i> < 0.05, **<i>p</i> < 0.01, ***<i>p</i> < 0.001.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01692/20210204/images/large/jm0c01692_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01692&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01692/20210204/images/medium/jm0c01692_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01692/20210204/images/large/jm0c01692_0004.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01692/20210204/images/medium/jm0c01692_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01692/20210204/images/large/jm0c01692_0005.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Effect of <b>4a</b> on the ATPase, helicase, and DNA binding abilities of RECQL5 protein. (A) Schematic representation of RECQL5 protein with different domains. (B) N-terminal HIS-tagged human RECQL5 was expressed in <i>E. coli</i> and purified. The Gel shows two different batches of preparations. (C) Schematic representation of the fluorescent-based RECQL5 helicase assay using a duplex forked DNA. One of the strands was labeled with 3′-BHQ1, which quenches fluorescence of 5′-FAM labeled on the other strand. Helicase activity driven strand-unwinding results in the separation of 5′-FAM labeled strand and enhancement of the fluorescence. (D) RECQL5 helicase activity measured by the above fluorescent-based assay in the absence and presence of <b>4a</b> or <b>4e</b>. Data are shown as mean ± SD (<i>n</i> = 5). **<i>p</i> < 0.01 and ***<i>p</i> < 0.001. (E) Predicted binding mode of <b>4a</b> with ATPase pocket of RECQL5 (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LB3">5LB3</a>). The diagram shows compound <b>4a</b> in purple and ADP in yellow. (F) Possible mode of interaction of <b>4a</b> with ADP binding pocket of RECQL5. (G) RECQL5-KO cells untreated or transfected with RFP and RECQL5 (WT)-RFP. RECQL5 (K58M)-RFP and expression of these proteins was assessed. (H,I) Clonogenic potential of the indicated cells after <b>4a</b> treatment (8–10 days). Data are shown as mean ± SD (<i>n</i> = 5). *<i>p</i> < 0.01 w.r.t RECQL5-KO cells at the respective concentration of <b>4a</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01692/20210204/images/large/jm0c01692_0004.jpeg jm0c01692_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01692&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01692/20210204/images/medium/jm0c01692_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01692/20210204/images/large/jm0c01692_0006.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Compound <b>4a</b> caused higher accumulation of DSBs, which led to preferential death of RECQL5-expressing cells. (A,B) Apoptosis of RECQL5-WT and RECQL5-KO in response to different doses of <b>4a</b> for 72 h. Data are shown as mean ± SD (<i>n</i> = 5). **<i>p</i> < 0.01 and ***<i>p</i> < 0.001. (C,D) DSBs in the cells, treated with <b>4a</b> for 24 h, was measured by analyzing γH2AX foci by immunofluorescence microscopy. Data are shown as mean ± SEM (<i>n</i> = 5). ***<i>p</i> < 0.001. (E,F) Cells were treated with <b>4a</b> for indicated time points. DDR signaling was analyzed by western blotting. Data are shown as mean of at least three experiments.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01692/20210204/images/large/jm0c01692_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01692&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01692/20210204/images/medium/jm0c01692_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01692/20210204/images/large/jm0c01692_0007.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01692/20210204/images/medium/jm0c01692_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01692/20210204/images/large/jm0c01692_0008.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Compound <b>4a</b>-stabilized RECQL5-RAD51 physical interaction and inhibited HRR. (A,B) RECQL5-WT and RECQL5-KO cells were treated with <b>4a</b> for indicated time points and chromatin bound proteins was analyzed by western blotting. (C,D) Cells were treated with <b>4a</b> for indicated time periods and RAD51 foci were analyzed by immunofluorescence microscopy. Highlighted box in the images were zoomed. Data are shown as mean ± SD (<i>n</i> = 6). ***<i>p</i> < 0.001. (E) Scheme of I-SceI-induced DSBs at the SceGFP site and its HR-based repair to functional GFP. (F) Cells were treated with <b>4a</b> for 48 h and relative HRR was analyzed by measuring GFP positive cells by flow cytometry. Data are shown as mean ± SD (<i>n</i> = 6). ***<i>p</i> < 0.001. (G) Level of RAD51 and RECQL5 in the nuclear and chromatin extract of the cells, treated with <b>4a</b>, was measured by western blotting. Data are shown as mean (<i>n</i> = 3–4). (H) RECQL5-KO (MCF-7) cells transfected with EV or RECQL5-FLAG. These cells were untreated or treated with <b>4a</b>, immuno-precipitated with anti-FLAG tagged beads and probed for FLAG and RAD51. (I) RECQL5-KO (MCF-7) cells transfected with EV, RECQL5 (WT)-FLAG, and RECQL5 (F666A)-FLAG and expression of RECQL5-FLAG was assessed. (J,K) Clonogenic potential of indicated cells after <b>4a</b> treatment (8–10 days). Data are shown as mean ± SD (<i>n</i> = 5). *<i>p</i> < 0.05 w.r.t RECQL5-KO cells at the respective concentration of <b>4a</b>. **<i>p</i> < 0.01.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01692/20210204/images/large/jm0c01692_0007.jpeg jm0c01692_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01692&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01692/20210204/images/medium/jm0c01692_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01692/20210204/images/large/jm0c01692_0009.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Compound <b>4a</b>-abrogated <i>de novo</i> and acquired cisplatin resistance in breast cancer cells. (A) MCF-7 cells were treated with IR (6 Gy, 24 h recovery) or with indicated therapeutics for 24 h. The level of RECQL5 in whole cell extract was analyzed by western blotting. (B) MCF-7 cells were treated with IR (6 Gy, 24 h recovery) or aphidicolin, cisplatin, and CPT for 24 h. The level of chromatin-bound RECQL5 was analyzed by western blotting. (C) Cisplatin-resistant (MCF-7<sup>CDDP</sup>) cells were generated after repeated and prolonged cisplatin treatment to MCF-7 cells. Expression of RECQL5 in MCF-7 and MCF-7<sup>CDDP</sup> cells were analyzed by western blotting. (D) MCF-7 and MCF-7<sup>CDDP</sup> cells were treated with cisplatin for 8 h, cisplatin was removed and cells were treated with <b>4a</b>. Clonogenic potential of these cells was assessed after 8–10 days. Data are shown as mean ± SD (<i>n</i> = 4). *<i>p</i> < 0.05 and **<i>p</i> < 0.01. (E) MCF-7 and MCF-7<sup>CDDP</sup> cells were treated with cisplatin (20 μM) for 8 h, cisplatin was removed and cells were treated with <b>4a</b> (20 μM) for 24 h. RAD51 foci was analyzed by immunofluorescence microscopy. Data are shown as mean ± SD (<i>n</i> = 6). **<i>p</i> < 0.01 and ***<i>p</i> < 0.001. (F) Cells were treated as mentioned in E and mitotic catastrophe was analyzed by immunofluorescence microscopy. Data are shown as mean ± SD (<i>n</i> = 6). *<i>p</i> < 0.05 w.r.t respective untreated nuclei.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01692/20210204/images/large/jm0c01692_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01692&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01692/20210204/images/medium/jm0c01692_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01692/20210204/images/large/jm0c01692_0010.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Efficacy of compound <b>4a</b> against the breast tumor in the preclinical mice model. (A) Schematic representation of the breast tumor (xenograft) development in NUDE mice and treatment protocol. (B–E) NUDE mice-bearing RECQL5-WT and RECQL5-KO tumors were treated with the vehicle or <b>4a</b> for 30 days. Tumor volume (TV) was assessed on every alternate day and plotted. Representative images of mouse bearing tumors and excised tumors are shown. Data are shown as mean ± SD (<i>n</i> = 8). *<i>p</i> < 0.05 w.r.t respective the vehicle-treated mice. **<i>p</i> < 0.01. (F) Mice were given oral gavage of <b>4a</b> (50 mg/kg body wt.). Blood was collected immediately after sacrificing mice, at different time points. <b>4a</b> was extracted from the serum and subjected to HPLC analysis. The serum concentration of <b>4a</b> is shown. Data is shown as mean ± SD (<i>n</i> = 4). *<i>p</i> < 0.05, ***<i>p</i> < 0.01 w.r.t 0 h.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01692/20210204/images/large/jm0c01692_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01692&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch3" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01692/20210204/images/medium/jm0c01692_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01692/20210204/images/large/jm0c01692_0013.jpeg" id="rightTab-gr12" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Schematic Representation of Mode of Action of<b>4a</b> in Preferential Killing of RECQL5-Expressing Cells<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01692/20210204/images/large/jm0c01692_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01692&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>CO: cross-over; SDSA: synthesis-dependent strand annealing.</p></p></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i66">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_65728" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_65728" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 42 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bray, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferlay, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soerjomataram, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siegel, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Torre, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jemal, A.</span></span> <span> </span><span class="NLM_article-title">Global Cancer Statistics 2018: GLOBOCAN Estimates of Incidence and Mortality Worldwide for 36 Cancers in 185 Countries</span>. <i>Ca-Cancer J. Clin.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>68</i></span>,  <span class="NLM_fpage">394</span>– <span class="NLM_lpage">424</span>, <span class="refDoi"> DOI: 10.3322/caac.21492</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=10.3322%2Fcaac.21492" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=30207593" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=1%3ACAS%3A280%3ADC%252BB3c3otVGlsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2018&pages=394-424&author=F.+Brayauthor=J.+Ferlayauthor=I.+Soerjomataramauthor=R.+L.+Siegelauthor=L.+A.+Torreauthor=A.+Jemal&title=Global+Cancer+Statistics+2018%3A+GLOBOCAN+Estimates+of+Incidence+and+Mortality+Worldwide+for+36+Cancers+in+185+Countries&doi=10.3322%2Fcaac.21492"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Global cancer statistics 2018: GLOBOCAN estimates of incidence and mortality worldwide for 36 cancers in 185 countries</span></div><div class="casAuthors">Bray Freddie; Ferlay Jacques; Soerjomataram Isabelle; Siegel Rebecca L; Torre Lindsey A; Jemal Ahmedin</div><div class="citationInfo"><span class="NLM_cas:title">CA: a cancer journal for clinicians</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">68</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">394-424</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">This article provides a status report on the global burden of cancer worldwide using the GLOBOCAN 2018 estimates of cancer incidence and mortality produced by the International Agency for Research on Cancer, with a focus on geographic variability across 20 world regions.  There will be an estimated 18.1 million new cancer cases (17.0 million excluding nonmelanoma skin cancer) and 9.6 million cancer deaths (9.5 million excluding nonmelanoma skin cancer) in 2018.  In both sexes combined, lung cancer is the most commonly diagnosed cancer (11.6% of the total cases) and the leading cause of cancer death (18.4% of the total cancer deaths), closely followed by female breast cancer (11.6%), prostate cancer (7.1%), and colorectal cancer (6.1%) for incidence and colorectal cancer (9.2%), stomach cancer (8.2%), and liver cancer (8.2%) for mortality.  Lung cancer is the most frequent cancer and the leading cause of cancer death among males, followed by prostate and colorectal cancer (for incidence) and liver and stomach cancer (for mortality).  Among females, breast cancer is the most commonly diagnosed cancer and the leading cause of cancer death, followed by colorectal and lung cancer (for incidence), and vice versa (for mortality); cervical cancer ranks fourth for both incidence and mortality.  The most frequently diagnosed cancer and the leading cause of cancer death, however, substantially vary across countries and within each country depending on the degree of economic development and associated social and life style factors.  It is noteworthy that high-quality cancer registry data, the basis for planning and implementing evidence-based cancer control programs, are not available in most low- and middle-income countries.  The Global Initiative for Cancer Registry Development is an international partnership that supports better estimation, as well as the collection and use of local data, to prioritize and evaluate national cancer control efforts.  CA: A Cancer Journal for Clinicians 2018;0:1-31. © 2018 American Cancer Society.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQaW5wv8ocuhLxanA7vODO6fW6udTcc2ebFwLic56H7rLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3c3otVGlsg%253D%253D&md5=48529f8f794092779d70a27eec5c9017</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.3322%2Fcaac.21492&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3322%252Fcaac.21492%26sid%3Dliteratum%253Aachs%26aulast%3DBray%26aufirst%3DF.%26aulast%3DFerlay%26aufirst%3DJ.%26aulast%3DSoerjomataram%26aufirst%3DI.%26aulast%3DSiegel%26aufirst%3DR.%2BL.%26aulast%3DTorre%26aufirst%3DL.%2BA.%26aulast%3DJemal%26aufirst%3DA.%26atitle%3DGlobal%2520Cancer%2520Statistics%25202018%253A%2520GLOBOCAN%2520Estimates%2520of%2520Incidence%2520and%2520Mortality%2520Worldwide%2520for%252036%2520Cancers%2520in%2520185%2520Countries%26jtitle%3DCa-Cancer%2520J.%2520Clin.%26date%3D2018%26volume%3D68%26spage%3D394%26epage%3D424%26doi%3D10.3322%2Fcaac.21492" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Forouzanfar, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foreman, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delossantos, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lozano, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, C. J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naghavi, M.</span></span> <span> </span><span class="NLM_article-title">Breast and Cervical Cancer in 187 Countries between 1980 and 2010: A Systematic Analysis</span>. <i>Lancet</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>378</i></span>,  <span class="NLM_fpage">1461</span>– <span class="NLM_lpage">1484</span>, <span class="refDoi"> DOI: 10.1016/S0140-6736(11)61351-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=10.1016%2FS0140-6736%2811%2961351-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=21924486" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=1%3ACAS%3A280%3ADC%252BC3MbjtFGkuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=378&publication_year=2011&pages=1461-1484&author=M.+H.+Forouzanfarauthor=K.+J.+Foremanauthor=A.+M.+Delossantosauthor=R.+Lozanoauthor=A.+D.+Lopezauthor=C.+J.+L.+Murrayauthor=M.+Naghavi&title=Breast+and+Cervical+Cancer+in+187+Countries+between+1980+and+2010%3A+A+Systematic+Analysis&doi=10.1016%2FS0140-6736%2811%2961351-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Breast and cervical cancer in 187 countries between 1980 and 2010: a systematic analysis</span></div><div class="casAuthors">Forouzanfar Mohammad H; Foreman Kyle J; Delossantos Allyne M; Lozano Rafael; Lopez Alan D; Murray Christopher J L; Naghavi Mohsen</div><div class="citationInfo"><span class="NLM_cas:title">Lancet (London, England)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">378</span>
        (<span class="NLM_cas:issue">9801</span>),
    <span class="NLM_cas:pages">1461-84</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Breast and cervical cancer are important causes of mortality in women aged ≥15 years.  We undertook annual age-specific assessments of breast and cervical cancer in 187 countries.  METHODS:  We systematically collected cancer registry data on mortality and incidence, vital registration, and verbal autopsy data for the period 1980-2010.  We modelled the mortality-to-incidence (MI) ratio using a hierarchical model.  Vital registration and verbal autopsy were supplemented with incidence multiplied by the MI ratio to yield a comprehensive database of mortality rates.  We used Gaussian process regression to develop estimates of mortality with uncertainty by age, sex, country, and year.  We used out-of-sample predictive validity to select the final model.  Estimates of incidence with uncertainty were also generated with mortality and MI ratios.  FINDINGS:  Global breast cancer incidence increased from 641,000 (95% uncertainty intervals 610,000-750,000) cases in 1980 to 1,643,000 (1,421,000-1,782,000) cases in 2010, an annual rate of increase of 3·1%.  Global cervical cancer incidence increased from 378,000 (256,000-489,000) cases per year in 1980 to 454,000 (318,000-620,000) cases per year in 2010-a 0·6% annual rate of increase.  Breast cancer killed 425,000 (359,000-453,000) women in 2010, of whom 68,000 (62,000-74,000) were aged 15-49 years in developing countries.  Cervical cancer death rates have been decreasing but the disease still killed 200,000 (139,000-276,000) women in 2010, of whom 46,000 (33,000-64,000) were aged 15-49 years in developing countries.  We recorded pronounced variation in the trend in breast cancer mortality across regions and countries.  INTERPRETATION:  More policy attention is needed to strengthen established health-system responses to reduce breast and cervical cancer, especially in developing countries.  FUNDING:  Susan G Komen for the Cure and the Bill & Melinda Gates Foundation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTjccp-xjq9WxfKb6ELK8t0fW6udTcc2ebFwLic56H7rLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3MbjtFGkuw%253D%253D&md5=19f693da0e637477c4014d7f3c56edc5</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2FS0140-6736%2811%2961351-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0140-6736%252811%252961351-2%26sid%3Dliteratum%253Aachs%26aulast%3DForouzanfar%26aufirst%3DM.%2BH.%26aulast%3DForeman%26aufirst%3DK.%2BJ.%26aulast%3DDelossantos%26aufirst%3DA.%2BM.%26aulast%3DLozano%26aufirst%3DR.%26aulast%3DLopez%26aufirst%3DA.%2BD.%26aulast%3DMurray%26aufirst%3DC.%2BJ.%2BL.%26aulast%3DNaghavi%26aufirst%3DM.%26atitle%3DBreast%2520and%2520Cervical%2520Cancer%2520in%2520187%2520Countries%2520between%25201980%2520and%25202010%253A%2520A%2520Systematic%2520Analysis%26jtitle%3DLancet%26date%3D2011%26volume%3D378%26spage%3D1461%26epage%3D1484%26doi%3D10.1016%2FS0140-6736%2811%2961351-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Waks, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winer, E. P.</span></span> <span> </span><span class="NLM_article-title">Breast Cancer Treatment: A Review</span>. <i>JAMA</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>321</i></span>,  <span class="NLM_fpage">288</span>, <span class="refDoi"> DOI: 10.1001/jama.2018.19323</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=10.1001%2Fjama.2018.19323" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=30667505" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhvV2qt7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=321&publication_year=2019&pages=288&author=A.+G.+Waksauthor=E.+P.+Winer&title=Breast+Cancer+Treatment%3A+A+Review&doi=10.1001%2Fjama.2018.19323"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Breast cancer treatment: a review</span></div><div class="casAuthors">Waks, Adrienne G.; Winer, Eric P.</div><div class="citationInfo"><span class="NLM_cas:title">JAMA, the Journal of the American Medical Association</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">321</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">288-300</span>CODEN:
                <span class="NLM_cas:coden">JAMAAP</span>;
        ISSN:<span class="NLM_cas:issn">1538-3598</span>.
    
            (<span class="NLM_cas:orgname">American Medical Association</span>)
        </div><div class="casAbstract">A review.  Importance Breast cancer will be diagnosed in 12% of women in the United States over the course of their lifetimes and more than 250 000 new cases of breast cancer were diagnosed in the United States in 2017.  This review focuses on current approaches and evolving strategies for local and systemic therapy of breast cancer. observations Breast cancer is categorized into 3 major subtypes based on the presence or absence of mol. markers for estrogen or progesterone receptors and human epidermal growth factor 2 (ERBB2; formerly HER2): hormone receptor pos./ERBB2 neg. (70% of patients), ERBB2 pos. (15%-20%), and triple-neg. (tumors lacking all 3 std. mol. markers; 15%).  More than 90% of breast cancers are not metastatic at the time of diagnosis.  For people presenting without metastatic disease, therapeutic goals are tumor eradication and preventing recurrence.  Triple-neg. breast cancer is more likely to recur than the other 2 subtypes, with 85% 5-yr breast cancer-specific survival for stage I triple-neg. tumors vs 94% to 99% for hormone receptor pos. and ERBB2 pos.  Systemic therapy for nonmetastatic breast cancer is detd. by subtype: patients with hormone receptor-pos. tumors receive endocrine therapy, and a minority receive chemotherapy as well; patients with ERBB2-pos. tumors receive ERBB2-targeted antibody or small-mol. inhibitor therapy combined with chemotherapy; and patients with triple-neg. tumors receive chemotherapy alone.  Local therapy for all patients with nonmetastatic breast cancer consists of surgical resection, with consideration of postoperative radiation if lumpectomy is performed.  Increasingly, some systemic therapy is delivered before surgery.  Tailoring postoperative treatment based on preoperative treatment response is under investigation.  Metastatic breast cancer is treated according to subtype, with goals of prolonging life and palliating symptoms.  Median overall survival for metastatic triple-neg. breast cancer is approx. 1 yr vs approx. 5 years for the other 2 subtypes. conclusions and relevance Breast cancer consists of 3 major tumor subtypes categorized according to estrogen or progesterone receptor expression and ERBB2 gene amplification.  The 3 subtypes have distinct risk profiles and treatment strategies.  Optimal therapy for each patient depends on tumor subtype, anat. cancer stage, and patient preferences.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGriTXwcPIcZ_LVg90H21EOLACvtfcHk0lhHi8KLLSIfRQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhvV2qt7c%253D&md5=548a8f22dc89eb559f95d779b37f3a9d</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1001%2Fjama.2018.19323&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1001%252Fjama.2018.19323%26sid%3Dliteratum%253Aachs%26aulast%3DWaks%26aufirst%3DA.%2BG.%26aulast%3DWiner%26aufirst%3DE.%2BP.%26atitle%3DBreast%2520Cancer%2520Treatment%253A%2520A%2520Review%26jtitle%3DJAMA%26date%3D2019%26volume%3D321%26spage%3D288%26doi%3D10.1001%2Fjama.2018.19323" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guan, X.</span></span> <span> </span><span class="NLM_article-title">Emerging Therapeutic Modalities of PARP Inhibitors in Breast Cancer</span>. <i>Cancer Treat. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>68</i></span>,  <span class="NLM_fpage">62</span>– <span class="NLM_lpage">68</span>, <span class="refDoi"> DOI: 10.1016/j.ctrv.2018.05.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=10.1016%2Fj.ctrv.2018.05.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=29870916" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtV2jur7O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2018&pages=62-68&author=X.+Wangauthor=Y.+Shiauthor=D.+Huangauthor=X.+Guan&title=Emerging+Therapeutic+Modalities+of+PARP+Inhibitors+in+Breast+Cancer&doi=10.1016%2Fj.ctrv.2018.05.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Emerging therapeutic modalities of PARP inhibitors in breast cancer</span></div><div class="casAuthors">Wang, Xin; Shi, Yaqin; Huang, Doudou; Guan, Xiaoxiang</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Treatment Reviews</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">68</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">62-68</span>CODEN:
                <span class="NLM_cas:coden">CTREDJ</span>;
        ISSN:<span class="NLM_cas:issn">0305-7372</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Inhibition of Poly (ADP-ribose) polymerase (PARP) has shown marked benefit for breast cancer with homologous recombination deficiency, whether driven by defects in BRCA1, BRCA2, or other pathway components.  Since the initial approval of olaparib, a mostly investigated PARP inhibitor (PARPi), the clin. development of PARPi in breast cancer treatment has been a major emphasis.  Researches in investigating platinum-PARPi combination use compared with platinum monotherapy demonstrated promising benefit in metastatic BRCA mutated breast cancer or TNBC, while no such superiority was obsd. in the neoadjuvant setting of TNBC.  Moreover, the utility of PARP inhibition in BRCA1/2 mutated breast cancer with different platinum-free interval was investigated.  There was a clear assocn. between clin. benefit with PARPi and platinum sensitivity, whereas partial efficacy of PARPi still occurs in platinum-resistant patients.  In addn., proof-of-principle studies of immunotherapy combined with PARPi in breast cancer have obtained promising results, indicating the potential benefit of the combination therapy in patients with breast cancer.  These efforts, contributing to maximize the utility of PARPi, may drive a new era of this agent after its first routine use.  In this review, we summarized the utility of combining platinum-PARPi in BRCA mutated breast cancer or TNBC compared with platinum monotherapy and provided promising prospects of PARPi as maintenance therapy in breast cancer, as well as providing a strong rationale for testing immunotherapy combined with PARPi in breast cancer to expand the clin. utility of PARPi.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrtoy0qBmshirVg90H21EOLACvtfcHk0lhHi8KLLSIfRQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtV2jur7O&md5=1c28f7585b4deace459da25b2ff90d32</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.ctrv.2018.05.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ctrv.2018.05.014%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DX.%26aulast%3DShi%26aufirst%3DY.%26aulast%3DHuang%26aufirst%3DD.%26aulast%3DGuan%26aufirst%3DX.%26atitle%3DEmerging%2520Therapeutic%2520Modalities%2520of%2520PARP%2520Inhibitors%2520in%2520Breast%2520Cancer%26jtitle%3DCancer%2520Treat.%2520Rev.%26date%3D2018%26volume%3D68%26spage%3D62%26epage%3D68%26doi%3D10.1016%2Fj.ctrv.2018.05.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Exman, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barroso-Sousa, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tolaney, S. M.</span></span> <span> </span><span class="NLM_article-title">Evidence to Date: Talazoparib in the Treatment of Breast Cancer</span>. <i>OncoTargets Ther.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">5177</span>– <span class="NLM_lpage">5187</span>, <span class="refDoi"> DOI: 10.2147/OTT.S184971</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=10.2147%2FOTT.S184971" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=31303769" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=1%3ACAS%3A528%3ADC%252BB3cXjsVGkt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2019&pages=5177-5187&author=P.+Exmanauthor=R.+Barroso-Sousaauthor=S.+M.+Tolaney&title=Evidence+to+Date%3A+Talazoparib+in+the+Treatment+of+Breast+Cancer&doi=10.2147%2FOTT.S184971"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Evidence to date: talazoparib in the treatment of breast cancer</span></div><div class="casAuthors">Exman, Pedro; Barroso-Sousa, Romualdo; Tolaney, Sara M.</div><div class="citationInfo"><span class="NLM_cas:title">OncoTargets and Therapy</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">5177-5187</span>CODEN:
                <span class="NLM_cas:coden">OTNHAZ</span>;
        ISSN:<span class="NLM_cas:issn">1178-6930</span>.
    
            (<span class="NLM_cas:orgname">Dove Medical Press Ltd.</span>)
        </div><div class="casAbstract">Approx. 5-10% of all patients diagnosed with breast cancer have germline BRCA1/2 mutations, which make their disease more susceptible to DNA-damaging agents and a new class of drugs known as poly(ADP-ribose) polymerase (PARP) inhibitors.  Talazoparib is a new PARP inhibitor that has been recently approved for use in patients with metastatic breast cancer with germline BRCA mutations after a phase III trial showed superior progression-free survival when compared to std. chemotherapy.  In this review, we analyze the development of talazoparib as well as its safety profile and the potential role of the combination therapy with std. cytotoxic drugs and with novel therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqKyX-RFMagpLVg90H21EOLACvtfcHk0lhHi8KLLSIfRQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXjsVGkt7s%253D&md5=3c49b35879d525ecce2bf383188c835a</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.2147%2FOTT.S184971&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2147%252FOTT.S184971%26sid%3Dliteratum%253Aachs%26aulast%3DExman%26aufirst%3DP.%26aulast%3DBarroso-Sousa%26aufirst%3DR.%26aulast%3DTolaney%26aufirst%3DS.%2BM.%26atitle%3DEvidence%2520to%2520Date%253A%2520Talazoparib%2520in%2520the%2520Treatment%2520of%2520Breast%2520Cancer%26jtitle%3DOncoTargets%2520Ther.%26date%3D2019%26volume%3D12%26spage%3D5177%26epage%3D5187%26doi%3D10.2147%2FOTT.S184971" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Popuri, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tadokoro, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Croteau, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bohr, V. A.</span></span> <span> </span><span class="NLM_article-title">Human RECQL5: Guarding the Crossroads of DNA Replication and Transcription and Providing Backup Capability</span>. <i>Crit. Rev. Biochem. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">289</span>– <span class="NLM_lpage">299</span>, <span class="refDoi"> DOI: 10.3109/10409238.2013.792770</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=10.3109%2F10409238.2013.792770" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=23627586" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=1%3ACAS%3A528%3ADC%252BC3sXos12gtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2013&pages=289-299&author=V.+Popuriauthor=T.+Tadokoroauthor=D.+L.+Croteauauthor=V.+A.+Bohr&title=Human+RECQL5%3A+Guarding+the+Crossroads+of+DNA+Replication+and+Transcription+and+Providing+Backup+Capability&doi=10.3109%2F10409238.2013.792770"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Human RECQL5: Guarding the crossroads of DNA replication and transcription and providing backup capability</span></div><div class="casAuthors">Popuri, Venkateswarlu; Tadokoro, Takashi; Croteau, Deborah L.; Bohr, Vilhelm A.</div><div class="citationInfo"><span class="NLM_cas:title">Critical Reviews in Biochemistry and Molecular Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">289-299</span>CODEN:
                <span class="NLM_cas:coden">CRBBEJ</span>;
        ISSN:<span class="NLM_cas:issn">1040-9238</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  DNA helicases are ubiquitous enzymes that catalyze the unwinding of duplex DNA and function in all metabolic processes in which access to single-stranded DNA is required, including DNA replication, repair, recombination, and RNA transcription.  RecQ helicases are a conserved family of DNA helicases that display highly specialized and vital roles in the maintenance of genome stability.  Mutations in 3 of the 5 human RecQ helicases (BLM, WRN, and RECQL4) are assocd. with the genetic disorders, Bloom syndrome, Werner syndrome, and Rothmund-Thomson syndrome, that are characterized by chromosomal instability, premature aging, and predisposition to cancer.  The biol. role of human RECQL5 is only partially understood and RECQL5 has not yet been assocd. with any human disease.  Illegitimate recombination and replication stress are hallmarks of human cancers and common instigators for genomic instability and cell death.  Gene Recql5-knockout mice are cancer prone and show increased chromosomal instability.  Recql5-deficient mouse embryonic fibroblasts are sensitive to camptothecin and display elevated levels of sister chromatid exchanges.  Unlike other human RecQ helicases, RECQL5 is recruited to single-stranded DNA breaks and is also proposed to play an essential role in RNA transcription.  Here, the authors review the established roles of RECQL5 at the crossroads of DNA replication, recombination, and transcription, and propose that human RECQL5 provides important backup functions in the absence of other DNA helicases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrj_4k8QzuNo7Vg90H21EOLACvtfcHk0lgsdlc9W3T1ZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXos12gtL8%253D&md5=52b229f58b912e59c0e46c50f718d6e6</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.3109%2F10409238.2013.792770&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3109%252F10409238.2013.792770%26sid%3Dliteratum%253Aachs%26aulast%3DPopuri%26aufirst%3DV.%26aulast%3DTadokoro%26aufirst%3DT.%26aulast%3DCroteau%26aufirst%3DD.%2BL.%26aulast%3DBohr%26aufirst%3DV.%2BA.%26atitle%3DHuman%2520RECQL5%253A%2520Guarding%2520the%2520Crossroads%2520of%2520DNA%2520Replication%2520and%2520Transcription%2520and%2520Providing%2520Backup%2520Capability%26jtitle%3DCrit.%2520Rev.%2520Biochem.%2520Mol.%2520Biol.%26date%3D2013%26volume%3D48%26spage%3D289%26epage%3D299%26doi%3D10.3109%2F10409238.2013.792770" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brosh, R. M.</span></span> <span> </span><span class="NLM_article-title">DNA Helicases Involved in DNA Repair and Their Roles in Cancer</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">542</span>– <span class="NLM_lpage">558</span>, <span class="refDoi"> DOI: 10.1038/nrc3560</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=10.1038%2Fnrc3560" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=23842644" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtVyktLfL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2013&pages=542-558&author=R.+M.+Brosh&title=DNA+Helicases+Involved+in+DNA+Repair+and+Their+Roles+in+Cancer&doi=10.1038%2Fnrc3560"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">DNA helicases involved in DNA repair and their roles in cancer</span></div><div class="casAuthors">Brosh, Robert M., Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">542-558</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Helicases have major roles in genome maintenance by unwinding structured nucleic acids.  Their prominence is marked by various cancers and genetic disorders that are linked to helicase defects.  Although considerable effort has been made to understand the functions of DNA helicases that are important for genomic stability and cellular homeostasis, the complexity of the DNA damage response leaves the authors with unanswered questions regarding how helicase-dependent DNA repair pathways are regulated and coordinated with cell cycle checkpoints.  Further studies may open the door to targeting helicases to improve cancer treatments based on DNA-damaging chemotherapy or radiation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQPs2MtOs8VrVg90H21EOLACvtfcHk0lgsdlc9W3T1ZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtVyktLfL&md5=e3cbd291237aecd2f801287d0e8dbb53</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fnrc3560&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc3560%26sid%3Dliteratum%253Aachs%26aulast%3DBrosh%26aufirst%3DR.%2BM.%26atitle%3DDNA%2520Helicases%2520Involved%2520in%2520DNA%2520Repair%2520and%2520Their%2520Roles%2520in%2520Cancer%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2013%26volume%3D13%26spage%3D542%26epage%3D558%26doi%3D10.1038%2Fnrc3560" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhi, L.-Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, S.-X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, B.</span></span> <span> </span><span class="NLM_article-title">Association of RECQL5 Gene Polymorphisms and Osteosarcoma in a Chinese Han Population</span>. <i>Tumor Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">3255</span>– <span class="NLM_lpage">3259</span>, <span class="refDoi"> DOI: 10.1007/s13277-013-1425-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=10.1007%2Fs13277-013-1425-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmtlSqsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2014&pages=3255-3259&author=L.-Q.+Zhiauthor=W.+Maauthor=H.+Zhangauthor=S.-X.+Zengauthor=B.+Chen&title=Association+of+RECQL5+Gene+Polymorphisms+and+Osteosarcoma+in+a+Chinese+Han+Population&doi=10.1007%2Fs13277-013-1425-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Association of RECQL5 gene polymorphisms and osteosarcoma in a Chinese Han population</span></div><div class="casAuthors">Zhi, Li-Qiang; Ma, Wei; Zhang, Hong; Zeng, Si-Xiang; Chen, Bo</div><div class="citationInfo"><span class="NLM_cas:title">Tumor Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">3255-3259</span>CODEN:
                <span class="NLM_cas:coden">TUMBEA</span>;
        ISSN:<span class="NLM_cas:issn">1010-4283</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Despite the knowledge on many genetic variants present in osteosarcoma, the complexity of this disease precludes placing its biol. into a simple conceptual framework.  RECQL is a DNA helicase involved in DNA mismatch repair and has been reported to be assocd. with many human cancers.  We aimed to investigate the assocn. of RECQL genetic polymorphism with osteosarcoma in a Chinese population.  We selected three polymorphisms of the RECQL5 gene (rs820196, rs820200, and rs4789223) in the present study.  TaqMan method was utilized for genotyping these three SNPs in 212 patients with osteosarcoma and 240 age- and sex-matched noncancer controls.  In our study, we found that CC genotype in rs820196 (17.5 vs 8.3 %, P = 0.005) and AA genotype in rs4789223 (21.7 vs 14.2, P < 0.001) were more frequent in osteosarcoma group compared to the control group, resp.  We also found that the C allele of rs820196 (OR = 1.492, 95 % CI 1.138∼1.951; P = 0.004) and A allele of rs4789223 (OR = 1.767, 95 % CI: 1.354 ∼ 2.301; P < 0.001) were common in the osteosarcoma patients than those in the control subjects, resp.  Haplotype anal. showed that TTA (OR = 3.469, 95 % CI 1.798∼6.695; P < 0.001) was assocd. with increased risk for osteosarcoma.  However, the TTG (OR = 0.578, 95 % CI 0.442∼0.756) was assocd. with decreased risk for osteosarcoma.  Our results suggested that RECQL5 genetic polymorphisms were assocd. with osteosarcoma in a Chinese population.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9zacLjGqyV7Vg90H21EOLACvtfcHk0lgsdlc9W3T1ZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmtlSqsbw%253D&md5=5b1021bf0d327dea0a3fbca97ae6ca5c</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1007%2Fs13277-013-1425-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs13277-013-1425-4%26sid%3Dliteratum%253Aachs%26aulast%3DZhi%26aufirst%3DL.-Q.%26aulast%3DMa%26aufirst%3DW.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DZeng%26aufirst%3DS.-X.%26aulast%3DChen%26aufirst%3DB.%26atitle%3DAssociation%2520of%2520RECQL5%2520Gene%2520Polymorphisms%2520and%2520Osteosarcoma%2520in%2520a%2520Chinese%2520Han%2520Population%26jtitle%3DTumor%2520Biol.%26date%3D2014%26volume%3D35%26spage%3D3255%26epage%3D3259%26doi%3D10.1007%2Fs13277-013-1425-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dong, Y. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Y. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, T.</span></span> <span> </span><span class="NLM_article-title">Single Nucleotide Polymorphism in the RECQL5 Gene Increased Osteosarcoma Susceptibility in a Chinese Han Population</span>. <i>GMR, Genet. Mol. Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">1899</span>– <span class="NLM_lpage">1902</span>, <span class="refDoi"> DOI: 10.4238/2015.March.13.18</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=10.4238%2F2015.March.13.18" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=25867335" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=1%3ACAS%3A528%3ADC%252BC2MXlsVejtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2015&pages=1899-1902&author=Y.+Z.+Dongauthor=Y.+X.+Huangauthor=T.+Lu&title=Single+Nucleotide+Polymorphism+in+the+RECQL5+Gene+Increased+Osteosarcoma+Susceptibility+in+a+Chinese+Han+Population&doi=10.4238%2F2015.March.13.18"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Single nucleotide polymorphism in the RECQL5 gene increased osteosarcoma susceptibility in a Chinese Han population</span></div><div class="casAuthors">Dong, Y. Z.; Huang, Y. X.; Lu, T.</div><div class="citationInfo"><span class="NLM_cas:title">GMR, Genetics and Molecular Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1899-1902</span>CODEN:
                <span class="NLM_cas:coden">GGMRBI</span>;
        ISSN:<span class="NLM_cas:issn">1676-5680</span>.
    
            (<span class="NLM_cas:orgname">FUNPEP</span>)
        </div><div class="casAbstract">In this study, we investigated the assocn. between a RECQL genetic polymorphism and osteosarcoma in a Chinese population.  We selected rs820196 in the RECQL5 gene and genotyped 185 patients with osteosarcoma and 201 age- and gender-matched non-cancer controls.  We found that the CC genotype was more frequent in the osteosarcoma group compared to the control group (P = 0.011).  We also found that the C allele was more common in osteosarcoma patients than that in control subjects (P = 0.004).  Our results suggested that the RECQL5 genetic polymorphism was assocd. with osteosarcoma in a Chinese population.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpcY1342JNvcbVg90H21EOLACvtfcHk0lgsdlc9W3T1ZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXlsVejtbo%253D&md5=ab44c87a23db294690669405d3d2d1d6</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.4238%2F2015.March.13.18&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4238%252F2015.March.13.18%26sid%3Dliteratum%253Aachs%26aulast%3DDong%26aufirst%3DY.%2BZ.%26aulast%3DHuang%26aufirst%3DY.%2BX.%26aulast%3DLu%26aufirst%3DT.%26atitle%3DSingle%2520Nucleotide%2520Polymorphism%2520in%2520the%2520RECQL5%2520Gene%2520Increased%2520Osteosarcoma%2520Susceptibility%2520in%2520a%2520Chinese%2520Han%2520Population%26jtitle%3DGMR%252C%2520Genet.%2520Mol.%2520Res.%26date%3D2015%26volume%3D14%26spage%3D1899%26epage%3D1902%26doi%3D10.4238%2F2015.March.13.18" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Qi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, X.</span></span> <span> </span><span class="NLM_article-title">Haplotype Analysis of RECQL5 Gene and Laryngeal Cancer</span>. <i>Tumor Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">2669</span>– <span class="NLM_lpage">2673</span>, <span class="refDoi"> DOI: 10.1007/s13277-013-1351-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=10.1007%2Fs13277-013-1351-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=1%3ACAS%3A528%3ADC%252BC2cXltlyrtb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2014&pages=2669-2673&author=Y.+Qiauthor=X.+Zhou&title=Haplotype+Analysis+of+RECQL5+Gene+and+Laryngeal+Cancer&doi=10.1007%2Fs13277-013-1351-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Haplotype analysis of RECQL5 gene and laryngeal cancer</span></div><div class="casAuthors">Qi, Ying; Zhou, Xu</div><div class="citationInfo"><span class="NLM_cas:title">Tumor Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">2669-2673</span>CODEN:
                <span class="NLM_cas:coden">TUMBEA</span>;
        ISSN:<span class="NLM_cas:issn">1010-4283</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">RECQL is a DNA helicase involved in DNA mismatch repair.  Previous studies indicated that the RECQL gene mutation was assocd. with human cancers.  In the present study, we investigated the assocn. between polymorphisms of RECQL gene and laryngeal cancer in a Chinese population.  Four polymorphisms of the RECQL5 gene (rs820186, rs820196, rs820200, and rs4789223) were genotyped by the TaqMan method in 275 patients with larynx cancer and 300 age- and sex-matched non-cancer controls.  We found that rs820196 polymorphism of RECQL5 was assocd. with larynx cancer, the CC genotype (16.4 % vs 9.3 %, P=0.013) and C allele (42 % vs 34.2 %, P=0.006) was common in larynx cancer patients than in the control subjects, resp.  Haplotype anal. showed that GCGA (OR=18.955, 95 % confidence interval (CI) 9.479∼37.905; P<0.001) and GTTG (OR=1.684; 95 % CI 1.327∼2.137; P<0.001) were assocd. with increased risk for larynx cancer.  However, ACGA (OR=0.605; 95 % CI 0.430∼0.852; P=0.003), GCGG (OR=0.636; 95 % CI 0.411∼0.982; P=0.039), and GTGG (OR=0.194; 95 % CI 0.104∼0.361; P<0.001) were assocd. with decreased risk for larynx cancer.  The present study indicated that RECQL5 genetic polymorphism and haplotypes were assocd. with larynx cancer in a Chinese population.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoArsd34p2XXrVg90H21EOLACvtfcHk0ljRqjTkPs9mIw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXltlyrtb0%253D&md5=c7ad62452eb5fd3b6099aaead36eaa19</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1007%2Fs13277-013-1351-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs13277-013-1351-5%26sid%3Dliteratum%253Aachs%26aulast%3DQi%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DX.%26atitle%3DHaplotype%2520Analysis%2520of%2520RECQL5%2520Gene%2520and%2520Laryngeal%2520Cancer%26jtitle%3DTumor%2520Biol.%26date%3D2014%26volume%3D35%26spage%3D2669%26epage%3D2673%26doi%3D10.1007%2Fs13277-013-1351-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">He, Y.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiao, Z.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, Y.-Y.</span></span> <span> </span><span class="NLM_article-title">Association between RECQL5 Genetic Polymorphisms and Susceptibility to Breast Cancer</span>. <i>Tumor Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">12201</span>– <span class="NLM_lpage">12204</span>, <span class="refDoi"> DOI: 10.1007/s13277-014-2528-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=10.1007%2Fs13277-014-2528-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFGgtrfO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2014&pages=12201-12204&author=Y.-J.+Heauthor=Z.-Y.+Qiaoauthor=B.+Gaoauthor=X.-H.+Zhangauthor=Y.-Y.+Wen&title=Association+between+RECQL5+Genetic+Polymorphisms+and+Susceptibility+to+Breast+Cancer&doi=10.1007%2Fs13277-014-2528-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Association between RECQL5 genetic polymorphisms and susceptibility to breast cancer</span></div><div class="casAuthors">He, Yu-Jun; Qiao, Zuo-Yi; Gao, Bo; Zhang, Xiao-Hua; Wen, Ya-Yuan</div><div class="citationInfo"><span class="NLM_cas:title">Tumor Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">12201-12204</span>CODEN:
                <span class="NLM_cas:coden">TUMBEA</span>;
        ISSN:<span class="NLM_cas:issn">1010-4283</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Previous studies indicated that the RECQL5 gene polymorphism was assocd. with human cancers.  However, the assocn. of RECQL5 gene polymorphism with breast cancer remains unclear.  In the present study, we investigated the assocn. between polymorphisms of the RECQL gene and breast cancer in a Chinese population.  We selected four polymorphisms of the RECQL5 gene (rs820186, rs820196, rs820200, and rs4789223) for the present study.  The genotyping was performed using the TaqMan method in 510 patients with breast cancer and 510 age- and sex-matched non-cancer controls.  We found that rs820196 and rs828200 polymorphisms of RECQL5 were assocd. with breast cancer.  For rs820196, the CC genotype (16.7 vs 9.4 %, P < 0.001) and C allele (42.5 vs 34.3 %, P < 0.001) were common in the breast cancer patients than in the control subjects, resp.  For rs828200, the GG genotype (23.7 vs 18.0 %, P < 0.001) and G allele (52.7 vs 43.8 %, P < 0.001) were common in the breast cancer patients than in the control subjects, resp.  Haplotype anal. showed that C-G (odds ratio (OR) = 2.247, 95 % confidence interval (CI) 1.854∼2.722; P < 0.001) was assocd. with increased risk for breast cancer.  However, the C-T (OR = 0.175, 95 % CI 0.110∼0.278; P < 0.001) and T-G (OR = 0.544; 95 % CI 0.428∼0.692; P < 0.001) were assocd. with decreased risk for breast cancer, resp.  The present study indicated that the RECQL5 genetic polymorphism and haplotypes were assocd. with breast cancer in a Chinese population.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrYEZFicetXTbVg90H21EOLACvtfcHk0ljRqjTkPs9mIw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvFGgtrfO&md5=ce91471378a894696b28ef0ba89f5a8f</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1007%2Fs13277-014-2528-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs13277-014-2528-2%26sid%3Dliteratum%253Aachs%26aulast%3DHe%26aufirst%3DY.-J.%26aulast%3DQiao%26aufirst%3DZ.-Y.%26aulast%3DGao%26aufirst%3DB.%26aulast%3DZhang%26aufirst%3DX.-H.%26aulast%3DWen%26aufirst%3DY.-Y.%26atitle%3DAssociation%2520between%2520RECQL5%2520Genetic%2520Polymorphisms%2520and%2520Susceptibility%2520to%2520Breast%2520Cancer%26jtitle%3DTumor%2520Biol.%26date%3D2014%26volume%3D35%26spage%3D12201%26epage%3D12204%26doi%3D10.1007%2Fs13277-014-2528-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Arora, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abdel-Fatah, T. M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agarwal, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doherty, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Croteau, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moseley, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hameed, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aleskandarany, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rakha, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patterson, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ball, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, S. Y. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellis, I. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bohr, V. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bryant, H. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Madhusudan, S.</span></span> <span> </span><span class="NLM_article-title">Clinicopathological and Prognostic Significance of RECQL5 Helicase Expression in Breast Cancers</span>. <i>Carcinogenesis</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>37</i></span>,  <span class="NLM_fpage">63</span>– <span class="NLM_lpage">71</span>, <span class="refDoi"> DOI: 10.1093/carcin/bgv163</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=10.1093%2Fcarcin%2Fbgv163" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=26586793" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht12rsLnN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2016&pages=63-71&author=A.+Aroraauthor=T.+M.+A.+Abdel-Fatahauthor=D.+Agarwalauthor=R.+Dohertyauthor=D.+L.+Croteauauthor=P.+M.+Moseleyauthor=K.+Hameedauthor=A.+Greenauthor=M.+A.+Aleskandaranyauthor=E.+A.+Rakhaauthor=K.+Pattersonauthor=G.+Ballauthor=S.+Y.+T.+Chanauthor=I.+O.+Ellisauthor=V.+A.+Bohrauthor=H.+E.+Bryantauthor=S.+Madhusudan&title=Clinicopathological+and+Prognostic+Significance+of+RECQL5+Helicase+Expression+in+Breast+Cancers&doi=10.1093%2Fcarcin%2Fbgv163"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Clinicopathological and prognostic significance of RECQL5 helicase expression in breast cancers</span></div><div class="casAuthors">Arora, Arvind; Abdel-Fatah, Tarek M. A.; Agarwal, Devika; Doherty, Rachel; Croteau, Deborah L.; Moseley, Paul M.; Hameed, Khalid; Green, Andrew; Aleskandarany, Mohammed A.; Rakha, Emad A.; Patterson, Karl; Ball, Graham; Chan, Stephen Y. T.; Ellis, Ian O.; Bohr, Vilhelm A.; Bryant, Helen E.; Madhusudan, Srinivasan</div><div class="citationInfo"><span class="NLM_cas:title">Carcinogenesis</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">63-71</span>CODEN:
                <span class="NLM_cas:coden">CRNGDP</span>;
        ISSN:<span class="NLM_cas:issn">0143-3334</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">RECQL5 is a member of the RecQ family of DNA helicases and has key roles in homologous recombination, base excision repair, replication and transcription.  The clinicopathol. significance of RECQL5 expression in breast cancer is unknown.  In this study, we have evaluated RECQL5 mRNA expression in 1977 breast cancers, and RECQL5 protein level in 1902 breast cancers [Nottingham Tenovus series (n = 1650) and ER- cohort (n = 252)].  Expression levels were correlated to aggressive phenotypes and survival outcomes.  High RECQL5 mRNA expression was significantly assocd. with high histol. grade (P = 0.007), HER2 overexpression (P = 0.032), ER+/HER2-/high proliferation genefu subtype (P < 0.0001), integrative mol. clusters (intClust 1and 9) (P < 0.0001) and poor survival (P < 0.0001).  In subgroup anal., high RECQL5 mRNA level remains significantly assocd. with poor BCSS in ER+ cohort (P < 0.0001) but not in ER- cohort (P = 0.116).  At the protein level, in tumors with low RAD51, high RECQL5 level was significantly assocd. with high histol. grade (P < 0.0001), higher mitotic index (P = 0.008), dedifferentiation (P = 0.025), pleomorphism (P = 0.027) and poor survival (P = 0.003).  In subgroup anal., high RECQL5/low RAD51 remains significantly assocd. with poor BCSS in ER+ cohort (P = 0.010), but not in ER-cohort (P = 0.628).  In multivariate anal., high RECQL5 mRNA and high RECQL5/low RAD51 nuclear protein coexpression independently influenced survival (P = 0.022) in whole cohort and in the ER+ subgroup.  Preclinically, we show that exogenous expression of RECQL5 in MCF10A cells can drive proliferation supporting an oncogenic function for RECQL5 in breast cancer.  We conclude that RECQL5 is a promising biomarker in breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLbW-Nlbo5p7Vg90H21EOLACvtfcHk0ljRqjTkPs9mIw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht12rsLnN&md5=688c8b4611c229def638bbd7dfd8b17d</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1093%2Fcarcin%2Fbgv163&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fcarcin%252Fbgv163%26sid%3Dliteratum%253Aachs%26aulast%3DArora%26aufirst%3DA.%26aulast%3DAbdel-Fatah%26aufirst%3DT.%2BM.%2BA.%26aulast%3DAgarwal%26aufirst%3DD.%26aulast%3DDoherty%26aufirst%3DR.%26aulast%3DCroteau%26aufirst%3DD.%2BL.%26aulast%3DMoseley%26aufirst%3DP.%2BM.%26aulast%3DHameed%26aufirst%3DK.%26aulast%3DGreen%26aufirst%3DA.%26aulast%3DAleskandarany%26aufirst%3DM.%2BA.%26aulast%3DRakha%26aufirst%3DE.%2BA.%26aulast%3DPatterson%26aufirst%3DK.%26aulast%3DBall%26aufirst%3DG.%26aulast%3DChan%26aufirst%3DS.%2BY.%2BT.%26aulast%3DEllis%26aufirst%3DI.%2BO.%26aulast%3DBohr%26aufirst%3DV.%2BA.%26aulast%3DBryant%26aufirst%3DH.%2BE.%26aulast%3DMadhusudan%26aufirst%3DS.%26atitle%3DClinicopathological%2520and%2520Prognostic%2520Significance%2520of%2520RECQL5%2520Helicase%2520Expression%2520in%2520Breast%2520Cancers%26jtitle%3DCarcinogenesis%26date%3D2016%26volume%3D37%26spage%3D63%26epage%3D71%26doi%3D10.1093%2Fcarcin%2Fbgv163" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Andrs, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hasanova, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oravetzova, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dobrovolna, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janscak, P.</span></span> <span> </span><span class="NLM_article-title">RECQ5: A Mysterious Helicase at the Interface of DNA Replication and Transcription</span>. <i>Genes</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">232</span>, <span class="refDoi"> DOI: 10.3390/genes11020232</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=10.3390%2Fgenes11020232" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=1%3ACAS%3A528%3ADC%252BB3cXotlKnt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=232&author=M.+Andrsauthor=Z.+Hasanovaauthor=A.+Oravetzovaauthor=J.+Dobrovolnaauthor=P.+Janscak&title=RECQ5%3A+A+Mysterious+Helicase+at+the+Interface+of+DNA+Replication+and+Transcription&doi=10.3390%2Fgenes11020232"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">RECQ5: a mysterious helicase at the interface of DNA replication and transcription</span></div><div class="casAuthors">Andrs, Martin; Hasanova, Zdenka; Oravetzova, Anna; Dobrovolna, Jana; Janscak, Pavel</div><div class="citationInfo"><span class="NLM_cas:title">Genes</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">232</span>CODEN:
                <span class="NLM_cas:coden">GENEG9</span>;
        ISSN:<span class="NLM_cas:issn">2073-4425</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">RECQ5 belongs to the RecQ family of DNA helicases.  It is conserved from Drosophila to humans and its deficiency results in genomic instability and cancer susceptibility in mice.  Human RECQ5 is known for its ability to regulate homologous recombination by disrupting RAD51 nucleoprotein filaments.  It also binds to RNA polymerase II (RNAPII) and neg. regulates transcript elongation by RNAPII.  Here, we summarize recent studies implicating RECQ5 in the prevention and resoln. of transcription-replication conflicts, a major intrinsic source of genomic instability during cancer development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzmhnNQYCHSLVg90H21EOLACvtfcHk0ljSIj7SzlzNZw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXotlKnt7o%253D&md5=d997f26d6598328f00ba07c086634216</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.3390%2Fgenes11020232&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fgenes11020232%26sid%3Dliteratum%253Aachs%26aulast%3DAndrs%26aufirst%3DM.%26aulast%3DHasanova%26aufirst%3DZ.%26aulast%3DOravetzova%26aufirst%3DA.%26aulast%3DDobrovolna%26aufirst%3DJ.%26aulast%3DJanscak%26aufirst%3DP.%26atitle%3DRECQ5%253A%2520A%2520Mysterious%2520Helicase%2520at%2520the%2520Interface%2520of%2520DNA%2520Replication%2520and%2520Transcription%26jtitle%3DGenes%26date%3D2020%26volume%3D11%26spage%3D232%26doi%3D10.3390%2Fgenes11020232" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schwendener, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raynard, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paliwal, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kanagaraj, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shevelev, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stark, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sung, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janscak, P.</span></span> <span> </span><span class="NLM_article-title">Physical Interaction of RECQ5 Helicase with RAD51 Facilitates Its Anti-Recombinase Activity</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>285</i></span>,  <span class="NLM_fpage">15739</span>– <span class="NLM_lpage">15745</span>, <span class="refDoi"> DOI: 10.1074/jbc.M110.110478</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=10.1074%2Fjbc.M110.110478" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=20348101" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=1%3ACAS%3A528%3ADC%252BC3cXmtVyks7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=285&publication_year=2010&pages=15739-15745&author=S.+Schwendenerauthor=S.+Raynardauthor=S.+Paliwalauthor=A.+Chengauthor=R.+Kanagarajauthor=I.+Shevelevauthor=J.+M.+Starkauthor=P.+Sungauthor=P.+Janscak&title=Physical+Interaction+of+RECQ5+Helicase+with+RAD51+Facilitates+Its+Anti-Recombinase+Activity&doi=10.1074%2Fjbc.M110.110478"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Physical Interaction of RECQ5 Helicase with RAD51 Facilitates Its Anti-recombinase Activity</span></div><div class="casAuthors">Schwendener, Sybille; Raynard, Steven; Paliwal, Shreya; Cheng, Anita; Kanagaraj, Radhakrishnan; Shevelev, Igor; Stark, Jeremy M.; Sung, Patrick; Janscak, Pavel</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">285</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">15739-15745</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Homologous recombination (HR) provides an efficient mechanism for error-free repair of DNA double-strand breaks (DSBs).  However, HR can also be harmful, as inappropriate or untimely HR events can give rise to lethal recombination intermediates and chromosome rearrangements.  A crit. step of HR is the formation of a RAD51 filament on single-stranded (ss)DNA, which mediates the invasion of a homologous DNA mol.  In mammalian cells, several DNA helicases have been implicated in the regulation of this process.  RECQ5, a member of the RecQ family of DNA helicases, interacts phys. with the RAD51 recombinase and disrupts RAD51 presynaptic filaments in a reaction dependent on ATP hydrolysis.  Here, we have precisely mapped the RAD51-interacting domain of RECQ5 and generated mutants that fail to interact with RAD51.  We show that although these mutants retain normal ATPase activity, they are impaired in their ability to displace RAD51 from ssDNA.  Moreover, we show that ablation of RECQ5-RAD51 complex formation by a point mutation alleviates the inhibitory effect of RECQ5 on HR-mediated DSB repair.  These findings provide support for the proposal that interaction with RAD51 is crit. for the anti-recombinase attribute of RECQ5.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDTDB-CKfN5rVg90H21EOLACvtfcHk0ljSIj7SzlzNZw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXmtVyks7Y%253D&md5=3185152ffc63f32bc0857e7c04ff2c17</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M110.110478&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M110.110478%26sid%3Dliteratum%253Aachs%26aulast%3DSchwendener%26aufirst%3DS.%26aulast%3DRaynard%26aufirst%3DS.%26aulast%3DPaliwal%26aufirst%3DS.%26aulast%3DCheng%26aufirst%3DA.%26aulast%3DKanagaraj%26aufirst%3DR.%26aulast%3DShevelev%26aufirst%3DI.%26aulast%3DStark%26aufirst%3DJ.%2BM.%26aulast%3DSung%26aufirst%3DP.%26aulast%3DJanscak%26aufirst%3DP.%26atitle%3DPhysical%2520Interaction%2520of%2520RECQ5%2520Helicase%2520with%2520RAD51%2520Facilitates%2520Its%2520Anti-Recombinase%2520Activity%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2010%26volume%3D285%26spage%3D15739%26epage%3D15745%26doi%3D10.1074%2Fjbc.M110.110478" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hosono, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abe, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishiai, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Islam, M. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arakawa, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takeda, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishii, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takata, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seki, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Enomoto, T.</span></span> <span> </span><span class="NLM_article-title">Tumor suppressor RecQL5 controls recombination induced by DNA crosslinking agents</span>. <i>Biochim. Biophys. Acta, Mol. Cell Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>1843</i></span>,  <span class="NLM_fpage">1002</span>– <span class="NLM_lpage">1012</span>, <span class="refDoi"> DOI: 10.1016/j.bbamcr.2014.01.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=10.1016%2Fj.bbamcr.2014.01.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjsF2gt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1843&publication_year=2014&pages=1002-1012&author=Y.+Hosonoauthor=T.+Abeauthor=M.+Ishiaiauthor=M.+N.+Islamauthor=H.+Arakawaauthor=W.+Wangauthor=S.+Takedaauthor=Y.+Ishiiauthor=M.+Takataauthor=M.+Sekiauthor=T.+Enomoto&title=Tumor+suppressor+RecQL5+controls+recombination+induced+by+DNA+crosslinking+agents&doi=10.1016%2Fj.bbamcr.2014.01.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Tumor suppressor RecQL5 controls recombination induced by DNA crosslinking agents</span></div><div class="casAuthors">Hosono, Yoshifumi; Abe, Takuya; Ishiai, Masamichi; Islam, M. Nurul; Arakawa, Hiroshi; Wang, Weidong; Takeda, Shunichi; Ishii, Yutaka; Takata, Minoru; Seki, Masayuki; Enomoto, Takemi</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Molecular Cell Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">1843</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1002-1012</span>CODEN:
                <span class="NLM_cas:coden">BBAMCO</span>;
        ISSN:<span class="NLM_cas:issn">0167-4889</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">RecQ family DNA helicases function in the maintenance of genome stability.  Mice deficient in RecQL5, one of five RecQ helicases, show a cancer predisposition phenotype, suggesting that RecQL5 plays a tumor suppressor role.  RecQL5 interacts with Rad51, a key factor in homologous recombination (HR), and displaces Rad51 from Rad51-single stranded DNA (ssDNA) filaments in vitro.  However, the precise roles of RecQL5 in the cell remain elusive.  Here, we present evidence suggesting that RecQL5 is involved in DNA interstrand crosslink (ICL) repair.  Chicken DT40 RECQL5 gene knockout (KO) cells showed sensitivity to ICL-inducing agents such as cisplatin (CDDP) and mitomycin C (MMC) and a higher no. of chromosome aberrations in the presence of MMC than wild-type cells.  The phenotypes of RECQL5 KO cells resembled those of Fanconi anemia gene KO cells.  Genetic anal. using corresponding gene knockout cells showed that RecQL5 is involved in the FANCD1 (BRCA2)-dependent ICL repair pathway in which Rad51-ssDNA filament formation is promoted by BRCA2.  The disappearance but not appearance of Rad51-foci was delayed in RECQL5 KO cells after MMC treatment.  Deletion of Rad54, which processes the Rad51-ssDNA filament in HR, in RECQL5 KO cells increased sensitivity to CDDP and further delayed the disappearance of Rad51-foci, suggesting that RecQL5 and Rad54 have different effects on the Rad51-ssDNA filament.  Furthermore, the frequency and variation of CDDP-induced gene conversion at the Ig locus were increased in RECQL5 KO cells.  These results suggest that RecQL5 plays a role in regulating the incidence and quality of ICL-induced recombination.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpyc66YbYJ5zrVg90H21EOLACvtfcHk0ljSIj7SzlzNZw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjsF2gt7s%253D&md5=8833579d58e3977a80f479a925af7760</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.bbamcr.2014.01.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbamcr.2014.01.005%26sid%3Dliteratum%253Aachs%26aulast%3DHosono%26aufirst%3DY.%26aulast%3DAbe%26aufirst%3DT.%26aulast%3DIshiai%26aufirst%3DM.%26aulast%3DIslam%26aufirst%3DM.%2BN.%26aulast%3DArakawa%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DTakeda%26aufirst%3DS.%26aulast%3DIshii%26aufirst%3DY.%26aulast%3DTakata%26aufirst%3DM.%26aulast%3DSeki%26aufirst%3DM.%26aulast%3DEnomoto%26aufirst%3DT.%26atitle%3DTumor%2520suppressor%2520RecQL5%2520controls%2520recombination%2520induced%2520by%2520DNA%2520crosslinking%2520agents%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Mol.%2520Cell%2520Res.%26date%3D2014%26volume%3D1843%26spage%3D1002%26epage%3D1012%26doi%3D10.1016%2Fj.bbamcr.2014.01.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Røe, O. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szulkin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderssen, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flatberg, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sandeck, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amundsen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erlandsen, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dobra, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sundstrøm, S. H.</span></span> <span> </span><span class="NLM_article-title">Molecular Resistance Fingerprint of Pemetrexed and Platinum in a Long-Term Survivor of Mesothelioma</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>7</i></span>, <span class="NLM_elocation-id">e40521</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0040521</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=10.1371%2Fjournal.pone.0040521" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=22905093" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&author=O.+D.+R%C3%B8eauthor=A.+Szulkinauthor=E.+Anderssenauthor=A.+Flatbergauthor=H.+Sandeckauthor=T.+Amundsenauthor=S.+E.+Erlandsenauthor=K.+Dobraauthor=S.+H.+Sundstr%C3%B8m&title=Molecular+Resistance+Fingerprint+of+Pemetrexed+and+Platinum+in+a+Long-Term+Survivor+of+Mesothelioma&doi=10.1371%2Fjournal.pone.0040521"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0040521&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0040521%26sid%3Dliteratum%253Aachs%26aulast%3DR%25C3%25B8e%26aufirst%3DO.%2BD.%26aulast%3DSzulkin%26aufirst%3DA.%26aulast%3DAnderssen%26aufirst%3DE.%26aulast%3DFlatberg%26aufirst%3DA.%26aulast%3DSandeck%26aufirst%3DH.%26aulast%3DAmundsen%26aufirst%3DT.%26aulast%3DErlandsen%26aufirst%3DS.%2BE.%26aulast%3DDobra%26aufirst%3DK.%26aulast%3DSundstr%25C3%25B8m%26aufirst%3DS.%2BH.%26atitle%3DMolecular%2520Resistance%2520Fingerprint%2520of%2520Pemetrexed%2520and%2520Platinum%2520in%2520a%2520Long-Term%2520Survivor%2520of%2520Mesothelioma%26jtitle%3DPLoS%2520One%26date%3D2012%26volume%3D7%26doi%3D10.1371%2Fjournal.pone.0040521" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Györffy, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lanczky, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eklund, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denkert, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Budczies, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szallasi, Z.</span></span> <span> </span><span class="NLM_article-title">An Online Survival Analysis Tool to Rapidly Assess the Effect of 22,277 Genes on Breast Cancer Prognosis Using Microarray Data of 1,809 Patients</span>. <i>Breast Cancer Res. Treat.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>123</i></span>,  <span class="NLM_fpage">725</span>– <span class="NLM_lpage">731</span>, <span class="refDoi"> DOI: 10.1007/s10549-009-0674-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=10.1007%2Fs10549-009-0674-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=20020197" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=1%3ACAS%3A280%3ADC%252BC3cfit1Sksw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=123&publication_year=2010&pages=725-731&author=B.+Gy%C3%B6rffyauthor=A.+Lanczkyauthor=A.+C.+Eklundauthor=C.+Denkertauthor=J.+Budcziesauthor=Q.+Liauthor=Z.+Szallasi&title=An+Online+Survival+Analysis+Tool+to+Rapidly+Assess+the+Effect+of+22%2C277+Genes+on+Breast+Cancer+Prognosis+Using+Microarray+Data+of+1%2C809+Patients&doi=10.1007%2Fs10549-009-0674-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">An online survival analysis tool to rapidly assess the effect of 22,277 genes on breast cancer prognosis using microarray data of 1,809 patients</span></div><div class="casAuthors">Gyorffy Balazs; Lanczky Andras; Eklund Aron C; Denkert Carsten; Budczies Jan; Li Qiyuan; Szallasi Zoltan</div><div class="citationInfo"><span class="NLM_cas:title">Breast cancer research and treatment</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">123</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">725-31</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Validating prognostic or predictive candidate genes in appropriately powered breast cancer cohorts are of utmost interest.  Our aim was to develop an online tool to draw survival plots, which can be used to assess the relevance of the expression levels of various genes on the clinical outcome both in untreated and treated breast cancer patients.  A background database was established using gene expression data and survival information of 1,809 patients downloaded from GEO (Affymetrix HGU133A and HGU133+2 microarrays).  The median relapse free survival is 6.43 years, 968/1,231 patients are estrogen-receptor (ER) positive, and 190/1,369 are lymph-node positive.  After quality control and normalization only probes present on both Affymetrix platforms were retained (n = 22,277).  In order to analyze the prognostic value of a particular gene, the cohorts are divided into two groups according to the median (or upper/lower quartile) expression of the gene.  The two groups can be compared in terms of relapse free survival, overall survival, and distant metastasis free survival.  A survival curve is displayed, and the hazard ratio with 95% confidence intervals and logrank P value are calculated and displayed.  Additionally, three subgroups of patients can be assessed: systematically untreated patients, endocrine-treated ER positive patients, and patients with a distribution of clinical characteristics representative of those seen in general clinical practice in the US.  Web address: www.kmplot.com .  We used this integrative data analysis tool to confirm the prognostic power of the proliferation-related genes TOP2A and TOP2B, MKI67, CCND2, CCND3, CCNDE2, as well as CDKN1A, and TK2.  We also validated the capability of microarrays to determine estrogen receptor status in 1,231 patients.  The tool is highly valuable for the preliminary assessment of biomarkers, especially for research groups with limited bioinformatic resources.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRFYBCvpM8ROty3lHz94U-GfW6udTcc2eafJck90Aov8Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3cfit1Sksw%253D%253D&md5=0f2317107ba5739e3fa48aba9b965eab</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1007%2Fs10549-009-0674-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10549-009-0674-9%26sid%3Dliteratum%253Aachs%26aulast%3DGy%25C3%25B6rffy%26aufirst%3DB.%26aulast%3DLanczky%26aufirst%3DA.%26aulast%3DEklund%26aufirst%3DA.%2BC.%26aulast%3DDenkert%26aufirst%3DC.%26aulast%3DBudczies%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DQ.%26aulast%3DSzallasi%26aufirst%3DZ.%26atitle%3DAn%2520Online%2520Survival%2520Analysis%2520Tool%2520to%2520Rapidly%2520Assess%2520the%2520Effect%2520of%252022%252C277%2520Genes%2520on%2520Breast%2520Cancer%2520Prognosis%2520Using%2520Microarray%2520Data%2520of%25201%252C809%2520Patients%26jtitle%3DBreast%2520Cancer%2520Res.%2520Treat.%26date%3D2010%26volume%3D123%26spage%3D725%26epage%3D731%26doi%3D10.1007%2Fs10549-009-0674-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mgimpatsang, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Konstantinidou, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shishkina, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dömling, A.</span></span> <span> </span><span class="NLM_article-title">1,3,4-Oxadiazoles by Ugi-Tetrazole and Huisgen Reaction</span>. <i>Org. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">7320</span>– <span class="NLM_lpage">7323</span>, <span class="refDoi"> DOI: 10.1021/acs.orglett.9b02614</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.orglett.9b02614" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhs1yntrzE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2019&pages=7320-7323&author=Q.+Wangauthor=K.+C.+Mgimpatsangauthor=M.+Konstantinidouauthor=S.+V.+Shishkinaauthor=A.+D%C3%B6mling&title=1%2C3%2C4-Oxadiazoles+by+Ugi-Tetrazole+and+Huisgen+Reaction&doi=10.1021%2Facs.orglett.9b02614"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">1,3,4-Oxadiazoles by Ugi-Tetrazole and Huisgen Reaction</span></div><div class="casAuthors">Wang, Qian; Mgimpatsang, Kumchok C.; Konstantinidou, Markella; Shishkina, Svitlana V.; Doemling, Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Organic Letters</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">7320-7323</span>CODEN:
                <span class="NLM_cas:coden">ORLEF7</span>;
        ISSN:<span class="NLM_cas:issn">1523-7052</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Amines, aldehydes, Me3SiN3, and tert-octyl isonitrile t-BuCH2CMe2N+≡C- underwent Ugi reactions to yield aminoalkyltetrazoles such as I; acid cleavage of the tert-octyl group followed by Huisgen reactions with acyl chlorides in pyridine yielded oxadiazoles such as II.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp4GnJ4E_VO3LVg90H21EOLACvtfcHk0ljzrNtTGCrt0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhs1yntrzE&md5=a4ddb092f5245191e3bc5084ee1ac990</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Facs.orglett.9b02614&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.orglett.9b02614%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DQ.%26aulast%3DMgimpatsang%26aufirst%3DK.%2BC.%26aulast%3DKonstantinidou%26aufirst%3DM.%26aulast%3DShishkina%26aufirst%3DS.%2BV.%26aulast%3DD%25C3%25B6mling%26aufirst%3DA.%26atitle%3D1%252C3%252C4-Oxadiazoles%2520by%2520Ugi-Tetrazole%2520and%2520Huisgen%2520Reaction%26jtitle%3DOrg.%2520Lett.%26date%3D2019%26volume%3D21%26spage%3D7320%26epage%3D7323%26doi%3D10.1021%2Facs.orglett.9b02614" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Newman, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aitkenhead, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savitsky, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gileadi, O.</span></span> <span> </span><span class="NLM_article-title">Insights into the RecQ Helicase Mechanism Revealed by the Structure of the Helicase Domain of Human RECQL5</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">4231</span>– <span class="NLM_lpage">4243</span>, <span class="refDoi"> DOI: 10.1093/nar/gkw1362</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=10.1093%2Fnar%2Fgkw1362" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=28100692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsVyqsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2017&pages=4231-4243&author=J.+A.+Newmanauthor=H.+Aitkenheadauthor=P.+Savitskyauthor=O.+Gileadi&title=Insights+into+the+RecQ+Helicase+Mechanism+Revealed+by+the+Structure+of+the+Helicase+Domain+of+Human+RECQL5&doi=10.1093%2Fnar%2Fgkw1362"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Insights into the RecQ helicase mechanism revealed by the structure of the helicase domain of human RECQL5</span></div><div class="casAuthors">Newman, Joseph A.; Aitkenhead, Hazel; Savitsky, Pavel; Gileadi, Opher</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">4231-4243</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">1362-4962</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">RecQ helicases are important maintainers of genome integrity with distinct roles in almost every cellular process requiring access to DNA.  RECQL5 is one of five human RecQ proteins and is particularly versatile in this regard, forming protein complexes with a diverse set of cellular partners in order to coordinate its helicase activity to various processes including replication, recombination and DNA repair.  In this study, we have detd. crystal structures of the core helicase domain of RECQL5 both with and without the nucleotide ADP in two distinctly different ('Open' and 'Closed') conformations.  Small angle X-ray scattering studies show that the 'Open' form of the protein predominates in soln. and we discuss implications of this with regards to the RECQL5 mechanism and conformational changes.  We have measured the ATPase, helicase and DNA binding properties of various RECQL5 constructs and variants and discuss the role of these regions and residues in the various RECQL5 activities.  Finally, we have performed a systematic comparison of the RECQL5 structures with other RecQ family structures and based on these comparisons we have constructed a model for the mechano-chem. cycle of the common catalytic core of these helicases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGokJCPaZR5xoLVg90H21EOLACvtfcHk0ljzrNtTGCrt0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsVyqsrY%253D&md5=d2480e40ee405d59871a039759b06698</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkw1362&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkw1362%26sid%3Dliteratum%253Aachs%26aulast%3DNewman%26aufirst%3DJ.%2BA.%26aulast%3DAitkenhead%26aufirst%3DH.%26aulast%3DSavitsky%26aufirst%3DP.%26aulast%3DGileadi%26aufirst%3DO.%26atitle%3DInsights%2520into%2520the%2520RecQ%2520Helicase%2520Mechanism%2520Revealed%2520by%2520the%2520Structure%2520of%2520the%2520Helicase%2520Domain%2520of%2520Human%2520RECQL5%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2017%26volume%3D45%26spage%3D4231%26epage%3D4243%26doi%3D10.1093%2Fnar%2Fgkw1362" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gray, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, J.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamath-Loeb, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blank, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sopher, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oshima, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loeb, L. A.</span></span> <span> </span><span class="NLM_article-title">The Werner syndrome protein is a DNA helicase</span>. <i>Nat. Genet.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">100</span>– <span class="NLM_lpage">103</span>, <span class="refDoi"> DOI: 10.1038/ng0997-100</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=10.1038%2Fng0997-100" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=9288107" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=1%3ACAS%3A528%3ADyaK2sXlvVGgsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=1997&pages=100-103&author=M.+D.+Grayauthor=J.-C.+Shenauthor=A.+S.+Kamath-Loebauthor=A.+Blankauthor=B.+L.+Sopherauthor=G.+M.+Martinauthor=J.+Oshimaauthor=L.+A.+Loeb&title=The+Werner+syndrome+protein+is+a+DNA+helicase&doi=10.1038%2Fng0997-100"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">The Werner syndrome protein is a DNA helicase</span></div><div class="casAuthors">Gray, Matthew D.; Shen, Jiang-Cheng; Kamath-Loeb, Ashwini S.; Blank, A.; Sopher, Bryce L.; Martin, George M.; Oshima, Junko; Loeb, Lawrence A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">100-103</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature America</span>)
        </div><div class="casAbstract">Werner syndrome (WS) is an uncommon autosomal recessive disorder characterized by premature aging.  The clin. manifestations of WS, including atherosclerosis and osteoporosis, appear early in adulthood, and death in the fourth to sixth decade commonly ensues from myocardial infarction or cancer.  In accord with the aging phenotype, cells from WS patients have a reduced replicative life span in culture.  Genomic instability is obsd. at the cytogenetic level in the form of chromosome breaks and translocations and at the mol. level by multiple large deletions.  The Werner syndrome gene (WRN) has recently been cloned.  The predicted product is a 1,432-amino-acid protein whose central domain is homologous to members of the RecQ family of DNA helicases.  Such homol. does not necessarily mean that WRN encodes an active helicase.  For example, the Saccharomyces cerevisiae RAD26 gene protein and the human transcription-repair coupling factor CSB (Cockayne syndrome B) are highly homologous to known helicases, yet neither encodes an active helicase.  Moreover,the Bloom's syndrome gene (BLM), discovered before WRN, is also homologous to the RecQ family of DNA helicases, though we still await demonstration that it encodes an active helicase.  Here we report that the WS protein does indeed catalyze DNA unwinding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoNMwhnmtR8FrVg90H21EOLACvtfcHk0lj9q4ne91VXUg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXlvVGgsLo%253D&md5=f96ec899a353b498845fb6528e9eeb9c</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1038%2Fng0997-100&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fng0997-100%26sid%3Dliteratum%253Aachs%26aulast%3DGray%26aufirst%3DM.%2BD.%26aulast%3DShen%26aufirst%3DJ.-C.%26aulast%3DKamath-Loeb%26aufirst%3DA.%2BS.%26aulast%3DBlank%26aufirst%3DA.%26aulast%3DSopher%26aufirst%3DB.%2BL.%26aulast%3DMartin%26aufirst%3DG.%2BM.%26aulast%3DOshima%26aufirst%3DJ.%26aulast%3DLoeb%26aufirst%3DL.%2BA.%26atitle%3DThe%2520Werner%2520syndrome%2520protein%2520is%2520a%2520DNA%2520helicase%26jtitle%3DNat.%2520Genet.%26date%3D1997%26volume%3D17%26spage%3D100%26epage%3D103%26doi%3D10.1038%2Fng0997-100" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Garcia, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiricny, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">West, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janscak, P.</span></span> <span> </span><span class="NLM_article-title">Human RECQ5β, a protein with DNA helicase and strand-annealing activities in a single polypeptide</span>. <i>EMBO J.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">2882</span>– <span class="NLM_lpage">2891</span>, <span class="refDoi"> DOI: 10.1038/sj.emboj.7600301</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=10.1038%2Fsj.emboj.7600301" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=15241474" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=1%3ACAS%3A528%3ADC%252BD2cXlvValt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2004&pages=2882-2891&author=P.+L.+Garciaauthor=Y.+Liuauthor=J.+Jiricnyauthor=S.+C.+Westauthor=P.+Janscak&title=Human+RECQ5%CE%B2%2C+a+protein+with+DNA+helicase+and+strand-annealing+activities+in+a+single+polypeptide&doi=10.1038%2Fsj.emboj.7600301"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Human RECQ5β, a protein with DNA helicase and strand-annealing activities in a single polypeptide</span></div><div class="casAuthors">Garcia, Patrick L.; Liu, Yilun; Jiricny, Josef; West, Stephen C.; Janscak, Pavel</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Journal</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">2882-2891</span>CODEN:
                <span class="NLM_cas:coden">EMJODG</span>;
        ISSN:<span class="NLM_cas:issn">0261-4189</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Proteins belonging to the highly conserved RecQ helicase family are essential for the maintenance of genomic stability.  Here, we describe the biochem. properties of the human RECQ5β protein.  Like BLM and WRN, RECQ5β is an ATP-dependent 3'-5' DNA helicase that can promote migration of Holliday junctions.  However, RECQ5β required the single-stranded DNA-binding protein RPA in order to mediate the efficient unwinding of oligonucleotide-based substrates.  Surprisingly, we found that RECQ5β possesses an intrinsic DNA strand-annealing activity that is inhibited by RPA.  Anal. of deletion variants of RECQ5β revealed that the DNA helicase activity resides in the conserved N-terminal portion of the protein, whereas strand annealing is mediated by the unique C-terminal domain.  Moreover, the strand-annealing activity of RECQ5β was strongly inhibited by ATPγS, a poorly hydrolyzable analog of ATP.  This effect was alleviated by mutations in the ATP-binding motif of RECQ5β, indicating that the ATP-bound form of the protein cannot promote strand annealing.  This is the first demonstration of a DNA helicase with an intrinsic DNA strand-annealing function residing in a sep. domain.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpE9WNEn1Qrk7Vg90H21EOLACvtfcHk0lj9q4ne91VXUg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXlvValt7k%253D&md5=01afeb2481055108192a40602ce44093</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Fsj.emboj.7600301&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.emboj.7600301%26sid%3Dliteratum%253Aachs%26aulast%3DGarcia%26aufirst%3DP.%2BL.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DJiricny%26aufirst%3DJ.%26aulast%3DWest%26aufirst%3DS.%2BC.%26aulast%3DJanscak%26aufirst%3DP.%26atitle%3DHuman%2520RECQ5%25CE%25B2%252C%2520a%2520protein%2520with%2520DNA%2520helicase%2520and%2520strand-annealing%2520activities%2520in%2520a%2520single%2520polypeptide%26jtitle%3DEMBO%2520J.%26date%3D2004%26volume%3D23%26spage%3D2882%26epage%3D2891%26doi%3D10.1038%2Fsj.emboj.7600301" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Aggarwal, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sommers, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoemaker, R. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brosh, R. M.</span></span> <span> </span><span class="NLM_article-title">Inhibition of Helicase Activity by a Small Molecule Impairs Werner Syndrome Helicase (WRN) Function in the Cellular Response to DNA Damage or Replication Stress</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>108</i></span>,  <span class="NLM_fpage">1525</span>– <span class="NLM_lpage">1530</span>, <span class="refDoi"> DOI: 10.1073/pnas.1006423108</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=10.1073%2Fpnas.1006423108" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=21220316" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs1SmtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=108&publication_year=2011&pages=1525-1530&author=M.+Aggarwalauthor=J.+A.+Sommersauthor=R.+H.+Shoemakerauthor=R.+M.+Brosh&title=Inhibition+of+Helicase+Activity+by+a+Small+Molecule+Impairs+Werner+Syndrome+Helicase+%28WRN%29+Function+in+the+Cellular+Response+to+DNA+Damage+or+Replication+Stress&doi=10.1073%2Fpnas.1006423108"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of helicase activity by a small molecule impairs Werner syndrome helicase (WRN) function in the cellular response to DNA damage or replication stress</span></div><div class="casAuthors">Aggarwal, Monika; Sommers, Joshua A.; Shoemaker, Robert H.; Brosh, Robert M., Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">108</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1525-1530, S1525/1-S1525/9</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Modulation of DNA repair proteins by small mols. has attracted great interest.  An in vitro helicase activity screen was used to identify mols. that modulate DNA unwinding by Werner syndrome helicase (WRN), mutated in the premature aging disorder Werner syndrome.  A small mol. from the National Cancer Institute Diversity Set designated NSC 19630 [1-(propoxymethyl)-maleimide] was identified that inhibited WRN helicase activity but did not affect other DNA helicases [Bloom syndrome (BLM), Fanconi anemia group J (FANCJ), RECQ1, RecQ, UvrD, or DnaB].  Exposure of human cells to NSC 19630 dramatically impaired growth and proliferation, induced apoptosis in a WRN-dependent manner, and resulted in elevated γ-H2AX and proliferating cell nuclear antigen (PCNA) foci.  NSC 19630 exposure led to delayed S-phase progression, consistent with the accumulation of stalled replication forks, and to DNA damage in a WRN-dependent manner.  Exposure to NSC 19630 sensitized cancer cells to the G-quadruplex-binding compd. telomestatin or a poly(ADP ribose) polymerase (PARP) inhibitor.  Sublethal dosage of NSC 19630 and the chemotherapy drug topotecan acted synergistically to inhibit cell proliferation and induce DNA damage.  The use of this WRN helicase inhibitor mol. may provide insight into the importance of WRN-mediated pathway(s) important for DNA repair and the replicational stress response.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqRIkmkwi9yCrVg90H21EOLACvtfcHk0ljk3LjNN3ednA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhs1SmtL8%253D&md5=d01ee377da5c65cdec72b0403494fa0e</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1006423108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1006423108%26sid%3Dliteratum%253Aachs%26aulast%3DAggarwal%26aufirst%3DM.%26aulast%3DSommers%26aufirst%3DJ.%2BA.%26aulast%3DShoemaker%26aufirst%3DR.%2BH.%26aulast%3DBrosh%26aufirst%3DR.%2BM.%26atitle%3DInhibition%2520of%2520Helicase%2520Activity%2520by%2520a%2520Small%2520Molecule%2520Impairs%2520Werner%2520Syndrome%2520Helicase%2520%2528WRN%2529%2520Function%2520in%2520the%2520Cellular%2520Response%2520to%2520DNA%2520Damage%2520or%2520Replication%2520Stress%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2011%26volume%3D108%26spage%3D1525%26epage%3D1530%26doi%3D10.1073%2Fpnas.1006423108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tiwari, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gopalan Kutty, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaudhary, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vasanth Raj, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shenoy, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mallikarjuna Rao, C.</span></span> <span> </span><span class="NLM_article-title">Synthesis and Evaluation of Selected 1,3,4-Oxadiazole Derivatives for in Vitro Cytotoxicity and in Vivo Anti-Tumour Activity</span>. <i>Cytotechnology</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>68</i></span>,  <span class="NLM_fpage">2553</span>– <span class="NLM_lpage">2565</span>, <span class="refDoi"> DOI: 10.1007/s10616-016-9979-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=10.1007%2Fs10616-016-9979-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=27282155" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=1%3ACAS%3A528%3ADC%252BC28XpslWmtrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2016&pages=2553-2565&author=A.+Tiwariauthor=N.+Gopalan+Kuttyauthor=N.+Kumarauthor=A.+Chaudharyauthor=P.+Vasanth+Rajauthor=R.+Shenoyauthor=C.+Mallikarjuna+Rao&title=Synthesis+and+Evaluation+of+Selected+1%2C3%2C4-Oxadiazole+Derivatives+for+in+Vitro+Cytotoxicity+and+in+Vivo+Anti-Tumour+Activity&doi=10.1007%2Fs10616-016-9979-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and evaluation of selected 1,3,4-oxadiazole derivatives for in vitro cytotoxicity and in vivo anti-tumor activity</span></div><div class="casAuthors">Tiwari, Amit; Gopalan Kutty, N.; Kumar, Nitesh; Chaudhary, Anil; Vasanth Raj, P.; Shenoy, Rekha; Mallikarjuna Rao, C.</div><div class="citationInfo"><span class="NLM_cas:title">Cytotechnology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">68</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2553-2565</span>CODEN:
                <span class="NLM_cas:coden">CYTOER</span>;
        ISSN:<span class="NLM_cas:issn">0920-9069</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">The oxadiazole moiety is known for its anticancer activity through its antiangiogenic and mitostatic potential.  Taking this as a cue, the present study was designed to investigate the anti-cancer potential of selected oxadiazole derivs.  Twelve 1,3,4-oxadiazole derivs. (AMK OX-1 to AMK OX-12) were synthesized and were tested for IC50 values through brine shrimp lethality assay and MTT assay on HeLa and A549 cell lines.  Four compds., AMK OX-8, 9, 11 and 12 showed potential cytotoxicity activity with low IC50 value.  These compds. produced considerable cytotoxic effect on Hep-2 and A549 cancer cell lines.  However, they were found to be comparatively safer to normal cell lines, viz., V-79 cell lines than to the tested cancer cell lines, such as HeLa, A 549, and Hep2 cell lines.  The mechanism of cytotoxicity was evaluated through nuclear staining and DNA ladder assay.  Although DNA ladder assay showed DNA fragmentation (apoptotic phenomenon) in Hep-2 cells treated with only AMK OX-12, the staining procedures using acridine orange, ethidium bromide and propidium iodide showed apoptotic bodies in cells treated with AMK OX-8, 9 and 12 also.  In JCI staining on isolated mitochondria of Hep2 cells, AMK OX-8, 9-11 and 12 displayed increasing fluorescence intensity with time which confirmed involvement of mitochondrial pathway and intrinsic pathway of apoptosis.  All four compds. were found to be safe in acute oral toxicity study in Swiss albino mice.  These derivs. were effective in reducing tumor size and wt. in the in vivo DLA-induced solid tumor model.  They were found to be significantly effective in reducing tumor vol. and tumor wt.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGof2fUc8OvrebVg90H21EOLACvtfcHk0ljk3LjNN3ednA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XpslWmtrs%253D&md5=9b76d230f99b5ec927a56c5c49c52112</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1007%2Fs10616-016-9979-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10616-016-9979-9%26sid%3Dliteratum%253Aachs%26aulast%3DTiwari%26aufirst%3DA.%26aulast%3DGopalan%2BKutty%26aufirst%3DN.%26aulast%3DKumar%26aufirst%3DN.%26aulast%3DChaudhary%26aufirst%3DA.%26aulast%3DVasanth%2BRaj%26aufirst%3DP.%26aulast%3DShenoy%26aufirst%3DR.%26aulast%3DMallikarjuna%2BRao%26aufirst%3DC.%26atitle%3DSynthesis%2520and%2520Evaluation%2520of%2520Selected%25201%252C3%252C4-Oxadiazole%2520Derivatives%2520for%2520in%2520Vitro%2520Cytotoxicity%2520and%2520in%2520Vivo%2520Anti-Tumour%2520Activity%26jtitle%3DCytotechnology%26date%3D2016%26volume%3D68%26spage%3D2553%26epage%3D2565%26doi%3D10.1007%2Fs10616-016-9979-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hekal, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">El-Naggar, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abu El-Azm, F. S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">El-Sayed, W. M.</span></span> <span> </span><span class="NLM_article-title">Synthesis of New Oxadiazol-Phthalazinone Derivatives with Anti-Proliferative Activity; Molecular Docking, pro-Apoptotic, and Enzyme Inhibition Profile</span>. <i>RSC Adv.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">3675</span>– <span class="NLM_lpage">3688</span>, <span class="refDoi"> DOI: 10.1039/C9RA09016A</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=10.1039%2FC9RA09016A" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhsFWjsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2020&pages=3675-3688&author=M.+H.+Hekalauthor=A.+M.+El-Naggarauthor=F.+S.+M.+Abu+El-Azmauthor=W.+M.+El-Sayed&title=Synthesis+of+New+Oxadiazol-Phthalazinone+Derivatives+with+Anti-Proliferative+Activity%3B+Molecular+Docking%2C+pro-Apoptotic%2C+and+Enzyme+Inhibition+Profile&doi=10.1039%2FC9RA09016A"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis of new oxadiazol-phthalazinone derivatives with anti-proliferative activity; molecular docking, pro-apoptotic, and enzyme inhibition profile</span></div><div class="casAuthors">Hekal, Mohamed H.; El-Naggar, Abeer M.; Abu El-Azm, Fatma S. M.; El-Sayed, Wael M.</div><div class="citationInfo"><span class="NLM_cas:title">RSC Advances</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3675-3688</span>CODEN:
                <span class="NLM_cas:coden">RSCACL</span>;
        ISSN:<span class="NLM_cas:issn">2046-2069</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Background and aim: The current study reports the synthesis and biol. evaluation of two novel series of 4-(5-mercapto-1,3,4-oxadiazol-2-yl)phthalazin-1(2H)-one derivs.  Methods: The synthetic reactions were carried out under both conventional and ultrasonic irradn. conditions.  The anti-proliferative activity of the newly synthesized compds. against two human epithelial cell lines; liver (HepG2) and breast (MCF-7) in addn. to normal fibroblasts (WI-38) was investigated.  In addn. to mol. docking studies, the possible mechanism(s) of action were also explored.  Results: In general, an improvement in synthetic rates and yields was obsd. when reactions were carried out under sonication compared with classical conditions.  The structures of the products were established based on anal. and spectral data.  These derivs. arrested the cell cycle progression and/or induced apoptosis.  This has been manifested by the elevation in the expression of p53 and caspase 3, down-regulation of cdk1, and a redn. in the concns. of MAPK and Topo II at submicromolar concns.  The latter results confirmed the mol. docking study.  Conclusions: Compd. 1 had the best profile on the gene and protein levels (arresting cell cycle and inducing apoptosis).  The ability of compds. 1 and 2e to inhibit both MAPK and Topo II nominates these derivs. as potential candidates for further anticancer and antitumor studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPx4mdMUtvR7Vg90H21EOLACvtfcHk0li7AhXMkw_rPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhsFWjsb8%253D&md5=d24537ae457e0d631561a4bfe2f49c70</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1039%2FC9RA09016A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC9RA09016A%26sid%3Dliteratum%253Aachs%26aulast%3DHekal%26aufirst%3DM.%2BH.%26aulast%3DEl-Naggar%26aufirst%3DA.%2BM.%26aulast%3DAbu%2BEl-Azm%26aufirst%3DF.%2BS.%2BM.%26aulast%3DEl-Sayed%26aufirst%3DW.%2BM.%26atitle%3DSynthesis%2520of%2520New%2520Oxadiazol-Phthalazinone%2520Derivatives%2520with%2520Anti-Proliferative%2520Activity%253B%2520Molecular%2520Docking%252C%2520pro-Apoptotic%252C%2520and%2520Enzyme%2520Inhibition%2520Profile%26jtitle%3DRSC%2520Adv.%26date%3D2020%26volume%3D10%26spage%3D3675%26epage%3D3688%26doi%3D10.1039%2FC9RA09016A" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barnes, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lou, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, G.</span></span> <span> </span><span class="NLM_article-title">Recql5 and Blm RecQ DNA Helicases Have Nonredundant Roles in Suppressing Crossovers</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">3431</span>– <span class="NLM_lpage">3442</span>, <span class="refDoi"> DOI: 10.1128/MCB.25.9.3431-3442.2005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=10.1128%2FMCB.25.9.3431-3442.2005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=15831450" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=1%3ACAS%3A528%3ADC%252BD2MXjvFSmsb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2005&pages=3431-3442&author=Y.+Huauthor=X.+Luauthor=E.+Barnesauthor=M.+Yanauthor=H.+Louauthor=G.+Luo&title=Recql5+and+Blm+RecQ+DNA+Helicases+Have+Nonredundant+Roles+in+Suppressing+Crossovers&doi=10.1128%2FMCB.25.9.3431-3442.2005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Recql5 and Blm RecQ DNA helicases have nonredundant roles in suppressing crossovers</span></div><div class="casAuthors">Hu, Yiduo; Lu, Xincheng; Barnes, Ellen; Yan, Min; Lou, Hua; Luo, Guangbin</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3431-3442</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">In eukaryotes, crossovers in mitotic cells can have deleterious consequences and therefore must be suppressed.  Mutations in BLM give rise to Bloom syndrome, a disease that is characterized by an elevated rate of crossovers and increased cancer susceptibility.  However, simple eukaryotes such as Saccharomyces cerevisiae have multiple pathways for suppressing crossovers, suggesting that mammals also have multiple pathways for controlling crossovers in their mitotic cells.  We show here that in mouse embryonic stem (ES) cells, mutations in either the Bloom syndrome homolog (Blm) or the Recql5 genes result in a significant increase in the frequency of sister chromatid exchange (SCE), whereas deleting both Blm and Recql5 lead to an even higher frequency of SCE.  These data indicate that Blm and Recql5 have nonredundant roles in suppressing crossovers in mouse ES cells.  Furthermore, we show that mouse embryonic fibroblasts derived from Recql5 knockout mice also exhibit a significantly increased frequency of SCE compared with the corresponding wild-type control.  Thus, this study identifies a previously unknown Recql5-dependent, Blm-independent pathway for suppressing crossovers during mitosis in mice.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq3DOZyXmRYxbVg90H21EOLACvtfcHk0li7AhXMkw_rPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXjvFSmsb0%253D&md5=e80d835a8ad2cb3527757705a24f8681</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1128%2FMCB.25.9.3431-3442.2005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.25.9.3431-3442.2005%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DY.%26aulast%3DLu%26aufirst%3DX.%26aulast%3DBarnes%26aufirst%3DE.%26aulast%3DYan%26aufirst%3DM.%26aulast%3DLou%26aufirst%3DH.%26aulast%3DLuo%26aufirst%3DG.%26atitle%3DRecql5%2520and%2520Blm%2520RecQ%2520DNA%2520Helicases%2520Have%2520Nonredundant%2520Roles%2520in%2520Suppressing%2520Crossovers%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2005%26volume%3D25%26spage%3D3431%26epage%3D3442%26doi%3D10.1128%2FMCB.25.9.3431-3442.2005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raynard, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sehorn, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bussen, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stark, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barnes, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janscak, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jasin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vogel, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sung, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, G.</span></span> <span> </span><span class="NLM_article-title">RECQL5/Recql5 Helicase Regulates Homologous Recombination and Suppresses Tumour Formation via Disruption of Rad51 Presynaptic Filaments</span>. <i>Genes Dev.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">3073</span>– <span class="NLM_lpage">3084</span>, <span class="refDoi"> DOI: 10.1101/gad.1609107</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=10.1101%2Fgad.1609107" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=18003859" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhsVaht7vO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2007&pages=3073-3084&author=Y.+Huauthor=S.+Raynardauthor=M.+G.+Sehornauthor=X.+Luauthor=W.+Bussenauthor=L.+Zhengauthor=J.+M.+Starkauthor=E.+L.+Barnesauthor=P.+Chiauthor=P.+Janscakauthor=M.+Jasinauthor=H.+Vogelauthor=P.+Sungauthor=G.+Luo&title=RECQL5%2FRecql5+Helicase+Regulates+Homologous+Recombination+and+Suppresses+Tumour+Formation+via+Disruption+of+Rad51+Presynaptic+Filaments&doi=10.1101%2Fgad.1609107"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">RECQL5/Recq15 helicase regulates homologous recombination and suppresses tumor formation via disruption of Rad51 presynaptic filaments</span></div><div class="casAuthors">Hu, Yiduo; Raynard, Steven; Sehorn, Michael G.; Lu, Xincheng; Bussen, Wendy; Zheng, Lu; Stark, Jeremy M.; Barnes, Ellen L.; Chi, Peter; Janscak, Pavel; Jasin, Maria; Vogel, Hannes; Sung, Patrick; Luo, Guangbin</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">3073-3084</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">Members of the RecQ helicase family play crit. roles in genome maintenance.  There are five RecQ homologs in mammals, and defects in three of these (BLM, WRN, and RECQL4) give rise to cancer predisposition syndromes in humans.  RECQL and RECQL5 have not been assocd. with a human disease.  Here we show that deletion of Recq15 in mice results in cancer susceptibility.  Racq15-deficient cells exhibit elevated frequencies of spontaneous DNA double-strand breaks and homologous recombination (HR) as scored using a reporter that harbors a direct repeat, and are prone to gross chromosomal rearrangements in response to replication stress.  To understand how RECQL5 regulates HR, we use purified proteins to demonstrate that human RECQL5 binds the Rad51 recombinase and inhibits Rad51-mediated D-loop formation.  By biochem. means and electron microscopy, we show that RECQL5 displaces Rad51 from single-stranded DNA (ssDNA) in a reaction that requires ATP hydrolysis and RPA.  Together, our results identify RECQL5 as an important tumor suppressor that may act by preventing inappropriate HR events via Rad51 presynaptic filament disruption.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqKBA40NBcFrVg90H21EOLACvtfcHk0ljg_0C3YAYwlg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhsVaht7vO&md5=85ddf3085111874a24beca1b9c4a91ed</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1101%2Fgad.1609107&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.1609107%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DY.%26aulast%3DRaynard%26aufirst%3DS.%26aulast%3DSehorn%26aufirst%3DM.%2BG.%26aulast%3DLu%26aufirst%3DX.%26aulast%3DBussen%26aufirst%3DW.%26aulast%3DZheng%26aufirst%3DL.%26aulast%3DStark%26aufirst%3DJ.%2BM.%26aulast%3DBarnes%26aufirst%3DE.%2BL.%26aulast%3DChi%26aufirst%3DP.%26aulast%3DJanscak%26aufirst%3DP.%26aulast%3DJasin%26aufirst%3DM.%26aulast%3DVogel%26aufirst%3DH.%26aulast%3DSung%26aufirst%3DP.%26aulast%3DLuo%26aufirst%3DG.%26atitle%3DRECQL5%252FRecql5%2520Helicase%2520Regulates%2520Homologous%2520Recombination%2520and%2520Suppresses%2520Tumour%2520Formation%2520via%2520Disruption%2520of%2520Rad51%2520Presynaptic%2520Filaments%26jtitle%3DGenes%2520Dev.%26date%3D2007%26volume%3D21%26spage%3D3073%26epage%3D3084%26doi%3D10.1101%2Fgad.1609107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Paliwal, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kanagaraj, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sturzenegger, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burdova, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janscak, P.</span></span> <span> </span><span class="NLM_article-title">Human RECQ5 Helicase Promotes Repair of DNA Double-Strand Breaks by Synthesis-Dependent Strand Annealing</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">2380</span>– <span class="NLM_lpage">2390</span>, <span class="refDoi"> DOI: 10.1093/nar/gkt1263</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=10.1093%2Fnar%2Fgkt1263" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=24319145" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjtlOrur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2014&pages=2380-2390&author=S.+Paliwalauthor=R.+Kanagarajauthor=A.+Sturzeneggerauthor=K.+Burdovaauthor=P.+Janscak&title=Human+RECQ5+Helicase+Promotes+Repair+of+DNA+Double-Strand+Breaks+by+Synthesis-Dependent+Strand+Annealing&doi=10.1093%2Fnar%2Fgkt1263"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Human RECQ5 helicase promotes repair of DNA double-strand breaks by synthesis-dependent strand annealing</span></div><div class="casAuthors">Paliwal, Shreya; Kanagaraj, Radhakrishnan; Sturzenegger, Andreas; Burdova, Kamila; Janscak, Pavel</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">2380-2390</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Most mitotic homologous recombination (HR) events proceed via a synthesis-dependent strand annealing mechanism to avoid crossing over, which may give rise to chromosomal rearrangements and loss of heterozygosity.  The mol. mechanisms controlling HR sub-pathway choice are poorly understood.  Here, we show that human RECQ5, a DNA helicase that can disrupt RAD51 nucleoprotein filaments, promotes formation of non-crossover products during DNA double-strand break-induced HR and counteracts the inhibitory effect of RAD51 on RAD52-mediated DNA annealing in vitro and in vivo.  Moreover, we demonstrate that RECQ5 deficiency is assocd. with an increased occupancy of RAD51 at a double-strand break site, and it also causes an elevation of sister chromatid exchanges on inactivation of the Holliday junction dissoln. pathway or on induction of a high load of DNA damage in the cell.  Collectively, our findings suggest that RECQ5 acts during the post-synaptic phase of synthesis-dependent strand annealing to prevent formation of aberrant RAD51 filaments on the extended invading strand, thus limiting its channeling into potentially hazardous crossover pathway of HR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqRn3s7AR2epbVg90H21EOLACvtfcHk0ljg_0C3YAYwlg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjtlOrur0%253D&md5=92435bd89e4ae52eefdb97e8844b09fb</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkt1263&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkt1263%26sid%3Dliteratum%253Aachs%26aulast%3DPaliwal%26aufirst%3DS.%26aulast%3DKanagaraj%26aufirst%3DR.%26aulast%3DSturzenegger%26aufirst%3DA.%26aulast%3DBurdova%26aufirst%3DK.%26aulast%3DJanscak%26aufirst%3DP.%26atitle%3DHuman%2520RECQ5%2520Helicase%2520Promotes%2520Repair%2520of%2520DNA%2520Double-Strand%2520Breaks%2520by%2520Synthesis-Dependent%2520Strand%2520Annealing%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2014%26volume%3D42%26spage%3D2380%26epage%3D2390%26doi%3D10.1093%2Fnar%2Fgkt1263" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Islam, M. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paquet, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fox, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dray, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, X.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sung, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span> <span> </span><span class="NLM_article-title">A Variant of the Breast Cancer Type 2 Susceptibility Protein (BRC) Repeat Is Essential for the RECQL5 Helicase to Interact with RAD51 Recombinase for Genome Stabilization</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>287</i></span>,  <span class="NLM_fpage">23808</span>– <span class="NLM_lpage">23818</span>, <span class="refDoi"> DOI: 10.1074/jbc.M112.375014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=10.1074%2Fjbc.M112.375014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=22645136" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=1%3ACAS%3A528%3ADC%252BC38XpvVejurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=287&publication_year=2012&pages=23808-23818&author=M.+N.+Islamauthor=N.+Paquetauthor=D.+Foxauthor=E.+Drayauthor=X.-F.+Zhengauthor=H.+Kleinauthor=P.+Sungauthor=W.+Wang&title=A+Variant+of+the+Breast+Cancer+Type+2+Susceptibility+Protein+%28BRC%29+Repeat+Is+Essential+for+the+RECQL5+Helicase+to+Interact+with+RAD51+Recombinase+for+Genome+Stabilization&doi=10.1074%2Fjbc.M112.375014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">A variant of the breast cancer type 2 susceptibility protein (BRC) repeat is essential for the RECQL5 helicase to interact with RAD51 recombinase for genome stabilization</span></div><div class="casAuthors">Islam, M. Nurul; Paquet, Nicolas; Fox, David, III; Dray, Eloise; Zheng, Xiao-Feng; Klein, Hannah; Sung, Patrick; Wang, Weidong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">287</span>
        (<span class="NLM_cas:issue">28</span>),
    <span class="NLM_cas:pages">23808-23818</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The BRC repeat is a structural motif in the tumor suppressor BRCA2 (breast cancer type 2 susceptibility protein), which promotes homologous recombination (HR) by regulating RAD51 recombinase activity.  To date, the BRC repeat has not been obsd. in other proteins, so that its role in HR is inferred only in the context of BRCA2.  Here, we identified a BRC repeat variant, named BRCv, in the RECQL5 helicase, which possesses anti-recombinase activity in vitro and suppresses HR and promotes cellular resistance to camptothecin-induced replication stress in vivo.  RECQL5-BRCv interacted with RAD51 through two conserved motifs similar to those in the BRCA2-BRC repeat.  Mutations of either motif compromised functions of RECQL5, including assocn. with RAD51, inhibition of RAD51-mediated D-loop formation, suppression of sister chromatid exchange, and resistance to camptothecin-induced replication stress.  Potential BRCvs were also found in other HR regulatory proteins, including Srs2 and Sgs1, which possess anti-recombinase activities similar to that of RECQL5.  A point mutation in the predicted Srs2-BRCv disrupted the ability of the protein to bind RAD51 and to inhibit D-loop formation.  Thus, BRC is a common RAD51 interaction module that can be utilized by different proteins to either promote HR, as in the case of BRCA2, or to suppress HR, as in RECQL5.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgx_Xhss7s9bVg90H21EOLACvtfcHk0ljg_0C3YAYwlg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XpvVejurw%253D&md5=e9774611014fb47f3d7d43f70f67cd34</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M112.375014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M112.375014%26sid%3Dliteratum%253Aachs%26aulast%3DIslam%26aufirst%3DM.%2BN.%26aulast%3DPaquet%26aufirst%3DN.%26aulast%3DFox%26aufirst%3DD.%26aulast%3DDray%26aufirst%3DE.%26aulast%3DZheng%26aufirst%3DX.-F.%26aulast%3DKlein%26aufirst%3DH.%26aulast%3DSung%26aufirst%3DP.%26aulast%3DWang%26aufirst%3DW.%26atitle%3DA%2520Variant%2520of%2520the%2520Breast%2520Cancer%2520Type%25202%2520Susceptibility%2520Protein%2520%2528BRC%2529%2520Repeat%2520Is%2520Essential%2520for%2520the%2520RECQL5%2520Helicase%2520to%2520Interact%2520with%2520RAD51%2520Recombinase%2520for%2520Genome%2520Stabilization%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2012%26volume%3D287%26spage%3D23808%26epage%3D23818%26doi%3D10.1074%2Fjbc.M112.375014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Helleday, T.</span></span> <span> </span><span class="NLM_article-title">Homologous Recombination in Cancer Development, Treatment and Development of Drug Resistance</span>. <i>Carcinogenesis</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">955</span>– <span class="NLM_lpage">960</span>, <span class="refDoi"> DOI: 10.1093/carcin/bgq064</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=10.1093%2Fcarcin%2Fbgq064" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=20351092" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=1%3ACAS%3A528%3ADC%252BC3cXmslSqsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2010&pages=955-960&author=T.+Helleday&title=Homologous+Recombination+in+Cancer+Development%2C+Treatment+and+Development+of+Drug+Resistance&doi=10.1093%2Fcarcin%2Fbgq064"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Homologous recombination in cancer development, treatment and development of drug resistance</span></div><div class="casAuthors">Helleday, Thomas</div><div class="citationInfo"><span class="NLM_cas:title">Carcinogenesis</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">955-960</span>CODEN:
                <span class="NLM_cas:coden">CRNGDP</span>;
        ISSN:<span class="NLM_cas:issn">0143-3334</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Although DNA double-strand breaks (DSBs) are substrates for homologous recombination (HR) repair, it is becoming apparent that DNA lesions produced at replication forks, for instance by many anticancer drugs, are more significant substrates for HR repair.  Cells defective in HR are hypersensitive to a wide variety of anticancer drugs, including those that do not produce DSBs.  Several cancers have mutations in or epigenetically silenced HR genes, which explain the genetic instability that drives cancer development.  There are an increasing no. of reports suggesting that mutation or epigenetic silencing of HR genes explains the sensitivity of cancers to current chemotherapy treatments.  Furthermore, there are also many examples of re-expression of HR genes in tumors to explain drug resistance.  Emerging data suggest that there are several different subpathways of HR, which can compensate for each other.  Unravelling the overlapping pathways in HR showed that BRCA1- and BRCA2-defective cells rely on the PARP protein for survival.  This synthetic lethal interaction is now being exploited for selective treatment of BRCA1- and BRCA2-defective cancers with PARP inhibitors.  Here, I discuss the diversity of HR and how it impacts on cancer with a particular focus on how HR can be exploited in future anticancer strategies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGraaL2tqlRLWrVg90H21EOLACvtfcHk0ljKEPsY_oIACA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXmslSqsbY%253D&md5=1385e375ac39dd982ff9892fc8492486</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1093%2Fcarcin%2Fbgq064&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fcarcin%252Fbgq064%26sid%3Dliteratum%253Aachs%26aulast%3DHelleday%26aufirst%3DT.%26atitle%3DHomologous%2520Recombination%2520in%2520Cancer%2520Development%252C%2520Treatment%2520and%2520Development%2520of%2520Drug%2520Resistance%26jtitle%3DCarcinogenesis%26date%3D2010%26volume%3D31%26spage%3D955%26epage%3D960%26doi%3D10.1093%2Fcarcin%2Fbgq064" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hengel, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spies, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spies, M.</span></span> <span> </span><span class="NLM_article-title">Small-Molecule Inhibitors Targeting DNA Repair and DNA Repair Deficiency in Research and Cancer Therapy</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">1101</span>– <span class="NLM_lpage">1119</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2017.08.027</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=10.1016%2Fj.chembiol.2017.08.027" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=28938088" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFGgurbE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2017&pages=1101-1119&author=S.+R.+Hengelauthor=M.+A.+Spiesauthor=M.+Spies&title=Small-Molecule+Inhibitors+Targeting+DNA+Repair+and+DNA+Repair+Deficiency+in+Research+and+Cancer+Therapy&doi=10.1016%2Fj.chembiol.2017.08.027"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Small-Molecule Inhibitors Targeting DNA Repair and DNA Repair Deficiency in Research and Cancer Therapy</span></div><div class="casAuthors">Hengel, Sarah R.; Spies, M. Ashley; Spies, Maria</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1101-1119</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">To maintain stable genomes and to avoid cancer and aging, cells need to repair a multitude of deleterious DNA lesions, which arise constantly in every cell.  Processes that support genome integrity in normal cells, however, allow cancer cells to develop resistance to radiation and DNA-damaging chemotherapeutics.  Chem. inhibition of the key DNA repair proteins and pharmacol. induced synthetic lethality have become instrumental in both dissecting the complex DNA repair networks and as promising anticancer agents.  The difficulty in capitalizing on synthetically lethal interactions in cancer cells is that many potential targets do not possess well-defined small-mol. binding determinates.  In this review, we discuss several successful campaigns to identify and leverage small-mol. inhibitors of the DNA repair proteins, from PARP1, a paradigm case for clin. successful small-mol. inhibitors, to coveted new targets, such as RAD51 recombinase, RAD52 DNA repair protein, MRE11 nuclease, and WRN DNA helicase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZweRKd9gHmLVg90H21EOLACvtfcHk0ljKEPsY_oIACA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFGgurbE&md5=b8b865f8df43100effd60037efdc96b5</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2017.08.027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2017.08.027%26sid%3Dliteratum%253Aachs%26aulast%3DHengel%26aufirst%3DS.%2BR.%26aulast%3DSpies%26aufirst%3DM.%2BA.%26aulast%3DSpies%26aufirst%3DM.%26atitle%3DSmall-Molecule%2520Inhibitors%2520Targeting%2520DNA%2520Repair%2520and%2520DNA%2520Repair%2520Deficiency%2520in%2520Research%2520and%2520Cancer%2520Therapy%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2017%26volume%3D24%26spage%3D1101%26epage%3D1119%26doi%3D10.1016%2Fj.chembiol.2017.08.027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tyagi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patro, B. S.</span></span> <span> </span><span class="NLM_article-title">Salinomycin Reduces Growth, Proliferation and Metastasis of Cisplatin Resistant Breast Cancer Cells via NF-KB Deregulation</span>. <i>Toxicol. In Vitro</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">125</span>– <span class="NLM_lpage">133</span>, <span class="refDoi"> DOI: 10.1016/j.tiv.2019.05.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=10.1016%2Fj.tiv.2019.05.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=31077746" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtVagtbvF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2019&pages=125-133&author=M.+Tyagiauthor=B.+S.+Patro&title=Salinomycin+Reduces+Growth%2C+Proliferation+and+Metastasis+of+Cisplatin+Resistant+Breast+Cancer+Cells+via+NF-KB+Deregulation&doi=10.1016%2Fj.tiv.2019.05.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Salinomycin reduces growth, proliferation and metastasis of cisplatin resistant breast cancer cells via NF-kB deregulation</span></div><div class="casAuthors">Tyagi, Mrityunjay; Patro, Birija Sankar</div><div class="citationInfo"><span class="NLM_cas:title">Toxicology In Vitro</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">125-133</span>CODEN:
                <span class="NLM_cas:coden">TIVIEQ</span>;
        ISSN:<span class="NLM_cas:issn">0887-2333</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Cisplatin (cis-diamminedichloro-platinum, CDDP), is a widely used platinum compd. for various solid tumors including breast cancer as first line of therapy.  However, its pos. effects are limited due to acquired drug resistance and severe side effects in non-malignant tissue, esp. due to dose-dependent nephro- and/or neuro-toxicity.  Salinomycin is an antibiotic with coccidiostat effect and has shown anticancer efficacy against various cancer cells with selectivity in targeting cancer stem cells.  In the present study, anticancer efficacy and mechanism of action of salinomycin in CDDP-resistant human breast cancer (MCF7DDP) cells has been examd.  Initially, we generated CDDP-resistant cells by a new protocol followed by checking the anticancer efficacy of salinomycin through MTT, clonogenic, annexin-V/PI and sub-G1 assay.  Our results demonstrated that salinomycin diminished both cell proliferation and metastatic migration of MCF7DDP cells.  Salinomycin also induced mitochondrial dysfunction in CDDP-resistant breast cancer cells.  The anal. of nuclear translocation of pro-survival transcription factors by western blotting showed a distinct role of p65 (NF-κB) in CDDP-mediated resistance in breast cancer.  Salinomycin abrogated nuclear translocation of NF-κB proteins and also caused a concurrent redn. in NF-κB regulated expression of pro-survival proteins e.g., survivin, XIAP and BCL-2 in CDDP-resistant cells.  These results suggest that a follow up treatment of salinomycin may be promising strategy against CDDP resistant breast cancer cells and metastasis and help in reducing CDDP-induced side effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWxauTvGowGrVg90H21EOLACvtfcHk0ljKEPsY_oIACA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtVagtbvF&md5=82ad43598dc72d07cd086ef1f4c7abc8</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2Fj.tiv.2019.05.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tiv.2019.05.004%26sid%3Dliteratum%253Aachs%26aulast%3DTyagi%26aufirst%3DM.%26aulast%3DPatro%26aufirst%3DB.%2BS.%26atitle%3DSalinomycin%2520Reduces%2520Growth%252C%2520Proliferation%2520and%2520Metastasis%2520of%2520Cisplatin%2520Resistant%2520Breast%2520Cancer%2520Cells%2520via%2520NF-KB%2520Deregulation%26jtitle%3DToxicol.%2520In%2520Vitro%26date%3D2019%26volume%3D60%26spage%3D125%26epage%3D133%26doi%3D10.1016%2Fj.tiv.2019.05.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sarbajna, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">West, S. C.</span></span> <span> </span><span class="NLM_article-title">Roles of SLX1-SLX4, MUS81-EME1, and GEN1 in Avoiding Genome Instability and Mitotic Catastrophe</span>. <i>Genes Dev.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">1124</span>– <span class="NLM_lpage">1136</span>, <span class="refDoi"> DOI: 10.1101/gad.238303.114</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=10.1101%2Fgad.238303.114" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=24831703" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=1%3ACAS%3A528%3ADC%252BC2cXpt12qs7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2014&pages=1124-1136&author=S.+Sarbajnaauthor=D.+Daviesauthor=S.+C.+West&title=Roles+of+SLX1-SLX4%2C+MUS81-EME1%2C+and+GEN1+in+Avoiding+Genome+Instability+and+Mitotic+Catastrophe&doi=10.1101%2Fgad.238303.114"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Roles of SLX1-SLX4, MUS81-EME1, and GEN1 in avoiding genome instability and mitotic catastrophe</span></div><div class="casAuthors">Sarbajna, Shriparna; Davies, Derek; West, Stephen C.</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1124-1136</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">The resoln. of recombination intermediates contg. Holliday junctions (HJs) is crit. for genome maintenance and proper chromosome segregation.  Three pathways for HJ processing exist in human cells and involve the following enzymes/complexes: BLM-TopoIIIα-RMI1-RMI2 (BTR complex), SLX1-SLX4-MUS81-EME1 (SLX-MUS complex), and GEN1.  Cycling cells preferentially use the BTR complex for the removal of double HJs in S phase, with SLX-MUS and GEN1 acting at temporally distinct phases of the cell cycle.  Cells lacking SLX-MUS and GEN1 exhibit chromosome missegregation, micronucleus formation, and elevated levels of 53BP1-pos. G1 nuclear bodies, suggesting that defects in chromosome segregation lead to the transmission of extensive DNA damage to daughter cells.  In addn., however, we found that the effects of SLX4, MUS81, and GEN1 depletion extend beyond mitosis, since genome instability is obsd. throughout all phases of the cell cycle.  This is exemplified in the form of impaired replication fork movement and S-phase progression, endogenous checkpoint activation, chromosome segmentation, and multinucleation.  In contrast to SLX4, SLX1, the nuclease subunit of the SLX1-SLX4 structure-selective nuclease, plays no role in the replication-related phenotypes assocd. with SLX4/MUS81 and GEN1 depletion.  These observations demonstrate that the SLX1-SLX4 nuclease and the SLX4 scaffold play divergent roles in the maintenance of genome integrity in human cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphztHqtWw8nLVg90H21EOLACvtfcHk0ljPL4ckKs7v1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXpt12qs7w%253D&md5=2d1beecb6ad11ac1857778539977aca2</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1101%2Fgad.238303.114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.238303.114%26sid%3Dliteratum%253Aachs%26aulast%3DSarbajna%26aufirst%3DS.%26aulast%3DDavies%26aufirst%3DD.%26aulast%3DWest%26aufirst%3DS.%2BC.%26atitle%3DRoles%2520of%2520SLX1-SLX4%252C%2520MUS81-EME1%252C%2520and%2520GEN1%2520in%2520Avoiding%2520Genome%2520Instability%2520and%2520Mitotic%2520Catastrophe%26jtitle%3DGenes%2520Dev.%26date%3D2014%26volume%3D28%26spage%3D1124%26epage%3D1136%26doi%3D10.1101%2Fgad.238303.114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Krajewska, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fehrmann, R. S. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de
Vries, E. G. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Vugt, M. A. T. M.</span></span> <span> </span><span class="NLM_article-title">Regulators of Homologous Recombination Repair as Novel Targets for Cancer Treatment</span>. <i>Front. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">96</span>, <span class="refDoi"> DOI: 10.3389/fgene.2015.00096</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=10.3389%2Ffgene.2015.00096" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=25852742" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=1%3ACAS%3A280%3ADC%252BC2MjitFynug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=96&author=M.+Krajewskaauthor=R.+S.+N.+Fehrmannauthor=E.+G.+E.+de%0AVriesauthor=M.+A.+T.+M.+van+Vugt&title=Regulators+of+Homologous+Recombination+Repair+as+Novel+Targets+for+Cancer+Treatment&doi=10.3389%2Ffgene.2015.00096"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Regulators of homologous recombination repair as novel targets for cancer treatment</span></div><div class="casAuthors">Krajewska Malgorzata; Fehrmann Rudolf S N; de Vries Elisabeth G E; van Vugt Marcel A T M</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in genetics</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">96</span>
        ISSN:<span class="NLM_cas:issn">1664-8021</span>.
    </div><div class="casAbstract">To cope with DNA damage, cells possess a complex signaling network called the 'DNA damage response', which coordinates cell cycle control with DNA repair.  The importance of this network is underscored by the cancer predisposition that frequently goes along with hereditary mutations in DNA repair genes.  One especially important DNA repair pathway in this respect is homologous recombination (HR) repair.  Defects in HR repair are observed in various cancers, including hereditary breast, and ovarian cancer.  Intriguingly, tumor cells with defective HR repair show increased sensitivity to chemotherapeutic reagents, including platinum-containing agents.  These observations suggest that HR-proficient tumor cells might be sensitized to chemotherapeutics if HR repair could be therapeutically inactivated.  HR repair is an extensively regulated process, which depends strongly on the activity of various other pathways, including cell cycle pathways, protein-control pathways, and growth factor-activated receptor signaling pathways.  In this review, we discuss how the mechanistic wiring of HR is controlled by cell-intrinsic or extracellular pathways.  Furthermore, we have performed a meta-analysis on available genome-wide RNA interference studies to identify additional pathways that control HR repair.  Finally, we discuss how these HR-regulatory pathways may provide therapeutic targets in the context of radio/chemosensitization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSOCPY1LIxia0cLvDNbOD4DfW6udTcc2eackwrbmlKaP7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2MjitFynug%253D%253D&md5=c7e4d6066151172de4130c8df50aec97</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.3389%2Ffgene.2015.00096&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffgene.2015.00096%26sid%3Dliteratum%253Aachs%26aulast%3DKrajewska%26aufirst%3DM.%26aulast%3DFehrmann%26aufirst%3DR.%2BS.%2BN.%26aulast%3Dde%2BVries%26aufirst%3DE.%2BG.%2BE.%26aulast%3Dvan%2BVugt%26aufirst%3DM.%2BA.%2BT.%2BM.%26atitle%3DRegulators%2520of%2520Homologous%2520Recombination%2520Repair%2520as%2520Novel%2520Targets%2520for%2520Cancer%2520Treatment%26jtitle%3DFront.%2520Genet.%26date%3D2015%26volume%3D6%26spage%3D96%26doi%3D10.3389%2Ffgene.2015.00096" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Orbegoso, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marquina, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">George, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banerjee, S.</span></span> <span> </span><span class="NLM_article-title">The Role of Cediranib in Ovarian Cancer</span>. <i>Expert Opin. Pharmacother.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">1637</span>– <span class="NLM_lpage">1648</span>, <span class="refDoi"> DOI: 10.1080/14656566.2017.1383384</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=10.1080%2F14656566.2017.1383384" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=28933580" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1GqtLfM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=1637-1648&author=C.+Orbegosoauthor=G.+Marquinaauthor=A.+Georgeauthor=S.+Banerjee&title=The+Role+of+Cediranib+in+Ovarian+Cancer&doi=10.1080%2F14656566.2017.1383384"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">The role of Cediranib in ovarian cancer</span></div><div class="casAuthors">Orbegoso, Cecilia; Marquina, Gloria; George, Angela; Banerjee, Susana</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Pharmacotherapy</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">1637-1648</span>CODEN:
                <span class="NLM_cas:coden">EOPHF7</span>;
        ISSN:<span class="NLM_cas:issn">1465-6566</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">Treatment options for relapsed ovarian cancer have increased over the decade with the addn. of targeted agents, such as PARP inhibitors and antiangiogenic agents.  Bevacizumab, a monoclonal antibody binding vascular endothelial growth factor (VEGF), was the first anti-angiogenic agent to be incorporated in the ovarian cancer treatment landscape.  Other mols. utilizing different mechanisms of action to target angiogenesis have been developed, including cediranib, an oral potent inhibitor of VEGF Tyrosine Kinase Inhibitor that has demonstrated activity in both phase II and phase III studies.  Herein we will review cediranib as well as the evidence for its use in ovarian cancer, both as monotherapy and in combination with chemotherapy, PARP inhibitors and immunotherapy.  A literature search was made in PubMed and on ClinicalTrials.gov for clin. trials with cediranib.  The addn. of cediranib for the treatment of ovarian cancer is promising, and has demonstrated a significant improvement in progression free survival in a phase III trial in combination with chemotherapy and maintenance treatment.  Cediranib is currently being explored in ovarian cancer and other gynaecol. malignancies aiming to improve patient care; further research will help define its role in std. clin. practice for patients with ovarian cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGruIr3HI2T3y7Vg90H21EOLACvtfcHk0ljPL4ckKs7v1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1GqtLfM&md5=0f304893af3b8bdaf3d4863b29353684</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1080%2F14656566.2017.1383384&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F14656566.2017.1383384%26sid%3Dliteratum%253Aachs%26aulast%3DOrbegoso%26aufirst%3DC.%26aulast%3DMarquina%26aufirst%3DG.%26aulast%3DGeorge%26aufirst%3DA.%26aulast%3DBanerjee%26aufirst%3DS.%26atitle%3DThe%2520Role%2520of%2520Cediranib%2520in%2520Ovarian%2520Cancer%26jtitle%3DExpert%2520Opin.%2520Pharmacother.%26date%3D2017%26volume%3D18%26spage%3D1637%26epage%3D1648%26doi%3D10.1080%2F14656566.2017.1383384" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kaplan, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gueble, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oeck, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yun, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glazer, P. M.</span></span> <span> </span><span class="NLM_article-title">Cediranib Suppresses Homology-Directed DNA Repair through down-Regulation of BRCA1/2 and RAD51</span>. <i>Sci. Transl. Med.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>11</i></span>, <span class="NLM_elocation-id">eaav4508</span> <span class="refDoi"> DOI: 10.1126/scitranslmed.aav4508</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=10.1126%2Fscitranslmed.aav4508" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=31092693" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2019&author=A.+R.+Kaplanauthor=S.+E.+Guebleauthor=Y.+Liuauthor=S.+Oeckauthor=H.+Kimauthor=Z.+Yunauthor=P.+M.+Glazer&title=Cediranib+Suppresses+Homology-Directed+DNA+Repair+through+down-Regulation+of+BRCA1%2F2+and+RAD51&doi=10.1126%2Fscitranslmed.aav4508"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1126%2Fscitranslmed.aav4508&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscitranslmed.aav4508%26sid%3Dliteratum%253Aachs%26aulast%3DKaplan%26aufirst%3DA.%2BR.%26aulast%3DGueble%26aufirst%3DS.%2BE.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DOeck%26aufirst%3DS.%26aulast%3DKim%26aufirst%3DH.%26aulast%3DYun%26aufirst%3DZ.%26aulast%3DGlazer%26aufirst%3DP.%2BM.%26atitle%3DCediranib%2520Suppresses%2520Homology-Directed%2520DNA%2520Repair%2520through%2520down-Regulation%2520of%2520BRCA1%252F2%2520and%2520RAD51%26jtitle%3DSci.%2520Transl.%2520Med.%26date%3D2019%26volume%3D11%26doi%3D10.1126%2Fscitranslmed.aav4508" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bagnolini, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milano, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manerba, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schipani, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ortega, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gioia, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Falchi, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balboni, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farabegoli, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Franco, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robertson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pellicciari, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pallavicini, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peri, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minucci, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Girotto, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Stefano, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberti, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cavalli, A.</span></span> <span> </span><span class="NLM_article-title">Synthetic Lethality in Pancreatic Cancer: Discovery of a New RAD51-BRCA2 Small Molecule Disruptor That Inhibits Homologous Recombination and Synergizes with Olaparib</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">2588</span>– <span class="NLM_lpage">2619</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b01526</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01526" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=1%3ACAS%3A528%3ADC%252BB3cXisFKrurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=2588-2619&author=G.+Bagnoliniauthor=D.+Milanoauthor=M.+Manerbaauthor=F.+Schipaniauthor=J.+A.+Ortegaauthor=D.+Gioiaauthor=F.+Falchiauthor=A.+Balboniauthor=F.+Farabegoliauthor=F.+De+Francoauthor=J.+Robertsonauthor=R.+Pellicciariauthor=I.+Pallaviciniauthor=S.+Periauthor=S.+Minucciauthor=S.+Girottoauthor=G.+Di+Stefanoauthor=M.+Robertiauthor=A.+Cavalli&title=Synthetic+Lethality+in+Pancreatic+Cancer%3A+Discovery+of+a+New+RAD51-BRCA2+Small+Molecule+Disruptor+That+Inhibits+Homologous+Recombination+and+Synergizes+with+Olaparib&doi=10.1021%2Facs.jmedchem.9b01526"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Synthetic Lethality in Pancreatic Cancer: Discovery of a New RAD51-BRCA2 Small Molecule Disruptor That Inhibits Homologous Recombination and Synergizes with Olaparib</span></div><div class="casAuthors">Bagnolini, Greta; Milano, Domenico; Manerba, Marcella; Schipani, Fabrizio; Ortega, Jose Antonio; Gioia, Dario; Falchi, Federico; Balboni, Andrea; Farabegoli, Fulvia; De Franco, Francesca; Robertson, Janet; Pellicciari, Roberto; Pallavicini, Isabella; Peri, Sebastiano; Minucci, Saverio; Girotto, Stefania; Di Stefano, Giuseppina; Roberti, Marinella; Cavalli, Andrea</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2588-2619</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Synthetic lethality is an innovative framework for discovering novel anticancer drug candidates.  One example is the use of PARP inhibitors (PARPi) in oncol. patients with BRCA mutations.  Here, we exploit a new paradigm based on the possibility of triggering synthetic lethality using only small org. mols. (dubbed "fully small-mol.-induced synthetic lethality").  We exploited this paradigm to target pancreatic cancer, one of the major unmet needs in oncol.  We discovered a dihydroquinolone pyrazoline-based mol. (35d) that disrupts the RAD51-BRCA2 protein-protein interaction, thus mimicking the effect of BRCA2 mutation. 35d inhibits the homologous recombination in a human pancreatic adenocarcinoma cell line.  In addn., it synergizes with olaparib (a PARPi) to trigger synthetic lethality.  This strategy aims to widen the use of PARPi in BRCA-competent and olaparib-resistant cancers, making fully small-mol.-induced synthetic lethality an innovative approach toward unmet oncol. needs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoiTSEdA8VygbVg90H21EOLACvtfcHk0lgzyGt8Ad2UDA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXisFKrurg%253D&md5=1cae61064fc0e637f9b5f00c47e0bdeb</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01526%26sid%3Dliteratum%253Aachs%26aulast%3DBagnolini%26aufirst%3DG.%26aulast%3DMilano%26aufirst%3DD.%26aulast%3DManerba%26aufirst%3DM.%26aulast%3DSchipani%26aufirst%3DF.%26aulast%3DOrtega%26aufirst%3DJ.%2BA.%26aulast%3DGioia%26aufirst%3DD.%26aulast%3DFalchi%26aufirst%3DF.%26aulast%3DBalboni%26aufirst%3DA.%26aulast%3DFarabegoli%26aufirst%3DF.%26aulast%3DDe%2BFranco%26aufirst%3DF.%26aulast%3DRobertson%26aufirst%3DJ.%26aulast%3DPellicciari%26aufirst%3DR.%26aulast%3DPallavicini%26aufirst%3DI.%26aulast%3DPeri%26aufirst%3DS.%26aulast%3DMinucci%26aufirst%3DS.%26aulast%3DGirotto%26aufirst%3DS.%26aulast%3DDi%2BStefano%26aufirst%3DG.%26aulast%3DRoberti%26aufirst%3DM.%26aulast%3DCavalli%26aufirst%3DA.%26atitle%3DSynthetic%2520Lethality%2520in%2520Pancreatic%2520Cancer%253A%2520Discovery%2520of%2520a%2520New%2520RAD51-BRCA2%2520Small%2520Molecule%2520Disruptor%2520That%2520Inhibits%2520Homologous%2520Recombination%2520and%2520Synergizes%2520with%2520Olaparib%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D2588%26epage%3D2619%26doi%3D10.1021%2Facs.jmedchem.9b01526" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, G. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dexheimer, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenthal, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chu, W. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mosedale, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bachrati, C. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schultz, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakurai, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savitsky, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McHugh, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bohr, V. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jadhav, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gileadi, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maloney, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simeonov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hickson, I. D.</span></span> <span> </span><span class="NLM_article-title">A Small Molecule Inhibitor of the BLM Helicase Modulates Chromosome Stability in Human Cells</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">55</span>– <span class="NLM_lpage">62</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2012.10.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=10.1016%2Fj.chembiol.2012.10.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=23352139" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1equr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2013&pages=55-62&author=G.+H.+Nguyenauthor=T.+S.+Dexheimerauthor=A.+S.+Rosenthalauthor=W.+K.+Chuauthor=D.+K.+Singhauthor=G.+Mosedaleauthor=C.+Z.+Bachratiauthor=L.+Schultzauthor=M.+Sakuraiauthor=P.+Savitskyauthor=M.+Abuauthor=P.+J.+McHughauthor=V.+A.+Bohrauthor=C.+C.+Harrisauthor=A.+Jadhavauthor=O.+Gileadiauthor=D.+J.+Maloneyauthor=A.+Simeonovauthor=I.+D.+Hickson&title=A+Small+Molecule+Inhibitor+of+the+BLM+Helicase+Modulates+Chromosome+Stability+in+Human+Cells&doi=10.1016%2Fj.chembiol.2012.10.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">A Small Molecule Inhibitor of the BLM Helicase Modulates Chromosome Stability in Human Cells</span></div><div class="casAuthors">Nguyen, Giang Huong; Dexheimer, Thomas S.; Rosenthal, Andrew S.; Chu, Wai Kit; Singh, Dharmendra Kumar; Mosedale, Georgina; Bachrati, Csanad Z.; Schultz, Lena; Sakurai, Masaaki; Savitsky, Pavel; Abu, Mika; McHugh, Peter J.; Bohr, Vilhelm A.; Harris, Curtis C.; Jadhav, Ajit; Gileadi, Opher; Maloney, David J.; Simeonov, Anton; Hickson, Ian D.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">55-62</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The Bloom's syndrome protein, BLM, is a member of the conserved RecQ helicase family.  Although cell lines lacking BLM exist, these exhibit progressive genomic instability that makes distinguishing primary from secondary effects of BLM loss problematic.  In order to be able to acutely disable BLM function in cells, we undertook a high throughput screen of a chem. compd. library for small mol. inhibitors of BLM.  We present ML216, a potent inhibitor of the DNA unwinding activity of BLM.  ML216 shows cell-based activity and can induce sister chromatid exchanges, enhance the toxicity of aphidicolin, and exert antiproliferative activity in cells expressing BLM, but not those lacking BLM.  These data indicate that ML216 shows strong selectivity for BLM in cultured cells.  We discuss the potential utility of such a BLM-targeting compd. as an anticancer agent.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhJ5QlteU3zLVg90H21EOLACvtfcHk0lgzyGt8Ad2UDA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1equr8%253D&md5=26bfa8e45f2253aa8807d055be0689f2</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2012.10.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2012.10.016%26sid%3Dliteratum%253Aachs%26aulast%3DNguyen%26aufirst%3DG.%2BH.%26aulast%3DDexheimer%26aufirst%3DT.%2BS.%26aulast%3DRosenthal%26aufirst%3DA.%2BS.%26aulast%3DChu%26aufirst%3DW.%2BK.%26aulast%3DSingh%26aufirst%3DD.%2BK.%26aulast%3DMosedale%26aufirst%3DG.%26aulast%3DBachrati%26aufirst%3DC.%2BZ.%26aulast%3DSchultz%26aufirst%3DL.%26aulast%3DSakurai%26aufirst%3DM.%26aulast%3DSavitsky%26aufirst%3DP.%26aulast%3DAbu%26aufirst%3DM.%26aulast%3DMcHugh%26aufirst%3DP.%2BJ.%26aulast%3DBohr%26aufirst%3DV.%2BA.%26aulast%3DHarris%26aufirst%3DC.%2BC.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DGileadi%26aufirst%3DO.%26aulast%3DMaloney%26aufirst%3DD.%2BJ.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DHickson%26aufirst%3DI.%2BD.%26atitle%3DA%2520Small%2520Molecule%2520Inhibitor%2520of%2520the%2520BLM%2520Helicase%2520Modulates%2520Chromosome%2520Stability%2520in%2520Human%2520Cells%26jtitle%3DChem.%2520Biol.%26date%3D2013%26volume%3D20%26spage%3D55%26epage%3D62%26doi%3D10.1016%2Fj.chembiol.2012.10.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yin, Q.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.-X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.-Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, Q.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ou, T.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, S.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.-G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, J.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S.-B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Z.-S.</span></span> <span> </span><span class="NLM_article-title">Discovery of Isaindigotone Derivatives as Novel Bloom’s Syndrome Protein (BLM) Helicase Inhibitors That Disrupt the BLM/DNA Interactions and Regulate the Homologous Recombination Repair</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">3147</span>– <span class="NLM_lpage">3162</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b00083</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b00083" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=1%3ACAS%3A528%3ADC%252BC1MXktV2msbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=3147-3162&author=Q.-K.+Yinauthor=C.-X.+Wangauthor=Y.-Q.+Wangauthor=Q.-L.+Guoauthor=Z.-L.+Zhangauthor=T.-M.+Ouauthor=S.-L.+Huangauthor=D.+Liauthor=H.-G.+Wangauthor=J.-H.+Tanauthor=S.-B.+Chenauthor=Z.-S.+Huang&title=Discovery+of+Isaindigotone+Derivatives+as+Novel+Bloom%E2%80%99s+Syndrome+Protein+%28BLM%29+Helicase+Inhibitors+That+Disrupt+the+BLM%2FDNA+Interactions+and+Regulate+the+Homologous+Recombination+Repair&doi=10.1021%2Facs.jmedchem.9b00083"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Isaindigotone Derivatives as Novel Bloom's Syndrome Protein (BLM) Helicase Inhibitors That Disrupt the BLM/DNA Interactions and Regulate the Homologous Recombination Repair</span></div><div class="casAuthors">Yin, Qi-Kun; Wang, Chen-Xi; Wang, Yu-Qing; Guo, Qian-Liang; Zhang, Zi-Lin; Ou, Tian-Miao; Huang, Shi-Liang; Li, Ding; Wang, Hong-Gen; Tan, Jia-Heng; Chen, Shuo-Bin; Huang, Zhi-Shu</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">3147-3162</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Homologous recombination repair (HRR), a crucial approach in DNA damage repair, is an attractive target in cancer therapy and drug design.  The Bloom syndrome protein (BLM) is a 3'-5' DNA helicase that performs an important role in HRR regulation.  However, limited studies about BLM inhibitors and their biol. effects have been reported.  Here, we identified a class of isaindigotone derivs. as novel BLM inhibitors by synthesis, screening, and evaluating.  Among them, compd. 29 was found as an effective BLM inhibitor with a high binding affinity and good inhibitory effect on BLM.  Cellular evaluation indicated that 29 effectively disrupted the recruitment of BLM at DNA double-strand break sites, promoted an accumulation of RAD51, and regulated the HRR process.  Meanwhile, 29 significantly induced DNA damage responses, as well as apoptosis and proliferation arrest in cancer cells.  Our finding provides a potential anticancer strategy based on interfering with BLM via small mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_jlhubiNMYLVg90H21EOLACvtfcHk0lgzyGt8Ad2UDA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXktV2msbg%253D&md5=63b56e7be57afe71d3b279e2f5fb2663</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b00083&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b00083%26sid%3Dliteratum%253Aachs%26aulast%3DYin%26aufirst%3DQ.-K.%26aulast%3DWang%26aufirst%3DC.-X.%26aulast%3DWang%26aufirst%3DY.-Q.%26aulast%3DGuo%26aufirst%3DQ.-L.%26aulast%3DZhang%26aufirst%3DZ.-L.%26aulast%3DOu%26aufirst%3DT.-M.%26aulast%3DHuang%26aufirst%3DS.-L.%26aulast%3DLi%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DH.-G.%26aulast%3DTan%26aufirst%3DJ.-H.%26aulast%3DChen%26aufirst%3DS.-B.%26aulast%3DHuang%26aufirst%3DZ.-S.%26atitle%3DDiscovery%2520of%2520Isaindigotone%2520Derivatives%2520as%2520Novel%2520Bloom%25E2%2580%2599s%2520Syndrome%2520Protein%2520%2528BLM%2529%2520Helicase%2520Inhibitors%2520That%2520Disrupt%2520the%2520BLM%252FDNA%2520Interactions%2520and%2520Regulate%2520the%2520Homologous%2520Recombination%2520Repair%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D3147%26epage%3D3162%26doi%3D10.1021%2Facs.jmedchem.9b00083" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Islam, M. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fox, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Enomoto, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span> <span> </span><span class="NLM_article-title">RecQL5 Promotes Genome Stabilization through Two Parallel Mechanisms-Interacting with RNA Polymerase II and Acting as a Helicase</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">2460</span>– <span class="NLM_lpage">2472</span>, <span class="refDoi"> DOI: 10.1128/MCB.01583-09</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=10.1128%2FMCB.01583-09" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=20231364" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=1%3ACAS%3A528%3ADC%252BC3cXmtFymsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2010&pages=2460-2472&author=M.+N.+Islamauthor=D.+Foxauthor=R.+Guoauthor=T.+Enomotoauthor=W.+Wang&title=RecQL5+Promotes+Genome+Stabilization+through+Two+Parallel+Mechanisms-Interacting+with+RNA+Polymerase+II+and+Acting+as+a+Helicase&doi=10.1128%2FMCB.01583-09"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">RecQL5 promotes genome stabilization through two parallel mechanisms-interacting with RNA polymerase II and acting as a helicase</span></div><div class="casAuthors">Islam, M. Nurul; Fox, David, III; Guo, Rong; Enomoto, Takemi; Wang, Weidong</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2460-2472</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The RecQL5 helicase is essential for maintaining genome stability and reducing cancer risk.  To elucidate its mechanism of action, we purified a RecQL5-assocd. complex and identified its major component as RNA polymerase II (Pol II).  Bioinformatics and structural modeling-guided mutagenesis revealed two conserved regions in RecQL5 as KIX and SRI domains, already known in transcriptional regulators for Pol II.  The RecQL5-KIX domain binds both initiation (Pol IIa) and elongation (Pol IIo) forms of the polymerase, whereas the RecQL5-SRI domain interacts only with the elongation form.  Fully functional RecQL5 requires both helicase activity and assocns. with the initiation polymerase, because mutants lacking either activity are partially defective in the suppression of sister chromatid exchange and resistance to camptothecin-induced DNA damage, and mutants lacking both activities are completely defective.  We propose that RecQL5 promotes genome stabilization through two parallel mechanisms: by participation in homologous recombination-dependent DNA repair as a RecQ helicase and by regulating the initiation of Pol II to reduce transcription-assocd. replication impairment and recombination.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrslpjg_dVw0bVg90H21EOLACvtfcHk0ljui7zb6oIlMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXmtFymsb4%253D&md5=0098f5ba745b0ee230b0ee3dd1901b62</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1128%2FMCB.01583-09&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.01583-09%26sid%3Dliteratum%253Aachs%26aulast%3DIslam%26aufirst%3DM.%2BN.%26aulast%3DFox%26aufirst%3DD.%26aulast%3DGuo%26aufirst%3DR.%26aulast%3DEnomoto%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DW.%26atitle%3DRecQL5%2520Promotes%2520Genome%2520Stabilization%2520through%2520Two%2520Parallel%2520Mechanisms-Interacting%2520with%2520RNA%2520Polymerase%2520II%2520and%2520Acting%2520as%2520a%2520Helicase%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2010%26volume%3D30%26spage%3D2460%26epage%3D2472%26doi%3D10.1128%2FMCB.01583-09" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Saha, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patro, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koli, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pai, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ray, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bandyopadhyay, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chattopadhyay, S.</span></span> <span> </span><span class="NLM_article-title">trans-4,4’-Dihydroxystilbene (DHS) inhibits human neuroblastoma tumor growth and induces mitochondrial and lysosomal damages in neuroblastoma cell lines</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">73905</span>– <span class="NLM_lpage">73924</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.17879</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=10.18632%2Foncotarget.17879" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=29088756" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=1%3ACAS%3A280%3ADC%252BC1M7mtlaksw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=73905-73924&author=B.+Sahaauthor=B.+S.+Patroauthor=M.+Koliauthor=G.+Paiauthor=J.+Rayauthor=S.+K.+Bandyopadhyayauthor=S.+Chattopadhyay&title=trans-4%2C4%E2%80%99-Dihydroxystilbene+%28DHS%29+inhibits+human+neuroblastoma+tumor+growth+and+induces+mitochondrial+and+lysosomal+damages+in+neuroblastoma+cell+lines&doi=10.18632%2Foncotarget.17879"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">trans-4,4'-Dihydroxystilbene (DHS) inhibits human neuroblastoma tumor growth and induces mitochondrial and lysosomal damages in neuroblastoma cell lines</span></div><div class="casAuthors">Saha Bhaskar; Bandyopadhyay Sandip K; Saha Bhaskar; Ray Jharna; Patro Birija Sankar; Koli Mrunesh; Pai Ganesh; Chattopadhyay Subrata; Patro Birija Sankar; Chattopadhyay Subrata</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">43</span>),
    <span class="NLM_cas:pages">73905-73924</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">In view of the inadequacy of neuroblastoma treatment, five hydroxystilbenes and resveratrol (Resv) were screened for their cytotoxic property against human neuroblastoma cell lines.  The mechanism of cytotoxic action of the most potent compound, trans-4,4'-dihydroxystilbene (DHS) was investigated in vitro using human neuroblastoma cell lines.  DHS was also tested in a mouse xenograft model of human neuroblastoma tumor.  The MTT, sub-G1, annexin V and clonogenic assays as well as microscopy established higher cytotoxicity of DHS than Resv to the IMR32 cell line.  DHS (20 μM) induced mitochondrial membrane permeabilization (MMP) in the cells, as revealed from JC-1 staining, cytochrome c and ApaF1 release and caspases-9/3 activation.  DHS also induced lysosomal membrane permeabilization (LMP) to release cathepsins B, L and D, and the cathepsins inhibitors partially reduced MMP/caspase-3 activation.  The ROS, produced by DHS activated the p38 and JNK MAPKs to augment the BAX activity and BID-cleavage, and induce LMP and MMP in the cells.  DHS (100 mg/kg) also inhibited human neuroblastoma tumor growth in SCID mice by 51%.  Hence, DHS may be a potential chemotherapeutic option against neuroblastoma.  The involvement of an independent LMP as well as a partially LMP-dependent MMP by DHS is attractive as it provides options to target both mitochondria and lysosome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQRKdIxzbJTPMrRqEXsZufAfW6udTcc2eZMqtRlY1KM6Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M7mtlaksw%253D%253D&md5=b45c3118eb6a1568040792ac15e48ad9</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.17879&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.17879%26sid%3Dliteratum%253Aachs%26aulast%3DSaha%26aufirst%3DB.%26aulast%3DPatro%26aufirst%3DB.%2BS.%26aulast%3DKoli%26aufirst%3DM.%26aulast%3DPai%26aufirst%3DG.%26aulast%3DRay%26aufirst%3DJ.%26aulast%3DBandyopadhyay%26aufirst%3DS.%2BK.%26aulast%3DChattopadhyay%26aufirst%3DS.%26atitle%3Dtrans-4%252C4%25E2%2580%2599-Dihydroxystilbene%2520%2528DHS%2529%2520inhibits%2520human%2520neuroblastoma%2520tumor%2520growth%2520and%2520induces%2520mitochondrial%2520and%2520lysosomal%2520damages%2520in%2520neuroblastoma%2520cell%2520lines%26jtitle%3DOncotarget%26date%3D2017%26volume%3D8%26spage%3D73905%26epage%3D73924%26doi%3D10.18632%2Foncotarget.17879" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Patro, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frohlich, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bohr, V. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevnsner, T.</span></span> <span> </span><span class="NLM_article-title">WRN Helicase Regulates the ATR-CHK1-Induced S-Phase Checkpoint Pathway in Response to Topoisomerase-I-DNA Covalent Complexes</span>. <i>J. Cell Sci.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>124</i></span>,  <span class="NLM_fpage">3967</span>– <span class="NLM_lpage">3979</span>, <span class="refDoi"> DOI: 10.1242/jcs.081372</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=10.1242%2Fjcs.081372" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=22159421" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsV2lu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=124&publication_year=2011&pages=3967-3979&author=B.+S.+Patroauthor=R.+Frohlichauthor=V.+A.+Bohrauthor=T.+Stevnsner&title=WRN+Helicase+Regulates+the+ATR-CHK1-Induced+S-Phase+Checkpoint+Pathway+in+Response+to+Topoisomerase-I-DNA+Covalent+Complexes&doi=10.1242%2Fjcs.081372"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">WRN helicase regulates the ATR-CHK1-induced S-phase checkpoint pathway in response to topoisomerase-I-DNA covalent complexes</span></div><div class="casAuthors">Patro, Birija Sankar; Froehlich, Rikke; Bohr, Vilhelm A.; Stevnsner, Tinna</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cell Science</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">124</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">3967-3979</span>CODEN:
                <span class="NLM_cas:coden">JNCSAI</span>;
        ISSN:<span class="NLM_cas:issn">0021-9533</span>.
    
            (<span class="NLM_cas:orgname">Company of Biologists Ltd.</span>)
        </div><div class="casAbstract">Checkpoints are cellular surveillance and signaling pathways that coordinate the response to DNA damage and replicative stress.  Consequently, failure of cellular checkpoints increases susceptibility to DNA damage and can lead to profound genome instability.  This study examines the role of a human RECQ helicase, WRN, in checkpoint activation in response to DNA damage.  Mutations in WRN lead to genomic instability and the premature aging condition Werner syndrome.  Here, the role of WRN in a DNA-damage-induced checkpoint was analyzed in U-2 OS (WRN wild type) and isogenic cells stably expressing WRN-targeted shRNA (WRN knockdown).  The results of our studies suggest that WRN has a crucial role in inducing an S-phase checkpoint in cells exposed to the topoisomerase I inhibitor campthothecin (CPT), but not in cells exposed to hydroxyurea.  Intriguingly, WRN decreases the rate of replication fork elongation, increases the accumulation of ssDNA and stimulates phosphorylation of CHK1, which releases CHK1 from chromatin in CPT-treated cells.  Importantly, knockdown of WRN expression abolished or delayed all these processes in response to CPT.  Together, our results strongly suggest an essential regulatory role for WRN in controlling the ATR-CHK1-mediated S-phase checkpoint in CPT-treated cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrFgrwlOxEQ7bVg90H21EOLACvtfcHk0ljui7zb6oIlMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsV2lu7s%253D&md5=079e1b6a1b371997113498274ab81f39</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1242%2Fjcs.081372&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1242%252Fjcs.081372%26sid%3Dliteratum%253Aachs%26aulast%3DPatro%26aufirst%3DB.%2BS.%26aulast%3DFrohlich%26aufirst%3DR.%26aulast%3DBohr%26aufirst%3DV.%2BA.%26aulast%3DStevnsner%26aufirst%3DT.%26atitle%3DWRN%2520Helicase%2520Regulates%2520the%2520ATR-CHK1-Induced%2520S-Phase%2520Checkpoint%2520Pathway%2520in%2520Response%2520to%2520Topoisomerase-I-DNA%2520Covalent%2520Complexes%26jtitle%3DJ.%2520Cell%2520Sci.%26date%3D2011%26volume%3D124%26spage%3D3967%26epage%3D3979%26doi%3D10.1242%2Fjcs.081372" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Han, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span> <span> </span><span class="NLM_article-title">Immunomodulatory Activity of a Water-Soluble Polysaccharide Obtained from Highland Barley on Immunosuppressive Mice Models</span>. <i>Food Funct.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">304</span>– <span class="NLM_lpage">314</span>, <span class="refDoi"> DOI: 10.1039/C8FO01991F</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=10.1039%2FC8FO01991F" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=30574978" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisFSmsL7L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=304-314&author=L.+Hanauthor=M.+Mengauthor=M.+Guoauthor=D.+Chengauthor=L.+Shiauthor=X.+Wangauthor=C.+Wang&title=Immunomodulatory+Activity+of+a+Water-Soluble+Polysaccharide+Obtained+from+Highland+Barley+on+Immunosuppressive+Mice+Models&doi=10.1039%2FC8FO01991F"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Immunomodulatory activity of a water-soluble polysaccharide obtained from highland barley on immunosuppression mice models</span></div><div class="casAuthors">Han, Lirong; Meng, Meng; Guo, Mingzhu; Cheng, Dai; Shi, Lei; Wang, Xu; Wang, Chunling</div><div class="citationInfo"><span class="NLM_cas:title">Food & Function</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">304-314</span>CODEN:
                <span class="NLM_cas:coden">FFOUAI</span>;
        ISSN:<span class="NLM_cas:issn">2042-6496</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A water-sol. polysaccharide (BP-1) was extd. and purified from highland barley (Hordeum vulgare L.) and its av. mol. wt. was about 6.7x104 Da.  In this study, the immunomodu-latory activity of BP-1 on the immunosuppressive BALB/c mice model and its mol. mechanism were elucidated.  It was found that the wt. indexes of spleens and thymus were significantly in-creased by BP-1 (80 mg· kg-1 and 160 mg· kg-1) treatment in immunosuppressive mice model.  The results showed that BP-1 (80 mg· kg-1 and 160 mg· kg-1) could significantly increase the no. of bone marrow cells (BMC) and peripheral blood white blood cells (WBC) in immunosuppressive mice model.  In addn., the result further confirmed that BP-1 could increase the serum levels of IL-2, TNF-α and IFN-γ, so as to improve the immune function of immunosuppressive mice.  The results showed that BP-1 (80 mg· kg-1 and 160 mg· kg-1) could promote the proliferation of spleen cells, the natural killer (NK) cells activity in vivo.  The quant. real-time polymerase chain reaction (qRT-PCR) and ELISA results revealed that BP-1 (80 mg· kg-1 and 160 mg· kg-1) could enhance the prodn. of IL-2, TNF-α, IFN-γ, IgG and IgM in the spleen of immunosuppressive mice.  The HE (hematoxylin and eosin) stained histopathol. images showed that BP-1 (80 mg· kg-1 and 160 mg· kg-1) could repair the damage inducing by CTX in the spleen cells of immunosuppressive mice.  The result of macrophages showed that BP-1 (80 mg· kg-1 and 160 mg· kg-1) could promote the pro-liferation and phagocytosis activity of macrophages in immunosuppressive mice.  Furthermore, BP-1 could activate macrophages by the TLR-4- TRAF6- TAK1- nuclear factor κB (NF-κB) p65 pathway in vivo.  These results suggested that BP-1 has a remarkable immunomodulatory activity on the immu-nosuppressive mice model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoS0HrEh7f80rVg90H21EOLACvtfcHk0ljlDBu6kg4OAA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisFSmsL7L&md5=684d7c09b2ebb6daad1f924c8554aab2</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1039%2FC8FO01991F&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC8FO01991F%26sid%3Dliteratum%253Aachs%26aulast%3DHan%26aufirst%3DL.%26aulast%3DMeng%26aufirst%3DM.%26aulast%3DGuo%26aufirst%3DM.%26aulast%3DCheng%26aufirst%3DD.%26aulast%3DShi%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DC.%26atitle%3DImmunomodulatory%2520Activity%2520of%2520a%2520Water-Soluble%2520Polysaccharide%2520Obtained%2520from%2520Highland%2520Barley%2520on%2520Immunosuppressive%2520Mice%2520Models%26jtitle%3DFood%2520Funct.%26date%3D2019%26volume%3D10%26spage%3D304%26epage%3D314%26doi%3D10.1039%2FC8FO01991F" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LB3" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LB3','PDB','5LB3'); return false;">PDB: 5LB3</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i63"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01692">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_33545"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.0c01692?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01692</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Effect of compounds on cell viability and clonogenic potential; effects of <b>4a</b> on different RECQL5-WT- and RECQL5-depleted breast cancer cells, DNA binding, and ATPase activity, role of <b>4a</b> in DSB generation, apoptosis, NHEJ- and HRR-regulating proteins, stabilizing RECQL5-RAD51 complex, and cell cycle; pharmacokinetic and toxicity effects of <b>4a;</b> HPLC purity; sequence of the primers and DNA oligos;NMR spectra; and oligonucleotides used in this study (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01692/suppl_file/jm0c01692_si_001.pdf">PDF</a>)</p></li><li><p class="inline">(<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01692/suppl_file/jm0c01692_si_002.pdb">PDB</a>)</p></li><li><p class="inline">(<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01692/suppl_file/jm0c01692_si_003.pdb">PDB</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01692/suppl_file/jm0c01692_si_001.pdf">jm0c01692_si_001.pdf (2.38 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01692/suppl_file/jm0c01692_si_002.pdb">jm0c01692_si_002.pdb (275.48 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01692/suppl_file/jm0c01692_si_003.pdb">jm0c01692_si_003.pdb (275.72 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.0c01692&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.0c01692%3Bwgroup%3Astring%3AACHS+website+Group%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2021.64.issue-3%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.0c01692" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                13MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"67993c726c8c3dca","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
