
<html lang="en"     class="pb-page"  data-request-id="6b4395e3-f2c0-4b41-839c-06a7a296934d"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;issue:issue:10.1021/jmcmar.2021.64.issue-11;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;article:article:10.1021/acs.jmedchem.1c00280;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Dynamics-Based Discovery of Novel, Potent Benzoic Acid Derivatives as Orally Bioavailable Selective Estrogen Receptor Degraders for ERα+ Breast Cancer" /></meta><meta name="dc.Creator" content="Xiaomeng  Zhang" /></meta><meta name="dc.Creator" content="Yazhou  Wang" /></meta><meta name="dc.Creator" content="Xue  Li" /></meta><meta name="dc.Creator" content="Jie  Wu" /></meta><meta name="dc.Creator" content="Liwen  Zhao" /></meta><meta name="dc.Creator" content="Wei  Li" /></meta><meta name="dc.Creator" content="Jian  Liu" /></meta><meta name="dc.Description" content="The estrogen receptor α (ERα) is identified as an effective target for the treatment of ERα+ breast cancer; thus, discovery of novel selective estrogen receptor degraders (SERDs) are developed as a..." /></meta><meta name="Description" content="The estrogen receptor α (ERα) is identified as an effective target for the treatment of ERα+ breast cancer; thus, discovery of novel selective estrogen receptor degraders (SERDs) are developed as a..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="May 31, 2021" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.1c00280" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2021 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.1c00280" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.1c00280" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.1c00280" /></link>
        
    
    

<title>Dynamics-Based Discovery of Novel, Potent Benzoic Acid Derivatives as Orally Bioavailable Selective Estrogen Receptor Degraders for ERα+ Breast Cancer | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.1c00280" /></meta><meta property="og:title" content="Dynamics-Based Discovery of Novel, Potent Benzoic Acid Derivatives as Orally Bioavailable Selective Estrogen Receptor Degraders for ERα+ Breast Cancer" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00280/20210602/images/large/jm1c00280_0012.jpeg" /></meta><meta property="og:description" content="The estrogen receptor α (ERα) is identified as an effective target for the treatment of ERα+ breast cancer; thus, discovery of novel selective estrogen receptor degraders (SERDs) are developed as an effective method to overcome the resistance of breast cancer. Herein, the hot-spot residues for protein–ligand interaction between SERDs and ERα are analyzed by molecular dynamic simulation technology, focusing on the hot-spot residues for four series of designed and synthesized SERDs. SAR studies revealed that while the acrylic acid moiety of AZD9496 is scaffold hopping into benzoic acid, compound D24 exhibits potent binding affinity with ERα, good degradation efficacy of ERα, and inhibitory effect against the MCF-7 breast cancer cell line. Besides, D24 also displays good antitumor efficacy in the MCF-7 human breast cancer xenograft model in vivo, favorable pharmacokinetic properties, excellent druggability, and good safety property, making D24 as a promising drug candidate of SERD for further evaluation." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.1c00280"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00280">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.1c00280&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.1c00280&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.1c00280&amp;href=/doi/10.1021/acs.jmedchem.1c00280" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2021</span><span class="cit-fg-volume">, 64</span><span class="cit-fg-issue">, 11</span><span class="cit-fg-pageRange">, 7575-7595</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/64/11" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.1c00264" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.1c00301" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Dynamics-Based Discovery of Novel, Potent Benzoic Acid Derivatives as Orally Bioavailable Selective Estrogen Receptor Degraders for ERα+ Breast Cancer</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Xiaomeng Zhang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xiaomeng Zhang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Pharmacy, Nanjing University of Chinese Medicine, Nanjing 210023, China</div><div class="loa-info-affiliations-info">R&D Center, Nanjing Sanhome Pharmaceutical Company Ltd., Nanjing 211135, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xiaomeng++Zhang">Xiaomeng Zhang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yazhou Wang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yazhou Wang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">R&D Center, Nanjing Sanhome Pharmaceutical Company Ltd., Nanjing 211135, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yazhou++Wang">Yazhou Wang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Xue Li</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xue Li</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">R&D Center, Nanjing Sanhome Pharmaceutical Company Ltd., Nanjing 211135, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xue++Li">Xue Li</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jie Wu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jie Wu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">R&D Center, Nanjing Sanhome Pharmaceutical Company Ltd., Nanjing 211135, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jie++Wu">Jie Wu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Liwen Zhao</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Liwen Zhao</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">R&D Center, Nanjing Sanhome Pharmaceutical Company Ltd., Nanjing 211135, China</div></div><span class="conrtib-corresp"><strong>*</strong>(L.Z.) Email: <a href="/cdn-cgi/l/email-protection#4d17252c22213a0d3e2c2325222028632e2220"><span class="__cf_email__" data-cfemail="42182a232d2e350231232c2a2d2f276c212d2f">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Liwen++Zhao">Liwen Zhao</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Wei Li</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Wei Li</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Pharmacy, Nanjing University of Chinese Medicine, Nanjing 210023, China</div></div><span class="conrtib-corresp"><strong>*</strong>(W.L.) Email: <a href="/cdn-cgi/l/email-protection#365a5f41575f5f76585c43555b18535243185558"><span class="__cf_email__" data-cfemail="234f4a54424a4a634d4956404e0d4647560d404d">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Wei++Li">Wei Li</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Jian Liu</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jian Liu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Pharmacy, Nanjing University of Chinese Medicine, Nanjing 210023, China</div></div><span class="conrtib-corresp"><strong>*</strong>(J.L.) Email: <a href="/cdn-cgi/l/email-protection#b5d9dcc0dfdcd4db838786f5dbdfc0d6d89bd0d1c09bd6db"><span class="__cf_email__" data-cfemail="b2dedbc7d8dbd3dc848081f2dcd8c7d1df9cd7d6c79cd1dc">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jian++Liu">Jian Liu</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0001-9482-9036" title="Orcid link">https://orcid.org/0000-0001-9482-9036</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.1c00280&amp;href=/doi/10.1021%2Facs.jmedchem.1c00280" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2021</span></span><span class="cit-volume">, 64</span><span class="cit-issue">, 11</span><span class="cit-pageRange">, 7575–7595</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">May 31, 2021</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>13 February 2021</li><li><span class="item_label"><b>Published</b> online</span>31 May 2021</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 10 June 2021</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.1c00280" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.1c00280</a></div><div class="article_header-article-copyright"><strong>Copyright © 2021 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D7575%26pageCount%3D21%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DXiaomeng%2BZhang%252C%2BYazhou%2BWang%252C%2BXue%2BLi%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D64%26issueNum%3D11%26contentID%3Dacs.jmedchem.1c00280%26title%3DDynamics-Based%2BDiscovery%2Bof%2BNovel%252C%2BPotent%2BBenzoic%2BAcid%2BDerivatives%2Bas%2BOrally%2BBioavailable%2BSelective%2BEstrogen%2BReceptor%2BDegraders%2Bfor%2BER%25CE%25B1%252B%2BBreast%2BCancer%26numPages%3D21%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D7595%26publicationDate%3DJune%2B2021">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.1c00280"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">907</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val">-</div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.1c00280" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Dynamics-Based Discovery of Novel, Potent Benzoic Acid Derivatives as Orally Bioavailable Selective Estrogen Receptor Degraders for ERα+ Breast Cancer&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Xiaomeng&quot;,&quot;last_name&quot;:&quot;Zhang&quot;},{&quot;first_name&quot;:&quot;Yazhou&quot;,&quot;last_name&quot;:&quot;Wang&quot;},{&quot;first_name&quot;:&quot;Xue&quot;,&quot;last_name&quot;:&quot;Li&quot;},{&quot;first_name&quot;:&quot;Jie&quot;,&quot;last_name&quot;:&quot;Wu&quot;},{&quot;first_name&quot;:&quot;Liwen&quot;,&quot;last_name&quot;:&quot;Zhao&quot;},{&quot;first_name&quot;:&quot;Wei&quot;,&quot;last_name&quot;:&quot;Li&quot;},{&quot;first_name&quot;:&quot;Jian&quot;,&quot;last_name&quot;:&quot;Liu&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2021&quot;,&quot;month&quot;:&quot;05&quot;,&quot;day&quot;:&quot;31&quot;,&quot;issue&quot;:&quot;11&quot;,&quot;volume&quot;:&quot;64&quot;,&quot;pages&quot;:&quot;7575-7595&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.1c00280&quot;},&quot;abstract&quot;:&quot;The estrogen receptor α (ERα) is identified as an effective target for the treatment of ERα+ breast cancer; thus, discovery of novel selective estrogen receptor degraders (SERDs) are developed as an effective method to overcome the resistance of breast cancer. Herein, the hot-spot residues for protein–ligand interaction between SERDs and ERα are analyzed by molecular dynamic simulation technology, focusing on the hot-spot residues for four series of designed and synthesized SERDs. SAR studies revealed that while the acrylic acid moiety of AZD9496 is scaffold hopping into benzoic acid, compound D24 exhibits potent binding affinity with ERα, good degradation efficacy of ERα, and inhibitory effect against the MCF-7 breast cancer cell line. Besides, D24 also displays good antitumor efficacy in the MCF-7 human breast cancer xenograft model in vivo, favorable pharmacokinetic properties, excellent druggability, and good safety property, making D24 as a promising drug candidate of SERD for further evaluation.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00280&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00280" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00280&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00280" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00280&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00280" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.1c00280&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00280&amp;href=/doi/10.1021/acs.jmedchem.1c00280" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.1c00280" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.1c00280" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (3 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00280&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.1c00280%26sid%3Dliteratum%253Aachs%26pmid%3D34056898%26genre%3Darticle%26aulast%3DZhang%26date%3D2021%26atitle%3DDynamics-Based%2BDiscovery%2Bof%2BNovel%252C%2BPotent%2BBenzoic%2BAcid%2BDerivatives%2Bas%2BOrally%2BBioavailable%2BSelective%2BEstrogen%2BReceptor%2BDegraders%2Bfor%2BER%25CE%25B1%252B%2BBreast%2BCancer%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D64%26issue%3D11%26spage%3D7575%26epage%3D7595%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291781" title="Degradation">Degradation</a>,</li><li><a href="/action/doSearch?ConceptID=290976" title="Free energy">Free energy</a>,</li><li><a href="/action/doSearch?ConceptID=292258" title="Cancer">Cancer</a>,</li><li><a href="/action/doSearch?ConceptID=291984" title="Molecular mechanics">Molecular mechanics</a>,</li><li><a href="/action/doSearch?ConceptID=292178" title="Organic compounds">Organic compounds</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/64/11" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/jmcmar.2021.64.issue-11/20210610/jmcmar.2021.64.issue-11.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00280/20210602/images/medium/jm1c00280_0012.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00280/20210602/images/large/jm1c00280_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00280&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">The estrogen receptor α (ERα) is identified as an effective target for the treatment of ERα+ breast cancer; thus, discovery of novel selective estrogen receptor degraders (SERDs) are developed as an effective method to overcome the resistance of breast cancer. Herein, the hot-spot residues for protein–ligand interaction between SERDs and ERα are analyzed by molecular dynamic simulation technology, focusing on the hot-spot residues for four series of designed and synthesized SERDs. SAR studies revealed that while the acrylic acid moiety of <b>AZD9496</b> is scaffold hopping into benzoic acid, compound <b>D24</b> exhibits potent binding affinity with ERα, good degradation efficacy of ERα, and inhibitory effect against the MCF-7 breast cancer cell line. Besides, <b>D24</b> also displays good antitumor efficacy in the MCF-7 human breast cancer xenograft model in vivo, favorable pharmacokinetic properties, excellent druggability, and good safety property, making <b>D24</b> as a promising drug candidate of SERD for further evaluation.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_45848" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_45848" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Cancer is characterized by uncontrolled growth and spread of abnormal cells, and it is one of the most serious health hazards that may occur in different tissues.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> Breast cancer (BC) is one of the most common malignant tumors in women, which has the highest incidence rate.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> In 2019, according to the annual report released by China National Cancer Registration Center, breast cancer accounted for 16.5% of all cancer cases in women and 1.2 million women died from that. Breast cancer is classified in a clinical setting by its molecular subtypes, which can be divided into estrogen receptor positive (ERα+), human epidermal growth factor receptor 2 positive (HER2+), and triple-negative subtype.<a onclick="showRef(event, 'ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref3 ref4">(3,4)</a> As members of the nuclear receptor family, ERα and ERβ are transcription factors mediating the estrogen effects most known and regulating gene expression.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> ERα distributes in many tissues, such as breast, uterus, bone, and aorta, etc.; approximately 74% of breast cancer patients have high expression of ERα, while the overexpression of ERα is the main driving factor of breast cancer cell proliferation.<a onclick="showRef(event, 'ref6 ref7'); return false;" href="javascript:void(0);" class="ref ref6 ref7">(6,7)</a> Therefore, ERα has become an important target for the treatment of breast cancer.</div><div class="NLM_p">Nowadays, endocrine therapy is the main treatment to estrogen receptor positive breast cancer.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> Conceptually, on the basis of related targets of breast cancer and estrogen signaling pathways, the antiestrogen drugs can be mainly divided into three types: aromatase inhibitors (AIs), selective estrogen receptor modulators (SERMs), and selective estrogen receptor degraders (SERDs).<a onclick="showRef(event, 'ref9 ref10'); return false;" href="javascript:void(0);" class="ref ref9 ref10">(9,10)</a> The aromatase can catalyze the androstenedione into estrogen in vivo; thus, aromatase inhibitors (<b>1</b>, letrozole; <b>2</b>, anastrozole; <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>) indicate antitumor activity by inhibiting the synthesis of estrogen which is responsible for the key biosynthetic step in the formation of mitogenic molecules.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> SERMs can bind to the ligand binding region (LBD) of ER competitively and form a dimer complex, which changes the conformation of ERα and results in blocking of the interaction between coactivator and activating function-2 (AF2) of ERα, and finally can block the estrogen signaling pathway and inhibit tumor formation.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> SERMs, which are represented by tamoxifen and its active metabolite 4-hydroxytamoxifen, have been proved to stabilize an antagonist conformation in breast cancer cells, leading to antiproliferative signaling.<a onclick="showRef(event, 'ref13 ref14'); return false;" href="javascript:void(0);" class="ref ref13 ref14">(13,14)</a> Unfortunately, these compounds are not pure antagonists of ERα, which often exhibit partial activation of ERα in the uterus, resulting in serious side effects after long-term administration, such as endometrial hyperplasia and endometrial cancer.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> Moreover, although many patients initially respond to AIs and SERMs, drug resistance often emerges. Resistance to endocrine therapy is multifactorial, involving mutation of ESR1 and upregulation of growth factor signaling.<a onclick="showRef(event, 'ref16 ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref16 ref17 ref18">(16−18)</a> Therefore, the development of novel, high-efficiency and low-toxicity ERα antagonists might provide an alternative therapeutic option for drug-resistance breast cancer therapy.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00280/20210602/images/medium/jm1c00280_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00280/20210602/images/large/jm1c00280_0001.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Representative selective aromatase inhibitors (AIs), selective estrogen receptor modulators (SERMs), and selective estrogen receptor degraders (SERDs).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00280/20210602/images/large/jm1c00280_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00280&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Fortunately, selective estrogen receptor degraders (SERDs) are small molecules that target ERα for proteasome-dependent degradation. Fulvestrant (<b>5</b>) not only antagonizes the ERα-driven tumor cell growth but also induces the degradation of ERα via ubiquitinoylation, which has been approved by the FDA.<a onclick="showRef(event, 'ref19 ref20 ref21'); return false;" href="javascript:void(0);" class="ref ref19 ref20 ref21">(19−21)</a> Furthermore, the clinical success of fulvestrant reveals that the degradation of ER is beneficial to women with ER+ breast cancer, especially those who have progressed under endocrine therapies such as SERMs or AIs.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> However, due to poor physicochemical properties and low oral bioavailability, the utilization of fulvestrant is limited in clinic, which is administered as a monthly intramuscular implant at a dose of up to 500 mg.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> The poor physicochemical properties of fulvestrant are due to the steroidal structure, which led to an effort in designing a nonsteroidal scaffold of SERDs in order to enhance the oral bioavailability.</div><div class="NLM_p">Several companies are engaged in the exploration of novel orally available SERDs, and several new compounds have entered clinical trials recently, including <b>GW-5638</b> (<b>6</b>), <b>GW-7604</b> (<b>7</b>), <b>GDC-0810</b> (<b>8</b>), <b>AZD9496</b> (<b>9</b>), and <b>LSZ-102</b> (<b>10</b>).<a onclick="showRef(event, 'ref24 ref25 ref26 ref27 ref28'); return false;" href="javascript:void(0);" class="ref ref24 ref25 ref26 ref27 ref28">(24−28)</a> When the basic side chain of tamoxifen is substituted by acrylic acid group, <b>GW-5638</b> can be obtained, which improves the antagonistic activity and induces the degradation of ER. <b>GW-5638</b> is the first oral SERD, which is a prodrug and can be metabolized into <b>GW-7604</b> with better activity in vivo. The crystal structure of <b>GW-5638</b> bound to the ER LBD demonstrates that the interaction between the acrylate side chain and Leu536 and Tyr537 at the N-terminus of H12 was mediated by a structural water molecule; as a result, <b>GW-5638</b> shows potential inhibitory effect against tamoxifen-resistant breast cancer tumor cells.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> However, <b>GW-5638</b> failed in clinical phase I trial, which may be due to the poor metabolic properties. Meanwhile, the <b>GW-5638</b> analogue <b>GDC-0810</b> is also exhibiting the same problem. Furthermore, <b>AZD9496</b>, a novel tricyclic scaffold with an acrylic acid side chain, also possesses ER degradation mechanism, but it has side effects of uterine hyperplasia.<a onclick="showRef(event, 'ref30 ref31'); return false;" href="javascript:void(0);" class="ref ref30 ref31">(30,31)</a> LSZ-102, the systematic structural optimization from Arzoxifene has some drawbacks, such as poor metabolic properties, low degradation activity of ERα in vivo, and so on. Hence, seeking novel scaffold SERDs with potent degradation activity and excellent physicochemical properties is crucial to the success of SERDs design.</div><div class="NLM_p">Additionally, ERα contains two activating regions, AF1 and AF2, which control transcription activation. AF2 is located in the C-terminus of the LBD, and its function is dependent on the positioning of H12.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> As a result, most SERDs in clinical trials caused the degradation of ERα, because of the displacement and destabilization of H12 caused by forming a hydrogen bond between the acrylate side chain and H12 (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). The interaction between H12 and degraders will affect the position of H12, as the means to promote ERα degradation. Herein, focusing on the hydrogen-bond and hydrophobic interaction with key amino acids nearby H12 (called hot spots), we present the dynamics-simulation-based discovery of novel SERDs, containing a benzoic acid side chain instead of acrylic group. Through the systematic structure optimization, the potent biological activity and good oral bioavailability of compound <b>D24</b> indicated it can serve as a good candidate.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00280/20210602/images/medium/jm1c00280_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00280/20210602/images/large/jm1c00280_0002.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Analysis of binding “hot spots” of ERα. (a) Crystal structure of <b>AZD9496</b> (blue; PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5ACC">5ACC</a>) superimposed on reference structure of <b>GW-5638</b> (white; PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5AVV">5AVV</a>) bound to ERα pocket. The red circle revealed that the acryl acid group interacted with hot spot 1: Trp383, Leu525, and Val534. The blue rectangle revealed that the pyrido[3,4-<i>b</i>]-indole scaffold interacted with hot spot 2: Leu346, Met388, Leu391, Phe404, Met421, and Ile424. (b) Backbone RMSD of the protein along with simulation time. The blue line represents <b>GW-5638</b>; the red line represents <b>AZD9496</b>. (c) Hot-spot residue contribution for protein–ligand interaction. (d) Calculation results of hot-spot residue mutation to alanine for <b>AZD9496</b> (red) and <b>GW-6838</b> (blue) with ERα mutants.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00280/20210602/images/large/jm1c00280_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00280&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_01752" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_01752" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Analysis of Binding “Hot Spot” of ERα</h3><div class="NLM_p">The essential understanding of protein–ligand interaction and quantitative characterization of binding affinity are the basis of rational drug design. The small molecular drugs usually prefer binding to several key amino acids (called hot spots) in protein–ligand and protein–protein structures, rather than all of the amino acids in the active sites of target proteins.<a onclick="showRef(event, 'ref33 ref34 ref35 ref36'); return false;" href="javascript:void(0);" class="ref ref33 ref34 ref35 ref36">(33−36)</a> To quantify the contribution of the amino acid residue to the protein–ligand binding affinity, free energy perturbation (FEP), thermodynamic integration (TI), and the molecular mechanics–generalized Born surface area (MM-GBSA) methods are employed to assess the contribution of each amino acid in the protein–ligand interaction.<a onclick="showRef(event, 'ref37 ref38 ref39 ref40 ref41 ref42 ref43'); return false;" href="javascript:void(0);" class="ref ref37 ref38 ref39 ref40 ref41 ref42 ref43">(37−43)</a> The MM-GBSA method is the most practical application in computing protein–ligand binding free energy, and it has been successfully used to predict hot-spot protein–ligand interaction in recent years. Besides, the computational alanine scanning approach also can be used to evaluate the specific residue that contributes to the protein–ligand binding free energy.<a onclick="showRef(event, 'ref44 ref45 ref46'); return false;" href="javascript:void(0);" class="ref ref44 ref45 ref46">(44−46)</a> Thus, the hot spot of ERα can be identified by combining use of MM-GBSA and alanine scanning approaches, and as a consequence, the binding interaction with the active sites of the target protein can be explored, which lead to the discovery of novel SERDs with improved affinity and unique structure.</div><div class="NLM_p">In order to better understand the binding mode of SERDs and ERα, we superimposed two ER crystal structures of <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5acc">5acc</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5AAV">5AAV</a>, which were complexed with <b>AZD9496</b> and <b>GW-5638</b>, respectively (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>a). The two crystal structures indicated that the binding modes of <b>AZD9496</b> and <b>GW-5638</b> with ERα were similar, and a few of the key interactions were found from the crystal structure of <b>AZD9496</b>. Once the benzenoid ring of the indole was replaced by the phenyl ring of <b>GW-5638</b>, the indole NH formed a critical hydrogen bond with the carbonyl of Leu346. Meanwhile, the ERα structures (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6B0F">6B0F</a>) that contain ligand <b>LSZ-102</b> with a hydroxyl moiety instead of an indole scaffold possessed a hydrogen-bonding network among residues Glu353, Arg394, and a hydroxyl group.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> Besides this H-bond interaction, the indole ring and its hydrophobic side chain of <b>AZD9496</b> were located at a lipophilic pocket, which were constituted by several hydrophobic residues, involving Leu346, Met388, Leu391, Phe404, Met421, and Ile424, etc. Furthermore, examining the interface between <b>AZD9496</b> and H12 of ERα in the crystal structure, it showed that the acrylic acid moiety was situated adjacent to Trp383, Leu525, and Val534, and the pendent aryl substituent also oriented toward the H12 region of the ERα; thus, the formation of hydrogen-bond and hydrophobic interaction with target were observed as the key for ERα downregulation.</div><div class="NLM_p">To better explore the hot-spot residues and design novel SERDs, molecular dynamic simulations for compounds <b>AZD9496</b> and <b>GW-5638</b> in complex with ERα (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5ACC">5ACC</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5AAV">5AAV</a>) were carried out for 100 ns, using the Desmond module in Schrodinger Maestro 2019. The root-mean-square-deviation (RMSD) values of protein backbone atoms relative to the initial structure were calculated to identify the protein stability during the simulation period. As a result, compounds <b>AZD9496</b> and <b>GW-5638</b> were converged to 1.30 and 1.48 Å (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>b), respectively, which revealed that the two compounds steadily combined with ERα. Subsequently, the binding free energies were calculated by using MM-GBSA method. The binding free energies were less than zero, implying the binding interaction between ERα and SERDs could be spontaneous. The total binding energies were −85.90 and −79.87 kcal/mol for the complexes of <b>AZD9496</b>–ERα and <b>GW-5638</b>–ERα (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Besides, the contributions favoring ligand binding were van der Waals energy, electrostatic interaction energy, and the nonpolar solvation interaction, whereas the polar solvation interaction was detrimental to bind with targets. As the structures of <b>AZD9496</b> and <b>GW-5638</b> mainly consist of hydrophobic groups and acrylic acid moiety, they could form hydrophobic and electrostatic interactions with ERα. Therefore, the electrostatic interaction energy and van der Waals energy were the two essential components of total binding free energy.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Total Binding Free Energy (kcal/mol) and Its Component (kcal/mol)</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="center" /></col><col align="center" /></col><col align="center" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">contribution</th><th class="colsep0 rowsep0" align="center"><b>AZD9496</b></th><th class="colsep0 rowsep0" align="center"><b>GW-5638</b></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center">Δ<i>E</i><sub>VDW</sub><a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="center">–52.88 ± 2.66</td><td class="colsep0 rowsep0" align="center">–47.11 ± 2.00</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center">Δ<i>E</i><sub>ele</sub><a class="ref internalNav" href="#t1fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="center">–40.58 ± 1.92</td><td class="colsep0 rowsep0" align="center">–35.64 ± 1.72</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center">Δ<i>E</i><sub>GB</sub><a class="ref internalNav" href="#t1fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="center">38.84 ± 3.08</td><td class="colsep0 rowsep0" align="center">32.66 ± 2.66</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center">Δ<i>E</i><sub>GA</sub><a class="ref internalNav" href="#t1fn4" aria-label="d">d</a></td><td class="colsep0 rowsep0" align="center">–30.98 ± 1.26</td><td class="colsep0 rowsep0" align="center">–29.78 ± 1.12</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center">Δ<i>G</i><sub>bind</sub><a class="ref internalNav" href="#t1fn5" aria-label="e">e</a></td><td class="colsep0 rowsep0" align="center">–85.60 ± 4.22</td><td class="colsep0 rowsep0" align="center">–79.87 ± 3.30</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">Contribution to the free energy of binding from the van der Waals energy.</p></div><div class="footnote" id="t1fn2"><sup><sup>b</sup></sup><p class="last">Contribution to the free energy of binding from the electrostatic energy.</p></div><div class="footnote" id="t1fn3"><sup><sup>c</sup></sup><p class="last">Contribution to the free energy of binding from the polar solvation energies.</p></div><div class="footnote" id="t1fn4"><sup><sup>d</sup></sup><p class="last">Contribution to the free energy of binding from the nonpolar solvation energies.</p></div><div class="footnote" id="t1fn5"><sup><sup>e</sup></sup><p class="last">Free energy of binding.</p></div></div></div><div class="NLM_p">Meanwhile, through analyzing the MD simulation result, one could also obtain the hydrophobic and hydrogen-bond contributions of each residue in the binding pocket of ERα. Then, the hydrophobic interaction between <b>AZD9496</b> and each residue was found, and the hydrophobic interaction of the <b>AZD9496</b>–ERα complex could be attributed to Leu346, Trp383, Met388, Leu391, Phe404, Met421, Ile424, Leu525, and Val534. Meanwhile, the hydrogen bond between the indole ring and acryl acid group of <b>AZD9496</b> and Leu346, Val534 were also dispensable for binding affinity of ERα. Thus, with respect to the complex <b>AZD9496</b>–ERα, Leu346 (−2.30 kcal/mol), Trp383 (−4.68 kcal/mol), Met388 (−1.32 kcal/mol), Leu391 (−1.86 kcal/mol), Phe404 (−1.56 kcal/mol), Met421 (−1.20 kcal/mol), Ile424 (−1.94 kcal/mol), Leu525 (−3.56 kcal/mol), and Val534 (−2.28 kcal/mol) were regarded as hot-spot residues (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>c).</div><div class="NLM_p last">In order to further evaluate the hot-spot residues, the computational alanine mutation approaches were also utilized. In computational alanine mutation approach to protein–ligand binding, a specific residue was mutated to alanine and the differences in binding free energies before and after the mutation were calculated (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>d). The results demonstrated that once the corresponding hot-spot residues were mutated to alanine, the binding affinity of ligand–protein became weaker. The detailed protein–ligand interactions could be concluded as follows: the acryl acid group formed H-bond interaction with Val534 and interacted with Trp383, Leu525, and Val534 by lipophilic contacts (hot spot 1). Meanwhile, the indole ring and 2-fluoro-2-methylpropane fragment of <b>AZD9496</b> underwent hydrophobic interactions with Leu346, Met388, Leu391, Phe404, Met421, and Ile424, and the NH atoms also formed a hydrogen bond with Leu346 (hot spot 2). Therefore, all of those interactions were beneficial to stabilize the conformation and combination of ligand lying in the binding pocket of ERα. The structural design of novel SERDs should focus on enhancing the interaction with hot-spot residues.</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Dynamic Simulation-Based Structure Design</h3><div class="NLM_p">To explore novel SERDs, four different series of compounds were designed and virtually screened (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>). To verify the rationality of the design strategy and estimate the binding affinity, the molecular docking and dynamic simulation study were combined which allowed the evaluation of the binding energies of the designed compounds (<b>F1</b>–<b>F9</b>) toward ERα (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5ACC">5ACC</a>). <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a> demonstrated the binding mode of a compound (<b>F1</b>–<b>F9</b>) to the hot spots of ERα, and <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a> listed the docking score and binding free energies.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00280/20210602/images/medium/jm1c00280_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00280/20210602/images/large/jm1c00280_0003.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Design strategy of target compounds.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00280/20210602/images/large/jm1c00280_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00280&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00280/20210602/images/medium/jm1c00280_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00280/20210602/images/large/jm1c00280_0004.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Docking result of design compounds (<b>F1</b>–<b>F9</b>) toward ERα (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5ACC">5ACC</a>) and drug candidate <b>AZD9496</b>. (a) <b>AZD9496</b> (blue) and <b>F1</b> (green); (b) <b>AZD9496</b> (blue), <b>F2</b> (white), and <b>F3</b> (pink); (c) <b>AZD9496</b> (blue), <b>F4</b> (white), <b>F5</b> (yellow), <b>F6</b> (pink); (d) <b>AZD9496</b> (blue), <b>F7</b> (white), <b>F8</b> (yellow), and <b>F9</b> (pink).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00280/20210602/images/large/jm1c00280_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00280&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Predicted Docking Energies, Binding Free Energies, and Binding Sites of Compounds <b>F1</b>–<b>F9</b></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center" char="."> </th><th class="rowsep1 colsep0" colspan="4" align="center">binding site</th><th class="colsep0 rowsep0" align="center" char="."> </th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center" char="."> </th><th class="rowsep1 colsep0" colspan="2" align="center">H-bond</th><th class="rowsep1 colsep0" colspan="2" align="center">hydrophobic</th><th class="colsep0 rowsep0" align="center" char="."> </th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center" char=".">docking score</th><th class="colsep0 rowsep0" align="center">hot spot 1</th><th class="colsep0 rowsep0" align="center">hot spot 2</th><th class="colsep0 rowsep0" align="center">hot spot 1</th><th class="colsep0 rowsep0" align="center">hot spot 2</th><th class="colsep0 rowsep0" align="center" char=".">binding free energy (kcal/mol)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>F1</b></td><td class="colsep0 rowsep0" align="char" char=".">–9.77</td><td class="colsep0 rowsep0" align="left">Val534</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">Trp383, Leu525, Val534</td><td class="colsep0 rowsep0" align="left">Leu346, Met388, Leu391, Phe404, Met421, Ile424</td><td class="colsep0 rowsep0" align="char" char=".">–68.69 ± 3.50</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>F2</b></td><td class="colsep0 rowsep0" align="char" char=".">–10.82</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">Leu346</td><td class="colsep0 rowsep0" align="left">Leu525, Val534</td><td class="colsep0 rowsep0" align="left">Leu346, Met388, Leu391, Phe404, Met421, Ile424</td><td class="colsep0 rowsep0" align="char" char=".">–75.22 ± 3.16</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>F3</b></td><td class="colsep0 rowsep0" align="char" char=".">–3.80</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">Leu346</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">Leu346, Met421, Ile424</td><td class="colsep0 rowsep0" align="char" char=".">–36.60 ± 2.65</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>F4</b></td><td class="colsep0 rowsep0" align="char" char=".">–11.31</td><td class="colsep0 rowsep0" align="left">Val534</td><td class="colsep0 rowsep0" align="left">Leu346</td><td class="colsep0 rowsep0" align="left">Trp383, Leu525, Val534</td><td class="colsep0 rowsep0" align="left">Leu346, Met388, Leu391, Phe404, Met421</td><td class="colsep0 rowsep0" align="char" char=".">–80.54 ± 1.84</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>F5</b></td><td class="colsep0 rowsep0" align="char" char=".">–11.24</td><td class="colsep0 rowsep0" align="left">Val534</td><td class="colsep0 rowsep0" align="left">Leu346</td><td class="colsep0 rowsep0" align="left">Trp383, Leu525, Val534</td><td class="colsep0 rowsep0" align="left">Leu346, Met388, Leu391, Phe404, Met421</td><td class="colsep0 rowsep0" align="char" char=".">–79.96 ± 2.74</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>F6</b></td><td class="colsep0 rowsep0" align="char" char=".">–10.80</td><td class="colsep0 rowsep0" align="left">Val534</td><td class="colsep0 rowsep0" align="left">Leu346</td><td class="colsep0 rowsep0" align="left">Trp383, Leu525, Val534</td><td class="colsep0 rowsep0" align="left">Leu346, Met388, Leu391, Phe404, Ile424</td><td class="colsep0 rowsep0" align="char" char=".">–74.72 ± 2.40</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>F7</b></td><td class="colsep0 rowsep0" align="char" char=".">–12.50</td><td class="colsep0 rowsep0" align="left">Val534</td><td class="colsep0 rowsep0" align="left">Leu346</td><td class="colsep0 rowsep0" align="left">Trp383, Leu525, Val534</td><td class="colsep0 rowsep0" align="left">Leu346, Met388, Leu391, Phe404, Met421, Ile424</td><td class="colsep0 rowsep0" align="char" char=".">–80.35 ± 3.38</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>F8</b></td><td class="colsep0 rowsep0" align="char" char=".">–13.49</td><td class="colsep0 rowsep0" align="left">Val534</td><td class="colsep0 rowsep0" align="left">Leu346</td><td class="colsep0 rowsep0" align="left">Trp383, Leu525, Val534</td><td class="colsep0 rowsep0" align="left">Leu346, Met388, Leu391, Phe404, Met421, Ile424</td><td class="colsep0 rowsep0" align="char" char=".">–92.35 ± 1.98</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>F9</b></td><td class="colsep0 rowsep0" align="char" char=".">–12.66</td><td class="colsep0 rowsep0" align="left">Val534</td><td class="colsep0 rowsep0" align="left">Leu346</td><td class="colsep0 rowsep0" align="left">Trp383, Leu525, Val534</td><td class="colsep0 rowsep0" align="left">Leu346, Met388, Leu391, Phe404, Met421, Ile424</td><td class="colsep0 rowsep0" align="char" char=".">–87.52 ± 2.53</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b><b>AZD9496</b></b></td><td class="colsep0 rowsep0" align="char" char=".">–12.29</td><td class="colsep0 rowsep0" align="left">Val534</td><td class="colsep0 rowsep0" align="left">Leu346</td><td class="colsep0 rowsep0" align="left">Trp383, Leu525, Val534</td><td class="colsep0 rowsep0" align="left">Leu346, Met388, Leu391, Phe404, Met421, Ile424</td><td class="colsep0 rowsep0" align="char" char=".">–85.60 ± 4.22</td></tr></tbody></table></div></div><div class="NLM_p last">With respect to <b>A</b> series compounds, the indole ring was substituted by <i>N</i>-methylindole (<b>F1</b>); however the docking energies and binding free energies decreased (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>), mainly because of the lack of hydrogen-bond interaction with Leu346 (<b>F1</b>, <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>). By use of scaffold hopping strategy, the <b>B</b> series compounds (<b>F2</b>, <b>F3</b>) were designed and obtained. Owing to the absence of hydrogen bond with Val534 and the hydrophobicity with Trp383 (hot spot 1), the binding free energy of <b>F2</b> was lower than that of <b>AZD9496</b>. Meanwhile, the theory calculation results illustrated that once the piperidine ring was cyclized with its linked methyl group (<b>F3</b>), the compound lost key interactions with hot spot 1 residue, Trp383, Leu525, and Val534; thus, the binding conformation changed dramatically, and the binding affinity was significantly reduced. Besides, the 2-fluoro-2-methylpropyl substituent surrounded by Met421 and Ile424 could be replaced by other groups, such as 3-fluoro-3-methylbutyl (<b>F4</b>), (1-(trifluoromethyl)cyclopropyl)methyl (<b>F5</b>), and phenyl (<b>F6</b>). As a result, although the binding abilities of those three compounds were similar to that of <b>AZD9496</b>, their pharmacological activity might be decreased due to the partial loss of binding with hot-spot residue. Finally, the three fragments of the <b>D</b> series (<b>F7</b>–<b>F9</b>) were obtained by bioisoterism and scaffold hopping method. When the benzene ring was substituted by pyridine ring (<b>F7</b>), there was no difference between <b>F7</b> and <b>AZD9496</b>. The docking orientations and interactions of <b>F8</b> were fitted as well as the <b>AZD9496</b>. Meanwhile, <b>F8</b> exhibited the favorable calculation results and binding affinity with SERDs through the strong hydrogen-bond and hydrophobic interaction with the hot spot 1 residue at H12 of ERα. Therefore, <b>F8</b> could be selected as a candidate for further structure optimization and pharmacological evaluation.</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Chemistry</h3><div class="NLM_p">The synthesis routes of the designed final compounds were demonstrated in <a class="ref internalNav" href="#sch1" aria-label="Schemes 1">Schemes 1</a><named-content content-type="anchor" rid="sch2" type="simple"></named-content><named-content content-type="anchor" rid="sch3" type="simple"></named-content>–<a class="ref internalNav" href="#sch4" aria-label="4">4</a>. Initially, treatment of the <i>N</i>-methyl of <b>AZD9496</b> in the mixture of NaH, MeI, and THF could afford the compound <b>A1</b> in 60% yield. With respect to the synthesis strategies of series <b>B</b>, the primary amine intermediates of <b>3</b>, <b>6</b>, and <b>10</b> with various scaffolds were first synthesized (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>). Because the commercial starting material <b>1</b> contained a good leaving group OTs, it proceeded as a nucleophilic substitution reaction with the fluorine donor TBAF. Then, under the hydrogenation conditions the protecting group Cbz was removed, affording the free amine intermediate <b>3</b> to be obtained in high yields. Accordingly, to access the desired intermediate <b>6</b>, the commercial reagent <b>4</b> was cyclized with the Grignard reagent EtMgBr, followed by the elimination of Ts from nitrogen atoms to furnish the intermediate <b>6</b> bearing a three-member ring in 76% yield. Meanwhile, the C–H alkylation reaction could be achieved from readily available alkene <b>7</b> and haloalkane <b>8</b> in the presence of Na<sub>2</sub>CO<sub>3</sub> as base and 2,2,2-trifluoroethanol as solvent at room temperature for 48 h. By exploring the classical reductive amination strategy, the ketone <b>9</b> was smoothly converted into amine <b>10</b> in moderate yield. Then, owing to the excellent reaction activity of the trifluoromethanesulfonate moiety, the C–N coupling reaction was easily carried out among amines <b>3</b>, <b>6</b>, <b>10</b>, and the other material 2-fluoro-2-methylpropyl trifluoromethanesulfonate (<b>11</b>) to provide coupling products <b>12a</b>–<b>12c</b>. Finally, through the Mannich reaction of the intermediate <b>12a</b>–<b>12c</b> with other building block methyl (<i>E</i>)-3-(3,5-difluoro-4-formylphenyl)acrylate (<b>13</b>) and relay ester hydrolysis under base environment, the desired carboxylic acid compounds <b>B1</b>–<b>B3</b> could be afforded.</div><figure id="sch1" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00280/20210602/images/medium/jm1c00280_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00280/20210602/images/large/jm1c00280_0008.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of <b>A</b> Series of Compounds<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00280/20210602/images/large/jm1c00280_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00280&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: MeI, NaH, THF, 0 °C, 4 h, 66%.</p></p></figure><figure id="sch2" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00280/20210602/images/medium/jm1c00280_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00280/20210602/images/large/jm1c00280_0009.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of <b>B</b> Series of Compounds<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00280/20210602/images/large/jm1c00280_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00280&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) TBAF, THF, 80 °C, 4 h, 73%; (b) Pd(OH)<sub>2</sub>, H<sub>2</sub>, EtOH, rt, 8 h, 98%; (c) MeTi(<i>i-</i>PrO)<sub>3</sub>, EtMgBr, BF<sub>3</sub>·Et<sub>2</sub>O, THF, 2 h, 76%; (d) NaOH, H<sub>2</sub>O, MeOH, 2 h, 80%; (e) Na<sub>2</sub>CO<sub>3</sub>, 2,2,2-trifluoroethanol, 48 h, 65%; (f) NaBH<sub>3</sub>CN, ammonium acetate, MeOH, rt, 8 h, 54%; (g) 2-fluoro-2-methylpropyl trifluoromethanesulfonate (<b>11</b>), DIPEA, dioxane, 90 °C, 2 h, 35–90%; (h) methyl (<i>E</i>)-3-(3,5-difluoro-4-formylphenyl)acrylate (<b>13</b>), HOAc, toluene, 80 °C, 8 h, 41–70%; (i) NaOH, H<sub>2</sub>O, THF, MeOH, rt, 75–83%.</p></p></figure><figure id="sch3" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00280/20210602/images/medium/jm1c00280_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00280/20210602/images/large/jm1c00280_0010.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of <b>C</b> Series of Compounds<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00280/20210602/images/large/jm1c00280_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00280&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) (i) alkyl trifluoromethanesulfonate, DIPEA, dioxane, 90 °C, 8 h, 63–72% and (ii) iodobenzene, CuI, Cs<sub>2</sub>CO<sub>3</sub>, DMF, microwave, 135 °C, 2 h, 34%; (b) methyl (<i>E</i>)-3-(3,5-difluoro-4-formylphenyl)acrylate (<b>13</b>), HOAc, toluene, 80 °C, 8 h, 41–72%; (c) NaOH, H<sub>2</sub>O, THF, MeOH, rt, 72–86%.</p></p></figure><figure id="sch4" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00280/20210602/images/medium/jm1c00280_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00280/20210602/images/large/jm1c00280_0011.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of <b>D</b> Series of Compounds<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00280/20210602/images/large/jm1c00280_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00280&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) methyl acrylate, Pd(OAc)<sub>2</sub>, Et<sub>3</sub>N, DMF, 80 °C, 8 h, 72–96%; (b) bis(pinacolato)diboron, PdCl<sub>2</sub>(dppf), KOAc, 80 °C, 12 h, 80–88%; (c) 4-bromo-2,6-difluorobenzaldehyde, PdCl<sub>2</sub>(dppf)<sub>2</sub>, Na<sub>2</sub>CO<sub>3</sub>, toluene, H<sub>2</sub>O, 100 °C, 16 h, 75–86%; (d) <b>17</b> or <b>20</b>, HOAc, toluene, 8 h, 45–78%; (e) NaOH, H<sub>2</sub>O, THF, MeOH, rt, 2 h, 70–92%; (f) 4-bromo-2,6-difluorobenzaldehyde, HOAc, toluene, 80 °C, 8 h, 84%; (g) NHR<sub>1</sub>R<sub>2</sub>, Pd<sub>2</sub>(dba)<sub>3</sub>, XPhos, Cs<sub>2</sub>CO<sub>3</sub>, toluene, 80 °C, 8 h, 60–81%;</p></p></figure><div class="NLM_p">In a similar way, the <b>C</b> series compounds <b>C1</b>–<b>C4</b> were prepared (<a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>). The critical C–N bond formation was performed by the free amine <b>14</b> with substituted alkyl-OTf or PhI via CuI-catalyzed coupling reaction. Then, the final product <b>C1</b>–<b>C4</b> could be obtained through consecutive reaction involving Mannich reaction between <b>15a</b>–<b>15d</b> and methyl (<i>E</i>)-3-(3,5-difluoro-4-formylphenyl)acrylate (<b>13</b>) and base-mediated ester hydrolysis.</div><div class="NLM_p last">Furthermore, the <b>D</b> series of compounds <b>D1</b>–<b>D42</b> could be synthesized by the analogous synthesis protocol (<a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>). The Heck coupling reaction of <b>16</b> and methyl acrylate, exploring Pd(OAc)<sub>2</sub> as catalyst, Et<sub>3</sub>N as base, and DMF as solvent, generated the various aldehyde derivatives <b>17</b>. Meanwhile, the readily available bromine-substituted heterocyclic compound <b>18</b> was smoothly converted into the corresponding borate ester <b>19</b> by palladium(II)-catalyzed coupling reaction; then the Suzuki coupling reactions of <b>19</b> and 4-bromo-2,6-difluorobenzaldehyde generated the other aldehyde derivatives <b>20</b>. Through cascade reaction sequences including Mannich reaction, transition-metal-catalyzed coupling, and ester hydrolysis reaction, the final products of <b>D1</b>–<b>D40</b> were obtained in moderate to high yield.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Initial Screen of ERα Binding, ERα Degradation, and MCF-7 Antiproliferative Activities</h3><div class="NLM_p">With respect to the large number of compounds synthesized, the three biological assays including the ERα binding, ERα degradation, and MCF-7 antiproliferative activities were conducted to evaluate the biological properties of these four series of compounds (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Tables <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a> and <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a>). Compound <b>AZD9496</b> was identified as a positive control. As a result, once the <i>N</i>-methylation of the indole scaffold of <b>AZD9496</b> had occurred, the biological activities of compound <b>A1</b> were decreased significantly, the reason being that the amino group was not performed as a H-bond donor to form H-bond interaction with Leu346.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. ER Binding, ER Downregulation Degradation, and MCF-7 Antiproliferative Activities for <b>A</b>, <b>B</b>, and <b>C</b> Series Compounds</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center">ERα binding IC<sub>50</sub> (nM)<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center">ERα degradation EC<sub>50</sub> (nM)<a class="ref internalNav" href="#t3fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center">MCF-7 IC<sub>50</sub> (nM)<a class="ref internalNav" href="#t3fn3" aria-label="c">c</a></th><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center">ERα binding IC<sub>50</sub> (nM)<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center">ERα degradation EC<sub>50</sub> (nM)<a class="ref internalNav" href="#t3fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center">MCF-7 IC<sub>50</sub> (nM)<a class="ref internalNav" href="#t3fn3" aria-label="c">c</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>A1</b></td><td class="colsep0 rowsep0" align="left">320.4 ± 10.9</td><td class="colsep0 rowsep0" align="left">14.8 ± 0.82</td><td class="colsep0 rowsep0" align="left">30.8 ± 1.40</td><td class="colsep0 rowsep0" align="left"><b>B1</b></td><td class="colsep0 rowsep0" align="left">20.6 ± 0.06</td><td class="colsep0 rowsep0" align="left">0.86 ± 0.05</td><td class="colsep0 rowsep0" align="left">0.83 ± 0.09</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>B2</b></td><td class="colsep0 rowsep0" align="left">720.6 ± 15.2</td><td class="colsep0 rowsep0" align="left">32.9 ± 1.26</td><td class="colsep0 rowsep0" align="left">39.3 ± 1.20</td><td class="colsep0 rowsep0" align="left"><b>B3</b></td><td class="colsep0 rowsep0" align="left">8.6 ± 0.12</td><td class="colsep0 rowsep0" align="left">0.35 ± 0.02</td><td class="colsep0 rowsep0" align="left">0.54 ± 0.11</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>C1</b></td><td class="colsep0 rowsep0" align="left">113.8 ± 3.55</td><td class="colsep0 rowsep0" align="left">4.48 ± 0.02</td><td class="colsep0 rowsep0" align="left">20.9 ± 0.56</td><td class="colsep0 rowsep0" align="left"><b>C2</b></td><td class="colsep0 rowsep0" align="left">135.4 ± 5.0</td><td class="colsep0 rowsep0" align="left">5.39 ± 0.08</td><td class="colsep0 rowsep0" align="left">35.3 ± 1.83</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>C3</b></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left">ND<a class="ref internalNav" href="#t3fn4" aria-label="d">d</a></td><td class="colsep0 rowsep0" align="left"><b>C4</b></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left">365.0 ± 3.68</td><td class="colsep0 rowsep0" align="left">ND<a class="ref internalNav" href="#t3fn4" aria-label="d">d</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>AZD9496</b></td><td class="colsep0 rowsep0" align="left">4.60 ± 0.04</td><td class="colsep0 rowsep0" align="left">0.23 ± 0.05</td><td class="colsep0 rowsep0" align="left">0.76 ± 0.06</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">Relative binding activity was determined by a Polar Screen ERα competitor Assay system from Invitrogen (no. A15883) with 150 nM full-length human ERα.</p></div><div class="footnote" id="t3fn2"><sup><sup>b</sup></sup><p class="last">ERα In-Cell Western in MCF-7 breast cancer cells cultured with DMEM containing 10% hormone-free FBS; readings taken after 4 h of incubation using an anti-ERα rabbit monoclonal antibody (<i>n</i> ≥ 3).</p></div><div class="footnote" id="t3fn3"><sup><sup>c</sup></sup><p class="last">MCF-7 proliferation assay in MEM medium containing 10% FBS; 5 day incubation (<i>n</i> ≥ 3).</p></div><div class="footnote" id="t3fn4"><sup><sup>d</sup></sup><p class="last">ND = not determined.</p></div></div></div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. ERα Binding, ERα Downregulation Degradation, and MCF-7 Antiproliferative Activities for <b>D</b> Series Compounds</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center">ERα binding IC<sub>50</sub> (nM)<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center">ERα degradation EC<sub>50</sub> (nM)<a class="ref internalNav" href="#t4fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center">MCF-7 IC<sub>50</sub> (nM)<a class="ref internalNav" href="#t4fn3" aria-label="c">c</a></th><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center">ERα binding IC<sub>50</sub> (nM)<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center">ERα degradation EC<sub>50</sub> (nM)<a class="ref internalNav" href="#t4fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center">MCF-7 IC<sub>50</sub> (nM)<a class="ref internalNav" href="#t4fn3" aria-label="c">c</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>D1</b></td><td class="colsep0 rowsep0" align="left">100.8 ± 2.80</td><td class="colsep0 rowsep0" align="left">3.23 ± 0.09</td><td class="colsep0 rowsep0" align="left">16.0 ± 0.19</td><td class="colsep0 rowsep0" align="left"><b>D2</b></td><td class="colsep0 rowsep0" align="left">25.4 ± 0.19</td><td class="colsep0 rowsep0" align="left">1.00 ± 0.06</td><td class="colsep0 rowsep0" align="left">10.2 ± 0.12</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>D3</b></td><td class="colsep0 rowsep0" align="left">120.7 ± 3.06</td><td class="colsep0 rowsep0" align="left">3.48 ± 0.06</td><td class="colsep0 rowsep0" align="left">16.3 ± 0.62</td><td class="colsep0 rowsep0" align="left"><b>D4</b></td><td class="colsep0 rowsep0" align="left">46.4 ± 1.06</td><td class="colsep0 rowsep0" align="left">2.51 ± 0.11</td><td class="colsep0 rowsep0" align="left">2.07 ± 0.08</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>D5</b></td><td class="colsep0 rowsep0" align="left">130.8 ± 2.45</td><td class="colsep0 rowsep0" align="left">0.48 ± 0.02</td><td class="colsep0 rowsep0" align="left">1.23 ± 0.06</td><td class="colsep0 rowsep0" align="left"><b>D6</b></td><td class="colsep0 rowsep0" align="left">50.8 ± 0.98</td><td class="colsep0 rowsep0" align="left">4.67 ± 0.25</td><td class="colsep0 rowsep0" align="left">12.8 ± 1.40</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>D7</b></td><td class="colsep0 rowsep0" align="left">72.9 ± 2.28</td><td class="colsep0 rowsep0" align="left">6.85 ± 0.05</td><td class="colsep0 rowsep0" align="left">21.0 ± 0.28</td><td class="colsep0 rowsep0" align="left"><b>D8</b></td><td class="colsep0 rowsep0" align="left">19.8 ± 0.45</td><td class="colsep0 rowsep0" align="left">0.53 ± 0.03</td><td class="colsep0 rowsep0" align="left">0.89 ± 0.07</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>D9</b></td><td class="colsep0 rowsep0" align="left">29.5 ± 0.94</td><td class="colsep0 rowsep0" align="left">1.47 ± 0.03</td><td class="colsep0 rowsep0" align="left">8.63 ± 0.16</td><td class="colsep0 rowsep0" align="left"><b>D10</b></td><td class="colsep0 rowsep0" align="left">88.9 ± 2.14</td><td class="colsep0 rowsep0" align="left">7.80 ± 0.52</td><td class="colsep0 rowsep0" align="left">1.03 ± 0.05</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>D11</b></td><td class="colsep0 rowsep0" align="left">25.2 ± 0.56</td><td class="colsep0 rowsep0" align="left">0.98 ± 0.10</td><td class="colsep0 rowsep0" align="left">7.48 ± 0.52</td><td class="colsep0 rowsep0" align="left"><b>D12</b></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left">ND</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>D13</b></td><td class="colsep0 rowsep0" align="left">15.6 ± 0.58</td><td class="colsep0 rowsep0" align="left">0.22 ± 0.01</td><td class="colsep0 rowsep0" align="left">2.03 ± 0.12</td><td class="colsep0 rowsep0" align="left"><b>D14</b></td><td class="colsep0 rowsep0" align="left">17.1 ± 0.52</td><td class="colsep0 rowsep0" align="left">0.23 ± 0.02</td><td class="colsep0 rowsep0" align="left">1.69 ± 0.04</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>D15</b></td><td class="colsep0 rowsep0" align="left">3.2 ± 0.08</td><td class="colsep0 rowsep0" align="left">0.10 ± 0.01</td><td class="colsep0 rowsep0" align="left">1.54 ± 0.07</td><td class="colsep0 rowsep0" align="left"><b>D16</b></td><td class="colsep0 rowsep0" align="left">21.8 ± 0.44</td><td class="colsep0 rowsep0" align="left">0.61 ± 0.04</td><td class="colsep0 rowsep0" align="left">2.60 ± 0.09</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>D17</b></td><td class="colsep0 rowsep0" align="left">19.8 ± 0.42</td><td class="colsep0 rowsep0" align="left">0.55 ± 0.06</td><td class="colsep0 rowsep0" align="left">4.34 ± 0.21</td><td class="colsep0 rowsep0" align="left"><b>D18</b></td><td class="colsep0 rowsep0" align="left">17.6 ± 0.06</td><td class="colsep0 rowsep0" align="left">1.51 ± 0.12</td><td class="colsep0 rowsep0" align="left">2.47 ± 0.11</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>D19</b></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left">ND<a class="ref internalNav" href="#t4fn4" aria-label="d">d</a></td><td class="colsep0 rowsep0" align="left"><b>D20</b></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left">ND</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>D21</b></td><td class="colsep0 rowsep0" align="left">110.4 ± 4.20</td><td class="colsep0 rowsep0" align="left">6.27 ± 0.13</td><td class="colsep0 rowsep0" align="left">6.98 ± 0.45</td><td class="colsep0 rowsep0" align="left"><b>D22</b></td><td class="colsep0 rowsep0" align="left">28.8 ± 0.96</td><td class="colsep0 rowsep0" align="left">1.28 ± 0.10</td><td class="colsep0 rowsep0" align="left">0.66 ± 0.03</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>D23</b></td><td class="colsep0 rowsep0" align="left">30.4 ± 0.98</td><td class="colsep0 rowsep0" align="left">1.56 ± 0.08</td><td class="colsep0 rowsep0" align="left">0.53 ± 0.10</td><td class="colsep0 rowsep0" align="left"><b>D24</b></td><td class="colsep0 rowsep0" align="left">17.1 ± 0.52</td><td class="colsep0 rowsep0" align="left">0.30 ± 0.01</td><td class="colsep0 rowsep0" align="left">0.27 ± 0.02</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>D25</b></td><td class="colsep0 rowsep0" align="left">19.0 ± 0.60</td><td class="colsep0 rowsep0" align="left">0.49 ± 0.06</td><td class="colsep0 rowsep0" align="left">1.19 ± 0.08</td><td class="colsep0 rowsep0" align="left"><b>D26</b></td><td class="colsep0 rowsep0" align="left">26.2 ± 1.03</td><td class="colsep0 rowsep0" align="left">1.08 ± 0.15</td><td class="colsep0 rowsep0" align="left">1.48 ± 0.04</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>D27</b></td><td class="colsep0 rowsep0" align="left">17.4 ± 0.18</td><td class="colsep0 rowsep0" align="left">0.26 ± 0.02</td><td class="colsep0 rowsep0" align="left">2.47 ± 0.32</td><td class="colsep0 rowsep0" align="left"><b>D28</b></td><td class="colsep0 rowsep0" align="left">19.6 ± 0.65</td><td class="colsep0 rowsep0" align="left">0.50 ± 0.02</td><td class="colsep0 rowsep0" align="left">1.27 ± 0.06</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>D29</b></td><td class="colsep0 rowsep0" align="left">20.0 ± 0.16</td><td class="colsep0 rowsep0" align="left">0.70 ± 0.04</td><td class="colsep0 rowsep0" align="left">2.91 ± 0.28</td><td class="colsep0 rowsep0" align="left"><b>D30</b></td><td class="colsep0 rowsep0" align="left">19.8 ± 0.70</td><td class="colsep0 rowsep0" align="left">0.54 ± 0.01</td><td class="colsep0 rowsep0" align="left">1.22 ± 0.04</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>D31</b></td><td class="colsep0 rowsep0" align="left">17.1 ± 0.32</td><td class="colsep0 rowsep0" align="left">0.25 ± 0.02</td><td class="colsep0 rowsep0" align="left">1.08 ± 0.10</td><td class="colsep0 rowsep0" align="left"><b>D32</b></td><td class="colsep0 rowsep0" align="left">32.9 ± 0.85</td><td class="colsep0 rowsep0" align="left">1.92 ± 0.08</td><td class="colsep0 rowsep0" align="left">10.8 ± 0.16</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>D33</b></td><td class="colsep0 rowsep0" align="left">22.4 ± 0.72</td><td class="colsep0 rowsep0" align="left">0.66 ± 0.06</td><td class="colsep0 rowsep0" align="left">3.22 ± 0.08</td><td class="colsep0 rowsep0" align="left"><b>D34</b></td><td class="colsep0 rowsep0" align="left">32.2 ± 1.26</td><td class="colsep0 rowsep0" align="left">1.70 ± 0.06</td><td class="colsep0 rowsep0" align="left">5.91 ± 0.09</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>D35</b></td><td class="colsep0 rowsep0" align="left">19.8 ± 0.50</td><td class="colsep0 rowsep0" align="left">0.58 ± 0.03</td><td class="colsep0 rowsep0" align="left">2.35 ± 0.05</td><td class="colsep0 rowsep0" align="left"><b>D36</b></td><td class="colsep0 rowsep0" align="left">31.2 ± 0.32</td><td class="colsep0 rowsep0" align="left">1.61 ± 0.10</td><td class="colsep0 rowsep0" align="left">5.60 ± 0.07</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>D37</b></td><td class="colsep0 rowsep0" align="left">18.4 ± 0.36</td><td class="colsep0 rowsep0" align="left">0.43 ± 0.02</td><td class="colsep0 rowsep0" align="left">2.05 ± 0.02</td><td class="colsep0 rowsep0" align="left"><b>D38</b></td><td class="colsep0 rowsep0" align="left">26.6 ± 1.04</td><td class="colsep0 rowsep0" align="left">0.85 ± 0.05</td><td class="colsep0 rowsep0" align="left">2.85 ± 0.05</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>D39</b></td><td class="colsep0 rowsep0" align="left">18.3 ± 0.20</td><td class="colsep0 rowsep0" align="left">0.34 ± 0.01</td><td class="colsep0 rowsep0" align="left">1.49 ± 0.04</td><td class="colsep0 rowsep0" align="left"><b>D40</b></td><td class="colsep0 rowsep0" align="left">19.8 ± 0.40</td><td class="colsep0 rowsep0" align="left">0.61 ± 0.04</td><td class="colsep0 rowsep0" align="left">1.34 ± 0.16</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>AZD9496</b></td><td class="colsep0 rowsep0" align="left">4.60 ± 0.04</td><td class="colsep0 rowsep0" align="left">0.23 ± 0.05</td><td class="colsep0 rowsep0" align="left">0.76 ± 0.06</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup><sup>a</sup></sup><p class="last">Relative binding activity was determined by a Polar Screen ERα competitor Assay system from Invitrogen (no. A15883) with 150 nM full-length human ERα.</p></div><div class="footnote" id="t4fn2"><sup><sup>b</sup></sup><p class="last">ERα In-Cell Western in MCF-7 breast cancer cells cultured with DMEM containing 10% hormone-free FBS; readings taken after 4 h of incubation using an anti-ERα rabbit monoclonal antibody (<i>n</i> ≥ 3).</p></div><div class="footnote" id="t4fn3"><sup><sup>c</sup></sup><p class="last">MCF-7 proliferation assay in MEM medium containing 10% FBS; 5 day incubation (<i>n</i> ≥ 3).</p></div><div class="footnote" id="t4fn4"><sup><sup>d</sup></sup><p class="last">ND = not determined.</p></div></div></div><div class="NLM_p">Meanwhile, when the various fragments were incorporated into the piperidine region of <b>AZD9496</b>, compounds <b>B1</b>–<b>B3</b> exhibited potent ER binding affinity, ERα degradation efficiency, and MCF-9 antiproliferative activity. Because <b>B3</b> contained a large volume heterocycle, it strengthened the hydrophobic interaction with the hot-spot residues, which revealed the best biological activities among the tested compounds (ERα binding IC<sub>50</sub> = 8.6 nM; ERα degradation EC<sub>50</sub> = 0.351 nM; MCF-7 IC<sub>50</sub> = 0.544 nM). This finding was consistent with the MM-GBSA calculation results. However, compared to the reference compound <b>AZD9496</b>, the biological activity of <b>B3</b> was relatively lower; thus, we did not select it for further biological evaluation.</div><div class="NLM_p">Besides, the 2-fluoro-2-methylpropyl group was substituted by other linkers at the same region, and the biological results demonstrated that the linker had notable effects on the ERα binding affinity. For example, compound <b>AZD9496</b> with 2-fluoro-2-methylpropyl exhibited the strongest binding affinity with protein (IC<sub>50</sub> = 4.6 nM), while replacing with bulkier cyclopropyl (<b>C1</b>) and long butyl (<b>C2</b>) led to the decrease of interaction between ligand and protein. Especially, compounds <b>C3</b> and <b>C4</b> almost lost the biological activities and were not inhibited by the growth and proliferation of MCF-7 cancer cells.</div><div class="NLM_p">Initially, the 2,6-difluorophenyl was surrounded by hydrophobic residues Trp383, Leu525, and Val534; thus, we further expanded the search for alternative degradation motifs by investigating the 2,6-difluorophenyl fragment of <b>AZD9496</b> replaced by various heterocyclic substituents (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>). Once the 2,6-difluorophenyl was substituted by a pyridine group, the binding affinity and degradation efficiency of compound <b>D1</b> were significantly decreased, leading to the cellular activity being reduced (IC<sub>50</sub> = 100.8 nM; EC<sub>50</sub> = 3.23 nM; MCF-7 IC<sub>50</sub> = 15.97 nM). The fluorine atom at phenyl was beneficial for the protein–ligand interaction and protein degradation. Moreover, the fluorine also enhanced the membrane permeability. As a result, adding the fluorine into the pyridine ring (<b>D2</b>) restored the biological activity (IC<sub>50</sub> = 25.4 nM; EC<sub>50</sub> = 1.0 nM; MCF-7 IC<sub>50</sub> = 10.23 nM). On the contrary, while the fluorine was replaced by other groups such as chlorine or methoxy, the potency was lost slightly compared to positive control <b>AZD9496</b> (<b>D4</b>, <b>D5</b>, or <b>D6</b> vs <b>AZD9496</b>). Furthermore, the positions of fluorine atoms were also critical to protein–ligand interaction; the potency of compound <b>D3</b> was obviously weaker than that of <b>AZD9496</b> (IC<sub>50</sub> = 120.7 nM; EC<sub>50</sub> = 3.48 nM; MCF-7 IC<sub>50</sub> = 16.25 nM). Thus, the 2,6-difluorophenyl scaffold was essential to the ERα binding and degradation.</div><div class="NLM_p">Because the acrylate side chain forms a critical hydrogen-bond interaction with Leu536 and Tyr537 (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5ACC">5ACC</a>), which led to the conformational change to the AF2 domain and promotion of ERα degradation, most SERDs in clinical trials are acrylates. However, the potential reaction ability of acrylate to Michael addition and the metabolism problem stimulated us to explore novel scaffolds.<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> First, we modified the acrylate acid chain to a diverse library of aliphatic acids (<b>D7</b>–<b>D11</b>); the four-membered cyclobutene moiety was favorable for the ERα degradation and MCF-7 antiproliferation through the investigation of ring size (<b>D8</b> vs <b>D7</b>, <b>D9</b>, and <b>D10</b>). Then, the diazaspiro heptane fragment occupying a space similar to that of the piperazine ring was introduced on the 2,6-difluoro-phenyl, which enabled the exploration of the novel heterocycle scaffold binding with ERα LBD. Compound <b>D11</b> containing diazaspiro heptane demonstrated better ERα degradation efficiency and MCF-7 cellular inhibitory effect than the corresponding compound <b>D10</b> (<b>D11</b>: IC<sub>50</sub> = 25.2 nM; EC<sub>50</sub> = 0.98 nM; MCF-7 IC<sub>50</sub> = 7.48 nM). Subsequently, we envisioned that the acrylate acid could be identified as a negative center to form electrostatic interaction with H12. However, when the carboxylic acid was replaced by another electron-deficient group (<b>D12</b>), the ERα degradation ability was almost lost. Most importantly, if acrylate acid chain was replaced by other electron-withdrawing groups <b>D13</b>–<b>D15</b>, the ERα degradation efficiencies of <b>D13</b>, <b>D14</b>, and <b>D15</b> were equivalent to reference compound <b>AZD9496</b> (<b>D13</b>, EC<sub>50</sub> = 0.22 nM; <b>D14</b>, EC<sub>50</sub> = 0.23 nM; <b>D15</b>, EC<sub>50</sub> = 0.10 nM), which indicated the importance of exploring novel groups at this region. Besides, other non-carboxylic acid substituents linked to 2,6-difluorophenyl (<b>D16</b>–<b>D23</b>) were not improving the potencies, and some of them (<b>D19</b> and <b>D20</b>) were inactivated by the ERα degradation. In conclusion, the biological activity of <b>AZD9496</b> containing an acrylate side chain was more favorable than other degraders <b>D7</b>–<b>D23</b> without an acrylate substituent.</div><div class="NLM_p">On the basis of scaffold hopping strategy, we further expanded the searches for alternative degradation motifs by investigating the acrylic acid side chain of <b>AZD9496</b> replaced by various heterocyclic substituents. Gratifyingly, as the acrylic acid was cyclized to benzoic acid, the potency of compound <b>D24</b> was significantly enhanced. Although the interaction between ERα and <b>D24</b> was weaker than <b>AZD9496</b>, the degradation ability of <b>D24</b> was close to that of <b>AZD9496</b>; even the MCF-7 antiproliferative activity of <b>D24</b> was better than that of <b>AZD9496</b> (<b>D24</b>: IC<sub>50</sub> = 17.1 nM; EC<sub>50</sub> = 0.30 nM; MCF-7 IC<sub>50</sub> = 0.27 nM). To better understand the binding pattern of <b>D24</b> and ERα, the molecular docking analysis of <b>D24</b> in the binding pocket of ERα was performed (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5ACC">5ACC</a>, <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>). As expected, the docking orientation and interaction of <b>D24</b> fitted as well as the <b>AZD9496</b>; <b>D24</b> formed stronger binding affinity with hot spot 1 and hot spot 2 residues (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>: <b>F11</b>). Thus, considering the excellent in vitro ERα degradation activity and inhibitory activity of cell viability of compound <b>D24</b>, we carried out further SAR studies and structure optimization of <b>D24</b>.</div><figure id="fig5" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00280/20210602/images/medium/jm1c00280_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00280/20210602/images/large/jm1c00280_0005.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Docking conformation (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5ACC">5ACC</a>) of <b>D24</b> (green) superimposed onto cocrystal structure of <b>AZD9496</b> (pink).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00280/20210602/images/large/jm1c00280_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00280&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The benzoic acid moiety of <b>D24</b> was considered to extend out of the ligand binding pocket between α-helices 3 (H3) and 11 (H11); thus the binding affinity was increased.<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> To enrich the structure types of <b>D24</b> and investigate the structure–activity relationship, we supposed that other aromatic rings should occupy similar hydrophobic interaction with target as benzene ring on the benzoic acid fragment. Different heterocycles involving thiazole (<b>D25</b>), amino thiazole (<b>D26</b>), pyrazole (<b>D27</b>), and pyridine (<b>D28</b>) were examined; as a result, once the benzene ring was replaced by another aromatic fragment, the antitumor activities were not improved (<b>D25</b>, <b>D26</b>, <b>D27</b>, and <b>D28</b> vs <b>D24</b>). Therefore, the benzoic acid moiety was beneficial to ERα degradation, and it could not be replaced by other substituents.</div><div class="NLM_p last">To enrich the SAR of <b>D24</b>, the compound libraries of <b>D29</b>–<b>D40</b> were designed, synthesized, and biologically evaluated. In general, the substitution of the aryl ring (<b>D30</b>–<b>D32</b>, <b>D34</b>), the substitution of the carboxyl to the meta-position (<b>D29</b>), and the carboxyl bioisostered to amide (<b>D33</b>) led to slight reductions of the degradation abilities and MCF-7 antiproliferative activities. Furthermore, the biological activity of <b>D35</b>–<b>D40</b> also revealed that 2,6-difluorophenyl was essential to potencies of SERDs. When the acrylate side chain of <b>AZD9496</b> was replaced by the benzoic acid moiety of <b>D24</b>, <b>D24</b> also possessed good ERα binding affinity, ERα degradation efficacy, and antiproliferative activity; thus <b>D24</b> was selected for further biological evaluation.</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Pharmacokinetics and Pharmacodynamics of <b>D24</b></h3><div id="sec2_5_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i17" class="anchor-spacer"></div><h4 class="article-section__title" id="_i17"> Western Blot Analysis of ERα Degradation In Vitro</h4><div class="NLM_p">To validate the possible mechanisms of ERα degradation action mediated by compound <b>D24</b>, the Western blot analyses of <b>AZD9496</b>, <b>D24</b>, and <b>estradiol (E2)</b> were performed (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>). The results indicated that both <b>D24</b> and <b>AZD9496</b> were identified as potent degraders of ERα in MCF-7 cells at 5 nM. In addition, <b>D24</b> possessed potent degradation activity at 0.6 nM, and the degradation mechanism of <b>D24</b> was similar to that of <b>AZD9496</b>, which indicated that <b>D24</b> and <b>AZD9496</b> could not fully degrade ERα in MCF-7 cells even at a higher biochemical concentration, resulting in no significant difference of degradation efficiency difference at concentrations of 5 and 40 nM.</div><figure id="fig6" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00280/20210602/images/medium/jm1c00280_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00280/20210602/images/large/jm1c00280_0006.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Degradation of ERα by compound <b>AZD9496</b>, <b>D24</b>, and estradiol (<b>E2</b>) by Western blot analysis. MCF-7 cells were incubated with <b>AZD9496</b>, <b>D24</b>, or estradiol at the doses above. Total protein was extracted at 48 h incubation with compound and immunoblotting performed to detect effects on ERα and PR (progesterone receptor) levels.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00280/20210602/images/large/jm1c00280_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00280&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_5_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i18" class="anchor-spacer"></div><h4 class="article-section__title" id="_i18"> In Vivo Efficacy in the MCF-7 Breast Cancer Xenograft Model</h4><div class="NLM_p">To further evaluate the antitumor activity of <b>D24</b>, efficacy was assessed in the MCF-7 human breast cancer xenograft model in nude. Treatment of the nude with <b>D24</b> at 2 mg/kg daily led to the significant tumor growth inhibition as compared to the <b>AZD9496</b> group dose at 6 mg/kg (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>a). Notably no significant weight fluctuations were observed during the whole course (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>b). The results indicated that <b>D24</b> could be developed as a potent SERD for the therapy agents of breast cancer.</div><figure id="fig7" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00280/20210602/images/medium/jm1c00280_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00280/20210602/images/large/jm1c00280_0007.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. In vivo efficacy and ERα by <b>AZD9496</b> and <b>D24</b> in the MCF-7 xenograft model. (a) Tumor growth inhibition (TGI) dose response upon oral administration of compounds <b>AZD9496</b> and <b>D24</b> (<i>n</i> = 6 per dose group, QD × 21days); (b) Nude mouse body weight measured every 2 days.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00280/20210602/images/large/jm1c00280_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00280&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_5_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i19" class="anchor-spacer"></div><h4 class="article-section__title" id="_i19"> In Vivo Pharmacokinetic Study of <b>D24</b></h4><div class="NLM_p">In view of the excellent <i>in vitro</i> ERα degradation ability and antiproliferative inhibitory of <b>D24</b>, and <i>in vivo</i> Efficacy of <b>D24</b> in the MCF-7 Breast Cancer Xenograft Model, the pharmacokinetic (PK) parameters of <b>D24</b> were evaluated in rats (<a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>) to assess whether sufficient exposure for toxicological studies could be achieved. Compound <b>D24</b> was performed in rats via IV (1 mg/kg) and PO (2 mg/kg) administration. While dosing of 2 mg/kg solution of <b>D24</b> to rats, the bioavailability of <b>D24</b> was attained to 80.5% that was twice of the positive control <b>AZD9496</b>. Meanwhile, the plasma exposure (AUC) of <b>D24</b> was 16073.7 h*ng/mL, and the half-life period was 12.1 h, which indicated that <b>D24</b> possessed better pharmacokinetic properties than <b>AZD9496</b>. As for the intravenous administration, <b>D24</b> also exhibited favorable drug exposure and low clearance, revealing that <b>D24</b> also had better pharmacokinetic profiles than <b>AZD9496</b>. Those results revealed that <b>D24</b> had an acceptable PK profile.</div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. Pharmacokinetic Study of Compounds <b>AZD9496</b> and <b>D24</b></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center">dose (mg/kg)</th><th class="colsep0 rowsep0" align="center" char="."><i>t</i><sub>1/2</sub> (h)</th><th class="colsep0 rowsep0" align="center"><i>t</i><sub>max</sub> (h)</th><th class="colsep0 rowsep0" align="center" char="."><i>C</i><sub>max</sub> (ng/mL)</th><th class="colsep0 rowsep0" align="center" char=".">AUC<sub>0–<i>t</i></sub> (h·ng/mL)<a class="ref internalNav" href="#t5fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center" char=".">MRT<sub>0–<i>t</i></sub> (h)<a class="ref internalNav" href="#t5fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center" char=".">Cl ((L/h)/kg)<a class="ref internalNav" href="#t5fn3" aria-label="c">c</a></th><th class="colsep0 rowsep0" align="center" char="."><i>V</i><sub>d</sub> (L/kg)</th><th class="colsep0 rowsep0" align="center" char="."><i>F</i> (%)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>AZD9496</b></td><td class="colsep0 rowsep0" align="left">2 p.o.</td><td class="colsep0 rowsep0" align="char" char=".">1.6 ± 0.1</td><td class="colsep0 rowsep0" align="left">0.6</td><td class="colsep0 rowsep0" align="char" char=".">532.8 ± 77.2</td><td class="colsep0 rowsep0" align="char" char=".">1355.7 ± 210.8</td><td class="colsep0 rowsep0" align="char" char=".">1.9</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char=".">41.6 ± 6.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">1 i.v.</td><td class="colsep0 rowsep0" align="char" char=".">1 ± 0.1</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="char" char=".">2988.9 ± 414.0</td><td class="colsep0 rowsep0" align="char" char=".">1627.6 ± 214.8</td><td class="colsep0 rowsep0" align="char" char=".">0.8</td><td class="colsep0 rowsep0" align="char" char=".">0.6 ± 0.1</td><td class="colsep0 rowsep0" align="char" char=".">0.9 ± 0.1</td><td class="colsep0 rowsep0" align="char" char="."> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>D24</b></td><td class="colsep0 rowsep0" align="left">2 p.o.</td><td class="colsep0 rowsep0" align="char" char=".">12.1 ± 2.7</td><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="char" char=".">1277.7 ± 91.6</td><td class="colsep0 rowsep0" align="char" char=".">16073.7 ± 835.9</td><td class="colsep0 rowsep0" align="char" char=".">8.7</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char=".">80.5 ± 4.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">1 i.v.</td><td class="colsep0 rowsep0" align="char" char=".">13.8 ± 1.5</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="char" char=".">2913.4 ± 904.7</td><td class="colsep0 rowsep0" align="char" char=".">9986.9 ± 332.6</td><td class="colsep0 rowsep0" align="char" char=".">8.8</td><td class="colsep0 rowsep0" align="char" char=".">0.1</td><td class="colsep0 rowsep0" align="char" char=".">1.4</td><td class="colsep0 rowsep0" align="char" char="."> </td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t5fn1"><div class="footnote" id="t5fn1"><sup><sup>a</sup></sup><p class="last">Area under the curve.</p></div><div class="footnote" id="t5fn2"><sup><sup>b</sup></sup><p class="last">Mean retention time.</p></div><div class="footnote" id="t5fn3"><sup><sup>c</sup></sup><p class="last">Total body clearance.</p></div></div></div></div><div id="sec2_5_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> In Vitro DMPK Profiles of <b>D24</b></h4><div class="NLM_p">In order to systematically assess the druggability, the physicochemical properties of <b>D24</b> was assayed (<a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">Table <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a></a>). To evaluate the metabolic stability, compound <b>AZD9496</b> and <b>D24</b> were incubated with liver microsomes of different species (mouse, rat, dog, monkey and human). As for human, dog and rat, <b>D24</b> possessed stronger metabolic stability than <b>AZD9496</b>, which proved that the druggability of benzoic acid motifs better than acrylic acid group. Meanwhile, the plasma stability and binding rate of plasma protein for <b>D24</b> in different species were similar, and the data of <b>D24</b> was also approximately to <b>AZD9496</b>. By screening a panel of eight cytochrome P450 enzymes, the compound <b>D24</b> revealed no inhibition activity (IC<sub>50</sub> > 30 μM) against six isoforms (CYP1A2, CYP2C9, CYP2C19, CYP2D6, CYP3A4-M, CYP3A4-T), however, the modest inhibition against CYP2B6 and CYP2C8 were found in both <b>D24</b> and <b>AZD9496</b>.</div><div class="NLM_table-wrap" id="tbl6"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 6. In Vitro DMPK Evaluation of <b>AZD9496</b> and <b>D24</b></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center">compd <b>AZD9496</b></th><th class="colsep0 rowsep0" align="center">compd <b>D24</b></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="3" align="center">Metabolic Stability in Liver Microsomes (<i>T</i><sub>1/2</sub> (min))</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">mouse</td><td class="colsep0 rowsep0" align="left"><38</td><td class="colsep0 rowsep0" align="left">27</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">rat</td><td class="colsep0 rowsep0" align="left"><17.3</td><td class="colsep0 rowsep0" align="left">27.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">human</td><td class="colsep0 rowsep0" align="left">19.8</td><td class="colsep0 rowsep0" align="left">20.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">dog</td><td class="colsep0 rowsep0" align="left"><13.8</td><td class="colsep0 rowsep0" align="left">38.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">monkey</td><td class="colsep0 rowsep0" align="left">31.1</td><td class="colsep0 rowsep0" align="left">25.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="3" align="center">Plasma Stability (<i>T</i><sub>1/2</sub> min)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">mouse</td><td class="colsep0 rowsep0" align="left">>289.1</td><td class="colsep0 rowsep0" align="left">>289.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">rat</td><td class="colsep0 rowsep0" align="left">>289.1</td><td class="colsep0 rowsep0" align="left">>289.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">human</td><td class="colsep0 rowsep0" align="left">>289.1</td><td class="colsep0 rowsep0" align="left">>289.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">dog</td><td class="colsep0 rowsep0" align="left">>289.1</td><td class="colsep0 rowsep0" align="left">>289.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">monkey</td><td class="colsep0 rowsep0" align="left">>289.1</td><td class="colsep0 rowsep0" align="left">>289.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="3" align="center">Binding Rate of Plasma Protein (%)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">human</td><td class="colsep0 rowsep0" align="left">99.79</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">mouse</td><td class="colsep0 rowsep0" align="left">99.73</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">rat</td><td class="colsep0 rowsep0" align="left">99.66</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">dog</td><td class="colsep0 rowsep0" align="left">99.91</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">monkey</td><td class="colsep0 rowsep0" align="left">99.69</td><td class="colsep0 rowsep0" align="left">99.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="3" align="center">CYP Inhibition IC<sub>50</sub> (μM)<a class="ref internalNav" href="#t6fn1" aria-label="a">a</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">1A2</td><td class="colsep0 rowsep0" align="left">>30</td><td class="colsep0 rowsep0" align="left">>30</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">2B6</td><td class="colsep0 rowsep0" align="left">21.78</td><td class="colsep0 rowsep0" align="left">19.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">2C8</td><td class="colsep0 rowsep0" align="left">4.7</td><td class="colsep0 rowsep0" align="left">9.22</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">2C9</td><td class="colsep0 rowsep0" align="left">24.19</td><td class="colsep0 rowsep0" align="left">>30</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">2C19</td><td class="colsep0 rowsep0" align="left">>30</td><td class="colsep0 rowsep0" align="left">>30</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">2D6</td><td class="colsep0 rowsep0" align="left">>30</td><td class="colsep0 rowsep0" align="left">>30</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">3A4-M</td><td class="colsep0 rowsep0" align="left">>30</td><td class="colsep0 rowsep0" align="left">>30</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">3A4-T</td><td class="colsep0 rowsep0" align="left">>30</td><td class="colsep0 rowsep0" align="left">>30</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="3" align="center">hERG IC<sub>50</sub> (μM)<a class="ref internalNav" href="#t6fn2" aria-label="b">b</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">>30</td><td class="colsep0 rowsep0" align="left">>30</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t6fn1"><div class="footnote" id="t6fn1"><sup><sup>a</sup></sup><p class="last">Inhibition of cytochrome P450 enzymes IC<sub>50</sub>.</p></div><div class="footnote" id="t6fn2"><sup><sup>b</sup></sup><p class="last">Inhibition of the hERG tail current was measured using a plate-based planar patch-clamp system.</p></div></div></div><div class="NLM_p last">To further understand the metabolism of <b>D24</b>, we employed an in vitro model of liver microsomes to incubate <b>D24</b> and then analyzed the metabolites of <b>D24</b> in different species with UPLC-UV/Q-TOF MS (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00280/suppl_file/jm1c00280_si_001.pdf" class="ext-link">Supporting Information</a>). The results of the metabolism of <b>D24</b> revealed that the main metabolic pathways of <b>D24</b> were oxidation and amino dealkylation; the proposed metabolic pathways of <b>D24</b> were shown in <a href="/doi/suppl/10.1021/acs.jmedchem.1c00280/suppl_file/jm1c00280_si_001.pdf" class="ext-link">Supporting Information</a>. The metabolism of <b>D24</b> in human model was appropriate to the monkey, and it was also found that few variations among hepatic metabolic pathways of <b>D24</b> in dogs, monkeys, and humans.</div></div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> Toxicity Study of Compound <b>D24</b></h3><div class="NLM_p">As a preclinical evaluation of <b>D24</b>, the toxicity study of <b>D24</b> was performed. Because the potential cardiotoxicity was detrimental for the drug candidate into clinic trial, thus it was incredible to evaluate the hERG toxicity of compound in the preclinical phase, whereafter the patch-clamp experiment was utilized to determine hERG inhibitory effect of compounds <b>AZD9496</b> and <b>D24</b> with cisapride as the positive control. As shown in <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">Table <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a></a>, the hERG inhibitory values of <b>AZD9496</b> and <b>D24</b> were higher than 30 μM, which indicated that <b>D24</b> had a relatively low risk of hERG toxicity, even no cardiotoxicity (IC<sub>50</sub> > 10 μM, very weak inhibition or no inhibition).<a onclick="showRef(event, 'ref49 ref50'); return false;" href="javascript:void(0);" class="ref ref49 ref50">(49,50)</a></div><div class="NLM_p last">Besides, to further assess of the safety of <b>D24</b> in vivo, we treated SD rats with <b>D24</b> and carried out maximum tolerated dose (MTD) determination. The animals were given 17.5 or 35 mg/kg <b>D24</b> by gavage in a single dose for 14 days. No animal death, obvious pathological changes, and weight loss were observed after high-dose administration (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00280/suppl_file/jm1c00280_si_001.pdf" class="ext-link">Supporting Information</a>). Thus, compound <b>D24</b> exhibited lower toxicity by oral administration, and the drug safety window of <b>D24</b> also proved that <b>D24</b> had relatively good druggability.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i24">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_30540" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_30540" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Most breast cancers are estrogen receptor α positive (ERα+); however, resistance to current endocrine therapy has been emerging. Although a variety of resistance mechanisms have been elucidated, there is evidence that ERα still plays the central role in many cases; thus the utilization of selective estrogen receptor degraders (SERDs) to block ERα is an effective method to overcome resistance. The steroid-based antiestrogen fulvestrant, the only approved SERD, is effective in metastatic breast cancer, but limited by poor pharmacokinetics, prompting the discovery of favorable pharmacokinetics and orally bioavailable SERDs, which are still in demand.</div><div class="NLM_p last">In this study, we carry out molecular dynamic simulation technology to analyze the hot-spot residues for protein–ligand interaction between SERDs and ERα; subsequently focusing on the hot-spot residues, diversity of the compound library is designed and synthesized. Among them, while the acrylic acid moiety of <b>AZD9496</b> is modified to benzoic acid, <b>D24</b> exhibits better antitumor activity than <b>AZD9496</b> in vitro (IC<sub>50</sub> = 17.1 nM; EC<sub>50</sub> = 0.30 nM; MCF-7 IC<sub>50</sub> = 0.27 nM). Western blot analysis proves the antitumor mechanism of <b>D24</b>; additionally, <b>D24</b> also displays good antitumor efficacy in the MCF-7 human breast cancer xenograft model in vivo. The favorable pharmacokinetic properties, especially good oral bioavailability and efficacious exposures, together with its excellent druggability and good safety property, provide a favorable profile of <b>D24</b>. Thus, <b>D24</b> can be selected as a drug candidate of SERDs for further evaluation; the related research works will be published in due course.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i25">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_72252" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_72252" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> General Procedures</h3><div class="NLM_p last">Unless otherwise specified, reagents were purchased from commercial suppliers and used without further purification. The reaction flask is dried at 100 °C in an oven, and then the reaction device was set up with hot and cooled under an argon atmosphere. All reactions proceeded under the protection of an argon stream. The reaction process was monitored by analytical thin layer chromatography and a UV lamp (254 and 365 nm). A Bruker Biospin AV400 (400 MHz, <sup>1</sup>H NMR; 100 MHz, <sup>13</sup>C NMR) instrument was used to measure the <sup>1</sup>H NMR and <sup>13</sup>C NMR spectra. Chemical shifts are reported in ppm (parts per million) and are referenced to either TMS or the solvent. The purity of all compounds for biological testing was determined using an HPLC and was identified as >95% purity. HPLC conditions: Xselect CSH C18 (4.6 × 150 mm<sup>2</sup>, 3.5 μm); 1–2 min, 90:10 (%) 0.1%TFA:ACN, gradient, 15–18 min, 10:90 (%) 0.1%TFA:ACN; flow rate, 1.2 mL/min; detection, UV 254 nm (unless otherwise noted).</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> General Procedures for the Synthesis of Compound <b>A1</b></h3><div id="sec4_2_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> (<i>E</i>)-3-(3,5-Difluoro-4-((1<i>R</i>,3<i>R</i>)-2-(2-fluoro-2-methylpropyl)-3,9-dimethyl-2,3,4,4<i>a</i>,9,9<i>a</i>-hexahydro-1<i>H</i>-pyrido[3,4-<i>b</i>]indol-1-yl)phenyl)acrylic acid (<b>A1</b>)</h4><div class="NLM_p last">To a suspension of NaH (0.0394 g, 1 mmol) in tetrahydrofuran (5 mL) was added a solution of <b>AZD9496</b> (0.293 g, 0.66 mmol) in THF (5 mL) dropwise in an ice bath. The mixture was stirred for 0.5 h followed by addition of MeI (50 μL, 0.66 mmol). The mixture was stirred for a further 0.5 h, quenched by water, acidified with 2 N HCl to pH ∼ 6, and extracted with ethyl acetate. The organic layers were combined, dried with anhydrous sodium sulfate, filtered, and concentrated and purified by silica gel column chromatography to give <b>A1</b> as a yellow solid (0.200 g, 66%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 7.54 (d, <i>J</i> = 16.0 Hz, 1H), 7.50–7.48 (m, 3H), 7.35 (d, <i>J</i> = 8.0 Hz, 1H), 7.12–7.09 (m, 2H), 6.68 (d, <i>J</i> = 16.0 Hz, 1H), 5.33 (s, 1H), 3.42–3.41 (m, 1H), 3.25 (s, 3H), 2.92–2.82 (m, 2H), 2.65–2.60 (m, 1H), 2.48–2.45 (m, 1H), 1.37 (d, <i>J</i> = 16.0 Hz, 3H), 1.21 (d, <i>J</i> = 16.0 Hz, 3H), 1.09 (d, <i>J</i> = 6.5 Hz, 3H). HPLC purity: 97.8% (<i>t</i><sub>R</sub> = 9.59 min). ESI-MS <i>m</i>/<i>z</i>: [M + H]<sup>−</sup> = 457.2 (Calcd: 457.21).</div></div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> General Procedures for the Syntheses of Compounds <b>B1</b>–<b>B3</b></h3><div id="sec4_3_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> Benzyl (<i>S</i>)-(1-Fluoro-3-(1<i>H</i>-indol-3-yl)propan-2-yl)carbamate (<b>2</b>)</h4><div class="NLM_p last">(<i>S</i>)-2-(((Benzyloxy)carbonyl)amino)-3-(1<i>H</i>-indol-3-yl)propyl 4-methylbenzenesulfonate (2.0 g, 4.18 mmol) was dissolved in TBAF solution (1.0 M in THF, 12 mL) and stirred at 80 °C for 4 h. The reaction was cooled to room temperature, quenched by water (20 mL), and extracted with ethyl acetate. The organic layers were combined, washed with brine (50 mL × 2), dried with anhydrous sodium sulfate, filtered, and concentrated and purified by silica gel column chromatography to give the fluorinated compound <b>2</b> as a yellow oil (1.0 g, 73%). ESI-MS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> = 327.1 (Calcd: 327.14).</div></div><div id="sec4_3_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> (<i>S</i>)-1-Fluoro-3-(1<i>H</i>-indol-3-yl)propan-2-amine (<b>3</b>)</h4><div class="NLM_p last">The suspension of the intermediate <b>2</b> (1.0 g, 3.06 mmol) and Pd(OH)<sub>2</sub> (0.05 g, 5%) in EtOH (30 mL) was stirred at hydrogen atmosphere (balloon) overnight. The mixture was filtered, and concentrated and purified by silica gel column chromatography to give the primary amine <b>3</b> as a yellow oil (quantitative yield). ESI-MS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> = 193.1 (Calcd: 193.11).</div></div><div id="sec4_3_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> (<i>S</i>)-2-Fluoro-<i>N</i>-(1-fluoro-3-(1<i>H</i>-indol-3-yl)propan-2-yl)-2-methylpropan-1-amine (<b>12a</b>)</h4><div class="NLM_p last">To a solution of the primary amine <b>3</b> (4.81 g, 25 mmol) and <i>N</i>,<i>N</i>-diisopropylethylamine (8.7 mL, 50 mmol) in dioxane (50 mL) was added 2-fluoro-2-methylpropyl trifluoromethanesulfonate (6.72 g, 30 mmol). The solution was heated to 90 °C for 2 h, cooled to room temperature, and concentrated and purified by silica gel column chromatography to give the secondary amine <b>12a</b> as a brown oil (6.0 g, 90%). ESI-MS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> = 267.1 (Calcd: 267.16).</div></div><div id="sec4_3_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> Methyl (<i>E</i>)-3-(3,5-Difluoro-4-((1<i>R</i>,3<i>S</i>)-2-(2-fluoro-2-methylpropyl)-3-(fluoromethyl)-2,3,4,9-tetrahydro-1<i>H</i>-pyrido[3,4-<i>b</i>]indol-1-yl)phenyl)acrylate</h4><div class="NLM_p last">To a solution of (<i>S</i>)-2-fluoro-<i>N</i>-(1-fluoro-3-(1<i>H</i>-indol-3-yl)propan-2-yl)-2-methylpropan-1-amine (<b>12a</b>; 0.532 g, 2 mmol) and glacial acetic acid (1 mL) in toluene (20 mL) was added methyl (<i>E</i>)-3-(3,5-difluoro-4-formylphenyl)acrylate (0.543 g, 2.4 mmol) under argon. The solution was heated to 80 °C overnight, cooled to room temperature, diluted with ethyl acetate (100 mL), washed with saturated sodium bicarbonate (50 mL) and brine (50 mL), dried with anhydrous sodium sulfate, filtered, and concentrated and purified by silica gel column chromatography to give the ester as a yellow solid (0.656 g, 70%). ESI-MS <i>m</i>/<i>z</i>: [M + H]<sup>−</sup> = 475.1 (Calcd: 475.19).</div></div><div id="sec4_3_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> (<i>E</i>)-3-(3,5-Difluoro-4-((1<i>R</i>,3<i>S</i>)-2-(2-fluoro-2-methylpropyl)-3-(fluoromethyl)-2,3,4,9-tetrahydro-1<i>H</i>-pyrido[3,4-<i>b</i>]indol-1-yl)phenyl)acrylic Acid (<b>B1</b>)</h4><div class="NLM_p last">To a solution of the ester (0.474 g, 1 mmol) in MeOH (3 mL) and THF (6 mL) was added dropwise a solution of NaOH (2 M, 2 mL) in an ice bath. The solution was stirred for 2 h, acidified with 2 N HCl to pH ∼ 6, and extracted with ethyl acetate (50 mL). The organic layers were combined, dried with anhydrous sodium sulfate, filtered, and concentrated and purified by silica gel column chromatography to give <b>B1</b> as a yellow solid (0.381 g, 83%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.80 (s, 1H), 7.65–7.61 (m, 2H), 7.49 (s, 1H), 7.23 (d, <i>J</i> = 16.0 Hz, 1H), 7.15–7.13 (m, 2H), 7.08 (d, <i>J</i> = 8.0 Hz, 1H), 6.45 (d, <i>J</i> = 16.0 Hz, 1H), 5.12 (s, 1H), 4.99–4.72 (m, 2H), 3.73 (d, <i>J</i> = 12.6 Hz, 1H), 3.42 (s, 1H), 2.86–2.73 (m, 2H), 2.52–2.42 (m, 1H), 1.24–1.16 (m, 6H). HPLC purity: 98.0% (<i>t</i><sub>R</sub> = 14.96 min). ESI-MS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> = 461.1 (Calcd: 461.18).</div></div><div id="sec4_3_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> 1-((1-Tosyl-1<i>H</i>-indol-3-yl)methyl)cyclopropan-1-amine (<b>5</b>)</h4><div class="NLM_p last">To a solution of 2-(1-tosyl-1<i>H</i>-indol-3-yl)acetonitrile (<b>4</b>; 3.83 g, 12.4 mmol) in THF (100 mL) was added dropwise MeTi(i-PrO)<sub>3</sub> (18.8 mL, 1 M in THF) under argon. After the solution was stirred for 0.5 h, ethylmagnesium bromide (18.8 mL, 1 M in THF) was added dropwise. After stirring for a further 1 h, boron trifluoride etherate (3.3 mL) was added. After additional stirring for a further 2 h, HCl (100 mL, 1 M (aq)) and ethyl acetate (200 mL) were added successively. NaOH (100 mL, 3 M (aq)) was added. The aqueous phase was extracted with ethyl acetate. The organic layers were combined, dried with anhydrous sodium sulfate, filtered, and concentrated and purified by silica gel column chromatography to give 1-((1-tosyl-1<i>H</i>-indol-3-yl)methyl)cyclopropan-1-amine (<b>5</b>) as a yellow oil (3.2 g, 76%). ESI-MS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> = 341.2 (Calcd: 341.12).</div></div><div id="sec4_3_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> 1-((1<i>H</i>-Indol-3-yl)methyl)cyclopropan-1-amine (<b>6</b>)</h4><div class="NLM_p last">To a solution of 1-((1-tosyl-1<i>H</i>-indol-3-yl)methyl)cyclopropan-1-amine (0.50 g, 1.47 mmol) in MeOH (25 mL) was added NaOH (5.0 mL, 10% (aq)) dropwise. The solution was heated for reflux for 12 h. After cooling to room temperature, brine (30 mL) was added. The mixture was extracted with ethyl acetate. The organic layers were combined, dried with anhydrous sodium sulfate, filtered, and concentrated to afford 1-((1<i>H</i>-indol-3-yl)methyl)cyclopropan-1-amine <b>6</b> as a light yellow oil for direct use. ESI-MS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> = 187.2 (Calcd: 187.12).</div></div><div id="sec4_3_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37"> (<i>E</i>)-3-(3,5-Difluoro-4-(2′-(2-fluoro-2-methylpropyl)-1′,2′,4′,9′-tetrahydrospiro[cyclopropane-1,3′-pyrido[3,4-<i>b</i>]indol]-1′-yl)phenyl)acrylic Acid (<b>B2</b>)</h4><div class="NLM_p last">Prepared using the same procedure as that for <b>B1</b>, <b>B2</b> was a light yellow solid (0.026 g, 76%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.81 (s, 1H), 7.49 (d, <i>J</i> = 16.0 Hz, 1H), 7.39–7.36 (m, 3H), 7.26 (d, <i>J</i> = 16.0 Hz, 1H), 7.04 (t, <i>J</i> = 7.2 Hz, 1H), 6.99 (t, <i>J</i> = 16.0 Hz, 1H), 6.65 (d, <i>J</i> = 16.0 Hz, 1H), 5.47 (d, <i>J</i> = 16.0 Hz, 1H), 3.31–3.27 (m, 1H), 3.00 (t, <i>J</i> = 12.0 Hz, 1H), 2.81–2.77 (m, 1H), 2.20 (d, <i>J</i> = 16.0 Hz, 1H), 1.31 (s, 6H), 0.50–0.46 (m, 2H), 0.34–0.30 (m, 2H). HPLC purity: 97.0% (<i>t</i><sub>R</sub> = 10.37 min). ESI-MS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> = 455.2 (Calcd: 455.19).</div></div><div id="sec4_3_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> 2-(1<i>H</i>-Indol-3-yl)cyclopentan-1-one (<b>9</b>)</h4><div class="NLM_p last">To a solution of 1<i>H</i>-indole (1.98 g, 16.9 mmol) and 2-chlorocyclopentan-1-one (2 g, 16.9 mmol) in 2,2,2-trifluoroethanol (35 mL) was added anhydrous sodium carbonate (2.15 g, 20.28 mmol). The solution was stirred for 48 h, filtered, and concentrated to afford 2-(1<i>H</i>-indol-3-yl)cyclopentan-1-one (<b>9</b>) for direct use in the next step. ESI-MS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> = 200.1 (Calcd: 200.10).</div></div><div id="sec4_3_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39"> 2-(1<i>H</i>-Indol-3-yl)cyclopentan-1-amine (<b>10</b>)</h4><div class="NLM_p last">To a solution of 2-(1<i>H</i>-indol-3-yl)cyclopentan-1-one (<b>9</b>; 4.78 g, 24.2 mmol) in methanol (60 mL) was added ammonium acetate (20.18 g, 262 mmol) and sodium cyanoborohydride (1.67g, 26.42 mmol). The mixture was stirred overnight before acidification to pH ∼ 2 by 2 N HCl. The solution was concentrated and extracted with dichloromethane. The aqueous phase was basified to pH ∼ 12 by 4 M NaOH. The solution was extracted with dichloromethane. The organic layers were combined, washed with brine, dried with anhydrous sodium sulfate, filtered, and concentrated to afford 2-(1<i>H</i>-indol-3-yl)cyclopentan-1-amine (<b>10</b> ; 4.13 g, 85%). ESI-MS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> = 201.1 (Calcd: 201.13).</div></div><div id="sec4_3_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40"> <i>N</i>-(2-Fluoro-2-methylpropyl)-2-(1<i>H</i>-indol-3-yl)cyclopentan-1-amine (<b>12c</b>)</h4><div class="NLM_p last">To a solution of 2-(1<i>H</i>-indol-3-yl)cyclopentan-1-amine (<b>10</b>; 3.5 g, 17.5 mmol) and <i>N</i>,<i>N</i>-diisopropylethylamine (4.52 g, 35 mmol) in dioxane (50 mL) was added 2-fluoro-2-methylpropyl trifluoromethanesulfonate (5.0 g, 22.32 mmol). The solution was heated to 90 °C for 2 h, cooled to room temperature, and concentrated and purified by silica gel column chromatography to give <i>N</i>-(2-fluoro-2-methylpropyl)-2-(1<i>H</i>-indol-3-yl)cyclopentan-1-amine (<b>12c</b>) as a brown oil (1.7 g, 35%). ESI-MS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> = 275.2 (Calcd: 275.18).</div></div><div id="sec4_3_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> Methyl (<i>E</i>)-3-(3,5-Difluoro-4-((5<i>R</i>)-4-(2-fluoro-2-methylpropyl)-1,2,3,3<i>a</i>,4,5,6, 10<i>c</i>-octahydrocyclopenta[5,6]pyrido[3,4-<i>b</i>]indol-5-yl)phenyl)acrylate</h4><div class="NLM_p last">To a solution of <i>N</i>-(2-fluoro-2-methylpropyl)-2-(1<i>H</i>-indol-3-yl)cyclopentan-1-amine (<b>12c</b>; 0.30 g, 1.1 mmol) and glacial acetic acid (0.6 mL) in toluene (20 mL) was added methyl (<i>E</i>)-3-(3,5-difluoro-4-formylphenyl)acrylate (0.251 g, 1.1 mmol) under argon. The solution was heated to 90 °C for 6 h, cooled to room temperature, concentrated and purified by silica gel column chromatography to give the ester as a yellow solid (0.215 g, 41%). ESI-MS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> = 483.2 (Calcd: 483.21).</div></div><div id="sec4_3_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42"> (<i>E</i>)-3-(3,5-Difluoro-4-((5<i>R</i>)-4-(2-fluoro-2-methylpropyl)-1,2,3,3<i>a</i>,4,5,6,10<i>c</i>-octahydrocyclopenta[5,6]pyrido[3,4-<i>b</i>]indol-5-yl)phenyl)acrylic Acid (<b>B3</b>)</h4><div class="NLM_p last">To a solution of the ester (0.215 g, 0.45 mmol) in MeOH (5 mL) and THF (10 mL) was added dropwise a solution of NaOH (2 N, 0.5 mL) in an ice bath. The solution was stirred for 2 h at room temperature, acidified with 1 N HCl to pH ∼ 6, and extracted with ethyl acetate (50 mL). The organic layers were combined, dried with anhydrous sodium sulfate, filtered, and concentrated and purified by silica gel column chromatography to give <b>B3</b> as a yellow solid (0.158 g, 75%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.76 (s, 1H), 7.56 (d, <i>J</i> = 8.0 Hz, 1H), 7.51–7.44 (m, 3H), 7.23 (d, <i>J</i> = 8.0 Hz, 1H), 7.01–6.94 (m, 2H), 6.66 (d, <i>J</i> = 16.0 Hz, 1H), 5.35 (s, 1H), 2.75–2.51 (m, 5H), 1.77–1.76 (m, 2H), 1.50–1.44 (m, 3H), 1.38–1.36 (m, 3H), 1.24–1.21 (m, 3H), 1.20–1.10 (m, 2H). HPLC purity: 98.2% (<i>t</i><sub>R</sub> = 16.79 min). ESI-Ms <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> = 469.2 (Calcd: 469.20).</div></div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i43" class="anchor-spacer"></div><h3 class="article-section__title" id="_i43"> General Procedures for the Syntheses of Compounds <b>C1</b>–<b>C4</b></h3><div id="sec4_4_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i44" class="anchor-spacer"></div><h4 class="article-section__title" id="_i44"> (<i>R</i>)-1-(1<i>H</i>-Indol-3-yl)-<i>N</i>-((1-(trifluoromethyl)cyclopropyl)methyl)propan-2-amine (<b>15a</b>)</h4><div class="NLM_p last">A solution of (1-(trifluoromethyl)cyclopropyl)methyl trifluoromethanesulfonate (0.296 g, 1 mmol), (<i>R</i>)-1-(1<i>H</i>-indol-3-yl)propan-2-amine (0.270 g, 1.2 mmol), and <i>N</i>,<i>N</i>-diisopropylethylamine (0.26 mL, 1.5 mmol) in dioxane (10 mL) was heated to 90 °C for 8 h. The mixture was diluted with ethyl acetate (100 mL) and washed with water (50 mL × 2) and brine. The organic phase was dried with anhydrous sodium sulfate, filtered, and concentrated and purified by silica gel column chromatography to give (<i>R</i>)-1-(1<i>H</i>-indol-3-yl)-<i>N</i>-((1-(trifluoromethyl)cyclopropyl)methyl)propan-2-amine (<b>15a</b>) as a yellow oil (0.212 g, 72%). ESI-MS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> = 297.2 (Calcd: 297.15).</div></div><div id="sec4_4_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i45" class="anchor-spacer"></div><h4 class="article-section__title" id="_i45"> (<i>E</i>)-3-(3,5-Difluoro-4-((1<i>R</i>,3<i>R</i>)-3-methyl-2-((1-(trifluoromethyl)cyclopropyl)methyl)-2,3,4,9-tetrahydro-1<i>H</i>-pyrido[3,4-<i>b</i>]indol-1-yl)phenyl)acrylic Acid (<b>C1</b>)</h4><div class="NLM_p last">Prepared using the same procedure as that for <b>B1</b>, <b>C1</b> was a white solid (0.033 g, 79%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.55 (s, 1H), 7.59 (d, <i>J</i> = 16.0 Hz, 1H), 7.51–7.49 (m, 2H), 7.42 (d, <i>J</i> = 8.0 Hz, 1H), 7.21 (d, <i>J</i> = 8.0 Hz, 1H), 7.04–6.83 (m, 2H), 6.58 (d, <i>J</i> = 16.0 Hz, 1H), 5.06 (s, 1H), 3.65–3.63 (m, 1H), 3.00–2.96 (m, 1H), 2.80–2.71 (m, 2H), 2.65–2.63 (m, 1H), 1.42–1.36 (m, 2H), 1.43–1.29 (m, 2H), 1.10 (d, <i>J</i> = 7.5 Hz, 3H). HPLC purity: 96.4% (<i>t</i><sub>R</sub> = 14.96 min). ESI-MS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> = 491.2 (Calcd: 491.17).</div></div><div id="sec4_4_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i46" class="anchor-spacer"></div><h4 class="article-section__title" id="_i46"> (<i>E</i>)-3-(4-((1<i>R</i>,3<i>R</i>)-2-Butyl-3-methyl-2,3,4,9-tetrahydro-1<i>H</i>-pyrido[3,4-<i>b</i>] indol-1-yl)-3,5-difluorophenyl)acrylic Acid (<b>C2</b>)</h4><div class="NLM_p last">Prepared using the same procedure as that for <b>B1</b>, <b>C2</b> was a white solid (0.045 g, 83%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.54 (s, 1H), 7.57–7.43 (m, 3H), 7.40 (d, <i>J</i> = 7.5 Hz, 1H), 7.18 (d, <i>J</i> = 7.5 Hz, 1H), 7.02–6.92 (m, 2H), 6.66 (d, <i>J</i> = 16.0 Hz, 1H), 5.11 (s, 1H), 3.47–3.41 (m, 1H), 2.71–2.54 (m, 1H), 2.43–2.25 (m, 2H), 1.38–1.35 (m, 1H), 1.33–1.30 (m, 2H),1.28–1.26 (m, 2H), 1.22 (d, <i>J</i> = 7.5 Hz, 3H), 0.75 (t, <i>J</i> = 7.5 Hz, 3H). HPLC purity: 95.9% (<i>t</i><sub>R</sub> = 10.37 min). ESI-MS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> = 425.2 (Calcd: 425.19).</div></div><div id="sec4_4_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47"> (<i>E</i>)-3-(3,5-Difluoro-4-((1<i>R</i>,3<i>R</i>)-2-(3-fluoro-3-methylbutyl)-3-methyl-2,3,4,9-tetrahydro-1<i>H</i>-pyrido[3,4-<i>b</i>]indol-1-yl)phenyl)acrylic Acid (<b>C3</b>)</h4><div class="NLM_p last">Prepared using the same procedure as that for <b>B1</b>, <b>C3</b> was a white solid (0.090 g, 81%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.56 (s, 1H), 7.53–7.46 (m, 3H), 7.41 (d, <i>J</i> = 7.5 Hz, 1H), 7.19 (d, <i>J</i> = 7.5 Hz, 1H), 7.02–6.92 (m, 2H), 6.65 (d, <i>J</i> = 16.0 Hz, 1H), 5.10 (s, 1H), 3.49–3.48 (m, 1H), 2.75–2.73 (m, 1H), 2.71–2.69 (m, 1H), 2.66–2.58 (m, 2H), 1.81–1.72 (m, 2H), 1.30–1.11 (m, 6H), 1.02 (d, <i>J</i> = 7.5 Hz, 3H). HPLC purity: 95.9% (<i>t</i><sub>R</sub> = 10.30 min). ESI-MS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> = 457.3 (Calcd: 457.20).</div></div><div id="sec4_4_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i48" class="anchor-spacer"></div><h4 class="article-section__title" id="_i48"> (<i>R</i>)-<i>N</i>-(1-(1<i>H</i>-Indol-3-yl)propan-2-yl)aniline (<b>15d</b>)</h4><div class="NLM_p last">To a solution of (<i>R</i>)-1-(1<i>H</i>-indol-3-yl)propan-2-amine (0.500 g, 2.87 mmol) and iodobenzene (0.702 g, 4 mmol) in DMF (5 mL) was added cesium carbonate (1.87 g, 3.7 mmol) and copper(I) iodide (0.055 g, 0.28 mmol) under argon. The sealed vial was irradiated in the microwave on a Biotage Smith Synthesis apparatus at 135 °C for 120 min. The mixture was diluted with ethyl acetate (50 mL) and filtered over Celite. The filtrate was washed with water (30 mL × 3) and brine, dried with anhydrous sodium sulfate, filtered, and concentrated and purified by silica gel column chromatography to give (<i>R</i>)-<i>N</i>-(1-(1<i>H</i>-indol-3-yl)propan-2-yl)aniline (<b>15d</b>; 0.243 g, 34%). ESI-MS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> = 251.1 (Calcd: 251.15).</div></div><div id="sec4_4_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i49" class="anchor-spacer"></div><h4 class="article-section__title" id="_i49"> Methyl (<i>E</i>)-3-(3,5-Difluoro-4-((1<i>R</i>,3<i>R</i>)-3-methyl-2-phenyl-2,3,4,9-tetrahydro-1<i>H</i>-pyrido[3,4<i>-b</i>]indol-1-yl)phenyl)acrylate</h4><div class="NLM_p last">To a solution of (<i>R</i>)-<i>N</i>-(1-(1<i>H</i>-indol-3-yl)propan-2-yl)aniline (0.20 g, 0.8 mmol) and glacial acetic acid (0.5 mL) in toluene (20 mL) was added methyl (<i>E</i>)-3-(3,5-difluoro-4-formylphenyl)acrylate (0.202 g, 0.89 mmol) under argon. The solution was heated to 80 °C overnight, cooled to room temperature, and concentrated and purified by silica gel column chromatography to give the ester as a brown oil (0.150 g, 41%). ESI-MS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> = 459.1 (Calcd: 459.18).</div></div><div id="sec4_4_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i50" class="anchor-spacer"></div><h4 class="article-section__title" id="_i50"> (<i>E</i>)-3-(3,5-Difluoro-4-((1<i>R</i>,3<i>R</i>)-3-methyl-2-phenyl-2,3,4,9-tetrahydro-1<i>H</i>-pyrido[3,4-<i>b</i>]indol-1-yl)phenyl)acrylic Acid (<b>C4</b>)</h4><div class="NLM_p last">To a solution of the ester (0.150 g, 0.33 mmol) in MeOH (2 mL) and THF (8 mL) was added dropwise a solution of NaOH (5 N, 2 mL) in an ice bath. The solution was stirred for 2 h at room temperature, acidified with 2 N HCl to pH ∼ 6, and extracted with ethyl acetate (50 mL). The organic layers were combined, dried with anhydrous sodium sulfate, filtered, and concentrated and purified by silica gel column chromatography to give compound <b>C4</b> as a light yellow solid (0.125 g, 86%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 7.53–7.52 (m, 1H), 7.37 (d, <i>J</i> = 16.0 Hz, 1H), 7.25–7.17 (m, 2H), 7.12–7.06 (m, 5H), 6. 96–6.94 (m, 1H), 6.87 (s, 1H), 6.59 (s, 1H), 6.54 (d, <i>J</i> = 16.0 Hz, 1H), 5.06 (s, 1H), 3.47–3.41 (m, 1H), 3.00–2.94 (m, 1H), 1.24–1.22 (m, 1H), 1.19–1.17 (m, 3H). HPLC purity: 97.1% (<i>t</i><sub>R</sub> = 10.31 min). ESI-MS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> = 445.0 (Calcd: 445.16).</div></div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i51" class="anchor-spacer"></div><h3 class="article-section__title" id="_i51"> General Procedures for the Syntheses of Compounds <b>D1</b>–<b>D40</b></h3><div id="sec4_5_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i52" class="anchor-spacer"></div><h4 class="article-section__title" id="_i52"> Methyl (<i>E</i>)-3-(6-Formylpyridin-3-yl)acrylate (<b>17</b>)</h4><div class="NLM_p last">To a solution of 5-bromopicolinaldehyde (1.46 g, 7.84 mmol) and palladium acetate (0. 239 g, 0.78 mmol) in DMF (30 mL) was added methyl acrylate (1.01 g, 11.76 mmol) and triethylamine (1.59 g, 15.69 mmol) under argon. The solution was heated to 80 °C overnight. After cooling to room temperature, brine (60 mL) was added. The mixture was extracted with ethyl acetate. The organic layers were combined, dried with anhydrous sodium sulfate, filtered, and concentrated and purified by silica gel column chromatography to give methyl (<i>E</i>)-3-(6-formylpyridin-3-yl) acrylate (<b>17</b>) as a yellow solid (1.45 g, 96%). ESI-MS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> = 192.1 (Calcd: 192.06).</div></div><div id="sec4_5_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i53" class="anchor-spacer"></div><h4 class="article-section__title" id="_i53"> (<i>E</i>)-3-(6-((1<i>S</i>,3<i>R</i>)-2-(2-Fluoro-2-methylpropyl)-3-methyl-2,3,4,9-tetrahydro-1<i>H</i>-pyrido [3,4-<i>b</i>]indol-1-yl)pyridin-3-yl)acrylic Acid (<b>D1</b>)</h4><div class="NLM_p last">Prepared using the same procedure as that for <b>B1</b>, <b>D1</b> was a white solid (0.077 g, 68%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.68 (s, 1H), 8.72 (s,1H), 8.11 (d, <i>J</i> = 16.0 Hz, 1H), 7.55 (m, 2H), 7.50 (d, <i>J</i> = 16.0 Hz, 1H), 7.30 (d, <i>J</i> = 8.0 Hz, 1H), 7.05–6.97 (m, 2H), 6.64 (d, <i>J</i> = 8.0 Hz, 1H), 5.10 (s, 1H), 3.19–3.12 (m, 1H), 2.93–2.86 (m, 1H), 2.52–2.43 (m, 2H), 1.23 (d, <i>J</i> = 20.0 Hz, 3H), 1.15 (d, <i>J</i> = 20.0 Hz, 3H), 1.07 (d, <i>J</i> = 8.0 Hz, 3H). HPLC purity: 97.8% (<i>t</i><sub>R</sub> = 9.59 min). ESI-MS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> = 408.3 (Calcd: 408.21).</div></div><div id="sec4_5_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i54" class="anchor-spacer"></div><h4 class="article-section__title" id="_i54"> (<i>E</i>)-3-(5-Fluoro-6-((1<i>S</i>,3<i>R</i>)-2-(2-fluoro-2-methylpropyl)-3-methyl-2,3,4,9-tetrahydro-1<i>H</i>-pyrido[3,4-<i>b</i>]indol-1-yl)pyridin-3-yl)acrylic Acid (<b>D2</b>)</h4><div class="NLM_p last">Prepared using the same procedure as that for <b>B1</b>, <b>D2</b> was a light yellow solid (0.030 g, 64%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.52 (s, 1H), 8.49 (s, 1H), 8.11 (d, <i>J</i> = 12.0 Hz, 1H), 7.54 (d, <i>J</i> = 16.0 Hz, 1H), 7.41 (d, <i>J</i> = 8.0 Hz, 1H), 7.19 (d, <i>J</i> = 8.0 Hz, 1H), 7.02–6.92 (m, 2H), 6.69 (d, <i>J</i> = 16.0 Hz, 1H), 5.44 (s, 1H), 3.41 (s, 1H), 2.91–2.86 (m, 2H), 2.60–2.53 (m, 1H), 2.46–2.33 (m, 1H), 1.23 (d, <i>J</i> = 20.0 Hz, 3H), 1.15 (d, <i>J</i> = 20.0 Hz, 3H), 1.07 (d, <i>J</i> = 8.0 Hz, 3H). HPLC purity: 96.6% (<i>t</i><sub>R</sub> = 9.90 min). ESI-MS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> = 426.1 (Calcd: 426.19).</div></div><div id="sec4_5_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i55" class="anchor-spacer"></div><h4 class="article-section__title" id="_i55"> (<i>E</i>)-3-(2,6-Difluoro-4-((1<i>R</i>,3<i>R</i>)-2-(2-fluoro-2-methylpropyl)-3-methyl-2,3,4,9-tetrahydro-1<i>H</i>-pyrido[3,4-<i>b</i>]indol-1-yl)phenyl)acrylic Acid (<b>D3</b>)</h4><div class="NLM_p last">Prepared using the same procedure as that for <b>B1</b>, <b>D3</b> was a light yellow solid (0.077 g, 79%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.87 (s, 1H), 7.53 (d, <i>J</i> = 16.0 Hz, 1H), 7.45 (d, <i>J</i> = 4.0 Hz, 1H), 7.33 (d, <i>J</i> = 4.0 Hz, 1H), 7.08–7.06 (m, 3H), 7.01–6.98 (m, 1H), 6.55 (d, <i>J</i> = 16.0 Hz, 1H), 5.04 (s, 1H), 3.16 (m, 1H), 2.92–2.83 (m, 1H), 2.69–2.66 (m, 1H), 2.54–2.53 (m, 1H), 2.53–2.51 (m, 1H), 1.29 (d, <i>J</i> = 16.0 Hz, 3H), 1.10 (d, <i>J</i> = 16.0 Hz, 3H), 1.07 (d, <i>J</i> = 8.0 Hz, 3H). HPLC purity: 96.7% (<i>t</i><sub>R</sub> = 12.15 min). ESI-MS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> = 443.1 (Calcd: 443.19).</div></div><div id="sec4_5_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i56" class="anchor-spacer"></div><h4 class="article-section__title" id="_i56"> (<i>E</i>)-3-(3,5-Dichloro-4-((1<i>R</i>,3<i>R</i>)-2-(2-fluoro-2-methylpropyl)-3-methyl-2,3,4,9-tetrahydro-1<i>H</i>-pyrido[3,4-<i>b</i>]indol-1-yl)phenyl)acrylic Acid (<b>D4</b>)</h4><div class="NLM_p last">Prepared using the same procedure as that for <b>B1</b>, <b>D4</b> was a white solid (0.069 g, 73%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.25 (s, 1H), 7.79 (s, 1H), 7.54 (s, 1H), 7.44–7.29 (m, 2H), 7.15 (d, <i>J</i> = 16.0 Hz, 1H), 7.04–6.83 (m, 2H), 6.58 (d, <i>J</i> = 16.0 Hz, 1H), 5.58 (s, 1H), 3.67 (s, 1H), 3.14–2.83 (m, 2H), 2.65–2.58 (m, 1H), 2.29–2.24 (m, 1H), 1.14–1.10 (m, 6H), 1.03–1.01 (m, 3H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 170.78, 142.53, 137.32, 136.28, 135.76, 130.57, 127.45, 121.40, 121.12, 119.31, 118.15, 110.77, 109.30, 97.13 (d, <i>J</i> = 166.0 Hz), 58.17 (d, <i>J</i> = 22.0 Hz), 55.71, 52.16 (d, <i>J</i> = 5.0 Hz), 27.72, 25.73 (d, <i>J</i> = 24.0 Hz), 24.35 (d, <i>J</i> = 24.0 Hz), 9.80. HPLC purity: 96.4% (<i>t</i><sub>R</sub> = 12.15 min). ESI-MS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> = 475.1 (Calcd: 475.13).</div></div><div id="sec4_5_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i57" class="anchor-spacer"></div><h4 class="article-section__title" id="_i57"> (<i>E</i>)-3-(3-Fluoro-4-((1<i>R</i>,3<i>R</i>)-2-(2-fluoro-2-methylpropyl)-3-methyl-2,3,4,9-tetrahydro-1<i>H</i>-pyrido[3,4-<i>b</i>]indol-1-yl)-5-methoxyphenyl)acrylic Acid (<b>D5</b>)</h4><div class="NLM_p last">Prepared using the same procedure as that for <b>B1</b>, <b>D5</b> was a white solid (0.045 g, 61%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.36 (s, 1H), 7.56 (d, <i>J</i> = 16.0 Hz, 1H), 7.38 (d, <i>J</i> = 8.0 Hz, 1H), 7.28 (s, 1H), 7.15 (d, <i>J</i> = 4.0 Hz, 1H), 7.06 (d, <i>J</i> = 16.0 Hz, 1H), 6.97–6.92 (m, 2H), 6.98–6.64 (d, 1H), 5.33 (s, 1H), 3.89 (s, 3H), 3.58–3.52 (m, 1H), 2.98–2.81 (m, 2H), 2.58–2.53 (m, 1H), 2.35–2.25 (m, 1H), 1.29 (d, <i>J</i> = 16.0 Hz, 3H), 1.10 (d, <i>J</i> = 16.0 Hz, 3H), 1.07 (d, <i>J</i> = 8.0 Hz, 3H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 169.88, 160.69 (d, <i>J</i> = 180.0 Hz), 144.07, 136.04, 133.25, 126.67, 120.98, 119.93, 119.13, 118.15, 114.10, 110.73, 106.08, 97.15 (d, <i>J</i> = 166.0 Hz), 62.36 (d, <i>J</i> = 22.0 Hz), 59.26, 56.11, 54.96, 27.72, 25.91 (d, <i>J</i> = 14.0 Hz), 24.32 (d, <i>J</i> = 25.0 Hz), 19.35. HPLC purity: 96.3% (<i>t</i><sub>R</sub> = 10.74 min). ESI-MS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> = 455.2 (Calcd: 455.21).</div></div><div id="sec4_5_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i58" class="anchor-spacer"></div><h4 class="article-section__title" id="_i58"> (<i>E</i>)-3-(4-((1<i>R</i>,3<i>R</i>)-2-(2-Fluoro-2-methylpropyl)-3-methyl-2,3,4,9-tetrahydro-1<i>H</i>-pyrido[3,4-<i>b</i>]indol-1-yl)-3,5-dimethoxyphenyl)acrylic Acid (<b>D6</b>)</h4><div class="NLM_p last">Prepared using the same procedure as that for <b>B1</b>, <b>D6</b> was a white solid (0.045 g, 61%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.1 (s,1H), 7.52 (d, <i>J</i> = 8.0 Hz, 1H), 7.36 (d, <i>J</i> = 16.0 Hz, 1H), 7.14 (d, <i>J</i> = 4.0 Hz, 1H), 6.94–6.85 (m, 4H), 6.63 (d, <i>J</i> = 16.0 Hz, 1H), 5.45 (s, 1H), 3.41–3.93 (m, 2H), 3.38 (s, 6H), 3.05 (m, 1H), 2.85 (d, 1H), 2.55 (m, 1H), 1.09 (d, <i>J</i> = 5.0 Hz, 3H), 1.04 (d, <i>J</i> = 5.0 Hz,3H), 1.03–1.01 (m, 3H). HPLC purity: 97.0% (<i>t</i><sub>R</sub> = 9.73 min). ESI-MS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> = 467.3 (Calcd: 467.23).</div></div><div id="sec4_5_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i59" class="anchor-spacer"></div><h4 class="article-section__title" id="_i59"> (1<i>R</i>,3<i>R</i>)-1-(4-Bromo-2,6-difluorophenyl)-2-(2-fluoro-2-methylpropyl)-3-methyl-2,3,4,9-tetrahydro-1<i>H</i>-pyrido[3,4-<i>b</i>]indole (<b>22</b>)</h4><div class="NLM_p last">To a solution of (<i>R</i>)-<i>N</i>-(1-(1<i>H</i>-indol-3-yl)propan-2-yl)aniline (1.0 g, 4.03 mmol) and glacial acetic acid (0.5 mL) in toluene (10 mL) was added 4-bromo-2,6-difluorobenzaldehyde (0.934 g, 4.23 mmol) under argon. The solution was heated to 80 °C overnight, cooled to room temperature, quenched by saturated sodium bicarbonate, and extracted with ethyl acetate. The organic layers were combined, dried with anhydrous sodium sulfate, filtered, and concentrated and purified by silica gel column chromatography to give (1<i>R</i>,3<i>R</i>)-1-(4-bromo-2,6-difluorophenyl)-2-(2-fluoro-2-methyl propyl)-3-methyl-2,3,4,9-tetrahydro-1<i>H</i>-pyrido[3,4-<i>b</i>]indole (<b>22</b>) as a white solid (1.50 g, 83%). ESI-MS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> = 451.1 (Calcd: 450.09).</div></div><div id="sec4_5_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i60" class="anchor-spacer"></div><h4 class="article-section__title" id="_i60"> Methyl 1-(3,5-Difluoro-4-((1<i>R</i>,3<i>R</i>)-2-(2-fluoro-2-methylpropyl)-3-methyl-2,3,4,9-tetrahydro-1<i>H</i>-pyrido[3,4-<i>b</i>]indol-1-yl)phenyl)azetidine-3-carboxylate (<b>23</b>)</h4><div class="NLM_p last">To a mixture of (1<i>R</i>,3<i>R</i>)-1-(4-bromo-2,6-difluorophenyl)-2-(2-fluoro-2-methyl propyl)-3-methyl-2,3,4,9-tetrahydro-1<i>H</i>-pyrido[3,4-<i>b</i>]indole (0.10 g, 0.22 mmol), Pd<sub>2</sub>(dba)<sub>3</sub> (0.010 g, 0.011 mmol), Xphos (0.010 g, 0.022 mmol), and Cs<sub>2</sub>CO<sub>3</sub> (0.202 g, 0.62 mmol) in toluene (5 mL) was added methyl azetidine-3-carboxylate (0.029 g, 0.25 mmol) under argon. The reaction was heated to 110 °C for 10 h. After cooling to room temperature, the mixture was concentrated and purified by silica gel column chromatography to give the desired ester <b>23</b> as a yellow solid (0.050 g, 47%). ESI-MS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> = 486.1 (Calcd: 486.23).</div></div><div id="sec4_5_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i61" class="anchor-spacer"></div><h4 class="article-section__title" id="_i61"> 1-(3,5-Difluoro-4-((1<i>R</i>,3<i>R</i>)-2-(2-fluoro-2-methylpropyl)-3-methyl-2,3,4,9-tetrahydro-1<i>H</i>-pyrido[3,4-<i>b</i>]indol-1-yl)phenyl)azetidine-3-carboxylic Acid (<b>D8</b>)</h4><div class="NLM_p last">Prepared using the same procedure for hydrolysis aforementioned, <b>D8</b> was a white solid (0.035 g, 74%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.45 (s, 1H), 7.38 (d, <i>J</i> = 8.0 Hz, 1H), 7.18 (d, <i>J</i> = 8.0 Hz, 1H), 7.00–6.91 (m, 2H), 6.18 (d, <i>J</i> = 8.0 Hz, 2H), 5.12 (s, 1H), 4.05–4.00 (m, 2H), 3.90–3.86 (m, 2H), 3.55–3.51 (m, 2H), 2.90–2.81 (m, 2H), 2.59–2.55 (m, 1H), 2.41–2.28 (m, 1H), 1.14–1.10 (m, 6H), 1.03–1.01 (m, 3H). HPLC purity: 96.3% (<i>t</i><sub>R</sub> = 10.74 min). HPLC purity: 96.4% (t<sub>R</sub> = 10.78 min). ESI-MS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> = 472.3 (Calcd: 472.21).</div></div><div id="sec4_5_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i62" class="anchor-spacer"></div><h4 class="article-section__title" id="_i62"> 1-((3,5-Difluoro-4-((3<i>R</i>)-2-(2-fluoro-2-methylpropyl)-3-methyl-2,3,4,9-tetrahydro-1<i>H</i>-pyrido[3,4-<i>b</i>]indol-1-yl)phenyl)amino)cyclopropane-1-carboxylic Acid (<b>D7</b>)</h4><div class="NLM_p last">Prepared using the same procedure as that for <b>D8</b>, <b>D7</b> was a white solid (0.041 g, 64%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.54 (s, 1H), 7.38 (d, <i>J</i> = 8.0 Hz, 1H), 7.20 (d, <i>J</i> = 8.0 Hz, 1H), 7.00–6.93 (m, 2H), 6.18–6.12 (m, 2H), 5.06 (s, 1H), 3.47–3.44 (m, 1H), 3.18–3.04 (m, 1H), 2.82–2.73 (m, 2H), 2.38–2.34 (m, 1H), 1.43–1.40 (m, 4H), 1.22–1.10 (dd, <i>J</i> = 16.0 Hz, 12.0 Hz, 6H), 1.08 (d, <i>J</i> = 8.0 Hz, 3H). HPLC purity: 99.3% (<i>t</i><sub>R</sub> = 10.13 min). ESI-MS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> = 472.3 (Calcd: 472.21).</div></div><div id="sec4_5_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i63" class="anchor-spacer"></div><h4 class="article-section__title" id="_i63"> 1-(3,5-Difluoro-4-((1<i>R</i>,3<i>R</i>)-2-(2-fluoro-2-methylpropyl)-3-methyl-2,3,4,9-tetrahydro-1<i>H</i>-pyrido[3,4-<i>b</i>]indol-1-yl)phenyl)pyrrolidine-3-carboxylic Acid (<b>D9</b>)</h4><div class="NLM_p last">Prepared using the same procedure as that for <b>D8</b>, <b>D9</b> was a white solid (0.055 g, 73%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.43 (s, 1H), 7.38 (d, <i>J</i> = 8.0 Hz, 1H), 7.19 (d, <i>J</i> = 8.0 Hz, 1H), 6.95 (m, 2H), 6.09 (d, <i>J</i> = 4.0 Hz, 2H), 5.05 (s, 1H), 3.50 (s, 1H), 3.40 (s, 2H), 3.39–3.33 (m, 2H), 3.27–3.20 (m, 2H), 2.81–2.73 (m, 2H), 2.37–2.32 (m, 1H), 2.19–2.12 (m, 2H), 1.23–1.16 (dd, <i>J</i> = 21.5 Hz, 16.0 Hz, 6H), 1.08 (d, <i>J</i> = 6.5 Hz, 3H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 178.17, 164.43, 161.92 (d, <i>J</i> = 11.0 Hz), 148.24, 136.11, 133.00, 127.86, 120.09 (d, <i>J</i> = 200.0 Hz), 118.08, 110.63, 108.03, 97.50 (d, <i>J</i> = 165.0 Hz), 95.14 (d, <i>J</i> = 28.0 Hz), 56.83 (d, <i>J</i> = 22.0 Hz), 51.35, 50.20, 49.85, 47.09, 42.81, 29.71, 28.44, 27.19, 25.51 (d, <i>J</i> = 24.0 Hz), 24.50 (d, <i>J</i> = 24.0 Hz). HPLC purity: 98.0% (<i>t</i><sub>R</sub> = 11.00 min). ESI-MS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> = 486.3 (Calcd: 486.23).</div></div><div id="sec4_5_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i64" class="anchor-spacer"></div><h4 class="article-section__title" id="_i64"> 1-(3,5-Difluoro-4-((1<i>R</i>,3<i>R</i>)-2-(2-fluoro-2-methylpropyl)-3-methyl-2,3,4,9-tetrahydro-1<i>H</i>-pyrido[3,4-<i>b</i>]indol-1-yl)phenyl)piperidine-4-carboxylic Acid (<b>D10</b>)</h4><div class="NLM_p last">Prepared using the same procedure as that for <b>D8</b>, <b>D10</b> was a white solid (0.035 g, 57%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.34 (s, 1H), 7.93 (s,1H), 7.64 (s, 1H), 7.49 (d, <i>J</i> = 8.0 Hz, 1H), 7.40 (d, <i>J</i> = 8.0 Hz, 1H), 7.25 (m, 1H), 7.17 (d, <i>J</i> = 4.0 Hz, 1H), 6.98–6.92 (m, 2H), 6.73 (d, <i>J</i> = 4.0 Hz, 1H), 5.62 (s, 1H), 3.79–3.74 (m, 2H), 3.68–3.65 (m, 4H), 2.98–2.91 (m, 2H), 2.67–2.61 (m, 2H), 2.28–2.22 (m, 1H), 1.24–1.12 (dd, <i>J</i> = 16.0 Hz, 12.0 Hz, 6H), 1.02 (d, <i>J</i> = 8.0 Hz, 3H). HPLC purity: 97.0% (<i>t</i><sub>R</sub> = 11.31 min). ESI-MS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> = 500.3 (Calcd: 500.24).</div></div><div id="sec4_5_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i65" class="anchor-spacer"></div><h4 class="article-section__title" id="_i65"> 2-(3,5-Difluoro-4-((1<i>R</i>,3<i>R</i>)-2-(2-fluoro-2-methylpropyl)-3-methyl-2,3,4,9-tetrahydro-1<i>H</i>-pyrido[3,4-<i>b</i>]indol-1-yl)phenyl)-2-azaspiro[3.3]heptane-6-carboxylic Acid (<b>D11</b>)</h4><div class="NLM_p last">Prepared using the same procedure as that for <b>D8</b>, <b>D11</b> was a white solid (0.054 g, 78%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.48 (s, 1H), 7.37 (d, <i>J</i> = 8.0 Hz, 1H), 7.17 (d, <i>J</i> = 8.0 Hz, 1H), 6.98–6.90 (m, 2H), 6.01 (d, <i>J</i> = 12.0 Hz, 2H), 5.04 (s, 1H), 3.83 (s, 2H), 3.75 (s, 2H), 3.52–3.45 (m, 1H), 2.99–2.92 (m, 1H), 2.87–2.77 (m, 2H), 2.39 (s, 1H), 2.45–2.30 (m, 5H), 1.14–1.10 (m, 6H), 1.03–1.01 (m, 3H). HPLC purity: 96.1% (<i>t</i><sub>R</sub> = 11.79 min). ESI-MS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> = 512.2 (Calcd: 512.24).</div></div><div id="sec4_5_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i66" class="anchor-spacer"></div><h4 class="article-section__title" id="_i66"> 1-(3,5-Difluoro-4-((1<i>R</i>,3<i>R</i>)-2-(2-fluoro-2-methylpropyl)-3-methyl-2,3,4,9-tetrahydro-1<i>H</i>-pyrido[3,4-<i>b</i>]indol-1-yl)phenyl)-<i>N</i>,<i>N</i>-dimethylazetidin-3-amine (<b>D12</b>)</h4><div class="NLM_p last">Prepared using the same procedure as that for <b>D8</b>, <b>D12</b> was a light yellow solid (0.044 g, 69%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.47 (s, 1H), 7.40 (d, <i>J</i> = 4.0 Hz, 1H), 7.21 (d, <i>J</i> = 8.0 Hz, 1H), 7.02–6.94 (m, 2H), 6.05 (d, <i>J</i> = 8.0 Hz, 2H), 5.09 (s, 1H), 3.93–3.90 (m, 2H), 3.62–3.54 (m, 3H), 3.21–3.18 (m, 1H), 2.92–2.80 (m, 2H), 2.60–2.50 (m, 1H), 2.12 (s, 6H), 1.43–1.38 (m, 6H), 1.12–1.09 (m, 3H). HPLC purity: 96.5% (<i>t</i><sub>R</sub> = 8.23 min). ESI-MS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> = 471.3 (Calcd: 471.27).</div></div><div id="sec4_5_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i67" class="anchor-spacer"></div><h4 class="article-section__title" id="_i67"> (1<i>R</i>,3<i>R</i>)-1-(2,6-Difluoro-4-(3-(fluoromethyl)azetidin-1-yl)phenyl)-2-(2-fluoro-2-methylpropyl)-3-methyl-2,3,4,9-tetrahydro-1<i>H</i>-pyrido[3,4-<i>b</i>]indole (<b>D13</b>)</h4><div class="NLM_p last">Prepared using the same procedure as that for <b>D8</b>, <b>D13</b> was a white solid (0.041 g, 65%). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.47 (s, 1H), 7.40 (d, <i>J</i> = 8.0 Hz, 1H), 7.21 (d, <i>J</i> = 8.0 Hz, 1H), 7.02–6.91 (m, 2H), 6.07 (d, <i>J</i> = 11.5 Hz, 2H), 5.09 (s, 1H), 4.67 (d, <i>J</i> = 5.0 Hz, 1H), 4.58 (d, <i>J</i> = 5.0 Hz, 1H), 3.95 (s, 2H), 3.66 (s, 2H), 3.32 (s, 1H), 3.13–3.04 (m, 2H), 2.93–2.86 (m, 2H), 2.43–2.38 (m, 2H), 1.23–1.14 (m, 6H), 1.06 (d, <i>J</i> = 6,5 Hz, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 176.23, 164.23 (d, <i>J</i> = 8.0 Hz), 161.77 (d, <i>J</i> = 11.0 Hz), 152.68, 136.67, 133.68, 127.49, 120.61, 118.47, 117.82, 111.35, 106.57, 105.02, 97.56 (d, <i>J</i> = 165.0 Hz), 94.76 (d, <i>J</i> = 27.0 Hz), 84.63 (d, <i>J</i> = 164.0 Hz), 56.28, 53.44 (d, <i>J</i> = 7.0 Hz), 50.92, 40.62, 29.78 (d, <i>J</i> = 20.0 Hz), 27.15, 25.44 (d, <i>J</i> = 24.0 Hz), 24.95 (d, <i>J</i> = 24.0 Hz). HPLC purity: 94.6% (<i>t</i><sub>R</sub> = 12.75 min). ESI-MS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> = 460.3 (Calcd: 460.23).</div></div><div id="sec4_5_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i68" class="anchor-spacer"></div><h4 class="article-section__title" id="_i68"> 6-(3,5-Difluoro-4-((1<i>R</i>,3<i>R</i>)-2-(2-fluoro-2-methylpropyl)-3-methyl-2,3,4,9-tetrahydro-1<i>H</i>-pyrido[3,4-<i>b</i>]indol-1-yl)phenyl)-2-oxa-6-azaspiro[3.3]heptane (<b>D14</b>)</h4><div class="NLM_p last">Prepared using the same procedure as that for <b>D8</b>, <b>D14</b> was a white solid (0.046 g, 51%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.47 (s, 1H), 7.37 (d, <i>J</i> = 8.0 Hz, 1H), 7.17 (d, <i>J</i> = 8.0 Hz, 1H), 6.99–6.91 (m, 2H), 6.05 (d, <i>J</i> = 8.0 Hz, 2H), 5.06 (s, 1H), 4.70 (s, 4H), 3.98 (s, 4H), 2.84–2.75 (m, 2H), 2.43–2.38 (m, 3H), 1.22–1.10 (m, 6H), 1.05–1.02 (m, 3H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 164.35 (d, <i>J</i> = 10.0 Hz), 161.87 (d, <i>J</i> = 10.0 Hz), 151.81 (t, <i>J</i> = 13.0 Hz), 136.11, 132.88, 128.95 (d, <i>J</i> = 165.0 Hz), 127.86, 121.13, 119.12, 118.11, 110.60, 108.21, 105.57 (t, <i>J</i> = 16.0 Hz), 97.45 (d, <i>J</i> = 165.0 Hz), 96.14, 94.82 (d, <i>J</i> = 26.0 Hz), 80.95, 61.29 (d, <i>J</i> = 6.0 Hz), 56.94 (d, <i>J</i> = 21.0 Hz), 51.28 (d, <i>J</i> = 4.0 Hz), 50.16, 38.80, 27.22, 25.43 (d, <i>J</i> = 25.0 Hz), 24.44 (d, <i>J</i> = 25.0 Hz). HPLC purity: 98.3% (<i>t</i><sub>R</sub> = 11.30 min). ESI-MS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> = 470.3 (Calcd: 470.23).</div></div><div id="sec4_5_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i69" class="anchor-spacer"></div><h4 class="article-section__title" id="_i69"> 1-(3,5-Difluoro-4-((1<i>R</i>,3<i>R</i>)-2-(2-fluoro-2-methylpropyl)-3-methyl-2,3,4,9-tetrahydro-1<i>H</i>-pyrido[3,4-<i>b</i>]indol-1-yl)phenyl)azetidine-3-carboxamide (<b>D15</b>)</h4><div class="NLM_p last">Prepared using the same procedure as that for <b>D8</b>, <b>D15</b> was a white solid (0.077 g, 43%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.47 (s, 1H), 7.47 (s, 1H), 7.37 (d, <i>J</i> = 8.0 Hz, 1H), 7.17 (d, <i>J</i> = 8.0 Hz, 1H), 7.02 (s, 1H), 6.99–6.90 (m, 2H), 6.06 (d, <i>J</i> = 12.0 Hz, 2H), 5.05 (s, 1H), 3.97–3.93 (m, 2H), 3.83–3.78 (m, 2H), 3.54–3.50 (m, 1H), 3.45–3.40 (m, 1H), 2.90–2.77 (m, 2H), 2.54–2.29 (m, 2H), 1.20–1.10 (dd, <i>J</i> = 20.0 Hz, 16.0 Hz, 6H), 1.03 (d, <i>J</i> = 8.0 Hz, 3H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 173.80, 164.32 (d, <i>J</i> = 10.0 Hz), 161.84 (d, <i>J</i> = 22.0 Hz), 152.21 (t, <i>J</i> = 13.0 Hz), 136.14, 132.94, 127.84, 121.10, 118.50 (d, <i>J</i> = 101.0 Hz), 110.69, 108.09, 105.67, 97.56 (d, <i>J</i> = 165.0 Hz), 96.14, 94.86 (d, <i>J</i> = 27.0 Hz), 77.35, 77.04, 76.72, 56.94 (d, <i>J</i> = 22.0 Hz), 54.42, 51.28 (d, <i>J</i> = 4.0 Hz), 50.08, 34.10, 27.27, 25.52 (d, <i>J</i> = 25.0 Hz), 24.37 (d, <i>J</i> = 25.0 Hz). HPLC purity: 96.2% (<i>t</i><sub>R</sub> = 11.26 min). ESI-MS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> = 471.3 (Calcd: 471.23).</div></div><div id="sec4_5_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i70" class="anchor-spacer"></div><h4 class="article-section__title" id="_i70"> (1<i>R</i>,3<i>R</i>)-1-(2,6-Difluoro-4-(3-(2-fluoroethoxy)azetidin-1-yl)phenyl)-2-(2-fluoro-2-methylpropyl)-3-methyl-2,3,4,9-tetrahydro-1<i>H</i>-pyrido[3,4-<i>b</i>]indole (<b>D16</b>)</h4><div class="NLM_p last">Prepared using the same procedure as that for <b>D8</b>, <b>D16</b> as a white solid (0.056 g, 59%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.45 (s, 1H), 8.23 (s, 1H),7.37 (d, <i>J</i> = 8.0 Hz, 1H), 7.19 (d, <i>J</i> = 8.0 Hz, 1H), 6.98–6.95 (m, 2H), 6.05 (d, <i>J</i> = 4.0 Hz, 1H), 5.07 (s, 1H), 4.50 (s, 1H), 4.47 (s, 2H), 3.71 (s, 2H), 3.62 (m, 4H), 2.91–2.80 (m, 2H), 2.41–2.34 (m, 2H), 1.23–1.14 (m, 6H), 1.07 (d, <i>J</i> = 6.5 Hz, 3H). HPLC purity: 96.0% (<i>t</i><sub>R</sub> = 12.34 min). ESI-MS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> = 490.3 (Calcd: 490.24).</div></div><div id="sec4_5_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i71" class="anchor-spacer"></div><h4 class="article-section__title" id="_i71"> 1-(3,5-Difluoro-4-((1<i>R</i>,3<i>R</i>)-2-(2-fluoro-2-methylpropyl)-3-methyl-2,3,4,9-tetrahydro-1<i>H</i>-pyrido[3,4-<i>b</i>]indol-1-yl)phenyl)azetidine-3-carbonitrile (<b>D17</b>)</h4><div class="NLM_p last">Prepared using the same procedure as that for <b>D8</b>, <b>D17</b> was a white solid (0.011 g, 31%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.49 (s, 1H), 7.38 (d, <i>J</i> = 8.0 Hz, 1H), 7.18 (d, <i>J</i> = 8.0 Hz, 1H), 6.98–6.93 (m, 2H), 6.15 (d, <i>J</i> = 8.0 Hz, 2H), 5.08 (s, 1H), 4.12–4.08 (m, 2H), 4.01–3.97 (m, 2H), 3.89–3.83 (m, 1H), 2.87–2.76 (m, 3H), 2.38–2.28 (m, 2H), 1.23–1.21 (m, 3H), 1.16–1.11 (m, 3H), 1.04–1.02 (m, 3H). HPLC purity: 99.2% (<i>t</i><sub>R</sub> = 12.15 min). ESI-MS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> = 453.1 (Calcd: 453.22).</div></div><div id="sec4_5_21" class="NLM_sec NLM_sec_level_3"><div id="ac_i72" class="anchor-spacer"></div><h4 class="article-section__title" id="_i72"> 4-(3,5-Difluoro-4-((1<i>R</i>,3<i>R</i>)-2-(2-fluoro-2-methylpropyl)-3-methyl-2,3,4,9-tetrahydro-1<i>H</i>-pyrido[3,4-<i>b</i>]indol-1-yl)phenyl)morpholine (<b>D18</b>)</h4><div class="NLM_p last">Prepared using the same procedure as that for <b>D8</b>, <b>D18</b> was a light yellow solid (0.045 g, 52%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.48 (s, 1H), 7.41 (d, <i>J</i> = 8.0 Hz, 1H), 7.21 (d, <i>J</i> = 8.0 Hz, 1H), 7.02–6.95 (m, 2H), 6.59 (d, <i>J</i> = 8.0 Hz, 2H), 5.12 (s, 1H), 3.76–3.71 (m, 4H), 3.59–3.54 (m, 1H), 3.19–3.13 (m, 4H), 2.90–2.81 (m, 2H), 2.59–2.51 (m, 1H), 2.43–2.40 (m, 1H), 1.20–1.10 (m, 6H), 1.03 (d, <i>J</i> = 8.0 Hz, 3H). HPLC purity: 95.1% (<i>t</i><sub>R</sub> = 12.40 min). ESI-MS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> = 458.3 (Calcd: 458.23).</div></div><div id="sec4_5_22" class="NLM_sec NLM_sec_level_3"><div id="ac_i73" class="anchor-spacer"></div><h4 class="article-section__title" id="_i73"> 1-(4-(3,5-Difluoro-4-((1<i>R</i>,3<i>R</i>)-2-(2-fluoro-2-methylpropyl)-3-methyl-2,3,4,9-tetrahydro-1<i>H</i>-pyrido[3,4-<i>b</i>]indol-1-yl)phenyl)piperazin-1-yl)prop-2-en-1-one (<b>D19</b>)</h4><div class="NLM_p last">Prepared from 1-(piperazin-1-yl)prop-2-en-1-one, using the same procedure as that for <b>D8</b>, <b>D19</b> as a white solid (0.020 g, 48%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.50 (s, 1H), 7.41 (d, <i>J</i> = 8.0 Hz, 1H), 7.21 (d, <i>J</i> = 8.0 Hz, 1H), 7.00–6.94 (m, 2H), 6.86 (dd, <i>J</i> = 16.0 Hz, 10.5 Hz, 1H), 6.61 (d, <i>J</i> = 12.0 Hz, 1H), 6.16 (dd, <i>J</i> = 16.0 Hz, 2.5 Hz, 1H), 5.73 (dd, <i>J</i> = 10.0 Hz, 2.0 Hz, 1H), 5.12 (s, 1H), 3.70–3.66 (m, 4H), 3.56–3.53 (m, 1H), 3.26 (s, 4H), 2.91–2.81 (m, 2H), 2.58–2.57 (m, 1H), 2.43–2.35 (m, 1H), 1.24–1.20 (m, 6H), 1.06 (d, <i>J</i> = 8.0 Hz, 3H). HPLC purity: 97.0% (<i>t</i><sub>R</sub> = 11.87 min). ESI-MS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> = 511.2 (Calcd: 511.26).</div></div><div id="sec4_5_23" class="NLM_sec NLM_sec_level_3"><div id="ac_i74" class="anchor-spacer"></div><h4 class="article-section__title" id="_i74"> 4-(3,5-Difluoro-4-((1<i>R</i>,3<i>R</i>)-2-(2-fluoro-2-methylpropyl)-3-methyl-2,3,4,9-tetrahydro-1<i>H</i>-pyrido[3,4-<i>b</i>]indol-1-yl)phenyl)thiomorpholine 1,1-Dioxide (<b>D20</b>)</h4><div class="NLM_p last">Prepared using the same procedure as that for <b>D8</b>, <b>D20</b> was a white solid (0.030 g, 35%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.50 (s, 1H), 7.41 (d, <i>J</i> = 8.0 Hz, 1H), 7.21 (d, <i>J</i> = 8.0 Hz, 1H), 7.03–6.95 (m, 2H), 6.71 (d, <i>J</i> = 7.5 Hz, 1H), 5.14 (s, 1H), 3.85 (s, 4H), 3.54–3.52 (m, 1H), 3.11–3.08 (m, 4H), 2.91–2.81 (m, 2H), 2.59–2.53 (m, 1H), 2.44–2.36 (m, 1H), 1.25–1.16 (m, 6H), 1.08 (d, <i>J</i> = 8.0 Hz, 3H). HPLC purity: 95.0% (<i>t</i><sub>R</sub> = 12.01 min). ESI-MS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> = 506.2 (Calcd: 506.20).</div></div><div id="sec4_5_24" class="NLM_sec NLM_sec_level_3"><div id="ac_i75" class="anchor-spacer"></div><h4 class="article-section__title" id="_i75"> (1<i>R</i>,3<i>R</i>)-1-(4-(4,4-Difluoropiperidin-1-yl)-2,6-difluorophenyl)-2-(2-fluoro-2-methylpropyl)-3-methyl-2,3,4,9-tetrahydro-1<i>H</i>-pyrido[3,4-<i>b</i>]indole (<b>D21</b>)</h4><div class="NLM_p last">Prepared using the same procedure as that for <b>D8</b>, <b>D21</b> was a light yellow solid (0.080 g, 77%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.50 (s, 1H), 7.41 (d, <i>J</i> = 8.0 Hz, 1H), 7.21 (d, <i>J</i> = 8.0 Hz, 1H), 7.03–6.95 (m, 2H), 6.67 (d, <i>J</i> = 8.0 Hz, 2H), 5.13 (s, 1H), 3.85 (s, 4H), 3.54–3.53 (m, 1H), 3.43–3.42 (m, 4H), 2.91–2.81 (m, 2H), 2.58–2.53 (m, 1H), 2.40–2.35 (m, 1H), 1.26 (s, 3H), 1.20 (s, 3H), 1.16 (s, 3H). HPLC purity: 97.4% (<i>t</i><sub>R</sub> = 14.70 min). ESI-MS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> = 492.3 (Calcd: 492.24).</div></div><div id="sec4_5_25" class="NLM_sec NLM_sec_level_3"><div id="ac_i76" class="anchor-spacer"></div><h4 class="article-section__title" id="_i76"> (<i>E</i>)-3-(3,5-Difluoro-4-((1<i>R</i>,3<i>R</i>)-2-(2-fluoro-2-methylpropyl)-3-methyl-2,3,4,9-tetrahydro-1<i>H</i>-pyrido[3,4-<i>b</i>]indol-1-yl)phenyl)-<i>N</i>-hydroxyacrylamide (<b>D22</b>)</h4><div class="NLM_p last">To a solution of hydroxylamine hydrochloride (1.0 g, 14.4 mmol) in MeOH (5 mL) was added a solution of KOH (1.19 g, 21.15 mmol) in MeOH (2.5 mL) at 0 °C. The solution was stirred at 0 °C for 30 min and filtered to remove the white solid. Methyl (<i>E</i>)-3-(3,5-difluoro-4-((1<i>R</i>,3<i>R</i>)-2-(2-fluoro-2-methylpropyl)-3-methyl-2,3,4,9-tetrahydro-1<i>H</i>-pyrido[3,4-<i>b</i>]indol-1-yl)phenyl)acrylate (0.2 g, 0.43 mmol) was dissolved in the hydroxylamine solution, stirred for 30 min, neutralized with acetic acid, and concentrated and purified by silica gel column chromatography to give <b>D22</b> as a white solid (0.140 g, 70%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.82 (s, 1H), 10.59 (s, 1H), 9.18 (s, 1H), 7.43 (s, 1H), 7.41 (d, <i>J</i> = 8.0 Hz, 1H), 7.28 (d, <i>J</i> = 12.0 Hz, 2H), 7.18 (d, <i>J</i> = 8.0 Hz, 1H), 7.02–6.95 (m, 2H), 6.53–6.49 (m, 1H), 5.21 (s, 1H), 3.52–3.50 (m, 1H), 2.92–2.83 (m, 2H), 2.60–2.55 (m, 1H), 2.39–2.30 (m, 1H), 1.23–1.10 (dd, <i>J</i> = 16.0 Hz, 4.0 Hz, 6H), 1.05 (d, <i>J</i> = 8.0 Hz, 3H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 164.29, 163.36, 160.85, 147.92, 138.48, 136.42 (d, <i>J</i> = 21.0 Hz), 131.59, 128.34 (d, <i>J</i> = 1.0 Hz), 127.42, 121.33, 118.68 (d, <i>J</i> = 102.0 Hz), 110.92, 108.58, 97.27 (d, <i>J</i> = 165.0 Hz), 77.36, 77.04, 76.72, 56.89, 51.27, 29.72, 26.72, 24.92 (d, <i>J</i> = 14.0 Hz), 24.67 (d, <i>J</i> = 14.0 Hz). HPLC purity: 95.8% (<i>t</i><sub>R</sub> = 13.50 min). ESI-MS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> = 458.2 (Calcd: 458.20).</div></div><div id="sec4_5_26" class="NLM_sec NLM_sec_level_3"><div id="ac_i77" class="anchor-spacer"></div><h4 class="article-section__title" id="_i77"> (<i>E</i>)-3-(3,5-Difluoro-4-((1<i>R</i>,3<i>R</i>)-2-(2-fluoro-2-methylpropyl)-3-methyl-2,3,4,9-tetrahydro-1<i>H</i>-pyrido[3,4-<i>b</i>]indol-1-yl)phenyl)acrylamide (<b>D23</b>)</h4><div class="NLM_p last">Prepared using the same procedure as that for <b>D24</b>, <b>D23</b> was a white solid (0.056 g, 57%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.59 (s, 1H), 7.60–7.47 (m, 2H), 7.41 (t, <i>J</i> = 4.5 Hz, 2H), 7.26 (d, <i>J</i> = 10.2 Hz, 2H), 7.21 (d, <i>J</i> = 3.7 Hz, 1H), 7.01–6.94 (m, 2H), 6.68 (d, <i>J</i> = 15.9 Hz, 1H), 3.52 (d, <i>J</i> = 6.3 Hz, 1H), 2.96–2.88 (m, 2H), 2.61–2.53 (m, 1H), 2.38–2.28 (m, 2H), 1.23–1.15 (m, 6H), 1.05 (d, <i>J</i> = 8.0 Hz, 3H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 166.84, 163.49, 161.03 (d, <i>J</i> = 8.0 Hz), 143.41 (d, <i>J</i> = 8.0 Hz), 139.65, 136.33, 132.89 (d, <i>J</i> = 13.0 Hz), 132.10, 132.98 (d, <i>J</i> = 5.0 Hz), 132.05 (d, <i>J</i> = 10.0 Hz), 131.94 (d, <i>J</i> = 2.0 Hz), 131.51, 127.56, 125.56 (d, <i>J</i> = 12.0 Hz), 121.79 (d, <i>J</i> = 67.0 Hz), 119.13 (d, <i>J</i> = 32.0 Hz), 111.06, 110.80, 108.80, 97.14 (d, <i>J</i> = 166.0 Hz), 96.13, 56.97, 51.27, 26.98, 25.08 (d, <i>J</i> = 25.0 Hz), 24.66 (d, <i>J</i> = 24.2 Hz), 21.98 (d, <i>J</i> = 4.0 Hz). HPLC purity: 96.8% (<i>t</i><sub>R</sub> = 12.96 min). ESI-MS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> = 442.2 (Calcd: 442.20).</div></div><div id="sec4_5_27" class="NLM_sec NLM_sec_level_3"><div id="ac_i78" class="anchor-spacer"></div><h4 class="article-section__title" id="_i78"> Methyl 3′,5′-Difluoro-4′-((1<i>R</i>,3<i>R</i>)-2-(2-fluoro-2-methylpropyl)-3-methyl-2,3,4,9-tetrahydro-1<i>H</i>-pyrido[3,4-<i>b</i>]indol-1-yl)-[1,1′-biphenyl]-4-carboxylate</h4><div class="NLM_p last">To a mixture of (1<i>R</i>,3<i>R</i>)-1-(4-bromo-2,6-difluorophenyl)-2-(2-fluoro-2-methyl propyl)-3-methyl-2,3,4,9-tetrahydro-1<i>H</i>-pyrido[3,4-<i>b</i>]indole (<b>22</b>; 0.12 g, 0.265 mmol), Pd(dppf)Cl<sub>2</sub> (0.020 g, 0.026 mmol), and Na<sub>2</sub>CO<sub>3</sub> (2 M, 0.5 mL) in toluene (5 mL) was added (4-(methoxycarbonyl)phenyl)boronic acid (0.072 g, 0.4 mmol) under argon. The reaction was heated to 110 °C for 3 h. After cooling to room temperature, the mixture was concentrated and purified by silica gel column chromatography to give the desired ester as a white solid (0.120 g, 45%). ESI-MS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> = 507.2 (Calcd: 507.22).</div></div><div id="sec4_5_28" class="NLM_sec NLM_sec_level_3"><div id="ac_i79" class="anchor-spacer"></div><h4 class="article-section__title" id="_i79"> 3′,5′-Difluoro-4′-((1<i>R</i>,3<i>R</i>)-2-(2-fluoro-2-methylpropyl)-3-methyl-2,3,4,9-tetrahydro-1<i>H</i>-pyrido[3,4-<i>b</i>]indol-1-yl)-[1,1′-biphenyl]-4-carboxylic Acid (<b>D24</b>)</h4><div class="NLM_p last">Prepared using the same procedure for hydrolysis aforementioned, <b>D24</b> was a light yellow solid (0.085 g, 73%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.24 (s, 1H), 7.99 (d, <i>J</i> = 8.0 Hz, 2H), 7.78 (d, <i>J</i> = 8.0 Hz, 2H), 7.42–7.39 (m, 3H), 7.19 (d, <i>J</i> = 8.0 Hz, 2H), 7.02–6.93 (m, 2H), 5.25 (s, 1H), 3.48–3.45 (m, 1H), 2.88–2.79 (m, 2H), 2.58–2.51 (m, 1H), 2.42–2.36 (m, 1H), 1.18 (d, <i>J</i> = 16.0 Hz, 3H), 1.14 (d, <i>J</i> = 16.0 Hz, 3H), 1.02 (d, <i>J</i> = 8.0 Hz, 3H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 171.36, 163.75 (d, <i>J</i> = 8.0 Hz), 161.25 (d, <i>J</i> = 9.0 Hz), 143.36, 141.85 (t, <i>J</i> = 10.0 Hz), 136.32, 131.61, 130.95, 129.32, 127.65, 126.93, 121.53, 118.82 (d, <i>J</i> = 110.0 Hz), 117.22 (d, <i>J</i> = 15.0 Hz), 110.78, 110.55 (d, <i>J</i> = 21.0 Hz), 109.02, 97.21 (d, <i>J</i> = 166.0 Hz), 57.00 (d, <i>J</i> = 21.0 Hz), 51.21, 26.97, 25.15 (d, <i>J</i> = 25.0 Hz), 24.72 (d, <i>J</i> = 25.0 Hz). <sup>19</sup>F NMR (376 MHz, CDCl<sub>3</sub>) δ −110.23, −139.16 (hept). ESI-MS <i>m</i>/<i>z</i>: [M – H]<sup>−</sup> = 491.2 (Calcd: 491.20). HPLC purity: 98.2% (<i>t</i><sub>R</sub> = 9.88 min). HRMS <i>m</i>/<i>z</i> calcd for C<sub>29</sub>H<sub>27</sub>F<sub>3</sub>N<sub>2</sub>O<sub>2</sub>: 491.1952 [M – H]<sup>−</sup> (found: 491.1956).</div></div><div id="sec4_5_29" class="NLM_sec NLM_sec_level_3"><div id="ac_i80" class="anchor-spacer"></div><h4 class="article-section__title" id="_i80"> Ethyl 5-(4,4,5,5-Tetramethyl-1,3,2-dioxaborolan-2-yl)thiazole-2-carboxylate (<b>19</b>)</h4><div class="NLM_p last">A solution of ethyl 5-bromothiazole-2-carboxylate (4.68 g, 20 mmol), 4,4,4′,4′,5,5,5′,5′-octamethyl-2,2′-bi(1,3,2-dioxaborolane) (5.59 g, 22 mmol), potassium acetate (5.88 g, 60 mmol), and Pd(dppf)Cl<sub>2</sub> (1.4 g, 1.8 mmol) in dioxane (100 mL) was stirred under argon at 80 °C for 12 h. After cooling to room temperature, brine (120 mL) was added. The mixture was extracted with ethyl acetate. The organic layers were combined, dried with anhydrous sodium sulfate, filtered, and concentrated and purified by silica gel column chromatography to give the desired coupling product.</div></div><div id="sec4_5_30" class="NLM_sec NLM_sec_level_3"><div id="ac_i81" class="anchor-spacer"></div><h4 class="article-section__title" id="_i81"> Ethyl 5-(3,5-Difluoro-4-formylphenyl)thiazole-2-carboxylate (<b>20</b>)</h4><div class="NLM_p last">A solution of ethyl 5-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)thiazole-2-carboxylate (1.45 g, 5.1 mmol), 4-bromo-2,6-difluorobenzaldehyde (0.936 g, 4.26 mmol), Pd(dppf)Cl<sub>2</sub> (0.062 g, 0.085 mmol), and Na<sub>2</sub>CO<sub>3</sub> (2 M, 4.3 mL) in toluene (10 mL) was stirred at 100 °C for 16 h. After cooling to room temperature, brine (120 mL) was added. The mixture was extracted with ethyl acetate. The organic layers were combined, dried with anhydrous sodium sulfate, filtered, and concentrated and purified by silica gel column chromatography to give ethyl 5-(3,5-difluoro-4-formylphenyl) thiazole-2-carboxylate <b>20</b> (1.24 g, 83%).</div></div><div id="sec4_5_31" class="NLM_sec NLM_sec_level_3"><div id="ac_i82" class="anchor-spacer"></div><h4 class="article-section__title" id="_i82"> 5-(3,5-Difluoro-4-((1<i>R</i>,3<i>R</i>)-2-(2-fluoro-2-methylpropyl)-3-methyl-2,3,4,9-tetrahydro-1<i>H</i>-pyrido[3,4-<i>b</i>]indol-1-yl)phenyl)thiazole-2-carboxylic Acid (<b>D25</b>)</h4><div class="NLM_p last">Prepared using the same procedure as that for <b>D24</b>, <b>D25</b> was a light yellow solid (0.044 g, 54% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 10.62 (s, 1H), 8.60 (s, 1H), 7.57 (d, <i>J</i> = 8.0 Hz, 1H), 7.45 (s, 1H), 7.42 (d, <i>J</i> = 8.0 Hz, 1H), 7.19 (d, <i>J</i> = 8.0 Hz, 1H), 7.02–6.94 (m, 2H), 5.25 (s, 1H), 3.36–3.33 (m, 1H), 2.94–2.84 (m, 2H), 2.61–2.58 (m, 1H), 2.48–2.45 (m, 1H), 1.23–1.20 (m, 6H), 1.05 (d, <i>J</i> = 8.0 Hz, 3H). HPLC purity: 99.1% (<i>t</i><sub>R</sub> = 17.09 min). ESI-MS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> = 500.0 (Calcd: 500.15).</div></div><div id="sec4_5_32" class="NLM_sec NLM_sec_level_3"><div id="ac_i83" class="anchor-spacer"></div><h4 class="article-section__title" id="_i83"> 5-((3,5-Difluoro-4-((1<i>R</i>,3<i>R</i>)-2-(2-fluoro-2-methylpropyl)-3-methyl-2,3,4,9-tetrahydro-1<i>H</i>-pyrido[3,4-<i>b</i>]indol-1-yl)phenyl)amino)thiazole-2-carboxylic Acid (<b>D26</b>)</h4><div class="NLM_p last">Prepared using the same procedure as that for <b>B1</b>, <b>D26</b> was a light yellow solid (0.044 g, 54% yield). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.59 (s, 1H), 9.14 (s, 1H), 8.12 (s, 1H), 7.69 (d, <i>J</i> = 12.0 Hz, 2H), 7.42 (d, <i>J</i> = 12.0 Hz, 1H), 7.19 (d, <i>J</i> = 8.0 Hz, 1H), 7.03–6.93 (m, 2H), 5.23 (s, 1H), 3.56–3.50 (m, 1H), 2.95–2.88 (m, 2H), 2.62–2.56 (m, 1H), 2.44–2.39 (m, 1H), 1.26–1.21 (m, 6H), 1.07–1.05 (m, 3H). HPLC purity: 96.8% (<i>t</i><sub>R</sub> = 13.11 min). ESI-MS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> = 515.0 (Calcd: 515.17).</div></div><div id="sec4_5_33" class="NLM_sec NLM_sec_level_3"><div id="ac_i84" class="anchor-spacer"></div><h4 class="article-section__title" id="_i84"> 1-(3,5-Difluoro-4-((1<i>R</i>,3<i>R</i>)-2-(2-fluoro-2-methylpropyl)-3-methyl-2,3,4,9-tetrahydro-1<i>H</i>-pyrido[3,4-<i>b</i>]indol-1-yl)phenyl)-1<i>H</i>-pyrazole-4-carboxylic Acid (<b>D27</b>)</h4><div class="NLM_p last">Prepared using the same procedure as that for <b>D9</b>, <b>D27</b> was a light yellow solid (0.057 g, 65% yield). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.65 (s, 1H), 9.14 (s, 1H), 8.34 (d, <i>J</i> = 8.0 Hz, 1H), 8.21 (d, <i>J</i> = 8.0 Hz, 1H), 7.86 (d, <i>J</i> = 8.0 Hz, 2H), 7.43 (d, <i>J</i> = 8.0 Hz, 1H), 7.20 (d, <i>J</i> = 8.0 Hz, 1H), 7.01–6.96 (m, 2H), 5.28 (s, 1H), 3.55 (s, 1H), 2.91–2.86 (m, 2H), 2.60–2.53 (m, 1H), 2.46–2.38 (m, 1H), 1.23 (d, <i>J</i> = 20.0 Hz, 3H), 1.15 (d, <i>J</i> = 20.0 Hz, 3H), 1.07 (d, <i>J</i> = 8.0 Hz, 3H). HPLC purity: 96.0% (<i>t</i><sub>R</sub> = 14.67 min). ESI-MS <i>m</i>/<i>z</i>: [M – H]<sup>−</sup> = 481.2 (Calcd: 481.19).</div></div><div id="sec4_5_34" class="NLM_sec NLM_sec_level_3"><div id="ac_i85" class="anchor-spacer"></div><h4 class="article-section__title" id="_i85"> Methyl 6-(3,5-Difluoro-4-formylphenyl)nicotinate</h4><div class="NLM_p last">A solution of methyl 6-bromonicotinate (0.400 g, 1.389 mmol), (3,5-difluoro-4-formylphenyl)boronic acid (0.284 g, 1.528 mmol), Na<sub>2</sub>CO<sub>3</sub> (2 M, 2 mL), and Pd(dppf)Cl<sub>2</sub> (0.02 g. 0.027 mmol) in toluene (15 mL) was heated at 110 °C under argon for 8 h. After cooling to room temperature, brine (30 mL) was added. The mixture was extracted with ethyl acetate. The organic layers were combined, dried with anhydrous sodium sulfate, filtered, and concentrated and purified by silica gel column chromatography to give methyl 6-(3,5-difluoro-4-formylphenyl) nicotinate as a yellow solid (0.15 g, 39%).</div></div><div id="sec4_5_35" class="NLM_sec NLM_sec_level_3"><div id="ac_i86" class="anchor-spacer"></div><h4 class="article-section__title" id="_i86"> 6-(3,5-Difluoro-4-((1<i>R</i>,3<i>R</i>)-2-(2-fluoro-2-methylpropyl)-3-methyl-2,3,4,9-tetrahydro-1<i>H</i>-pyrido[3,4-<i>b</i>]indol-1-yl)phenyl)nicotinic Acid (<b>D28</b>)</h4><div class="NLM_p last">Prepared using the same procedure as that for <b>D24</b>, <b>D28</b> was a light yellow solid (0.031 g, 55%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.63 (s, 1H), 8.22 (d, <i>J</i> = 8.0 Hz, 1H), 7.98 (d, <i>J</i> = 8.0 Hz, 1H), 7.61 (d, <i>J</i> = 8.0 Hz, 1H), 7.46–7.41 (m, 3H), 7.20 (d, <i>J</i> = 8.0 Hz, 1H), 7.02–6.94 (m, 2H), 5.28 (s, 1H), 3.56–3.52 (m, 1H), 2.94–2.90 (m, 2H), 2.62–2.60 (m, 1H), 2.50–2.45 (m, 1H), 1.21 (d, <i>J</i> = 12.0 Hz, 3H), 1.17 (d, <i>J</i> = 12.0 Hz, 3H), 1.09 (d, <i>J</i> = 12.0 Hz, 3H). HPLC purity: 97.0% (<i>t</i><sub>R</sub> = 15.62 min). ESI-MS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> = 494.2 (Calcd: 494.20).</div></div><div id="sec4_5_36" class="NLM_sec NLM_sec_level_3"><div id="ac_i87" class="anchor-spacer"></div><h4 class="article-section__title" id="_i87"> 3′,5′-Difluoro-4′-((1<i>R</i>,3<i>R</i>)-2-(2-fluoro-2-methylpropyl)-3-methyl-2,3,4,9-tetrahydro-1<i>H</i>-pyrido[3,4-<i>b</i>]indol-1-yl)-[1,1′-biphenyl]-3-carboxylic Acid (<b>D29</b>)</h4><div class="NLM_p last">Prepared using the same procedure as that for <b>D24</b>, <b>D29</b> was a light yellow solid (0.031 g, 55%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.69 (s, 1H), 7.61 (d, <i>J</i> = 8.0 Hz, 2H), 7.50 (d, <i>J</i> = 8.0 Hz, 1H), 7.42 (d, <i>J</i> = 8.0 Hz, 1H), 7.25–7.20 (m, 3H), 7.01–6.96 (m, 2H), 5.30 (s, 1H), 3.86 (s, 3H), 3.51–3.49 (m, 1H), 2.89–2.78 (m, 2H), 2.62–2.51 (m, 1H), 2.45–2.36 (m, 1H), 1.23–1.15 (m, 6H), 1.05 (d, <i>J</i> = 8.0 Hz, 3H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 170.88, 163.72 (d, <i>J</i> = 9.0 Hz), 161.17, 142.07, 138.58, 136.29, 131.67 (d, <i>J</i> = 15.0 Hz), 130.75, 130.09, 129.22, 128.44, 127.65, 121.43, 119.28, 118.75 (d, <i>J</i> = 106.0 Hz), 110.44, 110.20, 108.82, 97.25 (d, <i>J</i> = 166.0 Hz), 96.13, 57.02 (d, <i>J</i> = 22.0 Hz), 51.13 (d, <i>J</i> = 15.0 Hz), 27.04, 25.23 (d, <i>J</i> = 24.0 Hz), 24.63 (d, <i>J</i> = 25.0 Hz). HPLC purity: 96.2% (<i>t</i><sub>R</sub> = 15.87 min). ESI-MS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> = 493.2 (Calcd: 493.20).</div></div><div id="sec4_5_37" class="NLM_sec NLM_sec_level_3"><div id="ac_i88" class="anchor-spacer"></div><h4 class="article-section__title" id="_i88"> 3′,5′-Difluoro-4′-((1<i>R</i>,3<i>R</i>)-2-(2-fluoro-2-methylpropyl)-3-methyl-2,3,4,9-tetrahydro-1<i>H</i>-pyrido[3,4-<i>b</i>]indol-1-yl)-2-methoxy-[1,1′-biphenyl]-4-carboxylic Acid (<b>D30</b>)</h4><div class="NLM_p last">Prepared using the same procedure as that for <b>D24</b>, <b>D30</b> was a white solid (0.049 g, 58%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.62 (s, 1H), 7.71 (d, <i>J</i> = 8.0 Hz, 1H), 7.52 (d, <i>J</i> = 12.0 Hz, 2H), 7.42 (d, <i>J</i> = 8.0 Hz, 1H), 7.39 (d, <i>J</i> = 8.0 Hz, 1H), 7.20 (d, <i>J</i> = 8.0 Hz, 1H), 7.21–7.19 (m, 1H), 7.01–6.96 (m, 2H), 5.28 (s, 1H), 3.92 (s, 3H), 3.56–3.52 (m, 1H), 2.94–2.90 (m, 2H), 2.62–2.60 (m, 1H), 2.51–2.48 (m, 1H), 1.24–1.21 (m, 6H), 1.09 (d, <i>J</i> = 8.0 Hz, 3H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 170.63, 160.48, 156.17, 139.38, 136.22, 132.57, 131.82, 130.35, 127.59, 122.90, 121.39, 118.74 (d, <i>J</i> = 107.0 Hz), 116.44, 112.96, 112.72, 112.55, 110.75, 108.85, 97.23 (d, <i>J</i> = 165.0 Hz), 96.13, 56.74 (d, <i>J</i> = 21.0 Hz), 55.73, 51.15 (d, <i>J</i> = 28.0 Hz), 26.89, 25.18 (d, <i>J</i> = 24.0 Hz), 24.64 (d, <i>J</i> = 25.0 Hz). HPLC purity: 96.6% (<i>t</i><sub>R</sub> = 15.77 min). ESI-MS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> = 523.2 (Calcd: 523.21).</div></div><div id="sec4_5_38" class="NLM_sec NLM_sec_level_3"><div id="ac_i89" class="anchor-spacer"></div><h4 class="article-section__title" id="_i89"> 3′,5′-Difluoro-4′-((1<i>R</i>,3<i>R</i>)-2-(2-fluoro-2-methylpropyl)-3-methyl-2,3,4,9-tetrahydro-1<i>H</i>-pyrido[3,4-<i>b</i>]indol-1-yl)-3-methoxy-[1,1′-biphenyl]-4-carboxylic Acid (<b>D31</b>)</h4><div class="NLM_p last">Prepared using the same procedure as that for <b>D24</b>, <b>D31</b> was a white solid (0.120 g, 88%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.48 (s, 1H), 8.20 (d, <i>J</i> = 8.0 Hz, 2H), 7.68 (d, <i>J</i> = 8.0 Hz, 2H), 7.57–7.51 (m, 3H), 7.25–7.18 (m, 3H), 5.73 (s, 1H), 3.88–3.86 (m, 1H), 3.26–3.23 (m, 1H), 3.07–3.00 (m, 1H), 2.72–2.67 (m, 1H), 2.34–2.23 (m, 1H), 1.24–1.20 (m, 6H), 1.09 (d, <i>J</i> = 8.0 Hz, 3H). HPLC purity: 96.0% (t<sub>R</sub> = 15.77 min). ESI-MS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> = 523.2 (Calcd: 523.21).</div></div><div id="sec4_5_39" class="NLM_sec NLM_sec_level_3"><div id="ac_i90" class="anchor-spacer"></div><h4 class="article-section__title" id="_i90"> 2-Chloro-3′,5′-difluoro-4′-((1<i>R</i>,3<i>R</i>)-2-(2-fluoro-2-methylpropyl)-3-methyl-2,3,4,9-tetrahydro-1<i>H</i>-pyrido[3,4-<i>b</i>]indol-1-yl)-6-methoxy-[1,1′-biphenyl]-4-carboxylic Acid (<b>D32</b>)</h4><div class="NLM_p last">Prepared using the same procedure as that for <b>D24</b>, <b>D32</b> was a white solid (0.078 g, 76%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.42 (s, 1H), 8.01 (d, <i>J</i> = 8.0 Hz, 2H), 7.89 (d, <i>J</i> = 8.0 Hz, 2H), 7.39 (d, <i>J</i> = 8.0 Hz, 1H), 7.27 (s, 1H), 7.16 (d, <i>J</i> = 8.0 Hz, 1H), 7.07 (d, <i>J</i> = 8.0 Hz, 1H), 6.98–6.91 (m, 2H), 5.37 (s, 1H), 3.95 (s, 3H), 3.59–3.56 (m, 1H), 2.95–2.87 (m, 2H), 2.59–2.56 (m, 1H), 2.46–2.34 (m,1H), 1.24–1.20 (m, 6H), 1.09 (d, <i>J</i> = 8.0 Hz, 3H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 170.95, 164.38, 161.88, 159.88 (d, <i>J</i> = 8.0 Hz), 144.62, 141.67, 141.56, 136.10, 133.18, 130.77, 129.29, 127.90, 126.95, 120.97, 118.55 (d, <i>J</i> = 103.0 Hz), 117.07 (d, <i>J</i> = 12.0 Hz), 110.58, 107.93 (d, <i>J</i> = 18.0 Hz), 105.36, 97.54 (d, <i>J</i> = 165.0 Hz), 57.27 (d, <i>J</i> = 24.0 Hz), 56.21, 51.49, 50.25, 27.39, 25.48 (d, <i>J</i> = 24.0 Hz), 24.43 (d, <i>J</i> = 25.0 Hz). HPLC purity: 97.1% (<i>t</i><sub>R</sub> = 11.13 min). ESI-MS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> = 557.2 (Calcd: 557.17).</div></div><div id="sec4_5_40" class="NLM_sec NLM_sec_level_3"><div id="ac_i91" class="anchor-spacer"></div><h4 class="article-section__title" id="_i91"> 3′,5′-Difluoro-4′-((1<i>R</i>,3<i>R</i>)-2-(2-fluoro-2-methylpropyl)-3-methyl-2,3,4,9-tetrahydro-1<i>H</i>-pyrido[3,4-<i>b</i>]indol-1-yl)-[1,1′-biphenyl]-4-carboxamide (<b>D33</b>)</h4><div class="NLM_p last">A solution of methyl 3′,5′-difluoro-4′-((1<i>R</i>,3<i>R</i>)-2-(2-fluoro-2-methylpropyl)-3-methyl-2,3,4,9-tetrahydro-1<i>H</i>-pyrido[3,4-<i>b</i>]indol-1-yl)-[1,1′-biphenyl]-4-carboxylate (0.10 g, 0.2 mmol) in ammonia (ca. 7 mol/L, 20 mL) was stirred overnight at 65 °C. After cooling to room temperature, the mixture was concentrated and purified by silica gel column chromatography to give <b>D33</b> as a white solid (0.023 g, 24%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.38 (s, 1H), 8.02 (s, 1H), 7.74 (d, <i>J</i> = 8.0 Hz, 1H), 7.72 (d, <i>J</i> = 8.0 Hz, 1H), 7.42–7.39 (m 2H), 7.39 (d, <i>J</i> = 8.0 Hz, 1H), 7.17 (d, <i>J</i> = 8.0 Hz, 1H), 7.00–6.91 (m, 2H), 5.64 (s, 1H), 3.94 (s, 3H), 3.76–3.73 (m, 1H), 3.13–3.09 (m, 1H), 3.03–2.95 (m, 1H), 2.69–2.64 (m, 1H), 2.34–2.23 (m, 1H), 1.18 (d, <i>J</i> = 12.0 Hz, 3H), 1.13 (d, <i>J</i> = 12.0 Hz, 3H), 1.08–1.06 (m, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.73, 163.74 (d, <i>J</i> = 9.0 Hz), 161.26 (d, <i>J</i> = 8.0 Hz), 141.63 (d, <i>J</i> = 10.0 Hz), 140.17, 136.75, 134.59, 132.58, 128.67, 127.35, 126.97, 120.90, 118.33 (d, <i>J</i> = 64.0 Hz), 117.17, 111.42, 110.48 (d, <i>J</i> = 24.0 Hz), 107.16, 97.40 (d, <i>J</i> = 166.0 Hz), 56.73, 51.65, 51.09, 26.97, 25.16 (d, <i>J</i> = 24.0 Hz), 13.85. HPLC purity: 97.1% (<i>t</i><sub>R</sub> = 14.91 min). ESI-MS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> = 492.1 (Calcd: 492.22).</div></div><div id="sec4_5_41" class="NLM_sec NLM_sec_level_3"><div id="ac_i92" class="anchor-spacer"></div><h4 class="article-section__title" id="_i92"> 2-Chloro-3′,5′-difluoro-4′-((1<i>R</i>,3<i>R</i>)-2-(2-fluoro-2-methylpropyl)-3-methyl-2,3,4,9-tetrahydro-1<i>H</i>-pyrido[3,4-<i>b</i>]indol-1-yl)-[1,1′-biphenyl]-4-carboxylic Acid (<b>D34</b>)</h4><div class="NLM_p last">Prepared using the same procedure as that for <b>D24</b>, <b>D34</b> was a white solid (0.020 g, 76%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.70 (s, 1H), 8.03 (d, <i>J</i> = 8.0 Hz, 1H), 7.96 (d, <i>J</i> = 8.0 Hz, 1H),, 7.59 (d, <i>J</i> = 8.0 Hz, 1H), 7.43 (d, <i>J</i> = 8.0 Hz, 1H), 7.23–7.21 (m, 3H), 7.01–6.95 (m, 2H), 5.32 (s, 1H), 3.54–3.50 (m, 1H), 2.95–2.85 (m, 2H), 2.63–2.59 (m, 1H), 2.46–2.37 (m, 1H), 1.25 (d, <i>J</i> = 20.0 Hz, 3H), 1.17 (d, <i>J</i> = 20.0 Hz, 3H), 1.09 (d, <i>J</i> = 8.0 Hz, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 166.34, 162.90, 160.41, 141.45, 140.14, 136.74, 133.33, 132.35, 132.22, 131.84, 130.90, 128.70, 127.31, 120.93, 118.35 (d, <i>J</i> = 63.0 Hz), 117.86, 113.35 (d, <i>J</i> = 25.0 Hz), 111.45, 107.36, 97.38 (d, <i>J</i> = 166.0 Hz), 56.48, 56.26, 51.95, 51.03, 26.80, 25.12 (d, <i>J</i> = 24.0 Hz). HPLC purity: 98.9% (<i>t</i><sub>R</sub> = 16.81 min). ESI-MS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> = 527.3 (Calcd: 527.16).</div></div><div id="sec4_5_42" class="NLM_sec NLM_sec_level_3"><div id="ac_i93" class="anchor-spacer"></div><h4 class="article-section__title" id="_i93"> 3′,5′-Dichloro-4′-((1<i>R</i>,3<i>R</i>)-2-(2-fluoro-2-methylpropyl)-3-methyl-2,3,4,9-tetrahydro-1<i>H</i>-pyrido[3,4-<i>b</i>]indol-1-yl)-3-methoxy-[1,1′-biphenyl]-4-carboxylic Acid (<b>D35</b>)</h4><div class="NLM_p last">Prepared using the same procedure as that for <b>D24</b>, <b>D35</b> was a white solid (0.047 g, 72%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.62 (s, 1H), 8.06 (s, 1H), 7.97 (d, <i>J</i> = 8.0 Hz, 2H), 7.86 (d, <i>J</i> = 8.0 Hz, 2H), 7.52–7.42 (m, 4H), 7.22–7.18 (m, 1H), 7.03–6.94 (m, 2H), 5.28 (s, 1H), 3.56–3.52 (m, 1H), 2.94–2.89 (m, 2H), 2.63–2.58 (m, 1H), 2.46–2.37 (m, 1H), 1.26–1.19 (m, 6H), 1.08 (d, <i>J</i> = 8.0 Hz, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.40, 159.12, 141.69, 141.08, 138.07 (d, <i>J</i> = 15.0 Hz), 136.87, 135.13, 131.91, 131.77, 129.25, 127.19, 126.76, 121.59, 120.76, 118.95, 118.28 (d, <i>J</i> = 60.0 Hz), 111.48, 111.27, 107.36, 97.48 (d, <i>J</i> = 165.0 Hz), 56.52, 55.69, 52.16 (d, <i>J</i> = 5.0 Hz), 27.98, 25.92 (d, <i>J</i> = 24.0 Hz), 24.64 (d, <i>J</i> = 25.0 Hz), 9.99. HPLC purity: 97.5% (<i>t</i><sub>R</sub> = 17.35 min). ESI-MS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> = 555.1 (Calcd: 555.15).</div></div><div id="sec4_5_43" class="NLM_sec NLM_sec_level_3"><div id="ac_i94" class="anchor-spacer"></div><h4 class="article-section__title" id="_i94"> 3′,5′-Dichloro-4′-((1<i>R</i>,3<i>R</i>)-2-(2-fluoro-2-methylpropyl)-3-methyl-2,3,4,9-tetrahydro-1<i>H</i>-pyrido[3,4-<i>b</i>]indol-1-yl)-[1,1′-biphenyl]-4-carboxylic Acid (<b>D36</b>)</h4><div class="NLM_p last">Prepared using the same procedure as that for <b>D24</b>, <b>D36</b> was a white solid (0.061 g, 54%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.49 (s, 1H), 7.77 (s, 1H), 7.64 (d, <i>J</i> = 8.0 Hz, 1H), 7.60 (s, 1H), 7.41 (d, <i>J</i> = 8.0 Hz, 1H), 7.41–7.35 (m, 2H), 7.18 (d, <i>J</i> = 8.0 Hz, 1H), 7.00–6.92 (m, 2H), 5.38 (s, 1H), 3.91 (s, 3H), 3.70–3.66 (m, 1H), 3.08–3.03 (m, 2H), 2.64–2.60 (m, 1H), 2.38–2.27 (m, 1H), 1.16 (d, <i>J</i> = 8.0 Hz, 3H), 1.08 (d, <i>J</i> = 8.0 Hz, 3H), 1.06–1.04 (m, 3H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 142.56, 141.27, 136.25, 135.04, 130.94, 130.90, 129.64, 127.54, 126.95, 126.51, 121.37, 118.72 (d, <i>J</i> = 116.0 Hz), 110.74, 109.28, 97.28 (d, <i>J</i> = 166.0 Hz), 96.14, 55.58, 52.00 (d, <i>J</i> = 5.0 Hz), 27.76, 25.92 (d, <i>J</i> = 25.0 Hz), 24.34 (d, <i>J</i> = 24.0 Hz). HPLC purity: 97.3% (<i>t</i><sub>R</sub> = 17.41 min). ESI-MS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> = 525.2 (Calcd: 525.14).</div></div><div id="sec4_5_44" class="NLM_sec NLM_sec_level_3"><div id="ac_i95" class="anchor-spacer"></div><h4 class="article-section__title" id="_i95"> 3′-Chloro-5′-fluoro-4′-((1<i>R</i>,3<i>R</i>)-2-(2-fluoro-2-methylpropyl)-3-methyl-2,3,4,9-tetrahydro-1<i>H</i>-pyrido[3,4-<i>b</i>]indol-1-yl)-3-methoxy-[1,1′-biphenyl]-4-carboxylic Acid (<b>D37</b>)</h4><div class="NLM_p last">Prepared using the same procedure as that for <b>D24</b>, <b>D37</b> was a white solid (0.055 g, 81%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.51 (s, 1H), 8.01 (d, <i>J</i> = 8.0 Hz, 2H), 7.87 (d, <i>J</i> = 8.0 Hz, 2H), 7.74 (s, 1H), 7.59 (d, <i>J</i> = 8.0 Hz, 1H), 7.41 (d, <i>J</i> = 8.0 Hz, 1H), 7.18–7.15 (m, 1H), 6.99–6.94 (m, 2H), 5.38 (s, 1H), 3.70–3.66 (m, 1H), 3.09–2.98 (m, 2H), 2.64–2.60 (m, 1H), 2.37–2.26 (m, 1H), 1.14 (d, <i>J</i> = 8.0 Hz, 3H), 1.09 (d, <i>J</i> = 8.0 Hz, 3H), 1.08–1.05 (m, 3H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 164.69, 158.37, 151.73, 144.82, 141.19, 136.26, 134.65, 131.22, 127.66, 124.59, 121.50, 120.80, 119.38, 118.26, 117.48, 110.39 (d, <i>J</i> = 60.0 Hz), 108.77, 97.05 (d, <i>J</i> = 166.0 Hz), 58.26, 58.03, 56.96, 52.00, 27.68, 25.54 (d, <i>J</i> = 25.0 Hz), 24.34 (d, <i>J</i> = 25.0 Hz). HPLC purity: 96.4% (<i>t</i><sub>R</sub> = 9.59 min). ESI-MS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> = 539.1 (Calcd: 539.18).</div></div><div id="sec4_5_45" class="NLM_sec NLM_sec_level_3"><div id="ac_i96" class="anchor-spacer"></div><h4 class="article-section__title" id="_i96"> 3′-Chloro-5′-fluoro-4′-((1<i>R</i>,3<i>R</i>)-2-(2-fluoro-2-methylpropyl)-3-methyl-2,3,4,9-tetrahydro-1<i>H</i>-pyrido[3,4-<i>b</i>]indol-1-yl)-[1,1′-biphenyl]-4-carboxylic Acid (<b>D38</b>)</h4><div class="NLM_p last">Prepared using the same procedure as that for <b>D24</b>, <b>D38</b> was a white solid (0.047 g, 85%).<sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.65 (s, 1H), 9.14 (s, 1H), 8.34 (d, <i>J</i> = 8.0 Hz, 1H), 8.21 (d, <i>J</i> = 8.0 Hz, 1H), 7.86 (d, <i>J</i> = 8.0 Hz, 2H), 7.43 (d, <i>J</i> = 8.0 Hz, 1H), 7.20 (d, <i>J</i> = 8.0 Hz, 1H), 7.01–6.96 (m, 2H), 5.28 (s, 1H), 3.55 (s, 1H), 2.91–2.86 (m, 2H), 2.60–2.53 (m, 1H), 2.46–2.43 (m, 1H), 1.23 (d, <i>J</i> = 20.0 Hz, 3H), 1.15 (d, <i>J</i> = 20.0 Hz, 3H), 1.07 (d, <i>J</i> = 8.0 Hz, 3H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 170.93, 164.29, 161.75, 142.89, 141.78 (d, <i>J</i> = 9 Hz), 137.02, 136.26, 131.41, 130.92, 129.56, 127.68, 126.93, 126.52 (d, <i>J</i> = 12.0 Hz), 124.54, 121.41, 118.77 (d, <i>J</i> = 108.0 Hz), 113.68 (d, <i>J</i> = 24.0 Hz), 110.73, 108.65, 97.24 (d, <i>J</i> = 165.0 Hz), 57.99 (d, <i>J</i> = 21.0 Hz), 53.18, 51.84 (d, <i>J</i> = 5.0 Hz), 27.71, 25.69 (d, <i>J</i> = 25.0 Hz), 24.22 (d, <i>J</i> = 23.8 Hz). HPLC purity: 97.4% (<i>t</i><sub>R</sub> = 16.84 min). ESI-MS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> = 509.1 (Calcd: 509.17).</div></div><div id="sec4_5_46" class="NLM_sec NLM_sec_level_3"><div id="ac_i97" class="anchor-spacer"></div><h4 class="article-section__title" id="_i97"> 6-(3-Fluoro-4-((1<i>R</i>,3<i>R</i>)-2-(2-fluoro-2-methylpropyl)-3-methyl-2,3,4,9-tetrahydro-1<i>H</i>-pyrido[3,4-<i>b</i>]indol-1-yl)-5-methoxyphenyl)nicotinic Acid (<b>D39</b>)</h4><div class="NLM_p last">Prepared using the same procedure as that for <b>D24</b>, <b>D39</b> was a white solid (0.033 g, 65%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.77 (s, 1H), 7.65 (d, <i>J</i> = 8.0 Hz, 1H), 7.56 (d, <i>J</i> = 8.0 Hz, 1H), 7.41 (d, <i>J</i> = 8.0 Hz, 1H), 7.23 (d, <i>J</i> = 8.0 Hz, 1H), 7.02–6.98 (m, 4H), 5.13 (s, 1H), 3.79 (s, 3H), 3.52–3.49 (m, 1H), 2.91–2.85 (m, 2H), 2.62–2.58 (m, 1H), 2.51–2.49 (m, 1H), 1.23 (d, <i>J</i> = 20.0 Hz, 3H), 1.15 (d, <i>J</i> = 20.0 Hz, 3H), 1.09 (d, <i>J</i> = 8.0 Hz, 3H). HPLC purity: 97.5% (<i>t</i><sub>R</sub> = 16.84 min). ESI-MS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> = 506.2 (Calcd: 506.21).</div></div><div id="sec4_5_47" class="NLM_sec NLM_sec_level_3"><div id="ac_i98" class="anchor-spacer"></div><h4 class="article-section__title" id="_i98"> 3′-Fluoro-4′-((1<i>R</i>,3<i>R</i>)-2-(2-fluoro-2-methylpropyl)-3-methyl-2,3,4,9-tetrahydro-1<i>H</i>-pyrido[3,4-<i>b</i>]indol-1-yl)-5′-methoxy-[1,1′-biphenyl]-4-carboxylic Acid (<b>D40</b>)</h4><div class="NLM_p last">Prepared using the same procedure as that for <b>D24</b>, <b>D40</b> was a white solid (0.055 g, 71%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.42 (s, 1H), 8.01 (d, <i>J</i> = 8.3 Hz, 2H), 7.88 (d, <i>J</i> = 8.4 Hz, 2H), 7.39 (d, <i>J</i> = 7.3 Hz, 1H), 7.27 (s, 1H), 7.16 (d, <i>J</i> = 7.4 Hz, 1H), 7.07 (d, <i>J</i> = 11.5 Hz, 1H), 7.00–6.89 (m, 2H), 5.37 (s, 1H), 3.95 (s, 3H), 3.59 (d, <i>J</i> = 5.7 Hz, 1H), 2.99–2.82 (m, 2H), 2.57 (dd, <i>J</i> = 14.9, 3.0 Hz, 1H), 2.42–2.31 (m, 1H), 1.27–1.21 (m, 6H), 1.02 (d, <i>J</i> = 8.0 Hz, 1H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 171.28, 164.39, 161.88, 159.94, 159.85, 144.88, 141.56, 136.10, 133.06, 130.83, 128.86, 127.87, 127.03, 121.03, 119.10, 118.04, 110.60, 108.11, 107.87, 106.72 (d, <i>J</i> = 265.0 Hz), 97.48 (d, <i>J</i> = 166.0 Hz), 57.22 (d, <i>J</i> = 23.0 Hz), 56.22, 51.54, 50.34, 27.35, 25.47 (d, <i>J</i> = 24.0 Hz), 24.49 (d, <i>J</i> = 25.0 Hz). HPLC purity: 97.8% (<i>t</i><sub>R</sub> = 11.60 min). ESI-MS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> = 505.3 (Calcd: 505.22).</div></div><div id="sec4_5_48" class="NLM_sec NLM_sec_level_3"><div id="ac_i99" class="anchor-spacer"></div><h4 class="article-section__title" id="_i99"> Cell Lines and Culture Conditions</h4><div class="NLM_p last">MCF-7 cell line was purchased from ATCC (ATCC No. HTB-22) and cultured following the instructions. Briefly, MCF-7 cell line was cultured in MEM media (Eagle’s minimum essential medium, GIBCO, No. 11095-080) containing 10% FBS (GIBCO, No. 10091-148), 1% penicillin–streptomycin solution (GIBCO, No. 15140-122), 1% NEAA (GIBCO, No. 11140-050), 1 mM sodium pyruvate (GIBCO, No. 11360-070), and 0.01 mg/mL insulin (Jiangsu Wanbang Biopharmaceutical Co. Ltd.) in an incubator (37 °C, 5% CO<sub>2</sub>). Cells were detached by 0.25% trypsin-EDTA (GIBCO No. 25200-056) when reached to 80–90% confluence for further culture or assays. For immunofluorescence staining of ERα and PR level, MCF-7 cell line was cultured according to the protocol of Pharmaron Inc. The cells were cultured in DMEM containing 10% hormone-free FBS and 1% penicillin–streptomycin solution. The hormone-free FBS was produced by incubating 50 mL of FBS with 1 g of activated carbon for 24 h in 4 °C.</div></div><div id="sec4_5_49" class="NLM_sec NLM_sec_level_3"><div id="ac_i100" class="anchor-spacer"></div><h4 class="article-section__title" id="_i100"> Cell Proliferation Assay</h4><div class="NLM_p last">MCF-7 cell line was suspended at 2 × 10<sup>4</sup> cells/mL, and 100 μL/well was plated in a 96-well plate. Then the cells were incubated for 24 h at 37 °C and 5% CO<sub>2</sub> under humidified conditions. A 20 mM amount of solution of each compound was prepared with DMSO, and the solution was diluted to a series of 2× final concentration with complete RPMI 1640 media. Then 100 μL of the diluted solution was added to the well. After the cells were incubated for 5 days, the medium was discarded and Celltiter-Glo reagent was added to the well according to the instruction of the manufacture. Finally, the luminescence was read on a plate reader and the inhibition rate was calculated as %Inh = 100 – [(compd signal – min signal)/(max signal – min signal) × 100].</div></div><div id="sec4_5_50" class="NLM_sec NLM_sec_level_3"><div id="ac_i101" class="anchor-spacer"></div><h4 class="article-section__title" id="_i101"> ERα Binding Affinity</h4><div class="NLM_p last">ER binding assay was performed by Pharmaron Inc. Briefly, the compound was diluted with DMSO (final concentration is 0.3%) and added to the assay plate by Acho instrument. ERα and Fluormone ES2 were diluted with ES2 screening buffer, and the final concentrations were 150 and 4.5 nM, respectively. Then 20 μL of each solution was added to the plate and mixed well. The plate was centrifuged 1000 rpm at room temperature for 1 min and incubated for 60 min in dark. Finally, the fluorescence polarization value (mP) was read by Envision.</div></div><div id="sec4_5_51" class="NLM_sec NLM_sec_level_3"><div id="ac_i102" class="anchor-spacer"></div><h4 class="article-section__title" id="_i102"> ERα Degradation Activity</h4><div class="NLM_p last">ERα was measured by immunofluorescent staining, which was conducted by Pharmaron Inc. Briefly, MCF-7 cells were suspended at 1500 cells/40 and 40 μL was inoculated in each well of a 384-well plate. After the cells were equilibrated at room temperature for 20 min followed by incubation for 24 h (37 °C, 5% CO<sub>2</sub>), the compound was diluted with DMSO (final concentration is 0.3%) and added to the assay plate by Acho instrument. Then the cells were washed with PBS and fixed by 3.7% paraformaldehyde after incubation for 24 h. Then the cells were washed two times with PBS and penetrated with 0.5% Tween-20 for 1 h at room temperature, followed by being washed two times with PBST. The primary antibody of anti-ERα (CST No. 8644S) was diluted to 1:1000 and 1:250 in 1% milk in PBST, respectively. After incubation with primary antibody for 1.5 h in room temperature, the cells were washed two times with PBST and incubated with diluted secondary antibody (1:1000 in PBST) for 45 min at room temperature. Finally, the cells were washed three times with PBST and two times with PBS and the cells were scanned with Acumen.</div></div><div id="sec4_5_52" class="NLM_sec NLM_sec_level_3"><div id="ac_i103" class="anchor-spacer"></div><h4 class="article-section__title" id="_i103"> Western Blot Analysis</h4><div class="NLM_p last">Cells were lysed in RIPA lysis buffer (Beyotime) with protease inhibitors (Solarbio) and proteins run on 4–12% precast gels (GenScript). Membranes were probed overnight with primary antibodies (anti-ERα, CST No. 8644; anti-PR, CST No. 8757; anti-GAPDH, CST No. 5174) followed by incubation with HRP-tagged secondary antibodies (CST No. 7074) and visualized with Chemidoc digital imager (Bio-RAD).</div></div><div id="sec4_5_53" class="NLM_sec NLM_sec_level_3"><div id="ac_i104" class="anchor-spacer"></div><h4 class="article-section__title" id="_i104"> In Vivo Efficacy in the MCF-7 Breast Cancer Xenograft Model</h4><div class="NLM_p last">Female BALB/C nude mice (6 to 8 weeks old) in this study were purchased from Shanghai Sippe-Bk Lab Animal Co., Ltd. and were housed under standard specific-pathogen-free (SPF) conditions. All of the procedures and protocols were approved by The Institutional Animal Care and Use Committee (IACUC) of Nanjing University of Chinese Medicine and in accordance with the National Institutes of Health Guide for the Care and Use of Laboratory Animals. A well-established tumorigenesis assay was used to evaluate the antitumor effect of compounds <b>AZD9496</b> and <b>D24</b> in female BALB/C nude mice model. A 1 × 10<sup>7</sup> amount of MCF-7 cells in 50% Matrigel was subcutaneously injected in the right hind leg of the mouse. When tumors reached ∼170 mm<sup>3</sup>, the mice were randomly divided to 6 mice per group and assigned to compound or vehicle treatment. Six cohorts of the mice were orally treated (QD) with <b>AZD9496</b> at the doses of 0.6 and 6 mg/kg, with <b>D24</b> at the doses of 0.6, 2, and 0.6 mg/kg, and with vehicle, respectively. Mice were examined thrice a week for the development of tumors by electronic calipers, and tumor volume was calculated as <i>V</i> = 0.5 × length × width<sup>2</sup>.</div></div><div id="sec4_5_54" class="NLM_sec NLM_sec_level_3"><div id="ac_i105" class="anchor-spacer"></div><h4 class="article-section__title" id="_i105"> In Vivo Pharmacokinetic Assessment in the MCF-7 Breast Cancer Xenograft Model</h4><div class="NLM_p last">This study was conducted in BALB/C female subjects to investigate the pharmacokinetic profiles of test compounds. The mice supplied by Changzhou Cavensle Laboratory Animal Co. Ltd. were randomized and divided into two groups consisting of 3 mice/group. The body weights were in the range from 17 to 24 g. Prepared test compounds in solutol–physiological saline (2:98) to make 0.1 mg/mL solutions for oral gavage and for intravenous injection. Sample collection was performed as follows: (1) single oral administration (p.o.) group, 2 mg/kg and 0.2 mL/10g; (2) single tail vein injection (i.v.) group, 1 mg/kg and 0.1 mL/10g. Blood was collected from orbital venous plexus in a heparinized EP tube at 5, 15, and 30 min, 1, 2, 6, 10, and 24 h after intravenous or oral administration, and the contents of the blood were analyzed by LC-MS/MS (API 4500).</div></div><div id="sec4_5_55" class="NLM_sec NLM_sec_level_3"><div id="ac_i106" class="anchor-spacer"></div><h4 class="article-section__title" id="_i106"> Molecular Docking</h4><div class="NLM_p last">The binding mode of the ligand and protein was simulated by the Glide-Dock module of the Schrodinger Maestro 2019. The crystal structure of ERα (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5ACC">5ACC</a>) was obtained from the PDB; the protein was prepared by the Protein Preparation module in Schrodinger Maestro 2019. Waters were eliminated from the protein, and polar hydrogen was added. Meanwhile, the protonation states of all residues had been considered to involve His being protonated to HID, and the Asp, Arg, Glu, and Lys residues were preprocessed to deprotonated states and so on. Receptor grids were generated using Receptor Grid Generation; the generated binding site was just the binding pocket of ERα, including several key amino acid residues. The three-dimensional structure of ligand was constructed using ChemBio 3D Ultra software; then, it was energetically minimized by using MMFF94 with 5000 iterations and a minimum RMS gradient of 0.05. Furthermore, the ligands were also prepared by the Ligand Prepare module of Schrodinger Maestro 2019. Then, the ligands were docked into a rigid protein using Glide SP; the docking score was set to glide score, and the number of returned poses per ligand in each docking stage was set to 10. The other docking parameters were left at the default settings.</div></div><div id="sec4_5_56" class="NLM_sec NLM_sec_level_3"><div id="ac_i107" class="anchor-spacer"></div><h4 class="article-section__title" id="_i107"> Molecular Dynamic Simulation</h4><div class="NLM_p last">The molecular dynamics simulation (100 ns) was performed using Desmond in Schrodinger Maestro 2019. The protein and ligand were assigned the OPLS3 force field; the protonation states of charged residues were also preprocessed: The His residues were protonated to HID, and the Asp, Arg, Glu, and Lys residues were preprocessed to deprotonated states. Simulation procedure and parameter settings of the MD simulations were similar to the previous works.<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> The model was solvated in a truncated octahedron box of the transferable interaction potential (TIP3P) water molecules with a margin distance of 10 Å. Sodium ions were added to neutralize the systems. The particle mesh Ewald (PME) method was used for long-range electrostatic interactions with a cutoff distance of 10 Å. Next, the entire setup was subjected to a stepwise equilibration procedure: The first minimization with 2500 steps of steepest descent and 2500 steps of conjugate gradient was performed with the protein, ligand fixed by 10.0 kcal/(mol Å<sup>2</sup>). Then, 5000 steps of steepest descent and 15000 steps of the conjugate gradient were employed in the whole system with the constraints switched off. After energy minimization, the whole system was gradually heated from 0 to 300 K for 60 ps under NVT, followed by a 600 ps NPT simulation at 1 atm, with harmonic restraints of 2 kcal/(mol Å<sup>2</sup>) on the complex. Subsequently, 100 ns MD simulations were performed in NPT ensemble for each system. The temperature and pressure were kept constant using a Langevin thermostat and a Langevin barostat, respectively. The time step is set at 2 fs, and the hydrogen atoms were constrained by the SHAKE algorithm. The resulting root-mean-square derivation (RMSD), root-mean-square fluctuation (RMSF), the hydrogen bonding, and the hydrophobic interaction were analyzed throughout the trajectory using Simulation Interactions Diagram module in Schrodinger Maestro 2019. The hydrogen-bond and hydrophobic contributions of each residue also could be found; thus, the hot-spot residues interacting with inhibitors could be identified.</div></div><div id="sec4_5_57" class="NLM_sec NLM_sec_level_3"><div id="ac_i108" class="anchor-spacer"></div><h4 class="article-section__title" id="_i108"> Binding Free Energy Calculation</h4><div class="NLM_p">Subsequently, the binding free energies based on the protein–ligand complex generated by the MD procedure were calculated by using molecular mechanics–generalized Born surface area module in Schrodinger Maestro 2019. The binding free energy was calculated as follows: Δ<i>G</i><sub>bind</sub> = <i>G</i><sub>complex</sub> – (<i>G</i><sub>protein</sub> + <i>G</i><sub>ligand</sub>) = Δ<i>H</i> – <i>T</i>Δ<i>S</i> = Δ<i>E</i><sub>mm</sub> + Δ<i>E</i><sub>sol</sub> – <i>T</i>Δ<i>S</i> = (Δ<i>E</i><sub>VDW</sub> + Δ<i>E</i><sub>ele</sub> + Δ<i>E</i><sub>internal</sub>) + (Δ<i>E</i><sub>GA</sub> + Δ<i>E</i><sub>GB</sub>) – <i>T</i>Δ<i>S</i>.</div><div class="NLM_p last">The Δ<i>G</i><sub>bind</sub> includes the enthalpic contribution (Δ<i>H</i>) and entropic term (−<i>T</i>Δ<i>S</i>). Δ<i>H</i> is the sum of intermolecular energy (Δ<i>E</i><sub>mm</sub>) and solvation free energy (Δ<i>E</i><sub>sol</sub>). Δ<i>E</i><sub>mm</sub> represents the complex gas-phase interaction energy composed of the electrostatic interaction energy (Δ<i>E</i><sub>ele</sub>), van der Waals energy (Δ<i>E</i><sub>vdw</sub>), and internal energy (Δ<i>E</i><sub>internal</sub>). The solvation-free energy (Δ<i>E</i><sub>sol</sub>) consists of the electrostatic solvation energy (Δ<i>E</i><sub>GB</sub>) and the nonpolar contributions (Δ<i>E</i><sub>GA</sub>). The former was calculated by the generalized Born (GB) continuum model, and the latter was estimated by the solvent accessible surface area (SASA) approach. An ensemble of 1000 snapshots extracted from the last 10 ns of MD trajectory was used to calculate the binding free energy. The entropy was determined by normal mode calculations with the MM-GBSA module by selecting 100 of the 1000 snapshots used in the free energy calculations at regular intervals. The convergence criterion for the energy gradient to stop minimization was 0.5. In total, 1000 frames were used for each MM-GBSA calculation, while 100 frames were used for each normal mode analysis. All other parameters were defined as default values.</div></div><div id="sec4_5_58" class="NLM_sec NLM_sec_level_3"><div id="ac_i109" class="anchor-spacer"></div><h4 class="article-section__title" id="_i109"> Alanine Mutation of Hot-Spot Residue</h4><div class="NLM_p last">In computational alanine scanning approach to protein–ligand binding, one mutates a specific residue to alanine and computes the difference in binding free energies before and after the mutation. The hot-spot residues were mutated to alanine by Residue Mutation module in Schrodinger Maestro 2019; each mutated protein was saved as a separate protein receptor and complex. The complex was prepared by the Protein Preparation module once again, and the MD simulation and binding free energy calculations were carried out as mentioned above. The change in free energy between the original and mutated complexes (ΔΔ<i>G</i>) is calculated as ΔΔ<i>G</i> = Δ<i>G</i><sub>mut</sub> – Δ<i>G</i><sub>wt</sub>, where Δ<i>G</i><sub>mut</sub> is the complex with a single alanine mutation and Δ<i>G</i><sub>wt</sub> is the wild-type complex. A negative ΔΔ<i>G</i> suggests that the mutation stabilizes the complex, while a positive value suggests that the mutation destabilizes it.<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> A residue was considered as a hot-spot residue if ΔΔ<i>G</i> ≥ 1.0 kcal/mol.</div></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i110"><a href="/doi/suppl/10.1021/acs.jmedchem.1c00280" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_25043" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_25043" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.1c00280?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00280</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Table S1 listing the UPLC-UV/Q-TOF MS analysis results of compound <b>D24</b> metabolites, Table S2 listing compound <b>D24</b> toxicity data, Figure S1 showing <b>D24</b> putative metabolic pathway in liver cells; spectra for the target compounds (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00280/suppl_file/jm1c00280_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00280/suppl_file/jm1c00280_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00280/suppl_file/jm1c00280_si_001.pdf">jm1c00280_si_001.pdf (3.3 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00280/suppl_file/jm1c00280_si_002.csv">jm1c00280_si_002.csv (5.53 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.1c00280" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_70341" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_70341" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Liwen Zhao</span> - <span class="hlFld-Affiliation affiliation">R&D
Center, Nanjing Sanhome Pharmaceutical Company
Ltd., Nanjing 211135, China</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#80dae8e1efecf7c0f3e1eee8efede5aee3efed"><span class="__cf_email__" data-cfemail="114b79707e7d665162707f797e7c743f727e7c">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Wei Li</span> - <span class="hlFld-Affiliation affiliation">School
of Pharmacy, Nanjing University of Chinese
Medicine, Nanjing 210023, China</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#523e3b25333b3b123c3827313f7c3736277c313c"><span class="__cf_email__" data-cfemail="f79b9e80969e9eb7999d82949ad9929382d99499">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jian Liu</span> - <span class="hlFld-Affiliation affiliation">School
of Pharmacy, Nanjing University of Chinese
Medicine, Nanjing 210023, China</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0001-9482-9036" title="Orcid link">https://orcid.org/0000-0001-9482-9036</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#6a06031f00030b045c58592a04001f0907440f0e1f440904"><span class="__cf_email__" data-cfemail="e18d88948b88808fd7d3d2a18f8b94828ccf848594cf828f">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xiaomeng Zhang</span> - <span class="hlFld-Affiliation affiliation">School
of Pharmacy, Nanjing University of Chinese
Medicine, Nanjing 210023, China</span>; 
    <span class="hlFld-Affiliation affiliation">R&D
Center, Nanjing Sanhome Pharmaceutical Company
Ltd., Nanjing 211135, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yazhou Wang</span> - <span class="hlFld-Affiliation affiliation">R&D
Center, Nanjing Sanhome Pharmaceutical Company
Ltd., Nanjing 211135, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xue Li</span> - <span class="hlFld-Affiliation affiliation">R&D
Center, Nanjing Sanhome Pharmaceutical Company
Ltd., Nanjing 211135, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jie Wu</span> - <span class="hlFld-Affiliation affiliation">R&D
Center, Nanjing Sanhome Pharmaceutical Company
Ltd., Nanjing 211135, China</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>X.Z. and Y.W. contributed equally to this work.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d7e8507-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i112">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_14074" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_14074" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We gratefully acknowledge the National Science Foundation of China (Grant Nos. 81573304 and 81703342), Natural Science Foundation of Nanjing University of Chinese Medicine (Grant No. NZY81703342), Natural Science Foundation of Jiangsu (Grant No. BK20161050), and Postgraduate Research & Practice Innovation Program of Jiangsu Province (Grant No. KYCX19_1261).</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">ADMET</td><td class="NLM_def"><p class="first last">absorption, distribution, metabolism, excretion, and toxicity</p></td></tr><tr><td class="NLM_term">AF</td><td class="NLM_def"><p class="first last">activating function</p></td></tr><tr><td class="NLM_term">AIs</td><td class="NLM_def"><p class="first last">aromatase inhibitors</p></td></tr><tr><td class="NLM_term">AUC</td><td class="NLM_def"><p class="first last">area under the curve</p></td></tr><tr><td class="NLM_term">BC</td><td class="NLM_def"><p class="first last">breast cancer</p></td></tr><tr><td class="NLM_term">Cl</td><td class="NLM_def"><p class="first last">clearance</p></td></tr><tr><td class="NLM_term"><i>C</i><sub>max</sub></td><td class="NLM_def"><p class="first last">maximum concentration</p></td></tr><tr><td class="NLM_term">CYP</td><td class="NLM_def"><p class="first last">cytochrome P450</p></td></tr><tr><td class="NLM_term">ER</td><td class="NLM_def"><p class="first last">estrogen receptor</p></td></tr><tr><td class="NLM_term">E<sub>2</sub></td><td class="NLM_def"><p class="first last">estradiol</p></td></tr><tr><td class="NLM_term"><i>F</i> (%)</td><td class="NLM_def"><p class="first last">percent oral bioavailability</p></td></tr><tr><td class="NLM_term">GAPDH</td><td class="NLM_def"><p class="first last">glyceraldehyde 3-phosphate dehydrogenase</p></td></tr><tr><td class="NLM_term">HER2</td><td class="NLM_def"><p class="first last">human epidermal growth factor 2</p></td></tr><tr><td class="NLM_term">H12</td><td class="NLM_def"><p class="first last">helix 12</p></td></tr><tr><td class="NLM_term">LBD</td><td class="NLM_def"><p class="first last">ligand binding region</p></td></tr><tr><td class="NLM_term">i.v.</td><td class="NLM_def"><p class="first last">intravenous administration</p></td></tr><tr><td class="NLM_term">MM-GBSA</td><td class="NLM_def"><p class="first last">molecular mechanics–generalized Born surface area</p></td></tr><tr><td class="NLM_term">MTD</td><td class="NLM_def"><p class="first last">maximum tolerated dose</p></td></tr><tr><td class="NLM_term">p.o.</td><td class="NLM_def"><p class="first last">oral administration</p></td></tr><tr><td class="NLM_term">PR</td><td class="NLM_def"><p class="first last">progesterone receptor</p></td></tr><tr><td class="NLM_term">RT</td><td class="NLM_def"><p class="first last">room temperature</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structure–activity relationship</p></td></tr><tr><td class="NLM_term">SERDs</td><td class="NLM_def"><p class="first last">selective estrogen receptor degraders</p></td></tr><tr><td class="NLM_term">SERMs</td><td class="NLM_def"><p class="first last">selective estrogen receptor modulators</p></td></tr><tr><td class="NLM_term"><i>t</i><sub>1/2</sub></td><td class="NLM_def"><p class="first last">half-life</p></td></tr><tr><td class="NLM_term">TGI</td><td class="NLM_def"><p class="first last">tumor growth inhibition</p></td></tr><tr><td class="NLM_term"><i>V</i><sub>d</sub></td><td class="NLM_def"><p class="first last">volume of distribution;</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i114">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_24356" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_24356" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 52 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hanahan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinberg, R. A.</span></span> <span> </span><span class="NLM_article-title">Hallmarks of cancer: the next generation</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>144</i></span>,  <span class="NLM_fpage">646</span>– <span class="NLM_lpage">674</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2011.02.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=10.1016%2Fj.cell.2011.02.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=21376230" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A528%3ADC%252BC3MXjsFeqtrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=144&publication_year=2011&pages=646-674&author=D.+Hanahanauthor=R.+A.+Weinberg&title=Hallmarks+of+cancer%3A+the+next+generation&doi=10.1016%2Fj.cell.2011.02.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Hallmarks of cancer: the next generation</span></div><div class="casAuthors">Hanahan, Douglas; Weinberg, Robert A.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">144</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">646-674</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  The hallmarks of cancer comprise six biol. capabilities acquired during the multistep development of human tumors.  The hallmarks constitute an organizing principle for rationalizing the complexities of neoplastic disease.  They include sustaining proliferative signaling, evading growth suppressors, resisting cell death, enabling replicative immortality, inducing angiogenesis, and activating invasion and metastasis.  Underlying these hallmarks are genome instability, which generates the genetic diversity that expedites their acquisition, and inflammation, which fosters multiple hallmark functions.  Conceptual progress in the last decade has added two emerging hallmarks of potential generality to this list-reprogramming of energy metab. and evading immune destruction.  In addn. to cancer cells, tumors exhibit another dimension of complexity: they contain a repertoire of recruited, ostensibly normal cells that contribute to the acquisition of hallmark traits by creating the "tumor microenvironment.".  Recognition of the widespread applicability of these concepts will increasingly affect the development of new means to treat human cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTcpqXALcb6LVg90H21EOLACvtfcHk0li9iZpRK3N5Gw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXjsFeqtrk%253D&md5=b36f160c417a712e83cb83c577f0018e</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2011.02.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2011.02.013%26sid%3Dliteratum%253Aachs%26aulast%3DHanahan%26aufirst%3DD.%26aulast%3DWeinberg%26aufirst%3DR.%2BA.%26atitle%3DHallmarks%2520of%2520cancer%253A%2520the%2520next%2520generation%26jtitle%3DCell%26date%3D2011%26volume%3D144%26spage%3D646%26epage%3D674%26doi%3D10.1016%2Fj.cell.2011.02.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, W.-Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, R.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baade, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, H.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bray, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jemal, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, X.-Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, J.</span></span> <span> </span><span class="NLM_article-title">Cancer statistics in China, 2015</span>. <i>Ca-Cancer J. Clin.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">115</span>– <span class="NLM_lpage">132</span>, <span class="refDoi"> DOI: 10.3322/caac.21338</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=10.3322%2Fcaac.21338" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=26808342" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A280%3ADC%252BC28njvVykug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2016&pages=115-132&author=W.-Q.+Chenauthor=R.-S.+Zhengauthor=P.+D.+Baadeauthor=S.-W.+Zhangauthor=H.-M.+Zengauthor=F.+Brayauthor=A.+Jemalauthor=X.-Q.+Yuauthor=J.+He&title=Cancer+statistics+in+China%2C+2015&doi=10.3322%2Fcaac.21338"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Cancer statistics in China, 2015</span></div><div class="casAuthors">Chen Wanqing; Zheng Rongshou; Baade Peter D; Zhang Siwei; Zeng Hongmei; Bray Freddie; Jemal Ahmedin; Yu Xue Qin; Yu Xue Qin; He Jie</div><div class="citationInfo"><span class="NLM_cas:title">CA: a cancer journal for clinicians</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">115-32</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">With increasing incidence and mortality, cancer is the leading cause of death in China and is a major public health problem.  Because of China's massive population (1.37 billion), previous national incidence and mortality estimates have been limited to small samples of the population using data from the 1990s or based on a specific year.  With high-quality data from an additional number of population-based registries now available through the National Central Cancer Registry of China, the authors analyzed data from 72 local, population-based cancer registries (2009-2011), representing 6.5% of the population, to estimate the number of new cases and cancer deaths for 2015.  Data from 22 registries were used for trend analyses (2000-2011).  The results indicated that an estimated 4292,000 new cancer cases and 2814,000 cancer deaths would occur in China in 2015, with lung cancer being the most common incident cancer and the leading cause of cancer death.  Stomach, esophageal, and liver cancers were also commonly diagnosed and were identified as leading causes of cancer death.  Residents of rural areas had significantly higher age-standardized (Segi population) incidence and mortality rates for all cancers combined than urban residents (213.6 per 100,000 vs 191.5 per 100,000 for incidence; 149.0 per 100,000 vs 109.5 per 100,000 for mortality, respectively).  For all cancers combined, the incidence rates were stable during 2000 through 2011 for males (+0.2% per year; P = .1), whereas they increased significantly (+2.2% per year; P < .05) among females.  In contrast, the mortality rates since 2006 have decreased significantly for both males (-1.4% per year; P < .05) and females (-1.1% per year; P < .05).  Many of the estimated cancer cases and deaths can be prevented through reducing the prevalence of risk factors, while increasing the effectiveness of clinical care delivery, particularly for those living in rural areas and in disadvantaged populations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSkZeZPQ7Kk7ntlzo9xastyfW6udTcc2eZFSSEy0suFDbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC28njvVykug%253D%253D&md5=2affa5f8beab6f885ccd830cbb249707</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.3322%2Fcaac.21338&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3322%252Fcaac.21338%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DW.-Q.%26aulast%3DZheng%26aufirst%3DR.-S.%26aulast%3DBaade%26aufirst%3DP.%2BD.%26aulast%3DZhang%26aufirst%3DS.-W.%26aulast%3DZeng%26aufirst%3DH.-M.%26aulast%3DBray%26aufirst%3DF.%26aulast%3DJemal%26aufirst%3DA.%26aulast%3DYu%26aufirst%3DX.-Q.%26aulast%3DHe%26aufirst%3DJ.%26atitle%3DCancer%2520statistics%2520in%2520China%252C%25202015%26jtitle%3DCa-Cancer%2520J.%2520Clin.%26date%3D2016%26volume%3D66%26spage%3D115%26epage%3D132%26doi%3D10.3322%2Fcaac.21338" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nardone, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Angelis, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trivedi, M. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osborne, C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schiff, R.</span></span> <span> </span><span class="NLM_article-title">The changing role of ER in endocrine resistance</span>. <i>Breast.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">S60</span>– <span class="NLM_lpage">S66</span>, <span class="refDoi"> DOI: 10.1016/j.breast.2015.07.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=10.1016%2Fj.breast.2015.07.015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=26271713" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2015&pages=S60-S66&author=A.+Nardoneauthor=C.+De+Angelisauthor=M.+V.+Trivediauthor=C.+K.+Osborneauthor=R.+Schiff&title=The+changing+role+of+ER+in+endocrine+resistance&doi=10.1016%2Fj.breast.2015.07.015"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.breast.2015.07.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.breast.2015.07.015%26sid%3Dliteratum%253Aachs%26aulast%3DNardone%26aufirst%3DA.%26aulast%3DDe%2BAngelis%26aufirst%3DC.%26aulast%3DTrivedi%26aufirst%3DM.%2BV.%26aulast%3DOsborne%26aufirst%3DC.%2BK.%26aulast%3DSchiff%26aufirst%3DR.%26atitle%3DThe%2520changing%2520role%2520of%2520ER%2520in%2520endocrine%2520resistance%26jtitle%3DBreast.%26date%3D2015%26volume%3D24%26spage%3DS60%26epage%3DS66%26doi%3D10.1016%2Fj.breast.2015.07.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tong, C. W. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, W. C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">To, K. K. W.</span></span> <span> </span><span class="NLM_article-title">Recent advances in the treatment of breast cancer</span>. <i>Front. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">227</span>, <span class="refDoi"> DOI: 10.3389/fonc.2018.00227</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=10.3389%2Ffonc.2018.00227" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=29963498" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A280%3ADC%252BB3c%252FjvVylsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2018&pages=227&author=C.+W.+S.+Tongauthor=M.+Wuauthor=W.+C.+S.+Choauthor=K.+K.+W.+To&title=Recent+advances+in+the+treatment+of+breast+cancer&doi=10.3389%2Ffonc.2018.00227"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Recent Advances in the Treatment of Breast Cancer</span></div><div class="casAuthors">Tong Christy W S; Wu Mingxia; To Kenneth K W; Cho William C S</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in oncology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">227</span>
        ISSN:<span class="NLM_cas:issn">2234-943X</span>.
    </div><div class="casAbstract">Breast cancer (BC) is the most common malignancy in women.  It is classified into a few major molecular subtypes according to hormone and growth factor receptor expression.  Over the past few years, substantial advances have been made in the discovery of new drugs for treating BC.  Improved understanding of the biologic heterogeneity of BC has allowed the development of more effective and individualized approach to treatment.  In this review, we provide an update about the current treatment strategy and discuss the various emerging novel therapies for the major molecular subtypes of BC.  A brief account of the clinical development of inhibitors of poly(ADP-ribose) polymerase, cyclin-dependent kinases 4 and 6, phosphatidylinositol 3-kinase/protein kinase B/mammalian target of rapamycin pathway, histone deacetylation, multi-targeting tyrosine kinases, and immune checkpoints for personalized treatment of BC is included.  However, no targeted drug has been approved for the most aggressive subtype-triple negative breast cancer (TNBC).  Thus, we discuss the heterogeneity of TNBC and how molecular subtyping of TNBC may help drug discovery for this deadly disease.  The emergence of drug resistance also poses threat to the successful development of targeted therapy in various molecular subtypes of BC.  New clinical trials should incorporate advanced methods to identify changes induced by drug treatment, which may be associated with the upregulation of compensatory signaling pathways in drug resistant cancer cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcS9aYrYWlne0BJ5E29YNGvQfW6udTcc2eZmjUqGltV2Hbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3c%252FjvVylsw%253D%253D&md5=a116dcf63d5640023aac066b920bdc5f</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.3389%2Ffonc.2018.00227&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffonc.2018.00227%26sid%3Dliteratum%253Aachs%26aulast%3DTong%26aufirst%3DC.%2BW.%2BS.%26aulast%3DWu%26aufirst%3DM.%26aulast%3DCho%26aufirst%3DW.%2BC.%2BS.%26aulast%3DTo%26aufirst%3DK.%2BK.%2BW.%26atitle%3DRecent%2520advances%2520in%2520the%2520treatment%2520of%2520breast%2520cancer%26jtitle%3DFront.%2520Oncol.%26date%3D2018%26volume%3D8%26spage%3D227%26doi%3D10.3389%2Ffonc.2018.00227" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ariazi, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ariazi, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cordera, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jordan, V. C.</span></span> <span> </span><span class="NLM_article-title">Estrogen receptors as therapeutic targets in breast cancer</span>. <i>Curr. Top. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">181</span>– <span class="NLM_lpage">202</span>, <span class="refDoi"> DOI: 10.2174/156802606776173483</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=10.2174%2F156802606776173483" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=16515478" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A528%3ADC%252BD28XkvFWnsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2006&pages=181-202&author=E.+A.+Ariaziauthor=J.+L.+Ariaziauthor=F.+Corderaauthor=V.+C.+Jordan&title=Estrogen+receptors+as+therapeutic+targets+in+breast+cancer&doi=10.2174%2F156802606776173483"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Estrogen receptors as therapeutic targets in breast cancer</span></div><div class="casAuthors">Ariazi, Eric A.; Ariazi, Jennifer L.; Cordera, Fernando; Jordan, V. Craig</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">181-202</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  The estrogen receptor α (ERα) has proven to be the single most important target in breast cancer over the last 30 years.  The use of the selective ER modulator (SERM) tamoxifen for the treatment and prevention of breast cancer has changed therapeutics.  The SERM Raloxifene, approved for the treatment of osteoporosis, lacks tamoxifen's increased risk for endometrial cancer and is being evaluated for the prevention of breast cancer.  Other SERMs approved or under development for use against breast cancer or osteoporosis include Toremifene, GW5638, GW7604 (the active metabolite of GW5638), Idoxifene, Lasofoxifene, Arzoxifene, Bazedoxifene, EM-800, and Acolbifene (the active metabolite of EM-800).  Aromatase inhibitors (AIs) have recently proven to be more efficacious than tamoxifen as first-line therapy, efficacious for second-line therapy (e.g. against tamoxifen-resistant disease), and useful for extended adjuvant therapy after tamoxifen.  The AIs include the nonsteroidal agents Letrozole and Anastrole, and the steroidal agent Exemestane.  The pure antiestrogen Fulvestrant has proven to be just as effective as AIs.  Other pure antiestrogens, ZK-703, ZK-253, RU 58668, and TAS-108, show great promise.  The development of resistance to endocrine therapy remains a clin. important problem, and lab. models based on human breast cancer cells grown as tumors in immune-compromised mice have led to important insights into this problem.  Progesterone receptor-neg. status of ER-pos. breast cancers may reflect altered growth factor receptor signaling, and helps to explain why this subclass of tumors exhibits lower response rates to tamoxifen compared to cancers typed progesterone receptor-pos.  Crosstalk among plasma membrane-localized ER, growth factor receptor signaling, and nuclear-localized ER provide further insights into antihormonal-resistant breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGph8MK1y7HLJrVg90H21EOLACvtfcHk0lgO5vMcazaLNg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XkvFWnsrY%253D&md5=d5e3920a355d937d5bf69b6d71ef1a9b</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.2174%2F156802606776173483&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F156802606776173483%26sid%3Dliteratum%253Aachs%26aulast%3DAriazi%26aufirst%3DE.%2BA.%26aulast%3DAriazi%26aufirst%3DJ.%2BL.%26aulast%3DCordera%26aufirst%3DF.%26aulast%3DJordan%26aufirst%3DV.%2BC.%26atitle%3DEstrogen%2520receptors%2520as%2520therapeutic%2520targets%2520in%2520breast%2520cancer%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2006%26volume%3D6%26spage%3D181%26epage%3D202%26doi%3D10.2174%2F156802606776173483" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, H.-R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, T.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, K.-C.</span></span> <span> </span><span class="NLM_article-title">Functions and physiological roles of two types of estrogen receptors, ERα and ERβ, identified by estrogen receptor knockout mouse</span>. <i>Lab. Anim. Res.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">71</span>– <span class="NLM_lpage">76</span>, <span class="refDoi"> DOI: 10.5625/lar.2012.28.2.71</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=10.5625%2Flar.2012.28.2.71" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=22787479" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A280%3ADC%252BC38jptlOisQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2012&pages=71-76&author=H.-R.+Leeauthor=T.-H.+Kimauthor=K.-C.+Choi&title=Functions+and+physiological+roles+of+two+types+of+estrogen+receptors%2C+ER%CE%B1+and+ER%CE%B2%2C+identified+by+estrogen+receptor+knockout+mouse&doi=10.5625%2Flar.2012.28.2.71"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Functions and physiological roles of two types of estrogen receptors, ERα and ERβ, identified by estrogen receptor knockout mouse</span></div><div class="casAuthors">Lee Hye-Rim; Kim Tae-Hee; Choi Kyung-Chul</div><div class="citationInfo"><span class="NLM_cas:title">Laboratory animal research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">71-6</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Estrogens, a class of steroid hormones, regulate the growth, development, and physiology of the human reproductive system.  Estrogens also involve in the neuroendocrine, skeletal, adipogenesis, and cardiovascular systems.  Estrogen signaling pathways are selectively stimulated or inhibited depending on a balance between the activities of estrogen receptor (ER) α or ERβ in target organs.  ERs belong to the steroid hormone superfamily of nuclear receptors, which act as transcription factors after binding to estrogen.  The gene expression regulation by ERs is to modulate biological activities, such as reproductive organ development, bone modeling, cardiovascular system functioning, metabolism, and behavior in both females and males.  Understanding of the general physiological roles of ERs has been gained when estrogen levels were ablated by ovariectomy and then replenished by treatment with exogenous estrogen.  This technique is not sufficient to fully determine the exact function of estrogen signaling in general processes in living tissues.  However, a transgenic mouse model has been useful to study gene-specific functions.  ERα and ERβ have different biological functions, and knockout and transgenic animal models have distinct phenotypes.  Analysis of ERα and ERβ function using knockout mouse models has identified the roles of estrogen signaling in general physiologic processes.  Although transgenic mouse models do not always produce consistent results, they are the useful for studying the functions of these genes under specific pathological conditions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSOsUARkrX9ASEJCPKhJPuqfW6udTcc2eZmjUqGltV2Hbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC38jptlOisQ%253D%253D&md5=efbea11cb9e6901b1ac35af2fc1b878a</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.5625%2Flar.2012.28.2.71&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.5625%252Flar.2012.28.2.71%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DH.-R.%26aulast%3DKim%26aufirst%3DT.-H.%26aulast%3DChoi%26aufirst%3DK.-C.%26atitle%3DFunctions%2520and%2520physiological%2520roles%2520of%2520two%2520types%2520of%2520estrogen%2520receptors%252C%2520ER%25CE%25B1%2520and%2520ER%25CE%25B2%252C%2520identified%2520by%2520estrogen%2520receptor%2520knockout%2520mouse%26jtitle%3DLab.%2520Anim.%2520Res.%26date%3D2012%26volume%3D28%26spage%3D71%26epage%3D76%26doi%3D10.5625%2Flar.2012.28.2.71" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spezia, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lei, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vuppalapati, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luu, H. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haydon, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, T.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, G.</span></span> <span> </span><span class="NLM_article-title">Breast cancer development and progression: risk factors, cancer stem cells, signaling pathways, genomics, and molecular pathogenesis</span>. <i>Genes. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">77</span>– <span class="NLM_lpage">106</span>, <span class="refDoi"> DOI: 10.1016/j.gendis.2018.05.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=10.1016%2Fj.gendis.2018.05.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=30258937" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A528%3ADC%252BC1cXpvVKrtLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2018&pages=77-106&author=Y.+Fengauthor=M.+Speziaauthor=S.+Huangauthor=C.+Yuanauthor=Z.+Zengauthor=L.+Zhangauthor=X.+Jiauthor=W.+Liuauthor=B.+Huangauthor=W.+Luoauthor=B.+Liuauthor=Y.+Leiauthor=S.+Duauthor=A.+Vuppalapatiauthor=H.+H.+Luuauthor=R.+C.+Haydonauthor=T.-C.+Heauthor=G.+Ren&title=Breast+cancer+development+and+progression%3A+risk+factors%2C+cancer+stem+cells%2C+signaling+pathways%2C+genomics%2C+and+molecular+pathogenesis&doi=10.1016%2Fj.gendis.2018.05.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Breast cancer development and progression: Risk factors, cancer stem cells, signaling pathways, genomics, and molecular pathogenesis</span></div><div class="casAuthors">Feng, Yixiao; Spezia, Mia; Huang, Shifeng; Yuan, Chengfu; Zeng, Zongyue; Zhang, Linghuan; Ji, Xiaojuan; Liu, Wei; Huang, Bo; Luo, Wenping; Liu, Bo; Lei, Yan; Du, Scott; Vuppalapati, Akhila; Luu, Hue H.; Haydon, Rex C.; He, Tong-Chuan; Ren, Guosheng</div><div class="citationInfo"><span class="NLM_cas:title">Genes and Diseases</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">77-106</span>CODEN:
                <span class="NLM_cas:coden">GDEIAV</span>;
        ISSN:<span class="NLM_cas:issn">2352-3042</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">As the most commonly occurring cancer in women worldwide, breast cancer poses a formidable public health challenge on a global scale.  Breast cancer consists of a group of biol. and molecularly heterogeneous diseases originated from the breast.  While the risk factors assocd. with this cancer varies with respect to other cancers, genetic predisposition, most notably mutations in BRCA1 or BRCA2 gene, is an important causative factor for this malignancy.  Breast cancers can begin in different areas of the breast, such as the ducts, the lobules, or the tissue in between.  Within the large group of diverse breast carcinomas, there are various denoted types of breast cancer based on their invasiveness relative to the primary tumor sites.  It is important to distinguish between the various subtypes because they have different prognoses and treatment implications.  As there are remarkable parallels between normal development and breast cancer progression at the mol. level, it has been postulated that breast cancer may be derived from mammary cancer stem cells.  Normal breast development and mammary stem cells are regulated by several signaling pathways, such as estrogen receptors (ERs), HER2, and Wnt/β-catenin signaling pathways, which control stem cell proliferation, cell death, cell differentiation, and cell motility.  Furthermore, emerging evidence indicates that epigenetic regulations and noncoding RNAs may play important roles in breast cancer development and may contribute to the heterogeneity and metastatic aspects of breast cancer, esp. for triple-neg. breast cancer.  This review provides a comprehensive survey of the mol., cellular and genetic aspects of breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpD6nESJp4DErVg90H21EOLACvtfcHk0lhp1QEsamV8Dw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXpvVKrtLc%253D&md5=f9ae3c050fc7cf2241919a7cca7b82a8</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.gendis.2018.05.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.gendis.2018.05.001%26sid%3Dliteratum%253Aachs%26aulast%3DFeng%26aufirst%3DY.%26aulast%3DSpezia%26aufirst%3DM.%26aulast%3DHuang%26aufirst%3DS.%26aulast%3DYuan%26aufirst%3DC.%26aulast%3DZeng%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DJi%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DW.%26aulast%3DHuang%26aufirst%3DB.%26aulast%3DLuo%26aufirst%3DW.%26aulast%3DLiu%26aufirst%3DB.%26aulast%3DLei%26aufirst%3DY.%26aulast%3DDu%26aufirst%3DS.%26aulast%3DVuppalapati%26aufirst%3DA.%26aulast%3DLuu%26aufirst%3DH.%2BH.%26aulast%3DHaydon%26aufirst%3DR.%2BC.%26aulast%3DHe%26aufirst%3DT.-C.%26aulast%3DRen%26aufirst%3DG.%26atitle%3DBreast%2520cancer%2520development%2520and%2520progression%253A%2520risk%2520factors%252C%2520cancer%2520stem%2520cells%252C%2520signaling%2520pathways%252C%2520genomics%252C%2520and%2520molecular%2520pathogenesis%26jtitle%3DGenes.%2520Dis.%26date%3D2018%26volume%3D5%26spage%3D77%26epage%3D106%26doi%3D10.1016%2Fj.gendis.2018.05.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Eckhardt, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Francis, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parker, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, R. L.</span></span> <span> </span><span class="NLM_article-title">Strategies for the discovery and development of therapies for metastatic breast cancer</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">479</span>– <span class="NLM_lpage">497</span>, <span class="refDoi"> DOI: 10.1038/nrd2372</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=10.1038%2Fnrd2372" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=22653217" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A528%3ADC%252BC38XnslOht7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2012&pages=479-497&author=B.+L.+Eckhardtauthor=P.+A.+Francisauthor=B.+S.+Parkerauthor=R.+L.+Anderson&title=Strategies+for+the+discovery+and+development+of+therapies+for+metastatic+breast+cancer&doi=10.1038%2Fnrd2372"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Strategies for the discovery and development of therapies for metastatic breast cancer</span></div><div class="casAuthors">Eckhardt, Bedrich L.; Francis, Prudence A.; Parker, Belinda S.; Anderson, Robin L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">479-497</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Nearly all deaths caused by solid cancers occur as a result of metastasis - the formation of secondary tumors in distant organs such as the lungs, liver, brain and bone.  A major obstruction to the development of drugs with anti-metastatic efficacy is our fragmented understanding of how tumors 'evolve' and metastasize, at both the biol. and genetic levels.  Furthermore, although there is significant overlap in the metastatic process among different types of cancer, there are also marked differences in the propensity to metastasize, the extent of metastasis, the sites to which the tumor metastasizes, the kinetics of the process and the mechanisms involved.  Here, we consider the case of breast cancer, which has some marked distinguishing features compared with other types of cancer.  Considerable progress has been made in the development of preclin. models and in the identification of relevant signalling pathways and genetic regulators of metastatic breast cancer, and we discuss how these might facilitate the development of novel targeted anti-metastatic drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKLJhOlAUBKbVg90H21EOLACvtfcHk0lhp1QEsamV8Dw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XnslOht7c%253D&md5=f3b53f3ca5d65ccadce9bb35ef084640</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnrd2372&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2372%26sid%3Dliteratum%253Aachs%26aulast%3DEckhardt%26aufirst%3DB.%2BL.%26aulast%3DFrancis%26aufirst%3DP.%2BA.%26aulast%3DParker%26aufirst%3DB.%2BS.%26aulast%3DAnderson%26aufirst%3DR.%2BL.%26atitle%3DStrategies%2520for%2520the%2520discovery%2520and%2520development%2520of%2520therapies%2520for%2520metastatic%2520breast%2520cancer%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2012%26volume%3D11%26spage%3D479%26epage%3D497%26doi%3D10.1038%2Fnrd2372" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Patel, H. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bihani, T.</span></span> <span> </span><span class="NLM_article-title">Selective estrogen receptor modulators (SERMs) and selective estrogen receptor degraders (SERDs) in cancer treatment</span>. <i>Pharmacol. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>186</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">24</span>, <span class="refDoi"> DOI: 10.1016/j.pharmthera.2017.12.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=10.1016%2Fj.pharmthera.2017.12.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=29289555" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A528%3ADC%252BC1cXovVShtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=186&publication_year=2018&pages=1-24&author=H.+K.+Patelauthor=T.+Bihani&title=Selective+estrogen+receptor+modulators+%28SERMs%29+and+selective+estrogen+receptor+degraders+%28SERDs%29+in+cancer+treatment&doi=10.1016%2Fj.pharmthera.2017.12.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Selective estrogen receptor modulators (SERMs) and selective estrogen receptor degraders (SERDs) in cancer treatment</span></div><div class="casAuthors">Patel, Hitisha K.; Bihani, Teeru</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacology & Therapeutics</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">186</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1-24</span>CODEN:
                <span class="NLM_cas:coden">PHTHDT</span>;
        ISSN:<span class="NLM_cas:issn">0163-7258</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Breast cancer is the most frequently diagnosed cancer in women, with estrogen receptor pos. (ER+) breast cancer making up approx. 75% of all breast cancers diagnosed.  Given the dependence on active ER signaling in these tumors, the predominant treatment strategy has been to inhibit various aspects of this pathway including directly antagonizing ER with the use of selective estrogen receptor modulators (SERMs) and selective estrogen receptor degraders (SERDs).  Interestingly, the dependence on ER for breast cancer growth is often retained even after progression through several lines of antiestrogen therapy, making ER a bonafide biomarker for this cancer subtype and driving the continued research and development of novel ER-targeted therapeutics to treat this patient population.  This, combined with the continuous discovery of mechanisms underlying endocrine resistance, is resulting in a continually evolving treatment landscape for ER+ breast cancer.  This review discusses various ER antagonists investigated for the treatment of breast cancer, outlining their pharmacol. and tissue-specific mechanisms of action as well as their specified use within the ER+ breast cancer setting.  In addn., mechanisms of resistance to SERMs and SERDs, the use of ER antagonists in combination therapy strategies, and the ongoing development of novel drugs are also reviewed in the context of the changing clin. landscape of ER+ breast cancer.  Lastly, the role of SERMs and SERDs in non-breast cancer indications is also discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGre8o54q6VIOrVg90H21EOLACvtfcHk0lhp1QEsamV8Dw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXovVShtw%253D%253D&md5=9e33ceed330cba071400fd70c0b71bf4</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.pharmthera.2017.12.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pharmthera.2017.12.012%26sid%3Dliteratum%253Aachs%26aulast%3DPatel%26aufirst%3DH.%2BK.%26aulast%3DBihani%26aufirst%3DT.%26atitle%3DSelective%2520estrogen%2520receptor%2520modulators%2520%2528SERMs%2529%2520and%2520selective%2520estrogen%2520receptor%2520degraders%2520%2528SERDs%2529%2520in%2520cancer%2520treatment%26jtitle%3DPharmacol.%2520Ther.%26date%3D2018%26volume%3D186%26spage%3D1%26epage%3D24%26doi%3D10.1016%2Fj.pharmthera.2017.12.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boisen, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andersen, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sreekumar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stern, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oesterreich, S.</span></span> <span> </span><span class="NLM_article-title">Treating gynecologic malignancies with selective estrogen receptor downregulators (SERDs): promise and challenges</span>. <i>Mol. Cell. Endocrinol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>418</i></span>,  <span class="NLM_fpage">322</span>– <span class="NLM_lpage">333</span>, <span class="refDoi"> DOI: 10.1016/j.mce.2015.04.035</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=10.1016%2Fj.mce.2015.04.035" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=26276546" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVajtrzF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=418&publication_year=2015&pages=322-333&author=M.+M.+Boisenauthor=C.+L.+Andersenauthor=S.+Sreekumarauthor=A.+M.+Sternauthor=S.+Oesterreich&title=Treating+gynecologic+malignancies+with+selective+estrogen+receptor+downregulators+%28SERDs%29%3A+promise+and+challenges&doi=10.1016%2Fj.mce.2015.04.035"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Treating gynecologic malignancies with selective estrogen receptor downregulators (SERDs): promise and challenges</span></div><div class="casAuthors">Boisen, Michelle M.; Andersen, Courtney L.; Sreekumar, Sreeja; Stern, Andrew M.; Oesterreich, Steffi</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Endocrinology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">418</span>
        (<span class="NLM_cas:issue">Part_3</span>),
    <span class="NLM_cas:pages">322-333</span>CODEN:
                <span class="NLM_cas:coden">MCEND6</span>;
        ISSN:<span class="NLM_cas:issn">0303-7207</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ireland Ltd.</span>)
        </div><div class="casAbstract">Endometrial and ovarian cancers are estrogen-dependent gynecol. malignancies.  Although many are estrogen receptor (ER) pos., treatment with the selective estrogen receptor modulator (SERM) tamoxifen, a tissue selective partial-agonist, has demonstrated only modest clin. benefit.  Selective estrogen receptor downregulators (SERDs) are pure ER antagonists showing a benefit for advanced ER pos. breast cancer, which has bolstered their potential use for ER pos. gynecol. malignancies.  We summarize these preclin. and clin. data, suggesting that a subpopulation of patients with endometrial or ovarian cancer exists in which treatment with SERDs results in improved outcome.  However, the full potential of SERDs for a gynecol. malignancies will be realized only when the appropriate predictive biomarkers are identified.  Addnl., a further understanding ER signaling in the context of ovarian and endometrial tissues that appear to involve c-Src and other kinase pathways is needed to successfully address the emergence of resistance with rationally designed combination therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgqSQ46Mh85rVg90H21EOLACvtfcHk0lgPWOVX6elmmw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVajtrzF&md5=e0c259cf751f2acc6a368a22dc8d38af</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.mce.2015.04.035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.mce.2015.04.035%26sid%3Dliteratum%253Aachs%26aulast%3DBoisen%26aufirst%3DM.%2BM.%26aulast%3DAndersen%26aufirst%3DC.%2BL.%26aulast%3DSreekumar%26aufirst%3DS.%26aulast%3DStern%26aufirst%3DA.%2BM.%26aulast%3DOesterreich%26aufirst%3DS.%26atitle%3DTreating%2520gynecologic%2520malignancies%2520with%2520selective%2520estrogen%2520receptor%2520downregulators%2520%2528SERDs%2529%253A%2520promise%2520and%2520challenges%26jtitle%3DMol.%2520Cell.%2520Endocrinol.%26date%3D2015%26volume%3D418%26spage%3D322%26epage%3D333%26doi%3D10.1016%2Fj.mce.2015.04.035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Johnston, S. R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dowsett, M.</span></span> <span> </span><span class="NLM_article-title">Aromatase inhibitors for breast cancer: lessons from the laboratory</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">821</span>– <span class="NLM_lpage">831</span>, <span class="refDoi"> DOI: 10.1038/nrc1211</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=10.1038%2Fnrc1211" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=14668813" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A528%3ADC%252BD3sXptFKmur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2003&pages=821-831&author=S.+R.+D.+Johnstonauthor=M.+Dowsett&title=Aromatase+inhibitors+for+breast+cancer%3A+lessons+from+the+laboratory&doi=10.1038%2Fnrc1211"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Aromatase inhibitors for breast cancer: Lessons from the laboratory</span></div><div class="casAuthors">Johnston, Stephen R. D.; Dowsett, Mitch</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">821-831</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Endocrine therapy with tamoxifen has been the mainstay of treatment for hormone-sensitive breast cancer for more than 20 yr.  An alternative strategy of estrogen deprivation in postmenopausal women with aromatase inhibitors is set to replace tamoxifen based on better efficacy and a delay in the emergence of endocrine resistance.  There are fundamental differences in how tamoxifen and aromatase inhibitors alter estrogen-receptor signaling.  Understanding the response and resistance to these different therapies is central to further improving therapeutic options for women with breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0yq8HDYSRf7Vg90H21EOLACvtfcHk0lgPWOVX6elmmw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXptFKmur8%253D&md5=5e187fa66d3912a9c5b151c3c54fb251</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fnrc1211&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc1211%26sid%3Dliteratum%253Aachs%26aulast%3DJohnston%26aufirst%3DS.%2BR.%2BD.%26aulast%3DDowsett%26aufirst%3DM.%26atitle%3DAromatase%2520inhibitors%2520for%2520breast%2520cancer%253A%2520lessons%2520from%2520the%2520laboratory%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2003%26volume%3D3%26spage%3D821%26epage%3D831%26doi%3D10.1038%2Fnrc1211" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McDonnell, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wardell, S. E.</span></span> <span> </span><span class="NLM_article-title">The molecular mechanisms underlying the pharmacological actions of ER modulators: implications for new drug discovery in breast cancer</span>. <i>Curr. Opin. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">620</span>– <span class="NLM_lpage">628</span>, <span class="refDoi"> DOI: 10.1016/j.coph.2010.09.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=10.1016%2Fj.coph.2010.09.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=20926342" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVals73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2010&pages=620-628&author=D.+P.+McDonnellauthor=S.+E.+Wardell&title=The+molecular+mechanisms+underlying+the+pharmacological+actions+of+ER+modulators%3A+implications+for+new+drug+discovery+in+breast+cancer&doi=10.1016%2Fj.coph.2010.09.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">The molecular mechanisms underlying the pharmacological actions of ER modulators: implications for new drug discovery in breast cancer</span></div><div class="casAuthors">McDonnell, Donald P.; Wardell, Suzanne E.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Pharmacology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">620-628</span>CODEN:
                <span class="NLM_cas:coden">COPUBK</span>;
        ISSN:<span class="NLM_cas:issn">1471-4892</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Our understanding of the mol. mechanisms underlying the pharmacol. actions of estrogen receptor (ER) ligands has evolved considerably in recent years.  Much of this knowledge has come from a detailed dissection of the mechanism(s) of action of the Selective Estrogen Receptor Modulators (SERMs) tamoxifen and raloxifene, so called for their ability to function as ER agonists or antagonists depending on the tissue in which they operate.  These mechanistic insights have had a significant impact on the discovery of second generation SERMs, some of which are in late stage clin. development for the treatment/prevention of breast cancer as well as other estrogenopathies.  In addn. to the SERMs, however, have emerged the Selective Estrogen Degraders (SERDs), which as their name suggests, interact with and facilitate ER turnover in cells.  One drug of this class, fulvestrant, has been approved as a third line treatment for ER-pos. metastatic breast cancer.  Whereas the first generation SERMs/SERDs were discovered in a serendipitous manner, this review will highlight how our understanding of the mol. pharmacol. of ER ligands has been utilized in the development of the next generation of SERMs/SERDs, some of which are likely to have a major impact on the pharmacotherapy of breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8b2l5p-n0bbVg90H21EOLACvtfcHk0lgPWOVX6elmmw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsVals73P&md5=24d1490ca0b0570c21e9768c9de134d6</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.coph.2010.09.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.coph.2010.09.007%26sid%3Dliteratum%253Aachs%26aulast%3DMcDonnell%26aufirst%3DD.%2BP.%26aulast%3DWardell%26aufirst%3DS.%2BE.%26atitle%3DThe%2520molecular%2520mechanisms%2520underlying%2520the%2520pharmacological%2520actions%2520of%2520ER%2520modulators%253A%2520implications%2520for%2520new%2520drug%2520discovery%2520in%2520breast%2520cancer%26jtitle%3DCurr.%2520Opin.%2520Pharmacol.%26date%3D2010%26volume%3D10%26spage%3D620%26epage%3D628%26doi%3D10.1016%2Fj.coph.2010.09.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cuzick, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forbes, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sestak, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cawthorn, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamed, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holli, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howell, A.</span></span> <span> </span><span class="NLM_article-title">Long-term results of tamoxifen prophylaxis for breast cancer-96-month follow-up of the randomized IBIS-I trial</span>. <i>J. Natl. Cancer Inst.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>99</i></span>,  <span class="NLM_fpage">272</span>– <span class="NLM_lpage">282</span>, <span class="refDoi"> DOI: 10.1093/jnci/djk049</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=10.1093%2Fjnci%2Fdjk049" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=17312304" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A528%3ADC%252BD2sXjsVeitrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=99&publication_year=2007&pages=272-282&author=J.+Cuzickauthor=J.+F.+Forbesauthor=I.+Sestakauthor=S.+Cawthornauthor=H.+Hamedauthor=K.+Holliauthor=A.+Howell&title=Long-term+results+of+tamoxifen+prophylaxis+for+breast+cancer-96-month+follow-up+of+the+randomized+IBIS-I+trial&doi=10.1093%2Fjnci%2Fdjk049"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Long-term results of tamoxifen prophylaxis for breast cancer-96-month follow-up of the randomized IBIS-I trial</span></div><div class="casAuthors">Cuzick, Jack; Forbes, John F.; Sestak, Ivana; Cawthorn, Simon; Hamed, Hisham; Holli, Kaija; Howell, Anthony</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the National Cancer Institute</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">99</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">272-282</span>CODEN:
                <span class="NLM_cas:coden">JNCIEQ</span>;
        ISSN:<span class="NLM_cas:issn">0027-8874</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Background: Initial results from the first International Breast Cancer Intervention Study (IBIS-I) found that tamoxifen reduced the risk of invasive estrogen receptor (ER)-pos. tumors by 31% in women at increased risk for breast cancer, but most of the follow-up at this time was during the active treatment phase.  We report an updated anal. of IBIS-I that focuses on the period after active treatment was completed, a time for which little evidence from other trials is available.  Methods: A total of 7145 women who were aged 35-70 years and at increased risk of breast cancer were randomly assigned to receive either tamoxifen (20 mg/day) or placebo for 5 years.  The primary outcome measure was the incidence of breast cancer (including ductal carcinoma in situ), but side effects were also investigated.  Relative risks were computed as the ratio of incidence rates.  All statistical tests were two-sided.  Results: After a median follow-up of 96 mo after randomization, 142 breast cancers were diagnosed in the 3579 women in the tamoxifen group and 195 in the 3575 women in the placebo group (4.97 vs. 6.82 per 1000 woman-years, resp.; risk ratio [RR] = 0.73, 95% confidence interval [CI] = 0.58 to 0.91, P = .004).  The prophylactic effect of tamoxifen was fairly const. for the entire follow-up period, and no diminution of benefit was obsd. for up to 10 years after randomization.  However, side effects in the tamoxifen group were much lower after completion of the active treatment period than during active treatment.  For example, deep-vein thrombosis and pulmonary embolism were statistically significantly higher in the tamoxifen arm than in the placebo arm during active treatment (52 vs. 23 cases, RR = 2.26, 95% CI = 1.36 to 3.87) but not after tamoxifen was stopped (16 vs. 14 cases, RR = 1.14, 95% CI = 0.52 to 2.53).  The two arms did not differ in the risk of ER-neg. invasive tumors (35 in each arm, RR = 1.00, 95% CI = 0.61 to 1.65) across the entire follow-up period, but the risk of ER-pos. invasive breast cancer was 34% lower in the tamoxifen arm (87 vs. 132 cases, RR = 0.66, 95% CI = 0.50 to 0.87).  Conclusions: The risk-reducing effect of tamoxifen appears to persist for at least 10 years, but most side effects of tamoxifen do not continue after the 5-yr treatment period.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdtmvBUpbSlbVg90H21EOLACvtfcHk0lgX6FVKQ01qCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXjsVeitrs%253D&md5=e2e23ea674e36864f76d7cd5e8e98f3d</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1093%2Fjnci%2Fdjk049&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjnci%252Fdjk049%26sid%3Dliteratum%253Aachs%26aulast%3DCuzick%26aufirst%3DJ.%26aulast%3DForbes%26aufirst%3DJ.%2BF.%26aulast%3DSestak%26aufirst%3DI.%26aulast%3DCawthorn%26aufirst%3DS.%26aulast%3DHamed%26aufirst%3DH.%26aulast%3DHolli%26aufirst%3DK.%26aulast%3DHowell%26aufirst%3DA.%26atitle%3DLong-term%2520results%2520of%2520tamoxifen%2520prophylaxis%2520for%2520breast%2520cancer-96-month%2520follow-up%2520of%2520the%2520randomized%2520IBIS-I%2520trial%26jtitle%3DJ.%2520Natl.%2520Cancer%2520Inst.%26date%3D2007%26volume%3D99%26spage%3D272%26epage%3D282%26doi%3D10.1093%2Fjnci%2Fdjk049" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cuzick, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sestak, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baum, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buzdar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howell, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dowsett, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forbes, J. F.</span></span> <span> </span><span class="NLM_article-title">Effect of anastrozole and tamoxifen as adjuvant treatment for early-stage breast cancer: 10-year analysis of the ATAC trial</span>. <i>Lancet Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">1135</span>– <span class="NLM_lpage">1141</span>, <span class="refDoi"> DOI: 10.1016/S1470-2045(10)70257-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=10.1016%2FS1470-2045%2810%2970257-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=21087898" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVyhsLnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2010&pages=1135-1141&author=J.+Cuzickauthor=I.+Sestakauthor=M.+Baumauthor=A.+Buzdarauthor=A.+Howellauthor=M.+Dowsettauthor=J.+F.+Forbes&title=Effect+of+anastrozole+and+tamoxifen+as+adjuvant+treatment+for+early-stage+breast+cancer%3A+10-year+analysis+of+the+ATAC+trial&doi=10.1016%2FS1470-2045%2810%2970257-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Effect of anastrozole and tamoxifen as adjuvant treatment for early-stage breast cancer: 10-year analysis of the ATAC trial</span></div><div class="casAuthors">Cuzick, Jack; Sestak, Ivana; Baum, Michael; Buzdar, Aman; Howell, Anthony; Dowsett, Mitch; Forbes, John F.</div><div class="citationInfo"><span class="NLM_cas:title">Lancet Oncology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1135-1141</span>CODEN:
                <span class="NLM_cas:coden">LOANBN</span>;
        ISSN:<span class="NLM_cas:issn">1470-2045</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The Arimidex, Tamoxifen, Alone or in Combination (ATAC) trial was designed to compare the efficacy and safety of anastrozole (1 mg) with tamoxifen (20 mg), both given orally every day for 5 years, as adjuvant treatment for postmenopausal women with early-stage breast cancer.  In this anal., we assess the long-term outcomes after a median follow-up of 120 mo.  We used a proportional hazards model to assess the primary endpoint of disease-free survival, and the secondary endpoints of time to recurrence, time to distant recurrence, incidence of new contralateral breast cancer, overall survival, and death with or without recurrence in all randomized patients (anastrozole n = 3125, tamoxifen n = 3116) and hormone-receptor-pos. patients (anastrozole n = 2618, tamoxifen n = 2598).  After treatment completion, we continued to collect data on fractures and serious adverse events in a masked fashion (safety population: anastrozole n = 3092, tamoxifen n = 3094).  This study is registered as an , no. .  Patients were followed up for a median of 120 mo (range 0-145); there were 24 522 woman-years of follow-up in the anastrozole group and 23 950 woman-years in the tamoxifen group.  In the full study population, there were significant improvements in the anastrozole group compared with the tamoxifen group for disease-free survival (hazard ratio [HR] 0.91, 95% CI 0.83-0.99; p = 0.04), time to recurrence (0.84, 0.75-0.93; p = 0.001), and time to distant recurrence (0.87, 0.77-0.99; p = 0.03).  For hormone-receptor-pos. patients, the results were also significantly in favor of the anastrozole group for disease-free survival (HR 0.86, 95% CI 0.78-0.95; p = 0.003), time to recurrence (0.79, 0.70-0.89; p = 0.0002), and time to distant recurrence (0.85, 0.73-0.98; p = 0.02).  In hormone-receptor-pos. patients, abs. differences in time to recurrence between anastrozole and tamoxifen increased over time (2.7% at 5 years and 4.3% at 10 years) and recurrence rates remained significantly lower on anastrozole than tamoxifen after treatment completion (HR 0.81, 95% CI 0.67-0.98; p = 0.03), although the carryover benefit was smaller after 8 years.  There was weak evidence of fewer deaths after recurrence with anastrozole compared with tamoxifen treatment in the hormone-receptor-pos. subgroup (HR 0.87, 95% CI 0.74-1.02; p = 0.09), but there was little difference in overall mortality (0.95, 95% CI 0.84-1.06; p = 0.4).  Fractures were more frequent during active treatment in patients receiving anastrozole than those receiving tamoxifen (451 vs 351; OR 1.33, 95% CI 1.15-1.55; p<0.0001), but were similar in the post-treatment follow-up period (110 vs 112; OR 0.98, 95% CI 0.74-1.30; p = 0.9).  Treatment-related serious adverse events were less common in the anastrozole group than the tamoxifen group (223 anastrozole vs 369 tamoxifen; OR 0.57, 95% CI 0.48-0.69; p<0.0001), but were similar after treatment completion (66 vs 78; OR 0.84, 95% CI 0.60-1.19; p = 0.3).  No differences in non-breast cancer causes of death were apparent and the incidence of other cancers was similar between groups (425 vs 431) and continue to be higher with anastrozole for colorectal (66 vs 44) and lung cancer (51 vs 34), and lower for endometrial cancer (six vs 24), melanoma (eight vs 19), and ovarian cancer (17 vs 28).  No new safety concerns were reported.  These data confirm the long-term superior efficacy and safety of anastrozole over tamoxifen as initial adjuvant therapy for postmenopausal women with hormone-sensitive early breast cancer.  Funding: AstraZeneca.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkXdPIpkfRNrVg90H21EOLACvtfcHk0lgX6FVKQ01qCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsVyhsLnF&md5=63f3920eb78b3c4d3dea4ef91b4bc53c</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2FS1470-2045%2810%2970257-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1470-2045%252810%252970257-6%26sid%3Dliteratum%253Aachs%26aulast%3DCuzick%26aufirst%3DJ.%26aulast%3DSestak%26aufirst%3DI.%26aulast%3DBaum%26aufirst%3DM.%26aulast%3DBuzdar%26aufirst%3DA.%26aulast%3DHowell%26aufirst%3DA.%26aulast%3DDowsett%26aufirst%3DM.%26aulast%3DForbes%26aufirst%3DJ.%2BF.%26atitle%3DEffect%2520of%2520anastrozole%2520and%2520tamoxifen%2520as%2520adjuvant%2520treatment%2520for%2520early-stage%2520breast%2520cancer%253A%252010-year%2520analysis%2520of%2520the%2520ATAC%2520trial%26jtitle%3DLancet%2520Oncol.%26date%3D2010%26volume%3D11%26spage%3D1135%26epage%3D1141%26doi%3D10.1016%2FS1470-2045%2810%2970257-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nilsson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gustafsson, J. A.</span></span> <span> </span><span class="NLM_article-title">Estrogen receptors: therapies targeted to receptor subtypes</span>. <i>Clin. Pharmacol. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>89</i></span>,  <span class="NLM_fpage">44</span>– <span class="NLM_lpage">55</span>, <span class="refDoi"> DOI: 10.1038/clpt.2010.226</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=10.1038%2Fclpt.2010.226" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=21124311" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsFylt7zN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=89&publication_year=2011&pages=44-55&author=S.+Nilssonauthor=J.+A.+Gustafsson&title=Estrogen+receptors%3A+therapies+targeted+to+receptor+subtypes&doi=10.1038%2Fclpt.2010.226"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Estrogen Receptors: Therapies Targeted to Receptor Subtypes</span></div><div class="casAuthors">Nilsson, S.; Gustafsson, J.-Aa.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Pharmacology & Therapeutics (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">89</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">44-55</span>CODEN:
                <span class="NLM_cas:coden">CLPTAT</span>;
        ISSN:<span class="NLM_cas:issn">0009-9236</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Over the past two decades, we have learned that estrogens play important physiol. roles not only in women but also in men and that the biol. effects of estrogen are mediated by not one but two distinct estrogen receptors (ERs), ERα and ERβ.  Our appreciation of the physiol. importance of estrogen and the mechanisms by which it acts has significantly increased over the years; however, we are only now beginning to decipher the roles of ERα and ERβ in different organs and to elucidate how selective ligands, acting through either of the two ERs, can prevent or treat various age- or sex-specific diseases.  The specific roles of ERα and ERβ and the therapeutic potential of ER subtype-selective agonists in bone and metabolic homeostasis, depression, vasomotor symptoms, neurodegenerative diseases, and cancer are reviewed herein.  It must be stated, however, that appropriate clin. studies are necessary to validate these compds. as agents for the prevention and treatment of diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqq0tQWjd5LTLVg90H21EOLACvtfcHk0lgX6FVKQ01qCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsFylt7zN&md5=30f4fac2dcdfb20035b54783a7684a8c</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2Fclpt.2010.226&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fclpt.2010.226%26sid%3Dliteratum%253Aachs%26aulast%3DNilsson%26aufirst%3DS.%26aulast%3DGustafsson%26aufirst%3DJ.%2BA.%26atitle%3DEstrogen%2520receptors%253A%2520therapies%2520targeted%2520to%2520receptor%2520subtypes%26jtitle%3DClin.%2520Pharmacol.%2520Ther.%26date%3D2011%26volume%3D89%26spage%3D44%26epage%3D55%26doi%3D10.1038%2Fclpt.2010.226" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Toy, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weir, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Razavi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goeppert, A. U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mazzola, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morrow, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Stanchina, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlson, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uddin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fanning, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katzenellenbogen, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greene, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baselga, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chandarlapaty, S.</span></span> <span> </span><span class="NLM_article-title">Activating ESR1 mutations differentially affect the efficacy of ER antagonists</span>. <i>Cancer Discovery</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">277</span>– <span class="NLM_lpage">287</span>, <span class="refDoi"> DOI: 10.1158/2159-8290.CD-15-1523</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=10.1158%2F2159-8290.CD-15-1523" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=27986707" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjs1Klsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&pages=277-287&author=W.+Toyauthor=H.+Weirauthor=P.+Razaviauthor=M.+Lawsonauthor=A.+U.+Goeppertauthor=A.+M.+Mazzolaauthor=A.+Smithauthor=J.+Wilsonauthor=C.+Morrowauthor=W.+L.+Wongauthor=E.+De+Stanchinaauthor=K.+E.+Carlsonauthor=T.+S.+Martinauthor=S.+Uddinauthor=Z.+Liauthor=S.+Fanningauthor=J.+A.+Katzenellenbogenauthor=G.+Greeneauthor=J.+Baselgaauthor=S.+Chandarlapaty&title=Activating+ESR1+mutations+differentially+affect+the+efficacy+of+ER+antagonists&doi=10.1158%2F2159-8290.CD-15-1523"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Activating ESR1 Mutations Differentially Affect the Efficacy of ER Antagonists</span></div><div class="casAuthors">Toy, Weiyi; Weir, Hazel; Razavi, Pedram; Lawson, Mandy; Goeppert, Anne U.; Mazzola, Anne Marie; Smith, Aaron; Wilson, Joanne; Morrow, Christopher; Wong, Wai Lin; De Stanchina, Elisa; Carlson, Kathryn E.; Martin, Teresa S.; Uddin, Sharmeen; Li, Zhiqiang; Fanning, Sean; Katzenellenbogen, John A.; Greene, Geoffrey; Baselga, Jose; Chandarlapaty, Sarat</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">277-287</span>CODEN:
                <span class="NLM_cas:coden">CDAIB2</span>;
        ISSN:<span class="NLM_cas:issn">2159-8274</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Recent studies have identified somatic ESR1 mutations in patients with metastatic breast cancer and found some of them to promote estrogen-independent activation of the receptor.  The degree to which all recurrent mutants can drive estrogen-independent activities and reduced sensitivity to ER antagonists like fulvestrant is not established.  In this report, we characterize the spectrum of ESR1 mutations from more than 900 patients.  ESR1 mutations were detected in 10%, with D538G being the most frequent (36%), followed by Y537S (14%).  Several novel, activating mutations were also detected (e.g., L469V, V422del, and Y537D).  Although many mutations lead to constitutive activity and reduced sensitivity to ER antagonists, only select mutants such as Y537S caused a magnitude of change assocd. with fulvestrant resistance in vivo.  Correspondingly, tumors driven by Y537S, but not D5358G, E380Q, or S463P, were less effectively inhibited by fulvestrant than more potent and bioavailable antagonists, including AZD9496.  These data point to a need for antagonists with optimal pharmacokinetic properties to realize clin. efficacy against certain ESR1 mutants.  Significance: A diversity of activating ESR1 mutations exist, only some of which confer resistance to existing ER antagonists that might be overcome by next-generation inhibitors such as AZD9496.  Cancer Discov; 7(3); 277-87. ©2016 AACR.  This article is highlighted in the In This Issue feature, p.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZyS-eS6T9ZLVg90H21EOLACvtfcHk0li6vFGEmkyiSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjs1Klsbo%253D&md5=398dd5202db358ba8facab67055895c9</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1158%2F2159-8290.CD-15-1523&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F2159-8290.CD-15-1523%26sid%3Dliteratum%253Aachs%26aulast%3DToy%26aufirst%3DW.%26aulast%3DWeir%26aufirst%3DH.%26aulast%3DRazavi%26aufirst%3DP.%26aulast%3DLawson%26aufirst%3DM.%26aulast%3DGoeppert%26aufirst%3DA.%2BU.%26aulast%3DMazzola%26aufirst%3DA.%2BM.%26aulast%3DSmith%26aufirst%3DA.%26aulast%3DWilson%26aufirst%3DJ.%26aulast%3DMorrow%26aufirst%3DC.%26aulast%3DWong%26aufirst%3DW.%2BL.%26aulast%3DDe%2BStanchina%26aufirst%3DE.%26aulast%3DCarlson%26aufirst%3DK.%2BE.%26aulast%3DMartin%26aufirst%3DT.%2BS.%26aulast%3DUddin%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DFanning%26aufirst%3DS.%26aulast%3DKatzenellenbogen%26aufirst%3DJ.%2BA.%26aulast%3DGreene%26aufirst%3DG.%26aulast%3DBaselga%26aufirst%3DJ.%26aulast%3DChandarlapaty%26aufirst%3DS.%26atitle%3DActivating%2520ESR1%2520mutations%2520differentially%2520affect%2520the%2520efficacy%2520of%2520ER%2520antagonists%26jtitle%3DCancer%2520Discovery%26date%3D2017%26volume%3D7%26spage%3D277%26epage%3D287%26doi%3D10.1158%2F2159-8290.CD-15-1523" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Musgrove, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sutherland, R. L.</span></span> <span> </span><span class="NLM_article-title">Biological determinants of endocrine resistance in breast cancer</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">631</span>– <span class="NLM_lpage">643</span>, <span class="refDoi"> DOI: 10.1038/nrc2713</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=10.1038%2Fnrc2713" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=19701242" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtVegtrzL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2009&pages=631-643&author=E.+A.+Musgroveauthor=R.+L.+Sutherland&title=Biological+determinants+of+endocrine+resistance+in+breast+cancer&doi=10.1038%2Fnrc2713"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Biological determinants of endocrine resistance in breast cancer</span></div><div class="casAuthors">Musgrove, Elizabeth A.; Sutherland, Robert L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">631-643</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Endocrine therapies targeting estrogen action (anti-estrogens, such as tamoxifen, and aromatase inhibitors) decrease mortality from breast cancer, but their efficacy is limited by intrinsic and acquired therapeutic resistance.  Candidate mol. biomarkers and gene expression signatures of tamoxifen response emphasize the importance of deregulation of proliferation and survival signaling in endocrine resistance.  However, definition of the specific genetic lesions and mol. processes that det. clin. endocrine resistance is incomplete.  The development of large-scale computational and genetic approaches offers the promise of identifying the mediators of endocrine resistance that may be exploited as potential therapeutic targets and biomarkers of response in the clinic.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbi_1MVsJjz7Vg90H21EOLACvtfcHk0li6vFGEmkyiSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtVegtrzL&md5=e57a6a082e0f41d26422350657201de5</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1038%2Fnrc2713&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc2713%26sid%3Dliteratum%253Aachs%26aulast%3DMusgrove%26aufirst%3DE.%2BA.%26aulast%3DSutherland%26aufirst%3DR.%2BL.%26atitle%3DBiological%2520determinants%2520of%2520endocrine%2520resistance%2520in%2520breast%2520cancer%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2009%26volume%3D9%26spage%3D631%26epage%3D643%26doi%3D10.1038%2Fnrc2713" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mills, J. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rutkovsky, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giordano, A.</span></span> <span> </span><span class="NLM_article-title">Mechanisms of resistance in estrogen receptor positive breast cancer: overcoming resistance to tamoxifen/aromatase inhibitors</span>. <i>Curr. Opin. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">59</span>– <span class="NLM_lpage">65</span>, <span class="refDoi"> DOI: 10.1016/j.coph.2018.04.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=10.1016%2Fj.coph.2018.04.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=29719270" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A528%3ADC%252BC1cXotlOksbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2018&pages=59-65&author=J.+N.+Millsauthor=A.+C.+Rutkovskyauthor=A.+Giordano&title=Mechanisms+of+resistance+in+estrogen+receptor+positive+breast+cancer%3A+overcoming+resistance+to+tamoxifen%2Faromatase+inhibitors&doi=10.1016%2Fj.coph.2018.04.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanisms of resistance in estrogen receptor positive breast cancer: overcoming resistance to tamoxifen/aromatase inhibitors</span></div><div class="casAuthors">Mills, Jamie N.; Rutkovsky, Alex C.; Giordano, Antonio</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Pharmacology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">59-65</span>CODEN:
                <span class="NLM_cas:coden">COPUBK</span>;
        ISSN:<span class="NLM_cas:issn">1471-4892</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Several mechanisms of resistance have been identified, underscoring the complex nature of estrogen receptor (ER) signaling and the many connections between this pathway and other essential signaling pathways in breast cancer cells.  Many therapeutic targets of cell signaling and cell cycle pathways have met success with endocrine therapy and remain an ongoing area of investigation.  This review focuses on two major pathways that have recently emerged as important opportunities for therapeutic intervention in endocrine resistant breast tumors: PI3K/AKT/mTOR cell signaling and cyclinD1/cyclin-dependent kinase 4/6 cell cycle pathways.  Addnl., we highlight individual and combination strategies in current clin. trials that target these pathways and others under investigation for the treatment of ER pos. breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqVYQkU5BQS2LVg90H21EOLACvtfcHk0li6vFGEmkyiSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXotlOksbg%253D&md5=7eb380dd5c8e441b8dd2602b40aa94b5</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.coph.2018.04.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.coph.2018.04.009%26sid%3Dliteratum%253Aachs%26aulast%3DMills%26aufirst%3DJ.%2BN.%26aulast%3DRutkovsky%26aufirst%3DA.%2BC.%26aulast%3DGiordano%26aufirst%3DA.%26atitle%3DMechanisms%2520of%2520resistance%2520in%2520estrogen%2520receptor%2520positive%2520breast%2520cancer%253A%2520overcoming%2520resistance%2520to%2520tamoxifen%252Faromatase%2520inhibitors%26jtitle%3DCurr.%2520Opin.%2520Pharmacol.%26date%3D2018%26volume%3D41%26spage%3D59%26epage%3D65%26doi%3D10.1016%2Fj.coph.2018.04.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boer, K.</span></span> <span> </span><span class="NLM_article-title">Fulvestrant in advanced breast cancer: evidence to date and place in therapy</span>. <i>Ther. Adv. Med. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">465</span>– <span class="NLM_lpage">479</span>, <span class="refDoi"> DOI: 10.1177/1758834017711097</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=10.1177%2F1758834017711097" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=28717399" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvVSntrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2017&pages=465-479&author=K.+Boer&title=Fulvestrant+in+advanced+breast+cancer%3A+evidence+to+date+and+place+in+therapy&doi=10.1177%2F1758834017711097"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Fulvestrant in advanced breast cancer: evidence to date and place in therapy</span></div><div class="casAuthors">Boer, Katalin</div><div class="citationInfo"><span class="NLM_cas:title">Therapeutic Advances in Medical Oncology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">465-479</span>CODEN:
                <span class="NLM_cas:coden">TAMOB8</span>;
        ISSN:<span class="NLM_cas:issn">1758-8340</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications Ltd.</span>)
        </div><div class="casAbstract">A review.  Breast cancer is a classical hormone-dependent tumor; therefore, endocrine therapy is the mainstay of treatment for hormone receptor-pos., human epidermal growth factor 2-neg. advanced breast cancer.  Until recently, classical endocrine agents such as tamoxifen, steroidal and nonsteroidal aromatase inhibitors and fulvestrant have been widely used in postmenopausal patients to treat locally advanced or metastatic disease.  However, for patients with this subtype of breast cancer, the landscape of endocrine therapy is rapidly changing.  Therapies targeting estrogen modulation have evolved in recent years following the introduction of targeted agents, mTOR and CDK 4/6 inhibitors that are administered in combination with hormone therapy.  As a result, options for endocrine therapy have expanded in recent years, and a variety of single-agent or combinations of targeted drugs and endocrine therapies are accepted.  Fulvestrant is a selective estrogen receptor downregulator (SERD) which was introduced to clin. practice in 2002, initially with the indication to treat postmenopausal women with hormone-receptor-pos. advanced breast cancer as second-line therapy postdisease progression after aromatase inhibitors or tamoxifen.  Addnl., fulvestrant has also been shown to be active in patients previously untreated with endocrine therapy, either both in the neoadjuvant and the metastatic setting, alone or in combination with other targeted therapies.  Currently, the std. dose is 500 mg, which is administered with a loading dose.  Fulvestrant received a new FDA indication in Dec. 2016, in combination with palbociclib, both in pre/peri/postmenopausal women with breast cancer progressing after endocrine therapy.  This manuscript aims to give an overview of new efficacy data and the current role of fulvestrant in the systemic therapy of hormone-receptor-pos. advanced breast cancer, in the context of other available therapeutic modalities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpRi8P0FTgbVrVg90H21EOLACvtfcHk0liR_5peLd9wcQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvVSntrs%253D&md5=c8ce774327a3523edabfe23c557c5a19</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1177%2F1758834017711097&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1758834017711097%26sid%3Dliteratum%253Aachs%26aulast%3DBoer%26aufirst%3DK.%26atitle%3DFulvestrant%2520in%2520advanced%2520breast%2520cancer%253A%2520evidence%2520to%2520date%2520and%2520place%2520in%2520therapy%26jtitle%3DTher.%2520Adv.%2520Med.%2520Oncol.%26date%3D2017%26volume%3D9%26spage%3D465%26epage%3D479%26doi%3D10.1177%2F1758834017711097" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mehta, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barlow, W. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albain, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vandenberg, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dakhil, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tirumali, N. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lew, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayes, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gralow, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Livingston, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hortobagyi, G. N.</span></span> <span> </span><span class="NLM_article-title">Combination anastrozole and fulvestrant in metastatic breast cancer</span>. <i>N. Engl. J. Med.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>367</i></span>,  <span class="NLM_fpage">435</span>– <span class="NLM_lpage">444</span>, <span class="refDoi"> DOI: 10.1056/NEJMoa1201622</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=10.1056%2FNEJMoa1201622" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=22853014" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtlSltrbF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=367&publication_year=2012&pages=435-444&author=R.+S.+Mehtaauthor=W.+E.+Barlowauthor=K.+S.+Albainauthor=T.+A.+Vandenbergauthor=S.+R.+Dakhilauthor=N.+R.+Tirumaliauthor=D.+L.+Lewauthor=D.+F.+Hayesauthor=J.+R.+Gralowauthor=R.+B.+Livingstonauthor=G.+N.+Hortobagyi&title=Combination+anastrozole+and+fulvestrant+in+metastatic+breast+cancer&doi=10.1056%2FNEJMoa1201622"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Combination anastrozole and fulvestrant in metastatic breast cancer</span></div><div class="casAuthors">Mehta, Rita S.; Barlow, William E.; Albain, Kathy S.; Vandenberg, Ted A.; Dakhil, Shaker R.; Tirumali, Nagendra R.; Lew, Danika L.; Hayes, Daniel F.; Gralow, Julie R.; Livingston, Robert B.; Hortobagyi, Gabriel N.</div><div class="citationInfo"><span class="NLM_cas:title">New England Journal of Medicine</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">367</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">435-444</span>CODEN:
                <span class="NLM_cas:coden">NEJMAG</span>;
        ISSN:<span class="NLM_cas:issn">0028-4793</span>.
    
            (<span class="NLM_cas:orgname">Massachusetts Medical Society</span>)
        </div><div class="casAbstract">BACKGROUND: The aromatase inhibitor anastrozole inhibits estrogen synthesis.  Fulvestrant binds and accelerates degrdn. of estrogen receptors.  We hypothesized that these two agents in combination might be more effective than anastrozole alone in patients with hormone-receptor (HR)-pos. metastatic breast cancer.  METHODS: Postmenopausal women with previously untreated metastatic disease were randomly assigned, in a 1:1 ratio, to receive either 1 mg of anastrozole orally every day (group 1), with crossover to fulvestrant alone strongly encouraged if the disease progressed, or anastrozole and fulvestrant in combination (group 2).  Patients were stratified according to prior or no prior receipt of adjuvant tamoxifen therapy.  Fulvestrant was administered i.m. at a dose of 500 mg on day 1 and 250 mg on days 14 and 28 and monthly thereafter.  The primary end point was progression-free survival, with overall survival designated as a prespecified secondary outcome.  RESULTS: The median progression-free survival was 13.5 mo in group 1 and 15.0 mo in group 2 (hazard ratio for progression or death with combination therapy, 0.80; 95% confidence interval [CI], 0.68 to 0.94; P = 0.007 by the log-rank test).  The combination therapy was generally more effective than anastrozole alone in all subgroups, with no significant interactions.  Overall survival was also longer with combination therapy (median, 41.3 mo in group 1 and 47.7 mo in group 2; hazard ratio for death, 0.81; 95% CI, 0.65 to 1.00; P = 0.05 by the log-rank test), despite the fact that 41% of the patients in group 1 crossed over to fulvestrant after progression.  Three deaths that were possibly assocd. with treatment occurred in group 2.  The rates of grade 3 to 5 toxic effects did not differ significantly between the two groups.  CONCLUSIONS: The combination of anastrozole and fulvestrant was superior to anastrozole alone or sequential anastrozole and fulvestrant for the treatment of HR-pos. metastatic breast cancer, despite the use of a dose of fulvestrant that was below the current std.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphSxtK-2-aa7Vg90H21EOLACvtfcHk0liR_5peLd9wcQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtlSltrbF&md5=869d5b1663c60bc2a6fda960b4d8f00e</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1056%2FNEJMoa1201622&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMoa1201622%26sid%3Dliteratum%253Aachs%26aulast%3DMehta%26aufirst%3DR.%2BS.%26aulast%3DBarlow%26aufirst%3DW.%2BE.%26aulast%3DAlbain%26aufirst%3DK.%2BS.%26aulast%3DVandenberg%26aufirst%3DT.%2BA.%26aulast%3DDakhil%26aufirst%3DS.%2BR.%26aulast%3DTirumali%26aufirst%3DN.%2BR.%26aulast%3DLew%26aufirst%3DD.%2BL.%26aulast%3DHayes%26aufirst%3DD.%2BF.%26aulast%3DGralow%26aufirst%3DJ.%2BR.%26aulast%3DLivingston%26aufirst%3DR.%2BB.%26aulast%3DHortobagyi%26aufirst%3DG.%2BN.%26atitle%3DCombination%2520anastrozole%2520and%2520fulvestrant%2520in%2520metastatic%2520breast%2520cancer%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2012%26volume%3D367%26spage%3D435%26epage%3D444%26doi%3D10.1056%2FNEJMoa1201622" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Johnston, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheung, K. L.</span></span> <span> </span><span class="NLM_article-title">Fulvestrant - a novel endocrine therapy for breast cancer</span>. <i>Curr. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">902</span>– <span class="NLM_lpage">914</span>, <span class="refDoi"> DOI: 10.2174/092986710790820633</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=10.2174%2F092986710790820633" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=20156170" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A528%3ADC%252BC3cXkvV2jtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2010&pages=902-914&author=S.+J.+Johnstonauthor=K.+L.+Cheung&title=Fulvestrant+-+a+novel+endocrine+therapy+for+breast+cancer&doi=10.2174%2F092986710790820633"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Fulvestrant - a novel endocrine therapy for breast cancer</span></div><div class="casAuthors">Johnston, S. J.; Cheung, K. L.</div><div class="citationInfo"><span class="NLM_cas:title">Current Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">902-914</span>CODEN:
                <span class="NLM_cas:coden">CMCHE7</span>;
        ISSN:<span class="NLM_cas:issn">0929-8673</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Fulvestrant is a novel endocrine therapy for breast cancer, with a unique structure and mode of action.  It binds competitively to the estrogen receptor (ER), with high affinity, and down regulates ER by functional blockade and increased turnover.review.  Fulvestrant has reached the clinic via extensive pre-clin. and clin. trials, which demonstrated fulvestrant's unique characteristics and showed that they translate to equiv. or improved clin. efficacy compared to established endocrine agents.  Fulvestrant is currently licensed for use in postmenopausal women with hormone receptor pos. advanced breast cancer which has progressed on prior endocrine therapy.  As a pure estrogen antagonist, fulvestrant avoids the risk of detrimental side effects of selective ER modulators such as tamoxifen, which has partial agonist activity.  Fulvestrant, the only parenteral agent in this setting, has a good side effect profile and is well tolerated.  Due to its unique mode of action, fulvestrant lacks cross-resistance with existing agents.  Fulvestrant is the subject of much ongoing research, which utilizes knowledge of its novel mechanism and pharmacokinetic profile in order to optimize clin. efficacy and explore new roles, including first-line use in advanced breast cancer, use in combination with existing agents, in males, and in premenopausal women, and use as an adjuvant therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq-9Noer_gy07Vg90H21EOLACvtfcHk0liR_5peLd9wcQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXkvV2jtr0%253D&md5=9dab4f74acde17ff56508769d4822161</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.2174%2F092986710790820633&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F092986710790820633%26sid%3Dliteratum%253Aachs%26aulast%3DJohnston%26aufirst%3DS.%2BJ.%26aulast%3DCheung%26aufirst%3DK.%2BL.%26atitle%3DFulvestrant%2520-%2520a%2520novel%2520endocrine%2520therapy%2520for%2520breast%2520cancer%26jtitle%3DCurr.%2520Med.%2520Chem.%26date%3D2010%26volume%3D17%26spage%3D902%26epage%3D914%26doi%3D10.2174%2F092986710790820633" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ellis, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Llombart-Cussac, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feltl, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dewar, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jasiowka, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hewson, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rukazenkov, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robertson, J. F.</span></span> <span> </span><span class="NLM_article-title">Fulvestrant 500 mg versus anastrozole 1 mg for the first-line treatment of advanced breast cancer: overall survival analysis from the phase II first study</span>. <i>J. Clin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">3781</span>– <span class="NLM_lpage">3787</span>, <span class="refDoi"> DOI: 10.1200/JCO.2015.61.5831</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=10.1200%2FJCO.2015.61.5831" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=26371134" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtVSntb3K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2015&pages=3781-3787&author=M.+J.+Ellisauthor=A.+Llombart-Cussacauthor=D.+Feltlauthor=J.+A.+Dewarauthor=M.+Jasiowkaauthor=N.+Hewsonauthor=Y.+Rukazenkovauthor=J.+F.+Robertson&title=Fulvestrant+500+mg+versus+anastrozole+1+mg+for+the+first-line+treatment+of+advanced+breast+cancer%3A+overall+survival+analysis+from+the+phase+II+first+study&doi=10.1200%2FJCO.2015.61.5831"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Fulvestrant 500 mg versus anastrozole 1 mg for the first-line treatment of advanced breast cancer: overall survival analysis from the phase II FIRST study</span></div><div class="casAuthors">Ellis, Matthew J.; Llombart-Cussac, Antonio; Feltl, David; Dewar, John A.; Jasiowka, Marek; Hewson, Nicola; Rukazenkov, Yuri; Robertson, John F. R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Clinical Oncology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">32</span>),
    <span class="NLM_cas:pages">3781-3787</span>CODEN:
                <span class="NLM_cas:coden">JCONDN</span>;
        ISSN:<span class="NLM_cas:issn">0732-183X</span>.
    
            (<span class="NLM_cas:orgname">American Society of Clinical Oncology</span>)
        </div><div class="casAbstract">To compare overall survival (OS) for fulvestrant 500 mg vs. anastrozole as first-line endocrine therapy for advanced breast cancer.  The Fulvestrant First-Line Study Comparing Endocrine Treatments (FIRST) was a phase II, randomized, open-label, multicenter trial.  Postmenopausal women with estrogen receptor-pos., locally advanced/metastatic breast cancer who had no previous therapy for advanced disease received either fulvestrant 500 mg (days 0, 14, 28, and every 28 days thereafter) or anastrozole 1 mg (daily).  The primary end point (clin. benefit rate [72.5% and 67.0%]) and a follow-up anal. (median time to progression [23.4 mo and 13.1 mo]) have been reported previously for fulvestrant 500 mg and anastrozole, resp.  Subsequently, the protocol was amended to assess OS by unadjusted log-rank test after approx. 65% of patients had died.  Treatment effect on OS across several subgroups was examd.  Tolerability was evaluated by adverse event monitoring.  Results In total, 205 patients were randomly assigned (fulvestrant 500 mg, n = 102; anastrozole, n = 103).  At data cutoff, 61.8% (fulvestrant 500 mg, n = 63) and 71.8% (anastrozole, n = 74) had died.  The hazard ratio (95% CI) for OS with fulvestrant 500 mg vs. anastrozole was 0.70 (0.50 to 0.98; P = .04; median OS, 54.1 mo v 48.4 mo).  Treatment effects seemed generally consistent across the subgroups analyzed.  No new safety issues were obsd.  There are several limitations of this OS anal., including that it was not planned in the original protocol but instead was added after time-to-progression results were analyzed, and that not all patients participated in addnl. OS follow-up.  However, the present results suggest fulvestrant 500 mg extends OS vs. anastrozole.  This finding now awaits prospective confirmation in the larger phase III FALCON (Fulvestrant and Anastrozole Compared in Hormonal Therapy Na.ovrddot.ive Advanced Breast Cancer).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgcontOU3287Vg90H21EOLACvtfcHk0lj8doT8sv_MUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtVSntb3K&md5=45cb5f02224b33857d55e5cb5b69e775</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1200%2FJCO.2015.61.5831&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1200%252FJCO.2015.61.5831%26sid%3Dliteratum%253Aachs%26aulast%3DEllis%26aufirst%3DM.%2BJ.%26aulast%3DLlombart-Cussac%26aufirst%3DA.%26aulast%3DFeltl%26aufirst%3DD.%26aulast%3DDewar%26aufirst%3DJ.%2BA.%26aulast%3DJasiowka%26aufirst%3DM.%26aulast%3DHewson%26aufirst%3DN.%26aulast%3DRukazenkov%26aufirst%3DY.%26aulast%3DRobertson%26aufirst%3DJ.%2BF.%26atitle%3DFulvestrant%2520500%2520mg%2520versus%2520anastrozole%25201%2520mg%2520for%2520the%2520first-line%2520treatment%2520of%2520advanced%2520breast%2520cancer%253A%2520overall%2520survival%2520analysis%2520from%2520the%2520phase%2520II%2520first%2520study%26jtitle%3DJ.%2520Clin.%2520Oncol.%26date%3D2015%26volume%3D33%26spage%3D3781%26epage%3D3787%26doi%3D10.1200%2FJCO.2015.61.5831" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Howell, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sapunar, F.</span></span> <span> </span><span class="NLM_article-title">Fulvestrant revisited: efficacy and safety of the 500-mg dose</span>. <i>Clin. Breast Cancer</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">204</span>– <span class="NLM_lpage">210</span>, <span class="refDoi"> DOI: 10.1016/j.clbc.2011.02.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=10.1016%2Fj.clbc.2011.02.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=21729658" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlKisr7E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2011&pages=204-210&author=A.+Howellauthor=F.+Sapunar&title=Fulvestrant+revisited%3A+efficacy+and+safety+of+the+500-mg+dose&doi=10.1016%2Fj.clbc.2011.02.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Fulvestrant revisited: efficacy and safety of the 500-mg dose</span></div><div class="casAuthors">Howell, Anthony; Sapunar, Francisco</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Breast Cancer</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">204-210</span>CODEN:
                <span class="NLM_cas:coden">CBCLB7</span>;
        ISSN:<span class="NLM_cas:issn">1526-8209</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review.  Postmenopausal women with hormone receptor-pos. advanced breast cancer are candidates for endocrine therapy.  As the disease will eventually progress in most patients, it is important to investigate agents with novel modes of action to reduce the likelihood of treatment cross-resistance.  Fulvestrant is an estrogen receptor antagonist with no known agonist effects that has been shown to be as effective as anastrozole following failure on tamoxifen, at the approved dose of 250 mg/mo.  However, pharmacokinetic modeling and evidence of clin. efficacy in early trials, together with the favorable tolerability profile of fulvestrant 250 mg, led to suggestions that increasing the fulvestrant dose would lead to an improved benefit-risk profile.  This review describes the rationale behind the development of a 500 mg/mo higher dose of fulvestrant and details relevant clin. trials, including the pivotal phase III COmparisoN of Faslodex In Recurrent or Metastatic breast cancer (CONFIRM) study.  CONFIRM demonstrated a significant improvement in progression-free survival for fulvestrant 500 mg vs. 250 mg in postmenopausal patients who had progressed on previous endocrine therapy.  Here, we present and discuss a pooled safety anal. of CONFIRM and three further clin. studies demonstrating fulvestrant 500 mg to be well-tolerated with no evidence of dose-related adverse events.  Overall, these data indicate an improved benefit-risk profile for fulvestrant 500 mg vs. 250 mg following failure on prior endocrine therapy, and suggest that fulvestrant 500 mg may be considered in future as initial endocrine treatment for advanced breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4U1d7Q6edlbVg90H21EOLACvtfcHk0lj8doT8sv_MUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlKisr7E&md5=0d1063c9546320294193e92d3f2097dd</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fj.clbc.2011.02.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.clbc.2011.02.002%26sid%3Dliteratum%253Aachs%26aulast%3DHowell%26aufirst%3DA.%26aulast%3DSapunar%26aufirst%3DF.%26atitle%3DFulvestrant%2520revisited%253A%2520efficacy%2520and%2520safety%2520of%2520the%2520500-mg%2520dose%26jtitle%3DClin.%2520Breast%2520Cancer%26date%3D2011%26volume%3D11%26spage%3D204%26epage%3D210%26doi%3D10.1016%2Fj.clbc.2011.02.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Willson, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henke, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Momtahen, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Charifson, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Batchelor, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lubahn, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moore, L. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliver, B. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sauls, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Triantafillou, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolfe, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baer, P. G.</span></span> <span> </span><span class="NLM_article-title">3-[4-(1,2-Diphenylbut-1-enyl)phenyl]acrylic acid: a non-steroidal estrogen with functional selectivity for bone over uterus in rats</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>37</i></span>,  <span class="NLM_fpage">1550</span>– <span class="NLM_lpage">1552</span>, <span class="refDoi"> DOI: 10.1021/jm00037a002</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm00037a002" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A528%3ADyaK2cXktlCqt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=1994&pages=1550-1552&author=T.+M.+Willsonauthor=B.+R.+Henkeauthor=T.+M.+Momtahenauthor=P.+S.+Charifsonauthor=K.+W.+Batchelorauthor=D.+B.+Lubahnauthor=L.+B.+Mooreauthor=B.+B.+Oliverauthor=H.+R.+Saulsauthor=J.+A.+Triantafillouauthor=S.+G.+Wolfeauthor=P.+G.+Baer&title=3-%5B4-%281%2C2-Diphenylbut-1-enyl%29phenyl%5Dacrylic+acid%3A+a+non-steroidal+estrogen+with+functional+selectivity+for+bone+over+uterus+in+rats&doi=10.1021%2Fjm00037a002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">3-[4-(1,2-Diphenylbut-1-enyl)phenyl]acrylic Acid: A Non-Steroidal Estrogen with Functional Selectivity for Bone over Uterus in Rats</span></div><div class="casAuthors">Willson, Timothy M.; Henke, Brad R.; Momtahen, Tanya M.; Charifson, Paul S.; Batchelor, Kenneth W.; Lubahn, Dennis B.; Moore, Linda B.; Oliver, Beverly B.; Sauls, Howard R.; et al.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1550-2</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    </div><div class="casAbstract">Exploiting knowledge of the mol. biol. of the estrogen receptor and the existence of two independent transcription activation domains, a series of tissue-selective non-steroidal estrogens was designed.  The ethanolamine side-chain of tamoxifen was targeted for modification, resulting in the identification of a series of α,β-unsatd. acid and amide analogs I [R = OH, NEt2, morpholino, NH(CH2)3OMe, dicyclohexylamino, NEtCH2CH2NMe2, NH(CH2)3OH, N-methyloctylamino, NHEt,NH2].  I [R = OH] profiled as an estrogen with full agonist activity in bone while maintaining antagonist activity in the uterus of rats.  The demonstration that different parts of the triphenylethylene structure control expression of estrogenic activity in bone and uterus has implications in the design of tissue selective ligands for other members of the steroid/retinoid family of intracellular receptors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp4iuXq4R8fm7Vg90H21EOLACvtfcHk0lj8doT8sv_MUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXktlCqt7g%253D&md5=721906b3513f7e8cdc9ae08ecd99a93b</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fjm00037a002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm00037a002%26sid%3Dliteratum%253Aachs%26aulast%3DWillson%26aufirst%3DT.%2BM.%26aulast%3DHenke%26aufirst%3DB.%2BR.%26aulast%3DMomtahen%26aufirst%3DT.%2BM.%26aulast%3DCharifson%26aufirst%3DP.%2BS.%26aulast%3DBatchelor%26aufirst%3DK.%2BW.%26aulast%3DLubahn%26aufirst%3DD.%2BB.%26aulast%3DMoore%26aufirst%3DL.%2BB.%26aulast%3DOliver%26aufirst%3DB.%2BB.%26aulast%3DSauls%26aufirst%3DH.%2BR.%26aulast%3DTriantafillou%26aufirst%3DJ.%2BA.%26aulast%3DWolfe%26aufirst%3DS.%2BG.%26aulast%3DBaer%26aufirst%3DP.%2BG.%26atitle%3D3-%255B4-%25281%252C2-Diphenylbut-1-enyl%2529phenyl%255Dacrylic%2520acid%253A%2520a%2520non-steroidal%2520estrogen%2520with%2520functional%2520selectivity%2520for%2520bone%2520over%2520uterus%2520in%2520rats%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1994%26volume%3D37%26spage%3D1550%26epage%3D1552%26doi%3D10.1021%2Fjm00037a002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lai, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kahraman, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Govek, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagasawa, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonnefous, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Julien, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Douglas, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sensintaffar, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aparicio, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaufman, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prudente, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moon, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joseph, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Darimont, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brigham, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grillot, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heyman, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rix, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hager, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, N. D.</span></span> <span> </span><span class="NLM_article-title">Identification of GDC-0810 (ARN-810), an orally bioavailable selective estrogen receptor degrader (SERD) that demonstrates robust activity in tamoxifen-resistant breast cancer xenografts</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">4888</span>– <span class="NLM_lpage">4904</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00054</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00054" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmsFyisb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=4888-4904&author=A.+Laiauthor=M.+Kahramanauthor=S.+Govekauthor=J.+Nagasawaauthor=C.+Bonnefousauthor=J.+Julienauthor=K.+Douglasauthor=J.+Sensintaffarauthor=N.+Luauthor=K.+J.+Leeauthor=A.+Aparicioauthor=J.+Kaufmanauthor=J.+Qianauthor=G.+Shaoauthor=R.+Prudenteauthor=M.+J.+Moonauthor=J.+D.+Josephauthor=B.+Darimontauthor=D.+Brighamauthor=K.+Grillotauthor=R.+Heymanauthor=P.+J.+Rixauthor=J.+H.+Hagerauthor=N.+D.+Smith&title=Identification+of+GDC-0810+%28ARN-810%29%2C+an+orally+bioavailable+selective+estrogen+receptor+degrader+%28SERD%29+that+demonstrates+robust+activity+in+tamoxifen-resistant+breast+cancer+xenografts&doi=10.1021%2Facs.jmedchem.5b00054"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of GDC-0810 (ARN-810), an Orally Bioavailable Selective Estrogen Receptor Degrader (SERD) that Demonstrates Robust Activity in Tamoxifen-Resistant Breast Cancer Xenografts</span></div><div class="casAuthors">Lai, Andiliy; Kahraman, Mehmet; Govek, Steven; Nagasawa, Johnny; Bonnefous, Celine; Julien, Jackie; Douglas, Karensa; Sensintaffar, John; Lu, Nhin; Lee, Kyoung-jin; Aparicio, Anna; Kaufman, Josh; Qian, Jing; Shao, Gang; Prudente, Rene; Moon, Michael J.; Joseph, James D.; Darimont, Beatrice; Brigham, Daniel; Grillot, Kate; Heyman, Richard; Rix, Peter J.; Hager, Jeffrey H.; Smith, Nicholas D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">4888-4904</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Approx. 80% of breast cancers are estrogen receptor alpha (ER-α) pos., and although women typically initially respond well to antihormonal therapies such as tamoxifen and aromatase inhibitors, resistance often emerges.  Although a variety of resistance mechanism may be at play in this state, there is evidence that in many cases the ER still plays a central role, including mutations in the ER leading to constitutively active receptor.  Fulvestrant is a steroid-based, selective estrogen receptor degrader (SERD) that both antagonizes and degrades ER-α and is active in patients who have progressed on antihormonal agents.  However, fulvestrant suffers from poor pharmaceutical properties and must be administered by i.m. injections that limit the total amt. of drug that can be administered and hence lead to the potential for incomplete receptor blockade.  The authors describe the identification and characterization of a series of small-mol., orally bioavailable SERDs which are potent antagonists and degraders of ER-α and in which the ER-α degrading properties were prospectively optimized.  The lead compd. I (GDC-0810 or ARN-810) demonstrates robust activity in models of tamoxifen-sensitive and tamoxifen-resistant breast cancer, and is currently in clin. trials in women with locally advanced or metastatic estrogen receptor-pos. breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOegJFz-sqwbVg90H21EOLACvtfcHk0ljS9TIkUFncwg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXmsFyisb4%253D&md5=e836ec4d593ce6ed2d9622da5cd898e6</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00054&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00054%26sid%3Dliteratum%253Aachs%26aulast%3DLai%26aufirst%3DA.%26aulast%3DKahraman%26aufirst%3DM.%26aulast%3DGovek%26aufirst%3DS.%26aulast%3DNagasawa%26aufirst%3DJ.%26aulast%3DBonnefous%26aufirst%3DC.%26aulast%3DJulien%26aufirst%3DJ.%26aulast%3DDouglas%26aufirst%3DK.%26aulast%3DSensintaffar%26aufirst%3DJ.%26aulast%3DLu%26aufirst%3DN.%26aulast%3DLee%26aufirst%3DK.%2BJ.%26aulast%3DAparicio%26aufirst%3DA.%26aulast%3DKaufman%26aufirst%3DJ.%26aulast%3DQian%26aufirst%3DJ.%26aulast%3DShao%26aufirst%3DG.%26aulast%3DPrudente%26aufirst%3DR.%26aulast%3DMoon%26aufirst%3DM.%2BJ.%26aulast%3DJoseph%26aufirst%3DJ.%2BD.%26aulast%3DDarimont%26aufirst%3DB.%26aulast%3DBrigham%26aufirst%3DD.%26aulast%3DGrillot%26aufirst%3DK.%26aulast%3DHeyman%26aufirst%3DR.%26aulast%3DRix%26aufirst%3DP.%2BJ.%26aulast%3DHager%26aufirst%3DJ.%2BH.%26aulast%3DSmith%26aufirst%3DN.%2BD.%26atitle%3DIdentification%2520of%2520GDC-0810%2520%2528ARN-810%2529%252C%2520an%2520orally%2520bioavailable%2520selective%2520estrogen%2520receptor%2520degrader%2520%2528SERD%2529%2520that%2520demonstrates%2520robust%2520activity%2520in%2520tamoxifen-resistant%2520breast%2520cancer%2520xenografts%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D4888%26epage%3D4904%26doi%3D10.1021%2Facs.jmedchem.5b00054" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kahraman, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Govek, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagasawa, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonnefous, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Douglas, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sensintaffar, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aparicio, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaufman, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prudente, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joseph, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Darimont, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brigham, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heyman, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rix, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hager, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, N. D.</span></span> <span> </span><span class="NLM_article-title">Maximizing ER-α degradation maximizes activity in a tamoxifen-resistant breast cancer model: identification of GDC-0927</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">50</span>– <span class="NLM_lpage">55</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.8b00414</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.8b00414" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisVensr3M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=50-55&author=M.+Kahramanauthor=S.+P.+Govekauthor=J.+Y.+Nagasawaauthor=A.+Laiauthor=C.+Bonnefousauthor=K.+Douglasauthor=J.+Sensintaffarauthor=N.+Liuauthor=K.+Leeauthor=A.+Aparicioauthor=J.+Kaufmanauthor=J.+Qianauthor=G.+Shaoauthor=R.+Prudenteauthor=J.+D.+Josephauthor=B.+Darimontauthor=D.+Brighamauthor=R.+Heymanauthor=P.+J.+Rixauthor=J.+H.+Hagerauthor=N.+D.+Smith&title=Maximizing+ER-%CE%B1+degradation+maximizes+activity+in+a+tamoxifen-resistant+breast+cancer+model%3A+identification+of+GDC-0927&doi=10.1021%2Facsmedchemlett.8b00414"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Maximizing ER-α Degradation Maximizes Activity in a Tamoxifen-Resistant Breast Cancer Model: Identification of GDC-0927</span></div><div class="casAuthors">Kahraman, Mehmet; Govek, Steven P.; Nagasawa, Johnny Y.; Lai, Andiliy; Bonnefous, Celine; Douglas, Karensa; Sensintaffar, John; Liu, Nhin; Lee, KyoungJin; Aparicio, Anna; Kaufman, Josh; Qian, Jing; Shao, Gang; Prudente, Rene; Joseph, James D.; Darimont, Beatrice; Brigham, Daniel; Heyman, Richard; Rix, Peter J.; Hager, Jeffrey H.; Smith, Nicholas D.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">50-55</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The further optimization of ER-α degrdn. efficacy of a series of ER modulators by refining side-chain substitution led to efficacious selective estrogen receptor degraders (SERDs).  A fluoromethyl azetidine group was found to be preferred and resulted in the identification of bis-phenol chromene 17ha.  In a tamoxifen-resistant breast cancer xenograft model, 17ha (ER-α degrdn. efficacy = 97%) demonstrated tumor regression, together with robust redn. of intratumoral ER-α levels.  However, despite superior oral exposure, 5a (ER-α degrdn. efficacy = 91%) had inferior activity.  This result suggests that optimizing ER-α degrdn. efficacy leads to compds. with robust effects in a model of tamoxifen-resistant breast cancer.  Compd. 17ha (GDC-0927) was evaluated in clin. trials in women with metastatic estrogen receptor-pos. breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgyUydsTFnYrVg90H21EOLACvtfcHk0ljS9TIkUFncwg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisVensr3M&md5=324c1697ef350d169a97960de72ee8a9</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.8b00414&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.8b00414%26sid%3Dliteratum%253Aachs%26aulast%3DKahraman%26aufirst%3DM.%26aulast%3DGovek%26aufirst%3DS.%2BP.%26aulast%3DNagasawa%26aufirst%3DJ.%2BY.%26aulast%3DLai%26aufirst%3DA.%26aulast%3DBonnefous%26aufirst%3DC.%26aulast%3DDouglas%26aufirst%3DK.%26aulast%3DSensintaffar%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DN.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DAparicio%26aufirst%3DA.%26aulast%3DKaufman%26aufirst%3DJ.%26aulast%3DQian%26aufirst%3DJ.%26aulast%3DShao%26aufirst%3DG.%26aulast%3DPrudente%26aufirst%3DR.%26aulast%3DJoseph%26aufirst%3DJ.%2BD.%26aulast%3DDarimont%26aufirst%3DB.%26aulast%3DBrigham%26aufirst%3DD.%26aulast%3DHeyman%26aufirst%3DR.%26aulast%3DRix%26aufirst%3DP.%2BJ.%26aulast%3DHager%26aufirst%3DJ.%2BH.%26aulast%3DSmith%26aufirst%3DN.%2BD.%26atitle%3DMaximizing%2520ER-%25CE%25B1%2520degradation%2520maximizes%2520activity%2520in%2520a%2520tamoxifen-resistant%2520breast%2520cancer%2520model%253A%2520identification%2520of%2520GDC-0927%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2019%26volume%3D10%26spage%3D50%26epage%3D55%26doi%3D10.1021%2Facsmedchemlett.8b00414" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">De
Savi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradbury, R. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rabow, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norman, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Almeida, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andrews, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ballard, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buttar, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Callis, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Currie, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curwen, J. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donald, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feron, L. J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gingell, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glossop, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayter, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hussain, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karoutchi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamont, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacFaul, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moss, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pearson, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tonge, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, G. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weir, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, Z.</span></span> <span> </span><span class="NLM_article-title">Optimization of a novel binding motif to (E)-3-(3,5-difluoro-4-((1R,3R)-2-(2-fluoro-2-methylpropyl)-3-methyl-2,3,4,9-tetrahydro-1H-pyrido[3,4-b]indol-1-yl)phenyl)acrylic Acid (AZD9496), a potent and orally bioavailable selective estrogen receptor downregulator and antagonist</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">8128</span>– <span class="NLM_lpage">8140</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00984</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00984" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFKiu7fK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=8128-8140&author=C.+De%0ASaviauthor=R.+H.+Bradburyauthor=A.+A.+Rabowauthor=R.+A.+Normanauthor=C.+de+Almeidaauthor=D.+M.+Andrewsauthor=P.+Ballardauthor=D.+Buttarauthor=R.+J.+Callisauthor=G.+S.+Currieauthor=J.+O.+Curwenauthor=C.+D.+Daviesauthor=C.+S.+Donaldauthor=L.+J.+L.+Feronauthor=H.+Gingellauthor=S.+C.+Glossopauthor=B.+R.+Hayterauthor=S.+Hussainauthor=G.+Karoutchiauthor=S.+G.+Lamontauthor=P.+MacFaulauthor=T.+A.+Mossauthor=S.+E.+Pearsonauthor=M.+Tongeauthor=G.+E.+Walkerauthor=H.+M.+Weirauthor=Z.+Wilson&title=Optimization+of+a+novel+binding+motif+to+%28E%29-3-%283%2C5-difluoro-4-%28%281R%2C3R%29-2-%282-fluoro-2-methylpropyl%29-3-methyl-2%2C3%2C4%2C9-tetrahydro-1H-pyrido%5B3%2C4-b%5Dindol-1-yl%29phenyl%29acrylic+Acid+%28AZD9496%29%2C+a+potent+and+orally+bioavailable+selective+estrogen+receptor+downregulator+and+antagonist&doi=10.1021%2Facs.jmedchem.5b00984"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Optimization of a Novel Binding Motif to (E)-3-(3,5-Difluoro-4-((1R,3R)-2-(2-fluoro-2-methylpropyl)-3-methyl-2,3,4,9-tetrahydro-1H-pyrido[3,4-b]indol-1-yl)phenyl)acrylic Acid (AZD9496), a Potent and Orally Bioavailable Selective Estrogen Receptor Downregulator and Antagonist</span></div><div class="casAuthors">De Savi, Chris; Bradbury, Robert H.; Rabow, Alfred A.; Norman, Richard A.; de Almeida, Camila; Andrews, David M.; Ballard, Peter; Buttar, David; Callis, Rowena J.; Currie, Gordon S.; Curwen, Jon O.; Davies, Chris D.; Donald, Craig S.; Feron, Lyman J. L.; Gingell, Helen; Glossop, Steven C.; Hayter, Barry R.; Hussain, Syeed; Karoutchi, Galith; Lamont, Scott G.; MacFaul, Philip; Moss, Thomas A.; Pearson, Stuart E.; Tonge, Michael; Walker, Graeme E.; Weir, Hazel M.; Wilson, Zena</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">8128-8140</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The discovery of an orally bioavailable selective estrogen receptor downregulator (SERD) with equiv. potency and preclin. pharmacol. to the i.m. SERD fulvestrant is described.  A directed screen identified the 1-aryl-2,3,4,9-tetrahydro-1H-pyrido[3,4-b]indole motif as a novel, druglike ER ligand.  Aided by crystal structures of novel ligands bound to an ER construct, medicinal chem. iterations led to (E)-3-(3,5-difluoro-4-((1R,3R)-2-(2-fluoro-2-methylpropyl)-3-methyl-2,3,4,9-tetrahydro-1H-pyrido[3,4-b]indol-1-yl)phenyl)acrylic acid (30b, AZD9496), a clin. candidate with high oral bioavailability across preclin. species that is currently being evaluated in phase I clin. trials for the treatment of advanced estrogen receptor (ER) pos. breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpP-ZYYxmMtg7Vg90H21EOLACvtfcHk0lih1I72veDa2Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFKiu7fK&md5=df7eee70ca783a2933cd494b53f71488</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00984&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00984%26sid%3Dliteratum%253Aachs%26aulast%3DDe%2BSavi%26aufirst%3DC.%26aulast%3DBradbury%26aufirst%3DR.%2BH.%26aulast%3DRabow%26aufirst%3DA.%2BA.%26aulast%3DNorman%26aufirst%3DR.%2BA.%26aulast%3Dde%2BAlmeida%26aufirst%3DC.%26aulast%3DAndrews%26aufirst%3DD.%2BM.%26aulast%3DBallard%26aufirst%3DP.%26aulast%3DButtar%26aufirst%3DD.%26aulast%3DCallis%26aufirst%3DR.%2BJ.%26aulast%3DCurrie%26aufirst%3DG.%2BS.%26aulast%3DCurwen%26aufirst%3DJ.%2BO.%26aulast%3DDavies%26aufirst%3DC.%2BD.%26aulast%3DDonald%26aufirst%3DC.%2BS.%26aulast%3DFeron%26aufirst%3DL.%2BJ.%2BL.%26aulast%3DGingell%26aufirst%3DH.%26aulast%3DGlossop%26aufirst%3DS.%2BC.%26aulast%3DHayter%26aufirst%3DB.%2BR.%26aulast%3DHussain%26aufirst%3DS.%26aulast%3DKaroutchi%26aufirst%3DG.%26aulast%3DLamont%26aufirst%3DS.%2BG.%26aulast%3DMacFaul%26aufirst%3DP.%26aulast%3DMoss%26aufirst%3DT.%2BA.%26aulast%3DPearson%26aufirst%3DS.%2BE.%26aulast%3DTonge%26aufirst%3DM.%26aulast%3DWalker%26aufirst%3DG.%2BE.%26aulast%3DWeir%26aufirst%3DH.%2BM.%26aulast%3DWilson%26aufirst%3DZ.%26atitle%3DOptimization%2520of%2520a%2520novel%2520binding%2520motif%2520to%2520%2528E%2529-3-%25283%252C5-difluoro-4-%2528%25281R%252C3R%2529-2-%25282-fluoro-2-methylpropyl%2529-3-methyl-2%252C3%252C4%252C9-tetrahydro-1H-pyrido%255B3%252C4-b%255Dindol-1-yl%2529phenyl%2529acrylic%2520Acid%2520%2528AZD9496%2529%252C%2520a%2520potent%2520and%2520orally%2520bioavailable%2520selective%2520estrogen%2520receptor%2520downregulator%2520and%2520antagonist%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D8128%26epage%3D8140%26doi%3D10.1021%2Facs.jmedchem.5b00984" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tria, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abrams, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baird, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burks, H. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Firestone, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaither, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamann, L. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirby, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lombardo, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macchi, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDonnell, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mishina, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norris, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nunez, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Springer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomsen, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tiong-Yip, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peukert, S.</span></span> <span> </span><span class="NLM_article-title">Discovery of LSZ102, a potent, orally bioavailable selective estrogen receptor degrader (SERD) for the treatment of estrogen receptor positive breast cancer</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">2837</span>– <span class="NLM_lpage">2864</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01682</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01682" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A528%3ADC%252BC1cXltFynurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=2837-2864&author=G.+S.+Triaauthor=T.+Abramsauthor=J.+Bairdauthor=H.+E.+Burksauthor=B.+Firestoneauthor=L.+A.+Gaitherauthor=L.+G.+Hamannauthor=G.+Heauthor=C.+A.+Kirbyauthor=S.+Kimauthor=F.+Lombardoauthor=K.+J.+Macchiauthor=D.+P.+McDonnellauthor=Y.+Mishinaauthor=J.+D.+Norrisauthor=J.+Nunezauthor=C.+Springerauthor=Y.+Sunauthor=N.+M.+Thomsenauthor=C.+Wangauthor=J.+Wangauthor=B.+Yuauthor=C.+L.+Tiong-Yipauthor=S.+Peukert&title=Discovery+of+LSZ102%2C+a+potent%2C+orally+bioavailable+selective+estrogen+receptor+degrader+%28SERD%29+for+the+treatment+of+estrogen+receptor+positive+breast+cancer&doi=10.1021%2Facs.jmedchem.7b01682"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of LSZ102, a Potent, Orally Bioavailable Selective Estrogen Receptor Degrader (SERD) for the Treatment of Estrogen Receptor Positive Breast Cancer</span></div><div class="casAuthors">Tria, George S.; Abrams, Tinya; Baird, Jason; Burks, Heather E.; Firestone, Brant; Gaither, L. Alex; Hamann, Lawrence G.; He, Guo; Kirby, Christina A.; Kim, Sunkyu; Lombardo, Franco; Macchi, Kaitlin J.; McDonnell, Donald P.; Mishina, Yuji; Norris, John D.; Nunez, Jill; Springer, Clayton; Sun, Yingchuan; Thomsen, Noel M.; Wang, Chunrong; Wang, Jianling; Yu, Bing; Tiong-Yip, Choi-Lai; Peukert, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2837-2864</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In breast cancer, estrogen receptor alpha (ERα) pos. cancer accounts for approx. 74% of all diagnoses, and in these settings, it is a primary driver of cell proliferation.  Treatment of ERα pos. breast cancer has long relied on endocrine therapies such as selective estrogen receptor modulators, aromatase inhibitors, and selective estrogen receptor degraders (SERDs).  The steroid-based anti-estrogen fulvestrant, the only approved SERD, is effective in patients who have not previously been treated with endocrine therapy as well as in patients who have progressed after receiving other endocrine therapies.  Its efficacy, however, may be limited due to its poor physicochem. properties.  THe authors describe the design and synthesis of a series of potent benzothiophene-contg. compds. that exhibit oral bioavailability and preclin. activity as SERDs.  This article culminates in the identification of LSZ102 (I), a compd. in clin. development for the treatment of ERα pos. breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcHgOrWwlRvrVg90H21EOLACvtfcHk0lih1I72veDa2Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXltFynurc%253D&md5=dba954f9f11121fae1bb50979428a83b</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01682&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01682%26sid%3Dliteratum%253Aachs%26aulast%3DTria%26aufirst%3DG.%2BS.%26aulast%3DAbrams%26aufirst%3DT.%26aulast%3DBaird%26aufirst%3DJ.%26aulast%3DBurks%26aufirst%3DH.%2BE.%26aulast%3DFirestone%26aufirst%3DB.%26aulast%3DGaither%26aufirst%3DL.%2BA.%26aulast%3DHamann%26aufirst%3DL.%2BG.%26aulast%3DHe%26aufirst%3DG.%26aulast%3DKirby%26aufirst%3DC.%2BA.%26aulast%3DKim%26aufirst%3DS.%26aulast%3DLombardo%26aufirst%3DF.%26aulast%3DMacchi%26aufirst%3DK.%2BJ.%26aulast%3DMcDonnell%26aufirst%3DD.%2BP.%26aulast%3DMishina%26aufirst%3DY.%26aulast%3DNorris%26aufirst%3DJ.%2BD.%26aulast%3DNunez%26aufirst%3DJ.%26aulast%3DSpringer%26aufirst%3DC.%26aulast%3DSun%26aufirst%3DY.%26aulast%3DThomsen%26aufirst%3DN.%2BM.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DYu%26aufirst%3DB.%26aulast%3DTiong-Yip%26aufirst%3DC.%2BL.%26aulast%3DPeukert%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520LSZ102%252C%2520a%2520potent%252C%2520orally%2520bioavailable%2520selective%2520estrogen%2520receptor%2520degrader%2520%2528SERD%2529%2520for%2520the%2520treatment%2520of%2520estrogen%2520receptor%2520positive%2520breast%2520cancer%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D2837%26epage%3D2864%26doi%3D10.1021%2Facs.jmedchem.7b01682" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDonnell, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norris, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willson, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greene, G. L.</span></span> <span> </span><span class="NLM_article-title">Structural basis for an unexpected mode of SERM-mediated ER antagonism</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">413</span>– <span class="NLM_lpage">424</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2005.04.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=10.1016%2Fj.molcel.2005.04.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=15893725" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A528%3ADC%252BD2MXkslaqu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2005&pages=413-424&author=Y.-L.+Wuauthor=X.+Yangauthor=Z.+Renauthor=D.+P.+McDonnellauthor=J.+D.+Norrisauthor=T.+M.+Willsonauthor=G.+L.+Greene&title=Structural+basis+for+an+unexpected+mode+of+SERM-mediated+ER+antagonism&doi=10.1016%2Fj.molcel.2005.04.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis for an unexpected mode of SERM-mediated ER antagonism</span></div><div class="casAuthors">Wu, Ya-Ling; Yang, Xiaojing; Ren, Zhong; McDonnell, Donald P.; Norris, John D.; Willson, Timothy M.; Greene, Geoffrey L.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">413-424</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Tamoxifen is effective for the prevention and treatment of estrogen-dependent breast cancers, but is assocd. with an increased incidence of endometrial tumors.  The authors report the crystal structure of the estrogen receptor α (ERα) ligand binding domain (LBD) bound to the structurally similar compd. GW5638, which has therapeutic potential and does not stimulate the uterus.  Like tamoxifen, GW5638 relocates the carboxy-terminal helix (H12) to the known coactivator-docking site in the ERα LBD.  However, GW5638 repositions residues in H12 through specific contacts with the N terminus of this helix.  In contrast to tamoxifen, the resulting increase in exposed hydrophobic surface of ERα LBD correlates with a significant destabilization of ERα in MCF-7 cells.  Thus, the GW5638-ERα LBD structure reveals an unexpected mode of SERM-mediated ER antagonism, in which the stability of ERα is decreased through an altered position of H12.  This dual mechanism of antagonism may explain why GW5638 can inhibit tamoxifen-resistant breast tumors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBuyhLNSdbQbVg90H21EOLACvtfcHk0lhTrjKD9QBuYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXkslaqu7s%253D&md5=c60e01b31b57c07b934da01a34efb255</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2005.04.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2005.04.014%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DY.-L.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DRen%26aufirst%3DZ.%26aulast%3DMcDonnell%26aufirst%3DD.%2BP.%26aulast%3DNorris%26aufirst%3DJ.%2BD.%26aulast%3DWillson%26aufirst%3DT.%2BM.%26aulast%3DGreene%26aufirst%3DG.%2BL.%26atitle%3DStructural%2520basis%2520for%2520an%2520unexpected%2520mode%2520of%2520SERM-mediated%2520ER%2520antagonism%26jtitle%3DMol.%2520Cell%26date%3D2005%26volume%3D18%26spage%3D413%26epage%3D424%26doi%3D10.1016%2Fj.molcel.2005.04.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hamilton, E. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Armstrong, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baird, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jhaveri, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoch, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klinowska, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindemann, J. P. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morgan, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schiavon, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weir, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Im, S. A.</span></span> <span> </span><span class="NLM_article-title">A first-in-human study of the new oral selective estrogen receptor degrader AZD9496 for ER(+)/HER2(−) advanced breast cancer</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">3510</span>– <span class="NLM_lpage">3518</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.CCR-17-3102</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=10.1158%2F1078-0432.CCR-17-3102" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=29440181" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsVWrtr3J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2018&pages=3510-3518&author=E.+P.+Hamiltonauthor=M.+R.+Patelauthor=A.+C.+Armstrongauthor=R.+D.+Bairdauthor=K.+Jhaveriauthor=M.+Hochauthor=T.+Klinowskaauthor=J.+P.+O.+Lindemannauthor=S.+R.+Morganauthor=G.+Schiavonauthor=H.+M.+Weirauthor=S.+A.+Im&title=A+first-in-human+study+of+the+new+oral+selective+estrogen+receptor+degrader+AZD9496+for+ER%28%2B%29%2FHER2%28%E2%88%92%29+advanced+breast+cancer&doi=10.1158%2F1078-0432.CCR-17-3102"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">A First-in-Human Study of the New Oral Selective Estrogen Receptor Degrader AZD9496 for ER+/HER2- Advanced Breast Cancer</span></div><div class="casAuthors">Hamilton, Erika P.; Patel, Manish R.; Armstrong, Anne C.; Baird, Richard D.; Jhaveri, Komal; Hoch, Matthias; Klinowska, Teresa; Lindemann, Justin P. O.; Morgan, Shethah R.; Schiavon, Gaia; Weir, Hazel M.; Im, Seock-Ah</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">3510-3518</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">AZD9496 is an oral nonsteroidal, small-mol. inhibitor of estrogen receptor alpha (ERα) and a potent and selective antagonist and degrader of ERα.  This first-in-human phase I study detd. the safety and tolerability of ascending doses of oral AZD9496 in women with estrogen receptor (ER)+/HER2- advanced breast cancer, characterized its pharmacokinetic (PK) profile, and made preliminary assessment of antitumor activity.  Forty-five patients received AZD9496 [20 mg once daily (QD) to 600 mg twice daily (BID)] in a dose-escalation, dose-expansion rolling 6 design.  Safety, tolerability, and PK activity in each cohort were reviewed before escalating to the next dose.  PK was detd. by mass spectrometry.  Adverse events (AEs) were graded according to the Common Terminol. Criteria for Adverse Events (CTCAE) v4.0.  Objective tumor response was evaluated by Response Evaluation Criteria in Solid Tumors (RECIST) v1.1.  Most common causally related AEs were diarrhea (35.6%), fatigue (31.1%), and nausea (22.2%), and seven patients had grade ≥3 AEs.  Three patients experienced a dose-limiting toxicity: one each at 150 mg BID (abnormal hepatic function), 400 mg BID (diarrhea and elevated liver function tests), and 600 mg BID (diarrhea), and all were reversible.  The max. tolerated dose was not reached.  Partial response was confirmed in one patient, who also had decreased tumor marker Ca15.3.  Four patients had stable disease at 12 mo follow-up.  AZD9496 is well tolerated with an acceptable safety profile, showing evidence of prolonged disease stabilization in heavily pretreated patients with ER+/HER2- advanced breast cancer.  Clin Cancer Res; 24(15); 3510-8. 2018 AACR.  See related commentary by Jordan, p.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrT2Esco_KTEbVg90H21EOLACvtfcHk0lhTrjKD9QBuYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsVWrtr3J&md5=6d299ace6b73c7693554a54789883d9c</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-17-3102&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-17-3102%26sid%3Dliteratum%253Aachs%26aulast%3DHamilton%26aufirst%3DE.%2BP.%26aulast%3DPatel%26aufirst%3DM.%2BR.%26aulast%3DArmstrong%26aufirst%3DA.%2BC.%26aulast%3DBaird%26aufirst%3DR.%2BD.%26aulast%3DJhaveri%26aufirst%3DK.%26aulast%3DHoch%26aufirst%3DM.%26aulast%3DKlinowska%26aufirst%3DT.%26aulast%3DLindemann%26aufirst%3DJ.%2BP.%2BO.%26aulast%3DMorgan%26aufirst%3DS.%2BR.%26aulast%3DSchiavon%26aufirst%3DG.%26aulast%3DWeir%26aufirst%3DH.%2BM.%26aulast%3DIm%26aufirst%3DS.%2BA.%26atitle%3DA%2520first-in-human%2520study%2520of%2520the%2520new%2520oral%2520selective%2520estrogen%2520receptor%2520degrader%2520AZD9496%2520for%2520ER%2528%252B%2529%252FHER2%2528%25E2%2588%2592%2529%2520advanced%2520breast%2520cancer%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2018%26volume%3D24%26spage%3D3510%26epage%3D3518%26doi%3D10.1158%2F1078-0432.CCR-17-3102" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Weir, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradbury, R. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rabow, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buttar, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Callis, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curwen, J. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Almeida, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ballard, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hulse, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donald, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feron, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karoutchi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacFaul, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moss, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norman, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pearson, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tonge, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, G. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rowlinson, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sadler, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richmond, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ladd, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pazolli, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mazzola, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">D’Cruz, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Savi, C.</span></span> <span> </span><span class="NLM_article-title">AZD9496: an oral estrogen receptor inhibitor that blocks the growth of ER-positive and ESR1-mutant breast tumors in preclinical models</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>76</i></span>,  <span class="NLM_fpage">3307</span>– <span class="NLM_lpage">3318</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-15-2357</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=10.1158%2F0008-5472.CAN-15-2357" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=27020862" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A528%3ADC%252BC28XovVyktLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=76&publication_year=2016&pages=3307-3318&author=H.+M.+Weirauthor=R.+H.+Bradburyauthor=M.+Lawsonauthor=A.+A.+Rabowauthor=D.+Buttarauthor=R.+J.+Callisauthor=J.+O.+Curwenauthor=C.+de+Almeidaauthor=P.+Ballardauthor=M.+Hulseauthor=C.+S.+Donaldauthor=L.+J.+Feronauthor=G.+Karoutchiauthor=P.+MacFaulauthor=T.+Mossauthor=R.+A.+Normanauthor=S.+E.+Pearsonauthor=M.+Tongeauthor=G.+Daviesauthor=G.+E.+Walkerauthor=Z.+Wilsonauthor=R.+Rowlinsonauthor=S.+Powellauthor=C.+Sadlerauthor=G.+Richmondauthor=B.+Laddauthor=E.+Pazolliauthor=A.+M.+Mazzolaauthor=C.+D%E2%80%99Cruzauthor=C.+De+Savi&title=AZD9496%3A+an+oral+estrogen+receptor+inhibitor+that+blocks+the+growth+of+ER-positive+and+ESR1-mutant+breast+tumors+in+preclinical+models&doi=10.1158%2F0008-5472.CAN-15-2357"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">AZD9496: An Oral Estrogen Receptor Inhibitor That Blocks the Growth of ER-Positive and ESR1-Mutant Breast Tumors in Preclinical Models</span></div><div class="casAuthors">Weir, Hazel M.; Bradbury, Robert H.; Lawson, Mandy; Rabow, Alfred A.; Buttar, David; Callis, Rowena J.; Curwen, Jon O.; de Almeida, Camila; Ballard, Peter; Hulse, Michael; Donald, Craig S.; Feron, Lyman J. L.; Karoutchi, Galith; MacFaul, Philip; Moss, Thomas; Norman, Richard A.; Pearson, Stuart E.; Tonge, Michael; Davies, Gareth; Walker, Graeme E.; Wilson, Zena; Rowlinson, Rachel; Powell, Steve; Sadler, Claire; Richmond, Graham; Ladd, Brendon; Pazolli, Ermira; Mazzola, Anne Marie; D'Cruz, Celina; De Savi, Chris</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">76</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">3307-3318</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Fulvestrant is an estrogen receptor (ER) antagonist administered to breast cancer patients by monthly i.m. injection.  Given its present limitations of dosing and route of administration, a more flexible orally available compd. has been sought to pursue the potential benefits of this drug in patients with advanced metastatic disease.  Here we report the identification and characterization of AZD9496, a nonsteroidal small-mol. inhibitor of ERα, which is a potent and selective antagonist and downregulator of ERα in vitro and in vivo in ER-pos. models of breast cancer.  Significant tumor growth inhibition was obsd. as low as 0.5 mg/kg dose in the estrogen-dependent MCF-7 xenograft model, where this effect was accompanied by a dose-dependent decrease in PR protein levels, demonstrating potent antagonist activity.  Combining AZD9496 with PI3K pathway and CDK4/6 inhibitors led to further growth-inhibitory effects compared with monotherapy alone.  Tumor regressions were also seen in a long-term estrogen-deprived breast model, where significant downregulation of ERα protein was obsd.  AZD9496 bound and downregulated clin. relevant ESR1 mutants in vitro and inhibited tumor growth in an ESR1-mutant patient-derived xenograft model that included a D538G mutation.  Collectively, the pharmacol. evidence showed that AZD9496 is an oral, nonsteroidal, selective estrogen receptor antagonist and downregulator in ER+ breast cells that could provide meaningful benefit to ER+ breast cancer patients.  AZD9496 is currently being evaluated in a phase I clin. trial.  Cancer Res; 76(11); 3307-18. ©2016 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoP4X3e2iYdcLVg90H21EOLACvtfcHk0lhTrjKD9QBuYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XovVyktLo%253D&md5=4ff8ab9a9cd379acc031b7c6fd27134c</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-15-2357&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-15-2357%26sid%3Dliteratum%253Aachs%26aulast%3DWeir%26aufirst%3DH.%2BM.%26aulast%3DBradbury%26aufirst%3DR.%2BH.%26aulast%3DLawson%26aufirst%3DM.%26aulast%3DRabow%26aufirst%3DA.%2BA.%26aulast%3DButtar%26aufirst%3DD.%26aulast%3DCallis%26aufirst%3DR.%2BJ.%26aulast%3DCurwen%26aufirst%3DJ.%2BO.%26aulast%3Dde%2BAlmeida%26aufirst%3DC.%26aulast%3DBallard%26aufirst%3DP.%26aulast%3DHulse%26aufirst%3DM.%26aulast%3DDonald%26aufirst%3DC.%2BS.%26aulast%3DFeron%26aufirst%3DL.%2BJ.%26aulast%3DKaroutchi%26aufirst%3DG.%26aulast%3DMacFaul%26aufirst%3DP.%26aulast%3DMoss%26aufirst%3DT.%26aulast%3DNorman%26aufirst%3DR.%2BA.%26aulast%3DPearson%26aufirst%3DS.%2BE.%26aulast%3DTonge%26aufirst%3DM.%26aulast%3DDavies%26aufirst%3DG.%26aulast%3DWalker%26aufirst%3DG.%2BE.%26aulast%3DWilson%26aufirst%3DZ.%26aulast%3DRowlinson%26aufirst%3DR.%26aulast%3DPowell%26aufirst%3DS.%26aulast%3DSadler%26aufirst%3DC.%26aulast%3DRichmond%26aufirst%3DG.%26aulast%3DLadd%26aufirst%3DB.%26aulast%3DPazolli%26aufirst%3DE.%26aulast%3DMazzola%26aufirst%3DA.%2BM.%26aulast%3DD%25E2%2580%2599Cruz%26aufirst%3DC.%26aulast%3DDe%2BSavi%26aufirst%3DC.%26atitle%3DAZD9496%253A%2520an%2520oral%2520estrogen%2520receptor%2520inhibitor%2520that%2520blocks%2520the%2520growth%2520of%2520ER-positive%2520and%2520ESR1-mutant%2520breast%2520tumors%2520in%2520preclinical%2520models%26jtitle%3DCancer%2520Res.%26date%3D2016%26volume%3D76%26spage%3D3307%26epage%3D3318%26doi%3D10.1158%2F0008-5472.CAN-15-2357" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shiau, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barstad, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loria, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kushner, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agard, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greene, G. L.</span></span> <span> </span><span class="NLM_article-title">The structural basis of estrogen receptor/coactivator recognition and the antagonism of this interaction by tamoxifen</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>95</i></span>,  <span class="NLM_fpage">927</span>– <span class="NLM_lpage">937</span>, <span class="refDoi"> DOI: 10.1016/S0092-8674(00)81717-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=10.1016%2FS0092-8674%2800%2981717-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=9875847" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A528%3ADyaK1MXjslyltQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=95&publication_year=1998&pages=927-937&author=A.+K.+Shiauauthor=D.+Barstadauthor=P.+M.+Loriaauthor=L.+Chengauthor=P.+J.+Kushnerauthor=D.+A.+Agardauthor=G.+L.+Greene&title=The+structural+basis+of+estrogen+receptor%2Fcoactivator+recognition+and+the+antagonism+of+this+interaction+by+tamoxifen&doi=10.1016%2FS0092-8674%2800%2981717-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">The structural basis of estrogen receptor/coactivator recognition and the antagonism of this interaction by tamoxifen</span></div><div class="casAuthors">Shiau, Andrew K.; Barstad, Danielle; Loria, Paula M.; Cheng, Lin; Kushner, Peter J.; Agard, David A.; Greene, Geoffrey L.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, Massachusetts)</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">95</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">927-937</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Ligand-dependent activation of transcription by nuclear receptors (NRs) is mediated by interactions with coactivators.  Receptor agonists promote coactivator binding, and antagonists block coactivator binding.  Here we report the crystal structure of the human estrogen receptor α (hERα) ligand-binding domain (LBD) bound to both the agonist diethylstilbestrol (DES) and a peptide derived from the NR box II region of the coactivator GRIP1 and the crystal structure of the hERα LBD bound to the selective antagonist 4-hydroxytamoxifen (OHT).  In the DES-LBD-peptide complex, the peptide binds as a short α helix to a hydrophobic groove on the surface of the LBD.  In the OHT-LBD complex, helix 12 occludes the coactivator recognition groove by mimicking the interactions of the NR box peptide with the LBD.  These structures reveal the two distinct mechanisms by which structural features of OHT promote this "autoinhibitory" helix 12 conformation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrrjqf40oHhE7Vg90H21EOLACvtfcHk0ljm2fvKhF6oFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXjslyltQ%253D%253D&md5=074d616cf353f0d98cea5b26c4595c10</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2FS0092-8674%2800%2981717-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0092-8674%252800%252981717-1%26sid%3Dliteratum%253Aachs%26aulast%3DShiau%26aufirst%3DA.%2BK.%26aulast%3DBarstad%26aufirst%3DD.%26aulast%3DLoria%26aufirst%3DP.%2BM.%26aulast%3DCheng%26aufirst%3DL.%26aulast%3DKushner%26aufirst%3DP.%2BJ.%26aulast%3DAgard%26aufirst%3DD.%2BA.%26aulast%3DGreene%26aufirst%3DG.%2BL.%26atitle%3DThe%2520structural%2520basis%2520of%2520estrogen%2520receptor%252Fcoactivator%2520recognition%2520and%2520the%2520antagonism%2520of%2520this%2520interaction%2520by%2520tamoxifen%26jtitle%3DCell%26date%3D1998%26volume%3D95%26spage%3D927%26epage%3D937%26doi%3D10.1016%2FS0092-8674%2800%2981717-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Barillari, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marcou, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rognan, D.</span></span> <span> </span><span class="NLM_article-title">Hot-spots-guided receptor-based pharmacophores (HS-Pharm): a knowledge-based approach to identify ligand-anchoring atoms in protein cavities and prioritize structure-based pharmacophores</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">1396</span>– <span class="NLM_lpage">1410</span>, <span class="refDoi"> DOI: 10.1021/ci800064z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci800064z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A528%3ADC%252BD1cXnsFGktrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2008&pages=1396-1410&author=C.+Barillariauthor=G.+Marcouauthor=D.+Rognan&title=Hot-spots-guided+receptor-based+pharmacophores+%28HS-Pharm%29%3A+a+knowledge-based+approach+to+identify+ligand-anchoring+atoms+in+protein+cavities+and+prioritize+structure-based+pharmacophores&doi=10.1021%2Fci800064z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Hot-Spots-Guided Receptor-Based Pharmacophores (HS-Pharm): A Knowledge-Based Approach to Identify Ligand-Anchoring Atoms in Protein Cavities and Prioritize Structure-Based Pharmacophores</span></div><div class="casAuthors">Barillari, Caterina; Marcou, Gilles; Rognan, Didier</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1396-1410</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The design of biol. active compds. from ligand-free protein structures using a structure-based approach is still a major challenge.  In this paper, we present a fast knowledge-based approach (HS-Pharm) that allows the prioritization of cavity atoms that should be targeted for ligand binding, by training machine learning algorithms with atom-based fingerprints of known ligand-binding pockets.  The knowledge of hot spots for ligand binding is here used for focusing structure-based pharmacophore models.  Three targets of pharmacol. interest (neuraminidase, β2 adrenergic receptor, and cyclooxygenase-2) were used to test the evaluated methodol., and the derived structure-based pharmacophores were used in retrospective virtual screening studies.  The current study shows that structure-based pharmacophore screening is a powerful technique for the fast identification of potential hits in a chem. library, and that it is a valid alternative to virtual screening by mol. docking.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq1wuGUqc3VZbVg90H21EOLACvtfcHk0ljm2fvKhF6oFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXnsFGktrg%253D&md5=75c0bc3d0e89f62f8382d3ff7e85fecc</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Fci800064z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci800064z%26sid%3Dliteratum%253Aachs%26aulast%3DBarillari%26aufirst%3DC.%26aulast%3DMarcou%26aufirst%3DG.%26aulast%3DRognan%26aufirst%3DD.%26atitle%3DHot-spots-guided%2520receptor-based%2520pharmacophores%2520%2528HS-Pharm%2529%253A%2520a%2520knowledge-based%2520approach%2520to%2520identify%2520ligand-anchoring%2520atoms%2520in%2520protein%2520cavities%2520and%2520prioritize%2520structure-based%2520pharmacophores%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2008%26volume%3D48%26spage%3D1396%26epage%3D1410%26doi%3D10.1021%2Fci800064z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gohlke, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hendlich, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klebe, G.</span></span> <span> </span><span class="NLM_article-title">Predicting binding modes, binding affinities and ‘hot spots’ for protein-ligand complexes using a knowledge-based scoring function</span>. <i>Perspect. Drug Discovery Des.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">115</span>– <span class="NLM_lpage">144</span>, <span class="refDoi"> DOI: 10.1023/A:1008781006867</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=10.1023%2FA%3A1008781006867" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A528%3ADC%252BD3MXptV2q" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2000&pages=115-144&author=H.+Gohlkeauthor=M.+Hendlichauthor=G.+Klebe&title=Predicting+binding+modes%2C+binding+affinities+and+%E2%80%98hot+spots%E2%80%99+for+protein-ligand+complexes+using+a+knowledge-based+scoring+function&doi=10.1023%2FA%3A1008781006867"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Predicting binding modes, binding affinities and "hot spots" for protein-ligand complexes using a knowledge-based scoring function</span></div><div class="casAuthors">Gohlke, Holger; Hendlich, Manfred; Klebe, Gerhard</div><div class="citationInfo"><span class="NLM_cas:title">Perspectives in Drug Discovery and Design</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">115-144</span>CODEN:
                <span class="NLM_cas:coden">PDDDEC</span>;
        ISSN:<span class="NLM_cas:issn">0928-2866</span>.
    
            (<span class="NLM_cas:orgname">Kluwer Academic Publishers</span>)
        </div><div class="casAbstract">The development of a new knowledge-based scoring function (DrugScore) and its power to recognize binding modes close to expt., to predict binding affinities, and to identify "hot spots" in binding pockets is presented.  Structural information is extd. from crystallog. detd. protein-ligand complexes using ReLiBase and converted into distance-dependent pair-preferences and solvent-accessible surface (SAS) dependent singlet preferences of protein and ligand atoms.  The sum of the pair preferences and the singlet preferences is calcd. using the 3D structure of protein-ligand complexes either taken directly from the x-ray structure or generated by the docking tool FlexX.  DrugScore discriminates efficiently between well-docked ligand binding modes (root-mean-square deviation <2.0 Å with respect to a crystallog. detd. ref. complex) and computer-generated ones largely deviating from the native structure.  For two test sets (91 and 68 protein-ligand complexes, taken from the PDB) the calcd. score recognizes poses deviating <2 Å from the crystal structure on rank 1 in three quarters of all possible cases.  Compared to the scoring function in FlexX, this is a substantial improvement.  For five test sets of crystallog. detd. protein-ligand complexes as well as for two sets of ligand geometries generated by FlexX, the calcd. score is correlated with exptl. detd. binding affinities.  For a set of 16 crystallog. detd. serine protease inhibitor complexes, a R2 value of 0.86 and a std. deviation of 0.95 log units is achieved as best result; for a set of 64 thrombin and trypsin inhibitors docked into their target proteins, a R2 value of 0.48 and a std. deviation of 0.7 log units is calcd.  DrugScore performs better than other state-of-the-art scoring functions.  To assess DrugScore's capability to reproduce the geometry of directional interactions correctly, "hot spots" are identified and visualized in terms of isocontour surfaces inside the binding pocket.  A data set of 159 x-ray protein-ligand complexes is used to reproduce and highlight the actually obsd. ligand atom positions.  In 74% of all cases, the actually obsd. atom type corresponds to an atom type predicted by the most favorable score at the nearest grid point.  The prediction rate increases to 85% if at least an atom type of the same class of interaction is suggested.  DrugScore is fast to compute and includes implicitly solvation and entropy contributions.  Small deviations in the 3D structure are tolerated and, since only contacts to non-hydrogen atoms are regarded, it does not require any assumptions on protonation states.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhQrM1Fu-n5bVg90H21EOLACvtfcHk0ljm2fvKhF6oFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXptV2q&md5=4f330d644753d3729ad11d5ee5fa4f79</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1023%2FA%3A1008781006867&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FA%253A1008781006867%26sid%3Dliteratum%253Aachs%26aulast%3DGohlke%26aufirst%3DH.%26aulast%3DHendlich%26aufirst%3DM.%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3DPredicting%2520binding%2520modes%252C%2520binding%2520affinities%2520and%2520%25E2%2580%2598hot%2520spots%25E2%2580%2599%2520for%2520protein-ligand%2520complexes%2520using%2520a%2520knowledge-based%2520scoring%2520function%26jtitle%3DPerspect.%2520Drug%2520Discovery%2520Des.%26date%3D2000%26volume%3D20%26spage%3D115%26epage%3D144%26doi%3D10.1023%2FA%3A1008781006867" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Burgoyne, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jackson, R. M.</span></span> <span> </span><span class="NLM_article-title">Predicting protein interaction sites: binding hot-spots in protein-protein and protein-ligand interfaces</span>. <i>Bioinformatics</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">1335</span>– <span class="NLM_lpage">1342</span>, <span class="refDoi"> DOI: 10.1093/bioinformatics/btl079</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=10.1093%2Fbioinformatics%2Fbtl079" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=16522669" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A528%3ADC%252BD28XmtlOjsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2006&pages=1335-1342&author=N.+J.+Burgoyneauthor=R.+M.+Jackson&title=Predicting+protein+interaction+sites%3A+binding+hot-spots+in+protein-protein+and+protein-ligand+interfaces&doi=10.1093%2Fbioinformatics%2Fbtl079"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Predicting protein interaction sites: binding hot-spots in protein-protein and protein-ligand interfaces</span></div><div class="casAuthors">Burgoyne, Nicholas J.; Jackson, Richard M.</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1335-1342</span>CODEN:
                <span class="NLM_cas:coden">BOINFP</span>;
        ISSN:<span class="NLM_cas:issn">1367-4803</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Protein assemblies are currently poorly represented in structural databases and their structural elucidation is a key goal in biol.  Here we analyze clefts in protein surfaces, likely to correspond to binding 'hot-spots', and rank them according to sequence conservation and simple measures of phys. properties including hydrophobicity, desolvation, electrostatic and van der Waals potentials, to predict which are involved in binding in the native complex.  The resulting differences between predicting binding-sites at protein-protein and protein-ligand interfaces are striking.  There is a high level of prediction accuracy (≤93%) for protein-ligand interactions, based on the following attributes: van der Waals potential, electrostatic potential, desolvation and surface conservation.  Generally, the prediction accuracy for protein-protein interactions is lower, with the exception of enzymes.  Our results show that the ease of cleft desolvation is strongly predictive of interfaces and strongly maintained across all classes of protein-binding interface.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqD2TRwsq1iIbVg90H21EOLACvtfcHk0ljSWiCzEKMRtA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XmtlOjsb8%253D&md5=11cd0c8978e82ace530d45cb28449de8</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtl079&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtl079%26sid%3Dliteratum%253Aachs%26aulast%3DBurgoyne%26aufirst%3DN.%2BJ.%26aulast%3DJackson%26aufirst%3DR.%2BM.%26atitle%3DPredicting%2520protein%2520interaction%2520sites%253A%2520binding%2520hot-spots%2520in%2520protein-protein%2520and%2520protein-ligand%2520interfaces%26jtitle%3DBioinformatics%26date%3D2006%26volume%3D22%26spage%3D1335%26epage%3D1342%26doi%3D10.1093%2Fbioinformatics%2Fbtl079" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pei, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, L.</span></span> <span> </span><span class="NLM_article-title">Protein-protein interface analysis and hot spots identification for chemical ligand design</span>. <i>Curr. Pharm. Des.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">1192</span>– <span class="NLM_lpage">1200</span>, <span class="refDoi"> DOI: 10.2174/13816128113199990065</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=10.2174%2F13816128113199990065" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=23713772" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A528%3ADC%252BC2cXksFSgs7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2014&pages=1192-1200&author=J.+Chenauthor=X.+Maauthor=Y.+Yuanauthor=J.+Peiauthor=L.+Lai&title=Protein-protein+interface+analysis+and+hot+spots+identification+for+chemical+ligand+design&doi=10.2174%2F13816128113199990065"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Protein-Protein Interface Analysis and Hot Spots Identification for Chemical Ligand Design</span></div><div class="casAuthors">Chen, Jing; Ma, Xiaomin; Yuan, Yaxia; Pei, Jianfeng; Lai, Luhua</div><div class="citationInfo"><span class="NLM_cas:title">Current Pharmaceutical Design</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1192-1200</span>CODEN:
                <span class="NLM_cas:coden">CPDEFP</span>;
        ISSN:<span class="NLM_cas:issn">1381-6128</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Rational design for chem. compds. targeting protein-protein interactions has grown from a dream to reality after a decade of efforts.  There are an increasing no. of successful examples, though major challenges remain in the field.  In this paper, we will first give a brief review of the available methods that can be used to analyze protein-protein interface and predict hot spots for chem. ligand design.  New developments of binding sites detection, ligandability and hot spots prediction from the author's group will also be described.  Pocket V.3 is an improved program for identifying hot spots in protein-protein interface using only an apo protein structure.  It has been developed based on Pocket V.2 that can derive receptor-based pharmacophore model for ligand binding cavity.  Given similarities and differences between the essence of pharmacophore and hot spots for guiding design of chem. compds., not only energetic but also spatial properties of protein-protein interface are used in Pocket V.3 for dealing with protein-protein interface.  In order to illustrate the capability of Pocket V.3, two datasets have been used.  One is taken from ASEdb and BID having exptl. alanine scanning results for testing hot spots prediction.  The other is taken from the 2P2I database contg. complex structures of protein-ligand binding at the original protein-protein interface for testing hot spots application in ligand design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoeQadx7vaGVbVg90H21EOLACvtfcHk0ljSWiCzEKMRtA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXksFSgs7c%253D&md5=7d51f3f146bcba0d7b0ec56ebb289756</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.2174%2F13816128113199990065&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F13816128113199990065%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DMa%26aufirst%3DX.%26aulast%3DYuan%26aufirst%3DY.%26aulast%3DPei%26aufirst%3DJ.%26aulast%3DLai%26aufirst%3DL.%26atitle%3DProtein-protein%2520interface%2520analysis%2520and%2520hot%2520spots%2520identification%2520for%2520chemical%2520ligand%2520design%26jtitle%3DCurr.%2520Pharm.%2520Des.%26date%3D2014%26volume%3D20%26spage%3D1192%26epage%3D1200%26doi%3D10.2174%2F13816128113199990065" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rao, B. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, U. C.</span></span> <span> </span><span class="NLM_article-title">A free energy perturbation study of solvation in methanol and dimethyl sulfoxide</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">1990</span>,  <span class="NLM_volume"><i>112</i></span>,  <span class="NLM_fpage">3803</span>– <span class="NLM_lpage">3811</span>, <span class="refDoi"> DOI: 10.1021/ja00166a014</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja00166a014" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A528%3ADyaK3cXitFGrsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=112&publication_year=1990&pages=3803-3811&author=B.+G.+Raoauthor=U.+C.+Singh&title=A+free+energy+perturbation+study+of+solvation+in+methanol+and+dimethyl+sulfoxide&doi=10.1021%2Fja00166a014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">A free energy perturbation study of solvation in methanol and dimethyl sulfoxide</span></div><div class="casAuthors">Rao, B. G.; Singh, U. Chandra</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1990</span>),
    <span class="NLM_cas:volume">112</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3803-11</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    </div><div class="casAbstract">Relative differences in free energies of solvation, ΔG, between two different solutes in water, methanol, and DMSO have been detd. by a coordinate coupled free energy perturbation method.  The solutes examd. in this study are some closed shell ions, tetraalkylammonium ions, normal alkanes, and tetraalkylmethane mols.  The ΔG varies differently in different solvents for the mutation of a solute into another solute.  These differences are due to the differences in the solvation processes in different solvents.  For closed shell ions, the variation of ΔG shows greater sensitivity to change in the size of an anion in protic solvents than in dipolar aprotic solvent, while it is almost equally affected by the change in the size of a cation in all the three solvents.  The structural results of pure liq. and ionic solns. of DMSO obtained by the mol. dynamics simulations are discussed to rationalize the shape of the ΔG curve for mutations of the closed shell ions in DMSO.  In the case of large tetraalkylammonium ions and alkanes, the variation of ΔG in water is dictated by both the structure of the tight solvent shell around these solutes and the solute-solvent interaction energies.  On the other hand, the variation of ΔG for these solutes in methanol and DMSO are detd. mainly by the solute-solvent interaction energies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGruwwgK6SvhL7Vg90H21EOLACvtfcHk0ljx-q8g1w6nTg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3cXitFGrsb4%253D&md5=442ab2765b2ac287e7723a17087370dc</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Fja00166a014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja00166a014%26sid%3Dliteratum%253Aachs%26aulast%3DRao%26aufirst%3DB.%2BG.%26aulast%3DSingh%26aufirst%3DU.%2BC.%26atitle%3DA%2520free%2520energy%2520perturbation%2520study%2520of%2520solvation%2520in%2520methanol%2520and%2520dimethyl%2520sulfoxide%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1990%26volume%3D112%26spage%3D3803%26epage%3D3811%26doi%3D10.1021%2Fja00166a014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kollman, P.</span></span> <span> </span><span class="NLM_article-title">Free energy calculations: applications to chemical and biochemical phenomena</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>93</i></span>,  <span class="NLM_fpage">2395</span>– <span class="NLM_lpage">2417</span>, <span class="refDoi"> DOI: 10.1021/cr00023a004</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr00023a004" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A528%3ADyaK3sXmt1Sktr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=93&publication_year=1993&pages=2395-2417&author=P.+Kollman&title=Free+energy+calculations%3A+applications+to+chemical+and+biochemical+phenomena&doi=10.1021%2Fcr00023a004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Free energy calculations: Applications to chemical and biochemical phenomena</span></div><div class="casAuthors">Kollman, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">93</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2395-417</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    </div><div class="casAbstract">A review with 252 refs. about applications of free energy calcns. employing mol. dynamics or Monte Carlo methods to a variety of chem. and biochem. phenomena.  The focus is on applications of such calcns. to mol. solvation, mol. assocn., macromol. stability, and enzyme catalysis.  The mols. discussed range from monovalent ions and small mols. to proteins and nucleic acids.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrN1w4SyFDqCLVg90H21EOLACvtfcHk0ljx-q8g1w6nTg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXmt1Sktr0%253D&md5=f12326b24734ea092996cf22efb6ebd6</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fcr00023a004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr00023a004%26sid%3Dliteratum%253Aachs%26aulast%3DKollman%26aufirst%3DP.%26atitle%3DFree%2520energy%2520calculations%253A%2520applications%2520to%2520chemical%2520and%2520biochemical%2520phenomena%26jtitle%3DChem.%2520Rev.%26date%3D1993%26volume%3D93%26spage%3D2395%26epage%3D2417%26doi%3D10.1021%2Fcr00023a004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jorgensen, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, L. L.</span></span> <span> </span><span class="NLM_article-title">Perspective on free-energy perturbation calculations for chemical equilibria</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">869</span>– <span class="NLM_lpage">876</span>, <span class="refDoi"> DOI: 10.1021/ct800011m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct800011m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A528%3ADC%252BD1cXlslOisLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2008&pages=869-876&author=W.+L.+Jorgensenauthor=L.+L.+Thomas&title=Perspective+on+free-energy+perturbation+calculations+for+chemical+equilibria&doi=10.1021%2Fct800011m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Perspective on Free-Energy Perturbation Calculations for Chemical Equilibria</span></div><div class="casAuthors">Jorgensen, William L.; Thomas, Laura L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">869-876</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">An overview has been provided on the history of free-energy perturbation (FEP) calcns. and their application to computation of relative free energies of solvation.  In view of the expanding role of FEP calcns. for pursuing a wide range of problems in chem. and biochem., including mol. design, it is important to establish reliable protocols for their execution.  For 30 interconversions of substituted benzenes, which represent prototypical cases for mol. design, alternative sampling procedures and nos. of FEP windows were considered.  Simple overlap sampling with 11 or 15 windows and double-wide sampling with 14 windows emerged as viable procedures for such routine FEP calcns.; the present protocols with these sampling methods yield statistical uncertainties of 0.2 kcal/mol for free-energy differences.  The slightly greater accuracy obtainable with 21 to 25 windows does not clearly justify the increased costs, while overlap sampling with 6 or 8 windows may be useful for preliminary exploration of large nos. of mol. changes.  Application of the power-series expansion of the Zwanzig equation was also tested.  Use of the first-order term is generally reliable, while inclusion of the slowly convergent, second-order fluctuation term causes deterioration in the results for strongly hydrogen-bonded solutes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqukDgvYtLny7Vg90H21EOLACvtfcHk0likuiy81qz5MA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXlslOisLw%253D&md5=4f6cd47142dbbcd51e94aedd2d878538</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Fct800011m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct800011m%26sid%3Dliteratum%253Aachs%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26aulast%3DThomas%26aufirst%3DL.%2BL.%26atitle%3DPerspective%2520on%2520free-energy%2520perturbation%2520calculations%2520for%2520chemical%2520equilibria%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2008%26volume%3D4%26spage%3D869%26epage%3D876%26doi%3D10.1021%2Fct800011m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zacharias, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Straatsma, T. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCammon, J. A.</span></span> <span> </span><span class="NLM_article-title">Separation-shifted scaling, a new scaling method for Lennard-Jones interactions in thermodynamic integration</span>. <i>J. Chem. Phys.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>100</i></span>,  <span class="NLM_fpage">9025</span>– <span class="NLM_lpage">9031</span>, <span class="refDoi"> DOI: 10.1063/1.466707</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=10.1063%2F1.466707" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A528%3ADyaK2cXltFyrsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=100&publication_year=1994&pages=9025-9031&author=M.+Zachariasauthor=T.+P.+Straatsmaauthor=J.+A.+McCammon&title=Separation-shifted+scaling%2C+a+new+scaling+method+for+Lennard-Jones+interactions+in+thermodynamic+integration&doi=10.1063%2F1.466707"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Separation-shifted scaling, a new scaling method for Lennard-Jones interactions in thermodynamic integration</span></div><div class="casAuthors">Zacharias, M.; Straatsma, T. P.; McCammon, J. A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">100</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">9025-31</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    </div><div class="casAbstract">A new method of simultaneously scaling and shifting the Lennard-Jones (LJ) potential in mol. dynamics (MD) and thermodn. integration (TI) simulations is presented.  The approach allows the smooth creation or annihilation of atoms or mols. in an ensemble of solvent mols. during a mol. simulation.  By scaling and shifting the LJ potential in the direction of the interat. distance between particles, the method eliminates the problem of the creation or annihilation of a large repulsive LJ potential at the initial or final state of a TI.  The optimal degree of shifting and scaling the LJ potential as a function of a control variable λ was studied for the annihilation and creation of neon in aq. soln.  The method was further tested on the calcn. of the free energy of aq. solvation of a small mol., ethanol.  In contrast to linear scaling of the LJ potential during TI, the calcd. free energies using the new sepn.-shifted scaling approach are reasonably well converged after 200-500 ps of simulation and show smaller hysteresis comparing forward and reverse TI.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp749QLLl9dubVg90H21EOLACvtfcHk0likuiy81qz5MA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXltFyrsLw%253D&md5=82654e60072c5a954188276cea3b468a</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1063%2F1.466707&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.466707%26sid%3Dliteratum%253Aachs%26aulast%3DZacharias%26aufirst%3DM.%26aulast%3DStraatsma%26aufirst%3DT.%2BP.%26aulast%3DMcCammon%26aufirst%3DJ.%2BA.%26atitle%3DSeparation-shifted%2520scaling%252C%2520a%2520new%2520scaling%2520method%2520for%2520Lennard-Jones%2520interactions%2520in%2520thermodynamic%2520integration%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1994%26volume%3D100%26spage%3D9025%26epage%3D9031%26doi%3D10.1063%2F1.466707" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kollman, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Massova, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reyes, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuhn, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chong, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Calculating structures and free energies of complex molecules: combining molecular mechanics and continuum models</span>. <i>Acc. Chem. Res.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">889</span>– <span class="NLM_lpage">897</span>, <span class="refDoi"> DOI: 10.1021/ar000033j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ar000033j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A528%3ADC%252BD3cXmvFGiu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2000&pages=889-897&author=P.+A.+Kollmanauthor=I.+Massovaauthor=C.+Reyesauthor=B.+Kuhnauthor=S.+Huoauthor=L.+Chongauthor=M.+Leeauthor=T.+Leeauthor=Y.+Duanauthor=W.+Wang&title=Calculating+structures+and+free+energies+of+complex+molecules%3A+combining+molecular+mechanics+and+continuum+models&doi=10.1021%2Far000033j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Calculating Structures and Free Energies of Complex Molecules: Combining Molecular Mechanics and Continuum Models</span></div><div class="casAuthors">Kollman, Peter A.; Massova, Irina; Reyes, Carolina; Kuhn, Bernd; Huo, Shuanghong; Chong, Lillian; Lee, Matthew; Lee, Taisung; Duan, Yong; Wang, Wei; Donini, Oreola; Cieplak, Piotr; Srinivasan, Jaysharee; Case, David A.; Cheatham, Thomas E., III</div><div class="citationInfo"><span class="NLM_cas:title">Accounts of Chemical Research</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">889-897</span>CODEN:
                <span class="NLM_cas:coden">ACHRE4</span>;
        ISSN:<span class="NLM_cas:issn">0001-4842</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review, with 63 refs.  A historical perspective on the application of mol. dynamics (MD) to biol. macromols. is presented.  Recent developments combining state-of-the-art force fields with continuum solvation calcns. have allowed us to reach the fourth era of MD applications in which one can often derive both accurate structure and accurate relative free energies from mol. dynamics trajectories.  We illustrate such applications on nucleic acid duplexes, RNA hairpins, protein folding trajectories, and protein-ligand, protein-protein, and protein-nucleic acid interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXGhwlfA3kvLVg90H21EOLACvtfcHk0likuiy81qz5MA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXmvFGiu7g%253D&md5=8436ee610ae145894428db1a1deff73c</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Far000033j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Far000033j%26sid%3Dliteratum%253Aachs%26aulast%3DKollman%26aufirst%3DP.%2BA.%26aulast%3DMassova%26aufirst%3DI.%26aulast%3DReyes%26aufirst%3DC.%26aulast%3DKuhn%26aufirst%3DB.%26aulast%3DHuo%26aufirst%3DS.%26aulast%3DChong%26aufirst%3DL.%26aulast%3DLee%26aufirst%3DM.%26aulast%3DLee%26aufirst%3DT.%26aulast%3DDuan%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DW.%26atitle%3DCalculating%2520structures%2520and%2520free%2520energies%2520of%2520complex%2520molecules%253A%2520combining%2520molecular%2520mechanics%2520and%2520continuum%2520models%26jtitle%3DAcc.%2520Chem.%2520Res.%26date%3D2000%26volume%3D33%26spage%3D889%26epage%3D897%26doi%3D10.1021%2Far000033j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kuhn, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerber, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schulz-Gasch, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stahl, M.</span></span> <span> </span><span class="NLM_article-title">Validation and use of the MM-PBSA approach for drug discovery</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">4040</span>– <span class="NLM_lpage">4048</span>, <span class="refDoi"> DOI: 10.1021/jm049081q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm049081q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A528%3ADC%252BD2MXjvFKgu74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=4040-4048&author=B.+Kuhnauthor=P.+Gerberauthor=T.+Schulz-Gaschauthor=M.+Stahl&title=Validation+and+use+of+the+MM-PBSA+approach+for+drug+discovery&doi=10.1021%2Fjm049081q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Validation and Use of the MM-PBSA Approach for Drug Discovery</span></div><div class="casAuthors">Kuhn, Bernd; Gerber, Paul; Schulz-Gasch, Tanja; Stahl, Martin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">4040-4048</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The MM-PBSA approach has become a popular method for calcg. binding affinities of biomol. complexes.  Published application examples focus on small test sets and few proteins and, hence, are of limited relevance in assessing the general validity of this method.  To further characterize MM-PBSA, the authors report on a more extensive study involving a large no. of ligands and eight different proteins.  Our results show that applying the MM-PBSA energy function to a single, relaxed complex structure is an adequate and sometimes more accurate approach than the std. free energy averaging over mol. dynamics snapshots.  The use of MM-PBSA on a single structure is shown to be valuable (a) as a postdocking filter in further enriching virtual screening results, (b) as a helpful tool to prioritize de novo design solns., and (c) for distinguishing between good and weak binders (ΔpIC50 ≥ 2-3), but rarely to reproduce smaller free energy differences.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGorwcFgm2UDiLVg90H21EOLACvtfcHk0lj2x33M6IoZ2Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXjvFKgu74%253D&md5=277858e33c944736d7f725359c265e10</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Fjm049081q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm049081q%26sid%3Dliteratum%253Aachs%26aulast%3DKuhn%26aufirst%3DB.%26aulast%3DGerber%26aufirst%3DP.%26aulast%3DSchulz-Gasch%26aufirst%3DT.%26aulast%3DStahl%26aufirst%3DM.%26atitle%3DValidation%2520and%2520use%2520of%2520the%2520MM-PBSA%2520approach%2520for%2520drug%2520discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D4040%26epage%3D4048%26doi%3D10.1021%2Fjm049081q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Massova, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kollman, P. A.</span></span> <span> </span><span class="NLM_article-title">Combined molecular mechanical and continuum solvent approach (MM-PBSA/GBSA) to predict ligand binding</span>. <i>Perspect. Drug Discovery Des.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">113</span>– <span class="NLM_lpage">135</span>, <span class="refDoi"> DOI: 10.1023/A:1008763014207</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=10.1023%2FA%3A1008763014207" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A528%3ADC%252BD3cXis12ju7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2000&pages=113-135&author=I.+Massovaauthor=P.+A.+Kollman&title=Combined+molecular+mechanical+and+continuum+solvent+approach+%28MM-PBSA%2FGBSA%29+to+predict+ligand+binding&doi=10.1023%2FA%3A1008763014207"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Combined molecular mechanical and continuum solvent approach (MM-PBSA/GBSA) to predict ligand binding</span></div><div class="casAuthors">Massova, Irina; Kollman, Peter A.</div><div class="citationInfo"><span class="NLM_cas:title">Perspectives in Drug Discovery and Design</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">Hydrophobicity and Solvation in Drug Design, Pt. II</span>),
    <span class="NLM_cas:pages">113-135</span>CODEN:
                <span class="NLM_cas:coden">PDDDEC</span>;
        ISSN:<span class="NLM_cas:issn">0928-2866</span>.
    
            (<span class="NLM_cas:orgname">Kluwer Academic Publishers</span>)
        </div><div class="casAbstract">Significant progress has been achieved in computational methods to treat solvent effects in recent years.  Among various techniques, the continuum solvent approach appears to be practically promising because it can be used to calc. reliable interaction and solvation energies in complex systems.  A computational scanning mutagenesis method, one of such new approaches, has been recently developed.  It combines the mol. mech. and continuum solvent approaches and allows one to identify the "hot spots" in binding interfaces from a single trajectory of a wild type complex.  Such techniques can be also used as a tool to optimize the interacting species for the binding, or as a ranking procedure in high throughput screening.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp4IQPgMmXRBbVg90H21EOLACvtfcHk0lj2x33M6IoZ2Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXis12ju7Y%253D&md5=520df05a7c7ed02fdf76d3e1e6f377f9</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1023%2FA%3A1008763014207&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FA%253A1008763014207%26sid%3Dliteratum%253Aachs%26aulast%3DMassova%26aufirst%3DI.%26aulast%3DKollman%26aufirst%3DP.%2BA.%26atitle%3DCombined%2520molecular%2520mechanical%2520and%2520continuum%2520solvent%2520approach%2520%2528MM-PBSA%252FGBSA%2529%2520to%2520predict%2520ligand%2520binding%26jtitle%3DPerspect.%2520Drug%2520Discovery%2520Des.%26date%3D2000%26volume%3D18%26spage%3D113%26epage%3D135%26doi%3D10.1023%2FA%3A1008763014207" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Keskin, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gursoy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nussinov, R.</span></span> <span> </span><span class="NLM_article-title">Principles of protein-protein interactions: what are the preferred ways for proteins to interact?</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>108</i></span>,  <span class="NLM_fpage">1225</span>– <span class="NLM_lpage">1244</span>, <span class="refDoi"> DOI: 10.1021/cr040409x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr040409x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A528%3ADC%252BD1cXjsVOmtrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=108&publication_year=2008&pages=1225-1244&author=O.+Keskinauthor=A.+Gursoyauthor=B.+Maauthor=R.+Nussinov&title=Principles+of+protein-protein+interactions%3A+what+are+the+preferred+ways+for+proteins+to+interact%3F&doi=10.1021%2Fcr040409x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Principles of protein-protein interactions: What are the preferred ways for proteins To interact?</span></div><div class="casAuthors">Keskin, Ozlem; Gursoy, Attila; Ma, Buyong; Nussinov, Ruth</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">108</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1225-1244</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  An overview of the principles that govern protein-protein interactions is provided by the authors.  Within this framework the authors highlight what they consider are key components in the question of "what are the preferred ways for proteins To interact.".  The goal is to be able to predict how the proteins will interact.  A description of binding sites in terms of preferred residue, and particularly preferred region and architectural organizations, may lead to classification strategies assisting in predictions of the preferred ways for proteins to interact.  Within the recurring favorable architectures, there are preferred cooperative "hot spot" organizations.  Preferred organization is a key in chem. and in protein science, whether in amyloid microfilaments or in globular protein-protein assocns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpEjFoMyzUh67Vg90H21EOLACvtfcHk0lj6CpegFZ7ucg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXjsVOmtrs%253D&md5=1028d6a1c43c8ad1a98286268101c305</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Fcr040409x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr040409x%26sid%3Dliteratum%253Aachs%26aulast%3DKeskin%26aufirst%3DO.%26aulast%3DGursoy%26aufirst%3DA.%26aulast%3DMa%26aufirst%3DB.%26aulast%3DNussinov%26aufirst%3DR.%26atitle%3DPrinciples%2520of%2520protein-protein%2520interactions%253A%2520what%2520are%2520the%2520preferred%2520ways%2520for%2520proteins%2520to%2520interact%253F%26jtitle%3DChem.%2520Rev.%26date%3D2008%26volume%3D108%26spage%3D1225%26epage%3D1244%26doi%3D10.1021%2Fcr040409x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schreiber, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haran, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, H. X.</span></span> <span> </span><span class="NLM_article-title">Fundamental aspects of protein-protein association kinetics</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>109</i></span>,  <span class="NLM_fpage">839</span>– <span class="NLM_lpage">860</span>, <span class="refDoi"> DOI: 10.1021/cr800373w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr800373w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsFWns7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2009&pages=839-860&author=G.+Schreiberauthor=G.+Haranauthor=H.+X.+Zhou&title=Fundamental+aspects+of+protein-protein+association+kinetics&doi=10.1021%2Fcr800373w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Fundamental Aspects of Protein-Protein Association Kinetics</span></div><div class="casAuthors">Schreiber, G.; Haran, G.; Zhou, H.-X.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">839-860</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  This review focuses on recent advances in deciphering the kinetic pathway of protein complex formation, the nature of the precomplex formed through diffusion (which the authors term the "transient complex"), the transition state and other intermediates (such as the so-called encounter complex) along the assocn. pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqBsx0HWzD2W7Vg90H21EOLACvtfcHk0lj6CpegFZ7ucg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsFWns7w%253D&md5=fbc492ede3242af279ee42705d23fdec</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Fcr800373w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr800373w%26sid%3Dliteratum%253Aachs%26aulast%3DSchreiber%26aufirst%3DG.%26aulast%3DHaran%26aufirst%3DG.%26aulast%3DZhou%26aufirst%3DH.%2BX.%26atitle%3DFundamental%2520aspects%2520of%2520protein-protein%2520association%2520kinetics%26jtitle%3DChem.%2520Rev.%26date%3D2009%26volume%3D109%26spage%3D839%26epage%3D860%26doi%3D10.1021%2Fcr800373w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jones, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thornton, J. M.</span></span> <span> </span><span class="NLM_article-title">Principles of protein-protein interactions</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>93</i></span>,  <span class="NLM_fpage">13</span>– <span class="NLM_lpage">20</span>, <span class="refDoi"> DOI: 10.1073/pnas.93.1.13</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=10.1073%2Fpnas.93.1.13" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=8552589" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A528%3ADyaK28XjslOrsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=93&publication_year=1996&pages=13-20&author=S.+Jonesauthor=J.+M.+Thornton&title=Principles+of+protein-protein+interactions&doi=10.1073%2Fpnas.93.1.13"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Principles of protein-protein interactions</span></div><div class="casAuthors">Jones, Susan; Thornton, Janet M.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">93</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">13-20</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">A review, with 47 refs., examg. protein complexes in the Brookhaven Protein Databank to gain a better understanding of the principles governing the interactions involved in protein-protein recognition.  The factors that influence the formation of protein-protein complexes are explored in four different types of protein-protein complexes: homodimeric proteins, heterodimeric proteins, enzyme-inhibitor complexes, and antibody-protein complexes.  The comparison between the complexes highlights differences that reflect their biol. roles.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr13RlSj8fKFLVg90H21EOLACvtfcHk0lj6CpegFZ7ucg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XjslOrsQ%253D%253D&md5=2b03d8f835104e5a7bf5da83afd4a8d1</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1073%2Fpnas.93.1.13&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.93.1.13%26sid%3Dliteratum%253Aachs%26aulast%3DJones%26aufirst%3DS.%26aulast%3DThornton%26aufirst%3DJ.%2BM.%26atitle%3DPrinciples%2520of%2520protein-protein%2520interactions%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1996%26volume%3D93%26spage%3D13%26epage%3D20%26doi%3D10.1073%2Fpnas.93.1.13" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gutgesell, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karumudi, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tonetti, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thatcher, G. R. J.</span></span> <span> </span><span class="NLM_article-title">Novel selective estrogen receptor downregulators (SERDs) developed against treatment-resistant breast cancer</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">1325</span>– <span class="NLM_lpage">1342</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01355</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01355" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFWqu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=1325-1342&author=R.+Xiongauthor=J.+Zhaoauthor=L.+M.+Gutgesellauthor=Y.+Wangauthor=S.+Leeauthor=B.+Karumudiauthor=H.+Zhaoauthor=Y.+Luauthor=D.+A.+Tonettiauthor=G.+R.+J.+Thatcher&title=Novel+selective+estrogen+receptor+downregulators+%28SERDs%29+developed+against+treatment-resistant+breast+cancer&doi=10.1021%2Facs.jmedchem.6b01355"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Novel Selective Estrogen Receptor Downregulators (SERDs) Developed against Treatment-Resistant Breast Cancer</span></div><div class="casAuthors">Xiong, Rui; Zhao, Jiong; Gutgesell, Lauren M.; Wang, Yueting; Lee, Sue; Karumudi, Bhargava; Zhao, Huiping; Lu, Yunlong; Tonetti, Debra A.; Thatcher, Gregory R. J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1325-1342</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Resistance to the selective estrogen receptor modulator (SERM) tamoxifen and to aromatase inhibitors that lower circulating estradiol occurs in up to 50% of patients, generally leading to an endocrine-independent ER+ phenotype.  Selective ER downregulators (SERDs) are able to ablate ER and thus theor. to prevent survival of both endocrine-dependent and independent ER+ tumors.  The clin. SERD, fulvestrant, is hampered by i.m. administration and undesirable pharmacokinetics.  Novel SERDs were designed using the 6-OH-benzothiophene (BT) scaffold common to arzoxifene and raloxifene.  Treatment-resistant (TR) ER+ cell lines (MCF-7:5C and MCF-7:TAM1) were used for optimization, followed by validation in the parent endocrine-dependent cell line (MCF-7:WS8), in 2D and 3D cultures, using ERα in-cell westerns, ERE-luciferase, and cell viability assays, with GDC-0810 (ARN-810) used for comparison.  Two BT SERDs with superior in vitro activity to GDC-0810 were studied for bioavailability and shown to cause regression of a TR, endocrine-independent ER+ xenograft superior to GDC-0810.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLqe0A8df78rVg90H21EOLACvtfcHk0lgmEJDVPyI-pg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFWqu70%253D&md5=71dbdfa2152f9e9019e8d138ed41313e</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01355&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01355%26sid%3Dliteratum%253Aachs%26aulast%3DXiong%26aufirst%3DR.%26aulast%3DZhao%26aufirst%3DJ.%26aulast%3DGutgesell%26aufirst%3DL.%2BM.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DLee%26aufirst%3DS.%26aulast%3DKarumudi%26aufirst%3DB.%26aulast%3DZhao%26aufirst%3DH.%26aulast%3DLu%26aufirst%3DY.%26aulast%3DTonetti%26aufirst%3DD.%2BA.%26aulast%3DThatcher%26aufirst%3DG.%2BR.%2BJ.%26atitle%3DNovel%2520selective%2520estrogen%2520receptor%2520downregulators%2520%2528SERDs%2529%2520developed%2520against%2520treatment-resistant%2520breast%2520cancer%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D1325%26epage%3D1342%26doi%3D10.1021%2Facs.jmedchem.6b01355" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brzozowski, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pike, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dauter, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hubbard, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonn, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engström, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ö hman, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greene, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gustafsson, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlquist, M.</span></span> <span> </span><span class="NLM_article-title">Molecular basis of agonism and antagonism in the oestrogen receptor</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>389</i></span>,  <span class="NLM_fpage">753</span>– <span class="NLM_lpage">758</span>, <span class="refDoi"> DOI: 10.1038/39645</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=10.1038%2F39645" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=9338790" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A528%3ADyaK2sXmvVSnt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=389&publication_year=1997&pages=753-758&author=A.+M.+Brzozowskiauthor=A.+C.+Pikeauthor=Z.+Dauterauthor=R.+E.+Hubbardauthor=T.+Bonnauthor=O.+Engstr%C3%B6mauthor=L.+%C3%96+hmanauthor=G.+L.+Greeneauthor=J.+A.+Gustafssonauthor=M.+Carlquist&title=Molecular+basis+of+agonism+and+antagonism+in+the+oestrogen+receptor&doi=10.1038%2F39645"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular basis of agonism and antagonism in the estrogen receptor</span></div><div class="casAuthors">Brzozowski, Andrzej M.; Pike, Ashley C. W.; Dauter, Zbigniew; Hubbard, Roderick E.; Bonn, Tomas; Engstrom, Owe; Ohman, Lars; Greene, Geoffrey L.; Gustafsson, Jan-Ake; Carlquist, Mats</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London)</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">389</span>
        (<span class="NLM_cas:issue">6652</span>),
    <span class="NLM_cas:pages">753-758</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Macmillan Magazines</span>)
        </div><div class="casAbstract">Estrogens are involved in the growth, development and homeostasis of a no. of tissues.  The physiol. effects of these steroids are mediated by a ligand-inducible nuclear transcription factor, the estrogen receptor (ER).  Hormone binding to the ligand-binding domain (LBD) of the ER initiates a series of mol. events culminating in the activation or repression of target genes.  Transcriptional regulation arises from the direct interaction of the ER with components of the cellular transcription machinery.  Here we report the crystal structures of the LBD of ER in complex with the endogenous estrogen, 17β-estradiol, and the selective antagonist raloxifene, at resolns. of 3.1 and 2.6Å, resp.  The structures provide a mol. basis for the distinctive pharmacophore of the ER and its catholic binding properties.  Agonist and antagonist bind at the same site within the core of the LBD but demonstrate different binding modes.  In addn., each class of ligand induces a distinct conformation in the transactivation domain of the LBD, providing structural evidence of the mechanism of antagonism.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAXbSo66V1LrVg90H21EOLACvtfcHk0lgmEJDVPyI-pg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXmvVSnt74%253D&md5=2a63951b881b329ff142c9fbc82a13e3</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1038%2F39645&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F39645%26sid%3Dliteratum%253Aachs%26aulast%3DBrzozowski%26aufirst%3DA.%2BM.%26aulast%3DPike%26aufirst%3DA.%2BC.%26aulast%3DDauter%26aufirst%3DZ.%26aulast%3DHubbard%26aufirst%3DR.%2BE.%26aulast%3DBonn%26aufirst%3DT.%26aulast%3DEngstr%25C3%25B6m%26aufirst%3DO.%26aulast%3D%25C3%2596%2Bhman%26aufirst%3DL.%26aulast%3DGreene%26aufirst%3DG.%2BL.%26aulast%3DGustafsson%26aufirst%3DJ.%2BA.%26aulast%3DCarlquist%26aufirst%3DM.%26atitle%3DMolecular%2520basis%2520of%2520agonism%2520and%2520antagonism%2520in%2520the%2520oestrogen%2520receptor%26jtitle%3DNature%26date%3D1997%26volume%3D389%26spage%3D753%26epage%3D758%26doi%3D10.1038%2F39645" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mohammad, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gong, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">January, C. T.</span></span> <span> </span><span class="NLM_article-title">Blockage of the HERG human cardiac K<sup>+</sup> channel by the gastrointestinal prokinetic agent cisapride</span>. <i>Am. J. Physiol.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>273</i></span>,  <span class="NLM_fpage">H2534</span>– <span class="NLM_lpage">H2538</span>, <span class="refDoi"> DOI: 10.1152/ajpheart.1997.273.5.H2534</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=10.1152%2Fajpheart.1997.273.5.H2534" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=9374794" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A528%3ADyaK2sXnsVCqu7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=273&publication_year=1997&pages=H2534-H2538&author=S.+Mohammadauthor=Z.+Zhouauthor=Q.+Gongauthor=C.+T.+January&title=Blockage+of+the+HERG+human+cardiac+K%2B+channel+by+the+gastrointestinal+prokinetic+agent+cisapride&doi=10.1152%2Fajpheart.1997.273.5.H2534"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Blockage of the HERG human cardiac K+ channel by the gastrointestinal prokinetic agent cisapride</span></div><div class="casAuthors">Mohammad, Saeed; Zhou, Zhengfeng; Gong, Qiuming; January, Craig T.</div><div class="citationInfo"><span class="NLM_cas:title">American Journal of Physiology</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">273</span>
        (<span class="NLM_cas:issue">5, Pt. 2</span>),
    <span class="NLM_cas:pages">H2534-H2538</span>CODEN:
                <span class="NLM_cas:coden">AJPHAP</span>;
        ISSN:<span class="NLM_cas:issn">0002-9513</span>.
    
            (<span class="NLM_cas:orgname">American Physiological Society</span>)
        </div><div class="casAbstract">Cisapride, a gastrointestinal prokinetic agent, is known to caused long Q-T syndrome and ventricular arrhythmias.  The cellular mechanism is not known.  The human ether-a-go-go-related gene (HERG), which encodes the rapidly activating delayed rectifier K+ current and is important in cardiac repolarization, may serve as a target for the action of cisapride.  The authors tested the hypothesis that cisapride blocks HERG.  The whole cell path-clamp recording technique was used to study HERG channels stably expressed heterologously in HEK293 cells.  Under voltage-clamp conditions, cisapride block of HERG is dose dependent with a half-maximal inhibitory concn. of 6.5 nM at 22° (cells).  Currents rapidly recovered with drug washout.  The onset of block by cisapride required channel activation indicative of open or inactivated state blockage.  Block of HERG with cisapride after channel activation was voltage dependent.  At -20 mV, 10 nM cisapride reduced HERG tail-current amplitude by 5%, whereas, at +20 mV, the tail-current amplitude was reduced by 45% (cells).  At -20 and +20 mV, 100 nM cisapride reduced tail-current amplitude by 66 and 90%, resp.  The authors conclude that cisapride is a potent blocker of HERG channels expressed in HEK293 cells.  This effect may account for the clin. occurrence of Q-T prolongation and ventricular arrhythmias obsd. with cisapride.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoNimFBPcs477Vg90H21EOLACvtfcHk0ljYindGg-HrVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXnsVCqu7k%253D&md5=291fcfb4f2d9efd92120735a91d27b59</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1152%2Fajpheart.1997.273.5.H2534&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1152%252Fajpheart.1997.273.5.H2534%26sid%3Dliteratum%253Aachs%26aulast%3DMohammad%26aufirst%3DS.%26aulast%3DZhou%26aufirst%3DZ.%26aulast%3DGong%26aufirst%3DQ.%26aulast%3DJanuary%26aufirst%3DC.%2BT.%26atitle%3DBlockage%2520of%2520the%2520HERG%2520human%2520cardiac%2520K%252B%2520channel%2520by%2520the%2520gastrointestinal%2520prokinetic%2520agent%2520cisapride%26jtitle%3DAm.%2520J.%2520Physiol.%26date%3D1997%26volume%3D273%26spage%3DH2534%26epage%3DH2538%26doi%3D10.1152%2Fajpheart.1997.273.5.H2534" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pollard, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skinner, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lazic, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prior, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conlon, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valentin, J.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dota, C.</span></span> <span> </span><span class="NLM_article-title">An analysis of the relationship between preclinical and clinical QT interval-related data</span>. <i>Toxicol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>159</i></span>,  <span class="NLM_fpage">94</span>– <span class="NLM_lpage">101</span>, <span class="refDoi"> DOI: 10.1093/toxsci/kfx125</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=10.1093%2Ftoxsci%2Fkfx125" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=28903488" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A528%3ADC%252BC1cXmtFSgsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=159&publication_year=2017&pages=94-101&author=C.+E.+Pollardauthor=M.+Skinnerauthor=S.+E.+Lazicauthor=H.+M.+Priorauthor=K.+M.+Conlonauthor=J.-P.+Valentinauthor=C.+Dota&title=An+analysis+of+the+relationship+between+preclinical+and+clinical+QT+interval-related+data&doi=10.1093%2Ftoxsci%2Fkfx125"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">An analysis of the relationship between preclinical and clinical QT interval-related data</span></div><div class="casAuthors">Pollard, Christopher E.; Skinner, Matthew; Lazic, Stanley E.; Prior, Helen M.; Conlon, Kelly M.; Valentin, Jean-Pierre; Dota, Corina</div><div class="citationInfo"><span class="NLM_cas:title">Toxicological Sciences</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">159</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">94-101</span>CODEN:
                <span class="NLM_cas:coden">TOSCF2</span>;
        ISSN:<span class="NLM_cas:issn">1096-0929</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">There has been significant focus on drug-induced QT interval prolongation caused by block of the human ether-a-go-corelated gene (hERG)-encoded potassium channel.  Regulatory guidance has been implemented to assess QT interval prolongation risk: preclin. guidance requires a candidate drug potency as a hERG channel blocker to be defined and also its effect on QT interval in a non-rodent species; clin. guidance requires a "Thorough QT Study" during development, although some QT prolonging compds. are identified earlier via a Phase I study.  Clin., heart rate-cor. QT interval (QTc) data on 24 compds. (13 positives; 11 negatives) were compared with their effect on dog QTc and the concn. of compd. causing 50% inhibition (IC50) of hERG current.  Concordance was assessed by calcg. sensitivity and specificity across a range of decision thresholds, thus yielding receiver operating characteristic curves of sensitivity vs. (1-specificity).  The area under the curve of ROC curves (for which 0.5 and 1 indicate chance and perfect concordance, resp.) was used to summarize concordance.  Three aspects of preclin. data were compared with the clin. outcome (receiver operating characteristic area under the curve values shown in brackets): abs. hERG IC50 (0.78); safety margin between hERG IC50 and clin. peak free plasma exposure (0.80); safety margin between QTc effects in dogs and clin. peak free plasma exposure (0.81).  Pos. and neg. predictive values of abs. hERG IIC50 indicated that from an early drug discovery perspective, low potency compds. can be progressed on the basis of a low risk of causing a QTc increase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzer6VhKiwIbVg90H21EOLACvtfcHk0ljYindGg-HrVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXmtFSgsrc%253D&md5=442a63490fb478512d52828a9d8d0a08</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1093%2Ftoxsci%2Fkfx125&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Ftoxsci%252Fkfx125%26sid%3Dliteratum%253Aachs%26aulast%3DPollard%26aufirst%3DC.%2BE.%26aulast%3DSkinner%26aufirst%3DM.%26aulast%3DLazic%26aufirst%3DS.%2BE.%26aulast%3DPrior%26aufirst%3DH.%2BM.%26aulast%3DConlon%26aufirst%3DK.%2BM.%26aulast%3DValentin%26aufirst%3DJ.-P.%26aulast%3DDota%26aufirst%3DC.%26atitle%3DAn%2520analysis%2520of%2520the%2520relationship%2520between%2520preclinical%2520and%2520clinical%2520QT%2520interval-related%2520data%26jtitle%3DToxicol.%2520Sci.%26date%3D2017%26volume%3D159%26spage%3D94%26epage%3D101%26doi%3D10.1093%2Ftoxsci%2Fkfx125" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, L.</span></span> <span> </span><span class="NLM_article-title">Design, synthesis and biological evaluation of novel indazole-based derivatives as potent HDAC inhibitors via fragment-based virtual screening</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>192</i></span>,  <span class="NLM_fpage">112189</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2020.112189</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=10.1016%2Fj.ejmech.2020.112189" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=32151834" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A528%3ADC%252BB3cXksFels7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=192&publication_year=2020&pages=112189&author=J.+Liuauthor=J.+Zhouauthor=F.+Heauthor=L.+Gaoauthor=Y.+Wenauthor=L.+Gaoauthor=P.+Wangauthor=D.+Kangauthor=L.+Hu&title=Design%2C+synthesis+and+biological+evaluation+of+novel+indazole-based+derivatives+as+potent+HDAC+inhibitors+via+fragment-based+virtual+screening&doi=10.1016%2Fj.ejmech.2020.112189"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Design, synthesis and biological evaluation of novel indazole-based derivatives as potent HDAC inhibitors via fragment-based virtual screening</span></div><div class="casAuthors">Liu, Jian; Zhou, Jingxian; He, Fengjun; Gao, Liang; Wen, Yu; Gao, Lina; Wang, Ping; Kang, Di; Hu, Lihong</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">192</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">112189</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Based on fragment-based virtual screening and bioisosterism strategies, novel indazoles I [R = C6H5, 4-C5H4N, 3-C4H3S etc.; X = C; Y = (CH2)3, (CH2)4, (CH2)6, etc.] and pyrazolo[3,4-b] pyridine derivs. I [R = 3-MeO-C6H4, 4-MeO-C6H4; X = N; Y = (CH2)6] as HDACs inhibitors were synthesized and evaluated.  Most of these compds. displayed good to excellent inhibitory activities against HDACs, esp. compds. I [R = 4-MeO-C6H4, 4-Cl-C6H4; X = C; Y = (CH2)6] were identified as potent inhibitors of HDAC1 (IC50 = 2.7 nM and IC50 = 3.1 nM), HDAC2 (IC50 = 4.2 nM and IC50 = 3.6 nM) and HDAC8 (IC50 = 3.6 nM and IC50 = 3.3 nM).  Further anti-proliferation assays revealed that compds. I [R = 4-MeO-C6H4, 4-Cl-C6H4; X = C; Y = (CH2)6] showed better anti-proliferative activities against HCT-116 and HeLa cells than pos. control SAHA.  The western blot anal. results indicated that compds. I [R = 4-MeO-C6H4, 4-Cl-C6H4; X = C; Y = (CH2)6] noticeably up-regulated the level of acetylated α-tubulin and histone H3.  In addn., the two compds. I [R = 4-MeO-C6H4, 4-Cl-C6H4; X = C; Y = (CH2)6] could arrest cell cycle in G2/M phase and promote cell apoptosis, which was similar as the ref. compd. SAHA.  Through the mol. docking and dynamic studies, the potent HDAC inhibitory activities mainly caused by van der Waals and electrostatic interactions with the HDACs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8VNY0R6hLfrVg90H21EOLACvtfcHk0liiBTz6Sp8FfA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXksFels7w%253D&md5=ad3cca7c8927db6331c1216680e2857b</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2020.112189&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2020.112189%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DZhou%26aufirst%3DJ.%26aulast%3DHe%26aufirst%3DF.%26aulast%3DGao%26aufirst%3DL.%26aulast%3DWen%26aufirst%3DY.%26aulast%3DGao%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DP.%26aulast%3DKang%26aufirst%3DD.%26aulast%3DHu%26aufirst%3DL.%26atitle%3DDesign%252C%2520synthesis%2520and%2520biological%2520evaluation%2520of%2520novel%2520indazole-based%2520derivatives%2520as%2520potent%2520HDAC%2520inhibitors%2520via%2520fragment-based%2520virtual%2520screening%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2020%26volume%3D192%26spage%3D112189%26doi%3D10.1016%2Fj.ejmech.2020.112189" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Miller, B. R.,  3rd</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGee, T. D.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swails, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Homeyer, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gohlke, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roitberg, A. E.</span></span> <span> </span><span class="NLM_article-title">MMPBSA.py: An efficient program for end-state free energy calculations</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">3314</span>– <span class="NLM_lpage">3321</span>, <span class="refDoi"> DOI: 10.1021/ct300418h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct300418h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtV2gtrzP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2012&pages=3314-3321&author=B.+R.+Millerauthor=T.+D.+McGeeauthor=J.+M.+Swailsauthor=N.+Homeyerauthor=H.+Gohlkeauthor=A.+E.+Roitberg&title=MMPBSA.py%3A+An+efficient+program+for+end-state+free+energy+calculations&doi=10.1021%2Fct300418h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">MMPBSA.py: An Efficient Program for End-State Free Energy Calculations</span></div><div class="casAuthors">Miller, Bill R., III; McGee, T. Dwight, Jr.; Swails, Jason M.; Homeyer, Nadine; Gohlke, Holger; Roitberg, Adrian E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3314-3321</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">MM-PBSA is a post-processing end-state method to calc. free energies of mols. in soln.  MMPBSA.py is a program written in Python for streamlining end-state free energy calcns. using ensembles derived from mol. dynamics (MD) or Monte Carlo (MC) simulations.  Several implicit solvation models are available with MMPBSA.py, including the Poisson-Boltzmann Model, the Generalized Born Model, and the Ref. Interaction Site Model.  Vibrational frequencies may be calcd. using normal mode or quasi-harmonic anal. to approx. the solute entropy.  Specific interactions can also be dissected using free energy decompn. or alanine scanning.  A parallel implementation significantly speeds up the calcn. by dividing frames evenly across available processors.  MMPBSA.py is an efficient, user-friendly program with the flexibility to accommodate the needs of users performing end-state free energy calcns.  The source code can be downloaded at http://ambermd.org/ with AmberTools, released under the GNU General Public License.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZK3C1jqiSULVg90H21EOLACvtfcHk0liiBTz6Sp8FfA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtV2gtrzP&md5=cc4148bd8f70c7cad94fd3ec6f580e52</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1021%2Fct300418h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct300418h%26sid%3Dliteratum%253Aachs%26aulast%3DMiller%26aufirst%3DB.%2BR.%26aulast%3DMcGee%26aufirst%3DT.%2BD.%26aulast%3DSwails%26aufirst%3DJ.%2BM.%26aulast%3DHomeyer%26aufirst%3DN.%26aulast%3DGohlke%26aufirst%3DH.%26aulast%3DRoitberg%26aufirst%3DA.%2BE.%26atitle%3DMMPBSA.py%253A%2520An%2520efficient%2520program%2520for%2520end-state%2520free%2520energy%2520calculations%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2012%26volume%3D8%26spage%3D3314%26epage%3D3321%26doi%3D10.1021%2Fct300418h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article has not yet been cited by other publications.</p></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00280/20210602/images/medium/jm1c00280_0012.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00280/20210602/images/large/jm1c00280_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00280&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00280/20210602/images/medium/jm1c00280_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00280/20210602/images/large/jm1c00280_0001.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Representative selective aromatase inhibitors (AIs), selective estrogen receptor modulators (SERMs), and selective estrogen receptor degraders (SERDs).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00280/20210602/images/large/jm1c00280_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00280&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00280/20210602/images/medium/jm1c00280_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00280/20210602/images/large/jm1c00280_0002.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Analysis of binding “hot spots” of ERα. (a) Crystal structure of <b>AZD9496</b> (blue; PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5ACC">5ACC</a>) superimposed on reference structure of <b>GW-5638</b> (white; PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5AVV">5AVV</a>) bound to ERα pocket. The red circle revealed that the acryl acid group interacted with hot spot 1: Trp383, Leu525, and Val534. The blue rectangle revealed that the pyrido[3,4-<i>b</i>]-indole scaffold interacted with hot spot 2: Leu346, Met388, Leu391, Phe404, Met421, and Ile424. (b) Backbone RMSD of the protein along with simulation time. The blue line represents <b>GW-5638</b>; the red line represents <b>AZD9496</b>. (c) Hot-spot residue contribution for protein–ligand interaction. (d) Calculation results of hot-spot residue mutation to alanine for <b>AZD9496</b> (red) and <b>GW-6838</b> (blue) with ERα mutants.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00280/20210602/images/large/jm1c00280_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00280&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00280/20210602/images/medium/jm1c00280_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00280/20210602/images/large/jm1c00280_0003.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Design strategy of target compounds.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00280/20210602/images/large/jm1c00280_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00280&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00280/20210602/images/medium/jm1c00280_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00280/20210602/images/large/jm1c00280_0004.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Docking result of design compounds (<b>F1</b>–<b>F9</b>) toward ERα (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5ACC">5ACC</a>) and drug candidate <b>AZD9496</b>. (a) <b>AZD9496</b> (blue) and <b>F1</b> (green); (b) <b>AZD9496</b> (blue), <b>F2</b> (white), and <b>F3</b> (pink); (c) <b>AZD9496</b> (blue), <b>F4</b> (white), <b>F5</b> (yellow), <b>F6</b> (pink); (d) <b>AZD9496</b> (blue), <b>F7</b> (white), <b>F8</b> (yellow), and <b>F9</b> (pink).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00280/20210602/images/large/jm1c00280_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00280&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00280/20210602/images/medium/jm1c00280_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00280/20210602/images/large/jm1c00280_0008.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of <b>A</b> Series of Compounds<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00280/20210602/images/large/jm1c00280_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00280&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: MeI, NaH, THF, 0 °C, 4 h, 66%.</p></p></figure><figure data-id="sch2" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00280/20210602/images/medium/jm1c00280_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00280/20210602/images/large/jm1c00280_0009.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of <b>B</b> Series of Compounds<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00280/20210602/images/large/jm1c00280_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00280&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) TBAF, THF, 80 °C, 4 h, 73%; (b) Pd(OH)<sub>2</sub>, H<sub>2</sub>, EtOH, rt, 8 h, 98%; (c) MeTi(<i>i-</i>PrO)<sub>3</sub>, EtMgBr, BF<sub>3</sub>·Et<sub>2</sub>O, THF, 2 h, 76%; (d) NaOH, H<sub>2</sub>O, MeOH, 2 h, 80%; (e) Na<sub>2</sub>CO<sub>3</sub>, 2,2,2-trifluoroethanol, 48 h, 65%; (f) NaBH<sub>3</sub>CN, ammonium acetate, MeOH, rt, 8 h, 54%; (g) 2-fluoro-2-methylpropyl trifluoromethanesulfonate (<b>11</b>), DIPEA, dioxane, 90 °C, 2 h, 35–90%; (h) methyl (<i>E</i>)-3-(3,5-difluoro-4-formylphenyl)acrylate (<b>13</b>), HOAc, toluene, 80 °C, 8 h, 41–70%; (i) NaOH, H<sub>2</sub>O, THF, MeOH, rt, 75–83%.</p></p></figure><figure data-id="sch3" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00280/20210602/images/medium/jm1c00280_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00280/20210602/images/large/jm1c00280_0010.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of <b>C</b> Series of Compounds<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00280/20210602/images/large/jm1c00280_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00280&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) (i) alkyl trifluoromethanesulfonate, DIPEA, dioxane, 90 °C, 8 h, 63–72% and (ii) iodobenzene, CuI, Cs<sub>2</sub>CO<sub>3</sub>, DMF, microwave, 135 °C, 2 h, 34%; (b) methyl (<i>E</i>)-3-(3,5-difluoro-4-formylphenyl)acrylate (<b>13</b>), HOAc, toluene, 80 °C, 8 h, 41–72%; (c) NaOH, H<sub>2</sub>O, THF, MeOH, rt, 72–86%.</p></p></figure><figure data-id="sch4" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00280/20210602/images/medium/jm1c00280_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00280/20210602/images/large/jm1c00280_0011.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of <b>D</b> Series of Compounds<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00280/20210602/images/large/jm1c00280_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00280&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) methyl acrylate, Pd(OAc)<sub>2</sub>, Et<sub>3</sub>N, DMF, 80 °C, 8 h, 72–96%; (b) bis(pinacolato)diboron, PdCl<sub>2</sub>(dppf), KOAc, 80 °C, 12 h, 80–88%; (c) 4-bromo-2,6-difluorobenzaldehyde, PdCl<sub>2</sub>(dppf)<sub>2</sub>, Na<sub>2</sub>CO<sub>3</sub>, toluene, H<sub>2</sub>O, 100 °C, 16 h, 75–86%; (d) <b>17</b> or <b>20</b>, HOAc, toluene, 8 h, 45–78%; (e) NaOH, H<sub>2</sub>O, THF, MeOH, rt, 2 h, 70–92%; (f) 4-bromo-2,6-difluorobenzaldehyde, HOAc, toluene, 80 °C, 8 h, 84%; (g) NHR<sub>1</sub>R<sub>2</sub>, Pd<sub>2</sub>(dba)<sub>3</sub>, XPhos, Cs<sub>2</sub>CO<sub>3</sub>, toluene, 80 °C, 8 h, 60–81%;</p></p></figure><figure data-id="fig5" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00280/20210602/images/medium/jm1c00280_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00280/20210602/images/large/jm1c00280_0005.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Docking conformation (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5ACC">5ACC</a>) of <b>D24</b> (green) superimposed onto cocrystal structure of <b>AZD9496</b> (pink).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00280/20210602/images/large/jm1c00280_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00280&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00280/20210602/images/medium/jm1c00280_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00280/20210602/images/large/jm1c00280_0006.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Degradation of ERα by compound <b>AZD9496</b>, <b>D24</b>, and estradiol (<b>E2</b>) by Western blot analysis. MCF-7 cells were incubated with <b>AZD9496</b>, <b>D24</b>, or estradiol at the doses above. Total protein was extracted at 48 h incubation with compound and immunoblotting performed to detect effects on ERα and PR (progesterone receptor) levels.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00280/20210602/images/large/jm1c00280_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00280&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00280/20210602/images/medium/jm1c00280_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00280/20210602/images/large/jm1c00280_0007.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. In vivo efficacy and ERα by <b>AZD9496</b> and <b>D24</b> in the MCF-7 xenograft model. (a) Tumor growth inhibition (TGI) dose response upon oral administration of compounds <b>AZD9496</b> and <b>D24</b> (<i>n</i> = 6 per dose group, QD × 21days); (b) Nude mouse body weight measured every 2 days.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00280/20210602/images/large/jm1c00280_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00280&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i114">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_73253" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_73253" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 52 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hanahan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinberg, R. A.</span></span> <span> </span><span class="NLM_article-title">Hallmarks of cancer: the next generation</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>144</i></span>,  <span class="NLM_fpage">646</span>– <span class="NLM_lpage">674</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2011.02.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=10.1016%2Fj.cell.2011.02.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=21376230" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A528%3ADC%252BC3MXjsFeqtrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=144&publication_year=2011&pages=646-674&author=D.+Hanahanauthor=R.+A.+Weinberg&title=Hallmarks+of+cancer%3A+the+next+generation&doi=10.1016%2Fj.cell.2011.02.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Hallmarks of cancer: the next generation</span></div><div class="casAuthors">Hanahan, Douglas; Weinberg, Robert A.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">144</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">646-674</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  The hallmarks of cancer comprise six biol. capabilities acquired during the multistep development of human tumors.  The hallmarks constitute an organizing principle for rationalizing the complexities of neoplastic disease.  They include sustaining proliferative signaling, evading growth suppressors, resisting cell death, enabling replicative immortality, inducing angiogenesis, and activating invasion and metastasis.  Underlying these hallmarks are genome instability, which generates the genetic diversity that expedites their acquisition, and inflammation, which fosters multiple hallmark functions.  Conceptual progress in the last decade has added two emerging hallmarks of potential generality to this list-reprogramming of energy metab. and evading immune destruction.  In addn. to cancer cells, tumors exhibit another dimension of complexity: they contain a repertoire of recruited, ostensibly normal cells that contribute to the acquisition of hallmark traits by creating the "tumor microenvironment.".  Recognition of the widespread applicability of these concepts will increasingly affect the development of new means to treat human cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTcpqXALcb6LVg90H21EOLACvtfcHk0ljp02LBVge4Sw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXjsFeqtrk%253D&md5=b36f160c417a712e83cb83c577f0018e</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2011.02.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2011.02.013%26sid%3Dliteratum%253Aachs%26aulast%3DHanahan%26aufirst%3DD.%26aulast%3DWeinberg%26aufirst%3DR.%2BA.%26atitle%3DHallmarks%2520of%2520cancer%253A%2520the%2520next%2520generation%26jtitle%3DCell%26date%3D2011%26volume%3D144%26spage%3D646%26epage%3D674%26doi%3D10.1016%2Fj.cell.2011.02.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, W.-Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, R.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baade, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, H.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bray, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jemal, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, X.-Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, J.</span></span> <span> </span><span class="NLM_article-title">Cancer statistics in China, 2015</span>. <i>Ca-Cancer J. Clin.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">115</span>– <span class="NLM_lpage">132</span>, <span class="refDoi"> DOI: 10.3322/caac.21338</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=10.3322%2Fcaac.21338" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=26808342" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A280%3ADC%252BC28njvVykug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2016&pages=115-132&author=W.-Q.+Chenauthor=R.-S.+Zhengauthor=P.+D.+Baadeauthor=S.-W.+Zhangauthor=H.-M.+Zengauthor=F.+Brayauthor=A.+Jemalauthor=X.-Q.+Yuauthor=J.+He&title=Cancer+statistics+in+China%2C+2015&doi=10.3322%2Fcaac.21338"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Cancer statistics in China, 2015</span></div><div class="casAuthors">Chen Wanqing; Zheng Rongshou; Baade Peter D; Zhang Siwei; Zeng Hongmei; Bray Freddie; Jemal Ahmedin; Yu Xue Qin; Yu Xue Qin; He Jie</div><div class="citationInfo"><span class="NLM_cas:title">CA: a cancer journal for clinicians</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">115-32</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">With increasing incidence and mortality, cancer is the leading cause of death in China and is a major public health problem.  Because of China's massive population (1.37 billion), previous national incidence and mortality estimates have been limited to small samples of the population using data from the 1990s or based on a specific year.  With high-quality data from an additional number of population-based registries now available through the National Central Cancer Registry of China, the authors analyzed data from 72 local, population-based cancer registries (2009-2011), representing 6.5% of the population, to estimate the number of new cases and cancer deaths for 2015.  Data from 22 registries were used for trend analyses (2000-2011).  The results indicated that an estimated 4292,000 new cancer cases and 2814,000 cancer deaths would occur in China in 2015, with lung cancer being the most common incident cancer and the leading cause of cancer death.  Stomach, esophageal, and liver cancers were also commonly diagnosed and were identified as leading causes of cancer death.  Residents of rural areas had significantly higher age-standardized (Segi population) incidence and mortality rates for all cancers combined than urban residents (213.6 per 100,000 vs 191.5 per 100,000 for incidence; 149.0 per 100,000 vs 109.5 per 100,000 for mortality, respectively).  For all cancers combined, the incidence rates were stable during 2000 through 2011 for males (+0.2% per year; P = .1), whereas they increased significantly (+2.2% per year; P < .05) among females.  In contrast, the mortality rates since 2006 have decreased significantly for both males (-1.4% per year; P < .05) and females (-1.1% per year; P < .05).  Many of the estimated cancer cases and deaths can be prevented through reducing the prevalence of risk factors, while increasing the effectiveness of clinical care delivery, particularly for those living in rural areas and in disadvantaged populations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSkZeZPQ7Kk7ntlzo9xastyfW6udTcc2ebWYQoNHaDokLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC28njvVykug%253D%253D&md5=2affa5f8beab6f885ccd830cbb249707</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.3322%2Fcaac.21338&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3322%252Fcaac.21338%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DW.-Q.%26aulast%3DZheng%26aufirst%3DR.-S.%26aulast%3DBaade%26aufirst%3DP.%2BD.%26aulast%3DZhang%26aufirst%3DS.-W.%26aulast%3DZeng%26aufirst%3DH.-M.%26aulast%3DBray%26aufirst%3DF.%26aulast%3DJemal%26aufirst%3DA.%26aulast%3DYu%26aufirst%3DX.-Q.%26aulast%3DHe%26aufirst%3DJ.%26atitle%3DCancer%2520statistics%2520in%2520China%252C%25202015%26jtitle%3DCa-Cancer%2520J.%2520Clin.%26date%3D2016%26volume%3D66%26spage%3D115%26epage%3D132%26doi%3D10.3322%2Fcaac.21338" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nardone, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Angelis, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trivedi, M. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osborne, C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schiff, R.</span></span> <span> </span><span class="NLM_article-title">The changing role of ER in endocrine resistance</span>. <i>Breast.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">S60</span>– <span class="NLM_lpage">S66</span>, <span class="refDoi"> DOI: 10.1016/j.breast.2015.07.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=10.1016%2Fj.breast.2015.07.015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=26271713" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2015&pages=S60-S66&author=A.+Nardoneauthor=C.+De+Angelisauthor=M.+V.+Trivediauthor=C.+K.+Osborneauthor=R.+Schiff&title=The+changing+role+of+ER+in+endocrine+resistance&doi=10.1016%2Fj.breast.2015.07.015"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.breast.2015.07.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.breast.2015.07.015%26sid%3Dliteratum%253Aachs%26aulast%3DNardone%26aufirst%3DA.%26aulast%3DDe%2BAngelis%26aufirst%3DC.%26aulast%3DTrivedi%26aufirst%3DM.%2BV.%26aulast%3DOsborne%26aufirst%3DC.%2BK.%26aulast%3DSchiff%26aufirst%3DR.%26atitle%3DThe%2520changing%2520role%2520of%2520ER%2520in%2520endocrine%2520resistance%26jtitle%3DBreast.%26date%3D2015%26volume%3D24%26spage%3DS60%26epage%3DS66%26doi%3D10.1016%2Fj.breast.2015.07.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tong, C. W. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, W. C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">To, K. K. W.</span></span> <span> </span><span class="NLM_article-title">Recent advances in the treatment of breast cancer</span>. <i>Front. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">227</span>, <span class="refDoi"> DOI: 10.3389/fonc.2018.00227</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=10.3389%2Ffonc.2018.00227" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=29963498" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A280%3ADC%252BB3c%252FjvVylsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2018&pages=227&author=C.+W.+S.+Tongauthor=M.+Wuauthor=W.+C.+S.+Choauthor=K.+K.+W.+To&title=Recent+advances+in+the+treatment+of+breast+cancer&doi=10.3389%2Ffonc.2018.00227"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Recent Advances in the Treatment of Breast Cancer</span></div><div class="casAuthors">Tong Christy W S; Wu Mingxia; To Kenneth K W; Cho William C S</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in oncology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">227</span>
        ISSN:<span class="NLM_cas:issn">2234-943X</span>.
    </div><div class="casAbstract">Breast cancer (BC) is the most common malignancy in women.  It is classified into a few major molecular subtypes according to hormone and growth factor receptor expression.  Over the past few years, substantial advances have been made in the discovery of new drugs for treating BC.  Improved understanding of the biologic heterogeneity of BC has allowed the development of more effective and individualized approach to treatment.  In this review, we provide an update about the current treatment strategy and discuss the various emerging novel therapies for the major molecular subtypes of BC.  A brief account of the clinical development of inhibitors of poly(ADP-ribose) polymerase, cyclin-dependent kinases 4 and 6, phosphatidylinositol 3-kinase/protein kinase B/mammalian target of rapamycin pathway, histone deacetylation, multi-targeting tyrosine kinases, and immune checkpoints for personalized treatment of BC is included.  However, no targeted drug has been approved for the most aggressive subtype-triple negative breast cancer (TNBC).  Thus, we discuss the heterogeneity of TNBC and how molecular subtyping of TNBC may help drug discovery for this deadly disease.  The emergence of drug resistance also poses threat to the successful development of targeted therapy in various molecular subtypes of BC.  New clinical trials should incorporate advanced methods to identify changes induced by drug treatment, which may be associated with the upregulation of compensatory signaling pathways in drug resistant cancer cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcS9aYrYWlne0BJ5E29YNGvQfW6udTcc2ebWYQoNHaDokLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3c%252FjvVylsw%253D%253D&md5=a116dcf63d5640023aac066b920bdc5f</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.3389%2Ffonc.2018.00227&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffonc.2018.00227%26sid%3Dliteratum%253Aachs%26aulast%3DTong%26aufirst%3DC.%2BW.%2BS.%26aulast%3DWu%26aufirst%3DM.%26aulast%3DCho%26aufirst%3DW.%2BC.%2BS.%26aulast%3DTo%26aufirst%3DK.%2BK.%2BW.%26atitle%3DRecent%2520advances%2520in%2520the%2520treatment%2520of%2520breast%2520cancer%26jtitle%3DFront.%2520Oncol.%26date%3D2018%26volume%3D8%26spage%3D227%26doi%3D10.3389%2Ffonc.2018.00227" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ariazi, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ariazi, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cordera, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jordan, V. C.</span></span> <span> </span><span class="NLM_article-title">Estrogen receptors as therapeutic targets in breast cancer</span>. <i>Curr. Top. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">181</span>– <span class="NLM_lpage">202</span>, <span class="refDoi"> DOI: 10.2174/156802606776173483</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=10.2174%2F156802606776173483" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=16515478" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A528%3ADC%252BD28XkvFWnsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2006&pages=181-202&author=E.+A.+Ariaziauthor=J.+L.+Ariaziauthor=F.+Corderaauthor=V.+C.+Jordan&title=Estrogen+receptors+as+therapeutic+targets+in+breast+cancer&doi=10.2174%2F156802606776173483"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Estrogen receptors as therapeutic targets in breast cancer</span></div><div class="casAuthors">Ariazi, Eric A.; Ariazi, Jennifer L.; Cordera, Fernando; Jordan, V. Craig</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">181-202</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  The estrogen receptor α (ERα) has proven to be the single most important target in breast cancer over the last 30 years.  The use of the selective ER modulator (SERM) tamoxifen for the treatment and prevention of breast cancer has changed therapeutics.  The SERM Raloxifene, approved for the treatment of osteoporosis, lacks tamoxifen's increased risk for endometrial cancer and is being evaluated for the prevention of breast cancer.  Other SERMs approved or under development for use against breast cancer or osteoporosis include Toremifene, GW5638, GW7604 (the active metabolite of GW5638), Idoxifene, Lasofoxifene, Arzoxifene, Bazedoxifene, EM-800, and Acolbifene (the active metabolite of EM-800).  Aromatase inhibitors (AIs) have recently proven to be more efficacious than tamoxifen as first-line therapy, efficacious for second-line therapy (e.g. against tamoxifen-resistant disease), and useful for extended adjuvant therapy after tamoxifen.  The AIs include the nonsteroidal agents Letrozole and Anastrole, and the steroidal agent Exemestane.  The pure antiestrogen Fulvestrant has proven to be just as effective as AIs.  Other pure antiestrogens, ZK-703, ZK-253, RU 58668, and TAS-108, show great promise.  The development of resistance to endocrine therapy remains a clin. important problem, and lab. models based on human breast cancer cells grown as tumors in immune-compromised mice have led to important insights into this problem.  Progesterone receptor-neg. status of ER-pos. breast cancers may reflect altered growth factor receptor signaling, and helps to explain why this subclass of tumors exhibits lower response rates to tamoxifen compared to cancers typed progesterone receptor-pos.  Crosstalk among plasma membrane-localized ER, growth factor receptor signaling, and nuclear-localized ER provide further insights into antihormonal-resistant breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGph8MK1y7HLJrVg90H21EOLACvtfcHk0lgdILZz36w2tw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XkvFWnsrY%253D&md5=d5e3920a355d937d5bf69b6d71ef1a9b</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.2174%2F156802606776173483&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F156802606776173483%26sid%3Dliteratum%253Aachs%26aulast%3DAriazi%26aufirst%3DE.%2BA.%26aulast%3DAriazi%26aufirst%3DJ.%2BL.%26aulast%3DCordera%26aufirst%3DF.%26aulast%3DJordan%26aufirst%3DV.%2BC.%26atitle%3DEstrogen%2520receptors%2520as%2520therapeutic%2520targets%2520in%2520breast%2520cancer%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2006%26volume%3D6%26spage%3D181%26epage%3D202%26doi%3D10.2174%2F156802606776173483" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, H.-R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, T.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, K.-C.</span></span> <span> </span><span class="NLM_article-title">Functions and physiological roles of two types of estrogen receptors, ERα and ERβ, identified by estrogen receptor knockout mouse</span>. <i>Lab. Anim. Res.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">71</span>– <span class="NLM_lpage">76</span>, <span class="refDoi"> DOI: 10.5625/lar.2012.28.2.71</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=10.5625%2Flar.2012.28.2.71" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=22787479" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A280%3ADC%252BC38jptlOisQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2012&pages=71-76&author=H.-R.+Leeauthor=T.-H.+Kimauthor=K.-C.+Choi&title=Functions+and+physiological+roles+of+two+types+of+estrogen+receptors%2C+ER%CE%B1+and+ER%CE%B2%2C+identified+by+estrogen+receptor+knockout+mouse&doi=10.5625%2Flar.2012.28.2.71"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Functions and physiological roles of two types of estrogen receptors, ERα and ERβ, identified by estrogen receptor knockout mouse</span></div><div class="casAuthors">Lee Hye-Rim; Kim Tae-Hee; Choi Kyung-Chul</div><div class="citationInfo"><span class="NLM_cas:title">Laboratory animal research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">71-6</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Estrogens, a class of steroid hormones, regulate the growth, development, and physiology of the human reproductive system.  Estrogens also involve in the neuroendocrine, skeletal, adipogenesis, and cardiovascular systems.  Estrogen signaling pathways are selectively stimulated or inhibited depending on a balance between the activities of estrogen receptor (ER) α or ERβ in target organs.  ERs belong to the steroid hormone superfamily of nuclear receptors, which act as transcription factors after binding to estrogen.  The gene expression regulation by ERs is to modulate biological activities, such as reproductive organ development, bone modeling, cardiovascular system functioning, metabolism, and behavior in both females and males.  Understanding of the general physiological roles of ERs has been gained when estrogen levels were ablated by ovariectomy and then replenished by treatment with exogenous estrogen.  This technique is not sufficient to fully determine the exact function of estrogen signaling in general processes in living tissues.  However, a transgenic mouse model has been useful to study gene-specific functions.  ERα and ERβ have different biological functions, and knockout and transgenic animal models have distinct phenotypes.  Analysis of ERα and ERβ function using knockout mouse models has identified the roles of estrogen signaling in general physiologic processes.  Although transgenic mouse models do not always produce consistent results, they are the useful for studying the functions of these genes under specific pathological conditions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSOsUARkrX9ASEJCPKhJPuqfW6udTcc2eZ5MQZDaSOL5bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC38jptlOisQ%253D%253D&md5=efbea11cb9e6901b1ac35af2fc1b878a</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.5625%2Flar.2012.28.2.71&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.5625%252Flar.2012.28.2.71%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DH.-R.%26aulast%3DKim%26aufirst%3DT.-H.%26aulast%3DChoi%26aufirst%3DK.-C.%26atitle%3DFunctions%2520and%2520physiological%2520roles%2520of%2520two%2520types%2520of%2520estrogen%2520receptors%252C%2520ER%25CE%25B1%2520and%2520ER%25CE%25B2%252C%2520identified%2520by%2520estrogen%2520receptor%2520knockout%2520mouse%26jtitle%3DLab.%2520Anim.%2520Res.%26date%3D2012%26volume%3D28%26spage%3D71%26epage%3D76%26doi%3D10.5625%2Flar.2012.28.2.71" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spezia, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lei, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vuppalapati, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luu, H. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haydon, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, T.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, G.</span></span> <span> </span><span class="NLM_article-title">Breast cancer development and progression: risk factors, cancer stem cells, signaling pathways, genomics, and molecular pathogenesis</span>. <i>Genes. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">77</span>– <span class="NLM_lpage">106</span>, <span class="refDoi"> DOI: 10.1016/j.gendis.2018.05.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=10.1016%2Fj.gendis.2018.05.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=30258937" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A528%3ADC%252BC1cXpvVKrtLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2018&pages=77-106&author=Y.+Fengauthor=M.+Speziaauthor=S.+Huangauthor=C.+Yuanauthor=Z.+Zengauthor=L.+Zhangauthor=X.+Jiauthor=W.+Liuauthor=B.+Huangauthor=W.+Luoauthor=B.+Liuauthor=Y.+Leiauthor=S.+Duauthor=A.+Vuppalapatiauthor=H.+H.+Luuauthor=R.+C.+Haydonauthor=T.-C.+Heauthor=G.+Ren&title=Breast+cancer+development+and+progression%3A+risk+factors%2C+cancer+stem+cells%2C+signaling+pathways%2C+genomics%2C+and+molecular+pathogenesis&doi=10.1016%2Fj.gendis.2018.05.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Breast cancer development and progression: Risk factors, cancer stem cells, signaling pathways, genomics, and molecular pathogenesis</span></div><div class="casAuthors">Feng, Yixiao; Spezia, Mia; Huang, Shifeng; Yuan, Chengfu; Zeng, Zongyue; Zhang, Linghuan; Ji, Xiaojuan; Liu, Wei; Huang, Bo; Luo, Wenping; Liu, Bo; Lei, Yan; Du, Scott; Vuppalapati, Akhila; Luu, Hue H.; Haydon, Rex C.; He, Tong-Chuan; Ren, Guosheng</div><div class="citationInfo"><span class="NLM_cas:title">Genes and Diseases</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">77-106</span>CODEN:
                <span class="NLM_cas:coden">GDEIAV</span>;
        ISSN:<span class="NLM_cas:issn">2352-3042</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">As the most commonly occurring cancer in women worldwide, breast cancer poses a formidable public health challenge on a global scale.  Breast cancer consists of a group of biol. and molecularly heterogeneous diseases originated from the breast.  While the risk factors assocd. with this cancer varies with respect to other cancers, genetic predisposition, most notably mutations in BRCA1 or BRCA2 gene, is an important causative factor for this malignancy.  Breast cancers can begin in different areas of the breast, such as the ducts, the lobules, or the tissue in between.  Within the large group of diverse breast carcinomas, there are various denoted types of breast cancer based on their invasiveness relative to the primary tumor sites.  It is important to distinguish between the various subtypes because they have different prognoses and treatment implications.  As there are remarkable parallels between normal development and breast cancer progression at the mol. level, it has been postulated that breast cancer may be derived from mammary cancer stem cells.  Normal breast development and mammary stem cells are regulated by several signaling pathways, such as estrogen receptors (ERs), HER2, and Wnt/β-catenin signaling pathways, which control stem cell proliferation, cell death, cell differentiation, and cell motility.  Furthermore, emerging evidence indicates that epigenetic regulations and noncoding RNAs may play important roles in breast cancer development and may contribute to the heterogeneity and metastatic aspects of breast cancer, esp. for triple-neg. breast cancer.  This review provides a comprehensive survey of the mol., cellular and genetic aspects of breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpD6nESJp4DErVg90H21EOLACvtfcHk0lgdILZz36w2tw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXpvVKrtLc%253D&md5=f9ae3c050fc7cf2241919a7cca7b82a8</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.gendis.2018.05.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.gendis.2018.05.001%26sid%3Dliteratum%253Aachs%26aulast%3DFeng%26aufirst%3DY.%26aulast%3DSpezia%26aufirst%3DM.%26aulast%3DHuang%26aufirst%3DS.%26aulast%3DYuan%26aufirst%3DC.%26aulast%3DZeng%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DJi%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DW.%26aulast%3DHuang%26aufirst%3DB.%26aulast%3DLuo%26aufirst%3DW.%26aulast%3DLiu%26aufirst%3DB.%26aulast%3DLei%26aufirst%3DY.%26aulast%3DDu%26aufirst%3DS.%26aulast%3DVuppalapati%26aufirst%3DA.%26aulast%3DLuu%26aufirst%3DH.%2BH.%26aulast%3DHaydon%26aufirst%3DR.%2BC.%26aulast%3DHe%26aufirst%3DT.-C.%26aulast%3DRen%26aufirst%3DG.%26atitle%3DBreast%2520cancer%2520development%2520and%2520progression%253A%2520risk%2520factors%252C%2520cancer%2520stem%2520cells%252C%2520signaling%2520pathways%252C%2520genomics%252C%2520and%2520molecular%2520pathogenesis%26jtitle%3DGenes.%2520Dis.%26date%3D2018%26volume%3D5%26spage%3D77%26epage%3D106%26doi%3D10.1016%2Fj.gendis.2018.05.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Eckhardt, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Francis, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parker, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, R. L.</span></span> <span> </span><span class="NLM_article-title">Strategies for the discovery and development of therapies for metastatic breast cancer</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">479</span>– <span class="NLM_lpage">497</span>, <span class="refDoi"> DOI: 10.1038/nrd2372</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=10.1038%2Fnrd2372" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=22653217" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A528%3ADC%252BC38XnslOht7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2012&pages=479-497&author=B.+L.+Eckhardtauthor=P.+A.+Francisauthor=B.+S.+Parkerauthor=R.+L.+Anderson&title=Strategies+for+the+discovery+and+development+of+therapies+for+metastatic+breast+cancer&doi=10.1038%2Fnrd2372"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Strategies for the discovery and development of therapies for metastatic breast cancer</span></div><div class="casAuthors">Eckhardt, Bedrich L.; Francis, Prudence A.; Parker, Belinda S.; Anderson, Robin L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">479-497</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Nearly all deaths caused by solid cancers occur as a result of metastasis - the formation of secondary tumors in distant organs such as the lungs, liver, brain and bone.  A major obstruction to the development of drugs with anti-metastatic efficacy is our fragmented understanding of how tumors 'evolve' and metastasize, at both the biol. and genetic levels.  Furthermore, although there is significant overlap in the metastatic process among different types of cancer, there are also marked differences in the propensity to metastasize, the extent of metastasis, the sites to which the tumor metastasizes, the kinetics of the process and the mechanisms involved.  Here, we consider the case of breast cancer, which has some marked distinguishing features compared with other types of cancer.  Considerable progress has been made in the development of preclin. models and in the identification of relevant signalling pathways and genetic regulators of metastatic breast cancer, and we discuss how these might facilitate the development of novel targeted anti-metastatic drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKLJhOlAUBKbVg90H21EOLACvtfcHk0liPjvw8fgD6tg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XnslOht7c%253D&md5=f3b53f3ca5d65ccadce9bb35ef084640</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnrd2372&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2372%26sid%3Dliteratum%253Aachs%26aulast%3DEckhardt%26aufirst%3DB.%2BL.%26aulast%3DFrancis%26aufirst%3DP.%2BA.%26aulast%3DParker%26aufirst%3DB.%2BS.%26aulast%3DAnderson%26aufirst%3DR.%2BL.%26atitle%3DStrategies%2520for%2520the%2520discovery%2520and%2520development%2520of%2520therapies%2520for%2520metastatic%2520breast%2520cancer%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2012%26volume%3D11%26spage%3D479%26epage%3D497%26doi%3D10.1038%2Fnrd2372" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Patel, H. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bihani, T.</span></span> <span> </span><span class="NLM_article-title">Selective estrogen receptor modulators (SERMs) and selective estrogen receptor degraders (SERDs) in cancer treatment</span>. <i>Pharmacol. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>186</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">24</span>, <span class="refDoi"> DOI: 10.1016/j.pharmthera.2017.12.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=10.1016%2Fj.pharmthera.2017.12.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=29289555" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A528%3ADC%252BC1cXovVShtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=186&publication_year=2018&pages=1-24&author=H.+K.+Patelauthor=T.+Bihani&title=Selective+estrogen+receptor+modulators+%28SERMs%29+and+selective+estrogen+receptor+degraders+%28SERDs%29+in+cancer+treatment&doi=10.1016%2Fj.pharmthera.2017.12.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Selective estrogen receptor modulators (SERMs) and selective estrogen receptor degraders (SERDs) in cancer treatment</span></div><div class="casAuthors">Patel, Hitisha K.; Bihani, Teeru</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacology & Therapeutics</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">186</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1-24</span>CODEN:
                <span class="NLM_cas:coden">PHTHDT</span>;
        ISSN:<span class="NLM_cas:issn">0163-7258</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Breast cancer is the most frequently diagnosed cancer in women, with estrogen receptor pos. (ER+) breast cancer making up approx. 75% of all breast cancers diagnosed.  Given the dependence on active ER signaling in these tumors, the predominant treatment strategy has been to inhibit various aspects of this pathway including directly antagonizing ER with the use of selective estrogen receptor modulators (SERMs) and selective estrogen receptor degraders (SERDs).  Interestingly, the dependence on ER for breast cancer growth is often retained even after progression through several lines of antiestrogen therapy, making ER a bonafide biomarker for this cancer subtype and driving the continued research and development of novel ER-targeted therapeutics to treat this patient population.  This, combined with the continuous discovery of mechanisms underlying endocrine resistance, is resulting in a continually evolving treatment landscape for ER+ breast cancer.  This review discusses various ER antagonists investigated for the treatment of breast cancer, outlining their pharmacol. and tissue-specific mechanisms of action as well as their specified use within the ER+ breast cancer setting.  In addn., mechanisms of resistance to SERMs and SERDs, the use of ER antagonists in combination therapy strategies, and the ongoing development of novel drugs are also reviewed in the context of the changing clin. landscape of ER+ breast cancer.  Lastly, the role of SERMs and SERDs in non-breast cancer indications is also discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGre8o54q6VIOrVg90H21EOLACvtfcHk0liPjvw8fgD6tg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXovVShtw%253D%253D&md5=9e33ceed330cba071400fd70c0b71bf4</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.pharmthera.2017.12.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pharmthera.2017.12.012%26sid%3Dliteratum%253Aachs%26aulast%3DPatel%26aufirst%3DH.%2BK.%26aulast%3DBihani%26aufirst%3DT.%26atitle%3DSelective%2520estrogen%2520receptor%2520modulators%2520%2528SERMs%2529%2520and%2520selective%2520estrogen%2520receptor%2520degraders%2520%2528SERDs%2529%2520in%2520cancer%2520treatment%26jtitle%3DPharmacol.%2520Ther.%26date%3D2018%26volume%3D186%26spage%3D1%26epage%3D24%26doi%3D10.1016%2Fj.pharmthera.2017.12.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boisen, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andersen, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sreekumar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stern, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oesterreich, S.</span></span> <span> </span><span class="NLM_article-title">Treating gynecologic malignancies with selective estrogen receptor downregulators (SERDs): promise and challenges</span>. <i>Mol. Cell. Endocrinol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>418</i></span>,  <span class="NLM_fpage">322</span>– <span class="NLM_lpage">333</span>, <span class="refDoi"> DOI: 10.1016/j.mce.2015.04.035</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=10.1016%2Fj.mce.2015.04.035" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=26276546" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVajtrzF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=418&publication_year=2015&pages=322-333&author=M.+M.+Boisenauthor=C.+L.+Andersenauthor=S.+Sreekumarauthor=A.+M.+Sternauthor=S.+Oesterreich&title=Treating+gynecologic+malignancies+with+selective+estrogen+receptor+downregulators+%28SERDs%29%3A+promise+and+challenges&doi=10.1016%2Fj.mce.2015.04.035"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Treating gynecologic malignancies with selective estrogen receptor downregulators (SERDs): promise and challenges</span></div><div class="casAuthors">Boisen, Michelle M.; Andersen, Courtney L.; Sreekumar, Sreeja; Stern, Andrew M.; Oesterreich, Steffi</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Endocrinology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">418</span>
        (<span class="NLM_cas:issue">Part_3</span>),
    <span class="NLM_cas:pages">322-333</span>CODEN:
                <span class="NLM_cas:coden">MCEND6</span>;
        ISSN:<span class="NLM_cas:issn">0303-7207</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ireland Ltd.</span>)
        </div><div class="casAbstract">Endometrial and ovarian cancers are estrogen-dependent gynecol. malignancies.  Although many are estrogen receptor (ER) pos., treatment with the selective estrogen receptor modulator (SERM) tamoxifen, a tissue selective partial-agonist, has demonstrated only modest clin. benefit.  Selective estrogen receptor downregulators (SERDs) are pure ER antagonists showing a benefit for advanced ER pos. breast cancer, which has bolstered their potential use for ER pos. gynecol. malignancies.  We summarize these preclin. and clin. data, suggesting that a subpopulation of patients with endometrial or ovarian cancer exists in which treatment with SERDs results in improved outcome.  However, the full potential of SERDs for a gynecol. malignancies will be realized only when the appropriate predictive biomarkers are identified.  Addnl., a further understanding ER signaling in the context of ovarian and endometrial tissues that appear to involve c-Src and other kinase pathways is needed to successfully address the emergence of resistance with rationally designed combination therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgqSQ46Mh85rVg90H21EOLACvtfcHk0liPjvw8fgD6tg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVajtrzF&md5=e0c259cf751f2acc6a368a22dc8d38af</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.mce.2015.04.035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.mce.2015.04.035%26sid%3Dliteratum%253Aachs%26aulast%3DBoisen%26aufirst%3DM.%2BM.%26aulast%3DAndersen%26aufirst%3DC.%2BL.%26aulast%3DSreekumar%26aufirst%3DS.%26aulast%3DStern%26aufirst%3DA.%2BM.%26aulast%3DOesterreich%26aufirst%3DS.%26atitle%3DTreating%2520gynecologic%2520malignancies%2520with%2520selective%2520estrogen%2520receptor%2520downregulators%2520%2528SERDs%2529%253A%2520promise%2520and%2520challenges%26jtitle%3DMol.%2520Cell.%2520Endocrinol.%26date%3D2015%26volume%3D418%26spage%3D322%26epage%3D333%26doi%3D10.1016%2Fj.mce.2015.04.035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Johnston, S. R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dowsett, M.</span></span> <span> </span><span class="NLM_article-title">Aromatase inhibitors for breast cancer: lessons from the laboratory</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">821</span>– <span class="NLM_lpage">831</span>, <span class="refDoi"> DOI: 10.1038/nrc1211</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=10.1038%2Fnrc1211" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=14668813" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A528%3ADC%252BD3sXptFKmur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2003&pages=821-831&author=S.+R.+D.+Johnstonauthor=M.+Dowsett&title=Aromatase+inhibitors+for+breast+cancer%3A+lessons+from+the+laboratory&doi=10.1038%2Fnrc1211"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Aromatase inhibitors for breast cancer: Lessons from the laboratory</span></div><div class="casAuthors">Johnston, Stephen R. D.; Dowsett, Mitch</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">821-831</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Endocrine therapy with tamoxifen has been the mainstay of treatment for hormone-sensitive breast cancer for more than 20 yr.  An alternative strategy of estrogen deprivation in postmenopausal women with aromatase inhibitors is set to replace tamoxifen based on better efficacy and a delay in the emergence of endocrine resistance.  There are fundamental differences in how tamoxifen and aromatase inhibitors alter estrogen-receptor signaling.  Understanding the response and resistance to these different therapies is central to further improving therapeutic options for women with breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0yq8HDYSRf7Vg90H21EOLACvtfcHk0ljPKY8bUI0-Vg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXptFKmur8%253D&md5=5e187fa66d3912a9c5b151c3c54fb251</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fnrc1211&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc1211%26sid%3Dliteratum%253Aachs%26aulast%3DJohnston%26aufirst%3DS.%2BR.%2BD.%26aulast%3DDowsett%26aufirst%3DM.%26atitle%3DAromatase%2520inhibitors%2520for%2520breast%2520cancer%253A%2520lessons%2520from%2520the%2520laboratory%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2003%26volume%3D3%26spage%3D821%26epage%3D831%26doi%3D10.1038%2Fnrc1211" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McDonnell, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wardell, S. E.</span></span> <span> </span><span class="NLM_article-title">The molecular mechanisms underlying the pharmacological actions of ER modulators: implications for new drug discovery in breast cancer</span>. <i>Curr. Opin. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">620</span>– <span class="NLM_lpage">628</span>, <span class="refDoi"> DOI: 10.1016/j.coph.2010.09.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=10.1016%2Fj.coph.2010.09.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=20926342" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVals73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2010&pages=620-628&author=D.+P.+McDonnellauthor=S.+E.+Wardell&title=The+molecular+mechanisms+underlying+the+pharmacological+actions+of+ER+modulators%3A+implications+for+new+drug+discovery+in+breast+cancer&doi=10.1016%2Fj.coph.2010.09.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">The molecular mechanisms underlying the pharmacological actions of ER modulators: implications for new drug discovery in breast cancer</span></div><div class="casAuthors">McDonnell, Donald P.; Wardell, Suzanne E.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Pharmacology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">620-628</span>CODEN:
                <span class="NLM_cas:coden">COPUBK</span>;
        ISSN:<span class="NLM_cas:issn">1471-4892</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Our understanding of the mol. mechanisms underlying the pharmacol. actions of estrogen receptor (ER) ligands has evolved considerably in recent years.  Much of this knowledge has come from a detailed dissection of the mechanism(s) of action of the Selective Estrogen Receptor Modulators (SERMs) tamoxifen and raloxifene, so called for their ability to function as ER agonists or antagonists depending on the tissue in which they operate.  These mechanistic insights have had a significant impact on the discovery of second generation SERMs, some of which are in late stage clin. development for the treatment/prevention of breast cancer as well as other estrogenopathies.  In addn. to the SERMs, however, have emerged the Selective Estrogen Degraders (SERDs), which as their name suggests, interact with and facilitate ER turnover in cells.  One drug of this class, fulvestrant, has been approved as a third line treatment for ER-pos. metastatic breast cancer.  Whereas the first generation SERMs/SERDs were discovered in a serendipitous manner, this review will highlight how our understanding of the mol. pharmacol. of ER ligands has been utilized in the development of the next generation of SERMs/SERDs, some of which are likely to have a major impact on the pharmacotherapy of breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8b2l5p-n0bbVg90H21EOLACvtfcHk0ljPKY8bUI0-Vg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsVals73P&md5=24d1490ca0b0570c21e9768c9de134d6</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.coph.2010.09.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.coph.2010.09.007%26sid%3Dliteratum%253Aachs%26aulast%3DMcDonnell%26aufirst%3DD.%2BP.%26aulast%3DWardell%26aufirst%3DS.%2BE.%26atitle%3DThe%2520molecular%2520mechanisms%2520underlying%2520the%2520pharmacological%2520actions%2520of%2520ER%2520modulators%253A%2520implications%2520for%2520new%2520drug%2520discovery%2520in%2520breast%2520cancer%26jtitle%3DCurr.%2520Opin.%2520Pharmacol.%26date%3D2010%26volume%3D10%26spage%3D620%26epage%3D628%26doi%3D10.1016%2Fj.coph.2010.09.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cuzick, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forbes, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sestak, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cawthorn, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamed, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holli, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howell, A.</span></span> <span> </span><span class="NLM_article-title">Long-term results of tamoxifen prophylaxis for breast cancer-96-month follow-up of the randomized IBIS-I trial</span>. <i>J. Natl. Cancer Inst.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>99</i></span>,  <span class="NLM_fpage">272</span>– <span class="NLM_lpage">282</span>, <span class="refDoi"> DOI: 10.1093/jnci/djk049</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=10.1093%2Fjnci%2Fdjk049" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=17312304" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A528%3ADC%252BD2sXjsVeitrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=99&publication_year=2007&pages=272-282&author=J.+Cuzickauthor=J.+F.+Forbesauthor=I.+Sestakauthor=S.+Cawthornauthor=H.+Hamedauthor=K.+Holliauthor=A.+Howell&title=Long-term+results+of+tamoxifen+prophylaxis+for+breast+cancer-96-month+follow-up+of+the+randomized+IBIS-I+trial&doi=10.1093%2Fjnci%2Fdjk049"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Long-term results of tamoxifen prophylaxis for breast cancer-96-month follow-up of the randomized IBIS-I trial</span></div><div class="casAuthors">Cuzick, Jack; Forbes, John F.; Sestak, Ivana; Cawthorn, Simon; Hamed, Hisham; Holli, Kaija; Howell, Anthony</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the National Cancer Institute</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">99</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">272-282</span>CODEN:
                <span class="NLM_cas:coden">JNCIEQ</span>;
        ISSN:<span class="NLM_cas:issn">0027-8874</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Background: Initial results from the first International Breast Cancer Intervention Study (IBIS-I) found that tamoxifen reduced the risk of invasive estrogen receptor (ER)-pos. tumors by 31% in women at increased risk for breast cancer, but most of the follow-up at this time was during the active treatment phase.  We report an updated anal. of IBIS-I that focuses on the period after active treatment was completed, a time for which little evidence from other trials is available.  Methods: A total of 7145 women who were aged 35-70 years and at increased risk of breast cancer were randomly assigned to receive either tamoxifen (20 mg/day) or placebo for 5 years.  The primary outcome measure was the incidence of breast cancer (including ductal carcinoma in situ), but side effects were also investigated.  Relative risks were computed as the ratio of incidence rates.  All statistical tests were two-sided.  Results: After a median follow-up of 96 mo after randomization, 142 breast cancers were diagnosed in the 3579 women in the tamoxifen group and 195 in the 3575 women in the placebo group (4.97 vs. 6.82 per 1000 woman-years, resp.; risk ratio [RR] = 0.73, 95% confidence interval [CI] = 0.58 to 0.91, P = .004).  The prophylactic effect of tamoxifen was fairly const. for the entire follow-up period, and no diminution of benefit was obsd. for up to 10 years after randomization.  However, side effects in the tamoxifen group were much lower after completion of the active treatment period than during active treatment.  For example, deep-vein thrombosis and pulmonary embolism were statistically significantly higher in the tamoxifen arm than in the placebo arm during active treatment (52 vs. 23 cases, RR = 2.26, 95% CI = 1.36 to 3.87) but not after tamoxifen was stopped (16 vs. 14 cases, RR = 1.14, 95% CI = 0.52 to 2.53).  The two arms did not differ in the risk of ER-neg. invasive tumors (35 in each arm, RR = 1.00, 95% CI = 0.61 to 1.65) across the entire follow-up period, but the risk of ER-pos. invasive breast cancer was 34% lower in the tamoxifen arm (87 vs. 132 cases, RR = 0.66, 95% CI = 0.50 to 0.87).  Conclusions: The risk-reducing effect of tamoxifen appears to persist for at least 10 years, but most side effects of tamoxifen do not continue after the 5-yr treatment period.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdtmvBUpbSlbVg90H21EOLACvtfcHk0ljPKY8bUI0-Vg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXjsVeitrs%253D&md5=e2e23ea674e36864f76d7cd5e8e98f3d</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1093%2Fjnci%2Fdjk049&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjnci%252Fdjk049%26sid%3Dliteratum%253Aachs%26aulast%3DCuzick%26aufirst%3DJ.%26aulast%3DForbes%26aufirst%3DJ.%2BF.%26aulast%3DSestak%26aufirst%3DI.%26aulast%3DCawthorn%26aufirst%3DS.%26aulast%3DHamed%26aufirst%3DH.%26aulast%3DHolli%26aufirst%3DK.%26aulast%3DHowell%26aufirst%3DA.%26atitle%3DLong-term%2520results%2520of%2520tamoxifen%2520prophylaxis%2520for%2520breast%2520cancer-96-month%2520follow-up%2520of%2520the%2520randomized%2520IBIS-I%2520trial%26jtitle%3DJ.%2520Natl.%2520Cancer%2520Inst.%26date%3D2007%26volume%3D99%26spage%3D272%26epage%3D282%26doi%3D10.1093%2Fjnci%2Fdjk049" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cuzick, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sestak, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baum, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buzdar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howell, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dowsett, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forbes, J. F.</span></span> <span> </span><span class="NLM_article-title">Effect of anastrozole and tamoxifen as adjuvant treatment for early-stage breast cancer: 10-year analysis of the ATAC trial</span>. <i>Lancet Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">1135</span>– <span class="NLM_lpage">1141</span>, <span class="refDoi"> DOI: 10.1016/S1470-2045(10)70257-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=10.1016%2FS1470-2045%2810%2970257-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=21087898" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVyhsLnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2010&pages=1135-1141&author=J.+Cuzickauthor=I.+Sestakauthor=M.+Baumauthor=A.+Buzdarauthor=A.+Howellauthor=M.+Dowsettauthor=J.+F.+Forbes&title=Effect+of+anastrozole+and+tamoxifen+as+adjuvant+treatment+for+early-stage+breast+cancer%3A+10-year+analysis+of+the+ATAC+trial&doi=10.1016%2FS1470-2045%2810%2970257-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Effect of anastrozole and tamoxifen as adjuvant treatment for early-stage breast cancer: 10-year analysis of the ATAC trial</span></div><div class="casAuthors">Cuzick, Jack; Sestak, Ivana; Baum, Michael; Buzdar, Aman; Howell, Anthony; Dowsett, Mitch; Forbes, John F.</div><div class="citationInfo"><span class="NLM_cas:title">Lancet Oncology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1135-1141</span>CODEN:
                <span class="NLM_cas:coden">LOANBN</span>;
        ISSN:<span class="NLM_cas:issn">1470-2045</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The Arimidex, Tamoxifen, Alone or in Combination (ATAC) trial was designed to compare the efficacy and safety of anastrozole (1 mg) with tamoxifen (20 mg), both given orally every day for 5 years, as adjuvant treatment for postmenopausal women with early-stage breast cancer.  In this anal., we assess the long-term outcomes after a median follow-up of 120 mo.  We used a proportional hazards model to assess the primary endpoint of disease-free survival, and the secondary endpoints of time to recurrence, time to distant recurrence, incidence of new contralateral breast cancer, overall survival, and death with or without recurrence in all randomized patients (anastrozole n = 3125, tamoxifen n = 3116) and hormone-receptor-pos. patients (anastrozole n = 2618, tamoxifen n = 2598).  After treatment completion, we continued to collect data on fractures and serious adverse events in a masked fashion (safety population: anastrozole n = 3092, tamoxifen n = 3094).  This study is registered as an , no. .  Patients were followed up for a median of 120 mo (range 0-145); there were 24 522 woman-years of follow-up in the anastrozole group and 23 950 woman-years in the tamoxifen group.  In the full study population, there were significant improvements in the anastrozole group compared with the tamoxifen group for disease-free survival (hazard ratio [HR] 0.91, 95% CI 0.83-0.99; p = 0.04), time to recurrence (0.84, 0.75-0.93; p = 0.001), and time to distant recurrence (0.87, 0.77-0.99; p = 0.03).  For hormone-receptor-pos. patients, the results were also significantly in favor of the anastrozole group for disease-free survival (HR 0.86, 95% CI 0.78-0.95; p = 0.003), time to recurrence (0.79, 0.70-0.89; p = 0.0002), and time to distant recurrence (0.85, 0.73-0.98; p = 0.02).  In hormone-receptor-pos. patients, abs. differences in time to recurrence between anastrozole and tamoxifen increased over time (2.7% at 5 years and 4.3% at 10 years) and recurrence rates remained significantly lower on anastrozole than tamoxifen after treatment completion (HR 0.81, 95% CI 0.67-0.98; p = 0.03), although the carryover benefit was smaller after 8 years.  There was weak evidence of fewer deaths after recurrence with anastrozole compared with tamoxifen treatment in the hormone-receptor-pos. subgroup (HR 0.87, 95% CI 0.74-1.02; p = 0.09), but there was little difference in overall mortality (0.95, 95% CI 0.84-1.06; p = 0.4).  Fractures were more frequent during active treatment in patients receiving anastrozole than those receiving tamoxifen (451 vs 351; OR 1.33, 95% CI 1.15-1.55; p<0.0001), but were similar in the post-treatment follow-up period (110 vs 112; OR 0.98, 95% CI 0.74-1.30; p = 0.9).  Treatment-related serious adverse events were less common in the anastrozole group than the tamoxifen group (223 anastrozole vs 369 tamoxifen; OR 0.57, 95% CI 0.48-0.69; p<0.0001), but were similar after treatment completion (66 vs 78; OR 0.84, 95% CI 0.60-1.19; p = 0.3).  No differences in non-breast cancer causes of death were apparent and the incidence of other cancers was similar between groups (425 vs 431) and continue to be higher with anastrozole for colorectal (66 vs 44) and lung cancer (51 vs 34), and lower for endometrial cancer (six vs 24), melanoma (eight vs 19), and ovarian cancer (17 vs 28).  No new safety concerns were reported.  These data confirm the long-term superior efficacy and safety of anastrozole over tamoxifen as initial adjuvant therapy for postmenopausal women with hormone-sensitive early breast cancer.  Funding: AstraZeneca.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkXdPIpkfRNrVg90H21EOLACvtfcHk0lhdBER6bxXZnw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsVyhsLnF&md5=63f3920eb78b3c4d3dea4ef91b4bc53c</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2FS1470-2045%2810%2970257-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1470-2045%252810%252970257-6%26sid%3Dliteratum%253Aachs%26aulast%3DCuzick%26aufirst%3DJ.%26aulast%3DSestak%26aufirst%3DI.%26aulast%3DBaum%26aufirst%3DM.%26aulast%3DBuzdar%26aufirst%3DA.%26aulast%3DHowell%26aufirst%3DA.%26aulast%3DDowsett%26aufirst%3DM.%26aulast%3DForbes%26aufirst%3DJ.%2BF.%26atitle%3DEffect%2520of%2520anastrozole%2520and%2520tamoxifen%2520as%2520adjuvant%2520treatment%2520for%2520early-stage%2520breast%2520cancer%253A%252010-year%2520analysis%2520of%2520the%2520ATAC%2520trial%26jtitle%3DLancet%2520Oncol.%26date%3D2010%26volume%3D11%26spage%3D1135%26epage%3D1141%26doi%3D10.1016%2FS1470-2045%2810%2970257-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nilsson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gustafsson, J. A.</span></span> <span> </span><span class="NLM_article-title">Estrogen receptors: therapies targeted to receptor subtypes</span>. <i>Clin. Pharmacol. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>89</i></span>,  <span class="NLM_fpage">44</span>– <span class="NLM_lpage">55</span>, <span class="refDoi"> DOI: 10.1038/clpt.2010.226</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=10.1038%2Fclpt.2010.226" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=21124311" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsFylt7zN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=89&publication_year=2011&pages=44-55&author=S.+Nilssonauthor=J.+A.+Gustafsson&title=Estrogen+receptors%3A+therapies+targeted+to+receptor+subtypes&doi=10.1038%2Fclpt.2010.226"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Estrogen Receptors: Therapies Targeted to Receptor Subtypes</span></div><div class="casAuthors">Nilsson, S.; Gustafsson, J.-Aa.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Pharmacology & Therapeutics (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">89</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">44-55</span>CODEN:
                <span class="NLM_cas:coden">CLPTAT</span>;
        ISSN:<span class="NLM_cas:issn">0009-9236</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Over the past two decades, we have learned that estrogens play important physiol. roles not only in women but also in men and that the biol. effects of estrogen are mediated by not one but two distinct estrogen receptors (ERs), ERα and ERβ.  Our appreciation of the physiol. importance of estrogen and the mechanisms by which it acts has significantly increased over the years; however, we are only now beginning to decipher the roles of ERα and ERβ in different organs and to elucidate how selective ligands, acting through either of the two ERs, can prevent or treat various age- or sex-specific diseases.  The specific roles of ERα and ERβ and the therapeutic potential of ER subtype-selective agonists in bone and metabolic homeostasis, depression, vasomotor symptoms, neurodegenerative diseases, and cancer are reviewed herein.  It must be stated, however, that appropriate clin. studies are necessary to validate these compds. as agents for the prevention and treatment of diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqq0tQWjd5LTLVg90H21EOLACvtfcHk0lhdBER6bxXZnw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsFylt7zN&md5=30f4fac2dcdfb20035b54783a7684a8c</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2Fclpt.2010.226&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fclpt.2010.226%26sid%3Dliteratum%253Aachs%26aulast%3DNilsson%26aufirst%3DS.%26aulast%3DGustafsson%26aufirst%3DJ.%2BA.%26atitle%3DEstrogen%2520receptors%253A%2520therapies%2520targeted%2520to%2520receptor%2520subtypes%26jtitle%3DClin.%2520Pharmacol.%2520Ther.%26date%3D2011%26volume%3D89%26spage%3D44%26epage%3D55%26doi%3D10.1038%2Fclpt.2010.226" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Toy, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weir, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Razavi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goeppert, A. U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mazzola, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morrow, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Stanchina, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlson, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uddin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fanning, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katzenellenbogen, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greene, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baselga, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chandarlapaty, S.</span></span> <span> </span><span class="NLM_article-title">Activating ESR1 mutations differentially affect the efficacy of ER antagonists</span>. <i>Cancer Discovery</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">277</span>– <span class="NLM_lpage">287</span>, <span class="refDoi"> DOI: 10.1158/2159-8290.CD-15-1523</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=10.1158%2F2159-8290.CD-15-1523" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=27986707" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjs1Klsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&pages=277-287&author=W.+Toyauthor=H.+Weirauthor=P.+Razaviauthor=M.+Lawsonauthor=A.+U.+Goeppertauthor=A.+M.+Mazzolaauthor=A.+Smithauthor=J.+Wilsonauthor=C.+Morrowauthor=W.+L.+Wongauthor=E.+De+Stanchinaauthor=K.+E.+Carlsonauthor=T.+S.+Martinauthor=S.+Uddinauthor=Z.+Liauthor=S.+Fanningauthor=J.+A.+Katzenellenbogenauthor=G.+Greeneauthor=J.+Baselgaauthor=S.+Chandarlapaty&title=Activating+ESR1+mutations+differentially+affect+the+efficacy+of+ER+antagonists&doi=10.1158%2F2159-8290.CD-15-1523"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Activating ESR1 Mutations Differentially Affect the Efficacy of ER Antagonists</span></div><div class="casAuthors">Toy, Weiyi; Weir, Hazel; Razavi, Pedram; Lawson, Mandy; Goeppert, Anne U.; Mazzola, Anne Marie; Smith, Aaron; Wilson, Joanne; Morrow, Christopher; Wong, Wai Lin; De Stanchina, Elisa; Carlson, Kathryn E.; Martin, Teresa S.; Uddin, Sharmeen; Li, Zhiqiang; Fanning, Sean; Katzenellenbogen, John A.; Greene, Geoffrey; Baselga, Jose; Chandarlapaty, Sarat</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">277-287</span>CODEN:
                <span class="NLM_cas:coden">CDAIB2</span>;
        ISSN:<span class="NLM_cas:issn">2159-8274</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Recent studies have identified somatic ESR1 mutations in patients with metastatic breast cancer and found some of them to promote estrogen-independent activation of the receptor.  The degree to which all recurrent mutants can drive estrogen-independent activities and reduced sensitivity to ER antagonists like fulvestrant is not established.  In this report, we characterize the spectrum of ESR1 mutations from more than 900 patients.  ESR1 mutations were detected in 10%, with D538G being the most frequent (36%), followed by Y537S (14%).  Several novel, activating mutations were also detected (e.g., L469V, V422del, and Y537D).  Although many mutations lead to constitutive activity and reduced sensitivity to ER antagonists, only select mutants such as Y537S caused a magnitude of change assocd. with fulvestrant resistance in vivo.  Correspondingly, tumors driven by Y537S, but not D5358G, E380Q, or S463P, were less effectively inhibited by fulvestrant than more potent and bioavailable antagonists, including AZD9496.  These data point to a need for antagonists with optimal pharmacokinetic properties to realize clin. efficacy against certain ESR1 mutants.  Significance: A diversity of activating ESR1 mutations exist, only some of which confer resistance to existing ER antagonists that might be overcome by next-generation inhibitors such as AZD9496.  Cancer Discov; 7(3); 277-87. ©2016 AACR.  This article is highlighted in the In This Issue feature, p.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZyS-eS6T9ZLVg90H21EOLACvtfcHk0lhdBER6bxXZnw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjs1Klsbo%253D&md5=398dd5202db358ba8facab67055895c9</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1158%2F2159-8290.CD-15-1523&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F2159-8290.CD-15-1523%26sid%3Dliteratum%253Aachs%26aulast%3DToy%26aufirst%3DW.%26aulast%3DWeir%26aufirst%3DH.%26aulast%3DRazavi%26aufirst%3DP.%26aulast%3DLawson%26aufirst%3DM.%26aulast%3DGoeppert%26aufirst%3DA.%2BU.%26aulast%3DMazzola%26aufirst%3DA.%2BM.%26aulast%3DSmith%26aufirst%3DA.%26aulast%3DWilson%26aufirst%3DJ.%26aulast%3DMorrow%26aufirst%3DC.%26aulast%3DWong%26aufirst%3DW.%2BL.%26aulast%3DDe%2BStanchina%26aufirst%3DE.%26aulast%3DCarlson%26aufirst%3DK.%2BE.%26aulast%3DMartin%26aufirst%3DT.%2BS.%26aulast%3DUddin%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DFanning%26aufirst%3DS.%26aulast%3DKatzenellenbogen%26aufirst%3DJ.%2BA.%26aulast%3DGreene%26aufirst%3DG.%26aulast%3DBaselga%26aufirst%3DJ.%26aulast%3DChandarlapaty%26aufirst%3DS.%26atitle%3DActivating%2520ESR1%2520mutations%2520differentially%2520affect%2520the%2520efficacy%2520of%2520ER%2520antagonists%26jtitle%3DCancer%2520Discovery%26date%3D2017%26volume%3D7%26spage%3D277%26epage%3D287%26doi%3D10.1158%2F2159-8290.CD-15-1523" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Musgrove, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sutherland, R. L.</span></span> <span> </span><span class="NLM_article-title">Biological determinants of endocrine resistance in breast cancer</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">631</span>– <span class="NLM_lpage">643</span>, <span class="refDoi"> DOI: 10.1038/nrc2713</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=10.1038%2Fnrc2713" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=19701242" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtVegtrzL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2009&pages=631-643&author=E.+A.+Musgroveauthor=R.+L.+Sutherland&title=Biological+determinants+of+endocrine+resistance+in+breast+cancer&doi=10.1038%2Fnrc2713"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Biological determinants of endocrine resistance in breast cancer</span></div><div class="casAuthors">Musgrove, Elizabeth A.; Sutherland, Robert L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">631-643</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Endocrine therapies targeting estrogen action (anti-estrogens, such as tamoxifen, and aromatase inhibitors) decrease mortality from breast cancer, but their efficacy is limited by intrinsic and acquired therapeutic resistance.  Candidate mol. biomarkers and gene expression signatures of tamoxifen response emphasize the importance of deregulation of proliferation and survival signaling in endocrine resistance.  However, definition of the specific genetic lesions and mol. processes that det. clin. endocrine resistance is incomplete.  The development of large-scale computational and genetic approaches offers the promise of identifying the mediators of endocrine resistance that may be exploited as potential therapeutic targets and biomarkers of response in the clinic.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbi_1MVsJjz7Vg90H21EOLACvtfcHk0lg61pqZqlmvEg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtVegtrzL&md5=e57a6a082e0f41d26422350657201de5</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1038%2Fnrc2713&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc2713%26sid%3Dliteratum%253Aachs%26aulast%3DMusgrove%26aufirst%3DE.%2BA.%26aulast%3DSutherland%26aufirst%3DR.%2BL.%26atitle%3DBiological%2520determinants%2520of%2520endocrine%2520resistance%2520in%2520breast%2520cancer%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2009%26volume%3D9%26spage%3D631%26epage%3D643%26doi%3D10.1038%2Fnrc2713" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mills, J. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rutkovsky, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giordano, A.</span></span> <span> </span><span class="NLM_article-title">Mechanisms of resistance in estrogen receptor positive breast cancer: overcoming resistance to tamoxifen/aromatase inhibitors</span>. <i>Curr. Opin. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">59</span>– <span class="NLM_lpage">65</span>, <span class="refDoi"> DOI: 10.1016/j.coph.2018.04.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=10.1016%2Fj.coph.2018.04.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=29719270" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A528%3ADC%252BC1cXotlOksbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2018&pages=59-65&author=J.+N.+Millsauthor=A.+C.+Rutkovskyauthor=A.+Giordano&title=Mechanisms+of+resistance+in+estrogen+receptor+positive+breast+cancer%3A+overcoming+resistance+to+tamoxifen%2Faromatase+inhibitors&doi=10.1016%2Fj.coph.2018.04.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanisms of resistance in estrogen receptor positive breast cancer: overcoming resistance to tamoxifen/aromatase inhibitors</span></div><div class="casAuthors">Mills, Jamie N.; Rutkovsky, Alex C.; Giordano, Antonio</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Pharmacology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">59-65</span>CODEN:
                <span class="NLM_cas:coden">COPUBK</span>;
        ISSN:<span class="NLM_cas:issn">1471-4892</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Several mechanisms of resistance have been identified, underscoring the complex nature of estrogen receptor (ER) signaling and the many connections between this pathway and other essential signaling pathways in breast cancer cells.  Many therapeutic targets of cell signaling and cell cycle pathways have met success with endocrine therapy and remain an ongoing area of investigation.  This review focuses on two major pathways that have recently emerged as important opportunities for therapeutic intervention in endocrine resistant breast tumors: PI3K/AKT/mTOR cell signaling and cyclinD1/cyclin-dependent kinase 4/6 cell cycle pathways.  Addnl., we highlight individual and combination strategies in current clin. trials that target these pathways and others under investigation for the treatment of ER pos. breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqVYQkU5BQS2LVg90H21EOLACvtfcHk0lg61pqZqlmvEg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXotlOksbg%253D&md5=7eb380dd5c8e441b8dd2602b40aa94b5</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.coph.2018.04.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.coph.2018.04.009%26sid%3Dliteratum%253Aachs%26aulast%3DMills%26aufirst%3DJ.%2BN.%26aulast%3DRutkovsky%26aufirst%3DA.%2BC.%26aulast%3DGiordano%26aufirst%3DA.%26atitle%3DMechanisms%2520of%2520resistance%2520in%2520estrogen%2520receptor%2520positive%2520breast%2520cancer%253A%2520overcoming%2520resistance%2520to%2520tamoxifen%252Faromatase%2520inhibitors%26jtitle%3DCurr.%2520Opin.%2520Pharmacol.%26date%3D2018%26volume%3D41%26spage%3D59%26epage%3D65%26doi%3D10.1016%2Fj.coph.2018.04.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boer, K.</span></span> <span> </span><span class="NLM_article-title">Fulvestrant in advanced breast cancer: evidence to date and place in therapy</span>. <i>Ther. Adv. Med. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">465</span>– <span class="NLM_lpage">479</span>, <span class="refDoi"> DOI: 10.1177/1758834017711097</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=10.1177%2F1758834017711097" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=28717399" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvVSntrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2017&pages=465-479&author=K.+Boer&title=Fulvestrant+in+advanced+breast+cancer%3A+evidence+to+date+and+place+in+therapy&doi=10.1177%2F1758834017711097"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Fulvestrant in advanced breast cancer: evidence to date and place in therapy</span></div><div class="casAuthors">Boer, Katalin</div><div class="citationInfo"><span class="NLM_cas:title">Therapeutic Advances in Medical Oncology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">465-479</span>CODEN:
                <span class="NLM_cas:coden">TAMOB8</span>;
        ISSN:<span class="NLM_cas:issn">1758-8340</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications Ltd.</span>)
        </div><div class="casAbstract">A review.  Breast cancer is a classical hormone-dependent tumor; therefore, endocrine therapy is the mainstay of treatment for hormone receptor-pos., human epidermal growth factor 2-neg. advanced breast cancer.  Until recently, classical endocrine agents such as tamoxifen, steroidal and nonsteroidal aromatase inhibitors and fulvestrant have been widely used in postmenopausal patients to treat locally advanced or metastatic disease.  However, for patients with this subtype of breast cancer, the landscape of endocrine therapy is rapidly changing.  Therapies targeting estrogen modulation have evolved in recent years following the introduction of targeted agents, mTOR and CDK 4/6 inhibitors that are administered in combination with hormone therapy.  As a result, options for endocrine therapy have expanded in recent years, and a variety of single-agent or combinations of targeted drugs and endocrine therapies are accepted.  Fulvestrant is a selective estrogen receptor downregulator (SERD) which was introduced to clin. practice in 2002, initially with the indication to treat postmenopausal women with hormone-receptor-pos. advanced breast cancer as second-line therapy postdisease progression after aromatase inhibitors or tamoxifen.  Addnl., fulvestrant has also been shown to be active in patients previously untreated with endocrine therapy, either both in the neoadjuvant and the metastatic setting, alone or in combination with other targeted therapies.  Currently, the std. dose is 500 mg, which is administered with a loading dose.  Fulvestrant received a new FDA indication in Dec. 2016, in combination with palbociclib, both in pre/peri/postmenopausal women with breast cancer progressing after endocrine therapy.  This manuscript aims to give an overview of new efficacy data and the current role of fulvestrant in the systemic therapy of hormone-receptor-pos. advanced breast cancer, in the context of other available therapeutic modalities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpRi8P0FTgbVrVg90H21EOLACvtfcHk0lg61pqZqlmvEg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvVSntrs%253D&md5=c8ce774327a3523edabfe23c557c5a19</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1177%2F1758834017711097&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1758834017711097%26sid%3Dliteratum%253Aachs%26aulast%3DBoer%26aufirst%3DK.%26atitle%3DFulvestrant%2520in%2520advanced%2520breast%2520cancer%253A%2520evidence%2520to%2520date%2520and%2520place%2520in%2520therapy%26jtitle%3DTher.%2520Adv.%2520Med.%2520Oncol.%26date%3D2017%26volume%3D9%26spage%3D465%26epage%3D479%26doi%3D10.1177%2F1758834017711097" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mehta, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barlow, W. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albain, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vandenberg, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dakhil, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tirumali, N. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lew, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayes, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gralow, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Livingston, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hortobagyi, G. N.</span></span> <span> </span><span class="NLM_article-title">Combination anastrozole and fulvestrant in metastatic breast cancer</span>. <i>N. Engl. J. Med.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>367</i></span>,  <span class="NLM_fpage">435</span>– <span class="NLM_lpage">444</span>, <span class="refDoi"> DOI: 10.1056/NEJMoa1201622</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=10.1056%2FNEJMoa1201622" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=22853014" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtlSltrbF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=367&publication_year=2012&pages=435-444&author=R.+S.+Mehtaauthor=W.+E.+Barlowauthor=K.+S.+Albainauthor=T.+A.+Vandenbergauthor=S.+R.+Dakhilauthor=N.+R.+Tirumaliauthor=D.+L.+Lewauthor=D.+F.+Hayesauthor=J.+R.+Gralowauthor=R.+B.+Livingstonauthor=G.+N.+Hortobagyi&title=Combination+anastrozole+and+fulvestrant+in+metastatic+breast+cancer&doi=10.1056%2FNEJMoa1201622"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Combination anastrozole and fulvestrant in metastatic breast cancer</span></div><div class="casAuthors">Mehta, Rita S.; Barlow, William E.; Albain, Kathy S.; Vandenberg, Ted A.; Dakhil, Shaker R.; Tirumali, Nagendra R.; Lew, Danika L.; Hayes, Daniel F.; Gralow, Julie R.; Livingston, Robert B.; Hortobagyi, Gabriel N.</div><div class="citationInfo"><span class="NLM_cas:title">New England Journal of Medicine</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">367</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">435-444</span>CODEN:
                <span class="NLM_cas:coden">NEJMAG</span>;
        ISSN:<span class="NLM_cas:issn">0028-4793</span>.
    
            (<span class="NLM_cas:orgname">Massachusetts Medical Society</span>)
        </div><div class="casAbstract">BACKGROUND: The aromatase inhibitor anastrozole inhibits estrogen synthesis.  Fulvestrant binds and accelerates degrdn. of estrogen receptors.  We hypothesized that these two agents in combination might be more effective than anastrozole alone in patients with hormone-receptor (HR)-pos. metastatic breast cancer.  METHODS: Postmenopausal women with previously untreated metastatic disease were randomly assigned, in a 1:1 ratio, to receive either 1 mg of anastrozole orally every day (group 1), with crossover to fulvestrant alone strongly encouraged if the disease progressed, or anastrozole and fulvestrant in combination (group 2).  Patients were stratified according to prior or no prior receipt of adjuvant tamoxifen therapy.  Fulvestrant was administered i.m. at a dose of 500 mg on day 1 and 250 mg on days 14 and 28 and monthly thereafter.  The primary end point was progression-free survival, with overall survival designated as a prespecified secondary outcome.  RESULTS: The median progression-free survival was 13.5 mo in group 1 and 15.0 mo in group 2 (hazard ratio for progression or death with combination therapy, 0.80; 95% confidence interval [CI], 0.68 to 0.94; P = 0.007 by the log-rank test).  The combination therapy was generally more effective than anastrozole alone in all subgroups, with no significant interactions.  Overall survival was also longer with combination therapy (median, 41.3 mo in group 1 and 47.7 mo in group 2; hazard ratio for death, 0.81; 95% CI, 0.65 to 1.00; P = 0.05 by the log-rank test), despite the fact that 41% of the patients in group 1 crossed over to fulvestrant after progression.  Three deaths that were possibly assocd. with treatment occurred in group 2.  The rates of grade 3 to 5 toxic effects did not differ significantly between the two groups.  CONCLUSIONS: The combination of anastrozole and fulvestrant was superior to anastrozole alone or sequential anastrozole and fulvestrant for the treatment of HR-pos. metastatic breast cancer, despite the use of a dose of fulvestrant that was below the current std.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphSxtK-2-aa7Vg90H21EOLACvtfcHk0liVDSXIgSjXxw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtlSltrbF&md5=869d5b1663c60bc2a6fda960b4d8f00e</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1056%2FNEJMoa1201622&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMoa1201622%26sid%3Dliteratum%253Aachs%26aulast%3DMehta%26aufirst%3DR.%2BS.%26aulast%3DBarlow%26aufirst%3DW.%2BE.%26aulast%3DAlbain%26aufirst%3DK.%2BS.%26aulast%3DVandenberg%26aufirst%3DT.%2BA.%26aulast%3DDakhil%26aufirst%3DS.%2BR.%26aulast%3DTirumali%26aufirst%3DN.%2BR.%26aulast%3DLew%26aufirst%3DD.%2BL.%26aulast%3DHayes%26aufirst%3DD.%2BF.%26aulast%3DGralow%26aufirst%3DJ.%2BR.%26aulast%3DLivingston%26aufirst%3DR.%2BB.%26aulast%3DHortobagyi%26aufirst%3DG.%2BN.%26atitle%3DCombination%2520anastrozole%2520and%2520fulvestrant%2520in%2520metastatic%2520breast%2520cancer%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2012%26volume%3D367%26spage%3D435%26epage%3D444%26doi%3D10.1056%2FNEJMoa1201622" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Johnston, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheung, K. L.</span></span> <span> </span><span class="NLM_article-title">Fulvestrant - a novel endocrine therapy for breast cancer</span>. <i>Curr. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">902</span>– <span class="NLM_lpage">914</span>, <span class="refDoi"> DOI: 10.2174/092986710790820633</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=10.2174%2F092986710790820633" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=20156170" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A528%3ADC%252BC3cXkvV2jtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2010&pages=902-914&author=S.+J.+Johnstonauthor=K.+L.+Cheung&title=Fulvestrant+-+a+novel+endocrine+therapy+for+breast+cancer&doi=10.2174%2F092986710790820633"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Fulvestrant - a novel endocrine therapy for breast cancer</span></div><div class="casAuthors">Johnston, S. J.; Cheung, K. L.</div><div class="citationInfo"><span class="NLM_cas:title">Current Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">902-914</span>CODEN:
                <span class="NLM_cas:coden">CMCHE7</span>;
        ISSN:<span class="NLM_cas:issn">0929-8673</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Fulvestrant is a novel endocrine therapy for breast cancer, with a unique structure and mode of action.  It binds competitively to the estrogen receptor (ER), with high affinity, and down regulates ER by functional blockade and increased turnover.review.  Fulvestrant has reached the clinic via extensive pre-clin. and clin. trials, which demonstrated fulvestrant's unique characteristics and showed that they translate to equiv. or improved clin. efficacy compared to established endocrine agents.  Fulvestrant is currently licensed for use in postmenopausal women with hormone receptor pos. advanced breast cancer which has progressed on prior endocrine therapy.  As a pure estrogen antagonist, fulvestrant avoids the risk of detrimental side effects of selective ER modulators such as tamoxifen, which has partial agonist activity.  Fulvestrant, the only parenteral agent in this setting, has a good side effect profile and is well tolerated.  Due to its unique mode of action, fulvestrant lacks cross-resistance with existing agents.  Fulvestrant is the subject of much ongoing research, which utilizes knowledge of its novel mechanism and pharmacokinetic profile in order to optimize clin. efficacy and explore new roles, including first-line use in advanced breast cancer, use in combination with existing agents, in males, and in premenopausal women, and use as an adjuvant therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq-9Noer_gy07Vg90H21EOLACvtfcHk0liVDSXIgSjXxw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXkvV2jtr0%253D&md5=9dab4f74acde17ff56508769d4822161</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.2174%2F092986710790820633&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F092986710790820633%26sid%3Dliteratum%253Aachs%26aulast%3DJohnston%26aufirst%3DS.%2BJ.%26aulast%3DCheung%26aufirst%3DK.%2BL.%26atitle%3DFulvestrant%2520-%2520a%2520novel%2520endocrine%2520therapy%2520for%2520breast%2520cancer%26jtitle%3DCurr.%2520Med.%2520Chem.%26date%3D2010%26volume%3D17%26spage%3D902%26epage%3D914%26doi%3D10.2174%2F092986710790820633" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ellis, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Llombart-Cussac, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feltl, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dewar, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jasiowka, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hewson, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rukazenkov, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robertson, J. F.</span></span> <span> </span><span class="NLM_article-title">Fulvestrant 500 mg versus anastrozole 1 mg for the first-line treatment of advanced breast cancer: overall survival analysis from the phase II first study</span>. <i>J. Clin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">3781</span>– <span class="NLM_lpage">3787</span>, <span class="refDoi"> DOI: 10.1200/JCO.2015.61.5831</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=10.1200%2FJCO.2015.61.5831" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=26371134" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtVSntb3K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2015&pages=3781-3787&author=M.+J.+Ellisauthor=A.+Llombart-Cussacauthor=D.+Feltlauthor=J.+A.+Dewarauthor=M.+Jasiowkaauthor=N.+Hewsonauthor=Y.+Rukazenkovauthor=J.+F.+Robertson&title=Fulvestrant+500+mg+versus+anastrozole+1+mg+for+the+first-line+treatment+of+advanced+breast+cancer%3A+overall+survival+analysis+from+the+phase+II+first+study&doi=10.1200%2FJCO.2015.61.5831"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Fulvestrant 500 mg versus anastrozole 1 mg for the first-line treatment of advanced breast cancer: overall survival analysis from the phase II FIRST study</span></div><div class="casAuthors">Ellis, Matthew J.; Llombart-Cussac, Antonio; Feltl, David; Dewar, John A.; Jasiowka, Marek; Hewson, Nicola; Rukazenkov, Yuri; Robertson, John F. R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Clinical Oncology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">32</span>),
    <span class="NLM_cas:pages">3781-3787</span>CODEN:
                <span class="NLM_cas:coden">JCONDN</span>;
        ISSN:<span class="NLM_cas:issn">0732-183X</span>.
    
            (<span class="NLM_cas:orgname">American Society of Clinical Oncology</span>)
        </div><div class="casAbstract">To compare overall survival (OS) for fulvestrant 500 mg vs. anastrozole as first-line endocrine therapy for advanced breast cancer.  The Fulvestrant First-Line Study Comparing Endocrine Treatments (FIRST) was a phase II, randomized, open-label, multicenter trial.  Postmenopausal women with estrogen receptor-pos., locally advanced/metastatic breast cancer who had no previous therapy for advanced disease received either fulvestrant 500 mg (days 0, 14, 28, and every 28 days thereafter) or anastrozole 1 mg (daily).  The primary end point (clin. benefit rate [72.5% and 67.0%]) and a follow-up anal. (median time to progression [23.4 mo and 13.1 mo]) have been reported previously for fulvestrant 500 mg and anastrozole, resp.  Subsequently, the protocol was amended to assess OS by unadjusted log-rank test after approx. 65% of patients had died.  Treatment effect on OS across several subgroups was examd.  Tolerability was evaluated by adverse event monitoring.  Results In total, 205 patients were randomly assigned (fulvestrant 500 mg, n = 102; anastrozole, n = 103).  At data cutoff, 61.8% (fulvestrant 500 mg, n = 63) and 71.8% (anastrozole, n = 74) had died.  The hazard ratio (95% CI) for OS with fulvestrant 500 mg vs. anastrozole was 0.70 (0.50 to 0.98; P = .04; median OS, 54.1 mo v 48.4 mo).  Treatment effects seemed generally consistent across the subgroups analyzed.  No new safety issues were obsd.  There are several limitations of this OS anal., including that it was not planned in the original protocol but instead was added after time-to-progression results were analyzed, and that not all patients participated in addnl. OS follow-up.  However, the present results suggest fulvestrant 500 mg extends OS vs. anastrozole.  This finding now awaits prospective confirmation in the larger phase III FALCON (Fulvestrant and Anastrozole Compared in Hormonal Therapy Na.ovrddot.ive Advanced Breast Cancer).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgcontOU3287Vg90H21EOLACvtfcHk0liVDSXIgSjXxw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtVSntb3K&md5=45cb5f02224b33857d55e5cb5b69e775</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1200%2FJCO.2015.61.5831&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1200%252FJCO.2015.61.5831%26sid%3Dliteratum%253Aachs%26aulast%3DEllis%26aufirst%3DM.%2BJ.%26aulast%3DLlombart-Cussac%26aufirst%3DA.%26aulast%3DFeltl%26aufirst%3DD.%26aulast%3DDewar%26aufirst%3DJ.%2BA.%26aulast%3DJasiowka%26aufirst%3DM.%26aulast%3DHewson%26aufirst%3DN.%26aulast%3DRukazenkov%26aufirst%3DY.%26aulast%3DRobertson%26aufirst%3DJ.%2BF.%26atitle%3DFulvestrant%2520500%2520mg%2520versus%2520anastrozole%25201%2520mg%2520for%2520the%2520first-line%2520treatment%2520of%2520advanced%2520breast%2520cancer%253A%2520overall%2520survival%2520analysis%2520from%2520the%2520phase%2520II%2520first%2520study%26jtitle%3DJ.%2520Clin.%2520Oncol.%26date%3D2015%26volume%3D33%26spage%3D3781%26epage%3D3787%26doi%3D10.1200%2FJCO.2015.61.5831" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Howell, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sapunar, F.</span></span> <span> </span><span class="NLM_article-title">Fulvestrant revisited: efficacy and safety of the 500-mg dose</span>. <i>Clin. Breast Cancer</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">204</span>– <span class="NLM_lpage">210</span>, <span class="refDoi"> DOI: 10.1016/j.clbc.2011.02.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=10.1016%2Fj.clbc.2011.02.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=21729658" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlKisr7E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2011&pages=204-210&author=A.+Howellauthor=F.+Sapunar&title=Fulvestrant+revisited%3A+efficacy+and+safety+of+the+500-mg+dose&doi=10.1016%2Fj.clbc.2011.02.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Fulvestrant revisited: efficacy and safety of the 500-mg dose</span></div><div class="casAuthors">Howell, Anthony; Sapunar, Francisco</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Breast Cancer</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">204-210</span>CODEN:
                <span class="NLM_cas:coden">CBCLB7</span>;
        ISSN:<span class="NLM_cas:issn">1526-8209</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review.  Postmenopausal women with hormone receptor-pos. advanced breast cancer are candidates for endocrine therapy.  As the disease will eventually progress in most patients, it is important to investigate agents with novel modes of action to reduce the likelihood of treatment cross-resistance.  Fulvestrant is an estrogen receptor antagonist with no known agonist effects that has been shown to be as effective as anastrozole following failure on tamoxifen, at the approved dose of 250 mg/mo.  However, pharmacokinetic modeling and evidence of clin. efficacy in early trials, together with the favorable tolerability profile of fulvestrant 250 mg, led to suggestions that increasing the fulvestrant dose would lead to an improved benefit-risk profile.  This review describes the rationale behind the development of a 500 mg/mo higher dose of fulvestrant and details relevant clin. trials, including the pivotal phase III COmparisoN of Faslodex In Recurrent or Metastatic breast cancer (CONFIRM) study.  CONFIRM demonstrated a significant improvement in progression-free survival for fulvestrant 500 mg vs. 250 mg in postmenopausal patients who had progressed on previous endocrine therapy.  Here, we present and discuss a pooled safety anal. of CONFIRM and three further clin. studies demonstrating fulvestrant 500 mg to be well-tolerated with no evidence of dose-related adverse events.  Overall, these data indicate an improved benefit-risk profile for fulvestrant 500 mg vs. 250 mg following failure on prior endocrine therapy, and suggest that fulvestrant 500 mg may be considered in future as initial endocrine treatment for advanced breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4U1d7Q6edlbVg90H21EOLACvtfcHk0lh9yNaL1-VtUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlKisr7E&md5=0d1063c9546320294193e92d3f2097dd</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fj.clbc.2011.02.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.clbc.2011.02.002%26sid%3Dliteratum%253Aachs%26aulast%3DHowell%26aufirst%3DA.%26aulast%3DSapunar%26aufirst%3DF.%26atitle%3DFulvestrant%2520revisited%253A%2520efficacy%2520and%2520safety%2520of%2520the%2520500-mg%2520dose%26jtitle%3DClin.%2520Breast%2520Cancer%26date%3D2011%26volume%3D11%26spage%3D204%26epage%3D210%26doi%3D10.1016%2Fj.clbc.2011.02.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Willson, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henke, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Momtahen, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Charifson, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Batchelor, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lubahn, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moore, L. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliver, B. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sauls, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Triantafillou, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolfe, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baer, P. G.</span></span> <span> </span><span class="NLM_article-title">3-[4-(1,2-Diphenylbut-1-enyl)phenyl]acrylic acid: a non-steroidal estrogen with functional selectivity for bone over uterus in rats</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>37</i></span>,  <span class="NLM_fpage">1550</span>– <span class="NLM_lpage">1552</span>, <span class="refDoi"> DOI: 10.1021/jm00037a002</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm00037a002" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A528%3ADyaK2cXktlCqt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=1994&pages=1550-1552&author=T.+M.+Willsonauthor=B.+R.+Henkeauthor=T.+M.+Momtahenauthor=P.+S.+Charifsonauthor=K.+W.+Batchelorauthor=D.+B.+Lubahnauthor=L.+B.+Mooreauthor=B.+B.+Oliverauthor=H.+R.+Saulsauthor=J.+A.+Triantafillouauthor=S.+G.+Wolfeauthor=P.+G.+Baer&title=3-%5B4-%281%2C2-Diphenylbut-1-enyl%29phenyl%5Dacrylic+acid%3A+a+non-steroidal+estrogen+with+functional+selectivity+for+bone+over+uterus+in+rats&doi=10.1021%2Fjm00037a002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">3-[4-(1,2-Diphenylbut-1-enyl)phenyl]acrylic Acid: A Non-Steroidal Estrogen with Functional Selectivity for Bone over Uterus in Rats</span></div><div class="casAuthors">Willson, Timothy M.; Henke, Brad R.; Momtahen, Tanya M.; Charifson, Paul S.; Batchelor, Kenneth W.; Lubahn, Dennis B.; Moore, Linda B.; Oliver, Beverly B.; Sauls, Howard R.; et al.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1550-2</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    </div><div class="casAbstract">Exploiting knowledge of the mol. biol. of the estrogen receptor and the existence of two independent transcription activation domains, a series of tissue-selective non-steroidal estrogens was designed.  The ethanolamine side-chain of tamoxifen was targeted for modification, resulting in the identification of a series of α,β-unsatd. acid and amide analogs I [R = OH, NEt2, morpholino, NH(CH2)3OMe, dicyclohexylamino, NEtCH2CH2NMe2, NH(CH2)3OH, N-methyloctylamino, NHEt,NH2].  I [R = OH] profiled as an estrogen with full agonist activity in bone while maintaining antagonist activity in the uterus of rats.  The demonstration that different parts of the triphenylethylene structure control expression of estrogenic activity in bone and uterus has implications in the design of tissue selective ligands for other members of the steroid/retinoid family of intracellular receptors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp4iuXq4R8fm7Vg90H21EOLACvtfcHk0lh9yNaL1-VtUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXktlCqt7g%253D&md5=721906b3513f7e8cdc9ae08ecd99a93b</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fjm00037a002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm00037a002%26sid%3Dliteratum%253Aachs%26aulast%3DWillson%26aufirst%3DT.%2BM.%26aulast%3DHenke%26aufirst%3DB.%2BR.%26aulast%3DMomtahen%26aufirst%3DT.%2BM.%26aulast%3DCharifson%26aufirst%3DP.%2BS.%26aulast%3DBatchelor%26aufirst%3DK.%2BW.%26aulast%3DLubahn%26aufirst%3DD.%2BB.%26aulast%3DMoore%26aufirst%3DL.%2BB.%26aulast%3DOliver%26aufirst%3DB.%2BB.%26aulast%3DSauls%26aufirst%3DH.%2BR.%26aulast%3DTriantafillou%26aufirst%3DJ.%2BA.%26aulast%3DWolfe%26aufirst%3DS.%2BG.%26aulast%3DBaer%26aufirst%3DP.%2BG.%26atitle%3D3-%255B4-%25281%252C2-Diphenylbut-1-enyl%2529phenyl%255Dacrylic%2520acid%253A%2520a%2520non-steroidal%2520estrogen%2520with%2520functional%2520selectivity%2520for%2520bone%2520over%2520uterus%2520in%2520rats%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1994%26volume%3D37%26spage%3D1550%26epage%3D1552%26doi%3D10.1021%2Fjm00037a002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lai, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kahraman, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Govek, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagasawa, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonnefous, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Julien, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Douglas, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sensintaffar, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aparicio, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaufman, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prudente, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moon, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joseph, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Darimont, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brigham, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grillot, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heyman, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rix, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hager, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, N. D.</span></span> <span> </span><span class="NLM_article-title">Identification of GDC-0810 (ARN-810), an orally bioavailable selective estrogen receptor degrader (SERD) that demonstrates robust activity in tamoxifen-resistant breast cancer xenografts</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">4888</span>– <span class="NLM_lpage">4904</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00054</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00054" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmsFyisb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=4888-4904&author=A.+Laiauthor=M.+Kahramanauthor=S.+Govekauthor=J.+Nagasawaauthor=C.+Bonnefousauthor=J.+Julienauthor=K.+Douglasauthor=J.+Sensintaffarauthor=N.+Luauthor=K.+J.+Leeauthor=A.+Aparicioauthor=J.+Kaufmanauthor=J.+Qianauthor=G.+Shaoauthor=R.+Prudenteauthor=M.+J.+Moonauthor=J.+D.+Josephauthor=B.+Darimontauthor=D.+Brighamauthor=K.+Grillotauthor=R.+Heymanauthor=P.+J.+Rixauthor=J.+H.+Hagerauthor=N.+D.+Smith&title=Identification+of+GDC-0810+%28ARN-810%29%2C+an+orally+bioavailable+selective+estrogen+receptor+degrader+%28SERD%29+that+demonstrates+robust+activity+in+tamoxifen-resistant+breast+cancer+xenografts&doi=10.1021%2Facs.jmedchem.5b00054"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of GDC-0810 (ARN-810), an Orally Bioavailable Selective Estrogen Receptor Degrader (SERD) that Demonstrates Robust Activity in Tamoxifen-Resistant Breast Cancer Xenografts</span></div><div class="casAuthors">Lai, Andiliy; Kahraman, Mehmet; Govek, Steven; Nagasawa, Johnny; Bonnefous, Celine; Julien, Jackie; Douglas, Karensa; Sensintaffar, John; Lu, Nhin; Lee, Kyoung-jin; Aparicio, Anna; Kaufman, Josh; Qian, Jing; Shao, Gang; Prudente, Rene; Moon, Michael J.; Joseph, James D.; Darimont, Beatrice; Brigham, Daniel; Grillot, Kate; Heyman, Richard; Rix, Peter J.; Hager, Jeffrey H.; Smith, Nicholas D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">4888-4904</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Approx. 80% of breast cancers are estrogen receptor alpha (ER-α) pos., and although women typically initially respond well to antihormonal therapies such as tamoxifen and aromatase inhibitors, resistance often emerges.  Although a variety of resistance mechanism may be at play in this state, there is evidence that in many cases the ER still plays a central role, including mutations in the ER leading to constitutively active receptor.  Fulvestrant is a steroid-based, selective estrogen receptor degrader (SERD) that both antagonizes and degrades ER-α and is active in patients who have progressed on antihormonal agents.  However, fulvestrant suffers from poor pharmaceutical properties and must be administered by i.m. injections that limit the total amt. of drug that can be administered and hence lead to the potential for incomplete receptor blockade.  The authors describe the identification and characterization of a series of small-mol., orally bioavailable SERDs which are potent antagonists and degraders of ER-α and in which the ER-α degrading properties were prospectively optimized.  The lead compd. I (GDC-0810 or ARN-810) demonstrates robust activity in models of tamoxifen-sensitive and tamoxifen-resistant breast cancer, and is currently in clin. trials in women with locally advanced or metastatic estrogen receptor-pos. breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOegJFz-sqwbVg90H21EOLACvtfcHk0lh9yNaL1-VtUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXmsFyisb4%253D&md5=e836ec4d593ce6ed2d9622da5cd898e6</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00054&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00054%26sid%3Dliteratum%253Aachs%26aulast%3DLai%26aufirst%3DA.%26aulast%3DKahraman%26aufirst%3DM.%26aulast%3DGovek%26aufirst%3DS.%26aulast%3DNagasawa%26aufirst%3DJ.%26aulast%3DBonnefous%26aufirst%3DC.%26aulast%3DJulien%26aufirst%3DJ.%26aulast%3DDouglas%26aufirst%3DK.%26aulast%3DSensintaffar%26aufirst%3DJ.%26aulast%3DLu%26aufirst%3DN.%26aulast%3DLee%26aufirst%3DK.%2BJ.%26aulast%3DAparicio%26aufirst%3DA.%26aulast%3DKaufman%26aufirst%3DJ.%26aulast%3DQian%26aufirst%3DJ.%26aulast%3DShao%26aufirst%3DG.%26aulast%3DPrudente%26aufirst%3DR.%26aulast%3DMoon%26aufirst%3DM.%2BJ.%26aulast%3DJoseph%26aufirst%3DJ.%2BD.%26aulast%3DDarimont%26aufirst%3DB.%26aulast%3DBrigham%26aufirst%3DD.%26aulast%3DGrillot%26aufirst%3DK.%26aulast%3DHeyman%26aufirst%3DR.%26aulast%3DRix%26aufirst%3DP.%2BJ.%26aulast%3DHager%26aufirst%3DJ.%2BH.%26aulast%3DSmith%26aufirst%3DN.%2BD.%26atitle%3DIdentification%2520of%2520GDC-0810%2520%2528ARN-810%2529%252C%2520an%2520orally%2520bioavailable%2520selective%2520estrogen%2520receptor%2520degrader%2520%2528SERD%2529%2520that%2520demonstrates%2520robust%2520activity%2520in%2520tamoxifen-resistant%2520breast%2520cancer%2520xenografts%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D4888%26epage%3D4904%26doi%3D10.1021%2Facs.jmedchem.5b00054" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kahraman, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Govek, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagasawa, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonnefous, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Douglas, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sensintaffar, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aparicio, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaufman, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prudente, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joseph, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Darimont, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brigham, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heyman, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rix, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hager, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, N. D.</span></span> <span> </span><span class="NLM_article-title">Maximizing ER-α degradation maximizes activity in a tamoxifen-resistant breast cancer model: identification of GDC-0927</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">50</span>– <span class="NLM_lpage">55</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.8b00414</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.8b00414" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisVensr3M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=50-55&author=M.+Kahramanauthor=S.+P.+Govekauthor=J.+Y.+Nagasawaauthor=A.+Laiauthor=C.+Bonnefousauthor=K.+Douglasauthor=J.+Sensintaffarauthor=N.+Liuauthor=K.+Leeauthor=A.+Aparicioauthor=J.+Kaufmanauthor=J.+Qianauthor=G.+Shaoauthor=R.+Prudenteauthor=J.+D.+Josephauthor=B.+Darimontauthor=D.+Brighamauthor=R.+Heymanauthor=P.+J.+Rixauthor=J.+H.+Hagerauthor=N.+D.+Smith&title=Maximizing+ER-%CE%B1+degradation+maximizes+activity+in+a+tamoxifen-resistant+breast+cancer+model%3A+identification+of+GDC-0927&doi=10.1021%2Facsmedchemlett.8b00414"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Maximizing ER-α Degradation Maximizes Activity in a Tamoxifen-Resistant Breast Cancer Model: Identification of GDC-0927</span></div><div class="casAuthors">Kahraman, Mehmet; Govek, Steven P.; Nagasawa, Johnny Y.; Lai, Andiliy; Bonnefous, Celine; Douglas, Karensa; Sensintaffar, John; Liu, Nhin; Lee, KyoungJin; Aparicio, Anna; Kaufman, Josh; Qian, Jing; Shao, Gang; Prudente, Rene; Joseph, James D.; Darimont, Beatrice; Brigham, Daniel; Heyman, Richard; Rix, Peter J.; Hager, Jeffrey H.; Smith, Nicholas D.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">50-55</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The further optimization of ER-α degrdn. efficacy of a series of ER modulators by refining side-chain substitution led to efficacious selective estrogen receptor degraders (SERDs).  A fluoromethyl azetidine group was found to be preferred and resulted in the identification of bis-phenol chromene 17ha.  In a tamoxifen-resistant breast cancer xenograft model, 17ha (ER-α degrdn. efficacy = 97%) demonstrated tumor regression, together with robust redn. of intratumoral ER-α levels.  However, despite superior oral exposure, 5a (ER-α degrdn. efficacy = 91%) had inferior activity.  This result suggests that optimizing ER-α degrdn. efficacy leads to compds. with robust effects in a model of tamoxifen-resistant breast cancer.  Compd. 17ha (GDC-0927) was evaluated in clin. trials in women with metastatic estrogen receptor-pos. breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgyUydsTFnYrVg90H21EOLACvtfcHk0lieQZx2wpzvwg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisVensr3M&md5=324c1697ef350d169a97960de72ee8a9</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.8b00414&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.8b00414%26sid%3Dliteratum%253Aachs%26aulast%3DKahraman%26aufirst%3DM.%26aulast%3DGovek%26aufirst%3DS.%2BP.%26aulast%3DNagasawa%26aufirst%3DJ.%2BY.%26aulast%3DLai%26aufirst%3DA.%26aulast%3DBonnefous%26aufirst%3DC.%26aulast%3DDouglas%26aufirst%3DK.%26aulast%3DSensintaffar%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DN.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DAparicio%26aufirst%3DA.%26aulast%3DKaufman%26aufirst%3DJ.%26aulast%3DQian%26aufirst%3DJ.%26aulast%3DShao%26aufirst%3DG.%26aulast%3DPrudente%26aufirst%3DR.%26aulast%3DJoseph%26aufirst%3DJ.%2BD.%26aulast%3DDarimont%26aufirst%3DB.%26aulast%3DBrigham%26aufirst%3DD.%26aulast%3DHeyman%26aufirst%3DR.%26aulast%3DRix%26aufirst%3DP.%2BJ.%26aulast%3DHager%26aufirst%3DJ.%2BH.%26aulast%3DSmith%26aufirst%3DN.%2BD.%26atitle%3DMaximizing%2520ER-%25CE%25B1%2520degradation%2520maximizes%2520activity%2520in%2520a%2520tamoxifen-resistant%2520breast%2520cancer%2520model%253A%2520identification%2520of%2520GDC-0927%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2019%26volume%3D10%26spage%3D50%26epage%3D55%26doi%3D10.1021%2Facsmedchemlett.8b00414" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">De
Savi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradbury, R. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rabow, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norman, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Almeida, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andrews, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ballard, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buttar, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Callis, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Currie, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curwen, J. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donald, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feron, L. J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gingell, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glossop, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayter, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hussain, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karoutchi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamont, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacFaul, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moss, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pearson, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tonge, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, G. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weir, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, Z.</span></span> <span> </span><span class="NLM_article-title">Optimization of a novel binding motif to (E)-3-(3,5-difluoro-4-((1R,3R)-2-(2-fluoro-2-methylpropyl)-3-methyl-2,3,4,9-tetrahydro-1H-pyrido[3,4-b]indol-1-yl)phenyl)acrylic Acid (AZD9496), a potent and orally bioavailable selective estrogen receptor downregulator and antagonist</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">8128</span>– <span class="NLM_lpage">8140</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00984</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00984" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFKiu7fK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=8128-8140&author=C.+De%0ASaviauthor=R.+H.+Bradburyauthor=A.+A.+Rabowauthor=R.+A.+Normanauthor=C.+de+Almeidaauthor=D.+M.+Andrewsauthor=P.+Ballardauthor=D.+Buttarauthor=R.+J.+Callisauthor=G.+S.+Currieauthor=J.+O.+Curwenauthor=C.+D.+Daviesauthor=C.+S.+Donaldauthor=L.+J.+L.+Feronauthor=H.+Gingellauthor=S.+C.+Glossopauthor=B.+R.+Hayterauthor=S.+Hussainauthor=G.+Karoutchiauthor=S.+G.+Lamontauthor=P.+MacFaulauthor=T.+A.+Mossauthor=S.+E.+Pearsonauthor=M.+Tongeauthor=G.+E.+Walkerauthor=H.+M.+Weirauthor=Z.+Wilson&title=Optimization+of+a+novel+binding+motif+to+%28E%29-3-%283%2C5-difluoro-4-%28%281R%2C3R%29-2-%282-fluoro-2-methylpropyl%29-3-methyl-2%2C3%2C4%2C9-tetrahydro-1H-pyrido%5B3%2C4-b%5Dindol-1-yl%29phenyl%29acrylic+Acid+%28AZD9496%29%2C+a+potent+and+orally+bioavailable+selective+estrogen+receptor+downregulator+and+antagonist&doi=10.1021%2Facs.jmedchem.5b00984"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Optimization of a Novel Binding Motif to (E)-3-(3,5-Difluoro-4-((1R,3R)-2-(2-fluoro-2-methylpropyl)-3-methyl-2,3,4,9-tetrahydro-1H-pyrido[3,4-b]indol-1-yl)phenyl)acrylic Acid (AZD9496), a Potent and Orally Bioavailable Selective Estrogen Receptor Downregulator and Antagonist</span></div><div class="casAuthors">De Savi, Chris; Bradbury, Robert H.; Rabow, Alfred A.; Norman, Richard A.; de Almeida, Camila; Andrews, David M.; Ballard, Peter; Buttar, David; Callis, Rowena J.; Currie, Gordon S.; Curwen, Jon O.; Davies, Chris D.; Donald, Craig S.; Feron, Lyman J. L.; Gingell, Helen; Glossop, Steven C.; Hayter, Barry R.; Hussain, Syeed; Karoutchi, Galith; Lamont, Scott G.; MacFaul, Philip; Moss, Thomas A.; Pearson, Stuart E.; Tonge, Michael; Walker, Graeme E.; Weir, Hazel M.; Wilson, Zena</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">8128-8140</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The discovery of an orally bioavailable selective estrogen receptor downregulator (SERD) with equiv. potency and preclin. pharmacol. to the i.m. SERD fulvestrant is described.  A directed screen identified the 1-aryl-2,3,4,9-tetrahydro-1H-pyrido[3,4-b]indole motif as a novel, druglike ER ligand.  Aided by crystal structures of novel ligands bound to an ER construct, medicinal chem. iterations led to (E)-3-(3,5-difluoro-4-((1R,3R)-2-(2-fluoro-2-methylpropyl)-3-methyl-2,3,4,9-tetrahydro-1H-pyrido[3,4-b]indol-1-yl)phenyl)acrylic acid (30b, AZD9496), a clin. candidate with high oral bioavailability across preclin. species that is currently being evaluated in phase I clin. trials for the treatment of advanced estrogen receptor (ER) pos. breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpP-ZYYxmMtg7Vg90H21EOLACvtfcHk0lieQZx2wpzvwg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFKiu7fK&md5=df7eee70ca783a2933cd494b53f71488</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00984&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00984%26sid%3Dliteratum%253Aachs%26aulast%3DDe%2BSavi%26aufirst%3DC.%26aulast%3DBradbury%26aufirst%3DR.%2BH.%26aulast%3DRabow%26aufirst%3DA.%2BA.%26aulast%3DNorman%26aufirst%3DR.%2BA.%26aulast%3Dde%2BAlmeida%26aufirst%3DC.%26aulast%3DAndrews%26aufirst%3DD.%2BM.%26aulast%3DBallard%26aufirst%3DP.%26aulast%3DButtar%26aufirst%3DD.%26aulast%3DCallis%26aufirst%3DR.%2BJ.%26aulast%3DCurrie%26aufirst%3DG.%2BS.%26aulast%3DCurwen%26aufirst%3DJ.%2BO.%26aulast%3DDavies%26aufirst%3DC.%2BD.%26aulast%3DDonald%26aufirst%3DC.%2BS.%26aulast%3DFeron%26aufirst%3DL.%2BJ.%2BL.%26aulast%3DGingell%26aufirst%3DH.%26aulast%3DGlossop%26aufirst%3DS.%2BC.%26aulast%3DHayter%26aufirst%3DB.%2BR.%26aulast%3DHussain%26aufirst%3DS.%26aulast%3DKaroutchi%26aufirst%3DG.%26aulast%3DLamont%26aufirst%3DS.%2BG.%26aulast%3DMacFaul%26aufirst%3DP.%26aulast%3DMoss%26aufirst%3DT.%2BA.%26aulast%3DPearson%26aufirst%3DS.%2BE.%26aulast%3DTonge%26aufirst%3DM.%26aulast%3DWalker%26aufirst%3DG.%2BE.%26aulast%3DWeir%26aufirst%3DH.%2BM.%26aulast%3DWilson%26aufirst%3DZ.%26atitle%3DOptimization%2520of%2520a%2520novel%2520binding%2520motif%2520to%2520%2528E%2529-3-%25283%252C5-difluoro-4-%2528%25281R%252C3R%2529-2-%25282-fluoro-2-methylpropyl%2529-3-methyl-2%252C3%252C4%252C9-tetrahydro-1H-pyrido%255B3%252C4-b%255Dindol-1-yl%2529phenyl%2529acrylic%2520Acid%2520%2528AZD9496%2529%252C%2520a%2520potent%2520and%2520orally%2520bioavailable%2520selective%2520estrogen%2520receptor%2520downregulator%2520and%2520antagonist%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D8128%26epage%3D8140%26doi%3D10.1021%2Facs.jmedchem.5b00984" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tria, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abrams, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baird, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burks, H. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Firestone, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaither, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamann, L. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirby, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lombardo, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macchi, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDonnell, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mishina, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norris, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nunez, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Springer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomsen, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tiong-Yip, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peukert, S.</span></span> <span> </span><span class="NLM_article-title">Discovery of LSZ102, a potent, orally bioavailable selective estrogen receptor degrader (SERD) for the treatment of estrogen receptor positive breast cancer</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">2837</span>– <span class="NLM_lpage">2864</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01682</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01682" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A528%3ADC%252BC1cXltFynurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=2837-2864&author=G.+S.+Triaauthor=T.+Abramsauthor=J.+Bairdauthor=H.+E.+Burksauthor=B.+Firestoneauthor=L.+A.+Gaitherauthor=L.+G.+Hamannauthor=G.+Heauthor=C.+A.+Kirbyauthor=S.+Kimauthor=F.+Lombardoauthor=K.+J.+Macchiauthor=D.+P.+McDonnellauthor=Y.+Mishinaauthor=J.+D.+Norrisauthor=J.+Nunezauthor=C.+Springerauthor=Y.+Sunauthor=N.+M.+Thomsenauthor=C.+Wangauthor=J.+Wangauthor=B.+Yuauthor=C.+L.+Tiong-Yipauthor=S.+Peukert&title=Discovery+of+LSZ102%2C+a+potent%2C+orally+bioavailable+selective+estrogen+receptor+degrader+%28SERD%29+for+the+treatment+of+estrogen+receptor+positive+breast+cancer&doi=10.1021%2Facs.jmedchem.7b01682"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of LSZ102, a Potent, Orally Bioavailable Selective Estrogen Receptor Degrader (SERD) for the Treatment of Estrogen Receptor Positive Breast Cancer</span></div><div class="casAuthors">Tria, George S.; Abrams, Tinya; Baird, Jason; Burks, Heather E.; Firestone, Brant; Gaither, L. Alex; Hamann, Lawrence G.; He, Guo; Kirby, Christina A.; Kim, Sunkyu; Lombardo, Franco; Macchi, Kaitlin J.; McDonnell, Donald P.; Mishina, Yuji; Norris, John D.; Nunez, Jill; Springer, Clayton; Sun, Yingchuan; Thomsen, Noel M.; Wang, Chunrong; Wang, Jianling; Yu, Bing; Tiong-Yip, Choi-Lai; Peukert, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2837-2864</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In breast cancer, estrogen receptor alpha (ERα) pos. cancer accounts for approx. 74% of all diagnoses, and in these settings, it is a primary driver of cell proliferation.  Treatment of ERα pos. breast cancer has long relied on endocrine therapies such as selective estrogen receptor modulators, aromatase inhibitors, and selective estrogen receptor degraders (SERDs).  The steroid-based anti-estrogen fulvestrant, the only approved SERD, is effective in patients who have not previously been treated with endocrine therapy as well as in patients who have progressed after receiving other endocrine therapies.  Its efficacy, however, may be limited due to its poor physicochem. properties.  THe authors describe the design and synthesis of a series of potent benzothiophene-contg. compds. that exhibit oral bioavailability and preclin. activity as SERDs.  This article culminates in the identification of LSZ102 (I), a compd. in clin. development for the treatment of ERα pos. breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcHgOrWwlRvrVg90H21EOLACvtfcHk0lgqy27XWPUt0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXltFynurc%253D&md5=dba954f9f11121fae1bb50979428a83b</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01682&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01682%26sid%3Dliteratum%253Aachs%26aulast%3DTria%26aufirst%3DG.%2BS.%26aulast%3DAbrams%26aufirst%3DT.%26aulast%3DBaird%26aufirst%3DJ.%26aulast%3DBurks%26aufirst%3DH.%2BE.%26aulast%3DFirestone%26aufirst%3DB.%26aulast%3DGaither%26aufirst%3DL.%2BA.%26aulast%3DHamann%26aufirst%3DL.%2BG.%26aulast%3DHe%26aufirst%3DG.%26aulast%3DKirby%26aufirst%3DC.%2BA.%26aulast%3DKim%26aufirst%3DS.%26aulast%3DLombardo%26aufirst%3DF.%26aulast%3DMacchi%26aufirst%3DK.%2BJ.%26aulast%3DMcDonnell%26aufirst%3DD.%2BP.%26aulast%3DMishina%26aufirst%3DY.%26aulast%3DNorris%26aufirst%3DJ.%2BD.%26aulast%3DNunez%26aufirst%3DJ.%26aulast%3DSpringer%26aufirst%3DC.%26aulast%3DSun%26aufirst%3DY.%26aulast%3DThomsen%26aufirst%3DN.%2BM.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DYu%26aufirst%3DB.%26aulast%3DTiong-Yip%26aufirst%3DC.%2BL.%26aulast%3DPeukert%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520LSZ102%252C%2520a%2520potent%252C%2520orally%2520bioavailable%2520selective%2520estrogen%2520receptor%2520degrader%2520%2528SERD%2529%2520for%2520the%2520treatment%2520of%2520estrogen%2520receptor%2520positive%2520breast%2520cancer%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D2837%26epage%3D2864%26doi%3D10.1021%2Facs.jmedchem.7b01682" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDonnell, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norris, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willson, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greene, G. L.</span></span> <span> </span><span class="NLM_article-title">Structural basis for an unexpected mode of SERM-mediated ER antagonism</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">413</span>– <span class="NLM_lpage">424</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2005.04.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=10.1016%2Fj.molcel.2005.04.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=15893725" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A528%3ADC%252BD2MXkslaqu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2005&pages=413-424&author=Y.-L.+Wuauthor=X.+Yangauthor=Z.+Renauthor=D.+P.+McDonnellauthor=J.+D.+Norrisauthor=T.+M.+Willsonauthor=G.+L.+Greene&title=Structural+basis+for+an+unexpected+mode+of+SERM-mediated+ER+antagonism&doi=10.1016%2Fj.molcel.2005.04.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis for an unexpected mode of SERM-mediated ER antagonism</span></div><div class="casAuthors">Wu, Ya-Ling; Yang, Xiaojing; Ren, Zhong; McDonnell, Donald P.; Norris, John D.; Willson, Timothy M.; Greene, Geoffrey L.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">413-424</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Tamoxifen is effective for the prevention and treatment of estrogen-dependent breast cancers, but is assocd. with an increased incidence of endometrial tumors.  The authors report the crystal structure of the estrogen receptor α (ERα) ligand binding domain (LBD) bound to the structurally similar compd. GW5638, which has therapeutic potential and does not stimulate the uterus.  Like tamoxifen, GW5638 relocates the carboxy-terminal helix (H12) to the known coactivator-docking site in the ERα LBD.  However, GW5638 repositions residues in H12 through specific contacts with the N terminus of this helix.  In contrast to tamoxifen, the resulting increase in exposed hydrophobic surface of ERα LBD correlates with a significant destabilization of ERα in MCF-7 cells.  Thus, the GW5638-ERα LBD structure reveals an unexpected mode of SERM-mediated ER antagonism, in which the stability of ERα is decreased through an altered position of H12.  This dual mechanism of antagonism may explain why GW5638 can inhibit tamoxifen-resistant breast tumors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBuyhLNSdbQbVg90H21EOLACvtfcHk0lgqy27XWPUt0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXkslaqu7s%253D&md5=c60e01b31b57c07b934da01a34efb255</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2005.04.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2005.04.014%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DY.-L.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DRen%26aufirst%3DZ.%26aulast%3DMcDonnell%26aufirst%3DD.%2BP.%26aulast%3DNorris%26aufirst%3DJ.%2BD.%26aulast%3DWillson%26aufirst%3DT.%2BM.%26aulast%3DGreene%26aufirst%3DG.%2BL.%26atitle%3DStructural%2520basis%2520for%2520an%2520unexpected%2520mode%2520of%2520SERM-mediated%2520ER%2520antagonism%26jtitle%3DMol.%2520Cell%26date%3D2005%26volume%3D18%26spage%3D413%26epage%3D424%26doi%3D10.1016%2Fj.molcel.2005.04.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hamilton, E. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Armstrong, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baird, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jhaveri, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoch, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klinowska, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindemann, J. P. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morgan, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schiavon, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weir, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Im, S. A.</span></span> <span> </span><span class="NLM_article-title">A first-in-human study of the new oral selective estrogen receptor degrader AZD9496 for ER(+)/HER2(−) advanced breast cancer</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">3510</span>– <span class="NLM_lpage">3518</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.CCR-17-3102</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=10.1158%2F1078-0432.CCR-17-3102" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=29440181" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsVWrtr3J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2018&pages=3510-3518&author=E.+P.+Hamiltonauthor=M.+R.+Patelauthor=A.+C.+Armstrongauthor=R.+D.+Bairdauthor=K.+Jhaveriauthor=M.+Hochauthor=T.+Klinowskaauthor=J.+P.+O.+Lindemannauthor=S.+R.+Morganauthor=G.+Schiavonauthor=H.+M.+Weirauthor=S.+A.+Im&title=A+first-in-human+study+of+the+new+oral+selective+estrogen+receptor+degrader+AZD9496+for+ER%28%2B%29%2FHER2%28%E2%88%92%29+advanced+breast+cancer&doi=10.1158%2F1078-0432.CCR-17-3102"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">A First-in-Human Study of the New Oral Selective Estrogen Receptor Degrader AZD9496 for ER+/HER2- Advanced Breast Cancer</span></div><div class="casAuthors">Hamilton, Erika P.; Patel, Manish R.; Armstrong, Anne C.; Baird, Richard D.; Jhaveri, Komal; Hoch, Matthias; Klinowska, Teresa; Lindemann, Justin P. O.; Morgan, Shethah R.; Schiavon, Gaia; Weir, Hazel M.; Im, Seock-Ah</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">3510-3518</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">AZD9496 is an oral nonsteroidal, small-mol. inhibitor of estrogen receptor alpha (ERα) and a potent and selective antagonist and degrader of ERα.  This first-in-human phase I study detd. the safety and tolerability of ascending doses of oral AZD9496 in women with estrogen receptor (ER)+/HER2- advanced breast cancer, characterized its pharmacokinetic (PK) profile, and made preliminary assessment of antitumor activity.  Forty-five patients received AZD9496 [20 mg once daily (QD) to 600 mg twice daily (BID)] in a dose-escalation, dose-expansion rolling 6 design.  Safety, tolerability, and PK activity in each cohort were reviewed before escalating to the next dose.  PK was detd. by mass spectrometry.  Adverse events (AEs) were graded according to the Common Terminol. Criteria for Adverse Events (CTCAE) v4.0.  Objective tumor response was evaluated by Response Evaluation Criteria in Solid Tumors (RECIST) v1.1.  Most common causally related AEs were diarrhea (35.6%), fatigue (31.1%), and nausea (22.2%), and seven patients had grade ≥3 AEs.  Three patients experienced a dose-limiting toxicity: one each at 150 mg BID (abnormal hepatic function), 400 mg BID (diarrhea and elevated liver function tests), and 600 mg BID (diarrhea), and all were reversible.  The max. tolerated dose was not reached.  Partial response was confirmed in one patient, who also had decreased tumor marker Ca15.3.  Four patients had stable disease at 12 mo follow-up.  AZD9496 is well tolerated with an acceptable safety profile, showing evidence of prolonged disease stabilization in heavily pretreated patients with ER+/HER2- advanced breast cancer.  Clin Cancer Res; 24(15); 3510-8. 2018 AACR.  See related commentary by Jordan, p.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrT2Esco_KTEbVg90H21EOLACvtfcHk0lgqy27XWPUt0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsVWrtr3J&md5=6d299ace6b73c7693554a54789883d9c</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-17-3102&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-17-3102%26sid%3Dliteratum%253Aachs%26aulast%3DHamilton%26aufirst%3DE.%2BP.%26aulast%3DPatel%26aufirst%3DM.%2BR.%26aulast%3DArmstrong%26aufirst%3DA.%2BC.%26aulast%3DBaird%26aufirst%3DR.%2BD.%26aulast%3DJhaveri%26aufirst%3DK.%26aulast%3DHoch%26aufirst%3DM.%26aulast%3DKlinowska%26aufirst%3DT.%26aulast%3DLindemann%26aufirst%3DJ.%2BP.%2BO.%26aulast%3DMorgan%26aufirst%3DS.%2BR.%26aulast%3DSchiavon%26aufirst%3DG.%26aulast%3DWeir%26aufirst%3DH.%2BM.%26aulast%3DIm%26aufirst%3DS.%2BA.%26atitle%3DA%2520first-in-human%2520study%2520of%2520the%2520new%2520oral%2520selective%2520estrogen%2520receptor%2520degrader%2520AZD9496%2520for%2520ER%2528%252B%2529%252FHER2%2528%25E2%2588%2592%2529%2520advanced%2520breast%2520cancer%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2018%26volume%3D24%26spage%3D3510%26epage%3D3518%26doi%3D10.1158%2F1078-0432.CCR-17-3102" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Weir, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradbury, R. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rabow, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buttar, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Callis, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curwen, J. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Almeida, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ballard, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hulse, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donald, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feron, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karoutchi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacFaul, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moss, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norman, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pearson, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tonge, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, G. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rowlinson, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sadler, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richmond, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ladd, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pazolli, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mazzola, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">D’Cruz, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Savi, C.</span></span> <span> </span><span class="NLM_article-title">AZD9496: an oral estrogen receptor inhibitor that blocks the growth of ER-positive and ESR1-mutant breast tumors in preclinical models</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>76</i></span>,  <span class="NLM_fpage">3307</span>– <span class="NLM_lpage">3318</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-15-2357</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=10.1158%2F0008-5472.CAN-15-2357" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=27020862" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A528%3ADC%252BC28XovVyktLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=76&publication_year=2016&pages=3307-3318&author=H.+M.+Weirauthor=R.+H.+Bradburyauthor=M.+Lawsonauthor=A.+A.+Rabowauthor=D.+Buttarauthor=R.+J.+Callisauthor=J.+O.+Curwenauthor=C.+de+Almeidaauthor=P.+Ballardauthor=M.+Hulseauthor=C.+S.+Donaldauthor=L.+J.+Feronauthor=G.+Karoutchiauthor=P.+MacFaulauthor=T.+Mossauthor=R.+A.+Normanauthor=S.+E.+Pearsonauthor=M.+Tongeauthor=G.+Daviesauthor=G.+E.+Walkerauthor=Z.+Wilsonauthor=R.+Rowlinsonauthor=S.+Powellauthor=C.+Sadlerauthor=G.+Richmondauthor=B.+Laddauthor=E.+Pazolliauthor=A.+M.+Mazzolaauthor=C.+D%E2%80%99Cruzauthor=C.+De+Savi&title=AZD9496%3A+an+oral+estrogen+receptor+inhibitor+that+blocks+the+growth+of+ER-positive+and+ESR1-mutant+breast+tumors+in+preclinical+models&doi=10.1158%2F0008-5472.CAN-15-2357"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">AZD9496: An Oral Estrogen Receptor Inhibitor That Blocks the Growth of ER-Positive and ESR1-Mutant Breast Tumors in Preclinical Models</span></div><div class="casAuthors">Weir, Hazel M.; Bradbury, Robert H.; Lawson, Mandy; Rabow, Alfred A.; Buttar, David; Callis, Rowena J.; Curwen, Jon O.; de Almeida, Camila; Ballard, Peter; Hulse, Michael; Donald, Craig S.; Feron, Lyman J. L.; Karoutchi, Galith; MacFaul, Philip; Moss, Thomas; Norman, Richard A.; Pearson, Stuart E.; Tonge, Michael; Davies, Gareth; Walker, Graeme E.; Wilson, Zena; Rowlinson, Rachel; Powell, Steve; Sadler, Claire; Richmond, Graham; Ladd, Brendon; Pazolli, Ermira; Mazzola, Anne Marie; D'Cruz, Celina; De Savi, Chris</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">76</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">3307-3318</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Fulvestrant is an estrogen receptor (ER) antagonist administered to breast cancer patients by monthly i.m. injection.  Given its present limitations of dosing and route of administration, a more flexible orally available compd. has been sought to pursue the potential benefits of this drug in patients with advanced metastatic disease.  Here we report the identification and characterization of AZD9496, a nonsteroidal small-mol. inhibitor of ERα, which is a potent and selective antagonist and downregulator of ERα in vitro and in vivo in ER-pos. models of breast cancer.  Significant tumor growth inhibition was obsd. as low as 0.5 mg/kg dose in the estrogen-dependent MCF-7 xenograft model, where this effect was accompanied by a dose-dependent decrease in PR protein levels, demonstrating potent antagonist activity.  Combining AZD9496 with PI3K pathway and CDK4/6 inhibitors led to further growth-inhibitory effects compared with monotherapy alone.  Tumor regressions were also seen in a long-term estrogen-deprived breast model, where significant downregulation of ERα protein was obsd.  AZD9496 bound and downregulated clin. relevant ESR1 mutants in vitro and inhibited tumor growth in an ESR1-mutant patient-derived xenograft model that included a D538G mutation.  Collectively, the pharmacol. evidence showed that AZD9496 is an oral, nonsteroidal, selective estrogen receptor antagonist and downregulator in ER+ breast cells that could provide meaningful benefit to ER+ breast cancer patients.  AZD9496 is currently being evaluated in a phase I clin. trial.  Cancer Res; 76(11); 3307-18. ©2016 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoP4X3e2iYdcLVg90H21EOLACvtfcHk0lg5NMfClSVP8A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XovVyktLo%253D&md5=4ff8ab9a9cd379acc031b7c6fd27134c</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-15-2357&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-15-2357%26sid%3Dliteratum%253Aachs%26aulast%3DWeir%26aufirst%3DH.%2BM.%26aulast%3DBradbury%26aufirst%3DR.%2BH.%26aulast%3DLawson%26aufirst%3DM.%26aulast%3DRabow%26aufirst%3DA.%2BA.%26aulast%3DButtar%26aufirst%3DD.%26aulast%3DCallis%26aufirst%3DR.%2BJ.%26aulast%3DCurwen%26aufirst%3DJ.%2BO.%26aulast%3Dde%2BAlmeida%26aufirst%3DC.%26aulast%3DBallard%26aufirst%3DP.%26aulast%3DHulse%26aufirst%3DM.%26aulast%3DDonald%26aufirst%3DC.%2BS.%26aulast%3DFeron%26aufirst%3DL.%2BJ.%26aulast%3DKaroutchi%26aufirst%3DG.%26aulast%3DMacFaul%26aufirst%3DP.%26aulast%3DMoss%26aufirst%3DT.%26aulast%3DNorman%26aufirst%3DR.%2BA.%26aulast%3DPearson%26aufirst%3DS.%2BE.%26aulast%3DTonge%26aufirst%3DM.%26aulast%3DDavies%26aufirst%3DG.%26aulast%3DWalker%26aufirst%3DG.%2BE.%26aulast%3DWilson%26aufirst%3DZ.%26aulast%3DRowlinson%26aufirst%3DR.%26aulast%3DPowell%26aufirst%3DS.%26aulast%3DSadler%26aufirst%3DC.%26aulast%3DRichmond%26aufirst%3DG.%26aulast%3DLadd%26aufirst%3DB.%26aulast%3DPazolli%26aufirst%3DE.%26aulast%3DMazzola%26aufirst%3DA.%2BM.%26aulast%3DD%25E2%2580%2599Cruz%26aufirst%3DC.%26aulast%3DDe%2BSavi%26aufirst%3DC.%26atitle%3DAZD9496%253A%2520an%2520oral%2520estrogen%2520receptor%2520inhibitor%2520that%2520blocks%2520the%2520growth%2520of%2520ER-positive%2520and%2520ESR1-mutant%2520breast%2520tumors%2520in%2520preclinical%2520models%26jtitle%3DCancer%2520Res.%26date%3D2016%26volume%3D76%26spage%3D3307%26epage%3D3318%26doi%3D10.1158%2F0008-5472.CAN-15-2357" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shiau, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barstad, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loria, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kushner, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agard, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greene, G. L.</span></span> <span> </span><span class="NLM_article-title">The structural basis of estrogen receptor/coactivator recognition and the antagonism of this interaction by tamoxifen</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>95</i></span>,  <span class="NLM_fpage">927</span>– <span class="NLM_lpage">937</span>, <span class="refDoi"> DOI: 10.1016/S0092-8674(00)81717-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=10.1016%2FS0092-8674%2800%2981717-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=9875847" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A528%3ADyaK1MXjslyltQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=95&publication_year=1998&pages=927-937&author=A.+K.+Shiauauthor=D.+Barstadauthor=P.+M.+Loriaauthor=L.+Chengauthor=P.+J.+Kushnerauthor=D.+A.+Agardauthor=G.+L.+Greene&title=The+structural+basis+of+estrogen+receptor%2Fcoactivator+recognition+and+the+antagonism+of+this+interaction+by+tamoxifen&doi=10.1016%2FS0092-8674%2800%2981717-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">The structural basis of estrogen receptor/coactivator recognition and the antagonism of this interaction by tamoxifen</span></div><div class="casAuthors">Shiau, Andrew K.; Barstad, Danielle; Loria, Paula M.; Cheng, Lin; Kushner, Peter J.; Agard, David A.; Greene, Geoffrey L.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, Massachusetts)</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">95</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">927-937</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Ligand-dependent activation of transcription by nuclear receptors (NRs) is mediated by interactions with coactivators.  Receptor agonists promote coactivator binding, and antagonists block coactivator binding.  Here we report the crystal structure of the human estrogen receptor α (hERα) ligand-binding domain (LBD) bound to both the agonist diethylstilbestrol (DES) and a peptide derived from the NR box II region of the coactivator GRIP1 and the crystal structure of the hERα LBD bound to the selective antagonist 4-hydroxytamoxifen (OHT).  In the DES-LBD-peptide complex, the peptide binds as a short α helix to a hydrophobic groove on the surface of the LBD.  In the OHT-LBD complex, helix 12 occludes the coactivator recognition groove by mimicking the interactions of the NR box peptide with the LBD.  These structures reveal the two distinct mechanisms by which structural features of OHT promote this "autoinhibitory" helix 12 conformation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrrjqf40oHhE7Vg90H21EOLACvtfcHk0lg5NMfClSVP8A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXjslyltQ%253D%253D&md5=074d616cf353f0d98cea5b26c4595c10</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2FS0092-8674%2800%2981717-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0092-8674%252800%252981717-1%26sid%3Dliteratum%253Aachs%26aulast%3DShiau%26aufirst%3DA.%2BK.%26aulast%3DBarstad%26aufirst%3DD.%26aulast%3DLoria%26aufirst%3DP.%2BM.%26aulast%3DCheng%26aufirst%3DL.%26aulast%3DKushner%26aufirst%3DP.%2BJ.%26aulast%3DAgard%26aufirst%3DD.%2BA.%26aulast%3DGreene%26aufirst%3DG.%2BL.%26atitle%3DThe%2520structural%2520basis%2520of%2520estrogen%2520receptor%252Fcoactivator%2520recognition%2520and%2520the%2520antagonism%2520of%2520this%2520interaction%2520by%2520tamoxifen%26jtitle%3DCell%26date%3D1998%26volume%3D95%26spage%3D927%26epage%3D937%26doi%3D10.1016%2FS0092-8674%2800%2981717-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Barillari, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marcou, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rognan, D.</span></span> <span> </span><span class="NLM_article-title">Hot-spots-guided receptor-based pharmacophores (HS-Pharm): a knowledge-based approach to identify ligand-anchoring atoms in protein cavities and prioritize structure-based pharmacophores</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">1396</span>– <span class="NLM_lpage">1410</span>, <span class="refDoi"> DOI: 10.1021/ci800064z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci800064z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A528%3ADC%252BD1cXnsFGktrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2008&pages=1396-1410&author=C.+Barillariauthor=G.+Marcouauthor=D.+Rognan&title=Hot-spots-guided+receptor-based+pharmacophores+%28HS-Pharm%29%3A+a+knowledge-based+approach+to+identify+ligand-anchoring+atoms+in+protein+cavities+and+prioritize+structure-based+pharmacophores&doi=10.1021%2Fci800064z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Hot-Spots-Guided Receptor-Based Pharmacophores (HS-Pharm): A Knowledge-Based Approach to Identify Ligand-Anchoring Atoms in Protein Cavities and Prioritize Structure-Based Pharmacophores</span></div><div class="casAuthors">Barillari, Caterina; Marcou, Gilles; Rognan, Didier</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1396-1410</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The design of biol. active compds. from ligand-free protein structures using a structure-based approach is still a major challenge.  In this paper, we present a fast knowledge-based approach (HS-Pharm) that allows the prioritization of cavity atoms that should be targeted for ligand binding, by training machine learning algorithms with atom-based fingerprints of known ligand-binding pockets.  The knowledge of hot spots for ligand binding is here used for focusing structure-based pharmacophore models.  Three targets of pharmacol. interest (neuraminidase, β2 adrenergic receptor, and cyclooxygenase-2) were used to test the evaluated methodol., and the derived structure-based pharmacophores were used in retrospective virtual screening studies.  The current study shows that structure-based pharmacophore screening is a powerful technique for the fast identification of potential hits in a chem. library, and that it is a valid alternative to virtual screening by mol. docking.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq1wuGUqc3VZbVg90H21EOLACvtfcHk0lgTwzrwexmxOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXnsFGktrg%253D&md5=75c0bc3d0e89f62f8382d3ff7e85fecc</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Fci800064z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci800064z%26sid%3Dliteratum%253Aachs%26aulast%3DBarillari%26aufirst%3DC.%26aulast%3DMarcou%26aufirst%3DG.%26aulast%3DRognan%26aufirst%3DD.%26atitle%3DHot-spots-guided%2520receptor-based%2520pharmacophores%2520%2528HS-Pharm%2529%253A%2520a%2520knowledge-based%2520approach%2520to%2520identify%2520ligand-anchoring%2520atoms%2520in%2520protein%2520cavities%2520and%2520prioritize%2520structure-based%2520pharmacophores%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2008%26volume%3D48%26spage%3D1396%26epage%3D1410%26doi%3D10.1021%2Fci800064z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gohlke, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hendlich, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klebe, G.</span></span> <span> </span><span class="NLM_article-title">Predicting binding modes, binding affinities and ‘hot spots’ for protein-ligand complexes using a knowledge-based scoring function</span>. <i>Perspect. Drug Discovery Des.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">115</span>– <span class="NLM_lpage">144</span>, <span class="refDoi"> DOI: 10.1023/A:1008781006867</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=10.1023%2FA%3A1008781006867" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A528%3ADC%252BD3MXptV2q" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2000&pages=115-144&author=H.+Gohlkeauthor=M.+Hendlichauthor=G.+Klebe&title=Predicting+binding+modes%2C+binding+affinities+and+%E2%80%98hot+spots%E2%80%99+for+protein-ligand+complexes+using+a+knowledge-based+scoring+function&doi=10.1023%2FA%3A1008781006867"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Predicting binding modes, binding affinities and "hot spots" for protein-ligand complexes using a knowledge-based scoring function</span></div><div class="casAuthors">Gohlke, Holger; Hendlich, Manfred; Klebe, Gerhard</div><div class="citationInfo"><span class="NLM_cas:title">Perspectives in Drug Discovery and Design</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">115-144</span>CODEN:
                <span class="NLM_cas:coden">PDDDEC</span>;
        ISSN:<span class="NLM_cas:issn">0928-2866</span>.
    
            (<span class="NLM_cas:orgname">Kluwer Academic Publishers</span>)
        </div><div class="casAbstract">The development of a new knowledge-based scoring function (DrugScore) and its power to recognize binding modes close to expt., to predict binding affinities, and to identify "hot spots" in binding pockets is presented.  Structural information is extd. from crystallog. detd. protein-ligand complexes using ReLiBase and converted into distance-dependent pair-preferences and solvent-accessible surface (SAS) dependent singlet preferences of protein and ligand atoms.  The sum of the pair preferences and the singlet preferences is calcd. using the 3D structure of protein-ligand complexes either taken directly from the x-ray structure or generated by the docking tool FlexX.  DrugScore discriminates efficiently between well-docked ligand binding modes (root-mean-square deviation <2.0 Å with respect to a crystallog. detd. ref. complex) and computer-generated ones largely deviating from the native structure.  For two test sets (91 and 68 protein-ligand complexes, taken from the PDB) the calcd. score recognizes poses deviating <2 Å from the crystal structure on rank 1 in three quarters of all possible cases.  Compared to the scoring function in FlexX, this is a substantial improvement.  For five test sets of crystallog. detd. protein-ligand complexes as well as for two sets of ligand geometries generated by FlexX, the calcd. score is correlated with exptl. detd. binding affinities.  For a set of 16 crystallog. detd. serine protease inhibitor complexes, a R2 value of 0.86 and a std. deviation of 0.95 log units is achieved as best result; for a set of 64 thrombin and trypsin inhibitors docked into their target proteins, a R2 value of 0.48 and a std. deviation of 0.7 log units is calcd.  DrugScore performs better than other state-of-the-art scoring functions.  To assess DrugScore's capability to reproduce the geometry of directional interactions correctly, "hot spots" are identified and visualized in terms of isocontour surfaces inside the binding pocket.  A data set of 159 x-ray protein-ligand complexes is used to reproduce and highlight the actually obsd. ligand atom positions.  In 74% of all cases, the actually obsd. atom type corresponds to an atom type predicted by the most favorable score at the nearest grid point.  The prediction rate increases to 85% if at least an atom type of the same class of interaction is suggested.  DrugScore is fast to compute and includes implicitly solvation and entropy contributions.  Small deviations in the 3D structure are tolerated and, since only contacts to non-hydrogen atoms are regarded, it does not require any assumptions on protonation states.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhQrM1Fu-n5bVg90H21EOLACvtfcHk0lgTwzrwexmxOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXptV2q&md5=4f330d644753d3729ad11d5ee5fa4f79</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1023%2FA%3A1008781006867&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FA%253A1008781006867%26sid%3Dliteratum%253Aachs%26aulast%3DGohlke%26aufirst%3DH.%26aulast%3DHendlich%26aufirst%3DM.%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3DPredicting%2520binding%2520modes%252C%2520binding%2520affinities%2520and%2520%25E2%2580%2598hot%2520spots%25E2%2580%2599%2520for%2520protein-ligand%2520complexes%2520using%2520a%2520knowledge-based%2520scoring%2520function%26jtitle%3DPerspect.%2520Drug%2520Discovery%2520Des.%26date%3D2000%26volume%3D20%26spage%3D115%26epage%3D144%26doi%3D10.1023%2FA%3A1008781006867" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Burgoyne, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jackson, R. M.</span></span> <span> </span><span class="NLM_article-title">Predicting protein interaction sites: binding hot-spots in protein-protein and protein-ligand interfaces</span>. <i>Bioinformatics</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">1335</span>– <span class="NLM_lpage">1342</span>, <span class="refDoi"> DOI: 10.1093/bioinformatics/btl079</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=10.1093%2Fbioinformatics%2Fbtl079" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=16522669" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A528%3ADC%252BD28XmtlOjsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2006&pages=1335-1342&author=N.+J.+Burgoyneauthor=R.+M.+Jackson&title=Predicting+protein+interaction+sites%3A+binding+hot-spots+in+protein-protein+and+protein-ligand+interfaces&doi=10.1093%2Fbioinformatics%2Fbtl079"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Predicting protein interaction sites: binding hot-spots in protein-protein and protein-ligand interfaces</span></div><div class="casAuthors">Burgoyne, Nicholas J.; Jackson, Richard M.</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1335-1342</span>CODEN:
                <span class="NLM_cas:coden">BOINFP</span>;
        ISSN:<span class="NLM_cas:issn">1367-4803</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Protein assemblies are currently poorly represented in structural databases and their structural elucidation is a key goal in biol.  Here we analyze clefts in protein surfaces, likely to correspond to binding 'hot-spots', and rank them according to sequence conservation and simple measures of phys. properties including hydrophobicity, desolvation, electrostatic and van der Waals potentials, to predict which are involved in binding in the native complex.  The resulting differences between predicting binding-sites at protein-protein and protein-ligand interfaces are striking.  There is a high level of prediction accuracy (≤93%) for protein-ligand interactions, based on the following attributes: van der Waals potential, electrostatic potential, desolvation and surface conservation.  Generally, the prediction accuracy for protein-protein interactions is lower, with the exception of enzymes.  Our results show that the ease of cleft desolvation is strongly predictive of interfaces and strongly maintained across all classes of protein-binding interface.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqD2TRwsq1iIbVg90H21EOLACvtfcHk0lgTwzrwexmxOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XmtlOjsb8%253D&md5=11cd0c8978e82ace530d45cb28449de8</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtl079&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtl079%26sid%3Dliteratum%253Aachs%26aulast%3DBurgoyne%26aufirst%3DN.%2BJ.%26aulast%3DJackson%26aufirst%3DR.%2BM.%26atitle%3DPredicting%2520protein%2520interaction%2520sites%253A%2520binding%2520hot-spots%2520in%2520protein-protein%2520and%2520protein-ligand%2520interfaces%26jtitle%3DBioinformatics%26date%3D2006%26volume%3D22%26spage%3D1335%26epage%3D1342%26doi%3D10.1093%2Fbioinformatics%2Fbtl079" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pei, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, L.</span></span> <span> </span><span class="NLM_article-title">Protein-protein interface analysis and hot spots identification for chemical ligand design</span>. <i>Curr. Pharm. Des.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">1192</span>– <span class="NLM_lpage">1200</span>, <span class="refDoi"> DOI: 10.2174/13816128113199990065</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=10.2174%2F13816128113199990065" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=23713772" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A528%3ADC%252BC2cXksFSgs7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2014&pages=1192-1200&author=J.+Chenauthor=X.+Maauthor=Y.+Yuanauthor=J.+Peiauthor=L.+Lai&title=Protein-protein+interface+analysis+and+hot+spots+identification+for+chemical+ligand+design&doi=10.2174%2F13816128113199990065"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Protein-Protein Interface Analysis and Hot Spots Identification for Chemical Ligand Design</span></div><div class="casAuthors">Chen, Jing; Ma, Xiaomin; Yuan, Yaxia; Pei, Jianfeng; Lai, Luhua</div><div class="citationInfo"><span class="NLM_cas:title">Current Pharmaceutical Design</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1192-1200</span>CODEN:
                <span class="NLM_cas:coden">CPDEFP</span>;
        ISSN:<span class="NLM_cas:issn">1381-6128</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Rational design for chem. compds. targeting protein-protein interactions has grown from a dream to reality after a decade of efforts.  There are an increasing no. of successful examples, though major challenges remain in the field.  In this paper, we will first give a brief review of the available methods that can be used to analyze protein-protein interface and predict hot spots for chem. ligand design.  New developments of binding sites detection, ligandability and hot spots prediction from the author's group will also be described.  Pocket V.3 is an improved program for identifying hot spots in protein-protein interface using only an apo protein structure.  It has been developed based on Pocket V.2 that can derive receptor-based pharmacophore model for ligand binding cavity.  Given similarities and differences between the essence of pharmacophore and hot spots for guiding design of chem. compds., not only energetic but also spatial properties of protein-protein interface are used in Pocket V.3 for dealing with protein-protein interface.  In order to illustrate the capability of Pocket V.3, two datasets have been used.  One is taken from ASEdb and BID having exptl. alanine scanning results for testing hot spots prediction.  The other is taken from the 2P2I database contg. complex structures of protein-ligand binding at the original protein-protein interface for testing hot spots application in ligand design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoeQadx7vaGVbVg90H21EOLACvtfcHk0ljjZmHdVW5lQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXksFSgs7c%253D&md5=7d51f3f146bcba0d7b0ec56ebb289756</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.2174%2F13816128113199990065&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F13816128113199990065%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DMa%26aufirst%3DX.%26aulast%3DYuan%26aufirst%3DY.%26aulast%3DPei%26aufirst%3DJ.%26aulast%3DLai%26aufirst%3DL.%26atitle%3DProtein-protein%2520interface%2520analysis%2520and%2520hot%2520spots%2520identification%2520for%2520chemical%2520ligand%2520design%26jtitle%3DCurr.%2520Pharm.%2520Des.%26date%3D2014%26volume%3D20%26spage%3D1192%26epage%3D1200%26doi%3D10.2174%2F13816128113199990065" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rao, B. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, U. C.</span></span> <span> </span><span class="NLM_article-title">A free energy perturbation study of solvation in methanol and dimethyl sulfoxide</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">1990</span>,  <span class="NLM_volume"><i>112</i></span>,  <span class="NLM_fpage">3803</span>– <span class="NLM_lpage">3811</span>, <span class="refDoi"> DOI: 10.1021/ja00166a014</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja00166a014" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A528%3ADyaK3cXitFGrsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=112&publication_year=1990&pages=3803-3811&author=B.+G.+Raoauthor=U.+C.+Singh&title=A+free+energy+perturbation+study+of+solvation+in+methanol+and+dimethyl+sulfoxide&doi=10.1021%2Fja00166a014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">A free energy perturbation study of solvation in methanol and dimethyl sulfoxide</span></div><div class="casAuthors">Rao, B. G.; Singh, U. Chandra</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1990</span>),
    <span class="NLM_cas:volume">112</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3803-11</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    </div><div class="casAbstract">Relative differences in free energies of solvation, ΔG, between two different solutes in water, methanol, and DMSO have been detd. by a coordinate coupled free energy perturbation method.  The solutes examd. in this study are some closed shell ions, tetraalkylammonium ions, normal alkanes, and tetraalkylmethane mols.  The ΔG varies differently in different solvents for the mutation of a solute into another solute.  These differences are due to the differences in the solvation processes in different solvents.  For closed shell ions, the variation of ΔG shows greater sensitivity to change in the size of an anion in protic solvents than in dipolar aprotic solvent, while it is almost equally affected by the change in the size of a cation in all the three solvents.  The structural results of pure liq. and ionic solns. of DMSO obtained by the mol. dynamics simulations are discussed to rationalize the shape of the ΔG curve for mutations of the closed shell ions in DMSO.  In the case of large tetraalkylammonium ions and alkanes, the variation of ΔG in water is dictated by both the structure of the tight solvent shell around these solutes and the solute-solvent interaction energies.  On the other hand, the variation of ΔG for these solutes in methanol and DMSO are detd. mainly by the solute-solvent interaction energies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGruwwgK6SvhL7Vg90H21EOLACvtfcHk0ljjZmHdVW5lQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3cXitFGrsb4%253D&md5=442ab2765b2ac287e7723a17087370dc</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Fja00166a014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja00166a014%26sid%3Dliteratum%253Aachs%26aulast%3DRao%26aufirst%3DB.%2BG.%26aulast%3DSingh%26aufirst%3DU.%2BC.%26atitle%3DA%2520free%2520energy%2520perturbation%2520study%2520of%2520solvation%2520in%2520methanol%2520and%2520dimethyl%2520sulfoxide%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1990%26volume%3D112%26spage%3D3803%26epage%3D3811%26doi%3D10.1021%2Fja00166a014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kollman, P.</span></span> <span> </span><span class="NLM_article-title">Free energy calculations: applications to chemical and biochemical phenomena</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>93</i></span>,  <span class="NLM_fpage">2395</span>– <span class="NLM_lpage">2417</span>, <span class="refDoi"> DOI: 10.1021/cr00023a004</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr00023a004" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A528%3ADyaK3sXmt1Sktr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=93&publication_year=1993&pages=2395-2417&author=P.+Kollman&title=Free+energy+calculations%3A+applications+to+chemical+and+biochemical+phenomena&doi=10.1021%2Fcr00023a004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Free energy calculations: Applications to chemical and biochemical phenomena</span></div><div class="casAuthors">Kollman, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">93</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2395-417</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    </div><div class="casAbstract">A review with 252 refs. about applications of free energy calcns. employing mol. dynamics or Monte Carlo methods to a variety of chem. and biochem. phenomena.  The focus is on applications of such calcns. to mol. solvation, mol. assocn., macromol. stability, and enzyme catalysis.  The mols. discussed range from monovalent ions and small mols. to proteins and nucleic acids.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrN1w4SyFDqCLVg90H21EOLACvtfcHk0ljjZmHdVW5lQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXmt1Sktr0%253D&md5=f12326b24734ea092996cf22efb6ebd6</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fcr00023a004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr00023a004%26sid%3Dliteratum%253Aachs%26aulast%3DKollman%26aufirst%3DP.%26atitle%3DFree%2520energy%2520calculations%253A%2520applications%2520to%2520chemical%2520and%2520biochemical%2520phenomena%26jtitle%3DChem.%2520Rev.%26date%3D1993%26volume%3D93%26spage%3D2395%26epage%3D2417%26doi%3D10.1021%2Fcr00023a004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jorgensen, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, L. L.</span></span> <span> </span><span class="NLM_article-title">Perspective on free-energy perturbation calculations for chemical equilibria</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">869</span>– <span class="NLM_lpage">876</span>, <span class="refDoi"> DOI: 10.1021/ct800011m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct800011m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A528%3ADC%252BD1cXlslOisLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2008&pages=869-876&author=W.+L.+Jorgensenauthor=L.+L.+Thomas&title=Perspective+on+free-energy+perturbation+calculations+for+chemical+equilibria&doi=10.1021%2Fct800011m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Perspective on Free-Energy Perturbation Calculations for Chemical Equilibria</span></div><div class="casAuthors">Jorgensen, William L.; Thomas, Laura L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">869-876</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">An overview has been provided on the history of free-energy perturbation (FEP) calcns. and their application to computation of relative free energies of solvation.  In view of the expanding role of FEP calcns. for pursuing a wide range of problems in chem. and biochem., including mol. design, it is important to establish reliable protocols for their execution.  For 30 interconversions of substituted benzenes, which represent prototypical cases for mol. design, alternative sampling procedures and nos. of FEP windows were considered.  Simple overlap sampling with 11 or 15 windows and double-wide sampling with 14 windows emerged as viable procedures for such routine FEP calcns.; the present protocols with these sampling methods yield statistical uncertainties of 0.2 kcal/mol for free-energy differences.  The slightly greater accuracy obtainable with 21 to 25 windows does not clearly justify the increased costs, while overlap sampling with 6 or 8 windows may be useful for preliminary exploration of large nos. of mol. changes.  Application of the power-series expansion of the Zwanzig equation was also tested.  Use of the first-order term is generally reliable, while inclusion of the slowly convergent, second-order fluctuation term causes deterioration in the results for strongly hydrogen-bonded solutes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqukDgvYtLny7Vg90H21EOLACvtfcHk0ljjZmHdVW5lQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXlslOisLw%253D&md5=4f6cd47142dbbcd51e94aedd2d878538</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Fct800011m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct800011m%26sid%3Dliteratum%253Aachs%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26aulast%3DThomas%26aufirst%3DL.%2BL.%26atitle%3DPerspective%2520on%2520free-energy%2520perturbation%2520calculations%2520for%2520chemical%2520equilibria%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2008%26volume%3D4%26spage%3D869%26epage%3D876%26doi%3D10.1021%2Fct800011m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zacharias, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Straatsma, T. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCammon, J. A.</span></span> <span> </span><span class="NLM_article-title">Separation-shifted scaling, a new scaling method for Lennard-Jones interactions in thermodynamic integration</span>. <i>J. Chem. Phys.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>100</i></span>,  <span class="NLM_fpage">9025</span>– <span class="NLM_lpage">9031</span>, <span class="refDoi"> DOI: 10.1063/1.466707</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=10.1063%2F1.466707" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A528%3ADyaK2cXltFyrsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=100&publication_year=1994&pages=9025-9031&author=M.+Zachariasauthor=T.+P.+Straatsmaauthor=J.+A.+McCammon&title=Separation-shifted+scaling%2C+a+new+scaling+method+for+Lennard-Jones+interactions+in+thermodynamic+integration&doi=10.1063%2F1.466707"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Separation-shifted scaling, a new scaling method for Lennard-Jones interactions in thermodynamic integration</span></div><div class="casAuthors">Zacharias, M.; Straatsma, T. P.; McCammon, J. A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">100</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">9025-31</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    </div><div class="casAbstract">A new method of simultaneously scaling and shifting the Lennard-Jones (LJ) potential in mol. dynamics (MD) and thermodn. integration (TI) simulations is presented.  The approach allows the smooth creation or annihilation of atoms or mols. in an ensemble of solvent mols. during a mol. simulation.  By scaling and shifting the LJ potential in the direction of the interat. distance between particles, the method eliminates the problem of the creation or annihilation of a large repulsive LJ potential at the initial or final state of a TI.  The optimal degree of shifting and scaling the LJ potential as a function of a control variable λ was studied for the annihilation and creation of neon in aq. soln.  The method was further tested on the calcn. of the free energy of aq. solvation of a small mol., ethanol.  In contrast to linear scaling of the LJ potential during TI, the calcd. free energies using the new sepn.-shifted scaling approach are reasonably well converged after 200-500 ps of simulation and show smaller hysteresis comparing forward and reverse TI.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp749QLLl9dubVg90H21EOLACvtfcHk0lhCdqKvdWp36Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXltFyrsLw%253D&md5=82654e60072c5a954188276cea3b468a</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1063%2F1.466707&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.466707%26sid%3Dliteratum%253Aachs%26aulast%3DZacharias%26aufirst%3DM.%26aulast%3DStraatsma%26aufirst%3DT.%2BP.%26aulast%3DMcCammon%26aufirst%3DJ.%2BA.%26atitle%3DSeparation-shifted%2520scaling%252C%2520a%2520new%2520scaling%2520method%2520for%2520Lennard-Jones%2520interactions%2520in%2520thermodynamic%2520integration%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1994%26volume%3D100%26spage%3D9025%26epage%3D9031%26doi%3D10.1063%2F1.466707" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kollman, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Massova, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reyes, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuhn, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chong, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Calculating structures and free energies of complex molecules: combining molecular mechanics and continuum models</span>. <i>Acc. Chem. Res.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">889</span>– <span class="NLM_lpage">897</span>, <span class="refDoi"> DOI: 10.1021/ar000033j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ar000033j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A528%3ADC%252BD3cXmvFGiu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2000&pages=889-897&author=P.+A.+Kollmanauthor=I.+Massovaauthor=C.+Reyesauthor=B.+Kuhnauthor=S.+Huoauthor=L.+Chongauthor=M.+Leeauthor=T.+Leeauthor=Y.+Duanauthor=W.+Wang&title=Calculating+structures+and+free+energies+of+complex+molecules%3A+combining+molecular+mechanics+and+continuum+models&doi=10.1021%2Far000033j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Calculating Structures and Free Energies of Complex Molecules: Combining Molecular Mechanics and Continuum Models</span></div><div class="casAuthors">Kollman, Peter A.; Massova, Irina; Reyes, Carolina; Kuhn, Bernd; Huo, Shuanghong; Chong, Lillian; Lee, Matthew; Lee, Taisung; Duan, Yong; Wang, Wei; Donini, Oreola; Cieplak, Piotr; Srinivasan, Jaysharee; Case, David A.; Cheatham, Thomas E., III</div><div class="citationInfo"><span class="NLM_cas:title">Accounts of Chemical Research</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">889-897</span>CODEN:
                <span class="NLM_cas:coden">ACHRE4</span>;
        ISSN:<span class="NLM_cas:issn">0001-4842</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review, with 63 refs.  A historical perspective on the application of mol. dynamics (MD) to biol. macromols. is presented.  Recent developments combining state-of-the-art force fields with continuum solvation calcns. have allowed us to reach the fourth era of MD applications in which one can often derive both accurate structure and accurate relative free energies from mol. dynamics trajectories.  We illustrate such applications on nucleic acid duplexes, RNA hairpins, protein folding trajectories, and protein-ligand, protein-protein, and protein-nucleic acid interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXGhwlfA3kvLVg90H21EOLACvtfcHk0lhCdqKvdWp36Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXmvFGiu7g%253D&md5=8436ee610ae145894428db1a1deff73c</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Far000033j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Far000033j%26sid%3Dliteratum%253Aachs%26aulast%3DKollman%26aufirst%3DP.%2BA.%26aulast%3DMassova%26aufirst%3DI.%26aulast%3DReyes%26aufirst%3DC.%26aulast%3DKuhn%26aufirst%3DB.%26aulast%3DHuo%26aufirst%3DS.%26aulast%3DChong%26aufirst%3DL.%26aulast%3DLee%26aufirst%3DM.%26aulast%3DLee%26aufirst%3DT.%26aulast%3DDuan%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DW.%26atitle%3DCalculating%2520structures%2520and%2520free%2520energies%2520of%2520complex%2520molecules%253A%2520combining%2520molecular%2520mechanics%2520and%2520continuum%2520models%26jtitle%3DAcc.%2520Chem.%2520Res.%26date%3D2000%26volume%3D33%26spage%3D889%26epage%3D897%26doi%3D10.1021%2Far000033j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kuhn, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerber, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schulz-Gasch, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stahl, M.</span></span> <span> </span><span class="NLM_article-title">Validation and use of the MM-PBSA approach for drug discovery</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">4040</span>– <span class="NLM_lpage">4048</span>, <span class="refDoi"> DOI: 10.1021/jm049081q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm049081q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A528%3ADC%252BD2MXjvFKgu74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=4040-4048&author=B.+Kuhnauthor=P.+Gerberauthor=T.+Schulz-Gaschauthor=M.+Stahl&title=Validation+and+use+of+the+MM-PBSA+approach+for+drug+discovery&doi=10.1021%2Fjm049081q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Validation and Use of the MM-PBSA Approach for Drug Discovery</span></div><div class="casAuthors">Kuhn, Bernd; Gerber, Paul; Schulz-Gasch, Tanja; Stahl, Martin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">4040-4048</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The MM-PBSA approach has become a popular method for calcg. binding affinities of biomol. complexes.  Published application examples focus on small test sets and few proteins and, hence, are of limited relevance in assessing the general validity of this method.  To further characterize MM-PBSA, the authors report on a more extensive study involving a large no. of ligands and eight different proteins.  Our results show that applying the MM-PBSA energy function to a single, relaxed complex structure is an adequate and sometimes more accurate approach than the std. free energy averaging over mol. dynamics snapshots.  The use of MM-PBSA on a single structure is shown to be valuable (a) as a postdocking filter in further enriching virtual screening results, (b) as a helpful tool to prioritize de novo design solns., and (c) for distinguishing between good and weak binders (ΔpIC50 ≥ 2-3), but rarely to reproduce smaller free energy differences.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGorwcFgm2UDiLVg90H21EOLACvtfcHk0lhCdqKvdWp36Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXjvFKgu74%253D&md5=277858e33c944736d7f725359c265e10</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Fjm049081q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm049081q%26sid%3Dliteratum%253Aachs%26aulast%3DKuhn%26aufirst%3DB.%26aulast%3DGerber%26aufirst%3DP.%26aulast%3DSchulz-Gasch%26aufirst%3DT.%26aulast%3DStahl%26aufirst%3DM.%26atitle%3DValidation%2520and%2520use%2520of%2520the%2520MM-PBSA%2520approach%2520for%2520drug%2520discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D4040%26epage%3D4048%26doi%3D10.1021%2Fjm049081q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Massova, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kollman, P. A.</span></span> <span> </span><span class="NLM_article-title">Combined molecular mechanical and continuum solvent approach (MM-PBSA/GBSA) to predict ligand binding</span>. <i>Perspect. Drug Discovery Des.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">113</span>– <span class="NLM_lpage">135</span>, <span class="refDoi"> DOI: 10.1023/A:1008763014207</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=10.1023%2FA%3A1008763014207" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A528%3ADC%252BD3cXis12ju7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2000&pages=113-135&author=I.+Massovaauthor=P.+A.+Kollman&title=Combined+molecular+mechanical+and+continuum+solvent+approach+%28MM-PBSA%2FGBSA%29+to+predict+ligand+binding&doi=10.1023%2FA%3A1008763014207"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Combined molecular mechanical and continuum solvent approach (MM-PBSA/GBSA) to predict ligand binding</span></div><div class="casAuthors">Massova, Irina; Kollman, Peter A.</div><div class="citationInfo"><span class="NLM_cas:title">Perspectives in Drug Discovery and Design</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">Hydrophobicity and Solvation in Drug Design, Pt. II</span>),
    <span class="NLM_cas:pages">113-135</span>CODEN:
                <span class="NLM_cas:coden">PDDDEC</span>;
        ISSN:<span class="NLM_cas:issn">0928-2866</span>.
    
            (<span class="NLM_cas:orgname">Kluwer Academic Publishers</span>)
        </div><div class="casAbstract">Significant progress has been achieved in computational methods to treat solvent effects in recent years.  Among various techniques, the continuum solvent approach appears to be practically promising because it can be used to calc. reliable interaction and solvation energies in complex systems.  A computational scanning mutagenesis method, one of such new approaches, has been recently developed.  It combines the mol. mech. and continuum solvent approaches and allows one to identify the "hot spots" in binding interfaces from a single trajectory of a wild type complex.  Such techniques can be also used as a tool to optimize the interacting species for the binding, or as a ranking procedure in high throughput screening.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp4IQPgMmXRBbVg90H21EOLACvtfcHk0lhQhFDK-8wSpA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXis12ju7Y%253D&md5=520df05a7c7ed02fdf76d3e1e6f377f9</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1023%2FA%3A1008763014207&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FA%253A1008763014207%26sid%3Dliteratum%253Aachs%26aulast%3DMassova%26aufirst%3DI.%26aulast%3DKollman%26aufirst%3DP.%2BA.%26atitle%3DCombined%2520molecular%2520mechanical%2520and%2520continuum%2520solvent%2520approach%2520%2528MM-PBSA%252FGBSA%2529%2520to%2520predict%2520ligand%2520binding%26jtitle%3DPerspect.%2520Drug%2520Discovery%2520Des.%26date%3D2000%26volume%3D18%26spage%3D113%26epage%3D135%26doi%3D10.1023%2FA%3A1008763014207" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Keskin, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gursoy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nussinov, R.</span></span> <span> </span><span class="NLM_article-title">Principles of protein-protein interactions: what are the preferred ways for proteins to interact?</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>108</i></span>,  <span class="NLM_fpage">1225</span>– <span class="NLM_lpage">1244</span>, <span class="refDoi"> DOI: 10.1021/cr040409x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr040409x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A528%3ADC%252BD1cXjsVOmtrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=108&publication_year=2008&pages=1225-1244&author=O.+Keskinauthor=A.+Gursoyauthor=B.+Maauthor=R.+Nussinov&title=Principles+of+protein-protein+interactions%3A+what+are+the+preferred+ways+for+proteins+to+interact%3F&doi=10.1021%2Fcr040409x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Principles of protein-protein interactions: What are the preferred ways for proteins To interact?</span></div><div class="casAuthors">Keskin, Ozlem; Gursoy, Attila; Ma, Buyong; Nussinov, Ruth</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">108</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1225-1244</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  An overview of the principles that govern protein-protein interactions is provided by the authors.  Within this framework the authors highlight what they consider are key components in the question of "what are the preferred ways for proteins To interact.".  The goal is to be able to predict how the proteins will interact.  A description of binding sites in terms of preferred residue, and particularly preferred region and architectural organizations, may lead to classification strategies assisting in predictions of the preferred ways for proteins to interact.  Within the recurring favorable architectures, there are preferred cooperative "hot spot" organizations.  Preferred organization is a key in chem. and in protein science, whether in amyloid microfilaments or in globular protein-protein assocns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpEjFoMyzUh67Vg90H21EOLACvtfcHk0lhQhFDK-8wSpA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXjsVOmtrs%253D&md5=1028d6a1c43c8ad1a98286268101c305</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Fcr040409x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr040409x%26sid%3Dliteratum%253Aachs%26aulast%3DKeskin%26aufirst%3DO.%26aulast%3DGursoy%26aufirst%3DA.%26aulast%3DMa%26aufirst%3DB.%26aulast%3DNussinov%26aufirst%3DR.%26atitle%3DPrinciples%2520of%2520protein-protein%2520interactions%253A%2520what%2520are%2520the%2520preferred%2520ways%2520for%2520proteins%2520to%2520interact%253F%26jtitle%3DChem.%2520Rev.%26date%3D2008%26volume%3D108%26spage%3D1225%26epage%3D1244%26doi%3D10.1021%2Fcr040409x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schreiber, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haran, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, H. X.</span></span> <span> </span><span class="NLM_article-title">Fundamental aspects of protein-protein association kinetics</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>109</i></span>,  <span class="NLM_fpage">839</span>– <span class="NLM_lpage">860</span>, <span class="refDoi"> DOI: 10.1021/cr800373w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr800373w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsFWns7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2009&pages=839-860&author=G.+Schreiberauthor=G.+Haranauthor=H.+X.+Zhou&title=Fundamental+aspects+of+protein-protein+association+kinetics&doi=10.1021%2Fcr800373w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Fundamental Aspects of Protein-Protein Association Kinetics</span></div><div class="casAuthors">Schreiber, G.; Haran, G.; Zhou, H.-X.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">839-860</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  This review focuses on recent advances in deciphering the kinetic pathway of protein complex formation, the nature of the precomplex formed through diffusion (which the authors term the "transient complex"), the transition state and other intermediates (such as the so-called encounter complex) along the assocn. pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqBsx0HWzD2W7Vg90H21EOLACvtfcHk0lhQhFDK-8wSpA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsFWns7w%253D&md5=fbc492ede3242af279ee42705d23fdec</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Fcr800373w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr800373w%26sid%3Dliteratum%253Aachs%26aulast%3DSchreiber%26aufirst%3DG.%26aulast%3DHaran%26aufirst%3DG.%26aulast%3DZhou%26aufirst%3DH.%2BX.%26atitle%3DFundamental%2520aspects%2520of%2520protein-protein%2520association%2520kinetics%26jtitle%3DChem.%2520Rev.%26date%3D2009%26volume%3D109%26spage%3D839%26epage%3D860%26doi%3D10.1021%2Fcr800373w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jones, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thornton, J. M.</span></span> <span> </span><span class="NLM_article-title">Principles of protein-protein interactions</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>93</i></span>,  <span class="NLM_fpage">13</span>– <span class="NLM_lpage">20</span>, <span class="refDoi"> DOI: 10.1073/pnas.93.1.13</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=10.1073%2Fpnas.93.1.13" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=8552589" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A528%3ADyaK28XjslOrsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=93&publication_year=1996&pages=13-20&author=S.+Jonesauthor=J.+M.+Thornton&title=Principles+of+protein-protein+interactions&doi=10.1073%2Fpnas.93.1.13"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Principles of protein-protein interactions</span></div><div class="casAuthors">Jones, Susan; Thornton, Janet M.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">93</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">13-20</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">A review, with 47 refs., examg. protein complexes in the Brookhaven Protein Databank to gain a better understanding of the principles governing the interactions involved in protein-protein recognition.  The factors that influence the formation of protein-protein complexes are explored in four different types of protein-protein complexes: homodimeric proteins, heterodimeric proteins, enzyme-inhibitor complexes, and antibody-protein complexes.  The comparison between the complexes highlights differences that reflect their biol. roles.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr13RlSj8fKFLVg90H21EOLACvtfcHk0lhQhFDK-8wSpA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XjslOrsQ%253D%253D&md5=2b03d8f835104e5a7bf5da83afd4a8d1</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1073%2Fpnas.93.1.13&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.93.1.13%26sid%3Dliteratum%253Aachs%26aulast%3DJones%26aufirst%3DS.%26aulast%3DThornton%26aufirst%3DJ.%2BM.%26atitle%3DPrinciples%2520of%2520protein-protein%2520interactions%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1996%26volume%3D93%26spage%3D13%26epage%3D20%26doi%3D10.1073%2Fpnas.93.1.13" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gutgesell, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karumudi, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tonetti, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thatcher, G. R. J.</span></span> <span> </span><span class="NLM_article-title">Novel selective estrogen receptor downregulators (SERDs) developed against treatment-resistant breast cancer</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">1325</span>– <span class="NLM_lpage">1342</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01355</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01355" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFWqu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=1325-1342&author=R.+Xiongauthor=J.+Zhaoauthor=L.+M.+Gutgesellauthor=Y.+Wangauthor=S.+Leeauthor=B.+Karumudiauthor=H.+Zhaoauthor=Y.+Luauthor=D.+A.+Tonettiauthor=G.+R.+J.+Thatcher&title=Novel+selective+estrogen+receptor+downregulators+%28SERDs%29+developed+against+treatment-resistant+breast+cancer&doi=10.1021%2Facs.jmedchem.6b01355"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Novel Selective Estrogen Receptor Downregulators (SERDs) Developed against Treatment-Resistant Breast Cancer</span></div><div class="casAuthors">Xiong, Rui; Zhao, Jiong; Gutgesell, Lauren M.; Wang, Yueting; Lee, Sue; Karumudi, Bhargava; Zhao, Huiping; Lu, Yunlong; Tonetti, Debra A.; Thatcher, Gregory R. J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1325-1342</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Resistance to the selective estrogen receptor modulator (SERM) tamoxifen and to aromatase inhibitors that lower circulating estradiol occurs in up to 50% of patients, generally leading to an endocrine-independent ER+ phenotype.  Selective ER downregulators (SERDs) are able to ablate ER and thus theor. to prevent survival of both endocrine-dependent and independent ER+ tumors.  The clin. SERD, fulvestrant, is hampered by i.m. administration and undesirable pharmacokinetics.  Novel SERDs were designed using the 6-OH-benzothiophene (BT) scaffold common to arzoxifene and raloxifene.  Treatment-resistant (TR) ER+ cell lines (MCF-7:5C and MCF-7:TAM1) were used for optimization, followed by validation in the parent endocrine-dependent cell line (MCF-7:WS8), in 2D and 3D cultures, using ERα in-cell westerns, ERE-luciferase, and cell viability assays, with GDC-0810 (ARN-810) used for comparison.  Two BT SERDs with superior in vitro activity to GDC-0810 were studied for bioavailability and shown to cause regression of a TR, endocrine-independent ER+ xenograft superior to GDC-0810.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLqe0A8df78rVg90H21EOLACvtfcHk0lhGrfU__UpazA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFWqu70%253D&md5=71dbdfa2152f9e9019e8d138ed41313e</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01355&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01355%26sid%3Dliteratum%253Aachs%26aulast%3DXiong%26aufirst%3DR.%26aulast%3DZhao%26aufirst%3DJ.%26aulast%3DGutgesell%26aufirst%3DL.%2BM.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DLee%26aufirst%3DS.%26aulast%3DKarumudi%26aufirst%3DB.%26aulast%3DZhao%26aufirst%3DH.%26aulast%3DLu%26aufirst%3DY.%26aulast%3DTonetti%26aufirst%3DD.%2BA.%26aulast%3DThatcher%26aufirst%3DG.%2BR.%2BJ.%26atitle%3DNovel%2520selective%2520estrogen%2520receptor%2520downregulators%2520%2528SERDs%2529%2520developed%2520against%2520treatment-resistant%2520breast%2520cancer%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D1325%26epage%3D1342%26doi%3D10.1021%2Facs.jmedchem.6b01355" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brzozowski, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pike, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dauter, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hubbard, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonn, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engström, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ö hman, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greene, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gustafsson, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlquist, M.</span></span> <span> </span><span class="NLM_article-title">Molecular basis of agonism and antagonism in the oestrogen receptor</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>389</i></span>,  <span class="NLM_fpage">753</span>– <span class="NLM_lpage">758</span>, <span class="refDoi"> DOI: 10.1038/39645</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=10.1038%2F39645" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=9338790" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A528%3ADyaK2sXmvVSnt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=389&publication_year=1997&pages=753-758&author=A.+M.+Brzozowskiauthor=A.+C.+Pikeauthor=Z.+Dauterauthor=R.+E.+Hubbardauthor=T.+Bonnauthor=O.+Engstr%C3%B6mauthor=L.+%C3%96+hmanauthor=G.+L.+Greeneauthor=J.+A.+Gustafssonauthor=M.+Carlquist&title=Molecular+basis+of+agonism+and+antagonism+in+the+oestrogen+receptor&doi=10.1038%2F39645"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular basis of agonism and antagonism in the estrogen receptor</span></div><div class="casAuthors">Brzozowski, Andrzej M.; Pike, Ashley C. W.; Dauter, Zbigniew; Hubbard, Roderick E.; Bonn, Tomas; Engstrom, Owe; Ohman, Lars; Greene, Geoffrey L.; Gustafsson, Jan-Ake; Carlquist, Mats</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London)</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">389</span>
        (<span class="NLM_cas:issue">6652</span>),
    <span class="NLM_cas:pages">753-758</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Macmillan Magazines</span>)
        </div><div class="casAbstract">Estrogens are involved in the growth, development and homeostasis of a no. of tissues.  The physiol. effects of these steroids are mediated by a ligand-inducible nuclear transcription factor, the estrogen receptor (ER).  Hormone binding to the ligand-binding domain (LBD) of the ER initiates a series of mol. events culminating in the activation or repression of target genes.  Transcriptional regulation arises from the direct interaction of the ER with components of the cellular transcription machinery.  Here we report the crystal structures of the LBD of ER in complex with the endogenous estrogen, 17β-estradiol, and the selective antagonist raloxifene, at resolns. of 3.1 and 2.6Å, resp.  The structures provide a mol. basis for the distinctive pharmacophore of the ER and its catholic binding properties.  Agonist and antagonist bind at the same site within the core of the LBD but demonstrate different binding modes.  In addn., each class of ligand induces a distinct conformation in the transactivation domain of the LBD, providing structural evidence of the mechanism of antagonism.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAXbSo66V1LrVg90H21EOLACvtfcHk0lhGrfU__UpazA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXmvVSnt74%253D&md5=2a63951b881b329ff142c9fbc82a13e3</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1038%2F39645&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F39645%26sid%3Dliteratum%253Aachs%26aulast%3DBrzozowski%26aufirst%3DA.%2BM.%26aulast%3DPike%26aufirst%3DA.%2BC.%26aulast%3DDauter%26aufirst%3DZ.%26aulast%3DHubbard%26aufirst%3DR.%2BE.%26aulast%3DBonn%26aufirst%3DT.%26aulast%3DEngstr%25C3%25B6m%26aufirst%3DO.%26aulast%3D%25C3%2596%2Bhman%26aufirst%3DL.%26aulast%3DGreene%26aufirst%3DG.%2BL.%26aulast%3DGustafsson%26aufirst%3DJ.%2BA.%26aulast%3DCarlquist%26aufirst%3DM.%26atitle%3DMolecular%2520basis%2520of%2520agonism%2520and%2520antagonism%2520in%2520the%2520oestrogen%2520receptor%26jtitle%3DNature%26date%3D1997%26volume%3D389%26spage%3D753%26epage%3D758%26doi%3D10.1038%2F39645" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mohammad, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gong, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">January, C. T.</span></span> <span> </span><span class="NLM_article-title">Blockage of the HERG human cardiac K<sup>+</sup> channel by the gastrointestinal prokinetic agent cisapride</span>. <i>Am. J. Physiol.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>273</i></span>,  <span class="NLM_fpage">H2534</span>– <span class="NLM_lpage">H2538</span>, <span class="refDoi"> DOI: 10.1152/ajpheart.1997.273.5.H2534</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=10.1152%2Fajpheart.1997.273.5.H2534" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=9374794" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A528%3ADyaK2sXnsVCqu7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=273&publication_year=1997&pages=H2534-H2538&author=S.+Mohammadauthor=Z.+Zhouauthor=Q.+Gongauthor=C.+T.+January&title=Blockage+of+the+HERG+human+cardiac+K%2B+channel+by+the+gastrointestinal+prokinetic+agent+cisapride&doi=10.1152%2Fajpheart.1997.273.5.H2534"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Blockage of the HERG human cardiac K+ channel by the gastrointestinal prokinetic agent cisapride</span></div><div class="casAuthors">Mohammad, Saeed; Zhou, Zhengfeng; Gong, Qiuming; January, Craig T.</div><div class="citationInfo"><span class="NLM_cas:title">American Journal of Physiology</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">273</span>
        (<span class="NLM_cas:issue">5, Pt. 2</span>),
    <span class="NLM_cas:pages">H2534-H2538</span>CODEN:
                <span class="NLM_cas:coden">AJPHAP</span>;
        ISSN:<span class="NLM_cas:issn">0002-9513</span>.
    
            (<span class="NLM_cas:orgname">American Physiological Society</span>)
        </div><div class="casAbstract">Cisapride, a gastrointestinal prokinetic agent, is known to caused long Q-T syndrome and ventricular arrhythmias.  The cellular mechanism is not known.  The human ether-a-go-go-related gene (HERG), which encodes the rapidly activating delayed rectifier K+ current and is important in cardiac repolarization, may serve as a target for the action of cisapride.  The authors tested the hypothesis that cisapride blocks HERG.  The whole cell path-clamp recording technique was used to study HERG channels stably expressed heterologously in HEK293 cells.  Under voltage-clamp conditions, cisapride block of HERG is dose dependent with a half-maximal inhibitory concn. of 6.5 nM at 22° (cells).  Currents rapidly recovered with drug washout.  The onset of block by cisapride required channel activation indicative of open or inactivated state blockage.  Block of HERG with cisapride after channel activation was voltage dependent.  At -20 mV, 10 nM cisapride reduced HERG tail-current amplitude by 5%, whereas, at +20 mV, the tail-current amplitude was reduced by 45% (cells).  At -20 and +20 mV, 100 nM cisapride reduced tail-current amplitude by 66 and 90%, resp.  The authors conclude that cisapride is a potent blocker of HERG channels expressed in HEK293 cells.  This effect may account for the clin. occurrence of Q-T prolongation and ventricular arrhythmias obsd. with cisapride.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoNimFBPcs477Vg90H21EOLACvtfcHk0lhGrfU__UpazA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXnsVCqu7k%253D&md5=291fcfb4f2d9efd92120735a91d27b59</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1152%2Fajpheart.1997.273.5.H2534&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1152%252Fajpheart.1997.273.5.H2534%26sid%3Dliteratum%253Aachs%26aulast%3DMohammad%26aufirst%3DS.%26aulast%3DZhou%26aufirst%3DZ.%26aulast%3DGong%26aufirst%3DQ.%26aulast%3DJanuary%26aufirst%3DC.%2BT.%26atitle%3DBlockage%2520of%2520the%2520HERG%2520human%2520cardiac%2520K%252B%2520channel%2520by%2520the%2520gastrointestinal%2520prokinetic%2520agent%2520cisapride%26jtitle%3DAm.%2520J.%2520Physiol.%26date%3D1997%26volume%3D273%26spage%3DH2534%26epage%3DH2538%26doi%3D10.1152%2Fajpheart.1997.273.5.H2534" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pollard, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skinner, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lazic, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prior, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conlon, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valentin, J.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dota, C.</span></span> <span> </span><span class="NLM_article-title">An analysis of the relationship between preclinical and clinical QT interval-related data</span>. <i>Toxicol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>159</i></span>,  <span class="NLM_fpage">94</span>– <span class="NLM_lpage">101</span>, <span class="refDoi"> DOI: 10.1093/toxsci/kfx125</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=10.1093%2Ftoxsci%2Fkfx125" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=28903488" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A528%3ADC%252BC1cXmtFSgsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=159&publication_year=2017&pages=94-101&author=C.+E.+Pollardauthor=M.+Skinnerauthor=S.+E.+Lazicauthor=H.+M.+Priorauthor=K.+M.+Conlonauthor=J.-P.+Valentinauthor=C.+Dota&title=An+analysis+of+the+relationship+between+preclinical+and+clinical+QT+interval-related+data&doi=10.1093%2Ftoxsci%2Fkfx125"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">An analysis of the relationship between preclinical and clinical QT interval-related data</span></div><div class="casAuthors">Pollard, Christopher E.; Skinner, Matthew; Lazic, Stanley E.; Prior, Helen M.; Conlon, Kelly M.; Valentin, Jean-Pierre; Dota, Corina</div><div class="citationInfo"><span class="NLM_cas:title">Toxicological Sciences</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">159</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">94-101</span>CODEN:
                <span class="NLM_cas:coden">TOSCF2</span>;
        ISSN:<span class="NLM_cas:issn">1096-0929</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">There has been significant focus on drug-induced QT interval prolongation caused by block of the human ether-a-go-corelated gene (hERG)-encoded potassium channel.  Regulatory guidance has been implemented to assess QT interval prolongation risk: preclin. guidance requires a candidate drug potency as a hERG channel blocker to be defined and also its effect on QT interval in a non-rodent species; clin. guidance requires a "Thorough QT Study" during development, although some QT prolonging compds. are identified earlier via a Phase I study.  Clin., heart rate-cor. QT interval (QTc) data on 24 compds. (13 positives; 11 negatives) were compared with their effect on dog QTc and the concn. of compd. causing 50% inhibition (IC50) of hERG current.  Concordance was assessed by calcg. sensitivity and specificity across a range of decision thresholds, thus yielding receiver operating characteristic curves of sensitivity vs. (1-specificity).  The area under the curve of ROC curves (for which 0.5 and 1 indicate chance and perfect concordance, resp.) was used to summarize concordance.  Three aspects of preclin. data were compared with the clin. outcome (receiver operating characteristic area under the curve values shown in brackets): abs. hERG IC50 (0.78); safety margin between hERG IC50 and clin. peak free plasma exposure (0.80); safety margin between QTc effects in dogs and clin. peak free plasma exposure (0.81).  Pos. and neg. predictive values of abs. hERG IIC50 indicated that from an early drug discovery perspective, low potency compds. can be progressed on the basis of a low risk of causing a QTc increase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzer6VhKiwIbVg90H21EOLACvtfcHk0lhatTbJDjCMQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXmtFSgsrc%253D&md5=442a63490fb478512d52828a9d8d0a08</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1093%2Ftoxsci%2Fkfx125&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Ftoxsci%252Fkfx125%26sid%3Dliteratum%253Aachs%26aulast%3DPollard%26aufirst%3DC.%2BE.%26aulast%3DSkinner%26aufirst%3DM.%26aulast%3DLazic%26aufirst%3DS.%2BE.%26aulast%3DPrior%26aufirst%3DH.%2BM.%26aulast%3DConlon%26aufirst%3DK.%2BM.%26aulast%3DValentin%26aufirst%3DJ.-P.%26aulast%3DDota%26aufirst%3DC.%26atitle%3DAn%2520analysis%2520of%2520the%2520relationship%2520between%2520preclinical%2520and%2520clinical%2520QT%2520interval-related%2520data%26jtitle%3DToxicol.%2520Sci.%26date%3D2017%26volume%3D159%26spage%3D94%26epage%3D101%26doi%3D10.1093%2Ftoxsci%2Fkfx125" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, L.</span></span> <span> </span><span class="NLM_article-title">Design, synthesis and biological evaluation of novel indazole-based derivatives as potent HDAC inhibitors via fragment-based virtual screening</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>192</i></span>,  <span class="NLM_fpage">112189</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2020.112189</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=10.1016%2Fj.ejmech.2020.112189" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=32151834" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A528%3ADC%252BB3cXksFels7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=192&publication_year=2020&pages=112189&author=J.+Liuauthor=J.+Zhouauthor=F.+Heauthor=L.+Gaoauthor=Y.+Wenauthor=L.+Gaoauthor=P.+Wangauthor=D.+Kangauthor=L.+Hu&title=Design%2C+synthesis+and+biological+evaluation+of+novel+indazole-based+derivatives+as+potent+HDAC+inhibitors+via+fragment-based+virtual+screening&doi=10.1016%2Fj.ejmech.2020.112189"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Design, synthesis and biological evaluation of novel indazole-based derivatives as potent HDAC inhibitors via fragment-based virtual screening</span></div><div class="casAuthors">Liu, Jian; Zhou, Jingxian; He, Fengjun; Gao, Liang; Wen, Yu; Gao, Lina; Wang, Ping; Kang, Di; Hu, Lihong</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">192</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">112189</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Based on fragment-based virtual screening and bioisosterism strategies, novel indazoles I [R = C6H5, 4-C5H4N, 3-C4H3S etc.; X = C; Y = (CH2)3, (CH2)4, (CH2)6, etc.] and pyrazolo[3,4-b] pyridine derivs. I [R = 3-MeO-C6H4, 4-MeO-C6H4; X = N; Y = (CH2)6] as HDACs inhibitors were synthesized and evaluated.  Most of these compds. displayed good to excellent inhibitory activities against HDACs, esp. compds. I [R = 4-MeO-C6H4, 4-Cl-C6H4; X = C; Y = (CH2)6] were identified as potent inhibitors of HDAC1 (IC50 = 2.7 nM and IC50 = 3.1 nM), HDAC2 (IC50 = 4.2 nM and IC50 = 3.6 nM) and HDAC8 (IC50 = 3.6 nM and IC50 = 3.3 nM).  Further anti-proliferation assays revealed that compds. I [R = 4-MeO-C6H4, 4-Cl-C6H4; X = C; Y = (CH2)6] showed better anti-proliferative activities against HCT-116 and HeLa cells than pos. control SAHA.  The western blot anal. results indicated that compds. I [R = 4-MeO-C6H4, 4-Cl-C6H4; X = C; Y = (CH2)6] noticeably up-regulated the level of acetylated α-tubulin and histone H3.  In addn., the two compds. I [R = 4-MeO-C6H4, 4-Cl-C6H4; X = C; Y = (CH2)6] could arrest cell cycle in G2/M phase and promote cell apoptosis, which was similar as the ref. compd. SAHA.  Through the mol. docking and dynamic studies, the potent HDAC inhibitory activities mainly caused by van der Waals and electrostatic interactions with the HDACs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8VNY0R6hLfrVg90H21EOLACvtfcHk0lhatTbJDjCMQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXksFels7w%253D&md5=ad3cca7c8927db6331c1216680e2857b</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2020.112189&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2020.112189%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DZhou%26aufirst%3DJ.%26aulast%3DHe%26aufirst%3DF.%26aulast%3DGao%26aufirst%3DL.%26aulast%3DWen%26aufirst%3DY.%26aulast%3DGao%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DP.%26aulast%3DKang%26aufirst%3DD.%26aulast%3DHu%26aufirst%3DL.%26atitle%3DDesign%252C%2520synthesis%2520and%2520biological%2520evaluation%2520of%2520novel%2520indazole-based%2520derivatives%2520as%2520potent%2520HDAC%2520inhibitors%2520via%2520fragment-based%2520virtual%2520screening%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2020%26volume%3D192%26spage%3D112189%26doi%3D10.1016%2Fj.ejmech.2020.112189" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Miller, B. R.,  3rd</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGee, T. D.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swails, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Homeyer, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gohlke, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roitberg, A. E.</span></span> <span> </span><span class="NLM_article-title">MMPBSA.py: An efficient program for end-state free energy calculations</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">3314</span>– <span class="NLM_lpage">3321</span>, <span class="refDoi"> DOI: 10.1021/ct300418h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct300418h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtV2gtrzP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2012&pages=3314-3321&author=B.+R.+Millerauthor=T.+D.+McGeeauthor=J.+M.+Swailsauthor=N.+Homeyerauthor=H.+Gohlkeauthor=A.+E.+Roitberg&title=MMPBSA.py%3A+An+efficient+program+for+end-state+free+energy+calculations&doi=10.1021%2Fct300418h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">MMPBSA.py: An Efficient Program for End-State Free Energy Calculations</span></div><div class="casAuthors">Miller, Bill R., III; McGee, T. Dwight, Jr.; Swails, Jason M.; Homeyer, Nadine; Gohlke, Holger; Roitberg, Adrian E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3314-3321</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">MM-PBSA is a post-processing end-state method to calc. free energies of mols. in soln.  MMPBSA.py is a program written in Python for streamlining end-state free energy calcns. using ensembles derived from mol. dynamics (MD) or Monte Carlo (MC) simulations.  Several implicit solvation models are available with MMPBSA.py, including the Poisson-Boltzmann Model, the Generalized Born Model, and the Ref. Interaction Site Model.  Vibrational frequencies may be calcd. using normal mode or quasi-harmonic anal. to approx. the solute entropy.  Specific interactions can also be dissected using free energy decompn. or alanine scanning.  A parallel implementation significantly speeds up the calcn. by dividing frames evenly across available processors.  MMPBSA.py is an efficient, user-friendly program with the flexibility to accommodate the needs of users performing end-state free energy calcns.  The source code can be downloaded at http://ambermd.org/ with AmberTools, released under the GNU General Public License.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZK3C1jqiSULVg90H21EOLACvtfcHk0lhatTbJDjCMQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtV2gtrzP&md5=cc4148bd8f70c7cad94fd3ec6f580e52</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1021%2Fct300418h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct300418h%26sid%3Dliteratum%253Aachs%26aulast%3DMiller%26aufirst%3DB.%2BR.%26aulast%3DMcGee%26aufirst%3DT.%2BD.%26aulast%3DSwails%26aufirst%3DJ.%2BM.%26aulast%3DHomeyer%26aufirst%3DN.%26aulast%3DGohlke%26aufirst%3DH.%26aulast%3DRoitberg%26aufirst%3DA.%2BE.%26atitle%3DMMPBSA.py%253A%2520An%2520efficient%2520program%2520for%2520end-state%2520free%2520energy%2520calculations%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2012%26volume%3D8%26spage%3D3314%26epage%3D3321%26doi%3D10.1021%2Fct300418h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5ACC" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5ACC','PDB','5ACC'); return false;">PDB: 5ACC</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5AVV" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5AVV','PDB','5AVV'); return false;">PDB: 5AVV</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5acc" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5acc','PDB','5acc'); return false;">PDB: 5acc</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5AAV" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5AAV','PDB','5AAV'); return false;">PDB: 5AAV</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6B0F" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6B0F','PDB','6B0F'); return false;">PDB: 6B0F</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i110"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00280">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_72026"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.1c00280?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00280</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Table S1 listing the UPLC-UV/Q-TOF MS analysis results of compound <b>D24</b> metabolites, Table S2 listing compound <b>D24</b> toxicity data, Figure S1 showing <b>D24</b> putative metabolic pathway in liver cells; spectra for the target compounds (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00280/suppl_file/jm1c00280_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00280/suppl_file/jm1c00280_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00280/suppl_file/jm1c00280_si_001.pdf">jm1c00280_si_001.pdf (3.3 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00280/suppl_file/jm1c00280_si_002.csv">jm1c00280_si_002.csv (5.53 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.1c00280&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2021.64.issue-11%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.1c00280%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.1c00280" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                3MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799769cc95a24f5","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
